
<html lang="en"     class="pb-page"  data-request-id="014e69d8-f284-4ece-84bb-20ca343158c3"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.0c01118;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2020.63.issue-22"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structural and Functional Study of the Klebsiella pneumoniae VapBC Toxin–Antitoxin System, Including the Development of an Inhibitor That Activates VapC" /></meta><meta name="dc.Creator" content="Sung-Min  Kang" /></meta><meta name="dc.Creator" content="Chenglong  Jin" /></meta><meta name="dc.Creator" content="Do-Hee  Kim" /></meta><meta name="dc.Creator" content="Yuno  Lee" /></meta><meta name="dc.Creator" content="Bong-Jin  Lee" /></meta><meta name="dc.Description" content="Klebsiella pneumoniae is one of the most critical opportunistic pathogens. TA systems are promising drug targets because they are related to the survival of bacterial pathogens. However, structural..." /></meta><meta name="Description" content="Klebsiella pneumoniae is one of the most critical opportunistic pathogens. TA systems are promising drug targets because they are related to the survival of bacterial pathogens. However, structural..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 4, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01118" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01118" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01118" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01118" /></link>
        
    
    

<title>Structural and Functional Study of the Klebsiella pneumoniae VapBC Toxin–Antitoxin System, Including the Development of an Inhibitor That Activates VapC | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01118" /></meta><meta property="og:title" content="Structural and Functional Study of the Klebsiella pneumoniae VapBC Toxin–Antitoxin System, Including the Development of an Inhibitor That Activates VapC" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/large/jm0c01118_0007.jpeg" /></meta><meta property="og:description" content="Klebsiella pneumoniae is one of the most critical opportunistic pathogens. TA systems are promising drug targets because they are related to the survival of bacterial pathogens. However, structural information on TA systems in K. pneumoniae remains lacking; therefore, it is necessary to explore this information for the development of antibacterial agents. Here, we present the first crystal structure of the VapBC complex from K. pneumoniae at a resolution of 2.00 Å. We determined the toxin inhibitory mechanism of the VapB antitoxin through an Mg2+ switch, in which Mg2+ is displaced by R79 of VapB. This inhibitory mechanism of the active site is a novel finding and the first to be identified in a bacterial TA system. Furthermore, inhibitors, including peptides and small molecules, that activate the VapC toxin were discovered and investigated. These inhibitors can act as antimicrobial agents by disrupting the VapBC complex and activating VapC. Our comprehensive investigation of the K. pneumoniae VapBC system will help elucidate an unsolved conundrum in VapBC systems and develop potential antimicrobial agents." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01118"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01118">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01118&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01118&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01118&amp;href=/doi/10.1021/acs.jmedchem.0c01118" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 22</span><span class="cit-fg-pageRange">, 13669-13679</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/22" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00975" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01201" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structural and Functional Study of the <i>Klebsiella pneumoniae</i> VapBC Toxin–Antitoxin System, Including the Development of an Inhibitor That Activates VapC</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Sung-Min Kang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sung-Min Kang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Research Institute of Pharmaceutical Sciences, College of Pharmacy, Seoul National University, Seoul 08826, Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sung-Min++Kang">Sung-Min Kang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chenglong Jin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chenglong Jin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Research Institute of Pharmaceutical Sciences, College of Pharmacy, Seoul National University, Seoul 08826, Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chenglong++Jin">Chenglong Jin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Do-Hee Kim</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Do-Hee Kim</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy, Jeju National University, Jeju 63243, Korea</div><div class="loa-info-affiliations-info">Interdisciplinary Graduate Program in Advanced Convergence Technology & Science, Jeju National University, Jeju 63243, Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Do-Hee++Kim">Do-Hee Kim</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yuno Lee</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yuno Lee</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Korea Chemical Bank, Korea Research Institute of Chemical Technology, Daejeon 34114, Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yuno++Lee">Yuno Lee</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Bong-Jin Lee</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bong-Jin Lee</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Research Institute of Pharmaceutical Sciences, College of Pharmacy, Seoul National University, Seoul 08826, Korea</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#1a7678705a7477683469746f347b79347168"><span class="__cf_email__" data-cfemail="abc7c9c1ebc5c6d985d8c5de85cac885c0d9">[email protected]</span></a>. Phone: +82 2 880 7869. Fax: +82 2 872 3632.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bong-Jin++Lee">Bong-Jin Lee</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7896-2961" title="Orcid link">http://orcid.org/0000-0001-7896-2961</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01118&amp;href=/doi/10.1021%2Facs.jmedchem.0c01118" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 22</span><span class="cit-pageRange">, 13669–13679</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 4, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>30 June 2020</li><li><span class="item_label"><b>Published</b> online</span>4 November 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 November 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01118" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01118</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D13669%26pageCount%3D11%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DSung-Min%2BKang%252C%2BChenglong%2BJin%252C%2BDo-Hee%2BKim%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D22%26contentID%3Dacs.jmedchem.0c01118%26title%3DStructural%2Band%2BFunctional%2BStudy%2Bof%2Bthe%2BKlebsiella%2Bpneumoniae%2BVapBC%2BToxin%25E2%2580%2593Antitoxin%2BSystem%252C%2BIncluding%2Bthe%2BDevelopment%2Bof%2Ban%2BInhibitor%2BThat%2BActivates%2BVapC%26numPages%3D11%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D13679%26publicationDate%3DNovember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01118"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">693</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">2</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01118" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structural and Functional Study of the Klebsiella pneumoniae VapBC Toxin–Antitoxin System, Including the Development of an Inhibitor That Activates VapC&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Sung-Min&quot;,&quot;last_name&quot;:&quot;Kang&quot;},{&quot;first_name&quot;:&quot;Chenglong&quot;,&quot;last_name&quot;:&quot;Jin&quot;},{&quot;first_name&quot;:&quot;Do-Hee&quot;,&quot;last_name&quot;:&quot;Kim&quot;},{&quot;first_name&quot;:&quot;Yuno&quot;,&quot;last_name&quot;:&quot;Lee&quot;},{&quot;first_name&quot;:&quot;Bong-Jin&quot;,&quot;last_name&quot;:&quot;Lee&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;04&quot;,&quot;issue&quot;:&quot;22&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;13669-13679&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01118&quot;},&quot;abstract&quot;:&quot;Klebsiella pneumoniae is one of the most critical opportunistic pathogens. TA systems are promising drug targets because they are related to the survival of bacterial pathogens. However, structural information on TA systems in K. pneumoniae remains lacking; therefore, it is necessary to explore this information for the development of antibacterial agents. Here, we present the first crystal structure of the VapBC complex from K. pneumoniae at a resolution of 2.00 Å. We determined the toxin inhibitory mechanism of the VapB antitoxin through an Mg2+ switch, in which Mg2+ is displaced by R79 of VapB. This inhibitory mechanism of the active site is a novel finding and the first to be identified in a bacterial TA system. Furthermore, inhibitors, including peptides and small molecules, that activate the VapC toxin were discovered and investigated. These inhibitors can act as antimicrobial agents by disrupting the VapBC complex and activating VapC. Our comprehensive investigation of the K. pneumoniae VapBC system wil&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01118&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01118" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01118&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01118" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01118&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01118" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01118&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01118&amp;href=/doi/10.1021/acs.jmedchem.0c01118" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01118" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01118" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01118%26sid%3Dliteratum%253Aachs%26pmid%3D33146528%26genre%3Darticle%26aulast%3DKang%26date%3D2020%26atitle%3DStructural%2Band%2BFunctional%2BStudy%2Bof%2Bthe%2BKlebsiella%2Bpneumoniae%2BVapBC%2BToxin%25E2%2580%2593Antitoxin%2BSystem%252C%2BIncluding%2Bthe%2BDevelopment%2Bof%2Ban%2BInhibitor%2BThat%2BActivates%2BVapC%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D22%26spage%3D13669%26epage%3D13679%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290817" title="Interfaces">Interfaces</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=292276" title="Small molecules">Small molecules</a>,</li><li><a href="/action/doSearch?ConceptID=291735" title="Toxins">Toxins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/22" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/jmcmar.2020.63.issue-22/20201125/jmcmar.2020.63.issue-22.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/medium/jm0c01118_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/large/jm0c01118_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01118&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText"><i>Klebsiella pneumoniae</i> is one of the most critical opportunistic pathogens. TA systems are promising drug targets because they are related to the survival of bacterial pathogens. However, structural information on TA systems in <i>K. pneumoniae</i> remains lacking; therefore, it is necessary to explore this information for the development of antibacterial agents. Here, we present the first crystal structure of the VapBC complex from <i>K. pneumoniae</i> at a resolution of 2.00 Å. We determined the toxin inhibitory mechanism of the VapB antitoxin through an Mg<sup>2+</sup> switch, in which Mg<sup>2+</sup> is displaced by R79 of VapB. This inhibitory mechanism of the active site is a novel finding and the first to be identified in a bacterial TA system. Furthermore, inhibitors, including peptides and small molecules, that activate the VapC toxin were discovered and investigated. These inhibitors can act as antimicrobial agents by disrupting the VapBC complex and activating VapC. Our comprehensive investigation of the <i>K. pneumoniae</i> VapBC system will help elucidate an unsolved conundrum in VapBC systems and develop potential antimicrobial agents.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">1.  Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86254" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86254" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p"><i>Klebsiella pneumoniae</i> is one of the most critical opportunistic pathogens. It is widespread in the environment, including in water, on solids or on leaf surfaces. <i>K. pneumoniae</i> is responsible for infections of the upper respiratory tract, urinary tract, and bloodstream and causes pneumonia.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Currently, <i>K. pneumoniae</i> is considered a significant threat to the health of people worldwide. It possesses an extraordinarily plastic genome and a great tendency to develop antibiotic resistance. In particular, drug-resistant <i>K. pneumoniae</i>, with resistance even to carbapenems, has emerged as a global problem.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a></div><div class="NLM_p">The overall biological functions of toxin–antitoxin (TA) systems cover a wide variety. The initial role identified for a TA system was plasmid maintenance.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Shortly afterward, similar functions of TA systems, such as postsegregational killing, were also observed.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Later, it was shown that the TA systems mediate programmed cell death from stresses such as antibiotic treatment, phage infection, DNA damage, oxidative stress, and high temperature.<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a> Recently, it has been shown that TA systems are likely to function in the formation of bacterial persister cells.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a></div><div class="NLM_p">The four major types of TA systems are classified by the mechanism by which the antitoxin inhibits the toxin and by the nature of the antitoxin. In type I and III TA systems, antitoxins are RNA molecules that regulate the cellular levels of toxins. An RNA antitoxin inhibits the translation of an mRNA toxin (type I) and inhibits the activity of a protein toxin by direct binding (type III). However, antitoxins in type II and IV TA systems are proteins. These protein antitoxins bind directly to a protein toxin and inhibit the activity of the toxin (type II) or neutralize the toxic activity of the toxin without direct binding, such as by triggering opposite effects to the target (type IV). There are only single cases of type V and VI TA systems, each of which has a different regulatory principle.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a></div><div class="NLM_p">The bacterial type II TA system contains two proteins, a biophysically stable toxin and a relatively labile antitoxin. In the normal state, the expression levels of the toxin and antitoxin are well-harmonized, so bacteria can normally survive because the antitoxin counteracts any harmful effect of the toxin.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> However, the labile antitoxin degrades faster under stress conditions, including antibiotic treatment, nutrient starvation, and unfavorable temperature. Then, the toxin is freely released from the antitoxin, and the bacteria suffer damage because of the toxicity of the released toxin.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a></div><div class="NLM_p">Type II TA systems are more broadly distributed in pathogenic bacteria than in nonpathogenic bacteria.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> This suggests that TA systems are closely related to bacterial pathogenicity. Additionally, TA systems are not found in any eukaryotes and are therefore a promising drug target for bacterial pathogens.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Dozens of structures from the TA systems of pathogenic bacteria have been solved in structural and functional studies; however, the structural information on TA systems in <i>K. pneumoniae</i> is still unknown.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> To gain molecular insights into and validate the <i>K. pneumoniae</i> TA system as a future drug target, it is essential to explore the structure of <i>Klebsiella</i> pneumonia TA systems.</div><div class="NLM_p">VapBC is one of the largest families of type II TA systems. To date, at least 11 complex structures have been identified in the VapBC TA system: <i>Neisseria gonorrheae</i> FitAB,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a><i>Shigella flexneri</i> VapBC;<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a><i>Rickettsia felis</i> VapBC2;<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a><i>Caulobacter crescentus</i> VapC1;<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a><i>Haemophilus influenza</i> VapC1;<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> and <i>Mycobacterium tuberculosis</i> VapBC3,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> VapBC5,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> VapBC11,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> VapBC15,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> VapC26,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and VapBC30.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> These VapBC complexes highlight the great structural complexity of VapB, which contains a variable DNA-binding domain and flexible region, whereas VapC exhibits a conserved rigid domain.</div><div class="NLM_p">The VapC toxin contains a pilus retraction protein (PilT) N-terminal (PIN) domain, which is crucial for ribonuclease (RNase) activity. These PIN domains comprise approximately 130 amino acid proteins and contain a strictly conserved negatively charged acidic quartet of residues coordinated by Mg<sup>2+</sup> in the active site. On the other hand, the cognate antitoxin VapB folds into a ribbon–helix–helix (RHH) DNA-binding motif at the N-terminus and wraps the active site of VapC via its C-terminal helices.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> As VapB sterically blocks the catalytic site of VapC and acts as a tight binding inhibitor, the molecules that inhibit the binding between VapB and VapC can generate and release the free toxin. There are previous reports about these inhibitory molecules acting as binding competitors, leading to growth arrest or ultimately cell death in other pathogenic bacteria.<a onclick="showRef(event, 'ref23 ref25'); return false;" href="javascript:void(0);" class="ref ref23 ref25">(23,25)</a> The exploitation of TA systems as an antibacterial strategy via artificial activation of the toxin has been proposed as a potential approach for antibiotics therapy.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> In some recent studies, sequence-specific antisense peptide nucleic acid oligomers and peptide-based inhibitors led to bacterial cell death, highlighting this strategy.<a onclick="showRef(event, 'ref21 ref27'); return false;" href="javascript:void(0);" class="ref ref21 ref27">(21,27)</a></div><div class="NLM_p last">Here, we present the first crystal structure of the VapBC complex from <i>K. pneumoniae</i> at a resolution of 2.00 Å. First, through structural analysis, we found that R79 of VapB displaces the coordinated Mg<sup>2+</sup> from the neighboring D9, E43, and D90 of VapC to D9, D90, and D111 of VapC and abolishes the RNase activity of VapC. This finding was further verified by site-directed mutagenesis and <i>in vitro</i> RNase activity experiments. We designed peptides mimicking the binding interface of the VapBC complex and confirmed the complex disruption ability of peptides by the successful generation of RNase activity of the VapC toxin. Additionally, using the structural information, a total of 400 small molecules were screened. The final small-molecule candidate is thought to be a potent structure-based inhibitor with complex disrupting efficacy. This comprehensive investigation of the <i>K. pneumoniae</i> VapBC system will help us to decipher unsolved conundrums in VapBC systems and develop potential antimicrobial agents against <i>K. pneumoniae</i>.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">2.  Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86056" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86056" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4">2.1.  Overall Information on the VapBC Complex</h3><div class="NLM_p">First, we validated the <i>K. pneumonia</i> VapBC TA system experimentally. VapC expression resulted in considerable restraint of cell growth and a decrease in the colony-forming unit (cfu). However, suppressed cell viability was almost fully rescued by the expression of VapB (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_001.pdf" class="ext-link">Figure S1</a>). The bacterium used for validation of the <i>Klebsiella</i> pneumonia VapBC TA system is <i>Escherichia coli</i>. To solve the structure of VapBC, the single-wavelength anomalous diffraction (SAD) data for the SeMet-labeled VapBC complex were refined [Protein Data Dank (PDB) ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY2">7BY2</a>] and used to solve the phase problem. Then, the native data were refined (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY3">7BY3</a>) by molecular replacement employing the structure of the SeMet-labeled VapBC complex as the search model.</div><div class="NLM_p">As a result of the structural calculation of <i>K. pneumoniae</i> VapBC from native data, the overall conformation of the VapBC complex was obtained. The VapB antitoxin is divided into an N-terminal part containing a DNA-binding domain and a C-terminal part containing a toxin-binding region, which blocks the active site of the VapC toxin. Interestingly, the phased structure of VapBC from the SAD data showed an incomplete complex structure in which the C-terminus of VapB was digested; thus, the active site of VapC was not inhibited by VapB. Detailed new findings related to this observation will be described in the third subsection of the <a class="ref internalNav" href="#sec2" aria-label="Results and Discussion">Results and Discussion</a> section.</div><div class="NLM_p">In the crystal structures, we observed two types of intermolecular interfaces, which originated from crystal packing. The first is a toxin dimerization-driven heterotetramer (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). In this case, the interaction between α-helices in the adjacent toxin monomers is a main binding force. The other is an antitoxin dimerization-driven heterotetramer (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B), which is mediated by the DNA-binding domain of the antitoxin.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/medium/jm0c01118_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/large/jm0c01118_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Overall structural information on <i>K. pneumoniae</i> VapBC (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY3">7BY3</a>). (A–C) Antitoxin VapB subunits are shown in yellow and the toxin VapC subunits are shown in light purple. (A) Heterotetrameric assembly of the VapBC complex containing an intermolecular interface formed by the dimerization of VapC. (B) Heterotetrameric assembly of the VapBC complex containing an intermolecular interface formed by the dimerization of the N-terminal DNA-binding domain of VapB. The DNA-binding domain of VapB is labeled. (C) Schematic diagram of the VapBC heterodimer showing the secondary structural architecture. (D) Secondary structure analysis of the N-terminal DNA-binding domain of VapB. Prediction data using the PSIPRED server are presented with structural information on the N-terminus of VapB in the crystal structure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/large/jm0c01118_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01118&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">A schematic diagram of the VapBC heterodimer is helpful for understanding the spatial arrangement of the secondary structural elements that make up the heterodimeric architecture (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C). In our native structure, the first β-strand of the VapB N-terminus was not observed because of the poor electron density in the map. According to the prediction by PSIPRED,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> VapB is expected to have an RHH DNA-binding domain<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> in the order β1−α1−α2 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D) at the N-terminus.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5">2.2.  Structural Insights into <i>K. pneumoniae</i> VapBC Compared with Its Homologues</h3><div class="NLM_p">To identify the structural characteristics of <i>K. pneumoniae</i> VapBC, a comparative analysis was performed with its structural homologues, and the complex structure was solved. To reflect the latest information, comparisons were made with relatively recently reported homologues since 2017. First, an amino acid sequence alignment of the VapC toxins using Clustal Omega 1.2.1<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> and ESPript 3.0<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> was carried out. In this alignment, the primary sequences of <i>M. tuberculosis</i> VapC11 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6A7V">6A7V</a>)<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and VapC26 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5X3T">5X3T</a>),<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a><i>C. crescentus</i> VapC1 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K8J">5K8J</a>),<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> and <i>Hemophilus</i> influenza VapC1 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6NKL">6NKL</a>)<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> were compared with that of <i>K. pneumoniae</i> VapC. The alignment revealed seven highly conserved residues (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). The first, third, fourth, and sixth conserved residues are negatively charged and acidic residues, such as aspartate and glutamate, which make up the active site of the toxin. Among the remaining three residues, the locations of the second threonine and seventh phenylalanine are closely related to the location of the active site, and hydrogen bonding and hydrophobic interactions between these residues and the active-site residues support and contribute to the building and structural integrity of the active site (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/medium/jm0c01118_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/large/jm0c01118_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structural comparison of <i>K. pneumoniae</i> VapBC with its homologues. (A) Sequence alignment of <i>K. pneumoniae</i> VapC with its homologues. The secondary structural elements of these proteins are shown above the alignment. Highly conserved residues are highlighted in red and yellow. Among them, the acidic active-site residues and residual residues are marked as red and blue triangles, respectively. (B) Electrostatic surface of VapC (left) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY3">7BY3</a>). Positive and negative charges are colored blue and red, respectively. An acidic active site of VapC is shown enlarged in cartoon and stick representations (right) showing the locations of seven conserved residues in the structure of each homologue. Seven conserved residues are shown in stick representation. (C) Comparison of the VapBC complexes (PDB IDs: in the order of <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY3">7BY3</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6A7V">6A7V</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5X3T">5X3T</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K8J">5K8J</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6NKL">6NKL</a>). Each VapBC complex is shown in the same direction based on the VapC structure. For VapC, the same colors as in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B are used.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/large/jm0c01118_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01118&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Despite the low sequence similarity (13.7% on average) with the four homologues, <i>K. pneumoniae</i> VapC shares structural similarity with the other VapC proteins. VapC toxins adopt a canonical α/β/α sandwich fold composed of five parallel β-strands flanking seven α-helices. VapC toxins also similarly feature the PIN domain, which is essential for ribonuclease activity.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The conserved acidic residues in PIN domain proteins coordinate with divalent metals and trigger catalytic activity based on acid–base reactions.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></div><div class="NLM_p last">The four homologues were also compared with <i>K. pneumoniae</i> VapBC to determine the differences in the VapB structures and VapB- and VapC-binding patterns (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C). In comparison, the VapC structures had similar folds; however, the binding patterns of the VapC toxin with its cognate VapB and the VapB DNA-binding domains were quite different. The C-terminal toxin-binding region of VapB is intrinsically extremely flexible; thus, structural folding and integrity are achieved only after binding to the toxin.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> For this reason, the α-helical contents and the loop contents of each VapB in each binding region between these VapB antitoxins and VapC toxins seem to have large differences. In addition, there are two different DNA-binding domains in VapB, including the RHH<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> and AbrB-type<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> domains. Interestingly, DNA-binding domains in VapB differ not only in their type of domain but also within the same domain type, and the orientation of the domain varies considerably depending on the curve at the hinge region. This emphasizes that the structural variability of VapB proteins causes a diversity of transcription regulation mechanisms and further explains the specificity in the recognition of their own promoter regions.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6">2.3.  RNase Activity of VapC and Unveiling the Inhibition Mechanism of VapB</h3><div class="NLM_p">As mentioned in the first subsection, the structure obtained from the SAD data is in an open form with an active site exposed because of the natural digestion of the C-terminus of VapB (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). On the other hand, the native structure obtained from the native crystal shows a closed form with an active site in which the toxin-binding region of VapB is completely preserved (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). In accordance with the innate nature of VapC, its RNase activity is only shown when divalent metal ions such as Mg<sup>2+</sup> are present. <i>K. pneumoniae</i> VapC showed RNase activity only when Mg<sup>2+</sup> was added to the protein solution. In the open form without VapB interference in the active site, Mg<sup>2+</sup> is located in the midst of the conserved active-site residues (D9, E43, and D90) of VapC. However, in the closed form, VapB covers the active site of the toxin. R79 of VapB exactly blocks the original Mg<sup>2+</sup> site and displaces Mg<sup>2+</sup> to the center of residues D9, D90, and D111 of VapC, causing the active site to switch from the open state to the closed state (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). This inhibitory mechanism of the active site is a novel TA system mechanism. This inhibitory mechanism of the active site is first observed through two states of crystal structures.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/medium/jm0c01118_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/large/jm0c01118_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Active site and RNase activities of VapC. (A) Two forms of the VapBC complex structure: open form (left) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY2">7BY2</a>) and closed form (right) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY3">7BY3</a>). (B) Two forms of the active site are presented with a cartoon diagram of VapC. Four conserved residues are shown in stick representation. Active-site conformations are denoted as black dotted squares and enlarged. The 2m<i>Fo</i>-<i>DFc</i> electron density map of the Mg<sup>2+</sup> site contouring at σ (calculated with PHENIX) is also displayed. In the open form of the VapC active site, the coordinated Mg<sup>2+</sup> is located in the midst of residues D9, E43, and D90. In the closed form of the VapC active site, R79 of VapB triggers the displacement of Mg<sup>2+</sup>. Consequently, R79 of VapB fills the original location of Mg<sup>2+</sup>, and Mg<sup>2+</sup> moves to the middle of D9, D90, and D111. (C) <i>In vitro</i> RNase activities of VapC, VapC mutants, and VapBC. Proteins were treated with EDTA to remove metal ions, and 10 mM Mg<sup>2+</sup> was then added to proteins. In the assay, 10 μM proteins were used, and a general RNase inhibitor (40 units) was used to prevent unintended contamination. The kinetics for the RNase activities of these proteins were recorded for 1 h. The data shown are representative of three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/large/jm0c01118_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01118&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To expand the importance of these results, we mutated each residue comprising the active site of VapC and measured the RNase activities of each VapC mutant (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). From the graphs, which express the kinetics for 1 h and show a proportional plot for VapC, the residue with the largest contribution to activity was E43, and the residue with the least contribution was D111. In conclusion, as R79 of VapB causes the displacement of Mg<sup>2+</sup>, VapC active-site inhibition is achieved by completely abolishing the contribution of E43 to the RNase activity. However, based on the mutation experiments, when E43 of VapC is mutated to alanine, VapC still has approximately 40% of its RNase activity. This suggests that the RNase activity of VapC depends on the position of Mg<sup>2+</sup>, which could be displaced by VapB, as well as the contribution of each residue constituting the active site.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7">2.4.  Exploration of Peptides That Activate VapC by Disrupting the Interface between VapB and VapC</h3><div class="NLM_p">If the formation of the TA complex is artificially inhibited, toxins freely released from the TA complex can arrest the growth or eventually cause the death of bacteria.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Therefore, to design peptides capable of disrupting the <i>K. pneumoniae</i> VapBC complex, the interaction networks between VapB and VapC were analyzed based on the results of analyses from the proteins, interfaces, structures, and assemblies (PISA)<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> and protein interaction calculator (PIC)<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> servers. The results showed that the α3 helix of VapB (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A) and α1 helix of VapC (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B) were most important in the dimerization interaction between VapB and VapC.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/medium/jm0c01118_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/large/jm0c01118_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Heterodimeric interface of the VapBC complex (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY3">7BY3</a>) and efficacy of complex disruption using the designed peptides. (A,B) Residues involved in interaction networks are shown in stick representation. Schematic diagrams of the hydrophilic and hydrophobic interactions are provided in the table. (A) Intermolecular interaction network of the VapB α3 helix. (B) Intermolecular interaction network of the VapC α1 helix. (C,D) RNase activities resulting from peptides added to the VapBC complex. Activities generated by peptides are plotted in proportion to that of VapC. The pretreatments for the assay were the same as those described in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C. In these complex disruption assays, 100 μM peptides were added to 10 μM proteins, and their complex disruption efficacies were evaluated. Data are represented as the mean of three independent replicates ±SD. (C) Efficacy of peptides designed based on the sequence of the α3 helix of VapB. (D) Efficacy of peptides designed based on the sequence of the α1 helix of VapC.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/large/jm0c01118_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01118&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Therefore, several peptides were designed based on the sequence of the α3 helix of VapB and the α1 helix of VapC (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_001.pdf" class="ext-link">Table S3</a>). When a peptide is added to the VapBC complex, the inhibition efficacy of the peptide can be estimated by measuring the RNase activity generated by the free VapC, which is released by the disruption of the VapBC complex. In the complex disruption assay, although the α3 helix of VapB consists of G71–K80 (GMEYQRQLRK), the VapB<sup>71–78</sup> peptide consisting of G71–L78 (GMEYQRQL), which excludes R79 of VapB that interacts with three active-site residues (D9, E43, and D90) of VapC, showed high efficacy. Y74 and Q75 of VapB interact with E43 and D9 of VapC, respectively. As a result, when Y74 and Q75 were individually mutated to alanine, the VapB<sup>71–78, Y74A</sup> peptide showed the highest efficacy among all of the designed peptides. The VapB<sup>71–78, Y74A, Q75A</sup> peptide, in which both Y74 and Q75 are mutated to an alanine, showed a lower efficacy than the individual alanine-mutated VapB peptides. This might be due to defects in the structural integrity of the α helix because of multiple alanine substitutions (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C).</div><div class="NLM_p">When free VapC was treated with VapB-mimicking peptides, the overall RNase activity measured was slightly increased, but the pattern was similar to that resulting from VapBC treated with peptides (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_001.pdf" class="ext-link">S2A</a>). Because the designed peptides are made by mimicking the binding interface of VapB and VapC, arithmetically, the more residues that participate in the binding of VapB and VapC are present in the peptides, the greater the complex disruption capability should be. However, if a peptide contains the residues that interact with the nucleic acid-binding site of VapC, which is the active site of VapC, the disruption capability is inhibited. Moreover, the lack of structural integrity due to multiple alanine substitutions (both Y74A and Q75A) had little effect on the activity of VapC. In summary, among the residues involved in the binding of VapB and VapC included in the peptide, the more residues in the peptide that interact with the residues constituting the active site of VapC, the more the activity of VapC is unintentionally inhibited. In particular, as shown in the mutational study, this phenomenon is more pronounced when the peptide contains a residue that interacts with D43, which is most related with the activity of VapC.</div><div class="NLM_p last">In the VapBC complex structure, the α1 helix of VapC is composed of T10–F17 (TNILIDLF). Although D9 and S18 are not included in the α1 helix of VapC, the VapC<a onclick="showRef(event, 'ref9 ref10 ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18">(9−18)</a> peptides, including D9 and S18, which interact with VapB, showed lower efficacy than the peptide consisting of only the α1 helix (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D). As a result, the importance of the α-helical nature in the artificial disruption of the TA complex was reconfirmed.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8">2.5.  Exploration of Small Molecules That Activate VapC by Occluding the Interface Pocket</h3><div class="NLM_p">In parallel, based on the information on the interface between <i>K. pneumoniae</i> VapB and VapC, a total of 400 small-molecule candidates capable of binding to the VapB-binding pocket of the VapC surface were selected by virtual screening. The majority of these compounds are distributed in an MW range of 300–500 Da with an average of ∼5.1 HB acceptors, ∼2.2 HB donors, and 6.8 rotatable bonds. The predicted <i>A</i> log <i>P</i> values for most of the compounds are in the range of 0.5–5, with an average value of 2.8. Each compound was designed to disrupt the VapBC complex by binding not to the active-site cavity of VapC but to the cleft through the α2−α3 loop of VapB, linking the interface pockets (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/medium/jm0c01118_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/large/jm0c01118_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Surface pockets on VapC (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY3">7BY3</a>) and efficacy of final compounds expressed with molecular docking. (A) Surface of VapC with a cartoon representation of VapB. The negatively charged active-site cavity and surface cleft interface pockets are linked through the α2−α3 loop of VapB. The compound-binding site of VapC is located on the surface cleft interface pockets represented as the interface pocket valley. (B) RNase activities of compounds when the VapBC complex was added. Activities generated by compounds are plotted in proportion to that of VapC. The activities of compounds without the VapBC complex were also measured. Pretreatments for these assays were the same as those described in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C. In these complex disruption assays, compound (1, 3.16, and 10 μM) was added to 10 μM protein, and the complex disruption efficacy was evaluated. The activity was measured based on the initial velocity during the first hour. Data are represented as the mean of three independent replicates ±SD. (C,D) Results of molecular docking of the compounds on the VapC structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY3">7BY3</a>) with the detailed interaction networks. (C) Interaction data for compound 1. (D) Interaction data for compound 2. In the cartoon diagram, ligand-binding residues of VapC are shown in stick and labeled. 2D ligand-VapC interaction diagrams were generated using LigPlot<sup>+</sup>.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Compounds 1 and 2 are shown in green and pink, respectively. The residues involved in HBs are shown, and HBs are represented by blue dashes. The residues making nonbonded contacts with the ligand are shown in the red spoked arcs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/large/jm0c01118_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01118&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Using a similar assay method to that in the previous section, the inhibition efficacies of the compounds were ranked according to the amount of RNase activity by the free VapC released by the disruption of the VapBC complex. A total of 400 compounds were narrowed to 25 according to inhibition efficacy ranking through triplicate tests, and these 25 small molecules were further narrowed through additional triplicate testing to the two compounds with the highest efficacy, named compound 1 and compound 2 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). These two final compounds did not affect the RNase activity of VapC when added to free VapC (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_001.pdf" class="ext-link">Figure S2B</a>) because they are designed to bind to the binding interface between VapB and other regions of VapC, which is not the active site of VapC.</div><div class="NLM_p last">These two compounds had a higher efficacy at 1 μM, which is one hundredth of the peptide concentration (100 μM) previously used. Therefore, compounds 1 and 2 are very efficient at disrupting the VapBC complex. Compounds 1 and 2 exhibited IC<sub>50</sub> values of 0.42 and 0.40 μM, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_001.pdf" class="ext-link">Figure S3</a>). For better understanding, the results of molecular docking simulations of compounds 1 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C) and 2 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D) are also presented. The carbonyl group of compound 1 interacted with the positively charged amino group on the side chain of K50 by forming a strong HB. Residues E43, S18, and A48 also formed hydrogen and carbon–hydrogen bond interactions. Residues Y51, I14, and V46 formed π-alkyl and alkyl interactions. Similar to the HB of compound 1, the sulfonyl group of compound 2 was also found to have a strong HB with the positively charged amino group in the side chain of K50. Additionally, residues T10, N11, G47, and A48 formed carbon–hydrogen bonds and amide−π-stacked interactions. Residues I14 and V46 were involved in the formation of π–alkyl and alkyl interactions, respectively. F17 and Y51 formed π–π stacking interactions with the fluorobenzene ring of the compound. In conclusion, we suggest that these two final compounds are potential candidates for controlling drug-resistant <i>K. pneumoniae</i>. This is the first report on small molecules controlling a bacterial TA system.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9">2.6.  Insights into the Possibility of Competitive Inhibitor and Autoregulation Based on Binding Affinity</h3><div class="NLM_p">To further verify the peptides and small molecules selected through this study, the binding affinities of VapC with VapB, peptides, and small molecules were measured by isothermal titration calorimetry (ITC) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_001.pdf" class="ext-link">Figure S4</a>). VapB was most strongly coupled with VapC (<i>K</i><sub>d</sub> = 3 nM), and peptides and small molecules also showed affinities with VapC in the nanomolar range (<i>K</i><sub>d</sub> = 110–170 nM for peptides and 22–35 nM for small molecules) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_001.pdf" class="ext-link">Figure S4</a>). Identifications of these affinities confirm that the peptides were successfully rationally designed as competitive inhibitors for the VapB-binding region and that the proposed docking models of small molecules are reliable.</div><div class="NLM_p">The affinities of the peptides VapB<sup>71–78</sup>, VapB<sup>71–78, Y74A</sup>, and VapB<sup>71–78, Q75A</sup> to VapC were 110 nM, 170 nM, and 150 nM, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_001.pdf" class="ext-link">Figure S4</a>). By mutation of Y74 and Q75, some interactions with VapC disappear and the affinity with VapC decreases. However, Y74 and Q75 from VapB interact with the active site of VapC. Therefore, VapB<sup>71–78, Y74A</sup> and VapB<sup>71–78, Q75A</sup> show higher activities than VapB<sup>71–78</sup> because of the decrease in inhibitions of VapC activity, despite their decreased binding affinity with VapC.</div><div class="NLM_p last">In type II TA systems, the antitoxin and toxin proteins play important roles in transcriptional autoregulation.<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39,40)</a> To investigate the molecular mechanism of transcriptional regulation, the 28-bp DNA sequence at the promoter region was titrated into VapB and VapBC. The affinity of VapBC to DNA (<i>K</i><sub>d</sub> = 1.26 μM) was fivefold higher than that of VapB to DNA (<i>K</i><sub>d</sub> = 5.38 μM) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_001.pdf" class="ext-link">Figure S4</a>). This indicates that VapB serves as a primary repressor in autoregulation by directly binding to its own promoter, while VapC acts as a corepressor, enhancing the interaction between VapB and the operator site by forming a complex with VapB.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">3.  Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63233" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63233" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">The structure of the VapBC complex in this study is the first reported structure of a TA complex in <i>K. pneumoniae</i>. The VapC toxin inhibition mechanism by R79 of VapB displacing Mg<sup>2+</sup> was revealed through two crystal structures, the open and closed forms of the active site. This is a novel finding first identified in the field of bacterial TA systems. Through the rational design of peptides mimicking the VapBC-binding interface, which can disrupt the VapBC complex, the artificial activation of the toxin as an antibacterial strategy can be applied to <i>K. pneumoniae</i>. Furthermore, by discovering small compounds that effectively inhibit the formation of the VapBC complex even at low concentrations, controlling drug-resistant <i>K. pneumoniae</i> through modification with inhibitors may be possible.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">4.  Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88618" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88618" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12">4.1.  Cloning, Protein Purification, and Mutation</h3><div class="NLM_p">For the structural determination of the VapBC complex, the genes encoding <i>K. pneumoniae</i> (strain MGH78578) VapB (<i>kpn_04185</i>) and VapC (<i>kpn_04186</i>) were amplified by PCR using the primers VapB-F/VapB-R<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> and VapC-F/VapC-R (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_001.pdf" class="ext-link">Table S1</a>). The restriction enzymes <i>Nde1</i> and <i>Xho1</i> were used to doubly cleave all of the PCR products and vectors. Then, the cleaved PCR products were ligated to pET-21a (<i>kpn_04185</i>) and pET-28a (<i>kpn_04186</i>). The pET-28a vector, used for VapC (<i>kpn_04186</i>), has an N-terminal tag (MGSSHHHHHHSSGLVPRGSH). The cloned plasmids were cotransformed into competent <i>E. coli</i> BL21(DE3) cells (Novagen). The bacterial cells were grown at 37 °C using Luria broth (LB) until the optical density at 600 nm (OD<sub>600</sub>) reached 0.6. The cells were then induced by 0.5 mM IPTG and further incubated for 4 h at 37 °C. The induced cells were centrifuged at 11,355<i>g</i> and suspended in a buffer containing 20 mM Tris-HCl, pH 7.9, and 500 mM NaCl with 5% glycerol. The suspended cells were lysed by ultrasonication and centrifuged at 28,306<i>g</i>. After centrifugation, the supernatants containing soluble proteins were loaded onto an Ni<sup>2+</sup> affinity open column (Bio-Rad) equilibrated in advance with buffer A. The bound proteins were washed with buffer A containing 100 mM imidazole and eluted by an imidazole gradient (150–500 mM). The eluted proteins were diluted with a buffer containing 20 mM HEPES, pH 7.5, and 100 mM NaCl and further purified using a HiTrap Q HP anion-exchange chromatography column (GE Healthcare) with a NaCl gradient (200–800 mM). These eluted proteins were exchanged with a buffer containing 20 mM HEPES, pH 7.5, and 400 mM NaCl by size-exclusion chromatography on a HiLoad Superdex 200 prep-grade column (GE Healthcare) and concentrated to 20 mg/mL using an Amicon Ultra Centrifugal Filter Unit (Millipore).</div><div class="NLM_p">For the purification of the selenomethionine (SeMet)-labeled VapBC complex, the same procedure described above was used, except that the cells were grown in SeMet Medium Base plus Nutrient Mix (Molecular Dimensions) containing extra SeMet.</div><div class="NLM_p last">To create VapC mutations, forward (F) and reverse (R) primers for D9A, E43A, D90A, and D111A were used (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_001.pdf" class="ext-link">Table S1</a>). The mutations were created using the EZchange Site-Directed Mutagenesis Kit (Enzynomics). PCR products were ligated to pET-28a with an N-terminal tag (MGSSHHHHHHSSGLVPRGSH). The VapC and mutated VapC proteins were produced by identical steps to those of native VapBC complexes. The purities of the purified proteins were verified by sodium dodecyl sulphate–polyacrylamide gel electrophoresis at every step.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13">4.2.  Crystallization, Data Collection, and Processing</h3><div class="NLM_p last">To crystallize the VapBC complex, the sitting drop vapor diffusion method was employed using the Wizard Classic crystallization screen series (Rigaku) at 20 °C. The final crystallization condition was 0.1 M sodium citrate, pH 5.5, and 200 mM lithium sulfate. Crystals were cryoprotected with 20% glycerol and flash-cooled in liquid nitrogen. The diffraction data were collected using an ADSC Quantum Q270r CCD detector at beamlines 5C and 11C of Pohang Light Source, Republic of Korea. All raw data were scaled and processed using HKL2000.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> The detailed statistical information is shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_001.pdf" class="ext-link">Table S2</a>. During the structural calculation, <i>PHENIX</i><a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> was used to automatically build the model, and <i>Coot</i><a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> was utilized to refine it. <i>REFMAC</i> and <i>PHENIX</i><a onclick="showRef(event, 'ref42 ref44'); return false;" href="javascript:void(0);" class="ref ref42 ref44">(42,44)</a> were employed to obtain the plausible <i>R</i><sub>work</sub> and <i>R</i><sub>free</sub> values. <i>MolProbity</i><a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> was used to validate the overall geometry. PyMOL (the PyMOL Molecular Graphics System, Schrödinger, LLC., Cambridge, MA, USA) was used to generate figures.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14">4.3.  <i>In Vitro</i> RNase Assay for VapC and VapC Mutants</h3><div class="NLM_p last">The RNase activities of VapC and VapC mutants were confirmed using an RNase Alert Kit (Integrated DNA Technologies, Inc.). In this assay, each well contained 50 pm of synthetic RNA containing a fluorophore at one end and a quencher group at the other end. If the prepared RNA is contacted by RNase, the quencher is detached. Then, the resulting fluorescence (RFU) during the 1 h reaction period was detected using a SPECTRAmax GEMINI XS spectrofluorometer (Molecular Devices) at 490 and 520 nm as the excitation and emission wavelengths, respectively. For this assay, the purified VapC and VapC mutants were treated with 10 mM EDTA to remove metal ions and exchanged with a purification buffer containing 10 mM Mg<sup>2+</sup>. In each experiment, the concentration of both VapC and VapC mutants was 10 μM. Forty units of RiboLock (Thermo Scientific) were added to each well to prevent contamination. RiboLock does not inhibit prokaryotic RNases I and H. VapC is similar in architecture to RNase H and is thus not affected by RiboLock.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The final concentration of RNA was 5 pM, and the buffer components were 20 mM HEPES, pH 7.5, and 400 mM NaCl.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15">4.4.  Complex Disruption Assay by the Peptides Mimicking the VapBC Interface</h3><div class="NLM_p last">To disrupt the VapBC complex and activate the VapC toxin, several peptides mimicking the binding interface of VapBC were designed (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_001.pdf" class="ext-link">Table S3</a>). The peptides were designed through structural analysis and synthesized by ANYGEN (<a href="http://www.anygen.com" class="extLink">http://www.anygen.com</a>). In these assays, the peptide (100 μM) was added to the VapBC complex (10 μM), and the RFU from the activated VapC toxin was measured. Other experimental protocols used for the peptides were the same as those used for the RNase activity test of VapC and VapC mutants.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16">4.5.  Structure-Based Pharmacophore Generation and Molecular Docking Studies</h3><div class="NLM_p">The structure-based drug design was carried out with a structure-based pharmacophore (SBP) model and molecular docking simulation to virtually screen the primary hit compounds. SBP modeling, which extracts the chemical features from the protein–ligand complex structure, was performed to generate an SBP model of the crystal structure of the VapBC complex (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY3">7BY3</a>). Receptor-ligand pharmacophore generation was used in the pharmacophore generation tools of Discovery Studio (DS) 2018 software (DASSAULT SYSTEMES, 2017) with the default parameters. The SBP model, having four significant chemical features [two hydrogen-bond (HB) donors, one HB acceptor, and one hydrophobic interaction], was designed by considering only the VapB α2−α3 loop region (W64, C67, and Y74 of VapB) as a replacement ligand to inhibit the protein–protein interaction. The generated SBP model was used to retrieve a total of ∼610,000 Korea Chemical Bank (KCB) compounds and select ∼16,000 compounds by filtering.</div><div class="NLM_p last">Subsequently, the molecular docking simulation was performed with the resulting compounds and 7155 diverse set compounds, representing a total of ∼610,000 KCB compounds. The target-binding site of VapC was set from the cocrystal structure of VapB, especially the α2−α3 loop region with a 14 Å radius sphere. The 3D structure of VapC was refined and protonated by the Prepare protein tool implemented in DS 2018. To perform the docking simulation, CDOCKER<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> was used in DS 2018. CDOCKER is a CHARMm-based docking tool that uses a powerful ligand flexibility algorithm and a refinement process that performs random rotation and grid-based simulated annealing at high temperatures. The resulting hit compounds were ranked by the score, which considers both fit value and negative CDOCKER interaction energy (including van der Waals and electrostatic interactions).</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17">4.6.  Assay for Complex Disruption of Small Molecules That Occlude the Interface Pocket</h3><div class="NLM_p last">To disrupt the VapBC complex and activate the VapC toxin, 400 small molecules were initially selected by virtual screening using the pharmacophore model and molecular docking. These 400 small molecules were provided by KCB dissolved in dimethyl sulfoxide solution in 384-well plates. In this virtual screening, small molecules that are expected to occlude the interface pocket of the VapBC complex were explored and listed in the order of their virtual fitness scores as criteria for selecting initial candidates. Small molecules were added to the VapBC complex (10 μM), and their efficacies for activating VapC were tested. Other experimental protocols were the same as those described for the complex disruption assay by the peptides. The synthetic procedures of the final selected two compounds are described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_001.pdf" class="ext-link">Supporting Information</a>. The purity of all compounds was >95%, as determined by liquid chromatography–mass spectrometry.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18">4.7.  Growth Assay and Colony-Forming Unit Measurement</h3><div class="NLM_p last">The genes encoding VapC were additionally cloned into pBAD33. The cloned plasmid was transformed into competent <i>E. coli</i> BL21 (DE3) cells containing previously cloned genes encoding VapB. This compatible plasmid pair (pBAD33 and pET-21a) was grown overnight at 37 °C in M9 minimal medium plus 25 μg/mL chloramphenicol and 50 μg/mL ampicillin. The overnight cultures were diluted 50-fold and further grown for the next 5 h. At an OD<sub>600</sub> of 0.2, 0.2% arabinose was added. For the growth assay, 0.5 mM IPTG was added after 1 h. For the cfu measurement, every 30 min, cell samples were collected and streaked onto LB medium plates with 0.2% glucose and 0.5 mM IPTG. The plates were incubated for 16 h before counting.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19">4.8.  Isothermal Titration Calorimetry</h3><div class="NLM_p last">ITC was carried out using a MicroCal200 (GE Healthcare). Concentrations of proteins and DNA were determined using NanoDrop (Thermo) at 280 nm and 260 nm, respectively. Stock solutions of peptides and small molecules were prepared at the appropriate concentrations. Titrations were performed at 25 °C. Data were calculated by subtracting the background and fitted to a single-site-binding model with the origin-based software provided using MicroCal200.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01118" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18020" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18020" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01118?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01118</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Growth assay and cfu measurement; effects of the peptides and small molecules on free VapC; inhibition curves of compounds; ITC assays; primers used for cloning and producing mutations; data collection and refinement statistics for SeMet-substituted and native structures; peptides used to disrupt the binding interface of VapBC; and synthetic procedures (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Hit compounds and peptide analysis reports (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_003.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_001.pdf">jm0c01118_si_001.pdf (670.11 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_002.csv">jm0c01118_si_002.csv (0.16 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_003.pdf">jm0c01118_si_003.pdf (4.68 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01118" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18333" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18333" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bong-Jin Lee</span> - <span class="hlFld-Affiliation affiliation">The
Research Institute of Pharmaceutical Sciences, College of Pharmacy, Seoul National University, Seoul 08826, Korea</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7896-2961" title="Orcid link">http://orcid.org/0000-0001-7896-2961</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#87ebe5edc7e9eaf5a9f4e9f2a9e6e4a9ecf5"><span class="__cf_email__" data-cfemail="ee828c84ae80839cc09d809bc08f8dc0859c">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sung-Min Kang</span> - <span class="hlFld-Affiliation affiliation">The
Research Institute of Pharmaceutical Sciences, College of Pharmacy, Seoul National University, Seoul 08826, Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chenglong Jin</span> - <span class="hlFld-Affiliation affiliation">The
Research Institute of Pharmaceutical Sciences, College of Pharmacy, Seoul National University, Seoul 08826, Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Do-Hee Kim</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy, Jeju National University, Jeju 63243, Korea</span>; 
    <span class="hlFld-Affiliation affiliation">Interdisciplinary
Graduate Program in Advanced Convergence Technology & Science, Jeju National University, Jeju 63243, Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuno Lee</span> - <span class="hlFld-Affiliation affiliation">Korea
Chemical Bank, Korea Research Institute
of Chemical Technology, Daejeon 34114, Korea</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>S.-M.K., C.J., and D.-H.K. contributed equally to this paper.</p></li><li><span class="author-information-subsection-header">Funding</span><p>This research was funded by the National Research Foundation of Korea (NRF) grants funded by the Korean government (grant numbers: 2018R1A2A1A19018526, 2018R1A5A2024425, 2019R1C1C1002128, and 2019R1I1A1A01057713) and the 2019 BK21 Plus Project for Medicine, Dentistry and Pharmacy. This work was also funded by the International Program (ICR-20-05) of Institute for Protein Research (IPR), Osaka University.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">The structures from this study have been deposited to the PDB website under the PDB IDs <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY2">7BY2</a> (SeMet) and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY3">7BY3</a> (native).</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12116" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12116" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank the beamline (BL) staff members at Pohang Light Source (BL-5C, BL-7A, and BL-11C), Korea, and SPring-8 (BL44XU) under the Collaborative Research Program of the Institute for Protein Research, Osaka University (proposal no. 2019B6972). The chemical library used in this study was kindly provided by the Korea Chemical Bank (<a href="http://www.chembank.org" class="extLink">http://www.chembank.org</a>) of the Korea Research Institute of Chemical Technology. This work was supported by MSIT and PAL, Korea.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">TA</td><td class="NLM_def"><p class="first last">toxin–antitoxin</p></td></tr><tr><td class="NLM_term">PilT</td><td class="NLM_def"><p class="first last">pilus retraction protein</p></td></tr><tr><td class="NLM_term">PIN</td><td class="NLM_def"><p class="first last">pilus retraction protein N-terminal</p></td></tr><tr><td class="NLM_term">RNase</td><td class="NLM_def"><p class="first last">ribonuclease</p></td></tr><tr><td class="NLM_term">LB</td><td class="NLM_def"><p class="first last">Luria broth</p></td></tr><tr><td class="NLM_term">IPTG</td><td class="NLM_def"><p class="first last">isopropyl-β-<span class="smallcaps smallerCapital">d</span>-thiogalactopyranoside</p></td></tr><tr><td class="NLM_term">SeMet</td><td class="NLM_def"><p class="first last">selenomethionine</p></td></tr><tr><td class="NLM_term">F</td><td class="NLM_def"><p class="first last">forward</p></td></tr><tr><td class="NLM_term">R</td><td class="NLM_def"><p class="first last">reverse</p></td></tr><tr><td class="NLM_term">RFU</td><td class="NLM_def"><p class="first last">resulting fluorescence</p></td></tr><tr><td class="NLM_term">SBP</td><td class="NLM_def"><p class="first last">structure-based pharmacophore</p></td></tr><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">Protein Data Bank</p></td></tr><tr><td class="NLM_term">DS</td><td class="NLM_def"><p class="first last">Discovery Studio</p></td></tr><tr><td class="NLM_term">HB</td><td class="NLM_def"><p class="first last">hydrogen bond</p></td></tr><tr><td class="NLM_term">KCB</td><td class="NLM_def"><p class="first last">Korea Chemical Bank</p></td></tr><tr><td class="NLM_term">cfu</td><td class="NLM_def"><p class="first last">colony-forming unit</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">HTH</td><td class="NLM_def"><p class="first last">ribbon–helix–helix</p></td></tr><tr><td class="NLM_term">PISA</td><td class="NLM_def"><p class="first last">proteins, interfaces, structures, and assemblies</p></td></tr><tr><td class="NLM_term">PIC</td><td class="NLM_def"><p class="first last">protein interaction calculator</p></td></tr><tr><td class="NLM_term">BL</td><td class="NLM_def"><p class="first last">beamline</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12431" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12431" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 47 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-García, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blasco, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Contreras, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomas, M.</span></span> <span> </span><span class="NLM_article-title">Toxin-Antitoxin Systems in Clinical Pathogens</span>. <i>Toxins</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">227</span>, <span class="refDoi"> DOI: 10.3390/toxins8070227</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.3390%2Ftoxins8070227" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Cls7vK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=227&author=L.+Fern%C3%A1ndez-Garc%C3%ADaauthor=L.+Blascoauthor=M.+Lopezauthor=G.+Bouauthor=R.+Garc%C3%ADa-Contrerasauthor=T.+Woodauthor=M.+Tomas&title=Toxin-Antitoxin+Systems+in+Clinical+Pathogens&doi=10.3390%2Ftoxins8070227"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Toxin-antitoxin systems in clinical pathogens</span></div><div class="casAuthors">Fernandez-Garcia, Laura; Blasco, Lucia; Lopez, Maria; Bou, German; Garcia-Contreras, Rodolfo; Wood, Thomas; Tomas, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Toxins</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">227/1-227/23</span>CODEN:
                <span class="NLM_cas:coden">TOXIB7</span>;
        ISSN:<span class="NLM_cas:issn">2072-6651</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Toxin-antitoxin (TA) systems are prevalent in bacteria and archaea.  Although not essential for normal cell growth, TA systems are implicated in multiple cellular functions assocd. with survival under stress conditions.  Clin. strains of bacteria are currently causing major human health problems as a result of their multidrug resistance, persistence and strong pathogenicity.  Here, we present a review of the TA systems described to date and their biol. role in human pathogens belonging to the ESKAPE group (Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa and Enterobacter spp.) and others of clin. relevance (Escherichia coli, Burkholderia spp., Streptococcus spp. and Mycobacterium tuberculosis).  Better understanding of the mechanisms of action of TA systems will enable the development of new lines of treatment for infections caused by the above-mentioned pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeDgNrGshJpLVg90H21EOLACvtfcHk0liVumV8fyJKTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Cls7vK&md5=0919f098873bbf6158b04767420fa107</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.3390%2Ftoxins8070227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Ftoxins8070227%26sid%3Dliteratum%253Aachs%26aulast%3DFern%25C3%25A1ndez-Garc%25C3%25ADa%26aufirst%3DL.%26aulast%3DBlasco%26aufirst%3DL.%26aulast%3DLopez%26aufirst%3DM.%26aulast%3DBou%26aufirst%3DG.%26aulast%3DGarc%25C3%25ADa-Contreras%26aufirst%3DR.%26aulast%3DWood%26aufirst%3DT.%26aulast%3DTomas%26aufirst%3DM.%26atitle%3DToxin-Antitoxin%2520Systems%2520in%2520Clinical%2520Pathogens%26jtitle%3DToxins%26date%3D2016%26volume%3D8%26spage%3D227%26doi%3D10.3390%2Ftoxins8070227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gomez-Simmonds, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uhlemann, A.-C.</span></span> <span> </span><span class="NLM_article-title">Clinical Implications of Genomic Adaptation and Evolution of Carbapenem-Resistant Klebsiella pneumoniae</span>. <i>J. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>215</i></span>,  <span class="NLM_fpage">S18</span>– <span class="NLM_lpage">S27</span>, <span class="refDoi"> DOI: 10.1093/infdis/jiw378</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Finfdis%2Fjiw378" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=28375514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFCru7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=215&publication_year=2017&pages=S18-S27&author=A.+Gomez-Simmondsauthor=A.-C.+Uhlemann&title=Clinical+Implications+of+Genomic+Adaptation+and+Evolution+of+Carbapenem-Resistant+Klebsiella+pneumoniae&doi=10.1093%2Finfdis%2Fjiw378"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical implications of genomic adaptation and evolution of carbapenem-resistant Klebsiella pneumoniae</span></div><div class="casAuthors">Gomez-Simmonds, Angela; Uhlemann, Anne-Catrin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Infectious Diseases</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">215</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">S18-S27</span>CODEN:
                <span class="NLM_cas:coden">JIDIAQ</span>;
        ISSN:<span class="NLM_cas:issn">1537-6613</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Klebsiella pneumoniae poses a major challenge to healthcare worldwide as an important cause of multidrug-resistant infections.  Nosocomial clones, including epidemic sequence type 258 (ST258), have shown an affinity for acquiring and disseminating resistance plasmids, particularly variants of the K. pneumoniae carbapenemase.  By comparison, the resurgence of severe communityassocd. K. pneumoniae infections has led to increased recognition of hypervirulent strains belonging to the K1 and K2 capsular serotypes, predominantly in eastern Asia.  Genomic and functional studies suggest that a variety of virulence and immune evasive factors contribute to the success of nosocomial and community-assocd. clonal lineages, aided by mechanisms of genetic plasticity that contribute to uptake of genes assocd. with antimicrobial resistance and pathogenicity.  While there currently appears to be limited overlap between resistant and hypervirulent lineages, specific bacterial and host factors contributing to the emergence of dominant clones remain incompletely understood.  This review summarizes recent advances in our understanding of the mol. epidemiol., virulence potential, and host-pathogen interactions of K. pneumoniae.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGompRNHue-TUbVg90H21EOLACvtfcHk0liVumV8fyJKTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFCru7nM&md5=f59b65bb049121e3c95aaa564ec52da1</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1093%2Finfdis%2Fjiw378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Finfdis%252Fjiw378%26sid%3Dliteratum%253Aachs%26aulast%3DGomez-Simmonds%26aufirst%3DA.%26aulast%3DUhlemann%26aufirst%3DA.-C.%26atitle%3DClinical%2520Implications%2520of%2520Genomic%2520Adaptation%2520and%2520Evolution%2520of%2520Carbapenem-Resistant%2520Klebsiella%2520pneumoniae%26jtitle%3DJ.%2520Infect.%2520Dis.%26date%3D2017%26volume%3D215%26spage%3DS18%26epage%3DS27%26doi%3D10.1093%2Finfdis%2Fjiw378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ogura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiraga, S.</span></span> <span> </span><span class="NLM_article-title">Mini-F Plasmid Genes that Couple Host Cell Division to Plasmid Proliferation</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">4784</span>– <span class="NLM_lpage">4788</span>, <span class="refDoi"> DOI: 10.1073/pnas.80.15.4784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1073%2Fpnas.80.15.4784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=6308648" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADyaL3sXltFKqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=1983&pages=4784-4788&author=T.+Oguraauthor=S.+Hiraga&title=Mini-F+Plasmid+Genes+that+Couple+Host+Cell+Division+to+Plasmid+Proliferation&doi=10.1073%2Fpnas.80.15.4784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Mini-F plasmid genes that couple host cell division to plasmid proliferation</span></div><div class="casAuthors">Ogura, Teru; Hiraga, Sota</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4784-8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">A mechanism for stable maintenance of plasmids, besides the replication and partition mechanisms, was specified by genes of a mini-F plasmid.  An oriC plasmid carrying both a mini-F segment necessary for partition [coordinates 46.4-49.4 kilobase pairs (kb) on the F map] and another segment (42.9-43.6 kb), designated ccd (coupled cell division), is more stably maintained than are oriC plasmids carrying only the partition segment; the stability is comparable to that of the parental mini-F plasmid.  When replication of a plasmid carrying ccd is prevented and the plasmid copy no. decreases, to ≥1/cell, host cell division is inhibited but turbidity increases and chromosome replication are not.  The appearance of plasmid-free segregants is therefore effectively prevented under such conditions.  Exptl. results suggest that redn. of the copy no. of plasmids carrying the ccd region causes an inhibition of cell division and that the ccd region can be dissected into 2 functional regions; 1 (ccdB) inhibits cell division, and the other (ccdA) releases the inhibition.  The interplay of the ccdA and ccdB genes promotes stable plasmid maintenance by coupling host cell division to plasmid proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdSkN-dtoMX7Vg90H21EOLACvtfcHk0liVumV8fyJKTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXltFKqsLs%253D&md5=9966ff0c9dbc34be2a612623d3fbb311</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1073%2Fpnas.80.15.4784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.80.15.4784%26sid%3Dliteratum%253Aachs%26aulast%3DOgura%26aufirst%3DT.%26aulast%3DHiraga%26aufirst%3DS.%26atitle%3DMini-F%2520Plasmid%2520Genes%2520that%2520Couple%2520Host%2520Cell%2520Division%2520to%2520Plasmid%2520Proliferation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1983%26volume%3D80%26spage%3D4784%26epage%3D4788%26doi%3D10.1073%2Fpnas.80.15.4784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerdes, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molin, S.</span></span> <span> </span><span class="NLM_article-title">Unique Type of Plasmid Maintenance Function: Postsegregational Killing of Plasmid-Free Cells</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">3116</span>– <span class="NLM_lpage">3120</span>, <span class="refDoi"> DOI: 10.1073/pnas.83.10.3116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1073%2Fpnas.83.10.3116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=3517851" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADyaL28XktF2ms7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=1986&pages=3116-3120&author=K.+Gerdesauthor=P.+B.+Rasmussenauthor=S.+Molin&title=Unique+Type+of+Plasmid+Maintenance+Function%3A+Postsegregational+Killing+of+Plasmid-Free+Cells&doi=10.1073%2Fpnas.83.10.3116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Unique type of plasmid maintenance function:  postsegregational killing of plasmid-free cells</span></div><div class="casAuthors">Gerdes, Kenn; Rasmussen, Poul B.; Molin, Soeren</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3116-20</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">The stability locus parB+ of plasmid R1 specifies a unique type of plasmid maintenance function.  Two genes, hok (host killing) and sok (suppressor of killing), are required for the stabilizing activity.  The hok gene encodes a highly toxic gene product, whose overexpression causes a rapid killing and a concomitant dramatic change in morphol. of the host cell.  The other gene, sok, was found to encode a product that counteracts the hok gene-mediated killing.  The parB+ region was inserted in a plasmid with a temp.-sensitive replication system.  At nonpermissive temp., the parB+ plasmid was maintained in the population for a significantly longer period than the corresponding parB- plasmid.  Coupled to this extended maintenance, a large fraction of the population was shown to be nonviable plasmid-free cells with the characteristic hok-induced change in morphol.  Thus, the parB+ locus mediates plasmid stability by killing cells that have lost the parB+ plasmid during the preceding cell division, thereby ensuring that a growing bacterial culture predominantly consists of plasmid-contg. cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpinbKsCNHGErVg90H21EOLACvtfcHk0liVumV8fyJKTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28XktF2ms7k%253D&md5=a8c89d2381eef482d2151b9aab695071</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.83.10.3116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.83.10.3116%26sid%3Dliteratum%253Aachs%26aulast%3DGerdes%26aufirst%3DK.%26aulast%3DRasmussen%26aufirst%3DP.%2BB.%26aulast%3DMolin%26aufirst%3DS.%26atitle%3DUnique%2520Type%2520of%2520Plasmid%2520Maintenance%2520Function%253A%2520Postsegregational%2520Killing%2520of%2520Plasmid-Free%2520Cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1986%26volume%3D83%26spage%3D3116%26epage%3D3120%26doi%3D10.1073%2Fpnas.83.10.3116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hazan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sat, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelberg-Kulka, H.</span></span> <span> </span><span class="NLM_article-title">Escherichia coli mazEF-Mediated Cell Death is Triggered by Various Stressful Conditions</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>186</i></span>,  <span class="NLM_fpage">3663</span>– <span class="NLM_lpage">3669</span>, <span class="refDoi"> DOI: 10.1128/jb.186.11.3663-3669.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1128%2FJB.186.11.3663-3669.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=15150257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BD2cXks1CqtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=186&publication_year=2004&pages=3663-3669&author=R.+Hazanauthor=B.+Satauthor=H.+Engelberg-Kulka&title=Escherichia+coli+mazEF-Mediated+Cell+Death+is+Triggered+by+Various+Stressful+Conditions&doi=10.1128%2Fjb.186.11.3663-3669.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Escherichia coli mazEF-mediated cell death is triggered by various stressful conditions</span></div><div class="casAuthors">Hazan, Ronen; Sat, Boaz; Engelberg-Kulka, Hanna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">186</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3663-3669</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">MazEF is an Escherichia coli suicide module specific for a stable toxin and a labile antitoxin.  Inhibiting mazEF expression appeared to activate the module to cause cell death.  Here, the authors show that several stressful conditions, including high temps., DNA damage, and oxidative stress, also induce mazEF-mediated cell death.  They also show that this process takes place only during logarithmic growth and requires an intact relA gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbTDNCICt-J7Vg90H21EOLACvtfcHk0lgJ_ip4QCEFsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXks1CqtLY%253D&md5=fb645a023b387fc3d93eb9161a6d13c7</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1128%2FJB.186.11.3663-3669.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.186.11.3663-3669.2004%26sid%3Dliteratum%253Aachs%26aulast%3DHazan%26aufirst%3DR.%26aulast%3DSat%26aufirst%3DB.%26aulast%3DEngelberg-Kulka%26aufirst%3DH.%26atitle%3DEscherichia%2520coli%2520mazEF-Mediated%2520Cell%2520Death%2520is%2520Triggered%2520by%2520Various%2520Stressful%2520Conditions%26jtitle%3DJ.%2520Bacteriol.%26date%3D2004%26volume%3D186%26spage%3D3663%26epage%3D3669%26doi%3D10.1128%2Fjb.186.11.3663-3669.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sat, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khaner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glaser, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelberg-Kulka, H.</span></span> <span> </span><span class="NLM_article-title">Programmed Cell Death in Escherichia coli: Some Antibiotics can Trigger mazEF Lethality</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>183</i></span>,  <span class="NLM_fpage">2041</span>– <span class="NLM_lpage">2045</span>, <span class="refDoi"> DOI: 10.1128/jb.183.6.2041-2045.2001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1128%2FJB.183.6.2041-2045.2001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=11222603" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhvVeksbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=183&publication_year=2001&pages=2041-2045&author=B.+Satauthor=R.+Hazanauthor=T.+Fisherauthor=H.+Khanerauthor=G.+Glaserauthor=H.+Engelberg-Kulka&title=Programmed+Cell+Death+in+Escherichia+coli%3A+Some+Antibiotics+can+Trigger+mazEF+Lethality&doi=10.1128%2Fjb.183.6.2041-2045.2001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Programmed cell death in Escherichia coli: some antibiotics can trigger mazEF lethality</span></div><div class="casAuthors">Sat, Boaz; Hazan, Ronen; Fisher, Tova; Khaner, Hanita; Glaser, Gad; Engelberg-Kulka, Hanna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">183</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2041-2045</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The discovery of toxin-antitoxin gene pairs (also called addiction modules) on extrachromosomal elements of Escherichia coli, and particularly the discovery of homologous modules on the bacterial chromosome, suggest that a potential for programmed cell death may be inherent in bacterial cultures.  We have reported on the E. coli mazEF system, a regulatable addiction module located on the bacterial chromosome.  MazF is a stable toxin and MazE is a labile antitoxin.  Here we show that cell death mediated by the E. coli mazEF module can be triggered by several antibiotics (rifampicin, chloramphenicol, and spectinomycin) that are general inhibitors of transcription and/or translation.  These antibiotics inhibit the continuous expression of the labile antitoxin MazE, and as a result, the stable toxin MazF causes cell death.  Our results have implications for the possible mode(s) of action of this group of antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTJmRULMgG_bVg90H21EOLACvtfcHk0lgJ_ip4QCEFsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhvVeksbY%253D&md5=68f8ec328695b31a1e33c080bcac428b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1128%2FJB.183.6.2041-2045.2001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.183.6.2041-2045.2001%26sid%3Dliteratum%253Aachs%26aulast%3DSat%26aufirst%3DB.%26aulast%3DHazan%26aufirst%3DR.%26aulast%3DFisher%26aufirst%3DT.%26aulast%3DKhaner%26aufirst%3DH.%26aulast%3DGlaser%26aufirst%3DG.%26aulast%3DEngelberg-Kulka%26aufirst%3DH.%26atitle%3DProgrammed%2520Cell%2520Death%2520in%2520Escherichia%2520coli%253A%2520Some%2520Antibiotics%2520can%2520Trigger%2520mazEF%2520Lethality%26jtitle%3DJ.%2520Bacteriol.%26date%3D2001%26volume%3D183%26spage%3D2041%26epage%3D2045%26doi%3D10.1128%2Fjb.183.6.2041-2045.2001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, T. K.</span></span> <span> </span><span class="NLM_article-title">Toxin-Antitoxin Systems Influence Biofilm and Persister Cell Formation and the General Stress Response</span>. <i>Appl. Environ. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">5577</span>– <span class="NLM_lpage">5583</span>, <span class="refDoi"> DOI: 10.1128/aem.05068-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1128%2FAEM.05068-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=21685157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFGisL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2011&pages=5577-5583&author=X.+Wangauthor=T.+K.+Wood&title=Toxin-Antitoxin+Systems+Influence+Biofilm+and+Persister+Cell+Formation+and+the+General+Stress+Response&doi=10.1128%2Faem.05068-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Toxin-antitoxin systems influence biofilm and persister cell formation and the general stress response</span></div><div class="casAuthors">Wang, Xiaoxue; Wood, Thomas K.</div><div class="citationInfo"><span class="NLM_cas:title">Applied and Environmental Microbiology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5577-5583</span>CODEN:
                <span class="NLM_cas:coden">AEMIDF</span>;
        ISSN:<span class="NLM_cas:issn">0099-2240</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review.  In many genomes, toxin-antitoxin (TA) systems have been identified; however, their role in cell physiol. has been unclear.  Here, the authors examine the evidence that TA systems are involved in biofilm formation and persister cell formation and that these systems may be important regulators of the switch from the planktonic to the biofilm lifestyle as a stress response by their control of secondary messenger 3',5'-cyclic diguanylic acid.  Specifically, upon stress, the sequence-specific mRNA interferases MqsR and MazF mediate cell survival.  The authors propose that TA systems are not redundant, as they may have developed to respond to specific stresses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodVbjold1jJ7Vg90H21EOLACvtfcHk0lgJ_ip4QCEFsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFGisL%252FO&md5=70996635abf2d601c582499fd88c6b0c</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1128%2FAEM.05068-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAEM.05068-11%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWood%26aufirst%3DT.%2BK.%26atitle%3DToxin-Antitoxin%2520Systems%2520Influence%2520Biofilm%2520and%2520Persister%2520Cell%2520Formation%2520and%2520the%2520General%2520Stress%2520Response%26jtitle%3DAppl.%2520Environ.%2520Microbiol.%26date%3D2011%26volume%3D77%26spage%3D5577%26epage%3D5583%26doi%3D10.1128%2Faem.05068-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dörr, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vulić, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, K.</span></span> <span> </span><span class="NLM_article-title">Ciprofloxacin Causes Persister Formation by Inducing the TisB Toxin in Escherichia coli</span>. <i>PLoS Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e1000317</span> <span class="refDoi"> DOI: 10.1371/journal.pbio.1000317</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1371%2Fjournal.pbio.1000317" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=20186264" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2010&author=T.+D%C3%B6rrauthor=M.+Vuli%C4%87author=K.+Lewis&title=Ciprofloxacin+Causes+Persister+Formation+by+Inducing+the+TisB+Toxin+in+Escherichia+coli&doi=10.1371%2Fjournal.pbio.1000317"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pbio.1000317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pbio.1000317%26sid%3Dliteratum%253Aachs%26aulast%3DD%25C3%25B6rr%26aufirst%3DT.%26aulast%3DVuli%25C4%2587%26aufirst%3DM.%26aulast%3DLewis%26aufirst%3DK.%26atitle%3DCiprofloxacin%2520Causes%2520Persister%2520Formation%2520by%2520Inducing%2520the%2520TisB%2520Toxin%2520in%2520Escherichia%2520coli%26jtitle%3DPLoS%2520Biol.%26date%3D2010%26volume%3D8%26doi%3D10.1371%2Fjournal.pbio.1000317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harms, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brodersen, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitarai, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerdes, K.</span></span> <span> </span><span class="NLM_article-title">Toxins, Targets, and Triggers: An Overview of Toxin-Antitoxin Biology</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">768</span>– <span class="NLM_lpage">784</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2018.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1016%2Fj.molcel.2018.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=29398446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVSit7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2018&pages=768-784&author=A.+Harmsauthor=D.+E.+Brodersenauthor=N.+Mitaraiauthor=K.+Gerdes&title=Toxins%2C+Targets%2C+and+Triggers%3A+An+Overview+of+Toxin-Antitoxin+Biology&doi=10.1016%2Fj.molcel.2018.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Toxins, Targets, and Triggers: An Overview of Toxin-Antitoxin Biology</span></div><div class="casAuthors">Harms, Alexander; Brodersen, Ditlev Egeskov; Mitarai, Namiko; Gerdes, Kenn</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">768-784</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Bacterial toxin-antitoxin (TA) modules are abundant genetic elements that encode a toxin protein capable of inhibiting cell growth and an antitoxin that counteracts the toxin.  The majority of toxins are enzymes that interfere with translation or DNA replication, but a wide variety of mol. activities and cellular targets have been described.  Antitoxins are proteins or RNAs that often control their cognate toxins through direct interactions and, in conjunction with other signaling elements, through transcriptional and translational regulation of TA module expression.  Three major biol. functions of TA modules have been discovered, post-segregational killing ("plasmid addiction"), abortive infection (bacteriophage immunity through altruistic suicide), and persister formation (antibiotic tolerance through dormancy).  In this review, we summarize the current state of the field and highlight how multiple levels of regulation shape the conditions of toxin activation to achieve the different biol. functions of TA modules.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSdSgFK0WMK7Vg90H21EOLACvtfcHk0lioXim6H3IWcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVSit7s%253D&md5=fb6f8ea512c48716af378608d0b10eca</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2018.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2018.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DHarms%26aufirst%3DA.%26aulast%3DBrodersen%26aufirst%3DD.%2BE.%26aulast%3DMitarai%26aufirst%3DN.%26aulast%3DGerdes%26aufirst%3DK.%26atitle%3DToxins%252C%2520Targets%252C%2520and%2520Triggers%253A%2520An%2520Overview%2520of%2520Toxin-Antitoxin%2520Biology%26jtitle%3DMol.%2520Cell%26date%3D2018%26volume%3D70%26spage%3D768%26epage%3D784%26doi%3D10.1016%2Fj.molcel.2018.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Son, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, B.-J.</span></span> <span> </span><span class="NLM_article-title">Structural Overview of Toxin-Antitoxin Systems in Infectious Bacteria: A Target for Developing Antimicrobial Agents</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>1834</i></span>,  <span class="NLM_fpage">1155</span>– <span class="NLM_lpage">1167</span>, <span class="refDoi"> DOI: 10.1016/j.bbapap.2013.02.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1016%2Fj.bbapap.2013.02.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=23459128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnslCitb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1834&publication_year=2013&pages=1155-1167&author=S.+J.+Parkauthor=W.+S.+Sonauthor=B.-J.+Lee&title=Structural+Overview+of+Toxin-Antitoxin+Systems+in+Infectious+Bacteria%3A+A+Target+for+Developing+Antimicrobial+Agents&doi=10.1016%2Fj.bbapap.2013.02.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Structural overview of toxin-antitoxin systems in infectious bacteria: A target for developing antimicrobial agents</span></div><div class="casAuthors">Park, Sung Jean; Son, Woo Sung; Lee, Bong-Jin</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Proteins and Proteomics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">1834</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1155-1167</span>CODEN:
                <span class="NLM_cas:coden">BBAPBW</span>;
        ISSN:<span class="NLM_cas:issn">1570-9639</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">A review.  The bacterial toxin-antitoxin (TA) system is a module that may play a role in cell survival under stress conditions.  Generally, toxin mols. act as neg. regulators in cell survival and antitoxin mols. as pos. regulators.  Thus, the expression levels and interactions between toxins and antitoxins should be systematically harmonized so that bacteria can escape such harmful conditions.  Since TA systems are able to control the fate of bacteria, they are considered potent targets for the development of new antimicrobial agents.  TA systems are widely prevalent with a variety of systems existing in bacteria: there are three types of bacterial TA systems depending on the property of the antitoxin which binds either the protein toxin or mRNA coding the toxin protein.  Moreover, the multiplicity of TA genes has been obsd. even in species of bacteria.  Therefore, knowledge on TA systems such as the individual characteristics of TA systems, integrative working mechanisms of various TA systems in bacteria, interactions between toxin mols. and cellular targets, and so on is currently limited due to their complexity.  In this regard, it would be helpful to know the structural characteristics of TA modules for understanding TA systems in bacteria.  Until now, 85 out of the total structures deposited in PDB have been bacterial TA system proteins including TA complexes or isolated toxins/antitoxins.  Here, the authors summarize the structural information of TA systems and analyzed the structural characteristics of known TA modules from several bacteria, esp. focusing on the TA modules of several infectious bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojinQZ7DF55rVg90H21EOLACvtfcHk0lioXim6H3IWcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnslCitb8%253D&md5=6c50d7149dc61fc4d081317ee0cc158b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.bbapap.2013.02.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbapap.2013.02.027%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DS.%2BJ.%26aulast%3DSon%26aufirst%3DW.%2BS.%26aulast%3DLee%26aufirst%3DB.-J.%26atitle%3DStructural%2520Overview%2520of%2520Toxin-Antitoxin%2520Systems%2520in%2520Infectious%2520Bacteria%253A%2520A%2520Target%2520for%2520Developing%2520Antimicrobial%2520Agents%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2013%26volume%3D1834%26spage%3D1155%26epage%3D1167%26doi%3D10.1016%2Fj.bbapap.2013.02.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kang, S.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, B.-J.</span></span> <span> </span><span class="NLM_article-title">A Systematic Overview of Type II and III Toxin-Antitoxin Systems with a Focus on Druggability</span>. <i>Toxins</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">515</span>, <span class="refDoi"> DOI: 10.3390/toxins10120515</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.3390%2Ftoxins10120515" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFGmt7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=515&author=S.-M.+Kangauthor=D.-H.+Kimauthor=C.+Jinauthor=B.-J.+Lee&title=A+Systematic+Overview+of+Type+II+and+III+Toxin-Antitoxin+Systems+with+a+Focus+on+Druggability&doi=10.3390%2Ftoxins10120515"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A systematic overview of type II and III toxin-antitoxin systems with a focus on druggability</span></div><div class="casAuthors">Kang, Sung-Min; Kim, Do-Hee; Jin, Chenglong; Lee, Bong-Jin</div><div class="citationInfo"><span class="NLM_cas:title">Toxins</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">515</span>CODEN:
                <span class="NLM_cas:coden">TOXIB7</span>;
        ISSN:<span class="NLM_cas:issn">2072-6651</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Toxin-antitoxin (TA) systems are known to play various roles in physiol. processes, such as gene regulation, growth arrest and survival, in bacteria exposed to environmental stress.  Type II TA systems comprise natural complexes consisting of protein toxins and antitoxins.  Each toxin and antitoxin participates in distinct regulatory mechanisms depending on the type of TA system.  Recently, peptides designed by mimicking the interfaces between TA complexes showed its potential to activate the activity of toxin by competing its binding counterparts.  Type II TA systems occur more often in pathogenic bacteria than in their nonpathogenic kin.  Therefore, they can be possible drug targets, because of their high abundance in some pathogenic bacteria, such as Mycobacterium tuberculosis.  In addn., recent bioinformatic analyses have shown that type III TA systems are highly abundant in the intestinal microbiota, and recent clin. studies have shown that the intestinal microbiota is linked to inflammatory diseases, obesity and even several types of cancer.  We therefore focused on exploring the putative relationship between intestinal microbiota-related human diseases and type III TA systems.  In this paper, we review and discuss the development of possible druggable materials based on the mechanism of type II and type III TA system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNB2-Jmv9N1LVg90H21EOLACvtfcHk0lioXim6H3IWcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFGmt7zO&md5=8766577046b447ea3ee73ee1d90969cc</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.3390%2Ftoxins10120515&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Ftoxins10120515%26sid%3Dliteratum%253Aachs%26aulast%3DKang%26aufirst%3DS.-M.%26aulast%3DKim%26aufirst%3DD.-H.%26aulast%3DJin%26aufirst%3DC.%26aulast%3DLee%26aufirst%3DB.-J.%26atitle%3DA%2520Systematic%2520Overview%2520of%2520Type%2520II%2520and%2520III%2520Toxin-Antitoxin%2520Systems%2520with%2520a%2520Focus%2520on%2520Druggability%26jtitle%3DToxins%26date%3D2018%26volume%3D10%26spage%3D515%26doi%3D10.3390%2Ftoxins10120515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, H.-Y.</span></span> <span> </span><span class="NLM_article-title">TADB 2.0: An Updated Database of Bacterial Type II Toxin-Antitoxin Loci</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">D749</span>– <span class="NLM_lpage">D753</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx1033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgkx1033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=29106666" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGitr%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D749-D753&author=Y.+Xieauthor=Y.+Weiauthor=Y.+Shenauthor=X.+Liauthor=H.+Zhouauthor=C.+Taiauthor=Z.+Dengauthor=H.-Y.+Ou&title=TADB+2.0%3A+An+Updated+Database+of+Bacterial+Type+II+Toxin-Antitoxin+Loci&doi=10.1093%2Fnar%2Fgkx1033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">TADB 2.0: an updated database of bacterial type II toxin-antitoxin loci</span></div><div class="casAuthors">Xie, Yingzhou; Wei, Yiqing; Shen, Yue; Li, Xiaobin; Zhou, Hao; Tai, Cui; Deng, Zixin; Ou, Hong-Yu</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D749-D753</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">TADB2.0 is an updated database that provides comprehensive information about bacterial type II toxin-antitoxin (TA) loci.  Compared with the previous version, the database refined and the new data schema is employed.  With the aid of text mining and manual curation, it recorded 6193 type II TA loci in 870 replicons of bacteria and archaea, including 105 exptl. validated TA loci.  In addn., the newly developed tool TAfinder combines the homolog searches and the operon structure detection, allowing the prediction for type II TA pairs in bacterial genome sequences.  It also helps to investigate the genomic context of predicted TA loci for putative virulence factors, antimicrobial resistance determinants and mobile genetic elements via alignments to the specific public databases.  Addnl., the module TAfinder-Compare allows comparing the presence of the given TA loci across the close relative genomes.  With the recent updates, TADB2.0 might provide better support for understanding the important roles of type II TA systems in the prokaryotic life activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-4pI4aJE6HbVg90H21EOLACvtfcHk0lguLBs4lNUC-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlGitr%252FL&md5=5972896b514bff8ebfccb62bdb3b9a8a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx1033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx1033%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DTai%26aufirst%3DC.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DOu%26aufirst%3DH.-Y.%26atitle%3DTADB%25202.0%253A%2520An%2520Updated%2520Database%2520of%2520Bacterial%2520Type%2520II%2520Toxin-Antitoxin%2520Loci%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26spage%3DD749%26epage%3DD753%26doi%3D10.1093%2Fnar%2Fgkx1033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burley, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhikadiya, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Costanzo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalenberg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duarte, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guranović, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzenko, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalro, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namkoong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peisach, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Periskova, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prlić, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Randle, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sala, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekharan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, Y.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valasatava, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voigt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuravleva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zardecki, C.</span></span> <span> </span><span class="NLM_article-title">RCSB Protein Data Bank: Biological Macromolecular Structures Enabling Research and Education in Fundamental Biology, Biomedicine, Biotechnology and Energy</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">D464</span>– <span class="NLM_lpage">D474</span>, <span class="refDoi"> DOI: 10.1093/nar/gky1004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgky1004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=30357411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1Cgt7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D464-D474&author=S.+K.+Burleyauthor=H.+M.+Bermanauthor=C.+Bhikadiyaauthor=C.+Biauthor=L.+Chenauthor=L.+Di+Costanzoauthor=C.+Christieauthor=K.+Dalenbergauthor=J.+M.+Duarteauthor=S.+Duttaauthor=Z.+Fengauthor=S.+Ghoshauthor=D.+S.+Goodsellauthor=R.+K.+Greenauthor=V.+Guranovi%C4%87author=D.+Guzenkoauthor=B.+P.+Hudsonauthor=T.+Kalroauthor=Y.+Liangauthor=R.+Loweauthor=H.+Namkoongauthor=E.+Peisachauthor=I.+Periskovaauthor=A.+Prli%C4%87author=C.+Randleauthor=A.+Roseauthor=P.+Roseauthor=R.+Salaauthor=M.+Sekharanauthor=C.+Shaoauthor=L.+Tanauthor=Y.-P.+Taoauthor=Y.+Valasatavaauthor=M.+Voigtauthor=J.+Westbrookauthor=J.+Wooauthor=H.+Yangauthor=J.+Youngauthor=M.+Zhuravlevaauthor=C.+Zardecki&title=RCSB+Protein+Data+Bank%3A+Biological+Macromolecular+Structures+Enabling+Research+and+Education+in+Fundamental+Biology%2C+Biomedicine%2C+Biotechnology+and+Energy&doi=10.1093%2Fnar%2Fgky1004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">RCSB Protein Data Bank: biological macromolecular structures enabling research and education in fundamental biology, biomedicine, biotechnology and energy</span></div><div class="casAuthors">Burley, Stephen K.; Berman, Helen M.; Bhikadiya, Charmi; Bi, Chunxiao; Chen, Li; Di Costanzo, Luigi; Christie, Cole; Dalenberg, Ken; Duarte, Jose M.; Dutta, Shuchismita; Feng, Zukang; Ghosh, Sutapa; Goodsell, David S.; Green, Rachel K.; Guranovic, Vladimir; Guzenko, Dmytro; Hudson, Brian P.; Kalro, Tara; Liang, Yuhe; Lowe, Robert; Namkoong, Harry; Peisach, Ezra; Periskova, Irina; Prlic, Andreas; Randle, Chris; Rose, Alexander; Rose, Peter; Sala, Raul; Sekharan, Monica; Shao, Chenghua; Tan, Lihua; Tao, Yi-Ping; Valasatava, Yana; Voigt, Maria; Westbrook, John; Woo, Jesse; Yang, Huanwang; Young, Jasmine; Zhuravleva, Marina; Zardecki, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D464-D474</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  The Research Collab. for Structural Bioinformatics Protein Data Bank (RCSB PDB, rcsb.org), the US data center for the global PDB archive, serves thousands of Data Depositors in the Americas and Oceania and makes 3D macromol. structure data available at no charge and without usage restrictions to more than 1 million rcsb.org Users worldwide and 600 000 pdb101.rcsb.org education-focused Users around the globe.  PDB Data Depositors include structural biologists using macromol. crystallog., NMR spectroscopy and 3D electron microscopy.  PDB Data Consumers include researchers, educators and students studying Fundamental Biol., Biomedicine, Biotechnol. and Energy.  Recent reorganization of RCSB PDB activities into four integrated, interdependent services is described in detail, together with tools and resources added over the past 2 years to RCSB PDB web portals in support of a 'Structural View of Biol.'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqbbw8LsGXrrVg90H21EOLACvtfcHk0lguLBs4lNUC-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1Cgt7jO&md5=190361b23d7665a66208618c0ba86b5a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky1004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky1004%26sid%3Dliteratum%253Aachs%26aulast%3DBurley%26aufirst%3DS.%2BK.%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DBhikadiya%26aufirst%3DC.%26aulast%3DBi%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DDi%2BCostanzo%26aufirst%3DL.%26aulast%3DChristie%26aufirst%3DC.%26aulast%3DDalenberg%26aufirst%3DK.%26aulast%3DDuarte%26aufirst%3DJ.%2BM.%26aulast%3DDutta%26aufirst%3DS.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DGreen%26aufirst%3DR.%2BK.%26aulast%3DGuranovi%25C4%2587%26aufirst%3DV.%26aulast%3DGuzenko%26aufirst%3DD.%26aulast%3DHudson%26aufirst%3DB.%2BP.%26aulast%3DKalro%26aufirst%3DT.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DLowe%26aufirst%3DR.%26aulast%3DNamkoong%26aufirst%3DH.%26aulast%3DPeisach%26aufirst%3DE.%26aulast%3DPeriskova%26aufirst%3DI.%26aulast%3DPrli%25C4%2587%26aufirst%3DA.%26aulast%3DRandle%26aufirst%3DC.%26aulast%3DRose%26aufirst%3DA.%26aulast%3DRose%26aufirst%3DP.%26aulast%3DSala%26aufirst%3DR.%26aulast%3DSekharan%26aufirst%3DM.%26aulast%3DShao%26aufirst%3DC.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DTao%26aufirst%3DY.-P.%26aulast%3DValasatava%26aufirst%3DY.%26aulast%3DVoigt%26aufirst%3DM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DWoo%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DYoung%26aufirst%3DJ.%26aulast%3DZhuravleva%26aufirst%3DM.%26aulast%3DZardecki%26aufirst%3DC.%26atitle%3DRCSB%2520Protein%2520Data%2520Bank%253A%2520Biological%2520Macromolecular%2520Structures%2520Enabling%2520Research%2520and%2520Education%2520in%2520Fundamental%2520Biology%252C%2520Biomedicine%252C%2520Biotechnology%2520and%2520Energy%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26spage%3DD464%26epage%3DD474%26doi%3D10.1093%2Fnar%2Fgky1004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mattison, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilbur, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">So, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, R. G.</span></span> <span> </span><span class="NLM_article-title">Structure of FitAB from Neisseria gonorrhoeae Bound to DNA Reveals a Tetramer of Toxin-Antitoxin Heterodimers Containing Pin Domains and Ribbon-Helix-Helix Motifs</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">37942</span>– <span class="NLM_lpage">37951</span>, <span class="refDoi"> DOI: 10.1074/jbc.m605198200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1074%2Fjbc.M605198200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=16982615" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Kit7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=37942-37951&author=K.+Mattisonauthor=J.+S.+Wilburauthor=M.+Soauthor=R.+G.+Brennan&title=Structure+of+FitAB+from+Neisseria+gonorrhoeae+Bound+to+DNA+Reveals+a+Tetramer+of+Toxin-Antitoxin+Heterodimers+Containing+Pin+Domains+and+Ribbon-Helix-Helix+Motifs&doi=10.1074%2Fjbc.m605198200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of FitAB from Neisseria gonorrhoeae Bound to DNA Reveals a Tetramer of Toxin-Antitoxin Heterodimers Containing Pin Domains and Ribbon-Helix-Helix Motifs</span></div><div class="casAuthors">Mattison, Kirsten; Wilbur, J. Scott; So, Magdalene; Brennan, Richard G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">37942-37951</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Neisseria gonorrhoeae is a sexually transmitted pathogen that initiates infections in humans by adhering to the mucosal epithelium of the urogenital tract.  The bacterium then enters the apical region of the cell and traffics across the cell to exit into the subepithelial matrix.  Mutations in the fast intracellular trafficking (fitAB) locus cause the bacteria to transit a polarized epithelial monolayer more quickly than the wild-type parent and to replicate within cells at an accelerated rate.  Here, we describe the crystal structure of the toxin-antitoxin heterodimer, FitAB, bound to a high affinity 36-bp DNA fragment from the fitAB promoter.  FitA, the antitoxin, binds DNA through its ribbon-helix-helix motif and is tethered to FitB, the toxin, to form a heterodimer by the insertion of a four turn α-helix into an extensive FitB hydrophobic pocket.  FitB is composed of a PIN (PilT N terminus) domain, with a central, twisted, 5-stranded parallel β-sheet that is open on one side and flanked by five α-helixes.  FitB in the context of the FitAB complex does not display nuclease activity against tested PIN substrates.  The FitAB complex points to the mechanism by which antitoxins with RHH motifs can block the activity of toxins with PIN domains.  Interactions between two FitB mols. result in the formation of a tetramer of FitAB heterodimers, which binds to the 36-bp DNA fragment and provides an explanation for how FitB enhances the DNA binding affinity of FitA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQMCwZq_dD3LVg90H21EOLACvtfcHk0lidWi8iHbUIhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Kit7%252FF&md5=bccbcc839ddce4a994b6d76065e30a6a</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M605198200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M605198200%26sid%3Dliteratum%253Aachs%26aulast%3DMattison%26aufirst%3DK.%26aulast%3DWilbur%26aufirst%3DJ.%2BS.%26aulast%3DSo%26aufirst%3DM.%26aulast%3DBrennan%26aufirst%3DR.%2BG.%26atitle%3DStructure%2520of%2520FitAB%2520from%2520Neisseria%2520gonorrhoeae%2520Bound%2520to%2520DNA%2520Reveals%2520a%2520Tetramer%2520of%2520Toxin-Antitoxin%2520Heterodimers%2520Containing%2520Pin%2520Domains%2520and%2520Ribbon-Helix-Helix%2520Motifs%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D37942%26epage%3D37951%26doi%3D10.1074%2Fjbc.m605198200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dienemann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bøggild, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winther, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerdes, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brodersen, D. E.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the VapBC Toxin-Antitoxin Complex from Shigella Flexneri Reveals a Hetero-Octameric DNA-Binding Assembly</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>414</i></span>,  <span class="NLM_fpage">713</span>– <span class="NLM_lpage">722</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2011.10.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1016%2Fj.jmb.2011.10.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=22037005" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFKls7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=414&publication_year=2011&pages=713-722&author=C.+Dienemannauthor=A.+B%C3%B8ggildauthor=K.+S.+Wintherauthor=K.+Gerdesauthor=D.+E.+Brodersen&title=Crystal+Structure+of+the+VapBC+Toxin-Antitoxin+Complex+from+Shigella+Flexneri+Reveals+a+Hetero-Octameric+DNA-Binding+Assembly&doi=10.1016%2Fj.jmb.2011.10.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of the VapBC Toxin-Antitoxin Complex from Shigella flexneri Reveals a Hetero-Octameric DNA-Binding Assembly</span></div><div class="casAuthors">Dienemann, Christian; Boggild, Andreas; Winther, Kristoffer S.; Gerdes, Kenn; Brodersen, Ditlev E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">414</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">713-722</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Toxin-antitoxin (TA) loci are common in archaea and prokaryotes and allow cells to rapidly adapt to changing environmental conditions through release of active regulators of metab.  Many toxins are endonucleases that target cellular mRNA and tRNAs, while the antitoxins tightly wrap around the toxins to inhibit them under normal circumstances.  The antitoxins also bind to operators in the promoter regions of the cognate TA operon and thereby regulate transcription.  For enteric vapBC TA loci, the VapC toxins specifically cleave tRNAfMet and thus down-regulate protein synthesis.  Here, we describe the crystal structure of the intact Shigella flexneri VapBC TA complex, detd. to 2.7 Å resoln.  Both in soln. and in the crystal structure, four mols. of each protein combine to form a large and globular hetero-octameric assembly with SpoVT/AbrB-type DNA-binding domains at each end and a total mol. mass of about 100 kDa.  The structure gives new insights into the inhibition of VapC toxins by VapB and provides the mol. basis for understanding transcriptional regulation through VapB dimerization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGov8BLYjM0jq7Vg90H21EOLACvtfcHk0lidWi8iHbUIhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFKls7jF&md5=9771b57824d7407c17dae0cfa68103ce</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2011.10.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2011.10.024%26sid%3Dliteratum%253Aachs%26aulast%3DDienemann%26aufirst%3DC.%26aulast%3DB%25C3%25B8ggild%26aufirst%3DA.%26aulast%3DWinther%26aufirst%3DK.%2BS.%26aulast%3DGerdes%26aufirst%3DK.%26aulast%3DBrodersen%26aufirst%3DD.%2BE.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520VapBC%2520Toxin-Antitoxin%2520Complex%2520from%2520Shigella%2520Flexneri%2520Reveals%2520a%2520Hetero-Octameric%2520DNA-Binding%2520Assembly%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2011%26volume%3D414%26spage%3D713%26epage%3D722%26doi%3D10.1016%2Fj.jmb.2011.10.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maté, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincentelli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foos, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raoult, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cambillau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz-Lombardía, M.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the DNA-Bound VapBC2 Antitoxin/Toxin Pair from Rickettsia felis</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">3245</span>– <span class="NLM_lpage">3258</span>, <span class="refDoi"> DOI: 10.1093/nar/gkr1167</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgkr1167" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=22140099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC38Xls1Snu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=3245-3258&author=M.+J.+Mat%C3%A9author=R.+Vincentelliauthor=N.+Foosauthor=D.+Raoultauthor=C.+Cambillauauthor=M.+Ortiz-Lombard%C3%ADa&title=Crystal+Structure+of+the+DNA-Bound+VapBC2+Antitoxin%2FToxin+Pair+from+Rickettsia+felis&doi=10.1093%2Fnar%2Fgkr1167"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the DNA-bound VapBC2 antitoxin/toxin pair from Rickettsia felis</span></div><div class="casAuthors">Mate, Maria J.; Vincentelli, Renaud; Foos, Nicolas; Raoult, Didier; Cambillau, Christian; Ortiz-Lombardia, Miguel</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3245-3258</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Besides their commonly attributed role in the maintenance of low-copy no. plasmids, toxin/antitoxin (TA) loci, also called "addiction modules," have been found in chromosomes and assocd. with a no. of biol. functions such as redn. of protein synthesis, gene regulation and retardation of cell growth under nutritional stress.  The recent discovery of TA loci in obligatory intracellular species of the Rickettsia genus has prompted new research to establish whether they work as stress response elements or as addiction systems that might be toxic for the host cell.  VapBC2 is a TA locus from R. felis, a pathogen responsible for flea-borne spotted fever in humans.  The VapC2 toxin is a PIN-domain protein, whereas the antitoxin, VapB2, belongs to the family of swapped-hairpin β-barrel DNA-binding proteins.  We have used a combination of biophys. and structural methods to characterize this new toxin/antitoxin pair.  Our results show how VapB2 can block the VapC2 toxin.  They provide a first structural description of the interaction between a swapped-hairpin β-barrel protein and DNA.  Finally, these results suggest how the VapC2/VapB2 molar ratio can control the self-regulation of the TA locus transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqR28iWdUuBWLVg90H21EOLACvtfcHk0lidWi8iHbUIhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xls1Snu7c%253D&md5=78770d368e630da4d50499844b254e59</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr1167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr1167%26sid%3Dliteratum%253Aachs%26aulast%3DMat%25C3%25A9%26aufirst%3DM.%2BJ.%26aulast%3DVincentelli%26aufirst%3DR.%26aulast%3DFoos%26aufirst%3DN.%26aulast%3DRaoult%26aufirst%3DD.%26aulast%3DCambillau%26aufirst%3DC.%26aulast%3DOrtiz-Lombard%25C3%25ADa%26aufirst%3DM.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520DNA-Bound%2520VapBC2%2520Antitoxin%252FToxin%2520Pair%2520from%2520Rickettsia%2520felis%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3D3245%26epage%3D3258%26doi%3D10.1093%2Fnar%2Fgkr1167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bendtsen, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luckmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winther, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, C. N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brodersen, D. E.</span></span> <span> </span><span class="NLM_article-title">Toxin Inhibition in C. Crescentus VapBC1 is Mediated by a Flexible Pseudo-Palindromic Protein Motif and Modulated by DNA Binding</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">2875</span>– <span class="NLM_lpage">2886</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw1266</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgkw1266" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=27998932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC1cXivFygtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=2875-2886&author=K.+L.+Bendtsenauthor=K.+Xuauthor=M.+Luckmannauthor=K.+S.+Wintherauthor=S.+A.+Shahauthor=C.+N.+S.+Pedersenauthor=D.+E.+Brodersen&title=Toxin+Inhibition+in+C.+Crescentus+VapBC1+is+Mediated+by+a+Flexible+Pseudo-Palindromic+Protein+Motif+and+Modulated+by+DNA+Binding&doi=10.1093%2Fnar%2Fgkw1266"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Toxin inhibition in C. crescentus VapBC1 is mediated by a flexible pseudo-palindromic protein motif and modulated by DNA binding</span></div><div class="casAuthors">Bendtsen, Kirstine L.; Xu, Kehan; Luckmann, Majbritt; Winther, Kristoffer S.; Shah, Shiraz Ali; Pedersen, Christian N. S.; Brodersen, Ditlev E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2875-2886</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Expression of bacterial type II toxin-antitoxin (TA) systems is regulated at the transcriptional level through direct binding of the antitoxin to pseudo-palindromic sequences on operator DNA.  In this context, the toxin functions as a co-repressor by stimulating DNA binding through direct interaction with the antitoxin.  Here, we det. crystal structures of the complete 90 kDa heterooctameric VapBC1 complex from Caulobacter crescentus CB15 both in isolation and bound to its cognate DNA operator sequence at 1.6 and 2.7 Å resoln., resp.  DNA binding is assocd. with a dramatic architectural rearrangement of conserved TA interactions in which C-terminal extended structures of the antitoxin VapB1 swap positions to interlock the complex in the DNA-bound state.  We further show that a pseudo-palindromic protein sequence in the antitoxin is responsible for this interaction and required for binding and inactivation of the VapC1 toxin dimer.  Sequence anal. of 4127 orthologous VapB sequences reveals that such palindromic protein sequences are widespread and unique to bacterial and archaeal VapB antitoxins suggesting a general principle governing regulation of VapBC TA systems.  Finally, a structure of C-terminally truncated VapB1 bound to VapC1 reveals discrete states of the TA interaction that suggest a structural basis for toxin activation in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxpyA3YmKmQLVg90H21EOLACvtfcHk0ljBCSfv_4DFgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXivFygtb0%253D&md5=5898602c7b2dcd79ac4e82d9a56d6fbf</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1266&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1266%26sid%3Dliteratum%253Aachs%26aulast%3DBendtsen%26aufirst%3DK.%2BL.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DLuckmann%26aufirst%3DM.%26aulast%3DWinther%26aufirst%3DK.%2BS.%26aulast%3DShah%26aufirst%3DS.%2BA.%26aulast%3DPedersen%26aufirst%3DC.%2BN.%2BS.%26aulast%3DBrodersen%26aufirst%3DD.%2BE.%26atitle%3DToxin%2520Inhibition%2520in%2520C.%2520Crescentus%2520VapBC1%2520is%2520Mediated%2520by%2520a%2520Flexible%2520Pseudo-Palindromic%2520Protein%2520Motif%2520and%2520Modulated%2520by%2520DNA%2520Binding%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3D2875%26epage%3D2886%26doi%3D10.1093%2Fnar%2Fgkw1266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molinaro, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashipathy, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battaile, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coussens, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daines, D. A.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of VapBC-1 from Nontypeable Haemophilus influenzae and the Effect of PIN Domain Mutations on Survival during Infection</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>201</i></span>, <span class="NLM_elocation-id">e00026</span> <span class="refDoi"> DOI: 10.1128/jb.00026-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1128%2FJB.00026-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=30936373" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlWmsb3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=201&publication_year=2019&author=A.+L.+Molinaroauthor=M.+M.+Kashipathyauthor=S.+Lovellauthor=K.+P.+Battaileauthor=N.+P.+Coussensauthor=M.+Shenauthor=D.+A.+Daines&title=Crystal+Structure+of+VapBC-1+from+Nontypeable+Haemophilus+influenzae+and+the+Effect+of+PIN+Domain+Mutations+on+Survival+during+Infection&doi=10.1128%2Fjb.00026-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of VapBC-1 from nontypeable Haemophilus influenzae and the effect of PIN domain mutations on survival during infection</span></div><div class="casAuthors">Molinaro, Ashley L.; Kashipathy, Maithri M.; Lovell, Scott; Battaile, Kevin P.; Coussens, Nathan P.; Shen, Min; Daines, Dayle A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">201</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e00026</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">1098-5530</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Toxin-antitoxin (TA) gene pairs have been identified in nearly all bacterial genomes sequenced to date and are thought to facilitate persistence and antibiotic tolerance.  TA loci are classified into various types based upon the characteristics of their antitoxins, with those in type II expressing proteic antitoxins.  Many toxins from type II modules are RNases that maintain a PilT N-terminal (PIN) domain contg. conserved amino acids considered essential for activity.  The vapBC (virulence-assocd. protein) TA system is the largest subfamily in this class and has been linked to pathogenesis of nontypeable Haemophilus influenzae (NTHi).  In this study, the crystal structure of the VapBC-1 complex from NTHi was detd. to 2.20 Å resoln.  Based on this structure, aspartate-to-asparagine and glutamate-to-glutamine mutations of four conserved residues in the PIN domain of the VapC-1 toxin were constructed and the effects of the mutations on protein-protein interactions, growth of Escherichia coli, and pathogenesis ex vivo were tested.  Finally, a novel model system was designed and utilized that consists of an NTHi ΔvapBC-1 strain complemented in cis with the TA module contg. a mutated or wild-type toxin at an ectopic site on the chromosome.  This enabled the anal. of the effect of PIN domain toxin mutants in tandem with their wild-type antitoxin under the control of the vapBC-1 native promoter and in single copy.  This is the first report of a system facilitating the study of TA mutant operons in the background of NTHi during infections of primary human tissues ex vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozANUr-sm2srVg90H21EOLACvtfcHk0ljBCSfv_4DFgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlWmsb3P&md5=6a976fb8683204cc548a5a80cbb8400d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1128%2FJB.00026-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.00026-19%26sid%3Dliteratum%253Aachs%26aulast%3DMolinaro%26aufirst%3DA.%2BL.%26aulast%3DKashipathy%26aufirst%3DM.%2BM.%26aulast%3DLovell%26aufirst%3DS.%26aulast%3DBattaile%26aufirst%3DK.%2BP.%26aulast%3DCoussens%26aufirst%3DN.%2BP.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DDaines%26aufirst%3DD.%2BA.%26atitle%3DCrystal%2520Structure%2520of%2520VapBC-1%2520from%2520Nontypeable%2520Haemophilus%2520influenzae%2520and%2520the%2520Effect%2520of%2520PIN%2520Domain%2520Mutations%2520on%2520Survival%2520during%2520Infection%26jtitle%3DJ.%2520Bacteriol.%26date%3D2019%26volume%3D201%26doi%3D10.1128%2Fjb.00026-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Min, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miallau, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawaya, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cascio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenberg, D.</span></span> <span> </span><span class="NLM_article-title">The Crystal Structure of the Rv0301-Rv0300 VapBC-3 Toxin-Antitoxin Complex from M. tuberculosis Reveals a Mg(2)(+) Ion in the Active Site and a Putative RNA-Binding Site</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1754</span>– <span class="NLM_lpage">1767</span>, <span class="refDoi"> DOI: 10.1002/pro.2161</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1002%2Fpro.2161" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=23011806" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFWitrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2012&pages=1754-1767&author=A.+B.+Minauthor=L.+Miallauauthor=M.+R.+Sawayaauthor=J.+Habelauthor=D.+Cascioauthor=D.+Eisenberg&title=The+Crystal+Structure+of+the+Rv0301-Rv0300+VapBC-3+Toxin-Antitoxin+Complex+from+M.+tuberculosis+Reveals+a+Mg%282%29%28%2B%29+Ion+in+the+Active+Site+and+a+Putative+RNA-Binding+Site&doi=10.1002%2Fpro.2161"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The crystal structure of the Rv0301-Rv0300 VapBC-3 toxin-antitoxin complex from M. tuberculosis reveals a Mg2+ ion in the active site and a putative RNA-binding site</span></div><div class="casAuthors">Min, Andrew B.; Miallau, Linda; Sawaya, Michael R.; Habel, Jeff; Cascio, Duilio; Eisenberg, David</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1754-1767</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">VapBC pairs account for 45 out of 88 identified toxin-antitoxin (TA) pairs in the Mycobacterium tuberculosis (Mtb) H37Rv genome.  A working model suggests that under times of stress, antitoxin mols. are degraded, releasing the toxins to slow the metab. of the cell, which in the case of VapC toxins is via their RNase activity.  Otherwise the TA pairs remain bound to their promoters, autoinhibiting transcription.  The crystal structure of Rv0301-Rv0300, an Mtb VapBC TA complex detd. at 1.49 Å resoln., suggests a mechanism for these three functions: RNase activity, its inhibition by antitoxin, and its ability to bind promoter DNA.  The Rv0301 toxin consists of a core of five parallel beta strands flanked by alpha helixes.  Three proximal aspartates coordinate a Mg2+ ion forming the putative RNase active site.  The Rv0300 antitoxin monomer is extended in structure, consisting of an N-terminal beta strand followed by four helixes.  The last two helixes wrap around the toxin and terminate near the putative RNase active site, but with different conformations.  In one conformation, the C-terminal arginine interferes with Mg2+ ion coordination, suggesting a mechanism by which the antitoxin can inhibit toxin activity.  At the N-terminus of the antitoxin, two pairs of Ribbon-Helix-Helix (RHH) motifs are related by crystallog. twofold symmetry.  The resulting hetero-octameric complex is similar to the FitAB system, but the two RHH motifs are about 30 Å closer together in the Rv0301-Rv0300 complex, suggesting either a different span of the DNA recognition sequence or a conformational change.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRD_bgcnfHIbVg90H21EOLACvtfcHk0ljBCSfv_4DFgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFWitrbL&md5=0c256d6a1f84053b5f732c081b77460d</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fpro.2161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.2161%26sid%3Dliteratum%253Aachs%26aulast%3DMin%26aufirst%3DA.%2BB.%26aulast%3DMiallau%26aufirst%3DL.%26aulast%3DSawaya%26aufirst%3DM.%2BR.%26aulast%3DHabel%26aufirst%3DJ.%26aulast%3DCascio%26aufirst%3DD.%26aulast%3DEisenberg%26aufirst%3DD.%26atitle%3DThe%2520Crystal%2520Structure%2520of%2520the%2520Rv0301-Rv0300%2520VapBC-3%2520Toxin-Antitoxin%2520Complex%2520from%2520M.%2520tuberculosis%2520Reveals%2520a%2520Mg%25282%2529%2528%252B%2529%2520Ion%2520in%2520the%2520Active%2520Site%2520and%2520a%2520Putative%2520RNA-Binding%2520Site%26jtitle%3DProtein%2520Sci.%26date%3D2012%26volume%3D21%26spage%3D1754%26epage%3D1767%26doi%3D10.1002%2Fpro.2161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miallau, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arbing, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cascio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenberg, D.</span></span> <span> </span><span class="NLM_article-title">Structure and Proposed Activity of a Member of the VapBC Family of Toxin-Antitoxin Systems. VapBC-5 from Mycobacterium tuberculosis</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>284</i></span>,  <span class="NLM_fpage">276</span>– <span class="NLM_lpage">283</span>, <span class="refDoi"> DOI: 10.1074/jbc.m805061200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1074%2Fjbc.M805061200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=18952600" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFCjtLzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=276-283&author=L.+Miallauauthor=M.+Fallerauthor=J.+Chiangauthor=M.+Arbingauthor=F.+Guoauthor=D.+Cascioauthor=D.+Eisenberg&title=Structure+and+Proposed+Activity+of+a+Member+of+the+VapBC+Family+of+Toxin-Antitoxin+Systems.+VapBC-5+from+Mycobacterium+tuberculosis&doi=10.1074%2Fjbc.m805061200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and Proposed Activity of a Member of the VapBC Family of Toxin-Antitoxin Systems: VapBC-5 from Mycobacterium tuberculosis</span></div><div class="casAuthors">Miallau, Linda; Faller, Michael; Chiang, Janet; Arbing, Mark; Guo, Feng; Cascio, Duilio; Eisenberg, David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">276-283</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">In prokaryotes, cognate toxin-antitoxin pairs have long been known, but no three-dimensional structure has been available for any given complex from Mycobacterium tuberculosis.  Here we report the crystal structure and activity of a member of the VapBC family of complexes from M. tuberculosis.  The toxin VapC-5 is a compact, 150 residues, two domain α/β protein.  Bent around the toxin is the VapB-5 antitoxin, a 33-residue α-helix.  Assays suggest that the toxin is an Mg-enabled endoribonuclease, inhibited by the antitoxin.  The lack of DNase activity is consistent with earlier suggestions that the complex represses its own operon.  Furthermore, anal. of the interactions in the binding of the antitoxin to the toxin suggest that exquisite control is required to protect the bacteria cell from toxic VapC-5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrh7EhG7joYp7Vg90H21EOLACvtfcHk0lhygErSO55g4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFCjtLzJ&md5=7ae4516a98c956c30890e26cc8e12b0e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M805061200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M805061200%26sid%3Dliteratum%253Aachs%26aulast%3DMiallau%26aufirst%3DL.%26aulast%3DFaller%26aufirst%3DM.%26aulast%3DChiang%26aufirst%3DJ.%26aulast%3DArbing%26aufirst%3DM.%26aulast%3DGuo%26aufirst%3DF.%26aulast%3DCascio%26aufirst%3DD.%26aulast%3DEisenberg%26aufirst%3DD.%26atitle%3DStructure%2520and%2520Proposed%2520Activity%2520of%2520a%2520Member%2520of%2520the%2520VapBC%2520Family%2520of%2520Toxin-Antitoxin%2520Systems.%2520VapBC-5%2520from%2520Mycobacterium%2520tuberculosis%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D276%26epage%3D283%26doi%3D10.1074%2Fjbc.m805061200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deep, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiwari, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agarwal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaundal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kidwai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thakur, K. G.</span></span> <span> </span><span class="NLM_article-title">Structural, Functional and Biological Insights into the Role of Mycobacterium tuberculosis VapBC11 Toxin-Antitoxin System: Targeting a tRNase to Tackle Mycobacterial Adaptation</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">11639</span>– <span class="NLM_lpage">11655</span>, <span class="refDoi"> DOI: 10.1093/nar/gky924</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgky924" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=30329074" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC1MXovFWntbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=11639-11655&author=A.+Deepauthor=P.+Tiwariauthor=S.+Agarwalauthor=S.+Kaundalauthor=S.+Kidwaiauthor=R.+Singhauthor=K.+G.+Thakur&title=Structural%2C+Functional+and+Biological+Insights+into+the+Role+of+Mycobacterium+tuberculosis+VapBC11+Toxin-Antitoxin+System%3A+Targeting+a+tRNase+to+Tackle+Mycobacterial+Adaptation&doi=10.1093%2Fnar%2Fgky924"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Structural, functional and biological insights into the role of Mycobacterium tuberculosis VapBC11 toxin-antitoxin system: targeting a tRNase to tackle mycobacterial adaptation</span></div><div class="casAuthors">Deep, Amar; Tiwari, Prabhakar; Agarwal, Sakshi; Kaundal, Soni; Kidwai, Saqib; Singh, Ramandeep; Thakur, Krishan G.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11639-11655</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Toxin-antitoxin (TA) systems are involved in diverse physiol. processes in prokaryotes, but their exact role in Mycobacterium tuberculosis (Mtb) virulence and in vivo stress adaptation has not been extensively studied.  Here, we demonstrate that the VapBC11 TA module is essential for Mtb to establish infection in guinea pigs.  RNA-sequencing revealed that overexpression of VapC11 toxin results in metabolic slowdown, suggesting that modulation of the growth rate is an essential strategy for in vivo survival.  Interestingly, overexpression of VapC11 resulted in the upregulation of chromosomal TA genes, suggesting the existence of highly coordinated crosstalk among TA systems.  In this study, we also present the crystal structure of the VapBC11 heterooctameric complex at 1.67 Å resoln.  Binding kinetic studies suggest that the binding affinities of toxin-substrate and toxin-antitoxin interactions are comparable.  We used a combination of structural studies, mol. docking, mutational anal. and in vitro RNase assays to enhance our understanding of the mode of substrate recognition by the VapC11 toxin.  Furthermore, we have also designed peptide-based inhibitors to target VapC11 RNase activity.  Taken together, we propose that the structure-guided design of inhibitors against in vivo essential RNases might be a novel strategy to hasten clearance of intracellular Mtb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaZotFmWm1_rVg90H21EOLACvtfcHk0lhygErSO55g4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXovFWntbY%253D&md5=6c6d022bad4a8a1e4c2b53238624d75c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky924&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky924%26sid%3Dliteratum%253Aachs%26aulast%3DDeep%26aufirst%3DA.%26aulast%3DTiwari%26aufirst%3DP.%26aulast%3DAgarwal%26aufirst%3DS.%26aulast%3DKaundal%26aufirst%3DS.%26aulast%3DKidwai%26aufirst%3DS.%26aulast%3DSingh%26aufirst%3DR.%26aulast%3DThakur%26aufirst%3DK.%2BG.%26atitle%3DStructural%252C%2520Functional%2520and%2520Biological%2520Insights%2520into%2520the%2520Role%2520of%2520Mycobacterium%2520tuberculosis%2520VapBC11%2520Toxin-Antitoxin%2520System%253A%2520Targeting%2520a%2520tRNase%2520to%2520Tackle%2520Mycobacterial%2520Adaptation%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26spage%3D11639%26epage%3D11655%26doi%3D10.1093%2Fnar%2Fgky924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Das, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogenberg, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subhramanyam, U. K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilmanns, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gourinath, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, A.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the VapBC-15 Complex from Mycobacterium tuberculosis Reveals a Two-Metal Ion Dependent PIN-Domain Ribonuclease and a Variable Mode of Toxin-Antitoxin Assembly</span>. <i>J. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>188</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">258</span>, <span class="refDoi"> DOI: 10.1016/j.jsb.2014.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1016%2Fj.jsb.2014.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=25450593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWmtrbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=188&publication_year=2014&pages=249-258&author=U.+Dasauthor=V.+Pogenbergauthor=U.+K.+T.+Subhramanyamauthor=M.+Wilmannsauthor=S.+Gourinathauthor=A.+Srinivasan&title=Crystal+Structure+of+the+VapBC-15+Complex+from+Mycobacterium+tuberculosis+Reveals+a+Two-Metal+Ion+Dependent+PIN-Domain+Ribonuclease+and+a+Variable+Mode+of+Toxin-Antitoxin+Assembly&doi=10.1016%2Fj.jsb.2014.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the VapBC-15 complex from Mycobacterium tuberculosis reveals a two-metal ion dependent PIN-domain ribonuclease and a variable mode of toxin-antitoxin assembly</span></div><div class="casAuthors">Das, Uddipan; Pogenberg, Vivian; Subhramanyam, Udaya Kumar Tiruttani; Wilmanns, Matthias; Gourinath, Samudrala; Srinivasan, Alagiri</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Structural Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">188</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">249-258</span>CODEN:
                <span class="NLM_cas:coden">JSBIEM</span>;
        ISSN:<span class="NLM_cas:issn">1047-8477</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Although PIN (PilT N-terminal)-domain proteins are known to have RNase activity, their specific mechanism of action remains unknown.  VapCs form a family of RNases that possess a PIN-domain assembly and are known as toxins.  The activities of VapCs are impaired by VapB antitoxins.  Here we present the crystal structure of the VapBC-15 toxin-antitoxin complex from Mycobacterium tuberculosis detd. to 2.1 Å resoln.  The VapB-15 and VapC-15 components assemble into one heterotetramer (VapB2C2) and two heterotrimers (VapBC2) in each asym. unit of the crystal.  The active site of VapC-15 toxin consists of a cluster of acidic amino acid residues and two divalent metal ions, forming a well organised RNase active site.  The distribution of the catalytic-site residues of the VapC-15 toxin is similar to that of T4 RNase H and of Methanococcus jannaschii FEN-1, providing strong evidence that these three proteins share a similar mechanism of activity.  The presence of both VapB2C2 and VapBC2 emphasizes the fact that the same antitoxin can bind the toxin in 1:1 and 1:2 ratios.  The crystal structure detn. of the VapBC-15 complex reveals for the first time a PIN-domain RNase protein that shows two metal ions at the active site and a variable mode of toxin-antitoxin assembly.  The structure further shows that VapB-15 antitoxin binds to the same groove meant for the binding of putative substrate (RNA), resulting in the inhibition of VapC-15's toxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRdAXWGbiHdLVg90H21EOLACvtfcHk0lhygErSO55g4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWmtrbF&md5=875d5c535bc270fd236313aa81c75b42</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.jsb.2014.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsb.2014.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DU.%26aulast%3DPogenberg%26aufirst%3DV.%26aulast%3DSubhramanyam%26aufirst%3DU.%2BK.%2BT.%26aulast%3DWilmanns%26aufirst%3DM.%26aulast%3DGourinath%26aufirst%3DS.%26aulast%3DSrinivasan%26aufirst%3DA.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520VapBC-15%2520Complex%2520from%2520Mycobacterium%2520tuberculosis%2520Reveals%2520a%2520Two-Metal%2520Ion%2520Dependent%2520PIN-Domain%2520Ribonuclease%2520and%2520a%2520Variable%2520Mode%2520of%2520Toxin-Antitoxin%2520Assembly%26jtitle%3DJ.%2520Struct.%2520Biol.%26date%3D2014%26volume%3D188%26spage%3D249%26epage%3D258%26doi%3D10.1016%2Fj.jsb.2014.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kang, S.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Im, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, B.-J.</span></span> <span> </span><span class="NLM_article-title">Functional Details of the Mycobacterium tuberculosis VapBC26 Toxin-Antitoxin System based on a Structural Study: Insights into Unique Binding and Antibiotic Peptides</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">8564</span>– <span class="NLM_lpage">8580</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgkx489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=28575388" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjt1Wksrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=8564-8580&author=S.-M.+Kangauthor=D.-H.+Kimauthor=K.-Y.+Leeauthor=S.+J.+Parkauthor=H.-J.+Yoonauthor=S.+J.+Leeauthor=H.+Imauthor=B.-J.+Lee&title=Functional+Details+of+the+Mycobacterium+tuberculosis+VapBC26+Toxin-Antitoxin+System+based+on+a+Structural+Study%3A+Insights+into+Unique+Binding+and+Antibiotic+Peptides&doi=10.1093%2Fnar%2Fgkx489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Functional details of the Mycobacterium tuberculosis VapBC26 toxin-antitoxin system based on a structural study: insights into unique binding and antibiotic peptides</span></div><div class="casAuthors">Kang, Sung-Min; Kim, Do-Hee; Lee, Ki-Young; Park, Sung Jean; Yoon, Hye-Jin; Lee, Sang Jae; Im, Hookang; Lee, Bong-Jin</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">8564-8580</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Toxin-antitoxin (TA) systems are essential for bacterial persistence under stressful conditions.  In particular, Mycobacterium tuberculosis express VapBC TA genes that encode the stable VapC toxin and the labile VapB antitoxin.  Under normal conditions, these proteins interact to form a non-toxic TA complex, but the toxin is activated by release from the antitoxin in response to unfavorable conditions.  Here, we present the crystal structure of the M. tuberculosis VapBC26 complex and show that the VapC26 toxin contains a pilus retraction protein (PilT) N-terminal (PIN) domain that is essential for RNase activity and that, the VapB26 antitoxin folds into a ribbon-helix-helix DNA-binding motif at the N-terminus.  The active site of VapC26 is sterically blocked by the flexible C-terminal region of VapB26.  The C-terminal region of free VapB26 adopts an unfolded conformation but forms a helix upon binding to VapC26.  The results of RNase activity assays show that Mg2+ and Mn2+ are essential for the RNase activity of VapC26.  As shown in the NMR spectra, several residues of VapB26 participate in the specific binding to the promoter region of the VapBC26 operon.  In addn., toxin-mimicking peptides were designed that inhibit TA complex formation and thereby increase toxin activity, providing a novel approach to the development of new antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrI8pUoIBVlZrVg90H21EOLACvtfcHk0ljhLZSuvS5QxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjt1Wksrs%253D&md5=e8675ba6324e5ed0a84717a2afc95441</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx489%26sid%3Dliteratum%253Aachs%26aulast%3DKang%26aufirst%3DS.-M.%26aulast%3DKim%26aufirst%3DD.-H.%26aulast%3DLee%26aufirst%3DK.-Y.%26aulast%3DPark%26aufirst%3DS.%2BJ.%26aulast%3DYoon%26aufirst%3DH.-J.%26aulast%3DLee%26aufirst%3DS.%2BJ.%26aulast%3DIm%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DB.-J.%26atitle%3DFunctional%2520Details%2520of%2520the%2520Mycobacterium%2520tuberculosis%2520VapBC26%2520Toxin-Antitoxin%2520System%2520based%2520on%2520a%2520Structural%2520Study%253A%2520Insights%2520into%2520Unique%2520Binding%2520and%2520Antibiotic%2520Peptides%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3D8564%26epage%3D8580%26doi%3D10.1093%2Fnar%2Fgkx489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, I.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chae, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, B.-J.</span></span> <span> </span><span class="NLM_article-title">Structural an Functional Studies of the Mycobacterium tuberculosis VapBC30 Toxin-Antitoxin System: Implications for the Design of Novel Antimicrobial Peptides</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">7624</span>– <span class="NLM_lpage">7637</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv689</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgkv689" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=26150422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslOqtb3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=7624-7637&author=I.-G.+Leeauthor=S.+J.+Leeauthor=S.+Chaeauthor=K.-Y.+Leeauthor=J.-H.+Kimauthor=B.-J.+Lee&title=Structural+an+Functional+Studies+of+the+Mycobacterium+tuberculosis+VapBC30+Toxin-Antitoxin+System%3A+Implications+for+the+Design+of+Novel+Antimicrobial+Peptides&doi=10.1093%2Fnar%2Fgkv689"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and functional studies of the Mycobacterium tuberculosis VapBC30 toxin-antitoxin system: implications for the design of novel antimicrobial peptides</span></div><div class="casAuthors">Lee, In-Gyun; Lee, Sang Jae; Chae, Susanna; Lee, Ki-Young; Kim, Ji-Hun; Lee, Bong-Jin</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">7624-7637</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Toxin-antitoxin (TA) systems play important roles in bacterial physiol., such as multidrug tolerance, biofilm formation, and arrest of cellular growth under stress conditions.  To develop novel antimicrobial agents against tuberculosis, we focused on VapBC systems, which encompass more than half of TA systems in Mycobacterium tuberculosis.  Here, we report that the Mycobacterium tuberculosis VapC30 toxin regulates cellular growth through both magnesium and manganese ion-dependent RNase activity and is inhibited by the cognate VapB30 antitoxin.  We also detd. the 2.7-Å resoln. crystal structure of the M. tuberculosis VapBC30 complex, which revealed a novel process of inactivation of the VapC30 toxin via swapped blocking by the VapB30 antitoxin.  Our study on M. tuberculosis VapBC30 leads us to design two kinds of VapB30 and VapC30-based novel peptides which successfully disrupt the toxin-antitoxin complex and thus activate the RNase activity of the VapC30 toxin.  Our discovery herein possibly paves the way to treat tuberculosis for next generation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3UEanxZ2BarVg90H21EOLACvtfcHk0ljhLZSuvS5QxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslOqtb3I&md5=658d19f46e947c5aedec3556639ae976</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv689%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DI.-G.%26aulast%3DLee%26aufirst%3DS.%2BJ.%26aulast%3DChae%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DK.-Y.%26aulast%3DKim%26aufirst%3DJ.-H.%26aulast%3DLee%26aufirst%3DB.-J.%26atitle%3DStructural%2520an%2520Functional%2520Studies%2520of%2520the%2520Mycobacterium%2520tuberculosis%2520VapBC30%2520Toxin-Antitoxin%2520System%253A%2520Implications%2520for%2520the%2520Design%2520of%2520Novel%2520Antimicrobial%2520Peptides%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26spage%3D7624%26epage%3D7637%26doi%3D10.1093%2Fnar%2Fgkv689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, S.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, B.-J.</span></span> <span> </span><span class="NLM_article-title">Functional Insights into the Streptococcus pneumoniae HicBA Toxin-Antitoxin System Based on a Structural Study</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">6371</span>– <span class="NLM_lpage">6386</span>, <span class="refDoi"> DOI: 10.1093/nar/gky469</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgky469" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=29878152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosVyrtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=6371-6386&author=D.-H.+Kimauthor=S.-M.+Kangauthor=S.+J.+Parkauthor=C.+Jinauthor=H.-J.+Yoonauthor=B.-J.+Lee&title=Functional+Insights+into+the+Streptococcus+pneumoniae+HicBA+Toxin-Antitoxin+System+Based+on+a+Structural+Study&doi=10.1093%2Fnar%2Fgky469"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Functional insights into the Streptococcus pneumoniae HicBA toxin-antitoxin system based on a structural study</span></div><div class="casAuthors">Kim, Do-Hee; Kang, Sung-Min; Park, Sung Jean; Jin, Chenglong; Yoon, Hye-Jin; Lee, Bong-Jin</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6371-6386</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Streptococcus pneumonia has attracted increasing attention due to its resistance to existing antibiotics.  TA systems are essential for bacterial persistence under stressful conditions such as nutrient deprivation, antibiotic treatment, and immune system attacks.  In particular, S. pneumoniae expresses the HicBA TA gene, which encodes the stable HicA toxin and the labile HicB antitoxin.  These proteins interact to form a non-toxic TA complex under normal conditions, but the toxin is activated by release from the antitoxin in response to unfavorable growth conditions.  Here, we present the first crystal structure showing the complete conformation of the HicBA complex from S. pneumonia.  The structure reveals that the HicA toxin contains a double-stranded RNA-binding domain that is essential for RNA recognition and that the C-terminus of the HicB antitoxin folds into a ribbon-helix-helix DNA-binding motif.  The active site of HicA is sterically blocked by the N-terminal region of HicB.  RNase activity assays show that His36 is essential for the RNase activity of HicA, and NMR spectra show that several residues of HicB participate in binding to the promoter DNA of the HicBA operon.  A toxin-mimicking peptide that inhibits TA complex formation and thereby increases toxin activity was designed, providing a novel approach to the development of new antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobiPx_Ood1erVg90H21EOLACvtfcHk0liEia3BkQ4Gzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosVyrtLk%253D&md5=d67938b568850079c5eb7013d341a217</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky469&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky469%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DD.-H.%26aulast%3DKang%26aufirst%3DS.-M.%26aulast%3DPark%26aufirst%3DS.%2BJ.%26aulast%3DJin%26aufirst%3DC.%26aulast%3DYoon%26aufirst%3DH.-J.%26aulast%3DLee%26aufirst%3DB.-J.%26atitle%3DFunctional%2520Insights%2520into%2520the%2520Streptococcus%2520pneumoniae%2520HicBA%2520Toxin-Antitoxin%2520System%2520Based%2520on%2520a%2520Structural%2520Study%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26spage%3D6371%26epage%3D6386%26doi%3D10.1093%2Fnar%2Fgky469" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">Artificial Activation of Toxin-Antitoxin Systems as an Antibacterial Strategy</span>. <i>Trends Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">291</span>– <span class="NLM_lpage">298</span>, <span class="refDoi"> DOI: 10.1016/j.tim.2012.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1016%2Fj.tim.2012.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=22445361" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC38XksFejtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=291-298&author=J.+J.+Williamsauthor=P.+J.+Hergenrother&title=Artificial+Activation+of+Toxin-Antitoxin+Systems+as+an+Antibacterial+Strategy&doi=10.1016%2Fj.tim.2012.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Artificial activation of toxin-antitoxin systems as an antibacterial strategy</span></div><div class="casAuthors">Williams, Julia J.; Hergenrother, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Microbiology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">291-298</span>CODEN:
                <span class="NLM_cas:coden">TRMIEA</span>;
        ISSN:<span class="NLM_cas:issn">0966-842X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Toxin-antitoxin (TA) systems are unique modules that effect plasmid stabilization via post-segregational killing of the bacterial host.  The genes encoding TA systems also exist on bacterial chromosomes, and it has been speculated that these are involved in a variety of cellular processes.  Interest in TA systems has increased dramatically over the past 5 years as the ubiquitous nature of TA genes on bacterial genomes has been revealed.  The exploitation of TA systems as an antibacterial strategy via artificial activation of the toxin has been proposed and has considerable potential; however, efforts in this area remain in the early stages and several major questions remain.  This review investigates the tractability of targeting TA systems to kill bacteria, including fundamental requirements for success, recent advances, and challenges assocd. with artificial toxin activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeA7u5NkOjXrVg90H21EOLACvtfcHk0liEia3BkQ4Gzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XksFejtLc%253D&md5=64e94986065d2d63b7a88cc350604fb0</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2012.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2012.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DJ.%2BJ.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DArtificial%2520Activation%2520of%2520Toxin-Antitoxin%2520Systems%2520as%2520an%2520Antibacterial%2520Strategy%26jtitle%3DTrends%2520Microbiol.%26date%3D2012%26volume%3D20%26spage%3D291%26epage%3D298%26doi%3D10.1016%2Fj.tim.2012.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Równicki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pieńko, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czarnecki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolanowska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartosik, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trylska, J.</span></span> <span> </span><span class="NLM_article-title">Artificial Activation of Escherichia coli mazEF and hipBA Toxin-Antitoxin Systems by Antisense Peptide Nucleic Acids as an Antibacterial Strategy</span>. <i>Front. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2870</span>, <span class="refDoi"> DOI: 10.3389/fmicb.2018.02870</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.3389%2Ffmicb.2018.02870" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=30534121" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A280%3ADC%252BB3cngsVKhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=2870&author=M.+R%C3%B3wnickiauthor=T.+Pie%C5%84koauthor=J.+Czarneckiauthor=M.+Kolanowskaauthor=D.+Bartosikauthor=J.+Trylska&title=Artificial+Activation+of+Escherichia+coli+mazEF+and+hipBA+Toxin-Antitoxin+Systems+by+Antisense+Peptide+Nucleic+Acids+as+an+Antibacterial+Strategy&doi=10.3389%2Ffmicb.2018.02870"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Artificial Activation of Escherichia coli mazEF and hipBA Toxin-Antitoxin Systems by Antisense Peptide Nucleic Acids as an Antibacterial Strategy</span></div><div class="casAuthors">Rownicki Marcin; Pienko Tomasz; Kolanowska Monika; Trylska Joanna; Rownicki Marcin; Pienko Tomasz; Czarnecki Jakub; Bartosik Dariusz; Czarnecki Jakub; Kolanowska Monika</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in microbiology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2870</span>
        ISSN:<span class="NLM_cas:issn">1664-302X</span>.
    </div><div class="casAbstract">The search for new, non-standard targets is currently a high priority in the design of new antibacterial compounds.  Bacterial toxin-antitoxin systems (TAs) are genetic modules that encode a toxin protein that causes growth arrest by interfering with essential cellular processes, and a cognate antitoxin, which neutralizes the toxin activity.  TAs have no human analogs, are highly abundant in bacterial genomes, and therefore represent attractive alternative targets for antimicrobial drugs.  This study demonstrates how artificial activation of Escherichia coli mazEF and hipBA toxin-antitoxin systems using sequence-specific antisense peptide nucleic acid oligomers is an innovative antibacterial strategy.  The growth arrest observed in E. coli resulted from the inhibition of translation of the antitoxins by the antisense oligomers.  Furthermore, two other targets, related to the activities of mazEF and hipBA, were identified as promising sites of action for antibacterials.  These results show that TAs are susceptible to sequence-specific antisense agents and provide a proof-of-concept for their further exploitation in antimicrobial strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRmSJfCr0K06hJ5E29YNGvQfW6udTcc2eb91zKz0jjA77ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cngsVKhsw%253D%253D&md5=27f4f683c6663bf410cc2cb9a9f7326e</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.3389%2Ffmicb.2018.02870&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffmicb.2018.02870%26sid%3Dliteratum%253Aachs%26aulast%3DR%25C3%25B3wnicki%26aufirst%3DM.%26aulast%3DPie%25C5%2584ko%26aufirst%3DT.%26aulast%3DCzarnecki%26aufirst%3DJ.%26aulast%3DKolanowska%26aufirst%3DM.%26aulast%3DBartosik%26aufirst%3DD.%26aulast%3DTrylska%26aufirst%3DJ.%26atitle%3DArtificial%2520Activation%2520of%2520Escherichia%2520coli%2520mazEF%2520and%2520hipBA%2520Toxin-Antitoxin%2520Systems%2520by%2520Antisense%2520Peptide%2520Nucleic%2520Acids%2520as%2520an%2520Antibacterial%2520Strategy%26jtitle%3DFront.%2520Microbiol.%26date%3D2018%26volume%3D9%26spage%3D2870%26doi%3D10.3389%2Ffmicb.2018.02870" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buchan, D. W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D. T.</span></span> <span> </span><span class="NLM_article-title">The PSIPRED Protein Analysis Workbench: 20 Years on</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">W402</span>– <span class="NLM_lpage">W407</span>, <span class="refDoi"> DOI: 10.1093/nar/gkz297</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgkz297" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=31251384" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BB3cXktVyiur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=W402-W407&author=D.+W.+A.+Buchanauthor=D.+T.+Jones&title=The+PSIPRED+Protein+Analysis+Workbench%3A+20+Years+on&doi=10.1093%2Fnar%2Fgkz297"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">The PSIPRED protein analysis workbench: 20 years on</span></div><div class="casAuthors">Buchan, Daniel W. A.; Jones, David T.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W402-W407</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The PSIPRED Workbench is a web server offering a range of predictive methods to the bioscience community for 20 years.  Here, we present the work we have completed to update the PSIPRED Protein Anal. Workbench and make it ready for the next 20 years.  The main focus of our recent website upgrade work has been the acceleration of analyses in the face of increasing protein sequence database size.  We addnl. discuss any new software, the new hardware infrastructure, our web services and web site.  Lastly we survey updates to some of the key predictive algorithms available through our website.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq97_XW9o_NtLVg90H21EOLACvtfcHk0liTx7OdlWBHrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXktVyiur8%253D&md5=1bced7d73a52bc5220c0ebaf95d9ab70</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkz297&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkz297%26sid%3Dliteratum%253Aachs%26aulast%3DBuchan%26aufirst%3DD.%2BW.%2BA.%26aulast%3DJones%26aufirst%3DD.%2BT.%26atitle%3DThe%2520PSIPRED%2520Protein%2520Analysis%2520Workbench%253A%252020%2520Years%2520on%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26spage%3DW402%26epage%3DW407%26doi%3D10.1093%2Fnar%2Fgkz297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schreiter, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drennan, C. L.</span></span> <span> </span><span class="NLM_article-title">Ribbon-Helix-Helix Transcription Factors: Variations on a Theme</span>. <i>Nat. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">710</span>– <span class="NLM_lpage">720</span>, <span class="refDoi"> DOI: 10.1038/nrmicro1717</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1038%2Fnrmicro1717" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=17676053" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BD2sXptFyku7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2007&pages=710-720&author=E.+R.+Schreiterauthor=C.+L.+Drennan&title=Ribbon-Helix-Helix+Transcription+Factors%3A+Variations+on+a+Theme&doi=10.1038%2Fnrmicro1717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Ribbon-helix-helix transcription factors: variations on a theme</span></div><div class="casAuthors">Schreiter, Eric R.; Drennan, Catherine L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">710-720</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The ribbon-helix-helix (RHH) superfamily of transcription factors uses a conserved three-dimensional structural motif to bind to DNA in a sequence-specific manner.  This functionally diverse protein superfamily regulates the transcription of genes that are involved in the uptake of metals, amino-acid biosynthesis, cell division, the control of plasmid copy no., the lytic cycle of bacteriophages and, perhaps, many other cellular processes.  In this Anal., the structures of different RHH transcription factors are compared in order to evaluate the sequence motifs that are required for RHH-domain folding and DNA binding, as well as to identify conserved protein-DNA interactions in this superfamily.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDNof6E8hsMbVg90H21EOLACvtfcHk0liTx7OdlWBHrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXptFyku7o%253D&md5=c4f6c6b2f9a606341b2da8c8d823c09e</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro1717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro1717%26sid%3Dliteratum%253Aachs%26aulast%3DSchreiter%26aufirst%3DE.%2BR.%26aulast%3DDrennan%26aufirst%3DC.%2BL.%26atitle%3DRibbon-Helix-Helix%2520Transcription%2520Factors%253A%2520Variations%2520on%2520a%2520Theme%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2007%26volume%3D5%26spage%3D710%26epage%3D720%26doi%3D10.1038%2Fnrmicro1717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McWilliam, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uludag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Squizzato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowley, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, R.</span></span> <span> </span><span class="NLM_article-title">Analysis Tool Web Services from the EMBL-EBI</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">W597</span>– <span class="NLM_lpage">W600</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt376</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgkt376" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=23671338" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=W597-W600&author=H.+McWilliamauthor=W.+Liauthor=M.+Uludagauthor=S.+Squizzatoauthor=Y.+M.+Parkauthor=N.+Busoauthor=A.+P.+Cowleyauthor=R.+Lopez&title=Analysis+Tool+Web+Services+from+the+EMBL-EBI&doi=10.1093%2Fnar%2Fgkt376"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt376%26sid%3Dliteratum%253Aachs%26aulast%3DMcWilliam%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DUludag%26aufirst%3DM.%26aulast%3DSquizzato%26aufirst%3DS.%26aulast%3DPark%26aufirst%3DY.%2BM.%26aulast%3DBuso%26aufirst%3DN.%26aulast%3DCowley%26aufirst%3DA.%2BP.%26aulast%3DLopez%26aufirst%3DR.%26atitle%3DAnalysis%2520Tool%2520Web%2520Services%2520from%2520the%2520EMBL-EBI%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3DW597%26epage%3DW600%26doi%3D10.1093%2Fnar%2Fgkt376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robert, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gouet, P.</span></span> <span> </span><span class="NLM_article-title">Deciphering Key Features in Protein Structures with the New ENDscript Server</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">W320</span>– <span class="NLM_lpage">W324</span>, <span class="refDoi"> DOI: 10.1093/nar/gku316</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgku316" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=24753421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFCqs7vK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=W320-W324&author=X.+Robertauthor=P.+Gouet&title=Deciphering+Key+Features+in+Protein+Structures+with+the+New+ENDscript+Server&doi=10.1093%2Fnar%2Fgku316"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Deciphering key features in protein structures with the new ENDscript server</span></div><div class="casAuthors">Robert, Xavier; Gouet, Patrice</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W320-W324</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ENDscript 2 is a friendly Web server for extg. and rendering a comprehensive anal. of primary to quaternary protein structure information in an automated way.  This major upgrade has been fully re-engineered to enhance speed, accuracy and usability with interactive 3D visualization.  It takes advantage of the new version 3 of ESPript, our well-known sequence alignment renderer, improved to handle a large no. of data with reduced computation time.  From a single PDB entry or file, ENDscript produces high quality figures displaying multiple sequence alignment of proteins homologous to the query, colored according to residue conservation.  Furthermore, the exptl. secondary structure elements and a detailed set of relevant biophys. and structural data are depicted.  All this information and more are now mapped on interactive 3D PyMOL representations.  Thanks to its adaptive and rigorous algorithm, beginner to expert users can modify settings to fine-tune ENDscript to their needs.  ENDscript has also been upgraded as an open platform for the visualization of multiple biochem. and structural data coming from external biotool Web servers, with both 2D and 3D representations.  ENDscript 2 and ESPript 3 are freely available at http://endscript.ibcp.fr and http://espript.ibcp.fr, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5lD0RT2rA9rVg90H21EOLACvtfcHk0liTx7OdlWBHrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFCqs7vK&md5=96c315b6732a2be7fb72fa44be3511da</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku316%26sid%3Dliteratum%253Aachs%26aulast%3DRobert%26aufirst%3DX.%26aulast%3DGouet%26aufirst%3DP.%26atitle%3DDeciphering%2520Key%2520Features%2520in%2520Protein%2520Structures%2520with%2520the%2520New%2520ENDscript%2520Server%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DW320%26epage%3DW324%26doi%3D10.1093%2Fnar%2Fgku316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arcus, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rainey, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, S. J.</span></span> <span> </span><span class="NLM_article-title">The PIN-Domain Toxin-Antitoxin Array in Mycobacteria</span>. <i>Trends Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">360</span>– <span class="NLM_lpage">365</span>, <span class="refDoi"> DOI: 10.1016/j.tim.2005.06.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1016%2Fj.tim.2005.06.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=15993073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmvFSisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2005&pages=360-365&author=V.+L.+Arcusauthor=P.+B.+Raineyauthor=S.+J.+Turner&title=The+PIN-Domain+Toxin-Antitoxin+Array+in+Mycobacteria&doi=10.1016%2Fj.tim.2005.06.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The PIN-domain toxin-antitoxin array in mycobacteria</span></div><div class="casAuthors">Arcus, Vickery L.; Rainey, Paul B.; Turner, Susan J.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Microbiology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">360-365</span>CODEN:
                <span class="NLM_cas:coden">TRMIEA</span>;
        ISSN:<span class="NLM_cas:issn">0966-842X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  PIN-domains (homologs of the pilT N-terminal domain) are small protein domains of ∼140 amino acids.  They are found in a diverse range of organisms and recent evidence from bioinformatics, biochem., structural biol. and microbiol. suggest that the majority of the prokaryotic PIN-domain proteins are the toxic components of toxin-antitoxin (TA) operons.  Several microorganisms have a large cohort of these operons.  For example, the genome of Mycobacterium tuberculosis encodes 48 PIN-domain proteins, of which 38 are thought to be involved in TA interactions.  This large array of PIN-domain TA operons raises questions as to their evolutionary origin and contemporary functional significance.  We suggest that the evolutionary origin of genes encoding mycobacterial PIN-domain TA operons is linked to the mobile gene pool, but that TA operons can become resident within the chromosome of host cells from where they might be recruited to fulfil a variety of roles assocd. with retardation of cell growth and persistence in stressful environments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofELKYHG-bOrVg90H21EOLACvtfcHk0lgWGE17dGApaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmvFSisLk%253D&md5=b2ee960976b941a4858ec8bbc049055c</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2005.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2005.06.008%26sid%3Dliteratum%253Aachs%26aulast%3DArcus%26aufirst%3DV.%2BL.%26aulast%3DRainey%26aufirst%3DP.%2BB.%26aulast%3DTurner%26aufirst%3DS.%2BJ.%26atitle%3DThe%2520PIN-Domain%2520Toxin-Antitoxin%2520Array%2520in%2520Mycobacteria%26jtitle%3DTrends%2520Microbiol.%26date%3D2005%26volume%3D13%26spage%3D360%26epage%3D365%26doi%3D10.1016%2Fj.tim.2005.06.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arcus, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenzie, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, G. M.</span></span> <span> </span><span class="NLM_article-title">The PIN-Domain Ribonucleases and the Prokaryotic VapBC Toxin-Antitoxin Array</span>. <i>Protein Eng., Des. Sel.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">40</span>, <span class="refDoi"> DOI: 10.1093/protein/gzq081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fprotein%2Fgzq081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=21036780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A280%3ADC%252BC3M%252Fjt1yjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2011&pages=33-40&author=V.+L.+Arcusauthor=J.+L.+McKenzieauthor=J.+Robsonauthor=G.+M.+Cook&title=The+PIN-Domain+Ribonucleases+and+the+Prokaryotic+VapBC+Toxin-Antitoxin+Array&doi=10.1093%2Fprotein%2Fgzq081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The PIN-domain ribonucleases and the prokaryotic VapBC toxin-antitoxin array</span></div><div class="casAuthors">Arcus Vickery L; McKenzie Joanna L; Robson Jennifer; Cook Gregory M</div><div class="citationInfo"><span class="NLM_cas:title">Protein engineering, design & selection : PEDS</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">33-40</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The PIN-domains are small proteins of ~130 amino acids that are found in bacteria, archaea and eukaryotes and are defined by a group of three strictly conserved acidic amino acids.  The conserved three-dimensional structures of the PIN-domains cluster these acidic residues in an enzymatic active site.  PIN-domains cleave single-stranded RNA in a sequence-specific, Mg2+- or Mn2+-dependent manner.  These ribonucleases are toxic to the cells which express them and to offset this toxicity, they are co-expressed with tight binding protein inhibitors.  The genes encoding these two proteins are adjacent in the genome of all prokaryotic organisms where they are found.  This sequential arrangement of inhibitor-RNAse genes conforms to that of the so-called toxin-antitoxin (TA) modules and the PIN-domain TAs have been named VapBC TAs (virulence associated proteins, VapB is the inhibitor which contains a transcription factor domain and VapC is the PIN-domain ribonuclease).  The presence of large numbers of vapBC loci in disparate prokaryotes has motivated many researchers to investigate their biochemical and biological functions.  For example, the devastating human pathogen Mycobacterium tuberculosis has 45 vapBC loci encoded in its genome whereas its non-pathogenic relative, Mycobacterium smegmatis has just one vapBC operon.  On another branch of the prokaryotic tree, the nitrogen-fixing symbiont of legumes, Sinorhizobium meliloti has 21 vapBC loci and at least one of these loci have been implicated in the regulation of growth in the plant nodule.  A range of biological functions has been suggested for these operons and this review sets out to survey the PIN-domains and summarise the current knowledge about the vapBC TA systems and their roles in diverse bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSYcLBUbiNT5Rgd2Wuil24tfW6udTcc2ebhsd1WOPkLI7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3M%252Fjt1yjuw%253D%253D&md5=d3c1bb299ca62ea690b84063250d63ce</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2Fgzq081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252Fgzq081%26sid%3Dliteratum%253Aachs%26aulast%3DArcus%26aufirst%3DV.%2BL.%26aulast%3DMcKenzie%26aufirst%3DJ.%2BL.%26aulast%3DRobson%26aufirst%3DJ.%26aulast%3DCook%26aufirst%3DG.%2BM.%26atitle%3DThe%2520PIN-Domain%2520Ribonucleases%2520and%2520the%2520Prokaryotic%2520VapBC%2520Toxin-Antitoxin%2520Array%26jtitle%3DProtein%2520Eng.%252C%2520Des.%2520Sel.%26date%3D2011%26volume%3D24%26spage%3D33%26epage%3D40%26doi%3D10.1093%2Fprotein%2Fgzq081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coles, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Djuranovic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Söding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frickey, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koretke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truffault, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lupas, A. N.</span></span> <span> </span><span class="NLM_article-title">AbrB-Like Transcription Factors Assume a Swapped Hairpin Fold that is Evolutionarily Related to Double-Psi Beta Barrels</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">919</span>– <span class="NLM_lpage">928</span>, <span class="refDoi"> DOI: 10.1016/j.str.2005.03.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1016%2Fj.str.2005.03.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=15939023" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltFCksbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2005&pages=919-928&author=M.+Colesauthor=S.+Djuranovicauthor=J.+S%C3%B6dingauthor=T.+Frickeyauthor=K.+Koretkeauthor=V.+Truffaultauthor=J.+Martinauthor=A.+N.+Lupas&title=AbrB-Like+Transcription+Factors+Assume+a+Swapped+Hairpin+Fold+that+is+Evolutionarily+Related+to+Double-Psi+Beta+Barrels&doi=10.1016%2Fj.str.2005.03.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">AbrB-like Transcription Factors Assume a Swapped Hairpin Fold that Is Evolutionarily Related to Double-Psi β Barrels</span></div><div class="casAuthors">Coles, Murray; Djuranovic, Sergej; Soeding, Johannes; Frickey, Tancred; Koretke, Kristin; Truffault, Vincent; Martin, Joerg; Lupas, Andrei N.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">919-928</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">AbrB is a key transition-state regulator of Bacillus subtilis.  Based on the conservation of a βαβ structural unit, we proposed a β barrel fold for its DNA binding domain, similar to, but topol. distinct from, double-psi β barrels.  However, the NMR structure revealed a novel fold, the "looped-hinge helix.  " To understand this discrepancy, we undertook a bioinformatics study of AbrB and its homologs; these form a large superfamily, which includes SpoVT, PrlF, MraZ, addiction module antidotes (PemI, MazE), plasmid maintenance proteins (VagC, VapB), and archaeal PhoU homologs.  MazE and MraZ form swapped-hairpin β barrels.  We therefore reexamd. the fold of AbrB by NMR spectroscopy and found that it also forms a swapped-hairpin barrel.  The conservation of the core βαβ element supports a common evolutionary origin for swapped-hairpin and double-psi barrels, which we group into a higher-order class, the cradle-loop barrels, based on the peculiar shape of their ligand binding site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8QSWF06S1XLVg90H21EOLACvtfcHk0lgWGE17dGApaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltFCksbw%253D&md5=49c94348b872276896599815818eede9</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2005.03.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2005.03.017%26sid%3Dliteratum%253Aachs%26aulast%3DColes%26aufirst%3DM.%26aulast%3DDjuranovic%26aufirst%3DS.%26aulast%3DS%25C3%25B6ding%26aufirst%3DJ.%26aulast%3DFrickey%26aufirst%3DT.%26aulast%3DKoretke%26aufirst%3DK.%26aulast%3DTruffault%26aufirst%3DV.%26aulast%3DMartin%26aufirst%3DJ.%26aulast%3DLupas%26aufirst%3DA.%2BN.%26atitle%3DAbrB-Like%2520Transcription%2520Factors%2520Assume%2520a%2520Swapped%2520Hairpin%2520Fold%2520that%2520is%2520Evolutionarily%2520Related%2520to%2520Double-Psi%2520Beta%2520Barrels%26jtitle%3DStructure%26date%3D2005%26volume%3D13%26spage%3D919%26epage%3D928%26doi%3D10.1016%2Fj.str.2005.03.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henrick, K.</span></span> <span> </span><span class="NLM_article-title">Inference of Macromolecular Assemblies from Crystalline State</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>372</i></span>,  <span class="NLM_fpage">774</span>– <span class="NLM_lpage">797</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2007.05.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1016%2Fj.jmb.2007.05.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=17681537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpvFGktb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=372&publication_year=2007&pages=774-797&author=E.+Krissinelauthor=K.+Henrick&title=Inference+of+Macromolecular+Assemblies+from+Crystalline+State&doi=10.1016%2Fj.jmb.2007.05.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Inference of Macromolecular Assemblies from Crystalline State</span></div><div class="casAuthors">Krissinel, Evgeny; Henrick, Kim</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">372</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">774-797</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The authors discuss basic phys.-chem. principles underlying the formation of stable macromol. complexes, which in many cases are likely to be the biol. units performing a certain physiol. function.  The authors also consider available theor. approaches to the calcn. of macromol. affinity and entropy of complexation.  The latter is shown to play an important role and make a major effect on complex size and symmetry.  The authors develop a new method, based on chem. thermodn., for automatic detection of macromol. assemblies in the Protein Data Bank (PDB) entries that are the results of x-ray diffraction expts.  As found, biol. units may be recovered at 80-90% success rate, which makes x-ray crystallog. an important source of exptl. data on macromol. complexes and protein-protein interactions.  The method is implemented as a public WWW service (http://www.ebi.ac.uk/msd-srv/prot_int/pistart.html).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3cilBvVoBS7Vg90H21EOLACvtfcHk0lhz0-h0J_MxQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpvFGktb8%253D&md5=a5c764cfc7dc129f53ddc31ef9d475fa</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2007.05.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2007.05.022%26sid%3Dliteratum%253Aachs%26aulast%3DKrissinel%26aufirst%3DE.%26aulast%3DHenrick%26aufirst%3DK.%26atitle%3DInference%2520of%2520Macromolecular%2520Assemblies%2520from%2520Crystalline%2520State%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D372%26spage%3D774%26epage%3D797%26doi%3D10.1016%2Fj.jmb.2007.05.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tina, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhadra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, N.</span></span> <span> </span><span class="NLM_article-title">PIC: Protein Interactions Calculator</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">W473</span>– <span class="NLM_lpage">W476</span>, <span class="refDoi"> DOI: 10.1093/nar/gkm423</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgkm423" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=17584791" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A280%3ADC%252BD2svhsVOltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=W473-W476&author=K.+G.+Tinaauthor=R.+Bhadraauthor=N.+Srinivasan&title=PIC%3A+Protein+Interactions+Calculator&doi=10.1093%2Fnar%2Fgkm423"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">PIC: Protein Interactions Calculator</span></div><div class="casAuthors">Tina K G; Bhadra R; Srinivasan N</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic acids research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">Web Server issue</span>),
    <span class="NLM_cas:pages">W473-6</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Interactions within a protein structure and interactions between proteins in an assembly are essential considerations in understanding molecular basis of stability and functions of proteins and their complexes.  There are several weak and strong interactions that render stability to a protein structure or an assembly.  Protein Interactions Calculator (PIC) is a server which, given the coordinate set of 3D structure of a protein or an assembly, computes various interactions such as disulphide bonds, interactions between hydrophobic residues, ionic interactions, hydrogen bonds, aromatic-aromatic interactions, aromatic-sulphur interactions and cation-pi interactions within a protein or between proteins in a complex.  Interactions are calculated on the basis of standard, published criteria.  The identified interactions between residues can be visualized using a RasMol and Jmol interface.  The advantage with PIC server is the easy availability of inter-residue interaction calculations in a single site.  It also determines the accessible surface area and residue-depth, which is the distance of a residue from the surface of the protein.  User can also recognize specific kind of interactions, such as apolar-apolar residue interactions or ionic interactions, that are formed between buried or exposed residues or near the surface or deep inside.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSKn5zO1OOtqL5XHngOXiV_fW6udTcc2eYt8eDEB094q7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2svhsVOltQ%253D%253D&md5=6c59648eeba47a6175754ed6d7499c14</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm423%26sid%3Dliteratum%253Aachs%26aulast%3DTina%26aufirst%3DK.%2BG.%26aulast%3DBhadra%26aufirst%3DR.%26aulast%3DSrinivasan%26aufirst%3DN.%26atitle%3DPIC%253A%2520Protein%2520Interactions%2520Calculator%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DW473%26epage%3DW476%26doi%3D10.1093%2Fnar%2Fgkm423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luong, H. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, B.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.-W.</span></span> <span> </span><span class="NLM_article-title">Effects of Lysine-to-Arginine Substitution on Antimicrobial Activity of Cationic Stapled Heptapeptides</span>. <i>Arch. Pharmacal Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">1092</span>– <span class="NLM_lpage">1097</span>, <span class="refDoi"> DOI: 10.1007/s12272-018-1084-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1007%2Fs12272-018-1084-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=30361948" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVWjt7fO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2018&pages=1092-1097&author=H.+X.+Luongauthor=D.-H.+Kimauthor=B.-J.+Leeauthor=Y.-W.+Kim&title=Effects+of+Lysine-to-Arginine+Substitution+on+Antimicrobial+Activity+of+Cationic+Stapled+Heptapeptides&doi=10.1007%2Fs12272-018-1084-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of lysine-to-arginine substitution on antimicrobial activity of cationic stapled heptapeptides</span></div><div class="casAuthors">Luong, Huy X.; Kim, Do-Hee; Lee, Bong-Jin; Kim, Young-Woo</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Pharmacal Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1092-1097</span>CODEN:
                <span class="NLM_cas:coden">APHRDQ</span>;
        ISSN:<span class="NLM_cas:issn">0253-6269</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Korea</span>)
        </div><div class="casAbstract">We previously reported a series of amphipathic helixes of stapled heptapeptides as membrane-lytic antimicrobial peptides (AMPs).  These peptides possess 3 Lys residues as the sole cationic residues in their hydrophilic face of the helix.  Lys-to-Arg residue substitution is often shown to increase antimicrobial activity of many natural AMPs due to the more favorable interactions of guanidinium moiety of Arg with membranes.  In an effort to further improve the pharmacol. properties of our novel AMP series, we here examd. the impact of Lys-to-Arg substitution on their structures and antimicrobial and hemolytic activities.  The results indicated that the Lys-to-Arg substitution does not always guarantee enhancement in the antimicrobial potency of AMPs.  Instead, we obsd. varied potency and selectivity depending on the no. of substitutions and the positions substituted.  The results implied that, in the given helical scaffold stabilized by a hydrocarbon staple, antimicrobial potency and selectivity are influenced by a complex effect of various structural and chem. changes accompanied by Lys-to-Arg substitution rather than solely by the type of cationic residue.  These data show potential for use in our scaffold-assisted development of short, selective, and metabolically stable AMPs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ5fDZQtqhq7Vg90H21EOLACvtfcHk0lhz0-h0J_MxQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVWjt7fO&md5=9d5305d18f105c57dc46ad4d3717dab2</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1007%2Fs12272-018-1084-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12272-018-1084-5%26sid%3Dliteratum%253Aachs%26aulast%3DLuong%26aufirst%3DH.%2BX.%26aulast%3DKim%26aufirst%3DD.-H.%26aulast%3DLee%26aufirst%3DB.-J.%26aulast%3DKim%26aufirst%3DY.-W.%26atitle%3DEffects%2520of%2520Lysine-to-Arginine%2520Substitution%2520on%2520Antimicrobial%2520Activity%2520of%2520Cationic%2520Stapled%2520Heptapeptides%26jtitle%3DArch.%2520Pharmacal%2520Res.%26date%3D2018%26volume%3D41%26spage%3D1092%26epage%3D1097%26doi%3D10.1007%2Fs12272-018-1084-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laskowski, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swindells, M. B.</span></span> <span> </span><span class="NLM_article-title">LigPlot+: Multiple Ligand-Protein Interaction Diagrams for Drug Discovery</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2778</span>– <span class="NLM_lpage">2786</span>, <span class="refDoi"> DOI: 10.1021/ci200227u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200227u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1KjtrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2778-2786&author=R.+A.+Laskowskiauthor=M.+B.+Swindells&title=LigPlot%2B%3A+Multiple+Ligand-Protein+Interaction+Diagrams+for+Drug+Discovery&doi=10.1021%2Fci200227u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">LigPlot+: Multiple Ligand-Protein Interaction Diagrams for Drug Discovery</span></div><div class="casAuthors">Laskowski, Roman A.; Swindells, Mark B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2778-2786</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe a graphical system for automatically generating multiple 2D diagrams of ligand-protein interactions from 3D coordinates.  The diagrams portray the hydrogen-bond interaction patterns and hydrophobic contacts between the ligand(s) and the main-chain or side-chain elements of the protein.  The system is able to plot, in the same orientation, related sets of ligand-protein interactions.  This facilitates popular research tasks, such as analyzing a series of small mols. binding to the same protein target, a single ligand binding to homologous proteins, or the completely general case where both protein and ligand change.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYuWvYH7Nel7Vg90H21EOLACvtfcHk0liirEbv2tHFnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1KjtrzP&md5=576e59ad4f0c4304d01c6c80301ecf39</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fci200227u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200227u%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DSwindells%26aufirst%3DM.%2BB.%26atitle%3DLigPlot%252B%253A%2520Multiple%2520Ligand-Protein%2520Interaction%2520Diagrams%2520for%2520Drug%2520Discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2778%26epage%3D2786%26doi%3D10.1021%2Fci200227u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kedzierska, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lian, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, F.</span></span> <span> </span><span class="NLM_article-title">Toxin-Antitoxin Regulation: Bimodal Interaction of YefM-YoeB with Paired DNA Palindromes Exerts Transcriptional Autorepression</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">325</span>– <span class="NLM_lpage">339</span>, <span class="refDoi"> DOI: 10.1093/nar/gkl1028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgkl1028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=17170003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVyitr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=325-339&author=B.+Kedzierskaauthor=L.+Y.+Lianauthor=F.+Hayes&title=Toxin-Antitoxin+Regulation%3A+Bimodal+Interaction+of+YefM-YoeB+with+Paired+DNA+Palindromes+Exerts+Transcriptional+Autorepression&doi=10.1093%2Fnar%2Fgkl1028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Toxin-antitoxin regulation: bimodal interaction of YefM-YoeB with paired DNA palindromes exerts transcriptional autorepression</span></div><div class="casAuthors">Kedzierska, Barbara; Lian, Lu-Yun; Hayes, Finbarr</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">325-339</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Toxin-antitoxin (TA) complexes function in programmed cell death or stress response mechanisms in bacteria.  The YefM-YoeB TA complex of Escherichia coli consists of YoeB toxin that is counteracted by YefM antitoxin.  When liberated from the complex, YoeB acts as an endoribonuclease, preferentially cleaving 3' of purine nucleotides.  Here we demonstrate that yefM-yoeB is transcriptionally autoregulated.  YefM, a dimeric protein with extensive secondary structure revealed by CD and NMR spectroscopy, is the primary repressor, whereas YoeB is a repression enhancer.  The operator site 5' of yefM-yoeB comprises adjacent long and short palindromes with core 5'-TGTACA-3' motifs.  YefM binds the long palindrome, followed sequentially by short palindrome recognition.  In contrast, the repressor-corepressor complex recognizes both motifs more avidly, implying that YefM within the complex has an enhanced DNA-binding affinity compared to free YefM.  Operator interaction by YefM and YefM-YoeB is accompanied by structural transitions in the proteins.  Paired 5'-TGTACA-3' motifs are common in yefM-yoeB regulatory regions in diverse genomes suggesting that interaction of YefM-YoeB with these motifs is a conserved mechanism of operon autoregulation.  Artificial perturbation of transcriptional autorepression could elicit inappropriate YoeB toxin prodn. and induction of bacterial cell suicide, a potentially novel antibacterial strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCiSray-MBfrVg90H21EOLACvtfcHk0liirEbv2tHFnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVyitr4%253D&md5=8c355a0480750fbef4c5ec2f4ef097de</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl1028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl1028%26sid%3Dliteratum%253Aachs%26aulast%3DKedzierska%26aufirst%3DB.%26aulast%3DLian%26aufirst%3DL.%2BY.%26aulast%3DHayes%26aufirst%3DF.%26atitle%3DToxin-Antitoxin%2520Regulation%253A%2520Bimodal%2520Interaction%2520of%2520YefM-YoeB%2520with%2520Paired%2520DNA%2520Palindromes%2520Exerts%2520Transcriptional%2520Autorepression%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3D325%26epage%3D339%26doi%3D10.1093%2Fnar%2Fgkl1028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Overgaard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borch, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jørgensen, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerdes, K.</span></span> <span> </span><span class="NLM_article-title">Messenger RNA Interferase RelE Controls relBE Transcription by Conditional Cooperativity</span>. <i>Mol. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">841</span>– <span class="NLM_lpage">857</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2958.2008.06313.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1111%2Fj.1365-2958.2008.06313.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=18532983" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVehurzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2008&pages=841-857&author=M.+Overgaardauthor=J.+Borchauthor=M.+G.+J%C3%B8rgensenauthor=K.+Gerdes&title=Messenger+RNA+Interferase+RelE+Controls+relBE+Transcription+by+Conditional+Cooperativity&doi=10.1111%2Fj.1365-2958.2008.06313.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity</span></div><div class="casAuthors">Overgaard, Martin; Borch, Jonas; Joergensen, Mikkel G.; Gerdes, Kenn</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Microbiology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">841-857</span>CODEN:
                <span class="NLM_cas:coden">MOMIEE</span>;
        ISSN:<span class="NLM_cas:issn">0950-382X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Prokaryotic toxin-antitoxin (TA) loci consist of two genes in an operon that encodes a metabolically stable toxin and an unstable antitoxin.  The antitoxin neutralizes its cognate toxin by forming a tight complex with it.  In all cases known, the antitoxin auto-regulates TA operon transcription by binding to one or more operators in the promoter region while the toxin functions as a co-repressor of transcription.  Interestingly, the toxin can also stimulate TA operon transcription.  Here we analyze mechanistic aspects of how RelE of Escherichia coli can function both as a co-repressor and as a derepressor of relBE transcription.  When RelB was in excess to RelE, two trimeric RelB2•RelE complexes bound cooperatively to two adjacent operator sites in the relBE promoter region and repressed transcription.  In contrast, RelE in excess stimulated relBE transcription and released the RelB2•RelE complex from operator DNA.  A mutational anal. of the operator sites showed that RelE in excess counteracted cooperative binding of the RelB2•RelE complexes to the operator sites.  Thus, RelE controls relBE transcription by conditional cooperativity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnOKvpn90C8bVg90H21EOLACvtfcHk0liirEbv2tHFnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVehurzE&md5=6b2590d4ad21756e1bf896ac8c3d5573</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2958.2008.06313.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2958.2008.06313.x%26sid%3Dliteratum%253Aachs%26aulast%3DOvergaard%26aufirst%3DM.%26aulast%3DBorch%26aufirst%3DJ.%26aulast%3DJ%25C3%25B8rgensen%26aufirst%3DM.%2BG.%26aulast%3DGerdes%26aufirst%3DK.%26atitle%3DMessenger%2520RNA%2520Interferase%2520RelE%2520Controls%2520relBE%2520Transcription%2520by%2520Conditional%2520Cooperativity%26jtitle%3DMol.%2520Microbiol.%26date%3D2008%26volume%3D69%26spage%3D841%26epage%3D857%26doi%3D10.1111%2Fj.1365-2958.2008.06313.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">[20] Processing of X-Ray Diffraction Data Collected in Oscillation Mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1016/s0076-6879(97)76066-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=%5B20%5D+Processing+of+X-Ray+Diffraction+Data+Collected+in+Oscillation+Mode&doi=10.1016%2Fs0076-6879%2897%2976066-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0litk-b6WugLPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3D%255B20%255D%2520Processing%2520of%2520X-Ray%2520Diffraction%2520Data%2520Collected%2520in%2520Oscillation%2520Mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2Fs0076-6879%2897%2976066-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: A Comprehensive Python-Based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/s0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2Fs0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0litk-b6WugLPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2Fs0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0litk-b6WugLPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skubák, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span> <span> </span><span class="NLM_article-title">REFMAC5 for the Refinement of Macromolecular Crystal Structures</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1107/s0907444911001314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1107%2FS0907444911001314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=21460454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=355-367&author=G.+N.+Murshudovauthor=P.+Skub%C3%A1kauthor=A.+A.+Lebedevauthor=N.+S.+Pannuauthor=R.+A.+Steinerauthor=R.+A.+Nichollsauthor=M.+D.+Winnauthor=F.+Longauthor=A.+A.+Vagin&title=REFMAC5+for+the+Refinement+of+Macromolecular+Crystal+Structures&doi=10.1107%2Fs0907444911001314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">REFMAC5 for the refinement of macromolecular crystal structures</span></div><div class="casAuthors">Murshudov, Garib N.; Skubak, Pavol; Lebedev, Andrey A.; Pannu, Navraj S.; Steiner, Roberto A.; Nicholls, Robert A.; Winn, Martyn D.; Long, Fei; Vagin, Alexei A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">355-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">This paper describes various components of the macromol. crystallog. refinement program REFMAC5, which is distributed as part of the CCP4 suite.  REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS exptl. diffraction data.  To ensure chem. and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization.  Reliable models at resolns. at least as low as 4 Å can be achieved thanks to low-resoln. refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on at. displacement parameters (ADPs) based on the Kullback-Leibler divergence.  REFMAC5 addnl. offers TLS parameterization and, when high-resoln. data are available, fast refinement of anisotropic ADPs.  Refinement in the presence of twinning is performed in a fully automated fashion.  REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resoln. spectrum encountered in macromol. crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Jq-i2qIq5rVg90H21EOLACvtfcHk0litk-b6WugLPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D&md5=f8f3202d246908500057ad7c71015b7b</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1107%2FS0907444911001314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911001314%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DSkub%25C3%25A1k%26aufirst%3DP.%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DSteiner%26aufirst%3DR.%2BA.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DLong%26aufirst%3DF.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26atitle%3DREFMAC5%2520for%2520the%2520Refinement%2520of%2520Macromolecular%2520Crystal%2520Structures%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D355%26epage%3D367%26doi%3D10.1107%2Fs0907444911001314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Immormino, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: All-Atom Structure Validation for Macromolecular Crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1107/s0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1107%2Fs0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+All-Atom+Structure+Validation+for+Macromolecular+Crystallography&doi=10.1107%2Fs0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0liCCfEeMH49Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2Fs0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520All-Atom%2520Structure%2520Validation%2520for%2520Macromolecular%2520Crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2Fs0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delano, W. L.</span></span> <i>The PyMOL Molecular Graphics System</i>; <span class="NLM_publisher-name">DeLano Scientific</span>: <span class="NLM_publisher-loc">San Carlos, CA</span>, <span class="NLM_year">2002</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2002&author=W.+L.+Delano&title=The+PyMOL+Molecular+Graphics+System"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DDelano%26aufirst%3DW.%2BL.%26btitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26pub%3DDeLano%2520Scientific%26date%3D2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, C. L.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vieth, M.</span></span> <span> </span><span class="NLM_article-title">Detailed Analysis of Grid-based Molecular Docking: A Case Study of CDOCKER-A CHARMm-Based MD Docking Algorithm</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1549</span>– <span class="NLM_lpage">1562</span>, <span class="refDoi"> DOI: 10.1002/jcc.10306</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1002%2Fjcc.10306" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=12925999" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntFent70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2003&pages=1549-1562&author=G.+Wuauthor=D.+H.+Robertsonauthor=C.+L.+Brooksauthor=M.+Vieth&title=Detailed+Analysis+of+Grid-based+Molecular+Docking%3A+A+Case+Study+of+CDOCKER-A+CHARMm-Based+MD+Docking+Algorithm&doi=10.1002%2Fjcc.10306"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Detailed analysis of grid-based molecular docking: A case study of CDOCKER-A CHARMm-based MD docking algorithm</span></div><div class="casAuthors">Wu, Guosheng; Robertson, Daniel H.; Brooks, Charles L., III; Vieth, Michal</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1549-1562</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The influence of various factors on the accuracy of protein-ligand docking is examd.  The factors investigated include the role of a grid representation of protein-ligand interactions, the initial ligand conformation and orientation, the sampling rate of the energy hyper-surface, and the final minimization.  A representative docking method is used to study these factors, namely, CDOCKER, a mol. dynamics (MD) simulated-annealing-based algorithm.  A major emphasis in these studies is to compare the relative performance and accuracy of various grid-based approxns. to explicit all-atom force field calcns.  In these docking studies, the protein is kept rigid while the ligands are treated as fully flexible and a final minimization step is used to refine the docked poses.  A docking success rate of 74% is obsd. when an explicit all-atom representation of the protein (full force field) is used, while a lower accuracy of 66-76% is obsd. for grid-based methods.  All docking expts. considered a 41-member protein-ligand validation set.  A significant improvement in accuracy (76 vs. 66%) for the grid-based docking is achieved if the explicit all-atom force field is used in a final minimization step to refine the docking poses.  Statistical anal. shows that even lower-accuracy grid-based energy representations can be effectively used when followed with full force field minimization.  The results of these grid-based protocols are statistically indistinguishable from the detailed at. dockings and provide up to a sixfold redn. in computation time.  For the test case examd. here, improving the docking accuracy did not necessarily enhance the ability to est. binding affinities using the docked structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvNo4ih6vcd7Vg90H21EOLACvtfcHk0liCCfEeMH49Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXntFent70%253D&md5=34d16a1774c60ea18a5f7da58f478b71</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10306%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DG.%26aulast%3DRobertson%26aufirst%3DD.%2BH.%26aulast%3DBrooks%26aufirst%3DC.%2BL.%26aulast%3DVieth%26aufirst%3DM.%26atitle%3DDetailed%2520Analysis%2520of%2520Grid-based%2520Molecular%2520Docking%253A%2520A%2520Case%2520Study%2520of%2520CDOCKER-A%2520CHARMm-Based%2520MD%2520Docking%2520Algorithm%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2003%26volume%3D24%26spage%3D1549%26epage%3D1562%26doi%3D10.1002%2Fjcc.10306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 2 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Polina  Starkova</span>, <span class="hlFld-ContribAuthor ">Irina  Lazareva</span>, <span class="hlFld-ContribAuthor ">Alisa  Avdeeva</span>, <span class="hlFld-ContribAuthor ">Ofeliia  Sulian</span>, <span class="hlFld-ContribAuthor ">Darya  Likholetova</span>, <span class="hlFld-ContribAuthor ">Vladimir  Ageevets</span>, <span class="hlFld-ContribAuthor ">Marina  Lebedeva</span>, <span class="hlFld-ContribAuthor ">Vladimir  Gostev</span>, <span class="hlFld-ContribAuthor ">Julia  Sopova</span>, <span class="hlFld-ContribAuthor ">Sergey  Sidorenko</span>. </span><span class="cited-content_cbyCitation_article-title">Emergence of Hybrid Resistance and Virulence Plasmids Harboring New Delhi Metallo-β-Lactamase in Klebsiella pneumoniae in Russia. </span><span class="cited-content_cbyCitation_journal-name">Antibiotics</span><span> <strong>2021,</strong> <em>10 </em>
                                    (6)
                                     , 691. <a href="https://doi.org/10.3390/antibiotics10060691" title="DOI URL">https://doi.org/10.3390/antibiotics10060691</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/antibiotics10060691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fantibiotics10060691%26sid%3Dliteratum%253Aachs%26jtitle%3DAntibiotics%26atitle%3DEmergence%252Bof%252BHybrid%252BResistance%252Band%252BVirulence%252BPlasmids%252BHarboring%252BNew%252BDelhi%252BMetallo-%2525CE%2525B2-Lactamase%252Bin%252BKlebsiella%252Bpneumoniae%252Bin%252BRussia%26aulast%3DStarkova%26aufirst%3DPolina%26date%3D2021%26date%3D2021%26volume%3D10%26issue%3D6%26spage%3D691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sung-Min  Kang</span>, <span class="hlFld-ContribAuthor ">Heejo  Moon</span>, <span class="hlFld-ContribAuthor ">Sang-Woo  Han</span>, <span class="hlFld-ContribAuthor ">Byeong Wook  Kim</span>, <span class="hlFld-ContribAuthor ">Do-Hee  Kim</span>, <span class="hlFld-ContribAuthor ">Byeong Moon  Kim</span>, <span class="hlFld-ContribAuthor ">Bong-Jin  Lee</span>. </span><span class="cited-content_cbyCitation_article-title">Toxin-Activating Stapled Peptides Discovered by Structural Analysis Were Identified as New Therapeutic Candidates That Trigger Antibacterial Activity against Mycobacterium tuberculosis in the Mycobacterium smegmatis Model. </span><span class="cited-content_cbyCitation_journal-name">Microorganisms</span><span> <strong>2021,</strong> <em>9 </em>
                                    (3)
                                     , 568. <a href="https://doi.org/10.3390/microorganisms9030568" title="DOI URL">https://doi.org/10.3390/microorganisms9030568</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/microorganisms9030568&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmicroorganisms9030568%26sid%3Dliteratum%253Aachs%26jtitle%3DMicroorganisms%26atitle%3DToxin-Activating%252BStapled%252BPeptides%252BDiscovered%252Bby%252BStructural%252BAnalysis%252BWere%252BIdentified%252Bas%252BNew%252BTherapeutic%252BCandidates%252BThat%252BTrigger%252BAntibacterial%252BActivity%252Bagainst%252BMycobacterium%252Btuberculosis%252Bin%252Bthe%252BMycobacterium%252Bsmegmatis%252BModel%26aulast%3DKang%26aufirst%3DSung-Min%26date%3D2021%26date%3D2021%26volume%3D9%26issue%3D3%26spage%3D568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/medium/jm0c01118_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/large/jm0c01118_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01118&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/medium/jm0c01118_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/large/jm0c01118_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Overall structural information on <i>K. pneumoniae</i> VapBC (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY3">7BY3</a>). (A–C) Antitoxin VapB subunits are shown in yellow and the toxin VapC subunits are shown in light purple. (A) Heterotetrameric assembly of the VapBC complex containing an intermolecular interface formed by the dimerization of VapC. (B) Heterotetrameric assembly of the VapBC complex containing an intermolecular interface formed by the dimerization of the N-terminal DNA-binding domain of VapB. The DNA-binding domain of VapB is labeled. (C) Schematic diagram of the VapBC heterodimer showing the secondary structural architecture. (D) Secondary structure analysis of the N-terminal DNA-binding domain of VapB. Prediction data using the PSIPRED server are presented with structural information on the N-terminus of VapB in the crystal structure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/large/jm0c01118_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01118&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/medium/jm0c01118_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/large/jm0c01118_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structural comparison of <i>K. pneumoniae</i> VapBC with its homologues. (A) Sequence alignment of <i>K. pneumoniae</i> VapC with its homologues. The secondary structural elements of these proteins are shown above the alignment. Highly conserved residues are highlighted in red and yellow. Among them, the acidic active-site residues and residual residues are marked as red and blue triangles, respectively. (B) Electrostatic surface of VapC (left) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY3">7BY3</a>). Positive and negative charges are colored blue and red, respectively. An acidic active site of VapC is shown enlarged in cartoon and stick representations (right) showing the locations of seven conserved residues in the structure of each homologue. Seven conserved residues are shown in stick representation. (C) Comparison of the VapBC complexes (PDB IDs: in the order of <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY3">7BY3</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6A7V">6A7V</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5X3T">5X3T</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K8J">5K8J</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6NKL">6NKL</a>). Each VapBC complex is shown in the same direction based on the VapC structure. For VapC, the same colors as in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B are used.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/large/jm0c01118_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01118&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/medium/jm0c01118_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/large/jm0c01118_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Active site and RNase activities of VapC. (A) Two forms of the VapBC complex structure: open form (left) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY2">7BY2</a>) and closed form (right) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY3">7BY3</a>). (B) Two forms of the active site are presented with a cartoon diagram of VapC. Four conserved residues are shown in stick representation. Active-site conformations are denoted as black dotted squares and enlarged. The 2m<i>Fo</i>-<i>DFc</i> electron density map of the Mg<sup>2+</sup> site contouring at σ (calculated with PHENIX) is also displayed. In the open form of the VapC active site, the coordinated Mg<sup>2+</sup> is located in the midst of residues D9, E43, and D90. In the closed form of the VapC active site, R79 of VapB triggers the displacement of Mg<sup>2+</sup>. Consequently, R79 of VapB fills the original location of Mg<sup>2+</sup>, and Mg<sup>2+</sup> moves to the middle of D9, D90, and D111. (C) <i>In vitro</i> RNase activities of VapC, VapC mutants, and VapBC. Proteins were treated with EDTA to remove metal ions, and 10 mM Mg<sup>2+</sup> was then added to proteins. In the assay, 10 μM proteins were used, and a general RNase inhibitor (40 units) was used to prevent unintended contamination. The kinetics for the RNase activities of these proteins were recorded for 1 h. The data shown are representative of three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/large/jm0c01118_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01118&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/medium/jm0c01118_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/large/jm0c01118_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Heterodimeric interface of the VapBC complex (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY3">7BY3</a>) and efficacy of complex disruption using the designed peptides. (A,B) Residues involved in interaction networks are shown in stick representation. Schematic diagrams of the hydrophilic and hydrophobic interactions are provided in the table. (A) Intermolecular interaction network of the VapB α3 helix. (B) Intermolecular interaction network of the VapC α1 helix. (C,D) RNase activities resulting from peptides added to the VapBC complex. Activities generated by peptides are plotted in proportion to that of VapC. The pretreatments for the assay were the same as those described in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C. In these complex disruption assays, 100 μM peptides were added to 10 μM proteins, and their complex disruption efficacies were evaluated. Data are represented as the mean of three independent replicates ±SD. (C) Efficacy of peptides designed based on the sequence of the α3 helix of VapB. (D) Efficacy of peptides designed based on the sequence of the α1 helix of VapC.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/large/jm0c01118_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01118&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/medium/jm0c01118_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/large/jm0c01118_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Surface pockets on VapC (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY3">7BY3</a>) and efficacy of final compounds expressed with molecular docking. (A) Surface of VapC with a cartoon representation of VapB. The negatively charged active-site cavity and surface cleft interface pockets are linked through the α2−α3 loop of VapB. The compound-binding site of VapC is located on the surface cleft interface pockets represented as the interface pocket valley. (B) RNase activities of compounds when the VapBC complex was added. Activities generated by compounds are plotted in proportion to that of VapC. The activities of compounds without the VapBC complex were also measured. Pretreatments for these assays were the same as those described in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C. In these complex disruption assays, compound (1, 3.16, and 10 μM) was added to 10 μM protein, and the complex disruption efficacy was evaluated. The activity was measured based on the initial velocity during the first hour. Data are represented as the mean of three independent replicates ±SD. (C,D) Results of molecular docking of the compounds on the VapC structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY3">7BY3</a>) with the detailed interaction networks. (C) Interaction data for compound 1. (D) Interaction data for compound 2. In the cartoon diagram, ligand-binding residues of VapC are shown in stick and labeled. 2D ligand-VapC interaction diagrams were generated using LigPlot<sup>+</sup>.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Compounds 1 and 2 are shown in green and pink, respectively. The residues involved in HBs are shown, and HBs are represented by blue dashes. The residues making nonbonded contacts with the ligand are shown in the red spoked arcs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01118/20201120/images/large/jm0c01118_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01118&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i25">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43164" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43164" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 47 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-García, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blasco, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Contreras, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomas, M.</span></span> <span> </span><span class="NLM_article-title">Toxin-Antitoxin Systems in Clinical Pathogens</span>. <i>Toxins</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">227</span>, <span class="refDoi"> DOI: 10.3390/toxins8070227</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.3390%2Ftoxins8070227" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Cls7vK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=227&author=L.+Fern%C3%A1ndez-Garc%C3%ADaauthor=L.+Blascoauthor=M.+Lopezauthor=G.+Bouauthor=R.+Garc%C3%ADa-Contrerasauthor=T.+Woodauthor=M.+Tomas&title=Toxin-Antitoxin+Systems+in+Clinical+Pathogens&doi=10.3390%2Ftoxins8070227"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Toxin-antitoxin systems in clinical pathogens</span></div><div class="casAuthors">Fernandez-Garcia, Laura; Blasco, Lucia; Lopez, Maria; Bou, German; Garcia-Contreras, Rodolfo; Wood, Thomas; Tomas, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Toxins</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">227/1-227/23</span>CODEN:
                <span class="NLM_cas:coden">TOXIB7</span>;
        ISSN:<span class="NLM_cas:issn">2072-6651</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Toxin-antitoxin (TA) systems are prevalent in bacteria and archaea.  Although not essential for normal cell growth, TA systems are implicated in multiple cellular functions assocd. with survival under stress conditions.  Clin. strains of bacteria are currently causing major human health problems as a result of their multidrug resistance, persistence and strong pathogenicity.  Here, we present a review of the TA systems described to date and their biol. role in human pathogens belonging to the ESKAPE group (Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa and Enterobacter spp.) and others of clin. relevance (Escherichia coli, Burkholderia spp., Streptococcus spp. and Mycobacterium tuberculosis).  Better understanding of the mechanisms of action of TA systems will enable the development of new lines of treatment for infections caused by the above-mentioned pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeDgNrGshJpLVg90H21EOLACvtfcHk0lho2H6Tbyd8qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Cls7vK&md5=0919f098873bbf6158b04767420fa107</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.3390%2Ftoxins8070227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Ftoxins8070227%26sid%3Dliteratum%253Aachs%26aulast%3DFern%25C3%25A1ndez-Garc%25C3%25ADa%26aufirst%3DL.%26aulast%3DBlasco%26aufirst%3DL.%26aulast%3DLopez%26aufirst%3DM.%26aulast%3DBou%26aufirst%3DG.%26aulast%3DGarc%25C3%25ADa-Contreras%26aufirst%3DR.%26aulast%3DWood%26aufirst%3DT.%26aulast%3DTomas%26aufirst%3DM.%26atitle%3DToxin-Antitoxin%2520Systems%2520in%2520Clinical%2520Pathogens%26jtitle%3DToxins%26date%3D2016%26volume%3D8%26spage%3D227%26doi%3D10.3390%2Ftoxins8070227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gomez-Simmonds, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uhlemann, A.-C.</span></span> <span> </span><span class="NLM_article-title">Clinical Implications of Genomic Adaptation and Evolution of Carbapenem-Resistant Klebsiella pneumoniae</span>. <i>J. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>215</i></span>,  <span class="NLM_fpage">S18</span>– <span class="NLM_lpage">S27</span>, <span class="refDoi"> DOI: 10.1093/infdis/jiw378</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Finfdis%2Fjiw378" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=28375514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFCru7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=215&publication_year=2017&pages=S18-S27&author=A.+Gomez-Simmondsauthor=A.-C.+Uhlemann&title=Clinical+Implications+of+Genomic+Adaptation+and+Evolution+of+Carbapenem-Resistant+Klebsiella+pneumoniae&doi=10.1093%2Finfdis%2Fjiw378"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical implications of genomic adaptation and evolution of carbapenem-resistant Klebsiella pneumoniae</span></div><div class="casAuthors">Gomez-Simmonds, Angela; Uhlemann, Anne-Catrin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Infectious Diseases</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">215</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">S18-S27</span>CODEN:
                <span class="NLM_cas:coden">JIDIAQ</span>;
        ISSN:<span class="NLM_cas:issn">1537-6613</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Klebsiella pneumoniae poses a major challenge to healthcare worldwide as an important cause of multidrug-resistant infections.  Nosocomial clones, including epidemic sequence type 258 (ST258), have shown an affinity for acquiring and disseminating resistance plasmids, particularly variants of the K. pneumoniae carbapenemase.  By comparison, the resurgence of severe communityassocd. K. pneumoniae infections has led to increased recognition of hypervirulent strains belonging to the K1 and K2 capsular serotypes, predominantly in eastern Asia.  Genomic and functional studies suggest that a variety of virulence and immune evasive factors contribute to the success of nosocomial and community-assocd. clonal lineages, aided by mechanisms of genetic plasticity that contribute to uptake of genes assocd. with antimicrobial resistance and pathogenicity.  While there currently appears to be limited overlap between resistant and hypervirulent lineages, specific bacterial and host factors contributing to the emergence of dominant clones remain incompletely understood.  This review summarizes recent advances in our understanding of the mol. epidemiol., virulence potential, and host-pathogen interactions of K. pneumoniae.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGompRNHue-TUbVg90H21EOLACvtfcHk0ljw4gnBcB93Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFCru7nM&md5=f59b65bb049121e3c95aaa564ec52da1</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1093%2Finfdis%2Fjiw378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Finfdis%252Fjiw378%26sid%3Dliteratum%253Aachs%26aulast%3DGomez-Simmonds%26aufirst%3DA.%26aulast%3DUhlemann%26aufirst%3DA.-C.%26atitle%3DClinical%2520Implications%2520of%2520Genomic%2520Adaptation%2520and%2520Evolution%2520of%2520Carbapenem-Resistant%2520Klebsiella%2520pneumoniae%26jtitle%3DJ.%2520Infect.%2520Dis.%26date%3D2017%26volume%3D215%26spage%3DS18%26epage%3DS27%26doi%3D10.1093%2Finfdis%2Fjiw378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ogura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiraga, S.</span></span> <span> </span><span class="NLM_article-title">Mini-F Plasmid Genes that Couple Host Cell Division to Plasmid Proliferation</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">4784</span>– <span class="NLM_lpage">4788</span>, <span class="refDoi"> DOI: 10.1073/pnas.80.15.4784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1073%2Fpnas.80.15.4784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=6308648" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADyaL3sXltFKqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=1983&pages=4784-4788&author=T.+Oguraauthor=S.+Hiraga&title=Mini-F+Plasmid+Genes+that+Couple+Host+Cell+Division+to+Plasmid+Proliferation&doi=10.1073%2Fpnas.80.15.4784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Mini-F plasmid genes that couple host cell division to plasmid proliferation</span></div><div class="casAuthors">Ogura, Teru; Hiraga, Sota</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4784-8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">A mechanism for stable maintenance of plasmids, besides the replication and partition mechanisms, was specified by genes of a mini-F plasmid.  An oriC plasmid carrying both a mini-F segment necessary for partition [coordinates 46.4-49.4 kilobase pairs (kb) on the F map] and another segment (42.9-43.6 kb), designated ccd (coupled cell division), is more stably maintained than are oriC plasmids carrying only the partition segment; the stability is comparable to that of the parental mini-F plasmid.  When replication of a plasmid carrying ccd is prevented and the plasmid copy no. decreases, to ≥1/cell, host cell division is inhibited but turbidity increases and chromosome replication are not.  The appearance of plasmid-free segregants is therefore effectively prevented under such conditions.  Exptl. results suggest that redn. of the copy no. of plasmids carrying the ccd region causes an inhibition of cell division and that the ccd region can be dissected into 2 functional regions; 1 (ccdB) inhibits cell division, and the other (ccdA) releases the inhibition.  The interplay of the ccdA and ccdB genes promotes stable plasmid maintenance by coupling host cell division to plasmid proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdSkN-dtoMX7Vg90H21EOLACvtfcHk0ljw4gnBcB93Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXltFKqsLs%253D&md5=9966ff0c9dbc34be2a612623d3fbb311</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1073%2Fpnas.80.15.4784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.80.15.4784%26sid%3Dliteratum%253Aachs%26aulast%3DOgura%26aufirst%3DT.%26aulast%3DHiraga%26aufirst%3DS.%26atitle%3DMini-F%2520Plasmid%2520Genes%2520that%2520Couple%2520Host%2520Cell%2520Division%2520to%2520Plasmid%2520Proliferation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1983%26volume%3D80%26spage%3D4784%26epage%3D4788%26doi%3D10.1073%2Fpnas.80.15.4784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerdes, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molin, S.</span></span> <span> </span><span class="NLM_article-title">Unique Type of Plasmid Maintenance Function: Postsegregational Killing of Plasmid-Free Cells</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">3116</span>– <span class="NLM_lpage">3120</span>, <span class="refDoi"> DOI: 10.1073/pnas.83.10.3116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1073%2Fpnas.83.10.3116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=3517851" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADyaL28XktF2ms7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=1986&pages=3116-3120&author=K.+Gerdesauthor=P.+B.+Rasmussenauthor=S.+Molin&title=Unique+Type+of+Plasmid+Maintenance+Function%3A+Postsegregational+Killing+of+Plasmid-Free+Cells&doi=10.1073%2Fpnas.83.10.3116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Unique type of plasmid maintenance function:  postsegregational killing of plasmid-free cells</span></div><div class="casAuthors">Gerdes, Kenn; Rasmussen, Poul B.; Molin, Soeren</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3116-20</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">The stability locus parB+ of plasmid R1 specifies a unique type of plasmid maintenance function.  Two genes, hok (host killing) and sok (suppressor of killing), are required for the stabilizing activity.  The hok gene encodes a highly toxic gene product, whose overexpression causes a rapid killing and a concomitant dramatic change in morphol. of the host cell.  The other gene, sok, was found to encode a product that counteracts the hok gene-mediated killing.  The parB+ region was inserted in a plasmid with a temp.-sensitive replication system.  At nonpermissive temp., the parB+ plasmid was maintained in the population for a significantly longer period than the corresponding parB- plasmid.  Coupled to this extended maintenance, a large fraction of the population was shown to be nonviable plasmid-free cells with the characteristic hok-induced change in morphol.  Thus, the parB+ locus mediates plasmid stability by killing cells that have lost the parB+ plasmid during the preceding cell division, thereby ensuring that a growing bacterial culture predominantly consists of plasmid-contg. cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpinbKsCNHGErVg90H21EOLACvtfcHk0ljw4gnBcB93Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28XktF2ms7k%253D&md5=a8c89d2381eef482d2151b9aab695071</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.83.10.3116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.83.10.3116%26sid%3Dliteratum%253Aachs%26aulast%3DGerdes%26aufirst%3DK.%26aulast%3DRasmussen%26aufirst%3DP.%2BB.%26aulast%3DMolin%26aufirst%3DS.%26atitle%3DUnique%2520Type%2520of%2520Plasmid%2520Maintenance%2520Function%253A%2520Postsegregational%2520Killing%2520of%2520Plasmid-Free%2520Cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1986%26volume%3D83%26spage%3D3116%26epage%3D3120%26doi%3D10.1073%2Fpnas.83.10.3116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hazan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sat, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelberg-Kulka, H.</span></span> <span> </span><span class="NLM_article-title">Escherichia coli mazEF-Mediated Cell Death is Triggered by Various Stressful Conditions</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>186</i></span>,  <span class="NLM_fpage">3663</span>– <span class="NLM_lpage">3669</span>, <span class="refDoi"> DOI: 10.1128/jb.186.11.3663-3669.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1128%2FJB.186.11.3663-3669.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=15150257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BD2cXks1CqtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=186&publication_year=2004&pages=3663-3669&author=R.+Hazanauthor=B.+Satauthor=H.+Engelberg-Kulka&title=Escherichia+coli+mazEF-Mediated+Cell+Death+is+Triggered+by+Various+Stressful+Conditions&doi=10.1128%2Fjb.186.11.3663-3669.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Escherichia coli mazEF-mediated cell death is triggered by various stressful conditions</span></div><div class="casAuthors">Hazan, Ronen; Sat, Boaz; Engelberg-Kulka, Hanna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">186</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3663-3669</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">MazEF is an Escherichia coli suicide module specific for a stable toxin and a labile antitoxin.  Inhibiting mazEF expression appeared to activate the module to cause cell death.  Here, the authors show that several stressful conditions, including high temps., DNA damage, and oxidative stress, also induce mazEF-mediated cell death.  They also show that this process takes place only during logarithmic growth and requires an intact relA gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbTDNCICt-J7Vg90H21EOLACvtfcHk0ljw4gnBcB93Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXks1CqtLY%253D&md5=fb645a023b387fc3d93eb9161a6d13c7</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1128%2FJB.186.11.3663-3669.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.186.11.3663-3669.2004%26sid%3Dliteratum%253Aachs%26aulast%3DHazan%26aufirst%3DR.%26aulast%3DSat%26aufirst%3DB.%26aulast%3DEngelberg-Kulka%26aufirst%3DH.%26atitle%3DEscherichia%2520coli%2520mazEF-Mediated%2520Cell%2520Death%2520is%2520Triggered%2520by%2520Various%2520Stressful%2520Conditions%26jtitle%3DJ.%2520Bacteriol.%26date%3D2004%26volume%3D186%26spage%3D3663%26epage%3D3669%26doi%3D10.1128%2Fjb.186.11.3663-3669.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sat, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khaner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glaser, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelberg-Kulka, H.</span></span> <span> </span><span class="NLM_article-title">Programmed Cell Death in Escherichia coli: Some Antibiotics can Trigger mazEF Lethality</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>183</i></span>,  <span class="NLM_fpage">2041</span>– <span class="NLM_lpage">2045</span>, <span class="refDoi"> DOI: 10.1128/jb.183.6.2041-2045.2001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1128%2FJB.183.6.2041-2045.2001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=11222603" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhvVeksbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=183&publication_year=2001&pages=2041-2045&author=B.+Satauthor=R.+Hazanauthor=T.+Fisherauthor=H.+Khanerauthor=G.+Glaserauthor=H.+Engelberg-Kulka&title=Programmed+Cell+Death+in+Escherichia+coli%3A+Some+Antibiotics+can+Trigger+mazEF+Lethality&doi=10.1128%2Fjb.183.6.2041-2045.2001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Programmed cell death in Escherichia coli: some antibiotics can trigger mazEF lethality</span></div><div class="casAuthors">Sat, Boaz; Hazan, Ronen; Fisher, Tova; Khaner, Hanita; Glaser, Gad; Engelberg-Kulka, Hanna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">183</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2041-2045</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The discovery of toxin-antitoxin gene pairs (also called addiction modules) on extrachromosomal elements of Escherichia coli, and particularly the discovery of homologous modules on the bacterial chromosome, suggest that a potential for programmed cell death may be inherent in bacterial cultures.  We have reported on the E. coli mazEF system, a regulatable addiction module located on the bacterial chromosome.  MazF is a stable toxin and MazE is a labile antitoxin.  Here we show that cell death mediated by the E. coli mazEF module can be triggered by several antibiotics (rifampicin, chloramphenicol, and spectinomycin) that are general inhibitors of transcription and/or translation.  These antibiotics inhibit the continuous expression of the labile antitoxin MazE, and as a result, the stable toxin MazF causes cell death.  Our results have implications for the possible mode(s) of action of this group of antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTJmRULMgG_bVg90H21EOLACvtfcHk0lhoSoXWP8RYxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhvVeksbY%253D&md5=68f8ec328695b31a1e33c080bcac428b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1128%2FJB.183.6.2041-2045.2001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.183.6.2041-2045.2001%26sid%3Dliteratum%253Aachs%26aulast%3DSat%26aufirst%3DB.%26aulast%3DHazan%26aufirst%3DR.%26aulast%3DFisher%26aufirst%3DT.%26aulast%3DKhaner%26aufirst%3DH.%26aulast%3DGlaser%26aufirst%3DG.%26aulast%3DEngelberg-Kulka%26aufirst%3DH.%26atitle%3DProgrammed%2520Cell%2520Death%2520in%2520Escherichia%2520coli%253A%2520Some%2520Antibiotics%2520can%2520Trigger%2520mazEF%2520Lethality%26jtitle%3DJ.%2520Bacteriol.%26date%3D2001%26volume%3D183%26spage%3D2041%26epage%3D2045%26doi%3D10.1128%2Fjb.183.6.2041-2045.2001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, T. K.</span></span> <span> </span><span class="NLM_article-title">Toxin-Antitoxin Systems Influence Biofilm and Persister Cell Formation and the General Stress Response</span>. <i>Appl. Environ. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">5577</span>– <span class="NLM_lpage">5583</span>, <span class="refDoi"> DOI: 10.1128/aem.05068-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1128%2FAEM.05068-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=21685157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFGisL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2011&pages=5577-5583&author=X.+Wangauthor=T.+K.+Wood&title=Toxin-Antitoxin+Systems+Influence+Biofilm+and+Persister+Cell+Formation+and+the+General+Stress+Response&doi=10.1128%2Faem.05068-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Toxin-antitoxin systems influence biofilm and persister cell formation and the general stress response</span></div><div class="casAuthors">Wang, Xiaoxue; Wood, Thomas K.</div><div class="citationInfo"><span class="NLM_cas:title">Applied and Environmental Microbiology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5577-5583</span>CODEN:
                <span class="NLM_cas:coden">AEMIDF</span>;
        ISSN:<span class="NLM_cas:issn">0099-2240</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review.  In many genomes, toxin-antitoxin (TA) systems have been identified; however, their role in cell physiol. has been unclear.  Here, the authors examine the evidence that TA systems are involved in biofilm formation and persister cell formation and that these systems may be important regulators of the switch from the planktonic to the biofilm lifestyle as a stress response by their control of secondary messenger 3',5'-cyclic diguanylic acid.  Specifically, upon stress, the sequence-specific mRNA interferases MqsR and MazF mediate cell survival.  The authors propose that TA systems are not redundant, as they may have developed to respond to specific stresses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodVbjold1jJ7Vg90H21EOLACvtfcHk0lhoSoXWP8RYxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFGisL%252FO&md5=70996635abf2d601c582499fd88c6b0c</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1128%2FAEM.05068-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAEM.05068-11%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWood%26aufirst%3DT.%2BK.%26atitle%3DToxin-Antitoxin%2520Systems%2520Influence%2520Biofilm%2520and%2520Persister%2520Cell%2520Formation%2520and%2520the%2520General%2520Stress%2520Response%26jtitle%3DAppl.%2520Environ.%2520Microbiol.%26date%3D2011%26volume%3D77%26spage%3D5577%26epage%3D5583%26doi%3D10.1128%2Faem.05068-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dörr, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vulić, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, K.</span></span> <span> </span><span class="NLM_article-title">Ciprofloxacin Causes Persister Formation by Inducing the TisB Toxin in Escherichia coli</span>. <i>PLoS Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e1000317</span> <span class="refDoi"> DOI: 10.1371/journal.pbio.1000317</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1371%2Fjournal.pbio.1000317" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=20186264" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2010&author=T.+D%C3%B6rrauthor=M.+Vuli%C4%87author=K.+Lewis&title=Ciprofloxacin+Causes+Persister+Formation+by+Inducing+the+TisB+Toxin+in+Escherichia+coli&doi=10.1371%2Fjournal.pbio.1000317"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pbio.1000317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pbio.1000317%26sid%3Dliteratum%253Aachs%26aulast%3DD%25C3%25B6rr%26aufirst%3DT.%26aulast%3DVuli%25C4%2587%26aufirst%3DM.%26aulast%3DLewis%26aufirst%3DK.%26atitle%3DCiprofloxacin%2520Causes%2520Persister%2520Formation%2520by%2520Inducing%2520the%2520TisB%2520Toxin%2520in%2520Escherichia%2520coli%26jtitle%3DPLoS%2520Biol.%26date%3D2010%26volume%3D8%26doi%3D10.1371%2Fjournal.pbio.1000317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harms, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brodersen, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitarai, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerdes, K.</span></span> <span> </span><span class="NLM_article-title">Toxins, Targets, and Triggers: An Overview of Toxin-Antitoxin Biology</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">768</span>– <span class="NLM_lpage">784</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2018.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1016%2Fj.molcel.2018.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=29398446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVSit7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2018&pages=768-784&author=A.+Harmsauthor=D.+E.+Brodersenauthor=N.+Mitaraiauthor=K.+Gerdes&title=Toxins%2C+Targets%2C+and+Triggers%3A+An+Overview+of+Toxin-Antitoxin+Biology&doi=10.1016%2Fj.molcel.2018.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Toxins, Targets, and Triggers: An Overview of Toxin-Antitoxin Biology</span></div><div class="casAuthors">Harms, Alexander; Brodersen, Ditlev Egeskov; Mitarai, Namiko; Gerdes, Kenn</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">768-784</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Bacterial toxin-antitoxin (TA) modules are abundant genetic elements that encode a toxin protein capable of inhibiting cell growth and an antitoxin that counteracts the toxin.  The majority of toxins are enzymes that interfere with translation or DNA replication, but a wide variety of mol. activities and cellular targets have been described.  Antitoxins are proteins or RNAs that often control their cognate toxins through direct interactions and, in conjunction with other signaling elements, through transcriptional and translational regulation of TA module expression.  Three major biol. functions of TA modules have been discovered, post-segregational killing ("plasmid addiction"), abortive infection (bacteriophage immunity through altruistic suicide), and persister formation (antibiotic tolerance through dormancy).  In this review, we summarize the current state of the field and highlight how multiple levels of regulation shape the conditions of toxin activation to achieve the different biol. functions of TA modules.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSdSgFK0WMK7Vg90H21EOLACvtfcHk0lhoSoXWP8RYxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVSit7s%253D&md5=fb6f8ea512c48716af378608d0b10eca</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2018.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2018.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DHarms%26aufirst%3DA.%26aulast%3DBrodersen%26aufirst%3DD.%2BE.%26aulast%3DMitarai%26aufirst%3DN.%26aulast%3DGerdes%26aufirst%3DK.%26atitle%3DToxins%252C%2520Targets%252C%2520and%2520Triggers%253A%2520An%2520Overview%2520of%2520Toxin-Antitoxin%2520Biology%26jtitle%3DMol.%2520Cell%26date%3D2018%26volume%3D70%26spage%3D768%26epage%3D784%26doi%3D10.1016%2Fj.molcel.2018.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Son, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, B.-J.</span></span> <span> </span><span class="NLM_article-title">Structural Overview of Toxin-Antitoxin Systems in Infectious Bacteria: A Target for Developing Antimicrobial Agents</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>1834</i></span>,  <span class="NLM_fpage">1155</span>– <span class="NLM_lpage">1167</span>, <span class="refDoi"> DOI: 10.1016/j.bbapap.2013.02.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1016%2Fj.bbapap.2013.02.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=23459128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnslCitb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1834&publication_year=2013&pages=1155-1167&author=S.+J.+Parkauthor=W.+S.+Sonauthor=B.-J.+Lee&title=Structural+Overview+of+Toxin-Antitoxin+Systems+in+Infectious+Bacteria%3A+A+Target+for+Developing+Antimicrobial+Agents&doi=10.1016%2Fj.bbapap.2013.02.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Structural overview of toxin-antitoxin systems in infectious bacteria: A target for developing antimicrobial agents</span></div><div class="casAuthors">Park, Sung Jean; Son, Woo Sung; Lee, Bong-Jin</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Proteins and Proteomics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">1834</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1155-1167</span>CODEN:
                <span class="NLM_cas:coden">BBAPBW</span>;
        ISSN:<span class="NLM_cas:issn">1570-9639</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">A review.  The bacterial toxin-antitoxin (TA) system is a module that may play a role in cell survival under stress conditions.  Generally, toxin mols. act as neg. regulators in cell survival and antitoxin mols. as pos. regulators.  Thus, the expression levels and interactions between toxins and antitoxins should be systematically harmonized so that bacteria can escape such harmful conditions.  Since TA systems are able to control the fate of bacteria, they are considered potent targets for the development of new antimicrobial agents.  TA systems are widely prevalent with a variety of systems existing in bacteria: there are three types of bacterial TA systems depending on the property of the antitoxin which binds either the protein toxin or mRNA coding the toxin protein.  Moreover, the multiplicity of TA genes has been obsd. even in species of bacteria.  Therefore, knowledge on TA systems such as the individual characteristics of TA systems, integrative working mechanisms of various TA systems in bacteria, interactions between toxin mols. and cellular targets, and so on is currently limited due to their complexity.  In this regard, it would be helpful to know the structural characteristics of TA modules for understanding TA systems in bacteria.  Until now, 85 out of the total structures deposited in PDB have been bacterial TA system proteins including TA complexes or isolated toxins/antitoxins.  Here, the authors summarize the structural information of TA systems and analyzed the structural characteristics of known TA modules from several bacteria, esp. focusing on the TA modules of several infectious bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojinQZ7DF55rVg90H21EOLACvtfcHk0ljBKF3AvwtSpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnslCitb8%253D&md5=6c50d7149dc61fc4d081317ee0cc158b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.bbapap.2013.02.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbapap.2013.02.027%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DS.%2BJ.%26aulast%3DSon%26aufirst%3DW.%2BS.%26aulast%3DLee%26aufirst%3DB.-J.%26atitle%3DStructural%2520Overview%2520of%2520Toxin-Antitoxin%2520Systems%2520in%2520Infectious%2520Bacteria%253A%2520A%2520Target%2520for%2520Developing%2520Antimicrobial%2520Agents%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2013%26volume%3D1834%26spage%3D1155%26epage%3D1167%26doi%3D10.1016%2Fj.bbapap.2013.02.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kang, S.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, B.-J.</span></span> <span> </span><span class="NLM_article-title">A Systematic Overview of Type II and III Toxin-Antitoxin Systems with a Focus on Druggability</span>. <i>Toxins</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">515</span>, <span class="refDoi"> DOI: 10.3390/toxins10120515</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.3390%2Ftoxins10120515" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFGmt7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=515&author=S.-M.+Kangauthor=D.-H.+Kimauthor=C.+Jinauthor=B.-J.+Lee&title=A+Systematic+Overview+of+Type+II+and+III+Toxin-Antitoxin+Systems+with+a+Focus+on+Druggability&doi=10.3390%2Ftoxins10120515"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A systematic overview of type II and III toxin-antitoxin systems with a focus on druggability</span></div><div class="casAuthors">Kang, Sung-Min; Kim, Do-Hee; Jin, Chenglong; Lee, Bong-Jin</div><div class="citationInfo"><span class="NLM_cas:title">Toxins</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">515</span>CODEN:
                <span class="NLM_cas:coden">TOXIB7</span>;
        ISSN:<span class="NLM_cas:issn">2072-6651</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Toxin-antitoxin (TA) systems are known to play various roles in physiol. processes, such as gene regulation, growth arrest and survival, in bacteria exposed to environmental stress.  Type II TA systems comprise natural complexes consisting of protein toxins and antitoxins.  Each toxin and antitoxin participates in distinct regulatory mechanisms depending on the type of TA system.  Recently, peptides designed by mimicking the interfaces between TA complexes showed its potential to activate the activity of toxin by competing its binding counterparts.  Type II TA systems occur more often in pathogenic bacteria than in their nonpathogenic kin.  Therefore, they can be possible drug targets, because of their high abundance in some pathogenic bacteria, such as Mycobacterium tuberculosis.  In addn., recent bioinformatic analyses have shown that type III TA systems are highly abundant in the intestinal microbiota, and recent clin. studies have shown that the intestinal microbiota is linked to inflammatory diseases, obesity and even several types of cancer.  We therefore focused on exploring the putative relationship between intestinal microbiota-related human diseases and type III TA systems.  In this paper, we review and discuss the development of possible druggable materials based on the mechanism of type II and type III TA system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNB2-Jmv9N1LVg90H21EOLACvtfcHk0ljBKF3AvwtSpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFGmt7zO&md5=8766577046b447ea3ee73ee1d90969cc</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.3390%2Ftoxins10120515&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Ftoxins10120515%26sid%3Dliteratum%253Aachs%26aulast%3DKang%26aufirst%3DS.-M.%26aulast%3DKim%26aufirst%3DD.-H.%26aulast%3DJin%26aufirst%3DC.%26aulast%3DLee%26aufirst%3DB.-J.%26atitle%3DA%2520Systematic%2520Overview%2520of%2520Type%2520II%2520and%2520III%2520Toxin-Antitoxin%2520Systems%2520with%2520a%2520Focus%2520on%2520Druggability%26jtitle%3DToxins%26date%3D2018%26volume%3D10%26spage%3D515%26doi%3D10.3390%2Ftoxins10120515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, H.-Y.</span></span> <span> </span><span class="NLM_article-title">TADB 2.0: An Updated Database of Bacterial Type II Toxin-Antitoxin Loci</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">D749</span>– <span class="NLM_lpage">D753</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx1033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgkx1033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=29106666" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGitr%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D749-D753&author=Y.+Xieauthor=Y.+Weiauthor=Y.+Shenauthor=X.+Liauthor=H.+Zhouauthor=C.+Taiauthor=Z.+Dengauthor=H.-Y.+Ou&title=TADB+2.0%3A+An+Updated+Database+of+Bacterial+Type+II+Toxin-Antitoxin+Loci&doi=10.1093%2Fnar%2Fgkx1033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">TADB 2.0: an updated database of bacterial type II toxin-antitoxin loci</span></div><div class="casAuthors">Xie, Yingzhou; Wei, Yiqing; Shen, Yue; Li, Xiaobin; Zhou, Hao; Tai, Cui; Deng, Zixin; Ou, Hong-Yu</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D749-D753</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">TADB2.0 is an updated database that provides comprehensive information about bacterial type II toxin-antitoxin (TA) loci.  Compared with the previous version, the database refined and the new data schema is employed.  With the aid of text mining and manual curation, it recorded 6193 type II TA loci in 870 replicons of bacteria and archaea, including 105 exptl. validated TA loci.  In addn., the newly developed tool TAfinder combines the homolog searches and the operon structure detection, allowing the prediction for type II TA pairs in bacterial genome sequences.  It also helps to investigate the genomic context of predicted TA loci for putative virulence factors, antimicrobial resistance determinants and mobile genetic elements via alignments to the specific public databases.  Addnl., the module TAfinder-Compare allows comparing the presence of the given TA loci across the close relative genomes.  With the recent updates, TADB2.0 might provide better support for understanding the important roles of type II TA systems in the prokaryotic life activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-4pI4aJE6HbVg90H21EOLACvtfcHk0ljBKF3AvwtSpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlGitr%252FL&md5=5972896b514bff8ebfccb62bdb3b9a8a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx1033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx1033%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DTai%26aufirst%3DC.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DOu%26aufirst%3DH.-Y.%26atitle%3DTADB%25202.0%253A%2520An%2520Updated%2520Database%2520of%2520Bacterial%2520Type%2520II%2520Toxin-Antitoxin%2520Loci%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26spage%3DD749%26epage%3DD753%26doi%3D10.1093%2Fnar%2Fgkx1033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burley, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhikadiya, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Costanzo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalenberg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duarte, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guranović, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzenko, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalro, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namkoong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peisach, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Periskova, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prlić, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Randle, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sala, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekharan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, Y.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valasatava, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voigt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuravleva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zardecki, C.</span></span> <span> </span><span class="NLM_article-title">RCSB Protein Data Bank: Biological Macromolecular Structures Enabling Research and Education in Fundamental Biology, Biomedicine, Biotechnology and Energy</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">D464</span>– <span class="NLM_lpage">D474</span>, <span class="refDoi"> DOI: 10.1093/nar/gky1004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgky1004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=30357411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1Cgt7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D464-D474&author=S.+K.+Burleyauthor=H.+M.+Bermanauthor=C.+Bhikadiyaauthor=C.+Biauthor=L.+Chenauthor=L.+Di+Costanzoauthor=C.+Christieauthor=K.+Dalenbergauthor=J.+M.+Duarteauthor=S.+Duttaauthor=Z.+Fengauthor=S.+Ghoshauthor=D.+S.+Goodsellauthor=R.+K.+Greenauthor=V.+Guranovi%C4%87author=D.+Guzenkoauthor=B.+P.+Hudsonauthor=T.+Kalroauthor=Y.+Liangauthor=R.+Loweauthor=H.+Namkoongauthor=E.+Peisachauthor=I.+Periskovaauthor=A.+Prli%C4%87author=C.+Randleauthor=A.+Roseauthor=P.+Roseauthor=R.+Salaauthor=M.+Sekharanauthor=C.+Shaoauthor=L.+Tanauthor=Y.-P.+Taoauthor=Y.+Valasatavaauthor=M.+Voigtauthor=J.+Westbrookauthor=J.+Wooauthor=H.+Yangauthor=J.+Youngauthor=M.+Zhuravlevaauthor=C.+Zardecki&title=RCSB+Protein+Data+Bank%3A+Biological+Macromolecular+Structures+Enabling+Research+and+Education+in+Fundamental+Biology%2C+Biomedicine%2C+Biotechnology+and+Energy&doi=10.1093%2Fnar%2Fgky1004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">RCSB Protein Data Bank: biological macromolecular structures enabling research and education in fundamental biology, biomedicine, biotechnology and energy</span></div><div class="casAuthors">Burley, Stephen K.; Berman, Helen M.; Bhikadiya, Charmi; Bi, Chunxiao; Chen, Li; Di Costanzo, Luigi; Christie, Cole; Dalenberg, Ken; Duarte, Jose M.; Dutta, Shuchismita; Feng, Zukang; Ghosh, Sutapa; Goodsell, David S.; Green, Rachel K.; Guranovic, Vladimir; Guzenko, Dmytro; Hudson, Brian P.; Kalro, Tara; Liang, Yuhe; Lowe, Robert; Namkoong, Harry; Peisach, Ezra; Periskova, Irina; Prlic, Andreas; Randle, Chris; Rose, Alexander; Rose, Peter; Sala, Raul; Sekharan, Monica; Shao, Chenghua; Tan, Lihua; Tao, Yi-Ping; Valasatava, Yana; Voigt, Maria; Westbrook, John; Woo, Jesse; Yang, Huanwang; Young, Jasmine; Zhuravleva, Marina; Zardecki, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D464-D474</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  The Research Collab. for Structural Bioinformatics Protein Data Bank (RCSB PDB, rcsb.org), the US data center for the global PDB archive, serves thousands of Data Depositors in the Americas and Oceania and makes 3D macromol. structure data available at no charge and without usage restrictions to more than 1 million rcsb.org Users worldwide and 600 000 pdb101.rcsb.org education-focused Users around the globe.  PDB Data Depositors include structural biologists using macromol. crystallog., NMR spectroscopy and 3D electron microscopy.  PDB Data Consumers include researchers, educators and students studying Fundamental Biol., Biomedicine, Biotechnol. and Energy.  Recent reorganization of RCSB PDB activities into four integrated, interdependent services is described in detail, together with tools and resources added over the past 2 years to RCSB PDB web portals in support of a 'Structural View of Biol.'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqbbw8LsGXrrVg90H21EOLACvtfcHk0liTgEKdRYqCwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1Cgt7jO&md5=190361b23d7665a66208618c0ba86b5a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky1004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky1004%26sid%3Dliteratum%253Aachs%26aulast%3DBurley%26aufirst%3DS.%2BK.%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DBhikadiya%26aufirst%3DC.%26aulast%3DBi%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DDi%2BCostanzo%26aufirst%3DL.%26aulast%3DChristie%26aufirst%3DC.%26aulast%3DDalenberg%26aufirst%3DK.%26aulast%3DDuarte%26aufirst%3DJ.%2BM.%26aulast%3DDutta%26aufirst%3DS.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DGreen%26aufirst%3DR.%2BK.%26aulast%3DGuranovi%25C4%2587%26aufirst%3DV.%26aulast%3DGuzenko%26aufirst%3DD.%26aulast%3DHudson%26aufirst%3DB.%2BP.%26aulast%3DKalro%26aufirst%3DT.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DLowe%26aufirst%3DR.%26aulast%3DNamkoong%26aufirst%3DH.%26aulast%3DPeisach%26aufirst%3DE.%26aulast%3DPeriskova%26aufirst%3DI.%26aulast%3DPrli%25C4%2587%26aufirst%3DA.%26aulast%3DRandle%26aufirst%3DC.%26aulast%3DRose%26aufirst%3DA.%26aulast%3DRose%26aufirst%3DP.%26aulast%3DSala%26aufirst%3DR.%26aulast%3DSekharan%26aufirst%3DM.%26aulast%3DShao%26aufirst%3DC.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DTao%26aufirst%3DY.-P.%26aulast%3DValasatava%26aufirst%3DY.%26aulast%3DVoigt%26aufirst%3DM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DWoo%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DYoung%26aufirst%3DJ.%26aulast%3DZhuravleva%26aufirst%3DM.%26aulast%3DZardecki%26aufirst%3DC.%26atitle%3DRCSB%2520Protein%2520Data%2520Bank%253A%2520Biological%2520Macromolecular%2520Structures%2520Enabling%2520Research%2520and%2520Education%2520in%2520Fundamental%2520Biology%252C%2520Biomedicine%252C%2520Biotechnology%2520and%2520Energy%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26spage%3DD464%26epage%3DD474%26doi%3D10.1093%2Fnar%2Fgky1004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mattison, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilbur, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">So, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, R. G.</span></span> <span> </span><span class="NLM_article-title">Structure of FitAB from Neisseria gonorrhoeae Bound to DNA Reveals a Tetramer of Toxin-Antitoxin Heterodimers Containing Pin Domains and Ribbon-Helix-Helix Motifs</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">37942</span>– <span class="NLM_lpage">37951</span>, <span class="refDoi"> DOI: 10.1074/jbc.m605198200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1074%2Fjbc.M605198200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=16982615" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Kit7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=37942-37951&author=K.+Mattisonauthor=J.+S.+Wilburauthor=M.+Soauthor=R.+G.+Brennan&title=Structure+of+FitAB+from+Neisseria+gonorrhoeae+Bound+to+DNA+Reveals+a+Tetramer+of+Toxin-Antitoxin+Heterodimers+Containing+Pin+Domains+and+Ribbon-Helix-Helix+Motifs&doi=10.1074%2Fjbc.m605198200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of FitAB from Neisseria gonorrhoeae Bound to DNA Reveals a Tetramer of Toxin-Antitoxin Heterodimers Containing Pin Domains and Ribbon-Helix-Helix Motifs</span></div><div class="casAuthors">Mattison, Kirsten; Wilbur, J. Scott; So, Magdalene; Brennan, Richard G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">37942-37951</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Neisseria gonorrhoeae is a sexually transmitted pathogen that initiates infections in humans by adhering to the mucosal epithelium of the urogenital tract.  The bacterium then enters the apical region of the cell and traffics across the cell to exit into the subepithelial matrix.  Mutations in the fast intracellular trafficking (fitAB) locus cause the bacteria to transit a polarized epithelial monolayer more quickly than the wild-type parent and to replicate within cells at an accelerated rate.  Here, we describe the crystal structure of the toxin-antitoxin heterodimer, FitAB, bound to a high affinity 36-bp DNA fragment from the fitAB promoter.  FitA, the antitoxin, binds DNA through its ribbon-helix-helix motif and is tethered to FitB, the toxin, to form a heterodimer by the insertion of a four turn α-helix into an extensive FitB hydrophobic pocket.  FitB is composed of a PIN (PilT N terminus) domain, with a central, twisted, 5-stranded parallel β-sheet that is open on one side and flanked by five α-helixes.  FitB in the context of the FitAB complex does not display nuclease activity against tested PIN substrates.  The FitAB complex points to the mechanism by which antitoxins with RHH motifs can block the activity of toxins with PIN domains.  Interactions between two FitB mols. result in the formation of a tetramer of FitAB heterodimers, which binds to the 36-bp DNA fragment and provides an explanation for how FitB enhances the DNA binding affinity of FitA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQMCwZq_dD3LVg90H21EOLACvtfcHk0liTgEKdRYqCwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Kit7%252FF&md5=bccbcc839ddce4a994b6d76065e30a6a</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M605198200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M605198200%26sid%3Dliteratum%253Aachs%26aulast%3DMattison%26aufirst%3DK.%26aulast%3DWilbur%26aufirst%3DJ.%2BS.%26aulast%3DSo%26aufirst%3DM.%26aulast%3DBrennan%26aufirst%3DR.%2BG.%26atitle%3DStructure%2520of%2520FitAB%2520from%2520Neisseria%2520gonorrhoeae%2520Bound%2520to%2520DNA%2520Reveals%2520a%2520Tetramer%2520of%2520Toxin-Antitoxin%2520Heterodimers%2520Containing%2520Pin%2520Domains%2520and%2520Ribbon-Helix-Helix%2520Motifs%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D37942%26epage%3D37951%26doi%3D10.1074%2Fjbc.m605198200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dienemann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bøggild, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winther, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerdes, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brodersen, D. E.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the VapBC Toxin-Antitoxin Complex from Shigella Flexneri Reveals a Hetero-Octameric DNA-Binding Assembly</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>414</i></span>,  <span class="NLM_fpage">713</span>– <span class="NLM_lpage">722</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2011.10.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1016%2Fj.jmb.2011.10.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=22037005" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFKls7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=414&publication_year=2011&pages=713-722&author=C.+Dienemannauthor=A.+B%C3%B8ggildauthor=K.+S.+Wintherauthor=K.+Gerdesauthor=D.+E.+Brodersen&title=Crystal+Structure+of+the+VapBC+Toxin-Antitoxin+Complex+from+Shigella+Flexneri+Reveals+a+Hetero-Octameric+DNA-Binding+Assembly&doi=10.1016%2Fj.jmb.2011.10.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of the VapBC Toxin-Antitoxin Complex from Shigella flexneri Reveals a Hetero-Octameric DNA-Binding Assembly</span></div><div class="casAuthors">Dienemann, Christian; Boggild, Andreas; Winther, Kristoffer S.; Gerdes, Kenn; Brodersen, Ditlev E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">414</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">713-722</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Toxin-antitoxin (TA) loci are common in archaea and prokaryotes and allow cells to rapidly adapt to changing environmental conditions through release of active regulators of metab.  Many toxins are endonucleases that target cellular mRNA and tRNAs, while the antitoxins tightly wrap around the toxins to inhibit them under normal circumstances.  The antitoxins also bind to operators in the promoter regions of the cognate TA operon and thereby regulate transcription.  For enteric vapBC TA loci, the VapC toxins specifically cleave tRNAfMet and thus down-regulate protein synthesis.  Here, we describe the crystal structure of the intact Shigella flexneri VapBC TA complex, detd. to 2.7 Å resoln.  Both in soln. and in the crystal structure, four mols. of each protein combine to form a large and globular hetero-octameric assembly with SpoVT/AbrB-type DNA-binding domains at each end and a total mol. mass of about 100 kDa.  The structure gives new insights into the inhibition of VapC toxins by VapB and provides the mol. basis for understanding transcriptional regulation through VapB dimerization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGov8BLYjM0jq7Vg90H21EOLACvtfcHk0liTgEKdRYqCwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFKls7jF&md5=9771b57824d7407c17dae0cfa68103ce</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2011.10.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2011.10.024%26sid%3Dliteratum%253Aachs%26aulast%3DDienemann%26aufirst%3DC.%26aulast%3DB%25C3%25B8ggild%26aufirst%3DA.%26aulast%3DWinther%26aufirst%3DK.%2BS.%26aulast%3DGerdes%26aufirst%3DK.%26aulast%3DBrodersen%26aufirst%3DD.%2BE.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520VapBC%2520Toxin-Antitoxin%2520Complex%2520from%2520Shigella%2520Flexneri%2520Reveals%2520a%2520Hetero-Octameric%2520DNA-Binding%2520Assembly%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2011%26volume%3D414%26spage%3D713%26epage%3D722%26doi%3D10.1016%2Fj.jmb.2011.10.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maté, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincentelli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foos, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raoult, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cambillau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz-Lombardía, M.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the DNA-Bound VapBC2 Antitoxin/Toxin Pair from Rickettsia felis</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">3245</span>– <span class="NLM_lpage">3258</span>, <span class="refDoi"> DOI: 10.1093/nar/gkr1167</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgkr1167" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=22140099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC38Xls1Snu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=3245-3258&author=M.+J.+Mat%C3%A9author=R.+Vincentelliauthor=N.+Foosauthor=D.+Raoultauthor=C.+Cambillauauthor=M.+Ortiz-Lombard%C3%ADa&title=Crystal+Structure+of+the+DNA-Bound+VapBC2+Antitoxin%2FToxin+Pair+from+Rickettsia+felis&doi=10.1093%2Fnar%2Fgkr1167"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the DNA-bound VapBC2 antitoxin/toxin pair from Rickettsia felis</span></div><div class="casAuthors">Mate, Maria J.; Vincentelli, Renaud; Foos, Nicolas; Raoult, Didier; Cambillau, Christian; Ortiz-Lombardia, Miguel</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3245-3258</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Besides their commonly attributed role in the maintenance of low-copy no. plasmids, toxin/antitoxin (TA) loci, also called "addiction modules," have been found in chromosomes and assocd. with a no. of biol. functions such as redn. of protein synthesis, gene regulation and retardation of cell growth under nutritional stress.  The recent discovery of TA loci in obligatory intracellular species of the Rickettsia genus has prompted new research to establish whether they work as stress response elements or as addiction systems that might be toxic for the host cell.  VapBC2 is a TA locus from R. felis, a pathogen responsible for flea-borne spotted fever in humans.  The VapC2 toxin is a PIN-domain protein, whereas the antitoxin, VapB2, belongs to the family of swapped-hairpin β-barrel DNA-binding proteins.  We have used a combination of biophys. and structural methods to characterize this new toxin/antitoxin pair.  Our results show how VapB2 can block the VapC2 toxin.  They provide a first structural description of the interaction between a swapped-hairpin β-barrel protein and DNA.  Finally, these results suggest how the VapC2/VapB2 molar ratio can control the self-regulation of the TA locus transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqR28iWdUuBWLVg90H21EOLACvtfcHk0lhlpGr9VkiTJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xls1Snu7c%253D&md5=78770d368e630da4d50499844b254e59</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr1167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr1167%26sid%3Dliteratum%253Aachs%26aulast%3DMat%25C3%25A9%26aufirst%3DM.%2BJ.%26aulast%3DVincentelli%26aufirst%3DR.%26aulast%3DFoos%26aufirst%3DN.%26aulast%3DRaoult%26aufirst%3DD.%26aulast%3DCambillau%26aufirst%3DC.%26aulast%3DOrtiz-Lombard%25C3%25ADa%26aufirst%3DM.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520DNA-Bound%2520VapBC2%2520Antitoxin%252FToxin%2520Pair%2520from%2520Rickettsia%2520felis%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3D3245%26epage%3D3258%26doi%3D10.1093%2Fnar%2Fgkr1167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bendtsen, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luckmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winther, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, C. N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brodersen, D. E.</span></span> <span> </span><span class="NLM_article-title">Toxin Inhibition in C. Crescentus VapBC1 is Mediated by a Flexible Pseudo-Palindromic Protein Motif and Modulated by DNA Binding</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">2875</span>– <span class="NLM_lpage">2886</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw1266</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgkw1266" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=27998932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC1cXivFygtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=2875-2886&author=K.+L.+Bendtsenauthor=K.+Xuauthor=M.+Luckmannauthor=K.+S.+Wintherauthor=S.+A.+Shahauthor=C.+N.+S.+Pedersenauthor=D.+E.+Brodersen&title=Toxin+Inhibition+in+C.+Crescentus+VapBC1+is+Mediated+by+a+Flexible+Pseudo-Palindromic+Protein+Motif+and+Modulated+by+DNA+Binding&doi=10.1093%2Fnar%2Fgkw1266"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Toxin inhibition in C. crescentus VapBC1 is mediated by a flexible pseudo-palindromic protein motif and modulated by DNA binding</span></div><div class="casAuthors">Bendtsen, Kirstine L.; Xu, Kehan; Luckmann, Majbritt; Winther, Kristoffer S.; Shah, Shiraz Ali; Pedersen, Christian N. S.; Brodersen, Ditlev E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2875-2886</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Expression of bacterial type II toxin-antitoxin (TA) systems is regulated at the transcriptional level through direct binding of the antitoxin to pseudo-palindromic sequences on operator DNA.  In this context, the toxin functions as a co-repressor by stimulating DNA binding through direct interaction with the antitoxin.  Here, we det. crystal structures of the complete 90 kDa heterooctameric VapBC1 complex from Caulobacter crescentus CB15 both in isolation and bound to its cognate DNA operator sequence at 1.6 and 2.7 Å resoln., resp.  DNA binding is assocd. with a dramatic architectural rearrangement of conserved TA interactions in which C-terminal extended structures of the antitoxin VapB1 swap positions to interlock the complex in the DNA-bound state.  We further show that a pseudo-palindromic protein sequence in the antitoxin is responsible for this interaction and required for binding and inactivation of the VapC1 toxin dimer.  Sequence anal. of 4127 orthologous VapB sequences reveals that such palindromic protein sequences are widespread and unique to bacterial and archaeal VapB antitoxins suggesting a general principle governing regulation of VapBC TA systems.  Finally, a structure of C-terminally truncated VapB1 bound to VapC1 reveals discrete states of the TA interaction that suggest a structural basis for toxin activation in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxpyA3YmKmQLVg90H21EOLACvtfcHk0lhlpGr9VkiTJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXivFygtb0%253D&md5=5898602c7b2dcd79ac4e82d9a56d6fbf</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1266&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1266%26sid%3Dliteratum%253Aachs%26aulast%3DBendtsen%26aufirst%3DK.%2BL.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DLuckmann%26aufirst%3DM.%26aulast%3DWinther%26aufirst%3DK.%2BS.%26aulast%3DShah%26aufirst%3DS.%2BA.%26aulast%3DPedersen%26aufirst%3DC.%2BN.%2BS.%26aulast%3DBrodersen%26aufirst%3DD.%2BE.%26atitle%3DToxin%2520Inhibition%2520in%2520C.%2520Crescentus%2520VapBC1%2520is%2520Mediated%2520by%2520a%2520Flexible%2520Pseudo-Palindromic%2520Protein%2520Motif%2520and%2520Modulated%2520by%2520DNA%2520Binding%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3D2875%26epage%3D2886%26doi%3D10.1093%2Fnar%2Fgkw1266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molinaro, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashipathy, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battaile, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coussens, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daines, D. A.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of VapBC-1 from Nontypeable Haemophilus influenzae and the Effect of PIN Domain Mutations on Survival during Infection</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>201</i></span>, <span class="NLM_elocation-id">e00026</span> <span class="refDoi"> DOI: 10.1128/jb.00026-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1128%2FJB.00026-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=30936373" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlWmsb3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=201&publication_year=2019&author=A.+L.+Molinaroauthor=M.+M.+Kashipathyauthor=S.+Lovellauthor=K.+P.+Battaileauthor=N.+P.+Coussensauthor=M.+Shenauthor=D.+A.+Daines&title=Crystal+Structure+of+VapBC-1+from+Nontypeable+Haemophilus+influenzae+and+the+Effect+of+PIN+Domain+Mutations+on+Survival+during+Infection&doi=10.1128%2Fjb.00026-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of VapBC-1 from nontypeable Haemophilus influenzae and the effect of PIN domain mutations on survival during infection</span></div><div class="casAuthors">Molinaro, Ashley L.; Kashipathy, Maithri M.; Lovell, Scott; Battaile, Kevin P.; Coussens, Nathan P.; Shen, Min; Daines, Dayle A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">201</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e00026</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">1098-5530</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Toxin-antitoxin (TA) gene pairs have been identified in nearly all bacterial genomes sequenced to date and are thought to facilitate persistence and antibiotic tolerance.  TA loci are classified into various types based upon the characteristics of their antitoxins, with those in type II expressing proteic antitoxins.  Many toxins from type II modules are RNases that maintain a PilT N-terminal (PIN) domain contg. conserved amino acids considered essential for activity.  The vapBC (virulence-assocd. protein) TA system is the largest subfamily in this class and has been linked to pathogenesis of nontypeable Haemophilus influenzae (NTHi).  In this study, the crystal structure of the VapBC-1 complex from NTHi was detd. to 2.20 Å resoln.  Based on this structure, aspartate-to-asparagine and glutamate-to-glutamine mutations of four conserved residues in the PIN domain of the VapC-1 toxin were constructed and the effects of the mutations on protein-protein interactions, growth of Escherichia coli, and pathogenesis ex vivo were tested.  Finally, a novel model system was designed and utilized that consists of an NTHi ΔvapBC-1 strain complemented in cis with the TA module contg. a mutated or wild-type toxin at an ectopic site on the chromosome.  This enabled the anal. of the effect of PIN domain toxin mutants in tandem with their wild-type antitoxin under the control of the vapBC-1 native promoter and in single copy.  This is the first report of a system facilitating the study of TA mutant operons in the background of NTHi during infections of primary human tissues ex vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozANUr-sm2srVg90H21EOLACvtfcHk0lhlpGr9VkiTJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlWmsb3P&md5=6a976fb8683204cc548a5a80cbb8400d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1128%2FJB.00026-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.00026-19%26sid%3Dliteratum%253Aachs%26aulast%3DMolinaro%26aufirst%3DA.%2BL.%26aulast%3DKashipathy%26aufirst%3DM.%2BM.%26aulast%3DLovell%26aufirst%3DS.%26aulast%3DBattaile%26aufirst%3DK.%2BP.%26aulast%3DCoussens%26aufirst%3DN.%2BP.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DDaines%26aufirst%3DD.%2BA.%26atitle%3DCrystal%2520Structure%2520of%2520VapBC-1%2520from%2520Nontypeable%2520Haemophilus%2520influenzae%2520and%2520the%2520Effect%2520of%2520PIN%2520Domain%2520Mutations%2520on%2520Survival%2520during%2520Infection%26jtitle%3DJ.%2520Bacteriol.%26date%3D2019%26volume%3D201%26doi%3D10.1128%2Fjb.00026-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Min, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miallau, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawaya, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cascio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenberg, D.</span></span> <span> </span><span class="NLM_article-title">The Crystal Structure of the Rv0301-Rv0300 VapBC-3 Toxin-Antitoxin Complex from M. tuberculosis Reveals a Mg(2)(+) Ion in the Active Site and a Putative RNA-Binding Site</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1754</span>– <span class="NLM_lpage">1767</span>, <span class="refDoi"> DOI: 10.1002/pro.2161</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1002%2Fpro.2161" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=23011806" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFWitrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2012&pages=1754-1767&author=A.+B.+Minauthor=L.+Miallauauthor=M.+R.+Sawayaauthor=J.+Habelauthor=D.+Cascioauthor=D.+Eisenberg&title=The+Crystal+Structure+of+the+Rv0301-Rv0300+VapBC-3+Toxin-Antitoxin+Complex+from+M.+tuberculosis+Reveals+a+Mg%282%29%28%2B%29+Ion+in+the+Active+Site+and+a+Putative+RNA-Binding+Site&doi=10.1002%2Fpro.2161"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The crystal structure of the Rv0301-Rv0300 VapBC-3 toxin-antitoxin complex from M. tuberculosis reveals a Mg2+ ion in the active site and a putative RNA-binding site</span></div><div class="casAuthors">Min, Andrew B.; Miallau, Linda; Sawaya, Michael R.; Habel, Jeff; Cascio, Duilio; Eisenberg, David</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1754-1767</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">VapBC pairs account for 45 out of 88 identified toxin-antitoxin (TA) pairs in the Mycobacterium tuberculosis (Mtb) H37Rv genome.  A working model suggests that under times of stress, antitoxin mols. are degraded, releasing the toxins to slow the metab. of the cell, which in the case of VapC toxins is via their RNase activity.  Otherwise the TA pairs remain bound to their promoters, autoinhibiting transcription.  The crystal structure of Rv0301-Rv0300, an Mtb VapBC TA complex detd. at 1.49 Å resoln., suggests a mechanism for these three functions: RNase activity, its inhibition by antitoxin, and its ability to bind promoter DNA.  The Rv0301 toxin consists of a core of five parallel beta strands flanked by alpha helixes.  Three proximal aspartates coordinate a Mg2+ ion forming the putative RNase active site.  The Rv0300 antitoxin monomer is extended in structure, consisting of an N-terminal beta strand followed by four helixes.  The last two helixes wrap around the toxin and terminate near the putative RNase active site, but with different conformations.  In one conformation, the C-terminal arginine interferes with Mg2+ ion coordination, suggesting a mechanism by which the antitoxin can inhibit toxin activity.  At the N-terminus of the antitoxin, two pairs of Ribbon-Helix-Helix (RHH) motifs are related by crystallog. twofold symmetry.  The resulting hetero-octameric complex is similar to the FitAB system, but the two RHH motifs are about 30 Å closer together in the Rv0301-Rv0300 complex, suggesting either a different span of the DNA recognition sequence or a conformational change.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRD_bgcnfHIbVg90H21EOLACvtfcHk0lhC5mk_Kq4PDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFWitrbL&md5=0c256d6a1f84053b5f732c081b77460d</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fpro.2161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.2161%26sid%3Dliteratum%253Aachs%26aulast%3DMin%26aufirst%3DA.%2BB.%26aulast%3DMiallau%26aufirst%3DL.%26aulast%3DSawaya%26aufirst%3DM.%2BR.%26aulast%3DHabel%26aufirst%3DJ.%26aulast%3DCascio%26aufirst%3DD.%26aulast%3DEisenberg%26aufirst%3DD.%26atitle%3DThe%2520Crystal%2520Structure%2520of%2520the%2520Rv0301-Rv0300%2520VapBC-3%2520Toxin-Antitoxin%2520Complex%2520from%2520M.%2520tuberculosis%2520Reveals%2520a%2520Mg%25282%2529%2528%252B%2529%2520Ion%2520in%2520the%2520Active%2520Site%2520and%2520a%2520Putative%2520RNA-Binding%2520Site%26jtitle%3DProtein%2520Sci.%26date%3D2012%26volume%3D21%26spage%3D1754%26epage%3D1767%26doi%3D10.1002%2Fpro.2161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miallau, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arbing, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cascio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenberg, D.</span></span> <span> </span><span class="NLM_article-title">Structure and Proposed Activity of a Member of the VapBC Family of Toxin-Antitoxin Systems. VapBC-5 from Mycobacterium tuberculosis</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>284</i></span>,  <span class="NLM_fpage">276</span>– <span class="NLM_lpage">283</span>, <span class="refDoi"> DOI: 10.1074/jbc.m805061200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1074%2Fjbc.M805061200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=18952600" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFCjtLzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=276-283&author=L.+Miallauauthor=M.+Fallerauthor=J.+Chiangauthor=M.+Arbingauthor=F.+Guoauthor=D.+Cascioauthor=D.+Eisenberg&title=Structure+and+Proposed+Activity+of+a+Member+of+the+VapBC+Family+of+Toxin-Antitoxin+Systems.+VapBC-5+from+Mycobacterium+tuberculosis&doi=10.1074%2Fjbc.m805061200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and Proposed Activity of a Member of the VapBC Family of Toxin-Antitoxin Systems: VapBC-5 from Mycobacterium tuberculosis</span></div><div class="casAuthors">Miallau, Linda; Faller, Michael; Chiang, Janet; Arbing, Mark; Guo, Feng; Cascio, Duilio; Eisenberg, David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">276-283</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">In prokaryotes, cognate toxin-antitoxin pairs have long been known, but no three-dimensional structure has been available for any given complex from Mycobacterium tuberculosis.  Here we report the crystal structure and activity of a member of the VapBC family of complexes from M. tuberculosis.  The toxin VapC-5 is a compact, 150 residues, two domain α/β protein.  Bent around the toxin is the VapB-5 antitoxin, a 33-residue α-helix.  Assays suggest that the toxin is an Mg-enabled endoribonuclease, inhibited by the antitoxin.  The lack of DNase activity is consistent with earlier suggestions that the complex represses its own operon.  Furthermore, anal. of the interactions in the binding of the antitoxin to the toxin suggest that exquisite control is required to protect the bacteria cell from toxic VapC-5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrh7EhG7joYp7Vg90H21EOLACvtfcHk0lhC5mk_Kq4PDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFCjtLzJ&md5=7ae4516a98c956c30890e26cc8e12b0e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M805061200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M805061200%26sid%3Dliteratum%253Aachs%26aulast%3DMiallau%26aufirst%3DL.%26aulast%3DFaller%26aufirst%3DM.%26aulast%3DChiang%26aufirst%3DJ.%26aulast%3DArbing%26aufirst%3DM.%26aulast%3DGuo%26aufirst%3DF.%26aulast%3DCascio%26aufirst%3DD.%26aulast%3DEisenberg%26aufirst%3DD.%26atitle%3DStructure%2520and%2520Proposed%2520Activity%2520of%2520a%2520Member%2520of%2520the%2520VapBC%2520Family%2520of%2520Toxin-Antitoxin%2520Systems.%2520VapBC-5%2520from%2520Mycobacterium%2520tuberculosis%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D276%26epage%3D283%26doi%3D10.1074%2Fjbc.m805061200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deep, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiwari, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agarwal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaundal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kidwai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thakur, K. G.</span></span> <span> </span><span class="NLM_article-title">Structural, Functional and Biological Insights into the Role of Mycobacterium tuberculosis VapBC11 Toxin-Antitoxin System: Targeting a tRNase to Tackle Mycobacterial Adaptation</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">11639</span>– <span class="NLM_lpage">11655</span>, <span class="refDoi"> DOI: 10.1093/nar/gky924</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgky924" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=30329074" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC1MXovFWntbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=11639-11655&author=A.+Deepauthor=P.+Tiwariauthor=S.+Agarwalauthor=S.+Kaundalauthor=S.+Kidwaiauthor=R.+Singhauthor=K.+G.+Thakur&title=Structural%2C+Functional+and+Biological+Insights+into+the+Role+of+Mycobacterium+tuberculosis+VapBC11+Toxin-Antitoxin+System%3A+Targeting+a+tRNase+to+Tackle+Mycobacterial+Adaptation&doi=10.1093%2Fnar%2Fgky924"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Structural, functional and biological insights into the role of Mycobacterium tuberculosis VapBC11 toxin-antitoxin system: targeting a tRNase to tackle mycobacterial adaptation</span></div><div class="casAuthors">Deep, Amar; Tiwari, Prabhakar; Agarwal, Sakshi; Kaundal, Soni; Kidwai, Saqib; Singh, Ramandeep; Thakur, Krishan G.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11639-11655</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Toxin-antitoxin (TA) systems are involved in diverse physiol. processes in prokaryotes, but their exact role in Mycobacterium tuberculosis (Mtb) virulence and in vivo stress adaptation has not been extensively studied.  Here, we demonstrate that the VapBC11 TA module is essential for Mtb to establish infection in guinea pigs.  RNA-sequencing revealed that overexpression of VapC11 toxin results in metabolic slowdown, suggesting that modulation of the growth rate is an essential strategy for in vivo survival.  Interestingly, overexpression of VapC11 resulted in the upregulation of chromosomal TA genes, suggesting the existence of highly coordinated crosstalk among TA systems.  In this study, we also present the crystal structure of the VapBC11 heterooctameric complex at 1.67 Å resoln.  Binding kinetic studies suggest that the binding affinities of toxin-substrate and toxin-antitoxin interactions are comparable.  We used a combination of structural studies, mol. docking, mutational anal. and in vitro RNase assays to enhance our understanding of the mode of substrate recognition by the VapC11 toxin.  Furthermore, we have also designed peptide-based inhibitors to target VapC11 RNase activity.  Taken together, we propose that the structure-guided design of inhibitors against in vivo essential RNases might be a novel strategy to hasten clearance of intracellular Mtb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaZotFmWm1_rVg90H21EOLACvtfcHk0lhC5mk_Kq4PDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXovFWntbY%253D&md5=6c6d022bad4a8a1e4c2b53238624d75c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky924&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky924%26sid%3Dliteratum%253Aachs%26aulast%3DDeep%26aufirst%3DA.%26aulast%3DTiwari%26aufirst%3DP.%26aulast%3DAgarwal%26aufirst%3DS.%26aulast%3DKaundal%26aufirst%3DS.%26aulast%3DKidwai%26aufirst%3DS.%26aulast%3DSingh%26aufirst%3DR.%26aulast%3DThakur%26aufirst%3DK.%2BG.%26atitle%3DStructural%252C%2520Functional%2520and%2520Biological%2520Insights%2520into%2520the%2520Role%2520of%2520Mycobacterium%2520tuberculosis%2520VapBC11%2520Toxin-Antitoxin%2520System%253A%2520Targeting%2520a%2520tRNase%2520to%2520Tackle%2520Mycobacterial%2520Adaptation%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26spage%3D11639%26epage%3D11655%26doi%3D10.1093%2Fnar%2Fgky924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Das, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogenberg, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subhramanyam, U. K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilmanns, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gourinath, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, A.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the VapBC-15 Complex from Mycobacterium tuberculosis Reveals a Two-Metal Ion Dependent PIN-Domain Ribonuclease and a Variable Mode of Toxin-Antitoxin Assembly</span>. <i>J. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>188</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">258</span>, <span class="refDoi"> DOI: 10.1016/j.jsb.2014.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1016%2Fj.jsb.2014.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=25450593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWmtrbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=188&publication_year=2014&pages=249-258&author=U.+Dasauthor=V.+Pogenbergauthor=U.+K.+T.+Subhramanyamauthor=M.+Wilmannsauthor=S.+Gourinathauthor=A.+Srinivasan&title=Crystal+Structure+of+the+VapBC-15+Complex+from+Mycobacterium+tuberculosis+Reveals+a+Two-Metal+Ion+Dependent+PIN-Domain+Ribonuclease+and+a+Variable+Mode+of+Toxin-Antitoxin+Assembly&doi=10.1016%2Fj.jsb.2014.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the VapBC-15 complex from Mycobacterium tuberculosis reveals a two-metal ion dependent PIN-domain ribonuclease and a variable mode of toxin-antitoxin assembly</span></div><div class="casAuthors">Das, Uddipan; Pogenberg, Vivian; Subhramanyam, Udaya Kumar Tiruttani; Wilmanns, Matthias; Gourinath, Samudrala; Srinivasan, Alagiri</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Structural Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">188</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">249-258</span>CODEN:
                <span class="NLM_cas:coden">JSBIEM</span>;
        ISSN:<span class="NLM_cas:issn">1047-8477</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Although PIN (PilT N-terminal)-domain proteins are known to have RNase activity, their specific mechanism of action remains unknown.  VapCs form a family of RNases that possess a PIN-domain assembly and are known as toxins.  The activities of VapCs are impaired by VapB antitoxins.  Here we present the crystal structure of the VapBC-15 toxin-antitoxin complex from Mycobacterium tuberculosis detd. to 2.1 Å resoln.  The VapB-15 and VapC-15 components assemble into one heterotetramer (VapB2C2) and two heterotrimers (VapBC2) in each asym. unit of the crystal.  The active site of VapC-15 toxin consists of a cluster of acidic amino acid residues and two divalent metal ions, forming a well organised RNase active site.  The distribution of the catalytic-site residues of the VapC-15 toxin is similar to that of T4 RNase H and of Methanococcus jannaschii FEN-1, providing strong evidence that these three proteins share a similar mechanism of activity.  The presence of both VapB2C2 and VapBC2 emphasizes the fact that the same antitoxin can bind the toxin in 1:1 and 1:2 ratios.  The crystal structure detn. of the VapBC-15 complex reveals for the first time a PIN-domain RNase protein that shows two metal ions at the active site and a variable mode of toxin-antitoxin assembly.  The structure further shows that VapB-15 antitoxin binds to the same groove meant for the binding of putative substrate (RNA), resulting in the inhibition of VapC-15's toxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRdAXWGbiHdLVg90H21EOLACvtfcHk0lhTAeWaRTw1lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWmtrbF&md5=875d5c535bc270fd236313aa81c75b42</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.jsb.2014.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsb.2014.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DU.%26aulast%3DPogenberg%26aufirst%3DV.%26aulast%3DSubhramanyam%26aufirst%3DU.%2BK.%2BT.%26aulast%3DWilmanns%26aufirst%3DM.%26aulast%3DGourinath%26aufirst%3DS.%26aulast%3DSrinivasan%26aufirst%3DA.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520VapBC-15%2520Complex%2520from%2520Mycobacterium%2520tuberculosis%2520Reveals%2520a%2520Two-Metal%2520Ion%2520Dependent%2520PIN-Domain%2520Ribonuclease%2520and%2520a%2520Variable%2520Mode%2520of%2520Toxin-Antitoxin%2520Assembly%26jtitle%3DJ.%2520Struct.%2520Biol.%26date%3D2014%26volume%3D188%26spage%3D249%26epage%3D258%26doi%3D10.1016%2Fj.jsb.2014.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kang, S.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Im, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, B.-J.</span></span> <span> </span><span class="NLM_article-title">Functional Details of the Mycobacterium tuberculosis VapBC26 Toxin-Antitoxin System based on a Structural Study: Insights into Unique Binding and Antibiotic Peptides</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">8564</span>– <span class="NLM_lpage">8580</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgkx489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=28575388" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjt1Wksrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=8564-8580&author=S.-M.+Kangauthor=D.-H.+Kimauthor=K.-Y.+Leeauthor=S.+J.+Parkauthor=H.-J.+Yoonauthor=S.+J.+Leeauthor=H.+Imauthor=B.-J.+Lee&title=Functional+Details+of+the+Mycobacterium+tuberculosis+VapBC26+Toxin-Antitoxin+System+based+on+a+Structural+Study%3A+Insights+into+Unique+Binding+and+Antibiotic+Peptides&doi=10.1093%2Fnar%2Fgkx489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Functional details of the Mycobacterium tuberculosis VapBC26 toxin-antitoxin system based on a structural study: insights into unique binding and antibiotic peptides</span></div><div class="casAuthors">Kang, Sung-Min; Kim, Do-Hee; Lee, Ki-Young; Park, Sung Jean; Yoon, Hye-Jin; Lee, Sang Jae; Im, Hookang; Lee, Bong-Jin</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">8564-8580</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Toxin-antitoxin (TA) systems are essential for bacterial persistence under stressful conditions.  In particular, Mycobacterium tuberculosis express VapBC TA genes that encode the stable VapC toxin and the labile VapB antitoxin.  Under normal conditions, these proteins interact to form a non-toxic TA complex, but the toxin is activated by release from the antitoxin in response to unfavorable conditions.  Here, we present the crystal structure of the M. tuberculosis VapBC26 complex and show that the VapC26 toxin contains a pilus retraction protein (PilT) N-terminal (PIN) domain that is essential for RNase activity and that, the VapB26 antitoxin folds into a ribbon-helix-helix DNA-binding motif at the N-terminus.  The active site of VapC26 is sterically blocked by the flexible C-terminal region of VapB26.  The C-terminal region of free VapB26 adopts an unfolded conformation but forms a helix upon binding to VapC26.  The results of RNase activity assays show that Mg2+ and Mn2+ are essential for the RNase activity of VapC26.  As shown in the NMR spectra, several residues of VapB26 participate in the specific binding to the promoter region of the VapBC26 operon.  In addn., toxin-mimicking peptides were designed that inhibit TA complex formation and thereby increase toxin activity, providing a novel approach to the development of new antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrI8pUoIBVlZrVg90H21EOLACvtfcHk0lhTAeWaRTw1lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjt1Wksrs%253D&md5=e8675ba6324e5ed0a84717a2afc95441</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx489%26sid%3Dliteratum%253Aachs%26aulast%3DKang%26aufirst%3DS.-M.%26aulast%3DKim%26aufirst%3DD.-H.%26aulast%3DLee%26aufirst%3DK.-Y.%26aulast%3DPark%26aufirst%3DS.%2BJ.%26aulast%3DYoon%26aufirst%3DH.-J.%26aulast%3DLee%26aufirst%3DS.%2BJ.%26aulast%3DIm%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DB.-J.%26atitle%3DFunctional%2520Details%2520of%2520the%2520Mycobacterium%2520tuberculosis%2520VapBC26%2520Toxin-Antitoxin%2520System%2520based%2520on%2520a%2520Structural%2520Study%253A%2520Insights%2520into%2520Unique%2520Binding%2520and%2520Antibiotic%2520Peptides%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3D8564%26epage%3D8580%26doi%3D10.1093%2Fnar%2Fgkx489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, I.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chae, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, B.-J.</span></span> <span> </span><span class="NLM_article-title">Structural an Functional Studies of the Mycobacterium tuberculosis VapBC30 Toxin-Antitoxin System: Implications for the Design of Novel Antimicrobial Peptides</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">7624</span>– <span class="NLM_lpage">7637</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv689</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgkv689" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=26150422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslOqtb3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=7624-7637&author=I.-G.+Leeauthor=S.+J.+Leeauthor=S.+Chaeauthor=K.-Y.+Leeauthor=J.-H.+Kimauthor=B.-J.+Lee&title=Structural+an+Functional+Studies+of+the+Mycobacterium+tuberculosis+VapBC30+Toxin-Antitoxin+System%3A+Implications+for+the+Design+of+Novel+Antimicrobial+Peptides&doi=10.1093%2Fnar%2Fgkv689"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and functional studies of the Mycobacterium tuberculosis VapBC30 toxin-antitoxin system: implications for the design of novel antimicrobial peptides</span></div><div class="casAuthors">Lee, In-Gyun; Lee, Sang Jae; Chae, Susanna; Lee, Ki-Young; Kim, Ji-Hun; Lee, Bong-Jin</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">7624-7637</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Toxin-antitoxin (TA) systems play important roles in bacterial physiol., such as multidrug tolerance, biofilm formation, and arrest of cellular growth under stress conditions.  To develop novel antimicrobial agents against tuberculosis, we focused on VapBC systems, which encompass more than half of TA systems in Mycobacterium tuberculosis.  Here, we report that the Mycobacterium tuberculosis VapC30 toxin regulates cellular growth through both magnesium and manganese ion-dependent RNase activity and is inhibited by the cognate VapB30 antitoxin.  We also detd. the 2.7-Å resoln. crystal structure of the M. tuberculosis VapBC30 complex, which revealed a novel process of inactivation of the VapC30 toxin via swapped blocking by the VapB30 antitoxin.  Our study on M. tuberculosis VapBC30 leads us to design two kinds of VapB30 and VapC30-based novel peptides which successfully disrupt the toxin-antitoxin complex and thus activate the RNase activity of the VapC30 toxin.  Our discovery herein possibly paves the way to treat tuberculosis for next generation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3UEanxZ2BarVg90H21EOLACvtfcHk0lhTAeWaRTw1lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslOqtb3I&md5=658d19f46e947c5aedec3556639ae976</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv689%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DI.-G.%26aulast%3DLee%26aufirst%3DS.%2BJ.%26aulast%3DChae%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DK.-Y.%26aulast%3DKim%26aufirst%3DJ.-H.%26aulast%3DLee%26aufirst%3DB.-J.%26atitle%3DStructural%2520an%2520Functional%2520Studies%2520of%2520the%2520Mycobacterium%2520tuberculosis%2520VapBC30%2520Toxin-Antitoxin%2520System%253A%2520Implications%2520for%2520the%2520Design%2520of%2520Novel%2520Antimicrobial%2520Peptides%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26spage%3D7624%26epage%3D7637%26doi%3D10.1093%2Fnar%2Fgkv689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, S.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, B.-J.</span></span> <span> </span><span class="NLM_article-title">Functional Insights into the Streptococcus pneumoniae HicBA Toxin-Antitoxin System Based on a Structural Study</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">6371</span>– <span class="NLM_lpage">6386</span>, <span class="refDoi"> DOI: 10.1093/nar/gky469</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgky469" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=29878152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosVyrtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=6371-6386&author=D.-H.+Kimauthor=S.-M.+Kangauthor=S.+J.+Parkauthor=C.+Jinauthor=H.-J.+Yoonauthor=B.-J.+Lee&title=Functional+Insights+into+the+Streptococcus+pneumoniae+HicBA+Toxin-Antitoxin+System+Based+on+a+Structural+Study&doi=10.1093%2Fnar%2Fgky469"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Functional insights into the Streptococcus pneumoniae HicBA toxin-antitoxin system based on a structural study</span></div><div class="casAuthors">Kim, Do-Hee; Kang, Sung-Min; Park, Sung Jean; Jin, Chenglong; Yoon, Hye-Jin; Lee, Bong-Jin</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6371-6386</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Streptococcus pneumonia has attracted increasing attention due to its resistance to existing antibiotics.  TA systems are essential for bacterial persistence under stressful conditions such as nutrient deprivation, antibiotic treatment, and immune system attacks.  In particular, S. pneumoniae expresses the HicBA TA gene, which encodes the stable HicA toxin and the labile HicB antitoxin.  These proteins interact to form a non-toxic TA complex under normal conditions, but the toxin is activated by release from the antitoxin in response to unfavorable growth conditions.  Here, we present the first crystal structure showing the complete conformation of the HicBA complex from S. pneumonia.  The structure reveals that the HicA toxin contains a double-stranded RNA-binding domain that is essential for RNA recognition and that the C-terminus of the HicB antitoxin folds into a ribbon-helix-helix DNA-binding motif.  The active site of HicA is sterically blocked by the N-terminal region of HicB.  RNase activity assays show that His36 is essential for the RNase activity of HicA, and NMR spectra show that several residues of HicB participate in binding to the promoter DNA of the HicBA operon.  A toxin-mimicking peptide that inhibits TA complex formation and thereby increases toxin activity was designed, providing a novel approach to the development of new antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobiPx_Ood1erVg90H21EOLACvtfcHk0li7eQlSBQ4JPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosVyrtLk%253D&md5=d67938b568850079c5eb7013d341a217</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky469&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky469%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DD.-H.%26aulast%3DKang%26aufirst%3DS.-M.%26aulast%3DPark%26aufirst%3DS.%2BJ.%26aulast%3DJin%26aufirst%3DC.%26aulast%3DYoon%26aufirst%3DH.-J.%26aulast%3DLee%26aufirst%3DB.-J.%26atitle%3DFunctional%2520Insights%2520into%2520the%2520Streptococcus%2520pneumoniae%2520HicBA%2520Toxin-Antitoxin%2520System%2520Based%2520on%2520a%2520Structural%2520Study%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26spage%3D6371%26epage%3D6386%26doi%3D10.1093%2Fnar%2Fgky469" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">Artificial Activation of Toxin-Antitoxin Systems as an Antibacterial Strategy</span>. <i>Trends Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">291</span>– <span class="NLM_lpage">298</span>, <span class="refDoi"> DOI: 10.1016/j.tim.2012.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1016%2Fj.tim.2012.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=22445361" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC38XksFejtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=291-298&author=J.+J.+Williamsauthor=P.+J.+Hergenrother&title=Artificial+Activation+of+Toxin-Antitoxin+Systems+as+an+Antibacterial+Strategy&doi=10.1016%2Fj.tim.2012.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Artificial activation of toxin-antitoxin systems as an antibacterial strategy</span></div><div class="casAuthors">Williams, Julia J.; Hergenrother, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Microbiology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">291-298</span>CODEN:
                <span class="NLM_cas:coden">TRMIEA</span>;
        ISSN:<span class="NLM_cas:issn">0966-842X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Toxin-antitoxin (TA) systems are unique modules that effect plasmid stabilization via post-segregational killing of the bacterial host.  The genes encoding TA systems also exist on bacterial chromosomes, and it has been speculated that these are involved in a variety of cellular processes.  Interest in TA systems has increased dramatically over the past 5 years as the ubiquitous nature of TA genes on bacterial genomes has been revealed.  The exploitation of TA systems as an antibacterial strategy via artificial activation of the toxin has been proposed and has considerable potential; however, efforts in this area remain in the early stages and several major questions remain.  This review investigates the tractability of targeting TA systems to kill bacteria, including fundamental requirements for success, recent advances, and challenges assocd. with artificial toxin activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeA7u5NkOjXrVg90H21EOLACvtfcHk0li7eQlSBQ4JPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XksFejtLc%253D&md5=64e94986065d2d63b7a88cc350604fb0</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2012.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2012.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DJ.%2BJ.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DArtificial%2520Activation%2520of%2520Toxin-Antitoxin%2520Systems%2520as%2520an%2520Antibacterial%2520Strategy%26jtitle%3DTrends%2520Microbiol.%26date%3D2012%26volume%3D20%26spage%3D291%26epage%3D298%26doi%3D10.1016%2Fj.tim.2012.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Równicki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pieńko, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czarnecki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolanowska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartosik, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trylska, J.</span></span> <span> </span><span class="NLM_article-title">Artificial Activation of Escherichia coli mazEF and hipBA Toxin-Antitoxin Systems by Antisense Peptide Nucleic Acids as an Antibacterial Strategy</span>. <i>Front. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2870</span>, <span class="refDoi"> DOI: 10.3389/fmicb.2018.02870</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.3389%2Ffmicb.2018.02870" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=30534121" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A280%3ADC%252BB3cngsVKhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=2870&author=M.+R%C3%B3wnickiauthor=T.+Pie%C5%84koauthor=J.+Czarneckiauthor=M.+Kolanowskaauthor=D.+Bartosikauthor=J.+Trylska&title=Artificial+Activation+of+Escherichia+coli+mazEF+and+hipBA+Toxin-Antitoxin+Systems+by+Antisense+Peptide+Nucleic+Acids+as+an+Antibacterial+Strategy&doi=10.3389%2Ffmicb.2018.02870"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Artificial Activation of Escherichia coli mazEF and hipBA Toxin-Antitoxin Systems by Antisense Peptide Nucleic Acids as an Antibacterial Strategy</span></div><div class="casAuthors">Rownicki Marcin; Pienko Tomasz; Kolanowska Monika; Trylska Joanna; Rownicki Marcin; Pienko Tomasz; Czarnecki Jakub; Bartosik Dariusz; Czarnecki Jakub; Kolanowska Monika</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in microbiology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2870</span>
        ISSN:<span class="NLM_cas:issn">1664-302X</span>.
    </div><div class="casAbstract">The search for new, non-standard targets is currently a high priority in the design of new antibacterial compounds.  Bacterial toxin-antitoxin systems (TAs) are genetic modules that encode a toxin protein that causes growth arrest by interfering with essential cellular processes, and a cognate antitoxin, which neutralizes the toxin activity.  TAs have no human analogs, are highly abundant in bacterial genomes, and therefore represent attractive alternative targets for antimicrobial drugs.  This study demonstrates how artificial activation of Escherichia coli mazEF and hipBA toxin-antitoxin systems using sequence-specific antisense peptide nucleic acid oligomers is an innovative antibacterial strategy.  The growth arrest observed in E. coli resulted from the inhibition of translation of the antitoxins by the antisense oligomers.  Furthermore, two other targets, related to the activities of mazEF and hipBA, were identified as promising sites of action for antibacterials.  These results show that TAs are susceptible to sequence-specific antisense agents and provide a proof-of-concept for their further exploitation in antimicrobial strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRmSJfCr0K06hJ5E29YNGvQfW6udTcc2eYbH6bqJ738OLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cngsVKhsw%253D%253D&md5=27f4f683c6663bf410cc2cb9a9f7326e</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.3389%2Ffmicb.2018.02870&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffmicb.2018.02870%26sid%3Dliteratum%253Aachs%26aulast%3DR%25C3%25B3wnicki%26aufirst%3DM.%26aulast%3DPie%25C5%2584ko%26aufirst%3DT.%26aulast%3DCzarnecki%26aufirst%3DJ.%26aulast%3DKolanowska%26aufirst%3DM.%26aulast%3DBartosik%26aufirst%3DD.%26aulast%3DTrylska%26aufirst%3DJ.%26atitle%3DArtificial%2520Activation%2520of%2520Escherichia%2520coli%2520mazEF%2520and%2520hipBA%2520Toxin-Antitoxin%2520Systems%2520by%2520Antisense%2520Peptide%2520Nucleic%2520Acids%2520as%2520an%2520Antibacterial%2520Strategy%26jtitle%3DFront.%2520Microbiol.%26date%3D2018%26volume%3D9%26spage%3D2870%26doi%3D10.3389%2Ffmicb.2018.02870" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buchan, D. W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D. T.</span></span> <span> </span><span class="NLM_article-title">The PSIPRED Protein Analysis Workbench: 20 Years on</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">W402</span>– <span class="NLM_lpage">W407</span>, <span class="refDoi"> DOI: 10.1093/nar/gkz297</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgkz297" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=31251384" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BB3cXktVyiur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=W402-W407&author=D.+W.+A.+Buchanauthor=D.+T.+Jones&title=The+PSIPRED+Protein+Analysis+Workbench%3A+20+Years+on&doi=10.1093%2Fnar%2Fgkz297"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">The PSIPRED protein analysis workbench: 20 years on</span></div><div class="casAuthors">Buchan, Daniel W. A.; Jones, David T.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W402-W407</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The PSIPRED Workbench is a web server offering a range of predictive methods to the bioscience community for 20 years.  Here, we present the work we have completed to update the PSIPRED Protein Anal. Workbench and make it ready for the next 20 years.  The main focus of our recent website upgrade work has been the acceleration of analyses in the face of increasing protein sequence database size.  We addnl. discuss any new software, the new hardware infrastructure, our web services and web site.  Lastly we survey updates to some of the key predictive algorithms available through our website.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq97_XW9o_NtLVg90H21EOLACvtfcHk0lgaXxNu7VK4ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXktVyiur8%253D&md5=1bced7d73a52bc5220c0ebaf95d9ab70</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkz297&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkz297%26sid%3Dliteratum%253Aachs%26aulast%3DBuchan%26aufirst%3DD.%2BW.%2BA.%26aulast%3DJones%26aufirst%3DD.%2BT.%26atitle%3DThe%2520PSIPRED%2520Protein%2520Analysis%2520Workbench%253A%252020%2520Years%2520on%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26spage%3DW402%26epage%3DW407%26doi%3D10.1093%2Fnar%2Fgkz297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schreiter, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drennan, C. L.</span></span> <span> </span><span class="NLM_article-title">Ribbon-Helix-Helix Transcription Factors: Variations on a Theme</span>. <i>Nat. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">710</span>– <span class="NLM_lpage">720</span>, <span class="refDoi"> DOI: 10.1038/nrmicro1717</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1038%2Fnrmicro1717" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=17676053" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BD2sXptFyku7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2007&pages=710-720&author=E.+R.+Schreiterauthor=C.+L.+Drennan&title=Ribbon-Helix-Helix+Transcription+Factors%3A+Variations+on+a+Theme&doi=10.1038%2Fnrmicro1717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Ribbon-helix-helix transcription factors: variations on a theme</span></div><div class="casAuthors">Schreiter, Eric R.; Drennan, Catherine L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">710-720</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The ribbon-helix-helix (RHH) superfamily of transcription factors uses a conserved three-dimensional structural motif to bind to DNA in a sequence-specific manner.  This functionally diverse protein superfamily regulates the transcription of genes that are involved in the uptake of metals, amino-acid biosynthesis, cell division, the control of plasmid copy no., the lytic cycle of bacteriophages and, perhaps, many other cellular processes.  In this Anal., the structures of different RHH transcription factors are compared in order to evaluate the sequence motifs that are required for RHH-domain folding and DNA binding, as well as to identify conserved protein-DNA interactions in this superfamily.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDNof6E8hsMbVg90H21EOLACvtfcHk0lgaXxNu7VK4ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXptFyku7o%253D&md5=c4f6c6b2f9a606341b2da8c8d823c09e</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro1717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro1717%26sid%3Dliteratum%253Aachs%26aulast%3DSchreiter%26aufirst%3DE.%2BR.%26aulast%3DDrennan%26aufirst%3DC.%2BL.%26atitle%3DRibbon-Helix-Helix%2520Transcription%2520Factors%253A%2520Variations%2520on%2520a%2520Theme%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2007%26volume%3D5%26spage%3D710%26epage%3D720%26doi%3D10.1038%2Fnrmicro1717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McWilliam, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uludag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Squizzato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowley, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, R.</span></span> <span> </span><span class="NLM_article-title">Analysis Tool Web Services from the EMBL-EBI</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">W597</span>– <span class="NLM_lpage">W600</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt376</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgkt376" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=23671338" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=W597-W600&author=H.+McWilliamauthor=W.+Liauthor=M.+Uludagauthor=S.+Squizzatoauthor=Y.+M.+Parkauthor=N.+Busoauthor=A.+P.+Cowleyauthor=R.+Lopez&title=Analysis+Tool+Web+Services+from+the+EMBL-EBI&doi=10.1093%2Fnar%2Fgkt376"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt376%26sid%3Dliteratum%253Aachs%26aulast%3DMcWilliam%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DUludag%26aufirst%3DM.%26aulast%3DSquizzato%26aufirst%3DS.%26aulast%3DPark%26aufirst%3DY.%2BM.%26aulast%3DBuso%26aufirst%3DN.%26aulast%3DCowley%26aufirst%3DA.%2BP.%26aulast%3DLopez%26aufirst%3DR.%26atitle%3DAnalysis%2520Tool%2520Web%2520Services%2520from%2520the%2520EMBL-EBI%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3DW597%26epage%3DW600%26doi%3D10.1093%2Fnar%2Fgkt376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robert, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gouet, P.</span></span> <span> </span><span class="NLM_article-title">Deciphering Key Features in Protein Structures with the New ENDscript Server</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">W320</span>– <span class="NLM_lpage">W324</span>, <span class="refDoi"> DOI: 10.1093/nar/gku316</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgku316" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=24753421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFCqs7vK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=W320-W324&author=X.+Robertauthor=P.+Gouet&title=Deciphering+Key+Features+in+Protein+Structures+with+the+New+ENDscript+Server&doi=10.1093%2Fnar%2Fgku316"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Deciphering key features in protein structures with the new ENDscript server</span></div><div class="casAuthors">Robert, Xavier; Gouet, Patrice</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W320-W324</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ENDscript 2 is a friendly Web server for extg. and rendering a comprehensive anal. of primary to quaternary protein structure information in an automated way.  This major upgrade has been fully re-engineered to enhance speed, accuracy and usability with interactive 3D visualization.  It takes advantage of the new version 3 of ESPript, our well-known sequence alignment renderer, improved to handle a large no. of data with reduced computation time.  From a single PDB entry or file, ENDscript produces high quality figures displaying multiple sequence alignment of proteins homologous to the query, colored according to residue conservation.  Furthermore, the exptl. secondary structure elements and a detailed set of relevant biophys. and structural data are depicted.  All this information and more are now mapped on interactive 3D PyMOL representations.  Thanks to its adaptive and rigorous algorithm, beginner to expert users can modify settings to fine-tune ENDscript to their needs.  ENDscript has also been upgraded as an open platform for the visualization of multiple biochem. and structural data coming from external biotool Web servers, with both 2D and 3D representations.  ENDscript 2 and ESPript 3 are freely available at http://endscript.ibcp.fr and http://espript.ibcp.fr, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5lD0RT2rA9rVg90H21EOLACvtfcHk0lgaXxNu7VK4ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFCqs7vK&md5=96c315b6732a2be7fb72fa44be3511da</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku316%26sid%3Dliteratum%253Aachs%26aulast%3DRobert%26aufirst%3DX.%26aulast%3DGouet%26aufirst%3DP.%26atitle%3DDeciphering%2520Key%2520Features%2520in%2520Protein%2520Structures%2520with%2520the%2520New%2520ENDscript%2520Server%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DW320%26epage%3DW324%26doi%3D10.1093%2Fnar%2Fgku316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arcus, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rainey, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, S. J.</span></span> <span> </span><span class="NLM_article-title">The PIN-Domain Toxin-Antitoxin Array in Mycobacteria</span>. <i>Trends Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">360</span>– <span class="NLM_lpage">365</span>, <span class="refDoi"> DOI: 10.1016/j.tim.2005.06.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1016%2Fj.tim.2005.06.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=15993073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmvFSisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2005&pages=360-365&author=V.+L.+Arcusauthor=P.+B.+Raineyauthor=S.+J.+Turner&title=The+PIN-Domain+Toxin-Antitoxin+Array+in+Mycobacteria&doi=10.1016%2Fj.tim.2005.06.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The PIN-domain toxin-antitoxin array in mycobacteria</span></div><div class="casAuthors">Arcus, Vickery L.; Rainey, Paul B.; Turner, Susan J.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Microbiology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">360-365</span>CODEN:
                <span class="NLM_cas:coden">TRMIEA</span>;
        ISSN:<span class="NLM_cas:issn">0966-842X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  PIN-domains (homologs of the pilT N-terminal domain) are small protein domains of ∼140 amino acids.  They are found in a diverse range of organisms and recent evidence from bioinformatics, biochem., structural biol. and microbiol. suggest that the majority of the prokaryotic PIN-domain proteins are the toxic components of toxin-antitoxin (TA) operons.  Several microorganisms have a large cohort of these operons.  For example, the genome of Mycobacterium tuberculosis encodes 48 PIN-domain proteins, of which 38 are thought to be involved in TA interactions.  This large array of PIN-domain TA operons raises questions as to their evolutionary origin and contemporary functional significance.  We suggest that the evolutionary origin of genes encoding mycobacterial PIN-domain TA operons is linked to the mobile gene pool, but that TA operons can become resident within the chromosome of host cells from where they might be recruited to fulfil a variety of roles assocd. with retardation of cell growth and persistence in stressful environments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofELKYHG-bOrVg90H21EOLACvtfcHk0lgaXxNu7VK4ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmvFSisLk%253D&md5=b2ee960976b941a4858ec8bbc049055c</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2005.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2005.06.008%26sid%3Dliteratum%253Aachs%26aulast%3DArcus%26aufirst%3DV.%2BL.%26aulast%3DRainey%26aufirst%3DP.%2BB.%26aulast%3DTurner%26aufirst%3DS.%2BJ.%26atitle%3DThe%2520PIN-Domain%2520Toxin-Antitoxin%2520Array%2520in%2520Mycobacteria%26jtitle%3DTrends%2520Microbiol.%26date%3D2005%26volume%3D13%26spage%3D360%26epage%3D365%26doi%3D10.1016%2Fj.tim.2005.06.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arcus, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenzie, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, G. M.</span></span> <span> </span><span class="NLM_article-title">The PIN-Domain Ribonucleases and the Prokaryotic VapBC Toxin-Antitoxin Array</span>. <i>Protein Eng., Des. Sel.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">40</span>, <span class="refDoi"> DOI: 10.1093/protein/gzq081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fprotein%2Fgzq081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=21036780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A280%3ADC%252BC3M%252Fjt1yjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2011&pages=33-40&author=V.+L.+Arcusauthor=J.+L.+McKenzieauthor=J.+Robsonauthor=G.+M.+Cook&title=The+PIN-Domain+Ribonucleases+and+the+Prokaryotic+VapBC+Toxin-Antitoxin+Array&doi=10.1093%2Fprotein%2Fgzq081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The PIN-domain ribonucleases and the prokaryotic VapBC toxin-antitoxin array</span></div><div class="casAuthors">Arcus Vickery L; McKenzie Joanna L; Robson Jennifer; Cook Gregory M</div><div class="citationInfo"><span class="NLM_cas:title">Protein engineering, design & selection : PEDS</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">33-40</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The PIN-domains are small proteins of ~130 amino acids that are found in bacteria, archaea and eukaryotes and are defined by a group of three strictly conserved acidic amino acids.  The conserved three-dimensional structures of the PIN-domains cluster these acidic residues in an enzymatic active site.  PIN-domains cleave single-stranded RNA in a sequence-specific, Mg2+- or Mn2+-dependent manner.  These ribonucleases are toxic to the cells which express them and to offset this toxicity, they are co-expressed with tight binding protein inhibitors.  The genes encoding these two proteins are adjacent in the genome of all prokaryotic organisms where they are found.  This sequential arrangement of inhibitor-RNAse genes conforms to that of the so-called toxin-antitoxin (TA) modules and the PIN-domain TAs have been named VapBC TAs (virulence associated proteins, VapB is the inhibitor which contains a transcription factor domain and VapC is the PIN-domain ribonuclease).  The presence of large numbers of vapBC loci in disparate prokaryotes has motivated many researchers to investigate their biochemical and biological functions.  For example, the devastating human pathogen Mycobacterium tuberculosis has 45 vapBC loci encoded in its genome whereas its non-pathogenic relative, Mycobacterium smegmatis has just one vapBC operon.  On another branch of the prokaryotic tree, the nitrogen-fixing symbiont of legumes, Sinorhizobium meliloti has 21 vapBC loci and at least one of these loci have been implicated in the regulation of growth in the plant nodule.  A range of biological functions has been suggested for these operons and this review sets out to survey the PIN-domains and summarise the current knowledge about the vapBC TA systems and their roles in diverse bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSYcLBUbiNT5Rgd2Wuil24tfW6udTcc2eZwHnM_FV8JZrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3M%252Fjt1yjuw%253D%253D&md5=d3c1bb299ca62ea690b84063250d63ce</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2Fgzq081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252Fgzq081%26sid%3Dliteratum%253Aachs%26aulast%3DArcus%26aufirst%3DV.%2BL.%26aulast%3DMcKenzie%26aufirst%3DJ.%2BL.%26aulast%3DRobson%26aufirst%3DJ.%26aulast%3DCook%26aufirst%3DG.%2BM.%26atitle%3DThe%2520PIN-Domain%2520Ribonucleases%2520and%2520the%2520Prokaryotic%2520VapBC%2520Toxin-Antitoxin%2520Array%26jtitle%3DProtein%2520Eng.%252C%2520Des.%2520Sel.%26date%3D2011%26volume%3D24%26spage%3D33%26epage%3D40%26doi%3D10.1093%2Fprotein%2Fgzq081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coles, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Djuranovic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Söding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frickey, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koretke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truffault, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lupas, A. N.</span></span> <span> </span><span class="NLM_article-title">AbrB-Like Transcription Factors Assume a Swapped Hairpin Fold that is Evolutionarily Related to Double-Psi Beta Barrels</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">919</span>– <span class="NLM_lpage">928</span>, <span class="refDoi"> DOI: 10.1016/j.str.2005.03.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1016%2Fj.str.2005.03.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=15939023" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltFCksbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2005&pages=919-928&author=M.+Colesauthor=S.+Djuranovicauthor=J.+S%C3%B6dingauthor=T.+Frickeyauthor=K.+Koretkeauthor=V.+Truffaultauthor=J.+Martinauthor=A.+N.+Lupas&title=AbrB-Like+Transcription+Factors+Assume+a+Swapped+Hairpin+Fold+that+is+Evolutionarily+Related+to+Double-Psi+Beta+Barrels&doi=10.1016%2Fj.str.2005.03.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">AbrB-like Transcription Factors Assume a Swapped Hairpin Fold that Is Evolutionarily Related to Double-Psi β Barrels</span></div><div class="casAuthors">Coles, Murray; Djuranovic, Sergej; Soeding, Johannes; Frickey, Tancred; Koretke, Kristin; Truffault, Vincent; Martin, Joerg; Lupas, Andrei N.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">919-928</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">AbrB is a key transition-state regulator of Bacillus subtilis.  Based on the conservation of a βαβ structural unit, we proposed a β barrel fold for its DNA binding domain, similar to, but topol. distinct from, double-psi β barrels.  However, the NMR structure revealed a novel fold, the "looped-hinge helix.  " To understand this discrepancy, we undertook a bioinformatics study of AbrB and its homologs; these form a large superfamily, which includes SpoVT, PrlF, MraZ, addiction module antidotes (PemI, MazE), plasmid maintenance proteins (VagC, VapB), and archaeal PhoU homologs.  MazE and MraZ form swapped-hairpin β barrels.  We therefore reexamd. the fold of AbrB by NMR spectroscopy and found that it also forms a swapped-hairpin barrel.  The conservation of the core βαβ element supports a common evolutionary origin for swapped-hairpin and double-psi barrels, which we group into a higher-order class, the cradle-loop barrels, based on the peculiar shape of their ligand binding site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8QSWF06S1XLVg90H21EOLACvtfcHk0lg3jMWpei2ymA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltFCksbw%253D&md5=49c94348b872276896599815818eede9</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2005.03.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2005.03.017%26sid%3Dliteratum%253Aachs%26aulast%3DColes%26aufirst%3DM.%26aulast%3DDjuranovic%26aufirst%3DS.%26aulast%3DS%25C3%25B6ding%26aufirst%3DJ.%26aulast%3DFrickey%26aufirst%3DT.%26aulast%3DKoretke%26aufirst%3DK.%26aulast%3DTruffault%26aufirst%3DV.%26aulast%3DMartin%26aufirst%3DJ.%26aulast%3DLupas%26aufirst%3DA.%2BN.%26atitle%3DAbrB-Like%2520Transcription%2520Factors%2520Assume%2520a%2520Swapped%2520Hairpin%2520Fold%2520that%2520is%2520Evolutionarily%2520Related%2520to%2520Double-Psi%2520Beta%2520Barrels%26jtitle%3DStructure%26date%3D2005%26volume%3D13%26spage%3D919%26epage%3D928%26doi%3D10.1016%2Fj.str.2005.03.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henrick, K.</span></span> <span> </span><span class="NLM_article-title">Inference of Macromolecular Assemblies from Crystalline State</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>372</i></span>,  <span class="NLM_fpage">774</span>– <span class="NLM_lpage">797</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2007.05.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1016%2Fj.jmb.2007.05.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=17681537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpvFGktb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=372&publication_year=2007&pages=774-797&author=E.+Krissinelauthor=K.+Henrick&title=Inference+of+Macromolecular+Assemblies+from+Crystalline+State&doi=10.1016%2Fj.jmb.2007.05.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Inference of Macromolecular Assemblies from Crystalline State</span></div><div class="casAuthors">Krissinel, Evgeny; Henrick, Kim</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">372</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">774-797</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The authors discuss basic phys.-chem. principles underlying the formation of stable macromol. complexes, which in many cases are likely to be the biol. units performing a certain physiol. function.  The authors also consider available theor. approaches to the calcn. of macromol. affinity and entropy of complexation.  The latter is shown to play an important role and make a major effect on complex size and symmetry.  The authors develop a new method, based on chem. thermodn., for automatic detection of macromol. assemblies in the Protein Data Bank (PDB) entries that are the results of x-ray diffraction expts.  As found, biol. units may be recovered at 80-90% success rate, which makes x-ray crystallog. an important source of exptl. data on macromol. complexes and protein-protein interactions.  The method is implemented as a public WWW service (http://www.ebi.ac.uk/msd-srv/prot_int/pistart.html).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3cilBvVoBS7Vg90H21EOLACvtfcHk0lg3jMWpei2ymA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpvFGktb8%253D&md5=a5c764cfc7dc129f53ddc31ef9d475fa</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2007.05.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2007.05.022%26sid%3Dliteratum%253Aachs%26aulast%3DKrissinel%26aufirst%3DE.%26aulast%3DHenrick%26aufirst%3DK.%26atitle%3DInference%2520of%2520Macromolecular%2520Assemblies%2520from%2520Crystalline%2520State%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D372%26spage%3D774%26epage%3D797%26doi%3D10.1016%2Fj.jmb.2007.05.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tina, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhadra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, N.</span></span> <span> </span><span class="NLM_article-title">PIC: Protein Interactions Calculator</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">W473</span>– <span class="NLM_lpage">W476</span>, <span class="refDoi"> DOI: 10.1093/nar/gkm423</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgkm423" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=17584791" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A280%3ADC%252BD2svhsVOltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=W473-W476&author=K.+G.+Tinaauthor=R.+Bhadraauthor=N.+Srinivasan&title=PIC%3A+Protein+Interactions+Calculator&doi=10.1093%2Fnar%2Fgkm423"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">PIC: Protein Interactions Calculator</span></div><div class="casAuthors">Tina K G; Bhadra R; Srinivasan N</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic acids research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">Web Server issue</span>),
    <span class="NLM_cas:pages">W473-6</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Interactions within a protein structure and interactions between proteins in an assembly are essential considerations in understanding molecular basis of stability and functions of proteins and their complexes.  There are several weak and strong interactions that render stability to a protein structure or an assembly.  Protein Interactions Calculator (PIC) is a server which, given the coordinate set of 3D structure of a protein or an assembly, computes various interactions such as disulphide bonds, interactions between hydrophobic residues, ionic interactions, hydrogen bonds, aromatic-aromatic interactions, aromatic-sulphur interactions and cation-pi interactions within a protein or between proteins in a complex.  Interactions are calculated on the basis of standard, published criteria.  The identified interactions between residues can be visualized using a RasMol and Jmol interface.  The advantage with PIC server is the easy availability of inter-residue interaction calculations in a single site.  It also determines the accessible surface area and residue-depth, which is the distance of a residue from the surface of the protein.  User can also recognize specific kind of interactions, such as apolar-apolar residue interactions or ionic interactions, that are formed between buried or exposed residues or near the surface or deep inside.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSKn5zO1OOtqL5XHngOXiV_fW6udTcc2ebgpaYQ3xtILLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2svhsVOltQ%253D%253D&md5=6c59648eeba47a6175754ed6d7499c14</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm423%26sid%3Dliteratum%253Aachs%26aulast%3DTina%26aufirst%3DK.%2BG.%26aulast%3DBhadra%26aufirst%3DR.%26aulast%3DSrinivasan%26aufirst%3DN.%26atitle%3DPIC%253A%2520Protein%2520Interactions%2520Calculator%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DW473%26epage%3DW476%26doi%3D10.1093%2Fnar%2Fgkm423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luong, H. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, B.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.-W.</span></span> <span> </span><span class="NLM_article-title">Effects of Lysine-to-Arginine Substitution on Antimicrobial Activity of Cationic Stapled Heptapeptides</span>. <i>Arch. Pharmacal Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">1092</span>– <span class="NLM_lpage">1097</span>, <span class="refDoi"> DOI: 10.1007/s12272-018-1084-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1007%2Fs12272-018-1084-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=30361948" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVWjt7fO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2018&pages=1092-1097&author=H.+X.+Luongauthor=D.-H.+Kimauthor=B.-J.+Leeauthor=Y.-W.+Kim&title=Effects+of+Lysine-to-Arginine+Substitution+on+Antimicrobial+Activity+of+Cationic+Stapled+Heptapeptides&doi=10.1007%2Fs12272-018-1084-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of lysine-to-arginine substitution on antimicrobial activity of cationic stapled heptapeptides</span></div><div class="casAuthors">Luong, Huy X.; Kim, Do-Hee; Lee, Bong-Jin; Kim, Young-Woo</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Pharmacal Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1092-1097</span>CODEN:
                <span class="NLM_cas:coden">APHRDQ</span>;
        ISSN:<span class="NLM_cas:issn">0253-6269</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Korea</span>)
        </div><div class="casAbstract">We previously reported a series of amphipathic helixes of stapled heptapeptides as membrane-lytic antimicrobial peptides (AMPs).  These peptides possess 3 Lys residues as the sole cationic residues in their hydrophilic face of the helix.  Lys-to-Arg residue substitution is often shown to increase antimicrobial activity of many natural AMPs due to the more favorable interactions of guanidinium moiety of Arg with membranes.  In an effort to further improve the pharmacol. properties of our novel AMP series, we here examd. the impact of Lys-to-Arg substitution on their structures and antimicrobial and hemolytic activities.  The results indicated that the Lys-to-Arg substitution does not always guarantee enhancement in the antimicrobial potency of AMPs.  Instead, we obsd. varied potency and selectivity depending on the no. of substitutions and the positions substituted.  The results implied that, in the given helical scaffold stabilized by a hydrocarbon staple, antimicrobial potency and selectivity are influenced by a complex effect of various structural and chem. changes accompanied by Lys-to-Arg substitution rather than solely by the type of cationic residue.  These data show potential for use in our scaffold-assisted development of short, selective, and metabolically stable AMPs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ5fDZQtqhq7Vg90H21EOLACvtfcHk0lg3jMWpei2ymA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVWjt7fO&md5=9d5305d18f105c57dc46ad4d3717dab2</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1007%2Fs12272-018-1084-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12272-018-1084-5%26sid%3Dliteratum%253Aachs%26aulast%3DLuong%26aufirst%3DH.%2BX.%26aulast%3DKim%26aufirst%3DD.-H.%26aulast%3DLee%26aufirst%3DB.-J.%26aulast%3DKim%26aufirst%3DY.-W.%26atitle%3DEffects%2520of%2520Lysine-to-Arginine%2520Substitution%2520on%2520Antimicrobial%2520Activity%2520of%2520Cationic%2520Stapled%2520Heptapeptides%26jtitle%3DArch.%2520Pharmacal%2520Res.%26date%3D2018%26volume%3D41%26spage%3D1092%26epage%3D1097%26doi%3D10.1007%2Fs12272-018-1084-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laskowski, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swindells, M. B.</span></span> <span> </span><span class="NLM_article-title">LigPlot+: Multiple Ligand-Protein Interaction Diagrams for Drug Discovery</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2778</span>– <span class="NLM_lpage">2786</span>, <span class="refDoi"> DOI: 10.1021/ci200227u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200227u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1KjtrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2778-2786&author=R.+A.+Laskowskiauthor=M.+B.+Swindells&title=LigPlot%2B%3A+Multiple+Ligand-Protein+Interaction+Diagrams+for+Drug+Discovery&doi=10.1021%2Fci200227u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">LigPlot+: Multiple Ligand-Protein Interaction Diagrams for Drug Discovery</span></div><div class="casAuthors">Laskowski, Roman A.; Swindells, Mark B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2778-2786</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe a graphical system for automatically generating multiple 2D diagrams of ligand-protein interactions from 3D coordinates.  The diagrams portray the hydrogen-bond interaction patterns and hydrophobic contacts between the ligand(s) and the main-chain or side-chain elements of the protein.  The system is able to plot, in the same orientation, related sets of ligand-protein interactions.  This facilitates popular research tasks, such as analyzing a series of small mols. binding to the same protein target, a single ligand binding to homologous proteins, or the completely general case where both protein and ligand change.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYuWvYH7Nel7Vg90H21EOLACvtfcHk0liaQOlSOkmtzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1KjtrzP&md5=576e59ad4f0c4304d01c6c80301ecf39</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fci200227u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200227u%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DSwindells%26aufirst%3DM.%2BB.%26atitle%3DLigPlot%252B%253A%2520Multiple%2520Ligand-Protein%2520Interaction%2520Diagrams%2520for%2520Drug%2520Discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2778%26epage%3D2786%26doi%3D10.1021%2Fci200227u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kedzierska, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lian, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, F.</span></span> <span> </span><span class="NLM_article-title">Toxin-Antitoxin Regulation: Bimodal Interaction of YefM-YoeB with Paired DNA Palindromes Exerts Transcriptional Autorepression</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">325</span>– <span class="NLM_lpage">339</span>, <span class="refDoi"> DOI: 10.1093/nar/gkl1028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1093%2Fnar%2Fgkl1028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=17170003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVyitr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=325-339&author=B.+Kedzierskaauthor=L.+Y.+Lianauthor=F.+Hayes&title=Toxin-Antitoxin+Regulation%3A+Bimodal+Interaction+of+YefM-YoeB+with+Paired+DNA+Palindromes+Exerts+Transcriptional+Autorepression&doi=10.1093%2Fnar%2Fgkl1028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Toxin-antitoxin regulation: bimodal interaction of YefM-YoeB with paired DNA palindromes exerts transcriptional autorepression</span></div><div class="casAuthors">Kedzierska, Barbara; Lian, Lu-Yun; Hayes, Finbarr</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">325-339</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Toxin-antitoxin (TA) complexes function in programmed cell death or stress response mechanisms in bacteria.  The YefM-YoeB TA complex of Escherichia coli consists of YoeB toxin that is counteracted by YefM antitoxin.  When liberated from the complex, YoeB acts as an endoribonuclease, preferentially cleaving 3' of purine nucleotides.  Here we demonstrate that yefM-yoeB is transcriptionally autoregulated.  YefM, a dimeric protein with extensive secondary structure revealed by CD and NMR spectroscopy, is the primary repressor, whereas YoeB is a repression enhancer.  The operator site 5' of yefM-yoeB comprises adjacent long and short palindromes with core 5'-TGTACA-3' motifs.  YefM binds the long palindrome, followed sequentially by short palindrome recognition.  In contrast, the repressor-corepressor complex recognizes both motifs more avidly, implying that YefM within the complex has an enhanced DNA-binding affinity compared to free YefM.  Operator interaction by YefM and YefM-YoeB is accompanied by structural transitions in the proteins.  Paired 5'-TGTACA-3' motifs are common in yefM-yoeB regulatory regions in diverse genomes suggesting that interaction of YefM-YoeB with these motifs is a conserved mechanism of operon autoregulation.  Artificial perturbation of transcriptional autorepression could elicit inappropriate YoeB toxin prodn. and induction of bacterial cell suicide, a potentially novel antibacterial strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCiSray-MBfrVg90H21EOLACvtfcHk0liaQOlSOkmtzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVyitr4%253D&md5=8c355a0480750fbef4c5ec2f4ef097de</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl1028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl1028%26sid%3Dliteratum%253Aachs%26aulast%3DKedzierska%26aufirst%3DB.%26aulast%3DLian%26aufirst%3DL.%2BY.%26aulast%3DHayes%26aufirst%3DF.%26atitle%3DToxin-Antitoxin%2520Regulation%253A%2520Bimodal%2520Interaction%2520of%2520YefM-YoeB%2520with%2520Paired%2520DNA%2520Palindromes%2520Exerts%2520Transcriptional%2520Autorepression%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3D325%26epage%3D339%26doi%3D10.1093%2Fnar%2Fgkl1028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Overgaard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borch, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jørgensen, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerdes, K.</span></span> <span> </span><span class="NLM_article-title">Messenger RNA Interferase RelE Controls relBE Transcription by Conditional Cooperativity</span>. <i>Mol. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">841</span>– <span class="NLM_lpage">857</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2958.2008.06313.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1111%2Fj.1365-2958.2008.06313.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=18532983" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVehurzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2008&pages=841-857&author=M.+Overgaardauthor=J.+Borchauthor=M.+G.+J%C3%B8rgensenauthor=K.+Gerdes&title=Messenger+RNA+Interferase+RelE+Controls+relBE+Transcription+by+Conditional+Cooperativity&doi=10.1111%2Fj.1365-2958.2008.06313.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Messenger RNA interferase RelE controls relBE transcription by conditional cooperativity</span></div><div class="casAuthors">Overgaard, Martin; Borch, Jonas; Joergensen, Mikkel G.; Gerdes, Kenn</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Microbiology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">841-857</span>CODEN:
                <span class="NLM_cas:coden">MOMIEE</span>;
        ISSN:<span class="NLM_cas:issn">0950-382X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Prokaryotic toxin-antitoxin (TA) loci consist of two genes in an operon that encodes a metabolically stable toxin and an unstable antitoxin.  The antitoxin neutralizes its cognate toxin by forming a tight complex with it.  In all cases known, the antitoxin auto-regulates TA operon transcription by binding to one or more operators in the promoter region while the toxin functions as a co-repressor of transcription.  Interestingly, the toxin can also stimulate TA operon transcription.  Here we analyze mechanistic aspects of how RelE of Escherichia coli can function both as a co-repressor and as a derepressor of relBE transcription.  When RelB was in excess to RelE, two trimeric RelB2•RelE complexes bound cooperatively to two adjacent operator sites in the relBE promoter region and repressed transcription.  In contrast, RelE in excess stimulated relBE transcription and released the RelB2•RelE complex from operator DNA.  A mutational anal. of the operator sites showed that RelE in excess counteracted cooperative binding of the RelB2•RelE complexes to the operator sites.  Thus, RelE controls relBE transcription by conditional cooperativity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnOKvpn90C8bVg90H21EOLACvtfcHk0liaQOlSOkmtzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVehurzE&md5=6b2590d4ad21756e1bf896ac8c3d5573</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2958.2008.06313.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2958.2008.06313.x%26sid%3Dliteratum%253Aachs%26aulast%3DOvergaard%26aufirst%3DM.%26aulast%3DBorch%26aufirst%3DJ.%26aulast%3DJ%25C3%25B8rgensen%26aufirst%3DM.%2BG.%26aulast%3DGerdes%26aufirst%3DK.%26atitle%3DMessenger%2520RNA%2520Interferase%2520RelE%2520Controls%2520relBE%2520Transcription%2520by%2520Conditional%2520Cooperativity%26jtitle%3DMol.%2520Microbiol.%26date%3D2008%26volume%3D69%26spage%3D841%26epage%3D857%26doi%3D10.1111%2Fj.1365-2958.2008.06313.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">[20] Processing of X-Ray Diffraction Data Collected in Oscillation Mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1016/s0076-6879(97)76066-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=%5B20%5D+Processing+of+X-Ray+Diffraction+Data+Collected+in+Oscillation+Mode&doi=10.1016%2Fs0076-6879%2897%2976066-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lign7XFYgfHlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3D%255B20%255D%2520Processing%2520of%2520X-Ray%2520Diffraction%2520Data%2520Collected%2520in%2520Oscillation%2520Mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2Fs0076-6879%2897%2976066-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: A Comprehensive Python-Based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/s0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2Fs0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lign7XFYgfHlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2Fs0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lign7XFYgfHlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skubák, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span> <span> </span><span class="NLM_article-title">REFMAC5 for the Refinement of Macromolecular Crystal Structures</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1107/s0907444911001314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1107%2FS0907444911001314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=21460454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=355-367&author=G.+N.+Murshudovauthor=P.+Skub%C3%A1kauthor=A.+A.+Lebedevauthor=N.+S.+Pannuauthor=R.+A.+Steinerauthor=R.+A.+Nichollsauthor=M.+D.+Winnauthor=F.+Longauthor=A.+A.+Vagin&title=REFMAC5+for+the+Refinement+of+Macromolecular+Crystal+Structures&doi=10.1107%2Fs0907444911001314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">REFMAC5 for the refinement of macromolecular crystal structures</span></div><div class="casAuthors">Murshudov, Garib N.; Skubak, Pavol; Lebedev, Andrey A.; Pannu, Navraj S.; Steiner, Roberto A.; Nicholls, Robert A.; Winn, Martyn D.; Long, Fei; Vagin, Alexei A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">355-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">This paper describes various components of the macromol. crystallog. refinement program REFMAC5, which is distributed as part of the CCP4 suite.  REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS exptl. diffraction data.  To ensure chem. and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization.  Reliable models at resolns. at least as low as 4 Å can be achieved thanks to low-resoln. refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on at. displacement parameters (ADPs) based on the Kullback-Leibler divergence.  REFMAC5 addnl. offers TLS parameterization and, when high-resoln. data are available, fast refinement of anisotropic ADPs.  Refinement in the presence of twinning is performed in a fully automated fashion.  REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resoln. spectrum encountered in macromol. crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Jq-i2qIq5rVg90H21EOLACvtfcHk0lign7XFYgfHlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D&md5=f8f3202d246908500057ad7c71015b7b</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1107%2FS0907444911001314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911001314%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DSkub%25C3%25A1k%26aufirst%3DP.%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DSteiner%26aufirst%3DR.%2BA.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DLong%26aufirst%3DF.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26atitle%3DREFMAC5%2520for%2520the%2520Refinement%2520of%2520Macromolecular%2520Crystal%2520Structures%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D355%26epage%3D367%26doi%3D10.1107%2Fs0907444911001314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Immormino, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: All-Atom Structure Validation for Macromolecular Crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1107/s0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1107%2Fs0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+All-Atom+Structure+Validation+for+Macromolecular+Crystallography&doi=10.1107%2Fs0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0lgl2Vj6mEW7Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2Fs0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520All-Atom%2520Structure%2520Validation%2520for%2520Macromolecular%2520Crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2Fs0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delano, W. L.</span></span> <i>The PyMOL Molecular Graphics System</i>; <span class="NLM_publisher-name">DeLano Scientific</span>: <span class="NLM_publisher-loc">San Carlos, CA</span>, <span class="NLM_year">2002</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2002&author=W.+L.+Delano&title=The+PyMOL+Molecular+Graphics+System"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DDelano%26aufirst%3DW.%2BL.%26btitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26pub%3DDeLano%2520Scientific%26date%3D2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, C. L.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vieth, M.</span></span> <span> </span><span class="NLM_article-title">Detailed Analysis of Grid-based Molecular Docking: A Case Study of CDOCKER-A CHARMm-Based MD Docking Algorithm</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1549</span>– <span class="NLM_lpage">1562</span>, <span class="refDoi"> DOI: 10.1002/jcc.10306</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=10.1002%2Fjcc.10306" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=12925999" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntFent70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2003&pages=1549-1562&author=G.+Wuauthor=D.+H.+Robertsonauthor=C.+L.+Brooksauthor=M.+Vieth&title=Detailed+Analysis+of+Grid-based+Molecular+Docking%3A+A+Case+Study+of+CDOCKER-A+CHARMm-Based+MD+Docking+Algorithm&doi=10.1002%2Fjcc.10306"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Detailed analysis of grid-based molecular docking: A case study of CDOCKER-A CHARMm-based MD docking algorithm</span></div><div class="casAuthors">Wu, Guosheng; Robertson, Daniel H.; Brooks, Charles L., III; Vieth, Michal</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1549-1562</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The influence of various factors on the accuracy of protein-ligand docking is examd.  The factors investigated include the role of a grid representation of protein-ligand interactions, the initial ligand conformation and orientation, the sampling rate of the energy hyper-surface, and the final minimization.  A representative docking method is used to study these factors, namely, CDOCKER, a mol. dynamics (MD) simulated-annealing-based algorithm.  A major emphasis in these studies is to compare the relative performance and accuracy of various grid-based approxns. to explicit all-atom force field calcns.  In these docking studies, the protein is kept rigid while the ligands are treated as fully flexible and a final minimization step is used to refine the docked poses.  A docking success rate of 74% is obsd. when an explicit all-atom representation of the protein (full force field) is used, while a lower accuracy of 66-76% is obsd. for grid-based methods.  All docking expts. considered a 41-member protein-ligand validation set.  A significant improvement in accuracy (76 vs. 66%) for the grid-based docking is achieved if the explicit all-atom force field is used in a final minimization step to refine the docking poses.  Statistical anal. shows that even lower-accuracy grid-based energy representations can be effectively used when followed with full force field minimization.  The results of these grid-based protocols are statistically indistinguishable from the detailed at. dockings and provide up to a sixfold redn. in computation time.  For the test case examd. here, improving the docking accuracy did not necessarily enhance the ability to est. binding affinities using the docked structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvNo4ih6vcd7Vg90H21EOLACvtfcHk0lgl2Vj6mEW7Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXntFent70%253D&md5=34d16a1774c60ea18a5f7da58f478b71</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10306%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DG.%26aulast%3DRobertson%26aufirst%3DD.%2BH.%26aulast%3DBrooks%26aufirst%3DC.%2BL.%26aulast%3DVieth%26aufirst%3DM.%26atitle%3DDetailed%2520Analysis%2520of%2520Grid-based%2520Molecular%2520Docking%253A%2520A%2520Case%2520Study%2520of%2520CDOCKER-A%2520CHARMm-Based%2520MD%2520Docking%2520Algorithm%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2003%26volume%3D24%26spage%3D1549%26epage%3D1562%26doi%3D10.1002%2Fjcc.10306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY2','PDB','7BY2'); return false;">PDB: 7BY2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BY3','PDB','7BY3'); return false;">PDB: 7BY3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6A7V" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6A7V','PDB','6A7V'); return false;">PDB: 6A7V</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5X3T" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5X3T','PDB','5X3T'); return false;">PDB: 5X3T</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K8J" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K8J','PDB','5K8J'); return false;">PDB: 5K8J</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6NKL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6NKL','PDB','6NKL'); return false;">PDB: 6NKL</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i20"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01118">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_04377"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01118?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01118</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Growth assay and cfu measurement; effects of the peptides and small molecules on free VapC; inhibition curves of compounds; ITC assays; primers used for cloning and producing mutations; data collection and refinement statistics for SeMet-substituted and native structures; peptides used to disrupt the binding interface of VapBC; and synthetic procedures (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_002.csv">CSV</a>)</p></li><li><p class="inline">Hit compounds and peptide analysis reports (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_003.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_001.pdf">jm0c01118_si_001.pdf (670.11 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_002.csv">jm0c01118_si_002.csv (0.16 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01118/suppl_file/jm0c01118_si_003.pdf">jm0c01118_si_003.pdf (4.68 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01118&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01118%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-22" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01118" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67996cfe3c6e2290","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
