
<html lang="en"     class="pb-page"  data-request-id="7f30f7f1-d1af-40e9-98eb-4e897a7f7b8e"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2020.63.issue-23;article:article:10.1021/acs.jmedchem.0c01242;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Pan-SMARCA/PB1 Bromodomain Inhibitors and Their Role in Regulating Adipogenesis" /></meta><meta name="dc.Creator" content="Marek  Wanior" /></meta><meta name="dc.Creator" content="Franziska  Preuss" /></meta><meta name="dc.Creator" content="Xiaomin  Ni" /></meta><meta name="dc.Creator" content="Andreas  Krämer" /></meta><meta name="dc.Creator" content="Sebastian  Mathea" /></meta><meta name="dc.Creator" content="Tamara  Göbel" /></meta><meta name="dc.Creator" content="David  Heidenreich" /></meta><meta name="dc.Creator" content="Svenja  Simonyi" /></meta><meta name="dc.Creator" content="Astrid S.  Kahnt" /></meta><meta name="dc.Creator" content="Andreas C.  Joerger" /></meta><meta name="dc.Creator" content="Stefan  Knapp" /></meta><meta name="dc.Description" content="Accessibility of the human genome is modulated by the ATP-driven SWI/SNF chromatin remodeling multiprotein complexes BAF (BRG1/BRM-associated factor) and PBAF (polybromo-associated BAF factor), whi..." /></meta><meta name="Description" content="Accessibility of the human genome is modulated by the ATP-driven SWI/SNF chromatin remodeling multiprotein complexes BAF (BRG1/BRM-associated factor) and PBAF (polybromo-associated BAF factor), whi..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 20, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01242" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01242" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01242" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01242" /></link>
        
    
    

<title>Pan-SMARCA/PB1 Bromodomain Inhibitors and Their Role in Regulating Adipogenesis | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01242" /></meta><meta property="og:title" content="Pan-SMARCA/PB1 Bromodomain Inhibitors and Their Role in Regulating Adipogenesis" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/large/jm0c01242_0009.jpeg" /></meta><meta property="og:description" content="Accessibility of the human genome is modulated by the ATP-driven SWI/SNF chromatin remodeling multiprotein complexes BAF (BRG1/BRM-associated factor) and PBAF (polybromo-associated BAF factor), which involves reading of acetylated histone tails by the bromodomain-containing proteins SMARCA2 (BRM), SMARCA4 (BRG1), and polybromo-1. Dysregulation of chromatin remodeling leads to aberrant cell proliferation and differentiation. Here, we have characterized a set of potent and cell-active bromodomain inhibitors with pan-selectivity for canonical family VIII bromodomains. Targeted SWI/SNF bromodomain inhibition blocked the expression of key genes during adipogenesis, including the transcription factors PPARγ and C/EBPα, and impaired the differentiation of 3T3-L1 murine fibroblasts into adipocytes. Our data highlight the role of SWI/SNF bromodomains in adipogenesis and provide a framework for the development of SWI/SNF bromodomain inhibitors for indirect targeting of key transcription factors regulating cell differentiation." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01242"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01242">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01242&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01242&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01242&amp;href=/doi/10.1021/acs.jmedchem.0c01242" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 23</span><span class="cit-fg-pageRange">, 14680-14699</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/23" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01233" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01264" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Pan-SMARCA/PB1 Bromodomain Inhibitors and Their Role in Regulating Adipogenesis</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Marek Wanior</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marek Wanior</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Goethe University Frankfurt, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</div><div class="loa-info-affiliations-info">Structural Genomics Consortium (SGC), Buchmann Institute for Molecular Life Sciences (BMLS), Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marek++Wanior">Marek Wanior</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9569-4487" title="Orcid link">http://orcid.org/0000-0001-9569-4487</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Franziska Preuss</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Franziska Preuss</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Goethe University Frankfurt, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</div><div class="loa-info-affiliations-info">Structural Genomics Consortium (SGC), Buchmann Institute for Molecular Life Sciences (BMLS), Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Franziska++Preuss">Franziska Preuss</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiaomin Ni</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaomin Ni</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Goethe University Frankfurt, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</div><div class="loa-info-affiliations-info">Structural Genomics Consortium (SGC), Buchmann Institute for Molecular Life Sciences (BMLS), Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaomin++Ni">Xiaomin Ni</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andreas Krämer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andreas Krämer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Goethe University Frankfurt, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</div><div class="loa-info-affiliations-info">Structural Genomics Consortium (SGC), Buchmann Institute for Molecular Life Sciences (BMLS), Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</div><div class="loa-info-affiliations-info">Frankfurt Cancer Institute (FCI), Paul-Ehrlich-Str. 42-44, 60596 Frankfurt am Main, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andreas++Kr%C3%A4mer">Andreas Krämer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sebastian Mathea</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sebastian Mathea</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Goethe University Frankfurt, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</div><div class="loa-info-affiliations-info">Structural Genomics Consortium (SGC), Buchmann Institute for Molecular Life Sciences (BMLS), Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sebastian++Mathea">Sebastian Mathea</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8500-4569" title="Orcid link">http://orcid.org/0000-0001-8500-4569</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tamara Göbel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tamara Göbel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Goethe University Frankfurt, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tamara++G%C3%B6bel">Tamara Göbel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David Heidenreich</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David Heidenreich</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Goethe University Frankfurt, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</div><div class="loa-info-affiliations-info">Structural Genomics Consortium (SGC), Buchmann Institute for Molecular Life Sciences (BMLS), Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David++Heidenreich">David Heidenreich</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Svenja Simonyi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Svenja Simonyi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Goethe University Frankfurt, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Svenja++Simonyi">Svenja Simonyi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Astrid S. Kahnt</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Astrid S. Kahnt</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Goethe University Frankfurt, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Astrid+S.++Kahnt">Astrid S. Kahnt</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andreas C. Joerger</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andreas C. Joerger</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Goethe University Frankfurt, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</div><div class="loa-info-affiliations-info">Structural Genomics Consortium (SGC), Buchmann Institute for Molecular Life Sciences (BMLS), Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</div><div class="loa-info-affiliations-info">German Translational Cancer Network (DKTK), Frankfurt/Mainz Site, 60438 Frankfurt am Main, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#167c79736471736456667e77647b757e737b3863787f3b706477787d70636462387273"><span class="__cf_email__" data-cfemail="62080d071005071022120a03100f010a070f4c170c0b4f0410030c09041710164c0607">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andreas+C.++Joerger">Andreas C. Joerger</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1232-0138" title="Orcid link">http://orcid.org/0000-0002-1232-0138</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Stefan Knapp</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stefan Knapp</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Goethe University Frankfurt, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</div><div class="loa-info-affiliations-info">Structural Genomics Consortium (SGC), Buchmann Institute for Molecular Life Sciences (BMLS), Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</div><div class="loa-info-affiliations-info">German Translational Cancer Network (DKTK), Frankfurt/Mainz Site, 60438 Frankfurt am Main, Germany</div><div class="loa-info-affiliations-info">Frankfurt Cancer Institute (FCI), Paul-Ehrlich-Str. 42-44, 60596 Frankfurt am Main, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#3f54515e4f4f7f4f575e4d525c575a52114a515612594d5e5154594a4d4b115b5a"><span class="__cf_email__" data-cfemail="94fffaf5e4e4d4e4fcf5e6f9f7fcf1f9bae1fafdb9f2e6f5fafff2e1e6e0baf0f1">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stefan++Knapp">Stefan Knapp</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5995-6494" title="Orcid link">http://orcid.org/0000-0001-5995-6494</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01242&amp;href=/doi/10.1021%2Facs.jmedchem.0c01242" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 23</span><span class="cit-pageRange">, 14680–14699</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 20, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>16 July 2020</li><li><span class="item_label"><b>Published</b> online</span>20 November 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 December 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01242" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01242</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D14680%26pageCount%3D20%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMarek%2BWanior%252C%2BFranziska%2BPreuss%252C%2BXiaomin%2BNi%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D23%26contentID%3Dacs.jmedchem.0c01242%26title%3DPan-SMARCA%252FPB1%2BBromodomain%2BInhibitors%2Band%2BTheir%2BRole%2Bin%2BRegulating%2BAdipogenesis%26numPages%3D20%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D14699%26publicationDate%3DDecember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01242"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1160</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">1</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01242" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Pan-SMARCA/PB1 Bromodomain Inhibitors and Their Role in Regulating Adipogenesis&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Marek&quot;,&quot;last_name&quot;:&quot;Wanior&quot;},{&quot;first_name&quot;:&quot;Franziska&quot;,&quot;last_name&quot;:&quot;Preuss&quot;},{&quot;first_name&quot;:&quot;Xiaomin&quot;,&quot;last_name&quot;:&quot;Ni&quot;},{&quot;first_name&quot;:&quot;Andreas&quot;,&quot;last_name&quot;:&quot;Krämer&quot;},{&quot;first_name&quot;:&quot;Sebastian&quot;,&quot;last_name&quot;:&quot;Mathea&quot;},{&quot;first_name&quot;:&quot;Tamara&quot;,&quot;last_name&quot;:&quot;Göbel&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Heidenreich&quot;},{&quot;first_name&quot;:&quot;Svenja&quot;,&quot;last_name&quot;:&quot;Simonyi&quot;},{&quot;first_name&quot;:&quot;Astrid&quot;,&quot;last_name&quot;:&quot;S. Kahnt&quot;},{&quot;first_name&quot;:&quot;Andreas&quot;,&quot;last_name&quot;:&quot;C. Joerger&quot;},{&quot;first_name&quot;:&quot;Stefan&quot;,&quot;last_name&quot;:&quot;Knapp&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;20&quot;,&quot;issue&quot;:&quot;23&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;14680-14699&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01242&quot;},&quot;abstract&quot;:&quot;Accessibility of the human genome is modulated by the ATP-driven SWI/SNF chromatin remodeling multiprotein complexes BAF (BRG1/BRM-associated factor) and PBAF (polybromo-associated BAF factor), which involves reading of acetylated histone tails by the bromodomain-containing proteins SMARCA2 (BRM), SMARCA4 (BRG1), and polybromo-1. Dysregulation of chromatin remodeling leads to aberrant cell proliferation and differentiation. Here, we have characterized a set of potent and cell-active bromodomain inhibitors with pan-selectivity for canonical family VIII bromodomains. Targeted SWI/SNF bromodomain inhibition blocked the expression of key genes during adipogenesis, including the transcription factors PPARγ and C/EBPα, and impaired the differentiation of 3T3-L1 murine fibroblasts into adipocytes. Our data highlight the role of SWI/SNF bromodomains in adipogenesis and provide a framework for the development of SWI/SNF bromodomain inhibitors for indirect targeting of key transcription factors regulating cell differen&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01242&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01242" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01242&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01242" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01242&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01242" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01242&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01242&amp;href=/doi/10.1021/acs.jmedchem.0c01242" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01242" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01242" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01242&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01242%26sid%3Dliteratum%253Aachs%26pmid%3D33216538%26genre%3Darticle%26aulast%3DWanior%26date%3D2020%26atitle%3DPan-SMARCA%252FPB1%2BBromodomain%2BInhibitors%2Band%2BTheir%2BRole%2Bin%2BRegulating%2BAdipogenesis%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D23%26spage%3D14680%26epage%3D14699%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291881" title="Differentiation">Differentiation</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/23" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/jmcmar.2020.63.issue-23/20201210/jmcmar.2020.63.issue-23.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/medium/jm0c01242_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/large/jm0c01242_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01242&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Accessibility of the human genome is modulated by the ATP-driven SWI/SNF chromatin remodeling multiprotein complexes BAF (BRG1/BRM-associated factor) and PBAF (polybromo-associated BAF factor), which involves reading of acetylated histone tails by the bromodomain-containing proteins SMARCA2 (BRM), SMARCA4 (BRG1), and polybromo-1. Dysregulation of chromatin remodeling leads to aberrant cell proliferation and differentiation. Here, we have characterized a set of potent and cell-active bromodomain inhibitors with pan-selectivity for canonical family VIII bromodomains. Targeted SWI/SNF bromodomain inhibition blocked the expression of key genes during adipogenesis, including the transcription factors PPARγ and C/EBPα, and impaired the differentiation of 3T3-L1 murine fibroblasts into adipocytes. Our data highlight the role of SWI/SNF bromodomains in adipogenesis and provide a framework for the development of SWI/SNF bromodomain inhibitors for indirect targeting of key transcription factors regulating cell differentiation.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56501" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56501" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Access to the genetic information is primarily initiated through chromatin remodeling and epigenetic mechanisms. Errors in these processes lead to aberrant cell proliferation and abnormal cell differentiation.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> Bromodomains (BRDs) are epigenetic protein–protein interaction modules involved in the regulation of gene transcription. They recognize acetylation marks or combinations of acetylated histones in addition with other epigenetic proteins. Often, multiple bromodomains are present in nuclear complexes involved in chromatin remodeling, which complicates functional studies on the epigenetic reading process. The human genome encodes 61 BRDs in 46 BRD-containing proteins, which are divided into eight families. BRDs have a highly conserved structure of ∼110 amino acids, containing four α-helices (αA, αB, αC, and αZ) that are linked by loops of varying lengths. Two of these loops (BC and ZA) form a hydrophobic pocket with a highly conserved asparagine residue that preferentially anchors acetylated lysine <i>via</i> a hydrogen bond.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> Some BRDs have unique additional secondary structure elements, such as the BRD family VIII members with two short helical segments within the ZA loop (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a).<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Because dysregulation of acetylation levels has been implicated in the development of many diseases such as cancer, autoimmune disorders, inflammation, heart failure, or viral infections,<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a> chemical probes for a large set of BRDs have been developed to enable mechanistic studies on gene transcription programs and cellular disease models. Several bromodomain inhibitors, in particular targeting BET-bromodomains, have now entered clinical testing.<a onclick="showRef(event, 'ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9">(7−9)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/medium/jm0c01242_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/large/jm0c01242_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Sequence and structure of bromodomain family VIII members. (A) Sequence alignment of human polybromo-1 and SMARCA2/4 bromodomains. The bromodomains PB1(1) and PB1(6) have been excluded because of their inaccessible acetyllysine binding sites and lack of conserved acetyllysine-binding residues.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Used were the UniProt entries Q86U86, P51531 (short isoform 2), and P51532. Secondary structure elements are shown above the alignment, based on the crystals structure of PB1(2) (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3LJW">3LJW</a>). Key interacting residues in the ligand-binding pocket are highlighted. (B) Cartoon representation of the structure of the SMARCA4 BRD in complex with inhibitor PFI-3 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKD">5DKD</a>). The inhibitor is shown as a yellow stick model. Residues that are strictly conserved in the alignment above are highlighted in green. (C) Close-up view of the binding mode of PFI-3 in SMARCA4. Selected side chains interacting with the inhibitor are shown as stick models. A conserved structural water molecule at the bottom of the binding pocket is highlighted as a red sphere. (D) Chemical structures of published SMARCA2/4 and PB1(5) bromodomain inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/large/jm0c01242_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01242&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">ATP-dependent switch/sucrose nonfermentable (SWI/SNF) remodeling complexes bind genome-wide to approximately 20,000–40,000 sites<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> by reading specific histone marks, leading to activation or repression of gene transcription.<a onclick="showRef(event, 'ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13">(11−13)</a> They therefore play a crucial role in cell proliferation, cell differentiation and other biological processes such as muscle<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> or heart development.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> The human SWI/SNF complexes BAF (BRG1/BRM-associated factor) and PBAF (polybromo-associated BAF factor) can be distinguished based on their subunit composition.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> The core subunits BRG1/BRM contain an ATPase domain, a helicase domain as well as two bromodomains, SMARCA2/4 (SWI/SNF-related, matrix-associated actin-dependent regulator of chromatin, subfamily A2/4). The role of BAF as a tumor suppressor has been widely demonstrated. Loss-of-function mutations in these complexes are strong drivers of tumorigenesis, and mutations in BAF components are among the most frequently observed genetic alterations in cancer.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> For example, development of cancer is promoted through an error to counteract polycomb repressive complexes and to remove the transcriptional silencing mark H3K27me3.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></div><div class="NLM_p">A particularly interesting PBAF component is polybromo-1 (PB1, PBRM1), which contains six distinct tandem-acting BRDs, PB1(1–6), for which no homolog is found in BAF.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> PB1 is an important tumor suppressor, which is mutated in about 40% of clear cell renal carcinoma, making it the second most frequently mutated gene in kidney cancer after the Von-Hippel–Lindau tumor suppressor.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> The exact role of the six tandem PB1 BRDs in PBAF, a unique feature among BRD-containing proteins, is largely unexplored, though.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18,19)</a> Histone peptide arrays have shown that PB1(2) interacts strongly with H3K4me3 and H3K14ac, hallmarks of transcriptionally active chromatin.<a onclick="showRef(event, 'ref4 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref4 ref20 ref21">(4,20,21)</a> More recently, three independent studies confirmed that PB1 binding is associated with a strong interaction of PB1(2) with H3K14ac, but also shed some light on the role of the other BRDs, for example, showing that PB1(2), PB1(4) and PB1(5) work synergistically to facilitate high affinity binding to H3K14ac and tether PB1 to chromatin.<a onclick="showRef(event, 'ref18 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref18 ref22 ref23">(18,22,23)</a> The bromodomains PB1(1) and PB1(6) lack conserved residues necessary for binding to acetylated histone tails and have largely occluded binding sites and are therefore thought not to recognize acetylated sequences.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">BAF and PBAF subunits are mutated in approximately 20% of all human tumors.<a onclick="showRef(event, 'ref17 ref24'); return false;" href="javascript:void(0);" class="ref ref17 ref24">(17,24)</a> However, an intriguing therapeutic window exists in some cases.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Tumors with a SMARCA4-deficiency, for example, are vulnerable to SMARCA2 depletion,<a onclick="showRef(event, 'ref26 ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28 ref29">(26−29)</a> or a BRD9-dependency was reported for SMARCB1-mutant cells.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Inhibition of BAF/PBAF bromodomains with small molecules, such as the BRD7/9 bromodomain inhibitors BI-7273/BI-9564 or the SMARCA2/4 and PB1(5) bromodomain inhibitor PFI-3, had no significant effect on cell viability,<a onclick="showRef(event, 'ref31 ref32 ref33 ref34 ref35 ref36 ref37 ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref31 ref32 ref33 ref34 ref35 ref36 ref37 ref38 ref39">(31−39)</a> whereas degradation of the complex with proteolysis targeting chimeras (PROTACs) based on BRD inhibitors has very recently been shown to efficiently kill SMARCA4- or SMARCB1-deficient cancer cells but also nonmutated cells.<a onclick="showRef(event, 'ref40 ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref40 ref41 ref42">(40−42)</a> However, despite the lack of antiproliferative effects, PFI-3 impairs maintenance and differentiation of embryonic and trophoblast stem cells.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Furthermore, it has been shown that knockdown of SMARCA4 or other BAF subunits influences self-renewal and pluripotency of embryonic stem cells and other lineage specifications.<a onclick="showRef(event, 'ref44 ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref44 ref45 ref46">(44−46)</a> Thus, these data indicate that SWI/SNF bromodomains play a pivotal role in regulating cell differentiation.</div><div class="NLM_p">Here, we were interested in the role of SWI/SNF bromodomains in adipogenesis, a process that depends on the transcription factors peroxisome proliferator-activated receptor γ (PPARγ) and CCAAT/enhancer binding protein α (C/EBPα) as well as the activity of chromatin-modifying enzymes.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> During adipogenesis, the SWI/SNF complex interacts with the C/EBPα transactivation element III domain of C/EBPα<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> and forms a preinitiation complex with PPARγ.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> Interestingly, genome-wide analyses revealed a high enrichment of SWI/SNF complexes on acetylated histone marks, H3K27ac and H3K9ac, in the vicinity of PPARγ binding sites during adipogenesis.<a onclick="showRef(event, 'ref50 ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref50 ref51 ref52">(50−52)</a> Moreover, SWI/SNF was recently shown to be enriched on specific enhancer sites that are important for lineage specification, and to interact synergistically with p300 to read H3K27ac sites.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> These data suggest that adipocyte differentiation relies on a highly sensitive crosstalk between PPARγ, C/EBPα and SWI/SNF.</div><div class="NLM_p last">Because of the instability of PFI-3 and its narrow activity spectrum on PB1 bromodomains, we characterized a set of potent SMARCA2/4 and PB1(5) bromodomain inhibitors based on a 2-(6-aminopyridazin-3-yl)phenol scaffold. Through structure-guided design, this series was further developed into a bromodomain inhibitor with broader specificity, additionally targeting the BRD family VIII members PB1(2), PB1(3), and PB1(4). Overall, those SWI/SNF bromodomain inhibitors showed intriguing cellular activities by blocking differentiation of 3T3-L1 murine fibroblasts into adipocytes. Our data highlight the potential of SWI/SNF bromodomain inhibition to modulate cell differentiation and provide mechanistic insights into the molecular basis of adipogenesis.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52043" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52043" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Characterization of a Selective and Potent Inhibitor for BRD Family VIII</h3><div class="NLM_p">The identification of novel chemical probes remains a priority task in pharmaceutical industry and academia. Chemical probes are selective and potent small-molecule modulators of a particular biological target, with a clearly defined mode of action, and are particularly useful for studying functional roles and molecular mechanisms of a specific target protein.<a onclick="showRef(event, 'ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref54 ref55">(54,55)</a> The first chemical probe for the BRDs of SMARCA2/4 and PB1(5) was based on the expansion of an acetyllysine mimetic salicylic acid warhead, resulting in compound PFI-3 (<b>1</b>) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>d).<a onclick="showRef(event, 'ref31 ref43'); return false;" href="javascript:void(0);" class="ref ref31 ref43">(31,43)</a> The unique binding mode causing the displacement of structural water molecules that are usually tightly bound in other bromodomains provided excellent selectivity of this chemical probe.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> However, the instability of the prop-2-en-1-one motif linked to a ternary amine limited applications of this inhibitor in cellular systems that require long-term exposure. Several other inhibitors that used fused ring systems in order to improve stability such as quinazolinones<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> (<b>2</b>), isochromenopyrazole<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> (<b>3</b>) or compound DCSM06-05<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> (<b>4</b>) lacked potency and had limiting solubility. A recent breakthrough in developing potent and stable SMARCA2/4 and PB1(5) bromodomain inhibitors was the discovery of the 2-(6-aminopyridazin-3-yl)phenol scaffold (<b>5</b>) published by Genentech/Constellation Pharmaceuticals in the patent literature (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>d).<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> However, the authors only presented IC<sub>50</sub> values based on TR-FRET measurements without further characterization of their inhibitors. One of those inhibitors has recently been used as a starting point for the development of PROTAC ACBI1, a chemical tool to degrade BAF/PBAF <i>via</i> their SMARCA2/4 and PB1 bromodomains.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a></div><div class="NLM_p">To characterize this inhibitor class and to further develop them into inhibitors with an enhanced activity for a broader set of PB1 bromodomains, we resynthesized a selection of these compounds following the published protocol.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> The general synthesis route is outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Nucleophilic aromatic substitution of 4-bromo-6-chloropyridazin-3-amine <b>6</b> with the corresponding secondary amine yielded the intermediates <b>8–10</b>, which were then converted with (2-hydroxyphenyl)boronic acid <b>7</b> or <b>7a</b> to the inhibitors <b>22–24</b> and <b>35</b> using a palladium-catalyzed Suzuki–Miyaura cross-coupling reaction.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/medium/jm0c01242_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/large/jm0c01242_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Synthesis Route of 2-(6-Aminopyridazin-3-yl)phenol Derivatives<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/large/jm0c01242_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01242&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (I) DIEA, MeCN, reflux; (II) XPhos Pd G3, K<sub>2</sub>CO<sub>3</sub>, MeCN/H<sub>2</sub>O, microwave, 120 °C; (Ia) <i>N</i>-methylation of compounds <i>R</i>-<b>30</b> and <i>S</i>-<b>31</b> was performed with NaH, MeI, THF, 0–40 °C prior to step (I). Compound <b>34</b> was prepared <i>via</i> intermediate <b>33</b>. For final compounds <b>22–32</b> and <b>34–39</b>, see <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>. Additional compound characterization data are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_001.csv" class="ext-link">Supporting Information</a>.</p></p></figure><div class="NLM_p">The synthesized inhibitors <b>22–24</b> and the negative control <b>35</b> were subsequently screened for selectivity within BRD subfamily VIII by differential scanning fluorimetry (DSF, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_002.pdf" class="ext-link">Supporting Information</a> Figure S1a–c), which measures the stabilization of the protein upon ligand binding.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> Inhibitor <b>22</b> showed a high selectivity for SMARCA2/4 and PB1(5) and resulted in moderate thermal stabilization (Δ<i>T</i><sub>m</sub>) of PB1(2,3), but no significant thermostabilization when screened against a set of 16 representative members from the other seven BRD families (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_002.pdf" class="ext-link">Supporting Information</a> Table S1). A number of clinical kinase inhibitors have been shown to also inhibit bromodomains with therapeutically relevant potencies.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> We therefore screened inhibitor <b>22</b> against a representative panel of 85 protein kinases. No significant thermostabilization was observed in that screen, excluding those kinases as potential off targets (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_002.pdf" class="ext-link">Table S2</a>). Binding constants for <b>22</b> were determined by isothermal titration calorimetry (ITC), which gave dissociation constants (<i>K</i><sub>d</sub>) of 13, 35, and 36 nM for PB1(5), SMARCA2, and SMARCA4 as well as <i>K</i><sub>d</sub>s of 3.7 and 2.0 μM for PB1(2) and PB1(3), respectively (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Removal of the <i>tert</i>-butyl ester moiety in <b>23</b> reduced the ability to bind SMARCA2/4 and PB1(5) by a factor of 2 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>).</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/medium/jm0c01242_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/large/jm0c01242_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Bromodomain family VIII binders. (A) Heat map of the synthesized inhibitors screened against the acetyllysine-binding members of bromodomain family VIII using thermal shift assays. (B) Phylogenetic tree of bromodomains and subdivision into eight subgroups. Thermal shifts, Δ<i>T</i><sub>m</sub>, upon binding of <b>22</b> to different BRD family members are indicated. (C) ITC data for compound <b>22</b>. Corrected heat rates of reverse titration of SMARCA4 and ligand <b>22</b> are displayed (top). Integrated ligand binding heats were fitted by assuming a sigmoidal dose–response relationship (four parameters) (bottom). (D) Cocrystal structures of <b>23</b> with PB1(5) (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZS3">6ZS3</a>) and (E) <b>22</b> with PB1(5) (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZS4">6ZS4</a>), highlighting the interacting amino acids and the remaining conserved water upon ligand binding. 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps for the ligands are shown at contour levels of 1.5 σ (D) and 1.8 σ (E). Hydrogen bonds are shown as dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/large/jm0c01242_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01242&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Thermal Stabilization of BRD Family VIII Members upon Inhibitor Binding</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/medium/jm0c01242_0007.gif" alt="" id="gr6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Thermal shift Δ<i>T</i><sub>m</sub> values given are the mean of triplicate measurements. The standard error of the mean was ≤±0.5 °C for all measurements. DSF assays were performed at 5 μM protein concentration and a final compound concentration of 20 μM.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Binding Affinities of BRD Family VIII Inhibitors</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="center" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="6" align="center" char="±"><i>K</i><sub>d</sub> [nM]<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="."> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char="±">PB1(2)</th><th class="colsep0 rowsep0" align="center" char="±">PB1(3)</th><th class="colsep0 rowsep0" align="center">PB1(4)</th><th class="colsep0 rowsep0" align="center" char="±">PB1(5)</th><th class="colsep0 rowsep0" align="center" char="±">SMARCA2</th><th class="colsep0 rowsep0" align="center" char="±">SMARCA4</th><th class="colsep0 rowsep0" align="center" char=".">log <i>P</i><sub>o/w</sub><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>22</b></td><td class="colsep0 rowsep0" align="char" char="±">3655 ± 810</td><td class="colsep0 rowsep0" align="char" char="±">1963 ± 524</td><td class="colsep0 rowsep0" align="center">>10 μM</td><td class="colsep0 rowsep0" align="char" char="±">13 ± 2</td><td class="colsep0 rowsep0" align="char" char="±">35 ± 8</td><td class="colsep0 rowsep0" align="char" char="±">36 ± 4</td><td class="colsep0 rowsep0" align="char" char=".">1.82</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>23</b></td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="char" char="±">26 ± 7</td><td class="colsep0 rowsep0" align="char" char="±">135 ± 15</td><td class="colsep0 rowsep0" align="char" char="±">73 ± 16</td><td class="colsep0 rowsep0" align="char" char=".">0.74</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><i>R</i>-<b>26</b></td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">911 ± 155</td><td class="colsep0 rowsep0" align="center">2940 ± 374</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char=".">2.90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><i>S</i>-<b>27</b></td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">1867 ± 282</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char=".">2.77</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><i>R</i>-<b>28</b></td><td class="colsep0 rowsep0" align="char" char="±">252 ± 26</td><td class="colsep0 rowsep0" align="char" char="±">328 ± 47</td><td class="colsep0 rowsep0" align="center">3110 ± 440</td><td class="colsep0 rowsep0" align="char" char="±">238 ± 53</td><td class="colsep0 rowsep0" align="char" char="±">71 ± 10</td><td class="colsep0 rowsep0" align="char" char="±">100 ± 17</td><td class="colsep0 rowsep0" align="char" char=".">1.17</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>32</b></td><td class="colsep0 rowsep0" align="char" char="±">464 ± 44</td><td class="colsep0 rowsep0" align="char" char="±">448 ± 72</td><td class="colsep0 rowsep0" align="center">4157 ± 493</td><td class="colsep0 rowsep0" align="char" char="±">64 ± 10</td><td class="colsep0 rowsep0" align="char" char="±">69 ± 10</td><td class="colsep0 rowsep0" align="char" char="±">61 ± 16</td><td class="colsep0 rowsep0" align="char" char=".">1.14</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><i>S</i>,<i>S</i>-<b>37</b></td><td class="colsep0 rowsep0" align="char" char="±">892 ± 86</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char=".">0.30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><i>S</i>,<i>R</i>-<b>39</b></td><td class="colsep0 rowsep0" align="char" char="±">420 ± 45</td><td class="colsep0 rowsep0" align="char" char="±">840 ± 133</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char=".">0.26</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last"><i>K</i><sub>d</sub> values were derived from ITC measurements and calculated by assuming a sigmoidal dose–response relationship (four parameters). The errors of the fits were calculated using standard deviation and a confidence interval of 95%.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Log <i>P</i><sub>o/w</sub> values were predicted using SwissADME<a onclick="showRef(event, 'ref87'); return false;" href="javascript:void(0);" class="ref ref87">(87)</a> and are given as the mean of five independent algorithms.</p></div></div></div><div class="NLM_p last">To elucidate the binding mode of the compounds, we determined cocrystal structures of PB1(5) with <b>22</b> and <b>23</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>d,e) as well as of SMARCA4 with <b>23</b>. The overall binding mode of these ligands was reminiscent of the one observed for PFI-3 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b,c). Upon binding, the 2-(6-aminopyridazin-3-yl)phenol scaffold of the inhibitor displaces four out of five structural water molecules within the acetyl lysine binding pocket that are conserved in a multitude of BRDs.<a onclick="showRef(event, 'ref56 ref59 ref60'); return false;" href="javascript:void(0);" class="ref ref56 ref59 ref60">(56,59,60)</a> In addition to extensive hydrophobic interactions, for example, with Phe684, Leu687, Pro688, and Ile745 in PB1(5), the compounds also formed specific polar contacts. The conserved Asn739 located in the BC loop formed hydrogen bonds with two nitrogens of the amino-pyridazine. In addition, the phenolic moiety of the inhibitor formed a hydrogen bond with the side chain of Tyr696, another highly conserved residue, located on a short helical segment within the ZA loop (see sequence alignment in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a). Methylation of the phenolic hydroxyl moiety in <b>35</b> completely abolished protein–ligand binding as shown by DSF (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), making this compound a perfect negative control. The <i>tert</i>-butyl ester of compound <b>22</b> formed additional hydrophobic interactions with Leu693 at the entrance of the binding pocket of PB1(5), which explained the higher affinity of <b>22</b> compared with <b>23</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>e). Predicted octanol–water partition coefficients (log<i>P</i><sub>o/w</sub> values) of the inhibitors (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) suggested a distribution between the lipid and aqueous phases in the desired range for a drug-like molecule according to Lipinski’s rule of 5.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> Taken together, the biophysical and structural data confirmed <b>22</b> as a selective and potent inhibitor of the SMARCA2/4 and PB1(5) BRDs.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Identification of Inhibitors with Activity for PB1(2), PB1(3) and PB1(4)</h3><div class="NLM_p">In addition to binding to SMARCA2/4 and PB1(5) BRDs, most of the inhibitors also showed a moderate thermal stabilization of the second and third bromodomain of PB1 and weak activity for the fourth bromodomain (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a). This broader activity for bromodomains in PB1 can be rationalized by the sequence alignment of BRD subfamily VIII members in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a, showing that especially the binding region of acetylated H3 histone lysines in the ZA loop (including the short helical segment), the αB helix and the BC loop is highly conserved. In fact, most PB1(5) residues interacting with compound <b>23</b> within the acetyllysine binding pocket are either strictly conserved or have similar residue types in the different family members, apart from domains PB1(1) and PB1(6), which have occluded binding sites.</div><div class="NLM_p">Interestingly, it has been reported that the second, fourth and fifth bromodomain in PB1 work in tandem to recognize acetylated histones in chromatin.<a onclick="showRef(event, 'ref18 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref18 ref22 ref23">(18,22,23)</a> We therefore aimed at designing a chemical probe with pan-selectivity for the functional bromodomains in subfamily VIII. To achieve this, we introduced small decorations to the core scaffold that might potentially increase the affinity for PB1(2–4) while retaining the already high affinities for SMARCA2/4 and PB1(5). Targeting the hydroxyl of a surface-exposed tyrosine (Tyr217 in PB1(2)) on the flexible ZA loop at the entrance of the binding pocket, which is only present in the second, third and fourth PB1 domain (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a), appeared particular promising to achieve the desired binding profile.</div><div class="NLM_p">Our initial Δ<i>T</i><sub>m</sub> screens suggested the racemic inhibitor <i>rac</i>-<b>25</b>, a compound disclosed in the above-mentioned patent application by Genentech/Constellation Pharmaceuticals,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> as a good starting point for SAR studies to develop promiscuous BRD subfamily VIII inhibitors. In particular, <i>rac</i>-<b>25</b> gave Δ<i>T</i><sub>m</sub> values of 4.9 °C for PB1(2), 6.9 °C for PB1(3), and 1.6 °C for PB1(4) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Encouraged by these data, we synthesized inhibitors <b>26–32</b>, <b>34</b>, and <b>36–39</b><i>via</i> intermediates <b>11–21</b> and <b>33</b> as outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. First, the enantiomeric pure inhibitors <i>R</i>-<b>26</b> and <i>S</i>-<b>27</b> were synthesized and tested by DSF. The <i>R</i>-enantiomer <i>R</i>-<b>26</b> showed increased Δ<i>T</i><sub>m</sub>s (6.0 °C for PB1(2), 8.4 °C for PB1(3), and 2.1 °C for PB1(4)) compared with the <i>S</i>-enantiomer <i>S</i>-<b>27</b> or the racemic mixture, while retaining activity for SMARCA2/4 and PB1(5). <i>R</i>-<b>26</b> bound with a <i>K</i><sub>d</sub> of 0.9 μM for PB1(3) and 2.9 μM for PB1(4). Consistent with the DSF data, <i>S</i>-<b>27</b> showed a reduced affinity for PB1(3) (<i>K</i><sub>d</sub> = 1.9 μM), while <i>N</i>-methylation of the amino-pyridazine (<i>R</i>-<b>30</b> and <i>S</i>-<b>31</b>) completely abrogated binding to all BRDs of subfamily VIII (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). To elucidate the binding mode of <i>R</i>-<b>26</b>, we determined the crystal structure of PB1(2) in complex with the ligand (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>d). As expected, the 2-(6-aminopyridazin-3-yl)phenol core scaffold adopted the same binding mode as its parent compound <b>23</b> in PB1(5), which was characterized by extensive hydrophobic interactions and specific hydrogen bonds to the highly conserved Asn263 on the BC loop and Tyr220 on a helical segment within the ZA loop. The piperidine moiety packed against the side chain of Tyr217 at the entrance of the binding pocket, while the phenoxy moiety was solvent exposed in the crystal structure, without specific interactions with the protein. The phenoxy ether formed an intramolecular hydrogen bond with the primary amine of the ligand.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/medium/jm0c01242_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/large/jm0c01242_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Bromodomain family VIII binders. (A) Heat map of the synthesized inhibitors screened against the acetyllysine-binding members of bromodomain family VIII using thermal shift assays. (B) Phylogenetic tree of bromodomains and subdivision into 8 subgroups. Thermal shift, Δ<i>T</i><sub>m</sub>s, upon binding of <i>R</i>-<b>28</b> to representative BRD family members are indicated. (C) ITC data of compound <b>32</b>. Corrected heat rates of reverse titration of PB1(3) and ligand <b>32</b> are shown (top). Integrated ligand binding heats were fitted by assuming a sigmoidal dose–response relationship (four parameters) (bottom). (D) Cocrystal structure of the PB1(2)-<i>R</i>-<b>26</b> complex (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZN6">6ZN6</a>). A 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map for the ligand is shown at a contour level of 1.2 σ. (E) Cocrystal structure of the PB1(2)-<b>32</b> complex (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZNV">6ZNV</a>) superimposed onto the structure of ligand-free PB1(4) (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3TLP">3TLP</a>). A 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map for the ligand is shown at a contour level of 1.5 σ. Hydrogen bonds are shown as dashed lines. The numbering of residues is according to the canonical UniProt sequence Q86U86, which is different from the numbering in the actual PDB entry that is based on a different isoform.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/large/jm0c01242_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01242&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Thermal Stabilization of BRD Family VIII Members upon Inhibitor Binding</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/medium/jm0c01242_0008.gif" alt="" id="gr7" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">DSF assays were performed at a protein concentration of 5 μM and a final compound concentration of 20 μM. Δ<i>T</i><sub>m</sub> values are the mean of triplicate measurements. The standard error of the mean was ≤±0.5 °C, unless stated otherwise.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Standard error of the mean ≤±1.0 °C.</p></div></div><div></div></div><div class="NLM_p">A drawback of compound <i>R</i>-<b>26</b> was its low aqueous solubility (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), which hampered further studies. To increase the aqueous solubility of <i>R</i>-<b>26</b>, we removed the phenyl ring to yield inhibitors <i>R</i>-<b>28</b> and <i>S</i>-<b>29</b>, as this hydrophobic moiety was solvent-exposed in the crystal structure of PB1(2) in complex with <i>R</i>-<b>26</b> and did not significantly interact with the bromodomain. Consistent with the <i>R</i>-<b>26</b>/<i>S</i>-<b>27</b> pair, the <i>R</i>-enantiomer of the shortened compound, <i>R</i>-<b>28</b>, preferentially bound to PB1(2,3,4) compared with its enantiomer <i>S</i>-<b>29</b> (see DSF data in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>), with dissociation constants of 252 nM, 328 nM, and 3.1 μM for PB1(2), PB1(3), and PB1(4), respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_002.pdf" class="ext-link">Supporting Information</a> Figure S2, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_002.pdf" class="ext-link">S3</a>). As such, the affinity of <i>R</i>-<b>28</b> for PB1(3) was almost three times higher than the one of its less water-soluble precursor molecule <i>R</i>-<b>26</b>. Importantly, <i>R</i>-<b>28</b> retained high affinities for SMARCA2/4 and PB1(5), with <i>K</i><sub>d</sub> values of 71, 100 and 238 nM, respectively (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_p">Next, the piperidine ring of the inhibitor was modified to engineer a hydrogen bond with Tyr217 by introducing either a hydroxyl group (<b>32</b>) or a ketone (<b>34</b>) at position 4 (synthesized <i>via</i> intermediate <b>33</b>). The ketone-based inhibitor <b>34</b> bound more weakly to bromodomains of subfamily VIII, suggesting that the chair conformation of the piperidine was necessary for optimal binding. In contrast, compound <b>32</b> showed increased binding affinity for PB1(2,3,4) compared with <b>24</b>, indicating that Tyr217 may be targeted by this modification as anticipated. This novel interaction, which has not been exploited before, was confirmed by a crystal structure of PB1(2) in complex with <b>32</b> at high resolution (1.14 Å) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>e). Somewhat unexpectedly, this novel interaction did not further increase binding affinity for the second, third or fourth PB1 bromodomain compared with compound <i>R</i>-<b>28</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_002.pdf" class="ext-link">Supporting Information</a> Table S3).</div><div class="NLM_p">Based on these results, inhibitor decorations in <i>R</i>-<b>28</b> and <b>32</b> were merged because both inhibitors showed good to moderate affinities for PB1(2,3,4) and retained binding to PB1(5) and SMARCA2/4. As building blocks, the respective cis and trans isomers of piperidine-3,4-diol with relative configurations were used, followed by chiral purification on an <i>i</i>-amylose-1 LC column to obtain the four absolute stereoisomers <b>36–39</b> with a purity of >95% ee (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_001.csv" class="ext-link">Supporting Information</a> data chiral HPLC). The absolute stereochemistry was determined by cocrystal structures with PB1(5). <i>S,R</i>-<b>39</b> was the most potent of the four diastereomers in our biophysical assays (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>), but unfortunately none of those inhibitors improved binding affinity compared with <i>R</i>-<b>28</b> or <b>32</b>.</div><div class="NLM_p">Notably, the affinity of the pan-inhibitors for PB1(4) was on average about 10 times lower than for PB1(2) and PB1(3). This was most likely due to a unique sequence variation at the bottom of the binding pocket of PB1(4). In the complex of PB1(2) with <i>R</i>-<b>26</b> or <b>32</b>, the phenolic moiety of the inhibitor formed hydrophobic interactions with the side chain of Leu211, among others (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>d,e). This residue is conserved in the other BRDs as shown in the sequence alignment in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a, except for PB1(4), which contains a lysine at this position (Lys549) that points away from the binding pocket (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>e). This loss of hydrophobic interaction may therefore account for the observed reduction in binding affinity for PB1(4).</div><div class="NLM_p last">Taken together, our data establish <i>R</i>-<b>28</b> and <b>32</b> as pan-inhibitors of BRD subfamily VIII with moderate to good affinities for PB1(2,3,4,5) and SMARCA2/4 BRDs. Both compounds are, therefore, valuable molecules, in addition to <b>22</b> and the negative control compound <b>35</b>, for exploring SWI/SNF-related biology in normal physiology and diseases.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Effects of SMARCA/PB1 Bromodomain Inhibitors on Cell Proliferation and Cell Differentiation</h3><div class="NLM_p">The chromatin remodeling complexes BAF and PBAF influence stem cell differentiation and gene expression, and they are frequently mutated in cancer. We therefore investigated the effects of <b>22</b> and <i>R</i>-<b>28</b> on cell proliferation in two colorectal (HCT116, HT29) and two breast (MCF-7, MDA-MB-231) cancer cell lines that have shown a dependency on SMARCA2/4.<a onclick="showRef(event, 'ref62 ref63 ref64'); return false;" href="javascript:void(0);" class="ref ref62 ref63 ref64">(62−64)</a> Both inhibitors had no significant effect on metabolic activity in a cell viability assay using 2,3-bis-[(2-methoxy-4-nitro-5-sulfophenyl)-2<i>H</i>-tetrazolium-5-carboxanilide salt] (XTT) and <i>N</i>-methyl dibenzopyrazine methyl sulfate (PMS) in treatment doses of up to 50 μM after 4 days (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_002.pdf" class="ext-link">Supporting Information</a> Figure S3a–d). Compound <b>22</b> was also tested in the NCI-60 Human Cancer Cell Line Screen, again without any noticeable antiproliferative effects (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_002.pdf" class="ext-link">Supporting Information</a> Figure S4). These findings are consistent with earlier studies where PFI-3 had been profiled in twelve primary human cell-based systems without showing significant effects in a BioMAP screen run at short incubation times for detection of direct effects on gene transcription.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></div><div class="NLM_p">SWI/SNF chromatin remodeling complexes have been linked to several metabolic diseases<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> and cell differentiation processes.<a onclick="showRef(event, 'ref65 ref66'); return false;" href="javascript:void(0);" class="ref ref65 ref66">(65,66)</a> Because SWI/SNF chromatin remodelers play regulatory roles during adipocyte differentiation, we tested compounds <b>22</b>, <i>R</i>-<b>28</b>, and <b>32</b> in an adipogenesis assay using 3T3-L1 murine fibroblasts (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_002.pdf" class="ext-link">Supporting Information</a> S5). Differentiation of fibroblasts into adipocytes was induced by an activation cocktail (isobutylmethylxanthine (IBMX), insulin and dexamethasone) and addition of PPARγ agonist rosiglitazone at a final concentration of 2 μM. Intracellular lipid accumulation was measured 14 days after stimulation by oil red O staining. Using this setup, 3T3-L1 murine fibroblasts were cotreated with two concentrations of <b>22</b>, <i>R</i>-<b>28</b>, <b>32</b>, the negative control <b>35</b> and PFI-3 (1.0 and 5.0 μM). As expected, rosiglitazone alone induced adipocyte formation and led to robust lipid droplet accumulation. Cotreatment with PFI-3 impaired adipocyte differentiation only slightly at a final concentration of 5.0 μM. The modest effects of this inhibitor were probably due to its limiting long-term stability in aqueous environments because of its reactive Michael acceptor system.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> In contrast, <b>22</b>, <i>R</i>-<b>28</b>, and <b>32</b> blocked adipocyte differentiation at 1.0 μM in 3T3-L1 murine fibroblasts, while the negative control <b>35</b> did not affect lipid droplet accumulation at both concentrations tested.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/medium/jm0c01242_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/large/jm0c01242_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Effects of BRD family VIII inhibitors on adipogenesis. (A) Lipid accumulation during adipogenesis detected by staining with oil red O solution on day 14. 3T3-L1 murine fibroblasts were cotreated with rosiglitazone and SWI/SNF inhibitors during differentiation into adipocytes at indicated concentrations over 14 days. Induction of adipogenesis was performed by an activation cocktail (IBMX, insulin, and dexamethasone). Experiments were performed in four biological replicates. (B) Influence of SWI/SNF bromodomain inhibitors PFI-3 (<b>1</b>) and <b>22</b> on mRNA expression levels of adipocyte markers, PPARγ, C/EBPα, and FABP4, measured by quantitative PCR (qPCR) at different time points following the induction of differentiation. Results were analyzed with the comparative 2<sup>–ΔΔ<i>C</i><sub><i>T</i></sub></sup> method compared to agonist rosiglitazone treatment (left) or unstimulated control cells (w/o; right). Data represent the mean ± SEM of three independent biological experiments. Statistical significance of the difference in mRNA levels between <b>22</b> cotreated with rosiglitazone and rosiglitazone only treated cells (after 5 days) was calculated based on Δ<i>C</i><sub><i>T</i></sub> values using an unpaired Student <i>t</i>-test. **<i>P</i> ≤ 0.010, *<i>P</i> < 0.020, not significant (ns).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/large/jm0c01242_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01242&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> SMARCA/PB1 Bromodomain Inhibition Blocks the Expression of Key Genes during Adipogenesis</h3><div class="NLM_p">To provide further mechanistic insights into the effects of SWI/SNF bromodomain inhibition during adipogenesis, we analyzed mRNA levels of key transcriptional regulators by qPCR within the first five days after stimulation of adipogenesis (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_002.pdf" class="ext-link">Supporting Information</a> S6). The investigated target genes comprised the transcription factors PPARγ, C/EBPα, Krüppel-like factor 5 (KLF5) and early growth response protein 2 (Krox20) as well as the fatty acid-binding protein 4 (FABP4), which have all been implicated with adipogenesis.</div><div class="NLM_p">During the early stages of adipogenesis (0–1 days), transcription factors C/EBPβ and C/EBPδ are upregulated to promote preadipocytes to differentiate into adipocytes.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> This upregulation is enhanced through increased expression of KLF5<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> and Krox20.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> Then on days 1–3, C/EBPβ and C/EBPδ are usually downregulated, while C/EBPα and PPARγ are upregulated, leading to an accumulation of these late adipogenic transcription factors in matured adipocytes (3–7 days).<a onclick="showRef(event, 'ref47 ref70 ref71'); return false;" href="javascript:void(0);" class="ref ref47 ref70 ref71">(47,70,71)</a> Terminal differentiation is marked by elevated levels of insulin-regulated glucose transporter GLUT4, adiponectin, and FABP4.<a onclick="showRef(event, 'ref72 ref73 ref74'); return false;" href="javascript:void(0);" class="ref ref72 ref73 ref74">(72−74)</a></div><div class="NLM_p">The observed changes in mRNA levels upon inhibitor treatment (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b) were in line with the phenotypic changes. Expression levels of the positive regulators of adipogenesis, PPARγ and C/EBPα, and the downstream marker of matured adipocytes, FABP4, were remarkably reduced after addition of inhibitor <b>22</b> to rosiglitazone treated cells. The negative control compound <b>35</b>, did not affect the expression of these three genes, showing that the reduced expression levels with <b>22</b> were indeed due to on-target effects of the inhibitor. Compared with inhibitor PFI-3, <b>22</b> reduced the mRNA levels of PPARγ, C/EBPα, and FABP4 more efficiently even at 10 times lower doses. After 5 days, for example, FABP4 was almost completely downregulated to the level of the control culture without rosiglitazone. Interestingly, the mRNA levels of Krox20 and KLF5, positive regulators of the very early stage of adipogenesis, were not affected by the treatment (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_002.pdf" class="ext-link">Supporting Information</a> Figure S6), suggesting that expression of these transcription factors was independent of bromodomain family VIII action.</div><div class="NLM_p last">Taken together, our inhibition experiments showed that the bromodomains of SWI/SNF are directly involved in adipogenesis by regulating the expression of key adipogenic transcription factors PPARγ and C/EBPα as well as of adipogenesis marker FABP4.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00465" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00465" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Here, we have characterized a set of potent inhibitors, based on a 2-(6-aminopyridazin-3-yl)phenol scaffold, for the bromodomains SMARCA2/4 and PB1 of chromatin remodeling complexes BAF and PBAF. Inhibitor <b>22</b> was highly selective within a panel of 25 BRDs and strongly bound to SMARCA2/4 and PB1(5) with dissociation constants of 35, 36 and 13 nM, respectively. Structure-guided modification of <b>22</b> led to compounds <i>R</i>-<b>28</b> and <b>32</b> with pan-selectivity for the bromodomains of subfamily VIII with increased binding affinities for the BRDs PB1(2), PB1(3), and PB1(4). In the process, the first inhibitor-bound crystal structures of PB1(2) were determined. Intriguingly, compounds <b>22</b>, <i>R</i>-<b>28</b>, and <b>32</b> blocked adipogenesis of 3T3-L1 murine fibroblasts by reducing the mRNA levels of key markers of adipocyte differentiation, PPARγ, C/EBPα, and FABP4. The presented SWI/SNF inhibitors provide valuable tools for studying the molecular mechanisms of adipogenesis and other SWI/SNF-dependent lineage specifications.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35687" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35687" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Compound Synthesis</h3><div class="NLM_p">The structures of the synthesized compounds were verified by <sup>1</sup>H- and <sup>13</sup>C NMR, and mass spectrometry (ESI). Compound purity (254 and 280 nm >95%) was determined by analytical HPLC. All commercial chemicals and solvents were used without further purification. <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra were measured in DMSO-<i>d</i><sub>6</sub> or CDCl<sub>3</sub> on a Bruker DPX250, AV400, or AV500 spectrometer. Chemical shifts δ are reported in parts per million (ppm). Mass spectra were recorded by the mass spectrometry service team of the Goethe University using a Thermo Fisher Surveyor MSQ system (including TLC-MS for reaction control). High-resolution mass spectra were recorded on a MALDI LTQ ORBITRAP XL device from Thermo Fisher Scientific. Product purification was performed on a Reveleris Prep Flash Column Chromatography System from Grace/Büchi using prepacked silica or RP C18 columns. Product purification was also performed on an Agilent 1260 Infinity II LC System [Eclipse XDB-C18 column (7 μM, 21.2 × 250 mm)] using a gradient of water/MeCN + 0.1% TFA (98:2–5:95) over 40 min with a flow rate of 21 mL/min. Compound purity was analyzed on an Agilent 1260 Infinity II LC System [Eclipse XDB-C18 column (4.6 × 250 mm, 5 μm)] coupled to an Agilent InifinityLab LC/MSD using a gradient of water/MeCN + 0.1% TFA (98:2–5:95) over 25 min at a flow rate of 1 mL/min (see the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_001.csv" class="ext-link">Supporting Information</a> data for HPLC–MS traces of lead compounds).</div><div class="NLM_p">Final compounds <b>22–39</b> were synthesized from 4-bromo-6-chloropyridazin-3-amine <b>6</b><i>via</i> intermediates <b>8–21</b> as outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> <i>tert</i>-Butyl 4-(3-amino-6-chloro-pyridazin-4-yl)piperazine-1-carboxylate (<b>8</b>)</h4><div class="NLM_p last">General procedure A: Adapted from a patent procedure,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> 4-bromo-6-chloropyridazin-3-amine <b>6</b> (2.00 g, 9.59 mmol, 1.0 equiv) and Boc-piperazine (3.57 g, 19.2 mmol, 2.0 eq) were suspended in dry MeCN (10 mL). After adding DIEA (8.2 mL, 48 mmol, 5.0 equiv), the reaction mixture was stirred at 82 °C overnight. After complete educt conversion, the reaction mixture was cooled to room temperature and then extracted with 10 mL water and EtOAc (3 × 15 mL). The combined organic layers were washed with sat. NaHCO<sub>3</sub> (aq) (5 mL) and brine (5 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i>. The crude product was deposited onto silica and purified by flash silica column chromatography using cyclohexane/EtOAc (20–100%) as eluent to yield intermediate <b>8</b> (2.43 g, 7.74 mmol, 81%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ [ppm] 6.71 (s, 1H, pyridazine-C<i>H</i>), 5.26 (s, 2H, N<i>H</i><sub>2</sub>), 3.57 (t, <i>J</i> = 5.0 Hz, 4H, piperazine-2(C<i>H</i><sub>2</sub>)-3), 2.99 (t, <i>J</i> = 5.0 Hz, 4H, piperazine-2(C<i>H</i><sub>2</sub>)-2), 1.47 (s, 9H, <i>tert</i>-butyl-(C<i>H</i><sub>3</sub>)<sub>3</sub>); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ [ppm] 155.4, 154.6, 148.2, 141.4, 115.8, 80.5, 49.0, 28.5; MS-ESI (<i>m</i>/<i>z</i>): for [C<sub>13</sub>H<sub>20</sub>ClN<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup> calcd, 314.13 (<sup>35</sup>Cl), 316.13 (<sup>37</sup>Cl); found, 314.21.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> <i>tert</i>-Butyl 4-[3-amino-6-(2-hydroxyphenyl)pyridazin-4-yl]piperazine-1-carboxylate (<b>22</b>)</h4><div class="NLM_p last">General procedure B: Adapted from a patent procedure,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> a microwave-suitable flask was charged with <b>8</b> (850 mg, 2.71 mmol, 1.0 equiv), (2-hydroxyphenyl)boronic acid <b>7</b> (747 mg, 5.42 mmol, 2.0 equiv), K<sub>2</sub>CO<sub>3</sub> (824 mg, 5.96 mmol, 2.2 equiv), XPhos Gen. 3 (229 mg, 0.271 mmol, 0.1 equiv) and an argon-purged 1:1 mixture of 15 mL H<sub>2</sub>O/MeCN (181 mmol/L). The reaction mixture was irradiated in the microwave under dynamic conditions at 120 °C for 60 min until the complete educt was converted as indicated by TLC. After cooling to room temperature, the reaction mixture was extracted with EtOAc (3 × 10 mL). The combined organic layers were washed with sat. NaHCO<sub>3</sub> (5 mL) and brine (5 mL), dried over MgSO<sub>4</sub>, filtered, and the solvent was removed <i>in vacuo</i>. The crude product was deposited onto silica and purified by flash silica column chromatography using cyclohexane/EtOAc (20–70%) to yield the pure product <b>22</b> (760 mg, 205 mmol, 76%). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ [ppm] 13.73 (s, 1H, O<i>H</i>), 7.56 (d, <i>J</i> = 8.0 Hz, 1H, phenol-C<i>H</i>-3), 7.29 (s, 1H, pyridazine-C<i>H</i>), 7.27 (t, <i>J</i> = 8.1 Hz, 1H, phenol-C<i>H</i>-5), 7.04 (d, <i>J</i> = 7.5 Hz, 1H, phenol-C<i>H</i>-6), 6.90 (t, <i>J</i> = 8.0 Hz, 1H, phenol-C<i>H</i>-4), 4.98 (s, 2H, N<i>H</i><sub>2</sub>), 3.63 (t, <i>J</i> = 5.0 Hz, 4H, piperazine-2(C<i>H</i><sub>2</sub>)-3), 3.08 (t, <i>J</i> = 5.0 Hz, 4H, piperazine-2(C<i>H</i><sub>2</sub>)-2), 1.49 (s, 9H, <i>tert</i>-butyl (C<i>H</i><sub>3</sub>)<sub>3</sub>; <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): δ [ppm] 159.3, 155.2, 154.6, 154.3, 141.0, 131.0, 125.2, 118.8, 118.6, 117.5, 111.1, 80.5, 49.3, 28.5; HRMS-ESI (<i>m</i>/<i>z</i>): for [C<sub>19</sub>H<sub>25</sub>N<sub>5</sub>O<sub>3</sub> + H]<sup>+</sup> calcd, 372.203; found, 372.204. HPLC–UV (254 nm) = 98.9%, (280 nm) = 99.3%.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> 6-Chloro-4-morpholino-pyridazin-3-amine (<b>9</b>)</h4><div class="NLM_p last"><b>6</b> (400 mg, 1.92 mmol) was reacted with morpholine according to general procedure A. Purification was performed by flash silica column chromatography using 20–100% cyclohexane/EtOAc to yield compound <b>9</b> (214 mg, 0.997 mmol, 52%) as a bright yellow solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 6.89 (s, 1H, pyridazine-C<i>H</i>), 6.22 (s, 2H, N<i>H</i><sub>2</sub>), 3.75 (t, <i>J</i> = 5.0 Hz, 4H, morpholin-2(C<i>H</i><sub>2</sub>)-3), 2.98 (t, <i>J</i> = 5.0 Hz, 4H, morpholin-2(C<i>H</i><sub>2</sub>)-2); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 155.4, 145.8, 140.5, 114.5, 65.6, 48.6; MS-ESI (<i>m</i>/<i>z</i>): for [C<sub>8</sub>H<sub>11</sub>ClN<sub>4</sub>O + H]<sup>+</sup> calcd, 215.06 (<sup>35</sup>Cl), 217.06 (<sup>37</sup>Cl); found, 215.04, 217.05.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> 6-Chloro-4-[(<i>rac</i>)-3-phenoxy-1-piperidyl]-pyridazin-3-amine (<i>Rac</i>-<b>10</b>)</h4><div class="NLM_p last"><b>6</b> (200 mg, 0.959 mmol) was reacted with <i>rac</i>-3-phenoxypiperidine according to general procedure A. Purification was performed by flash silica column chromatography using 20–100% cyclohexane/EtOAc to yield compound <i>rac</i>-<b>10</b> (178 mg, 0.584 mmol, 61%) as a slightly yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ [ppm] 7.31 (t, <i>J</i> = 7.06 Hz, 2H, phenyl-(<i>m</i>)-C<i>H</i>), 6.99 (t, <i>J</i> = 7.3 Hz, 1H, phenyl-(<i>p</i>)-C<i>H</i>), 6.93 (d, <i>J</i> = 8.4 Hz, 2H, phenyl-(<i>o</i>)-C<i>H</i>), 6.71 (s, 1H, pyridazine-C<i>H</i>), 5.10 (s, 2H, N<i>H</i><sub>2</sub>), 4.58–4.52 (m, 1H, piperidine-C<i>H</i>-3<sub>ax/eq</sub>), 3.55 (dd, <i>J</i> = 12.4 Hz, <i>J</i> = 4.9 Hz, 1H, piperidine-C<i>H</i>-2<sub>ax</sub>), 3.16–3.11 (m, 1H, piperidine-C<i>H</i>-6<sub>ax</sub>), 3.05–3.02 (m, 2H, piperidine-C<i>H</i><sub>2</sub>-6<sub>eq</sub>/2<sub>eq</sub>), 2.11–1.96 (m, 2H, piperidine-C<i>H</i><sub>2</sub>-5<sub>ax</sub>/4<sub>ax</sub>), 1.94–1.83 (m, 1H, piperidine-C<i>H</i>-4<sub>eq</sub>), 1.77–1.68 (m, 1H, piperidine-C<i>H</i>-5<sub>eq</sub>); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ [ppm] 156.9, 155.2, 148.0, 141.4, 129.9, 121.7, 116.1, 115.4, 71.1, 52.8, 49.6, 28.0, 21.8; MS-ESI (<i>m</i>/<i>z</i>): for [C<sub>15</sub>H<sub>17</sub>ClN<sub>4</sub>O + H]<sup>+</sup> calcd, 305.11 (<sup>35</sup>Cl), 307.11 (<sup>37</sup>Cl); found, 305.16, 307.17.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> 6-Chloro-4-[(3<i>R</i>)-3-phenoxy-1-piperidyl]-pyridazin-3-amine (<i>R</i>-<b>11</b>)</h4><div class="NLM_p last"><b>6</b> (100 mg, 0.480 mmol) was reacted with (3<i>R</i>)-3-phenoxypiperidine (1.4 equiv) according to general procedure A. Purification was performed by flash silica column chromatography using 10–70% cyclohexane/EtOAc to yield compound <i>R</i>-<b>11</b> (71 mg, 0.233 mmol, 49%) as a slightly yellow solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ [ppm] 7.30 (dd, <i>J</i> = 8.6 Hz, <i>J</i> = 7.4 Hz, 2H, phenyl-(<i>m</i>)-C<i>H</i>), 6.99 (t, <i>J</i> = 7.4 Hz, 1H, phenyl-(<i>p</i>)-C<i>H</i>), 6.93 (d, <i>J</i> = 7.8 Hz, 2H, phenyl-(<i>o</i>)-C<i>H</i>), 6.70 (s, 1H, pyridazine-C<i>H</i>), 5.10 (s, 2H, N<i>H</i><sub>2</sub>), 4.61–4.53 (m, 1H, piperidine-C<i>H</i>-3<sub>ax</sub>), 3.54 (dd, <i>J</i> = 12.3 Hz, <i>J</i> = 4.6 Hz, 1H, piperidine-C<i>H</i>-2<sub>ax</sub>), 3.23–3.09 (m, 1H, piperidine-C<i>H</i>-6<sub>ax</sub>), 3.09–2.88 (m, 2H, piperidine-C<i>H</i><sub>2</sub>-6<sub>eq</sub>/2<sub>eq</sub>), 2.10–1.95 (m, 2H, piperidine-C<i>H</i><sub>2</sub>-5<sub>ax</sub>/4<sub>ax</sub>), 1.93–1.85 (m, 1H, piperidine-C<i>H</i>-4<sub>eq</sub>), 1.79–1.61 (m, 1H, piperidine-C<i>H</i>-5<sub>eq</sub>); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): δ [ppm] 156.9, 155.2, 148.0, 141.4, 129.8, 121.6, 116.0, 115.4, 71.0, 52.8, 49.6, 28.0, 21.8; MS-ESI (<i>m</i>/<i>z</i>): for [C<sub>15</sub>H<sub>17</sub>ClN<sub>4</sub>O + H]<sup>+</sup> calcd, 305.11 (<sup>35</sup>Cl), 307.11 (<sup>37</sup>Cl); found, 305.09, 307.10.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> 6-Chloro-4-[(3<i>S</i>)-3-phenoxy-1-piperidyl]-pyridazin-3-amine (<i>S</i>-<b>12</b>)</h4><div class="NLM_p last"><b>6</b> (120 mg, 0.576 mmol) was reacted with (3<i>S</i>)-3-phenoxypiperidine (1.4 equiv) according to general procedure A. Purification was performed by flash silica column chromatography using 10–100% cyclohexane/EtOAc to yield compound <i>S</i>-<b>12</b> (103 mg, 0.380 mmol, 59%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ [ppm] 7.30 (dd, <i>J</i> = 8.7 Hz, <i>J</i> = 7.4 Hz, 2H, phenyl-(<i>m</i>)-C<i>H</i>), 6.98 (t, <i>J</i> = 7.4 Hz, 1H, phenyl-(<i>p</i>)-C<i>H</i>), 6.93 (d, <i>J</i> = 7.8 Hz, 2H, phenyl-(<i>o</i>)-C<i>H</i>), 6.70 (s, 1H, pyridazine-C<i>H</i>), 5.14 (s, 2H, N<i>H</i><sub>2</sub>), 4.62–4.47 (m, 1H, piperidine-C<i>H</i>-3<sub>eq</sub>), 3.54 (dd, <i>J</i> = 12.3 Hz, <i>J</i> = 4.8 Hz, 1H, piperidine-C<i>H</i>-2<sub>ax</sub>), 3.28–3.08 (m, 1H, piperidine-C<i>H</i>-6<sub>ax</sub>), 3.08–2.88 (m, 2H, piperidine-C<i>H</i><sub>2</sub>-6<sub>eq</sub>/2<sub>eq</sub>), 2.13–1.94 (m, 2H, piperidine-C<i>H</i><sub>2</sub>-5<sub>ax</sub>/4<sub>ax</sub>), 1.93–1.84 (m, 1H, piperidine-C<i>H</i>-4<sub>eq</sub>), 1.77–1.65 (m, 1H, piperidine-C<i>H</i>-5<sub>eq</sub>); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): δ [ppm] 156.9, 155.2, 148.0, 141.4, 129.8, 121.6, 116.0, 115.4, 71.0, 52.8, 49.6, 28.0, 21.8; MS-ESI (<i>m</i>/<i>z</i>): for [C<sub>15</sub>H<sub>17</sub>ClN<sub>4</sub>O + H]<sup>+</sup> calcd, 305.11 (<sup>35</sup>Cl), 307.11 (<sup>37</sup>Cl); found, 305.09, 307.10.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> (3<i>R</i>)-1-(3-Amino-6-chloro-pyridazin-4-yl)piperidin-3-ol (<i>R</i>-<b>13</b>)</h4><div class="NLM_p last"><b>6</b> (417 mg, 2.00 mmol) was reacted with (3<i>R</i>)-piperidin-3-ol hydrochloride according to general procedure A. Purification was performed by flash silica column chromatography using 1–10% DCM/MeOH to yield compound <i>R</i>-<b>13</b> (108 mg, 0.472 mmol, 23%) as a white solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 6.84 (s, 1H, pyridazine-C<i>H</i>), 6.09 (s, 2H, N<i>H</i><sub>2</sub>), 4.93 (d, <i>J</i> = 5.1 Hz, 1H, O<i>H</i>), 3.78–3.72 (m, 1H, piperidine-C<i>H</i>-3<sub>ax</sub>), 3.13 (dd, <i>J</i> = 11.7, 2.5 Hz, 1H, piperidine-C<i>H</i>-2<sub>ax</sub>), 3.10–3.01 (m, 1H, piperidine-C<i>H</i>-6<sub>ax</sub>), 2.75 (t, <i>J</i> = 9.2 Hz, 1H, piperidine-C<i>H</i>-6<sub>eq</sub>), 2.67 (dd, <i>J</i> = 11.6, 7.5 Hz, 1H, piperidine-C<i>H</i>-2<sub>eq</sub>), 1.87–1.71 (m, 2H, piperidine-C<i>H</i><sub>2</sub>-5<sub>ax</sub>/4<sub>ax</sub>), 1.65–1.53 (m, 1H, piperidine-C<i>H</i>-4<sub>eq</sub>), 1.47–1.32 (m, 1H, piperidine-C<i>H</i>-5<sub>eq</sub>); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 155.6, 145.9, 141.2, 114.3, 64.5, 55.2, 48.7, 32.1, 21.8; MS-ESI (<i>m</i>/<i>z</i>): for [C<sub>9</sub>H<sub>13</sub>ClN<sub>4</sub>O + H]<sup>+</sup> calcd, 229.08 (<sup>35</sup>Cl), 231.07 (<sup>37</sup>Cl); found, 229.03, 231.05.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> (3<i>S</i>)-1-(3-Amino-6-chloro-pyridazin-4-yl)piperidin-3-ol (<i>S</i>-<b>14</b>)</h4><div class="NLM_p last"><b>6</b> (379 mg, 1.82 mmol) was reacted with (3<i>S</i>)-piperidin-3-ol hydrochloride according to general procedure A. Purification was performed by flash silica column chromatography using 1–10% DCM/MeOH to yield compound <i>S</i>-<b>14</b> (244 mg, 1.07 mmol, 59%) as a white solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 6.84 (s, 1H, pyridazine-C<i>H</i>), 6.09 (s, 2H, N<i>H</i><sub>2</sub>), 4.93 (d, <i>J</i> = 5.1 Hz, 1H, O<i>H</i>), 3.78–3.72 (m, 1H, piperidine-C<i>H</i>-3<sub>eq</sub>), 3.13 (dd, <i>J</i> = 11.7, 2.6 Hz, 1H, piperidine-C<i>H</i>-2<sub>ax</sub>), 3.10–3.01 (m, 1H, piperidine-C<i>H</i>-6<sub>ax</sub>), 2.74 (t, <i>J</i> = 9.2 Hz, 1H, piperidine-C<i>H</i>-6<sub>eq</sub>), 2.67 (dd, <i>J</i> = 11.6, 7.5 Hz, 1H, piperidine-C<i>H</i>-2<sub>eq</sub>), 1.87–1.71 (m, 2H, piperidine-C<i>H</i><sub>2</sub>-5<sub>ax</sub>/4<sub>ax</sub>), 1.65–1.53 (m, 1H, piperidine-C<i>H</i>-4<sub>eq</sub>), 1.47–1.32 (m, 1H, piperidine-C<i>H</i>-5<sub>eq</sub>); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 155.6, 145.9, 141.2, 114.3, 64.5, 55.2, 48.7, 32.11, 21.85; MS-ESI (<i>m</i>/<i>z</i>): for [C<sub>9</sub>H<sub>13</sub>ClN<sub>4</sub>O + H]<sup>+</sup> calcd, 229.08 (<sup>35</sup>Cl), 231.07 (<sup>37</sup>Cl); found, 229.03, 231.05.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 6-Chloro-4-(1,4-dioxa-8-azaspiro[4.5]decan-8-yl)pyridazine-3-amine (<b>15</b>)</h4><div class="NLM_p last"><b>6</b> (500 mg, 2.40 mmol) was reacted with 1,4-dioxa-8-azaspiro[4.5]decane according to general procedure A. Purification was performed by flash silica column chromatography using 20–100% cyclohexane/EtOAc to yield compound <b>15</b> (468 mg, 1.73 mmol, 72%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ [ppm] 6.72 (s, 1H, pyridazine-C<i>H</i>), 5.27 (s, 2H, N<i>H</i><sub>2</sub>), 3.98 (s, 4H, 2(OC<i>H</i><sub>2</sub>)), 3.13 (t, <i>J</i> = 5.0 Hz, 4H, piperidine-2(C<i>H</i><sub>2</sub>)-2), 1.84 (t, <i>J</i> = 5.0 Hz, 4H, piperidine-2(C<i>H</i><sub>2</sub>)-3); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): δ [ppm] 155.4, 148.1, 141.8, 115.6, 106.2, 64.5, 47.4, 35.0; MS-ESI (<i>m</i>/<i>z</i>): for [C<sub>11</sub>H<sub>15</sub>ClN<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup> calcd, 271.09 (<sup>35</sup>Cl), 273.09 (<sup>37</sup>Cl); found, 271.14, 273.14.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 1-(3-Amino-6-chloro-pyridazin-4-yl)piperidin-4-ol (<b>16</b>)</h4><div class="NLM_p last"><b>6</b> (500 mg, 2.40 mmol) was reacted with piperidin-4-ol according to general procedure A. Purification was performed by flash silica column chromatography using 1–10% DCM/MeOH to yield compound <b>16</b> (244 mg, 1.07 mmol, 45%) as a slightly yellow solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 6.85 (s, 1H, pyridazine-C<i>H</i>), 6.05 (s, 2H, N<i>H</i><sub>2</sub>), 4.74 (d, <i>J</i> = 3.8 Hz, 1H, O<i>H</i>), 3.71–3.58 (m, 1H, piperidine-C<i>H</i>-4), 3.28–3.15 (m, 2H, piperidine-2(C<i>H</i>)-2<sub>ax</sub>), 2.79–2.65 (m, 2H, piperidine-2(C<i>H</i>)-2<sub>eq</sub>), 1.90–1.77 (m, 2H, piperidine-2(C<i>H</i>)-3<sub>ax</sub>), 1.68–1.47 (m, 2H, piperidine-2(C<i>H</i>)-3<sub>eq</sub>).; <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 155.5, 145.9, 141.4, 114.2, 65.3, 46.2, 33.5; MS-ESI (<i>m</i>/<i>z</i>): for [C<sub>9</sub>H<sub>13</sub>ClN<sub>4</sub>O + H]<sup>+</sup> calcd, 229.08 (<sup>35</sup>Cl), 231.07 (<sup>37</sup>Cl); found, 229.03, 231.05.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 4-Bromo-6-chloro-<i>N</i>-methyl-pyridazin-3-amine (<b>17</b>)</h4><div class="NLM_p last">To a suspension of <b>6</b> (1.00 g, 4.80 mmol) in dry THF (35 mL) at 0 °C was added first NaH (60% mineral oil, 288 mg, 1.5 equiv, 7.20 mmol), followed by 2 M iodomethane in <i>tert</i>-butylmethylether (2.5 mL, 1.05 equiv, 5.04 mmol) after 15 min stirring. The reaction mixture was allowed to warm to room temperature and was then stirred overnight at 40 °C. The reaction was quenched by pouring into a sat. NH<sub>4</sub>Cl (aq) solution. The solution was extracted with EtOAc (3 × 20 mL), the organic layer was separated and dried over MgSO<sub>4</sub>, filtered, and the solvent was removed under reduced pressure. The crude product was deposited onto silica and purified by flash silica column chromatography using 10–50% cyclohexane/EtOAc to yield compound <b>17</b> (390 mg, 1.75 mmol, 37%) as a yellow solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ [ppm] 7.46 (s, 1H, pyridazine-C<i>H</i>), 5.13 (s, 1H, N<i>H</i>), 3.14 (d, <i>J</i> = 5.0 Hz, 3H, C<i>H</i><sub>3</sub>); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): δ [ppm] 155.6, 145.0, 130.7, 116.9, 29.4; MS-ESI (<i>m</i>/<i>z</i>): for [C<sub>5</sub>H<sub>5</sub>BrClN<sub>3</sub> + H]<sup>+</sup> calcd, for <sup>35</sup>Cl, <sup>37</sup>Cl, <sup>79</sup>Br, <sup>81</sup>Br: 221.94, 223.93, 223.93, 225.93; found, 221.92, 223.92, 225.87.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 6-Chloro-<i>N</i>-methyl-4-[(3<i>S</i>)-3-phenoxy-1-piperidyl]-pyridazin-3-amine (<i>S</i>-<b>18</b>)</h4><div class="NLM_p last"><b>17</b> (100 mg, 0.449 mmol) was reacted with (3<i>S</i>)-3-phenoxypiperidine (1.5 equiv) according to general procedure A. Purification was performed by flash silica column chromatography using 10–70% cyclohexane/EtOAc to yield compound <i>S</i>-<b>18</b> (91 mg, 0.285 mmol, 64%) as a yellow solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ [ppm] 7.32 (t, <i>J</i> = 8.0 Hz, 2H, phenyl-(<i>m</i>)-C<i>H</i>), 6.99 (t, <i>J</i> = 7.4 Hz, 1H, phenyl-(<i>p</i>)-C<i>H</i>), 6.94 (d, <i>J</i> = 8.2 Hz, 2H, phenyl-(<i>o</i>)-C<i>H</i>), 6.62 (s, 1H pyridazine-C<i>H</i>), 5.13–5.02 (m, 1H, N<i>H</i>), 4.63–4.50 (m, 1H, piperidine-C<i>H</i>-3<sub>eq</sub>), 3.54 (d, <i>J</i> = 10.1 Hz, 1H, piperidine-C<i>H</i>-2<sub>ax</sub>), 3.11–2.97 (m, 2H, piperidine-C<i>H</i>-6<sub>ax</sub>/6<sub>eq</sub>), 2.89 (d, <i>J</i> = 5.0 Hz, 3H, C<i>H</i><sub>3</sub>), 2.84 (d, <i>J</i> = 12.5 Hz, 1H, piperidine-C<i>H</i>-2<sub>eq</sub>), 2.12–2.01 (m, 2H, piperidine-C<i>H</i><sub>2</sub>-5<sub>ax</sub>/4<sub>ax</sub>), 1.90–1.78 (m, 1H, piperidine-C<i>H</i>-4<sub>eq</sub>), 1.77–1.65 (m, 1H, piperidine-C<i>H</i>-5<sub>eq</sub>); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): δ [ppm] 156.8, 154.0, 146.7, 141.6, 129.9, 121.5, 115.6, 114.7, 70.5, 52.8, 49.8, 28.7, 27.9, 21.6; MS-ESI (<i>m</i>/<i>z</i>): for [C<sub>16</sub>H<sub>19</sub>ClN<sub>4</sub>O + H]<sup>+</sup> calcd, 319.12 (<sup>35</sup>Cl), 321.12 (<sup>37</sup>Cl); found, 319.06, 321.07.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 6-Chloro-<i>N</i>-methyl-4-[(3<i>R</i>)-3-phenoxy-1-piperidyl]-pyridazin-3-amine (<i>R</i>-<b>19</b>)</h4><div class="NLM_p last"><b>17</b> (200 mg, 0.899 mmol) was reacted with (3<i>R</i>)-3-phenoxypiperidine (1.5 equiv) according to general procedure A. Purification was performed by flash silica column chromatography using 10–70% cyclohexane/EtOAc to yield com/pound <i>R</i>-<b>19</b> (156 mg, 0.489 mmol, 54%) as a yellow solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ [ppm] 7.33 (t, <i>J</i> = 7.9 Hz, 2H, phenyl-(<i>m</i>)-C<i>H</i>), 7.00 (t, <i>J</i> = 7.4 Hz, 1H, phenyl-(<i>p</i>)-C<i>H</i>), 6.94 (d, <i>J</i> = 8.0 Hz, 2H, phenyl-(<i>o</i>)-C<i>H</i>), 6.63 (s, 1H, pyridazine-C<i>H</i>), 5.11–4.98 (m, 1H, N<i>H</i>), 4.61–4.53 (m, 1H, piperidine-C<i>H</i>-3<sub>ax</sub>), 3.56 (d, <i>J</i> = 10.3 Hz, 1H, piperidine-C<i>H</i>-2<sub>ax</sub>), 3.11–2.95 (m, 2H, piperidine-C<i>H</i>-6<sub>ax</sub>/6<sub>eq</sub>), 2.91 (d, <i>J</i> = 5.0 Hz, 3H, C<i>H</i><sub>3</sub>), 2.84 (d, <i>J</i> = 12.6 Hz, 1H, piperidine-C<i>H</i>-2<sub>eq</sub>), 2.12–2.02 (m, 2H, piperidine-C<i>H</i><sub>2</sub>-5<sub>ax</sub>/4<sub>ax</sub>), 1.90–1.78 (m, 1H, piperidine-C<i>H</i>-4<sub>eq</sub>), 1.77–1.68 (m, 1H, piperidine-C<i>H</i>-5<sub>eq</sub>); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): δ [ppm] 156.9, 154.1, 146.7, 141.6, 130.0, 121.6, 115.6, 114.7, 70.5, 52.9, 49.9, 28.7, 28.0, 21.6; MS-ESI (<i>m</i>/<i>z</i>): for [C<sub>16</sub>H<sub>19</sub>ClN<sub>4</sub>O + H]<sup>+</sup> calcd, 319.12 (<sup>35</sup>Cl), 321.12 (<sup>37</sup>Cl); found, 319.03, 321.03.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> <i>trans</i>-1-(3-Amino-6-chloro-pyridazin-4-yl)piperidine-3,4-diol (<i>Trans</i>-<b>20</b>)</h4><div class="NLM_p last"><b>6</b> (200 mg, 0.959 mmol) was reacted with <i>trans</i>-piperidine-3,4-diol hydrochloride (1.7 equiv) according to general procedure A. Purification was performed by flash C18 column chromatography using water/MeCN + 0.1% TFA (98/2–5/95) to yield <i>trans</i>-<b>20</b> (90 mg, 0.368 mmol, 38%) as a white solid. <sup>1</sup>H NMR (500 MHz, <i>d</i><sub>4</sub>-MeOH): δ [ppm] 6.93 (s, 1H, pyridazine-C<i>H</i>), 3.68–3.64 (m, 1H, piperidine-C<i>H</i>-3<sub>ax/eq</sub>), 3.62–3.59 (m, 1H, piperidine-C<i>H</i>-4<sub>ax/eq</sub>), 3.40 (dd, <i>J</i> = 12.2, 2.4 Hz, 1H, piperidine-C<i>H</i>-2<sub>ax</sub>), 3.37–3.32 (m, 1H, piperidine-C<i>H</i>-6<sub>ax</sub>), 2.91–2.80 (m, 2H, piperidine-C<i>H</i>-6<sub>eq</sub>/2<sub>eq</sub>), 2.15–2.05 (m, 1H, piperidine-C<i>H</i>-5<sub>ax</sub>), 1.76–1.68 (m, 1H, piperidine-C<i>H</i>-5<sub>eq</sub>). <sup>13</sup>C NMR (125 MHz, <i>d</i><sub>4</sub>-MeOH): δ [ppm] 157.0, 148.3, 143.8, 116.4, 72.0, 71.5, 53.1, 47.4, 31.5. MS-ESI (<i>m</i>/<i>z</i>): for [C<sub>9</sub>H<sub>13</sub>ClN<sub>4</sub>O + H]<sup>+</sup> calcd, 245.07 (<sup>35</sup>Cl), 247.07 (<sup>37</sup>Cl); found, 245.08, 247.10.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> <i>cis</i>-1-(3-Amino-6-chloro-pyridazin-4-yl)piperidine-3,4-diol (<i>Cis</i>-<b>21</b>)</h4><div class="NLM_p last"><b>6</b> (200 mg, 0.959 mmol) was reacted with <i>cis</i>-piperidine-3,4-diol hydrochloride (1.7 equiv) according to general procedure A. Purification was performed by flash C18 column chromatography using water/MeCN + 0.1% TFA (98/2–5/95) to yield <i>cis</i>-<b>21</b> (36 mg, 0.147 mmol, 15%) as a white solid. <sup>1</sup>H NMR (500 MHz, <i>d</i><sub>4</sub>-MeOH): δ [ppm] 6.93 (s, 1H, pyridazine-C<i>H</i>), 3.92–3.90 (m, 1H, piperidine-C<i>H</i>-3<sub>ax/eq</sub>), 3.80 (dt, <i>J</i> = 9.0, 3.7 Hz, 1H, piperidine-C<i>H</i>-4<sub>ax/eq</sub>), 3.47 (dd, <i>J</i> = 12.3, 5.0 Hz, 1H, piperidine-C<i>H</i>-2<sub>ax</sub>), 3.30–3.23 (m, 1H, piperidine-C<i>H</i>-6<sub>ax</sub>), 3.01–2.92 (m, 1H, piperidine-C<i>H</i>-6<sub>eq</sub>), 2.90 (dd, <i>J</i> = 12.6, 1.9 Hz, 1H, piperidine-C<i>H</i>-2<sub>eq</sub>), 2.03–1.95 (m, 1H, piperidine-C<i>H</i>-5<sub>ax</sub>), 1.85–1.82 (m, 1H, piperidine-C<i>H</i>-5<sub>eq</sub>). <sup>13</sup>C NMR (125 MHz, <i>d</i><sub>4</sub>-MeOH): δ [ppm] 157.0, 148.2, 143.9, 116.4, 69.8, 69.3, 52.8, 47.4, 30.0. MS-ESI (<i>m</i>/<i>z</i>): for [C<sub>9</sub>H<sub>13</sub>ClN<sub>4</sub>O + H]<sup>+</sup> calcd, 245.07 (<sup>35</sup>Cl), 247.07 (<sup>37</sup>Cl); found, 245.07, 247.10.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 2-(6-Amino-5-piperazin-1-yl-pyridazin-3-yl)phenol Hydrochloride (<b>23</b>)</h4><div class="NLM_p last">Compound <b>22</b> (500 mg, 1.35 mmol) was treated with 10 mL of a 4 M HCl solution in 1,4-dioxan for 2 h at room temperature. The resulting precipitate was filtered through a Büchner funnel, washed with 10 mL 1,4-dioxan, 5 mL DCM and finally dried under reduced pressure to give <b>23</b> (414 mg, 1.35 mmol, quant.) as a white HCl-solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 10. 96 (br s, 1H, O<i>H</i>), 9.84 (s, 2H, piperazine-NH·HCl), 7.79 (br s, 3H, NH<sub>2</sub>·HCl), 7.62 (s, 1H, pyridazine-C<i>H</i>), 7.57 (d, <i>J</i> = 7.5 Hz, 1H, phenol-C<i>H</i>-3), 7.37 (t, <i>J</i> = 7.5 Hz, 1H, phenol-C<i>H</i>-5), 7.15 (d, <i>J</i> = 8.1 Hz, 1H, phenol-C<i>H</i>-6), 6.96 (t, <i>J</i> = 7.4 Hz, 1H, phenol-C<i>H</i>-4), 3.47 (br s, 4H, piperazine-2(C<i>H</i><sub>2</sub>)-2), 3.32 (br s, 4H, piperazine-2(C<i>H</i><sub>2</sub>)-3); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 155.6, 132.2, 129.9, 119.5, 116.7, 45.4, 41.8; HRMS-ESI (<i>m</i>/<i>z</i>): for [C<sub>14</sub>H<sub>17</sub>N<sub>5</sub>O + H]<sup>+</sup> calcd, 272.150; found, 272.149. HPLC–UV (254 nm) = 99.8%, (280 nm) = 98.9%.</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 2-(6-Amino-5-morpholino-pyridazin-3-yl)phenol (<b>24</b>)</h4><div class="NLM_p last"><b>9</b> (100 mg, 0.466 mmol) was reacted with (2-hydroxyphenyl)boronic acid <b>7</b> according to general procedure B. Purification was performed by flash silica column chromatography using 20–100% cyclohexane/EtOAc to yield compound <b>24</b> (54 mg, 0.198 mmol, 43%) as a white solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 7.93 (dd, <i>J</i> = 8.3, 1.5 Hz, 1H, phenol-C<i>H</i>-3), 7.52 (s, 1H, pyridazine-C<i>H</i>), 7.24 (dt, <i>J</i> = 8.0, 1.5 Hz, 1H, phenol-C<i>H</i>-5), 6.93–6.86 (m, 2H, phenol-C<i>H</i>-6/4), 6.37 (s, 2H, N<i>H</i><sub>2</sub>), 3.81 (t, <i>J</i> = 4.5 Hz, 4H, morpholin-2(C<i>H</i><sub>2</sub>)-3), 3.09 (t, <i>J</i> = 4.5 Hz, 4H, morpholin-2(C<i>H</i><sub>2</sub>)-2); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 158.5, 154.6, 153.1, 140.1, 130.1, 126.1, 118.5, 117.8, 117.4, 110.4, 65.7, 48.9; HRMS-ESI (<i>m</i>/<i>z</i>): for [C<sub>14</sub>H<sub>16</sub>N<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup> calcd, 273.135; found, 273.135. HPLC–UV (254 nm) = 98.9%, (280 nm) = 99.0%.</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 2-[6-Amino-5-(3-phenoxy-1-piperidyl)pyridazin-3-yl]phenol (<i>Rac</i>-<b>25</b>)</h4><div class="NLM_p last"><i>Rac</i>-<b>10</b> (55 mg, 0.180 mmol) was reacted with (2-hydroxyphenyl)boronic acid <b>7</b> according to general procedure B. Purification was performed by flash silica column chromatography using 20–100% cyclohexane/EtOAc to yield compound <i>rac</i>-<b>25</b> (43 mg, 0.119 mmol, 66%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ [ppm] 13.68 (s, 1H, O<i>H</i>), 7.58 (dd, <i>J</i> = 8.0, 1.4 Hz, 1H, phenol-C<i>H</i>-3), 7.36–7.31 (m, 2H, phenyl-(<i>m</i>)-C<i>H</i>), 7.29 (s, 1H, pyridazine-C<i>H</i>), 7.27 (dt, <i>J</i> = 8.0, 1.5 Hz, 1H, phenol-C<i>H</i>-5), 7.05 (dd, <i>J</i> = 8.2, 1.0 Hz, 1H, phenol-C<i>H</i>-6), 7.01 (t, <i>J</i> = 7.4 Hz, 1H, phenyl-(<i>p</i>)-C<i>H</i>), 6.98–6.94 (m, 2H, phenyl-(<i>o</i>)-C<i>H</i>), 6.89 (dt, <i>J</i> = 7.0, 1.0 Hz, 1H, phenol-C<i>H</i>-4), 5.14 (s, 2H, N<i>H</i><sub>2</sub>), 4.68–4.51 (m, 1H, piperidine-C<i>H</i>-3<sub>ax/eq</sub>), 3.75–3.52 (m, 1H, piperidine-C<i>H</i>-2<sub>ax</sub>), 3.32–3.20 (m, 1H, piperidine-C<i>H</i>-6<sub>ax</sub>), 3.20–3.13 (m, 1H, piperidine-C<i>H</i><sub>2</sub>-6<sub>eq</sub>), 3.10 (d, <i>J</i> = 12.3 Hz, 1H, piperidine-C<i>H</i><sub>2</sub>-2<sub>eq</sub>), 2.17–2.01 (m, 2H, piperidine-C<i>H</i>-5<sub>ax</sub>/4<sub>ax</sub>), 1.95–1.87 (m, 1H, piperidine-C<i>H</i>-4<sub>eq</sub>), 1.82–1.72 (m, 1H, piperidine-C<i>H</i>-5<sub>eq</sub>); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): δ [ppm] 159.3, 156.9, 154.9, 154.2, 141.2, 130.8, 129.9, 125.2, 121.7, 118.7, 118.5, 117.7, 116.0, 110.9, 71.2, 53.2, 49.8, 28.0, 21.9; HRMS-ESI (<i>m</i>/<i>z</i>): for [C<sub>21</sub>H<sub>22</sub>N<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup> calcd, 363.174.135; found, 363.175. HPLC–UV (254 nm) = 98.5%, (280 nm) = 98.3%.</div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 2-[6-Amino-5-[(3<i>R</i>)-3-phenoxy-1-piperidyl]pyridazin-3-yl]phenol (<i>R</i>-<b>26</b>)</h4><div class="NLM_p last"><i>R</i>-<b>11</b> (100 mg, 0.328 mmol) was reacted with (2-hydroxyphenyl)boronic acid <b>7</b> according to general procedure B. Purification was performed by flash silica column chromatography using 1–10% DCM/MeOH, followed by a preparative RP-C18-HPLC separation with a gradient (98–5%) of H<sub>2</sub>O/MeCN and 0.1% TFA. The product fractions were combined and the solvents were removed under reduced pressure. To remove TFA excess the product was dissolved in 2 mL CHCl<sub>3</sub> with 50 mg NaHCO<sub>3</sub>. The mixture was stirred for 1 h at room temperature, filtered through a spin column and finally dried to yield compound <i>R</i>-<b>26</b> (68 mg, 0.188 mmol, 58%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ [ppm] 13.81 (s, 1H, O<i>H</i>), 7.55 (dd, <i>J</i> = 8.0, 1.2 Hz, 1H, phenol-C<i>H</i>-3), 7.30 (t, <i>J</i> = 7.5 Hz, 2H, phenyl-(<i>m</i>)-C<i>H</i>), 7.27–7.21 (m, 2H, pyridazine-C<i>H</i>/phenol-C<i>H</i>-5), 7.02 (d, <i>J</i> = 8.2 Hz, 1H, phenol-C<i>H</i>-6), 6.98 (t, <i>J</i> = 7.4 Hz, 1H, phenyl-(<i>p</i>)-C<i>H</i>), 6.94 (d, <i>J</i> = 8.0 Hz, 2H, phenyl-(<i>o</i>)-C<i>H</i>), 6.87 (t, <i>J</i> = 7.9 Hz, 1H, phenol-C<i>H</i>-4), 5.12 (s, 2H, N<i>H</i><sub>2</sub>), 4.62–4.51 (m, 1H, piperidine-C<i>H</i>-3<sub>ax</sub>), 3.59 (dd, <i>J</i> = 12.1, 4.1 Hz, 1H, piperidine-C<i>H</i>-2<sub>ax</sub>), 3.29–3.15 (m, 1H, piperidine-C<i>H</i>-6<sub>ax</sub>), 3.15–2.99 (m, 2H, piperidine-C<i>H</i><sub>2</sub>-6<sub>eq</sub>/2<sub>eq</sub>), 2.14–1.97 (m, 2H, piperidine-C<i>H</i>-5<sub>ax</sub>/4<sub>ax</sub>), 1.95–1.84 (m, 1H, piperidine-C<i>H</i>-4<sub>eq</sub>), 1.79–1.69 (m, 1H, piperidine-C<i>H</i>-5<sub>eq</sub>); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): δ [ppm] 159.3, 156.9, 154.8, 154.3, 141.1, 130.7, 129.9, 125.2, 121.6, 118.7, 118.4, 117.7, 116.0, 110.7, 71.1, 53.1, 49.7, 28.0, 21.9; HRMS-ESI (<i>m</i>/<i>z</i>): for [C<sub>21</sub>H<sub>22</sub>N<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup> calcd, 363.181; found, 363.182. HPLC–UV (254 nm) = 98.2%, (280 nm) = 98.3%.</div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 2-[6-Amino-5-[(3<i>S</i>)-3-phenoxy-1-piperidyl]pyridazin-3-yl]phenol (<i>S</i>-<b>27</b>)</h4><div class="NLM_p last"><i>S</i>-<b>12</b> (100 mg, 0.328 mmol) was reacted with (2-hydroxyphenyl)boronic acid <b>7</b> according to general procedure B. Purification was performed by flash silica column chromatography using 1–5% DCM/MeOH, followed by a preparative RP-C18-HPLC separation with a gradient (98–5%) of H<sub>2</sub>O/MeCN and 0.1% TFA. The product fractions were combined and the solvents were removed under reduced pressure. To remove TFA excess the product was dissolved in 2 mL CHCl<sub>3</sub> with 50 mg NaHCO<sub>3</sub>. The mixture was stirred for 1 h at room temperature, filtered through a spin column and finally dried to yield compound <i>S</i>-<b>27</b> (65 mg, 0.179 mmol, 55%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ [ppm] 13.86 (s, 1H, O<i>H</i>), 7.55 (dd, <i>J</i> = 7.9, 1.1 Hz, 1H, phenol-C<i>H</i>-3), 7.30 (t, <i>J</i> = 7.9 Hz, 2H, phenyl-(<i>m</i>)-C<i>H</i>), 7.27–7.22 (m, 2H, pyridazine-C<i>H</i>/phenol-C<i>H</i>-5), 7.02 (d, <i>J</i> = 8.2 Hz, 1H, phenol-C<i>H</i>-6), 6.98 (t, <i>J</i> = 7.4 Hz, 1H, phenyl-(<i>p</i>)-C<i>H</i>), 6.94 (d, <i>J</i> = 8.1 Hz, 2H, phenyl-(<i>o</i>)-C<i>H</i>), 6.87 (t, <i>J</i> = 7.5 Hz, 1H, phenol-C<i>H</i>-4), 5.11 (s, 2H, N<i>H</i><sub>2</sub>), 4.62–4.48 (m, 1H, piperidine-C<i>H</i>-3<sub>eq</sub>), 3.58 (dd, <i>J</i> = 12.1, 4.2 H z, 1H, piperidine-C<i>H</i>-2<sub>ax</sub>), 3.29–3.15 (m, 1H, piperidine-C<i>H</i>-6<sub>ax</sub>), 3.15–3.02 (m, 2H, piperidine-C<i>H</i>-6<sub>eq</sub>/2<sub>eq</sub>), 2.15–1.96 (m, 2H, piperidine-C<i>H</i><sub>2</sub>-5<sub>ax</sub>/4<sub>ax</sub>), 1.96–1.81 (m, 1H, piperidine-C<i>H</i>-4<sub>eq</sub>), 1.81–1.64 (m, 1H, piperidine-C<i>H</i>-5<sub>eq</sub>); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): δ [ppm] 159.3, 156.9, 154.8, 154.3, 141.1, 130.7, 129.9, 125.2, 121.6, 118.7, 118.4, 117.7, 116.0, 110.7, 71.1, 53.1, 49.7, 28.0, 21.9; HRMS-ESI (<i>m</i>/<i>z</i>): for [C<sub>21</sub>H<sub>22</sub>N<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup> calcd, 363.181; found, 363.181. HPLC–UV (254 nm) = 99.6%, (280 nm) = 99.1%.</div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> (3<i>R</i>)-1-[3-Amino-6-(2-hydroxyphenyl)pyridazin-4-yl]piperidin-3-ol (<i>R</i>-<b>28</b>)</h4><div class="NLM_p last"><i>R</i>-<b>13</b> (73 mg, 0.319 mmol) was reacted with (2-hydroxyphenyl)boronic acid <b>7</b> according to general procedure B. Purification was performed by flash silica column chromatography using 20–100% cyclohexane/EtOAc to yield compound <i>R</i>-<b>28</b> (36 mg, 0.126 mmol, 39%) as a white solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 7.91 (dd, <i>J</i> = 8.4, 1.4 Hz, 1H, phenol-C<i>H</i>-3), 7.49 (s, 1H, pyridazine-C<i>H</i>), 7.23 (td, <i>J</i> = 8.0, 1.5 Hz, 1H, phenol-C<i>H</i>-5), 6.89–6.87 (m, 2H, phenol-C<i>H</i>-6/4), 6.26 (s, 2H, N<i>H</i><sub>2</sub>), 4.98 (d, <i>J</i> = 4.7 Hz, 1H, piperidine-O<i>H</i>), 3.87–3.73 (m, 1H, piperidine-C<i>H</i>-3<sub>ax</sub>), 3.22 (dd, <i>J</i> = 10.6, 2.0 Hz, 1H, piperidine-C<i>H</i>-2<sub>ax</sub>), 3.20–3.10 (m, 1H, piperidine-C<i>H</i>-6<sub>ax</sub>), 2.87 (t, <i>J</i> = 9.1 Hz, 1H, piperidine-C<i>H</i>-6<sub>eq</sub>), 2.79 (dd, <i>J</i> = 11.4, 7.5 Hz, 1H, piperidine-C<i>H</i>-2<sub>eq</sub>), 1.92–1.78 (m, 2H, piperidine-C<i>H</i><sub>2</sub>-5<sub>ax</sub>/4<sub>ax</sub>), 1.71–1.60 (m, 1H, piperidine-C<i>H</i>-5<sub>eq</sub>), 1.50–1.38 (m, 1H, piperidine-C<i>H</i>-4<sub>eq</sub>); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 158.6, 154.9, 153.2, 140.8, 130.1, 126.1, 118.4, 117.8, 117.4, 110.4, 64.7, 55.6, 49.0, 32.2, 22.0; HRMS-ESI (<i>m</i>/<i>z</i>): for [C<sub>15</sub>H<sub>18</sub>N<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup> calcd, 287.151; found, 287.151. HPLC–UV (254 nm) = 99.0%, (280 nm) = 98.6%.</div></div><div id="sec4_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> (3<i>S</i>)-1-[3-Amino-6-(2-hydroxyphenyl)pyridazin-4-yl]piperidin-3-ol (<i>S</i>-<b>29</b>)</h4><div class="NLM_p last"><i>S</i>-<b>14</b> (150 mg, 0.656 mmol) was reacted with (2-hydroxyphenyl)boronic acid <b>7</b> according to general procedure B. Purification was performed by flash silica column chromatography using 20–100% cyclohexane/EtOAc to yield compound <i>S</i>-<b>29</b> (85 mg, 0.297 mmol, 45%) as a white solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 7.91 (dd, <i>J</i> = 8.3, 1.3 Hz, 1H, phenol-C<i>H</i>-3), 7.49 (s, 1H, pyridazine-C<i>H</i>), 7.23 (td, <i>J</i> = 8.0, 1.5 Hz, 1H, phenol-C<i>H</i>-5), 6.89–6.87 (m, phenol-C<i>H</i>-6/4), 6.26 (s, 2H, N<i>H</i><sub>2</sub>), 4.98 (d, <i>J</i> = 4.6 Hz, 1H, piperidine-O<i>H</i>), 3.87–3.72 (m, 1H, piperidine-C<i>H</i>-3<sub>eq</sub>), 3.22 (d, <i>J</i> = 9.7 Hz, 1H, piperidine-C<i>H</i>-2<sub>ax</sub>), 3.20–3.10 (m, 1H, piperidine-C<i>H</i>-6<sub>ax</sub>), 2.87 (t, <i>J</i> = 9.1 Hz, 1H, piperidine-C<i>H</i>-6<sub>eq</sub>), 2.79 (dd, <i>J</i> = 11.4, 7.5 Hz, 1H, piperidine-C<i>H</i>-2<sub>eq</sub>), 1.92–1.75 (m, 2H, piperidine-C<i>H</i><sub>2</sub>-5<sub>ax</sub>4<sub>ax</sub>), 1.72–1.55 (m, 1H, piperidine-C<i>H</i>-5<sub>eq</sub>), 1.51–1.36 (m, 1H, piperidine-C<i>H</i>-4<sub>eq</sub>); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 158.6, 154.9, 153.2, 140.8, 130.1, 126.1, 118.4, 117.8, 117.4, 110.4, 64.7, 55.6, 49.0, 32.2, 22.0; HRMS-ESI (<i>m</i>/<i>z</i>): for [C<sub>15</sub>H<sub>18</sub>N<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup> calcd, 287.151; found, 287.153. HPLC–UV (254 nm) = 98.8%, (280 nm) = 98.7%.</div></div><div id="sec4_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 2-[6-(Methylamino)-5-[(3<i>R</i>)-3-phenoxy-1-piperidyl]pyridazine-3-yl]phenol (<i>R</i>-<b>30</b>)</h4><div class="NLM_p last"><i>R</i>-<b>19</b> (100 mg, 0.314 mmol) was reacted with (2-hydroxyphenyl)boronic acid <b>7</b> according to general procedure B. Purification was performed by flash silica column chromatography using 10–70% cyclohexane/EtOAc to yield compound <i>R</i>-<b>30</b> (87 mg, 0.231 mmol, 74%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ [ppm] 7.59 (d, <i>J</i> = 7.8 Hz, 1H, phenol-C<i>H</i>-3), 7.38 (t, <i>J</i> = 7.9 Hz, 2H, phenyl-(<i>m</i>)-C<i>H</i>), 7.29 (t, <i>J</i> = 5.8 Hz, 1H, phenol-C<i>H</i>-5), 7.26 (s, 1H, pyridazine-C<i>H</i>), 7.08 (d, <i>J</i> = 8.5 Hz, 1H, phenol-C<i>H</i>-6), 7.04 (t, <i>J</i> = 7.4 Hz, 1H, phenyl-(<i>p</i>)-C<i>H</i>), 7.00 (d, <i>J</i> = 8.1 Hz, 2H, phenyl-(<i>o</i>)-C<i>H</i>), 6.91 (t, <i>J</i> = 7.5 Hz, 1H, phenol-C<i>H</i>-5), 5.32 (d, <i>J</i> = 4.6 Hz, 1H, N<i>H</i>), 4.68–4.60 (m, 1H, piperidine-C<i>H</i>-3<sub>ax</sub>), 3.65 (d, <i>J</i> = 10.5 Hz, 1H, piperidine-C<i>H</i>-2<sub>ax</sub>), 3.27–3.11 (m, 2H, piperidine-C<i>H</i>-6<sub>ax</sub>/6<sub>eq</sub>), 2.99 (d, <i>J</i> = 5.0 Hz, 3H, C<i>H</i><sub>3</sub>), 2.94 (d, <i>J</i> = 12.6 Hz, 1H, piperidine-C<i>H</i><sub>2</sub>-2<sub>eq</sub>), 2.19–2.06 (m, 2H, piperidine-C<i>H</i>-5<sub>ax</sub>/4<sub>ax</sub>), 1.97–1.85 (m, 1H, piperidine-C<i>H</i>-4<sub>eq</sub>), 1.85–1.75 (m, 1H, piperidine-C<i>H</i>-<sub>eq</sub>); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): δ [ppm] 159.3, 156.9, 153.4, 153.4, 141.3, 130.5, 130.0, 125.1, 121.6, 118.5, 118.4, 117.9, 115.6, 110.2, 70.7, 53.2, 50.1, 28.8, 28.1, 21.7; HRMS-ESI (<i>m</i>/<i>z</i>): for [C<sub>22</sub>H<sub>24</sub>N<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup> calcd, 377.197; found, 377.198. HPLC–UV (254 nm) = 99.5%, (280 nm) = 99.8%.</div></div><div id="sec4_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 2-[6-(Methylamino)-5-[(3<i>S</i>)-3-phenoxy-1-piperidyl]pyridazine-3-yl]phenol (<i>S</i>-<b>31</b>)</h4><div class="NLM_p last"><i>S</i>-<b>18</b> (80 mg, 0.251 mmol) was reacted with (2-hydroxyphenyl)boronic acid <b>7</b> according to general procedure B. Purification was performed by flash silica column chromatography using 10–70% cyclohexane/EtOAc to yield compound <i>S</i>-<b>31</b> (64 mg, 0.170 mmol, 68%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ [ppm] 7.59 (dd, <i>J</i> = 8.0, 1.3 Hz, 1H, phenol-C<i>H</i>-3), 7.38 (t, <i>J</i> = 7.5 Hz, 2H, phenyl-(<i>m</i>)-C<i>H</i>), 7.28 (t, <i>J</i> = 7.8 Hz, 1H, phenol-C<i>H</i>-5), 7.25 (s, 1H, pyridazine-C<i>H</i>), 7.07 (d, <i>J</i> = 8.4 Hz, 1H, phenol-C<i>H</i>-6), 7.04 (t, <i>J</i> = 7.4 Hz, 1H, phenyl-(<i>p</i>)-C<i>H</i>), 7.00 (d, <i>J</i> = 7.9 Hz, 2H, phenyl-(<i>o</i>)-C<i>H</i>), 6.91 (td, <i>J</i> = 8.1, 1.1 Hz, 1H, phenol-C<i>H</i>-4), 5.32 (d, <i>J</i> = 4.8 Hz, 1H, N<i>H</i>), 4.69–4.55 (m, 1H, piperidine-C<i>H</i>-3<sub>eq</sub>), 3.64 (d, <i>J</i> = 10.4 Hz, 1H, piperidine-C<i>H</i>-2<sub>ax</sub>), 3.26–3.07 (m, 2H, piperidine-C<i>H</i>-6<sub>ax</sub>/6<sub>eq</sub>), 2.98 (d, <i>J</i> = 5.0 Hz, 3H, C<i>H</i><sub>3</sub>), 2.93 (d, <i>J</i> = 12.6 Hz, 1H, piperidine-C<i>H</i><sub>2</sub>-2<sub>eq</sub>), 2.19–2.06 (m, 2H, piperidine-C<i>H</i>-5<sub>ax</sub>/4<sub>ax</sub>), 1.96–1.84 (m, 1H, piperidine-C<i>H</i>-4<sub>eq</sub>), 1.84–1.75 (m, 1H, piperidine-C<i>H</i>-5<sub>eq</sub>); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): δ [ppm] 159.3, 156.9, 153.4, 153.3, 141.3, 130.4, 130.0, 125.1, 121.6, 118.5, 118.4, 117.8, 115.5, 110.1, 70.7, 53.2, 50.0, 28.7, 28.0, 21.7; HRMS-ESI (<i>m</i>/<i>z</i>): for [C<sub>22</sub>H<sub>24</sub>N<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup> calcd, 377.197; found, 377.198. HPLC–UV (254 nm) = 99.2%, (280 nm) = 99.4%.</div></div><div id="sec4_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 1-[3-Amino-6-(2-hydroxyphenyl)pyridazin-4-yl]piperidin-4-ol (<b>32</b>)</h4><div class="NLM_p last"><b>16</b> (130 mg, 0.568 mmol) was reacted with (2-hydroxyphenyl)boronic acid <b>7</b> according to general procedure B. Purification was performed by flash silica column chromatography using 20–100% cyclohexane/EtOAc to yield compound <b>32</b> (81 mg, 0.283 mmol, 50%) as a white solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 7.91 (dd, <i>J</i> = 7.7, 0.6 Hz, 1H, phenol-C<i>H</i>-3), 7.50 (s, 1H, pyridazine-C<i>H</i>), 7.23 (dt, <i>J</i> = 7.5, 1.5 Hz, 1H, phenol-C<i>H</i>-5), 6.93–6.83 (m, 2H, phenol-C<i>H</i>-6/4), 6.22 (s, 2H, N<i>H</i><sub>2</sub>), 4.77 (d, <i>J</i> = 3.1 Hz, 1H, piperidine-O<i>H</i>), 3.76–3.59 (m, 1H, piperidine-C<i>H</i>-4), 3.33 (d, <i>J</i> = 6.0 Hz, 2H, piperidine-2(C<i>H</i>)-2<sub>ax</sub>), 2.83 (t, <i>J</i> = 9.6 Hz, 2H, piperidine-2(C<i>H</i>)-2<sub>eq</sub>), 1.95–1.84 (m, 2H, piperidine-2(C<i>H</i>)-3<sub>ax</sub>), 1.73–1.59 (m, 2H, piperidine-2(C<i>H</i>)-3<sub>eq</sub>); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 158.6, 154.8, 153.1, 141.0, 130.0, 126.0, 118.4, 117.8, 117.4, 110.3, 65.4, 46.5, 33.70; HRMS-ESI (<i>m</i>/<i>z</i>): for [C<sub>15</sub>H<sub>18</sub>N<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup> calcd, 287.151; found, 287.154. HPLC–UV (254 nm) = 98.8%, (280 nm) = 98.7%.</div></div><div id="sec4_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 2-[6-Amino-5-(1,4-dioxa-8-azaspiro[4.5]decan-8-yl)pyridazin-3-yl]phenol (<b>33</b>)</h4><div class="NLM_p last"><b>15</b> (150 mg, 0.554 mmol) was reacted with (2-hydroxyphenyl)boronic acid <b>7</b> according to general procedure B. Purification was performed by flash silica column chromatography using 20–100% cyclohexane/EtOAc to yield compound <b>33</b> (73 mg, 0.222 mmol, 40%) as a white solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 7.94 (d, <i>J</i> = 7.1 Hz, 1H, phenol-C<i>H</i>-3), 7.54 (s, 1H, pyridazine-C<i>H</i>), 7.23 (td, <i>J</i> = 8.4, 1.4 Hz, 1H, phenol-C<i>H</i>-5), 6.94–6.80 (m, 2H, phenol-C<i>H</i>-6/4), 6.31 (s, 2H, N<i>H</i><sub>2</sub>), 3.93 (s, 4H, 2(OC<i>H</i><sub>2</sub>)), 3.15 (t, <i>J</i> = 6.0 Hz, 4H, piperidine-2(C<i>H</i><sub>2</sub>)-2), 1.87 (t, <i>J</i> = 5.5 Hz, 4H, piperidine-2(C<i>H</i><sub>2</sub>)-3); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 159.0, 155.3, 153.6, 141.1, 130.5, 126.6, 118.9, 118.2, 117.8, 111.2, 106.5, 64.2, 47.5, 34.6; MS-ESI (<i>m</i>/<i>z</i>): for [C<sub>17</sub>H<sub>20</sub>N<sub>4</sub>O<sub>3</sub> + H]<sup>+</sup> calcd, 328.15; found, 328.34.</div></div><div id="sec4_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 1-[3-Amino-6-(2-hydroxyphenyl)pyridazin-4-yl]piperidin-4-one (<b>34</b>)</h4><div class="NLM_p last"><b>33</b> (35 mg, 0.107 mmol) was dissolved in 5 mL THF. After adding 1 mL 6 M HCl solution, the reaction mixture was stirred overnight at room temperature. The reaction was quenched by adding 15 mL of sat. aq NaHCO<sub>3</sub> solution, which was then extracted with EtOAc (4 × 10 mL). The combined organic layers were washed with water (3 × 10 mL), dried over MgSO<sub>4</sub>, filtered and concentrated under reduced pressure. The crude product was deposited onto silica and purified by flash silica column chromatography using 20–100% cyclohexane/EtOAc to yield compound <b>34</b> (30 mg, 0.070 mmol, 66%) as a white solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 7.92 (d, <i>J</i> = 7.1 Hz, 1H, phenol-C<i>H</i>-3), 7.59 (s, 1H, pyridazine-C<i>H</i>), 7.24 (td, <i>J</i> = 7.0, 1.5 Hz, 1H, phenol-C<i>H</i>-5), 6.97–6.83 (m, 2H, phenol-C<i>H</i>-6/4), 6.48 (d, <i>J</i> = 4.9 Hz, 2H, N<i>H</i><sub>2</sub>), 3.41 (t, <i>J</i> = 5.9 Hz, 4H, piperidine-2(C<i>H</i><sub>2</sub>)-2), 2.62 (t, <i>J</i> = 6.0 Hz, 4H, piperidine-2(C<i>H</i><sub>2</sub>)-3); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 207.3, 158.4, 154.8, 153.1, 139.8, 130.1, 126.2, 118.5, 117.8, 117.3, 111.3, 48.1, 40.5; HRMS-ESI (<i>m</i>/<i>z</i>): for [C<sub>15</sub>H<sub>16</sub>N<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup> calcd, 285.135; found, 285.138. HPLC–UV (254 nm) = 97.5%, (280 nm) = 97.2%.</div></div><div id="sec4_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> <i>tert</i>-Butyl 4-[3-amino-6-(2-methoxyphenyl)yridazine-4-yl]piperazine-1-carboxylate (<b>35</b>)</h4><div class="NLM_p last"><b>8</b> (55 mg, 0.175 mmol) was reacted with (2-methoxyphenyl)boronic acid <b>7a</b> according to general procedure B. Purification was performed by flash silica column chromatography using 1–10% DCM/MeOH to yield compound <b>35</b> (42 mg, 0.109 mmol, 62%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ [ppm] 7.77 (dd, <i>J</i> = 7.6, 1.7 Hz, 1H, phenol-C<i>H</i>-3), 7.36 (ddd, <i>J</i> = 8.3, 7.6, 1.8 Hz, 1H, phenol-C<i>H</i>-5), 7.26 (s, 1H, pyridazine-C<i>H</i>), 7.06 (td, <i>J</i> = 7.5, 0.9 Hz, 1H, phenol-C<i>H</i>-4), 6.97 (d, <i>J</i> = 8.3 Hz, 1H, phenol-C<i>H</i>-6), 5.27 (s, 2H, N<i>H</i><sub>2</sub>), 3.82 (s, 3H, OC<i>H</i><sub>3</sub>), 3.59 (t, <i>J</i> = 5.0 Hz, 4H, piperazine-2(C<i>H</i><sub>2</sub>)-3), 3.00 (t, <i>J</i> = 4.5 Hz, 4H, piperazine-2(C<i>H</i><sub>2</sub>)-2), 1.47 (s, 9H, <i>tert</i>-butyl (C<i>H</i><sub>3</sub>)<sub>3</sub>); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): δ [ppm] 156.7, 154.7, 154.6, 152.9, 138.2, 130.8, 130.1, 126.8, 121.2, 117.1, 111.3, 80.2, 55.7, 49.2, 28.4; HRMS-ESI (<i>m</i>/<i>z</i>): for [C<sub>20</sub>H<sub>27</sub>N<sub>5</sub>O<sub>3</sub> + H]<sup>+</sup> calcd, 386.211; found, 386.212. HPLC–UV (254 nm) = 99.2%, (280 nm) = 99.5%.</div></div><div id="sec4_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> (3<i>R</i>,4<i>R</i>)-1-[3-Amino-6-(2-hydroxyphenyl)pyridazine-4-yl]piperidine-3,4-diol (<i>R</i>,<i>R</i>-<b>36</b>) and (3<i>S</i>,4<i>S</i>)-1-[3-Amino-6-(2-hydroxyphenyl)pyridazine-4-yl]piperidine-3,4-diol (<i>S</i>,<i>S</i>-<b>37</b>)</h4><div class="NLM_p last"><i>Trans</i>-<b>20</b> (80 mg, 0.327 mmol) was reacted with (2-hydroxyphenyl)boronic acid <b>7</b> according to general procedure B. Purification was performed by flash C18 column chromatography using water/MeCN + 0.1% TFA (98/2–5/95) to yield the trans-mixture (34 mg, 0.112 mmol, 34%). Chiral preparative separation (1–2 mg/HPLC run) and chiral analytics were performed on an Agilent HPLC using a chiral <i>i</i>-Amylose-I LC column (see section: chiral separation of racemic mixture for details) to yield <i>R</i>,<i>R</i>-<b>36</b> and <i>S</i>,<i>S</i>-<b>37</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 7.91 (dd, <i>J</i> = 8.3, 1.3 Hz, 1H, phenol-C<i>H</i>-3), 7.50 (s, 1H, pyridazine-C<i>H</i>), 7.23 (dt, <i>J</i> = 8.1, 1.2 Hz, 1H, phenol-C<i>H</i>-5), 6.91–6.84 (m, 2H, phenol-C<i>H</i>-6/4), 6.27 (s, 2H, N<i>H</i><sub>2</sub>), 5.10 (d, <i>J</i> = 5.1 Hz, 1H, piperidine-O<i>H</i>-3), 4.91 (d, <i>J</i> = 3.6 Hz, 1H, piperidine-O<i>H</i>-4), 3.58–3.52 (m, 1H, piperidine-C<i>H</i>-3<sub>ax/eq</sub>), 3.49–3.44 (m, 1H, piperidine-C<i>H</i>-4<sub>ax/eq</sub>), 3.34–3.27 (m, 2H, piperidine-C<i>H</i>-2<sub>ax</sub>/6<sub>ax</sub>), 2.89–2.83 (m, 1H, piperidine-C<i>H</i>-6<sub>eq</sub>), 2.82–2.77 (m, 1H, piperidine-C<i>H</i>-2<sub>eq</sub>), 2.02–1.94 (m, 1H, piperidine-C<i>H</i>-5<sub>ax</sub>), 1.68–1.60 (m, 1H, piperidine-C<i>H</i>-5<sub>eq</sub>). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 158.6, 154.8, 153.2, 140.7, 130.1, 126.2, 118.5, 117.8, 117.4, 110.4, 70.1, 69.8, 52.1, 46.1, 30.3; HRMS-ESI (<i>m</i>/<i>z</i>): for [C<sub>15</sub>H<sub>18</sub>N<sub>4</sub>O<sub>3</sub> + H]<sup>+</sup> calcd, 303.145; found, 303.145.</div></div><div id="sec4_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> (3<i>R</i>,4<i>S</i>)-1-[3-Amino-6-(2-hydroxyphenyl)pyridazine-4-yl]piperidine-3,4-diol (<i>R</i>,<i>S</i>-<b>38</b>) and (3<i>S</i>,4<i>R</i>)-1-[3-Amino-6-(2-hydroxyphenyl)pyridazine-4-yl]piperidine-3,4-diol (<i>S</i>,<i>R</i>-<b>39</b>)</h4><div class="NLM_p last"><i>Cis</i>-<b>21</b> (36 mg, 0.147 mmol) was reacted with (2-hydroxyphenyl)boronic acid <b>7</b> according to general procedure B. Purification was performed by flash C18 column chromatography using water/MeCN + 0.1% TFA (98/2–5/95) to yield the cis-mixture (16 mg, 0.053 mmol, 36%). Chiral preparative separation (1–2 mg/HPLC run) and chiral analytics were performed on an Agilent HPLC using a chiral <i>i</i>-Amylose-I LC column (see section: chiral separation of racemic mixtures for details) to yield <i>R</i>,<i>S</i>-<b>38</b> and <i>S</i>,<i>R</i>-<b>39</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 7.90 (dd, <i>J</i> = 8.3, 1.3 Hz, 1H, phenol-C<i>H</i>-3), 7.49 (s, 1H, pyridazine-C<i>H</i>), 7.23 (dt, <i>J</i> = 15.3, 1.4 Hz, 1H, phenol-C<i>H</i>-5), 6.92–6.83 (m, 2H, phenol-C<i>H</i>-6/4), 6.31 (s, 2H, N<i>H</i><sub>2</sub>), 4.87 (d, <i>J</i> = 4.8 Hz, 1H, piperidine-O<i>H</i>-3), 4.65 (d, <i>J</i> = 2.6 Hz, 1H, piperidine-O<i>H</i>-4), 3.82–3.77 (m, 1H, piperidine-C<i>H</i>-3<sub>ax/eq</sub>), 3.71–3.67 (m, 1H, piperidine-C<i>H</i>-4<sub>ax/eq</sub>), 3.32–3.27 (m, 1H, piperidine-C<i>H</i>-2<sub>ax</sub>), 3.25–3.17 (m, 1H, piperidine-C<i>H</i>-6<sub>ax</sub>), 3.03–2.96 (m, 1H, piperidine-C<i>H</i>-6<sub>eq</sub>), 2.94–2.87 (m, 1H, piperidine-C<i>H</i>-2<sub>eq</sub>), 1.91–1.82 (m, 1H, piperidine-C<i>H</i>-5<sub>ax</sub>), 1.79–1.70 (m, 1H, piperidine-C<i>H</i>-5<sub>eq</sub>). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ [ppm] 158.6, 154.8, 153.2, 140.7, 130.1, 126.1, 118.5, 117.8, 117.4, 110.3, 67.9, 67.7, 51.4, 45.7, 29.4; HRMS-ESI (<i>m</i>/<i>z</i>): for [C<sub>15</sub>H<sub>18</sub>N<sub>4</sub>O<sub>3</sub> + H]<sup>+</sup> calcd, 303.145; found, 303.145.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> Chiral Separation of Cis/Trans Mixtures</h3><div class="NLM_p last">Chiral separation of compounds <i>R</i>,<i>R</i>-<b>36</b> and <i>S</i>,<i>S</i>-<b>37</b> or <i>R</i>,<i>S</i>-<b>38</b> and <i>S</i>,<i>R</i>-<b>39</b>, respectively, was performed on a preparative chiral HPLC column using an Agilent Technologies 1260 Infinity II LC System with Agilent InfinityLab single quadrupole LC/MSD system. Before chiral preparative separation, analytical runs were performed to evaluate a separation method using a Phenomenex Lux 5 μm <i>i</i>-Amylose 1 LC Column (250 × 4.6 mm). As appropriate separation method, a mixture of acetonitrile/methanol/glacial acetic acid/triethylamine (isocratic; 98:2:0.1:0.2; v/v/v/v) was chosen with a flow rate of 0.5 mL/min for 35 min. For preparative separation, a semi preparative chiral column from Phenomenex (Lux 5 μm <i>i</i>-Amylose 1 LC Column (250 × 10 mm)) was used. Acetonitrile/methanol/glacial acetic acid/triethylamine (isocratic; 98:2:0.1:0.2; v/v/v/v) was used as mobile phase with a flow rate of 2 mL/min over 35 min to elute the pure stereoisomers. The enantiomeric purity (95+%) was determined on the analytical column as described above (see the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_001.csv" class="ext-link">Supporting Information</a> data).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Protein Purification</h3><div class="NLM_p last">Bromodomains were produced following published protocols.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> The buffers for the final gel filtration purification step on Superdex 75 varied for the different bromodomain variants purified. SMARCA2/4: 300 mM NaCl, 10 mM HEPES (pH 7.5), 0.2 mM TCEP, 5% (v/v) glycerol. PB1(2/3/4): 150 mM NaCl, 20 mM HEPES (pH 7.4), 0.5 mM TCEP, 5% (v/v) glycerol. PB1(5): 500 mM NaCl, 10 mM HEPES (pH 7.5), 0.2 mM TCEP, 5% (v/v) glycerol. Proteins samples were flash frozen in liquid nitrogen and stored at −80 °C.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> Differential Scanning Fluorimetry</h3><div class="NLM_p last">Ligand binding to proteins was detected using DSF on an MX3005P qPCR system from Agilent Technologies or a QuantStudio 5 Real-Time PCR System (384-well plates) from Thermo Fisher Scientific as described elsewhere.<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> Briefly, proteins were buffered in 10 mM HEPES (pH 7.5), 150 mM NaCl and diluted to a final concentration of 2.5 μM (protein kinase domains) or 5.0 μM (bromodomains), and the fluorescent dye SYPRO Orange was added at a dilution of 1:1000. Compounds were dissolved in DMSO (50 mM) and added at a final concentration of 10 μM (protein kinases) or 20 μM (bromodomains) to 10 or 20 μL of protein-dye mix in a 96 or 384-well plate. Real-time melting curves were then recorded by heating the samples from 25 to 96 °C in 71 cycles (heating rate of 270 K/h, excitation/emission filters = 492/610 nm), and the melting point, <i>T</i><sub>m</sub>, was calculated using the Boltzmann equation.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> Isothermal Titration Calorimetry</h3><div class="NLM_p last">Binding constant (<i>K</i><sub>d</sub>), stoichiometry (<i>n</i>) and thermodynamic binding parameters (Δ<i>H</i>, Δ<i>S</i> and Δ<i>G</i>) of ligand–protein interactions were determined on a Nano-ITC from TA Instruments as described elsewhere.<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> Before measurement, the proteins were rebuffered to 20 mM HEPES (pH 7.5), 150 mM NaCl, 0.5 mM TCEP, and 5% glycerol by gel filtration. Compounds were diluted to a final concentration of 20–30 μM in buffer and placed into the sample cell. Proteins (125–200 μM) were added using an initial injection of 2 μL, followed by 27 injections of 8 μL at 15 or 25 °C. Collected data were corrected by subtraction of pure DMSO injection heats. Data were analyzed by assuming a sigmoidal dose–response relationship (four parameters). Errors of fits were calculated using standard deviation and a confidence interval of 95%.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> Crystallization and Structure Determination</h3><div class="NLM_p last">Crystals of bromodomain-inhibitor complexes were grown <i>via</i> the sitting drop vapor diffusion technique using a Mosquito crystallization robot (TTP Labtech) and SWISSCI 3-lens crystallization plates. Inhibitors were added at a threefold molar excess to the protein solution in gel filtration buffer (see above) prior to setting up the plates. Crystals were obtained under the following conditions. SMARCA4 with <b>23</b>: 20% medium-molecular-weight PEG smear, 0.1 M Tris pH 8.5 (20 °C); PB1(5) with <b>23</b>: 22% (w/v) PEG 3350, 0.2 M sodium malonate, 10% (v/v) ethylene glycol, 0.1 M bis-tris propane pH 7.1 (20 °C); PB1(5) with <b>22</b>: 30% PEG 5000 MME, 0.2 M (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 0.1 M MES pH 5.8 (4 °C); PB1(2) with <i>R</i>-<b>26</b>: 0.2 M sodium chloride, 25% (w/v) PEG3350, 0.1 M bis-tris pH 5.5 (4 °C); PB1(2) with <b>32</b>: 0.005 M zinc acetate, 20% (w/v) PEG 3350 (4 °C). Crystals were cryoprotected with mother liquor supplemented with 20% ethylene glycol and flash frozen in liquid nitrogen. X-ray data sets were collected at 100 K at BESSY beamline 14.2, Berlin, Germany, and beamline X06SA of the Swiss Light Source, Villigen, Switzerland. The diffraction data were integrated with either MOSFLM<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> or XDS,<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> and scaled with AIMLESS,<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> which is part of the CCP4 program suite.<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> The structures were solved by molecular replacement with PHASER,<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> using either PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKD">5DKD</a> (SMARCA4), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FH7">5FH7</a> (PB1(5)) or <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3LJW">3LJW</a> (PB1(2)) as a starting model. Refinement of the structures was then performed by using iterative cycles of manual model building in COOT<a onclick="showRef(event, 'ref82'); return false;" href="javascript:void(0);" class="ref ref82">(82)</a> and refinement in PHENIX.<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a> Dictionary files for the ligands were generated using the Grade Web Server (<a href="http://grade.globalphasing.org" class="extLink">http://grade.globalphasing.org</a>). A summary of the data collection and refinement statistics is given in the <a class="ref internalNav" href="#notes1" aria-label="Supporting Information">Supporting Information</a> Table S5. Structural figures were prepared using PyMOL (<a href="http://www.pymol.org" class="extLink">www.pymol.org</a>).</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> Cell Viability XTT Assay</h3><div class="NLM_p last">Cell viability was measured according to a published protocol.<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a> Breast (MCF-7, MDA-MB-231) or colon (HCT116, SW480) cancer cells were seeded in 96-well plates (2 × 10<sup>4</sup> cells/well) in 200 μL of Dulbecco’s modified Eagle medium (DMEM) supplemented with 10% fetal bovine serum (FBS) and 1% penicillin streptomycin solution (10,000 units penicillin and 10 mg of streptomycin/mL; PS) or McCoy’s 5A medium (supplemented with 10% FBS and 1% PS) and incubated overnight at 37 °C in 5% CO<sub>2</sub>. Cells were treated with vehicle (DMSO) or increasing doses (1.5, 3.1, 6.3, 12.5, 25, 50, and 100 μM) of compounds <i>R</i>-<b>28</b> and <b>22</b> for 72 h. Compounds were dispended using an Echo 550 liquid handler from Labcyte Inc. Before analysis, a mixture of 10 mL of media supplemented with 1 mg/mL XTT and 50 μL of 100 mM PMS in PBS (phosphate-buffered saline) was prepared. After compound treatment, the media was removed and 100 μL of the freshly prepared media mixture was added to each well. Cell viability/metabolic activity was measured on a PHERAstar FSX from BMG Labtech (absorbance: 450 nm, 660 nm) after 1 h. Blank signal was subtracted from results (media without cells).</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> Adipocyte Differentiation</h3><div class="NLM_p last">All adipogenesis-related experiments were performed as previously described.<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a> In general, 3T3-L1 cells were differentiated into adipocytes over 14 days according to the published protocol.<a onclick="showRef(event, 'ref86'); return false;" href="javascript:void(0);" class="ref ref86">(86)</a> For this, cells were seeded in 6-well plates [2.5 × 10<sup>5</sup> cells/well, DMEM supplemented with 10% newborn calf serum, 1% PS and 1% sodium pyruvate], and differentiation was induced after 48 h by the addition of a differentiation cocktail containing 1 μg/mL insulin, 0.25 μM dexamethasone and 0.5 mM IBMX in DMEM supplemented with 10% fetal calf serum (FCS), 1% sodium pyruvate, and 1% PS. In addition, the PPARγ agonist rosiglitazone (2 μM) was added to stimulate and enhance differentiation. After 2 days, the medium was replaced by DMEM supplemented with 10% FCS, 1% sodium pyruvate, 1% PS and 1 μg/mL insulin, and the culture was grown for 2 more days. Then, cells were kept for lipid droplet accumulation in DMEM supplemented with 10% FCS, 1% sodium pyruvate and 1% PS until day 14. Compounds <b>22</b>, <i>R</i>-<b>28</b>, <b>32</b>, and the controls PFI-3 and <b>35</b> were added at 1 and 5 μM during seeding and kept at this concentration throughout the complete differentiation process (day 1–14). Compounds were refreshed after 48 and 96 h.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> Oil Red Staining</h3><div class="NLM_p last">Differentiated 3T3-L1 cells were washed with PBS and subsequently fixed for 60 min with formaldehyde (4% in PBS). Afterward, cells were thoroughly rinsed with 60% isopropanol and incubated with oil red O solution [0.3% in 60% (v/v) isopropanol] for 120 min. This was followed by the removal of the staining solution and thorough rinsing of the cell layer with ultrapure water. Finally, the plates were dried, and photos of the stained cell layers were taken.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> Analysis of Adipogenesis-Related Genes by qPCR</h3><div class="NLM_p last">3T3-L1 cells were differentiated into adipocytes for 7 days as described above. Samples were taken at indicated times and concentrations as provided in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b. After harvesting the samples and washing the cell pellets with PBS, the 3T3-L1 cells were lyzed using TRIzol reagent (Ambion Life Technologies). Subsequently, mRNA isolation was performed according to the manufacturers protocol. Remaining DNA impurities were digested using the DNAse I kit from Thermo Scientific, and mRNA integrity was confirmed on an agarose gel. In order to obtain cDNA, reverse transcription was performed using the High Capacity RNA-to-cDNA kit from Applied Biosystems according to the manufacturer’s protocol. qPCR was performed with Power SYBR Green Master Mix (Applied Biosystems) with a StepOnePlus Real-Time PCR system from Applied Biosystems using specific primers for murine as indicated in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_002.pdf" class="ext-link">Supporting Information</a> Table S4. Relative mRNA expression was determined by the 2<sup>–ΔΔ<i>CT</i></sup> method compared to murine non-POU domain-containing octamer-binding protein (NONO). For incubation times and compound concentrations, see <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b. All samples were measured in triplicates, and experiments were repeated independently at least three times.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i54"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01242" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26647" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26647" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01242?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01242</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings with activity data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_001.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Supporting analytical chemistry data for intermediates and final products and additional tables and figures (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_002.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_001.csv">jm0c01242_si_001.csv (2.56 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_002.pdf">jm0c01242_si_002.pdf (7.03 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors of the bromodomain-ligand complexes are available in the Protein Data Bank (PDB) under accession codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZS2">6ZS2</a> (SMARCA4-<b>23</b> complex), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZS3">6ZS3</a> (PB1(5)-<b>23</b> complex), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZS4">6ZS4</a> (PB1(5)-<b>22</b> complex), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZN6">6ZN6</a> (PB1(2)-<i>R</i>-<b>26</b> complex), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZNV">6ZNV</a> (PB1(2)-<b>32</b> complex).</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c01242" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04404" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04404" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andreas C. Joerger</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Goethe University
Frankfurt, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium (SGC), Buchmann Institute
for Molecular Life Sciences (BMLS), Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">German
Translational Cancer Network (DKTK), Frankfurt/Mainz Site, 60438 Frankfurt
am Main, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1232-0138" title="Orcid link">http://orcid.org/0000-0002-1232-0138</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#cca6a3a9beaba9be8cbca4adbea1afa4a9a1e2b9a2a5e1aabeada2a7aab9beb8e2a8a9"><span class="__cf_email__" data-cfemail="caa0a5afb8adafb88abaa2abb8a7a9a2afa7e4bfa4a3e7acb8aba4a1acbfb8bee4aeaf">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefan Knapp</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Goethe University
Frankfurt, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium (SGC), Buchmann Institute
for Molecular Life Sciences (BMLS), Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">German
Translational Cancer Network (DKTK), Frankfurt/Mainz Site, 60438 Frankfurt
am Main, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Frankfurt
Cancer Institute (FCI), Paul-Ehrlich-Str. 42-44, 60596 Frankfurt am Main, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5995-6494" title="Orcid link">http://orcid.org/0000-0001-5995-6494</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#aec5c0cfdedeeedec6cfdcc3cdc6cbc380dbc0c783c8dccfc0c5c8dbdcda80cacb"><span class="__cf_email__" data-cfemail="244f4a45545464544c455649474c41490a514a4d094256454a4f425156500a4041">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marek Wanior</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Goethe University
Frankfurt, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium (SGC), Buchmann Institute
for Molecular Life Sciences (BMLS), Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9569-4487" title="Orcid link">http://orcid.org/0000-0001-9569-4487</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Franziska Preuss</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Goethe University
Frankfurt, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium (SGC), Buchmann Institute
for Molecular Life Sciences (BMLS), Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaomin Ni</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Goethe University
Frankfurt, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium (SGC), Buchmann Institute
for Molecular Life Sciences (BMLS), Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andreas Krämer</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Goethe University
Frankfurt, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium (SGC), Buchmann Institute
for Molecular Life Sciences (BMLS), Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Frankfurt
Cancer Institute (FCI), Paul-Ehrlich-Str. 42-44, 60596 Frankfurt am Main, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sebastian Mathea</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Goethe University
Frankfurt, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium (SGC), Buchmann Institute
for Molecular Life Sciences (BMLS), Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8500-4569" title="Orcid link">http://orcid.org/0000-0001-8500-4569</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tamara Göbel</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Goethe University
Frankfurt, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Heidenreich</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Goethe University
Frankfurt, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium (SGC), Buchmann Institute
for Molecular Life Sciences (BMLS), Max-von-Laue-Str. 15, 60438 Frankfurt am Main, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Svenja Simonyi</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Goethe University
Frankfurt, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Astrid S. Kahnt</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Goethe University
Frankfurt, Max-von-Laue-Str. 9, 60438 Frankfurt am Main, Germany</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>M.W. synthesized all compounds and evaluated them in cellular experiments. M.W. and F.P. performed biophysical studies. F.P., X.N., A.K., S.M., D.H., and A.C.J. performed crystallographic studies. M.W., S.S., T.G., and A.S.K. performed and analyzed adipocyte stem cell differentiation experiments. M.W., A.C.J., and S.K. designed the study and wrote the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i57">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22110" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22110" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">M.W. is grateful for support by the Research Training Group Translational Research Innovation—Pharma (TRIP), supported by the Else Kröner-Fresenius Foundation (EKFS). We are also grateful for support by the SGC, a registered charity (no. 1097737) that receives funds from; AbbVie, Bayer AG, Boehringer Ingelheim, the Canada Foundation for Innovation, Eshelman Institute for Innovation, Genentech, Genome Canada through Ontario Genomics Institute [OGI-196], EU/EFPIA/OICR/McGill/KTH/Diamond, Innovative Medicines Initiative 2 Joint Undertaking [EUbOPEN grant 875510], Janssen, Merck KGaA (aka EMD in Canada and US), Merck & Co (aka MSD outside Canada and US), Pfizer, the São Paulo Research Foundation-FAPESP, Takeda and Wellcome [106169/ZZ14/Z]. S.K. and A.C.J. are grateful for support by the German cancer network DKTK and the FCI. A.C.J. is supported by German Research Foundation (DFG) grant JO 1473/1-3. We thank the NIH for screening our compounds in the NCI-60 human tumor cell lines panel. The data collection at SLS was supported by funding from the European Union’s Horizon 2020 research and innovation program under grant agreement number 730872, project CALIPSOplus. We thank the staff at BESSY beamline 14.2 and beamline X06SA of the Swiss Light Source for assistance during data collection.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">BAF</td><td class="NLM_def"><p class="first last">BRG1- or BRM-associated factors</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extra-terminal motif</p></td></tr><tr><td class="NLM_term">BRD</td><td class="NLM_def"><p class="first last">bromodomain</p></td></tr><tr><td class="NLM_term">BRG1</td><td class="NLM_def"><p class="first last">brahma-related gene-1</p></td></tr><tr><td class="NLM_term">BRM</td><td class="NLM_def"><p class="first last">brahma</p></td></tr><tr><td class="NLM_term">C/EBPα/β/δ</td><td class="NLM_def"><p class="first last">CCAAT/enhancer-binding protein α/β/δ</p></td></tr><tr><td class="NLM_term">DSF</td><td class="NLM_def"><p class="first last">differential scanning fluorimetry</p></td></tr><tr><td class="NLM_term">FABP4</td><td class="NLM_def"><p class="first last">fatty acid-binding protein 4</p></td></tr><tr><td class="NLM_term">GLUT4</td><td class="NLM_def"><p class="first last">glucose transporter type 4</p></td></tr><tr><td class="NLM_term">IBMX</td><td class="NLM_def"><p class="first last">isobutylmethylxanthine</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">KLF5</td><td class="NLM_def"><p class="first last">Krüppel-like factor 5</p></td></tr><tr><td class="NLM_term">Krox20</td><td class="NLM_def"><p class="first last">early growth response protein 2</p></td></tr><tr><td class="NLM_term">PBAF</td><td class="NLM_def"><p class="first last">polybromo-associated BAF</p></td></tr><tr><td class="NLM_term">PBRM1/PB1</td><td class="NLM_def"><p class="first last">polybromo 1</p></td></tr><tr><td class="NLM_term">PMS</td><td class="NLM_def"><p class="first last"><i>N</i>-methyl dibenzopyrazine methyl sulfate</p></td></tr><tr><td class="NLM_term">PPARγ</td><td class="NLM_def"><p class="first last">peroxisome proliferator-activated receptor γ</p></td></tr><tr><td class="NLM_term">PROTACs</td><td class="NLM_def"><p class="first last">proteolysis targeting chimeras</p></td></tr><tr><td class="NLM_term">p300</td><td class="NLM_def"><p class="first last">histone acetyltransferase p300</p></td></tr><tr><td class="NLM_term">qPCR</td><td class="NLM_def"><p class="first last">quantitative real-time polymerase chain reaction</p></td></tr><tr><td class="NLM_term">SMARCA2/4 or SMARCB1</td><td class="NLM_def"><p class="first last">SWI/SNF-related matrix-associated actin-dependent regulator of chromatin, subfamily A2/4 or subfamily B1</p></td></tr><tr><td class="NLM_term">SWI/SNF</td><td class="NLM_def"><p class="first last">switch/sucrose nonfermentable</p></td></tr><tr><td class="NLM_term">XTT</td><td class="NLM_def"><p class="first last">2,3-bis-[(2-methoxy-4-nitro-5-sulfophenyl)-2<i>H</i>-tetrazolium-5-carboxanilide salt]</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i59">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03784" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03784" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 87 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span> <span> </span><span class="NLM_article-title">Cancer epigenetics: from mechanism to therapy</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>150</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.06.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2Fj.cell.2012.06.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=22770212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvFGjsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2012&pages=12-27&author=M.+A.+Dawsonauthor=T.+Kouzarides&title=Cancer+epigenetics%3A+from+mechanism+to+therapy&doi=10.1016%2Fj.cell.2012.06.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer Epigenetics: From Mechanism to Therapy</span></div><div class="casAuthors">Dawson, Mark A.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-27</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The epigenetic regulation of DNA-templated processes has been intensely studied over the last 15 years.  DNA methylation, histone modification, nucleosome remodeling, and RNA-mediated targeting regulate many biol. processes that are fundamental to the genesis of cancer.  Here, we present the basic principles behind these epigenetic pathways and highlight the evidence suggesting that their misregulation can culminate in cancer.  This information, along with the promising clin. and preclin. results seen with epigenetic drugs against chromatin regulators, signifies that it is time to embrace the central role of epigenetics in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuyeeliq_34rVg90H21EOLACvtfcHk0ljWUaRL15WPOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvFGjsLg%253D&md5=bc0176364205a2e66e1971fe789d36ae</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.06.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.06.013%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DCancer%2520epigenetics%253A%2520from%2520mechanism%2520to%2520therapy%26jtitle%3DCell%26date%3D2012%26volume%3D150%26spage%3D12%26epage%3D27%26doi%3D10.1016%2Fj.cell.2012.06.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klemm, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shipony, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenleaf, W. J.</span></span> <span> </span><span class="NLM_article-title">Chromatin accessibility and the regulatory epigenome</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">207</span>– <span class="NLM_lpage">220</span>, <span class="refDoi"> DOI: 10.1038/s41576-018-0089-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fs41576-018-0089-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=30675018" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmtFGqsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=207-220&author=S.+L.+Klemmauthor=Z.+Shiponyauthor=W.+J.+Greenleaf&title=Chromatin+accessibility+and+the+regulatory+epigenome&doi=10.1038%2Fs41576-018-0089-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin accessibility and the regulatory epigenome</span></div><div class="casAuthors">Klemm, Sandy L.; Shipony, Zohar; Greenleaf, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">207-220</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Phys. access to DNA is a highly dynamic property of chromatin that plays an essential role in establishing and maintaining cellular identity.  The organization of accessible chromatin across the genome reflects a network of permissible phys. interactions through which enhancers, promoters, insulators and chromatin-binding factors cooperatively regulate gene expression.  This landscape of accessibility changes dynamically in response to both external stimuli and developmental cues, and emerging evidence suggests that homeostatic maintenance of accessibility is itself dynamically regulated through a competitive interplay between chromatin-binding factors and nucleosomes.  In this Review, we examine how the accessible genome is measured and explore the role of transcription factors in initiating accessibility remodeling; our goal is to illustrate how chromatin accessibility defines regulatory elements within the genome and how these epigenetic features are dynamically established to control gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoP-5OBDLCo5LVg90H21EOLACvtfcHk0lgR4CFv3rTpsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmtFGqsbk%253D&md5=60707e7c4467e9f7b0edbba256e66832</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fs41576-018-0089-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41576-018-0089-8%26sid%3Dliteratum%253Aachs%26aulast%3DKlemm%26aufirst%3DS.%2BL.%26aulast%3DShipony%26aufirst%3DZ.%26aulast%3DGreenleaf%26aufirst%3DW.%2BJ.%26atitle%3DChromatin%2520accessibility%2520and%2520the%2520regulatory%2520epigenome%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2019%26volume%3D20%26spage%3D207%26epage%3D220%26doi%3D10.1038%2Fs41576-018-0089-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ornaghi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballario, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuhaus, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filetici, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Travers, A. A.</span></span> <span> </span><span class="NLM_article-title">The structural basis for the recognition of acetylated histone H4 by the bromodomain of histone acetyltransferase gcn5p</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">6141</span>– <span class="NLM_lpage">6149</span>, <span class="refDoi"> DOI: 10.1093/emboj/19.22.6141</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1093%2Femboj%2F19.22.6141" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=11080160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD3cXovVaitro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=6141-6149&author=D.+J.+Owenauthor=P.+Ornaghiauthor=J.+C.+Yangauthor=N.+Loweauthor=P.+R.+Evansauthor=P.+Ballarioauthor=D.+Neuhausauthor=P.+Fileticiauthor=A.+A.+Travers&title=The+structural+basis+for+the+recognition+of+acetylated+histone+H4+by+the+bromodomain+of+histone+acetyltransferase+gcn5p&doi=10.1093%2Femboj%2F19.22.6141"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The structural basis for the recognition of acetylated histone H4 by the bromodomain of histone acetyltransferase Gcn5p</span></div><div class="casAuthors">Owen, David J.; Ornaghi, Prisca; Yang, Ji-Chun; Lowe, Nicholas; Evans, Philip R.; Ballario, Paola; Neuhaus, David; Filetici, Patrizia; Travers, Andrew A.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6141-6149</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The bromodomain is an ∼110 amino acid module found in histone acetyltransferases and the ATPase component of certain nucleosome remodeling complexes.  We report the crystal structure at 1.9 Å resoln. of the Saccharomyces cerevisiae Gcn5p bromodomain complexed with a peptide corresponding to residues 15-29 of histone H4 acetylated at the ζ-N of lysine 16.  We show that this bromodomain preferentially binds to peptides contg. an N-acetyl lysine residue.  Only residues 16-19 of the acetylated peptide interact with the bromodomain.  The primary interaction is the N-acetyl lysine binding in a cleft with the specificity provided by the interaction of the amide nitrogen of a conserved asparagine with the oxygen of the acetyl carbonyl group.  A network of water-mediated H-bonds with protein main chain carbonyl groups at the base of the cleft contributes to the binding.  Addnl. side chain binding occurs on a shallow depression that is hydrophobic at one end and can accommodate charge interactions at the other.  These findings suggest that the Gcn5p bromodomain may discriminate between different acetylated lysine residues depending on the context in which they are displayed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp0LsEWlzQl7Vg90H21EOLACvtfcHk0lgR4CFv3rTpsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXovVaitro%253D&md5=1cc0a6434f61362e83830becc7bdc594</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F19.22.6141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F19.22.6141%26sid%3Dliteratum%253Aachs%26aulast%3DOwen%26aufirst%3DD.%2BJ.%26aulast%3DOrnaghi%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DJ.%2BC.%26aulast%3DLowe%26aufirst%3DN.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DBallario%26aufirst%3DP.%26aulast%3DNeuhaus%26aufirst%3DD.%26aulast%3DFiletici%26aufirst%3DP.%26aulast%3DTravers%26aufirst%3DA.%2BA.%26atitle%3DThe%2520structural%2520basis%2520for%2520the%2520recognition%2520of%2520acetylated%2520histone%2520H4%2520by%2520the%2520bromodomain%2520of%2520histone%2520acetyltransferase%2520gcn5p%26jtitle%3DEMBO%2520J.%26date%3D2000%26volume%3D19%26spage%3D6141%26epage%3D6149%26doi%3D10.1093%2Femboj%2F19.22.6141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.-P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+M%C3%BCllerauthor=T.+Pawsonauthor=A.-C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0lgR4CFv3rTpsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Bromodomains as therapeutic targets</span>. <i>Expert Rev. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>13</i></span>, <span class="NLM_elocation-id">e29</span> <span class="refDoi"> DOI: 10.1017/s1462399411001992</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1017%2FS1462399411001992" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=21933453" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1OhurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&author=S.+Mullerauthor=P.+Filippakopoulosauthor=S.+Knapp&title=Bromodomains+as+therapeutic+targets&doi=10.1017%2Fs1462399411001992"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomains as therapeutic targets</span></div><div class="casAuthors">Muller, Susanne; Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Expert Reviews in Molecular Medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e29/1-e29/21</span>CODEN:
                <span class="NLM_cas:coden">ERMMFS</span>;
        ISSN:<span class="NLM_cas:issn">1462-3994</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">A review.  Acetylation of lysine residues is a post-translational modification with broad relevance to cellular signalling and disease biol.  Enzymes that 'write' (histone acetyltransferases, HATs) and 'erase' (histone deacetylases, HDACs) acetylation sites are an area of extensive research in current drug development, but very few potent inhibitors that modulate the 'reading process' mediated by acetyl lysines have been described.  The principal readers of ε-N-acetyl lysine (Kac) marks are bromodomains (BRDs), which are a diverse family of evolutionary conserved protein-interaction modules.  The conserved BRD fold contains a deep, largely hydrophobic acetyl lysine binding site, which represents an attractive pocket for the development of small, pharmaceutically active mols.  Proteins that contain BRDs have been implicated in the development of a large variety of diseases.  Recently, two highly potent and selective inhibitors that target BRDs of the BET (bromodomains and extra-terminal) family provided compelling data supporting targeting of these BRDs in inflammation and in an aggressive type of squamous cell carcinoma.  It is likely that BRDs will emerge alongside HATs and HDACs as interesting targets for drug development for the large no. of diseases that are caused by aberrant acetylation of lysine residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSuKacpsr6xLVg90H21EOLACvtfcHk0lggZbyWmAqBeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1OhurjI&md5=dc9383afcbe6aedc265041c21213aa0b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1017%2FS1462399411001992&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS1462399411001992%26sid%3Dliteratum%253Aachs%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DBromodomains%2520as%2520therapeutic%2520targets%26jtitle%3DExpert%2520Rev.%2520Mol.%2520Med.%26date%3D2011%26volume%3D13%26doi%3D10.1017%2Fs1462399411001992" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">356</span>, <span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0lggZbyWmAqBeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lixmqYXSGEefw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moustakim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, P. G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Chemical probes and inhibitors of bromodomains outside the BET family</span>. <i>Medchemcomm</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">2246</span>– <span class="NLM_lpage">2264</span>, <span class="refDoi"> DOI: 10.1039/c6md00373g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1039%2Fc6md00373g" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=29170712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVOhsbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=2246-2264&author=M.+Moustakimauthor=P.+G.+K.+Clarkauthor=D.+A.+Hayauthor=D.+J.+Dixonauthor=P.+E.+Brennan&title=Chemical+probes+and+inhibitors+of+bromodomains+outside+the+BET+family&doi=10.1039%2Fc6md00373g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical probes and inhibitors of bromodomains outside the BET family</span></div><div class="casAuthors">Moustakim, Moses; Clark, Peter G. K.; Hay, Duncan A.; Dixon, Darren J.; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2246-2264</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">In the last five years, the development of inhibitors of bromodomains has emerged as an area of intensive worldwide research.  Emerging evidence has implicated a no. of non-BET bromodomains in the onset and progression of diseases such as cancer, HIV infection and inflammation.  The development and use of small mol. chem. probes has been fundamental to pre-clin. evaluation of bromodomains as targets.  Recent efforts are described highlighting the development of potent, selective and cell active non-BET bromodomain inhibitors and their therapeutic potential.  Over half of typical bromodomains now have reported ligands, but those with atypical binding site residues remain resistant to chem. probe discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYJvhj3Z0Ry7Vg90H21EOLACvtfcHk0lixmqYXSGEefw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVOhsbbF&md5=df4c5718b15ff7d401a0cd9974abc236</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1039%2Fc6md00373g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc6md00373g%26sid%3Dliteratum%253Aachs%26aulast%3DMoustakim%26aufirst%3DM.%26aulast%3DClark%26aufirst%3DP.%2BG.%2BK.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DDixon%26aufirst%3DD.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DChemical%2520probes%2520and%2520inhibitors%2520of%2520bromodomains%2520outside%2520the%2520BET%2520family%26jtitle%3DMedchemcomm%26date%3D2016%26volume%3D7%26spage%3D2246%26epage%3D2264%26doi%3D10.1039%2Fc6md00373g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">Discovery of chemical inhibitors of human bromodomains</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">11625</span>– <span class="NLM_lpage">11668</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00205</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00205" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs12qurzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2015&pages=11625-11668&author=G.+Zhangauthor=S.+G.+Smithauthor=M.-M.+Zhou&title=Discovery+of+chemical+inhibitors+of+human+bromodomains&doi=10.1021%2Facs.chemrev.5b00205"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Chemical Inhibitors of Human Bromodomains</span></div><div class="casAuthors">Zhang, Guangtao; Smith, Steven G.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11625-11668</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The structure-guided drug design strategy has played an important role in the recent discovery and development of bromodomain inhibitors that has yielded a multitude of chemotypes as described in this review.  Going forward, we expect that this structure-based approach will continue to guide medicinal chemists in lead optimization of bromodomain inhibitors and facilitate the efforts to develop new inhibitors for less druggable bromodomains.  Collectively, these emerging new chem. inhibitors will empower our ongoing investigation to better understand the roles of bromodomain proteins in control of gene transcription in human biol. and disease pathol.  These small-mol. inhibitors can also be developed into new epigenetic therapies for new effective treatments of various human diseases including cancer and inflammatory disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHbsgfBguOabVg90H21EOLACvtfcHk0ljwc0h9n_N6nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs12qurzP&md5=4014c771384b4a768a80644af22c4485</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00205%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DS.%2BG.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DDiscovery%2520of%2520chemical%2520inhibitors%2520of%2520human%2520bromodomains%26jtitle%3DChem.%2520Rev.%26date%3D2015%26volume%3D115%26spage%3D11625%26epage%3D11668%26doi%3D10.1021%2Facs.chemrev.5b00205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Euskirchen, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auerbach, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gianoulis, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozowsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhardwaj, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstein, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, M.</span></span> <span> </span><span class="NLM_article-title">Diverse roles and interactions of the SWI/SNF chromatin remodeling complex revealed using global approaches</span>. <i>PLoS Genet.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e1002008</span> <span class="refDoi"> DOI: 10.1371/journal.pgen.1002008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1371%2Fjournal.pgen.1002008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=21408204" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjt1Kiuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&author=G.+M.+Euskirchenauthor=R.+K.+Auerbachauthor=E.+Davidovauthor=T.+A.+Gianoulisauthor=G.+Zhongauthor=J.+Rozowskyauthor=N.+Bhardwajauthor=M.+B.+Gersteinauthor=M.+Snyder&title=Diverse+roles+and+interactions+of+the+SWI%2FSNF+chromatin+remodeling+complex+revealed+using+global+approaches&doi=10.1371%2Fjournal.pgen.1002008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Diverse roles and interactions of the SWI/SNF chromatin remodeling complex revealed using global approaches</span></div><div class="casAuthors">Euskirchen, Ghia M.; Auerbach, Raymond K.; Davidov, Eugene; Gianoulis, Tara A.; Zhong, Guoneng; Rozowsky, Joel; Bhardwaj, Nitin; Gerstein, Mark B.; Snyder, Michael</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Genetics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e1002008</span>CODEN:
                <span class="NLM_cas:coden">PGLEB5</span>;
        ISSN:<span class="NLM_cas:issn">1553-7404</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">A systems understanding of nuclear organization and events is crit. for detg. how cells divide, differentiate, and respond to stimuli and for identifying the causes of diseases.  Chromatin remodeling complexes such as SWI/SNF have been implicated in a wide variety of cellular processes including gene expression, nuclear organization, centromere function, and chromosomal stability, and mutations in SWI/SNF components have been linked to several types of cancer.  To better understand the biol. processes in which chromatin remodeling proteins participate, we globally mapped binding regions for several components of the SWI/SNF complex throughout the human genome using ChIP-Seq.  SWI/SNF components were found to lie near regulatory elements integral to transcription (e.g. 5' ends, RNA Polymerases II and III, and enhancers) as well as regions crit. for chromosome organization (e.g. CTCF, lamins, and DNA replication origins).  Interestingly we also find that certain configurations of SWI/SNF subunits are assocd. with transcripts that have higher levels of expression, whereas other configurations of SWI/SNF factors are assocd. with transcripts that have lower levels of expression.  To further elucidate the assocn. of SWI/SNF subunits with each other as well as with other nuclear proteins, we also analyzed SWI/SNF immunopptd. complexes by mass spectrometry.  Individual SWI/SNF factors are assocd. with their own family members, as well as with cellular constituents such as nuclear matrix proteins, key transcription factors, and centromere components, implying a ubiquitous role in gene regulation and nuclear function.  We find an overrepresentation of both SWI/SNF-assocd. regions and proteins in cell cycle and chromosome organization.  Taken together the results from our ChIP and immunopptn. expts. suggest that SWI/SNF facilitates gene regulation and genome function more broadly and through a greater diversity of interactions than previously appreciated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhpZiFTTXYj7Vg90H21EOLACvtfcHk0ljwc0h9n_N6nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjt1Kiuro%253D&md5=cfd31a2824d8ca9180783224fca4b1e0</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pgen.1002008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pgen.1002008%26sid%3Dliteratum%253Aachs%26aulast%3DEuskirchen%26aufirst%3DG.%2BM.%26aulast%3DAuerbach%26aufirst%3DR.%2BK.%26aulast%3DDavidov%26aufirst%3DE.%26aulast%3DGianoulis%26aufirst%3DT.%2BA.%26aulast%3DZhong%26aufirst%3DG.%26aulast%3DRozowsky%26aufirst%3DJ.%26aulast%3DBhardwaj%26aufirst%3DN.%26aulast%3DGerstein%26aufirst%3DM.%2BB.%26aulast%3DSnyder%26aufirst%3DM.%26atitle%3DDiverse%2520roles%2520and%2520interactions%2520of%2520the%2520SWI%252FSNF%2520chromatin%2520remodeling%2520complex%2520revealed%2520using%2520global%2520approaches%26jtitle%3DPLoS%2520Genet.%26date%3D2011%26volume%3D7%26doi%3D10.1371%2Fjournal.pgen.1002008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kadoch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crabtree, G. R.</span></span> <span> </span><span class="NLM_article-title">Mammalian SWI/SNF chromatin remodeling complexes and cancer: Mechanistic insights gained from human genomics</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1</i></span>, <span class="NLM_elocation-id">e1500447</span> <span class="refDoi"> DOI: 10.1126/sciadv.1500447</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1126%2Fsciadv.1500447" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=26601204" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC28XnslOrsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&author=C.+Kadochauthor=G.+R.+Crabtree&title=Mammalian+SWI%2FSNF+chromatin+remodeling+complexes+and+cancer%3A+Mechanistic+insights+gained+from+human+genomics&doi=10.1126%2Fsciadv.1500447"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Mammalian SWI/SNF chromatin remodelling complexes and cancer: Mechanistic insights gained from human genomics</span></div><div class="casAuthors">Kadoch, Cigall; Crabtree, Gerald R.</div><div class="citationInfo"><span class="NLM_cas:title">Science Advances</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e1500447/1-e1500447/17</span>CODEN:
                <span class="NLM_cas:coden">SACDAF</span>;
        ISSN:<span class="NLM_cas:issn">2375-2548</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Over the past 4 years, nearly 100 exome sequencing studies have revealed the high frequency of mutations in the genes encoding the subunits of ATP-dependent chromatin remodelers in human cancer.  Most of these mutations are within the genes encoding subunits of the BAF (Brg/Brahma-assocd. factors) or mSWI/SNF complex, which is one of two dozen predicted ATP-dependent chromatin remodeling complexes in mammals.  Considering BAF complexes as a single entity, the 15 subunits encoded by 29 genes are mutated in >20% of human cancer, across a broad range of tumor types.  These observations demonstrate that there is little redundancy in the oncogenic function of BAF complexes with the other remodeling complexes, underscoring their unique roles.  Several important conclusionsemerge fromthese genomic data: specific subunits appear to bemutated in specific cancers, highlighting tissue-specific protective roles; mutations can function as tumor suppressors or oncogenes;mutations can be homozygous or, more commonly, heterozygous, implying their dosage-sensitive roles in an unknown yet fundamental process used to suppress the genesis of cancer.  These new human genetic findings paired with biochem. studies are challenging old ideas on how chromatin remodeling complexes function, generating new hypotheseswith respect to their normal and oncogenicmechanisms and highlighting potential avenues for therapeutic intervention in human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUweEWQp4d2LVg90H21EOLACvtfcHk0ljwc0h9n_N6nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnslOrsrY%253D&md5=e4f8ae87a6c179d61091582f03885f5b</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.1500447&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.1500447%26sid%3Dliteratum%253Aachs%26aulast%3DKadoch%26aufirst%3DC.%26aulast%3DCrabtree%26aufirst%3DG.%2BR.%26atitle%3DMammalian%2520SWI%252FSNF%2520chromatin%2520remodeling%2520complexes%2520and%2520cancer%253A%2520Mechanistic%2520insights%2520gained%2520from%2520human%2520genomics%26jtitle%3DSci.%2520Adv.%26date%3D2015%26volume%3D1%26doi%3D10.1126%2Fsciadv.1500447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alfert, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerl, K.</span></span> <span> </span><span class="NLM_article-title">The BAF complex in development and disease</span>. <i>Epigenet. Chromatin</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">19</span>, <span class="refDoi"> DOI: 10.1186/s13072-019-0264-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1186%2Fs13072-019-0264-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=30898143" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2019&pages=19&author=A.+Alfertauthor=N.+Morenoauthor=K.+Kerl&title=The+BAF+complex+in+development+and+disease&doi=10.1186%2Fs13072-019-0264-y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1186%2Fs13072-019-0264-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13072-019-0264-y%26sid%3Dliteratum%253Aachs%26aulast%3DAlfert%26aufirst%3DA.%26aulast%3DMoreno%26aufirst%3DN.%26aulast%3DKerl%26aufirst%3DK.%26atitle%3DThe%2520BAF%2520complex%2520in%2520development%2520and%2520disease%26jtitle%3DEpigenet.%2520Chromatin%26date%3D2019%26volume%3D12%26spage%3D19%26doi%3D10.1186%2Fs13072-019-0264-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hodges, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crabtree, G. R.</span></span> <span> </span><span class="NLM_article-title">The many roles of BAF (mSWI/SNF) and PBAF complexes in cancer</span>. <i>Cold Spring Harbor Perspect. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">a026930</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a026930</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1101%2Fcshperspect.a026930" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=27413115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1ygsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=a026930&author=C.+Hodgesauthor=J.+G.+Kirklandauthor=G.+R.+Crabtree&title=The+many+roles+of+BAF+%28mSWI%2FSNF%29+and+PBAF+complexes+in+cancer&doi=10.1101%2Fcshperspect.a026930"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The many roles of BAF (mSWI/SNF) and PBAF complexes in cancer</span></div><div class="casAuthors">Hodges, Courtney; Kirkland, Jacob G.; Crabtree, Gerald R.</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">a026930/1-a026930/25</span>CODEN:
                <span class="NLM_cas:coden">CSHPFV</span>;
        ISSN:<span class="NLM_cas:issn">2157-1422</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">During the last decade, a host of epigenetic mechanisms were found to contribute to cancer and other human diseases.  Several genomic studies have revealed that ∼ 20% of malignancies have alterations of the subunits of polymorphic BRG-/BRM-assocd. factor (BAF) and Polybromo-assocd. BAF (PBAF) complexes, making them among the most frequently mutated complexes in cancer.  Recurrent mutations arise in genes encoding several BAF/ PBAF subunits, including ARID1A, ARID2, PBRM1, SMARCA4, and SMARCB1.  These subunits share some degree of conservation with subunits from related ATP (ATP)-dependent chromatin remodeling complexes in model organisms, in which a large body of work provides insight into their roles in cancer.  Here, we review the roles of BAF- and PBAF-like complexes in these organisms, and relate these findings to recent discoveries in cancer epigenomics.  We review several roles of BAF and PBAF complexes in cancer, including transcriptional regulation, DNA repair, and regulation of chromatin architecture and topol.  More recent results highlight the need for new techniquesto study these complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCynky-OxCc7Vg90H21EOLACvtfcHk0ljkrLTbd53qEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1ygsLs%253D&md5=71976f9f18bfad4c44fe26991679a194</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a026930&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a026930%26sid%3Dliteratum%253Aachs%26aulast%3DHodges%26aufirst%3DC.%26aulast%3DKirkland%26aufirst%3DJ.%2BG.%26aulast%3DCrabtree%26aufirst%3DG.%2BR.%26atitle%3DThe%2520many%2520roles%2520of%2520BAF%2520%2528mSWI%252FSNF%2529%2520and%2520PBAF%2520complexes%2520in%2520cancer%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Med.%26date%3D2016%26volume%3D6%26spage%3Da026930%26doi%3D10.1101%2Fcshperspect.a026930" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
La Serna, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbalzano, A. N.</span></span> <span> </span><span class="NLM_article-title">Mammalian SWI/SNF complexes promote MyoD-mediated muscle differentiation</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">187</span>– <span class="NLM_lpage">190</span>, <span class="refDoi"> DOI: 10.1038/84826</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2F84826" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=11175787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhtFGkur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2001&pages=187-190&author=I.+L.+de%0ALa+Sernaauthor=K.+A.+Carlsonauthor=A.+N.+Imbalzano&title=Mammalian+SWI%2FSNF+complexes+promote+MyoD-mediated+muscle+differentiation&doi=10.1038%2F84826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Mammalian SWI/SNF complexes promote MyoD-mediated muscle differentiation</span></div><div class="casAuthors">de la Serna, Ivana; Carlson, Kerri A.; Imbalzano, Anthony N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">187-190</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">Mammalian SWI/SNF complexes are ATP-dependent chromatin remodeling enzymes that have been implicated in the regulation of gene expression, cell-cycle control and oncogenesis.  MyoD is a muscle-specific regulator able to induce myogenesis in numerous cell types.  To ascertain the requirement for chromatin remodeling enzymes in cellular differentiation processes, we examd. MyoD-mediated induction of muscle differentiation in fibroblasts expressing dominant-neg. versions of the human brahma-related gene-1 (BRG1) or human brahma (BRM), the ATPase subunits of two distinct SWI/SNF enzymes.  We find that induction of the myogenic phenotype is completely abrogated in the presence of the mutant enzymes.  We further demonstrate that failure to induce muscle-specific gene expression correlates with inhibition of chromatin remodeling in the promoter region of an endogenous muscle-specific gene.  Our results demonstrate that SWI/SNF enzymes promote MyoD-mediated muscle differentiation and indicate that these enzymes function by altering chromatin structure in promoter regions of endogenous, differentiation-specific loci.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxtfpSZ1WGhLVg90H21EOLACvtfcHk0ljkrLTbd53qEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhtFGkur8%253D&md5=2d61d6ddae277103498e3e98ef563e7c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2F84826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F84826%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BLa%2BSerna%26aufirst%3DI.%2BL.%26aulast%3DCarlson%26aufirst%3DK.%2BA.%26aulast%3DImbalzano%26aufirst%3DA.%2BN.%26atitle%3DMammalian%2520SWI%252FSNF%2520complexes%2520promote%2520MyoD-mediated%2520muscle%2520differentiation%26jtitle%3DNat.%2520Genet.%26date%3D2001%26volume%3D27%26spage%3D187%26epage%3D190%26doi%3D10.1038%2F84826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lickert, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeuchi, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Both, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walls, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAuliffe, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee Adamson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mark Henkelman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrana, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossant, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruneau, B. G.</span></span> <span> </span><span class="NLM_article-title">Baf60c is essential for function of BAF chromatin remodelling complexes in heart development</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>432</i></span>,  <span class="NLM_fpage">107</span>– <span class="NLM_lpage">112</span>, <span class="refDoi"> DOI: 10.1038/nature03071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fnature03071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=15525990" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpt1eitLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=107-112&author=H.+Lickertauthor=J.+K.+Takeuchiauthor=I.+von+Bothauthor=J.+R.+Wallsauthor=F.+McAuliffeauthor=S.+Lee+Adamsonauthor=R.+Mark+Henkelmanauthor=J.+L.+Wranaauthor=J.+Rossantauthor=B.+G.+Bruneau&title=Baf60c+is+essential+for+function+of+BAF+chromatin+remodelling+complexes+in+heart+development&doi=10.1038%2Fnature03071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Baf60c is essential for function of BAF chromatin remodelling complexes in heart development</span></div><div class="casAuthors">Lickert, Heiko; Takeuchi, Jun K.; von Both, Ingo; Walls, Johnathon R.; McAuliffe, Fionnuala; Adamson, S. Lee; Henkelman, R. Mark; Wrana, Jeffrey L.; Rossant, Janet; Bruneau, Benoit G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7013</span>),
    <span class="NLM_cas:pages">107-112</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Tissue-specific transcription factors regulate several important aspects of embryonic development.  They must function in the context of DNA assembled into the higher-order structure of chromatin.  Enzymic complexes such as the Swi/Snf-like BAF complexes remodel chromatin to allow the transcriptional machinery access to gene regulatory elements.  Here the authors show that Smarcd3, encoding Baf60c, a subunit of the BAF complexes, is expressed specifically in the heart and somites in the early mouse embryo.  Smarcd3 silencing by RNA interference in mouse embryos derived from embryonic stem cells causes defects in heart morphogenesis that reflect impaired expansion of the anterior/secondary heart field, and also results in abnormal cardiac and skeletal muscle differentiation.  An intermediate redn. in Smarcd3 expression leads to defects in outflow tract remodelling reminiscent of human congenital heart defects.  Baf60c overexpressed in cell culture can mediate interactions between cardiac transcription factors and the BAF complex ATPase Brg1, thereby potentiating the activation of target genes.  These results reveal tissue-specific and dose-dependent roles for Baf60c in recruiting BAF chromatin remodelling complexes to heart-specific enhancers, providing a novel mechanism to ensure transcriptional regulation during organogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9V5xD2mMuhLVg90H21EOLACvtfcHk0liSb__MxqdgJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpt1eitLg%253D&md5=63dcaa76fa4c7c4671860bd8f8a2b6e7</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnature03071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03071%26sid%3Dliteratum%253Aachs%26aulast%3DLickert%26aufirst%3DH.%26aulast%3DTakeuchi%26aufirst%3DJ.%2BK.%26aulast%3Dvon%2BBoth%26aufirst%3DI.%26aulast%3DWalls%26aufirst%3DJ.%2BR.%26aulast%3DMcAuliffe%26aufirst%3DF.%26aulast%3DLee%2BAdamson%26aufirst%3DS.%26aulast%3DMark%2BHenkelman%26aufirst%3DR.%26aulast%3DWrana%26aufirst%3DJ.%2BL.%26aulast%3DRossant%26aufirst%3DJ.%26aulast%3DBruneau%26aufirst%3DB.%2BG.%26atitle%3DBaf60c%2520is%2520essential%2520for%2520function%2520of%2520BAF%2520chromatin%2520remodelling%2520complexes%2520in%2520heart%2520development%26jtitle%3DNature%26date%3D2004%26volume%3D432%26spage%3D107%26epage%3D112%26doi%3D10.1038%2Fnature03071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bracken, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brien, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verrijzer, C. P.</span></span> <span> </span><span class="NLM_article-title">Dangerous liaisons: interplay between SWI/SNF, NuRD, and polycomb in chromatin regulation and cancer</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">936</span>– <span class="NLM_lpage">959</span>, <span class="refDoi"> DOI: 10.1101/gad.326066.119</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1101%2Fgad.326066.119" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=31123059" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslKmtbrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2019&pages=936-959&author=A.+P.+Brackenauthor=G.+L.+Brienauthor=C.+P.+Verrijzer&title=Dangerous+liaisons%3A+interplay+between+SWI%2FSNF%2C+NuRD%2C+and+polycomb+in+chromatin+regulation+and+cancer&doi=10.1101%2Fgad.326066.119"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Dangerous liaisons: interplay between SWI/SNF, NuRD, and Polycomb in chromatin regulation and cancer</span></div><div class="casAuthors">Bracken, Adrian P.; Brien, Gerard L.; Verrijzer, C. Peter</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">15-16</span>),
    <span class="NLM_cas:pages">936-959</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Changes in chromatin structure mediated by ATP-dependent nucleosome remodelers and histone modifying enzymes are integral to the process of gene regulation.  Here, we review the roles of the SWI/SNF (switch/sucrose nonfermenting) and NuRD (nucleosome remodeling and deacetylase) and the Polycomb system in chromatin regulation and cancer.  First, we discuss the basic mol. mechanism of nucleosome remodeling, and how this controls gene transcription.  Next, we provide an overview of the functional organization and biochem. activities of SWI/SNF, NuRD, and Polycomb complexes.  We describe how, in metazoans, the balance of these activities is central to the proper regulation of gene expression and cellular identity during development.  Whereas SWI/SNF counteracts Polycomb, NuRD facilitates Polycomb repression on chromatin.  Finally, we discuss how disruptions of this regulatory equil. contribute to oncogenesis, and how new insights into the biol. functions of remodelers and Polycombs are opening avenues for therapeutic interventions on a broad range of cancer types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4BhoBUmy0wbVg90H21EOLACvtfcHk0liSb__MxqdgJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslKmtbrJ&md5=2ac67cb91973994d332837415c084306</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1101%2Fgad.326066.119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.326066.119%26sid%3Dliteratum%253Aachs%26aulast%3DBracken%26aufirst%3DA.%2BP.%26aulast%3DBrien%26aufirst%3DG.%2BL.%26aulast%3DVerrijzer%26aufirst%3DC.%2BP.%26atitle%3DDangerous%2520liaisons%253A%2520interplay%2520between%2520SWI%252FSNF%252C%2520NuRD%252C%2520and%2520polycomb%2520in%2520chromatin%2520regulation%2520and%2520cancer%26jtitle%3DGenes%2520Dev.%26date%3D2019%26volume%3D33%26spage%3D936%26epage%3D959%26doi%3D10.1101%2Fgad.326066.119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Varela, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarpey, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raine, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ong, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teague, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bignell, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalgliesh, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galappaththige, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latimer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLaren, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menzies, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mudie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stebbings, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Largaespada, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wessels, L. F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahnoski, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anema, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">A.Tuveson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Mancera, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustonen, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rust, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan-on, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subimerb, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dykema, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furge, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teh, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stratton, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Futreal, P. A.</span></span> <span> </span><span class="NLM_article-title">Exome sequencing identifies frequent mutation of the SWI/SNF complex gene PBRM1 in renal carcinoma</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>469</i></span>,  <span class="NLM_fpage">539</span>– <span class="NLM_lpage">542</span>, <span class="refDoi"> DOI: 10.1038/nature09639</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fnature09639" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=21248752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnsFaquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=469&publication_year=2011&pages=539-542&author=I.+Varelaauthor=P.+Tarpeyauthor=K.+Raineauthor=D.+Huangauthor=C.+K.+Ongauthor=P.+Stephensauthor=H.+Daviesauthor=D.+Jonesauthor=M.-L.+Linauthor=J.+Teagueauthor=G.+Bignellauthor=A.+Butlerauthor=J.+Choauthor=G.+L.+Dalglieshauthor=D.+Galappaththigeauthor=C.+Greenmanauthor=C.+Hardyauthor=M.+Jiaauthor=C.+Latimerauthor=K.+W.+Lauauthor=J.+Marshallauthor=S.+McLarenauthor=A.+Menziesauthor=L.+Mudieauthor=L.+Stebbingsauthor=D.+A.+Largaespadaauthor=L.+F.+A.+Wesselsauthor=S.+Richardauthor=R.+J.+Kahnoskiauthor=J.+Anemaauthor=D.+A.Tuvesonauthor=P.+A.+Perez-Manceraauthor=V.+Mustonenauthor=A.+Fischerauthor=D.+J.+Adamsauthor=A.+Rustauthor=W.+Chan-onauthor=C.+Subimerbauthor=K.+Dykemaauthor=K.+Furgeauthor=P.+J.+Campbellauthor=B.+T.+Tehauthor=M.+R.+Strattonauthor=P.+A.+Futreal&title=Exome+sequencing+identifies+frequent+mutation+of+the+SWI%2FSNF+complex+gene+PBRM1+in+renal+carcinoma&doi=10.1038%2Fnature09639"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Exome sequencing identifies frequent mutation of the SWI/SNF complex gene PBRM1 in renal carcinoma</span></div><div class="casAuthors">Varela, Ignacio; Tarpey, Patrick; Raine, Keiran; Huang, Dachuan; Ong, Choon Kiat; Stephens, Philip; Davies, Helen; Jones, David; Lin, Meng-Lay; Teague, Jon; Bignell, Graham; Butler, Adam; Cho, Juok; Dalgliesh, Gillian L.; Galappaththige, Danushka; Greenman, Chris; Hardy, Claire; Jia, Mingming; Latimer, Calli; Lau, King Wai; Marshall, John; McLaren, Stuart; Menzies, Andrew; Mudie, Laura; Stebbings, Lucy; Largaespada, David A.; Wessels, L. F. A.; Richard, Stephane; Kahnoski, Richard J.; Anema, John; Tuveson, David A.; Perez-Mancera, Pedro A.; Mustonen, Ville; Fischer, Andrej; Adams, David J.; Rust, Alistair; Chan-on, Waraporn; Subimerb, Chutima; Dykema, Karl; Furge, Kyle; Campbell, Peter J.; Teh, Bin Tean; Stratton, Michael R.; Futreal, P. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">469</span>
        (<span class="NLM_cas:issue">7331</span>),
    <span class="NLM_cas:pages">539-542</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The genetics of renal cancer is dominated by inactivation of the VHL tumor suppressor gene in clear cell carcinoma (ccRCC), the commonest histol. subtype.  A recent large-scale screen of ∼3500 genes by PCR-based exon re-sequencing identified several new cancer genes in ccRCC including UTX (also known as KDM6A), JARID1C (also known as KDM5C) and SETD2 (ref. 2).  These genes encode enzymes that demethylate (UTX, JARID1C) or methylate (SETD2) key lysine residues of histone H3.  Modification of the methylation state of these lysine residues of histone H3 regulates chromatin structure and is implicated in transcriptional control.  However, together these mutations are present in fewer than 15% of ccRCC, suggesting the existence of addnl., currently unidentified cancer genes.  Here, we have sequenced the protein coding exome in a series of primary ccRCC and report the identification of the SWI/SNF chromatin remodeling complex gene PBRM1 (ref. 4) as a second major ccRCC cancer gene, with truncating mutations in 41% (92/227) of cases.  These data further elucidate the somatic genetic architecture of ccRCC and emphasize the marked contribution of aberrant chromatin biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKCmhiZqUc6LVg90H21EOLACvtfcHk0liSb__MxqdgJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnsFaquw%253D%253D&md5=24c3de69a0b7bc7f90c1f14e7732e4c4</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnature09639&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09639%26sid%3Dliteratum%253Aachs%26aulast%3DVarela%26aufirst%3DI.%26aulast%3DTarpey%26aufirst%3DP.%26aulast%3DRaine%26aufirst%3DK.%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DOng%26aufirst%3DC.%2BK.%26aulast%3DStephens%26aufirst%3DP.%26aulast%3DDavies%26aufirst%3DH.%26aulast%3DJones%26aufirst%3DD.%26aulast%3DLin%26aufirst%3DM.-L.%26aulast%3DTeague%26aufirst%3DJ.%26aulast%3DBignell%26aufirst%3DG.%26aulast%3DButler%26aufirst%3DA.%26aulast%3DCho%26aufirst%3DJ.%26aulast%3DDalgliesh%26aufirst%3DG.%2BL.%26aulast%3DGalappaththige%26aufirst%3DD.%26aulast%3DGreenman%26aufirst%3DC.%26aulast%3DHardy%26aufirst%3DC.%26aulast%3DJia%26aufirst%3DM.%26aulast%3DLatimer%26aufirst%3DC.%26aulast%3DLau%26aufirst%3DK.%2BW.%26aulast%3DMarshall%26aufirst%3DJ.%26aulast%3DMcLaren%26aufirst%3DS.%26aulast%3DMenzies%26aufirst%3DA.%26aulast%3DMudie%26aufirst%3DL.%26aulast%3DStebbings%26aufirst%3DL.%26aulast%3DLargaespada%26aufirst%3DD.%2BA.%26aulast%3DWessels%26aufirst%3DL.%2BF.%2BA.%26aulast%3DRichard%26aufirst%3DS.%26aulast%3DKahnoski%26aufirst%3DR.%2BJ.%26aulast%3DAnema%26aufirst%3DJ.%26aulast%3DA.Tuveson%26aufirst%3DD.%26aulast%3DPerez-Mancera%26aufirst%3DP.%2BA.%26aulast%3DMustonen%26aufirst%3DV.%26aulast%3DFischer%26aufirst%3DA.%26aulast%3DAdams%26aufirst%3DD.%2BJ.%26aulast%3DRust%26aufirst%3DA.%26aulast%3DChan-on%26aufirst%3DW.%26aulast%3DSubimerb%26aufirst%3DC.%26aulast%3DDykema%26aufirst%3DK.%26aulast%3DFurge%26aufirst%3DK.%26aulast%3DCampbell%26aufirst%3DP.%2BJ.%26aulast%3DTeh%26aufirst%3DB.%2BT.%26aulast%3DStratton%26aufirst%3DM.%2BR.%26aulast%3DFutreal%26aufirst%3DP.%2BA.%26atitle%3DExome%2520sequencing%2520identifies%2520frequent%2520mutation%2520of%2520the%2520SWI%252FSNF%2520complex%2520gene%2520PBRM1%2520in%2520renal%2520carcinoma%26jtitle%3DNature%26date%3D2011%26volume%3D469%26spage%3D539%26epage%3D542%26doi%3D10.1038%2Fnature09639" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Porter, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dykhuizen, E. C.</span></span> <span> </span><span class="NLM_article-title">Individual bromodomains of polybromo-1 contribute to chromatin association and tumor suppression in clear cell renal carcinoma</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">2601</span>– <span class="NLM_lpage">2610</span>, <span class="refDoi"> DOI: 10.1074/jbc.m116.746875</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1074%2Fjbc.M116.746875" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=28053089" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlyht73J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=2601-2610&author=E.+G.+Porterauthor=E.+C.+Dykhuizen&title=Individual+bromodomains+of+polybromo-1+contribute+to+chromatin+association+and+tumor+suppression+in+clear+cell+renal+carcinoma&doi=10.1074%2Fjbc.m116.746875"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Individual bromodomains of polybromo-1 contribute to chromatin association and tumor suppression in clear cell renal carcinoma</span></div><div class="casAuthors">Porter, Elizabeth G.; Dykhuizen, Emily C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2601-2610</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The architecture of chromatin is governed, in part, by ATP-dependent chromatin remodelers.  These multiprotein complexes contain targeting domains that recognize post-translational marks on histones.  One such targeting domain is the bromodomain (BD), which recognizes acetyl-lysines and recruits proteins to sites of acetylation across the genome.  Polybromo1 (PBRM1), a subunit of the Polybromo-assocd. BRG1- or hBRM-assocd. factors (PBAF) chromatin remodeler, contains six tandem BDs and is frequently mutated in clear cell renal cell carcinoma (ccRCC).  Mutations in the PBRM1 gene often lead to the loss of protein expression; however, missense mutations in PBRM1 have been identified and tend to cluster in the BDs, particularly BD2 and BD4, suggesting that individual BDs are crit. for PBRM1 function.  To study the role of these six BDs, we inactivated each of the six BDs of PBRM1 and re-expressed these mutants in Caki2 cells (ccRCC cells with the loss of function mutation in PBRM1).  Four of the six BDs abrogated PBRM1 tumor suppressor function, gene regulation, and chromatin affinity with the degree of importance correlating strongly to the rate of missense mutations in patients.  Furthermore, we identified BD2 as the most crit. for PBRM1 and confirmed BD2-mediated assocn. to histone H3 peptides acetylated at lysine 14 (H3K14Ac), validating the importance of this specific acetylation mark for PBRM1 binding.  From these data, we conclude that four of the BDs act together to target PBRM1 to sites on chromatin; when a single BD is mutated, PBRM1 no longer controls gene expression properly, leading to increased cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8VdeFhQpTwLVg90H21EOLACvtfcHk0lgCj7ojSKdBEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlyht73J&md5=8c74fe7140ae901f95c280e68c073f76</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.746875&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.746875%26sid%3Dliteratum%253Aachs%26aulast%3DPorter%26aufirst%3DE.%2BG.%26aulast%3DDykhuizen%26aufirst%3DE.%2BC.%26atitle%3DIndividual%2520bromodomains%2520of%2520polybromo-1%2520contribute%2520to%2520chromatin%2520association%2520and%2520tumor%2520suppression%2520in%2520clear%2520cell%2520renal%2520carcinoma%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D2601%26epage%3D2610%26doi%3D10.1074%2Fjbc.m116.746875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruthenburg, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David Allis, C.</span></span> <span> </span><span class="NLM_article-title">Multivalent engagement of chromatin modifications by linked binding modules</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">983</span>– <span class="NLM_lpage">994</span>, <span class="refDoi"> DOI: 10.1038/nrm2298</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fnrm2298" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=18037899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlCrsb3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=983-994&author=A.+J.+Ruthenburgauthor=H.+Liauthor=D.+J.+Patelauthor=C.+David+Allis&title=Multivalent+engagement+of+chromatin+modifications+by+linked+binding+modules&doi=10.1038%2Fnrm2298"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Multivalent engagement of chromatin modifications by linked binding modules</span></div><div class="casAuthors">Ruthenburg, Alexander J.; Li, Haitao; Patel, Dinshaw J.; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">983-994</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Histone post-translational modifications (PTMs) play crucial roles in genome management, in part by recruiting specific factors that alter the structural properties of chromatin.  These so-called effector complexes often comprise multiple histone-binding modules that may act in concert to regulate chromatin structure and DNA-related activities.  Various chem. modifications on histones and regions of assocd. DNA play crucial roles in genome management by binding specific factors that, in turn, serve to alter the structural properties of chromatin.  These so-called effector proteins have typically been studied with the biochemist's paring knife; the capacity to recognize specific chromatin PTMs has been mapped to an increasing no. of domains that frequently appear in the nuclear subset of the proteome, often present in large, multisubunit complexes that bristle with modification-dependent binding potential.  The authors propose that multivalent interactions on a single histone tail and beyond may have a significant, if not dominant, role in chromatin transactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkIrlnyvt64LVg90H21EOLACvtfcHk0ljTI_nCOvIwMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlCrsb3L&md5=0d27fc9ba2e2a009e0a9903e1c65bdf7</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnrm2298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm2298%26sid%3Dliteratum%253Aachs%26aulast%3DRuthenburg%26aufirst%3DA.%2BJ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DDavid%2BAllis%26aufirst%3DC.%26atitle%3DMultivalent%2520engagement%2520of%2520chromatin%2520modifications%2520by%2520linked%2520binding%2520modules%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2007%26volume%3D8%26spage%3D983%26epage%3D994%26doi%3D10.1038%2Fnrm2298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekaran, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, M.</span></span> <span> </span><span class="NLM_article-title">Polybromo-1-bromodomains bind histone H3 at specific acetyl-lysine positions</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">661</span>– <span class="NLM_lpage">666</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2007.01.193</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2Fj.bbrc.2007.01.193" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=17320048" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD2sXisVWjurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2007&pages=661-666&author=R.+Chandrasekaranauthor=M.+Thompson&title=Polybromo-1-bromodomains+bind+histone+H3+at+specific+acetyl-lysine+positions&doi=10.1016%2Fj.bbrc.2007.01.193"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Polybromo-1-bromodomains bind histone H3 at specific acetyl-lysine positions</span></div><div class="casAuthors">Chandrasekaran, Renu; Thompson, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">661-666</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The human polybromo-1 protein is thought to localize the Polybromo, BRG1-assocd. factors chromatin-remodeling complex to kinetochores during mitosis via direct interaction of its six tandem bromodomains with acetylated nucleosomes.  Bromodomains are acetyl-lysine binding modules roughly 100 amino acids in length originally found in chromatin assocd. proteins.  Previous studies verified acetyl-histone binding by each bromodomain, but site-specificity, a central tenet of the histone code hypothesis, was not examd.  Here, the acetylation site-dependence of bromodomain-histone interactions was examd. using steady-state fluorescence anisotropy.  Results indicate that single bromodomains bind specific acetyl-lysine sites within the histone tail with sub-micromolar affinity.  Identification of duplicate target sites suggests that native Pb1 interacts with both copies of histone H3 upon nucleosome assembly.  Quant. anal. of single bromodomain-histone interactions can be used to develop hypotheses regarding the histone acetylation pattern that acts as the binding target of the native polybromo-1 protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCjqNIdWtx37Vg90H21EOLACvtfcHk0ljTI_nCOvIwMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXisVWjurY%253D&md5=b344831ee2de7822c90937be0849a408</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2007.01.193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2007.01.193%26sid%3Dliteratum%253Aachs%26aulast%3DChandrasekaran%26aufirst%3DR.%26aulast%3DThompson%26aufirst%3DM.%26atitle%3DPolybromo-1-bromodomains%2520bind%2520histone%2520H3%2520at%2520specific%2520acetyl-lysine%2520positions%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2007%26volume%3D355%26spage%3D661%26epage%3D666%26doi%3D10.1016%2Fj.bbrc.2007.01.193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Charlop-Powers, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">Structural insights into selective histone H3 recognition by the human polybromo bromodomain 2</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">529</span>– <span class="NLM_lpage">538</span>, <span class="refDoi"> DOI: 10.1038/cr.2010.43</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fcr.2010.43" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=20368734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsFSqu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=529-538&author=Z.+Charlop-Powersauthor=L.+Zengauthor=Q.+Zhangauthor=M.-M.+Zhou&title=Structural+insights+into+selective+histone+H3+recognition+by+the+human+polybromo+bromodomain+2&doi=10.1038%2Fcr.2010.43"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into selective histone H3 recognition by the human Polybromo bromodomain 2</span></div><div class="casAuthors">Charlop-Powers, Zachary; Zeng, Lei; Zhang, Qiang; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">529-538</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The Polybromo (PB) protein functions as a key component of the human PBAF chromatin remodeling complex in regulation of gene transcription.  PB is made up of modular domains including six bromodomains that are known as acetyl-lysine binding domains.  However, histone-binding specificity of the bromodomains of PB has remained elusive.  In this study, we report biochem. characterization of all six PB bromodomains' binding to a suite of lysine-acetylated peptides derived from known acetylation sites on human core histones.  We demonstrate that bromodomain 2 of PB preferentially recognizes acetylated lysine 14 of histone H3 (H3K14ac), a post-translational mark known for gene transcriptional activation.  We further describe the mol. basis of the selective H3K14ac recognition of bromodomain 2 by solving the protein structures in both the free and bound forms using X-ray crystallog. and NMR, resp.  Cell Research (2010) 20:529-538. doi:10.1038/cr.2010.43; published online 6 Apr. 2010.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq2k1NkpJHTrVg90H21EOLACvtfcHk0liFjFwzRRvecg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsFSqu7w%253D&md5=1895bb2b0e8aafbef170dec26226994f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fcr.2010.43&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2010.43%26sid%3Dliteratum%253Aachs%26aulast%3DCharlop-Powers%26aufirst%3DZ.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DStructural%2520insights%2520into%2520selective%2520histone%2520H3%2520recognition%2520by%2520the%2520human%2520polybromo%2520bromodomain%25202%26jtitle%3DCell%2520Res.%26date%3D2010%26volume%3D20%26spage%3D529%26epage%3D538%26doi%3D10.1038%2Fcr.2010.43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Slaughter, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanle, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McFadden, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollis, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suttle, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strahl, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. J.</span></span> <span> </span><span class="NLM_article-title">PBRM1 bromodomains variably influence nucleosome interactions and cellular function</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">13592</span>– <span class="NLM_lpage">13603</span>, <span class="refDoi"> DOI: 10.1074/jbc.ra118.003381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1074%2Fjbc.RA118.003381" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=29986887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1altr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=13592-13603&author=M.+J.+Slaughterauthor=E.+K.+Shanleauthor=A.+W.+McFaddenauthor=E.+S.+Hollisauthor=L.+E.+Suttleauthor=B.+D.+Strahlauthor=I.+J.+Davis&title=PBRM1+bromodomains+variably+influence+nucleosome+interactions+and+cellular+function&doi=10.1074%2Fjbc.ra118.003381"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">PBRM1 bromodomains variably influence nucleosome interactions and cellular function</span></div><div class="casAuthors">Slaughter, Mariesa J.; Shanle, Erin K.; McFadden, Andrew W.; Hollis, Emily S.; Suttle, Lindsey E.; Strahl, Brian D.; Davis, Ian J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">13592-13603</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Chromatin remodelers use bromodomains (BDs) to recognize histones.  Polybromo 1 (PBRM1 or BAF180) is hypothesized to function as the nucleosome-recognition subunit of the PBAF chromatin-remodeling complex and is frequently mutated in clear cell renal cell carcinoma (ccRCC).  Previous studies have applied in vitro methods to explore the binding specificities of the six individual PBRM1 BDs.  However, BD targeting to histones and the influence of neighboring BD on nucleosome recognition have not been well characterized.  Here, using histone microarrays and intact nucleosomes to investigate the histone-binding characteristics of the six PBRM1 BDs individually and combined, we demonstrate that BD2 and BD4 of PBRM1 mediate binding to acetylated histone peptides and to modified recombinant and cellular nucleosomes.  Moreover, we show that neighboring BDs variably modulate these chromatin interactions, with BD1 and BD5 enhancing nucleosome interactions of BD2 and BD4, resp., whereas BD3 attenuated these interactions.  We also found that binding pocket missense mutations in BD4 obsd. in ccRCC disrupt PBRM1-chromatin interactions and that these mutations in BD4, but not similar mutations in BD2, in the context of full-length PBRM1, accelerate ccRCC cell proliferation.  Taken together, our biochem. and mutational analyses have identified BD4 as being critically important for maintaining proper PBRM1 function and demonstrate that BD4 mutations increase ccRCC cell growth.  Because of the link between PBRM1 status and sensitivity to immune checkpoint inhibitor treatment, these data also suggest the relevance of BD4 as a potential clin. target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOqqSdkL2uHLVg90H21EOLACvtfcHk0liFjFwzRRvecg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1altr3L&md5=07a66a8302d15e1c2eebb405e69dfc1f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.RA118.003381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.RA118.003381%26sid%3Dliteratum%253Aachs%26aulast%3DSlaughter%26aufirst%3DM.%2BJ.%26aulast%3DShanle%26aufirst%3DE.%2BK.%26aulast%3DMcFadden%26aufirst%3DA.%2BW.%26aulast%3DHollis%26aufirst%3DE.%2BS.%26aulast%3DSuttle%26aufirst%3DL.%2BE.%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DDavis%26aufirst%3DI.%2BJ.%26atitle%3DPBRM1%2520bromodomains%2520variably%2520influence%2520nucleosome%2520interactions%2520and%2520cellular%2520function%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26spage%3D13592%26epage%3D13603%26doi%3D10.1074%2Fjbc.ra118.003381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alicea-Velázquez, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langbein, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greer, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosgrove, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span> <span> </span><span class="NLM_article-title">High affinity binding of H3K14ac through collaboration of bromodomains 2, 4 and 5 is critical for the molecular and tumor suppressor functions of PBRM1</span>. <i>Mol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">811</span>– <span class="NLM_lpage">828</span>, <span class="refDoi"> DOI: 10.1002/1878-0261.12434</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1002%2F1878-0261.12434" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=30585695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFaqs7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2019&pages=811-828&author=L.+Liaoauthor=N.+L.+Alicea-Vel%C3%A1zquezauthor=L.+Langbeinauthor=X.+Niuauthor=W.+Caiauthor=E.+A.+Choauthor=M.+Zhangauthor=C.+B.+Greerauthor=Q.+Yanauthor=M.+S.+Cosgroveauthor=H.+Yang&title=High+affinity+binding+of+H3K14ac+through+collaboration+of+bromodomains+2%2C+4+and+5+is+critical+for+the+molecular+and+tumor+suppressor+functions+of+PBRM1&doi=10.1002%2F1878-0261.12434"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">High affinity binding of H3K14ac through collaboration of bromodomains 2, 4 and 5 is critical for the molecular and tumor suppressor functions of PBRM1</span></div><div class="casAuthors">Liao, Lili; Alicea-Velazquez, Nilda L.; Langbein, Lauren; Niu, Xiaohua; Cai, Weijia; Cho, Eun-Ah; Zhang, Meiling; Greer, Celeste B.; Yan, Qin; Cosgrove, Michael S.; Yang, Haifeng</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Oncology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">811-828</span>CODEN:
                <span class="NLM_cas:coden">MOONC3</span>;
        ISSN:<span class="NLM_cas:issn">1878-0261</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Polybromo-1 is an important tumor suppressor in kidney cancer.  While BD2 has been found to bind acetylated histone H3 lysine 14 (H3K14ac), and the importance of H3K14ac recognition for the mol.  We discovered that full-length PBRM1, but not its individual BDs, strongly binds H3K14ac.  BDs 2, 4, and 5 were found to collaborate to facilitate strong binding to H3K14ac.  Quant. measurement of the interactions between purified BD proteins and H3K14ac or nonacetylated peptides confirmed the tight and specific assocn. of the former.  Interestingly, while the structural integrity of BD4 was found to be required for H3K14ac recognition, the conserved acetyl-lysine binding site of BD4 was not.  Furthermore, simultaneous point mutations in BDs 2, 4, and 5 prevented recognition of H3K14ac, altered promoter binding and gene expression, and caused PBRM1 to relocalize to the cytoplasm.  In contrast, tumor-derived point mutations in BD2 alone lowered PBRM1's affinity to H3K14ac and also disrupted promoter binding and gene expression without altering cellular localization.  Finally, overexpression of PBRM1 variants contg. point mutations in BDs 2, 4, and 5 or BD2 alone failed to suppress tumor growth in a xenograft model.  Taken together, our study demonstrates that BDs 2,4, and 5 of PBRM1 collaborate to generate high affinity to H3K14ac and tether PBRM1 to chromatin.  Mutations in BD2 alone weaken these interactions, and this is sufficient to abolish its mol. and tumor suppressor functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWdNbAce1d3rVg90H21EOLACvtfcHk0liFjFwzRRvecg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFaqs7vI&md5=d6763d01ce78da05205564b73b5e6b75</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2F1878-0261.12434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1878-0261.12434%26sid%3Dliteratum%253Aachs%26aulast%3DLiao%26aufirst%3DL.%26aulast%3DAlicea-Vel%25C3%25A1zquez%26aufirst%3DN.%2BL.%26aulast%3DLangbein%26aufirst%3DL.%26aulast%3DNiu%26aufirst%3DX.%26aulast%3DCai%26aufirst%3DW.%26aulast%3DCho%26aufirst%3DE.%2BA.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DGreer%26aufirst%3DC.%2BB.%26aulast%3DYan%26aufirst%3DQ.%26aulast%3DCosgrove%26aufirst%3DM.%2BS.%26aulast%3DYang%26aufirst%3DH.%26atitle%3DHigh%2520affinity%2520binding%2520of%2520H3K14ac%2520through%2520collaboration%2520of%2520bromodomains%25202%252C%25204%2520and%25205%2520is%2520critical%2520for%2520the%2520molecular%2520and%2520tumor%2520suppressor%2520functions%2520of%2520PBRM1%26jtitle%3DMol.%2520Oncol.%26date%3D2019%26volume%3D13%26spage%3D811%26epage%3D828%26doi%3D10.1002%2F1878-0261.12434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kadoch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodges, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elias, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranish, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crabtree, G. R.</span></span> <span> </span><span class="NLM_article-title">Proteomic and bioinformatic analysis of mammalian SWI/SNF complexes identifies extensive roles in human malignancy</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">592</span>– <span class="NLM_lpage">601</span>, <span class="refDoi"> DOI: 10.1038/ng.2628</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fng.2628" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=23644491" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntVart7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2013&pages=592-601&author=C.+Kadochauthor=D.+C.+Hargreavesauthor=C.+Hodgesauthor=L.+Eliasauthor=L.+Hoauthor=J.+Ranishauthor=G.+R.+Crabtree&title=Proteomic+and+bioinformatic+analysis+of+mammalian+SWI%2FSNF+complexes+identifies+extensive+roles+in+human+malignancy&doi=10.1038%2Fng.2628"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic and bioinformatic analysis of mammalian SWI/SNF complexes identifies extensive roles in human malignancy</span></div><div class="casAuthors">Kadoch, Cigall; Hargreaves, Diana C.; Hodges, Courtney; Elias, Laura; Ho, Lena; Ranish, Jeff; Crabtree, Gerald R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">592-601</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Subunits of mammalian SWI/SNF (mSWI/SNF or BAF) complexes have recently been implicated as tumor suppressors in human malignancies.  To understand the full extent of their involvement, we conducted a proteomic anal. of endogenous mSWI/SNF complexes, which identified several new dedicated, stable subunits not found in yeast SWI/SNF complexes, including BCL7A, BCL7B and BCL7C, BCL11A and BCL11B, BRD9 and SS18.  Incorporating these new members, we detd. mSWI/SNF subunit mutation frequency in exome and whole-genome sequencing studies of primary human tumors.  Notably, mSWI/SNF subunits are mutated in 19.6% of all human tumors reported in 44 studies.  Our anal. suggests that specific subunits protect against cancer in specific tissues.  In addn., mutations affecting more than one subunit, defined here as compd. heterozygosity, are prevalent in certain cancers.  Our studies demonstrate that mSWI/SNF is the most frequently mutated chromatin-regulatory complex (CRC) in human cancer, exhibiting a broad mutation pattern, similar to that of TP53.  Thus, proper functioning of polymorphic BAF complexes may constitute a major mechanism of tumor suppression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP873DCpn7JrVg90H21EOLACvtfcHk0lhioM9TqanPgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntVart7s%253D&md5=0f49cf3c59e7bc04b74970fe656b1e17</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fng.2628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.2628%26sid%3Dliteratum%253Aachs%26aulast%3DKadoch%26aufirst%3DC.%26aulast%3DHargreaves%26aufirst%3DD.%2BC.%26aulast%3DHodges%26aufirst%3DC.%26aulast%3DElias%26aufirst%3DL.%26aulast%3DHo%26aufirst%3DL.%26aulast%3DRanish%26aufirst%3DJ.%26aulast%3DCrabtree%26aufirst%3DG.%2BR.%26atitle%3DProteomic%2520and%2520bioinformatic%2520analysis%2520of%2520mammalian%2520SWI%252FSNF%2520complexes%2520identifies%2520extensive%2520roles%2520in%2520human%2520malignancy%26jtitle%3DNat.%2520Genet.%26date%3D2013%26volume%3D45%26spage%3D592%26epage%3D601%26doi%3D10.1038%2Fng.2628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chabanon, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Postel-Vinay, S.</span></span> <span> </span><span class="NLM_article-title">Exploiting epigenetic vulnerabilities in solid tumors: Novel therapeutic opportunities in the treatment of SWI/SNF-defective cancers</span>. <i>Semin. Cancer Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">180</span>– <span class="NLM_lpage">198</span>, <span class="refDoi"> DOI: 10.1016/j.semcancer.2019.09.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2Fj.semcancer.2019.09.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=31568814" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFyntbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2020&pages=180-198&author=R.+M.+Chabanonauthor=D.+Morelauthor=S.+Postel-Vinay&title=Exploiting+epigenetic+vulnerabilities+in+solid+tumors%3A+Novel+therapeutic+opportunities+in+the+treatment+of+SWI%2FSNF-defective+cancers&doi=10.1016%2Fj.semcancer.2019.09.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting epigenetic vulnerabilities in solid tumors: Novel therapeutic opportunities in the treatment of SWI/SNF-defective cancers</span></div><div class="casAuthors">Chabanon, Roman M.; Morel, Daphne; Postel-Vinay, Sophie</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Cancer Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">180-198</span>CODEN:
                <span class="NLM_cas:coden">SECBE7</span>;
        ISSN:<span class="NLM_cas:issn">1044-579X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Mammalian switch/sucrose non-fermentable (mSWI/SNF) family complexes are pivotal elements of the chromatin remodeling machinery, which contribute to the regulation of several major cellular functions.  Large-scale exome-wide sequencing studies have identified mutations in genes encoding mSWI/SNF subunits in 20% of all human cancers, establishing mSWI/SNF deficiency as a recurrent oncogenic alteration.  Accumulating evidence now supports that several mSWI/SNF defects represent targetable vulnerabilities in cancer; notably, recent research advances have unveiled unexpected synthetic lethal opportunities that foster the development of novel biomarker-driven and mechanism-based therapeutic approaches for the treatment of mSWI/SNF-deficient tumors.  Here, we review the latest breakthroughs and discoveries that inform our understanding of the mSWI/SNF complexes biol. in carcinogenesis, and discuss the most promising therapeutic strategies to target mSWI/SNF defects in human solid malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa-p9-2CfaObVg90H21EOLACvtfcHk0lhioM9TqanPgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFyntbbJ&md5=0aec46e82354754eb4177fe5e8ca7330</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.semcancer.2019.09.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.semcancer.2019.09.018%26sid%3Dliteratum%253Aachs%26aulast%3DChabanon%26aufirst%3DR.%2BM.%26aulast%3DMorel%26aufirst%3DD.%26aulast%3DPostel-Vinay%26aufirst%3DS.%26atitle%3DExploiting%2520epigenetic%2520vulnerabilities%2520in%2520solid%2520tumors%253A%2520Novel%2520therapeutic%2520opportunities%2520in%2520the%2520treatment%2520of%2520SWI%252FSNF-defective%2520cancers%26jtitle%3DSemin.%2520Cancer%2520Biol.%26date%3D2020%26volume%3D61%26spage%3D180%26epage%3D198%26doi%3D10.1016%2Fj.semcancer.2019.09.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oike, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogiwara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tominaga, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ando, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuta, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizukami, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isomura, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komachi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furuta, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, S.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokota, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohno, T.</span></span> <span> </span><span class="NLM_article-title">A synthetic lethality-based strategy to treat cancers harboring a genetic deficiency in the chromatin remodeling factor BRG1</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">5508</span>– <span class="NLM_lpage">5518</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-12-4593</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1158%2F0008-5472.CAN-12-4593" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=23872584" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlKkurfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=5508-5518&author=T.+Oikeauthor=H.+Ogiwaraauthor=Y.+Tominagaauthor=K.+Itoauthor=O.+Andoauthor=K.+Tsutaauthor=T.+Mizukamiauthor=Y.+Shimadaauthor=H.+Isomuraauthor=M.+Komachiauthor=K.+Furutaauthor=S.-I.+Watanabeauthor=T.+Nakanoauthor=J.+Yokotaauthor=T.+Kohno&title=A+synthetic+lethality-based+strategy+to+treat+cancers+harboring+a+genetic+deficiency+in+the+chromatin+remodeling+factor+BRG1&doi=10.1158%2F0008-5472.can-12-4593"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A Synthetic Lethality-Based Strategy to Treat Cancers Harboring a Genetic Deficiency in the Chromatin Remodeling Factor BRG1</span></div><div class="casAuthors">Oike, Takahiro; Ogiwara, Hideaki; Tominaga, Yuichi; Ito, Kentaro; Ando, Osamu; Tsuta, Koji; Mizukami, Tatsuji; Shimada, Yoko; Isomura, Hisanori; Komachi, Mayumi; Furuta, Koh; Watanabe, Shun-ichi; Nakano, Takashi; Yokota, Jun; Kohno, Takashi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5508-5518</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The occurrence of inactivating mutations in SWI/SNF chromatin-remodeling genes in common cancers has attracted a great deal of interest.  However, mechanistic strategies to target tumor cells carrying such mutations are yet to be developed.  This study proposes a synthetic-lethality therapy for treating cancers deficient in the SWI/SNF catalytic (ATPase) subunit, BRG1/SMARCA4.  The strategy relies upon inhibition of BRM/SMARCA2, another catalytic SWI/SNF subunit with a BRG1-related activity.  Immunohistochem. anal. of a cohort of non-small- cell lung carcinomas (NSCLC) indicated that 15.5% (16 of 103) of the cohort, corresponding to preferentially undifferentiated tumors, was deficient in BRG1 expression.  All BRG1-deficient cases were neg. for alterations in known therapeutic target genes, for example, EGFR and DDR2 gene mutations, ALK gene fusions, or FGFR1 gene amplifications.  RNA interference (RNAi)-mediated silencing of BRM suppressed the growth of BRG1-deficient cancer cells relative to BRG1-proficient cancer cells, inducing senescence via activation of p21/CDKN1A.  This growth suppression was reversed by transduction of wild-type but not ATPase-deficient BRG1.  In support of these in vitro results, a conditional RNAi study conducted in vivo revealed that BRM depletion suppressed the growth of BRG1-deficient tumor xenografts.  Our results offer a rationale to develop BRM-ATPase inhibitors as a strategy to treat BRG1/SMARCA4-deficient cancers, including NSCLCs that lack mutations in presently known therapeutic target genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcN-XkB72ouLVg90H21EOLACvtfcHk0lhioM9TqanPgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlKkurfO&md5=78fd175ce9dd7e2469cfe1ba4e6b49e1</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-12-4593&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-12-4593%26sid%3Dliteratum%253Aachs%26aulast%3DOike%26aufirst%3DT.%26aulast%3DOgiwara%26aufirst%3DH.%26aulast%3DTominaga%26aufirst%3DY.%26aulast%3DIto%26aufirst%3DK.%26aulast%3DAndo%26aufirst%3DO.%26aulast%3DTsuta%26aufirst%3DK.%26aulast%3DMizukami%26aufirst%3DT.%26aulast%3DShimada%26aufirst%3DY.%26aulast%3DIsomura%26aufirst%3DH.%26aulast%3DKomachi%26aufirst%3DM.%26aulast%3DFuruta%26aufirst%3DK.%26aulast%3DWatanabe%26aufirst%3DS.-I.%26aulast%3DNakano%26aufirst%3DT.%26aulast%3DYokota%26aufirst%3DJ.%26aulast%3DKohno%26aufirst%3DT.%26atitle%3DA%2520synthetic%2520lethality-based%2520strategy%2520to%2520treat%2520cancers%2520harboring%2520a%2520genetic%2520deficiency%2520in%2520the%2520chromatin%2520remodeling%2520factor%2520BRG1%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D5508%26epage%3D5518%26doi%3D10.1158%2F0008-5472.can-12-4593" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahal, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buxton, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frias, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagdasarian, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindeman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramadan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phadke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaskelioff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saenz-Vash, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myer, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLaughlin, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mickanin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, C. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmeier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagani, Z.</span></span> <span> </span><span class="NLM_article-title">Functional epigenetics approach identifies BRM/SMARCA2 as a critical synthetic lethal target in BRG1-deficient cancers</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">3128</span>– <span class="NLM_lpage">3133</span>, <span class="refDoi"> DOI: 10.1073/pnas.1316793111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1073%2Fpnas.1316793111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=24520176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtVKjurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=3128-3133&author=G.+R.+Hoffmanauthor=R.+Rahalauthor=F.+Buxtonauthor=K.+Xiangauthor=G.+McAllisterauthor=E.+Friasauthor=L.+Bagdasarianauthor=J.+Huberauthor=A.+Lindemanauthor=D.+Chenauthor=R.+Romeroauthor=N.+Ramadanauthor=T.+Phadkeauthor=K.+Haasauthor=M.+Jaskelioffauthor=B.+G.+Wilsonauthor=M.+J.+Meyerauthor=V.+Saenz-Vashauthor=H.+Zhaiauthor=V.+E.+Myerauthor=J.+A.+Porterauthor=N.+Keenauthor=M.+E.+McLaughlinauthor=C.+Mickaninauthor=C.+W.+M.+Robertsauthor=F.+Stegmeierauthor=Z.+Jagani&title=Functional+epigenetics+approach+identifies+BRM%2FSMARCA2+as+a+critical+synthetic+lethal+target+in+BRG1-deficient+cancers&doi=10.1073%2Fpnas.1316793111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Functional epigenetics approach identifies BRM/SMARCA2 as a critical synthetic lethal target in BRG1-deficient cancers</span></div><div class="casAuthors">Hoffman, Gregory R.; Rahal, Rami; Buxton, Frank; Xiang, Kay; McAllister, Gregory; Frias, Elizabeth; Bagdasarian, Linda; Huber, Janina; Lindeman, Alicia; Chen, Dongshu; Romero, Rodrigo; Ramadan, Nadire; Phadke, Tanushree; Haas, Kristy; Jaskelioff, Mariela; Wilson, Boris G.; Meyer, Matthew J.; Saenz-Vash, Veronica; Zhai, Huili; Myer, Vic E.; Porter, Jeffery A.; Keen, Nicholas; McLaughlin, Margaret E.; Mickanin, Craig; Roberts, Charles W. M.; Stegmeier, Frank; Jagani, Zainab</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3128-3133</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Defects in epigenetic regulation play a fundamental role in the development of cancer, and epigenetic regulators have recently emerged as promising therapeutic candidates.  We therefore set out to systematically interrogate epigenetic cancer dependencies by screening an epigenome-focused deep-coverage design shRNA (DECODER) library across 58 cancer cell lines.  This screen identified BRM/SMARCA2, a DNA-dependent ATPase of the mammalian SWI/SNF (mSWI/SNF) chromatin remodeling complex, as being essential for the growth of tumor cells that harbor loss of function mutations in BRG1/SMARCA4.  Depletion of BRM in BRG1-deficient cancer cells leads to a cell cycle arrest, induction of senescence, and increased levels of global H3K9me3.  We further demonstrate the selective dependency of BRG1-mutant tumors on BRM in vivo.  Genetic alterations of the mSWI/SNF chromatin remodeling complexes are the most frequent among chromatin regulators in cancers, with BRG1/SMARCA4 mutations occurring in ∼10-15% of lung adenocarcinomas.  Our findings position BRM as an attractive therapeutic target for BRG1 mutated cancers.  Because BRG1 and BRM function as mutually exclusive catalytic subunits of the mSWI/SNF complex, we propose that such synthetic lethality may be explained by paralog insufficiency, in which loss of one family member unveils crit. dependence on paralogous subunits.  This concept of "cancer-selective paralog dependency" may provide a more general strategy for targeting other tumor suppressor lesions/complexes with paralogous subunits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjF1zG7em3s7Vg90H21EOLACvtfcHk0lg3-L_RJdMc5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtVKjurc%253D&md5=f9c36efd50b517ef389b39ca6a65307d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1316793111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1316793111%26sid%3Dliteratum%253Aachs%26aulast%3DHoffman%26aufirst%3DG.%2BR.%26aulast%3DRahal%26aufirst%3DR.%26aulast%3DBuxton%26aufirst%3DF.%26aulast%3DXiang%26aufirst%3DK.%26aulast%3DMcAllister%26aufirst%3DG.%26aulast%3DFrias%26aufirst%3DE.%26aulast%3DBagdasarian%26aufirst%3DL.%26aulast%3DHuber%26aufirst%3DJ.%26aulast%3DLindeman%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DRomero%26aufirst%3DR.%26aulast%3DRamadan%26aufirst%3DN.%26aulast%3DPhadke%26aufirst%3DT.%26aulast%3DHaas%26aufirst%3DK.%26aulast%3DJaskelioff%26aufirst%3DM.%26aulast%3DWilson%26aufirst%3DB.%2BG.%26aulast%3DMeyer%26aufirst%3DM.%2BJ.%26aulast%3DSaenz-Vash%26aufirst%3DV.%26aulast%3DZhai%26aufirst%3DH.%26aulast%3DMyer%26aufirst%3DV.%2BE.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DMcLaughlin%26aufirst%3DM.%2BE.%26aulast%3DMickanin%26aufirst%3DC.%26aulast%3DRoberts%26aufirst%3DC.%2BW.%2BM.%26aulast%3DStegmeier%26aufirst%3DF.%26aulast%3DJagani%26aufirst%3DZ.%26atitle%3DFunctional%2520epigenetics%2520approach%2520identifies%2520BRM%252FSMARCA2%2520as%2520a%2520critical%2520synthetic%2520lethal%2520target%2520in%2520BRG1-deficient%2520cancers%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2014%26volume%3D111%26spage%3D3128%26epage%3D3133%26doi%3D10.1073%2Fpnas.1316793111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vangamudi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paul, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kost-Alimova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nottebaum, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahadeshwar, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Protopopov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Futreal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tieu, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peoples, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heffernan, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marszalek, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toniatti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrocchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhelle, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Draetta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, J. N.</span></span> <span> </span><span class="NLM_article-title">The SMARCA2/4 ATPase domain surpasses the bromodomain as a drug target in SWI/SNF-mutant cancers: Insights from cDNA rescue and PFI-3 inhibitor studies</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">3865</span>– <span class="NLM_lpage">3878</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-14-3798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1158%2F0008-5472.CAN-14-3798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=26139243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsV2gtL7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2015&pages=3865-3878&author=B.+Vangamudiauthor=T.+A.+Paulauthor=P.+K.+Shahauthor=M.+Kost-Alimovaauthor=L.+Nottebaumauthor=X.+Shiauthor=Y.+Zhanauthor=E.+Leoauthor=H.+S.+Mahadeshwarauthor=A.+Protopopovauthor=A.+Futrealauthor=T.+N.+Tieuauthor=M.+Peoplesauthor=T.+P.+Heffernanauthor=J.+R.+Marszalekauthor=C.+Toniattiauthor=A.+Petrocchiauthor=D.+Verhelleauthor=D.+R.+Owenauthor=G.+Draettaauthor=P.+Jonesauthor=W.+S.+Palmerauthor=S.+Sharmaauthor=J.+N.+Andersen&title=The+SMARCA2%2F4+ATPase+domain+surpasses+the+bromodomain+as+a+drug+target+in+SWI%2FSNF-mutant+cancers%3A+Insights+from+cDNA+rescue+and+PFI-3+inhibitor+studies&doi=10.1158%2F0008-5472.can-14-3798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">The SMARCA2/4 ATPase Domain Surpasses the Bromodomain as a Drug Target in SWI/SNF-Mutant Cancers: Insights from cDNA Rescue and PFI-3 Inhibitor Studies</span></div><div class="casAuthors">Vangamudi, Bhavatarini; Paul, Thomas A.; Shah, Parantu K.; Kost-Alimova, Maria; Nottebaum, Lisa; Shi, Xi; Zhan, Yanai; Leo, Elisabetta; Mahadeshwar, Harshad S.; Protopopov, Alexei; Futreal, Andrew; Tieu, Trang N.; Peoples, Mike; Heffernan, Timothy P.; Marszalek, Joseph R.; Toniatti, Carlo; Petrocchi, Alessia; Verhelle, Dominique; Owen, Dafydd R.; Draetta, Giulio; Jones, Philip; Palmer, Wylie S.; Sharma, Shikhar; Andersen, Jannik N.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3865-3878</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The SWI/SNF multisubunit complex modulates chromatin structure through the activity of two mutually exclusive catalytic subunits, SMARCA2 and SMARCA4, which both contain a bromodomain and an ATPase domain.  Using RNAi, cancer-specific vulnerabilities have been identified in SWI/SNF-mutant tumors, including SMARCA4-deficient lung cancer; however, the contribution of conserved, druggable protein domains to this anticancer phenotype is unknown.  Here, we functionally deconstruct the SMARCA2/4 paralog dependence of cancer cells using bioinformatics, genetic, and pharmacol. tools.  We evaluate a selective SMARCA2/4 bromodomain inhibitor (PFI-3) and characterize its activity in chromatin-binding and cell-functional assays focusing on cells with altered SWI/SNF complex (e.g., lung, synovial sarcoma, leukemia, and rhabdoid tumors).  We demonstrate that PFI-3 is a potent, cell-permeable probe capable of displacing ectopically expressed, GFP-tagged SMARCA2-bromodomain from chromatin, yet contrary to target knockdown, the inhibitor fails to display an antiproliferative phenotype.  Mechanistically, the lack of pharmacol. efficacy is reconciled by the failure of bromodomain inhibition to displace endogenous, full-length SMARCA2 from chromatin as detd. by in situ cell extn., chromatin immunopptn., and target gene expression studies.  Furthermore, using inducible RNAi and cDNA complementation (bromodomain- and ATPase-dead constructs), we unequivocally identify the ATPase domain, and not the bromodomain of SMARCA2, as the relevant therapeutic target with the catalytic activity suppressing defined transcriptional programs.  Taken together, our complementary genetic and pharmacol. studies exemplify a general strategy for multidomain protein drug-target validation and in case of SMARCA2/4 highlight the potential for drugging the more challenging helicase/ATPase domain to deliver on the promise of synthetic-lethality therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrix1Qe07Ed3LVg90H21EOLACvtfcHk0lg3-L_RJdMc5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsV2gtL7K&md5=bb35d59dabe56c253fef822a9fb337a8</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-14-3798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-14-3798%26sid%3Dliteratum%253Aachs%26aulast%3DVangamudi%26aufirst%3DB.%26aulast%3DPaul%26aufirst%3DT.%2BA.%26aulast%3DShah%26aufirst%3DP.%2BK.%26aulast%3DKost-Alimova%26aufirst%3DM.%26aulast%3DNottebaum%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DZhan%26aufirst%3DY.%26aulast%3DLeo%26aufirst%3DE.%26aulast%3DMahadeshwar%26aufirst%3DH.%2BS.%26aulast%3DProtopopov%26aufirst%3DA.%26aulast%3DFutreal%26aufirst%3DA.%26aulast%3DTieu%26aufirst%3DT.%2BN.%26aulast%3DPeoples%26aufirst%3DM.%26aulast%3DHeffernan%26aufirst%3DT.%2BP.%26aulast%3DMarszalek%26aufirst%3DJ.%2BR.%26aulast%3DToniatti%26aufirst%3DC.%26aulast%3DPetrocchi%26aufirst%3DA.%26aulast%3DVerhelle%26aufirst%3DD.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DDraetta%26aufirst%3DG.%26aulast%3DJones%26aufirst%3DP.%26aulast%3DPalmer%26aufirst%3DW.%2BS.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DAndersen%26aufirst%3DJ.%2BN.%26atitle%3DThe%2520SMARCA2%252F4%2520ATPase%2520domain%2520surpasses%2520the%2520bromodomain%2520as%2520a%2520drug%2520target%2520in%2520SWI%252FSNF-mutant%2520cancers%253A%2520Insights%2520from%2520cDNA%2520rescue%2520and%2520PFI-3%2520inhibitor%2520studies%26jtitle%3DCancer%2520Res.%26date%3D2015%26volume%3D75%26spage%3D3865%26epage%3D3878%26doi%3D10.1158%2F0008-5472.can-14-3798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schick, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rendeiro, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Runggatscher, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ringler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boidol, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinkel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Májek, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vulliard, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parapatics, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidl, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menche, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehmelt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petronczki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bock, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubicek, S.</span></span> <span> </span><span class="NLM_article-title">Systematic characterization of BAF mutations provides insights into intracomplex synthetic lethalities in human cancers</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">1399</span>– <span class="NLM_lpage">1410</span>, <span class="refDoi"> DOI: 10.1038/s41588-019-0477-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fs41588-019-0477-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=31427792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1WjurjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2019&pages=1399-1410&author=S.+Schickauthor=A.+F.+Rendeiroauthor=K.+Runggatscherauthor=A.+Ringlerauthor=B.+Boidolauthor=M.+Hinkelauthor=P.+M%C3%A1jekauthor=L.+Vulliardauthor=T.+Penzauthor=K.+Parapaticsauthor=C.+Schmidlauthor=J.+Mencheauthor=G.+Boehmeltauthor=M.+Petronczkiauthor=A.+C.+M%C3%BCllerauthor=C.+Bockauthor=S.+Kubicek&title=Systematic+characterization+of+BAF+mutations+provides+insights+into+intracomplex+synthetic+lethalities+in+human+cancers&doi=10.1038%2Fs41588-019-0477-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic characterization of BAF mutations provides insights into intracomplex synthetic lethalities in human cancers</span></div><div class="casAuthors">Schick, Sandra; Rendeiro, Andre F.; Runggatscher, Kathrin; Ringler, Anna; Boidol, Bernd; Hinkel, Melanie; Majek, Peter; Vulliard, Loan; Penz, Thomas; Parapatics, Katja; Schmidl, Christian; Menche, Joerg; Boehmelt, Guido; Petronczki, Mark; Mueller, Andre C.; Bock, Christoph; Kubicek, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1399-1410</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Aberrations in genes coding for subunits of the BRG1/BRM assocd. factor (BAF) chromatin remodeling complexes are highly abundant in human cancers.  Currently, it is not understood how these mostly loss-of-function mutations contribute to cancer development and how they can be targeted therapeutically.  The cancer-type-specific occurrence patterns of certain subunit mutations suggest subunit-specific effects on BAF complex function, possibly by the formation of aberrant residual complexes.  Here, we systematically characterize the effects of individual subunit loss on complex compn., chromatin accessibility and gene expression in a panel of knockout cell lines deficient for 22 BAF subunits.  We observe strong, specific and sometimes discordant alterations dependent on the targeted subunit and show that these explain intracomplex codependencies, including the synthetic lethal interactions SMARCA4-ARID2, SMARCA4-ACTB and SMARCC1-SMARCC2.  These data provide insights into the role of different BAF subcomplexes in genome-wide chromatin organization and suggest approaches to therapeutically target BAF-mutant cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriYa3fDjh9k7Vg90H21EOLACvtfcHk0liY_x2QwV6OCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1WjurjJ&md5=291cc74adc398d34900ffa6a8aec8e1b</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fs41588-019-0477-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41588-019-0477-9%26sid%3Dliteratum%253Aachs%26aulast%3DSchick%26aufirst%3DS.%26aulast%3DRendeiro%26aufirst%3DA.%2BF.%26aulast%3DRunggatscher%26aufirst%3DK.%26aulast%3DRingler%26aufirst%3DA.%26aulast%3DBoidol%26aufirst%3DB.%26aulast%3DHinkel%26aufirst%3DM.%26aulast%3DM%25C3%25A1jek%26aufirst%3DP.%26aulast%3DVulliard%26aufirst%3DL.%26aulast%3DPenz%26aufirst%3DT.%26aulast%3DParapatics%26aufirst%3DK.%26aulast%3DSchmidl%26aufirst%3DC.%26aulast%3DMenche%26aufirst%3DJ.%26aulast%3DBoehmelt%26aufirst%3DG.%26aulast%3DPetronczki%26aufirst%3DM.%26aulast%3DM%25C3%25BCller%26aufirst%3DA.%2BC.%26aulast%3DBock%26aufirst%3DC.%26aulast%3DKubicek%26aufirst%3DS.%26atitle%3DSystematic%2520characterization%2520of%2520BAF%2520mutations%2520provides%2520insights%2520into%2520intracomplex%2520synthetic%2520lethalities%2520in%2520human%2520cancers%26jtitle%3DNat.%2520Genet.%26date%3D2019%26volume%3D51%26spage%3D1399%26epage%3D1410%26doi%3D10.1038%2Fs41588-019-0477-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Michel, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Avino, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassel, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mashtalir, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenzie, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McBride, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valencia, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bocker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, L. M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remillard, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lareau, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zullow, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fortoul, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadoch, C.</span></span> <span> </span><span class="NLM_article-title">A non-canonical SWI/SNF complex is a synthetic lethal target in cancers driven by BAF complex perturbation</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1410</span>– <span class="NLM_lpage">1420</span>, <span class="refDoi"> DOI: 10.1038/s41556-018-0221-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fs41556-018-0221-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=30397315" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitV2qu7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2018&pages=1410-1420&author=B.+C.+Michelauthor=A.+R.+D%E2%80%99Avinoauthor=S.+H.+Casselauthor=N.+Mashtalirauthor=Z.+M.+McKenzieauthor=M.+J.+McBrideauthor=A.+M.+Valenciaauthor=Q.+Zhouauthor=M.+Bockerauthor=L.+M.+M.+Soaresauthor=J.+Panauthor=D.+I.+Remillardauthor=C.+A.+Lareauauthor=H.+J.+Zullowauthor=N.+Fortoulauthor=N.+S.+Grayauthor=J.+E.+Bradnerauthor=H.+M.+Chanauthor=C.+Kadoch&title=A+non-canonical+SWI%2FSNF+complex+is+a+synthetic+lethal+target+in+cancers+driven+by+BAF+complex+perturbation&doi=10.1038%2Fs41556-018-0221-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A non-canonical SWI/SNF complex is a synthetic lethal target in cancers driven by BAF complex perturbation</span></div><div class="casAuthors">Michel, Brittany C.; D'Avino, Andrew R.; Cassel, Seth H.; Mashtalir, Nazar; McKenzie, Zachary M.; McBride, Matthew J.; Valencia, Alfredo M.; Zhou, Qianhe; Bocker, Michael; Soares, Luis M. M.; Pan, Joshua; Remillard, David I.; Lareau, Caleb A.; Zullow, Hayley J.; Fortoul, Nora; Gray, Nathanael S.; Bradner, James E.; Chan, Ho Man; Kadoch, Cigall</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1410-1420</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Mammalian SWI/SNF chromatin remodeling complexes exist in three distinct, final-form assemblies: canonical BAF (cBAF), PBAF and a newly characterized non-canonical complex (ncBAF).  However, their complex-specific targeting on chromatin, functions and roles in disease remain largely undefined.  Here, we comprehensively mapped complex assemblies on chromatin and found that ncBAF complexes uniquely localize to CTCF sites and promoters.  We identified ncBAF subunits as synthetic lethal targets specific to synovial sarcoma and malignant rhabdoid tumors, which both exhibit cBAF complex (SMARCB1 subunit) perturbation.  Chem. and biol. depletion of the ncBAF subunit, BRD9, rapidly attenuates synovial sarcoma and malignant rhabdoid tumor cell proliferation.  Importantly, in cBAF-perturbed cancers, ncBAF complexes maintain gene expression at retained CTCF-promoter sites and function in a manner distinct from fusion oncoprotein-bound complexes.  Together, these findings unmask the unique targeting and functional roles of ncBAF complexes and present new cancer-specific therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWZOWaYPPJk7Vg90H21EOLACvtfcHk0liY_x2QwV6OCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitV2qu7nL&md5=1c8118a4e3a2a276a15aa42f5e8dd03c</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fs41556-018-0221-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41556-018-0221-1%26sid%3Dliteratum%253Aachs%26aulast%3DMichel%26aufirst%3DB.%2BC.%26aulast%3DD%25E2%2580%2599Avino%26aufirst%3DA.%2BR.%26aulast%3DCassel%26aufirst%3DS.%2BH.%26aulast%3DMashtalir%26aufirst%3DN.%26aulast%3DMcKenzie%26aufirst%3DZ.%2BM.%26aulast%3DMcBride%26aufirst%3DM.%2BJ.%26aulast%3DValencia%26aufirst%3DA.%2BM.%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DBocker%26aufirst%3DM.%26aulast%3DSoares%26aufirst%3DL.%2BM.%2BM.%26aulast%3DPan%26aufirst%3DJ.%26aulast%3DRemillard%26aufirst%3DD.%2BI.%26aulast%3DLareau%26aufirst%3DC.%2BA.%26aulast%3DZullow%26aufirst%3DH.%2BJ.%26aulast%3DFortoul%26aufirst%3DN.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DChan%26aufirst%3DH.%2BM.%26aulast%3DKadoch%26aufirst%3DC.%26atitle%3DA%2520non-canonical%2520SWI%252FSNF%2520complex%2520is%2520a%2520synthetic%2520lethal%2520target%2520in%2520cancers%2520driven%2520by%2520BAF%2520complex%2520perturbation%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2018%26volume%3D20%26spage%3D1410%26epage%3D1420%26doi%3D10.1038%2Fs41556-018-0221-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerstenberger, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trzupek, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedele, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parikh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samas, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Mahony, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berg, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrey, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samardjiev, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasipak, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padilla-Benavides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbalzano, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nickerson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span> <span> </span><span class="NLM_article-title">Identification of a chemical probe for family VIII bromodomains through optimization of a fragment hit</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4800</span>– <span class="NLM_lpage">4811</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00012</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00012" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmsl2rs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4800-4811&author=B.+S.+Gerstenbergerauthor=J.+D.+Trzupekauthor=C.+Tallantauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=P.+E.+Brennanauthor=V.+Fedeleauthor=S.+Martinauthor=S.+Picaudauthor=C.+Rogersauthor=M.+Parikhauthor=A.+Taylorauthor=B.+Samasauthor=A.+O%E2%80%99Mahonyauthor=E.+Bergauthor=G.+Pallaresauthor=A.+D.+Torreyauthor=D.+K.+Treiberauthor=I.+J.+Samardjievauthor=B.+T.+Nasipakauthor=T.+Padilla-Benavidesauthor=Q.+Wuauthor=A.+N.+Imbalzanoauthor=J.+A.+Nickersonauthor=M.+E.+Bunnageauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=D.+R.+Owen&title=Identification+of+a+chemical+probe+for+family+VIII+bromodomains+through+optimization+of+a+fragment+hit&doi=10.1021%2Facs.jmedchem.6b00012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Chemical Probe for Family VIII Bromodomains through Optimization of a Fragment Hit</span></div><div class="casAuthors">Gerstenberger, Brian S.; Trzupek, John D.; Tallant, Cynthia; Fedorov, Oleg; Filippakopoulos, Panagis; Brennan, Paul E.; Fedele, Vita; Martin, Sarah; Picaud, Sarah; Rogers, Catherine; Parikh, Mihir; Taylor, Alexandria; Samas, Brian; O'Mahony, Alison; Berg, Ellen; Pallares, Gabriel; Torrey, Adam V.; Treiber, Daniel K.; Samardjiev, Ivan J.; Nasipak, Brian T.; Padilla-Benavides, Teresita; Wu, Qiong; Imbalzano, Anthony N.; Nickerson, Jeffrey A.; Bunnage, Mark E.; Muller, Susanne; Knapp, Stefan; Owen, Dafydd R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4800-4811</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The acetyl post-translational modification of chromatin at selected histone lysine residues is interpreted by an acetyl-lysine specific interaction with bromodomain reader modules.  Here we report the discovery of the potent, acetyl-lysine-competitive, and cell active inhibitor PFI-3 that binds to certain family VIII bromodomains while displaying significant, broader bromodomain family selectivity.  The high specificity of PFI-3 for family VIII was achieved through a novel bromodomain binding mode of a phenolic headgroup that led to the unusual displacement of water mols. that are generally retained by most other bromodomain inhibitors reported to date.  The medicinal chem. program that led to PFI-3 from an initial fragment screening hit is described in detail, and addnl. analogs with differing family VIII bromodomain selectivity profiles are also reported.  We also describe the full pharmacol. characterization of PFI-3 as a chem. probe, along with phenotypic data on adipocyte and myoblast cell differentiation assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZlrg_xjnZGbVg90H21EOLACvtfcHk0ljyJPf_auzl6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmsl2rs74%253D&md5=6cb73e76a19f7fb44153f8cbfeff9cf6</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00012%26sid%3Dliteratum%253Aachs%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DTrzupek%26aufirst%3DJ.%2BD.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DFedele%26aufirst%3DV.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DParikh%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DA.%26aulast%3DSamas%26aufirst%3DB.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DBerg%26aufirst%3DE.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DTorrey%26aufirst%3DA.%2BD.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DSamardjiev%26aufirst%3DI.%2BJ.%26aulast%3DNasipak%26aufirst%3DB.%2BT.%26aulast%3DPadilla-Benavides%26aufirst%3DT.%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DImbalzano%26aufirst%3DA.%2BN.%26aulast%3DNickerson%26aufirst%3DJ.%2BA.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26atitle%3DIdentification%2520of%2520a%2520chemical%2520probe%2520for%2520family%2520VIII%2520bromodomains%2520through%2520optimization%2520of%2520a%2520fragment%2520hit%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4800%26epage%3D4811%26doi%3D10.1021%2Facs.jmedchem.6b00012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cockcroft, X.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiegen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hohmann, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knesl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornigg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nar, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaaf, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeeb, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehmelt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of an in vivo active selective BRD9 inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4462</span>– <span class="NLM_lpage">4475</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01865</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01865" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjt1Sit7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4462-4475&author=L.+J.+Martinauthor=M.+Koeglauthor=G.+Baderauthor=X.-L.+Cockcroftauthor=O.+Fedorovauthor=D.+Fiegenauthor=T.+Gerstbergerauthor=M.+H.+Hofmannauthor=A.+F.+Hohmannauthor=D.+Kesslerauthor=S.+Knappauthor=P.+Kneslauthor=S.+Korniggauthor=S.+M%C3%BCllerauthor=H.+Narauthor=C.+Rogersauthor=K.+Rumpelauthor=O.+Schaafauthor=S.+Steurerauthor=C.+Tallantauthor=C.+R.+Vakocauthor=M.+Zeebauthor=A.+Zoephelauthor=M.+Pearsonauthor=G.+Boehmeltauthor=D.+McConnell&title=Structure-based+design+of+an+in+vivo+active+selective+BRD9+inhibitor&doi=10.1021%2Facs.jmedchem.5b01865"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of an in Vivo Active Selective BRD9 Inhibitor</span></div><div class="casAuthors">Martin, Laetitia J.; Koegl, Manfred; Bader, Gerd; Cockcroft, Xiao-Ling; Fedorov, Oleg; Fiegen, Dennis; Gerstberger, Thomas; Hofmann, Marco H.; Hohmann, Anja F.; Kessler, Dirk; Knapp, Stefan; Knesl, Petr; Kornigg, Stefan; Mueller, Susanne; Nar, Herbert; Rogers, Catherine; Rumpel, Klaus; Schaaf, Otmar; Steurer, Steffen; Tallant, Cynthia; Vakoc, Christopher R.; Zeeb, Markus; Zoephel, Andreas; Pearson, Mark; Boehmelt, Guido; McConnell, Darryl</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4462-4475</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Components of the chromatin remodelling switch/sucrose nonfermentable (SWI/SNF) complex are recurrently mutated in tumors, suggesting that altering the activity of the complex plays a role in oncogenesis.  However, the role that the individual subunits play in this process is not clear.  We set out to develop an inhibitor compd. targeting the bromodomain of BRD9 in order to evaluate its function within the SWI/SNF complex.  Here, we present the discovery and development of a potent and selective BRD9 bromodomain inhibitor series based on a new pyridinone-like scaffold.  Crystallog. information on the inhibitors bound to BRD9 guided their development with respect to potency for BRD9 and selectivity against BRD4.  These compds. modulate BRD9 bromodomain cellular function and display antitumor activity in an AML xenograft model.  Two chem. probes, BI-7273 (1) and BI-9564 (2), were identified that should prove to be useful in further exploring BRD9 bromodomain biol. in both in vitro and in vivo settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWHs4hA8bijbVg90H21EOLACvtfcHk0ljyJPf_auzl6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjt1Sit7w%253D&md5=b1f4394a7a410273c800b65ee9697c7e</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01865&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01865%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DL.%2BJ.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DCockcroft%26aufirst%3DX.-L.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFiegen%26aufirst%3DD.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DHofmann%26aufirst%3DM.%2BH.%26aulast%3DHohmann%26aufirst%3DA.%2BF.%26aulast%3DKessler%26aufirst%3DD.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKnesl%26aufirst%3DP.%26aulast%3DKornigg%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DNar%26aufirst%3DH.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DSchaaf%26aufirst%3DO.%26aulast%3DSteurer%26aufirst%3DS.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DZeeb%26aufirst%3DM.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DBoehmelt%26aufirst%3DG.%26aulast%3DMcConnell%26aufirst%3DD.%26atitle%3DStructure-based%2520design%2520of%2520an%2520in%2520vivo%2520active%2520selective%2520BRD9%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4462%26epage%3D4475%26doi%3D10.1021%2Facs.jmedchem.5b01865" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sutherell, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yapp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siejka, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenton, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ley, S. V.</span></span> <span> </span><span class="NLM_article-title">Identification and development of 2,3-dihydropyrrolo1,2-aquinazolin-5(1H)-one inhibitors targeting bromodomains within the switch/sucrose nonfermenting complex</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">5095</span>– <span class="NLM_lpage">5101</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01997</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01997" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmsl2rt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5095-5101&author=C.+L.+Sutherellauthor=C.+Tallantauthor=O.+P.+Monteiroauthor=C.+Yappauthor=J.+E.+Fuchsauthor=O.+Fedorovauthor=P.+Siejkaauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=J.+D.+Brentonauthor=P.+E.+Brennanauthor=S.+V.+Ley&title=Identification+and+development+of+2%2C3-dihydropyrrolo1%2C2-aquinazolin-5%281H%29-one+inhibitors+targeting+bromodomains+within+the+switch%2Fsucrose+nonfermenting+complex&doi=10.1021%2Facs.jmedchem.5b01997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Development of 2,3-Dihydropyrrolo[1,2-a]quinazolin-5(1H)-one Inhibitors Targeting Bromodomains within the Switch/Sucrose Nonfermenting Complex</span></div><div class="casAuthors">Sutherell, Charlotte L.; Tallant, Cynthia; Monteiro, Octovia P.; Yapp, Clarence; Fuchs, Julian E.; Fedorov, Oleg; Siejka, Paulina; Muller, Suzanne; Knapp, Stefan; Brenton, James D.; Brennan, Paul E.; Ley, Steven V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5095-5101</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain contg. proteins PB1, SMARCA4, and SMARCA2 are important components of SWI/SNF chromatin remodeling complexes.  We identified bromodomain inhibitors that target these proteins and display unusual binding modes involving water displacement from the KAc binding site.  The best compd. (I) binds the fifth bromodomain of PB1 with a KD of 124 nM, SMARCA2B and SMARCA4 with KD values of 262 and 417 nM, resp., and displays excellent selectivity over bromodomains other than PB1, SMARCA2, and SMARCA4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfnWlpl3TimLVg90H21EOLACvtfcHk0lgORBFl90-L8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmsl2rt7s%253D&md5=a27afa2c97bd74462a09c9ec231f61ef</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01997%26sid%3Dliteratum%253Aachs%26aulast%3DSutherell%26aufirst%3DC.%2BL.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DFuchs%26aufirst%3DJ.%2BE.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DSiejka%26aufirst%3DP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrenton%26aufirst%3DJ.%2BD.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DLey%26aufirst%3DS.%2BV.%26atitle%3DIdentification%2520and%2520development%2520of%25202%252C3-dihydropyrrolo1%252C2-aquinazolin-5%25281H%2529-one%2520inhibitors%2520targeting%2520bromodomains%2520within%2520the%2520switch%252Fsucrose%2520nonfermenting%2520complex%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5095%26epage%3D5101%26doi%3D10.1021%2Facs.jmedchem.5b01997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Myrianthopoulos, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaboriaud-Kolar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grigoriou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brownlee, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heidenreich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wanior, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drosos, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mexia, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagratuni, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kastritis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terpos, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skaltsounis, A.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Downs, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikros, E.</span></span> <span> </span><span class="NLM_article-title">Discovery and optimization of a selective ligand for the switch/sucrose nonfermenting-related bromodomains of polybromo protein-1 by the use of virtual screening and hydration analysis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">8787</span>– <span class="NLM_lpage">8803</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00355</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00355" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVyqtLrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8787-8803&author=V.+Myrianthopoulosauthor=N.+Gaboriaud-Kolarauthor=C.+Tallantauthor=M.-L.+Hallauthor=S.+Grigoriouauthor=P.+M.+Brownleeauthor=O.+Fedorovauthor=C.+Rogersauthor=D.+Heidenreichauthor=M.+Waniorauthor=N.+Drososauthor=N.+Mexiaauthor=P.+Savitskyauthor=T.+Bagratuniauthor=E.+Kastritisauthor=E.+Terposauthor=P.+Filippakopoulosauthor=S.+M%C3%BCllerauthor=A.-L.+Skaltsounisauthor=J.+A.+Downsauthor=S.+Knappauthor=E.+Mikros&title=Discovery+and+optimization+of+a+selective+ligand+for+the+switch%2Fsucrose+nonfermenting-related+bromodomains+of+polybromo+protein-1+by+the+use+of+virtual+screening+and+hydration+analysis&doi=10.1021%2Facs.jmedchem.6b00355"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of a Selective Ligand for the Switch/Sucrose Nonfermenting-Related Bromodomains of Polybromo Protein-1 by the Use of Virtual Screening and Hydration Analysis</span></div><div class="casAuthors">Myrianthopoulos, Vassilios; Gaboriaud-Kolar, Nicolas; Tallant, Cynthia; Hall, Michelle-Lynn; Grigoriou, Stylianos; Brownlee, Peter Moore; Fedorov, Oleg; Rogers, Catherine; Heidenreich, David; Wanior, Marek; Drosos, Nikolaos; Mexia, Nikitia; Savitsky, Pavel; Bagratuni, Tina; Kastritis, Efstathios; Terpos, Evangelos; Filippakopoulos, Panagis; Muller, Susanne; Skaltsounis, Alexios-Leandros; Downs, Jessica Ann; Knapp, Stefan; Mikros, Emmanuel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8787-8803</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are epigenetic interaction domains currently recognized as emerging drug targets for development of anticancer or anti-inflammatory agents.  In this study, development of a selective ligand of the fifth BRD of Polybromo protein-1 (PB1(5)) related to Switch/Sucrose Non-Fermenting (SWI/SNF) chromatin remodeling complexes is presented.  A compd. collection was evaluated by consensus virtual screening and a hit was identified.  The biophys. study of protein-ligand interactions was performed using x-ray crystallog. and isothermal titrn. calorimetry.  Collective data supported the hypothesis that affinity improvement could be achieved by enhancing interactions of the complex with the solvent.  The derived SAR along with free energy calcns. and a consensus hydration anal. using WaterMap and SZmap algorithms guided rational design of a set of novel analogs.  The most potent analog demonstrated high affinity of 3.3 μM and an excellent selectivity profile, thus comprising a promising lead for the development of chem. probes targeting PB1(5).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrMsChShl81rVg90H21EOLACvtfcHk0lgORBFl90-L8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVyqtLrL&md5=026dcc048ad31b7bf37c9a41b7562018</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00355&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00355%26sid%3Dliteratum%253Aachs%26aulast%3DMyrianthopoulos%26aufirst%3DV.%26aulast%3DGaboriaud-Kolar%26aufirst%3DN.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DHall%26aufirst%3DM.-L.%26aulast%3DGrigoriou%26aufirst%3DS.%26aulast%3DBrownlee%26aufirst%3DP.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DHeidenreich%26aufirst%3DD.%26aulast%3DWanior%26aufirst%3DM.%26aulast%3DDrosos%26aufirst%3DN.%26aulast%3DMexia%26aufirst%3DN.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DBagratuni%26aufirst%3DT.%26aulast%3DKastritis%26aufirst%3DE.%26aulast%3DTerpos%26aufirst%3DE.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DSkaltsounis%26aufirst%3DA.-L.%26aulast%3DDowns%26aufirst%3DJ.%2BA.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DMikros%26aufirst%3DE.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520a%2520selective%2520ligand%2520for%2520the%2520switch%252Fsucrose%2520nonfermenting-related%2520bromodomains%2520of%2520polybromo%2520protein-1%2520by%2520the%2520use%2520of%2520virtual%2520screening%2520and%2520hydration%2520analysis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8787%26epage%3D8803%26doi%3D10.1021%2Facs.jmedchem.6b00355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, L.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, H.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Identification of small molecule inhibitors targeting the SMARCA2 bromodomain from a high-throughput screening assay</span>. <i>Acta Pharmacol. Sin.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">1544</span>– <span class="NLM_lpage">1552</span>, <span class="refDoi"> DOI: 10.1038/aps.2017.188</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Faps.2017.188" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=29795359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsF2ltL7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2018&pages=1544-1552&author=T.+Luauthor=J.-C.+Huauthor=W.-C.+Luauthor=J.+Hanauthor=H.+Dingauthor=H.+Jiangauthor=Y.-Y.+Zhangauthor=L.-Y.+Yueauthor=S.-J.+Chenauthor=H.-L.+Jiangauthor=K.-X.+Chenauthor=H.-F.+Chaiauthor=C.+Luo&title=Identification+of+small+molecule+inhibitors+targeting+the+SMARCA2+bromodomain+from+a+high-throughput+screening+assay&doi=10.1038%2Faps.2017.188"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of small molecule inhibitors targeting the SMARCA2 bromodomain from a high-throughput screening assay</span></div><div class="casAuthors">Lu, Tian; Hu, Jun-chi; Lu, Wen-chao; Han, Jie; Ding, Hong; Jiang, Hao; Zhang, Yuan-yuan; Yue, Li-yan; Chen, Shi-jie; Jiang, Hua-liang; Chen, Kai-xian; Chai, Hui-fang; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Acta Pharmacologica Sinica</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1544-1552</span>CODEN:
                <span class="NLM_cas:coden">APSCG5</span>;
        ISSN:<span class="NLM_cas:issn">1671-4083</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">SMARCA2 is a crit. catalytic subunit of the switch/sucrose non-fermenting (SWI/SNF) chromatin remodeling complexes.  Dysregulation of SMARCA2 is assocd. with several diseases, including some cancers.  SMARCA2 is multi-domain protein contg. a bromodomain (BRD) that specifically recognizes acetylated lysine residues in histone tails, thus playing an important role in chromatin remodeling.  Many potent and specific inhibitors targeting other BRDs have recently been discovered and have been widely used for cancer treatments and biol. research.  However, hit discovery targeting SMARCA2-BRD is particularly lacking.  To date, there is a paucity of reported high-throughput screening (HTS) assays targeting the SMARCA2-BRD interface.  In this study, we developed an AlphaScreen HTS system for the discovery of SMARCA2-BRD inhibitors and optimized the physicochem. conditions including pH, salt concns. and detergent levels.  Through an established AlphaScreen-based high-throughput screening assay against an inhouse compd. library, DCSM06 was identified as a novel SMARCA2-BRD inhibitor with an IC50 value of 39.9±3.0 μmol/L.  Surface plasmon resonance demonstrated the binding between SMARCA2-BRD and DCSM06 (Kd=38.6 μmol/L).  A similarity-based analog search led to identification of DCSM06-05 with an IC50 value of 9.0±1.4 μmol/L.  Mol. docking was performed to predict the binding mode of DCSM06-05 and to decipher the structural basis of the infiuence of chem. modifications on inhibitor potency.  DCSM06-05 may be used as a starting point for further medicinal chem. optimization and could function as a chem. tool for SMARCA2-related functional studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPiXXz-0iPmrVg90H21EOLACvtfcHk0liLMl9QfZbMyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsF2ltL7F&md5=8f7d612af8147bd832236015322cd444</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Faps.2017.188&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Faps.2017.188%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DT.%26aulast%3DHu%26aufirst%3DJ.-C.%26aulast%3DLu%26aufirst%3DW.-C.%26aulast%3DHan%26aufirst%3DJ.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DY.-Y.%26aulast%3DYue%26aufirst%3DL.-Y.%26aulast%3DChen%26aufirst%3DS.-J.%26aulast%3DJiang%26aufirst%3DH.-L.%26aulast%3DChen%26aufirst%3DK.-X.%26aulast%3DChai%26aufirst%3DH.-F.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520small%2520molecule%2520inhibitors%2520targeting%2520the%2520SMARCA2%2520bromodomain%2520from%2520a%2520high-throughput%2520screening%2520assay%26jtitle%3DActa%2520Pharmacol.%2520Sin.%26date%3D2018%26volume%3D39%26spage%3D1544%26epage%3D1552%26doi%3D10.1038%2Faps.2017.188" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, P. G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vieira, L. C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singleton, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baronio, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Méndez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. J.</span></span> <span> </span><span class="NLM_article-title">LP99: Discovery and synthesis of the first selective BRD7/9 bromodomain inhibitor</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">6217</span>– <span class="NLM_lpage">6221</span>, <span class="refDoi"> DOI: 10.1002/anie.201501394</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1002%2Fanie.201501394" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtlGqsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=6217-6221&author=P.+G.+K.+Clarkauthor=L.+C.+C.+Vieiraauthor=C.+Tallantauthor=O.+Fedorovauthor=D.+C.+Singletonauthor=C.+M.+Rogersauthor=O.+P.+Monteiroauthor=J.+M.+Bennettauthor=R.+Baronioauthor=S.+M%C3%BCllerauthor=D.+L.+Danielsauthor=J.+M%C3%A9ndezauthor=S.+Knappauthor=P.+E.+Brennanauthor=D.+J.+Dixon&title=LP99%3A+Discovery+and+synthesis+of+the+first+selective+BRD7%2F9+bromodomain+inhibitor&doi=10.1002%2Fanie.201501394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">LP99: Discovery and Synthesis of the First Selective BRD7/9 Bromodomain Inhibitor</span></div><div class="casAuthors">Clark, Peter G. K.; Vieira, Lucas C. C.; Tallant, Cynthia; Fedorov, Oleg; Singleton, Dean C.; Rogers, Catherine M.; Monteiro, Octovia P.; Bennett, James M.; Baronio, Roberta; Mueller, Susanne; Daniels, Danette L.; Mendez, Jacqui; Knapp, Stefan; Brennan, Paul E.; Dixon, Darren J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6217-6221</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The bromodomain-contg. proteins BRD9 and BRD7 are part of the human SWI/SNF chromatin-remodeling complexes BAF and PBAF.  To date, no selective inhibitor for BRD7/9 has been reported despite its potential value as a biol. tool or as a lead for future therapeutics.  The quinolone-fused lactam LP99 (I) is now reported as the first potent and selective inhibitor of the BRD7 and BRD9 bromodomains.  Development of LP99 from a fragment hit was expedited through balancing structure-based inhibitor design and biophys. characterization against tractable chem. synthesis: Complexity-building nitro-Mannich/lactamization cascade processes allowed for early structure-activity relationship studies whereas an enantioselective organocatalytic nitro-Mannich reaction enabled the synthesis of the lead scaffold in enantioenriched form and on scale.  This epigenetic probe was shown to inhibit the assocn. of BRD7 and BRD9 to acetylated histones in vitro and in cells.  Moreover, LP99 was used to demonstrate that BRD7/9 plays a role in regulating pro-inflammatory cytokine secretion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplfOmbNMpdPbVg90H21EOLACvtfcHk0liLMl9QfZbMyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtlGqsrg%253D&md5=9dee2b352045f7ae5efcbdc83f240a3f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fanie.201501394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201501394%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DP.%2BG.%2BK.%26aulast%3DVieira%26aufirst%3DL.%2BC.%2BC.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DSingleton%26aufirst%3DD.%2BC.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DBennett%26aufirst%3DJ.%2BM.%26aulast%3DBaronio%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DM%25C3%25A9ndez%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DDixon%26aufirst%3DD.%2BJ.%26atitle%3DLP99%253A%2520Discovery%2520and%2520synthesis%2520of%2520the%2520first%2520selective%2520BRD7%252F9%2520bromodomain%2520inhibitor%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D6217%26epage%3D6221%26doi%3D10.1002%2Fanie.201501394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of potent and selective inhibitors of BRD7 and BRD9 bromodomains</span>. <i>Medchemcomm</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1381</span>– <span class="NLM_lpage">1386</span>, <span class="refDoi"> DOI: 10.1039/c5md00152h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1039%2Fc5md00152h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVSgtrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=1381-1386&author=D.+A.+Hayauthor=C.+M.+Rogersauthor=O.+Fedorovauthor=C.+Tallantauthor=S.+Martinauthor=O.+P.+Monteiroauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=C.+J.+Schofieldauthor=P.+E.+Brennan&title=Design+and+synthesis+of+potent+and+selective+inhibitors+of+BRD7+and+BRD9+bromodomains&doi=10.1039%2Fc5md00152h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Design and synthesis of potent and selective inhibitors of BRD7 and BRD9 bromodomains</span></div><div class="casAuthors">Hay, Duncan A.; Rogers, Catherine M.; Fedorov, Oleg; Tallant, Cynthia; Martin, Sarah; Monteiro, Octovia P.; Muller, Susanne; Knapp, Stefan; Schofield, Christopher J.; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1381-1386</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Emerging evidence suggests bromodomain-contg. proteins 7 and 9 (BRD7 and BRD9) have roles in the regulation of human transcription and disease including cancer.  We describe potent and selective inhibitors of the BRD7 and BRD9 bromodomains intended for use as tools to elucidate the biol. roles of BRD7 and BRD9 in healthy and diseased cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrynQ2AJnreDLVg90H21EOLACvtfcHk0lg-on5qiAhxIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVSgtrvJ&md5=6a93d7300f7815eaf5ca654f873078d3</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1039%2Fc5md00152h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc5md00152h%26sid%3Dliteratum%253Aachs%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520potent%2520and%2520selective%2520inhibitors%2520of%2520BRD7%2520and%2520BRD9%2520bromodomains%26jtitle%3DMedchemcomm%26date%3D2015%26volume%3D6%26spage%3D1381%26epage%3D1386%26doi%3D10.1039%2Fc5md00152h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span>; <span class="NLM_string-name">Cote, A.</span>; <span class="NLM_string-name">Crawford, T. D.</span>; <span class="NLM_string-name">Duplessis, M.</span>; <span class="NLM_string-name">Good, A. C.</span>; <span class="NLM_string-name">Leblanc, Y.</span>; <span class="NLM_string-name">Magnuson, S. R.</span>; <span class="NLM_string-name">Nasveschuk, C. G.</span>; <span class="NLM_string-name">Romero, A. F.</span>; <span class="NLM_string-name">Tang, Y.</span>; <span class="NLM_string-name">Taylor, A. M.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Pyridazine Compounds and Uses thereof</span>. U.S. Patent <span class="NLM_patent">WO 2016138114 A1</span>, Feburary 24, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=B.+K.+Albrecht&author=A.+Cote&author=T.+D.+Crawford&author=M.+Duplessis&author=A.+C.+Good&author=Y.+Leblanc&author=S.+R.+Magnuson&author=C.+G.+Nasveschuk&author=A.+F.+Romero&author=Y.+Tang&author=A.+M.+Taylor&title=Therapeutic+Pyridazine+Compounds+and+Uses+thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26atitle%3DTherapeutic%2520Pyridazine%2520Compounds%2520and%2520Uses%2520thereof%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Theodoulou, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bit, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leveridge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molnar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Discovery of I-BRD9, a selective cell active chemical probe for bromodomain containing protein 9 inhibition</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1425</span>– <span class="NLM_lpage">1439</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00256</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00256" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtlaktb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1425-1439&author=N.+H.+Theodoulouauthor=P.+Bamboroughauthor=A.+J.+Bannisterauthor=I.+Becherauthor=R.+A.+Bitauthor=K.+H.+Cheauthor=C.-W.+Chungauthor=A.+Dittmannauthor=G.+Drewesauthor=D.+H.+Drewryauthor=L.+Gordonauthor=P.+Grandiauthor=M.+Leveridgeauthor=M.+Lindonauthor=A.-M.+Michonauthor=J.+Molnarauthor=S.+C.+Robsonauthor=N.+C.+O.+Tomkinsonauthor=T.+Kouzaridesauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Discovery+of+I-BRD9%2C+a+selective+cell+active+chemical+probe+for+bromodomain+containing+protein+9+inhibition&doi=10.1021%2Facs.jmedchem.5b00256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of I-BRD9, a Selective Cell Active Chemical Probe for Bromodomain Containing Protein 9 Inhibition</span></div><div class="casAuthors">Theodoulou, Natalie H.; Bamborough, Paul; Bannister, Andrew J.; Becher, Isabelle; Bit, Rino A.; Che, Ka Hing; Chung, Chun-wa; Dittmann, Antje; Drewes, Gerard; Drewry, David H.; Gordon, Laurie; Grandi, Paola; Leveridge, Melanie; Lindon, Matthew; Michon, Anne-Marie; Molnar, Judit; Robson, Samuel C.; Tomkinson, Nicholas C. O.; Kouzarides, Tony; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1425-1439</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Acetylation of histone lysine residues is one of the most well-studied post-translational modifications of chromatin, selectively recognized by bromodomain "reader" modules.  Inhibitors of the bromodomain and extra terminal domain (BET) family of bromodomains have shown profound anticancer and anti-inflammatory properties, generating much interest in targeting other bromodomain-contg. proteins for disease treatment.  Herein, we report the discovery of I-BRD9, the first selective cellular chem. probe for bromodomain-contg. protein 9 (BRD9).  I-BRD9 was identified through structure-based design, leading to greater than 700-fold selectivity over the BET family and 200-fold over the highly homologous bromodomain-contg. protein 7 (BRD7).  I-BRD9 was used to identify genes regulated by BRD9 in Kasumi-1 cells involved in oncol. and immune response pathways and to the best of our knowledge, represents the first selective tool compd. available to elucidate the cellular phenotype of BRD9 bromodomain inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVmjhksIDyTrVg90H21EOLACvtfcHk0lg-on5qiAhxIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtlaktb8%253D&md5=4e8f6aa664797fe54772b64c81cf2a83</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00256%26sid%3Dliteratum%253Aachs%26aulast%3DTheodoulou%26aufirst%3DN.%2BH.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DBecher%26aufirst%3DI.%26aulast%3DBit%26aufirst%3DR.%2BA.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DDittmann%26aufirst%3DA.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DLeveridge%26aufirst%3DM.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DMolnar%26aufirst%3DJ.%26aulast%3DRobson%26aufirst%3DS.%2BC.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DDiscovery%2520of%2520I-BRD9%252C%2520a%2520selective%2520cell%2520active%2520chemical%2520probe%2520for%2520bromodomain%2520containing%2520protein%25209%2520inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1425%26epage%3D1439%26doi%3D10.1021%2Facs.jmedchem.5b00256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Remillard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brien, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonnett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wühr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Degradation of the BAF complex factor BRD9 by heterobifunctional ligands</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">5738</span>– <span class="NLM_lpage">5743</span>, <span class="refDoi"> DOI: 10.1002/anie.201611281</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1002%2Fanie.201611281" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtFGrurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=5738-5743&author=D.+Remillardauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=G.+L.+Brienauthor=M.+Sonnettauthor=H.-S.+Seoauthor=S.+Dastjerdiauthor=M.+W%C3%BChrauthor=S.+Dhe-Paganonauthor=S.+A.+Armstrongauthor=J.+E.+Bradner&title=Degradation+of+the+BAF+complex+factor+BRD9+by+heterobifunctional+ligands&doi=10.1002%2Fanie.201611281"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Degradation of the BAF Complex Factor BRD9 by Heterobifunctional Ligands</span></div><div class="casAuthors">Remillard, David; Buckley, Dennis L.; Paulk, Joshiawa; Brien, Gerard L.; Sonnett, Matthew; Seo, Hyuk-Soo; Dastierdi, Shiva; Wuehr, Martin; Dhe-Paganon, Sirano; Armstrong, Scott A.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5738-5743</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The bromodomain-contg. protein BRD9, a subunit of the human BAF (SWI/SNF) nucleosome remodeling complex, has emerged as an attractive therapeutic target in cancer.  Despite the development of chem. probes targeting the BRD9 bromodomain, there is a limited understanding of BRD9 function beyond acetyl-lysine recognition.  We have therefore created the first BRD9-directed chem. degraders, through iterative design and testing of heterobifunctional ligands that bridge the BRD9 bromodomain and the cereblon E3 ubiquitin ligase complex.  Degraders of BRD9 exhibit markedly enhanced potency compared to parental ligands (10- to 100-fold).  Parallel study of degraders with divergent BRD9-binding chemotypes in models of acute myeloid leukemia resolves bromodomain polypharmacol. in this emerging drug class.  Together, these findings reveal the tractability of non-BET bromodomain contg. proteins to chem. degrdn., and highlight lead compd. dBRD9 as a tool for the study of BRD9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ9FU2Ipaq27Vg90H21EOLACvtfcHk0li3coH00khDDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtFGrurk%253D&md5=2eb6562cc9d976c7ae6070c9a419e9ea</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fanie.201611281&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201611281%26sid%3Dliteratum%253Aachs%26aulast%3DRemillard%26aufirst%3DD.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DBrien%26aufirst%3DG.%2BL.%26aulast%3DSonnett%26aufirst%3DM.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DW%25C3%25BChr%26aufirst%3DM.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDegradation%2520of%2520the%2520BAF%2520complex%2520factor%2520BRD9%2520by%2520heterobifunctional%2520ligands%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D5738%26epage%3D5743%26doi%3D10.1002%2Fanie.201611281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Farnaby, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitworth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diers, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trainor, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zollman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karolyi-Oezguer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riedmueller, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wachter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dank, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharps, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traxler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnitzer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petermann, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstabl, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss-Puxbaum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrenhöfer-Wölfer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wöhrle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehmelt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinnenthal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnhof, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiechens, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen-Hughes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettmayer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">BAF complex vulnerabilities in cancer demonstrated via structure-based PROTAC design</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">672</span>– <span class="NLM_lpage">680</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0294-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fs41589-019-0294-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=31178587" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFeqtLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=672-680&author=W.+Farnabyauthor=M.+Koeglauthor=M.+J.+Royauthor=C.+Whitworthauthor=E.+Diersauthor=N.+Trainorauthor=D.+Zollmanauthor=S.+Steurerauthor=J.+Karolyi-Oezguerauthor=C.+Riedmuellerauthor=T.+Gmaschitzauthor=J.+Wachterauthor=C.+Dankauthor=M.+Galantauthor=B.+Sharpsauthor=K.+Rumpelauthor=E.+Traxlerauthor=T.+Gerstbergerauthor=R.+Schnitzerauthor=O.+Petermannauthor=P.+Grebauthor=H.+Weinstablauthor=G.+Baderauthor=A.+Zoephelauthor=A.+Weiss-Puxbaumauthor=K.+Ehrenh%C3%B6fer-W%C3%B6lferauthor=S.+W%C3%B6hrleauthor=G.+Boehmeltauthor=J.+Rinnenthalauthor=H.+Arnhofauthor=N.+Wiechensauthor=M.-Y.+Wuauthor=T.+Owen-Hughesauthor=P.+Ettmayerauthor=M.+Pearsonauthor=D.+B.+McConnellauthor=A.+Ciulli&title=BAF+complex+vulnerabilities+in+cancer+demonstrated+via+structure-based+PROTAC+design&doi=10.1038%2Fs41589-019-0294-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">BAF complex vulnerabilities in cancer demonstrated via structure-based PROTAC design</span></div><div class="casAuthors">Farnaby, William; Koegl, Manfred; Roy, Michael J.; Whitworth, Claire; Diers, Emelyne; Trainor, Nicole; Zollman, David; Steurer, Steffen; Karolyi-Oezguer, Jale; Riedmueller, Carina; Gmaschitz, Teresa; Wachter, Johannes; Dank, Christian; Galant, Michael; Sharps, Bernadette; Rumpel, Klaus; Traxler, Elisabeth; Gerstberger, Thomas; Schnitzer, Renate; Petermann, Oliver; Greb, Peter; Weinstabl, Harald; Bader, Gerd; Zoephel, Andreas; Weiss-Puxbaum, Alexander; Ehrenhofer-Wolfer, Katharina; Wohrle, Simon; Boehmelt, Guido; Rinnenthal, Joerg; Arnhof, Heribert; Wiechens, Nicola; Wu, Meng-Ying; Owen-Hughes, Tom; Ettmayer, Peter; Pearson, Mark; McConnell, Darryl B.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">672-680</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Targeting subunits of BAF/PBAF chromatin remodeling complexes has been proposed as an approach to exploit cancer vulnerabilities.  Here, we develop proteolysis targeting chimera (PROTAC) degraders of the BAF ATPase subunits SMARCA2 and SMARCA4 using a bromodomain ligand and recruitment of the E3 ubiquitin ligase VHL.  High-resoln. ternary complex crystal structures and biophys. investigation guided rational and efficient optimization toward ACBI1(I), a potent and cooperative degrader of SMARCA2, SMARCA4 and PBRM1.  I induced anti-proliferative effects and cell death caused by SMARCA2 depletion in SMARCA4 mutant cancer cells, and in acute myeloid leukemia cells dependent on SMARCA4 ATPase activity.  These findings exemplify a successful biophysics- and structure-based PROTAC design approach to degrade high profile drug targets, and pave the way toward new therapeutics for the treatment of tumors sensitive to the loss of BAF complex ATPases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonEh4pux4Bs7Vg90H21EOLACvtfcHk0li3coH00khDDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFeqtLfJ&md5=e95fad12c727aedcf08aed9320dae905</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0294-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0294-6%26sid%3Dliteratum%253Aachs%26aulast%3DFarnaby%26aufirst%3DW.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DRoy%26aufirst%3DM.%2BJ.%26aulast%3DWhitworth%26aufirst%3DC.%26aulast%3DDiers%26aufirst%3DE.%26aulast%3DTrainor%26aufirst%3DN.%26aulast%3DZollman%26aufirst%3DD.%26aulast%3DSteurer%26aufirst%3DS.%26aulast%3DKarolyi-Oezguer%26aufirst%3DJ.%26aulast%3DRiedmueller%26aufirst%3DC.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DWachter%26aufirst%3DJ.%26aulast%3DDank%26aufirst%3DC.%26aulast%3DGalant%26aufirst%3DM.%26aulast%3DSharps%26aufirst%3DB.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DTraxler%26aufirst%3DE.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DSchnitzer%26aufirst%3DR.%26aulast%3DPetermann%26aufirst%3DO.%26aulast%3DGreb%26aufirst%3DP.%26aulast%3DWeinstabl%26aufirst%3DH.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DWeiss-Puxbaum%26aufirst%3DA.%26aulast%3DEhrenh%25C3%25B6fer-W%25C3%25B6lfer%26aufirst%3DK.%26aulast%3DW%25C3%25B6hrle%26aufirst%3DS.%26aulast%3DBoehmelt%26aufirst%3DG.%26aulast%3DRinnenthal%26aufirst%3DJ.%26aulast%3DArnhof%26aufirst%3DH.%26aulast%3DWiechens%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DM.-Y.%26aulast%3DOwen-Hughes%26aufirst%3DT.%26aulast%3DEttmayer%26aufirst%3DP.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DMcConnell%26aufirst%3DD.%2BB.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DBAF%2520complex%2520vulnerabilities%2520in%2520cancer%2520demonstrated%2520via%2520structure-based%2520PROTAC%2520design%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D672%26epage%3D680%26doi%3D10.1038%2Fs41589-019-0294-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brien, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remillard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hemming, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chabon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wynne, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cagney, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Mierlo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baltissen, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vermeulen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fröhling, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span> <span> </span><span class="NLM_article-title">Targeted degradation of BRD9 reverses oncogenic gene expression in synovial sarcoma</span>. <i>Elife</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e41305</span> <span class="refDoi"> DOI: 10.7554/elife.41305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.7554%2FeLife.41305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=30431433" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlyqs7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&author=G.+L.+Brienauthor=D.+Remillardauthor=J.+Shiauthor=M.+L.+Hemmingauthor=J.+Chabonauthor=K.+Wynneauthor=E.+T.+Dillonauthor=G.+Cagneyauthor=G.+van+Mierloauthor=M.+P.+Baltissenauthor=M.+Vermeulenauthor=J.+Qiauthor=S.+Fr%C3%B6hlingauthor=N.+S.+Grayauthor=J.+E.+Bradnerauthor=C.+R.+Vakocauthor=S.+A.+Armstrong&title=Targeted+degradation+of+BRD9+reverses+oncogenic+gene+expression+in+synovial+sarcoma&doi=10.7554%2Felife.41305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted degradation of BRD9 reverses oncogenic gene expression in synovial sarcoma</span></div><div class="casAuthors">Brien, Gerard L.; Remillard, David; Shi, Junwei; Hemming, Matthew L.; Chabon, Jonathon; Wynne, Kieran; Dillon, Euge Ne T.; Cagney, Gerard; Van Mierlo, Guido; Baltissen, Marijke P.; Vermeulen, Michiel; Qi, Jun; Froehling, Stefan; Gray, Nathanael S.; Bradner, James E.; Vakoc, Christopher R.; Armstrong, Scott A.</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e41305/1-e41305/26</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Synovial sarcoma tumors contain a characteristic fusion protein, SS18-SSX, which drives disease development.  Targeting oncogenic fusion proteins presents an attractive therapeutic opportunity.  However, SS18-SSX has proven intractable for therapeutic intervention.  Using a domain-focused CRISPR screen we identified the bromodomain of BRD9 as a crit. functional dependency in synovial sarcoma.  BRD9 is a component of SS18-SSX contg. BAF complexes in synovial sarcoma cells; and integration of BRD9 into these complexes is crit. for cell growth.  Moreover BRD9 and SS18-SSX co-localize extensively on the synovial sarcoma genome.  Remarkably, synovial sarcoma cells are highly sensitive to a novel small mol. degrader of BRD9, while other sarcoma subtypes are unaffected.  Degrdn. of BRD9 induces downregulation of oncogenic transcriptional programs and inhibits tumor progression in vivo.  We demonstrate that BRD9 supports oncogenic mechanisms underlying the SS18-SSX fusion in synovial sarcoma and highlight targeted degrdn. of BRD9 as a potential therapeutic opportunity in this disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfd6-D9YlmrrVg90H21EOLACvtfcHk0ljHMuu5FMP3Ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlyqs7fF&md5=cbda64457406cd0d8fbd1211678b04b3</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.7554%2FeLife.41305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.41305%26sid%3Dliteratum%253Aachs%26aulast%3DBrien%26aufirst%3DG.%2BL.%26aulast%3DRemillard%26aufirst%3DD.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DHemming%26aufirst%3DM.%2BL.%26aulast%3DChabon%26aufirst%3DJ.%26aulast%3DWynne%26aufirst%3DK.%26aulast%3DDillon%26aufirst%3DE.%2BT.%26aulast%3DCagney%26aufirst%3DG.%26aulast%3Dvan%2BMierlo%26aufirst%3DG.%26aulast%3DBaltissen%26aufirst%3DM.%2BP.%26aulast%3DVermeulen%26aufirst%3DM.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DFr%25C3%25B6hling%26aufirst%3DS.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26atitle%3DTargeted%2520degradation%2520of%2520BRD9%2520reverses%2520oncogenic%2520gene%2520expression%2520in%2520synovial%2520sarcoma%26jtitle%3DElife%26date%3D2018%26volume%3D7%26doi%3D10.7554%2Felife.41305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castex, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aldeghi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trzupek, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstenberger, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggin, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schüle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Günther, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span> <span> </span><span class="NLM_article-title">Selective targeting of the BRG/PB1 bromodomains impairs embryonic and trophoblast stem cell maintenance</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1</i></span>, <span class="NLM_elocation-id">e1500723</span> <span class="refDoi"> DOI: 10.1126/sciadv.1500723</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1126%2Fsciadv.1500723" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=26702435" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC28XovFaktLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&author=O.+Fedorovauthor=J.+Castexauthor=C.+Tallantauthor=D.+R.+Owenauthor=S.+Martinauthor=M.+Aldeghiauthor=O.+Monteiroauthor=P.+Filippakopoulosauthor=S.+Picaudauthor=J.+D.+Trzupekauthor=B.+S.+Gerstenbergerauthor=C.+Bountraauthor=D.+Willmannauthor=C.+Wellsauthor=M.+Philpottauthor=C.+Rogersauthor=P.+C.+Bigginauthor=P.+E.+Brennanauthor=M.+E.+Bunnageauthor=R.+Sch%C3%BCleauthor=T.+G%C3%BCntherauthor=S.+Knappauthor=S.+M%C3%BCller&title=Selective+targeting+of+the+BRG%2FPB1+bromodomains+impairs+embryonic+and+trophoblast+stem+cell+maintenance&doi=10.1126%2Fsciadv.1500723"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Selective targeting of the BRG/PB1 bromodomains impairs embryonic and trophoblast stem cell maintenance</span></div><div class="casAuthors">Fedorov, Oleg; Castex, Josefina; Tallant, Cynthia; Owen, Dafydd R.; Martin, Sarah; Aldeghi, Matteo; Monteiro, Octovia; Filippakopoulos, Panagis; Picaud, Sarah; Trzupek, John D.; Gerstenberger, Brian S.; Bountra, Chas; Willmann, Dominica; Wells, Christopher; Philpott, Martin; Rogers, Catherine; Biggin, Philip C.; Brennan, Paul E.; Bunnage, Mark E.; Schule, Roland; Gunther, Thomas; Knapp, Stefan; Muller, Susanne</div><div class="citationInfo"><span class="NLM_cas:title">Science Advances</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1500723/1-e1500723/11</span>CODEN:
                <span class="NLM_cas:coden">SACDAF</span>;
        ISSN:<span class="NLM_cas:issn">2375-2548</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Mammalian SWI/SNF [also called Brg/Brahma-assocd. factors (BAFs)] are evolutionarily conserved chromatin remodeling complexes regulating gene transcription programs during development and stem cell differentiation.  BAF complexes contain an ATP (ATP)-driven remodeling enzyme (either BRG1 or BRM) and multiple protein interaction domains including bromodomains, an evolutionary conserved acetyl lysine-dependent protein interaction motif that recruits transcriptional regulators to acetylated chromatin.  We report a potent and cell active protein interaction inhibitor, PFI-3, that selectively binds to essential BAF bromodomains.  The high specificity of PFI-3 was achieved on the basis of a novel binding mode of a salicylic acid head group that led to the replacement of water mols. typically maintained in other bromodomain inhibitor complexes.  We show that exposure of embryonic stem cells to PFI-3 led to deprivation of stemness and deregulated lineage specification.  Furthermore, differentiation of trophoblast stem cells in the presence of PFI-3 was markedly enhanced.  The data present a key function of BAF bromodomains in stem cell maintenance and differentiation, introducing a novel versatile chem. probe for studies on acetylation-dependent cellular processes controlled by BAF remodeling complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiGyzl98dGOLVg90H21EOLACvtfcHk0ljHMuu5FMP3Ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XovFaktLw%253D&md5=52757e667cc5afeb4c3a151fc8ef3442</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.1500723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.1500723%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DCastex%26aufirst%3DJ.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DAldeghi%26aufirst%3DM.%26aulast%3DMonteiro%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DTrzupek%26aufirst%3DJ.%2BD.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DWillmann%26aufirst%3DD.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DBiggin%26aufirst%3DP.%2BC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DG%25C3%25BCnther%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26atitle%3DSelective%2520targeting%2520of%2520the%2520BRG%252FPB1%2520bromodomains%2520impairs%2520embryonic%2520and%2520trophoblast%2520stem%2520cell%2520maintenance%26jtitle%3DSci.%2520Adv.%26date%3D2015%26volume%3D1%26doi%3D10.1126%2Fsciadv.1500723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ho, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ronan, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staahl, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lessard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nesvizhskii, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranish, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crabtree, G. R.</span></span> <span> </span><span class="NLM_article-title">An embryonic stem cell chromatin remodeling complex, esBAF, is essential for embryonic stem cell self-renewal and pluripotency</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">5181</span>– <span class="NLM_lpage">5186</span>, <span class="refDoi"> DOI: 10.1073/pnas.0812889106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1073%2Fpnas.0812889106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=19279220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksVert70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=5181-5186&author=L.+Hoauthor=J.+L.+Ronanauthor=J.+Wuauthor=B.+T.+Staahlauthor=L.+Chenauthor=A.+Kuoauthor=J.+Lessardauthor=A.+I.+Nesvizhskiiauthor=J.+Ranishauthor=G.+R.+Crabtree&title=An+embryonic+stem+cell+chromatin+remodeling+complex%2C+esBAF%2C+is+essential+for+embryonic+stem+cell+self-renewal+and+pluripotency&doi=10.1073%2Fpnas.0812889106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">An embryonic stem cell chromatin remodeling complex, esBAF, is essential for embryonic stem cell self-renewal and pluripotency</span></div><div class="casAuthors">Ho, Lena; Ronan, Jehnna L.; Wu, Jiang; Staahl, Brett T.; Chen, Lei; Kuo, Ann; Lessard, Julie; Nesvizhskii, Alexey I.; Ranish, Jeff; Crabtree, Gerald R.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5181-5186</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Mammalian SWI/SNF [also called BAF (Brg/Brahma-assocd. factors)] ATP-dependent chromatin remodeling complexes are essential for the formation of the totipotent and pluripotent cells of the early embryo.  In addn., subunits of this complex have been recovered in screens for genes required for nuclear reprogramming in Xenopus and mouse embryonic stem cell (ES cell) morphol.  However, the mechanism underlying the roles of these complexes is unclear.  Here, the authors show that BAF complexes are required for the self-renewal and pluripotency of mouse ES cells but not for the proliferation of fibroblasts or other cells.  The authors' proteomic studies revealed that ES cells expressed distinctive complexes (esBAF) defined by the presence of Brg (Brahma-related gene), BAF155, and BAF60A, and the absence of Brm (Brahma), BAF170, and BAF60C.  The authors showed that this specialized subunit compn. was required for ES cell maintenance and pluripotency.  The authors' proteomic anal. also revealed that esBAF complexes interact directly with key regulators of pluripotency, suggesting that esBAF complexes are specialized to interact with ES cell-specific regulators, providing a potential explanation for the requirement of BAF complexes in pluripotency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIHWi94C9GwbVg90H21EOLACvtfcHk0lihf6SaflBk3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksVert70%253D&md5=c9e9b1417ceac18a02b1d29be51cefdd</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0812889106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0812889106%26sid%3Dliteratum%253Aachs%26aulast%3DHo%26aufirst%3DL.%26aulast%3DRonan%26aufirst%3DJ.%2BL.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DStaahl%26aufirst%3DB.%2BT.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DKuo%26aufirst%3DA.%26aulast%3DLessard%26aufirst%3DJ.%26aulast%3DNesvizhskii%26aufirst%3DA.%2BI.%26aulast%3DRanish%26aufirst%3DJ.%26aulast%3DCrabtree%26aufirst%3DG.%2BR.%26atitle%3DAn%2520embryonic%2520stem%2520cell%2520chromatin%2520remodeling%2520complex%252C%2520esBAF%252C%2520is%2520essential%2520for%2520embryonic%2520stem%2520cell%2520self-renewal%2520and%2520pluripotency%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2009%26volume%3D106%26spage%3D5181%26epage%3D5186%26doi%3D10.1073%2Fpnas.0812889106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mailman, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Archer, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">Transcriptional repression by the BRG1-SWI/SNF complex affects the pluripotency of human embryonic stem cells</span>. <i>Stem Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">460</span>– <span class="NLM_lpage">474</span>, <span class="refDoi"> DOI: 10.1016/j.stemcr.2014.07.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2Fj.stemcr.2014.07.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=25241744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlKrtr%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&pages=460-474&author=X.+Zhangauthor=B.+Liauthor=W.+Liauthor=L.+Maauthor=D.+Zhengauthor=L.+Liauthor=W.+Yangauthor=M.+Chuauthor=W.+Chenauthor=R.+B.+Mailmanauthor=J.+Zhuauthor=G.+Fanauthor=T.+K.+Archerauthor=Y.+Wang&title=Transcriptional+repression+by+the+BRG1-SWI%2FSNF+complex+affects+the+pluripotency+of+human+embryonic+stem+cells&doi=10.1016%2Fj.stemcr.2014.07.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional Repression by the BRG1-SWI/SNF Complex Affects the Pluripotency of Human Embryonic Stem Cells</span></div><div class="casAuthors">Zhang, Xiaoli; Li, Bing; Li, Wenguo; Ma, Lijuan; Zheng, Dongyan; Li, Leping; Yang, Weijing; Chu, Min; Chen, Wei; Mailman, Richard B.; Zhu, Jun; Fan, Guoping; Archer, Trevor K.; Wang, Yuan</div><div class="citationInfo"><span class="NLM_cas:title">Stem Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">460-474</span>CODEN:
                <span class="NLM_cas:coden">SCRTFL</span>;
        ISSN:<span class="NLM_cas:issn">2213-6711</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The SWI/SNF complex plays an important role in mouse embryonic stem cells (mESCs), but it remains to be detd. whether this complex is required for the pluripotency of human ESCs (hESCs).  Using RNAi, we demonstrated that depletion of BRG1, the catalytic subunit of the SWI/SNF complex, led to impaired self-renewing ability and dysregulated lineage specification of hESCs.  A unique compn. of the BRG1-SWI/SNF complex in hESCs was further defined by the presence of BRG1, BAF250A, BAF170, BAF155, BAF53A, and BAF47.  Genome-wide expression analyses revealed that BRG1 participated in a broad range of biol. processes in hESCs through pathways different from those in mESCs.  In addn., chromatin immunopptn. sequencing (ChIP-seq) demonstrated that BRG1 played a repressive role in transcriptional regulation by modulating the acetylation levels of H3K27 at the enhancers of lineage-specific genes.  Our data thus provide valuable insights into mol. mechanisms by which transcriptional repression affects the self-renewal and differentiation of hESCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGA2baO4Y6DLVg90H21EOLACvtfcHk0lihf6SaflBk3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlKrtr%252FJ&md5=b7a9d386e584eeed58b412f25cd07d61</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.stemcr.2014.07.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.stemcr.2014.07.004%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DZheng%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DChu%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DMailman%26aufirst%3DR.%2BB.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DFan%26aufirst%3DG.%26aulast%3DArcher%26aufirst%3DT.%2BK.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DTranscriptional%2520repression%2520by%2520the%2520BRG1-SWI%252FSNF%2520complex%2520affects%2520the%2520pluripotency%2520of%2520human%2520embryonic%2520stem%2520cells%26jtitle%3DStem%2520Cell%2520Rep.%26date%3D2014%26volume%3D3%26spage%3D460%26epage%3D474%26doi%3D10.1016%2Fj.stemcr.2014.07.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klochendler-Yeivin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiette, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barra, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muchardt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babinet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yaniv, M.</span></span> <span> </span><span class="NLM_article-title">The murine SNF5/INI1 chromatin remodeling factor is essential for embryonic development and tumor suppression</span>. <i>EMBO Rep.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">500</span>– <span class="NLM_lpage">506</span>, <span class="refDoi"> DOI: 10.1093/embo-reports/kvd129</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1093%2Fembo-reports%2Fkvd129" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=11263494" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjt1ejtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2000&pages=500-506&author=A.+Klochendler-Yeivinauthor=L.+Fietteauthor=J.+Barraauthor=C.+Muchardtauthor=C.+Babinetauthor=M.+Yaniv&title=The+murine+SNF5%2FINI1+chromatin+remodeling+factor+is+essential+for+embryonic+development+and+tumor+suppression&doi=10.1093%2Fembo-reports%2Fkvd129"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">The murine SNF5/INI1 chromatin remodeling factor is essential for embryonic development and tumor suppression</span></div><div class="casAuthors">Klochendler-Yeivin, Agnes; Fiette, Laurence; Barra, Jaqueline; Muchardt, Christian; Babinet, Charles; Yaniv, Moshe</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Reports</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">500-506</span>CODEN:
                <span class="NLM_cas:coden">ERMEAX</span>;
        ISSN:<span class="NLM_cas:issn">1469-221X</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The assembly of eukaryotic DNA into nucleosomes and derived higher order structures constitutes a barrier for transcription, replication and repair.  A no. of chromatin remodeling complexes, as well as histone acetylation, were shown to facilitate gene activation.  To investigate the function of two closely related mammalian SWI/SNF complexes in vivo, we inactivated the murine SNF5/INI1 gene, a common subunit of these two complexes.  Mice lacking SNF5 protein stop developing at the peri-implantation stage, showing that the SWI/SNF complex is essential for early development and viability of early embryonic cells.  Furthermore, heterozygous mice develop nervous system and soft tissue sarcomas.  In these tumors the wild-type allele was lost, providing further evidence that SNF5 functions as a tumor suppressor gene in certain cell types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2fPp6F-3PPLVg90H21EOLACvtfcHk0lihf6SaflBk3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjt1ejtLc%253D&md5=876ba7fa7fffebb2e53bc359f8b3cffe</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1093%2Fembo-reports%2Fkvd129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fembo-reports%252Fkvd129%26sid%3Dliteratum%253Aachs%26aulast%3DKlochendler-Yeivin%26aufirst%3DA.%26aulast%3DFiette%26aufirst%3DL.%26aulast%3DBarra%26aufirst%3DJ.%26aulast%3DMuchardt%26aufirst%3DC.%26aulast%3DBabinet%26aufirst%3DC.%26aulast%3DYaniv%26aufirst%3DM.%26atitle%3DThe%2520murine%2520SNF5%252FINI1%2520chromatin%2520remodeling%2520factor%2520is%2520essential%2520for%2520embryonic%2520development%2520and%2520tumor%2520suppression%26jtitle%3DEMBO%2520Rep.%26date%3D2000%26volume%3D1%26spage%3D500%26epage%3D506%26doi%3D10.1093%2Fembo-reports%2Fkvd129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siersbæk, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandrup, S.</span></span> <span> </span><span class="NLM_article-title">Transcriptional networks and chromatin remodeling controlling adipogenesis</span>. <i>Trends Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">56</span>– <span class="NLM_lpage">64</span>, <span class="refDoi"> DOI: 10.1016/j.tem.2011.10.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2Fj.tem.2011.10.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=22079269" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2012&pages=56-64&author=R.+Siersb%C3%A6kauthor=R.+Nielsenauthor=S.+Mandrup&title=Transcriptional+networks+and+chromatin+remodeling+controlling+adipogenesis&doi=10.1016%2Fj.tem.2011.10.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional networks and chromatin remodeling controlling adipogenesis</span></div><div class="casAuthors">Siersbaek, Rasmus; Nielsen, Ronni; Mandrup, Susanne</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">56-64</span>CODEN:
                <span class="NLM_cas:coden">TENME4</span>;
        ISSN:<span class="NLM_cas:issn">1043-2760</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Adipocyte differentiation is tightly controlled by a transcriptional cascade, which directs the extensive reprogramming of gene expression required to convert fibroblast-like precursor cells into mature lipid-laden adipocytes.  Recent global analyses of transcription factor binding and chromatin remodeling have revealed snapshots' of this cascade and the chromatin landscape at specific time-points of differentiation.  These studies demonstrate that multiple adipogenic transcription factors co-occupy hotspots characterized by an open chromatin structure and specific epigenetic modifications.  Such transcription factor hotspots are likely to represent key signaling nodes which integrate multiple adipogenic signals at specific chromatin sites, thereby facilitating coordinated action on gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwAmbNvL0wPbVg90H21EOLACvtfcHk0ljuW5-8H-qkmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCntLo%253D&md5=610d28862d9a593d7581eb9c64986fb1</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.tem.2011.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tem.2011.10.001%26sid%3Dliteratum%253Aachs%26aulast%3DSiersb%25C3%25A6k%26aufirst%3DR.%26aulast%3DNielsen%26aufirst%3DR.%26aulast%3DMandrup%26aufirst%3DS.%26atitle%3DTranscriptional%2520networks%2520and%2520chromatin%2520remodeling%2520controlling%2520adipogenesis%26jtitle%3DTrends%2520Endocrinol.%2520Metab.%26date%3D2012%26volume%3D23%26spage%3D56%26epage%3D64%26doi%3D10.1016%2Fj.tem.2011.10.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowenz-Leutz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leutz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nerlov, C.</span></span> <span> </span><span class="NLM_article-title">Cooperation between C/EBPalpha TBP/TFIIB and SWI/SNF recruiting domains is required for adipocyte differentiation</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">3208</span>– <span class="NLM_lpage">3216</span>, <span class="refDoi"> DOI: 10.1101/gad.209901</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1101%2Fgad.209901" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=11731483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD3MXptVKmsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2001&pages=3208-3216&author=T.+A.+Pedersenauthor=E.+Kowenz-Leutzauthor=A.+Leutzauthor=C.+Nerlov&title=Cooperation+between+C%2FEBPalpha+TBP%2FTFIIB+and+SWI%2FSNF+recruiting+domains+is+required+for+adipocyte+differentiation&doi=10.1101%2Fgad.209901"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Cooperation between C/EBPα TBP/TFIIB and SWI/SNF recruiting domains is required for adipocyte differentiation</span></div><div class="casAuthors">Pedersen, Thomas Askov; Kowenz-Leutz, Elisabeth; Leutz, Achim; Nerlov, Claus</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3208-3216</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Chromatin remodeling is an important step in promoter activation during cellular lineage commitment and differentiation.  We show that the ability of the C/EBPα transcription factor to direct adipocyte differentiation of uncommitted fibroblast precursors and to activate SWI/SNF-dependent myeloid-specific genes depends on a domain, C/EBPα transactivation element III (TE-III), that binds the SWI/SNF chromatin remodeling complex.  TE-III collaborates with C/EBPα TBP/TFIIB interaction motifs during induction of adipogenesis and adipocyte-specific gene expression.  These results indicate that C/EBPα acts as a lineage-instructive transcription factor through SWI/SNF-dependent modification of the chromatin structure of lineage-specific genes, followed by direct promoter activation via recruitment of the basal transcription-initiation complex, and provide a mechanism by which C/EBPα can mediate differentiation along multiple cellular lineages.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsI37_rDsNbrVg90H21EOLACvtfcHk0ljuW5-8H-qkmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXptVKmsLs%253D&md5=62fcbfc140893f5e34eb4c10acb38926</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1101%2Fgad.209901&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.209901%26sid%3Dliteratum%253Aachs%26aulast%3DPedersen%26aufirst%3DT.%2BA.%26aulast%3DKowenz-Leutz%26aufirst%3DE.%26aulast%3DLeutz%26aufirst%3DA.%26aulast%3DNerlov%26aufirst%3DC.%26atitle%3DCooperation%2520between%2520C%252FEBPalpha%2520TBP%252FTFIIB%2520and%2520SWI%252FSNF%2520recruiting%2520domains%2520is%2520required%2520for%2520adipocyte%2520differentiation%26jtitle%3DGenes%2520Dev.%26date%3D2001%26volume%3D15%26spage%3D3208%26epage%3D3216%26doi%3D10.1101%2Fgad.209901" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbalzano, A. N.</span></span> <span> </span><span class="NLM_article-title">Temporal recruitment of transcription factors and SWI/SNF chromatin-remodeling enzymes during adipogenic induction of the peroxisome proliferator-activated receptor gamma nuclear hormone receptor</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">4651</span>– <span class="NLM_lpage">4663</span>, <span class="refDoi"> DOI: 10.1128/mcb.24.11.4651-4663.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1128%2FMCB.24.11.4651-4663.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=15143161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVyhsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=4651-4663&author=N.+Salmaauthor=H.+Xiaoauthor=E.+Muellerauthor=A.+N.+Imbalzano&title=Temporal+recruitment+of+transcription+factors+and+SWI%2FSNF+chromatin-remodeling+enzymes+during+adipogenic+induction+of+the+peroxisome+proliferator-activated+receptor+gamma+nuclear+hormone+receptor&doi=10.1128%2Fmcb.24.11.4651-4663.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Temporal recruitment of transcription factors and SWI/SNF chromatin-remodeling enzymes during adipogenic induction of the peroxisome proliferator-activated receptor γ nuclear hormone receptor</span></div><div class="casAuthors">Salma, Nunciada; Xiao, Hengyi; Mueller, Elisabetta; Imbalzano, Anthony N.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4651-4663</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The peroxisome proliferator-activated receptor gamma (PPARγ) regulates adipogenesis, lipid metab., and glucose homeostasis, and roles have emerged for this receptor in the pathogenesis and treatment of diabetes, atherosclerosis, and cancer.  We report here that induction of the PPARγ activator and adipogenesis forced by overexpression of adipogenic regulatory proteins is blocked upon expression of dominant-neg. BRG1 or hBRM, the ATPase subunits of distinct SWI/SNF chromatin-remodeling enzymes.  We demonstrate that histone hyperacetylation and the binding of C/EBP activators, polymerase II (Pol II), and general transcription factors (GTFs) initially occurred at the inducible PPARγ2 promoter in the absence of SWI/SNF function.  However, the polymerase and GTFs were subsequently lost from the promoter in cells expressing dominant-neg. SWI/SNF, explaining the inhibition of PPARγ2 expression.  To corroborate these data, we analyzed interactions at the PPARγ2 promoter in differentiating preadipocytes.  Changes in promoter structure, histone hyperacetylation, and binding of C/EBP activators, Pol II, and most GTFs preceded the interaction of SWI/SNF enzymes with the PPARγ2 promoter.  However, transcription of the PPARγ2 gene occurred only upon subsequent assocn. of SWI/SNF and TFIIH with the promoter.  Thus, induction of the PPARγ nuclear hormone receptor during adipogenesis requires SWI/SNF enzymes to facilitate preinitiation complex function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Nj3AMxjesbVg90H21EOLACvtfcHk0ljuW5-8H-qkmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVyhsrk%253D&md5=0e1d196bbbf840bd00f2c86151f80f11</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1128%2FMCB.24.11.4651-4663.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.24.11.4651-4663.2004%26sid%3Dliteratum%253Aachs%26aulast%3DSalma%26aufirst%3DN.%26aulast%3DXiao%26aufirst%3DH.%26aulast%3DMueller%26aufirst%3DE.%26aulast%3DImbalzano%26aufirst%3DA.%2BN.%26atitle%3DTemporal%2520recruitment%2520of%2520transcription%2520factors%2520and%2520SWI%252FSNF%2520chromatin-remodeling%2520enzymes%2520during%2520adipogenic%2520induction%2520of%2520the%2520peroxisome%2520proliferator-activated%2520receptor%2520gamma%2520nuclear%2520hormone%2520receptor%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D4651%26epage%3D4663%26doi%3D10.1128%2Fmcb.24.11.4651-4663.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mikkelsen, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gimble, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, E. D.</span></span> <span> </span><span class="NLM_article-title">Comparative epigenomic analysis of murine and human adipogenesis</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>143</i></span>,  <span class="NLM_fpage">156</span>– <span class="NLM_lpage">169</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2010.09.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2Fj.cell.2010.09.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=20887899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Cmu7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=143&publication_year=2010&pages=156-169&author=T.+S.+Mikkelsenauthor=Z.+Xuauthor=X.+Zhangauthor=L.+Wangauthor=J.+M.+Gimbleauthor=E.+S.+Landerauthor=E.+D.+Rosen&title=Comparative+epigenomic+analysis+of+murine+and+human+adipogenesis&doi=10.1016%2Fj.cell.2010.09.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative Epigenomic Analysis of Murine and Human Adipogenesis</span></div><div class="casAuthors">Mikkelsen, Tarjei S.; Xu, Zhao; Zhang, Xiaolan; Wang, Li; Gimble, Jeffrey M.; Lander, Eric S.; Rosen, Evan D.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">143</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">156-169</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We report the generation and comparative anal. of genome-wide chromatin state maps, PPARγ and CTCF localization maps, and gene expression profiles from murine and human models of adipogenesis.  The data provide high-resoln. views of chromatin remodeling during cellular differentiation and allow identification of thousands of putative preadipocyte- and adipocyte-specific cis-regulatory elements based on dynamic chromatin signatures.  We find that the specific locations of most such elements differ between the two models, including at orthologous loci with similar expression patterns.  Based on sequence anal. and reporter assays, we show that these differences are detd., in part, by evolutionary turnover of transcription factor motifs in the genome sequences and that this turnover may be facilitated by the presence of multiple distal regulatory elements at adipogenesis-dependent loci.  We also utilize the close relationship between open chromatin marks and transcription factor motifs to identify and validate PLZF and SRF as regulators of adipogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaVAJqwXqgl7Vg90H21EOLACvtfcHk0liF1NmEb8h8xA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Cmu7zK&md5=cbd32d847744e8c44275871ae0530864</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2010.09.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2010.09.006%26sid%3Dliteratum%253Aachs%26aulast%3DMikkelsen%26aufirst%3DT.%2BS.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DGimble%26aufirst%3DJ.%2BM.%26aulast%3DLander%26aufirst%3DE.%2BS.%26aulast%3DRosen%26aufirst%3DE.%2BD.%26atitle%3DComparative%2520epigenomic%2520analysis%2520of%2520murine%2520and%2520human%2520adipogenesis%26jtitle%3DCell%26date%3D2010%26volume%3D143%26spage%3D156%26epage%3D169%26doi%3D10.1016%2Fj.cell.2010.09.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lefterova, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haakonsson, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazar, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandrup, S.</span></span> <span> </span><span class="NLM_article-title">PPARγ and the global map of adipogenesis and beyond</span>. <i>Trends Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">302</span>, <span class="refDoi"> DOI: 10.1016/j.tem.2014.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2Fj.tem.2014.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=24793638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntFygtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=293-302&author=M.+I.+Lefterovaauthor=A.+K.+Haakonssonauthor=M.+A.+Lazarauthor=S.+Mandrup&title=PPAR%CE%B3+and+the+global+map+of+adipogenesis+and+beyond&doi=10.1016%2Fj.tem.2014.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">PPARγ and the global map of adipogenesis and beyond</span></div><div class="casAuthors">Lefterova, Martina I.; Haakonsson, Anders K.; Lazar, Mitchell A.; Mandrup, Susanne</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">293-302</span>CODEN:
                <span class="NLM_cas:coden">TENME4</span>;
        ISSN:<span class="NLM_cas:issn">1043-2760</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Peroxisome proliferator-activated receptor γ (PPARγ) is a member of the nuclear receptor (NR) superfamily of ligand-dependent transcription factors (TFs) and function as a master regulator of adipocyte differentiation and metab.  We review recent breakthroughs in the understanding of PPARγ gene regulation and function in the chromatin context.  It is now clear that multiple TFs team up to induce PPARγ during adipogenesis, and that other TFs cooperate with PPARγ to ensure adipocyte-specific genomic binding and function.  We discuss how this differs in other PPARγ-expressing cells such as macrophages and how these genome-wide mechanisms are preserved across species despite modest conservation of specific binding sites.  These emerging considerations inform our understanding of PPARγ function as well as of adipocyte development and physiol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiop59iTWwo7Vg90H21EOLACvtfcHk0liF1NmEb8h8xA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntFygtbk%253D&md5=7dfcb1c26d4232969257146f5f7b3636</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.tem.2014.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tem.2014.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DLefterova%26aufirst%3DM.%2BI.%26aulast%3DHaakonsson%26aufirst%3DA.%2BK.%26aulast%3DLazar%26aufirst%3DM.%2BA.%26aulast%3DMandrup%26aufirst%3DS.%26atitle%3DPPAR%25CE%25B3%2520and%2520the%2520global%2520map%2520of%2520adipogenesis%2520and%2520beyond%26jtitle%3DTrends%2520Endocrinol.%2520Metab.%26date%3D2014%26volume%3D25%26spage%3D293%26epage%3D302%26doi%3D10.1016%2Fj.tem.2014.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lefterova, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steger, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schupp, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schug, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cristancho, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoeckert, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazar, M. A.</span></span> <span> </span><span class="NLM_article-title">PPARgamma and C/EBP factors orchestrate adipocyte biology via adjacent binding on a genome-wide scale</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">2941</span>– <span class="NLM_lpage">2952</span>, <span class="refDoi"> DOI: 10.1101/gad.1709008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1101%2Fgad.1709008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=18981473" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOqtbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=2941-2952&author=M.+I.+Lefterovaauthor=Y.+Zhangauthor=D.+J.+Stegerauthor=M.+Schuppauthor=J.+Schugauthor=A.+Cristanchoauthor=D.+Fengauthor=D.+Zhuoauthor=C.+J.+Stoeckertauthor=X.+S.+Liuauthor=M.+A.+Lazar&title=PPARgamma+and+C%2FEBP+factors+orchestrate+adipocyte+biology+via+adjacent+binding+on+a+genome-wide+scale&doi=10.1101%2Fgad.1709008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">PPARγ and C/EBP factors orchestrate adipocyte biology via adjacent binding on a genome-wide scale</span></div><div class="casAuthors">Lefterova, Martina I.; Zhang, Yong; Steger, David J.; Schupp, Michael; Schug, Jonathan; Cristancho, Ana; Feng, Dan; Zhuo, David; Stoeckert, Christian J., Jr.; Liu, X. Shirley; Lazar, Mitchell A.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2941-2952</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Peroxisome proliferator-activated receptor γ(PPARγ), a nuclear receptor and the target of anti-diabetic thiazolinedione drugs, is known as the master regulator of adipocyte biol.  Although it regulates hundreds of adipocyte genes, PPARγ binding to endogenous genes has rarely been demonstrated.  Here, utilizing chromatin immunopptn. (ChIP) coupled with whole genome tiling arrays, we identified 5299 genomic regions of PPARγ binding in mouse 3T3-L1 adipocytes.  The consensus PPARγ/RXRα "DR-1"-binding motif was found at most of the sites, and ChIP for RXRα showed colocalization at nearly all locations tested.  Bioinformatics anal. also revealed CCAAT/enhancer-binding protein (C/EBP)-binding motifs in the vicinity of most PPARγ-binding sites, and genome-wide anal. of C/EBPα binding demonstrated that it localized to 3350 of the locations bound by PPARγ.  Importantly, most genes induced in adipogenesis were bound by both PPARγ and C/EBPα, while very few were PPARγ-specific.  C/EBPβ also plays a role at many of these genes, such that both C/EBPα and β are required along with PPARγ for robust adipocyte-specific gene expression.  Thus, PPARγ and C/EBP factors cooperatively orchestrate adipocyte biol. by adjacent binding on an unanticipated scale.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWnMWvt4R6krVg90H21EOLACvtfcHk0lhO3tgMMdBl1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOqtbnF&md5=566a2a51ce6b20a9722063b0e3e2952f</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1101%2Fgad.1709008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1709008%26sid%3Dliteratum%253Aachs%26aulast%3DLefterova%26aufirst%3DM.%2BI.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DSteger%26aufirst%3DD.%2BJ.%26aulast%3DSchupp%26aufirst%3DM.%26aulast%3DSchug%26aufirst%3DJ.%26aulast%3DCristancho%26aufirst%3DA.%26aulast%3DFeng%26aufirst%3DD.%26aulast%3DZhuo%26aufirst%3DD.%26aulast%3DStoeckert%26aufirst%3DC.%2BJ.%26aulast%3DLiu%26aufirst%3DX.%2BS.%26aulast%3DLazar%26aufirst%3DM.%2BA.%26atitle%3DPPARgamma%2520and%2520C%252FEBP%2520factors%2520orchestrate%2520adipocyte%2520biology%2520via%2520adjacent%2520binding%2520on%2520a%2520genome-wide%2520scale%26jtitle%3DGenes%2520Dev.%26date%3D2008%26volume%3D22%26spage%3D2941%26epage%3D2952%26doi%3D10.1101%2Fgad.1709008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alver, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manchester, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haswell, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, C. W. M.</span></span> <span> </span><span class="NLM_article-title">The SWI/SNF chromatin remodelling complex is required for maintenance of lineage specific enhancers</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">14648</span>, <span class="refDoi"> DOI: 10.1038/ncomms14648</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fncomms14648" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=28262751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A280%3ADC%252BC1czjtFeitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=14648&author=B.+H.+Alverauthor=K.+H.+Kimauthor=P.+Luauthor=X.+Wangauthor=H.+E.+Manchesterauthor=W.+Wangauthor=J.+R.+Haswellauthor=P.+J.+Parkauthor=C.+W.+M.+Roberts&title=The+SWI%2FSNF+chromatin+remodelling+complex+is+required+for+maintenance+of+lineage+specific+enhancers&doi=10.1038%2Fncomms14648"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">The SWI/SNF chromatin remodelling complex is required for maintenance of lineage specific enhancers</span></div><div class="casAuthors">Alver Burak H; Park Peter J; Kim Kimberly H; Lu Ping; Wang Xiaofeng; Manchester Haley E; Wang Weishan; Haswell Jeffrey R; Roberts Charles W M; Kim Kimberly H; Lu Ping; Wang Xiaofeng; Manchester Haley E; Wang Weishan; Haswell Jeffrey R; Roberts Charles W M; Kim Kimberly H; Lu Ping; Wang Xiaofeng; Manchester Haley E; Wang Weishan; Haswell Jeffrey R; Roberts Charles W M; Roberts Charles W M</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">14648</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Genes encoding subunits of SWI/SNF (BAF) chromatin remodelling complexes are collectively altered in over 20% of human malignancies, but the mechanisms by which these complexes alter chromatin to modulate transcription and cell fate are poorly understood.  Utilizing mouse embryonic fibroblast and cancer cell line models, here we show via ChIP-seq and biochemical assays that SWI/SNF complexes are preferentially targeted to distal lineage specific enhancers and interact with p300 to modulate histone H3 lysine 27 acetylation.  We identify a greater requirement for SWI/SNF at typical enhancers than at most super-enhancers and at enhancers in untranscribed regions than in transcribed regions.  Our data further demonstrate that SWI/SNF-dependent distal enhancers are essential for controlling expression of genes linked to developmental processes.  Our findings thus establish SWI/SNF complexes as regulators of the enhancer landscape and provide insight into the roles of SWI/SNF in cellular fate control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTe-gGPhesmNRdUSIfHIn4FfW6udTcc2eZo7Rm3TyBDWbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1czjtFeitA%253D%253D&md5=43052bae8dacb83273af786fbf5c6a1f</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2Fncomms14648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms14648%26sid%3Dliteratum%253Aachs%26aulast%3DAlver%26aufirst%3DB.%2BH.%26aulast%3DKim%26aufirst%3DK.%2BH.%26aulast%3DLu%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DManchester%26aufirst%3DH.%2BE.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DHaswell%26aufirst%3DJ.%2BR.%26aulast%3DPark%26aufirst%3DP.%2BJ.%26aulast%3DRoberts%26aufirst%3DC.%2BW.%2BM.%26atitle%3DThe%2520SWI%252FSNF%2520chromatin%2520remodelling%2520complex%2520is%2520required%2520for%2520maintenance%2520of%2520lineage%2520specific%2520enhancers%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D14648%26doi%3D10.1038%2Fncomms14648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buser-Doepner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederiksen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimshaw, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hepworth, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mader, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller-Fahrnow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Hagan, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundström, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toledo-Sherman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walpole, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuercher, W. J.</span></span> <span> </span><span class="NLM_article-title">The promise and peril of chemical probes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">536</span>– <span class="NLM_lpage">541</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fnchembio.1867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=26196764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=536-541&author=C.+H.+Arrowsmithauthor=J.+E.+Audiaauthor=C.+Austinauthor=J.+Baellauthor=J.+Bennettauthor=J.+Blaggauthor=C.+Bountraauthor=P.+E.+Brennanauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=C.+Buser-Doepnerauthor=R.+M.+Campbellauthor=A.+J.+Carterauthor=P.+Cohenauthor=R.+A.+Copelandauthor=B.+Cravattauthor=J.+L.+Dahlinauthor=D.+Dhanakauthor=A.+M.+Edwardsauthor=M.+Frederiksenauthor=S.+V.+Fryeauthor=N.+Grayauthor=C.+E.+Grimshawauthor=D.+Hepworthauthor=T.+Howeauthor=K.+V.+M.+Huberauthor=J.+Jinauthor=S.+Knappauthor=J.+D.+Kotzauthor=R.+G.+Krugerauthor=D.+Loweauthor=M.+M.+Maderauthor=B.+Marsdenauthor=A.+Mueller-Fahrnowauthor=S.+M%C3%BCllerauthor=R.+C.+O%E2%80%99Haganauthor=J.+P.+Overingtonauthor=D.+R.+Owenauthor=S.+H.+Rosenbergauthor=R.+Rossauthor=B.+Rothauthor=M.+Schapiraauthor=S.+L.+Schreiberauthor=B.+Shoichetauthor=M.+Sundstr%C3%B6mauthor=G.+Superti-Furgaauthor=J.+Tauntonauthor=L.+Toledo-Shermanauthor=C.+Walpoleauthor=M.+A.+Waltersauthor=T.+M.+Willsonauthor=P.+Workmanauthor=R.+N.+Youngauthor=W.+J.+Zuercher&title=The+promise+and+peril+of+chemical+probes&doi=10.1038%2Fnchembio.1867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">The promise and peril of chemical probes</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Audia, James E.; Austin, Christopher; Baell, Jonathan; Bennett, Jonathan; Blagg, Julian; Bountra, Chas; Brennan, Paul E.; Brown, Peter J.; Bunnage, Mark E.; Buser-Doepner, Carolyn; Campbell, Robert M.; Carter, Adrian J.; Cohen, Philip; Copeland, Robert A.; Cravatt, Ben; Dahlin, Jayme L.; Dhanak, Dashyant; Edwards, Aled M.; Frye, Stephen V.; Gray, Nathanael; Grimshaw, Charles E.; Hepworth, David; Howe, Trevor; Huber, Kilian V. M.; Jin, Jian; Knapp, Stefan; Kotz, Joanne D.; Kruger, Ryan G.; Lowe, Derek; Mader, Mary M.; Marsden, Brian; Mueller-Fahrnow, Anke; Muller, Susanne; O'Hagan, Ronan C.; Overington, John P.; Owen, Dafydd R.; Rosenberg, Saul H.; Roth, Brian; Ross, Ruth; Schapira, Matthieu; Schreiber, Stuart L.; Shoichet, Brian; Sundstrom, Michael; Superti-Furga, Giulio; Taunton, Jack; Toledo-Sherman, Leticia; Walpole, Chris; Walters, Michael A.; Willson, Timothy M.; Workman, Paul; Young, Robert N.; Zuercher, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">536-541</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. probes are powerful reagents with increasing impacts on biomedical research.  However, probes of poor quality or that are used incorrectly generate misleading results.  To help address these shortcomings, we will create a community-driven wiki resource to improve quality and convey current best practice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-li4dxq487rVg90H21EOLACvtfcHk0lhO3tgMMdBl1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO&md5=d8c7a7af94921ab4cdcc871487cceadd</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1867%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DAustin%26aufirst%3DC.%26aulast%3DBaell%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DBuser-Doepner%26aufirst%3DC.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DCarter%26aufirst%3DA.%2BJ.%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DCravatt%26aufirst%3DB.%26aulast%3DDahlin%26aufirst%3DJ.%2BL.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26aulast%3DFrederiksen%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DGrimshaw%26aufirst%3DC.%2BE.%26aulast%3DHepworth%26aufirst%3DD.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKotz%26aufirst%3DJ.%2BD.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DLowe%26aufirst%3DD.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DMueller-Fahrnow%26aufirst%3DA.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Hagan%26aufirst%3DR.%2BC.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DRoss%26aufirst%3DR.%26aulast%3DRoth%26aufirst%3DB.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%26aulast%3DSundstr%25C3%25B6m%26aufirst%3DM.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DToledo-Sherman%26aufirst%3DL.%26aulast%3DWalpole%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DR.%2BN.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26atitle%3DThe%2520promise%2520and%2520peril%2520of%2520chemical%2520probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D536%26epage%3D541%26doi%3D10.1038%2Fnchembio.1867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chekler, E. L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span> <span> </span><span class="NLM_article-title">Target validation using chemical probes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">195</span>– <span class="NLM_lpage">199</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fnchembio.1197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=23508172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFagsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=195-199&author=M.+E.+Bunnageauthor=E.+L.+P.+Cheklerauthor=L.+H.+Jones&title=Target+validation+using+chemical+probes&doi=10.1038%2Fnchembio.1197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Target validation using chemical probes</span></div><div class="casAuthors">Bunnage, Mark E.; Chekler, Eugene L. Piatnitski; Jones, Lyn H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">195-199</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review on the fully profiled chem. probes essential to support the unbiased interpretation of biol. expts. necessary for rigorous preclin. target validation.  We believe that by developing a 'chem. probe tool kit', and a framework for its use, chem. biol. can have a more central role in identifying targets of potential relevance to disease, avoiding many of the biases that complicate target validation as practiced currently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLwRTT4pXGxbVg90H21EOLACvtfcHk0lhrThs9HeU-sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFagsr4%253D&md5=af0c362882191de96a8e8698ff2c9ce8</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1197%26sid%3Dliteratum%253Aachs%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DChekler%26aufirst%3DE.%2BL.%2BP.%26aulast%3DJones%26aufirst%3DL.%2BH.%26atitle%3DTarget%2520validation%2520using%2520chemical%2520probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D195%26epage%3D199%26doi%3D10.1038%2Fnchembio.1197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aldeghi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodkin, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Essex, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggin, P. C.</span></span> <span> </span><span class="NLM_article-title">Large-scale analysis of water stability in bromodomain binding pockets with grand canonical Monte Carlo</span>. <i>Commun. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">19</span>, <span class="refDoi"> DOI: 10.1038/s42004-018-0019-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fs42004-018-0019-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=29863194" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2018&pages=19&author=M.+Aldeghiauthor=G.+A.+Rossauthor=M.+J.+Bodkinauthor=J.+W.+Essexauthor=S.+Knappauthor=P.+C.+Biggin&title=Large-scale+analysis+of+water+stability+in+bromodomain+binding+pockets+with+grand+canonical+Monte+Carlo&doi=10.1038%2Fs42004-018-0019-x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fs42004-018-0019-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42004-018-0019-x%26sid%3Dliteratum%253Aachs%26aulast%3DAldeghi%26aufirst%3DM.%26aulast%3DRoss%26aufirst%3DG.%2BA.%26aulast%3DBodkin%26aufirst%3DM.%2BJ.%26aulast%3DEssex%26aufirst%3DJ.%2BW.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBiggin%26aufirst%3DP.%2BC.%26atitle%3DLarge-scale%2520analysis%2520of%2520water%2520stability%2520in%2520bromodomain%2520binding%2520pockets%2520with%2520grand%2520canonical%2520Monte%2520Carlo%26jtitle%3DCommun.%2520Chem.%26date%3D2018%26volume%3D1%26spage%3D19%26doi%3D10.1038%2Fs42004-018-0019-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niesen, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Kinase inhibitor selectivity profiling using differential scanning fluorimetry</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>795</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">118</span>, <span class="refDoi"> DOI: 10.1007/978-1-61779-337-0_7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1007%2F978-1-61779-337-0_7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=21960218" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlersLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=795&publication_year=2012&pages=109-118&author=O.+Fedorovauthor=F.+H.+Niesenauthor=S.+Knapp&title=Kinase+inhibitor+selectivity+profiling+using+differential+scanning+fluorimetry&doi=10.1007%2F978-1-61779-337-0_7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitor selectivity profiling using differential scanning fluorimetry</span></div><div class="casAuthors">Fedorov, Oleg; Niesen, Frank H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">795</span>
        (<span class="NLM_cas:issue">Kinase Inhibitors</span>),
    <span class="NLM_cas:pages">109-118</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Fast, robust, and inexpensive screening methods are the heart of drug discovery processes.  Moreover, it is useful to have access to several established assay formats, for validation purposes.  If a targeted protein is an enzyme, the logical and widely used approach is the direct measurement of the effect of the added ligands on its activity.  A variety of enzymic assay formats have been successfully applied for inhibitor screening of protein kinases.  However, enzymic assays require an active enzyme with a known substrate and often time-consuming assay optimization.  Several alternative approaches have been recently developed that detect binding of ligands to proteins.  This chapter overviews and provides the exptl. protocol of the successful application of differential scanning fluorimetry (DSF) in our lab. for fast and robust screening of medium-sized (<10,000) inhibitor libraries.  DSF monitors the thermal stabilization of the native protein structure upon ligand binding.  It allows selectivity profiling of any protein kinase without prior knowledge of either substrate or activity of the kinase under investigation.  Comparative studies revealed that generated data is highly reproducible and correlates well with the results from other ligand binding methodologies, direct binding consts. as well as enzymic assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJ3JcrKLX6AbVg90H21EOLACvtfcHk0lhrThs9HeU-sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlersLvM&md5=ce44a4a0c4e28e61b865da2301df6db7</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-337-0_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-337-0_7%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DKinase%2520inhibitor%2520selectivity%2520profiling%2520using%2520differential%2520scanning%2520fluorimetry%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2012%26volume%3D795%26spage%3D109%26epage%3D118%26doi%3D10.1007%2F978-1-61779-337-0_7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Mahony, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasater, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">312</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fnchembio.1471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=24584101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=305-312&author=P.+Ciceriauthor=S.+M%C3%BCllerauthor=A.+O%E2%80%99Mahonyauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=J.+P.+Huntauthor=E.+A.+Lasaterauthor=G.+Pallaresauthor=S.+Picaudauthor=C.+Wellsauthor=S.+Martinauthor=L.+M.+Wodickaauthor=N.+P.+Shahauthor=D.+K.+Treiberauthor=S.+Knapp&title=Dual+kinase-bromodomain+inhibitors+for+rationally+designed+polypharmacology&doi=10.1038%2Fnchembio.1471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span></div><div class="casAuthors">Ciceri, Pietro; Mueller, Susanne; O'Mahony, Alison; Fedorov, Oleg; Filippakopoulos, Panagis; Hunt, Jeremy P.; Lasater, Elisabeth A.; Pallares, Gabriel; Picaud, Sarah; Wells, Christopher; Martin, Sarah; Wodicka, Lisa M.; Shah, Neil P.; Treiber, Daniel K.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">305-312</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Concomitant inhibition of multiple cancer-driving kinases is an established strategy to improve the durability of clin. responses to targeted therapies.  The difficulty of discovering kinase inhibitors with an appropriate multitarget profile has, however, necessitated the application of combination therapies, which can pose major clin. development challenges.  Epigenetic reader domains of the bromodomain family have recently emerged as new targets for cancer therapy.  Here the authors report that several clin. kinase inhibitors also inhibit bromodomains with therapeutically relevant potencies and are best classified as dual kinase-bromodomain inhibitors.  Nanomolar activity on BRD4 by BI-2536 and TG-101348, which are clin. PLK1 and JAK2-FLT3 kinase inhibitors, resp., is particularly noteworthy as these combinations of activities on independent oncogenic pathways exemplify a new strategy for rational single-agent polypharmacol. targeting.  Furthermore, structure-activity relationships and co-crystal structures identify design features that enable a general platform for the rational design of dual kinase-bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotPedDED4FJLVg90H21EOLACvtfcHk0ljWaHXq7mZzqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D&md5=a00117db2648ea38dd03750faf3b3cb9</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1471%26sid%3Dliteratum%253Aachs%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLasater%26aufirst%3DE.%2BA.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DShah%26aufirst%3DN.%2BP.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DDual%2520kinase-bromodomain%2520inhibitors%2520for%2520rationally%2520designed%2520polypharmacology%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D305%26epage%3D312%26doi%3D10.1038%2Fnchembio.1471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gallenkamp, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelato, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haendler, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinmann, H.</span></span> <span> </span><span class="NLM_article-title">Bromodomains and their pharmacological inhibitors</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">438</span>– <span class="NLM_lpage">464</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201300434</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1002%2Fcmdc.201300434" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=24497428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsl2gtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=438-464&author=D.+Gallenkampauthor=K.+A.+Gelatoauthor=B.+Haendlerauthor=H.+Weinmann&title=Bromodomains+and+their+pharmacological+inhibitors&doi=10.1002%2Fcmdc.201300434"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomains and Their Pharmacological Inhibitors</span></div><div class="casAuthors">Gallenkamp, Daniel; Gelato, Kathy A.; Haendler, Bernard; Weinmann, Hilmar</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">438-464</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Over 60 bromodomains belonging to proteins with very different functions have been identified in humans.  Several of them interact with acetylated lysine residues, leading to the recruitment and stabilization of protein complexes.  The bromodomain and extra-terminal domain (BET) proteins contain tandem bromodomains which bind to acetylated histones and are thereby implicated in a no. of DNA-centered processes, including the regulation of gene expression.  The recent identification of inhibitors of BET and non-BET bromodomains is one of the few examples in which effective blockade of a protein-protein interaction can be achieved with a small mol.  This has led to major strides in the understanding of the function of bromodomain-contg. proteins and their involvement in diseases such as cancer and inflammation.  Indeed, BET bromodomain inhibitors are now being clin. evaluated for the treatment of hematol. tumors and have also been tested in clin. trials for the relatively rare BRD-NUT midline carcinoma.  This review gives an overview of the newest developments in the field, with a focus on the biol. of selected bromodomain proteins on the one hand, and on reported pharmacol. inhibitors on the other, including recent examples from the patent literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon-8_ymytCPbVg90H21EOLACvtfcHk0ljWaHXq7mZzqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsl2gtL4%253D&md5=2ce7c6b5c37001fb8fb901fe04aeb606</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300434%26sid%3Dliteratum%253Aachs%26aulast%3DGallenkamp%26aufirst%3DD.%26aulast%3DGelato%26aufirst%3DK.%2BA.%26aulast%3DHaendler%26aufirst%3DB.%26aulast%3DWeinmann%26aufirst%3DH.%26atitle%3DBromodomains%2520and%2520their%2520pharmacological%2520inhibitors%26jtitle%3DChemMedChem%26date%3D2014%26volume%3D9%26spage%3D438%26epage%3D464%26doi%3D10.1002%2Fcmdc.201300434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span> <span> </span><span class="NLM_article-title">Protein dynamics and structural waters in bromodomains</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0186570</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0186570</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1371%2Fjournal.pone.0186570" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=29077715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFGms7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=X.+Zhangauthor=K.+Chenauthor=Y.-D.+Wuauthor=O.+Wiest&title=Protein+dynamics+and+structural+waters+in+bromodomains&doi=10.1371%2Fjournal.pone.0186570"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Protein dynamics and structural waters in bromodomains</span></div><div class="casAuthors">Zhang, Xiaoxiao; Chen, Kai; Wu, Yun-Dong; Wiest, Olaf</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e0186570/1-e0186570/16</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Bromodomains are epigenetic readers of acetylated lysines that are integral parts of histone tails.  The 61 bromodomains in humans are structurally highly conserved but specifically bind to widely varying recognition motifs, suggesting that dynamic rather than static factors are responsible for recognition selectivity.  To test this hypothesis, the dynamics of the binding sites and structural water mols. of four bromodomains (ATAD2, BAZ2B, BRD2(1) and CREBBP) representing four different subtypes is studied with 1 μs MD simulations using the RSFF2 force field.  The different dynamics of the ZA-loops and BC-loops between the four bromodomains leads to distinct patterns for the opening and closing of the binding pocket.  This in turn dets. the structural and energetic properties of the structural waters in the binding pocket, suggesting that these waters are not only important for the recognition itself, as has been proposed previously, but also contribute to the selectivity of different bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdqzaoNSjS-7Vg90H21EOLACvtfcHk0lh7afrcm71h4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFGms7bN&md5=3ed02a6025d67e5c0e8b511c3dc6756e</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0186570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0186570%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DWu%26aufirst%3DY.-D.%26aulast%3DWiest%26aufirst%3DO.%26atitle%3DProtein%2520dynamics%2520and%2520structural%2520waters%2520in%2520bromodomains%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0186570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominy, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feeney, P. J.</span></span> <span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1016/s0169-409x(00)00129-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2Fs0169-409x%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2Fs0169-409x%2800%2900129-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0lgGkF0y7gcpsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fs0169-409x%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0169-409x%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D3%26epage%3D26%26doi%3D10.1016%2Fs0169-409x%2800%2900129-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span> <span> </span><span class="NLM_article-title">The chromatin-remodeling enzyme BRG1 promotes colon cancer progression via positive regulation of WNT3A</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">86051</span>– <span class="NLM_lpage">86063</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.13326</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.18632%2Foncotarget.13326" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=27852072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A280%3ADC%252BC2snmvV2ltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=86051-86063&author=S.+Linauthor=T.+Jiangauthor=L.+Yeauthor=Z.+Hanauthor=Y.+Liuauthor=C.+Liuauthor=C.+Yuanauthor=S.+Zhaoauthor=J.+Chenauthor=J.+Wangauthor=H.+Tangauthor=S.+Luauthor=L.+Yangauthor=X.+Wangauthor=D.+Yanauthor=Z.+Pengauthor=J.+Fan&title=The+chromatin-remodeling+enzyme+BRG1+promotes+colon+cancer+progression+via+positive+regulation+of+WNT3A&doi=10.18632%2Foncotarget.13326"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">The chromatin-remodeling enzyme BRG1 promotes colon cancer progression via positive regulation of WNT3A</span></div><div class="casAuthors">Lin Shengtao; Ye Ling; Liu Yuan; Liu Chenchen; Yuan Chenwei; Zhao Senlin; Chen Jian; Wang Jingtao; Wang Xiaoliang; Yan Dongwang; Peng Zhihai; Fan Junwei; Jiang Tao; Han Zhongbo; Tang Huamei; Lu Su; Yang Liguang</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">86051-86063</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In this study, we aimed to elucidate the clinical significance and underlying mechanisms of BRG1 in colon cancer.  In the clinical analysis, overexpression of BRG1 correlates with colon cancer progression in two cohorts (n = 191 and n = 75).  Kaplan-Meier survival analysis revealed that BRG1 is a prognosis predictor for overall survival (P < 0.001) and disease-free survival (P = 0.001).  Knocking down BRG1 expression significantly suppressed the proliferation and invasion in colon cancer cells.  The expression pattern of WNT3A is consistent with BRG1 in colon cancer tissues and WNT3A expression was inhibited in BRG1 knockdown cells.  In addition, restoring WNT3A expression rescues the inhibition of cell proliferation and invasion induced by BRG1.  In this study, we demonstrate that BRG1 may contribute to colon cancer progression through upregulating WNT3A expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSuPnIftE7c1GZyxfu64m4ffW6udTcc2ebPSgXn6mkZ2rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2snmvV2ltQ%253D%253D&md5=701bd407c4a814b307e53704bf38076d</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.13326&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.13326%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DS.%26aulast%3DJiang%26aufirst%3DT.%26aulast%3DYe%26aufirst%3DL.%26aulast%3DHan%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DYuan%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DYan%26aufirst%3DD.%26aulast%3DPeng%26aufirst%3DZ.%26aulast%3DFan%26aufirst%3DJ.%26atitle%3DThe%2520chromatin-remodeling%2520enzyme%2520BRG1%2520promotes%2520colon%2520cancer%2520progression%2520via%2520positive%2520regulation%2520of%2520WNT3A%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D86051%26epage%3D86063%26doi%3D10.18632%2Foncotarget.13326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madany, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobson, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnabl, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Wijnen, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lian, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muthuswami, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbalzano, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nickerson, J. A.</span></span> <span> </span><span class="NLM_article-title">The BRG1 chromatin remodeling enzyme links cancer cell metabolism and proliferation</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">38270</span>– <span class="NLM_lpage">38281</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.9505</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.18632%2Foncotarget.9505" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=27223259" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A280%3ADC%252BC2s%252FjtFGkug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=38270-38281&author=Q.+Wuauthor=P.+Madanyauthor=J.+R.+Dobsonauthor=J.+M.+Schnablauthor=S.+Sharmaauthor=T.+C.+Smithauthor=A.+J.+van+Wijnenauthor=J.+L.+Steinauthor=J.+B.+Lianauthor=G.+S.+Steinauthor=R.+Muthuswamiauthor=A.+N.+Imbalzanoauthor=J.+A.+Nickerson&title=The+BRG1+chromatin+remodeling+enzyme+links+cancer+cell+metabolism+and+proliferation&doi=10.18632%2Foncotarget.9505"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">The BRG1 chromatin remodeling enzyme links cancer cell metabolism and proliferation</span></div><div class="casAuthors">Wu Qiong; Madany Pasil; Dobson Jason R; Schnabl Jake M; Smith Tara C; Imbalzano Anthony N; Nickerson Jeffrey A; Sharma Soni; Muthuswami Rohini; van Wijnen Andre J; Stein Janet L; Lian Jane B; Stein Gary S</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">38270-38281</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cancer cells reprogram cellular metabolism to meet the demands of growth.  Identification of the regulatory machinery that regulates cancer-specific metabolic changes may open new avenues for anti-cancer therapeutics.  The epigenetic regulator BRG1 is a catalytic ATPase for some mammalian SWI/SNF chromatin remodeling enzymes.  BRG1 is a well-characterized tumor suppressor in some human cancers, but is frequently overexpressed without mutation in other cancers, including breast cancer.  Here we demonstrate that BRG1 upregulates de novo lipogenesis and that this is crucial for cancer cell proliferation.  Knockdown of BRG1 attenuates lipid synthesis by impairing the transcription of enzymes catalyzing fatty acid and lipid synthesis.  Remarkably, exogenous addition of palmitate, the key intermediate in fatty acid synthesis, rescued the cancer cell proliferation defect caused by BRG1 knockdown.  Our work suggests that targeting BRG1 to reduce lipid metabolism and, thereby, to reduce proliferation, has promise for epigenetic therapy in triple negative breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSvsaxPn9YAplVwdslNBF-tfW6udTcc2eY3XLuoPSudx7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s%252FjtFGkug%253D%253D&md5=a5847bf6a0604889bd5fad01cd7b6738</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.9505&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.9505%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DMadany%26aufirst%3DP.%26aulast%3DDobson%26aufirst%3DJ.%2BR.%26aulast%3DSchnabl%26aufirst%3DJ.%2BM.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DSmith%26aufirst%3DT.%2BC.%26aulast%3Dvan%2BWijnen%26aufirst%3DA.%2BJ.%26aulast%3DStein%26aufirst%3DJ.%2BL.%26aulast%3DLian%26aufirst%3DJ.%2BB.%26aulast%3DStein%26aufirst%3DG.%2BS.%26aulast%3DMuthuswami%26aufirst%3DR.%26aulast%3DImbalzano%26aufirst%3DA.%2BN.%26aulast%3DNickerson%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520BRG1%2520chromatin%2520remodeling%2520enzyme%2520links%2520cancer%2520cell%2520metabolism%2520and%2520proliferation%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D38270%26epage%3D38281%26doi%3D10.18632%2Foncotarget.9505" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mei, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span> <span> </span><span class="NLM_article-title">BRG1 is a prognostic marker and potential therapeutic target in human breast cancer</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e59772</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0059772</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1371%2Fjournal.pone.0059772" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=23533649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsVOntLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=J.+Baiauthor=P.+Meiauthor=C.+Zhangauthor=F.+Chenauthor=C.+Liauthor=Z.+Panauthor=H.+Liuauthor=J.+Zheng&title=BRG1+is+a+prognostic+marker+and+potential+therapeutic+target+in+human+breast+cancer&doi=10.1371%2Fjournal.pone.0059772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">BRG1 is a prognostic marker and potential therapeutic target in human breast cancer</span></div><div class="casAuthors">Bai, Jin; Mei, Pengjin; Zhang, Cuipeng; Chen, Feifei; Li, Chen; Pan, Zhenqiang; Liu, Hui; Zheng, Junnian</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e59772</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">BRG1, a core component of the SWI/SNF chromatin-remodeling complex, has been implicated in cancer development; however, the biol. significance of BRG1 in breast cancer remains unknown.  We explored the role of BRG1 in human breast cancer pathogenesis.  Using tissue microarray and immunohistochem., we evaluated BRG1 staining in 437 breast cancer specimens and investigated its role in breast cancer cell proliferation, migration and invasion.  Our Kaplan-Meier survival curves showed that high BRG1 expression is inversely correlated with both overall (P = 0.000) and disease-specific (P = 0.000) 5-yr patient survival.  Furthermore, we found that knockdown of BRG1 by RNA interference markedly inhibits cell proliferation and causes cessation of cell cycle.  This reduced cell proliferation is due to G1 phase arrest as cyclin D1 and cyclin E are diminished whereas p27 is upregulated.  Moreover, BRG1 depletion induces the expression of TIMP-2 but reduces MMP-2, thereby inhibiting the ability of cells to migrate and to invade.  These results highlight the importance of BRG1 in breast cancer pathogenesis and BRG1 may serve as a prognostic marker as well as a potentially selective therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqreHzAF2rxbrVg90H21EOLACvtfcHk0ljJfKY5mc00og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsVOntLc%253D&md5=15194af1bd7257d61c2e55b2b0cb3392</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0059772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0059772%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DJ.%26aulast%3DMei%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DPan%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DJ.%26atitle%3DBRG1%2520is%2520a%2520prognostic%2520marker%2520and%2520potential%2520therapeutic%2520target%2520in%2520human%2520breast%2520cancer%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0059772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gresh, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourachot, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reimann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guigas, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiette, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbay, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muchardt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hue, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pontoglio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yaniv, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klochendler-Yeivin, A.</span></span> <span> </span><span class="NLM_article-title">The SWI/SNF chromatin-remodeling complex subunit SNF5 is essential for hepatocyte differentiation</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">3313</span>– <span class="NLM_lpage">3324</span>, <span class="refDoi"> DOI: 10.1038/sj.emboj.7600802</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fsj.emboj.7600802" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=16138077" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVajsrnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2005&pages=3313-3324&author=L.+Greshauthor=B.+Bourachotauthor=A.+Reimannauthor=B.+Guigasauthor=L.+Fietteauthor=S.+Garbayauthor=C.+Muchardtauthor=L.+Hueauthor=M.+Pontoglioauthor=M.+Yanivauthor=A.+Klochendler-Yeivin&title=The+SWI%2FSNF+chromatin-remodeling+complex+subunit+SNF5+is+essential+for+hepatocyte+differentiation&doi=10.1038%2Fsj.emboj.7600802"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">The SWI/SNF chromatin-remodeling complex subunit SNF5 is essential for hepatocyte differentiation</span></div><div class="casAuthors">Gresh, Lionel; Bourachot, Brigitte; Reimann, Andreas; Guigas, Bruno; Fiette, Laurence; Garbay, Serge; Muchardt, Christian; Hue, Louis; Pontoglio, Marco; Yaniv, Moshe; Klochendler-Yeivin, Agnes</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3313-3324</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Regulation of gene expression underlies cell differentiation and organogenesis.  Both transcription factors and chromatin modifiers are crucial for this process.  To study the role of the ATP-dependent SWI/SNF chromatin-remodeling complex in cell differentiation, we inactivated the gene encoding the core complex subunit SNF5/INI1 in the developing liver.  Hepatic SNF5 deletion caused neonatal death due to severe hypoglycemia; mutant animals fail to store glycogen and have impaired energetic metab.  The formation of a hepatic epithelium is also affected in SNF5-deficient livers.  Transcriptome analyses showed that SNF5 inactivation is accompanied by defective transcriptional activation of 70% of the genes that are normally upregulated during liver development.  These include genes involved in glycogen synthesis, gluconeogenesis and cell-cell adhesion.  A fraction of hepatic developmentally activated genes were normally expressed, suggesting that cell differentiation was not completely blocked.  Moreover, SNF5-deleted cells showed increased proliferation, and we identified several misexpressed genes that may contribute to cell cycle deregulation in these cells.  Our results emphasize the role of chromatin remodeling in the activation of cell-type-specific genetic programs and in driving cell differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt7d7ilEBZyLVg90H21EOLACvtfcHk0lh1M87JXe_v7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVajsrnP&md5=a1d1407c27f128cc0dc6f14acf1e868c</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7600802&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7600802%26sid%3Dliteratum%253Aachs%26aulast%3DGresh%26aufirst%3DL.%26aulast%3DBourachot%26aufirst%3DB.%26aulast%3DReimann%26aufirst%3DA.%26aulast%3DGuigas%26aufirst%3DB.%26aulast%3DFiette%26aufirst%3DL.%26aulast%3DGarbay%26aufirst%3DS.%26aulast%3DMuchardt%26aufirst%3DC.%26aulast%3DHue%26aufirst%3DL.%26aulast%3DPontoglio%26aufirst%3DM.%26aulast%3DYaniv%26aufirst%3DM.%26aulast%3DKlochendler-Yeivin%26aufirst%3DA.%26atitle%3DThe%2520SWI%252FSNF%2520chromatin-remodeling%2520complex%2520subunit%2520SNF5%2520is%2520essential%2520for%2520hepatocyte%2520differentiation%26jtitle%3DEMBO%2520J.%26date%3D2005%26volume%3D24%26spage%3D3313%26epage%3D3324%26doi%3D10.1038%2Fsj.emboj.7600802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, T.</span></span> <span> </span><span class="NLM_article-title">The chromatin-remodeling enzyme BRG1 plays an essential role in primitive erythropoiesis and vascular development</span>. <i>Development</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">493</span>– <span class="NLM_lpage">500</span>, <span class="refDoi"> DOI: 10.1242/dev.010090</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1242%2Fdev.010090" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=18094026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjt1CntrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2008&pages=493-500&author=C.+T.+Griffinauthor=J.+Brennanauthor=T.+Magnuson&title=The+chromatin-remodeling+enzyme+BRG1+plays+an+essential+role+in+primitive+erythropoiesis+and+vascular+development&doi=10.1242%2Fdev.010090"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">The chromatin-remodeling enzyme BRG1 plays an essential role in primitive erythropoiesis and vascular development</span></div><div class="casAuthors">Griffin, Courtney T.; Brennan, Jennifer; Magnuson, Terry</div><div class="citationInfo"><span class="NLM_cas:title">Development (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">493-500</span>CODEN:
                <span class="NLM_cas:coden">DEVPED</span>;
        ISSN:<span class="NLM_cas:issn">0950-1991</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">ATP-dependent chromatin-remodeling complexes contribute to the proper temporal and spatial patterns of gene expression in mammalian embryos and therefore play important roles in a no. of developmental processes.  SWI/SNF-like chromatin-remodeling complexes use one of two different ATPases as their catalytic subunit: brahma (BRM, also known as SMARCA2) and brahma-related gene 1 (BRG1, also known as SMARCA4).  We have conditionally deleted a floxed Brg1 allele with a Tie2-Cre transgene, which is expressed in developing hematopoietic and endothelial cells.  Brg1fl/fl:Tie2-Cre+ embryos die at midgestation from anemia, as mutant primitive erythrocytes fail to transcribe embryonic α- and β-globins, and subsequently undergo apoptosis.  Addnl., vascular remodeling of the extraembryonic yolk sac is abnormal in Brg1fl/fl:Tie2-Cre+ embryos.  Importantly, Brm deficiency does not exacerbate the erythropoietic or vascular abnormalities found in Brg1fl/fl:Tie2-Cre+ embryos, implying that Brg1-contg. SWI/SNF-like complexes, rather than Brm-contg. complexes, play a crucial role in primitive erythropoiesis and in early vascular development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcF-UQpNXLHrVg90H21EOLACvtfcHk0lh1M87JXe_v7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjt1CntrY%253D&md5=ad1b61ee0637c08b5b0cc8436838dd22</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1242%2Fdev.010090&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fdev.010090%26sid%3Dliteratum%253Aachs%26aulast%3DGriffin%26aufirst%3DC.%2BT.%26aulast%3DBrennan%26aufirst%3DJ.%26aulast%3DMagnuson%26aufirst%3DT.%26atitle%3DThe%2520chromatin-remodeling%2520enzyme%2520BRG1%2520plays%2520an%2520essential%2520role%2520in%2520primitive%2520erythropoiesis%2520and%2520vascular%2520development%26jtitle%3DDevelopment%26date%3D2008%26volume%3D135%26spage%3D493%26epage%3D500%26doi%3D10.1242%2Fdev.010090" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cornelius, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDougald, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, M. D.</span></span> <span> </span><span class="NLM_article-title">Regulation of adipocyte development</span>. <i>Annu. Rev. Nutr.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">99</span>– <span class="NLM_lpage">129</span>, <span class="refDoi"> DOI: 10.1146/annurev.nu.14.070194.000531</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1146%2Fannurev.nu.14.070194.000531" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=7946535" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADyaK2cXmsFWrsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1994&pages=99-129&author=P.+Corneliusauthor=O.+A.+MacDougaldauthor=M.+D.+Lane&title=Regulation+of+adipocyte+development&doi=10.1146%2Fannurev.nu.14.070194.000531"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of adipocyte development</span></div><div class="casAuthors">Cornelius, Peter; MacDougald, Ormond A.; Lane, M. Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Nutrition</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">99-129</span>CODEN:
                <span class="NLM_cas:coden">ARNTD8</span>;
        ISSN:<span class="NLM_cas:issn">0199-9885</span>.
    </div><div class="casAbstract">A review, with 216 refs., discussing cell culture models for adipocyte development, external modulators of preadipocyte differentiation, a developmental program, and transcriptional control of adipocyte genes during differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpadKhkXFwuArVg90H21EOLACvtfcHk0lh1M87JXe_v7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmsFWrsL8%253D&md5=e2c5c546ba15ef193ab4323b73b8ed06</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1146%2Fannurev.nu.14.070194.000531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.nu.14.070194.000531%26sid%3Dliteratum%253Aachs%26aulast%3DCornelius%26aufirst%3DP.%26aulast%3DMacDougald%26aufirst%3DO.%2BA.%26aulast%3DLane%26aufirst%3DM.%2BD.%26atitle%3DRegulation%2520of%2520adipocyte%2520development%26jtitle%3DAnnu.%2520Rev.%2520Nutr.%26date%3D1994%26volume%3D14%26spage%3D99%26epage%3D129%26doi%3D10.1146%2Fannurev.nu.14.070194.000531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oishi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manabe, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsushima, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shindo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishimura, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maemura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamauchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubota, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akira, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadowaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, R.</span></span> <span> </span><span class="NLM_article-title">Krüppel-like transcription factor KLF5 is a key regulator of adipocyte differentiation</span>. <i>Cell Metab.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">27</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1016/j.cmet.2004.11.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2Fj.cmet.2004.11.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=16054042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlaqs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2005&pages=27-39&author=Y.+Oishiauthor=I.+Manabeauthor=K.+Tobeauthor=K.+Tsushimaauthor=T.+Shindoauthor=K.+Fujiuauthor=G.+Nishimuraauthor=K.+Maemuraauthor=T.+Yamauchiauthor=N.+Kubotaauthor=R.+Suzukiauthor=T.+Kitamuraauthor=S.+Akiraauthor=T.+Kadowakiauthor=R.+Nagai&title=Kr%C3%BCppel-like+transcription+factor+KLF5+is+a+key+regulator+of+adipocyte+differentiation&doi=10.1016%2Fj.cmet.2004.11.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Krueppel-like transcription factor KLF5 is a key regulator of adipocyte differentiation</span></div><div class="casAuthors">Oishi, Yumiko; Manabe, Ichiro; Tobe, Kazuyuki; Tsushima, Kensuke; Shindo, Takayuki; Fujiu, Katsuhito; Nishimura, Go; Maemura, Koji; Yamauchi, Toshimasa; Kubota, Naoto; Suzuki, Ryo; Kitamura, Toshio; Akira, Shizuo; Kadowaki, Takashi; Nagai, Ryozo</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-39</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Krueppel-like factor 5 (KLF5) is a zinc-finger transcription factor known to play a pivotal role in the pathogenesis of cardiovascular disease.  Here, we show that neonatal heterozygous KLF5 knockout mice exhibit a marked deficiency in white adipose tissue development, suggesting that KLF5 is also required for adipogenesis.  In 3T3-L1 preadipocytes, KLF5 expression was induced at an early stage of differentiation, and this was followed by expression of PPARγ2.  Constitutive overexpression of dominant-neg. KLF5 inhibited adipocyte differentiation, whereas overexpression of wild-type KLF5 induced differentiation even without hormonal stimulation.  Moreover, embryonic fibroblasts obtained from KLF5+/- mice showed much attenuated adipocyte differentiation, confirming the key role played by KLF5 in adipocyte differentiation.  KLF5 expression is induced by C/EBPβ and δ.  KLF5, in turn, acts in concert with C/EBPβ/δ to activate the PPARγ2 promoter.  This study establishes KLF5 as a key component of the transcription factor network controlling adipocyte differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVvhxLphZlcLVg90H21EOLACvtfcHk0lji7F6HRCejnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlaqs78%253D&md5=d56b443c5f82c63fcd4ee7f56806c758</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2004.11.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2004.11.005%26sid%3Dliteratum%253Aachs%26aulast%3DOishi%26aufirst%3DY.%26aulast%3DManabe%26aufirst%3DI.%26aulast%3DTobe%26aufirst%3DK.%26aulast%3DTsushima%26aufirst%3DK.%26aulast%3DShindo%26aufirst%3DT.%26aulast%3DFujiu%26aufirst%3DK.%26aulast%3DNishimura%26aufirst%3DG.%26aulast%3DMaemura%26aufirst%3DK.%26aulast%3DYamauchi%26aufirst%3DT.%26aulast%3DKubota%26aufirst%3DN.%26aulast%3DSuzuki%26aufirst%3DR.%26aulast%3DKitamura%26aufirst%3DT.%26aulast%3DAkira%26aufirst%3DS.%26aulast%3DKadowaki%26aufirst%3DT.%26aulast%3DNagai%26aufirst%3DR.%26atitle%3DKr%25C3%25BCppel-like%2520transcription%2520factor%2520KLF5%2520is%2520a%2520key%2520regulator%2520of%2520adipocyte%2520differentiation%26jtitle%3DCell%2520Metab.%26date%3D2005%26volume%3D1%26spage%3D27%26epage%3D39%26doi%3D10.1016%2Fj.cmet.2004.11.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrens, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anand, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spiegelman, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, J. M.</span></span> <span> </span><span class="NLM_article-title">Krox20 stimulates adipogenesis via C/EBPbeta-dependent and -independent mechanisms</span>. <i>Cell Metab.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">93</span>– <span class="NLM_lpage">106</span>, <span class="refDoi"> DOI: 10.1016/j.cmet.2004.12.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2Fj.cmet.2004.12.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=16054051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD2MXit12rtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2005&pages=93-106&author=Z.+Chenauthor=J.+I.+Torrensauthor=A.+Anandauthor=B.+M.+Spiegelmanauthor=J.+M.+Friedman&title=Krox20+stimulates+adipogenesis+via+C%2FEBPbeta-dependent+and+-independent+mechanisms&doi=10.1016%2Fj.cmet.2004.12.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Krox20 stimulates adipogenesis via C/EBPβ-dependent and -independent mechanisms</span></div><div class="casAuthors">Chen, Zhu; Torrens, Javier I.; Anand, Ashim; Spiegelman, Bruce M.; Friedman, Jeffrey M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">93-106</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Krox20 is a zinc finger-contg. transcription factor that is abundantly expressed in adipose tissue.  However, its role in fat cell differentiation has not been established.  In cultured 3T3-L1 cells, Krox20 is rapidly induced by serum stimulation.  Overexpression of Krox20 in both 3T3-L1 preadipocytes and multipotent NIH3T3 cells promotes adipogenesis in a hormone-dependent manner.  Conversely, RNAi-mediated loss of Krox20 function reduced adipogenesis in 3T3-L1 cells.  Ectopic expression of Krox20 can transactivate the C/EBPβ promoter and increase C/EBPβ gene expression in 3T3-L1 preadipocytes.  RNAi-mediated knockdown of C/EPBβ diminished Krox20's proadipogenic effect.  Finally, coexpression of Krox20 and C/EBPβ in naive NIH3T3 cells resulted in the pronounced induction of a fully differentiated adipocyte phenotype, an effect previously obsd. only with PPARγ.  These data indicate that Krox20 is necessary for adipogenesis and that, when overexpressed, Krox20 potently stimulates adipogenesis via C/EBPβ-dependent and -independent mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpR_wstSnX5FbVg90H21EOLACvtfcHk0lji7F6HRCejnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXit12rtbw%253D&md5=5c7737f5325c5b290a82db8b6d3074b6</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2004.12.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2004.12.009%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DTorrens%26aufirst%3DJ.%2BI.%26aulast%3DAnand%26aufirst%3DA.%26aulast%3DSpiegelman%26aufirst%3DB.%2BM.%26aulast%3DFriedman%26aufirst%3DJ.%2BM.%26atitle%3DKrox20%2520stimulates%2520adipogenesis%2520via%2520C%252FEBPbeta-dependent%2520and%2520-independent%2520mechanisms%26jtitle%3DCell%2520Metab.%26date%3D2005%26volume%3D1%26spage%3D93%26epage%3D106%26doi%3D10.1016%2Fj.cmet.2004.12.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gimble, J. M.</span></span> <span> </span><span class="NLM_article-title">CAAT/enhancer binding proteins directly modulate transcription from the peroxisome proliferator-activated receptor gamma 2 promoter</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>240</i></span>,  <span class="NLM_fpage">99</span>– <span class="NLM_lpage">103</span>, <span class="refDoi"> DOI: 10.1006/bbrc.1997.7627</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1006%2Fbbrc.1997.7627" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=9367890" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADyaK2sXnsVWisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=240&publication_year=1997&pages=99-103&author=S.+L.+Clarkeauthor=C.+E.+Robinsonauthor=J.+M.+Gimble&title=CAAT%2Fenhancer+binding+proteins+directly+modulate+transcription+from+the+peroxisome+proliferator-activated+receptor+gamma+2+promoter&doi=10.1006%2Fbbrc.1997.7627"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">CAAT/enhancer binding proteins directly modulate transcription from the peroxisome proliferator-activated receptor γ2 promoter</span></div><div class="casAuthors">Clarke, Stephen L.; Robinson, Claudius E.; Gimble, Jeffrey M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">240</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">99-103</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">The CCAAT/enhancer binding proteins (C/EBPs) and the peroxisome proliferator-activated receptors (PPARs) together regulate adipogenesis.  The current work uses co-transfection studies to examine the C/EBP dependence of PPARy2 transcription.  Both C/EBPα and C/EBPδ expression vectors activated transcription from a PPARγ2 promoter/luciferase expression vector by 5-6 fold in UMR106 cells.  The simultaneous transfection of the C/EBP homologous protein (CHOP) (also known as growth arrest DNA damage protein 153 or gadd153) inhibited this C/EBP-dependent activation in a concn. dependent manner.  The CHOP protein is known to heterodimerize with other C/EBP proteins to form transcriptionally inactive complexes.  Mutation of the two C/EBP DNA recognition elements at -340 bp and -327 bp within the PPARγ2 promoter reduced the inductive effects of both C/EBPα and C/EBPδ.  These findings demonstrate that proteins within the C/EBP family directly modulate transcription from the PPARγ2 promoter.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq5EeNFa8uOrVg90H21EOLACvtfcHk0lheNxcTT37svw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXnsVWisrY%253D&md5=6ec5e3a65195a1dad3837245f26e666e</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1006%2Fbbrc.1997.7627&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fbbrc.1997.7627%26sid%3Dliteratum%253Aachs%26aulast%3DClarke%26aufirst%3DS.%2BL.%26aulast%3DRobinson%26aufirst%3DC.%2BE.%26aulast%3DGimble%26aufirst%3DJ.%2BM.%26atitle%3DCAAT%252Fenhancer%2520binding%2520proteins%2520directly%2520modulate%2520transcription%2520from%2520the%2520peroxisome%2520proliferator-activated%2520receptor%2520gamma%25202%2520promoter%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1997%26volume%3D240%26spage%3D99%26epage%3D103%26doi%3D10.1006%2Fbbrc.1997.7627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bucher, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farmer, S. R.</span></span> <span> </span><span class="NLM_article-title">Conditional ectopic expression of C/EBP beta in NIH-3T3 cells induces PPAR gamma and stimulates adipogenesis</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2350</span>– <span class="NLM_lpage">2363</span>, <span class="refDoi"> DOI: 10.1101/gad.9.19.2350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1101%2Fgad.9.19.2350" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=7557387" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADyaK2MXos12it74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1995&pages=2350-2363&author=Z.+Wuauthor=Y.+Xieauthor=N.+L.+Bucherauthor=S.+R.+Farmer&title=Conditional+ectopic+expression+of+C%2FEBP+beta+in+NIH-3T3+cells+induces+PPAR+gamma+and+stimulates+adipogenesis&doi=10.1101%2Fgad.9.19.2350"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Conditional ectopic expression of C/EBPβ in NIH-3T3 cells induces PPARγ and stimulates adipogenesis</span></div><div class="casAuthors">Wu, Zhidan; Xie, Yuhong; Bucher, Nancy L. R.; Farmer, Stephen R.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2350-63</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Activation of adipogenesis in 3T3 preadipocytes by exposure to the adipogenic inducers dexamethasone, methylisobutylxanthine, insulin, and fetal bovine serum is accompanied by a transient burst of C/EBPβ protein expression that precedes the induction of the fat gene program.  In this study we have investigated the role of C/EBPβ in initiating the adipogenic program by overexpressing C/EBPβ in multipotential NIH-3T3 fibroblasts.  Conditional ectopic expression of C/EBPβ was accomplished by using an artificial transcriptional regulatory system based on the Escherichia coli tetracycline repressor to generate a stable cell line, β2, that expresses C/EBPβ mRNA and protein in a tightly controlled tetracycline dose-dependent manner.  Induction of C/EBPβ DNA-binding activity in NIH-3T3 β2 cells exposed to dexamethasone in the presence of insulin and fetal bovine serum activates the expression of an adipocyte-specific nuclear hormone receptor, PPARγ, that stimulates the conversion of these fibroblasts into committed preadipocytes.  Either ectopic expression of C/EBPβ or treatment with dexamethasone alone is incapable of inducing PPARγ expression, but when present together, they have a synergistic effect on the adipogenic program.  Exposure of these stimulated cells to a PPAR activator 5,8,11,14-eicosatetraynoic acid (ETYA) results in the accumulation of fat droplets and expression of the adipocyte-enriched genes aP2 and glycerol phosphate dehydrogenase (GPD).  The no. of β2 cells that can differentiate into adipocytes is related to the concn. of tetracycline and, therefore, the amt. of the exogenous C/EBPβ protein expressed.  C/EBPβ can induce PPARγ mRNA in the absence of ETYA; however, expression of aP2 mRNA and max. fat deposition is dependent on the PPAR activator.  Our results suggest that enhanced expression of C/EBPβ converts multipotential mesenchymal precursor cells into preadipocytes that respond to adipogenic inducers, including dexamethasone and PPAR activators to differentiate into adipocytes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq19Iwq-lGFzLVg90H21EOLACvtfcHk0lheNxcTT37svw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXos12it74%253D&md5=76f2dc035e3aa33fca6731feb8dc0fc5</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1101%2Fgad.9.19.2350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.9.19.2350%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DBucher%26aufirst%3DN.%2BL.%26aulast%3DFarmer%26aufirst%3DS.%2BR.%26atitle%3DConditional%2520ectopic%2520expression%2520of%2520C%252FEBP%2520beta%2520in%2520NIH-3T3%2520cells%2520induces%2520PPAR%2520gamma%2520and%2520stimulates%2520adipogenesis%26jtitle%3DGenes%2520Dev.%26date%3D1995%26volume%3D9%26spage%3D2350%26epage%3D2363%26doi%3D10.1101%2Fgad.9.19.2350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shepherd, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahn, B. B.</span></span> <span> </span><span class="NLM_article-title">Glucose transporters and insulin action-implications for insulin resistance and diabetes mellitus</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>341</i></span>,  <span class="NLM_fpage">248</span>– <span class="NLM_lpage">257</span>, <span class="refDoi"> DOI: 10.1056/nejm199907223410406</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1056%2FNEJM199907223410406" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10413738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADyaK1MXltVOju74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=1999&pages=248-257&author=P.+R.+Shepherdauthor=B.+B.+Kahn&title=Glucose+transporters+and+insulin+action-implications+for+insulin+resistance+and+diabetes+mellitus&doi=10.1056%2Fnejm199907223410406"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Glucose transporters and insulin action: implications for insulin resistance and diabetes mellitus</span></div><div class="casAuthors">Shepherd, Peter R.; Kahn, Barbara B.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">248-257</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review, with 103 refs.  This review discusses recent advances, their contribution to our understanding of the pathogenesis of diabetes mellitus, and their implications for the design of new therapies to prevent and treat diabetes and its complications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZ8ajzYZcxc7Vg90H21EOLACvtfcHk0lheNxcTT37svw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXltVOju74%253D&md5=197ffdc34708ec60d21c64024a1d2dff</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1056%2FNEJM199907223410406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJM199907223410406%26sid%3Dliteratum%253Aachs%26aulast%3DShepherd%26aufirst%3DP.%2BR.%26aulast%3DKahn%26aufirst%3DB.%2BB.%26atitle%3DGlucose%2520transporters%2520and%2520insulin%2520action-implications%2520for%2520insulin%2520resistance%2520and%2520diabetes%2520mellitus%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D1999%26volume%3D341%26spage%3D248%26epage%3D257%26doi%3D10.1056%2Fnejm199907223410406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garvey, W. T.</span></span> <span> </span><span class="NLM_article-title">Adiponectin promotes adipocyte differentiation, insulin sensitivity, and lipid accumulation</span>. <i>J. Lipid Res.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">1369</span>– <span class="NLM_lpage">1379</span>, <span class="refDoi"> DOI: 10.1194/jlr.m400373-jlr200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1194%2Fjlr.M400373-JLR200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=15834118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmtFKksL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2005&pages=1369-1379&author=Y.+Fuauthor=N.+Luoauthor=R.+L.+Kleinauthor=W.+T.+Garvey&title=Adiponectin+promotes+adipocyte+differentiation%2C+insulin+sensitivity%2C+and+lipid+accumulation&doi=10.1194%2Fjlr.m400373-jlr200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Adiponectin promotes adipocyte differentiation, insulin sensitivity, and lipid accumulation</span></div><div class="casAuthors">Fu, Yuchang; Luo, Nanlan; Klein, Richard L.; Garvey, W. Timothy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Lipid Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1369-1379</span>CODEN:
                <span class="NLM_cas:coden">JLPRAW</span>;
        ISSN:<span class="NLM_cas:issn">0022-2275</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology, Inc.</span>)
        </div><div class="casAbstract">Adiponectin is secreted from adipocytes, and low circulating levels have been epidemiol. assocd. with obesity, insulin resistance, type 2 diabetes, and cardiovascular disease.  To investigate whether adiponectin could exert autocrine effects in adipocytes, the authors expressed the adiponectin gene in 3T3-L1 fibroblasts.  The authors obsd. that 3T3-L1 fibroblasts expressing adiponectin have a fast growth phase and reach confluence more rapidly compared with control cells or LacZ-transduced cells.  Furthermore, cells with overexpressed adiponectin were obsd. to differentiate into adipocytes more rapidly, and during adipogenesis, they exhibited more prolonged and robust gene expression for related transcriptional factors, CCAAT/enhancer binding protein α (C/EBP2), peroxisome proliferator-activated receptor γ (PPARγ), and adipocyte detn. and differentiation factor 1/sterol-regulatory element binding protein 1c (ADD1/SREBP1c) and earlier suppression of PPARγ coactivator-1α (PGC-1α).  In fully differentiated adipocytes, adiponectin-overexpressing cells accumulated more and larger lipid droplets compared with control cells.  Also, adiponectin increased insulin's ability to maximally stimulate glucose uptake by 78% through increased glucose transporter 4 (GLUT4) gene expression and increased GLUT4 recruitment to the plasma membrane.  These data suggest a new role for adiponectin as an autocrine factor in adipose tissues: promoting cell proliferation and differentiation from preadipocytes into adipocytes, augmenting programmed gene expression responsible for adipogenesis, and increasing lipid content and insulin responsiveness of the glucose transport system in adipocytes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrex0wPiPGuDLVg90H21EOLACvtfcHk0litKvd9Ggqaig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmtFKksL0%253D&md5=a9cbc435c4ba1871e40e11b6cfe31abd</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1194%2Fjlr.M400373-JLR200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1194%252Fjlr.M400373-JLR200%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DN.%26aulast%3DKlein%26aufirst%3DR.%2BL.%26aulast%3DGarvey%26aufirst%3DW.%2BT.%26atitle%3DAdiponectin%2520promotes%2520adipocyte%2520differentiation%252C%2520insulin%2520sensitivity%252C%2520and%2520lipid%2520accumulation%26jtitle%3DJ.%2520Lipid%2520Res.%26date%3D2005%26volume%3D46%26spage%3D1369%26epage%3D1379%26doi%3D10.1194%2Fjlr.m400373-jlr200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Furuhashi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saitoh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miura, T.</span></span> <span> </span><span class="NLM_article-title">Fatty acid-binding protein 4 (FABP4): Pathophysiological insights and potent clinical biomarker of metabolic and cardiovascular diseases</span>. <i>Clin. Med. Insights: Cardiol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.4137/CMC.S17067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.4137%2FCMC.S17067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=25674026" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2015&pages=23-33&author=M.+Furuhashiauthor=S.+Saitohauthor=K.+Shimamotoauthor=T.+Miura&title=Fatty+acid-binding+protein+4+%28FABP4%29%3A+Pathophysiological+insights+and+potent+clinical+biomarker+of+metabolic+and+cardiovascular+diseases&doi=10.4137%2FCMC.S17067"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.4137%2FCMC.S17067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4137%252FCMC.S17067%26sid%3Dliteratum%253Aachs%26aulast%3DFuruhashi%26aufirst%3DM.%26aulast%3DSaitoh%26aufirst%3DS.%26aulast%3DShimamoto%26aufirst%3DK.%26aulast%3DMiura%26aufirst%3DT.%26atitle%3DFatty%2520acid-binding%2520protein%25204%2520%2528FABP4%2529%253A%2520Pathophysiological%2520insights%2520and%2520potent%2520clinical%2520biomarker%2520of%2520metabolic%2520and%2520cardiovascular%2520diseases%26jtitle%3DClin.%2520Med.%2520Insights%253A%2520Cardiol.%26date%3D2015%26volume%3D8%26spage%3D23%26epage%3D33%26doi%3D10.4137%2FCMC.S17067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niesen, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span> <span> </span><span class="NLM_article-title">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">2212</span>– <span class="NLM_lpage">2221</span>, <span class="refDoi"> DOI: 10.1038/nprot.2007.321</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fnprot.2007.321" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=17853878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2212-2221&author=F.+H.+Niesenauthor=H.+Berglundauthor=M.+Vedadi&title=The+use+of+differential+scanning+fluorimetry+to+detect+ligand+interactions+that+promote+protein+stability&doi=10.1038%2Fnprot.2007.321"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span></div><div class="casAuthors">Niesen, Frank H.; Berglund, Helena; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2212-2221</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Differential scanning fluorometry (DSF) is a rapid and inexpensive screening method to identify low-mol.-wt. ligands that bind and stabilize purified proteins.  The temp. at which a protein unfolds is measured by an increase in the fluorescence of a dye with affinity for hydrophobic parts of the protein, which are exposed as the protein unfolds.  A simple fitting procedure allows quick calcn. of the transition midpoint; the difference in the temp. of this midpoint in the presence and absence of ligand is related to the binding affinity of the small mol., which can be a low-mol.-wt. compd., a peptide or a nucleic acid.  DSF is best performed using a conventional real-time PCR instrument.  Ligand solns. from a storage plate are added to a soln. of protein and dye, distributed into the wells of the PCR plate and fluorescence intensity measured as the temp. is raised gradually.  Results can be obtained in a single day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomViqp6Tq7krVg90H21EOLACvtfcHk0litKvd9Ggqaig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N&md5=8a74dbb348ab81179614ab4ae5797d94</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.321%26sid%3Dliteratum%253Aachs%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DBerglund%26aufirst%3DH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DThe%2520use%2520of%2520differential%2520scanning%2520fluorimetry%2520to%2520detect%2520ligand%2520interactions%2520that%2520promote%2520protein%2520stability%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D2%26spage%3D2212%26epage%3D2221%26doi%3D10.1038%2Fnprot.2007.321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiseman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williston, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandts, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.-N.</span></span> <span> </span><span class="NLM_article-title">Rapid measurement of binding constants and heats of binding using a new titration calorimeter</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>179</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">137</span>, <span class="refDoi"> DOI: 10.1016/0003-2697(89)90213-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2F0003-2697%2889%2990213-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=2757186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADyaL1MXktlKjsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=1989&pages=131-137&author=T.+Wisemanauthor=S.+Willistonauthor=J.+F.+Brandtsauthor=L.-N.+Lin&title=Rapid+measurement+of+binding+constants+and+heats+of+binding+using+a+new+titration+calorimeter&doi=10.1016%2F0003-2697%2889%2990213-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid measurement of binding constants and heats of binding using a new titration calorimeter</span></div><div class="casAuthors">Wiseman, Thomas; Williston, Samuel; Brandts, John F.; Lin, Lung Nan</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">179</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">131-7</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">A new titrn. calorimeter is described and results are presented for the binding of 2'-CMP to the active site of RNase A.  The instrument characteristics include very high sensitivity, rapid calorimetric response, and fast thermal equilibration.  Convenient software is available for instrument operation, data collection, data redn., and deconvolution to obtain least-squares ests. of binding parameters n, ΔH°, ΔS°, and the binding const. K.  Sample through-put for the instrument is high, and under favorable conditions binding consts. as large as 108 M-1 can be measured.  The bovine RNase A (RNase)/2'-CMP system was studied over a 50-fold range of RNase concn. and at 2 different temps.  The binding consts. were in the 105 to 106 M-1 range, depending on conditions, and heats of binding ∼-15,000 cal/mol.  Repeat detns. suggested errors of only a few percent in n, ΔH°, and K values over the most favorable concn. range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJRWydqrfWArVg90H21EOLACvtfcHk0litKvd9Ggqaig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXktlKjsrw%253D&md5=53fcdb7144fd85e57963add0bcdb6041</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1016%2F0003-2697%2889%2990213-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-2697%252889%252990213-3%26sid%3Dliteratum%253Aachs%26aulast%3DWiseman%26aufirst%3DT.%26aulast%3DWilliston%26aufirst%3DS.%26aulast%3DBrandts%26aufirst%3DJ.%2BF.%26aulast%3DLin%26aufirst%3DL.-N.%26atitle%3DRapid%2520measurement%2520of%2520binding%2520constants%2520and%2520heats%2520of%2520binding%2520using%2520a%2520new%2520titration%2520calorimeter%26jtitle%3DAnal.%2520Biochem.%26date%3D1989%26volume%3D179%26spage%3D131%26epage%3D137%26doi%3D10.1016%2F0003-2697%2889%2990213-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Battye, T. G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kontogiannis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span> <span> </span><span class="NLM_article-title">iMOSFLM: a new graphical interface for diffraction-image processing with MOSFLM</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">271</span>– <span class="NLM_lpage">281</span>, <span class="refDoi"> DOI: 10.1107/s0907444910048675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1107%2Fs0907444910048675" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=21460445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=271-281&author=T.+G.+G.+Battyeauthor=L.+Kontogiannisauthor=O.+Johnsonauthor=H.+R.+Powellauthor=A.+G.+W.+Leslie&title=iMOSFLM%3A+a+new+graphical+interface+for+diffraction-image+processing+with+MOSFLM&doi=10.1107%2Fs0907444910048675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">iMOSFLM: A new graphical interface for diffraction-image processing with MOSFLM</span></div><div class="casAuthors">Battye, T. Geoff G.; Kontogiannis, Luke; Johnson, Owen; Powell, Harold R.; Leslie, Andrew G. W.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">271-281</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">IMOSFLM is a graphical user interface to the diffraction data-integration program MOSFLM.  It is designed to simplify data processing by dividing the process into a series of steps, which are normally carried out sequentially.  Each step has its own display pane, allowing control over parameters that influence that step and providing graphical feedback to the user.  Suitable values for integration parameters are set automatically, but addnl. menus provide a detailed level of control for experienced users.  The image display and the interfaces to the different tasks (indexing, strategy calcn., cell refinement, integration and history) are described.  The most important parameters for each step and the best way of assessing success or failure are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeZl2UOMg3_bVg90H21EOLACvtfcHk0ljUsZKPJGEYGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLw%253D&md5=852651253f8d9c9a81e0513b6633c705</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1107%2Fs0907444910048675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444910048675%26sid%3Dliteratum%253Aachs%26aulast%3DBattye%26aufirst%3DT.%2BG.%2BG.%26aulast%3DKontogiannis%26aufirst%3DL.%26aulast%3DJohnson%26aufirst%3DO.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26atitle%3DiMOSFLM%253A%2520a%2520new%2520graphical%2520interface%2520for%2520diffraction-image%2520processing%2520with%2520MOSFLM%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D271%26epage%3D281%26doi%3D10.1107%2Fs0907444910048675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2Fs0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0ljUsZKPJGEYGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2Fs0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span> <span> </span><span class="NLM_article-title">An introduction to data reduction: space-group determination, scaling and intensity statistics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">282</span>– <span class="NLM_lpage">292</span>, <span class="refDoi"> DOI: 10.1107/s090744491003982x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1107%2Fs090744491003982x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=21460446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=282-292&author=P.+R.+Evans&title=An+introduction+to+data+reduction%3A+space-group+determination%2C+scaling+and+intensity+statistics&doi=10.1107%2Fs090744491003982x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">An introduction to data reduction: Space-group determination, scaling and intensity statistics</span></div><div class="casAuthors">Evans, Philip R.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">282-292</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  This paper presents an overview of how to run the CCP4 programs for data redn. (SCALA, POINTLESS and CTRUNCATE) through the CCP4 graphical interface ccp4i and points out some issues that need to be considered, together with a few examples.  It covers detn. of the point-group symmetry of the diffraction data (the Laue group), which is required for the subsequent scaling step, examn. of systematic absences, which in many cases will allow inference of the space group, putting multiple data sets on a common indexing system when there are alternatives, the scaling step itself, which produces a large set of data-quality indicators, estn. of |F| from intensity and finally examn. of intensity statistics to detect crystal pathologies such as twinning.  An appendix outlines the scoring schemes used by the program POINTLESS to assign probabilities to possible Laue and space groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEq8RhfQOW8LVg90H21EOLACvtfcHk0ljUsZKPJGEYGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLo%253D&md5=f425388a87744d721d324118d50e4f9a</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1107%2Fs090744491003982x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs090744491003982x%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26atitle%3DAn%2520introduction%2520to%2520data%2520reduction%253A%2520space-group%2520determination%252C%2520scaling%2520and%2520intensity%2520statistics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D282%26epage%3D292%26doi%3D10.1107%2Fs090744491003982x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/s0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2Fs0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lgkWww5Xsn40g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2Fs0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/s0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2Fs0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lgkWww5Xsn40g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2Fs0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0ljPqBYhv19Gog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/s0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2Fs0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0ljPqBYhv19Gog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2Fs0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roehm, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodgers, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatfield, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glasebrook, A. L.</span></span> <span> </span><span class="NLM_article-title">An improved colorimetric assay for cell proliferation and viability utilizing the tetrazolium salt XTT</span>. <i>J. Immunol. Methods</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">257</span>– <span class="NLM_lpage">265</span>, <span class="refDoi"> DOI: 10.1016/0022-1759(91)90114-u</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2F0022-1759%2891%2990114-U" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1919029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A280%3ADyaK38%252FhtlWqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=1991&pages=257-265&author=N.+W.+Roehmauthor=G.+H.+Rodgersauthor=S.+M.+Hatfieldauthor=A.+L.+Glasebrook&title=An+improved+colorimetric+assay+for+cell+proliferation+and+viability+utilizing+the+tetrazolium+salt+XTT&doi=10.1016%2F0022-1759%2891%2990114-u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">An improved colorimetric assay for cell proliferation and viability utilizing the tetrazolium salt XTT</span></div><div class="casAuthors">Roehm N W; Rodgers G H; Hatfield S M; Glasebrook A L</div><div class="citationInfo"><span class="NLM_cas:title">Journal of immunological methods</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">257-65</span>
        ISSN:<span class="NLM_cas:issn">0022-1759</span>.
    </div><div class="casAbstract">A new tetrazolium salt XTT, sodium 3'-[1-[(phenylamino)-carbonyl]-3,4-tetrazolium]-bis(4-methoxy-6- nitro)benzene-sulfonic acid hydrate, was evaluated for use in a colorimetric assay for cell viability and proliferation by normal activated T cells and several cytokine dependent cell lines.  Cleavage of XTT by dehydrogenase enzymes of metabolically active cells yields a highly colored formazan product which is water soluble.  This feature obviates the need for formazan crystal solubilization prior to absorbance measurements, as required when using other tetrazolium salts such as MTT.  Bioreduction of XTT by all the murine cells examined was not particularly efficient, but could be potentiated by addition of electron coupling agents such as phenazine methosulfate (PMS) or menadione (MEN).  Optimal concentrations of PMS or MEN were determined for the metabolism of XTT by the T cell lines HT-2 and 11.6, NFS-60 a myeloid leukemia, MC/9 a mast cell line and mitogen activated splenic T cells.  When used in combination with PMS, each of these cells generated higher formazan absorbance values with XTT than were observed with MTT.  Thus the use of XTT in colorimetric proliferation assays offer significant advantages over MTT, resulting from reduced assay time and sample handling, while offering equivalent sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSIr3tbWsUOa8rRqEXsZufAfW6udTcc2eYC7oMUgyaLxLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK38%252FhtlWqsw%253D%253D&md5=744ee298cf13e12bda5f93b5ec26be2d</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1016%2F0022-1759%2891%2990114-U&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-1759%252891%252990114-U%26sid%3Dliteratum%253Aachs%26aulast%3DRoehm%26aufirst%3DN.%2BW.%26aulast%3DRodgers%26aufirst%3DG.%2BH.%26aulast%3DHatfield%26aufirst%3DS.%2BM.%26aulast%3DGlasebrook%26aufirst%3DA.%2BL.%26atitle%3DAn%2520improved%2520colorimetric%2520assay%2520for%2520cell%2520proliferation%2520and%2520viability%2520utilizing%2520the%2520tetrazolium%2520salt%2520XTT%26jtitle%3DJ.%2520Immunol.%2520Methods%26date%3D1991%26volume%3D142%26spage%3D257%26epage%3D265%26doi%3D10.1016%2F0022-1759%2891%2990114-u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Göbel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heering, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angioni, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittmann, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Buscato, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kottke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weizel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schader, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maier, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geisslinger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schubert-Zsilavecz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinhilber, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proschak, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahnt, A. S.</span></span> <span> </span><span class="NLM_article-title">Zafirlukast is a dual modulator of human soluble epoxide hydrolase and peroxisome proliferator-activated receptor γ</span>. <i>Front. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">263</span>, <span class="refDoi"> DOI: 10.3389/fphar.2019.00263</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.3389%2Ffphar.2019.00263" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=30949053" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A280%3ADC%252BB3M%252FhvVOhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=263&author=T.+G%C3%B6belauthor=O.+Diehlauthor=J.+Heeringauthor=D.+Merkauthor=C.+Angioniauthor=S.+K.+Wittmannauthor=E.+La+Buscatoauthor=R.+Kottkeauthor=L.+Weizelauthor=T.+Schaderauthor=T.+J.+Maierauthor=G.+Geisslingerauthor=M.+Schubert-Zsilaveczauthor=D.+Steinhilberauthor=E.+Proschakauthor=A.+S.+Kahnt&title=Zafirlukast+is+a+dual+modulator+of+human+soluble+epoxide+hydrolase+and+peroxisome+proliferator-activated+receptor+%CE%B3&doi=10.3389%2Ffphar.2019.00263"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Zafirlukast Is a Dual Modulator of Human Soluble Epoxide Hydrolase and Peroxisome Proliferator-Activated Receptor γ</span></div><div class="casAuthors">Gobel Tamara; Diehl Olaf; Merk Daniel; Wittmann Sandra K; Buscato Estel la; Kottke Ramona; Weizel Lilia; Schader Tim; Schubert-Zsilavecz Manfred; Steinhilber Dieter; Proschak Ewgenij; Kahnt Astrid S; Heering Jan; Geisslinger Gerd; Angioni Carlo; Geisslinger Gerd; Maier Thorsten J</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in pharmacology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">263</span>
        ISSN:<span class="NLM_cas:issn">1663-9812</span>.
    </div><div class="casAbstract">Cysteinyl leukotriene receptor 1 antagonists (CysLT1RA) are frequently used as add-on medication for the treatment of asthma.  Recently, these compounds have shown protective effects in cardiovascular diseases.  This prompted us to investigate their influence on soluble epoxide hydrolase (sEH) and peroxisome proliferator activated receptor (PPAR) activities, two targets known to play an important role in CVD and the metabolic syndrome.  Montelukast, pranlukast and zafirlukast inhibited human sEH with IC50 values of 1.9, 14.1, and 0.8 μM, respectively.  In contrast, only montelukast and zafirlukast activated PPARγ in the reporter gene assay with EC50 values of 1.17 μM (21.9% max. activation) and 2.49 μM (148% max. activation), respectively.  PPARα and δ were not affected by any of the compounds.  The activation of PPARγ was further investigated in 3T3-L1 adipocytes.  Analysis of lipid accumulation, mRNA and protein expression of target genes as well as PPARγ phosphorylation revealed that montelukast was not able to induce adipocyte differentiation.  In contrast, zafirlukast triggered moderate lipid accumulation compared to rosiglitazone and upregulated PPARγ target genes.  In addition, we found that montelukast and zafirlukast display antagonistic activities concerning recruitment of the PPARγ cofactor CBP upon ligand binding suggesting that both compounds act as PPARγ modulators.  In addition, zafirlukast impaired the TNFα triggered phosphorylation of PPARγ2 on serine 273.  Thus, zafirlukast is a novel dual sEH/PPARγ modulator representing an excellent starting point for the further development of this compound class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcShvSmLL7MEOacY7HFp97qMfW6udTcc2eb8A21UViPqk7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M%252FhvVOhsw%253D%253D&md5=bb60a84a4417a8bc878c5a420043a395</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2019.00263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2019.00263%26sid%3Dliteratum%253Aachs%26aulast%3DG%25C3%25B6bel%26aufirst%3DT.%26aulast%3DDiehl%26aufirst%3DO.%26aulast%3DHeering%26aufirst%3DJ.%26aulast%3DMerk%26aufirst%3DD.%26aulast%3DAngioni%26aufirst%3DC.%26aulast%3DWittmann%26aufirst%3DS.%2BK.%26aulast%3DLa%2BBuscato%26aufirst%3DE.%26aulast%3DKottke%26aufirst%3DR.%26aulast%3DWeizel%26aufirst%3DL.%26aulast%3DSchader%26aufirst%3DT.%26aulast%3DMaier%26aufirst%3DT.%2BJ.%26aulast%3DGeisslinger%26aufirst%3DG.%26aulast%3DSchubert-Zsilavecz%26aufirst%3DM.%26aulast%3DSteinhilber%26aufirst%3DD.%26aulast%3DProschak%26aufirst%3DE.%26aulast%3DKahnt%26aufirst%3DA.%2BS.%26atitle%3DZafirlukast%2520is%2520a%2520dual%2520modulator%2520of%2520human%2520soluble%2520epoxide%2520hydrolase%2520and%2520peroxisome%2520proliferator-activated%2520receptor%2520%25CE%25B3%26jtitle%3DFront.%2520Pharmacol.%26date%3D2019%26volume%3D10%26spage%3D263%26doi%3D10.3389%2Ffphar.2019.00263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zebisch, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voigt, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wabitsch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandsch, M.</span></span> <span> </span><span class="NLM_article-title">Protocol for effective differentiation of 3T3-L1 cells to adipocytes</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>425</i></span>,  <span class="NLM_fpage">88</span>– <span class="NLM_lpage">90</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2012.03.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2Fj.ab.2012.03.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=22425542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC38XlvFyrurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=425&publication_year=2012&pages=88-90&author=K.+Zebischauthor=V.+Voigtauthor=M.+Wabitschauthor=M.+Brandsch&title=Protocol+for+effective+differentiation+of+3T3-L1+cells+to+adipocytes&doi=10.1016%2Fj.ab.2012.03.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Protocol for effective differentiation of 3T3-L1 cells to adipocytes</span></div><div class="casAuthors">Zebisch, Katja; Voigt, Valerie; Wabitsch, Martin; Brandsch, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">425</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">88-90</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In this note, we present a detailed procedure for highly effective and reproducible 3T3-L1 cell differentiation.  Due to their potential to differentiate from fibroblasts to adipocytes, 3T3-L1 cells are widely used for studying adipogenesis and the biochem. of adipocytes.  However, using different kits and protocols published so far, we were not able to obtain full differentiation of the currently available American Type Culture Collection (ATCC) 3T3-L1 cell lots.  Using rosiglitazone (2 μM) as an addnl. prodifferentiative agent, we achieved apparently complete differentiation of 3T3-L1 cells within 10 to 12 days that persisted for at least up to cell culture passage 10.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBHtWKXZs9I7Vg90H21EOLACvtfcHk0liB5pzvlp1IbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlvFyrurw%253D&md5=6e59b59b38fb7a42a80d0289c12766e6</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2012.03.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2012.03.005%26sid%3Dliteratum%253Aachs%26aulast%3DZebisch%26aufirst%3DK.%26aulast%3DVoigt%26aufirst%3DV.%26aulast%3DWabitsch%26aufirst%3DM.%26aulast%3DBrandsch%26aufirst%3DM.%26atitle%3DProtocol%2520for%2520effective%2520differentiation%2520of%25203T3-L1%2520cells%2520to%2520adipocytes%26jtitle%3DAnal.%2520Biochem.%26date%3D2012%26volume%3D425%26spage%3D88%26epage%3D90%26doi%3D10.1016%2Fj.ab.2012.03.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Daina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michielin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoete, V.</span></span> <span> </span><span class="NLM_article-title">SwissADME: a free web tool to evaluate pharmacokinetics, drug-likeness and medicinal chemistry friendliness of small molecules</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">42717</span>, <span class="refDoi"> DOI: 10.1038/srep42717</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fsrep42717" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=28256516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A280%3ADC%252BC1czisFSrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=42717&author=A.+Dainaauthor=O.+Michielinauthor=V.+Zoete&title=SwissADME%3A+a+free+web+tool+to+evaluate+pharmacokinetics%2C+drug-likeness+and+medicinal+chemistry+friendliness+of+small+molecules&doi=10.1038%2Fsrep42717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">SwissADME: a free web tool to evaluate pharmacokinetics, drug-likeness and medicinal chemistry friendliness of small molecules</span></div><div class="casAuthors">Daina Antoine; Michielin Olivier; Zoete Vincent; Michielin Olivier; Michielin Olivier</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">42717</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">To be effective as a drug, a potent molecule must reach its target in the body in sufficient concentration, and stay there in a bioactive form long enough for the expected biologic events to occur.  Drug development involves assessment of absorption, distribution, metabolism and excretion (ADME) increasingly earlier in the discovery process, at a stage when considered compounds are numerous but access to the physical samples is limited.  In that context, computer models constitute valid alternatives to experiments.  Here, we present the new SwissADME web tool that gives free access to a pool of fast yet robust predictive models for physicochemical properties, pharmacokinetics, drug-likeness and medicinal chemistry friendliness, among which in-house proficient methods such as the BOILED-Egg, iLOGP and Bioavailability Radar.  Easy efficient input and interpretation are ensured thanks to a user-friendly interface through the login-free website http://www.swissadme.ch.  Specialists, but also nonexpert in cheminformatics or computational chemistry can predict rapidly key parameters for a collection of molecules to support their drug discovery endeavours.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRl_Uk5dOMKs6qcneEtCi92fW6udTcc2eb8A21UViPqk7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1czisFSrtg%253D%253D&md5=9715b8cb8a34b17c4c73ff69a5a8cc50</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1038%2Fsrep42717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep42717%26sid%3Dliteratum%253Aachs%26aulast%3DDaina%26aufirst%3DA.%26aulast%3DMichielin%26aufirst%3DO.%26aulast%3DZoete%26aufirst%3DV.%26atitle%3DSwissADME%253A%2520a%2520free%2520web%2520tool%2520to%2520evaluate%2520pharmacokinetics%252C%2520drug-likeness%2520and%2520medicinal%2520chemistry%2520friendliness%2520of%2520small%2520molecules%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D42717%26doi%3D10.1038%2Fsrep42717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 1 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marek  Wanior</span>, <span class="hlFld-ContribAuthor ">Andreas  Krämer</span>, <span class="hlFld-ContribAuthor ">Stefan  Knapp</span>, <span class="hlFld-ContribAuthor ">Andreas C.  Joerger</span>. </span><span class="cited-content_cbyCitation_article-title">Exploiting vulnerabilities of SWI/SNF chromatin remodelling complexes for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Oncogene</span><span> <strong>2021,</strong> <em>40 </em>
                                    (21)
                                     , 3637-3654. <a href="https://doi.org/10.1038/s41388-021-01781-x" title="DOI URL">https://doi.org/10.1038/s41388-021-01781-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41388-021-01781-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41388-021-01781-x%26sid%3Dliteratum%253Aachs%26jtitle%3DOncogene%26atitle%3DExploiting%252Bvulnerabilities%252Bof%252BSWI%25252FSNF%252Bchromatin%252Bremodelling%252Bcomplexes%252Bfor%252Bcancer%252Btherapy%26aulast%3DWanior%26aufirst%3DMarek%26date%3D2021%26date%3D2021%26volume%3D40%26issue%3D21%26spage%3D3637%26epage%3D3654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/medium/jm0c01242_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/large/jm0c01242_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01242&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/medium/jm0c01242_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/large/jm0c01242_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Sequence and structure of bromodomain family VIII members. (A) Sequence alignment of human polybromo-1 and SMARCA2/4 bromodomains. The bromodomains PB1(1) and PB1(6) have been excluded because of their inaccessible acetyllysine binding sites and lack of conserved acetyllysine-binding residues.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Used were the UniProt entries Q86U86, P51531 (short isoform 2), and P51532. Secondary structure elements are shown above the alignment, based on the crystals structure of PB1(2) (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3LJW">3LJW</a>). Key interacting residues in the ligand-binding pocket are highlighted. (B) Cartoon representation of the structure of the SMARCA4 BRD in complex with inhibitor PFI-3 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKD">5DKD</a>). The inhibitor is shown as a yellow stick model. Residues that are strictly conserved in the alignment above are highlighted in green. (C) Close-up view of the binding mode of PFI-3 in SMARCA4. Selected side chains interacting with the inhibitor are shown as stick models. A conserved structural water molecule at the bottom of the binding pocket is highlighted as a red sphere. (D) Chemical structures of published SMARCA2/4 and PB1(5) bromodomain inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/large/jm0c01242_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01242&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/medium/jm0c01242_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/large/jm0c01242_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Synthesis Route of 2-(6-Aminopyridazin-3-yl)phenol Derivatives<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/large/jm0c01242_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01242&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (I) DIEA, MeCN, reflux; (II) XPhos Pd G3, K<sub>2</sub>CO<sub>3</sub>, MeCN/H<sub>2</sub>O, microwave, 120 °C; (Ia) <i>N</i>-methylation of compounds <i>R</i>-<b>30</b> and <i>S</i>-<b>31</b> was performed with NaH, MeI, THF, 0–40 °C prior to step (I). Compound <b>34</b> was prepared <i>via</i> intermediate <b>33</b>. For final compounds <b>22–32</b> and <b>34–39</b>, see <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>. Additional compound characterization data are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_001.csv" class="ext-link">Supporting Information</a>.</p></p></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/medium/jm0c01242_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/large/jm0c01242_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Bromodomain family VIII binders. (A) Heat map of the synthesized inhibitors screened against the acetyllysine-binding members of bromodomain family VIII using thermal shift assays. (B) Phylogenetic tree of bromodomains and subdivision into eight subgroups. Thermal shifts, Δ<i>T</i><sub>m</sub>, upon binding of <b>22</b> to different BRD family members are indicated. (C) ITC data for compound <b>22</b>. Corrected heat rates of reverse titration of SMARCA4 and ligand <b>22</b> are displayed (top). Integrated ligand binding heats were fitted by assuming a sigmoidal dose–response relationship (four parameters) (bottom). (D) Cocrystal structures of <b>23</b> with PB1(5) (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZS3">6ZS3</a>) and (E) <b>22</b> with PB1(5) (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZS4">6ZS4</a>), highlighting the interacting amino acids and the remaining conserved water upon ligand binding. 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps for the ligands are shown at contour levels of 1.5 σ (D) and 1.8 σ (E). Hydrogen bonds are shown as dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/large/jm0c01242_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01242&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/medium/jm0c01242_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/large/jm0c01242_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Bromodomain family VIII binders. (A) Heat map of the synthesized inhibitors screened against the acetyllysine-binding members of bromodomain family VIII using thermal shift assays. (B) Phylogenetic tree of bromodomains and subdivision into 8 subgroups. Thermal shift, Δ<i>T</i><sub>m</sub>s, upon binding of <i>R</i>-<b>28</b> to representative BRD family members are indicated. (C) ITC data of compound <b>32</b>. Corrected heat rates of reverse titration of PB1(3) and ligand <b>32</b> are shown (top). Integrated ligand binding heats were fitted by assuming a sigmoidal dose–response relationship (four parameters) (bottom). (D) Cocrystal structure of the PB1(2)-<i>R</i>-<b>26</b> complex (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZN6">6ZN6</a>). A 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map for the ligand is shown at a contour level of 1.2 σ. (E) Cocrystal structure of the PB1(2)-<b>32</b> complex (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZNV">6ZNV</a>) superimposed onto the structure of ligand-free PB1(4) (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3TLP">3TLP</a>). A 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map for the ligand is shown at a contour level of 1.5 σ. Hydrogen bonds are shown as dashed lines. The numbering of residues is according to the canonical UniProt sequence Q86U86, which is different from the numbering in the actual PDB entry that is based on a different isoform.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/large/jm0c01242_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01242&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/medium/jm0c01242_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/large/jm0c01242_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Effects of BRD family VIII inhibitors on adipogenesis. (A) Lipid accumulation during adipogenesis detected by staining with oil red O solution on day 14. 3T3-L1 murine fibroblasts were cotreated with rosiglitazone and SWI/SNF inhibitors during differentiation into adipocytes at indicated concentrations over 14 days. Induction of adipogenesis was performed by an activation cocktail (IBMX, insulin, and dexamethasone). Experiments were performed in four biological replicates. (B) Influence of SWI/SNF bromodomain inhibitors PFI-3 (<b>1</b>) and <b>22</b> on mRNA expression levels of adipocyte markers, PPARγ, C/EBPα, and FABP4, measured by quantitative PCR (qPCR) at different time points following the induction of differentiation. Results were analyzed with the comparative 2<sup>–ΔΔ<i>C</i><sub><i>T</i></sub></sup> method compared to agonist rosiglitazone treatment (left) or unstimulated control cells (w/o; right). Data represent the mean ± SEM of three independent biological experiments. Statistical significance of the difference in mRNA levels between <b>22</b> cotreated with rosiglitazone and rosiglitazone only treated cells (after 5 days) was calculated based on Δ<i>C</i><sub><i>T</i></sub> values using an unpaired Student <i>t</i>-test. **<i>P</i> ≤ 0.010, *<i>P</i> < 0.020, not significant (ns).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01242/20201203/images/large/jm0c01242_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01242&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i59">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01144" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01144" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 87 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span> <span> </span><span class="NLM_article-title">Cancer epigenetics: from mechanism to therapy</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>150</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.06.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2Fj.cell.2012.06.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=22770212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvFGjsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2012&pages=12-27&author=M.+A.+Dawsonauthor=T.+Kouzarides&title=Cancer+epigenetics%3A+from+mechanism+to+therapy&doi=10.1016%2Fj.cell.2012.06.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer Epigenetics: From Mechanism to Therapy</span></div><div class="casAuthors">Dawson, Mark A.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-27</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The epigenetic regulation of DNA-templated processes has been intensely studied over the last 15 years.  DNA methylation, histone modification, nucleosome remodeling, and RNA-mediated targeting regulate many biol. processes that are fundamental to the genesis of cancer.  Here, we present the basic principles behind these epigenetic pathways and highlight the evidence suggesting that their misregulation can culminate in cancer.  This information, along with the promising clin. and preclin. results seen with epigenetic drugs against chromatin regulators, signifies that it is time to embrace the central role of epigenetics in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuyeeliq_34rVg90H21EOLACvtfcHk0ljnCR-W5mX3vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvFGjsLg%253D&md5=bc0176364205a2e66e1971fe789d36ae</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.06.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.06.013%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DCancer%2520epigenetics%253A%2520from%2520mechanism%2520to%2520therapy%26jtitle%3DCell%26date%3D2012%26volume%3D150%26spage%3D12%26epage%3D27%26doi%3D10.1016%2Fj.cell.2012.06.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klemm, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shipony, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenleaf, W. J.</span></span> <span> </span><span class="NLM_article-title">Chromatin accessibility and the regulatory epigenome</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">207</span>– <span class="NLM_lpage">220</span>, <span class="refDoi"> DOI: 10.1038/s41576-018-0089-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fs41576-018-0089-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=30675018" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmtFGqsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=207-220&author=S.+L.+Klemmauthor=Z.+Shiponyauthor=W.+J.+Greenleaf&title=Chromatin+accessibility+and+the+regulatory+epigenome&doi=10.1038%2Fs41576-018-0089-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin accessibility and the regulatory epigenome</span></div><div class="casAuthors">Klemm, Sandy L.; Shipony, Zohar; Greenleaf, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">207-220</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Phys. access to DNA is a highly dynamic property of chromatin that plays an essential role in establishing and maintaining cellular identity.  The organization of accessible chromatin across the genome reflects a network of permissible phys. interactions through which enhancers, promoters, insulators and chromatin-binding factors cooperatively regulate gene expression.  This landscape of accessibility changes dynamically in response to both external stimuli and developmental cues, and emerging evidence suggests that homeostatic maintenance of accessibility is itself dynamically regulated through a competitive interplay between chromatin-binding factors and nucleosomes.  In this Review, we examine how the accessible genome is measured and explore the role of transcription factors in initiating accessibility remodeling; our goal is to illustrate how chromatin accessibility defines regulatory elements within the genome and how these epigenetic features are dynamically established to control gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoP-5OBDLCo5LVg90H21EOLACvtfcHk0lgQ0Wqz2je75Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmtFGqsbk%253D&md5=60707e7c4467e9f7b0edbba256e66832</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fs41576-018-0089-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41576-018-0089-8%26sid%3Dliteratum%253Aachs%26aulast%3DKlemm%26aufirst%3DS.%2BL.%26aulast%3DShipony%26aufirst%3DZ.%26aulast%3DGreenleaf%26aufirst%3DW.%2BJ.%26atitle%3DChromatin%2520accessibility%2520and%2520the%2520regulatory%2520epigenome%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2019%26volume%3D20%26spage%3D207%26epage%3D220%26doi%3D10.1038%2Fs41576-018-0089-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ornaghi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballario, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuhaus, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filetici, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Travers, A. A.</span></span> <span> </span><span class="NLM_article-title">The structural basis for the recognition of acetylated histone H4 by the bromodomain of histone acetyltransferase gcn5p</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">6141</span>– <span class="NLM_lpage">6149</span>, <span class="refDoi"> DOI: 10.1093/emboj/19.22.6141</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1093%2Femboj%2F19.22.6141" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=11080160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD3cXovVaitro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=6141-6149&author=D.+J.+Owenauthor=P.+Ornaghiauthor=J.+C.+Yangauthor=N.+Loweauthor=P.+R.+Evansauthor=P.+Ballarioauthor=D.+Neuhausauthor=P.+Fileticiauthor=A.+A.+Travers&title=The+structural+basis+for+the+recognition+of+acetylated+histone+H4+by+the+bromodomain+of+histone+acetyltransferase+gcn5p&doi=10.1093%2Femboj%2F19.22.6141"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The structural basis for the recognition of acetylated histone H4 by the bromodomain of histone acetyltransferase Gcn5p</span></div><div class="casAuthors">Owen, David J.; Ornaghi, Prisca; Yang, Ji-Chun; Lowe, Nicholas; Evans, Philip R.; Ballario, Paola; Neuhaus, David; Filetici, Patrizia; Travers, Andrew A.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6141-6149</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The bromodomain is an ∼110 amino acid module found in histone acetyltransferases and the ATPase component of certain nucleosome remodeling complexes.  We report the crystal structure at 1.9 Å resoln. of the Saccharomyces cerevisiae Gcn5p bromodomain complexed with a peptide corresponding to residues 15-29 of histone H4 acetylated at the ζ-N of lysine 16.  We show that this bromodomain preferentially binds to peptides contg. an N-acetyl lysine residue.  Only residues 16-19 of the acetylated peptide interact with the bromodomain.  The primary interaction is the N-acetyl lysine binding in a cleft with the specificity provided by the interaction of the amide nitrogen of a conserved asparagine with the oxygen of the acetyl carbonyl group.  A network of water-mediated H-bonds with protein main chain carbonyl groups at the base of the cleft contributes to the binding.  Addnl. side chain binding occurs on a shallow depression that is hydrophobic at one end and can accommodate charge interactions at the other.  These findings suggest that the Gcn5p bromodomain may discriminate between different acetylated lysine residues depending on the context in which they are displayed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp0LsEWlzQl7Vg90H21EOLACvtfcHk0lgQ0Wqz2je75Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXovVaitro%253D&md5=1cc0a6434f61362e83830becc7bdc594</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F19.22.6141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F19.22.6141%26sid%3Dliteratum%253Aachs%26aulast%3DOwen%26aufirst%3DD.%2BJ.%26aulast%3DOrnaghi%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DJ.%2BC.%26aulast%3DLowe%26aufirst%3DN.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DBallario%26aufirst%3DP.%26aulast%3DNeuhaus%26aufirst%3DD.%26aulast%3DFiletici%26aufirst%3DP.%26aulast%3DTravers%26aufirst%3DA.%2BA.%26atitle%3DThe%2520structural%2520basis%2520for%2520the%2520recognition%2520of%2520acetylated%2520histone%2520H4%2520by%2520the%2520bromodomain%2520of%2520histone%2520acetyltransferase%2520gcn5p%26jtitle%3DEMBO%2520J.%26date%3D2000%26volume%3D19%26spage%3D6141%26epage%3D6149%26doi%3D10.1093%2Femboj%2F19.22.6141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.-P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+M%C3%BCllerauthor=T.+Pawsonauthor=A.-C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0lgQ0Wqz2je75Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Bromodomains as therapeutic targets</span>. <i>Expert Rev. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>13</i></span>, <span class="NLM_elocation-id">e29</span> <span class="refDoi"> DOI: 10.1017/s1462399411001992</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1017%2FS1462399411001992" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=21933453" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1OhurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&author=S.+Mullerauthor=P.+Filippakopoulosauthor=S.+Knapp&title=Bromodomains+as+therapeutic+targets&doi=10.1017%2Fs1462399411001992"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomains as therapeutic targets</span></div><div class="casAuthors">Muller, Susanne; Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Expert Reviews in Molecular Medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e29/1-e29/21</span>CODEN:
                <span class="NLM_cas:coden">ERMMFS</span>;
        ISSN:<span class="NLM_cas:issn">1462-3994</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">A review.  Acetylation of lysine residues is a post-translational modification with broad relevance to cellular signalling and disease biol.  Enzymes that 'write' (histone acetyltransferases, HATs) and 'erase' (histone deacetylases, HDACs) acetylation sites are an area of extensive research in current drug development, but very few potent inhibitors that modulate the 'reading process' mediated by acetyl lysines have been described.  The principal readers of ε-N-acetyl lysine (Kac) marks are bromodomains (BRDs), which are a diverse family of evolutionary conserved protein-interaction modules.  The conserved BRD fold contains a deep, largely hydrophobic acetyl lysine binding site, which represents an attractive pocket for the development of small, pharmaceutically active mols.  Proteins that contain BRDs have been implicated in the development of a large variety of diseases.  Recently, two highly potent and selective inhibitors that target BRDs of the BET (bromodomains and extra-terminal) family provided compelling data supporting targeting of these BRDs in inflammation and in an aggressive type of squamous cell carcinoma.  It is likely that BRDs will emerge alongside HATs and HDACs as interesting targets for drug development for the large no. of diseases that are caused by aberrant acetylation of lysine residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSuKacpsr6xLVg90H21EOLACvtfcHk0lijBaHORspHoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1OhurjI&md5=dc9383afcbe6aedc265041c21213aa0b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1017%2FS1462399411001992&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS1462399411001992%26sid%3Dliteratum%253Aachs%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DBromodomains%2520as%2520therapeutic%2520targets%26jtitle%3DExpert%2520Rev.%2520Mol.%2520Med.%26date%3D2011%26volume%3D13%26doi%3D10.1017%2Fs1462399411001992" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">356</span>, <span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0lijBaHORspHoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0ljJ45AKbnczhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moustakim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, P. G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Chemical probes and inhibitors of bromodomains outside the BET family</span>. <i>Medchemcomm</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">2246</span>– <span class="NLM_lpage">2264</span>, <span class="refDoi"> DOI: 10.1039/c6md00373g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1039%2Fc6md00373g" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=29170712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVOhsbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=2246-2264&author=M.+Moustakimauthor=P.+G.+K.+Clarkauthor=D.+A.+Hayauthor=D.+J.+Dixonauthor=P.+E.+Brennan&title=Chemical+probes+and+inhibitors+of+bromodomains+outside+the+BET+family&doi=10.1039%2Fc6md00373g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical probes and inhibitors of bromodomains outside the BET family</span></div><div class="casAuthors">Moustakim, Moses; Clark, Peter G. K.; Hay, Duncan A.; Dixon, Darren J.; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2246-2264</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">In the last five years, the development of inhibitors of bromodomains has emerged as an area of intensive worldwide research.  Emerging evidence has implicated a no. of non-BET bromodomains in the onset and progression of diseases such as cancer, HIV infection and inflammation.  The development and use of small mol. chem. probes has been fundamental to pre-clin. evaluation of bromodomains as targets.  Recent efforts are described highlighting the development of potent, selective and cell active non-BET bromodomain inhibitors and their therapeutic potential.  Over half of typical bromodomains now have reported ligands, but those with atypical binding site residues remain resistant to chem. probe discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYJvhj3Z0Ry7Vg90H21EOLACvtfcHk0ljJ45AKbnczhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVOhsbbF&md5=df4c5718b15ff7d401a0cd9974abc236</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1039%2Fc6md00373g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc6md00373g%26sid%3Dliteratum%253Aachs%26aulast%3DMoustakim%26aufirst%3DM.%26aulast%3DClark%26aufirst%3DP.%2BG.%2BK.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DDixon%26aufirst%3DD.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DChemical%2520probes%2520and%2520inhibitors%2520of%2520bromodomains%2520outside%2520the%2520BET%2520family%26jtitle%3DMedchemcomm%26date%3D2016%26volume%3D7%26spage%3D2246%26epage%3D2264%26doi%3D10.1039%2Fc6md00373g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">Discovery of chemical inhibitors of human bromodomains</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">11625</span>– <span class="NLM_lpage">11668</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00205</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00205" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs12qurzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2015&pages=11625-11668&author=G.+Zhangauthor=S.+G.+Smithauthor=M.-M.+Zhou&title=Discovery+of+chemical+inhibitors+of+human+bromodomains&doi=10.1021%2Facs.chemrev.5b00205"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Chemical Inhibitors of Human Bromodomains</span></div><div class="casAuthors">Zhang, Guangtao; Smith, Steven G.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11625-11668</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The structure-guided drug design strategy has played an important role in the recent discovery and development of bromodomain inhibitors that has yielded a multitude of chemotypes as described in this review.  Going forward, we expect that this structure-based approach will continue to guide medicinal chemists in lead optimization of bromodomain inhibitors and facilitate the efforts to develop new inhibitors for less druggable bromodomains.  Collectively, these emerging new chem. inhibitors will empower our ongoing investigation to better understand the roles of bromodomain proteins in control of gene transcription in human biol. and disease pathol.  These small-mol. inhibitors can also be developed into new epigenetic therapies for new effective treatments of various human diseases including cancer and inflammatory disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHbsgfBguOabVg90H21EOLACvtfcHk0ljJ45AKbnczhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs12qurzP&md5=4014c771384b4a768a80644af22c4485</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00205%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DS.%2BG.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DDiscovery%2520of%2520chemical%2520inhibitors%2520of%2520human%2520bromodomains%26jtitle%3DChem.%2520Rev.%26date%3D2015%26volume%3D115%26spage%3D11625%26epage%3D11668%26doi%3D10.1021%2Facs.chemrev.5b00205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Euskirchen, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auerbach, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gianoulis, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozowsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhardwaj, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstein, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, M.</span></span> <span> </span><span class="NLM_article-title">Diverse roles and interactions of the SWI/SNF chromatin remodeling complex revealed using global approaches</span>. <i>PLoS Genet.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e1002008</span> <span class="refDoi"> DOI: 10.1371/journal.pgen.1002008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1371%2Fjournal.pgen.1002008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=21408204" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjt1Kiuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&author=G.+M.+Euskirchenauthor=R.+K.+Auerbachauthor=E.+Davidovauthor=T.+A.+Gianoulisauthor=G.+Zhongauthor=J.+Rozowskyauthor=N.+Bhardwajauthor=M.+B.+Gersteinauthor=M.+Snyder&title=Diverse+roles+and+interactions+of+the+SWI%2FSNF+chromatin+remodeling+complex+revealed+using+global+approaches&doi=10.1371%2Fjournal.pgen.1002008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Diverse roles and interactions of the SWI/SNF chromatin remodeling complex revealed using global approaches</span></div><div class="casAuthors">Euskirchen, Ghia M.; Auerbach, Raymond K.; Davidov, Eugene; Gianoulis, Tara A.; Zhong, Guoneng; Rozowsky, Joel; Bhardwaj, Nitin; Gerstein, Mark B.; Snyder, Michael</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Genetics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e1002008</span>CODEN:
                <span class="NLM_cas:coden">PGLEB5</span>;
        ISSN:<span class="NLM_cas:issn">1553-7404</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">A systems understanding of nuclear organization and events is crit. for detg. how cells divide, differentiate, and respond to stimuli and for identifying the causes of diseases.  Chromatin remodeling complexes such as SWI/SNF have been implicated in a wide variety of cellular processes including gene expression, nuclear organization, centromere function, and chromosomal stability, and mutations in SWI/SNF components have been linked to several types of cancer.  To better understand the biol. processes in which chromatin remodeling proteins participate, we globally mapped binding regions for several components of the SWI/SNF complex throughout the human genome using ChIP-Seq.  SWI/SNF components were found to lie near regulatory elements integral to transcription (e.g. 5' ends, RNA Polymerases II and III, and enhancers) as well as regions crit. for chromosome organization (e.g. CTCF, lamins, and DNA replication origins).  Interestingly we also find that certain configurations of SWI/SNF subunits are assocd. with transcripts that have higher levels of expression, whereas other configurations of SWI/SNF factors are assocd. with transcripts that have lower levels of expression.  To further elucidate the assocn. of SWI/SNF subunits with each other as well as with other nuclear proteins, we also analyzed SWI/SNF immunopptd. complexes by mass spectrometry.  Individual SWI/SNF factors are assocd. with their own family members, as well as with cellular constituents such as nuclear matrix proteins, key transcription factors, and centromere components, implying a ubiquitous role in gene regulation and nuclear function.  We find an overrepresentation of both SWI/SNF-assocd. regions and proteins in cell cycle and chromosome organization.  Taken together the results from our ChIP and immunopptn. expts. suggest that SWI/SNF facilitates gene regulation and genome function more broadly and through a greater diversity of interactions than previously appreciated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhpZiFTTXYj7Vg90H21EOLACvtfcHk0livw4ZQ_zpNEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjt1Kiuro%253D&md5=cfd31a2824d8ca9180783224fca4b1e0</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pgen.1002008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pgen.1002008%26sid%3Dliteratum%253Aachs%26aulast%3DEuskirchen%26aufirst%3DG.%2BM.%26aulast%3DAuerbach%26aufirst%3DR.%2BK.%26aulast%3DDavidov%26aufirst%3DE.%26aulast%3DGianoulis%26aufirst%3DT.%2BA.%26aulast%3DZhong%26aufirst%3DG.%26aulast%3DRozowsky%26aufirst%3DJ.%26aulast%3DBhardwaj%26aufirst%3DN.%26aulast%3DGerstein%26aufirst%3DM.%2BB.%26aulast%3DSnyder%26aufirst%3DM.%26atitle%3DDiverse%2520roles%2520and%2520interactions%2520of%2520the%2520SWI%252FSNF%2520chromatin%2520remodeling%2520complex%2520revealed%2520using%2520global%2520approaches%26jtitle%3DPLoS%2520Genet.%26date%3D2011%26volume%3D7%26doi%3D10.1371%2Fjournal.pgen.1002008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kadoch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crabtree, G. R.</span></span> <span> </span><span class="NLM_article-title">Mammalian SWI/SNF chromatin remodeling complexes and cancer: Mechanistic insights gained from human genomics</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1</i></span>, <span class="NLM_elocation-id">e1500447</span> <span class="refDoi"> DOI: 10.1126/sciadv.1500447</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1126%2Fsciadv.1500447" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=26601204" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC28XnslOrsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&author=C.+Kadochauthor=G.+R.+Crabtree&title=Mammalian+SWI%2FSNF+chromatin+remodeling+complexes+and+cancer%3A+Mechanistic+insights+gained+from+human+genomics&doi=10.1126%2Fsciadv.1500447"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Mammalian SWI/SNF chromatin remodelling complexes and cancer: Mechanistic insights gained from human genomics</span></div><div class="casAuthors">Kadoch, Cigall; Crabtree, Gerald R.</div><div class="citationInfo"><span class="NLM_cas:title">Science Advances</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e1500447/1-e1500447/17</span>CODEN:
                <span class="NLM_cas:coden">SACDAF</span>;
        ISSN:<span class="NLM_cas:issn">2375-2548</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Over the past 4 years, nearly 100 exome sequencing studies have revealed the high frequency of mutations in the genes encoding the subunits of ATP-dependent chromatin remodelers in human cancer.  Most of these mutations are within the genes encoding subunits of the BAF (Brg/Brahma-assocd. factors) or mSWI/SNF complex, which is one of two dozen predicted ATP-dependent chromatin remodeling complexes in mammals.  Considering BAF complexes as a single entity, the 15 subunits encoded by 29 genes are mutated in >20% of human cancer, across a broad range of tumor types.  These observations demonstrate that there is little redundancy in the oncogenic function of BAF complexes with the other remodeling complexes, underscoring their unique roles.  Several important conclusionsemerge fromthese genomic data: specific subunits appear to bemutated in specific cancers, highlighting tissue-specific protective roles; mutations can function as tumor suppressors or oncogenes;mutations can be homozygous or, more commonly, heterozygous, implying their dosage-sensitive roles in an unknown yet fundamental process used to suppress the genesis of cancer.  These new human genetic findings paired with biochem. studies are challenging old ideas on how chromatin remodeling complexes function, generating new hypotheseswith respect to their normal and oncogenicmechanisms and highlighting potential avenues for therapeutic intervention in human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUweEWQp4d2LVg90H21EOLACvtfcHk0livw4ZQ_zpNEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnslOrsrY%253D&md5=e4f8ae87a6c179d61091582f03885f5b</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.1500447&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.1500447%26sid%3Dliteratum%253Aachs%26aulast%3DKadoch%26aufirst%3DC.%26aulast%3DCrabtree%26aufirst%3DG.%2BR.%26atitle%3DMammalian%2520SWI%252FSNF%2520chromatin%2520remodeling%2520complexes%2520and%2520cancer%253A%2520Mechanistic%2520insights%2520gained%2520from%2520human%2520genomics%26jtitle%3DSci.%2520Adv.%26date%3D2015%26volume%3D1%26doi%3D10.1126%2Fsciadv.1500447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alfert, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerl, K.</span></span> <span> </span><span class="NLM_article-title">The BAF complex in development and disease</span>. <i>Epigenet. Chromatin</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">19</span>, <span class="refDoi"> DOI: 10.1186/s13072-019-0264-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1186%2Fs13072-019-0264-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=30898143" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2019&pages=19&author=A.+Alfertauthor=N.+Morenoauthor=K.+Kerl&title=The+BAF+complex+in+development+and+disease&doi=10.1186%2Fs13072-019-0264-y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1186%2Fs13072-019-0264-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13072-019-0264-y%26sid%3Dliteratum%253Aachs%26aulast%3DAlfert%26aufirst%3DA.%26aulast%3DMoreno%26aufirst%3DN.%26aulast%3DKerl%26aufirst%3DK.%26atitle%3DThe%2520BAF%2520complex%2520in%2520development%2520and%2520disease%26jtitle%3DEpigenet.%2520Chromatin%26date%3D2019%26volume%3D12%26spage%3D19%26doi%3D10.1186%2Fs13072-019-0264-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hodges, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crabtree, G. R.</span></span> <span> </span><span class="NLM_article-title">The many roles of BAF (mSWI/SNF) and PBAF complexes in cancer</span>. <i>Cold Spring Harbor Perspect. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">a026930</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a026930</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1101%2Fcshperspect.a026930" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=27413115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1ygsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=a026930&author=C.+Hodgesauthor=J.+G.+Kirklandauthor=G.+R.+Crabtree&title=The+many+roles+of+BAF+%28mSWI%2FSNF%29+and+PBAF+complexes+in+cancer&doi=10.1101%2Fcshperspect.a026930"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The many roles of BAF (mSWI/SNF) and PBAF complexes in cancer</span></div><div class="casAuthors">Hodges, Courtney; Kirkland, Jacob G.; Crabtree, Gerald R.</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">a026930/1-a026930/25</span>CODEN:
                <span class="NLM_cas:coden">CSHPFV</span>;
        ISSN:<span class="NLM_cas:issn">2157-1422</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">During the last decade, a host of epigenetic mechanisms were found to contribute to cancer and other human diseases.  Several genomic studies have revealed that ∼ 20% of malignancies have alterations of the subunits of polymorphic BRG-/BRM-assocd. factor (BAF) and Polybromo-assocd. BAF (PBAF) complexes, making them among the most frequently mutated complexes in cancer.  Recurrent mutations arise in genes encoding several BAF/ PBAF subunits, including ARID1A, ARID2, PBRM1, SMARCA4, and SMARCB1.  These subunits share some degree of conservation with subunits from related ATP (ATP)-dependent chromatin remodeling complexes in model organisms, in which a large body of work provides insight into their roles in cancer.  Here, we review the roles of BAF- and PBAF-like complexes in these organisms, and relate these findings to recent discoveries in cancer epigenomics.  We review several roles of BAF and PBAF complexes in cancer, including transcriptional regulation, DNA repair, and regulation of chromatin architecture and topol.  More recent results highlight the need for new techniquesto study these complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCynky-OxCc7Vg90H21EOLACvtfcHk0livw4ZQ_zpNEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1ygsLs%253D&md5=71976f9f18bfad4c44fe26991679a194</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a026930&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a026930%26sid%3Dliteratum%253Aachs%26aulast%3DHodges%26aufirst%3DC.%26aulast%3DKirkland%26aufirst%3DJ.%2BG.%26aulast%3DCrabtree%26aufirst%3DG.%2BR.%26atitle%3DThe%2520many%2520roles%2520of%2520BAF%2520%2528mSWI%252FSNF%2529%2520and%2520PBAF%2520complexes%2520in%2520cancer%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Med.%26date%3D2016%26volume%3D6%26spage%3Da026930%26doi%3D10.1101%2Fcshperspect.a026930" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
La Serna, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbalzano, A. N.</span></span> <span> </span><span class="NLM_article-title">Mammalian SWI/SNF complexes promote MyoD-mediated muscle differentiation</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">187</span>– <span class="NLM_lpage">190</span>, <span class="refDoi"> DOI: 10.1038/84826</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2F84826" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=11175787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhtFGkur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2001&pages=187-190&author=I.+L.+de%0ALa+Sernaauthor=K.+A.+Carlsonauthor=A.+N.+Imbalzano&title=Mammalian+SWI%2FSNF+complexes+promote+MyoD-mediated+muscle+differentiation&doi=10.1038%2F84826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Mammalian SWI/SNF complexes promote MyoD-mediated muscle differentiation</span></div><div class="casAuthors">de la Serna, Ivana; Carlson, Kerri A.; Imbalzano, Anthony N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">187-190</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">Mammalian SWI/SNF complexes are ATP-dependent chromatin remodeling enzymes that have been implicated in the regulation of gene expression, cell-cycle control and oncogenesis.  MyoD is a muscle-specific regulator able to induce myogenesis in numerous cell types.  To ascertain the requirement for chromatin remodeling enzymes in cellular differentiation processes, we examd. MyoD-mediated induction of muscle differentiation in fibroblasts expressing dominant-neg. versions of the human brahma-related gene-1 (BRG1) or human brahma (BRM), the ATPase subunits of two distinct SWI/SNF enzymes.  We find that induction of the myogenic phenotype is completely abrogated in the presence of the mutant enzymes.  We further demonstrate that failure to induce muscle-specific gene expression correlates with inhibition of chromatin remodeling in the promoter region of an endogenous muscle-specific gene.  Our results demonstrate that SWI/SNF enzymes promote MyoD-mediated muscle differentiation and indicate that these enzymes function by altering chromatin structure in promoter regions of endogenous, differentiation-specific loci.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxtfpSZ1WGhLVg90H21EOLACvtfcHk0liUpKrhhh6GUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhtFGkur8%253D&md5=2d61d6ddae277103498e3e98ef563e7c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2F84826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F84826%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BLa%2BSerna%26aufirst%3DI.%2BL.%26aulast%3DCarlson%26aufirst%3DK.%2BA.%26aulast%3DImbalzano%26aufirst%3DA.%2BN.%26atitle%3DMammalian%2520SWI%252FSNF%2520complexes%2520promote%2520MyoD-mediated%2520muscle%2520differentiation%26jtitle%3DNat.%2520Genet.%26date%3D2001%26volume%3D27%26spage%3D187%26epage%3D190%26doi%3D10.1038%2F84826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lickert, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeuchi, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Both, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walls, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAuliffe, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee Adamson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mark Henkelman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrana, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossant, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruneau, B. G.</span></span> <span> </span><span class="NLM_article-title">Baf60c is essential for function of BAF chromatin remodelling complexes in heart development</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>432</i></span>,  <span class="NLM_fpage">107</span>– <span class="NLM_lpage">112</span>, <span class="refDoi"> DOI: 10.1038/nature03071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fnature03071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=15525990" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpt1eitLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=107-112&author=H.+Lickertauthor=J.+K.+Takeuchiauthor=I.+von+Bothauthor=J.+R.+Wallsauthor=F.+McAuliffeauthor=S.+Lee+Adamsonauthor=R.+Mark+Henkelmanauthor=J.+L.+Wranaauthor=J.+Rossantauthor=B.+G.+Bruneau&title=Baf60c+is+essential+for+function+of+BAF+chromatin+remodelling+complexes+in+heart+development&doi=10.1038%2Fnature03071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Baf60c is essential for function of BAF chromatin remodelling complexes in heart development</span></div><div class="casAuthors">Lickert, Heiko; Takeuchi, Jun K.; von Both, Ingo; Walls, Johnathon R.; McAuliffe, Fionnuala; Adamson, S. Lee; Henkelman, R. Mark; Wrana, Jeffrey L.; Rossant, Janet; Bruneau, Benoit G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7013</span>),
    <span class="NLM_cas:pages">107-112</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Tissue-specific transcription factors regulate several important aspects of embryonic development.  They must function in the context of DNA assembled into the higher-order structure of chromatin.  Enzymic complexes such as the Swi/Snf-like BAF complexes remodel chromatin to allow the transcriptional machinery access to gene regulatory elements.  Here the authors show that Smarcd3, encoding Baf60c, a subunit of the BAF complexes, is expressed specifically in the heart and somites in the early mouse embryo.  Smarcd3 silencing by RNA interference in mouse embryos derived from embryonic stem cells causes defects in heart morphogenesis that reflect impaired expansion of the anterior/secondary heart field, and also results in abnormal cardiac and skeletal muscle differentiation.  An intermediate redn. in Smarcd3 expression leads to defects in outflow tract remodelling reminiscent of human congenital heart defects.  Baf60c overexpressed in cell culture can mediate interactions between cardiac transcription factors and the BAF complex ATPase Brg1, thereby potentiating the activation of target genes.  These results reveal tissue-specific and dose-dependent roles for Baf60c in recruiting BAF chromatin remodelling complexes to heart-specific enhancers, providing a novel mechanism to ensure transcriptional regulation during organogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9V5xD2mMuhLVg90H21EOLACvtfcHk0liUpKrhhh6GUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpt1eitLg%253D&md5=63dcaa76fa4c7c4671860bd8f8a2b6e7</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnature03071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03071%26sid%3Dliteratum%253Aachs%26aulast%3DLickert%26aufirst%3DH.%26aulast%3DTakeuchi%26aufirst%3DJ.%2BK.%26aulast%3Dvon%2BBoth%26aufirst%3DI.%26aulast%3DWalls%26aufirst%3DJ.%2BR.%26aulast%3DMcAuliffe%26aufirst%3DF.%26aulast%3DLee%2BAdamson%26aufirst%3DS.%26aulast%3DMark%2BHenkelman%26aufirst%3DR.%26aulast%3DWrana%26aufirst%3DJ.%2BL.%26aulast%3DRossant%26aufirst%3DJ.%26aulast%3DBruneau%26aufirst%3DB.%2BG.%26atitle%3DBaf60c%2520is%2520essential%2520for%2520function%2520of%2520BAF%2520chromatin%2520remodelling%2520complexes%2520in%2520heart%2520development%26jtitle%3DNature%26date%3D2004%26volume%3D432%26spage%3D107%26epage%3D112%26doi%3D10.1038%2Fnature03071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bracken, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brien, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verrijzer, C. P.</span></span> <span> </span><span class="NLM_article-title">Dangerous liaisons: interplay between SWI/SNF, NuRD, and polycomb in chromatin regulation and cancer</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">936</span>– <span class="NLM_lpage">959</span>, <span class="refDoi"> DOI: 10.1101/gad.326066.119</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1101%2Fgad.326066.119" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=31123059" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslKmtbrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2019&pages=936-959&author=A.+P.+Brackenauthor=G.+L.+Brienauthor=C.+P.+Verrijzer&title=Dangerous+liaisons%3A+interplay+between+SWI%2FSNF%2C+NuRD%2C+and+polycomb+in+chromatin+regulation+and+cancer&doi=10.1101%2Fgad.326066.119"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Dangerous liaisons: interplay between SWI/SNF, NuRD, and Polycomb in chromatin regulation and cancer</span></div><div class="casAuthors">Bracken, Adrian P.; Brien, Gerard L.; Verrijzer, C. Peter</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">15-16</span>),
    <span class="NLM_cas:pages">936-959</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Changes in chromatin structure mediated by ATP-dependent nucleosome remodelers and histone modifying enzymes are integral to the process of gene regulation.  Here, we review the roles of the SWI/SNF (switch/sucrose nonfermenting) and NuRD (nucleosome remodeling and deacetylase) and the Polycomb system in chromatin regulation and cancer.  First, we discuss the basic mol. mechanism of nucleosome remodeling, and how this controls gene transcription.  Next, we provide an overview of the functional organization and biochem. activities of SWI/SNF, NuRD, and Polycomb complexes.  We describe how, in metazoans, the balance of these activities is central to the proper regulation of gene expression and cellular identity during development.  Whereas SWI/SNF counteracts Polycomb, NuRD facilitates Polycomb repression on chromatin.  Finally, we discuss how disruptions of this regulatory equil. contribute to oncogenesis, and how new insights into the biol. functions of remodelers and Polycombs are opening avenues for therapeutic interventions on a broad range of cancer types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4BhoBUmy0wbVg90H21EOLACvtfcHk0liUpKrhhh6GUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslKmtbrJ&md5=2ac67cb91973994d332837415c084306</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1101%2Fgad.326066.119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.326066.119%26sid%3Dliteratum%253Aachs%26aulast%3DBracken%26aufirst%3DA.%2BP.%26aulast%3DBrien%26aufirst%3DG.%2BL.%26aulast%3DVerrijzer%26aufirst%3DC.%2BP.%26atitle%3DDangerous%2520liaisons%253A%2520interplay%2520between%2520SWI%252FSNF%252C%2520NuRD%252C%2520and%2520polycomb%2520in%2520chromatin%2520regulation%2520and%2520cancer%26jtitle%3DGenes%2520Dev.%26date%3D2019%26volume%3D33%26spage%3D936%26epage%3D959%26doi%3D10.1101%2Fgad.326066.119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Varela, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarpey, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raine, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ong, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teague, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bignell, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalgliesh, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galappaththige, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latimer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLaren, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menzies, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mudie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stebbings, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Largaespada, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wessels, L. F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahnoski, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anema, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">A.Tuveson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Mancera, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustonen, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rust, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan-on, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subimerb, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dykema, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furge, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teh, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stratton, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Futreal, P. A.</span></span> <span> </span><span class="NLM_article-title">Exome sequencing identifies frequent mutation of the SWI/SNF complex gene PBRM1 in renal carcinoma</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>469</i></span>,  <span class="NLM_fpage">539</span>– <span class="NLM_lpage">542</span>, <span class="refDoi"> DOI: 10.1038/nature09639</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fnature09639" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=21248752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnsFaquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=469&publication_year=2011&pages=539-542&author=I.+Varelaauthor=P.+Tarpeyauthor=K.+Raineauthor=D.+Huangauthor=C.+K.+Ongauthor=P.+Stephensauthor=H.+Daviesauthor=D.+Jonesauthor=M.-L.+Linauthor=J.+Teagueauthor=G.+Bignellauthor=A.+Butlerauthor=J.+Choauthor=G.+L.+Dalglieshauthor=D.+Galappaththigeauthor=C.+Greenmanauthor=C.+Hardyauthor=M.+Jiaauthor=C.+Latimerauthor=K.+W.+Lauauthor=J.+Marshallauthor=S.+McLarenauthor=A.+Menziesauthor=L.+Mudieauthor=L.+Stebbingsauthor=D.+A.+Largaespadaauthor=L.+F.+A.+Wesselsauthor=S.+Richardauthor=R.+J.+Kahnoskiauthor=J.+Anemaauthor=D.+A.Tuvesonauthor=P.+A.+Perez-Manceraauthor=V.+Mustonenauthor=A.+Fischerauthor=D.+J.+Adamsauthor=A.+Rustauthor=W.+Chan-onauthor=C.+Subimerbauthor=K.+Dykemaauthor=K.+Furgeauthor=P.+J.+Campbellauthor=B.+T.+Tehauthor=M.+R.+Strattonauthor=P.+A.+Futreal&title=Exome+sequencing+identifies+frequent+mutation+of+the+SWI%2FSNF+complex+gene+PBRM1+in+renal+carcinoma&doi=10.1038%2Fnature09639"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Exome sequencing identifies frequent mutation of the SWI/SNF complex gene PBRM1 in renal carcinoma</span></div><div class="casAuthors">Varela, Ignacio; Tarpey, Patrick; Raine, Keiran; Huang, Dachuan; Ong, Choon Kiat; Stephens, Philip; Davies, Helen; Jones, David; Lin, Meng-Lay; Teague, Jon; Bignell, Graham; Butler, Adam; Cho, Juok; Dalgliesh, Gillian L.; Galappaththige, Danushka; Greenman, Chris; Hardy, Claire; Jia, Mingming; Latimer, Calli; Lau, King Wai; Marshall, John; McLaren, Stuart; Menzies, Andrew; Mudie, Laura; Stebbings, Lucy; Largaespada, David A.; Wessels, L. F. A.; Richard, Stephane; Kahnoski, Richard J.; Anema, John; Tuveson, David A.; Perez-Mancera, Pedro A.; Mustonen, Ville; Fischer, Andrej; Adams, David J.; Rust, Alistair; Chan-on, Waraporn; Subimerb, Chutima; Dykema, Karl; Furge, Kyle; Campbell, Peter J.; Teh, Bin Tean; Stratton, Michael R.; Futreal, P. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">469</span>
        (<span class="NLM_cas:issue">7331</span>),
    <span class="NLM_cas:pages">539-542</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The genetics of renal cancer is dominated by inactivation of the VHL tumor suppressor gene in clear cell carcinoma (ccRCC), the commonest histol. subtype.  A recent large-scale screen of ∼3500 genes by PCR-based exon re-sequencing identified several new cancer genes in ccRCC including UTX (also known as KDM6A), JARID1C (also known as KDM5C) and SETD2 (ref. 2).  These genes encode enzymes that demethylate (UTX, JARID1C) or methylate (SETD2) key lysine residues of histone H3.  Modification of the methylation state of these lysine residues of histone H3 regulates chromatin structure and is implicated in transcriptional control.  However, together these mutations are present in fewer than 15% of ccRCC, suggesting the existence of addnl., currently unidentified cancer genes.  Here, we have sequenced the protein coding exome in a series of primary ccRCC and report the identification of the SWI/SNF chromatin remodeling complex gene PBRM1 (ref. 4) as a second major ccRCC cancer gene, with truncating mutations in 41% (92/227) of cases.  These data further elucidate the somatic genetic architecture of ccRCC and emphasize the marked contribution of aberrant chromatin biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKCmhiZqUc6LVg90H21EOLACvtfcHk0lgZSKZVT__o_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnsFaquw%253D%253D&md5=24c3de69a0b7bc7f90c1f14e7732e4c4</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnature09639&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09639%26sid%3Dliteratum%253Aachs%26aulast%3DVarela%26aufirst%3DI.%26aulast%3DTarpey%26aufirst%3DP.%26aulast%3DRaine%26aufirst%3DK.%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DOng%26aufirst%3DC.%2BK.%26aulast%3DStephens%26aufirst%3DP.%26aulast%3DDavies%26aufirst%3DH.%26aulast%3DJones%26aufirst%3DD.%26aulast%3DLin%26aufirst%3DM.-L.%26aulast%3DTeague%26aufirst%3DJ.%26aulast%3DBignell%26aufirst%3DG.%26aulast%3DButler%26aufirst%3DA.%26aulast%3DCho%26aufirst%3DJ.%26aulast%3DDalgliesh%26aufirst%3DG.%2BL.%26aulast%3DGalappaththige%26aufirst%3DD.%26aulast%3DGreenman%26aufirst%3DC.%26aulast%3DHardy%26aufirst%3DC.%26aulast%3DJia%26aufirst%3DM.%26aulast%3DLatimer%26aufirst%3DC.%26aulast%3DLau%26aufirst%3DK.%2BW.%26aulast%3DMarshall%26aufirst%3DJ.%26aulast%3DMcLaren%26aufirst%3DS.%26aulast%3DMenzies%26aufirst%3DA.%26aulast%3DMudie%26aufirst%3DL.%26aulast%3DStebbings%26aufirst%3DL.%26aulast%3DLargaespada%26aufirst%3DD.%2BA.%26aulast%3DWessels%26aufirst%3DL.%2BF.%2BA.%26aulast%3DRichard%26aufirst%3DS.%26aulast%3DKahnoski%26aufirst%3DR.%2BJ.%26aulast%3DAnema%26aufirst%3DJ.%26aulast%3DA.Tuveson%26aufirst%3DD.%26aulast%3DPerez-Mancera%26aufirst%3DP.%2BA.%26aulast%3DMustonen%26aufirst%3DV.%26aulast%3DFischer%26aufirst%3DA.%26aulast%3DAdams%26aufirst%3DD.%2BJ.%26aulast%3DRust%26aufirst%3DA.%26aulast%3DChan-on%26aufirst%3DW.%26aulast%3DSubimerb%26aufirst%3DC.%26aulast%3DDykema%26aufirst%3DK.%26aulast%3DFurge%26aufirst%3DK.%26aulast%3DCampbell%26aufirst%3DP.%2BJ.%26aulast%3DTeh%26aufirst%3DB.%2BT.%26aulast%3DStratton%26aufirst%3DM.%2BR.%26aulast%3DFutreal%26aufirst%3DP.%2BA.%26atitle%3DExome%2520sequencing%2520identifies%2520frequent%2520mutation%2520of%2520the%2520SWI%252FSNF%2520complex%2520gene%2520PBRM1%2520in%2520renal%2520carcinoma%26jtitle%3DNature%26date%3D2011%26volume%3D469%26spage%3D539%26epage%3D542%26doi%3D10.1038%2Fnature09639" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Porter, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dykhuizen, E. C.</span></span> <span> </span><span class="NLM_article-title">Individual bromodomains of polybromo-1 contribute to chromatin association and tumor suppression in clear cell renal carcinoma</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">2601</span>– <span class="NLM_lpage">2610</span>, <span class="refDoi"> DOI: 10.1074/jbc.m116.746875</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1074%2Fjbc.M116.746875" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=28053089" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlyht73J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=2601-2610&author=E.+G.+Porterauthor=E.+C.+Dykhuizen&title=Individual+bromodomains+of+polybromo-1+contribute+to+chromatin+association+and+tumor+suppression+in+clear+cell+renal+carcinoma&doi=10.1074%2Fjbc.m116.746875"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Individual bromodomains of polybromo-1 contribute to chromatin association and tumor suppression in clear cell renal carcinoma</span></div><div class="casAuthors">Porter, Elizabeth G.; Dykhuizen, Emily C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2601-2610</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The architecture of chromatin is governed, in part, by ATP-dependent chromatin remodelers.  These multiprotein complexes contain targeting domains that recognize post-translational marks on histones.  One such targeting domain is the bromodomain (BD), which recognizes acetyl-lysines and recruits proteins to sites of acetylation across the genome.  Polybromo1 (PBRM1), a subunit of the Polybromo-assocd. BRG1- or hBRM-assocd. factors (PBAF) chromatin remodeler, contains six tandem BDs and is frequently mutated in clear cell renal cell carcinoma (ccRCC).  Mutations in the PBRM1 gene often lead to the loss of protein expression; however, missense mutations in PBRM1 have been identified and tend to cluster in the BDs, particularly BD2 and BD4, suggesting that individual BDs are crit. for PBRM1 function.  To study the role of these six BDs, we inactivated each of the six BDs of PBRM1 and re-expressed these mutants in Caki2 cells (ccRCC cells with the loss of function mutation in PBRM1).  Four of the six BDs abrogated PBRM1 tumor suppressor function, gene regulation, and chromatin affinity with the degree of importance correlating strongly to the rate of missense mutations in patients.  Furthermore, we identified BD2 as the most crit. for PBRM1 and confirmed BD2-mediated assocn. to histone H3 peptides acetylated at lysine 14 (H3K14Ac), validating the importance of this specific acetylation mark for PBRM1 binding.  From these data, we conclude that four of the BDs act together to target PBRM1 to sites on chromatin; when a single BD is mutated, PBRM1 no longer controls gene expression properly, leading to increased cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8VdeFhQpTwLVg90H21EOLACvtfcHk0ljyDA1CB_ewAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlyht73J&md5=8c74fe7140ae901f95c280e68c073f76</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.746875&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.746875%26sid%3Dliteratum%253Aachs%26aulast%3DPorter%26aufirst%3DE.%2BG.%26aulast%3DDykhuizen%26aufirst%3DE.%2BC.%26atitle%3DIndividual%2520bromodomains%2520of%2520polybromo-1%2520contribute%2520to%2520chromatin%2520association%2520and%2520tumor%2520suppression%2520in%2520clear%2520cell%2520renal%2520carcinoma%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D2601%26epage%3D2610%26doi%3D10.1074%2Fjbc.m116.746875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruthenburg, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David Allis, C.</span></span> <span> </span><span class="NLM_article-title">Multivalent engagement of chromatin modifications by linked binding modules</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">983</span>– <span class="NLM_lpage">994</span>, <span class="refDoi"> DOI: 10.1038/nrm2298</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fnrm2298" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=18037899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlCrsb3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=983-994&author=A.+J.+Ruthenburgauthor=H.+Liauthor=D.+J.+Patelauthor=C.+David+Allis&title=Multivalent+engagement+of+chromatin+modifications+by+linked+binding+modules&doi=10.1038%2Fnrm2298"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Multivalent engagement of chromatin modifications by linked binding modules</span></div><div class="casAuthors">Ruthenburg, Alexander J.; Li, Haitao; Patel, Dinshaw J.; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">983-994</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Histone post-translational modifications (PTMs) play crucial roles in genome management, in part by recruiting specific factors that alter the structural properties of chromatin.  These so-called effector complexes often comprise multiple histone-binding modules that may act in concert to regulate chromatin structure and DNA-related activities.  Various chem. modifications on histones and regions of assocd. DNA play crucial roles in genome management by binding specific factors that, in turn, serve to alter the structural properties of chromatin.  These so-called effector proteins have typically been studied with the biochemist's paring knife; the capacity to recognize specific chromatin PTMs has been mapped to an increasing no. of domains that frequently appear in the nuclear subset of the proteome, often present in large, multisubunit complexes that bristle with modification-dependent binding potential.  The authors propose that multivalent interactions on a single histone tail and beyond may have a significant, if not dominant, role in chromatin transactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkIrlnyvt64LVg90H21EOLACvtfcHk0ljyDA1CB_ewAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlCrsb3L&md5=0d27fc9ba2e2a009e0a9903e1c65bdf7</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnrm2298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm2298%26sid%3Dliteratum%253Aachs%26aulast%3DRuthenburg%26aufirst%3DA.%2BJ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DDavid%2BAllis%26aufirst%3DC.%26atitle%3DMultivalent%2520engagement%2520of%2520chromatin%2520modifications%2520by%2520linked%2520binding%2520modules%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2007%26volume%3D8%26spage%3D983%26epage%3D994%26doi%3D10.1038%2Fnrm2298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekaran, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, M.</span></span> <span> </span><span class="NLM_article-title">Polybromo-1-bromodomains bind histone H3 at specific acetyl-lysine positions</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">661</span>– <span class="NLM_lpage">666</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2007.01.193</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2Fj.bbrc.2007.01.193" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=17320048" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD2sXisVWjurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2007&pages=661-666&author=R.+Chandrasekaranauthor=M.+Thompson&title=Polybromo-1-bromodomains+bind+histone+H3+at+specific+acetyl-lysine+positions&doi=10.1016%2Fj.bbrc.2007.01.193"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Polybromo-1-bromodomains bind histone H3 at specific acetyl-lysine positions</span></div><div class="casAuthors">Chandrasekaran, Renu; Thompson, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">661-666</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The human polybromo-1 protein is thought to localize the Polybromo, BRG1-assocd. factors chromatin-remodeling complex to kinetochores during mitosis via direct interaction of its six tandem bromodomains with acetylated nucleosomes.  Bromodomains are acetyl-lysine binding modules roughly 100 amino acids in length originally found in chromatin assocd. proteins.  Previous studies verified acetyl-histone binding by each bromodomain, but site-specificity, a central tenet of the histone code hypothesis, was not examd.  Here, the acetylation site-dependence of bromodomain-histone interactions was examd. using steady-state fluorescence anisotropy.  Results indicate that single bromodomains bind specific acetyl-lysine sites within the histone tail with sub-micromolar affinity.  Identification of duplicate target sites suggests that native Pb1 interacts with both copies of histone H3 upon nucleosome assembly.  Quant. anal. of single bromodomain-histone interactions can be used to develop hypotheses regarding the histone acetylation pattern that acts as the binding target of the native polybromo-1 protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCjqNIdWtx37Vg90H21EOLACvtfcHk0ljyDA1CB_ewAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXisVWjurY%253D&md5=b344831ee2de7822c90937be0849a408</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2007.01.193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2007.01.193%26sid%3Dliteratum%253Aachs%26aulast%3DChandrasekaran%26aufirst%3DR.%26aulast%3DThompson%26aufirst%3DM.%26atitle%3DPolybromo-1-bromodomains%2520bind%2520histone%2520H3%2520at%2520specific%2520acetyl-lysine%2520positions%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2007%26volume%3D355%26spage%3D661%26epage%3D666%26doi%3D10.1016%2Fj.bbrc.2007.01.193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Charlop-Powers, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">Structural insights into selective histone H3 recognition by the human polybromo bromodomain 2</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">529</span>– <span class="NLM_lpage">538</span>, <span class="refDoi"> DOI: 10.1038/cr.2010.43</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fcr.2010.43" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=20368734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsFSqu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=529-538&author=Z.+Charlop-Powersauthor=L.+Zengauthor=Q.+Zhangauthor=M.-M.+Zhou&title=Structural+insights+into+selective+histone+H3+recognition+by+the+human+polybromo+bromodomain+2&doi=10.1038%2Fcr.2010.43"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into selective histone H3 recognition by the human Polybromo bromodomain 2</span></div><div class="casAuthors">Charlop-Powers, Zachary; Zeng, Lei; Zhang, Qiang; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">529-538</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The Polybromo (PB) protein functions as a key component of the human PBAF chromatin remodeling complex in regulation of gene transcription.  PB is made up of modular domains including six bromodomains that are known as acetyl-lysine binding domains.  However, histone-binding specificity of the bromodomains of PB has remained elusive.  In this study, we report biochem. characterization of all six PB bromodomains' binding to a suite of lysine-acetylated peptides derived from known acetylation sites on human core histones.  We demonstrate that bromodomain 2 of PB preferentially recognizes acetylated lysine 14 of histone H3 (H3K14ac), a post-translational mark known for gene transcriptional activation.  We further describe the mol. basis of the selective H3K14ac recognition of bromodomain 2 by solving the protein structures in both the free and bound forms using X-ray crystallog. and NMR, resp.  Cell Research (2010) 20:529-538. doi:10.1038/cr.2010.43; published online 6 Apr. 2010.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq2k1NkpJHTrVg90H21EOLACvtfcHk0lgysJ8Af283uQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsFSqu7w%253D&md5=1895bb2b0e8aafbef170dec26226994f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fcr.2010.43&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2010.43%26sid%3Dliteratum%253Aachs%26aulast%3DCharlop-Powers%26aufirst%3DZ.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DStructural%2520insights%2520into%2520selective%2520histone%2520H3%2520recognition%2520by%2520the%2520human%2520polybromo%2520bromodomain%25202%26jtitle%3DCell%2520Res.%26date%3D2010%26volume%3D20%26spage%3D529%26epage%3D538%26doi%3D10.1038%2Fcr.2010.43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Slaughter, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanle, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McFadden, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollis, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suttle, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strahl, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. J.</span></span> <span> </span><span class="NLM_article-title">PBRM1 bromodomains variably influence nucleosome interactions and cellular function</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">13592</span>– <span class="NLM_lpage">13603</span>, <span class="refDoi"> DOI: 10.1074/jbc.ra118.003381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1074%2Fjbc.RA118.003381" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=29986887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1altr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=13592-13603&author=M.+J.+Slaughterauthor=E.+K.+Shanleauthor=A.+W.+McFaddenauthor=E.+S.+Hollisauthor=L.+E.+Suttleauthor=B.+D.+Strahlauthor=I.+J.+Davis&title=PBRM1+bromodomains+variably+influence+nucleosome+interactions+and+cellular+function&doi=10.1074%2Fjbc.ra118.003381"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">PBRM1 bromodomains variably influence nucleosome interactions and cellular function</span></div><div class="casAuthors">Slaughter, Mariesa J.; Shanle, Erin K.; McFadden, Andrew W.; Hollis, Emily S.; Suttle, Lindsey E.; Strahl, Brian D.; Davis, Ian J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">13592-13603</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Chromatin remodelers use bromodomains (BDs) to recognize histones.  Polybromo 1 (PBRM1 or BAF180) is hypothesized to function as the nucleosome-recognition subunit of the PBAF chromatin-remodeling complex and is frequently mutated in clear cell renal cell carcinoma (ccRCC).  Previous studies have applied in vitro methods to explore the binding specificities of the six individual PBRM1 BDs.  However, BD targeting to histones and the influence of neighboring BD on nucleosome recognition have not been well characterized.  Here, using histone microarrays and intact nucleosomes to investigate the histone-binding characteristics of the six PBRM1 BDs individually and combined, we demonstrate that BD2 and BD4 of PBRM1 mediate binding to acetylated histone peptides and to modified recombinant and cellular nucleosomes.  Moreover, we show that neighboring BDs variably modulate these chromatin interactions, with BD1 and BD5 enhancing nucleosome interactions of BD2 and BD4, resp., whereas BD3 attenuated these interactions.  We also found that binding pocket missense mutations in BD4 obsd. in ccRCC disrupt PBRM1-chromatin interactions and that these mutations in BD4, but not similar mutations in BD2, in the context of full-length PBRM1, accelerate ccRCC cell proliferation.  Taken together, our biochem. and mutational analyses have identified BD4 as being critically important for maintaining proper PBRM1 function and demonstrate that BD4 mutations increase ccRCC cell growth.  Because of the link between PBRM1 status and sensitivity to immune checkpoint inhibitor treatment, these data also suggest the relevance of BD4 as a potential clin. target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOqqSdkL2uHLVg90H21EOLACvtfcHk0lgysJ8Af283uQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1altr3L&md5=07a66a8302d15e1c2eebb405e69dfc1f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.RA118.003381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.RA118.003381%26sid%3Dliteratum%253Aachs%26aulast%3DSlaughter%26aufirst%3DM.%2BJ.%26aulast%3DShanle%26aufirst%3DE.%2BK.%26aulast%3DMcFadden%26aufirst%3DA.%2BW.%26aulast%3DHollis%26aufirst%3DE.%2BS.%26aulast%3DSuttle%26aufirst%3DL.%2BE.%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DDavis%26aufirst%3DI.%2BJ.%26atitle%3DPBRM1%2520bromodomains%2520variably%2520influence%2520nucleosome%2520interactions%2520and%2520cellular%2520function%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26spage%3D13592%26epage%3D13603%26doi%3D10.1074%2Fjbc.ra118.003381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alicea-Velázquez, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langbein, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greer, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosgrove, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span> <span> </span><span class="NLM_article-title">High affinity binding of H3K14ac through collaboration of bromodomains 2, 4 and 5 is critical for the molecular and tumor suppressor functions of PBRM1</span>. <i>Mol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">811</span>– <span class="NLM_lpage">828</span>, <span class="refDoi"> DOI: 10.1002/1878-0261.12434</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1002%2F1878-0261.12434" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=30585695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFaqs7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2019&pages=811-828&author=L.+Liaoauthor=N.+L.+Alicea-Vel%C3%A1zquezauthor=L.+Langbeinauthor=X.+Niuauthor=W.+Caiauthor=E.+A.+Choauthor=M.+Zhangauthor=C.+B.+Greerauthor=Q.+Yanauthor=M.+S.+Cosgroveauthor=H.+Yang&title=High+affinity+binding+of+H3K14ac+through+collaboration+of+bromodomains+2%2C+4+and+5+is+critical+for+the+molecular+and+tumor+suppressor+functions+of+PBRM1&doi=10.1002%2F1878-0261.12434"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">High affinity binding of H3K14ac through collaboration of bromodomains 2, 4 and 5 is critical for the molecular and tumor suppressor functions of PBRM1</span></div><div class="casAuthors">Liao, Lili; Alicea-Velazquez, Nilda L.; Langbein, Lauren; Niu, Xiaohua; Cai, Weijia; Cho, Eun-Ah; Zhang, Meiling; Greer, Celeste B.; Yan, Qin; Cosgrove, Michael S.; Yang, Haifeng</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Oncology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">811-828</span>CODEN:
                <span class="NLM_cas:coden">MOONC3</span>;
        ISSN:<span class="NLM_cas:issn">1878-0261</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Polybromo-1 is an important tumor suppressor in kidney cancer.  While BD2 has been found to bind acetylated histone H3 lysine 14 (H3K14ac), and the importance of H3K14ac recognition for the mol.  We discovered that full-length PBRM1, but not its individual BDs, strongly binds H3K14ac.  BDs 2, 4, and 5 were found to collaborate to facilitate strong binding to H3K14ac.  Quant. measurement of the interactions between purified BD proteins and H3K14ac or nonacetylated peptides confirmed the tight and specific assocn. of the former.  Interestingly, while the structural integrity of BD4 was found to be required for H3K14ac recognition, the conserved acetyl-lysine binding site of BD4 was not.  Furthermore, simultaneous point mutations in BDs 2, 4, and 5 prevented recognition of H3K14ac, altered promoter binding and gene expression, and caused PBRM1 to relocalize to the cytoplasm.  In contrast, tumor-derived point mutations in BD2 alone lowered PBRM1's affinity to H3K14ac and also disrupted promoter binding and gene expression without altering cellular localization.  Finally, overexpression of PBRM1 variants contg. point mutations in BDs 2, 4, and 5 or BD2 alone failed to suppress tumor growth in a xenograft model.  Taken together, our study demonstrates that BDs 2,4, and 5 of PBRM1 collaborate to generate high affinity to H3K14ac and tether PBRM1 to chromatin.  Mutations in BD2 alone weaken these interactions, and this is sufficient to abolish its mol. and tumor suppressor functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWdNbAce1d3rVg90H21EOLACvtfcHk0lgysJ8Af283uQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFaqs7vI&md5=d6763d01ce78da05205564b73b5e6b75</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2F1878-0261.12434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1878-0261.12434%26sid%3Dliteratum%253Aachs%26aulast%3DLiao%26aufirst%3DL.%26aulast%3DAlicea-Vel%25C3%25A1zquez%26aufirst%3DN.%2BL.%26aulast%3DLangbein%26aufirst%3DL.%26aulast%3DNiu%26aufirst%3DX.%26aulast%3DCai%26aufirst%3DW.%26aulast%3DCho%26aufirst%3DE.%2BA.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DGreer%26aufirst%3DC.%2BB.%26aulast%3DYan%26aufirst%3DQ.%26aulast%3DCosgrove%26aufirst%3DM.%2BS.%26aulast%3DYang%26aufirst%3DH.%26atitle%3DHigh%2520affinity%2520binding%2520of%2520H3K14ac%2520through%2520collaboration%2520of%2520bromodomains%25202%252C%25204%2520and%25205%2520is%2520critical%2520for%2520the%2520molecular%2520and%2520tumor%2520suppressor%2520functions%2520of%2520PBRM1%26jtitle%3DMol.%2520Oncol.%26date%3D2019%26volume%3D13%26spage%3D811%26epage%3D828%26doi%3D10.1002%2F1878-0261.12434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kadoch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodges, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elias, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranish, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crabtree, G. R.</span></span> <span> </span><span class="NLM_article-title">Proteomic and bioinformatic analysis of mammalian SWI/SNF complexes identifies extensive roles in human malignancy</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">592</span>– <span class="NLM_lpage">601</span>, <span class="refDoi"> DOI: 10.1038/ng.2628</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fng.2628" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=23644491" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntVart7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2013&pages=592-601&author=C.+Kadochauthor=D.+C.+Hargreavesauthor=C.+Hodgesauthor=L.+Eliasauthor=L.+Hoauthor=J.+Ranishauthor=G.+R.+Crabtree&title=Proteomic+and+bioinformatic+analysis+of+mammalian+SWI%2FSNF+complexes+identifies+extensive+roles+in+human+malignancy&doi=10.1038%2Fng.2628"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic and bioinformatic analysis of mammalian SWI/SNF complexes identifies extensive roles in human malignancy</span></div><div class="casAuthors">Kadoch, Cigall; Hargreaves, Diana C.; Hodges, Courtney; Elias, Laura; Ho, Lena; Ranish, Jeff; Crabtree, Gerald R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">592-601</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Subunits of mammalian SWI/SNF (mSWI/SNF or BAF) complexes have recently been implicated as tumor suppressors in human malignancies.  To understand the full extent of their involvement, we conducted a proteomic anal. of endogenous mSWI/SNF complexes, which identified several new dedicated, stable subunits not found in yeast SWI/SNF complexes, including BCL7A, BCL7B and BCL7C, BCL11A and BCL11B, BRD9 and SS18.  Incorporating these new members, we detd. mSWI/SNF subunit mutation frequency in exome and whole-genome sequencing studies of primary human tumors.  Notably, mSWI/SNF subunits are mutated in 19.6% of all human tumors reported in 44 studies.  Our anal. suggests that specific subunits protect against cancer in specific tissues.  In addn., mutations affecting more than one subunit, defined here as compd. heterozygosity, are prevalent in certain cancers.  Our studies demonstrate that mSWI/SNF is the most frequently mutated chromatin-regulatory complex (CRC) in human cancer, exhibiting a broad mutation pattern, similar to that of TP53.  Thus, proper functioning of polymorphic BAF complexes may constitute a major mechanism of tumor suppression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP873DCpn7JrVg90H21EOLACvtfcHk0ljpi40laoUF2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntVart7s%253D&md5=0f49cf3c59e7bc04b74970fe656b1e17</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fng.2628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.2628%26sid%3Dliteratum%253Aachs%26aulast%3DKadoch%26aufirst%3DC.%26aulast%3DHargreaves%26aufirst%3DD.%2BC.%26aulast%3DHodges%26aufirst%3DC.%26aulast%3DElias%26aufirst%3DL.%26aulast%3DHo%26aufirst%3DL.%26aulast%3DRanish%26aufirst%3DJ.%26aulast%3DCrabtree%26aufirst%3DG.%2BR.%26atitle%3DProteomic%2520and%2520bioinformatic%2520analysis%2520of%2520mammalian%2520SWI%252FSNF%2520complexes%2520identifies%2520extensive%2520roles%2520in%2520human%2520malignancy%26jtitle%3DNat.%2520Genet.%26date%3D2013%26volume%3D45%26spage%3D592%26epage%3D601%26doi%3D10.1038%2Fng.2628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chabanon, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Postel-Vinay, S.</span></span> <span> </span><span class="NLM_article-title">Exploiting epigenetic vulnerabilities in solid tumors: Novel therapeutic opportunities in the treatment of SWI/SNF-defective cancers</span>. <i>Semin. Cancer Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">180</span>– <span class="NLM_lpage">198</span>, <span class="refDoi"> DOI: 10.1016/j.semcancer.2019.09.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2Fj.semcancer.2019.09.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=31568814" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFyntbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2020&pages=180-198&author=R.+M.+Chabanonauthor=D.+Morelauthor=S.+Postel-Vinay&title=Exploiting+epigenetic+vulnerabilities+in+solid+tumors%3A+Novel+therapeutic+opportunities+in+the+treatment+of+SWI%2FSNF-defective+cancers&doi=10.1016%2Fj.semcancer.2019.09.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting epigenetic vulnerabilities in solid tumors: Novel therapeutic opportunities in the treatment of SWI/SNF-defective cancers</span></div><div class="casAuthors">Chabanon, Roman M.; Morel, Daphne; Postel-Vinay, Sophie</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Cancer Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">180-198</span>CODEN:
                <span class="NLM_cas:coden">SECBE7</span>;
        ISSN:<span class="NLM_cas:issn">1044-579X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Mammalian switch/sucrose non-fermentable (mSWI/SNF) family complexes are pivotal elements of the chromatin remodeling machinery, which contribute to the regulation of several major cellular functions.  Large-scale exome-wide sequencing studies have identified mutations in genes encoding mSWI/SNF subunits in 20% of all human cancers, establishing mSWI/SNF deficiency as a recurrent oncogenic alteration.  Accumulating evidence now supports that several mSWI/SNF defects represent targetable vulnerabilities in cancer; notably, recent research advances have unveiled unexpected synthetic lethal opportunities that foster the development of novel biomarker-driven and mechanism-based therapeutic approaches for the treatment of mSWI/SNF-deficient tumors.  Here, we review the latest breakthroughs and discoveries that inform our understanding of the mSWI/SNF complexes biol. in carcinogenesis, and discuss the most promising therapeutic strategies to target mSWI/SNF defects in human solid malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa-p9-2CfaObVg90H21EOLACvtfcHk0ljpi40laoUF2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFyntbbJ&md5=0aec46e82354754eb4177fe5e8ca7330</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.semcancer.2019.09.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.semcancer.2019.09.018%26sid%3Dliteratum%253Aachs%26aulast%3DChabanon%26aufirst%3DR.%2BM.%26aulast%3DMorel%26aufirst%3DD.%26aulast%3DPostel-Vinay%26aufirst%3DS.%26atitle%3DExploiting%2520epigenetic%2520vulnerabilities%2520in%2520solid%2520tumors%253A%2520Novel%2520therapeutic%2520opportunities%2520in%2520the%2520treatment%2520of%2520SWI%252FSNF-defective%2520cancers%26jtitle%3DSemin.%2520Cancer%2520Biol.%26date%3D2020%26volume%3D61%26spage%3D180%26epage%3D198%26doi%3D10.1016%2Fj.semcancer.2019.09.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oike, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogiwara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tominaga, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ando, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuta, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizukami, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isomura, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komachi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furuta, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, S.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokota, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohno, T.</span></span> <span> </span><span class="NLM_article-title">A synthetic lethality-based strategy to treat cancers harboring a genetic deficiency in the chromatin remodeling factor BRG1</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">5508</span>– <span class="NLM_lpage">5518</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-12-4593</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1158%2F0008-5472.CAN-12-4593" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=23872584" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlKkurfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=5508-5518&author=T.+Oikeauthor=H.+Ogiwaraauthor=Y.+Tominagaauthor=K.+Itoauthor=O.+Andoauthor=K.+Tsutaauthor=T.+Mizukamiauthor=Y.+Shimadaauthor=H.+Isomuraauthor=M.+Komachiauthor=K.+Furutaauthor=S.-I.+Watanabeauthor=T.+Nakanoauthor=J.+Yokotaauthor=T.+Kohno&title=A+synthetic+lethality-based+strategy+to+treat+cancers+harboring+a+genetic+deficiency+in+the+chromatin+remodeling+factor+BRG1&doi=10.1158%2F0008-5472.can-12-4593"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A Synthetic Lethality-Based Strategy to Treat Cancers Harboring a Genetic Deficiency in the Chromatin Remodeling Factor BRG1</span></div><div class="casAuthors">Oike, Takahiro; Ogiwara, Hideaki; Tominaga, Yuichi; Ito, Kentaro; Ando, Osamu; Tsuta, Koji; Mizukami, Tatsuji; Shimada, Yoko; Isomura, Hisanori; Komachi, Mayumi; Furuta, Koh; Watanabe, Shun-ichi; Nakano, Takashi; Yokota, Jun; Kohno, Takashi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5508-5518</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The occurrence of inactivating mutations in SWI/SNF chromatin-remodeling genes in common cancers has attracted a great deal of interest.  However, mechanistic strategies to target tumor cells carrying such mutations are yet to be developed.  This study proposes a synthetic-lethality therapy for treating cancers deficient in the SWI/SNF catalytic (ATPase) subunit, BRG1/SMARCA4.  The strategy relies upon inhibition of BRM/SMARCA2, another catalytic SWI/SNF subunit with a BRG1-related activity.  Immunohistochem. anal. of a cohort of non-small- cell lung carcinomas (NSCLC) indicated that 15.5% (16 of 103) of the cohort, corresponding to preferentially undifferentiated tumors, was deficient in BRG1 expression.  All BRG1-deficient cases were neg. for alterations in known therapeutic target genes, for example, EGFR and DDR2 gene mutations, ALK gene fusions, or FGFR1 gene amplifications.  RNA interference (RNAi)-mediated silencing of BRM suppressed the growth of BRG1-deficient cancer cells relative to BRG1-proficient cancer cells, inducing senescence via activation of p21/CDKN1A.  This growth suppression was reversed by transduction of wild-type but not ATPase-deficient BRG1.  In support of these in vitro results, a conditional RNAi study conducted in vivo revealed that BRM depletion suppressed the growth of BRG1-deficient tumor xenografts.  Our results offer a rationale to develop BRM-ATPase inhibitors as a strategy to treat BRG1/SMARCA4-deficient cancers, including NSCLCs that lack mutations in presently known therapeutic target genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcN-XkB72ouLVg90H21EOLACvtfcHk0ljpi40laoUF2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlKkurfO&md5=78fd175ce9dd7e2469cfe1ba4e6b49e1</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-12-4593&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-12-4593%26sid%3Dliteratum%253Aachs%26aulast%3DOike%26aufirst%3DT.%26aulast%3DOgiwara%26aufirst%3DH.%26aulast%3DTominaga%26aufirst%3DY.%26aulast%3DIto%26aufirst%3DK.%26aulast%3DAndo%26aufirst%3DO.%26aulast%3DTsuta%26aufirst%3DK.%26aulast%3DMizukami%26aufirst%3DT.%26aulast%3DShimada%26aufirst%3DY.%26aulast%3DIsomura%26aufirst%3DH.%26aulast%3DKomachi%26aufirst%3DM.%26aulast%3DFuruta%26aufirst%3DK.%26aulast%3DWatanabe%26aufirst%3DS.-I.%26aulast%3DNakano%26aufirst%3DT.%26aulast%3DYokota%26aufirst%3DJ.%26aulast%3DKohno%26aufirst%3DT.%26atitle%3DA%2520synthetic%2520lethality-based%2520strategy%2520to%2520treat%2520cancers%2520harboring%2520a%2520genetic%2520deficiency%2520in%2520the%2520chromatin%2520remodeling%2520factor%2520BRG1%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D5508%26epage%3D5518%26doi%3D10.1158%2F0008-5472.can-12-4593" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahal, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buxton, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frias, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagdasarian, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindeman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramadan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phadke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaskelioff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saenz-Vash, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myer, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLaughlin, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mickanin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, C. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmeier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagani, Z.</span></span> <span> </span><span class="NLM_article-title">Functional epigenetics approach identifies BRM/SMARCA2 as a critical synthetic lethal target in BRG1-deficient cancers</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">3128</span>– <span class="NLM_lpage">3133</span>, <span class="refDoi"> DOI: 10.1073/pnas.1316793111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1073%2Fpnas.1316793111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=24520176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtVKjurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=3128-3133&author=G.+R.+Hoffmanauthor=R.+Rahalauthor=F.+Buxtonauthor=K.+Xiangauthor=G.+McAllisterauthor=E.+Friasauthor=L.+Bagdasarianauthor=J.+Huberauthor=A.+Lindemanauthor=D.+Chenauthor=R.+Romeroauthor=N.+Ramadanauthor=T.+Phadkeauthor=K.+Haasauthor=M.+Jaskelioffauthor=B.+G.+Wilsonauthor=M.+J.+Meyerauthor=V.+Saenz-Vashauthor=H.+Zhaiauthor=V.+E.+Myerauthor=J.+A.+Porterauthor=N.+Keenauthor=M.+E.+McLaughlinauthor=C.+Mickaninauthor=C.+W.+M.+Robertsauthor=F.+Stegmeierauthor=Z.+Jagani&title=Functional+epigenetics+approach+identifies+BRM%2FSMARCA2+as+a+critical+synthetic+lethal+target+in+BRG1-deficient+cancers&doi=10.1073%2Fpnas.1316793111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Functional epigenetics approach identifies BRM/SMARCA2 as a critical synthetic lethal target in BRG1-deficient cancers</span></div><div class="casAuthors">Hoffman, Gregory R.; Rahal, Rami; Buxton, Frank; Xiang, Kay; McAllister, Gregory; Frias, Elizabeth; Bagdasarian, Linda; Huber, Janina; Lindeman, Alicia; Chen, Dongshu; Romero, Rodrigo; Ramadan, Nadire; Phadke, Tanushree; Haas, Kristy; Jaskelioff, Mariela; Wilson, Boris G.; Meyer, Matthew J.; Saenz-Vash, Veronica; Zhai, Huili; Myer, Vic E.; Porter, Jeffery A.; Keen, Nicholas; McLaughlin, Margaret E.; Mickanin, Craig; Roberts, Charles W. M.; Stegmeier, Frank; Jagani, Zainab</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3128-3133</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Defects in epigenetic regulation play a fundamental role in the development of cancer, and epigenetic regulators have recently emerged as promising therapeutic candidates.  We therefore set out to systematically interrogate epigenetic cancer dependencies by screening an epigenome-focused deep-coverage design shRNA (DECODER) library across 58 cancer cell lines.  This screen identified BRM/SMARCA2, a DNA-dependent ATPase of the mammalian SWI/SNF (mSWI/SNF) chromatin remodeling complex, as being essential for the growth of tumor cells that harbor loss of function mutations in BRG1/SMARCA4.  Depletion of BRM in BRG1-deficient cancer cells leads to a cell cycle arrest, induction of senescence, and increased levels of global H3K9me3.  We further demonstrate the selective dependency of BRG1-mutant tumors on BRM in vivo.  Genetic alterations of the mSWI/SNF chromatin remodeling complexes are the most frequent among chromatin regulators in cancers, with BRG1/SMARCA4 mutations occurring in ∼10-15% of lung adenocarcinomas.  Our findings position BRM as an attractive therapeutic target for BRG1 mutated cancers.  Because BRG1 and BRM function as mutually exclusive catalytic subunits of the mSWI/SNF complex, we propose that such synthetic lethality may be explained by paralog insufficiency, in which loss of one family member unveils crit. dependence on paralogous subunits.  This concept of "cancer-selective paralog dependency" may provide a more general strategy for targeting other tumor suppressor lesions/complexes with paralogous subunits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjF1zG7em3s7Vg90H21EOLACvtfcHk0lhF6Fgae2naEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtVKjurc%253D&md5=f9c36efd50b517ef389b39ca6a65307d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1316793111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1316793111%26sid%3Dliteratum%253Aachs%26aulast%3DHoffman%26aufirst%3DG.%2BR.%26aulast%3DRahal%26aufirst%3DR.%26aulast%3DBuxton%26aufirst%3DF.%26aulast%3DXiang%26aufirst%3DK.%26aulast%3DMcAllister%26aufirst%3DG.%26aulast%3DFrias%26aufirst%3DE.%26aulast%3DBagdasarian%26aufirst%3DL.%26aulast%3DHuber%26aufirst%3DJ.%26aulast%3DLindeman%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DRomero%26aufirst%3DR.%26aulast%3DRamadan%26aufirst%3DN.%26aulast%3DPhadke%26aufirst%3DT.%26aulast%3DHaas%26aufirst%3DK.%26aulast%3DJaskelioff%26aufirst%3DM.%26aulast%3DWilson%26aufirst%3DB.%2BG.%26aulast%3DMeyer%26aufirst%3DM.%2BJ.%26aulast%3DSaenz-Vash%26aufirst%3DV.%26aulast%3DZhai%26aufirst%3DH.%26aulast%3DMyer%26aufirst%3DV.%2BE.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DMcLaughlin%26aufirst%3DM.%2BE.%26aulast%3DMickanin%26aufirst%3DC.%26aulast%3DRoberts%26aufirst%3DC.%2BW.%2BM.%26aulast%3DStegmeier%26aufirst%3DF.%26aulast%3DJagani%26aufirst%3DZ.%26atitle%3DFunctional%2520epigenetics%2520approach%2520identifies%2520BRM%252FSMARCA2%2520as%2520a%2520critical%2520synthetic%2520lethal%2520target%2520in%2520BRG1-deficient%2520cancers%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2014%26volume%3D111%26spage%3D3128%26epage%3D3133%26doi%3D10.1073%2Fpnas.1316793111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vangamudi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paul, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kost-Alimova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nottebaum, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahadeshwar, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Protopopov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Futreal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tieu, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peoples, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heffernan, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marszalek, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toniatti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrocchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhelle, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Draetta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, J. N.</span></span> <span> </span><span class="NLM_article-title">The SMARCA2/4 ATPase domain surpasses the bromodomain as a drug target in SWI/SNF-mutant cancers: Insights from cDNA rescue and PFI-3 inhibitor studies</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">3865</span>– <span class="NLM_lpage">3878</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-14-3798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1158%2F0008-5472.CAN-14-3798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=26139243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsV2gtL7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2015&pages=3865-3878&author=B.+Vangamudiauthor=T.+A.+Paulauthor=P.+K.+Shahauthor=M.+Kost-Alimovaauthor=L.+Nottebaumauthor=X.+Shiauthor=Y.+Zhanauthor=E.+Leoauthor=H.+S.+Mahadeshwarauthor=A.+Protopopovauthor=A.+Futrealauthor=T.+N.+Tieuauthor=M.+Peoplesauthor=T.+P.+Heffernanauthor=J.+R.+Marszalekauthor=C.+Toniattiauthor=A.+Petrocchiauthor=D.+Verhelleauthor=D.+R.+Owenauthor=G.+Draettaauthor=P.+Jonesauthor=W.+S.+Palmerauthor=S.+Sharmaauthor=J.+N.+Andersen&title=The+SMARCA2%2F4+ATPase+domain+surpasses+the+bromodomain+as+a+drug+target+in+SWI%2FSNF-mutant+cancers%3A+Insights+from+cDNA+rescue+and+PFI-3+inhibitor+studies&doi=10.1158%2F0008-5472.can-14-3798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">The SMARCA2/4 ATPase Domain Surpasses the Bromodomain as a Drug Target in SWI/SNF-Mutant Cancers: Insights from cDNA Rescue and PFI-3 Inhibitor Studies</span></div><div class="casAuthors">Vangamudi, Bhavatarini; Paul, Thomas A.; Shah, Parantu K.; Kost-Alimova, Maria; Nottebaum, Lisa; Shi, Xi; Zhan, Yanai; Leo, Elisabetta; Mahadeshwar, Harshad S.; Protopopov, Alexei; Futreal, Andrew; Tieu, Trang N.; Peoples, Mike; Heffernan, Timothy P.; Marszalek, Joseph R.; Toniatti, Carlo; Petrocchi, Alessia; Verhelle, Dominique; Owen, Dafydd R.; Draetta, Giulio; Jones, Philip; Palmer, Wylie S.; Sharma, Shikhar; Andersen, Jannik N.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3865-3878</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The SWI/SNF multisubunit complex modulates chromatin structure through the activity of two mutually exclusive catalytic subunits, SMARCA2 and SMARCA4, which both contain a bromodomain and an ATPase domain.  Using RNAi, cancer-specific vulnerabilities have been identified in SWI/SNF-mutant tumors, including SMARCA4-deficient lung cancer; however, the contribution of conserved, druggable protein domains to this anticancer phenotype is unknown.  Here, we functionally deconstruct the SMARCA2/4 paralog dependence of cancer cells using bioinformatics, genetic, and pharmacol. tools.  We evaluate a selective SMARCA2/4 bromodomain inhibitor (PFI-3) and characterize its activity in chromatin-binding and cell-functional assays focusing on cells with altered SWI/SNF complex (e.g., lung, synovial sarcoma, leukemia, and rhabdoid tumors).  We demonstrate that PFI-3 is a potent, cell-permeable probe capable of displacing ectopically expressed, GFP-tagged SMARCA2-bromodomain from chromatin, yet contrary to target knockdown, the inhibitor fails to display an antiproliferative phenotype.  Mechanistically, the lack of pharmacol. efficacy is reconciled by the failure of bromodomain inhibition to displace endogenous, full-length SMARCA2 from chromatin as detd. by in situ cell extn., chromatin immunopptn., and target gene expression studies.  Furthermore, using inducible RNAi and cDNA complementation (bromodomain- and ATPase-dead constructs), we unequivocally identify the ATPase domain, and not the bromodomain of SMARCA2, as the relevant therapeutic target with the catalytic activity suppressing defined transcriptional programs.  Taken together, our complementary genetic and pharmacol. studies exemplify a general strategy for multidomain protein drug-target validation and in case of SMARCA2/4 highlight the potential for drugging the more challenging helicase/ATPase domain to deliver on the promise of synthetic-lethality therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrix1Qe07Ed3LVg90H21EOLACvtfcHk0lhF6Fgae2naEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsV2gtL7K&md5=bb35d59dabe56c253fef822a9fb337a8</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-14-3798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-14-3798%26sid%3Dliteratum%253Aachs%26aulast%3DVangamudi%26aufirst%3DB.%26aulast%3DPaul%26aufirst%3DT.%2BA.%26aulast%3DShah%26aufirst%3DP.%2BK.%26aulast%3DKost-Alimova%26aufirst%3DM.%26aulast%3DNottebaum%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DZhan%26aufirst%3DY.%26aulast%3DLeo%26aufirst%3DE.%26aulast%3DMahadeshwar%26aufirst%3DH.%2BS.%26aulast%3DProtopopov%26aufirst%3DA.%26aulast%3DFutreal%26aufirst%3DA.%26aulast%3DTieu%26aufirst%3DT.%2BN.%26aulast%3DPeoples%26aufirst%3DM.%26aulast%3DHeffernan%26aufirst%3DT.%2BP.%26aulast%3DMarszalek%26aufirst%3DJ.%2BR.%26aulast%3DToniatti%26aufirst%3DC.%26aulast%3DPetrocchi%26aufirst%3DA.%26aulast%3DVerhelle%26aufirst%3DD.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DDraetta%26aufirst%3DG.%26aulast%3DJones%26aufirst%3DP.%26aulast%3DPalmer%26aufirst%3DW.%2BS.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DAndersen%26aufirst%3DJ.%2BN.%26atitle%3DThe%2520SMARCA2%252F4%2520ATPase%2520domain%2520surpasses%2520the%2520bromodomain%2520as%2520a%2520drug%2520target%2520in%2520SWI%252FSNF-mutant%2520cancers%253A%2520Insights%2520from%2520cDNA%2520rescue%2520and%2520PFI-3%2520inhibitor%2520studies%26jtitle%3DCancer%2520Res.%26date%3D2015%26volume%3D75%26spage%3D3865%26epage%3D3878%26doi%3D10.1158%2F0008-5472.can-14-3798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schick, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rendeiro, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Runggatscher, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ringler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boidol, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinkel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Májek, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vulliard, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parapatics, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidl, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menche, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehmelt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petronczki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bock, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubicek, S.</span></span> <span> </span><span class="NLM_article-title">Systematic characterization of BAF mutations provides insights into intracomplex synthetic lethalities in human cancers</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">1399</span>– <span class="NLM_lpage">1410</span>, <span class="refDoi"> DOI: 10.1038/s41588-019-0477-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fs41588-019-0477-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=31427792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1WjurjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2019&pages=1399-1410&author=S.+Schickauthor=A.+F.+Rendeiroauthor=K.+Runggatscherauthor=A.+Ringlerauthor=B.+Boidolauthor=M.+Hinkelauthor=P.+M%C3%A1jekauthor=L.+Vulliardauthor=T.+Penzauthor=K.+Parapaticsauthor=C.+Schmidlauthor=J.+Mencheauthor=G.+Boehmeltauthor=M.+Petronczkiauthor=A.+C.+M%C3%BCllerauthor=C.+Bockauthor=S.+Kubicek&title=Systematic+characterization+of+BAF+mutations+provides+insights+into+intracomplex+synthetic+lethalities+in+human+cancers&doi=10.1038%2Fs41588-019-0477-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic characterization of BAF mutations provides insights into intracomplex synthetic lethalities in human cancers</span></div><div class="casAuthors">Schick, Sandra; Rendeiro, Andre F.; Runggatscher, Kathrin; Ringler, Anna; Boidol, Bernd; Hinkel, Melanie; Majek, Peter; Vulliard, Loan; Penz, Thomas; Parapatics, Katja; Schmidl, Christian; Menche, Joerg; Boehmelt, Guido; Petronczki, Mark; Mueller, Andre C.; Bock, Christoph; Kubicek, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1399-1410</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Aberrations in genes coding for subunits of the BRG1/BRM assocd. factor (BAF) chromatin remodeling complexes are highly abundant in human cancers.  Currently, it is not understood how these mostly loss-of-function mutations contribute to cancer development and how they can be targeted therapeutically.  The cancer-type-specific occurrence patterns of certain subunit mutations suggest subunit-specific effects on BAF complex function, possibly by the formation of aberrant residual complexes.  Here, we systematically characterize the effects of individual subunit loss on complex compn., chromatin accessibility and gene expression in a panel of knockout cell lines deficient for 22 BAF subunits.  We observe strong, specific and sometimes discordant alterations dependent on the targeted subunit and show that these explain intracomplex codependencies, including the synthetic lethal interactions SMARCA4-ARID2, SMARCA4-ACTB and SMARCC1-SMARCC2.  These data provide insights into the role of different BAF subcomplexes in genome-wide chromatin organization and suggest approaches to therapeutically target BAF-mutant cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriYa3fDjh9k7Vg90H21EOLACvtfcHk0lgqwnb-k9Ctfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1WjurjJ&md5=291cc74adc398d34900ffa6a8aec8e1b</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fs41588-019-0477-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41588-019-0477-9%26sid%3Dliteratum%253Aachs%26aulast%3DSchick%26aufirst%3DS.%26aulast%3DRendeiro%26aufirst%3DA.%2BF.%26aulast%3DRunggatscher%26aufirst%3DK.%26aulast%3DRingler%26aufirst%3DA.%26aulast%3DBoidol%26aufirst%3DB.%26aulast%3DHinkel%26aufirst%3DM.%26aulast%3DM%25C3%25A1jek%26aufirst%3DP.%26aulast%3DVulliard%26aufirst%3DL.%26aulast%3DPenz%26aufirst%3DT.%26aulast%3DParapatics%26aufirst%3DK.%26aulast%3DSchmidl%26aufirst%3DC.%26aulast%3DMenche%26aufirst%3DJ.%26aulast%3DBoehmelt%26aufirst%3DG.%26aulast%3DPetronczki%26aufirst%3DM.%26aulast%3DM%25C3%25BCller%26aufirst%3DA.%2BC.%26aulast%3DBock%26aufirst%3DC.%26aulast%3DKubicek%26aufirst%3DS.%26atitle%3DSystematic%2520characterization%2520of%2520BAF%2520mutations%2520provides%2520insights%2520into%2520intracomplex%2520synthetic%2520lethalities%2520in%2520human%2520cancers%26jtitle%3DNat.%2520Genet.%26date%3D2019%26volume%3D51%26spage%3D1399%26epage%3D1410%26doi%3D10.1038%2Fs41588-019-0477-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Michel, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Avino, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassel, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mashtalir, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenzie, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McBride, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valencia, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bocker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, L. M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remillard, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lareau, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zullow, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fortoul, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadoch, C.</span></span> <span> </span><span class="NLM_article-title">A non-canonical SWI/SNF complex is a synthetic lethal target in cancers driven by BAF complex perturbation</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1410</span>– <span class="NLM_lpage">1420</span>, <span class="refDoi"> DOI: 10.1038/s41556-018-0221-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fs41556-018-0221-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=30397315" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitV2qu7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2018&pages=1410-1420&author=B.+C.+Michelauthor=A.+R.+D%E2%80%99Avinoauthor=S.+H.+Casselauthor=N.+Mashtalirauthor=Z.+M.+McKenzieauthor=M.+J.+McBrideauthor=A.+M.+Valenciaauthor=Q.+Zhouauthor=M.+Bockerauthor=L.+M.+M.+Soaresauthor=J.+Panauthor=D.+I.+Remillardauthor=C.+A.+Lareauauthor=H.+J.+Zullowauthor=N.+Fortoulauthor=N.+S.+Grayauthor=J.+E.+Bradnerauthor=H.+M.+Chanauthor=C.+Kadoch&title=A+non-canonical+SWI%2FSNF+complex+is+a+synthetic+lethal+target+in+cancers+driven+by+BAF+complex+perturbation&doi=10.1038%2Fs41556-018-0221-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A non-canonical SWI/SNF complex is a synthetic lethal target in cancers driven by BAF complex perturbation</span></div><div class="casAuthors">Michel, Brittany C.; D'Avino, Andrew R.; Cassel, Seth H.; Mashtalir, Nazar; McKenzie, Zachary M.; McBride, Matthew J.; Valencia, Alfredo M.; Zhou, Qianhe; Bocker, Michael; Soares, Luis M. M.; Pan, Joshua; Remillard, David I.; Lareau, Caleb A.; Zullow, Hayley J.; Fortoul, Nora; Gray, Nathanael S.; Bradner, James E.; Chan, Ho Man; Kadoch, Cigall</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1410-1420</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Mammalian SWI/SNF chromatin remodeling complexes exist in three distinct, final-form assemblies: canonical BAF (cBAF), PBAF and a newly characterized non-canonical complex (ncBAF).  However, their complex-specific targeting on chromatin, functions and roles in disease remain largely undefined.  Here, we comprehensively mapped complex assemblies on chromatin and found that ncBAF complexes uniquely localize to CTCF sites and promoters.  We identified ncBAF subunits as synthetic lethal targets specific to synovial sarcoma and malignant rhabdoid tumors, which both exhibit cBAF complex (SMARCB1 subunit) perturbation.  Chem. and biol. depletion of the ncBAF subunit, BRD9, rapidly attenuates synovial sarcoma and malignant rhabdoid tumor cell proliferation.  Importantly, in cBAF-perturbed cancers, ncBAF complexes maintain gene expression at retained CTCF-promoter sites and function in a manner distinct from fusion oncoprotein-bound complexes.  Together, these findings unmask the unique targeting and functional roles of ncBAF complexes and present new cancer-specific therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWZOWaYPPJk7Vg90H21EOLACvtfcHk0lgqwnb-k9Ctfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitV2qu7nL&md5=1c8118a4e3a2a276a15aa42f5e8dd03c</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fs41556-018-0221-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41556-018-0221-1%26sid%3Dliteratum%253Aachs%26aulast%3DMichel%26aufirst%3DB.%2BC.%26aulast%3DD%25E2%2580%2599Avino%26aufirst%3DA.%2BR.%26aulast%3DCassel%26aufirst%3DS.%2BH.%26aulast%3DMashtalir%26aufirst%3DN.%26aulast%3DMcKenzie%26aufirst%3DZ.%2BM.%26aulast%3DMcBride%26aufirst%3DM.%2BJ.%26aulast%3DValencia%26aufirst%3DA.%2BM.%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DBocker%26aufirst%3DM.%26aulast%3DSoares%26aufirst%3DL.%2BM.%2BM.%26aulast%3DPan%26aufirst%3DJ.%26aulast%3DRemillard%26aufirst%3DD.%2BI.%26aulast%3DLareau%26aufirst%3DC.%2BA.%26aulast%3DZullow%26aufirst%3DH.%2BJ.%26aulast%3DFortoul%26aufirst%3DN.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DChan%26aufirst%3DH.%2BM.%26aulast%3DKadoch%26aufirst%3DC.%26atitle%3DA%2520non-canonical%2520SWI%252FSNF%2520complex%2520is%2520a%2520synthetic%2520lethal%2520target%2520in%2520cancers%2520driven%2520by%2520BAF%2520complex%2520perturbation%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2018%26volume%3D20%26spage%3D1410%26epage%3D1420%26doi%3D10.1038%2Fs41556-018-0221-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerstenberger, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trzupek, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedele, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parikh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samas, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Mahony, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berg, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrey, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samardjiev, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasipak, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padilla-Benavides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbalzano, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nickerson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span> <span> </span><span class="NLM_article-title">Identification of a chemical probe for family VIII bromodomains through optimization of a fragment hit</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4800</span>– <span class="NLM_lpage">4811</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00012</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00012" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmsl2rs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4800-4811&author=B.+S.+Gerstenbergerauthor=J.+D.+Trzupekauthor=C.+Tallantauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=P.+E.+Brennanauthor=V.+Fedeleauthor=S.+Martinauthor=S.+Picaudauthor=C.+Rogersauthor=M.+Parikhauthor=A.+Taylorauthor=B.+Samasauthor=A.+O%E2%80%99Mahonyauthor=E.+Bergauthor=G.+Pallaresauthor=A.+D.+Torreyauthor=D.+K.+Treiberauthor=I.+J.+Samardjievauthor=B.+T.+Nasipakauthor=T.+Padilla-Benavidesauthor=Q.+Wuauthor=A.+N.+Imbalzanoauthor=J.+A.+Nickersonauthor=M.+E.+Bunnageauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=D.+R.+Owen&title=Identification+of+a+chemical+probe+for+family+VIII+bromodomains+through+optimization+of+a+fragment+hit&doi=10.1021%2Facs.jmedchem.6b00012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Chemical Probe for Family VIII Bromodomains through Optimization of a Fragment Hit</span></div><div class="casAuthors">Gerstenberger, Brian S.; Trzupek, John D.; Tallant, Cynthia; Fedorov, Oleg; Filippakopoulos, Panagis; Brennan, Paul E.; Fedele, Vita; Martin, Sarah; Picaud, Sarah; Rogers, Catherine; Parikh, Mihir; Taylor, Alexandria; Samas, Brian; O'Mahony, Alison; Berg, Ellen; Pallares, Gabriel; Torrey, Adam V.; Treiber, Daniel K.; Samardjiev, Ivan J.; Nasipak, Brian T.; Padilla-Benavides, Teresita; Wu, Qiong; Imbalzano, Anthony N.; Nickerson, Jeffrey A.; Bunnage, Mark E.; Muller, Susanne; Knapp, Stefan; Owen, Dafydd R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4800-4811</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The acetyl post-translational modification of chromatin at selected histone lysine residues is interpreted by an acetyl-lysine specific interaction with bromodomain reader modules.  Here we report the discovery of the potent, acetyl-lysine-competitive, and cell active inhibitor PFI-3 that binds to certain family VIII bromodomains while displaying significant, broader bromodomain family selectivity.  The high specificity of PFI-3 for family VIII was achieved through a novel bromodomain binding mode of a phenolic headgroup that led to the unusual displacement of water mols. that are generally retained by most other bromodomain inhibitors reported to date.  The medicinal chem. program that led to PFI-3 from an initial fragment screening hit is described in detail, and addnl. analogs with differing family VIII bromodomain selectivity profiles are also reported.  We also describe the full pharmacol. characterization of PFI-3 as a chem. probe, along with phenotypic data on adipocyte and myoblast cell differentiation assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZlrg_xjnZGbVg90H21EOLACvtfcHk0lgAwjh6jMPCyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmsl2rs74%253D&md5=6cb73e76a19f7fb44153f8cbfeff9cf6</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00012%26sid%3Dliteratum%253Aachs%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DTrzupek%26aufirst%3DJ.%2BD.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DFedele%26aufirst%3DV.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DParikh%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DA.%26aulast%3DSamas%26aufirst%3DB.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DBerg%26aufirst%3DE.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DTorrey%26aufirst%3DA.%2BD.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DSamardjiev%26aufirst%3DI.%2BJ.%26aulast%3DNasipak%26aufirst%3DB.%2BT.%26aulast%3DPadilla-Benavides%26aufirst%3DT.%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DImbalzano%26aufirst%3DA.%2BN.%26aulast%3DNickerson%26aufirst%3DJ.%2BA.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26atitle%3DIdentification%2520of%2520a%2520chemical%2520probe%2520for%2520family%2520VIII%2520bromodomains%2520through%2520optimization%2520of%2520a%2520fragment%2520hit%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4800%26epage%3D4811%26doi%3D10.1021%2Facs.jmedchem.6b00012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cockcroft, X.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiegen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hohmann, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knesl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornigg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nar, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaaf, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeeb, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehmelt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of an in vivo active selective BRD9 inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4462</span>– <span class="NLM_lpage">4475</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01865</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01865" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjt1Sit7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4462-4475&author=L.+J.+Martinauthor=M.+Koeglauthor=G.+Baderauthor=X.-L.+Cockcroftauthor=O.+Fedorovauthor=D.+Fiegenauthor=T.+Gerstbergerauthor=M.+H.+Hofmannauthor=A.+F.+Hohmannauthor=D.+Kesslerauthor=S.+Knappauthor=P.+Kneslauthor=S.+Korniggauthor=S.+M%C3%BCllerauthor=H.+Narauthor=C.+Rogersauthor=K.+Rumpelauthor=O.+Schaafauthor=S.+Steurerauthor=C.+Tallantauthor=C.+R.+Vakocauthor=M.+Zeebauthor=A.+Zoephelauthor=M.+Pearsonauthor=G.+Boehmeltauthor=D.+McConnell&title=Structure-based+design+of+an+in+vivo+active+selective+BRD9+inhibitor&doi=10.1021%2Facs.jmedchem.5b01865"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of an in Vivo Active Selective BRD9 Inhibitor</span></div><div class="casAuthors">Martin, Laetitia J.; Koegl, Manfred; Bader, Gerd; Cockcroft, Xiao-Ling; Fedorov, Oleg; Fiegen, Dennis; Gerstberger, Thomas; Hofmann, Marco H.; Hohmann, Anja F.; Kessler, Dirk; Knapp, Stefan; Knesl, Petr; Kornigg, Stefan; Mueller, Susanne; Nar, Herbert; Rogers, Catherine; Rumpel, Klaus; Schaaf, Otmar; Steurer, Steffen; Tallant, Cynthia; Vakoc, Christopher R.; Zeeb, Markus; Zoephel, Andreas; Pearson, Mark; Boehmelt, Guido; McConnell, Darryl</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4462-4475</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Components of the chromatin remodelling switch/sucrose nonfermentable (SWI/SNF) complex are recurrently mutated in tumors, suggesting that altering the activity of the complex plays a role in oncogenesis.  However, the role that the individual subunits play in this process is not clear.  We set out to develop an inhibitor compd. targeting the bromodomain of BRD9 in order to evaluate its function within the SWI/SNF complex.  Here, we present the discovery and development of a potent and selective BRD9 bromodomain inhibitor series based on a new pyridinone-like scaffold.  Crystallog. information on the inhibitors bound to BRD9 guided their development with respect to potency for BRD9 and selectivity against BRD4.  These compds. modulate BRD9 bromodomain cellular function and display antitumor activity in an AML xenograft model.  Two chem. probes, BI-7273 (1) and BI-9564 (2), were identified that should prove to be useful in further exploring BRD9 bromodomain biol. in both in vitro and in vivo settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWHs4hA8bijbVg90H21EOLACvtfcHk0liS9kf1Gf-Bxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjt1Sit7w%253D&md5=b1f4394a7a410273c800b65ee9697c7e</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01865&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01865%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DL.%2BJ.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DCockcroft%26aufirst%3DX.-L.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFiegen%26aufirst%3DD.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DHofmann%26aufirst%3DM.%2BH.%26aulast%3DHohmann%26aufirst%3DA.%2BF.%26aulast%3DKessler%26aufirst%3DD.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKnesl%26aufirst%3DP.%26aulast%3DKornigg%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DNar%26aufirst%3DH.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DSchaaf%26aufirst%3DO.%26aulast%3DSteurer%26aufirst%3DS.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DZeeb%26aufirst%3DM.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DBoehmelt%26aufirst%3DG.%26aulast%3DMcConnell%26aufirst%3DD.%26atitle%3DStructure-based%2520design%2520of%2520an%2520in%2520vivo%2520active%2520selective%2520BRD9%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4462%26epage%3D4475%26doi%3D10.1021%2Facs.jmedchem.5b01865" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sutherell, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yapp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siejka, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenton, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ley, S. V.</span></span> <span> </span><span class="NLM_article-title">Identification and development of 2,3-dihydropyrrolo1,2-aquinazolin-5(1H)-one inhibitors targeting bromodomains within the switch/sucrose nonfermenting complex</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">5095</span>– <span class="NLM_lpage">5101</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01997</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01997" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmsl2rt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5095-5101&author=C.+L.+Sutherellauthor=C.+Tallantauthor=O.+P.+Monteiroauthor=C.+Yappauthor=J.+E.+Fuchsauthor=O.+Fedorovauthor=P.+Siejkaauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=J.+D.+Brentonauthor=P.+E.+Brennanauthor=S.+V.+Ley&title=Identification+and+development+of+2%2C3-dihydropyrrolo1%2C2-aquinazolin-5%281H%29-one+inhibitors+targeting+bromodomains+within+the+switch%2Fsucrose+nonfermenting+complex&doi=10.1021%2Facs.jmedchem.5b01997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Development of 2,3-Dihydropyrrolo[1,2-a]quinazolin-5(1H)-one Inhibitors Targeting Bromodomains within the Switch/Sucrose Nonfermenting Complex</span></div><div class="casAuthors">Sutherell, Charlotte L.; Tallant, Cynthia; Monteiro, Octovia P.; Yapp, Clarence; Fuchs, Julian E.; Fedorov, Oleg; Siejka, Paulina; Muller, Suzanne; Knapp, Stefan; Brenton, James D.; Brennan, Paul E.; Ley, Steven V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5095-5101</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain contg. proteins PB1, SMARCA4, and SMARCA2 are important components of SWI/SNF chromatin remodeling complexes.  We identified bromodomain inhibitors that target these proteins and display unusual binding modes involving water displacement from the KAc binding site.  The best compd. (I) binds the fifth bromodomain of PB1 with a KD of 124 nM, SMARCA2B and SMARCA4 with KD values of 262 and 417 nM, resp., and displays excellent selectivity over bromodomains other than PB1, SMARCA2, and SMARCA4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfnWlpl3TimLVg90H21EOLACvtfcHk0liS9kf1Gf-Bxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmsl2rt7s%253D&md5=a27afa2c97bd74462a09c9ec231f61ef</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01997%26sid%3Dliteratum%253Aachs%26aulast%3DSutherell%26aufirst%3DC.%2BL.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DFuchs%26aufirst%3DJ.%2BE.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DSiejka%26aufirst%3DP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrenton%26aufirst%3DJ.%2BD.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DLey%26aufirst%3DS.%2BV.%26atitle%3DIdentification%2520and%2520development%2520of%25202%252C3-dihydropyrrolo1%252C2-aquinazolin-5%25281H%2529-one%2520inhibitors%2520targeting%2520bromodomains%2520within%2520the%2520switch%252Fsucrose%2520nonfermenting%2520complex%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5095%26epage%3D5101%26doi%3D10.1021%2Facs.jmedchem.5b01997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Myrianthopoulos, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaboriaud-Kolar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grigoriou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brownlee, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heidenreich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wanior, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drosos, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mexia, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagratuni, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kastritis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terpos, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skaltsounis, A.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Downs, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikros, E.</span></span> <span> </span><span class="NLM_article-title">Discovery and optimization of a selective ligand for the switch/sucrose nonfermenting-related bromodomains of polybromo protein-1 by the use of virtual screening and hydration analysis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">8787</span>– <span class="NLM_lpage">8803</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00355</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00355" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVyqtLrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8787-8803&author=V.+Myrianthopoulosauthor=N.+Gaboriaud-Kolarauthor=C.+Tallantauthor=M.-L.+Hallauthor=S.+Grigoriouauthor=P.+M.+Brownleeauthor=O.+Fedorovauthor=C.+Rogersauthor=D.+Heidenreichauthor=M.+Waniorauthor=N.+Drososauthor=N.+Mexiaauthor=P.+Savitskyauthor=T.+Bagratuniauthor=E.+Kastritisauthor=E.+Terposauthor=P.+Filippakopoulosauthor=S.+M%C3%BCllerauthor=A.-L.+Skaltsounisauthor=J.+A.+Downsauthor=S.+Knappauthor=E.+Mikros&title=Discovery+and+optimization+of+a+selective+ligand+for+the+switch%2Fsucrose+nonfermenting-related+bromodomains+of+polybromo+protein-1+by+the+use+of+virtual+screening+and+hydration+analysis&doi=10.1021%2Facs.jmedchem.6b00355"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of a Selective Ligand for the Switch/Sucrose Nonfermenting-Related Bromodomains of Polybromo Protein-1 by the Use of Virtual Screening and Hydration Analysis</span></div><div class="casAuthors">Myrianthopoulos, Vassilios; Gaboriaud-Kolar, Nicolas; Tallant, Cynthia; Hall, Michelle-Lynn; Grigoriou, Stylianos; Brownlee, Peter Moore; Fedorov, Oleg; Rogers, Catherine; Heidenreich, David; Wanior, Marek; Drosos, Nikolaos; Mexia, Nikitia; Savitsky, Pavel; Bagratuni, Tina; Kastritis, Efstathios; Terpos, Evangelos; Filippakopoulos, Panagis; Muller, Susanne; Skaltsounis, Alexios-Leandros; Downs, Jessica Ann; Knapp, Stefan; Mikros, Emmanuel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8787-8803</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are epigenetic interaction domains currently recognized as emerging drug targets for development of anticancer or anti-inflammatory agents.  In this study, development of a selective ligand of the fifth BRD of Polybromo protein-1 (PB1(5)) related to Switch/Sucrose Non-Fermenting (SWI/SNF) chromatin remodeling complexes is presented.  A compd. collection was evaluated by consensus virtual screening and a hit was identified.  The biophys. study of protein-ligand interactions was performed using x-ray crystallog. and isothermal titrn. calorimetry.  Collective data supported the hypothesis that affinity improvement could be achieved by enhancing interactions of the complex with the solvent.  The derived SAR along with free energy calcns. and a consensus hydration anal. using WaterMap and SZmap algorithms guided rational design of a set of novel analogs.  The most potent analog demonstrated high affinity of 3.3 μM and an excellent selectivity profile, thus comprising a promising lead for the development of chem. probes targeting PB1(5).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrMsChShl81rVg90H21EOLACvtfcHk0liXz_j44729CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVyqtLrL&md5=026dcc048ad31b7bf37c9a41b7562018</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00355&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00355%26sid%3Dliteratum%253Aachs%26aulast%3DMyrianthopoulos%26aufirst%3DV.%26aulast%3DGaboriaud-Kolar%26aufirst%3DN.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DHall%26aufirst%3DM.-L.%26aulast%3DGrigoriou%26aufirst%3DS.%26aulast%3DBrownlee%26aufirst%3DP.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DHeidenreich%26aufirst%3DD.%26aulast%3DWanior%26aufirst%3DM.%26aulast%3DDrosos%26aufirst%3DN.%26aulast%3DMexia%26aufirst%3DN.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DBagratuni%26aufirst%3DT.%26aulast%3DKastritis%26aufirst%3DE.%26aulast%3DTerpos%26aufirst%3DE.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DSkaltsounis%26aufirst%3DA.-L.%26aulast%3DDowns%26aufirst%3DJ.%2BA.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DMikros%26aufirst%3DE.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520a%2520selective%2520ligand%2520for%2520the%2520switch%252Fsucrose%2520nonfermenting-related%2520bromodomains%2520of%2520polybromo%2520protein-1%2520by%2520the%2520use%2520of%2520virtual%2520screening%2520and%2520hydration%2520analysis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8787%26epage%3D8803%26doi%3D10.1021%2Facs.jmedchem.6b00355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, L.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, H.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Identification of small molecule inhibitors targeting the SMARCA2 bromodomain from a high-throughput screening assay</span>. <i>Acta Pharmacol. Sin.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">1544</span>– <span class="NLM_lpage">1552</span>, <span class="refDoi"> DOI: 10.1038/aps.2017.188</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Faps.2017.188" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=29795359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsF2ltL7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2018&pages=1544-1552&author=T.+Luauthor=J.-C.+Huauthor=W.-C.+Luauthor=J.+Hanauthor=H.+Dingauthor=H.+Jiangauthor=Y.-Y.+Zhangauthor=L.-Y.+Yueauthor=S.-J.+Chenauthor=H.-L.+Jiangauthor=K.-X.+Chenauthor=H.-F.+Chaiauthor=C.+Luo&title=Identification+of+small+molecule+inhibitors+targeting+the+SMARCA2+bromodomain+from+a+high-throughput+screening+assay&doi=10.1038%2Faps.2017.188"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of small molecule inhibitors targeting the SMARCA2 bromodomain from a high-throughput screening assay</span></div><div class="casAuthors">Lu, Tian; Hu, Jun-chi; Lu, Wen-chao; Han, Jie; Ding, Hong; Jiang, Hao; Zhang, Yuan-yuan; Yue, Li-yan; Chen, Shi-jie; Jiang, Hua-liang; Chen, Kai-xian; Chai, Hui-fang; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Acta Pharmacologica Sinica</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1544-1552</span>CODEN:
                <span class="NLM_cas:coden">APSCG5</span>;
        ISSN:<span class="NLM_cas:issn">1671-4083</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">SMARCA2 is a crit. catalytic subunit of the switch/sucrose non-fermenting (SWI/SNF) chromatin remodeling complexes.  Dysregulation of SMARCA2 is assocd. with several diseases, including some cancers.  SMARCA2 is multi-domain protein contg. a bromodomain (BRD) that specifically recognizes acetylated lysine residues in histone tails, thus playing an important role in chromatin remodeling.  Many potent and specific inhibitors targeting other BRDs have recently been discovered and have been widely used for cancer treatments and biol. research.  However, hit discovery targeting SMARCA2-BRD is particularly lacking.  To date, there is a paucity of reported high-throughput screening (HTS) assays targeting the SMARCA2-BRD interface.  In this study, we developed an AlphaScreen HTS system for the discovery of SMARCA2-BRD inhibitors and optimized the physicochem. conditions including pH, salt concns. and detergent levels.  Through an established AlphaScreen-based high-throughput screening assay against an inhouse compd. library, DCSM06 was identified as a novel SMARCA2-BRD inhibitor with an IC50 value of 39.9±3.0 μmol/L.  Surface plasmon resonance demonstrated the binding between SMARCA2-BRD and DCSM06 (Kd=38.6 μmol/L).  A similarity-based analog search led to identification of DCSM06-05 with an IC50 value of 9.0±1.4 μmol/L.  Mol. docking was performed to predict the binding mode of DCSM06-05 and to decipher the structural basis of the infiuence of chem. modifications on inhibitor potency.  DCSM06-05 may be used as a starting point for further medicinal chem. optimization and could function as a chem. tool for SMARCA2-related functional studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPiXXz-0iPmrVg90H21EOLACvtfcHk0lhncw2jLuCMuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsF2ltL7F&md5=8f7d612af8147bd832236015322cd444</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Faps.2017.188&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Faps.2017.188%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DT.%26aulast%3DHu%26aufirst%3DJ.-C.%26aulast%3DLu%26aufirst%3DW.-C.%26aulast%3DHan%26aufirst%3DJ.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DY.-Y.%26aulast%3DYue%26aufirst%3DL.-Y.%26aulast%3DChen%26aufirst%3DS.-J.%26aulast%3DJiang%26aufirst%3DH.-L.%26aulast%3DChen%26aufirst%3DK.-X.%26aulast%3DChai%26aufirst%3DH.-F.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520small%2520molecule%2520inhibitors%2520targeting%2520the%2520SMARCA2%2520bromodomain%2520from%2520a%2520high-throughput%2520screening%2520assay%26jtitle%3DActa%2520Pharmacol.%2520Sin.%26date%3D2018%26volume%3D39%26spage%3D1544%26epage%3D1552%26doi%3D10.1038%2Faps.2017.188" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, P. G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vieira, L. C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singleton, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baronio, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Méndez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. J.</span></span> <span> </span><span class="NLM_article-title">LP99: Discovery and synthesis of the first selective BRD7/9 bromodomain inhibitor</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">6217</span>– <span class="NLM_lpage">6221</span>, <span class="refDoi"> DOI: 10.1002/anie.201501394</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1002%2Fanie.201501394" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtlGqsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=6217-6221&author=P.+G.+K.+Clarkauthor=L.+C.+C.+Vieiraauthor=C.+Tallantauthor=O.+Fedorovauthor=D.+C.+Singletonauthor=C.+M.+Rogersauthor=O.+P.+Monteiroauthor=J.+M.+Bennettauthor=R.+Baronioauthor=S.+M%C3%BCllerauthor=D.+L.+Danielsauthor=J.+M%C3%A9ndezauthor=S.+Knappauthor=P.+E.+Brennanauthor=D.+J.+Dixon&title=LP99%3A+Discovery+and+synthesis+of+the+first+selective+BRD7%2F9+bromodomain+inhibitor&doi=10.1002%2Fanie.201501394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">LP99: Discovery and Synthesis of the First Selective BRD7/9 Bromodomain Inhibitor</span></div><div class="casAuthors">Clark, Peter G. K.; Vieira, Lucas C. C.; Tallant, Cynthia; Fedorov, Oleg; Singleton, Dean C.; Rogers, Catherine M.; Monteiro, Octovia P.; Bennett, James M.; Baronio, Roberta; Mueller, Susanne; Daniels, Danette L.; Mendez, Jacqui; Knapp, Stefan; Brennan, Paul E.; Dixon, Darren J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6217-6221</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The bromodomain-contg. proteins BRD9 and BRD7 are part of the human SWI/SNF chromatin-remodeling complexes BAF and PBAF.  To date, no selective inhibitor for BRD7/9 has been reported despite its potential value as a biol. tool or as a lead for future therapeutics.  The quinolone-fused lactam LP99 (I) is now reported as the first potent and selective inhibitor of the BRD7 and BRD9 bromodomains.  Development of LP99 from a fragment hit was expedited through balancing structure-based inhibitor design and biophys. characterization against tractable chem. synthesis: Complexity-building nitro-Mannich/lactamization cascade processes allowed for early structure-activity relationship studies whereas an enantioselective organocatalytic nitro-Mannich reaction enabled the synthesis of the lead scaffold in enantioenriched form and on scale.  This epigenetic probe was shown to inhibit the assocn. of BRD7 and BRD9 to acetylated histones in vitro and in cells.  Moreover, LP99 was used to demonstrate that BRD7/9 plays a role in regulating pro-inflammatory cytokine secretion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplfOmbNMpdPbVg90H21EOLACvtfcHk0ljtGufcnAH8Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtlGqsrg%253D&md5=9dee2b352045f7ae5efcbdc83f240a3f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fanie.201501394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201501394%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DP.%2BG.%2BK.%26aulast%3DVieira%26aufirst%3DL.%2BC.%2BC.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DSingleton%26aufirst%3DD.%2BC.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DBennett%26aufirst%3DJ.%2BM.%26aulast%3DBaronio%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DM%25C3%25A9ndez%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DDixon%26aufirst%3DD.%2BJ.%26atitle%3DLP99%253A%2520Discovery%2520and%2520synthesis%2520of%2520the%2520first%2520selective%2520BRD7%252F9%2520bromodomain%2520inhibitor%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D6217%26epage%3D6221%26doi%3D10.1002%2Fanie.201501394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of potent and selective inhibitors of BRD7 and BRD9 bromodomains</span>. <i>Medchemcomm</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1381</span>– <span class="NLM_lpage">1386</span>, <span class="refDoi"> DOI: 10.1039/c5md00152h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1039%2Fc5md00152h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVSgtrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=1381-1386&author=D.+A.+Hayauthor=C.+M.+Rogersauthor=O.+Fedorovauthor=C.+Tallantauthor=S.+Martinauthor=O.+P.+Monteiroauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=C.+J.+Schofieldauthor=P.+E.+Brennan&title=Design+and+synthesis+of+potent+and+selective+inhibitors+of+BRD7+and+BRD9+bromodomains&doi=10.1039%2Fc5md00152h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Design and synthesis of potent and selective inhibitors of BRD7 and BRD9 bromodomains</span></div><div class="casAuthors">Hay, Duncan A.; Rogers, Catherine M.; Fedorov, Oleg; Tallant, Cynthia; Martin, Sarah; Monteiro, Octovia P.; Muller, Susanne; Knapp, Stefan; Schofield, Christopher J.; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1381-1386</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Emerging evidence suggests bromodomain-contg. proteins 7 and 9 (BRD7 and BRD9) have roles in the regulation of human transcription and disease including cancer.  We describe potent and selective inhibitors of the BRD7 and BRD9 bromodomains intended for use as tools to elucidate the biol. roles of BRD7 and BRD9 in healthy and diseased cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrynQ2AJnreDLVg90H21EOLACvtfcHk0ljtGufcnAH8Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVSgtrvJ&md5=6a93d7300f7815eaf5ca654f873078d3</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1039%2Fc5md00152h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc5md00152h%26sid%3Dliteratum%253Aachs%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520potent%2520and%2520selective%2520inhibitors%2520of%2520BRD7%2520and%2520BRD9%2520bromodomains%26jtitle%3DMedchemcomm%26date%3D2015%26volume%3D6%26spage%3D1381%26epage%3D1386%26doi%3D10.1039%2Fc5md00152h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span>; <span class="NLM_string-name">Cote, A.</span>; <span class="NLM_string-name">Crawford, T. D.</span>; <span class="NLM_string-name">Duplessis, M.</span>; <span class="NLM_string-name">Good, A. C.</span>; <span class="NLM_string-name">Leblanc, Y.</span>; <span class="NLM_string-name">Magnuson, S. R.</span>; <span class="NLM_string-name">Nasveschuk, C. G.</span>; <span class="NLM_string-name">Romero, A. F.</span>; <span class="NLM_string-name">Tang, Y.</span>; <span class="NLM_string-name">Taylor, A. M.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Pyridazine Compounds and Uses thereof</span>. U.S. Patent <span class="NLM_patent">WO 2016138114 A1</span>, Feburary 24, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=B.+K.+Albrecht&author=A.+Cote&author=T.+D.+Crawford&author=M.+Duplessis&author=A.+C.+Good&author=Y.+Leblanc&author=S.+R.+Magnuson&author=C.+G.+Nasveschuk&author=A.+F.+Romero&author=Y.+Tang&author=A.+M.+Taylor&title=Therapeutic+Pyridazine+Compounds+and+Uses+thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26atitle%3DTherapeutic%2520Pyridazine%2520Compounds%2520and%2520Uses%2520thereof%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Theodoulou, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bit, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leveridge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molnar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Discovery of I-BRD9, a selective cell active chemical probe for bromodomain containing protein 9 inhibition</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1425</span>– <span class="NLM_lpage">1439</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00256</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00256" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtlaktb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1425-1439&author=N.+H.+Theodoulouauthor=P.+Bamboroughauthor=A.+J.+Bannisterauthor=I.+Becherauthor=R.+A.+Bitauthor=K.+H.+Cheauthor=C.-W.+Chungauthor=A.+Dittmannauthor=G.+Drewesauthor=D.+H.+Drewryauthor=L.+Gordonauthor=P.+Grandiauthor=M.+Leveridgeauthor=M.+Lindonauthor=A.-M.+Michonauthor=J.+Molnarauthor=S.+C.+Robsonauthor=N.+C.+O.+Tomkinsonauthor=T.+Kouzaridesauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Discovery+of+I-BRD9%2C+a+selective+cell+active+chemical+probe+for+bromodomain+containing+protein+9+inhibition&doi=10.1021%2Facs.jmedchem.5b00256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of I-BRD9, a Selective Cell Active Chemical Probe for Bromodomain Containing Protein 9 Inhibition</span></div><div class="casAuthors">Theodoulou, Natalie H.; Bamborough, Paul; Bannister, Andrew J.; Becher, Isabelle; Bit, Rino A.; Che, Ka Hing; Chung, Chun-wa; Dittmann, Antje; Drewes, Gerard; Drewry, David H.; Gordon, Laurie; Grandi, Paola; Leveridge, Melanie; Lindon, Matthew; Michon, Anne-Marie; Molnar, Judit; Robson, Samuel C.; Tomkinson, Nicholas C. O.; Kouzarides, Tony; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1425-1439</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Acetylation of histone lysine residues is one of the most well-studied post-translational modifications of chromatin, selectively recognized by bromodomain "reader" modules.  Inhibitors of the bromodomain and extra terminal domain (BET) family of bromodomains have shown profound anticancer and anti-inflammatory properties, generating much interest in targeting other bromodomain-contg. proteins for disease treatment.  Herein, we report the discovery of I-BRD9, the first selective cellular chem. probe for bromodomain-contg. protein 9 (BRD9).  I-BRD9 was identified through structure-based design, leading to greater than 700-fold selectivity over the BET family and 200-fold over the highly homologous bromodomain-contg. protein 7 (BRD7).  I-BRD9 was used to identify genes regulated by BRD9 in Kasumi-1 cells involved in oncol. and immune response pathways and to the best of our knowledge, represents the first selective tool compd. available to elucidate the cellular phenotype of BRD9 bromodomain inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVmjhksIDyTrVg90H21EOLACvtfcHk0liXWy5izlFdnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtlaktb8%253D&md5=4e8f6aa664797fe54772b64c81cf2a83</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00256%26sid%3Dliteratum%253Aachs%26aulast%3DTheodoulou%26aufirst%3DN.%2BH.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DBecher%26aufirst%3DI.%26aulast%3DBit%26aufirst%3DR.%2BA.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DDittmann%26aufirst%3DA.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DLeveridge%26aufirst%3DM.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DMolnar%26aufirst%3DJ.%26aulast%3DRobson%26aufirst%3DS.%2BC.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DDiscovery%2520of%2520I-BRD9%252C%2520a%2520selective%2520cell%2520active%2520chemical%2520probe%2520for%2520bromodomain%2520containing%2520protein%25209%2520inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1425%26epage%3D1439%26doi%3D10.1021%2Facs.jmedchem.5b00256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Remillard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brien, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonnett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wühr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Degradation of the BAF complex factor BRD9 by heterobifunctional ligands</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">5738</span>– <span class="NLM_lpage">5743</span>, <span class="refDoi"> DOI: 10.1002/anie.201611281</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1002%2Fanie.201611281" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtFGrurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=5738-5743&author=D.+Remillardauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=G.+L.+Brienauthor=M.+Sonnettauthor=H.-S.+Seoauthor=S.+Dastjerdiauthor=M.+W%C3%BChrauthor=S.+Dhe-Paganonauthor=S.+A.+Armstrongauthor=J.+E.+Bradner&title=Degradation+of+the+BAF+complex+factor+BRD9+by+heterobifunctional+ligands&doi=10.1002%2Fanie.201611281"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Degradation of the BAF Complex Factor BRD9 by Heterobifunctional Ligands</span></div><div class="casAuthors">Remillard, David; Buckley, Dennis L.; Paulk, Joshiawa; Brien, Gerard L.; Sonnett, Matthew; Seo, Hyuk-Soo; Dastierdi, Shiva; Wuehr, Martin; Dhe-Paganon, Sirano; Armstrong, Scott A.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5738-5743</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The bromodomain-contg. protein BRD9, a subunit of the human BAF (SWI/SNF) nucleosome remodeling complex, has emerged as an attractive therapeutic target in cancer.  Despite the development of chem. probes targeting the BRD9 bromodomain, there is a limited understanding of BRD9 function beyond acetyl-lysine recognition.  We have therefore created the first BRD9-directed chem. degraders, through iterative design and testing of heterobifunctional ligands that bridge the BRD9 bromodomain and the cereblon E3 ubiquitin ligase complex.  Degraders of BRD9 exhibit markedly enhanced potency compared to parental ligands (10- to 100-fold).  Parallel study of degraders with divergent BRD9-binding chemotypes in models of acute myeloid leukemia resolves bromodomain polypharmacol. in this emerging drug class.  Together, these findings reveal the tractability of non-BET bromodomain contg. proteins to chem. degrdn., and highlight lead compd. dBRD9 as a tool for the study of BRD9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ9FU2Ipaq27Vg90H21EOLACvtfcHk0ljRcVbkRhnzHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtFGrurk%253D&md5=2eb6562cc9d976c7ae6070c9a419e9ea</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fanie.201611281&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201611281%26sid%3Dliteratum%253Aachs%26aulast%3DRemillard%26aufirst%3DD.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DBrien%26aufirst%3DG.%2BL.%26aulast%3DSonnett%26aufirst%3DM.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DW%25C3%25BChr%26aufirst%3DM.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDegradation%2520of%2520the%2520BAF%2520complex%2520factor%2520BRD9%2520by%2520heterobifunctional%2520ligands%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D5738%26epage%3D5743%26doi%3D10.1002%2Fanie.201611281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Farnaby, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitworth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diers, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trainor, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zollman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karolyi-Oezguer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riedmueller, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wachter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dank, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharps, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traxler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnitzer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petermann, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstabl, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss-Puxbaum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrenhöfer-Wölfer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wöhrle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehmelt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinnenthal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnhof, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiechens, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen-Hughes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettmayer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">BAF complex vulnerabilities in cancer demonstrated via structure-based PROTAC design</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">672</span>– <span class="NLM_lpage">680</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0294-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fs41589-019-0294-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=31178587" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFeqtLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=672-680&author=W.+Farnabyauthor=M.+Koeglauthor=M.+J.+Royauthor=C.+Whitworthauthor=E.+Diersauthor=N.+Trainorauthor=D.+Zollmanauthor=S.+Steurerauthor=J.+Karolyi-Oezguerauthor=C.+Riedmuellerauthor=T.+Gmaschitzauthor=J.+Wachterauthor=C.+Dankauthor=M.+Galantauthor=B.+Sharpsauthor=K.+Rumpelauthor=E.+Traxlerauthor=T.+Gerstbergerauthor=R.+Schnitzerauthor=O.+Petermannauthor=P.+Grebauthor=H.+Weinstablauthor=G.+Baderauthor=A.+Zoephelauthor=A.+Weiss-Puxbaumauthor=K.+Ehrenh%C3%B6fer-W%C3%B6lferauthor=S.+W%C3%B6hrleauthor=G.+Boehmeltauthor=J.+Rinnenthalauthor=H.+Arnhofauthor=N.+Wiechensauthor=M.-Y.+Wuauthor=T.+Owen-Hughesauthor=P.+Ettmayerauthor=M.+Pearsonauthor=D.+B.+McConnellauthor=A.+Ciulli&title=BAF+complex+vulnerabilities+in+cancer+demonstrated+via+structure-based+PROTAC+design&doi=10.1038%2Fs41589-019-0294-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">BAF complex vulnerabilities in cancer demonstrated via structure-based PROTAC design</span></div><div class="casAuthors">Farnaby, William; Koegl, Manfred; Roy, Michael J.; Whitworth, Claire; Diers, Emelyne; Trainor, Nicole; Zollman, David; Steurer, Steffen; Karolyi-Oezguer, Jale; Riedmueller, Carina; Gmaschitz, Teresa; Wachter, Johannes; Dank, Christian; Galant, Michael; Sharps, Bernadette; Rumpel, Klaus; Traxler, Elisabeth; Gerstberger, Thomas; Schnitzer, Renate; Petermann, Oliver; Greb, Peter; Weinstabl, Harald; Bader, Gerd; Zoephel, Andreas; Weiss-Puxbaum, Alexander; Ehrenhofer-Wolfer, Katharina; Wohrle, Simon; Boehmelt, Guido; Rinnenthal, Joerg; Arnhof, Heribert; Wiechens, Nicola; Wu, Meng-Ying; Owen-Hughes, Tom; Ettmayer, Peter; Pearson, Mark; McConnell, Darryl B.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">672-680</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Targeting subunits of BAF/PBAF chromatin remodeling complexes has been proposed as an approach to exploit cancer vulnerabilities.  Here, we develop proteolysis targeting chimera (PROTAC) degraders of the BAF ATPase subunits SMARCA2 and SMARCA4 using a bromodomain ligand and recruitment of the E3 ubiquitin ligase VHL.  High-resoln. ternary complex crystal structures and biophys. investigation guided rational and efficient optimization toward ACBI1(I), a potent and cooperative degrader of SMARCA2, SMARCA4 and PBRM1.  I induced anti-proliferative effects and cell death caused by SMARCA2 depletion in SMARCA4 mutant cancer cells, and in acute myeloid leukemia cells dependent on SMARCA4 ATPase activity.  These findings exemplify a successful biophysics- and structure-based PROTAC design approach to degrade high profile drug targets, and pave the way toward new therapeutics for the treatment of tumors sensitive to the loss of BAF complex ATPases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonEh4pux4Bs7Vg90H21EOLACvtfcHk0ljVftpEqDeS1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFeqtLfJ&md5=e95fad12c727aedcf08aed9320dae905</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0294-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0294-6%26sid%3Dliteratum%253Aachs%26aulast%3DFarnaby%26aufirst%3DW.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DRoy%26aufirst%3DM.%2BJ.%26aulast%3DWhitworth%26aufirst%3DC.%26aulast%3DDiers%26aufirst%3DE.%26aulast%3DTrainor%26aufirst%3DN.%26aulast%3DZollman%26aufirst%3DD.%26aulast%3DSteurer%26aufirst%3DS.%26aulast%3DKarolyi-Oezguer%26aufirst%3DJ.%26aulast%3DRiedmueller%26aufirst%3DC.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DWachter%26aufirst%3DJ.%26aulast%3DDank%26aufirst%3DC.%26aulast%3DGalant%26aufirst%3DM.%26aulast%3DSharps%26aufirst%3DB.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DTraxler%26aufirst%3DE.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DSchnitzer%26aufirst%3DR.%26aulast%3DPetermann%26aufirst%3DO.%26aulast%3DGreb%26aufirst%3DP.%26aulast%3DWeinstabl%26aufirst%3DH.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DWeiss-Puxbaum%26aufirst%3DA.%26aulast%3DEhrenh%25C3%25B6fer-W%25C3%25B6lfer%26aufirst%3DK.%26aulast%3DW%25C3%25B6hrle%26aufirst%3DS.%26aulast%3DBoehmelt%26aufirst%3DG.%26aulast%3DRinnenthal%26aufirst%3DJ.%26aulast%3DArnhof%26aufirst%3DH.%26aulast%3DWiechens%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DM.-Y.%26aulast%3DOwen-Hughes%26aufirst%3DT.%26aulast%3DEttmayer%26aufirst%3DP.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DMcConnell%26aufirst%3DD.%2BB.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DBAF%2520complex%2520vulnerabilities%2520in%2520cancer%2520demonstrated%2520via%2520structure-based%2520PROTAC%2520design%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D672%26epage%3D680%26doi%3D10.1038%2Fs41589-019-0294-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brien, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remillard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hemming, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chabon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wynne, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cagney, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Mierlo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baltissen, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vermeulen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fröhling, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span> <span> </span><span class="NLM_article-title">Targeted degradation of BRD9 reverses oncogenic gene expression in synovial sarcoma</span>. <i>Elife</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e41305</span> <span class="refDoi"> DOI: 10.7554/elife.41305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.7554%2FeLife.41305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=30431433" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlyqs7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&author=G.+L.+Brienauthor=D.+Remillardauthor=J.+Shiauthor=M.+L.+Hemmingauthor=J.+Chabonauthor=K.+Wynneauthor=E.+T.+Dillonauthor=G.+Cagneyauthor=G.+van+Mierloauthor=M.+P.+Baltissenauthor=M.+Vermeulenauthor=J.+Qiauthor=S.+Fr%C3%B6hlingauthor=N.+S.+Grayauthor=J.+E.+Bradnerauthor=C.+R.+Vakocauthor=S.+A.+Armstrong&title=Targeted+degradation+of+BRD9+reverses+oncogenic+gene+expression+in+synovial+sarcoma&doi=10.7554%2Felife.41305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted degradation of BRD9 reverses oncogenic gene expression in synovial sarcoma</span></div><div class="casAuthors">Brien, Gerard L.; Remillard, David; Shi, Junwei; Hemming, Matthew L.; Chabon, Jonathon; Wynne, Kieran; Dillon, Euge Ne T.; Cagney, Gerard; Van Mierlo, Guido; Baltissen, Marijke P.; Vermeulen, Michiel; Qi, Jun; Froehling, Stefan; Gray, Nathanael S.; Bradner, James E.; Vakoc, Christopher R.; Armstrong, Scott A.</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e41305/1-e41305/26</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Synovial sarcoma tumors contain a characteristic fusion protein, SS18-SSX, which drives disease development.  Targeting oncogenic fusion proteins presents an attractive therapeutic opportunity.  However, SS18-SSX has proven intractable for therapeutic intervention.  Using a domain-focused CRISPR screen we identified the bromodomain of BRD9 as a crit. functional dependency in synovial sarcoma.  BRD9 is a component of SS18-SSX contg. BAF complexes in synovial sarcoma cells; and integration of BRD9 into these complexes is crit. for cell growth.  Moreover BRD9 and SS18-SSX co-localize extensively on the synovial sarcoma genome.  Remarkably, synovial sarcoma cells are highly sensitive to a novel small mol. degrader of BRD9, while other sarcoma subtypes are unaffected.  Degrdn. of BRD9 induces downregulation of oncogenic transcriptional programs and inhibits tumor progression in vivo.  We demonstrate that BRD9 supports oncogenic mechanisms underlying the SS18-SSX fusion in synovial sarcoma and highlight targeted degrdn. of BRD9 as a potential therapeutic opportunity in this disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfd6-D9YlmrrVg90H21EOLACvtfcHk0lhiBkZJG40Y0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlyqs7fF&md5=cbda64457406cd0d8fbd1211678b04b3</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.7554%2FeLife.41305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.41305%26sid%3Dliteratum%253Aachs%26aulast%3DBrien%26aufirst%3DG.%2BL.%26aulast%3DRemillard%26aufirst%3DD.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DHemming%26aufirst%3DM.%2BL.%26aulast%3DChabon%26aufirst%3DJ.%26aulast%3DWynne%26aufirst%3DK.%26aulast%3DDillon%26aufirst%3DE.%2BT.%26aulast%3DCagney%26aufirst%3DG.%26aulast%3Dvan%2BMierlo%26aufirst%3DG.%26aulast%3DBaltissen%26aufirst%3DM.%2BP.%26aulast%3DVermeulen%26aufirst%3DM.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DFr%25C3%25B6hling%26aufirst%3DS.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26atitle%3DTargeted%2520degradation%2520of%2520BRD9%2520reverses%2520oncogenic%2520gene%2520expression%2520in%2520synovial%2520sarcoma%26jtitle%3DElife%26date%3D2018%26volume%3D7%26doi%3D10.7554%2Felife.41305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castex, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aldeghi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trzupek, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstenberger, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggin, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schüle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Günther, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span> <span> </span><span class="NLM_article-title">Selective targeting of the BRG/PB1 bromodomains impairs embryonic and trophoblast stem cell maintenance</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1</i></span>, <span class="NLM_elocation-id">e1500723</span> <span class="refDoi"> DOI: 10.1126/sciadv.1500723</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1126%2Fsciadv.1500723" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=26702435" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC28XovFaktLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&author=O.+Fedorovauthor=J.+Castexauthor=C.+Tallantauthor=D.+R.+Owenauthor=S.+Martinauthor=M.+Aldeghiauthor=O.+Monteiroauthor=P.+Filippakopoulosauthor=S.+Picaudauthor=J.+D.+Trzupekauthor=B.+S.+Gerstenbergerauthor=C.+Bountraauthor=D.+Willmannauthor=C.+Wellsauthor=M.+Philpottauthor=C.+Rogersauthor=P.+C.+Bigginauthor=P.+E.+Brennanauthor=M.+E.+Bunnageauthor=R.+Sch%C3%BCleauthor=T.+G%C3%BCntherauthor=S.+Knappauthor=S.+M%C3%BCller&title=Selective+targeting+of+the+BRG%2FPB1+bromodomains+impairs+embryonic+and+trophoblast+stem+cell+maintenance&doi=10.1126%2Fsciadv.1500723"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Selective targeting of the BRG/PB1 bromodomains impairs embryonic and trophoblast stem cell maintenance</span></div><div class="casAuthors">Fedorov, Oleg; Castex, Josefina; Tallant, Cynthia; Owen, Dafydd R.; Martin, Sarah; Aldeghi, Matteo; Monteiro, Octovia; Filippakopoulos, Panagis; Picaud, Sarah; Trzupek, John D.; Gerstenberger, Brian S.; Bountra, Chas; Willmann, Dominica; Wells, Christopher; Philpott, Martin; Rogers, Catherine; Biggin, Philip C.; Brennan, Paul E.; Bunnage, Mark E.; Schule, Roland; Gunther, Thomas; Knapp, Stefan; Muller, Susanne</div><div class="citationInfo"><span class="NLM_cas:title">Science Advances</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1500723/1-e1500723/11</span>CODEN:
                <span class="NLM_cas:coden">SACDAF</span>;
        ISSN:<span class="NLM_cas:issn">2375-2548</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Mammalian SWI/SNF [also called Brg/Brahma-assocd. factors (BAFs)] are evolutionarily conserved chromatin remodeling complexes regulating gene transcription programs during development and stem cell differentiation.  BAF complexes contain an ATP (ATP)-driven remodeling enzyme (either BRG1 or BRM) and multiple protein interaction domains including bromodomains, an evolutionary conserved acetyl lysine-dependent protein interaction motif that recruits transcriptional regulators to acetylated chromatin.  We report a potent and cell active protein interaction inhibitor, PFI-3, that selectively binds to essential BAF bromodomains.  The high specificity of PFI-3 was achieved on the basis of a novel binding mode of a salicylic acid head group that led to the replacement of water mols. typically maintained in other bromodomain inhibitor complexes.  We show that exposure of embryonic stem cells to PFI-3 led to deprivation of stemness and deregulated lineage specification.  Furthermore, differentiation of trophoblast stem cells in the presence of PFI-3 was markedly enhanced.  The data present a key function of BAF bromodomains in stem cell maintenance and differentiation, introducing a novel versatile chem. probe for studies on acetylation-dependent cellular processes controlled by BAF remodeling complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiGyzl98dGOLVg90H21EOLACvtfcHk0ljyoZoC97ojog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XovFaktLw%253D&md5=52757e667cc5afeb4c3a151fc8ef3442</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.1500723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.1500723%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DCastex%26aufirst%3DJ.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DAldeghi%26aufirst%3DM.%26aulast%3DMonteiro%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DTrzupek%26aufirst%3DJ.%2BD.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DWillmann%26aufirst%3DD.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DBiggin%26aufirst%3DP.%2BC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DG%25C3%25BCnther%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26atitle%3DSelective%2520targeting%2520of%2520the%2520BRG%252FPB1%2520bromodomains%2520impairs%2520embryonic%2520and%2520trophoblast%2520stem%2520cell%2520maintenance%26jtitle%3DSci.%2520Adv.%26date%3D2015%26volume%3D1%26doi%3D10.1126%2Fsciadv.1500723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ho, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ronan, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staahl, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lessard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nesvizhskii, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranish, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crabtree, G. R.</span></span> <span> </span><span class="NLM_article-title">An embryonic stem cell chromatin remodeling complex, esBAF, is essential for embryonic stem cell self-renewal and pluripotency</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">5181</span>– <span class="NLM_lpage">5186</span>, <span class="refDoi"> DOI: 10.1073/pnas.0812889106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1073%2Fpnas.0812889106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=19279220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksVert70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=5181-5186&author=L.+Hoauthor=J.+L.+Ronanauthor=J.+Wuauthor=B.+T.+Staahlauthor=L.+Chenauthor=A.+Kuoauthor=J.+Lessardauthor=A.+I.+Nesvizhskiiauthor=J.+Ranishauthor=G.+R.+Crabtree&title=An+embryonic+stem+cell+chromatin+remodeling+complex%2C+esBAF%2C+is+essential+for+embryonic+stem+cell+self-renewal+and+pluripotency&doi=10.1073%2Fpnas.0812889106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">An embryonic stem cell chromatin remodeling complex, esBAF, is essential for embryonic stem cell self-renewal and pluripotency</span></div><div class="casAuthors">Ho, Lena; Ronan, Jehnna L.; Wu, Jiang; Staahl, Brett T.; Chen, Lei; Kuo, Ann; Lessard, Julie; Nesvizhskii, Alexey I.; Ranish, Jeff; Crabtree, Gerald R.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5181-5186</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Mammalian SWI/SNF [also called BAF (Brg/Brahma-assocd. factors)] ATP-dependent chromatin remodeling complexes are essential for the formation of the totipotent and pluripotent cells of the early embryo.  In addn., subunits of this complex have been recovered in screens for genes required for nuclear reprogramming in Xenopus and mouse embryonic stem cell (ES cell) morphol.  However, the mechanism underlying the roles of these complexes is unclear.  Here, the authors show that BAF complexes are required for the self-renewal and pluripotency of mouse ES cells but not for the proliferation of fibroblasts or other cells.  The authors' proteomic studies revealed that ES cells expressed distinctive complexes (esBAF) defined by the presence of Brg (Brahma-related gene), BAF155, and BAF60A, and the absence of Brm (Brahma), BAF170, and BAF60C.  The authors showed that this specialized subunit compn. was required for ES cell maintenance and pluripotency.  The authors' proteomic anal. also revealed that esBAF complexes interact directly with key regulators of pluripotency, suggesting that esBAF complexes are specialized to interact with ES cell-specific regulators, providing a potential explanation for the requirement of BAF complexes in pluripotency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIHWi94C9GwbVg90H21EOLACvtfcHk0lg973cVE70qOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksVert70%253D&md5=c9e9b1417ceac18a02b1d29be51cefdd</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0812889106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0812889106%26sid%3Dliteratum%253Aachs%26aulast%3DHo%26aufirst%3DL.%26aulast%3DRonan%26aufirst%3DJ.%2BL.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DStaahl%26aufirst%3DB.%2BT.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DKuo%26aufirst%3DA.%26aulast%3DLessard%26aufirst%3DJ.%26aulast%3DNesvizhskii%26aufirst%3DA.%2BI.%26aulast%3DRanish%26aufirst%3DJ.%26aulast%3DCrabtree%26aufirst%3DG.%2BR.%26atitle%3DAn%2520embryonic%2520stem%2520cell%2520chromatin%2520remodeling%2520complex%252C%2520esBAF%252C%2520is%2520essential%2520for%2520embryonic%2520stem%2520cell%2520self-renewal%2520and%2520pluripotency%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2009%26volume%3D106%26spage%3D5181%26epage%3D5186%26doi%3D10.1073%2Fpnas.0812889106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mailman, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Archer, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">Transcriptional repression by the BRG1-SWI/SNF complex affects the pluripotency of human embryonic stem cells</span>. <i>Stem Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">460</span>– <span class="NLM_lpage">474</span>, <span class="refDoi"> DOI: 10.1016/j.stemcr.2014.07.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2Fj.stemcr.2014.07.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=25241744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlKrtr%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&pages=460-474&author=X.+Zhangauthor=B.+Liauthor=W.+Liauthor=L.+Maauthor=D.+Zhengauthor=L.+Liauthor=W.+Yangauthor=M.+Chuauthor=W.+Chenauthor=R.+B.+Mailmanauthor=J.+Zhuauthor=G.+Fanauthor=T.+K.+Archerauthor=Y.+Wang&title=Transcriptional+repression+by+the+BRG1-SWI%2FSNF+complex+affects+the+pluripotency+of+human+embryonic+stem+cells&doi=10.1016%2Fj.stemcr.2014.07.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional Repression by the BRG1-SWI/SNF Complex Affects the Pluripotency of Human Embryonic Stem Cells</span></div><div class="casAuthors">Zhang, Xiaoli; Li, Bing; Li, Wenguo; Ma, Lijuan; Zheng, Dongyan; Li, Leping; Yang, Weijing; Chu, Min; Chen, Wei; Mailman, Richard B.; Zhu, Jun; Fan, Guoping; Archer, Trevor K.; Wang, Yuan</div><div class="citationInfo"><span class="NLM_cas:title">Stem Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">460-474</span>CODEN:
                <span class="NLM_cas:coden">SCRTFL</span>;
        ISSN:<span class="NLM_cas:issn">2213-6711</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The SWI/SNF complex plays an important role in mouse embryonic stem cells (mESCs), but it remains to be detd. whether this complex is required for the pluripotency of human ESCs (hESCs).  Using RNAi, we demonstrated that depletion of BRG1, the catalytic subunit of the SWI/SNF complex, led to impaired self-renewing ability and dysregulated lineage specification of hESCs.  A unique compn. of the BRG1-SWI/SNF complex in hESCs was further defined by the presence of BRG1, BAF250A, BAF170, BAF155, BAF53A, and BAF47.  Genome-wide expression analyses revealed that BRG1 participated in a broad range of biol. processes in hESCs through pathways different from those in mESCs.  In addn., chromatin immunopptn. sequencing (ChIP-seq) demonstrated that BRG1 played a repressive role in transcriptional regulation by modulating the acetylation levels of H3K27 at the enhancers of lineage-specific genes.  Our data thus provide valuable insights into mol. mechanisms by which transcriptional repression affects the self-renewal and differentiation of hESCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGA2baO4Y6DLVg90H21EOLACvtfcHk0lg973cVE70qOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlKrtr%252FJ&md5=b7a9d386e584eeed58b412f25cd07d61</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.stemcr.2014.07.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.stemcr.2014.07.004%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DZheng%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DChu%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DMailman%26aufirst%3DR.%2BB.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DFan%26aufirst%3DG.%26aulast%3DArcher%26aufirst%3DT.%2BK.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DTranscriptional%2520repression%2520by%2520the%2520BRG1-SWI%252FSNF%2520complex%2520affects%2520the%2520pluripotency%2520of%2520human%2520embryonic%2520stem%2520cells%26jtitle%3DStem%2520Cell%2520Rep.%26date%3D2014%26volume%3D3%26spage%3D460%26epage%3D474%26doi%3D10.1016%2Fj.stemcr.2014.07.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klochendler-Yeivin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiette, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barra, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muchardt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babinet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yaniv, M.</span></span> <span> </span><span class="NLM_article-title">The murine SNF5/INI1 chromatin remodeling factor is essential for embryonic development and tumor suppression</span>. <i>EMBO Rep.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">500</span>– <span class="NLM_lpage">506</span>, <span class="refDoi"> DOI: 10.1093/embo-reports/kvd129</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1093%2Fembo-reports%2Fkvd129" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=11263494" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjt1ejtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2000&pages=500-506&author=A.+Klochendler-Yeivinauthor=L.+Fietteauthor=J.+Barraauthor=C.+Muchardtauthor=C.+Babinetauthor=M.+Yaniv&title=The+murine+SNF5%2FINI1+chromatin+remodeling+factor+is+essential+for+embryonic+development+and+tumor+suppression&doi=10.1093%2Fembo-reports%2Fkvd129"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">The murine SNF5/INI1 chromatin remodeling factor is essential for embryonic development and tumor suppression</span></div><div class="casAuthors">Klochendler-Yeivin, Agnes; Fiette, Laurence; Barra, Jaqueline; Muchardt, Christian; Babinet, Charles; Yaniv, Moshe</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Reports</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">500-506</span>CODEN:
                <span class="NLM_cas:coden">ERMEAX</span>;
        ISSN:<span class="NLM_cas:issn">1469-221X</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The assembly of eukaryotic DNA into nucleosomes and derived higher order structures constitutes a barrier for transcription, replication and repair.  A no. of chromatin remodeling complexes, as well as histone acetylation, were shown to facilitate gene activation.  To investigate the function of two closely related mammalian SWI/SNF complexes in vivo, we inactivated the murine SNF5/INI1 gene, a common subunit of these two complexes.  Mice lacking SNF5 protein stop developing at the peri-implantation stage, showing that the SWI/SNF complex is essential for early development and viability of early embryonic cells.  Furthermore, heterozygous mice develop nervous system and soft tissue sarcomas.  In these tumors the wild-type allele was lost, providing further evidence that SNF5 functions as a tumor suppressor gene in certain cell types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2fPp6F-3PPLVg90H21EOLACvtfcHk0lj5JwMCDg6FhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjt1ejtLc%253D&md5=876ba7fa7fffebb2e53bc359f8b3cffe</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1093%2Fembo-reports%2Fkvd129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fembo-reports%252Fkvd129%26sid%3Dliteratum%253Aachs%26aulast%3DKlochendler-Yeivin%26aufirst%3DA.%26aulast%3DFiette%26aufirst%3DL.%26aulast%3DBarra%26aufirst%3DJ.%26aulast%3DMuchardt%26aufirst%3DC.%26aulast%3DBabinet%26aufirst%3DC.%26aulast%3DYaniv%26aufirst%3DM.%26atitle%3DThe%2520murine%2520SNF5%252FINI1%2520chromatin%2520remodeling%2520factor%2520is%2520essential%2520for%2520embryonic%2520development%2520and%2520tumor%2520suppression%26jtitle%3DEMBO%2520Rep.%26date%3D2000%26volume%3D1%26spage%3D500%26epage%3D506%26doi%3D10.1093%2Fembo-reports%2Fkvd129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siersbæk, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandrup, S.</span></span> <span> </span><span class="NLM_article-title">Transcriptional networks and chromatin remodeling controlling adipogenesis</span>. <i>Trends Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">56</span>– <span class="NLM_lpage">64</span>, <span class="refDoi"> DOI: 10.1016/j.tem.2011.10.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2Fj.tem.2011.10.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=22079269" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2012&pages=56-64&author=R.+Siersb%C3%A6kauthor=R.+Nielsenauthor=S.+Mandrup&title=Transcriptional+networks+and+chromatin+remodeling+controlling+adipogenesis&doi=10.1016%2Fj.tem.2011.10.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional networks and chromatin remodeling controlling adipogenesis</span></div><div class="casAuthors">Siersbaek, Rasmus; Nielsen, Ronni; Mandrup, Susanne</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">56-64</span>CODEN:
                <span class="NLM_cas:coden">TENME4</span>;
        ISSN:<span class="NLM_cas:issn">1043-2760</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Adipocyte differentiation is tightly controlled by a transcriptional cascade, which directs the extensive reprogramming of gene expression required to convert fibroblast-like precursor cells into mature lipid-laden adipocytes.  Recent global analyses of transcription factor binding and chromatin remodeling have revealed snapshots' of this cascade and the chromatin landscape at specific time-points of differentiation.  These studies demonstrate that multiple adipogenic transcription factors co-occupy hotspots characterized by an open chromatin structure and specific epigenetic modifications.  Such transcription factor hotspots are likely to represent key signaling nodes which integrate multiple adipogenic signals at specific chromatin sites, thereby facilitating coordinated action on gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwAmbNvL0wPbVg90H21EOLACvtfcHk0ljsxlIfH9gF4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCntLo%253D&md5=610d28862d9a593d7581eb9c64986fb1</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.tem.2011.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tem.2011.10.001%26sid%3Dliteratum%253Aachs%26aulast%3DSiersb%25C3%25A6k%26aufirst%3DR.%26aulast%3DNielsen%26aufirst%3DR.%26aulast%3DMandrup%26aufirst%3DS.%26atitle%3DTranscriptional%2520networks%2520and%2520chromatin%2520remodeling%2520controlling%2520adipogenesis%26jtitle%3DTrends%2520Endocrinol.%2520Metab.%26date%3D2012%26volume%3D23%26spage%3D56%26epage%3D64%26doi%3D10.1016%2Fj.tem.2011.10.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowenz-Leutz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leutz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nerlov, C.</span></span> <span> </span><span class="NLM_article-title">Cooperation between C/EBPalpha TBP/TFIIB and SWI/SNF recruiting domains is required for adipocyte differentiation</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">3208</span>– <span class="NLM_lpage">3216</span>, <span class="refDoi"> DOI: 10.1101/gad.209901</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1101%2Fgad.209901" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=11731483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD3MXptVKmsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2001&pages=3208-3216&author=T.+A.+Pedersenauthor=E.+Kowenz-Leutzauthor=A.+Leutzauthor=C.+Nerlov&title=Cooperation+between+C%2FEBPalpha+TBP%2FTFIIB+and+SWI%2FSNF+recruiting+domains+is+required+for+adipocyte+differentiation&doi=10.1101%2Fgad.209901"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Cooperation between C/EBPα TBP/TFIIB and SWI/SNF recruiting domains is required for adipocyte differentiation</span></div><div class="casAuthors">Pedersen, Thomas Askov; Kowenz-Leutz, Elisabeth; Leutz, Achim; Nerlov, Claus</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3208-3216</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Chromatin remodeling is an important step in promoter activation during cellular lineage commitment and differentiation.  We show that the ability of the C/EBPα transcription factor to direct adipocyte differentiation of uncommitted fibroblast precursors and to activate SWI/SNF-dependent myeloid-specific genes depends on a domain, C/EBPα transactivation element III (TE-III), that binds the SWI/SNF chromatin remodeling complex.  TE-III collaborates with C/EBPα TBP/TFIIB interaction motifs during induction of adipogenesis and adipocyte-specific gene expression.  These results indicate that C/EBPα acts as a lineage-instructive transcription factor through SWI/SNF-dependent modification of the chromatin structure of lineage-specific genes, followed by direct promoter activation via recruitment of the basal transcription-initiation complex, and provide a mechanism by which C/EBPα can mediate differentiation along multiple cellular lineages.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsI37_rDsNbrVg90H21EOLACvtfcHk0ljsxlIfH9gF4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXptVKmsLs%253D&md5=62fcbfc140893f5e34eb4c10acb38926</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1101%2Fgad.209901&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.209901%26sid%3Dliteratum%253Aachs%26aulast%3DPedersen%26aufirst%3DT.%2BA.%26aulast%3DKowenz-Leutz%26aufirst%3DE.%26aulast%3DLeutz%26aufirst%3DA.%26aulast%3DNerlov%26aufirst%3DC.%26atitle%3DCooperation%2520between%2520C%252FEBPalpha%2520TBP%252FTFIIB%2520and%2520SWI%252FSNF%2520recruiting%2520domains%2520is%2520required%2520for%2520adipocyte%2520differentiation%26jtitle%3DGenes%2520Dev.%26date%3D2001%26volume%3D15%26spage%3D3208%26epage%3D3216%26doi%3D10.1101%2Fgad.209901" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbalzano, A. N.</span></span> <span> </span><span class="NLM_article-title">Temporal recruitment of transcription factors and SWI/SNF chromatin-remodeling enzymes during adipogenic induction of the peroxisome proliferator-activated receptor gamma nuclear hormone receptor</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">4651</span>– <span class="NLM_lpage">4663</span>, <span class="refDoi"> DOI: 10.1128/mcb.24.11.4651-4663.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1128%2FMCB.24.11.4651-4663.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=15143161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVyhsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=4651-4663&author=N.+Salmaauthor=H.+Xiaoauthor=E.+Muellerauthor=A.+N.+Imbalzano&title=Temporal+recruitment+of+transcription+factors+and+SWI%2FSNF+chromatin-remodeling+enzymes+during+adipogenic+induction+of+the+peroxisome+proliferator-activated+receptor+gamma+nuclear+hormone+receptor&doi=10.1128%2Fmcb.24.11.4651-4663.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Temporal recruitment of transcription factors and SWI/SNF chromatin-remodeling enzymes during adipogenic induction of the peroxisome proliferator-activated receptor γ nuclear hormone receptor</span></div><div class="casAuthors">Salma, Nunciada; Xiao, Hengyi; Mueller, Elisabetta; Imbalzano, Anthony N.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4651-4663</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The peroxisome proliferator-activated receptor gamma (PPARγ) regulates adipogenesis, lipid metab., and glucose homeostasis, and roles have emerged for this receptor in the pathogenesis and treatment of diabetes, atherosclerosis, and cancer.  We report here that induction of the PPARγ activator and adipogenesis forced by overexpression of adipogenic regulatory proteins is blocked upon expression of dominant-neg. BRG1 or hBRM, the ATPase subunits of distinct SWI/SNF chromatin-remodeling enzymes.  We demonstrate that histone hyperacetylation and the binding of C/EBP activators, polymerase II (Pol II), and general transcription factors (GTFs) initially occurred at the inducible PPARγ2 promoter in the absence of SWI/SNF function.  However, the polymerase and GTFs were subsequently lost from the promoter in cells expressing dominant-neg. SWI/SNF, explaining the inhibition of PPARγ2 expression.  To corroborate these data, we analyzed interactions at the PPARγ2 promoter in differentiating preadipocytes.  Changes in promoter structure, histone hyperacetylation, and binding of C/EBP activators, Pol II, and most GTFs preceded the interaction of SWI/SNF enzymes with the PPARγ2 promoter.  However, transcription of the PPARγ2 gene occurred only upon subsequent assocn. of SWI/SNF and TFIIH with the promoter.  Thus, induction of the PPARγ nuclear hormone receptor during adipogenesis requires SWI/SNF enzymes to facilitate preinitiation complex function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Nj3AMxjesbVg90H21EOLACvtfcHk0lj3BgeJwEgWRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVyhsrk%253D&md5=0e1d196bbbf840bd00f2c86151f80f11</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1128%2FMCB.24.11.4651-4663.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.24.11.4651-4663.2004%26sid%3Dliteratum%253Aachs%26aulast%3DSalma%26aufirst%3DN.%26aulast%3DXiao%26aufirst%3DH.%26aulast%3DMueller%26aufirst%3DE.%26aulast%3DImbalzano%26aufirst%3DA.%2BN.%26atitle%3DTemporal%2520recruitment%2520of%2520transcription%2520factors%2520and%2520SWI%252FSNF%2520chromatin-remodeling%2520enzymes%2520during%2520adipogenic%2520induction%2520of%2520the%2520peroxisome%2520proliferator-activated%2520receptor%2520gamma%2520nuclear%2520hormone%2520receptor%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D4651%26epage%3D4663%26doi%3D10.1128%2Fmcb.24.11.4651-4663.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mikkelsen, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gimble, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, E. D.</span></span> <span> </span><span class="NLM_article-title">Comparative epigenomic analysis of murine and human adipogenesis</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>143</i></span>,  <span class="NLM_fpage">156</span>– <span class="NLM_lpage">169</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2010.09.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2Fj.cell.2010.09.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=20887899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Cmu7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=143&publication_year=2010&pages=156-169&author=T.+S.+Mikkelsenauthor=Z.+Xuauthor=X.+Zhangauthor=L.+Wangauthor=J.+M.+Gimbleauthor=E.+S.+Landerauthor=E.+D.+Rosen&title=Comparative+epigenomic+analysis+of+murine+and+human+adipogenesis&doi=10.1016%2Fj.cell.2010.09.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative Epigenomic Analysis of Murine and Human Adipogenesis</span></div><div class="casAuthors">Mikkelsen, Tarjei S.; Xu, Zhao; Zhang, Xiaolan; Wang, Li; Gimble, Jeffrey M.; Lander, Eric S.; Rosen, Evan D.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">143</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">156-169</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We report the generation and comparative anal. of genome-wide chromatin state maps, PPARγ and CTCF localization maps, and gene expression profiles from murine and human models of adipogenesis.  The data provide high-resoln. views of chromatin remodeling during cellular differentiation and allow identification of thousands of putative preadipocyte- and adipocyte-specific cis-regulatory elements based on dynamic chromatin signatures.  We find that the specific locations of most such elements differ between the two models, including at orthologous loci with similar expression patterns.  Based on sequence anal. and reporter assays, we show that these differences are detd., in part, by evolutionary turnover of transcription factor motifs in the genome sequences and that this turnover may be facilitated by the presence of multiple distal regulatory elements at adipogenesis-dependent loci.  We also utilize the close relationship between open chromatin marks and transcription factor motifs to identify and validate PLZF and SRF as regulators of adipogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaVAJqwXqgl7Vg90H21EOLACvtfcHk0lj0eL7HOG9zhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Cmu7zK&md5=cbd32d847744e8c44275871ae0530864</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2010.09.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2010.09.006%26sid%3Dliteratum%253Aachs%26aulast%3DMikkelsen%26aufirst%3DT.%2BS.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DGimble%26aufirst%3DJ.%2BM.%26aulast%3DLander%26aufirst%3DE.%2BS.%26aulast%3DRosen%26aufirst%3DE.%2BD.%26atitle%3DComparative%2520epigenomic%2520analysis%2520of%2520murine%2520and%2520human%2520adipogenesis%26jtitle%3DCell%26date%3D2010%26volume%3D143%26spage%3D156%26epage%3D169%26doi%3D10.1016%2Fj.cell.2010.09.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lefterova, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haakonsson, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazar, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandrup, S.</span></span> <span> </span><span class="NLM_article-title">PPARγ and the global map of adipogenesis and beyond</span>. <i>Trends Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">302</span>, <span class="refDoi"> DOI: 10.1016/j.tem.2014.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2Fj.tem.2014.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=24793638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntFygtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=293-302&author=M.+I.+Lefterovaauthor=A.+K.+Haakonssonauthor=M.+A.+Lazarauthor=S.+Mandrup&title=PPAR%CE%B3+and+the+global+map+of+adipogenesis+and+beyond&doi=10.1016%2Fj.tem.2014.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">PPARγ and the global map of adipogenesis and beyond</span></div><div class="casAuthors">Lefterova, Martina I.; Haakonsson, Anders K.; Lazar, Mitchell A.; Mandrup, Susanne</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">293-302</span>CODEN:
                <span class="NLM_cas:coden">TENME4</span>;
        ISSN:<span class="NLM_cas:issn">1043-2760</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Peroxisome proliferator-activated receptor γ (PPARγ) is a member of the nuclear receptor (NR) superfamily of ligand-dependent transcription factors (TFs) and function as a master regulator of adipocyte differentiation and metab.  We review recent breakthroughs in the understanding of PPARγ gene regulation and function in the chromatin context.  It is now clear that multiple TFs team up to induce PPARγ during adipogenesis, and that other TFs cooperate with PPARγ to ensure adipocyte-specific genomic binding and function.  We discuss how this differs in other PPARγ-expressing cells such as macrophages and how these genome-wide mechanisms are preserved across species despite modest conservation of specific binding sites.  These emerging considerations inform our understanding of PPARγ function as well as of adipocyte development and physiol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiop59iTWwo7Vg90H21EOLACvtfcHk0lj0eL7HOG9zhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntFygtbk%253D&md5=7dfcb1c26d4232969257146f5f7b3636</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.tem.2014.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tem.2014.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DLefterova%26aufirst%3DM.%2BI.%26aulast%3DHaakonsson%26aufirst%3DA.%2BK.%26aulast%3DLazar%26aufirst%3DM.%2BA.%26aulast%3DMandrup%26aufirst%3DS.%26atitle%3DPPAR%25CE%25B3%2520and%2520the%2520global%2520map%2520of%2520adipogenesis%2520and%2520beyond%26jtitle%3DTrends%2520Endocrinol.%2520Metab.%26date%3D2014%26volume%3D25%26spage%3D293%26epage%3D302%26doi%3D10.1016%2Fj.tem.2014.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lefterova, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steger, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schupp, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schug, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cristancho, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoeckert, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazar, M. A.</span></span> <span> </span><span class="NLM_article-title">PPARgamma and C/EBP factors orchestrate adipocyte biology via adjacent binding on a genome-wide scale</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">2941</span>– <span class="NLM_lpage">2952</span>, <span class="refDoi"> DOI: 10.1101/gad.1709008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1101%2Fgad.1709008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=18981473" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOqtbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=2941-2952&author=M.+I.+Lefterovaauthor=Y.+Zhangauthor=D.+J.+Stegerauthor=M.+Schuppauthor=J.+Schugauthor=A.+Cristanchoauthor=D.+Fengauthor=D.+Zhuoauthor=C.+J.+Stoeckertauthor=X.+S.+Liuauthor=M.+A.+Lazar&title=PPARgamma+and+C%2FEBP+factors+orchestrate+adipocyte+biology+via+adjacent+binding+on+a+genome-wide+scale&doi=10.1101%2Fgad.1709008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">PPARγ and C/EBP factors orchestrate adipocyte biology via adjacent binding on a genome-wide scale</span></div><div class="casAuthors">Lefterova, Martina I.; Zhang, Yong; Steger, David J.; Schupp, Michael; Schug, Jonathan; Cristancho, Ana; Feng, Dan; Zhuo, David; Stoeckert, Christian J., Jr.; Liu, X. Shirley; Lazar, Mitchell A.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2941-2952</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Peroxisome proliferator-activated receptor γ(PPARγ), a nuclear receptor and the target of anti-diabetic thiazolinedione drugs, is known as the master regulator of adipocyte biol.  Although it regulates hundreds of adipocyte genes, PPARγ binding to endogenous genes has rarely been demonstrated.  Here, utilizing chromatin immunopptn. (ChIP) coupled with whole genome tiling arrays, we identified 5299 genomic regions of PPARγ binding in mouse 3T3-L1 adipocytes.  The consensus PPARγ/RXRα "DR-1"-binding motif was found at most of the sites, and ChIP for RXRα showed colocalization at nearly all locations tested.  Bioinformatics anal. also revealed CCAAT/enhancer-binding protein (C/EBP)-binding motifs in the vicinity of most PPARγ-binding sites, and genome-wide anal. of C/EBPα binding demonstrated that it localized to 3350 of the locations bound by PPARγ.  Importantly, most genes induced in adipogenesis were bound by both PPARγ and C/EBPα, while very few were PPARγ-specific.  C/EBPβ also plays a role at many of these genes, such that both C/EBPα and β are required along with PPARγ for robust adipocyte-specific gene expression.  Thus, PPARγ and C/EBP factors cooperatively orchestrate adipocyte biol. by adjacent binding on an unanticipated scale.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWnMWvt4R6krVg90H21EOLACvtfcHk0lgDHtQ_ziBxww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOqtbnF&md5=566a2a51ce6b20a9722063b0e3e2952f</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1101%2Fgad.1709008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1709008%26sid%3Dliteratum%253Aachs%26aulast%3DLefterova%26aufirst%3DM.%2BI.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DSteger%26aufirst%3DD.%2BJ.%26aulast%3DSchupp%26aufirst%3DM.%26aulast%3DSchug%26aufirst%3DJ.%26aulast%3DCristancho%26aufirst%3DA.%26aulast%3DFeng%26aufirst%3DD.%26aulast%3DZhuo%26aufirst%3DD.%26aulast%3DStoeckert%26aufirst%3DC.%2BJ.%26aulast%3DLiu%26aufirst%3DX.%2BS.%26aulast%3DLazar%26aufirst%3DM.%2BA.%26atitle%3DPPARgamma%2520and%2520C%252FEBP%2520factors%2520orchestrate%2520adipocyte%2520biology%2520via%2520adjacent%2520binding%2520on%2520a%2520genome-wide%2520scale%26jtitle%3DGenes%2520Dev.%26date%3D2008%26volume%3D22%26spage%3D2941%26epage%3D2952%26doi%3D10.1101%2Fgad.1709008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alver, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manchester, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haswell, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, C. W. M.</span></span> <span> </span><span class="NLM_article-title">The SWI/SNF chromatin remodelling complex is required for maintenance of lineage specific enhancers</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">14648</span>, <span class="refDoi"> DOI: 10.1038/ncomms14648</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fncomms14648" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=28262751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A280%3ADC%252BC1czjtFeitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=14648&author=B.+H.+Alverauthor=K.+H.+Kimauthor=P.+Luauthor=X.+Wangauthor=H.+E.+Manchesterauthor=W.+Wangauthor=J.+R.+Haswellauthor=P.+J.+Parkauthor=C.+W.+M.+Roberts&title=The+SWI%2FSNF+chromatin+remodelling+complex+is+required+for+maintenance+of+lineage+specific+enhancers&doi=10.1038%2Fncomms14648"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">The SWI/SNF chromatin remodelling complex is required for maintenance of lineage specific enhancers</span></div><div class="casAuthors">Alver Burak H; Park Peter J; Kim Kimberly H; Lu Ping; Wang Xiaofeng; Manchester Haley E; Wang Weishan; Haswell Jeffrey R; Roberts Charles W M; Kim Kimberly H; Lu Ping; Wang Xiaofeng; Manchester Haley E; Wang Weishan; Haswell Jeffrey R; Roberts Charles W M; Kim Kimberly H; Lu Ping; Wang Xiaofeng; Manchester Haley E; Wang Weishan; Haswell Jeffrey R; Roberts Charles W M; Roberts Charles W M</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">14648</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Genes encoding subunits of SWI/SNF (BAF) chromatin remodelling complexes are collectively altered in over 20% of human malignancies, but the mechanisms by which these complexes alter chromatin to modulate transcription and cell fate are poorly understood.  Utilizing mouse embryonic fibroblast and cancer cell line models, here we show via ChIP-seq and biochemical assays that SWI/SNF complexes are preferentially targeted to distal lineage specific enhancers and interact with p300 to modulate histone H3 lysine 27 acetylation.  We identify a greater requirement for SWI/SNF at typical enhancers than at most super-enhancers and at enhancers in untranscribed regions than in transcribed regions.  Our data further demonstrate that SWI/SNF-dependent distal enhancers are essential for controlling expression of genes linked to developmental processes.  Our findings thus establish SWI/SNF complexes as regulators of the enhancer landscape and provide insight into the roles of SWI/SNF in cellular fate control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTe-gGPhesmNRdUSIfHIn4FfW6udTcc2eYAx1pkS7oj07ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1czjtFeitA%253D%253D&md5=43052bae8dacb83273af786fbf5c6a1f</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2Fncomms14648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms14648%26sid%3Dliteratum%253Aachs%26aulast%3DAlver%26aufirst%3DB.%2BH.%26aulast%3DKim%26aufirst%3DK.%2BH.%26aulast%3DLu%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DManchester%26aufirst%3DH.%2BE.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DHaswell%26aufirst%3DJ.%2BR.%26aulast%3DPark%26aufirst%3DP.%2BJ.%26aulast%3DRoberts%26aufirst%3DC.%2BW.%2BM.%26atitle%3DThe%2520SWI%252FSNF%2520chromatin%2520remodelling%2520complex%2520is%2520required%2520for%2520maintenance%2520of%2520lineage%2520specific%2520enhancers%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D14648%26doi%3D10.1038%2Fncomms14648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buser-Doepner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederiksen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimshaw, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hepworth, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mader, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller-Fahrnow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Hagan, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundström, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toledo-Sherman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walpole, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuercher, W. J.</span></span> <span> </span><span class="NLM_article-title">The promise and peril of chemical probes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">536</span>– <span class="NLM_lpage">541</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fnchembio.1867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=26196764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=536-541&author=C.+H.+Arrowsmithauthor=J.+E.+Audiaauthor=C.+Austinauthor=J.+Baellauthor=J.+Bennettauthor=J.+Blaggauthor=C.+Bountraauthor=P.+E.+Brennanauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=C.+Buser-Doepnerauthor=R.+M.+Campbellauthor=A.+J.+Carterauthor=P.+Cohenauthor=R.+A.+Copelandauthor=B.+Cravattauthor=J.+L.+Dahlinauthor=D.+Dhanakauthor=A.+M.+Edwardsauthor=M.+Frederiksenauthor=S.+V.+Fryeauthor=N.+Grayauthor=C.+E.+Grimshawauthor=D.+Hepworthauthor=T.+Howeauthor=K.+V.+M.+Huberauthor=J.+Jinauthor=S.+Knappauthor=J.+D.+Kotzauthor=R.+G.+Krugerauthor=D.+Loweauthor=M.+M.+Maderauthor=B.+Marsdenauthor=A.+Mueller-Fahrnowauthor=S.+M%C3%BCllerauthor=R.+C.+O%E2%80%99Haganauthor=J.+P.+Overingtonauthor=D.+R.+Owenauthor=S.+H.+Rosenbergauthor=R.+Rossauthor=B.+Rothauthor=M.+Schapiraauthor=S.+L.+Schreiberauthor=B.+Shoichetauthor=M.+Sundstr%C3%B6mauthor=G.+Superti-Furgaauthor=J.+Tauntonauthor=L.+Toledo-Shermanauthor=C.+Walpoleauthor=M.+A.+Waltersauthor=T.+M.+Willsonauthor=P.+Workmanauthor=R.+N.+Youngauthor=W.+J.+Zuercher&title=The+promise+and+peril+of+chemical+probes&doi=10.1038%2Fnchembio.1867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">The promise and peril of chemical probes</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Audia, James E.; Austin, Christopher; Baell, Jonathan; Bennett, Jonathan; Blagg, Julian; Bountra, Chas; Brennan, Paul E.; Brown, Peter J.; Bunnage, Mark E.; Buser-Doepner, Carolyn; Campbell, Robert M.; Carter, Adrian J.; Cohen, Philip; Copeland, Robert A.; Cravatt, Ben; Dahlin, Jayme L.; Dhanak, Dashyant; Edwards, Aled M.; Frye, Stephen V.; Gray, Nathanael; Grimshaw, Charles E.; Hepworth, David; Howe, Trevor; Huber, Kilian V. M.; Jin, Jian; Knapp, Stefan; Kotz, Joanne D.; Kruger, Ryan G.; Lowe, Derek; Mader, Mary M.; Marsden, Brian; Mueller-Fahrnow, Anke; Muller, Susanne; O'Hagan, Ronan C.; Overington, John P.; Owen, Dafydd R.; Rosenberg, Saul H.; Roth, Brian; Ross, Ruth; Schapira, Matthieu; Schreiber, Stuart L.; Shoichet, Brian; Sundstrom, Michael; Superti-Furga, Giulio; Taunton, Jack; Toledo-Sherman, Leticia; Walpole, Chris; Walters, Michael A.; Willson, Timothy M.; Workman, Paul; Young, Robert N.; Zuercher, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">536-541</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. probes are powerful reagents with increasing impacts on biomedical research.  However, probes of poor quality or that are used incorrectly generate misleading results.  To help address these shortcomings, we will create a community-driven wiki resource to improve quality and convey current best practice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-li4dxq487rVg90H21EOLACvtfcHk0lgjG7QX758Cvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO&md5=d8c7a7af94921ab4cdcc871487cceadd</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1867%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DAustin%26aufirst%3DC.%26aulast%3DBaell%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DBuser-Doepner%26aufirst%3DC.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DCarter%26aufirst%3DA.%2BJ.%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DCravatt%26aufirst%3DB.%26aulast%3DDahlin%26aufirst%3DJ.%2BL.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26aulast%3DFrederiksen%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DGrimshaw%26aufirst%3DC.%2BE.%26aulast%3DHepworth%26aufirst%3DD.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKotz%26aufirst%3DJ.%2BD.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DLowe%26aufirst%3DD.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DMueller-Fahrnow%26aufirst%3DA.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Hagan%26aufirst%3DR.%2BC.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DRoss%26aufirst%3DR.%26aulast%3DRoth%26aufirst%3DB.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%26aulast%3DSundstr%25C3%25B6m%26aufirst%3DM.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DToledo-Sherman%26aufirst%3DL.%26aulast%3DWalpole%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DR.%2BN.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26atitle%3DThe%2520promise%2520and%2520peril%2520of%2520chemical%2520probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D536%26epage%3D541%26doi%3D10.1038%2Fnchembio.1867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chekler, E. L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span> <span> </span><span class="NLM_article-title">Target validation using chemical probes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">195</span>– <span class="NLM_lpage">199</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fnchembio.1197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=23508172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFagsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=195-199&author=M.+E.+Bunnageauthor=E.+L.+P.+Cheklerauthor=L.+H.+Jones&title=Target+validation+using+chemical+probes&doi=10.1038%2Fnchembio.1197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Target validation using chemical probes</span></div><div class="casAuthors">Bunnage, Mark E.; Chekler, Eugene L. Piatnitski; Jones, Lyn H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">195-199</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review on the fully profiled chem. probes essential to support the unbiased interpretation of biol. expts. necessary for rigorous preclin. target validation.  We believe that by developing a 'chem. probe tool kit', and a framework for its use, chem. biol. can have a more central role in identifying targets of potential relevance to disease, avoiding many of the biases that complicate target validation as practiced currently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLwRTT4pXGxbVg90H21EOLACvtfcHk0lgjmFbVkUfAmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFagsr4%253D&md5=af0c362882191de96a8e8698ff2c9ce8</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1197%26sid%3Dliteratum%253Aachs%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DChekler%26aufirst%3DE.%2BL.%2BP.%26aulast%3DJones%26aufirst%3DL.%2BH.%26atitle%3DTarget%2520validation%2520using%2520chemical%2520probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D195%26epage%3D199%26doi%3D10.1038%2Fnchembio.1197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aldeghi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodkin, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Essex, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggin, P. C.</span></span> <span> </span><span class="NLM_article-title">Large-scale analysis of water stability in bromodomain binding pockets with grand canonical Monte Carlo</span>. <i>Commun. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">19</span>, <span class="refDoi"> DOI: 10.1038/s42004-018-0019-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fs42004-018-0019-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=29863194" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2018&pages=19&author=M.+Aldeghiauthor=G.+A.+Rossauthor=M.+J.+Bodkinauthor=J.+W.+Essexauthor=S.+Knappauthor=P.+C.+Biggin&title=Large-scale+analysis+of+water+stability+in+bromodomain+binding+pockets+with+grand+canonical+Monte+Carlo&doi=10.1038%2Fs42004-018-0019-x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fs42004-018-0019-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42004-018-0019-x%26sid%3Dliteratum%253Aachs%26aulast%3DAldeghi%26aufirst%3DM.%26aulast%3DRoss%26aufirst%3DG.%2BA.%26aulast%3DBodkin%26aufirst%3DM.%2BJ.%26aulast%3DEssex%26aufirst%3DJ.%2BW.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBiggin%26aufirst%3DP.%2BC.%26atitle%3DLarge-scale%2520analysis%2520of%2520water%2520stability%2520in%2520bromodomain%2520binding%2520pockets%2520with%2520grand%2520canonical%2520Monte%2520Carlo%26jtitle%3DCommun.%2520Chem.%26date%3D2018%26volume%3D1%26spage%3D19%26doi%3D10.1038%2Fs42004-018-0019-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niesen, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Kinase inhibitor selectivity profiling using differential scanning fluorimetry</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>795</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">118</span>, <span class="refDoi"> DOI: 10.1007/978-1-61779-337-0_7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1007%2F978-1-61779-337-0_7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=21960218" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlersLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=795&publication_year=2012&pages=109-118&author=O.+Fedorovauthor=F.+H.+Niesenauthor=S.+Knapp&title=Kinase+inhibitor+selectivity+profiling+using+differential+scanning+fluorimetry&doi=10.1007%2F978-1-61779-337-0_7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitor selectivity profiling using differential scanning fluorimetry</span></div><div class="casAuthors">Fedorov, Oleg; Niesen, Frank H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">795</span>
        (<span class="NLM_cas:issue">Kinase Inhibitors</span>),
    <span class="NLM_cas:pages">109-118</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Fast, robust, and inexpensive screening methods are the heart of drug discovery processes.  Moreover, it is useful to have access to several established assay formats, for validation purposes.  If a targeted protein is an enzyme, the logical and widely used approach is the direct measurement of the effect of the added ligands on its activity.  A variety of enzymic assay formats have been successfully applied for inhibitor screening of protein kinases.  However, enzymic assays require an active enzyme with a known substrate and often time-consuming assay optimization.  Several alternative approaches have been recently developed that detect binding of ligands to proteins.  This chapter overviews and provides the exptl. protocol of the successful application of differential scanning fluorimetry (DSF) in our lab. for fast and robust screening of medium-sized (<10,000) inhibitor libraries.  DSF monitors the thermal stabilization of the native protein structure upon ligand binding.  It allows selectivity profiling of any protein kinase without prior knowledge of either substrate or activity of the kinase under investigation.  Comparative studies revealed that generated data is highly reproducible and correlates well with the results from other ligand binding methodologies, direct binding consts. as well as enzymic assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJ3JcrKLX6AbVg90H21EOLACvtfcHk0lgjmFbVkUfAmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlersLvM&md5=ce44a4a0c4e28e61b865da2301df6db7</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-337-0_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-337-0_7%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DKinase%2520inhibitor%2520selectivity%2520profiling%2520using%2520differential%2520scanning%2520fluorimetry%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2012%26volume%3D795%26spage%3D109%26epage%3D118%26doi%3D10.1007%2F978-1-61779-337-0_7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Mahony, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasater, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">312</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fnchembio.1471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=24584101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=305-312&author=P.+Ciceriauthor=S.+M%C3%BCllerauthor=A.+O%E2%80%99Mahonyauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=J.+P.+Huntauthor=E.+A.+Lasaterauthor=G.+Pallaresauthor=S.+Picaudauthor=C.+Wellsauthor=S.+Martinauthor=L.+M.+Wodickaauthor=N.+P.+Shahauthor=D.+K.+Treiberauthor=S.+Knapp&title=Dual+kinase-bromodomain+inhibitors+for+rationally+designed+polypharmacology&doi=10.1038%2Fnchembio.1471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span></div><div class="casAuthors">Ciceri, Pietro; Mueller, Susanne; O'Mahony, Alison; Fedorov, Oleg; Filippakopoulos, Panagis; Hunt, Jeremy P.; Lasater, Elisabeth A.; Pallares, Gabriel; Picaud, Sarah; Wells, Christopher; Martin, Sarah; Wodicka, Lisa M.; Shah, Neil P.; Treiber, Daniel K.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">305-312</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Concomitant inhibition of multiple cancer-driving kinases is an established strategy to improve the durability of clin. responses to targeted therapies.  The difficulty of discovering kinase inhibitors with an appropriate multitarget profile has, however, necessitated the application of combination therapies, which can pose major clin. development challenges.  Epigenetic reader domains of the bromodomain family have recently emerged as new targets for cancer therapy.  Here the authors report that several clin. kinase inhibitors also inhibit bromodomains with therapeutically relevant potencies and are best classified as dual kinase-bromodomain inhibitors.  Nanomolar activity on BRD4 by BI-2536 and TG-101348, which are clin. PLK1 and JAK2-FLT3 kinase inhibitors, resp., is particularly noteworthy as these combinations of activities on independent oncogenic pathways exemplify a new strategy for rational single-agent polypharmacol. targeting.  Furthermore, structure-activity relationships and co-crystal structures identify design features that enable a general platform for the rational design of dual kinase-bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotPedDED4FJLVg90H21EOLACvtfcHk0lhiqEpYmDDRiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D&md5=a00117db2648ea38dd03750faf3b3cb9</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1471%26sid%3Dliteratum%253Aachs%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLasater%26aufirst%3DE.%2BA.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DShah%26aufirst%3DN.%2BP.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DDual%2520kinase-bromodomain%2520inhibitors%2520for%2520rationally%2520designed%2520polypharmacology%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D305%26epage%3D312%26doi%3D10.1038%2Fnchembio.1471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gallenkamp, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelato, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haendler, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinmann, H.</span></span> <span> </span><span class="NLM_article-title">Bromodomains and their pharmacological inhibitors</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">438</span>– <span class="NLM_lpage">464</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201300434</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1002%2Fcmdc.201300434" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=24497428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsl2gtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=438-464&author=D.+Gallenkampauthor=K.+A.+Gelatoauthor=B.+Haendlerauthor=H.+Weinmann&title=Bromodomains+and+their+pharmacological+inhibitors&doi=10.1002%2Fcmdc.201300434"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomains and Their Pharmacological Inhibitors</span></div><div class="casAuthors">Gallenkamp, Daniel; Gelato, Kathy A.; Haendler, Bernard; Weinmann, Hilmar</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">438-464</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Over 60 bromodomains belonging to proteins with very different functions have been identified in humans.  Several of them interact with acetylated lysine residues, leading to the recruitment and stabilization of protein complexes.  The bromodomain and extra-terminal domain (BET) proteins contain tandem bromodomains which bind to acetylated histones and are thereby implicated in a no. of DNA-centered processes, including the regulation of gene expression.  The recent identification of inhibitors of BET and non-BET bromodomains is one of the few examples in which effective blockade of a protein-protein interaction can be achieved with a small mol.  This has led to major strides in the understanding of the function of bromodomain-contg. proteins and their involvement in diseases such as cancer and inflammation.  Indeed, BET bromodomain inhibitors are now being clin. evaluated for the treatment of hematol. tumors and have also been tested in clin. trials for the relatively rare BRD-NUT midline carcinoma.  This review gives an overview of the newest developments in the field, with a focus on the biol. of selected bromodomain proteins on the one hand, and on reported pharmacol. inhibitors on the other, including recent examples from the patent literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon-8_ymytCPbVg90H21EOLACvtfcHk0lhiqEpYmDDRiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsl2gtL4%253D&md5=2ce7c6b5c37001fb8fb901fe04aeb606</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300434%26sid%3Dliteratum%253Aachs%26aulast%3DGallenkamp%26aufirst%3DD.%26aulast%3DGelato%26aufirst%3DK.%2BA.%26aulast%3DHaendler%26aufirst%3DB.%26aulast%3DWeinmann%26aufirst%3DH.%26atitle%3DBromodomains%2520and%2520their%2520pharmacological%2520inhibitors%26jtitle%3DChemMedChem%26date%3D2014%26volume%3D9%26spage%3D438%26epage%3D464%26doi%3D10.1002%2Fcmdc.201300434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span> <span> </span><span class="NLM_article-title">Protein dynamics and structural waters in bromodomains</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0186570</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0186570</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1371%2Fjournal.pone.0186570" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=29077715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFGms7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=X.+Zhangauthor=K.+Chenauthor=Y.-D.+Wuauthor=O.+Wiest&title=Protein+dynamics+and+structural+waters+in+bromodomains&doi=10.1371%2Fjournal.pone.0186570"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Protein dynamics and structural waters in bromodomains</span></div><div class="casAuthors">Zhang, Xiaoxiao; Chen, Kai; Wu, Yun-Dong; Wiest, Olaf</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e0186570/1-e0186570/16</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Bromodomains are epigenetic readers of acetylated lysines that are integral parts of histone tails.  The 61 bromodomains in humans are structurally highly conserved but specifically bind to widely varying recognition motifs, suggesting that dynamic rather than static factors are responsible for recognition selectivity.  To test this hypothesis, the dynamics of the binding sites and structural water mols. of four bromodomains (ATAD2, BAZ2B, BRD2(1) and CREBBP) representing four different subtypes is studied with 1 μs MD simulations using the RSFF2 force field.  The different dynamics of the ZA-loops and BC-loops between the four bromodomains leads to distinct patterns for the opening and closing of the binding pocket.  This in turn dets. the structural and energetic properties of the structural waters in the binding pocket, suggesting that these waters are not only important for the recognition itself, as has been proposed previously, but also contribute to the selectivity of different bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdqzaoNSjS-7Vg90H21EOLACvtfcHk0lgEeCe9IuOLkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFGms7bN&md5=3ed02a6025d67e5c0e8b511c3dc6756e</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0186570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0186570%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DWu%26aufirst%3DY.-D.%26aulast%3DWiest%26aufirst%3DO.%26atitle%3DProtein%2520dynamics%2520and%2520structural%2520waters%2520in%2520bromodomains%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0186570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominy, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feeney, P. J.</span></span> <span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1016/s0169-409x(00)00129-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2Fs0169-409x%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2Fs0169-409x%2800%2900129-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0lgkS5kMP7fsAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fs0169-409x%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0169-409x%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D3%26epage%3D26%26doi%3D10.1016%2Fs0169-409x%2800%2900129-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span> <span> </span><span class="NLM_article-title">The chromatin-remodeling enzyme BRG1 promotes colon cancer progression via positive regulation of WNT3A</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">86051</span>– <span class="NLM_lpage">86063</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.13326</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.18632%2Foncotarget.13326" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=27852072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A280%3ADC%252BC2snmvV2ltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=86051-86063&author=S.+Linauthor=T.+Jiangauthor=L.+Yeauthor=Z.+Hanauthor=Y.+Liuauthor=C.+Liuauthor=C.+Yuanauthor=S.+Zhaoauthor=J.+Chenauthor=J.+Wangauthor=H.+Tangauthor=S.+Luauthor=L.+Yangauthor=X.+Wangauthor=D.+Yanauthor=Z.+Pengauthor=J.+Fan&title=The+chromatin-remodeling+enzyme+BRG1+promotes+colon+cancer+progression+via+positive+regulation+of+WNT3A&doi=10.18632%2Foncotarget.13326"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">The chromatin-remodeling enzyme BRG1 promotes colon cancer progression via positive regulation of WNT3A</span></div><div class="casAuthors">Lin Shengtao; Ye Ling; Liu Yuan; Liu Chenchen; Yuan Chenwei; Zhao Senlin; Chen Jian; Wang Jingtao; Wang Xiaoliang; Yan Dongwang; Peng Zhihai; Fan Junwei; Jiang Tao; Han Zhongbo; Tang Huamei; Lu Su; Yang Liguang</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">86051-86063</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In this study, we aimed to elucidate the clinical significance and underlying mechanisms of BRG1 in colon cancer.  In the clinical analysis, overexpression of BRG1 correlates with colon cancer progression in two cohorts (n = 191 and n = 75).  Kaplan-Meier survival analysis revealed that BRG1 is a prognosis predictor for overall survival (P < 0.001) and disease-free survival (P = 0.001).  Knocking down BRG1 expression significantly suppressed the proliferation and invasion in colon cancer cells.  The expression pattern of WNT3A is consistent with BRG1 in colon cancer tissues and WNT3A expression was inhibited in BRG1 knockdown cells.  In addition, restoring WNT3A expression rescues the inhibition of cell proliferation and invasion induced by BRG1.  In this study, we demonstrate that BRG1 may contribute to colon cancer progression through upregulating WNT3A expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSuPnIftE7c1GZyxfu64m4ffW6udTcc2eahnogSPkiUobntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2snmvV2ltQ%253D%253D&md5=701bd407c4a814b307e53704bf38076d</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.13326&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.13326%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DS.%26aulast%3DJiang%26aufirst%3DT.%26aulast%3DYe%26aufirst%3DL.%26aulast%3DHan%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DYuan%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DYan%26aufirst%3DD.%26aulast%3DPeng%26aufirst%3DZ.%26aulast%3DFan%26aufirst%3DJ.%26atitle%3DThe%2520chromatin-remodeling%2520enzyme%2520BRG1%2520promotes%2520colon%2520cancer%2520progression%2520via%2520positive%2520regulation%2520of%2520WNT3A%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D86051%26epage%3D86063%26doi%3D10.18632%2Foncotarget.13326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madany, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobson, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnabl, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Wijnen, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lian, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muthuswami, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbalzano, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nickerson, J. A.</span></span> <span> </span><span class="NLM_article-title">The BRG1 chromatin remodeling enzyme links cancer cell metabolism and proliferation</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">38270</span>– <span class="NLM_lpage">38281</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.9505</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.18632%2Foncotarget.9505" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=27223259" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A280%3ADC%252BC2s%252FjtFGkug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=38270-38281&author=Q.+Wuauthor=P.+Madanyauthor=J.+R.+Dobsonauthor=J.+M.+Schnablauthor=S.+Sharmaauthor=T.+C.+Smithauthor=A.+J.+van+Wijnenauthor=J.+L.+Steinauthor=J.+B.+Lianauthor=G.+S.+Steinauthor=R.+Muthuswamiauthor=A.+N.+Imbalzanoauthor=J.+A.+Nickerson&title=The+BRG1+chromatin+remodeling+enzyme+links+cancer+cell+metabolism+and+proliferation&doi=10.18632%2Foncotarget.9505"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">The BRG1 chromatin remodeling enzyme links cancer cell metabolism and proliferation</span></div><div class="casAuthors">Wu Qiong; Madany Pasil; Dobson Jason R; Schnabl Jake M; Smith Tara C; Imbalzano Anthony N; Nickerson Jeffrey A; Sharma Soni; Muthuswami Rohini; van Wijnen Andre J; Stein Janet L; Lian Jane B; Stein Gary S</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">38270-38281</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cancer cells reprogram cellular metabolism to meet the demands of growth.  Identification of the regulatory machinery that regulates cancer-specific metabolic changes may open new avenues for anti-cancer therapeutics.  The epigenetic regulator BRG1 is a catalytic ATPase for some mammalian SWI/SNF chromatin remodeling enzymes.  BRG1 is a well-characterized tumor suppressor in some human cancers, but is frequently overexpressed without mutation in other cancers, including breast cancer.  Here we demonstrate that BRG1 upregulates de novo lipogenesis and that this is crucial for cancer cell proliferation.  Knockdown of BRG1 attenuates lipid synthesis by impairing the transcription of enzymes catalyzing fatty acid and lipid synthesis.  Remarkably, exogenous addition of palmitate, the key intermediate in fatty acid synthesis, rescued the cancer cell proliferation defect caused by BRG1 knockdown.  Our work suggests that targeting BRG1 to reduce lipid metabolism and, thereby, to reduce proliferation, has promise for epigenetic therapy in triple negative breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSvsaxPn9YAplVwdslNBF-tfW6udTcc2eY1M1F3EuG0Ubntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s%252FjtFGkug%253D%253D&md5=a5847bf6a0604889bd5fad01cd7b6738</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.9505&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.9505%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DMadany%26aufirst%3DP.%26aulast%3DDobson%26aufirst%3DJ.%2BR.%26aulast%3DSchnabl%26aufirst%3DJ.%2BM.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DSmith%26aufirst%3DT.%2BC.%26aulast%3Dvan%2BWijnen%26aufirst%3DA.%2BJ.%26aulast%3DStein%26aufirst%3DJ.%2BL.%26aulast%3DLian%26aufirst%3DJ.%2BB.%26aulast%3DStein%26aufirst%3DG.%2BS.%26aulast%3DMuthuswami%26aufirst%3DR.%26aulast%3DImbalzano%26aufirst%3DA.%2BN.%26aulast%3DNickerson%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520BRG1%2520chromatin%2520remodeling%2520enzyme%2520links%2520cancer%2520cell%2520metabolism%2520and%2520proliferation%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D38270%26epage%3D38281%26doi%3D10.18632%2Foncotarget.9505" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mei, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span> <span> </span><span class="NLM_article-title">BRG1 is a prognostic marker and potential therapeutic target in human breast cancer</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e59772</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0059772</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1371%2Fjournal.pone.0059772" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=23533649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsVOntLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=J.+Baiauthor=P.+Meiauthor=C.+Zhangauthor=F.+Chenauthor=C.+Liauthor=Z.+Panauthor=H.+Liuauthor=J.+Zheng&title=BRG1+is+a+prognostic+marker+and+potential+therapeutic+target+in+human+breast+cancer&doi=10.1371%2Fjournal.pone.0059772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">BRG1 is a prognostic marker and potential therapeutic target in human breast cancer</span></div><div class="casAuthors">Bai, Jin; Mei, Pengjin; Zhang, Cuipeng; Chen, Feifei; Li, Chen; Pan, Zhenqiang; Liu, Hui; Zheng, Junnian</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e59772</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">BRG1, a core component of the SWI/SNF chromatin-remodeling complex, has been implicated in cancer development; however, the biol. significance of BRG1 in breast cancer remains unknown.  We explored the role of BRG1 in human breast cancer pathogenesis.  Using tissue microarray and immunohistochem., we evaluated BRG1 staining in 437 breast cancer specimens and investigated its role in breast cancer cell proliferation, migration and invasion.  Our Kaplan-Meier survival curves showed that high BRG1 expression is inversely correlated with both overall (P = 0.000) and disease-specific (P = 0.000) 5-yr patient survival.  Furthermore, we found that knockdown of BRG1 by RNA interference markedly inhibits cell proliferation and causes cessation of cell cycle.  This reduced cell proliferation is due to G1 phase arrest as cyclin D1 and cyclin E are diminished whereas p27 is upregulated.  Moreover, BRG1 depletion induces the expression of TIMP-2 but reduces MMP-2, thereby inhibiting the ability of cells to migrate and to invade.  These results highlight the importance of BRG1 in breast cancer pathogenesis and BRG1 may serve as a prognostic marker as well as a potentially selective therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqreHzAF2rxbrVg90H21EOLACvtfcHk0lgFC-iy7o41oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsVOntLc%253D&md5=15194af1bd7257d61c2e55b2b0cb3392</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0059772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0059772%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DJ.%26aulast%3DMei%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DPan%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DJ.%26atitle%3DBRG1%2520is%2520a%2520prognostic%2520marker%2520and%2520potential%2520therapeutic%2520target%2520in%2520human%2520breast%2520cancer%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0059772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gresh, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourachot, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reimann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guigas, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiette, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbay, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muchardt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hue, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pontoglio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yaniv, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klochendler-Yeivin, A.</span></span> <span> </span><span class="NLM_article-title">The SWI/SNF chromatin-remodeling complex subunit SNF5 is essential for hepatocyte differentiation</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">3313</span>– <span class="NLM_lpage">3324</span>, <span class="refDoi"> DOI: 10.1038/sj.emboj.7600802</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fsj.emboj.7600802" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=16138077" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVajsrnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2005&pages=3313-3324&author=L.+Greshauthor=B.+Bourachotauthor=A.+Reimannauthor=B.+Guigasauthor=L.+Fietteauthor=S.+Garbayauthor=C.+Muchardtauthor=L.+Hueauthor=M.+Pontoglioauthor=M.+Yanivauthor=A.+Klochendler-Yeivin&title=The+SWI%2FSNF+chromatin-remodeling+complex+subunit+SNF5+is+essential+for+hepatocyte+differentiation&doi=10.1038%2Fsj.emboj.7600802"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">The SWI/SNF chromatin-remodeling complex subunit SNF5 is essential for hepatocyte differentiation</span></div><div class="casAuthors">Gresh, Lionel; Bourachot, Brigitte; Reimann, Andreas; Guigas, Bruno; Fiette, Laurence; Garbay, Serge; Muchardt, Christian; Hue, Louis; Pontoglio, Marco; Yaniv, Moshe; Klochendler-Yeivin, Agnes</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3313-3324</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Regulation of gene expression underlies cell differentiation and organogenesis.  Both transcription factors and chromatin modifiers are crucial for this process.  To study the role of the ATP-dependent SWI/SNF chromatin-remodeling complex in cell differentiation, we inactivated the gene encoding the core complex subunit SNF5/INI1 in the developing liver.  Hepatic SNF5 deletion caused neonatal death due to severe hypoglycemia; mutant animals fail to store glycogen and have impaired energetic metab.  The formation of a hepatic epithelium is also affected in SNF5-deficient livers.  Transcriptome analyses showed that SNF5 inactivation is accompanied by defective transcriptional activation of 70% of the genes that are normally upregulated during liver development.  These include genes involved in glycogen synthesis, gluconeogenesis and cell-cell adhesion.  A fraction of hepatic developmentally activated genes were normally expressed, suggesting that cell differentiation was not completely blocked.  Moreover, SNF5-deleted cells showed increased proliferation, and we identified several misexpressed genes that may contribute to cell cycle deregulation in these cells.  Our results emphasize the role of chromatin remodeling in the activation of cell-type-specific genetic programs and in driving cell differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt7d7ilEBZyLVg90H21EOLACvtfcHk0lhSFO-gUeObIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVajsrnP&md5=a1d1407c27f128cc0dc6f14acf1e868c</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7600802&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7600802%26sid%3Dliteratum%253Aachs%26aulast%3DGresh%26aufirst%3DL.%26aulast%3DBourachot%26aufirst%3DB.%26aulast%3DReimann%26aufirst%3DA.%26aulast%3DGuigas%26aufirst%3DB.%26aulast%3DFiette%26aufirst%3DL.%26aulast%3DGarbay%26aufirst%3DS.%26aulast%3DMuchardt%26aufirst%3DC.%26aulast%3DHue%26aufirst%3DL.%26aulast%3DPontoglio%26aufirst%3DM.%26aulast%3DYaniv%26aufirst%3DM.%26aulast%3DKlochendler-Yeivin%26aufirst%3DA.%26atitle%3DThe%2520SWI%252FSNF%2520chromatin-remodeling%2520complex%2520subunit%2520SNF5%2520is%2520essential%2520for%2520hepatocyte%2520differentiation%26jtitle%3DEMBO%2520J.%26date%3D2005%26volume%3D24%26spage%3D3313%26epage%3D3324%26doi%3D10.1038%2Fsj.emboj.7600802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, T.</span></span> <span> </span><span class="NLM_article-title">The chromatin-remodeling enzyme BRG1 plays an essential role in primitive erythropoiesis and vascular development</span>. <i>Development</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">493</span>– <span class="NLM_lpage">500</span>, <span class="refDoi"> DOI: 10.1242/dev.010090</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1242%2Fdev.010090" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=18094026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjt1CntrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2008&pages=493-500&author=C.+T.+Griffinauthor=J.+Brennanauthor=T.+Magnuson&title=The+chromatin-remodeling+enzyme+BRG1+plays+an+essential+role+in+primitive+erythropoiesis+and+vascular+development&doi=10.1242%2Fdev.010090"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">The chromatin-remodeling enzyme BRG1 plays an essential role in primitive erythropoiesis and vascular development</span></div><div class="casAuthors">Griffin, Courtney T.; Brennan, Jennifer; Magnuson, Terry</div><div class="citationInfo"><span class="NLM_cas:title">Development (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">493-500</span>CODEN:
                <span class="NLM_cas:coden">DEVPED</span>;
        ISSN:<span class="NLM_cas:issn">0950-1991</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">ATP-dependent chromatin-remodeling complexes contribute to the proper temporal and spatial patterns of gene expression in mammalian embryos and therefore play important roles in a no. of developmental processes.  SWI/SNF-like chromatin-remodeling complexes use one of two different ATPases as their catalytic subunit: brahma (BRM, also known as SMARCA2) and brahma-related gene 1 (BRG1, also known as SMARCA4).  We have conditionally deleted a floxed Brg1 allele with a Tie2-Cre transgene, which is expressed in developing hematopoietic and endothelial cells.  Brg1fl/fl:Tie2-Cre+ embryos die at midgestation from anemia, as mutant primitive erythrocytes fail to transcribe embryonic α- and β-globins, and subsequently undergo apoptosis.  Addnl., vascular remodeling of the extraembryonic yolk sac is abnormal in Brg1fl/fl:Tie2-Cre+ embryos.  Importantly, Brm deficiency does not exacerbate the erythropoietic or vascular abnormalities found in Brg1fl/fl:Tie2-Cre+ embryos, implying that Brg1-contg. SWI/SNF-like complexes, rather than Brm-contg. complexes, play a crucial role in primitive erythropoiesis and in early vascular development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcF-UQpNXLHrVg90H21EOLACvtfcHk0lhSFO-gUeObIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjt1CntrY%253D&md5=ad1b61ee0637c08b5b0cc8436838dd22</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1242%2Fdev.010090&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fdev.010090%26sid%3Dliteratum%253Aachs%26aulast%3DGriffin%26aufirst%3DC.%2BT.%26aulast%3DBrennan%26aufirst%3DJ.%26aulast%3DMagnuson%26aufirst%3DT.%26atitle%3DThe%2520chromatin-remodeling%2520enzyme%2520BRG1%2520plays%2520an%2520essential%2520role%2520in%2520primitive%2520erythropoiesis%2520and%2520vascular%2520development%26jtitle%3DDevelopment%26date%3D2008%26volume%3D135%26spage%3D493%26epage%3D500%26doi%3D10.1242%2Fdev.010090" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cornelius, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDougald, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, M. D.</span></span> <span> </span><span class="NLM_article-title">Regulation of adipocyte development</span>. <i>Annu. Rev. Nutr.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">99</span>– <span class="NLM_lpage">129</span>, <span class="refDoi"> DOI: 10.1146/annurev.nu.14.070194.000531</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1146%2Fannurev.nu.14.070194.000531" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=7946535" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADyaK2cXmsFWrsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1994&pages=99-129&author=P.+Corneliusauthor=O.+A.+MacDougaldauthor=M.+D.+Lane&title=Regulation+of+adipocyte+development&doi=10.1146%2Fannurev.nu.14.070194.000531"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of adipocyte development</span></div><div class="casAuthors">Cornelius, Peter; MacDougald, Ormond A.; Lane, M. Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Nutrition</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">99-129</span>CODEN:
                <span class="NLM_cas:coden">ARNTD8</span>;
        ISSN:<span class="NLM_cas:issn">0199-9885</span>.
    </div><div class="casAbstract">A review, with 216 refs., discussing cell culture models for adipocyte development, external modulators of preadipocyte differentiation, a developmental program, and transcriptional control of adipocyte genes during differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpadKhkXFwuArVg90H21EOLACvtfcHk0liEMcYvHSrwVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmsFWrsL8%253D&md5=e2c5c546ba15ef193ab4323b73b8ed06</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1146%2Fannurev.nu.14.070194.000531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.nu.14.070194.000531%26sid%3Dliteratum%253Aachs%26aulast%3DCornelius%26aufirst%3DP.%26aulast%3DMacDougald%26aufirst%3DO.%2BA.%26aulast%3DLane%26aufirst%3DM.%2BD.%26atitle%3DRegulation%2520of%2520adipocyte%2520development%26jtitle%3DAnnu.%2520Rev.%2520Nutr.%26date%3D1994%26volume%3D14%26spage%3D99%26epage%3D129%26doi%3D10.1146%2Fannurev.nu.14.070194.000531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oishi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manabe, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsushima, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shindo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishimura, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maemura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamauchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubota, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akira, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadowaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, R.</span></span> <span> </span><span class="NLM_article-title">Krüppel-like transcription factor KLF5 is a key regulator of adipocyte differentiation</span>. <i>Cell Metab.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">27</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1016/j.cmet.2004.11.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2Fj.cmet.2004.11.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=16054042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlaqs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2005&pages=27-39&author=Y.+Oishiauthor=I.+Manabeauthor=K.+Tobeauthor=K.+Tsushimaauthor=T.+Shindoauthor=K.+Fujiuauthor=G.+Nishimuraauthor=K.+Maemuraauthor=T.+Yamauchiauthor=N.+Kubotaauthor=R.+Suzukiauthor=T.+Kitamuraauthor=S.+Akiraauthor=T.+Kadowakiauthor=R.+Nagai&title=Kr%C3%BCppel-like+transcription+factor+KLF5+is+a+key+regulator+of+adipocyte+differentiation&doi=10.1016%2Fj.cmet.2004.11.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Krueppel-like transcription factor KLF5 is a key regulator of adipocyte differentiation</span></div><div class="casAuthors">Oishi, Yumiko; Manabe, Ichiro; Tobe, Kazuyuki; Tsushima, Kensuke; Shindo, Takayuki; Fujiu, Katsuhito; Nishimura, Go; Maemura, Koji; Yamauchi, Toshimasa; Kubota, Naoto; Suzuki, Ryo; Kitamura, Toshio; Akira, Shizuo; Kadowaki, Takashi; Nagai, Ryozo</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-39</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Krueppel-like factor 5 (KLF5) is a zinc-finger transcription factor known to play a pivotal role in the pathogenesis of cardiovascular disease.  Here, we show that neonatal heterozygous KLF5 knockout mice exhibit a marked deficiency in white adipose tissue development, suggesting that KLF5 is also required for adipogenesis.  In 3T3-L1 preadipocytes, KLF5 expression was induced at an early stage of differentiation, and this was followed by expression of PPARγ2.  Constitutive overexpression of dominant-neg. KLF5 inhibited adipocyte differentiation, whereas overexpression of wild-type KLF5 induced differentiation even without hormonal stimulation.  Moreover, embryonic fibroblasts obtained from KLF5+/- mice showed much attenuated adipocyte differentiation, confirming the key role played by KLF5 in adipocyte differentiation.  KLF5 expression is induced by C/EBPβ and δ.  KLF5, in turn, acts in concert with C/EBPβ/δ to activate the PPARγ2 promoter.  This study establishes KLF5 as a key component of the transcription factor network controlling adipocyte differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVvhxLphZlcLVg90H21EOLACvtfcHk0liEMcYvHSrwVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlaqs78%253D&md5=d56b443c5f82c63fcd4ee7f56806c758</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2004.11.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2004.11.005%26sid%3Dliteratum%253Aachs%26aulast%3DOishi%26aufirst%3DY.%26aulast%3DManabe%26aufirst%3DI.%26aulast%3DTobe%26aufirst%3DK.%26aulast%3DTsushima%26aufirst%3DK.%26aulast%3DShindo%26aufirst%3DT.%26aulast%3DFujiu%26aufirst%3DK.%26aulast%3DNishimura%26aufirst%3DG.%26aulast%3DMaemura%26aufirst%3DK.%26aulast%3DYamauchi%26aufirst%3DT.%26aulast%3DKubota%26aufirst%3DN.%26aulast%3DSuzuki%26aufirst%3DR.%26aulast%3DKitamura%26aufirst%3DT.%26aulast%3DAkira%26aufirst%3DS.%26aulast%3DKadowaki%26aufirst%3DT.%26aulast%3DNagai%26aufirst%3DR.%26atitle%3DKr%25C3%25BCppel-like%2520transcription%2520factor%2520KLF5%2520is%2520a%2520key%2520regulator%2520of%2520adipocyte%2520differentiation%26jtitle%3DCell%2520Metab.%26date%3D2005%26volume%3D1%26spage%3D27%26epage%3D39%26doi%3D10.1016%2Fj.cmet.2004.11.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrens, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anand, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spiegelman, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, J. M.</span></span> <span> </span><span class="NLM_article-title">Krox20 stimulates adipogenesis via C/EBPbeta-dependent and -independent mechanisms</span>. <i>Cell Metab.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">93</span>– <span class="NLM_lpage">106</span>, <span class="refDoi"> DOI: 10.1016/j.cmet.2004.12.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2Fj.cmet.2004.12.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=16054051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD2MXit12rtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2005&pages=93-106&author=Z.+Chenauthor=J.+I.+Torrensauthor=A.+Anandauthor=B.+M.+Spiegelmanauthor=J.+M.+Friedman&title=Krox20+stimulates+adipogenesis+via+C%2FEBPbeta-dependent+and+-independent+mechanisms&doi=10.1016%2Fj.cmet.2004.12.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Krox20 stimulates adipogenesis via C/EBPβ-dependent and -independent mechanisms</span></div><div class="casAuthors">Chen, Zhu; Torrens, Javier I.; Anand, Ashim; Spiegelman, Bruce M.; Friedman, Jeffrey M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">93-106</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Krox20 is a zinc finger-contg. transcription factor that is abundantly expressed in adipose tissue.  However, its role in fat cell differentiation has not been established.  In cultured 3T3-L1 cells, Krox20 is rapidly induced by serum stimulation.  Overexpression of Krox20 in both 3T3-L1 preadipocytes and multipotent NIH3T3 cells promotes adipogenesis in a hormone-dependent manner.  Conversely, RNAi-mediated loss of Krox20 function reduced adipogenesis in 3T3-L1 cells.  Ectopic expression of Krox20 can transactivate the C/EBPβ promoter and increase C/EBPβ gene expression in 3T3-L1 preadipocytes.  RNAi-mediated knockdown of C/EPBβ diminished Krox20's proadipogenic effect.  Finally, coexpression of Krox20 and C/EBPβ in naive NIH3T3 cells resulted in the pronounced induction of a fully differentiated adipocyte phenotype, an effect previously obsd. only with PPARγ.  These data indicate that Krox20 is necessary for adipogenesis and that, when overexpressed, Krox20 potently stimulates adipogenesis via C/EBPβ-dependent and -independent mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpR_wstSnX5FbVg90H21EOLACvtfcHk0liEMcYvHSrwVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXit12rtbw%253D&md5=5c7737f5325c5b290a82db8b6d3074b6</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2004.12.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2004.12.009%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DTorrens%26aufirst%3DJ.%2BI.%26aulast%3DAnand%26aufirst%3DA.%26aulast%3DSpiegelman%26aufirst%3DB.%2BM.%26aulast%3DFriedman%26aufirst%3DJ.%2BM.%26atitle%3DKrox20%2520stimulates%2520adipogenesis%2520via%2520C%252FEBPbeta-dependent%2520and%2520-independent%2520mechanisms%26jtitle%3DCell%2520Metab.%26date%3D2005%26volume%3D1%26spage%3D93%26epage%3D106%26doi%3D10.1016%2Fj.cmet.2004.12.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gimble, J. M.</span></span> <span> </span><span class="NLM_article-title">CAAT/enhancer binding proteins directly modulate transcription from the peroxisome proliferator-activated receptor gamma 2 promoter</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>240</i></span>,  <span class="NLM_fpage">99</span>– <span class="NLM_lpage">103</span>, <span class="refDoi"> DOI: 10.1006/bbrc.1997.7627</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1006%2Fbbrc.1997.7627" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=9367890" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADyaK2sXnsVWisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=240&publication_year=1997&pages=99-103&author=S.+L.+Clarkeauthor=C.+E.+Robinsonauthor=J.+M.+Gimble&title=CAAT%2Fenhancer+binding+proteins+directly+modulate+transcription+from+the+peroxisome+proliferator-activated+receptor+gamma+2+promoter&doi=10.1006%2Fbbrc.1997.7627"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">CAAT/enhancer binding proteins directly modulate transcription from the peroxisome proliferator-activated receptor γ2 promoter</span></div><div class="casAuthors">Clarke, Stephen L.; Robinson, Claudius E.; Gimble, Jeffrey M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">240</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">99-103</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">The CCAAT/enhancer binding proteins (C/EBPs) and the peroxisome proliferator-activated receptors (PPARs) together regulate adipogenesis.  The current work uses co-transfection studies to examine the C/EBP dependence of PPARy2 transcription.  Both C/EBPα and C/EBPδ expression vectors activated transcription from a PPARγ2 promoter/luciferase expression vector by 5-6 fold in UMR106 cells.  The simultaneous transfection of the C/EBP homologous protein (CHOP) (also known as growth arrest DNA damage protein 153 or gadd153) inhibited this C/EBP-dependent activation in a concn. dependent manner.  The CHOP protein is known to heterodimerize with other C/EBP proteins to form transcriptionally inactive complexes.  Mutation of the two C/EBP DNA recognition elements at -340 bp and -327 bp within the PPARγ2 promoter reduced the inductive effects of both C/EBPα and C/EBPδ.  These findings demonstrate that proteins within the C/EBP family directly modulate transcription from the PPARγ2 promoter.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq5EeNFa8uOrVg90H21EOLACvtfcHk0lh5ma7za34Oag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXnsVWisrY%253D&md5=6ec5e3a65195a1dad3837245f26e666e</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1006%2Fbbrc.1997.7627&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fbbrc.1997.7627%26sid%3Dliteratum%253Aachs%26aulast%3DClarke%26aufirst%3DS.%2BL.%26aulast%3DRobinson%26aufirst%3DC.%2BE.%26aulast%3DGimble%26aufirst%3DJ.%2BM.%26atitle%3DCAAT%252Fenhancer%2520binding%2520proteins%2520directly%2520modulate%2520transcription%2520from%2520the%2520peroxisome%2520proliferator-activated%2520receptor%2520gamma%25202%2520promoter%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1997%26volume%3D240%26spage%3D99%26epage%3D103%26doi%3D10.1006%2Fbbrc.1997.7627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bucher, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farmer, S. R.</span></span> <span> </span><span class="NLM_article-title">Conditional ectopic expression of C/EBP beta in NIH-3T3 cells induces PPAR gamma and stimulates adipogenesis</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2350</span>– <span class="NLM_lpage">2363</span>, <span class="refDoi"> DOI: 10.1101/gad.9.19.2350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1101%2Fgad.9.19.2350" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=7557387" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADyaK2MXos12it74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1995&pages=2350-2363&author=Z.+Wuauthor=Y.+Xieauthor=N.+L.+Bucherauthor=S.+R.+Farmer&title=Conditional+ectopic+expression+of+C%2FEBP+beta+in+NIH-3T3+cells+induces+PPAR+gamma+and+stimulates+adipogenesis&doi=10.1101%2Fgad.9.19.2350"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Conditional ectopic expression of C/EBPβ in NIH-3T3 cells induces PPARγ and stimulates adipogenesis</span></div><div class="casAuthors">Wu, Zhidan; Xie, Yuhong; Bucher, Nancy L. R.; Farmer, Stephen R.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2350-63</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Activation of adipogenesis in 3T3 preadipocytes by exposure to the adipogenic inducers dexamethasone, methylisobutylxanthine, insulin, and fetal bovine serum is accompanied by a transient burst of C/EBPβ protein expression that precedes the induction of the fat gene program.  In this study we have investigated the role of C/EBPβ in initiating the adipogenic program by overexpressing C/EBPβ in multipotential NIH-3T3 fibroblasts.  Conditional ectopic expression of C/EBPβ was accomplished by using an artificial transcriptional regulatory system based on the Escherichia coli tetracycline repressor to generate a stable cell line, β2, that expresses C/EBPβ mRNA and protein in a tightly controlled tetracycline dose-dependent manner.  Induction of C/EBPβ DNA-binding activity in NIH-3T3 β2 cells exposed to dexamethasone in the presence of insulin and fetal bovine serum activates the expression of an adipocyte-specific nuclear hormone receptor, PPARγ, that stimulates the conversion of these fibroblasts into committed preadipocytes.  Either ectopic expression of C/EBPβ or treatment with dexamethasone alone is incapable of inducing PPARγ expression, but when present together, they have a synergistic effect on the adipogenic program.  Exposure of these stimulated cells to a PPAR activator 5,8,11,14-eicosatetraynoic acid (ETYA) results in the accumulation of fat droplets and expression of the adipocyte-enriched genes aP2 and glycerol phosphate dehydrogenase (GPD).  The no. of β2 cells that can differentiate into adipocytes is related to the concn. of tetracycline and, therefore, the amt. of the exogenous C/EBPβ protein expressed.  C/EBPβ can induce PPARγ mRNA in the absence of ETYA; however, expression of aP2 mRNA and max. fat deposition is dependent on the PPAR activator.  Our results suggest that enhanced expression of C/EBPβ converts multipotential mesenchymal precursor cells into preadipocytes that respond to adipogenic inducers, including dexamethasone and PPAR activators to differentiate into adipocytes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq19Iwq-lGFzLVg90H21EOLACvtfcHk0lh5ma7za34Oag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXos12it74%253D&md5=76f2dc035e3aa33fca6731feb8dc0fc5</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1101%2Fgad.9.19.2350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.9.19.2350%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DBucher%26aufirst%3DN.%2BL.%26aulast%3DFarmer%26aufirst%3DS.%2BR.%26atitle%3DConditional%2520ectopic%2520expression%2520of%2520C%252FEBP%2520beta%2520in%2520NIH-3T3%2520cells%2520induces%2520PPAR%2520gamma%2520and%2520stimulates%2520adipogenesis%26jtitle%3DGenes%2520Dev.%26date%3D1995%26volume%3D9%26spage%3D2350%26epage%3D2363%26doi%3D10.1101%2Fgad.9.19.2350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shepherd, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahn, B. B.</span></span> <span> </span><span class="NLM_article-title">Glucose transporters and insulin action-implications for insulin resistance and diabetes mellitus</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>341</i></span>,  <span class="NLM_fpage">248</span>– <span class="NLM_lpage">257</span>, <span class="refDoi"> DOI: 10.1056/nejm199907223410406</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1056%2FNEJM199907223410406" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10413738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADyaK1MXltVOju74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=1999&pages=248-257&author=P.+R.+Shepherdauthor=B.+B.+Kahn&title=Glucose+transporters+and+insulin+action-implications+for+insulin+resistance+and+diabetes+mellitus&doi=10.1056%2Fnejm199907223410406"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Glucose transporters and insulin action: implications for insulin resistance and diabetes mellitus</span></div><div class="casAuthors">Shepherd, Peter R.; Kahn, Barbara B.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">248-257</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review, with 103 refs.  This review discusses recent advances, their contribution to our understanding of the pathogenesis of diabetes mellitus, and their implications for the design of new therapies to prevent and treat diabetes and its complications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZ8ajzYZcxc7Vg90H21EOLACvtfcHk0liQ-ZGsrg9VXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXltVOju74%253D&md5=197ffdc34708ec60d21c64024a1d2dff</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1056%2FNEJM199907223410406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJM199907223410406%26sid%3Dliteratum%253Aachs%26aulast%3DShepherd%26aufirst%3DP.%2BR.%26aulast%3DKahn%26aufirst%3DB.%2BB.%26atitle%3DGlucose%2520transporters%2520and%2520insulin%2520action-implications%2520for%2520insulin%2520resistance%2520and%2520diabetes%2520mellitus%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D1999%26volume%3D341%26spage%3D248%26epage%3D257%26doi%3D10.1056%2Fnejm199907223410406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garvey, W. T.</span></span> <span> </span><span class="NLM_article-title">Adiponectin promotes adipocyte differentiation, insulin sensitivity, and lipid accumulation</span>. <i>J. Lipid Res.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">1369</span>– <span class="NLM_lpage">1379</span>, <span class="refDoi"> DOI: 10.1194/jlr.m400373-jlr200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1194%2Fjlr.M400373-JLR200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=15834118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmtFKksL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2005&pages=1369-1379&author=Y.+Fuauthor=N.+Luoauthor=R.+L.+Kleinauthor=W.+T.+Garvey&title=Adiponectin+promotes+adipocyte+differentiation%2C+insulin+sensitivity%2C+and+lipid+accumulation&doi=10.1194%2Fjlr.m400373-jlr200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Adiponectin promotes adipocyte differentiation, insulin sensitivity, and lipid accumulation</span></div><div class="casAuthors">Fu, Yuchang; Luo, Nanlan; Klein, Richard L.; Garvey, W. Timothy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Lipid Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1369-1379</span>CODEN:
                <span class="NLM_cas:coden">JLPRAW</span>;
        ISSN:<span class="NLM_cas:issn">0022-2275</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology, Inc.</span>)
        </div><div class="casAbstract">Adiponectin is secreted from adipocytes, and low circulating levels have been epidemiol. assocd. with obesity, insulin resistance, type 2 diabetes, and cardiovascular disease.  To investigate whether adiponectin could exert autocrine effects in adipocytes, the authors expressed the adiponectin gene in 3T3-L1 fibroblasts.  The authors obsd. that 3T3-L1 fibroblasts expressing adiponectin have a fast growth phase and reach confluence more rapidly compared with control cells or LacZ-transduced cells.  Furthermore, cells with overexpressed adiponectin were obsd. to differentiate into adipocytes more rapidly, and during adipogenesis, they exhibited more prolonged and robust gene expression for related transcriptional factors, CCAAT/enhancer binding protein α (C/EBP2), peroxisome proliferator-activated receptor γ (PPARγ), and adipocyte detn. and differentiation factor 1/sterol-regulatory element binding protein 1c (ADD1/SREBP1c) and earlier suppression of PPARγ coactivator-1α (PGC-1α).  In fully differentiated adipocytes, adiponectin-overexpressing cells accumulated more and larger lipid droplets compared with control cells.  Also, adiponectin increased insulin's ability to maximally stimulate glucose uptake by 78% through increased glucose transporter 4 (GLUT4) gene expression and increased GLUT4 recruitment to the plasma membrane.  These data suggest a new role for adiponectin as an autocrine factor in adipose tissues: promoting cell proliferation and differentiation from preadipocytes into adipocytes, augmenting programmed gene expression responsible for adipogenesis, and increasing lipid content and insulin responsiveness of the glucose transport system in adipocytes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrex0wPiPGuDLVg90H21EOLACvtfcHk0liQ-ZGsrg9VXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmtFKksL0%253D&md5=a9cbc435c4ba1871e40e11b6cfe31abd</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1194%2Fjlr.M400373-JLR200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1194%252Fjlr.M400373-JLR200%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DN.%26aulast%3DKlein%26aufirst%3DR.%2BL.%26aulast%3DGarvey%26aufirst%3DW.%2BT.%26atitle%3DAdiponectin%2520promotes%2520adipocyte%2520differentiation%252C%2520insulin%2520sensitivity%252C%2520and%2520lipid%2520accumulation%26jtitle%3DJ.%2520Lipid%2520Res.%26date%3D2005%26volume%3D46%26spage%3D1369%26epage%3D1379%26doi%3D10.1194%2Fjlr.m400373-jlr200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Furuhashi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saitoh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miura, T.</span></span> <span> </span><span class="NLM_article-title">Fatty acid-binding protein 4 (FABP4): Pathophysiological insights and potent clinical biomarker of metabolic and cardiovascular diseases</span>. <i>Clin. Med. Insights: Cardiol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.4137/CMC.S17067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.4137%2FCMC.S17067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=25674026" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2015&pages=23-33&author=M.+Furuhashiauthor=S.+Saitohauthor=K.+Shimamotoauthor=T.+Miura&title=Fatty+acid-binding+protein+4+%28FABP4%29%3A+Pathophysiological+insights+and+potent+clinical+biomarker+of+metabolic+and+cardiovascular+diseases&doi=10.4137%2FCMC.S17067"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.4137%2FCMC.S17067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4137%252FCMC.S17067%26sid%3Dliteratum%253Aachs%26aulast%3DFuruhashi%26aufirst%3DM.%26aulast%3DSaitoh%26aufirst%3DS.%26aulast%3DShimamoto%26aufirst%3DK.%26aulast%3DMiura%26aufirst%3DT.%26atitle%3DFatty%2520acid-binding%2520protein%25204%2520%2528FABP4%2529%253A%2520Pathophysiological%2520insights%2520and%2520potent%2520clinical%2520biomarker%2520of%2520metabolic%2520and%2520cardiovascular%2520diseases%26jtitle%3DClin.%2520Med.%2520Insights%253A%2520Cardiol.%26date%3D2015%26volume%3D8%26spage%3D23%26epage%3D33%26doi%3D10.4137%2FCMC.S17067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niesen, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span> <span> </span><span class="NLM_article-title">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">2212</span>– <span class="NLM_lpage">2221</span>, <span class="refDoi"> DOI: 10.1038/nprot.2007.321</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fnprot.2007.321" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=17853878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2212-2221&author=F.+H.+Niesenauthor=H.+Berglundauthor=M.+Vedadi&title=The+use+of+differential+scanning+fluorimetry+to+detect+ligand+interactions+that+promote+protein+stability&doi=10.1038%2Fnprot.2007.321"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span></div><div class="casAuthors">Niesen, Frank H.; Berglund, Helena; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2212-2221</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Differential scanning fluorometry (DSF) is a rapid and inexpensive screening method to identify low-mol.-wt. ligands that bind and stabilize purified proteins.  The temp. at which a protein unfolds is measured by an increase in the fluorescence of a dye with affinity for hydrophobic parts of the protein, which are exposed as the protein unfolds.  A simple fitting procedure allows quick calcn. of the transition midpoint; the difference in the temp. of this midpoint in the presence and absence of ligand is related to the binding affinity of the small mol., which can be a low-mol.-wt. compd., a peptide or a nucleic acid.  DSF is best performed using a conventional real-time PCR instrument.  Ligand solns. from a storage plate are added to a soln. of protein and dye, distributed into the wells of the PCR plate and fluorescence intensity measured as the temp. is raised gradually.  Results can be obtained in a single day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomViqp6Tq7krVg90H21EOLACvtfcHk0liQ-ZGsrg9VXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N&md5=8a74dbb348ab81179614ab4ae5797d94</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.321%26sid%3Dliteratum%253Aachs%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DBerglund%26aufirst%3DH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DThe%2520use%2520of%2520differential%2520scanning%2520fluorimetry%2520to%2520detect%2520ligand%2520interactions%2520that%2520promote%2520protein%2520stability%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D2%26spage%3D2212%26epage%3D2221%26doi%3D10.1038%2Fnprot.2007.321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiseman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williston, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandts, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.-N.</span></span> <span> </span><span class="NLM_article-title">Rapid measurement of binding constants and heats of binding using a new titration calorimeter</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>179</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">137</span>, <span class="refDoi"> DOI: 10.1016/0003-2697(89)90213-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2F0003-2697%2889%2990213-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=2757186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADyaL1MXktlKjsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=1989&pages=131-137&author=T.+Wisemanauthor=S.+Willistonauthor=J.+F.+Brandtsauthor=L.-N.+Lin&title=Rapid+measurement+of+binding+constants+and+heats+of+binding+using+a+new+titration+calorimeter&doi=10.1016%2F0003-2697%2889%2990213-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid measurement of binding constants and heats of binding using a new titration calorimeter</span></div><div class="casAuthors">Wiseman, Thomas; Williston, Samuel; Brandts, John F.; Lin, Lung Nan</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">179</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">131-7</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">A new titrn. calorimeter is described and results are presented for the binding of 2'-CMP to the active site of RNase A.  The instrument characteristics include very high sensitivity, rapid calorimetric response, and fast thermal equilibration.  Convenient software is available for instrument operation, data collection, data redn., and deconvolution to obtain least-squares ests. of binding parameters n, ΔH°, ΔS°, and the binding const. K.  Sample through-put for the instrument is high, and under favorable conditions binding consts. as large as 108 M-1 can be measured.  The bovine RNase A (RNase)/2'-CMP system was studied over a 50-fold range of RNase concn. and at 2 different temps.  The binding consts. were in the 105 to 106 M-1 range, depending on conditions, and heats of binding ∼-15,000 cal/mol.  Repeat detns. suggested errors of only a few percent in n, ΔH°, and K values over the most favorable concn. range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJRWydqrfWArVg90H21EOLACvtfcHk0lhh_cMHJ8jl7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXktlKjsrw%253D&md5=53fcdb7144fd85e57963add0bcdb6041</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1016%2F0003-2697%2889%2990213-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-2697%252889%252990213-3%26sid%3Dliteratum%253Aachs%26aulast%3DWiseman%26aufirst%3DT.%26aulast%3DWilliston%26aufirst%3DS.%26aulast%3DBrandts%26aufirst%3DJ.%2BF.%26aulast%3DLin%26aufirst%3DL.-N.%26atitle%3DRapid%2520measurement%2520of%2520binding%2520constants%2520and%2520heats%2520of%2520binding%2520using%2520a%2520new%2520titration%2520calorimeter%26jtitle%3DAnal.%2520Biochem.%26date%3D1989%26volume%3D179%26spage%3D131%26epage%3D137%26doi%3D10.1016%2F0003-2697%2889%2990213-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Battye, T. G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kontogiannis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span> <span> </span><span class="NLM_article-title">iMOSFLM: a new graphical interface for diffraction-image processing with MOSFLM</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">271</span>– <span class="NLM_lpage">281</span>, <span class="refDoi"> DOI: 10.1107/s0907444910048675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1107%2Fs0907444910048675" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=21460445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=271-281&author=T.+G.+G.+Battyeauthor=L.+Kontogiannisauthor=O.+Johnsonauthor=H.+R.+Powellauthor=A.+G.+W.+Leslie&title=iMOSFLM%3A+a+new+graphical+interface+for+diffraction-image+processing+with+MOSFLM&doi=10.1107%2Fs0907444910048675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">iMOSFLM: A new graphical interface for diffraction-image processing with MOSFLM</span></div><div class="casAuthors">Battye, T. Geoff G.; Kontogiannis, Luke; Johnson, Owen; Powell, Harold R.; Leslie, Andrew G. W.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">271-281</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">IMOSFLM is a graphical user interface to the diffraction data-integration program MOSFLM.  It is designed to simplify data processing by dividing the process into a series of steps, which are normally carried out sequentially.  Each step has its own display pane, allowing control over parameters that influence that step and providing graphical feedback to the user.  Suitable values for integration parameters are set automatically, but addnl. menus provide a detailed level of control for experienced users.  The image display and the interfaces to the different tasks (indexing, strategy calcn., cell refinement, integration and history) are described.  The most important parameters for each step and the best way of assessing success or failure are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeZl2UOMg3_bVg90H21EOLACvtfcHk0lhh_cMHJ8jl7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLw%253D&md5=852651253f8d9c9a81e0513b6633c705</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1107%2Fs0907444910048675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444910048675%26sid%3Dliteratum%253Aachs%26aulast%3DBattye%26aufirst%3DT.%2BG.%2BG.%26aulast%3DKontogiannis%26aufirst%3DL.%26aulast%3DJohnson%26aufirst%3DO.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26atitle%3DiMOSFLM%253A%2520a%2520new%2520graphical%2520interface%2520for%2520diffraction-image%2520processing%2520with%2520MOSFLM%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D271%26epage%3D281%26doi%3D10.1107%2Fs0907444910048675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2Fs0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lhowmj5BRXosg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2Fs0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span> <span> </span><span class="NLM_article-title">An introduction to data reduction: space-group determination, scaling and intensity statistics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">282</span>– <span class="NLM_lpage">292</span>, <span class="refDoi"> DOI: 10.1107/s090744491003982x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1107%2Fs090744491003982x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=21460446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=282-292&author=P.+R.+Evans&title=An+introduction+to+data+reduction%3A+space-group+determination%2C+scaling+and+intensity+statistics&doi=10.1107%2Fs090744491003982x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">An introduction to data reduction: Space-group determination, scaling and intensity statistics</span></div><div class="casAuthors">Evans, Philip R.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">282-292</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  This paper presents an overview of how to run the CCP4 programs for data redn. (SCALA, POINTLESS and CTRUNCATE) through the CCP4 graphical interface ccp4i and points out some issues that need to be considered, together with a few examples.  It covers detn. of the point-group symmetry of the diffraction data (the Laue group), which is required for the subsequent scaling step, examn. of systematic absences, which in many cases will allow inference of the space group, putting multiple data sets on a common indexing system when there are alternatives, the scaling step itself, which produces a large set of data-quality indicators, estn. of |F| from intensity and finally examn. of intensity statistics to detect crystal pathologies such as twinning.  An appendix outlines the scoring schemes used by the program POINTLESS to assign probabilities to possible Laue and space groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEq8RhfQOW8LVg90H21EOLACvtfcHk0lhowmj5BRXosg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLo%253D&md5=f425388a87744d721d324118d50e4f9a</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1107%2Fs090744491003982x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs090744491003982x%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26atitle%3DAn%2520introduction%2520to%2520data%2520reduction%253A%2520space-group%2520determination%252C%2520scaling%2520and%2520intensity%2520statistics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D282%26epage%3D292%26doi%3D10.1107%2Fs090744491003982x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/s0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2Fs0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lhowmj5BRXosg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2Fs0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/s0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2Fs0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lgXSjEFEXw7Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2Fs0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lgXSjEFEXw7Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/s0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2Fs0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lhv0ErTNGFajg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2Fs0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roehm, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodgers, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatfield, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glasebrook, A. L.</span></span> <span> </span><span class="NLM_article-title">An improved colorimetric assay for cell proliferation and viability utilizing the tetrazolium salt XTT</span>. <i>J. Immunol. Methods</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">257</span>– <span class="NLM_lpage">265</span>, <span class="refDoi"> DOI: 10.1016/0022-1759(91)90114-u</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2F0022-1759%2891%2990114-U" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1919029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A280%3ADyaK38%252FhtlWqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=1991&pages=257-265&author=N.+W.+Roehmauthor=G.+H.+Rodgersauthor=S.+M.+Hatfieldauthor=A.+L.+Glasebrook&title=An+improved+colorimetric+assay+for+cell+proliferation+and+viability+utilizing+the+tetrazolium+salt+XTT&doi=10.1016%2F0022-1759%2891%2990114-u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">An improved colorimetric assay for cell proliferation and viability utilizing the tetrazolium salt XTT</span></div><div class="casAuthors">Roehm N W; Rodgers G H; Hatfield S M; Glasebrook A L</div><div class="citationInfo"><span class="NLM_cas:title">Journal of immunological methods</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">257-65</span>
        ISSN:<span class="NLM_cas:issn">0022-1759</span>.
    </div><div class="casAbstract">A new tetrazolium salt XTT, sodium 3'-[1-[(phenylamino)-carbonyl]-3,4-tetrazolium]-bis(4-methoxy-6- nitro)benzene-sulfonic acid hydrate, was evaluated for use in a colorimetric assay for cell viability and proliferation by normal activated T cells and several cytokine dependent cell lines.  Cleavage of XTT by dehydrogenase enzymes of metabolically active cells yields a highly colored formazan product which is water soluble.  This feature obviates the need for formazan crystal solubilization prior to absorbance measurements, as required when using other tetrazolium salts such as MTT.  Bioreduction of XTT by all the murine cells examined was not particularly efficient, but could be potentiated by addition of electron coupling agents such as phenazine methosulfate (PMS) or menadione (MEN).  Optimal concentrations of PMS or MEN were determined for the metabolism of XTT by the T cell lines HT-2 and 11.6, NFS-60 a myeloid leukemia, MC/9 a mast cell line and mitogen activated splenic T cells.  When used in combination with PMS, each of these cells generated higher formazan absorbance values with XTT than were observed with MTT.  Thus the use of XTT in colorimetric proliferation assays offer significant advantages over MTT, resulting from reduced assay time and sample handling, while offering equivalent sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSIr3tbWsUOa8rRqEXsZufAfW6udTcc2eb51T8REpKNzrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK38%252FhtlWqsw%253D%253D&md5=744ee298cf13e12bda5f93b5ec26be2d</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1016%2F0022-1759%2891%2990114-U&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-1759%252891%252990114-U%26sid%3Dliteratum%253Aachs%26aulast%3DRoehm%26aufirst%3DN.%2BW.%26aulast%3DRodgers%26aufirst%3DG.%2BH.%26aulast%3DHatfield%26aufirst%3DS.%2BM.%26aulast%3DGlasebrook%26aufirst%3DA.%2BL.%26atitle%3DAn%2520improved%2520colorimetric%2520assay%2520for%2520cell%2520proliferation%2520and%2520viability%2520utilizing%2520the%2520tetrazolium%2520salt%2520XTT%26jtitle%3DJ.%2520Immunol.%2520Methods%26date%3D1991%26volume%3D142%26spage%3D257%26epage%3D265%26doi%3D10.1016%2F0022-1759%2891%2990114-u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Göbel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heering, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angioni, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittmann, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Buscato, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kottke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weizel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schader, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maier, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geisslinger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schubert-Zsilavecz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinhilber, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proschak, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahnt, A. S.</span></span> <span> </span><span class="NLM_article-title">Zafirlukast is a dual modulator of human soluble epoxide hydrolase and peroxisome proliferator-activated receptor γ</span>. <i>Front. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">263</span>, <span class="refDoi"> DOI: 10.3389/fphar.2019.00263</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.3389%2Ffphar.2019.00263" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=30949053" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A280%3ADC%252BB3M%252FhvVOhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=263&author=T.+G%C3%B6belauthor=O.+Diehlauthor=J.+Heeringauthor=D.+Merkauthor=C.+Angioniauthor=S.+K.+Wittmannauthor=E.+La+Buscatoauthor=R.+Kottkeauthor=L.+Weizelauthor=T.+Schaderauthor=T.+J.+Maierauthor=G.+Geisslingerauthor=M.+Schubert-Zsilaveczauthor=D.+Steinhilberauthor=E.+Proschakauthor=A.+S.+Kahnt&title=Zafirlukast+is+a+dual+modulator+of+human+soluble+epoxide+hydrolase+and+peroxisome+proliferator-activated+receptor+%CE%B3&doi=10.3389%2Ffphar.2019.00263"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Zafirlukast Is a Dual Modulator of Human Soluble Epoxide Hydrolase and Peroxisome Proliferator-Activated Receptor γ</span></div><div class="casAuthors">Gobel Tamara; Diehl Olaf; Merk Daniel; Wittmann Sandra K; Buscato Estel la; Kottke Ramona; Weizel Lilia; Schader Tim; Schubert-Zsilavecz Manfred; Steinhilber Dieter; Proschak Ewgenij; Kahnt Astrid S; Heering Jan; Geisslinger Gerd; Angioni Carlo; Geisslinger Gerd; Maier Thorsten J</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in pharmacology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">263</span>
        ISSN:<span class="NLM_cas:issn">1663-9812</span>.
    </div><div class="casAbstract">Cysteinyl leukotriene receptor 1 antagonists (CysLT1RA) are frequently used as add-on medication for the treatment of asthma.  Recently, these compounds have shown protective effects in cardiovascular diseases.  This prompted us to investigate their influence on soluble epoxide hydrolase (sEH) and peroxisome proliferator activated receptor (PPAR) activities, two targets known to play an important role in CVD and the metabolic syndrome.  Montelukast, pranlukast and zafirlukast inhibited human sEH with IC50 values of 1.9, 14.1, and 0.8 μM, respectively.  In contrast, only montelukast and zafirlukast activated PPARγ in the reporter gene assay with EC50 values of 1.17 μM (21.9% max. activation) and 2.49 μM (148% max. activation), respectively.  PPARα and δ were not affected by any of the compounds.  The activation of PPARγ was further investigated in 3T3-L1 adipocytes.  Analysis of lipid accumulation, mRNA and protein expression of target genes as well as PPARγ phosphorylation revealed that montelukast was not able to induce adipocyte differentiation.  In contrast, zafirlukast triggered moderate lipid accumulation compared to rosiglitazone and upregulated PPARγ target genes.  In addition, we found that montelukast and zafirlukast display antagonistic activities concerning recruitment of the PPARγ cofactor CBP upon ligand binding suggesting that both compounds act as PPARγ modulators.  In addition, zafirlukast impaired the TNFα triggered phosphorylation of PPARγ2 on serine 273.  Thus, zafirlukast is a novel dual sEH/PPARγ modulator representing an excellent starting point for the further development of this compound class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcShvSmLL7MEOacY7HFp97qMfW6udTcc2eYwblcD3yeXJ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M%252FhvVOhsw%253D%253D&md5=bb60a84a4417a8bc878c5a420043a395</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2019.00263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2019.00263%26sid%3Dliteratum%253Aachs%26aulast%3DG%25C3%25B6bel%26aufirst%3DT.%26aulast%3DDiehl%26aufirst%3DO.%26aulast%3DHeering%26aufirst%3DJ.%26aulast%3DMerk%26aufirst%3DD.%26aulast%3DAngioni%26aufirst%3DC.%26aulast%3DWittmann%26aufirst%3DS.%2BK.%26aulast%3DLa%2BBuscato%26aufirst%3DE.%26aulast%3DKottke%26aufirst%3DR.%26aulast%3DWeizel%26aufirst%3DL.%26aulast%3DSchader%26aufirst%3DT.%26aulast%3DMaier%26aufirst%3DT.%2BJ.%26aulast%3DGeisslinger%26aufirst%3DG.%26aulast%3DSchubert-Zsilavecz%26aufirst%3DM.%26aulast%3DSteinhilber%26aufirst%3DD.%26aulast%3DProschak%26aufirst%3DE.%26aulast%3DKahnt%26aufirst%3DA.%2BS.%26atitle%3DZafirlukast%2520is%2520a%2520dual%2520modulator%2520of%2520human%2520soluble%2520epoxide%2520hydrolase%2520and%2520peroxisome%2520proliferator-activated%2520receptor%2520%25CE%25B3%26jtitle%3DFront.%2520Pharmacol.%26date%3D2019%26volume%3D10%26spage%3D263%26doi%3D10.3389%2Ffphar.2019.00263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zebisch, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voigt, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wabitsch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandsch, M.</span></span> <span> </span><span class="NLM_article-title">Protocol for effective differentiation of 3T3-L1 cells to adipocytes</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>425</i></span>,  <span class="NLM_fpage">88</span>– <span class="NLM_lpage">90</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2012.03.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1016%2Fj.ab.2012.03.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=22425542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A528%3ADC%252BC38XlvFyrurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=425&publication_year=2012&pages=88-90&author=K.+Zebischauthor=V.+Voigtauthor=M.+Wabitschauthor=M.+Brandsch&title=Protocol+for+effective+differentiation+of+3T3-L1+cells+to+adipocytes&doi=10.1016%2Fj.ab.2012.03.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Protocol for effective differentiation of 3T3-L1 cells to adipocytes</span></div><div class="casAuthors">Zebisch, Katja; Voigt, Valerie; Wabitsch, Martin; Brandsch, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">425</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">88-90</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In this note, we present a detailed procedure for highly effective and reproducible 3T3-L1 cell differentiation.  Due to their potential to differentiate from fibroblasts to adipocytes, 3T3-L1 cells are widely used for studying adipogenesis and the biochem. of adipocytes.  However, using different kits and protocols published so far, we were not able to obtain full differentiation of the currently available American Type Culture Collection (ATCC) 3T3-L1 cell lots.  Using rosiglitazone (2 μM) as an addnl. prodifferentiative agent, we achieved apparently complete differentiation of 3T3-L1 cells within 10 to 12 days that persisted for at least up to cell culture passage 10.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBHtWKXZs9I7Vg90H21EOLACvtfcHk0ligiAz_GNVoUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlvFyrurw%253D&md5=6e59b59b38fb7a42a80d0289c12766e6</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2012.03.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2012.03.005%26sid%3Dliteratum%253Aachs%26aulast%3DZebisch%26aufirst%3DK.%26aulast%3DVoigt%26aufirst%3DV.%26aulast%3DWabitsch%26aufirst%3DM.%26aulast%3DBrandsch%26aufirst%3DM.%26atitle%3DProtocol%2520for%2520effective%2520differentiation%2520of%25203T3-L1%2520cells%2520to%2520adipocytes%26jtitle%3DAnal.%2520Biochem.%26date%3D2012%26volume%3D425%26spage%3D88%26epage%3D90%26doi%3D10.1016%2Fj.ab.2012.03.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Daina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michielin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoete, V.</span></span> <span> </span><span class="NLM_article-title">SwissADME: a free web tool to evaluate pharmacokinetics, drug-likeness and medicinal chemistry friendliness of small molecules</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">42717</span>, <span class="refDoi"> DOI: 10.1038/srep42717</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=10.1038%2Fsrep42717" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=28256516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;key=1%3ACAS%3A280%3ADC%252BC1czisFSrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=42717&author=A.+Dainaauthor=O.+Michielinauthor=V.+Zoete&title=SwissADME%3A+a+free+web+tool+to+evaluate+pharmacokinetics%2C+drug-likeness+and+medicinal+chemistry+friendliness+of+small+molecules&doi=10.1038%2Fsrep42717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">SwissADME: a free web tool to evaluate pharmacokinetics, drug-likeness and medicinal chemistry friendliness of small molecules</span></div><div class="casAuthors">Daina Antoine; Michielin Olivier; Zoete Vincent; Michielin Olivier; Michielin Olivier</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">42717</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">To be effective as a drug, a potent molecule must reach its target in the body in sufficient concentration, and stay there in a bioactive form long enough for the expected biologic events to occur.  Drug development involves assessment of absorption, distribution, metabolism and excretion (ADME) increasingly earlier in the discovery process, at a stage when considered compounds are numerous but access to the physical samples is limited.  In that context, computer models constitute valid alternatives to experiments.  Here, we present the new SwissADME web tool that gives free access to a pool of fast yet robust predictive models for physicochemical properties, pharmacokinetics, drug-likeness and medicinal chemistry friendliness, among which in-house proficient methods such as the BOILED-Egg, iLOGP and Bioavailability Radar.  Easy efficient input and interpretation are ensured thanks to a user-friendly interface through the login-free website http://www.swissadme.ch.  Specialists, but also nonexpert in cheminformatics or computational chemistry can predict rapidly key parameters for a collection of molecules to support their drug discovery endeavours.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRl_Uk5dOMKs6qcneEtCi92fW6udTcc2eYGaaZOxbb5Nrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1czisFSrtg%253D%253D&md5=9715b8cb8a34b17c4c73ff69a5a8cc50</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1038%2Fsrep42717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep42717%26sid%3Dliteratum%253Aachs%26aulast%3DDaina%26aufirst%3DA.%26aulast%3DMichielin%26aufirst%3DO.%26aulast%3DZoete%26aufirst%3DV.%26atitle%3DSwissADME%253A%2520a%2520free%2520web%2520tool%2520to%2520evaluate%2520pharmacokinetics%252C%2520drug-likeness%2520and%2520medicinal%2520chemistry%2520friendliness%2520of%2520small%2520molecules%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D42717%26doi%3D10.1038%2Fsrep42717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3LJW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3LJW','PDB','3LJW'); return false;">PDB: 3LJW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKD','PDB','5DKD'); return false;">PDB: 5DKD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZS3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZS3','PDB','6ZS3'); return false;">PDB: 6ZS3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZS4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZS4','PDB','6ZS4'); return false;">PDB: 6ZS4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZN6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZN6','PDB','6ZN6'); return false;">PDB: 6ZN6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZNV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZNV','PDB','6ZNV'); return false;">PDB: 6ZNV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3TLP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3TLP','PDB','3TLP'); return false;">PDB: 3TLP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FH7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FH7','PDB','5FH7'); return false;">PDB: 5FH7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZS2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZS2','PDB','6ZS2'); return false;">PDB: 6ZS2</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i54"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01242">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_78077"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01242?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01242</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings with activity data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_001.csv">CSV</a>)</p></li><li><p class="inline">Supporting analytical chemistry data for intermediates and final products and additional tables and figures (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_002.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_001.csv">jm0c01242_si_001.csv (2.56 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01242/suppl_file/jm0c01242_si_002.pdf">jm0c01242_si_002.pdf (7.03 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors of the bromodomain-ligand complexes are available in the Protein Data Bank (PDB) under accession codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZS2">6ZS2</a> (SMARCA4-<b>23</b> complex), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZS3">6ZS3</a> (PB1(5)-<b>23</b> complex), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZS4">6ZS4</a> (PB1(5)-<b>22</b> complex), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZN6">6ZN6</a> (PB1(2)-<i>R</i>-<b>26</b> complex), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZNV">6ZNV</a> (PB1(2)-<b>32</b> complex).</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01242&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-23%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01242%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01242" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799a86c19ee24eb","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
