
<html lang="en"     class="pb-page"  data-request-id="9b2dcf0e-a72a-4371-8c4e-0298d3729445"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/jm3018332;issue:issue:10.1021/jmcmar.2013.56.issue-5;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Exploiting an Allosteric Binding Site of PRMT3 Yields Potent and Selective Inhibitors" /></meta><meta name="dc.Creator" content="Feng  Liu" /></meta><meta name="dc.Creator" content="Fengling  Li" /></meta><meta name="dc.Creator" content="Anqi  Ma" /></meta><meta name="dc.Creator" content="Elena  Dobrovetsky" /></meta><meta name="dc.Creator" content="Aiping  Dong" /></meta><meta name="dc.Creator" content="Cen  Gao" /></meta><meta name="dc.Creator" content="Ilia  Korboukh" /></meta><meta name="dc.Creator" content="Jing  Liu" /></meta><meta name="dc.Creator" content="David  Smil" /></meta><meta name="dc.Creator" content="Peter J.  Brown" /></meta><meta name="dc.Creator" content="Stephen V.  Frye" /></meta><meta name="dc.Creator" content="Cheryl H.  Arrowsmith" /></meta><meta name="dc.Creator" content="Matthieu  Schapira" /></meta><meta name="dc.Creator" content="Masoud  Vedadi" /></meta><meta name="dc.Creator" content="Jian  Jin" /></meta><meta name="dc.Description" content="Protein arginine methyltransferases (PRMTs) play an important role in diverse biological processes. Among the nine known human PRMTs, PRMT3 has been implicated in ribosomal biosynthesis via asymmet..." /></meta><meta name="Description" content="Protein arginine methyltransferases (PRMTs) play an important role in diverse biological processes. Among the nine known human PRMTs, PRMT3 has been implicated in ribosomal biosynthesis via asymmet..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 27, 2013" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm3018332" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2013 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm3018332" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm3018332" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm3018332" /></link>
        
    
    

<title>Exploiting an Allosteric Binding Site of PRMT3 Yields Potent and Selective Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm3018332" /></meta><meta property="og:title" content="Exploiting an Allosteric Binding Site of PRMT3 Yields Potent and Selective Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0017.jpeg" /></meta><meta property="og:description" content="Protein arginine methyltransferases (PRMTs) play an important role in diverse biological processes. Among the nine known human PRMTs, PRMT3 has been implicated in ribosomal biosynthesis via asymmetric dimethylation of the 40S ribosomal protein S2 and in cancer via interaction with the DAL-1 tumor suppressor protein. However, few selective inhibitors of PRMTs have been discovered. We recently disclosed the first selective PRMT3 inhibitor, which occupies a novel allosteric binding site and is noncompetitive with both the peptide substrate and cofactor. Here we report comprehensive structure–activity relationship studies of this series, which resulted in the discovery of multiple PRMT3 inhibitors with submicromolar potencies. An X-ray crystal structure of compound 14u in complex with PRMT3 confirmed that this inhibitor occupied the same allosteric binding site as our initial lead compound. These studies provide the first experimental evidence that potent and selective inhibitors can be created by exploiting the allosteric binding site of PRMT3." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm3018332"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm3018332">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm3018332&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm3018332&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm3018332&amp;href=/doi/10.1021/jm3018332" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2013</span><span class="cit-fg-volume">, 56</span><span class="cit-fg-issue">, 5</span><span class="cit-fg-pageRange">, 2110-2124</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/56/5" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm301819k" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm301859s" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Exploiting an Allosteric Binding Site of PRMT3 Yields Potent and Selective Inhibitors</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Feng++Liu">Feng Liu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Fengling++Li">Fengling Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Anqi++Ma">Anqi Ma</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Elena++Dobrovetsky">Elena Dobrovetsky</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Aiping++Dong">Aiping Dong</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Cen++Gao">Cen Gao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ilia++Korboukh">Ilia Korboukh</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jing++Liu">Jing Liu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=David++Smil">David Smil</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Peter+J.++Brown">Peter J. Brown</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Stephen+V.++Frye">Stephen V. Frye</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Cheryl+H.++Arrowsmith">Cheryl H. Arrowsmith</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Matthieu++Schapira">Matthieu Schapira</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Masoud++Vedadi">Masoud Vedadi</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Jin">Jian Jin</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Center for Integrative Chemical Biology and Drug Discovery, Division of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div><div class="corresp-info"><strong>*</strong>Phone: (919) 843-8459 (J.J.); (416) 946-0897 (M.V.). Fax: (919) 843-8465 (J.J.). E-mail: <a href="/cdn-cgi/l/email-protection#513b38303f3b383f11243f327f343524"><span class="__cf_email__" data-cfemail="ff95969e91959691bf8a919cd19a9b8a">[email protected]</span></a> (J.J.); <a href="/cdn-cgi/l/email-protection#315c1f4754555055587144455e435e5f455e1f5250"><span class="__cf_email__" data-cfemail="472a6931222326232e073233283528293328692426">[email protected]</span></a> (M.V.).</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm3018332&amp;href=/doi/10.1021%2Fjm3018332" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2013</span></span><span class="cit-volume">, 56</span><span class="cit-issue">, 5</span><span class="cit-pageRange">, 2110–2124</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 27, 2013</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>12 December 2012</li><li><span class="item_label"><b>Published</b> online</span>27 February 2013</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 March 2013</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm3018332" title="DOI URL">https://doi.org/10.1021/jm3018332</a></div><div class="article_header-article-copyright"><strong>Copyright © 2013 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2110%26pageCount%3D15%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DFeng%2BLiu%252C%2BFengling%2BLi%252C%2BAnqi%2BMa%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D56%26issueNum%3D5%26contentID%3Djm3018332%26title%3DExploiting%2Ban%2BAllosteric%2BBinding%2BSite%2Bof%2BPRMT3%2BYields%2BPotent%2Band%2BSelective%2BInhibitors%26numPages%3D15%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2124%26publicationDate%3DMarch%2B2013">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm3018332"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2495</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">55</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm3018332" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Exploiting an Allosteric Binding Site of PRMT3 Yields Potent and Selective Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Feng&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Fengling&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Anqi&quot;,&quot;last_name&quot;:&quot;Ma&quot;},{&quot;first_name&quot;:&quot;Elena&quot;,&quot;last_name&quot;:&quot;Dobrovetsky&quot;},{&quot;first_name&quot;:&quot;Aiping&quot;,&quot;last_name&quot;:&quot;Dong&quot;},{&quot;first_name&quot;:&quot;Cen&quot;,&quot;last_name&quot;:&quot;Gao&quot;},{&quot;first_name&quot;:&quot;Ilia&quot;,&quot;last_name&quot;:&quot;Korboukh&quot;},{&quot;first_name&quot;:&quot;Jing&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Smil&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;J. Brown&quot;},{&quot;first_name&quot;:&quot;Stephen&quot;,&quot;last_name&quot;:&quot;V. Frye&quot;},{&quot;first_name&quot;:&quot;Cheryl&quot;,&quot;last_name&quot;:&quot;H. Arrowsmith&quot;},{&quot;first_name&quot;:&quot;Matthieu&quot;,&quot;last_name&quot;:&quot;Schapira&quot;},{&quot;first_name&quot;:&quot;Masoud&quot;,&quot;last_name&quot;:&quot;Vedadi&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Jin&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2013&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;27&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;volume&quot;:&quot;56&quot;,&quot;pages&quot;:&quot;2110-2124&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm3018332&quot;},&quot;abstract&quot;:&quot;Protein arginine methyltransferases (PRMTs) play an important role in diverse biological processes. Among the nine known human PRMTs, PRMT3 has been implicated in ribosomal biosynthesis via asymmetric dimethylation of the 40S ribosomal protein S2 and in cancer via interaction with the DAL-1 tumor suppressor protein. However, few selective inhibitors of PRMTs have been discovered. We recently disclosed the first selective PRMT3 inhibitor, which occupies a novel allosteric binding site and is noncompetitive with both the peptide substrate and cofactor. Here we report comprehensive structure–activity relationship studies of this series, which resulted in the discovery of multiple PRMT3 inhibitors with submicromolar potencies. An X-ray crystal structure of compound 14u in complex with PRMT3 confirmed that this inhibitor occupied the same allosteric binding site as our initial lead compound. These studies provide the first experimental evidence that potent and selective inhibitors can be created by exploiting the &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm3018332&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm3018332" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm3018332&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm3018332" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm3018332&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm3018332" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm3018332&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm3018332&amp;href=/doi/10.1021/jm3018332" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm3018332" /></input><a href="/doi/pdf/10.1021/jm3018332" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm3018332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm3018332%26sid%3Dliteratum%253Aachs%26pmid%3D23445220%26genre%3Darticle%26aulast%3DLiu%26date%3D2013%26atitle%3DExploiting%2Ban%2BAllosteric%2BBinding%2BSite%2Bof%2BPRMT3%2BYields%2BPotent%2Band%2BSelective%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D56%26issue%3D5%26spage%3D2110%26epage%3D2124%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291011" title="High-performance liquid chromatography">High-performance liquid chromatography</a>,</li><li><a href="/action/doSearch?ConceptID=292271" title="Urea">Urea</a>,</li><li><a href="/action/doSearch?ConceptID=291337" title="Carbon">Carbon</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291774" title="Nuclear magnetic resonance spectroscopy">Nuclear magnetic resonance spectroscopy</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/56/5" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jmcmar.2013.56.issue-5/production/jmcmar.2013.56.issue-5.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3018332&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Protein arginine methyltransferases (PRMTs) play an important role in diverse biological processes. Among the nine known human PRMTs, PRMT3 has been implicated in ribosomal biosynthesis via asymmetric dimethylation of the 40S ribosomal protein S2 and in cancer via interaction with the DAL-1 tumor suppressor protein. However, few selective inhibitors of PRMTs have been discovered. We recently disclosed the first selective PRMT3 inhibitor, which occupies a novel allosteric binding site and is noncompetitive with both the peptide substrate and cofactor. Here we report comprehensive structure–activity relationship studies of this series, which resulted in the discovery of multiple PRMT3 inhibitors with submicromolar potencies. An X-ray crystal structure of compound <b>14u</b> in complex with PRMT3 confirmed that this inhibitor occupied the same allosteric binding site as our initial lead compound. These studies provide the first experimental evidence that potent and selective inhibitors can be created by exploiting the allosteric binding site of PRMT3.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23750" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23750" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Among epigenetic writers, histone methyltransferases (HMTs, also known as protein methyltransferases (PMTs)) are divided into two categories: protein lysine methyltransferases (PKMTs) and protein arginine methyltransferases (PRMTs), which catalyze lysine and arginine methylation, respectively.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> Nine human PRMTs have been identified to date and are further divided into three subtypes.<a onclick="showRef(event, 'ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5">(3-5)</a> Type I PRMTs (PRMT1–4,6,8) catalyze arginine monomethylation and asymmetric dimethylation, while type II PRMTs such as PRMT5 catalyze arginine monomethylation and symmetric dimethylation. Recently, PRMT7 was characterized as a type III PRMT which catalyzes only arginine monomethylation.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> In addition to histone substrates, PRMTs also methylate many nonhistone proteins.<a onclick="showRef(event, 'ref3 ref7'); return false;" href="javascript:void(0);" class="ref ref3 ref7">(3, 7)</a> PRMTs usually methylate GAR (glycine- and arginine-rich) motifs in their substrates,<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> except for PRMT4, which instead methylates PGM (proline-, glycine-, methionine-, and arginine-rich) motifs.<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10, 11)</a> PRMT5 methylates both GAR and PGM motifs.<a onclick="showRef(event, 'ref10 ref12'); return false;" href="javascript:void(0);" class="ref ref10 ref12">(10, 12)</a> Protein arginine methylation catalyzed by PRMTs plays a key role in various biological processes, including gene expression, transcriptional regulation, signal transduction, protein and RNA subcellular localization, RNA splicing, and DNA damage repair.<a onclick="showRef(event, 'ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5">(3-5)</a> Dysregulation of PRMTs has been implicated in a number of human conditions, including cancer.<a onclick="showRef(event, 'ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5">(3-5)</a></div><div class="NLM_p">PRMT3 (protein arginine methyltransferase 3), a type I PRMT, was first reported in 1998<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and subsequently shown to catalyze asymmetric dimethylation of GAR motifs in its main substrate: the 40S ribosomal protein S2 (rpS2).<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14, 15)</a> The dimethylation of rpS2 by PRMT3 results in stabilization of rpS2 and influences ribosomal biosynthesis.<a onclick="showRef(event, 'ref4 ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref4 ref14 ref15 ref16">(4, 14-16)</a> PRMT3 has also been reported to methylate the recombinant mammalian nuclear poly(A)-binding protein (PABPN1)<a onclick="showRef(event, 'ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19">(17-19)</a> and a histone peptide (H4 1–24) in vitro.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> In addition, the protein complex consisting of PRMT3, the von Hippel–Lindau (VHL) tumor suppressor protein, and ARF (alternative reading frame) methylates the tumor suppressor p53.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Importantly, the tumor suppressor DAL-1 (differentially expressed in adenocarcinoma of the lung-1) inhibits the methyltransferase activity of PRMT3 via its interaction with PRMT3, suggesting that DAL-1 may affect tumor growth by regulating PRMT3 function.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Therefore, pharmacologic inhibition of PRMT3 may offer a potentially viable option for treating the tumors that display epigenetic down-regulation of DAL-1. In addition, PRMT3 expression levels are elevated in myocardial tissue from patients with atherosclerosis,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> potentially implicating PRMT3 in this and related diseases. Lastly, PRMT3 function has been reported to be essential for dendritic spine maturation in rats.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></div><div class="NLM_p last">Selective small-molecule inhibitors of PRMTs and PKMTs are important tools for investigating the biology of this emerging target class and testing disease and therapeutic hypotheses regarding these enzymes.<a onclick="showRef(event, 'ref5 ref25 ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref5 ref25 ref26 ref27 ref28">(5, 25-28)</a> However, only a limited number of selective inhibitors of PRMTs<a onclick="showRef(event, 'ref29 ref30 ref31 ref32 ref33 ref34 ref35 ref37 ref39 ref40 ref41 ref42 ref43 ref44 ref36 ref38'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31 ref32 ref33 ref34 ref35 ref37 ref39 ref40 ref41 ref42 ref43 ref44 ref36 ref38">(29-42, 44, 46)</a> and PKMTs<a onclick="showRef(event, 'ref45 ref36 ref47 ref38 ref49 ref50 ref51 ref52 ref53 ref54 ref55 ref56 ref57 ref58'); return false;" href="javascript:void(0);" class="ref ref45 ref36 ref47 ref38 ref49 ref50 ref51 ref52 ref53 ref54 ref55 ref56 ref57 ref58">(43-56)</a> have been reported. In particular, selective small-molecule inhibitors of PRMT3 were lacking. We recently disclosed the first selective, allosteric inhibitor (compound <b>1</b> in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>A) of PRMT3.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(57)</a> This inhibitor occupies neither the substrate binding groove nor the cofactor binding site. Instead, it occupies a novel allosteric binding site revealed by the X-ray crystal structure of compound <b>1</b> in complex with PRMT3 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SMQ">3SMQ</a>). Subsequent mechanism of action (MOA) studies confirmed that this inhibitor is noncompetitive with both the peptide substrate and the cofactor.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(57)</a> Here we report structure–activity relationship (SAR) studies that focused on extensively exploring three regions of the scaffold represented by compound <b>1</b>. We describe the design, synthesis, and biochemical evaluation of novel compounds, which resulted in the discovery of potent and selective PRMT3 inhibitors. We obtained an X-ray crystal structure of the compound <b>14u</b>–PRMT3 complex, which confirmed that this inhibitor occupied the same allosteric binding site previously described. These studies establish that the allosteric binding site of PRMT3 is druggable and can be exploited to generate potent and selective inhibitors.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61107" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61107" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Design and Synthesis</h3><div class="NLM_p">To improve the potency of compound <b>1</b> (PRMT3, IC<sub>50</sub> = 2.5 μM)<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(57)</a> and demonstrate that potent and selective inhibitors can be generated by exploiting the allosteric binding site of PRMT3, we investigated the following three regions of the chemical series represented by compound <b>1</b>: (1) the left-hand side (LHS) bicyclic aromatic moiety, (2) the middle urea moiety, and (3) the right-hand side (RHS) moiety (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>A). We previously reported that the uncommon cyclohexenylethyl group of compound <b>1</b> could be replaced by a more common and potentially stable group, the cyclohexylethyl group of compound <b>2</b> (PRMT3, IC<sub>50</sub> = 1.0 μM), without any potency loss, and showed that the alkene functionality was unnecessary (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>A).<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(57)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Structure of PRMT3 inhibitors <b>1</b> and <b>2</b> and three regions explored for SAR. (B) X-ray crystal structure of the inhibitor <b>1</b>–PRMT3 complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SMQ">3SMQ</a>). Key interactions include (1) a hydrogen bond between the 2-nitrogen of the benzothiadiazole and T466, (2) two hydrogen bonds between the two amino groups of the middle urea moiety and E422, (3) a hydrogen bond between the oxygen of the urea moiety and R396, and (4) hydrophobic interactions between the cyclohexenylethyl group and a nonpolar surface.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3018332&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Because the X-ray crystal structure of compound <b>1</b> in complex with PRMT3 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SMQ">3SMQ</a>) reveals that the LHS benzothiadiazole moiety fits tightly in the allosteric pocket and forms a hydrogen bond with T466 (threonine 466) (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>B),<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(57)</a> we designed the compounds outlined in Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a> and Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> to determine whether the benzothiadiazole ring can be replaced with other fused bicyclic aromatic groups. A representative synthesis of these compounds (<b>3</b>–<b>7</b>) is shown in Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>. The desired compounds were prepared via a standard urea formation reaction from commercially available anilines and 2-cyclohexylethylamine using 1,1′-carbonyldiimidazole (CDI) as the coupling reagent.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0006.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>3</b>–<b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3018332&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) CDI, THF, 2-cyclohexylethylamine, rt, 41–88%.</p></p></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR of the LHS Bicyclic Aromatic Moiety</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0010.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0011.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">IC<sub>50</sub> determination experiments were performed in triplicate.</p></div><div class="footnote" id="t1fn2"><sup>Table b</sup><p class="last">The IC<sub>50</sub> value was previously reported.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(57)</a></p></div></div><div></div></div><div class="NLM_p">We previously synthesized several analogues to confirm the hydrogen bonds between the middle urea moiety of compound <b>1</b> and the R396 (arginine 396) and E422 (glutamate 422) of PRMT3 (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>B).<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(57)</a> To further investigate the middle urea moiety, we designed and synthesized the urea bioisosteres and more rigid analogues outlined in Scheme <a class="ref internalNav" href="#sch2" aria-label="2">2</a>. The key intermediate <b>8</b> was prepared according to published procedures.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(58)</a> Commercially available 6-nitrobenzothiazole was first ring-opened to yield the disulfide intermediate, which was then converted to the 6-nitrobenzothiadiazole intermediate. Subsequent reduction of the nitro group afforded the intermediate <b>8</b>. The nucleophilic substitution reaction of the intermediate <b>8</b> with 3,4-diethoxycyclobut-3-ene-1,2-dione<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(59)</a> formed the intermediate <b>9a</b>, which was then converted to the desired products <b>10</b> and <b>11</b> by a second nucleophilic substitution reaction with 2-cyclohexylethylamine or 2-amino-1-phenylethanone. Similarly, nucleophilic substitution reaction of the intermediate <b>8</b> with diphenyl cyanocarbonimidate yielded the intermediate <b>9b</b>, which was subsequently converted to the desired product <b>12</b> by a second substitution reaction. The aminotriazole <b>13</b> was formed from 3-cyclohexylpropanehydrazide and the intermediate <b>9c,</b> which was prepared from the intermediate <b>8</b>.</div><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0007.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds for Exploring the Middle Urea Moiety<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3018332&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) N<sub>2</sub>H<sub>4</sub>·H<sub>2</sub>O, EtOH, reflux; (b) H<sub>2</sub>O<sub>2</sub>, rt, 87% over two steps; (c) KNO<sub>2</sub>/H<sub>2</sub>SO<sub>4</sub>, 0–10 °C, 62%; (d) SnCl<sub>2</sub>/HCl, 65 °C, 5 min, 61%; (e) 3,4-diethoxycyclobut-3-ene-1,2-dione, Zn(OTf)<sub>2</sub>, EtOH, rt, 92%; (f) 2-cyclohexylethylamine or 2-amino-1-phenylethanone, <i>i-</i>PrOH, 120 °C, microwave, 78–85%; (g) diphenyl cyanocarbonimidate, <i>i</i>-PrOH, 100 °C, microwave; (h) 2-cyclohexylethylamine, <i>i</i>-PrOH, 100 °C, microwave, 38% over two steps; (i) BrCN, THF, 40 °C; (j) 3-cyclohexylpropanehydrazide, HCl, <i>i-</i>PrOH, 160 °C, microwave, 32% over two steps.</p></p></figure><div class="NLM_p">In the X-ray crystal structure of the inhibitor <b>1</b>–PRMT3 complex, the RHS cyclohexenylethyl moiety makes hydrophobic interactions with a surface formed from the two subunits of the PRMT3 homodimer (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>B). Because it was not clear what was required for these hydrophobic interactions and whether additional interactions could be made, we extensively explored this region by introducing functional groups with varying steric and electronic characteristics to either an aliphatic or an aromatic moiety. Synthesis of compounds <b>14a</b>–<b>v</b> and <b>15 a</b>–<b>e</b> (see structures in Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>) is outlined in Scheme <a class="ref internalNav" href="#sch3" aria-label="3">3</a>. Compounds <b>14a–v</b> were prepared via a urea formation reaction from the intermediate <b>8</b> and various amines. Compounds <b>15a–e</b> were obtained via a urea formation reaction followed by the removal of the Boc protecting group.</div><figure id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0008.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of the Compounds Outlined in Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3018332&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) CDI, DMF, various amines, rt, 35–55%; (b) TFA, DCM, rt, 100%.</p></p></figure><div class="NLM_p">To further investigate the benzoyl moiety of compound <b>14u</b>, the first PRMT3 inhibitor with submicromolar potency (see below and Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>), we designed and synthesized the compounds outlined in Scheme <a class="ref internalNav" href="#sch4" aria-label="4">4</a> and Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>. Compound <b>16</b> was prepared via a urea formation reaction followed by the removal of the Boc protecting group, similar to the synthesis of compounds <b>15a–e</b>. Compounds <b>17–21</b> and <b>24</b> were synthesized via a urea formation reaction from the intermediate <b>8</b> and various amines. Compounds <b>22</b> and <b>23</b> were obtained via oxidation of respective alcohols <b>21</b> and <b>19</b> using the sulfur trioxide–pyridine complex. Compounds <b>16</b>, <b>17</b>, <b>19</b>, <b>20</b>, and <b>21</b> were prepared as racemic mixtures.</div><figure id="sch4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0009.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of the Compounds Outlined in Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3018332&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) CDI, DMF, various amines, rt, 35–55%; (b) TFA, DCM, rt, 100%; (c) PySO<sub>3</sub>, Et<sub>3</sub>N, DCM/DMSO, rt, 76–78%.</p></p></figure><div class="NLM_p last">These synthesized compounds were evaluated in a radioactive biochemical assay which measures the transfer of the tritiated methyl group from the cofactor [<sup>3</sup>H]-<i>S</i>-adenosylmethionine (SAM) to a peptide substrate. IC<sub>50</sub> values of these compounds in this assay are summarized in Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>–<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> SAR of the LHS Bicyclic Aromatic Moiety</h3><div class="NLM_p last">We previously reported that the replacement of the benzothiadiazole moiety (compound <b>2</b>) with the benzothiazole (compound <b>3</b>), which abolished the hydrogen bond with T466, resulted in complete loss of potency.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(57)</a> This SAR finding confirmed the key hydrogen bond interaction between the middle nitrogen of the benzothiadiazole moiety and T466, revealed by the X-ray crystal structure of compound <b>1</b> in complex with PRMT3. Interestingly, the replacement of the benzothiadiazole moiety (compound <b>2</b>) with the benzotriazole (compound <b>4</b>), indazole (compound <b>5</b> or <b>6</b>), or benzoxazolone (compound <b>7</b>) moiety also led to total loss of potency (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>), suggesting that, in addition to making the hydrogen bond interaction with T466, the benzothiadiazole moiety is particularly well accommodated by the PRMT3 allosteric binding pocket due to its unique electronic nature and steric fit. Because these relatively minor modifications to the benzothiadiazole moiety were not tolerated, we did not attempt to introduce more drastic modifications such as substituted monocyclic aromatic or nonaromatic groups and have kept this LHS moiety constant during the subsequent SAR exploration of this hit series.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> SAR of the Middle Urea Moiety</h3><div class="NLM_p">The X-ray crystal structure of the inhibitor <b>1</b>–PRMT3 complex reveals that the carbonyl group of the middle urea moiety forms a hydrogen bond with the guanidinium group of R396 and the two amino groups of the urea moiety form two hydrogen bonds with the carboxylate of E422 (glutamate 422) (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>B).<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(57)</a> We previously synthesized a number of analogues for probing these hydrogen bond interactions and found that the replacement of the oxygen of the urea moiety, which abolished the hydrogen bond with R396, and <i>N</i>-methylation of either nitrogen of the urea moiety, which impaired the hydrogen bonds with E422, led to complete loss of potency.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(57)</a> These SAR results confirmed the key hydrogen bond interactions between the urea moiety of inhibitor <b>1</b> and the allosteric binding site of PRMT3.</div><div class="NLM_p">We further investigated the middle urea moiety by design, synthesis, and biological evaluation of the urea bioisosteres and more rigid analogues outlined in Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>. The diaminosquarate <b>10</b> displayed >100-fold potency loss compared with its urea analogue (<b>2</b>). Similarly, the diaminosquarate <b>11</b> was >200-fold less potent compared with its urea analogue (<b>14u</b>), which contains a preferred RHS moiety (see below and Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). In addition, the cyanoguanidine <b>12</b> and aminotriazole <b>13</b> showed no inhibitory activity against PRMT3 (IC<sub>50</sub> values >100 μM). Taken together, these results suggest that the urea moiety is ideal for interacting with the allosteric binding pocket of PRMT3. We therefore held this moiety constant during subsequent SAR exploration.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Bioisosteres of the Middle Urea Moiety</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0012.gif" alt="" id="fx3" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>Table a</sup><p class="last">IC<sub>50</sub> determination experiments were performed in triplicate.</p></div></div><div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> SAR of the RHS Moiety</h3><div class="NLM_p">We previously reported that the uncommon cyclohexenylethyl group of compound <b>1</b> could be replaced by a more common and potentially stable group, the cyclohexylethyl group of compound <b>2</b>, without any potency loss (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>).<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(57)</a> To improve potency, we extensively investigated this region by introducing functional groups with varying steric and electronic characteristics to either an aliphatic or an aromatic moiety (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). The replacement of the cyclohexyl ring with a heterocycle such as tetrahydropyran (compound <b>14a</b>), 1-piperidine (compound <b>14b</b>), morpholine (compound <b>14c</b>), 1-pyrrolidine (<b>14e</b>), and 4-piperidine (compound <b>15a</b>) generally resulted in a decrease in potency. Similarly, compounds <b>15b</b>–<b>e</b>, which contain a respective terminal amino group to potentially form additional hydrogen bond interaction(s) with backbone amides, were also less potent than compound <b>2</b>. A longer chain (three-carbon versus two-carbon) was tolerated but did not lead to a significant increase in potency (<b>14d</b> versus <b>14c</b>, <b>15d</b> versus <b>15c</b>, and <b>15e</b> versus <b>15a</b>). On the other hand, a shorter chain (one-carbon versus two-carbon) resulted in a complete loss of potency (<b>14f</b> versus <b>2</b>). The replacement of the cyclohexyl group with a hydrogen (compound <b>14g</b>) or <i>tert</i>-butyl group (compound <b>14h</b>) also led to a significant decrease in potency. We next explored the replacement of the cyclohexylethyl group with a motif containing an aromatic ring. Although the unsubstituted benzyl group (compound <b>14i</b>), three pyridinylmethyl groups (compounds <b>14j</b>–<b>l</b>), and two benzyl groups substituted with an electron-donating group (compounds <b>14m</b>,<b>n</b>) led to a significant decrease in potency compared with the cyclohexylethyl group (compound <b>2</b>), we were pleased to find that strong electron-withdrawing groups such as nitro (compound <b>14o</b>) and methylsulfonyl (compound <b>14p</b>) groups resulted in similar or higher potency compared with the cyclohexylethyl group (compound <b>2</b>). Other electron-withdrawing groups such as cyano (compound <b>14q</b>), trifluoromethyl (compound <b>14r</b>), and fluoro (compound <b>14s</b>) groups gave either similar or slightly lower potency compared with the cyclohexylethyl group (compound <b>2</b>) but significantly better potency compared with the unsubstituted compound (<b>14i</b>) or electron-donating groups (compounds <b>14m</b>,<b>n</b>). Interestingly, moving the trifluoromethyl group from the <i>para</i>-position (compound <b>14r</b>) to the <i>ortho</i>-position (compound <b>14t</b>) resulted in a significant decrease in potency. Further exploration of this aromatic moiety led to the discovery of compound <b>14u</b>, which contains a benzoyl group, with an IC<sub>50</sub> of 0.48 μM and good selectivity for PRMT3 over other methyltransferases (see below). We also found that a longer chain (three-carbon versus one-carbon) maintained good potency (<b>14v</b> versus <b>14s</b>), which is consistent with the finding in the aliphatic subseries (see above).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. SAR of the RHS Moiety: Part I<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0013.gif" alt="" id="fx4" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0014.gif" alt="" id="fx5" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>Table a</sup><p class="last">IC<sub>50</sub> determination experiments were performed in triplicate.</p></div></div><div></div></div><div class="NLM_p">Encouraged by the discovery of the first PRMT3 inhibitors with submicromolar potencies, we further investigated the benzoyl moiety of compound <b>14u</b> (Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>). The replacement of the carbonyl group (compound <b>14u</b>) with an α-amino (compound <b>16</b>), α-hydroxyl (compound <b>17</b>), or α,α-difluoro (compound <b>18</b>) group led to a decrease in potency. Adding a <i>p</i>- or <i>o</i>-fluoro group to the phenyl group did not significantly change the potency (compounds <b>19</b> and <b>20</b> versus compound <b>17</b>). Interestingly, compound <b>21</b>, which possesses a cyclohexyl group, was slightly more potent than compound <b>17</b>, which possesses a phenyl group instead. Although the ketone <b>22</b> had potency similar to that of its alcohol analogue (compound <b>21</b>), the ketone <b>23</b> was about 10-fold more potent than its corresponding alcohol (<b>19</b>). Further modifications of the ketone <b>22</b> led to the discovery of the amide <b>24</b>, which is the most potent PRMT3 inhibitor to date with an IC<sub>50</sub> of 0.23 μM. Taken together, these results indicate that modifications to the RHS moiety are well tolerated and can lead to a significant increase in potency. These studies resulted in the discovery of multiple PRMT3 inhibitors with submicromolar potencies. Among them, compound <b>14u</b> was further evaluated in subsequent selectivity, X-ray crystallography, and computational modeling studies.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. SAR of the RHS Moiety: Part II<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0015.gif" alt="" id="fx6" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0016.gif" alt="" id="fx7" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>Table a</sup><p class="last">IC<sub>50</sub> determination experiments were performed in triplicate.</p></div></div><div></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Selectivity Assessment of Compound <b>14u</b></h3><div class="NLM_p">We next determined the selectivity of compound <b>14u</b> against a number of methyltransferases, including PRMTs, PKMTs, and DNMT1 (DNA methyltransferase 1). As shown in Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>, compound <b>14u</b> was at least 40-fold selective for PRMT3 over G9a, GLP, and SUV39H2 and completely inactive against PRMT5, SETD7, PRC2, SETD8, SETDB1, SUV420H1, SUV420H2, MLL1, SMYD3, SMYD2, DOT1L, and DNMT1. These results are consistent with the selectivity results of compound <b>1</b> reported previously,<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(57)</a> suggesting that targeting the allosteric binding site of PRMT3 can yield selective inhibitors.</div><figure id="fig2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0003.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Compound <b>14u</b> was selective for PRMT3 over other PRMTs, PKMTs, and DNMT1. The effects of compound <b>14u</b> on the activity of the following methyltransferases were assessed: (A) PRMT3 (●, IC<sub>50</sub> = 0.48 ± 0.01 μM), G9a (○, IC<sub>50</sub> = 20 ± 1 μM), GLP (▼, IC<sub>50</sub> = 25 ± 8 μM), and SUV39H2 (△, IC<sub>50</sub> = 22 ± 1 μM); (B) SETD7 (●), PRC2 complex (○), SETD8 (▼), SETDB1 (△), SUV420H1 (■), SUV420H2 (□), MLL1 complex (◇), SMYD3 (▲), SMYD2 (∇), DOT1L (□), PRMT5–MEP50 complex (◆), and DNMT1 (⬢).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3018332&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> X-ray Crystal Structure of the Compound <b>14u</b>–PRMT3 Complex</h3><div class="NLM_p">To better understand structural determinants for the increased potency of compound <b>14u</b>, we solved the crystal structure of PRMT3 in complex with compound <b>14u</b>. Similar to the binding mode of compound <b>1</b> reported previously,<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(57)</a> compound <b>14u</b> binds at the allosteric site located at the interface between the two subunits of the PRMT3 homodimer (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>A). The benzothiadiazole ring and urea moiety engage hydrogen bonds with surrounding side chains (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>B). The benzoyl group is stacked against the guanidinium group of R396, which may explain, in part, the increased binding affinity over that of the parent compound. Nevertheless, we were surprised to observe that the additional carbonyl group is not better exploited in the crystal structure.</div><figure id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0004.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. X-ray crystal structure of PRMT3 in complex with the inhibitor <b>14u</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HSG">4HSG</a>). (A) Compound <b>14u</b> binds at the allosteric site of PRMT3 and contacts both subunits of the homodimer, as was observed with compound <b>1</b>.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(57)</a> (B) Critical interactions include hydrogen bonds with T466, E422, and R396. The letter preceding the residue number indicates the PRMT3 chain (A or B).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3018332&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Modeling of Compound <b>14u</b> Binding to PRMT3 Identifies a Potential Hydrogen Bond</h3><div class="NLM_p">Considering the significant increase in potency accompanying the introduction of a carbonyl group to the linker moiety, we were surprised by the absence of a hydrogen bond at this position in the crystal structure. We also noted that alternate rotameric states of K392 could bring its side chain within hydrogen-bonding distance of compound <b>14u</b>’s carbonyl oxygen. To see whether such a conformation was energetically relevant, we ran a Monte Carlo-based energy minimization of the K392 side chain and found that the second lowest energy conformation was indeed forming a hydrogen bond with compound <b>14u</b>, within less than 0.5 kcal/mol of the lowest energy state (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>). This suggests that K392 oscillates between two conformational states that are nearly equivalent energetically, one forming a hydrogen bond with compound <b>14u</b>, the other not.</div><figure id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0005.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Monte Carlo energy minimization identifies a low-energy conformation of K392, which interacts with the carbonyl oxygen of compound <b>14u</b> via a hydrogen bond. Three independent energy minimization simulations (conformational samplings 1–3) converge toward similar low-energy conformations for K392. Conformational states placing the ε-nitrogen of K392 within hydrogen-bonding distance of <b>14u</b> are listed in bold. The four lowest energy states from sampling 1 (A–D) and the fourth state from sampling 2 (E) are shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3018332&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17340" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17340" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We designed, synthesized, and evaluated biochemically a series of novel compounds that explore three regions of the scaffold represented by compound <b>1</b>. The key SAR results revealed by these studies include the following: (1) Modifications to the LHS benzothiadiazole moiety were not tolerated, which is consistent with the key finding revealed by the compound <b>1</b>–PRMT3 X-ray cocrystal structure that the benzothiadiazole ring fits tightly in the allosteric pocket. (2) Similarly, modifications to the middle urea moiety were not tolerated, again consistent with the findings by the X-ray cocrystal structure. (3) On the other hand, modifications to the RHS cyclohexenylethyl moiety were well tolerated. Our extensive exploration of the RHS cyclohexenylethyl group resulted in the discovery of multiple PRMT3 inhibitors with submicromolar potencies. Among them, compound <b>14u</b> was selective for PRMT3 over 15 other PRMTs, PKMTs, and DNMT1. The X-ray crystal structure of the compound <b>14u</b>–PRMT3 complex confirmed that this inhibitor interacts with the same allosteric binding site. Taken together, these studies provide the first experimental evidence that the allosteric binding site of PRMT3 can be exploited to generate potent and selective inhibitors.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13644" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13644" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Chemistry General Procedures</h3><div class="NLM_p">HPLC spectra for all compounds were acquired using an Agilent 6110 series system with a UV detector set to 254 nm. Samples were injected (5 μL) onto an Agilent Eclipse Plus, 4.6 × 50 mm, 1.8 μM, C18 column at room temperature. Method 1: A linear gradient from 50% to 100% B (MeOH + 0.1% acetic acid) in 5.0 min was followed by pumping 100% B for another 2 min with A being H<sub>2</sub>O + 0.1% acetic acid. Method 2: A linear gradient from 10% to 100% B (MeOH + 0.1% acetic acid) in 5.0 min was followed by pumping 100% B for another 2 min with A being H<sub>2</sub>O + 0.1% acetic acid. The flow rate was 1.0 mL/min. Mass spectrometry (MS) data were acquired in positive ion mode using an Agilent 6110 single-quadrupole mass spectrometer with an electrospray ionization (ESI) source. Nuclear magnetic resonance (NMR) spectra were recorded on a Varian Mercury spectrometer at 400 MHz for proton (<sup>1</sup>H NMR) and 100 MHz for carbon (<sup>13</sup>C NMR); chemical shifts are reported in parts per million (δ). Preparative HPLC was performed on an Agilent Prep 1200 series with a UV detector set to 254 nm. Samples were injected onto a Phenomenex Luna, 75 × 30 mm, 5 μM, C<sub>18</sub> column at room temperature. The flow rate was 30 mL/min. A linear gradient was used with 10% (or 50%) MeOH (A) in 0.1% TFA in H<sub>2</sub>O (B) to 100% MeOH (A). HPLC was used to establish the purity of target compounds; all compounds had >95% purity using the HPLC methods described above. High-resolution (positive ion) mass spectrometry (HRMS) for compounds <b>14u</b> and <b>24</b> was performed using a Thermo LTqFT mass spectrometer under FT control at 100 000 resolution.</div><div class="NLM_p last">Synthesis of compounds <b>1</b>–<b>3</b> was reported previously.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(57)</a></div></div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> 1-(1<i>H</i>-Benzo[<i>d</i>][1,2,3]triazol-6-yl)-3-(2-cyclohexylethyl)urea (<b>4</b>)</h3><div class="NLM_p last">To a solution of 6-amino-1,2,3-benzotriazole (72 mg, 0.538 mmol) in DMF (2.0 mL) was added <i>N</i>,<i>N′</i>-carbonyldiimidazole (CDI; 106 mg, 0.654 mmol), and the resulting mixture was stirred for 5 h at rt. Cyclohexylethylamine (109 mg, 0.861 mmol) was then added. After being stirred for 6 h at rt, the resulting mixture was purified by preparative HPLC (50–100% methanol/0.1% TFA in H<sub>2</sub>O) to afford the title compound <b>4</b> as a white solid (65 mg, 42%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 15.25 (s, 1H), 8.73 (s, 1H), 8.08 (s, 1H), 7.84 (d, <i>J</i> = 9.2 Hz, 1H), 7.05 (d, <i>J</i> = 8.5 Hz, 1H), 6.16 (s, 1H), 3.17–3.08 (m, 2H), 1.81–1.55 (m, 5H), 1.42–1.05 (m, 6H), 0.97–0.81 (m, 2H); HPLC (method 2) 95%, <i>t</i><sub>R</sub> 5.51 min; MS (ESI) <i>m</i>/<i>z</i> 288 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> 1-(2-Cyclohexylethyl)-3-(1<i>H</i>-indazol-6-yl)urea (<b>5</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>5</b>. The title compound <b>5</b> was obtained as a white solid (122 mg, 51%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 12.69 (s, 1H), 8.51 (s, 1H), 7.90 (s, 1H), 7.88 (s, 1H), 7.55 (d, <i>J</i> = 8.6 Hz, 1H), 6.81 (dd, <i>J</i> = 8.7, 1.7 Hz, 1H), 6.09 (t, <i>J</i> = 5.5 Hz, 1H), 3.19–3.06 (m, 2H), 1.86–1.50 (m, 5H), 1.44–1.04 (m, 6H), 0.96–0.82 (m, 2H); HPLC (method 2) 96%, <i>t</i><sub>R</sub> 5.59 min; MS (ESI) <i>m</i>/<i>z</i> 287 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> 1-(2-Cyclohexylethyl)-3-(1<i>H</i>-indazol-5-yl)urea (<b>6</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>6</b>. The title compound <b>6</b> was obtained as a white solid (78 mg, 47%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>4</sub>-MeOH) δ 7.95 (d, <i>J</i> = 0.8 Hz, 1H), 7.78–7.75 (m, 1H), 7.45 (d, <i>J</i> = 8.9 Hz, 1H), 7.30 (dd, <i>J</i> = 8.9, 2.0 Hz, 1H), 3.26–3.21 (m, 2H), 1.83–1.62 (m, 5H), 1.47–1.13 (m, 6H), 1.02–0.91 (m, 2H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>4</sub>-MeOH) δ 159.0, 138.7, 134.4, 134.2, 124.5, 123.5, 111.6, 111.3, 38.8, 38.6, 36.6, 34.4 (two carbons), 27.7, 27.4 (two carbons); HPLC (method 2) 98%, <i>t</i><sub>R</sub> 5.45 min; MS  287 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> 1-(2-Cyclohexylethyl)-3-(2-oxo-2,3-dihydrobenzo[<i>d</i>]oxazol-6-yl)urea (<b>7</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>7</b>. The title compound <b>7</b> was obtained as a white solid (138 mg, 76%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 11.36 (s, 1H), 8.40 (s, 1H), 7.53 (d, <i>J</i> = 1.6 Hz, 1H), 7.00–6.85 (m, 2H), 6.04 (t, <i>J</i> = 5.6 Hz, 1H), 3.15–3.03 (m, 2H), 1.75–1.54 (m, 5H), 1.38–1.06 (m, 6H), 0.97–0.80 (m, 2H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>6</sub>-DMSO) δ 155.7, 155.1, 143.89, 136.1, 124.4, 113.5, 109.9, 100.8, 37.7, 37.2, 35.0, 33.1 (two carbons), 26.6, 26.2 (two carbons); HPLC (method 1) 100%, <i>t</i><sub>R</sub> 4.59 min; MS (ESI) <i>m</i>/<i>z</i> 304 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> 6-Amino-1,2,3-benzothiadiazole (<b>8</b>)</h3><div class="NLM_p last">6-Amino-1,2,3-benzothiadiazole (<b>8</b>) was synthesized according to previously reported procedures.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(57)</a></div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 3-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-ylamino)-4-((2-cyclohexylethyl)amino)cyclobut-3-ene-1,2-dione (<b>10</b>)</h3><div class="NLM_p last">To a stirred solution of 3,4-diethoxycyclobut-3-ene-1,2-dione (0.26 g, 1.5 mmol, 1.2 equiv) and zinc trifluoromethanesulfonate (59 mg, 0.12 mmol) in ethanol (4.0 mL) at room temperature was added <b>8</b> (184 mg, 1.22 mmol). After the solution was stirred for 1 h, a white precipitate was formed, which was centrifuged to remove the ethanol and washed with ethanol (2 mL) twice, yielding the intermediate as a white solid (309 mg, 92%). A mixture of the intermediate (67 mg, 0.244 mmol) and 2-cyclohexylethylamine (32 mg, 0.250 mmol) in 0.2 mL of <i>i</i>-PrOH was heated by microwave irradiation to 120 °C for 10 min in a sealed tube. After concentration in vacuo, the crude product was purified by preparative HPLC with a gradient from 50% MeOH (A) in 0.1% TFA in H<sub>2</sub>O (B) to 100% MeOH (A) to afford the title compound <b>10</b> as a white solid (74 mg, 85%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 10.12 (s, 1H), 8.62 (d, <i>J</i> = 9.0 Hz, 1H), 8.25 (s, 1H), 7.93–7.70 (m, 2H), 3.74–3.57 (m, 2H), 1.83–1.55 (m, 5H), 1.49 (q, <i>J</i> = 7.0 Hz, 2H), 1.43–1.28 (m, 1H), 1.29–1.05 (m, 3H), 1.01–0.82 (m, 2H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>6</sub>-DMSO) δ 184.9, 180.1, 169.8, 162.6, 153.9, 142.4, 140.6, 124.3, 119.3, 106.5, 41.6, 38.0, 34.1, 32.5 (two carbons), 26.0, 25.7 (two carbons); HPLC (method 1) 96%, <i>t</i><sub>R</sub> 5.39 min; MS (ESI) <i>m</i>/<i>z</i> 357 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 3-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-ylamino)-4-((2-oxo-2-phenylethyl)amino)cyclobut-3-ene-1,2-dione (<b>11</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>10</b> was followed for synthesis of compound <b>11</b>. The title compound <b>11</b> was obtained as a white solid (80 mg, 78%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 10.53 (s, 1H), 8.66 (d, <i>J</i> = 9.0 Hz, 1H), 8.32 (s, 1H), 8.17 (s, 1H), 8.05–7.96 (m, 2H), 7.87 (d, <i>J</i> = 8.9 Hz, 1H), 7.77–7.68 (m, 1H), 7.60 (t, <i>J</i> = 7.7 Hz, 2H), 5.29 (d, <i>J</i> = 5.6 Hz, 2H); HPLC (method 1) 98%, <i>t</i><sub>R</sub> 3.66 min; MS (ESI) <i>m</i>/<i>z</i> 365 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> (<i>E</i>)-1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-2-cyano-3-(2-cyclohexylethyl)guanidine (<b>12</b>)</h3><div class="NLM_p last">A mixture of <b>8</b> (65 mg, 0.430 mmol) and diphenyl cyanocarbonimidate (82 mg, 0.344 mmol) in 0.4 mL of <i>i</i>-PrOH was heated by microwave irradiation to 100 °C for 60 min in a sealed tube. To the resulting mixture was added 2-cyclohexylethylamine (109 mg, 0.86 mmol), and then the resulting mixuture was heated by microwave irradiation to 100 °C for another 30 min. The resulting residue was purified by preparative HPLC with a gradient from 50% MeOH (A) in 0.1% TFA in H<sub>2</sub>O (B) to 100% MeOH (A) to afford the title compound <b>12</b> as a white solid (43 mg, 38% in two steps): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 9.50 (s, 1H), 8.61 (d, <i>J</i> = 9.0 Hz, 1H), 8.22 (s, 1H), 7.65 (t, <i>J</i> = 5.6 Hz, 1H), 7.57 (dd, <i>J</i> = 9.0, 1.9 Hz, 1H), 3.33–3.25 (m, 2H), 1.77–1.56 (m, 5H), 1.48–1.39 (m, 2H), 1.36–1.06 (m, 4H), 0.98–0.82 (m, 2H); HPLC (method 1) 100%, <i>t</i><sub>R</sub> 5.12 min; MS (ESI) <i>m</i>/<i>z</i> 329 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> <i>N</i>-(5-(2-Cyclohexylethyl)-4<i>H</i>-1,2,4-triazol-3-yl)benzo[<i>d</i>][1,2,3]thiadiazol-6-amine (<b>13</b>)</h3><div class="NLM_p last">To a stirred solution of <b>8</b> (82 mg, 0.54 mmol) in THF (2 mL) was added cyanogen bromide solution (0.11 mL, 5.0 M in CH<sub>3</sub>CN) at rt. The resulting mixture was stirred overnight at 40 °C. After removal of the solvents, to the solution of the residue in THF (1 mL) were added 2 drops of HCl solution in <i>i</i>-PrOH (5 N) and 3-cyclohexylpropanehydrazide (184 mg, 1.08 mmol). The resulting mixture was heated by microwave irradiation to 160 °C for 30 min. After being concentrated, the residue was purified by preparative HPLC with a gradient from 50% MeOH (A) in 0.1% TFA in H<sub>2</sub>O (B) to 100% MeOH (A) to afford the title compound <b>13</b> as a white solid (57 mg, 32% in two steps): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>4</sub>-MeOH) δ 8.50 (d, <i>J</i> = 2.1 Hz, 1H), 8.44 (d, <i>J</i> = 9.1 Hz, 1H), 7.59 (dd, <i>J</i> = 9.1, 2.1 Hz, 1H), 2.87–2.79 (m, 2H), 1.86–1.71 (m, 4H), 1.71–1.63 (m, 3H), 1.37–1.14 (m, 4H), 1.06–0.92 (m, 2H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>4</sub>-MeOH) δ 157.8, 157.3, 154.8, 144.6, 144.0, 124.6, 119.9, 104.6, 38.4, 36.06, 34.0 (two carbons), 27.6, 27.3 (two carbons), 24.3; HPLC (method 1) 97%, <i>t</i><sub>R</sub> 5.76 min; MS (ESI) <i>m</i>/<i>z</i> 329 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)urea (<b>14a</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>14a</b>. The title compound <b>14a</b> was obtained as a white solid (52 mg, 47%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 9.12 (s, 1H), 8.52 (d, <i>J</i> = 1.9 Hz, 1H), 8.50 (d, <i>J</i> = 9.1 Hz, 1H), 7.59 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 6.41 (t, <i>J</i> = 5.6 Hz, 1H), 3.82 (dd, <i>J</i> = 11.3, 3.6 Hz, 2H), 3.30–3.21 (m, 2H), 3.21–3.11 (m, 2H), 1.66– 1.47 (m, 3H), 1.40 (q, <i>J</i> = 6.8 Hz, 2H), 1.23–1.07 (m, 2H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>6</sub>-DMSO) δ 154.8, 153.2, 142.3, 142.2, 123.4, 119.1, 105.1, 67.0 (two carbons), 36.7, 36.3, 32.6 (two carbons), 31.9; HPLC (method 1) 99%, <i>t</i><sub>R</sub> 3.23 min; MS (ESI) <i>m</i>/<i>z</i> 307 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(2-(piperidin-1-yl)ethyl)urea (<b>14b</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of <b>14b</b>. The title compound <b>14b</b> was obtained as a white solid (48 mg, 45%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-acetone) δ 10.04 (s, 1H), 8.65 (d, <i>J</i> = 1.9 Hz, 1H), 8.48 (d, <i>J</i> = 9.1 Hz, 1H), 8.00 (s, 1H), 7.67 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 3.92–3.83 (m, 2H), 3.77–3.68 (m, 2H), 3.50–3.42 (m, 2H), 3.20–3.06 (m, 2H), 2.03–1.93 (m, 4H), 1.92–1.83 (m, 1H), 1.66–1.52 (m, 1H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>6</sub>-acetone) δ 154.9, 143.5, 124.4, 120.1, 120.0, 106.2, 106.1, 59.6, 59.5, 54.6, 54.5, 35.3, 23.9, 22.5; HPLC (method 2) 98%, <i>t</i><sub>R</sub> 3.08 min; MS (ESI) <i>m</i>/<i>z</i> 306 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(2-morpholinoethyl)urea (<b>14c</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>14c</b>. The title compound <b>14c</b> was obtained as a white solid (53 mg, 47%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>4</sub>-MeOH) δ 8.51 (d, <i>J</i> = 1.9 Hz, 1H), 8.45 (d, <i>J</i> = 9.1 Hz, 1H), 7.56 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 4.15–4.00 (m, 2H), 3.81 (t, <i>J</i> = 11.6 Hz, 2H), 3.74–3.61 (m, 4H), 3.35 (t, <i>J</i> = 5.6 Hz, 2H), 3.20 (t, <i>J</i> = 10.4 Hz, 2H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>4</sub>-MeOH) δ 158.4, 155.5, 144.0, 142.7, 124.5, 120.9, 107.4, 65.1 (two carbons), 59.4, 53.6 (two carbons), 35.6; HPLC (method 2) 99%, <i>t</i><sub>R</sub> 2.63 min; MS (ESI) <i>m</i>/<i>z</i> 308 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(3-morpholinopropyl)urea (<b>14d</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>14d</b>. The title compound <b>14d</b> was obtained as a white solid (42 mg, 39%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>4</sub>-MeOH) δ 8.48–8.46 (m, 1H), 8.44 (d, <i>J</i> = 9.1 Hz, 1H), 7.55 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 4.14–4.02 (m, 2H), 3.79 (t, <i>J</i> = 12.0 Hz, 2H), 3.58–3.47 (m, 2H), 3.37 (t, <i>J</i> = 6.5 Hz, 2H), 3.29–3.23 (m, 2H), 3.16 (td, <i>J</i> = 12.4, 3.3 Hz, 2H), 2.07–1.97 (m, 2H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>4</sub>-MeOH) δ 158.2, 155.4, 144.0, 142.9, 124.5, 120.8, 107.1, 65.1 (two carbons), 56.0, 53.2 (two carbons), 37.6, 25.8; HPLC (method 2) 98%, <i>t</i><sub>R</sub> 2.83 min; MS (ESI) <i>m</i>/<i>z</i> 322 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(2-(pyrrolidin-1-yl)ethyl)urea (<b>14e</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>14e</b>. The title compound <b>14e</b> was obtained as a white solid (48 mg, 45%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>4</sub>-MeOH) δ 8.50 (d, <i>J</i> = 2.0 Hz, 1H), 8.44 (d, <i>J</i> = 9.1 Hz, 1H), 7.56 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 3.90–3.75 (m, 2H), 3.62 (t, <i>J</i> = 5.7 Hz, 2H), 3.39 (t, <i>J</i> = 5.7 Hz, 2H), 3.23–3.06 (m, 2H), 2.25–2.10 (m, 2H), 2.11–1.95 (m, 2H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>4</sub>-MeOH) δ 158.0, 155.4, 143.9, 142.8, 124.5, 120.8, 107.2, 56.8, 55.6 (two carbons), 37.5, 23.9 (two carbons); HPLC (method 2) 100%, <i>t</i><sub>R</sub> 2.69 min; MS (ESI) <i>m</i>/<i>z</i> 292 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(cyclohexylmethyl)urea (<b>14f</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>14f</b>. The title compound <b>14f</b> was obtained as a white solid (62 mg, 48%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 9.07 (s, 1H), 8.55–8.42 (m, 2H), 7.58 (dd, <i>J</i> = 9.0, 2.1 Hz, 1H), 6.43 (t, <i>J</i> = 5.8 Hz, 1H), 2.98 (t, <i>J</i> = 6.3 Hz, 2H), 1.73–1.64 (m, 4H), 1.64–1.54 (m, 1H), 1.47–1.35 (m, 1H), 1.26–1.07 (m, 3H), 0.97–0.85 (m, 2H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>6</sub>-DMSO) δ 154.8, 153.2, 142.3, 142.2, 123.4, 119.1, 105.1, 45.3, 37.9, 30.3 (two carbons), 26.04, 25.4 (two carbons); HPLC (method 1) 98%, <i>t</i><sub>R</sub> 5.28 min; MS (ESI) <i>m</i>/<i>z</i> 291 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-ethylurea (<b>14g</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>14g</b>. The title compound <b>14g</b> was obtained as a white solid (67 mg, 55%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 9.13 (s, 1H), 8.52 (d, <i>J</i> = 1.8 Hz, 1H), 8.50 (d, <i>J</i> = 9.1 Hz, 1H), 7.59 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 6.40 (t, <i>J</i> = 5.5 Hz, 1H), 3.22–3.09 (m, 2H), 1.08 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>6</sub>-DMSO) δ 154.7, 153.2, 142.3, 142.2, 123.4, 119.1, 105.2, 34.1, 15.3; HPLC (method 1) 100%, <i>t</i><sub>R</sub> 2.03 min; MS (ESI) <i>m</i>/<i>z</i> 223 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(3,3-dimethylbutyl)urea (<b>14h</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>14h</b>. The title compound <b>14h</b> was obtained as a white solid (54 mg, 49%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 9.13 (s, 1H), 8.52 (d, <i>J</i> = 1.7 Hz, 1H), 8.50 (d, <i>J</i> = 9.0 Hz, 1H), 7.59 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 6.33 (t, <i>J</i> = 5.5 Hz, 1H), 3.20–3.07 (m, 2H), 1.45–1.32 (m, 2H), 0.92 (s, 9H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>6</sub>-DMSO) δ 155.1, 153.7, 142.8, 142.7, 123.9, 119.6, 105.6, 43.8, 36.3, 30.0, 29.7 (three carbons); HPLC (method 1) 97%, <i>t</i><sub>R</sub> 5.01 min; MS (ESI) <i>m</i>/<i>z</i> 279 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-benzylurea (<b>14i</b>)</h3><div class="NLM_p last">1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-benzylurea (<b>14i</b>) was synthesized according to previously reported procedures.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(57)</a></div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(pyridin-2-ylmethyl)urea (<b>14j</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>14j</b>. The title compound <b>14j</b> was obtained as a white solid (57 mg, 46%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 9.83 (s, 1H), 8.74 (dd, <i>J</i> = 5.4, 0.8 Hz, 1H), 8.57–8.48 (m, 2H), 8.30 (td, <i>J</i> = 7.9, 1.6 Hz, 1H), 7.79 (d, <i>J</i> = 8.0 Hz, 1H), 7.75–7.70 (m, 1H), 7.64 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 7.42 (t, <i>J</i> = 5.5 Hz, 1H), 4.63 (d, <i>J</i> = 5.3 Hz, 2H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>6</sub>-DMSO) δ 156.5, 155.2, 153.5, 143.9, 143.0, 142.3, 141.8, 124.2, 123.8, 123.5, 119.3, 105.7, 42.5; HPLC (method 2) 99%, <i>t</i><sub>R</sub> 4.15 min; MS (ESI) <i>m</i>/<i>z</i> 286 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(pyridin-3-ylmethyl)urea (<b>14k</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>14k</b>. The title compound <b>14k</b> was obtained as a white solid (45 mg, 41%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 9.72 (s, 1H), 8.84 (d, <i>J</i> = 1.4 Hz, 1H), 8.78 (d, <i>J</i> = 4.8 Hz, 1H), 8.55 (d, <i>J</i> = 1.9 Hz, 1H), 8.52 (d, <i>J</i> = 9.1 Hz, 1H), 8.41 (d, <i>J</i> = 8.1 Hz, 1H), 7.95 (dd, <i>J</i> = 8.0, 5.6 Hz, 1H), 7.63 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 7.42 (t, <i>J</i> = 5.9 Hz, 1H), 4.52 (d, <i>J</i> = 5.8 Hz, 2H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>6</sub>-DMSO) δ 155.2, 153.4, 143.0, 142.3, 142.0, 141.9, 141.8, 139.6, 126.3, 123.5, 119.3, 105.6, 40.3; HPLC (method 2) 99%, <i>t</i><sub>R</sub> 3.67 min; MS (ESI) <i>m</i>/<i>z</i> 286 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(pyridin-4-ylmethyl)urea (<b>14l</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>14l</b>. The title compound <b>14l</b> was obtained as a white solid (51 mg, 44%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 9.87 (s, 1H), 8.83 (d, <i>J</i> = 6.6 Hz, 2H), 8.58–8.49 (m, 2H), 7.91 (d, <i>J</i> = 6.5 Hz, 2H), 7.65 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 7.56 (t, <i>J</i> = 5.9 Hz, 1H), 4.61 (d, <i>J</i> = 5.8 Hz, 2H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>6</sub>-DMSO) δ 160.0, 155.2, 153.4, 142.5 (two carbons), 142.3, 141.9, 124.4 (two carbons), 123.5, 119.3, 105.7, 42.6; HPLC (method 2) 97%, <i>t</i><sub>R</sub> 3.19 min; MS (ESI) <i>m</i>/<i>z</i> 286 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(4-(dimethylamino)benzyl)urea (<b>14m</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>14m</b>. The title compound <b>14m</b> was obtained as a white solid (31 mg, 36%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 9.39 (s, 1H), 8.54 (d, <i>J</i> = 1.9 Hz, 1H), 8.51 (d, <i>J</i> = 9.1 Hz, 1H), 7.61 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 7.33 (d, <i>J</i> = 8.6 Hz, 2H), 7.16 (d, <i>J</i> = 8.1 Hz, 2H), 7.02 (t, <i>J</i> = 5.1 Hz, 1H), 4.29 (d, <i>J</i> = 5.1 Hz, 2H), 3.02 (s, 6H); HPLC (method 2) 97%, <i>t</i><sub>R</sub> 4.74 min; MS (ESI) <i>m</i>/<i>z</i> 328 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(4-methoxybenzyl)urea (<b>14n</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>14n</b>. The title compound <b>14n</b> was obtained as a white solid (61 mg, 53%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 9.22 (s, 1H), 8.54 (d, <i>J</i> = 1.8 Hz, 1H), 8.51 (d, <i>J</i> = 9.1 Hz, 1H), 7.60 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 7.30–7.21 (m, 2H), 6.93–6.86 (m, 2H), 6.83 (t, <i>J</i> = 5.8 Hz, 1H), 4.26 (d, <i>J</i> = 5.8 Hz, 2H), 3.73 (s, 3H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>6</sub>-DMSO) δ 158.2, 154.8, 153.3, 142.3, 142.1, 131.8, 128.6 (two carbons), 123.5, 119.2, 113.7 (two carbons), 105.3, 55.1, 42.3; HPLC (method 1) 99%, <i>t</i><sub>R</sub> 3.77 min; MS (ESI) <i>m</i>/<i>z</i> 315 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(4-nitrobenzyl)urea (<b>14o</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>14o</b>. The title compound <b>14o</b> was obtained as a white solid (26 mg, 35%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 9.43 (s, 1H), 8.54 (d, <i>J</i> = 1.9 Hz, 1H), 8.52 (d, <i>J</i> = 9.1 Hz, 1H), 8.26–8.16 (m, 2H), 7.63 (dd, <i>J</i> = 9.1, 2.1 Hz, 1H), 7.61–7.54 (m, 2H), 7.08 (t, <i>J</i> = 6.0 Hz, 1H), 4.48 (d, <i>J</i> = 6.0 Hz, 2H); HPLC (method 1) 97%, <i>t</i><sub>R</sub> 3.59 min; MS (ESI) <i>m</i>/<i>z</i> 330 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(4-(methylsulfonyl)benzyl)urea (<b>14p</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>14p</b>. The title compound <b>14p</b> was obtained as a white solid (37 mg, 39%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 9.40 (s, 1H), 8.55 (d, <i>J</i> = 2.0 Hz, 1H), 8.52 (d, <i>J</i> = 9.1 Hz, 1H), 7.93–7.87 (m, 2H), 7.63 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 7.58 (d, <i>J</i> = 8.3 Hz, 2H), 7.06 (t, <i>J</i> = 6.0 Hz, 1H), 4.46 (d, <i>J</i> = 6.0 Hz, 2H), 3.19 (s, 3H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>6</sub>-DMSO) δ 154.9, 153.4, 146.4, 142.3, 141.9, 139.2, 127.8 (two carbons), 127.1 (two carbons), 123.5, 119.3, 105.6, 43.6, 42.5; HPLC (method 1) 100%, <i>t</i><sub>R</sub> 1.92 min; MS (ESI) <i>m</i>/<i>z</i> 363 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(4-cyanobenzyl)urea (<b>14q</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>14q</b>. The title compound <b>14q</b> was obtained as a white solid (34 mg, 41%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 9.40 (s, 1H), 8.54 (d, <i>J</i> = 2.0 Hz, 1H), 8.52 (d, <i>J</i> = 9.1 Hz, 1H), 7.81 (d, <i>J</i> = 8.1 Hz, 2H), 7.62 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 7.51 (d, <i>J</i> = 8.1 Hz, 2H), 7.05 (t, <i>J</i> = 6.0 Hz, 1H), 4.43 (d, <i>J</i> = 6.0 Hz, 2H); HPLC (method 1) 97%, <i>t</i><sub>R</sub> 4.86 min; MS (ESI) <i>m</i>/<i>z</i> 310 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(4-(trifluoromethyl)benzyl)urea (<b>14r</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>14r</b>. The title compound <b>14r</b> was obtained as a white solid (52 mg, 47%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 9.38 (s, 1H), 8.55 (d, <i>J</i> = 1.9 Hz, 1H), 8.52 (d, <i>J</i> = 9.0 Hz, 1H), 7.70 (d, <i>J</i> = 8.1 Hz, 2H), 7.62 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 7.54 (d, <i>J</i> = 8.0 Hz, 2H), 7.04 (t, <i>J</i> = 6.0 Hz, 1H), 4.44 (d, <i>J</i> = 6.0 Hz, 2H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>6</sub>-DMSO) δ 155.0, 153.4, 145.1 (q, <i>J</i> = 1.3 Hz), 142.3, 142.0, 127.7, 127.5 (q, <i>J</i> = 31.6 Hz), 125.2 (q, <i>J</i> = 3.8 Hz),124.4 (q, <i>J</i> = 271.9 Hz), 123.5, 119.3, 105.6, 42.5; HPLC (method 1) 98%, <i>t</i><sub>R</sub> 4.92 min; MS (ESI) <i>m</i>/<i>z</i> 353 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(4-fluorobenzyl)urea (<b>14s</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>14s</b>. The title compound <b>14s</b> was obtained as a white solid (41 mg, 48%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 9.29 (s, 1H), 8.54 (d, <i>J</i> = 1.8 Hz, 1H), 8.51 (d, <i>J</i> = 9.1 Hz, 1H), 7.61 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 7.43–7.31 (m, 2H), 7.23–7.10 (m, 2H), 6.93 (t, <i>J</i> = 6.0 Hz, 1H), 4.32 (d, <i>J</i> = 5.9 Hz, 2H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>6</sub>-DMSO) δ 161.15 (d, <i>J</i> = 242.1 Hz), 154.85, 153.33, 142.28, 142.01, 136.22 (d, <i>J</i> = 3.0 Hz), 129.15 (d, <i>J</i> = 8.2 Hz, two carbons), 123.46, 119.23, 115.01 (d, <i>J</i> = 21.2 Hz, two carbons). 105.44, 42.12; HPLC (method 1) 99%, <i>t</i><sub>R</sub> 4.20 min; MS (ESI) <i>m</i>/<i>z</i> 303 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(2-(trifluoromethyl)benzyl)urea (<b>14t</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>14t</b>. The title compound <b>14t</b> was obtained as a white solid (32 mg, 41%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 9.43 (s, 1H), 8.55 (d, <i>J</i> = 1.9 Hz, 1H), 8.52 (d, <i>J</i> = 9.1 Hz, 1H), 7.73 (d, <i>J</i> = 7.7 Hz, 1H), 7.68 (t, <i>J</i> = 7.6 Hz, 1H), 7.65–7.58 (m, 2H), 7.48 (t, <i>J</i> = 7.5 Hz, 1H), 6.99 (t, <i>J</i> = 6.0 Hz, 1H), 4.54 (d, <i>J</i> = 5.8 Hz, 2H); HPLC (method 1) 99%, <i>t</i><sub>R</sub> 4.76 min; MS (ESI) <i>m</i>/<i>z</i> 353 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(2-oxo-2-phenylethyl)urea (<b>14u</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>14u</b>. The title compound <b>14u</b> was obtained as a white solid (42 mg, 45%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>4</sub>-MeOH) δ 8.51 (d, <i>J</i> = 1.9 Hz, 1H), 8.46 (d, <i>J</i> = 9.1 Hz, 1H), 8.05 (dt, <i>J</i> = 8.5, 1.7 Hz, 2H), 7.68 – 7.63 (m, 1H), 7.55 (ddd, <i>J</i> = 9.2, 5.1, 1.9 Hz, 3H), 4.80 (s, 2H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>4</sub>-MeOH) δ 196.7, 157.5, 155.4, 144.1, 143.0, 136.3, 134.9, 130.0 (two carbons), 129.0 (two carbons), 124.6, 120.7, 107.0, 47.9; HPLC (method 1) 97%, <i>t</i><sub>R</sub> 3.68 min; MS (ESI) <i>m</i>/<i>z</i> 313 [M + H]<sup>+</sup>; HRMS calcd. for C<sub>15</sub>H<sub>12</sub>N<sub>4</sub>O<sub>2</sub>S + H: 313.0759; found: 313.0747 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(3-(4-fluorophenyl)propyl)urea (<b>14v</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>14v</b>. The title compound <b>14v</b> was obtained as a white solid (42 mg, 45%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 9.15 (s, 1H), 8.52 (d, <i>J</i> = 1.9 Hz, 1H), 8.50 (d, <i>J</i> = 9.1 Hz, 1H), 7.60 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 7.29–7.22 (m, 2H), 7.14–7.05 (m, 2H), 6.49 (t, <i>J</i> = 5.6 Hz, 1H), 3.18–3.09 (m, 2H), 2.65–2.57 (m, 2H), 1.80–1.69 (m, 2H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>6</sub>-DMSO) δ 160.6 (d, <i>J</i> = 241.0 Hz), 154.8, 153.2, 142.3, 142.2, 137.7 (d, <i>J</i> = 3.1 Hz), 130.0 (d, <i>J</i> = 7.9 Hz, two carbons), 123.4, 119.2, 114.9 (d, <i>J</i> = 20.9 Hz, two carbons), 105.2, 38.6, 31.6, 31.4; HPLC (method 1) 99%, <i>t</i><sub>R</sub> 5.02 min; MS (ESI) <i>m</i>/<i>z</i> 331 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(2-(piperidin-4-yl)ethyl)urea (<b>15a</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of Boc-protected compound <b>15a</b>, which was purified by flash column chromatography on silica gel (0–10% MeOH/DCM gradient). To a solution of Boc-protected compound <b>15a</b> (70 mg, 0.172 mmol) in DCM (5 mL) was added TFA (0.5 mL). After being stirred overnight, the resulting mixture was purified by preparative HPLC (10–100% methanol/0.1% TFA in H<sub>2</sub>O) to afford the title compound <b>15a</b> as a mono-TFA salt (39 mg, 37% in two steps): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>4</sub>-MeOH) δ 8.45 (d, <i>J</i> = 2.0 Hz, 1H), 8.43 (d, <i>J</i> = 9.2 Hz, 1H), 7.53 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 3.42–3.34 (m, 2H), 3.33–3.27 (m, 2H), 2.97 (td, <i>J</i> = 12.8, 2.6 Hz, 2H), 2.06–1.97 (m, 2H), 1.79–1.66 (m, 1H), 1.56 (q, <i>J</i> = 6.9 Hz, 2H), 1.48–1.34 (m, 2H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>4</sub>-MeOH) δ 157.6, 155.3, 144.1, 143.2, 124.5, 120.7, 106.8, 45.3 (two carbons), 37.9, 37.4, 32.5, 29.9 (two carbons); HPLC (method 2) 99%, <i>t</i><sub>R</sub> 3.13 min; MS (ESI) <i>m</i>/<i>z</i> 306 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> 1-(2-(<i>cis</i>-4-Aminocyclohexyl)ethyl)-3-(benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)urea (<b>15b</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>15a</b> was followed for synthesis of compound <b>15b</b>. The title compound <b>15b</b> was obtained as a mono-TFA salt (42 mg, 41% in two steps): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>4</sub>-MeOH) δ 8.45 (d, <i>J</i> = 1.9 Hz, 1H), 8.43 (d, <i>J</i> = 9.1 Hz, 1H), 7.53 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 3.29–3.23 (m, 3H), 1.87–1.63 (m, 7H), 1.63–1.42 (m, 4H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>4</sub>-MeOH) δ 157.6, 155.3, 144.1, 143.2, 124.5, 120.7, 106.8, 50.1, 38.7, 34.8, 32.6, 27.9 (two carbons), 27.8(two carbons); HPLC (method 2) 98%, <i>t</i><sub>R</sub> 3.56 min; MS (ESI) <i>m</i>/<i>z</i> 320 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> 1-(2-(<i>trans</i>-4-Aminocyclohexyl)ethyl)-3-(benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)urea (<b>15c</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>15a</b> was followed for synthesis of compound <b>15c</b>. The title compound <b>15c</b> was obtained as a mono-TFA salt (49 mg, 47% in two steps): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>4</sub>-MeOH) δ 8.45 (d, <i>J</i> = 1.9 Hz, 1H), 8.43 (d, <i>J</i> = 9.1 Hz, 1H), 7.52 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 3.28 (t, <i>J</i> = 7.1 Hz, 2H), 3.09–2.98 (m, 1H), 2.10–2.00 (m, 2H), 1.98–1.90 (m, 2H), 1.53–1.33 (m, 5H), 1.17–1.03 (m, 2H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>4</sub>-MeOH) δ 157.6, 155.3, 144.1, 143.2, 124.5, 120.7, 106.7, 51.6, 38.5, 37.8, 35.1, 31.8 (two carbons), 31.7 (two carbons); HPLC (method 2) 98%, <i>t</i><sub>R</sub> 3.43 min; MS (ESI) <i>m</i>/<i>z</i> 320 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> 1-(3-(<i>trans</i>-4-Aminocyclohexyl)propyl)-3-(benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)urea (<b>15d</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>15a</b> was followed for synthesis of compound <b>15d</b>. The title compound <b>15d</b> was obtained as a mono-TFA salt (29 mg, 35% in two steps): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>4</sub>-MeOH) δ 8.48–8.41 (m, 2H), 7.52 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 3.22 (t, <i>J</i> = 7.1 Hz, 2H), 3.02 (tt, <i>J</i> = 11.9, 3.9 Hz, 1H), 2.08–1.97 (m, 2H), 1.97–1.86 (m, 2H), 1.62–1.53 (m, 2H), 1.44–1.25 (m, 5H), 1.13–1.00 (m, 2H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>4</sub>-MeOH) δ 157.6, 155.3, 144.1, 143.2, 124.5, 120.7, 106.8, 51.7, 40.9, 37.3, 34.7, 32.0 (two carbons), 31.8 (two carbons), 28.5; HPLC (method 2) 98%, <i>t</i><sub>R</sub> 3.76 min; MS (ESI) <i>m</i>/<i>z</i> 334 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(3-(piperidin-4-yl)propyl)urea (<b>15e</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>15a</b> was followed for synthesis of compound <b>15e</b>. The title compound <b>15e</b> was obtained as a mono-TFA salt (49 mg, 45% in two steps): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>4</sub>-MeOH) δ 8.43 (d, <i>J</i> = 2.0 Hz, 1H), 8.40 (d, <i>J</i> = 9.1 Hz, 1H), 7.52 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 3.41–3.32 (m, 2H), 3.22 (t, <i>J</i> = 7.0 Hz, 2H), 2.95 (td, <i>J</i> = 12.8, 2.5 Hz, 2H), 1.97–1.89 (m, 2H), 1.66–1.53 (m, 3H), 1.43–1.29 (m, 4H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>4</sub>-MeOH) δ 157.6, 155.2, 144.0, 143.2, 124.4, 120.7, 106.7, 45.3 (two carbons), 40.7, 34.5, 34.1, 30.0 (two carbons), 27.9; HPLC (method 2) 99%, <i>t</i><sub>R</sub> 3.32 min; MS (ESI) <i>m</i>/<i>z</i> 320 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> 1-(2-Amino-2-phenylethyl)-3-(benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)urea (<b>16</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>15a</b> was followed for synthesis of compound <b>16</b>. The title compound <b>16</b> was obtained as a mono-TFA salt (48 mg, 42% in two steps): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>4</sub>-MeOH) δ 8.51–8.47 (m, 1H), 8.45–8.39 (m, 1H), 7.54 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 7.52–7.41 (m, 5H), 4.53 (dd, <i>J</i> = 7.9, 5.5 Hz, 1H), 3.79 (dd, <i>J</i> = 14.5, 8.0 Hz, 1H), 3.67 (dd, <i>J</i> = 14.5, 5.5 Hz, 1H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>4</sub>-MeOH) δ 157.8, 155.4, 143.9, 142.8, 136.3, 130.5, 130.4 (two carbons), 128.3 (two carbons), 124.5, 120.8, 107.1, 56.8, 44.7; HPLC (method 2) 99%, <i>t</i><sub>R</sub> 3.49 min; MS (ESI) <i>m</i>/<i>z</i> 314 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(2-hydroxy-2-phenylethyl)urea (<b>17</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>17</b>. The title compound <b>17</b> was obtained as a white solid (52 mg, 50%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>4</sub>-MeOH) δ 8.44–8.41 (m, 1H), 8.41–8.36 (m, 1H), 7.45 (dd, <i>J</i> = 9.0, 2.0 Hz, 1H), 7.43–7.39 (m, 2H), 7.37–7.31 (m, 2H), 7.29–7.23 (m, 1H), 4.80 (dd, <i>J</i> = 7.9, 4.5 Hz, 1H), 3.56 (dd, <i>J</i> = 13.7, 4.5 Hz, 1H), 3.36 (dd, <i>J</i> = 13.7, 7.9 Hz, 1H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>4</sub>-MeOH) δ 157.5, 155.2, 144.0, 143.9, 143.0, 129.4 (two carbons), 128.6, 127.1 (two carbons), 124.4, 120.6, 106.8, 73.9, 48.3; HPLC (method 1) 99%, <i>t</i><sub>R</sub> 3.27 min; MS (ESI) <i>m</i>/<i>z</i> 315 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(2,2-difluoro-2-phenylethyl)urea (<b>18</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>18</b>. The title compound <b>18</b> was obtained as a white solid (42 mg, 38%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 9.26 (s, 1H), 8.55–8.48 (m, 2H), 7.63–7.55 (m, 3H), 7.53–7.47 (m, 3H), 6.81 (t, <i>J</i> = 6.3 Hz, 1H), 3.95 (td, <i>J</i> = 15.0, 6.3 Hz, 2H); HPLC (method 1) 98%, <i>t</i><sub>R</sub> 4.15 min; MS (ESI) <i>m</i>/<i>z</i> 335 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(2-(4-fluorophenyl)-2-hydroxyethyl)urea (<b>19</b>)</h3><div class="NLM_p last">To a stirred solution of 4-fluorobenzaldehyde (1.24 g, 10.0 mmol) and nitromethane (2.14 mL, 40.0 mmol) in ethanol (40 mL) was added 4.2 mL of 10% aq NaOH at 0 °C. The resulting mixture was stirred for 2 h at 0 °C and quenched with acetic acid (0.61 mL, 10.5 mmol). After addition of 30 mL of saturated aq NaHCO<sub>3</sub> and extraction with ethyl acetate (100 mL × 3), the combined organic phase was dried with sodium sulfate, filtered, and concentrated to afford the desired crude product 1-(4-fluorophenyl)-2-nitroethanol. A stirred suspension of the intermediate and palladium carbon in methanol (100 mL) was treated with hydrogen at 1 atm overnight at rt. The product 2-amino-1-(4-fluorophenyl)ethanol was obtained after filtration and concentration. The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>19</b> from 2-amino-1-(4-fluorophenyl)ethanol. The title compound <b>19</b> was obtained as a white solid (47 mg, 49%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>4</sub>-MeOH) δ 8.46 (dd, <i>J</i> = 2.0, 0.5 Hz, 1H), 8.43 (dd, <i>J</i> = 9.0, 0.5 Hz, 1H), 7.49 (dd, <i>J</i> = 9.0, 2.0 Hz, 1H), 7.46–7.40 (m, 2H), 7.12–7.03 (m, 2H), 4.80 (dd, <i>J</i> = 7.7, 4.5 Hz, 1H), 3.53 (dd, <i>J</i> = 13.7, 4.5 Hz, 1H), 3.35 (dd, <i>J</i> = 13.8, 7.8 Hz, 1H); HPLC (method 1) 98%, <i>t</i><sub>R</sub> 3.59 min; MS (ESI) <i>m</i>/<i>z</i> 333 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(2-(2-fluorophenyl)-2-hydroxyethyl)urea (<b>20</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>19</b> was followed for synthesis of compound <b>20</b>. The title compound <b>20</b> was obtained as a white solid (57 mg, 54%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>4</sub>-MeOH) δ 8.45–8.37 (m, 2H), 7.58 (td, <i>J</i> = 7.5, 1.6 Hz, 1H), 7.46 (dd, <i>J</i> = 9.0, 2.0 Hz, 1H), 7.29 (tdd, <i>J</i> = 7.3, 5.3, 1.8 Hz, 1H), 7.19 (td, <i>J</i> = 7.5, 0.8 Hz, 1H), 7.06 (ddd, <i>J</i> = 10.5, 8.2, 0.9 Hz, 1H), 5.13 (dd, <i>J</i> = 7.3, 4.5 Hz, 1H), 3.60 (dd, <i>J</i> = 13.7, 4.5 Hz, 1H), 3.44 (dd, <i>J</i> = 13.7, 7.4 Hz, 1H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>4</sub>-MeOH) δ 159.9 (d, <i>J</i> = 244.4 Hz), 156.14, 153.88, 142.65, 141.60, 129.4 (d, <i>J</i> = 13.7 Hz), 128.9 (d, <i>J</i> = 8.3 Hz), 127.5 (d, <i>J</i> = 4.4 Hz), 124.0 (d, <i>J</i> = 3.5 Hz), 123.1, 119.2, 114.7 (d, <i>J</i> = 22.0 Hz), 105.4, 66.4, 45.6; HPLC (method 1) 100%, <i>t</i><sub>R</sub> 3.26 min; MS (ESI) <i>m</i>/<i>z</i> 333 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(2-cyclohexyl-2-hydroxyethyl)urea (<b>21</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>19</b> was followed for synthesis of compound <b>21</b>. The title compound <b>21</b> was obtained as a white solid (57 mg, 48%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>4</sub>-MeOH) δ 8.46 (dd, <i>J</i> = 2.0, 0.5 Hz, 1H), 8.42 (dd, <i>J</i> = 9.1, 0.5 Hz, 1H), 7.49 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 3.51 (dd, <i>J</i> = 13.6, 3.4 Hz, 1H), 3.41 (ddd, <i>J</i> = 8.2, 6.4, 3.4 Hz, 1H), 3.09 (dd, <i>J</i> = 13.6, 8.2 Hz, 1H), 1.97–1.87 (m, 1H), 1.84–1.63 (m, 4H), 1.47–1.35 (m, 1H), 1.35–0.99 (m, 5H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>4</sub>-MeOH) δ 157.7, 155.3, 144.1, 143.1, 124.5, 120.6, 106.7, 76.0, 44.5, 43.2, 30.3, 29.4, 27.6, 27.4, 27.2; HPLC (method 1) 100%, <i>t</i><sub>R</sub> 4.51 min; MS (ESI) <i>m</i>/<i>z</i> 321 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_43" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(2-cyclohexyl-2-oxoethyl)urea (<b>22</b>)</h3><div class="NLM_p last">To a solution of compound <b>21</b> (46 mg, 0.143 mmol) in 2.2 mL of DCM/DMSO (2:1) were added triethylamine (0.10 mL, 0.72 mmol) and Py·SO<sub>3</sub> (92 mg, 0.574 mmol) at rt. The resulting mixture was stirred for 1 h at rt, quenched with 1.0 mL of water, concentrated, and purified by preparative HPLC (50–100% methanol/0.1% TFA in H<sub>2</sub>O) to afford the title compound <b>23</b> as a white solid (35 mg, 78%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 9.50 (s, 1H), 8.54–8.49 (m, 2H), 7.59 (dd, <i>J</i> = 9.2, 2.0 Hz, 1H), 6.60 (t, <i>J</i> = 5.2 Hz, 1H), 4.12 (d, <i>J</i> = 5.2 Hz, 2H), 2.49–2.42 (m, 1H), 1.86–1.55 (m, 5H), 1.33–1.09 (m, 5H); HPLC (method 1) 98%, <i>t</i><sub>R</sub> 4.38 min; MS (ESI) <i>m</i>/<i>z</i> 319 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_44" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(2-(4-fluorophenyl)-2-oxoethyl)urea (<b>23</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>22</b> was followed for synthesis of compound <b>23</b>. The title compound <b>23</b> was obtained as a white solid (36 mg, 76%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 9.60 (s, 1H), 8.56–8.51 (m, 2H), 8.17–8.06 (m, 2H), 7.63 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 7.44–7.34 (m, 2H), 6.78 (t, <i>J</i> = 5.2 Hz, 1H), 4.72 (d, <i>J</i> = 5.2 Hz, 2H); <sup>13</sup>C NMR (100 MHz, <i>d</i><sub>6</sub>-DMSO) δ 194.5, 165.2 (d, <i>J</i> = 252.2 Hz), 154.8, 153.4, 142.3, 141.9, 131.6 (d, <i>J</i> = 2.9 Hz), 130.9 (d, <i>J</i> = 9.6 Hz, two carbons), 123.6, 119.1, 115.9 (d, <i>J</i> = 21.9 Hz, two carbons), 105.4, 46.8; HPLC (method 1) 98%, <i>t</i><sub>R</sub> 3.82 min; MS (ESI) <i>m</i>/<i>z</i> 331 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_45" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> 1-(Benzo[<i>d</i>][1,2,3]thiadiazol-6-yl)-3-(2-oxo-2-(piperidin-1-yl)ethyl)urea (<b>24</b>)</h3><div class="NLM_p last">The procedure used for preparation of compound <b>4</b> was followed for synthesis of compound <b>24</b>. The title compound <b>24</b> was obtained as a white solid (57 mg, 51%): <sup>1</sup>H NMR (400 MHz, <i>d</i><sub><i>6</i></sub>-DMSO) δ 9.60 (s, 1H), 8.57–8.46 (m, 2H), 7.60 (dd, <i>J</i> = 9.2, 2.0 Hz, 1H), 6.60 (t, <i>J</i> = 4.5 Hz, 1H), 4.01 (d, <i>J</i> = 4.5 Hz, 2H), 3.53–3.45 (m, 2H), 3.38–3.31 (m, 2H), 1.63–1.48 (m, 4H), 1.48–1.39 (m, 2H). <sup>13</sup>C NMR (100 MHz, <i>d</i><sub><i>6</i></sub>-DMSO) δ 166.6, 154.6, 153.3, 142.3, 142.0, 123.6, 119.0, 105.2, 44.8, 42.3, 41.2, 25.8, 25.2, 23.9. HPLC (method 1) 98%, <i>t</i><sub>R</sub> 3.26 min; MS (ESI) <i>m</i>/<i>z</i> 320 [M + H]<sup>+</sup>; HRMS calcd. for C<sub>14</sub>H<sub>17</sub>N<sub>5</sub>O<sub>2</sub>S + H: 320.1181; found: 320.1181 [M+H]<sup>+</sup>.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> Methyltransferase Activity Assays</h3><div class="NLM_p">Methyltransferase activity assays were performed by monitoring the incorporation of tritium-labeled methyl group to biotinylated peptide substrates using scintillation proximity assay (SPA) for PRMT3, SETD7, G9a, GLP, SETDB1, SETD8, SUV420H1, SUV420H2, SUV39H2, PRC2 trimeric complex (EZH2:EED:SUZ12), MLL1 tetrameric complex (MLL:WDR5:RbBP5:ASH2L), PRMT5–MEP50 complex, and SMYD2. Assay components for all assays are summarized in Table S1 (<a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>). The reaction buffer for SMYD2 and SMYD3 was 50 mM Tris, pH 9.0, 5 mM DTT, and 0.01% TritonX-100, that for G9a, GLP, and SUV39H2 was 25 mM potassium phosphate, pH 8.0, 1 mM EDTA, 2 mM MgCl<sub>2</sub>, and 0.01% Triton X-100, and that for the other HMTs was 20 mM Tris, pH 8.0, 5 mM DTT, and 0.01% TritonX-100. To stop the enzymatic reactions, 10 μL of 7.5 M guanidine hydrochloride was added, followed by 180 μL of buffer, and the resulting reaction mixture was mixed and transferred to a 96-well FlashPlate (catalog no. SMP103, Perkin-Elmer, <a href="http://www.perkinelmer.com" class="extLink">www.perkinelmer.com</a>). After mixing, the reaction mixtures were incubated, and the counts per minute (cpm) were measured using a Topcount plate reader (Perkin-Elmer). The cpm values in the absence of compound for each data set were defined as 100% activity. In the absence of the enzyme, the cpm values in each data set were defined as background (0%). IC<sub>50</sub> values were determined using compound concentrations ranging from 100 nM to 100 μM. The IC<sub>50</sub> values were determined using SigmaPlot software.</div><div class="NLM_p">For DNMT1, the assay was performed as described above using hemimethylated dsDNA as a substrate. The dsDNA substrate was prepared by annealing two complementary strands (biotinylated forward strand, B-GAGCCCGTAAGCCCGTTCAGGTCG; reverse strand, CGACCTGAACGGGCTTACGGGCTC) synthesized by Eurofins MWG Operon. The reaction buffer was 20 mM Tris–HCl, pH 8.0, 5 mM DTT, and 0.01% Triton X-100.</div><div class="NLM_p last">Methyltransferase activity assays for DOT1L and SMYD3 were performed using filter plates (catalog no. MSFBN6B10, Millipore, <a href="http://www.millipore.com" class="extLink">www.millipore.com</a>). Reaction mixtures in 20 mM Tris–HCl, pH 8.0, 5 mM DTT, 2 mM MgCl<sub>2</sub>, and 0.01% Triton X-100 were incubated at room temperature for 1 h, 100 μL 10% TCA was added, and the resulting reaction mixture was mixed and transferred to a filter plate. The plates were centrifuged at 2000 rpm for 2 min followed by two additional 10% TCA washes and one ethanol wash (180 μL) followed by centrifugation. The plates were dried, 100 μL of MicroO was added, and the plates were centrifuged. A 70 μL volume of MicroO was added, and the cpm values were measured using a Topcount plate reader.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> Cocrystallization Protocols and Structure Refinement</h3><div class="NLM_p">PRMT3 was incubated at 1.1 mg/mL overnight with compound <b>14u</b> at a 1:30 molar ratio (PRMT3:compound <b>14u</b>). Following incubation, protein was concentrated to 3 mg/mL and crystallized using the sitting drop diffusion method at 20 °C by mixing 1 μL of the protein solution with 1 μL of the reservoir solution containing 25% PEG 3350, 0.2 M LiSO<sub>4</sub>, and 0.1 M Hepes, pH 7.5. Prior to freezing, the crystals were soaked for 10 min in the same buffer with 10% glycerol.</div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> Data Collection and Processing</h4><div class="NLM_p last">The native data set was collected on APS beamline 19-ID at 100 K. Program HKL2000 was used for data processing and scaling.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(60)</a></div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> Structure Determination and Refinement</h4><div class="NLM_p last">The structure of PRMT3 in complex with compound <b>14u</b> was determined by molecular replacement using MOLREP.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(61)</a> PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2FYT">2FYT</a> was used as a search template. The graphic program Coot<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(62)</a> was used for manual model refinement and visualization. Refmac5<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(63)</a> was used to refine the model. MolProbity<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(64)</a> was used to validate the refined structure; 98.0% of the residues are the favored regions of the Ramachandran plot, and none of them are in the disallowed regions. The structure has been deposited in the RCSB with PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HSG">4HSG</a>.</div></div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> Modeling</h3><div class="NLM_p last">Monte Carlo energy minimization simulations were run in triplicate with ICM version 3.7-2c (Molsoft, San Diego) using the “ssearch” command. Briefly, the conformational space accessible to K392 was sampled in the internal coordinate space, with global conformational sampling steps followed by local energy minimization and calculation of the complete energy potential, including surface and advanced electrostatics terms.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(65)</a> Compound <b>14u</b> and residues surrounding K392 were kept static.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i71"><a href="/doi/suppl/10.1021/jm3018332">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00342" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00342" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Methyltransferase assay components and conditions, data collection and refinement statistics of the compound <b>14u</b>–PRMT3 cocrystal structure, and <sup>1</sup>H and <sup>13</sup>C NMR spectra of compounds <b>14u</b> and <b>24</b>. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm3018332/suppl_file/jm3018332_si_001.pdf">jm3018332_si_001.pdf (465.11 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm3018332" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23135" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23135" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masoud Vedadi</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#6a00030b040003042a1f0409440f0e1f4a07441c0f0e0b0e032a1f1e051805041e0544090b"><span class="__cf_email__" data-cfemail="3953505857535057794c575a175c5d4c">[email protected]</span> <span class="__cf_email__" data-cfemail="204d0e5645444144496055544f524f4e544f0e4341">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Jin</span> - <span class="hlFld-Affiliation affiliation">Center for
Integrative Chemical
Biology and Drug Discovery, Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University
of North Carolina at Chapel Hill, Chapel Hill, North
Carolina 27599, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a7cdcec6c9cdcec9e7d2c9c489c2c3d287ca89d1c2c3c6c3cee7d2d3c8d5c8c9d3c889c4c6"><span class="__cf_email__" data-cfemail="513b38303f3b383f11243f327f343524">[email protected]</span> <span class="__cf_email__" data-cfemail="a2cf8cd4c7c6c3c6cbe2d7d6cdd0cdccd6cd8cc1c3">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Feng Liu</span> - <span class="hlFld-Affiliation affiliation">Center for
Integrative Chemical
Biology and Drug Discovery, Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University
of North Carolina at Chapel Hill, Chapel Hill, North
Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fengling Li</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anqi Ma</span> - <span class="hlFld-Affiliation affiliation">Center for
Integrative Chemical
Biology and Drug Discovery, Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University
of North Carolina at Chapel Hill, Chapel Hill, North
Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elena Dobrovetsky</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Aiping Dong</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cen Gao</span> - <span class="hlFld-Affiliation affiliation">Center for
Integrative Chemical
Biology and Drug Discovery, Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University
of North Carolina at Chapel Hill, Chapel Hill, North
Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ilia Korboukh</span> - <span class="hlFld-Affiliation affiliation">Center for
Integrative Chemical
Biology and Drug Discovery, Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University
of North Carolina at Chapel Hill, Chapel Hill, North
Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jing Liu</span> - <span class="hlFld-Affiliation affiliation">Center for
Integrative Chemical
Biology and Drug Discovery, Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University
of North Carolina at Chapel Hill, Chapel Hill, North
Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Smil</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter J. Brown</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephen V. Frye</span> - <span class="hlFld-Affiliation affiliation">Center for
Integrative Chemical
Biology and Drug Discovery, Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University
of North Carolina at Chapel Hill, Chapel Hill, North
Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheryl H. Arrowsmith</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthieu Schapira</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d92e4371-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i73">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18242" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18242" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank the University Cancer Research Fund (UCRF) and the Carolina Partnership from the University of North Carolina at Chapel Hill for financial support. The Structural Genomics Consortium is a registered charity (number 1097737) that receives funds from Canadian Institutes of Health Research, Eli Lilly Canada, Genome Canada, GlaxoSmithKline, the Ontario Ministry of Economic Development and Innovation, the Novartis Research Foundation, Pfizer, AbbVie, Takeda, and the Wellcome Trust. C.H.A. holds a Canada Research Chair in Structural Genomics.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i74" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i74"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i75" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i75"> Abbreviations Used</h2><tr><td class="NLM_term">PRMTs</td><td class="NLM_def"><p class="first last">protein arginine methyltransferases</p></td></tr><tr><td class="NLM_term">PRMT3</td><td class="NLM_def"><p class="first last">protein arginine methyltransferase 3</p></td></tr><tr><td class="NLM_term">rpS2</td><td class="NLM_def"><p class="first last">40S ribosomal protein S2</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">HMTs</td><td class="NLM_def"><p class="first last">histone methyltransferases</p></td></tr><tr><td class="NLM_term">PMTs</td><td class="NLM_def"><p class="first last">protein methyltransferases</p></td></tr><tr><td class="NLM_term">PKMTs</td><td class="NLM_def"><p class="first last">protein lysine methyltransferases</p></td></tr><tr><td class="NLM_term">GAR</td><td class="NLM_def"><p class="first last">glycine- and arginine-rich</p></td></tr><tr><td class="NLM_term">PGM</td><td class="NLM_def"><p class="first last">proline-, glycine-, methionine-, and arginine-rich</p></td></tr><tr><td class="NLM_term">PABPN1</td><td class="NLM_def"><p class="first last">nuclear poly(A)-binding protein</p></td></tr><tr><td class="NLM_term">MOA</td><td class="NLM_def"><p class="first last">mechanism of action</p></td></tr><tr><td class="NLM_term">LHS</td><td class="NLM_def"><p class="first last">left-hand side</p></td></tr><tr><td class="NLM_term">RHS</td><td class="NLM_def"><p class="first last">right-hand side</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosylmethionine</p></td></tr><tr><td class="NLM_term">DNMT1</td><td class="NLM_def"><p class="first last">DNA methyltransferase 1</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i76">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78382" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78382" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 65 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Epigenetic protein families: a new frontier for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">384</span><span class="NLM_x">–</span> <span class="NLM_lpage">400</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1038%2Fnrd3674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=22498752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=384-400&author=C.+H.+Arrowsmithauthor=C.+Bountraauthor=P.+V.+Fishauthor=K.+Leeauthor=M.+Schapira&title=Epigenetic+protein+families%3A+a+new+frontier+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic protein families: a new frontier for drug discovery</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Bountra, Chas; Fish, Paul V.; Lee, Kevin; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">384-400</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetic regulation of gene expression is a dynamic and reversible process that establishes normal cellular phenotypes but also contributes to human diseases.  At the mol. level, epigenetic regulation involves hierarchical covalent modification of DNA and the proteins that package DNA, such as histones.  Here, we review the key protein families that mediate epigenetic signalling through the acetylation and methylation of histones, including histone deacetylases, protein methyltransferases, lysine demethylases, bromodomain-contg. proteins and proteins that bind to methylated histones.  These protein families are emerging as druggable classes of enzymes and druggable classes of protein-protein interaction domains.  In this article, we discuss the known links with disease, basic mol. mechanisms of action and recent progress in the pharmacol. modulation of each class of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcD99eCigNtLVg90H21EOLACvtfcHk0lhxNHmwsiKlDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D&md5=eb82034466a43107aa74a18ccf6d29f5</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd3674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3674%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DEpigenetic%2520protein%2520families%253A%2520a%2520new%2520frontier%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D384%26epage%3D400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solomon, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span> </span><span class="NLM_article-title">Protein methyltransferases as a target class for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">724</span><span class="NLM_x">–</span> <span class="NLM_lpage">732</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1038%2Fnrd2974" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=19721445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVKmsrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=724-732&author=R.+A.+Copelandauthor=M.+E.+Solomonauthor=V.+M.+Richon&title=Protein+methyltransferases+as+a+target+class+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Protein methyltransferases as a target class for drug discovery</span></div><div class="casAuthors">Copeland, Robert A.; Solomon, Michael E.; Richon, Victoria M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">724-732</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The protein methyltransferases (PMTs) - which methylate protein lysine and arginine residues and have crucial roles in gene transcription - are emerging as an important group of enzymes that play key parts in normal physiol. and human diseases.  The collection of human PMTs is a large and diverse group of enzymes that have a common mechanism of catalysis.  Here, we review the biol., biochem. and structural data that together present PMTs as a novel, chem. tractable target class for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpySXIqKk9Lw7Vg90H21EOLACvtfcHk0lhxNHmwsiKlDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVKmsrvI&md5=0626e14396442b47c7da2aaa6a0ac6e3</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd2974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2974%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DSolomon%26aufirst%3DM.%2BE.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26atitle%3DProtein%2520methyltransferases%2520as%2520a%2520target%2520class%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26spage%3D724%26epage%3D732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richard, S.</span><span> </span><span class="NLM_article-title">Arginine methylation an emerging regulator of protein function</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">263</span><span class="NLM_x">–</span> <span class="NLM_lpage">272</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1016%2Fj.molcel.2005.04.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=15866169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkt1Cgtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2005&pages=263-272&author=M.+T.+Bedfordauthor=S.+Richard&title=Arginine+methylation+an+emerging+regulator+of+protein+function"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Arginine methylation: An emerging regulator of protein function</span></div><div class="casAuthors">Bedford, Mark T.; Richard, Stephane</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">263-272</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Arginine methylation is now coming out of the shadows of protein phosphorylation and entering the mainstream, largely due to the identification of the family of enzymes that lay down this modification.  In addn., modification-specific antibodies and proteomic approaches have facilitated the identification of an array of substrates for protein arginine methyltransferases.  This review describes recent insights into the mol. processes regulated by arginine methylation in normal and diseased cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDYRpc_643mbVg90H21EOLACvtfcHk0lgsxH2deGO_iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXkt1Cgtb4%253D&md5=63525cb1e96138be5563fbd897c2bde1</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2005.04.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2005.04.003%26sid%3Dliteratum%253Aachs%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DRichard%26aufirst%3DS.%26atitle%3DArginine%2520methylation%2520an%2520emerging%2520regulator%2520of%2520protein%2520function%26jtitle%3DMol.%2520Cell%26date%3D2005%26volume%3D18%26spage%3D263%26epage%3D272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Di Lorenzo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">Histone arginine methylation</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">585</span><span class="NLM_x">, </span> <span class="NLM_fpage">2024</span><span class="NLM_x">–</span> <span class="NLM_lpage">2031</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=585&publication_year=2011&pages=2024-2031&author=A.+Di+Lorenzoauthor=M.+T.+Bedford&title=Histone+arginine+methylation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BLorenzo%26aufirst%3DA.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DHistone%2520arginine%2520methylation%26jtitle%3DFEBS%2520Lett.%26date%3D2011%26volume%3D585%26spage%3D2024%26epage%3D2031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Yost, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Targets in epigenetics: inhibiting the methyl writers of the histone code</span> <span class="citation_source-journal">Curr. Chem. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">72</span><span class="NLM_x">–</span> <span class="NLM_lpage">84</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.2174%2F1875397301005010072" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=21966347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1ent7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2011&pages=72-84&author=J.+M.+Yostauthor=I.+Korboukhauthor=F.+Liuauthor=C.+Gaoauthor=J.+Jin&title=Targets+in+epigenetics%3A+inhibiting+the+methyl+writers+of+the+histone+code"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Targets in epigenetics: inhibiting the methyl writers of the histone code</span></div><div class="casAuthors">Yost, Julianne M.; Korboukh, Ilia; Liu, Feng; Gao, Cen; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Current Chemical Genomics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">72-84</span>CODEN:
                <span class="NLM_cas:coden">CCGUAQ</span>;
        ISSN:<span class="NLM_cas:issn">1875-3973</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Growing evidence suggests that protein lysine methyltransferases (PKMTs) and protein arginine methyltransferases (PRMTs) are assocd. with the development of various human diseases, including cancer, inflammation, and psychiatric disorders.  Given the significant role of these proteins in human disease, efforts to discover selective small-mol. inhibitors of these enzymes are quickly gaining momentum.  In this review, we focus on the recent progress in the discovery of selective PKMT and PRMT inhibitors.  A future perspective on developing methyltransferase inhibitors is also offered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHi_nKenm0brVg90H21EOLACvtfcHk0lgsxH2deGO_iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1ent7nP&md5=996a0411f27dcb4db88e5a4885439d94</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.2174%2F1875397301005010072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1875397301005010072%26sid%3Dliteratum%253Aachs%26aulast%3DYost%26aufirst%3DJ.%2BM.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DTargets%2520in%2520epigenetics%253A%2520inhibiting%2520the%2520methyl%2520writers%2520of%2520the%2520histone%2520code%26jtitle%3DCurr.%2520Chem.%2520Genomics%26date%3D2011%26volume%3D5%26spage%3D72%26epage%3D84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Zurita-Lopez, C. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandberg, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, S. G.</span><span> </span><span class="NLM_article-title">Human protein arginine methyltransferase 7 (PRMT7) is a type III enzyme forming omega-NG-monomethylated arginine residues</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">287</span><span class="NLM_x">, </span> <span class="NLM_fpage">7859</span><span class="NLM_x">–</span> <span class="NLM_lpage">7870</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=7859-7870&author=C.+I.+Zurita-Lopezauthor=T.+Sandbergauthor=R.+Kellyauthor=S.+G.+Clarke&title=Human+protein+arginine+methyltransferase+7+%28PRMT7%29+is+a+type+III+enzyme+forming+omega-NG-monomethylated+arginine+residues"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZurita-Lopez%26aufirst%3DC.%2BI.%26aulast%3DSandberg%26aufirst%3DT.%26aulast%3DKelly%26aufirst%3DR.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DHuman%2520protein%2520arginine%2520methyltransferase%25207%2520%2528PRMT7%2529%2520is%2520a%2520type%2520III%2520enzyme%2520forming%2520omega-NG-monomethylated%2520arginine%2520residues%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D7859%26epage%3D7870" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Jansson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durant, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheahan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edelmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N. B.</span><span> </span><span class="NLM_article-title">Arginine methylation regulates the p53 response</span> <span class="citation_source-journal">Nat. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1431</span><span class="NLM_x">–</span> <span class="NLM_lpage">1439</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1038%2Fncb1802" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=19011621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVentbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2008&pages=1431-1439&author=M.+Janssonauthor=S.+T.+Durantauthor=E.+C.+Choauthor=S.+Sheahanauthor=M.+Edelmannauthor=B.+Kesslerauthor=N.+B.+La+Thangue&title=Arginine+methylation+regulates+the+p53+response"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Arginine methylation regulates the p53 response</span></div><div class="casAuthors">Jansson, Martin; Durant, Stephen T.; Cho, Er-Chieh; Sheahan, Sharon; Edelmann, Mariola; Kessler, Benedikt; La Thangue, Nicholas B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1431-1439</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Activation of the p53 tumor suppressor protein in response to DNA damage leads to apoptosis or cell-cycle arrest.  Enzymic modifications are widely believed to affect and regulate p53 activity.  We describe here a level of post-translational control that has an important functional consequence on the p53 response.  We show that the protein arginine methyltransferase (PRMT) 5, as a co-factor in a DNA damage responsive Strap (stress-responsive activator of p300)-contg. co-activator complex that interacts with p53, is responsible for methylating p53.  Arginine methylation is regulated during the p53 response and affects the target gene specificity of p53.  Furthermore, PRMT5 depletion triggers p53-dependent apoptosis.  Thus, methylation on arginine residues is an underlying mechanism of control during the p53 response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa6VSI-iJwVLVg90H21EOLACvtfcHk0lh4yhaP27iMoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVentbfN&md5=4e01b57e90fd6ae7622e6cea3be498ce</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fncb1802&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb1802%26sid%3Dliteratum%253Aachs%26aulast%3DJansson%26aufirst%3DM.%26aulast%3DDurant%26aufirst%3DS.%2BT.%26aulast%3DCho%26aufirst%3DE.%2BC.%26aulast%3DSheahan%26aufirst%3DS.%26aulast%3DEdelmann%26aufirst%3DM.%26aulast%3DKessler%26aufirst%3DB.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26atitle%3DArginine%2520methylation%2520regulates%2520the%2520p53%2520response%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2008%26volume%3D10%26spage%3D1431%26epage%3D1439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">Arginine methylation at a glance</span> <span class="citation_source-journal">J. Cell Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">120</span><span class="NLM_x">, </span> <span class="NLM_fpage">4243</span><span class="NLM_x">–</span> <span class="NLM_lpage">4246</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1242%2Fjcs.019885" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=18057026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD1cXns1GktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2007&pages=4243-4246&author=M.+T.+Bedford&title=Arginine+methylation+at+a+glance"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Arginine methylation at a glance</span></div><div class="casAuthors">Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">4243-4246</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">A review on the enzymes involved in methylation of arginine is presented.  Many different isoenzymes of the PRMT (protein arginine N Me transferase) from PRMT1 to PRMT9 are discussed.  Some of the substrates of the PRMT enzymes are also discussed along with a brief discussion of the role of protein protein interactions in regulation of arginine methylation is also given.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVWHJj0lIjM7Vg90H21EOLACvtfcHk0lh4yhaP27iMoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXns1GktA%253D%253D&md5=d6adb4f9193c22342c1d30f16a468590</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1242%2Fjcs.019885&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.019885%26sid%3Dliteratum%253Aachs%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DArginine%2520methylation%2520at%2520a%2520glance%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2007%26volume%3D120%26spage%3D4243%26epage%3D4246" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Pahlich, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bschir, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiavi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belyanskaya, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gehring, H.</span><span> </span><span class="NLM_article-title">Different methylation characteristics of protein arginine methyltransferase 1 and 3 toward the Ewing sarcoma protein and a peptide</span> <span class="citation_source-journal">Proteins: Struct., Funct., Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">164</span><span class="NLM_x">–</span> <span class="NLM_lpage">175</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=164-175&author=S.+Pahlichauthor=K.+Bschirauthor=C.+Chiaviauthor=L.+Belyanskayaauthor=H.+Gehring&title=Different+methylation+characteristics+of+protein+arginine+methyltransferase+1+and+3+toward+the+Ewing+sarcoma+protein+and+a+peptide"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPahlich%26aufirst%3DS.%26aulast%3DBschir%26aufirst%3DK.%26aulast%3DChiavi%26aufirst%3DC.%26aulast%3DBelyanskaya%26aufirst%3DL.%26aulast%3DGehring%26aufirst%3DH.%26atitle%3DDifferent%2520methylation%2520characteristics%2520of%2520protein%2520arginine%2520methyltransferase%25201%2520and%25203%2520toward%2520the%2520Ewing%2520sarcoma%2520protein%2520and%2520a%2520peptide%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Bioinf.%26date%3D2005%26volume%3D61%26spage%3D164%26epage%3D175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Cheng, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cote, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaaban, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">The arginine methyltransferase CARM1 regulates the coupling of transcription and mRNA processing</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">83</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1016%2Fj.molcel.2006.11.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=17218272" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFSnur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=71-83&author=D.+H.+Chengauthor=J.+Coteauthor=S.+Shaabanauthor=M.+T.+Bedford&title=The+arginine+methyltransferase+CARM1+regulates+the+coupling+of+transcription+and+mRNA+processing"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The arginine methyltransferase CARM1 regulates the coupling of transcription and mRNA processing</span></div><div class="casAuthors">Cheng, Donghang; Cote, Jocelyn; Shaaban, Salam; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-83</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The coactivator-assocd. arginine methyl-transferase CARM1 is recruited by many different transcription factors as a pos. regulator.  To understand the mechanism by which CARM1 functions, we sought to isolate its substrates.  We developed a small-pool screening approach for this purpose and identified CA150, SAP49, SmB, and U1C as splicing factors that are specifically methylated by CARM1.  We further showed that CA150, a mol. that links transcription to splicing, interacts with the Tudor domain of the spinal muscular atrophy protein SMN in a CARM1-dependent fashion.  Expts. with an exogenous splicing reporter and the endogenous CD44 gene revealed that CARM1 promotes exon skipping in an enzyme-dependent manner.  The identification of splicing factors that are methylated by CARM1, and protein-protein interactions that are regulated by CARM1, strongly implicates this enzyme in the regulation of alternative splicing and points toward its involvement in spinal muscular atrophy pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqulZNAke_ZsrVg90H21EOLACvtfcHk0ljYciOiRYlhQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFSnur8%253D&md5=29f6dccb415e1146e6455a03613f9fa2</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.11.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.11.019%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DD.%2BH.%26aulast%3DCote%26aufirst%3DJ.%26aulast%3DShaaban%26aufirst%3DS.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DThe%2520arginine%2520methyltransferase%2520CARM1%2520regulates%2520the%2520coupling%2520of%2520transcription%2520and%2520mRNA%2520processing%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D25%26spage%3D71%26epage%3D83" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reed, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leder, P.</span><span> </span><span class="NLM_article-title">WW domain-mediated interactions reveal a spliceosome-associated protein that binds a third class of proline-rich motif: the proline glycine and methionine-rich motif</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">95</span><span class="NLM_x">, </span> <span class="NLM_fpage">10602</span><span class="NLM_x">–</span> <span class="NLM_lpage">10607</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1998&pages=10602-10607&author=M.+T.+Bedfordauthor=R.+Reedauthor=P.+Leder&title=WW+domain-mediated+interactions+reveal+a+spliceosome-associated+protein+that+binds+a+third+class+of+proline-rich+motif%3A+the+proline+glycine+and+methionine-rich+motif"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DReed%26aufirst%3DR.%26aulast%3DLeder%26aufirst%3DP.%26atitle%3DWW%2520domain-mediated%2520interactions%2520reveal%2520a%2520spliceosome-associated%2520protein%2520that%2520binds%2520a%2520third%2520class%2520of%2520proline-rich%2520motif%253A%2520the%2520proline%2520glycine%2520and%2520methionine-rich%2520motif%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1998%26volume%3D95%26spage%3D10602%26epage%3D10607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Branscombe, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frankel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pestka, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, S.</span><span> </span><span class="NLM_article-title">PRMT5 (Janus kinase-binding protein 1) catalyzes the formation of symmetric dimethylarginine residues in proteins</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">32971</span><span class="NLM_x">–</span> <span class="NLM_lpage">32976</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=32971-32976&author=T.+L.+Branscombeauthor=A.+Frankelauthor=J.+H.+Leeauthor=J.+R.+Cookauthor=Z.+Yangauthor=S.+Pestkaauthor=S.+Clarke&title=PRMT5+%28Janus+kinase-binding+protein+1%29+catalyzes+the+formation+of+symmetric+dimethylarginine+residues+in+proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBranscombe%26aufirst%3DT.%2BL.%26aulast%3DFrankel%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DCook%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DPestka%26aufirst%3DS.%26aulast%3DClarke%26aufirst%3DS.%26atitle%3DPRMT5%2520%2528Janus%2520kinase-binding%2520protein%25201%2529%2520catalyzes%2520the%2520formation%2520of%2520symmetric%2520dimethylarginine%2520residues%2520in%2520proteins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D32971%26epage%3D32976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Tang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gary, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herschman, H. R.</span><span> </span><span class="NLM_article-title">PRMT 3, a type I protein arginine N-methyltransferase that differs from PRMT1 in its oligomerization, subcellular localization, substrate specificity, and regulation</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">273</span><span class="NLM_x">, </span> <span class="NLM_fpage">16935</span><span class="NLM_x">–</span> <span class="NLM_lpage">16945</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1998&pages=16935-16945&author=J.+Tangauthor=J.+D.+Garyauthor=S.+Clarkeauthor=H.+R.+Herschman&title=PRMT+3%2C+a+type+I+protein+arginine+N-methyltransferase+that+differs+from+PRMT1+in+its+oligomerization%2C+subcellular+localization%2C+substrate+specificity%2C+and+regulation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DGary%26aufirst%3DJ.%2BD.%26aulast%3DClarke%26aufirst%3DS.%26aulast%3DHerschman%26aufirst%3DH.%2BR.%26atitle%3DPRMT%25203%252C%2520a%2520type%2520I%2520protein%2520arginine%2520N-methyltransferase%2520that%2520differs%2520from%2520PRMT1%2520in%2520its%2520oligomerization%252C%2520subcellular%2520localization%252C%2520substrate%2520specificity%252C%2520and%2520regulation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1998%26volume%3D273%26spage%3D16935%26epage%3D16945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Bachand, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silver, P. A.</span><span> </span><span class="NLM_article-title">PRMT3 is a ribosomal protein methyltransferase that affects the cellular levels of ribosomal subunits</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">2641</span><span class="NLM_x">–</span> <span class="NLM_lpage">2650</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=2641-2650&author=F.+Bachandauthor=P.+A.+Silver&title=PRMT3+is+a+ribosomal+protein+methyltransferase+that+affects+the+cellular+levels+of+ribosomal+subunits"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBachand%26aufirst%3DF.%26aulast%3DSilver%26aufirst%3DP.%2BA.%26atitle%3DPRMT3%2520is%2520a%2520ribosomal%2520protein%2520methyltransferase%2520that%2520affects%2520the%2520cellular%2520levels%2520of%2520ribosomal%2520subunits%26jtitle%3DEMBO%2520J.%26date%3D2004%26volume%3D23%26spage%3D2641%26epage%3D2650" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Swiercz, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Person, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">Ribosomal protein S2 is a substrate for mammalian PRMT3 (protein arginine methyltransferase 3)</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">386</span><span class="NLM_x">, </span> <span class="NLM_fpage">85</span><span class="NLM_x">–</span> <span class="NLM_lpage">91</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=386&publication_year=2005&pages=85-91&author=R.+Swierczauthor=M.+D.+Personauthor=M.+T.+Bedford&title=Ribosomal+protein+S2+is+a+substrate+for+mammalian+PRMT3+%28protein+arginine+methyltransferase+3%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSwiercz%26aufirst%3DR.%26aulast%3DPerson%26aufirst%3DM.%2BD.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DRibosomal%2520protein%2520S2%2520is%2520a%2520substrate%2520for%2520mammalian%2520PRMT3%2520%2528protein%2520arginine%2520methyltransferase%25203%2529%26jtitle%3DBiochem.%2520J.%26date%3D2005%26volume%3D386%26spage%3D85%26epage%3D91" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Swiercz, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">Ribosomal protein rpS2 is hypomethylated in PRMT3-deficient mice</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">16917</span><span class="NLM_x">–</span> <span class="NLM_lpage">16923</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1074%2Fjbc.M609778200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=17439947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmt1agsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=16917-16923&author=R.+Swierczauthor=D.+Chengauthor=D.+Kimauthor=M.+T.+Bedford&title=Ribosomal+protein+rpS2+is+hypomethylated+in+PRMT3-deficient+mice"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Ribosomal Protein rpS2 Is Hypomethylated in PRMT3-deficient Mice</span></div><div class="casAuthors">Swiercz, Rafal; Cheng, Donghang; Kim, Daehoon; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">16917-16923</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">PRMT3 is a type I arginine methyltransferase that resides in the cytoplasm.  A large proportion of this cytosolic PRMT3 is found assocd. with ribosomes.  It is tethered to the ribosomes through its interaction with rpS2, which is also its substrate.  Here we show that mouse embryos with a targeted disruption of PRMT3 are small in size but survive after birth and attain a normal size in adulthood, thus displaying Minute-like characteristics.  The ribosome protein rpS2 is hypomethylated in the absence of PRMT3, demonstrating that it is a bona fide, in vivo PRMT3 substrate that cannot be modified by other PRMTs.  Finally, the levels 40 S, 60 S, and 80 S monosomes and polyribosomes are unaffected by the loss of PRMT3, but there are addnl. as yet unidentified proteins that co-fractionate with ribosomes that are also dedicated PRMT3 substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-lEvztlUF4rVg90H21EOLACvtfcHk0liKwVU8LJ1DRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmt1agsLw%253D&md5=4d97b44b6706559086aa2554df786b31</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M609778200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M609778200%26sid%3Dliteratum%253Aachs%26aulast%3DSwiercz%26aufirst%3DR.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DRibosomal%2520protein%2520rpS2%2520is%2520hypomethylated%2520in%2520PRMT3-deficient%2520mice%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D16917%26epage%3D16923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rucknagel, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schierhorn, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nemeth, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linder, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herschman, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahle, E.</span><span> </span><span class="NLM_article-title">Unusual sites of arginine methylation in poly(A)-binding protein II and in vitro methylation by protein arginine methyltransferases PRMT1 and PRMT3</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">274</span><span class="NLM_x">, </span> <span class="NLM_fpage">13229</span><span class="NLM_x">–</span> <span class="NLM_lpage">13234</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1999&pages=13229-13234&author=J.+J.+Smithauthor=K.+P.+Rucknagelauthor=A.+Schierhornauthor=J.+Tangauthor=A.+Nemethauthor=M.+Linderauthor=H.+R.+Herschmanauthor=E.+Wahle&title=Unusual+sites+of+arginine+methylation+in+poly%28A%29-binding+protein+II+and+in+vitro+methylation+by+protein+arginine+methyltransferases+PRMT1+and+PRMT3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DRucknagel%26aufirst%3DK.%2BP.%26aulast%3DSchierhorn%26aufirst%3DA.%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DNemeth%26aufirst%3DA.%26aulast%3DLinder%26aufirst%3DM.%26aulast%3DHerschman%26aufirst%3DH.%2BR.%26aulast%3DWahle%26aufirst%3DE.%26atitle%3DUnusual%2520sites%2520of%2520arginine%2520methylation%2520in%2520poly%2528A%2529-binding%2520protein%2520II%2520and%2520in%2520vitro%2520methylation%2520by%2520protein%2520arginine%2520methyltransferases%2520PRMT1%2520and%2520PRMT3%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1999%26volume%3D274%26spage%3D13229%26epage%3D13234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Fronz, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolbel, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrich, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schierhorn, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beck-Sickinger, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ostareck-Lederer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahle, E.</span><span> </span><span class="NLM_article-title">Promiscuous modification of the nuclear poly(A)-binding protein by multiple protein-arginine methyltransferases does not affect the aggregation behavior</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">20408</span><span class="NLM_x">–</span> <span class="NLM_lpage">20420</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=20408-20420&author=K.+Fronzauthor=S.+Ottoauthor=K.+Kolbelauthor=U.+Kuhnauthor=H.+Friedrichauthor=A.+Schierhornauthor=A.+G.+Beck-Sickingerauthor=A.+Ostareck-Ledererauthor=E.+Wahle&title=Promiscuous+modification+of+the+nuclear+poly%28A%29-binding+protein+by+multiple+protein-arginine+methyltransferases+does+not+affect+the+aggregation+behavior"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFronz%26aufirst%3DK.%26aulast%3DOtto%26aufirst%3DS.%26aulast%3DKolbel%26aufirst%3DK.%26aulast%3DKuhn%26aufirst%3DU.%26aulast%3DFriedrich%26aufirst%3DH.%26aulast%3DSchierhorn%26aufirst%3DA.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26aulast%3DOstareck-Lederer%26aufirst%3DA.%26aulast%3DWahle%26aufirst%3DE.%26atitle%3DPromiscuous%2520modification%2520of%2520the%2520nuclear%2520poly%2528A%2529-binding%2520protein%2520by%2520multiple%2520protein-arginine%2520methyltransferases%2520does%2520not%2520affect%2520the%2520aggregation%2520behavior%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D20408%26epage%3D20420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Tavanez, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bengoechea, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berciano, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lafarga, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carmo-Fonseca, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Enguita, F. J.</span><span> </span><span class="NLM_article-title">Hsp70 chaperones and type I PRMTs are sequestered at intranuclear inclusions caused by polyalanine expansions in PABPN1</span> <span class="citation_source-journal">PloS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">e6418</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=e6418&author=J.+P.+Tavanezauthor=R.+Bengoecheaauthor=M.+T.+Bercianoauthor=M.+Lafargaauthor=M.+Carmo-Fonsecaauthor=F.+J.+Enguita&title=Hsp70+chaperones+and+type+I+PRMTs+are+sequestered+at+intranuclear+inclusions+caused+by+polyalanine+expansions+in+PABPN1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTavanez%26aufirst%3DJ.%2BP.%26aulast%3DBengoechea%26aufirst%3DR.%26aulast%3DBerciano%26aufirst%3DM.%2BT.%26aulast%3DLafarga%26aufirst%3DM.%26aulast%3DCarmo-Fonseca%26aufirst%3DM.%26aulast%3DEnguita%26aufirst%3DF.%2BJ.%26atitle%3DHsp70%2520chaperones%2520and%2520type%2520I%2520PRMTs%2520are%2520sequestered%2520at%2520intranuclear%2520inclusions%2520caused%2520by%2520polyalanine%2520expansions%2520in%2520PABPN1%26jtitle%3DPloS%2520One%26date%3D2009%26volume%3D4%26spage%3De6418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">Fluorescence-based methods for screening writers and readers of histone methyl marks</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">84</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1177%2F1087057111422256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=21972038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC38Xit1Olt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=71-84&author=A.+Allali-Hassaniauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=T.+Hajianauthor=C.+H.+Arrowsmithauthor=M.+Vedadi&title=Fluorescence-based+methods+for+screening+writers+and+readers+of+histone+methyl+marks"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence-based methods for screening writers and readers of histone methyl marks</span></div><div class="casAuthors">Allali-Hassani, Abdellah; Wasney, Gregory A.; Siarheyeva, Alena; Hajian, Taraneh; Arrowsmith, Cheryl H.; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-84</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The histone methyltransferase (HMT) family of proteins consists of enzymes that methylate lysine or arginine residues on histone tails as well as other proteins.  Such modifications affect chromatin structure and play a significant regulatory role in gene expression.  Many HMTs have been implicated in tumorigenesis and progression of multiple malignancies and play essential roles in embryonic development and stem cell renewal.  Overexpression of some HMTs has been obsd. and is correlated pos. with various types of cancer.  Here the authors report development of a continuous fluorescence-based methyltransferase assay in a 384-well format and its application in detg. kinetic parameters for EHMT1, G9a, PRMT3, SETD7, and SUV39H2 as well as for screening against libraries of small mols. to identify enzyme inhibitors.  They also report the development of a peptide displacement assay using fluorescence polarization in a 384-well format to assay and screen protein peptide interactions such as those of WDR5 and EED, components of MLL and EZH2 methyltransferase complexes.  Using these high-throughput screening methods, the authors have identified potent inhibitors and ligands for some of these proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0s8Ex9rZOXrVg90H21EOLACvtfcHk0ljsLC65o3rcsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xit1Olt7g%253D&md5=105a20c2473ab603ade42a38764def09</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1177%2F1087057111422256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057111422256%26sid%3Dliteratum%253Aachs%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DFluorescence-based%2520methods%2520for%2520screening%2520writers%2520and%2520readers%2520of%2520histone%2520methyl%2520marks%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2012%26volume%3D17%26spage%3D71%26epage%3D84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Lai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hakala, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weintraub, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiio, Y.</span><span> </span><span class="NLM_article-title">Proteomic dissection of the von Hippel-Lindau (VHL) interactome</span> <span class="citation_source-journal">J. Proteome Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">5175</span><span class="NLM_x">–</span> <span class="NLM_lpage">5182</span></span><div class="citationLinks">[<a href="/doi/10.1021/pr200642c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht12jsbzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=5175-5182&author=Y.+Laiauthor=M.+Songauthor=K.+Hakalaauthor=S.+T.+Weintraubauthor=Y.+Shiio&title=Proteomic+dissection+of+the+von+Hippel-Lindau+%28VHL%29+interactome"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic Dissection of the von Hippel-Lindau (VHL) Interactome</span></div><div class="casAuthors">Lai, Yanlai; Song, Meihua; Hakala, Kevin; Weintraub, Susan T.; Shiio, Yuzuru</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5175-5182</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The von Hippel-Lindau (VHL) tumor suppressor gene encodes a component of a ubiquitin ligase complex contg. elongin B, elongin C, cullin 2, and Rbx1, which acts as a neg. regulator of hypoxia inducible factor (HIF).  VHL ubiquitinates and degrades the alpha subunits of HIF, and this is proposed to suppress tumorigenesis and tumor angiogenesis.  Several lines of evidence also suggest important roles for HIF-independent VHL functions in the maintenance of primary cilium, extracellular matrix formation, and tumor suppression.  We undertook a series of proteomic analyses to gain a comprehensive picture of the VHL-interacting proteins.  We found that the ARF tumor suppressor interacts with VHL30, a longer VHL isoform, but not with VHL19, a shorter VHL isoform.  ARF was found to release VHL30 from the E3 ligase complex, promoting the binding of VHL30 to a protein arginine methyltransferase, PRMT3.  Our anal. of the VHL19 interactome also uncovered that VHL19 displays an affinity to collagens and their biosynthesis enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqTYZ6RvZ7erVg90H21EOLACvtfcHk0ljcsgfaXn8rCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht12jsbzL&md5=51500af3ec4256311dbd835c47a90200</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fpr200642c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr200642c%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DM.%26aulast%3DHakala%26aufirst%3DK.%26aulast%3DWeintraub%26aufirst%3DS.%2BT.%26aulast%3DShiio%26aufirst%3DY.%26atitle%3DProteomic%2520dissection%2520of%2520the%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520interactome%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2011%26volume%3D10%26spage%3D5175%26epage%3D5182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Singh, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miranda, T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frankel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roemer, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robb, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gutmann, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herschman, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newsham, I. F.</span><span> </span><span class="NLM_article-title">DAL-1/4.1B tumor suppressor interacts with protein arginine N-methyltransferase 3 (PRMT3) and inhibits its ability to methylate substrates in vitro and in vivo</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">7761</span><span class="NLM_x">–</span> <span class="NLM_lpage">7771</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=7761-7771&author=V.+Singhauthor=T.+B.+Mirandaauthor=W.+Jiangauthor=A.+Frankelauthor=M.+E.+Roemerauthor=V.+A.+Robbauthor=D.+H.+Gutmannauthor=H.+R.+Herschmanauthor=S.+Clarkeauthor=I.+F.+Newsham&title=DAL-1%2F4.1B+tumor+suppressor+interacts+with+protein+arginine+N-methyltransferase+3+%28PRMT3%29+and+inhibits+its+ability+to+methylate+substrates+in+vitro+and+in+vivo"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DV.%26aulast%3DMiranda%26aufirst%3DT.%2BB.%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DFrankel%26aufirst%3DA.%26aulast%3DRoemer%26aufirst%3DM.%2BE.%26aulast%3DRobb%26aufirst%3DV.%2BA.%26aulast%3DGutmann%26aufirst%3DD.%2BH.%26aulast%3DHerschman%26aufirst%3DH.%2BR.%26aulast%3DClarke%26aufirst%3DS.%26aulast%3DNewsham%26aufirst%3DI.%2BF.%26atitle%3DDAL-1%252F4.1B%2520tumor%2520suppressor%2520interacts%2520with%2520protein%2520arginine%2520N-methyltransferase%25203%2520%2528PRMT3%2529%2520and%2520inhibits%2520its%2520ability%2520to%2520methylate%2520substrates%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DOncogene%26date%3D2004%26volume%3D23%26spage%3D7761%26epage%3D7771" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niroomand, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zankl, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katus, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jahn, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tiefenbacher, C. P.</span><span> </span><span class="NLM_article-title">Expression of nitric oxide related enzymes in coronary heart disease</span> <span class="citation_source-journal">Basic Res. Cardiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">346</span><span class="NLM_x">–</span> <span class="NLM_lpage">353</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1007%2Fs00395-006-0592-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=16705470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD28Xos1ylurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2006&pages=346-353&author=X.+Chenauthor=F.+Niroomandauthor=Z.+Liuauthor=A.+Zanklauthor=H.+A.+Katusauthor=L.+Jahnauthor=C.+P.+Tiefenbacher&title=Expression+of+nitric+oxide+related+enzymes+in+coronary+heart+disease"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of nitric oxide related enzymes in coronary heart disease</span></div><div class="casAuthors">Chen, X.; Niroomand, F.; Liu, Z.; Zankl, A.; Katus, H. A.; Jahn, L.; Tiefenbacher, C. P.</div><div class="citationInfo"><span class="NLM_cas:title">Basic Research in Cardiology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">346-353</span>CODEN:
                <span class="NLM_cas:coden">BRCAB7</span>;
        ISSN:<span class="NLM_cas:issn">0300-8428</span>.
    
            (<span class="NLM_cas:orgname">Steinkopff Verlag</span>)
        </div><div class="casAbstract">Enzymes involved in the metab. nitric oxide (NO) and reactive oxygen species (ROS) may play a role for the decreased availability of NO in atherosclerosis.  We, therefore, hypothesized that the pattern of gene expression of these enzymes is altered in atherosclerosis.  Myocardial tissue from patients with coronary heart disease (CHD) or without CHD (control group) was investigated.  The level of enzymes related to NO/ROS metab. was detd. both at mRNA level and protein level by rt-PCR, real-time PCR, and western blot.  The expression of NOS1-3 (synthesis of NO), arginasel (redn. of L-arginine), p22phox (active subunit of NADPH oxidase), GTPCH (rate limiting enzyme for tetrahydrobiopterin), SOD1-3 (scavengers of superoxide anions), PRTMT1-3, and DDAH2 (involved in the metab. of ADMA) was detd.  All enzymes were found to be expressed in human myocardium.  NOS isoforms were decreased in CHD in protein level, but only the down regulation of NOS3 expression reached statistical significance.  The expression of PRMT1 and PRMT3 was increased.  In addn., the expression of DDAH2 was reduced, both theor. leading to an increase of ADMA concn.  SOD3 was down regulated in tissue from patients with CHD.  Taken together, in myocardial tissue from patients with atherosclerosis, the expression of genes increasing ADMA levels is enhanced in contrast to a reduced expression of genes promoting NO synthesis.  These results may contribute to the explanation of increased oxidative stress in atherosclerosis on the level of gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreZhed3XxK3bVg90H21EOLACvtfcHk0ljcsgfaXn8rCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xos1ylurk%253D&md5=e40c38e03c6dc050a5a894bd0be20155</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1007%2Fs00395-006-0592-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00395-006-0592-5%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DNiroomand%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DZankl%26aufirst%3DA.%26aulast%3DKatus%26aufirst%3DH.%2BA.%26aulast%3DJahn%26aufirst%3DL.%26aulast%3DTiefenbacher%26aufirst%3DC.%2BP.%26atitle%3DExpression%2520of%2520nitric%2520oxide%2520related%2520enzymes%2520in%2520coronary%2520heart%2520disease%26jtitle%3DBasic%2520Res.%2520Cardiol.%26date%3D2006%26volume%3D101%26spage%3D346%26epage%3D353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Miyata, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mori, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tohyama, M.</span><span> </span><span class="NLM_article-title">PRMT3 is essential for dendritic spine maturation in rat hippocampal neurons</span> <span class="citation_source-journal">Brain Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1352</span><span class="NLM_x">, </span> <span class="NLM_fpage">11</span><span class="NLM_x">–</span> <span class="NLM_lpage">20</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1016%2Fj.brainres.2010.07.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=20647003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVKisL3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1352&publication_year=2010&pages=11-20&author=S.+Miyataauthor=Y.+Moriauthor=M.+Tohyama&title=PRMT3+is+essential+for+dendritic+spine+maturation+in+rat+hippocampal+neurons"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT3 is essential for dendritic spine maturation in rat hippocampal neurons</span></div><div class="casAuthors">Miyata, Shingo; Mori, Yasutake; Tohyama, Masaya</div><div class="citationInfo"><span class="NLM_cas:title">Brain Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1352</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11-20</span>CODEN:
                <span class="NLM_cas:coden">BRREAP</span>;
        ISSN:<span class="NLM_cas:issn">0006-8993</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Protein arginine N-methyltransferase 3 (PRMT3) is a cytoplasmic enzyme that utilizes S-adenosyl-L-methionine (AdoMet) to methylate specific proteins, most of which contain GAR (glycine-arginine rich) motifs.  PRMT3 has been shown to play a role in the proper maturation of the 80S ribosome by binding to and catalyzing the methylation of rpS2, a component of the 40S ribosomal subunit.  However, the other roles of PRMT3 are fairly unclear, particularly in the brain, which is abundant in methylated proteins.  In this study, we perturbed PRMT3 expression in cultured rat hippocampal neurons by transiently introducing siRNA oligonucleotides that were designed to hybridize with PRMT3 mRNA and then we examd. the morphol. and functional effects of neuronal PRMT3 depletion.  PRMT3-defective neurons showed deformed spines without any change in spine no.; less BDNF-induced protein translation of αCaMKII; and diminished rpS2 protein stability.  Furthermore, overexpression of a methylation-resistant rpS2, whose methylated arginine residues were deleted, produced phenotypes that were similar to those assocd. with PRMT3 down regulation.  These findings demonstrated that PRMT3 possibly plays a pivotal role in neuronal translation by interaction with rpS2 and that it contributes to activity-dependent changes in the dendritic spines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZvFaoRRY5lbVg90H21EOLACvtfcHk0liLL4L7QXAYiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVKisL3L&md5=ccac2a5a931c483e516003342916bb17</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.brainres.2010.07.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.brainres.2010.07.033%26sid%3Dliteratum%253Aachs%26aulast%3DMiyata%26aufirst%3DS.%26aulast%3DMori%26aufirst%3DY.%26aulast%3DTohyama%26aufirst%3DM.%26atitle%3DPRMT3%2520is%2520essential%2520for%2520dendritic%2520spine%2520maturation%2520in%2520rat%2520hippocampal%2520neurons%26jtitle%3DBrain%2520Res.%26date%3D2010%26volume%3D1352%26spage%3D11%26epage%3D20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Bissinger, E.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heinke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span> </span><span class="NLM_article-title">Targeting epigenetic modifiers: inhibitors of histone methyltransferases</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">114</span><span class="NLM_x">–</span> <span class="NLM_lpage">124</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=114-124&author=E.-M.+Bissingerauthor=R.+Heinkeauthor=W.+Sipplauthor=M.+Jung&title=Targeting+epigenetic+modifiers%3A+inhibitors+of+histone+methyltransferases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBissinger%26aufirst%3DE.-M.%26aulast%3DHeinke%26aufirst%3DR.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DTargeting%2520epigenetic%2520modifiers%253A%2520inhibitors%2520of%2520histone%2520methyltransferases%26jtitle%3DMedChemComm%26date%3D2010%26volume%3D1%26spage%3D114%26epage%3D124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">He, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span> </span><span class="NLM_article-title">Targeting protein lysine methylation and demethylation in cancers</span> <span class="citation_source-journal">Acta Biochim. Biophys. Sin.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">70</span><span class="NLM_x">–</span> <span class="NLM_lpage">79</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1093%2Fabbs%2Fgmr109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=22194015" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htr7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2012&pages=70-79&author=Y.+Heauthor=I.+Korboukhauthor=J.+Jinauthor=J.+Huang&title=Targeting+protein+lysine+methylation+and+demethylation+in+cancers"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein lysine methylation and demethylation in cancers</span></div><div class="casAuthors">He, Yunlong; Korboukh, Ilia; Jin, Jian; Huang, Jing</div><div class="citationInfo"><span class="NLM_cas:title">Acta Biochimica et Biophysica Sinica</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">70-79</span>CODEN:
                <span class="NLM_cas:coden">ABBSC2</span>;
        ISSN:<span class="NLM_cas:issn">1672-9145</span>.
    
            (<span class="NLM_cas:orgname">ABBS Editorial Office</span>)
        </div><div class="casAbstract">A review.  During the last decade, we saw an explosion of studies investigating the role of lysine methylation/demethylation of histones and non-histone proteins, such as p53, NF-kappaB, and E2F1.  These Ying-Yang' post-translational modifications are important to fine-tuning the activity of these proteins.  Lysine methylation and demethylation are catalyzed by protein lysine methyltransferases (PKMTs) and protein lysine demethylases (PKDMs).  PKMTs, PKDMs, and their substrates have been shown to play important roles in cancers.  Although the underlying mechanisms of tumorigenesis are still largely unknown, growing evidence is starting to link aberrant regulation of methylation to tumorigenesis.  This review focuses on summarizing the recent progress in understanding of the function of protein lysine methylation, and in the discovery of small mol. inhibitors for PKMTs and PKDMs.  We also discuss the potential and the caveats of targeting protein lysine methylation for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO9Pis9NgA27Vg90H21EOLACvtfcHk0liLL4L7QXAYiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htr7P&md5=bca793734c9f24cf45bbb529a41207db</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1093%2Fabbs%2Fgmr109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fabbs%252Fgmr109%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DY.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DJ.%26atitle%3DTargeting%2520protein%2520lysine%2520methylation%2520and%2520demethylation%2520in%2520cancers%26jtitle%3DActa%2520Biochim.%2520Biophys.%2520Sin.%26date%3D2012%26volume%3D44%26spage%3D70%26epage%3D79" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span> </span><span class="NLM_article-title">Epigenetic chemical probes</span> <span class="citation_source-journal">Clin. Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">92</span><span class="NLM_x">, </span> <span class="NLM_fpage">689</span><span class="NLM_x">–</span> <span class="NLM_lpage">693</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2012&pages=689-693&author=S.+Mullerauthor=P.+J.+Brown&title=Epigenetic+chemical+probes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26atitle%3DEpigenetic%2520chemical%2520probes%26jtitle%3DClin.%2520Pharmacol.%2520Ther.%26date%3D2012%26volume%3D92%26spage%3D689%26epage%3D693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Blancafort, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span> </span><span class="NLM_article-title">Writing and re-writing the epigenetic code of cancer cells: from engineered proteins to small molecules</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">563</span><span class="NLM_x">–</span> <span class="NLM_lpage">576</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2013&pages=563-576&author=P.+Blancafortauthor=J.+Jinauthor=S.+V.+Frye&title=Writing+and+re-writing+the+epigenetic+code+of+cancer+cells%3A+from+engineered+proteins+to+small+molecules"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBlancafort%26aufirst%3DP.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DWriting%2520and%2520re-writing%2520the%2520epigenetic%2520code%2520of%2520cancer%2520cells%253A%2520from%2520engineered%2520proteins%2520to%2520small%2520molecules%26jtitle%3DMol.%2520Pharmacol.%26date%3D2013%26volume%3D83%26spage%3D563%26epage%3D576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Cheng, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yadav, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swanson, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinstein, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">Small molecule regulators of protein arginine methyltransferases</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">279</span><span class="NLM_x">, </span> <span class="NLM_fpage">23892</span><span class="NLM_x">–</span> <span class="NLM_lpage">23899</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=23892-23899&author=D.+H.+Chengauthor=N.+Yadavauthor=R.+W.+Kingauthor=M.+S.+Swansonauthor=E.+J.+Weinsteinauthor=M.+T.+Bedford&title=Small+molecule+regulators+of+protein+arginine+methyltransferases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DD.%2BH.%26aulast%3DYadav%26aufirst%3DN.%26aulast%3DKing%26aufirst%3DR.%2BW.%26aulast%3DSwanson%26aufirst%3DM.%2BS.%26aulast%3DWeinstein%26aufirst%3DE.%2BJ.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DSmall%2520molecule%2520regulators%2520of%2520protein%2520arginine%2520methyltransferases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D23892%26epage%3D23899" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Bonham, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hemmers, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finn, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mowen, K. A.</span><span> </span><span class="NLM_article-title">Effects of a novel arginine methyltransferase inhibitor on T-helper cell cytokine production</span> <span class="citation_source-journal">FEBS J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">277</span><span class="NLM_x">, </span> <span class="NLM_fpage">2096</span><span class="NLM_x">–</span> <span class="NLM_lpage">2108</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2010&pages=2096-2108&author=K.+Bonhamauthor=S.+Hemmersauthor=Y.+H.+Limauthor=D.+M.+Hillauthor=M.+G.+Finnauthor=K.+A.+Mowen&title=Effects+of+a+novel+arginine+methyltransferase+inhibitor+on+T-helper+cell+cytokine+production"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBonham%26aufirst%3DK.%26aulast%3DHemmers%26aufirst%3DS.%26aulast%3DLim%26aufirst%3DY.%2BH.%26aulast%3DHill%26aufirst%3DD.%2BM.%26aulast%3DFinn%26aufirst%3DM.%2BG.%26aulast%3DMowen%26aufirst%3DK.%2BA.%26atitle%3DEffects%2520of%2520a%2520novel%2520arginine%2520methyltransferase%2520inhibitor%2520on%2520T-helper%2520cell%2520cytokine%2520production%26jtitle%3DFEBS%2520J.%26date%3D2010%26volume%3D277%26spage%3D2096%26epage%3D2108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Trapp, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meier, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hongwiset, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kassack, M. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span> </span><span class="NLM_article-title">Structure-activity studies on suramin analogues as inhibitors of NAD(+)-dependent histone deacetylases (sirtuins)</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">1419</span><span class="NLM_x">–</span> <span class="NLM_lpage">1431</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=1419-1431&author=J.+Trappauthor=R.+Meierauthor=D.+Hongwisetauthor=M.+U.+Kassackauthor=W.+Sipplauthor=M.+Jung&title=Structure-activity+studies+on+suramin+analogues+as+inhibitors+of+NAD%28%2B%29-dependent+histone+deacetylases+%28sirtuins%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTrapp%26aufirst%3DJ.%26aulast%3DMeier%26aufirst%3DR.%26aulast%3DHongwiset%26aufirst%3DD.%26aulast%3DKassack%26aufirst%3DM.%2BU.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DStructure-activity%2520studies%2520on%2520suramin%2520analogues%2520as%2520inhibitors%2520of%2520NAD%2528%252B%2529-dependent%2520histone%2520deacetylases%2520%2528sirtuins%2529%26jtitle%3DChemMedChem%26date%3D2007%26volume%3D2%26spage%3D1419%26epage%3D1431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Ragno, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeoni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castellano, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vicidomini, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caroli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tramontano, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonaccini, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sbardella, G.</span><span> </span><span class="NLM_article-title">Small molecule inhibitors of histone arginine methyltransferases: homology modeling, molecular docking, binding mode analysis, and biological evaluations</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1241</span><span class="NLM_x">–</span> <span class="NLM_lpage">1253</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm061213n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=1241-1253&author=R.+Ragnoauthor=S.+Simeoniauthor=S.+Castellanoauthor=C.+Vicidominiauthor=A.+Maiauthor=A.+Caroliauthor=A.+Tramontanoauthor=C.+Bonacciniauthor=P.+Trojerauthor=I.+Bauerauthor=G.+Broschauthor=G.+Sbardella&title=Small+molecule+inhibitors+of+histone+arginine+methyltransferases%3A+homology+modeling%2C+molecular+docking%2C+binding+mode+analysis%2C+and+biological+evaluations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm061213n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061213n%26sid%3Dliteratum%253Aachs%26aulast%3DRagno%26aufirst%3DR.%26aulast%3DSimeoni%26aufirst%3DS.%26aulast%3DCastellano%26aufirst%3DS.%26aulast%3DVicidomini%26aufirst%3DC.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DCaroli%26aufirst%3DA.%26aulast%3DTramontano%26aufirst%3DA.%26aulast%3DBonaccini%26aufirst%3DC.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DBauer%26aufirst%3DI.%26aulast%3DBrosch%26aufirst%3DG.%26aulast%3DSbardella%26aufirst%3DG.%26atitle%3DSmall%2520molecule%2520inhibitors%2520of%2520histone%2520arginine%2520methyltransferases%253A%2520homology%2520modeling%252C%2520molecular%2520docking%252C%2520binding%2520mode%2520analysis%252C%2520and%2520biological%2520evaluations%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D1241%26epage%3D1253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nebbioso, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrone, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sbardella, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Bellis, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miceli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altucci, L.</span><span> </span><span class="NLM_article-title">Epigenetic multiple ligands: mixed histone/protein methyltransferase, acetyltransferase, and class III deacetylase (sirtuin) inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2279</span><span class="NLM_x">–</span> <span class="NLM_lpage">2290</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm701595q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2279-2290&author=A.+Maiauthor=D.+Chengauthor=M.+T.+Bedfordauthor=S.+Valenteauthor=A.+Nebbiosoauthor=A.+Perroneauthor=G.+Broschauthor=G.+Sbardellaauthor=F.+De+Bellisauthor=M.+Miceliauthor=L.+Altucci&title=Epigenetic+multiple+ligands%3A+mixed+histone%2Fprotein+methyltransferase%2C+acetyltransferase%2C+and+class+III+deacetylase+%28sirtuin%29+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm701595q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701595q%26sid%3Dliteratum%253Aachs%26aulast%3DMai%26aufirst%3DA.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DNebbioso%26aufirst%3DA.%26aulast%3DPerrone%26aufirst%3DA.%26aulast%3DBrosch%26aufirst%3DG.%26aulast%3DSbardella%26aufirst%3DG.%26aulast%3DDe%2BBellis%26aufirst%3DF.%26aulast%3DMiceli%26aufirst%3DM.%26aulast%3DAltucci%26aufirst%3DL.%26atitle%3DEpigenetic%2520multiple%2520ligands%253A%2520mixed%2520histone%252Fprotein%2520methyltransferase%252C%2520acetyltransferase%252C%2520and%2520class%2520III%2520deacetylase%2520%2528sirtuin%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2279%26epage%3D2290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Purandare, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huynh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaccaro, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poss, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oconnell, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nowak, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayaraman, L.</span><span> </span><span class="NLM_article-title">Pyrazole inhibitors of coactivator associated arginine methyltransferase 1 (CARM1)</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">4438</span><span class="NLM_x">–</span> <span class="NLM_lpage">4441</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1016%2Fj.bmcl.2008.06.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=18619839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptVCjsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=4438-4441&author=A.+V.+Purandareauthor=Z.+Chenauthor=T.+Huynhauthor=S.+Pangauthor=J.+Gengauthor=W.+Vaccaroauthor=M.+A.+Possauthor=J.+Oconnellauthor=K.+Nowakauthor=L.+Jayaraman&title=Pyrazole+inhibitors+of+coactivator+associated+arginine+methyltransferase+1+%28CARM1%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Pyrazole inhibitors of coactivator associated arginine methyltransferase 1 (CARM1)</span></div><div class="casAuthors">Purandare, Ashok V.; Chen, Zhong; Huynh, Tram; Pang, Suhong; Geng, Jieping; Vaccaro, Wayne; Poss, Michael A.; Oconnell, Jonathan; Nowak, Kimberly; Jayaraman, Lata</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4438-4441</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">This study reports the identification and Hits to Leads optimization of inhibitors of coactivator assocd. arginine methyltransferase (CARM1).  Compd. (I) is a potent, selective inhibitor of CARM1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjsEj-ubHXsrVg90H21EOLACvtfcHk0lhd6yKnQt-efQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptVCjsr8%253D&md5=0ef68fcfa0dedbe54d1fa5b8ae9ca376</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.06.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.06.026%26sid%3Dliteratum%253Aachs%26aulast%3DPurandare%26aufirst%3DA.%2BV.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DHuynh%26aufirst%3DT.%26aulast%3DPang%26aufirst%3DS.%26aulast%3DGeng%26aufirst%3DJ.%26aulast%3DVaccaro%26aufirst%3DW.%26aulast%3DPoss%26aufirst%3DM.%2BA.%26aulast%3DOconnell%26aufirst%3DJ.%26aulast%3DNowak%26aufirst%3DK.%26aulast%3DJayaraman%26aufirst%3DL.%26atitle%3DPyrazole%2520inhibitors%2520of%2520coactivator%2520associated%2520arginine%2520methyltransferase%25201%2520%2528CARM1%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D4438%26epage%3D4441" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Huynh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bandiera, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bindi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vianello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roletto, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thieffine, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galvani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaccaro, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poss, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trainor, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorenzi, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gottardis, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayaraman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Purandare, A. V.</span><span> </span><span class="NLM_article-title">Optimization of pyrazole inhibitors of coactivator associated arginine methyltransferase 1 (CARM1)</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2924</span><span class="NLM_x">–</span> <span class="NLM_lpage">2927</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1016%2Fj.bmcl.2009.04.075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=19419866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtVekur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=2924-2927&author=T.+Huynhauthor=Z.+Chenauthor=S.+Pangauthor=J.+Gengauthor=T.+Bandieraauthor=S.+Bindiauthor=P.+Vianelloauthor=F.+Rolettoauthor=S.+Thieffineauthor=A.+Galvaniauthor=W.+Vaccaroauthor=M.+A.+Possauthor=G.+L.+Trainorauthor=M.+V.+Lorenziauthor=M.+Gottardisauthor=L.+Jayaramanauthor=A.+V.+Purandare&title=Optimization+of+pyrazole+inhibitors+of+coactivator+associated+arginine+methyltransferase+1+%28CARM1%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of pyrazole inhibitors of coactivator associated arginine methyltransferase 1 (CARM1)</span></div><div class="casAuthors">Huynh, Tram; Chen, Zhong; Pang, Suhong; Geng, Jieping; Bandiera, Tiziano; Bindi, Simona; Vianello, Paola; Roletto, Fulvia; Thieffine, Sandrine; Galvani, Arturo; Vaccaro, Wayne; Poss, Michael A.; Trainor, George L.; Lorenzi, Matthew V.; Gottardis, Marco; Jayaraman, Lata; Purandare, Ashok V.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2924-2927</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Design, synthesis, and SAR development led to the identification of the potent, novel, and selective pyrazole based inhibitor I of CARM1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphFardNepde7Vg90H21EOLACvtfcHk0lgYtqcm2tRmTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtVekur4%253D&md5=c37a2738bbd001d4758f8480746626fb</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.04.075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.04.075%26sid%3Dliteratum%253Aachs%26aulast%3DHuynh%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DPang%26aufirst%3DS.%26aulast%3DGeng%26aufirst%3DJ.%26aulast%3DBandiera%26aufirst%3DT.%26aulast%3DBindi%26aufirst%3DS.%26aulast%3DVianello%26aufirst%3DP.%26aulast%3DRoletto%26aufirst%3DF.%26aulast%3DThieffine%26aufirst%3DS.%26aulast%3DGalvani%26aufirst%3DA.%26aulast%3DVaccaro%26aufirst%3DW.%26aulast%3DPoss%26aufirst%3DM.%2BA.%26aulast%3DTrainor%26aufirst%3DG.%2BL.%26aulast%3DLorenzi%26aufirst%3DM.%2BV.%26aulast%3DGottardis%26aufirst%3DM.%26aulast%3DJayaraman%26aufirst%3DL.%26aulast%3DPurandare%26aufirst%3DA.%2BV.%26atitle%3DOptimization%2520of%2520pyrazole%2520inhibitors%2520of%2520coactivator%2520associated%2520arginine%2520methyltransferase%25201%2520%2528CARM1%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D2924%26epage%3D2927" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Allan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manku, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therrien, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Styhler, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robert, M.-F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goulet, A.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petschner, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahil, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacLeod, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deziel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Besterman, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahhab, A.</span><span> </span><span class="NLM_article-title"><i>N</i>-Benzyl-1-heteroaryl-3-(trifluoromethyl)-1<i>H</i>-pyrazole-5-carboxamides as inhibitors of co-activator associated arginine methyltransferase 1 (CARM1)</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1218</span><span class="NLM_x">–</span> <span class="NLM_lpage">1223</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=1218-1223&author=M.+Allanauthor=S.+Mankuauthor=E.+Therrienauthor=N.+Nguyenauthor=S.+Styhlerauthor=M.-F.+Robertauthor=A.-C.+Gouletauthor=A.+J.+Petschnerauthor=G.+Rahilauthor=A.+R.+MacLeodauthor=R.+Dezielauthor=J.+M.+Bestermanauthor=H.+Nguyenauthor=A.+Wahhab&title=N-Benzyl-1-heteroaryl-3-%28trifluoromethyl%29-1H-pyrazole-5-carboxamides+as+inhibitors+of+co-activator+associated+arginine+methyltransferase+1+%28CARM1%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAllan%26aufirst%3DM.%26aulast%3DManku%26aufirst%3DS.%26aulast%3DTherrien%26aufirst%3DE.%26aulast%3DNguyen%26aufirst%3DN.%26aulast%3DStyhler%26aufirst%3DS.%26aulast%3DRobert%26aufirst%3DM.-F.%26aulast%3DGoulet%26aufirst%3DA.-C.%26aulast%3DPetschner%26aufirst%3DA.%2BJ.%26aulast%3DRahil%26aufirst%3DG.%26aulast%3DMacLeod%26aufirst%3DA.%2BR.%26aulast%3DDeziel%26aufirst%3DR.%26aulast%3DBesterman%26aufirst%3DJ.%2BM.%26aulast%3DNguyen%26aufirst%3DH.%26aulast%3DWahhab%26aufirst%3DA.%26atitle%3DN-Benzyl-1-heteroaryl-3-%2528trifluoromethyl%2529-1H-pyrazole-5-carboxamides%2520as%2520inhibitors%2520of%2520co-activator%2520associated%2520arginine%2520methyltransferase%25201%2520%2528CARM1%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D1218%26epage%3D1223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heinke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gust, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span> </span><span class="NLM_article-title">Target-based approach to inhibitors of histone arginine methyltransferases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">2319</span><span class="NLM_x">–</span> <span class="NLM_lpage">2325</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm061250e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=2319-2325&author=A.+Spannhoffauthor=R.+Heinkeauthor=I.+Bauerauthor=P.+Trojerauthor=E.+Metzgerauthor=R.+Gustauthor=R.+Schuleauthor=G.+Broschauthor=W.+Sipplauthor=M.+Jung&title=Target-based+approach+to+inhibitors+of+histone+arginine+methyltransferases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm061250e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061250e%26sid%3Dliteratum%253Aachs%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DHeinke%26aufirst%3DR.%26aulast%3DBauer%26aufirst%3DI.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DGust%26aufirst%3DR.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DBrosch%26aufirst%3DG.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DTarget-based%2520approach%2520to%2520inhibitors%2520of%2520histone%2520arginine%2520methyltransferases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D2319%26epage%3D2325" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Heinke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meier, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span> </span><span class="NLM_article-title">Virtual screening and biological characterization of novel histone arginine methyltransferase PRMT1 inhibitors</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">69</span><span class="NLM_x">–</span> <span class="NLM_lpage">77</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=69-77&author=R.+Heinkeauthor=A.+Spannhoffauthor=R.+Meierauthor=P.+Trojerauthor=I.+Bauerauthor=M.+Jungauthor=W.+Sippl&title=Virtual+screening+and+biological+characterization+of+novel+histone+arginine+methyltransferase+PRMT1+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHeinke%26aufirst%3DR.%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DMeier%26aufirst%3DR.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DBauer%26aufirst%3DI.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DSippl%26aufirst%3DW.%26atitle%3DVirtual%2520screening%2520and%2520biological%2520characterization%2520of%2520novel%2520histone%2520arginine%2520methyltransferase%2520PRMT1%2520inhibitors%26jtitle%3DChemMedChem%26date%3D2009%26volume%3D4%26spage%3D69%26epage%3D77" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machmur, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heinke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanefeld, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span> </span><span class="NLM_article-title">A novel arginine methyltransferase inhibitor with cellular activity</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">4150</span><span class="NLM_x">–</span> <span class="NLM_lpage">4153</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=4150-4153&author=A.+Spannhoffauthor=R.+Machmurauthor=R.+Heinkeauthor=P.+Trojerauthor=I.+Bauerauthor=G.+Broschauthor=R.+Schuleauthor=W.+Hanefeldauthor=W.+Sipplauthor=M.+Jung&title=A+novel+arginine+methyltransferase+inhibitor+with+cellular+activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DMachmur%26aufirst%3DR.%26aulast%3DHeinke%26aufirst%3DR.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DBauer%26aufirst%3DI.%26aulast%3DBrosch%26aufirst%3DG.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DHanefeld%26aufirst%3DW.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DA%2520novel%2520arginine%2520methyltransferase%2520inhibitor%2520with%2520cellular%2520activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D4150%26epage%3D4153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Feng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, Y. G.</span><span> </span><span class="NLM_article-title">Discovery and mechanistic study of a class of protein arginine methylation inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">6028</span><span class="NLM_x">–</span> <span class="NLM_lpage">6039</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm100416n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=6028-6039&author=Y.+Fengauthor=M.+Y.+Liauthor=B.+H.+Wangauthor=Y.+G.+Zheng&title=Discovery+and+mechanistic+study+of+a+class+of+protein+arginine+methylation+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm100416n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100416n%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DM.%2BY.%26aulast%3DWang%26aufirst%3DB.%2BH.%26aulast%3DZheng%26aufirst%3DY.%2BG.%26atitle%3DDiscovery%2520and%2520mechanistic%2520study%2520of%2520a%2520class%2520of%2520protein%2520arginine%2520methylation%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D6028%26epage%3D6039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Selvi, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Batta, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kishore, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mantelingu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varier, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanyam, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pradhan, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dasgupta, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sriram, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agrawal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kundu, T. K.</span><span> </span><span class="NLM_article-title">Identification of a novel inhibitor of coactivator-associated arginine methyltransferase 1 (CARM1)-mediated methylation of histone H3 Arg-17</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">7143</span><span class="NLM_x">–</span> <span class="NLM_lpage">7152</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1074%2Fjbc.M109.063933" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=20022955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC3cXisFalu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=7143-7152&author=B.+R.+Selviauthor=K.+Battaauthor=A.+H.+Kishoreauthor=K.+Mantelinguauthor=R.+A.+Varierauthor=K.+Balasubramanyamauthor=S.+K.+Pradhanauthor=D.+Dasguptaauthor=S.+Sriramauthor=S.+Agrawalauthor=T.+K.+Kundu&title=Identification+of+a+novel+inhibitor+of+coactivator-associated+arginine+methyltransferase+1+%28CARM1%29-mediated+methylation+of+histone+H3+Arg-17"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Novel Inhibitor of Coactivator-associated Arginine Methyltransferase 1 (CARM1)-mediated Methylation of Histone H3 Arg-17</span></div><div class="casAuthors">Selvi, B. Ruthrotha; Batta, Kiran; Kishore, A. Hari; Mantelingu, Kempegowda; Varier, Radhika A.; Balasubramanyam, Karanam; Pradhan, Suman Kalyan; Dasgupta, Dipak; Sriram, Sokalingam; Agrawal, Shipra; Kundu, Tapas K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">7143-7152</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Methylation of the arginine residues of histones by methyltransferases has important consequences for chromatin structure and gene regulation; however, the mol. mechanism(s) of methyltransferase regulation is still unclear, as is the biol. significance of methylation at particular arginine residues.  Here, we report a novel specific inhibitor of coactivator-assocd. arginine methyltransferase 1 (CARM1; also known as PRMT4) that selectively inhibits methylation at arginine 17 of histone H3 (H3R17).  Remarkably, this plant-derived inhibitor, called TBBD (ellagic acid), binds to the substrate (histone) preferentially at the signature motif, "KAPRK", where the proline residue (Pro-16) plays a crit. role for interaction and subsequent enzyme inhibition.  In a promoter-specific context, inhibition of H3R17 methylation represses expression of p21, a p53-responsive gene, thus implicating a possible role for H3 Arg-17 methylation in tumor suppressor function.  These data establish TBBD as a novel specific inhibitor of arginine methylation and demonstrate substrate sequence-directed inhibition of enzyme activity by a small mol. and its physiol. consequence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoacMKhuo6Pl7Vg90H21EOLACvtfcHk0lixm7FPdD4K5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXisFalu7c%253D&md5=6be021194a5bbd5deeea1990df6238fd</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.063933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.063933%26sid%3Dliteratum%253Aachs%26aulast%3DSelvi%26aufirst%3DB.%2BR.%26aulast%3DBatta%26aufirst%3DK.%26aulast%3DKishore%26aufirst%3DA.%2BH.%26aulast%3DMantelingu%26aufirst%3DK.%26aulast%3DVarier%26aufirst%3DR.%2BA.%26aulast%3DBalasubramanyam%26aufirst%3DK.%26aulast%3DPradhan%26aufirst%3DS.%2BK.%26aulast%3DDasgupta%26aufirst%3DD.%26aulast%3DSriram%26aufirst%3DS.%26aulast%3DAgrawal%26aufirst%3DS.%26aulast%3DKundu%26aufirst%3DT.%2BK.%26atitle%3DIdentification%2520of%2520a%2520novel%2520inhibitor%2520of%2520coactivator-associated%2520arginine%2520methyltransferase%25201%2520%2528CARM1%2529-mediated%2520methylation%2520of%2520histone%2520H3%2520Arg-17%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D7143%26epage%3D7152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinha, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chai, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muniyan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chou, Y. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kathy, Li, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">George Zheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Pharmacophore-based virtual screening and biological evaluation of small molecule inhibitors for protein arginine methylation</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">7978</span><span class="NLM_x">–</span> <span class="NLM_lpage">7987</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm300521m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7978-7987&author=J.+Wangauthor=L.+Chenauthor=S.+H.+Sinhaauthor=Z.+Liangauthor=H.+Chaiauthor=S.+Muniyanauthor=Y.+W.+Chouauthor=C.+Yangauthor=L.+Yanauthor=Y.+Fengauthor=Li%2C+K.+Kathyauthor=M.+F.+Linauthor=H.+Jiangauthor=Y.+George+Zhengauthor=C.+Luo&title=Pharmacophore-based+virtual+screening+and+biological+evaluation+of+small+molecule+inhibitors+for+protein+arginine+methylation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjm300521m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300521m%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DSinha%26aufirst%3DS.%2BH.%26aulast%3DLiang%26aufirst%3DZ.%26aulast%3DChai%26aufirst%3DH.%26aulast%3DMuniyan%26aufirst%3DS.%26aulast%3DChou%26aufirst%3DY.%2BW.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DYan%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DKathy%26aufirst%3DLi%252C%2BK.%26aulast%3DLin%26aufirst%3DM.%2BF.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DGeorge%2BZheng%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DPharmacophore-based%2520virtual%2520screening%2520and%2520biological%2520evaluation%2520of%2520small%2520molecule%2520inhibitors%2520for%2520protein%2520arginine%2520methylation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D7978%26epage%3D7987" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Sullivan, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">August, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickey, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teodoro, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rea, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mechtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kowalski, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Homon, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span> </span><span class="NLM_article-title">Reversal of H3K9me2 by a small-molecule inhibitor for the G9a histone methyltransferase</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">473</span><span class="NLM_x">–</span> <span class="NLM_lpage">481</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1016%2Fj.molcel.2007.01.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=17289593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitl2jurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=473-481&author=S.+Kubicekauthor=R.+J.+O%E2%80%99Sullivanauthor=E.+M.+Augustauthor=E.+R.+Hickeyauthor=Q.+Zhangauthor=M.+L.+Teodoroauthor=S.+Reaauthor=K.+Mechtlerauthor=J.+A.+Kowalskiauthor=C.+A.+Homonauthor=T.+A.+Kellyauthor=T.+Jenuwein&title=Reversal+of+H3K9me2+by+a+small-molecule+inhibitor+for+the+G9a+histone+methyltransferase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of H3K9me2 by a small-molecule inhibitor for the G9a histone methyltransferase</span></div><div class="casAuthors">Kubicek, Stefan; O'Sullivan, Roderick J.; August, E. Michael; Hickey, Eugene R.; Zhang, Qiang; Teodoro, Miguel L.; Rea, Stephen; Mechtler, Karl; Kowalski, Jennifer A.; Homon, Carol Ann; Kelly, Terence A.; Jenuwein, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">473-481</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone lysine methylation has important roles in the organization of chromatin domains and the regulation of gene expression.  To analyze its function and modulate its activity, we screened for specific inhibitors against histone lysine methyltransferases (HMTases) using recombinant G9a as the target enzyme.  From a chem. library comprising 125,000 preselected compds., seven hits were identified.  Of those, one inhibitor, BIX-01294 (diazepinquinazolin-amine deriv.), does not compete with the cofactor S-adenosyl-methionine, and selectively impairs the G9a HMTase and the generation of H3K9me2 in vitro.  In cellular assays, transient incubation of several cell lines with BIX-01294 lowers bulk H3K9me2 levels that are restored upon removal of the inhibitor.  Importantly, chromatin immunopptn. at several G9a target genes demonstrates reversible redn. of promoter-proximal H3K9me2 in inhibitor-treated mouse ES cells and fibroblasts.  Our data identify a biol. active HMTase inhibitor that allows for the transient modulation of H3K9me2 marks in mammalian chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3SW8czS4WjbVg90H21EOLACvtfcHk0liA5EdayQKYCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitl2jurs%253D&md5=d2f5290f1252e630a9b05489b0775c2c</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.01.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.01.017%26sid%3Dliteratum%253Aachs%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DR.%2BJ.%26aulast%3DAugust%26aufirst%3DE.%2BM.%26aulast%3DHickey%26aufirst%3DE.%2BR.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DTeodoro%26aufirst%3DM.%2BL.%26aulast%3DRea%26aufirst%3DS.%26aulast%3DMechtler%26aufirst%3DK.%26aulast%3DKowalski%26aufirst%3DJ.%2BA.%26aulast%3DHomon%26aufirst%3DC.%2BA.%26aulast%3DKelly%26aufirst%3DT.%2BA.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DReversal%2520of%2520H3K9me2%2520by%2520a%2520small-molecule%2520inhibitor%2520for%2520the%2520G9a%2520histone%2520methyltransferase%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D25%26spage%3D473%26epage%3D481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Protein lysine methyltransferase G9a inhibitors: design, synthesis, and structure activity relationships of 2,4-diamino-7-aminoalkoxy-quinazolines</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5844</span><span class="NLM_x">–</span> <span class="NLM_lpage">5857</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm100478y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC3cXos1ekurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5844-5857&author=F.+Liuauthor=X.+Chenauthor=A.+Allali-Hassaniauthor=A.+M.+Quinnauthor=T.+J.+Wigleauthor=G.+A.+Wasneyauthor=A.+Dongauthor=G.+Senisterraauthor=I.+Chauauthor=A.+Siarheyevaauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Jadhavauthor=J.+M.+Heroldauthor=W.+P.+Janzenauthor=C.+H.+Arrowsmithauthor=S.+V.+Fryeauthor=P.+J.+Brownauthor=A.+Simeonovauthor=M.+Vedadiauthor=J.+Jin&title=Protein+lysine+methyltransferase+G9a+inhibitors%3A+design%2C+synthesis%2C+and+structure+activity+relationships+of+2%2C4-diamino-7-aminoalkoxy-quinazolines"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Lysine Methyltransferase G9a Inhibitors: Design, Synthesis, and Structure Activity Relationships of 2,4-Diamino-7-aminoalkoxy-quinazolines</span></div><div class="casAuthors">Liu, Feng; Chen, Xin; Allali-Hassani, Abdellah; Quinn, Amy M.; Wigle, Tim J.; Wasney, Gregory A.; Dong, Aiping; Senisterra, Guillermo; Chau, Irene; Siarheyeva, Alena; Norris, Jacqueline L.; Kireev, Dmitri B.; Jadhav, Ajit; Herold, J. Martin; Janzen, William P.; Arrowsmith, Cheryl H.; Frye, Stephen V.; Brown, Peter J.; Simeonov, Anton; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5844-5857</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein lysine methyltransferase G9a, which catalyzes methylation of lysine 9 of histone H3 (H3K9) and lysine 373 (K373) of p53, is overexpressed in human cancers.  Genetic knockdown of G9a inhibits cancer cell growth; and the dimethylation of p53 K373 results in the inactivation of p53.  Initial SAR exploration of the 2,4-diamino-6,7-dimethoxyquinazoline template represented by I ( R1 = Bn; R2 = Me)(BIX01294), a selective small mol. inhibitor of G9a and GLP (G9a-like protein), led to the discovery of I [R1 = Me; R2 = (CH2)3NMe2](II)(UNC0224) as a potent G9a inhibitor with excellent selectivity.  A high resoln. X-ray crystal structure of the G9a-II complex, the first cocrystal structure of G9a with a small mol. inhibitor, was obtained.  On the basis of the structural insights revealed by this cocrystal structure, optimization of the 7-(dimethylamino)propoxy side chain of II resulted in the discovery of I [R1 = Me; R2 = (CH2)2O(CH2)2NMe2](UNC0321, Morrison Ki = 63 pM), which is the first G9a inhibitor with picomolar potency and the most potent G9a inhibitor to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgS0YxdNdNxLVg90H21EOLACvtfcHk0lgsphiJBBQ3AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXos1ekurk%253D&md5=8bd703c0a975a33bf9e5c019f1f95ee6</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm100478y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100478y%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProtein%2520lysine%2520methyltransferase%2520G9a%2520inhibitors%253A%2520design%252C%2520synthesis%252C%2520and%2520structure%2520activity%2520relationships%2520of%25202%252C4-diamino-7-aminoalkoxy-quinazolines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5844%26epage%3D5857" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Chang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horton, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snyder, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span> </span><span class="NLM_article-title">Adding a lysine mimic in the design of potent inhibitors of histone lysine methyltransferases</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">400</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=400&publication_year=2010&pages=1-7&author=Y.+Changauthor=T.+Ganeshauthor=J.+R.+Hortonauthor=A.+Spannhoffauthor=J.+Liuauthor=A.+Sunauthor=X.+Zhangauthor=M.+T.+Bedfordauthor=Y.+Shinkaiauthor=J.+P.+Snyderauthor=X.+Cheng&title=Adding+a+lysine+mimic+in+the+design+of+potent+inhibitors+of+histone+lysine+methyltransferases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DY.%26aulast%3DGanesh%26aufirst%3DT.%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DShinkai%26aufirst%3DY.%26aulast%3DSnyder%26aufirst%3DJ.%2BP.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DAdding%2520a%2520lysine%2520mimic%2520in%2520the%2520design%2520of%2520potent%2520inhibitors%2520of%2520histone%2520lysine%2520methyltransferases%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D400%26spage%3D1%26epage%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozieradzki, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Discovery of a 2,4-diamino-7-aminoalkoxyquinazoline as a potent and selective inhibitor of histone lysine methyltransferase G9a</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">7950</span><span class="NLM_x">–</span> <span class="NLM_lpage">7953</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm901543m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlOiurzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=7950-7953&author=F.+Liuauthor=X.+Chenauthor=A.+Allali-Hassaniauthor=A.+M.+Quinnauthor=G.+A.+Wasneyauthor=A.+Dongauthor=D.+Barsyteauthor=I.+Kozieradzkiauthor=G.+Senisterraauthor=I.+Chauauthor=A.+Siarheyevaauthor=D.+B.+Kireevauthor=A.+Jadhavauthor=J.+M.+Heroldauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=A.+Simeonovauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+a+2%2C4-diamino-7-aminoalkoxyquinazoline+as+a+potent+and+selective+inhibitor+of+histone+lysine+methyltransferase+G9a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a 2,4-Diamino-7-aminoalkoxyquinazoline as a Potent and Selective Inhibitor of Histone Lysine Methyltransferase G9a</span></div><div class="casAuthors">Liu, Feng; Chen, Xin; Allali-Hassani, Abdellah; Quinn, Amy M.; Wasney, Gregory A.; Dong, Aiping; Barsyte, Dalia; Kozieradzki, Ivona; Senisterra, Guillermo; Chau, Irene; Siarheyeva, Alena; Kireev, Dmitri B.; Jadhav, Ajit; Herold, J. Martin; Frye, Stephen V.; Arrowsmith, Cheryl H.; Brown, Peter J.; Simeonov, Anton; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7950-7953</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SAR exploration of the 2,4-diamino-6,7-dimethoxyquinazoline template led to the discovery of 8 (I) as a potent and selective G9a inhibitor.  A high resoln. X-ray crystal structure of the G9a-8 complex, the first cocrystal structure of G9a with a small mol. inhibitor, was obtained.  The cocrystal structure validated our binding hypothesis and will enable structure-based design of novel inhibitors.  8 Is a useful tool for investigating the biol. of G9a and its roles in chromatin remodeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjWt5XsGXyj7Vg90H21EOLACvtfcHk0lh9re3Z0pVidA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlOiurzN&md5=2d7e34a8c4b5bf1cafd4e01ad2a86de1</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm901543m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901543m%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DBarsyte%26aufirst%3DD.%26aulast%3DKozieradzki%26aufirst%3DI.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%25202%252C4-diamino-7-aminoalkoxyquinazoline%2520as%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520histone%2520lysine%2520methyltransferase%2520G9a%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D7950%26epage%3D7953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yates, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Optimization of cellular activity of G9a inhibitors 7-aminoalkoxy-quinazolines</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">6139</span><span class="NLM_x">–</span> <span class="NLM_lpage">6150</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm200903z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvVWltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6139-6150&author=F.+Liuauthor=D.+Barsyte-Lovejoyauthor=A.+Allali-Hassaniauthor=Y.+Heauthor=J.+M.+Heroldauthor=X.+Chenauthor=C.+M.+Yatesauthor=S.+V.+Fryeauthor=P.+J.+Brownauthor=J.+Huangauthor=M.+Vedadiauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=Optimization+of+cellular+activity+of+G9a+inhibitors+7-aminoalkoxy-quinazolines"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of cellular activity of G9a inhibitors 7-aminoalkoxy-quinazolines</span></div><div class="casAuthors">Liu, Feng; Barsyte-Lovejoy, Dalia; Allali-Hassani, Abdellah; He, Yunlong; Herold, J. Martin; Chen, Xin; Yates, Christopher M.; Frye, Stephen V.; Brown, Peter J.; Huang, Jing; Vedadi, Masoud; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6139-6150</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein lysine methyltransferase G9a plays key roles in the transcriptional repression of a variety of genes via dimethylation of lysine 9 on histone H3 (H3K9me2) of chromatin as well as dimethylation of nonhistone proteins including tumor suppressor p53.  We previously reported the discovery of UNC0321, the most potent G9a inhibitor to date, via structure-based design and structure-activity relationship (SAR) exploration of the quinazoline scaffold represented by BIX01294.  Despite its very high in vitro potency, compd. UNC0321 lacks sufficient cellular potency.  The design and synthesis of several generations of new analogs aimed at improving cell membrane permeability while maintaining high in vitro potency resulted in the discovery of a no. of novel G9a inhibitors such as UNC0646 I and UNC0631 II with excellent potency in a variety of cell lines and excellent sepn. of functional potency vs. cell toxicity.  The design, synthesis, and cellular SAR of these potent G9a inhibitors are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC91yKB7S34rVg90H21EOLACvtfcHk0lh9re3Z0pVidA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvVWltLg%253D&md5=f3f54b91e6b24aa248498f9ff1fdb1ae</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fjm200903z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200903z%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DYates%26aufirst%3DC.%2BM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DOptimization%2520of%2520cellular%2520activity%2520of%2520G9a%2520inhibitors%25207-aminoalkoxy-quinazolines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6139%26epage%3D6150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rival-Gervier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labrie, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiMaggio, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempel, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangano, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpson, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattenden, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tripathy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petronis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">566</span><span class="NLM_x">–</span> <span class="NLM_lpage">574</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1038%2Fnchembio.599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=21743462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=566-574&author=M.+Vedadiauthor=D.+Barsyte-Lovejoyauthor=F.+Liuauthor=S.+Rival-Gervierauthor=A.+Allali-Hassaniauthor=V.+Labrieauthor=T.+J.+Wigleauthor=P.+A.+DiMaggioauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=A.+Dongauthor=W.+Tempelauthor=S.-C.+Wangauthor=X.+Chenauthor=I.+Chauauthor=T.+Manganoauthor=X.-P.+Huangauthor=C.+D.+Simpsonauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Tripathyauthor=A.+Edwardsauthor=B.+L.+Rothauthor=W.+P.+Janzenauthor=B.+A.+Garciaauthor=A.+Petronisauthor=J.+Ellisauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=A+chemical+probe+selectively+inhibits+G9a+and+GLP+methyltransferase+activity+in+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span></div><div class="casAuthors">Vedadi, Masoud; Barsyte-Lovejoy, Dalia; Liu, Feng; Rival-Gervier, Sylvie; Allali-Hassani, Abdellah; Labrie, Viviane; Wigle, Tim J.; DiMaggio, Peter A.; Wasney, Gregory A.; Siarheyeva, Alena; Dong, Aiping; Tempel, Wolfram; Wang, Sun-Chong; Chen, Xin; Chau, Irene; Mangano, Thomas J.; Huang, Xi-ping; Simpson, Catherine D.; Pattenden, Samantha G.; Norris, Jacqueline L.; Kireev, Dmitri B.; Tripathy, Ashutosh; Edwards, Aled; Roth, Bryan L.; Janzen, William P.; Garcia, Benjamin A.; Petronis, Arturas; Ellis, James; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">566-574</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases G9a and GLP modulate the transcriptional repression of a variety of genes via dimethylation of Lys9 on histone H3 (H3K9me2) as well as dimethylation of non-histone targets.  Here we report the discovery of UNC0638, an inhibitor of G9a and GLP with excellent potency and selectivity over a wide range of epigenetic and non-epigenetic targets.  UNC0638 treatment of a variety of cell lines resulted in lower global H3K9me2 levels, equiv. to levels obsd. for small hairpin RNA knockdown of G9a and GLP with the functional potency of UNC0638 being well sepd. from its toxicity.  UNC0638 markedly reduced the clonogenicity of MCF7 cells, reduced the abundance of H3K9me2 marks at promoters of known G9a-regulated endogenous genes and disproportionately affected several genomic loci encoding microRNAs.  In mouse embryonic stem cells, UNC0638 reactivated G9a-silenced genes and a retroviral reporter gene in a concn.-dependent manner without promoting differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphv-2bZcBqwLVg90H21EOLACvtfcHk0lhS-5XXOVQbog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D&md5=a944e99a910194c1a15240fc6dc8f852</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.599%26sid%3Dliteratum%253Aachs%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DRival-Gervier%26aufirst%3DS.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLabrie%26aufirst%3DV.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DDiMaggio%26aufirst%3DP.%2BA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DS.-C.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMangano%26aufirst%3DT.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DSimpson%26aufirst%3DC.%2BD.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DEdwards%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DPetronis%26aufirst%3DA.%26aulast%3DEllis%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520chemical%2520probe%2520selectively%2520inhibits%2520G9a%2520and%2520GLP%2520methyltransferase%2520activity%2520in%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D566%26epage%3D574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Daigle, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therkelsen, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sneeringer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnston, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernt, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tseng, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span> </span><span class="NLM_article-title">Selective killing of mixed lineage leukemia cells by a potent small-molecule DOT1L inhibitor</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">53</span><span class="NLM_x">–</span> <span class="NLM_lpage">65</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1016%2Fj.ccr.2011.06.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=21741596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslCms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2011&pages=53-65&author=S.+R.+Daigleauthor=E.+J.+Olhavaauthor=C.+A.+Therkelsenauthor=C.+R.+Majerauthor=C.+J.+Sneeringerauthor=J.+Songauthor=L.+D.+Johnstonauthor=M.+P.+Scottauthor=J.+J.+Smithauthor=Y.+Xiaoauthor=L.+Jinauthor=K.+W.+Kuntzauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=K.+M.+Berntauthor=J.+C.+Tsengauthor=A.+L.+Kungauthor=S.+A.+Armstrongauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollock&title=Selective+killing+of+mixed+lineage+leukemia+cells+by+a+potent+small-molecule+DOT1L+inhibitor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Killing of Mixed Lineage Leukemia Cells by a Potent Small-Molecule DOT1L Inhibitor</span></div><div class="casAuthors">Daigle, Scott R.; Olhava, Edward J.; Therkelsen, Carly A.; Majer, Christina R.; Sneeringer, Christopher J.; Song, Jeffrey; Johnston, L. Danielle; Scott, Margaret Porter; Smith, Jesse J.; Xiao, Yonghong; Jin, Lei; Kuntz, Kevin W.; Chesworth, Richard; Moyer, Mikel P.; Bernt, Kathrin M.; Tseng, Jen-Chieh; Kung, Andrew L.; Armstrong, Scott A.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">53-65</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Mislocated enzymic activity of DOT1L has been proposed as a driver of leukemogenesis in mixed lineage leukemia (MLL).  The characterization of EPZ004777, a potent, selective inhibitor of DOT1L is reported.  Treatment of MLL cells with the compd. selectively inhibits H3K79 methylation and blocks expression of leukemogenic genes.  Exposure of leukemic cells to EPZ004777 results in selective killing of those cells bearing the MLL gene translocation, with little effect on non-MLL-translocated cells.  Finally, in vivo administration of EPZ004777 leads to extension of survival in a mouse MLL xenograft model.  These results provide compelling support for DOT1L inhibition as a basis for targeted therapeutics against MLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodbBbxHFmyybVg90H21EOLACvtfcHk0lhS-5XXOVQbog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslCms7c%253D&md5=1d39f99c2983b6b5fccc079110092c3d</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2011.06.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2011.06.009%26sid%3Dliteratum%253Aachs%26aulast%3DDaigle%26aufirst%3DS.%2BR.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DTherkelsen%26aufirst%3DC.%2BA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DBernt%26aufirst%3DK.%2BM.%26aulast%3DTseng%26aufirst%3DJ.%2BC.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26atitle%3DSelective%2520killing%2520of%2520mixed%2520lineage%2520leukemia%2520cells%2520by%2520a%2520potent%2520small-molecule%2520DOT1L%2520inhibitor%26jtitle%3DCancer%2520Cell%26date%3D2011%26volume%3D20%26spage%3D53%26epage%3D65" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Ferguson, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larsen, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grande, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia-Arenas, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Godin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keen, N.</span><span> </span><span class="NLM_article-title">Structural basis of substrate methylation and inhibition of SMYD2</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1262</span><span class="NLM_x">–</span> <span class="NLM_lpage">1273</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1016%2Fj.str.2011.06.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=21782458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaru7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=1262-1273&author=A.+D.+Fergusonauthor=N.+A.+Larsenauthor=T.+Howardauthor=H.+Pollardauthor=I.+Greenauthor=C.+Grandeauthor=T.+Cheungauthor=R.+Garcia-Arenasauthor=S.+Cowenauthor=J.+Wuauthor=R.+Godinauthor=H.+Chenauthor=N.+Keen&title=Structural+basis+of+substrate+methylation+and+inhibition+of+SMYD2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of Substrate Methylation and Inhibition of SMYD2</span></div><div class="casAuthors">Ferguson, Andrew D.; Larsen, Nicholas A.; Howard, Tina; Pollard, Hannah; Green, Isabelle; Grande, Christie; Cheung, Tony; Garcia-Arenas, Renee; Cowen, Scott; Wu, Jiaquan; Godin, Robert; Chen, Huawei; Keen, Nicholas</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1262-1273</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases are important regulators of epigenetic signaling.  These enzymes catalyze the transfer of donor Me groups from S-adenosylmethionine to specific acceptor lysines on histones, leading to changes in chromatin structure and transcriptional regulation.  These enzymes also methylate nonhistone protein substrates, revealing an addnl. mechanism to regulate cellular physiol.  The oncogenic protein SMYD2 represses the functional activities of the tumor suppressor proteins p53 and Rb, making it an attractive drug target.  Here we report the discovery of AZ505, a potent and selective inhibitor of SMYD2 that was identified from a high throughput chem. screen.  We also present the crystal structures of SMYD2 with p53 substrate and product peptides, and notably, in complex with AZ505.  This substrate competitive inhibitor is bound in the peptide binding groove of SMYD2.  These results have implications for the development of SMYD2 inhibitors, and indicate the potential for developing novel therapies targeting this target class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfo0RkV17Th7Vg90H21EOLACvtfcHk0lhJciIRAEOVWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaru7fK&md5=31a08f46bffea75609e1c5bb25b7c610</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2011.06.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2011.06.011%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DLarsen%26aufirst%3DN.%2BA.%26aulast%3DHoward%26aufirst%3DT.%26aulast%3DPollard%26aufirst%3DH.%26aulast%3DGreen%26aufirst%3DI.%26aulast%3DGrande%26aufirst%3DC.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DGarcia-Arenas%26aufirst%3DR.%26aulast%3DCowen%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DGodin%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DKeen%26aufirst%3DN.%26atitle%3DStructural%2520basis%2520of%2520substrate%2520methylation%2520and%2520inhibition%2520of%2520SMYD2%26jtitle%3DStructure%26date%3D2011%26volume%3D19%26spage%3D1262%26epage%3D1273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anglin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, B. V. V</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Selective inhibitors of histone methyltransferase DOT1L: design, synthesis and crystallographic studies</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">16746</span><span class="NLM_x">–</span> <span class="NLM_lpage">16749</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja206312b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=16746-16749&author=Y.+Yaoauthor=P.+Chenauthor=J.+Diaoauthor=G.+Chengauthor=L.+Dengauthor=J.+L.+Anglinauthor=B.+V.+V+Prasadauthor=Y.+Song&title=Selective+inhibitors+of+histone+methyltransferase+DOT1L%3A+design%2C+synthesis+and+crystallographic+studies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fja206312b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja206312b%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DDiao%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DG.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DAnglin%26aufirst%3DJ.%2BL.%26aulast%3DPrasad%26aufirst%3DB.%2BV.%2BV%26aulast%3DSong%26aufirst%3DY.%26atitle%3DSelective%2520inhibitors%2520of%2520histone%2520methyltransferase%2520DOT1L%253A%2520design%252C%2520synthesis%2520and%2520crystallographic%2520studies%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D16746%26epage%3D16749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Yuan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paulk, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kemp, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shamji, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span> </span><span class="NLM_article-title">A small-molecule probe of the histone methyltransferase G9a induces cellular senescence in pancreatic adenocarcinoma</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1152</span><span class="NLM_x">–</span> <span class="NLM_lpage">1157</span></span><div class="citationLinks">[<a href="/doi/10.1021/cb300139y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1152-1157&author=Y.+Yuanauthor=Q.+Wangauthor=J.+Paulkauthor=S.+Kubicekauthor=M.+M.+Kempauthor=D.+J.+Adamsauthor=A.+F.+Shamjiauthor=B.+K.+Wagnerauthor=S.+L.+Schreiber&title=A+small-molecule+probe+of+the+histone+methyltransferase+G9a+induces+cellular+senescence+in+pancreatic+adenocarcinoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fcb300139y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300139y%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DKemp%26aufirst%3DM.%2BM.%26aulast%3DAdams%26aufirst%3DD.%2BJ.%26aulast%3DShamji%26aufirst%3DA.%2BF.%26aulast%3DWagner%26aufirst%3DB.%2BK.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DA%2520small-molecule%2520probe%2520of%2520the%2520histone%2520methyltransferase%2520G9a%2520induces%2520cellular%2520senescence%2520in%2520pancreatic%2520adenocarcinoma%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1152%26epage%3D1157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganji, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korenchuk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Aller, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iii, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diaz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lafrance, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mellinger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duquenne, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHugh, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brandt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verma, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tummino, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creasy, C. L.</span><span> </span><span class="NLM_article-title">EZH2 inhibition as a therapeutic strategy for lymphoma with EZH2-activating mutations</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">492</span><span class="NLM_x">, </span> <span class="NLM_fpage">108</span><span class="NLM_x">–</span> <span class="NLM_lpage">112</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=108-112&author=M.+T.+McCabeauthor=H.+M.+Ottauthor=G.+Ganjiauthor=S.+Korenchukauthor=C.+Thompsonauthor=G.+S.+Van+Allerauthor=Y.+Liuauthor=A.+P.+Gravesauthor=A.+D.+Iiiauthor=E.+Diazauthor=L.+V.+Lafranceauthor=M.+Mellingerauthor=C.+Duquenneauthor=X.+Tianauthor=R.+G.+Krugerauthor=C.+F.+McHughauthor=M.+Brandtauthor=W.+H.+Millerauthor=D.+Dhanakauthor=S.+K.+Vermaauthor=P.+J.+Tumminoauthor=C.+L.+Creasy&title=EZH2+inhibition+as+a+therapeutic+strategy+for+lymphoma+with+EZH2-activating+mutations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DOtt%26aufirst%3DH.%2BM.%26aulast%3DGanji%26aufirst%3DG.%26aulast%3DKorenchuk%26aufirst%3DS.%26aulast%3DThompson%26aufirst%3DC.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DIii%26aufirst%3DA.%2BD.%26aulast%3DDiaz%26aufirst%3DE.%26aulast%3DLafrance%26aufirst%3DL.%2BV.%26aulast%3DMellinger%26aufirst%3DM.%26aulast%3DDuquenne%26aufirst%3DC.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DBrandt%26aufirst%3DM.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DVerma%26aufirst%3DS.%2BK.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26atitle%3DEZH2%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520for%2520lymphoma%2520with%2520EZH2-activating%2520mutations%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D108%26epage%3D112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Knutson, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warholic, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sneeringer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sacks, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keilhack, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span> </span><span class="NLM_article-title">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">890</span><span class="NLM_x">–</span> <span class="NLM_lpage">896</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1038%2Fnchembio.1084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=23023262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=890-896&author=S.+K.+Knutsonauthor=T.+J.+Wigleauthor=N.+M.+Warholicauthor=C.+J.+Sneeringerauthor=C.+J.+Allainauthor=C.+R.+Klausauthor=J.+D.+Sacksauthor=A.+Raimondiauthor=C.+R.+Majerauthor=J.+Songauthor=M.+P.+Scottauthor=L.+Jinauthor=J.+J.+Smithauthor=E.+J.+Olhavaauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=V.+M.+Richonauthor=R.+A.+Copelandauthor=H.+Keilhackauthor=R.+M.+Pollockauthor=K.+W.+Kuntz&title=A+selective+inhibitor+of+EZH2+blocks+H3K27+methylation+and+kills+mutant+lymphoma+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span></div><div class="casAuthors">Knutson, Sarah K.; Wigle, Tim J.; Warholic, Natalie M.; Sneeringer, Christopher J.; Allain, Christina J.; Klaus, Christine R.; Sacks, Joelle D.; Raimondi, Alejandra; Majer, Christina R.; Song, Jeffrey; Scott, Margaret Porter; Jin, Lei; Smith, Jesse J.; Olhava, Edward J.; Chesworth, Richard; Moyer, Mikel P.; Richon, Victoria M.; Copeland, Robert A.; Keilhack, Heike; Pollock, Roy M.; Kuntz, Kevin W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">890-896</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">EZH2 catalyzes trimethylation of histone H3 lysine 27 (H3K27).  Point mutations of EZH2 at Tyr 641 and Ala 677 occur in subpopulations of non-Hodgkin's lymphoma, where they drive H3K27 hypertrimethylation.  Here the authors report the discovery of EPZ005687, a potent inhibitor of EZH2 (Ki of 24 nM).  EPZ005687 has greater than 500-fold selectivity against 15 other protein methyltransferases and has 50-fold selectivity against the closely related enzyme EZH1.  The compd. reduces H3K27 methylation in various lymphoma cells; this translates into apoptotic cell killing in heterozygous Tyr 641 or Ala 677 mutant cells, with minimal effects on the proliferation of wild-type cells.  These data suggest that genetic alteration of EZH2 (for example, mutations at Tyr 641 or Ala 677) results in a crit. dependency on enzymic activity for proliferation (i.e., the equiv. of oncogene addiction), thus portending the clin. use of EZH2 inhibitors for cancers in which EZH2 is genetically altered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohlduVbTgWJrVg90H21EOLACvtfcHk0lgnTl4tVhG4qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP&md5=a9ad7a41505e3fd4aa087d9f5dae8825</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1084%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DSacks%26aufirst%3DJ.%2BD.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DKeilhack%26aufirst%3DH.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520EZH2%2520blocks%2520H3K27%2520methylation%2520and%2520kills%2520mutant%2520lymphoma%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D890%26epage%3D896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Verma, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaFrance, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duquenne, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suarez, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newlander, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romeril, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgess, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brackley, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scherzer, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aller, G. S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machutta, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diaz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tummino, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creasy, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. H.</span><span> </span><span class="NLM_article-title">Identification of potent, selective, cell-active inhibitors of the histone lysine methyltransferase EZH2</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1091</span><span class="NLM_x">–</span> <span class="NLM_lpage">1096</span></span><div class="citationLinks">[<a href="/doi/10.1021/ml3003346" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFChs77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=1091-1096&author=S.+K.+Vermaauthor=X.+Tianauthor=L.+V.+LaFranceauthor=C.+Duquenneauthor=D.+P.+Suarezauthor=K.+A.+Newlanderauthor=S.+P.+Romerilauthor=J.+L.+Burgessauthor=S.+W.+Grantauthor=J.+A.+Brackleyauthor=A.+P.+Gravesauthor=D.+A.+Scherzerauthor=A.+Shuauthor=C.+Thompsonauthor=H.+M.+Ottauthor=G.+S.+V.+Allerauthor=C.+A.+Machuttaauthor=E.+Diazauthor=Y.+Jiangauthor=N.+W.+Johnsonauthor=S.+D.+Knightauthor=R.+G.+Krugerauthor=M.+T.+McCabeauthor=D.+Dhanakauthor=P.+J.+Tumminoauthor=C.+L.+Creasyauthor=W.+H.+Miller&title=Identification+of+potent%2C+selective%2C+cell-active+inhibitors+of+the+histone+lysine+methyltransferase+EZH2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Potent, Selective, Cell-Active Inhibitors of the Histone Lysine Methyltransferase EZH2</span></div><div class="casAuthors">Verma, Sharad K.; Tian, Xinrong; LaFrance, Louis V.; Duquenne, Celine; Suarez, Dominic P.; Newlander, Kenneth A.; Romeril, Stuart P.; Burgess, Joelle L.; Grant, Seth W.; Brackley, James A.; Graves, Alan P.; Scherzer, Daryl A.; Shu, Art; Thompson, Christine; Ott, Heidi M.; Van Aller, Glenn S.; Machutta, Carl A.; Diaz, Elsie; Jiang, Yong; Johnson, Neil W.; Knight, Steven D.; Kruger, Ryan G.; McCabe, Michael T.; Dhanak, Dashyant; Tummino, Peter J.; Creasy, Caretha L.; Miller, William H.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1091-1096</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The histone H3-lysine 27 (H3K27) methyltransferase EZH2 plays a crit. role in regulating gene expression, and its aberrant activity is linked to the onset and progression of cancer.  As part of a drug discovery program targeting EZH2, we have identified highly potent, selective, S-(S'-adenosyl)-L-methionine (SAM)-competitive, and cell-active EZH2 inhibitors, including GSK926 I (R1 = Me, R2 = Q) and GSK343 I (R1 = n-Pr, R2 = Q1).  These compds. are small mol. chem. tools that would be useful to further explore the biol. of EZH2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8NUzf0zjo0bVg90H21EOLACvtfcHk0lhg8363ljbQbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFChs77K&md5=a6540bb712ee52cba05bd0b074d8b26a</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fml3003346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml3003346%26sid%3Dliteratum%253Aachs%26aulast%3DVerma%26aufirst%3DS.%2BK.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DLaFrance%26aufirst%3DL.%2BV.%26aulast%3DDuquenne%26aufirst%3DC.%26aulast%3DSuarez%26aufirst%3DD.%2BP.%26aulast%3DNewlander%26aufirst%3DK.%2BA.%26aulast%3DRomeril%26aufirst%3DS.%2BP.%26aulast%3DBurgess%26aufirst%3DJ.%2BL.%26aulast%3DGrant%26aufirst%3DS.%2BW.%26aulast%3DBrackley%26aufirst%3DJ.%2BA.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DScherzer%26aufirst%3DD.%2BA.%26aulast%3DShu%26aufirst%3DA.%26aulast%3DThompson%26aufirst%3DC.%26aulast%3DOtt%26aufirst%3DH.%2BM.%26aulast%3DAller%26aufirst%3DG.%2BS.%2BV.%26aulast%3DMachutta%26aufirst%3DC.%2BA.%26aulast%3DDiaz%26aufirst%3DE.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DJohnson%26aufirst%3DN.%2BW.%26aulast%3DKnight%26aufirst%3DS.%2BD.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26atitle%3DIdentification%2520of%2520potent%252C%2520selective%252C%2520cell-active%2520inhibitors%2520of%2520the%2520histone%2520lysine%2520methyltransferase%2520EZH2%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D1091%26epage%3D1096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Zheng, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibáñez, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blum, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amaya, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, M.</span><span> </span><span class="NLM_article-title">Sinefungin derivatives as inhibitors and structure probes of protein lysine methyltransferase SETD2</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">134</span><span class="NLM_x">, </span> <span class="NLM_fpage">18004</span><span class="NLM_x">–</span> <span class="NLM_lpage">18014</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja307060p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVOhsrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=18004-18014&author=W.+Zhengauthor=G.+Ib%C3%A1%C3%B1ezauthor=H.+Wuauthor=G.+Blumauthor=H.+Zengauthor=A.+Dongauthor=F.+Liauthor=T.+Hajianauthor=A.+Allali-Hassaniauthor=M.+F.+Amayaauthor=A.+Siarheyevaauthor=W.+Yuauthor=P.+J.+Brownauthor=M.+Schapiraauthor=M.+Vedadiauthor=J.+Minauthor=M.+Luo&title=Sinefungin+derivatives+as+inhibitors+and+structure+probes+of+protein+lysine+methyltransferase+SETD2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Sinefungin Derivatives as Inhibitors and Structure Probes of Protein Lysine Methyltransferase SETD2</span></div><div class="casAuthors">Zheng, Weihong; Ibanez, Glorymar; Wu, Hong; Blum, Gil; Zeng, Hong; Dong, Aiping; Li, Fengling; Hajian, Taraneh; Allali-Hassani, Abdellah; Amaya, Maria F.; Siarheyeva, Alena; Yu, Wenyu; Brown, Peter J.; Schapira, Matthieu; Vedadi, Masoud; Min, Jinrong; Luo, Minkui</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">18004-18014</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Epigenetic regulation is involved in numerous physiol. and pathogenic processes.  Among the key regulators that orchestrate epigenetic signaling are over 50 human protein lysine methyltransferases (PKMTs).  Interrogation of the functions of individual PKMTs can be facilitated by target-specific PKMT inhibitors.  Given the emerging need for such small mols., an approach was envisioned to identify target-specific methyltransferase inhibitors by screening privileged small-mol. scaffolds against diverse methyltransferases.  In this work, the feasibility of such an approach was demonstrated by identifying the inhibitors of SETD2.  N-Pr sinefungin (Pr-SNF) was shown to interact preferentially with SETD2 by matching the distinct transition-state features of SETD2's catalytically active conformer.  With Pr-SNF as a structure probe, the dual roles of SETD2's post-SET loop in regulating substrate access through a distinct topol. reconfiguration was revealed.  Privileged sinefungin scaffolds are expected to have broad use as structure and chem. probes of methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv-sWI1BJjg7Vg90H21EOLACvtfcHk0lhg8363ljbQbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVOhsrbL&md5=30f5ba653007349df64e095f1c2cf188</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fja307060p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja307060p%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DIb%25C3%25A1%25C3%25B1ez%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DBlum%26aufirst%3DG.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DAmaya%26aufirst%3DM.%2BF.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DM.%26atitle%3DSinefungin%2520derivatives%2520as%2520inhibitors%2520and%2520structure%2520probes%2520of%2520protein%2520lysine%2520methyltransferase%2520SETD2%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D18004%26epage%3D18014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobrovetsky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolshan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loppnau, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, Peter J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">An allosteric inhibitor of protein arginine methyltransferase 3</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1425</span><span class="NLM_x">–</span> <span class="NLM_lpage">1435</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1016%2Fj.str.2012.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=22795084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVektr3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=1425-1435&author=A.+Siarheyevaauthor=G.+Senisterraauthor=A.+Allali-Hassaniauthor=A.+Dongauthor=E.+Dobrovetskyauthor=G.+A.+Wasneyauthor=I.+Chauauthor=R.+Marcellusauthor=T.+Hajianauthor=F.+Liuauthor=I.+Korboukhauthor=D.+Smilauthor=Y.+Bolshanauthor=J.+Minauthor=H.+Wuauthor=H.+Zengauthor=P.+Loppnauauthor=G.+Podaauthor=C.+Griffinauthor=A.+Amanauthor=Peter+J.+Brownauthor=J.+Jinauthor=R.+Al-awarauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadi&title=An+allosteric+inhibitor+of+protein+arginine+methyltransferase+3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">An Allosteric Inhibitor of Protein Arginine Methyltransferase 3</span></div><div class="casAuthors">Siarheyeva, Alena; Senisterra, Guillermo; Allali-Hassani, Abdellah; Dong, Aiping; Dobrovetsky, Elena; Wasney, Gregory A.; Chau, Irene; Marcellus, Richard; Hajian, Taraneh; Liu, Feng; Korboukh, Ilia; Smil, David; Bolshan, Yuri; Min, Jinrong; Wu, Hong; Zeng, Hong; Loppnau, Peter; Poda, Gennadiy; Griffin, Carly; Aman, Ahmed; Brown, Peter J.; Jin, Jian; Al-awar, Rima; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1425-1435</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">PRMT3, a protein arginine methyltransferase, has been shown to influence ribosomal biosynthesis by catalyzing the dimethylation of the 40S ribosomal protein S2.  Although PRMT3 has been reported to be a cytosolic protein, it has been shown to methylate histone H4 peptide (H4 1-24) in vitro.  Here, we report the identification of a PRMT3 inhibitor (1-(benzo[d][1,2,3]thiadiazol-6-yl)-3-(2-cyclohexenylethyl)urea; compd. 1) with IC50 value of 2.5 μM by screening a library of 16,000 compds. using H4 (1-24) peptide as a substrate.  The crystal structure of PRMT3 in complex with compd. 1 as well as kinetic anal. reveals an allosteric mechanism of inhibition.  Mutating PRMT3 residues within the allosteric site or using compd. 1 analogs that disrupt interactions with allosteric site residues both abrogated binding and inhibitory activity.  These data demonstrate an allosteric mechanism for inhibition of protein arginine methyltransferases, an emerging class of therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruLurDJu10tLVg90H21EOLACvtfcHk0lgl4EHSEByQTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVektr3N&md5=c745acac0beb494318f4577293281ddf</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2012.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2012.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBolshan%26aufirst%3DY.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DLoppnau%26aufirst%3DP.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DGriffin%26aufirst%3DC.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DPeter%2BJ.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DAl-awar%26aufirst%3DR.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DAn%2520allosteric%2520inhibitor%2520of%2520protein%2520arginine%2520methyltransferase%25203%26jtitle%3DStructure%26date%3D2012%26volume%3D20%26spage%3D1425%26epage%3D1435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Ward, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poesche, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgins, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heard, D.</span><span> </span><span class="NLM_article-title">458. 1, 2, 3-Benzothiadiazole. Part I. Nitro-, amino-, and hydroxy-derivatives</span> <span class="citation_source-journal">J. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1962</span><span class="NLM_x">, </span> <span class="NLM_fpage">2374</span><span class="NLM_x">–</span> <span class="NLM_lpage">2379</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1962&pages=2374-2379&author=E.+Wardauthor=W.+Poescheauthor=D.+Higginsauthor=D.+Heard&title=458.+1%2C+2%2C+3-Benzothiadiazole.+Part+I.+Nitro-%2C+amino-%2C+and+hydroxy-derivatives"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DE.%26aulast%3DPoesche%26aufirst%3DW.%26aulast%3DHiggins%26aufirst%3DD.%26aulast%3DHeard%26aufirst%3DD.%26atitle%3D458.%25201%252C%25202%252C%25203-Benzothiadiazole.%2520Part%2520I.%2520Nitro-%252C%2520amino-%252C%2520and%2520hydroxy-derivatives%26jtitle%3DJ.%2520Chem.%2520Soc.%26date%3D1962%26spage%3D2374%26epage%3D2379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Rostami, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chudzinski, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lough, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, M. S.</span><span> </span><span class="NLM_article-title"><i>N</i>,<i>N</i>′-Diarylsquaramides: general, high-yielding synthesis and applications in colorimetric anion sensing</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">75</span><span class="NLM_x">, </span> <span class="NLM_fpage">3983</span><span class="NLM_x">–</span> <span class="NLM_lpage">3992</span></span><div class="citationLinks">[<a href="/doi/10.1021/jo100104g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2010&pages=3983-3992&author=A.+Rostamiauthor=A.+Colinauthor=X.+Y.+Liauthor=M.+G.+Chudzinskiauthor=A.+J.+Loughauthor=M.+S.+Taylor&title=N%2CN%E2%80%B2-Diarylsquaramides%3A+general%2C+high-yielding+synthesis+and+applications+in+colorimetric+anion+sensing"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fjo100104g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo100104g%26sid%3Dliteratum%253Aachs%26aulast%3DRostami%26aufirst%3DA.%26aulast%3DColin%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DX.%2BY.%26aulast%3DChudzinski%26aufirst%3DM.%2BG.%26aulast%3DLough%26aufirst%3DA.%2BJ.%26aulast%3DTaylor%26aufirst%3DM.%2BS.%26atitle%3DN%252CN%25E2%2580%25B2-Diarylsquaramides%253A%2520general%252C%2520high-yielding%2520synthesis%2520and%2520applications%2520in%2520colorimetric%2520anion%2520sensing%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2010%26volume%3D75%26spage%3D3983%26epage%3D3992" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Minor, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cymborowski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otwinowski, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chruszcz, M.</span><span> </span><span class="NLM_article-title">HKL-3000: the integration of data reduction and structure solution—from diffraction images to an initial model in minutes</span> <span class="citation_source-journal">Acta Crystallogr., D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">859</span><span class="NLM_x">–</span> <span class="NLM_lpage">866</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1107%2FS0907444906019949" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=16855301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=859-866&author=W.+Minorauthor=M.+Cymborowskiauthor=Z.+Otwinowskiauthor=M.+Chruszcz&title=HKL-3000%3A+the+integration+of+data+reduction+and+structure+solution%E2%80%94from+diffraction+images+to+an+initial+model+in+minutes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">HKL-3000: the integration of data reduction and structure solution - from diffraction images to an initial model in minutes</span></div><div class="casAuthors">Minor, Wladek; Cymborowski, Marcin; Otwinowski, Zbyszek; Chruszcz, Maksymilian</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">859-866</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A new approach that integrates data collection, data redn., phasing and model building significantly accelerates the process of structure detn. and on av. minimizes the no. of data sets and synchrotron time required for structure soln.  Initial testing of the HKL-3000 system (the beta version was named HKL-2000_ph) with more than 140 novel structure detns. has proven its high value for MAD/SAD expts.  The heuristics for choosing the best computational strategy at different data resoln. limits of phasing signal and crystal diffraction are being optimized.  The typical end result is an interpretable electron-d. map with a partially built structure and, in some cases, an almost complete refined model.  The current development is oriented towards very fast structure soln. in order to provide feedback during the diffraction expt.  Work is also proceeding towards improving the quality of phasing calcn. and model building.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0c0DYtjFP7rVg90H21EOLACvtfcHk0lgs8guia-iBvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D&md5=2cd2fe288f954837abb4acae4c008a2f</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1107%2FS0907444906019949&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906019949%26sid%3Dliteratum%253Aachs%26aulast%3DMinor%26aufirst%3DW.%26aulast%3DCymborowski%26aufirst%3DM.%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DChruszcz%26aufirst%3DM.%26atitle%3DHKL-3000%253A%2520the%2520integration%2520of%2520data%2520reduction%2520and%2520structure%2520solution%25E2%2580%2594from%2520diffraction%2520images%2520to%2520an%2520initial%2520model%2520in%2520minutes%26jtitle%3DActa%2520Crystallogr.%252C%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D859%26epage%3D866" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teplyakov, A.</span><span> </span><span class="NLM_article-title">MOLREP: an automated program for molecular replacement</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1022</span><span class="NLM_x">–</span> <span class="NLM_lpage">1025</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+an+automated+program+for+molecular+replacement"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0lgs8guia-iBvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520an%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lgeM5ydCCWI2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lgeM5ydCCWI2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arendall, W. B.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keedy, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Immormino, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span> <span class="citation_source-journal">Acta Crystallogr., D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">12</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0lgeM5ydCCWI2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Abagyan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Totrov, M.</span><span> </span><span class="NLM_article-title">Biased probability Monte Carlo conformational searches and electrostatic calculations for peptides and proteins</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">235</span><span class="NLM_x">, </span> <span class="NLM_fpage">983</span><span class="NLM_x">–</span> <span class="NLM_lpage">1002</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1006%2Fjmbi.1994.1052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=8289329" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADyaK2cXhsFyis7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=235&publication_year=1994&pages=983-1002&author=R.+Abagyanauthor=M.+Totrov&title=Biased+probability+Monte+Carlo+conformational+searches+and+electrostatic+calculations+for+peptides+and+proteins"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Biased probability Monte Carlo conformational searches and electrostatic calculations for peptides and proteins</span></div><div class="casAuthors">Abagyan, Ruben; Totrov, Maxim</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">235</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">983-1002</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">Two major components are required for a successful prediction of the three-dimensional structure of peptides and proteins: an efficient global optimization procedure which is capable of finding an appropriate local min. for the strongly anisotropic function of hundreds of variables and a set of free energy components for a protein mol. in soln. which are computationally inexpensive enough to be used in the search procedure, yet sufficiently accurate to ensure the uniqueness of the native conformation.  The authors here found an efficient way to make a random step in a Monte Carlo procedure given knowledge of the energy or statistical properties of conformational subspaces (e.g., φ-ψ zones or side-chain torsion angles).  This biased probability Monte Carlo (BPMC) procedure randomly selects the subspace first, then makes a step to a new random position independent of the previous position, but according to the predefined continuous probability distribution.  The random step is followed by a local minimization in torsion angle space.  The positions, sizes and preferences for high-probability zones on φ-ψ maps and χ-angle maps were calcd. for different residue types from the representative set of 191 and 161 protein 3D-structures, resp.  A fast and precise method to evaluate the electrostatic energy of a protein in soln. is developed and combined with the BPMC procedure.  The method is based on the modified spherical image charge approxn., efficiently projected onto a mol. of arbitrary shape.  Comparison with the finite-difference solns. of the Poisson-Boltzmann equation shows high accuracy for the authors' approach.  The BPMC procedure is applied successfully to the structure prediction of 12- and 16-residue synthetic peptides and the detn. of protein structure from NMR data, with the Ig binding domain of streptococcal protein G as an example.  The BPMC runs display much better convergence properties than the non-biased simulations.  The advantage of a true global optimization procedure for NMR structure detn. is its ability to cope with local min. originating from data errors and ambiguities in NMR data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZIMjvYsp0UbVg90H21EOLACvtfcHk0liX8kGOPcZHOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXhsFyis7k%253D&md5=d88445e688ef9c522fb87e4ca101f778</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1994.1052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1994.1052%26sid%3Dliteratum%253Aachs%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DTotrov%26aufirst%3DM.%26atitle%3DBiased%2520probability%2520Monte%2520Carlo%2520conformational%2520searches%2520and%2520electrostatic%2520calculations%2520for%2520peptides%2520and%2520proteins%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1994%26volume%3D235%26spage%3D983%26epage%3D1002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref37':['cit37'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref36':['cit36'],'ref47':['cit47'],'ref38':['cit38'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 55 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Yudao Shen, Fengling Li, Magdalena M. Szewczyk, Levon Halabelian, Irene Chau, Mohammad S. Eram, Carlo Dela Seña, Kwang-Su Park, Fanye Meng, He Chen, Hong Zeng, Aiping Dong, Hong Wu, Viacheslav V. Trush, David McLeod, Carlos A. Zepeda-Velázquez, Robert M. Campbell, Mary M. Mader, Brian M. Watson, Matthieu Schapira, Cheryl H. Arrowsmith, Rima Al-Awar, Dalia Barsyte-Lovejoy, H. Ümit Kaniskan, Peter J. Brown, Masoud Vedadi, <span class="NLM_string-name hlFld-ContribAuthor">Jian Jin</span>. </span><span class="cited-content_cbyCitation_article-title">A First-in-Class, Highly Selective and Cell-Active Allosteric Inhibitor of Protein Arginine Methyltransferase 6. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (7)
                                     , 3697-3706. <a href="https://doi.org/10.1021/acs.jmedchem.0c02160" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02160</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02160&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02160%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DA%252BFirst-in-Class%25252C%252BHighly%252BSelective%252Band%252BCell-Active%252BAllosteric%252BInhibitor%252Bof%252BProtein%252BArginine%252BMethyltransferase%252B6%26aulast%3DShen%26aufirst%3DYudao%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D14122020%26date%3D16022021%26volume%3D64%26issue%3D7%26spage%3D3697%26epage%3D3706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Rachel L. Palte, Sebastian E. Schneider, Michael D. Altman, Robert P. Hayes, Shuhei Kawamura, Brian M. Lacey, My Sam Mansueto, Michael Reutershan, Phieng Siliphaivanh, Christopher Sondey, Haiyan Xu, Zangwei Xu, Yingchun Ye, <span class="NLM_string-name hlFld-ContribAuthor">Michelle R. Machacek</span>. </span><span class="cited-content_cbyCitation_article-title">Allosteric Modulation of Protein Arginine Methyltransferase 5 (PRMT5). </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (9)
                                     , 1688-1693. <a href="https://doi.org/10.1021/acsmedchemlett.9b00525" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00525</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00525%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DAllosteric%252BModulation%252Bof%252BProtein%252BArginine%252BMethyltransferase%252B5%252B%252528PRMT5%252529%26aulast%3DPalte%26aufirst%3DRachel%2BL.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D14112019%26date%3D30072020%26date%3D07082020%26volume%3D11%26issue%3D9%26spage%3D1688%26epage%3D1693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yudao Shen, Fengling Li, Magdalena M. Szewczyk, Levon Halabelian, Kwang-su Park, Irene Chau, Aiping Dong, Hong Zeng, He Chen, Fanye Meng, Dalia Barsyte-Lovejoy, Cheryl H. Arrowsmith, Peter J. Brown, Jing Liu, Masoud Vedadi, <span class="NLM_string-name hlFld-ContribAuthor">Jian Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a First-in-Class Protein Arginine Methyltransferase 6 (PRMT6) Covalent Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (10)
                                     , 5477-5487. <a href="https://doi.org/10.1021/acs.jmedchem.0c00406" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00406</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00406%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BFirst-in-Class%252BProtein%252BArginine%252BMethyltransferase%252B6%252B%252528PRMT6%252529%252BCovalent%252BInhibitor%26aulast%3DShen%26aufirst%3DYudao%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D09032020%26date%3D14052020%26date%3D05052020%26volume%3D63%26issue%3D10%26spage%3D5477%26epage%3D5487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Michael L.  Martini</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Inhibitors of Protein Methyltransferases and Demethylases. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2018,</strong> <em>118 </em>
                                    (3)
                                     , 989-1068. <a href="https://doi.org/10.1021/acs.chemrev.6b00801" title="DOI URL">https://doi.org/10.1021/acs.chemrev.6b00801</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DInhibitors%252Bof%252BProtein%252BMethyltransferases%252Band%252BDemethylases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2018%26date%3D2017%26date%3D2016%26date%3D30112016%26date%3D24032017%26date%3D14022018%26volume%3D118%26issue%3D3%26spage%3D989%26epage%3D1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Mohammad S.  Eram</span>, <span class="hlFld-ContribAuthor ">Kehao  Zhao</span>, <span class="hlFld-ContribAuthor ">Magdalena M.  Szewczyk</span>, <span class="hlFld-ContribAuthor ">Xiaobao  Yang</span>, <span class="hlFld-ContribAuthor ">Keith  Schmidt</span>, <span class="hlFld-ContribAuthor ">Xiao  Luo</span>, <span class="hlFld-ContribAuthor ">Sean  Xiao</span>, <span class="hlFld-ContribAuthor ">Miao  Dai</span>, <span class="hlFld-ContribAuthor ">Feng  He</span>, <span class="hlFld-ContribAuthor ">Irene  Zang</span>, <span class="hlFld-ContribAuthor ">Ying  Lin</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Elena  Dobrovetsky</span>, <span class="hlFld-ContribAuthor ">David  Smil</span>, <span class="hlFld-ContribAuthor ">Sun-Joon  Min</span>, <span class="hlFld-ContribAuthor ">Jennifer  Lin-Jones</span>, <span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>, <span class="hlFld-ContribAuthor ">Peter  Atadja</span>, <span class="hlFld-ContribAuthor ">En  Li</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Feng  Liu</span>, <span class="hlFld-ContribAuthor ">Zhengtian  Yu</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of Potent and Selective Allosteric Inhibitors of Protein Arginine Methyltransferase 3 (PRMT3). </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (3)
                                     , 1204-1217. <a href="https://doi.org/10.1021/acs.jmedchem.7b01674" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01674</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01674%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BPotent%252Band%252BSelective%252BAllosteric%252BInhibitors%252Bof%252BProtein%252BArginine%252BMethyltransferase%252B3%252B%252528PRMT3%252529%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D13112017%26date%3D05012018%26date%3D08022018%26date%3D15122017%26volume%3D61%26issue%3D3%26spage%3D1204%26epage%3D1217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yudao  Shen</span>, <span class="hlFld-ContribAuthor ">Magdalena M.  Szewczyk</span>, <span class="hlFld-ContribAuthor ">Mohammad S.  Eram</span>, <span class="hlFld-ContribAuthor ">David  Smil</span>, <span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Renato  Ferreira de Freitas</span>, <span class="hlFld-ContribAuthor ">Guillermo  Senisterra</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>, <span class="hlFld-ContribAuthor ">Jing  Liu</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of a Potent, Selective, and Cell-Active Dual Inhibitor of Protein Arginine Methyltransferase 4 and Protein Arginine Methyltransferase 6. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (19)
                                     , 9124-9139. <a href="https://doi.org/10.1021/acs.jmedchem.6b01033" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01033</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01033%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BPotent%25252C%252BSelective%25252C%252Band%252BCell-Active%252BDual%252BInhibitor%252Bof%252BProtein%252BArginine%252BMethyltransferase%252B4%252Band%252BProtein%252BArginine%252BMethyltransferase%252B6%26aulast%3DShen%26aufirst%3DYudao%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D12072016%26date%3D15092016%26date%3D13102016%26date%3D01092016%26volume%3D59%26issue%3D19%26spage%3D9124%26epage%3D9139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">P. Ann  Boriack-Sjodin</span> and <span class="hlFld-ContribAuthor ">Kerren K.  Swinger</span>  . </span><span class="cited-content_cbyCitation_article-title">Protein Methyltransferases: A Distinct, Diverse, and Dynamic Family of Enzymes. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2016,</strong> <em>55 </em>
                                    (11)
                                     , 1557-1569. <a href="https://doi.org/10.1021/acs.biochem.5b01129" title="DOI URL">https://doi.org/10.1021/acs.biochem.5b01129</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.5b01129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.5b01129%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DProtein%252BMethyltransferases%25253A%252BA%252BDistinct%25252C%252BDiverse%25252C%252Band%252BDynamic%252BFamily%252Bof%252BEnzymes%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BAnn%26date%3D2016%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D16102015%26date%3D09122015%26date%3D22122015%26date%3D22032016%26date%3D10122015%26volume%3D55%26issue%3D11%26spage%3D1557%26epage%3D1569" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mohammad S.  Eram</span>, <span class="hlFld-ContribAuthor ">Yudao  Shen</span>, <span class="hlFld-ContribAuthor ">Magdalena M.  Szewczyk</span>, <span class="hlFld-ContribAuthor ">Hong  Wu</span>, <span class="hlFld-ContribAuthor ">Guillermo  Senisterra</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Kyle V.  Butler</span>, <span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Brandon A.  Speed</span>, <span class="hlFld-ContribAuthor ">Carlo  dela Seña</span>, <span class="hlFld-ContribAuthor ">Aiping  Dong</span>, <span class="hlFld-ContribAuthor ">Hong  Zeng</span>, <span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>, <span class="hlFld-ContribAuthor ">Jing  Liu</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">A Potent, Selective, and Cell-Active Inhibitor of Human Type I Protein Arginine Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2016,</strong> <em>11 </em>
                                    (3)
                                     , 772-781. <a href="https://doi.org/10.1021/acschembio.5b00839" title="DOI URL">https://doi.org/10.1021/acschembio.5b00839</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.5b00839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.5b00839%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DA%252BPotent%25252C%252BSelective%25252C%252Band%252BCell-Active%252BInhibitor%252Bof%252BHuman%252BType%252BI%252BProtein%252BArginine%252BMethyltransferases%26aulast%3DEram%26aufirst%3DMohammad%2BS.%26date%3D2016%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D15102015%26date%3D24112015%26date%3D08122015%26date%3D18032016%26date%3D24112015%26volume%3D11%26issue%3D3%26spage%3D772%26epage%3D781" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ran  Zhou</span>, <span class="hlFld-ContribAuthor ">Yiqian  Xie</span>, <span class="hlFld-ContribAuthor ">Hao  Hu</span>, <span class="hlFld-ContribAuthor ">Guang  Hu</span>, <span class="hlFld-ContribAuthor ">Viral Sanjay  Patel</span>, <span class="hlFld-ContribAuthor ">Jin  Zhang</span>, <span class="hlFld-ContribAuthor ">Kunqian  Yu</span>, <span class="hlFld-ContribAuthor ">Yiran  Huang</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Zhongjie  Liang</span>, <span class="hlFld-ContribAuthor ">Yujun George  Zheng</span>, and <span class="hlFld-ContribAuthor ">Cheng  Luo</span>  . </span><span class="cited-content_cbyCitation_article-title">Molecular Mechanism underlying PRMT1 Dimerization for SAM Binding and Methylase Activity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2015,</strong> <em>55 </em>
                                    (12)
                                     , 2623-2632. <a href="https://doi.org/10.1021/acs.jcim.5b00454" title="DOI URL">https://doi.org/10.1021/acs.jcim.5b00454</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.5b00454&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.5b00454%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMolecular%252BMechanism%252Bunderlying%252BPRMT1%252BDimerization%252Bfor%252BSAM%252BBinding%252Band%252BMethylase%252BActivity%26aulast%3DZhou%26aufirst%3DRan%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D20072015%26date%3D20112015%26date%3D28122015%26date%3D12112015%26volume%3D55%26issue%3D12%26spage%3D2623%26epage%3D2632" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fanwang  Meng</span>, <span class="hlFld-ContribAuthor ">Sufang  Cheng</span>, <span class="hlFld-ContribAuthor ">Hong  Ding</span>, <span class="hlFld-ContribAuthor ">Shien  Liu</span>, <span class="hlFld-ContribAuthor ">Yan  Liu</span>, <span class="hlFld-ContribAuthor ">Kongkai  Zhu</span>, <span class="hlFld-ContribAuthor ">Shijie  Chen</span>, <span class="hlFld-ContribAuthor ">Junyan  Lu</span>, <span class="hlFld-ContribAuthor ">Yiqian  Xie</span>, <span class="hlFld-ContribAuthor ">Linjuan  Li</span>, <span class="hlFld-ContribAuthor ">Rongfeng  Liu</span>, <span class="hlFld-ContribAuthor ">Zhe  Shi</span>, <span class="hlFld-ContribAuthor ">Yu  Zhou</span>, <span class="hlFld-ContribAuthor ">Yu-Chih  Liu</span>, <span class="hlFld-ContribAuthor ">Mingyue  Zheng</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Wencong  Lu</span>, <span class="hlFld-ContribAuthor ">Hong  Liu</span>, and <span class="hlFld-ContribAuthor ">Cheng  Luo</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery and Optimization of Novel, Selective Histone Methyltransferase SET7 Inhibitors by Pharmacophore- and Docking-Based Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2015,</strong> <em>58 </em>
                                    (20)
                                     , 8166-8181. <a href="https://doi.org/10.1021/acs.jmedchem.5b01154" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01154</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01154%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252BOptimization%252Bof%252BNovel%25252C%252BSelective%252BHistone%252BMethyltransferase%252BSET7%252BInhibitors%252Bby%252BPharmacophore-%252Band%252BDocking-Based%252BVirtual%252BScreening%26aulast%3DMeng%26aufirst%3DFanwang%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D06072015%26date%3D30092015%26date%3D22102015%26date%3D21092015%26volume%3D58%26issue%3D20%26spage%3D8166%26epage%3D8181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Kyle D.  Konze</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Selective Inhibitors of Protein Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2015,</strong> <em>58 </em>
                                    (4)
                                     , 1596-1629. <a href="https://doi.org/10.1021/jm501234a" title="DOI URL">https://doi.org/10.1021/jm501234a</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm501234a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm501234a%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSelective%252BInhibitors%252Bof%252BProtein%252BMethyltransferases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2015%26date%3D2014%26date%3D2014%26date%3D12082014%26date%3D02122014%26date%3D26022015%26date%3D19112014%26volume%3D58%26issue%3D4%26spage%3D1596%26epage%3D1629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anqi  Ma</span>, <span class="hlFld-ContribAuthor ">Wenyu  Yu</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Rachel M.  Bleich</span>, <span class="hlFld-ContribAuthor ">J. Martin  Herold</span>, <span class="hlFld-ContribAuthor ">Kyle V.  Butler</span>, <span class="hlFld-ContribAuthor ">Jacqueline L.  Norris</span>, <span class="hlFld-ContribAuthor ">Victoria  Korboukh</span>, <span class="hlFld-ContribAuthor ">Ashutosh  Tripathy</span>, <span class="hlFld-ContribAuthor ">William P.  Janzen</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of a Selective, Substrate-Competitive Inhibitor of the Lysine Methyltransferase SETD8. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2014,</strong> <em>57 </em>
                                    (15)
                                     , 6822-6833. <a href="https://doi.org/10.1021/jm500871s" title="DOI URL">https://doi.org/10.1021/jm500871s</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm500871s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm500871s%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BSelective%25252C%252BSubstrate-Competitive%252BInhibitor%252Bof%252Bthe%252BLysine%252BMethyltransferase%252BSETD8%26aulast%3DMa%26aufirst%3DAnqi%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D09062014%26date%3D25072014%26date%3D14082014%26date%3D17072014%26volume%3D57%26issue%3D15%26spage%3D6822%26epage%3D6833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Feng  Liu</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Yan  Xiong</span>, <span class="hlFld-ContribAuthor ">Victoria  Korboukh</span>, <span class="hlFld-ContribAuthor ">Xi-Ping  Huang</span>, <span class="hlFld-ContribAuthor ">Abdellah  Allali-Hassani</span>, <span class="hlFld-ContribAuthor ">William P.  Janzen</span>, <span class="hlFld-ContribAuthor ">Bryan L.  Roth</span>, <span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of an in Vivo Chemical Probe of the Lysine Methyltransferases G9a and GLP. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2013,</strong> <em>56 </em>
                                    (21)
                                     , 8931-8942. <a href="https://doi.org/10.1021/jm401480r" title="DOI URL">https://doi.org/10.1021/jm401480r</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm401480r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm401480r%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ban%252Bin%252BVivo%252BChemical%252BProbe%252Bof%252Bthe%252BLysine%252BMethyltransferases%252BG9a%252Band%252BGLP%26aulast%3DLiu%26aufirst%3DFeng%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D24092013%26date%3D31102013%26date%3D14112013%26date%3D08102013%26volume%3D56%26issue%3D21%26spage%3D8931%26epage%3D8942" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kyle D.  Konze</span>, <span class="hlFld-ContribAuthor ">Anqi  Ma</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>, <span class="hlFld-ContribAuthor ">Trevor  Parton</span>, <span class="hlFld-ContribAuthor ">Christopher J.  MacNevin</span>, <span class="hlFld-ContribAuthor ">Feng  Liu</span>, <span class="hlFld-ContribAuthor ">Cen  Gao</span>, <span class="hlFld-ContribAuthor ">Xi-Ping  Huang</span>, <span class="hlFld-ContribAuthor ">Ekaterina  Kuznetsova</span>, <span class="hlFld-ContribAuthor ">Marie  Rougie</span>, <span class="hlFld-ContribAuthor ">Alice  Jiang</span>, <span class="hlFld-ContribAuthor ">Samantha G.  Pattenden</span>, <span class="hlFld-ContribAuthor ">Jacqueline L.  Norris</span>, <span class="hlFld-ContribAuthor ">Lindsey I.  James</span>, <span class="hlFld-ContribAuthor ">Bryan L.  Roth</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Klaus M.  Hahn</span>, <span class="hlFld-ContribAuthor ">Gang Greg  Wang</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">An Orally Bioavailable Chemical Probe of the Lysine Methyltransferases EZH2 and EZH1. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2013,</strong> <em>8 </em>
                                    (6)
                                     , 1324-1334. <a href="https://doi.org/10.1021/cb400133j" title="DOI URL">https://doi.org/10.1021/cb400133j</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/cb400133j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fcb400133j%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DAn%252BOrally%252BBioavailable%252BChemical%252BProbe%252Bof%252Bthe%252BLysine%252BMethyltransferases%252BEZH2%252Band%252BEZH1%26aulast%3DKonze%26aufirst%3DKyle%2BD.%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D22022013%26date%3D08042013%26date%3D24042013%26date%3D21062013%26date%3D08042013%26volume%3D8%26issue%3D6%26spage%3D1324%26epage%3D1334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Amar Pratap  Singh</span>, <span class="hlFld-ContribAuthor ">Rakesh  Kumar</span>, <span class="hlFld-ContribAuthor ">Dinesh  Gupta</span>. </span><span class="cited-content_cbyCitation_article-title">Structural insights into the mechanism of human methyltransferase hPRMT4. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2021,</strong> <em> </em>, 1-14. <a href="https://doi.org/10.1080/07391102.2021.1950567" title="DOI URL">https://doi.org/10.1080/07391102.2021.1950567</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2021.1950567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2021.1950567%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DStructural%252Binsights%252Binto%252Bthe%252Bmechanism%252Bof%252Bhuman%252Bmethyltransferase%252BhPRMT4%26aulast%3DSingh%26aufirst%3DAmar%2BPratap%26date%3D2021%26date%3D2021%26spage%3D1%26epage%3D14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jimmy E.  Jakobsson</span>, <span class="hlFld-ContribAuthor ">Sanjay  Telu</span>, <span class="hlFld-ContribAuthor ">Shuiyu  Lu</span>, <span class="hlFld-ContribAuthor ">Susovan  Jana</span>, <span class="hlFld-ContribAuthor ">Victor W.  Pike</span>. </span><span class="cited-content_cbyCitation_article-title">Broad Scope and High‐Yield Access to Unsymmetrical Acyclic [
              11
              C]Ureas for Biomedical Imaging from [
              11
              C]Carbonyl Difluoride. </span><span class="cited-content_cbyCitation_journal-name">Chemistry – A European Journal</span><span> <strong>2021,</strong> <em>27 </em>
                                    (40)
                                     , 10369-10376. <a href="https://doi.org/10.1002/chem.202100690" title="DOI URL">https://doi.org/10.1002/chem.202100690</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.202100690&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.202100690%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%25E2%2580%2593%2520A%2520European%2520Journal%26atitle%3DBroad%252BScope%252Band%252BHigh%2525E2%252580%252590Yield%252BAccess%252Bto%252BUnsymmetrical%252BAcyclic%252B%25255B%252B11%252BC%25255DUreas%252Bfor%252BBiomedical%252BImaging%252Bfrom%252B%25255B%252B11%252BC%25255DCarbonyl%252BDifluoride%26aulast%3DJakobsson%26aufirst%3DJimmy%2BE.%26date%3D2021%26date%3D2021%26volume%3D27%26issue%3D40%26spage%3D10369%26epage%3D10376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Iredia D.  Iyamu</span>, <span class="hlFld-ContribAuthor ">Ayad A.  Al-Hamashi</span>, <span class="hlFld-ContribAuthor ">Rong  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">A Pan-Inhibitor for Protein Arginine Methyltransferase Family Enzymes. </span><span class="cited-content_cbyCitation_journal-name">Biomolecules</span><span> <strong>2021,</strong> <em>11 </em>
                                    (6)
                                     , 854. <a href="https://doi.org/10.3390/biom11060854" title="DOI URL">https://doi.org/10.3390/biom11060854</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/biom11060854&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fbiom11060854%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomolecules%26atitle%3DA%252BPan-Inhibitor%252Bfor%252BProtein%252BArginine%252BMethyltransferase%252BFamily%252BEnzymes%26aulast%3DIyamu%26aufirst%3DIredia%2BD.%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D6%26spage%3D854" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">P. Ann  Boriack‐Sjodin</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic Proteins as Emerging Drug Targets. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 423-448. <a href="https://doi.org/10.1002/9781118681121.ch18" title="DOI URL">https://doi.org/10.1002/9781118681121.ch18</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9781118681121.ch18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9781118681121.ch18%26sid%3Dliteratum%253Aachs%26atitle%3DEpigenetic%252BProteins%252Bas%252BEmerging%252BDrug%252BTargets%26aulast%3DBoriack%25E2%2580%2590Sjodin%26aufirst%3DP.%2BAnn%26date%3D2020%26date%3D2020%26spage%3D423%26epage%3D448%26pub%3DWiley%26atitle%3DStructural%252BBiology%252Bin%252BDrug%252BDiscovery%26aulast%3DRenaud%26aufirst%3DJean%25E2%2580%2590Paul%26date%3D2020%26date%3D2020%26volume%3D62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sunil Kumar  Tewary</span>, <span class="hlFld-ContribAuthor ">Y. George  Zheng</span>, <span class="hlFld-ContribAuthor ">Meng-Chiao  Ho</span>. </span><span class="cited-content_cbyCitation_article-title">Protein arginine methyltransferases: insights into the enzyme structure and mechanism at the atomic level. </span><span class="cited-content_cbyCitation_journal-name">Cellular and Molecular Life Sciences</span><span> <strong>2019,</strong> <em>76 </em>
                                    (15)
                                     , 2917-2932. <a href="https://doi.org/10.1007/s00018-019-03145-x" title="DOI URL">https://doi.org/10.1007/s00018-019-03145-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00018-019-03145-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00018-019-03145-x%26sid%3Dliteratum%253Aachs%26jtitle%3DCellular%2520and%2520Molecular%2520Life%2520Sciences%26atitle%3DProtein%252Barginine%252Bmethyltransferases%25253A%252Binsights%252Binto%252Bthe%252Benzyme%252Bstructure%252Band%252Bmechanism%252Bat%252Bthe%252Batomic%252Blevel%26aulast%3DTewary%26aufirst%3DSunil%2BKumar%26date%3D2019%26date%3D2019%26volume%3D76%26issue%3D15%26spage%3D2917%26epage%3D2932" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthijs J.  van Haren</span>, <span class="hlFld-ContribAuthor ">Nathaniel I.  Martin</span>. </span><span class="cited-content_cbyCitation_article-title">PRMT Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 159-196. <a href="https://doi.org/10.1007/7355_2019_73" title="DOI URL">https://doi.org/10.1007/7355_2019_73</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_73&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_73%26sid%3Dliteratum%253Aachs%26atitle%3DPRMT%252BInhibitors%26aulast%3Dvan%2BHaren%26aufirst%3DMatthijs%2BJ.%26date%3D2019%26date%3D2019%26spage%3D159%26epage%3D196%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Amit K.  Behera</span>, <span class="hlFld-ContribAuthor ">Tapas K.  Kundu</span>. </span><span class="cited-content_cbyCitation_article-title">Asymmetric Dimethylation on Arginine (ADMA) of Histones in Development, Differentiation and Disease. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 495-520. <a href="https://doi.org/10.1007/978-3-030-14792-1_20" title="DOI URL">https://doi.org/10.1007/978-3-030-14792-1_20</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-14792-1_20&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-14792-1_20%26sid%3Dliteratum%253Aachs%26atitle%3DAsymmetric%252BDimethylation%252Bon%252BArginine%252B%252528ADMA%252529%252Bof%252BHistones%252Bin%252BDevelopment%25252C%252BDifferentiation%252Band%252BDisease%26aulast%3DBehera%26aufirst%3DAmit%2BK.%26date%3D2019%26date%3D2019%26spage%3D495%26epage%3D520%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DThe%252BDNA%25252C%252BRNA%25252C%252Band%252BHistone%252BMethylomes%26aulast%3DJurga%26aufirst%3DStefan%26date%3D2019%26volume%3D71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ming-Chuan  Hsu</span>, <span class="hlFld-ContribAuthor ">Mei-Ren  Pan</span>, <span class="hlFld-ContribAuthor ">Pei-Yi  Chu</span>, <span class="hlFld-ContribAuthor ">Ya-Li  Tsai</span>, <span class="hlFld-ContribAuthor ">Chia-Hua  Tsai</span>, <span class="hlFld-ContribAuthor ">Yan-Shen  Shan</span>, <span class="hlFld-ContribAuthor ">Li-Tzong  Chen</span>, <span class="hlFld-ContribAuthor ">Wen-Chun  Hung</span>. </span><span class="cited-content_cbyCitation_article-title">Protein Arginine Methyltransferase 3 Enhances Chemoresistance in Pancreatic Cancer by Methylating hnRNPA1 to Increase ABCG2 Expression. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2019,</strong> <em>11 </em>
                                    (1)
                                     , 8. <a href="https://doi.org/10.3390/cancers11010008" title="DOI URL">https://doi.org/10.3390/cancers11010008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers11010008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers11010008%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DProtein%252BArginine%252BMethyltransferase%252B3%252BEnhances%252BChemoresistance%252Bin%252BPancreatic%252BCancer%252Bby%252BMethylating%252BhnRNPA1%252Bto%252BIncrease%252BABCG2%252BExpression%26aulast%3DHsu%26aufirst%3DMing-Chuan%26date%3D2019%26date%3D2018%26volume%3D11%26issue%3D1%26spage%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Masayoshi  Sakurai</span>, <span class="hlFld-ContribAuthor ">Nobuhiro  Kihara</span>, <span class="hlFld-ContribAuthor ">Nobuhiro  Watanabe</span>, <span class="hlFld-ContribAuthor ">Yoshihiro  Ikari</span>, <span class="hlFld-ContribAuthor ">Toshikazu  Takata</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of α-Aminocarbonyl Compounds via Hetero Diels–Alder Reaction. </span><span class="cited-content_cbyCitation_journal-name">Chemistry Letters</span><span> <strong>2018,</strong> <em>47 </em>
                                    (2)
                                     , 144-147. <a href="https://doi.org/10.1246/cl.170970" title="DOI URL">https://doi.org/10.1246/cl.170970</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1246/cl.170970&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1246%2Fcl.170970%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520Letters%26atitle%3DSynthesis%252Bof%252B%2525CE%2525B1-Aminocarbonyl%252BCompounds%252Bvia%252BHetero%252BDiels%2525E2%252580%252593Alder%252BReaction%26aulast%3DSakurai%26aufirst%3DMasayoshi%26date%3D2018%26volume%3D47%26issue%3D2%26spage%3D144%26epage%3D147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Beth E  Zucconi</span>, <span class="hlFld-ContribAuthor ">Philip A  Cole</span>. </span><span class="cited-content_cbyCitation_article-title">Allosteric regulation of epigenetic modifying enzymes. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2017,</strong> <em>39 </em>, 109-115. <a href="https://doi.org/10.1016/j.cbpa.2017.05.015" title="DOI URL">https://doi.org/10.1016/j.cbpa.2017.05.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2017.05.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2017.05.015%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DAllosteric%252Bregulation%252Bof%252Bepigenetic%252Bmodifying%252Benzymes%26aulast%3DZucconi%26aufirst%3DBeth%2BE%26date%3D2017%26volume%3D39%26spage%3D109%26epage%3D115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jernej  Murn</span>, <span class="hlFld-ContribAuthor ">Yang  Shi</span>. </span><span class="cited-content_cbyCitation_article-title">The winding path of protein methylation research: milestones and new frontiers. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Molecular Cell Biology</span><span> <strong>2017,</strong> <em>18 </em>
                                    (8)
                                     , 517-527. <a href="https://doi.org/10.1038/nrm.2017.35" title="DOI URL">https://doi.org/10.1038/nrm.2017.35</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nrm.2017.35&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnrm.2017.35%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Molecular%2520Cell%2520Biology%26atitle%3DThe%252Bwinding%252Bpath%252Bof%252Bprotein%252Bmethylation%252Bresearch%25253A%252Bmilestones%252Band%252Bnew%252Bfrontiers%26aulast%3DMurn%26aufirst%3DJernej%26date%3D2017%26date%3D2017%26volume%3D18%26issue%3D8%26spage%3D517%26epage%3D527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shawn  Johnstone</span>, <span class="hlFld-ContribAuthor ">Jeffrey S.  Albert</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacological property optimization for allosteric ligands: A medicinal chemistry perspective. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2017,</strong> <em>27 </em>
                                    (11)
                                     , 2239-2258. <a href="https://doi.org/10.1016/j.bmcl.2017.03.084" title="DOI URL">https://doi.org/10.1016/j.bmcl.2017.03.084</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2017.03.084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2017.03.084%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DPharmacological%252Bproperty%252Boptimization%252Bfor%252Ballosteric%252Bligands%25253A%252BA%252Bmedicinal%252Bchemistry%252Bperspective%26aulast%3DJohnstone%26aufirst%3DShawn%26date%3D2017%26volume%3D27%26issue%3D11%26spage%3D2239%26epage%3D2258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Levi  Blazer</span>, <span class="hlFld-ContribAuthor ">Mohammad S.  Eram</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Taraneh  Hajian</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting human SET1/MLL family of proteins. </span><span class="cited-content_cbyCitation_journal-name">Protein Science</span><span> <strong>2017,</strong> <em>26 </em>
                                    (4)
                                     , 662-676. <a href="https://doi.org/10.1002/pro.3129" title="DOI URL">https://doi.org/10.1002/pro.3129</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/pro.3129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fpro.3129%26sid%3Dliteratum%253Aachs%26jtitle%3DProtein%2520Science%26atitle%3DTargeting%252Bhuman%252BSET1%25252FMLL%252Bfamily%252Bof%252Bproteins%26aulast%3DVedadi%26aufirst%3DMasoud%26date%3D2017%26date%3D2017%26volume%3D26%26issue%3D4%26spage%3D662%26epage%3D676" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mikhail A.  Prezent</span>, <span class="hlFld-ContribAuthor ">Elena D.  Daeva</span>, <span class="hlFld-ContribAuthor ">Sergey V.  Baranin</span>, <span class="hlFld-ContribAuthor ">Igor V.  Zavarzin</span>. </span><span class="cited-content_cbyCitation_article-title">A new synthesis of 2-(aminoalkyl)-1,2,4-triazolo[1,5- a ]pyrimidines. </span><span class="cited-content_cbyCitation_journal-name">Mendeleev Communications</span><span> <strong>2017,</strong> <em>27 </em>
                                    (2)
                                     , 169-171. <a href="https://doi.org/10.1016/j.mencom.2017.03.021" title="DOI URL">https://doi.org/10.1016/j.mencom.2017.03.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.mencom.2017.03.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.mencom.2017.03.021%26sid%3Dliteratum%253Aachs%26jtitle%3DMendeleev%2520Communications%26atitle%3DA%252Bnew%252Bsynthesis%252Bof%252B2-%252528aminoalkyl%252529-1%25252C2%25252C4-triazolo%25255B1%25252C5-%252Ba%252B%25255Dpyrimidines%26aulast%3DPrezent%26aufirst%3DMikhail%2BA.%26date%3D2017%26volume%3D27%26issue%3D2%26spage%3D169%26epage%3D171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kenneth W.  Duncan</span>, <span class="hlFld-ContribAuthor ">John E.  Campbell</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic Modulators. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 227-227. <a href="https://doi.org/10.1007/7355_2017_30" title="DOI URL">https://doi.org/10.1007/7355_2017_30</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2017_30&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2017_30%26sid%3Dliteratum%253Aachs%26atitle%3DEpigenetic%252BModulators%26aulast%3DDuncan%26aufirst%3DKenneth%2BW.%26date%3D2017%26spage%3D227%26epage%3D227%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DCancer%252BII%26aulast%3DWaring%26aufirst%3DMichael%2BJ.%26date%3D2018%26volume%3D28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wen Fong  Ooi</span>, <span class="hlFld-ContribAuthor ">Xiaosai  Yao</span>, <span class="hlFld-ContribAuthor ">Patrick  Tan</span>, <span class="hlFld-ContribAuthor ">Bin Tean  Teh</span>. </span><span class="cited-content_cbyCitation_article-title">Misregulation of Histone Methylation Regulators in Cancer. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 221-248. <a href="https://doi.org/10.1007/978-3-319-59786-7_8" title="DOI URL">https://doi.org/10.1007/978-3-319-59786-7_8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-319-59786-7_8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-319-59786-7_8%26sid%3Dliteratum%253Aachs%26atitle%3DMisregulation%252Bof%252BHistone%252BMethylation%252BRegulators%252Bin%252BCancer%26aulast%3DOoi%26aufirst%3DWen%2BFong%26date%3D2017%26date%3D2017%26spage%3D221%26epage%3D248%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DDNA%252Band%252BHistone%252BMethylation%252Bas%252BCancer%252BTargets%26aulast%3DKaneda%26aufirst%3DAtsushi%26date%3D2017%26volume%3D339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xuejiao  Song</span>, <span class="hlFld-ContribAuthor ">Lidan  Zhang</span>, <span class="hlFld-ContribAuthor ">Tiantao  Gao</span>, <span class="hlFld-ContribAuthor ">Tinghong  Ye</span>, <span class="hlFld-ContribAuthor ">Yongxia  Zhu</span>, <span class="hlFld-ContribAuthor ">Qian  Lei</span>, <span class="hlFld-ContribAuthor ">Qiang  Feng</span>, <span class="hlFld-ContribAuthor ">Bing  He</span>, <span class="hlFld-ContribAuthor ">Hongxia  Deng</span>, <span class="hlFld-ContribAuthor ">Luoting  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Selective inhibition of EZH2 by ZLD10A blocks H3K27 methylation and kills mutant lymphoma cells proliferation. </span><span class="cited-content_cbyCitation_journal-name">Biomedicine & Pharmacotherapy</span><span> <strong>2016,</strong> <em>81 </em>, 288-294. <a href="https://doi.org/10.1016/j.biopha.2016.04.019" title="DOI URL">https://doi.org/10.1016/j.biopha.2016.04.019</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.biopha.2016.04.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.biopha.2016.04.019%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomedicine%2520%2526%2520Pharmacotherapy%26atitle%3DSelective%252Binhibition%252Bof%252BEZH2%252Bby%252BZLD10A%252Bblocks%252BH3K27%252Bmethylation%252Band%252Bkills%252Bmutant%252Blymphoma%252Bcells%252Bproliferation%26aulast%3DSong%26aufirst%3DXuejiao%26date%3D2016%26volume%3D81%26spage%3D288%26epage%3D294" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Manuela  Terranova-Barberio</span>, <span class="hlFld-ContribAuthor ">Scott  Thomas</span>, <span class="hlFld-ContribAuthor ">Pamela N  Munster</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic modifiers in immunotherapy: a focus on checkpoint inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Immunotherapy</span><span> <strong>2016,</strong> <em>8 </em>
                                    (6)
                                     , 705-719. <a href="https://doi.org/10.2217/imt-2016-0014" title="DOI URL">https://doi.org/10.2217/imt-2016-0014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2217/imt-2016-0014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2217%2Fimt-2016-0014%26sid%3Dliteratum%253Aachs%26jtitle%3DImmunotherapy%26atitle%3DEpigenetic%252Bmodifiers%252Bin%252Bimmunotherapy%25253A%252Ba%252Bfocus%252Bon%252Bcheckpoint%252Binhibitors%26aulast%3DTerranova-Barberio%26aufirst%3DManuela%26date%3D2016%26volume%3D8%26issue%3D6%26spage%3D705%26epage%3D719" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hao  Hu</span>, <span class="hlFld-ContribAuthor ">Kun  Qian</span>, <span class="hlFld-ContribAuthor ">Meng-Chiao  Ho</span>, <span class="hlFld-ContribAuthor ">Y. George  Zheng</span>. </span><span class="cited-content_cbyCitation_article-title">Small Molecule Inhibitors of Protein Arginine Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Investigational Drugs</span><span> <strong>2016,</strong> <em>25 </em>
                                    (3)
                                     , 335-358. <a href="https://doi.org/10.1517/13543784.2016.1144747" title="DOI URL">https://doi.org/10.1517/13543784.2016.1144747</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1517/13543784.2016.1144747&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1517%2F13543784.2016.1144747%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Investigational%2520Drugs%26atitle%3DSmall%252BMolecule%252BInhibitors%252Bof%252BProtein%252BArginine%252BMethyltransferases%26aulast%3DHu%26aufirst%3DHao%26date%3D2016%26date%3D2016%26volume%3D25%26issue%3D3%26spage%3D335%26epage%3D358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xuejiao  Song</span>, <span class="hlFld-ContribAuthor ">Tiantao  Gao</span>, <span class="hlFld-ContribAuthor ">Ningyu  Wang</span>, <span class="hlFld-ContribAuthor ">Qiang  Feng</span>, <span class="hlFld-ContribAuthor ">Xinyu  You</span>, <span class="hlFld-ContribAuthor ">Tinghong  Ye</span>, <span class="hlFld-ContribAuthor ">Qian  Lei</span>, <span class="hlFld-ContribAuthor ">Yongxia  Zhu</span>, <span class="hlFld-ContribAuthor ">Menghua  Xiong</span>, <span class="hlFld-ContribAuthor ">Yong  Xia</span>, <span class="hlFld-ContribAuthor ">Fangfang  Yang</span>, <span class="hlFld-ContribAuthor ">Yaojie  Shi</span>, <span class="hlFld-ContribAuthor ">Yuquan  Wei</span>, <span class="hlFld-ContribAuthor ">Lidan  Zhang</span>, <span class="hlFld-ContribAuthor ">Luoting  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Selective inhibition of EZH2 by ZLD1039 blocks H3K27methylation and leads to potent anti-tumor activity in breast cancer. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2016,</strong> <em>6 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/srep20864" title="DOI URL">https://doi.org/10.1038/srep20864</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/srep20864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fsrep20864%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DSelective%252Binhibition%252Bof%252BEZH2%252Bby%252BZLD1039%252Bblocks%252BH3K27methylation%252Band%252Bleads%252Bto%252Bpotent%252Banti-tumor%252Bactivity%252Bin%252Bbreast%252Bcancer%26aulast%3DSong%26aufirst%3DXuejiao%26date%3D2016%26date%3D2016%26volume%3D6%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aman  Iqbal</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetics and Fragment-Based Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2016,</strong>,, 355-370. <a href="https://doi.org/10.1002/9783527683604.ch15" title="DOI URL">https://doi.org/10.1002/9783527683604.ch15</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527683604.ch15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527683604.ch15%26sid%3Dliteratum%253Aachs%26atitle%3DEpigenetics%252Band%252BFragment-Based%252BDrug%252BDiscovery%26aulast%3DIqbal%26aufirst%3DAman%26date%3D2016%26date%3D2016%26spage%3D355%26epage%3D370%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DFragment-based%252BDrug%252BDiscovery%252BLessons%252Band%252BOutlook%26aulast%3DErlanson%26aufirst%3DDaniel%2BA.%26date%3D2016%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kun  Qian</span>, <span class="hlFld-ContribAuthor ">Y.  George Zheng</span>. </span><span class="cited-content_cbyCitation_article-title">Current Development of Protein Arginine Methyltransferase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2016,</strong>,, 231-256. <a href="https://doi.org/10.1016/B978-0-12-802808-7.00008-3" title="DOI URL">https://doi.org/10.1016/B978-0-12-802808-7.00008-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-802808-7.00008-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-802808-7.00008-3%26sid%3Dliteratum%253Aachs%26atitle%3DCurrent%252BDevelopment%252Bof%252BProtein%252BArginine%252BMethyltransferase%252BInhibitors%26aulast%3DQian%26aufirst%3DKun%26date%3D2016%26spage%3D231%26epage%3D256%26pub%3DElsevier%26atitle%3DEpi-Informatics%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nataliya P.  Belskaya</span>, <span class="hlFld-ContribAuthor ">Kseniya I.  Lugovik</span>, <span class="hlFld-ContribAuthor ">Vasiliy A.  Bakulev</span>, <span class="hlFld-ContribAuthor ">Jenny  Bauer</span>, <span class="hlFld-ContribAuthor ">Igor  Kitanovic</span>, <span class="hlFld-ContribAuthor ">Pavlo  Holenya</span>, <span class="hlFld-ContribAuthor ">Maksim  Zakhartsev</span>, <span class="hlFld-ContribAuthor ">Stefan  Wölfl</span>. </span><span class="cited-content_cbyCitation_article-title">The new facile and straightforward method for the synthesis of 4 H -1,2,3-thiadiazolo[5,4- b ]indoles and determination of their antiproliferative activity. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>108 </em>, 245-257. <a href="https://doi.org/10.1016/j.ejmech.2015.11.011" title="DOI URL">https://doi.org/10.1016/j.ejmech.2015.11.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2015.11.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2015.11.011%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThe%252Bnew%252Bfacile%252Band%252Bstraightforward%252Bmethod%252Bfor%252Bthe%252Bsynthesis%252Bof%2525C2%2525A04%252BH%252B-1%25252C2%25252C3-thiadiazolo%25255B5%25252C4-%252Bb%252B%25255Dindoles%252Band%252Bdetermination%252Bof%252Btheir%252Bantiproliferative%252Bactivity%26aulast%3DBelskaya%26aufirst%3DNataliya%2BP.%26date%3D2016%26volume%3D108%26spage%3D245%26epage%3D257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ying  Yang</span>, <span class="hlFld-ContribAuthor ">Congcong  Wang</span>, <span class="hlFld-ContribAuthor ">Xinxue  Li</span>, <span class="hlFld-ContribAuthor ">Qianyun  Chai</span>, <span class="hlFld-ContribAuthor ">Yutong  Fei</span>, <span class="hlFld-ContribAuthor ">Ruyu  Xia</span>, <span class="hlFld-ContribAuthor ">Rongqian  Xu</span>, <span class="hlFld-ContribAuthor ">Li  Yang</span>, <span class="hlFld-ContribAuthor ">Jianping  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Chinese herbal medicine for Henoch–Schönlein purpura in children without renal damage: A systematic review of randomized controlled trials. </span><span class="cited-content_cbyCitation_journal-name">Complementary Therapies in Medicine</span><span> <strong>2015,</strong> <em>23 </em>
                                    (5)
                                     , 741-750. <a href="https://doi.org/10.1016/j.ctim.2015.07.010" title="DOI URL">https://doi.org/10.1016/j.ctim.2015.07.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ctim.2015.07.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ctim.2015.07.010%26sid%3Dliteratum%253Aachs%26jtitle%3DComplementary%2520Therapies%2520in%2520Medicine%26atitle%3DChinese%252Bherbal%252Bmedicine%252Bfor%252BHenoch%2525E2%252580%252593Sch%2525C3%2525B6nlein%252Bpurpura%252Bin%252Bchildren%252Bwithout%252Brenal%252Bdamage%25253A%252BA%252Bsystematic%252Breview%252Bof%252Brandomized%252Bcontrolled%252Btrials%26aulast%3DYang%26aufirst%3DYing%26date%3D2015%26volume%3D23%26issue%3D5%26spage%3D741%26epage%3D750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Juan  Bayo</span>, <span class="hlFld-ContribAuthor ">Maithili P  Dalvi</span>, <span class="hlFld-ContribAuthor ">Elisabeth D  Martinez</span>. </span><span class="cited-content_cbyCitation_article-title">Successful strategies in the discovery of small-molecule epigenetic modulators with anticancer potential. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2015,</strong> <em>7 </em>
                                    (16)
                                     , 2243-2261. <a href="https://doi.org/10.4155/fmc.15.140" title="DOI URL">https://doi.org/10.4155/fmc.15.140</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc.15.140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc.15.140%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DSuccessful%252Bstrategies%252Bin%252Bthe%252Bdiscovery%252Bof%252Bsmall-molecule%252Bepigenetic%252Bmodulators%252Bwith%252Banticancer%252Bpotential%26aulast%3DBayo%26aufirst%3DJuan%26date%3D2015%26volume%3D7%26issue%3D16%26spage%3D2243%26epage%3D2261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Saedeh  Dadgar</span>, <span class="hlFld-ContribAuthor ">Wely B.  Floriano</span>. </span><span class="cited-content_cbyCitation_article-title">Systematic discovery of molecular probes targeting multiple non-orthosteric and spatially distinct sites in the botulinum neurotoxin subtype A (BoNT/A). </span><span class="cited-content_cbyCitation_journal-name">Molecular and Cellular Probes</span><span> <strong>2015,</strong> <em>29 </em>
                                    (3)
                                     , 135-143. <a href="https://doi.org/10.1016/j.mcp.2015.02.002" title="DOI URL">https://doi.org/10.1016/j.mcp.2015.02.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.mcp.2015.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.mcp.2015.02.002%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520and%2520Cellular%2520Probes%26atitle%3DSystematic%252Bdiscovery%252Bof%252Bmolecular%252Bprobes%252Btargeting%252Bmultiple%252Bnon-orthosteric%252Band%252Bspatially%252Bdistinct%252Bsites%252Bin%252Bthe%252Bbotulinum%252Bneurotoxin%252Bsubtype%252BA%252B%252528BoNT%25252FA%252529%26aulast%3DDadgar%26aufirst%3DSaedeh%26date%3D2015%26volume%3D29%26issue%3D3%26spage%3D135%26epage%3D143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elayne  Chan-Penebre</span>, <span class="hlFld-ContribAuthor ">Kristy G  Kuplast</span>, <span class="hlFld-ContribAuthor ">Christina R  Majer</span>, <span class="hlFld-ContribAuthor ">P Ann  Boriack-Sjodin</span>, <span class="hlFld-ContribAuthor ">Tim J  Wigle</span>, <span class="hlFld-ContribAuthor ">L Danielle  Johnston</span>, <span class="hlFld-ContribAuthor ">Nathalie  Rioux</span>, <span class="hlFld-ContribAuthor ">Michael J  Munchhof</span>, <span class="hlFld-ContribAuthor ">Lei  Jin</span>, <span class="hlFld-ContribAuthor ">Suzanne L  Jacques</span>, <span class="hlFld-ContribAuthor ">Kip A  West</span>, <span class="hlFld-ContribAuthor ">Trupti  Lingaraj</span>, <span class="hlFld-ContribAuthor ">Kimberly  Stickland</span>, <span class="hlFld-ContribAuthor ">Scott A  Ribich</span>, <span class="hlFld-ContribAuthor ">Alejandra  Raimondi</span>, <span class="hlFld-ContribAuthor ">Margaret Porter  Scott</span>, <span class="hlFld-ContribAuthor ">Nigel J  Waters</span>, <span class="hlFld-ContribAuthor ">Roy M  Pollock</span>, <span class="hlFld-ContribAuthor ">Jesse J  Smith</span>, <span class="hlFld-ContribAuthor ">Olena  Barbash</span>, <span class="hlFld-ContribAuthor ">Melissa  Pappalardi</span>, <span class="hlFld-ContribAuthor ">Thau F  Ho</span>, <span class="hlFld-ContribAuthor ">Kelvin  Nurse</span>, <span class="hlFld-ContribAuthor ">Khyati P  Oza</span>, <span class="hlFld-ContribAuthor ">Kathleen T  Gallagher</span>, <span class="hlFld-ContribAuthor ">Ryan  Kruger</span>, <span class="hlFld-ContribAuthor ">Mikel P  Moyer</span>, <span class="hlFld-ContribAuthor ">Robert A  Copeland</span>, <span class="hlFld-ContribAuthor ">Richard  Chesworth</span>, <span class="hlFld-ContribAuthor ">Kenneth W  Duncan</span>. </span><span class="cited-content_cbyCitation_article-title">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models. </span><span class="cited-content_cbyCitation_journal-name">Nature Chemical Biology</span><span> <strong>2015,</strong> <em>11 </em>
                                    (6)
                                     , 432-437. <a href="https://doi.org/10.1038/nchembio.1810" title="DOI URL">https://doi.org/10.1038/nchembio.1810</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nchembio.1810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnchembio.1810%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Chemical%2520Biology%26atitle%3DA%252Bselective%252Binhibitor%252Bof%252BPRMT5%252Bwith%252Bin%252Bvivo%252Band%252Bin%252Bvitro%252Bpotency%252Bin%252BMCL%252Bmodels%26aulast%3DChan-Penebre%26aufirst%3DElayne%26date%3D2015%26date%3D2015%26volume%3D11%26issue%3D6%26spage%3D432%26epage%3D437" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">M. A.  Prezent</span>, <span class="hlFld-ContribAuthor ">E. D.  Daeva</span>, <span class="hlFld-ContribAuthor ">S. V.  Baranin</span>, <span class="hlFld-ContribAuthor ">V. A.  Dorokhov</span>. </span><span class="cited-content_cbyCitation_article-title">A new approach to the synthesis of 3-amino- and 3-benzoylamino-5-aminoalkyl-1,2,4-triazoles. </span><span class="cited-content_cbyCitation_journal-name">Russian Chemical Bulletin</span><span> <strong>2015,</strong> <em>64 </em>
                                    (5)
                                     , 1089-1092. <a href="https://doi.org/10.1007/s11172-015-0983-6" title="DOI URL">https://doi.org/10.1007/s11172-015-0983-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11172-015-0983-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11172-015-0983-6%26sid%3Dliteratum%253Aachs%26jtitle%3DRussian%2520Chemical%2520Bulletin%26atitle%3DA%252Bnew%252Bapproach%252Bto%252Bthe%252Bsynthesis%252Bof%252B3-amino-%252Band%252B3-benzoylamino-5-aminoalkyl-1%25252C2%25252C4-triazoles%26aulast%3DPrezent%26aufirst%3DM.%2BA.%26date%3D2015%26date%3D2016%26volume%3D64%26issue%3D5%26spage%3D1089%26epage%3D1092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Magdalena M.  Szewczyk</span>, <span class="hlFld-ContribAuthor ">Zhengtian  Yu</span>, <span class="hlFld-ContribAuthor ">Mohammad S.  Eram</span>, <span class="hlFld-ContribAuthor ">Xiaobao  Yang</span>, <span class="hlFld-ContribAuthor ">Keith  Schmidt</span>, <span class="hlFld-ContribAuthor ">Xiao  Luo</span>, <span class="hlFld-ContribAuthor ">Miao  Dai</span>, <span class="hlFld-ContribAuthor ">Feng  He</span>, <span class="hlFld-ContribAuthor ">Irene  Zang</span>, <span class="hlFld-ContribAuthor ">Ying  Lin</span>, <span class="hlFld-ContribAuthor ">Steven  Kennedy</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Elena  Dobrovetsky</span>, <span class="hlFld-ContribAuthor ">Aiping  Dong</span>, <span class="hlFld-ContribAuthor ">David  Smil</span>, <span class="hlFld-ContribAuthor ">Sun-Joon  Min</span>, <span class="hlFld-ContribAuthor ">Melissa  Landon</span>, <span class="hlFld-ContribAuthor ">Jennifer  Lin-Jones</span>, <span class="hlFld-ContribAuthor ">Xi-Ping  Huang</span>, <span class="hlFld-ContribAuthor ">Bryan L.  Roth</span>, <span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>, <span class="hlFld-ContribAuthor ">Peter  Atadja</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Kehao  Zhao</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>. </span><span class="cited-content_cbyCitation_article-title">A Potent, Selective and Cell-Active Allosteric Inhibitor of Protein Arginine Methyltransferase 3 (PRMT3). </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2015,</strong> <em>127 </em>
                                    (17)
                                     , 5255-5259. <a href="https://doi.org/10.1002/ange.201412154" title="DOI URL">https://doi.org/10.1002/ange.201412154</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.201412154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.201412154%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DA%252BPotent%25252C%252BSelective%252Band%252BCell-Active%252BAllosteric%252BInhibitor%252Bof%252BProtein%252BArginine%252BMethyltransferase%2525E2%252580%2525853%252B%252528PRMT3%252529%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2015%26date%3D2015%26volume%3D127%26issue%3D17%26spage%3D5255%26epage%3D5259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Magdalena M.  Szewczyk</span>, <span class="hlFld-ContribAuthor ">Zhengtian  Yu</span>, <span class="hlFld-ContribAuthor ">Mohammad S.  Eram</span>, <span class="hlFld-ContribAuthor ">Xiaobao  Yang</span>, <span class="hlFld-ContribAuthor ">Keith  Schmidt</span>, <span class="hlFld-ContribAuthor ">Xiao  Luo</span>, <span class="hlFld-ContribAuthor ">Miao  Dai</span>, <span class="hlFld-ContribAuthor ">Feng  He</span>, <span class="hlFld-ContribAuthor ">Irene  Zang</span>, <span class="hlFld-ContribAuthor ">Ying  Lin</span>, <span class="hlFld-ContribAuthor ">Steven  Kennedy</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Elena  Dobrovetsky</span>, <span class="hlFld-ContribAuthor ">Aiping  Dong</span>, <span class="hlFld-ContribAuthor ">David  Smil</span>, <span class="hlFld-ContribAuthor ">Sun-Joon  Min</span>, <span class="hlFld-ContribAuthor ">Melissa  Landon</span>, <span class="hlFld-ContribAuthor ">Jennifer  Lin-Jones</span>, <span class="hlFld-ContribAuthor ">Xi-Ping  Huang</span>, <span class="hlFld-ContribAuthor ">Bryan L.  Roth</span>, <span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>, <span class="hlFld-ContribAuthor ">Peter  Atadja</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Kehao  Zhao</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>. </span><span class="cited-content_cbyCitation_article-title">A Potent, Selective and Cell-Active Allosteric Inhibitor of Protein Arginine Methyltransferase 3 (PRMT3). </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2015,</strong> <em>54 </em>
                                    (17)
                                     , 5166-5170. <a href="https://doi.org/10.1002/anie.201412154" title="DOI URL">https://doi.org/10.1002/anie.201412154</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.201412154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.201412154%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DA%252BPotent%25252C%252BSelective%252Band%252BCell-Active%252BAllosteric%252BInhibitor%252Bof%252BProtein%252BArginine%252BMethyltransferase%2525E2%252580%2525853%252B%252528PRMT3%252529%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2015%26date%3D2015%26volume%3D54%26issue%3D17%26spage%3D5166%26epage%3D5170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Min  Zhang</span>, <span class="hlFld-ContribAuthor ">Yuefa  Gong</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of 2,3,3a,4,5,6-Hexahydrobenzo[b]thiophene-3a-carbaldehydes via a Tandem Reaction of Cyclic β-Thiocyanatoenals with Electron-Deficient Alkenes Triggered by Fluoride. </span><span class="cited-content_cbyCitation_journal-name">Journal of Heterocyclic Chemistry</span><span> <strong>2015,</strong> <em>52 </em>
                                    (2)
                                     , 573-577. <a href="https://doi.org/10.1002/jhet.2115" title="DOI URL">https://doi.org/10.1002/jhet.2115</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jhet.2115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjhet.2115%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Heterocyclic%2520Chemistry%26atitle%3DSynthesis%252Bof%252B2%25252C3%25252C3a%25252C4%25252C5%25252C6-Hexahydrobenzo%25255Bb%25255Dthiophene-3a-carbaldehydes%252Bvia%252Ba%252BTandem%252BReaction%252Bof%252BCyclic%252B%2525CE%2525B2-Thiocyanatoenals%252Bwith%252BElectron-Deficient%252BAlkenes%252BTriggered%252Bby%252BFluoride%26aulast%3DZhang%26aufirst%3DMin%26date%3D2015%26date%3D2014%26volume%3D52%26issue%3D2%26spage%3D573%26epage%3D577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Keqin Kathy  Li</span>, <span class="hlFld-ContribAuthor ">Kenneth  Huang</span>, <span class="hlFld-ContribAuthor ">Shukkoor  Kondengaden</span>, <span class="hlFld-ContribAuthor ">Jonathan  Wooten</span>, <span class="hlFld-ContribAuthor ">Hamed  Reyhanfard</span>, <span class="hlFld-ContribAuthor ">Zhang  Qing</span>, <span class="hlFld-ContribAuthor ">Bingxue Chris  Zhai</span>, <span class="hlFld-ContribAuthor ">Peng George  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Histone Methyltransferase Inhibitors for Cancer Therapy. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 363-395. <a href="https://doi.org/10.1016/B978-0-12-801080-8.00017-X" title="DOI URL">https://doi.org/10.1016/B978-0-12-801080-8.00017-X</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-801080-8.00017-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-801080-8.00017-X%26sid%3Dliteratum%253Aachs%26atitle%3DHistone%252BMethyltransferase%252BInhibitors%252Bfor%252BCancer%252BTherapy%26aulast%3DLi%26aufirst%3DKeqin%2BKathy%26date%3D2015%26spage%3D363%26epage%3D395%26pub%3DElsevier%26atitle%3DEpigenetic%252BTechnological%252BApplications%26date%3D2015%26volume%3D41" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aili  Chen</span>, <span class="hlFld-ContribAuthor ">Gang  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Animal Model Study of Epigenetic Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 447-477. <a href="https://doi.org/10.1016/B978-0-12-801080-8.00020-X" title="DOI URL">https://doi.org/10.1016/B978-0-12-801080-8.00020-X</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-801080-8.00020-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-801080-8.00020-X%26sid%3Dliteratum%253Aachs%26atitle%3DAnimal%252BModel%252BStudy%252Bof%252BEpigenetic%252BInhibitors%26aulast%3DChen%26aufirst%3DAili%26date%3D2015%26spage%3D447%26epage%3D477%26pub%3DElsevier%26atitle%3DEpigenetic%252BTechnological%252BApplications%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>, <span class="hlFld-ContribAuthor ">Renato  Ferreira de Freitas</span>. </span><span class="cited-content_cbyCitation_article-title">Structural biology and chemistry of protein arginine methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2014,</strong> <em>5 </em>
                                    (12)
                                     , 1779-1788. <a href="https://doi.org/10.1039/C4MD00269E" title="DOI URL">https://doi.org/10.1039/C4MD00269E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C4MD00269E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC4MD00269E%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DStructural%252Bbiology%252Band%252Bchemistry%252Bof%252Bprotein%252Barginine%252Bmethyltransferases%26aulast%3DSchapira%26aufirst%3DMatthieu%26date%3D2014%26date%3D2014%26volume%3D5%26issue%3D12%26spage%3D1779%26epage%3D1788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anqi  Ma</span>, <span class="hlFld-ContribAuthor ">Wenyu  Yu</span>, <span class="hlFld-ContribAuthor ">Yan  Xiong</span>, <span class="hlFld-ContribAuthor ">Kyle V.  Butler</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Structure–activity relationship studies of SETD8 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2014,</strong> <em>5 </em>
                                    (12)
                                     , 1892-1898. <a href="https://doi.org/10.1039/C4MD00317A" title="DOI URL">https://doi.org/10.1039/C4MD00317A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C4MD00317A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC4MD00317A%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DStructure%2525E2%252580%252593activity%252Brelationship%252Bstudies%252Bof%252BSETD8%252Binhibitors%26aulast%3DMa%26aufirst%3DAnqi%26date%3D2014%26date%3D2014%26volume%3D5%26issue%3D12%26spage%3D1892%26epage%3D1898" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yanli  Liu</span>, <span class="hlFld-ContribAuthor ">Ke  Liu</span>, <span class="hlFld-ContribAuthor ">Su  Qin</span>, <span class="hlFld-ContribAuthor ">Chao  Xu</span>, <span class="hlFld-ContribAuthor ">Jinrong  Min</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic targets and drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Pharmacology & Therapeutics</span><span> <strong>2014,</strong> <em>143 </em>
                                    (3)
                                     , 275-294. <a href="https://doi.org/10.1016/j.pharmthera.2014.03.007" title="DOI URL">https://doi.org/10.1016/j.pharmthera.2014.03.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pharmthera.2014.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pharmthera.2014.03.007%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacology%2520%2526%2520Therapeutics%26atitle%3DEpigenetic%252Btargets%252Band%252Bdrug%252Bdiscovery%26aulast%3DLiu%26aufirst%3DYanli%26date%3D2014%26volume%3D143%26issue%3D3%26spage%3D275%26epage%3D294" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert M.  Campbell</span>, <span class="hlFld-ContribAuthor ">Peter J.  Tummino</span>. </span><span class="cited-content_cbyCitation_article-title">Cancer epigenetics drug discovery and development: the challenge of hitting the mark. </span><span class="cited-content_cbyCitation_journal-name">Journal of Clinical Investigation</span><span> <strong>2014,</strong> <em>124 </em>
                                    (1)
                                     , 64-69. <a href="https://doi.org/10.1172/JCI71605" title="DOI URL">https://doi.org/10.1172/JCI71605</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1172/JCI71605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1172%2FJCI71605%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Clinical%2520Investigation%26atitle%3DCancer%252Bepigenetics%252Bdrug%252Bdiscovery%252Band%252Bdevelopment%25253A%252Bthe%252Bchallenge%252Bof%252Bhitting%252Bthe%252Bmark%26aulast%3DCampbell%26aufirst%3DRobert%2BM.%26date%3D2014%26date%3D2014%26volume%3D124%26issue%3D1%26spage%3D64%26epage%3D69" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yong-Jin  Wu</span>, <span class="hlFld-ContribAuthor ">Bingwei V.  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Five-Membered Ring Systems. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2014,</strong>,, 279-301. <a href="https://doi.org/10.1016/B978-0-08-100017-5.00009-1" title="DOI URL">https://doi.org/10.1016/B978-0-08-100017-5.00009-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-08-100017-5.00009-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-08-100017-5.00009-1%26sid%3Dliteratum%253Aachs%26atitle%3DFive-Membered%252BRing%252BSystems%26aulast%3DWu%26aufirst%3DYong-Jin%26date%3D2014%26spage%3D279%26epage%3D301%26pub%3DElsevier%26date%3D2014%26volume%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>. </span><span class="cited-content_cbyCitation_article-title">Drug discovery in academic institutions. </span><span class="cited-content_cbyCitation_journal-name">Hematology</span><span> <strong>2013,</strong> <em>2013 </em>
                                    (1)
                                     , 300-305. <a href="https://doi.org/10.1182/asheducation-2013.1.300" title="DOI URL">https://doi.org/10.1182/asheducation-2013.1.300</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1182/asheducation-2013.1.300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1182%2Fasheducation-2013.1.300%26sid%3Dliteratum%253Aachs%26jtitle%3DHematology%26atitle%3DDrug%252Bdiscovery%252Bin%252Bacademic%252Binstitutions%26aulast%3DFrye%26aufirst%3DStephen%2BV.%26date%3D2013%26volume%3D2013%26issue%3D1%26spage%3D300%26epage%3D305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stefan  Knapp</span>, <span class="hlFld-ContribAuthor ">Hilmar  Weinmann</span>. </span><span class="cited-content_cbyCitation_article-title">Small-Molecule Modulators for Epigenetics Targets. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2013,</strong> <em>8 </em>
                                    (11)
                                     , 1885-1891. <a href="https://doi.org/10.1002/cmdc.201300344" title="DOI URL">https://doi.org/10.1002/cmdc.201300344</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201300344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201300344%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DSmall-Molecule%252BModulators%252Bfor%252BEpigenetics%252BTargets%26aulast%3DKnapp%26aufirst%3DStefan%26date%3D2013%26date%3D2013%26volume%3D8%26issue%3D11%26spage%3D1885%26epage%3D1891" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kristian  Helin</span>, <span class="hlFld-ContribAuthor ">Dashyant  Dhanak</span>. </span><span class="cited-content_cbyCitation_article-title">Chromatin proteins and modifications as drug targets. </span><span class="cited-content_cbyCitation_journal-name">Nature</span><span> <strong>2013,</strong> <em>502 </em>
                                    (7472)
                                     , 480-488. <a href="https://doi.org/10.1038/nature12751" title="DOI URL">https://doi.org/10.1038/nature12751</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nature12751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnature12751%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%26atitle%3DChromatin%252Bproteins%252Band%252Bmodifications%252Bas%252Bdrug%252Btargets%26aulast%3DHelin%26aufirst%3DKristian%26date%3D2013%26date%3D2013%26volume%3D502%26issue%3D7472%26spage%3D480%26epage%3D488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3018332&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Structure of PRMT3 inhibitors <b>1</b> and <b>2</b> and three regions explored for SAR. (B) X-ray crystal structure of the inhibitor <b>1</b>–PRMT3 complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SMQ">3SMQ</a>). Key interactions include (1) a hydrogen bond between the 2-nitrogen of the benzothiadiazole and T466, (2) two hydrogen bonds between the two amino groups of the middle urea moiety and E422, (3) a hydrogen bond between the oxygen of the urea moiety and R396, and (4) hydrophobic interactions between the cyclohexenylethyl group and a nonpolar surface.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3018332&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0006.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>3</b>–<b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3018332&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) CDI, THF, 2-cyclohexylethylamine, rt, 41–88%.</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0007.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds for Exploring the Middle Urea Moiety<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3018332&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) N<sub>2</sub>H<sub>4</sub>·H<sub>2</sub>O, EtOH, reflux; (b) H<sub>2</sub>O<sub>2</sub>, rt, 87% over two steps; (c) KNO<sub>2</sub>/H<sub>2</sub>SO<sub>4</sub>, 0–10 °C, 62%; (d) SnCl<sub>2</sub>/HCl, 65 °C, 5 min, 61%; (e) 3,4-diethoxycyclobut-3-ene-1,2-dione, Zn(OTf)<sub>2</sub>, EtOH, rt, 92%; (f) 2-cyclohexylethylamine or 2-amino-1-phenylethanone, <i>i-</i>PrOH, 120 °C, microwave, 78–85%; (g) diphenyl cyanocarbonimidate, <i>i</i>-PrOH, 100 °C, microwave; (h) 2-cyclohexylethylamine, <i>i</i>-PrOH, 100 °C, microwave, 38% over two steps; (i) BrCN, THF, 40 °C; (j) 3-cyclohexylpropanehydrazide, HCl, <i>i-</i>PrOH, 160 °C, microwave, 32% over two steps.</p></p></figure><figure data-id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0008.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of the Compounds Outlined in Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3018332&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) CDI, DMF, various amines, rt, 35–55%; (b) TFA, DCM, rt, 100%.</p></p></figure><figure data-id="sch4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0009.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of the Compounds Outlined in Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3018332&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) CDI, DMF, various amines, rt, 35–55%; (b) TFA, DCM, rt, 100%; (c) PySO<sub>3</sub>, Et<sub>3</sub>N, DCM/DMSO, rt, 76–78%.</p></p></figure><figure data-id="fig2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0003.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Compound <b>14u</b> was selective for PRMT3 over other PRMTs, PKMTs, and DNMT1. The effects of compound <b>14u</b> on the activity of the following methyltransferases were assessed: (A) PRMT3 (●, IC<sub>50</sub> = 0.48 ± 0.01 μM), G9a (○, IC<sub>50</sub> = 20 ± 1 μM), GLP (▼, IC<sub>50</sub> = 25 ± 8 μM), and SUV39H2 (△, IC<sub>50</sub> = 22 ± 1 μM); (B) SETD7 (●), PRC2 complex (○), SETD8 (▼), SETDB1 (△), SUV420H1 (■), SUV420H2 (□), MLL1 complex (◇), SMYD3 (▲), SMYD2 (∇), DOT1L (□), PRMT5–MEP50 complex (◆), and DNMT1 (⬢).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3018332&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0004.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. X-ray crystal structure of PRMT3 in complex with the inhibitor <b>14u</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HSG">4HSG</a>). (A) Compound <b>14u</b> binds at the allosteric site of PRMT3 and contacts both subunits of the homodimer, as was observed with compound <b>1</b>.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(57)</a> (B) Critical interactions include hydrogen bonds with T466, E422, and R396. The letter preceding the residue number indicates the PRMT3 chain (A or B).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3018332&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/medium/jm-2012-018332_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0005.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Monte Carlo energy minimization identifies a low-energy conformation of K392, which interacts with the carbonyl oxygen of compound <b>14u</b> via a hydrogen bond. Three independent energy minimization simulations (conformational samplings 1–3) converge toward similar low-energy conformations for K392. Conformational states placing the ε-nitrogen of K392 within hydrogen-bonding distance of <b>14u</b> are listed in bold. The four lowest energy states from sampling 1 (A–D) and the fourth state from sampling 2 (E) are shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-5/jm3018332/production/images/large/jm-2012-018332_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3018332&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i76">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26648" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26648" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 65 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Epigenetic protein families: a new frontier for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">384</span><span class="NLM_x">–</span> <span class="NLM_lpage">400</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1038%2Fnrd3674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=22498752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=384-400&author=C.+H.+Arrowsmithauthor=C.+Bountraauthor=P.+V.+Fishauthor=K.+Leeauthor=M.+Schapira&title=Epigenetic+protein+families%3A+a+new+frontier+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic protein families: a new frontier for drug discovery</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Bountra, Chas; Fish, Paul V.; Lee, Kevin; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">384-400</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetic regulation of gene expression is a dynamic and reversible process that establishes normal cellular phenotypes but also contributes to human diseases.  At the mol. level, epigenetic regulation involves hierarchical covalent modification of DNA and the proteins that package DNA, such as histones.  Here, we review the key protein families that mediate epigenetic signalling through the acetylation and methylation of histones, including histone deacetylases, protein methyltransferases, lysine demethylases, bromodomain-contg. proteins and proteins that bind to methylated histones.  These protein families are emerging as druggable classes of enzymes and druggable classes of protein-protein interaction domains.  In this article, we discuss the known links with disease, basic mol. mechanisms of action and recent progress in the pharmacol. modulation of each class of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcD99eCigNtLVg90H21EOLACvtfcHk0lganfVG3P8OwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D&md5=eb82034466a43107aa74a18ccf6d29f5</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd3674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3674%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DEpigenetic%2520protein%2520families%253A%2520a%2520new%2520frontier%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D384%26epage%3D400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solomon, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span> </span><span class="NLM_article-title">Protein methyltransferases as a target class for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">724</span><span class="NLM_x">–</span> <span class="NLM_lpage">732</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1038%2Fnrd2974" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=19721445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVKmsrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=724-732&author=R.+A.+Copelandauthor=M.+E.+Solomonauthor=V.+M.+Richon&title=Protein+methyltransferases+as+a+target+class+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Protein methyltransferases as a target class for drug discovery</span></div><div class="casAuthors">Copeland, Robert A.; Solomon, Michael E.; Richon, Victoria M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">724-732</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The protein methyltransferases (PMTs) - which methylate protein lysine and arginine residues and have crucial roles in gene transcription - are emerging as an important group of enzymes that play key parts in normal physiol. and human diseases.  The collection of human PMTs is a large and diverse group of enzymes that have a common mechanism of catalysis.  Here, we review the biol., biochem. and structural data that together present PMTs as a novel, chem. tractable target class for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpySXIqKk9Lw7Vg90H21EOLACvtfcHk0ljrGamXN_tuYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVKmsrvI&md5=0626e14396442b47c7da2aaa6a0ac6e3</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd2974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2974%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DSolomon%26aufirst%3DM.%2BE.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26atitle%3DProtein%2520methyltransferases%2520as%2520a%2520target%2520class%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26spage%3D724%26epage%3D732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richard, S.</span><span> </span><span class="NLM_article-title">Arginine methylation an emerging regulator of protein function</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">263</span><span class="NLM_x">–</span> <span class="NLM_lpage">272</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1016%2Fj.molcel.2005.04.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=15866169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkt1Cgtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2005&pages=263-272&author=M.+T.+Bedfordauthor=S.+Richard&title=Arginine+methylation+an+emerging+regulator+of+protein+function"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Arginine methylation: An emerging regulator of protein function</span></div><div class="casAuthors">Bedford, Mark T.; Richard, Stephane</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">263-272</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Arginine methylation is now coming out of the shadows of protein phosphorylation and entering the mainstream, largely due to the identification of the family of enzymes that lay down this modification.  In addn., modification-specific antibodies and proteomic approaches have facilitated the identification of an array of substrates for protein arginine methyltransferases.  This review describes recent insights into the mol. processes regulated by arginine methylation in normal and diseased cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDYRpc_643mbVg90H21EOLACvtfcHk0ljrGamXN_tuYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXkt1Cgtb4%253D&md5=63525cb1e96138be5563fbd897c2bde1</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2005.04.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2005.04.003%26sid%3Dliteratum%253Aachs%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DRichard%26aufirst%3DS.%26atitle%3DArginine%2520methylation%2520an%2520emerging%2520regulator%2520of%2520protein%2520function%26jtitle%3DMol.%2520Cell%26date%3D2005%26volume%3D18%26spage%3D263%26epage%3D272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Di Lorenzo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">Histone arginine methylation</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">585</span><span class="NLM_x">, </span> <span class="NLM_fpage">2024</span><span class="NLM_x">–</span> <span class="NLM_lpage">2031</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=585&publication_year=2011&pages=2024-2031&author=A.+Di+Lorenzoauthor=M.+T.+Bedford&title=Histone+arginine+methylation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BLorenzo%26aufirst%3DA.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DHistone%2520arginine%2520methylation%26jtitle%3DFEBS%2520Lett.%26date%3D2011%26volume%3D585%26spage%3D2024%26epage%3D2031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Yost, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Targets in epigenetics: inhibiting the methyl writers of the histone code</span> <span class="citation_source-journal">Curr. Chem. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">72</span><span class="NLM_x">–</span> <span class="NLM_lpage">84</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.2174%2F1875397301005010072" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=21966347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1ent7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2011&pages=72-84&author=J.+M.+Yostauthor=I.+Korboukhauthor=F.+Liuauthor=C.+Gaoauthor=J.+Jin&title=Targets+in+epigenetics%3A+inhibiting+the+methyl+writers+of+the+histone+code"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Targets in epigenetics: inhibiting the methyl writers of the histone code</span></div><div class="casAuthors">Yost, Julianne M.; Korboukh, Ilia; Liu, Feng; Gao, Cen; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Current Chemical Genomics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">72-84</span>CODEN:
                <span class="NLM_cas:coden">CCGUAQ</span>;
        ISSN:<span class="NLM_cas:issn">1875-3973</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Growing evidence suggests that protein lysine methyltransferases (PKMTs) and protein arginine methyltransferases (PRMTs) are assocd. with the development of various human diseases, including cancer, inflammation, and psychiatric disorders.  Given the significant role of these proteins in human disease, efforts to discover selective small-mol. inhibitors of these enzymes are quickly gaining momentum.  In this review, we focus on the recent progress in the discovery of selective PKMT and PRMT inhibitors.  A future perspective on developing methyltransferase inhibitors is also offered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHi_nKenm0brVg90H21EOLACvtfcHk0liQQrltZf0bzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1ent7nP&md5=996a0411f27dcb4db88e5a4885439d94</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.2174%2F1875397301005010072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1875397301005010072%26sid%3Dliteratum%253Aachs%26aulast%3DYost%26aufirst%3DJ.%2BM.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DTargets%2520in%2520epigenetics%253A%2520inhibiting%2520the%2520methyl%2520writers%2520of%2520the%2520histone%2520code%26jtitle%3DCurr.%2520Chem.%2520Genomics%26date%3D2011%26volume%3D5%26spage%3D72%26epage%3D84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Zurita-Lopez, C. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandberg, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, S. G.</span><span> </span><span class="NLM_article-title">Human protein arginine methyltransferase 7 (PRMT7) is a type III enzyme forming omega-NG-monomethylated arginine residues</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">287</span><span class="NLM_x">, </span> <span class="NLM_fpage">7859</span><span class="NLM_x">–</span> <span class="NLM_lpage">7870</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=7859-7870&author=C.+I.+Zurita-Lopezauthor=T.+Sandbergauthor=R.+Kellyauthor=S.+G.+Clarke&title=Human+protein+arginine+methyltransferase+7+%28PRMT7%29+is+a+type+III+enzyme+forming+omega-NG-monomethylated+arginine+residues"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZurita-Lopez%26aufirst%3DC.%2BI.%26aulast%3DSandberg%26aufirst%3DT.%26aulast%3DKelly%26aufirst%3DR.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DHuman%2520protein%2520arginine%2520methyltransferase%25207%2520%2528PRMT7%2529%2520is%2520a%2520type%2520III%2520enzyme%2520forming%2520omega-NG-monomethylated%2520arginine%2520residues%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D7859%26epage%3D7870" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Jansson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durant, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheahan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edelmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N. B.</span><span> </span><span class="NLM_article-title">Arginine methylation regulates the p53 response</span> <span class="citation_source-journal">Nat. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1431</span><span class="NLM_x">–</span> <span class="NLM_lpage">1439</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1038%2Fncb1802" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=19011621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVentbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2008&pages=1431-1439&author=M.+Janssonauthor=S.+T.+Durantauthor=E.+C.+Choauthor=S.+Sheahanauthor=M.+Edelmannauthor=B.+Kesslerauthor=N.+B.+La+Thangue&title=Arginine+methylation+regulates+the+p53+response"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Arginine methylation regulates the p53 response</span></div><div class="casAuthors">Jansson, Martin; Durant, Stephen T.; Cho, Er-Chieh; Sheahan, Sharon; Edelmann, Mariola; Kessler, Benedikt; La Thangue, Nicholas B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1431-1439</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Activation of the p53 tumor suppressor protein in response to DNA damage leads to apoptosis or cell-cycle arrest.  Enzymic modifications are widely believed to affect and regulate p53 activity.  We describe here a level of post-translational control that has an important functional consequence on the p53 response.  We show that the protein arginine methyltransferase (PRMT) 5, as a co-factor in a DNA damage responsive Strap (stress-responsive activator of p300)-contg. co-activator complex that interacts with p53, is responsible for methylating p53.  Arginine methylation is regulated during the p53 response and affects the target gene specificity of p53.  Furthermore, PRMT5 depletion triggers p53-dependent apoptosis.  Thus, methylation on arginine residues is an underlying mechanism of control during the p53 response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa6VSI-iJwVLVg90H21EOLACvtfcHk0ljmKEAlXmE05A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVentbfN&md5=4e01b57e90fd6ae7622e6cea3be498ce</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fncb1802&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb1802%26sid%3Dliteratum%253Aachs%26aulast%3DJansson%26aufirst%3DM.%26aulast%3DDurant%26aufirst%3DS.%2BT.%26aulast%3DCho%26aufirst%3DE.%2BC.%26aulast%3DSheahan%26aufirst%3DS.%26aulast%3DEdelmann%26aufirst%3DM.%26aulast%3DKessler%26aufirst%3DB.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26atitle%3DArginine%2520methylation%2520regulates%2520the%2520p53%2520response%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2008%26volume%3D10%26spage%3D1431%26epage%3D1439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">Arginine methylation at a glance</span> <span class="citation_source-journal">J. Cell Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">120</span><span class="NLM_x">, </span> <span class="NLM_fpage">4243</span><span class="NLM_x">–</span> <span class="NLM_lpage">4246</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1242%2Fjcs.019885" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=18057026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD1cXns1GktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2007&pages=4243-4246&author=M.+T.+Bedford&title=Arginine+methylation+at+a+glance"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Arginine methylation at a glance</span></div><div class="casAuthors">Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">4243-4246</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">A review on the enzymes involved in methylation of arginine is presented.  Many different isoenzymes of the PRMT (protein arginine N Me transferase) from PRMT1 to PRMT9 are discussed.  Some of the substrates of the PRMT enzymes are also discussed along with a brief discussion of the role of protein protein interactions in regulation of arginine methylation is also given.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVWHJj0lIjM7Vg90H21EOLACvtfcHk0ljmKEAlXmE05A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXns1GktA%253D%253D&md5=d6adb4f9193c22342c1d30f16a468590</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1242%2Fjcs.019885&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.019885%26sid%3Dliteratum%253Aachs%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DArginine%2520methylation%2520at%2520a%2520glance%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2007%26volume%3D120%26spage%3D4243%26epage%3D4246" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Pahlich, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bschir, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiavi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belyanskaya, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gehring, H.</span><span> </span><span class="NLM_article-title">Different methylation characteristics of protein arginine methyltransferase 1 and 3 toward the Ewing sarcoma protein and a peptide</span> <span class="citation_source-journal">Proteins: Struct., Funct., Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">164</span><span class="NLM_x">–</span> <span class="NLM_lpage">175</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=164-175&author=S.+Pahlichauthor=K.+Bschirauthor=C.+Chiaviauthor=L.+Belyanskayaauthor=H.+Gehring&title=Different+methylation+characteristics+of+protein+arginine+methyltransferase+1+and+3+toward+the+Ewing+sarcoma+protein+and+a+peptide"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPahlich%26aufirst%3DS.%26aulast%3DBschir%26aufirst%3DK.%26aulast%3DChiavi%26aufirst%3DC.%26aulast%3DBelyanskaya%26aufirst%3DL.%26aulast%3DGehring%26aufirst%3DH.%26atitle%3DDifferent%2520methylation%2520characteristics%2520of%2520protein%2520arginine%2520methyltransferase%25201%2520and%25203%2520toward%2520the%2520Ewing%2520sarcoma%2520protein%2520and%2520a%2520peptide%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Bioinf.%26date%3D2005%26volume%3D61%26spage%3D164%26epage%3D175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Cheng, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cote, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaaban, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">The arginine methyltransferase CARM1 regulates the coupling of transcription and mRNA processing</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">83</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1016%2Fj.molcel.2006.11.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=17218272" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFSnur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=71-83&author=D.+H.+Chengauthor=J.+Coteauthor=S.+Shaabanauthor=M.+T.+Bedford&title=The+arginine+methyltransferase+CARM1+regulates+the+coupling+of+transcription+and+mRNA+processing"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The arginine methyltransferase CARM1 regulates the coupling of transcription and mRNA processing</span></div><div class="casAuthors">Cheng, Donghang; Cote, Jocelyn; Shaaban, Salam; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-83</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The coactivator-assocd. arginine methyl-transferase CARM1 is recruited by many different transcription factors as a pos. regulator.  To understand the mechanism by which CARM1 functions, we sought to isolate its substrates.  We developed a small-pool screening approach for this purpose and identified CA150, SAP49, SmB, and U1C as splicing factors that are specifically methylated by CARM1.  We further showed that CA150, a mol. that links transcription to splicing, interacts with the Tudor domain of the spinal muscular atrophy protein SMN in a CARM1-dependent fashion.  Expts. with an exogenous splicing reporter and the endogenous CD44 gene revealed that CARM1 promotes exon skipping in an enzyme-dependent manner.  The identification of splicing factors that are methylated by CARM1, and protein-protein interactions that are regulated by CARM1, strongly implicates this enzyme in the regulation of alternative splicing and points toward its involvement in spinal muscular atrophy pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqulZNAke_ZsrVg90H21EOLACvtfcHk0lja6gP7g8XflA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFSnur8%253D&md5=29f6dccb415e1146e6455a03613f9fa2</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.11.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.11.019%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DD.%2BH.%26aulast%3DCote%26aufirst%3DJ.%26aulast%3DShaaban%26aufirst%3DS.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DThe%2520arginine%2520methyltransferase%2520CARM1%2520regulates%2520the%2520coupling%2520of%2520transcription%2520and%2520mRNA%2520processing%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D25%26spage%3D71%26epage%3D83" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reed, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leder, P.</span><span> </span><span class="NLM_article-title">WW domain-mediated interactions reveal a spliceosome-associated protein that binds a third class of proline-rich motif: the proline glycine and methionine-rich motif</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">95</span><span class="NLM_x">, </span> <span class="NLM_fpage">10602</span><span class="NLM_x">–</span> <span class="NLM_lpage">10607</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1998&pages=10602-10607&author=M.+T.+Bedfordauthor=R.+Reedauthor=P.+Leder&title=WW+domain-mediated+interactions+reveal+a+spliceosome-associated+protein+that+binds+a+third+class+of+proline-rich+motif%3A+the+proline+glycine+and+methionine-rich+motif"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DReed%26aufirst%3DR.%26aulast%3DLeder%26aufirst%3DP.%26atitle%3DWW%2520domain-mediated%2520interactions%2520reveal%2520a%2520spliceosome-associated%2520protein%2520that%2520binds%2520a%2520third%2520class%2520of%2520proline-rich%2520motif%253A%2520the%2520proline%2520glycine%2520and%2520methionine-rich%2520motif%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1998%26volume%3D95%26spage%3D10602%26epage%3D10607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Branscombe, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frankel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pestka, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, S.</span><span> </span><span class="NLM_article-title">PRMT5 (Janus kinase-binding protein 1) catalyzes the formation of symmetric dimethylarginine residues in proteins</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">32971</span><span class="NLM_x">–</span> <span class="NLM_lpage">32976</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=32971-32976&author=T.+L.+Branscombeauthor=A.+Frankelauthor=J.+H.+Leeauthor=J.+R.+Cookauthor=Z.+Yangauthor=S.+Pestkaauthor=S.+Clarke&title=PRMT5+%28Janus+kinase-binding+protein+1%29+catalyzes+the+formation+of+symmetric+dimethylarginine+residues+in+proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBranscombe%26aufirst%3DT.%2BL.%26aulast%3DFrankel%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DCook%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DPestka%26aufirst%3DS.%26aulast%3DClarke%26aufirst%3DS.%26atitle%3DPRMT5%2520%2528Janus%2520kinase-binding%2520protein%25201%2529%2520catalyzes%2520the%2520formation%2520of%2520symmetric%2520dimethylarginine%2520residues%2520in%2520proteins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D32971%26epage%3D32976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Tang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gary, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herschman, H. R.</span><span> </span><span class="NLM_article-title">PRMT 3, a type I protein arginine N-methyltransferase that differs from PRMT1 in its oligomerization, subcellular localization, substrate specificity, and regulation</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">273</span><span class="NLM_x">, </span> <span class="NLM_fpage">16935</span><span class="NLM_x">–</span> <span class="NLM_lpage">16945</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1998&pages=16935-16945&author=J.+Tangauthor=J.+D.+Garyauthor=S.+Clarkeauthor=H.+R.+Herschman&title=PRMT+3%2C+a+type+I+protein+arginine+N-methyltransferase+that+differs+from+PRMT1+in+its+oligomerization%2C+subcellular+localization%2C+substrate+specificity%2C+and+regulation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DGary%26aufirst%3DJ.%2BD.%26aulast%3DClarke%26aufirst%3DS.%26aulast%3DHerschman%26aufirst%3DH.%2BR.%26atitle%3DPRMT%25203%252C%2520a%2520type%2520I%2520protein%2520arginine%2520N-methyltransferase%2520that%2520differs%2520from%2520PRMT1%2520in%2520its%2520oligomerization%252C%2520subcellular%2520localization%252C%2520substrate%2520specificity%252C%2520and%2520regulation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1998%26volume%3D273%26spage%3D16935%26epage%3D16945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Bachand, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silver, P. A.</span><span> </span><span class="NLM_article-title">PRMT3 is a ribosomal protein methyltransferase that affects the cellular levels of ribosomal subunits</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">2641</span><span class="NLM_x">–</span> <span class="NLM_lpage">2650</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=2641-2650&author=F.+Bachandauthor=P.+A.+Silver&title=PRMT3+is+a+ribosomal+protein+methyltransferase+that+affects+the+cellular+levels+of+ribosomal+subunits"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBachand%26aufirst%3DF.%26aulast%3DSilver%26aufirst%3DP.%2BA.%26atitle%3DPRMT3%2520is%2520a%2520ribosomal%2520protein%2520methyltransferase%2520that%2520affects%2520the%2520cellular%2520levels%2520of%2520ribosomal%2520subunits%26jtitle%3DEMBO%2520J.%26date%3D2004%26volume%3D23%26spage%3D2641%26epage%3D2650" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Swiercz, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Person, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">Ribosomal protein S2 is a substrate for mammalian PRMT3 (protein arginine methyltransferase 3)</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">386</span><span class="NLM_x">, </span> <span class="NLM_fpage">85</span><span class="NLM_x">–</span> <span class="NLM_lpage">91</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=386&publication_year=2005&pages=85-91&author=R.+Swierczauthor=M.+D.+Personauthor=M.+T.+Bedford&title=Ribosomal+protein+S2+is+a+substrate+for+mammalian+PRMT3+%28protein+arginine+methyltransferase+3%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSwiercz%26aufirst%3DR.%26aulast%3DPerson%26aufirst%3DM.%2BD.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DRibosomal%2520protein%2520S2%2520is%2520a%2520substrate%2520for%2520mammalian%2520PRMT3%2520%2528protein%2520arginine%2520methyltransferase%25203%2529%26jtitle%3DBiochem.%2520J.%26date%3D2005%26volume%3D386%26spage%3D85%26epage%3D91" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Swiercz, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">Ribosomal protein rpS2 is hypomethylated in PRMT3-deficient mice</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">16917</span><span class="NLM_x">–</span> <span class="NLM_lpage">16923</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1074%2Fjbc.M609778200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=17439947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmt1agsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=16917-16923&author=R.+Swierczauthor=D.+Chengauthor=D.+Kimauthor=M.+T.+Bedford&title=Ribosomal+protein+rpS2+is+hypomethylated+in+PRMT3-deficient+mice"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Ribosomal Protein rpS2 Is Hypomethylated in PRMT3-deficient Mice</span></div><div class="casAuthors">Swiercz, Rafal; Cheng, Donghang; Kim, Daehoon; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">16917-16923</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">PRMT3 is a type I arginine methyltransferase that resides in the cytoplasm.  A large proportion of this cytosolic PRMT3 is found assocd. with ribosomes.  It is tethered to the ribosomes through its interaction with rpS2, which is also its substrate.  Here we show that mouse embryos with a targeted disruption of PRMT3 are small in size but survive after birth and attain a normal size in adulthood, thus displaying Minute-like characteristics.  The ribosome protein rpS2 is hypomethylated in the absence of PRMT3, demonstrating that it is a bona fide, in vivo PRMT3 substrate that cannot be modified by other PRMTs.  Finally, the levels 40 S, 60 S, and 80 S monosomes and polyribosomes are unaffected by the loss of PRMT3, but there are addnl. as yet unidentified proteins that co-fractionate with ribosomes that are also dedicated PRMT3 substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-lEvztlUF4rVg90H21EOLACvtfcHk0lh-gZeRf7wmYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmt1agsLw%253D&md5=4d97b44b6706559086aa2554df786b31</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M609778200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M609778200%26sid%3Dliteratum%253Aachs%26aulast%3DSwiercz%26aufirst%3DR.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DRibosomal%2520protein%2520rpS2%2520is%2520hypomethylated%2520in%2520PRMT3-deficient%2520mice%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D16917%26epage%3D16923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rucknagel, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schierhorn, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nemeth, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linder, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herschman, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahle, E.</span><span> </span><span class="NLM_article-title">Unusual sites of arginine methylation in poly(A)-binding protein II and in vitro methylation by protein arginine methyltransferases PRMT1 and PRMT3</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">274</span><span class="NLM_x">, </span> <span class="NLM_fpage">13229</span><span class="NLM_x">–</span> <span class="NLM_lpage">13234</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1999&pages=13229-13234&author=J.+J.+Smithauthor=K.+P.+Rucknagelauthor=A.+Schierhornauthor=J.+Tangauthor=A.+Nemethauthor=M.+Linderauthor=H.+R.+Herschmanauthor=E.+Wahle&title=Unusual+sites+of+arginine+methylation+in+poly%28A%29-binding+protein+II+and+in+vitro+methylation+by+protein+arginine+methyltransferases+PRMT1+and+PRMT3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DRucknagel%26aufirst%3DK.%2BP.%26aulast%3DSchierhorn%26aufirst%3DA.%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DNemeth%26aufirst%3DA.%26aulast%3DLinder%26aufirst%3DM.%26aulast%3DHerschman%26aufirst%3DH.%2BR.%26aulast%3DWahle%26aufirst%3DE.%26atitle%3DUnusual%2520sites%2520of%2520arginine%2520methylation%2520in%2520poly%2528A%2529-binding%2520protein%2520II%2520and%2520in%2520vitro%2520methylation%2520by%2520protein%2520arginine%2520methyltransferases%2520PRMT1%2520and%2520PRMT3%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1999%26volume%3D274%26spage%3D13229%26epage%3D13234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Fronz, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolbel, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrich, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schierhorn, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beck-Sickinger, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ostareck-Lederer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahle, E.</span><span> </span><span class="NLM_article-title">Promiscuous modification of the nuclear poly(A)-binding protein by multiple protein-arginine methyltransferases does not affect the aggregation behavior</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">20408</span><span class="NLM_x">–</span> <span class="NLM_lpage">20420</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=20408-20420&author=K.+Fronzauthor=S.+Ottoauthor=K.+Kolbelauthor=U.+Kuhnauthor=H.+Friedrichauthor=A.+Schierhornauthor=A.+G.+Beck-Sickingerauthor=A.+Ostareck-Ledererauthor=E.+Wahle&title=Promiscuous+modification+of+the+nuclear+poly%28A%29-binding+protein+by+multiple+protein-arginine+methyltransferases+does+not+affect+the+aggregation+behavior"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFronz%26aufirst%3DK.%26aulast%3DOtto%26aufirst%3DS.%26aulast%3DKolbel%26aufirst%3DK.%26aulast%3DKuhn%26aufirst%3DU.%26aulast%3DFriedrich%26aufirst%3DH.%26aulast%3DSchierhorn%26aufirst%3DA.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26aulast%3DOstareck-Lederer%26aufirst%3DA.%26aulast%3DWahle%26aufirst%3DE.%26atitle%3DPromiscuous%2520modification%2520of%2520the%2520nuclear%2520poly%2528A%2529-binding%2520protein%2520by%2520multiple%2520protein-arginine%2520methyltransferases%2520does%2520not%2520affect%2520the%2520aggregation%2520behavior%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D20408%26epage%3D20420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Tavanez, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bengoechea, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berciano, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lafarga, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carmo-Fonseca, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Enguita, F. J.</span><span> </span><span class="NLM_article-title">Hsp70 chaperones and type I PRMTs are sequestered at intranuclear inclusions caused by polyalanine expansions in PABPN1</span> <span class="citation_source-journal">PloS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">e6418</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=e6418&author=J.+P.+Tavanezauthor=R.+Bengoecheaauthor=M.+T.+Bercianoauthor=M.+Lafargaauthor=M.+Carmo-Fonsecaauthor=F.+J.+Enguita&title=Hsp70+chaperones+and+type+I+PRMTs+are+sequestered+at+intranuclear+inclusions+caused+by+polyalanine+expansions+in+PABPN1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTavanez%26aufirst%3DJ.%2BP.%26aulast%3DBengoechea%26aufirst%3DR.%26aulast%3DBerciano%26aufirst%3DM.%2BT.%26aulast%3DLafarga%26aufirst%3DM.%26aulast%3DCarmo-Fonseca%26aufirst%3DM.%26aulast%3DEnguita%26aufirst%3DF.%2BJ.%26atitle%3DHsp70%2520chaperones%2520and%2520type%2520I%2520PRMTs%2520are%2520sequestered%2520at%2520intranuclear%2520inclusions%2520caused%2520by%2520polyalanine%2520expansions%2520in%2520PABPN1%26jtitle%3DPloS%2520One%26date%3D2009%26volume%3D4%26spage%3De6418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">Fluorescence-based methods for screening writers and readers of histone methyl marks</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">84</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1177%2F1087057111422256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=21972038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC38Xit1Olt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=71-84&author=A.+Allali-Hassaniauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=T.+Hajianauthor=C.+H.+Arrowsmithauthor=M.+Vedadi&title=Fluorescence-based+methods+for+screening+writers+and+readers+of+histone+methyl+marks"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence-based methods for screening writers and readers of histone methyl marks</span></div><div class="casAuthors">Allali-Hassani, Abdellah; Wasney, Gregory A.; Siarheyeva, Alena; Hajian, Taraneh; Arrowsmith, Cheryl H.; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-84</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The histone methyltransferase (HMT) family of proteins consists of enzymes that methylate lysine or arginine residues on histone tails as well as other proteins.  Such modifications affect chromatin structure and play a significant regulatory role in gene expression.  Many HMTs have been implicated in tumorigenesis and progression of multiple malignancies and play essential roles in embryonic development and stem cell renewal.  Overexpression of some HMTs has been obsd. and is correlated pos. with various types of cancer.  Here the authors report development of a continuous fluorescence-based methyltransferase assay in a 384-well format and its application in detg. kinetic parameters for EHMT1, G9a, PRMT3, SETD7, and SUV39H2 as well as for screening against libraries of small mols. to identify enzyme inhibitors.  They also report the development of a peptide displacement assay using fluorescence polarization in a 384-well format to assay and screen protein peptide interactions such as those of WDR5 and EED, components of MLL and EZH2 methyltransferase complexes.  Using these high-throughput screening methods, the authors have identified potent inhibitors and ligands for some of these proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0s8Ex9rZOXrVg90H21EOLACvtfcHk0lhU0Y2XD8-e_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xit1Olt7g%253D&md5=105a20c2473ab603ade42a38764def09</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1177%2F1087057111422256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057111422256%26sid%3Dliteratum%253Aachs%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DFluorescence-based%2520methods%2520for%2520screening%2520writers%2520and%2520readers%2520of%2520histone%2520methyl%2520marks%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2012%26volume%3D17%26spage%3D71%26epage%3D84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Lai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hakala, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weintraub, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiio, Y.</span><span> </span><span class="NLM_article-title">Proteomic dissection of the von Hippel-Lindau (VHL) interactome</span> <span class="citation_source-journal">J. Proteome Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">5175</span><span class="NLM_x">–</span> <span class="NLM_lpage">5182</span></span><div class="citationLinks">[<a href="/doi/10.1021/pr200642c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht12jsbzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=5175-5182&author=Y.+Laiauthor=M.+Songauthor=K.+Hakalaauthor=S.+T.+Weintraubauthor=Y.+Shiio&title=Proteomic+dissection+of+the+von+Hippel-Lindau+%28VHL%29+interactome"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic Dissection of the von Hippel-Lindau (VHL) Interactome</span></div><div class="casAuthors">Lai, Yanlai; Song, Meihua; Hakala, Kevin; Weintraub, Susan T.; Shiio, Yuzuru</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5175-5182</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The von Hippel-Lindau (VHL) tumor suppressor gene encodes a component of a ubiquitin ligase complex contg. elongin B, elongin C, cullin 2, and Rbx1, which acts as a neg. regulator of hypoxia inducible factor (HIF).  VHL ubiquitinates and degrades the alpha subunits of HIF, and this is proposed to suppress tumorigenesis and tumor angiogenesis.  Several lines of evidence also suggest important roles for HIF-independent VHL functions in the maintenance of primary cilium, extracellular matrix formation, and tumor suppression.  We undertook a series of proteomic analyses to gain a comprehensive picture of the VHL-interacting proteins.  We found that the ARF tumor suppressor interacts with VHL30, a longer VHL isoform, but not with VHL19, a shorter VHL isoform.  ARF was found to release VHL30 from the E3 ligase complex, promoting the binding of VHL30 to a protein arginine methyltransferase, PRMT3.  Our anal. of the VHL19 interactome also uncovered that VHL19 displays an affinity to collagens and their biosynthesis enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqTYZ6RvZ7erVg90H21EOLACvtfcHk0lgM2-7Zk8dfKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht12jsbzL&md5=51500af3ec4256311dbd835c47a90200</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fpr200642c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr200642c%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DM.%26aulast%3DHakala%26aufirst%3DK.%26aulast%3DWeintraub%26aufirst%3DS.%2BT.%26aulast%3DShiio%26aufirst%3DY.%26atitle%3DProteomic%2520dissection%2520of%2520the%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520interactome%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2011%26volume%3D10%26spage%3D5175%26epage%3D5182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Singh, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miranda, T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frankel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roemer, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robb, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gutmann, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herschman, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newsham, I. F.</span><span> </span><span class="NLM_article-title">DAL-1/4.1B tumor suppressor interacts with protein arginine N-methyltransferase 3 (PRMT3) and inhibits its ability to methylate substrates in vitro and in vivo</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">7761</span><span class="NLM_x">–</span> <span class="NLM_lpage">7771</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=7761-7771&author=V.+Singhauthor=T.+B.+Mirandaauthor=W.+Jiangauthor=A.+Frankelauthor=M.+E.+Roemerauthor=V.+A.+Robbauthor=D.+H.+Gutmannauthor=H.+R.+Herschmanauthor=S.+Clarkeauthor=I.+F.+Newsham&title=DAL-1%2F4.1B+tumor+suppressor+interacts+with+protein+arginine+N-methyltransferase+3+%28PRMT3%29+and+inhibits+its+ability+to+methylate+substrates+in+vitro+and+in+vivo"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DV.%26aulast%3DMiranda%26aufirst%3DT.%2BB.%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DFrankel%26aufirst%3DA.%26aulast%3DRoemer%26aufirst%3DM.%2BE.%26aulast%3DRobb%26aufirst%3DV.%2BA.%26aulast%3DGutmann%26aufirst%3DD.%2BH.%26aulast%3DHerschman%26aufirst%3DH.%2BR.%26aulast%3DClarke%26aufirst%3DS.%26aulast%3DNewsham%26aufirst%3DI.%2BF.%26atitle%3DDAL-1%252F4.1B%2520tumor%2520suppressor%2520interacts%2520with%2520protein%2520arginine%2520N-methyltransferase%25203%2520%2528PRMT3%2529%2520and%2520inhibits%2520its%2520ability%2520to%2520methylate%2520substrates%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DOncogene%26date%3D2004%26volume%3D23%26spage%3D7761%26epage%3D7771" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niroomand, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zankl, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katus, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jahn, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tiefenbacher, C. P.</span><span> </span><span class="NLM_article-title">Expression of nitric oxide related enzymes in coronary heart disease</span> <span class="citation_source-journal">Basic Res. Cardiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">346</span><span class="NLM_x">–</span> <span class="NLM_lpage">353</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1007%2Fs00395-006-0592-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=16705470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD28Xos1ylurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2006&pages=346-353&author=X.+Chenauthor=F.+Niroomandauthor=Z.+Liuauthor=A.+Zanklauthor=H.+A.+Katusauthor=L.+Jahnauthor=C.+P.+Tiefenbacher&title=Expression+of+nitric+oxide+related+enzymes+in+coronary+heart+disease"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of nitric oxide related enzymes in coronary heart disease</span></div><div class="casAuthors">Chen, X.; Niroomand, F.; Liu, Z.; Zankl, A.; Katus, H. A.; Jahn, L.; Tiefenbacher, C. P.</div><div class="citationInfo"><span class="NLM_cas:title">Basic Research in Cardiology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">346-353</span>CODEN:
                <span class="NLM_cas:coden">BRCAB7</span>;
        ISSN:<span class="NLM_cas:issn">0300-8428</span>.
    
            (<span class="NLM_cas:orgname">Steinkopff Verlag</span>)
        </div><div class="casAbstract">Enzymes involved in the metab. nitric oxide (NO) and reactive oxygen species (ROS) may play a role for the decreased availability of NO in atherosclerosis.  We, therefore, hypothesized that the pattern of gene expression of these enzymes is altered in atherosclerosis.  Myocardial tissue from patients with coronary heart disease (CHD) or without CHD (control group) was investigated.  The level of enzymes related to NO/ROS metab. was detd. both at mRNA level and protein level by rt-PCR, real-time PCR, and western blot.  The expression of NOS1-3 (synthesis of NO), arginasel (redn. of L-arginine), p22phox (active subunit of NADPH oxidase), GTPCH (rate limiting enzyme for tetrahydrobiopterin), SOD1-3 (scavengers of superoxide anions), PRTMT1-3, and DDAH2 (involved in the metab. of ADMA) was detd.  All enzymes were found to be expressed in human myocardium.  NOS isoforms were decreased in CHD in protein level, but only the down regulation of NOS3 expression reached statistical significance.  The expression of PRMT1 and PRMT3 was increased.  In addn., the expression of DDAH2 was reduced, both theor. leading to an increase of ADMA concn.  SOD3 was down regulated in tissue from patients with CHD.  Taken together, in myocardial tissue from patients with atherosclerosis, the expression of genes increasing ADMA levels is enhanced in contrast to a reduced expression of genes promoting NO synthesis.  These results may contribute to the explanation of increased oxidative stress in atherosclerosis on the level of gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreZhed3XxK3bVg90H21EOLACvtfcHk0lgM2-7Zk8dfKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xos1ylurk%253D&md5=e40c38e03c6dc050a5a894bd0be20155</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1007%2Fs00395-006-0592-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00395-006-0592-5%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DNiroomand%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DZankl%26aufirst%3DA.%26aulast%3DKatus%26aufirst%3DH.%2BA.%26aulast%3DJahn%26aufirst%3DL.%26aulast%3DTiefenbacher%26aufirst%3DC.%2BP.%26atitle%3DExpression%2520of%2520nitric%2520oxide%2520related%2520enzymes%2520in%2520coronary%2520heart%2520disease%26jtitle%3DBasic%2520Res.%2520Cardiol.%26date%3D2006%26volume%3D101%26spage%3D346%26epage%3D353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Miyata, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mori, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tohyama, M.</span><span> </span><span class="NLM_article-title">PRMT3 is essential for dendritic spine maturation in rat hippocampal neurons</span> <span class="citation_source-journal">Brain Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1352</span><span class="NLM_x">, </span> <span class="NLM_fpage">11</span><span class="NLM_x">–</span> <span class="NLM_lpage">20</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1016%2Fj.brainres.2010.07.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=20647003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVKisL3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1352&publication_year=2010&pages=11-20&author=S.+Miyataauthor=Y.+Moriauthor=M.+Tohyama&title=PRMT3+is+essential+for+dendritic+spine+maturation+in+rat+hippocampal+neurons"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT3 is essential for dendritic spine maturation in rat hippocampal neurons</span></div><div class="casAuthors">Miyata, Shingo; Mori, Yasutake; Tohyama, Masaya</div><div class="citationInfo"><span class="NLM_cas:title">Brain Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1352</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11-20</span>CODEN:
                <span class="NLM_cas:coden">BRREAP</span>;
        ISSN:<span class="NLM_cas:issn">0006-8993</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Protein arginine N-methyltransferase 3 (PRMT3) is a cytoplasmic enzyme that utilizes S-adenosyl-L-methionine (AdoMet) to methylate specific proteins, most of which contain GAR (glycine-arginine rich) motifs.  PRMT3 has been shown to play a role in the proper maturation of the 80S ribosome by binding to and catalyzing the methylation of rpS2, a component of the 40S ribosomal subunit.  However, the other roles of PRMT3 are fairly unclear, particularly in the brain, which is abundant in methylated proteins.  In this study, we perturbed PRMT3 expression in cultured rat hippocampal neurons by transiently introducing siRNA oligonucleotides that were designed to hybridize with PRMT3 mRNA and then we examd. the morphol. and functional effects of neuronal PRMT3 depletion.  PRMT3-defective neurons showed deformed spines without any change in spine no.; less BDNF-induced protein translation of αCaMKII; and diminished rpS2 protein stability.  Furthermore, overexpression of a methylation-resistant rpS2, whose methylated arginine residues were deleted, produced phenotypes that were similar to those assocd. with PRMT3 down regulation.  These findings demonstrated that PRMT3 possibly plays a pivotal role in neuronal translation by interaction with rpS2 and that it contributes to activity-dependent changes in the dendritic spines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZvFaoRRY5lbVg90H21EOLACvtfcHk0lg_RXJdJR3eXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVKisL3L&md5=ccac2a5a931c483e516003342916bb17</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.brainres.2010.07.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.brainres.2010.07.033%26sid%3Dliteratum%253Aachs%26aulast%3DMiyata%26aufirst%3DS.%26aulast%3DMori%26aufirst%3DY.%26aulast%3DTohyama%26aufirst%3DM.%26atitle%3DPRMT3%2520is%2520essential%2520for%2520dendritic%2520spine%2520maturation%2520in%2520rat%2520hippocampal%2520neurons%26jtitle%3DBrain%2520Res.%26date%3D2010%26volume%3D1352%26spage%3D11%26epage%3D20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Bissinger, E.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heinke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span> </span><span class="NLM_article-title">Targeting epigenetic modifiers: inhibitors of histone methyltransferases</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">114</span><span class="NLM_x">–</span> <span class="NLM_lpage">124</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=114-124&author=E.-M.+Bissingerauthor=R.+Heinkeauthor=W.+Sipplauthor=M.+Jung&title=Targeting+epigenetic+modifiers%3A+inhibitors+of+histone+methyltransferases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBissinger%26aufirst%3DE.-M.%26aulast%3DHeinke%26aufirst%3DR.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DTargeting%2520epigenetic%2520modifiers%253A%2520inhibitors%2520of%2520histone%2520methyltransferases%26jtitle%3DMedChemComm%26date%3D2010%26volume%3D1%26spage%3D114%26epage%3D124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">He, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span> </span><span class="NLM_article-title">Targeting protein lysine methylation and demethylation in cancers</span> <span class="citation_source-journal">Acta Biochim. Biophys. Sin.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">70</span><span class="NLM_x">–</span> <span class="NLM_lpage">79</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1093%2Fabbs%2Fgmr109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=22194015" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htr7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2012&pages=70-79&author=Y.+Heauthor=I.+Korboukhauthor=J.+Jinauthor=J.+Huang&title=Targeting+protein+lysine+methylation+and+demethylation+in+cancers"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein lysine methylation and demethylation in cancers</span></div><div class="casAuthors">He, Yunlong; Korboukh, Ilia; Jin, Jian; Huang, Jing</div><div class="citationInfo"><span class="NLM_cas:title">Acta Biochimica et Biophysica Sinica</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">70-79</span>CODEN:
                <span class="NLM_cas:coden">ABBSC2</span>;
        ISSN:<span class="NLM_cas:issn">1672-9145</span>.
    
            (<span class="NLM_cas:orgname">ABBS Editorial Office</span>)
        </div><div class="casAbstract">A review.  During the last decade, we saw an explosion of studies investigating the role of lysine methylation/demethylation of histones and non-histone proteins, such as p53, NF-kappaB, and E2F1.  These Ying-Yang' post-translational modifications are important to fine-tuning the activity of these proteins.  Lysine methylation and demethylation are catalyzed by protein lysine methyltransferases (PKMTs) and protein lysine demethylases (PKDMs).  PKMTs, PKDMs, and their substrates have been shown to play important roles in cancers.  Although the underlying mechanisms of tumorigenesis are still largely unknown, growing evidence is starting to link aberrant regulation of methylation to tumorigenesis.  This review focuses on summarizing the recent progress in understanding of the function of protein lysine methylation, and in the discovery of small mol. inhibitors for PKMTs and PKDMs.  We also discuss the potential and the caveats of targeting protein lysine methylation for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO9Pis9NgA27Vg90H21EOLACvtfcHk0lg_RXJdJR3eXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htr7P&md5=bca793734c9f24cf45bbb529a41207db</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1093%2Fabbs%2Fgmr109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fabbs%252Fgmr109%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DY.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DJ.%26atitle%3DTargeting%2520protein%2520lysine%2520methylation%2520and%2520demethylation%2520in%2520cancers%26jtitle%3DActa%2520Biochim.%2520Biophys.%2520Sin.%26date%3D2012%26volume%3D44%26spage%3D70%26epage%3D79" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span> </span><span class="NLM_article-title">Epigenetic chemical probes</span> <span class="citation_source-journal">Clin. Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">92</span><span class="NLM_x">, </span> <span class="NLM_fpage">689</span><span class="NLM_x">–</span> <span class="NLM_lpage">693</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2012&pages=689-693&author=S.+Mullerauthor=P.+J.+Brown&title=Epigenetic+chemical+probes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26atitle%3DEpigenetic%2520chemical%2520probes%26jtitle%3DClin.%2520Pharmacol.%2520Ther.%26date%3D2012%26volume%3D92%26spage%3D689%26epage%3D693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Blancafort, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span> </span><span class="NLM_article-title">Writing and re-writing the epigenetic code of cancer cells: from engineered proteins to small molecules</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">563</span><span class="NLM_x">–</span> <span class="NLM_lpage">576</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2013&pages=563-576&author=P.+Blancafortauthor=J.+Jinauthor=S.+V.+Frye&title=Writing+and+re-writing+the+epigenetic+code+of+cancer+cells%3A+from+engineered+proteins+to+small+molecules"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBlancafort%26aufirst%3DP.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DWriting%2520and%2520re-writing%2520the%2520epigenetic%2520code%2520of%2520cancer%2520cells%253A%2520from%2520engineered%2520proteins%2520to%2520small%2520molecules%26jtitle%3DMol.%2520Pharmacol.%26date%3D2013%26volume%3D83%26spage%3D563%26epage%3D576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Cheng, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yadav, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swanson, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinstein, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">Small molecule regulators of protein arginine methyltransferases</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">279</span><span class="NLM_x">, </span> <span class="NLM_fpage">23892</span><span class="NLM_x">–</span> <span class="NLM_lpage">23899</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=23892-23899&author=D.+H.+Chengauthor=N.+Yadavauthor=R.+W.+Kingauthor=M.+S.+Swansonauthor=E.+J.+Weinsteinauthor=M.+T.+Bedford&title=Small+molecule+regulators+of+protein+arginine+methyltransferases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DD.%2BH.%26aulast%3DYadav%26aufirst%3DN.%26aulast%3DKing%26aufirst%3DR.%2BW.%26aulast%3DSwanson%26aufirst%3DM.%2BS.%26aulast%3DWeinstein%26aufirst%3DE.%2BJ.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DSmall%2520molecule%2520regulators%2520of%2520protein%2520arginine%2520methyltransferases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D23892%26epage%3D23899" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Bonham, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hemmers, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finn, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mowen, K. A.</span><span> </span><span class="NLM_article-title">Effects of a novel arginine methyltransferase inhibitor on T-helper cell cytokine production</span> <span class="citation_source-journal">FEBS J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">277</span><span class="NLM_x">, </span> <span class="NLM_fpage">2096</span><span class="NLM_x">–</span> <span class="NLM_lpage">2108</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2010&pages=2096-2108&author=K.+Bonhamauthor=S.+Hemmersauthor=Y.+H.+Limauthor=D.+M.+Hillauthor=M.+G.+Finnauthor=K.+A.+Mowen&title=Effects+of+a+novel+arginine+methyltransferase+inhibitor+on+T-helper+cell+cytokine+production"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBonham%26aufirst%3DK.%26aulast%3DHemmers%26aufirst%3DS.%26aulast%3DLim%26aufirst%3DY.%2BH.%26aulast%3DHill%26aufirst%3DD.%2BM.%26aulast%3DFinn%26aufirst%3DM.%2BG.%26aulast%3DMowen%26aufirst%3DK.%2BA.%26atitle%3DEffects%2520of%2520a%2520novel%2520arginine%2520methyltransferase%2520inhibitor%2520on%2520T-helper%2520cell%2520cytokine%2520production%26jtitle%3DFEBS%2520J.%26date%3D2010%26volume%3D277%26spage%3D2096%26epage%3D2108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Trapp, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meier, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hongwiset, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kassack, M. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span> </span><span class="NLM_article-title">Structure-activity studies on suramin analogues as inhibitors of NAD(+)-dependent histone deacetylases (sirtuins)</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">1419</span><span class="NLM_x">–</span> <span class="NLM_lpage">1431</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=1419-1431&author=J.+Trappauthor=R.+Meierauthor=D.+Hongwisetauthor=M.+U.+Kassackauthor=W.+Sipplauthor=M.+Jung&title=Structure-activity+studies+on+suramin+analogues+as+inhibitors+of+NAD%28%2B%29-dependent+histone+deacetylases+%28sirtuins%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTrapp%26aufirst%3DJ.%26aulast%3DMeier%26aufirst%3DR.%26aulast%3DHongwiset%26aufirst%3DD.%26aulast%3DKassack%26aufirst%3DM.%2BU.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DStructure-activity%2520studies%2520on%2520suramin%2520analogues%2520as%2520inhibitors%2520of%2520NAD%2528%252B%2529-dependent%2520histone%2520deacetylases%2520%2528sirtuins%2529%26jtitle%3DChemMedChem%26date%3D2007%26volume%3D2%26spage%3D1419%26epage%3D1431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Ragno, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeoni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castellano, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vicidomini, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caroli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tramontano, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonaccini, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sbardella, G.</span><span> </span><span class="NLM_article-title">Small molecule inhibitors of histone arginine methyltransferases: homology modeling, molecular docking, binding mode analysis, and biological evaluations</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1241</span><span class="NLM_x">–</span> <span class="NLM_lpage">1253</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm061213n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=1241-1253&author=R.+Ragnoauthor=S.+Simeoniauthor=S.+Castellanoauthor=C.+Vicidominiauthor=A.+Maiauthor=A.+Caroliauthor=A.+Tramontanoauthor=C.+Bonacciniauthor=P.+Trojerauthor=I.+Bauerauthor=G.+Broschauthor=G.+Sbardella&title=Small+molecule+inhibitors+of+histone+arginine+methyltransferases%3A+homology+modeling%2C+molecular+docking%2C+binding+mode+analysis%2C+and+biological+evaluations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm061213n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061213n%26sid%3Dliteratum%253Aachs%26aulast%3DRagno%26aufirst%3DR.%26aulast%3DSimeoni%26aufirst%3DS.%26aulast%3DCastellano%26aufirst%3DS.%26aulast%3DVicidomini%26aufirst%3DC.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DCaroli%26aufirst%3DA.%26aulast%3DTramontano%26aufirst%3DA.%26aulast%3DBonaccini%26aufirst%3DC.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DBauer%26aufirst%3DI.%26aulast%3DBrosch%26aufirst%3DG.%26aulast%3DSbardella%26aufirst%3DG.%26atitle%3DSmall%2520molecule%2520inhibitors%2520of%2520histone%2520arginine%2520methyltransferases%253A%2520homology%2520modeling%252C%2520molecular%2520docking%252C%2520binding%2520mode%2520analysis%252C%2520and%2520biological%2520evaluations%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D1241%26epage%3D1253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nebbioso, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrone, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sbardella, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Bellis, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miceli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altucci, L.</span><span> </span><span class="NLM_article-title">Epigenetic multiple ligands: mixed histone/protein methyltransferase, acetyltransferase, and class III deacetylase (sirtuin) inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2279</span><span class="NLM_x">–</span> <span class="NLM_lpage">2290</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm701595q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2279-2290&author=A.+Maiauthor=D.+Chengauthor=M.+T.+Bedfordauthor=S.+Valenteauthor=A.+Nebbiosoauthor=A.+Perroneauthor=G.+Broschauthor=G.+Sbardellaauthor=F.+De+Bellisauthor=M.+Miceliauthor=L.+Altucci&title=Epigenetic+multiple+ligands%3A+mixed+histone%2Fprotein+methyltransferase%2C+acetyltransferase%2C+and+class+III+deacetylase+%28sirtuin%29+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm701595q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701595q%26sid%3Dliteratum%253Aachs%26aulast%3DMai%26aufirst%3DA.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DNebbioso%26aufirst%3DA.%26aulast%3DPerrone%26aufirst%3DA.%26aulast%3DBrosch%26aufirst%3DG.%26aulast%3DSbardella%26aufirst%3DG.%26aulast%3DDe%2BBellis%26aufirst%3DF.%26aulast%3DMiceli%26aufirst%3DM.%26aulast%3DAltucci%26aufirst%3DL.%26atitle%3DEpigenetic%2520multiple%2520ligands%253A%2520mixed%2520histone%252Fprotein%2520methyltransferase%252C%2520acetyltransferase%252C%2520and%2520class%2520III%2520deacetylase%2520%2528sirtuin%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2279%26epage%3D2290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Purandare, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huynh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaccaro, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poss, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oconnell, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nowak, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayaraman, L.</span><span> </span><span class="NLM_article-title">Pyrazole inhibitors of coactivator associated arginine methyltransferase 1 (CARM1)</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">4438</span><span class="NLM_x">–</span> <span class="NLM_lpage">4441</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1016%2Fj.bmcl.2008.06.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=18619839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptVCjsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=4438-4441&author=A.+V.+Purandareauthor=Z.+Chenauthor=T.+Huynhauthor=S.+Pangauthor=J.+Gengauthor=W.+Vaccaroauthor=M.+A.+Possauthor=J.+Oconnellauthor=K.+Nowakauthor=L.+Jayaraman&title=Pyrazole+inhibitors+of+coactivator+associated+arginine+methyltransferase+1+%28CARM1%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Pyrazole inhibitors of coactivator associated arginine methyltransferase 1 (CARM1)</span></div><div class="casAuthors">Purandare, Ashok V.; Chen, Zhong; Huynh, Tram; Pang, Suhong; Geng, Jieping; Vaccaro, Wayne; Poss, Michael A.; Oconnell, Jonathan; Nowak, Kimberly; Jayaraman, Lata</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4438-4441</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">This study reports the identification and Hits to Leads optimization of inhibitors of coactivator assocd. arginine methyltransferase (CARM1).  Compd. (I) is a potent, selective inhibitor of CARM1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjsEj-ubHXsrVg90H21EOLACvtfcHk0lgjQqPoHNwuzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptVCjsr8%253D&md5=0ef68fcfa0dedbe54d1fa5b8ae9ca376</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.06.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.06.026%26sid%3Dliteratum%253Aachs%26aulast%3DPurandare%26aufirst%3DA.%2BV.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DHuynh%26aufirst%3DT.%26aulast%3DPang%26aufirst%3DS.%26aulast%3DGeng%26aufirst%3DJ.%26aulast%3DVaccaro%26aufirst%3DW.%26aulast%3DPoss%26aufirst%3DM.%2BA.%26aulast%3DOconnell%26aufirst%3DJ.%26aulast%3DNowak%26aufirst%3DK.%26aulast%3DJayaraman%26aufirst%3DL.%26atitle%3DPyrazole%2520inhibitors%2520of%2520coactivator%2520associated%2520arginine%2520methyltransferase%25201%2520%2528CARM1%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D4438%26epage%3D4441" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Huynh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bandiera, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bindi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vianello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roletto, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thieffine, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galvani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaccaro, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poss, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trainor, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorenzi, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gottardis, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayaraman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Purandare, A. V.</span><span> </span><span class="NLM_article-title">Optimization of pyrazole inhibitors of coactivator associated arginine methyltransferase 1 (CARM1)</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2924</span><span class="NLM_x">–</span> <span class="NLM_lpage">2927</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1016%2Fj.bmcl.2009.04.075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=19419866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtVekur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=2924-2927&author=T.+Huynhauthor=Z.+Chenauthor=S.+Pangauthor=J.+Gengauthor=T.+Bandieraauthor=S.+Bindiauthor=P.+Vianelloauthor=F.+Rolettoauthor=S.+Thieffineauthor=A.+Galvaniauthor=W.+Vaccaroauthor=M.+A.+Possauthor=G.+L.+Trainorauthor=M.+V.+Lorenziauthor=M.+Gottardisauthor=L.+Jayaramanauthor=A.+V.+Purandare&title=Optimization+of+pyrazole+inhibitors+of+coactivator+associated+arginine+methyltransferase+1+%28CARM1%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of pyrazole inhibitors of coactivator associated arginine methyltransferase 1 (CARM1)</span></div><div class="casAuthors">Huynh, Tram; Chen, Zhong; Pang, Suhong; Geng, Jieping; Bandiera, Tiziano; Bindi, Simona; Vianello, Paola; Roletto, Fulvia; Thieffine, Sandrine; Galvani, Arturo; Vaccaro, Wayne; Poss, Michael A.; Trainor, George L.; Lorenzi, Matthew V.; Gottardis, Marco; Jayaraman, Lata; Purandare, Ashok V.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2924-2927</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Design, synthesis, and SAR development led to the identification of the potent, novel, and selective pyrazole based inhibitor I of CARM1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphFardNepde7Vg90H21EOLACvtfcHk0limlnjdEoolaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtVekur4%253D&md5=c37a2738bbd001d4758f8480746626fb</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.04.075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.04.075%26sid%3Dliteratum%253Aachs%26aulast%3DHuynh%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DPang%26aufirst%3DS.%26aulast%3DGeng%26aufirst%3DJ.%26aulast%3DBandiera%26aufirst%3DT.%26aulast%3DBindi%26aufirst%3DS.%26aulast%3DVianello%26aufirst%3DP.%26aulast%3DRoletto%26aufirst%3DF.%26aulast%3DThieffine%26aufirst%3DS.%26aulast%3DGalvani%26aufirst%3DA.%26aulast%3DVaccaro%26aufirst%3DW.%26aulast%3DPoss%26aufirst%3DM.%2BA.%26aulast%3DTrainor%26aufirst%3DG.%2BL.%26aulast%3DLorenzi%26aufirst%3DM.%2BV.%26aulast%3DGottardis%26aufirst%3DM.%26aulast%3DJayaraman%26aufirst%3DL.%26aulast%3DPurandare%26aufirst%3DA.%2BV.%26atitle%3DOptimization%2520of%2520pyrazole%2520inhibitors%2520of%2520coactivator%2520associated%2520arginine%2520methyltransferase%25201%2520%2528CARM1%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D2924%26epage%3D2927" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Allan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manku, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therrien, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Styhler, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robert, M.-F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goulet, A.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petschner, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahil, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacLeod, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deziel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Besterman, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahhab, A.</span><span> </span><span class="NLM_article-title"><i>N</i>-Benzyl-1-heteroaryl-3-(trifluoromethyl)-1<i>H</i>-pyrazole-5-carboxamides as inhibitors of co-activator associated arginine methyltransferase 1 (CARM1)</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1218</span><span class="NLM_x">–</span> <span class="NLM_lpage">1223</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=1218-1223&author=M.+Allanauthor=S.+Mankuauthor=E.+Therrienauthor=N.+Nguyenauthor=S.+Styhlerauthor=M.-F.+Robertauthor=A.-C.+Gouletauthor=A.+J.+Petschnerauthor=G.+Rahilauthor=A.+R.+MacLeodauthor=R.+Dezielauthor=J.+M.+Bestermanauthor=H.+Nguyenauthor=A.+Wahhab&title=N-Benzyl-1-heteroaryl-3-%28trifluoromethyl%29-1H-pyrazole-5-carboxamides+as+inhibitors+of+co-activator+associated+arginine+methyltransferase+1+%28CARM1%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAllan%26aufirst%3DM.%26aulast%3DManku%26aufirst%3DS.%26aulast%3DTherrien%26aufirst%3DE.%26aulast%3DNguyen%26aufirst%3DN.%26aulast%3DStyhler%26aufirst%3DS.%26aulast%3DRobert%26aufirst%3DM.-F.%26aulast%3DGoulet%26aufirst%3DA.-C.%26aulast%3DPetschner%26aufirst%3DA.%2BJ.%26aulast%3DRahil%26aufirst%3DG.%26aulast%3DMacLeod%26aufirst%3DA.%2BR.%26aulast%3DDeziel%26aufirst%3DR.%26aulast%3DBesterman%26aufirst%3DJ.%2BM.%26aulast%3DNguyen%26aufirst%3DH.%26aulast%3DWahhab%26aufirst%3DA.%26atitle%3DN-Benzyl-1-heteroaryl-3-%2528trifluoromethyl%2529-1H-pyrazole-5-carboxamides%2520as%2520inhibitors%2520of%2520co-activator%2520associated%2520arginine%2520methyltransferase%25201%2520%2528CARM1%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D1218%26epage%3D1223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heinke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gust, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span> </span><span class="NLM_article-title">Target-based approach to inhibitors of histone arginine methyltransferases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">2319</span><span class="NLM_x">–</span> <span class="NLM_lpage">2325</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm061250e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=2319-2325&author=A.+Spannhoffauthor=R.+Heinkeauthor=I.+Bauerauthor=P.+Trojerauthor=E.+Metzgerauthor=R.+Gustauthor=R.+Schuleauthor=G.+Broschauthor=W.+Sipplauthor=M.+Jung&title=Target-based+approach+to+inhibitors+of+histone+arginine+methyltransferases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm061250e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061250e%26sid%3Dliteratum%253Aachs%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DHeinke%26aufirst%3DR.%26aulast%3DBauer%26aufirst%3DI.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DGust%26aufirst%3DR.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DBrosch%26aufirst%3DG.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DTarget-based%2520approach%2520to%2520inhibitors%2520of%2520histone%2520arginine%2520methyltransferases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D2319%26epage%3D2325" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Heinke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meier, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span> </span><span class="NLM_article-title">Virtual screening and biological characterization of novel histone arginine methyltransferase PRMT1 inhibitors</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">69</span><span class="NLM_x">–</span> <span class="NLM_lpage">77</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=69-77&author=R.+Heinkeauthor=A.+Spannhoffauthor=R.+Meierauthor=P.+Trojerauthor=I.+Bauerauthor=M.+Jungauthor=W.+Sippl&title=Virtual+screening+and+biological+characterization+of+novel+histone+arginine+methyltransferase+PRMT1+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHeinke%26aufirst%3DR.%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DMeier%26aufirst%3DR.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DBauer%26aufirst%3DI.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DSippl%26aufirst%3DW.%26atitle%3DVirtual%2520screening%2520and%2520biological%2520characterization%2520of%2520novel%2520histone%2520arginine%2520methyltransferase%2520PRMT1%2520inhibitors%26jtitle%3DChemMedChem%26date%3D2009%26volume%3D4%26spage%3D69%26epage%3D77" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machmur, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heinke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanefeld, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span> </span><span class="NLM_article-title">A novel arginine methyltransferase inhibitor with cellular activity</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">4150</span><span class="NLM_x">–</span> <span class="NLM_lpage">4153</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=4150-4153&author=A.+Spannhoffauthor=R.+Machmurauthor=R.+Heinkeauthor=P.+Trojerauthor=I.+Bauerauthor=G.+Broschauthor=R.+Schuleauthor=W.+Hanefeldauthor=W.+Sipplauthor=M.+Jung&title=A+novel+arginine+methyltransferase+inhibitor+with+cellular+activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DMachmur%26aufirst%3DR.%26aulast%3DHeinke%26aufirst%3DR.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DBauer%26aufirst%3DI.%26aulast%3DBrosch%26aufirst%3DG.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DHanefeld%26aufirst%3DW.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DA%2520novel%2520arginine%2520methyltransferase%2520inhibitor%2520with%2520cellular%2520activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D4150%26epage%3D4153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Feng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, Y. G.</span><span> </span><span class="NLM_article-title">Discovery and mechanistic study of a class of protein arginine methylation inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">6028</span><span class="NLM_x">–</span> <span class="NLM_lpage">6039</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm100416n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=6028-6039&author=Y.+Fengauthor=M.+Y.+Liauthor=B.+H.+Wangauthor=Y.+G.+Zheng&title=Discovery+and+mechanistic+study+of+a+class+of+protein+arginine+methylation+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm100416n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100416n%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DM.%2BY.%26aulast%3DWang%26aufirst%3DB.%2BH.%26aulast%3DZheng%26aufirst%3DY.%2BG.%26atitle%3DDiscovery%2520and%2520mechanistic%2520study%2520of%2520a%2520class%2520of%2520protein%2520arginine%2520methylation%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D6028%26epage%3D6039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Selvi, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Batta, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kishore, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mantelingu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varier, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanyam, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pradhan, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dasgupta, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sriram, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agrawal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kundu, T. K.</span><span> </span><span class="NLM_article-title">Identification of a novel inhibitor of coactivator-associated arginine methyltransferase 1 (CARM1)-mediated methylation of histone H3 Arg-17</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">7143</span><span class="NLM_x">–</span> <span class="NLM_lpage">7152</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1074%2Fjbc.M109.063933" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=20022955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC3cXisFalu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=7143-7152&author=B.+R.+Selviauthor=K.+Battaauthor=A.+H.+Kishoreauthor=K.+Mantelinguauthor=R.+A.+Varierauthor=K.+Balasubramanyamauthor=S.+K.+Pradhanauthor=D.+Dasguptaauthor=S.+Sriramauthor=S.+Agrawalauthor=T.+K.+Kundu&title=Identification+of+a+novel+inhibitor+of+coactivator-associated+arginine+methyltransferase+1+%28CARM1%29-mediated+methylation+of+histone+H3+Arg-17"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Novel Inhibitor of Coactivator-associated Arginine Methyltransferase 1 (CARM1)-mediated Methylation of Histone H3 Arg-17</span></div><div class="casAuthors">Selvi, B. Ruthrotha; Batta, Kiran; Kishore, A. Hari; Mantelingu, Kempegowda; Varier, Radhika A.; Balasubramanyam, Karanam; Pradhan, Suman Kalyan; Dasgupta, Dipak; Sriram, Sokalingam; Agrawal, Shipra; Kundu, Tapas K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">7143-7152</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Methylation of the arginine residues of histones by methyltransferases has important consequences for chromatin structure and gene regulation; however, the mol. mechanism(s) of methyltransferase regulation is still unclear, as is the biol. significance of methylation at particular arginine residues.  Here, we report a novel specific inhibitor of coactivator-assocd. arginine methyltransferase 1 (CARM1; also known as PRMT4) that selectively inhibits methylation at arginine 17 of histone H3 (H3R17).  Remarkably, this plant-derived inhibitor, called TBBD (ellagic acid), binds to the substrate (histone) preferentially at the signature motif, "KAPRK", where the proline residue (Pro-16) plays a crit. role for interaction and subsequent enzyme inhibition.  In a promoter-specific context, inhibition of H3R17 methylation represses expression of p21, a p53-responsive gene, thus implicating a possible role for H3 Arg-17 methylation in tumor suppressor function.  These data establish TBBD as a novel specific inhibitor of arginine methylation and demonstrate substrate sequence-directed inhibition of enzyme activity by a small mol. and its physiol. consequence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoacMKhuo6Pl7Vg90H21EOLACvtfcHk0lhi8c4ZNqeLow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXisFalu7c%253D&md5=6be021194a5bbd5deeea1990df6238fd</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.063933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.063933%26sid%3Dliteratum%253Aachs%26aulast%3DSelvi%26aufirst%3DB.%2BR.%26aulast%3DBatta%26aufirst%3DK.%26aulast%3DKishore%26aufirst%3DA.%2BH.%26aulast%3DMantelingu%26aufirst%3DK.%26aulast%3DVarier%26aufirst%3DR.%2BA.%26aulast%3DBalasubramanyam%26aufirst%3DK.%26aulast%3DPradhan%26aufirst%3DS.%2BK.%26aulast%3DDasgupta%26aufirst%3DD.%26aulast%3DSriram%26aufirst%3DS.%26aulast%3DAgrawal%26aufirst%3DS.%26aulast%3DKundu%26aufirst%3DT.%2BK.%26atitle%3DIdentification%2520of%2520a%2520novel%2520inhibitor%2520of%2520coactivator-associated%2520arginine%2520methyltransferase%25201%2520%2528CARM1%2529-mediated%2520methylation%2520of%2520histone%2520H3%2520Arg-17%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D7143%26epage%3D7152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinha, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chai, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muniyan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chou, Y. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kathy, Li, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">George Zheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Pharmacophore-based virtual screening and biological evaluation of small molecule inhibitors for protein arginine methylation</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">7978</span><span class="NLM_x">–</span> <span class="NLM_lpage">7987</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm300521m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7978-7987&author=J.+Wangauthor=L.+Chenauthor=S.+H.+Sinhaauthor=Z.+Liangauthor=H.+Chaiauthor=S.+Muniyanauthor=Y.+W.+Chouauthor=C.+Yangauthor=L.+Yanauthor=Y.+Fengauthor=Li%2C+K.+Kathyauthor=M.+F.+Linauthor=H.+Jiangauthor=Y.+George+Zhengauthor=C.+Luo&title=Pharmacophore-based+virtual+screening+and+biological+evaluation+of+small+molecule+inhibitors+for+protein+arginine+methylation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjm300521m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300521m%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DSinha%26aufirst%3DS.%2BH.%26aulast%3DLiang%26aufirst%3DZ.%26aulast%3DChai%26aufirst%3DH.%26aulast%3DMuniyan%26aufirst%3DS.%26aulast%3DChou%26aufirst%3DY.%2BW.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DYan%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DKathy%26aufirst%3DLi%252C%2BK.%26aulast%3DLin%26aufirst%3DM.%2BF.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DGeorge%2BZheng%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DPharmacophore-based%2520virtual%2520screening%2520and%2520biological%2520evaluation%2520of%2520small%2520molecule%2520inhibitors%2520for%2520protein%2520arginine%2520methylation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D7978%26epage%3D7987" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Sullivan, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">August, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickey, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teodoro, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rea, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mechtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kowalski, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Homon, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span> </span><span class="NLM_article-title">Reversal of H3K9me2 by a small-molecule inhibitor for the G9a histone methyltransferase</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">473</span><span class="NLM_x">–</span> <span class="NLM_lpage">481</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1016%2Fj.molcel.2007.01.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=17289593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitl2jurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=473-481&author=S.+Kubicekauthor=R.+J.+O%E2%80%99Sullivanauthor=E.+M.+Augustauthor=E.+R.+Hickeyauthor=Q.+Zhangauthor=M.+L.+Teodoroauthor=S.+Reaauthor=K.+Mechtlerauthor=J.+A.+Kowalskiauthor=C.+A.+Homonauthor=T.+A.+Kellyauthor=T.+Jenuwein&title=Reversal+of+H3K9me2+by+a+small-molecule+inhibitor+for+the+G9a+histone+methyltransferase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of H3K9me2 by a small-molecule inhibitor for the G9a histone methyltransferase</span></div><div class="casAuthors">Kubicek, Stefan; O'Sullivan, Roderick J.; August, E. Michael; Hickey, Eugene R.; Zhang, Qiang; Teodoro, Miguel L.; Rea, Stephen; Mechtler, Karl; Kowalski, Jennifer A.; Homon, Carol Ann; Kelly, Terence A.; Jenuwein, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">473-481</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone lysine methylation has important roles in the organization of chromatin domains and the regulation of gene expression.  To analyze its function and modulate its activity, we screened for specific inhibitors against histone lysine methyltransferases (HMTases) using recombinant G9a as the target enzyme.  From a chem. library comprising 125,000 preselected compds., seven hits were identified.  Of those, one inhibitor, BIX-01294 (diazepinquinazolin-amine deriv.), does not compete with the cofactor S-adenosyl-methionine, and selectively impairs the G9a HMTase and the generation of H3K9me2 in vitro.  In cellular assays, transient incubation of several cell lines with BIX-01294 lowers bulk H3K9me2 levels that are restored upon removal of the inhibitor.  Importantly, chromatin immunopptn. at several G9a target genes demonstrates reversible redn. of promoter-proximal H3K9me2 in inhibitor-treated mouse ES cells and fibroblasts.  Our data identify a biol. active HMTase inhibitor that allows for the transient modulation of H3K9me2 marks in mammalian chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3SW8czS4WjbVg90H21EOLACvtfcHk0ljnu_UqsnQS0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitl2jurs%253D&md5=d2f5290f1252e630a9b05489b0775c2c</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.01.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.01.017%26sid%3Dliteratum%253Aachs%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DR.%2BJ.%26aulast%3DAugust%26aufirst%3DE.%2BM.%26aulast%3DHickey%26aufirst%3DE.%2BR.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DTeodoro%26aufirst%3DM.%2BL.%26aulast%3DRea%26aufirst%3DS.%26aulast%3DMechtler%26aufirst%3DK.%26aulast%3DKowalski%26aufirst%3DJ.%2BA.%26aulast%3DHomon%26aufirst%3DC.%2BA.%26aulast%3DKelly%26aufirst%3DT.%2BA.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DReversal%2520of%2520H3K9me2%2520by%2520a%2520small-molecule%2520inhibitor%2520for%2520the%2520G9a%2520histone%2520methyltransferase%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D25%26spage%3D473%26epage%3D481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Protein lysine methyltransferase G9a inhibitors: design, synthesis, and structure activity relationships of 2,4-diamino-7-aminoalkoxy-quinazolines</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5844</span><span class="NLM_x">–</span> <span class="NLM_lpage">5857</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm100478y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC3cXos1ekurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5844-5857&author=F.+Liuauthor=X.+Chenauthor=A.+Allali-Hassaniauthor=A.+M.+Quinnauthor=T.+J.+Wigleauthor=G.+A.+Wasneyauthor=A.+Dongauthor=G.+Senisterraauthor=I.+Chauauthor=A.+Siarheyevaauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Jadhavauthor=J.+M.+Heroldauthor=W.+P.+Janzenauthor=C.+H.+Arrowsmithauthor=S.+V.+Fryeauthor=P.+J.+Brownauthor=A.+Simeonovauthor=M.+Vedadiauthor=J.+Jin&title=Protein+lysine+methyltransferase+G9a+inhibitors%3A+design%2C+synthesis%2C+and+structure+activity+relationships+of+2%2C4-diamino-7-aminoalkoxy-quinazolines"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Lysine Methyltransferase G9a Inhibitors: Design, Synthesis, and Structure Activity Relationships of 2,4-Diamino-7-aminoalkoxy-quinazolines</span></div><div class="casAuthors">Liu, Feng; Chen, Xin; Allali-Hassani, Abdellah; Quinn, Amy M.; Wigle, Tim J.; Wasney, Gregory A.; Dong, Aiping; Senisterra, Guillermo; Chau, Irene; Siarheyeva, Alena; Norris, Jacqueline L.; Kireev, Dmitri B.; Jadhav, Ajit; Herold, J. Martin; Janzen, William P.; Arrowsmith, Cheryl H.; Frye, Stephen V.; Brown, Peter J.; Simeonov, Anton; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5844-5857</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein lysine methyltransferase G9a, which catalyzes methylation of lysine 9 of histone H3 (H3K9) and lysine 373 (K373) of p53, is overexpressed in human cancers.  Genetic knockdown of G9a inhibits cancer cell growth; and the dimethylation of p53 K373 results in the inactivation of p53.  Initial SAR exploration of the 2,4-diamino-6,7-dimethoxyquinazoline template represented by I ( R1 = Bn; R2 = Me)(BIX01294), a selective small mol. inhibitor of G9a and GLP (G9a-like protein), led to the discovery of I [R1 = Me; R2 = (CH2)3NMe2](II)(UNC0224) as a potent G9a inhibitor with excellent selectivity.  A high resoln. X-ray crystal structure of the G9a-II complex, the first cocrystal structure of G9a with a small mol. inhibitor, was obtained.  On the basis of the structural insights revealed by this cocrystal structure, optimization of the 7-(dimethylamino)propoxy side chain of II resulted in the discovery of I [R1 = Me; R2 = (CH2)2O(CH2)2NMe2](UNC0321, Morrison Ki = 63 pM), which is the first G9a inhibitor with picomolar potency and the most potent G9a inhibitor to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgS0YxdNdNxLVg90H21EOLACvtfcHk0lhtiNZAmk_Hag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXos1ekurk%253D&md5=8bd703c0a975a33bf9e5c019f1f95ee6</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm100478y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100478y%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProtein%2520lysine%2520methyltransferase%2520G9a%2520inhibitors%253A%2520design%252C%2520synthesis%252C%2520and%2520structure%2520activity%2520relationships%2520of%25202%252C4-diamino-7-aminoalkoxy-quinazolines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5844%26epage%3D5857" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Chang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horton, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snyder, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span> </span><span class="NLM_article-title">Adding a lysine mimic in the design of potent inhibitors of histone lysine methyltransferases</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">400</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=400&publication_year=2010&pages=1-7&author=Y.+Changauthor=T.+Ganeshauthor=J.+R.+Hortonauthor=A.+Spannhoffauthor=J.+Liuauthor=A.+Sunauthor=X.+Zhangauthor=M.+T.+Bedfordauthor=Y.+Shinkaiauthor=J.+P.+Snyderauthor=X.+Cheng&title=Adding+a+lysine+mimic+in+the+design+of+potent+inhibitors+of+histone+lysine+methyltransferases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DY.%26aulast%3DGanesh%26aufirst%3DT.%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DShinkai%26aufirst%3DY.%26aulast%3DSnyder%26aufirst%3DJ.%2BP.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DAdding%2520a%2520lysine%2520mimic%2520in%2520the%2520design%2520of%2520potent%2520inhibitors%2520of%2520histone%2520lysine%2520methyltransferases%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D400%26spage%3D1%26epage%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozieradzki, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Discovery of a 2,4-diamino-7-aminoalkoxyquinazoline as a potent and selective inhibitor of histone lysine methyltransferase G9a</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">7950</span><span class="NLM_x">–</span> <span class="NLM_lpage">7953</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm901543m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlOiurzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=7950-7953&author=F.+Liuauthor=X.+Chenauthor=A.+Allali-Hassaniauthor=A.+M.+Quinnauthor=G.+A.+Wasneyauthor=A.+Dongauthor=D.+Barsyteauthor=I.+Kozieradzkiauthor=G.+Senisterraauthor=I.+Chauauthor=A.+Siarheyevaauthor=D.+B.+Kireevauthor=A.+Jadhavauthor=J.+M.+Heroldauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=A.+Simeonovauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+a+2%2C4-diamino-7-aminoalkoxyquinazoline+as+a+potent+and+selective+inhibitor+of+histone+lysine+methyltransferase+G9a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a 2,4-Diamino-7-aminoalkoxyquinazoline as a Potent and Selective Inhibitor of Histone Lysine Methyltransferase G9a</span></div><div class="casAuthors">Liu, Feng; Chen, Xin; Allali-Hassani, Abdellah; Quinn, Amy M.; Wasney, Gregory A.; Dong, Aiping; Barsyte, Dalia; Kozieradzki, Ivona; Senisterra, Guillermo; Chau, Irene; Siarheyeva, Alena; Kireev, Dmitri B.; Jadhav, Ajit; Herold, J. Martin; Frye, Stephen V.; Arrowsmith, Cheryl H.; Brown, Peter J.; Simeonov, Anton; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7950-7953</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SAR exploration of the 2,4-diamino-6,7-dimethoxyquinazoline template led to the discovery of 8 (I) as a potent and selective G9a inhibitor.  A high resoln. X-ray crystal structure of the G9a-8 complex, the first cocrystal structure of G9a with a small mol. inhibitor, was obtained.  The cocrystal structure validated our binding hypothesis and will enable structure-based design of novel inhibitors.  8 Is a useful tool for investigating the biol. of G9a and its roles in chromatin remodeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjWt5XsGXyj7Vg90H21EOLACvtfcHk0lik0Q9fbnrIQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlOiurzN&md5=2d7e34a8c4b5bf1cafd4e01ad2a86de1</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm901543m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901543m%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DBarsyte%26aufirst%3DD.%26aulast%3DKozieradzki%26aufirst%3DI.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%25202%252C4-diamino-7-aminoalkoxyquinazoline%2520as%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520histone%2520lysine%2520methyltransferase%2520G9a%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D7950%26epage%3D7953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yates, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Optimization of cellular activity of G9a inhibitors 7-aminoalkoxy-quinazolines</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">6139</span><span class="NLM_x">–</span> <span class="NLM_lpage">6150</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm200903z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvVWltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6139-6150&author=F.+Liuauthor=D.+Barsyte-Lovejoyauthor=A.+Allali-Hassaniauthor=Y.+Heauthor=J.+M.+Heroldauthor=X.+Chenauthor=C.+M.+Yatesauthor=S.+V.+Fryeauthor=P.+J.+Brownauthor=J.+Huangauthor=M.+Vedadiauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=Optimization+of+cellular+activity+of+G9a+inhibitors+7-aminoalkoxy-quinazolines"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of cellular activity of G9a inhibitors 7-aminoalkoxy-quinazolines</span></div><div class="casAuthors">Liu, Feng; Barsyte-Lovejoy, Dalia; Allali-Hassani, Abdellah; He, Yunlong; Herold, J. Martin; Chen, Xin; Yates, Christopher M.; Frye, Stephen V.; Brown, Peter J.; Huang, Jing; Vedadi, Masoud; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6139-6150</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein lysine methyltransferase G9a plays key roles in the transcriptional repression of a variety of genes via dimethylation of lysine 9 on histone H3 (H3K9me2) of chromatin as well as dimethylation of nonhistone proteins including tumor suppressor p53.  We previously reported the discovery of UNC0321, the most potent G9a inhibitor to date, via structure-based design and structure-activity relationship (SAR) exploration of the quinazoline scaffold represented by BIX01294.  Despite its very high in vitro potency, compd. UNC0321 lacks sufficient cellular potency.  The design and synthesis of several generations of new analogs aimed at improving cell membrane permeability while maintaining high in vitro potency resulted in the discovery of a no. of novel G9a inhibitors such as UNC0646 I and UNC0631 II with excellent potency in a variety of cell lines and excellent sepn. of functional potency vs. cell toxicity.  The design, synthesis, and cellular SAR of these potent G9a inhibitors are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC91yKB7S34rVg90H21EOLACvtfcHk0lik0Q9fbnrIQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvVWltLg%253D&md5=f3f54b91e6b24aa248498f9ff1fdb1ae</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fjm200903z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200903z%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DYates%26aufirst%3DC.%2BM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DOptimization%2520of%2520cellular%2520activity%2520of%2520G9a%2520inhibitors%25207-aminoalkoxy-quinazolines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6139%26epage%3D6150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rival-Gervier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labrie, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiMaggio, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempel, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangano, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpson, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattenden, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tripathy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petronis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">566</span><span class="NLM_x">–</span> <span class="NLM_lpage">574</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1038%2Fnchembio.599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=21743462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=566-574&author=M.+Vedadiauthor=D.+Barsyte-Lovejoyauthor=F.+Liuauthor=S.+Rival-Gervierauthor=A.+Allali-Hassaniauthor=V.+Labrieauthor=T.+J.+Wigleauthor=P.+A.+DiMaggioauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=A.+Dongauthor=W.+Tempelauthor=S.-C.+Wangauthor=X.+Chenauthor=I.+Chauauthor=T.+Manganoauthor=X.-P.+Huangauthor=C.+D.+Simpsonauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Tripathyauthor=A.+Edwardsauthor=B.+L.+Rothauthor=W.+P.+Janzenauthor=B.+A.+Garciaauthor=A.+Petronisauthor=J.+Ellisauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=A+chemical+probe+selectively+inhibits+G9a+and+GLP+methyltransferase+activity+in+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span></div><div class="casAuthors">Vedadi, Masoud; Barsyte-Lovejoy, Dalia; Liu, Feng; Rival-Gervier, Sylvie; Allali-Hassani, Abdellah; Labrie, Viviane; Wigle, Tim J.; DiMaggio, Peter A.; Wasney, Gregory A.; Siarheyeva, Alena; Dong, Aiping; Tempel, Wolfram; Wang, Sun-Chong; Chen, Xin; Chau, Irene; Mangano, Thomas J.; Huang, Xi-ping; Simpson, Catherine D.; Pattenden, Samantha G.; Norris, Jacqueline L.; Kireev, Dmitri B.; Tripathy, Ashutosh; Edwards, Aled; Roth, Bryan L.; Janzen, William P.; Garcia, Benjamin A.; Petronis, Arturas; Ellis, James; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">566-574</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases G9a and GLP modulate the transcriptional repression of a variety of genes via dimethylation of Lys9 on histone H3 (H3K9me2) as well as dimethylation of non-histone targets.  Here we report the discovery of UNC0638, an inhibitor of G9a and GLP with excellent potency and selectivity over a wide range of epigenetic and non-epigenetic targets.  UNC0638 treatment of a variety of cell lines resulted in lower global H3K9me2 levels, equiv. to levels obsd. for small hairpin RNA knockdown of G9a and GLP with the functional potency of UNC0638 being well sepd. from its toxicity.  UNC0638 markedly reduced the clonogenicity of MCF7 cells, reduced the abundance of H3K9me2 marks at promoters of known G9a-regulated endogenous genes and disproportionately affected several genomic loci encoding microRNAs.  In mouse embryonic stem cells, UNC0638 reactivated G9a-silenced genes and a retroviral reporter gene in a concn.-dependent manner without promoting differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphv-2bZcBqwLVg90H21EOLACvtfcHk0livdCQsUGI-Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D&md5=a944e99a910194c1a15240fc6dc8f852</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.599%26sid%3Dliteratum%253Aachs%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DRival-Gervier%26aufirst%3DS.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLabrie%26aufirst%3DV.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DDiMaggio%26aufirst%3DP.%2BA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DS.-C.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMangano%26aufirst%3DT.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DSimpson%26aufirst%3DC.%2BD.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DEdwards%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DPetronis%26aufirst%3DA.%26aulast%3DEllis%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520chemical%2520probe%2520selectively%2520inhibits%2520G9a%2520and%2520GLP%2520methyltransferase%2520activity%2520in%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D566%26epage%3D574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Daigle, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therkelsen, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sneeringer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnston, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernt, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tseng, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span> </span><span class="NLM_article-title">Selective killing of mixed lineage leukemia cells by a potent small-molecule DOT1L inhibitor</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">53</span><span class="NLM_x">–</span> <span class="NLM_lpage">65</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1016%2Fj.ccr.2011.06.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=21741596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslCms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2011&pages=53-65&author=S.+R.+Daigleauthor=E.+J.+Olhavaauthor=C.+A.+Therkelsenauthor=C.+R.+Majerauthor=C.+J.+Sneeringerauthor=J.+Songauthor=L.+D.+Johnstonauthor=M.+P.+Scottauthor=J.+J.+Smithauthor=Y.+Xiaoauthor=L.+Jinauthor=K.+W.+Kuntzauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=K.+M.+Berntauthor=J.+C.+Tsengauthor=A.+L.+Kungauthor=S.+A.+Armstrongauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollock&title=Selective+killing+of+mixed+lineage+leukemia+cells+by+a+potent+small-molecule+DOT1L+inhibitor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Killing of Mixed Lineage Leukemia Cells by a Potent Small-Molecule DOT1L Inhibitor</span></div><div class="casAuthors">Daigle, Scott R.; Olhava, Edward J.; Therkelsen, Carly A.; Majer, Christina R.; Sneeringer, Christopher J.; Song, Jeffrey; Johnston, L. Danielle; Scott, Margaret Porter; Smith, Jesse J.; Xiao, Yonghong; Jin, Lei; Kuntz, Kevin W.; Chesworth, Richard; Moyer, Mikel P.; Bernt, Kathrin M.; Tseng, Jen-Chieh; Kung, Andrew L.; Armstrong, Scott A.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">53-65</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Mislocated enzymic activity of DOT1L has been proposed as a driver of leukemogenesis in mixed lineage leukemia (MLL).  The characterization of EPZ004777, a potent, selective inhibitor of DOT1L is reported.  Treatment of MLL cells with the compd. selectively inhibits H3K79 methylation and blocks expression of leukemogenic genes.  Exposure of leukemic cells to EPZ004777 results in selective killing of those cells bearing the MLL gene translocation, with little effect on non-MLL-translocated cells.  Finally, in vivo administration of EPZ004777 leads to extension of survival in a mouse MLL xenograft model.  These results provide compelling support for DOT1L inhibition as a basis for targeted therapeutics against MLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodbBbxHFmyybVg90H21EOLACvtfcHk0livdCQsUGI-Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslCms7c%253D&md5=1d39f99c2983b6b5fccc079110092c3d</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2011.06.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2011.06.009%26sid%3Dliteratum%253Aachs%26aulast%3DDaigle%26aufirst%3DS.%2BR.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DTherkelsen%26aufirst%3DC.%2BA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DBernt%26aufirst%3DK.%2BM.%26aulast%3DTseng%26aufirst%3DJ.%2BC.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26atitle%3DSelective%2520killing%2520of%2520mixed%2520lineage%2520leukemia%2520cells%2520by%2520a%2520potent%2520small-molecule%2520DOT1L%2520inhibitor%26jtitle%3DCancer%2520Cell%26date%3D2011%26volume%3D20%26spage%3D53%26epage%3D65" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Ferguson, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larsen, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grande, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia-Arenas, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Godin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keen, N.</span><span> </span><span class="NLM_article-title">Structural basis of substrate methylation and inhibition of SMYD2</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1262</span><span class="NLM_x">–</span> <span class="NLM_lpage">1273</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1016%2Fj.str.2011.06.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=21782458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaru7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=1262-1273&author=A.+D.+Fergusonauthor=N.+A.+Larsenauthor=T.+Howardauthor=H.+Pollardauthor=I.+Greenauthor=C.+Grandeauthor=T.+Cheungauthor=R.+Garcia-Arenasauthor=S.+Cowenauthor=J.+Wuauthor=R.+Godinauthor=H.+Chenauthor=N.+Keen&title=Structural+basis+of+substrate+methylation+and+inhibition+of+SMYD2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of Substrate Methylation and Inhibition of SMYD2</span></div><div class="casAuthors">Ferguson, Andrew D.; Larsen, Nicholas A.; Howard, Tina; Pollard, Hannah; Green, Isabelle; Grande, Christie; Cheung, Tony; Garcia-Arenas, Renee; Cowen, Scott; Wu, Jiaquan; Godin, Robert; Chen, Huawei; Keen, Nicholas</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1262-1273</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases are important regulators of epigenetic signaling.  These enzymes catalyze the transfer of donor Me groups from S-adenosylmethionine to specific acceptor lysines on histones, leading to changes in chromatin structure and transcriptional regulation.  These enzymes also methylate nonhistone protein substrates, revealing an addnl. mechanism to regulate cellular physiol.  The oncogenic protein SMYD2 represses the functional activities of the tumor suppressor proteins p53 and Rb, making it an attractive drug target.  Here we report the discovery of AZ505, a potent and selective inhibitor of SMYD2 that was identified from a high throughput chem. screen.  We also present the crystal structures of SMYD2 with p53 substrate and product peptides, and notably, in complex with AZ505.  This substrate competitive inhibitor is bound in the peptide binding groove of SMYD2.  These results have implications for the development of SMYD2 inhibitors, and indicate the potential for developing novel therapies targeting this target class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfo0RkV17Th7Vg90H21EOLACvtfcHk0lhjY6YJ2PPXKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaru7fK&md5=31a08f46bffea75609e1c5bb25b7c610</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2011.06.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2011.06.011%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DLarsen%26aufirst%3DN.%2BA.%26aulast%3DHoward%26aufirst%3DT.%26aulast%3DPollard%26aufirst%3DH.%26aulast%3DGreen%26aufirst%3DI.%26aulast%3DGrande%26aufirst%3DC.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DGarcia-Arenas%26aufirst%3DR.%26aulast%3DCowen%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DGodin%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DKeen%26aufirst%3DN.%26atitle%3DStructural%2520basis%2520of%2520substrate%2520methylation%2520and%2520inhibition%2520of%2520SMYD2%26jtitle%3DStructure%26date%3D2011%26volume%3D19%26spage%3D1262%26epage%3D1273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anglin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, B. V. V</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Selective inhibitors of histone methyltransferase DOT1L: design, synthesis and crystallographic studies</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">16746</span><span class="NLM_x">–</span> <span class="NLM_lpage">16749</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja206312b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=16746-16749&author=Y.+Yaoauthor=P.+Chenauthor=J.+Diaoauthor=G.+Chengauthor=L.+Dengauthor=J.+L.+Anglinauthor=B.+V.+V+Prasadauthor=Y.+Song&title=Selective+inhibitors+of+histone+methyltransferase+DOT1L%3A+design%2C+synthesis+and+crystallographic+studies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fja206312b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja206312b%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DDiao%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DG.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DAnglin%26aufirst%3DJ.%2BL.%26aulast%3DPrasad%26aufirst%3DB.%2BV.%2BV%26aulast%3DSong%26aufirst%3DY.%26atitle%3DSelective%2520inhibitors%2520of%2520histone%2520methyltransferase%2520DOT1L%253A%2520design%252C%2520synthesis%2520and%2520crystallographic%2520studies%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D16746%26epage%3D16749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Yuan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paulk, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kemp, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shamji, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span> </span><span class="NLM_article-title">A small-molecule probe of the histone methyltransferase G9a induces cellular senescence in pancreatic adenocarcinoma</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1152</span><span class="NLM_x">–</span> <span class="NLM_lpage">1157</span></span><div class="citationLinks">[<a href="/doi/10.1021/cb300139y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1152-1157&author=Y.+Yuanauthor=Q.+Wangauthor=J.+Paulkauthor=S.+Kubicekauthor=M.+M.+Kempauthor=D.+J.+Adamsauthor=A.+F.+Shamjiauthor=B.+K.+Wagnerauthor=S.+L.+Schreiber&title=A+small-molecule+probe+of+the+histone+methyltransferase+G9a+induces+cellular+senescence+in+pancreatic+adenocarcinoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fcb300139y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300139y%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DKemp%26aufirst%3DM.%2BM.%26aulast%3DAdams%26aufirst%3DD.%2BJ.%26aulast%3DShamji%26aufirst%3DA.%2BF.%26aulast%3DWagner%26aufirst%3DB.%2BK.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DA%2520small-molecule%2520probe%2520of%2520the%2520histone%2520methyltransferase%2520G9a%2520induces%2520cellular%2520senescence%2520in%2520pancreatic%2520adenocarcinoma%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1152%26epage%3D1157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganji, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korenchuk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Aller, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iii, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diaz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lafrance, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mellinger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duquenne, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHugh, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brandt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verma, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tummino, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creasy, C. L.</span><span> </span><span class="NLM_article-title">EZH2 inhibition as a therapeutic strategy for lymphoma with EZH2-activating mutations</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">492</span><span class="NLM_x">, </span> <span class="NLM_fpage">108</span><span class="NLM_x">–</span> <span class="NLM_lpage">112</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=108-112&author=M.+T.+McCabeauthor=H.+M.+Ottauthor=G.+Ganjiauthor=S.+Korenchukauthor=C.+Thompsonauthor=G.+S.+Van+Allerauthor=Y.+Liuauthor=A.+P.+Gravesauthor=A.+D.+Iiiauthor=E.+Diazauthor=L.+V.+Lafranceauthor=M.+Mellingerauthor=C.+Duquenneauthor=X.+Tianauthor=R.+G.+Krugerauthor=C.+F.+McHughauthor=M.+Brandtauthor=W.+H.+Millerauthor=D.+Dhanakauthor=S.+K.+Vermaauthor=P.+J.+Tumminoauthor=C.+L.+Creasy&title=EZH2+inhibition+as+a+therapeutic+strategy+for+lymphoma+with+EZH2-activating+mutations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DOtt%26aufirst%3DH.%2BM.%26aulast%3DGanji%26aufirst%3DG.%26aulast%3DKorenchuk%26aufirst%3DS.%26aulast%3DThompson%26aufirst%3DC.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DIii%26aufirst%3DA.%2BD.%26aulast%3DDiaz%26aufirst%3DE.%26aulast%3DLafrance%26aufirst%3DL.%2BV.%26aulast%3DMellinger%26aufirst%3DM.%26aulast%3DDuquenne%26aufirst%3DC.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DBrandt%26aufirst%3DM.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DVerma%26aufirst%3DS.%2BK.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26atitle%3DEZH2%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520for%2520lymphoma%2520with%2520EZH2-activating%2520mutations%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D108%26epage%3D112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Knutson, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warholic, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sneeringer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sacks, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keilhack, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span> </span><span class="NLM_article-title">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">890</span><span class="NLM_x">–</span> <span class="NLM_lpage">896</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1038%2Fnchembio.1084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=23023262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=890-896&author=S.+K.+Knutsonauthor=T.+J.+Wigleauthor=N.+M.+Warholicauthor=C.+J.+Sneeringerauthor=C.+J.+Allainauthor=C.+R.+Klausauthor=J.+D.+Sacksauthor=A.+Raimondiauthor=C.+R.+Majerauthor=J.+Songauthor=M.+P.+Scottauthor=L.+Jinauthor=J.+J.+Smithauthor=E.+J.+Olhavaauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=V.+M.+Richonauthor=R.+A.+Copelandauthor=H.+Keilhackauthor=R.+M.+Pollockauthor=K.+W.+Kuntz&title=A+selective+inhibitor+of+EZH2+blocks+H3K27+methylation+and+kills+mutant+lymphoma+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span></div><div class="casAuthors">Knutson, Sarah K.; Wigle, Tim J.; Warholic, Natalie M.; Sneeringer, Christopher J.; Allain, Christina J.; Klaus, Christine R.; Sacks, Joelle D.; Raimondi, Alejandra; Majer, Christina R.; Song, Jeffrey; Scott, Margaret Porter; Jin, Lei; Smith, Jesse J.; Olhava, Edward J.; Chesworth, Richard; Moyer, Mikel P.; Richon, Victoria M.; Copeland, Robert A.; Keilhack, Heike; Pollock, Roy M.; Kuntz, Kevin W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">890-896</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">EZH2 catalyzes trimethylation of histone H3 lysine 27 (H3K27).  Point mutations of EZH2 at Tyr 641 and Ala 677 occur in subpopulations of non-Hodgkin's lymphoma, where they drive H3K27 hypertrimethylation.  Here the authors report the discovery of EPZ005687, a potent inhibitor of EZH2 (Ki of 24 nM).  EPZ005687 has greater than 500-fold selectivity against 15 other protein methyltransferases and has 50-fold selectivity against the closely related enzyme EZH1.  The compd. reduces H3K27 methylation in various lymphoma cells; this translates into apoptotic cell killing in heterozygous Tyr 641 or Ala 677 mutant cells, with minimal effects on the proliferation of wild-type cells.  These data suggest that genetic alteration of EZH2 (for example, mutations at Tyr 641 or Ala 677) results in a crit. dependency on enzymic activity for proliferation (i.e., the equiv. of oncogene addiction), thus portending the clin. use of EZH2 inhibitors for cancers in which EZH2 is genetically altered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohlduVbTgWJrVg90H21EOLACvtfcHk0liWJCyI-465QQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP&md5=a9ad7a41505e3fd4aa087d9f5dae8825</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1084%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DSacks%26aufirst%3DJ.%2BD.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DKeilhack%26aufirst%3DH.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520EZH2%2520blocks%2520H3K27%2520methylation%2520and%2520kills%2520mutant%2520lymphoma%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D890%26epage%3D896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Verma, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaFrance, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duquenne, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suarez, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newlander, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romeril, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgess, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brackley, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scherzer, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aller, G. S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machutta, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diaz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tummino, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creasy, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. H.</span><span> </span><span class="NLM_article-title">Identification of potent, selective, cell-active inhibitors of the histone lysine methyltransferase EZH2</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1091</span><span class="NLM_x">–</span> <span class="NLM_lpage">1096</span></span><div class="citationLinks">[<a href="/doi/10.1021/ml3003346" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFChs77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=1091-1096&author=S.+K.+Vermaauthor=X.+Tianauthor=L.+V.+LaFranceauthor=C.+Duquenneauthor=D.+P.+Suarezauthor=K.+A.+Newlanderauthor=S.+P.+Romerilauthor=J.+L.+Burgessauthor=S.+W.+Grantauthor=J.+A.+Brackleyauthor=A.+P.+Gravesauthor=D.+A.+Scherzerauthor=A.+Shuauthor=C.+Thompsonauthor=H.+M.+Ottauthor=G.+S.+V.+Allerauthor=C.+A.+Machuttaauthor=E.+Diazauthor=Y.+Jiangauthor=N.+W.+Johnsonauthor=S.+D.+Knightauthor=R.+G.+Krugerauthor=M.+T.+McCabeauthor=D.+Dhanakauthor=P.+J.+Tumminoauthor=C.+L.+Creasyauthor=W.+H.+Miller&title=Identification+of+potent%2C+selective%2C+cell-active+inhibitors+of+the+histone+lysine+methyltransferase+EZH2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Potent, Selective, Cell-Active Inhibitors of the Histone Lysine Methyltransferase EZH2</span></div><div class="casAuthors">Verma, Sharad K.; Tian, Xinrong; LaFrance, Louis V.; Duquenne, Celine; Suarez, Dominic P.; Newlander, Kenneth A.; Romeril, Stuart P.; Burgess, Joelle L.; Grant, Seth W.; Brackley, James A.; Graves, Alan P.; Scherzer, Daryl A.; Shu, Art; Thompson, Christine; Ott, Heidi M.; Van Aller, Glenn S.; Machutta, Carl A.; Diaz, Elsie; Jiang, Yong; Johnson, Neil W.; Knight, Steven D.; Kruger, Ryan G.; McCabe, Michael T.; Dhanak, Dashyant; Tummino, Peter J.; Creasy, Caretha L.; Miller, William H.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1091-1096</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The histone H3-lysine 27 (H3K27) methyltransferase EZH2 plays a crit. role in regulating gene expression, and its aberrant activity is linked to the onset and progression of cancer.  As part of a drug discovery program targeting EZH2, we have identified highly potent, selective, S-(S'-adenosyl)-L-methionine (SAM)-competitive, and cell-active EZH2 inhibitors, including GSK926 I (R1 = Me, R2 = Q) and GSK343 I (R1 = n-Pr, R2 = Q1).  These compds. are small mol. chem. tools that would be useful to further explore the biol. of EZH2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8NUzf0zjo0bVg90H21EOLACvtfcHk0lic6-6Qy6A-fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFChs77K&md5=a6540bb712ee52cba05bd0b074d8b26a</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fml3003346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml3003346%26sid%3Dliteratum%253Aachs%26aulast%3DVerma%26aufirst%3DS.%2BK.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DLaFrance%26aufirst%3DL.%2BV.%26aulast%3DDuquenne%26aufirst%3DC.%26aulast%3DSuarez%26aufirst%3DD.%2BP.%26aulast%3DNewlander%26aufirst%3DK.%2BA.%26aulast%3DRomeril%26aufirst%3DS.%2BP.%26aulast%3DBurgess%26aufirst%3DJ.%2BL.%26aulast%3DGrant%26aufirst%3DS.%2BW.%26aulast%3DBrackley%26aufirst%3DJ.%2BA.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DScherzer%26aufirst%3DD.%2BA.%26aulast%3DShu%26aufirst%3DA.%26aulast%3DThompson%26aufirst%3DC.%26aulast%3DOtt%26aufirst%3DH.%2BM.%26aulast%3DAller%26aufirst%3DG.%2BS.%2BV.%26aulast%3DMachutta%26aufirst%3DC.%2BA.%26aulast%3DDiaz%26aufirst%3DE.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DJohnson%26aufirst%3DN.%2BW.%26aulast%3DKnight%26aufirst%3DS.%2BD.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26atitle%3DIdentification%2520of%2520potent%252C%2520selective%252C%2520cell-active%2520inhibitors%2520of%2520the%2520histone%2520lysine%2520methyltransferase%2520EZH2%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D1091%26epage%3D1096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Zheng, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibáñez, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blum, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amaya, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, M.</span><span> </span><span class="NLM_article-title">Sinefungin derivatives as inhibitors and structure probes of protein lysine methyltransferase SETD2</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">134</span><span class="NLM_x">, </span> <span class="NLM_fpage">18004</span><span class="NLM_x">–</span> <span class="NLM_lpage">18014</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja307060p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVOhsrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=18004-18014&author=W.+Zhengauthor=G.+Ib%C3%A1%C3%B1ezauthor=H.+Wuauthor=G.+Blumauthor=H.+Zengauthor=A.+Dongauthor=F.+Liauthor=T.+Hajianauthor=A.+Allali-Hassaniauthor=M.+F.+Amayaauthor=A.+Siarheyevaauthor=W.+Yuauthor=P.+J.+Brownauthor=M.+Schapiraauthor=M.+Vedadiauthor=J.+Minauthor=M.+Luo&title=Sinefungin+derivatives+as+inhibitors+and+structure+probes+of+protein+lysine+methyltransferase+SETD2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Sinefungin Derivatives as Inhibitors and Structure Probes of Protein Lysine Methyltransferase SETD2</span></div><div class="casAuthors">Zheng, Weihong; Ibanez, Glorymar; Wu, Hong; Blum, Gil; Zeng, Hong; Dong, Aiping; Li, Fengling; Hajian, Taraneh; Allali-Hassani, Abdellah; Amaya, Maria F.; Siarheyeva, Alena; Yu, Wenyu; Brown, Peter J.; Schapira, Matthieu; Vedadi, Masoud; Min, Jinrong; Luo, Minkui</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">18004-18014</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Epigenetic regulation is involved in numerous physiol. and pathogenic processes.  Among the key regulators that orchestrate epigenetic signaling are over 50 human protein lysine methyltransferases (PKMTs).  Interrogation of the functions of individual PKMTs can be facilitated by target-specific PKMT inhibitors.  Given the emerging need for such small mols., an approach was envisioned to identify target-specific methyltransferase inhibitors by screening privileged small-mol. scaffolds against diverse methyltransferases.  In this work, the feasibility of such an approach was demonstrated by identifying the inhibitors of SETD2.  N-Pr sinefungin (Pr-SNF) was shown to interact preferentially with SETD2 by matching the distinct transition-state features of SETD2's catalytically active conformer.  With Pr-SNF as a structure probe, the dual roles of SETD2's post-SET loop in regulating substrate access through a distinct topol. reconfiguration was revealed.  Privileged sinefungin scaffolds are expected to have broad use as structure and chem. probes of methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv-sWI1BJjg7Vg90H21EOLACvtfcHk0li5GvhV7oaN2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVOhsrbL&md5=30f5ba653007349df64e095f1c2cf188</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fja307060p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja307060p%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DIb%25C3%25A1%25C3%25B1ez%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DBlum%26aufirst%3DG.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DAmaya%26aufirst%3DM.%2BF.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DM.%26atitle%3DSinefungin%2520derivatives%2520as%2520inhibitors%2520and%2520structure%2520probes%2520of%2520protein%2520lysine%2520methyltransferase%2520SETD2%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D18004%26epage%3D18014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobrovetsky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolshan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loppnau, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, Peter J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">An allosteric inhibitor of protein arginine methyltransferase 3</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1425</span><span class="NLM_x">–</span> <span class="NLM_lpage">1435</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1016%2Fj.str.2012.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=22795084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVektr3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=1425-1435&author=A.+Siarheyevaauthor=G.+Senisterraauthor=A.+Allali-Hassaniauthor=A.+Dongauthor=E.+Dobrovetskyauthor=G.+A.+Wasneyauthor=I.+Chauauthor=R.+Marcellusauthor=T.+Hajianauthor=F.+Liuauthor=I.+Korboukhauthor=D.+Smilauthor=Y.+Bolshanauthor=J.+Minauthor=H.+Wuauthor=H.+Zengauthor=P.+Loppnauauthor=G.+Podaauthor=C.+Griffinauthor=A.+Amanauthor=Peter+J.+Brownauthor=J.+Jinauthor=R.+Al-awarauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadi&title=An+allosteric+inhibitor+of+protein+arginine+methyltransferase+3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">An Allosteric Inhibitor of Protein Arginine Methyltransferase 3</span></div><div class="casAuthors">Siarheyeva, Alena; Senisterra, Guillermo; Allali-Hassani, Abdellah; Dong, Aiping; Dobrovetsky, Elena; Wasney, Gregory A.; Chau, Irene; Marcellus, Richard; Hajian, Taraneh; Liu, Feng; Korboukh, Ilia; Smil, David; Bolshan, Yuri; Min, Jinrong; Wu, Hong; Zeng, Hong; Loppnau, Peter; Poda, Gennadiy; Griffin, Carly; Aman, Ahmed; Brown, Peter J.; Jin, Jian; Al-awar, Rima; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1425-1435</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">PRMT3, a protein arginine methyltransferase, has been shown to influence ribosomal biosynthesis by catalyzing the dimethylation of the 40S ribosomal protein S2.  Although PRMT3 has been reported to be a cytosolic protein, it has been shown to methylate histone H4 peptide (H4 1-24) in vitro.  Here, we report the identification of a PRMT3 inhibitor (1-(benzo[d][1,2,3]thiadiazol-6-yl)-3-(2-cyclohexenylethyl)urea; compd. 1) with IC50 value of 2.5 μM by screening a library of 16,000 compds. using H4 (1-24) peptide as a substrate.  The crystal structure of PRMT3 in complex with compd. 1 as well as kinetic anal. reveals an allosteric mechanism of inhibition.  Mutating PRMT3 residues within the allosteric site or using compd. 1 analogs that disrupt interactions with allosteric site residues both abrogated binding and inhibitory activity.  These data demonstrate an allosteric mechanism for inhibition of protein arginine methyltransferases, an emerging class of therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruLurDJu10tLVg90H21EOLACvtfcHk0li5GvhV7oaN2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVektr3N&md5=c745acac0beb494318f4577293281ddf</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2012.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2012.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBolshan%26aufirst%3DY.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DLoppnau%26aufirst%3DP.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DGriffin%26aufirst%3DC.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DPeter%2BJ.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DAl-awar%26aufirst%3DR.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DAn%2520allosteric%2520inhibitor%2520of%2520protein%2520arginine%2520methyltransferase%25203%26jtitle%3DStructure%26date%3D2012%26volume%3D20%26spage%3D1425%26epage%3D1435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Ward, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poesche, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgins, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heard, D.</span><span> </span><span class="NLM_article-title">458. 1, 2, 3-Benzothiadiazole. Part I. Nitro-, amino-, and hydroxy-derivatives</span> <span class="citation_source-journal">J. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1962</span><span class="NLM_x">, </span> <span class="NLM_fpage">2374</span><span class="NLM_x">–</span> <span class="NLM_lpage">2379</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1962&pages=2374-2379&author=E.+Wardauthor=W.+Poescheauthor=D.+Higginsauthor=D.+Heard&title=458.+1%2C+2%2C+3-Benzothiadiazole.+Part+I.+Nitro-%2C+amino-%2C+and+hydroxy-derivatives"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DE.%26aulast%3DPoesche%26aufirst%3DW.%26aulast%3DHiggins%26aufirst%3DD.%26aulast%3DHeard%26aufirst%3DD.%26atitle%3D458.%25201%252C%25202%252C%25203-Benzothiadiazole.%2520Part%2520I.%2520Nitro-%252C%2520amino-%252C%2520and%2520hydroxy-derivatives%26jtitle%3DJ.%2520Chem.%2520Soc.%26date%3D1962%26spage%3D2374%26epage%3D2379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Rostami, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chudzinski, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lough, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, M. S.</span><span> </span><span class="NLM_article-title"><i>N</i>,<i>N</i>′-Diarylsquaramides: general, high-yielding synthesis and applications in colorimetric anion sensing</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">75</span><span class="NLM_x">, </span> <span class="NLM_fpage">3983</span><span class="NLM_x">–</span> <span class="NLM_lpage">3992</span></span><div class="citationLinks">[<a href="/doi/10.1021/jo100104g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2010&pages=3983-3992&author=A.+Rostamiauthor=A.+Colinauthor=X.+Y.+Liauthor=M.+G.+Chudzinskiauthor=A.+J.+Loughauthor=M.+S.+Taylor&title=N%2CN%E2%80%B2-Diarylsquaramides%3A+general%2C+high-yielding+synthesis+and+applications+in+colorimetric+anion+sensing"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fjo100104g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo100104g%26sid%3Dliteratum%253Aachs%26aulast%3DRostami%26aufirst%3DA.%26aulast%3DColin%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DX.%2BY.%26aulast%3DChudzinski%26aufirst%3DM.%2BG.%26aulast%3DLough%26aufirst%3DA.%2BJ.%26aulast%3DTaylor%26aufirst%3DM.%2BS.%26atitle%3DN%252CN%25E2%2580%25B2-Diarylsquaramides%253A%2520general%252C%2520high-yielding%2520synthesis%2520and%2520applications%2520in%2520colorimetric%2520anion%2520sensing%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2010%26volume%3D75%26spage%3D3983%26epage%3D3992" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Minor, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cymborowski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otwinowski, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chruszcz, M.</span><span> </span><span class="NLM_article-title">HKL-3000: the integration of data reduction and structure solution—from diffraction images to an initial model in minutes</span> <span class="citation_source-journal">Acta Crystallogr., D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">859</span><span class="NLM_x">–</span> <span class="NLM_lpage">866</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1107%2FS0907444906019949" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=16855301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=859-866&author=W.+Minorauthor=M.+Cymborowskiauthor=Z.+Otwinowskiauthor=M.+Chruszcz&title=HKL-3000%3A+the+integration+of+data+reduction+and+structure+solution%E2%80%94from+diffraction+images+to+an+initial+model+in+minutes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">HKL-3000: the integration of data reduction and structure solution - from diffraction images to an initial model in minutes</span></div><div class="casAuthors">Minor, Wladek; Cymborowski, Marcin; Otwinowski, Zbyszek; Chruszcz, Maksymilian</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">859-866</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A new approach that integrates data collection, data redn., phasing and model building significantly accelerates the process of structure detn. and on av. minimizes the no. of data sets and synchrotron time required for structure soln.  Initial testing of the HKL-3000 system (the beta version was named HKL-2000_ph) with more than 140 novel structure detns. has proven its high value for MAD/SAD expts.  The heuristics for choosing the best computational strategy at different data resoln. limits of phasing signal and crystal diffraction are being optimized.  The typical end result is an interpretable electron-d. map with a partially built structure and, in some cases, an almost complete refined model.  The current development is oriented towards very fast structure soln. in order to provide feedback during the diffraction expt.  Work is also proceeding towards improving the quality of phasing calcn. and model building.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0c0DYtjFP7rVg90H21EOLACvtfcHk0ljAIf-scBFYjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D&md5=2cd2fe288f954837abb4acae4c008a2f</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1107%2FS0907444906019949&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906019949%26sid%3Dliteratum%253Aachs%26aulast%3DMinor%26aufirst%3DW.%26aulast%3DCymborowski%26aufirst%3DM.%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DChruszcz%26aufirst%3DM.%26atitle%3DHKL-3000%253A%2520the%2520integration%2520of%2520data%2520reduction%2520and%2520structure%2520solution%25E2%2580%2594from%2520diffraction%2520images%2520to%2520an%2520initial%2520model%2520in%2520minutes%26jtitle%3DActa%2520Crystallogr.%252C%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D859%26epage%3D866" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teplyakov, A.</span><span> </span><span class="NLM_article-title">MOLREP: an automated program for molecular replacement</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1022</span><span class="NLM_x">–</span> <span class="NLM_lpage">1025</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+an+automated+program+for+molecular+replacement"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0ljAIf-scBFYjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520an%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lgQd915WDOqhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lgQd915WDOqhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arendall, W. B.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keedy, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Immormino, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span> <span class="citation_source-journal">Acta Crystallogr., D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">12</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0lhymHAm0ckVUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Abagyan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Totrov, M.</span><span> </span><span class="NLM_article-title">Biased probability Monte Carlo conformational searches and electrostatic calculations for peptides and proteins</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">235</span><span class="NLM_x">, </span> <span class="NLM_fpage">983</span><span class="NLM_x">–</span> <span class="NLM_lpage">1002</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Fjm3018332&amp;key=10.1006%2Fjmbi.1994.1052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Fjm3018332&amp;key=8289329" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Fjm3018332&amp;key=1%3ACAS%3A528%3ADyaK2cXhsFyis7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=235&publication_year=1994&pages=983-1002&author=R.+Abagyanauthor=M.+Totrov&title=Biased+probability+Monte+Carlo+conformational+searches+and+electrostatic+calculations+for+peptides+and+proteins"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Biased probability Monte Carlo conformational searches and electrostatic calculations for peptides and proteins</span></div><div class="casAuthors">Abagyan, Ruben; Totrov, Maxim</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">235</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">983-1002</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">Two major components are required for a successful prediction of the three-dimensional structure of peptides and proteins: an efficient global optimization procedure which is capable of finding an appropriate local min. for the strongly anisotropic function of hundreds of variables and a set of free energy components for a protein mol. in soln. which are computationally inexpensive enough to be used in the search procedure, yet sufficiently accurate to ensure the uniqueness of the native conformation.  The authors here found an efficient way to make a random step in a Monte Carlo procedure given knowledge of the energy or statistical properties of conformational subspaces (e.g., φ-ψ zones or side-chain torsion angles).  This biased probability Monte Carlo (BPMC) procedure randomly selects the subspace first, then makes a step to a new random position independent of the previous position, but according to the predefined continuous probability distribution.  The random step is followed by a local minimization in torsion angle space.  The positions, sizes and preferences for high-probability zones on φ-ψ maps and χ-angle maps were calcd. for different residue types from the representative set of 191 and 161 protein 3D-structures, resp.  A fast and precise method to evaluate the electrostatic energy of a protein in soln. is developed and combined with the BPMC procedure.  The method is based on the modified spherical image charge approxn., efficiently projected onto a mol. of arbitrary shape.  Comparison with the finite-difference solns. of the Poisson-Boltzmann equation shows high accuracy for the authors' approach.  The BPMC procedure is applied successfully to the structure prediction of 12- and 16-residue synthetic peptides and the detn. of protein structure from NMR data, with the Ig binding domain of streptococcal protein G as an example.  The BPMC runs display much better convergence properties than the non-biased simulations.  The advantage of a true global optimization procedure for NMR structure detn. is its ability to cope with local min. originating from data errors and ambiguities in NMR data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZIMjvYsp0UbVg90H21EOLACvtfcHk0lhymHAm0ckVUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXhsFyis7k%253D&md5=d88445e688ef9c522fb87e4ca101f778</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1994.1052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1994.1052%26sid%3Dliteratum%253Aachs%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DTotrov%26aufirst%3DM.%26atitle%3DBiased%2520probability%2520Monte%2520Carlo%2520conformational%2520searches%2520and%2520electrostatic%2520calculations%2520for%2520peptides%2520and%2520proteins%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1994%26volume%3D235%26spage%3D983%26epage%3D1002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SMQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SMQ','PDB','3SMQ'); return false;">PDB: 3SMQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HSG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HSG','PDB','4HSG'); return false;">PDB: 4HSG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2FYT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2FYT','PDB','2FYT'); return false;">PDB: 2FYT</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i71"><a href="/doi/suppl/10.1021/jm3018332">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_42668"></div></div></div></div></div><hr /></hr><p class="last">Methyltransferase assay components and conditions, data collection and refinement statistics of the compound <b>14u</b>–PRMT3 cocrystal structure, and <sup>1</sup>H and <sup>13</sup>C NMR spectra of compounds <b>14u</b> and <b>24</b>. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm3018332/suppl_file/jm3018332_si_001.pdf">jm3018332_si_001.pdf (465.11 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm3018332&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Fjm3018332%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2013.56.issue-5%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm3018332" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679999aa8cfa3c88","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
