
<html lang="en"     class="pb-page"  data-request-id="9c7d320b-03aa-4248-9ed9-d5c203925dcb"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.6b00709;issue:issue:10.1021/jmcmar.2016.59.issue-15;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery of a Selective Aurora A Kinase Inhibitor by Virtual Screening" /></meta><meta name="dc.Creator" content="Falco  Kilchmann" /></meta><meta name="dc.Creator" content="Maria J.  Marcaida" /></meta><meta name="dc.Creator" content="Sachin  Kotak" /></meta><meta name="dc.Creator" content="Thomas  Schick" /></meta><meta name="dc.Creator" content="Silvan D.  Boss" /></meta><meta name="dc.Creator" content="Mahendra  Awale" /></meta><meta name="dc.Creator" content="Pierre  Gönczy" /></meta><meta name="dc.Creator" content="Jean-Louis  Reymond" /></meta><meta name="dc.Description" content="Here we report the discovery of a selective inhibitor of Aurora A, a key regulator of cell division and potential anticancer target. We used the atom category extended ligand overlap score (xLOS), ..." /></meta><meta name="Description" content="Here we report the discovery of a selective inhibitor of Aurora A, a key regulator of cell division and potential anticancer target. We used the atom category extended ligand overlap score (xLOS), ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 20, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b00709" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b00709" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b00709" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b00709" /></link>
        
    
    

<title>Discovery of a Selective Aurora A Kinase Inhibitor by Virtual Screening | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b00709" /></meta><meta property="og:title" content="Discovery of a Selective Aurora A Kinase Inhibitor by Virtual Screening" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0013.jpeg" /></meta><meta property="og:description" content="Here we report the discovery of a selective inhibitor of Aurora A, a key regulator of cell division and potential anticancer target. We used the atom category extended ligand overlap score (xLOS), a 3D ligand-based virtual screening method recently developed in our group, to select 437 shape and pharmacophore analogs of reference kinase inhibitors. Biochemical screening uncovered two inhibitor series with scaffolds unprecedented among kinase inhibitors. One of them was successfully optimized by structure-based design to a potent Aurora A inhibitor (IC50 = 2 nM) with very high kinome selectivity for Aurora kinases. This inhibitor locks Aurora A in an inactive conformation and disrupts binding to its activator protein TPX2, which impairs Aurora A localization at the mitotic spindle and induces cell division defects. This phenotype can be rescued by inhibitor-resistant Aurora A mutants. The inhibitor furthermore does not induce Aurora B specific effects in cells." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b00709"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b00709">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b00709&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b00709&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b00709&amp;href=/doi/10.1021/acs.jmedchem.6b00709" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 15</span><span class="cit-fg-pageRange">, 7188-7211</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/15" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b00705" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.6b00730" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of a Selective Aurora A Kinase Inhibitor by Virtual Screening</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Falco++Kilchmann">Falco Kilchmann</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Maria+J.++Marcaida">Maria J. Marcaida</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sachin++Kotak">Sachin Kotak</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Thomas++Schick">Thomas Schick</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Silvan+D.++Boss">Silvan D. Boss</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mahendra++Awale">Mahendra Awale</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Pierre++G%C3%B6nczy">Pierre Gönczy</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jean-Louis++Reymond">Jean-Louis Reymond</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Chemistry and Biochemistry, National Center of Competence in Research NCCR Chemical Biology and NCCR TransCure, University of Berne, Freiestrasse 3, 3012 Berne, Switzerland</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Swiss Institute for Experimental Cancer Research (ISREC), School of Life Sciences, National Center of Competence in Research NCCR Chemical Biology, Swiss Federal Institute of Technology (EPFL), CH-1015 Lausanne, Switzerland</span></div><div class="corresp-info"><strong>*</strong>Phone: +41216930711. E-mail: <a href="/cdn-cgi/l/email-protection#97e7fef2e5e5f2b9f0f8f9f4edeed7f2e7f1fbb9f4ff"><span class="__cf_email__" data-cfemail="66160f0314140348010908051c1f260316000a48050e">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>Phone: +41316314325. E-mail: <a href="/cdn-cgi/l/email-protection#432926222d6e2f2c362a306d31263a2e2c2d27032720216d362d2a21266d202b"><span class="__cf_email__" data-cfemail="543e31353a79383b213d277a26312d393b3a30143037367a213a3d36317a373c">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b00709&amp;href=/doi/10.1021%2Facs.jmedchem.6b00709" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 15</span><span class="cit-pageRange">, 7188–7211</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 8, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>10 May 2016</li><li><span class="item_label"><b>Published</b> online</span>20 July 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 August 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b00709" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b00709</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7188%26pageCount%3D24%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DFalco%2BKilchmann%252C%2BMaria%2BJ.%2BMarcaida%252C%2BSachin%2BKotak%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D15%26contentID%3Dacs.jmedchem.6b00709%26title%3DDiscovery%2Bof%2Ba%2BSelective%2BAurora%2BA%2BKinase%2BInhibitor%2Bby%2BVirtual%2BScreening%26numPages%3D24%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7211%26publicationDate%3DAugust%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b00709"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">4472</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">39</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b00709" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of a Selective Aurora A Kinase Inhibitor by Virtual Screening&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Falco&quot;,&quot;last_name&quot;:&quot;Kilchmann&quot;},{&quot;first_name&quot;:&quot;Maria&quot;,&quot;last_name&quot;:&quot;J. Marcaida&quot;},{&quot;first_name&quot;:&quot;Sachin&quot;,&quot;last_name&quot;:&quot;Kotak&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;Schick&quot;},{&quot;first_name&quot;:&quot;Silvan&quot;,&quot;last_name&quot;:&quot;D. Boss&quot;},{&quot;first_name&quot;:&quot;Mahendra&quot;,&quot;last_name&quot;:&quot;Awale&quot;},{&quot;first_name&quot;:&quot;Pierre&quot;,&quot;last_name&quot;:&quot;Gönczy&quot;},{&quot;first_name&quot;:&quot;Jean-Louis&quot;,&quot;last_name&quot;:&quot;Reymond&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;20&quot;,&quot;issue&quot;:&quot;15&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;7188-7211&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b00709&quot;},&quot;abstract&quot;:&quot;Here we report the discovery of a selective inhibitor of Aurora A, a key regulator of cell division and potential anticancer target. We used the atom category extended ligand overlap score (xLOS), a 3D ligand-based virtual screening method recently developed in our group, to select 437 shape and pharmacophore analogs of reference kinase inhibitors. Biochemical screening uncovered two inhibitor series with scaffolds unprecedented among kinase inhibitors. One of them was successfully optimized by structure-based design to a potent Aurora A inhibitor (IC50 = 2 nM) with very high kinome selectivity for Aurora kinases. This inhibitor locks Aurora A in an inactive conformation and disrupts binding to its activator protein TPX2, which impairs Aurora A localization at the mitotic spindle and induces cell division defects. This phenotype can be rescued by inhibitor-resistant Aurora A mutants. The inhibitor furthermore does not induce Aurora B specific effects in cells.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00709&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00709" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00709&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00709" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00709&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00709" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b00709&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00709&amp;href=/doi/10.1021/acs.jmedchem.6b00709" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b00709" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b00709" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b00709&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b00709%26sid%3Dliteratum%253Aachs%26pmid%3D27391133%26genre%3Darticle%26aulast%3DKilchmann%26date%3D2016%26atitle%3DDiscovery%2Bof%2Ba%2BSelective%2BAurora%2BA%2BKinase%2BInhibitor%2Bby%2BVirtual%2BScreening%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D15%26spage%3D7188%26epage%3D7211%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (4)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291074" title="Purification">Purification</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292201" title="Scaffolds">Scaffolds</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/15" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/jmcmar.2016.59.issue-15/20160811-01/jmcmar.2016.59.issue-15.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Here we report the discovery of a selective inhibitor of Aurora A, a key regulator of cell division and potential anticancer target. We used the atom category extended ligand overlap score (xLOS), a 3D ligand-based virtual screening method recently developed in our group, to select 437 shape and pharmacophore analogs of reference kinase inhibitors. Biochemical screening uncovered two inhibitor series with scaffolds unprecedented among kinase inhibitors. One of them was successfully optimized by structure-based design to a potent Aurora A inhibitor (IC<sub>50</sub> = 2 nM) with very high kinome selectivity for Aurora kinases. This inhibitor locks Aurora A in an inactive conformation and disrupts binding to its activator protein TPX2, which impairs Aurora A localization at the mitotic spindle and induces cell division defects. This phenotype can be rescued by inhibitor-resistant Aurora A mutants. The inhibitor furthermore does not induce Aurora B specific effects in cells.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87521" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87521" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Computer-aided drug design exploits accumulated experimental and theoretical knowledge of drug-target interactions to preselect small sets of compounds for experimental evaluation enabling faster and more efficient drug discovery.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1-3)</a> While structure-based methods exploit structural information on the protein targets,<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4, 5)</a> ligand-based virtual screening (LBVS) methods employ similarity to previously known compounds to propose new analogs for testing.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> In LBVS substructure similarity searches, for example, using Tanimoto similarity with Daylight-type substructure fingerprints,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> can be used to find available analogs with varying substituents around conserved scaffolds. On the other hand, shape and pharmacophore similarity methods, in particular those comparing the 3D structure of molecules either by overlays<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> (e.g., FlexS,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Topomer shape similarity searching,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Surflex-Sim,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> pharmACOphore,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> ROCS<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a>) or via 3D-fingerprints<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> (e.g., USRCAT,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> 3DXfp<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a>), can suggest analogs with entirely different scaffolds and thus open the way to new compound series. This transition to new scaffolds while preserving shapes and pharmacophores is referred to as scaffold-hopping.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div><div class="NLM_p">Here we applied scaffold hopping LBVS to identify a new and selective inhibitor of Aurora A, an evolutionary conserved Ser/Thr kinase essential for proper progression through mitosis and one of the most intensively researched kinase targets due to its significance in cancer.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18, 19)</a> Although many potent inhibitors of this kinase are known, most of these also inhibit Aurora B, a chromosomal passenger protein required notably for cytokinesis and which shares over 85% sequence homology with Aurora A in the kinase domain.<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20, 21)</a> Alisertib (MLN8237, <b>1</b>), one of the most selective Aurora A inhibitors reported to date,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> is currently under advanced clinical investigation for the treatment of various solid and hematological malignancies (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>).<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Interestingly <b>1</b> and its close relative MLN8054 (<b>2</b>)<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> belong to a scaffold that is unique among the 4874 scaffolds (as defined by Bemis and Murcko)<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> occurring in 11,286 kinase inhibitors with potencies better than 50 nM listed in ChEMBL.<a onclick="showRef(event, 'ref26 ref27 ref28 ref29 ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28 ref29 ref30 ref31 ref32">(26-32)</a> Furthermore, the diversity of scaffolds is quite high among Aurora A kinase inhibitors, with 174 different scaffolds occurring among 329 inhibitors with potencies better than 50 nM listed in ChEMBL. This suggests that each inhibitor represents an already optimized compound for each scaffold and that discovering further selective inhibitors for this particular kinase might require identifying different scaffolds, for example using scaffold hopping LBVS.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23, 24)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0001.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of reference kinase inhibitors and new inhibitors discussed in this paper.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To perform scaffold hopping LBVS, we used the atom category extended ligand overlap score (xLOS),<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> a 3D shape and pharmacophore similarity algorithm recently developed in our group. The xLOS algorithm compares molecules by maximizing the ligand overlap score<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34, 35)</a> computed separately for three atom categories (hydrophobic, hydrogen-bond acceptor, and hydrogen-bond donor atoms) previously found to be useful for pharmacophore fingerprint design.<a onclick="showRef(event, 'ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref36 ref37">(36, 37)</a> The method shows excellent scaffold hopping properties, which may be attributed to the fact that the algorithm compares molecules by atom positions only without considering atom connectivity patterns. Furthermore, xLOS penalizes nonoverlapping parts of seed and query molecules and therefore gives higher scores to molecules with a relatively small size, which are most suitable for initial activity screening since these may later be optimized by the addition of substituents.<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29, 30)</a> In our initial implementation we used xLOS to optimize weak and unselective inhibitors of the calcium channel TRPV6 by early scaffold redesign, which enabled the identification of a new scaffold leading to the first potent and selective inhibitor for this target.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The method was also recently used for the identification of a new noncompetitive inhibitor with an original scaffold for the iron transporter DMT1, for which only very few inhibitors have been described to date.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></div><div class="NLM_p last">As detailed below, we used xLOS to search commercial catalogs for shape and pharmacophore analogs of 16 known kinase inhibitors and assembled a kinase screening set of 437 compounds featuring mostly different scaffolds compared to known kinase inhibitors and kinase screening compounds. Biochemical screening against Aurora A gave 18 hits featuring 12 different scaffolds, two of which were pursued further. While attempts to optimize benzimidazole <b>5</b> only led to modest improvements with its hydroxyl analog <b>6</b>, we succeeded in modifying the phenylimino-thiazolidinone hit <b>7</b> and its close analog <b>8</b> identified in a second round of LBVS using structure-based drug design and obtained the potent inhibitor <b>9</b>. Compound <b>9</b> is a type II kinase inhibitor displaying strong and selective binding to Aurora kinases against the entire kinome. Inhibitor <b>9</b> induces a selective Aurora A phenotype in cells without any Aurora B specific effects.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28530" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28530" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Virtual Screening</h3><div class="NLM_p">We assembled a kinase discovery set by searching the ZINC database comprising ∼12 million compounds<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> for shape and pharmacophore analogs of 16 known inhibitors of Aurora A and other kinases (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_001.pdf" class="ext-link">Supplementary Figure 1</a>) using the xLOS algorithm described above. We focused our selection on two vendor catalogs totaling ∼1.2 million compounds and purchased 437 compounds selected by clustering the 1000 top scoring compounds against each of the 16 references, taking into account scaffold diversity, potential for derivatization, and absence of reactive functional groups (aldehydes, quinones, chloroketones, imines, etc.). Pan assay interference compounds (PAINS) were however not specifically excluded.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Indeed the recently available automated tool at the docking.org Web site<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> flags 27 of the 437 compounds (6.2%) as PAINS, which is significantly higher than for DrugBank (67 PAINS from 1775 drugs, 3.8%) or a random selection of ChEMBL (86 PAINS from 2000 ChEMBL compounds, 4.3%). The selected compounds had relatively high xLOS scores indicating significant shape and pharmacophore similarity to the reference inhibitors. These compounds also showed relatively low substructure similarity to these reference inhibitors as measured by the Tanimoto similarity coefficient relative to a 1024-bit Daylight-type substructure fingerprint (TSfp <0.7), although this similarity was still higher than the average Tanimoto value of 0.23 for all compounds used in LBVS (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>a). Our screening set was comparable to commercially available kinase discovery sets in terms of molecular properties (molecular size, number of hydrogen-bond donor and acceptor atoms, number of rotatable bonds). Both of these discovery sets contained compounds that were on average smaller and with fewer hydrogen-bond donor atoms and rotatable bonds compared to kinase inhibitors reported in ChEMBL, in particular the more potent ones, reflecting the difference between screening compounds and optimized inhibitors (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>c). Most importantly, our screening set showed very low overlap with known kinase inhibitors and discovery sets in terms of molecules and scaffolds, highlighting its potential for identifying scaffolds not previously used for kinase inhibition (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0002.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Kinase library design and screening. (a) Combined frequency histogram as a function of xLOS score (black line, ∼1.2 million commercial cpds from two catalogs, right vertical axis) and scatter plot of substructure similarity (TSfp: Tanimoto coefficient of a 1024-bit Daylight type substructure fingerprint) versus xLOS score for the selected 437 screening compounds (black dots) and active Aurora A hits (red dots) against their most similar reference inhibitor. (b) Combined frequency histogram and similarity scatter plot as (a) for 212 compounds selected in the second round of virtual screening compared to their reference first round hit. (c) Property profile of kinase screening sets and inhibitors in terms of heavy atom count (HAC), hydrogen-bond acceptor atoms (HBA), hydrogen-bond donor atoms (HBD), and rotatable bond count (RBC). See <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> for details of the composition of the sets.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Compound and Scaffold Overlap between Different Kinase Inhibitor and Screening Sets<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char=".">kinase ChEMBL</th><th class="colsep0 rowsep0" align="center" char=".">screening commercial</th><th class="colsep0 rowsep0" align="center" char=".">kinase 50 nM</th><th class="colsep0 rowsep0" align="center" char=".">aurora 50 nM</th><th class="colsep0 rowsep0" align="center" char=".">screening xLOS</th><th class="colsep0 rowsep0" align="center" char=".">hits xLOS</th><th class="colsep0 rowsep0" align="center" char=".">unique</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="8" align="left">Shared Compounds</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">kinase ChEMBL</td><td class="colsep0 rowsep0" align="char" char=".">51643</td><td class="colsep0 rowsep0" align="char" char=".">139</td><td class="colsep0 rowsep0" align="char" char=".">11286</td><td class="colsep0 rowsep0" align="char" char=".">329</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">40222</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">screening commercial</td><td class="colsep0 rowsep0" align="char" char=".">139</td><td class="colsep0 rowsep0" align="char" char=".">23979</td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">9</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">23831</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">kinase 50 nM</td><td class="colsep0 rowsep0" align="char" char=".">11286</td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="char" char=".">11286</td><td class="colsep0 rowsep0" align="char" char=".">329</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">aurora 50 nM</td><td class="colsep0 rowsep0" align="char" char=".">329</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">329</td><td class="colsep0 rowsep0" align="char" char=".">329</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">screening xLOS</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char=".">9</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">437</td><td class="colsep0 rowsep0" align="char" char=".">15</td><td class="colsep0 rowsep0" align="char" char=".">410</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hits xLOS</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">15</td><td class="colsep0 rowsep0" align="char" char=".">24</td><td class="colsep0 rowsep0" align="char" char=".">9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="8" align="left">Shared Scaffolds</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">kinase ChEMBL</td><td class="colsep0 rowsep0" align="char" char=".">23400</td><td class="colsep0 rowsep0" align="char" char=".">836</td><td class="colsep0 rowsep0" align="char" char=".">4874</td><td class="colsep0 rowsep0" align="char" char=".">174</td><td class="colsep0 rowsep0" align="char" char=".">66</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">17719</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">screening commercial</td><td class="colsep0 rowsep0" align="char" char=".">836</td><td class="colsep0 rowsep0" align="char" char=".">13447</td><td class="colsep0 rowsep0" align="char" char=".">61</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">58</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char=".">12584</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">kinase 50 nM</td><td class="colsep0 rowsep0" align="char" char=".">4874</td><td class="colsep0 rowsep0" align="char" char=".">61</td><td class="colsep0 rowsep0" align="char" char=".">4874</td><td class="colsep0 rowsep0" align="char" char=".">174</td><td class="colsep0 rowsep0" align="char" char=".">9</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">aurora 50 nM</td><td class="colsep0 rowsep0" align="char" char=".">174</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">174</td><td class="colsep0 rowsep0" align="char" char=".">174</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">screening xLOS</td><td class="colsep0 rowsep0" align="char" char=".">66</td><td class="colsep0 rowsep0" align="char" char=".">58</td><td class="colsep0 rowsep0" align="char" char=".">9</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">247</td><td class="colsep0 rowsep0" align="char" char=".">15</td><td class="colsep0 rowsep0" align="char" char=".">145</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hits xLOS</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">15</td><td class="colsep0 rowsep0" align="char" char=".">18</td><td class="colsep0 rowsep0" align="char" char=".">3 (12)<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">Kinase ChEMBL: all kinase inhibitors found in ChEMBL; screening commercial: aggregated series of commercially available screening compounds for kinases; kinase 50 nM: ChEMBL kinase inhibitors reported with IC<sub>50</sub> < 50 nM; aurora 50 nM: ChEMBL aurora inhibitors with IC<sub>50</sub> < 50 nM; screening xLOS: screening set in the present study. Scaffolds are defined according to Bemis and Murcko.</p></div><div class="footnote" id="t1fn2"><sup>b</sup><p class="last">Scaffolds unique to the hits not considering overlap with the xLOS screening set.</p></div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Hit Identification</h3><div class="NLM_p last">We conducted a time-resolved FRET assay<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> to test the impact of our kinase screening set on Aurora A activity and found that 15 compounds inhibited Aurora A by more than 50% at 10 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_001.pdf" class="ext-link">Supplementary Figure 2</a>), corresponding to a 3.4% hit rate comparable to other LBVS guided screens.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> None of these hits were flagged as PAINS. An additional 205 analogs of these 15 hits were selected by further similarity searching using xLOS, which featured mostly very close analogs of the hits displaying both high xLOS and substructure similarity scores to these hits (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b). In this case a subsequent check at docking.org flagged six compounds as PAINS (see <a href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_001.pdf" class="ext-link">Supporting Information</a>), none of which appeared as hits. Biochemical testing of these second round LBVS compounds provided an additional 9 hits with comparable inhibition (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_001.pdf" class="ext-link">Supplementary Figure 3</a>). The compendium of 24 hits featured 18 scaffolds, 12 of which did not appear among commercial kinase screening compounds or known kinase inhibitors. Two of these hits were investigated closer, which were benzimidazole <b>5</b> (IC<sub>50</sub> = 6.5 μM), with a xLOS score of 1.1 relative to the known CDK and Aurora A/B inhibitor <b>3</b> (JNJ-7706621),<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> and the N-substituted thiazolidinone <b>7</b> (IC<sub>50</sub> = 2.0 μM), with a xLOS score of 1.01 relative to the known CDK inhibitor <b>4</b> (AT-7519) (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>).<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Hit <b>7</b> was confirmed in the second round of virtual screening by the identification of a more potent analog <b>8</b> (IC<sub>50</sub> = 0.39 μM). Both series represented scaffolds previously unknown among kinase inhibitors. Indeed only a single derivative of <b>5</b> and no bioactivity have been reported for this scaffold,<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> whereas the closest relative of <b>5</b> is the natural product indirubin (<b>10</b>), which is known to inhibit cyclin-dependent kinases (CDKs), but with which <b>5</b> only shares the cyclic skeleton (scaffold not considering heteroatoms and double bonds).<a onclick="showRef(event, 'ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref47 ref48">(47, 48)</a> Furthermore, although a variety of bioactive compounds with N-substituted thiazolidinones are known,<a onclick="showRef(event, 'ref49 ref50 ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref49 ref50 ref51 ref52">(49-52)</a> only the N-unsubstituted thiazolidinone scaffold has been used in relationship with kinases in the case of compound <b>12</b> and analogs developed as PIM kinase and CDK2 inhibitors.<a onclick="showRef(event, 'ref53 ref54'); return false;" href="javascript:void(0);" class="ref ref53 ref54">(53, 54)</a> Note that hit <b>7</b>, <b>8</b>, our optimized inhibitor <b>9</b>, and the known inhibitor <b>12</b> as well as the various reported bioactive thiazolidinones<a onclick="showRef(event, 'ref49 ref50 ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref49 ref50 ref51 ref52">(49-52)</a> are not flagged by the PAINS tool at docking.org and that their thiazolidinone core is allowed in the ZINC clean set.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> In the present context the steep structure–activity relationship (SAR) profile observed during the optimization of hit <b>7</b> and the relatively low glutathione (GSH) reactivity observed for <b>9</b> (see below <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>) confirm that this structural type, although listed as “rhodanine-like” PAINS in the original publication,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> does not behave as a typical PAINS.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Synthesis</h3><div class="NLM_p">Benzimidazole hit <b>5</b> was resynthesized from 1,2-diaminobenzene by condensation with carbonyl diimidazole to form the corresponding cyclic urea <b>13</b>, reaction with phosphorus oxychloride to form chlorobenzimidazole <b>19</b>, coupling with monoboc protected 1,2-diaminobenzene to intermediate <b>25</b>, and finally cyclization of the Boc group under basic conditions to form the cyclic urea group. This synthetic route was used to prepare <b>5</b> and its analogs <b>6</b> and <b>33</b>–<b>42</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Coupling of the monoboc protected 4-methyl-1,2-diaminobenzene <b>43</b> with chlorobenzimidazole <b>52</b> and cyclization gave benzimidazole <b>44</b> as an isomeric mixture. Oxidation of the methyl groups afforded the carboxylic acids <b>45</b> and <b>46</b> as a separable pair of isomers, which were subsequently converted to the corresponding <i>N</i>-methyl amide <b>47</b> and <b>48</b>. Finally syntheses involving <i>N</i>-methyl chlorobenzimidazole <b>49</b> on the one hand and boc protected <i>N</i>-methyl-1,2-diaminobenzene <b>52</b> on the other hand afforded the selectively <i>N</i>-methylated analogs <b>51</b> and <b>53</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0009.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Benzimidazole Hit <b>5</b> and Derivatives</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0010.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Further Analogs of Benzimidazole <b>5</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Analogs of phenylimino-thiazolidinone hit <b>7</b> were prepared by addition of various anilines to alkyl isothiocyanates, followed by regioselective cyclization with ethyl chloroacetate to form the thiazolidinone nucleus, and finally Knoevenagel condensation with aromatic aldehydes (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>).<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> The 2-aminopyridine analog <b>79</b>, prepared by condensation of intermediate <b>54</b> with Boc-protected 2-amino-4-formyl-pyridine followed by acidic deprotection (see <a href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_001.pdf" class="ext-link">Supporting Information</a>), was further elaborated by coupling to carboxylic acids by amide-bond formation to analogs <b>80</b>–<b>83</b> or to aromatic bromides by palladium coupling to analogs <b>84</b>–<b>88</b> and <b>9</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). Alternatively, the exocyclic double bond of <b>79</b> was either cyclopropanated to yield diastereoisomeric cyclopropanes <i>rac</i>-<b>89</b>/<i>rac</i>-<b>90</b> or reduced by hydrogenation to yield <i>rac</i>-<b>91</b>, which was subsequently extended by coupling to 6-bromo-nicotinic acid to yield <i>rac</i>-<b>92</b>.</div><figure id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0011.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Phenylimino-thiazolidinones</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0012.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of further thiazolidinone analogs</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> The Indirubin-Like Benzimidazole <b>5</b> Is a Type I Inhibitor</h3><div class="NLM_p">The benzimidazole derivative <b>5</b> was investigated first. Analogs <b>6</b>, <b>33</b>–<b>42</b>, <b>45</b>–<b>48</b>, <b>51</b>, and <b>52</b> provided an initial SAR profile. Many of these derivatives lost potency or only provided modest improvements compared to the initial hit compound. In particular N-methylation of the benzimidazole nitrogen atoms resulted in the inactive analogs <b>51</b> and <b>52</b>, suggesting that these hydrogen-bond donor atoms were essential for activity and might therefore interact with the hinge region of the kinase. Furthermore, a 3-fold gain in activity was observed upon introduction of a hydroxyl group at position 5 with <b>6</b> (IC<sub>50</sub> = 2.3 μM, <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>a). The 5-carboxy analog <b>46</b> was almost as active, but activity returned to lower levels with the corresponding <i>N</i>-methyl carboxamide <b>48</b>.</div><figure id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0003.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Indirubin-like scaffold <b>5</b>. (a) Structure–activity relationship profile. Aurora A inhibition was measured using an HTRF assay (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). (b) Crystal structure of the Aurora A ATP-binding site with bound <b>6</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZS0">4ZS0</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_001.pdf" class="ext-link">Supplementary Table S1</a>), with highlighted salt bridge between residues Lys162 and Glu181 (blue), DFG motif (pink) ,and HRD motif (yellow) characteristic of an active kinase conformation. (c) Schematic of contact residues between <b>6</b> and Aurora A. Dashed lines indicate hydrogen bonds, and red arcs represent hydrophobic interactions.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To understand the inhibition mechanism of the best inhibitor <b>6</b> at the atomic level, we solved the crystal structure of <b>6</b> bound to Aurora A, revealing a typical Type I mode of inhibition including the activation loop in the active, DFG-in conformation (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZS0">4ZS0</a>, 3.0–3.1 Å resolution, <a href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_001.pdf" class="ext-link">Supplementary Table 1</a> and <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>b).<a onclick="showRef(event, 'ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref56 ref57">(56, 57)</a> Compound <b>6</b> engaged in two hydrogen bonds between its benzimidazole and Ala213 in the hinge region of Aurora A, as suggested by the loss of activity observed upon N-methylation of the benzimidazole with analogs <b>51</b> and <b>52</b>. These interactions were analogous to those between the related 6-bromoindirubin (<b>11</b>) and its target CDK2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2BHE">2BHE</a>).<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> Moreover, <b>6</b> made additional hydrophobic contacts with Val147, Leu139, Leu194, and Leu263 and a hydrogen bond between its 6-hydroxyl group and Asp274 presumably explaining its stronger binding compared to <b>5</b> (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>c). Although no breakthrough derivative (IC<sub>50</sub> < 100 nM) could be identified in this series, the small size and high ligand efficiency of <b>6</b> (LE = 0.39) suggest that its rather original scaffold might be amenable to further optimization for Aurora A or other kinases.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> The Phenylimino-Thiazolidinone <b>7</b> Can Be Optimized to the Potent and Selective Inhibitor <b>9</b></h3><div class="NLM_p">The phenylimino-thiazolidinones <b>7</b> was investigated next. Initial SAR profiling indicated potential for optimization upon variation of the <i>N</i>-phenyl substituents with <b>8</b> and pinpointed to the essential role of the 4-pyridine ring since several analogs of <b>7</b> with alkyl-, halogen-, hydroxy-, or methoxy-substituted phenyl rings or a 3-pyridyl ring at that position were inactive (data not shown). Additional analogs were investigated to test if the initial gain in activity with <b>8</b> could be improved further (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>a). Activity profiling confirmed the optimal 4-ethylphenylimino substituent of <b>8</b> (<b>63</b>–<b>70</b> and <b>93</b>) and the need for a small alkyl substituent on the endocyclic nitrogen atom of the thiazolidinone (<b>75</b>–<b>78</b>). Furthermore, crystal structures of <b>76</b> and <b>78</b> established the <i>Z</i>-stereochemistry of both exocyclic double bonds of the thiazolidinone core (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_001.pdf" class="ext-link">Supplementary Figure 4</a>).</div><figure id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0004.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Optimization of the phenylimino-thiazolidinone hit <b>7</b> to the selective Aurora A inhibitor <b>9</b>. (a) Structure and activities of analogs with variations of R<sup>1</sup> and R<sup>2</sup>. (b) Optimization of the 2-pyridyl substituent. SI = selectivity index: IC<sub>50</sub> (Aurora B/INCENP)/IC<sub>50</sub> (Aurora A). (c) Structure and IC<sub>50</sub> curves for Aurora A and Aurora B/INCENP of the most potent inhibitor <b>9</b>. (d) Kinome profiling and <i>K</i><sub>D</sub> values for <b>9</b> toward its nine best target kinases.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A further crystal structure of <b>77</b> in complex with Aurora A (see below) suggested that placing an amino group at the 2-pyridyl position substituted with acyl or aromatic groups<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> might enhance hinge-binding interactions. Significant potency gains were indeed obtained by this approach, reaching low nanomolar values with <b>81</b>–<b>83</b> and <b>87</b>, <b>9</b> and <b>88</b> displaying substituents with a carboxylate or tetrazole group presumably interacting with Arg137 (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>b).</div><div class="NLM_p last">These inhibitors were remarkably selective for Aurora A against Aurora B used in complex with its physiological activator INCENP (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>c).<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> The most potent inhibitor <b>9</b> (IC<sub>50</sub> = 2 nM) was selected for further in depth analysis because this compound was also the most potent when tested on cells (see below). A kinome scan of 456 kinases using an active site directed binding competition assay<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> followed by determination of binding affinities showed that <b>9</b> was remarkably selective and bound tightly only to Aurora A (<i>K</i><sub>D</sub> = 5 nM), Aurora B (without INCENP, <i>K</i><sub>D</sub> = 10 nM), and Aurora C (<i>K</i><sub>D</sub> = 11 nM) (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>d).<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> Note that the biochemical inhibition and binding affinity measurements were both carried out with free Aurora A, which is autophosphorylated at T288, and gave comparable values. By contrast and due to the inactivity of the kinase alone, the biochemical inhibition of Aurora B was measured for its complex with its activator protein INCENP, to which it is always bound in cells, giving a 15-fold weaker value for inhibition of catalysis compared to the <i>K</i><sub>D</sub> value for binding measured with free Aurora B.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Binding of <b>9</b> to Aurora A Excludes the Activator Protein TPX2</h3><div class="NLM_p">To understand the inhibition mechanism at the atomic level, we solved the crystal structures of Aurora A in complex with <b>77</b> from the initial SAR study, with the highest affinity inhibitor <b>9</b> and with its phenyl tetrazole analog <b>88</b> (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZTQ">4ZTQ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZTR">4ZTR</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZTS">4ZTS</a>, resolution 2.8–2.9 Å, <a href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_001.pdf" class="ext-link">Supplementary Table 1</a>, <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>). All three inhibitors bind to the ATP pocket in the adenine binding region (Leu139, Val147, Ala160, Leu263) and occupy the hydrophobic back pocket (Leu194, Arg195, Leu196, Leu210, Phe275). The inhibitors induce an inactive DFG motif conformation also found in other Aurora A inhibitor complexes (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JBQ">4JBQ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JAI">4JAI</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2J5O">2J5O</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2BMC">2BMC</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3P9J">3P9J</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R22">3R22</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FDN">3FDN</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3K5U">3K5U</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BOG">4BOG</a>) with four characteristic features:<a onclick="showRef(event, 'ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref56 ref57">(56, 57)</a> (1) disruption of the R-spine (residues Leu196, Gln185, Phe275, His254, and Asp311); (2) an outward displacement of the αC helix compared to the active state (approximately 2.5 Å); (3) absence of the salt bridge between Lys162 and Glu181; and (4) disruption of the Asp256-Thr292-Lys258 hydrogen-bond network. In this conformation Asp274 is pointing away from the ATP pocket, and Phe275 is rotated upward thereby contacting the phenylimino moiety of the inhibitors. Furthermore, Trp277 (or Arg255 of the HRD motif in the case of <b>9</b>) forms H-bonds with Gln185 and Asp274, an arrangement clearly different from classical DFG-in or DFG-out conformations,<a onclick="showRef(event, 'ref63 ref64'); return false;" href="javascript:void(0);" class="ref ref63 ref64">(63, 64)</a> and which is specific of Aurora kinases, thus probably contributing to inhibitor selectivity.<a onclick="showRef(event, 'ref65 ref66'); return false;" href="javascript:void(0);" class="ref ref65 ref66">(65, 66)</a> The pyridine/aminopyridine group of the inhibitors engages in one or two hydrogen bonds with Ala213 in the hinge region. Finally, the terminal carboxylate of <b>9</b> forms a salt bridge with Arg137 and Arg220, analogous to <b>1</b> in the structure bound to Aurora A (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X81">2X81</a>),<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> thus explaining its stronger affinity compared to <b>77</b>.</div><figure id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0005.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Crystal structures of the 4-thiazolidinone derivatives bound to Aurora A. The 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map contoured at 1σ around the compounds is shown to highlight their unambiguous position in the pocket. Lys162 and Glu181 (blue), DFG motif (pink), and HRD motif (yellow) are highlighted. The catalytic pocket of Aurora A acquires an inactive conformation upon binding to <b>77</b> (a), <b>9</b> (b) and <b>88</b> (c). The lower panels show schematic representations of the interactions between the compounds and the protein. Dashed lines indicate hydrogen bonds, in particular interactions between the aminopyridine portion of the inhibitors and the hinge region (A213) and between the anionic carboxyl or tetrazole group and the guanidinium groups of R137 and R220. Residues involved in hydrophobic interactions are highlighted with a red arc.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Tryptophan fluorescence experiments showed that binding of <b>9</b> and of its analogs <b>77</b> and <b>88</b> perturbed the environment of Trp277, reflecting the rearrangement of the DFG loop and Trp277 interactions with Gln185 occurring upon binding of these inhibitors, as suggested by inspection of the crystal structures. By contrast, the type I inhibitor <b>6</b>, which does not induce rearrangement of the DFG loop, had no effect on tryptophan fluorescence (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_001.pdf" class="ext-link">Supplementary Figure 5</a>). Further analysis of these crystal structures indicated that this rearrangement should be incompatible with the rotation of the αC-helix and the subsequent salt bridge formation between Lys162 and Glu181 that take place upon binding to the microtubule-associated activator protein TPX2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1OL5">1OL5</a> or <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C3P">4C3P</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_001.pdf" class="ext-link">Supplementary Figure 6</a>).<a onclick="showRef(event, 'ref68 ref69'); return false;" href="javascript:void(0);" class="ref ref68 ref69">(68, 69)</a></div><div class="NLM_p">To test this hypothesis, we measured the binding of Aurora A to labeled TPX2<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5 ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26 ref27 ref28 ref29 ref30 ref31 ref32 ref33 ref34 ref35 ref36 ref37 ref38 ref39 ref40 ref41 ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5 ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26 ref27 ref28 ref29 ref30 ref31 ref32 ref33 ref34 ref35 ref36 ref37 ref38 ref39 ref40 ref41 ref42 ref43">(1-43)</a> using microscale thermophoresis (<i>K</i><sub>D</sub> ∼ 800 nM). As reported in <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>a, we found that this binding was indeed abolished in the presence of excess <b>9</b>. The incompatibility between <b>9</b> and TPX2 was further evidenced by the fact that TPX2 binding, which induced a 3-fold increase in the activity of Aurora A similar to other reports (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>b),<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> reduced inhibition by <b>9</b> from IC<sub>50</sub> = 2.0 nM to 1.4 μM (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>c). In control human cells, TPX2 helps in the localization of Aurora A to the spindle microtubules, but not to the centrosomes.<a onclick="showRef(event, 'ref70 ref71'); return false;" href="javascript:void(0);" class="ref ref70 ref71">(70, 71)</a> Our findings raised the possibility that addition of <b>9</b> should prevent localization from taking place on the spindle microtubules.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> Indeed, we found that Aurora A remained localized at centrosomes in the presence of <b>9</b> but was notably displaced from the spindle microtubules, as is the case for <b>1</b> (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>d).<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a></div><figure id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0006.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Binding of Aurora A to <b>9</b> excludes TPX2. (a) MST analysis of TPX2 binding to Aurora A kinase in presence and absence of compound <b>9</b>. Three independent measurements were performed; error bars represent the standard deviation (SD). (b) Aurora A kinase activity in the presence of TPX2 or the control protein BSA. (c) IC<sub>50</sub> curves for <b>9</b> in the presence of different TPX2 concentrations. (d) <b>9</b> impairs localization of Aurora A at the mitotic spindle. Representative images of HeLa Kyoto cells treated with compound <b>9</b> overnight and stained for DNA (blue), Aurora A (green), and TPX2 (red). TPX2 colocalizes with Aurora A in control cells (top row). <b>9</b> impairs Aurora A localization at the spindle microtubules but does not affect centrosomal Aurora A. Aurora A is also present at centrosomes but not on spindle microtubules in cells treated with <b>1</b> (MLN8237).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Compound <b>9</b> Selectively Inhibits Aurora A in Cells</h3><div class="NLM_p">We next set out to test the impact of <b>9</b> on mitotic progression of human tissue culture cells. Treatment of HeLa cells with <b>9</b> resulted in a dose-dependent increase in the mitotic index, as observed also with the reference inhibitor <b>1</b> and as expected from the role of Aurora A for timely progression through mitosis (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>a).<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Inhibitor <b>9</b> also decreased phosphorylation of Aurora A Thr288, an autophosphorylation site that marks Aurora A kinase activity (IC<sub>50</sub> = 760 nM, <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>b).<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> Furthermore, <b>9</b> induced defective chromosome alignment during metaphase, with a phenotypic EC<sub>50</sub> value of 6 μM, in line with the requirement of Aurora A activity for proper spindle dynamics (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>c,d).<a onclick="showRef(event, 'ref18 ref24 ref74'); return false;" href="javascript:void(0);" class="ref ref18 ref24 ref74">(18, 24, 74)</a></div><figure id="fig7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0007.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Inhibitor <b>9</b> yields an Aurora A specific inhibition phenotype in cells without impairing Aurora B function. (a) Mitotic index of HeLa Kyoto cells after overnight incubation with indicated compounds. The experiment was repeated twice and error bars represent SD. (b) Centrosomal pThr288 Aurora A levels in HeLa Kyoto cells after overnight incubation with indicated concentrations of <b>9</b>. More than 10 cells were analyzed in every condition. The experiment was repeated twice and the error bars represent SD. (c) Chromosome misalignment phenotype of HeLa Kyoto cells incubated with indicated compounds. The experiment was repeated twice and error bars represent SD. (d) Representative images of HeLa Kyoto cells after overnight incubation with 10 μM <b>9</b> or 250 nM <b>1</b>. The cells were stained for pThr288 Aurora A (red), α-tubulin (green), and DNA (blue). Impaired alignment of chromosomes on the metaphase plate is indicated by white arrows. (e) Chromosome misalignment phenotype of HeLa Kyoto cells. The cells were transfected with the indicated GFP-Aurora A constructs before incubation with 10 μM <b>9</b> overnight. The bars represent mean phenotype ± SD of two independent experiments. At least 100 cells were analyzed in every condition. Note that cells expressing the highest amount of GFP-Aurora A, GFP-Aurora A Arg137Ala, and GFP-Aurora A Trp277Ala were considered for the analysis; cells expressing lower levels of the mutant transgenes exhibited less or no rescue (data not shown). (f) Representative images of HeLa Kyoto cells after overnight incubation with indicated compounds and staining for pHis-H3 (red) and DNA (blue). (g) Western blot of lysates from cells synchronized in prometaphase with 100 nM nocodazole and treated with the indicated inhibitors. β-actin was used as loading control. (h) DNA content analysis of cells treated for 48 h with the indicated compounds. The cells were stained with propidium iodide and analyzed using flow cytometry.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Many Aurora A kinase inhibitors also target Aurora B in the cellular context. To address whether this may be the case also for <b>9</b>, we examined the distribution of phospho-Histone H3 Ser10, a histone modification that is imparted during early mitosis by Aurora B.<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> We found that this feature remained unchanged in cells treated with <b>9</b>, whereas it was absent in cells treated with the Aurora B inhibitor ZM447439 (<b>94</b>) (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>f,g).<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> Furthermore, the massive accumulation of cells with 8N and 16N DNA contents observed by flow cytometry upon treatment with <b>94</b>, which is a hallmark of defective cytokinesis following Aurora B inactivation, also did not occur with <b>9</b> even when provided at 10 μM (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>h).<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> We conclude that in contrast to other Aurora A inhibitors, including the reference inhibitor <b>1</b>,<a onclick="showRef(event, 'ref23 ref77'); return false;" href="javascript:void(0);" class="ref ref23 ref77">(23, 77)</a> our inhibitor <b>9</b> had no effect on Aurora B activity in cells.</div><div class="NLM_p last">To unequivocally test whether the effect of <b>9</b> on human cells was specific to Aurora A inhibition, we performed phenotypic rescue experiments using cells expressing GFP-Aurora A-Arg137Ala or GFP-Aurora A-Trp277Ala mutants, which were predicted by examination of the crystal structure to be insensitive to <b>9</b>. As shown in <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>e, we found that these proteins were insensitive to the addition of <b>9</b>, demonstrating that the phenotype normally induced by this compound was indeed caused by Aurora A inhibition.<a onclick="showRef(event, 'ref65 ref67'); return false;" href="javascript:void(0);" class="ref ref65 ref67">(65, 67)</a></div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Compound <b>9</b> Shows Moderate Reactivity with Glutathione</h3><div class="NLM_p">Despite of its high selectivity toward Aurora A, the cellular activity of <b>9</b> (EC<sub>50</sub> > 1 μM) was much weaker than its biochemical activity (IC<sub>50</sub> = 2 nM). To understand whether this lower activity might be caused by covalent reaction of the electrophilic double bond of <b>9</b>, we measured its reactivity toward the intracellular nucleophile GSH under physiological conditions. Conversion to a GSH adduct was indeed detected, however only to a limited extent (50% conversion after 24 h with 5 mM GSH, pH 7.4, 37 °C, <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>), suggesting that a significant portion of the inhibitor remained unreacted within the cell and was available for inhibition of Aurora A. Nevertheless analogs of <b>9</b> lacking the electrophilic double bond were prepared as described above (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>) and investigated as alternatives. Diastereomeric cyclopropanes <i>rac</i>-<b>89</b>/<i>rac</i>-<b>90</b> were more than 400-fold less active than their precursor <b>79</b>. On the other hand, its reduced double bond analog <i>rac</i>-<b>91</b> was only 7-fold less active, and the reduced double bond analog of <b>9</b>, <i>rac</i>-<b>92</b>, was also still quite potent (IC<sub>50</sub> = 24 nM). However, this derivative did not show any activity in cells despite the fact that it cannot form GSH adducts.</div><figure id="fig8" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0008.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. In vitro reactivity of <b>9</b> toward GSH. Inhibitor <b>9</b> (30 μM) was incubated with GSH (5 mM) in H<sub>2</sub>O/CH<sub>3</sub>CN (4:1) at pH = 7.0, and the reaction was followed by LC-MS. (a) Reaction scheme and structures of modified analogs. (b) LC chromatogram of the reaction mixture after <i>t</i> = 0 and 14 h. The product formed with a retention time <i>t</i><sub>ret</sub> = 1.29 min is identified by MS as the GSH-adduct of <b>9</b>. 9-methylcarbazole (<i>t</i><sub>ret</sub> = 2.44 min) was used as internal control. (c) Concentration of compound <b>9</b> during the course of the reaction as determined by integration of the absorption peaks (at 310 nm) of compound <b>9</b> at <i>t</i><sub>ret</sub> = 1.78 min.The average of two independent experiments is shown, SD is <5% for all data points. (d) Aurora A inhibition in the HTRF biochemical assay. (e) Cellular pThr288 Aurora A levels of cells incubated with <b>9</b> or <i>rac</i>-<b>92</b> overnight at the indicated concentrations. The experiment was repeated twice and the error bars represent SD. Derivative <i>rac</i>-<b>92</b> does not reduce pThr288 levels in cells, although having an IC<sub>50</sub> of 24 nM in the biochemical assay. Numbers represent average IC<sub>50</sub> ± SD of two independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Taken together, these data suggest that GSH reactivity is insufficient to explain the discrepancy between the IC<sub>50</sub> values of cellular versus biochemical activity of <b>9</b> on Aurora A and B presented above (Aurora A: biochemical IC<sub>50</sub> = 2.0 nM, cellular IC<sub>50</sub> (pT288) = 760 nM, Aurora B/INCENP: biochemical IC<sub>50</sub> = 149 nM, cellular IC<sub>50</sub> (pH3) > 25 μM). This activity difference is probably a consequence of the higher ATP concentration in cells (1 mM) compared to in vitro assays (20 μM) and the presence of competing ligands such as TPX2 (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>d) as well as the presence of a carboxylate group which might reduce cellular uptake. The same effects probably explain the weaker cellular versus biochemical activities reported for inhibitor <b>1</b> (Aurora A: biochemical IC<sub>50</sub> = 0.04 nM, cellular IC<sub>50</sub> (pT288) = 6.7 nM, Aurora B/INCENP: biochemical IC<sub>50</sub> = 1.1 nM, cellular IC<sub>50</sub> (pH3) = 1.5 μM) as well as for MK-5108, a further selective Aurora A inhibitor (Aurora A: biochemical IC<sub>50</sub> = 0.064 nM, cellular IC<sub>50</sub> (pT288) = 300 nM, Aurora B/INCENP: biochemical IC<sub>50</sub> = 1.49 nM, cellular IC<sub>50</sub> (pH3) > 10 μM).<a onclick="showRef(event, 'ref22 ref23 ref78'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref78">(22, 23, 78)</a></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32673" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32673" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">To identify new inhibitors of Aurora A kinase, we performed scaffold hopping LBVS using our recently reported method xLOS, a 3D shape and pharmacophore similarity algorithm, and assembled a kinase discovery set with very low compound and scaffold overlap with known kinase inhibitors and screening compounds. We then performed biochemical activity screening against Aurora A and optimization for two screening hits, benzimidazole <b>5</b> and thiazolidinone <b>7</b>. While benzimidazole <b>5</b> did not yield potent inhibitors, we successfully optimized thiazolidinone <b>7</b> by elaboration of the hinge-binding motif using structure-based drug design and discovered the potent inhibitor <b>9</b> with IC<sub>50</sub> = 2 nM. This type II inhibitor locks the kinase in an inactive conformation and competes with the activator protein TPX2. Inhibitor <b>9</b> shows strong binding selectivity for Aurora kinases against the entire kinome and displays a highly specific Aurora A inhibition phenotype without Aurora B specific effects.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30471" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30471" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Virtual Screening</h3><div class="NLM_p last">Virtual screening was performed using xLOS, an in-house developed 3D ligand-based virtual screening method. Two rounds of virtual screening were performed. In the first round, the ZINC database containing ∼12 million commercial compounds<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> was screened using each of the 16 known inhibitors of Aurora A and other kinases as seed molecule. For each seed (one stereoisomer) and database (all possible stereoisomers), a single lowest-energy 3D model was generated using CORINA program available from Molecular Networks Pvt. Ltd. These 3D models were used by xLOS to calculate the xLOS similarity score between the seed and database molecule as described previously.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> For database molecules, only the best scoring stereoisomer was retained in the list. The search was limited to catalogs from Princeton Molecular Research Inc. (∼800,000 cpds) and Otava Chemicals (∼300,000 cpds). For each of the 16 reference molecules, the top 1000 scoring compounds were selected. The resulting 16,000 compounds were further analyzed by clustering and visual analysis to select structurally diverse compounds. In the second round, 15 hits identified in first round of virtual screening were used in the same manner.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Scaffold Analysis</h3><div class="NLM_p last">The Kinase ChEMBL was downloaded from the ChEMBL database Web site (<a href="https://www.ebi.ac.uk/chembl/" class="extLink">https://www.ebi.ac.uk/chembl/</a>). The screening commercial set was created by combining commercially available screening compounds for kinases from three different vendors namely ChemBridge (<a href="http://www.chembridge.com/" class="extLink">http://www.chembridge.com/</a>), Life Chemicals (<a href="http://www.lifechemicals.com/" class="extLink">http://www.lifechemicals.com/</a>), and Otava (<a href="http://www.otavachemicals.com/" class="extLink">http://www.otavachemicals.com/</a>). The Kinase 50 nM and Aurora 50 nM sets were created by selecting the compounds from ChEMBL database with reported IC<sub>50</sub> of ≤50 nM against any kinases and Aurora kinase targets, respectively. Molecules were processed using in-house written java program utilizing Java Chemistry library (JChem) from ChemAxon, Pvt. Ltd. as a starting point. Molecules were checked for valence error, counterions were removed, and ionization states were adjusted to pH 7.4. Molecules were stored in nonstereo smiles format, and duplicate molecules were removed from each database. Within each database, each molecule was converted to Bemis–Murcko Scaffold (BMS) using the StructuralFrameworksPlugin available in the JChem library from ChemAxon Pvt. Ltd. For each database, the list of unique BMS was generated, followed by calculation of the number of compounds represented by each BMS. For the shared scaffold analysis, the unique SMILES of scaffolds were used for comparing molecules.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Chemistry</h3><div class="NLM_p">All reagents were purchased from commercial sources and were used without further purification. Flash chromatography purifications were performed with silica Gel 60 (Fluka, 0.040–0.063 nm, 230–400 mesh ASTM). Low-resolution mass spectra were obtained by electron spray ionization (ESI-MS) in the positive mode on a Thermo Scientific LCQ Fleet. High-resolution mass spectra were obtained by electron spray ionization (HR-ESI-MS) in the positive mode recorded on a Thermo Scientific LTQ Orbitrap XL. <sup>1</sup>H and <sup>13</sup>C NMR spectra were measured on a Bruker Avance 300 spectrometer (at 300 and 75 MHz, respectively) or on a Bruker AVANCE II 400 spectrometer (at 400 and 101 MHz, respectively). <sup>1</sup>H and <sup>13</sup>C chemical shifts are quoted relative to solvent signals, and resonance multiplicities are reported as s (singlet), d (doublet), t (triplet), q (quartet), p (pentet), and m (multiplet); br = broad peak. Compound purities were assessed by analytical reversed-phase HPLC (RP-HPLC) at a detection wavelength of 254 or 310 nm. The purity of tested compounds was >95%, unless otherwise stated (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_001.pdf" class="ext-link">Supplementary Table 2</a>). Analytical RP-HPLC was performed on a Dionex Ultimate 3000 RSLC System (DAD-3000 RS Photodiode Array Detector) using a Dionex Acclaim RSLC 120 column (C18, 3.0 × 50 mm, particle size 2.2 μm, 120 Å pore size) and a flow rate of 1.2 mL min<sup>–1</sup>. Data were recorded and processed with Dionex Chromeleon Management System (version 6.8) and Xcalibur (version 2.2, Thermo Scientific). Eluents for analytical RP-HPLC were as follows: (A) Milli-Q-deionized water containing 0.05% TFA and (D) HPLC-grade acetonitrile/Milli-Q-deionized water (9:1) containing 0.05% TFA. Conditions for analytical RP-HPLC were as follows: From A/D (7:3) to 100% D (2.2 min) followed by 100% D (1 min). Preparative RP-HPLC was performed with a Waters Prep LC4000 Chromatography System using a Reprospher 100 column (Dr. Maisch GmbH, C18-DE, 100 × 30 mm, particle size 5 μM, pore size 100 Å) and a Waters 489 Tunable Absorbance Detector operating at 214 nm. Eluents for preparative RP-HPLC were as follows: (A) Milli-Q-deionized water containing 0.1% TFA and (D) HPLC-grade acetonitrile/Milli-Q-deionized water (9:1) containing 0.1% TFA.</div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 1′<i>H</i>-[1,2′-Bibenzo[<i>d</i>]imidazol]-2(3<i>H</i>)-one (<b>5</b>)</h4><div class="NLM_p last">A solution of compound <b>25</b> (150 mg, 0.463 mmol, 1 equiv) and K<sub>2</sub>CO<sub>3</sub> (194 mg, 1.39 mmol, 3 equiv) in DMF (4 mL) was stirred at 110 °C for 14 h. DMF was evaporated in vacuo, and EtOAc (20 mL) and brine (20 mL) were added. The organic phase was separated and washed with H<sub>2</sub>O (3 × 20 mL). EtOAc was removed in vacuo. The crude was purified by FC (SiO<sub>2</sub>; hexane/EtOAc 1:2) to afford <b>5</b> as gray powder (46 mg, 0.19 mmol, 40%). Mp >300 °C (dec.). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.69 (s, 1H), 8.59–8.18 (m, 1H), 7.87–7.44 (m, 2H), 7.50–7.08 (m, 5H); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 152.42, 143.91, 128.56, 127.02, 123.41, 121.68, 121.61, 113.41, 109.45; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>14</sub>H<sub>11</sub>ON<sub>4</sub> 251.0927, found 251.0923.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 1-(1<i>H</i>-Benzo[<i>d</i>]imidazol-2-yl)-5-hydroxy-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>6</b>)</h4><div class="NLM_p last">Purification of compound <b>40</b> by HPLC (Reprospher C<sub>18</sub>; 10–60% D/A) gave as a second fraction <b>6</b> (19 mg, 0.071 mmol, 70%) as a white solid. Mp 297.4–299.8 °C (dec.). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 11.47 (s, 1H), 9.41 (s, 1H), 8.17 (d, <i>J</i> = 8.6 Hz, 1H), 7.59 (dd, <i>J</i> = 5.9, 3.2 Hz, 2H), 7.17 (dd, <i>J</i> = 6.0, 3.2 Hz, 2H), 6.61 (dd, <i>J</i> = 8.6, 2.3 Hz, 1H), 6.57 (d, <i>J</i> = 2.2 Hz, 1H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): δ 154.17, 152.65, 144.10, 129.51, 121.60, 119.57, 114.26, 108.40, 97.02; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>14</sub>H<sub>11</sub>O<sub>2</sub>N<sub>4</sub> 267.0877, found 267.0872.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> (2<i>Z</i>,5<i>Z</i>)-2-((3,4-Dimethylphenyl)imino)-3-methyl-5-(pyridin-4-ylmethylene)thiazolidin-4-one (<b>7</b>)</h4><div class="NLM_p last">This compound was purchased from Princeton Biomolecular (no. OSSK_393940). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.57 (d, <i>J</i> = 4.0 Hz, 2H), 7.55 (d, <i>J</i> = 1.5 Hz, 1H), 7.23 (d, <i>J</i> = 4.4 Hz, 2H), 7.07 (d, <i>J</i> = 7.8 Hz, 1H), 6.72–6.66 (m, 2H), 3.39 (s, 3H), 2.21 (s, 6H); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 166.18, 150.13, 149.07, 145.49, 141.33, 137.91, 133.64, 130.54, 127.68, 126.85, 123.42, 122.32, 118.04, 29.99, 19.99, 19.37; HR-ESI-MS (<i>m</i>/<i>z</i>): 324.1175, calcd for [M + H]<sup>+</sup> C<sub>18</sub>H<sub>18</sub>ON<sub>3</sub>S: 324.1165.</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> (2<i>Z</i>,5<i>Z</i>)-2-((4-Ethylphenyl)imino)-3-methyl-5-(pyridin-4-ylmethylene)thiazolidin-4-one (<b>8</b>)</h4><div class="NLM_p last">This compound was initially purchased and later resynthesized as follows: A solution of <b>54</b> (702 mg, 3.00 mmol, 1.0 equiv) and piperidine (0.65 mL, 6.6 mmol, 2.2 equiv) in EtOH (5 mL) was treated with 4-pyridinecarboxaldehyde (0.56 mL, 6.0 mmol, 2.0 equiv) and heated to 78 °C. After 12 h, the reaction mixture was cooled to 0 °C and filtered, and the filter residue washed with ice-cold EtOH (3 × 5 mL). The yellow solid was further purified by FC (SiO<sub>2</sub>; hexane/EtOAc 2:1) to afford the <b>8</b> as yellow solid (203 mg, 0.627 mmol, 20.9%). Mp 167.3–168.0 °C. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.58–8.56 (m, 2H), 7.55 (d, <i>J</i> = 1.5 Hz, 1H), 7.24–7.14 (m, 4H), 6.86 (d, <i>J</i> = 8.2 Hz, 2H), 3.39 (s, 3H), 2.61 (q, <i>J</i> = 7.6 Hz, 2H), 1.20 (t, <i>J</i> = 7.6 Hz, 3H); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 166.15, 150.30, 150.25, 149.15, 145.27, 141.29, 128.88, 127.00, 123.35, 120.94, 29.99, 28.39, 15.52; ESI-MS (<i>m</i>/<i>z</i>): 324.12.</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 6-((4-((<i>Z</i>)-((<i>Z</i>)-2-((4-Ethylphenyl)imino)-3-methyl-4-oxothiazolidin-5-ylidene)methyl)pyridin-2-yl)amino)nicotinic acid (<b>9</b>)</h4><div class="NLM_p last">Degassed dioxane (1.2 mL) was added to a flask containing amine <b>79</b> (100 mg, 0.296 mmol, 1.0 equiv), Pd<sub>2</sub>(dba)<sub>3</sub> (5.1 mg, 0.0088 mmol, 0.03 equiv), Xantphos (10 mg, 0.018 mmol, 0.06 equiv), Cs<sub>2</sub>CO<sub>3</sub> (116 mg, 0.355 mmol, 1.2 equiv), and 6-bromonicotinic acid (65.9 mg, 0.326 mmol, 1.1 equiv) under argon. The suspension was degassed, refilled with argon (5×), and then heated to 100 °C. The reaction mixture was stirred overnight at 100 °C, then cooled to room temperature, filtered, and concentrated in vacuo. The crude was purified by RP-HPLC (A/D 70:30 to A/D 45:55, 40 min gradient) to afford <b>9</b> as yellow solid after lyophylization (38 mg, 0.083 mmol, 28%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.71 (s, 1H), 8.70 (d, <i>J</i> = 2.3 Hz, 1H), 8.37–8.36 (m, 1H), 8.16–8.15 (m, 2H), 7.67 (s, 1H), 7.47 (d, <i>J</i> = 8.8 Hz, 1H), 7.32 (d, <i>J</i> = 8.3 Hz, 2H), 7.19 (dd, <i>J</i> = 5.4, 1.7 Hz, 1H), 7.00 (d, <i>J</i> = 8.2 Hz, 2H), 3.36 (s, 3H), 2.61 (q, <i>J</i> = 7.6 Hz, 2H), 1.17 (t, <i>J</i> = 7.6 Hz, 3H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.07, 165.47, 156.34, 153.46, 149.42, 148.97, 147.38, 145.14, 142.61, 140.51, 138.74, 128.78, 127.07, 126.83, 120.82, 118.02, 111.55, 110.76, 29.73, 27.53, 15.48; HR-ESI-MS (<i>m</i>/<i>z</i>): 460.1435, calcd for [M + H]<sup>+</sup> C<sub>24</sub>H<sub>22</sub>O<sub>3</sub>N<sub>5</sub>S: 460.1438.</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 1,3-Dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>13</b>)</h4><div class="NLM_p last">A solution of 1,2-diaminobenzene (10.0 g, 92.5 mmol) in DMF (93 mL) was treated with <i>N</i>,<i>N</i>′-carbonyldiimidazole (30.2 g, 186 mmol) and stirred at room temperature for 18 h. After removal of the DMF in vacuo, the resulting white solid was washed with H<sub>2</sub>O (4 × 50 mL) and dried on high vacuum to give <b>13</b> (12.4 g, 99%) as a white solid. Mp >300 °C (dec.). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 10.57 (s, 1H), 6.91 (s, 2H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): δ 155.29, 129.68, 120.41, 108.49; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>7</sub>H<sub>7</sub>ON <sub>2</sub> 135.0553, found 135.0550.</div></div><div id="sec4_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 4-Methyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>14</b>)</h4><div class="NLM_p last">A solution of 2,3-diaminotoluene (3.50 g, 28.7 mmol) in DMF (28.7 mL) was treated with <i>N</i>,<i>N</i>′-carbonyldiimidazole (4.65 g, 28.7 mmol), and the solution was stirred at room temperature for 15 h. After addition of H<sub>2</sub>O (100 mL), the reaction mixture was extracted with EtOAc (3 × 200 mL), and the combined organic phases were dried over MgSO<sub>4</sub> and concentrated to give <b>14</b> (4.24 g, 28.6 mmol, quant.) as a brown solid. Mp 218.6–220.4 °C.  <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 10.73 (s, 1H), 10.60 (s, 1H), 6.85–6.78 (m, 1H), 6.77–6.70 (m, 2H), 2.25 (s, 3H); <sup>13</sup>  C NMR (DMSO- <i>d</i><sub>6</sub>, 75 MHz): δ 155.68, 129.35, 128.67, 121.70, 120.48, 118.29, 106.21, 16.25; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>8</sub>H<sub>8</sub>ON <sub>2</sub>Na 171.0529, found 171.0527.</div></div><div id="sec4_3_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 5-Methyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>15</b>)</h4><div class="NLM_p last">Following the <a class="ref internalNav" href="#sec4_3_7" aria-label="procedure">procedure</a> for <b>14</b>, conversion of 3,4-diaminotoluene (1.70 g, 13.9 mmol) gave <b>15</b> as light-gray solid (2.04 g, 13.8 mmol, 99%) after filtration and washing with H<sub>2</sub>O (4 × 30 mL). Mp 277.2–279.2 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 10.46 (d, <i>J</i> = 10.9 Hz, 2H), 6.77 (s, 1H), 6.73 (s, 2H), 2.27 (s, 3H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): δ 155.41, 129.84, 129.33, 127.43, 120.86, 108.99, 108.14, 21.01; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>8</sub>H<sub>9</sub>N <sub>2</sub>O 149.0709, found 149.0704.</div></div><div id="sec4_3_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 5-Bromo-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>16</b>)</h4><div class="NLM_p last">Following the <a class="ref internalNav" href="#sec4_3_7" aria-label="procedure">procedure</a> for <b>14</b>, conversion of 3,4-diaminobromobenzene (2.50 g, 13.4 mmol) gave <b>16</b> as brown solid (2.83 g, 13.3 mmol, quant.) after filtration and washing with H<sub>2</sub>O (5 × 70 mL). Mp >300 °C (dec.). <sup>1</sup>H NMR (DMSO- <i>d</i><sub>6</sub>, 300 MHz): δ 10.76 (s, 2H),  7.08–7.03 (m,  <i>J</i> = 10.2 Hz, 2H), 6.86 (d, <i>J</i> = 8.0 Hz, 1H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): δ 155.07, 131.23, 129.01, 122.92, 112.02, 111.05, 110.09; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>7</sub>H<sub>6</sub>ON <sub>2</sub>Br 212.9658, found 212.9662.</div></div><div id="sec4_3_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 5-(Trifluoromethyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>17</b>)</h4><div class="NLM_p last">Following the <a class="ref internalNav" href="#sec4_3_7" aria-label="procedure">procedure</a> for <b>14</b>, conversion of 3,4-diaminotrifluoromethylbenzene (2.50 g, 14.2 mmol) gave <b>17</b> as brown solid (2.06 g, 10.2 mmol, 73%) after filtration and washing with H<sub>2</sub>O (5 × 20 mL). Mp 280.1–282.0 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 11.06 (s, 1H), 10.96 (s, 1H), 7.29 (m, 1H), 7.16 (s, 1H), 7.09 (d, <i>J</i> = 8.2 Hz, 1H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): δ 155.35, 132.87, 129.86, 121.27, 120.85, 117.87, 108.56, 104.93; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>8</sub>H<sub>6</sub>ON <sub>2</sub>F<sub>3</sub> 203.0427, found 203.0423.</div></div><div id="sec4_3_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 5,6-Dichloro-1<i>H</i>-benzo[<i>d</i>[imidazol-2(3<i>H</i>)-one (<b>18</b>)</h4><div class="NLM_p last">Following the <a class="ref internalNav" href="#sec4_3_7" aria-label="procedure">procedure</a> for <b>14</b>, conversion of 4,5-diamino-1,2-dichlorobenzene (2.50 g, 14.2 mmol) gave <b>18</b> as brown solid (2.67 g, 13.2 mmol, 95%) after filtration and washing with H<sub>2</sub>O (5 × 20 mL). Mp >300 °C (dec.). <sup>1</sup>H NMR (DMSO- <i>d</i><sub>6</sub>, 300 MHz): δ 10.91 (s, 2H), 7.10 (s, 2H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): δ 155.18, 129.85, 122.36, 109.75; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>7</sub>H<sub>5</sub>ON <sub>2</sub>Cl<sub>2</sub> 202.9773, found 202.9770.</div></div><div id="sec4_3_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 2-chloro-1<i>H</i>-benzo[<i>d</i>]imidazole (<b>19</b>)</h4><div class="NLM_p last">A solution of <b>13</b> (4.31 g, 32.1 mmol) in POCl<sub>3</sub> (30 mL) was refluxed at 100 °C for 15 h. The reaction mixture was slowly poured into icecold H<sub>2</sub>O (150 mL), and the pH was adjusted to pH = 7 using NaOH (6 M). The yellowish solid was filtered and washed with H<sub>2</sub>O (3 × 50 mL) to give <b>19</b> (4.42 g, 91%) as a yellowish solid. Mp 192.1–193.5 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 13.22 (s, 1H), 7.51 (m, 2H), 7.21 (m, 2H); <sup>13</sup>C NMR (DMSO- <i>d</i><sub>6</sub>, 75 MHz): δ 150.90, 129.08, 126.59, 124.40, 121.68, 112.15, 109.69; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>7</sub>H<sub>6</sub>N <sub>2</sub>Cl 153.0214, found 153.0214.</div></div><div id="sec4_3_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 2-Chloro-4-methyl-1<i>H</i>-benzo[<i>d</i>]imidazole (<b>20</b>)</h4><div class="NLM_p last">A solution of <b>14</b> (3.50 g, 23.6 mmol) in POCl<sub>3</sub> (22.0 mL) was refluxed at 100 °C for 16 h. H<sub>2</sub>O was slowly added to the reaction mixture and then poured into ice-cold H<sub>2</sub>O (100 mL), and the pH was adjusted to pH = 7 using NaOH (5M) to precipitate a white solid. The white suspension was extracted with EtOAc (3 × 100 mL), dried over MgSO<sub>4</sub>, and concentrated to give a white crude product. Purification by flash chromatography (SiO<sub>2</sub>; 10–40% EtOAc/hexane) gave <b>20</b> (1.00 g, 6.02 mmol, 26%) as a white solid. Mp 138.2–140.9 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 13.18 (s, 1H), 7.31 (dd, <i>J</i> = 38.8, 7.1 Hz, 1H), 7.10 (t, <i>J</i> = 6.8 Hz, 1H), 7.00 (d, <i>J</i> = 7.3 Hz, 1H), 2.47 (s, 3H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): mixture of two isomers δ 142.08, 141.71, 138.33, 137.68, 134.39, 134.20, 127.89, 123.33, 122.78, 122.17, 121.97, 120.98, 115.68, 108.42, 16.65, 16.33; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>8</sub>H<sub>8</sub>N <sub>2</sub>Cl 167.0371, found 167.0367.</div></div><div id="sec4_3_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 2-Chloro-5-methyl-1<i>H</i>-benzo[<i>d</i>]imidazole (<b>21</b>)</h4><div class="NLM_p last">Following the <a class="ref internalNav" href="#sec4_3_13" aria-label="procedure">procedure</a> for <b>20</b>, conversion of <b>15</b> (1.00 g, 6.75 mmol) gave <b>21</b> as brown solid (530 mg, 3.19 mmol, 48%) after extraction with EtOAc (3 × 50 mL). Mp 161.7–162.8 °C; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 13.04 (s, 1H), 7.53–7.13 (m, 2H), 7.03 (m, 1H), 2.39 (s, 3H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): δ 155.44, 137.98, 135.53, 130.07, 123.66, 120.85, 117.29, 21.15; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>8</sub>H<sub>8</sub>N <sub>2</sub>Cl 167.0371, found 167.0367.</div></div><div id="sec4_3_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 5-Bromo-2-chloro-1<i>H</i>benzo[<i>d</i>]imidazole (<b>22</b>)</h4><div class="NLM_p last">Following the <a class="ref internalNav" href="#sec4_3_13" aria-label="procedure">procedure</a> for <b>20</b>, conversion of <b>16</b> (2.50 g, 11.8 mmol) gave <b>22</b> as brown solid (2.40 g, 10.4 mmol, 89%) after extraction with EtOAc (3 × 100 mL). Mp 198.0–199.2 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 13.46 (s, 1H), 7.75 (d, <i>J</i> = 1.5 Hz, 1H), 7.50 (d, <i>J</i> = 8.5 Hz, 1H), 7.39 (dd, <i>J</i> = 8.6, 1.9 Hz, 1H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): δ 152.14, 139.90, 135.70, 130.29, 125.35, 117.33, 114.69; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>7</sub>H<sub>5</sub>N <sub>2</sub>BrCl 230.9319, found 230.9315.</div></div><div id="sec4_3_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 2-Chloro-5-(trifluoromethyl)-1<i>H</i>-benzo[<i>d</i>]imidazole (<b>23</b>)</h4><div class="NLM_p last">Following the <a class="ref internalNav" href="#sec4_3_13" aria-label="procedure">procedure</a> for <b>20</b>, conversion of <b>17</b> (1.50 g, 7.42 mmol) gave <b>23</b> as light-brown solid (1.11 g, 5.05 mmol, 72%) after filtration and washing with H<sub>2</sub>O (5 × 50 mL). Mp 184.7–185.9 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 13.76 (s, 1H), 7.89 (s, 1H), 7.71 (d, <i>J</i> = 8.4 Hz, 1H), 7.55 (dd, <i>J</i> = 8.5, 1.3 Hz, 1H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): δ 141.51, 130.16, 126.56, 123.34, 122.96, 119.35, 119.29, 119.25; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>8</sub>H<sub>5</sub>N <sub>2</sub>ClF<sub>3</sub> 221.0088, found 221.0083.</div></div><div id="sec4_3_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 2,5,6-Trichloro-1<i>H</i>-benzo[<i>d</i>]imidazole (<b>24</b>)</h4><div class="NLM_p last">Following the <a class="ref internalNav" href="#sec4_3_13" aria-label="procedure">procedure</a> for <b>20</b>, conversion of <b>18</b> (2.50 g, 12.4 mmol) gave <b>24</b> as brown solid (1.99 g, 9.00 mmol, 73%) after extraction with EtOAc (3 × 100 mL). Mp 218.1–219.9 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 13.65 (s, 1H), 7.82 (s, 2H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): δ 141.92, 138.36, 124.76, 116.11; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>7</sub>H<sub>4</sub>N <sub>2</sub>Cl<sub>3</sub> 220.9435, found 220.9437.</div></div><div id="sec4_3_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> <i>tert</i>-Butyl(2-((1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)amino)phenyl)carbamate (<b>25</b>)</h4><div class="NLM_p last">A solution of <b>19</b> (350 mg, 2.29 mmol, 1.0 equiv), <i>tert</i>-butyl (2-aminophenyl)carbamate (477 mg, 2.29 mmol, 1.0 equiv), and KH<sub>2</sub>PO<sub>4</sub> (312 mg, 2.29 mmol, 1.0 equiv) in <i>n</i>-BuOH (5 mL) was heated to 90 °C and stirred for 12 h. The reaction mixture was cooled to room temperature, EtOAc (20 mL) and H<sub>2</sub>O (20 mL) were added, and the organic layer was extracted with H<sub>2</sub>O (3 × 20 mL). Evaporation of EtOAc in vacuo afforded <b>25</b> as pale brown solid (145 mg, 0.458 mmol, 20%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.81 (s, 2H), 10.82 (s, 1H), 9.13 (s, 1H), 7.82 (dd, <i>J</i> = 8.0, 1.5 Hz, 1H), 7.53 (dd, <i>J</i> = 7.9, 1.7 Hz, 1H), 7.42 (dt, <i>J</i> = 6.6, 3.1 Hz, 3H), 7.33–7.18 (m, 3H), 1.45 (s, 9H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 153.02, 149.32, 134.79, 129.87, 128.07, 127.33, 126.82, 124.63, 123.33, 123.12, 111.59, 79.44, 28.03; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>18</sub>H<sub>21</sub>O<sub>2</sub>N<sub>4</sub> 325.1659, found 325.1654.</div></div><div id="sec4_3_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> <i>tert</i>-Butyl 2-(4-Methyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2-ylamino)phenylcarbamate (<b>26</b>)</h4><div class="NLM_p last">A solution of <b>20</b> (0.810 g, 4.88 mmol) and <i>tert</i>-butyl (2-aminophenyl)carbamate (1.02 g, 4.88 mmol) in <i>n</i>-BuOH (9.8 mL) was treated with KH<sub>2</sub>PO<sub>4</sub> (0.664 g, 4.88 mmol) and stirred at 115 °C for 17 h. After addition of H<sub>2</sub>O (40 mL), the reaction mixture was extracted with EtOAc (3 × 15 mL), and the combined organic phases were dried over MgSO<sub>4</sub> and concentrated to give a brown crude product. Purification by flash chromatography (SiO<sub>2</sub>; 10–100% EtOAc/hexane) gave <b>26</b> (425 mg, 1.26 mmol, 26%) as a brownish solid. Mp 106.3–109.0 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 11.16 (s, 1H), 8.93 (s, 1H), 8.09 (s, 1H), 7.39 (s, 1H), 7.15 (dd,  <i>J</i> = 16.1, 7.7 Hz, 2H), 6.99 (t, <i>J</i> = 7.4 Hz, 1H), 6.93–6.75 (m, 3H), 2.41 (s, 3H), 1.45 (s, 9H); <sup>13</sup>C NMR (DMSO- <i>d</i><sub>6</sub>, 75 MHz): δ 172.02, 153.75, 150.79, 129.66, 125.18, 121.93, 120.98, 120.37, 120.06, 108.45, 79.11, 28.10, 16.59; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>19</sub>H<sub>23</sub>O<sub>2</sub>N<sub>4</sub> 339.1816, found 339.1814.</div></div><div id="sec4_3_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> <i>tert</i>-Butyl 2-(5-Methyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2-ylamino)phenylcarbamate (<b>27</b>)</h4><div class="NLM_p last">Following the <a class="ref internalNav" href="#sec4_3_19" aria-label="procedure">procedure</a> for <b>26</b>, conversion of <b>21</b> (300 mg, 1.81 mmol) gave <b>27</b> as brown solid (170 mg, 0.51 mmol, 27%) after purification by flash chromatography (SiO<sub>2</sub>; 1–15% MeOH/DCM). Mp 166.4–166.8 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 10.44 (d, <i>J</i> = 10.9 Hz, 1H), 8.74 (s, 1H),8.07 (d, <i>J</i> = 7.8 Hz, 1H), 7.40 (d, <i>J</i> = 7.6 Hz, 1H),  7.22–7.11 (m, 2H), 7.10–6.97 (m, 2H), 6.78 (m, 2H), 2.35 (s, 3H), 1.45 (s, 9H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): δ 155.36, 153.65, 129.81, 129.28, 129.04, 128.20, 127.40, 125.08, 122.11, 121.24, 120.81, 120.53, 108.94, 108.09, 79.15, 28.07, 21.21; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>19</sub>H<sub>23</sub>O<sub>2</sub>N<sub>4</sub> 339.1816, found 339.1809.</div></div><div id="sec4_3_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> <i>tert</i>-Butyl 2-(5-Bromo-1<i>H</i>-benzo[<i>d</i>]imidazol-2-ylamino)phenylcarbamate (<b>28</b>)</h4><div class="NLM_p last">Following the <a class="ref internalNav" href="#sec4_3_19" aria-label="procedure">procedure</a> for <b>26</b>, conversion of <b>22</b> (700 mg, 3.04 mmol) gave <b>28</b> as brown solid (480 mg, 1.19 mmol, 40%) after purification by flash chromatography (SiO<sub>2</sub>; 0–10% MeOH/DCM). Mp 176.2–178.2 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 11.26 (s, 1H), 8.62 (d, <i>J</i> = 27.6 Hz, 2H), 8.08 (s, 1H), 7.42 (dd, <i>J</i> = 18.1, 5.0 Hz, 2H), 7.28–7.07 (m, 3H), 7.02 (t, <i>J</i> = 7.1 Hz, 1H), 1.45 (s, 9H); <sup>13</sup> C NMR (DMSO- <i>d</i><sub>6</sub>, 75 MHz): δ 153.66, 152.14, 133.26, 128.46, 125.14, 122.55, 120.90, 79.22, 28.06; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>18</sub>H<sub>20</sub>O<sub>2</sub>N<sub>4</sub>Br 403.0764, found 403.0757.</div></div><div id="sec4_3_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> <i>tert</i>-Butyl 2-(5-(Trifluoromethyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-ylamino)phenylcarbamate (<b>29</b>)</h4><div class="NLM_p last">Following the <a class="ref internalNav" href="#sec4_3_19" aria-label="procedure">procedure</a> for <b>26</b>, conversion of <b>23</b> (500 mg, 2.27 mmol) was gave <b>29</b> as brown solid (750 mg, 1.91 mmol, 84%) after purification by flash chromatography (SiO<sub>2</sub>; 1:1 hexane/EtOAc). Mp 176.5–177.6 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 11.40 (s, 1H), 8.68 (s, 1H), 8.09 (d, <i>J</i> = 7.9 Hz, 1H), 7.60 (s, 1H), 7.43 (t, <i>J</i> = 8.0 Hz, 2H), 7.31 (m, 1H), 7.19 (td, <i>J</i> = 8.1, 1.5 Hz, 1H), 7.05 (td, <i>J</i> = 7.7, 1.4 Hz, 1H), 1.45 (s, 9H); <sup>13</sup>C NMR (DMSO- <i>d</i><sub>6</sub>, 75 MHz): δ 153.64, 133.04, 128.69, 127.15, 125.13, 123.56, 122.80, 121.17, 79.24, 28.05; HR-ESI-MS ( <i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>19</sub>H<sub>20</sub>O<sub>2</sub>N<sub>4</sub>F<sub>3</sub> 393.1533, found 393.1532.</div></div><div id="sec4_3_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> <i>tert</i>-Butyl 2-(5,6-Dichloro-1<i>H</i>-benzo[<i>d</i>]imidazol-2-ylamino)phenylcarbamate (<b>30</b>)</h4><div class="NLM_p last">Following the <a class="ref internalNav" href="#sec4_3_19" aria-label="procedure">procedure</a> for <b>26</b>, conversion of <b>24</b> (700 mg, 3.18 mmol) gave <b>30</b> as brown solid (700 mg, 1.78 mmol, 56%) after washing the obtained solid after extraction with hexane/EtOAc (4:1, 20 mL). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 11.30 (s, 1H), 8.67 (d, <i>J</i> = 17.8 Hz, 2H), 8.08 (d, <i>J</i> = 7.9 Hz, 1H), 7.50 (s, 2H), 7.39 (d, <i>J</i> = 9.4 Hz, 1H), 7.18 (td, <i>J</i> = 7.9, 1.5 Hz, 1H), 7.04 (td, <i>J</i> = 7.7, 1.4 Hz, 1H), 1.45 (s, 9H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): δ 153.70, 153.32, 133.02, 128.72, 125.78, 125.15, 124.19, 123.64, 122.83, 121.20, 110.92, 79.27, 28.09; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>18</sub>H<sub>19</sub>O<sub>2</sub>N<sub>4</sub>Cl<sub>2</sub> 393.0880, found 393.0882.</div></div><div id="sec4_3_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 2-(1<i>H</i>-Benzo[<i>d</i>]imidazol-2-ylamino)-4-chloroaniline (<b>31</b>)</h4><div class="NLM_p last">A solution of <i>tert</i>-butyl 2-amino-4-chlorophenylcarbamate (300 mg, 1.24 mmol) and <b>19</b> (190 mg, 1.24 mmol) in <i>n</i>-BuOH (2.5 mL) was treated with KH<sub>2</sub>PO<sub>4</sub> (170 mg, 1.24 mmol) and stirred at 90 °C for 17 h. After addition of H<sub>2</sub>O (50 mL), the reaction mixture was extracted with EtOAc (3 × 15 mL), and the combined organic phases were dried over MgSO<sub>4</sub> and concentrated to give a dark-violet solid. Purification by flash chromatography (SiO<sub>2</sub>; 10–100% EtOAc/hexane) gave <b>31</b> (300 mg, 1.16 mmol, 92%) as a violet solid. Mp 122.8–124.3 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 10.67 (s, 1H), 9.12 (s, 2H), 7.43–7.36 (m, 3H), 7.26–7.17 (m, 4H), 6.89–6.84 (m, 1H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): δ 149.15, 144.17, 129.89, 128.71, 127.43, 123.14, 120.37, 118.81, 117.14, 111.54; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>13</sub>H<sub>12</sub>N<sub>4</sub>Cl 259.0745, found 259.0752.</div></div><div id="sec4_3_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> <i>tert</i>-Butyl 2-(1<i>H</i>-Benzo[<i>d</i>]imidazol-2-ylamino)-5-chlorophenylcarbamate (<b>32</b>)</h4><div class="NLM_p last">A solution of <i>tert</i>-butyl 2-amino-5-chlorophenylcarbamate (260 mg, 1.07 mmol) and <b>19</b> (0.16 g, 1.07 mmol) in <i>n</i>-BuOH (2.1 mL) was treated with KH<sub>2</sub>PO<sub>4</sub> (150 mg, 1.07 mmol) and stirred at 90 °C for 24 h. After addition of H<sub>2</sub>O (40 mL), the reaction mixture was extracted with EtOAc (3 × 20 mL), and the combined organic phases were dried over MgSO<sub>4</sub> and concentrated to give a brownish solid. Purification by flash chromatography (SiO<sub>2</sub>; 10–100% EtOAc/hexane) gave <b>32</b> (160 mg, 41%) as a violet solid. Mp 154.5–155.8 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 12.39 (s, 1H), 9.24 (s, 1H), 7.92 (d, <i>J</i> = 2.2 Hz, 1H), 7.64 (d, <i>J</i> = 8.5 Hz, 1H), 7.37 (m, 2H), 7.25 (m, 3H), 1.45 (s, 9H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): δ 152.85, 149.85, 136.09, 131.75, 130.68, 128.87, 125.70, 124.05, 122.96, 122.03, 111.63, 80.10, 27.98; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>18</sub>H<sub>20</sub>O<sub>2</sub>N<sub>4</sub>Cl 359.1269, found 359.1275.</div></div><div id="sec4_3_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 1-(4-Methyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>33</b>)</h4><div class="NLM_p last">A solution of <b>26</b> (170 mg, 0.503 mmol) in DMF (2.5 mL) was treated with K<sub>2</sub>CO<sub>3</sub> (207 mg, 1.51 mmol) and stirred at 110 °C for 22 h. After addition of H<sub>2</sub>O (20 mL), the reaction mixture was extracted with EtOAc (3 × 15 mL), and the combined organic phases were dried over MgSO<sub>4</sub> and concentrated to give a brown crude product. Purification by HPLC (Reprospher C<sub>18</sub>; 10–70% D/A) gave <b>33</b> (86.0 mg, 0.33 mmol, 65%) as a white solid. Mp 278.3–280.7 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 11.66 (s, 1H), 8.37 (dd, <i>J</i> = 5.4, 3.4 Hz, 1H), 7.44 (d, <i>J</i> = 7.8 Hz, 1H), 7.25–7.12 (m, 3H), 7.09 (t, <i>J</i> = 7.6 Hz, 1H), 7.01 (d, <i>J</i> = 7.3 Hz, 1H), 2.58 (s, 3H);  <sup>13</sup> C NMR (DMSO- <i>d</i><sub>6</sub>, 75 MHz): δ 152.43, 143.13, 128.58, 127.23, 125.85, 123.32, 122.12, 121.74, 121.56, 113.19, 110.63, 109.41, 16.46; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>15</sub>H<sub>13</sub>ON<sub>4</sub> 265.1084, found 265.1078.</div></div><div id="sec4_3_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 1-(5-Methyl-1<i>H</i>-Benzo[<i>d</i>]imidazol-2-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>34</b>)</h4><div class="NLM_p last">A solution of <b>27</b> (150 mg, 0.442 mmol) in DMF (2.2 mL) was treated with K<sub>2</sub>CO<sub>3</sub> (180 mg, 1.32 mmol) and stirred at 110 °C for 47 h. After addition of H<sub>2</sub>O (30 mL), the reaction mixture was extracted with EtOAc (3 × 20 mL), and the combined organic phases were dried over MgSO<sub>4</sub> and concentrated to give a brown solid. Purification by flash chromatography (SiO<sub>2</sub>; 1–10% MeOH/DCM) gave <b>34</b> (20 mg, 0.076 mmol, 14%) as a gray solid. Mp >300 °C (dec.). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 12.29 (s, 1H), 11.64 (s, 1H), 8.44–8.32 (m, 1H), 7.49 (m, 1H), 7.41 (d, <i>J</i> = 5.1 Hz, 1H),  7.24–7.11 (m, 3H), 7.01 (dd, <i>J</i> = 8.2, 1.2 Hz, 1H), 2.42 (s, 3H);  <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): mixture of two isomers δ 152.39, 143.83, 143.48, 140.81, 138.53, 132.57, 130.87, 130.42, 130.32, 128.53, 127.06, 123.29, 122.91, 122.92, 121.54, 117.65, 117.33, 113.35, 111.67, 111.38, 109.36, 21.37, 21.25; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>15</sub>H<sub>13</sub>ON<sub>4</sub> 265.1084, found 265.1083.</div></div><div id="sec4_3_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 1-(5-Bromo-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>35</b>)</h4><div class="NLM_p last">A solution of <b>28</b> (480 mg, 1.20 mmol) in DMF (6 mL) was treated with K<sub>2</sub>CO<sub>3</sub> (500 mg, 3.60 mmol) and stirred at 110 °C for 15 h. After addition of H<sub>2</sub>O (50 mL), the reaction mixture was extracted with EtOAc (3 × 30 mL), and the combined organic phases were dried over MgSO<sub>4</sub> and concentrated to give a light-brown solid. Purification by flash chromatography (SiO<sub>2</sub>; 1–10% MeOH/DCM) gave <b>35</b> (10 mg, 0.030 mmol, 2%) as a light brown solid. Mp >300 °C (dec.). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 12.57 (d, <i>J</i> = 15.7 Hz, 1H), 11.72 (s, 1H), 8.46–8.34 (m, 1H), 7.86–7.74 (m, 1H), 7.58 (dd, <i>J</i> = 8.5, 3.6 Hz, 1H), 7.33 (m, 1H),  7.26–7.12 (m, 3H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): two sets of signals: first set with high intensity δ 152.35, 128.61, 126.81, 124.43, 123.56, 121.64, 113.64, 113.55, 109.49: second set with low intensity δ 155.20, 155.00, 145.01, 144.74, 142.07, 139.68, 133.67, 131.58, 131.19, 129.63, 128.97, 122.85, 120.32, 120.10, 119.42, 114.51, 110.98, 110.02, 108.40; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>14</sub>H<sub>10</sub>ON<sub>4</sub>Br 329.0033, found 329.0035.</div></div><div id="sec4_3_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 1-(5-(Trifluoromethyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>36</b>)</h4><div class="NLM_p last">A solution of <b>29</b> (400 mg, 1.02 mmol) in DMF (6.4 mL) was treated with K<sub>2</sub>CO<sub>3</sub> (420 mg, 3.06 mmol) and stirred at 110 °C for 46 h. After addition of H<sub>2</sub>O (50 mL), the reaction mixture was extracted with EtOAc (3 × 20 mL), and the combined organic phases were dried over MgSO<sub>4</sub> and concentrated to give a white solid. Purification by flash chromatography (SiO<sub>2</sub>; 0–3% MeOH/DCM) gave <b>36</b> (40.0 mg, 0.13 mmol, 12%) as a white solid. Mp 269.7–270.5 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 12.79 (d, <i>J</i> = 9.8 Hz, 1H), 11.75 (d, <i>J</i> = 6.9 Hz, 1H), 8.44 (td, <i>J</i> = 6.7, 2.8 Hz, 1H), 7.97 (s, 1H), 7.80 (d, <i>J</i> = 8.3 Hz, 1H), 7.57–7.47 (m, 1H), 7.20 (dd, <i>J</i> = 21.7, 9.1 Hz, 3H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): δ 152.35, 146.40, 145.92, 143.22, 140.16, 135.04, 132.03, 128.69, 126.80, 123.69, 121.69, 118.23, 113.61, 112.67, 109.56; <sup>19</sup>F NMR (DMSO- <i>d</i><sub>6</sub>, 376 MHz): δ  −58.81 (t,  <i>J</i> = 27.2 Hz); HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>15</sub>H<sub>10</sub>ON<sub>4</sub>F<sub>3</sub> 319.0812, found 319.0822.</div></div><div id="sec4_3_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 1-(5,6-Dichloro-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>37</b>)</h4><div class="NLM_p last">A solution of <b>30</b> (500 mg, 1.28 mmol) in DMF (6.4 mL) was treated with K<sub>2</sub>CO<sub>3</sub> (530 mg, 3.84 mmol) and stirred at 110 °C for 19 h. After addition of H<sub>2</sub>O (50 mL), the reaction mixture was extracted with EtOAc (3 × 20 mL), and the combined organic phases were dried over MgSO<sub>4</sub> and concentrated to give a brown solid. Purification by flash chromatography (SiO<sub>2</sub>; 0–10% MeOH/DCM) gave <b>37</b> (21 mg, 0.066 mmol, 5%) as a white solid. Mp >300 °C (dec.). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 12.65 (s, 1H), 11.75 (s, 1H), 8.43–8.33 (m, 1H), 7.84 (d, <i>J</i> = 24.9 Hz, 2H), 7.19 (m, 3H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): δ 152.31, 146.01, 128.65, 126.66, 123.87, 123.72, 121.69, 118.82, 113.63, 113.14, 109.56; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>14</sub>H<sub>9</sub>ON<sub>4</sub>Cl<sub>2</sub> 319.0148, found 319.0150.</div></div><div id="sec4_3_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 1-(1<i>H</i>-Benzo[<i>d</i>]imidazol-2-yl)-7-hydroxy-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>38</b>)</h4><div class="NLM_p last">A solution of 3-<i>tert</i>-butyldimethylsiloxy-1,2-diaminobenzene (500 mg, 2.10 mmol) and <b>19</b> (319 mg, 2.10 mmol) in <i>n</i>-BuOH (4.2 mL) was treated with KH<sub>2</sub>PO<sub>4</sub> (286 mg, 2.10 mmol) and stirred at 120 °C for 16 h. After addition of H<sub>2</sub>O (40 mL), the reaction mixture was extracted with EtOAc (3 × 40 mL), and the combined organic phases were dried over MgSO<sub>4</sub> and concentrated to give a white solid. A solution of this intermediate (65 mg, 0.27 mmol) in DMF (0.3 mL) was then treated with <i>N</i>,<i>N</i>′-carbonyldiimidazole(44 mg, 0.27 mmol) and stirred at room temperature for 17 h. After addition of H<sub>2</sub>O (10 mL), the reaction mixture was extracted with EtOAc (3 × 10 mL), and the combined organic phases were dried over MgSO<sub>4</sub> and concentrated to give a white solid. Purification by HPLC (Reprospher C<sub>18</sub>; 30–40% D/A) afforded a good separation from <b>42</b> to give <b>38</b> (11 mg, 0.041 mmol, 15%) as a white solid. Mp 220.6–222.2 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 11.65 (s, 1H), 9.67 (s, 1H), 8.44 (dd, <i>J</i> = 6.1, 3.0 Hz, 1H), 7.21 (dd, <i>J</i> = 5.4, 3.7 Hz, 2H), 7.17–7.12 (m, 1H), 7.11–7.05 (m, 1H), 6.99 (t, <i>J</i> = 7.9 Hz, 1H), 6.59 (dd, <i>J</i> = 7.7, 0.9 Hz, 1H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): δ 157.98, 152.52, 142.61, 128.53, 127.09, 123.38, 122.58, 121.58, 113.53, 109.42, 107.35; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>14</sub>H<sub>11</sub>O<sub>2</sub>N<sub>4</sub> 267.0877, found 267.0871.</div></div><div id="sec4_3_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 1-(1<i>H</i>-Benzo[<i>d</i>]imidazol-2-yl)-6-chloro-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>39</b>)</h4><div class="NLM_p last">A solution of <b>31</b> (150 mg, 0.582 mmol) in DMF (0.6 mL) was treated with <i>N</i>,<i>N</i>′-carbonyldiimidazole (94.1 mg, 0.582 mmol) and stirred at room temperature for 40 h. After addition of H<sub>2</sub>O (30 mL), the reaction mixture was extracted with EtOAc (3 × 15 mL), and the combined organic phases were dried over MgSO<sub>4</sub> and concentrated to give a violet solid. Purification by HPLC (Reprospher C<sub>18</sub>; 30–100% D/A) gave <b>39</b> (5.0 mg, 0.018 mmol, 3%) as a white solid. Mp >300 °C (dec.). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 11.84 (s, 1H), 8.46 (d, <i>J</i> = 2.1 Hz, 1H), 7.67–7.62 (m, 2H), 7.29–7.25 (m, 1H), 7.22–7.14 (m, 3H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): δ 152.33, 143.57, 127.86, 127.63, 125.56, 123.11, 121.80, 113.26, 110.65; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>14</sub>H<sub>10</sub>ON<sub>4</sub>Cl 285.0538, found 285.0543.</div></div><div id="sec4_3_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 1-(1<i>H</i>-Benzo[<i>d</i>]imidazol-2-yl)-6-hydroxy-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>40</b>)</h4><div class="NLM_p">A solution of 4-<i>tert</i>-butyldimethylsiloxy-1,2-diaminobenzene (0.60 g, 2.52 mmol) and <b>52</b> (0.38 g, 2.5 mmol) in <i>n</i>-BuOH (5.0 mL) was treated with KH<sub>2</sub>PO<sub>4</sub> (0.34 g, 2.5 mmol) and stirred at 120 °C for 18 h. After addition of H<sub>2</sub>O (40 mL), the reaction mixture was extracted with EtOAc (3 × 100 mL), and the combined organic phases were dried over MgSO<sub>4</sub> and concentrated to give a white product, which was used for the next step without further purification.</div><div class="NLM_p">A solution of this intermediate (150 mg, 0.424 mmol) in DMF (0.4 mL) was treated with <i>N</i>,<i>N</i>′-carbonyldiimidazole (68.0 mg, 0.424 mmol) and stirred at room temperature for 17 h. After addition of H<sub>2</sub>O (20 mL), the reaction mixture was extracted with EtOAc (3 × 20 mL), and the combined organic phases were dried over MgSO<sub>4</sub> and concentrated to give a white solid. Purification by HPLC (Reprospher C<sub>18</sub>; 65–85% D/A) gave 1-(1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-5/6-<i>tert</i>-butyldimethylsilylether-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (52 mg, 0.14 mmol, 33%) as a white solid. Mp 201.2–204.0 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): mixture of two isomers with the same chemical shift δ 11.53 (s, 1H), 8.25 (d, <i>J</i> = 8.6 Hz, 1H), 7.60 (dd, <i>J</i> = 5.9, 3.2 Hz, 2H), 7.18 (dd, <i>J</i> = 6.0, 3.1 Hz, 2H), 6.71 (dd, <i>J</i> = 8.6, 2.3 Hz, 1H), 6.60 (d, <i>J</i> = 2.2 Hz, 1H), 0.97 (s, 9H), 0.21 (s, 6H); <sup>13</sup>  C NMR (DMSO- <i>d</i><sub>6</sub>, 75 MHz): mixture of two isomers with the same chemical shift δ 158.43, 152.63, 151.53, 144.01, 129.47, 121.61, 121.51, 114.76, 114.25, 113.02, 101.51, 25.59, 17.95,   −4.56; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>20</sub>H<sub>25</sub>O<sub>2</sub>N<sub>4</sub>Si 381.1741, found 381.1737.</div><div class="NLM_p last">This intermediate (37 mg, 0.10 mmol) in THF (0.6 mL) was treated with a solution of TBAF·3H<sub>2</sub>O (32 mg, 0.10 mmol) in THF (0.2 mL) and stirred at 0 °C for 5 min and at room temperature for 1 h. The reaction mixture was extracted with HCl (1 M)/ice water (1:1) and EtOAc (3 × 10 mL), and the combined organic phases were dried over MgSO<sub>4</sub> and concentrated to give a brown crude product. Purification by HPLC (Reprospher C<sub>18</sub>; 10–60% D/A) gave <b>40</b> (2.0 mg, 0.008 mmol, 7%) as the first fraction (separated from <b>6</b>) as a white solid. Mp 151.4–154.4 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 11.63 (s, 1H), 9.11 (s, 1H), 8.32 (dd, <i>J</i> = 6.0, 3.1 Hz, 1H), 7.39 (d, <i>J</i> = 8.6 Hz, 1H), 7.22–7.16 (m, 2H), 7.16–7.11 (m, 1H), 6.99 (d, <i>J</i> = 2.2 Hz, 1H), 6.66 (dd, <i>J</i> = 8.6, 2.3 Hz, 1H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz): δ 153.12, 152.35, 142.90, 128.49, 127.07, 123.22, 121.51, 113.24, 111.05, 109.35; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>14</sub>H<sub>11</sub>O<sub>2</sub>N<sub>4</sub> 267.0877, found 267.0872.</div></div><div id="sec4_3_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 1-(1<i>H</i>-Benzo[<i>d</i>]imidazol-2-yl)-5-chloro-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>41</b>)</h4><div class="NLM_p last">A solution of <b>32</b> (120 mg, 0.34 mmol) in DMF (1.7 mL) was treated with K<sub>2</sub>CO<sub>3</sub> (140 mg, 1.02 mmol) and stirred at 110 °C for 38 h. After addition of H<sub>2</sub>O (20 mL), the reaction mixture was extracted with EtOAc (3 × 15 mL), and the combined organic phases were dried over MgSO<sub>4</sub> and concentrated to give a gray solid. Purification by flash chromatography (SiO<sub>2</sub>; 0–5% MeOH/DCM) gave <b>41</b> (14 mg, 0.049 mmol, 14%) as a white solid. Mp >300 °C (dec.). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 12.44 (s, 1H), 11.84 (s, 1H), 8.40 (d, <i>J</i> = 8.6 Hz, 1H),  7.65–7.58 (m, 2H), 7.27 (dd, <i>J</i> = 8.6, 2.1 Hz, 1H), 7.22–7.16 (m, 3H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): δ 152.32, 143.61, 129.87, 127.45, 125.97, 121.38, 114.64, 109.38; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>14</sub>H<sub>10</sub>ON<sub>4</sub>Cl 285.0538, found 285.0544.</div></div><div id="sec4_3_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 1-(1<i>H</i>-Benzo[<i>d</i>]imidazol-2-yl)-4-hydroxy-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>42</b>)</h4><div class="NLM_p last">Purification of the reaction to form <b>38</b> by HPLC (Reprospher C<sub>18</sub>; 30–40% C/A) afforded also a second fraction in form of <b>42</b> (14 mg, 0.053 mmol, 19%) as a white solid. Mp 153.2–155.7 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 11.51 (s, 1H), 9.95 (s, 1H), 7.88 (d, <i>J</i> = 8.0 Hz, 1H), 7.61 (dd, <i>J</i> = 6.0, 3.2 Hz, 2H), 7.18 (dd, <i>J</i> = 6.0, 3.2 Hz, 2H), 7.00 (t, <i>J</i> = 8.1 Hz, 1H), 6.71 (d, <i>J</i> = 7.4 Hz, 1H); <sup>13</sup>  C NMR (DMSO- <i>d</i><sub>6</sub>, 75 MHz): δ 152.37, 144.07, 141.61, 128.27, 122.00, 121.60, 116.73, 110.31, 105.00; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>14</sub>H<sub>11</sub>O<sub>2</sub>N<sub>4</sub> 267.0877, found 267.0874.</div></div><div id="sec4_3_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> <i>tert</i>-Butyl 2-Amino-4-methylphenylcarbamate and <i>tert</i>-Butyl 2-Amino-5-methylphenylcarbamate (<b>43</b>)</h4><div class="NLM_p last">An ice-cold solution of 3,4-diaminotoluene (5.00 g, 40.9 mmol) and guanidine hydrochloride (0.587 g, 6.14 mmol) in EtOH (41 mL) was treated with Boc<sub>2</sub>O (8.93 g, 40.9 mmol), and the solution was stirred at room temperature for 2 h. EtOH was removed in vacuo. Flash chromatography (SiO<sub>2</sub>; 1:6 EtOAc/hexane) gave <b>43</b> (6.22 g, 28.1 mmol, 68%) as a white solid. Mp 101.9–104.1 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): mixture of two isomers δ 8.20 (d, <i>J</i> = 16.8 Hz, 1H), 7.02 (d, <i>J</i> = 7.9 Hz, 1H), 6.50 (tdd, <i>J</i> = 51.4, 8.0, 1.6 Hz, 2H), 4.66 (d, <i>J</i> = 33.8 Hz, 2H), 2.13 (s, 3H), 1.45 (d, <i>J</i> = 3.2 Hz, 9H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): mixture of two isomers δ 153.72, 153.53, 141.19, 133.88, 125.30, 124.84, 124.68, 123.82, 121.24, 117.01, 116.12, 115.85, 78.56, 78.44, 28.18, 20.69, 20.19; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>12</sub>H<sub>19</sub>O<sub>2</sub>N<sub>2</sub> 223.1441, found 223.1442.</div></div><div id="sec4_3_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 1-(1<i>H</i>-Benzo[<i>d</i>]imidazol-2-yl)-5-methyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one and 1-(1<i>H</i>-Benzo[<i>d</i>]imidazol-2-yl)-6-methyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>44</b>)</h4><div class="NLM_p">A solution of <b>43</b> (1.00 g, 4.50 mmol) and <b>19</b> (0.684 g, 4.50 mmol) in <i>n</i>-BuOH (9.0 mL) was treated with KH<sub>2</sub>PO<sub>4</sub> (612 mg, 1.81 mmol, 4.50 mmol) and stirred at 95 °C for 15 h. After addition of H<sub>2</sub>O (50 mL), the reaction mixture was extracted with EtOAc (3 × 30 mL), and the combined organic phases were dried over MgSO<sub>4</sub> and concentrated to give a white solid, which was used for the next step without further purification.</div><div class="NLM_p last">This product (800 mg, 2.37 mmol) was dissolved in DMF (12 mL) and treated with K<sub>2</sub>CO<sub>3</sub> (983 mg, 7.11 mmol) and stirred at 110 °C for 18 h. After addition of H<sub>2</sub>O (50 mL), the reaction mixture was extracted with EtOAc (3 × 40 mL), and the combined organic phases were dried over MgSO<sub>4</sub> and concentrated to give a brownish solid. Purification by HPLC (Reprospher C<sub>18</sub>; 40–55% D/A) gave <b>44</b> (123 mg, 20%) as a white solid. Mp 271.9–273.5 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): mixture of two isomers (ratio 5:1) with the same chemical shift δ 11.55 (s, 1H), 8.23 (s, 1H), 7.63 (dd, <i>J</i> = 6.0, 3.2 Hz, 2H), 7.19 (dd, <i>J</i> = 5.9, 3.2 Hz, 2H), 7.03 (d, <i>J</i> = 0.8 Hz, 2H), 2.42 (s, 3H); <sup>13</sup> C NMR (DMSO- <i>d</i><sub>6</sub>, 75 MHz): mixture of two isomers δ 152.58, 152.54, 144.02, 143.97, 130.78, 127.14, 126.34, 123.90, 122.14, 121.68, 113.79, 109.14, 21.30, 21.04; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>15</sub>H<sub>13</sub>ON<sub>4</sub> 265.1084, found 265.1082.</div></div><div id="sec4_3_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 3-(1<i>H</i>-Benzo[<i>d</i>]imidazol-2-yl)-2,3-dihydro-2-oxo-1<i>H</i>-benzo[<i>d</i>]imidazole-6-carboxylic acid (<b>45</b>)</h4><div class="NLM_p last">H<sub>5</sub>IO<sub>6</sub> (249 mg, 1.09 mmol) was dissolved in MeCN (2.9 mL) under vigorous stirring, and CrO<sub>3</sub> (6.0 mg, 0.2 equiv) was added to the transparent solution and stirred at room temperature for 5 min. The orange solution was slowly treated with <b>44</b> (82.0 mg, 0.310 mmol) and stirred for 1 h. H<sub>2</sub>O (5.0 mL) was added, and the reaction mixture was extracted with EtOAc (4 × 20 mL). The combined organic phases were dried over MgSO<sub>4</sub> and concentrated to give a white solid. Separation from <b>46</b> by HPLC (Reprospher C<sub>18</sub>; 30–65% D/A) gave <b>45</b> (42.0 mg, 0.14 mmol, 46%) as a white solid. Mp >300 °C (dec.). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 12.91 (s, 1H), 12.05 (s, 1H), 8.98 (s, 1H), 7.87 (dd, <i>J</i> = 8.2, 1.5 Hz, 1H), 7.66 (dd, <i>J</i> = 5.8, 3.1 Hz, 2H),  7.27–7.16 (m, 3H); <sup>13</sup> C NMR (DMSO- <i>d</i><sub>6</sub>, 75 MHz): δ 167.31, 152.63, 143.60, 132.50, 126.92, 125.57, 124.10, 121.77, 114.51, 109.06; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>15</sub>H<sub>11</sub>O<sub>3</sub>N<sub>4</sub> 295.0826, found 295.0824.</div></div><div id="sec4_3_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 1-(1<i>H</i>-Benzo[<i>d</i>]imidazol-2-yl)-2,3-dihydro-2-oxo-1<i>H</i>-benzo[<i>d</i>]imidazole-5-carboxylic acid (<b>46</b>)</h4><div class="NLM_p last">Purification of <b>45</b> by HPLC (Reprospher C<sub>18</sub>; 30–65% D/A) gave as a second fraction product <b>46</b> (16 mg, 0.054 mmol, 18%) as a white solid. Mp >300 °C (dec.). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 11.91 (s, 1H), 8.47 (d, <i>J</i> = 8.4 Hz, 1H), 7.87 (dd, <i>J</i> = 8.5, 1.6 Hz, 1H), 7.68–7.61 (m, 3H), 7.21 (dd, <i>J</i> = 6.0, 3.2 Hz, 2H); <sup>13</sup>C NMR (DMSO- <i>d</i><sub>6</sub>, 101 MHz): δ 167.04, 152.52, 143.47, 128.61, 125.73, 123.66, 121.83, 112.97, 110.01; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>15</sub>H<sub>11</sub>O<sub>3</sub>N<sub>4</sub> 295.0826, found 295.0827.</div></div><div id="sec4_3_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 3-(1<i>H</i>-Benzo[<i>d</i>]imidazol-2-yl)-2,3-dihydro-<i>N</i>-methyl-2-oxo-1<i>H</i>-benzo[<i>d</i>]imidazole-6-carboxamide (<b>47</b>)</h4><div class="NLM_p last">A solution of <b>45</b> (4.0 mg, 0.014 mmol) in DMF (0.5 mL) was treated with BOP (6.6 mg, 0.015 mmol), triethylamine (0.011 mL, 0.070 mmol), and methylamine hydrochloride (4.7 mg, 0.070 mmol) and stirred at room temperature for 4 h. The reaction crude was directly purified by HPLC (Reprospher C<sub>18</sub>; 15–60% C/A) to give <b>47</b> (2.5 mg, 0.0081 mmol, 58%) as a white solid. Mp >300 °C (dec.). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 11.92 (s, 1H), 8.81 (d, <i>J</i> = 1.3 Hz, 1H), 8.45 (d, <i>J</i> = 4.5 Hz, 1H), 7.70 (dd, <i>J</i> = 8.2, 1.7 Hz, 1H), 7.66 (dd, <i>J</i> = 6.0, 3.2 Hz, 2H), 7.24–7.16 (m, 3H), 2.81 (d, <i>J</i> = 4.5 Hz, 3H); <sup>13</sup> C NMR (DMSO- <i>d</i><sub>6</sub>, 101 MHz): δ 166.73, 152.66, 143.55, 130.86, 128.57, 126.99, 122.57, 121.75, 112.72, 108.69, 26.35; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>16</sub>H<sub>14</sub>O<sub>2</sub>N<sub>5</sub> 308.1142, found 308.1141.</div></div><div id="sec4_3_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 1-(1<i>H</i>-Benzo[<i>d</i>]imidazol-2-yl)-2,3-dihydro-<i>N</i>-methyl-2-oxo-1<i>H</i>-benzo[<i>d</i>]imidazole-5-carboxamide (<b>48</b>)</h4><div class="NLM_p last">A solution of <b>46</b> (2.5 mg, 0.0085 mmol) in DMF (0.5 mL) was treated with BOP (4.5 mg, 0.010 mmol), triethylamine (0.011 mL, 0.050 mmol), and methylamine hydrochloride (3.4 mg, 0.050 mmol) and stirred at room temperature for 4 h. The reaction crude was directly purified by HPLC (Reprospher C<sub>18</sub>; 15–60% D/A) to give <b>48</b> (1.5 mg, 0.0049 mmol, 54%) as a white solid. Mp 172.5–175.0 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 11.91 (s, 1H), 8.50 (d, <i>J</i> = 4.6 Hz, 1H), 8.41 (d,  <i>J</i> = 8.4 Hz, 1H), 7.74 (dd, <i>J</i> = 8.4, 1.7 Hz, 1H), 7.66–7.60 (m, 3H), 7.20 (dd, <i>J</i> = 6.0, 3.2 Hz, 2H), 2.80 (d, <i>J</i> = 4.5 Hz, 3H); <sup>13</sup>C NMR (DMSO- <i>d</i><sub>6</sub>, 101 MHz): δ 166.17, 152.62, 144.64, 143.62, 131.02, 129.81, 129.03, 121.76, 120.98, 112.70, 108.27, 26.29; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>16</sub>H<sub>14</sub>O<sub>2</sub>N<sub>5</sub> 308.1142, found 308.1141.</div></div><div id="sec4_3_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 2-Chloro-1-methyl-1<i>H</i>-benzo[<i>d</i>]imidazole (<b>49</b>)</h4><div class="NLM_p last">A solution of <b>19</b> (1.00 g, 6.58 mmol) in DMF (10 mL) at 0 °C was treated with NaH (0.284 g, 11. Eight mmol) and stirred for 15 min. MeI (0.431 mL, 6.91 mmol) was added, and the mixture was stirred for 15 min. The reaction mixture was then poured into H<sub>2</sub>O (75 mL), and the white precipitate was filtered and washed with H<sub>2</sub>O (3 × 30 mL). The white solid was dried in vacuo to give <b>49</b> (604 mg, 3.64 mmol, 55%) as a white solid. Mp 115.2–117.8 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 7.59 (m, 2H), 7.28 (m, 2H), 3.79 (s, 3H); <sup>13</sup>C NMR (DMSO- <i>d</i><sub>6</sub>, 75 MHz): δ 141.02, 140.24, 135.61, 122.80, 122.29, 118.46, 110.46, 30.49; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>8</sub>H<sub>8</sub>ClN <sub>2</sub> 167.0371, found 167.0370.</div></div><div id="sec4_3_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 2-Chloro-1-methyl-1<i>H</i>-benzo[<i>d</i>]imidazole (<b>50</b>)</h4><div class="NLM_p last">A solution of <b>49</b> (500 mg, 3.01 mmol) and <i>tert</i>-butyl (2-aminophenyl)carbamate (627 mg, 3.01 mmol) in <i>n</i>-BuOH (6.0 mL) was treated with KH<sub>2</sub>PO<sub>4</sub> (410 mg, 3.01 mmol) and stirred at 90 °C for 22 h. Reaction control indicated incomplete reaction. The temperature was elevated to 120 °C, and the reaction mixture was stirred for additional 22 h. After addition of H<sub>2</sub>O (60 mL), the reaction mixture was extracted with EtOAc (3 × 30 mL), and the combined organic phases were dried over MgSO<sub>4</sub> and concentrated to give a reddish solid. Purification by flash chromatography (SiO<sub>2</sub>; 5–100% EtOAc/hexane) gave <b>50</b> (302 mg, 0.89 mmol, 30%) as an off-white solid. Mp 212.7–214.9 °C.  <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 8.99 (s, 1H), 7.78 (d, <i>J</i> = 7.7 Hz, 1H), 7.48 (d, <i>J</i> = 7.7 Hz, 1H), 7.31–7.25 (m, 2H), 7.16–6.99 (m, 5H), 3.67 (s, 3H), 1.45 (d, <i>J</i> = 0.8 Hz, 9H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): mixture of isomers δ 155.23, 153.62, 152.67, 151.39, 141.99, 141.43, 141.63, 134.92, 134.54, 132.39, 130.07, 129.65, 129.48, 128.98, 124.70, 124.34, 124.06, 123.37, 123.29, 122.90, 122.84, 122.66, 122.32, 121.32, 120.79, 120.43, 120.34, 119.58, 119.16, 116.23, 115.90, 110.69, 110.37, 110.02, 109.58, 108.42, 107.97, 107.49, 79.40, 48.57, 28.03; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>19</sub>H<sub>23</sub>O<sub>2</sub>N<sub>4</sub> 339.1816, found 339.1815.</div></div><div id="sec4_3_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 1-(1-Methyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>51</b>)</h4><div class="NLM_p last">A solution of <b>50</b> (222 mg, 0.656 mmol) in DMF (3.3 mL) was treated with K<sub>2</sub>CO<sub>3</sub> (274 mg, 1.98 mmol) and stirred at 110 °C for 16 h. After addition of H<sub>2</sub>O (40 mL), the reaction mixture was extracted with EtOAc (3 × 20 mL), and the combined organic phases were dried over MgSO<sub>4</sub> and concentrated to give a solid. Purification by flash chromatography (SiO<sub>2</sub>; 5–100% EtOAc/hexane) gave <b>51</b> (84.0 mg, 0.32 mmol, 48%) as a white solid, mp 239.4–241.6 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 11.45 (s, 1H), 7.74–7.65 (m, 2H), 7.41–7.29 (m, 2H), 7.17–7.03 (m, 4H), 3.75 (s, 3H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): δ 152.65, 141.97, 140.62, 134.91, 129.46, 128.96, 122.89, 122.83, 122.31, 121.31, 119.14, 110.69, 110.01, 109.56, 30.42; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>15</sub>H<sub>13</sub>ON<sub>4</sub> 265.1084, found 265.1082.</div></div><div id="sec4_3_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> <i>tert</i>-Butyl 2-Aminophenylmethylcarbamate (<b>52</b>)</h4><div class="NLM_p last">A solution of <i>tert</i>-butyl methyl 2-nitrophenylcarbamate (6.30 g, 25.0 mmol) in dioxane/H<sub>2</sub>O (5:1, 166 mL) was treated with Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub> (17.4 g, 100 mmol) and NaHCO<sub>3</sub> (21.0 g, 250 mmol) and refluxed at 100 °C for 45 min. After addition of H<sub>2</sub>O (90 mL), the reaction mixture was extracted with EtOAc (3 × 70 mL), and the combined organic phases were dried over MgSO<sub>4</sub> and concentrated to give an orange solid. Purification by flash chromatography (Si<sub>2</sub>O; 5–100% EtOAc/hexane) gave <b>52</b> (1.19 g, 5.36 mmol, 22%) as a light brown solid. Mp 87.4–89.6 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ  7.00–6.83 (m, 2H), 6.69 (d, <i>J</i> = 7.7 Hz, 1H), 6.50 (t, <i>J</i> = 7.4 Hz, 1H), 4.85 (s, 2H), 2.98 (s, 3H), 1.30 (s, 9H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): δ 154.39, 144.09, 128.28, 127.83, 127.39, 116.02, 115.38, 78.53, 35.63, 27.99; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>12</sub>H<sub>19</sub>O<sub>2</sub>N<sub>2</sub> 223.1441, found 223.1443.</div></div><div id="sec4_3_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 1-(1<i>H</i>-Benzo[<i>d</i>]imidazol-2-yl)-3-methyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>53</b>)</h4><div class="NLM_p last">A solution of <b>52</b> (1.10 g, 4.95 mmol) and <b>19</b> (0.684 g, 4.95 mmol) in <i>n</i>-BuOH (10 mL) was treated with KH<sub>2</sub>PO<sub>4</sub> (0.612 g, 4.95 mmol) and stirred at 120 °C for 24 h. After addition of H<sub>2</sub>O (40 mL), the reaction mixture was extracted with EtOAc (3 × 30 mL), and the combined organic phases were dried over MgSO<sub>4</sub> and concentrated to give a white crude solid. Purification by HPLC (Reprospher C<sub>18</sub>; 20–80% D/A) gave <b>53</b> (8.00 mg, 0.03 mmol, 1%) as a white solid. Mp 258.3–260.5 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ 12.47 (s, 1H), 8.51–8.39 (m, 1H), 7.67–7.59 (m, 2H), 7.37–7.17 (m, 5H), 3.48 (s, 3H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz): δ 151.68, 143.78, 130.10, 125.78, 123.37, 122.01, 121.70, 113.29, 108.53, 27.14; HR-ESI-MS (<i>m</i>/<i>z</i>): calcd for [M + H]<sup>+</sup> C<sub>15</sub>H<sub>13</sub>ON<sub>4</sub> 265.1084, found 265.1087.</div></div><div id="sec4_3_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> (<i>Z</i>)-2-((4-Ethylphenyl)imino)-3-methylthiazolidin-4-one (<b>54</b>)</h4><div class="NLM_p last">A solution of 4-ethylaniline (2.15 mL, 17.3 mmol, 1.15 equiv) in EtOH (50 mL) was treated with methyl isothiocyanate (1.01 g, 15.0 mmol, 1.0 equiv) and stirred at 78 °C for 12 h. The solution was then treated with NaOAc (6.23 g, 75.0 mmol, 5.0 equiv) and ethyl chloroacetate (3.22 mL, 30.0 mmol, 2.0 equiv) and stirred at 78 °C for another 5 h. EtOH was evaporated in vacuo to afford crude <b>54</b> as yellow oil (2.25 g, 9.60 mmol, 64.0%), which was used for the next step without further purification. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.19 (d, <i>J</i> = 8.4 Hz, 2H), 7.88–6.83 (m, 2H), 4.00 (s, 2H), 3.15 (s, 3H), 2.59 (q, <i>J</i> = 7.6 Hz, 2H), 1.18 (t, <i>J</i> = 7.6 Hz, 3H); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.91, 155.36, 145.75, 139.53, 128.50, 120.76, 32.54, 29.15, 27.57, ESI-MS (<i>m</i>/<i>z</i>): 235.09.</div></div><div id="sec4_3_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> (<i>Z</i>)-2-((3-Ethylphenyl)imino)-3-methylthiazolidin-4-one (<b>55</b>)</h4><div class="NLM_p last">Using the same <a class="ref internalNav" href="#sec4_3_47" aria-label="procedure">procedure</a> as for <b>54</b>, this compound was obtained from 3-ethylaniline (2.10 g, 17.3 mmol, 1.15 equiv) as brown oil (3.08 g, 13.2 mmol, 88%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.18 (t, <i>J</i> = 7.6 Hz, 1H), 6.93–6.89 (m, 1H), 6.75–6.71 (m, 2H), 3.73 (s, 2H), 3.24 (s, 3H), 2.57 (q, <i>J</i> = 7.6 Hz, 2H), 1.16 (t, <i>J</i> = 7.6 Hz, 3H); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 171.80, 155.10, 147.75, 145.60, 129.17, 124.44, 120.63, 118.21, 32.84, 29.75, 28.84, 15.47; HR-ESI-MS (<i>m</i>/<i>z</i>): 235.0897, calcd for [M + H]<sup>+</sup> C<sub>12</sub>H<sub>15</sub>ON<sub>2</sub>S: 235.0900.</div></div><div id="sec4_3_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> (<i>Z</i>)-2-((2-Ethylphenyl)imino)-3-methylthiazolidin-4-one (<b>56</b>)</h4><div class="NLM_p last">Using the same <a class="ref internalNav" href="#sec4_3_47" aria-label="procedure">procedure</a> as for <b>54</b>, this compound was obtained from 2-ethylaniline (2.09 g, 17.3 mmol, 1.15 equiv) as brown oil (3.23 g, 13.8 mmol, 92%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.18–7.00 (m, 3H), 6.80–6.77 (m, 1H), 3.73 (s, 2H), 3.27 (s, 3H), 2.47 (q, <i>J</i> = 7.5 Hz, 2H), 1.09 (t, <i>J</i> = 7.5 Hz, 3H); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 171.81, 154.40, 145.87, 136.08, 129.04, 126.54, 125.03, 119.88, 32.93, 29.72, 29.67, 24.66, 14.49; HR-ESI-MS (<i>m</i>/<i>z</i>): 235.0898, calcd for [M + H]<sup>+</sup> C<sub>12</sub>H<sub>15</sub>ON<sub>2</sub>S: 235.0900.</div></div><div id="sec4_3_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> (<i>Z</i>)-3-Methyl-2-((4-propylphenyl)imino)thiazolidin-4-one (<b>57</b>)</h4><div class="NLM_p last">Using the same <a class="ref internalNav" href="#sec4_3_47" aria-label="procedure">procedure</a> as for <b>54</b>, this compound was obtained from 4-propylaniline (2.34 g, 17.3 mmol, 1.15 equiv) as brown oil (3.40 g, 13.7 mmol, 91%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.08 (d, <i>J</i> = 8.3 Hz, 2H), 6.82 (d, <i>J</i> = 8.3 Hz, 2H), 3.73 (s, 2H), 3.25 (s, 3H), 2.53–2.48 (m, 2H), 1.63–1.51 (m, 2H), 0.87 (t, <i>J</i> = 7.3 Hz, 3H); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 171.13, 144.61, 138.59, 128.64, 120.26, 36.91, 32.17, 29.13, 23.91, 13.19; HR-ESI-MS (<i>m</i>/<i>z</i>): 249.1049, calcd for [M + H]<sup>+</sup> C<sub>13</sub>H<sub>17</sub>ON<sub>2</sub>S: 249.1056.</div></div><div id="sec4_3_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> (<i>Z</i>)-2-((4-Isopropylphenyl)imino)-3-methylthiazolidin-4-one (<b>58</b>)</h4><div class="NLM_p last">Using the same <a class="ref internalNav" href="#sec4_3_47" aria-label="procedure">procedure</a> as for <b>54</b>, this compound was obtained from 4-isopropylaniline (2.34 g, 17.3 mmol, 1.15 equiv) as brown oil (3.62 g, 14.6 mmol, 97%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.11 (d, <i>J</i> = 8.4 Hz, 2H), 6.81 (d, <i>J</i> = 8.3 Hz, 2H), 3.70 (s, 2H), 3.21 (s, 3H), 2.88–2.74 (m, 1H), 1.16 (d, <i>J</i> = 7.0 Hz, 6H); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 171.83, 154.52, 145.53, 145.21, 127.21, 120.88, 33.60, 32.80, 29.67, 24.10; HR-ESI-MS (<i>m</i>/<i>z</i>): 249.1054, calcd for [M + H]<sup>+</sup> C<sub>13</sub>H<sub>17</sub>ON<sub>2</sub>S: 249.1056.</div></div><div id="sec4_3_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> (<i>Z</i>)-2-((4-Butylphenyl)imino)-3-methylthiazolidin-4-one (<b>59</b>)</h4><div class="NLM_p last">Using the same <a class="ref internalNav" href="#sec4_3_47" aria-label="procedure">procedure</a> as for <b>54</b>, this compound was obtained from 4-butylaniline (2.58 g, 17.3 mmol, 1.15 equiv) as brown oil (3.07 g, 11.7 mmol, 78%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.08 (d, <i>J</i> = 8.3 Hz, 2H), 6.81 (d, <i>J</i> = 8.3 Hz, 2H), 3.73 (s, 2H), 3.24 (s, 3H), 2.75–2.39 (m, 2H), 1.66–1.42 (m, 2H), 1.42–1.16 (m, 2H), 1.09–0.66 (m, 3H); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 171.79, 154.87, 145.27, 139.43, 129.22, 120.90, 35.17, 33.66, 32.82, 29.76, 22.39, 14.02; HR-ESI-MS (<i>m</i>/<i>z</i>): 263.1204, calcd for [M + H]<sup>+</sup> C<sub>14</sub>H<sub>19</sub>ON<sub>2</sub>S: 263.1213.</div></div><div id="sec4_3_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> (<i>Z</i>)-2-((4-(<i>tert</i>-Butyl)phenyl)imino)-3-methylthiazolidin-4-one (<b>60</b>)</h4><div class="NLM_p last">Using the same <a class="ref internalNav" href="#sec4_3_47" aria-label="procedure">procedure</a> as for <b>54</b>, this compound was obtained from 4-<i>tert</i>-butylaniline (2.58 g, 17.3 mmol, 1.15 equiv) as brown oil (3.20 g, 12.2 mmol, 81%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.28 (d, <i>J</i> = 8.6 Hz, 2H), 6.84 (d, <i>J</i> = 8.6 Hz, 2H), 3.73 (s, 2H), 3.24 (s, 3H), 1.25 (s, 9H); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 171.79, 154.65, 147.63, 144.98, 126.14, 120.63, 34.45, 32.82, 31.48, 29.76; HR-ESI-MS (<i>m</i>/<i>z</i>): 263.1206, calcd for [M + H]<sup>+</sup> C<sub>14</sub>H<sub>19</sub>ON<sub>2</sub>S: 263.1213.</div></div><div id="sec4_3_54" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> (<i>Z</i>)-3-Methyl-2-((4-(trifluoromethyl)phenyl)imino)thiazolidin-4-one (<b>61</b>)</h4><div class="NLM_p last">Using the same <a class="ref internalNav" href="#sec4_3_47" aria-label="procedure">procedure</a> as for <b>54</b>, this compound was obtained from 4-(trifluoromethyl)aniline (2.79 g, 17.3 mmol, 1.15 equiv) as brown oil (3.95 g, 14.4 mmol, 96%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.52 (d, <i>J</i> = 8.4 Hz, 2H), 6.97 (d, <i>J</i> = 8.1 Hz, 2H), 3.76 (s, 2H), 3.22 (s, 3H); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 171.59, 155.99, 151.18, 151.15, 127.24–121.37 (m), 32.87, 29.65, 29.61; HR-ESI-MS (<i>m</i>/<i>z</i>): 275.0458, calcd for [M + H]<sup>+</sup> C<sub>11</sub>H<sub>10</sub>ON<sub>2</sub>F<sub>3</sub>S: 275.0460.</div></div><div id="sec4_3_55" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> (<i>Z</i>)-2-((4-Bromophenyl)imino)-3-methylthiazolidin-4-one (<b>62</b>)</h4><div class="NLM_p last">Using the same <a class="ref internalNav" href="#sec4_3_47" aria-label="procedure">procedure</a> as for <b>54</b>, this compound was obtained from 4-bromoaniline (2.98 g, 17.3 mmol, 1.15 equiv) as brown oil (3.82 g, 13.4 mmol, 89%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.38 (d, <i>J</i> = 8.5 Hz, 2H), 6.78 (d, <i>J</i> = 8.6 Hz, 2H), 3.75 (s, 2H), 3.23 (s, 3H); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 171.61, 155.77, 146.89, 132.37, 122.95, 117.80, 32.90, 29.77, 29.72; HR-ESI-MS (<i>m</i>/<i>z</i>): 284.9687, calcd for [M + H]<sup>+</sup> C<sub>10</sub>H<sub>10</sub>ON<sub>2</sub>BrS: 284.9692.</div></div><div id="sec4_3_56" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> (2<i>Z</i>,5<i>Z</i>)-2-((3-Ethylphenyl)imino)-3-methyl-5-(pyridin-4-ylmethylene)thiazolidin-4-one (<b>63</b>)</h4><div class="NLM_p last">Using the same <a class="ref internalNav" href="#sec4_3_4" aria-label="procedure">procedure</a> as for <b>8</b>, this compound was obtained from <b>55</b> (702 mg, 3.00 mmol, 1.0 equiv) as yellow solid (795 mg, 2.46 mmol, 82%). Mp 116.2–116.8 °C.<sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.68–8.66 (m, 2H), 7.72 (s, 1H), 7.47 (d, <i>J</i> = 6.2 Hz, 2H), 7.34 (t, <i>J</i> = 7.6 Hz, 1H), 7.07 (dt, <i>J</i> = 7.6, 1.3 Hz, 1H), 6.88–6.84 (m, 2H), 3.36 (s, 3H), 2.64 (q, <i>J</i> = 7.6 Hz, 2H), 1.21 (t, <i>J</i> = 7.6 Hz, 3H); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.42, 150.41, 149.29, 147.55, 145.27, 140.41, 129.38, 126.86, 126.67, 124.52, 123.17, 120.42, 117.93, 29.70, 28.01, 15.34; HR-ESI-MS (<i>m</i>/<i>z</i>): 324.1164, calcd for [M + H]<sup>+</sup> C<sub>18</sub>H<sub>18</sub>ON<sub>3</sub>S: 324.1165.</div></div><div id="sec4_3_57" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> (2<i>Z</i>,5<i>Z</i>)-2-((2-Ethylphenyl)imino)-3-methyl-5-(pyridin-4-ylmethylene)thiazolidin-4-one (<b>64</b>)</h4><div class="NLM_p last">Using the same <a class="ref internalNav" href="#sec4_3_4" aria-label="procedure">procedure</a> as for <b>8</b>, this compound was obtained from <b>56</b> (702 mg, 3.00 mmol, 1.0 equiv) as yellow solid (882 mg, 2.73 mmol, 91%). Mp 107.7–110.8 °C. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.76–8.65 (m, 2H), 7.72 (s, 1H), 7.46 (d, <i>J</i> = 6.2 Hz, 2H), 7.33–7.14 (m, 3H), 6.93 (dd, <i>J</i> = 7.6, 1.4 Hz, 1H), 2.57–2.50 (m, 2H), 1.11 (t, <i>J</i> = 7.5 Hz, 3H); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.44, 150.43, 148.98, 145.69, 140.38, 135.66, 129.02, 126.86, 126.78, 126.74, 125.26, 123.15, 119.78, 29.67, 24.07, 14.50; HR-ESI-MS (<i>m</i>/<i>z</i>): 324.1164, calcd for [M + H]<sup>+</sup> C<sub>18</sub>H<sub>18</sub>ON<sub>3</sub>S: 324.1165.</div></div><div id="sec4_3_58" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> (2<i>Z</i>,5<i>Z</i>)-3-Methyl-2-((4-propylphenyl)imino)-5-(pyridin-4-ylmethylene)thiazolidin-4-one (<b>65</b>)</h4><div class="NLM_p last">Using the same <a class="ref internalNav" href="#sec4_3_4" aria-label="procedure">procedure</a> as for <b>8</b>, this compound was obtained from <b>57</b> (744 mg, 3.00 mmol, 1.0 equiv) as yellow solid (485 mg, 1.44 mmol, 48%). Mp 131.6–133.3 °C. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.59–8.57 (m, 2H), 7.56 (s, 1H), 7.24 (dd, <i>J</i> = 4.5, 1.7 Hz, 2H), 7.13 (d, <i>J</i> = 8.2 Hz, 2H), 6.85 (d, <i>J</i> = 8.3 Hz, 2H), 3.40 (s, 3H), 2.56–2.51 (m, 2H), 1.64–1.56 (m, 2H), 0.90 (t, <i>J</i> = 7.3 Hz, 3H); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 166.13, 150.12, 149.06, 145.24, 141.34, 139.82, 129.48, 127.63, 126.91, 123.40, 120.86, 37.59, 30.01, 24.56, 13.89; HR-ESI-MS (<i>m</i>/<i>z</i>): 338.1320, calcd for [M + H]<sup>+</sup> C<sub>19</sub>H<sub>20</sub>ON<sub>3</sub>S: 338.1322.</div></div><div id="sec4_3_59" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> (2<i>Z</i>,5<i>Z</i>)-2-((4-Isopropylphenyl)imino)-3-methyl-5-(pyridin-4-ylmethylene)thiazolidin-4-one (<b>66</b>)</h4><div class="NLM_p last">Using the same <a class="ref internalNav" href="#sec4_3_4" aria-label="procedure">procedure</a> as for <b>8</b>, this compound was obtained from <b>58</b> (744 mg, 3.00 mmol, 1.0 equiv) as yellow solid (880 mg, 2.61 mmol, 87%). Mp 159.2–159.3 °C. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.58 (d, <i>J</i> = 6.2 Hz, 2H), 7.55 (s, 1H), 7.23 (d, <i>J</i> = 6.2 Hz, 2H), 7.18 (d, <i>J</i> = 8.4 Hz, 2H), 6.86 (d, <i>J</i> = 8.3 Hz, 2H), 3.39 (s, 3H), 2.86 (dt, <i>J</i> = 14.0, 7.1 Hz, 1H), 1.21 (d, <i>J</i> = 6.9 Hz, 6H); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 166.08, 150.18, 148.87, 145.90, 145.21, 141.23, 127.55, 127.43, 126.89, 123.33, 120.89, 33.63, 29.96, 24.03; HR-ESI-MS (<i>m</i>/<i>z</i>): 338.1321, calcd for [M + H]<sup>+</sup> C<sub>19</sub>H<sub>20</sub>ON<sub>3</sub>S: 338.1322.</div></div><div id="sec4_3_60" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> (2<i>Z</i>,5<i>Z</i>)-2-((4-Butylphenyl)imino)-3-methyl-5-(pyridin-4-ylmethylene)thiazolidin-4-one (<b>67</b>)</h4><div class="NLM_p last">Using the same <a class="ref internalNav" href="#sec4_3_4" aria-label="procedure">procedure</a> as for <b>8</b>, this compound was obtained from <b>59</b> (786 mg, 3.00 mmol, 1.0 equiv) as yellow solid (748 mg, 2.13 mmol, 71%). Mp 104.6–105.2 °C. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.57 (d, <i>J</i> = 6.2 Hz, 2H), 7.55 (s, 1H), 7.22 (d, <i>J</i> = 6.2 Hz, 2H), 7.13 (d, <i>J</i> = 8.3 Hz, 2H), 6.84 (d, <i>J</i> = 8.3 Hz, 2H), 3.39 (s, 3H), 2.88–2.31 (m, 2H), 1.77–1.42 (m, 2H), 1.31 (q, <i>J</i> = 7.4 Hz, 3H), 0.87 (t, <i>J</i> = 7.3 Hz, 3H); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 166.13, 150.26, 149.07, 145.21, 141.18, 140.02, 129.42, 127.50, 126.98, 123.36, 120.88, 35.20, 33.64, 29.99, 22.45, 14.03; HR-ESI-MS (<i>m</i>/<i>z</i>): 352.1471, calcd for [M + H]<sup>+</sup> C<sub>20</sub>H<sub>22</sub>ON<sub>3</sub>S: 352.1478.</div></div><div id="sec4_3_61" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> (2<i>Z</i>,5<i>Z</i>)-2-((4-(<i>tert</i>-Butyl)phenyl)imino)-3-methyl-5-(pyridin-4-ylmethylene)thiazolidin-4-one (<b>68</b>)</h4><div class="NLM_p last">Using the same <a class="ref internalNav" href="#sec4_3_4" aria-label="procedure">procedure</a> as for <b>8</b>, this compound was obtained from <b>60</b> (786 mg, 3.00 mmol, 1.0 equiv) as yellow solid (453 mg, 1.29 mmol, 43%). Mp 154.9–158.4 °C. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.58–8.56 (m, 2H), 7.55 (s, 1H), 7.33 (d, <i>J</i> = 8.6 Hz, 2H), 7.21 (d, <i>J</i> = 6.2 Hz, 2H), 6.87 (d, <i>J</i> = 8.5 Hz, 2H), 3.38 (s, 3H), 1.27 (s, 9H); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 166.16, 150.56, 148.92, 148.20, 144.87, 140.90, 127.28, 127.09, 126.37, 123.28, 120.63, 34.53, 31.47, 29.99; HR-ESI-MS (<i>m</i>/<i>z</i>): 352.1470, calcd for [M + H]<sup>+</sup> C<sub>20</sub>H<sub>22</sub>ON<sub>3</sub>S: 352.1478.</div></div><div id="sec4_3_62" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> (2<i>Z</i>,5<i>Z</i>)-3-Methyl-5-(pyridin-4-ylmethylene)-2-((4-(trifluoromethyl)phenyl)imino)thiazolidin-4-one (<b>69</b>)</h4><div class="NLM_p last">Using the same <a class="ref internalNav" href="#sec4_3_4" aria-label="procedure">procedure</a> as for <b>8</b>, this compound was obtained from <b>61</b> (822 mg, 3.00 mmol, 1.0 equiv) as yellow solid (643 mg, 1.77 mmol, 59%). Mp 167.7–168.6 °C. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.69–8.67 (m, 12), 7.82–7.77 (m, 3H), 7.50 (dd, <i>J</i> = 4.6, 1.7 Hz, 2H), 7.26 (d, <i>J</i> = 8.6 Hz, 2H), 3.38 (s, 3H); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.37, 151.10, 150.87, 150.54, 140.14, 127.62, 126.79, 126.74, 126.10, 126.07, 125.48, 125.05, 124.63, 123.19, 122.50, 121.79, 29.72; HR-ESI-MS (<i>m</i>/<i>z</i>): 364.0724, calcd for [M + H]<sup>+</sup> C<sub>17</sub>H<sub>13</sub>ON<sub>3</sub>F<sub>3</sub>S: 364.0726.</div></div><div id="sec4_3_63" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> (2<i>Z</i>,5<i>Z</i>)-2-((4-Bromophenyl)imino)-3-methyl-5-(pyridin-4-ylmethylene)thiazolidin-4-one (<b>70</b>)</h4><div class="NLM_p last">Using the same <a class="ref internalNav" href="#sec4_3_4" aria-label="procedure">procedure</a> as for <b>8</b>, this compound was obtained from <b>62</b> (855 mg, 3.00 mmol, 1.0 equiv) as yellow solid (426 mg, 1.77 mmol, 59%). Mp 191.9–192.3 °C. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.59 (d, <i>J</i> = 6.2 Hz, 2H), 7.58 (s, 1H), 7.43 (d, <i>J</i> = 8.6 Hz, 2H), 7.22 (d, <i>J</i> = 6.2 Hz, 2H), 6.81 (d, <i>J</i> = 8.6 Hz, 2H), 3.38 (s, 3H); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 165.93, 150.30, 150.23, 146.75, 140.94, 132.56, 127.63, 126.83, 123.27, 122.85, 118.44, 29.93; HR-ESI-MS (<i>m</i>/<i>z</i>): 373.9961, calcd for [M + H]<sup>+</sup> C<sub>16</sub>H<sub>13</sub>ON<sub>3</sub>BrS: 373.9957.</div></div><div id="sec4_3_64" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> (<i>Z</i>)-2-((4-Ethylphenyl)imino)thiazolidin-4-one (<b>71</b>)</h4><div class="NLM_p last">4-Ethylphenylisothiocyanate (2.00 g, 12.3 mmol, 1.0 equiv) was refluxed in a saturated NH<sub>3</sub> solution of MeOH (10 mL) for 18 h. The solution was cooled to room temperature and evaporated in vacuo to afford a pale yellow solid. The solid was added to a solution of ethyl chloroacetate (2.63 mL, 24.6 mmol, 2 equiv) and sodium acetate (2.98 g, 36.9 mmol, 3 equiv) in ethanol (33 mL), and the mixture was stirred at 78 °C for 6 h. After removal of EtOH in vacuo, the resulting solid was dissolved in EtOAc (200 mL) and washed with brine (5 × 40 mL) and water (5 × 40 mL). Evaporation of EtOAc in vacuo afforded the product as brown solid (2.41 g, 10.9 mmol, 89.0%), which was used for the next steps without further purification.</div></div><div id="sec4_3_65" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> (<i>Z</i>)-2-((4-Ethylphenyl)imino)-3-ethylthiazolidin-4-one (<b>72</b>)</h4><div class="NLM_p last">Using the same <a class="ref internalNav" href="#sec4_3_47" aria-label="procedure">procedure</a> as for <b>54</b>, this compound was obtained from ethyl isothiocyanate (1.31 g, 15.0 mmol, 1.0 equiv) as yellow oil (2.61 g, 10.8 mmol, 72%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.21 (dd, <i>J</i> = 8.1, 1.5 Hz, 2H), 6.88 (dd, <i>J</i> = 8.2, 1.5 Hz, 2H), 4.02 (d, <i>J</i> = 1.4 Hz, 2H), 3.78 (qd, <i>J</i> = 7.0, 1.4 Hz, 2H), 2.61 (dt, <i>J</i> = 8.4, 7.0 Hz, 2H), 1.22–1.17 (m, 6H); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.65, 154.45, 145.75, 139.53, 128.49, 120.77, 37.35, 32.46, 27.57, 15.54, 12.27; ESI-MS (<i>m</i>/<i>z</i>): 249.11</div></div><div id="sec4_3_66" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> (<i>Z</i>)-2-((4-Ethylphenyl)imino)-3-(2-methoxyethyl)thiazolidin-4-one (<b>73</b>)</h4><div class="NLM_p last">A solution of 4-ethylaniline (2.15 mL, 17.3 mmol, 1.15 equiv) in EtOH (50 mL) was treated with 2-methoxyethyl isothiocyanate (1.76 g, 15.0 mmol, 1.0 equiv) and stirred at 78 °C for 12 h. The solution was then treated with NaOAc (6.23 g, 75.0 mmol, 5.0 equiv) and ethyl chloroacetate (3.22 mL, 30.0 mmol, 2.0 equiv) and stirred at 78 °C for another 5 h. EtOH was evaporated in vacuo to afford the crude title compound as brown solid (4.18 g, 15.0 mmol, quant.). Mp 51.0–51.1 °C. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.09 (d, <i>J</i> = 8.2 Hz, 2H), 6.81 (d, <i>J</i> = 8.2 Hz, 2H), 3.99 (t,  <i>J</i> = 5.6 Hz, 2H), 3.72 (s, 2H), 3.62 (t, <i>J</i> = 5.6 Hz, 2H), 3.29 (s, 3H), 2.55 (q, <i>J</i> = 7.6 Hz, 2H), 1.15 (t, <i>J</i> = 7.6 Hz, 3H); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 171.82, 154.23, 145.33, 140.66, 128.64, 120.95, 68.52, 58.65, 42.09, 32.74, 28.37, 15.60; HR-ESI-MS (<i>m</i>/<i>z</i>): 279.1159, calcd for [M + H]<sup>+</sup> C<sub>14</sub>H <sub>19</sub>O<sub>2</sub>N<sub>2</sub>S: 279.1162.</div></div><div id="sec4_3_67" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> (<i>Z</i>)-3-(Cyclohexylmethyl)-2-((4-ethylphenyl)imino)thiazolidin-4-one (<b>74</b>)</h4><div class="NLM_p last">Using the same <a class="ref internalNav" href="#sec4_3_47" aria-label="procedure">procedure</a> as for <b>54</b>, this compound was obtained from cyclohexylmethyl isothiocyanate (2.33 g, 15.0 mmol, 1.0 equiv) as brown oil (2.80 g, 8.85 mmol, 59%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.10 (d, <i>J</i> = 8.2 Hz, 2H), 6.81 (d, <i>J</i> = 8.2 Hz, 2H), 3.73 (s, 2H), 3.65 (d, <i>J</i> = 7.4 Hz, 2H), 2.57 (q, <i>J</i> = 7.6 Hz, 2H), 1.86 (ddd, <i>J</i> = 11.0, 7.7, 3.5 Hz, 1H), 1.65–1.61 (m, 5H), 1.23–0.92 (m, 8H); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 172.10, 154.70, 145.45, 140.67, 128.65, 120.93, 49.11, 35.88, 32.65, 30.69, 28.39, 26.32, 25.78, 15.64; HR-ESI-MS (<i>m</i>/<i>z</i>): 317.1674, calcd for [M + H]<sup>+</sup> C<sub>18</sub>H<sub>25</sub>ON<sub>2</sub>S: 317.1682.</div></div><div id="sec4_3_68" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> (2<i>Z</i>,5<i>Z</i>)-2-((4-Ethylphenyl)imino)-5-(pyridin-4-ylmethylene)thiazolidin-4-one (<b>75</b>)</h4><div class="NLM_p last">Using the same <a class="ref internalNav" href="#sec4_3_4" aria-label="procedure">procedure</a> as for <b>8</b>, this compound was obtained from <b>71</b> (660 mg, 3.00 mmol, 1.0 equiv) as yellow solid (417 mg, 1.35 mmol, 45%). Mp >300 °C (dec.). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.15 (s, 1H), 8.74 (dd, <i>J</i> = 4.5, 1.7 Hz, 1H), 8.67 (dd, <i>J</i> = 4.5, 1.7 Hz, 1H), 7.71–7.69 (m, 2H), 7.60–7.56 (m, 1H), 7.47 (dd, <i>J</i> = 4.6, 1.7 Hz, 1H), 7.29 (dd, <i>J</i> = 8.6, 7.0 Hz, 2H), 7.02 (d, <i>J</i> = 8.1 Hz, 1H), 2.63 (qd, <i>J</i> = 7.6, 3.3 Hz, 2H), 1.25–1.18 (m, 3H); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 150.51, 150.47, 128.78, 128.38, 127.03, 126.34, 123.98, 123.10, 121.55, 120.82, 27.60, 15.52, 15.44; ESI-MS (<i>m/</i>z): 310.10.</div></div><div id="sec4_3_69" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> (2<i>Z</i>,5<i>Z</i>)-3-Ethyl-2-((4-ethylphenyl)imino)-5-(pyridin-4-ylmethylene)thiazolidin-4-one (<b>76</b>)</h4><div class="NLM_p last">This compound was prepared from <b>72</b> (744 mg, 3.00 mmol, 1.0 equiv) using the same <a class="ref internalNav" href="#sec4_3_4" aria-label="procedure">procedure</a> as for <b>8</b>. The yellow precipitate was further purified by FC (SiO<sub>2</sub>; hexane/EtOAc 2:1) to afford the product as yellow solid (182 mg, 0.54 mmol, 18%). Mp 177.7–178.5 °C. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.59–8.57 (m, 2H), 7.55 (s, 1H), 7.25–7.14 (m, 4H), 6.86 (d, <i>J</i> = 8.2 Hz, 2H), 4.00 (q, <i>J</i> = 7.1 Hz, 2H), 2.61 (q, <i>J</i> = 7.6 Hz, 2H), 1.30 (t, <i>J</i> = 7.1 Hz, 3H), 1.20 (t, <i>J</i> = 7.6 Hz, 3H); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 165.86, 150.03, 148.31, 145.35, 141.50, 141.26, 128.87, 127.91, 126.67, 123.42, 120.95, 38.78, 28.40, 15.54, 12.81; ESI-MS (<i>m/</i>z): 338.13.</div></div><div id="sec4_3_70" class="NLM_sec NLM_sec_level_3"><div id="ac_i90" class="anchor-spacer"></div><h4 class="article-section__title" id="_i90"> (2<i>Z</i>,5<i>Z</i>)-2-((4-Ethylphenyl)imino)-3-(2-methoxyethyl)-5-(pyridin-4-ylmethylene)thiazolidin-4-one (<b>77</b>)</h4><div class="NLM_p last">A solution of <b>73</b> (834 mg, 3.00 mmol, 1.0 equiv) and piperidine (0.65 mL, 6.6 mmol, 2.2 equiv) in EtOH (5 mL) was treated with 4-pyridinecarboxaldehyde (0.56 mL, 6.0 mmol, 2.0 equiv) and heated to 78 °C. After 12 h, the reaction mixture was cooled to 0 °C and filtered, and the filter residue washed with ice-cold EtOH (3 × 5 mL). The yellow solid was further purified by FC (SiO<sub>2</sub>; hexane/EtOAc 2:1) to afford <b>77</b> as yellow solid (361 mg, 0.984 mmol, 32.8%). Mp 112.9–114.0 °C. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.57 (d, <i>J</i> = 5.0 Hz, 2H), 7.55 (s, 1H), 7.23 (d, <i>J</i> = 6.2 Hz, 2H), 7.14 (d, <i>J</i> = 8.2 Hz, 1H), 6.84 (d, <i>J</i> = 8.2 Hz, 2H), 4.14 (t, <i>J</i> = 5.6 Hz, 2H), 3.69 (t, <i>J</i> = 5.6 Hz, 2H), 2.59 (q, <i>J</i> = 7.6 Hz, 3H), 1.19 (t, <i>J</i> = 7.6 Hz, 3H); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 166.10, 150.01, 148.54, 145.22, 141.44, 141.28, 128.86, 127.58, 127.01, 123.42, 120.93, 68.61, 58.72, 42.46, 28.39, 15.55; HR-ESI-MS (<i>m</i>/<i>z</i>): 368.1433, calcd for [M + H]<sup>+</sup> C<sub>20</sub>H<sub>22</sub>O<sub>2</sub>N<sub>3</sub>S: 368.1429.</div></div><div id="sec4_3_71" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91"> (2<i>Z</i>,5<i>Z</i>)-3-(Cyclohexylmethyl)-2-((4-ethylphenyl)imino)-5-(pyridin-4-ylmethylene)thiazolidin-4-one (<b>78</b>)</h4><div class="NLM_p last">Using the same <a class="ref internalNav" href="#sec4_3_4" aria-label="procedure">procedure</a> as for <b>8</b>, this compound was obtained from <b>74</b> (948 mg, 3.00 mmol, 1.0 equiv) as yellow solid (767 mg, 1.89 mmol, 63%). Mp 147.8–148.7 °C. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.57 (d, <i>J</i> = 5.3 Hz, 2H), 7.53 (s, 0H), 7.22 (d, <i>J</i> = 5.1 Hz, 1H), 7.15 (d, <i>J</i> = 7.9 Hz, 4H), 6.84 (d, <i>J</i> = 8.0 Hz, 2H), 3.79 (d, <i>J</i> = 7.4 Hz, 12), 2.60 (q, <i>J</i> = 7.6 Hz, 1H), 1.92 (dd, <i>J</i> = 13.3, 7.6 Hz, 5H), 1.67–1.62 (m, 2H), 1.22–0.99 (m, 8H); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 166.44, 150.30, 148.92, 145.45, 141.25, 141.19, 128.85, 127.51, 126.83, 123.35, 120.89, 49.44, 36.20, 30.73, 28.42, 26.30, 25.77, 15.60; HR-ESI-MS (<i>m</i>/<i>z</i>): 406.1941, calcd for [M + H]<sup>+</sup> C<sub>24</sub>H<sub>28</sub>ON<sub>3</sub>S: 406.1948.</div></div><div id="sec4_3_72" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92"> (2<i>Z</i>,5<i>Z</i>)-5-((2-Aminopyridin-4-yl)methylene)-2-((4-ethylphenyl)imino)-3-methylthiazolidin-4-one (<b>79</b>)</h4><div class="NLM_p last">A solution of 4-thiazolidinone <b>54</b> (2.69 g, 11.5 mmol, 1.0 equiv) and piperidine (3.90 mL, 39.6 mmol, 3.44 equiv) in EtOH (6 mL) was treated with <i>tert</i>-butyl(4-formylpyridin-2-yl)carbamate (3.85 g, 17.3 mmol, 1.5 equiv) and stirred at 75 °C for 15 h. The solution was cooled to 0 °C and filtered. The yellow solid on the filter was washed with ice-cold EtOH (2 × 20 mL) and then added to TFA (6 mL). After 4 h, the solution was diluted with H<sub>2</sub>O (100 mL) and CH<sub>2</sub>Cl<sub>2</sub> (100 mL) and neutralized with NaOH (1 M) to pH = 7. The organic phase was washed with H<sub>2</sub>O (3 × 20 mL), dried over MgSO<sub>4</sub>·2H<sub>2</sub>O, and evaporated in vacuo to afford <b>79</b> as a yellow solid (2.45 g, 7.25 mmol, 63.1%). Mp 191.2–199.7 °C. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.96 (d, <i>J</i> = 5.6 Hz, 1H), 7.52 (s, 1H), 7.25 (d, <i>J</i> = 8.3 Hz, 2H), 6.94 (d, <i>J</i> = 8.3 Hz, 2H), 6.65 (dd, <i>J</i> = 5.8, 1.6 Hz, 1H), 6.59 (d, <i>J</i> = 1.5 Hz, 1H), 3.33 (s, 3H), 2.62 (q, <i>J</i> = 7.6 Hz, 2H), 1.20 (t, <i>J</i> = 7.6 Hz, 3H); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.45, 158.85, 149.19, 145.86, 145.13, 142.69, 140.30, 128.78, 127.36, 126.35, 120.86, 111.78, 108.14, 29.72, 27.57, 15.41; HR-ESI-MS (<i>m</i>/<i>z</i>): 339.1271, calcd for [M + H]<sup>+</sup> C<sub>18</sub>H<sub>19</sub>ON<sub>4</sub>S: 339.1274.</div></div><div id="sec4_3_73" class="NLM_sec NLM_sec_level_3"><div id="ac_i93" class="anchor-spacer"></div><h4 class="article-section__title" id="_i93"> 3-Amino-<i>N</i>-(4-((<i>Z</i>)-((<i>Z</i>)-2-((4-ethylphenyl)imino)-3-methyl-4-oxothiazolidin-5-ylidene)methyl)pyridin-2-yl)propanamide (TFA salt) (<b>80</b>)</h4><div class="NLM_p last">A solution of <b>79</b> (100 mg, 0.296 mmol, 1.0 equiv), EDC·HCl (170 mg, 0.888 mmol, 3.0 equiv), and pyridine (71 μL, 0.888 mmol, 3.0 equiv) in DMF (2.0 mL) was treated with <i>N</i>-(<i>tert</i>-butoxycarbonyl)-3-aminopropionic acid (84 mg, 0.444 mmol, 1.5 equiv) and stirred overnight. Then, the solution was treated with EtOAc (10 mL), washed with H<sub>2</sub>O (3 × 10 mL), and evaporated in vacuo. The crude was redissolved in TFA/THF (1:1) and stirred for 5 h. The mixture was then diluted with H<sub>2</sub>O and neutralized with NaOH (1 M) to pH = 7. The organic phase was washed with H<sub>2</sub>O (3 × 20 mL), dried over MgSO<sub>4</sub>·2H<sub>2</sub>O, and evaporated in vacuo. The crude was purified by RP-HPLC (A(70%)/D(30%) to A(45%)/D(55%), 40 min gradient), and <b>80</b> was obtained as yellow solid after lyophilization (7.7 mg, 0.015 mmol, 5.1%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.86 (s, 1H), 8.41 (d, <i>J</i> = 5.3 Hz, 1H), 8.20 (s, 1H), 7.76 (s, 3H), 7.65 (s, 1H), 7.27–7.21 (m, 3H), 6.96 (d, <i>J</i> = 8.3 Hz, 2H), 3.35 (s, 3H), 3.07 (q, <i>J</i> = 6.2 Hz, 2H), 2.77 (t, <i>J</i> = 6.7 Hz, 2H), 2.63 (q, <i>J</i> = 7.6 Hz, 2H), 1.21 (t, <i>J</i> = 7.6 Hz, 3H); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 169.46, 165.39, 152.52, 149.06, 145.05, 142.31, 140.41, 128.74, 127.03, 126.80, 120.89, 119.12, 112.44, 34.61, 33.34, 29.77, 27.59, 15.50; HR-ESI-MS (<i>m</i>/<i>z</i>): 410.1638, calcd for [M + H]<sup>+</sup> C<sub>21</sub>H<sub>24</sub>O<sub>2</sub>N<sub>2</sub>S: 410.1645.</div></div><div id="sec4_3_74" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> 3-((4-((<i>Z</i>)-((<i>Z</i>)-2-((4-Ethylphenyl)imino)-3-methyl-4-oxothiazolidin-5-ylidene)methyl)pyridin-2-yl)amino)-3-oxopropanoic acid (<b>81</b>)</h4><div class="NLM_p last">Using the same <a class="ref internalNav" href="#sec4_3_73" aria-label="procedure">procedure</a> as for <b>80</b>, this compound was obtained from mono-<i>tert</i>-butyl malonate (70 mg, 0.44 mmol, 1.5 equiv) as yellow solid (37 mg, 0.086 mmol, 29%). Mp 116.1–116.2 °C. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.82 (s, 1H), 8.41 (d, <i>J</i> = 5.1 Hz, 1H), 8.21 (s, 1H), 7.66 (s, 1H), 7.26 (d, <i>J</i> = 8.3 Hz, 2H), 7.22 (dd, <i>J</i> = 5.3, 1.7 Hz, 1H), 6.96 (d, <i>J</i> = 8.2 Hz, 2H), 3.47 (s, 2H), 3.36 (s, 3H), 2.64 (q, <i>J</i> = 7.5 Hz, 2H), 1.22 (t, <i>J</i> = 7.6 Hz, 3H); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 169.11, 165.71, 165.36, 152.45, 149.01, 148.89, 145.01, 142.42, 140.35, 128.75, 127.04, 126.82, 120.85, 119.12, 112.35, 43.60, 29.75, 27.59, 15.44; HR-ESI-MS (<i>m</i>/<i>z</i>): 425.1274, calcd for [M + H]<sup>+</sup> C<sub>21</sub>H<sub>21</sub>O<sub>4</sub>N<sub>4</sub>S: 425.1278.</div></div><div id="sec4_3_75" class="NLM_sec NLM_sec_level_3"><div id="ac_i95" class="anchor-spacer"></div><h4 class="article-section__title" id="_i95"> 3-((4-((<i>Z</i>)-((<i>Z</i>)-2-((4-Ethylphenyl)imino)-3-methyl-4-oxothiazolidin-5-ylidene)methyl)pyridin-2-yl)amino)-4-oxobutanoic acid (<b>82</b>)</h4><div class="NLM_p last">Succinic anhydride (443 mg, 4.43 mmol, 10 equiv) was added to a solution of amine <b>79</b> (150 mg, 0.443 mmol, 1 equiv) and pyridine (700 μL, 8.86 mmol, 20 equiv) in DMF (0.9 mL). The mixture was stirred for 12 h, and then EtOAc (40 mL) was added. The organic phase was washed with 1 M HCl (40 mL), brine (2 × 40 mL), and H<sub>2</sub>O (20 mL). The solvent was evaporated in vacuo, and the crude was purified by RP-HPLC to afford the product as yellow solid after lyophylization (5.8 mg, 0.013 mmol, 3.1%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.69 (d, <i>J</i> = 1.8 Hz, 1H), 8.40 (dd, <i>J</i> = 5.3, 1.7 Hz, 1H), 8.24 (s, 1H), 7.66 (d, <i>J</i> = 1.7 Hz, 1H), 7.28 (dd, <i>J</i> = 8.4, 1.9 Hz, 2H), 7.20 (d, <i>J</i> = 5.3 Hz, 1H), 6.98 (dd, <i>J</i> = 8.4, 1.8 Hz, 2H), 3.37 (d, <i>J</i> = 1.8 Hz, 3H), 2.64 (dd, <i>J</i> = 4.8, 2.3 Hz, 4H), 1.23 (td, <i>J</i> = 7.6, 1.7 Hz, 3H); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.66, 171.30, 165.40, 152.79, 149.23, 148.88, 145.07, 142.24, 140.41, 128.73, 127.11, 126.71, 120.90, 118.66, 112.39, 30.89, 29.73, 28.42, 27.59, 15.46; HR-ESI-MS (<i>m</i>/<i>z</i>): 439.1431, calcd for [M + H]<sup>+</sup> C<sub>22</sub>H<sub>23</sub>O<sub>4</sub>N<sub>4</sub>S: 439.1435.</div></div><div id="sec4_3_76" class="NLM_sec NLM_sec_level_3"><div id="ac_i96" class="anchor-spacer"></div><h4 class="article-section__title" id="_i96"> 3-((4-((<i>Z</i>)-((<i>Z</i>)-2-((4-Ethylphenyl)imino)-3-methyl-4-oxothiazolidin-5-ylidene)methyl)pyridin-2-yl)amino)-5-oxopentanoic acid (<b>83</b>)</h4><div class="NLM_p last">Glutaric anhydride (506 mg, 4.44 mmol, 15 equiv) was added to a solution of amine <b>79</b> (100 mg, 0.296 mmol, 1 equiv) and pyridine (239 μL, 2.96 mmol, 10 equiv) in DMF (2.0 mL). The solution was heated to 100 °C for 1 h. The yellow precipitate was filtered and washed with EtOAc (2 × 30 mL) and H<sub>2</sub>O (30 mL). The product was dried on high vacuum overnight to afford 90 mg (0.20 mmol, 67%) of the product as yellow solid. Mp 266.8–273.9 °C.  <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.09 (s, 1H), 10.62 (s, 1H), 8.39 (d, <i>J</i> = 5.2 Hz, 1H), 8.27 (d, <i>J</i> = 1.8 Hz, 1H), 7.66 (s, 1H), 7.27 (d, <i>J</i> = 8.3 Hz, 2H), 7.20 (dd, <i>J</i> = 5.3, 1.6 Hz, 1H), 6.98 (d, <i>J</i> = 8.2 Hz, 2H), 3.37 (s, 3H), 2.64 (q, <i>J</i> = 7.6 Hz, 2H), 2.44 (t, <i>J</i> = 7.3 Hz, 2H), 2.27 (t, <i>J</i> = 7.4 Hz, 2H), 1.79 (p, <i>J</i> = 7.4 Hz, 2H), 1.22 (t, <i>J</i> = 7.6 Hz, 3H); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 174.02, 171.90, 165.41, 152.82, 149.27, 148.91, 145.09, 142.16, 140.41, 128.70, 127.13, 126.67, 120.90, 118.77, 112.43, 35.04, 32.91, 29.72, 27.58, 20.10, 15.46; HR-ESI-MS (<i>m</i>/<i>z</i>): 453.1594, calcd for [M + H]<sup>+</sup> C<sub>23</sub>H <sub>25</sub>O<sub>4</sub>N<sub>4</sub>S: 453.1591.</div></div><div id="sec4_3_77" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> (2<i>Z</i>,5<i>Z</i>)-2-((4-Ethylphenyl)imino)-3-methyl-5-((2-(phenylamino)pyridin-4-yl)methylene)thiazolidin-4-one (<b>84</b>)</h4><div class="NLM_p last">Degassed dioxane (1.2 mL) was added to a flask containing amine <b>79</b> (100 mg, 0.296 mmol, 1.0 equiv), Pd<sub>2</sub>(dba)<sub>3</sub> (5.1 mg, 0.0088 mmol, 0.03 equiv), Xantphos (10 mg, 0.018 mmol, 0.06 equiv), Cs<sub>2</sub>CO<sub>3</sub> (116 mg, 0.355 mmol, 1.2 equiv), and phenyl bromide (51 mg, 0.33 mmol, 1.1 equiv) under argon. The suspension was degassed and refilled with argon (5×) and then heated to 100 °C. The reaction mixture was stirred overnight at 100 °C, then cooled to room temperature, filtered, and concentrated in vacuo. The crude was purified by FC (SiO<sub>2</sub>; DCM, 0 to 2% MeOH) to afford <b>84</b> as a yellow solid (43 mg, 0.104 mmol, 35%). Mp 181–182 °C. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.09 (d, <i>J</i> = 5.4 Hz, 1H), 7.46 (s, 1H), 7.19–7.15 (m, 7H), 7.01–6.96 (m, 1H), 6.86–6.80 (m, 3H), 6.67 (dd, <i>J</i> = 5.4, 1.6 Hz, 1H), 3.38 (s, 3H), 2.65 (q, <i>J</i> = 7.5 Hz, 2H), 1.23 (t, <i>J</i> = 7.6 Hz, 3H); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 165.16, 155.39, 148.36, 146.90, 144.35, 142.25, 140.13, 138.25, 128.49, 128.45, 127.85, 127.80, 126.57, 126.14, 122.79, 120.15, 119.94, 113.60, 106.55, 28.86, 27.37, 14.50; HR-ESI-MS (<i>m</i>/<i>z</i>): 415.1581, calcd for [M + H]<sup>+</sup> C<sub>24</sub>H<sub>23</sub>ON<sub>4</sub>S: 415.1587.</div></div><div id="sec4_3_78" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> (2<i>Z</i>,5<i>Z</i>)-2-((4-Ethylphenyl)imino)-3-methyl-5-((2-(pyridin-2-ylamino)pyridin-4-yl)methylene)thiazolidin-4-one (<b>85</b>)</h4><div class="NLM_p last">Using the same <a class="ref internalNav" href="#sec4_3_77" aria-label="procedure">procedure</a> as for <b>84</b> and after purification by RP-HPLC (A(70%)/D(30%) to A(45%)/D(55%), 40 min gradient), this compound was obtained from 2-bromopyridine (52 mg, 0.33 mmol, 1.1 equiv) as yellow solid after lyophilization (6.0 mg, 0.015 mmol, 4.9%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.39 (d, <i>J</i> = 5.4 Hz, 1H), 8.05 (s, 1H), 7.90 (s, 12), 7.72 (s, 1H), 7.36–7.30 (m, 3H), 7.24 (dd, <i>J</i> = 5.5, 1.7 Hz, 1H), 7.05–6.99 (m, 3H), 3.38 (s, 3H), 2.68 (q, <i>J</i> = 7.5 Hz, 2H), 1.25 (t, <i>J</i> = 7.5 Hz, 3H); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.52, 149.45, 145.34, 140.36, 128.77, 127.92, 126.61, 120.87, 117.68, 116.47, 113.33, 110.71, 69.76, 29.75, 27.58, 15.44.); HR-ESI-MS (<i>m</i>/<i>z</i>): 416.1531, calcd for [M + H]<sup>+</sup> C<sub>23</sub>H<sub>22</sub>ON<sub>5</sub>S: 416.1540.</div></div><div id="sec4_3_79" class="NLM_sec NLM_sec_level_3"><div id="ac_i99" class="anchor-spacer"></div><h4 class="article-section__title" id="_i99"> (2<i>Z</i>,5<i>Z</i>)-2-((4-Ethylphenyl)imino)-5-((2-((4-(hydroxymethyl)phenyl)amino)pyridin-4-yl)methylene)-3-methylthiazolidin-4-one (<b>86</b>)</h4><div class="NLM_p last">Using the same <a class="ref internalNav" href="#sec4_3_73" aria-label="procedure">procedure</a> as for <b>84</b> and after purification by RP-HPLC (A(70%)/D(30%) to A(45%)/D(55%), 40 min gradient), this compound was obtained from 4-hydroxymethyl-1-bromobenzene (62 mg, 0.33 mmol, 1.1 equiv) as yellow solid after lyophilization (3.0 mg, 0.0070 mmol, 2.3%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.43 (s, 1H), 8.19 (d, <i>J</i> = 5.4 Hz, 1H), 7.62 (s, 1H), 7.55 (d, <i>J</i> = 8.5 Hz, 2H), 7.29 (d, <i>J</i> = 8.3 Hz, 2H), 7.22 (d, <i>J</i> = 8.5 Hz, 2H), 6.99 (d, <i>J</i> = 8.3 Hz, 2H), 6.90–6.87 (m, 2H), 4.44 (s, 2H), 3.37 (s, 3H), 2.66 (q, <i>J</i> = 7.6 Hz, 2H), 1.24 (t, <i>J</i> = 7.6 Hz, 3H); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.45, 155.98, 149.13, 147.32, 145.10, 142.06, 140.33, 139.21, 135.66, 128.79, 127.40, 127.14, 126.23, 120.86, 118.71, 113.86, 109.43, 62.73, 29.74, 27.56, 15.40; HR-ESI-MS (<i>m</i>/<i>z</i>): 445.1685, calcd for [M + H]<sup>+</sup> C<sub>25</sub>H<sub>25</sub>O<sub>2</sub>N<sub>4</sub>S: 445.1693.</div></div><div id="sec4_3_80" class="NLM_sec NLM_sec_level_3"><div id="ac_i100" class="anchor-spacer"></div><h4 class="article-section__title" id="_i100"> 4-((4-((<i>Z</i>)-((<i>Z</i>)-2-((4-Ethylphenyl)imino)-3-methyl-4-oxothiazolidin-5-ylidene)methyl)pyridin-2-yl)amino)benzoic acid (<b>87</b>)</h4><div class="NLM_p last">Using the same <a class="ref internalNav" href="#sec4_3_73" aria-label="procedure">procedure</a> as for <b>84</b> and after purification by RP-HPLC (A(70%)/D(30%) to A(45%)/D(55%), 40 min gradient), this compound was obtained from 4-bromobenzoic acid (66 mg, 0.33 mmol, 1.1 equiv) as yellow solid after lyophilization (15 mg, 0.033 mmol, 11%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.74 (s, 1H), 8.31 (d, <i>J</i> = 6.0 Hz, 1H), 7.87–7.84 (m, 2H), 7.79–7.76 (m, 2H), 7.64 (s, 1H), 7.29 (d, <i>J</i> = 8.3 Hz, 2H), 7.00–6.97 (m, 4H), 3.37 (s, 3H), 2.65 (q, <i>J</i> = 7.6 Hz, 2H), 1.23 (t, <i>J</i> = 7.6 Hz, 3H); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.07, 165.46, 155.68, 149.08, 148.31, 145.43, 145.09, 141.79, 140.34, 130.43, 128.80, 127.35, 126.15, 122.23, 120.86, 116.86, 115.31, 110.23, 29.76, 27.55, 15.40; HR-ESI-MS (<i>m</i>/<i>z</i>): 459.1479, calcd for [M + H]<sup>+</sup> C<sub>25</sub>H<sub>23</sub>O<sub>3</sub>N<sub>4</sub>S: 459.1485.</div></div><div id="sec4_3_81" class="NLM_sec NLM_sec_level_3"><div id="ac_i101" class="anchor-spacer"></div><h4 class="article-section__title" id="_i101"> (2<i>Z</i>,5<i>Z</i>)-5-((2-((4-(1<i>H</i>-Tetrazol-5-yl)phenyl)amino)pyridin-4-yl)methylene)-2-((4-ethylphenyl)imino)-3-methylthiazolidin-4-one (<b>88</b>)</h4><div class="NLM_p last">Degassed dioxane (1.2 mL) was added to a flask containing amine <b>79</b> (100 mg, 0.296 mmol, 1.0 equiv), Pd<sub>2</sub>(dba)<sub>3</sub> (5.1 mg, 0.0088 mmol, 0.03 equiv), Xantphos (10 mg, 0.018 mmol, 0.06 equiv), Cs<sub>2</sub>CO<sub>3</sub> (116 mg, 0.355 mmol, 1.2 equiv), and 5-(4-bromophenyl)-1H-tetrazole (73.4 mg, 0.326 mmol, 1.1 equiv) under argon. The suspension was degassed, refilled with argon (5×), and then heated to 100 °C. The reaction mixture was stirred overnight at 100 °C, then cooled to room temperature, filtered, and concentrated in vacuo. The crude was purified by RP-HPLC (A/D 70:30 to A/D 45:55, 40 min gradient) to afford the title compound as yellow solid after lyophilization (12 mg, 0.024 mmol, 8.2%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.72 (s, 1H), 8.30–8.29 (m, 1H), 7.95–7.92 (m, 2H), 7.89–7.87 (m, 2H), 7.62 (s, 1H), 7.27 (d, <i>J</i> = 8.3 Hz, 2H), 6.98–6.94 (m, 4H), 2.62 (q, <i>J</i> = 7.6 Hz, 2H), 1.21 (t, <i>J</i> = 7.6 Hz, 3H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.45, 155.70, 149.07, 148.18, 145.08, 143.91, 141.81, 140.31, 128.79, 127.67, 127.34, 126.13, 120.85, 118.03, 115.11, 110.23, 29.74, 27.53, 15.38; HR-ESI-MS (<i>m</i>/<i>z</i>): 483.1712, calcd for [M + H]<sup>+</sup> C<sub>25</sub>H<sub>23</sub>ON<sub>8</sub>S: 483.1710.</div></div><div id="sec4_3_82" class="NLM_sec NLM_sec_level_3"><div id="ac_i102" class="anchor-spacer"></div><h4 class="article-section__title" id="_i102"> (<i>Z</i>)- and (<i>E</i>)-1-(2-Aminopyridin-4-yl)-5-((4-ethylphenyl)imino)-6-methyl-4-thia-6-azaspiro[2.4]heptan-7-one (<i>rac</i>-<b>89</b> and <i>rac</i>-<b>90</b>)</h4><div class="NLM_p">NaH (190 mg, 4.72 mmol, 4.0 equiv) was added to a solution of Me<sub>3</sub>SOI (520 mg, 2.36 mmol, 2.0 equiv) in DMF (12 mL) at 0 °C. The reaction mixture was stirred for 15 min at 0 °C. Then, amine <b>79</b> (400 mg, 1.18 mmol, 1.0 equiv) dissolved in DMF (12 mL) was added to the reaction mixture at 0 °C. After 50 min, EtOAc (20 mL) and H<sub>2</sub>O (20 mL) were added, and the pH of the aqueous phase was adjusted to pH = 8 using 1 M HCl. The organic phase was separated, washed with H<sub>2</sub>O (3 × 20 mL), and evaporated in vacuo. The crude was purified by FC (SiO<sub>2</sub>; CH<sub>2</sub>Cl<sub>2</sub>, 0–3% MeOH) to give a major isomer as off-white foam (156 mg, 0.437 mmol, 37.0%) and a minor isomer as a colorless oil. The minor isomer was further purified by RP-HPLC to afford 5.0 mg (0.014 mmol, 1.3%) of a white solid. The (<i>E</i>) or (<i>Z</i>) configuration of the two isomers could not be assigned.</div><div id="sec4_3_82_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i103" class="anchor-spacer"></div><h5 class="article-section__title" id="_i103"> Major Isomer</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.90 (d, <i>J</i> = 6.1 Hz, 1H), 7.14 (d, <i>J</i> = 8.3 Hz, 2H), 6.83 (d, <i>J</i> = 8.3 Hz, 2H), 6.33 (dd, <i>J</i> = 5.6, 1.6 Hz, 1H), 6.28 (s, 1H), 5.01 (br s, 1H), 3.36 (s, 3H), 2.96 (dd, <i>J</i> = 9.5, 7.7 Hz, 1H), 2.61 (q, <i>J</i> = 7.6 Hz, 2H), 2.16 (dd, <i>J</i> = 9.5, 6.0 Hz, 1H), 1.66 (dd, <i>J</i> = 7.7, 6.0 Hz, 1H), 1.22 (t, <i>J</i> = 7.6 Hz, 3H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 173.46, 157.78, 152.30, 147.70, 145.83, 145.55, 140.69, 128.72, 128.66, 120.77, 120.74, 112.90, 108.21, 38.09, 30.90, 29.95, 28.29, 20.48, 15.44; HR-ESI-MS (<i>m</i>/<i>z</i>): 353.1425, calcd for [M + H]<sup>+</sup> C<sub>19</sub>H<sub>21</sub>ON<sub>4</sub>S: 353.1431.</div></div><div id="sec4_3_82_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i104" class="anchor-spacer"></div><h5 class="article-section__title" id="_i104"> Minor Isomer</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.63 (s, 1H), 7.13 (d, <i>J</i> = 6.4 Hz, 2H), 6.86 (s, 3H), 6.68 (s, 1H), 3.22 (s, 3H), 2.58 (m, <i>J</i> = 7.6 Hz, 3H), 2.29 (s, 1H), 1.83 (s, 1H), 1.18 (t, <i>J</i> = 7.5 Hz, 3H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 169.60, 153.07, 151.21, 150.88, 144.23, 140.24, 133.43, 127.79, 119.84, 113.16, 112.64, 35.00, 33.03, 29.21, 27.34, 19.35, 14.51; HR-ESI-MS (<i>m</i>/<i>z</i>): 353.1427, calcd for [M + H]<sup>+</sup> C<sub>19</sub>H<sub>21</sub>ON<sub>4</sub>S: 353.1431</div></div></div><div id="sec4_3_83" class="NLM_sec NLM_sec_level_3"><div id="ac_i105" class="anchor-spacer"></div><h4 class="article-section__title" id="_i105"> (±)-(<i>Z</i>)-5-((2-Aminopyridin-4-yl)methyl)-2-((4-ethylphenyl)imino)-3-methylthiazolidin-4-one (<i>rac</i>-<b>91</b>)</h4><div class="NLM_p last">A flask containing amine <b>79</b> (150 mg, 0.444 mmol, 1.0 equiv), Pd/C (47 mg) and EtOAc (9 mL) was evacuated and refilled with argon (3×). Then, H<sub>2</sub> was added via a septum inlet. The black suspension was stirred for 12 h and then filtered over Celite. The Celite was washed with EtOAc (3 × 20 mL). EtOAc was evaporated in vacuo, and the remaining crude was purified by FC (SiO<sub>2</sub>; CH<sub>2</sub>Cl<sub>2</sub>, 0 to 5% MeOH) to give the product as colorless oil (95 mg, 0.28 mmol, 63%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.80 (d, <i>J</i> = 5.2 Hz, 1H), 7.18 (d, <i>J</i> = 8.3 Hz, 2H), 6.82 (d, <i>J</i> = 8.2 Hz, 2H), 6.35 (dd, <i>J</i> = 5.3, 1.5 Hz, 1H), 6.25 (s, 1H), 5.90 (s, 2H), 4.75–4.70 (m, 1H), 3.29 (dd, <i>J</i> = 14.2, 4.1 Hz, 1H), 3.18 (s, 3H), 2.85 (dd, <i>J</i> = 14.2, 10.1 Hz, 1H), 2.58 (q, <i>J</i> = 7.6 Hz, 2H), 1.18 (t, <i>J</i> = 7.6 Hz, 3H); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.41, 159.88, 153.71, 147.86, 146.52, 145.67, 139.60, 128.50, 120.71, 112.29, 107.77, 47.97, 37.29, 29.26, 27.54, 15.49; HR-ESI-MS (<i>m</i>/<i>z</i>): 341.1424, calcd for [M + H]<sup>+</sup> C<sub>18</sub>H<sub>21</sub>ON<sub>4</sub>S: 341.1431.</div></div><div id="sec4_3_84" class="NLM_sec NLM_sec_level_3"><div id="ac_i106" class="anchor-spacer"></div><h4 class="article-section__title" id="_i106"> (±)-(<i>Z</i>)-6-((4-((2-((4-Ethylphenyl)imino)-3-methyl-4-oxothiazolidin-5-yl)methyl)pyridin-2-yl)amino)nicotinic acid (<i>rac</i>-<b>92</b>)</h4><div class="NLM_p last">Degassed dioxane (0.6 mL) was added to a flask containing <i>rac</i>-<b>91</b> (50 mg, 0.248 mmol, 1.0 equiv), Pd<sub>2</sub>(dba)<sub>3</sub> (2.6 mg, 0.0044 mmol, 0.03 equiv), Xantphos (5.0 mg, 0.009 mmol, 0.06 equiv), Cs<sub>2</sub>CO<sub>3</sub> (58 mg, 0.178 mmol, 1.2 equiv), and 6-bromonicotinic acid (33 mg, 0.163 mmol, 1.1 equiv) under argon. The suspension was degassed and refilled with argon (5×) and then heated to 100 °C. The reaction mixture was stirred overnight at 100 °C, then cooled to room temperature, filtered, and concentrated in vacuo. The crude was purified by RP-HPLC (A(70%)/D(30%) to A(45%)/D(55%), 40 min gradient) to give the product as off-white solid after lyophilization (18 mg, 0.040 mmol, 16%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.13 (s, 1H), 8.82–8.81 (m, 1H), 8.27 (d, <i>J</i> = 5.7 Hz, 1H), 8.23 (dd, <i>J</i> = 8.8, 2.3 Hz, 1H), 7.54 (d, <i>J</i> = 8.8 Hz, 1H), 7.48 (s, 1H), 7.14 (d, <i>J</i> = 8.3 Hz, 2H), 7.05 (d, <i>J</i> = 5.6 Hz, 1H), 6.79 (d, <i>J</i> = 8.3 Hz, 2H), 4.83 (dd, <i>J</i> = 9.1, 4.6 Hz, 1H), 3.49 (dd, <i>J</i> = 14.4, 4.6 Hz, 1H), 3.23–3.19 (m, 1H), 3.17 (s, 3H), 2.55 (q, <i>J</i> = 7.7 Hz, 2H), 1.15 (t, <i>J</i> = 7.6 Hz, 3H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.13, 165.79, 155.83, 153.44, 151.65, 148.28, 145.57, 139.64, 139.37, 128.46, 120.65, 119.64, 118.17, 113.77, 111.97, 47.38, 37.38, 29.32, 27.49, 15.43; HR-ESI-MS (<i>m</i>/<i>z</i>): 462.1586, calcd for [M + H]<sup>+</sup> C<sub>24</sub>H<sub>24</sub>O<sub>3</sub>N<sub>5</sub>S: 462.1594.</div></div><div id="sec4_3_85" class="NLM_sec NLM_sec_level_3"><div id="ac_i107" class="anchor-spacer"></div><h4 class="article-section__title" id="_i107"> (2<i>Z</i>,5<i>Z</i>)-3-Methyl-2-(phenylimino)-5-(pyridin-4-ylmethylene)thiazolidin-4-one (<b>93</b>)</h4><div class="NLM_p last">This compound was purchased from Princeton Biomolecular (no. OSSL_047658). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.68–8.66 (m, 2H), 7.73 (s, 1H), 7.48–7.42 (m, 2H), 7.25–7.21 (m, 1H), 7.06–7.03 (m, 1H), 3.37 (s, 3H).</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i108" class="anchor-spacer"></div><h3 class="article-section__title" id="_i108"> Aurora A Purification</h3><div class="NLM_p last">The clone of human Aurora A kinase domain (residues 122–403 in the pET24-d vector) was kindly provided by the Montoya laboratory (University of Copenhagen). The construct was transformed into <i>E. coli</i> BL21 (DE3) Rosetta cells, and protein expression was induced with 0.5 mM IPTG at 20 °C for 12 h. The cells were harvested at 6000×<i>g</i> for 25 min at 4 °C and resuspended in lysis buffer (50 mM Tris-HCl, 500 mM NaCl, 1 mM PMSF, pH = 8.0). Disruption of the cells was performed by sonication cooled on ice, after which the debris were removed by centrifugation at 10,000×<i>g</i> for 30 min at 4 °C. Aurora A was purified by affinity chromatography using Ni-NTA resin from Qiagen following the manufacturer’s instructions. After loading, the resin was washed with lysis buffer, followed by a second wash with 6% elution buffer (50 mM Tris-HCl, 500 mM NaCl, 500 mM imidazole, pH = 8.0). Protein was eluted with 100% elution buffer. The eluate was exchanged into final buffer (20 mM Tris-HCl, 200 mM NaCl, 0.5 mM EDTA, 2 mM DTT, pH = 8.0) using a HiPrep 26/10 desalting column (GE Healthcare). The His-tag was cleaved with TEV protease at 4 °C overnight. The tag and minor impurities were removed by a second nickel affinity chromatography step. Aggregated and soluble Aurora A were separated from one another by size exclusion chromatography using a HiLoad 16/60 Superdex 200 column (GE Healthcare) equilibrated in final buffer. Soluble Aurora A was concentrated using Vivaspin-15 concentrators (Sartorius Stedim Biotech). Protein concentration was determined by UV absorbance. Aurora A was flash frozen in liquid nitrogen and stored at −18 °C.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i109" class="anchor-spacer"></div><h3 class="article-section__title" id="_i109"> HTRF Kinase Assay</h3><div class="NLM_p last">Aurora A and Aurora B kinases were assayed using the homogeneous time-resolved fluorescence (HTRF) KinEASE STK2 kit from Cisbio (France). For Aurora A, which was expressed and purified as described above, the enzymatic reaction (total volume 10 μL) was carried out with 3 nM Aurora A kinase domain, 1 μM biotinylated STK2 substrate, 20 μM ATP (∼<i>K</i><sub>m</sub>) in kinase buffer (50 mM HEPES (pH 7.0), 0.02% NaN<sub>3</sub>, 0.1 mM Na<sub>3</sub>VO<sub>4</sub>, 0.01% BSA, 5 mM MgCl<sub>2</sub>, 0.01% Triton X-100, 1 mM DTT), and either the test compound or the DMSO control (final DMSO concentration was 2%). For Aurora B, the enzyme reaction (total volume 10 μL) was carried out with 8 nM Aurora B/INCENP complex (Millipore, no. 14-835), 1 μM biotinylated STK2 substrate, 20 μM ATP (<i>K</i><sub>m</sub> = 26 μM) in kinase buffer (50 mM HEPES (pH 7.0), 0.02% NaN<sub>3</sub>, 0.1 mM Na<sub>3</sub>VO<sub>4</sub>, 0.01% BSA, 5 mM MgCl<sub>2</sub>, 0.01% Triton X-100, 1 mM DTT), and either the test compound or the DMSO control (final DMSO concentration was 2%). For both enzymes, the reactions were run for 30 min at room temperature and stopped by the addition of 10 μL of detection buffer containing EDTA, antiphospho-Ser/Thr antibody labeled with europium cryptate, and XL-665 conjugated streptavidin (62.5 nM final concentration). After incubation at room temperature for 1 h, fluorescence was measured at 620 nm (europium cryptate) and 665 nm (XL-665) after excitation at 317 nm (lag time 60 μs, integration time 500 μs) using a Tecan Infinite M1000 PRO microplate reader (Greiner 384-well plates, white, nonbinding, small volume). The ratio of fluorescence (665 nm/620 nm) was calculated for each well, and the results were expressed as follows: specific signal = ratio (sample) – ratio (negative control), where ratio = 665/620 nm × 10<sup>4</sup>. Compounds were measured in 3-fold serial dilutions at 10 different concentrations, covering the concentration range from no to full inhibition. Each concentration was measured in duplicates, and two independent determinations were made for each IC<sub>50</sub> value. IC<sub>50</sub> curves were generated using a four-parameter logistic model (XLfit from IDBS).</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i110" class="anchor-spacer"></div><h3 class="article-section__title" id="_i110"> Kinome Profiling</h3><div class="NLM_p last">The kinome profiling of compound <b>9</b> was conducted by DiscoveRx. In total, 456 kinases were assayed (scanMAX). For most assays, kinase-tagged T7 phage strains were grown in parallel in 24-well blocks in an <i>E. coli</i> host derived from the BL21 strain. <i>E. coli</i> were grown to log-phase and infected with T7 phage from a frozen stock (multiplicity of infection = 0.4) and incubated with shaking at 32 °C until lysis (90–150 min). The lysates were centrifuged (6000<i>g</i>) and filtered (0.2 μm) to remove cell debris. The remaining kinases were produced in HEK-293 cells and subsequently tagged with DNA for qPCR detection. Streptavidin-coated magnetic beads were treated with biotinylated small molecule ligands for 30 min at room temperature to generate affinity resins for kinase assays. The liganded beads were blocked with excess biotin and washed with blocking buffer (SeaBlock (Pierce), 1% BSA, 0.05% Tween 20, 1 mM DTT) to remove unbound ligand and to reduce nonspecific phage binding. Binding reactions were assembled by combining kinases, liganded affinity beads, and test compounds in 1× binding buffer (20% SeaBlock, 0.17× PBS, 0.05% Tween 20, 6 mM DTT). Test compounds were prepared as 40× stocks in 100% DMSO and directly diluted into the assay. All reactions were performed in polypropylene 384-well plates in a final volume of 0.04 mL. The assay plates were incubated at room temperature with shaking for 1 h, and the affinity beads were washed with wash buffer (1× PBS, 0.05% Tween 20). The beads were then resuspended in elution buffer (1× PBS, 0.05% Tween 20, 0.5 μM nonbiotinylated affinity ligand) and incubated at room temperature with shaking for 30 min. The kinase concentration in the eluates was measured by qPCR. Compound 114 was screened at 1 μM, and results for primary screen binding interactions were reported as % of control (PoC). PoC = (test compound signal – positive control signal)/(DMSO signal – positive control signal). PoC values at 1 μM <b>9</b> for all kinases were visualized using TREEspot (DiscoveRx). For the determination of <i>K</i><sub>D</sub> values, an 11-point 3-fold serial dilution of <b>9</b> was prepared in 100% DMSO at 100× final test concentration and subsequently diluted to 1× in the assay (final DMSO concentration = 1%). <i>K</i><sub>D</sub> values were determined with a standard dose–response curve using the Hill equation.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i111" class="anchor-spacer"></div><h3 class="article-section__title" id="_i111"> Crystallization, Structure Solution, and Refinement</h3><div class="NLM_p">Aurora A kinase domain was concentrated to 150 μM and mixed with MgSO<sub>4</sub> and the corresponding compound (dissolved in DMSO) to a final concentration of 2 mM and 150 μM, respectively. Crystallization experiments were set up after incubation of the complex for 15 min on ice. The crystals were obtained by hanging drop vapor diffusion at 18 °C, mixing 1 μL of sample solution with 1 μL of reservoir solution and equilibrating against 500 μL of reservoir solution. The crystals were flash frozen in liquid nitrogen under cryo-protection. The crystallization solutions found for each complex were: 4% v/v Tacsimate pH 8.0, 12% w/v PEG 3350, (10% MPD added as cryo-protectant) for <b>6</b>/Aurora A; 0.3 M ammonium citrate dibasic, 25% PEG 3350 (10% MPD added as cryo-protectant) for <b>77</b>/Aurora A; 0.1 M Tris pH 8.0, 28% PEG 4K (20% glycerol added as cryo-protectant) for <b>9</b>/Aurora A; and 5% MPD, 0.1 M Hepes pH 7.5, 10% PEG 10K (20% MPD added as cryo-protectant) for <b>88</b>/Aurora A.</div><div class="NLM_p last">Data were collected on beamline X06DA at the Swiss Light Source, Paul Scherrer Institut, Villigen, Switzerland, except the data set for the <b>77</b>/Aurora A complex that was collected on beamline X06SA, also at the Swiss Light Source. The wavelength was 1 Å and the temperature 100 K. The data were integrated and scaled with MOSFLM/SCALA<a onclick="showRef(event, 'ref79 ref80'); return false;" href="javascript:void(0);" class="ref ref79 ref80">(79, 80)</a> or XDS<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> (crystallographic data statistics are shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_001.pdf" class="ext-link">Supporting Information</a>). The structures were solved by molecular replacement using Phaser in CCP4<a onclick="showRef(event, 'ref82'); return false;" href="javascript:void(0);" class="ref ref82">(82)</a> and PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1OL5">1OL5</a> without ligand as template. All complexes crystallized in the <i>P</i>6<sub>1</sub>22 space group with one molecule per asymmetric unit. A first round of rigid body refinement was performed, followed by simulated annealing<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a> and initial electron density maps showed the presence of the compounds. These were fitted using LigandFit in Phenix.<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a> Several rounds of refinement (Phenix and Refmac5<a onclick="showRef(event, 'ref82'); return false;" href="javascript:void(0);" class="ref ref82">(82)</a> (using TLS) and model building (Coot)<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a> were subsequently carried out until convergence was reached. The final models have no Ramachandran outliers (except one residue in the <b>6</b>/Aurora A complex structure), and they lack 8 and 15 residues at the N- and C-terminus, respectively. In addition, in the complex structures of Aurora A with <b>77</b>, <b>9</b>, and <b>88</b>, 13 residues in the activation loop were not visible (residues 279–291). In the <b>6</b>/Aurora A structure, 5 residues were not modeled (residues 287–291). The figures were compiled using Pymol<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a> and Ligplot.<a onclick="showRef(event, 'ref86'); return false;" href="javascript:void(0);" class="ref ref86">(86)</a></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i112" class="anchor-spacer"></div><h3 class="article-section__title" id="_i112"> TPX2 Purification</h3><div class="NLM_p last">The clone of human TPX2 N-terminal domain (residues 1–43), preceded by GST and a TEV cleavage site, was a generous gift from the Conti laboratory (MPI, Martinsried). The protein was expressed in <i>E. coli</i> Bl21 (DE3) cells using 0.1 mM IPTG at 20 °C for 12 h. Cells were broken, and the soluble fraction was isolated as explained above. TPX2 was purified using a GSTrap (GE Healthcare) equilibrated in 50 mM Tris, 150 mM NaCl, pH = 7.5. GST-TPX2 was eluted using final buffer plus 10 mM reduced GSH. The GST-tag was cleaved with TEV protease at 4 °C overnight. TPX2 (1–43) was isolated in a final step of size exclusion chromatography using a HiLoad 16/60 Superdex 75 column (GE Healthcare) equilibrated in final buffer. Protein concentration was determined by UV absorbance. TPX2 was flash frozen in liquid nitrogen and stored at −18 °C.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i113" class="anchor-spacer"></div><h3 class="article-section__title" id="_i113"> Microscale Thermophoresis</h3><div class="NLM_p last">TPX2 was labeled using the Monolith NT.115 protein labeling kit RED (NanoTemper Technologies). The labeling was performed according to the manufacturer’s instructions in the supplied labeling buffer applying a concentration of 44 μM peptide at room temperature for 30 min. Labeled TPX2 was adjusted to 400 nM with final buffer and 0.05% Tween 20 (NanoTemper Technologies). Aurora A kinase with or without ligand was dissolved in final buffer supplemented with 0.05% Tween 20, and a series of 12 dilutions (1:1) were prepared in the identical buffer, keeping the DMSO/ligand concentration constant (final concentrations 0.5% and 125 μM, respectively). Final protein concentrations were between 4.4 nM and 150 μM. For the thermophoresis experiment, each protein dilution was mixed with one volume of labeled TPX2, which led to a final concentration of 200 nM for fluorescently labeled TPX2, 125 μM for the ligand and 2.2 nM to 75 μM for Aurora A. After 30 min incubation at 4 °C and centrifugation at 9600x<i>g</i> for 2 min, the solution was filled into Monolith NT hydrophilic capillaries (NanoTemper Technologies). Thermophoresis was measured at room temperature with 5 s/30 s/5 s laser off/on/off times. Instrument parameters were adjusted to 15% LED intensity and 80% microscale thermophoresis (MST) power. Data of three independent measurements were analyzed (NT.Analysis software, version 1.5.41, NanoTemper Technologies) using the thermophoresis signal after 10 s. Points measured above 4.7 μM Aurora A were rejected.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i114" class="anchor-spacer"></div><h3 class="article-section__title" id="_i114"> Cell Culture Experiments</h3><div class="NLM_p last">HeLa Kyoto cells were cultured in high-glucose DMEM with GlutaMAX (Life Technologies) media supplemented with 10% fetal calf serum (FCS) in a humidified 5% CO<sub>2</sub> incubator at 37 °C. For plasmid transfections, cells were seeded at 80–90% confluence. Four μg of plasmid DNA in 100 μL OptiMEM and 4 μL of Lipofectamine 2000 (Life Technologies) in 100 μL OptiMEM were incubated in parallel for 5 min, mixed for 20 min, and added to each well. All Aurora A clones were constructed using full-length Aurora A as a template with appropriate PCR primer pairs. The amplified products were subcloned into a pcDNA3-GFP vector.<a onclick="showRef(event, 'ref87'); return false;" href="javascript:void(0);" class="ref ref87">(87)</a> GFP-Aurora A<sub>R137A</sub> and GFP-Aurora A<sub>W277A</sub> were engineered using a site-directed mutagenesis kit (Agilent Technologies, no. 210515) with appropriate primers. For determining the mitotic index and the chromosome misalignment phenotype, cells were incubated with various concentrations of compounds for 20 h before analysis. Cells were likewise treated with 0.25 μM MLN8237 (Selleckchem, no. S1133) or 5 μM ZM447439 (Selleckchem, no. S1103). For flow cytometry analysis, 50 nM MLN8037 and 2.5 μM ZM447439 were used. For rescue experiments, HeLa Kyoto cells were first transfected with GFP-Aurora A, GFP-Aurora A<sub>R137A</sub> or GFP-Aurora A<sub>W277A</sub> for 20 h, followed by incubation with <b>9</b> for another 20 h before cells were fixed with cold methanol and stained. To assess the impact of compound <b>9</b> on cell cycle progression, DNA content was analyzed after propidium iodide staining using FACS (BD Accuri C6 flow cytometer).</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i115" class="anchor-spacer"></div><h3 class="article-section__title" id="_i115"> Indirect Immunofluorescence</h3><div class="NLM_p last">For immunofluorescence, cells were fixed in cold methanol for 7 min, washed in PBT (PBS supplemented with 0.05% Tween 20) and stained with the following primary antibodies for 1 to 4 h at room temperature: 1:200 rabbit anti-pT288 (Cell Signaling, C39D8), 1:200 rabbit anti-pH3S10 (Cell Signaling, D2C8), 1:500 mouse anti-α-tubulin (Transduction Laboratories, 612709), 1:300 mouse anti-GFP (MAB3580, Millipore), 1:200 anti-TPX2 (Santa Cruz Biotechnology, sc-32863), anti-Aurora A (Invitrogen, 458900). Secondary antibodies were antimouse conjugated to Alexa488 or Alexa568 as well as antirabbit conjugated to Alexa488 or Alexa568, all used at 1:500 (Invitrogen). Confocal images were acquired on a Zeiss LSM 710 confocal microscope equipped with a Axiocam MRm (B/W) CCD camera using a 63× NA 1.0 oil objective and processed in ImageJ and Adobe Photoshop, maintaining relative image intensities within a series.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-6"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i116"><a href="/doi/suppl/10.1021/acs.jmedchem.6b00709">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67847" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67847" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b00709" class="ext-link">10.1021/acs.jmedchem.6b00709</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Structures of references inhibitors used for xLOS virtual screening (Supplementary Figure 1), first round hits (Supplementary Figure 2) and second round hits (Supplementary Figure 3), ORTEP diagrams of <b>76</b> and <b>78</b> (Supplementary Figure 4), X-ray data collection table (Supplementary Table 1), Trp fluorescence data (Supplementary Figure 5), analysis of Aurora A conformations (Supplementary Figure 6), compound purities (Supplementary Table 2), and images of <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra for final compounds and selected intermediates (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">(<a href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_002.txt" class="ext-link">TXT</a>)</p></li><li><p class="inline">(<a href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_003.txt" class="ext-link">TXT</a>)</p></li><li><p class="inline">SMILES files of kinase screening set and second round LBVS annotated with PAINS flags (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_004.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_001.pdf">jm6b00709_si_001.pdf (9.0 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_002.txt">jm6b00709_si_002.txt (20.85 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_003.txt">jm6b00709_si_003.txt (13.45 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_004.csv">jm6b00709_si_004.csv (3.4 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">4ZS0 (Aurora A complex with <b>6</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZTQ">4ZTQ</a> (Aurora A complex with <b>77</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZTR">4ZTR</a> (Aurora A complex with <b>9</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZTS">4ZTS</a> (Aurora A complex with <b>88</b>). Authors will release the atomic coordinates an experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.6b00709" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54567" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54567" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pierre Gönczy</span> - <span class="hlFld-Affiliation affiliation">Swiss
Institute for Experimental Cancer Research (ISREC), School of Life
Sciences, National Center of Competence in Research NCCR Chemical
Biology, Swiss Federal Institute of Technology
(EPFL), CH-1015 Lausanne, Switzerland</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#44342d213636216a232b2a273e3d04213422286a272c"><span class="__cf_email__" data-cfemail="e5958c80979780cb828a8b869f9ca580958389cb868d">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jean-Louis Reymond</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Biochemistry, National Center of Competence in Research
NCCR Chemical Biology and NCCR TransCure, University of Berne, Freiestrasse 3, 3012 Berne, Switzerland</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#412b24202f6c2d2e3428326f3324382c2e2f25012522236f342f2823246f2229"><span class="__cf_email__" data-cfemail="a7cdc2c6c98acbc8d2ced489d5c2decac8c9c3e7c3c4c589d2c9cec5c289c4cf">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Falco Kilchmann</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Biochemistry, National Center of Competence in Research
NCCR Chemical Biology and NCCR TransCure, University of Berne, Freiestrasse 3, 3012 Berne, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Maria J. Marcaida</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Biochemistry, National Center of Competence in Research
NCCR Chemical Biology and NCCR TransCure, University of Berne, Freiestrasse 3, 3012 Berne, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sachin Kotak</span> - <span class="hlFld-Affiliation affiliation">Swiss
Institute for Experimental Cancer Research (ISREC), School of Life
Sciences, National Center of Competence in Research NCCR Chemical
Biology, Swiss Federal Institute of Technology
(EPFL), CH-1015 Lausanne, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas Schick</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Biochemistry, National Center of Competence in Research
NCCR Chemical Biology and NCCR TransCure, University of Berne, Freiestrasse 3, 3012 Berne, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Silvan D. Boss</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Biochemistry, National Center of Competence in Research
NCCR Chemical Biology and NCCR TransCure, University of Berne, Freiestrasse 3, 3012 Berne, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mahendra Awale</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Biochemistry, National Center of Competence in Research
NCCR Chemical Biology and NCCR TransCure, University of Berne, Freiestrasse 3, 3012 Berne, Switzerland</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>These authors contributed equally. F.K. designed and carried out the study. M.J.M. designed, supervised, and performed crystallographic and biophysical studies. S.K. designed and performed cell culture experiments. T.S. performed crystallographic and biophysical studies. S.D.B. performed the synthesis of the indirubin-like series. M.A. performed and carried out virtual screening and scaffold analyses. P.G. and J.L.R. designed and supervised the study. All authors interpreted data. F.K., M.J.M., S.K., P.G. and J.L.R. wrote the paper.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e8986-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i121">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64636" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64636" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported financially by the University of Bern and the Swiss National Science Foundation programs NCCR Chemical Biology and NCCR TransCure. M.J.M. was funded by a Marie Heim-Vögtlin grant. We would like to thank Vincent Olieric at the SLS for his assistance and members of the Dal Peraro laboratory for helpful discussions and Clémence Delalande at the University of Bern for help with compound characterization.</p></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i122">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63523" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63523" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 87 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Virtual Ligand Screening: Strategies, Perspectives and Limitations</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">580</span><span class="NLM_x">–</span> <span class="NLM_lpage">594</span><span class="refDoi"> DOI: 10.1016/j.drudis.2006.05.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2Fj.drudis.2006.05.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=16793526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGqtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=580-594&author=G.+Klebe&title=Virtual+Ligand+Screening%3A+Strategies%2C+Perspectives+and+Limitations&doi=10.1016%2Fj.drudis.2006.05.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual ligand screening: strategies, perspectives and limitations</span></div><div class="casAuthors">Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">13 & 14</span>),
    <span class="NLM_cas:pages">580-594</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In contrast to high-throughput screening, in virtual ligand screening (VS), compds. are selected using computer programs to predict their binding to a target receptor.  A key prerequisite is knowledge about the spatial and energetic criteria responsible for protein-ligand binding.  The concepts and prerequisites to perform VS are summarized here, and explanations are sought for the enduring limitations of the technol.  Target selection, anal. and prepn. are discussed, as well as considerations about the compilation of candidate ligand libraries.  The tools and strategies of a VS campaign, and the accuracy of scoring and ranking of the results, are also considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN3dWlP-WDY7Vg90H21EOLACvtfcHk0ljeUh4HCgDetw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGqtLo%253D&md5=5adfd125d48082238a6c5ad0e8343c59</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.05.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.05.012%26sid%3Dliteratum%253Aachs%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DVirtual%2520Ligand%2520Screening%253A%2520Strategies%252C%2520Perspectives%2520and%2520Limitations%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2006%26volume%3D11%26spage%3D580%26epage%3D594%26doi%3D10.1016%2Fj.drudis.2006.05.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Scior, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tresadern, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medina-Franco, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez-Mayorga, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cuanalo-Contreras, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agrafiotis, D. K.</span><span> </span><span class="NLM_article-title">Recognizing Pitfalls in Virtual Screening: A Critical Review</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">867</span><span class="NLM_x">–</span> <span class="NLM_lpage">881</span><span class="refDoi"> DOI: 10.1021/ci200528d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200528d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC38Xkt1eiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=867-881&author=T.+Sciorauthor=A.+Benderauthor=G.+Tresadernauthor=J.+L.+Medina-Francoauthor=K.+Martinez-Mayorgaauthor=T.+Langerauthor=K.+Cuanalo-Contrerasauthor=D.+K.+Agrafiotis&title=Recognizing+Pitfalls+in+Virtual+Screening%3A+A+Critical+Review&doi=10.1021%2Fci200528d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Recognizing Pitfalls in Virtual Screening: A Critical Review</span></div><div class="casAuthors">Scior, Thomas; Bender, Andreas; Tresadern, Gary; Medina-Franco, Jose L.; Martinez-Mayorga, Karina; Langer, Thierry; Cuanalo-Contreras, Karina; Agrafiotis, Dimitris K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">867-881</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The aim of virtual screening (VS) is to identify bioactive compds. through computational means, by employing knowledge about the protein target (structure-based VS) or known bioactive ligands (ligand-based VS).  In VS, a large no. of mols. are ranked according to their likelihood to be bioactive compds., with the aim to enrich the top fraction of the resulting list (which can be tested in bioassays afterward).  At its core, VS attempts to improve the odds of identifying bioactive mols. by maximizing the true pos. rate, i.e., by ranking the truly active mols. as high as possible (and, correspondingly, the truly inactive ones as low as possible).  In choosing the right approach, the researcher is faced with many questions: where does the optimal balance between efficiency and accuracy lie when evaluating a particular algorithm; do some methods perform better than others and in what particular situations; and what do retrospective results tell us about the prospective utility of a particular method.  Given the multitude of settings, parameters, and data sets the practitioner can choose from, there are many pitfalls that lurk along the way which might render VS less efficient or downright useless.  This review attempts to catalog published and unpublished problems, shortcomings, failures, and tech. traps of VS methods with the aim to avoid pitfalls by making the user aware of them in the first place.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGois-3Bq6mm-LVg90H21EOLACvtfcHk0lhilD31hRrJNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xkt1eiurg%253D&md5=ad61b3289039c03844085a8b8de9e1e4</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fci200528d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200528d%26sid%3Dliteratum%253Aachs%26aulast%3DScior%26aufirst%3DT.%26aulast%3DBender%26aufirst%3DA.%26aulast%3DTresadern%26aufirst%3DG.%26aulast%3DMedina-Franco%26aufirst%3DJ.%2BL.%26aulast%3DMartinez-Mayorga%26aufirst%3DK.%26aulast%3DLanger%26aufirst%3DT.%26aulast%3DCuanalo-Contreras%26aufirst%3DK.%26aulast%3DAgrafiotis%26aufirst%3DD.%2BK.%26atitle%3DRecognizing%2520Pitfalls%2520in%2520Virtual%2520Screening%253A%2520A%2520Critical%2520Review%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D867%26epage%3D881%26doi%3D10.1021%2Fci200528d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Heikamp, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">The Future of Virtual Compound Screening</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">33</span><span class="NLM_x">–</span> <span class="NLM_lpage">40</span><span class="refDoi"> DOI: 10.1111/cbdd.12054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1111%2Fcbdd.12054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=23253129" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlt1ymuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2013&pages=33-40&author=K.+Heikampauthor=J.+Bajorath&title=The+Future+of+Virtual+Compound+Screening&doi=10.1111%2Fcbdd.12054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The future of virtual compound screening</span></div><div class="casAuthors">Heikamp, Kathrin; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-40</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  We provide a future perspective of the virtual screening field.  A no. of challenges will be highlighted that virtual screening will likely face when compd. data will further grow at or beyond current rates and when much more target information will become available.  These challenges go beyond computational efficiency issues (that will of course also play a crit. role).  For example, for structure-based approaches, the accuracy of scoring functions and energy calcns. will need to be improved.  For ligand-based approaches, the compd. class-dependence of similarity methods needs to be further explored and relationships between mol. similarity and activity similarity need to be established.  We also comment on the current and future value of virtual screening.  Opportunities for further development in a postgenome era are also discussed.  It is hoped that some of the views and hypotheses we articulate might stimulate further discussion about the virtual screening field going forward.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOYfHCsPxEabVg90H21EOLACvtfcHk0lhilD31hRrJNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlt1ymuw%253D%253D&md5=de1141b27ae39110f78b83fca24a2238</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12054%26sid%3Dliteratum%253Aachs%26aulast%3DHeikamp%26aufirst%3DK.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DThe%2520Future%2520of%2520Virtual%2520Compound%2520Screening%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2013%26volume%3D81%26spage%3D33%26epage%3D40%26doi%3D10.1111%2Fcbdd.12054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Kolb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferreira, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Docking and Chemoinformatic Screens for New Ligands and Targets</span> <span class="citation_source-journal">Curr. Opin. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">429</span><span class="NLM_x">–</span> <span class="NLM_lpage">436</span><span class="refDoi"> DOI: 10.1016/j.copbio.2009.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2Fj.copbio.2009.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=19733475" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1OrtLbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2009&pages=429-436&author=P.+Kolbauthor=R.+S.+Ferreiraauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Docking+and+Chemoinformatic+Screens+for+New+Ligands+and+Targets&doi=10.1016%2Fj.copbio.2009.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Docking and chemoinformatic screens for new ligands and targets</span></div><div class="casAuthors">Kolb, Peter; Ferreira, Rafaela S.; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Biotechnology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">429-436</span>CODEN:
                <span class="NLM_cas:coden">CUOBE3</span>;
        ISSN:<span class="NLM_cas:issn">0958-1669</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Computer-based docking screens are now widely used to discover new ligands for targets of known structure; in the last two years alone, the discovery of ligands for more than 20 proteins has been reported.  Recently, investigators have also turned to predicting new substrates for enzymes of unknown function, taking docking in a wholly new direction.  Increasingly, the hit rates, the true-positives, and the false-positives from the docking screens are being compared to those from empirical, high-throughput screens, revealing the strengths, weaknesses, and complementarities of both techniques.  The recent efflorescence of GPCR structures has made these quintessential drug targets available to structure-based approaches.  Consistent with their druggability', the docking screens have returned high hit rates and potent mols.  Finally, in the last several years, an approach almost exactly opposite to docking has also appeared; this pharmacol. network approach begins not with the structure of the target but rather those of drug mols. and asks, given a pattern of chem. in the ligands, what targets may a particular drug bind to.  This method, which returns to an older, pharmacol. logic, has been surprisingly successful in predicting new off-targets' for established drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryVNL3mMiQjbVg90H21EOLACvtfcHk0lg71L1fh0vVsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1OrtLbL&md5=4b6b2457a735e18be5780d9299ebffe6</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.copbio.2009.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.copbio.2009.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DFerreira%26aufirst%3DR.%2BS.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDocking%2520and%2520Chemoinformatic%2520Screens%2520for%2520New%2520Ligands%2520and%2520Targets%26jtitle%3DCurr.%2520Opin.%2520Biotechnol.%26date%3D2009%26volume%3D20%26spage%3D429%26epage%3D436%26doi%3D10.1016%2Fj.copbio.2009.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Forino, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Easton, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houghton, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pellecchia, M.</span><span> </span><span class="NLM_article-title">Virtual Docking Approaches to Protein Kinase B Inhibition</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">2278</span><span class="NLM_x">–</span> <span class="NLM_lpage">2281</span><span class="refDoi"> DOI: 10.1021/jm048962u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm048962u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=2278-2281&author=M.+Forinoauthor=D.+Jungauthor=J.+B.+Eastonauthor=P.+J.+Houghtonauthor=M.+Pellecchia&title=Virtual+Docking+Approaches+to+Protein+Kinase+B+Inhibition&doi=10.1021%2Fjm048962u"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fjm048962u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm048962u%26sid%3Dliteratum%253Aachs%26aulast%3DForino%26aufirst%3DM.%26aulast%3DJung%26aufirst%3DD.%26aulast%3DEaston%26aufirst%3DJ.%2BB.%26aulast%3DHoughton%26aufirst%3DP.%2BJ.%26aulast%3DPellecchia%26aufirst%3DM.%26atitle%3DVirtual%2520Docking%2520Approaches%2520to%2520Protein%2520Kinase%2520B%2520Inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D2278%26epage%3D2281%26doi%3D10.1021%2Fjm048962u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Geppert, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Current Trends in Ligand-Based Virtual Screening: Molecular Representations, Data Mining Methods, New Application Areas, and Performance Evaluation</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">205</span><span class="NLM_x">–</span> <span class="NLM_lpage">216</span><span class="refDoi"> DOI: 10.1021/ci900419k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900419k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvFWmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=205-216&author=H.+Geppertauthor=M.+Vogtauthor=J.+Bajorath&title=Current+Trends+in+Ligand-Based+Virtual+Screening%3A+Molecular+Representations%2C+Data+Mining+Methods%2C+New+Application+Areas%2C+and+Performance+Evaluation&doi=10.1021%2Fci900419k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Current Trends in Ligand-Based Virtual Screening: Molecular Representations, Data Mining Methods, New Application Areas, and Performance Evaluation</span></div><div class="casAuthors">Geppert, Hanna; Vogt, Martin; Bajorath, Jurgen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">205-216</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc1HgbkXa9o7Vg90H21EOLACvtfcHk0lg71L1fh0vVsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvFWmsA%253D%253D&md5=aeb674ef7c6c93711eb21452732e21d8</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fci900419k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900419k%26sid%3Dliteratum%253Aachs%26aulast%3DGeppert%26aufirst%3DH.%26aulast%3DVogt%26aufirst%3DM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DCurrent%2520Trends%2520in%2520Ligand-Based%2520Virtual%2520Screening%253A%2520Molecular%2520Representations%252C%2520Data%2520Mining%2520Methods%252C%2520New%2520Application%2520Areas%252C%2520and%2520Performance%2520Evaluation%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D205%26epage%3D216%26doi%3D10.1021%2Fci900419k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Willett, P.</span><span> </span><span class="NLM_article-title">Similarity-Based Virtual Screening Using 2d Fingerprints</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1046</span><span class="NLM_x">–</span> <span class="NLM_lpage">1053</span><span class="refDoi"> DOI: 10.1016/j.drudis.2006.10.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2Fj.drudis.2006.10.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=17129822" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1eqtb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=1046-1053&author=P.+Willett&title=Similarity-Based+Virtual+Screening+Using+2d+Fingerprints&doi=10.1016%2Fj.drudis.2006.10.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Similarity-based virtual screening using 2D fingerprints</span></div><div class="casAuthors">Willett, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">23 & 24</span>),
    <span class="NLM_cas:pages">1046-1053</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  This paper summarizes recent work at the University of Sheffield on virtual screening methods that use 2D fingerprint measures of structural similarity.  A detailed comparison of a large no. of similarity coeffs. demonstrates that the well-known Tanimoto coeff. remains the method of choice for the computation of fingerprint-based similarity, despite possessing some inherent biases related to the sizes of the mols. that are being sought.  Group fusion involves combining the results of similarity searches based on multiple ref. structures and a single similarity measure.  We demonstrate the effectiveness of this approach to screening, and also describe an approx. form of group fusion, turbo similarity searching, that can be used when just a single ref. structure is available.  This paper discusses the use of binary-encoded fragment substructures to scan databases to find mols. that are structurally similar to a bioactive query compd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3TRt6CMU5VrVg90H21EOLACvtfcHk0liJoK4oDKLlHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1eqtb%252FN&md5=095fe152b1ae1abbea69f02123651874</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.10.005%26sid%3Dliteratum%253Aachs%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DSimilarity-Based%2520Virtual%2520Screening%2520Using%25202d%2520Fingerprints%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2006%26volume%3D11%26spage%3D1046%26epage%3D1053%26doi%3D10.1016%2Fj.drudis.2006.10.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Giganti, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guillemain, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spadoni, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilges, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zagury, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montes, M.</span><span> </span><span class="NLM_article-title">Comparative Evaluation of 3d Virtual Ligand Screening Methods: Impact of the Molecular Alignment on Enrichment</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">992</span><span class="NLM_x">–</span> <span class="NLM_lpage">1004</span><span class="refDoi"> DOI: 10.1021/ci900507g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900507g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntVOns7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=992-1004&author=D.+Gigantiauthor=H.+Guillemainauthor=J.+L.+Spadoniauthor=M.+Nilgesauthor=J.+F.+Zaguryauthor=M.+Montes&title=Comparative+Evaluation+of+3d+Virtual+Ligand+Screening+Methods%3A+Impact+of+the+Molecular+Alignment+on+Enrichment&doi=10.1021%2Fci900507g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative Evaluation of 3D Virtual Ligand Screening Methods: Impact of the Molecular Alignment on Enrichment</span></div><div class="casAuthors">Giganti, David; Guillemain, Helene; Spadoni, Jean-Louis; Nilges, Michael; Zagury, Jean-Francois; Montes, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">992-1004</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the early stage of drug discovery programs, when the structure of a complex involving a target and a small mol. is available, structure-based virtual ligand screening methods are generally preferred.  However, ligand-based strategies like shape-similarity search methods can also be applied.  Shape-similarity search methods consist in exploring a pseudo-binding-site derived from the known small mol. used as a ref.  Several of these methods use conformational sampling algorithms which are also shared by corresponding docking methods: for example Surflex-dock/Surflex-sim, FlexX/FlexS, ICM, and OMEGA-FRED/OMEGA-ROCS.  Using 11 systems issued from the challenging "own" subsets of the Directory of Useful Decoys (DUD-own), we evaluated and compared the performance of the above-cited programs in terms of mol. alignment accuracy, enrichment in active compds., and enrichment in different chemotypes (scaffold-hopping).  Since mol. alignment is a crucial aspect of performance for the different methods, we have assessed its impact on enrichment.  We have also illustrated the paradox of retrieving active compds. with good scores even if they are inaccurately positioned.  Finally, we have highlighted possible pos. aspects of using shape-based approaches in drug-discovery protocols when the structure of the target in complex with a small mol. is known.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbvXSUiRzBC7Vg90H21EOLACvtfcHk0liJoK4oDKLlHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntVOns7o%253D&md5=5d455690f7d4260839ef6c228b9b5364</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fci900507g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900507g%26sid%3Dliteratum%253Aachs%26aulast%3DGiganti%26aufirst%3DD.%26aulast%3DGuillemain%26aufirst%3DH.%26aulast%3DSpadoni%26aufirst%3DJ.%2BL.%26aulast%3DNilges%26aufirst%3DM.%26aulast%3DZagury%26aufirst%3DJ.%2BF.%26aulast%3DMontes%26aufirst%3DM.%26atitle%3DComparative%2520Evaluation%2520of%25203d%2520Virtual%2520Ligand%2520Screening%2520Methods%253A%2520Impact%2520of%2520the%2520Molecular%2520Alignment%2520on%2520Enrichment%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D992%26epage%3D1004%26doi%3D10.1021%2Fci900507g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Lemmen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lengauer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Flexs: A Method for Fast Flexible Ligand Superposition</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">4502</span><span class="NLM_x">–</span> <span class="NLM_lpage">4520</span><span class="refDoi"> DOI: 10.1021/jm981037l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm981037l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=4502-4520&author=C.+Lemmenauthor=T.+Lengauerauthor=G.+Klebe&title=Flexs%3A+A+Method+for+Fast+Flexible+Ligand+Superposition&doi=10.1021%2Fjm981037l"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm981037l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm981037l%26sid%3Dliteratum%253Aachs%26aulast%3DLemmen%26aufirst%3DC.%26aulast%3DLengauer%26aufirst%3DT.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DFlexs%253A%2520A%2520Method%2520for%2520Fast%2520Flexible%2520Ligand%2520Superposition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D4502%26epage%3D4520%26doi%3D10.1021%2Fjm981037l" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Andrews, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cramer, R. D.</span><span> </span><span class="NLM_article-title">Toward General Methods of Targeted Library Design: Topomer Shape Similarity Searching with Diverse Structures as Queries</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">1723</span><span class="NLM_x">–</span> <span class="NLM_lpage">1740</span><span class="refDoi"> DOI: 10.1021/jm000003m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm000003m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD3cXitF2gsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=1723-1740&author=K.+M.+Andrewsauthor=R.+D.+Cramer&title=Toward+General+Methods+of+Targeted+Library+Design%3A+Topomer+Shape+Similarity+Searching+with+Diverse+Structures+as+Queries&doi=10.1021%2Fjm000003m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Toward general methods of targeted library design. Topomer shape similarity searching with diverse structures as queries</span></div><div class="casAuthors">Andrews, Katherine M.; Cramer, Richard D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1723-1740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A promising strategy for selecting synthetic targets is similarity-based searching of very large "virtual libraries", which comprise all structures accessible by linking 2 or 3 com. available building blocks with combinatorial syntheses.  To assess the general applicability of this strategy, leading structures taken from each of 34 recent medicinal chem. publications were used as queries to search a virtual library contg. 2.6 × 1013 products from 7 reactions, using a topomer shape similarity metric.  85% Of these searches succeeded, by yielding, with a search radius no greater than 120 topomer shape units, either at least 400 hits or hits from at least 6 sublibraries.  From these 34 sets of search results, 122 representative structures were selected, illustrating potential "lead hops", or otherwise novel structures.  Overall shape similarity to the query structure was confirmed for up to 95% of these representative structures, according to FLEXS, an algorithmically distinct program.  Exptl., there were 28 structures among those reported in the 34 query publications that were identified within the virtual library.  Among these, the frequency of high activity was 87% for the 16 structures whose similarity to their query was 90 topomer units or less, compared to a frequency of 50% for the other 12 structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfSsMO0zff6bVg90H21EOLACvtfcHk0ligkqxbD3TLYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXitF2gsbw%253D&md5=ea40f3ae6042b09413ad43c0f0061431</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm000003m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm000003m%26sid%3Dliteratum%253Aachs%26aulast%3DAndrews%26aufirst%3DK.%2BM.%26aulast%3DCramer%26aufirst%3DR.%2BD.%26atitle%3DToward%2520General%2520Methods%2520of%2520Targeted%2520Library%2520Design%253A%2520Topomer%2520Shape%2520Similarity%2520Searching%2520with%2520Diverse%2520Structures%2520as%2520Queries%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D1723%26epage%3D1740%26doi%3D10.1021%2Fjm000003m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Jain, A. N.</span><span> </span><span class="NLM_article-title">Ligand-Based Structural Hypotheses for Virtual Screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">947</span><span class="NLM_x">–</span> <span class="NLM_lpage">961</span><span class="refDoi"> DOI: 10.1021/jm030520f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030520f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD2cXkt1GnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=947-961&author=A.+N.+Jain&title=Ligand-Based+Structural+Hypotheses+for+Virtual+Screening&doi=10.1021%2Fjm030520f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand-Based Structural Hypotheses for Virtual Screening</span></div><div class="casAuthors">Jain, Ajay N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">947-961</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The majority of drug targets for small mol. therapeutics are proteins whose three-dimensional structure is not known to sufficient resoln. to permit structure-based design.  All three-dimensional QSAR approaches have a requirement for some hypothesis of ligand conformation and alignment, and predictions of mol. activity critically depend on this ligand-based binding site hypothesis.  The mol. similarity function used in the Surflex docking system, coupled with quant. pressure to minimize overall mol. vol., forms an effective objective function for generating hypotheses of bioactive conformations of sets of small mols. binding to their cognate proteins.  Results are presented, assessing utility of the method for ligands of the serotonin, histamine, muscarinic, and GABAA receptors.  The Surflex similarity module (Surflex-Sim) was able, in each case, to distinguish true ligands from random compds. using models constructed from just two or three known ligands.  True pos. rates of 60% were achieved with false pos. rates of 0-3%; the theor. enrichment rates were over 150-fold compared with random screening.  The methods are practically applicable for rational design of ligands and for high-throughput virtual screening and offer competitive performance to many structure-based docking algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSKeodYur23bVg90H21EOLACvtfcHk0lgvFsORzYT8cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXkt1GnsQ%253D%253D&md5=7442ef3b414134cc32add2d159ce199b</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm030520f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030520f%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DLigand-Based%2520Structural%2520Hypotheses%2520for%2520Virtual%2520Screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D947%26epage%3D961%26doi%3D10.1021%2Fjm030520f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Korb, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monecke, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hessler, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stutzle, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Exner, T. E.</span><span> </span><span class="NLM_article-title">Pharmacophore: Multiple Flexible Ligand Alignment Based on Ant Colony Optimization</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1669</span><span class="NLM_x">–</span> <span class="NLM_lpage">1681</span><span class="refDoi"> DOI: 10.1021/ci1000218</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci1000218" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtV2jtr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1669-1681&author=O.+Korbauthor=P.+Moneckeauthor=G.+Hesslerauthor=T.+Stutzleauthor=T.+E.+Exner&title=Pharmacophore%3A+Multiple+Flexible+Ligand+Alignment+Based+on+Ant+Colony+Optimization&doi=10.1021%2Fci1000218"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">pharmACOphore: Multiple Flexible Ligand Alignment Based on Ant Colony Optimization</span></div><div class="casAuthors">Korb, Oliver; Monecke, Peter; Hessler, Gerhard; Stuetzle, Thomas; Exner, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1669-1681</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The flexible superimposition of biol. active ligands is a crucial step in ligand-based drug design.  Here we present pharmACOphore, a new approach for pairwise as well as multiple flexible alignment of ligands based on ant colony optimization.  An empirical scoring function is used, which describes ligand similarity by minimizing the distance of pharmacophoric features.  The scoring function was parametrized on pairwise alignments of ligand sets for four proteins from diverse protein families (cyclooxygenase-2, cyclin-dependent kinase 2, factor Xa and peroxisome proliferator-activated receptor γ).  The derived parameters were assessed with respect to pose prediction performance on the independent FlexS data set in exhausting pairwise alignments.  Addnl., multiple flexible alignment expts. were carried out for the pharmacol. relevant targets trypsin and poly (ADP-ribose) polymerase (PARP).  The results obtained show that the new procedure provides a robust and efficient way for the pairwise as well as multiple flexible alignment of small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV9Q-o6EevhrVg90H21EOLACvtfcHk0lgvFsORzYT8cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtV2jtr7O&md5=01deb5cd5dea2f3dd9262b5221d8381c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fci1000218&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci1000218%26sid%3Dliteratum%253Aachs%26aulast%3DKorb%26aufirst%3DO.%26aulast%3DMonecke%26aufirst%3DP.%26aulast%3DHessler%26aufirst%3DG.%26aulast%3DStutzle%26aufirst%3DT.%26aulast%3DExner%26aufirst%3DT.%2BE.%26atitle%3DPharmacophore%253A%2520Multiple%2520Flexible%2520Ligand%2520Alignment%2520Based%2520on%2520Ant%2520Colony%2520Optimization%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1669%26epage%3D1681%26doi%3D10.1021%2Fci1000218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Nicholls, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGaughey, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Good, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warren, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathieu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muchmore, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haigh, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nevins, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelley, B.</span><span> </span><span class="NLM_article-title">Molecular Shape and Medicinal Chemistry: A Perspective</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">3862</span><span class="NLM_x">–</span> <span class="NLM_lpage">3886</span><span class="refDoi"> DOI: 10.1021/jm900818s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm900818s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvF2kt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=3862-3886&author=A.+Nichollsauthor=G.+B.+McGaugheyauthor=R.+P.+Sheridanauthor=A.+C.+Goodauthor=G.+Warrenauthor=M.+Mathieuauthor=S.+W.+Muchmoreauthor=S.+P.+Brownauthor=J.+A.+Grantauthor=J.+A.+Haighauthor=N.+Nevinsauthor=A.+N.+Jainauthor=B.+Kelley&title=Molecular+Shape+and+Medicinal+Chemistry%3A+A+Perspective&doi=10.1021%2Fjm900818s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Shape and Medicinal Chemistry: A Perspective</span></div><div class="casAuthors">Nicholls, Anthony; McGaughey, Georgia B.; Sheridan, Robert P.; Good, Andrew C.; Warren, Gregory; Mathieu, Magali; Muchmore, Steven W.; Brown, Scott P.; Grant, J. Andrew; Haigh, James A.; Nevins, Neysa; Jain, Ajay N.; Kelley, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3862-3886</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review article with 111 refs. summarized perspectives of mol. shape and medicinal chem. in drug screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqtbXOEoSmy7Vg90H21EOLACvtfcHk0lgvFsORzYT8cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvF2kt7k%253D&md5=85664344e13872527a3dfb2296d34864</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm900818s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm900818s%26sid%3Dliteratum%253Aachs%26aulast%3DNicholls%26aufirst%3DA.%26aulast%3DMcGaughey%26aufirst%3DG.%2BB.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DGood%26aufirst%3DA.%2BC.%26aulast%3DWarren%26aufirst%3DG.%26aulast%3DMathieu%26aufirst%3DM.%26aulast%3DMuchmore%26aufirst%3DS.%2BW.%26aulast%3DBrown%26aufirst%3DS.%2BP.%26aulast%3DGrant%26aufirst%3DJ.%2BA.%26aulast%3DHaigh%26aufirst%3DJ.%2BA.%26aulast%3DNevins%26aufirst%3DN.%26aulast%3DJain%26aufirst%3DA.%2BN.%26aulast%3DKelley%26aufirst%3DB.%26atitle%3DMolecular%2520Shape%2520and%2520Medicinal%2520Chemistry%253A%2520A%2520Perspective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D3862%26epage%3D3886%26doi%3D10.1021%2Fjm900818s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Underwood, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kearsley, S. K.</span><span> </span><span class="NLM_article-title">Chemical Similarity Using Geometric Atom Pair Descriptors</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">128</span><span class="NLM_x">–</span> <span class="NLM_lpage">136</span><span class="refDoi"> DOI: 10.1021/ci950275b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci950275b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADyaK28XhsFKitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1996&pages=128-136&author=R.+P.+Sheridanauthor=M.+D.+Millerauthor=D.+J.+Underwoodauthor=S.+K.+Kearsley&title=Chemical+Similarity+Using+Geometric+Atom+Pair+Descriptors&doi=10.1021%2Fci950275b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Similarity Using Geometric Atom Pair Descriptors</span></div><div class="casAuthors">Sheridan, Robert P.; Miller, Michael D.; Underwood, Dennis J.; Kearsley, Simon K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">128-36</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this paper we describe the geometric atom pair, the 3D analog of the topol. atom pair descriptor (Carhart et al. J. Chem. Inf. Comput. Sci. 1985, 25, 64-73).  We show the results of geometric similarity searches using the CONCORD-build structures of typical small druglike mols. as probes.  The database to be searched is a 3D version of the Derwent Std. Drug File that contains an av. of 10 explicit conformations per compd.  Using objective criteria for detg. how good a descriptor is in selecting active compds. from large databases, we compare the results using the geometric vs. the topol. atom pair.  We find that geometric and topol. atom pairs are about equally effective in selecting active compds. from large databases.  How the two types of descriptors rank active compds. is generally about the same as well, but occasionally active compds. will be seen as very similar to a probe in geometric descriptors, but as fairly dissimilar in topol. descriptors.  These are of two types: (1) compds. where equiv. groups are in the same spatial arrangement as in the probe but are connected by very different bond paths and (2) compds. that can superimpose onto the probe when they are in a folded conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBZVblsDYTNLVg90H21EOLACvtfcHk0lgvXI3gHyxCjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhsFKitA%253D%253D&md5=99ed1470e6973f090bcbb4025c3a0445</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fci950275b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci950275b%26sid%3Dliteratum%253Aachs%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DMiller%26aufirst%3DM.%2BD.%26aulast%3DUnderwood%26aufirst%3DD.%2BJ.%26aulast%3DKearsley%26aufirst%3DS.%2BK.%26atitle%3DChemical%2520Similarity%2520Using%2520Geometric%2520Atom%2520Pair%2520Descriptors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1996%26volume%3D36%26spage%3D128%26epage%3D136%26doi%3D10.1021%2Fci950275b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Schreyer, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T.</span><span> </span><span class="NLM_article-title">Usrcat: Real-Time Ultrafast Shape Recognition with Pharmacophoric Constraints</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">27</span><span class="NLM_x">–</span> <span class="NLM_lpage">39</span><span class="refDoi"> DOI: 10.1186/1758-2946-4-27</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1186%2F1758-2946-4-27" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVGjt7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=27-39&author=A.+M.+Schreyerauthor=T.+Blundell&title=Usrcat%3A+Real-Time+Ultrafast+Shape+Recognition+with+Pharmacophoric+Constraints&doi=10.1186%2F1758-2946-4-27"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">USRCAT: real-time Ultrafast Shape Recognition with pharmacophoric constraints</span></div><div class="casAuthors">Schreyer, Adrian M.; Blundell, Tom</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">27</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Background: Ligand-based virtual screening using mol. shape is an important tool for researchers who wish to find novel chem. scaffolds in compd. libraries.  The Ultrafast Shape Recognition (USR) algorithm is capable of screening millions of compds. and is therefore suitable for usage in a web service.  The algorithm however is agnostic of atom types and cannot discriminate compds. with similar shape but distinct pharmacophoric features.  To solve this problem, an extension of USR called USRCAT, has been developed that includes pharmacophoric information while retaining the performance benefits of the original method.  Results: The USRCAT extension is shown to outperform the traditional USR method in a retrospective virtual screening benchmark.  Also, a relational database implementation is described that is capable of screening a million conformers in milliseconds and allows the inclusion of complex query parameters.  Conclusions: USRCAT provides a soln. to the lack of atom type information in the USR algorithm.  Researchers, particularly those with only limited resources, who wish to use ligand-based virtual screening in order to discover new hits, will benefit the most.  Online chem. databases that offer a shape-based similarity method might also find advantage in using USRCAT due to its accuracy and performance.  The source code is freely available and can easily be modified to fit specific needs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxouS95m5dGrVg90H21EOLACvtfcHk0lgvXI3gHyxCjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVGjt7fJ&md5=9e7e82f660a7ec4e542d539bed081a25</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-4-27&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-4-27%26sid%3Dliteratum%253Aachs%26aulast%3DSchreyer%26aufirst%3DA.%2BM.%26aulast%3DBlundell%26aufirst%3DT.%26atitle%3DUsrcat%253A%2520Real-Time%2520Ultrafast%2520Shape%2520Recognition%2520with%2520Pharmacophoric%2520Constraints%26jtitle%3DJ.%2520Cheminf.%26date%3D2012%26volume%3D4%26spage%3D27%26epage%3D39%26doi%3D10.1186%2F1758-2946-4-27" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Awale, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reymond, J. L.</span><span> </span><span class="NLM_article-title">Stereoselective Virtual Screening of the Zinc Database Using Atom Pair 3d-Fingerprints</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="refDoi"> DOI: 10.1186/s13321-014-0051-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1186%2Fs13321-014-0051-5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=3&author=M.+Awaleauthor=X.+Jinauthor=J.+L.+Reymond&title=Stereoselective+Virtual+Screening+of+the+Zinc+Database+Using+Atom+Pair+3d-Fingerprints&doi=10.1186%2Fs13321-014-0051-5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1186%2Fs13321-014-0051-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-014-0051-5%26sid%3Dliteratum%253Aachs%26aulast%3DAwale%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DX.%26aulast%3DReymond%26aufirst%3DJ.%2BL.%26atitle%3DStereoselective%2520Virtual%2520Screening%2520of%2520the%2520Zinc%2520Database%2520Using%2520Atom%2520Pair%25203d-Fingerprints%26jtitle%3DJ.%2520Cheminf.%26date%3D2015%26volume%3D7%26spage%3D3%26doi%3D10.1186%2Fs13321-014-0051-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Schneider, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidhart, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giller, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmid, G.</span><span> </span><span class="NLM_article-title">″Scaffold-Hopping″ by Topological Pharmacophore Search: A Contribution to Virtual Screening</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">2894</span><span class="NLM_x">–</span> <span class="NLM_lpage">2896</span><span class="refDoi"> DOI: 10.1002/(SICI)1521-3773(19991004)38:19<2894::AID-ANIE2894>3.3.CO;2-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1002%2F%28SICI%291521-3773%2819991004%2938%3A19%3C2894%3A%3AAID-ANIE2894%3E3.3.CO%3B2-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=2894-2896&author=G.+Schneiderauthor=W.+Neidhartauthor=T.+Gillerauthor=G.+Schmid&title=%E2%80%B3Scaffold-Hopping%E2%80%B3+by+Topological+Pharmacophore+Search%3A+A+Contribution+to+Virtual+Screening&doi=10.1002%2F%28SICI%291521-3773%2819991004%2938%3A19%3C2894%3A%3AAID-ANIE2894%3E3.3.CO%3B2-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291521-3773%2819991004%2938%3A19%3C2894%3A%3AAID-ANIE2894%3E3.3.CO%3B2-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291521-3773%252819991004%252938%253A19%253C2894%253A%253AAID-ANIE2894%253E3.3.CO%253B2-6%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DNeidhart%26aufirst%3DW.%26aulast%3DGiller%26aufirst%3DT.%26aulast%3DSchmid%26aufirst%3DG.%26atitle%3D%25E2%2580%25B3Scaffold-Hopping%25E2%2580%25B3%2520by%2520Topological%2520Pharmacophore%2520Search%253A%2520A%2520Contribution%2520to%2520Virtual%2520Screening%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1999%26volume%3D38%26spage%3D2894%26epage%3D2896%26doi%3D10.1002%2F%28SICI%291521-3773%2819991004%2938%3A19%3C2894%3A%3AAID-ANIE2894%3E3.3.CO%3B2-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Marumoto, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saya, H.</span><span> </span><span class="NLM_article-title">Aurora-a - a Guardian of Poles</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">42</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1038/nrc1526</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1038%2Fnrc1526" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=15630414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlvVen" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2005&pages=42-50&author=T.+Marumotoauthor=D.+Zhangauthor=H.+Saya&title=Aurora-a+-+a+Guardian+of+Poles&doi=10.1038%2Fnrc1526"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Aurora-A - A guardian of poles</span></div><div class="casAuthors">Marumoto, Tomotoshi; Zhang, Dongwei; Saya, Hideyuki</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">42-50</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The three human homologs of Aurora kinases (A, B and C) are essential for proper execution of various mitotic events and are important for maintaining genomic integrity.  Aurora-A is mainly localized at spindle poles and the mitotic spindle during mitosis, where it regulates the functions of centrosomes, spindles and kinetochores required for proper mitotic progression.  Recent studies have revealed that Aurora-A is frequently overexpressed in various cancer cells, indicating its involvement in tumorigenesis.  What are the normal physiol. roles of Aurora-A, how are these regulated and how might the enzyme function during tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZ5DkKXAtszrVg90H21EOLACvtfcHk0li4YwII4H1Qhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlvVen&md5=5eef2a40508d408be784301b20e6822d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnrc1526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1526%26sid%3Dliteratum%253Aachs%26aulast%3DMarumoto%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DSaya%26aufirst%3DH.%26atitle%3DAurora-a%2520-%2520a%2520Guardian%2520of%2520Poles%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2005%26volume%3D5%26spage%3D42%26epage%3D50%26doi%3D10.1038%2Fnrc1526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Metz, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soni, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merta, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kifle, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span> </span><span class="NLM_article-title">Navigating the Kinome</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">200</span><span class="NLM_x">–</span> <span class="NLM_lpage">202</span><span class="refDoi"> DOI: 10.1038/nchembio.530</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1038%2Fnchembio.530" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=21336281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1aitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=200-202&author=J.+T.+Metzauthor=E.+F.+Johnsonauthor=N.+B.+Soniauthor=P.+J.+Mertaauthor=L.+Kifleauthor=P.+J.+Hajduk&title=Navigating+the+Kinome&doi=10.1038%2Fnchembio.530"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Navigating the kinome</span></div><div class="casAuthors">Metz, James T.; Johnson, Eric F.; Soni, Niru B.; Merta, Philip J.; Kifle, Lemma; Hajduk, Philip J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">200-202</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although it is increasingly being recognized that drug-target interaction networks can be powerful tools for the interrogation of systems biol. and the rational design of multitargeted drugs, there is no generalized, statistically validated approach to harmonizing sequence-dependent and pharmacol.-dependent networks.  Here we demonstrate the creation of a comprehensive kinome interaction network based not only on sequence comparisons but also on multiple pharmacol. parameters derived from activity profiling data.  The framework described for statistical interpretation of these network connections also enables rigorous investigation of chemotype-specific interaction networks, which is crit. for multitargeted drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOgzF9Q0eH87Vg90H21EOLACvtfcHk0li4YwII4H1Qhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1aitrc%253D&md5=09234ec451a4918e5aca6d65e0b65aee</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.530%26sid%3Dliteratum%253Aachs%26aulast%3DMetz%26aufirst%3DJ.%2BT.%26aulast%3DJohnson%26aufirst%3DE.%2BF.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DMerta%26aufirst%3DP.%2BJ.%26aulast%3DKifle%26aufirst%3DL.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DNavigating%2520the%2520Kinome%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D200%26epage%3D202%26doi%3D10.1038%2Fnchembio.530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Carmena, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Earnshaw, W. C.</span><span> </span><span class="NLM_article-title">The Cellular Geography of Aurora Kinases</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">842</span><span class="NLM_x">–</span> <span class="NLM_lpage">854</span><span class="refDoi"> DOI: 10.1038/nrm1245</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1038%2Fnrm1245" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=14625535" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpvFais7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2003&pages=842-854&author=M.+Carmenaauthor=W.+C.+Earnshaw&title=The+Cellular+Geography+of+Aurora+Kinases&doi=10.1038%2Fnrm1245"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The cellular geography of Aurora kinases</span></div><div class="casAuthors">Carmena, Mar; Earnshaw, William C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">842-854</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Aurora is the name given to a family of highly conserved protein kinases with essential roles in many aspects of cell division.  Yeasts have a single Aurora kinase, whereas mammals have three: Aurora A, B and C.  During mitosis, Aurora kinases regulate the structure and function of the cytoskeleton and chromosomes and the interactions between these two at the kinetochore.  They also regulate signalling by the spindle-assembly checkpoint pathway and cytokinesis.  Perturbation of Aurora kinase expression or function might lead to cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrU7PA68xW-NLVg90H21EOLACvtfcHk0liZYkElnZxwFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpvFais7g%253D&md5=4887100c6893eb6a40537cdc9916c1a4</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnrm1245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1245%26sid%3Dliteratum%253Aachs%26aulast%3DCarmena%26aufirst%3DM.%26aulast%3DEarnshaw%26aufirst%3DW.%2BC.%26atitle%3DThe%2520Cellular%2520Geography%2520of%2520Aurora%2520Kinases%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2003%26volume%3D4%26spage%3D842%26epage%3D854%26doi%3D10.1038%2Fnrm1245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Shagisultanova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunbrack, R. L.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golemis, E. A.</span><span> </span><span class="NLM_article-title">Issues in Interpreting the in Vivo Activity of Aurora-A</span> <span class="citation_source-journal">Expert Opin. Ther. Targets</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">187</span><span class="NLM_x">–</span> <span class="NLM_lpage">200</span><span class="refDoi"> DOI: 10.1517/14728222.2014.981154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1517%2F14728222.2014.981154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=25384454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlCrt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2015&pages=187-200&author=E.+Shagisultanovaauthor=R.+L.+Dunbrackauthor=E.+A.+Golemis&title=Issues+in+Interpreting+the+in+Vivo+Activity+of+Aurora-A&doi=10.1517%2F14728222.2014.981154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Issues in interpreting the in vivo activity of Aurora-A</span></div><div class="casAuthors">Shagisultanova, Elena; Dunbrack, Roland L., Jr.; Golemis, Erica A.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">187-200</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">Introduction: Based on its role as a mitotic regulatory kinase, overexpressed and assocd. with aneuploidy in cancer, small-mol. inhibitors have been developed for Aurora-A (AURKA) kinase.  In preclin. and clin. assessments, these agents have shown efficacy in inducing stable disease or therapeutic response.  In optimizing the use of Aurora-A inhibitors, it is crit. to have robust capacity to measure the kinase activity of Aurora-A in tumors.  Areas covered: We provide an overview of mol. mechanisms of mitotic and non-mitotic activation of Aurora-A kinase, and interaction of Aurora-A with its regulatory partners.  Typically, Aurora-A activity is measured by use of phospho-antibodies targeting an autophosphorylated T288 epitope.  However, recent studies have identified alternative means of Aurora-A activation control, including allosteric regulation by partners, phosphorylation on alternative activating residues (S51, S98), dephosphorylation on inhibitory sites (S342) and T288 phosphorylation by alternative kinases such as Pak enzymes.  Addnl. work has shown that the relative abundance of Aurora-A partners can affect the activity of Aurora-A inhibitors, and that Aurora-A activation also occurs in interphase cells.  Expert opinion: Taken together, this work suggests the need for comprehensive anal. of Aurora-A activity and expression of Aurora-A partners in order to stratify patients for likely therapeutic response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnUIECo5Cq47Vg90H21EOLACvtfcHk0liZYkElnZxwFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlCrt7c%253D&md5=2ad2457af5ceae42abb5c70445c9693f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1517%2F14728222.2014.981154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728222.2014.981154%26sid%3Dliteratum%253Aachs%26aulast%3DShagisultanova%26aufirst%3DE.%26aulast%3DDunbrack%26aufirst%3DR.%2BL.%26aulast%3DGolemis%26aufirst%3DE.%2BA.%26atitle%3DIssues%2520in%2520Interpreting%2520the%2520in%2520Vivo%2520Activity%2520of%2520Aurora-A%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2015%26volume%3D19%26spage%3D187%26epage%3D200%26doi%3D10.1517%2F14728222.2014.981154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">de Groot, C. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anzola, J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motamedi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoon, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gahman, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiau, A. K.</span><span> </span><span class="NLM_article-title">A Cell Biologist’s Field Guide to Aurora Kinase Inhibitors</span> <span class="citation_source-journal">Front. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">285</span><span class="refDoi"> DOI: 10.3389/fonc.2015.00285</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.3389%2Ffonc.2015.00285" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=26732741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A280%3ADC%252BC28rnsFSjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=285&author=C.+O.+de+Grootauthor=J.+E.+Hsiaauthor=J.+V.+Anzolaauthor=A.+Motamediauthor=M.+Yoonauthor=Y.+L.+Wongauthor=D.+Jenkinsauthor=H.+J.+Leeauthor=M.+B.+Martinezauthor=R.+L.+Davisauthor=T.+C.+Gahmanauthor=A.+Desaiauthor=A.+K.+Shiau&title=A+Cell+Biologist%E2%80%99s+Field+Guide+to+Aurora+Kinase+Inhibitors&doi=10.3389%2Ffonc.2015.00285"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A Cell Biologist's Field Guide to Aurora Kinase Inhibitors</span></div><div class="casAuthors">de Groot Christian O; Hsia Judy E; Anzola John V; Motamedi Amir; Yoon Michelle; Jenkins David; Lee Hyun J; Martinez Mallory B; Davis Robert L; Gahman Timothy C; Shiau Andrew K; Wong Yao Liang; Desai Arshad</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">285</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">Aurora kinases are essential for cell division and are frequently misregulated in human cancers.  Based on their potential as cancer therapeutics, a plethora of small molecule Aurora kinase inhibitors have been developed, with a subset having been adopted as tools in cell biology.  Here, we fill a gap in the characterization of Aurora kinase inhibitors by using biochemical and cell-based assays to systematically profile a panel of 10 commercially available compounds with reported selectivity for Aurora A (MLN8054, MLN8237, MK-5108, MK-8745, Genentech Aurora Inhibitor 1), Aurora B (Hesperadin, ZM447439, AZD1152-HQPA, GSK1070916), or Aurora A/B (VX-680).  We quantify the in vitro effect of each inhibitor on the activity of Aurora A alone, as well as Aurora A and Aurora B bound to fragments of their activators, TPX2 and INCENP, respectively.  We also report kinome profiling results for a subset of these compounds to highlight potential off-target effects.  In a cellular context, we demonstrate that immunofluorescence-based detection of LATS2 and histone H3 phospho-epitopes provides a facile and reliable means to assess potency and specificity of Aurora A versus Aurora B inhibition, and that G2 duration measured in a live imaging assay is a specific readout of Aurora A activity.  Our analysis also highlights variation between HeLa, U2OS, and hTERT-RPE1 cells that impacts selective Aurora A inhibition.  For Aurora B, all four tested compounds exhibit excellent selectivity and do not significantly inhibit Aurora A at effective doses.  For Aurora A, MK-5108 and MK-8745 are significantly more selective than the commonly used inhibitors MLN8054 and MLN8237.  A crystal structure of an Aurora A/MK-5108 complex that we determined suggests the chemical basis for this higher specificity.  Taken together, our quantitative biochemical and cell-based analyses indicate that AZD1152-HQPA and MK-8745 are the best current tools for selectively inhibiting Aurora B and Aurora A, respectively.  However, MK-8745 is not nearly as ideal as AZD1152-HQPA in that it requires high concentrations to achieve full inhibition in a cellular context, indicating a need for more potent Aurora A-selective inhibitors.  We conclude with a set of "good practice" guidelines for the use of Aurora inhibitors in cell biology experiments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQItUi6CXjsM669lAbTj9rGfW6udTcc2eY_c0b7JP7G-7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28rnsFSjuw%253D%253D&md5=ad773bddf44e4bf624ee83f400912885</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2015.00285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2015.00285%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BGroot%26aufirst%3DC.%2BO.%26aulast%3DHsia%26aufirst%3DJ.%2BE.%26aulast%3DAnzola%26aufirst%3DJ.%2BV.%26aulast%3DMotamedi%26aufirst%3DA.%26aulast%3DYoon%26aufirst%3DM.%26aulast%3DWong%26aufirst%3DY.%2BL.%26aulast%3DJenkins%26aufirst%3DD.%26aulast%3DLee%26aufirst%3DH.%2BJ.%26aulast%3DMartinez%26aufirst%3DM.%2BB.%26aulast%3DDavis%26aufirst%3DR.%2BL.%26aulast%3DGahman%26aufirst%3DT.%2BC.%26aulast%3DDesai%26aufirst%3DA.%26aulast%3DShiau%26aufirst%3DA.%2BK.%26atitle%3DA%2520Cell%2520Biologist%25E2%2580%2599s%2520Field%2520Guide%2520to%2520Aurora%2520Kinase%2520Inhibitors%26jtitle%3DFront.%2520Oncol.%26date%3D2015%26volume%3D5%26spage%3D285%26doi%3D10.3389%2Ffonc.2015.00285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Manfredi, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ecsedy, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chakravarty, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silverman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoar, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroud, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinde, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huck, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wysong, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janowick, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hyer, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leroy, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gershman, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silva, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Germanos, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolen, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Claiborne, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sells, T. B.</span><span> </span><span class="NLM_article-title">Characterization of Alisertib (Mln8237), an Investigational Small-Molecule Inhibitor of Aurora a Kinase Using Novel in Vivo Pharmacodynamic Assays</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">7614</span><span class="NLM_x">–</span> <span class="NLM_lpage">7624</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-11-1536</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1158%2F1078-0432.CCR-11-1536" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=22016509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1CqtrjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=7614-7624&author=M.+G.+Manfrediauthor=J.+A.+Ecsedyauthor=A.+Chakravartyauthor=L.+Silvermanauthor=M.+Zhangauthor=K.+M.+Hoarauthor=S.+G.+Stroudauthor=W.+Chenauthor=V.+Shindeauthor=J.+J.+Huckauthor=D.+R.+Wysongauthor=D.+A.+Janowickauthor=M.+L.+Hyerauthor=P.+J.+Leroyauthor=R.+E.+Gershmanauthor=M.+D.+Silvaauthor=M.+S.+Germanosauthor=J.+B.+Bolenauthor=C.+F.+Claiborneauthor=T.+B.+Sells&title=Characterization+of+Alisertib+%28Mln8237%29%2C+an+Investigational+Small-Molecule+Inhibitor+of+Aurora+a+Kinase+Using+Novel+in+Vivo+Pharmacodynamic+Assays&doi=10.1158%2F1078-0432.CCR-11-1536"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of Alisertib (MLN8237), an Investigational Small-Molecule Inhibitor of Aurora A Kinase Using Novel In Vivo Pharmacodynamic Assays</span></div><div class="casAuthors">Manfredi, Mark G.; Ecsedy, Jeffrey A.; Chakravarty, Arijit; Silverman, Lee; Zhang, Mengkun; Hoar, Kara M.; Stroud, Stephen G.; Chen, Wei; Shinde, Vaishali; Huck, Jessica J.; Wysong, Deborah R.; Janowick, David A.; Hyer, Marc L.; LeRoy, Patrick J.; Gershman, Rachel E.; Silva, Matthew D.; Germanos, Melissa S.; Bolen, Joseph B.; Claiborne, Christopher F.; Sells, Todd B.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7614-7624</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PURPOSE: Small-mol. inhibitors of Aurora A (AAK) and B (ABK) kinases, which play important roles in mitosis, are currently being pursued in oncol. clin. trials.  We developed three novel assays to quant. measure biomarkers of AAK inhibition in vivo.  Here, we describe preclin. characterization of alisertib (MLN8237), a selective AAK inhibitor, incorporating these novel pharmacodynamic assays.  Exptl. Design: We investigated the selectivity of alisertib for AAK and ABK and studied the antitumor and antiproliferative activity of alisertib in vitro and in vivo.  Novel assays were used to assess chromosome alignment and mitotic spindle bipolarity in human tumor xenografts using immunofluorescent detection of DNA and alpha-tubulin, resp.  In addn., 18F-3'-fluoro-3'-deoxy-L-thymidine positron emission tomog. (FLT-PET) was used to noninvasively measure effects of alisertib on in vivo tumor cell proliferation.  RESULTS: Alisertib inhibited AAK over ABK with a selectivity of more than 200-fold in cells and produced a dose-dependent decrease in bipolar and aligned chromosomes in the HCT-116 xenograft model, a phenotype consistent with AAK inhibition.  Alisertib inhibited proliferation of human tumor cell lines in vitro and produced tumor growth inhibition in solid tumor xenograft models and regressions in in vivo lymphoma models.  In addn., a dose of alisertib that caused tumor stasis, as measured by vol., resulted in a decrease in FLT uptake, suggesting that noninvasive imaging could provide value over traditional measurements of response.  CONCLUSIONS: Alisertib is a selective and potent inhibitor of AAK.  The novel methods of measuring Aurora A pathway inhibition and application of tumor imaging described here may be valuable for clin. evaluation of small-mol. inhibitors.  Clin Cancer Res; 17(24); 7614-24.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYGGGKlJwzprVg90H21EOLACvtfcHk0liD6fDZc4Q9Pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1CqtrjL&md5=de00b44d9e6cd58a2a642770c3121128</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-11-1536&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-11-1536%26sid%3Dliteratum%253Aachs%26aulast%3DManfredi%26aufirst%3DM.%2BG.%26aulast%3DEcsedy%26aufirst%3DJ.%2BA.%26aulast%3DChakravarty%26aufirst%3DA.%26aulast%3DSilverman%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DHoar%26aufirst%3DK.%2BM.%26aulast%3DStroud%26aufirst%3DS.%2BG.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DShinde%26aufirst%3DV.%26aulast%3DHuck%26aufirst%3DJ.%2BJ.%26aulast%3DWysong%26aufirst%3DD.%2BR.%26aulast%3DJanowick%26aufirst%3DD.%2BA.%26aulast%3DHyer%26aufirst%3DM.%2BL.%26aulast%3DLeroy%26aufirst%3DP.%2BJ.%26aulast%3DGershman%26aufirst%3DR.%2BE.%26aulast%3DSilva%26aufirst%3DM.%2BD.%26aulast%3DGermanos%26aufirst%3DM.%2BS.%26aulast%3DBolen%26aufirst%3DJ.%2BB.%26aulast%3DClaiborne%26aufirst%3DC.%2BF.%26aulast%3DSells%26aufirst%3DT.%2BB.%26atitle%3DCharacterization%2520of%2520Alisertib%2520%2528Mln8237%2529%252C%2520an%2520Investigational%2520Small-Molecule%2520Inhibitor%2520of%2520Aurora%2520a%2520Kinase%2520Using%2520Novel%2520in%2520Vivo%2520Pharmacodynamic%2520Assays%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2011%26volume%3D17%26spage%3D7614%26epage%3D7624%26doi%3D10.1158%2F1078-0432.CCR-11-1536" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Hoar, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chakravarty, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabino, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wysong, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowman, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roy, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ecsedy, J. A.</span><span> </span><span class="NLM_article-title">Mln8054, a Small-Molecule Inhibitor of Aurora a, Causes Spindle Pole and Chromosome Congression Defects Leading to Aneuploidy</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">4513</span><span class="NLM_x">–</span> <span class="NLM_lpage">4525</span><span class="refDoi"> DOI: 10.1128/MCB.02364-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1128%2FMCB.02364-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=17438137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmvFansbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=4513-4525&author=K.+Hoarauthor=A.+Chakravartyauthor=C.+Rabinoauthor=D.+Wysongauthor=D.+Bowmanauthor=N.+Royauthor=J.+A.+Ecsedy&title=Mln8054%2C+a+Small-Molecule+Inhibitor+of+Aurora+a%2C+Causes+Spindle+Pole+and+Chromosome+Congression+Defects+Leading+to+Aneuploidy&doi=10.1128%2FMCB.02364-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">MLN8054, a small-molecule inhibitor of Aurora A, causes spindle pole and chromosome congress-ion defects leading to aneuploidy</span></div><div class="casAuthors">Hoar, Kara; Chakravarty, Arijit; Rabino, Claudia; Wysong, Deborah; Bowman, Douglas; Roy, Natalie; Ecsedy, Jeffrey A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4513-4525</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Aurora A kinase plays an essential role in the proper assembly and function of the mitotic spindle, as its perturbation causes defects in centrosome sepn., spindle pole organization, and chromosome congression.  Moreover, Aurora A disruption leads to cell death via a mechanism that involves aneuploidy generation.  However, the link between the immediate functional consequences of Aurora A inhibition and the development of aneuploidy is not clearly defined.  In this study, we delineate the sequence of events that lead to aneuploidy following Aurora A inhibition using MLN8054, a selective Aurora A small-mol. inhibitor.  Human tumor cells treated with MLN8054 show a high incidence of abnormal mitotic spindles, often with unsepd. centrosomes.  Although these spindle defects result in mitotic delays, cells ultimately divide at a frequency near that of untreated cells.  We show that many of the spindles in the dividing cells are bipolar, although they lack centrosomes at one or more spindle poles.  MLN8054-treated cells frequently show alignment defects during metaphase, lagging chromosomes in anaphase, and chromatin bridges during telophase.  Consistent with the chromosome segregation defects, cells treated with MLN8054 develop aneuploidy over time.  Taken together, these results suggest that Aurora A inhibition kills tumor cells through the development of deleterious aneuploidy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhlzReVziYI7Vg90H21EOLACvtfcHk0liD6fDZc4Q9Pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmvFansbs%253D&md5=0c86835e0c42615fd69416856a05cdc6</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1128%2FMCB.02364-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.02364-06%26sid%3Dliteratum%253Aachs%26aulast%3DHoar%26aufirst%3DK.%26aulast%3DChakravarty%26aufirst%3DA.%26aulast%3DRabino%26aufirst%3DC.%26aulast%3DWysong%26aufirst%3DD.%26aulast%3DBowman%26aufirst%3DD.%26aulast%3DRoy%26aufirst%3DN.%26aulast%3DEcsedy%26aufirst%3DJ.%2BA.%26atitle%3DMln8054%252C%2520a%2520Small-Molecule%2520Inhibitor%2520of%2520Aurora%2520a%252C%2520Causes%2520Spindle%2520Pole%2520and%2520Chromosome%2520Congression%2520Defects%2520Leading%2520to%2520Aneuploidy%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2007%26volume%3D27%26spage%3D4513%26epage%3D4525%26doi%3D10.1128%2FMCB.02364-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Bemis, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span> </span><span class="NLM_article-title">The Properties of Known Drugs. 1. Molecular Frameworks</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">2887</span><span class="NLM_x">–</span> <span class="NLM_lpage">2893</span><span class="refDoi"> DOI: 10.1021/jm9602928</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9602928" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2887-2893&author=G.+W.+Bemisauthor=M.+A.+Murcko&title=The+Properties+of+Known+Drugs.+1.+Molecular+Frameworks&doi=10.1021%2Fjm9602928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">The Properties of Known Drugs. 1. Molecular Frameworks</span></div><div class="casAuthors">Bemis, Guy W.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2887-2893</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the common features present in drug mols., we use shape description methods to analyze a database of com. available drugs and prep. a list of common drug shapes.  A useful way of organizing this structural data is to group the atoms of each drug mol. into ring, linker, framework, and side chain atoms.  On the basis of the two-dimensional mol. structures (without regard to atom type, hybridization, and bond order), there are 1179 different frameworks among the 5120 compds. analyzed.  However, the shapes of half of the drugs in the database are described by the 32 most frequently occurring frameworks.  This suggests that the diversity of shapes in the set of known drugs is extremely low.  In our second method of anal., in which atom type, hybridization, and bond order are considered, more diversity is seen; there are 2506 different frameworks among the 5120 compds. in the database, and the most frequently occurring 42 frameworks account for only one-fourth of the drugs.  We discuss the possible interpretations of these findings and the way they may be used to guide future drug discovery research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoodEb3sfDSdrVg90H21EOLACvtfcHk0lib4CY1Osvl5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D&md5=5e2c4fdfea9434456a0cca83de4185b3</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm9602928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9602928%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DG.%2BW.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DThe%2520Properties%2520of%2520Known%2520Drugs.%25201.%2520Molecular%2520Frameworks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2887%26epage%3D2893%26doi%3D10.1021%2Fjm9602928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Schuffenhauer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ertl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roggo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wetzel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koch, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldmann, H.</span><span> </span><span class="NLM_article-title">The Scaffold Tree – Visualization of the Scaffold Universe by Hierarchical Scaffold Classification</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">47</span><span class="NLM_x">–</span> <span class="NLM_lpage">58</span><span class="refDoi"> DOI: 10.1021/ci600338x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci600338x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlCit7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=47-58&author=A.+Schuffenhauerauthor=P.+Ertlauthor=S.+Roggoauthor=S.+Wetzelauthor=M.+A.+Kochauthor=H.+Waldmann&title=The+Scaffold+Tree+%E2%80%93+Visualization+of+the+Scaffold+Universe+by+Hierarchical+Scaffold+Classification&doi=10.1021%2Fci600338x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The Scaffold Tree - Visualization of the Scaffold Universe by Hierarchical Scaffold Classification</span></div><div class="casAuthors">Schuffenhauer, Ansgar; Ertl, Peter; Roggo, Silvio; Wetzel, Stefan; Koch, Marcus A.; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-58</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A hierarchical classification of chem. scaffolds (mol. framework, which is obtained by pruning all terminal side chains) has been introduced.  The mol. frameworks form the leaf nodes in the hierarchy trees.  By an iterative removal of rings, scaffolds forming the higher levels in the hierarchy tree are obtained.  Prioritization rules ensure that less characteristic, peripheral rings are removed first.  All scaffolds in the hierarchy tree are well-defined chem. entities making the classification chem. intuitive.  The classification is deterministic, data-set-independent, and scales linearly with the no. of compds. included in the data set.  The application of the classification is demonstrated on two data sets extd. from the PubChem database, namely, pyruvate kinase binders and a collection of pesticides.  The examples shown demonstrate that the classification procedure handles robustly synthetic structures and natural products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8XfR2xvGasbVg90H21EOLACvtfcHk0lib4CY1Osvl5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlCit7jO&md5=89cec5a01a37802ab0826ecababfaf0a</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fci600338x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600338x%26sid%3Dliteratum%253Aachs%26aulast%3DSchuffenhauer%26aufirst%3DA.%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DRoggo%26aufirst%3DS.%26aulast%3DWetzel%26aufirst%3DS.%26aulast%3DKoch%26aufirst%3DM.%2BA.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DThe%2520Scaffold%2520Tree%2520%25E2%2580%2593%2520Visualization%2520of%2520the%2520Scaffold%2520Universe%2520by%2520Hierarchical%2520Scaffold%2520Classification%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D47%26epage%3D58%26doi%3D10.1021%2Fci600338x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Welsch, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snyder, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stockwell, B. R.</span><span> </span><span class="NLM_article-title">Privileged Scaffolds for Library Design and Drug Discovery</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">347</span><span class="NLM_x">–</span> <span class="NLM_lpage">361</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2010.02.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2Fj.cbpa.2010.02.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=20303320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=347-361&author=M.+E.+Welschauthor=S.+A.+Snyderauthor=B.+R.+Stockwell&title=Privileged+Scaffolds+for+Library+Design+and+Drug+Discovery&doi=10.1016%2Fj.cbpa.2010.02.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Privileged scaffolds for library design and drug discovery</span></div><div class="casAuthors">Welsch, Matthew E.; Snyder, Scott A.; Stockwell, Brent R.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">347-361</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This review explores the concept of using privileged scaffolds to identify biol. active compds. through building chem. libraries.  We hope to accomplish three main objectives: to provide one of the most comprehensive listings of privileged scaffolds; to reveal through four selected examples the present state of the art in privileged scaffold library synthesis (in hopes of inspiring new and even more creative approaches); and also to offer some thoughts on how new privileged scaffolds might be identified and exploited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWQPxdsWawiLVg90H21EOLACvtfcHk0ljv16_3SVe6yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsLs%253D&md5=fd0c2a642f063fd6f42f2c34fd186f8a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2010.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2010.02.018%26sid%3Dliteratum%253Aachs%26aulast%3DWelsch%26aufirst%3DM.%2BE.%26aulast%3DSnyder%26aufirst%3DS.%2BA.%26aulast%3DStockwell%26aufirst%3DB.%2BR.%26atitle%3DPrivileged%2520Scaffolds%2520for%2520Library%2520Design%2520and%2520Drug%2520Discovery%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D14%26spage%3D347%26epage%3D361%26doi%3D10.1016%2Fj.cbpa.2010.02.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Aronov, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McClain, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moody, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span> </span><span class="NLM_article-title">Kinase-Likeness and Kinase-Privileged Fragments: Toward Virtual Polypharmacology</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1214</span><span class="NLM_x">–</span> <span class="NLM_lpage">1222</span><span class="refDoi"> DOI: 10.1021/jm701021b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm701021b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=1214-1222&author=A.+M.+Aronovauthor=B.+McClainauthor=C.+S.+Moodyauthor=M.+A.+Murcko&title=Kinase-Likeness+and+Kinase-Privileged+Fragments%3A+Toward+Virtual+Polypharmacology&doi=10.1021%2Fjm701021b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm701021b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701021b%26sid%3Dliteratum%253Aachs%26aulast%3DAronov%26aufirst%3DA.%2BM.%26aulast%3DMcClain%26aufirst%3DB.%26aulast%3DMoody%26aufirst%3DC.%2BS.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DKinase-Likeness%2520and%2520Kinase-Privileged%2520Fragments%253A%2520Toward%2520Virtual%2520Polypharmacology%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D1214%26epage%3D1222%26doi%3D10.1021%2Fjm701021b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Yan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, J.</span><span> </span><span class="NLM_article-title">Aurora-a Kinase Inhibitor Scaffolds and Binding Modes</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">260</span><span class="NLM_x">–</span> <span class="NLM_lpage">269</span><span class="refDoi"> DOI: 10.1016/j.drudis.2010.12.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2Fj.drudis.2010.12.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=21147253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjs1WrtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=260-269&author=A.+Yanauthor=L.+Wangauthor=S.+Xuauthor=J.+Xu&title=Aurora-a+Kinase+Inhibitor+Scaffolds+and+Binding+Modes&doi=10.1016%2Fj.drudis.2010.12.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Aurora-A kinase inhibitor scaffolds and binding modes</span></div><div class="casAuthors">Yan, Aixia; Wang, Liyu; Xu, Shuyu; Xu, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5/6</span>),
    <span class="NLM_cas:pages">260-269</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Aurora kinases (A-C) belong to the serine/threonine protein kinase family.  In recent years, the constitutive or elevated expression of Aurora kinases has been found in cancer cells and oncogene transfected cells.  In this review, we summarize the common binding modes of Aurora-A kinase inhibitors, the hot spot residues in the binding sites and the privileged inhibitor structures.  Our review of the reported chem. scaffolds of Aurora-A kinase inhibitors and their binding modes could provide a useful framework from which new design strategies for inhibitors might be assessed or developed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqi7pOvViuZ0rVg90H21EOLACvtfcHk0ljv16_3SVe6yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjs1WrtLk%253D&md5=5c3cf3953c13de2af4596348f10432de</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2010.12.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2010.12.003%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DJ.%26atitle%3DAurora-a%2520Kinase%2520Inhibitor%2520Scaffolds%2520and%2520Binding%2520Modes%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2011%26volume%3D16%26spage%3D260%26epage%3D269%26doi%3D10.1016%2Fj.drudis.2010.12.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Uitdehaag, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verkaar, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alwan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Man, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buijsman, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaman, G. J.</span><span> </span><span class="NLM_article-title">A Guide to Picking the Most Selective Kinase Inhibitor Tool Compounds for Pharmacological Validation of Drug Targets</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">166</span><span class="NLM_x">, </span> <span class="NLM_fpage">858</span><span class="NLM_x">–</span> <span class="NLM_lpage">876</span><span class="refDoi"> DOI: 10.1111/j.1476-5381.2012.01859.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1111%2Fj.1476-5381.2012.01859.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=22250956" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC38XotlWmsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2012&pages=858-876&author=J.+C.+Uitdehaagauthor=F.+Verkaarauthor=H.+Alwanauthor=J.+de+Manauthor=R.+C.+Buijsmanauthor=G.+J.+Zaman&title=A+Guide+to+Picking+the+Most+Selective+Kinase+Inhibitor+Tool+Compounds+for+Pharmacological+Validation+of+Drug+Targets&doi=10.1111%2Fj.1476-5381.2012.01859.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A guide to picking the most selective kinase inhibitor tool compounds for pharmacological validation of drug targets</span></div><div class="casAuthors">Uitdehaag, Joost C. M.; Verkaar, Folkert; Alwan, Husam; de Man, Jos; Buijsman, Rogier C.; Zaman, Guido J. R.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">166</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">858-876</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  To establish the druggability of a target, genetic validation needs to be supplemented with pharmacol. validation.  Pharmacol. studies, esp. in the kinase field, are hampered by the fact that many ref. inhibitors are not fully selective for one target.  Fortunately, the initial trickle of selective inhibitors released in the public domain has steadily swelled into a stream.  However, rationally picking the most selective tool compd. out of the increasing amts. of available inhibitors has become progressively difficult due to the lack of accurate quant. descriptors of drug selectivity.  A recently published approach, termed "selectivity entropy", is an improved way of expressing selectivity as a single-value parameter and enables rank ordering of inhibitors.  We provide a guide to select the best tool compds. for pharmacol. validation expts. of candidate drug targets using selectivity entropy.  In addn., we recommend which inhibitors to use for studying the biol. of the 20 most investigated kinases that are clin. relevant: Abl (ABL1), AKT1, ALK, Aurora A/B, CDKs, MET, CSF1R (FMS), EGFR, FLT3, ERBB2 (HER2), IKBKB (IKK2), JAK2/3, JNK1/2/3 (MAPK8/9/10), MEK1/2, PLK1, PI3Ks, p38α (MAPK14), BRAF, SRC and VEGFR2 (KDR).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtE6alezH-ArVg90H21EOLACvtfcHk0ljv16_3SVe6yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XotlWmsLw%253D&md5=26238277768843c7f6e44f037dcfe174</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2012.01859.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2012.01859.x%26sid%3Dliteratum%253Aachs%26aulast%3DUitdehaag%26aufirst%3DJ.%2BC.%26aulast%3DVerkaar%26aufirst%3DF.%26aulast%3DAlwan%26aufirst%3DH.%26aulast%3Dde%2BMan%26aufirst%3DJ.%26aulast%3DBuijsman%26aufirst%3DR.%2BC.%26aulast%3DZaman%26aufirst%3DG.%2BJ.%26atitle%3DA%2520Guide%2520to%2520Picking%2520the%2520Most%2520Selective%2520Kinase%2520Inhibitor%2520Tool%2520Compounds%2520for%2520Pharmacological%2520Validation%2520of%2520Drug%2520Targets%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2012%26volume%3D166%26spage%3D858%26epage%3D876%26doi%3D10.1111%2Fj.1476-5381.2012.01859.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Hu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Exploring the Scaffold Universe of Kinase Inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">315</span><span class="NLM_x">–</span> <span class="NLM_lpage">332</span><span class="refDoi"> DOI: 10.1021/jm501237k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501237k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=315-332&author=Y.+Huauthor=J.+Bajorath&title=Exploring+the+Scaffold+Universe+of+Kinase+Inhibitors&doi=10.1021%2Fjm501237k"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm501237k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501237k%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DExploring%2520the%2520Scaffold%2520Universe%2520of%2520Kinase%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D315%26epage%3D332%26doi%3D10.1021%2Fjm501237k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Gaulton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellis, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bento, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hersey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Light, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGlinchey, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michalovich, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Lazikani, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span> </span><span class="NLM_article-title">Chembl: A Large-Scale Bioactivity Database for Drug Discovery</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">D1100</span><span class="NLM_x">–</span> <span class="NLM_lpage">D1107</span><span class="refDoi"> DOI: 10.1093/nar/gkr777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1093%2Fnar%2Fgkr777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=21948594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D1100-D1107&author=A.+Gaultonauthor=L.+J.+Bellisauthor=A.+P.+Bentoauthor=J.+Chambersauthor=M.+Daviesauthor=A.+Herseyauthor=Y.+Lightauthor=S.+McGlincheyauthor=D.+Michalovichauthor=B.+Al-Lazikaniauthor=J.+P.+Overington&title=Chembl%3A+A+Large-Scale+Bioactivity+Database+for+Drug+Discovery&doi=10.1093%2Fnar%2Fgkr777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">ChEMBL: a large-scale bioactivity database for drug discovery</span></div><div class="casAuthors">Gaulton, Anna; Bellis, Louisa J.; Bento, A. Patricia; Chambers, Jon; Davies, Mark; Hersey, Anne; Light, Yvonne; McGlinchey, Shaun; Michalovich, David; Al-Lazikani, Bissan; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1100-D1107</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an Open Data database contg. binding, functional and ADMET information for a large no. of drug-like bioactive compds.  These data are manually abstracted from the primary published literature on a regular basis, then further curated and standardized to maximize their quality and utility across a wide range of chem. biol. and drug-discovery research problems.  Currently, the database contains 5.4 million bioactivity measurements for more than 1 million compds. and 5200 protein targets.  Access is available through a web-based interface, data downloads and web services at: https://www.ebi.ac.uk/chembldb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9e9jRENTtubVg90H21EOLACvtfcHk0li4tf0VrTdexg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN&md5=aedf7793e1ca54b6a4fa272ea3ef7d0e</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr777%26sid%3Dliteratum%253Aachs%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DMichalovich%26aufirst%3DD.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DChembl%253A%2520A%2520Large-Scale%2520Bioactivity%2520Database%2520for%2520Drug%2520Discovery%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DD1100%26epage%3DD1107%26doi%3D10.1093%2Fnar%2Fgkr777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Simonin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Awale, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brand, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Deursen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fine, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kovacs, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Häfliger, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gyimesi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sithampari, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charles, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hediger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reymond, J. L.</span><span> </span><span class="NLM_article-title">Optimization of Trpv6 Calcium Channel Inhibitors Using a New 3d Ligand Based Virtual Screening Method</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">14748</span><span class="NLM_x">–</span> <span class="NLM_lpage">14752</span><span class="refDoi"> DOI: 10.1002/anie.201507320</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1002%2Fanie.201507320" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=14748-14752&author=C.+Simoninauthor=M.+Awaleauthor=M.+Brandauthor=R.+van+Deursenauthor=J.+Schwartzauthor=M.+Fineauthor=G.+Kovacsauthor=P.+H%C3%A4fligerauthor=G.+Gyimesiauthor=A.+Sithampariauthor=R.+P.+Charlesauthor=M.+Hedigerauthor=J.+L.+Reymond&title=Optimization+of+Trpv6+Calcium+Channel+Inhibitors+Using+a+New+3d+Ligand+Based+Virtual+Screening+Method&doi=10.1002%2Fanie.201507320"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507320&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507320%26sid%3Dliteratum%253Aachs%26aulast%3DSimonin%26aufirst%3DC.%26aulast%3DAwale%26aufirst%3DM.%26aulast%3DBrand%26aufirst%3DM.%26aulast%3Dvan%2BDeursen%26aufirst%3DR.%26aulast%3DSchwartz%26aufirst%3DJ.%26aulast%3DFine%26aufirst%3DM.%26aulast%3DKovacs%26aufirst%3DG.%26aulast%3DH%25C3%25A4fliger%26aufirst%3DP.%26aulast%3DGyimesi%26aufirst%3DG.%26aulast%3DSithampari%26aufirst%3DA.%26aulast%3DCharles%26aufirst%3DR.%2BP.%26aulast%3DHediger%26aufirst%3DM.%26aulast%3DReymond%26aufirst%3DJ.%2BL.%26atitle%3DOptimization%2520of%2520Trpv6%2520Calcium%2520Channel%2520Inhibitors%2520Using%2520a%2520New%25203d%2520Ligand%2520Based%2520Virtual%2520Screening%2520Method%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D14748%26epage%3D14752%26doi%3D10.1002%2Fanie.201507320" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Kearsley, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, G.</span><span> </span><span class="NLM_article-title">An Alternative Method for the Alignment of Molecular Structures: Maximizing Electrostatic and Steric Overlap</span> <span class="citation_source-journal">Tetrahedron Comput. Methodol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">615</span><span class="NLM_x">–</span> <span class="NLM_lpage">633</span><span class="refDoi"> DOI: 10.1016/0898-5529(90)90162-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2F0898-5529%2890%2990162-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADyaK3sXmtlGj" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1990&pages=615-633&author=S.+K.+Kearsleyauthor=G.+Smith&title=An+Alternative+Method+for+the+Alignment+of+Molecular+Structures%3A+Maximizing+Electrostatic+and+Steric+Overlap&doi=10.1016%2F0898-5529%2890%2990162-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">An alternative method for the alignment of molecular structures:  maximizing electrostatic and steric overlap</span></div><div class="casAuthors">Kearsley, Simon K.; Smith, Graham M.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Computer Methodology</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6c</span>),
    <span class="NLM_cas:pages">615-33</span>CODEN:
                <span class="NLM_cas:coden">TCMTE6</span>;
        ISSN:<span class="NLM_cas:issn">0898-5529</span>.
    </div><div class="casAbstract">The SEAL method has been developed which will optimize the alignment of two 3-dimensional structures using their at. partial charges and steric vols. as factors.  In addn., this method will perform the superimposition many times using randomly generated starting configurations and keep only the best unique results based on the value of the alignment function.  The computer generated alignments of methotrexate and dihydrofolic acid are compared with the alignments found in the active site of dihydrfolate reductase.  The marine neurotoxins, saxitoxin and tetrodotoxin, are aligned and the results compared to other alignment techniques.  The relationship of the SEAL alignments to the potential field error is explored.  The program is on disk.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFtoxyn792HbVg90H21EOLACvtfcHk0li-YufoWWG-DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXmtlGj&md5=f03505ee6ce9275af178be0e7ba29f7e</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2F0898-5529%2890%2990162-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0898-5529%252890%252990162-2%26sid%3Dliteratum%253Aachs%26aulast%3DKearsley%26aufirst%3DS.%2BK.%26aulast%3DSmith%26aufirst%3DG.%26atitle%3DAn%2520Alternative%2520Method%2520for%2520the%2520Alignment%2520of%2520Molecular%2520Structures%253A%2520Maximizing%2520Electrostatic%2520and%2520Steric%2520Overlap%26jtitle%3DTetrahedron%2520Comput.%2520Methodol.%26date%3D1990%26volume%3D3%26spage%3D615%26epage%3D633%26doi%3D10.1016%2F0898-5529%2890%2990162-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Kennewell, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ducrot, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luttmann, C.</span><span> </span><span class="NLM_article-title">Identification of Target-Specific Bioisosteric Fragments from Ligand-Protein Crystallographic Data</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">385</span><span class="NLM_x">–</span> <span class="NLM_lpage">394</span><span class="refDoi"> DOI: 10.1007/s10822-006-9072-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1007%2Fs10822-006-9072-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2006&pages=385-394&author=E.+A.+Kennewellauthor=P.+Willettauthor=P.+Ducrotauthor=C.+Luttmann&title=Identification+of+Target-Specific+Bioisosteric+Fragments+from+Ligand-Protein+Crystallographic+Data&doi=10.1007%2Fs10822-006-9072-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1007%2Fs10822-006-9072-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-006-9072-0%26sid%3Dliteratum%253Aachs%26aulast%3DKennewell%26aufirst%3DE.%2BA.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DDucrot%26aufirst%3DP.%26aulast%3DLuttmann%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520Target-Specific%2520Bioisosteric%2520Fragments%2520from%2520Ligand-Protein%2520Crystallographic%2520Data%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2006%26volume%3D20%26spage%3D385%26epage%3D394%26doi%3D10.1007%2Fs10822-006-9072-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Awale, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reymond, J. L.</span><span> </span><span class="NLM_article-title">Atom Pair 2d-Fingerprints Perceive 3d-Molecular Shape and Pharmacophores for Very Fast Virtual Screening of Zinc and Gdb-17</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1892</span><span class="NLM_x">–</span> <span class="NLM_lpage">1897</span><span class="refDoi"> DOI: 10.1021/ci500232g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500232g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpvFCltr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1892-1897&author=M.+Awaleauthor=J.+L.+Reymond&title=Atom+Pair+2d-Fingerprints+Perceive+3d-Molecular+Shape+and+Pharmacophores+for+Very+Fast+Virtual+Screening+of+Zinc+and+Gdb-17&doi=10.1021%2Fci500232g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Atom Pair 2D-Fingerprints Perceive 3D-Molecular Shape and Pharmacophores for Very Fast Virtual Screening of ZINC and GDB-17</span></div><div class="casAuthors">Awale, Mahendra; Reymond, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1892-1907</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Three-dimensional (3D) mol. shape and pharmacophores are important determinants of the biol. activity of org. mols.; however, a precise computation of 3D-shape is generally too slow for virtual screening of very large databases.  A reinvestigation of the concept of atom pairs initially reported by Carhart et al. and extended by Schneider et al. showed that a simple atom pair fingerprint (APfp) counting atom pairs at increasing topol. distances in 2D-structures without atom property assignment correlates with various representations of mol. shape extd. from the 3D-structures.  A related 55-dimensional atom pair fingerprint extended with atom properties (Xfp) provided an efficient pharmacophore fingerprint with good performance for ligand-based virtual screening such as the recovery of active compds. from decoys in DUD, and overlap with the ROCS 3D-pharmacophore scoring function.  The APfp and Xfp data were organized for web-based extremely fast nearest-neighbor searching in ZINC (13.5 M compds.) and GDB-17 (50 M random subset) freely accessible at www.gdb.unibe.ch.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiXcOsBelbq7Vg90H21EOLACvtfcHk0li-YufoWWG-DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpvFCltr4%253D&md5=f6a994a6c16db780b485087fd3925be7</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fci500232g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500232g%26sid%3Dliteratum%253Aachs%26aulast%3DAwale%26aufirst%3DM.%26aulast%3DReymond%26aufirst%3DJ.%2BL.%26atitle%3DAtom%2520Pair%25202d-Fingerprints%2520Perceive%25203d-Molecular%2520Shape%2520and%2520Pharmacophores%2520for%2520Very%2520Fast%2520Virtual%2520Screening%2520of%2520Zinc%2520and%2520Gdb-17%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1892%26epage%3D1897%26doi%3D10.1021%2Fci500232g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Awale, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reymond, J. L.</span><span> </span><span class="NLM_article-title">Stereoselective Virtual Screening of the Zinc Database Using Atom Pair 3d-Fingerprints</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="refDoi"> DOI: 10.1186/s13321-014-0051-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1186%2Fs13321-014-0051-5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=3&author=M.+Awaleauthor=X.+Jinauthor=J.+L.+Reymond&title=Stereoselective+Virtual+Screening+of+the+Zinc+Database+Using+Atom+Pair+3d-Fingerprints&doi=10.1186%2Fs13321-014-0051-5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1186%2Fs13321-014-0051-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-014-0051-5%26sid%3Dliteratum%253Aachs%26aulast%3DAwale%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DX.%26aulast%3DReymond%26aufirst%3DJ.%2BL.%26atitle%3DStereoselective%2520Virtual%2520Screening%2520of%2520the%2520Zinc%2520Database%2520Using%2520Atom%2520Pair%25203d-Fingerprints%26jtitle%3DJ.%2520Cheminf.%26date%3D2015%26volume%3D7%26spage%3D3%26doi%3D10.1186%2Fs13321-014-0051-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Montalbetti, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simonin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simonin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Awale, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reymond, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hediger, M. A.</span><span> </span><span class="NLM_article-title">Discovery and Characterization of a Novel Non-Competitive Inhibitor of the Divalent Metal Transporter Dmt1/Slc11a2</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">96</span><span class="NLM_x">, </span> <span class="NLM_fpage">216</span><span class="NLM_x">–</span> <span class="NLM_lpage">224</span><span class="refDoi"> DOI: 10.1016/j.bcp.2015.05.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2Fj.bcp.2015.05.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=26047847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWgtrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=2015&pages=216-224&author=N.+Montalbettiauthor=A.+Simoninauthor=C.+Simoninauthor=M.+Awaleauthor=J.+L.+Reymondauthor=M.+A.+Hediger&title=Discovery+and+Characterization+of+a+Novel+Non-Competitive+Inhibitor+of+the+Divalent+Metal+Transporter+Dmt1%2FSlc11a2&doi=10.1016%2Fj.bcp.2015.05.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of a novel non-competitive inhibitor of the divalent metal transporter DMT1/SLC11A2</span></div><div class="casAuthors">Montalbetti, Nicolas; Simonin, Alexandre; Simonin, Celine; Awale, Mahendra; Reymond, Jean-Louis; Hediger, Matthias A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">216-224</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Divalent metal transporter-1 (SLC11A2/DMT1) uses the H+ electrochem. gradient as the driving force to transport divalent metal ions such as Fe2+, Mn2+ and others metals into mammalian cells.  DMT1 is ubiquitously expressed, most notably in proximal duodenum, immature erythroid cells, brain and kidney.  This transporter mediates H+-coupled transport of ferrous iron across the apical membrane of enterocytes.  In addn., in cells such as to erythroid precursors, following transferrin receptor (TfR) mediated endocytosis; it mediates H+-coupled exit of ferrous iron from endocytic vesicles into the cytosol.  Dysfunction of human DMT1 is assocd. with several pathologies such as iron deficiency anemia hemochromatosis, Parkinson's disease and Alzheimer's disease, as well as colorectal cancer and esophageal adenocarcinoma, making DMT1 an attractive target for drug discovery.  In the present study, we performed a ligand-based virtual screening of the Princeton database (700,000 com. available compds.) to search for pharmacophore shape analogs of recently reported DMT1 inhibitors.  We discovered a new compd., named pyrimidinone 8, which mediates a reversible linear non-competitive inhibition of human DMT1 (hDMT1) transport activity with a Ki of ∼20 μM.  This compd. does not affect hDMT1 cell surface expression and shows no dependence on extracellular pH.  To our knowledge, this is the first exptl. evidence that hDMT1 can be allosterically modulated by pharmacol. agents.  Pyrimidinone 8 represents a novel versatile tool compd. and it may serve as a lead structure for the development of therapeutic compds. for pre-clin. assessment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCuVTNtEKIvLVg90H21EOLACvtfcHk0lg0ivaCzyY0AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWgtrnN&md5=dda3c8979ea2c053a409da1cae3db243</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2015.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2015.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DMontalbetti%26aufirst%3DN.%26aulast%3DSimonin%26aufirst%3DA.%26aulast%3DSimonin%26aufirst%3DC.%26aulast%3DAwale%26aufirst%3DM.%26aulast%3DReymond%26aufirst%3DJ.%2BL.%26aulast%3DHediger%26aufirst%3DM.%2BA.%26atitle%3DDiscovery%2520and%2520Characterization%2520of%2520a%2520Novel%2520Non-Competitive%2520Inhibitor%2520of%2520the%2520Divalent%2520Metal%2520Transporter%2520Dmt1%252FSlc11a2%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2015%26volume%3D96%26spage%3D216%26epage%3D224%26doi%3D10.1016%2Fj.bcp.2015.05.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sterling, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolstad, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span> </span><span class="NLM_article-title">Zinc: A Free Tool to Discover Chemistry for Biology</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1757</span><span class="NLM_x">–</span> <span class="NLM_lpage">1768</span><span class="refDoi"> DOI: 10.1021/ci3001277</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3001277" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1757-1768&author=J.+J.+Irwinauthor=T.+Sterlingauthor=M.+M.+Mysingerauthor=E.+S.+Bolstadauthor=R.+G.+Coleman&title=Zinc%3A+A+Free+Tool+to+Discover+Chemistry+for+Biology&doi=10.1021%2Fci3001277"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC: A Free Tool to Discover Chemistry for Biology</span></div><div class="casAuthors">Irwin, John J.; Sterling, Teague; Mysinger, Michael M.; Bolstad, Erin S.; Coleman, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1757-1768</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ZINC is a free public resource for ligand discovery.  The database contains over twenty million com. available mols. in biol. relevant representations that may be downloaded in popular ready-to-dock formats and subsets.  The Web site also enables searches by structure, biol. activity, phys. property, vendor, catalog no., name, and CAS no.  Small custom subsets may be created, edited, shared, docked, downloaded, and conveyed to a vendor for purchase.  The database is maintained and curated for a high purchasing success rate and is freely available at zinc.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotyhntxgvRpbVg90H21EOLACvtfcHk0lg0ivaCzyY0AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D&md5=97f2ede64afc6b5e3ea2f279e38e32a0</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fci3001277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3001277%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DBolstad%26aufirst%3DE.%2BS.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26atitle%3DZinc%253A%2520A%2520Free%2520Tool%2520to%2520Discover%2520Chemistry%2520for%2520Biology%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1757%26epage%3D1768%26doi%3D10.1021%2Fci3001277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Baell, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holloway, G. A.</span><span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (Pains) from Screening Libraries and for Their Exclusion in Bioassays</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2719</span><span class="NLM_x">–</span> <span class="NLM_lpage">2740</span><span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28Pains%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lielOUJ7SkFXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528Pains%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Sterling, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span> </span><span class="NLM_article-title">Zinc 15--Ligand Discovery for Everyone</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">2324</span><span class="NLM_x">–</span> <span class="NLM_lpage">2337</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00559</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00559" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1OhurbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2324-2337&author=T.+Sterlingauthor=J.+J.+Irwin&title=Zinc+15%2D%2DLigand+Discovery+for+Everyone&doi=10.1021%2Facs.jcim.5b00559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC 15 - Ligand Discovery for Everyone</span></div><div class="casAuthors">Sterling, Teague; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2324-2337</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many questions about the biol. activity and availability of small mols. remain inaccessible to investigators who could most benefit from their answers.  To narrow the gap between chemoinformatics and biol., we have developed a suite of ligand annotation, purchasability, target, and biol. assocn. tools, incorporated into ZINC and meant for investigators who are not computer specialists.  The new version contains over 120 million purchasable "drug-like" compds. - effectively all org. mols. that are for sale - a quarter of which are available for immediate delivery.  ZINC connects purchasable compds. to high-value ones such as metabolites, drugs, natural products, and annotated compds. from the literature.  Compds. may be accessed by the genes for which they are annotated as well as the major and minor target classes to which those genes belong.  It offers new anal. tools that are easy for nonspecialists yet with few limitations for experts.  ZINC retains its original 3D roots - all mols. are available in biol. relevant, ready-to-dock formats.  ZINC is freely available at http://zinc15.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWfkDnal3D3bVg90H21EOLACvtfcHk0lielOUJ7SkFXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1OhurbF&md5=e767c26c1d0aff7432c403c59a5d2b8b</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00559%26sid%3Dliteratum%253Aachs%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DZinc%252015--Ligand%2520Discovery%2520for%2520Everyone%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D2324%26epage%3D2337%26doi%3D10.1021%2Facs.jcim.5b00559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Degorce, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Card, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soh, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trinquet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapik, G. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, B.</span><span> </span><span class="NLM_article-title">Htrf: A Technology Tailored for Drug Discovery - a Review of Theoretical Aspects and Recent Applications</span> <span class="citation_source-journal">Curr. Chem. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">22</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span><span class="refDoi"> DOI: 10.2174/1875397300903010022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.2174%2F1875397300903010022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=20161833" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD1MXosFWnsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2009&pages=22-32&author=F.+Degorceauthor=A.+Cardauthor=S.+Sohauthor=E.+Trinquetauthor=G.+P.+Knapikauthor=B.+Xie&title=Htrf%3A+A+Technology+Tailored+for+Drug+Discovery+-+a+Review+of+Theoretical+Aspects+and+Recent+Applications&doi=10.2174%2F1875397300903010022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">HTRF: a technology tailored for drug discovery - a review of theoretical aspects and recent applications</span></div><div class="casAuthors">Degorce, Francois; Card, Amy; Soh, Sharon; Trinquet, Eric; Knapik, Glenn P.; Xie, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Current Chemical Genomics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">22-32</span>CODEN:
                <span class="NLM_cas:coden">CCGUAQ</span>;
        ISSN:<span class="NLM_cas:issn">1875-3973</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  HTRF (Homogeneous Time Resolved Fluorescence) is the most frequently used generic assay technol. to measure analytes in a homogenous format, which is the ideal platform used for drug target studies in high-throughput screening (HTS).  This technol. combines fluorescence resonance energy transfer technol. (FRET) with time-resolved measurement (TR).  In TR-FRET assays, a signal is generated through fluorescent resonance energy transfer between a donor and an acceptor mol. when in close proximity to each other.  Buffer and media interference is dramatically reduced by dual-wavelength detection, and the final signal is proportional to the extent of product formation.  The HTRF assay is usually sensitive and robust that can be miniaturized into the 384 and 1536-well plate formats.  This assay technol. has been applied to many antibody-based assays including GPCR signaling (cAMP and IP-One), kinases, cytokines and biomarkers, bioprocess (antibody and protein prodn.), as well as the assays for protein-protein, protein-peptide, and protein-DNA/RNA interactions.  Since its introduction to the drug-screening world over ten years ago, researchers have used HTRF to expedite the study of GPCRs, kinases, new biomarkers, protein-protein interactions, and other targets of interest.  HTRF has also been utilized as an alternative method for bioprocess monitoring.  The first-generation HTRF technol., which uses Europium cryptate as a fluorescence donor to monitor reactions between biomols., was extended in 2008 through the introduction of a second-generation donor, Terbium cryptate (Tb), enhancing screening performance.  Terbium cryptate possesses different photophys. properties compared to Europium, including increased quantum yield and a higher molar extinction coeff.  In addn. to being compatible with the same acceptor fluorophors used with Europium, it can serve as a donor fluorophore to green-emitting fluors because it has multiple emission peaks including one at 490 nm.  Moreover, all Terbium HTRF assays can be read on the same HTRF-compatible instruments as Europium HTRF assays.  Overall, HTRF is a highly sensitive, robust technol. for the detection of mol. interactions in vitro and is widely used for primary and secondary screening phases of drug development.  This review addresses the general principles of HTRF and its current applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKDP8Vd8k247Vg90H21EOLACvtfcHk0lielOUJ7SkFXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXosFWnsr4%253D&md5=9590149ceeed17ce424522ef85aa32f9</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.2174%2F1875397300903010022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1875397300903010022%26sid%3Dliteratum%253Aachs%26aulast%3DDegorce%26aufirst%3DF.%26aulast%3DCard%26aufirst%3DA.%26aulast%3DSoh%26aufirst%3DS.%26aulast%3DTrinquet%26aufirst%3DE.%26aulast%3DKnapik%26aufirst%3DG.%2BP.%26aulast%3DXie%26aufirst%3DB.%26atitle%3DHtrf%253A%2520A%2520Technology%2520Tailored%2520for%2520Drug%2520Discovery%2520-%2520a%2520Review%2520of%2520Theoretical%2520Aspects%2520and%2520Recent%2520Applications%26jtitle%3DCurr.%2520Chem.%2520Genomics%26date%3D2009%26volume%3D3%26spage%3D22%26epage%3D32%26doi%3D10.2174%2F1875397300903010022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Ke, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coumar, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiao, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsieh, H. P.</span><span> </span><span class="NLM_article-title">Ligand Efficiency Based Approach for Efficient Virtual Screening of Compound Libraries</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">226</span><span class="NLM_x">–</span> <span class="NLM_lpage">235</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2014.06.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2Fj.ejmech.2014.06.029" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2014&pages=226-235&author=Y.+Y.+Keauthor=M.+S.+Coumarauthor=H.+Y.+Shiaoauthor=W.+C.+Wangauthor=C.+W.+Chenauthor=J.+S.+Songauthor=C.+H.+Chenauthor=W.+H.+Linauthor=S.+H.+Wuauthor=J.+T.+Hsuauthor=C.+M.+Changauthor=H.+P.+Hsieh&title=Ligand+Efficiency+Based+Approach+for+Efficient+Virtual+Screening+of+Compound+Libraries&doi=10.1016%2Fj.ejmech.2014.06.029"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2014.06.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2014.06.029%26sid%3Dliteratum%253Aachs%26aulast%3DKe%26aufirst%3DY.%2BY.%26aulast%3DCoumar%26aufirst%3DM.%2BS.%26aulast%3DShiao%26aufirst%3DH.%2BY.%26aulast%3DWang%26aufirst%3DW.%2BC.%26aulast%3DChen%26aufirst%3DC.%2BW.%26aulast%3DSong%26aufirst%3DJ.%2BS.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DLin%26aufirst%3DW.%2BH.%26aulast%3DWu%26aufirst%3DS.%2BH.%26aulast%3DHsu%26aufirst%3DJ.%2BT.%26aulast%3DChang%26aufirst%3DC.%2BM.%26aulast%3DHsieh%26aufirst%3DH.%2BP.%26atitle%3DLigand%2520Efficiency%2520Based%2520Approach%2520for%2520Efficient%2520Virtual%2520Screening%2520of%2520Compound%2520Libraries%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2014%26volume%3D83%26spage%3D226%26epage%3D235%26doi%3D10.1016%2Fj.ejmech.2014.06.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Emanuel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rugg, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gruninger, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuentes-Pesquera, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Connolly, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wetter, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hollister, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Napier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jolliffe, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Middleton, S. A.</span><span> </span><span class="NLM_article-title">The in Vitro and in Vivo Effects of Jnj-7706621: A Dual Inhibitor of Cyclin-Dependent Kinases and Aurora Kinases</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">9038</span><span class="NLM_x">–</span> <span class="NLM_lpage">9046</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-05-0882</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1158%2F0008-5472.CAN-05-0882" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2005&pages=9038-9046&author=S.+Emanuelauthor=C.+A.+Ruggauthor=R.+H.+Gruningerauthor=R.+Linauthor=A.+Fuentes-Pesqueraauthor=P.+J.+Connollyauthor=S.+K.+Wetterauthor=B.+Hollisterauthor=W.+W.+Krugerauthor=C.+Napierauthor=L.+Jolliffeauthor=S.+A.+Middleton&title=The+in+Vitro+and+in+Vivo+Effects+of+Jnj-7706621%3A+A+Dual+Inhibitor+of+Cyclin-Dependent+Kinases+and+Aurora+Kinases&doi=10.1158%2F0008-5472.CAN-05-0882"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-05-0882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-05-0882%26sid%3Dliteratum%253Aachs%26aulast%3DEmanuel%26aufirst%3DS.%26aulast%3DRugg%26aufirst%3DC.%2BA.%26aulast%3DGruninger%26aufirst%3DR.%2BH.%26aulast%3DLin%26aufirst%3DR.%26aulast%3DFuentes-Pesquera%26aufirst%3DA.%26aulast%3DConnolly%26aufirst%3DP.%2BJ.%26aulast%3DWetter%26aufirst%3DS.%2BK.%26aulast%3DHollister%26aufirst%3DB.%26aulast%3DKruger%26aufirst%3DW.%2BW.%26aulast%3DNapier%26aufirst%3DC.%26aulast%3DJolliffe%26aufirst%3DL.%26aulast%3DMiddleton%26aufirst%3DS.%2BA.%26atitle%3DThe%2520in%2520Vitro%2520and%2520in%2520Vivo%2520Effects%2520of%2520Jnj-7706621%253A%2520A%2520Dual%2520Inhibitor%2520of%2520Cyclin-Dependent%2520Kinases%2520and%2520Aurora%2520Kinases%26jtitle%3DCancer%2520Res.%26date%3D2005%26volume%3D65%26spage%3D9038%26epage%3D9046%26doi%3D10.1158%2F0008-5472.CAN-05-0882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Wyatt, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodhead, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berdini, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boulstridge, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carr, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cross, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Devine, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Early, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feltell, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMenamin, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Navarro, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Reilly, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reule, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saxty, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seavers, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Squires, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trewartha, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolford, A. J.</span><span> </span><span class="NLM_article-title">Identification of N-(4-Piperidinyl)-4-(2,6-Dichlorobenzoylamino)-1h-Pyrazole-3-Carboxamide (At7519), a Novel Cyclin Dependent Kinase Inhibitor Using Fragment-Based X-Ray Crystallography and Structure Based Drug Design</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">4986</span><span class="NLM_x">–</span> <span class="NLM_lpage">4999</span><span class="refDoi"> DOI: 10.1021/jm800382h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800382h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=4986-4999&author=P.+G.+Wyattauthor=A.+J.+Woodheadauthor=V.+Berdiniauthor=J.+A.+Boulstridgeauthor=M.+G.+Carrauthor=D.+M.+Crossauthor=D.+J.+Davisauthor=L.+A.+Devineauthor=T.+R.+Earlyauthor=R.+E.+Feltellauthor=E.+J.+Lewisauthor=R.+L.+McMenaminauthor=E.+F.+Navarroauthor=M.+A.+O%E2%80%99Brienauthor=M.+O%E2%80%99Reillyauthor=M.+Reuleauthor=G.+Saxtyauthor=L.+C.+Seaversauthor=D.+M.+Smithauthor=M.+S.+Squiresauthor=G.+Trewarthaauthor=M.+T.+Walkerauthor=A.+J.+Woolford&title=Identification+of+N-%284-Piperidinyl%29-4-%282%2C6-Dichlorobenzoylamino%29-1h-Pyrazole-3-Carboxamide+%28At7519%29%2C+a+Novel+Cyclin+Dependent+Kinase+Inhibitor+Using+Fragment-Based+X-Ray+Crystallography+and+Structure+Based+Drug+Design&doi=10.1021%2Fjm800382h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm800382h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800382h%26sid%3Dliteratum%253Aachs%26aulast%3DWyatt%26aufirst%3DP.%2BG.%26aulast%3DWoodhead%26aufirst%3DA.%2BJ.%26aulast%3DBerdini%26aufirst%3DV.%26aulast%3DBoulstridge%26aufirst%3DJ.%2BA.%26aulast%3DCarr%26aufirst%3DM.%2BG.%26aulast%3DCross%26aufirst%3DD.%2BM.%26aulast%3DDavis%26aufirst%3DD.%2BJ.%26aulast%3DDevine%26aufirst%3DL.%2BA.%26aulast%3DEarly%26aufirst%3DT.%2BR.%26aulast%3DFeltell%26aufirst%3DR.%2BE.%26aulast%3DLewis%26aufirst%3DE.%2BJ.%26aulast%3DMcMenamin%26aufirst%3DR.%2BL.%26aulast%3DNavarro%26aufirst%3DE.%2BF.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DM.%2BA.%26aulast%3DO%25E2%2580%2599Reilly%26aufirst%3DM.%26aulast%3DReule%26aufirst%3DM.%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DSeavers%26aufirst%3DL.%2BC.%26aulast%3DSmith%26aufirst%3DD.%2BM.%26aulast%3DSquires%26aufirst%3DM.%2BS.%26aulast%3DTrewartha%26aufirst%3DG.%26aulast%3DWalker%26aufirst%3DM.%2BT.%26aulast%3DWoolford%26aufirst%3DA.%2BJ.%26atitle%3DIdentification%2520of%2520N-%25284-Piperidinyl%2529-4-%25282%252C6-Dichlorobenzoylamino%2529-1h-Pyrazole-3-Carboxamide%2520%2528At7519%2529%252C%2520a%2520Novel%2520Cyclin%2520Dependent%2520Kinase%2520Inhibitor%2520Using%2520Fragment-Based%2520X-Ray%2520Crystallography%2520and%2520Structure%2520Based%2520Drug%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D4986%26epage%3D4999%26doi%3D10.1021%2Fjm800382h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Deng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roessler, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brdar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faessler, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sales, Z. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mani, N. S.</span><span> </span><span class="NLM_article-title">Direct, Metal-Free Amination of Heterocyclic Amides/Ureas with Nh-Heterocycles and N-Substituted Anilines in Pocl<sub>3</sub></span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">76</span><span class="NLM_x">, </span> <span class="NLM_fpage">8262</span><span class="NLM_x">–</span> <span class="NLM_lpage">8269</span><span class="refDoi"> DOI: 10.1021/jo201425q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo201425q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFCgsrfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2011&pages=8262-8269&author=X.+Dengauthor=A.+Roesslerauthor=I.+Brdarauthor=R.+Faesslerauthor=J.+Wuauthor=Z.+S.+Salesauthor=N.+S.+Mani&title=Direct%2C+Metal-Free+Amination+of+Heterocyclic+Amides%2FUreas+with+Nh-Heterocycles+and+N-Substituted+Anilines+in+Pocl3&doi=10.1021%2Fjo201425q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Direct, metal-free amination of heterocyclic amides/ureas with NH-heterocycles and N-substituted anilines in POCl3</span></div><div class="casAuthors">Deng, Xiaohu; Roessler, Armin; Brdar, Ivana; Faessler, Roger; Wu, Jiejun; Sales, Zachary S.; Mani, Neelakandha S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8262-8269</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A POCl3-mediated, direct amination reaction of heterocyclic amides/ureas with NH-heterocycles or N-substituted anilines is described.  Compared to the existing methods, this operationally simple protocol provides unique reactivity and functional group compatibility because of the metal-free, acidic reaction conditions.  The yields are generally excellent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprHfZJEC5aPbVg90H21EOLACvtfcHk0lh2yBVJ7iRBQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFCgsrfM&md5=2407dead7288be61402476dfc7303cf9</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjo201425q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo201425q%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DRoessler%26aufirst%3DA.%26aulast%3DBrdar%26aufirst%3DI.%26aulast%3DFaessler%26aufirst%3DR.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DSales%26aufirst%3DZ.%2BS.%26aulast%3DMani%26aufirst%3DN.%2BS.%26atitle%3DDirect%252C%2520Metal-Free%2520Amination%2520of%2520Heterocyclic%2520Amides%252FUreas%2520with%2520Nh-Heterocycles%2520and%2520N-Substituted%2520Anilines%2520in%2520Pocl3%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2011%26volume%3D76%26spage%3D8262%26epage%3D8269%26doi%3D10.1021%2Fjo201425q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Davies, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tunnah, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meijer, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marko, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eisenbrand, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Endicott, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noble, M. E.</span><span> </span><span class="NLM_article-title">Inhibitor Binding to Active and Inactive Cdk2: The Crystal Structure of Cdk2-Cyclin a/Indirubin-5-Sulphonate</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">389</span><span class="NLM_x">–</span> <span class="NLM_lpage">397</span><span class="refDoi"> DOI: 10.1016/S0969-2126(01)00598-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2FS0969-2126%2801%2900598-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=11377199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjslCisLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2001&pages=389-397&author=T.+G.+Daviesauthor=P.+Tunnahauthor=L.+Meijerauthor=D.+Markoauthor=G.+Eisenbrandauthor=J.+A.+Endicottauthor=M.+E.+Noble&title=Inhibitor+Binding+to+Active+and+Inactive+Cdk2%3A+The+Crystal+Structure+of+Cdk2-Cyclin+a%2FIndirubin-5-Sulphonate&doi=10.1016%2FS0969-2126%2801%2900598-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitor Binding to Active and Inactive CDK2 The Crystal Structure of CDK2-Cyclin A/Indirubin-5-Sulphonate</span></div><div class="casAuthors">Davies, T. G.; Tunnah, P.; Meijer, L.; Marko, D.; Eisenbrand, G.; Endicott, J. A.; Noble, M. E. M.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">389-397</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Background: Cyclin-dependent kinase 2 (CDK2) is an important target for structure-based design of antitumor agents.  Monomeric CDK2 is inactive.  Activation requires rearrangements to key structural elements of the enzyme's active site, which accompany cyclin binding and phosphorylation.  To assess the validity of using monomeric CDK2 as a model for the active kinase in structure-based drug design, we have solved the structure of the inhibitor indirubin-5-sulfonate (E226) complexed with phospho-CDK2-cyclin A and compared it with the structure of E226 bound to inactive, monomeric CDK2.  Results: Activation of monomeric CDK2 leads to a rotation of its N-terminal domain relative to the C-terminal lobe.  The accompanying change in position of E226 follows that of the N-terminal domain, and its interactions with residues forming part of the adenine binding pocket are conserved.  The environment of the ATP-ribose site, not explored by E226, is significantly different in the binary complex compared to the monomeric complex due to movement of the glycine loop.  Conformational changes also result in subtle differences in hydrogen bonding and electrostatic interactions between E226's sulfonate and CDK2's phosphate binding site.  Affinities calcd. by LUDI for the interaction of E226 with active or inactive CDK2 differ by a factor of approx. ten.  Conclusions: The accuracy of monomeric CDK2 as an inhibitor design template is restricted to the adenine binding site.  The general flexibility obsd. for the glycine loop and subtle changes to the phosphate binding site suggest a need to study interactions between inhibitors and active CDK2 in structure-based drug design programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptJwFSSxY29rVg90H21EOLACvtfcHk0lh2yBVJ7iRBQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjslCisLw%253D&md5=5e8e5c5500ab6809cb4db066f6477901</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2801%2900598-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252801%252900598-6%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DT.%2BG.%26aulast%3DTunnah%26aufirst%3DP.%26aulast%3DMeijer%26aufirst%3DL.%26aulast%3DMarko%26aufirst%3DD.%26aulast%3DEisenbrand%26aufirst%3DG.%26aulast%3DEndicott%26aufirst%3DJ.%2BA.%26aulast%3DNoble%26aufirst%3DM.%2BE.%26atitle%3DInhibitor%2520Binding%2520to%2520Active%2520and%2520Inactive%2520Cdk2%253A%2520The%2520Crystal%2520Structure%2520of%2520Cdk2-Cyclin%2520a%252FIndirubin-5-Sulphonate%26jtitle%3DStructure%26date%3D2001%26volume%3D9%26spage%3D389%26epage%3D397%26doi%3D10.1016%2FS0969-2126%2801%2900598-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Hoessel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leclerc, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Endicott, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nobel, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawrie, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tunnah, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leost, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damiens, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marie, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marko, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niederberger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eisenbrand, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meijer, L.</span><span> </span><span class="NLM_article-title">Indirubin, the Active Constituent of a Chinese Antileukaemia Medicine, Inhibits Cyclin-Dependent Kinases</span> <span class="citation_source-journal">Nat. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">60</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span><span class="refDoi"> DOI: 10.1038/9035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1038%2F9035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10559866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADyaK1MXksVKitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=1999&pages=60-67&author=R.+Hoesselauthor=S.+Leclercauthor=J.+A.+Endicottauthor=M.+E.+Nobelauthor=A.+Lawrieauthor=P.+Tunnahauthor=M.+Leostauthor=E.+Damiensauthor=D.+Marieauthor=D.+Markoauthor=E.+Niederbergerauthor=W.+Tangauthor=G.+Eisenbrandauthor=L.+Meijer&title=Indirubin%2C+the+Active+Constituent+of+a+Chinese+Antileukaemia+Medicine%2C+Inhibits+Cyclin-Dependent+Kinases&doi=10.1038%2F9035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Indirubin, the active constituent of a Chinese antileukemia medicine, inhibits cyclin-dependent kinases</span></div><div class="casAuthors">Hoessel, Ralph; Leclerc, Sophie; Endicott, Jane A.; Nobel, Martin E. M.; Lawrie, Alison; Tunnah, Paul; Leost, Maryse; Damiens, Eve; Marie, Dominique; Marko, Doris; Niederberger, Ellen; Tang, Weici; Eisenbrand, Gerhard; Meijer, Laurent</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">60-67</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines Ltd</span>)
        </div><div class="casAbstract">Indirubin is the active ingredient of Danggui Longhui Wan, a mixt. of plants that is used in traditional Chinese medicine to treat chronic diseases.  Here indirubin and its analogs were identified as potent inhibitors of cyclin-dependent kinases (CDKs).  The crystal structure of CDK2 in complex with indirubin derivs. shows that indirubin interacts with the kinase's ATP-binding site through van der Waals interactions and three hydrogen bonds.  Indirubin-3'-monoxime inhibits the proliferation of a large range of cells, mainly through arresting the cells in the G2/M phase of the cell cycle.  These results have implications for therapeutic optimization of indigoids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpb5I1r-KcwErVg90H21EOLACvtfcHk0lhvoOTzGq1bYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXksVKitrk%253D&md5=bbbb09fbb3259de2e28fb58a5f507a7d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2F9035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F9035%26sid%3Dliteratum%253Aachs%26aulast%3DHoessel%26aufirst%3DR.%26aulast%3DLeclerc%26aufirst%3DS.%26aulast%3DEndicott%26aufirst%3DJ.%2BA.%26aulast%3DNobel%26aufirst%3DM.%2BE.%26aulast%3DLawrie%26aufirst%3DA.%26aulast%3DTunnah%26aufirst%3DP.%26aulast%3DLeost%26aufirst%3DM.%26aulast%3DDamiens%26aufirst%3DE.%26aulast%3DMarie%26aufirst%3DD.%26aulast%3DMarko%26aufirst%3DD.%26aulast%3DNiederberger%26aufirst%3DE.%26aulast%3DTang%26aufirst%3DW.%26aulast%3DEisenbrand%26aufirst%3DG.%26aulast%3DMeijer%26aufirst%3DL.%26atitle%3DIndirubin%252C%2520the%2520Active%2520Constituent%2520of%2520a%2520Chinese%2520Antileukaemia%2520Medicine%252C%2520Inhibits%2520Cyclin-Dependent%2520Kinases%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D1999%26volume%3D1%26spage%3D60%26epage%3D67%26doi%3D10.1038%2F9035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Ottana, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carotti, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maccari, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landini, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiricosta, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caciagli, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vigorita, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mini, E.</span><span> </span><span class="NLM_article-title">In Vitro Antiproliferative Activity against Human Colon Cancer Cell Lines of Representative 4-Thiazolidinones. Part I</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">3930</span><span class="NLM_x">–</span> <span class="NLM_lpage">3393</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2005.05.093</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2Fj.bmcl.2005.05.093" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2005&pages=3930-3393&author=R.+Ottanaauthor=S.+Carottiauthor=R.+Maccariauthor=I.+Landiniauthor=G.+Chiricostaauthor=B.+Caciagliauthor=M.+G.+Vigoritaauthor=E.+Mini&title=In+Vitro+Antiproliferative+Activity+against+Human+Colon+Cancer+Cell+Lines+of+Representative+4-Thiazolidinones.+Part+I&doi=10.1016%2Fj.bmcl.2005.05.093"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2005.05.093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2005.05.093%26sid%3Dliteratum%253Aachs%26aulast%3DOttana%26aufirst%3DR.%26aulast%3DCarotti%26aufirst%3DS.%26aulast%3DMaccari%26aufirst%3DR.%26aulast%3DLandini%26aufirst%3DI.%26aulast%3DChiricosta%26aufirst%3DG.%26aulast%3DCaciagli%26aufirst%3DB.%26aulast%3DVigorita%26aufirst%3DM.%2BG.%26aulast%3DMini%26aufirst%3DE.%26atitle%3DIn%2520Vitro%2520Antiproliferative%2520Activity%2520against%2520Human%2520Colon%2520Cancer%2520Cell%2520Lines%2520of%2520Representative%25204-Thiazolidinones.%2520Part%2520I%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2005%26volume%3D15%26spage%3D3930%26epage%3D3393%26doi%3D10.1016%2Fj.bmcl.2005.05.093" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Vicini, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geronikaki, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anastasia, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Incerti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zani, F.</span><span> </span><span class="NLM_article-title">Synthesis and Antimicrobial Activity of Novel 2-Thiazolylimino-5-Arylidene-4-Thiazolidinones</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">3859</span><span class="NLM_x">–</span> <span class="NLM_lpage">3864</span><span class="refDoi"> DOI: 10.1016/j.bmc.2006.01.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2Fj.bmc.2006.01.043" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2006&pages=3859-3864&author=P.+Viciniauthor=A.+Geronikakiauthor=K.+Anastasiaauthor=M.+Incertiauthor=F.+Zani&title=Synthesis+and+Antimicrobial+Activity+of+Novel+2-Thiazolylimino-5-Arylidene-4-Thiazolidinones&doi=10.1016%2Fj.bmc.2006.01.043"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2006.01.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2006.01.043%26sid%3Dliteratum%253Aachs%26aulast%3DVicini%26aufirst%3DP.%26aulast%3DGeronikaki%26aufirst%3DA.%26aulast%3DAnastasia%26aufirst%3DK.%26aulast%3DIncerti%26aufirst%3DM.%26aulast%3DZani%26aufirst%3DF.%26atitle%3DSynthesis%2520and%2520Antimicrobial%2520Activity%2520of%2520Novel%25202-Thiazolylimino-5-Arylidene-4-Thiazolidinones%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2006%26volume%3D14%26spage%3D3859%26epage%3D3864%26doi%3D10.1016%2Fj.bmc.2006.01.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">El-Barbary, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khodair, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, C.</span><span> </span><span class="NLM_article-title">Synthesis and Evaluation of Antiviral Activity of 2′-Deoxyuridines with 5-Methylene-2-Thiohydantoin Substituents in the 5-Position</span> <span class="citation_source-journal">Monatsh. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">593</span><span class="NLM_x">–</span> <span class="NLM_lpage">598</span><span class="refDoi"> DOI: 10.1007/BF00811853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1007%2FBF00811853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADyaK2cXlsFyjtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=1994&pages=593-598&author=A.+A.+El-Barbaryauthor=A.+I.+Khodairauthor=E.+B.+Pedersenauthor=C.+Nielsen&title=Synthesis+and+Evaluation+of+Antiviral+Activity+of+2%E2%80%B2-Deoxyuridines+with+5-Methylene-2-Thiohydantoin+Substituents+in+the+5-Position&doi=10.1007%2FBF00811853"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and evaluation of antiviral activity of 2'-deoxyuridines with 5-methylene-2-thiohydantoin substituents in the 5-position</span></div><div class="casAuthors">El-Barbary, A. A.; Khodair, A. I.; Pedersen, E. B.; Nielsen, C.</div><div class="citationInfo"><span class="NLM_cas:title">Monatshefte fuer Chemie</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">593-8</span>CODEN:
                <span class="NLM_cas:coden">MOCMB7</span>;
        ISSN:<span class="NLM_cas:issn">0026-9247</span>.
    </div><div class="casAbstract">Title deoxyribonucleosides, e.g. I (R = H, Me), were prepd. via glycosidation of thiohydantoins.  Antiviral and antitumor activities of I are reported.  None of I showed any significant inhibitory activities against HIV-1.  Only the nucleobase of I (R = Me) and α-I (R = Me) showed cytotoxicity against MT-4 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopDzJOcymgArVg90H21EOLACvtfcHk0lhh8QNdMMj7BQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXlsFyjtbg%253D&md5=b4c3f91b0cf3131d00cab25f354b2ced</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1007%2FBF00811853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00811853%26sid%3Dliteratum%253Aachs%26aulast%3DEl-Barbary%26aufirst%3DA.%2BA.%26aulast%3DKhodair%26aufirst%3DA.%2BI.%26aulast%3DPedersen%26aufirst%3DE.%2BB.%26aulast%3DNielsen%26aufirst%3DC.%26atitle%3DSynthesis%2520and%2520Evaluation%2520of%2520Antiviral%2520Activity%2520of%25202%25E2%2580%25B2-Deoxyuridines%2520with%25205-Methylene-2-Thiohydantoin%2520Substituents%2520in%2520the%25205-Position%26jtitle%3DMonatsh.%2520Chem.%26date%3D1994%26volume%3D125%26spage%3D593%26epage%3D598%26doi%3D10.1007%2FBF00811853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Bhandari, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bothara, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patil, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chitre, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarkate, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gore, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dangre, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khachane, C. V.</span><span> </span><span class="NLM_article-title">Design, Synthesis and Pharmacological Screening of Novel Antihypertensive Agents Using Hybrid Approach</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">390</span><span class="NLM_x">–</span> <span class="NLM_lpage">400</span><span class="refDoi"> DOI: 10.1016/j.bmc.2008.10.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2Fj.bmc.2008.10.032" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=390-400&author=S.+V.+Bhandariauthor=K.+G.+Botharaauthor=A.+A.+Patilauthor=T.+S.+Chitreauthor=A.+P.+Sarkateauthor=S.+T.+Goreauthor=S.+C.+Dangreauthor=C.+V.+Khachane&title=Design%2C+Synthesis+and+Pharmacological+Screening+of+Novel+Antihypertensive+Agents+Using+Hybrid+Approach&doi=10.1016%2Fj.bmc.2008.10.032"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2008.10.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2008.10.032%26sid%3Dliteratum%253Aachs%26aulast%3DBhandari%26aufirst%3DS.%2BV.%26aulast%3DBothara%26aufirst%3DK.%2BG.%26aulast%3DPatil%26aufirst%3DA.%2BA.%26aulast%3DChitre%26aufirst%3DT.%2BS.%26aulast%3DSarkate%26aufirst%3DA.%2BP.%26aulast%3DGore%26aufirst%3DS.%2BT.%26aulast%3DDangre%26aufirst%3DS.%2BC.%26aulast%3DKhachane%26aufirst%3DC.%2BV.%26atitle%3DDesign%252C%2520Synthesis%2520and%2520Pharmacological%2520Screening%2520of%2520Novel%2520Antihypertensive%2520Agents%2520Using%2520Hybrid%2520Approach%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2009%26volume%3D17%26spage%3D390%26epage%3D400%26doi%3D10.1016%2Fj.bmc.2008.10.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Richardson, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nunns, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williamson, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parratt, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dokurno, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howes, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borgognoni, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drysdale, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finch, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kierstan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lentzen, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmonite, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surgenor, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Torrance, C. J.</span><span> </span><span class="NLM_article-title">Discovery of a Potent Cdk2 Inhibitor with a Novel Binding Mode, Using Virtual Screening and Initial, Structure-Guided Lead Scoping</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">3880</span><span class="NLM_x">–</span> <span class="NLM_lpage">3885</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2007.04.110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2Fj.bmcl.2007.04.110" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=3880-3885&author=C.+M.+Richardsonauthor=C.+L.+Nunnsauthor=D.+S.+Williamsonauthor=M.+J.+Parrattauthor=P.+Dokurnoauthor=R.+Howesauthor=J.+Borgognoniauthor=M.+J.+Drysdaleauthor=H.+Finchauthor=R.+E.+Hubbardauthor=P.+S.+Jacksonauthor=P.+Kierstanauthor=G.+Lentzenauthor=J.+D.+Mooreauthor=J.+B.+Murrayauthor=H.+Simmoniteauthor=A.+E.+Surgenorauthor=C.+J.+Torrance&title=Discovery+of+a+Potent+Cdk2+Inhibitor+with+a+Novel+Binding+Mode%2C+Using+Virtual+Screening+and+Initial%2C+Structure-Guided+Lead+Scoping&doi=10.1016%2Fj.bmcl.2007.04.110"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.04.110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.04.110%26sid%3Dliteratum%253Aachs%26aulast%3DRichardson%26aufirst%3DC.%2BM.%26aulast%3DNunns%26aufirst%3DC.%2BL.%26aulast%3DWilliamson%26aufirst%3DD.%2BS.%26aulast%3DParratt%26aufirst%3DM.%2BJ.%26aulast%3DDokurno%26aufirst%3DP.%26aulast%3DHowes%26aufirst%3DR.%26aulast%3DBorgognoni%26aufirst%3DJ.%26aulast%3DDrysdale%26aufirst%3DM.%2BJ.%26aulast%3DFinch%26aufirst%3DH.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DJackson%26aufirst%3DP.%2BS.%26aulast%3DKierstan%26aufirst%3DP.%26aulast%3DLentzen%26aufirst%3DG.%26aulast%3DMoore%26aufirst%3DJ.%2BD.%26aulast%3DMurray%26aufirst%3DJ.%2BB.%26aulast%3DSimmonite%26aufirst%3DH.%26aulast%3DSurgenor%26aufirst%3DA.%2BE.%26aulast%3DTorrance%26aufirst%3DC.%2BJ.%26atitle%3DDiscovery%2520of%2520a%2520Potent%2520Cdk2%2520Inhibitor%2520with%2520a%2520Novel%2520Binding%2520Mode%252C%2520Using%2520Virtual%2520Screening%2520and%2520Initial%252C%2520Structure-Guided%2520Lead%2520Scoping%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D3880%26epage%3D3885%26doi%3D10.1016%2Fj.bmcl.2007.04.110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Dakin, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Block, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Code, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dowling, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferguson, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hird, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keeton, E. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamb, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyne, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, X.</span><span> </span><span class="NLM_article-title">Discovery of Novel Benzylidene-1,3-Thiazolidine-2,4-Diones as Potent and Selective Inhibitors of the Pim-1, Pim-2, and Pim-3 Protein Kinases</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">4599</span><span class="NLM_x">–</span> <span class="NLM_lpage">4604</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2012.05.098</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2Fj.bmcl.2012.05.098" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=22727640" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC38Xpt1Shtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=4599-4604&author=L.+A.+Dakinauthor=M.+H.+Blockauthor=H.+Chenauthor=E.+Codeauthor=J.+E.+Dowlingauthor=X.+Fengauthor=A.+D.+Fergusonauthor=I.+Greenauthor=A.+W.+Hirdauthor=T.+Howardauthor=E.+K.+Keetonauthor=M.+L.+Lambauthor=P.+D.+Lyneauthor=H.+Pollardauthor=J.+Readauthor=A.+J.+Wuauthor=T.+Zhangauthor=X.+Zheng&title=Discovery+of+Novel+Benzylidene-1%2C3-Thiazolidine-2%2C4-Diones+as+Potent+and+Selective+Inhibitors+of+the+Pim-1%2C+Pim-2%2C+and+Pim-3+Protein+Kinases&doi=10.1016%2Fj.bmcl.2012.05.098"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel benzylidene-1,3-thiazolidine-2,4-diones as potent and selective inhibitors of the PIM-1, PIM-2, and PIM-3 protein kinases</span></div><div class="casAuthors">Dakin, Les A.; Block, Michael H.; Chen, Huawei; Code, Erin; Dowling, James E.; Feng, Xiaomei; Ferguson, Andrew D.; Green, Isabelle; Hird, Alexander W.; Howard, Tina; Keeton, Erika K.; Lamb, Michelle L.; Lyne, Paul D.; Pollard, Hannah; Read, Jon; Wu, Allan J.; Zhang, Tao; Zheng, Xiaolan</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4599-4604</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Novel substituted benzylidene-1,3-thiazolidine-2,4-diones (TZDs) have been identified as potent and highly selective inhibitors of the PIM kinases.  The synthesis and SAR of these compds. are described, along with x-ray crystallog. of deriv. I, anti-proliferative, and selectivity data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLer-JRufp_rVg90H21EOLACvtfcHk0lhwjfFj_1zdEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xpt1Shtb4%253D&md5=cd1570938d48864bb739d2ccc37888f2</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.05.098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.05.098%26sid%3Dliteratum%253Aachs%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DBlock%26aufirst%3DM.%2BH.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DCode%26aufirst%3DE.%26aulast%3DDowling%26aufirst%3DJ.%2BE.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DGreen%26aufirst%3DI.%26aulast%3DHird%26aufirst%3DA.%2BW.%26aulast%3DHoward%26aufirst%3DT.%26aulast%3DKeeton%26aufirst%3DE.%2BK.%26aulast%3DLamb%26aufirst%3DM.%2BL.%26aulast%3DLyne%26aufirst%3DP.%2BD.%26aulast%3DPollard%26aufirst%3DH.%26aulast%3DRead%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DA.%2BJ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DZheng%26aufirst%3DX.%26atitle%3DDiscovery%2520of%2520Novel%2520Benzylidene-1%252C3-Thiazolidine-2%252C4-Diones%2520as%2520Potent%2520and%2520Selective%2520Inhibitors%2520of%2520the%2520Pim-1%252C%2520Pim-2%252C%2520and%2520Pim-3%2520Protein%2520Kinases%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D4599%26epage%3D4604%26doi%3D10.1016%2Fj.bmcl.2012.05.098" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Zhou, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ekins, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swaan, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, B.</span><span> </span><span class="NLM_article-title">Design, Synthesis, Cytoselective Toxicity, Structure-Activity Relationships, and Pharmacophore of Thiazolidinone Derivatives Targeting Drug-Resistant Lung Cancer Cells</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1242</span><span class="NLM_x">–</span> <span class="NLM_lpage">1251</span><span class="refDoi"> DOI: 10.1021/jm7012024</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm7012024" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhslWjurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=1242-1251&author=H.+Zhouauthor=S.+Wuauthor=S.+Zhaiauthor=A.+Liuauthor=Y.+Sunauthor=R.+Liauthor=Y.+Zhangauthor=S.+Ekinsauthor=P.+W.+Swaanauthor=B.+Fangauthor=B.+Zhangauthor=B.+Yan&title=Design%2C+Synthesis%2C+Cytoselective+Toxicity%2C+Structure-Activity+Relationships%2C+and+Pharmacophore+of+Thiazolidinone+Derivatives+Targeting+Drug-Resistant+Lung+Cancer+Cells&doi=10.1021%2Fjm7012024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, Cytoselective Toxicity, Structure-Activity Relationships, and Pharmacophore of Thiazolidinone Derivatives Targeting Drug-Resistant Lung Cancer Cells</span></div><div class="casAuthors">Zhou, Hongyu; Wu, Shuhong; Zhai, Shumei; Liu, Aifeng; Sun, Ying; Li, Rongshi; Zhang, Ying; Ekins, Sean; Swaan, Peter W.; Fang, Bingliang; Zhang, Bin; Yan, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1242-1251</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A library of 372 thiazolidinones I [R1 = H, Me2CH, HOCH2CH2, 2-pyridylmethyl, etc.; R2 = H, 2-Me, 3-F, 4-EtO2C, etc.; R3 = H, 4-CN, 2-HO, 2,4,5-(MeO)3, etc.] has been synthesized in three steps in moderate to excellent yields.  Of these thiazolidinones, ten cytoselective compds. have been identified by applying iterative library approaches.  These compds. selectively killed both non-small cell lung cancer cell line H460 and its paclitaxel-resistant variant H460taxR at an IC50 between 0.21 and 2.93 μM while showing much less toxicity to normal human fibroblasts at concns. up to 195 μM.  Structure-activity relationship studies revealed that (1) the nitrogen atom on the 4-thiazolidinone ring cannot be substituted, (2) several substitutions on arylimine arom. ring are tolerated at various positions, and (3) the substitution on benzylidene arom. ring is restricted to R3 = 4-NMe2.  A pharmacophore derived from active mols. suggested that two hydrogen bond acceptors and three hydrophobic regions were common features.  Activities against P-gp-overexpressing and paclitaxel-resistant cell line H460taxR and modeling using a previously validated P-gp substrate pharmacophore suggested that active compds. were not likely P-gp substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2qhZeAPQHM7Vg90H21EOLACvtfcHk0lhwjfFj_1zdEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhslWjurY%253D&md5=61a08d341d76d81978dc04a25e4b3fbe</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjm7012024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm7012024%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DS.%26aulast%3DZhai%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DA.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DEkins%26aufirst%3DS.%26aulast%3DSwaan%26aufirst%3DP.%2BW.%26aulast%3DFang%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DYan%26aufirst%3DB.%26atitle%3DDesign%252C%2520Synthesis%252C%2520Cytoselective%2520Toxicity%252C%2520Structure-Activity%2520Relationships%252C%2520and%2520Pharmacophore%2520of%2520Thiazolidinone%2520Derivatives%2520Targeting%2520Drug-Resistant%2520Lung%2520Cancer%2520Cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D1242%26epage%3D1251%26doi%3D10.1021%2Fjm7012024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Kornev, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haste, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eyck, L. F.</span><span> </span><span class="NLM_article-title">Surface Comparison of Active and Inactive Protein Kinases Identifies a Conserved Activation Mechanism</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">17783</span><span class="NLM_x">–</span> <span class="NLM_lpage">17788</span><span class="refDoi"> DOI: 10.1073/pnas.0607656103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1073%2Fpnas.0607656103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=17095602" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1KnurfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=17783-17788&author=A.+P.+Kornevauthor=N.+M.+Hasteauthor=S.+S.+Taylorauthor=L.+F.+Eyck&title=Surface+Comparison+of+Active+and+Inactive+Protein+Kinases+Identifies+a+Conserved+Activation+Mechanism&doi=10.1073%2Fpnas.0607656103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Surface comparison of active and inactive protein kinases identifies a conserved activation mechanism</span></div><div class="casAuthors">Kornev, Alexandr P.; Haste, Nina M.; Taylor, Susan S.; Ten Eyck, Lynn F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">17783-17788</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The surface comparison of different serine-threonine and tyrosine kinases reveals a set of 30 residues whose spatial positions are highly conserved.  The comparison between active and inactive conformations identified the residues whose positions are the most sensitive to activation.  Based on these results, we propose a model of protein kinase activation.  This model explains how the presence of a phosphate group in the activation loop dets. the position of the catalytically important aspartate in the Asp-Phe-Gly (DFG) motif.  According to the model, the most important feature of the activation is a "spine" formation that is dynamically assembled in all active kinases.  The spine is comprised of four hydrophobic residues that we detected in a set of 23 eukaryotic and prokaryotic kinases.  It spans the mol. and plays a coordinating role in activated kinases.  The spine is disordered in the inactive kinases and can explain how stabilization of the whole mol. is achieved upon phosphorylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwv4ByA5owH7Vg90H21EOLACvtfcHk0ljoGo_nNHhMbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1KnurfJ&md5=f474acffd8b17d8352ac9f6fc9bbbf8f</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0607656103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0607656103%26sid%3Dliteratum%253Aachs%26aulast%3DKornev%26aufirst%3DA.%2BP.%26aulast%3DHaste%26aufirst%3DN.%2BM.%26aulast%3DTaylor%26aufirst%3DS.%2BS.%26aulast%3DEyck%26aufirst%3DL.%2BF.%26atitle%3DSurface%2520Comparison%2520of%2520Active%2520and%2520Inactive%2520Protein%2520Kinases%2520Identifies%2520a%2520Conserved%2520Activation%2520Mechanism%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2006%26volume%3D103%26spage%3D17783%26epage%3D17788%26doi%3D10.1073%2Fpnas.0607656103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Kornev, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ten Eyck, L. F.</span><span> </span><span class="NLM_article-title">A Helix Scaffold for the Assembly of Active Protein Kinases</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">14377</span><span class="NLM_x">–</span> <span class="NLM_lpage">14382</span><span class="refDoi"> DOI: 10.1073/pnas.0807988105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1073%2Fpnas.0807988105" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=14377-14382&author=A.+P.+Kornevauthor=S.+S.+Taylorauthor=L.+F.+Ten+Eyck&title=A+Helix+Scaffold+for+the+Assembly+of+Active+Protein+Kinases&doi=10.1073%2Fpnas.0807988105"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0807988105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0807988105%26sid%3Dliteratum%253Aachs%26aulast%3DKornev%26aufirst%3DA.%2BP.%26aulast%3DTaylor%26aufirst%3DS.%2BS.%26aulast%3DTen%2BEyck%26aufirst%3DL.%2BF.%26atitle%3DA%2520Helix%2520Scaffold%2520for%2520the%2520Assembly%2520of%2520Active%2520Protein%2520Kinases%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D14377%26epage%3D14382%26doi%3D10.1073%2Fpnas.0807988105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Jautelat, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brumby, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schafer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Briem, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eisenbrand, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwahn, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lucking, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prien, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siemeister, G.</span><span> </span><span class="NLM_article-title">From the Insoluble Dye Indirubin Towards Highly Active, Soluble Cdk2-Inhibitors</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">531</span><span class="NLM_x">–</span> <span class="NLM_lpage">540</span><span class="refDoi"> DOI: 10.1002/cbic.200400108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1002%2Fcbic.200400108" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=531-540&author=R.+Jautelatauthor=T.+Brumbyauthor=M.+Schaferauthor=H.+Briemauthor=G.+Eisenbrandauthor=S.+Schwahnauthor=M.+Krugerauthor=U.+Luckingauthor=O.+Prienauthor=G.+Siemeister&title=From+the+Insoluble+Dye+Indirubin+Towards+Highly+Active%2C+Soluble+Cdk2-Inhibitors&doi=10.1002%2Fcbic.200400108"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1002%2Fcbic.200400108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.200400108%26sid%3Dliteratum%253Aachs%26aulast%3DJautelat%26aufirst%3DR.%26aulast%3DBrumby%26aufirst%3DT.%26aulast%3DSchafer%26aufirst%3DM.%26aulast%3DBriem%26aufirst%3DH.%26aulast%3DEisenbrand%26aufirst%3DG.%26aulast%3DSchwahn%26aufirst%3DS.%26aulast%3DKruger%26aufirst%3DM.%26aulast%3DLucking%26aufirst%3DU.%26aulast%3DPrien%26aufirst%3DO.%26aulast%3DSiemeister%26aufirst%3DG.%26atitle%3DFrom%2520the%2520Insoluble%2520Dye%2520Indirubin%2520Towards%2520Highly%2520Active%252C%2520Soluble%2520Cdk2-Inhibitors%26jtitle%3DChemBioChem%26date%3D2005%26volume%3D6%26spage%3D531%26epage%3D540%26doi%3D10.1002%2Fcbic.200400108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Yin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huffmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNamara, J. M.</span><span> </span><span class="NLM_article-title">Pd-Catalyzed N-Arylation of Heteroarylamines</span> <span class="citation_source-journal">Org. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">3481</span><span class="refDoi"> DOI: 10.1021/ol0265923</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol0265923" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD38Xms1Omur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2002&pages=3481&author=J.+Yinauthor=M.+M.+Zhaoauthor=M.+Huffmannauthor=J.+M.+McNamara&title=Pd-Catalyzed+N-Arylation+of+Heteroarylamines&doi=10.1021%2Fol0265923"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Pd-Catalyzed N-Arylation of Heteroarylamines</span></div><div class="casAuthors">Yin, Jingjun; Zhao, Matthew M.; Huffman, Mark A.; McNamara, James M.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3481-3484</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The palladium-catalyzed N-(hetero)arylation of a no. of heteroarylamines including 2-aminopyridines, 2-aminothiazoles, and their analogs has been realized using Xantphos as the ligand.  Weak bases such as Cs2CO3, Na2CO3, and K3PO4 were used in most cases to allow for the introduction of functional groups.  Choice of the base and solvent was crit. for the success of these reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpto0A1OmazwbVg90H21EOLACvtfcHk0ljoGo_nNHhMbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xms1Omur8%253D&md5=95f39b69250ce41d7c97d66ecc7f1ae8</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fol0265923&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol0265923%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DM.%2BM.%26aulast%3DHuffmann%26aufirst%3DM.%26aulast%3DMcNamara%26aufirst%3DJ.%2BM.%26atitle%3DPd-Catalyzed%2520N-Arylation%2520of%2520Heteroarylamines%26jtitle%3DOrg.%2520Lett.%26date%3D2002%26volume%3D4%26spage%3D3481%26doi%3D10.1021%2Fol0265923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Honda, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nigg, E. A.</span><span> </span><span class="NLM_article-title">Exploring the Functional Interactions between Aurora B, Incenp, and Survivin in Mitosis</span> <span class="citation_source-journal">Mol. Biol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">3325</span><span class="NLM_x">–</span> <span class="NLM_lpage">3341</span><span class="refDoi"> DOI: 10.1091/mbc.E02-11-0769</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1091%2Fmbc.E02-11-0769" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=12925766" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvFWks7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2003&pages=3325-3341&author=R.+Hondaauthor=R.+Kornerauthor=E.+A.+Nigg&title=Exploring+the+Functional+Interactions+between+Aurora+B%2C+Incenp%2C+and+Survivin+in+Mitosis&doi=10.1091%2Fmbc.E02-11-0769"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring the functional interactions between Aurora B, INCENP, and survivin in mitosis</span></div><div class="casAuthors">Honda, Reiko; Koerner, Roman; Nigg, Erich A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biology of the Cell</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3325-3341</span>CODEN:
                <span class="NLM_cas:coden">MBCEEV</span>;
        ISSN:<span class="NLM_cas:issn">1059-1524</span>.
    
            (<span class="NLM_cas:orgname">American Society for Cell Biology</span>)
        </div><div class="casAbstract">The function of the Aurora B kinase at centromeres and the central spindle is crucial for chromosome segregation and cytokinesis, resp.  Herein, we have investigated the regulation of human Aurora B by its complex partners inner centromere protein (INCENP) and survivin.  We found that overexpression of a catalytically inactive, dominant-neg. mutant of Aurora B impaired the localization of the entire Aurora B/INCENP/survivin complex to centromeres and the central spindle and severely disturbed mitotic progression.  Similar results were also obsd. after depletion, by RNA interference, of either Aurora B, INCENP, or survivin.  These data suggest that Aurora B kinase activity and the formation of the Aurora B/INCENP/survivin complex both contribute to its proper localization.  Using recombinant proteins, we found that Aurora B kinase activity was stimulated by INCENP and that the C-terminal region of INCENP was sufficient for activation.  Under identical assay conditions, survivin did not detectably influence kinase activity.  Human INCENP was a substrate of Aurora B and mass spectrometry identified three consecutive residues (threonine 893, serine 894, and serine 895) contg. at least two phosphorylation sites.  A nonphosphorylatable mutant (TSS893-895AAA) was a poor activator of Aurora B, demonstrating that INCENP phosphorylation is important for kinase activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHqQu6K_cbk7Vg90H21EOLACvtfcHk0lhQFTQlTQhGjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvFWks7w%253D&md5=8cb2fcfe367500c9f411f1ad621f7aa4</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1091%2Fmbc.E02-11-0769&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1091%252Fmbc.E02-11-0769%26sid%3Dliteratum%253Aachs%26aulast%3DHonda%26aufirst%3DR.%26aulast%3DKorner%26aufirst%3DR.%26aulast%3DNigg%26aufirst%3DE.%2BA.%26atitle%3DExploring%2520the%2520Functional%2520Interactions%2520between%2520Aurora%2520B%252C%2520Incenp%252C%2520and%2520Survivin%2520in%2520Mitosis%26jtitle%3DMol.%2520Biol.%2520Cell%26date%3D2003%26volume%3D14%26spage%3D3325%26epage%3D3341%26doi%3D10.1091%2Fmbc.E02-11-0769" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Fabian, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biggs, W. H.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azimioara, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benedetti, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galvin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerlach, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grotzfeld, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lelias, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehta, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velasco, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span> </span><span class="NLM_article-title">A Small Molecule-Kinase Interaction Map for Clinical Kinase Inhibitors</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span><span class="refDoi"> DOI: 10.1038/nbt1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.+M.+Leliasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+Small+Molecule-Kinase+Interaction+Map+for+Clinical+Kinase+Inhibitors&doi=10.1038%2Fnbt1068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0lhQFTQlTQhGjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DLelias%26aufirst%3DJ.%2BM.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520Small%2520Molecule-Kinase%2520Interaction%2520Map%2520for%2520Clinical%2520Kinase%2520Inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336%26doi%3D10.1038%2Fnbt1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Karaman, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallant, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faraoni, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrison, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pritchard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">A Quantitative Analysis of Kinase Inhibitor Selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span><span class="refDoi"> DOI: 10.1038/nbt1358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1038%2Fnbt1358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=18183025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=127-132&author=M.+W.+Karamanauthor=S.+Herrgardauthor=D.+K.+Treiberauthor=P.+Gallantauthor=C.+E.+Atteridgeauthor=B.+T.+Campbellauthor=K.+W.+Chanauthor=P.+Ciceriauthor=M.+I.+Davisauthor=P.+T.+Edeenauthor=R.+Faraoniauthor=M.+Floydauthor=J.+P.+Huntauthor=D.+J.+Lockhartauthor=Z.+V.+Milanovauthor=M.+J.+Morrisonauthor=G.+Pallaresauthor=H.+K.+Patelauthor=S.+Pritchardauthor=L.+M.+Wodickaauthor=P.+P.+Zarrinkar&title=A+Quantitative+Analysis+of+Kinase+Inhibitor+Selectivity&doi=10.1038%2Fnbt1358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Karaman, Mazen W.; Herrgard, Sanna; Treiber, Daniel K.; Gallant, Paul; Atteridge, Corey E.; Campbell, Brian T.; Chan, Katrina W.; Ciceri, Pietro; Davis, Mindy I.; Edeen, Philip T.; Faraoni, Raffaella; Floyd, Mark; Hunt, Jeremy P.; Lockhart, Daniel J.; Milanov, Zdravko V.; Morrison, Michael J.; Pallares, Gabriel; Patel, Hitesh K.; Pritchard, Stephanie; Wodicka, Lisa M.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-132</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors are a new class of therapeutics with a propensity to inhibit multiple targets.  The biol. consequences of multikinase activity are poorly defined, and an important step toward understanding the relationship between selectivity, efficacy and safety is the exploration of how inhibitors interact with the human kinome.  The authors present interaction maps for 38 kinase inhibitors across a panel of 317 kinases representing >50% of the predicted human protein kinome.  The data constitute the most comprehensive study of kinase inhibitor selectivity to date and reveal a wide diversity of interaction patterns.  To enable a global anal. of the results, the authors introduce the concept of a selectivity score as a general tool to quantify and differentiate the obsd. interaction patterns.  The authors further investigate the impact of panel size and find that small assay panels do not provide a robust measure of selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiArJSq2_lPrVg90H21EOLACvtfcHk0liGbLz3xSUnhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D&md5=346265d412853ced636ad4128ed8a76f</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1038%2Fnbt1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1358%26sid%3Dliteratum%253Aachs%26aulast%3DKaraman%26aufirst%3DM.%2BW.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DCampbell%26aufirst%3DB.%2BT.%26aulast%3DChan%26aufirst%3DK.%2BW.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFaraoni%26aufirst%3DR.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DMorrison%26aufirst%3DM.%2BJ.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DPritchard%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DA%2520Quantitative%2520Analysis%2520of%2520Kinase%2520Inhibitor%2520Selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D127%26epage%3D132%26doi%3D10.1038%2Fnbt1358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Knighton, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ten Eyck, L. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashford, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xuong, N. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sowadski, J. M.</span><span> </span><span class="NLM_article-title">Crystal Structure of the Catalytic Subunit of Cyclic Adenosine Monophosphate-Dependent Protein Kinase</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">253</span><span class="NLM_x">, </span> <span class="NLM_fpage">407</span><span class="NLM_x">–</span> <span class="NLM_lpage">414</span><span class="refDoi"> DOI: 10.1126/science.1862342</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1126%2Fscience.1862342" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1862342" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADyaK3MXmsFSqsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=253&publication_year=1991&pages=407-414&author=D.+R.+Knightonauthor=J.+H.+Zhengauthor=L.+F.+Ten+Eyckauthor=V.+A.+Ashfordauthor=N.+H.+Xuongauthor=S.+S.+Taylorauthor=J.+M.+Sowadski&title=Crystal+Structure+of+the+Catalytic+Subunit+of+Cyclic+Adenosine+Monophosphate-Dependent+Protein+Kinase&doi=10.1126%2Fscience.1862342"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the catalytic subunit of cyclic adenosine monophosphate-dependent protein kinase</span></div><div class="casAuthors">Knighton, Daniel R.; Zheng, Jianhua; Ten Eyck, Lynn F.; Ashford, Victor A.; Nguyen Huu Xuong; Taylor, Susan S.; Sowadski, Janusz M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">253</span>
        (<span class="NLM_cas:issue">5018</span>),
    <span class="NLM_cas:pages">407-14</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">The crystal structure of the catalytic subunit of cyclic adenosine monophosphate-dependent protein kinase complexed with a 20-amino acid substrate analog inhibitor has been solved and partially refined at 2.7 Å resoln. to an R factor of 0.212.  The magnesium ATP (MgATP) binding site was located by difference Fourier synthesis.  The enzyme structure is bilobal with a deep cleft between the lobes.  The cleft is filled by MgATP and a portion of the inhibitor peptide.  The smaller lobe, consisting mostly of amino-terminal sequence, is assocd. with nucleotide binding, and its largely antiparallel β sheet architecture constitutes an unusual nucleotide binding motif.  The large lobe is dominated by helical structure with a single β sheet at the domain interface.  This lobe is primarily involved in peptide binding and catalysis.  Residues 40 through 280 constitute a conserved catalytic core that is shared by more than 100 protein kinases.  Most of the invariant amino acids in this conserved catalytic core are clustered at the site of nucleotide binding and catalysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-BPvdNuboTbVg90H21EOLACvtfcHk0liGbLz3xSUnhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXmsFSqsLw%253D&md5=1b1dc6aaa1a63fc665b81f8967e143b1</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1126%2Fscience.1862342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1862342%26sid%3Dliteratum%253Aachs%26aulast%3DKnighton%26aufirst%3DD.%2BR.%26aulast%3DZheng%26aufirst%3DJ.%2BH.%26aulast%3DTen%2BEyck%26aufirst%3DL.%2BF.%26aulast%3DAshford%26aufirst%3DV.%2BA.%26aulast%3DXuong%26aufirst%3DN.%2BH.%26aulast%3DTaylor%26aufirst%3DS.%2BS.%26aulast%3DSowadski%26aufirst%3DJ.%2BM.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520Catalytic%2520Subunit%2520of%2520Cyclic%2520Adenosine%2520Monophosphate-Dependent%2520Protein%2520Kinase%26jtitle%3DScience%26date%3D1991%26volume%3D253%26spage%3D407%26epage%3D414%26doi%3D10.1126%2Fscience.1862342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Schindler, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bornmann, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pellicena, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarkson, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuriyan, J.</span><span> </span><span class="NLM_article-title">Structural Mechanism for Sti-571 Inhibition of Abelson Tyrosine Kinase</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">289</span><span class="NLM_x">, </span> <span class="NLM_fpage">1938</span><span class="NLM_x">–</span> <span class="NLM_lpage">1942</span><span class="refDoi"> DOI: 10.1126/science.289.5486.1938</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1126%2Fscience.289.5486.1938" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10988075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD3cXms1altLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2000&pages=1938-1942&author=T.+Schindlerauthor=W.+Bornmannauthor=P.+Pellicenaauthor=W.+T.+Millerauthor=B.+Clarksonauthor=J.+Kuriyan&title=Structural+Mechanism+for+Sti-571+Inhibition+of+Abelson+Tyrosine+Kinase&doi=10.1126%2Fscience.289.5486.1938"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Structural mechanism for STI-571 inhibition of Abelson tyrosine kinase</span></div><div class="casAuthors">Schindler, Thomas; Bornmann, William; Pellicena, Patricia; Miller, W. Todd; Clarkson, Bayard; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">5486</span>),
    <span class="NLM_cas:pages">1938-1942</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The inadvertent activation of the Abelson tyrosine kinase (Abl) causes chronic myelogenous leukemia (CML).  A small-mol. inhibitor of Abl (STI-571) is effective in the treatment of CML.  We report the crystal structure of the catalytic domain of Abl, complexed to a variant of STI-571.  Crit. to the binding of STI-571 is the adoption by the kinase of an inactive conformation, in which a centrally located "activation loop" is not phosphorylated.  The conformation of this loop is distinct from that in active protein kinases, as well as in the inactive form of the closely related Src kinases.  These results suggest that compds. that exploit the distinctive inactivation mechanisms of individual protein kinases can achieve both high affinity and high specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9eAZqIQ1aebVg90H21EOLACvtfcHk0lh8JnNwPdDi9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXms1altLc%253D&md5=56bb233f010f099d0073020ac6b9db59</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1126%2Fscience.289.5486.1938&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.289.5486.1938%26sid%3Dliteratum%253Aachs%26aulast%3DSchindler%26aufirst%3DT.%26aulast%3DBornmann%26aufirst%3DW.%26aulast%3DPellicena%26aufirst%3DP.%26aulast%3DMiller%26aufirst%3DW.%2BT.%26aulast%3DClarkson%26aufirst%3DB.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DStructural%2520Mechanism%2520for%2520Sti-571%2520Inhibition%2520of%2520Abelson%2520Tyrosine%2520Kinase%26jtitle%3DScience%26date%3D2000%26volume%3D289%26spage%3D1938%26epage%3D1942%26doi%3D10.1126%2Fscience.289.5486.1938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Girdler, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gascoigne, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eyers, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartmuth, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crafter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foote, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keen, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, S. S.</span><span> </span><span class="NLM_article-title">Validating Aurora B as an Anti-Cancer Drug Target</span> <span class="citation_source-journal">J. Cell Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">3664</span><span class="NLM_x">–</span> <span class="NLM_lpage">3675</span><span class="refDoi"> DOI: 10.1242/jcs.03145</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1242%2Fjcs.03145" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=16912073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVymu7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2006&pages=3664-3675&author=F.+Girdlerauthor=K.+E.+Gascoigneauthor=P.+A.+Eyersauthor=S.+Hartmuthauthor=C.+Crafterauthor=K.+M.+Footeauthor=N.+J.+Keenauthor=S.+S.+Taylor&title=Validating+Aurora+B+as+an+Anti-Cancer+Drug+Target&doi=10.1242%2Fjcs.03145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Validating Aurora B as an anti-cancer drug target</span></div><div class="casAuthors">Girdler, Fiona; Gascoigne, Karen E.; Eyers, Patrick A.; Hartmuth, Sonya; Crafter, Claire; Foote, Kevin M.; Keen, Nicholas J.; Taylor, Stephen S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3664-3675</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">The Aurora kinases, a family of mitotic regulators, have received much attention as potential targets for novel anti-cancer therapeutics.  Several Aurora kinase inhibitors have been described including ZM447439, which prevents chromosome alignment, spindle checkpoint function and cytokinesis.  Subsequently, ZM447439-treated cells exit mitosis without dividing and lose viability.  Because ZM447439 inhibits both Aurora A and B, we set out to det. which phenotypes are due to inhibition of which kinase.  Using mol. genetic approaches, we show that inhibition of Aurora B kinase activity phenocopies ZM447439.  Furthermore, a novel ZM compd., which is 100 times more selective for Aurora B over Aurora A in vitro, induces identical phenotypes.  Importantly, inhibition of Aurora B kinase activity induces a penetrant anti-proliferative phenotype, indicating that Aurora B is an attractive anti-cancer drug target.  Using mol. genetic and chem.-genetic approaches, we also probe the role of Aurora A kinase activity.  We show that simultaneous repression of Aurora A plus induction of a catalytic mutant induces a monopolar phenotype.  Consistently, another novel ZM-related inhibitor, which is 20 times as potent against Aurora A compared with ZM447439, induces a monopolar phenotype.  Expression of a drug-resistant Aurora A mutant reverts this phenotype, demonstrating that Aurora A kinase activity is required for spindle bipolarity in human cells.  Because small mol.-mediated inhibition of Aurora A and Aurora B yields distinct phenotypes, our observations indicate that the Auroras may present two avenues for anti-cancer drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXyhwkCpOaDrVg90H21EOLACvtfcHk0lh8JnNwPdDi9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVymu7zP&md5=707af873d03b36f20a389470e0aab1ff</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1242%2Fjcs.03145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.03145%26sid%3Dliteratum%253Aachs%26aulast%3DGirdler%26aufirst%3DF.%26aulast%3DGascoigne%26aufirst%3DK.%2BE.%26aulast%3DEyers%26aufirst%3DP.%2BA.%26aulast%3DHartmuth%26aufirst%3DS.%26aulast%3DCrafter%26aufirst%3DC.%26aulast%3DFoote%26aufirst%3DK.%2BM.%26aulast%3DKeen%26aufirst%3DN.%2BJ.%26aulast%3DTaylor%26aufirst%3DS.%2BS.%26atitle%3DValidating%2520Aurora%2520B%2520as%2520an%2520Anti-Cancer%2520Drug%2520Target%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2006%26volume%3D119%26spage%3D3664%26epage%3D3675%26doi%3D10.1242%2Fjcs.03145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Dodson, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kosmopoulou, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richards, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atrash, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bavetsias, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blagg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayliss, R.</span><span> </span><span class="NLM_article-title">Crystal Structure of an Aurora-a Mutant That Mimics Aurora-B Bound to Mln8054: Insights into Selectivity and Drug Design</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">427</span><span class="NLM_x">, </span> <span class="NLM_fpage">19</span><span class="NLM_x">–</span> <span class="NLM_lpage">28</span><span class="refDoi"> DOI: 10.1042/BJ20091530</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1042%2FBJ20091530" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=20067443" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFyntr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=427&publication_year=2010&pages=19-28&author=C.+A.+Dodsonauthor=M.+Kosmopoulouauthor=M.+W.+Richardsauthor=B.+Atrashauthor=V.+Bavetsiasauthor=J.+Blaggauthor=R.+Bayliss&title=Crystal+Structure+of+an+Aurora-a+Mutant+That+Mimics+Aurora-B+Bound+to+Mln8054%3A+Insights+into+Selectivity+and+Drug+Design&doi=10.1042%2FBJ20091530"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of an Aurora-A mutant that mimics Aurora-B bound to MLN8054: insights into selectivity and drug design</span></div><div class="casAuthors">Dodson, Charlotte A.; Kosmopoulou, Magda; Richards, Mark W.; Atrash, Butrus; Bavetsias, Vassilios; Blagg, Julian; Bayliss, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">427</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19-28</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The prodn. of selective protein kinase inhibitors is often frustrated by the similarity of the enzyme active sites.  For this reason, it is challenging to design inhibitors that discriminate between the three Aurora kinases, which are important targets in cancer drug discovery.  We have used a triple-point mutant of Aurora-A (AurAx3) which mimics the active site of Aurora-B to investigate the structural basis of MLN8054 selectivity.  The bias toward Aurora-A inhibition by MLN8054 is fully recapitulated by AurAx3 in vitro.  X-ray crystal structures of the complex suggest that the basis for the discrimination is electrostatic repulsion due to the T217E substitution, which we have confirmed using a single-point mutant.  The activation loop of Aurora-A in the AurAx3-MLN8054 complex exhibits an unusual conformation in which Asp274 and Phe275 side chains point into the interior of the protein.  There is to our knowledge no documented precedent for this conformation, which we have termed DFG-up.  The sequence requirements of the DFG-up conformation suggest that it might be accessible to only a fraction of kinases.  MLN8054 thus circumvents the problem of highly homologous active sites.  Binding of MLN8054 to Aurora-A switches the character of a pocket within the active site from polar to a hydrophobic pocket, similar to what is obsd. in the structure of Aurora-A bound to a compd. that induces DFG-out.  We propose that targeting this pocket may be a productive route in the design of selective kinase inhibitors and describe the structural basis for the rational design of these compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdJs5KY2VkabVg90H21EOLACvtfcHk0lj3C9FsTWqo5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFyntr8%253D&md5=fdf1a176a5fdf5fcd71fd57c3ef92817</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1042%2FBJ20091530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20091530%26sid%3Dliteratum%253Aachs%26aulast%3DDodson%26aufirst%3DC.%2BA.%26aulast%3DKosmopoulou%26aufirst%3DM.%26aulast%3DRichards%26aufirst%3DM.%2BW.%26aulast%3DAtrash%26aufirst%3DB.%26aulast%3DBavetsias%26aufirst%3DV.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBayliss%26aufirst%3DR.%26atitle%3DCrystal%2520Structure%2520of%2520an%2520Aurora-a%2520Mutant%2520That%2520Mimics%2520Aurora-B%2520Bound%2520to%2520Mln8054%253A%2520Insights%2520into%2520Selectivity%2520and%2520Drug%2520Design%26jtitle%3DBiochem.%2520J.%26date%3D2010%26volume%3D427%26spage%3D19%26epage%3D28%26doi%3D10.1042%2FBJ20091530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Sloane, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trikic, M. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamers, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mason, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savory, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eyers, P. A.</span><span> </span><span class="NLM_article-title">Drug-Resistant Aurora a Mutants for Cellular Target Validation of the Small Molecule Kinase Inhibitors Mln8054 and Mln8237</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">563</span><span class="NLM_x">–</span> <span class="NLM_lpage">576</span><span class="refDoi"> DOI: 10.1021/cb100053q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb100053q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlvVaktbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=563-576&author=D.+A.+Sloaneauthor=M.+Z.+Trikicauthor=M.+L.+Chuauthor=M.+B.+Lamersauthor=C.+S.+Masonauthor=I.+Muellerauthor=W.+J.+Savoryauthor=D.+H.+Williamsauthor=P.+A.+Eyers&title=Drug-Resistant+Aurora+a+Mutants+for+Cellular+Target+Validation+of+the+Small+Molecule+Kinase+Inhibitors+Mln8054+and+Mln8237&doi=10.1021%2Fcb100053q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-Resistant Aurora A Mutants for Cellular Target Validation of the Small Molecule Kinase Inhibitors MLN8054 and MLN8237</span></div><div class="casAuthors">Sloane, Dominic A.; Trikic, Michael Z.; Chu, Matthew L. H.; Lamers, Maria B. A. C.; Mason, Clive S.; Mueller, Ilka; Savory, Wendy J.; Williams, David H.; Eyers, Patrick A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">563-576</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Aurora kinases regulate multiple aspects of mitotic progression, and their overexpression in diverse tumor types makes them appealing oncol. targets.  An intensive research effort over the past decade has led to the discovery of chem. distinct families of small mol. Aurora kinase inhibitors, many of which have demonstrated therapeutic potential in model systems.  These agents are also important tools to help dissect signaling pathways that are orchestrated by Aurora kinases, and the antiproliferative target of pan-Aurora inhibitors such as VX-680 has been validated using chem. genetic techniques.  In many cases the nonspecific nature of Aurora inhibitors toward unrelated kinases is well established, potentially broadening the spectrum of cancers to which these compds. might be applied.  However, unambiguously demonstrating the mol. target(s) for clin. kinase inhibitors is an important challenge, one that is absolutely crit. for deciphering the mol. basis of compd. specificity, resistance, and efficacy.  In this paper, we have investigated amino acid requirements for Aurora A sensitivity to the benzazepine-based Aurora inhibitor MLN8054 and the close analog MLN8237, a second-generation compd. that is in phase II clin. trials.  A crystallog. anal. facilitated the design and biochem. investigation of a panel of resistant Aurora A mutants, a subset of which were then selected as candidate drug-resistance targets for further evaluation.  Using inducible human cell lines, we show that cells expressing near-physiol. levels of a functional but partially drug-resistant Aurora A T217D mutant survive in the presence of MLN8054 or MLN8237, authenticating Aurora A as a crit. antiproliferative target of these compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjg-t6yjgWhLVg90H21EOLACvtfcHk0lj5oBNWhATA5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlvVaktbs%253D&md5=f20bc2f2f827fd8f640570078d6505c6</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fcb100053q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb100053q%26sid%3Dliteratum%253Aachs%26aulast%3DSloane%26aufirst%3DD.%2BA.%26aulast%3DTrikic%26aufirst%3DM.%2BZ.%26aulast%3DChu%26aufirst%3DM.%2BL.%26aulast%3DLamers%26aufirst%3DM.%2BB.%26aulast%3DMason%26aufirst%3DC.%2BS.%26aulast%3DMueller%26aufirst%3DI.%26aulast%3DSavory%26aufirst%3DW.%2BJ.%26aulast%3DWilliams%26aufirst%3DD.%2BH.%26aulast%3DEyers%26aufirst%3DP.%2BA.%26atitle%3DDrug-Resistant%2520Aurora%2520a%2520Mutants%2520for%2520Cellular%2520Target%2520Validation%2520of%2520the%2520Small%2520Molecule%2520Kinase%2520Inhibitors%2520Mln8054%2520and%2520Mln8237%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2010%26volume%3D5%26spage%3D563%26epage%3D576%26doi%3D10.1021%2Fcb100053q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Eyers, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erikson, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maller, J. L.</span><span> </span><span class="NLM_article-title">A Novel Mechanism for Activation of the Protein Kinase Aurora A</span> <span class="citation_source-journal">Curr. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">691</span><span class="NLM_x">–</span> <span class="NLM_lpage">697</span><span class="refDoi"> DOI: 10.1016/S0960-9822(03)00166-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2FS0960-9822%2803%2900166-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=12699628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjtVKhu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=691-697&author=P.+A.+Eyersauthor=E.+Eriksonauthor=L.+G.+Chenauthor=J.+L.+Maller&title=A+Novel+Mechanism+for+Activation+of+the+Protein+Kinase+Aurora+A&doi=10.1016%2FS0960-9822%2803%2900166-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Mechanism for Activation of the Protein Kinase Aurora A</span></div><div class="casAuthors">Eyers, Patrick A.; Erikson, Eleanor; Chen, Lin G.; Maller, James L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">691-697</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Segregation of chromosomes during mitosis requires interplay between several classes of protein on the spindle, including protein kinases, protein phosphatases, and microtubule binding motor proteins.  Aurora A is an oncogenic cell cycle-regulated protein kinase that is subject to phosphorylation-dependent activation.  Aurora A localization to the mitotic spindle depends on the motor binding protein TPX2 (Targeting Protein for Xenopus kinesin-like protein 2), but the protein(s) involved in Aurora A activation are unknown.  Here, we purify an activator of Aurora A from Xenopus eggs and identify it as TPX2.  Remarkably, Aurora A that has been fully deactivated by Protein Phosphatase 2A (PP2A) becomes phosphorylated and reactivated by recombinant TPX2 in an ATP-dependent manner.  Increased phosphorylation and activation of Aurora A requires its own kinase activity, suggesting that TPX2 stimulates autophosphorylation and autoactivation of the enzyme.  Consistently, wild-type Aurora A, but not a kinase inactive mutant, becomes autophosphorylated on the regulatory T loop residue (Thr 295) after TPX2 treatment.  Active Aurora A from bacteria is further activated at least 7-fold by recombinant TPX2, and TPX2 also impairs the ability of protein phosphatases to inactivate Aurora A in vitro.  This concerted mechanism of stimulation of activation and inhibition of deactivation implies that TPX2 is the likely regulator of Aurora A activity at the mitotic spindle and may explain why loss of TPX2 in model systems perturbs spindle assembly.  Our finding that a known binding protein, and not a conventional protein kinase, is the relevant activator for Aurora A suggests a biochem. model in which the dynamic localization of TPX2 on mitotic structures directly modulates the activity of Aurora A for spindle assembly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouvFlsVOOTQ7Vg90H21EOLACvtfcHk0lj5oBNWhATA5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjtVKhu7k%253D&md5=98dc1619636f156c8a5748b7fdfa09fc</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2FS0960-9822%2803%2900166-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-9822%252803%252900166-0%26sid%3Dliteratum%253Aachs%26aulast%3DEyers%26aufirst%3DP.%2BA.%26aulast%3DErikson%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DL.%2BG.%26aulast%3DMaller%26aufirst%3DJ.%2BL.%26atitle%3DA%2520Novel%2520Mechanism%2520for%2520Activation%2520of%2520the%2520Protein%2520Kinase%2520Aurora%2520A%26jtitle%3DCurr.%2520Biol.%26date%3D2003%26volume%3D13%26spage%3D691%26epage%3D697%26doi%3D10.1016%2FS0960-9822%2803%2900166-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">Neumayer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belzil, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gruss, O. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, M. D.</span><span> </span><span class="NLM_article-title">Tpx2: Of Spindle Assembly, DNA Damage Response, and Cancer</span> <span class="citation_source-journal">Cell. Mol. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x">, </span> <span class="NLM_fpage">3027</span><span class="NLM_x">–</span> <span class="NLM_lpage">3047</span><span class="refDoi"> DOI: 10.1007/s00018-014-1582-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1007%2Fs00018-014-1582-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=24556998" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC2cXivFCiu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2014&pages=3027-3047&author=G.+Neumayerauthor=C.+Belzilauthor=O.+J.+Grussauthor=M.+D.+Nguyen&title=Tpx2%3A+Of+Spindle+Assembly%2C+DNA+Damage+Response%2C+and+Cancer&doi=10.1007%2Fs00018-014-1582-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">TPX2: of spindle assembly, DNA damage response, and cancer</span></div><div class="casAuthors">Neumayer, Gernot; Belzil, Camille; Gruss, Oliver J.; Nguyen, Minh Dang</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3027-3047</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Basel</span>)
        </div><div class="casAbstract">A review.  For more than 15 years, TPX2 has been studied as a factor crit. for mitosis and spindle assembly.  These functions of TPX2 are attributed to its Ran-regulated microtubule-assocd. protein properties and to its control of the Aurora A kinase.  Overexpressed in cancers, TPX2 is being established as marker for the diagnosis and prognosis of malignancies.  During interphase, TPX2 resides preferentially in the nucleus where its function had remained elusive until recently.  The latest finding that TPX2 plays a role in amplification of the DNA damage response, combined with the characterization of TPX2 knockout mice, open new perspectives to understand the biol. of this protein.  This review provides an historic overview of the discovery of TPX2 and summarizes its cytoskeletal and signaling roles with relevance to cancer therapies.  Finally, the review aims to reconcile discrepancies between the exptl. and pathol. effects of TPX2 overexpression and advances new roles for compartmentalized TPX2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo81lT1JcE_QrVg90H21EOLACvtfcHk0lj5oBNWhATA5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXivFCiu7Y%253D&md5=061d1c7591591ceb6a8974663ad1c125</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1007%2Fs00018-014-1582-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-014-1582-7%26sid%3Dliteratum%253Aachs%26aulast%3DNeumayer%26aufirst%3DG.%26aulast%3DBelzil%26aufirst%3DC.%26aulast%3DGruss%26aufirst%3DO.%2BJ.%26aulast%3DNguyen%26aufirst%3DM.%2BD.%26atitle%3DTpx2%253A%2520Of%2520Spindle%2520Assembly%252C%2520DNA%2520Damage%2520Response%252C%2520and%2520Cancer%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2014%26volume%3D71%26spage%3D3027%26epage%3D3047%26doi%3D10.1007%2Fs00018-014-1582-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Zorba, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buosi, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kutter, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kern, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pontiggia, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kern, D.</span><span> </span><span class="NLM_article-title">Molecular Mechanism of Aurora a Kinase Autophosphorylation and Its Allosteric Activation by Tpx2</span> <span class="citation_source-journal">eLife</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">e02667</span><span class="refDoi"> DOI: 10.7554/eLife.02667</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.7554%2FeLife.02667" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&pages=e02667&author=A.+Zorbaauthor=V.+Buosiauthor=S.+Kutterauthor=N.+Kernauthor=F.+Pontiggiaauthor=Y.+J.+Choauthor=D.+Kern&title=Molecular+Mechanism+of+Aurora+a+Kinase+Autophosphorylation+and+Its+Allosteric+Activation+by+Tpx2&doi=10.7554%2FeLife.02667"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.7554%2FeLife.02667&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.02667%26sid%3Dliteratum%253Aachs%26aulast%3DZorba%26aufirst%3DA.%26aulast%3DBuosi%26aufirst%3DV.%26aulast%3DKutter%26aufirst%3DS.%26aulast%3DKern%26aufirst%3DN.%26aulast%3DPontiggia%26aufirst%3DF.%26aulast%3DCho%26aufirst%3DY.%2BJ.%26aulast%3DKern%26aufirst%3DD.%26atitle%3DMolecular%2520Mechanism%2520of%2520Aurora%2520a%2520Kinase%2520Autophosphorylation%2520and%2520Its%2520Allosteric%2520Activation%2520by%2520Tpx2%26jtitle%3DeLife%26date%3D2014%26volume%3D3%26spage%3De02667%26doi%3D10.7554%2FeLife.02667" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">Kufer, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sillje, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gruss, O. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meraldi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nigg, E. A.</span><span> </span><span class="NLM_article-title">Human Tpx2 Is Required for Targeting Aurora-a Kinase to the Spindle</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">158</span><span class="NLM_x">, </span> <span class="NLM_fpage">617</span><span class="NLM_x">–</span> <span class="NLM_lpage">623</span><span class="refDoi"> DOI: 10.1083/jcb.200204155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1083%2Fjcb.200204155" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=158&publication_year=2002&pages=617-623&author=T.+A.+Kuferauthor=H.+H.+Silljeauthor=R.+Kornerauthor=O.+J.+Grussauthor=P.+Meraldiauthor=E.+A.+Nigg&title=Human+Tpx2+Is+Required+for+Targeting+Aurora-a+Kinase+to+the+Spindle&doi=10.1083%2Fjcb.200204155"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1083%2Fjcb.200204155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.200204155%26sid%3Dliteratum%253Aachs%26aulast%3DKufer%26aufirst%3DT.%2BA.%26aulast%3DSillje%26aufirst%3DH.%2BH.%26aulast%3DKorner%26aufirst%3DR.%26aulast%3DGruss%26aufirst%3DO.%2BJ.%26aulast%3DMeraldi%26aufirst%3DP.%26aulast%3DNigg%26aufirst%3DE.%2BA.%26atitle%3DHuman%2520Tpx2%2520Is%2520Required%2520for%2520Targeting%2520Aurora-a%2520Kinase%2520to%2520the%2520Spindle%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2002%26volume%3D158%26spage%3D617%26epage%3D623%26doi%3D10.1083%2Fjcb.200204155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group">Kesisova, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakos, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsolou, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Angelis, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chatzaki, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agianian, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giannis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koffa, M. D.</span><span> </span><span class="NLM_article-title">Tripolin a, a Novel Small-Molecule Inhibitor of Aurora a Kinase, Reveals New Regulation of Hurp’s Distribution on Microtubules</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e58485</span><span class="refDoi"> DOI: 10.1371/journal.pone.0058485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1371%2Fjournal.pone.0058485" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e58485&author=I.+A.+Kesisovaauthor=K.+C.+Nakosauthor=A.+Tsolouauthor=D.+Angelisauthor=J.+Lewisauthor=A.+Chatzakiauthor=B.+Agianianauthor=A.+Giannisauthor=M.+D.+Koffa&title=Tripolin+a%2C+a+Novel+Small-Molecule+Inhibitor+of+Aurora+a+Kinase%2C+Reveals+New+Regulation+of+Hurp%E2%80%99s+Distribution+on+Microtubules&doi=10.1371%2Fjournal.pone.0058485"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0058485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0058485%26sid%3Dliteratum%253Aachs%26aulast%3DKesisova%26aufirst%3DI.%2BA.%26aulast%3DNakos%26aufirst%3DK.%2BC.%26aulast%3DTsolou%26aufirst%3DA.%26aulast%3DAngelis%26aufirst%3DD.%26aulast%3DLewis%26aufirst%3DJ.%26aulast%3DChatzaki%26aufirst%3DA.%26aulast%3DAgianian%26aufirst%3DB.%26aulast%3DGiannis%26aufirst%3DA.%26aulast%3DKoffa%26aufirst%3DM.%2BD.%26atitle%3DTripolin%2520a%252C%2520a%2520Novel%2520Small-Molecule%2520Inhibitor%2520of%2520Aurora%2520a%2520Kinase%252C%2520Reveals%2520New%2520Regulation%2520of%2520Hurp%25E2%2580%2599s%2520Distribution%2520on%2520Microtubules%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De58485%26doi%3D10.1371%2Fjournal.pone.0058485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group">Littlepage, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andresson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deanehan, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amundadottir, L. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruderman, J. V.</span><span> </span><span class="NLM_article-title">Identification of Phosphorylated Residues That Affect the Activity of the Mitotic Kinase Aurora-A</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">15440</span><span class="NLM_x">–</span> <span class="NLM_lpage">15445</span><span class="refDoi"> DOI: 10.1073/pnas.202606599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1073%2Fpnas.202606599" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=15440-15445&author=L.+E.+Littlepageauthor=H.+Wuauthor=T.+Andressonauthor=J.+K.+Deanehanauthor=L.+T.+Amundadottirauthor=J.+V.+Ruderman&title=Identification+of+Phosphorylated+Residues+That+Affect+the+Activity+of+the+Mitotic+Kinase+Aurora-A&doi=10.1073%2Fpnas.202606599"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1073%2Fpnas.202606599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.202606599%26sid%3Dliteratum%253Aachs%26aulast%3DLittlepage%26aufirst%3DL.%2BE.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DAndresson%26aufirst%3DT.%26aulast%3DDeanehan%26aufirst%3DJ.%2BK.%26aulast%3DAmundadottir%26aufirst%3DL.%2BT.%26aulast%3DRuderman%26aufirst%3DJ.%2BV.%26atitle%3DIdentification%2520of%2520Phosphorylated%2520Residues%2520That%2520Affect%2520the%2520Activity%2520of%2520the%2520Mitotic%2520Kinase%2520Aurora-A%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D15440%26epage%3D15445%26doi%3D10.1073%2Fpnas.202606599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group">Marumoto, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nitta, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirota, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kohmura, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saya, H.</span><span> </span><span class="NLM_article-title">Aurora-a Kinase Maintains the Fidelity of Early and Late Mitotic Events in Hela Cells</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">278</span><span class="NLM_x">, </span> <span class="NLM_fpage">51786</span><span class="NLM_x">–</span> <span class="NLM_lpage">51795</span><span class="refDoi"> DOI: 10.1074/jbc.M306275200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1074%2Fjbc.M306275200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=51786-51795&author=T.+Marumotoauthor=S.+Hondaauthor=T.+Haraauthor=M.+Nittaauthor=T.+Hirotaauthor=E.+Kohmuraauthor=H.+Saya&title=Aurora-a+Kinase+Maintains+the+Fidelity+of+Early+and+Late+Mitotic+Events+in+Hela+Cells&doi=10.1074%2Fjbc.M306275200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M306275200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M306275200%26sid%3Dliteratum%253Aachs%26aulast%3DMarumoto%26aufirst%3DT.%26aulast%3DHonda%26aufirst%3DS.%26aulast%3DHara%26aufirst%3DT.%26aulast%3DNitta%26aufirst%3DM.%26aulast%3DHirota%26aufirst%3DT.%26aulast%3DKohmura%26aufirst%3DE.%26aulast%3DSaya%26aufirst%3DH.%26atitle%3DAurora-a%2520Kinase%2520Maintains%2520the%2520Fidelity%2520of%2520Early%2520and%2520Late%2520Mitotic%2520Events%2520in%2520Hela%2520Cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D51786%26epage%3D51795%26doi%3D10.1074%2Fjbc.M306275200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group">Giet, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glover, D. M.</span><span> </span><span class="NLM_article-title">Drosophila Aurora B Kinase Is Required for Histone H3 Phosphorylation and Condensin Recruitment During Chromosome Condensation and to Organize the Central Spindle During Cytokinesis</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">152</span><span class="NLM_x">, </span> <span class="NLM_fpage">669</span><span class="NLM_x">–</span> <span class="NLM_lpage">682</span><span class="refDoi"> DOI: 10.1083/jcb.152.4.669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1083%2Fjcb.152.4.669" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2001&pages=669-682&author=R.+Gietauthor=D.+M.+Glover&title=Drosophila+Aurora+B+Kinase+Is+Required+for+Histone+H3+Phosphorylation+and+Condensin+Recruitment+During+Chromosome+Condensation+and+to+Organize+the+Central+Spindle+During+Cytokinesis&doi=10.1083%2Fjcb.152.4.669"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1083%2Fjcb.152.4.669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.152.4.669%26sid%3Dliteratum%253Aachs%26aulast%3DGiet%26aufirst%3DR.%26aulast%3DGlover%26aufirst%3DD.%2BM.%26atitle%3DDrosophila%2520Aurora%2520B%2520Kinase%2520Is%2520Required%2520for%2520Histone%2520H3%2520Phosphorylation%2520and%2520Condensin%2520Recruitment%2520During%2520Chromosome%2520Condensation%2520and%2520to%2520Organize%2520the%2520Central%2520Spindle%2520During%2520Cytokinesis%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2001%26volume%3D152%26spage%3D669%26epage%3D682%26doi%3D10.1083%2Fjcb.152.4.669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group">Ditchfield, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, V. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tighe, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellston, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haworth, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mortlock, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, S. S.</span><span> </span><span class="NLM_article-title">Aurora B Couples Chromosome Alignment with Anaphase by Targeting Bubr1, Mad2, and Cenp-E to Kinetochores</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">161</span><span class="NLM_x">, </span> <span class="NLM_fpage">267</span><span class="NLM_x">–</span> <span class="NLM_lpage">280</span><span class="refDoi"> DOI: 10.1083/jcb.200208091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1083%2Fjcb.200208091" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=161&publication_year=2003&pages=267-280&author=C.+Ditchfieldauthor=V.+L.+Johnsonauthor=A.+Tigheauthor=R.+Ellstonauthor=C.+Haworthauthor=T.+Johnsonauthor=A.+Mortlockauthor=N.+Keenauthor=S.+S.+Taylor&title=Aurora+B+Couples+Chromosome+Alignment+with+Anaphase+by+Targeting+Bubr1%2C+Mad2%2C+and+Cenp-E+to+Kinetochores&doi=10.1083%2Fjcb.200208091"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1083%2Fjcb.200208091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.200208091%26sid%3Dliteratum%253Aachs%26aulast%3DDitchfield%26aufirst%3DC.%26aulast%3DJohnson%26aufirst%3DV.%2BL.%26aulast%3DTighe%26aufirst%3DA.%26aulast%3DEllston%26aufirst%3DR.%26aulast%3DHaworth%26aufirst%3DC.%26aulast%3DJohnson%26aufirst%3DT.%26aulast%3DMortlock%26aufirst%3DA.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DTaylor%26aufirst%3DS.%2BS.%26atitle%3DAurora%2520B%2520Couples%2520Chromosome%2520Alignment%2520with%2520Anaphase%2520by%2520Targeting%2520Bubr1%252C%2520Mad2%252C%2520and%2520Cenp-E%2520to%2520Kinetochores%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2003%26volume%3D161%26spage%3D267%26epage%3D280%26doi%3D10.1083%2Fjcb.200208091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group">Harrington, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bebbington, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rasmussen, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ajose-Adeogun, A. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakayama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graham, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demur, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hercend, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diu-Hercend, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golec, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, K. M.</span><span> </span><span class="NLM_article-title">Vx-680, a Potent and Selective Small-Molecule Inhibitor of the Aurora Kinases, Suppresses Tumor Growth in Vivo</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">262</span><span class="NLM_x">–</span> <span class="NLM_lpage">267</span><span class="refDoi"> DOI: 10.1038/nm1003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1038%2Fnm1003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=14981513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhs1Wnur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=262-267&author=E.+A.+Harringtonauthor=D.+Bebbingtonauthor=J.+Mooreauthor=R.+K.+Rasmussenauthor=A.+O.+Ajose-Adeogunauthor=T.+Nakayamaauthor=J.+A.+Grahamauthor=C.+Demurauthor=T.+Hercendauthor=A.+Diu-Hercendauthor=M.+Suauthor=J.+M.+Golecauthor=K.+M.+Miller&title=Vx-680%2C+a+Potent+and+Selective+Small-Molecule+Inhibitor+of+the+Aurora+Kinases%2C+Suppresses+Tumor+Growth+in+Vivo&doi=10.1038%2Fnm1003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">VX-680, a potent and selective small-molecule inhibitor of the Aurora kinases, suppresses tumor growth in vivo</span></div><div class="casAuthors">Harrington, Elizabeth A.; Bebbington, David; Moore, Jeff; Rasmussen, Richele K.; Ajose-Adeogun, Abi O.; Nakayama, Tomoko; Graham, Joanne A.; Demur, Cecile; Hercend, Thierry; Diu-Hercend, Anita; Su, Michael; Golec, Julian M. C.; Miller, Karen M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">262-267</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The Aurora kinases are essential for the regulation of chromosome segregation and cytokinesis during mitosis.  Aberrant expression and activity of these kinases occur in a wide range of human tumors, and lead to aneuploidy and tumorigenesis.  Here we report the discovery of a highly potent and selective small-mol. inhibitor of Aurora kinases, VX-680, that blocks cell-cycle progression and induces apoptosis in a diverse range of human tumor types.  This compd. causes profound inhibition of tumor growth in a variety of in vivo xenograft models, leading to regression of leukemia, colon and pancreatic tumors at well-tolerated doses.  Our data indicate that Aurora kinase inhibition provides a new approach for the treatment of multiple human malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKgm1YUgqkWLVg90H21EOLACvtfcHk0lijM7cCD6uPLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhs1Wnur4%253D&md5=954ba274e280d8d08a7f2e331011432d</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1038%2Fnm1003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm1003%26sid%3Dliteratum%253Aachs%26aulast%3DHarrington%26aufirst%3DE.%2BA.%26aulast%3DBebbington%26aufirst%3DD.%26aulast%3DMoore%26aufirst%3DJ.%26aulast%3DRasmussen%26aufirst%3DR.%2BK.%26aulast%3DAjose-Adeogun%26aufirst%3DA.%2BO.%26aulast%3DNakayama%26aufirst%3DT.%26aulast%3DGraham%26aufirst%3DJ.%2BA.%26aulast%3DDemur%26aufirst%3DC.%26aulast%3DHercend%26aufirst%3DT.%26aulast%3DDiu-Hercend%26aufirst%3DA.%26aulast%3DSu%26aufirst%3DM.%26aulast%3DGolec%26aufirst%3DJ.%2BM.%26aulast%3DMiller%26aufirst%3DK.%2BM.%26atitle%3DVx-680%252C%2520a%2520Potent%2520and%2520Selective%2520Small-Molecule%2520Inhibitor%2520of%2520the%2520Aurora%2520Kinases%252C%2520Suppresses%2520Tumor%2520Growth%2520in%2520Vivo%26jtitle%3DNat.%2520Med.%26date%3D2004%26volume%3D10%26spage%3D262%26epage%3D267%26doi%3D10.1038%2Fnm1003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group">Shimomura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasako, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakatsuru, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mita, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ichikawa, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kodera, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakai, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nambu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyamoto, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahashi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawanishi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohkubo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotani, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwasawa, Y.</span><span> </span><span class="NLM_article-title">Mk-5108, a Highly Selective Aurora-a Kinase Inhibitor, Shows Antitumor Activity Alone and in Combination with Docetaxel</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">157</span><span class="NLM_x">–</span> <span class="NLM_lpage">166</span><span class="refDoi"> DOI: 10.1158/1535-7163.MCT-09-0609</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1158%2F1535-7163.MCT-09-0609" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=20053775" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktVOisQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=157-166&author=T.+Shimomuraauthor=S.+Hasakoauthor=Y.+Nakatsuruauthor=T.+Mitaauthor=K.+Ichikawaauthor=T.+Koderaauthor=T.+Sakaiauthor=T.+Nambuauthor=M.+Miyamotoauthor=I.+Takahashiauthor=S.+Mikiauthor=N.+Kawanishiauthor=M.+Ohkuboauthor=H.+Kotaniauthor=Y.+Iwasawa&title=Mk-5108%2C+a+Highly+Selective+Aurora-a+Kinase+Inhibitor%2C+Shows+Antitumor+Activity+Alone+and+in+Combination+with+Docetaxel&doi=10.1158%2F1535-7163.MCT-09-0609"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">MK-5108, a Highly Selective Aurora-A Kinase Inhibitor, Shows Antitumor Activity Alone and in Combination with Docetaxel</span></div><div class="casAuthors">Shimomura, Toshiyasu; Hasako, Shinichi; Nakatsuru, Yoko; Mita, Takashi; Ichikawa, Koji; Kodera, Tsutomu; Sakai, Takumi; Nambu, Tadahiro; Miyamoto, Mayu; Takahashi, Ikuko; Miki, Satomi; Kawanishi, Nobuhiko; Ohkubo, Mitsuru; Kotani, Hidehito; Iwasawa, Yoshikazu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">157-166</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Aurora-A kinase is a one of the key regulators during mitosis progression.  Aurora-A kinase is a potential target for anticancer therapies because overexpression of Aurora-A, which is frequently obsd. in some human cancers, results in aberrant mitosis leading to chromosomal instability and possibly tumorigenesis.  MK-5108 is a novel small mol. with potent inhibitory activity against Aurora-A kinase.  Although most of the Aurora-kinase inhibitors target both Aurora-A and Aurora-B, MK-5108 specifically inhibited Aurora-A kinase in a panel of protein kinase assays.  Inhibition of Aurora-A by MK-5108 in cultured cells induced cell cycle arrest at the G2-M phase in flow cytometry anal.  The effect was confirmed by the accumulation of cells with expression of phosphorylated Histone H3 and inhibition of Aurora-A autophosphorylation by immunostaining assays.  MK-5108 also induced phosphorylated Histone H3 in skin and xenograft tumor tissues in a nude rat xenograft model.  MK-5108 inhibited growth of human tumor cell lines in culture and in different xenograft models.  Furthermore, the combination of MK-5108 and docetaxel showed enhanced antitumor activities compared with control and docetaxel alone-treated animals without exacerbating the adverse effects of docetaxel.  MK-5108 is currently tested in clin. trials and offers a new therapeutic approach to combat human cancers as a single agent or in combination with existing taxane therapies.  Mol Cancer Ther; 9(1); 157-66.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRD-HWMVBoWrVg90H21EOLACvtfcHk0lh62jt9BzR-hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktVOisQ%253D%253D&md5=267f0e3f702e2489e8b0c942f9299850</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-09-0609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-09-0609%26sid%3Dliteratum%253Aachs%26aulast%3DShimomura%26aufirst%3DT.%26aulast%3DHasako%26aufirst%3DS.%26aulast%3DNakatsuru%26aufirst%3DY.%26aulast%3DMita%26aufirst%3DT.%26aulast%3DIchikawa%26aufirst%3DK.%26aulast%3DKodera%26aufirst%3DT.%26aulast%3DSakai%26aufirst%3DT.%26aulast%3DNambu%26aufirst%3DT.%26aulast%3DMiyamoto%26aufirst%3DM.%26aulast%3DTakahashi%26aufirst%3DI.%26aulast%3DMiki%26aufirst%3DS.%26aulast%3DKawanishi%26aufirst%3DN.%26aulast%3DOhkubo%26aufirst%3DM.%26aulast%3DKotani%26aufirst%3DH.%26aulast%3DIwasawa%26aufirst%3DY.%26atitle%3DMk-5108%252C%2520a%2520Highly%2520Selective%2520Aurora-a%2520Kinase%2520Inhibitor%252C%2520Shows%2520Antitumor%2520Activity%2520Alone%2520and%2520in%2520Combination%2520with%2520Docetaxel%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2010%26volume%3D9%26spage%3D157%26epage%3D166%26doi%3D10.1158%2F1535-7163.MCT-09-0609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group">Evans, P.</span><span> </span><span class="NLM_article-title">Scaling and Assessment of Data Quality</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">72</span><span class="NLM_x">–</span> <span class="NLM_lpage">82</span><span class="refDoi"> DOI: 10.1107/S0907444905036693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1107%2FS0907444905036693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=16369096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=72-82&author=P.+Evans&title=Scaling+and+Assessment+of+Data+Quality&doi=10.1107%2FS0907444905036693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Scaling and assessment of data quality</span></div><div class="casAuthors">Evans, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-82</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The various phys. factors affecting measured diffraction intensities are discussed, as are the scaling models which may be used to put the data on a consistent scale.  After scaling, the intensities can be analyzed to set the real resoln. of the data set, to detect bad regions (e.g. bad images), to analyze radiation damage and to assess the overall quality of the data set.  The significance of any anomalous signal may be assessed by probability and correlation anal.  The algorithms used by the CCP4 scaling program SCALA are described.  A requirement for the scaling and merging of intensities is knowledge of the Laue group and point-group symmetries: the possible symmetry of the diffraction pattern may be detd. from scores such as correlation coeffs. between observations which might be symmetry-related.  These scoring functions are implemented in a new program POINTLESS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGravLSLU62RhLVg90H21EOLACvtfcHk0lh62jt9BzR-hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM&md5=293d3876e534c0c57813990515bb3c76</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1107%2FS0907444905036693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444905036693%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%26atitle%3DScaling%2520and%2520Assessment%2520of%2520Data%2520Quality%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D72%26epage%3D82%26doi%3D10.1107%2FS0907444905036693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group">Leslie, A. G. W.; Powell, H. R.; Read, R. J.; Sussman, J. L.</span>, <span> </span><span class="NLM_article-title">Processing Diffraction Data with Mosflm</span>.  <span class="citation_source-book">Nato Sci. Ser. II Math.</span>; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Dordrecht, The Netherlands</span>,<span class="NLM_x"> </span><span class="NLM_year">2007</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">245</span>, p  <span class="NLM_fpage">41</span>–.<span class="refDoi"> DOI: 10.1007/978-1-4020-6316-9_4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1007%2F978-1-4020-6316-9_4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&pages=41&author=A.+G.+W.+Leslie&author=H.+R.+Powell&author=R.+J.+Read&author=J.+L.+Sussman&title=Nato+Sci.+Ser.+II+Math."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1007%2F978-1-4020-6316-9_4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-4020-6316-9_4%26sid%3Dliteratum%253Aachs%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26atitle%3DProcessing%2520Diffraction%2520Data%2520with%2520Mosflm%26btitle%3DNato%2520Sci.%2520Ser.%2520II%2520Math.%26pub%3DSpringer%26date%3D2007%26volume%3D245%26spage%3D41%26doi%3D10.1007%2F978-1-4020-6316-9_4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">Xds</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span><span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lhgpi-vX8L4Eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballard, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, K. S.</span><span> </span><span class="NLM_article-title">Overview of the Ccp4 Suite and Current Developments</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=K.+S.+Wilson&title=Overview+of+the+Ccp4+Suite+and+Current+Developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lhgpi-vX8L4Eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520Ccp4%2520Suite%2520and%2520Current%2520Developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkoczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">Phenix: A Comprehensive Python-Based System for Macromolecular Structure Solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=Phenix%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lgg3EpnL_9OtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPhenix%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and Development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lgg3EpnL_9OtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group">DeLano, W. L.</span>The PyMOL Molecular Graphics System; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">Cambridge, MA</span>,<span class="NLM_x"> </span><span class="NLM_year">2002</span><span class="NLM_x">; </span><a href="http://www.pymol.org" class="extLink">http://www.pymol.org</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2002&author=W.+L.+DeLano&title=The+PyMOL+Molecular+Graphics+System"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DDeLano%26aufirst%3DW.%2BL.%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group">Wallace, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laskowski, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thornton, J. M.</span><span> </span><span class="NLM_article-title">Ligplot - a Program to Generate Schematic Diagrams of Protein Ligand Interactions</span> <span class="citation_source-journal">Protein Eng., Des. Sel.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">134</span><span class="refDoi"> DOI: 10.1093/protein/8.2.127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1093%2Fprotein%2F8.2.127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADyaK2MXltVCgtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1995&pages=127-134&author=A.+C.+Wallaceauthor=R.+A.+Laskowskiauthor=J.+M.+Thornton&title=Ligplot+-+a+Program+to+Generate+Schematic+Diagrams+of+Protein+Ligand+Interactions&doi=10.1093%2Fprotein%2F8.2.127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">LIGPLOT: a program to generate schematic diagrams of protein-ligand interactions</span></div><div class="casAuthors">Wallace, Andrew C.; Laskowski, Roman A.; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Engineering</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">127-34</span>CODEN:
                <span class="NLM_cas:coden">PRENE9</span>;
        ISSN:<span class="NLM_cas:issn">0269-2139</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The LIGPLOT program automatically generates schematic 2-D representations of protein-ligand complexes from std. Protein Data Bank file input.  The output is a color, or black-and-white, PostScript file giving a simple and informative representation of the intermol. interactions and their strengths, including hydrogen bonds, hydrophobic interactions and atom accessibilities.  The program is completely general for any ligand and can also be used to show other types of interaction in proteins and nucleic acids.  It was designed to facilitate the rapid inspection of many enzyme complexes, but has found many other applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpp9SrWw0klVbVg90H21EOLACvtfcHk0lgg3EpnL_9OtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXltVCgtLc%253D&md5=ed74e39b0cd6be4b2b8907d3ff6d9619</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F8.2.127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F8.2.127%26sid%3Dliteratum%253Aachs%26aulast%3DWallace%26aufirst%3DA.%2BC.%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DLigplot%2520-%2520a%2520Program%2520to%2520Generate%2520Schematic%2520Diagrams%2520of%2520Protein%2520Ligand%2520Interactions%26jtitle%3DProtein%2520Eng.%252C%2520Des.%2520Sel.%26date%3D1995%26volume%3D8%26spage%3D127%26epage%3D134%26doi%3D10.1093%2Fprotein%2F8.2.127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group">Merdes, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heald, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samejima, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Earnshaw, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cleveland, D. W.</span><span> </span><span class="NLM_article-title">Formation of Spindle Poles by Dynein/Dynactin-Dependent Transport of Numa</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">149</span><span class="NLM_x">, </span> <span class="NLM_fpage">851</span><span class="NLM_x">–</span> <span class="NLM_lpage">862</span><span class="refDoi"> DOI: 10.1083/jcb.149.4.851</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1083%2Fjcb.149.4.851" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2000&pages=851-862&author=A.+Merdesauthor=R.+Healdauthor=K.+Samejimaauthor=W.+C.+Earnshawauthor=D.+W.+Cleveland&title=Formation+of+Spindle+Poles+by+Dynein%2FDynactin-Dependent+Transport+of+Numa&doi=10.1083%2Fjcb.149.4.851"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1083%2Fjcb.149.4.851&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.149.4.851%26sid%3Dliteratum%253Aachs%26aulast%3DMerdes%26aufirst%3DA.%26aulast%3DHeald%26aufirst%3DR.%26aulast%3DSamejima%26aufirst%3DK.%26aulast%3DEarnshaw%26aufirst%3DW.%2BC.%26aulast%3DCleveland%26aufirst%3DD.%2BW.%26atitle%3DFormation%2520of%2520Spindle%2520Poles%2520by%2520Dynein%252FDynactin-Dependent%2520Transport%2520of%2520Numa%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2000%26volume%3D149%26spage%3D851%26epage%3D862%26doi%3D10.1083%2Fjcb.149.4.851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 39 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Mookda Pattarawarapan, Dolnapa Yamano, Nittaya Wiriya, Saranphong Yimklan, <span class="NLM_string-name hlFld-ContribAuthor">Wong Phakhodee</span>. </span><span class="cited-content_cbyCitation_article-title">Simultaneous Formation and Functionalization of Aryliminophosphoranes Using 1,3-Dihydro-1H-benzimidazol-2-ones as Precursors. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2020,</strong> <em>85 </em>
                                    (20)
                                     , 13330-13338. <a href="https://doi.org/10.1021/acs.joc.0c01979" title="DOI URL">https://doi.org/10.1021/acs.joc.0c01979</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.0c01979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.0c01979%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DSimultaneous%252BFormation%252Band%252BFunctionalization%252Bof%252BAryliminophosphoranes%252BUsing%252B1%25252C3-Dihydro-1H-benzimidazol-2-ones%252Bas%252BPrecursors%26aulast%3DPattarawarapan%26aufirst%3DMookda%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D17082020%26date%3D02102020%26volume%3D85%26issue%3D20%26spage%3D13330%26epage%3D13338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Baowen Qi, Ling Zhong, Jun He, Hongjia Zhang, Fengqiong Li, Ting Wang, Jing Zou, Yao-Xin Lin, Chengchen Zhang, Xiaoqiang Guo, Rui Li, <span class="NLM_string-name hlFld-ContribAuthor">Jianyou Shi</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Inhibitors of Aurora/PLK Targets as Anticancer Agents. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (17)
                                     , 7697-7707. <a href="https://doi.org/10.1021/acs.jmedchem.9b00353" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00353</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00353%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BInhibitors%252Bof%252BAurora%25252FPLK%252BTargets%252Bas%252BAnticancer%252BAgents%26aulast%3DQi%26aufirst%3DBaowen%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D26022019%26date%3D21082019%26date%3D05082019%26volume%3D62%26issue%3D17%26spage%3D7697%26epage%3D7707" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Richard Lamb, Vincent L. Revil-Baudard, <span class="NLM_string-name hlFld-ContribAuthor">Samir Z. Zard</span>. </span><span class="cited-content_cbyCitation_article-title">A Direct Approach to Orthogonally Protected α-Amino Aldehydes. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2019,</strong> <em>21 </em>
                                    (16)
                                     , 6352-6356. <a href="https://doi.org/10.1021/acs.orglett.9b02237" title="DOI URL">https://doi.org/10.1021/acs.orglett.9b02237</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.9b02237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.9b02237%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DA%252BDirect%252BApproach%252Bto%252BOrthogonally%252BProtected%252B%2525CE%2525B1-Amino%252BAldehydes%26aulast%3DLamb%26aufirst%3DRichard%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D29062019%26date%3D25072019%26volume%3D21%26issue%3D16%26spage%3D6352%26epage%3D6356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xiaodong Dou, Huixia Huang, Yibo Li, Lan Jiang, Yanxing Wang, Hongwei Jin, Ning Jiao, Lihe Zhang, Liangren Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Zhenming Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Multistage Screening Reveals 3-Substituted Indolin-2-one Derivatives as Novel and Isoform-Selective c-Jun N-terminal Kinase 3 (JNK3) Inhibitors: Implications to Drug Discovery for Potential Treatment of Neurodegenerative Diseases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (14)
                                     , 6645-6664. <a href="https://doi.org/10.1021/acs.jmedchem.9b00537" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00537</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00537%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DMultistage%252BScreening%252BReveals%252B3-Substituted%252BIndolin-2-one%252BDerivatives%252Bas%252BNovel%252Band%252BIsoform-Selective%252Bc-Jun%252BN-terminal%252BKinase%252B3%252B%252528JNK3%252529%252BInhibitors%25253A%252BImplications%252Bto%252BDrug%252BDiscovery%252Bfor%252BPotential%252BTreatment%252Bof%252BNeurodegenerative%252BDiseases%26aulast%3DDou%26aufirst%3DXiaodong%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D27032019%26date%3D15072019%26date%3D03072019%26volume%3D62%26issue%3D14%26spage%3D6645%26epage%3D6664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yue  Kong</span>, <span class="hlFld-ContribAuthor ">Andreas  Bender</span>, and <span class="hlFld-ContribAuthor ">Aixia  Yan</span>  . </span><span class="cited-content_cbyCitation_article-title">Identification of Novel Aurora Kinase A (AURKA) Inhibitors via Hierarchical Ligand-Based Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (1)
                                     , 36-47. <a href="https://doi.org/10.1021/acs.jcim.7b00300" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00300</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00300%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DIdentification%252Bof%252BNovel%252BAurora%252BKinase%252BA%252B%252528AURKA%252529%252BInhibitors%252Bvia%252BHierarchical%252BLigand-Based%252BVirtual%252BScreening%26aulast%3DKong%26aufirst%3DYue%26date%3D2018%26date%3D2017%26date%3D2017%26date%3D31052017%26date%3D21122017%26date%3D22012018%26date%3D04122017%26volume%3D58%26issue%3D1%26spage%3D36%26epage%3D47" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Swarit  Jasial</span>, <span class="hlFld-ContribAuthor ">Ye  Hu</span>, and <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>  . </span><span class="cited-content_cbyCitation_article-title">How Frequently Are Pan-Assay Interference Compounds Active? Large-Scale Analysis of Screening Data Reveals Diverse Activity Profiles, Low Global Hit Frequency, and Many Consistently Inactive Compounds. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (9)
                                     , 3879-3886. <a href="https://doi.org/10.1021/acs.jmedchem.7b00154" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00154</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00154%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHow%252BFrequently%252BAre%252BPan-Assay%252BInterference%252BCompounds%252BActive%25253F%252BLarge-Scale%252BAnalysis%252Bof%252BScreening%252BData%252BReveals%252BDiverse%252BActivity%252BProfiles%25252C%252BLow%252BGlobal%252BHit%252BFrequency%25252C%252Band%252BMany%252BConsistently%252BInactive%252BCompounds%26aulast%3DJasial%26aufirst%3DSwarit%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D30012017%26date%3D25042017%26date%3D11052017%26date%3D19042017%26volume%3D60%26issue%3D9%26spage%3D3879%26epage%3D3886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mahendra  Awale</span>, <span class="hlFld-ContribAuthor ">Daniel  Probst</span>, and <span class="hlFld-ContribAuthor ">Jean-Louis  Reymond</span>  . </span><span class="cited-content_cbyCitation_article-title">WebMolCS: A Web-Based Interface for Visualizing Molecules in Three-Dimensional Chemical Spaces. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (4)
                                     , 643-649. <a href="https://doi.org/10.1021/acs.jcim.6b00690" title="DOI URL">https://doi.org/10.1021/acs.jcim.6b00690</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.6b00690&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.6b00690%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DWebMolCS%25253A%252BA%252BWeb-Based%252BInterface%252Bfor%252BVisualizing%252BMolecules%252Bin%252BThree-Dimensional%252BChemical%252BSpaces%26aulast%3DAwale%26aufirst%3DMahendra%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D11112016%26date%3D30032017%26date%3D24042017%26date%3D19032017%26volume%3D57%26issue%3D4%26spage%3D643%26epage%3D649" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Courtney  Aldrich</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Carolyn  Bertozzi</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Gunda I.  Georg</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Laura  Kiessling</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Craig  Lindsley</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Dennis  Liotta</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Kenneth M.  Merz, Jr.</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Alanna  Schepartz</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Shaomeng  Wang</span> (<span class="NLM_role">Editor-in-Chief</span>)  . </span><span class="cited-content_cbyCitation_article-title">The Ecstasy and Agony of Assay Interference Compounds. </span><span class="cited-content_cbyCitation_journal-name">ACS Infectious Diseases</span><span> <strong>2017,</strong> <em>3 </em>
                                    (4)
                                     , 259-262. <a href="https://doi.org/10.1021/acsinfecdis.7b00023" title="DOI URL">https://doi.org/10.1021/acsinfecdis.7b00023</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsinfecdis.7b00023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsinfecdis.7b00023%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Infectious%2520Diseases%26atitle%3DThe%252BEcstasy%252Band%252BAgony%252Bof%252BAssay%252BInterference%252BCompounds%26aulast%3DAldrich%26aufirst%3DCourtney%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D10022017%26date%3D28022017%26date%3D14042017%26volume%3D3%26issue%3D4%26spage%3D259%26epage%3D262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Courtney  Aldrich</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Carolyn  Bertozzi</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Gunda I.  Georg</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Laura  Kiessling</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Craig  Lindsley</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Dennis  Liotta</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Kenneth M.  Merz, Jr.</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Alanna  Schepartz</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Shaomeng  Wang</span> (<span class="NLM_role">Editor-in-Chief</span>)  . </span><span class="cited-content_cbyCitation_article-title">The Ecstasy and Agony of Assay Interference Compounds. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2017,</strong> <em>8 </em>
                                    (4)
                                     , 379-382. <a href="https://doi.org/10.1021/acsmedchemlett.7b00056" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.7b00056</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.7b00056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.7b00056%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DThe%252BEcstasy%252Band%252BAgony%252Bof%252BAssay%252BInterference%252BCompounds%26aulast%3DAldrich%26aufirst%3DCourtney%26date%3D2017%26date%3D2017%26date%3D14022017%26date%3D28022017%26date%3D13042017%26volume%3D8%26issue%3D4%26spage%3D379%26epage%3D382" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Courtney  Aldrich</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Carolyn  Bertozzi</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Gunda I.  Georg</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Laura  Kiessling</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Craig  Lindsley</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Dennis  Liotta</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Kenneth M.  Merz, Jr.</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Alanna  Schepartz</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Shaomeng  Wang</span> (<span class="NLM_role">Editor-in-Chief</span>)  . </span><span class="cited-content_cbyCitation_article-title">The Ecstasy and Agony of Assay Interference Compounds. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (3)
                                     , 387-390. <a href="https://doi.org/10.1021/acs.jcim.7b00105" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00105</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00105%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DThe%252BEcstasy%252Band%252BAgony%252Bof%252BAssay%252BInterference%252BCompounds%26aulast%3DAldrich%26aufirst%3DCourtney%26date%3D2017%26date%3D2017%26date%3D24022017%26date%3D28022017%26date%3D27032017%26volume%3D57%26issue%3D3%26spage%3D387%26epage%3D390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Courtney  Aldrich</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Carolyn  Bertozzi</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Gunda I.  Georg</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Laura  Kiessling</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Craig  Lindsley</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Dennis  Liotta</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Kenneth M.  Merz, Jr.</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Alanna  Schepartz</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Shaomeng  Wang</span> (<span class="NLM_role">Editor-in-Chief</span>)  . </span><span class="cited-content_cbyCitation_article-title">The Ecstasy and Agony of Assay Interference Compounds. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (6)
                                     , 2165-2168. <a href="https://doi.org/10.1021/acs.jmedchem.7b00229" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00229</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00229&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00229%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThe%252BEcstasy%252Band%252BAgony%252Bof%252BAssay%252BInterference%252BCompounds%26aulast%3DAldrich%26aufirst%3DCourtney%26date%3D2017%26date%3D2017%26date%3D15022017%26date%3D28022017%26date%3D23032017%26volume%3D60%26issue%3D6%26spage%3D2165%26epage%3D2168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Courtney  Aldrich</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Carolyn  Bertozzi</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Gunda I.  Georg</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Laura  Kiessling</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Craig  Lindsley</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Dennis  Liotta</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Kenneth M.  Merz, Jr.</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Alanna  Schepartz</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Shaomeng  Wang</span> (<span class="NLM_role">Editor-in-Chief</span>)  . </span><span class="cited-content_cbyCitation_article-title">The Ecstasy and Agony of Assay Interference Compounds. </span><span class="cited-content_cbyCitation_journal-name">ACS Central Science</span><span> <strong>2017,</strong> <em>3 </em>
                                    (3)
                                     , 143-147. <a href="https://doi.org/10.1021/acscentsci.7b00069" title="DOI URL">https://doi.org/10.1021/acscentsci.7b00069</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscentsci.7b00069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscentsci.7b00069%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Central%2520Science%26atitle%3DThe%252BEcstasy%252Band%252BAgony%252Bof%252BAssay%252BInterference%252BCompounds%26aulast%3DAldrich%26aufirst%3DCourtney%26date%3D2017%26date%3D2017%26date%3D09022017%26date%3D28022017%26date%3D22032017%26volume%3D3%26issue%3D3%26spage%3D143%26epage%3D147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Courtney  Aldrich</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Carolyn  Bertozzi</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Gunda I.  Georg</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Laura  Kiessling</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Craig  Lindsley</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Dennis  Liotta</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Kenneth M.  Merz, Jr.</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Alanna  Schepartz</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Shaomeng  Wang</span> (<span class="NLM_role">Editor-in-Chief</span>)  . </span><span class="cited-content_cbyCitation_article-title">The Ecstasy and Agony of Assay Interference Compounds. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2017,</strong> <em>12 </em>
                                    (3)
                                     , 575-578. <a href="https://doi.org/10.1021/acschembio.7b00119" title="DOI URL">https://doi.org/10.1021/acschembio.7b00119</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.7b00119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.7b00119%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DThe%252BEcstasy%252Band%252BAgony%252Bof%252BAssay%252BInterference%252BCompounds%26aulast%3DAldrich%26aufirst%3DCourtney%26date%3D2017%26date%3D2017%26date%3D10022017%26date%3D28022017%26date%3D17032017%26volume%3D12%26issue%3D3%26spage%3D575%26epage%3D578" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Courtney  Aldrich</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Carolyn  Bertozzi</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Gunda I.  Georg</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Laura  Kiessling</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Craig  Lindsley</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Dennis  Liotta</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Kenneth M.  Merz, Jr.</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Alanna  Schepartz</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Shaomeng  Wang</span> (<span class="NLM_role">Editor-in-Chief</span>)  . </span><span class="cited-content_cbyCitation_article-title">The Ecstasy and Agony of Assay Interference Compounds. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Neuroscience</span><span> <strong>2017,</strong> <em>8 </em>
                                    (3)
                                     , 420-423. <a href="https://doi.org/10.1021/acschemneuro.7b00064" title="DOI URL">https://doi.org/10.1021/acschemneuro.7b00064</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschemneuro.7b00064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschemneuro.7b00064%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Neuroscience%26atitle%3DThe%252BEcstasy%252Band%252BAgony%252Bof%252BAssay%252BInterference%252BCompounds%26aulast%3DAldrich%26aufirst%3DCourtney%26date%3D2017%26date%3D2017%26date%3D10022017%26date%3D28022017%26date%3D15032017%26volume%3D8%26issue%3D3%26spage%3D420%26epage%3D423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Courtney  Aldrich</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Carolyn  Bertozzi</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Gunda I.  Georg</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Laura  Kiessling</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Craig  Lindsley</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Dennis  Liotta</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Kenneth M.  Merz, Jr.</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Alanna  Schepartz</span> (<span class="NLM_role">Editor-in-Chief</span>)  , <span class="hlFld-ContribAuthor ">Shaomeng  Wang</span> (<span class="NLM_role">Editor-in-Chief</span>)  . </span><span class="cited-content_cbyCitation_article-title">The Ecstasy and Agony of Assay Interference Compounds. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2017,</strong> <em>56 </em>
                                    (10)
                                     , 1363-1366. <a href="https://doi.org/10.1021/acs.biochem.7b00110" title="DOI URL">https://doi.org/10.1021/acs.biochem.7b00110</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.7b00110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.7b00110%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DThe%252BEcstasy%252Band%252BAgony%252Bof%252BAssay%252BInterference%252BCompounds%26aulast%3DAldrich%26aufirst%3DCourtney%26date%3D2017%26date%3D2017%26date%3D11022017%26date%3D28022017%26date%3D14032017%26volume%3D56%26issue%3D10%26spage%3D1363%26epage%3D1366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maria  Winiewska-Szajewska</span>, <span class="hlFld-ContribAuthor ">Agnieszka Monika  Maciejewska</span>, <span class="hlFld-ContribAuthor ">Elżbieta  Speina</span>, <span class="hlFld-ContribAuthor ">Jarosław  Poznański</span>, <span class="hlFld-ContribAuthor ">Daniel  Paprocki</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of Novel Halogenated Heterocycles Based on o-Phenylenediamine and Their Interactions with the Catalytic Subunit of Protein Kinase CK2. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (11)
                                     , 3163. <a href="https://doi.org/10.3390/molecules26113163" title="DOI URL">https://doi.org/10.3390/molecules26113163</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26113163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26113163%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DSynthesis%252Bof%252BNovel%252BHalogenated%252BHeterocycles%252BBased%252Bon%252Bo-Phenylenediamine%252Band%252BTheir%252BInteractions%252Bwith%252Bthe%252BCatalytic%252BSubunit%252Bof%252BProtein%252BKinase%252BCK2%26aulast%3DWiniewska-Szajewska%26aufirst%3DMaria%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D11%26spage%3D3163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Haiwen  Meng</span>, <span class="hlFld-ContribAuthor ">Kunhui  Sun</span>, <span class="hlFld-ContribAuthor ">Zhimin  Xu</span>, <span class="hlFld-ContribAuthor ">Lifang  Tian</span>, <span class="hlFld-ContribAuthor ">Yahui  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">P(III)‐Assisted Electrochemical Access to Ureas via in situ Generation of Isocyanates from Hydroxamic Acids. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Organic Chemistry</span><span> <strong>2021,</strong> <em>2021 </em>
                                    (11)
                                     , 1768-1772. <a href="https://doi.org/10.1002/ejoc.202100113" title="DOI URL">https://doi.org/10.1002/ejoc.202100113</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ejoc.202100113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fejoc.202100113%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DP%252528III%252529%2525E2%252580%252590Assisted%252BElectrochemical%252BAccess%252Bto%252BUreas%252Bvia%252Bin%2525E2%252580%252585situ%252BGeneration%252Bof%252BIsocyanates%252Bfrom%252BHydroxamic%252BAcids%26aulast%3DMeng%26aufirst%3DHaiwen%26date%3D2021%26date%3D2021%26volume%3D2021%26issue%3D11%26spage%3D1768%26epage%3D1772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">D.  Faixová</span>, <span class="hlFld-ContribAuthor ">G.  Hrčková</span>, <span class="hlFld-ContribAuthor ">T.  Mačák Kubašková</span>, <span class="hlFld-ContribAuthor ">D.  Mudroňová</span>. </span><span class="cited-content_cbyCitation_article-title">Antiparasitic effects of selected isoflavones on flatworms. </span><span class="cited-content_cbyCitation_journal-name">Helminthologia</span><span> <strong>2021,</strong> <em>58 </em>
                                    (1)
                                     , 1-16. <a href="https://doi.org/10.2478/helm-2021-0004" title="DOI URL">https://doi.org/10.2478/helm-2021-0004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2478/helm-2021-0004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2478%2Fhelm-2021-0004%26sid%3Dliteratum%253Aachs%26jtitle%3DHelminthologia%26atitle%3DAntiparasitic%252Beffects%252Bof%252Bselected%252Bisoflavones%252Bon%252Bflatworms%26aulast%3DFaixov%25C3%25A1%26aufirst%3DD.%26date%3D2021%26date%3D2021%26volume%3D58%26issue%3D1%26spage%3D1%26epage%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jasim Ali  Abdullah</span>, <span class="hlFld-ContribAuthor ">Bilal J M  Aldahham</span>, <span class="hlFld-ContribAuthor ">Muwafaq Ayesh  Rabeea</span>, <span class="hlFld-ContribAuthor ">Fatmah Ali  Asmary</span>, <span class="hlFld-ContribAuthor ">Hassna Mohammed  Alhajri</span>, <span class="hlFld-ContribAuthor ">Md Ataul  Islam</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, characterization and in-silico assessment of novel thiazolidinone derivatives for cyclin-dependent kinases-2 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Structure</span><span> <strong>2021,</strong> <em>1223 </em>, 129311. <a href="https://doi.org/10.1016/j.molstruc.2020.129311" title="DOI URL">https://doi.org/10.1016/j.molstruc.2020.129311</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molstruc.2020.129311&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molstruc.2020.129311%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Structure%26atitle%3DSynthesis%25252C%252Bcharacterization%252Band%252Bin-silico%252Bassessment%252Bof%252Bnovel%252Bthiazolidinone%252Bderivatives%252Bfor%252Bcyclin-dependent%252Bkinases-2%252Binhibitors%26aulast%3DAbdullah%26aufirst%3DJasim%2BAli%26date%3D2021%26volume%3D1223%26spage%3D129311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Janine L.  Gray</span>, <span class="hlFld-ContribAuthor ">Frank  Delft</span>, <span class="hlFld-ContribAuthor ">Paul E.  Brennan</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting der kleinen GTPasen über ihre regulatorischen Proteine. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2020,</strong> <em>132 </em>
                                    (16)
                                     , 6402-6428. <a href="https://doi.org/10.1002/ange.201900585" title="DOI URL">https://doi.org/10.1002/ange.201900585</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.201900585&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.201900585%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DTargeting%252Bder%252Bkleinen%252BGTPasen%252B%2525C3%2525BCber%252Bihre%252Bregulatorischen%252BProteine%26aulast%3DGray%26aufirst%3DJanine%2BL.%26date%3D2020%26date%3D2020%26volume%3D132%26issue%3D16%26spage%3D6402%26epage%3D6428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Janine L.  Gray</span>, <span class="hlFld-ContribAuthor ">Frank  Delft</span>, <span class="hlFld-ContribAuthor ">Paul E.  Brennan</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting the Small GTPase Superfamily through Their Regulatory Proteins. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2020,</strong> <em>59 </em>
                                    (16)
                                     , 6342-6366. <a href="https://doi.org/10.1002/anie.201900585" title="DOI URL">https://doi.org/10.1002/anie.201900585</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.201900585&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.201900585%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DTargeting%252Bthe%252BSmall%252BGTPase%252BSuperfamily%252Bthrough%252BTheir%252BRegulatory%252BProteins%26aulast%3DGray%26aufirst%3DJanine%2BL.%26date%3D2020%26date%3D2020%26volume%3D59%26issue%3D16%26spage%3D6342%26epage%3D6366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yifan  Lin</span>, <span class="hlFld-ContribAuthor ">Erfei  Li</span>, <span class="hlFld-ContribAuthor ">Xin  Wu</span>, <span class="hlFld-ContribAuthor ">Luyu  Wang</span>, <span class="hlFld-ContribAuthor ">Haofeng  Wang</span>, <span class="hlFld-ContribAuthor ">Xiaoyu  Li</span>, <span class="hlFld-ContribAuthor ">Honglan  Kang</span>, <span class="hlFld-ContribAuthor ">Liejin  Zhou</span>, <span class="hlFld-ContribAuthor ">Guodong  Shen</span>, <span class="hlFld-ContribAuthor ">Xin  Lv</span>. </span><span class="cited-content_cbyCitation_article-title">One-pot synthesis of 2-azolylimidazole derivatives through a domino addition/A
              3
              coupling/cyclization process under copper catalysis. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2020,</strong> <em>18 </em>
                                    (7)
                                     , 1476-1486. <a href="https://doi.org/10.1039/C9OB02532D" title="DOI URL">https://doi.org/10.1039/C9OB02532D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9OB02532D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9OB02532D%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DOne-pot%252Bsynthesis%252Bof%252B2-azolylimidazole%252Bderivatives%252Bthrough%252Ba%252Bdomino%252Baddition%25252FA%252B3%252Bcoupling%25252Fcyclization%252Bprocess%252Bunder%252Bcopper%252Bcatalysis%26aulast%3DLin%26aufirst%3DYifan%26date%3D2020%26date%3D2020%26volume%3D18%26issue%3D7%26spage%3D1476%26epage%3D1486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ahmed M.  Shawky</span>, <span class="hlFld-ContribAuthor ">Mohammed A.S.  Abourehab</span>, <span class="hlFld-ContribAuthor ">Ashraf N.  Abdalla</span>, <span class="hlFld-ContribAuthor ">Ahmed M.  Gouda</span>. </span><span class="cited-content_cbyCitation_article-title">Optimization of pyrrolizine-based Schiff bases with 4-thiazolidinone motif: Design, synthesis and investigation of cytotoxicity and anti-inflammatory potency. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>185 </em>, 111780. <a href="https://doi.org/10.1016/j.ejmech.2019.111780" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.111780</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.111780&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.111780%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOptimization%252Bof%252Bpyrrolizine-based%252BSchiff%252Bbases%252Bwith%252B4-thiazolidinone%252Bmotif%25253A%252BDesign%25252C%252Bsynthesis%252Band%252Binvestigation%252Bof%252Bcytotoxicity%252Band%252Banti-inflammatory%252Bpotency%26aulast%3DShawky%26aufirst%3DAhmed%2BM.%26date%3D2020%26volume%3D185%26spage%3D111780" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Benquan  Liu</span>, <span class="hlFld-ContribAuthor ">Huiqin  He</span>, <span class="hlFld-ContribAuthor ">Hongyi  Luo</span>, <span class="hlFld-ContribAuthor ">Tingting  Zhang</span>, <span class="hlFld-ContribAuthor ">Jingwei  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Artificial intelligence and big data facilitated targeted drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Stroke and Vascular Neurology</span><span> <strong>2019,</strong> <em>4 </em>
                                    (4)
                                     , 206-213. <a href="https://doi.org/10.1136/svn-2019-000290" title="DOI URL">https://doi.org/10.1136/svn-2019-000290</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1136/svn-2019-000290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1136%2Fsvn-2019-000290%26sid%3Dliteratum%253Aachs%26jtitle%3DStroke%2520and%2520Vascular%2520Neurology%26atitle%3DArtificial%252Bintelligence%252Band%252Bbig%252Bdata%252Bfacilitated%252Btargeted%252Bdrug%252Bdiscovery%26aulast%3DLiu%26aufirst%3DBenquan%26date%3D2019%26date%3D2019%26volume%3D4%26issue%3D4%26spage%3D206%26epage%3D213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yorley  Duarte</span>, <span class="hlFld-ContribAuthor ">Valeria  Márquez‐Miranda</span>, <span class="hlFld-ContribAuthor ">Matthieu J.  Miossec</span>, <span class="hlFld-ContribAuthor ">Fernando  González‐Nilo</span>. </span><span class="cited-content_cbyCitation_article-title">Integration of target discovery, drug discovery and drug delivery: A review on computational strategies. </span><span class="cited-content_cbyCitation_journal-name">Wiley Interdisciplinary Reviews: Nanomedicine and Nanobiotechnology</span><span> <strong>2019,</strong> <em>11 </em>
                                    (4)
                                     <a href="https://doi.org/10.1002/wnan.1554" title="DOI URL">https://doi.org/10.1002/wnan.1554</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/wnan.1554&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fwnan.1554%26sid%3Dliteratum%253Aachs%26jtitle%3DWiley%2520Interdisciplinary%2520Reviews%253A%2520Nanomedicine%2520and%2520Nanobiotechnology%26atitle%3DIntegration%252Bof%252Btarget%252Bdiscovery%25252C%252Bdrug%252Bdiscovery%252Band%252Bdrug%252Bdelivery%25253A%252BA%252Breview%252Bon%252Bcomputational%252Bstrategies%26aulast%3DDuarte%26aufirst%3DYorley%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Péter  Bana</span>, <span class="hlFld-ContribAuthor ">Áron  Szigetvári</span>, <span class="hlFld-ContribAuthor ">János  Kóti</span>, <span class="hlFld-ContribAuthor ">János  Éles</span>, <span class="hlFld-ContribAuthor ">István  Greiner</span>. </span><span class="cited-content_cbyCitation_article-title">Flow-oriented synthetic design in the continuous preparation of the aryl piperazine drug flibanserin. </span><span class="cited-content_cbyCitation_journal-name">Reaction Chemistry & Engineering</span><span> <strong>2019,</strong> <em>4 </em>
                                    (4)
                                     , 652-657. <a href="https://doi.org/10.1039/C8RE00266E" title="DOI URL">https://doi.org/10.1039/C8RE00266E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8RE00266E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8RE00266E%26sid%3Dliteratum%253Aachs%26jtitle%3DReaction%2520Chemistry%2520%2526%2520Engineering%26atitle%3DFlow-oriented%252Bsynthetic%252Bdesign%252Bin%252Bthe%252Bcontinuous%252Bpreparation%252Bof%252Bthe%252Baryl%252Bpiperazine%252Bdrug%252Bflibanserin%26aulast%3DBana%26aufirst%3DP%25C3%25A9ter%26date%3D2019%26date%3D2019%26volume%3D4%26issue%3D4%26spage%3D652%26epage%3D657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Clémence  Delalande</span>, <span class="hlFld-ContribAuthor ">Mahendra  Awale</span>, <span class="hlFld-ContribAuthor ">Matthias  Rubin</span>, <span class="hlFld-ContribAuthor ">Daniel  Probst</span>, <span class="hlFld-ContribAuthor ">Lijo C.  Ozhathil</span>, <span class="hlFld-ContribAuthor ">Jürg  Gertsch</span>, <span class="hlFld-ContribAuthor ">Hugues  Abriel</span>, <span class="hlFld-ContribAuthor ">Jean-Louis  Reymond</span>. </span><span class="cited-content_cbyCitation_article-title">Optimizing TRPM4 inhibitors in the MHFP6 chemical space. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>166 </em>, 167-177. <a href="https://doi.org/10.1016/j.ejmech.2019.01.048" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.01.048</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.01.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.01.048%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOptimizing%252BTRPM4%252Binhibitors%252Bin%252Bthe%252BMHFP6%252Bchemical%252Bspace%26aulast%3DDelalande%26aufirst%3DCl%25C3%25A9mence%26date%3D2019%26volume%3D166%26spage%3D167%26epage%3D177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marion  Poirier</span>, <span class="hlFld-ContribAuthor ">Mahendra  Awale</span>, <span class="hlFld-ContribAuthor ">Matthias A.  Roelli</span>, <span class="hlFld-ContribAuthor ">Guy T.  Giuffredi</span>, <span class="hlFld-ContribAuthor ">Lars  Ruddigkeit</span>, <span class="hlFld-ContribAuthor ">Lasse  Evensen</span>, <span class="hlFld-ContribAuthor ">Amandine  Stooss</span>, <span class="hlFld-ContribAuthor ">Serafina  Calarco</span>, <span class="hlFld-ContribAuthor ">James B.  Lorens</span>, <span class="hlFld-ContribAuthor ">Roch-Philippe  Charles</span>, <span class="hlFld-ContribAuthor ">Jean-Louis  Reymond</span>. </span><span class="cited-content_cbyCitation_article-title">Identifying Lysophosphatidic Acid Acyltransferase β (LPAAT-β) as the Target of a Nanomolar Angiogenesis Inhibitor from a Phenotypic Screen Using the Polypharmacology Browser PPB2. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2019,</strong> <em>14 </em>
                                    (2)
                                     , 224-236. <a href="https://doi.org/10.1002/cmdc.201800554" title="DOI URL">https://doi.org/10.1002/cmdc.201800554</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201800554&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201800554%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DIdentifying%252BLysophosphatidic%252BAcid%252BAcyltransferase%2525E2%252580%252585%2525CE%2525B2%252B%252528LPAAT-%2525CE%2525B2%252529%252Bas%252Bthe%252BTarget%252Bof%252Ba%252BNanomolar%252BAngiogenesis%252BInhibitor%252Bfrom%252Ba%252BPhenotypic%252BScreen%252BUsing%252Bthe%252BPolypharmacology%252BBrowser%252BPPB2%26aulast%3DPoirier%26aufirst%3DMarion%26date%3D2019%26date%3D2018%26volume%3D14%26issue%3D2%26spage%3D224%26epage%3D236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Magambo Phillip  Kimuda</span>, <span class="hlFld-ContribAuthor ">Dustin  Laming</span>, <span class="hlFld-ContribAuthor ">Heinrich C.  Hoppe</span>, <span class="hlFld-ContribAuthor ">Özlem  Tastan Bishop</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of Novel Potential Inhibitors of Pteridine Reductase 1 in Trypanosoma brucei via Computational Structure-Based Approaches and in Vitro Inhibition Assays. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (1)
                                     , 142. <a href="https://doi.org/10.3390/molecules24010142" title="DOI URL">https://doi.org/10.3390/molecules24010142</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24010142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24010142%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DIdentification%252Bof%252BNovel%252BPotential%252BInhibitors%252Bof%252BPteridine%252BReductase%252B1%252Bin%252BTrypanosoma%252Bbrucei%252Bvia%252BComputational%252BStructure-Based%252BApproaches%252Band%252Bin%252BVitro%252BInhibition%252BAssays%26aulast%3DKimuda%26aufirst%3DMagambo%2BPhillip%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D1%26spage%3D142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiao-Long  Qiu</span>, <span class="hlFld-ContribAuthor ">Shao-Hua  Gou</span>, <span class="hlFld-ContribAuthor ">Wanguo  Wei</span>, <span class="hlFld-ContribAuthor ">Ping  Zou</span>, <span class="hlFld-ContribAuthor ">Lin  Hu</span>, <span class="hlFld-ContribAuthor ">Xi-Meng  Yan</span>, <span class="hlFld-ContribAuthor ">Xiao-Yue  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Practical Synthesis of Pimobendan. </span><span class="cited-content_cbyCitation_journal-name">HETEROCYCLES</span><span> <strong>2019,</strong> <em>98 </em>
                                    (5)
                                     , 674. <a href="https://doi.org/10.3987/COM-19-14059" title="DOI URL">https://doi.org/10.3987/COM-19-14059</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3987/COM-19-14059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3987%2FCOM-19-14059%26sid%3Dliteratum%253Aachs%26jtitle%3DHETEROCYCLES%26atitle%3DPractical%252BSynthesis%252Bof%252BPimobendan%26aulast%3DQiu%26aufirst%3DXiao-Long%26date%3D2019%26volume%3D98%26issue%3D5%26spage%3D674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wei  Wang</span>, <span class="hlFld-ContribAuthor ">Xiu  Feng</span>, <span class="hlFld-ContribAuthor ">Huan-Xiang  Liu</span>, <span class="hlFld-ContribAuthor ">Shi-Wu  Chen</span>, <span class="hlFld-ContribAuthor ">Ling  Hui</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and biological evaluation of 2,4-disubstituted phthalazinones as Aurora kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2018,</strong> <em>26 </em>
                                    (12)
                                     , 3217-3226. <a href="https://doi.org/10.1016/j.bmc.2018.04.048" title="DOI URL">https://doi.org/10.1016/j.bmc.2018.04.048</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2018.04.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2018.04.048%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252B2%25252C4-disubstituted%252Bphthalazinones%252Bas%252BAurora%252Bkinase%252Binhibitors%26aulast%3DWang%26aufirst%3DWei%26date%3D2018%26volume%3D26%26issue%3D12%26spage%3D3217%26epage%3D3226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Warintra  Pitsawong</span>, <span class="hlFld-ContribAuthor ">Vanessa  Buosi</span>, <span class="hlFld-ContribAuthor ">Renee  Otten</span>, <span class="hlFld-ContribAuthor ">Roman V  Agafonov</span>, <span class="hlFld-ContribAuthor ">Adelajda  Zorba</span>, <span class="hlFld-ContribAuthor ">Nadja  Kern</span>, <span class="hlFld-ContribAuthor ">Steffen  Kutter</span>, <span class="hlFld-ContribAuthor ">Gunther  Kern</span>, <span class="hlFld-ContribAuthor ">Ricardo AP  Pádua</span>, <span class="hlFld-ContribAuthor ">Xavier  Meniche</span>, <span class="hlFld-ContribAuthor ">Dorothee  Kern</span>. </span><span class="cited-content_cbyCitation_article-title">Dynamics of human protein kinase Aurora A linked to drug selectivity. </span><span class="cited-content_cbyCitation_journal-name">eLife</span><span> <strong>2018,</strong> <em>7 </em><a href="https://doi.org/10.7554/eLife.36656" title="DOI URL">https://doi.org/10.7554/eLife.36656</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.7554/eLife.36656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.7554%2FeLife.36656%26sid%3Dliteratum%253Aachs%26jtitle%3DeLife%26atitle%3DDynamics%252Bof%252Bhuman%252Bprotein%252Bkinase%252BAurora%252BA%252Blinked%252Bto%252Bdrug%252Bselectivity%26aulast%3DPitsawong%26aufirst%3DWarintra%26date%3D2018%26date%3D2018%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lijo Cherian  Ozhathil</span>, <span class="hlFld-ContribAuthor ">Clémence  Delalande</span>, <span class="hlFld-ContribAuthor ">Beatrice  Bianchi</span>, <span class="hlFld-ContribAuthor ">Gabor  Nemeth</span>, <span class="hlFld-ContribAuthor ">Sven  Kappel</span>, <span class="hlFld-ContribAuthor ">Urs  Thomet</span>, <span class="hlFld-ContribAuthor ">Daniela  Ross-Kaschitza</span>, <span class="hlFld-ContribAuthor ">Céline  Simonin</span>, <span class="hlFld-ContribAuthor ">Matthias  Rubin</span>, <span class="hlFld-ContribAuthor ">Jürg  Gertsch</span>, <span class="hlFld-ContribAuthor ">Martin  Lochner</span>, <span class="hlFld-ContribAuthor ">Christine  Peinelt</span>, <span class="hlFld-ContribAuthor ">Jean-Louis  Reymond</span>, <span class="hlFld-ContribAuthor ">Hugues  Abriel</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of potent and selective small molecule inhibitors of the cation channel TRPM4. </span><span class="cited-content_cbyCitation_journal-name">British Journal of Pharmacology</span><span> <strong>2018,</strong> <em>175 </em>
                                    (12)
                                     , 2504-2519. <a href="https://doi.org/10.1111/bph.14220" title="DOI URL">https://doi.org/10.1111/bph.14220</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/bph.14220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fbph.14220%26sid%3Dliteratum%253Aachs%26jtitle%3DBritish%2520Journal%2520of%2520Pharmacology%26atitle%3DIdentification%252Bof%252Bpotent%252Band%252Bselective%252Bsmall%252Bmolecule%252Binhibitors%252Bof%252Bthe%252Bcation%252Bchannel%252BTRPM4%26aulast%3DOzhathil%26aufirst%3DLijo%2BCherian%26date%3D2018%26date%3D2018%26volume%3D175%26issue%3D12%26spage%3D2504%26epage%3D2519" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pooja R.  Tambe</span>, <span class="hlFld-ContribAuthor ">Ganapati D.  Yadav</span>. </span><span class="cited-content_cbyCitation_article-title">Selective carbonylation of o-phenylene diamine using carbon dioxide as feedstock for synthesis of 1, 3-dihydro-benzimidazol-2-one over La-Zr mixed oxide. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cleaner Production</span><span> <strong>2017,</strong> <em>166 </em>, 285-298. <a href="https://doi.org/10.1016/j.jclepro.2017.08.025" title="DOI URL">https://doi.org/10.1016/j.jclepro.2017.08.025</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jclepro.2017.08.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jclepro.2017.08.025%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cleaner%2520Production%26atitle%3DSelective%252Bcarbonylation%252Bof%252Bo-phenylene%252Bdiamine%252Busing%252Bcarbon%252Bdioxide%252Bas%252Bfeedstock%252Bfor%252Bsynthesis%252Bof%252B1%25252C%252B3-dihydro-benzimidazol-2-one%252Bover%252BLa-Zr%252Bmixed%252Boxide%26aulast%3DTambe%26aufirst%3DPooja%2BR.%26date%3D2017%26volume%3D166%26spage%3D285%26epage%3D298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Deelip S.  Rekunge</span>, <span class="hlFld-ContribAuthor ">Chetan K.  Khatri</span>, <span class="hlFld-ContribAuthor ">Ganesh U.  Chaturbhuj</span>. </span><span class="cited-content_cbyCitation_article-title">Sulfated polyborate-catalyzed efficient and expeditious synthesis of (un)symmetrical ureas and benzimidazolones. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron Letters</span><span> <strong>2017,</strong> <em>58 </em>
                                    (45)
                                     , 4304-4307. <a href="https://doi.org/10.1016/j.tetlet.2017.10.001" title="DOI URL">https://doi.org/10.1016/j.tetlet.2017.10.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tetlet.2017.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tetlet.2017.10.001%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%2520Letters%26atitle%3DSulfated%252Bpolyborate-catalyzed%252Befficient%252Band%252Bexpeditious%252Bsynthesis%252Bof%252B%252528un%252529symmetrical%252Bureas%252Band%252Bbenzimidazolones%26aulast%3DRekunge%26aufirst%3DDeelip%2BS.%26date%3D2017%26volume%3D58%26issue%3D45%26spage%3D4304%26epage%3D4307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Arun Prasath  Damodaran</span>, <span class="hlFld-ContribAuthor ">Lucie  Vaufrey</span>, <span class="hlFld-ContribAuthor ">Olivia  Gavard</span>, <span class="hlFld-ContribAuthor ">Claude  Prigent</span>. </span><span class="cited-content_cbyCitation_article-title">Aurora A Kinase Is a Priority Pharmaceutical Target for the Treatment of Cancers. </span><span class="cited-content_cbyCitation_journal-name">Trends in Pharmacological Sciences</span><span> <strong>2017,</strong> <em>38 </em>
                                    (8)
                                     , 687-700. <a href="https://doi.org/10.1016/j.tips.2017.05.003" title="DOI URL">https://doi.org/10.1016/j.tips.2017.05.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tips.2017.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tips.2017.05.003%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Pharmacological%2520Sciences%26atitle%3DAurora%252BA%252BKinase%252BIs%252Ba%252BPriority%252BPharmaceutical%252BTarget%252Bfor%252Bthe%252BTreatment%252Bof%252BCancers%26aulast%3DDamodaran%26aufirst%3DArun%2BPrasath%26date%3D2017%26volume%3D38%26issue%3D8%26spage%3D687%26epage%3D700" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Didier  Rognan</span>. </span><span class="cited-content_cbyCitation_article-title">The impact of in silico screening in the discovery of novel and safer drug candidates. </span><span class="cited-content_cbyCitation_journal-name">Pharmacology & Therapeutics</span><span> <strong>2017,</strong> <em>175 </em>, 47-66. <a href="https://doi.org/10.1016/j.pharmthera.2017.02.034" title="DOI URL">https://doi.org/10.1016/j.pharmthera.2017.02.034</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pharmthera.2017.02.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pharmthera.2017.02.034%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacology%2520%2526%2520Therapeutics%26atitle%3DThe%252Bimpact%252Bof%252Bin%252Bsilico%252Bscreening%252Bin%252Bthe%252Bdiscovery%252Bof%252Bnovel%252Band%252Bsafer%252Bdrug%252Bcandidates%26aulast%3DRognan%26aufirst%3DDidier%26date%3D2017%26volume%3D175%26spage%3D47%26epage%3D66" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chunlin  Zhuang</span>, <span class="hlFld-ContribAuthor ">Zhongli  Wu</span>, <span class="hlFld-ContribAuthor ">Chengguo  Xing</span>, <span class="hlFld-ContribAuthor ">Zhenyuan  Miao</span>. </span><span class="cited-content_cbyCitation_article-title">Small molecules inhibiting Keap1–Nrf2 protein–protein interactions: a novel approach to activate Nrf2 function. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2017,</strong> <em>8 </em>
                                    (2)
                                     , 286-294. <a href="https://doi.org/10.1039/C6MD00500D" title="DOI URL">https://doi.org/10.1039/C6MD00500D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C6MD00500D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC6MD00500D%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DSmall%252Bmolecules%252Binhibiting%252BKeap1%2525E2%252580%252593Nrf2%252Bprotein%2525E2%252580%252593protein%252Binteractions%25253A%252Ba%252Bnovel%252Bapproach%252Bto%252Bactivate%252BNrf2%252Bfunction%26aulast%3DZhuang%26aufirst%3DChunlin%26date%3D2017%26date%3D2017%26volume%3D8%26issue%3D2%26spage%3D286%26epage%3D294" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chaeuk  Im</span>. </span><span class="cited-content_cbyCitation_article-title">Docking and three-dimensional quantitative structure–activity relationship analyses of imidazole and thiazolidine derivatives as Aurora A kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Archives of Pharmacal Research</span><span> <strong>2016,</strong> <em>39 </em>
                                    (12)
                                     , 1635-1643. <a href="https://doi.org/10.1007/s12272-016-0870-1" title="DOI URL">https://doi.org/10.1007/s12272-016-0870-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s12272-016-0870-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs12272-016-0870-1%26sid%3Dliteratum%253Aachs%26jtitle%3DArchives%2520of%2520Pharmacal%2520Research%26atitle%3DDocking%252Band%252Bthree-dimensional%252Bquantitative%252Bstructure%2525E2%252580%252593activity%252Brelationship%252Banalyses%252Bof%252Bimidazole%252Band%252Bthiazolidine%252Bderivatives%252Bas%252BAurora%252BA%252Bkinase%252Binhibitors%26aulast%3DIm%26aufirst%3DChaeuk%26date%3D2016%26date%3D2016%26volume%3D39%26issue%3D12%26spage%3D1635%26epage%3D1643" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0001.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of reference kinase inhibitors and new inhibitors discussed in this paper.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0002.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Kinase library design and screening. (a) Combined frequency histogram as a function of xLOS score (black line, ∼1.2 million commercial cpds from two catalogs, right vertical axis) and scatter plot of substructure similarity (TSfp: Tanimoto coefficient of a 1024-bit Daylight type substructure fingerprint) versus xLOS score for the selected 437 screening compounds (black dots) and active Aurora A hits (red dots) against their most similar reference inhibitor. (b) Combined frequency histogram and similarity scatter plot as (a) for 212 compounds selected in the second round of virtual screening compared to their reference first round hit. (c) Property profile of kinase screening sets and inhibitors in terms of heavy atom count (HAC), hydrogen-bond acceptor atoms (HBA), hydrogen-bond donor atoms (HBD), and rotatable bond count (RBC). See <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> for details of the composition of the sets.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0009.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Benzimidazole Hit <b>5</b> and Derivatives</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0010.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Further Analogs of Benzimidazole <b>5</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0011.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Phenylimino-thiazolidinones</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0012.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of further thiazolidinone analogs</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0003.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Indirubin-like scaffold <b>5</b>. (a) Structure–activity relationship profile. Aurora A inhibition was measured using an HTRF assay (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). (b) Crystal structure of the Aurora A ATP-binding site with bound <b>6</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZS0">4ZS0</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_001.pdf" class="ext-link">Supplementary Table S1</a>), with highlighted salt bridge between residues Lys162 and Glu181 (blue), DFG motif (pink) ,and HRD motif (yellow) characteristic of an active kinase conformation. (c) Schematic of contact residues between <b>6</b> and Aurora A. Dashed lines indicate hydrogen bonds, and red arcs represent hydrophobic interactions.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0004.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Optimization of the phenylimino-thiazolidinone hit <b>7</b> to the selective Aurora A inhibitor <b>9</b>. (a) Structure and activities of analogs with variations of R<sup>1</sup> and R<sup>2</sup>. (b) Optimization of the 2-pyridyl substituent. SI = selectivity index: IC<sub>50</sub> (Aurora B/INCENP)/IC<sub>50</sub> (Aurora A). (c) Structure and IC<sub>50</sub> curves for Aurora A and Aurora B/INCENP of the most potent inhibitor <b>9</b>. (d) Kinome profiling and <i>K</i><sub>D</sub> values for <b>9</b> toward its nine best target kinases.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0005.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Crystal structures of the 4-thiazolidinone derivatives bound to Aurora A. The 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map contoured at 1σ around the compounds is shown to highlight their unambiguous position in the pocket. Lys162 and Glu181 (blue), DFG motif (pink), and HRD motif (yellow) are highlighted. The catalytic pocket of Aurora A acquires an inactive conformation upon binding to <b>77</b> (a), <b>9</b> (b) and <b>88</b> (c). The lower panels show schematic representations of the interactions between the compounds and the protein. Dashed lines indicate hydrogen bonds, in particular interactions between the aminopyridine portion of the inhibitors and the hinge region (A213) and between the anionic carboxyl or tetrazole group and the guanidinium groups of R137 and R220. Residues involved in hydrophobic interactions are highlighted with a red arc.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0006.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Binding of Aurora A to <b>9</b> excludes TPX2. (a) MST analysis of TPX2 binding to Aurora A kinase in presence and absence of compound <b>9</b>. Three independent measurements were performed; error bars represent the standard deviation (SD). (b) Aurora A kinase activity in the presence of TPX2 or the control protein BSA. (c) IC<sub>50</sub> curves for <b>9</b> in the presence of different TPX2 concentrations. (d) <b>9</b> impairs localization of Aurora A at the mitotic spindle. Representative images of HeLa Kyoto cells treated with compound <b>9</b> overnight and stained for DNA (blue), Aurora A (green), and TPX2 (red). TPX2 colocalizes with Aurora A in control cells (top row). <b>9</b> impairs Aurora A localization at the spindle microtubules but does not affect centrosomal Aurora A. Aurora A is also present at centrosomes but not on spindle microtubules in cells treated with <b>1</b> (MLN8237).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0007.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Inhibitor <b>9</b> yields an Aurora A specific inhibition phenotype in cells without impairing Aurora B function. (a) Mitotic index of HeLa Kyoto cells after overnight incubation with indicated compounds. The experiment was repeated twice and error bars represent SD. (b) Centrosomal pThr288 Aurora A levels in HeLa Kyoto cells after overnight incubation with indicated concentrations of <b>9</b>. More than 10 cells were analyzed in every condition. The experiment was repeated twice and the error bars represent SD. (c) Chromosome misalignment phenotype of HeLa Kyoto cells incubated with indicated compounds. The experiment was repeated twice and error bars represent SD. (d) Representative images of HeLa Kyoto cells after overnight incubation with 10 μM <b>9</b> or 250 nM <b>1</b>. The cells were stained for pThr288 Aurora A (red), α-tubulin (green), and DNA (blue). Impaired alignment of chromosomes on the metaphase plate is indicated by white arrows. (e) Chromosome misalignment phenotype of HeLa Kyoto cells. The cells were transfected with the indicated GFP-Aurora A constructs before incubation with 10 μM <b>9</b> overnight. The bars represent mean phenotype ± SD of two independent experiments. At least 100 cells were analyzed in every condition. Note that cells expressing the highest amount of GFP-Aurora A, GFP-Aurora A Arg137Ala, and GFP-Aurora A Trp277Ala were considered for the analysis; cells expressing lower levels of the mutant transgenes exhibited less or no rescue (data not shown). (f) Representative images of HeLa Kyoto cells after overnight incubation with indicated compounds and staining for pHis-H3 (red) and DNA (blue). (g) Western blot of lysates from cells synchronized in prometaphase with 100 nM nocodazole and treated with the indicated inhibitors. β-actin was used as loading control. (h) DNA content analysis of cells treated for 48 h with the indicated compounds. The cells were stained with propidium iodide and analyzed using flow cytometry.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/medium/jm-2016-007096_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0008.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. In vitro reactivity of <b>9</b> toward GSH. Inhibitor <b>9</b> (30 μM) was incubated with GSH (5 mM) in H<sub>2</sub>O/CH<sub>3</sub>CN (4:1) at pH = 7.0, and the reaction was followed by LC-MS. (a) Reaction scheme and structures of modified analogs. (b) LC chromatogram of the reaction mixture after <i>t</i> = 0 and 14 h. The product formed with a retention time <i>t</i><sub>ret</sub> = 1.29 min is identified by MS as the GSH-adduct of <b>9</b>. 9-methylcarbazole (<i>t</i><sub>ret</sub> = 2.44 min) was used as internal control. (c) Concentration of compound <b>9</b> during the course of the reaction as determined by integration of the absorption peaks (at 310 nm) of compound <b>9</b> at <i>t</i><sub>ret</sub> = 1.78 min.The average of two independent experiments is shown, SD is <5% for all data points. (d) Aurora A inhibition in the HTRF biochemical assay. (e) Cellular pThr288 Aurora A levels of cells incubated with <b>9</b> or <i>rac</i>-<b>92</b> overnight at the indicated concentrations. The experiment was repeated twice and the error bars represent SD. Derivative <i>rac</i>-<b>92</b> does not reduce pThr288 levels in cells, although having an IC<sub>50</sub> of 24 nM in the biochemical assay. Numbers represent average IC<sub>50</sub> ± SD of two independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-15/acs.jmedchem.6b00709/20160811/images/large/jm-2016-007096_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00709&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i122">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71727" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71727" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 87 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Virtual Ligand Screening: Strategies, Perspectives and Limitations</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">580</span><span class="NLM_x">–</span> <span class="NLM_lpage">594</span><span class="refDoi"> DOI: 10.1016/j.drudis.2006.05.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2Fj.drudis.2006.05.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=16793526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGqtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=580-594&author=G.+Klebe&title=Virtual+Ligand+Screening%3A+Strategies%2C+Perspectives+and+Limitations&doi=10.1016%2Fj.drudis.2006.05.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual ligand screening: strategies, perspectives and limitations</span></div><div class="casAuthors">Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">13 & 14</span>),
    <span class="NLM_cas:pages">580-594</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In contrast to high-throughput screening, in virtual ligand screening (VS), compds. are selected using computer programs to predict their binding to a target receptor.  A key prerequisite is knowledge about the spatial and energetic criteria responsible for protein-ligand binding.  The concepts and prerequisites to perform VS are summarized here, and explanations are sought for the enduring limitations of the technol.  Target selection, anal. and prepn. are discussed, as well as considerations about the compilation of candidate ligand libraries.  The tools and strategies of a VS campaign, and the accuracy of scoring and ranking of the results, are also considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN3dWlP-WDY7Vg90H21EOLACvtfcHk0lh065e-Eumq7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGqtLo%253D&md5=5adfd125d48082238a6c5ad0e8343c59</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.05.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.05.012%26sid%3Dliteratum%253Aachs%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DVirtual%2520Ligand%2520Screening%253A%2520Strategies%252C%2520Perspectives%2520and%2520Limitations%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2006%26volume%3D11%26spage%3D580%26epage%3D594%26doi%3D10.1016%2Fj.drudis.2006.05.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Scior, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tresadern, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medina-Franco, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez-Mayorga, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cuanalo-Contreras, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agrafiotis, D. K.</span><span> </span><span class="NLM_article-title">Recognizing Pitfalls in Virtual Screening: A Critical Review</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">867</span><span class="NLM_x">–</span> <span class="NLM_lpage">881</span><span class="refDoi"> DOI: 10.1021/ci200528d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200528d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC38Xkt1eiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=867-881&author=T.+Sciorauthor=A.+Benderauthor=G.+Tresadernauthor=J.+L.+Medina-Francoauthor=K.+Martinez-Mayorgaauthor=T.+Langerauthor=K.+Cuanalo-Contrerasauthor=D.+K.+Agrafiotis&title=Recognizing+Pitfalls+in+Virtual+Screening%3A+A+Critical+Review&doi=10.1021%2Fci200528d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Recognizing Pitfalls in Virtual Screening: A Critical Review</span></div><div class="casAuthors">Scior, Thomas; Bender, Andreas; Tresadern, Gary; Medina-Franco, Jose L.; Martinez-Mayorga, Karina; Langer, Thierry; Cuanalo-Contreras, Karina; Agrafiotis, Dimitris K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">867-881</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The aim of virtual screening (VS) is to identify bioactive compds. through computational means, by employing knowledge about the protein target (structure-based VS) or known bioactive ligands (ligand-based VS).  In VS, a large no. of mols. are ranked according to their likelihood to be bioactive compds., with the aim to enrich the top fraction of the resulting list (which can be tested in bioassays afterward).  At its core, VS attempts to improve the odds of identifying bioactive mols. by maximizing the true pos. rate, i.e., by ranking the truly active mols. as high as possible (and, correspondingly, the truly inactive ones as low as possible).  In choosing the right approach, the researcher is faced with many questions: where does the optimal balance between efficiency and accuracy lie when evaluating a particular algorithm; do some methods perform better than others and in what particular situations; and what do retrospective results tell us about the prospective utility of a particular method.  Given the multitude of settings, parameters, and data sets the practitioner can choose from, there are many pitfalls that lurk along the way which might render VS less efficient or downright useless.  This review attempts to catalog published and unpublished problems, shortcomings, failures, and tech. traps of VS methods with the aim to avoid pitfalls by making the user aware of them in the first place.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGois-3Bq6mm-LVg90H21EOLACvtfcHk0lh0LbhVGgpMqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xkt1eiurg%253D&md5=ad61b3289039c03844085a8b8de9e1e4</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fci200528d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200528d%26sid%3Dliteratum%253Aachs%26aulast%3DScior%26aufirst%3DT.%26aulast%3DBender%26aufirst%3DA.%26aulast%3DTresadern%26aufirst%3DG.%26aulast%3DMedina-Franco%26aufirst%3DJ.%2BL.%26aulast%3DMartinez-Mayorga%26aufirst%3DK.%26aulast%3DLanger%26aufirst%3DT.%26aulast%3DCuanalo-Contreras%26aufirst%3DK.%26aulast%3DAgrafiotis%26aufirst%3DD.%2BK.%26atitle%3DRecognizing%2520Pitfalls%2520in%2520Virtual%2520Screening%253A%2520A%2520Critical%2520Review%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D867%26epage%3D881%26doi%3D10.1021%2Fci200528d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Heikamp, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">The Future of Virtual Compound Screening</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">33</span><span class="NLM_x">–</span> <span class="NLM_lpage">40</span><span class="refDoi"> DOI: 10.1111/cbdd.12054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1111%2Fcbdd.12054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=23253129" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlt1ymuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2013&pages=33-40&author=K.+Heikampauthor=J.+Bajorath&title=The+Future+of+Virtual+Compound+Screening&doi=10.1111%2Fcbdd.12054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The future of virtual compound screening</span></div><div class="casAuthors">Heikamp, Kathrin; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-40</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  We provide a future perspective of the virtual screening field.  A no. of challenges will be highlighted that virtual screening will likely face when compd. data will further grow at or beyond current rates and when much more target information will become available.  These challenges go beyond computational efficiency issues (that will of course also play a crit. role).  For example, for structure-based approaches, the accuracy of scoring functions and energy calcns. will need to be improved.  For ligand-based approaches, the compd. class-dependence of similarity methods needs to be further explored and relationships between mol. similarity and activity similarity need to be established.  We also comment on the current and future value of virtual screening.  Opportunities for further development in a postgenome era are also discussed.  It is hoped that some of the views and hypotheses we articulate might stimulate further discussion about the virtual screening field going forward.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOYfHCsPxEabVg90H21EOLACvtfcHk0lh0LbhVGgpMqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlt1ymuw%253D%253D&md5=de1141b27ae39110f78b83fca24a2238</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12054%26sid%3Dliteratum%253Aachs%26aulast%3DHeikamp%26aufirst%3DK.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DThe%2520Future%2520of%2520Virtual%2520Compound%2520Screening%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2013%26volume%3D81%26spage%3D33%26epage%3D40%26doi%3D10.1111%2Fcbdd.12054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Kolb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferreira, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Docking and Chemoinformatic Screens for New Ligands and Targets</span> <span class="citation_source-journal">Curr. Opin. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">429</span><span class="NLM_x">–</span> <span class="NLM_lpage">436</span><span class="refDoi"> DOI: 10.1016/j.copbio.2009.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2Fj.copbio.2009.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=19733475" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1OrtLbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2009&pages=429-436&author=P.+Kolbauthor=R.+S.+Ferreiraauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Docking+and+Chemoinformatic+Screens+for+New+Ligands+and+Targets&doi=10.1016%2Fj.copbio.2009.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Docking and chemoinformatic screens for new ligands and targets</span></div><div class="casAuthors">Kolb, Peter; Ferreira, Rafaela S.; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Biotechnology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">429-436</span>CODEN:
                <span class="NLM_cas:coden">CUOBE3</span>;
        ISSN:<span class="NLM_cas:issn">0958-1669</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Computer-based docking screens are now widely used to discover new ligands for targets of known structure; in the last two years alone, the discovery of ligands for more than 20 proteins has been reported.  Recently, investigators have also turned to predicting new substrates for enzymes of unknown function, taking docking in a wholly new direction.  Increasingly, the hit rates, the true-positives, and the false-positives from the docking screens are being compared to those from empirical, high-throughput screens, revealing the strengths, weaknesses, and complementarities of both techniques.  The recent efflorescence of GPCR structures has made these quintessential drug targets available to structure-based approaches.  Consistent with their druggability', the docking screens have returned high hit rates and potent mols.  Finally, in the last several years, an approach almost exactly opposite to docking has also appeared; this pharmacol. network approach begins not with the structure of the target but rather those of drug mols. and asks, given a pattern of chem. in the ligands, what targets may a particular drug bind to.  This method, which returns to an older, pharmacol. logic, has been surprisingly successful in predicting new off-targets' for established drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryVNL3mMiQjbVg90H21EOLACvtfcHk0lh0LbhVGgpMqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1OrtLbL&md5=4b6b2457a735e18be5780d9299ebffe6</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.copbio.2009.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.copbio.2009.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DFerreira%26aufirst%3DR.%2BS.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDocking%2520and%2520Chemoinformatic%2520Screens%2520for%2520New%2520Ligands%2520and%2520Targets%26jtitle%3DCurr.%2520Opin.%2520Biotechnol.%26date%3D2009%26volume%3D20%26spage%3D429%26epage%3D436%26doi%3D10.1016%2Fj.copbio.2009.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Forino, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Easton, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houghton, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pellecchia, M.</span><span> </span><span class="NLM_article-title">Virtual Docking Approaches to Protein Kinase B Inhibition</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">2278</span><span class="NLM_x">–</span> <span class="NLM_lpage">2281</span><span class="refDoi"> DOI: 10.1021/jm048962u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm048962u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=2278-2281&author=M.+Forinoauthor=D.+Jungauthor=J.+B.+Eastonauthor=P.+J.+Houghtonauthor=M.+Pellecchia&title=Virtual+Docking+Approaches+to+Protein+Kinase+B+Inhibition&doi=10.1021%2Fjm048962u"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fjm048962u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm048962u%26sid%3Dliteratum%253Aachs%26aulast%3DForino%26aufirst%3DM.%26aulast%3DJung%26aufirst%3DD.%26aulast%3DEaston%26aufirst%3DJ.%2BB.%26aulast%3DHoughton%26aufirst%3DP.%2BJ.%26aulast%3DPellecchia%26aufirst%3DM.%26atitle%3DVirtual%2520Docking%2520Approaches%2520to%2520Protein%2520Kinase%2520B%2520Inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D2278%26epage%3D2281%26doi%3D10.1021%2Fjm048962u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Geppert, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Current Trends in Ligand-Based Virtual Screening: Molecular Representations, Data Mining Methods, New Application Areas, and Performance Evaluation</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">205</span><span class="NLM_x">–</span> <span class="NLM_lpage">216</span><span class="refDoi"> DOI: 10.1021/ci900419k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900419k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvFWmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=205-216&author=H.+Geppertauthor=M.+Vogtauthor=J.+Bajorath&title=Current+Trends+in+Ligand-Based+Virtual+Screening%3A+Molecular+Representations%2C+Data+Mining+Methods%2C+New+Application+Areas%2C+and+Performance+Evaluation&doi=10.1021%2Fci900419k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Current Trends in Ligand-Based Virtual Screening: Molecular Representations, Data Mining Methods, New Application Areas, and Performance Evaluation</span></div><div class="casAuthors">Geppert, Hanna; Vogt, Martin; Bajorath, Jurgen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">205-216</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc1HgbkXa9o7Vg90H21EOLACvtfcHk0lj_I4pKz8xB0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvFWmsA%253D%253D&md5=aeb674ef7c6c93711eb21452732e21d8</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fci900419k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900419k%26sid%3Dliteratum%253Aachs%26aulast%3DGeppert%26aufirst%3DH.%26aulast%3DVogt%26aufirst%3DM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DCurrent%2520Trends%2520in%2520Ligand-Based%2520Virtual%2520Screening%253A%2520Molecular%2520Representations%252C%2520Data%2520Mining%2520Methods%252C%2520New%2520Application%2520Areas%252C%2520and%2520Performance%2520Evaluation%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D205%26epage%3D216%26doi%3D10.1021%2Fci900419k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Willett, P.</span><span> </span><span class="NLM_article-title">Similarity-Based Virtual Screening Using 2d Fingerprints</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1046</span><span class="NLM_x">–</span> <span class="NLM_lpage">1053</span><span class="refDoi"> DOI: 10.1016/j.drudis.2006.10.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2Fj.drudis.2006.10.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=17129822" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1eqtb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=1046-1053&author=P.+Willett&title=Similarity-Based+Virtual+Screening+Using+2d+Fingerprints&doi=10.1016%2Fj.drudis.2006.10.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Similarity-based virtual screening using 2D fingerprints</span></div><div class="casAuthors">Willett, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">23 & 24</span>),
    <span class="NLM_cas:pages">1046-1053</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  This paper summarizes recent work at the University of Sheffield on virtual screening methods that use 2D fingerprint measures of structural similarity.  A detailed comparison of a large no. of similarity coeffs. demonstrates that the well-known Tanimoto coeff. remains the method of choice for the computation of fingerprint-based similarity, despite possessing some inherent biases related to the sizes of the mols. that are being sought.  Group fusion involves combining the results of similarity searches based on multiple ref. structures and a single similarity measure.  We demonstrate the effectiveness of this approach to screening, and also describe an approx. form of group fusion, turbo similarity searching, that can be used when just a single ref. structure is available.  This paper discusses the use of binary-encoded fragment substructures to scan databases to find mols. that are structurally similar to a bioactive query compd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3TRt6CMU5VrVg90H21EOLACvtfcHk0lj_I4pKz8xB0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1eqtb%252FN&md5=095fe152b1ae1abbea69f02123651874</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.10.005%26sid%3Dliteratum%253Aachs%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DSimilarity-Based%2520Virtual%2520Screening%2520Using%25202d%2520Fingerprints%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2006%26volume%3D11%26spage%3D1046%26epage%3D1053%26doi%3D10.1016%2Fj.drudis.2006.10.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Giganti, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guillemain, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spadoni, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilges, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zagury, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montes, M.</span><span> </span><span class="NLM_article-title">Comparative Evaluation of 3d Virtual Ligand Screening Methods: Impact of the Molecular Alignment on Enrichment</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">992</span><span class="NLM_x">–</span> <span class="NLM_lpage">1004</span><span class="refDoi"> DOI: 10.1021/ci900507g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900507g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntVOns7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=992-1004&author=D.+Gigantiauthor=H.+Guillemainauthor=J.+L.+Spadoniauthor=M.+Nilgesauthor=J.+F.+Zaguryauthor=M.+Montes&title=Comparative+Evaluation+of+3d+Virtual+Ligand+Screening+Methods%3A+Impact+of+the+Molecular+Alignment+on+Enrichment&doi=10.1021%2Fci900507g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative Evaluation of 3D Virtual Ligand Screening Methods: Impact of the Molecular Alignment on Enrichment</span></div><div class="casAuthors">Giganti, David; Guillemain, Helene; Spadoni, Jean-Louis; Nilges, Michael; Zagury, Jean-Francois; Montes, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">992-1004</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the early stage of drug discovery programs, when the structure of a complex involving a target and a small mol. is available, structure-based virtual ligand screening methods are generally preferred.  However, ligand-based strategies like shape-similarity search methods can also be applied.  Shape-similarity search methods consist in exploring a pseudo-binding-site derived from the known small mol. used as a ref.  Several of these methods use conformational sampling algorithms which are also shared by corresponding docking methods: for example Surflex-dock/Surflex-sim, FlexX/FlexS, ICM, and OMEGA-FRED/OMEGA-ROCS.  Using 11 systems issued from the challenging "own" subsets of the Directory of Useful Decoys (DUD-own), we evaluated and compared the performance of the above-cited programs in terms of mol. alignment accuracy, enrichment in active compds., and enrichment in different chemotypes (scaffold-hopping).  Since mol. alignment is a crucial aspect of performance for the different methods, we have assessed its impact on enrichment.  We have also illustrated the paradox of retrieving active compds. with good scores even if they are inaccurately positioned.  Finally, we have highlighted possible pos. aspects of using shape-based approaches in drug-discovery protocols when the structure of the target in complex with a small mol. is known.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbvXSUiRzBC7Vg90H21EOLACvtfcHk0lj_I4pKz8xB0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntVOns7o%253D&md5=5d455690f7d4260839ef6c228b9b5364</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fci900507g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900507g%26sid%3Dliteratum%253Aachs%26aulast%3DGiganti%26aufirst%3DD.%26aulast%3DGuillemain%26aufirst%3DH.%26aulast%3DSpadoni%26aufirst%3DJ.%2BL.%26aulast%3DNilges%26aufirst%3DM.%26aulast%3DZagury%26aufirst%3DJ.%2BF.%26aulast%3DMontes%26aufirst%3DM.%26atitle%3DComparative%2520Evaluation%2520of%25203d%2520Virtual%2520Ligand%2520Screening%2520Methods%253A%2520Impact%2520of%2520the%2520Molecular%2520Alignment%2520on%2520Enrichment%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D992%26epage%3D1004%26doi%3D10.1021%2Fci900507g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Lemmen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lengauer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Flexs: A Method for Fast Flexible Ligand Superposition</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">4502</span><span class="NLM_x">–</span> <span class="NLM_lpage">4520</span><span class="refDoi"> DOI: 10.1021/jm981037l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm981037l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=4502-4520&author=C.+Lemmenauthor=T.+Lengauerauthor=G.+Klebe&title=Flexs%3A+A+Method+for+Fast+Flexible+Ligand+Superposition&doi=10.1021%2Fjm981037l"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm981037l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm981037l%26sid%3Dliteratum%253Aachs%26aulast%3DLemmen%26aufirst%3DC.%26aulast%3DLengauer%26aufirst%3DT.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DFlexs%253A%2520A%2520Method%2520for%2520Fast%2520Flexible%2520Ligand%2520Superposition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D4502%26epage%3D4520%26doi%3D10.1021%2Fjm981037l" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Andrews, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cramer, R. D.</span><span> </span><span class="NLM_article-title">Toward General Methods of Targeted Library Design: Topomer Shape Similarity Searching with Diverse Structures as Queries</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">1723</span><span class="NLM_x">–</span> <span class="NLM_lpage">1740</span><span class="refDoi"> DOI: 10.1021/jm000003m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm000003m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD3cXitF2gsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=1723-1740&author=K.+M.+Andrewsauthor=R.+D.+Cramer&title=Toward+General+Methods+of+Targeted+Library+Design%3A+Topomer+Shape+Similarity+Searching+with+Diverse+Structures+as+Queries&doi=10.1021%2Fjm000003m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Toward general methods of targeted library design. Topomer shape similarity searching with diverse structures as queries</span></div><div class="casAuthors">Andrews, Katherine M.; Cramer, Richard D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1723-1740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A promising strategy for selecting synthetic targets is similarity-based searching of very large "virtual libraries", which comprise all structures accessible by linking 2 or 3 com. available building blocks with combinatorial syntheses.  To assess the general applicability of this strategy, leading structures taken from each of 34 recent medicinal chem. publications were used as queries to search a virtual library contg. 2.6 × 1013 products from 7 reactions, using a topomer shape similarity metric.  85% Of these searches succeeded, by yielding, with a search radius no greater than 120 topomer shape units, either at least 400 hits or hits from at least 6 sublibraries.  From these 34 sets of search results, 122 representative structures were selected, illustrating potential "lead hops", or otherwise novel structures.  Overall shape similarity to the query structure was confirmed for up to 95% of these representative structures, according to FLEXS, an algorithmically distinct program.  Exptl., there were 28 structures among those reported in the 34 query publications that were identified within the virtual library.  Among these, the frequency of high activity was 87% for the 16 structures whose similarity to their query was 90 topomer units or less, compared to a frequency of 50% for the other 12 structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfSsMO0zff6bVg90H21EOLACvtfcHk0lh9qwSFVbmn-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXitF2gsbw%253D&md5=ea40f3ae6042b09413ad43c0f0061431</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm000003m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm000003m%26sid%3Dliteratum%253Aachs%26aulast%3DAndrews%26aufirst%3DK.%2BM.%26aulast%3DCramer%26aufirst%3DR.%2BD.%26atitle%3DToward%2520General%2520Methods%2520of%2520Targeted%2520Library%2520Design%253A%2520Topomer%2520Shape%2520Similarity%2520Searching%2520with%2520Diverse%2520Structures%2520as%2520Queries%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D1723%26epage%3D1740%26doi%3D10.1021%2Fjm000003m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Jain, A. N.</span><span> </span><span class="NLM_article-title">Ligand-Based Structural Hypotheses for Virtual Screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">947</span><span class="NLM_x">–</span> <span class="NLM_lpage">961</span><span class="refDoi"> DOI: 10.1021/jm030520f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030520f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD2cXkt1GnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=947-961&author=A.+N.+Jain&title=Ligand-Based+Structural+Hypotheses+for+Virtual+Screening&doi=10.1021%2Fjm030520f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand-Based Structural Hypotheses for Virtual Screening</span></div><div class="casAuthors">Jain, Ajay N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">947-961</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The majority of drug targets for small mol. therapeutics are proteins whose three-dimensional structure is not known to sufficient resoln. to permit structure-based design.  All three-dimensional QSAR approaches have a requirement for some hypothesis of ligand conformation and alignment, and predictions of mol. activity critically depend on this ligand-based binding site hypothesis.  The mol. similarity function used in the Surflex docking system, coupled with quant. pressure to minimize overall mol. vol., forms an effective objective function for generating hypotheses of bioactive conformations of sets of small mols. binding to their cognate proteins.  Results are presented, assessing utility of the method for ligands of the serotonin, histamine, muscarinic, and GABAA receptors.  The Surflex similarity module (Surflex-Sim) was able, in each case, to distinguish true ligands from random compds. using models constructed from just two or three known ligands.  True pos. rates of 60% were achieved with false pos. rates of 0-3%; the theor. enrichment rates were over 150-fold compared with random screening.  The methods are practically applicable for rational design of ligands and for high-throughput virtual screening and offer competitive performance to many structure-based docking algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSKeodYur23bVg90H21EOLACvtfcHk0lh9qwSFVbmn-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXkt1GnsQ%253D%253D&md5=7442ef3b414134cc32add2d159ce199b</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm030520f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030520f%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DLigand-Based%2520Structural%2520Hypotheses%2520for%2520Virtual%2520Screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D947%26epage%3D961%26doi%3D10.1021%2Fjm030520f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Korb, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monecke, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hessler, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stutzle, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Exner, T. E.</span><span> </span><span class="NLM_article-title">Pharmacophore: Multiple Flexible Ligand Alignment Based on Ant Colony Optimization</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1669</span><span class="NLM_x">–</span> <span class="NLM_lpage">1681</span><span class="refDoi"> DOI: 10.1021/ci1000218</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci1000218" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtV2jtr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1669-1681&author=O.+Korbauthor=P.+Moneckeauthor=G.+Hesslerauthor=T.+Stutzleauthor=T.+E.+Exner&title=Pharmacophore%3A+Multiple+Flexible+Ligand+Alignment+Based+on+Ant+Colony+Optimization&doi=10.1021%2Fci1000218"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">pharmACOphore: Multiple Flexible Ligand Alignment Based on Ant Colony Optimization</span></div><div class="casAuthors">Korb, Oliver; Monecke, Peter; Hessler, Gerhard; Stuetzle, Thomas; Exner, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1669-1681</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The flexible superimposition of biol. active ligands is a crucial step in ligand-based drug design.  Here we present pharmACOphore, a new approach for pairwise as well as multiple flexible alignment of ligands based on ant colony optimization.  An empirical scoring function is used, which describes ligand similarity by minimizing the distance of pharmacophoric features.  The scoring function was parametrized on pairwise alignments of ligand sets for four proteins from diverse protein families (cyclooxygenase-2, cyclin-dependent kinase 2, factor Xa and peroxisome proliferator-activated receptor γ).  The derived parameters were assessed with respect to pose prediction performance on the independent FlexS data set in exhausting pairwise alignments.  Addnl., multiple flexible alignment expts. were carried out for the pharmacol. relevant targets trypsin and poly (ADP-ribose) polymerase (PARP).  The results obtained show that the new procedure provides a robust and efficient way for the pairwise as well as multiple flexible alignment of small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV9Q-o6EevhrVg90H21EOLACvtfcHk0lh9qwSFVbmn-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtV2jtr7O&md5=01deb5cd5dea2f3dd9262b5221d8381c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fci1000218&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci1000218%26sid%3Dliteratum%253Aachs%26aulast%3DKorb%26aufirst%3DO.%26aulast%3DMonecke%26aufirst%3DP.%26aulast%3DHessler%26aufirst%3DG.%26aulast%3DStutzle%26aufirst%3DT.%26aulast%3DExner%26aufirst%3DT.%2BE.%26atitle%3DPharmacophore%253A%2520Multiple%2520Flexible%2520Ligand%2520Alignment%2520Based%2520on%2520Ant%2520Colony%2520Optimization%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1669%26epage%3D1681%26doi%3D10.1021%2Fci1000218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Nicholls, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGaughey, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Good, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warren, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathieu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muchmore, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haigh, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nevins, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelley, B.</span><span> </span><span class="NLM_article-title">Molecular Shape and Medicinal Chemistry: A Perspective</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">3862</span><span class="NLM_x">–</span> <span class="NLM_lpage">3886</span><span class="refDoi"> DOI: 10.1021/jm900818s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm900818s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvF2kt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=3862-3886&author=A.+Nichollsauthor=G.+B.+McGaugheyauthor=R.+P.+Sheridanauthor=A.+C.+Goodauthor=G.+Warrenauthor=M.+Mathieuauthor=S.+W.+Muchmoreauthor=S.+P.+Brownauthor=J.+A.+Grantauthor=J.+A.+Haighauthor=N.+Nevinsauthor=A.+N.+Jainauthor=B.+Kelley&title=Molecular+Shape+and+Medicinal+Chemistry%3A+A+Perspective&doi=10.1021%2Fjm900818s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Shape and Medicinal Chemistry: A Perspective</span></div><div class="casAuthors">Nicholls, Anthony; McGaughey, Georgia B.; Sheridan, Robert P.; Good, Andrew C.; Warren, Gregory; Mathieu, Magali; Muchmore, Steven W.; Brown, Scott P.; Grant, J. Andrew; Haigh, James A.; Nevins, Neysa; Jain, Ajay N.; Kelley, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3862-3886</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review article with 111 refs. summarized perspectives of mol. shape and medicinal chem. in drug screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqtbXOEoSmy7Vg90H21EOLACvtfcHk0liDTJvog2rWyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvF2kt7k%253D&md5=85664344e13872527a3dfb2296d34864</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm900818s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm900818s%26sid%3Dliteratum%253Aachs%26aulast%3DNicholls%26aufirst%3DA.%26aulast%3DMcGaughey%26aufirst%3DG.%2BB.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DGood%26aufirst%3DA.%2BC.%26aulast%3DWarren%26aufirst%3DG.%26aulast%3DMathieu%26aufirst%3DM.%26aulast%3DMuchmore%26aufirst%3DS.%2BW.%26aulast%3DBrown%26aufirst%3DS.%2BP.%26aulast%3DGrant%26aufirst%3DJ.%2BA.%26aulast%3DHaigh%26aufirst%3DJ.%2BA.%26aulast%3DNevins%26aufirst%3DN.%26aulast%3DJain%26aufirst%3DA.%2BN.%26aulast%3DKelley%26aufirst%3DB.%26atitle%3DMolecular%2520Shape%2520and%2520Medicinal%2520Chemistry%253A%2520A%2520Perspective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D3862%26epage%3D3886%26doi%3D10.1021%2Fjm900818s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Underwood, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kearsley, S. K.</span><span> </span><span class="NLM_article-title">Chemical Similarity Using Geometric Atom Pair Descriptors</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">128</span><span class="NLM_x">–</span> <span class="NLM_lpage">136</span><span class="refDoi"> DOI: 10.1021/ci950275b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci950275b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADyaK28XhsFKitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1996&pages=128-136&author=R.+P.+Sheridanauthor=M.+D.+Millerauthor=D.+J.+Underwoodauthor=S.+K.+Kearsley&title=Chemical+Similarity+Using+Geometric+Atom+Pair+Descriptors&doi=10.1021%2Fci950275b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Similarity Using Geometric Atom Pair Descriptors</span></div><div class="casAuthors">Sheridan, Robert P.; Miller, Michael D.; Underwood, Dennis J.; Kearsley, Simon K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">128-36</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this paper we describe the geometric atom pair, the 3D analog of the topol. atom pair descriptor (Carhart et al. J. Chem. Inf. Comput. Sci. 1985, 25, 64-73).  We show the results of geometric similarity searches using the CONCORD-build structures of typical small druglike mols. as probes.  The database to be searched is a 3D version of the Derwent Std. Drug File that contains an av. of 10 explicit conformations per compd.  Using objective criteria for detg. how good a descriptor is in selecting active compds. from large databases, we compare the results using the geometric vs. the topol. atom pair.  We find that geometric and topol. atom pairs are about equally effective in selecting active compds. from large databases.  How the two types of descriptors rank active compds. is generally about the same as well, but occasionally active compds. will be seen as very similar to a probe in geometric descriptors, but as fairly dissimilar in topol. descriptors.  These are of two types: (1) compds. where equiv. groups are in the same spatial arrangement as in the probe but are connected by very different bond paths and (2) compds. that can superimpose onto the probe when they are in a folded conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBZVblsDYTNLVg90H21EOLACvtfcHk0liDTJvog2rWyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhsFKitA%253D%253D&md5=99ed1470e6973f090bcbb4025c3a0445</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fci950275b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci950275b%26sid%3Dliteratum%253Aachs%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DMiller%26aufirst%3DM.%2BD.%26aulast%3DUnderwood%26aufirst%3DD.%2BJ.%26aulast%3DKearsley%26aufirst%3DS.%2BK.%26atitle%3DChemical%2520Similarity%2520Using%2520Geometric%2520Atom%2520Pair%2520Descriptors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1996%26volume%3D36%26spage%3D128%26epage%3D136%26doi%3D10.1021%2Fci950275b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Schreyer, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T.</span><span> </span><span class="NLM_article-title">Usrcat: Real-Time Ultrafast Shape Recognition with Pharmacophoric Constraints</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">27</span><span class="NLM_x">–</span> <span class="NLM_lpage">39</span><span class="refDoi"> DOI: 10.1186/1758-2946-4-27</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1186%2F1758-2946-4-27" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVGjt7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=27-39&author=A.+M.+Schreyerauthor=T.+Blundell&title=Usrcat%3A+Real-Time+Ultrafast+Shape+Recognition+with+Pharmacophoric+Constraints&doi=10.1186%2F1758-2946-4-27"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">USRCAT: real-time Ultrafast Shape Recognition with pharmacophoric constraints</span></div><div class="casAuthors">Schreyer, Adrian M.; Blundell, Tom</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">27</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Background: Ligand-based virtual screening using mol. shape is an important tool for researchers who wish to find novel chem. scaffolds in compd. libraries.  The Ultrafast Shape Recognition (USR) algorithm is capable of screening millions of compds. and is therefore suitable for usage in a web service.  The algorithm however is agnostic of atom types and cannot discriminate compds. with similar shape but distinct pharmacophoric features.  To solve this problem, an extension of USR called USRCAT, has been developed that includes pharmacophoric information while retaining the performance benefits of the original method.  Results: The USRCAT extension is shown to outperform the traditional USR method in a retrospective virtual screening benchmark.  Also, a relational database implementation is described that is capable of screening a million conformers in milliseconds and allows the inclusion of complex query parameters.  Conclusions: USRCAT provides a soln. to the lack of atom type information in the USR algorithm.  Researchers, particularly those with only limited resources, who wish to use ligand-based virtual screening in order to discover new hits, will benefit the most.  Online chem. databases that offer a shape-based similarity method might also find advantage in using USRCAT due to its accuracy and performance.  The source code is freely available and can easily be modified to fit specific needs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxouS95m5dGrVg90H21EOLACvtfcHk0liDTJvog2rWyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVGjt7fJ&md5=9e7e82f660a7ec4e542d539bed081a25</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-4-27&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-4-27%26sid%3Dliteratum%253Aachs%26aulast%3DSchreyer%26aufirst%3DA.%2BM.%26aulast%3DBlundell%26aufirst%3DT.%26atitle%3DUsrcat%253A%2520Real-Time%2520Ultrafast%2520Shape%2520Recognition%2520with%2520Pharmacophoric%2520Constraints%26jtitle%3DJ.%2520Cheminf.%26date%3D2012%26volume%3D4%26spage%3D27%26epage%3D39%26doi%3D10.1186%2F1758-2946-4-27" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Awale, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reymond, J. L.</span><span> </span><span class="NLM_article-title">Stereoselective Virtual Screening of the Zinc Database Using Atom Pair 3d-Fingerprints</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="refDoi"> DOI: 10.1186/s13321-014-0051-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1186%2Fs13321-014-0051-5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=3&author=M.+Awaleauthor=X.+Jinauthor=J.+L.+Reymond&title=Stereoselective+Virtual+Screening+of+the+Zinc+Database+Using+Atom+Pair+3d-Fingerprints&doi=10.1186%2Fs13321-014-0051-5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1186%2Fs13321-014-0051-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-014-0051-5%26sid%3Dliteratum%253Aachs%26aulast%3DAwale%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DX.%26aulast%3DReymond%26aufirst%3DJ.%2BL.%26atitle%3DStereoselective%2520Virtual%2520Screening%2520of%2520the%2520Zinc%2520Database%2520Using%2520Atom%2520Pair%25203d-Fingerprints%26jtitle%3DJ.%2520Cheminf.%26date%3D2015%26volume%3D7%26spage%3D3%26doi%3D10.1186%2Fs13321-014-0051-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Schneider, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidhart, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giller, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmid, G.</span><span> </span><span class="NLM_article-title">″Scaffold-Hopping″ by Topological Pharmacophore Search: A Contribution to Virtual Screening</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">2894</span><span class="NLM_x">–</span> <span class="NLM_lpage">2896</span><span class="refDoi"> DOI: 10.1002/(SICI)1521-3773(19991004)38:19<2894::AID-ANIE2894>3.3.CO;2-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1002%2F%28SICI%291521-3773%2819991004%2938%3A19%3C2894%3A%3AAID-ANIE2894%3E3.3.CO%3B2-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=2894-2896&author=G.+Schneiderauthor=W.+Neidhartauthor=T.+Gillerauthor=G.+Schmid&title=%E2%80%B3Scaffold-Hopping%E2%80%B3+by+Topological+Pharmacophore+Search%3A+A+Contribution+to+Virtual+Screening&doi=10.1002%2F%28SICI%291521-3773%2819991004%2938%3A19%3C2894%3A%3AAID-ANIE2894%3E3.3.CO%3B2-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291521-3773%2819991004%2938%3A19%3C2894%3A%3AAID-ANIE2894%3E3.3.CO%3B2-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291521-3773%252819991004%252938%253A19%253C2894%253A%253AAID-ANIE2894%253E3.3.CO%253B2-6%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DNeidhart%26aufirst%3DW.%26aulast%3DGiller%26aufirst%3DT.%26aulast%3DSchmid%26aufirst%3DG.%26atitle%3D%25E2%2580%25B3Scaffold-Hopping%25E2%2580%25B3%2520by%2520Topological%2520Pharmacophore%2520Search%253A%2520A%2520Contribution%2520to%2520Virtual%2520Screening%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1999%26volume%3D38%26spage%3D2894%26epage%3D2896%26doi%3D10.1002%2F%28SICI%291521-3773%2819991004%2938%3A19%3C2894%3A%3AAID-ANIE2894%3E3.3.CO%3B2-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Marumoto, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saya, H.</span><span> </span><span class="NLM_article-title">Aurora-a - a Guardian of Poles</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">42</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1038/nrc1526</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1038%2Fnrc1526" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=15630414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlvVen" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2005&pages=42-50&author=T.+Marumotoauthor=D.+Zhangauthor=H.+Saya&title=Aurora-a+-+a+Guardian+of+Poles&doi=10.1038%2Fnrc1526"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Aurora-A - A guardian of poles</span></div><div class="casAuthors">Marumoto, Tomotoshi; Zhang, Dongwei; Saya, Hideyuki</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">42-50</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The three human homologs of Aurora kinases (A, B and C) are essential for proper execution of various mitotic events and are important for maintaining genomic integrity.  Aurora-A is mainly localized at spindle poles and the mitotic spindle during mitosis, where it regulates the functions of centrosomes, spindles and kinetochores required for proper mitotic progression.  Recent studies have revealed that Aurora-A is frequently overexpressed in various cancer cells, indicating its involvement in tumorigenesis.  What are the normal physiol. roles of Aurora-A, how are these regulated and how might the enzyme function during tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZ5DkKXAtszrVg90H21EOLACvtfcHk0lhvE3Vtt1SyyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlvVen&md5=5eef2a40508d408be784301b20e6822d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnrc1526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1526%26sid%3Dliteratum%253Aachs%26aulast%3DMarumoto%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DSaya%26aufirst%3DH.%26atitle%3DAurora-a%2520-%2520a%2520Guardian%2520of%2520Poles%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2005%26volume%3D5%26spage%3D42%26epage%3D50%26doi%3D10.1038%2Fnrc1526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Metz, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soni, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merta, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kifle, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span> </span><span class="NLM_article-title">Navigating the Kinome</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">200</span><span class="NLM_x">–</span> <span class="NLM_lpage">202</span><span class="refDoi"> DOI: 10.1038/nchembio.530</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1038%2Fnchembio.530" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=21336281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1aitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=200-202&author=J.+T.+Metzauthor=E.+F.+Johnsonauthor=N.+B.+Soniauthor=P.+J.+Mertaauthor=L.+Kifleauthor=P.+J.+Hajduk&title=Navigating+the+Kinome&doi=10.1038%2Fnchembio.530"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Navigating the kinome</span></div><div class="casAuthors">Metz, James T.; Johnson, Eric F.; Soni, Niru B.; Merta, Philip J.; Kifle, Lemma; Hajduk, Philip J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">200-202</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although it is increasingly being recognized that drug-target interaction networks can be powerful tools for the interrogation of systems biol. and the rational design of multitargeted drugs, there is no generalized, statistically validated approach to harmonizing sequence-dependent and pharmacol.-dependent networks.  Here we demonstrate the creation of a comprehensive kinome interaction network based not only on sequence comparisons but also on multiple pharmacol. parameters derived from activity profiling data.  The framework described for statistical interpretation of these network connections also enables rigorous investigation of chemotype-specific interaction networks, which is crit. for multitargeted drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOgzF9Q0eH87Vg90H21EOLACvtfcHk0lhvE3Vtt1SyyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1aitrc%253D&md5=09234ec451a4918e5aca6d65e0b65aee</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.530%26sid%3Dliteratum%253Aachs%26aulast%3DMetz%26aufirst%3DJ.%2BT.%26aulast%3DJohnson%26aufirst%3DE.%2BF.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DMerta%26aufirst%3DP.%2BJ.%26aulast%3DKifle%26aufirst%3DL.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DNavigating%2520the%2520Kinome%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D200%26epage%3D202%26doi%3D10.1038%2Fnchembio.530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Carmena, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Earnshaw, W. C.</span><span> </span><span class="NLM_article-title">The Cellular Geography of Aurora Kinases</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">842</span><span class="NLM_x">–</span> <span class="NLM_lpage">854</span><span class="refDoi"> DOI: 10.1038/nrm1245</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1038%2Fnrm1245" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=14625535" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpvFais7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2003&pages=842-854&author=M.+Carmenaauthor=W.+C.+Earnshaw&title=The+Cellular+Geography+of+Aurora+Kinases&doi=10.1038%2Fnrm1245"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The cellular geography of Aurora kinases</span></div><div class="casAuthors">Carmena, Mar; Earnshaw, William C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">842-854</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Aurora is the name given to a family of highly conserved protein kinases with essential roles in many aspects of cell division.  Yeasts have a single Aurora kinase, whereas mammals have three: Aurora A, B and C.  During mitosis, Aurora kinases regulate the structure and function of the cytoskeleton and chromosomes and the interactions between these two at the kinetochore.  They also regulate signalling by the spindle-assembly checkpoint pathway and cytokinesis.  Perturbation of Aurora kinase expression or function might lead to cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrU7PA68xW-NLVg90H21EOLACvtfcHk0lhvE3Vtt1SyyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpvFais7g%253D&md5=4887100c6893eb6a40537cdc9916c1a4</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnrm1245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1245%26sid%3Dliteratum%253Aachs%26aulast%3DCarmena%26aufirst%3DM.%26aulast%3DEarnshaw%26aufirst%3DW.%2BC.%26atitle%3DThe%2520Cellular%2520Geography%2520of%2520Aurora%2520Kinases%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2003%26volume%3D4%26spage%3D842%26epage%3D854%26doi%3D10.1038%2Fnrm1245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Shagisultanova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunbrack, R. L.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golemis, E. A.</span><span> </span><span class="NLM_article-title">Issues in Interpreting the in Vivo Activity of Aurora-A</span> <span class="citation_source-journal">Expert Opin. Ther. Targets</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">187</span><span class="NLM_x">–</span> <span class="NLM_lpage">200</span><span class="refDoi"> DOI: 10.1517/14728222.2014.981154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1517%2F14728222.2014.981154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=25384454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlCrt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2015&pages=187-200&author=E.+Shagisultanovaauthor=R.+L.+Dunbrackauthor=E.+A.+Golemis&title=Issues+in+Interpreting+the+in+Vivo+Activity+of+Aurora-A&doi=10.1517%2F14728222.2014.981154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Issues in interpreting the in vivo activity of Aurora-A</span></div><div class="casAuthors">Shagisultanova, Elena; Dunbrack, Roland L., Jr.; Golemis, Erica A.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">187-200</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">Introduction: Based on its role as a mitotic regulatory kinase, overexpressed and assocd. with aneuploidy in cancer, small-mol. inhibitors have been developed for Aurora-A (AURKA) kinase.  In preclin. and clin. assessments, these agents have shown efficacy in inducing stable disease or therapeutic response.  In optimizing the use of Aurora-A inhibitors, it is crit. to have robust capacity to measure the kinase activity of Aurora-A in tumors.  Areas covered: We provide an overview of mol. mechanisms of mitotic and non-mitotic activation of Aurora-A kinase, and interaction of Aurora-A with its regulatory partners.  Typically, Aurora-A activity is measured by use of phospho-antibodies targeting an autophosphorylated T288 epitope.  However, recent studies have identified alternative means of Aurora-A activation control, including allosteric regulation by partners, phosphorylation on alternative activating residues (S51, S98), dephosphorylation on inhibitory sites (S342) and T288 phosphorylation by alternative kinases such as Pak enzymes.  Addnl. work has shown that the relative abundance of Aurora-A partners can affect the activity of Aurora-A inhibitors, and that Aurora-A activation also occurs in interphase cells.  Expert opinion: Taken together, this work suggests the need for comprehensive anal. of Aurora-A activity and expression of Aurora-A partners in order to stratify patients for likely therapeutic response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnUIECo5Cq47Vg90H21EOLACvtfcHk0lj-QCihB3_MdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlCrt7c%253D&md5=2ad2457af5ceae42abb5c70445c9693f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1517%2F14728222.2014.981154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728222.2014.981154%26sid%3Dliteratum%253Aachs%26aulast%3DShagisultanova%26aufirst%3DE.%26aulast%3DDunbrack%26aufirst%3DR.%2BL.%26aulast%3DGolemis%26aufirst%3DE.%2BA.%26atitle%3DIssues%2520in%2520Interpreting%2520the%2520in%2520Vivo%2520Activity%2520of%2520Aurora-A%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2015%26volume%3D19%26spage%3D187%26epage%3D200%26doi%3D10.1517%2F14728222.2014.981154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">de Groot, C. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anzola, J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motamedi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoon, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gahman, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiau, A. K.</span><span> </span><span class="NLM_article-title">A Cell Biologist’s Field Guide to Aurora Kinase Inhibitors</span> <span class="citation_source-journal">Front. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">285</span><span class="refDoi"> DOI: 10.3389/fonc.2015.00285</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.3389%2Ffonc.2015.00285" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=26732741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A280%3ADC%252BC28rnsFSjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=285&author=C.+O.+de+Grootauthor=J.+E.+Hsiaauthor=J.+V.+Anzolaauthor=A.+Motamediauthor=M.+Yoonauthor=Y.+L.+Wongauthor=D.+Jenkinsauthor=H.+J.+Leeauthor=M.+B.+Martinezauthor=R.+L.+Davisauthor=T.+C.+Gahmanauthor=A.+Desaiauthor=A.+K.+Shiau&title=A+Cell+Biologist%E2%80%99s+Field+Guide+to+Aurora+Kinase+Inhibitors&doi=10.3389%2Ffonc.2015.00285"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A Cell Biologist's Field Guide to Aurora Kinase Inhibitors</span></div><div class="casAuthors">de Groot Christian O; Hsia Judy E; Anzola John V; Motamedi Amir; Yoon Michelle; Jenkins David; Lee Hyun J; Martinez Mallory B; Davis Robert L; Gahman Timothy C; Shiau Andrew K; Wong Yao Liang; Desai Arshad</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">285</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">Aurora kinases are essential for cell division and are frequently misregulated in human cancers.  Based on their potential as cancer therapeutics, a plethora of small molecule Aurora kinase inhibitors have been developed, with a subset having been adopted as tools in cell biology.  Here, we fill a gap in the characterization of Aurora kinase inhibitors by using biochemical and cell-based assays to systematically profile a panel of 10 commercially available compounds with reported selectivity for Aurora A (MLN8054, MLN8237, MK-5108, MK-8745, Genentech Aurora Inhibitor 1), Aurora B (Hesperadin, ZM447439, AZD1152-HQPA, GSK1070916), or Aurora A/B (VX-680).  We quantify the in vitro effect of each inhibitor on the activity of Aurora A alone, as well as Aurora A and Aurora B bound to fragments of their activators, TPX2 and INCENP, respectively.  We also report kinome profiling results for a subset of these compounds to highlight potential off-target effects.  In a cellular context, we demonstrate that immunofluorescence-based detection of LATS2 and histone H3 phospho-epitopes provides a facile and reliable means to assess potency and specificity of Aurora A versus Aurora B inhibition, and that G2 duration measured in a live imaging assay is a specific readout of Aurora A activity.  Our analysis also highlights variation between HeLa, U2OS, and hTERT-RPE1 cells that impacts selective Aurora A inhibition.  For Aurora B, all four tested compounds exhibit excellent selectivity and do not significantly inhibit Aurora A at effective doses.  For Aurora A, MK-5108 and MK-8745 are significantly more selective than the commonly used inhibitors MLN8054 and MLN8237.  A crystal structure of an Aurora A/MK-5108 complex that we determined suggests the chemical basis for this higher specificity.  Taken together, our quantitative biochemical and cell-based analyses indicate that AZD1152-HQPA and MK-8745 are the best current tools for selectively inhibiting Aurora B and Aurora A, respectively.  However, MK-8745 is not nearly as ideal as AZD1152-HQPA in that it requires high concentrations to achieve full inhibition in a cellular context, indicating a need for more potent Aurora A-selective inhibitors.  We conclude with a set of "good practice" guidelines for the use of Aurora inhibitors in cell biology experiments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQItUi6CXjsM669lAbTj9rGfW6udTcc2eaxb1NgYsl5Gbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28rnsFSjuw%253D%253D&md5=ad773bddf44e4bf624ee83f400912885</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2015.00285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2015.00285%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BGroot%26aufirst%3DC.%2BO.%26aulast%3DHsia%26aufirst%3DJ.%2BE.%26aulast%3DAnzola%26aufirst%3DJ.%2BV.%26aulast%3DMotamedi%26aufirst%3DA.%26aulast%3DYoon%26aufirst%3DM.%26aulast%3DWong%26aufirst%3DY.%2BL.%26aulast%3DJenkins%26aufirst%3DD.%26aulast%3DLee%26aufirst%3DH.%2BJ.%26aulast%3DMartinez%26aufirst%3DM.%2BB.%26aulast%3DDavis%26aufirst%3DR.%2BL.%26aulast%3DGahman%26aufirst%3DT.%2BC.%26aulast%3DDesai%26aufirst%3DA.%26aulast%3DShiau%26aufirst%3DA.%2BK.%26atitle%3DA%2520Cell%2520Biologist%25E2%2580%2599s%2520Field%2520Guide%2520to%2520Aurora%2520Kinase%2520Inhibitors%26jtitle%3DFront.%2520Oncol.%26date%3D2015%26volume%3D5%26spage%3D285%26doi%3D10.3389%2Ffonc.2015.00285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Manfredi, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ecsedy, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chakravarty, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silverman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoar, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroud, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinde, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huck, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wysong, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janowick, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hyer, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leroy, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gershman, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silva, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Germanos, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolen, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Claiborne, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sells, T. B.</span><span> </span><span class="NLM_article-title">Characterization of Alisertib (Mln8237), an Investigational Small-Molecule Inhibitor of Aurora a Kinase Using Novel in Vivo Pharmacodynamic Assays</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">7614</span><span class="NLM_x">–</span> <span class="NLM_lpage">7624</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-11-1536</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1158%2F1078-0432.CCR-11-1536" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=22016509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1CqtrjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=7614-7624&author=M.+G.+Manfrediauthor=J.+A.+Ecsedyauthor=A.+Chakravartyauthor=L.+Silvermanauthor=M.+Zhangauthor=K.+M.+Hoarauthor=S.+G.+Stroudauthor=W.+Chenauthor=V.+Shindeauthor=J.+J.+Huckauthor=D.+R.+Wysongauthor=D.+A.+Janowickauthor=M.+L.+Hyerauthor=P.+J.+Leroyauthor=R.+E.+Gershmanauthor=M.+D.+Silvaauthor=M.+S.+Germanosauthor=J.+B.+Bolenauthor=C.+F.+Claiborneauthor=T.+B.+Sells&title=Characterization+of+Alisertib+%28Mln8237%29%2C+an+Investigational+Small-Molecule+Inhibitor+of+Aurora+a+Kinase+Using+Novel+in+Vivo+Pharmacodynamic+Assays&doi=10.1158%2F1078-0432.CCR-11-1536"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of Alisertib (MLN8237), an Investigational Small-Molecule Inhibitor of Aurora A Kinase Using Novel In Vivo Pharmacodynamic Assays</span></div><div class="casAuthors">Manfredi, Mark G.; Ecsedy, Jeffrey A.; Chakravarty, Arijit; Silverman, Lee; Zhang, Mengkun; Hoar, Kara M.; Stroud, Stephen G.; Chen, Wei; Shinde, Vaishali; Huck, Jessica J.; Wysong, Deborah R.; Janowick, David A.; Hyer, Marc L.; LeRoy, Patrick J.; Gershman, Rachel E.; Silva, Matthew D.; Germanos, Melissa S.; Bolen, Joseph B.; Claiborne, Christopher F.; Sells, Todd B.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7614-7624</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PURPOSE: Small-mol. inhibitors of Aurora A (AAK) and B (ABK) kinases, which play important roles in mitosis, are currently being pursued in oncol. clin. trials.  We developed three novel assays to quant. measure biomarkers of AAK inhibition in vivo.  Here, we describe preclin. characterization of alisertib (MLN8237), a selective AAK inhibitor, incorporating these novel pharmacodynamic assays.  Exptl. Design: We investigated the selectivity of alisertib for AAK and ABK and studied the antitumor and antiproliferative activity of alisertib in vitro and in vivo.  Novel assays were used to assess chromosome alignment and mitotic spindle bipolarity in human tumor xenografts using immunofluorescent detection of DNA and alpha-tubulin, resp.  In addn., 18F-3'-fluoro-3'-deoxy-L-thymidine positron emission tomog. (FLT-PET) was used to noninvasively measure effects of alisertib on in vivo tumor cell proliferation.  RESULTS: Alisertib inhibited AAK over ABK with a selectivity of more than 200-fold in cells and produced a dose-dependent decrease in bipolar and aligned chromosomes in the HCT-116 xenograft model, a phenotype consistent with AAK inhibition.  Alisertib inhibited proliferation of human tumor cell lines in vitro and produced tumor growth inhibition in solid tumor xenograft models and regressions in in vivo lymphoma models.  In addn., a dose of alisertib that caused tumor stasis, as measured by vol., resulted in a decrease in FLT uptake, suggesting that noninvasive imaging could provide value over traditional measurements of response.  CONCLUSIONS: Alisertib is a selective and potent inhibitor of AAK.  The novel methods of measuring Aurora A pathway inhibition and application of tumor imaging described here may be valuable for clin. evaluation of small-mol. inhibitors.  Clin Cancer Res; 17(24); 7614-24.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYGGGKlJwzprVg90H21EOLACvtfcHk0lj-QCihB3_MdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1CqtrjL&md5=de00b44d9e6cd58a2a642770c3121128</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-11-1536&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-11-1536%26sid%3Dliteratum%253Aachs%26aulast%3DManfredi%26aufirst%3DM.%2BG.%26aulast%3DEcsedy%26aufirst%3DJ.%2BA.%26aulast%3DChakravarty%26aufirst%3DA.%26aulast%3DSilverman%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DHoar%26aufirst%3DK.%2BM.%26aulast%3DStroud%26aufirst%3DS.%2BG.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DShinde%26aufirst%3DV.%26aulast%3DHuck%26aufirst%3DJ.%2BJ.%26aulast%3DWysong%26aufirst%3DD.%2BR.%26aulast%3DJanowick%26aufirst%3DD.%2BA.%26aulast%3DHyer%26aufirst%3DM.%2BL.%26aulast%3DLeroy%26aufirst%3DP.%2BJ.%26aulast%3DGershman%26aufirst%3DR.%2BE.%26aulast%3DSilva%26aufirst%3DM.%2BD.%26aulast%3DGermanos%26aufirst%3DM.%2BS.%26aulast%3DBolen%26aufirst%3DJ.%2BB.%26aulast%3DClaiborne%26aufirst%3DC.%2BF.%26aulast%3DSells%26aufirst%3DT.%2BB.%26atitle%3DCharacterization%2520of%2520Alisertib%2520%2528Mln8237%2529%252C%2520an%2520Investigational%2520Small-Molecule%2520Inhibitor%2520of%2520Aurora%2520a%2520Kinase%2520Using%2520Novel%2520in%2520Vivo%2520Pharmacodynamic%2520Assays%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2011%26volume%3D17%26spage%3D7614%26epage%3D7624%26doi%3D10.1158%2F1078-0432.CCR-11-1536" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Hoar, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chakravarty, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabino, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wysong, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowman, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roy, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ecsedy, J. A.</span><span> </span><span class="NLM_article-title">Mln8054, a Small-Molecule Inhibitor of Aurora a, Causes Spindle Pole and Chromosome Congression Defects Leading to Aneuploidy</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">4513</span><span class="NLM_x">–</span> <span class="NLM_lpage">4525</span><span class="refDoi"> DOI: 10.1128/MCB.02364-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1128%2FMCB.02364-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=17438137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmvFansbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=4513-4525&author=K.+Hoarauthor=A.+Chakravartyauthor=C.+Rabinoauthor=D.+Wysongauthor=D.+Bowmanauthor=N.+Royauthor=J.+A.+Ecsedy&title=Mln8054%2C+a+Small-Molecule+Inhibitor+of+Aurora+a%2C+Causes+Spindle+Pole+and+Chromosome+Congression+Defects+Leading+to+Aneuploidy&doi=10.1128%2FMCB.02364-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">MLN8054, a small-molecule inhibitor of Aurora A, causes spindle pole and chromosome congress-ion defects leading to aneuploidy</span></div><div class="casAuthors">Hoar, Kara; Chakravarty, Arijit; Rabino, Claudia; Wysong, Deborah; Bowman, Douglas; Roy, Natalie; Ecsedy, Jeffrey A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4513-4525</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Aurora A kinase plays an essential role in the proper assembly and function of the mitotic spindle, as its perturbation causes defects in centrosome sepn., spindle pole organization, and chromosome congression.  Moreover, Aurora A disruption leads to cell death via a mechanism that involves aneuploidy generation.  However, the link between the immediate functional consequences of Aurora A inhibition and the development of aneuploidy is not clearly defined.  In this study, we delineate the sequence of events that lead to aneuploidy following Aurora A inhibition using MLN8054, a selective Aurora A small-mol. inhibitor.  Human tumor cells treated with MLN8054 show a high incidence of abnormal mitotic spindles, often with unsepd. centrosomes.  Although these spindle defects result in mitotic delays, cells ultimately divide at a frequency near that of untreated cells.  We show that many of the spindles in the dividing cells are bipolar, although they lack centrosomes at one or more spindle poles.  MLN8054-treated cells frequently show alignment defects during metaphase, lagging chromosomes in anaphase, and chromatin bridges during telophase.  Consistent with the chromosome segregation defects, cells treated with MLN8054 develop aneuploidy over time.  Taken together, these results suggest that Aurora A inhibition kills tumor cells through the development of deleterious aneuploidy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhlzReVziYI7Vg90H21EOLACvtfcHk0li-E0VE_Bqr4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmvFansbs%253D&md5=0c86835e0c42615fd69416856a05cdc6</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1128%2FMCB.02364-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.02364-06%26sid%3Dliteratum%253Aachs%26aulast%3DHoar%26aufirst%3DK.%26aulast%3DChakravarty%26aufirst%3DA.%26aulast%3DRabino%26aufirst%3DC.%26aulast%3DWysong%26aufirst%3DD.%26aulast%3DBowman%26aufirst%3DD.%26aulast%3DRoy%26aufirst%3DN.%26aulast%3DEcsedy%26aufirst%3DJ.%2BA.%26atitle%3DMln8054%252C%2520a%2520Small-Molecule%2520Inhibitor%2520of%2520Aurora%2520a%252C%2520Causes%2520Spindle%2520Pole%2520and%2520Chromosome%2520Congression%2520Defects%2520Leading%2520to%2520Aneuploidy%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2007%26volume%3D27%26spage%3D4513%26epage%3D4525%26doi%3D10.1128%2FMCB.02364-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Bemis, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span> </span><span class="NLM_article-title">The Properties of Known Drugs. 1. Molecular Frameworks</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">2887</span><span class="NLM_x">–</span> <span class="NLM_lpage">2893</span><span class="refDoi"> DOI: 10.1021/jm9602928</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9602928" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2887-2893&author=G.+W.+Bemisauthor=M.+A.+Murcko&title=The+Properties+of+Known+Drugs.+1.+Molecular+Frameworks&doi=10.1021%2Fjm9602928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">The Properties of Known Drugs. 1. Molecular Frameworks</span></div><div class="casAuthors">Bemis, Guy W.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2887-2893</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the common features present in drug mols., we use shape description methods to analyze a database of com. available drugs and prep. a list of common drug shapes.  A useful way of organizing this structural data is to group the atoms of each drug mol. into ring, linker, framework, and side chain atoms.  On the basis of the two-dimensional mol. structures (without regard to atom type, hybridization, and bond order), there are 1179 different frameworks among the 5120 compds. analyzed.  However, the shapes of half of the drugs in the database are described by the 32 most frequently occurring frameworks.  This suggests that the diversity of shapes in the set of known drugs is extremely low.  In our second method of anal., in which atom type, hybridization, and bond order are considered, more diversity is seen; there are 2506 different frameworks among the 5120 compds. in the database, and the most frequently occurring 42 frameworks account for only one-fourth of the drugs.  We discuss the possible interpretations of these findings and the way they may be used to guide future drug discovery research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoodEb3sfDSdrVg90H21EOLACvtfcHk0li-E0VE_Bqr4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D&md5=5e2c4fdfea9434456a0cca83de4185b3</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm9602928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9602928%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DG.%2BW.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DThe%2520Properties%2520of%2520Known%2520Drugs.%25201.%2520Molecular%2520Frameworks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2887%26epage%3D2893%26doi%3D10.1021%2Fjm9602928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Schuffenhauer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ertl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roggo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wetzel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koch, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldmann, H.</span><span> </span><span class="NLM_article-title">The Scaffold Tree – Visualization of the Scaffold Universe by Hierarchical Scaffold Classification</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">47</span><span class="NLM_x">–</span> <span class="NLM_lpage">58</span><span class="refDoi"> DOI: 10.1021/ci600338x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci600338x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlCit7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=47-58&author=A.+Schuffenhauerauthor=P.+Ertlauthor=S.+Roggoauthor=S.+Wetzelauthor=M.+A.+Kochauthor=H.+Waldmann&title=The+Scaffold+Tree+%E2%80%93+Visualization+of+the+Scaffold+Universe+by+Hierarchical+Scaffold+Classification&doi=10.1021%2Fci600338x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The Scaffold Tree - Visualization of the Scaffold Universe by Hierarchical Scaffold Classification</span></div><div class="casAuthors">Schuffenhauer, Ansgar; Ertl, Peter; Roggo, Silvio; Wetzel, Stefan; Koch, Marcus A.; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-58</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A hierarchical classification of chem. scaffolds (mol. framework, which is obtained by pruning all terminal side chains) has been introduced.  The mol. frameworks form the leaf nodes in the hierarchy trees.  By an iterative removal of rings, scaffolds forming the higher levels in the hierarchy tree are obtained.  Prioritization rules ensure that less characteristic, peripheral rings are removed first.  All scaffolds in the hierarchy tree are well-defined chem. entities making the classification chem. intuitive.  The classification is deterministic, data-set-independent, and scales linearly with the no. of compds. included in the data set.  The application of the classification is demonstrated on two data sets extd. from the PubChem database, namely, pyruvate kinase binders and a collection of pesticides.  The examples shown demonstrate that the classification procedure handles robustly synthetic structures and natural products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8XfR2xvGasbVg90H21EOLACvtfcHk0li-E0VE_Bqr4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlCit7jO&md5=89cec5a01a37802ab0826ecababfaf0a</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fci600338x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600338x%26sid%3Dliteratum%253Aachs%26aulast%3DSchuffenhauer%26aufirst%3DA.%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DRoggo%26aufirst%3DS.%26aulast%3DWetzel%26aufirst%3DS.%26aulast%3DKoch%26aufirst%3DM.%2BA.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DThe%2520Scaffold%2520Tree%2520%25E2%2580%2593%2520Visualization%2520of%2520the%2520Scaffold%2520Universe%2520by%2520Hierarchical%2520Scaffold%2520Classification%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D47%26epage%3D58%26doi%3D10.1021%2Fci600338x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Welsch, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snyder, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stockwell, B. R.</span><span> </span><span class="NLM_article-title">Privileged Scaffolds for Library Design and Drug Discovery</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">347</span><span class="NLM_x">–</span> <span class="NLM_lpage">361</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2010.02.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2Fj.cbpa.2010.02.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=20303320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=347-361&author=M.+E.+Welschauthor=S.+A.+Snyderauthor=B.+R.+Stockwell&title=Privileged+Scaffolds+for+Library+Design+and+Drug+Discovery&doi=10.1016%2Fj.cbpa.2010.02.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Privileged scaffolds for library design and drug discovery</span></div><div class="casAuthors">Welsch, Matthew E.; Snyder, Scott A.; Stockwell, Brent R.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">347-361</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This review explores the concept of using privileged scaffolds to identify biol. active compds. through building chem. libraries.  We hope to accomplish three main objectives: to provide one of the most comprehensive listings of privileged scaffolds; to reveal through four selected examples the present state of the art in privileged scaffold library synthesis (in hopes of inspiring new and even more creative approaches); and also to offer some thoughts on how new privileged scaffolds might be identified and exploited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWQPxdsWawiLVg90H21EOLACvtfcHk0lgoq4Zh-Uw2mA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsLs%253D&md5=fd0c2a642f063fd6f42f2c34fd186f8a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2010.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2010.02.018%26sid%3Dliteratum%253Aachs%26aulast%3DWelsch%26aufirst%3DM.%2BE.%26aulast%3DSnyder%26aufirst%3DS.%2BA.%26aulast%3DStockwell%26aufirst%3DB.%2BR.%26atitle%3DPrivileged%2520Scaffolds%2520for%2520Library%2520Design%2520and%2520Drug%2520Discovery%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D14%26spage%3D347%26epage%3D361%26doi%3D10.1016%2Fj.cbpa.2010.02.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Aronov, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McClain, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moody, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span> </span><span class="NLM_article-title">Kinase-Likeness and Kinase-Privileged Fragments: Toward Virtual Polypharmacology</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1214</span><span class="NLM_x">–</span> <span class="NLM_lpage">1222</span><span class="refDoi"> DOI: 10.1021/jm701021b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm701021b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=1214-1222&author=A.+M.+Aronovauthor=B.+McClainauthor=C.+S.+Moodyauthor=M.+A.+Murcko&title=Kinase-Likeness+and+Kinase-Privileged+Fragments%3A+Toward+Virtual+Polypharmacology&doi=10.1021%2Fjm701021b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm701021b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701021b%26sid%3Dliteratum%253Aachs%26aulast%3DAronov%26aufirst%3DA.%2BM.%26aulast%3DMcClain%26aufirst%3DB.%26aulast%3DMoody%26aufirst%3DC.%2BS.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DKinase-Likeness%2520and%2520Kinase-Privileged%2520Fragments%253A%2520Toward%2520Virtual%2520Polypharmacology%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D1214%26epage%3D1222%26doi%3D10.1021%2Fjm701021b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Yan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, J.</span><span> </span><span class="NLM_article-title">Aurora-a Kinase Inhibitor Scaffolds and Binding Modes</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">260</span><span class="NLM_x">–</span> <span class="NLM_lpage">269</span><span class="refDoi"> DOI: 10.1016/j.drudis.2010.12.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2Fj.drudis.2010.12.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=21147253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjs1WrtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=260-269&author=A.+Yanauthor=L.+Wangauthor=S.+Xuauthor=J.+Xu&title=Aurora-a+Kinase+Inhibitor+Scaffolds+and+Binding+Modes&doi=10.1016%2Fj.drudis.2010.12.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Aurora-A kinase inhibitor scaffolds and binding modes</span></div><div class="casAuthors">Yan, Aixia; Wang, Liyu; Xu, Shuyu; Xu, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5/6</span>),
    <span class="NLM_cas:pages">260-269</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Aurora kinases (A-C) belong to the serine/threonine protein kinase family.  In recent years, the constitutive or elevated expression of Aurora kinases has been found in cancer cells and oncogene transfected cells.  In this review, we summarize the common binding modes of Aurora-A kinase inhibitors, the hot spot residues in the binding sites and the privileged inhibitor structures.  Our review of the reported chem. scaffolds of Aurora-A kinase inhibitors and their binding modes could provide a useful framework from which new design strategies for inhibitors might be assessed or developed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqi7pOvViuZ0rVg90H21EOLACvtfcHk0lgoq4Zh-Uw2mA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjs1WrtLk%253D&md5=5c3cf3953c13de2af4596348f10432de</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2010.12.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2010.12.003%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DJ.%26atitle%3DAurora-a%2520Kinase%2520Inhibitor%2520Scaffolds%2520and%2520Binding%2520Modes%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2011%26volume%3D16%26spage%3D260%26epage%3D269%26doi%3D10.1016%2Fj.drudis.2010.12.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Uitdehaag, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verkaar, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alwan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Man, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buijsman, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaman, G. J.</span><span> </span><span class="NLM_article-title">A Guide to Picking the Most Selective Kinase Inhibitor Tool Compounds for Pharmacological Validation of Drug Targets</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">166</span><span class="NLM_x">, </span> <span class="NLM_fpage">858</span><span class="NLM_x">–</span> <span class="NLM_lpage">876</span><span class="refDoi"> DOI: 10.1111/j.1476-5381.2012.01859.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1111%2Fj.1476-5381.2012.01859.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=22250956" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC38XotlWmsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2012&pages=858-876&author=J.+C.+Uitdehaagauthor=F.+Verkaarauthor=H.+Alwanauthor=J.+de+Manauthor=R.+C.+Buijsmanauthor=G.+J.+Zaman&title=A+Guide+to+Picking+the+Most+Selective+Kinase+Inhibitor+Tool+Compounds+for+Pharmacological+Validation+of+Drug+Targets&doi=10.1111%2Fj.1476-5381.2012.01859.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A guide to picking the most selective kinase inhibitor tool compounds for pharmacological validation of drug targets</span></div><div class="casAuthors">Uitdehaag, Joost C. M.; Verkaar, Folkert; Alwan, Husam; de Man, Jos; Buijsman, Rogier C.; Zaman, Guido J. R.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">166</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">858-876</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  To establish the druggability of a target, genetic validation needs to be supplemented with pharmacol. validation.  Pharmacol. studies, esp. in the kinase field, are hampered by the fact that many ref. inhibitors are not fully selective for one target.  Fortunately, the initial trickle of selective inhibitors released in the public domain has steadily swelled into a stream.  However, rationally picking the most selective tool compd. out of the increasing amts. of available inhibitors has become progressively difficult due to the lack of accurate quant. descriptors of drug selectivity.  A recently published approach, termed "selectivity entropy", is an improved way of expressing selectivity as a single-value parameter and enables rank ordering of inhibitors.  We provide a guide to select the best tool compds. for pharmacol. validation expts. of candidate drug targets using selectivity entropy.  In addn., we recommend which inhibitors to use for studying the biol. of the 20 most investigated kinases that are clin. relevant: Abl (ABL1), AKT1, ALK, Aurora A/B, CDKs, MET, CSF1R (FMS), EGFR, FLT3, ERBB2 (HER2), IKBKB (IKK2), JAK2/3, JNK1/2/3 (MAPK8/9/10), MEK1/2, PLK1, PI3Ks, p38α (MAPK14), BRAF, SRC and VEGFR2 (KDR).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtE6alezH-ArVg90H21EOLACvtfcHk0lgoq4Zh-Uw2mA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XotlWmsLw%253D&md5=26238277768843c7f6e44f037dcfe174</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2012.01859.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2012.01859.x%26sid%3Dliteratum%253Aachs%26aulast%3DUitdehaag%26aufirst%3DJ.%2BC.%26aulast%3DVerkaar%26aufirst%3DF.%26aulast%3DAlwan%26aufirst%3DH.%26aulast%3Dde%2BMan%26aufirst%3DJ.%26aulast%3DBuijsman%26aufirst%3DR.%2BC.%26aulast%3DZaman%26aufirst%3DG.%2BJ.%26atitle%3DA%2520Guide%2520to%2520Picking%2520the%2520Most%2520Selective%2520Kinase%2520Inhibitor%2520Tool%2520Compounds%2520for%2520Pharmacological%2520Validation%2520of%2520Drug%2520Targets%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2012%26volume%3D166%26spage%3D858%26epage%3D876%26doi%3D10.1111%2Fj.1476-5381.2012.01859.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Hu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Exploring the Scaffold Universe of Kinase Inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">315</span><span class="NLM_x">–</span> <span class="NLM_lpage">332</span><span class="refDoi"> DOI: 10.1021/jm501237k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501237k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=315-332&author=Y.+Huauthor=J.+Bajorath&title=Exploring+the+Scaffold+Universe+of+Kinase+Inhibitors&doi=10.1021%2Fjm501237k"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm501237k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501237k%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DExploring%2520the%2520Scaffold%2520Universe%2520of%2520Kinase%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D315%26epage%3D332%26doi%3D10.1021%2Fjm501237k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Gaulton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellis, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bento, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hersey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Light, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGlinchey, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michalovich, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Lazikani, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span> </span><span class="NLM_article-title">Chembl: A Large-Scale Bioactivity Database for Drug Discovery</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">D1100</span><span class="NLM_x">–</span> <span class="NLM_lpage">D1107</span><span class="refDoi"> DOI: 10.1093/nar/gkr777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1093%2Fnar%2Fgkr777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=21948594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D1100-D1107&author=A.+Gaultonauthor=L.+J.+Bellisauthor=A.+P.+Bentoauthor=J.+Chambersauthor=M.+Daviesauthor=A.+Herseyauthor=Y.+Lightauthor=S.+McGlincheyauthor=D.+Michalovichauthor=B.+Al-Lazikaniauthor=J.+P.+Overington&title=Chembl%3A+A+Large-Scale+Bioactivity+Database+for+Drug+Discovery&doi=10.1093%2Fnar%2Fgkr777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">ChEMBL: a large-scale bioactivity database for drug discovery</span></div><div class="casAuthors">Gaulton, Anna; Bellis, Louisa J.; Bento, A. Patricia; Chambers, Jon; Davies, Mark; Hersey, Anne; Light, Yvonne; McGlinchey, Shaun; Michalovich, David; Al-Lazikani, Bissan; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1100-D1107</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an Open Data database contg. binding, functional and ADMET information for a large no. of drug-like bioactive compds.  These data are manually abstracted from the primary published literature on a regular basis, then further curated and standardized to maximize their quality and utility across a wide range of chem. biol. and drug-discovery research problems.  Currently, the database contains 5.4 million bioactivity measurements for more than 1 million compds. and 5200 protein targets.  Access is available through a web-based interface, data downloads and web services at: https://www.ebi.ac.uk/chembldb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9e9jRENTtubVg90H21EOLACvtfcHk0ljpq5G6ectqpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN&md5=aedf7793e1ca54b6a4fa272ea3ef7d0e</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr777%26sid%3Dliteratum%253Aachs%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DMichalovich%26aufirst%3DD.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DChembl%253A%2520A%2520Large-Scale%2520Bioactivity%2520Database%2520for%2520Drug%2520Discovery%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DD1100%26epage%3DD1107%26doi%3D10.1093%2Fnar%2Fgkr777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Simonin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Awale, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brand, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Deursen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fine, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kovacs, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Häfliger, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gyimesi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sithampari, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charles, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hediger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reymond, J. L.</span><span> </span><span class="NLM_article-title">Optimization of Trpv6 Calcium Channel Inhibitors Using a New 3d Ligand Based Virtual Screening Method</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">14748</span><span class="NLM_x">–</span> <span class="NLM_lpage">14752</span><span class="refDoi"> DOI: 10.1002/anie.201507320</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1002%2Fanie.201507320" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=14748-14752&author=C.+Simoninauthor=M.+Awaleauthor=M.+Brandauthor=R.+van+Deursenauthor=J.+Schwartzauthor=M.+Fineauthor=G.+Kovacsauthor=P.+H%C3%A4fligerauthor=G.+Gyimesiauthor=A.+Sithampariauthor=R.+P.+Charlesauthor=M.+Hedigerauthor=J.+L.+Reymond&title=Optimization+of+Trpv6+Calcium+Channel+Inhibitors+Using+a+New+3d+Ligand+Based+Virtual+Screening+Method&doi=10.1002%2Fanie.201507320"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507320&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507320%26sid%3Dliteratum%253Aachs%26aulast%3DSimonin%26aufirst%3DC.%26aulast%3DAwale%26aufirst%3DM.%26aulast%3DBrand%26aufirst%3DM.%26aulast%3Dvan%2BDeursen%26aufirst%3DR.%26aulast%3DSchwartz%26aufirst%3DJ.%26aulast%3DFine%26aufirst%3DM.%26aulast%3DKovacs%26aufirst%3DG.%26aulast%3DH%25C3%25A4fliger%26aufirst%3DP.%26aulast%3DGyimesi%26aufirst%3DG.%26aulast%3DSithampari%26aufirst%3DA.%26aulast%3DCharles%26aufirst%3DR.%2BP.%26aulast%3DHediger%26aufirst%3DM.%26aulast%3DReymond%26aufirst%3DJ.%2BL.%26atitle%3DOptimization%2520of%2520Trpv6%2520Calcium%2520Channel%2520Inhibitors%2520Using%2520a%2520New%25203d%2520Ligand%2520Based%2520Virtual%2520Screening%2520Method%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D14748%26epage%3D14752%26doi%3D10.1002%2Fanie.201507320" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Kearsley, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, G.</span><span> </span><span class="NLM_article-title">An Alternative Method for the Alignment of Molecular Structures: Maximizing Electrostatic and Steric Overlap</span> <span class="citation_source-journal">Tetrahedron Comput. Methodol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">615</span><span class="NLM_x">–</span> <span class="NLM_lpage">633</span><span class="refDoi"> DOI: 10.1016/0898-5529(90)90162-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2F0898-5529%2890%2990162-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADyaK3sXmtlGj" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1990&pages=615-633&author=S.+K.+Kearsleyauthor=G.+Smith&title=An+Alternative+Method+for+the+Alignment+of+Molecular+Structures%3A+Maximizing+Electrostatic+and+Steric+Overlap&doi=10.1016%2F0898-5529%2890%2990162-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">An alternative method for the alignment of molecular structures:  maximizing electrostatic and steric overlap</span></div><div class="casAuthors">Kearsley, Simon K.; Smith, Graham M.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Computer Methodology</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6c</span>),
    <span class="NLM_cas:pages">615-33</span>CODEN:
                <span class="NLM_cas:coden">TCMTE6</span>;
        ISSN:<span class="NLM_cas:issn">0898-5529</span>.
    </div><div class="casAbstract">The SEAL method has been developed which will optimize the alignment of two 3-dimensional structures using their at. partial charges and steric vols. as factors.  In addn., this method will perform the superimposition many times using randomly generated starting configurations and keep only the best unique results based on the value of the alignment function.  The computer generated alignments of methotrexate and dihydrofolic acid are compared with the alignments found in the active site of dihydrfolate reductase.  The marine neurotoxins, saxitoxin and tetrodotoxin, are aligned and the results compared to other alignment techniques.  The relationship of the SEAL alignments to the potential field error is explored.  The program is on disk.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFtoxyn792HbVg90H21EOLACvtfcHk0ljpq5G6ectqpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXmtlGj&md5=f03505ee6ce9275af178be0e7ba29f7e</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2F0898-5529%2890%2990162-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0898-5529%252890%252990162-2%26sid%3Dliteratum%253Aachs%26aulast%3DKearsley%26aufirst%3DS.%2BK.%26aulast%3DSmith%26aufirst%3DG.%26atitle%3DAn%2520Alternative%2520Method%2520for%2520the%2520Alignment%2520of%2520Molecular%2520Structures%253A%2520Maximizing%2520Electrostatic%2520and%2520Steric%2520Overlap%26jtitle%3DTetrahedron%2520Comput.%2520Methodol.%26date%3D1990%26volume%3D3%26spage%3D615%26epage%3D633%26doi%3D10.1016%2F0898-5529%2890%2990162-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Kennewell, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ducrot, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luttmann, C.</span><span> </span><span class="NLM_article-title">Identification of Target-Specific Bioisosteric Fragments from Ligand-Protein Crystallographic Data</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">385</span><span class="NLM_x">–</span> <span class="NLM_lpage">394</span><span class="refDoi"> DOI: 10.1007/s10822-006-9072-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1007%2Fs10822-006-9072-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2006&pages=385-394&author=E.+A.+Kennewellauthor=P.+Willettauthor=P.+Ducrotauthor=C.+Luttmann&title=Identification+of+Target-Specific+Bioisosteric+Fragments+from+Ligand-Protein+Crystallographic+Data&doi=10.1007%2Fs10822-006-9072-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1007%2Fs10822-006-9072-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-006-9072-0%26sid%3Dliteratum%253Aachs%26aulast%3DKennewell%26aufirst%3DE.%2BA.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DDucrot%26aufirst%3DP.%26aulast%3DLuttmann%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520Target-Specific%2520Bioisosteric%2520Fragments%2520from%2520Ligand-Protein%2520Crystallographic%2520Data%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2006%26volume%3D20%26spage%3D385%26epage%3D394%26doi%3D10.1007%2Fs10822-006-9072-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Awale, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reymond, J. L.</span><span> </span><span class="NLM_article-title">Atom Pair 2d-Fingerprints Perceive 3d-Molecular Shape and Pharmacophores for Very Fast Virtual Screening of Zinc and Gdb-17</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1892</span><span class="NLM_x">–</span> <span class="NLM_lpage">1897</span><span class="refDoi"> DOI: 10.1021/ci500232g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500232g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpvFCltr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1892-1897&author=M.+Awaleauthor=J.+L.+Reymond&title=Atom+Pair+2d-Fingerprints+Perceive+3d-Molecular+Shape+and+Pharmacophores+for+Very+Fast+Virtual+Screening+of+Zinc+and+Gdb-17&doi=10.1021%2Fci500232g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Atom Pair 2D-Fingerprints Perceive 3D-Molecular Shape and Pharmacophores for Very Fast Virtual Screening of ZINC and GDB-17</span></div><div class="casAuthors">Awale, Mahendra; Reymond, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1892-1907</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Three-dimensional (3D) mol. shape and pharmacophores are important determinants of the biol. activity of org. mols.; however, a precise computation of 3D-shape is generally too slow for virtual screening of very large databases.  A reinvestigation of the concept of atom pairs initially reported by Carhart et al. and extended by Schneider et al. showed that a simple atom pair fingerprint (APfp) counting atom pairs at increasing topol. distances in 2D-structures without atom property assignment correlates with various representations of mol. shape extd. from the 3D-structures.  A related 55-dimensional atom pair fingerprint extended with atom properties (Xfp) provided an efficient pharmacophore fingerprint with good performance for ligand-based virtual screening such as the recovery of active compds. from decoys in DUD, and overlap with the ROCS 3D-pharmacophore scoring function.  The APfp and Xfp data were organized for web-based extremely fast nearest-neighbor searching in ZINC (13.5 M compds.) and GDB-17 (50 M random subset) freely accessible at www.gdb.unibe.ch.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiXcOsBelbq7Vg90H21EOLACvtfcHk0lhfOEEro7GCmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpvFCltr4%253D&md5=f6a994a6c16db780b485087fd3925be7</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fci500232g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500232g%26sid%3Dliteratum%253Aachs%26aulast%3DAwale%26aufirst%3DM.%26aulast%3DReymond%26aufirst%3DJ.%2BL.%26atitle%3DAtom%2520Pair%25202d-Fingerprints%2520Perceive%25203d-Molecular%2520Shape%2520and%2520Pharmacophores%2520for%2520Very%2520Fast%2520Virtual%2520Screening%2520of%2520Zinc%2520and%2520Gdb-17%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1892%26epage%3D1897%26doi%3D10.1021%2Fci500232g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Awale, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reymond, J. L.</span><span> </span><span class="NLM_article-title">Stereoselective Virtual Screening of the Zinc Database Using Atom Pair 3d-Fingerprints</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="refDoi"> DOI: 10.1186/s13321-014-0051-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1186%2Fs13321-014-0051-5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=3&author=M.+Awaleauthor=X.+Jinauthor=J.+L.+Reymond&title=Stereoselective+Virtual+Screening+of+the+Zinc+Database+Using+Atom+Pair+3d-Fingerprints&doi=10.1186%2Fs13321-014-0051-5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1186%2Fs13321-014-0051-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-014-0051-5%26sid%3Dliteratum%253Aachs%26aulast%3DAwale%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DX.%26aulast%3DReymond%26aufirst%3DJ.%2BL.%26atitle%3DStereoselective%2520Virtual%2520Screening%2520of%2520the%2520Zinc%2520Database%2520Using%2520Atom%2520Pair%25203d-Fingerprints%26jtitle%3DJ.%2520Cheminf.%26date%3D2015%26volume%3D7%26spage%3D3%26doi%3D10.1186%2Fs13321-014-0051-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Montalbetti, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simonin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simonin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Awale, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reymond, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hediger, M. A.</span><span> </span><span class="NLM_article-title">Discovery and Characterization of a Novel Non-Competitive Inhibitor of the Divalent Metal Transporter Dmt1/Slc11a2</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">96</span><span class="NLM_x">, </span> <span class="NLM_fpage">216</span><span class="NLM_x">–</span> <span class="NLM_lpage">224</span><span class="refDoi"> DOI: 10.1016/j.bcp.2015.05.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2Fj.bcp.2015.05.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=26047847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWgtrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=2015&pages=216-224&author=N.+Montalbettiauthor=A.+Simoninauthor=C.+Simoninauthor=M.+Awaleauthor=J.+L.+Reymondauthor=M.+A.+Hediger&title=Discovery+and+Characterization+of+a+Novel+Non-Competitive+Inhibitor+of+the+Divalent+Metal+Transporter+Dmt1%2FSlc11a2&doi=10.1016%2Fj.bcp.2015.05.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of a novel non-competitive inhibitor of the divalent metal transporter DMT1/SLC11A2</span></div><div class="casAuthors">Montalbetti, Nicolas; Simonin, Alexandre; Simonin, Celine; Awale, Mahendra; Reymond, Jean-Louis; Hediger, Matthias A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">216-224</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Divalent metal transporter-1 (SLC11A2/DMT1) uses the H+ electrochem. gradient as the driving force to transport divalent metal ions such as Fe2+, Mn2+ and others metals into mammalian cells.  DMT1 is ubiquitously expressed, most notably in proximal duodenum, immature erythroid cells, brain and kidney.  This transporter mediates H+-coupled transport of ferrous iron across the apical membrane of enterocytes.  In addn., in cells such as to erythroid precursors, following transferrin receptor (TfR) mediated endocytosis; it mediates H+-coupled exit of ferrous iron from endocytic vesicles into the cytosol.  Dysfunction of human DMT1 is assocd. with several pathologies such as iron deficiency anemia hemochromatosis, Parkinson's disease and Alzheimer's disease, as well as colorectal cancer and esophageal adenocarcinoma, making DMT1 an attractive target for drug discovery.  In the present study, we performed a ligand-based virtual screening of the Princeton database (700,000 com. available compds.) to search for pharmacophore shape analogs of recently reported DMT1 inhibitors.  We discovered a new compd., named pyrimidinone 8, which mediates a reversible linear non-competitive inhibition of human DMT1 (hDMT1) transport activity with a Ki of ∼20 μM.  This compd. does not affect hDMT1 cell surface expression and shows no dependence on extracellular pH.  To our knowledge, this is the first exptl. evidence that hDMT1 can be allosterically modulated by pharmacol. agents.  Pyrimidinone 8 represents a novel versatile tool compd. and it may serve as a lead structure for the development of therapeutic compds. for pre-clin. assessment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCuVTNtEKIvLVg90H21EOLACvtfcHk0lhfOEEro7GCmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWgtrnN&md5=dda3c8979ea2c053a409da1cae3db243</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2015.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2015.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DMontalbetti%26aufirst%3DN.%26aulast%3DSimonin%26aufirst%3DA.%26aulast%3DSimonin%26aufirst%3DC.%26aulast%3DAwale%26aufirst%3DM.%26aulast%3DReymond%26aufirst%3DJ.%2BL.%26aulast%3DHediger%26aufirst%3DM.%2BA.%26atitle%3DDiscovery%2520and%2520Characterization%2520of%2520a%2520Novel%2520Non-Competitive%2520Inhibitor%2520of%2520the%2520Divalent%2520Metal%2520Transporter%2520Dmt1%252FSlc11a2%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2015%26volume%3D96%26spage%3D216%26epage%3D224%26doi%3D10.1016%2Fj.bcp.2015.05.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sterling, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolstad, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span> </span><span class="NLM_article-title">Zinc: A Free Tool to Discover Chemistry for Biology</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1757</span><span class="NLM_x">–</span> <span class="NLM_lpage">1768</span><span class="refDoi"> DOI: 10.1021/ci3001277</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3001277" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1757-1768&author=J.+J.+Irwinauthor=T.+Sterlingauthor=M.+M.+Mysingerauthor=E.+S.+Bolstadauthor=R.+G.+Coleman&title=Zinc%3A+A+Free+Tool+to+Discover+Chemistry+for+Biology&doi=10.1021%2Fci3001277"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC: A Free Tool to Discover Chemistry for Biology</span></div><div class="casAuthors">Irwin, John J.; Sterling, Teague; Mysinger, Michael M.; Bolstad, Erin S.; Coleman, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1757-1768</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ZINC is a free public resource for ligand discovery.  The database contains over twenty million com. available mols. in biol. relevant representations that may be downloaded in popular ready-to-dock formats and subsets.  The Web site also enables searches by structure, biol. activity, phys. property, vendor, catalog no., name, and CAS no.  Small custom subsets may be created, edited, shared, docked, downloaded, and conveyed to a vendor for purchase.  The database is maintained and curated for a high purchasing success rate and is freely available at zinc.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotyhntxgvRpbVg90H21EOLACvtfcHk0lhfOEEro7GCmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D&md5=97f2ede64afc6b5e3ea2f279e38e32a0</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fci3001277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3001277%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DBolstad%26aufirst%3DE.%2BS.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26atitle%3DZinc%253A%2520A%2520Free%2520Tool%2520to%2520Discover%2520Chemistry%2520for%2520Biology%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1757%26epage%3D1768%26doi%3D10.1021%2Fci3001277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Baell, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holloway, G. A.</span><span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (Pains) from Screening Libraries and for Their Exclusion in Bioassays</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2719</span><span class="NLM_x">–</span> <span class="NLM_lpage">2740</span><span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28Pains%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0ljjtPzFuTA0Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528Pains%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Sterling, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span> </span><span class="NLM_article-title">Zinc 15--Ligand Discovery for Everyone</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">2324</span><span class="NLM_x">–</span> <span class="NLM_lpage">2337</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00559</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00559" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1OhurbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2324-2337&author=T.+Sterlingauthor=J.+J.+Irwin&title=Zinc+15%2D%2DLigand+Discovery+for+Everyone&doi=10.1021%2Facs.jcim.5b00559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC 15 - Ligand Discovery for Everyone</span></div><div class="casAuthors">Sterling, Teague; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2324-2337</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many questions about the biol. activity and availability of small mols. remain inaccessible to investigators who could most benefit from their answers.  To narrow the gap between chemoinformatics and biol., we have developed a suite of ligand annotation, purchasability, target, and biol. assocn. tools, incorporated into ZINC and meant for investigators who are not computer specialists.  The new version contains over 120 million purchasable "drug-like" compds. - effectively all org. mols. that are for sale - a quarter of which are available for immediate delivery.  ZINC connects purchasable compds. to high-value ones such as metabolites, drugs, natural products, and annotated compds. from the literature.  Compds. may be accessed by the genes for which they are annotated as well as the major and minor target classes to which those genes belong.  It offers new anal. tools that are easy for nonspecialists yet with few limitations for experts.  ZINC retains its original 3D roots - all mols. are available in biol. relevant, ready-to-dock formats.  ZINC is freely available at http://zinc15.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWfkDnal3D3bVg90H21EOLACvtfcHk0ljjtPzFuTA0Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1OhurbF&md5=e767c26c1d0aff7432c403c59a5d2b8b</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00559%26sid%3Dliteratum%253Aachs%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DZinc%252015--Ligand%2520Discovery%2520for%2520Everyone%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D2324%26epage%3D2337%26doi%3D10.1021%2Facs.jcim.5b00559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Degorce, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Card, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soh, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trinquet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapik, G. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, B.</span><span> </span><span class="NLM_article-title">Htrf: A Technology Tailored for Drug Discovery - a Review of Theoretical Aspects and Recent Applications</span> <span class="citation_source-journal">Curr. Chem. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">22</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span><span class="refDoi"> DOI: 10.2174/1875397300903010022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.2174%2F1875397300903010022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=20161833" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD1MXosFWnsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2009&pages=22-32&author=F.+Degorceauthor=A.+Cardauthor=S.+Sohauthor=E.+Trinquetauthor=G.+P.+Knapikauthor=B.+Xie&title=Htrf%3A+A+Technology+Tailored+for+Drug+Discovery+-+a+Review+of+Theoretical+Aspects+and+Recent+Applications&doi=10.2174%2F1875397300903010022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">HTRF: a technology tailored for drug discovery - a review of theoretical aspects and recent applications</span></div><div class="casAuthors">Degorce, Francois; Card, Amy; Soh, Sharon; Trinquet, Eric; Knapik, Glenn P.; Xie, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Current Chemical Genomics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">22-32</span>CODEN:
                <span class="NLM_cas:coden">CCGUAQ</span>;
        ISSN:<span class="NLM_cas:issn">1875-3973</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  HTRF (Homogeneous Time Resolved Fluorescence) is the most frequently used generic assay technol. to measure analytes in a homogenous format, which is the ideal platform used for drug target studies in high-throughput screening (HTS).  This technol. combines fluorescence resonance energy transfer technol. (FRET) with time-resolved measurement (TR).  In TR-FRET assays, a signal is generated through fluorescent resonance energy transfer between a donor and an acceptor mol. when in close proximity to each other.  Buffer and media interference is dramatically reduced by dual-wavelength detection, and the final signal is proportional to the extent of product formation.  The HTRF assay is usually sensitive and robust that can be miniaturized into the 384 and 1536-well plate formats.  This assay technol. has been applied to many antibody-based assays including GPCR signaling (cAMP and IP-One), kinases, cytokines and biomarkers, bioprocess (antibody and protein prodn.), as well as the assays for protein-protein, protein-peptide, and protein-DNA/RNA interactions.  Since its introduction to the drug-screening world over ten years ago, researchers have used HTRF to expedite the study of GPCRs, kinases, new biomarkers, protein-protein interactions, and other targets of interest.  HTRF has also been utilized as an alternative method for bioprocess monitoring.  The first-generation HTRF technol., which uses Europium cryptate as a fluorescence donor to monitor reactions between biomols., was extended in 2008 through the introduction of a second-generation donor, Terbium cryptate (Tb), enhancing screening performance.  Terbium cryptate possesses different photophys. properties compared to Europium, including increased quantum yield and a higher molar extinction coeff.  In addn. to being compatible with the same acceptor fluorophors used with Europium, it can serve as a donor fluorophore to green-emitting fluors because it has multiple emission peaks including one at 490 nm.  Moreover, all Terbium HTRF assays can be read on the same HTRF-compatible instruments as Europium HTRF assays.  Overall, HTRF is a highly sensitive, robust technol. for the detection of mol. interactions in vitro and is widely used for primary and secondary screening phases of drug development.  This review addresses the general principles of HTRF and its current applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKDP8Vd8k247Vg90H21EOLACvtfcHk0ljjtPzFuTA0Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXosFWnsr4%253D&md5=9590149ceeed17ce424522ef85aa32f9</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.2174%2F1875397300903010022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1875397300903010022%26sid%3Dliteratum%253Aachs%26aulast%3DDegorce%26aufirst%3DF.%26aulast%3DCard%26aufirst%3DA.%26aulast%3DSoh%26aufirst%3DS.%26aulast%3DTrinquet%26aufirst%3DE.%26aulast%3DKnapik%26aufirst%3DG.%2BP.%26aulast%3DXie%26aufirst%3DB.%26atitle%3DHtrf%253A%2520A%2520Technology%2520Tailored%2520for%2520Drug%2520Discovery%2520-%2520a%2520Review%2520of%2520Theoretical%2520Aspects%2520and%2520Recent%2520Applications%26jtitle%3DCurr.%2520Chem.%2520Genomics%26date%3D2009%26volume%3D3%26spage%3D22%26epage%3D32%26doi%3D10.2174%2F1875397300903010022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Ke, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coumar, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiao, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsieh, H. P.</span><span> </span><span class="NLM_article-title">Ligand Efficiency Based Approach for Efficient Virtual Screening of Compound Libraries</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">226</span><span class="NLM_x">–</span> <span class="NLM_lpage">235</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2014.06.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2Fj.ejmech.2014.06.029" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2014&pages=226-235&author=Y.+Y.+Keauthor=M.+S.+Coumarauthor=H.+Y.+Shiaoauthor=W.+C.+Wangauthor=C.+W.+Chenauthor=J.+S.+Songauthor=C.+H.+Chenauthor=W.+H.+Linauthor=S.+H.+Wuauthor=J.+T.+Hsuauthor=C.+M.+Changauthor=H.+P.+Hsieh&title=Ligand+Efficiency+Based+Approach+for+Efficient+Virtual+Screening+of+Compound+Libraries&doi=10.1016%2Fj.ejmech.2014.06.029"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2014.06.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2014.06.029%26sid%3Dliteratum%253Aachs%26aulast%3DKe%26aufirst%3DY.%2BY.%26aulast%3DCoumar%26aufirst%3DM.%2BS.%26aulast%3DShiao%26aufirst%3DH.%2BY.%26aulast%3DWang%26aufirst%3DW.%2BC.%26aulast%3DChen%26aufirst%3DC.%2BW.%26aulast%3DSong%26aufirst%3DJ.%2BS.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DLin%26aufirst%3DW.%2BH.%26aulast%3DWu%26aufirst%3DS.%2BH.%26aulast%3DHsu%26aufirst%3DJ.%2BT.%26aulast%3DChang%26aufirst%3DC.%2BM.%26aulast%3DHsieh%26aufirst%3DH.%2BP.%26atitle%3DLigand%2520Efficiency%2520Based%2520Approach%2520for%2520Efficient%2520Virtual%2520Screening%2520of%2520Compound%2520Libraries%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2014%26volume%3D83%26spage%3D226%26epage%3D235%26doi%3D10.1016%2Fj.ejmech.2014.06.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Emanuel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rugg, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gruninger, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuentes-Pesquera, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Connolly, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wetter, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hollister, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Napier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jolliffe, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Middleton, S. A.</span><span> </span><span class="NLM_article-title">The in Vitro and in Vivo Effects of Jnj-7706621: A Dual Inhibitor of Cyclin-Dependent Kinases and Aurora Kinases</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">9038</span><span class="NLM_x">–</span> <span class="NLM_lpage">9046</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-05-0882</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1158%2F0008-5472.CAN-05-0882" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2005&pages=9038-9046&author=S.+Emanuelauthor=C.+A.+Ruggauthor=R.+H.+Gruningerauthor=R.+Linauthor=A.+Fuentes-Pesqueraauthor=P.+J.+Connollyauthor=S.+K.+Wetterauthor=B.+Hollisterauthor=W.+W.+Krugerauthor=C.+Napierauthor=L.+Jolliffeauthor=S.+A.+Middleton&title=The+in+Vitro+and+in+Vivo+Effects+of+Jnj-7706621%3A+A+Dual+Inhibitor+of+Cyclin-Dependent+Kinases+and+Aurora+Kinases&doi=10.1158%2F0008-5472.CAN-05-0882"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-05-0882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-05-0882%26sid%3Dliteratum%253Aachs%26aulast%3DEmanuel%26aufirst%3DS.%26aulast%3DRugg%26aufirst%3DC.%2BA.%26aulast%3DGruninger%26aufirst%3DR.%2BH.%26aulast%3DLin%26aufirst%3DR.%26aulast%3DFuentes-Pesquera%26aufirst%3DA.%26aulast%3DConnolly%26aufirst%3DP.%2BJ.%26aulast%3DWetter%26aufirst%3DS.%2BK.%26aulast%3DHollister%26aufirst%3DB.%26aulast%3DKruger%26aufirst%3DW.%2BW.%26aulast%3DNapier%26aufirst%3DC.%26aulast%3DJolliffe%26aufirst%3DL.%26aulast%3DMiddleton%26aufirst%3DS.%2BA.%26atitle%3DThe%2520in%2520Vitro%2520and%2520in%2520Vivo%2520Effects%2520of%2520Jnj-7706621%253A%2520A%2520Dual%2520Inhibitor%2520of%2520Cyclin-Dependent%2520Kinases%2520and%2520Aurora%2520Kinases%26jtitle%3DCancer%2520Res.%26date%3D2005%26volume%3D65%26spage%3D9038%26epage%3D9046%26doi%3D10.1158%2F0008-5472.CAN-05-0882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Wyatt, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodhead, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berdini, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boulstridge, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carr, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cross, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Devine, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Early, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feltell, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMenamin, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Navarro, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Reilly, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reule, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saxty, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seavers, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Squires, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trewartha, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolford, A. J.</span><span> </span><span class="NLM_article-title">Identification of N-(4-Piperidinyl)-4-(2,6-Dichlorobenzoylamino)-1h-Pyrazole-3-Carboxamide (At7519), a Novel Cyclin Dependent Kinase Inhibitor Using Fragment-Based X-Ray Crystallography and Structure Based Drug Design</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">4986</span><span class="NLM_x">–</span> <span class="NLM_lpage">4999</span><span class="refDoi"> DOI: 10.1021/jm800382h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800382h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=4986-4999&author=P.+G.+Wyattauthor=A.+J.+Woodheadauthor=V.+Berdiniauthor=J.+A.+Boulstridgeauthor=M.+G.+Carrauthor=D.+M.+Crossauthor=D.+J.+Davisauthor=L.+A.+Devineauthor=T.+R.+Earlyauthor=R.+E.+Feltellauthor=E.+J.+Lewisauthor=R.+L.+McMenaminauthor=E.+F.+Navarroauthor=M.+A.+O%E2%80%99Brienauthor=M.+O%E2%80%99Reillyauthor=M.+Reuleauthor=G.+Saxtyauthor=L.+C.+Seaversauthor=D.+M.+Smithauthor=M.+S.+Squiresauthor=G.+Trewarthaauthor=M.+T.+Walkerauthor=A.+J.+Woolford&title=Identification+of+N-%284-Piperidinyl%29-4-%282%2C6-Dichlorobenzoylamino%29-1h-Pyrazole-3-Carboxamide+%28At7519%29%2C+a+Novel+Cyclin+Dependent+Kinase+Inhibitor+Using+Fragment-Based+X-Ray+Crystallography+and+Structure+Based+Drug+Design&doi=10.1021%2Fjm800382h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm800382h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800382h%26sid%3Dliteratum%253Aachs%26aulast%3DWyatt%26aufirst%3DP.%2BG.%26aulast%3DWoodhead%26aufirst%3DA.%2BJ.%26aulast%3DBerdini%26aufirst%3DV.%26aulast%3DBoulstridge%26aufirst%3DJ.%2BA.%26aulast%3DCarr%26aufirst%3DM.%2BG.%26aulast%3DCross%26aufirst%3DD.%2BM.%26aulast%3DDavis%26aufirst%3DD.%2BJ.%26aulast%3DDevine%26aufirst%3DL.%2BA.%26aulast%3DEarly%26aufirst%3DT.%2BR.%26aulast%3DFeltell%26aufirst%3DR.%2BE.%26aulast%3DLewis%26aufirst%3DE.%2BJ.%26aulast%3DMcMenamin%26aufirst%3DR.%2BL.%26aulast%3DNavarro%26aufirst%3DE.%2BF.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DM.%2BA.%26aulast%3DO%25E2%2580%2599Reilly%26aufirst%3DM.%26aulast%3DReule%26aufirst%3DM.%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DSeavers%26aufirst%3DL.%2BC.%26aulast%3DSmith%26aufirst%3DD.%2BM.%26aulast%3DSquires%26aufirst%3DM.%2BS.%26aulast%3DTrewartha%26aufirst%3DG.%26aulast%3DWalker%26aufirst%3DM.%2BT.%26aulast%3DWoolford%26aufirst%3DA.%2BJ.%26atitle%3DIdentification%2520of%2520N-%25284-Piperidinyl%2529-4-%25282%252C6-Dichlorobenzoylamino%2529-1h-Pyrazole-3-Carboxamide%2520%2528At7519%2529%252C%2520a%2520Novel%2520Cyclin%2520Dependent%2520Kinase%2520Inhibitor%2520Using%2520Fragment-Based%2520X-Ray%2520Crystallography%2520and%2520Structure%2520Based%2520Drug%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D4986%26epage%3D4999%26doi%3D10.1021%2Fjm800382h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Deng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roessler, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brdar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faessler, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sales, Z. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mani, N. S.</span><span> </span><span class="NLM_article-title">Direct, Metal-Free Amination of Heterocyclic Amides/Ureas with Nh-Heterocycles and N-Substituted Anilines in Pocl<sub>3</sub></span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">76</span><span class="NLM_x">, </span> <span class="NLM_fpage">8262</span><span class="NLM_x">–</span> <span class="NLM_lpage">8269</span><span class="refDoi"> DOI: 10.1021/jo201425q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo201425q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFCgsrfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2011&pages=8262-8269&author=X.+Dengauthor=A.+Roesslerauthor=I.+Brdarauthor=R.+Faesslerauthor=J.+Wuauthor=Z.+S.+Salesauthor=N.+S.+Mani&title=Direct%2C+Metal-Free+Amination+of+Heterocyclic+Amides%2FUreas+with+Nh-Heterocycles+and+N-Substituted+Anilines+in+Pocl3&doi=10.1021%2Fjo201425q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Direct, metal-free amination of heterocyclic amides/ureas with NH-heterocycles and N-substituted anilines in POCl3</span></div><div class="casAuthors">Deng, Xiaohu; Roessler, Armin; Brdar, Ivana; Faessler, Roger; Wu, Jiejun; Sales, Zachary S.; Mani, Neelakandha S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8262-8269</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A POCl3-mediated, direct amination reaction of heterocyclic amides/ureas with NH-heterocycles or N-substituted anilines is described.  Compared to the existing methods, this operationally simple protocol provides unique reactivity and functional group compatibility because of the metal-free, acidic reaction conditions.  The yields are generally excellent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprHfZJEC5aPbVg90H21EOLACvtfcHk0lhwe4AXuYWG7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFCgsrfM&md5=2407dead7288be61402476dfc7303cf9</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjo201425q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo201425q%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DRoessler%26aufirst%3DA.%26aulast%3DBrdar%26aufirst%3DI.%26aulast%3DFaessler%26aufirst%3DR.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DSales%26aufirst%3DZ.%2BS.%26aulast%3DMani%26aufirst%3DN.%2BS.%26atitle%3DDirect%252C%2520Metal-Free%2520Amination%2520of%2520Heterocyclic%2520Amides%252FUreas%2520with%2520Nh-Heterocycles%2520and%2520N-Substituted%2520Anilines%2520in%2520Pocl3%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2011%26volume%3D76%26spage%3D8262%26epage%3D8269%26doi%3D10.1021%2Fjo201425q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Davies, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tunnah, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meijer, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marko, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eisenbrand, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Endicott, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noble, M. E.</span><span> </span><span class="NLM_article-title">Inhibitor Binding to Active and Inactive Cdk2: The Crystal Structure of Cdk2-Cyclin a/Indirubin-5-Sulphonate</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">389</span><span class="NLM_x">–</span> <span class="NLM_lpage">397</span><span class="refDoi"> DOI: 10.1016/S0969-2126(01)00598-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2FS0969-2126%2801%2900598-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=11377199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjslCisLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2001&pages=389-397&author=T.+G.+Daviesauthor=P.+Tunnahauthor=L.+Meijerauthor=D.+Markoauthor=G.+Eisenbrandauthor=J.+A.+Endicottauthor=M.+E.+Noble&title=Inhibitor+Binding+to+Active+and+Inactive+Cdk2%3A+The+Crystal+Structure+of+Cdk2-Cyclin+a%2FIndirubin-5-Sulphonate&doi=10.1016%2FS0969-2126%2801%2900598-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitor Binding to Active and Inactive CDK2 The Crystal Structure of CDK2-Cyclin A/Indirubin-5-Sulphonate</span></div><div class="casAuthors">Davies, T. G.; Tunnah, P.; Meijer, L.; Marko, D.; Eisenbrand, G.; Endicott, J. A.; Noble, M. E. M.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">389-397</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Background: Cyclin-dependent kinase 2 (CDK2) is an important target for structure-based design of antitumor agents.  Monomeric CDK2 is inactive.  Activation requires rearrangements to key structural elements of the enzyme's active site, which accompany cyclin binding and phosphorylation.  To assess the validity of using monomeric CDK2 as a model for the active kinase in structure-based drug design, we have solved the structure of the inhibitor indirubin-5-sulfonate (E226) complexed with phospho-CDK2-cyclin A and compared it with the structure of E226 bound to inactive, monomeric CDK2.  Results: Activation of monomeric CDK2 leads to a rotation of its N-terminal domain relative to the C-terminal lobe.  The accompanying change in position of E226 follows that of the N-terminal domain, and its interactions with residues forming part of the adenine binding pocket are conserved.  The environment of the ATP-ribose site, not explored by E226, is significantly different in the binary complex compared to the monomeric complex due to movement of the glycine loop.  Conformational changes also result in subtle differences in hydrogen bonding and electrostatic interactions between E226's sulfonate and CDK2's phosphate binding site.  Affinities calcd. by LUDI for the interaction of E226 with active or inactive CDK2 differ by a factor of approx. ten.  Conclusions: The accuracy of monomeric CDK2 as an inhibitor design template is restricted to the adenine binding site.  The general flexibility obsd. for the glycine loop and subtle changes to the phosphate binding site suggest a need to study interactions between inhibitors and active CDK2 in structure-based drug design programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptJwFSSxY29rVg90H21EOLACvtfcHk0ljDHnr88keJIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjslCisLw%253D&md5=5e8e5c5500ab6809cb4db066f6477901</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2801%2900598-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252801%252900598-6%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DT.%2BG.%26aulast%3DTunnah%26aufirst%3DP.%26aulast%3DMeijer%26aufirst%3DL.%26aulast%3DMarko%26aufirst%3DD.%26aulast%3DEisenbrand%26aufirst%3DG.%26aulast%3DEndicott%26aufirst%3DJ.%2BA.%26aulast%3DNoble%26aufirst%3DM.%2BE.%26atitle%3DInhibitor%2520Binding%2520to%2520Active%2520and%2520Inactive%2520Cdk2%253A%2520The%2520Crystal%2520Structure%2520of%2520Cdk2-Cyclin%2520a%252FIndirubin-5-Sulphonate%26jtitle%3DStructure%26date%3D2001%26volume%3D9%26spage%3D389%26epage%3D397%26doi%3D10.1016%2FS0969-2126%2801%2900598-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Hoessel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leclerc, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Endicott, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nobel, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawrie, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tunnah, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leost, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damiens, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marie, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marko, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niederberger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eisenbrand, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meijer, L.</span><span> </span><span class="NLM_article-title">Indirubin, the Active Constituent of a Chinese Antileukaemia Medicine, Inhibits Cyclin-Dependent Kinases</span> <span class="citation_source-journal">Nat. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">60</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span><span class="refDoi"> DOI: 10.1038/9035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1038%2F9035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10559866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADyaK1MXksVKitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=1999&pages=60-67&author=R.+Hoesselauthor=S.+Leclercauthor=J.+A.+Endicottauthor=M.+E.+Nobelauthor=A.+Lawrieauthor=P.+Tunnahauthor=M.+Leostauthor=E.+Damiensauthor=D.+Marieauthor=D.+Markoauthor=E.+Niederbergerauthor=W.+Tangauthor=G.+Eisenbrandauthor=L.+Meijer&title=Indirubin%2C+the+Active+Constituent+of+a+Chinese+Antileukaemia+Medicine%2C+Inhibits+Cyclin-Dependent+Kinases&doi=10.1038%2F9035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Indirubin, the active constituent of a Chinese antileukemia medicine, inhibits cyclin-dependent kinases</span></div><div class="casAuthors">Hoessel, Ralph; Leclerc, Sophie; Endicott, Jane A.; Nobel, Martin E. M.; Lawrie, Alison; Tunnah, Paul; Leost, Maryse; Damiens, Eve; Marie, Dominique; Marko, Doris; Niederberger, Ellen; Tang, Weici; Eisenbrand, Gerhard; Meijer, Laurent</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">60-67</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines Ltd</span>)
        </div><div class="casAbstract">Indirubin is the active ingredient of Danggui Longhui Wan, a mixt. of plants that is used in traditional Chinese medicine to treat chronic diseases.  Here indirubin and its analogs were identified as potent inhibitors of cyclin-dependent kinases (CDKs).  The crystal structure of CDK2 in complex with indirubin derivs. shows that indirubin interacts with the kinase's ATP-binding site through van der Waals interactions and three hydrogen bonds.  Indirubin-3'-monoxime inhibits the proliferation of a large range of cells, mainly through arresting the cells in the G2/M phase of the cell cycle.  These results have implications for therapeutic optimization of indigoids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpb5I1r-KcwErVg90H21EOLACvtfcHk0ljDHnr88keJIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXksVKitrk%253D&md5=bbbb09fbb3259de2e28fb58a5f507a7d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2F9035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F9035%26sid%3Dliteratum%253Aachs%26aulast%3DHoessel%26aufirst%3DR.%26aulast%3DLeclerc%26aufirst%3DS.%26aulast%3DEndicott%26aufirst%3DJ.%2BA.%26aulast%3DNobel%26aufirst%3DM.%2BE.%26aulast%3DLawrie%26aufirst%3DA.%26aulast%3DTunnah%26aufirst%3DP.%26aulast%3DLeost%26aufirst%3DM.%26aulast%3DDamiens%26aufirst%3DE.%26aulast%3DMarie%26aufirst%3DD.%26aulast%3DMarko%26aufirst%3DD.%26aulast%3DNiederberger%26aufirst%3DE.%26aulast%3DTang%26aufirst%3DW.%26aulast%3DEisenbrand%26aufirst%3DG.%26aulast%3DMeijer%26aufirst%3DL.%26atitle%3DIndirubin%252C%2520the%2520Active%2520Constituent%2520of%2520a%2520Chinese%2520Antileukaemia%2520Medicine%252C%2520Inhibits%2520Cyclin-Dependent%2520Kinases%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D1999%26volume%3D1%26spage%3D60%26epage%3D67%26doi%3D10.1038%2F9035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Ottana, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carotti, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maccari, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landini, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiricosta, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caciagli, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vigorita, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mini, E.</span><span> </span><span class="NLM_article-title">In Vitro Antiproliferative Activity against Human Colon Cancer Cell Lines of Representative 4-Thiazolidinones. Part I</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">3930</span><span class="NLM_x">–</span> <span class="NLM_lpage">3393</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2005.05.093</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2Fj.bmcl.2005.05.093" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2005&pages=3930-3393&author=R.+Ottanaauthor=S.+Carottiauthor=R.+Maccariauthor=I.+Landiniauthor=G.+Chiricostaauthor=B.+Caciagliauthor=M.+G.+Vigoritaauthor=E.+Mini&title=In+Vitro+Antiproliferative+Activity+against+Human+Colon+Cancer+Cell+Lines+of+Representative+4-Thiazolidinones.+Part+I&doi=10.1016%2Fj.bmcl.2005.05.093"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2005.05.093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2005.05.093%26sid%3Dliteratum%253Aachs%26aulast%3DOttana%26aufirst%3DR.%26aulast%3DCarotti%26aufirst%3DS.%26aulast%3DMaccari%26aufirst%3DR.%26aulast%3DLandini%26aufirst%3DI.%26aulast%3DChiricosta%26aufirst%3DG.%26aulast%3DCaciagli%26aufirst%3DB.%26aulast%3DVigorita%26aufirst%3DM.%2BG.%26aulast%3DMini%26aufirst%3DE.%26atitle%3DIn%2520Vitro%2520Antiproliferative%2520Activity%2520against%2520Human%2520Colon%2520Cancer%2520Cell%2520Lines%2520of%2520Representative%25204-Thiazolidinones.%2520Part%2520I%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2005%26volume%3D15%26spage%3D3930%26epage%3D3393%26doi%3D10.1016%2Fj.bmcl.2005.05.093" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Vicini, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geronikaki, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anastasia, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Incerti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zani, F.</span><span> </span><span class="NLM_article-title">Synthesis and Antimicrobial Activity of Novel 2-Thiazolylimino-5-Arylidene-4-Thiazolidinones</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">3859</span><span class="NLM_x">–</span> <span class="NLM_lpage">3864</span><span class="refDoi"> DOI: 10.1016/j.bmc.2006.01.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2Fj.bmc.2006.01.043" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2006&pages=3859-3864&author=P.+Viciniauthor=A.+Geronikakiauthor=K.+Anastasiaauthor=M.+Incertiauthor=F.+Zani&title=Synthesis+and+Antimicrobial+Activity+of+Novel+2-Thiazolylimino-5-Arylidene-4-Thiazolidinones&doi=10.1016%2Fj.bmc.2006.01.043"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2006.01.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2006.01.043%26sid%3Dliteratum%253Aachs%26aulast%3DVicini%26aufirst%3DP.%26aulast%3DGeronikaki%26aufirst%3DA.%26aulast%3DAnastasia%26aufirst%3DK.%26aulast%3DIncerti%26aufirst%3DM.%26aulast%3DZani%26aufirst%3DF.%26atitle%3DSynthesis%2520and%2520Antimicrobial%2520Activity%2520of%2520Novel%25202-Thiazolylimino-5-Arylidene-4-Thiazolidinones%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2006%26volume%3D14%26spage%3D3859%26epage%3D3864%26doi%3D10.1016%2Fj.bmc.2006.01.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">El-Barbary, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khodair, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, C.</span><span> </span><span class="NLM_article-title">Synthesis and Evaluation of Antiviral Activity of 2′-Deoxyuridines with 5-Methylene-2-Thiohydantoin Substituents in the 5-Position</span> <span class="citation_source-journal">Monatsh. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">593</span><span class="NLM_x">–</span> <span class="NLM_lpage">598</span><span class="refDoi"> DOI: 10.1007/BF00811853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1007%2FBF00811853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADyaK2cXlsFyjtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=1994&pages=593-598&author=A.+A.+El-Barbaryauthor=A.+I.+Khodairauthor=E.+B.+Pedersenauthor=C.+Nielsen&title=Synthesis+and+Evaluation+of+Antiviral+Activity+of+2%E2%80%B2-Deoxyuridines+with+5-Methylene-2-Thiohydantoin+Substituents+in+the+5-Position&doi=10.1007%2FBF00811853"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and evaluation of antiviral activity of 2'-deoxyuridines with 5-methylene-2-thiohydantoin substituents in the 5-position</span></div><div class="casAuthors">El-Barbary, A. A.; Khodair, A. I.; Pedersen, E. B.; Nielsen, C.</div><div class="citationInfo"><span class="NLM_cas:title">Monatshefte fuer Chemie</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">593-8</span>CODEN:
                <span class="NLM_cas:coden">MOCMB7</span>;
        ISSN:<span class="NLM_cas:issn">0026-9247</span>.
    </div><div class="casAbstract">Title deoxyribonucleosides, e.g. I (R = H, Me), were prepd. via glycosidation of thiohydantoins.  Antiviral and antitumor activities of I are reported.  None of I showed any significant inhibitory activities against HIV-1.  Only the nucleobase of I (R = Me) and α-I (R = Me) showed cytotoxicity against MT-4 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopDzJOcymgArVg90H21EOLACvtfcHk0lh-cZ6xx0qRhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXlsFyjtbg%253D&md5=b4c3f91b0cf3131d00cab25f354b2ced</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1007%2FBF00811853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00811853%26sid%3Dliteratum%253Aachs%26aulast%3DEl-Barbary%26aufirst%3DA.%2BA.%26aulast%3DKhodair%26aufirst%3DA.%2BI.%26aulast%3DPedersen%26aufirst%3DE.%2BB.%26aulast%3DNielsen%26aufirst%3DC.%26atitle%3DSynthesis%2520and%2520Evaluation%2520of%2520Antiviral%2520Activity%2520of%25202%25E2%2580%25B2-Deoxyuridines%2520with%25205-Methylene-2-Thiohydantoin%2520Substituents%2520in%2520the%25205-Position%26jtitle%3DMonatsh.%2520Chem.%26date%3D1994%26volume%3D125%26spage%3D593%26epage%3D598%26doi%3D10.1007%2FBF00811853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Bhandari, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bothara, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patil, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chitre, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarkate, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gore, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dangre, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khachane, C. V.</span><span> </span><span class="NLM_article-title">Design, Synthesis and Pharmacological Screening of Novel Antihypertensive Agents Using Hybrid Approach</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">390</span><span class="NLM_x">–</span> <span class="NLM_lpage">400</span><span class="refDoi"> DOI: 10.1016/j.bmc.2008.10.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2Fj.bmc.2008.10.032" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=390-400&author=S.+V.+Bhandariauthor=K.+G.+Botharaauthor=A.+A.+Patilauthor=T.+S.+Chitreauthor=A.+P.+Sarkateauthor=S.+T.+Goreauthor=S.+C.+Dangreauthor=C.+V.+Khachane&title=Design%2C+Synthesis+and+Pharmacological+Screening+of+Novel+Antihypertensive+Agents+Using+Hybrid+Approach&doi=10.1016%2Fj.bmc.2008.10.032"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2008.10.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2008.10.032%26sid%3Dliteratum%253Aachs%26aulast%3DBhandari%26aufirst%3DS.%2BV.%26aulast%3DBothara%26aufirst%3DK.%2BG.%26aulast%3DPatil%26aufirst%3DA.%2BA.%26aulast%3DChitre%26aufirst%3DT.%2BS.%26aulast%3DSarkate%26aufirst%3DA.%2BP.%26aulast%3DGore%26aufirst%3DS.%2BT.%26aulast%3DDangre%26aufirst%3DS.%2BC.%26aulast%3DKhachane%26aufirst%3DC.%2BV.%26atitle%3DDesign%252C%2520Synthesis%2520and%2520Pharmacological%2520Screening%2520of%2520Novel%2520Antihypertensive%2520Agents%2520Using%2520Hybrid%2520Approach%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2009%26volume%3D17%26spage%3D390%26epage%3D400%26doi%3D10.1016%2Fj.bmc.2008.10.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Richardson, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nunns, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williamson, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parratt, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dokurno, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howes, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borgognoni, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drysdale, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finch, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kierstan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lentzen, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmonite, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surgenor, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Torrance, C. J.</span><span> </span><span class="NLM_article-title">Discovery of a Potent Cdk2 Inhibitor with a Novel Binding Mode, Using Virtual Screening and Initial, Structure-Guided Lead Scoping</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">3880</span><span class="NLM_x">–</span> <span class="NLM_lpage">3885</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2007.04.110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2Fj.bmcl.2007.04.110" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=3880-3885&author=C.+M.+Richardsonauthor=C.+L.+Nunnsauthor=D.+S.+Williamsonauthor=M.+J.+Parrattauthor=P.+Dokurnoauthor=R.+Howesauthor=J.+Borgognoniauthor=M.+J.+Drysdaleauthor=H.+Finchauthor=R.+E.+Hubbardauthor=P.+S.+Jacksonauthor=P.+Kierstanauthor=G.+Lentzenauthor=J.+D.+Mooreauthor=J.+B.+Murrayauthor=H.+Simmoniteauthor=A.+E.+Surgenorauthor=C.+J.+Torrance&title=Discovery+of+a+Potent+Cdk2+Inhibitor+with+a+Novel+Binding+Mode%2C+Using+Virtual+Screening+and+Initial%2C+Structure-Guided+Lead+Scoping&doi=10.1016%2Fj.bmcl.2007.04.110"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.04.110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.04.110%26sid%3Dliteratum%253Aachs%26aulast%3DRichardson%26aufirst%3DC.%2BM.%26aulast%3DNunns%26aufirst%3DC.%2BL.%26aulast%3DWilliamson%26aufirst%3DD.%2BS.%26aulast%3DParratt%26aufirst%3DM.%2BJ.%26aulast%3DDokurno%26aufirst%3DP.%26aulast%3DHowes%26aufirst%3DR.%26aulast%3DBorgognoni%26aufirst%3DJ.%26aulast%3DDrysdale%26aufirst%3DM.%2BJ.%26aulast%3DFinch%26aufirst%3DH.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DJackson%26aufirst%3DP.%2BS.%26aulast%3DKierstan%26aufirst%3DP.%26aulast%3DLentzen%26aufirst%3DG.%26aulast%3DMoore%26aufirst%3DJ.%2BD.%26aulast%3DMurray%26aufirst%3DJ.%2BB.%26aulast%3DSimmonite%26aufirst%3DH.%26aulast%3DSurgenor%26aufirst%3DA.%2BE.%26aulast%3DTorrance%26aufirst%3DC.%2BJ.%26atitle%3DDiscovery%2520of%2520a%2520Potent%2520Cdk2%2520Inhibitor%2520with%2520a%2520Novel%2520Binding%2520Mode%252C%2520Using%2520Virtual%2520Screening%2520and%2520Initial%252C%2520Structure-Guided%2520Lead%2520Scoping%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D3880%26epage%3D3885%26doi%3D10.1016%2Fj.bmcl.2007.04.110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Dakin, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Block, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Code, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dowling, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferguson, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hird, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keeton, E. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamb, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyne, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, X.</span><span> </span><span class="NLM_article-title">Discovery of Novel Benzylidene-1,3-Thiazolidine-2,4-Diones as Potent and Selective Inhibitors of the Pim-1, Pim-2, and Pim-3 Protein Kinases</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">4599</span><span class="NLM_x">–</span> <span class="NLM_lpage">4604</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2012.05.098</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2Fj.bmcl.2012.05.098" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=22727640" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC38Xpt1Shtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=4599-4604&author=L.+A.+Dakinauthor=M.+H.+Blockauthor=H.+Chenauthor=E.+Codeauthor=J.+E.+Dowlingauthor=X.+Fengauthor=A.+D.+Fergusonauthor=I.+Greenauthor=A.+W.+Hirdauthor=T.+Howardauthor=E.+K.+Keetonauthor=M.+L.+Lambauthor=P.+D.+Lyneauthor=H.+Pollardauthor=J.+Readauthor=A.+J.+Wuauthor=T.+Zhangauthor=X.+Zheng&title=Discovery+of+Novel+Benzylidene-1%2C3-Thiazolidine-2%2C4-Diones+as+Potent+and+Selective+Inhibitors+of+the+Pim-1%2C+Pim-2%2C+and+Pim-3+Protein+Kinases&doi=10.1016%2Fj.bmcl.2012.05.098"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel benzylidene-1,3-thiazolidine-2,4-diones as potent and selective inhibitors of the PIM-1, PIM-2, and PIM-3 protein kinases</span></div><div class="casAuthors">Dakin, Les A.; Block, Michael H.; Chen, Huawei; Code, Erin; Dowling, James E.; Feng, Xiaomei; Ferguson, Andrew D.; Green, Isabelle; Hird, Alexander W.; Howard, Tina; Keeton, Erika K.; Lamb, Michelle L.; Lyne, Paul D.; Pollard, Hannah; Read, Jon; Wu, Allan J.; Zhang, Tao; Zheng, Xiaolan</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4599-4604</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Novel substituted benzylidene-1,3-thiazolidine-2,4-diones (TZDs) have been identified as potent and highly selective inhibitors of the PIM kinases.  The synthesis and SAR of these compds. are described, along with x-ray crystallog. of deriv. I, anti-proliferative, and selectivity data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLer-JRufp_rVg90H21EOLACvtfcHk0lh-cZ6xx0qRhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xpt1Shtb4%253D&md5=cd1570938d48864bb739d2ccc37888f2</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.05.098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.05.098%26sid%3Dliteratum%253Aachs%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DBlock%26aufirst%3DM.%2BH.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DCode%26aufirst%3DE.%26aulast%3DDowling%26aufirst%3DJ.%2BE.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DGreen%26aufirst%3DI.%26aulast%3DHird%26aufirst%3DA.%2BW.%26aulast%3DHoward%26aufirst%3DT.%26aulast%3DKeeton%26aufirst%3DE.%2BK.%26aulast%3DLamb%26aufirst%3DM.%2BL.%26aulast%3DLyne%26aufirst%3DP.%2BD.%26aulast%3DPollard%26aufirst%3DH.%26aulast%3DRead%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DA.%2BJ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DZheng%26aufirst%3DX.%26atitle%3DDiscovery%2520of%2520Novel%2520Benzylidene-1%252C3-Thiazolidine-2%252C4-Diones%2520as%2520Potent%2520and%2520Selective%2520Inhibitors%2520of%2520the%2520Pim-1%252C%2520Pim-2%252C%2520and%2520Pim-3%2520Protein%2520Kinases%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D4599%26epage%3D4604%26doi%3D10.1016%2Fj.bmcl.2012.05.098" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Zhou, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ekins, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swaan, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, B.</span><span> </span><span class="NLM_article-title">Design, Synthesis, Cytoselective Toxicity, Structure-Activity Relationships, and Pharmacophore of Thiazolidinone Derivatives Targeting Drug-Resistant Lung Cancer Cells</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1242</span><span class="NLM_x">–</span> <span class="NLM_lpage">1251</span><span class="refDoi"> DOI: 10.1021/jm7012024</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm7012024" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhslWjurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=1242-1251&author=H.+Zhouauthor=S.+Wuauthor=S.+Zhaiauthor=A.+Liuauthor=Y.+Sunauthor=R.+Liauthor=Y.+Zhangauthor=S.+Ekinsauthor=P.+W.+Swaanauthor=B.+Fangauthor=B.+Zhangauthor=B.+Yan&title=Design%2C+Synthesis%2C+Cytoselective+Toxicity%2C+Structure-Activity+Relationships%2C+and+Pharmacophore+of+Thiazolidinone+Derivatives+Targeting+Drug-Resistant+Lung+Cancer+Cells&doi=10.1021%2Fjm7012024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, Cytoselective Toxicity, Structure-Activity Relationships, and Pharmacophore of Thiazolidinone Derivatives Targeting Drug-Resistant Lung Cancer Cells</span></div><div class="casAuthors">Zhou, Hongyu; Wu, Shuhong; Zhai, Shumei; Liu, Aifeng; Sun, Ying; Li, Rongshi; Zhang, Ying; Ekins, Sean; Swaan, Peter W.; Fang, Bingliang; Zhang, Bin; Yan, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1242-1251</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A library of 372 thiazolidinones I [R1 = H, Me2CH, HOCH2CH2, 2-pyridylmethyl, etc.; R2 = H, 2-Me, 3-F, 4-EtO2C, etc.; R3 = H, 4-CN, 2-HO, 2,4,5-(MeO)3, etc.] has been synthesized in three steps in moderate to excellent yields.  Of these thiazolidinones, ten cytoselective compds. have been identified by applying iterative library approaches.  These compds. selectively killed both non-small cell lung cancer cell line H460 and its paclitaxel-resistant variant H460taxR at an IC50 between 0.21 and 2.93 μM while showing much less toxicity to normal human fibroblasts at concns. up to 195 μM.  Structure-activity relationship studies revealed that (1) the nitrogen atom on the 4-thiazolidinone ring cannot be substituted, (2) several substitutions on arylimine arom. ring are tolerated at various positions, and (3) the substitution on benzylidene arom. ring is restricted to R3 = 4-NMe2.  A pharmacophore derived from active mols. suggested that two hydrogen bond acceptors and three hydrophobic regions were common features.  Activities against P-gp-overexpressing and paclitaxel-resistant cell line H460taxR and modeling using a previously validated P-gp substrate pharmacophore suggested that active compds. were not likely P-gp substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2qhZeAPQHM7Vg90H21EOLACvtfcHk0lhmXtzOUcVJ-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhslWjurY%253D&md5=61a08d341d76d81978dc04a25e4b3fbe</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjm7012024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm7012024%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DS.%26aulast%3DZhai%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DA.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DEkins%26aufirst%3DS.%26aulast%3DSwaan%26aufirst%3DP.%2BW.%26aulast%3DFang%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DYan%26aufirst%3DB.%26atitle%3DDesign%252C%2520Synthesis%252C%2520Cytoselective%2520Toxicity%252C%2520Structure-Activity%2520Relationships%252C%2520and%2520Pharmacophore%2520of%2520Thiazolidinone%2520Derivatives%2520Targeting%2520Drug-Resistant%2520Lung%2520Cancer%2520Cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D1242%26epage%3D1251%26doi%3D10.1021%2Fjm7012024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Kornev, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haste, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eyck, L. F.</span><span> </span><span class="NLM_article-title">Surface Comparison of Active and Inactive Protein Kinases Identifies a Conserved Activation Mechanism</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">17783</span><span class="NLM_x">–</span> <span class="NLM_lpage">17788</span><span class="refDoi"> DOI: 10.1073/pnas.0607656103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1073%2Fpnas.0607656103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=17095602" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1KnurfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=17783-17788&author=A.+P.+Kornevauthor=N.+M.+Hasteauthor=S.+S.+Taylorauthor=L.+F.+Eyck&title=Surface+Comparison+of+Active+and+Inactive+Protein+Kinases+Identifies+a+Conserved+Activation+Mechanism&doi=10.1073%2Fpnas.0607656103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Surface comparison of active and inactive protein kinases identifies a conserved activation mechanism</span></div><div class="casAuthors">Kornev, Alexandr P.; Haste, Nina M.; Taylor, Susan S.; Ten Eyck, Lynn F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">17783-17788</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The surface comparison of different serine-threonine and tyrosine kinases reveals a set of 30 residues whose spatial positions are highly conserved.  The comparison between active and inactive conformations identified the residues whose positions are the most sensitive to activation.  Based on these results, we propose a model of protein kinase activation.  This model explains how the presence of a phosphate group in the activation loop dets. the position of the catalytically important aspartate in the Asp-Phe-Gly (DFG) motif.  According to the model, the most important feature of the activation is a "spine" formation that is dynamically assembled in all active kinases.  The spine is comprised of four hydrophobic residues that we detected in a set of 23 eukaryotic and prokaryotic kinases.  It spans the mol. and plays a coordinating role in activated kinases.  The spine is disordered in the inactive kinases and can explain how stabilization of the whole mol. is achieved upon phosphorylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwv4ByA5owH7Vg90H21EOLACvtfcHk0lhmXtzOUcVJ-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1KnurfJ&md5=f474acffd8b17d8352ac9f6fc9bbbf8f</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0607656103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0607656103%26sid%3Dliteratum%253Aachs%26aulast%3DKornev%26aufirst%3DA.%2BP.%26aulast%3DHaste%26aufirst%3DN.%2BM.%26aulast%3DTaylor%26aufirst%3DS.%2BS.%26aulast%3DEyck%26aufirst%3DL.%2BF.%26atitle%3DSurface%2520Comparison%2520of%2520Active%2520and%2520Inactive%2520Protein%2520Kinases%2520Identifies%2520a%2520Conserved%2520Activation%2520Mechanism%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2006%26volume%3D103%26spage%3D17783%26epage%3D17788%26doi%3D10.1073%2Fpnas.0607656103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Kornev, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ten Eyck, L. F.</span><span> </span><span class="NLM_article-title">A Helix Scaffold for the Assembly of Active Protein Kinases</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">14377</span><span class="NLM_x">–</span> <span class="NLM_lpage">14382</span><span class="refDoi"> DOI: 10.1073/pnas.0807988105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1073%2Fpnas.0807988105" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=14377-14382&author=A.+P.+Kornevauthor=S.+S.+Taylorauthor=L.+F.+Ten+Eyck&title=A+Helix+Scaffold+for+the+Assembly+of+Active+Protein+Kinases&doi=10.1073%2Fpnas.0807988105"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0807988105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0807988105%26sid%3Dliteratum%253Aachs%26aulast%3DKornev%26aufirst%3DA.%2BP.%26aulast%3DTaylor%26aufirst%3DS.%2BS.%26aulast%3DTen%2BEyck%26aufirst%3DL.%2BF.%26atitle%3DA%2520Helix%2520Scaffold%2520for%2520the%2520Assembly%2520of%2520Active%2520Protein%2520Kinases%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D14377%26epage%3D14382%26doi%3D10.1073%2Fpnas.0807988105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Jautelat, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brumby, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schafer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Briem, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eisenbrand, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwahn, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lucking, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prien, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siemeister, G.</span><span> </span><span class="NLM_article-title">From the Insoluble Dye Indirubin Towards Highly Active, Soluble Cdk2-Inhibitors</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">531</span><span class="NLM_x">–</span> <span class="NLM_lpage">540</span><span class="refDoi"> DOI: 10.1002/cbic.200400108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1002%2Fcbic.200400108" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=531-540&author=R.+Jautelatauthor=T.+Brumbyauthor=M.+Schaferauthor=H.+Briemauthor=G.+Eisenbrandauthor=S.+Schwahnauthor=M.+Krugerauthor=U.+Luckingauthor=O.+Prienauthor=G.+Siemeister&title=From+the+Insoluble+Dye+Indirubin+Towards+Highly+Active%2C+Soluble+Cdk2-Inhibitors&doi=10.1002%2Fcbic.200400108"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1002%2Fcbic.200400108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.200400108%26sid%3Dliteratum%253Aachs%26aulast%3DJautelat%26aufirst%3DR.%26aulast%3DBrumby%26aufirst%3DT.%26aulast%3DSchafer%26aufirst%3DM.%26aulast%3DBriem%26aufirst%3DH.%26aulast%3DEisenbrand%26aufirst%3DG.%26aulast%3DSchwahn%26aufirst%3DS.%26aulast%3DKruger%26aufirst%3DM.%26aulast%3DLucking%26aufirst%3DU.%26aulast%3DPrien%26aufirst%3DO.%26aulast%3DSiemeister%26aufirst%3DG.%26atitle%3DFrom%2520the%2520Insoluble%2520Dye%2520Indirubin%2520Towards%2520Highly%2520Active%252C%2520Soluble%2520Cdk2-Inhibitors%26jtitle%3DChemBioChem%26date%3D2005%26volume%3D6%26spage%3D531%26epage%3D540%26doi%3D10.1002%2Fcbic.200400108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Yin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huffmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNamara, J. M.</span><span> </span><span class="NLM_article-title">Pd-Catalyzed N-Arylation of Heteroarylamines</span> <span class="citation_source-journal">Org. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">3481</span><span class="refDoi"> DOI: 10.1021/ol0265923</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol0265923" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD38Xms1Omur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2002&pages=3481&author=J.+Yinauthor=M.+M.+Zhaoauthor=M.+Huffmannauthor=J.+M.+McNamara&title=Pd-Catalyzed+N-Arylation+of+Heteroarylamines&doi=10.1021%2Fol0265923"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Pd-Catalyzed N-Arylation of Heteroarylamines</span></div><div class="casAuthors">Yin, Jingjun; Zhao, Matthew M.; Huffman, Mark A.; McNamara, James M.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3481-3484</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The palladium-catalyzed N-(hetero)arylation of a no. of heteroarylamines including 2-aminopyridines, 2-aminothiazoles, and their analogs has been realized using Xantphos as the ligand.  Weak bases such as Cs2CO3, Na2CO3, and K3PO4 were used in most cases to allow for the introduction of functional groups.  Choice of the base and solvent was crit. for the success of these reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpto0A1OmazwbVg90H21EOLACvtfcHk0lik9_IJiopffQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xms1Omur8%253D&md5=95f39b69250ce41d7c97d66ecc7f1ae8</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fol0265923&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol0265923%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DM.%2BM.%26aulast%3DHuffmann%26aufirst%3DM.%26aulast%3DMcNamara%26aufirst%3DJ.%2BM.%26atitle%3DPd-Catalyzed%2520N-Arylation%2520of%2520Heteroarylamines%26jtitle%3DOrg.%2520Lett.%26date%3D2002%26volume%3D4%26spage%3D3481%26doi%3D10.1021%2Fol0265923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Honda, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nigg, E. A.</span><span> </span><span class="NLM_article-title">Exploring the Functional Interactions between Aurora B, Incenp, and Survivin in Mitosis</span> <span class="citation_source-journal">Mol. Biol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">3325</span><span class="NLM_x">–</span> <span class="NLM_lpage">3341</span><span class="refDoi"> DOI: 10.1091/mbc.E02-11-0769</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1091%2Fmbc.E02-11-0769" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=12925766" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvFWks7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2003&pages=3325-3341&author=R.+Hondaauthor=R.+Kornerauthor=E.+A.+Nigg&title=Exploring+the+Functional+Interactions+between+Aurora+B%2C+Incenp%2C+and+Survivin+in+Mitosis&doi=10.1091%2Fmbc.E02-11-0769"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring the functional interactions between Aurora B, INCENP, and survivin in mitosis</span></div><div class="casAuthors">Honda, Reiko; Koerner, Roman; Nigg, Erich A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biology of the Cell</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3325-3341</span>CODEN:
                <span class="NLM_cas:coden">MBCEEV</span>;
        ISSN:<span class="NLM_cas:issn">1059-1524</span>.
    
            (<span class="NLM_cas:orgname">American Society for Cell Biology</span>)
        </div><div class="casAbstract">The function of the Aurora B kinase at centromeres and the central spindle is crucial for chromosome segregation and cytokinesis, resp.  Herein, we have investigated the regulation of human Aurora B by its complex partners inner centromere protein (INCENP) and survivin.  We found that overexpression of a catalytically inactive, dominant-neg. mutant of Aurora B impaired the localization of the entire Aurora B/INCENP/survivin complex to centromeres and the central spindle and severely disturbed mitotic progression.  Similar results were also obsd. after depletion, by RNA interference, of either Aurora B, INCENP, or survivin.  These data suggest that Aurora B kinase activity and the formation of the Aurora B/INCENP/survivin complex both contribute to its proper localization.  Using recombinant proteins, we found that Aurora B kinase activity was stimulated by INCENP and that the C-terminal region of INCENP was sufficient for activation.  Under identical assay conditions, survivin did not detectably influence kinase activity.  Human INCENP was a substrate of Aurora B and mass spectrometry identified three consecutive residues (threonine 893, serine 894, and serine 895) contg. at least two phosphorylation sites.  A nonphosphorylatable mutant (TSS893-895AAA) was a poor activator of Aurora B, demonstrating that INCENP phosphorylation is important for kinase activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHqQu6K_cbk7Vg90H21EOLACvtfcHk0lik9_IJiopffQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvFWks7w%253D&md5=8cb2fcfe367500c9f411f1ad621f7aa4</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1091%2Fmbc.E02-11-0769&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1091%252Fmbc.E02-11-0769%26sid%3Dliteratum%253Aachs%26aulast%3DHonda%26aufirst%3DR.%26aulast%3DKorner%26aufirst%3DR.%26aulast%3DNigg%26aufirst%3DE.%2BA.%26atitle%3DExploring%2520the%2520Functional%2520Interactions%2520between%2520Aurora%2520B%252C%2520Incenp%252C%2520and%2520Survivin%2520in%2520Mitosis%26jtitle%3DMol.%2520Biol.%2520Cell%26date%3D2003%26volume%3D14%26spage%3D3325%26epage%3D3341%26doi%3D10.1091%2Fmbc.E02-11-0769" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Fabian, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biggs, W. H.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azimioara, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benedetti, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galvin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerlach, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grotzfeld, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lelias, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehta, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velasco, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span> </span><span class="NLM_article-title">A Small Molecule-Kinase Interaction Map for Clinical Kinase Inhibitors</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span><span class="refDoi"> DOI: 10.1038/nbt1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.+M.+Leliasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+Small+Molecule-Kinase+Interaction+Map+for+Clinical+Kinase+Inhibitors&doi=10.1038%2Fnbt1068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0lik9_IJiopffQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DLelias%26aufirst%3DJ.%2BM.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520Small%2520Molecule-Kinase%2520Interaction%2520Map%2520for%2520Clinical%2520Kinase%2520Inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336%26doi%3D10.1038%2Fnbt1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Karaman, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallant, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faraoni, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrison, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pritchard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">A Quantitative Analysis of Kinase Inhibitor Selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span><span class="refDoi"> DOI: 10.1038/nbt1358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1038%2Fnbt1358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=18183025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=127-132&author=M.+W.+Karamanauthor=S.+Herrgardauthor=D.+K.+Treiberauthor=P.+Gallantauthor=C.+E.+Atteridgeauthor=B.+T.+Campbellauthor=K.+W.+Chanauthor=P.+Ciceriauthor=M.+I.+Davisauthor=P.+T.+Edeenauthor=R.+Faraoniauthor=M.+Floydauthor=J.+P.+Huntauthor=D.+J.+Lockhartauthor=Z.+V.+Milanovauthor=M.+J.+Morrisonauthor=G.+Pallaresauthor=H.+K.+Patelauthor=S.+Pritchardauthor=L.+M.+Wodickaauthor=P.+P.+Zarrinkar&title=A+Quantitative+Analysis+of+Kinase+Inhibitor+Selectivity&doi=10.1038%2Fnbt1358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Karaman, Mazen W.; Herrgard, Sanna; Treiber, Daniel K.; Gallant, Paul; Atteridge, Corey E.; Campbell, Brian T.; Chan, Katrina W.; Ciceri, Pietro; Davis, Mindy I.; Edeen, Philip T.; Faraoni, Raffaella; Floyd, Mark; Hunt, Jeremy P.; Lockhart, Daniel J.; Milanov, Zdravko V.; Morrison, Michael J.; Pallares, Gabriel; Patel, Hitesh K.; Pritchard, Stephanie; Wodicka, Lisa M.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-132</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors are a new class of therapeutics with a propensity to inhibit multiple targets.  The biol. consequences of multikinase activity are poorly defined, and an important step toward understanding the relationship between selectivity, efficacy and safety is the exploration of how inhibitors interact with the human kinome.  The authors present interaction maps for 38 kinase inhibitors across a panel of 317 kinases representing >50% of the predicted human protein kinome.  The data constitute the most comprehensive study of kinase inhibitor selectivity to date and reveal a wide diversity of interaction patterns.  To enable a global anal. of the results, the authors introduce the concept of a selectivity score as a general tool to quantify and differentiate the obsd. interaction patterns.  The authors further investigate the impact of panel size and find that small assay panels do not provide a robust measure of selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiArJSq2_lPrVg90H21EOLACvtfcHk0lguvSVAuJaNJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D&md5=346265d412853ced636ad4128ed8a76f</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1038%2Fnbt1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1358%26sid%3Dliteratum%253Aachs%26aulast%3DKaraman%26aufirst%3DM.%2BW.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DCampbell%26aufirst%3DB.%2BT.%26aulast%3DChan%26aufirst%3DK.%2BW.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFaraoni%26aufirst%3DR.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DMorrison%26aufirst%3DM.%2BJ.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DPritchard%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DA%2520Quantitative%2520Analysis%2520of%2520Kinase%2520Inhibitor%2520Selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D127%26epage%3D132%26doi%3D10.1038%2Fnbt1358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Knighton, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ten Eyck, L. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashford, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xuong, N. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sowadski, J. M.</span><span> </span><span class="NLM_article-title">Crystal Structure of the Catalytic Subunit of Cyclic Adenosine Monophosphate-Dependent Protein Kinase</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">253</span><span class="NLM_x">, </span> <span class="NLM_fpage">407</span><span class="NLM_x">–</span> <span class="NLM_lpage">414</span><span class="refDoi"> DOI: 10.1126/science.1862342</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1126%2Fscience.1862342" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1862342" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADyaK3MXmsFSqsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=253&publication_year=1991&pages=407-414&author=D.+R.+Knightonauthor=J.+H.+Zhengauthor=L.+F.+Ten+Eyckauthor=V.+A.+Ashfordauthor=N.+H.+Xuongauthor=S.+S.+Taylorauthor=J.+M.+Sowadski&title=Crystal+Structure+of+the+Catalytic+Subunit+of+Cyclic+Adenosine+Monophosphate-Dependent+Protein+Kinase&doi=10.1126%2Fscience.1862342"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the catalytic subunit of cyclic adenosine monophosphate-dependent protein kinase</span></div><div class="casAuthors">Knighton, Daniel R.; Zheng, Jianhua; Ten Eyck, Lynn F.; Ashford, Victor A.; Nguyen Huu Xuong; Taylor, Susan S.; Sowadski, Janusz M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">253</span>
        (<span class="NLM_cas:issue">5018</span>),
    <span class="NLM_cas:pages">407-14</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">The crystal structure of the catalytic subunit of cyclic adenosine monophosphate-dependent protein kinase complexed with a 20-amino acid substrate analog inhibitor has been solved and partially refined at 2.7 Å resoln. to an R factor of 0.212.  The magnesium ATP (MgATP) binding site was located by difference Fourier synthesis.  The enzyme structure is bilobal with a deep cleft between the lobes.  The cleft is filled by MgATP and a portion of the inhibitor peptide.  The smaller lobe, consisting mostly of amino-terminal sequence, is assocd. with nucleotide binding, and its largely antiparallel β sheet architecture constitutes an unusual nucleotide binding motif.  The large lobe is dominated by helical structure with a single β sheet at the domain interface.  This lobe is primarily involved in peptide binding and catalysis.  Residues 40 through 280 constitute a conserved catalytic core that is shared by more than 100 protein kinases.  Most of the invariant amino acids in this conserved catalytic core are clustered at the site of nucleotide binding and catalysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-BPvdNuboTbVg90H21EOLACvtfcHk0lguvSVAuJaNJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXmsFSqsLw%253D&md5=1b1dc6aaa1a63fc665b81f8967e143b1</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1126%2Fscience.1862342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1862342%26sid%3Dliteratum%253Aachs%26aulast%3DKnighton%26aufirst%3DD.%2BR.%26aulast%3DZheng%26aufirst%3DJ.%2BH.%26aulast%3DTen%2BEyck%26aufirst%3DL.%2BF.%26aulast%3DAshford%26aufirst%3DV.%2BA.%26aulast%3DXuong%26aufirst%3DN.%2BH.%26aulast%3DTaylor%26aufirst%3DS.%2BS.%26aulast%3DSowadski%26aufirst%3DJ.%2BM.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520Catalytic%2520Subunit%2520of%2520Cyclic%2520Adenosine%2520Monophosphate-Dependent%2520Protein%2520Kinase%26jtitle%3DScience%26date%3D1991%26volume%3D253%26spage%3D407%26epage%3D414%26doi%3D10.1126%2Fscience.1862342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Schindler, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bornmann, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pellicena, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarkson, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuriyan, J.</span><span> </span><span class="NLM_article-title">Structural Mechanism for Sti-571 Inhibition of Abelson Tyrosine Kinase</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">289</span><span class="NLM_x">, </span> <span class="NLM_fpage">1938</span><span class="NLM_x">–</span> <span class="NLM_lpage">1942</span><span class="refDoi"> DOI: 10.1126/science.289.5486.1938</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1126%2Fscience.289.5486.1938" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10988075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD3cXms1altLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2000&pages=1938-1942&author=T.+Schindlerauthor=W.+Bornmannauthor=P.+Pellicenaauthor=W.+T.+Millerauthor=B.+Clarksonauthor=J.+Kuriyan&title=Structural+Mechanism+for+Sti-571+Inhibition+of+Abelson+Tyrosine+Kinase&doi=10.1126%2Fscience.289.5486.1938"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Structural mechanism for STI-571 inhibition of Abelson tyrosine kinase</span></div><div class="casAuthors">Schindler, Thomas; Bornmann, William; Pellicena, Patricia; Miller, W. Todd; Clarkson, Bayard; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">5486</span>),
    <span class="NLM_cas:pages">1938-1942</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The inadvertent activation of the Abelson tyrosine kinase (Abl) causes chronic myelogenous leukemia (CML).  A small-mol. inhibitor of Abl (STI-571) is effective in the treatment of CML.  We report the crystal structure of the catalytic domain of Abl, complexed to a variant of STI-571.  Crit. to the binding of STI-571 is the adoption by the kinase of an inactive conformation, in which a centrally located "activation loop" is not phosphorylated.  The conformation of this loop is distinct from that in active protein kinases, as well as in the inactive form of the closely related Src kinases.  These results suggest that compds. that exploit the distinctive inactivation mechanisms of individual protein kinases can achieve both high affinity and high specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9eAZqIQ1aebVg90H21EOLACvtfcHk0lguvSVAuJaNJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXms1altLc%253D&md5=56bb233f010f099d0073020ac6b9db59</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1126%2Fscience.289.5486.1938&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.289.5486.1938%26sid%3Dliteratum%253Aachs%26aulast%3DSchindler%26aufirst%3DT.%26aulast%3DBornmann%26aufirst%3DW.%26aulast%3DPellicena%26aufirst%3DP.%26aulast%3DMiller%26aufirst%3DW.%2BT.%26aulast%3DClarkson%26aufirst%3DB.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DStructural%2520Mechanism%2520for%2520Sti-571%2520Inhibition%2520of%2520Abelson%2520Tyrosine%2520Kinase%26jtitle%3DScience%26date%3D2000%26volume%3D289%26spage%3D1938%26epage%3D1942%26doi%3D10.1126%2Fscience.289.5486.1938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Girdler, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gascoigne, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eyers, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartmuth, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crafter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foote, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keen, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, S. S.</span><span> </span><span class="NLM_article-title">Validating Aurora B as an Anti-Cancer Drug Target</span> <span class="citation_source-journal">J. Cell Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">3664</span><span class="NLM_x">–</span> <span class="NLM_lpage">3675</span><span class="refDoi"> DOI: 10.1242/jcs.03145</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1242%2Fjcs.03145" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=16912073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVymu7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2006&pages=3664-3675&author=F.+Girdlerauthor=K.+E.+Gascoigneauthor=P.+A.+Eyersauthor=S.+Hartmuthauthor=C.+Crafterauthor=K.+M.+Footeauthor=N.+J.+Keenauthor=S.+S.+Taylor&title=Validating+Aurora+B+as+an+Anti-Cancer+Drug+Target&doi=10.1242%2Fjcs.03145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Validating Aurora B as an anti-cancer drug target</span></div><div class="casAuthors">Girdler, Fiona; Gascoigne, Karen E.; Eyers, Patrick A.; Hartmuth, Sonya; Crafter, Claire; Foote, Kevin M.; Keen, Nicholas J.; Taylor, Stephen S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3664-3675</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">The Aurora kinases, a family of mitotic regulators, have received much attention as potential targets for novel anti-cancer therapeutics.  Several Aurora kinase inhibitors have been described including ZM447439, which prevents chromosome alignment, spindle checkpoint function and cytokinesis.  Subsequently, ZM447439-treated cells exit mitosis without dividing and lose viability.  Because ZM447439 inhibits both Aurora A and B, we set out to det. which phenotypes are due to inhibition of which kinase.  Using mol. genetic approaches, we show that inhibition of Aurora B kinase activity phenocopies ZM447439.  Furthermore, a novel ZM compd., which is 100 times more selective for Aurora B over Aurora A in vitro, induces identical phenotypes.  Importantly, inhibition of Aurora B kinase activity induces a penetrant anti-proliferative phenotype, indicating that Aurora B is an attractive anti-cancer drug target.  Using mol. genetic and chem.-genetic approaches, we also probe the role of Aurora A kinase activity.  We show that simultaneous repression of Aurora A plus induction of a catalytic mutant induces a monopolar phenotype.  Consistently, another novel ZM-related inhibitor, which is 20 times as potent against Aurora A compared with ZM447439, induces a monopolar phenotype.  Expression of a drug-resistant Aurora A mutant reverts this phenotype, demonstrating that Aurora A kinase activity is required for spindle bipolarity in human cells.  Because small mol.-mediated inhibition of Aurora A and Aurora B yields distinct phenotypes, our observations indicate that the Auroras may present two avenues for anti-cancer drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXyhwkCpOaDrVg90H21EOLACvtfcHk0liG-otgq91hXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVymu7zP&md5=707af873d03b36f20a389470e0aab1ff</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1242%2Fjcs.03145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.03145%26sid%3Dliteratum%253Aachs%26aulast%3DGirdler%26aufirst%3DF.%26aulast%3DGascoigne%26aufirst%3DK.%2BE.%26aulast%3DEyers%26aufirst%3DP.%2BA.%26aulast%3DHartmuth%26aufirst%3DS.%26aulast%3DCrafter%26aufirst%3DC.%26aulast%3DFoote%26aufirst%3DK.%2BM.%26aulast%3DKeen%26aufirst%3DN.%2BJ.%26aulast%3DTaylor%26aufirst%3DS.%2BS.%26atitle%3DValidating%2520Aurora%2520B%2520as%2520an%2520Anti-Cancer%2520Drug%2520Target%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2006%26volume%3D119%26spage%3D3664%26epage%3D3675%26doi%3D10.1242%2Fjcs.03145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Dodson, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kosmopoulou, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richards, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atrash, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bavetsias, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blagg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayliss, R.</span><span> </span><span class="NLM_article-title">Crystal Structure of an Aurora-a Mutant That Mimics Aurora-B Bound to Mln8054: Insights into Selectivity and Drug Design</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">427</span><span class="NLM_x">, </span> <span class="NLM_fpage">19</span><span class="NLM_x">–</span> <span class="NLM_lpage">28</span><span class="refDoi"> DOI: 10.1042/BJ20091530</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1042%2FBJ20091530" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=20067443" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFyntr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=427&publication_year=2010&pages=19-28&author=C.+A.+Dodsonauthor=M.+Kosmopoulouauthor=M.+W.+Richardsauthor=B.+Atrashauthor=V.+Bavetsiasauthor=J.+Blaggauthor=R.+Bayliss&title=Crystal+Structure+of+an+Aurora-a+Mutant+That+Mimics+Aurora-B+Bound+to+Mln8054%3A+Insights+into+Selectivity+and+Drug+Design&doi=10.1042%2FBJ20091530"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of an Aurora-A mutant that mimics Aurora-B bound to MLN8054: insights into selectivity and drug design</span></div><div class="casAuthors">Dodson, Charlotte A.; Kosmopoulou, Magda; Richards, Mark W.; Atrash, Butrus; Bavetsias, Vassilios; Blagg, Julian; Bayliss, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">427</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19-28</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The prodn. of selective protein kinase inhibitors is often frustrated by the similarity of the enzyme active sites.  For this reason, it is challenging to design inhibitors that discriminate between the three Aurora kinases, which are important targets in cancer drug discovery.  We have used a triple-point mutant of Aurora-A (AurAx3) which mimics the active site of Aurora-B to investigate the structural basis of MLN8054 selectivity.  The bias toward Aurora-A inhibition by MLN8054 is fully recapitulated by AurAx3 in vitro.  X-ray crystal structures of the complex suggest that the basis for the discrimination is electrostatic repulsion due to the T217E substitution, which we have confirmed using a single-point mutant.  The activation loop of Aurora-A in the AurAx3-MLN8054 complex exhibits an unusual conformation in which Asp274 and Phe275 side chains point into the interior of the protein.  There is to our knowledge no documented precedent for this conformation, which we have termed DFG-up.  The sequence requirements of the DFG-up conformation suggest that it might be accessible to only a fraction of kinases.  MLN8054 thus circumvents the problem of highly homologous active sites.  Binding of MLN8054 to Aurora-A switches the character of a pocket within the active site from polar to a hydrophobic pocket, similar to what is obsd. in the structure of Aurora-A bound to a compd. that induces DFG-out.  We propose that targeting this pocket may be a productive route in the design of selective kinase inhibitors and describe the structural basis for the rational design of these compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdJs5KY2VkabVg90H21EOLACvtfcHk0liG-otgq91hXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFyntr8%253D&md5=fdf1a176a5fdf5fcd71fd57c3ef92817</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1042%2FBJ20091530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20091530%26sid%3Dliteratum%253Aachs%26aulast%3DDodson%26aufirst%3DC.%2BA.%26aulast%3DKosmopoulou%26aufirst%3DM.%26aulast%3DRichards%26aufirst%3DM.%2BW.%26aulast%3DAtrash%26aufirst%3DB.%26aulast%3DBavetsias%26aufirst%3DV.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBayliss%26aufirst%3DR.%26atitle%3DCrystal%2520Structure%2520of%2520an%2520Aurora-a%2520Mutant%2520That%2520Mimics%2520Aurora-B%2520Bound%2520to%2520Mln8054%253A%2520Insights%2520into%2520Selectivity%2520and%2520Drug%2520Design%26jtitle%3DBiochem.%2520J.%26date%3D2010%26volume%3D427%26spage%3D19%26epage%3D28%26doi%3D10.1042%2FBJ20091530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Sloane, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trikic, M. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamers, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mason, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savory, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eyers, P. A.</span><span> </span><span class="NLM_article-title">Drug-Resistant Aurora a Mutants for Cellular Target Validation of the Small Molecule Kinase Inhibitors Mln8054 and Mln8237</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">563</span><span class="NLM_x">–</span> <span class="NLM_lpage">576</span><span class="refDoi"> DOI: 10.1021/cb100053q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb100053q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlvVaktbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=563-576&author=D.+A.+Sloaneauthor=M.+Z.+Trikicauthor=M.+L.+Chuauthor=M.+B.+Lamersauthor=C.+S.+Masonauthor=I.+Muellerauthor=W.+J.+Savoryauthor=D.+H.+Williamsauthor=P.+A.+Eyers&title=Drug-Resistant+Aurora+a+Mutants+for+Cellular+Target+Validation+of+the+Small+Molecule+Kinase+Inhibitors+Mln8054+and+Mln8237&doi=10.1021%2Fcb100053q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-Resistant Aurora A Mutants for Cellular Target Validation of the Small Molecule Kinase Inhibitors MLN8054 and MLN8237</span></div><div class="casAuthors">Sloane, Dominic A.; Trikic, Michael Z.; Chu, Matthew L. H.; Lamers, Maria B. A. C.; Mason, Clive S.; Mueller, Ilka; Savory, Wendy J.; Williams, David H.; Eyers, Patrick A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">563-576</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Aurora kinases regulate multiple aspects of mitotic progression, and their overexpression in diverse tumor types makes them appealing oncol. targets.  An intensive research effort over the past decade has led to the discovery of chem. distinct families of small mol. Aurora kinase inhibitors, many of which have demonstrated therapeutic potential in model systems.  These agents are also important tools to help dissect signaling pathways that are orchestrated by Aurora kinases, and the antiproliferative target of pan-Aurora inhibitors such as VX-680 has been validated using chem. genetic techniques.  In many cases the nonspecific nature of Aurora inhibitors toward unrelated kinases is well established, potentially broadening the spectrum of cancers to which these compds. might be applied.  However, unambiguously demonstrating the mol. target(s) for clin. kinase inhibitors is an important challenge, one that is absolutely crit. for deciphering the mol. basis of compd. specificity, resistance, and efficacy.  In this paper, we have investigated amino acid requirements for Aurora A sensitivity to the benzazepine-based Aurora inhibitor MLN8054 and the close analog MLN8237, a second-generation compd. that is in phase II clin. trials.  A crystallog. anal. facilitated the design and biochem. investigation of a panel of resistant Aurora A mutants, a subset of which were then selected as candidate drug-resistance targets for further evaluation.  Using inducible human cell lines, we show that cells expressing near-physiol. levels of a functional but partially drug-resistant Aurora A T217D mutant survive in the presence of MLN8054 or MLN8237, authenticating Aurora A as a crit. antiproliferative target of these compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjg-t6yjgWhLVg90H21EOLACvtfcHk0liG-otgq91hXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlvVaktbs%253D&md5=f20bc2f2f827fd8f640570078d6505c6</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fcb100053q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb100053q%26sid%3Dliteratum%253Aachs%26aulast%3DSloane%26aufirst%3DD.%2BA.%26aulast%3DTrikic%26aufirst%3DM.%2BZ.%26aulast%3DChu%26aufirst%3DM.%2BL.%26aulast%3DLamers%26aufirst%3DM.%2BB.%26aulast%3DMason%26aufirst%3DC.%2BS.%26aulast%3DMueller%26aufirst%3DI.%26aulast%3DSavory%26aufirst%3DW.%2BJ.%26aulast%3DWilliams%26aufirst%3DD.%2BH.%26aulast%3DEyers%26aufirst%3DP.%2BA.%26atitle%3DDrug-Resistant%2520Aurora%2520a%2520Mutants%2520for%2520Cellular%2520Target%2520Validation%2520of%2520the%2520Small%2520Molecule%2520Kinase%2520Inhibitors%2520Mln8054%2520and%2520Mln8237%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2010%26volume%3D5%26spage%3D563%26epage%3D576%26doi%3D10.1021%2Fcb100053q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Eyers, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erikson, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maller, J. L.</span><span> </span><span class="NLM_article-title">A Novel Mechanism for Activation of the Protein Kinase Aurora A</span> <span class="citation_source-journal">Curr. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">691</span><span class="NLM_x">–</span> <span class="NLM_lpage">697</span><span class="refDoi"> DOI: 10.1016/S0960-9822(03)00166-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1016%2FS0960-9822%2803%2900166-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=12699628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjtVKhu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=691-697&author=P.+A.+Eyersauthor=E.+Eriksonauthor=L.+G.+Chenauthor=J.+L.+Maller&title=A+Novel+Mechanism+for+Activation+of+the+Protein+Kinase+Aurora+A&doi=10.1016%2FS0960-9822%2803%2900166-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Mechanism for Activation of the Protein Kinase Aurora A</span></div><div class="casAuthors">Eyers, Patrick A.; Erikson, Eleanor; Chen, Lin G.; Maller, James L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">691-697</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Segregation of chromosomes during mitosis requires interplay between several classes of protein on the spindle, including protein kinases, protein phosphatases, and microtubule binding motor proteins.  Aurora A is an oncogenic cell cycle-regulated protein kinase that is subject to phosphorylation-dependent activation.  Aurora A localization to the mitotic spindle depends on the motor binding protein TPX2 (Targeting Protein for Xenopus kinesin-like protein 2), but the protein(s) involved in Aurora A activation are unknown.  Here, we purify an activator of Aurora A from Xenopus eggs and identify it as TPX2.  Remarkably, Aurora A that has been fully deactivated by Protein Phosphatase 2A (PP2A) becomes phosphorylated and reactivated by recombinant TPX2 in an ATP-dependent manner.  Increased phosphorylation and activation of Aurora A requires its own kinase activity, suggesting that TPX2 stimulates autophosphorylation and autoactivation of the enzyme.  Consistently, wild-type Aurora A, but not a kinase inactive mutant, becomes autophosphorylated on the regulatory T loop residue (Thr 295) after TPX2 treatment.  Active Aurora A from bacteria is further activated at least 7-fold by recombinant TPX2, and TPX2 also impairs the ability of protein phosphatases to inactivate Aurora A in vitro.  This concerted mechanism of stimulation of activation and inhibition of deactivation implies that TPX2 is the likely regulator of Aurora A activity at the mitotic spindle and may explain why loss of TPX2 in model systems perturbs spindle assembly.  Our finding that a known binding protein, and not a conventional protein kinase, is the relevant activator for Aurora A suggests a biochem. model in which the dynamic localization of TPX2 on mitotic structures directly modulates the activity of Aurora A for spindle assembly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouvFlsVOOTQ7Vg90H21EOLACvtfcHk0lilgEWjq6WAnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjtVKhu7k%253D&md5=98dc1619636f156c8a5748b7fdfa09fc</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2FS0960-9822%2803%2900166-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-9822%252803%252900166-0%26sid%3Dliteratum%253Aachs%26aulast%3DEyers%26aufirst%3DP.%2BA.%26aulast%3DErikson%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DL.%2BG.%26aulast%3DMaller%26aufirst%3DJ.%2BL.%26atitle%3DA%2520Novel%2520Mechanism%2520for%2520Activation%2520of%2520the%2520Protein%2520Kinase%2520Aurora%2520A%26jtitle%3DCurr.%2520Biol.%26date%3D2003%26volume%3D13%26spage%3D691%26epage%3D697%26doi%3D10.1016%2FS0960-9822%2803%2900166-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">Neumayer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belzil, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gruss, O. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, M. D.</span><span> </span><span class="NLM_article-title">Tpx2: Of Spindle Assembly, DNA Damage Response, and Cancer</span> <span class="citation_source-journal">Cell. Mol. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x">, </span> <span class="NLM_fpage">3027</span><span class="NLM_x">–</span> <span class="NLM_lpage">3047</span><span class="refDoi"> DOI: 10.1007/s00018-014-1582-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1007%2Fs00018-014-1582-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=24556998" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC2cXivFCiu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2014&pages=3027-3047&author=G.+Neumayerauthor=C.+Belzilauthor=O.+J.+Grussauthor=M.+D.+Nguyen&title=Tpx2%3A+Of+Spindle+Assembly%2C+DNA+Damage+Response%2C+and+Cancer&doi=10.1007%2Fs00018-014-1582-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">TPX2: of spindle assembly, DNA damage response, and cancer</span></div><div class="casAuthors">Neumayer, Gernot; Belzil, Camille; Gruss, Oliver J.; Nguyen, Minh Dang</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3027-3047</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Basel</span>)
        </div><div class="casAbstract">A review.  For more than 15 years, TPX2 has been studied as a factor crit. for mitosis and spindle assembly.  These functions of TPX2 are attributed to its Ran-regulated microtubule-assocd. protein properties and to its control of the Aurora A kinase.  Overexpressed in cancers, TPX2 is being established as marker for the diagnosis and prognosis of malignancies.  During interphase, TPX2 resides preferentially in the nucleus where its function had remained elusive until recently.  The latest finding that TPX2 plays a role in amplification of the DNA damage response, combined with the characterization of TPX2 knockout mice, open new perspectives to understand the biol. of this protein.  This review provides an historic overview of the discovery of TPX2 and summarizes its cytoskeletal and signaling roles with relevance to cancer therapies.  Finally, the review aims to reconcile discrepancies between the exptl. and pathol. effects of TPX2 overexpression and advances new roles for compartmentalized TPX2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo81lT1JcE_QrVg90H21EOLACvtfcHk0lilgEWjq6WAnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXivFCiu7Y%253D&md5=061d1c7591591ceb6a8974663ad1c125</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1007%2Fs00018-014-1582-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-014-1582-7%26sid%3Dliteratum%253Aachs%26aulast%3DNeumayer%26aufirst%3DG.%26aulast%3DBelzil%26aufirst%3DC.%26aulast%3DGruss%26aufirst%3DO.%2BJ.%26aulast%3DNguyen%26aufirst%3DM.%2BD.%26atitle%3DTpx2%253A%2520Of%2520Spindle%2520Assembly%252C%2520DNA%2520Damage%2520Response%252C%2520and%2520Cancer%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2014%26volume%3D71%26spage%3D3027%26epage%3D3047%26doi%3D10.1007%2Fs00018-014-1582-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Zorba, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buosi, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kutter, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kern, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pontiggia, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kern, D.</span><span> </span><span class="NLM_article-title">Molecular Mechanism of Aurora a Kinase Autophosphorylation and Its Allosteric Activation by Tpx2</span> <span class="citation_source-journal">eLife</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">e02667</span><span class="refDoi"> DOI: 10.7554/eLife.02667</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.7554%2FeLife.02667" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&pages=e02667&author=A.+Zorbaauthor=V.+Buosiauthor=S.+Kutterauthor=N.+Kernauthor=F.+Pontiggiaauthor=Y.+J.+Choauthor=D.+Kern&title=Molecular+Mechanism+of+Aurora+a+Kinase+Autophosphorylation+and+Its+Allosteric+Activation+by+Tpx2&doi=10.7554%2FeLife.02667"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.7554%2FeLife.02667&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.02667%26sid%3Dliteratum%253Aachs%26aulast%3DZorba%26aufirst%3DA.%26aulast%3DBuosi%26aufirst%3DV.%26aulast%3DKutter%26aufirst%3DS.%26aulast%3DKern%26aufirst%3DN.%26aulast%3DPontiggia%26aufirst%3DF.%26aulast%3DCho%26aufirst%3DY.%2BJ.%26aulast%3DKern%26aufirst%3DD.%26atitle%3DMolecular%2520Mechanism%2520of%2520Aurora%2520a%2520Kinase%2520Autophosphorylation%2520and%2520Its%2520Allosteric%2520Activation%2520by%2520Tpx2%26jtitle%3DeLife%26date%3D2014%26volume%3D3%26spage%3De02667%26doi%3D10.7554%2FeLife.02667" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">Kufer, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sillje, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gruss, O. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meraldi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nigg, E. A.</span><span> </span><span class="NLM_article-title">Human Tpx2 Is Required for Targeting Aurora-a Kinase to the Spindle</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">158</span><span class="NLM_x">, </span> <span class="NLM_fpage">617</span><span class="NLM_x">–</span> <span class="NLM_lpage">623</span><span class="refDoi"> DOI: 10.1083/jcb.200204155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1083%2Fjcb.200204155" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=158&publication_year=2002&pages=617-623&author=T.+A.+Kuferauthor=H.+H.+Silljeauthor=R.+Kornerauthor=O.+J.+Grussauthor=P.+Meraldiauthor=E.+A.+Nigg&title=Human+Tpx2+Is+Required+for+Targeting+Aurora-a+Kinase+to+the+Spindle&doi=10.1083%2Fjcb.200204155"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1083%2Fjcb.200204155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.200204155%26sid%3Dliteratum%253Aachs%26aulast%3DKufer%26aufirst%3DT.%2BA.%26aulast%3DSillje%26aufirst%3DH.%2BH.%26aulast%3DKorner%26aufirst%3DR.%26aulast%3DGruss%26aufirst%3DO.%2BJ.%26aulast%3DMeraldi%26aufirst%3DP.%26aulast%3DNigg%26aufirst%3DE.%2BA.%26atitle%3DHuman%2520Tpx2%2520Is%2520Required%2520for%2520Targeting%2520Aurora-a%2520Kinase%2520to%2520the%2520Spindle%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2002%26volume%3D158%26spage%3D617%26epage%3D623%26doi%3D10.1083%2Fjcb.200204155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group">Kesisova, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakos, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsolou, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Angelis, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chatzaki, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agianian, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giannis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koffa, M. D.</span><span> </span><span class="NLM_article-title">Tripolin a, a Novel Small-Molecule Inhibitor of Aurora a Kinase, Reveals New Regulation of Hurp’s Distribution on Microtubules</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e58485</span><span class="refDoi"> DOI: 10.1371/journal.pone.0058485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1371%2Fjournal.pone.0058485" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e58485&author=I.+A.+Kesisovaauthor=K.+C.+Nakosauthor=A.+Tsolouauthor=D.+Angelisauthor=J.+Lewisauthor=A.+Chatzakiauthor=B.+Agianianauthor=A.+Giannisauthor=M.+D.+Koffa&title=Tripolin+a%2C+a+Novel+Small-Molecule+Inhibitor+of+Aurora+a+Kinase%2C+Reveals+New+Regulation+of+Hurp%E2%80%99s+Distribution+on+Microtubules&doi=10.1371%2Fjournal.pone.0058485"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0058485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0058485%26sid%3Dliteratum%253Aachs%26aulast%3DKesisova%26aufirst%3DI.%2BA.%26aulast%3DNakos%26aufirst%3DK.%2BC.%26aulast%3DTsolou%26aufirst%3DA.%26aulast%3DAngelis%26aufirst%3DD.%26aulast%3DLewis%26aufirst%3DJ.%26aulast%3DChatzaki%26aufirst%3DA.%26aulast%3DAgianian%26aufirst%3DB.%26aulast%3DGiannis%26aufirst%3DA.%26aulast%3DKoffa%26aufirst%3DM.%2BD.%26atitle%3DTripolin%2520a%252C%2520a%2520Novel%2520Small-Molecule%2520Inhibitor%2520of%2520Aurora%2520a%2520Kinase%252C%2520Reveals%2520New%2520Regulation%2520of%2520Hurp%25E2%2580%2599s%2520Distribution%2520on%2520Microtubules%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De58485%26doi%3D10.1371%2Fjournal.pone.0058485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group">Littlepage, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andresson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deanehan, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amundadottir, L. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruderman, J. V.</span><span> </span><span class="NLM_article-title">Identification of Phosphorylated Residues That Affect the Activity of the Mitotic Kinase Aurora-A</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">15440</span><span class="NLM_x">–</span> <span class="NLM_lpage">15445</span><span class="refDoi"> DOI: 10.1073/pnas.202606599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1073%2Fpnas.202606599" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=15440-15445&author=L.+E.+Littlepageauthor=H.+Wuauthor=T.+Andressonauthor=J.+K.+Deanehanauthor=L.+T.+Amundadottirauthor=J.+V.+Ruderman&title=Identification+of+Phosphorylated+Residues+That+Affect+the+Activity+of+the+Mitotic+Kinase+Aurora-A&doi=10.1073%2Fpnas.202606599"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1073%2Fpnas.202606599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.202606599%26sid%3Dliteratum%253Aachs%26aulast%3DLittlepage%26aufirst%3DL.%2BE.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DAndresson%26aufirst%3DT.%26aulast%3DDeanehan%26aufirst%3DJ.%2BK.%26aulast%3DAmundadottir%26aufirst%3DL.%2BT.%26aulast%3DRuderman%26aufirst%3DJ.%2BV.%26atitle%3DIdentification%2520of%2520Phosphorylated%2520Residues%2520That%2520Affect%2520the%2520Activity%2520of%2520the%2520Mitotic%2520Kinase%2520Aurora-A%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D15440%26epage%3D15445%26doi%3D10.1073%2Fpnas.202606599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group">Marumoto, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nitta, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirota, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kohmura, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saya, H.</span><span> </span><span class="NLM_article-title">Aurora-a Kinase Maintains the Fidelity of Early and Late Mitotic Events in Hela Cells</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">278</span><span class="NLM_x">, </span> <span class="NLM_fpage">51786</span><span class="NLM_x">–</span> <span class="NLM_lpage">51795</span><span class="refDoi"> DOI: 10.1074/jbc.M306275200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1074%2Fjbc.M306275200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=51786-51795&author=T.+Marumotoauthor=S.+Hondaauthor=T.+Haraauthor=M.+Nittaauthor=T.+Hirotaauthor=E.+Kohmuraauthor=H.+Saya&title=Aurora-a+Kinase+Maintains+the+Fidelity+of+Early+and+Late+Mitotic+Events+in+Hela+Cells&doi=10.1074%2Fjbc.M306275200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M306275200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M306275200%26sid%3Dliteratum%253Aachs%26aulast%3DMarumoto%26aufirst%3DT.%26aulast%3DHonda%26aufirst%3DS.%26aulast%3DHara%26aufirst%3DT.%26aulast%3DNitta%26aufirst%3DM.%26aulast%3DHirota%26aufirst%3DT.%26aulast%3DKohmura%26aufirst%3DE.%26aulast%3DSaya%26aufirst%3DH.%26atitle%3DAurora-a%2520Kinase%2520Maintains%2520the%2520Fidelity%2520of%2520Early%2520and%2520Late%2520Mitotic%2520Events%2520in%2520Hela%2520Cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D51786%26epage%3D51795%26doi%3D10.1074%2Fjbc.M306275200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group">Giet, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glover, D. M.</span><span> </span><span class="NLM_article-title">Drosophila Aurora B Kinase Is Required for Histone H3 Phosphorylation and Condensin Recruitment During Chromosome Condensation and to Organize the Central Spindle During Cytokinesis</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">152</span><span class="NLM_x">, </span> <span class="NLM_fpage">669</span><span class="NLM_x">–</span> <span class="NLM_lpage">682</span><span class="refDoi"> DOI: 10.1083/jcb.152.4.669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1083%2Fjcb.152.4.669" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2001&pages=669-682&author=R.+Gietauthor=D.+M.+Glover&title=Drosophila+Aurora+B+Kinase+Is+Required+for+Histone+H3+Phosphorylation+and+Condensin+Recruitment+During+Chromosome+Condensation+and+to+Organize+the+Central+Spindle+During+Cytokinesis&doi=10.1083%2Fjcb.152.4.669"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1083%2Fjcb.152.4.669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.152.4.669%26sid%3Dliteratum%253Aachs%26aulast%3DGiet%26aufirst%3DR.%26aulast%3DGlover%26aufirst%3DD.%2BM.%26atitle%3DDrosophila%2520Aurora%2520B%2520Kinase%2520Is%2520Required%2520for%2520Histone%2520H3%2520Phosphorylation%2520and%2520Condensin%2520Recruitment%2520During%2520Chromosome%2520Condensation%2520and%2520to%2520Organize%2520the%2520Central%2520Spindle%2520During%2520Cytokinesis%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2001%26volume%3D152%26spage%3D669%26epage%3D682%26doi%3D10.1083%2Fjcb.152.4.669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group">Ditchfield, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, V. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tighe, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellston, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haworth, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mortlock, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, S. S.</span><span> </span><span class="NLM_article-title">Aurora B Couples Chromosome Alignment with Anaphase by Targeting Bubr1, Mad2, and Cenp-E to Kinetochores</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">161</span><span class="NLM_x">, </span> <span class="NLM_fpage">267</span><span class="NLM_x">–</span> <span class="NLM_lpage">280</span><span class="refDoi"> DOI: 10.1083/jcb.200208091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1083%2Fjcb.200208091" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=161&publication_year=2003&pages=267-280&author=C.+Ditchfieldauthor=V.+L.+Johnsonauthor=A.+Tigheauthor=R.+Ellstonauthor=C.+Haworthauthor=T.+Johnsonauthor=A.+Mortlockauthor=N.+Keenauthor=S.+S.+Taylor&title=Aurora+B+Couples+Chromosome+Alignment+with+Anaphase+by+Targeting+Bubr1%2C+Mad2%2C+and+Cenp-E+to+Kinetochores&doi=10.1083%2Fjcb.200208091"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1083%2Fjcb.200208091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.200208091%26sid%3Dliteratum%253Aachs%26aulast%3DDitchfield%26aufirst%3DC.%26aulast%3DJohnson%26aufirst%3DV.%2BL.%26aulast%3DTighe%26aufirst%3DA.%26aulast%3DEllston%26aufirst%3DR.%26aulast%3DHaworth%26aufirst%3DC.%26aulast%3DJohnson%26aufirst%3DT.%26aulast%3DMortlock%26aufirst%3DA.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DTaylor%26aufirst%3DS.%2BS.%26atitle%3DAurora%2520B%2520Couples%2520Chromosome%2520Alignment%2520with%2520Anaphase%2520by%2520Targeting%2520Bubr1%252C%2520Mad2%252C%2520and%2520Cenp-E%2520to%2520Kinetochores%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2003%26volume%3D161%26spage%3D267%26epage%3D280%26doi%3D10.1083%2Fjcb.200208091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group">Harrington, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bebbington, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rasmussen, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ajose-Adeogun, A. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakayama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graham, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demur, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hercend, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diu-Hercend, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golec, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, K. M.</span><span> </span><span class="NLM_article-title">Vx-680, a Potent and Selective Small-Molecule Inhibitor of the Aurora Kinases, Suppresses Tumor Growth in Vivo</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">262</span><span class="NLM_x">–</span> <span class="NLM_lpage">267</span><span class="refDoi"> DOI: 10.1038/nm1003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1038%2Fnm1003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=14981513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhs1Wnur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=262-267&author=E.+A.+Harringtonauthor=D.+Bebbingtonauthor=J.+Mooreauthor=R.+K.+Rasmussenauthor=A.+O.+Ajose-Adeogunauthor=T.+Nakayamaauthor=J.+A.+Grahamauthor=C.+Demurauthor=T.+Hercendauthor=A.+Diu-Hercendauthor=M.+Suauthor=J.+M.+Golecauthor=K.+M.+Miller&title=Vx-680%2C+a+Potent+and+Selective+Small-Molecule+Inhibitor+of+the+Aurora+Kinases%2C+Suppresses+Tumor+Growth+in+Vivo&doi=10.1038%2Fnm1003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">VX-680, a potent and selective small-molecule inhibitor of the Aurora kinases, suppresses tumor growth in vivo</span></div><div class="casAuthors">Harrington, Elizabeth A.; Bebbington, David; Moore, Jeff; Rasmussen, Richele K.; Ajose-Adeogun, Abi O.; Nakayama, Tomoko; Graham, Joanne A.; Demur, Cecile; Hercend, Thierry; Diu-Hercend, Anita; Su, Michael; Golec, Julian M. C.; Miller, Karen M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">262-267</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The Aurora kinases are essential for the regulation of chromosome segregation and cytokinesis during mitosis.  Aberrant expression and activity of these kinases occur in a wide range of human tumors, and lead to aneuploidy and tumorigenesis.  Here we report the discovery of a highly potent and selective small-mol. inhibitor of Aurora kinases, VX-680, that blocks cell-cycle progression and induces apoptosis in a diverse range of human tumor types.  This compd. causes profound inhibition of tumor growth in a variety of in vivo xenograft models, leading to regression of leukemia, colon and pancreatic tumors at well-tolerated doses.  Our data indicate that Aurora kinase inhibition provides a new approach for the treatment of multiple human malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKgm1YUgqkWLVg90H21EOLACvtfcHk0lhR1qqcqSAJfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhs1Wnur4%253D&md5=954ba274e280d8d08a7f2e331011432d</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1038%2Fnm1003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm1003%26sid%3Dliteratum%253Aachs%26aulast%3DHarrington%26aufirst%3DE.%2BA.%26aulast%3DBebbington%26aufirst%3DD.%26aulast%3DMoore%26aufirst%3DJ.%26aulast%3DRasmussen%26aufirst%3DR.%2BK.%26aulast%3DAjose-Adeogun%26aufirst%3DA.%2BO.%26aulast%3DNakayama%26aufirst%3DT.%26aulast%3DGraham%26aufirst%3DJ.%2BA.%26aulast%3DDemur%26aufirst%3DC.%26aulast%3DHercend%26aufirst%3DT.%26aulast%3DDiu-Hercend%26aufirst%3DA.%26aulast%3DSu%26aufirst%3DM.%26aulast%3DGolec%26aufirst%3DJ.%2BM.%26aulast%3DMiller%26aufirst%3DK.%2BM.%26atitle%3DVx-680%252C%2520a%2520Potent%2520and%2520Selective%2520Small-Molecule%2520Inhibitor%2520of%2520the%2520Aurora%2520Kinases%252C%2520Suppresses%2520Tumor%2520Growth%2520in%2520Vivo%26jtitle%3DNat.%2520Med.%26date%3D2004%26volume%3D10%26spage%3D262%26epage%3D267%26doi%3D10.1038%2Fnm1003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group">Shimomura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasako, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakatsuru, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mita, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ichikawa, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kodera, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakai, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nambu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyamoto, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahashi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawanishi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohkubo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotani, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwasawa, Y.</span><span> </span><span class="NLM_article-title">Mk-5108, a Highly Selective Aurora-a Kinase Inhibitor, Shows Antitumor Activity Alone and in Combination with Docetaxel</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">157</span><span class="NLM_x">–</span> <span class="NLM_lpage">166</span><span class="refDoi"> DOI: 10.1158/1535-7163.MCT-09-0609</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1158%2F1535-7163.MCT-09-0609" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=20053775" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktVOisQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=157-166&author=T.+Shimomuraauthor=S.+Hasakoauthor=Y.+Nakatsuruauthor=T.+Mitaauthor=K.+Ichikawaauthor=T.+Koderaauthor=T.+Sakaiauthor=T.+Nambuauthor=M.+Miyamotoauthor=I.+Takahashiauthor=S.+Mikiauthor=N.+Kawanishiauthor=M.+Ohkuboauthor=H.+Kotaniauthor=Y.+Iwasawa&title=Mk-5108%2C+a+Highly+Selective+Aurora-a+Kinase+Inhibitor%2C+Shows+Antitumor+Activity+Alone+and+in+Combination+with+Docetaxel&doi=10.1158%2F1535-7163.MCT-09-0609"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">MK-5108, a Highly Selective Aurora-A Kinase Inhibitor, Shows Antitumor Activity Alone and in Combination with Docetaxel</span></div><div class="casAuthors">Shimomura, Toshiyasu; Hasako, Shinichi; Nakatsuru, Yoko; Mita, Takashi; Ichikawa, Koji; Kodera, Tsutomu; Sakai, Takumi; Nambu, Tadahiro; Miyamoto, Mayu; Takahashi, Ikuko; Miki, Satomi; Kawanishi, Nobuhiko; Ohkubo, Mitsuru; Kotani, Hidehito; Iwasawa, Yoshikazu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">157-166</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Aurora-A kinase is a one of the key regulators during mitosis progression.  Aurora-A kinase is a potential target for anticancer therapies because overexpression of Aurora-A, which is frequently obsd. in some human cancers, results in aberrant mitosis leading to chromosomal instability and possibly tumorigenesis.  MK-5108 is a novel small mol. with potent inhibitory activity against Aurora-A kinase.  Although most of the Aurora-kinase inhibitors target both Aurora-A and Aurora-B, MK-5108 specifically inhibited Aurora-A kinase in a panel of protein kinase assays.  Inhibition of Aurora-A by MK-5108 in cultured cells induced cell cycle arrest at the G2-M phase in flow cytometry anal.  The effect was confirmed by the accumulation of cells with expression of phosphorylated Histone H3 and inhibition of Aurora-A autophosphorylation by immunostaining assays.  MK-5108 also induced phosphorylated Histone H3 in skin and xenograft tumor tissues in a nude rat xenograft model.  MK-5108 inhibited growth of human tumor cell lines in culture and in different xenograft models.  Furthermore, the combination of MK-5108 and docetaxel showed enhanced antitumor activities compared with control and docetaxel alone-treated animals without exacerbating the adverse effects of docetaxel.  MK-5108 is currently tested in clin. trials and offers a new therapeutic approach to combat human cancers as a single agent or in combination with existing taxane therapies.  Mol Cancer Ther; 9(1); 157-66.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRD-HWMVBoWrVg90H21EOLACvtfcHk0lhR1qqcqSAJfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktVOisQ%253D%253D&md5=267f0e3f702e2489e8b0c942f9299850</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-09-0609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-09-0609%26sid%3Dliteratum%253Aachs%26aulast%3DShimomura%26aufirst%3DT.%26aulast%3DHasako%26aufirst%3DS.%26aulast%3DNakatsuru%26aufirst%3DY.%26aulast%3DMita%26aufirst%3DT.%26aulast%3DIchikawa%26aufirst%3DK.%26aulast%3DKodera%26aufirst%3DT.%26aulast%3DSakai%26aufirst%3DT.%26aulast%3DNambu%26aufirst%3DT.%26aulast%3DMiyamoto%26aufirst%3DM.%26aulast%3DTakahashi%26aufirst%3DI.%26aulast%3DMiki%26aufirst%3DS.%26aulast%3DKawanishi%26aufirst%3DN.%26aulast%3DOhkubo%26aufirst%3DM.%26aulast%3DKotani%26aufirst%3DH.%26aulast%3DIwasawa%26aufirst%3DY.%26atitle%3DMk-5108%252C%2520a%2520Highly%2520Selective%2520Aurora-a%2520Kinase%2520Inhibitor%252C%2520Shows%2520Antitumor%2520Activity%2520Alone%2520and%2520in%2520Combination%2520with%2520Docetaxel%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2010%26volume%3D9%26spage%3D157%26epage%3D166%26doi%3D10.1158%2F1535-7163.MCT-09-0609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group">Evans, P.</span><span> </span><span class="NLM_article-title">Scaling and Assessment of Data Quality</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">72</span><span class="NLM_x">–</span> <span class="NLM_lpage">82</span><span class="refDoi"> DOI: 10.1107/S0907444905036693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1107%2FS0907444905036693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=16369096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=72-82&author=P.+Evans&title=Scaling+and+Assessment+of+Data+Quality&doi=10.1107%2FS0907444905036693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Scaling and assessment of data quality</span></div><div class="casAuthors">Evans, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-82</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The various phys. factors affecting measured diffraction intensities are discussed, as are the scaling models which may be used to put the data on a consistent scale.  After scaling, the intensities can be analyzed to set the real resoln. of the data set, to detect bad regions (e.g. bad images), to analyze radiation damage and to assess the overall quality of the data set.  The significance of any anomalous signal may be assessed by probability and correlation anal.  The algorithms used by the CCP4 scaling program SCALA are described.  A requirement for the scaling and merging of intensities is knowledge of the Laue group and point-group symmetries: the possible symmetry of the diffraction pattern may be detd. from scores such as correlation coeffs. between observations which might be symmetry-related.  These scoring functions are implemented in a new program POINTLESS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGravLSLU62RhLVg90H21EOLACvtfcHk0lhR1qqcqSAJfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM&md5=293d3876e534c0c57813990515bb3c76</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1107%2FS0907444905036693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444905036693%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%26atitle%3DScaling%2520and%2520Assessment%2520of%2520Data%2520Quality%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D72%26epage%3D82%26doi%3D10.1107%2FS0907444905036693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group">Leslie, A. G. W.; Powell, H. R.; Read, R. J.; Sussman, J. L.</span>, <span> </span><span class="NLM_article-title">Processing Diffraction Data with Mosflm</span>.  <span class="citation_source-book">Nato Sci. Ser. II Math.</span>; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Dordrecht, The Netherlands</span>,<span class="NLM_x"> </span><span class="NLM_year">2007</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">245</span>, p  <span class="NLM_fpage">41</span>–.<span class="refDoi"> DOI: 10.1007/978-1-4020-6316-9_4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1007%2F978-1-4020-6316-9_4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&pages=41&author=A.+G.+W.+Leslie&author=H.+R.+Powell&author=R.+J.+Read&author=J.+L.+Sussman&title=Nato+Sci.+Ser.+II+Math."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1007%2F978-1-4020-6316-9_4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-4020-6316-9_4%26sid%3Dliteratum%253Aachs%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26atitle%3DProcessing%2520Diffraction%2520Data%2520with%2520Mosflm%26btitle%3DNato%2520Sci.%2520Ser.%2520II%2520Math.%26pub%3DSpringer%26date%3D2007%26volume%3D245%26spage%3D41%26doi%3D10.1007%2F978-1-4020-6316-9_4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">Xds</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span><span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0liAjweLPdPVqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballard, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, K. S.</span><span> </span><span class="NLM_article-title">Overview of the Ccp4 Suite and Current Developments</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=K.+S.+Wilson&title=Overview+of+the+Ccp4+Suite+and+Current+Developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0liAjweLPdPVqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520Ccp4%2520Suite%2520and%2520Current%2520Developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkoczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">Phenix: A Comprehensive Python-Based System for Macromolecular Structure Solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=Phenix%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0liAjweLPdPVqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPhenix%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and Development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lgwuitBZho2Vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group">DeLano, W. L.</span>The PyMOL Molecular Graphics System; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">Cambridge, MA</span>,<span class="NLM_x"> </span><span class="NLM_year">2002</span><span class="NLM_x">; </span><a href="http://www.pymol.org" class="extLink">http://www.pymol.org</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2002&author=W.+L.+DeLano&title=The+PyMOL+Molecular+Graphics+System"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DDeLano%26aufirst%3DW.%2BL.%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group">Wallace, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laskowski, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thornton, J. M.</span><span> </span><span class="NLM_article-title">Ligplot - a Program to Generate Schematic Diagrams of Protein Ligand Interactions</span> <span class="citation_source-journal">Protein Eng., Des. Sel.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">134</span><span class="refDoi"> DOI: 10.1093/protein/8.2.127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1093%2Fprotein%2F8.2.127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=1%3ACAS%3A528%3ADyaK2MXltVCgtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1995&pages=127-134&author=A.+C.+Wallaceauthor=R.+A.+Laskowskiauthor=J.+M.+Thornton&title=Ligplot+-+a+Program+to+Generate+Schematic+Diagrams+of+Protein+Ligand+Interactions&doi=10.1093%2Fprotein%2F8.2.127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">LIGPLOT: a program to generate schematic diagrams of protein-ligand interactions</span></div><div class="casAuthors">Wallace, Andrew C.; Laskowski, Roman A.; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Engineering</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">127-34</span>CODEN:
                <span class="NLM_cas:coden">PRENE9</span>;
        ISSN:<span class="NLM_cas:issn">0269-2139</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The LIGPLOT program automatically generates schematic 2-D representations of protein-ligand complexes from std. Protein Data Bank file input.  The output is a color, or black-and-white, PostScript file giving a simple and informative representation of the intermol. interactions and their strengths, including hydrogen bonds, hydrophobic interactions and atom accessibilities.  The program is completely general for any ligand and can also be used to show other types of interaction in proteins and nucleic acids.  It was designed to facilitate the rapid inspection of many enzyme complexes, but has found many other applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpp9SrWw0klVbVg90H21EOLACvtfcHk0lgwuitBZho2Vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXltVCgtLc%253D&md5=ed74e39b0cd6be4b2b8907d3ff6d9619</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F8.2.127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F8.2.127%26sid%3Dliteratum%253Aachs%26aulast%3DWallace%26aufirst%3DA.%2BC.%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DLigplot%2520-%2520a%2520Program%2520to%2520Generate%2520Schematic%2520Diagrams%2520of%2520Protein%2520Ligand%2520Interactions%26jtitle%3DProtein%2520Eng.%252C%2520Des.%2520Sel.%26date%3D1995%26volume%3D8%26spage%3D127%26epage%3D134%26doi%3D10.1093%2Fprotein%2F8.2.127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group">Merdes, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heald, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samejima, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Earnshaw, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cleveland, D. W.</span><span> </span><span class="NLM_article-title">Formation of Spindle Poles by Dynein/Dynactin-Dependent Transport of Numa</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">149</span><span class="NLM_x">, </span> <span class="NLM_fpage">851</span><span class="NLM_x">–</span> <span class="NLM_lpage">862</span><span class="refDoi"> DOI: 10.1083/jcb.149.4.851</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;key=10.1083%2Fjcb.149.4.851" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2000&pages=851-862&author=A.+Merdesauthor=R.+Healdauthor=K.+Samejimaauthor=W.+C.+Earnshawauthor=D.+W.+Cleveland&title=Formation+of+Spindle+Poles+by+Dynein%2FDynactin-Dependent+Transport+of+Numa&doi=10.1083%2Fjcb.149.4.851"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1083%2Fjcb.149.4.851&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.149.4.851%26sid%3Dliteratum%253Aachs%26aulast%3DMerdes%26aufirst%3DA.%26aulast%3DHeald%26aufirst%3DR.%26aulast%3DSamejima%26aufirst%3DK.%26aulast%3DEarnshaw%26aufirst%3DW.%2BC.%26aulast%3DCleveland%26aufirst%3DD.%2BW.%26atitle%3DFormation%2520of%2520Spindle%2520Poles%2520by%2520Dynein%252FDynactin-Dependent%2520Transport%2520of%2520Numa%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2000%26volume%3D149%26spage%3D851%26epage%3D862%26doi%3D10.1083%2Fjcb.149.4.851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZS0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZS0','PDB','4ZS0'); return false;">PDB: 4ZS0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2BHE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2BHE','PDB','2BHE'); return false;">PDB: 2BHE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZTQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZTQ','PDB','4ZTQ'); return false;">PDB: 4ZTQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZTR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZTR','PDB','4ZTR'); return false;">PDB: 4ZTR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZTS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZTS','PDB','4ZTS'); return false;">PDB: 4ZTS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JBQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JBQ','PDB','4JBQ'); return false;">PDB: 4JBQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JAI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JAI','PDB','4JAI'); return false;">PDB: 4JAI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2J5O" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2J5O','PDB','2J5O'); return false;">PDB: 2J5O</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2BMC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2BMC','PDB','2BMC'); return false;">PDB: 2BMC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3P9J" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3P9J','PDB','3P9J'); return false;">PDB: 3P9J</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R22" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R22','PDB','3R22'); return false;">PDB: 3R22</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FDN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FDN','PDB','3FDN'); return false;">PDB: 3FDN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3K5U" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3K5U','PDB','3K5U'); return false;">PDB: 3K5U</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BOG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BOG','PDB','4BOG'); return false;">PDB: 4BOG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X81" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X81','PDB','2X81'); return false;">PDB: 2X81</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1OL5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1OL5','PDB','1OL5'); return false;">PDB: 1OL5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C3P" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C3P','PDB','4C3P'); return false;">PDB: 4C3P</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-6"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i116"><a href="/doi/suppl/10.1021/acs.jmedchem.6b00709">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_62332"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b00709">10.1021/acs.jmedchem.6b00709</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Structures of references inhibitors used for xLOS virtual screening (Supplementary Figure 1), first round hits (Supplementary Figure 2) and second round hits (Supplementary Figure 3), ORTEP diagrams of <b>76</b> and <b>78</b> (Supplementary Figure 4), X-ray data collection table (Supplementary Table 1), Trp fluorescence data (Supplementary Figure 5), analysis of Aurora A conformations (Supplementary Figure 6), compound purities (Supplementary Table 2), and images of <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra for final compounds and selected intermediates (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_001.pdf">PDF</a>)</p></li><li><p class="inline">(<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_002.txt">TXT</a>)</p></li><li><p class="inline">(<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_003.txt">TXT</a>)</p></li><li><p class="inline">SMILES files of kinase screening set and second round LBVS annotated with PAINS flags (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_004.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_001.pdf">jm6b00709_si_001.pdf (9.0 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_002.txt">jm6b00709_si_002.txt (20.85 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_003.txt">jm6b00709_si_003.txt (13.45 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00709/suppl_file/jm6b00709_si_004.csv">jm6b00709_si_004.csv (3.4 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">4ZS0 (Aurora A complex with <b>6</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZTQ">4ZTQ</a> (Aurora A complex with <b>77</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZTR">4ZTR</a> (Aurora A complex with <b>9</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZTS">4ZTS</a> (Aurora A complex with <b>88</b>). Authors will release the atomic coordinates an experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b00709&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b00709%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-15%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b00709" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799c56109d63bec","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
