
<html lang="en"     class="pb-page"  data-request-id="b1f0d416-c534-44eb-8990-9cecb70fafb7"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2021.64.issue-10;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.0c02271;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Novel, Potent Inhibitors of Hydroxy Acid Oxidase 1 (HAO1) Using DNA-Encoded Chemical Library Screening" /></meta><meta name="dc.Creator" content="Esther C. Y.  Lee" /></meta><meta name="dc.Creator" content="Andrew J.  McRiner" /></meta><meta name="dc.Creator" content="Katy E.  Georgiadis" /></meta><meta name="dc.Creator" content="Julie  Liu" /></meta><meta name="dc.Creator" content="Zooey  Wang" /></meta><meta name="dc.Creator" content="Andrew D.  Ferguson" /></meta><meta name="dc.Creator" content="Benjamin  Levin" /></meta><meta name="dc.Creator" content="Moritz  von Rechenberg" /></meta><meta name="dc.Creator" content="Christopher D.  Hupp" /></meta><meta name="dc.Creator" content="Michael I.  Monteiro" /></meta><meta name="dc.Creator" content="Anthony D.  Keefe" /></meta><meta name="dc.Creator" content="Allison  Olszewski" /></meta><meta name="dc.Creator" content="Charles J.  Eyermann" /></meta><meta name="dc.Creator" content="Paolo  Centrella" /></meta><meta name="dc.Creator" content="Yanbin  Liu" /></meta><meta name="dc.Creator" content="Shilpi  Arora" /></meta><meta name="dc.Creator" content="John W.  Cuozzo" /></meta><meta name="dc.Creator" content="Ying  Zhang" /></meta><meta name="dc.Creator" content="Matthew A.  Clark" /></meta><meta name="dc.Creator" content="Christelle  Huguet" /></meta><meta name="dc.Creator" content="Anna  Kohlmann" /></meta><meta name="dc.Description" content="Inhibition of hydroxy acid oxidase 1 (HAO1) is a strategy to mitigate the accumulation of toxic oxalate that results from reduced activity of alanine–glyoxylate aminotransferase (AGXT) in primary h..." /></meta><meta name="Description" content="Inhibition of hydroxy acid oxidase 1 (HAO1) is a strategy to mitigate the accumulation of toxic oxalate that results from reduced activity of alanine–glyoxylate aminotransferase (AGXT) in primary h..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 6, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c02271" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02271" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c02271" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02271" /></link>
        
    
    

<title>Discovery of Novel, Potent Inhibitors of Hydroxy Acid Oxidase 1 (HAO1) Using DNA-Encoded Chemical Library Screening | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c02271" /></meta><meta property="og:title" content="Discovery of Novel, Potent Inhibitors of Hydroxy Acid Oxidase 1 (HAO1) Using DNA-Encoded Chemical Library Screening" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0015.jpeg" /></meta><meta property="og:description" content="Inhibition of hydroxy acid oxidase 1 (HAO1) is a strategy to mitigate the accumulation of toxic oxalate that results from reduced activity of alanine–glyoxylate aminotransferase (AGXT) in primary hyperoxaluria 1 (PH1) patients. DNA-Encoded Chemical Library (DECL) screening provided two novel chemical series of potent HAO1 inhibitors, represented by compounds 3–6. Compound 5 was further optimized via various structure–activity relationship (SAR) exploration methods to 29, a compound with improved potency and absorption, distribution, metabolism, and excretion (ADME)/pharmacokinetic (PK) properties. Since carboxylic acid-containing compounds are often poorly permeable and have potential active glucuronide metabolites, we undertook a brief, initial exploration of acid replacements with the aim of identifying non-acid-containing HAO1 inhibitors. Structure-based drug design initiated with Compound 5 led to the identification of a nonacid inhibitor of HAO1, 31, which has weaker potency and increased permeability." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c02271"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02271">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c02271&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c02271&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c02271&amp;href=/doi/10.1021/acs.jmedchem.0c02271" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 10</span><span class="cit-fg-pageRange">, 6730-6744</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/10" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c02202" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00023" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Novel, Potent Inhibitors of Hydroxy Acid Oxidase 1 (HAO1) Using DNA-Encoded Chemical Library Screening</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Esther C. Y. Lee</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Esther C. Y. Lee</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#c8adbbbca0adbae6a4adad88a9bbbcbaa9b2ada6adaba9e6aba7a5"><span class="__cf_email__" data-cfemail="e98c9a9d818c9bc7858c8ca9889a9d9b88938c878c8a88c78a8684">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Esther+C.+Y.++Lee">Esther C. Y. Lee</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0329-9186" title="Orcid link">http://orcid.org/0000-0003-0329-9186</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andrew J. McRiner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrew J. McRiner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrew+J.++McRiner">Andrew J. McRiner</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Katy E. Georgiadis</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Katy E. Georgiadis</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Katy+E.++Georgiadis">Katy E. Georgiadis</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Julie Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Julie Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Julie++Liu">Julie Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zooey Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zooey Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zooey++Wang">Zooey Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andrew D. Ferguson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrew D. Ferguson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrew+D.++Ferguson">Andrew D. Ferguson</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1018-9631" title="Orcid link">http://orcid.org/0000-0002-1018-9631</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Benjamin Levin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Benjamin Levin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Benjamin++Levin">Benjamin Levin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Moritz von Rechenberg</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Moritz von Rechenberg</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Moritz++von+Rechenberg">Moritz von Rechenberg</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christopher D. Hupp</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christopher D. Hupp</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christopher+D.++Hupp">Christopher D. Hupp</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michael I. Monteiro</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael I. Monteiro</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael+I.++Monteiro">Michael I. Monteiro</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anthony D. Keefe</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anthony D. Keefe</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#2e4f454b4b484b6e56034d464b435c56004d4143"><span class="__cf_email__" data-cfemail="86e7ede3e3e0e3c6feabe5eee3ebf4fea8e5e9eb">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anthony+D.++Keefe">Anthony D. Keefe</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Allison Olszewski</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Allison Olszewski</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Allison++Olszewski">Allison Olszewski</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Charles J. Eyermann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Charles J. Eyermann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Charles+J.++Eyermann">Charles J. Eyermann</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paolo Centrella</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paolo Centrella</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paolo++Centrella">Paolo Centrella</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yanbin Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yanbin Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yanbin++Liu">Yanbin Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shilpi Arora</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shilpi Arora</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shilpi++Arora">Shilpi Arora</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">John W. Cuozzo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">John W. Cuozzo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=John+W.++Cuozzo">John W. Cuozzo</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6229-0395" title="Orcid link">http://orcid.org/0000-0002-6229-0395</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ying Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ying Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ying++Zhang">Ying Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthew A. Clark</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew A. Clark</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+A.++Clark">Matthew A. Clark</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christelle Huguet</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christelle Huguet</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christelle++Huguet">Christelle Huguet</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Anna Kohlmann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anna Kohlmann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anna++Kohlmann">Anna Kohlmann</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02271&amp;href=/doi/10.1021%2Facs.jmedchem.0c02271" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 10</span><span class="cit-pageRange">, 6730–6744</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 6, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>31 December 2020</li><li><span class="item_label"><b>Published</b> online</span>6 May 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 27 May 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c02271" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02271</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6730%26pageCount%3D15%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DEsther%2BC.%2BY.%2BLee%252C%2BAndrew%2BJ.%2BMcRiner%252C%2BKaty%2BE.%2BGeorgiadis%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D10%26contentID%3Dacs.jmedchem.0c02271%26title%3DDiscovery%2Bof%2BNovel%252C%2BPotent%2BInhibitors%2Bof%2BHydroxy%2BAcid%2BOxidase%2B1%2B%2528HAO1%2529%2BUsing%2BDNA-Encoded%2BChemical%2BLibrary%2BScreening%26numPages%3D15%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6744%26publicationDate%3DMay%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c02271"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1431</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c02271" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Novel, Potent Inhibitors of Hydroxy Acid Oxidase 1 (HAO1) Using DNA-Encoded Chemical Library Screening&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Esther&quot;,&quot;last_name&quot;:&quot;C. Y. Lee&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;J. McRiner&quot;},{&quot;first_name&quot;:&quot;Katy&quot;,&quot;last_name&quot;:&quot;E. Georgiadis&quot;},{&quot;first_name&quot;:&quot;Julie&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Zooey&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;D. Ferguson&quot;},{&quot;first_name&quot;:&quot;Benjamin&quot;,&quot;last_name&quot;:&quot;Levin&quot;},{&quot;first_name&quot;:&quot;Moritz&quot;,&quot;last_name&quot;:&quot;von Rechenberg&quot;},{&quot;first_name&quot;:&quot;Christopher&quot;,&quot;last_name&quot;:&quot;D. Hupp&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;I. Monteiro&quot;},{&quot;first_name&quot;:&quot;Anthony&quot;,&quot;last_name&quot;:&quot;D. Keefe&quot;},{&quot;first_name&quot;:&quot;Allison&quot;,&quot;last_name&quot;:&quot;Olszewski&quot;},{&quot;first_name&quot;:&quot;Charles&quot;,&quot;last_name&quot;:&quot;J. Eyermann&quot;},{&quot;first_name&quot;:&quot;Paolo&quot;,&quot;last_name&quot;:&quot;Centrella&quot;},{&quot;first_name&quot;:&quot;Yanbin&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Shilpi&quot;,&quot;last_name&quot;:&quot;Arora&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;W. Cuozzo&quot;},{&quot;first_name&quot;:&quot;Ying&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;A. Clark&quot;},{&quot;first_name&quot;:&quot;Christelle&quot;,&quot;last_name&quot;:&quot;Huguet&quot;},{&quot;first_name&quot;:&quot;Anna&quot;,&quot;last_name&quot;:&quot;Kohlmann&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;06&quot;,&quot;issue&quot;:&quot;10&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;6730-6744&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c02271&quot;},&quot;abstract&quot;:&quot;Inhibition of hydroxy acid oxidase 1 (HAO1) is a strategy to mitigate the accumulation of toxic oxalate that results from reduced activity of alanine–glyoxylate aminotransferase (AGXT) in primary hyperoxaluria 1 (PH1) patients. DNA-Encoded Chemical Library (DECL) screening provided two novel chemical series of potent HAO1 inhibitors, represented by compounds 3–6. Compound 5 was further optimized via various structure–activity relationship (SAR) exploration methods to 29, a compound with improved potency and absorption, distribution, metabolism, and excretion (ADME)/pharmacokinetic (PK) properties. Since carboxylic acid-containing compounds are often poorly permeable and have potential active glucuronide metabolites, we undertook a brief, initial exploration of acid replacements with the aim of identifying non-acid-containing HAO1 inhibitors. Structure-based drug design initiated with Compound 5 led to the identification of a nonacid inhibitor of HAO1, 31, which has weaker potency and increased permeability.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02271&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02271" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02271&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02271" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02271&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02271" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02271&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02271&amp;href=/doi/10.1021/acs.jmedchem.0c02271" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c02271" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c02271" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c02271%26sid%3Dliteratum%253Aachs%26pmid%3D33955740%26genre%3Darticle%26aulast%3DLee%26date%3D2021%26atitle%3DDiscovery%2Bof%2BNovel%252C%2BPotent%2BInhibitors%2Bof%2BHydroxy%2BAcid%2BOxidase%2B1%2B%2528HAO1%2529%2BUsing%2BDNA-Encoded%2BChemical%2BLibrary%2BScreening%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D10%26spage%3D6730%26epage%3D6744%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291651" title="Anions">Anions</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292504" title="Permeability">Permeability</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/10" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/jmcmar.2021.64.issue-10/20210527/jmcmar.2021.64.issue-10.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/medium/jm0c02271_0015.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02271&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Inhibition of hydroxy acid oxidase 1 (HAO1) is a strategy to mitigate the accumulation of toxic oxalate that results from reduced activity of alanine–glyoxylate aminotransferase (AGXT) in primary hyperoxaluria 1 (PH1) patients. DNA-Encoded Chemical Library (DECL) screening provided two novel chemical series of potent HAO1 inhibitors, represented by compounds <b>3</b>–<b>6</b>. Compound <b>5</b> was further optimized via various structure–activity relationship (SAR) exploration methods to <b>29</b>, a compound with improved potency and absorption, distribution, metabolism, and excretion (ADME)/pharmacokinetic (PK) properties. Since carboxylic acid-containing compounds are often poorly permeable and have potential active glucuronide metabolites, we undertook a brief, initial exploration of acid replacements with the aim of identifying non-acid-containing HAO1 inhibitors. Structure-based drug design initiated with Compound <b>5</b> led to the identification of a nonacid inhibitor of HAO1, <b>31</b>, which has weaker potency and increased permeability.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05136" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05136" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Primary hyperoxaluria (PH) is a family of rare, genetic liver disorders characterized by the overproduction of oxalate, which is normally eliminated through the kidneys. The three known genetic types of PH result from mutations in three different genes in the glyoxylate metabolism pathway, AGXT (PH1), GRHPR (PH2), and HOGA1 (PH3). AGXT converts glyoxylate, the product of hydroxy acid oxidase 1 (HAO1), to glycine in the liver. Defective glyoxylate metabolism in liver peroxisomes of PH1 patients leads to accumulation of toxic levels of oxalate and recurring formation of kidney stones that result in progressive kidney deterioration, accumulation of oxalate in other organs, and ultimately end-stage renal disease.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> One therapeutic strategy to mitigate the accumulation of oxalate is via the inhibition of HAO1. Targeting HAO1 has been validated and shown to be safe in genetically modified mice.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> The sole phenotype of (<i>Hao1</i><sup>–<i>/</i>–</sup><i>)</i> mice is elevated levels of urine glycolate, whereas in double knockout mice (<i>Agxt1</i><sup>–<i>/</i>–</sup><i>, Hao1</i><sup>–<i>/</i>–</sup>), oxalate levels were decreased compared to single knockout (<i>Agxt1</i><sup>–<i>/</i>–</sup>) hyperoxaluric mice. Additionally, a small-molecule inhibitor of HAO1, 4-carboxy-5-[(4-chlorophenyl)sulfanyl]-1,2,3-thiadiaozle (<b>CCPST</b>),<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> was shown to reduce the oxalate generation in <i>Agxt1</i><sup>–<i>/</i>–</sup> mouse hepatocytes as well as in <i>Agxt1</i><sup>–<i>/</i>–</sup> mice at high doses.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a></div><div class="NLM_p">Currently, there are no approved treatments for PH1 leaving patients in need of both liver and kidney transplants. However, Alnylam Pharmaceuticals<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> has an siRNA therapy targeting <i>HAO1</i> in Phase 3 clinical evaluation (lumasiran) and Dicerna Pharmaceuticals<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> has an siRNA therapy in Phase 2 clinical trials that target PH1 and PH2 (nedosiran). The results so far look encouraging with normalization or near-normalization of urinary oxalate levels in the majority of participants and favorable safety and tolerability profiles after a single dose. In 2019, patent applications published describing small-molecule inhibitors of HAO1 from Orfan Biotech,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Biomarin Pharmaceuticals,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> and OxaluRx,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> all sharing the common chemical feature of an aryl carboxylic acid similar to <b>CCPST</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). <b>CCPST</b>, as previously noted, is able to lower oxalate <i>in vivo</i> only at very high doses due to its modest potency (mHAO1 IC<sub>50</sub> = 43–198 μM).<a onclick="showRef(event, 'ref2 ref11'); return false;" href="javascript:void(0);" class="ref ref2 ref11">(2,11)</a> At such high doses, treatment could be complicated by unwanted side effects. This limitation has prompted others to focus on optimizing the potency of <b>CCPST</b>. Other known inhibitors also have an acidic functionality, for example <b>CDST</b> (HAO1 IC<sub>50</sub> not reported),<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> hydroxy pyrrole dione, <b>1</b> (porcineHAO1 IC<sub>50</sub> = 87 nM),<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and a more recent salicylic acid derivative, <b>2</b> (mHAO1 IC<sub>50</sub> = 4.4 μM).<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> As shown in the crystal structures of <b>CCPST</b><a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> and <b>CDST</b>,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> the acid moiety makes interactions with Arg167, Arg263, and Tyr26, similar to the natural substrate, glyoxylate.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/medium/jm0c02271_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. HAO1 inhibitors: <b>CCPST</b>,<a onclick="showRef(event, 'ref2 ref3 ref11'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref11">(2,3,11)</a><b>CDST</b>,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Compound <b>1</b>,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and Compound <b>2</b>.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02271&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In our efforts to identify a small-molecule inhibitor of HAO1, we applied our DNA-Encoded Chemical Library (DECL) platform, which has been an attractive strategy to provide small molecule leads for a wide range of targets.<a onclick="showRef(event, 'ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20 ref21">(12−21)</a> This platform allows the simultaneous screening of billions of diverse compounds.<a onclick="showRef(event, 'ref22 ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref24 ref25 ref26">(22−26)</a> Additionally, known binders can be incorporated into the screens to help understand the binding modes of identified hits. In this paper, we report the discovery of potent, novel, carboxylic acid-containing HAO1 inhibitors via a DECL screen, their absorption, distribution, metabolism, and excretion (ADME) optimization, and structural understanding of protein–ligand interactions. In addition, initial efforts to scaffold hop using the structural information led to the identification of a novel, noncarboxylic acid chemotype hit.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66757" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66757" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Affinity-Mediated Selection of DNA-Encoded Chemical Libraries</h3><div class="NLM_p">Our collection of DNA-encoded chemical libraries were combined and incubated with no target, human HAO1 (hHAO1) protein at 7.7 or 1.5 μM, or the off-target LDHB (lactate dehydrogenase B) at 9.1 μM. An additional parallel selection condition of HAO1 protein with Compound <b>1</b> was performed to identify competitive features. Proteins and associated library members were captured using an anti-C-Tag matrix followed by stringent washing and elution at 85 °C. The methodology was similar to as described previously by X-Chem.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> The C-Tag is a short peptide sequence (glutamate–proline–glutamate–alanine) for which immobilized antibody affinity matrices are commercially available (CaptureSelect XL, Thermo Fisher Scientific).</div><div class="NLM_p">Individual sequence reads were translated back into the corresponding building block and library schemes and statistical prevalence data were calculated for all building block combinations in each library across all selection conditions. A large number of strongly enriched building block combinations were observed across all libraries. During analysis, our attention was drawn to clusters of co-enriched building block combinations in three different libraries. Compound <b>3</b>, which subsequently became the founding member of Series 1, was found in Library A. Library A was synthesized from a diverse set of aryl bromo amines, carboxylates, and boronates (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). This library is comprised of 283 million different three-cycle compounds, each of which is identifiable using the sequence of the unique oligonucleotide tag combination to which it is covalently attached. The cluster of related compounds that enriched from this library was defined by a range of substituted 2-bromobenzylamines, 6-chloroindole-2-carboxylic acid, and a 4-carboxy-3-chlorophenylboronate. These building block combinations were observed to have enriched in both the 7.7 and 1.5 μM HAO1 selection conditions to an approximately equal extent, which is consistent with a <i>K</i><sub>D</sub> value of <150 nM. No enrichment was observed to occur for this same building block combination as a result of selection in the presence of HAO1 and a saturating concentration of Compound <b>1</b>, consistent with the hypothesis that these enriched library compounds bind to HAO1 in a manner that is competitive with Compound <b>1</b>. Furthermore, no enrichment was observed for this building block combination against the off-target LDHB. Assembly of the most enriched building block combination provided Compound <b>3</b> for which the predictions were confirmed. Library B also attracted our attention during selection output data analysis. Library B was formed from primary amines and carboxylates/aldehydes (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Similar to the observations made in Library A, a cluster of co-enriched building blocks in Library B were enriched in selection with both 7.7 and 1.5 μM HAO1, and no enrichment was observed to occur for the same building block combination as a result of selection in the presence of HAO1 and a saturating concentration of Compound <b>1</b>, nor against the off-target LDHB. This cluster of structurally related compounds enriched in Library B was defined by a single enriched building block at the first diversity point, 5-aminoindazole-3-carboxylic acid for which no similar building blocks were included in the library (ECFP6 Tanimoto Distance >0.65). A range of structurally related building blocks co-enriched at the second diversity point including phenyls and pyridines with 2-substituents, many of which were ethers. The most highly enriched non-nitro-containing combination provided Compound <b>4</b>. The defining building block of this cluster of compounds, the 1<i>H</i>-indazole-3-carboxylate, was also found to have enriched in an independently synthesized library, Library C. Library C was synthesized using a diverse set of formyl acids and amines (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) and is comprised of 740 000 different compounds. Co-enriched building blocks at the second diversity element were comprised of a range of metasubstituted formyl benzoic acids or biaryl carboxylic acids. The synthesis of the most highly enriched combination of building blocks from this cluster provided Compounds <b>5</b> and <b>6</b>. For each of the enriched clusters containing the compounds that defined Series 1 and 2, “structure–activity relationship (SAR)” or unique preference was evident at every diversity point. We interpreted this to indicate that none of the diversity points could be removed while retaining activity, a prediction that was subsequently determined to be reliable. The methyl group on the amide of Compound <b>3</b>, the methyl group on the tertiary aromatic amine of Compound <b>4</b>, and a methyl group on the dimethylamino oxadiazole ring of Compounds <b>5</b> and <b>6</b> represent the DNA linker attachment points. Compounds <b>3</b>–<b>6</b> were the most potent screening hits to be discovered in this project. Target engagement was confirmed for both series using SPR (BIAcore) with Compound <b>5</b> determined to have a <i>K</i><sub>D</sub> of 160 nM with 1:1 binding of FMN (Flavin mononucleotide)-loaded HAO1 and a variant of Compound <b>3</b> with a methyl group replacing the biphenyl chlorine to have a <i>K</i><sub>D</sub> of 7 μM also with 1:1 binding (data not shown).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/medium/jm0c02271_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Hit Compound <b>3</b> (founding member of Series 1) and Compounds <b>4–6</b> (founding members of Series 2) and the DNA-encoded chemical libraries within which they were found.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02271&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Synthesis of Off-DNA Compounds</h3><div class="NLM_p">The synthesis of Compound <b>3</b> was initiated first by amide coupling of 1-(2-bromo-4-fluorophenyl)-<i>N</i>-methylmethanamine(I) and 6-chloro-1<i>H</i>-indole-2-carboxylic acid to provide <i>N</i>-(2-bromo-4-fluorobenzyl)-6-chloro-<i>N</i>-methyl-1<i>H</i>-indole-2-carboxamide(II) (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Subsequent Suzuki coupling of <b>II</b> with 4-borono-2-chlorobenzoic acid afforded Compound <b>3</b>. The synthesis of representative Series 2 compounds (<b>4</b>–6) is shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Compound <b>4</b> was synthesized via reductive amination of 2-propoxynicotinaldehyde(III) and 5-amino-1<i>H</i>-indazole-3-carboxylic acid with sodium borohydride in acetic acid/methanol. Compounds <b>5</b> and <b>6</b> were synthesized starting from oxadiazole <b>V</b>, which is formed via reaction of 3-hydroxy-1,1-dimethylguanidine(IV) and 3-formyl-2-hydroxybenzoic acid in the presence of hexafluorophosphate azabenzotriazole tetramethyl uranium (HATU), sodium bicarbonate in dimethylformamide. Reductive amination of <b>V</b> with 5-amino-1<i>H</i>-indazole-3-carboxylic acid and sodium borohydride in acetic acid/methanol afforded Compound <b>5</b>. Further treatment with paraformaldehyde provided the <i>N</i>-methylated compound <b>6</b>.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/medium/jm0c02271_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0008.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compound <b>3</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02271&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (a) 6-chloro-1<i>H</i>-indole-2-carboxylic acid, NaHCO<sub>3</sub>, and HATU; (b) 4-borono-2-chlorobenzoic acid, Pd(dppf)Cl<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O,100 °C, and 1 h.</p></p></figure><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/medium/jm0c02271_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0009.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>4</b>–<b>6</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02271&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (a) substituted 3-formyl-2-hydroxybenzoic acid, HATU, NaHCO<sub>3</sub>, and <i>N</i>,<i>N</i>-dimethylformamide (DMF); (b) 5-amino-1<i>H</i>-indazole-3-carboxylic acid, NaBH<sub>3</sub>CN, AcOH, and MeOH; and (c) paraformaldehyde.</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Initial Hit Expansion</h3><div class="NLM_p">Compounds were evaluated in a fluorescence-based HAO1 activity assay that determines the inhibition of hHAO1 activity via detection of the product, hydrogen peroxide, by a coupled horseradish peroxidase (HRP) reaction. Some compounds were also tested in an HRP interference assay (same as the former but without HAO1 protein) to confirm that the inhibitory activity of compounds was specific to HAO1 and was not an artifact. Compounds <b>3–6</b> were confirmed to have functional activity. Biaryl acid Compound <b>3</b> from Series 1 gave an IC<sub>50</sub> = 110 nM. From Series 2, Compound <b>4</b> was less potent (HAO1 IC<sub>50</sub> = 510 nM) whereas Compounds <b>5</b> and <b>6</b> were more potent (HAO1 IC<sub>50</sub> = 37 and 21 nM, respectively). In the HRP interference assay, all four compounds showed no signal (IC<sub>50</sub> > 30 μM for Compounds <b>3</b>, <b>4</b>, and <b>6</b>; IC<sub>50</sub> > 13 μM for Compound <b>5</b>). Compounds <b>5</b> and <b>6</b> from Series 2 were less lipophilic and had lower molecular weights making them more ligand efficient than Compound <b>3</b> (LE = 0.36 (Compounds <b>5</b> and <b>6</b>) vs 0.30 (Compound <b>3</b>)). Initial assessment of Series 1 tractability with close analogue designs resulted in large drop-offs in activity and given its higher molecular weight, we decided to focus our optimization efforts on Series 2.</div><div class="NLM_p">In the absence of a crystal structure at this point in the project, we first assessed the tractability of Series 2 with more drastic modifications to understand which parts of the molecule were driving potency. For ease of synthesis, we chose to perform initial modifications from Compound <b>5</b> rather than the <i>N</i>-methylated <b>6</b>. Published structures of HAO1 with glyoxylate and inhibitors indicate that carboxylic acid engages in a key protein–ligand interaction in the binding site, and therefore is likely indispensable.<a onclick="showRef(event, 'ref3 ref9'); return false;" href="javascript:void(0);" class="ref ref3 ref9">(3,9)</a> This was confirmed by removal of the carboxylic functionality or replacement with an amide, which resulted in a reduction in potency by 324- and 119-fold, respectively (Compounds <b>8</b> and <b>9</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Furthermore, removal of the entire indazole acid led to the loss of all detectable activity (Compound <b>7</b>). Changing indazole acid to indole acid also led to an 108-fold loss in potency (Compound <b>10</b>). Interestingly, modifying the benzylic amine to an ether or removal of the biaryl portion of the molecule also led to a complete loss in potency (Compounds <b>11</b> and <b>12</b>). Deletion of the oxadiazole moiety (Compounds <b>13</b> and <b>14</b>) led to 595- and 460-fold reduction, respectively. Unexpectedly, replacing the oxadiazole with a dimethyl amide isostere also led to significant potency loss (Compound <b>15</b>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Tractability Assessment of Series 2</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/medium/jm0c02271_0010.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Determined from at least two replicates.</p></div></div><div></div></div><div class="NLM_p">Concurrently, we obtained various <i>in vitro</i> ADME data for Compound <b>5</b>. Passive permeability (<i>P</i><sub>app</sub>) was measured in the wild-type Madin–Darby canine kidney (MDCK) assay, kinetic solubility was assessed at both pH 2 and 7.4, and both plasma protein binding (PPB) and microsomal stability were analyzed for both human and mouse species. The permeability for compound <b>5</b> was low (<i>P</i><sub>app</sub><i>A</i>–B = 0.82 × 10<sup>–6</sup> cm/s) and the PPB was high for both species (% bound 99.98 and 99.53% in human and mouse, respectively) (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Solubility was high at pH = 7.4 as the acid is ionized, but low at pH = 2. Clearance was low in both human and mouse (CL<sub>int</sub> = 6.9 and 29.6 mL/min/kg), respectively. Evaluation of calculated physicochemical properties showed that the compounds were very polar with low lipophilicity, high topical polar surface area (tPSA), and low <i>c</i> log <i>P</i> and <i>c</i> log <i>D</i>. With the ADME data, our first priority was to improve permeability while maintaining reasonable metabolic stability.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Subtle Modifications to Compound <b>5</b> and Their <i>In Vitro</i> ADME Parameters</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/medium/jm0c02271_0011.gif" alt="" id="fx2" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">R<sub>1</sub></th><th class="colsep0 rowsep0" align="center">R<sub>2</sub></th><th class="colsep0 rowsep0" align="center">R<sub>3</sub></th><th class="colsep0 rowsep0" align="center">R<sub>4</sub></th><th class="colsep0 rowsep0" align="center" char=".">HAO1 IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="."><i>c</i> log <i>P</i><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char="."><i>c</i> log <i>D</i><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">tPSA<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center"><i>P</i><sub>app</sub> (<i>A</i>–<i>B</i>) (×10<sup>–6</sup> cm/s)<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">sol (pH 2/7.4) (μM)<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center">h/m PPB (% bound)<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center">h/m LM CL<sub>int</sub> (mL/min/kg)<a class="ref internalNav" href="#t2fn6" aria-label="f">f</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">37</td><td class="colsep0 rowsep0" align="char" char=".">2.2</td><td class="colsep0 rowsep0" align="char" char=".">–1.0</td><td class="colsep0 rowsep0" align="char" char=".">131</td><td class="colsep0 rowsep0" align="left">0.82</td><td class="colsep0 rowsep0" align="left">0.4/157</td><td class="colsep0 rowsep0" align="left">99.98/99.53</td><td class="colsep0 rowsep0" align="left">6.9/28.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">11 000</td><td class="colsep0 rowsep0" align="char" char=".">2.9</td><td class="colsep0 rowsep0" align="char" char=".">2.6</td><td class="colsep0 rowsep0" align="char" char=".">120</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">4200</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="char" char=".">–0.9</td><td class="colsep0 rowsep0" align="char" char=".">122</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">2100</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="char" char=".">–0.6</td><td class="colsep0 rowsep0" align="char" char=".">120</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">170</td><td class="colsep0 rowsep0" align="char" char=".">2.4</td><td class="colsep0 rowsep0" align="char" char=".">–0.4</td><td class="colsep0 rowsep0" align="char" char=".">111</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">308/255</td><td class="colsep0 rowsep0" align="left">99.12/98.95</td><td class="colsep0 rowsep0" align="left">0.5/2.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">180</td><td class="colsep0 rowsep0" align="char" char=".">3.1</td><td class="colsep0 rowsep0" align="char" char=".">0.2</td><td class="colsep0 rowsep0" align="char" char=".">111</td><td class="colsep0 rowsep0" align="left">0.7</td><td class="colsep0 rowsep0" align="left">7/296</td><td class="colsep0 rowsep0" align="left">99.8/99.35</td><td class="colsep0 rowsep0" align="left">5.8/12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">3.1</td><td class="colsep0 rowsep0" align="char" char=".">–0.3</td><td class="colsep0 rowsep0" align="char" char=".">122</td><td class="colsep0 rowsep0" align="left">3.4</td><td class="colsep0 rowsep0" align="left">16/135</td><td class="colsep0 rowsep0" align="left">>99.98/99.91</td><td class="colsep0 rowsep0" align="left">14/46</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Determined from at least two replicates.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Calculated from ChemAxon.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Passive permeability (<i>P</i><sub>app</sub>) from apical to basolateral (AB) from Madin–Darby canine kidney (MDCK) cells.</p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last">Kinetic solubility was determined at pH2 and pH7.4.</p></div><div class="footnote" id="t2fn5"><sup><sup>e</sup></sup><p class="last">Plasma protein binding (PPB) was analyzed via the Equilibrium Dialysis Method.</p></div><div class="footnote" id="t2fn6"><sup><sup>f</sup></sup><p class="last">Metabolic stability was measured in pooled human and male mouse liver microsomes (LM).</p></div><div class="footnote" id="t2fn7"><sup><sup>g</sup></sup><p class="last">ND: not determined.</p></div></div></div><div class="NLM_p">A general tactic to improve the permeability of very polar compounds (high tPSA and low <i>c</i> log <i>D</i> and <i>c</i> log <i>P</i>) is to remove unnecessary hydrogen-bond donors to lower the tPSA and at the same time increase lipophilicity. Methylation of carboxylic acid <b>15</b> to give ester <b>16</b> led to an expected decrease in potency (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Methylation of indazole (<b>17</b>) and phenol (<b>18</b>) also led to a decrease in potency by 113- and 57-fold, respectively. Removing phenolic OH (<b>19</b>) or replacing it with a chlorine (<b>20</b>) led to a 5-fold loss in potency in both instances. However, for compounds that retained potency the change in tPSA (from 131 to 111) and <i>c</i> log <i>P</i> (from 2.2 to 2.4 (<b>19</b>) or 3.1 (<b>20</b>)) did not lead to improved permeability compared to parent Compound <b>5</b>. Interestingly, the des-hydroxy Compound <b>19</b> provided very low human and mouse microsomal clearance (CL<sub>int</sub> = 0.5 and 2.9 mg/min/kg), respectively, as well as high solubility at both low and high pH. Initially synthesized <i>N</i>-methylated Compound <b>6</b> had similar potency to the des-methyl <b>5</b>; however, permeability was much better for this <i>N</i>-methylated analogue <b>6</b> (<i>P</i><sub>app</sub><i>A</i>–<i>B</i> = 3.4 vs 0.82 × 10<sup>–6</sup> cm/s). A further extension to ethyl (<b>32</b>) and isopropyl groups (<b>33</b>) provided similar potency (HAO1 IC<sub>50</sub> = 24 and 35 nM, respectively); however, permeability decreased as the substituent size increased (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02271/suppl_file/jm0c02271_si_001.pdf" class="ext-link">Table S1</a>). Isopropyl <b>33</b> also had very high mouse microsomal clearance (approximately 11-fold higher) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02271/suppl_file/jm0c02271_si_001.pdf" class="ext-link">Table S1</a>). Larger groups such as phenethyl (<b>34</b>) led to a decrease in potency by 6-fold and also very poor permeability, as might be expected when the tPSA is maintained and the MW is increased (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02271/suppl_file/jm0c02271_si_001.pdf" class="ext-link">Table S1</a>).</div><div class="NLM_p">Then, we evaluated the oxadiazole portion of Compound <b>5</b>. Replacing dimethylamine with a methyl group (<b>35</b>) afforded similar potency and permeability to Compound <b>5</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02271/suppl_file/jm0c02271_si_001.pdf" class="ext-link">Table S2</a>). Cyclopropyl oxadiazole <b>36</b> and trifluoromethyl oxadiazole <b>37</b> both led to a decrease in potency by 4-fold with a slight increase in permeability (<i>P</i><sub>app</sub><i>A</i>–<i>B</i> = 2.8 × 10<sup>–6</sup> and 1.9 × 10<sup>–6</sup> cm/s, respectively) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02271/suppl_file/jm0c02271_si_001.pdf" class="ext-link">Table S2</a>). Isopropyl oxadiazole <b>21</b> maintained potency and improved permeability (<i>P</i><sub>app</sub><i>A</i>–<i>B</i> = 3.7 × 10<sup>–6</sup> cm/s) (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). However, further modifications on this scaffold could not improve both potency and permeability (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02271/suppl_file/jm0c02271_si_001.pdf" class="ext-link">Table S3</a>).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Replacement of Oxadiazole Ring<a class="ref internalNav" href="#t2fn7" aria-label="g">g</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/medium/jm0c02271_0012.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/medium/jm0c02271_0013.gif" alt="" id="fx4" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Determined from at least two replicates.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Calculated from ChemAxon.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">Passive permeability (<i>P</i><sub>app</sub>) from apical to basolateral (AB) from Madin–Darby canine kidney (MDCK) cells.</p></div><div class="footnote" id="t3fn4"><sup><sup>d</sup></sup><p class="last">Kinetic solubility was determined at pH2 and pH7.4.</p></div><div class="footnote" id="t3fn5"><sup><sup>e</sup></sup><p class="last">Plasma protein binding (PPB) was analyzed via the Equilibrium Dialysis Method.</p></div><div class="footnote" id="t3fn6"><sup><sup>f</sup></sup><p class="last">Metabolic stability was measured in pooled human and male mouse liver microsomes (LM).</p></div><div class="footnote" id="t3fn7"><sup><sup>g</sup></sup><p class="last">ND: not determined.</p></div></div><div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Heterocycle SAR</h3><div class="NLM_p">As we were not able to obtain both improved potency and permeability with the oxadiazole moiety, we chose to evaluate other five-membered heterocycles. Pyrazole <b>22</b> provided an 18-fold loss in potency, while both oxazole <b>23</b> and thiazole <b>24</b> exhibited similar potencies (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). The permeability of oxazole <b>23</b> was reasonable (<i>P</i><sub>app</sub><i>A</i>–<i>B</i> = 3.6 × 10<sup>–6</sup> cm/s); however, the mouse microsomal clearance was higher (CL<sub>int</sub> = 84 vs 38 mL/min/kg) compared to <b>21</b>. In the case of thiazole <b>24</b>, the permeability was moderate but the microsomal clearance in both human and mouse was low (CL<sub>int</sub> = 10 and 8 mL/min/kg, respectively).</div><div class="NLM_p">Since thiazole <b>24</b> had a good balance of potency and physicochemical properties, we decided to further investigate additional thiazole modifications. Methylation of the benzylic nitrogen led to a similarly potent compound (<b>25</b>); however, unlike the oxadiazole series, permeability decreased significantly (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Interestingly, when we remove the phenolic OH while keeping methylated benzyl nitrogen (<b>26</b>), we observed a notable increase in permeability albeit at the cost of potency. Then, we wanted to evaluate varying the substituents on thiazole. We assessed both mono- (<b>27</b>) and dimethyl thiazole (<b>29</b>) (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Both compounds were very potent (HAO1 IC<sub>50</sub> = 23 and 25 nM, respectively) and had good permeability (<b>27</b>, <i>P</i><sub>app</sub><i>A</i>–B = 4.5 × 10<sup>–6</sup> cm/s and <b>29</b>, <i>P</i><sub>app</sub><i>A</i>–B= 3.8 × 10<sup>–6</sup> cm/s). Methylating the benzylic amine and removal of phenolic OH led to compounds <b>28</b> and <b>30</b> with a 26- and 12-fold loss in potency, respectively. However, Compound <b>30</b> demonstrated the best permeability overall (<i>P</i><sub>app</sub><i>A</i>–B = 8.3 × 10<sup>–6</sup> cm/s), consistent with the trend observed in the isopropy<i>l</i> thiazole SAR (Compound <b>26</b>).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Thiazole Optimization<a class="ref internalNav" href="#t4fn7" aria-label="g">g</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/medium/jm0c02271_0014.gif" alt="" id="fx5" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">R<sub>1</sub></th><th class="colsep0 rowsep0" align="center">R<sub>2</sub></th><th class="colsep0 rowsep0" align="center">R<sub>3</sub></th><th class="colsep0 rowsep0" align="center">R<sub>4</sub></th><th class="colsep0 rowsep0" align="center" char=".">HAO1 IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="."><i>c</i> log <i>P</i><a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char="."><i>c</i> log <i>D</i><a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">tPSA<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center"><i>P</i><sub>app</sub> (<i>A</i>–<i>B</i>) (×10<sup>–6</sup> cm/s)<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">sol (pH 2/7.4) (μM)<a class="ref internalNav" href="#t4fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center">h/m PPB (% bound)<a class="ref internalNav" href="#t4fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center">h/mLM CL<sub>int</sub> (mL/min/kg)<a class="ref internalNav" href="#t4fn6" aria-label="f">f</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">iPr</td><td class="colsep0 rowsep0" align="char" char=".">65</td><td class="colsep0 rowsep0" align="char" char=".">3.7</td><td class="colsep0 rowsep0" align="char" char=".">0.8</td><td class="colsep0 rowsep0" align="char" char=".">106</td><td class="colsep0 rowsep0" align="left">1.5</td><td class="colsep0 rowsep0" align="left">97/102</td><td class="colsep0 rowsep0" align="left">99.97/99.96</td><td class="colsep0 rowsep0" align="left">10/8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">iPr</td><td class="colsep0 rowsep0" align="char" char=".">35</td><td class="colsep0 rowsep0" align="char" char=".">4.6</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td><td class="colsep0 rowsep0" align="char" char=".">98</td><td class="colsep0 rowsep0" align="left"><0.05</td><td class="colsep0 rowsep0" align="left">175/6</td><td class="colsep0 rowsep0" align="left">>99.96 (m)</td><td class="colsep0 rowsep0" align="left">13 (m)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">iPr</td><td class="colsep0 rowsep0" align="char" char=".">710</td><td class="colsep0 rowsep0" align="char" char=".">4.6</td><td class="colsep0 rowsep0" align="char" char=".">1.8</td><td class="colsep0 rowsep0" align="char" char=".">77</td><td class="colsep0 rowsep0" align="left">8.5</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="char" char=".">23</td><td class="colsep0 rowsep0" align="char" char=".">2.4</td><td class="colsep0 rowsep0" align="char" char=".">–0.5</td><td class="colsep0 rowsep0" align="char" char=".">106</td><td class="colsep0 rowsep0" align="left">4.5</td><td class="colsep0 rowsep0" align="left">5/44</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">34 (m)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="char" char=".">650</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">77</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="char" char=".">25</td><td class="colsep0 rowsep0" align="char" char=".">3.0</td><td class="colsep0 rowsep0" align="char" char=".">0.2</td><td class="colsep0 rowsep0" align="char" char=".">106</td><td class="colsep0 rowsep0" align="left">3.8</td><td class="colsep0 rowsep0" align="left">20/15</td><td class="colsep0 rowsep0" align="left">>99.95*/>99.96</td><td class="colsep0 rowsep0" align="left">21/15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="char" char=".">290</td><td class="colsep0 rowsep0" align="char" char=".">3.8</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="char" char=".">77</td><td class="colsep0 rowsep0" align="left">8.3</td><td class="colsep0 rowsep0" align="left">276/254</td><td class="colsep0 rowsep0" align="left">99.88 (m)</td><td class="colsep0 rowsep0" align="left">45 (m)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Determined from at least two replicates.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">Calculated from ChemAxon.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">Passive permeability (<i>P</i><sub>app</sub>) from apical to basolateral (AB) from Madin–Darby canine kidney (MDCK) cells.</p></div><div class="footnote" id="t4fn4"><sup><sup>d</sup></sup><p class="last">Kinetic solubility was determined at pH 2 and pH 7.4.</p></div><div class="footnote" id="t4fn5"><sup><sup>e</sup></sup><p class="last">Plasma protein binding (PPB) was analyzed via the Equilibrium Dialysis Method.</p></div><div class="footnote" id="t4fn6"><sup><sup>f</sup></sup><p class="last">Metabolic stability was measured in pooled human and male mouse liver microsomes (LM).</p></div><div class="footnote" id="t4fn7"><sup><sup>g</sup></sup><p class="last">ND: not determined.</p></div></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Cell-Based Phenotypic Rescue Assay</h3><div class="NLM_p">Compound <b>5</b> and our most potent compounds with improved permeability (<b>6</b>, <b>21</b>, and <b>29</b>; along with literature compounds, <b>CCPST</b> and <b>1</b>) were evaluated by measuring percent rescue of viability in Chinese Hamster Ovary (CHO) cells that overexpress hHAO1 (CHO-GO cells).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Addition of glycolic acid to CHO-GO cells results in the production of toxic levels of oxalate in the cytosol leading to cell death. We demonstrated the utility of using these cells to evaluate HAO1 inhibition by showing that reduction of h<i>HAO1</i> expression with HAO1-degrading siRNAs restored viability in CHO-GO cells treated with glycolic acid (85% cell rescue, <a href="/doi/suppl/10.1021/acs.jmedchem.0c02271/suppl_file/jm0c02271_si_001.pdf" class="ext-link">Figure S2</a>). HAO1 inhibition by these siRNAs resulted in >80% reduction of hHAO1 protein levels in CHO-GO cells relative to negative control siRNAs. CHO-WT cells were unaffected by the inhibition of hHAO1 using these siRNAs. Since the level of hHAO1 overexpression in CHO-GO cells is high (approximately 20 μM hHAO1 protein, based on the specific activity of hHAO1 measured in CHO-GO cell lysates, data not shown), and the siRNA results indicated that near-complete inhibition would be required for cell rescue, we expected a significant reduction in the potency of our inhibitors in the cell-rescue assay as compared to the biochemical assay. Despite the high stringency of the assay, we were excited to see that our potent compounds <b>5</b>, <b>6</b>, <b>21</b>, and <b>29</b> all showed at least 50% rescue at 30 μM (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). In comparison, <b>CCPST</b> was inactive and Compound <b>1</b> exhibited a 3.4% rescue at the same concentration. Compound <b>5</b> was further evaluated at multiple doses (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). When the cells were treated with Compound <b>5</b> alone, no cell death was observed, demonstrating that it had no deleterious effect on viability. Glycolic acid treatment of CHO-GO cells results in cell death. With increasing concentrations of Compound <b>5</b>, there is a dose-dependent rescue of viability, wherein Compound <b>5</b> demonstrates an EC<sub>50</sub> of 17 μM. No effect on viability was observed in CHO-WT cells upon treatment with any of these inhibitors (data not shown).</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/medium/jm0c02271_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cell-rescue assay of Compound <b>5</b> at various doses. (CHO-GO cells were plated in 96-well plates, preincubated with compound at different concentrations followed by glycolic acid (GA) treatment. Viability was monitored using live cells imager (IncuCyte Zoom). Each data point represents an average of three wells).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02271&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Cellular Rescue Assay for Compounds <b>CCPST</b>, <b>1</b>, <b>5</b>, <b>6</b>, <b>21</b>, and <b>29</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">HAO1 IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center">cell rescue % at 30 μM<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CCPST</td><td class="colsep0 rowsep0" align="left">>30 000</td><td class="colsep0 rowsep0" align="left">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">190</td><td class="colsep0 rowsep0" align="left">3.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">37</td><td class="colsep0 rowsep0" align="left">82</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">21</td><td class="colsep0 rowsep0" align="left">57</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">47</td><td class="colsep0 rowsep0" align="left">82</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left">25</td><td class="colsep0 rowsep0" align="left">76</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">Cell-rescue values represent the average of at least <i>n</i> = 3 separate assays (except for compound <b>29</b>, <i>n</i> = 1; CCPST, <i>n</i> = 2).</p></div></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> X-ray Crystallography and Structural Analysis</h3><div class="NLM_p">During the course of SAR optimization, we obtained the X-ray crystal structures of several analogues in complex with hHAO1 (Compound <b>3</b>, Compound <b>4</b>, and Compound <b>5</b>; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W45">6W45</a> (Compound <b>3</b>, representing Series 1), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W44">6W44</a> (Compound <b>4</b>), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W4C">6W4C</a> (Compound <b>5</b>), both of which represent Series 2) (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). All three compounds bind in the active site of HAO1, occupying the glycolate/glyoxylate binding site and engaging the same amino acid residues. Although all three compounds contain a carboxylic acid, their binding modes differ slightly: the Series 2 compounds form a salt bridge with Arg167 and Arg263; Series 1 Compound <b>3</b> forms hydrogen bonds with His260 and Tyr132 and connects to Arg167 and Arg263 through an extensive water-mediated hydrogen-bonding network. The differences in the left-hand portion of these molecules highlight the flexibility in the binding pocket of HAO1. The conformational plasticity in loop 4 (also known as the gating loop) as well as the side chain of Trp110 has been previously described.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> It was observed that the Trp110 side chain can assume alternative conformations to accommodate inhibitors of varying sizes. Furthermore, it has been proposed that motion of the Trp110 side chain drives loop movement via a series of concerted interactions with other residues in the binding site.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> In all three structures, a large section of loop 4 is disordered, suggesting that the loop is in an open conformation as the closed conformation would be incompatible with the presence of the bound inhibitor. The side chain of Trp110 samples both closed (Compound <b>5</b>) and open (Compound <b>4</b>) conformations.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/medium/jm0c02271_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Overlay of Compound <b>3</b> PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W45">6W45</a> (green), Compound <b>4</b> PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W44">6W44</a> (pink), and Compound <b>5</b> PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W4C">6W4C</a> (gray). Inhibitor binding is dependent on the presence of the cofactor FMN. Trp110 side chain can assume different conformations to accommodate different inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02271&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The X-ray crystal structure of Compound <b>5</b> sheds light on previously observed SAR. The indazole moiety stacks with the FMN cofactor (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). The N1 of the indazole is within hydrogen-bonding distance of the N2 of His260. This is consistent with the observation that the methylation of indazole N1 (<b>17</b>) led to an 113-fold loss in potency (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The N2 of indazole forms a hydrogen bond with the hydroxyl group of Tyr132, which explains why indole <b>10</b> has a loss of 108-fold in potency (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The benzylic amine orients toward Trp110, and the benzylic carbon introduces a turn that places the rest of the molecule at an angle of 72° to the indole. Substitutions at the benzylic amine (Me, Et, and <i>i</i>Pr) retain potency, due to the ability of Trp110 to assume alternative conformations as described above (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02271/suppl_file/jm0c02271_si_001.pdf" class="ext-link">Table S1</a>). The oxadiazole forms π–π stacking interactions with Tyr26, which explains why Compounds <b>13</b>–<b>15</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) demonstrate a significant loss in potency. The phenolic hydroxy forms an internal hydrogen bond with the nitrogen of the oxadiazole, stabilizing a planar conformation of this portion of the inhibitor. Removal of the hydroxyl group (<b>19</b>) results in a 5-fold loss in potency (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Finally, the dimethylamine is situated in a pocket formed by residues Asp170, Asn173, Arg174, Ser22, and Ala206. Since a large portion of loop 4 (residues 175–205) is disordered in the -oxadiazole (DNA-attachment point in the context of the original DECL member) appear to remain at least partially solvent-exposed. The oxadiazole moiety occupies the space normacrystal structure, the phenol and dimethyl-aminolly filled by the Phe175 of loop 4, the gating loop. This loop is typically ordered in structures with bound substrates.<a onclick="showRef(event, 'ref3 ref9'); return false;" href="javascript:void(0);" class="ref ref3 ref9">(3,9)</a></div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/medium/jm0c02271_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0006.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Protein–ligand interactions in the Compound <b>5</b>/HAO1 complex PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W4C">6W4C</a>. The dashed gray ribbon indicates disordered loop 4.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02271&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Carboxylic Acid Bioisostere Replacement</h3><div class="NLM_p">To mitigate risks associated with poor permeability and the potential for reactive glucuronide metabolites of carboxylic acid-containing compounds, we sought to utilize structural information to identify acid isosteres that would make similar contacts within the observed active site. In one approach, we performed a virtual screen of the Enamine Real and Premium collections to identify potential replacements for the indazole acid group. Additionally, specific isostere-containing molecules were selected from the Enamine collection using substructure searches, then docked into the crystal structure of HAO1 (the structure with Compound <b>5</b> bound).</div><div class="NLM_p">To supplement these commercially available cost-efficient indazole acid isosteric replacement compounds, structure-based drug design (SBDD) was used to dock traditionally designed analogues of Compound <b>5</b> bearing isosteric acid replacements, into the active site of HAO1. Of a handful that were synthesized, Compound <b>31</b>, containing a benzotriazole moiety as a replacement of the indazole acid, demonstrated inhibition of HAO1 in our enzymatic assay (IC<sub>50</sub> = 2.9 μM) and 28% cell rescue at 30 μM (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). As hypothesized, replacing the carboxylic acid group increases the permeability (<i>P</i><sub>app</sub><i>A</i>–B = 13.1 × 10<sup>–6</sup> cm/s). Further ADME studies revealed that mouse PPB is also reduced (98.44% bound), which also likely contributes to benzotriazole compounds showing decreased stability in liver microsomes versus the carboxylic acids derivatives. The docking mode of this compound suggests that although the triazole group occupies a slightly different orientation, the overall geometry of the inhibitor in the binding site is similar to that of Compound <b>5</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). The nitrogens at positions 1 and 2 of triazole are predicted to form hydrogen bonds with Arg263, while the protonated nitrogen at position 3 is predicted to form a hydrogen bond to His260. The pyridinyl nitrogen is predicted to be hydrogen bonded to Tyr132. A limited set of additional benzotriazole analogues were synthesized, but no improvement in potency in the HAO1 biochemical assay was achieved. However, we feel that the absence of a carboxylic acid moiety and correspondingly increased permeability would make Compound <b>31</b> a better starting point for further optimization.</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/medium/jm0c02271_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0007.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. <b>A</b>. Compound <b>31</b>. <b>B</b>. Docking mode of Compound <b>31</b>, a benzotriazole designed as an isosteric replacement to Compound <b>5</b>, assumes a shifted orientation in the binding site. The docking pose was minimized with MM-GBSA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02271&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Pharmacokinetic (PK) Studies</h3><div class="NLM_p">The cell-active compounds described in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a> were also evaluated in mouse intravenous (IV) and oral (PO) PK studies. Compound <b>5</b> showed moderate <i>in vivo</i> unbound clearance (CL<sub>obs,unbound</sub> = 447 mL/min/kg); however, exposure was also moderate (AUC<sub>inf</sub> = 11.2 μM*h) and oral bioavailability was low (<i>F</i> = 6%) (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). Compound <b>6</b>, despite having a slight improvement in MDCK permeability, did not afford any improvement in exposure potentially due to an increased <i>in vivo</i> unbound clearance. With the highest <i>in vivo</i> unbound clearance and short half-life, <b>21</b> provided the lowest exposure and bioavailability. Finally, with improved permeability over Compound <b>5</b>, we were able to achieve improved exposure and oral bioavailability with Compound <b>29</b> (AUC<sub>inf</sub> = 46.1 μM*h and <i>F</i> = 19%), despite higher unbound clearance, resulting in the achievement of a compound with sufficient plasma exposure to be recommended for <i>in vivo</i> efficacy assessment.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. PK<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a> Data of Compounds <b>5</b>, <b>6</b>, <b>21</b>, and <b>29</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">Cl<sub>obs, unbound</sub> IV (mL/min/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>1/2</sub> IV (h)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>inf</sub>PO (μM*h)</th><th class="colsep0 rowsep0" align="center" char=".">%<i>F</i></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="char" char=".">447</td><td class="colsep0 rowsep0" align="char" char=".">1.8</td><td class="colsep0 rowsep0" align="char" char=".">11.2</td><td class="colsep0 rowsep0" align="char" char=".">6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="char" char=".">2444</td><td class="colsep0 rowsep0" align="char" char=".">3.3</td><td class="colsep0 rowsep0" align="char" char=".">6.8</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="char" char=".">5436</td><td class="colsep0 rowsep0" align="char" char=".">0.4</td><td class="colsep0 rowsep0" align="char" char=".">0.46</td><td class="colsep0 rowsep0" align="char" char=".">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="char" char=".">4500</td><td class="colsep0 rowsep0" align="char" char=".">3.3</td><td class="colsep0 rowsep0" align="char" char=".">46.1</td><td class="colsep0 rowsep0" align="char" char=".">19</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">Male CD1 mice were used for cassette i.v. (0.3 mg/kg) and p.o. (10 mg/kg) PK studies. Pharmacokinetic parameters were calculated from plasma concentration-time data and are reported as an average of three animals.</p></div></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18854" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18854" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We have discovered two novel series of HAO1 inhibitors from a DNA-Encoded Chemical Library DECL screen and obtained crystal structures for members of both series. Optimization efforts were prioritized for the more tractable Series 2. Compound <b>5</b> showed good potency in the biochemical assay and also demonstrated rescue activity in a stringent CHO-GO cell assay; however, the compound had poor permeability and poor oral bioavailability and exposure as demonstrated in mouse PK studies. Optimization of Compound <b>5</b> led to Compound <b>29</b> with good potency, permeability, and improved exposure and oral bioavailability. The potency and properties of compound <b>29</b> show a significant advance over the current literature HAO1 inhibitors, where <b>CCPST</b> was inactive and Compound <b>1</b> was 22-fold less active in our cell assay. Additionally, we were able to utilize SBDD with the crystal structure of Compound <b>5</b> in complex with HAO1 to identify a nonacid HAO1 inhibitor, <b>31</b>. Although the potency of this non-carboxylic acid-containing inhibitor is weaker than the Series 2 carboxylic acid inhibitors, it has activity in our stringent cell assay, where CCPST is inactive, and exhibits an 8-fold improvement over Compound <b>1</b>. Together with very reasonable permeability, Compound <b>31</b> is an appealing starting point for future optimization. Compounds <b>5</b>, <b>29</b>, and <b>31</b> are valuable tools to further advance the study of HAO1 inhibitors and their use in understanding the role of HAO1 in primary hyperoxaluria.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87352" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87352" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> General Synthetic Methods</h3><div class="NLM_p">Unless otherwise noted, all chemicals, reagents, and solvents were used as purchased without further purification. All final compounds have >95% purity as determined by analytical HPLC with UV detection at 214 nm. <sup>1</sup>H NMR are referenced to residual solvent signals, and δ are reported as chemical shift (δ ppm), splitting patterns, coupling constant (Hz), and integration. Splitting patterns are denoted as follows: s, singlet; d, doublet; dd, double of doublets; t, triplet; m, multiplet; and brs, broad singlet. <sup>13</sup>C NMR are referenced to residual solvent signals and are reported as chemical shifts (ppm).</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 3-Chloro-2′-((6-chloro-<i>N</i>-methyl-1<i>H</i>-indole-2-carboxamido)methyl)-5′-fluoro-[1,1′-biphenyl]-4-carboxylic Acid (<b>3</b>)</h4><div id="sec4_1_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24"> Step 1: <i>N</i>-(2-Bromo-4-fluorobenzyl)-6-chloro-<i>N</i>-methyl-1<i>H</i>-indole-2-carboxamide</h5><div class="NLM_p last">1-(2-Bromo-4-fluorophenyl)-<i>N</i>-methylmethanamine (217 mg, 1.00 mmol, 1.0 equiv), 6-chloro-1<i>H</i>-indole-2-carboxylic acid (195 mg, 1.00 mmol, 1.0 equiv), NaHCO<sub>3</sub> (420 mg, 5.00 mmol, 5.0 equiv), HATU (456 mg, 1.20 mmol, 1.2 equiv), and DMF (5 mL) were placed into a 50 mL round-bottom flask. The resulting solution was stirred for 2 h at 25 °C. The crude product was purified by flash chromatography with the following conditions (Column: XBridge C18 OBD Prep Column, 100 Å, 5 μm, 19 mm × 250 mm; Mobile Phase A: Water (0.1%FA), Mobile Phase B: ACN; flow rate: 20 mL/min; Gradient: 20% B to 40% B in 7 min; 254 and 220 nm; <i>R<sub>t</sub></i>: 7 min). This resulted in 250 mg (63%) of <i>N</i>-(2-bromo-4-fluorobenzyl)-6-chloro-<i>N</i>-methyl-1<i>H</i>-indole-2-carboxamide as a white solid. ES/MS (<i>m</i>/<i>z</i>): 395 (M + H)<sup>+</sup>; HPLC <i>t<sub>R</sub></i> = 1.417 min.</div></div><div id="sec99" class="NLM_sec NLM_sec_level_4"><div id="ac_i25" class="anchor-spacer"></div><h5 class="article-section__title" id="_i25"> Step 2: 3-Chloro-2′-((6-chloro-<i>N</i>-methyl-1<i>H</i>-indole-2-carboxamido)methyl)-5′-fluoro-[1,1′-biphenyl]-4-carboxylic Acid (<b>3</b>)</h5><div class="NLM_p last"><i>N</i>-(2-Bromo-4-fluorobenzyl)-6-chloro-<i>N</i>-methyl-1<i>H</i>-indole-2-carboxamide (100 mg, 0.25 mmol, 1.0 equiv), dioxane (2 mL), water (2 mL), 4-borono-2-chlorobenzoic acid (50 mg, 0.25 mmol, 1.0 equiv), Pd(dppf)Cl<sub>2</sub> (22 mg, 0.03 mmol, 0.1 equiv), and Na<sub>2</sub>CO<sub>3</sub> (80 mg, 0.76 mmol, 3.0 equiv) were placed into a 50 mL round-bottom flask. The resulting solution was stirred for 1 h at 100 °C. The crude product was purified by flash chromatography with the following conditions (Column: XBridge C18 OBD Prep Column, 100 Å, 5 μm, 19 mm × 250 mm; Mobile Phase A: Water (0.1%FA), Mobile Phase B: ACN; flow rate: 20 mL/min; Gradient: 20–40% B in 7 min; 254 and 220 nm; <i>R<sub>t</sub></i>: 7 min). This resulted in 13.4 mg (13%) of 3-chloro-2′-((6-chloro-<i>N</i>-methyl-1<i>H</i>-indole-2-carboxamido)methyl)-5′-fluoro-[1,1′-biphenyl]-4-carboxylic acid as a white solid. <sup>1</sup>H (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.42 (s, 1H); 11.67 (s, 1H); 7.78 (d, <i>J</i> = 8.0 Hz, 1H); 7.55–7.63 (m, 2H); 7.34–7.46 (m, 3H); 7.27–7.31 (m, 1H); 7.17–7.24 (m, 1H); 7.03 (dd, <i>J</i> = 8.0, 4.0 Hz, 2H); 4.69 (s, 2H); and 3.08 (s, 3H). ES/MS (<i>m</i>/<i>z</i>): 471 (M + H)<sup>+</sup>; HPLC <i>t<sub>R</sub></i> = 1.942 min.</div></div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 5-(Methyl((2-propoxypyridin-3-yl)methyl)amino)-1<i>H</i>-indazole-3-carboxylic Acid (<b>4</b>)</h4><div class="NLM_p last">To a solution of 2-propoxynicotinaldehyde (500 mg, 3.03 mmol), 5-amino-1<i>H</i>-indazole-3-carboxylic acid (532 mg, 3.03 mmol) in AcOH:MeOH = 1:5 (10 mL) was stirred at r.t. for 2 h and NaBH<sub>3</sub>CN (381 mg, 6.06 mmol) was added. The reaction was stirred at r.t. overnight, and then paraformaldehyde (136.4 mg, 4.55 mmol) was added. The reaction was stirred at r.t. for another 4 h. The reaction was quenched by water (20 mL) and extracted with ethyl acetate (30 mL × 3); the combined organic layer was dried over with anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtration, the solution was concentrated under vacuum and purified by prep-HPLC to give 5-(methyl((2-propoxypyridin-3-yl)methyl)amino)-1<i>H</i>-indazole-3-carboxylic acid (56 mg, 24%) as a yellow solid. <sup>1</sup>H NMR (400 MHz, dimethyl sulfoxide (DMSO)) δ 13.47 (s, 1H), 12.61 (s, 1H), 8.03–8.01 (d, <i>J</i> = 4.0 Hz, 1H), 7.47–7.44 (d, <i>J</i> = 12.0 Hz, 1H), 7.37–7.36 (d, <i>J</i> = 4.0 Hz, 1H), 7.17 (s, 1H), 7.05 (m, 1H), 6.88–6.86 (d, <i>J</i> = 8.0 Hz, 1H), 4.51 (s, 2H), 4.30–4.27 (t, <i>J</i> = 4.0 Hz, 2H), 3.05 (s, 3H), 1.70–1.74 (m, 2H), and 1.02–0.98 (t, <i>J</i> = <i>8.0 Hz</i>, 3H). ES/MS (<i>m</i>/<i>z</i>): 341 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 5-((3-(3-(Dimethylamino)-1,2,4-oxadiazol-5-yl)-2-hydroxybenzyl)amino)-1<i>H</i>-indazole-3-carboxylic Acid (<b>5</b>)</h4><div id="sec4_1_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> Step 1: 3-(3-(Dimethylamino)-1,2,4-oxadiazol-5-yl)-2-hydroxybenzaldehyde</h5><div class="NLM_p last">A solution of 3-hydroxy-1,1-dimethylguanidine (3.0 g, crude), 3-formyl-2-hydroxybenzoic acid (1.3 g, 7.3 mmol), HATU (5.5 g, 15 mmol), and NaHCO<sub>3</sub> (1.8 g, 21.85 mmol) in DMF (10 mL) was stirred at rt overnight. The reaction mixture was diluted with H<sub>2</sub>O (30 mL) and extracted with ethyl acetate (30 mL × 3), then dried over with anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtration, the solution was concentrated under vacuum to give the crude product. The crude product was purified by Combiflash column (CH<sub>2</sub>Cl<sub>2</sub>:MeOH = 2:1) to give 3-(3-(dimethylamino)-1,2,4-oxadiazol-5-yl)-2-hydroxybenzaldehyde (480 mg, 28%) as a white solid. ES/MS (<i>m</i>/<i>z</i>): 234 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29"> Step 2: 5-((3-(3-(Dimethylamino)-1,2,4-oxadiazol-5-yl)-2-hydroxybenzyl)amino)-1<i>H</i>-indazole-3-carboxylic Acid (<b>5</b>)</h5><div class="NLM_p last">To a solution of 3-(3-(dimethylamino)-1,2,4-oxadiazol-5-yl)-2-hydroxybenzaldehyde (120 mg, 0.50 mmol), 5-amino-1<i>H</i>-indazole-3-carboxylic acid (180 mg, 0.7 mmol) in AcOH:MeOH = 1:10 (10 mL) was stirred at rt for 2 h. NaBH<sub>3</sub>CN (41 mg, 0.65 mmol) was added and the reaction was stirred at rt overnight. The reaction was quenched with water (10 mL) and extracted with ethyl acetate (20 mL x 3), then dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtration, the solution was concentrated under vacuum to give a residue which was purified by prep-HPLC to give 5-((3-(3-(dimethylamino)-1,2,4-oxadiazol-5-yl)-2-hydroxybenzyl)amino)-1<i>H</i>-indazole-3-carboxylic (40 mg, 21%) as a yellow solid. <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 13.40 (s, 1H), 12.52 (s, 1H), 10.63 (s, 1H), 7.78 (dd, <i>J</i> = 8.0, 1.6 Hz, 1H), 7.56 (dd, <i>J</i> = 7.6, 1.2 Hz, 1H), 7.40 (d, <i>J</i> = 9.2 Hz, 1H), 7.04–6.96 (m, 3H), 6.28 (brs, 1H), 4.38 (d, <i>J</i> = 4.4 Hz, 2H), and 3.04 (s, 6H); <sup>13</sup>C NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 37.8, 41.9, 108.0, 118.2, 119.8, 126.0, 127.6, 133.3, 154.9, 168.8, and 172.3. ES/MS (<i>m</i>/<i>z</i>): 395 (M + H)<sup>+</sup>.</div></div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 5-((3-(3-(Dimethylamino)-1,2,4-oxadiazol-5-yl)-2-hydroxybenzyl)(methyl)amino)-1<i>H</i>-indazole-3-carboxylic Acid (<b>6</b>)</h4><div class="NLM_p last">A solution of 3-(3-(dimethylamino)-1,2,4-oxadiazol-5-yl)-2-hydroxybenzaldehyde (100 mg, 0.43 mmol), 5-amino-1<i>H</i>-indazole-3-carboxylic acid (114 mg, 0.64 mmol) in AcOH:MeOH = 1:5 (15 mL) was stirred at rt for 2 h, NaBH<sub>3</sub>CN (54 mg, 0.86 mmol) was added. The reaction was stirred at rt overnight. Paraformaldehyde (26 mg, 0.86 mmol) was added to the reaction solution. The reaction solution was stirred at rt for 2 h. The reaction was quenched by water (10 mL) and extracted with ethyl acetate (20 mL x 3); the combined organic layer was dried over with anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtration, the solution was concentrated under vacuum and purified by prep-HPLC to give product 5-((3-(3-(dimethylamino)-1,2,4-oxadiazol-5-yl)-2-hydroxybenzyl)(methyl)amino)-1<i>H</i>-indazole-3-carboxylic acid (22 mg, 13%) as a yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO) δ 13.50 (s, 1H), 12.63 (s, 1H), 10.64 (s, 1H), 7.79–7.77 (d, <i>J</i> = 8.0 Hz, 1H), 7.49–7.47 (d, <i>J</i> = 8.0 Hz, 1H), 7.22 (m, 1H), 7.11 (s, 1H), 7.11–7.09 (d, <i>J</i> = 8.0 Hz, 1H), 7.01–6.97 (t, <i>J</i> = 8.0 Hz, 1H), 4.67 (s, 2H), 3.09 (s, 3H), and 3.04 (s, 6H). <sup>13</sup>C NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 38.3, 108.8, 112.5, 120.4, 127.2, 155.5, 164.3, 169.3, 172.7, and 178.5. ES/MS (<i>m</i>/<i>z</i>): 409 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 5-((2-Hydroxy-3-(3-isopropyl-1,2,4-oxadiazol-5-yl)benzyl)amino)-1<i>H</i>-indazole-3-carboxylic Acid (<b>21</b>)</h4><div id="sec4_1_5_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> Step 1: 2-Hydroxy-3-(3-isopropyl-1,2,4-oxadiazol-5-yl)benzaldehyde</h5><div class="NLM_p last">To a solution of 3-formyl-2-hydroxybenzoic acid (332 mg, 2 mmol) in 10 mL of CH<sub>2</sub>Cl<sub>2</sub> was added oxalyl chloride (1 mL) at rt, accelerated by a catalytic amount of DMF, and then the reaction was stirred at rt for 2 h. The solution was concentrated and added to a solution of <i>N</i>-hydroxyisobutyrimidamide (204 mg, 1 mmol) and <i>N,N</i>-diisopropylethylamine (DIEA, 516 mg, 4 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) at rt. The reaction mixture was stirred at rt for 2 h. Then, heated to reflux for overnight, the solution was concentrated and purified using combiflash (CH<sub>2</sub>Cl<sub>2</sub>:MeOH = 20:1) to give 2-hydroxy-3-(3-isopropyl-1,2,4-oxadiazol-5-yl)benzaldehyde (200 mg, 43%) as a white solid. ES/MS (<i>m</i>/<i>z</i>): 233 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_5_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> Step 2: 5-((2-hydroxy-3-(3-isopropyl-1,2,4-oxadiazol-5-yl)benzyl)amino)-1<i>H</i>-indazole-3-carboxylic Acid (<b>21</b>)</h5><div class="NLM_p last">To a solution of 2-hydroxy-3-(3-isopropyl-1,2,4-oxadiazol-5-yl)benzaldehyde (200 mg, 0.86 mmol), 5-amino-1<i>H</i>-indazole-3-carboxylic acid (177 mg, 1 mmol) and AcOH (0.1 mL) in 10 mL of MeOH were stirred at rt for 0.5 h, NaBH<sub>3</sub>CN (124 mg, 2 mmol) was added, and then the reaction was stirred at rt for overnight; the solution was concentrated and purified by Prep-HPLC(Chromatographic columns: -Gemini-C18 150 mm × 21.2 mm, 5 μm; Mobile Phase: ACN–H<sub>2</sub>O (0.1%FA); Gradient: 45–50) to give 5-((2-hydroxy-3-(3-isopropyl-1,2,4-oxadiazol-5-yl)benzyl)amino)-1<i>H</i>-indazole-3-carboxylic acid (15 mg, 4%) as a yellow solid. <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.87 (dd, <i>J = 7.9, 1.4 Hz</i>, 1H), 7.64 (d<i>, J = 7.4 Hz</i>, 1H), 7.41 (d, <i>J</i> = 9.0 Hz, 1H), 7.18 (d, <i>J</i> = 1.5 Hz, 1H), 7.09–6.98 (m, 2H), 4.52 (s, 2H), 3.24 (m, 1H), and 1.45 (d, <i>J</i> = 7.0 Hz, 6H). <sup>13</sup>C NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 20.6, 26.4, 42.2, 108.3, 118.5, 120.5, 124.5, 126.7, 128.5, 133.9, 145.2, 155.5, 173.9, and 174.0. ES/MS (<i>m</i>/<i>z</i>): 394 (M + H)<sup>+</sup>.</div></div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 5-([[3-(4,5-Dimethyl-1,3-thiazol-2-yl)-2-hydroxyphenyl]methyl]amino)-1<i>H</i>-indazole-3-carboxylic Acid (29)</h4><div id="sec4_1_6_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> Step 1: 3-(4,5-Dimethyl-1,3-thiazol-2-yl)-2-methoxybenzaldehyde</h5><div class="NLM_p last">To a stirred solution of 2-bromo-4,5-dimethyl-1,3-thiazole (400 mg, 2.08 mmol, 1.0 equiv) and 3-formyl-2-methoxyphenylboronic acid (562 mg, 3.12 mmol, 1.5 equiv) in dioxane (8 mL) and H<sub>2</sub>O (0.8 mL) were added Cs<sub>2</sub>CO<sub>3</sub> (2.04 g, 6.25 mmol, 3.0 equiv) and Pd(PPh<sub>3</sub>)<sub>4</sub> (241 mg, 0.21 mmol, 0.1 equiv). The resulting mixture was stirred at 100 °C for 2 h under a nitrogen atmosphere. The resulting mixture was concentrated under reduced pressure. The residue was purified using C18 flash chromatography with the following conditions (Mobile Phase A: Water (0.1%FA), Mobile Phase B: ACN; flow rate: 60 mL/min; and gradient: 0–52% B in 50 min; 254/220 nm) to afford 3-(4,5-dimethyl-1,3-thiazol-2-yl)-2-methoxybenzaldehyde (430 mg, 84%) as a brown solid. ES/MS (<i>m</i>/<i>z</i>): 248 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_6_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> Step 2: 3-(4,5-Dimethyl-1,3-thiazol-2-yl)-2-hydroxybenzaldehyde</h5><div class="NLM_p last">To a stirred solution of 3-(4,5-dimethyl-1,3-thiazol-2-yl)-2-methoxybenzaldehyde (400 mg, 1.62 mmol, 1.0 equiv) in CH<sub>2</sub>Cl<sub>2</sub> (4 mL) was added BBr<sub>3</sub> (1 M, 3.23 mL, 3.23 mmol, 2.0 equiv) dropwise at 0 °C under the nitrogen atmosphere. The resulting mixture was stirred at room temperature for 1 h under nitrogen atmosphere. The resulting mixture was poured into water (20 mL). The resulting mixture was extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 30 mL). The combined organic layers were washed with brine (1 × 50 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtration, the filtrate was concentrated under reduced pressure to afford 3-(4,5-dimethyl-1,3-thiazol-2-yl)-2-hydroxybenzaldehyde (400 mg, crude) as a yellow solid. ES/MS (<i>m</i>/<i>z</i>): 234 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_6_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> Step 3: 5-([[3-(4,5-Dimethyl-1,3-thiazol-2-yl)-2-hydroxyphenyl]methyl]amino)-1<i>H</i>-indazole-3-carboxylic Acid (<b>29</b>)</h5><div class="NLM_p last">To a stirred solution of 3-(4,5-dimethyl-1,3-thiazol-2-yl)-2-hydroxybenzaldehyde (288 mg, 1.24 mmol, 1 equiv) and 5-amino-1<i>H</i>-indazole-3-carboxylic acid (219 mg, 1.24 mmol, 1.0 equiv) in DMSO (2 mL) were added CH<sub>3</sub>COOH (74 mg, 1.24 mmol, 1.0 equiv) and NaBH<sub>3</sub>CN (310 mg, 4.94 mmol, 4.0 equiv). The resulting mixture was stirred at room temperature for 1 h under nitrogen atmosphere. The resulting mixture was purified using Prep-HPLC with the following conditions (Column: Xselect CSH OBD Column 30 mm × 150 mm 5 μm, n; Mobile Phase A:Water (0.1% FA), Mobile Phase B:ACN; flow rate: 60 mL/min; gradient: 56–77% B in 8 min; 254 nm; RT1:8.12) to afford 5-([[3-(4,5-dimethyl-1,3-thiazol-2-yl)-2-hydroxyphenyl]methyl]amino)-1<i>H</i>-indazole-3-carboxylic acid (25.00 mg, 5.1%) as a yellow solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 13.38 (s, 1H); 12.65 (s, 1H); 12.41 (s, 1H); 7.52 (dd, <i>J</i> = 7.8, 1.6 Hz, 1H); 7.30–7.42 (m, 2H); 6.94–7.01 (m, 2H); 6.88 (t, <i>J</i> = 7.7 Hz, 1H); 6.20 (s, 1H); 4.35 (m, 2H); 2.42 (d, <i>J</i> = 1.0 Hz, 3H); and 2.37 (d, <i>J</i> = 1.0 Hz, 3H). <sup>13</sup>C NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.3, 14.6, 42.2, 116.6, 118.6, 119.6, 124.5, 125.8, 126.0, 127.9, 130.1, 145.4, 146.9, 154.3, and 164.3. ES/MS (<i>m</i>/<i>z</i>): 395 (M + H)<sup>+</sup>.</div></div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 2-(((1<i>H</i>-[1,2,3]Triazolo[4,5-<i>b</i>]pyridin-6-yl)amino)methyl)-6-(3-(dimethylamino)-1,2,4-oxadiazol-5-yl)phenol (<b>31</b>)</h4><div id="sec4_1_7_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> Step 1: 6-Nitro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>b</i>]pyridine</h5><div class="NLM_p last">To a stirred solution of 5-nitropyridine-2,3-diamine (1.3 g, 8.44 mmol) in H<sub>2</sub>O (41 mL) was added H<sub>2</sub>SO<sub>4</sub> (10 mL) followed by NaNO<sub>2</sub> (639 mg, 9.28 mmol) portion wise at 0 °C and stirred for 1 h at 0 °C. The reaction was monitored by thin-layer chromatography (TLC). Then, the reaction mixture was filtered and the solid was dried over reduced pressure to obtain 6-nitro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>b</i>]pyridine (800 mg, 61%) as brick red solid. ES/MS (<i>m</i>/<i>z</i>): 166 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_7_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> Step 2: 1<i>H</i>-[1,2,3]Triazolo[4,5-<i>b</i>]pyridin-6-amine</h5><div class="NLM_p last">To a stirred solution of 6-nitro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>b</i>]pyridine (500 mg, 3.03 mmol) in THF:MeOH (1:1) (10 mL) was added Pd/C (100 mg, 20%w/w) at room temperature and stirred at hydrogen gas for 16 h. The reaction was monitored by TLC. Then, the reaction mixture was filtered through celite and the filtrate was concentrated under reduced pressure to obtain 1<i>H</i>-[1,2,3]triazolo[4,5-<i>b</i>]pyridin-6-amine (400 mg, 97%) as pale yellow solid. ES/MS (<i>m</i>/<i>z</i>):136 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_7_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> Step 3: 2-(((1<i>H</i>-[1,2,3]Triazolo[4,5-<i>b</i>]pyridin-6-yl)amino)methyl)-6-(3-(dimethylamino)-1,2,4-oxadiazol-5-yl)phenol (<b>31</b>)</h5><div class="NLM_p last">To a stirred solution of 3-(3-(dimethylamino)-1,2,4-oxadiazol-5-yl)-2-hydroxybenzaldehyde (220 mg, 0.94 mmol) in MeOH at RT were added K<sub>2</sub>CO<sub>3</sub> (195 mg, 1.41 mmol) followed by 1<i>H</i>-[1,2,3]triazolo[4,5-<i>b</i>]pyridin-6-amine (191.1 mg, 1.41 mmol) and stirred the reaction mixture at 65 °C for 16 h. Then, the reaction mixture cooled to room temperature, and NaCNBH<sub>3</sub> (58.5 mg, 0.94 mmol) was added to reaction mixture and stirred at RT for 2 h. After completion of the reaction (by TLC), the reaction mixture was filtered and filtrate was concentrated under reduced pressure. The resulting crude material was purified by SFC to afford 2-(((1<i>H</i>-[1,2,3]triazolo[4,5-<i>b</i>]pyridin-6-yl)amino)methyl)-6-(3-(dimethylamino)-1,2,4-oxadiazol-5-yl)phenol (41 mg, 12%) as an off-white solid.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> Purification by Preparative SFC</h4><div class="NLM_p">Purification by Preparative SFC. Column/dimensions: Chiralpak-IC (30 × 250) mm, 5 μm; % CO<sub>2</sub>: 70%; % Co-solvent: 30% (methanol); total flow: 100.0 g/min; back pressure: 110 bar; temperature: 30 °C; and UV: 220 nm. Solubility: MEOH.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 15.06 (brs, 1H), 10.64 (brs, 1H), 8.29 (d, <i>J</i> = 2.0 Hz, 1H), 7.81 (dd, <i>J</i> = 8.0, 1.6 Hz, 1H), 7.57 (dd, <i>J</i> = 7.6, 1.6 Hz, 1H), 7.12–7.02 (m, 2H), 6.79 (brs, 1H), 4.43 (d, <i>J</i> = 6.0 Hz, 2H), and 3.03 (s, 6H). <sup>13</sup>C NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 38.3, 41.6, 108.8, 120.5, 126.9, 127.0, 133.9, 141.0, 144.5, 155.5, 169.3, and 172.8. ES/MS (<i>m</i>/<i>z</i>): 353 (M + H)<sup>+</sup>.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> ADME/PK</h3><div class="NLM_p last">Determination of microsomal stability, permeability, solubility, and plasma protein binding were performed at Pharmaron (Beijing, China) under their standard assay conditions. PK studies were also performed at Pharmaron (Beijing, China) and were carried out according to the guidelines approved by the Institutional Animal Care and Use Committee (IACUC) of Pharmaron. Cassette iv (intravenous) PK was performed in male CD1 mice (3 mice per group) with up to five compounds each dosed at 0.3 mg/kg. The concentration of compounds was measured in plasma at nine timepoints (2, 5, 15, 30 min, 1, 2, 4, 6, 8 h post dose), with 0.03 mL of blood collected via the dorsal metatarsal vein at each timepoint. Single-dose oral (PO) PK was performed in male CD1 mice with each compound dosed at 10 mg/kg. The concentration of compounds was measured in plasma at nine timepoints (15, 30 min, 1, 2, 4, 6, 8, 12, 24 h post dose), with 0.03 mL of blood collected via the dorsal metatarsal vein at each timepoint.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> HAO1/GO Enzyme Assay</h3><div class="NLM_p last">The inhibitory activity of the compounds against purified recombinant human HAO1 enzymatic activity was determined using a 10 nM final concentration of HAO1 in the presence of 50 μM glycolic acid as substrate in 50 mM HEPES buffer, pH 8.0. The amount of hydrogen peroxide generated during the conversion of glycolic acid to glyoxylate was detected by a coupled horseradish peroxidase (HRP) reaction. Compounds were serially diluted and dispensed at a volume of 0.6 μL/well (10% DMSO) in an OptiPlate-384 black microplate (Perkin Elmer, Cat# 6007270) and were preincubated at room temperature for 30 min with 30 μL/well of 20 nM hHAO1. Then, 15 μL of 200 μM Amplex Red (Thermo Fisher, Cat# A12222) and 2 U/mL of HRP (Sigma-Aldrich, Cat#: P6140) were added to each well at final concentrations of 50 μM and 0.5 U/mL, respectively. The reaction was initiated by the addition of 15 μL/well of 200 μM glycolic acid to a final concentration of 50 μM in a final volume of 60 μL. The rate of the reaction (RFU/min) was measured on a SpectrMax Paradigm Microplate Reader (Ex 563 nm, Em 587 nm) for 15 min at 37 °C. The percent inhibition of compounds was calculated relative to the maximum rate in the absence of compound (HAO1 with 1% DMSO only) and the minimum rate in the absence of enzyme (reaction mixture without HAO1). Percent inhibition was plotted versus the compound concentrations to calculate IC<sub>50</sub> values for compounds tested using a four-parameter curve fit.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> CHO Cell Rescue Assay</h3><div class="NLM_p">CHO cells stably transformed with a vector expressing glycolate oxidase (CHO-GO) were cultured in Ham’s F-12/Glutamax media supplemented with 10% fetal bovine serum (FBS) and 100 units/mL penicillin/100 μg/mL streptomycin with additional supplementation of 400 μg/mL zeocin. CHO-WT cells were maintained in the same media conditions without the addition of zeocin.</div><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> siRNA Transfection</h4><div class="NLM_p last">CHO-GO cells were transfected using RNAiMax reagent with two specific HAO1 siRNAs (#8 or #7, Qiagen, Hilden, Germany), nonsilencing siRNA control (NC), or cell death positive control (Qiagen, Hilden, Germany). The next day media was replaced and 250 μM glycolic acid was added. Percent confluence vs time using live cell imaging (IncuCyte Zoom) was acquired up to 96 h. Cells were lysed in RIPA buffer and western blot was run to probe for HAO1 protein expression using HAO1 antibody (Abcam, Cambridge, U.K.). Tubulin was used as a loading control (Sigma, St Louis, MO).</div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> Inhibitor Studies</h4><div class="NLM_p">Cells were plated in 96-well plate (2000 cells/well) in standard medium (Ham’s F-12 Glutamax F-12 with 10% FBS and antibiotics) and incubated with compound at different concentrations for 1 h followed by addition of glycolic acid (final concentration 125 or 250 μM; Sigma, St Louis, MO). The final DMSO concentration in the assay was 0.5%. Viability was monitored using live cell imager (IncuCyte Zoom) for up to 96 h. CHO-WT cells were used as a control.</div><div class="NLM_p last">Data was analyzed using customized software developed internally. EC<sub>50</sub> values (half-maximum effective concentration) were calculated from dose–response curves using Prism software (Graph pad). At each compound concentration, the percent rescue was calculated as the percent confluence of CHO-GO cells treated with the compound in presence of glycolic acid divided by the percent confluence of CHO-GO cells treated with the compound in the absence of glycolic acid.</div></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> Affinity-Mediated Selection of DNA-Encoded Chemical Libraries</h3><div class="NLM_p last">Forty-eight different DNA-encoded chemical libraries were combined, and this mixture was incubated with no target, HAO1 at 7.7 or 1.5 μM, HAO1 at 7.7 μM, or the off-target LDHB at 9.1 μM each in a 60 μL of volume in a model cytosolic buffer (20 mM HEPES, 134 mM potassium acetate, 8 mM sodium acetate, 4 mM sodium chloride, 0.8 mM magnesium acetate, 1 mg/mL sheared salmon sperm DNA, 1 mM TCEP, 5 mM imidazole, 0.02% Tween-20, and pH 7.2). An additional parallel selection condition of HAO1 protein with Compound 1 was performed to identify competitive features. Separate incubations were set up for each selection condition with 50 pmoles of each library. After a 1 h incubation, the mixtures were captured on 5 μL of CaptureSelect XL Anti-C-Tag matrix (Thermo Fisher Scientific, Waltham, MA) constrained within a 20 μL Phytip mounted on a Phynexus ME-200 automated 12-tip pipettor. After capture, the matrices were washed eight times with changes of buffer and then heat-eluted at 85 °C into the same buffer. The second round of selection was performed using the round-one eluate as input and fresh proteins and Compound <b>1</b> repeating the conditions of the first round. The encoding DNA in the output of the second round of selection was amplified using PCR with primers that introduce the Illumina READ1 and READ2 sequences and the amplified sample was submitted for sequencing using an Illumina 2500 instrument in high-output mode. Two hundred fifty-two million single-end reads were generated across all 48 libraries and all selection conditions.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> Crystallography</h3><div class="NLM_p last">Human HAO1 protein (residues 1–168) for crystallization was expressed with a TEV cleavable N-terminal 6xHis-tag in <i>Escherichia. coli</i> and expressed and purified as described previously.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> One millimolar FMN was added throughout the purification process to ensure full occupancy of the cofactor binding site. Purified protein at ∼20 mg/mL was incubated with 1 mM sodium glycolate on ice for 1 h followed by centrifugation. Initial protein crystals in complex with glycolate were grown using the sitting drop vapor diffusion; 200 nL of protein solution was mixed with 180 nL of reservoir solution (0.1 M Tris HCl pH 8.5, 25% (v/v) PEG 550 MME), and 20 nL streaking seeding solution. Seed stock was prepared by crushing a large crystal of lysozyme in a reservoir solution. Trays were incubated at 20 °C, and crystals of sufficient size appeared within 2 days. Crystals were soaked in a reservoir solution containing 5 mM compound solubilized in DMSO overnight. Crystals were transferred to a soaking solution containing 25% glycerol and vitrified in liquid nitrogen. X-ray diffraction data was collected at cryogenic temperature using synchrotron radiation at the Shanghai Synchrotron Radiation Source. Diffraction data was processed with HKL3000 (Supporting Information Section, <a href="/doi/suppl/10.1021/acs.jmedchem.0c02271/suppl_file/jm0c02271_si_001.pdf" class="ext-link">Table S4</a>).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> The structure was solved by molecular replacement using Phaser.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Protein and inhibitors were modeled into the electron density using Coot,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> and the model was refined using REFMAC (Supporting Information Section, <a href="/doi/suppl/10.1021/acs.jmedchem.0c02271/suppl_file/jm0c02271_si_001.pdf" class="ext-link">Table S5</a>).<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Atomic coordinates and structure factors have been deposited in the Protein Data Bank under accession codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W45">6W45</a> (Compound <b>3</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W44">6W44</a> (Compound <b>4</b>), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W4C">6W4C</a> (Compound <b>5</b>). Authors will release the atomic coordinates and experimental data upon article publication.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> Modeling</h3><div class="NLM_p">All calculations were carried out using the tools in the Schrödinger Suite.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></div><div class="NLM_p">The X-ray crystal structures of human HAO1 with compounds <b>3</b>–<b>5</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W45">6W45</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W44">6W44</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W4C">6W4C</a>) were used in modeling studies. To explore the binding cavity, SiteMap<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33,34)</a> was applied to crystal structures. SiteMap was applied with default settings; the bound ligand in each case was used to define binding site regions.</div><div class="NLM_p">The preparation of the protein for docking was performed with PrepWizard<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> using the standard protocol, including the addition of hydrogens and the assignment of bond order and of correct protonation states. The hydrogen-bonding network of the protein was optimized. Docking experiments were performed using Glide SP.<a onclick="showRef(event, 'ref36 ref37 ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref36 ref37 ref38 ref39">(36−39)</a> A hydrogen-bond constraint to Arg263 was used.</div><div class="NLM_p">The ligands were treated with LigPrep<a onclick="showRef(event, 'ref32 ref40'); return false;" href="javascript:void(0);" class="ref ref32 ref40">(32,40)</a> to obtain three-dimensional (3D) structures, and protonation states were assigned using Epik. MM-GB/SA in Prime<a onclick="showRef(event, 'ref41 ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref41 ref42 ref43">(41−43)</a> was performed on top scoring ligand poses, with default settings.</div><div class="NLM_p last">For virtual screening, the Virtual Screening Workflow protocol available in Maestro was used, with enhanced sampling during the docking stages and postprocessing with MM-GBSA.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i51"><a href="/doi/suppl/10.1021/acs.jmedchem.0c02271" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12307" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12307" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c02271?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02271</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">SAR (Tables S1–S3); crystal data collect and reduction (Table S4); crystal refinement statistics (Table S5); protein production; compound interference assay; HAO1 siRNA rescue glycolic acid-induced cell-death in CHO-GO cells; synthesis and characterization for compounds <b>7–20, 22–28, 30</b>, and <b>32–39</b>; and LCMS and NMR Spectra for compounds <b>3–6</b>, <b>21</b>, <b>29</b>, and <b>31</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02271/suppl_file/jm0c02271_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02271/suppl_file/jm0c02271_si_002.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">Associated biological assay data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02271/suppl_file/jm0c02271_si_003.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02271/suppl_file/jm0c02271_si_001.pdf">jm0c02271_si_001.pdf (5.46 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02271/suppl_file/jm0c02271_si_002.xlsx">jm0c02271_si_002.xlsx (11.01 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02271/suppl_file/jm0c02271_si_003.pdb">jm0c02271_si_003.pdb (266.05 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The PDB ID for X-ray crystal structures of human HAO1 with compounds <b>3</b>, <b>4</b>, and <b>5</b> are <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W45">6W45</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W44">6W44</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W4C">6W4C</a>, respectively. The authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c02271" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24370" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24370" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Esther C. Y. Lee</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0329-9186" title="Orcid link">http://orcid.org/0000-0003-0329-9186</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#0e6b7d7a666b7c20626b6b4e6f7d7a7c6f746b606b6d6f206d6163"><span class="__cf_email__" data-cfemail="4e2b3d3a262b3c60222b2b0e2f3d3a3c2f342b202b2d2f602d2123">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anthony D. Keefe</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#67060c02020102271f4a040f020a151f4904080a"><span class="__cf_email__" data-cfemail="83e2e8e6e6e5e6c3fbaee0ebe6eef1fbade0ecee">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew J. McRiner</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Auron Therapeutics, Wellesley, Massachusetts 02481, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Katy E. Georgiadis</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Julie Liu</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Accent Therapeutics, Lexington, Massachusetts 02421, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zooey Wang</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew D. Ferguson</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Takeda, Cambridge, Massachusetts, 02139, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1018-9631" title="Orcid link">http://orcid.org/0000-0002-1018-9631</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Benjamin Levin</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Dewpoint Therapeutics, Boston, Massachusetts, 02210, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Moritz von Rechenberg</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Relay Therapeutics, Cambridge Massachusetts, 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christopher D. Hupp</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael I. Monteiro</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Relay Therapeutics, Cambridge Massachusetts, 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Allison Olszewski</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Civetta Therapeutics, Cambridge, Massachusetts 02138, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Charles J. Eyermann</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Boston Drug Design LLC, Lincoln, Massachusetts, 01773, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paolo Centrella</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yanbin Liu</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Cyteir Therapeutics, Lexington, Massachusetts, 02421, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shilpi Arora</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Exo Therapeutics, Cambridge, Massachusetts, 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John W. Cuozzo</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Relay Therapeutics, Cambridge Massachusetts, 02139, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6229-0395" title="Orcid link">http://orcid.org/0000-0002-6229-0395</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ying Zhang</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew A. Clark</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christelle Huguet</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Ipsen, Cambridge, Massachusetts, 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anna Kohlmann</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Black Diamond Therapeutics, Cambridge, Massachusetts, 02142, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions from all authors. All authors have given approval to the final version of the manuscript.</p></li><li><span class="author-information-subsection-header">Funding</span><p>The work included in the manuscript was performed as part of internal drug discovery efforts at X-Chem Pharmaceuticals. No other grant support was used.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): All authors are or were employed by X-Chem Pharmaceuticals at the time the work was done.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e3720-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i67">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43420" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43420" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">X-Chem gratefully acknowledges the contributions of F.W., Z.L., and J.W. from WuXi Biortus Biosciences for their contribution to the protein science and structural biology experiments.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">ADME</td><td class="NLM_def"><p class="first last">absorption, distribution, metabolism, and excretion</p></td></tr><tr><td class="NLM_term">AGT</td><td class="NLM_def"><p class="first last">alanine–glyoxylate aminotransferase</p></td></tr><tr><td class="NLM_term">CCPST</td><td class="NLM_def"><p class="first last">4-carboxy-5-[(4-chlorophenyl)sulfanyl]-1,2,3-thiadiazole</p></td></tr><tr><td class="NLM_term">CDST</td><td class="NLM_def"><p class="first last">4-carboxy-5-dodecylsulfanyl-1,2,3-triazole</p></td></tr><tr><td class="NLM_term">CL<sub>int</sub></td><td class="NLM_def"><p class="first last">intrinsic clearance</p></td></tr><tr><td class="NLM_term">DECL</td><td class="NLM_def"><p class="first last">DNA-Encoded Chemical Library</p></td></tr><tr><td class="NLM_term">F</td><td class="NLM_def"><p class="first last">oral bioavailability</p></td></tr><tr><td class="NLM_term">FMN</td><td class="NLM_def"><p class="first last">Flavin mononuleotide</p></td></tr><tr><td class="NLM_term">GO</td><td class="NLM_def"><p class="first last">glycolate oxidase</p></td></tr><tr><td class="NLM_term">HAO1</td><td class="NLM_def"><p class="first last">hydroxyl acid oxidase 1</p></td></tr><tr><td class="NLM_term">hHAO1</td><td class="NLM_def"><p class="first last">human hydroxy acid oxidase 1</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">hexafluorophosphate azabenzotriazole tetramethyl uranium</p></td></tr><tr><td class="NLM_term">HLM</td><td class="NLM_def"><p class="first last">human liver microsomes</p></td></tr><tr><td class="NLM_term">HRP</td><td class="NLM_def"><p class="first last">horseradish peroxidase</p></td></tr><tr><td class="NLM_term">LDHB</td><td class="NLM_def"><p class="first last">lactate dehydrogenase B</p></td></tr><tr><td class="NLM_term">MLM</td><td class="NLM_def"><p class="first last">mouse liver microsomes</p></td></tr><tr><td class="NLM_term"><i>P</i><sub>app</sub></td><td class="NLM_def"><p class="first last">apparent passive permeability</p></td></tr><tr><td class="NLM_term">PH1</td><td class="NLM_def"><p class="first last">primary hyperoxaluria type1</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetics</p></td></tr><tr><td class="NLM_term">PPB</td><td class="NLM_def"><p class="first last">plasma protein binding</p></td></tr><tr><td class="NLM_term">MDCK</td><td class="NLM_def"><p class="first last">Madin–Darby Canine Kidney</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">SFC</td><td class="NLM_def"><p class="first last">supercritical fluid chromatography</p></td></tr><tr><td class="NLM_term">SBDD</td><td class="NLM_def"><p class="first last">structure-based drug design</p></td></tr><tr><td class="NLM_term">tPSA</td><td class="NLM_def"><p class="first last">topological polar surface area</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i69">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78471" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78471" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 43 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cochat, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumsby, G.</span></span> <span> </span><span class="NLM_article-title">Primary Hyperoxaluria</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>369</i></span>,  <span class="NLM_fpage">649</span>– <span class="NLM_lpage">658</span>, <span class="refDoi"> DOI: 10.1056/NEJMra1301564</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1056%2FNEJMra1301564" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=23944302" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlClsrvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=369&publication_year=2013&pages=649-658&author=P.+Cochatauthor=G.+Rumsby&title=Primary+Hyperoxaluria&doi=10.1056%2FNEJMra1301564"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Primary hyperoxaluria</span></div><div class="casAuthors">Cochat, Pierre; Rumsby, Gill</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">369</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">649-658</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review.  The primary hyperoxaluria are a group of autosomal recessive disorders involving the overprodn. of oxalate.  Although the initial recognition of the disease is attributed to Lepoutre, who reported it in 1925,1 the elucidation of the underlying biochem. abnormalities occurred many years later.  This review discusses the major biochem., genetic, and therapeutic advances that have led to a better understanding of primary hyperoxaluria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGUV6DhnokFrVg90H21EOLACvtfcHk0lgLYiLp2J9bdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlClsrvL&md5=40380b669174cdf5be2bf4af052a8118</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1056%2FNEJMra1301564&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra1301564%26sid%3Dliteratum%253Aachs%26aulast%3DCochat%26aufirst%3DP.%26aulast%3DRumsby%26aufirst%3DG.%26atitle%3DPrimary%2520Hyperoxaluria%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2013%26volume%3D369%26spage%3D649%26epage%3D658%26doi%3D10.1056%2FNEJMra1301564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Higueras, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luis-Lima, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salido, E.</span></span> <span> </span><span class="NLM_article-title">Glycolate oxidase is a safe and efficient target for substrate reduction therapy in a mouse model of primary hyperoxaluria type 1</span>. <i>Mol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">719</span>– <span class="NLM_lpage">725</span>, <span class="refDoi"> DOI: 10.1038/mt.2015.224</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1038%2Fmt.2015.224" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=26689264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1altrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=719-725&author=C.+Martin-Higuerasauthor=S.+Luis-Limaauthor=E.+Salido&title=Glycolate+oxidase+is+a+safe+and+efficient+target+for+substrate+reduction+therapy+in+a+mouse+model+of+primary+hyperoxaluria+type+1&doi=10.1038%2Fmt.2015.224"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Glycolate Oxidase Is a Safe and Efficient Target for Substrate Reduction Therapy in a Mouse Model of Primary Hyperoxaluria Type I</span></div><div class="casAuthors">Martin-Higueras, Cristina; Luis-Lima, Sergio; Salido, Eduardo</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Therapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">719-725</span>CODEN:
                <span class="NLM_cas:coden">MTOHCK</span>;
        ISSN:<span class="NLM_cas:issn">1525-0016</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Primary hyperoxaluria type 1 (PH1) is caused by deficient alanine-glyoxylate aminotransferase, the human peroxisomal enzyme that detoxifies glyoxylate.  Glycolate is one of the best-known substrates leading to glyoxylate prodn., via peroxisomal glycolate oxidase (GO).  Using genetically modified mice, we herein report GO as a safe and efficient target for substrate redn. therapy (SRT) in PH1.  We first generated a GO-deficient mouse (Hao1-/-) that presented high urine glycolate levels but no addnl. phenotype.  Next, we produced double KO mice (Agxt1-/-Hao1-/-) that showed low levels of oxalate excretion compared with hyperoxaluric mice model (Agxt1-/-).  Previous studies have identified some GO inhibitors, such as 4-carboxy-5-[(4-chlorophenyl)sulfanyl]-1,2,3-thiadiazole (CCPST).  We herein report that CCPST inhibits GO in Agxt1-/- hepatocytes and significantly reduces their oxalate prodn., starting at 25 μM.  We also tested the ability of orally administered CCPST to reduce oxalate excretion in Agxt1-/- mice, showing that 30-50% redn. in urine oxalate can be achieved.  In summary, we present proof-of-concept evidence for SRT in PH1.  These encouraging results should be followed by a medicinal chem. program that might yield more potent GO inhibitors and eventually could result in a pharmacol. treatment for this rare and severe inborn error of metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnK0-rOTRtBrVg90H21EOLACvtfcHk0lgLYiLp2J9bdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1altrw%253D&md5=3a990cc2a6760420d33438382fef91d6</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fmt.2015.224&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmt.2015.224%26sid%3Dliteratum%253Aachs%26aulast%3DMartin-Higueras%26aufirst%3DC.%26aulast%3DLuis-Lima%26aufirst%3DS.%26aulast%3DSalido%26aufirst%3DE.%26atitle%3DGlycolate%2520oxidase%2520is%2520a%2520safe%2520and%2520efficient%2520target%2520for%2520substrate%2520reduction%2520therapy%2520in%2520a%2520mouse%2520model%2520of%2520primary%2520hyperoxaluria%2520type%25201%26jtitle%3DMol.%2520Ther.%26date%3D2016%26volume%3D24%26spage%3D719%26epage%3D725%26doi%3D10.1038%2Fmt.2015.224" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bourhis, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vignaud, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietrancosta, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gueritte, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guenard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lederer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindqvist, Y.</span></span> <span> </span><span class="NLM_article-title">Structure of human glycolate oxidase in complex with the inhibitor 4-carboxy-5-[(4-chlorophenyl)-sulfanyl]-1,2,3-thiadiazole</span>. <i>Acta Crystallogr., Sect. F: Struct. Biol. Cryst. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>F65</i></span>,  <span class="NLM_fpage">1246</span>– <span class="NLM_lpage">1253</span>, <span class="refDoi"> DOI: 10.1107/S1744309109041670</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1107%2FS1744309109041670" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=F65&publication_year=2009&pages=1246-1253&author=J.-M.+Bourhisauthor=C.+Vignaudauthor=N.+Pietrancostaauthor=F.+Gueritteauthor=D.+Guenardauthor=F.+Ledererauthor=Y.+Lindqvist&title=Structure+of+human+glycolate+oxidase+in+complex+with+the+inhibitor+4-carboxy-5-%5B%284-chlorophenyl%29-sulfanyl%5D-1%2C2%2C3-thiadiazole&doi=10.1107%2FS1744309109041670"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1107%2FS1744309109041670&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1744309109041670%26sid%3Dliteratum%253Aachs%26aulast%3DBourhis%26aufirst%3DJ.-M.%26aulast%3DVignaud%26aufirst%3DC.%26aulast%3DPietrancosta%26aufirst%3DN.%26aulast%3DGueritte%26aufirst%3DF.%26aulast%3DGuenard%26aufirst%3DD.%26aulast%3DLederer%26aufirst%3DF.%26aulast%3DLindqvist%26aufirst%3DY.%26atitle%3DStructure%2520of%2520human%2520glycolate%2520oxidase%2520in%2520complex%2520with%2520the%2520inhibitor%25204-carboxy-5-%255B%25284-chlorophenyl%2529-sulfanyl%255D-1%252C2%252C3-thiadiazole%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520F%253A%2520Struct.%2520Biol.%2520Cryst.%2520Commun.%26date%3D2009%26volume%3DF65%26spage%3D1246%26epage%3D1253%26doi%3D10.1107%2FS1744309109041670" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liebow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Racie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hettinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bettencourt, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najafian, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haslett, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erbe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Querbes, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, J.</span></span> <span> </span><span class="NLM_article-title">An investigational RNAi therapeutic targeting glycolate oxidase reduces oxalate production in models of primary hyperoxaluria</span>. <i>J. Am. Soc. Nephrol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">494</span>– <span class="NLM_lpage">503</span>, <span class="refDoi"> DOI: 10.1681/ASN.2016030338</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1681%2FASN.2016030338" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=27432743" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFGisbfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2017&pages=494-503&author=A.+Liebowauthor=X.+Liauthor=T.+Racieauthor=J.+Hettingerauthor=B.+R.+Bettencourtauthor=N.+Najafianauthor=P.+Haslettauthor=K.+Fitzgeraldauthor=R.+P.+Holmesauthor=D.+Erbeauthor=W.+Querbesauthor=J.+Knight&title=An+investigational+RNAi+therapeutic+targeting+glycolate+oxidase+reduces+oxalate+production+in+models+of+primary+hyperoxaluria&doi=10.1681%2FASN.2016030338"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">An investigational RNAI therapeutic targeting glycolate oxidase reduces oxalate production in models of primary hyperoxaluria</span></div><div class="casAuthors">Liebow, Abigail; Li, Xingsheng; Racie, Timothy; Hettinger, Julia; Bettencourt, Brian R.; Najafian, Nader; Haslett, Patrick; Fitzgerald, Kevin; Holmes, Ross P.; Erbe, David; Querbes, William; Knight, John</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Society of Nephrology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">494-503</span>CODEN:
                <span class="NLM_cas:coden">JASNEU</span>;
        ISSN:<span class="NLM_cas:issn">1046-6673</span>.
    
            (<span class="NLM_cas:orgname">American Society of Nephrology</span>)
        </div><div class="casAbstract">Primary hyperoxaluria type 1 (PH1), an inherited rare disease of glyoxylate metab., arises from mutations in the enzyme alanine-glyoxylate aminotransferase.  The resulting deficiency in this enzyme leads to abnor- mally high oxalate prodn. resulting in calcium oxalate crystal formation and deposition in the kidney and many other tissues, with systemic oxalosis and ESRD being a common outcome.  Although a small subset of patients manages the disease with vitamin B6 treatments, the only effective treatment for most is a combined liver-kidney transplant, which requires life-long immune suppression and carries significant mortality risk.  In this report, we discuss the development of ALN-GO1, an investigational RNA interference (RNAi) therapeutic targeting glycolate oxidase, to deplete the substrate for oxalate synthesis.  S.c. administration of ALN-GO1 resulted in potent, dose-dependent, and durable silencing of the mRNA encoding glycolate oxi- dase and increased serum glycolate concns. in wild-type mice, rats, and nonhuman primates.  ALN- GO1 also increased urinary glycolate concns. in normal nonhuman primates and in a genetic mouse model of PH1.  Notably, ALN-GO1 reduced urinary oxalate concn. up to 50% after a single dose in the genetic mouse model of PH1, and up to 98% after multiple doses in a rat model of hyperoxaluria.  These data demonstratethe ability of ALN-GO1 to reduce oxalate prodn. in preclin. models of PH1 across multiple species and provide a clear rationale for clin. trials with this compd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokWZDdJbT9zLVg90H21EOLACvtfcHk0lgLYiLp2J9bdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFGisbfE&md5=a010927bb16ddfe9fd85c1792dcd95ca</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1681%2FASN.2016030338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1681%252FASN.2016030338%26sid%3Dliteratum%253Aachs%26aulast%3DLiebow%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DRacie%26aufirst%3DT.%26aulast%3DHettinger%26aufirst%3DJ.%26aulast%3DBettencourt%26aufirst%3DB.%2BR.%26aulast%3DNajafian%26aufirst%3DN.%26aulast%3DHaslett%26aufirst%3DP.%26aulast%3DFitzgerald%26aufirst%3DK.%26aulast%3DHolmes%26aufirst%3DR.%2BP.%26aulast%3DErbe%26aufirst%3DD.%26aulast%3DQuerbes%26aufirst%3DW.%26aulast%3DKnight%26aufirst%3DJ.%26atitle%3DAn%2520investigational%2520RNAi%2520therapeutic%2520targeting%2520glycolate%2520oxidase%2520reduces%2520oxalate%2520production%2520in%2520models%2520of%2520primary%2520hyperoxaluria%26jtitle%3DJ.%2520Am.%2520Soc.%2520Nephrol.%26date%3D2017%26volume%3D28%26spage%3D494%26epage%3D503%26doi%3D10.1681%2FASN.2016030338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dutta, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avitahl-Curtis, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pursell, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diwanji, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apponi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ying, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shui, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxena, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cyr, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nazef, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abrams, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dudek, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salido, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, C.</span></span> <span> </span><span class="NLM_article-title">Inhibition of glycolate oxidase with dicer-substrate siRNA reduces calcium oxalate deposition in mouse model of primary hyperoxaluria type 1</span>. <i>Mol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">770</span>– <span class="NLM_lpage">778</span>, <span class="refDoi"> DOI: 10.1038/mt.2016.4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1038%2Fmt.2016.4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=26758691" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1altr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=770-778&author=C.+Duttaauthor=N.+Avitahl-Curtisauthor=N.+Pursellauthor=M.+L.+Cohenauthor=B.+Holmesauthor=R.+Diwanjiauthor=W.+Zhouauthor=L.+Apponiauthor=M.+Koserauthor=B.+Yingauthor=D.+Chenauthor=X.+Shuiauthor=U.+Saxenaauthor=W.+A.+Cyrauthor=A.+Shahauthor=N.+Nazefauthor=W.+Wangauthor=M.+Abramsauthor=H.+Dudekauthor=E.+Salidoauthor=B.+D.+Brownauthor=C.+Lai&title=Inhibition+of+glycolate+oxidase+with+dicer-substrate+siRNA+reduces+calcium+oxalate+deposition+in+mouse+model+of+primary+hyperoxaluria+type+1&doi=10.1038%2Fmt.2016.4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Glycolate Oxidase With Dicer-substrate siRNA Reduces Calcium Oxalate Deposition in a Mouse Model of Primary Hyperoxaluria Type 1</span></div><div class="casAuthors">Dutta, Chaitali; Avitahl-Curtis, Nicole; Pursell, Natalie; Larsson Cohen, Marita; Holmes, Benjamin; Diwanji, Rohan; Zhou, Wei; Apponi, Luciano; Koser, Martin; Ying, Bo; Chen, Dongyu; Shui, Xue; Saxena, Utsav; Cyr, Wendy A.; Shah, Anee; Nazef, Naim; Wang, Weimin; Abrams, Marc; Dudek, Henryk; Salido, Eduardo; Brown, Bob D.; Lai, Chengjung</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Therapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">770-778</span>CODEN:
                <span class="NLM_cas:coden">MTOHCK</span>;
        ISSN:<span class="NLM_cas:issn">1525-0016</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Primary hyperoxaluria type 1 (PH1) is an autosomal recessive, metabolic disorder caused by mutations of alanine-glyoxylate aminotransferase (AGT), a key hepatic enzyme in the detoxification of glyoxylate arising from multiple normal metabolic pathways to glycine.  Accumulation of glyoxylate, a precursor of oxalate, leads to the overprodn. of oxalate in the liver, which accumulates to high levels in kidneys and urine.  Crystalization of calcium oxalate (CaOx) in the kidney ultimately results in renal failure.  Currently, the only treatment effective in redn. of oxalate prodn. in patients who do not respond to high-dose vitamin B6 therapy is a combined liver/kidney transplant.  We explored an alternative approach to prevent glyoxylate prodn. using Dicer-substrate small interfering RNAs (DsiRNAs) targeting hydroxyacid oxidase 1 (HAO1) mRNA which encodes glycolate oxidase (GO), to reduce the hepatic conversion of glycolate to glyoxylate.  This approach efficiently reduces GO mRNA and protein in the livers of mice and nonhuman primates.  Redn. of hepatic GO leads to normalization of urine oxalate levels and reduces CaOx deposition in a preclin. mouse model of PH1.  Our results support the use of DsiRNA to reduce liver GO levels as a potential therapeutic approach to treat PH1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYr_lKKb91mrVg90H21EOLACvtfcHk0lifsf9aI1PcFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1altr8%253D&md5=8105de778740769b9cdb5a4db7b6551f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fmt.2016.4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmt.2016.4%26sid%3Dliteratum%253Aachs%26aulast%3DDutta%26aufirst%3DC.%26aulast%3DAvitahl-Curtis%26aufirst%3DN.%26aulast%3DPursell%26aufirst%3DN.%26aulast%3DCohen%26aufirst%3DM.%2BL.%26aulast%3DHolmes%26aufirst%3DB.%26aulast%3DDiwanji%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DApponi%26aufirst%3DL.%26aulast%3DKoser%26aufirst%3DM.%26aulast%3DYing%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DShui%26aufirst%3DX.%26aulast%3DSaxena%26aufirst%3DU.%26aulast%3DCyr%26aufirst%3DW.%2BA.%26aulast%3DShah%26aufirst%3DA.%26aulast%3DNazef%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DAbrams%26aufirst%3DM.%26aulast%3DDudek%26aufirst%3DH.%26aulast%3DSalido%26aufirst%3DE.%26aulast%3DBrown%26aufirst%3DB.%2BD.%26aulast%3DLai%26aufirst%3DC.%26atitle%3DInhibition%2520of%2520glycolate%2520oxidase%2520with%2520dicer-substrate%2520siRNA%2520reduces%2520calcium%2520oxalate%2520deposition%2520in%2520mouse%2520model%2520of%2520primary%2520hyperoxaluria%2520type%25201%26jtitle%3DMol.%2520Ther.%26date%3D2016%26volume%3D24%26spage%3D770%26epage%3D778%26doi%3D10.1038%2Fmt.2016.4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Maag, H.</span>; <span class="NLM_string-name">Fernandes, M. X.</span></span> <span> </span><span class="NLM_article-title">Glycolate oxidase inhibitors and use thereof</span>. <span class="NLM_patent">WO2019133813A1</span>, July 4, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=H.+Maag&author=M.+X.+Fernandes&title=Glycolate+oxidase+inhibitors+and+use+thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMaag%26aufirst%3DH.%26atitle%3DGlycolate%2520oxidase%2520inhibitors%2520and%2520use%2520thereof%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit6b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Maag, H.</span>; <span class="NLM_string-name">Fernandes, M. X.</span>; <span class="NLM_string-name">Zamboni, R.</span>; <span class="NLM_string-name">Akbariromani, E.</span>; <span class="NLM_string-name">Beaulieu, M.-A.</span>; <span class="NLM_string-name">Leblanc, Y.</span>; <span class="NLM_string-name">Thakur, P.</span></span> <span> </span><span class="NLM_article-title">Triazole clycolate oxidase inhibitors</span>. <span class="NLM_patent">WO2020010309A9</span>, January 9, <span class="NLM_year">2020</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=H.+Maag&author=M.+X.+Fernandes&author=R.+Zamboni&author=E.+Akbariromani&author=M.-A.+Beaulieu&author=Y.+Leblanc&author=P.+Thakur&title=Triazole+clycolate+oxidase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMaag%26aufirst%3DH.%26atitle%3DTriazole%2520clycolate%2520oxidase%2520inhibitors%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span>; <span class="NLM_string-name">Chao, Q.</span></span> <span> </span><span class="NLM_article-title">Glycolate oxidase inhibitors for the treatment of disease</span>. <span class="NLM_patent">WO2019133770A2</span>, July 4, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=B.+Wang&author=Q.+Chao&title=Glycolate+oxidase+inhibitors+for+the+treatment+of+disease"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DB.%26atitle%3DGlycolate%2520oxidase%2520inhibitors%2520for%2520the%2520treatment%2520of%2520disease%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keller, B. T.</span>; <span class="NLM_string-name">Talley, J. J.</span></span> <span> </span><span class="NLM_article-title">Compounds and methods for treating oxalate-related diseases</span>. <span class="NLM_patent">WO2019165159A1</span>, August 29, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=B.+T.+Keller&author=J.+J.+Talley&title=Compounds+and+methods+for+treating+oxalate-related+diseases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DKeller%26aufirst%3DB.%2BT.%26atitle%3DCompounds%2520and%2520methods%2520for%2520treating%2520oxalate-related%2520diseases%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowther, W. T.</span></span> <span> </span><span class="NLM_article-title">Active site and loop 4 movements within human glycolate oxidase: Implications for substrate specificity and drug design</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">2439</span>– <span class="NLM_lpage">2449</span>, <span class="refDoi"> DOI: 10.1021/bi701710r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi701710r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptlylsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=2439-2449&author=M.+S.+Murrayauthor=R.+P.+Holmesauthor=W.+T.+Lowther&title=Active+site+and+loop+4+movements+within+human+glycolate+oxidase%3A+Implications+for+substrate+specificity+and+drug+design&doi=10.1021%2Fbi701710r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Active Site and Loop 4 Movements within Human Glycolate Oxidase: Implications for Substrate Specificity and Drug Design</span></div><div class="casAuthors">Murray, Michael S.; Holmes, Ross P.; Lowther, W. Todd</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2439-2449</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human glycolate oxidase (GO) catalyzes the FMN-dependent oxidn. of glycolate to glyoxylate and glyoxylate to oxalate, a key metabolite in kidney stone formation.  We report herein the structures of recombinant GO complexed with sulfate, glyoxylate, and an inhibitor, 4-carboxy-5-dodecylsulfanyl-1,2,3-triazole (CDST), detd. by X-ray crystallog.  In contrast to most α-hydroxy acid oxidases including spinach glycolate oxidase, a loop region, known as loop 4, is completely visible when the GO active site contains a small ligand.  The lack of electron d. for this loop in the GO-CDST complex, which mimics a large substrate, suggests that a disordered to ordered transition may occur with the binding of substrates.  The conformational flexibility of Trp110 appears to be responsible for enabling GO to react with α-hydroxy acids of various chain lengths.  Moreover, the movement of Trp110 disrupts a hydrogen-bonding network between Trp110, Leu191, Tyr134, and Tyr208.  This loss of interactions is the first indication that active site movements are directly linked to changes in the conformation of loop 4.  The kinetic parameters for the oxidn. of glycolate, glyoxylate, and 2-hydroxy octanoate indicate that the oxidn. of glycolate to glyoxylate is the primary reaction catalyzed by GO, while the oxidn. of glyoxylate to oxalate is most likely not relevant under normal conditions.  However, drugs that exploit the unique structural features of GO may ultimately prove to be useful for decreasing glycolate and glyoxylate levels in primary hyperoxaluria type 1 patients who have the inability to convert peroxisomal glyoxylate to glycine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprc1FGNg286bVg90H21EOLACvtfcHk0ljD6h7VzaB1Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptlylsQ%253D%253D&md5=0eacdcc4727c929f4669852dd0c627c7</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fbi701710r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi701710r%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DM.%2BS.%26aulast%3DHolmes%26aufirst%3DR.%2BP.%26aulast%3DLowther%26aufirst%3DW.%2BT.%26atitle%3DActive%2520site%2520and%2520loop%25204%2520movements%2520within%2520human%2520glycolate%2520oxidase%253A%2520Implications%2520for%2520substrate%2520specificity%2520and%2520drug%2520design%26jtitle%3DBiochemistry%26date%3D2008%26volume%3D47%26spage%3D2439%26epage%3D2449%26doi%3D10.1021%2Fbi701710r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Randall, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Streeter, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziegler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cragoe, E. J.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwam, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michelson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, H. W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eichler, E.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of glycolic acid oxidase. 4-Substituted 3-hydroxy-1H-pyrrole-2,5-dione derivatives</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">700</span>– <span class="NLM_lpage">714</span>, <span class="refDoi"> DOI: 10.1021/jm00359a015</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00359a015" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADyaL3sXktVSitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1983&pages=700-714&author=C.+S.+Rooneyauthor=W.+C.+Randallauthor=K.+B.+Streeterauthor=C.+Zieglerauthor=E.+J.+Cragoeauthor=H.+Schwamauthor=S.+R.+Michelsonauthor=H.+W.+R.+Williamsauthor=E.+Eichler&title=Inhibitors+of+glycolic+acid+oxidase.+4-Substituted+3-hydroxy-1H-pyrrole-2%2C5-dione+derivatives&doi=10.1021%2Fjm00359a015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10aR"><div class="casContent"><span class="casTitleNuber">10a</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of glycolic acid oxidase.  4-Substituted 3-hydroxy-1H-pyrrole-2,5-dione derivatives</span></div><div class="casAuthors">Rooney, C. S.; Randall, W. C.; Streeter, K. B.; Ziegler, C.; Cragoe, E. J., Jr.; Schwam, H.; Michelson, S. R.; Williams, H. W. R.; Eichler, E.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">700-14</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">The title compds. I (R = aryl, aryloxy, heterocycle, etc.; R1 = H, Me, or Ph) and related ones were prepd. and evaluated as orally absorbable inhibitors of glycolic acid oxidase (GAO)  [9028-71-1] and to examine their effect on oxalate prodn. in animals.  Inhibitors of GAO are of interest as potentially useful drugs for treatment of Ca oxalate renal lithiasis and the primary hyperoxalurias.  Rat liver perfusion with 3 of the most potent I have shown effective inhibition of the conversion of glycolate to oxalate in this organ.  4-(4'-Bromo[1,1'-biphenyl]-4-yl)-3-hydroxy-1H-pyrrole-2,5-dione (I; R = C6H4-4-(4-BrC6H4); R1 = H)  [77529-42-1] administered orally to ethylene glycol treated rats showed redn. in oxalate excretion in urine.  QSAR are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9vZzyUCBtEbVg90H21EOLACvtfcHk0ljD6h7VzaB1Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXktVSitrk%253D&md5=e638bd095f104778375ac10a55ee34d2</span></div><a href="/servlet/linkout?suffix=cit10a&amp;dbid=16384&amp;doi=10.1021%2Fjm00359a015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00359a015%26sid%3Dliteratum%253Aachs%26aulast%3DRooney%26aufirst%3DC.%2BS.%26aulast%3DRandall%26aufirst%3DW.%2BC.%26aulast%3DStreeter%26aufirst%3DK.%2BB.%26aulast%3DZiegler%26aufirst%3DC.%26aulast%3DCragoe%26aufirst%3DE.%2BJ.%26aulast%3DSchwam%26aufirst%3DH.%26aulast%3DMichelson%26aufirst%3DS.%2BR.%26aulast%3DWilliams%26aufirst%3DH.%2BW.%2BR.%26aulast%3DEichler%26aufirst%3DE.%26atitle%3DInhibitors%2520of%2520glycolic%2520acid%2520oxidase.%25204-Substituted%25203-hydroxy-1H-pyrrole-2%252C5-dione%2520derivatives%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1983%26volume%3D26%26spage%3D700%26epage%3D714%26doi%3D10.1021%2Fjm00359a015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit10b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, H. W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eichler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Randall, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cragoe, E. J.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Streeter, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwam, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michelson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patchett, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taub, D.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of glycolic acid oxidase. 4-Substituted 2,4-dioxobutanoic acid derivatives</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1196</span>– <span class="NLM_lpage">1200</span>, <span class="refDoi"> DOI: 10.1021/jm00362a020</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00362a020" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADyaL3sXktlyntr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1983&pages=1196-1200&author=H.+W.+R.+Williamsauthor=E.+Eichlerauthor=W.+C.+Randallauthor=C.+S.+Rooneyauthor=E.+J.+Cragoeauthor=K.+B.+Streeterauthor=H.+Schwamauthor=S.+R.+Michelsonauthor=A.+A.+Patchettauthor=D.+Taub&title=Inhibitors+of+glycolic+acid+oxidase.+4-Substituted+2%2C4-dioxobutanoic+acid+derivatives&doi=10.1021%2Fjm00362a020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10bR"><div class="casContent"><span class="casTitleNuber">10b</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of glycolic acid oxidase.  4-Substituted-2,4-dioxobutanoic acid derivatives</span></div><div class="casAuthors">Williams, H. W. R.; Eichler, E.; Randall, W. C.; Rooney, C. S.; Cragoe, E. J., Jr.; Streeter, K. B.; Schwam, H.; Michelson, S. R.; Patchett, A. A.; Taub, D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1196-200</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">Fourteen new 4-substituted 2,4-dioxobutanoic acids were synthesized.  These compds., all of which contain lipophilic 4-substituents, are potent inhibitors in vitro of porcine liver glycolic acid oxidase.  The I50 values of the 2 most potent representatives, 4-(4-BrC6H4)C6H4COCH2COCO2H and I are 6 × 10-8M.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEVw46DI-rQ7Vg90H21EOLACvtfcHk0ljD6h7VzaB1Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXktlyntr0%253D&md5=45d00f0ed3f769b7e46f9e9b5dcdc697</span></div><a href="/servlet/linkout?suffix=cit10b&amp;dbid=16384&amp;doi=10.1021%2Fjm00362a020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00362a020%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DH.%2BW.%2BR.%26aulast%3DEichler%26aufirst%3DE.%26aulast%3DRandall%26aufirst%3DW.%2BC.%26aulast%3DRooney%26aufirst%3DC.%2BS.%26aulast%3DCragoe%26aufirst%3DE.%2BJ.%26aulast%3DStreeter%26aufirst%3DK.%2BB.%26aulast%3DSchwam%26aufirst%3DH.%26aulast%3DMichelson%26aufirst%3DS.%2BR.%26aulast%3DPatchett%26aufirst%3DA.%2BA.%26aulast%3DTaub%26aufirst%3DD.%26atitle%3DInhibitors%2520of%2520glycolic%2520acid%2520oxidase.%25204-Substituted%25202%252C4-dioxobutanoic%2520acid%2520derivatives%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1983%26volume%3D26%26spage%3D1196%26epage%3D1200%26doi%3D10.1021%2Fjm00362a020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moya-Garzón, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higueras, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penalver, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandes, M. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franco-Montalban, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez-Vidal, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salido, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz-Gavilan, M.</span></span> <span> </span><span class="NLM_article-title">Salicylic acid derivatives inhibit oxalate production in mouse hepatocytes with primary hyperoxaluria type 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">7144</span>– <span class="NLM_lpage">7167</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00399</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00399" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlGit7zM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=7144-7167&author=M.+D.+Moya-Garz%C3%B3nauthor=C.+M.+Higuerasauthor=P.+Penalverauthor=M.+Romeraauthor=M.+X.+Fernandesauthor=F.+Franco-Montalbanauthor=J.+A.+Gomez-Vidalauthor=E.+Salidoauthor=M.+Diaz-Gavilan&title=Salicylic+acid+derivatives+inhibit+oxalate+production+in+mouse+hepatocytes+with+primary+hyperoxaluria+type+1&doi=10.1021%2Facs.jmedchem.8b00399"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Salicylic Acid Derivatives Inhibit Oxalate Production in Mouse Hepatocytes with Primary Hyperoxaluria Type 1</span></div><div class="casAuthors">Moya-Garzon, Maria Dolores; Martin Higueras, Cristina; Penalver, Pablo; Romera, Manuela; Fernandes, Miguel X.; Franco-Montalban, Francisco; Gomez-Vidal, Jose A.; Salido, Eduardo; Diaz-Gavilan, Monica</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7144-7167</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Primary hyperoxaluria type 1 (PH1) is a rare life-threatening genetic disease related to glyoxylate metab. and characterized by accumulation of calcium oxalate crystals.  Current therapies involve hepatic and/or renal transplantation, procedures that have significant morbidity and mortality and require long-term immunosuppression.  Thus, a pharmacol. treatment is urgently needed.  We introduce here an unprecedented activity of salicylic acid derivs. as agents capable of decreasing oxalate output in hyperoxaluric hepatocytes at the low micromolar range, which means a potential use in the treatment of PH1.  Though correlation of this phenotypic activity with glycolate oxidase (GO) inhibition is still to be verified, most of the salicylic acids described here are GO inhibitors with IC50 values down to 3 μM.  Binding mode of salicylic acids inside GO has been studied using in silico methods, and preliminary structure-activity relationships have been established.  The drug-like structure and ease of synthesis of our compds. make them promising hits for structural optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwMGp3szKEj7Vg90H21EOLACvtfcHk0lhiTvoj5HaIMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlGit7zM&md5=457a9b87854dd7efae5abd0380dc2da5</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00399%26sid%3Dliteratum%253Aachs%26aulast%3DMoya-Garz%25C3%25B3n%26aufirst%3DM.%2BD.%26aulast%3DHigueras%26aufirst%3DC.%2BM.%26aulast%3DPenalver%26aufirst%3DP.%26aulast%3DRomera%26aufirst%3DM.%26aulast%3DFernandes%26aufirst%3DM.%2BX.%26aulast%3DFranco-Montalban%26aufirst%3DF.%26aulast%3DGomez-Vidal%26aufirst%3DJ.%2BA.%26aulast%3DSalido%26aufirst%3DE.%26aulast%3DDiaz-Gavilan%26aufirst%3DM.%26atitle%3DSalicylic%2520acid%2520derivatives%2520inhibit%2520oxalate%2520production%2520in%2520mouse%2520hepatocytes%2520with%2520primary%2520hyperoxaluria%2520type%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D7144%26epage%3D7167%26doi%3D10.1021%2Facs.jmedchem.8b00399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arico-Muendel, C. C.</span></span> <span> </span><span class="NLM_article-title">From haystack to needle: Finding value with DNA encoded library technology at GSK</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1898</span>– <span class="NLM_lpage">1909</span>, <span class="refDoi"> DOI: 10.1039/C6MD00341A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1039%2FC6MD00341A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1ehu7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=1898-1909&author=C.+C.+Arico-Muendel&title=From+haystack+to+needle%3A+Finding+value+with+DNA+encoded+library+technology+at+GSK&doi=10.1039%2FC6MD00341A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">From haystack to needle: finding value with DNA encoded library technology at GSK</span></div><div class="casAuthors">Arico-Muendel, Christopher C.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1898-1909</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">DNA encoded library technol. (ELT) provides access to broad chem. diversity through affinity selection.  As an early adopter, GlaxoSmithKline enabled the development of ELT from proof of concept to full fledged contributor to the small mol. therapeutic pipeline.  Unique benefits of the GSK incubation include uptake of large nos. of building blocks and privileged scaffolds, guidance on library design, and opportunities to add value beyond lead discovery.  Sol. epoxide hydrolase and RIP1 kinase represent two case studies of ELT hits that have advanced into clin. studies.  ELT also shows promise as a strategy to forecast target tractability.  As an emerging technol., it benefits from incubation at the interface of industry and academia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGhx5inxOj_7Vg90H21EOLACvtfcHk0lhiTvoj5HaIMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1ehu7jL&md5=1862af051a956ff1893143e107cff4f4</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1039%2FC6MD00341A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00341A%26sid%3Dliteratum%253Aachs%26aulast%3DArico-Muendel%26aufirst%3DC.%2BC.%26atitle%3DFrom%2520haystack%2520to%2520needle%253A%2520Finding%2520value%2520with%2520DNA%2520encoded%2520library%2520technology%2520at%2520GSK%26jtitle%3DMedChemComm%26date%3D2016%26volume%3D7%26spage%3D1898%26epage%3D1909%26doi%3D10.1039%2FC6MD00341A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halford, B.</span></span> <span> </span><span class="NLM_article-title">How DNA-encoded libraries are revolutionizing drug discovery</span>. <i>Chem. Eng. News</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">28</span>– <span class="NLM_lpage">33</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2017&pages=28-33&author=B.+Halford&title=How+DNA-encoded+libraries+are+revolutionizing+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHalford%26aufirst%3DB.%26atitle%3DHow%2520DNA-encoded%2520libraries%2520are%2520revolutionizing%2520drug%2520discovery%26jtitle%3DChem.%2520Eng.%2520News%26date%3D2017%26volume%3D95%26spage%3D28%26epage%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goodnow, R. A.,  Jr.</span>; <span class="NLM_string-name">Davie, C. P.</span></span> <span> </span><span class="NLM_article-title">DNA-encoded library: a brief guide to its evolution and impact on drug discovery</span>. In  <i>Annual Reports in Medicinal Chemistry</i>; <span class="NLM_publisher-name">Elsevier Inc.</span>, <span class="NLM_year">2017</span>; Vol.  <span class="NLM_volume">50</span>, pp  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">15</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=1-15&author=R.+A.+Goodnow&author=C.+P.+Davie&title=Annual+Reports+in+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DGoodnow%26aufirst%3DR.%2BA.%26atitle%3DDNA-encoded%2520library%253A%2520a%2520brief%2520guide%2520to%2520its%2520evolution%2520and%2520impact%2520on%2520drug%2520discovery%26btitle%3DAnnual%2520Reports%2520in%2520Medicinal%2520Chemistry%26pub%3DElsevier%2520Inc%26date%3D2017%26volume%3D50%26spage%3D1%26epage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gikunju, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habeshian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hupp, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soutter, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent BTK inhibitor with a novel binding mode using parallel selections with a DNA-encoded chemical library</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">864</span>– <span class="NLM_lpage">871</span>, <span class="refDoi"> DOI: 10.1002/cbic.201600573</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1002%2Fcbic.201600573" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=28056160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC2sXit1Chu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=864-871&author=J.+W.+Cuozzoauthor=P.+A.+Centrellaauthor=D.+Gikunjuauthor=S.+Habeshianauthor=C.+D.+Huppauthor=A.+D.+Keefeauthor=E.+A.+Sigelauthor=H.+H.+Soutterauthor=H.+A.+Thomsonauthor=Y.+Zhangauthor=M.+A.+Clark&title=Discovery+of+a+potent+BTK+inhibitor+with+a+novel+binding+mode+using+parallel+selections+with+a+DNA-encoded+chemical+library&doi=10.1002%2Fcbic.201600573"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent BTK Inhibitor with a Novel Binding Mode by Using Parallel Selections with a DNA-Encoded Chemical Library</span></div><div class="casAuthors">Cuozzo, John W.; Centrella, Paolo A.; Gikunju, Diana; Habeshian, Sevan; Hupp, Christopher D.; Keefe, Anthony D.; Sigel, Eric A.; Soutter, Holly H.; Thomson, Heather A.; Zhang, Ying; Clark, Matthew A.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">864-871</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">We have identified and characterized novel potent inhibitors of Bruton's tyrosine kinase (BTK) from a single DNA-encoded library of over 110 million compds. by using multiple parallel selection conditions, including variation in target concn. and addn. of known binders to provide competition information.  Distinct binding profiles were obsd. by comparing enrichments of library building block combinations under these conditions; one enriched only at high concns. of BTK and was competitive with ATP, and another enriched at both high and low concns. of BTK and was not competitive with ATP.  A compd. representing the latter profile showed low nanomolar potency in biochem. and cellular BTK assays.  Results from kinetic mechanism of action studies were consistent with the selection profiles.  Anal. of the co-crystal structure of the most potent compd. demonstrated a novel binding mode that revealed a new pocket in BTK.  Our results demonstrate that profile-based selection strategies using DNA-encoded libraries form the basis of a new methodol. to rapidly identify small mol. inhibitors with novel binding modes to clin. relevant targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxaUUXy-jOi7Vg90H21EOLACvtfcHk0lhiTvoj5HaIMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXit1Chu7Y%253D&md5=22423b0496eb68130236292433bc7251</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201600573&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201600573%26sid%3Dliteratum%253Aachs%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DGikunju%26aufirst%3DD.%26aulast%3DHabeshian%26aufirst%3DS.%26aulast%3DHupp%26aufirst%3DC.%2BD.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DSoutter%26aufirst%3DH.%2BH.%26aulast%3DThomson%26aufirst%3DH.%2BA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DClark%26aufirst%3DM.%2BA.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520BTK%2520inhibitor%2520with%2520a%2520novel%2520binding%2520mode%2520using%2520parallel%2520selections%2520with%2520a%2520DNA-encoded%2520chemical%2520library%26jtitle%3DChemBioChem%26date%3D2017%26volume%3D18%26spage%3D864%26epage%3D871%26doi%3D10.1002%2Fcbic.201600573" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulvihill, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renzetti, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resnicow, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruebsam, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">Novel Autotaxin Inhibitor for the Treatment of Idiopathic Pulmonary Fibrosis: A Clinical Candidate Discovered Using DNA-Encoded Chemistry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">7840</span>– <span class="NLM_lpage">7856</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00688</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00688" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1CrtL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=7840-7856&author=J.+W.+Cuozzoauthor=M.+A.+Clarkauthor=A.+D.+Keefeauthor=A.+Kohlmannauthor=M.+Mulvihillauthor=H.+Niauthor=L.+M.+Renzettiauthor=D.+I.+Resnicowauthor=F.+Ruebsamauthor=E.+A.+Sigelauthor=H.+A.+Thomsonauthor=C.+Wangauthor=Z.+Xieauthor=Y.+Zhang&title=Novel+Autotaxin+Inhibitor+for+the+Treatment+of+Idiopathic+Pulmonary+Fibrosis%3A+A+Clinical+Candidate+Discovered+Using+DNA-Encoded+Chemistry&doi=10.1021%2Facs.jmedchem.0c00688"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Autotaxin Inhibitor for the Treatment of Idiopathic Pulmonary Fibrosis: A Clinical Candidate Discovered Using DNA-Encoded Chemistry</span></div><div class="casAuthors">Cuozzo, John W.; Clark, Matthew A.; Keefe, Anthony D.; Kohlmann, Anna; Mulvihill, Mark; Ni, Haihong; Renzetti, Louis M.; Resnicow, Daniel I.; Ruebsam, Frank; Sigel, Eric A.; Thomson, Heather A.; Wang, Ce; Xie, Zhifeng; Zhang, Ying</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">7840-7856</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The activity of the secreted phosphodiesterase autotaxin produces the inflammatory signaling mol. LPA and has been assocd. with a no. of human diseases including idiopathic pulmonary fibrosis (IPF).  We screened a single DNA-encoded chem. library (DECL) of 225 million compds. and identified a series of potent inhibitors.  Optimization of this series led to the discovery of compd. 1 (X-165)(I), a highly potent, selective, and bioavailable small mol.  Cocrystn. of compd. 1 with human autotaxin demonstrated that it has a novel binding mode occupying both the hydrophobic pocket and a channel near the autotaxin active site.  Compd. 1 inhibited the prodn. of LPA in human and mouse plasma at nanomolar levels and showed efficacy in a mouse model of human lung fibrosis.  After successfully completing IND-enabling studies, compd. 1 was approved by the FDA for a Phase I clin. trial.  These results demonstrate that DECL hits can be readily optimized into clin. candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaofuGyFvOJrVg90H21EOLACvtfcHk0lgVgDfpN53Ngw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1CrtL%252FM&md5=c945f9bb71089c94314a27afce04fb96</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00688%26sid%3Dliteratum%253Aachs%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DKohlmann%26aufirst%3DA.%26aulast%3DMulvihill%26aufirst%3DM.%26aulast%3DNi%26aufirst%3DH.%26aulast%3DRenzetti%26aufirst%3DL.%2BM.%26aulast%3DResnicow%26aufirst%3DD.%2BI.%26aulast%3DRuebsam%26aufirst%3DF.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DThomson%26aufirst%3DH.%2BA.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DXie%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DNovel%2520Autotaxin%2520Inhibitor%2520for%2520the%2520Treatment%2520of%2520Idiopathic%2520Pulmonary%2520Fibrosis%253A%2520A%2520Clinical%2520Candidate%2520Discovered%2520Using%2520DNA-Encoded%2520Chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D7840%26epage%3D7856%26doi%3D10.1021%2Facs.jmedchem.0c00688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bazzaz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackett, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collingwood, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disch, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gikunju, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guilinger, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardaker, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennessy, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jetson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoull, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olszewski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pflug, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawlins, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivers, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schimpl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Underwood, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warwicker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter-Holt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodcock, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">Generating Selective Leads for Mer Kinase Inhibitors-Example of a Comprehensive Lead-Generation Strategy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">3165</span>– <span class="NLM_lpage">3184</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01904</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01904" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BB3MXlslWgsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2021&pages=3165-3184&author=J.+W.+M.+Nissinkauthor=A.+Bazzazauthor=C.+Blackettauthor=M.+A.+Clarkauthor=O.+Collingwoodauthor=J.+S.+Dischauthor=D.+Gikunjuauthor=K.+Goldbergauthor=J.+P.+Guilingerauthor=E.+Hardakerauthor=E.+J.+Hennessyauthor=R.+Jetsonauthor=A.+D.+Keefeauthor=W.+McCoullauthor=L.+McMurrayauthor=A.+Olszewskiauthor=R.+Overmanauthor=A.+Pflugauthor=M.+Prestonauthor=P.+B.+Rawlinsauthor=E.+Riversauthor=M.+Schimplauthor=P.+Smithauthor=C.+Trumanauthor=E.+Underwoodauthor=J.+Warwickerauthor=J.+Winter-Holtauthor=S.+Woodcockauthor=Y.+Zhang&title=Generating+Selective+Leads+for+Mer+Kinase+Inhibitors-Example+of+a+Comprehensive+Lead-Generation+Strategy&doi=10.1021%2Facs.jmedchem.0c01904"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Generating Selective Leads for Mer Kinase Inhibitors-Example of a Comprehensive Lead-Generation Strategy</span></div><div class="casAuthors">Nissink, J. Willem M.; Bazzaz, Sana; Blackett, Carolyn; Clark, Matthew A.; Collingwood, Olga; Disch, Jeremy S.; Gikunju, Diana; Goldberg, Kristin; Guilinger, John P.; Hardaker, Elizabeth; Hennessy, Edward J.; Jetson, Rachael; Keefe, Anthony D.; McCoull, William; McMurray, Lindsay; Olszewski, Allison; Overman, Ross; Pflug, Alexander; Preston, Marian; Rawlins, Philip B.; Rivers, Emma; Schimpl, Marianne; Smith, Paul; Truman, Caroline; Underwood, Elizabeth; Warwicker, Juli; Winter-Holt, Jon; Woodcock, Simon; Zhang, Ying</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3165-3184</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mer is a member of the TAM (Tyro3, Axl, Mer) kinase family that has been assocd. with cancer progression, metastasis, and drug resistance.  Their essential function in immune homeostasis has prompted an interest in their role as modulators of antitumor immune response in the tumor microenvironment.  Here we illustrate the outcomes of an extensive lead-generation campaign for identification of Mer inhibitors, focusing on the results from concurrent, orthogonal high-throughput screening approaches.  Data mining, HT (high-throughput), and DECL (DNA-encoded chem. library) screens offered means to evaluate large nos. of compds.  We discuss campaign strategy and screening outcomes, and exemplify series resulting from prioritization of hits that were identified.  Concurrent execution of HT and DECL screening successfully yielded a large no. of potent, selective, and novel starting points, covering a range of selectivity profiles across the TAM family members and modes of kinase binding, and offered excellent start points for lead development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ_gjdZIheGLVg90H21EOLACvtfcHk0lgVgDfpN53Ngw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXlslWgsbY%253D&md5=35131754287b08fc9c30d3989d42d52f</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01904%26sid%3Dliteratum%253Aachs%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DBazzaz%26aufirst%3DA.%26aulast%3DBlackett%26aufirst%3DC.%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DCollingwood%26aufirst%3DO.%26aulast%3DDisch%26aufirst%3DJ.%2BS.%26aulast%3DGikunju%26aufirst%3DD.%26aulast%3DGoldberg%26aufirst%3DK.%26aulast%3DGuilinger%26aufirst%3DJ.%2BP.%26aulast%3DHardaker%26aufirst%3DE.%26aulast%3DHennessy%26aufirst%3DE.%2BJ.%26aulast%3DJetson%26aufirst%3DR.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DMcCoull%26aufirst%3DW.%26aulast%3DMcMurray%26aufirst%3DL.%26aulast%3DOlszewski%26aufirst%3DA.%26aulast%3DOverman%26aufirst%3DR.%26aulast%3DPflug%26aufirst%3DA.%26aulast%3DPreston%26aufirst%3DM.%26aulast%3DRawlins%26aufirst%3DP.%2BB.%26aulast%3DRivers%26aufirst%3DE.%26aulast%3DSchimpl%26aufirst%3DM.%26aulast%3DSmith%26aufirst%3DP.%26aulast%3DTruman%26aufirst%3DC.%26aulast%3DUnderwood%26aufirst%3DE.%26aulast%3DWarwicker%26aufirst%3DJ.%26aulast%3DWinter-Holt%26aufirst%3DJ.%26aulast%3DWoodcock%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DGenerating%2520Selective%2520Leads%2520for%2520Mer%2520Kinase%2520Inhibitors-Example%2520of%2520a%2520Comprehensive%2520Lead-Generation%2520Strategy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2021%26volume%3D64%26spage%3D3165%26epage%3D3184%26doi%3D10.1021%2Facs.jmedchem.0c01904" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Veerman, J. J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruseker, Y. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heijne, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Bruggen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hekking, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hupp, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McRiner, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulvihill, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Rijnsbergen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zech, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renzetti, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babiss, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, G.</span></span> <span> </span><span class="NLM_article-title">Discovery of 2,4-1H-Imidazole Carboxamides as Potent and Selective TAK1 Inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">555</span>– <span class="NLM_lpage">562</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.0c00547</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.0c00547" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BB3MXltl2lt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2021&pages=555-562&author=J.+J.+N.+Veermanauthor=Y.+B.+Brusekerauthor=E.+Damenauthor=E.+H.+Heijneauthor=W.+van+Bruggenauthor=K.+F.+Hekkingauthor=R.+Winkelauthor=C.+D.+Huppauthor=A.+D.+Keefeauthor=J.+Liuauthor=H.+A.+Thomsonauthor=Y.+Zhangauthor=J.+W.+Cuozzoauthor=A.+J.+McRinerauthor=M.+J.+Mulvihillauthor=P.+van+Rijnsbergenauthor=B.+Zechauthor=L.+M.+Renzettiauthor=L.+Babissauthor=G.+Muller&title=Discovery+of+2%2C4-1H-Imidazole+Carboxamides+as+Potent+and+Selective+TAK1+Inhibitors&doi=10.1021%2Facsmedchemlett.0c00547"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 2,4-1H-Imidazole Carboxamides as Potent and Selective TAK1 Inhibitors</span></div><div class="casAuthors">Veerman, Johan J. N.; Bruseker, Yorik B.; Damen, Eddy; Heijne, Erik H.; van Bruggen, Wendy; Hekking, Koen F. W.; Winkel, Rob; Hupp, Christopher D.; Keefe, Anthony D.; Liu, Julie; Thomson, Heather A.; Zhang, Ying; Cuozzo, John W.; McRiner, Andrew J.; Mulvihill, Mark J.; van Rijnsbergen, Peter; Zech, Birgit; Renzetti, Louis M.; Babiss, Lee; Mueller, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">555-562</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Herein we report the discovery of 2,4-1H-imidazole carboxamides as novel, biochem. potent, and kinome selective inhibitors of transforming growth factor β-activated kinase 1 (TAK1).  The target was subjected to a DNA-encoded chem. library (DECL) screen.  After hit anal. a cluster of compds. was identified, which was based on a central pyrrole-2,4-1H-dicarboxamide scaffold, showing remarkable kinome selectivity.  A scaffold-hop to the corresponding imidazole resulted in increased biochem. potency.  Next, X-ray crystallog. revealed a distinct binding mode compared to other TAK1 inhibitors.  A benzylamide was found in a perpendicular orientation with respect to the core hinge-binding imidazole.  Addnl., an unusual amide flip was obsd. in the kinase hinge region.  Using structure-based drug design (SBDD), key substitutions at the pyrrolidine amide and the glycine resulted in a significant increase in biochem. potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsfdQbRX4C1LVg90H21EOLACvtfcHk0lhI02liLHlUZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXltl2lt7o%253D&md5=26c25af441806342c2048ce49c6be192</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.0c00547&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.0c00547%26sid%3Dliteratum%253Aachs%26aulast%3DVeerman%26aufirst%3DJ.%2BJ.%2BN.%26aulast%3DBruseker%26aufirst%3DY.%2BB.%26aulast%3DDamen%26aufirst%3DE.%26aulast%3DHeijne%26aufirst%3DE.%2BH.%26aulast%3Dvan%2BBruggen%26aufirst%3DW.%26aulast%3DHekking%26aufirst%3DK.%2BF.%26aulast%3DWinkel%26aufirst%3DR.%26aulast%3DHupp%26aufirst%3DC.%2BD.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DThomson%26aufirst%3DH.%2BA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DMcRiner%26aufirst%3DA.%2BJ.%26aulast%3DMulvihill%26aufirst%3DM.%2BJ.%26aulast%3Dvan%2BRijnsbergen%26aufirst%3DP.%26aulast%3DZech%26aufirst%3DB.%26aulast%3DRenzetti%26aufirst%3DL.%2BM.%26aulast%3DBabiss%26aufirst%3DL.%26aulast%3DMuller%26aufirst%3DG.%26atitle%3DDiscovery%2520of%25202%252C4-1H-Imidazole%2520Carboxamides%2520as%2520Potent%2520and%2520Selective%2520TAK1%2520Inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2021%26volume%3D12%26spage%3D555%26epage%3D562%26doi%3D10.1021%2Facsmedchemlett.0c00547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mimasu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamagishi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiyohara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matusda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yahata, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hupp, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kakefuda, K.</span></span> <span> </span><span class="NLM_article-title">Full-length in meso structure and mechanism of rat kynurenine 3-monooxygenase inhibition</span>. <i>Commun. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">159</span> <span class="refDoi"> DOI: 10.1038/s42003-021-01666-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1038%2Fs42003-021-01666-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=33542467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BB3MXjsVCltbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2021&author=S.+Mimasuauthor=H.+Yamagishiauthor=S.+Kuboauthor=M.+Kiyoharaauthor=T.+Matusdaauthor=T.+Yahataauthor=H.+A.+Thomsonauthor=C.+H.+Huppauthor=J.+Liuauthor=T.+Okudaauthor=K.+Kakefuda&title=Full-length+in+meso+structure+and+mechanism+of+rat+kynurenine+3-monooxygenase+inhibition&doi=10.1038%2Fs42003-021-01666-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Full-length in meso structure and mechanism of rat kynurenine 3-monooxygenase inhibition</span></div><div class="casAuthors">Mimasu, Shinya; Yamagishi, Hiroaki; Kubo, Satoshi; Kiyohara, Mie; Matsuda, Toshihiro; Yahata, Toshiko; Thomson, Heather A.; Hupp, Christopher D.; Liu, Julie; Okuda, Takao; Kakefuda, Kenichi</div><div class="citationInfo"><span class="NLM_cas:title">Communications Biology</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">159</span>CODEN:
                <span class="NLM_cas:coden">CBOIDQ</span>;
        ISSN:<span class="NLM_cas:issn">2399-3642</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: The structural mechanisms of single-pass transmembrane enzymes remain elusive.  Kynurenine 3-monooxygenase (KMO) is a mitochondrial protein involved in the eukaryotic tryptophan catabolic pathway and is linked to various diseases.  Here, we report the mammalian full-length structure of KMO in its membrane-embedded form, complexed with compd. 3 (identified internally) and compd. 4 (identified via DNA-encoded chem. library screening) at 3.0 Å resoln.  Despite predictions suggesting that KMO has two transmembrane domains, we show that KMO is actually a single-pass transmembrane protein, with the other transmembrane domain lying laterally along the membrane, where it forms part of the ligand-binding pocket.  Further exploration of compd. 3 led to identification of the brain-penetrant compd., 5.  We show that KMO is dimeric, and that mutations at the dimeric interface abolish its activity.  These results will provide insight for the drug discovery of addnl. blood-brain-barrier mols., and help illuminate the complex biol. behind single-pass transmembrane enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlMxD8lAIIpLVg90H21EOLACvtfcHk0lhI02liLHlUZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXjsVCltbs%253D&md5=07139f96f3bc4f8688fa0078e99166d3</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fs42003-021-01666-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42003-021-01666-5%26sid%3Dliteratum%253Aachs%26aulast%3DMimasu%26aufirst%3DS.%26aulast%3DYamagishi%26aufirst%3DH.%26aulast%3DKubo%26aufirst%3DS.%26aulast%3DKiyohara%26aufirst%3DM.%26aulast%3DMatusda%26aufirst%3DT.%26aulast%3DYahata%26aufirst%3DT.%26aulast%3DThomson%26aufirst%3DH.%2BA.%26aulast%3DHupp%26aufirst%3DC.%2BH.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DOkuda%26aufirst%3DT.%26aulast%3DKakefuda%26aufirst%3DK.%26atitle%3DFull-length%2520in%2520meso%2520structure%2520and%2520mechanism%2520of%2520rat%2520kynurenine%25203-monooxygenase%2520inhibition%26jtitle%3DCommun.%2520Biol.%26date%3D2021%26volume%3D4%26doi%3D10.1038%2Fs42003-021-01666-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundstrom, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geschwindner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poon, E. K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohman, R.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordqvist, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friden-Saxin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dekker, N.</span></span> <span> </span><span class="NLM_article-title">Protease-activated receptor-2 ligands reveal orthosteric and allosteric mechanisms of receptor inhibition</span>. <i>Commun. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>3</i></span>, <span class="NLM_elocation-id">782</span> <span class="refDoi"> DOI: 10.1038/s42003-020-01504-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1038%2Fs42003-020-01504-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=33335291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis12it7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2020&author=A.+J.+Kennedyauthor=L.+Sundstromauthor=S.+Geschwindnerauthor=E.+K.+Y.+Poonauthor=Y.+Jiangauthor=R.+Chenauthor=R.+Cookeauthor=S.+Johnstoneauthor=A.+Madinauthor=J.+Limauthor=Q.+Liuauthor=R.-J.+Lohmanauthor=A.+Nordqvistauthor=M.+Friden-Saxinauthor=W.+Yangauthor=D.+G.+Brownauthor=D.+P.+Fairlieauthor=N.+Dekker&title=Protease-activated+receptor-2+ligands+reveal+orthosteric+and+allosteric+mechanisms+of+receptor+inhibition&doi=10.1038%2Fs42003-020-01504-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Protease-activated receptor-2 ligands reveal orthosteric and allosteric mechanisms of receptor inhibition</span></div><div class="casAuthors">Kennedy, Amanda J.; Sundstroem, Linda; Geschwindner, Stefan; Poon, Eunice K. Y.; Jiang, Yuhong; Chen, Rongfeng; Cooke, Rob; Johnstone, Shawn; Madin, Andrew; Lim, Junxian; Liu, Qingqi; Lohman, Rink-Jan; Nordqvist, Anneli; Friden-Saxin, Maria; Yang, Wenzhen; Brown, Dean G.; Fairlie, David P.; Dekker, Niek</div><div class="citationInfo"><span class="NLM_cas:title">Communications Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">782</span>CODEN:
                <span class="NLM_cas:coden">CBOIDQ</span>;
        ISSN:<span class="NLM_cas:issn">2399-3642</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Protease-activated receptor-2 (PAR2) has been implicated in multiple pathophysiologies but drug discovery is challenging due to low small mol. tractability and a complex activation mechanism.  Here we report the pharmacol. profiling of a potent new agonist, suggested by mol. modeling to bind in the putative orthosteric site, and two novel PAR2 antagonists with distinctly different mechanisms of inhibition.  We identify coupling between different PAR2 binding sites.  One antagonist is a competitive inhibitor that binds to the orthosteric site, while a second antagonist is a neg. allosteric modulator that binds at a remote site.  The allosteric modulator shows probe dependence, more effectively inhibiting peptide than protease activation of PAR2 signalling.  Importantly, both antagonists are active in vivo, inhibiting PAR2 agonist-induced acute paw inflammation in rats and preventing activation of mast cells and neutrophils.  These results highlight two distinct mechanisms of inhibition that potentially could be targeted for future development of drugs that modulate PAR2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAX4X0epY4cbVg90H21EOLACvtfcHk0lhI02liLHlUZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis12it7zK&md5=e4401ba74eac9c73c2cbe157f0270f4a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fs42003-020-01504-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42003-020-01504-0%26sid%3Dliteratum%253Aachs%26aulast%3DKennedy%26aufirst%3DA.%2BJ.%26aulast%3DSundstrom%26aufirst%3DL.%26aulast%3DGeschwindner%26aufirst%3DS.%26aulast%3DPoon%26aufirst%3DE.%2BK.%2BY.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DR.%26aulast%3DCooke%26aufirst%3DR.%26aulast%3DJohnstone%26aufirst%3DS.%26aulast%3DMadin%26aufirst%3DA.%26aulast%3DLim%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DLohman%26aufirst%3DR.-J.%26aulast%3DNordqvist%26aufirst%3DA.%26aulast%3DFriden-Saxin%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26aulast%3DDekker%26aufirst%3DN.%26atitle%3DProtease-activated%2520receptor-2%2520ligands%2520reveal%2520orthosteric%2520and%2520allosteric%2520mechanisms%2520of%2520receptor%2520inhibition%26jtitle%3DCommun.%2520Biol.%26date%3D2020%26volume%3D3%26doi%3D10.1038%2Fs42003-020-01504-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Belyanskaya, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazaar, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israel, D. I.</span></span> <span> </span><span class="NLM_article-title">Discovering drugs with DNA-encoded library technology: From concept to clinic with an inhibitor of soluble epoxide hydrolase</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">837</span>– <span class="NLM_lpage">842</span>, <span class="refDoi"> DOI: 10.1002/cbic.201700014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1002%2Fcbic.201700014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=28281333" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVWjurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=837-842&author=S.+L.+Belyanskayaauthor=Y.+Dingauthor=J.+F.+Callahanauthor=A.+L.+Lazaarauthor=D.+I.+Israel&title=Discovering+drugs+with+DNA-encoded+library+technology%3A+From+concept+to+clinic+with+an+inhibitor+of+soluble+epoxide+hydrolase&doi=10.1002%2Fcbic.201700014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Discovering Drugs with DNA-Encoded Library Technology: From Concept to Clinic with an Inhibitor of Soluble Epoxide Hydrolase</span></div><div class="casAuthors">Belyanskaya, Svetlana L.; Ding, Yun; Callahan, James F.; Lazaar, Aili L.; Israel, David I.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">837-842</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">DNA-encoded chem. library technol. was developed with the vision of its becoming a transformational platform for drug discovery.  The hope was that a new paradigm for the discovery of low-mol.-wt. drugs would be enabled by combining the vast mol. diversity achievable with combinatorial chem., the information-encoding attributes of DNA, the power of mol. biol., and a streamlined selection-based discovery process.  Here, we describe the discovery and early clin. development of GSK2256294, an inhibitor of sol. epoxide hydrolase (sEH, EPHX2), by using encoded-library technol. (ELT).  GSK2256294 is an orally bioavailable, potent and selective inhibitor of sEH that has a long half life and produced no serious adverse events in a first-time-in-human clin. study.  To our knowledge, GSK2256294 is the first mol. discovered from this technol. to enter human clin. testing and represents a realization of the vision that DNA-encoded chem. library technol. can efficiently yield mols. with favorable properties that can be readily progressed into high-quality drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7WTPZp2SE0rVg90H21EOLACvtfcHk0lgHY73-d0o3WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVWjurk%253D&md5=a34e28c8817864356f6ff689e9236940</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201700014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201700014%26sid%3Dliteratum%253Aachs%26aulast%3DBelyanskaya%26aufirst%3DS.%2BL.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DLazaar%26aufirst%3DA.%2BL.%26aulast%3DIsrael%26aufirst%3DD.%2BI.%26atitle%3DDiscovering%2520drugs%2520with%2520DNA-encoded%2520library%2520technology%253A%2520From%2520concept%2520to%2520clinic%2520with%2520an%2520inhibitor%2520of%2520soluble%2520epoxide%2520hydrolase%26jtitle%3DChemBioChem%26date%3D2017%26volume%3D18%26spage%3D837%26epage%3D842%26doi%3D10.1002%2Fcbic.201700014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arico-Muendel, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belyanskaya, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benjamin, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creaser, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davie, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franzen, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gefter, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, N. J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israel, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavarana, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollmann, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lind, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mataruse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medeiros, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Keefe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliff, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rise, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satz, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svendsen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Vloten, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, B. A.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and selection of DNA-encoded small-molecule libraries</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">647</span>– <span class="NLM_lpage">654</span>, <span class="refDoi"> DOI: 10.1038/nchembio.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1038%2Fnchembio.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=19648931" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1Ogtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=647-654&author=M.+A.+Clarkauthor=R.+A.+Acharyaauthor=C.+C.+Arico-Muendelauthor=S.+L.+Belyanskayaauthor=D.+R.+Benjaminauthor=N.+R.+Carlsonauthor=P.+A.+Centrellaauthor=C.+H.+Chiuauthor=S.+P.+Creaserauthor=J.+W.+Cuozzoauthor=C.+P.+Davieauthor=Y.+Dingauthor=G.+J.+Franklinauthor=K.+D.+Franzenauthor=M.+L.+Gefterauthor=S.+P.+Haleauthor=N.+J.+V.+Hansenauthor=D.+I.+Israelauthor=J.+Jiangauthor=M.+J.+Kavaranaauthor=M.+S.+Kellyauthor=C.+S.+Kollmannauthor=F.+Liauthor=K.+Lindauthor=S.+Mataruseauthor=P.+F.+Medeirosauthor=J.+A.+Messerauthor=P.+Myersauthor=H.+O%E2%80%99Keefeauthor=M.+C.+Oliffauthor=C.+E.+Riseauthor=A.+L.+Satzauthor=S.+R.+Skinnerauthor=J.+L.+Svendsenauthor=L.+Tangauthor=K.+van+Vlotenauthor=R.+W.+Wagnerauthor=G.+Yaoauthor=B.+Zhaoauthor=B.+A.+Morgan&title=Design%2C+synthesis+and+selection+of+DNA-encoded+small-molecule+libraries&doi=10.1038%2Fnchembio.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and selection of DNA-encoded small-molecule libraries</span></div><div class="casAuthors">Clark, Matthew A.; Acharya, Raksha A.; Arico-Muendel, Christopher C.; Belyanskaya, Svetlana L.; Benjamin, Dennis R.; Carlson, Neil R.; Centrella, Paolo A.; Chiu, Cynthia H.; Creaser, Steffen P.; Cuozzo, John W.; Davie, Christopher P.; Ding, Yun; Franklin, G. Joseph; Franzen, Kurt D.; Gefter, Malcolm L.; Hale, Steven P.; Hansen, Nils J. V.; Israel, David I.; Jiang, Jinwei; Kavarana, Malcolm J.; Kelley, Michael S.; Kollmann, Christopher S.; Li, Fan; Lind, Kenneth; Mataruse, Sibongile; Medeiros, Patricia F.; Messer, Jeffrey A.; Myers, Paul; O'Keefe, Heather; Oliff, Matthew C.; Rise, Cecil E.; Satz, Alexander L.; Skinner, Steven R.; Svendsen, Jennifer L.; Tang, Lujia; van Vloten, Kurt; Wagner, Richard W.; Yao, Gang; Zhao, Baoguang; Morgan, Barry A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">647-654</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Biochem. combinatorial techniques such as phage display, RNA display and oligonucleotide aptamers have proven to be reliable methods for generation of ligands to protein targets.  Adapting these techniques to small synthetic mols. has been a long-sought goal.  We report the synthesis and interrogation of an 800-million-member DNA-encoded library in which small mols. are covalently attached to an encoding oligonucleotide.  The library was assembled by a combination of chem. and enzymic synthesis, and interrogated by affinity selection.  We describe methods for the selection and deconvolution of the chem. display library, and the discovery of inhibitors for two enzymes: Aurora A kinase and p38 MAP kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp32j3MHOgNurVg90H21EOLACvtfcHk0lgHY73-d0o3WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1Ogtbo%253D&md5=8a40148f5fca24ecf912ed360802e19d</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.211%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DAcharya%26aufirst%3DR.%2BA.%26aulast%3DArico-Muendel%26aufirst%3DC.%2BC.%26aulast%3DBelyanskaya%26aufirst%3DS.%2BL.%26aulast%3DBenjamin%26aufirst%3DD.%2BR.%26aulast%3DCarlson%26aufirst%3DN.%2BR.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DChiu%26aufirst%3DC.%2BH.%26aulast%3DCreaser%26aufirst%3DS.%2BP.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DDavie%26aufirst%3DC.%2BP.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DFranklin%26aufirst%3DG.%2BJ.%26aulast%3DFranzen%26aufirst%3DK.%2BD.%26aulast%3DGefter%26aufirst%3DM.%2BL.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DHansen%26aufirst%3DN.%2BJ.%2BV.%26aulast%3DIsrael%26aufirst%3DD.%2BI.%26aulast%3DJiang%26aufirst%3DJ.%26aulast%3DKavarana%26aufirst%3DM.%2BJ.%26aulast%3DKelly%26aufirst%3DM.%2BS.%26aulast%3DKollmann%26aufirst%3DC.%2BS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLind%26aufirst%3DK.%26aulast%3DMataruse%26aufirst%3DS.%26aulast%3DMedeiros%26aufirst%3DP.%2BF.%26aulast%3DMesser%26aufirst%3DJ.%2BA.%26aulast%3DMyers%26aufirst%3DP.%26aulast%3DO%25E2%2580%2599Keefe%26aufirst%3DH.%26aulast%3DOliff%26aufirst%3DM.%2BC.%26aulast%3DRise%26aufirst%3DC.%2BE.%26aulast%3DSatz%26aufirst%3DA.%2BL.%26aulast%3DSkinner%26aufirst%3DS.%2BR.%26aulast%3DSvendsen%26aufirst%3DJ.%2BL.%26aulast%3DTang%26aufirst%3DL.%26aulast%3Dvan%2BVloten%26aufirst%3DK.%26aulast%3DWagner%26aufirst%3DR.%2BW.%26aulast%3DYao%26aufirst%3DG.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DMorgan%26aufirst%3DB.%2BA.%26atitle%3DDesign%252C%2520synthesis%2520and%2520selection%2520of%2520DNA-encoded%2520small-molecule%2520libraries%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D647%26epage%3D654%26doi%3D10.1038%2Fnchembio.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span> <span> </span><span class="NLM_article-title">Selecting chemicals: the emerging utility of DNA-encoded libraries</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">396</span>– <span class="NLM_lpage">403</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2010.02.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1016%2Fj.cbpa.2010.02.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=20346729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=396-403&author=M.+A.+Clark&title=Selecting+chemicals%3A+the+emerging+utility+of+DNA-encoded+libraries&doi=10.1016%2Fj.cbpa.2010.02.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Selecting chemicals: the emerging utility of DNA-encoded libraries</span></div><div class="casAuthors">Clark, Matthew A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">396-403</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Over the past 10 years, a handful of academic and industrial research groups have developed strategies for the synthesis and interrogation of DNA-encoded small-mol. libraries.  These strategies can be divided into those in which DNA directs small-mol. synthesis and those in which it records the synthesis.  These libraries have started to yield novel modulators of biol. targets, including: SH3-domain-binding peptoids, macrocyclic peptide-based Bcl-XL/BH3 interaction disruptors, ligands for TNF, albumin, streptavidin and others, and small-mol. kinase inhibitors.  The DNA-encoded library field holds the potential to address the general problem of biol. ligand discovery, including pharmaceutical lead generation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBheFYsHS5irVg90H21EOLACvtfcHk0livVe0G2cJOuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsb4%253D&md5=163a93c8ca0f6f4b96352153b466fdc2</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2010.02.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2010.02.017%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DM.%2BA.%26atitle%3DSelecting%2520chemicals%253A%2520the%2520emerging%2520utility%2520of%2520DNA-encoded%2520libraries%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D14%26spage%3D396%26epage%3D403%26doi%3D10.1016%2Fj.cbpa.2010.02.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goodnow, R. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span> <span> </span><span class="NLM_article-title">DNA-encoded chemistry: enabling the deeper sampling of chemical space</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">147</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.213</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1038%2Fnrd.2016.213" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=27932801" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVKkurjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=131-147&author=R.+A.+Goodnowauthor=C.+E.+Dumelinauthor=A.+D.+Keefe&title=DNA-encoded+chemistry%3A+enabling+the+deeper+sampling+of+chemical+space&doi=10.1038%2Fnrd.2016.213"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-encoded chemistry: enabling the deeper sampling of chemical space</span></div><div class="casAuthors">Goodnow, Robert A. Jr; Dumelin, Christoph E.; Keefe, Anthony D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">131-147</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  DNA-encoded chem. library technologies are increasingly being adopted in drug discovery for hit and lead generation.  DNA-encoded chem. enables the exploration of chem. spaces four to five orders of magnitude more deeply than is achievable by traditional high-throughput screening methods.  Operation of this technol. requires developing a range of capabilities including aq. synthetic chem., building block acquisition, oligonucleotide conjugation, large-scale mol. biol. transformations, selection methodologies, PCR, sequencing, sequence data anal. and the anal. of large chem. spaces.  This Review provides an overview of the development and applications of DNA-encoded chem., highlighting the challenges and future directions for the use of this technol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdEd70MeexerVg90H21EOLACvtfcHk0livVe0G2cJOuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVKkurjM&md5=7ca97b01d75e600378799baf77665c07</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.213%26sid%3Dliteratum%253Aachs%26aulast%3DGoodnow%26aufirst%3DR.%2BA.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26atitle%3DDNA-encoded%2520chemistry%253A%2520enabling%2520the%2520deeper%2520sampling%2520of%2520chemical%2520space%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D131%26epage%3D147%26doi%3D10.1038%2Fnrd.2016.213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Franzini, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheuermann, J.</span></span> <span> </span><span class="NLM_article-title">DNA-encoded chemical libraries: Advancing beyond conventional small-molecule libraries</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1247</span>– <span class="NLM_lpage">1255</span>, <span class="refDoi"> DOI: 10.1021/ar400284t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar400284t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltF2ls7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2014&pages=1247-1255&author=R.+M.+Franziniauthor=D.+Neriauthor=J.+Scheuermann&title=DNA-encoded+chemical+libraries%3A+Advancing+beyond+conventional+small-molecule+libraries&doi=10.1021%2Far400284t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-Encoded Chemical Libraries: Advancing beyond Conventional Small-Molecule Libraries</span></div><div class="casAuthors">Franzini, Raphael M.; Neri, Dario; Scheuermann, Jorg</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1247-1255</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  DNA-encoded chem. libraries (DECLs) represent a promising tool in drug discovery.  DECL technol. allows the synthesis and screening of chem. libraries of unprecedented size at moderate costs.  In analogy to phage-display technol., where large antibody libraries are displayed on the surface of filamentous phage and are genetically encoded in the phage genome, DECLs feature the display of individual small org. chem. moieties on DNA fragments serving as amplifiable identification barcodes.  The DNA-tag facilitates the synthesis and allows the simultaneous screening of very large sets of compds. (up to billions of mols.), because the hit compds. can easily be identified and quantified by PCR-amplification of the DNA-barcode followed by high-throughput DNA sequencing.  Several approaches have been used to generate DECLs, differing both in the methods used for library encoding and for the combinatorial assembly of chem. moieties.  For example, DECLs can be used for fragment-based drug discovery, displaying a single mol. on DNA or two chem. moieties at the extremities of complementary DNA strands.  DECLs can vary substantially in the chem. structures and the library size.  While ultralarge libraries contg. billions of compds. have been reported contg. four or more sets of building blocks, also smaller libraries have been shown to be efficient for ligand discovery.  In general, it has been found that the overall library size is a poor predictor for library performance and that the no. and diversity of the building blocks are rather important indicators.  Smaller libraries consisting of two to three sets of building blocks better fulfill the criteria of drug-likeness and often have higher quality.  In this Account, we present advances in the DECL field from proof-of-principle studies to practical applications for drug discovery, both in industry and in academia.  DECL technol. can yield specific binders to a variety of target proteins and is likely to become a std. tool for pharmaceutical hit discovery, lead expansion, and Chem. Biol. research.  The introduction of new methodologies for library encoding and for compd. synthesis in the presence of DNA is an exciting research field and will crucially contribute to the performance and the propagation of the technol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcaCmssNgPLLVg90H21EOLACvtfcHk0livVe0G2cJOuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltF2ls7g%253D&md5=c37bd1625b668e869fcc26d5ef0a017a</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Far400284t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far400284t%26sid%3Dliteratum%253Aachs%26aulast%3DFranzini%26aufirst%3DR.%2BM.%26aulast%3DNeri%26aufirst%3DD.%26aulast%3DScheuermann%26aufirst%3DJ.%26atitle%3DDNA-encoded%2520chemical%2520libraries%253A%2520Advancing%2520beyond%2520conventional%2520small-molecule%2520libraries%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2014%26volume%3D47%26spage%3D1247%26epage%3D1255%26doi%3D10.1021%2Far400284t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, G. T.</span></span> <span> </span><span class="NLM_article-title">DNA-Encoded library screening as core platform technology in drug discovery: Its synthetic method development and applications in DEL synthesis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">6578</span>– <span class="NLM_lpage">6599</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01782</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01782" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis12nsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=6578-6599&author=M.+Songauthor=G.+T.+Hwang&title=DNA-Encoded+library+screening+as+core+platform+technology+in+drug+discovery%3A+Its+synthetic+method+development+and+applications+in+DEL+synthesis&doi=10.1021%2Facs.jmedchem.9b01782"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-Encoded Library Screening as Core Platform Technology in Drug Discovery: Its Synthetic Method Development and Applications in DEL Synthesis</span></div><div class="casAuthors">Song, Minsoo; Hwang, Gil Tae</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6578-6599</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  DNA-encoded library technol. (DELT) was introduced to our medicinal chem. society more than 20 years ago.  The application of DELT in the development of clin. candidates has been actively reported in the literature recently.  A few representative examples include RIP1K inhibitors for inflammatory diseases and sEH inhibitors for endothelial dysfunction or abnormal tissue repair, among many others.  Here, the authors would like to recall the recent developments in on-DNA synthetic methodologies for DEL construction and to analyze recent examples in the literature of DELT-based drug development efforts pursued in both the academic and industrial sectors.  With this perspective, we hope to provide a useful summary of recent DELT-based drug discovery research and to discuss the future scope of DELT in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOdR3yKaXgmrVg90H21EOLACvtfcHk0ljgcTa2rgZoqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis12nsb8%253D&md5=c462a02125c354e82883013f0be12e60</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01782&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01782%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DM.%26aulast%3DHwang%26aufirst%3DG.%2BT.%26atitle%3DDNA-Encoded%2520library%2520screening%2520as%2520core%2520platform%2520technology%2520in%2520drug%2520discovery%253A%2520Its%2520synthetic%2520method%2520development%2520and%2520applications%2520in%2520DEL%2520synthesis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D6578%26epage%3D6599%26doi%3D10.1021%2Facs.jmedchem.9b01782" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fargue, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southall, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKew, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danpure, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span> <span> </span><span class="NLM_article-title">High throughput cell-based assay for identification of glycolate oxidase inhibitors as a potential treatment for primary hyperoxaluria type 1</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">34060</span> <span class="refDoi"> DOI: 10.1038/srep34060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1038%2Fsrep34060" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=27670739" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsF2qsL7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&author=M.+Wangauthor=M.+Xuauthor=Y.+Longauthor=S.+Fargueauthor=N.+Southallauthor=X.+Huauthor=J.+C.+McKewauthor=C.+J.+Danpureauthor=W.+Zheng&title=High+throughput+cell-based+assay+for+identification+of+glycolate+oxidase+inhibitors+as+a+potential+treatment+for+primary+hyperoxaluria+type+1&doi=10.1038%2Fsrep34060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">High throughput cell-based assay for identification of glycolate oxidase inhibitors as a potential treatment for Primary Hyperoxaluria Type 1</span></div><div class="casAuthors">Wang, Mengqiao; Xu, Miao; Long, Yan; Fargue, Sonia; Southall, Noel; Hu, Xin; McKew, John C.; Danpure, Christopher J.; Zheng, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">34060</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Glycolate oxidase (GO) and alanine:glyoxylate aminotransferase (AGT) are both involved in the peroxisomal glyoxylate pathway.  Deficiency in AGT function causes the accumulation of intracellular oxalate and the primary hyperoxaluria type 1 (PH1).  AGT enhancers or GO inhibitors may restore the abnormal peroxisomal glyoxylate pathway in PH1 patients.  With stably transformed cells which mimic the glyoxylate metabolic pathway, we developed an indirect glycolate cytotoxicity assay in a 1,536-well plate format for high throughput screening.  This assay can be used to identify compds. that reduce indirect glycolate-induced cytotoxicity by either enhancing AGT activity or inhibiting GO.  A pilot screen of 4,096 known compds. identified two membrane permeable GO inhibitors: dichromate salt and colistimethate.  We also developed a GO enzyme assay using the hydrogen peroxide-Amplex red reporter system.  The IC50 values of potassium dichromate, sodium dichromate, and colistimethate sodium were 0.096, 0.108, and 2.3 μM in the GO enzyme assay, resp.  Further enzyme kinetic study revealed that both types of compds. inhibit GO activity by the mixed linear inhibition.  Our results demonstrate that the cell-based assay and GO enzyme assay developed in this study are useful for further screening of large compd. libraries for drug development to treat PH1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7wOw_TIVi3LVg90H21EOLACvtfcHk0ljgcTa2rgZoqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsF2qsL7P&md5=487ef1a45249a9251349c6898c8e9588</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fsrep34060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep34060%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DLong%26aufirst%3DY.%26aulast%3DFargue%26aufirst%3DS.%26aulast%3DSouthall%26aufirst%3DN.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DMcKew%26aufirst%3DJ.%2BC.%26aulast%3DDanpure%26aufirst%3DC.%2BJ.%26aulast%3DZheng%26aufirst%3DW.%26atitle%3DHigh%2520throughput%2520cell-based%2520assay%2520for%2520identification%2520of%2520glycolate%2520oxidase%2520inhibitors%2520as%2520a%2520potential%2520treatment%2520for%2520primary%2520hyperoxaluria%2520type%25201%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26doi%3D10.1038%2Fsrep34060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cymborowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chruszcz, M.</span></span> <span> </span><span class="NLM_article-title">HKL-3000: the integration of data reduction and structure solution - from diffraction images to an initial model in minutes</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">859</span>– <span class="NLM_lpage">866</span>, <span class="refDoi"> DOI: 10.1107/S0907444906019949</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1107%2FS0907444906019949" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=16855301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=859-866&author=W.+Minorauthor=M.+Cymborowskiauthor=Z.+Otwinowskiauthor=M.+Chruszcz&title=HKL-3000%3A+the+integration+of+data+reduction+and+structure+solution+-+from+diffraction+images+to+an+initial+model+in+minutes&doi=10.1107%2FS0907444906019949"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">HKL-3000: the integration of data reduction and structure solution - from diffraction images to an initial model in minutes</span></div><div class="casAuthors">Minor, Wladek; Cymborowski, Marcin; Otwinowski, Zbyszek; Chruszcz, Maksymilian</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">859-866</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A new approach that integrates data collection, data redn., phasing and model building significantly accelerates the process of structure detn. and on av. minimizes the no. of data sets and synchrotron time required for structure soln.  Initial testing of the HKL-3000 system (the beta version was named HKL-2000_ph) with more than 140 novel structure detns. has proven its high value for MAD/SAD expts.  The heuristics for choosing the best computational strategy at different data resoln. limits of phasing signal and crystal diffraction are being optimized.  The typical end result is an interpretable electron-d. map with a partially built structure and, in some cases, an almost complete refined model.  The current development is oriented towards very fast structure soln. in order to provide feedback during the diffraction expt.  Work is also proceeding towards improving the quality of phasing calcn. and model building.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0c0DYtjFP7rVg90H21EOLACvtfcHk0ljgcTa2rgZoqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D&md5=2cd2fe288f954837abb4acae4c008a2f</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS0907444906019949&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906019949%26sid%3Dliteratum%253Aachs%26aulast%3DMinor%26aufirst%3DW.%26aulast%3DCymborowski%26aufirst%3DM.%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DChruszcz%26aufirst%3DM.%26atitle%3DHKL-3000%253A%2520the%2520integration%2520of%2520data%2520reduction%2520and%2520structure%2520solution%2520-%2520from%2520diffraction%2520images%2520to%2520an%2520initial%2520model%2520in%2520minutes%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D859%26epage%3D866%26doi%3D10.1107%2FS0907444906019949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lhEONtvvLmFLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhEONtvvLmFLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lhEONtvvLmFLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span> <i>Schrodinger Release
2019-3</i>; <span class="NLM_publisher-name">Schrodinger,
LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Schrodinger+Release%0A2019-3%3B+Schrodinger%2C%0ALLC%3A+New+York%2C+NY%2C+2019."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSchrodinger%2520Release%250A2019-3%26pub%3DSchrodinger%252C%250ALLC%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T.</span></span> <span> </span><span class="NLM_article-title">Identifying and characterizing binding sites and assessing druggability</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1021/ci800324m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800324m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=377-389&author=T.+Halgren&title=Identifying+and+characterizing+binding+sites+and+assessing+druggability&doi=10.1021%2Fci800324m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying and Characterizing Binding Sites and Assessing Druggability</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">377-389</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Identification and characterization of binding sites is key in the process of structure-based drug design.  In some cases there may not be any information about the binding site for a target of interest.  In other cases, a putative binding site has been identified by computational or exptl. means, but the druggability of the target is not known.  Even when a site for a given target is known, it may be desirable to find addnl. sites whose targeting could produce a desired biol. response.  A new program, called SiteMap, is presented for identifying and analyzing binding sites and for predicting target druggability.  In a large-scale validation, SiteMap correctly identifies the known binding site as the top-ranked site in 86% of the cases, with best results (>98%) coming for sites that bind ligands with subnanomolar affinity.  In addn., a modified version of the score employed for binding-site identification allows SiteMap to accurately classify the druggability of proteins as measured by their ability to bind passively absorbed small mols. tightly.  In characterizing binding sites, SiteMap provides quant. and graphical information that can help guide efforts to critically assess virtual hits in a lead-discovery application or to modify ligand structure to enhance potency or improve phys. properties in a lead-optimization context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfAt0-gvcXDLVg90H21EOLACvtfcHk0lhEONtvvLmFLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D&md5=f03c7b27e30ae03d10e4127ed1f1ed3c</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fci800324m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800324m%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%26atitle%3DIdentifying%2520and%2520characterizing%2520binding%2520sites%2520and%2520assessing%2520druggability%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D377%26epage%3D389%26doi%3D10.1021%2Fci800324m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T.</span></span> <span> </span><span class="NLM_article-title">New method for fast and accurate binding-site identification and analysis</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">146</span>– <span class="NLM_lpage">148</span>, <span class="refDoi"> DOI: 10.1111/j.1747-0285.2007.00483.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1111%2Fj.1747-0285.2007.00483.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=17381729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkt1OnsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2007&pages=146-148&author=T.+Halgren&title=New+method+for+fast+and+accurate+binding-site+identification+and+analysis&doi=10.1111%2Fj.1747-0285.2007.00483.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">New method for fast and accurate binding-site identification and analysis</span></div><div class="casAuthors">Halgren, Tom</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">146-148</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">Structure-based drug design seeks to exploit the structure of protein-ligand or protein-protein binding sites, but the site is not always known at the outset.  Even when the site is known, the researcher may wish to identify alternative prospective binding sites that may result in different biol. effects or new class of compds.  It is also vital in lead optimization to clearly understand the degree to which known binders or docking hits satisfy or violate complementarity to the receptor.  SiteMap is a new technique for identifying potential binding sites and for predicting their druggability in lead-discovery applications and for characterizing binding sites and critically assessing prospective ligands in lead-optimization applications.  In large-scale validation tests, SiteMap correctly identifies the known binding site in > 96% of the cases, with best results (> 98%) coming for sites that bind ligands tightly.  It also accurately distinguishes between sites that bind ligands and sites that don't.  In binding-site anal., SiteMap provides a wealth of quant. and graphical information that can help guide efforts to modify ligand structure to enhance potency or improve phys. properties.  These attributes allow SiteMap to nicely complement techniques such as docking and computational lead optimization in structure-base drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr245hmZWXilrVg90H21EOLACvtfcHk0ljupekYXFjr3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkt1OnsLg%253D&md5=c233276ba9580393b282f255781f9ce6</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2007.00483.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2007.00483.x%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%26atitle%3DNew%2520method%2520for%2520fast%2520and%2520accurate%2520binding-site%2520identification%2520and%2520analysis%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2007%26volume%3D69%26spage%3D146%26epage%3D148%26doi%3D10.1111%2Fj.1747-0285.2007.00483.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sastry, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adzhigirey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annabhimoju, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Protein and ligand preparation: Parameters, protocols, and influence on virtual screening enrichments</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">221</span>– <span class="NLM_lpage">234</span>, <span class="refDoi"> DOI: 10.1007/s10822-013-9644-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1007%2Fs10822-013-9644-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=23579614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A280%3ADC%252BC3srksFekug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=221-234&author=G.+M.+Sastryauthor=M.+Adzhigireyauthor=T.+Dayauthor=R.+Annabhimojuauthor=W.+Sherman&title=Protein+and+ligand+preparation%3A+Parameters%2C+protocols%2C+and+influence+on+virtual+screening+enrichments&doi=10.1007%2Fs10822-013-9644-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span></div><div class="casAuthors">Sastry G Madhavi; Adzhigirey Matvey; Day Tyler; Annabhimoju Ramakrishna; Sherman Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of computer-aided molecular design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">221-34</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Structure-based virtual screening plays an important role in drug discovery and complements other screening approaches.  In general, protein crystal structures are prepared prior to docking in order to add hydrogen atoms, optimize hydrogen bonds, remove atomic clashes, and perform other operations that are not part of the x-ray crystal structure refinement process.  In addition, ligands must be prepared to create 3-dimensional geometries, assign proper bond orders, and generate accessible tautomer and ionization states prior to virtual screening.  While the prerequisite for proper system preparation is generally accepted in the field, an extensive study of the preparation steps and their effect on virtual screening enrichments has not been performed.  In this work, we systematically explore each of the steps involved in preparing a system for virtual screening.  We first explore a large number of parameters using the Glide validation set of 36 crystal structures and 1,000 decoys.  We then apply a subset of protocols to the DUD database.  We show that database enrichment is improved with proper preparation and that neglecting certain steps of the preparation process produces a systematic degradation in enrichments, which can be large for some targets.  We provide examples illustrating the structural changes introduced by the preparation that impact database enrichment.  While the work presented here was performed with the Protein Preparation Wizard and Glide, the insights and guidance are expected to be generalizable to structure-based virtual screening with other docking methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQd2Qiai-OqeXqj_yg7twvpfW6udTcc2ebuPO_s15fjerntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3srksFekug%253D%253D&md5=9eb1a820e121aca2742ed53f24481ace</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1007%2Fs10822-013-9644-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-013-9644-8%26sid%3Dliteratum%253Aachs%26aulast%3DSastry%26aufirst%3DG.%2BM.%26aulast%3DAdzhigirey%26aufirst%3DM.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DAnnabhimoju%26aufirst%3DR.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DProtein%2520and%2520ligand%2520preparation%253A%2520Parameters%252C%2520protocols%252C%2520and%2520influence%2520on%2520virtual%2520screening%2520enrichments%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26spage%3D221%26epage%3D234%26doi%3D10.1007%2Fs10822-013-9644-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanschagrin, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span> <span> </span><span class="NLM_article-title">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein-Ligand Complexes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">6177</span>– <span class="NLM_lpage">6196</span>, <span class="refDoi"> DOI: 10.1021/jm051256o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm051256o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6177-6196&author=R.+A.+Friesnerauthor=R.+B.+Murphyauthor=M.+P.+Repaskyauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=T.+A.+Halgrenauthor=P.+C.+Sanschagrinauthor=D.+T.+Mainz&title=Extra+Precision+Glide%3A+Docking+and+Scoring+Incorporating+a+Model+of+Hydrophobic+Enclosure+for+Protein-Ligand+Complexes&doi=10.1021%2Fjm051256o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein-Ligand Complexes</span></div><div class="casAuthors">Friesner, Richard A.; Murphy, Robert B.; Repasky, Matthew P.; Frye, Leah L.; Greenwood, Jeremy R.; Halgren, Thomas A.; Sanschagrin, Paul C.; Mainz, Daniel T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6177-6196</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel scoring function to est. protein-ligand binding affinities has been developed and implemented as the Glide 4.0 XP scoring function and docking protocol.  In addn. to unique water desolvation energy terms, protein-ligand structural motifs leading to enhanced binding affinity are included:(1) hydrophobic enclosure where groups of lipophilic ligand atoms are enclosed on opposite faces by lipophilic protein atoms, (2) neutral-neutral single or correlated hydrogen bonds in a hydrophobically enclosed environment, and (3) five categories of charged-charged hydrogen bonds.  The XP scoring function and docking protocol have been developed to reproduce exptl. binding affinities for a set of 198 complexes (RMSDs of 2.26 and 1.73 kcal/mol over all and well-docked ligands, resp.) and to yield quality enrichments for a set of fifteen screens of pharmaceutical importance.  Enrichment results demonstrate the importance of the novel XP mol. recognition and water scoring in sepg. active and inactive ligands and avoiding false positives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohZiwN12vaY7Vg90H21EOLACvtfcHk0liGoh8Uq0bRgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D&md5=ea428c82ead0d8c27f8c1a7b694a1edf</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26atitle%3DExtra%2520Precision%2520Glide%253A%2520Docking%2520and%2520Scoring%2520Incorporating%2520a%2520Model%2520of%2520Hydrophobic%2520Enclosure%2520for%2520Protein-Ligand%2520Complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6177%26epage%3D6196%26doi%3D10.1021%2Fjm051256o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 2. Enrichment Factors in Database Screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+2.+Enrichment+Factors+in+Database+Screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0liGoh8Uq0bRgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25202.%2520Enrichment%2520Factors%2520in%2520Database%2520Screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 1. Method and Assessment of Docking Accuracy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1739</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+1.+Method+and+Assessment+of+Docking+Accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0liGoh8Uq0bRgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25201.%2520Method%2520and%2520Assessment%2520of%2520Docking%2520Accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span> <i>Schrodinger Release
2019-3: Glide</i>; <span class="NLM_publisher-name">Schrodinger, LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Schrodinger+Release%0A2019-3%3A+Glide%3B+Schrodinger%2C+LLC%3A+New+York%2C+NY%2C+2019."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSchrodinger%2520Release%250A2019-3%253A%2520Glide%26pub%3DSchrodinger%252C%2520LLC%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span> <i>Schrodinger Release 2019-3: Ligprep</i>; <span class="NLM_publisher-name">Schrodinger, LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Schrodinger+Release+2019-3%3A+Ligprep%3B+Schrodinger%2C+LLC%3A+New+York%2C+NY%2C+2019."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSchrodinger%2520Release%25202019-3%253A%2520Ligprep%26pub%3DSchrodinger%252C%2520LLC%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pincus, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rapp, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T. J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honig, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">A Hierarchical Approach to All-Atom Protein Loop Prediction</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">351</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1002/prot.10613</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1002%2Fprot.10613" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=15048827" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtFKhsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2004&pages=351-367&author=M.+P.+Jacobsonauthor=D.+L.+Pincusauthor=C.+S.+Rappauthor=T.+J.+F.+Dayauthor=B.+Honigauthor=D.+E.+Shawauthor=R.+A.+Friesner&title=A+Hierarchical+Approach+to+All-Atom+Protein+Loop+Prediction&doi=10.1002%2Fprot.10613"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A hierarchical approach to all-atom protein loop prediction</span></div><div class="casAuthors">Jacobson, Matthew P.; Pincus, David L.; Rapp, Chaya S.; Day, Tyler J. F.; Honig, Barry; Shaw, David E.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">351-367</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The application of all-atom force fields (and explicit or implicit solvent models) to protein homol.-modeling tasks such as side-chain and loop prediction remains challenging both because of the expense of the individual energy calcns. and because of the difficulty of sampling the rugged all-atom energy surface.  Here the authors address this challenge for the problem of loop prediction through the development of numerous new algorithms, with an emphasis on multiscale and hierarchical techniques.  As a first step in evaluating the performance of the authors' loop prediction algorithm, the authors have applied it to the problem of reconstructing loops in native structures; the authors also explicitly include crystal packing to provide a fair comparison with crystal structures.  In brief, large nos. of loops are generated by using a dihedral angle-based buildup procedure followed by iterative cycles of clustering, side-chain optimization, and complete energy minimization of selected loop structures.  The authors evaluate this method by the largest test set yet used for validation of a loop prediction method, with a total of 833 loops ranging from 4 to 12 residues in length.  Av./median backbone root-mean-square deviations (RMSDs) to the native structures (superimposing the body of the protein, not the loop itself) are 0.42/0.24 Å for 5 residue loops, 1.00/0.44 Å for 8 residue loops, and 2.47/1.83 Å for 11 residue loops.  Median RMSDs are substantially lower than the avs. because of a small no. of outliers; the causes of these failures are examd. in some detail, and many can be attributed to errors in assignment of protonation states of titratable residues, omission of ligands from the simulation, and, in a few cases, probable errors in the exptl. detd. structures.  When these obvious problems in the data sets are filtered out, av. RMSDs to the native structures improve to 0.43 Å for 5 residue loops, 0.84 Å for 8 residue loops, and 1.63 Å for 11 residue loops.  In the vast majority of cases, the method locates energy min. that are lower than or equal to that of the minimized native loop, thus indicating that sampling rarely limits prediction accuracy.  The overall results are, to the authors' knowledge, the best reported to date, and the authors attribute this success to the combination of an accurate all-atom energy function, efficient methods for loop buildup and side-chain optimization, and, esp. for the longer loops, the hierarchical refinement protocol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHqYMnqaWXvLVg90H21EOLACvtfcHk0lhp4ATiULFKxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtFKhsrc%253D&md5=e0eff655eeefb30ea00ae041ea9099c8</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fprot.10613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10613%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DPincus%26aufirst%3DD.%2BL.%26aulast%3DRapp%26aufirst%3DC.%2BS.%26aulast%3DDay%26aufirst%3DT.%2BJ.%2BF.%26aulast%3DHonig%26aufirst%3DB.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DA%2520Hierarchical%2520Approach%2520to%2520All-Atom%2520Protein%2520Loop%2520Prediction%26jtitle%3DProteins%26date%3D2004%26volume%3D55%26spage%3D351%26epage%3D367%26doi%3D10.1002%2Fprot.10613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honig, B.</span></span> <span> </span><span class="NLM_article-title">On the Role of Crystal Packing Forces in Determining Protein Sidechain Conformations</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>320</i></span>,  <span class="NLM_fpage">597</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1016/S0022-2836(02)00470-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1016%2FS0022-2836%2802%2900470-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=12096912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BD38XltVKmu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=320&publication_year=2002&pages=597-608&author=M.+P.+Jacobsonauthor=R.+A.+Friesnerauthor=Z.+Xiangauthor=B.+Honig&title=On+the+Role+of+Crystal+Packing+Forces+in+Determining+Protein+Sidechain+Conformations&doi=10.1016%2FS0022-2836%2802%2900470-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">On the Role of the Crystal Environment in Determining Protein Side-chain Conformations</span></div><div class="casAuthors">Jacobson, Matthew P.; Friesner, Richard A.; Xiang, Zhexin; Honig, Barry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">320</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">597-608</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">The role of crystal packing in detg. the obsd. conformations of amino acid side-chains in protein crystals is investigated by (1) anal. of a database of proteins that have been crystd. in different unit cells (space group or unit cell dimensions) and (2) theor. predictions of side-chain conformations with the crystal environment explicitly represented.  Both of these approaches indicate that the crystal environment plays an important role in detg. the conformations of polar side-chains on the surfaces of proteins.  Inclusion of the crystal environment permits a more sensitive measurement of the achievable accuracy of side-chain prediction programs, when validating against structures obtained by x-ray crystallog.  Our side-chain prediction program uses an all-atom force field and a Generalized Born model of solvation and is thus capable of modeling simple packing effects (i.e. van der Waals interactions), electrostatic effects, and desolvation, which are all important mechanisms by which the crystal environment impacts obsd. side-chain conformations.  Our results are also relevant to the understanding of changes in side-chain conformation that may result from ligand docking and protein-protein assocn., insofar as the results reveal how side-chain conformations change in response to their local environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgHEOVpIEiErVg90H21EOLACvtfcHk0lhp4ATiULFKxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltVKmu70%253D&md5=006de6bd2d0f233ab32d6798dc1a3fbc</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2900470-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252900470-9%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DXiang%26aufirst%3DZ.%26aulast%3DHonig%26aufirst%3DB.%26atitle%3DOn%2520the%2520Role%2520of%2520Crystal%2520Packing%2520Forces%2520in%2520Determining%2520Protein%2520Sidechain%2520Conformations%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2002%26volume%3D320%26spage%3D597%26epage%3D608%26doi%3D10.1016%2FS0022-2836%2802%2900470-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span> <i>Schrodinger Release
2019-3: Prime</i>; <span class="NLM_publisher-name">Schrodinger, LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Schrodinger+Release%0A2019-3%3A+Prime%3B+Schrodinger%2C+LLC%3A+New+York%2C+NY%2C+2019."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSchrodinger%2520Release%250A2019-3%253A%2520Prime%26pub%3DSchrodinger%252C%2520LLC%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6a','cit6b'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10a','cit10b'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/medium/jm0c02271_0015.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02271&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/medium/jm0c02271_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. HAO1 inhibitors: <b>CCPST</b>,<a onclick="showRef(event, 'ref2 ref3 ref11'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref11">(2,3,11)</a><b>CDST</b>,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Compound <b>1</b>,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and Compound <b>2</b>.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02271&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/medium/jm0c02271_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Hit Compound <b>3</b> (founding member of Series 1) and Compounds <b>4–6</b> (founding members of Series 2) and the DNA-encoded chemical libraries within which they were found.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02271&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/medium/jm0c02271_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0008.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compound <b>3</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02271&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (a) 6-chloro-1<i>H</i>-indole-2-carboxylic acid, NaHCO<sub>3</sub>, and HATU; (b) 4-borono-2-chlorobenzoic acid, Pd(dppf)Cl<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O,100 °C, and 1 h.</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/medium/jm0c02271_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0009.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>4</b>–<b>6</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02271&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (a) substituted 3-formyl-2-hydroxybenzoic acid, HATU, NaHCO<sub>3</sub>, and <i>N</i>,<i>N</i>-dimethylformamide (DMF); (b) 5-amino-1<i>H</i>-indazole-3-carboxylic acid, NaBH<sub>3</sub>CN, AcOH, and MeOH; and (c) paraformaldehyde.</p></p></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/medium/jm0c02271_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cell-rescue assay of Compound <b>5</b> at various doses. (CHO-GO cells were plated in 96-well plates, preincubated with compound at different concentrations followed by glycolic acid (GA) treatment. Viability was monitored using live cells imager (IncuCyte Zoom). Each data point represents an average of three wells).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02271&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/medium/jm0c02271_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Overlay of Compound <b>3</b> PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W45">6W45</a> (green), Compound <b>4</b> PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W44">6W44</a> (pink), and Compound <b>5</b> PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W4C">6W4C</a> (gray). Inhibitor binding is dependent on the presence of the cofactor FMN. Trp110 side chain can assume different conformations to accommodate different inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02271&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/medium/jm0c02271_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0006.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Protein–ligand interactions in the Compound <b>5</b>/HAO1 complex PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W4C">6W4C</a>. The dashed gray ribbon indicates disordered loop 4.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02271&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/medium/jm0c02271_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0007.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. <b>A</b>. Compound <b>31</b>. <b>B</b>. Docking mode of Compound <b>31</b>, a benzotriazole designed as an isosteric replacement to Compound <b>5</b>, assumes a shifted orientation in the binding site. The docking pose was minimized with MM-GBSA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02271/20210520/images/large/jm0c02271_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02271&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i69">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44530" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44530" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 43 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cochat, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumsby, G.</span></span> <span> </span><span class="NLM_article-title">Primary Hyperoxaluria</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>369</i></span>,  <span class="NLM_fpage">649</span>– <span class="NLM_lpage">658</span>, <span class="refDoi"> DOI: 10.1056/NEJMra1301564</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1056%2FNEJMra1301564" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=23944302" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlClsrvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=369&publication_year=2013&pages=649-658&author=P.+Cochatauthor=G.+Rumsby&title=Primary+Hyperoxaluria&doi=10.1056%2FNEJMra1301564"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Primary hyperoxaluria</span></div><div class="casAuthors">Cochat, Pierre; Rumsby, Gill</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">369</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">649-658</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review.  The primary hyperoxaluria are a group of autosomal recessive disorders involving the overprodn. of oxalate.  Although the initial recognition of the disease is attributed to Lepoutre, who reported it in 1925,1 the elucidation of the underlying biochem. abnormalities occurred many years later.  This review discusses the major biochem., genetic, and therapeutic advances that have led to a better understanding of primary hyperoxaluria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGUV6DhnokFrVg90H21EOLACvtfcHk0li5V9aAAgGdxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlClsrvL&md5=40380b669174cdf5be2bf4af052a8118</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1056%2FNEJMra1301564&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra1301564%26sid%3Dliteratum%253Aachs%26aulast%3DCochat%26aufirst%3DP.%26aulast%3DRumsby%26aufirst%3DG.%26atitle%3DPrimary%2520Hyperoxaluria%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2013%26volume%3D369%26spage%3D649%26epage%3D658%26doi%3D10.1056%2FNEJMra1301564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Higueras, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luis-Lima, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salido, E.</span></span> <span> </span><span class="NLM_article-title">Glycolate oxidase is a safe and efficient target for substrate reduction therapy in a mouse model of primary hyperoxaluria type 1</span>. <i>Mol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">719</span>– <span class="NLM_lpage">725</span>, <span class="refDoi"> DOI: 10.1038/mt.2015.224</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1038%2Fmt.2015.224" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=26689264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1altrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=719-725&author=C.+Martin-Higuerasauthor=S.+Luis-Limaauthor=E.+Salido&title=Glycolate+oxidase+is+a+safe+and+efficient+target+for+substrate+reduction+therapy+in+a+mouse+model+of+primary+hyperoxaluria+type+1&doi=10.1038%2Fmt.2015.224"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Glycolate Oxidase Is a Safe and Efficient Target for Substrate Reduction Therapy in a Mouse Model of Primary Hyperoxaluria Type I</span></div><div class="casAuthors">Martin-Higueras, Cristina; Luis-Lima, Sergio; Salido, Eduardo</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Therapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">719-725</span>CODEN:
                <span class="NLM_cas:coden">MTOHCK</span>;
        ISSN:<span class="NLM_cas:issn">1525-0016</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Primary hyperoxaluria type 1 (PH1) is caused by deficient alanine-glyoxylate aminotransferase, the human peroxisomal enzyme that detoxifies glyoxylate.  Glycolate is one of the best-known substrates leading to glyoxylate prodn., via peroxisomal glycolate oxidase (GO).  Using genetically modified mice, we herein report GO as a safe and efficient target for substrate redn. therapy (SRT) in PH1.  We first generated a GO-deficient mouse (Hao1-/-) that presented high urine glycolate levels but no addnl. phenotype.  Next, we produced double KO mice (Agxt1-/-Hao1-/-) that showed low levels of oxalate excretion compared with hyperoxaluric mice model (Agxt1-/-).  Previous studies have identified some GO inhibitors, such as 4-carboxy-5-[(4-chlorophenyl)sulfanyl]-1,2,3-thiadiazole (CCPST).  We herein report that CCPST inhibits GO in Agxt1-/- hepatocytes and significantly reduces their oxalate prodn., starting at 25 μM.  We also tested the ability of orally administered CCPST to reduce oxalate excretion in Agxt1-/- mice, showing that 30-50% redn. in urine oxalate can be achieved.  In summary, we present proof-of-concept evidence for SRT in PH1.  These encouraging results should be followed by a medicinal chem. program that might yield more potent GO inhibitors and eventually could result in a pharmacol. treatment for this rare and severe inborn error of metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnK0-rOTRtBrVg90H21EOLACvtfcHk0lhBh6QMK97WLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1altrw%253D&md5=3a990cc2a6760420d33438382fef91d6</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fmt.2015.224&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmt.2015.224%26sid%3Dliteratum%253Aachs%26aulast%3DMartin-Higueras%26aufirst%3DC.%26aulast%3DLuis-Lima%26aufirst%3DS.%26aulast%3DSalido%26aufirst%3DE.%26atitle%3DGlycolate%2520oxidase%2520is%2520a%2520safe%2520and%2520efficient%2520target%2520for%2520substrate%2520reduction%2520therapy%2520in%2520a%2520mouse%2520model%2520of%2520primary%2520hyperoxaluria%2520type%25201%26jtitle%3DMol.%2520Ther.%26date%3D2016%26volume%3D24%26spage%3D719%26epage%3D725%26doi%3D10.1038%2Fmt.2015.224" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bourhis, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vignaud, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietrancosta, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gueritte, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guenard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lederer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindqvist, Y.</span></span> <span> </span><span class="NLM_article-title">Structure of human glycolate oxidase in complex with the inhibitor 4-carboxy-5-[(4-chlorophenyl)-sulfanyl]-1,2,3-thiadiazole</span>. <i>Acta Crystallogr., Sect. F: Struct. Biol. Cryst. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>F65</i></span>,  <span class="NLM_fpage">1246</span>– <span class="NLM_lpage">1253</span>, <span class="refDoi"> DOI: 10.1107/S1744309109041670</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1107%2FS1744309109041670" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=F65&publication_year=2009&pages=1246-1253&author=J.-M.+Bourhisauthor=C.+Vignaudauthor=N.+Pietrancostaauthor=F.+Gueritteauthor=D.+Guenardauthor=F.+Ledererauthor=Y.+Lindqvist&title=Structure+of+human+glycolate+oxidase+in+complex+with+the+inhibitor+4-carboxy-5-%5B%284-chlorophenyl%29-sulfanyl%5D-1%2C2%2C3-thiadiazole&doi=10.1107%2FS1744309109041670"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1107%2FS1744309109041670&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1744309109041670%26sid%3Dliteratum%253Aachs%26aulast%3DBourhis%26aufirst%3DJ.-M.%26aulast%3DVignaud%26aufirst%3DC.%26aulast%3DPietrancosta%26aufirst%3DN.%26aulast%3DGueritte%26aufirst%3DF.%26aulast%3DGuenard%26aufirst%3DD.%26aulast%3DLederer%26aufirst%3DF.%26aulast%3DLindqvist%26aufirst%3DY.%26atitle%3DStructure%2520of%2520human%2520glycolate%2520oxidase%2520in%2520complex%2520with%2520the%2520inhibitor%25204-carboxy-5-%255B%25284-chlorophenyl%2529-sulfanyl%255D-1%252C2%252C3-thiadiazole%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520F%253A%2520Struct.%2520Biol.%2520Cryst.%2520Commun.%26date%3D2009%26volume%3DF65%26spage%3D1246%26epage%3D1253%26doi%3D10.1107%2FS1744309109041670" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liebow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Racie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hettinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bettencourt, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najafian, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haslett, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erbe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Querbes, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, J.</span></span> <span> </span><span class="NLM_article-title">An investigational RNAi therapeutic targeting glycolate oxidase reduces oxalate production in models of primary hyperoxaluria</span>. <i>J. Am. Soc. Nephrol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">494</span>– <span class="NLM_lpage">503</span>, <span class="refDoi"> DOI: 10.1681/ASN.2016030338</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1681%2FASN.2016030338" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=27432743" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFGisbfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2017&pages=494-503&author=A.+Liebowauthor=X.+Liauthor=T.+Racieauthor=J.+Hettingerauthor=B.+R.+Bettencourtauthor=N.+Najafianauthor=P.+Haslettauthor=K.+Fitzgeraldauthor=R.+P.+Holmesauthor=D.+Erbeauthor=W.+Querbesauthor=J.+Knight&title=An+investigational+RNAi+therapeutic+targeting+glycolate+oxidase+reduces+oxalate+production+in+models+of+primary+hyperoxaluria&doi=10.1681%2FASN.2016030338"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">An investigational RNAI therapeutic targeting glycolate oxidase reduces oxalate production in models of primary hyperoxaluria</span></div><div class="casAuthors">Liebow, Abigail; Li, Xingsheng; Racie, Timothy; Hettinger, Julia; Bettencourt, Brian R.; Najafian, Nader; Haslett, Patrick; Fitzgerald, Kevin; Holmes, Ross P.; Erbe, David; Querbes, William; Knight, John</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Society of Nephrology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">494-503</span>CODEN:
                <span class="NLM_cas:coden">JASNEU</span>;
        ISSN:<span class="NLM_cas:issn">1046-6673</span>.
    
            (<span class="NLM_cas:orgname">American Society of Nephrology</span>)
        </div><div class="casAbstract">Primary hyperoxaluria type 1 (PH1), an inherited rare disease of glyoxylate metab., arises from mutations in the enzyme alanine-glyoxylate aminotransferase.  The resulting deficiency in this enzyme leads to abnor- mally high oxalate prodn. resulting in calcium oxalate crystal formation and deposition in the kidney and many other tissues, with systemic oxalosis and ESRD being a common outcome.  Although a small subset of patients manages the disease with vitamin B6 treatments, the only effective treatment for most is a combined liver-kidney transplant, which requires life-long immune suppression and carries significant mortality risk.  In this report, we discuss the development of ALN-GO1, an investigational RNA interference (RNAi) therapeutic targeting glycolate oxidase, to deplete the substrate for oxalate synthesis.  S.c. administration of ALN-GO1 resulted in potent, dose-dependent, and durable silencing of the mRNA encoding glycolate oxi- dase and increased serum glycolate concns. in wild-type mice, rats, and nonhuman primates.  ALN- GO1 also increased urinary glycolate concns. in normal nonhuman primates and in a genetic mouse model of PH1.  Notably, ALN-GO1 reduced urinary oxalate concn. up to 50% after a single dose in the genetic mouse model of PH1, and up to 98% after multiple doses in a rat model of hyperoxaluria.  These data demonstratethe ability of ALN-GO1 to reduce oxalate prodn. in preclin. models of PH1 across multiple species and provide a clear rationale for clin. trials with this compd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokWZDdJbT9zLVg90H21EOLACvtfcHk0lhBh6QMK97WLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFGisbfE&md5=a010927bb16ddfe9fd85c1792dcd95ca</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1681%2FASN.2016030338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1681%252FASN.2016030338%26sid%3Dliteratum%253Aachs%26aulast%3DLiebow%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DRacie%26aufirst%3DT.%26aulast%3DHettinger%26aufirst%3DJ.%26aulast%3DBettencourt%26aufirst%3DB.%2BR.%26aulast%3DNajafian%26aufirst%3DN.%26aulast%3DHaslett%26aufirst%3DP.%26aulast%3DFitzgerald%26aufirst%3DK.%26aulast%3DHolmes%26aufirst%3DR.%2BP.%26aulast%3DErbe%26aufirst%3DD.%26aulast%3DQuerbes%26aufirst%3DW.%26aulast%3DKnight%26aufirst%3DJ.%26atitle%3DAn%2520investigational%2520RNAi%2520therapeutic%2520targeting%2520glycolate%2520oxidase%2520reduces%2520oxalate%2520production%2520in%2520models%2520of%2520primary%2520hyperoxaluria%26jtitle%3DJ.%2520Am.%2520Soc.%2520Nephrol.%26date%3D2017%26volume%3D28%26spage%3D494%26epage%3D503%26doi%3D10.1681%2FASN.2016030338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dutta, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avitahl-Curtis, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pursell, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diwanji, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apponi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ying, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shui, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxena, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cyr, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nazef, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abrams, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dudek, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salido, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, C.</span></span> <span> </span><span class="NLM_article-title">Inhibition of glycolate oxidase with dicer-substrate siRNA reduces calcium oxalate deposition in mouse model of primary hyperoxaluria type 1</span>. <i>Mol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">770</span>– <span class="NLM_lpage">778</span>, <span class="refDoi"> DOI: 10.1038/mt.2016.4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1038%2Fmt.2016.4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=26758691" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1altr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=770-778&author=C.+Duttaauthor=N.+Avitahl-Curtisauthor=N.+Pursellauthor=M.+L.+Cohenauthor=B.+Holmesauthor=R.+Diwanjiauthor=W.+Zhouauthor=L.+Apponiauthor=M.+Koserauthor=B.+Yingauthor=D.+Chenauthor=X.+Shuiauthor=U.+Saxenaauthor=W.+A.+Cyrauthor=A.+Shahauthor=N.+Nazefauthor=W.+Wangauthor=M.+Abramsauthor=H.+Dudekauthor=E.+Salidoauthor=B.+D.+Brownauthor=C.+Lai&title=Inhibition+of+glycolate+oxidase+with+dicer-substrate+siRNA+reduces+calcium+oxalate+deposition+in+mouse+model+of+primary+hyperoxaluria+type+1&doi=10.1038%2Fmt.2016.4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Glycolate Oxidase With Dicer-substrate siRNA Reduces Calcium Oxalate Deposition in a Mouse Model of Primary Hyperoxaluria Type 1</span></div><div class="casAuthors">Dutta, Chaitali; Avitahl-Curtis, Nicole; Pursell, Natalie; Larsson Cohen, Marita; Holmes, Benjamin; Diwanji, Rohan; Zhou, Wei; Apponi, Luciano; Koser, Martin; Ying, Bo; Chen, Dongyu; Shui, Xue; Saxena, Utsav; Cyr, Wendy A.; Shah, Anee; Nazef, Naim; Wang, Weimin; Abrams, Marc; Dudek, Henryk; Salido, Eduardo; Brown, Bob D.; Lai, Chengjung</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Therapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">770-778</span>CODEN:
                <span class="NLM_cas:coden">MTOHCK</span>;
        ISSN:<span class="NLM_cas:issn">1525-0016</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Primary hyperoxaluria type 1 (PH1) is an autosomal recessive, metabolic disorder caused by mutations of alanine-glyoxylate aminotransferase (AGT), a key hepatic enzyme in the detoxification of glyoxylate arising from multiple normal metabolic pathways to glycine.  Accumulation of glyoxylate, a precursor of oxalate, leads to the overprodn. of oxalate in the liver, which accumulates to high levels in kidneys and urine.  Crystalization of calcium oxalate (CaOx) in the kidney ultimately results in renal failure.  Currently, the only treatment effective in redn. of oxalate prodn. in patients who do not respond to high-dose vitamin B6 therapy is a combined liver/kidney transplant.  We explored an alternative approach to prevent glyoxylate prodn. using Dicer-substrate small interfering RNAs (DsiRNAs) targeting hydroxyacid oxidase 1 (HAO1) mRNA which encodes glycolate oxidase (GO), to reduce the hepatic conversion of glycolate to glyoxylate.  This approach efficiently reduces GO mRNA and protein in the livers of mice and nonhuman primates.  Redn. of hepatic GO leads to normalization of urine oxalate levels and reduces CaOx deposition in a preclin. mouse model of PH1.  Our results support the use of DsiRNA to reduce liver GO levels as a potential therapeutic approach to treat PH1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYr_lKKb91mrVg90H21EOLACvtfcHk0ljCzj0tXVDKkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1altr8%253D&md5=8105de778740769b9cdb5a4db7b6551f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fmt.2016.4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmt.2016.4%26sid%3Dliteratum%253Aachs%26aulast%3DDutta%26aufirst%3DC.%26aulast%3DAvitahl-Curtis%26aufirst%3DN.%26aulast%3DPursell%26aufirst%3DN.%26aulast%3DCohen%26aufirst%3DM.%2BL.%26aulast%3DHolmes%26aufirst%3DB.%26aulast%3DDiwanji%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DApponi%26aufirst%3DL.%26aulast%3DKoser%26aufirst%3DM.%26aulast%3DYing%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DShui%26aufirst%3DX.%26aulast%3DSaxena%26aufirst%3DU.%26aulast%3DCyr%26aufirst%3DW.%2BA.%26aulast%3DShah%26aufirst%3DA.%26aulast%3DNazef%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DAbrams%26aufirst%3DM.%26aulast%3DDudek%26aufirst%3DH.%26aulast%3DSalido%26aufirst%3DE.%26aulast%3DBrown%26aufirst%3DB.%2BD.%26aulast%3DLai%26aufirst%3DC.%26atitle%3DInhibition%2520of%2520glycolate%2520oxidase%2520with%2520dicer-substrate%2520siRNA%2520reduces%2520calcium%2520oxalate%2520deposition%2520in%2520mouse%2520model%2520of%2520primary%2520hyperoxaluria%2520type%25201%26jtitle%3DMol.%2520Ther.%26date%3D2016%26volume%3D24%26spage%3D770%26epage%3D778%26doi%3D10.1038%2Fmt.2016.4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Maag, H.</span>; <span class="NLM_string-name">Fernandes, M. X.</span></span> <span> </span><span class="NLM_article-title">Glycolate oxidase inhibitors and use thereof</span>. <span class="NLM_patent">WO2019133813A1</span>, July 4, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=H.+Maag&author=M.+X.+Fernandes&title=Glycolate+oxidase+inhibitors+and+use+thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMaag%26aufirst%3DH.%26atitle%3DGlycolate%2520oxidase%2520inhibitors%2520and%2520use%2520thereof%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit6b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Maag, H.</span>; <span class="NLM_string-name">Fernandes, M. X.</span>; <span class="NLM_string-name">Zamboni, R.</span>; <span class="NLM_string-name">Akbariromani, E.</span>; <span class="NLM_string-name">Beaulieu, M.-A.</span>; <span class="NLM_string-name">Leblanc, Y.</span>; <span class="NLM_string-name">Thakur, P.</span></span> <span> </span><span class="NLM_article-title">Triazole clycolate oxidase inhibitors</span>. <span class="NLM_patent">WO2020010309A9</span>, January 9, <span class="NLM_year">2020</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=H.+Maag&author=M.+X.+Fernandes&author=R.+Zamboni&author=E.+Akbariromani&author=M.-A.+Beaulieu&author=Y.+Leblanc&author=P.+Thakur&title=Triazole+clycolate+oxidase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMaag%26aufirst%3DH.%26atitle%3DTriazole%2520clycolate%2520oxidase%2520inhibitors%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span>; <span class="NLM_string-name">Chao, Q.</span></span> <span> </span><span class="NLM_article-title">Glycolate oxidase inhibitors for the treatment of disease</span>. <span class="NLM_patent">WO2019133770A2</span>, July 4, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=B.+Wang&author=Q.+Chao&title=Glycolate+oxidase+inhibitors+for+the+treatment+of+disease"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DB.%26atitle%3DGlycolate%2520oxidase%2520inhibitors%2520for%2520the%2520treatment%2520of%2520disease%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keller, B. T.</span>; <span class="NLM_string-name">Talley, J. J.</span></span> <span> </span><span class="NLM_article-title">Compounds and methods for treating oxalate-related diseases</span>. <span class="NLM_patent">WO2019165159A1</span>, August 29, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=B.+T.+Keller&author=J.+J.+Talley&title=Compounds+and+methods+for+treating+oxalate-related+diseases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DKeller%26aufirst%3DB.%2BT.%26atitle%3DCompounds%2520and%2520methods%2520for%2520treating%2520oxalate-related%2520diseases%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowther, W. T.</span></span> <span> </span><span class="NLM_article-title">Active site and loop 4 movements within human glycolate oxidase: Implications for substrate specificity and drug design</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">2439</span>– <span class="NLM_lpage">2449</span>, <span class="refDoi"> DOI: 10.1021/bi701710r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi701710r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptlylsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=2439-2449&author=M.+S.+Murrayauthor=R.+P.+Holmesauthor=W.+T.+Lowther&title=Active+site+and+loop+4+movements+within+human+glycolate+oxidase%3A+Implications+for+substrate+specificity+and+drug+design&doi=10.1021%2Fbi701710r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Active Site and Loop 4 Movements within Human Glycolate Oxidase: Implications for Substrate Specificity and Drug Design</span></div><div class="casAuthors">Murray, Michael S.; Holmes, Ross P.; Lowther, W. Todd</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2439-2449</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human glycolate oxidase (GO) catalyzes the FMN-dependent oxidn. of glycolate to glyoxylate and glyoxylate to oxalate, a key metabolite in kidney stone formation.  We report herein the structures of recombinant GO complexed with sulfate, glyoxylate, and an inhibitor, 4-carboxy-5-dodecylsulfanyl-1,2,3-triazole (CDST), detd. by X-ray crystallog.  In contrast to most α-hydroxy acid oxidases including spinach glycolate oxidase, a loop region, known as loop 4, is completely visible when the GO active site contains a small ligand.  The lack of electron d. for this loop in the GO-CDST complex, which mimics a large substrate, suggests that a disordered to ordered transition may occur with the binding of substrates.  The conformational flexibility of Trp110 appears to be responsible for enabling GO to react with α-hydroxy acids of various chain lengths.  Moreover, the movement of Trp110 disrupts a hydrogen-bonding network between Trp110, Leu191, Tyr134, and Tyr208.  This loss of interactions is the first indication that active site movements are directly linked to changes in the conformation of loop 4.  The kinetic parameters for the oxidn. of glycolate, glyoxylate, and 2-hydroxy octanoate indicate that the oxidn. of glycolate to glyoxylate is the primary reaction catalyzed by GO, while the oxidn. of glyoxylate to oxalate is most likely not relevant under normal conditions.  However, drugs that exploit the unique structural features of GO may ultimately prove to be useful for decreasing glycolate and glyoxylate levels in primary hyperoxaluria type 1 patients who have the inability to convert peroxisomal glyoxylate to glycine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprc1FGNg286bVg90H21EOLACvtfcHk0ljCzj0tXVDKkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptlylsQ%253D%253D&md5=0eacdcc4727c929f4669852dd0c627c7</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fbi701710r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi701710r%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DM.%2BS.%26aulast%3DHolmes%26aufirst%3DR.%2BP.%26aulast%3DLowther%26aufirst%3DW.%2BT.%26atitle%3DActive%2520site%2520and%2520loop%25204%2520movements%2520within%2520human%2520glycolate%2520oxidase%253A%2520Implications%2520for%2520substrate%2520specificity%2520and%2520drug%2520design%26jtitle%3DBiochemistry%26date%3D2008%26volume%3D47%26spage%3D2439%26epage%3D2449%26doi%3D10.1021%2Fbi701710r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Randall, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Streeter, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziegler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cragoe, E. J.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwam, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michelson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, H. W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eichler, E.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of glycolic acid oxidase. 4-Substituted 3-hydroxy-1H-pyrrole-2,5-dione derivatives</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">700</span>– <span class="NLM_lpage">714</span>, <span class="refDoi"> DOI: 10.1021/jm00359a015</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00359a015" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADyaL3sXktVSitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1983&pages=700-714&author=C.+S.+Rooneyauthor=W.+C.+Randallauthor=K.+B.+Streeterauthor=C.+Zieglerauthor=E.+J.+Cragoeauthor=H.+Schwamauthor=S.+R.+Michelsonauthor=H.+W.+R.+Williamsauthor=E.+Eichler&title=Inhibitors+of+glycolic+acid+oxidase.+4-Substituted+3-hydroxy-1H-pyrrole-2%2C5-dione+derivatives&doi=10.1021%2Fjm00359a015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10aR"><div class="casContent"><span class="casTitleNuber">10a</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of glycolic acid oxidase.  4-Substituted 3-hydroxy-1H-pyrrole-2,5-dione derivatives</span></div><div class="casAuthors">Rooney, C. S.; Randall, W. C.; Streeter, K. B.; Ziegler, C.; Cragoe, E. J., Jr.; Schwam, H.; Michelson, S. R.; Williams, H. W. R.; Eichler, E.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">700-14</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">The title compds. I (R = aryl, aryloxy, heterocycle, etc.; R1 = H, Me, or Ph) and related ones were prepd. and evaluated as orally absorbable inhibitors of glycolic acid oxidase (GAO)  [9028-71-1] and to examine their effect on oxalate prodn. in animals.  Inhibitors of GAO are of interest as potentially useful drugs for treatment of Ca oxalate renal lithiasis and the primary hyperoxalurias.  Rat liver perfusion with 3 of the most potent I have shown effective inhibition of the conversion of glycolate to oxalate in this organ.  4-(4'-Bromo[1,1'-biphenyl]-4-yl)-3-hydroxy-1H-pyrrole-2,5-dione (I; R = C6H4-4-(4-BrC6H4); R1 = H)  [77529-42-1] administered orally to ethylene glycol treated rats showed redn. in oxalate excretion in urine.  QSAR are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9vZzyUCBtEbVg90H21EOLACvtfcHk0liBAoAjy25P4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXktVSitrk%253D&md5=e638bd095f104778375ac10a55ee34d2</span></div><a href="/servlet/linkout?suffix=cit10a&amp;dbid=16384&amp;doi=10.1021%2Fjm00359a015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00359a015%26sid%3Dliteratum%253Aachs%26aulast%3DRooney%26aufirst%3DC.%2BS.%26aulast%3DRandall%26aufirst%3DW.%2BC.%26aulast%3DStreeter%26aufirst%3DK.%2BB.%26aulast%3DZiegler%26aufirst%3DC.%26aulast%3DCragoe%26aufirst%3DE.%2BJ.%26aulast%3DSchwam%26aufirst%3DH.%26aulast%3DMichelson%26aufirst%3DS.%2BR.%26aulast%3DWilliams%26aufirst%3DH.%2BW.%2BR.%26aulast%3DEichler%26aufirst%3DE.%26atitle%3DInhibitors%2520of%2520glycolic%2520acid%2520oxidase.%25204-Substituted%25203-hydroxy-1H-pyrrole-2%252C5-dione%2520derivatives%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1983%26volume%3D26%26spage%3D700%26epage%3D714%26doi%3D10.1021%2Fjm00359a015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit10b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, H. W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eichler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Randall, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cragoe, E. J.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Streeter, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwam, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michelson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patchett, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taub, D.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of glycolic acid oxidase. 4-Substituted 2,4-dioxobutanoic acid derivatives</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1196</span>– <span class="NLM_lpage">1200</span>, <span class="refDoi"> DOI: 10.1021/jm00362a020</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00362a020" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADyaL3sXktlyntr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1983&pages=1196-1200&author=H.+W.+R.+Williamsauthor=E.+Eichlerauthor=W.+C.+Randallauthor=C.+S.+Rooneyauthor=E.+J.+Cragoeauthor=K.+B.+Streeterauthor=H.+Schwamauthor=S.+R.+Michelsonauthor=A.+A.+Patchettauthor=D.+Taub&title=Inhibitors+of+glycolic+acid+oxidase.+4-Substituted+2%2C4-dioxobutanoic+acid+derivatives&doi=10.1021%2Fjm00362a020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10bR"><div class="casContent"><span class="casTitleNuber">10b</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of glycolic acid oxidase.  4-Substituted-2,4-dioxobutanoic acid derivatives</span></div><div class="casAuthors">Williams, H. W. R.; Eichler, E.; Randall, W. C.; Rooney, C. S.; Cragoe, E. J., Jr.; Streeter, K. B.; Schwam, H.; Michelson, S. R.; Patchett, A. A.; Taub, D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1196-200</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">Fourteen new 4-substituted 2,4-dioxobutanoic acids were synthesized.  These compds., all of which contain lipophilic 4-substituents, are potent inhibitors in vitro of porcine liver glycolic acid oxidase.  The I50 values of the 2 most potent representatives, 4-(4-BrC6H4)C6H4COCH2COCO2H and I are 6 × 10-8M.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEVw46DI-rQ7Vg90H21EOLACvtfcHk0liBAoAjy25P4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXktlyntr0%253D&md5=45d00f0ed3f769b7e46f9e9b5dcdc697</span></div><a href="/servlet/linkout?suffix=cit10b&amp;dbid=16384&amp;doi=10.1021%2Fjm00362a020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00362a020%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DH.%2BW.%2BR.%26aulast%3DEichler%26aufirst%3DE.%26aulast%3DRandall%26aufirst%3DW.%2BC.%26aulast%3DRooney%26aufirst%3DC.%2BS.%26aulast%3DCragoe%26aufirst%3DE.%2BJ.%26aulast%3DStreeter%26aufirst%3DK.%2BB.%26aulast%3DSchwam%26aufirst%3DH.%26aulast%3DMichelson%26aufirst%3DS.%2BR.%26aulast%3DPatchett%26aufirst%3DA.%2BA.%26aulast%3DTaub%26aufirst%3DD.%26atitle%3DInhibitors%2520of%2520glycolic%2520acid%2520oxidase.%25204-Substituted%25202%252C4-dioxobutanoic%2520acid%2520derivatives%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1983%26volume%3D26%26spage%3D1196%26epage%3D1200%26doi%3D10.1021%2Fjm00362a020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moya-Garzón, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higueras, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penalver, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandes, M. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franco-Montalban, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez-Vidal, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salido, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz-Gavilan, M.</span></span> <span> </span><span class="NLM_article-title">Salicylic acid derivatives inhibit oxalate production in mouse hepatocytes with primary hyperoxaluria type 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">7144</span>– <span class="NLM_lpage">7167</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00399</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00399" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlGit7zM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=7144-7167&author=M.+D.+Moya-Garz%C3%B3nauthor=C.+M.+Higuerasauthor=P.+Penalverauthor=M.+Romeraauthor=M.+X.+Fernandesauthor=F.+Franco-Montalbanauthor=J.+A.+Gomez-Vidalauthor=E.+Salidoauthor=M.+Diaz-Gavilan&title=Salicylic+acid+derivatives+inhibit+oxalate+production+in+mouse+hepatocytes+with+primary+hyperoxaluria+type+1&doi=10.1021%2Facs.jmedchem.8b00399"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Salicylic Acid Derivatives Inhibit Oxalate Production in Mouse Hepatocytes with Primary Hyperoxaluria Type 1</span></div><div class="casAuthors">Moya-Garzon, Maria Dolores; Martin Higueras, Cristina; Penalver, Pablo; Romera, Manuela; Fernandes, Miguel X.; Franco-Montalban, Francisco; Gomez-Vidal, Jose A.; Salido, Eduardo; Diaz-Gavilan, Monica</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7144-7167</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Primary hyperoxaluria type 1 (PH1) is a rare life-threatening genetic disease related to glyoxylate metab. and characterized by accumulation of calcium oxalate crystals.  Current therapies involve hepatic and/or renal transplantation, procedures that have significant morbidity and mortality and require long-term immunosuppression.  Thus, a pharmacol. treatment is urgently needed.  We introduce here an unprecedented activity of salicylic acid derivs. as agents capable of decreasing oxalate output in hyperoxaluric hepatocytes at the low micromolar range, which means a potential use in the treatment of PH1.  Though correlation of this phenotypic activity with glycolate oxidase (GO) inhibition is still to be verified, most of the salicylic acids described here are GO inhibitors with IC50 values down to 3 μM.  Binding mode of salicylic acids inside GO has been studied using in silico methods, and preliminary structure-activity relationships have been established.  The drug-like structure and ease of synthesis of our compds. make them promising hits for structural optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwMGp3szKEj7Vg90H21EOLACvtfcHk0liBAoAjy25P4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlGit7zM&md5=457a9b87854dd7efae5abd0380dc2da5</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00399%26sid%3Dliteratum%253Aachs%26aulast%3DMoya-Garz%25C3%25B3n%26aufirst%3DM.%2BD.%26aulast%3DHigueras%26aufirst%3DC.%2BM.%26aulast%3DPenalver%26aufirst%3DP.%26aulast%3DRomera%26aufirst%3DM.%26aulast%3DFernandes%26aufirst%3DM.%2BX.%26aulast%3DFranco-Montalban%26aufirst%3DF.%26aulast%3DGomez-Vidal%26aufirst%3DJ.%2BA.%26aulast%3DSalido%26aufirst%3DE.%26aulast%3DDiaz-Gavilan%26aufirst%3DM.%26atitle%3DSalicylic%2520acid%2520derivatives%2520inhibit%2520oxalate%2520production%2520in%2520mouse%2520hepatocytes%2520with%2520primary%2520hyperoxaluria%2520type%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D7144%26epage%3D7167%26doi%3D10.1021%2Facs.jmedchem.8b00399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arico-Muendel, C. C.</span></span> <span> </span><span class="NLM_article-title">From haystack to needle: Finding value with DNA encoded library technology at GSK</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1898</span>– <span class="NLM_lpage">1909</span>, <span class="refDoi"> DOI: 10.1039/C6MD00341A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1039%2FC6MD00341A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1ehu7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=1898-1909&author=C.+C.+Arico-Muendel&title=From+haystack+to+needle%3A+Finding+value+with+DNA+encoded+library+technology+at+GSK&doi=10.1039%2FC6MD00341A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">From haystack to needle: finding value with DNA encoded library technology at GSK</span></div><div class="casAuthors">Arico-Muendel, Christopher C.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1898-1909</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">DNA encoded library technol. (ELT) provides access to broad chem. diversity through affinity selection.  As an early adopter, GlaxoSmithKline enabled the development of ELT from proof of concept to full fledged contributor to the small mol. therapeutic pipeline.  Unique benefits of the GSK incubation include uptake of large nos. of building blocks and privileged scaffolds, guidance on library design, and opportunities to add value beyond lead discovery.  Sol. epoxide hydrolase and RIP1 kinase represent two case studies of ELT hits that have advanced into clin. studies.  ELT also shows promise as a strategy to forecast target tractability.  As an emerging technol., it benefits from incubation at the interface of industry and academia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGhx5inxOj_7Vg90H21EOLACvtfcHk0ljlzV_EdDfzVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1ehu7jL&md5=1862af051a956ff1893143e107cff4f4</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1039%2FC6MD00341A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00341A%26sid%3Dliteratum%253Aachs%26aulast%3DArico-Muendel%26aufirst%3DC.%2BC.%26atitle%3DFrom%2520haystack%2520to%2520needle%253A%2520Finding%2520value%2520with%2520DNA%2520encoded%2520library%2520technology%2520at%2520GSK%26jtitle%3DMedChemComm%26date%3D2016%26volume%3D7%26spage%3D1898%26epage%3D1909%26doi%3D10.1039%2FC6MD00341A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halford, B.</span></span> <span> </span><span class="NLM_article-title">How DNA-encoded libraries are revolutionizing drug discovery</span>. <i>Chem. Eng. News</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">28</span>– <span class="NLM_lpage">33</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2017&pages=28-33&author=B.+Halford&title=How+DNA-encoded+libraries+are+revolutionizing+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHalford%26aufirst%3DB.%26atitle%3DHow%2520DNA-encoded%2520libraries%2520are%2520revolutionizing%2520drug%2520discovery%26jtitle%3DChem.%2520Eng.%2520News%26date%3D2017%26volume%3D95%26spage%3D28%26epage%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goodnow, R. A.,  Jr.</span>; <span class="NLM_string-name">Davie, C. P.</span></span> <span> </span><span class="NLM_article-title">DNA-encoded library: a brief guide to its evolution and impact on drug discovery</span>. In  <i>Annual Reports in Medicinal Chemistry</i>; <span class="NLM_publisher-name">Elsevier Inc.</span>, <span class="NLM_year">2017</span>; Vol.  <span class="NLM_volume">50</span>, pp  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">15</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=1-15&author=R.+A.+Goodnow&author=C.+P.+Davie&title=Annual+Reports+in+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DGoodnow%26aufirst%3DR.%2BA.%26atitle%3DDNA-encoded%2520library%253A%2520a%2520brief%2520guide%2520to%2520its%2520evolution%2520and%2520impact%2520on%2520drug%2520discovery%26btitle%3DAnnual%2520Reports%2520in%2520Medicinal%2520Chemistry%26pub%3DElsevier%2520Inc%26date%3D2017%26volume%3D50%26spage%3D1%26epage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gikunju, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habeshian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hupp, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soutter, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent BTK inhibitor with a novel binding mode using parallel selections with a DNA-encoded chemical library</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">864</span>– <span class="NLM_lpage">871</span>, <span class="refDoi"> DOI: 10.1002/cbic.201600573</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1002%2Fcbic.201600573" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=28056160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC2sXit1Chu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=864-871&author=J.+W.+Cuozzoauthor=P.+A.+Centrellaauthor=D.+Gikunjuauthor=S.+Habeshianauthor=C.+D.+Huppauthor=A.+D.+Keefeauthor=E.+A.+Sigelauthor=H.+H.+Soutterauthor=H.+A.+Thomsonauthor=Y.+Zhangauthor=M.+A.+Clark&title=Discovery+of+a+potent+BTK+inhibitor+with+a+novel+binding+mode+using+parallel+selections+with+a+DNA-encoded+chemical+library&doi=10.1002%2Fcbic.201600573"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent BTK Inhibitor with a Novel Binding Mode by Using Parallel Selections with a DNA-Encoded Chemical Library</span></div><div class="casAuthors">Cuozzo, John W.; Centrella, Paolo A.; Gikunju, Diana; Habeshian, Sevan; Hupp, Christopher D.; Keefe, Anthony D.; Sigel, Eric A.; Soutter, Holly H.; Thomson, Heather A.; Zhang, Ying; Clark, Matthew A.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">864-871</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">We have identified and characterized novel potent inhibitors of Bruton's tyrosine kinase (BTK) from a single DNA-encoded library of over 110 million compds. by using multiple parallel selection conditions, including variation in target concn. and addn. of known binders to provide competition information.  Distinct binding profiles were obsd. by comparing enrichments of library building block combinations under these conditions; one enriched only at high concns. of BTK and was competitive with ATP, and another enriched at both high and low concns. of BTK and was not competitive with ATP.  A compd. representing the latter profile showed low nanomolar potency in biochem. and cellular BTK assays.  Results from kinetic mechanism of action studies were consistent with the selection profiles.  Anal. of the co-crystal structure of the most potent compd. demonstrated a novel binding mode that revealed a new pocket in BTK.  Our results demonstrate that profile-based selection strategies using DNA-encoded libraries form the basis of a new methodol. to rapidly identify small mol. inhibitors with novel binding modes to clin. relevant targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxaUUXy-jOi7Vg90H21EOLACvtfcHk0ljlzV_EdDfzVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXit1Chu7Y%253D&md5=22423b0496eb68130236292433bc7251</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201600573&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201600573%26sid%3Dliteratum%253Aachs%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DGikunju%26aufirst%3DD.%26aulast%3DHabeshian%26aufirst%3DS.%26aulast%3DHupp%26aufirst%3DC.%2BD.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DSoutter%26aufirst%3DH.%2BH.%26aulast%3DThomson%26aufirst%3DH.%2BA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DClark%26aufirst%3DM.%2BA.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520BTK%2520inhibitor%2520with%2520a%2520novel%2520binding%2520mode%2520using%2520parallel%2520selections%2520with%2520a%2520DNA-encoded%2520chemical%2520library%26jtitle%3DChemBioChem%26date%3D2017%26volume%3D18%26spage%3D864%26epage%3D871%26doi%3D10.1002%2Fcbic.201600573" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulvihill, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renzetti, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resnicow, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruebsam, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">Novel Autotaxin Inhibitor for the Treatment of Idiopathic Pulmonary Fibrosis: A Clinical Candidate Discovered Using DNA-Encoded Chemistry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">7840</span>– <span class="NLM_lpage">7856</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00688</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00688" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1CrtL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=7840-7856&author=J.+W.+Cuozzoauthor=M.+A.+Clarkauthor=A.+D.+Keefeauthor=A.+Kohlmannauthor=M.+Mulvihillauthor=H.+Niauthor=L.+M.+Renzettiauthor=D.+I.+Resnicowauthor=F.+Ruebsamauthor=E.+A.+Sigelauthor=H.+A.+Thomsonauthor=C.+Wangauthor=Z.+Xieauthor=Y.+Zhang&title=Novel+Autotaxin+Inhibitor+for+the+Treatment+of+Idiopathic+Pulmonary+Fibrosis%3A+A+Clinical+Candidate+Discovered+Using+DNA-Encoded+Chemistry&doi=10.1021%2Facs.jmedchem.0c00688"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Autotaxin Inhibitor for the Treatment of Idiopathic Pulmonary Fibrosis: A Clinical Candidate Discovered Using DNA-Encoded Chemistry</span></div><div class="casAuthors">Cuozzo, John W.; Clark, Matthew A.; Keefe, Anthony D.; Kohlmann, Anna; Mulvihill, Mark; Ni, Haihong; Renzetti, Louis M.; Resnicow, Daniel I.; Ruebsam, Frank; Sigel, Eric A.; Thomson, Heather A.; Wang, Ce; Xie, Zhifeng; Zhang, Ying</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">7840-7856</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The activity of the secreted phosphodiesterase autotaxin produces the inflammatory signaling mol. LPA and has been assocd. with a no. of human diseases including idiopathic pulmonary fibrosis (IPF).  We screened a single DNA-encoded chem. library (DECL) of 225 million compds. and identified a series of potent inhibitors.  Optimization of this series led to the discovery of compd. 1 (X-165)(I), a highly potent, selective, and bioavailable small mol.  Cocrystn. of compd. 1 with human autotaxin demonstrated that it has a novel binding mode occupying both the hydrophobic pocket and a channel near the autotaxin active site.  Compd. 1 inhibited the prodn. of LPA in human and mouse plasma at nanomolar levels and showed efficacy in a mouse model of human lung fibrosis.  After successfully completing IND-enabling studies, compd. 1 was approved by the FDA for a Phase I clin. trial.  These results demonstrate that DECL hits can be readily optimized into clin. candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaofuGyFvOJrVg90H21EOLACvtfcHk0ljlzV_EdDfzVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1CrtL%252FM&md5=c945f9bb71089c94314a27afce04fb96</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00688%26sid%3Dliteratum%253Aachs%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DKohlmann%26aufirst%3DA.%26aulast%3DMulvihill%26aufirst%3DM.%26aulast%3DNi%26aufirst%3DH.%26aulast%3DRenzetti%26aufirst%3DL.%2BM.%26aulast%3DResnicow%26aufirst%3DD.%2BI.%26aulast%3DRuebsam%26aufirst%3DF.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DThomson%26aufirst%3DH.%2BA.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DXie%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DNovel%2520Autotaxin%2520Inhibitor%2520for%2520the%2520Treatment%2520of%2520Idiopathic%2520Pulmonary%2520Fibrosis%253A%2520A%2520Clinical%2520Candidate%2520Discovered%2520Using%2520DNA-Encoded%2520Chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D7840%26epage%3D7856%26doi%3D10.1021%2Facs.jmedchem.0c00688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bazzaz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackett, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collingwood, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disch, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gikunju, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guilinger, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardaker, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennessy, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jetson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoull, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olszewski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pflug, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawlins, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivers, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schimpl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Underwood, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warwicker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter-Holt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodcock, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">Generating Selective Leads for Mer Kinase Inhibitors-Example of a Comprehensive Lead-Generation Strategy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">3165</span>– <span class="NLM_lpage">3184</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01904</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01904" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BB3MXlslWgsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2021&pages=3165-3184&author=J.+W.+M.+Nissinkauthor=A.+Bazzazauthor=C.+Blackettauthor=M.+A.+Clarkauthor=O.+Collingwoodauthor=J.+S.+Dischauthor=D.+Gikunjuauthor=K.+Goldbergauthor=J.+P.+Guilingerauthor=E.+Hardakerauthor=E.+J.+Hennessyauthor=R.+Jetsonauthor=A.+D.+Keefeauthor=W.+McCoullauthor=L.+McMurrayauthor=A.+Olszewskiauthor=R.+Overmanauthor=A.+Pflugauthor=M.+Prestonauthor=P.+B.+Rawlinsauthor=E.+Riversauthor=M.+Schimplauthor=P.+Smithauthor=C.+Trumanauthor=E.+Underwoodauthor=J.+Warwickerauthor=J.+Winter-Holtauthor=S.+Woodcockauthor=Y.+Zhang&title=Generating+Selective+Leads+for+Mer+Kinase+Inhibitors-Example+of+a+Comprehensive+Lead-Generation+Strategy&doi=10.1021%2Facs.jmedchem.0c01904"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Generating Selective Leads for Mer Kinase Inhibitors-Example of a Comprehensive Lead-Generation Strategy</span></div><div class="casAuthors">Nissink, J. Willem M.; Bazzaz, Sana; Blackett, Carolyn; Clark, Matthew A.; Collingwood, Olga; Disch, Jeremy S.; Gikunju, Diana; Goldberg, Kristin; Guilinger, John P.; Hardaker, Elizabeth; Hennessy, Edward J.; Jetson, Rachael; Keefe, Anthony D.; McCoull, William; McMurray, Lindsay; Olszewski, Allison; Overman, Ross; Pflug, Alexander; Preston, Marian; Rawlins, Philip B.; Rivers, Emma; Schimpl, Marianne; Smith, Paul; Truman, Caroline; Underwood, Elizabeth; Warwicker, Juli; Winter-Holt, Jon; Woodcock, Simon; Zhang, Ying</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3165-3184</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mer is a member of the TAM (Tyro3, Axl, Mer) kinase family that has been assocd. with cancer progression, metastasis, and drug resistance.  Their essential function in immune homeostasis has prompted an interest in their role as modulators of antitumor immune response in the tumor microenvironment.  Here we illustrate the outcomes of an extensive lead-generation campaign for identification of Mer inhibitors, focusing on the results from concurrent, orthogonal high-throughput screening approaches.  Data mining, HT (high-throughput), and DECL (DNA-encoded chem. library) screens offered means to evaluate large nos. of compds.  We discuss campaign strategy and screening outcomes, and exemplify series resulting from prioritization of hits that were identified.  Concurrent execution of HT and DECL screening successfully yielded a large no. of potent, selective, and novel starting points, covering a range of selectivity profiles across the TAM family members and modes of kinase binding, and offered excellent start points for lead development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ_gjdZIheGLVg90H21EOLACvtfcHk0lj_XQRLgIYrIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXlslWgsbY%253D&md5=35131754287b08fc9c30d3989d42d52f</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01904&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01904%26sid%3Dliteratum%253Aachs%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DBazzaz%26aufirst%3DA.%26aulast%3DBlackett%26aufirst%3DC.%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DCollingwood%26aufirst%3DO.%26aulast%3DDisch%26aufirst%3DJ.%2BS.%26aulast%3DGikunju%26aufirst%3DD.%26aulast%3DGoldberg%26aufirst%3DK.%26aulast%3DGuilinger%26aufirst%3DJ.%2BP.%26aulast%3DHardaker%26aufirst%3DE.%26aulast%3DHennessy%26aufirst%3DE.%2BJ.%26aulast%3DJetson%26aufirst%3DR.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DMcCoull%26aufirst%3DW.%26aulast%3DMcMurray%26aufirst%3DL.%26aulast%3DOlszewski%26aufirst%3DA.%26aulast%3DOverman%26aufirst%3DR.%26aulast%3DPflug%26aufirst%3DA.%26aulast%3DPreston%26aufirst%3DM.%26aulast%3DRawlins%26aufirst%3DP.%2BB.%26aulast%3DRivers%26aufirst%3DE.%26aulast%3DSchimpl%26aufirst%3DM.%26aulast%3DSmith%26aufirst%3DP.%26aulast%3DTruman%26aufirst%3DC.%26aulast%3DUnderwood%26aufirst%3DE.%26aulast%3DWarwicker%26aufirst%3DJ.%26aulast%3DWinter-Holt%26aufirst%3DJ.%26aulast%3DWoodcock%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DGenerating%2520Selective%2520Leads%2520for%2520Mer%2520Kinase%2520Inhibitors-Example%2520of%2520a%2520Comprehensive%2520Lead-Generation%2520Strategy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2021%26volume%3D64%26spage%3D3165%26epage%3D3184%26doi%3D10.1021%2Facs.jmedchem.0c01904" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Veerman, J. J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruseker, Y. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heijne, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Bruggen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hekking, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hupp, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McRiner, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulvihill, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Rijnsbergen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zech, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renzetti, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babiss, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, G.</span></span> <span> </span><span class="NLM_article-title">Discovery of 2,4-1H-Imidazole Carboxamides as Potent and Selective TAK1 Inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">555</span>– <span class="NLM_lpage">562</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.0c00547</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.0c00547" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BB3MXltl2lt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2021&pages=555-562&author=J.+J.+N.+Veermanauthor=Y.+B.+Brusekerauthor=E.+Damenauthor=E.+H.+Heijneauthor=W.+van+Bruggenauthor=K.+F.+Hekkingauthor=R.+Winkelauthor=C.+D.+Huppauthor=A.+D.+Keefeauthor=J.+Liuauthor=H.+A.+Thomsonauthor=Y.+Zhangauthor=J.+W.+Cuozzoauthor=A.+J.+McRinerauthor=M.+J.+Mulvihillauthor=P.+van+Rijnsbergenauthor=B.+Zechauthor=L.+M.+Renzettiauthor=L.+Babissauthor=G.+Muller&title=Discovery+of+2%2C4-1H-Imidazole+Carboxamides+as+Potent+and+Selective+TAK1+Inhibitors&doi=10.1021%2Facsmedchemlett.0c00547"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 2,4-1H-Imidazole Carboxamides as Potent and Selective TAK1 Inhibitors</span></div><div class="casAuthors">Veerman, Johan J. N.; Bruseker, Yorik B.; Damen, Eddy; Heijne, Erik H.; van Bruggen, Wendy; Hekking, Koen F. W.; Winkel, Rob; Hupp, Christopher D.; Keefe, Anthony D.; Liu, Julie; Thomson, Heather A.; Zhang, Ying; Cuozzo, John W.; McRiner, Andrew J.; Mulvihill, Mark J.; van Rijnsbergen, Peter; Zech, Birgit; Renzetti, Louis M.; Babiss, Lee; Mueller, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">555-562</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Herein we report the discovery of 2,4-1H-imidazole carboxamides as novel, biochem. potent, and kinome selective inhibitors of transforming growth factor β-activated kinase 1 (TAK1).  The target was subjected to a DNA-encoded chem. library (DECL) screen.  After hit anal. a cluster of compds. was identified, which was based on a central pyrrole-2,4-1H-dicarboxamide scaffold, showing remarkable kinome selectivity.  A scaffold-hop to the corresponding imidazole resulted in increased biochem. potency.  Next, X-ray crystallog. revealed a distinct binding mode compared to other TAK1 inhibitors.  A benzylamide was found in a perpendicular orientation with respect to the core hinge-binding imidazole.  Addnl., an unusual amide flip was obsd. in the kinase hinge region.  Using structure-based drug design (SBDD), key substitutions at the pyrrolidine amide and the glycine resulted in a significant increase in biochem. potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsfdQbRX4C1LVg90H21EOLACvtfcHk0lj_XQRLgIYrIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXltl2lt7o%253D&md5=26c25af441806342c2048ce49c6be192</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.0c00547&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.0c00547%26sid%3Dliteratum%253Aachs%26aulast%3DVeerman%26aufirst%3DJ.%2BJ.%2BN.%26aulast%3DBruseker%26aufirst%3DY.%2BB.%26aulast%3DDamen%26aufirst%3DE.%26aulast%3DHeijne%26aufirst%3DE.%2BH.%26aulast%3Dvan%2BBruggen%26aufirst%3DW.%26aulast%3DHekking%26aufirst%3DK.%2BF.%26aulast%3DWinkel%26aufirst%3DR.%26aulast%3DHupp%26aufirst%3DC.%2BD.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DThomson%26aufirst%3DH.%2BA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DMcRiner%26aufirst%3DA.%2BJ.%26aulast%3DMulvihill%26aufirst%3DM.%2BJ.%26aulast%3Dvan%2BRijnsbergen%26aufirst%3DP.%26aulast%3DZech%26aufirst%3DB.%26aulast%3DRenzetti%26aufirst%3DL.%2BM.%26aulast%3DBabiss%26aufirst%3DL.%26aulast%3DMuller%26aufirst%3DG.%26atitle%3DDiscovery%2520of%25202%252C4-1H-Imidazole%2520Carboxamides%2520as%2520Potent%2520and%2520Selective%2520TAK1%2520Inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2021%26volume%3D12%26spage%3D555%26epage%3D562%26doi%3D10.1021%2Facsmedchemlett.0c00547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mimasu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamagishi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiyohara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matusda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yahata, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hupp, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kakefuda, K.</span></span> <span> </span><span class="NLM_article-title">Full-length in meso structure and mechanism of rat kynurenine 3-monooxygenase inhibition</span>. <i>Commun. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">159</span> <span class="refDoi"> DOI: 10.1038/s42003-021-01666-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1038%2Fs42003-021-01666-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=33542467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BB3MXjsVCltbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2021&author=S.+Mimasuauthor=H.+Yamagishiauthor=S.+Kuboauthor=M.+Kiyoharaauthor=T.+Matusdaauthor=T.+Yahataauthor=H.+A.+Thomsonauthor=C.+H.+Huppauthor=J.+Liuauthor=T.+Okudaauthor=K.+Kakefuda&title=Full-length+in+meso+structure+and+mechanism+of+rat+kynurenine+3-monooxygenase+inhibition&doi=10.1038%2Fs42003-021-01666-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Full-length in meso structure and mechanism of rat kynurenine 3-monooxygenase inhibition</span></div><div class="casAuthors">Mimasu, Shinya; Yamagishi, Hiroaki; Kubo, Satoshi; Kiyohara, Mie; Matsuda, Toshihiro; Yahata, Toshiko; Thomson, Heather A.; Hupp, Christopher D.; Liu, Julie; Okuda, Takao; Kakefuda, Kenichi</div><div class="citationInfo"><span class="NLM_cas:title">Communications Biology</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">159</span>CODEN:
                <span class="NLM_cas:coden">CBOIDQ</span>;
        ISSN:<span class="NLM_cas:issn">2399-3642</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: The structural mechanisms of single-pass transmembrane enzymes remain elusive.  Kynurenine 3-monooxygenase (KMO) is a mitochondrial protein involved in the eukaryotic tryptophan catabolic pathway and is linked to various diseases.  Here, we report the mammalian full-length structure of KMO in its membrane-embedded form, complexed with compd. 3 (identified internally) and compd. 4 (identified via DNA-encoded chem. library screening) at 3.0 Å resoln.  Despite predictions suggesting that KMO has two transmembrane domains, we show that KMO is actually a single-pass transmembrane protein, with the other transmembrane domain lying laterally along the membrane, where it forms part of the ligand-binding pocket.  Further exploration of compd. 3 led to identification of the brain-penetrant compd., 5.  We show that KMO is dimeric, and that mutations at the dimeric interface abolish its activity.  These results will provide insight for the drug discovery of addnl. blood-brain-barrier mols., and help illuminate the complex biol. behind single-pass transmembrane enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlMxD8lAIIpLVg90H21EOLACvtfcHk0ljwMUARfnJjrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXjsVCltbs%253D&md5=07139f96f3bc4f8688fa0078e99166d3</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fs42003-021-01666-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42003-021-01666-5%26sid%3Dliteratum%253Aachs%26aulast%3DMimasu%26aufirst%3DS.%26aulast%3DYamagishi%26aufirst%3DH.%26aulast%3DKubo%26aufirst%3DS.%26aulast%3DKiyohara%26aufirst%3DM.%26aulast%3DMatusda%26aufirst%3DT.%26aulast%3DYahata%26aufirst%3DT.%26aulast%3DThomson%26aufirst%3DH.%2BA.%26aulast%3DHupp%26aufirst%3DC.%2BH.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DOkuda%26aufirst%3DT.%26aulast%3DKakefuda%26aufirst%3DK.%26atitle%3DFull-length%2520in%2520meso%2520structure%2520and%2520mechanism%2520of%2520rat%2520kynurenine%25203-monooxygenase%2520inhibition%26jtitle%3DCommun.%2520Biol.%26date%3D2021%26volume%3D4%26doi%3D10.1038%2Fs42003-021-01666-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundstrom, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geschwindner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poon, E. K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohman, R.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordqvist, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friden-Saxin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dekker, N.</span></span> <span> </span><span class="NLM_article-title">Protease-activated receptor-2 ligands reveal orthosteric and allosteric mechanisms of receptor inhibition</span>. <i>Commun. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>3</i></span>, <span class="NLM_elocation-id">782</span> <span class="refDoi"> DOI: 10.1038/s42003-020-01504-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1038%2Fs42003-020-01504-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=33335291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis12it7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2020&author=A.+J.+Kennedyauthor=L.+Sundstromauthor=S.+Geschwindnerauthor=E.+K.+Y.+Poonauthor=Y.+Jiangauthor=R.+Chenauthor=R.+Cookeauthor=S.+Johnstoneauthor=A.+Madinauthor=J.+Limauthor=Q.+Liuauthor=R.-J.+Lohmanauthor=A.+Nordqvistauthor=M.+Friden-Saxinauthor=W.+Yangauthor=D.+G.+Brownauthor=D.+P.+Fairlieauthor=N.+Dekker&title=Protease-activated+receptor-2+ligands+reveal+orthosteric+and+allosteric+mechanisms+of+receptor+inhibition&doi=10.1038%2Fs42003-020-01504-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Protease-activated receptor-2 ligands reveal orthosteric and allosteric mechanisms of receptor inhibition</span></div><div class="casAuthors">Kennedy, Amanda J.; Sundstroem, Linda; Geschwindner, Stefan; Poon, Eunice K. Y.; Jiang, Yuhong; Chen, Rongfeng; Cooke, Rob; Johnstone, Shawn; Madin, Andrew; Lim, Junxian; Liu, Qingqi; Lohman, Rink-Jan; Nordqvist, Anneli; Friden-Saxin, Maria; Yang, Wenzhen; Brown, Dean G.; Fairlie, David P.; Dekker, Niek</div><div class="citationInfo"><span class="NLM_cas:title">Communications Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">782</span>CODEN:
                <span class="NLM_cas:coden">CBOIDQ</span>;
        ISSN:<span class="NLM_cas:issn">2399-3642</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Protease-activated receptor-2 (PAR2) has been implicated in multiple pathophysiologies but drug discovery is challenging due to low small mol. tractability and a complex activation mechanism.  Here we report the pharmacol. profiling of a potent new agonist, suggested by mol. modeling to bind in the putative orthosteric site, and two novel PAR2 antagonists with distinctly different mechanisms of inhibition.  We identify coupling between different PAR2 binding sites.  One antagonist is a competitive inhibitor that binds to the orthosteric site, while a second antagonist is a neg. allosteric modulator that binds at a remote site.  The allosteric modulator shows probe dependence, more effectively inhibiting peptide than protease activation of PAR2 signalling.  Importantly, both antagonists are active in vivo, inhibiting PAR2 agonist-induced acute paw inflammation in rats and preventing activation of mast cells and neutrophils.  These results highlight two distinct mechanisms of inhibition that potentially could be targeted for future development of drugs that modulate PAR2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAX4X0epY4cbVg90H21EOLACvtfcHk0ljwMUARfnJjrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis12it7zK&md5=e4401ba74eac9c73c2cbe157f0270f4a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fs42003-020-01504-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42003-020-01504-0%26sid%3Dliteratum%253Aachs%26aulast%3DKennedy%26aufirst%3DA.%2BJ.%26aulast%3DSundstrom%26aufirst%3DL.%26aulast%3DGeschwindner%26aufirst%3DS.%26aulast%3DPoon%26aufirst%3DE.%2BK.%2BY.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DR.%26aulast%3DCooke%26aufirst%3DR.%26aulast%3DJohnstone%26aufirst%3DS.%26aulast%3DMadin%26aufirst%3DA.%26aulast%3DLim%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DLohman%26aufirst%3DR.-J.%26aulast%3DNordqvist%26aufirst%3DA.%26aulast%3DFriden-Saxin%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26aulast%3DDekker%26aufirst%3DN.%26atitle%3DProtease-activated%2520receptor-2%2520ligands%2520reveal%2520orthosteric%2520and%2520allosteric%2520mechanisms%2520of%2520receptor%2520inhibition%26jtitle%3DCommun.%2520Biol.%26date%3D2020%26volume%3D3%26doi%3D10.1038%2Fs42003-020-01504-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Belyanskaya, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazaar, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israel, D. I.</span></span> <span> </span><span class="NLM_article-title">Discovering drugs with DNA-encoded library technology: From concept to clinic with an inhibitor of soluble epoxide hydrolase</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">837</span>– <span class="NLM_lpage">842</span>, <span class="refDoi"> DOI: 10.1002/cbic.201700014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1002%2Fcbic.201700014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=28281333" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVWjurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=837-842&author=S.+L.+Belyanskayaauthor=Y.+Dingauthor=J.+F.+Callahanauthor=A.+L.+Lazaarauthor=D.+I.+Israel&title=Discovering+drugs+with+DNA-encoded+library+technology%3A+From+concept+to+clinic+with+an+inhibitor+of+soluble+epoxide+hydrolase&doi=10.1002%2Fcbic.201700014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Discovering Drugs with DNA-Encoded Library Technology: From Concept to Clinic with an Inhibitor of Soluble Epoxide Hydrolase</span></div><div class="casAuthors">Belyanskaya, Svetlana L.; Ding, Yun; Callahan, James F.; Lazaar, Aili L.; Israel, David I.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">837-842</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">DNA-encoded chem. library technol. was developed with the vision of its becoming a transformational platform for drug discovery.  The hope was that a new paradigm for the discovery of low-mol.-wt. drugs would be enabled by combining the vast mol. diversity achievable with combinatorial chem., the information-encoding attributes of DNA, the power of mol. biol., and a streamlined selection-based discovery process.  Here, we describe the discovery and early clin. development of GSK2256294, an inhibitor of sol. epoxide hydrolase (sEH, EPHX2), by using encoded-library technol. (ELT).  GSK2256294 is an orally bioavailable, potent and selective inhibitor of sEH that has a long half life and produced no serious adverse events in a first-time-in-human clin. study.  To our knowledge, GSK2256294 is the first mol. discovered from this technol. to enter human clin. testing and represents a realization of the vision that DNA-encoded chem. library technol. can efficiently yield mols. with favorable properties that can be readily progressed into high-quality drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7WTPZp2SE0rVg90H21EOLACvtfcHk0ljwMUARfnJjrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVWjurk%253D&md5=a34e28c8817864356f6ff689e9236940</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201700014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201700014%26sid%3Dliteratum%253Aachs%26aulast%3DBelyanskaya%26aufirst%3DS.%2BL.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DLazaar%26aufirst%3DA.%2BL.%26aulast%3DIsrael%26aufirst%3DD.%2BI.%26atitle%3DDiscovering%2520drugs%2520with%2520DNA-encoded%2520library%2520technology%253A%2520From%2520concept%2520to%2520clinic%2520with%2520an%2520inhibitor%2520of%2520soluble%2520epoxide%2520hydrolase%26jtitle%3DChemBioChem%26date%3D2017%26volume%3D18%26spage%3D837%26epage%3D842%26doi%3D10.1002%2Fcbic.201700014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arico-Muendel, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belyanskaya, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benjamin, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creaser, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davie, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franzen, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gefter, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, N. J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israel, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavarana, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollmann, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lind, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mataruse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medeiros, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Keefe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliff, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rise, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satz, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svendsen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Vloten, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, B. A.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and selection of DNA-encoded small-molecule libraries</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">647</span>– <span class="NLM_lpage">654</span>, <span class="refDoi"> DOI: 10.1038/nchembio.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1038%2Fnchembio.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=19648931" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1Ogtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=647-654&author=M.+A.+Clarkauthor=R.+A.+Acharyaauthor=C.+C.+Arico-Muendelauthor=S.+L.+Belyanskayaauthor=D.+R.+Benjaminauthor=N.+R.+Carlsonauthor=P.+A.+Centrellaauthor=C.+H.+Chiuauthor=S.+P.+Creaserauthor=J.+W.+Cuozzoauthor=C.+P.+Davieauthor=Y.+Dingauthor=G.+J.+Franklinauthor=K.+D.+Franzenauthor=M.+L.+Gefterauthor=S.+P.+Haleauthor=N.+J.+V.+Hansenauthor=D.+I.+Israelauthor=J.+Jiangauthor=M.+J.+Kavaranaauthor=M.+S.+Kellyauthor=C.+S.+Kollmannauthor=F.+Liauthor=K.+Lindauthor=S.+Mataruseauthor=P.+F.+Medeirosauthor=J.+A.+Messerauthor=P.+Myersauthor=H.+O%E2%80%99Keefeauthor=M.+C.+Oliffauthor=C.+E.+Riseauthor=A.+L.+Satzauthor=S.+R.+Skinnerauthor=J.+L.+Svendsenauthor=L.+Tangauthor=K.+van+Vlotenauthor=R.+W.+Wagnerauthor=G.+Yaoauthor=B.+Zhaoauthor=B.+A.+Morgan&title=Design%2C+synthesis+and+selection+of+DNA-encoded+small-molecule+libraries&doi=10.1038%2Fnchembio.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and selection of DNA-encoded small-molecule libraries</span></div><div class="casAuthors">Clark, Matthew A.; Acharya, Raksha A.; Arico-Muendel, Christopher C.; Belyanskaya, Svetlana L.; Benjamin, Dennis R.; Carlson, Neil R.; Centrella, Paolo A.; Chiu, Cynthia H.; Creaser, Steffen P.; Cuozzo, John W.; Davie, Christopher P.; Ding, Yun; Franklin, G. Joseph; Franzen, Kurt D.; Gefter, Malcolm L.; Hale, Steven P.; Hansen, Nils J. V.; Israel, David I.; Jiang, Jinwei; Kavarana, Malcolm J.; Kelley, Michael S.; Kollmann, Christopher S.; Li, Fan; Lind, Kenneth; Mataruse, Sibongile; Medeiros, Patricia F.; Messer, Jeffrey A.; Myers, Paul; O'Keefe, Heather; Oliff, Matthew C.; Rise, Cecil E.; Satz, Alexander L.; Skinner, Steven R.; Svendsen, Jennifer L.; Tang, Lujia; van Vloten, Kurt; Wagner, Richard W.; Yao, Gang; Zhao, Baoguang; Morgan, Barry A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">647-654</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Biochem. combinatorial techniques such as phage display, RNA display and oligonucleotide aptamers have proven to be reliable methods for generation of ligands to protein targets.  Adapting these techniques to small synthetic mols. has been a long-sought goal.  We report the synthesis and interrogation of an 800-million-member DNA-encoded library in which small mols. are covalently attached to an encoding oligonucleotide.  The library was assembled by a combination of chem. and enzymic synthesis, and interrogated by affinity selection.  We describe methods for the selection and deconvolution of the chem. display library, and the discovery of inhibitors for two enzymes: Aurora A kinase and p38 MAP kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp32j3MHOgNurVg90H21EOLACvtfcHk0liv_-DkbHnheA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1Ogtbo%253D&md5=8a40148f5fca24ecf912ed360802e19d</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.211%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DAcharya%26aufirst%3DR.%2BA.%26aulast%3DArico-Muendel%26aufirst%3DC.%2BC.%26aulast%3DBelyanskaya%26aufirst%3DS.%2BL.%26aulast%3DBenjamin%26aufirst%3DD.%2BR.%26aulast%3DCarlson%26aufirst%3DN.%2BR.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DChiu%26aufirst%3DC.%2BH.%26aulast%3DCreaser%26aufirst%3DS.%2BP.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DDavie%26aufirst%3DC.%2BP.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DFranklin%26aufirst%3DG.%2BJ.%26aulast%3DFranzen%26aufirst%3DK.%2BD.%26aulast%3DGefter%26aufirst%3DM.%2BL.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DHansen%26aufirst%3DN.%2BJ.%2BV.%26aulast%3DIsrael%26aufirst%3DD.%2BI.%26aulast%3DJiang%26aufirst%3DJ.%26aulast%3DKavarana%26aufirst%3DM.%2BJ.%26aulast%3DKelly%26aufirst%3DM.%2BS.%26aulast%3DKollmann%26aufirst%3DC.%2BS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLind%26aufirst%3DK.%26aulast%3DMataruse%26aufirst%3DS.%26aulast%3DMedeiros%26aufirst%3DP.%2BF.%26aulast%3DMesser%26aufirst%3DJ.%2BA.%26aulast%3DMyers%26aufirst%3DP.%26aulast%3DO%25E2%2580%2599Keefe%26aufirst%3DH.%26aulast%3DOliff%26aufirst%3DM.%2BC.%26aulast%3DRise%26aufirst%3DC.%2BE.%26aulast%3DSatz%26aufirst%3DA.%2BL.%26aulast%3DSkinner%26aufirst%3DS.%2BR.%26aulast%3DSvendsen%26aufirst%3DJ.%2BL.%26aulast%3DTang%26aufirst%3DL.%26aulast%3Dvan%2BVloten%26aufirst%3DK.%26aulast%3DWagner%26aufirst%3DR.%2BW.%26aulast%3DYao%26aufirst%3DG.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DMorgan%26aufirst%3DB.%2BA.%26atitle%3DDesign%252C%2520synthesis%2520and%2520selection%2520of%2520DNA-encoded%2520small-molecule%2520libraries%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D647%26epage%3D654%26doi%3D10.1038%2Fnchembio.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span> <span> </span><span class="NLM_article-title">Selecting chemicals: the emerging utility of DNA-encoded libraries</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">396</span>– <span class="NLM_lpage">403</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2010.02.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1016%2Fj.cbpa.2010.02.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=20346729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=396-403&author=M.+A.+Clark&title=Selecting+chemicals%3A+the+emerging+utility+of+DNA-encoded+libraries&doi=10.1016%2Fj.cbpa.2010.02.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Selecting chemicals: the emerging utility of DNA-encoded libraries</span></div><div class="casAuthors">Clark, Matthew A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">396-403</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Over the past 10 years, a handful of academic and industrial research groups have developed strategies for the synthesis and interrogation of DNA-encoded small-mol. libraries.  These strategies can be divided into those in which DNA directs small-mol. synthesis and those in which it records the synthesis.  These libraries have started to yield novel modulators of biol. targets, including: SH3-domain-binding peptoids, macrocyclic peptide-based Bcl-XL/BH3 interaction disruptors, ligands for TNF, albumin, streptavidin and others, and small-mol. kinase inhibitors.  The DNA-encoded library field holds the potential to address the general problem of biol. ligand discovery, including pharmaceutical lead generation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBheFYsHS5irVg90H21EOLACvtfcHk0liv_-DkbHnheA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsb4%253D&md5=163a93c8ca0f6f4b96352153b466fdc2</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2010.02.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2010.02.017%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DM.%2BA.%26atitle%3DSelecting%2520chemicals%253A%2520the%2520emerging%2520utility%2520of%2520DNA-encoded%2520libraries%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D14%26spage%3D396%26epage%3D403%26doi%3D10.1016%2Fj.cbpa.2010.02.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goodnow, R. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span> <span> </span><span class="NLM_article-title">DNA-encoded chemistry: enabling the deeper sampling of chemical space</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">147</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.213</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1038%2Fnrd.2016.213" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=27932801" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVKkurjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=131-147&author=R.+A.+Goodnowauthor=C.+E.+Dumelinauthor=A.+D.+Keefe&title=DNA-encoded+chemistry%3A+enabling+the+deeper+sampling+of+chemical+space&doi=10.1038%2Fnrd.2016.213"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-encoded chemistry: enabling the deeper sampling of chemical space</span></div><div class="casAuthors">Goodnow, Robert A. Jr; Dumelin, Christoph E.; Keefe, Anthony D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">131-147</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  DNA-encoded chem. library technologies are increasingly being adopted in drug discovery for hit and lead generation.  DNA-encoded chem. enables the exploration of chem. spaces four to five orders of magnitude more deeply than is achievable by traditional high-throughput screening methods.  Operation of this technol. requires developing a range of capabilities including aq. synthetic chem., building block acquisition, oligonucleotide conjugation, large-scale mol. biol. transformations, selection methodologies, PCR, sequencing, sequence data anal. and the anal. of large chem. spaces.  This Review provides an overview of the development and applications of DNA-encoded chem., highlighting the challenges and future directions for the use of this technol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdEd70MeexerVg90H21EOLACvtfcHk0liv_-DkbHnheA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVKkurjM&md5=7ca97b01d75e600378799baf77665c07</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.213%26sid%3Dliteratum%253Aachs%26aulast%3DGoodnow%26aufirst%3DR.%2BA.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26atitle%3DDNA-encoded%2520chemistry%253A%2520enabling%2520the%2520deeper%2520sampling%2520of%2520chemical%2520space%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D131%26epage%3D147%26doi%3D10.1038%2Fnrd.2016.213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Franzini, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheuermann, J.</span></span> <span> </span><span class="NLM_article-title">DNA-encoded chemical libraries: Advancing beyond conventional small-molecule libraries</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1247</span>– <span class="NLM_lpage">1255</span>, <span class="refDoi"> DOI: 10.1021/ar400284t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar400284t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltF2ls7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2014&pages=1247-1255&author=R.+M.+Franziniauthor=D.+Neriauthor=J.+Scheuermann&title=DNA-encoded+chemical+libraries%3A+Advancing+beyond+conventional+small-molecule+libraries&doi=10.1021%2Far400284t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-Encoded Chemical Libraries: Advancing beyond Conventional Small-Molecule Libraries</span></div><div class="casAuthors">Franzini, Raphael M.; Neri, Dario; Scheuermann, Jorg</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1247-1255</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  DNA-encoded chem. libraries (DECLs) represent a promising tool in drug discovery.  DECL technol. allows the synthesis and screening of chem. libraries of unprecedented size at moderate costs.  In analogy to phage-display technol., where large antibody libraries are displayed on the surface of filamentous phage and are genetically encoded in the phage genome, DECLs feature the display of individual small org. chem. moieties on DNA fragments serving as amplifiable identification barcodes.  The DNA-tag facilitates the synthesis and allows the simultaneous screening of very large sets of compds. (up to billions of mols.), because the hit compds. can easily be identified and quantified by PCR-amplification of the DNA-barcode followed by high-throughput DNA sequencing.  Several approaches have been used to generate DECLs, differing both in the methods used for library encoding and for the combinatorial assembly of chem. moieties.  For example, DECLs can be used for fragment-based drug discovery, displaying a single mol. on DNA or two chem. moieties at the extremities of complementary DNA strands.  DECLs can vary substantially in the chem. structures and the library size.  While ultralarge libraries contg. billions of compds. have been reported contg. four or more sets of building blocks, also smaller libraries have been shown to be efficient for ligand discovery.  In general, it has been found that the overall library size is a poor predictor for library performance and that the no. and diversity of the building blocks are rather important indicators.  Smaller libraries consisting of two to three sets of building blocks better fulfill the criteria of drug-likeness and often have higher quality.  In this Account, we present advances in the DECL field from proof-of-principle studies to practical applications for drug discovery, both in industry and in academia.  DECL technol. can yield specific binders to a variety of target proteins and is likely to become a std. tool for pharmaceutical hit discovery, lead expansion, and Chem. Biol. research.  The introduction of new methodologies for library encoding and for compd. synthesis in the presence of DNA is an exciting research field and will crucially contribute to the performance and the propagation of the technol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcaCmssNgPLLVg90H21EOLACvtfcHk0ljuXUCIIg6E9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltF2ls7g%253D&md5=c37bd1625b668e869fcc26d5ef0a017a</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Far400284t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far400284t%26sid%3Dliteratum%253Aachs%26aulast%3DFranzini%26aufirst%3DR.%2BM.%26aulast%3DNeri%26aufirst%3DD.%26aulast%3DScheuermann%26aufirst%3DJ.%26atitle%3DDNA-encoded%2520chemical%2520libraries%253A%2520Advancing%2520beyond%2520conventional%2520small-molecule%2520libraries%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2014%26volume%3D47%26spage%3D1247%26epage%3D1255%26doi%3D10.1021%2Far400284t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, G. T.</span></span> <span> </span><span class="NLM_article-title">DNA-Encoded library screening as core platform technology in drug discovery: Its synthetic method development and applications in DEL synthesis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">6578</span>– <span class="NLM_lpage">6599</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01782</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01782" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis12nsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=6578-6599&author=M.+Songauthor=G.+T.+Hwang&title=DNA-Encoded+library+screening+as+core+platform+technology+in+drug+discovery%3A+Its+synthetic+method+development+and+applications+in+DEL+synthesis&doi=10.1021%2Facs.jmedchem.9b01782"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-Encoded Library Screening as Core Platform Technology in Drug Discovery: Its Synthetic Method Development and Applications in DEL Synthesis</span></div><div class="casAuthors">Song, Minsoo; Hwang, Gil Tae</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6578-6599</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  DNA-encoded library technol. (DELT) was introduced to our medicinal chem. society more than 20 years ago.  The application of DELT in the development of clin. candidates has been actively reported in the literature recently.  A few representative examples include RIP1K inhibitors for inflammatory diseases and sEH inhibitors for endothelial dysfunction or abnormal tissue repair, among many others.  Here, the authors would like to recall the recent developments in on-DNA synthetic methodologies for DEL construction and to analyze recent examples in the literature of DELT-based drug development efforts pursued in both the academic and industrial sectors.  With this perspective, we hope to provide a useful summary of recent DELT-based drug discovery research and to discuss the future scope of DELT in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOdR3yKaXgmrVg90H21EOLACvtfcHk0ljuXUCIIg6E9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis12nsb8%253D&md5=c462a02125c354e82883013f0be12e60</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01782&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01782%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DM.%26aulast%3DHwang%26aufirst%3DG.%2BT.%26atitle%3DDNA-Encoded%2520library%2520screening%2520as%2520core%2520platform%2520technology%2520in%2520drug%2520discovery%253A%2520Its%2520synthetic%2520method%2520development%2520and%2520applications%2520in%2520DEL%2520synthesis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D6578%26epage%3D6599%26doi%3D10.1021%2Facs.jmedchem.9b01782" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fargue, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southall, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKew, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danpure, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span> <span> </span><span class="NLM_article-title">High throughput cell-based assay for identification of glycolate oxidase inhibitors as a potential treatment for primary hyperoxaluria type 1</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">34060</span> <span class="refDoi"> DOI: 10.1038/srep34060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1038%2Fsrep34060" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=27670739" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsF2qsL7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&author=M.+Wangauthor=M.+Xuauthor=Y.+Longauthor=S.+Fargueauthor=N.+Southallauthor=X.+Huauthor=J.+C.+McKewauthor=C.+J.+Danpureauthor=W.+Zheng&title=High+throughput+cell-based+assay+for+identification+of+glycolate+oxidase+inhibitors+as+a+potential+treatment+for+primary+hyperoxaluria+type+1&doi=10.1038%2Fsrep34060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">High throughput cell-based assay for identification of glycolate oxidase inhibitors as a potential treatment for Primary Hyperoxaluria Type 1</span></div><div class="casAuthors">Wang, Mengqiao; Xu, Miao; Long, Yan; Fargue, Sonia; Southall, Noel; Hu, Xin; McKew, John C.; Danpure, Christopher J.; Zheng, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">34060</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Glycolate oxidase (GO) and alanine:glyoxylate aminotransferase (AGT) are both involved in the peroxisomal glyoxylate pathway.  Deficiency in AGT function causes the accumulation of intracellular oxalate and the primary hyperoxaluria type 1 (PH1).  AGT enhancers or GO inhibitors may restore the abnormal peroxisomal glyoxylate pathway in PH1 patients.  With stably transformed cells which mimic the glyoxylate metabolic pathway, we developed an indirect glycolate cytotoxicity assay in a 1,536-well plate format for high throughput screening.  This assay can be used to identify compds. that reduce indirect glycolate-induced cytotoxicity by either enhancing AGT activity or inhibiting GO.  A pilot screen of 4,096 known compds. identified two membrane permeable GO inhibitors: dichromate salt and colistimethate.  We also developed a GO enzyme assay using the hydrogen peroxide-Amplex red reporter system.  The IC50 values of potassium dichromate, sodium dichromate, and colistimethate sodium were 0.096, 0.108, and 2.3 μM in the GO enzyme assay, resp.  Further enzyme kinetic study revealed that both types of compds. inhibit GO activity by the mixed linear inhibition.  Our results demonstrate that the cell-based assay and GO enzyme assay developed in this study are useful for further screening of large compd. libraries for drug development to treat PH1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7wOw_TIVi3LVg90H21EOLACvtfcHk0ljuXUCIIg6E9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsF2qsL7P&md5=487ef1a45249a9251349c6898c8e9588</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fsrep34060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep34060%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DLong%26aufirst%3DY.%26aulast%3DFargue%26aufirst%3DS.%26aulast%3DSouthall%26aufirst%3DN.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DMcKew%26aufirst%3DJ.%2BC.%26aulast%3DDanpure%26aufirst%3DC.%2BJ.%26aulast%3DZheng%26aufirst%3DW.%26atitle%3DHigh%2520throughput%2520cell-based%2520assay%2520for%2520identification%2520of%2520glycolate%2520oxidase%2520inhibitors%2520as%2520a%2520potential%2520treatment%2520for%2520primary%2520hyperoxaluria%2520type%25201%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26doi%3D10.1038%2Fsrep34060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cymborowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chruszcz, M.</span></span> <span> </span><span class="NLM_article-title">HKL-3000: the integration of data reduction and structure solution - from diffraction images to an initial model in minutes</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">859</span>– <span class="NLM_lpage">866</span>, <span class="refDoi"> DOI: 10.1107/S0907444906019949</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1107%2FS0907444906019949" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=16855301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=859-866&author=W.+Minorauthor=M.+Cymborowskiauthor=Z.+Otwinowskiauthor=M.+Chruszcz&title=HKL-3000%3A+the+integration+of+data+reduction+and+structure+solution+-+from+diffraction+images+to+an+initial+model+in+minutes&doi=10.1107%2FS0907444906019949"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">HKL-3000: the integration of data reduction and structure solution - from diffraction images to an initial model in minutes</span></div><div class="casAuthors">Minor, Wladek; Cymborowski, Marcin; Otwinowski, Zbyszek; Chruszcz, Maksymilian</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">859-866</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A new approach that integrates data collection, data redn., phasing and model building significantly accelerates the process of structure detn. and on av. minimizes the no. of data sets and synchrotron time required for structure soln.  Initial testing of the HKL-3000 system (the beta version was named HKL-2000_ph) with more than 140 novel structure detns. has proven its high value for MAD/SAD expts.  The heuristics for choosing the best computational strategy at different data resoln. limits of phasing signal and crystal diffraction are being optimized.  The typical end result is an interpretable electron-d. map with a partially built structure and, in some cases, an almost complete refined model.  The current development is oriented towards very fast structure soln. in order to provide feedback during the diffraction expt.  Work is also proceeding towards improving the quality of phasing calcn. and model building.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0c0DYtjFP7rVg90H21EOLACvtfcHk0lgx8YavxY5Kaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D&md5=2cd2fe288f954837abb4acae4c008a2f</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS0907444906019949&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906019949%26sid%3Dliteratum%253Aachs%26aulast%3DMinor%26aufirst%3DW.%26aulast%3DCymborowski%26aufirst%3DM.%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DChruszcz%26aufirst%3DM.%26atitle%3DHKL-3000%253A%2520the%2520integration%2520of%2520data%2520reduction%2520and%2520structure%2520solution%2520-%2520from%2520diffraction%2520images%2520to%2520an%2520initial%2520model%2520in%2520minutes%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D859%26epage%3D866%26doi%3D10.1107%2FS0907444906019949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lgx8YavxY5Kaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0liMMjGslapv8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0liMMjGslapv8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span> <i>Schrodinger Release
2019-3</i>; <span class="NLM_publisher-name">Schrodinger,
LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Schrodinger+Release%0A2019-3%3B+Schrodinger%2C%0ALLC%3A+New+York%2C+NY%2C+2019."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSchrodinger%2520Release%250A2019-3%26pub%3DSchrodinger%252C%250ALLC%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T.</span></span> <span> </span><span class="NLM_article-title">Identifying and characterizing binding sites and assessing druggability</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1021/ci800324m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800324m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=377-389&author=T.+Halgren&title=Identifying+and+characterizing+binding+sites+and+assessing+druggability&doi=10.1021%2Fci800324m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying and Characterizing Binding Sites and Assessing Druggability</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">377-389</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Identification and characterization of binding sites is key in the process of structure-based drug design.  In some cases there may not be any information about the binding site for a target of interest.  In other cases, a putative binding site has been identified by computational or exptl. means, but the druggability of the target is not known.  Even when a site for a given target is known, it may be desirable to find addnl. sites whose targeting could produce a desired biol. response.  A new program, called SiteMap, is presented for identifying and analyzing binding sites and for predicting target druggability.  In a large-scale validation, SiteMap correctly identifies the known binding site as the top-ranked site in 86% of the cases, with best results (>98%) coming for sites that bind ligands with subnanomolar affinity.  In addn., a modified version of the score employed for binding-site identification allows SiteMap to accurately classify the druggability of proteins as measured by their ability to bind passively absorbed small mols. tightly.  In characterizing binding sites, SiteMap provides quant. and graphical information that can help guide efforts to critically assess virtual hits in a lead-discovery application or to modify ligand structure to enhance potency or improve phys. properties in a lead-optimization context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfAt0-gvcXDLVg90H21EOLACvtfcHk0liMMjGslapv8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D&md5=f03c7b27e30ae03d10e4127ed1f1ed3c</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fci800324m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800324m%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%26atitle%3DIdentifying%2520and%2520characterizing%2520binding%2520sites%2520and%2520assessing%2520druggability%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D377%26epage%3D389%26doi%3D10.1021%2Fci800324m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T.</span></span> <span> </span><span class="NLM_article-title">New method for fast and accurate binding-site identification and analysis</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">146</span>– <span class="NLM_lpage">148</span>, <span class="refDoi"> DOI: 10.1111/j.1747-0285.2007.00483.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1111%2Fj.1747-0285.2007.00483.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=17381729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkt1OnsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2007&pages=146-148&author=T.+Halgren&title=New+method+for+fast+and+accurate+binding-site+identification+and+analysis&doi=10.1111%2Fj.1747-0285.2007.00483.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">New method for fast and accurate binding-site identification and analysis</span></div><div class="casAuthors">Halgren, Tom</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">146-148</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">Structure-based drug design seeks to exploit the structure of protein-ligand or protein-protein binding sites, but the site is not always known at the outset.  Even when the site is known, the researcher may wish to identify alternative prospective binding sites that may result in different biol. effects or new class of compds.  It is also vital in lead optimization to clearly understand the degree to which known binders or docking hits satisfy or violate complementarity to the receptor.  SiteMap is a new technique for identifying potential binding sites and for predicting their druggability in lead-discovery applications and for characterizing binding sites and critically assessing prospective ligands in lead-optimization applications.  In large-scale validation tests, SiteMap correctly identifies the known binding site in > 96% of the cases, with best results (> 98%) coming for sites that bind ligands tightly.  It also accurately distinguishes between sites that bind ligands and sites that don't.  In binding-site anal., SiteMap provides a wealth of quant. and graphical information that can help guide efforts to modify ligand structure to enhance potency or improve phys. properties.  These attributes allow SiteMap to nicely complement techniques such as docking and computational lead optimization in structure-base drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr245hmZWXilrVg90H21EOLACvtfcHk0liV_1Mkpx43oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkt1OnsLg%253D&md5=c233276ba9580393b282f255781f9ce6</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2007.00483.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2007.00483.x%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%26atitle%3DNew%2520method%2520for%2520fast%2520and%2520accurate%2520binding-site%2520identification%2520and%2520analysis%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2007%26volume%3D69%26spage%3D146%26epage%3D148%26doi%3D10.1111%2Fj.1747-0285.2007.00483.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sastry, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adzhigirey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annabhimoju, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Protein and ligand preparation: Parameters, protocols, and influence on virtual screening enrichments</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">221</span>– <span class="NLM_lpage">234</span>, <span class="refDoi"> DOI: 10.1007/s10822-013-9644-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1007%2Fs10822-013-9644-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=23579614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A280%3ADC%252BC3srksFekug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=221-234&author=G.+M.+Sastryauthor=M.+Adzhigireyauthor=T.+Dayauthor=R.+Annabhimojuauthor=W.+Sherman&title=Protein+and+ligand+preparation%3A+Parameters%2C+protocols%2C+and+influence+on+virtual+screening+enrichments&doi=10.1007%2Fs10822-013-9644-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span></div><div class="casAuthors">Sastry G Madhavi; Adzhigirey Matvey; Day Tyler; Annabhimoju Ramakrishna; Sherman Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of computer-aided molecular design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">221-34</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Structure-based virtual screening plays an important role in drug discovery and complements other screening approaches.  In general, protein crystal structures are prepared prior to docking in order to add hydrogen atoms, optimize hydrogen bonds, remove atomic clashes, and perform other operations that are not part of the x-ray crystal structure refinement process.  In addition, ligands must be prepared to create 3-dimensional geometries, assign proper bond orders, and generate accessible tautomer and ionization states prior to virtual screening.  While the prerequisite for proper system preparation is generally accepted in the field, an extensive study of the preparation steps and their effect on virtual screening enrichments has not been performed.  In this work, we systematically explore each of the steps involved in preparing a system for virtual screening.  We first explore a large number of parameters using the Glide validation set of 36 crystal structures and 1,000 decoys.  We then apply a subset of protocols to the DUD database.  We show that database enrichment is improved with proper preparation and that neglecting certain steps of the preparation process produces a systematic degradation in enrichments, which can be large for some targets.  We provide examples illustrating the structural changes introduced by the preparation that impact database enrichment.  While the work presented here was performed with the Protein Preparation Wizard and Glide, the insights and guidance are expected to be generalizable to structure-based virtual screening with other docking methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQd2Qiai-OqeXqj_yg7twvpfW6udTcc2eZ4EKQytMCWULntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3srksFekug%253D%253D&md5=9eb1a820e121aca2742ed53f24481ace</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1007%2Fs10822-013-9644-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-013-9644-8%26sid%3Dliteratum%253Aachs%26aulast%3DSastry%26aufirst%3DG.%2BM.%26aulast%3DAdzhigirey%26aufirst%3DM.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DAnnabhimoju%26aufirst%3DR.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DProtein%2520and%2520ligand%2520preparation%253A%2520Parameters%252C%2520protocols%252C%2520and%2520influence%2520on%2520virtual%2520screening%2520enrichments%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26spage%3D221%26epage%3D234%26doi%3D10.1007%2Fs10822-013-9644-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanschagrin, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span> <span> </span><span class="NLM_article-title">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein-Ligand Complexes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">6177</span>– <span class="NLM_lpage">6196</span>, <span class="refDoi"> DOI: 10.1021/jm051256o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm051256o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6177-6196&author=R.+A.+Friesnerauthor=R.+B.+Murphyauthor=M.+P.+Repaskyauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=T.+A.+Halgrenauthor=P.+C.+Sanschagrinauthor=D.+T.+Mainz&title=Extra+Precision+Glide%3A+Docking+and+Scoring+Incorporating+a+Model+of+Hydrophobic+Enclosure+for+Protein-Ligand+Complexes&doi=10.1021%2Fjm051256o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein-Ligand Complexes</span></div><div class="casAuthors">Friesner, Richard A.; Murphy, Robert B.; Repasky, Matthew P.; Frye, Leah L.; Greenwood, Jeremy R.; Halgren, Thomas A.; Sanschagrin, Paul C.; Mainz, Daniel T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6177-6196</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel scoring function to est. protein-ligand binding affinities has been developed and implemented as the Glide 4.0 XP scoring function and docking protocol.  In addn. to unique water desolvation energy terms, protein-ligand structural motifs leading to enhanced binding affinity are included:(1) hydrophobic enclosure where groups of lipophilic ligand atoms are enclosed on opposite faces by lipophilic protein atoms, (2) neutral-neutral single or correlated hydrogen bonds in a hydrophobically enclosed environment, and (3) five categories of charged-charged hydrogen bonds.  The XP scoring function and docking protocol have been developed to reproduce exptl. binding affinities for a set of 198 complexes (RMSDs of 2.26 and 1.73 kcal/mol over all and well-docked ligands, resp.) and to yield quality enrichments for a set of fifteen screens of pharmaceutical importance.  Enrichment results demonstrate the importance of the novel XP mol. recognition and water scoring in sepg. active and inactive ligands and avoiding false positives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohZiwN12vaY7Vg90H21EOLACvtfcHk0liV_1Mkpx43oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D&md5=ea428c82ead0d8c27f8c1a7b694a1edf</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26atitle%3DExtra%2520Precision%2520Glide%253A%2520Docking%2520and%2520Scoring%2520Incorporating%2520a%2520Model%2520of%2520Hydrophobic%2520Enclosure%2520for%2520Protein-Ligand%2520Complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6177%26epage%3D6196%26doi%3D10.1021%2Fjm051256o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 2. Enrichment Factors in Database Screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+2.+Enrichment+Factors+in+Database+Screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lhZcCOn4RL1hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25202.%2520Enrichment%2520Factors%2520in%2520Database%2520Screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 1. Method and Assessment of Docking Accuracy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1739</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+1.+Method+and+Assessment+of+Docking+Accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lhZcCOn4RL1hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25201.%2520Method%2520and%2520Assessment%2520of%2520Docking%2520Accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span> <i>Schrodinger Release
2019-3: Glide</i>; <span class="NLM_publisher-name">Schrodinger, LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Schrodinger+Release%0A2019-3%3A+Glide%3B+Schrodinger%2C+LLC%3A+New+York%2C+NY%2C+2019."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSchrodinger%2520Release%250A2019-3%253A%2520Glide%26pub%3DSchrodinger%252C%2520LLC%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span> <i>Schrodinger Release 2019-3: Ligprep</i>; <span class="NLM_publisher-name">Schrodinger, LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Schrodinger+Release+2019-3%3A+Ligprep%3B+Schrodinger%2C+LLC%3A+New+York%2C+NY%2C+2019."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSchrodinger%2520Release%25202019-3%253A%2520Ligprep%26pub%3DSchrodinger%252C%2520LLC%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pincus, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rapp, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T. J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honig, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">A Hierarchical Approach to All-Atom Protein Loop Prediction</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">351</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1002/prot.10613</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1002%2Fprot.10613" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=15048827" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtFKhsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2004&pages=351-367&author=M.+P.+Jacobsonauthor=D.+L.+Pincusauthor=C.+S.+Rappauthor=T.+J.+F.+Dayauthor=B.+Honigauthor=D.+E.+Shawauthor=R.+A.+Friesner&title=A+Hierarchical+Approach+to+All-Atom+Protein+Loop+Prediction&doi=10.1002%2Fprot.10613"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A hierarchical approach to all-atom protein loop prediction</span></div><div class="casAuthors">Jacobson, Matthew P.; Pincus, David L.; Rapp, Chaya S.; Day, Tyler J. F.; Honig, Barry; Shaw, David E.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">351-367</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The application of all-atom force fields (and explicit or implicit solvent models) to protein homol.-modeling tasks such as side-chain and loop prediction remains challenging both because of the expense of the individual energy calcns. and because of the difficulty of sampling the rugged all-atom energy surface.  Here the authors address this challenge for the problem of loop prediction through the development of numerous new algorithms, with an emphasis on multiscale and hierarchical techniques.  As a first step in evaluating the performance of the authors' loop prediction algorithm, the authors have applied it to the problem of reconstructing loops in native structures; the authors also explicitly include crystal packing to provide a fair comparison with crystal structures.  In brief, large nos. of loops are generated by using a dihedral angle-based buildup procedure followed by iterative cycles of clustering, side-chain optimization, and complete energy minimization of selected loop structures.  The authors evaluate this method by the largest test set yet used for validation of a loop prediction method, with a total of 833 loops ranging from 4 to 12 residues in length.  Av./median backbone root-mean-square deviations (RMSDs) to the native structures (superimposing the body of the protein, not the loop itself) are 0.42/0.24 Å for 5 residue loops, 1.00/0.44 Å for 8 residue loops, and 2.47/1.83 Å for 11 residue loops.  Median RMSDs are substantially lower than the avs. because of a small no. of outliers; the causes of these failures are examd. in some detail, and many can be attributed to errors in assignment of protonation states of titratable residues, omission of ligands from the simulation, and, in a few cases, probable errors in the exptl. detd. structures.  When these obvious problems in the data sets are filtered out, av. RMSDs to the native structures improve to 0.43 Å for 5 residue loops, 0.84 Å for 8 residue loops, and 1.63 Å for 11 residue loops.  In the vast majority of cases, the method locates energy min. that are lower than or equal to that of the minimized native loop, thus indicating that sampling rarely limits prediction accuracy.  The overall results are, to the authors' knowledge, the best reported to date, and the authors attribute this success to the combination of an accurate all-atom energy function, efficient methods for loop buildup and side-chain optimization, and, esp. for the longer loops, the hierarchical refinement protocol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHqYMnqaWXvLVg90H21EOLACvtfcHk0lis1yvFwZ3qpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtFKhsrc%253D&md5=e0eff655eeefb30ea00ae041ea9099c8</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fprot.10613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10613%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DPincus%26aufirst%3DD.%2BL.%26aulast%3DRapp%26aufirst%3DC.%2BS.%26aulast%3DDay%26aufirst%3DT.%2BJ.%2BF.%26aulast%3DHonig%26aufirst%3DB.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DA%2520Hierarchical%2520Approach%2520to%2520All-Atom%2520Protein%2520Loop%2520Prediction%26jtitle%3DProteins%26date%3D2004%26volume%3D55%26spage%3D351%26epage%3D367%26doi%3D10.1002%2Fprot.10613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honig, B.</span></span> <span> </span><span class="NLM_article-title">On the Role of Crystal Packing Forces in Determining Protein Sidechain Conformations</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>320</i></span>,  <span class="NLM_fpage">597</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1016/S0022-2836(02)00470-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=10.1016%2FS0022-2836%2802%2900470-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=12096912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;key=1%3ACAS%3A528%3ADC%252BD38XltVKmu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=320&publication_year=2002&pages=597-608&author=M.+P.+Jacobsonauthor=R.+A.+Friesnerauthor=Z.+Xiangauthor=B.+Honig&title=On+the+Role+of+Crystal+Packing+Forces+in+Determining+Protein+Sidechain+Conformations&doi=10.1016%2FS0022-2836%2802%2900470-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">On the Role of the Crystal Environment in Determining Protein Side-chain Conformations</span></div><div class="casAuthors">Jacobson, Matthew P.; Friesner, Richard A.; Xiang, Zhexin; Honig, Barry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">320</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">597-608</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">The role of crystal packing in detg. the obsd. conformations of amino acid side-chains in protein crystals is investigated by (1) anal. of a database of proteins that have been crystd. in different unit cells (space group or unit cell dimensions) and (2) theor. predictions of side-chain conformations with the crystal environment explicitly represented.  Both of these approaches indicate that the crystal environment plays an important role in detg. the conformations of polar side-chains on the surfaces of proteins.  Inclusion of the crystal environment permits a more sensitive measurement of the achievable accuracy of side-chain prediction programs, when validating against structures obtained by x-ray crystallog.  Our side-chain prediction program uses an all-atom force field and a Generalized Born model of solvation and is thus capable of modeling simple packing effects (i.e. van der Waals interactions), electrostatic effects, and desolvation, which are all important mechanisms by which the crystal environment impacts obsd. side-chain conformations.  Our results are also relevant to the understanding of changes in side-chain conformation that may result from ligand docking and protein-protein assocn., insofar as the results reveal how side-chain conformations change in response to their local environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgHEOVpIEiErVg90H21EOLACvtfcHk0lis1yvFwZ3qpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltVKmu70%253D&md5=006de6bd2d0f233ab32d6798dc1a3fbc</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2900470-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252900470-9%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DXiang%26aufirst%3DZ.%26aulast%3DHonig%26aufirst%3DB.%26atitle%3DOn%2520the%2520Role%2520of%2520Crystal%2520Packing%2520Forces%2520in%2520Determining%2520Protein%2520Sidechain%2520Conformations%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2002%26volume%3D320%26spage%3D597%26epage%3D608%26doi%3D10.1016%2FS0022-2836%2802%2900470-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span> <i>Schrodinger Release
2019-3: Prime</i>; <span class="NLM_publisher-name">Schrodinger, LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Schrodinger+Release%0A2019-3%3A+Prime%3B+Schrodinger%2C+LLC%3A+New+York%2C+NY%2C+2019."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSchrodinger%2520Release%250A2019-3%253A%2520Prime%26pub%3DSchrodinger%252C%2520LLC%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W45" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W45','PDB','6W45'); return false;">PDB: 6W45</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W44" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W44','PDB','6W44'); return false;">PDB: 6W44</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W4C" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W4C','PDB','6W4C'); return false;">PDB: 6W4C</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i51"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02271">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_10123"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c02271?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02271</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">SAR (Tables S1–S3); crystal data collect and reduction (Table S4); crystal refinement statistics (Table S5); protein production; compound interference assay; HAO1 siRNA rescue glycolic acid-induced cell-death in CHO-GO cells; synthesis and characterization for compounds <b>7–20, 22–28, 30</b>, and <b>32–39</b>; and LCMS and NMR Spectra for compounds <b>3–6</b>, <b>21</b>, <b>29</b>, and <b>31</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02271/suppl_file/jm0c02271_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02271/suppl_file/jm0c02271_si_002.xlsx">XLSX</a>)</p></li><li><p class="inline">Associated biological assay data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02271/suppl_file/jm0c02271_si_003.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02271/suppl_file/jm0c02271_si_001.pdf">jm0c02271_si_001.pdf (5.46 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02271/suppl_file/jm0c02271_si_002.xlsx">jm0c02271_si_002.xlsx (11.01 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02271/suppl_file/jm0c02271_si_003.pdb">jm0c02271_si_003.pdb (266.05 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The PDB ID for X-ray crystal structures of human HAO1 with compounds <b>3</b>, <b>4</b>, and <b>5</b> are <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W45">6W45</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W44">6W44</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W4C">6W4C</a>, respectively. The authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c02271&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-10%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c02271%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c02271" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799181fbbc63cb8","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
