
<html lang="en"     class="pb-page"  data-request-id="8c2b1a68-0e95-4175-8168-caaa99638b8d"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.7b01372;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2017.60.issue-24;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="A Unique Approach to Design Potent and Selective Cyclic Adenosine Monophosphate Response Element Binding Protein, Binding Protein (CBP) Inhibitors" /></meta><meta name="dc.Creator" content="Sarah M.  Bronner" /></meta><meta name="dc.Creator" content="Jeremy  Murray" /></meta><meta name="dc.Creator" content="F. Anthony  Romero" /></meta><meta name="dc.Creator" content="Kwong Wah  Lai" /></meta><meta name="dc.Creator" content="Vickie  Tsui" /></meta><meta name="dc.Creator" content="Patrick  Cyr" /></meta><meta name="dc.Creator" content="Maureen H.  Beresini" /></meta><meta name="dc.Creator" content="Gladys  de leon Boenig" /></meta><meta name="dc.Creator" content="Zhongguo  Chen" /></meta><meta name="dc.Creator" content="Edna F.  Choo" /></meta><meta name="dc.Creator" content="Kevin R.  Clark" /></meta><meta name="dc.Creator" content="Terry D.  Crawford" /></meta><meta name="dc.Creator" content="Hariharan  Jayaram" /></meta><meta name="dc.Creator" content="Susan  Kaufman" /></meta><meta name="dc.Creator" content="Ruina  Li" /></meta><meta name="dc.Creator" content="Yingjie  Li" /></meta><meta name="dc.Creator" content="Jiangpeng  Liao" /></meta><meta name="dc.Creator" content="Xiaorong  Liang" /></meta><meta name="dc.Creator" content="Wenfeng  Liu" /></meta><meta name="dc.Creator" content="Justin  Ly" /></meta><meta name="dc.Creator" content="Jonathan  Maher" /></meta><meta name="dc.Creator" content="John  Wai" /></meta><meta name="dc.Creator" content="Fei  Wang" /></meta><meta name="dc.Creator" content="Aijun  Zheng" /></meta><meta name="dc.Creator" content="Xiaoyu  Zhu" /></meta><meta name="dc.Creator" content="Steven  Magnuson" /></meta><meta name="dc.Description" content="The epigenetic regulator CBP/P300 presents a novel therapeutic target for oncology. Previously, we disclosed the development of potent and selective CBP bromodomain inhibitors by first identifying ..." /></meta><meta name="Description" content="The epigenetic regulator CBP/P300 presents a novel therapeutic target for oncology. Previously, we disclosed the development of potent and selective CBP bromodomain inhibitors by first identifying ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 11, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b01372" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01372" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b01372" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01372" /></link>
        
    
    

<title>A Unique Approach to Design Potent and Selective Cyclic Adenosine Monophosphate Response Element Binding Protein, Binding Protein (CBP) Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b01372" /></meta><meta property="og:title" content="A Unique Approach to Design Potent and Selective Cyclic Adenosine Monophosphate Response Element Binding Protein, Binding Protein (CBP) Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0025.jpeg" /></meta><meta property="og:description" content="The epigenetic regulator CBP/P300 presents a novel therapeutic target for oncology. Previously, we disclosed the development of potent and selective CBP bromodomain inhibitors by first identifying pharmacophores that bind the KAc region and then building into the LPF shelf. Herein, we report the “hybridization” of a variety of KAc-binding fragments with a tetrahydroquinoline scaffold that makes optimal interactions with the LPF shelf, imparting enhanced potency and selectivity to the hybridized ligand. To demonstrate the utility of our hybridization approach, two analogues containing unique Asn binders and the optimized tetrahydroquinoline moiety were rapidly optimized to yield single-digit nanomolar inhibitors of CBP with exquisite selectivity over BRD4(1) and the broader bromodomain family." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b01372"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b01372">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b01372&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b01372&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b01372&amp;href=/doi/10.1021/acs.jmedchem.7b01372" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 24</span><span class="cit-fg-pageRange">, 10151-10171</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/24" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b01348" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b01454" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">A Unique Approach to Design Potent and Selective Cyclic Adenosine Monophosphate Response Element Binding Protein, Binding Protein (CBP) Inhibitors</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sarah+M.++Bronner">Sarah M. Bronner</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-4370-6727" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jeremy++Murray">Jeremy Murray</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=F.+Anthony++Romero">F. Anthony Romero</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0001-7917-1459" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kwong+Wah++Lai">Kwong Wah Lai</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Vickie++Tsui">Vickie Tsui</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Patrick++Cyr">Patrick Cyr</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0001-5320-3483" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Maureen+H.++Beresini">Maureen H. Beresini</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Gladys++de+leon+Boenig">Gladys de leon Boenig</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Zhongguo++Chen">Zhongguo Chen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Edna+F.++Choo">Edna F. Choo</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-4200-8302" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kevin+R.++Clark">Kevin R. Clark</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Terry+D.++Crawford">Terry D. Crawford</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-4663-4116" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hariharan++Jayaram">Hariharan Jayaram</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Susan++Kaufman">Susan Kaufman</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ruina++Li">Ruina Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yingjie++Li">Yingjie Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jiangpeng++Liao">Jiangpeng Liao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xiaorong++Liang">Xiaorong Liang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Wenfeng++Liu">Wenfeng Liu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Justin++Ly">Justin Ly</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jonathan++Maher">Jonathan Maher</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=John++Wai">John Wai</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Fei++Wang">Fei Wang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Aijun++Zheng">Aijun Zheng</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xiaoyu++Zhu">Xiaoyu Zhu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Steven++Magnuson">Steven Magnuson</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Genentech, Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Wuxi Apptec Co., Ltd., 288 Fute Zhong Road, Waigaoqiao Free Trade Zone, Shanghai 200131, People’s Republic of China</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Editas Medicine, Inc., 11 Hurley Street, Cambridge, Massachusetts 02141, United States</span></div><div class="corresp-info"><strong>*</strong>Phone: +1-650-467-4426. E-mail: <a href="/cdn-cgi/l/email-protection#375545585959524519445645565f77505259521954585a"><span class="__cf_email__" data-cfemail="6200100d0c0c07104c110310030a22">[email protected]</span>gene.com</a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01372&amp;href=/doi/10.1021%2Facs.jmedchem.7b01372" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 24</span><span class="cit-pageRange">, 10151–10171</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 20, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>23 September 2017</li><li><span class="item_label"><b>Published</b> online</span>11 December 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 28 December 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b01372" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01372</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10151%26pageCount%3D21%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DSarah%2BM.%2BBronner%252C%2BJeremy%2BMurray%252C%2BF.%2BAnthony%2BRomero%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D24%26contentID%3Dacs.jmedchem.7b01372%26title%3DA%2BUnique%2BApproach%2Bto%2BDesign%2BPotent%2Band%2BSelective%2BCyclic%2BAdenosine%2BMonophosphate%2BResponse%2BElement%2BBinding%2BProtein%252C%2BBinding%2BProtein%2B%2528CBP%2529%2BInhibitors%26numPages%3D21%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10171%26publicationDate%3DDecember%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b01372"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2580</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">10</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b01372" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;A Unique Approach to Design Potent and Selective Cyclic Adenosine Monophosphate Response Element Binding Protein, Binding Protein (CBP) Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Sarah&quot;,&quot;last_name&quot;:&quot;M. Bronner&quot;},{&quot;first_name&quot;:&quot;Jeremy&quot;,&quot;last_name&quot;:&quot;Murray&quot;},{&quot;first_name&quot;:&quot;F.&quot;,&quot;last_name&quot;:&quot;Anthony Romero&quot;},{&quot;first_name&quot;:&quot;Kwong&quot;,&quot;last_name&quot;:&quot;Wah Lai&quot;},{&quot;first_name&quot;:&quot;Vickie&quot;,&quot;last_name&quot;:&quot;Tsui&quot;},{&quot;first_name&quot;:&quot;Patrick&quot;,&quot;last_name&quot;:&quot;Cyr&quot;},{&quot;first_name&quot;:&quot;Maureen&quot;,&quot;last_name&quot;:&quot;H. Beresini&quot;},{&quot;first_name&quot;:&quot;Gladys&quot;,&quot;last_name&quot;:&quot;de leon Boenig&quot;},{&quot;first_name&quot;:&quot;Zhongguo&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Edna&quot;,&quot;last_name&quot;:&quot;F. Choo&quot;},{&quot;first_name&quot;:&quot;Kevin&quot;,&quot;last_name&quot;:&quot;R. Clark&quot;},{&quot;first_name&quot;:&quot;Terry&quot;,&quot;last_name&quot;:&quot;D. Crawford&quot;},{&quot;first_name&quot;:&quot;Hariharan&quot;,&quot;last_name&quot;:&quot;Jayaram&quot;},{&quot;first_name&quot;:&quot;Susan&quot;,&quot;last_name&quot;:&quot;Kaufman&quot;},{&quot;first_name&quot;:&quot;Ruina&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Yingjie&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Jiangpeng&quot;,&quot;last_name&quot;:&quot;Liao&quot;},{&quot;first_name&quot;:&quot;Xiaorong&quot;,&quot;last_name&quot;:&quot;Liang&quot;},{&quot;first_name&quot;:&quot;Wenfeng&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Justin&quot;,&quot;last_name&quot;:&quot;Ly&quot;},{&quot;first_name&quot;:&quot;Jonathan&quot;,&quot;last_name&quot;:&quot;Maher&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;Wai&quot;},{&quot;first_name&quot;:&quot;Fei&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Aijun&quot;,&quot;last_name&quot;:&quot;Zheng&quot;},{&quot;first_name&quot;:&quot;Xiaoyu&quot;,&quot;last_name&quot;:&quot;Zhu&quot;},{&quot;first_name&quot;:&quot;Steven&quot;,&quot;last_name&quot;:&quot;Magnuson&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;11&quot;,&quot;issue&quot;:&quot;24&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;10151-10171&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b01372&quot;},&quot;abstract&quot;:&quot;The epigenetic regulator CBP/P300 presents a novel therapeutic target for oncology. Previously, we disclosed the development of potent and selective CBP bromodomain inhibitors by first identifying pharmacophores that bind the KAc region and then building into the LPF shelf. Herein, we report the “hybridization” of a variety of KAc-binding fragments with a tetrahydroquinoline scaffold that makes optimal interactions with the LPF shelf, imparting enhanced potency and selectivity to the hybridized ligand. To demonstrate the utility of our hybridization approach, two analogues containing unique Asn binders and the optimized tetrahydroquinoline moiety were rapidly optimized to yield single-digit nanomolar inhibitors of CBP with exquisite selectivity over BRD4(1) and the broader bromodomain family.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01372&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01372" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01372&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01372" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01372&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01372" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01372&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01372&amp;href=/doi/10.1021/acs.jmedchem.7b01372" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b01372" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b01372" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (9 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b01372%26sid%3Dliteratum%253Aachs%26pmid%3D29155580%26genre%3Darticle%26aulast%3DBronner%26date%3D2017%26atitle%3DA%2BUnique%2BApproach%2Bto%2BDesign%2BPotent%2Band%2BSelective%2BCyclic%2BAdenosine%2BMonophosphate%2BResponse%2BElement%2BBinding%2BProtein%252C%2BBinding%2BProtein%2B%2528CBP%2529%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D24%26spage%3D10151%26epage%3D10171%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291051" title="Hybridization">Hybridization</a>,</li><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=291862" title="Indoles">Indoles</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/24" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/jmcmar.2017.60.issue-24/20171228/jmcmar.2017.60.issue-24.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0025.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0025.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The epigenetic regulator CBP/P300 presents a novel therapeutic target for oncology. Previously, we disclosed the development of potent and selective CBP bromodomain inhibitors by first identifying pharmacophores that bind the KAc region and then building into the LPF shelf. Herein, we report the “hybridization” of a variety of KAc-binding fragments with a tetrahydroquinoline scaffold that makes optimal interactions with the LPF shelf, imparting enhanced potency and selectivity to the hybridized ligand. To demonstrate the utility of our hybridization approach, two analogues containing unique Asn binders and the optimized tetrahydroquinoline moiety were rapidly optimized to yield single-digit nanomolar inhibitors of CBP with exquisite selectivity over BRD4(1) and the broader bromodomain family.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03704" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03704" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Epigenetics is an exciting field of research with unharnessed therapeutic potential.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> Applications to human health offered by the ability to modify gene expression include cancer, aging, neuropsychiatric disease, and obesity.<a onclick="showRef(event, 'ref3 ref4 ref5 ref6 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5 ref6 ref7 ref8 ref9">(3-9)</a> Although prospective therapies that result in permanent and potentially inheritable modification of chemical marks on DNA<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> are associated with greater risk,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> epigenetic mechanisms that modify histones and alter rates of transcription represent attractive therapeutic targets.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a></div><div class="NLM_p">Acetylation of lysine residues on histones is a reversible post-translational modification (PTM) that loosens the DNA–histone interaction, thus altering the chromatin structure and allowing for upregulation of transcription.<a onclick="showRef(event, 'ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref15">(13-15)</a> Proteins that recognize PTMs such as acetylated lysine marks are collectively referred to as readers and are involved in the recruitment of additional proteins to form large transcriptional complexes.<a onclick="showRef(event, 'ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18">(16-18)</a> Bromodomains (BrDs) are arguably the most well-studied class of epigenetic readers and have recently received considerable attention as promising therapeutic targets.<a onclick="showRef(event, 'ref19 ref20 ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21 ref22 ref23">(19-23)</a> Cyclic adenosine monophosphate response element binding protein, binding protein (CBP, CREBBP, or CREB-binding protein) and adenoviral E1A binding protein of 300 kDa (P300 or EP300), two closely related multidomain transcription-activating proteins containing a histone acetyltransferase (HAT) as well as a bromodomain, have an important role in histone acetylation.<a onclick="showRef(event, 'ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26">(24-26)</a> The CBP/P300 coactivator family (hereafter together referred to as “CBP”) has been implicated in various cancers including leukemia and prostate cancer.<a onclick="showRef(event, 'ref27 ref28 ref29 ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref29 ref30 ref31 ref32">(27-32)</a> Furthermore, CBP has been found to regulate expression of <i>MYC</i>,<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33, 34)</a> a transcription factor that is upregulated in many human cancers and also has relevance to cancer immunotherapy,<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35, 36)</a> thus contributing toward a rapidly growing interest in finding suitable CBP inhibitors. Despite the therapeutic opportunities offered by CBP, few potent and selective CBP BrD inhibitors have been reported.<a onclick="showRef(event, 'ref37 ref38 ref39 ref40 ref41 ref42 ref43 ref44 ref45 ref46 ref47 ref48 ref49 ref50 ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref37 ref38 ref39 ref40 ref41 ref42 ref43 ref44 ref45 ref46 ref47 ref48 ref49 ref50 ref51 ref52">(37-52)</a></div><div class="NLM_p">Recently, Genentech (GNE) identified CBP bromodomain inhibitor GNE-781 (<b>3</b>, <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>), which has unparalleled potency and selectivity over other bromodomains.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> Screening hit <b>1</b> provided a starting point for the development of the pyrazolopiperidine series of CBP inhibitors, and additional potency and stability optimization led to identification of in vivo tool compound GNE-272 (<b>2</b>).<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> Ultimately, cell potency and selectivity over the bromodomain 1 of bromodomain-containing protein 4 [BRD4(1)] were further improved via modification/rigidification of the aniline, exploration of the pyrazole substituent, and by utilizing a methyl urea in place of the <i>N</i>-acetyl group (<b>2</b> → <b>3</b>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Discovery and optimization of pyrazolopiperidine CBP inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We previously reported a cocrystal structure of CBP BrD-bound <b>3</b> (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>), which is illustrative of the interactions that govern potency and selectivity.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> The interaction between Asn1168 and the urea moiety mimics the endogenous ligands, which are acetylated lysine residues. The selectivity over other bromodomains is attributed in part to the half-chair conformation of the piperidine ring in the tetrahydroquinoline (THQ) and the THQ sharing shape complementarity with the base of the LPF shelf (Leu1109, Pro1110, and Phe1111). In BRD4(1), the WPF shelf contains a more sterically demanding Trp81 compared to the Leu1109 residue of CBP, and this difference provides an opportunity for selectivity.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Additionally, a favorable interaction between the negatively polarized fluorine atoms in the difluoromethyl and the guanidinium group of Arg1173 is observed.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> This interaction is likely to contribute to both CBP potency and improved selectivity over BRD4 because the equivalent residue to Arg1173 is Asp145 and Glu438 in BRD4(1) and BRD4(2), respectively.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Cocrystal structure of CBP BrD-bound <b>3</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W0E">5W0E</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64843" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64843" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">During the course of development of <b>3</b>, we sought to develop a structurally unique second chemical series with comparable potency and selectivity to that of our lead to minimize risk of bringing only one series forward. Intrigued with the possibility of introducing diversity in the Asn-binding pharmacophore, we decided to reexamine the benzodiazepinone core of CBP inhibitor <b>4</b>, which was previously developed during our program (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>).<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> The benzodiazepinone scaffold had a clear vector for the THQ-CF<sub>2</sub>H pyrazole and exchanging the indazole in <b>4</b> with the optimized LPF binder led to “THQ hybrid” <b>5</b>. This modification resulted in an ∼9-fold increase in CBP biochemical potency as measured by the time-resolved fluorescence energy transfer (TR-FRET) assay and an increase in selectivity over all other bromodomains assayed including bromodomain-containing protein 9 (BRD9) and cat eye syndrome critical protein 2 (CECR2). Furthermore, this result demonstrated that attaching the THQ moiety onto an Asn-binding pharmacophore, in this case the benzodiazepinone, is an effective strategy to improve selectivity and potency.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Effect of hybridization of the benzodiazepinone core of <b>4</b> with THQ-CF<sub>2</sub>H pyrazole on CBP potency and selectivity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As part of our screening efforts, we executed a high-throughput screen (HTS) targeting alternative Asn-binding cores. Cores with high ligand efficiency (LE) were selected as possible analogues for hybridization. In each case, the cocrystal structure of the CBP-bound ligand was overlaid with the cocrystal structure of <b>3</b> to evaluate whether the core had an appropriate vector for the THQ functionality. When cocrystal structures were not available, computational docking was utilized to approximate the binding orientation of the HTS hit within the CBP bromodomain.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> Targets were selected based on synthetic ease and whether the THQ fragment was predicted to have a similar spatial orientation in the receptor to that of lead molecule <b>3</b>. For example, a crystal structure of ligand efficient HTS hit <b>6</b> when compared to lead compound <b>3</b> indicated that appendage of the THQ onto C5 of the Asn-binding indole core might lead to a hybrid with the desired geometry (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>a). Thiazolone <b>7</b> was identified as a hit in the primary HTS (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>b). Although this fragment had a CBP IC<sub>50</sub> > 100 μM in the reconfirmation screen, <b>7</b> was nonetheless a core of interest because it was a frequently appearing motif in the HTS and also because similar scaffolds are known inhibitors of the bromodomains of tripartite motif-containing 24 protein (TRIM24) and bromodomain and PHD finger-containing protein (BRPF).<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> Docking of thiazolone <b>7</b> suggested two possible binding orientations and two potential vectors for the THQ moiety. To our delight, THQ hybrids <b>16</b> and <b>17</b> led to increases in potency from 99,000 nM to 9.5 nM in the case of the indole and >100,000 nM to 69 nM with the thiazolone (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>) relative to the parent HTS hit.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (a) Overlay of cocrystal structures of CBP BrD-bound HTS hit <b>6</b> (purple; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6AXQ">6AXQ</a>) and <b>3</b> (cyan; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W0E">5W0E</a>). (b) Docked structures of HTS hit <b>7</b> (orange and pink) overlaid with cocrystal structure of CBP BrD-bound <b>3</b> (cyan; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W0E">5W0E</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Hybridization of HTS hits and known CBP inhibitors with THQ-LPF binder.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a similar fashion, HTS hits <b>8</b>–<b>13</b> were combined with the THQ moiety to create hybrids containing their original Asn-binding cores (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>). In some cases, the Asn-binding moiety present in the core was slightly modified to provide improved potency utilizing our prior structure–activity relationship (SAR) learnings.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> The resulting THQ hybrids <b>18</b>–<b>25</b> demonstrated varying magnitudes of potency and selectivity increases but nonetheless led to improved CBP inhibitors. Hybridization even rescued scaffolds that originally displayed selectivity for BRD4(1); hit <b>10</b>, which showed a 4.4-fold preference for BRD4(1) inhibition, was transformed into a 530-fold selective CBP inhibitor via hybridization. Interestingly, the pharmacophore of <b>23</b>, although derived from HTS hit <b>13</b>, also closely resembled that of early weak CBP inhibitor MS2126 reported by Zhou.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> The same design exercise was performed with published CBP inhibitors with an equally fruitful outcome; hybridization of <b>14</b><a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> increased the CBP potency by 9.5-fold, and modification of <b>15</b><a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> resulted in a 34-fold improvement.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a></div><div class="NLM_p">X-ray cocrystal structures of CBP-bound hybrids <b>16</b> and <b>17</b> were obtained and are depicted in <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a> overlaid with lead <b>3</b>. The comparison of the crystal structures with different Asn-binding motifs indicates that the binding pocket of the CBP bromodomain can accommodate some flexibility in the location of the Asn-binding core, but the THQ-CF<sub>2</sub>H pyrazole substituents have very similar orientations (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>). This suggests that complementarity between THQ and the subpocket on CBP enhances both activity and selectivity in most cases. This structural comparison illustrates the crucial effect of the LPF shelf on selectivity and potency of bromodomain inhibitors and exemplifies the reduced importance of the KAc binding region. Although the latter is necessary and is the anchor of the inhibitor, this binding region only procures modest selectivity and potency.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a></div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Overlay of cocrystal structures of CBP BrD-bound hybrids <b>16</b> (magenta; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6AY3">6AY3</a>), <b>17</b> (green; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6AY5">6AY5</a>), and <b>3</b> (cyan; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W0E">5W0E</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The hybridization studies described above provided a good starting point for rapid development of structurally unique and potent CBP inhibitors. Hybridization of the indole and thiazolone cores (<b>6</b> → <b>16</b> and <b>7</b> → <b>17</b>) resulted in two of the most dramatic increases in CBP biochemical potency (10,400- and >1450-fold, respectively) while also imparting exquisite selectivity over BRD4(1) (1300- and 260-fold, respectively). It was anticipated that additional SAR could easily render inhibitors with similar potency to that of our lead molecule <b>3</b>. X-ray cocrystal structures also suggested that both hybrids had available vectors to grow into the loop connecting helices B and C (BC loop), which previously had been found to be a productive technique to improve potency (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>a/b).<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> This further supported the decision to pursue the indole and thiazolone series.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (a) Overlay of cocrystal structures of CBP BrD-bound thiazolone hybrid <b>17</b> (pink; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6AY5">6AY5</a>) and <b>3</b> (cyan; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W0E">5W0E</a>). (b) Overlay of cocrystal structures of CBP BrD-bound indole hybrid <b>16</b> (magenta; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6AY3">6AY3</a>) and <b>3</b> (cyan; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W0E">5W0E</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Improvement of potency and metabolic stability were the primary goals of optimization of the thiazolone series; clearance of <b>17</b> as measured by mouse, rat, and human liver microsome-predicted hepatic clearance assays (LM Cl<sub>hep</sub> M/R/H) was undesirably high at 71, 39, and 17 mL min<sup>–1</sup> kg<sup>–1</sup>, respectively (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>). Initially, SAR of the Asn binder was examined using de novo design and ideas inspired by frequently appearing scaffolds in the HTS that had previously been overlooked because of lower LE. Although other pharmacophores did not yield enhanced affinity, introduction of the benzimidazolone Asn binder resulted in a CBP IC<sub>50</sub> of 20 nM (<b>26</b>, <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>). Expansion into the BC loop with morpholine <b>27</b> provided a good balance of biochemical potency and selectivity over BRD4(1) while imparting modest in vitro stability gains. Furthermore, <b>27</b> had excellent potency in a cellular assay of MYC expresssion<a onclick="showRef(event, 'ref48 ref60'); return false;" href="javascript:void(0);" class="ref ref48 ref60">(48, 60)</a> with an EC<sub>50</sub> of 12 nM.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Optimization of the thiazolone–THQ hybrid.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Efforts to optimize the indole series also centered on improvement of potency and metabolic stability. Indole hybrid <b>16</b> had moderate in vitro clearance (LM Cl<sub>hep</sub> M/R/H = 47/20/13 mL/min/kg), but efforts to improve potency by growing into the BC loop by substitution at C7 of the indole resulted in decreased metabolic stability. To address these limitations, a nitrogen walk around the indole skeleton was performed leading to the identification of azaindole <b>28</b> that demonstrated improved CBP potency and metabolic stability (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>). Azaindole <b>28</b> proved to be a suitable starting point for expansion into the critical BC loop, and isopropyl substitution at C7 provided a good balance of biochemical potency (CBP IC<sub>50</sub> = 2.2 nM) and in vitro clearance (LM Cl<sub>hep</sub> M/R/H = 50/25/13 mL min<sup>–1</sup> kg<sup>–1</sup>). Furthermore, <b>29</b> displayed an impressive improvement in potency in the MYC assay relative to hybrids <b>28</b> and <b>16</b> (EC<sub>50</sub> = 53, 360, and 2200 nM, respectively). Compound <b>29</b> was tested in an in vivo pharmacokinetics (PK) experiment, exhibiting low clearance (18.2 mL min<sup>–1</sup> kg<sup>–1</sup>) and moderate bioavailability (52%) when dosed to mice.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Optimization of the indole–THQ hybrid.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To probe the broader bromodomain selectivity of optimized THQ hybrids, <b>27</b> and <b>29</b> were submitted to the DiscoveRx BROMOscan selectivity panel, which assesses binding in a competitive fashion by quantitating the reduction in capture of DNA-tagged bromodomain on ligand-coated beads in the presence of compound.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> Benzimidazolone <b>27</b> proved to have exquisite selectivity over other bromodomains. Out of 39 bromodomain-containing constructs surveyed, only four had any activity other than CBP and P300 with BRD9 being the most potent off target (<i>K</i><sub>d</sub> = 3600 nM). Other off target hits were BRD7, TRIM24(BrD), and BRD3(2) with potencies of <i>K</i><sub>d</sub> = 5100, 7600, and 8700 nM, respectively. Azaindole <b>29</b> also displayed exceptional selectivity in the BROMOscan selectivity panel, and no activity was detected in any potential bromodomain off-targets (<i>K</i><sub>d</sub> > 10,000 nM).</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32528" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32528" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The target compounds were synthesized as depicted in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a>–<a class="ref internalNav" href="#sch15" aria-label="15">15</a>. In all cases, aryl bromide coupling partners were obtained from either commercial sources or accessed through diverse synthetic sequences of varying complexity. Buchwald−Hartwig amination of intermediate <b>31</b> was used to install the requisite tetrahydroquinoline, and conditions utilizing Pd-PEPPSI-IPent catalyst were found to be suitable for accessing most Buchwald−Hartwig adducts.</div><figure id="sch1" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Preparation of Compound <b>5</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch2" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Preparation of Compound <b>16</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch3" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Preparation of Compound <b>17</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch4" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Preparation of Compound <b>18</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch5" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0014.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Preparation of Compound <b>19</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch6" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0015.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Preparation of Compound <b>20</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch7" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0016.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Preparation of Compound <b>21</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch8" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0017.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Preparation of Compound <b>22</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch9" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0018.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 9. Preparation of Compound <b>23</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch10" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Scheme 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0019.jpeg" id="gr19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 10. Preparation of Compound <b>24</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch11" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Scheme 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0020.jpeg" id="gr20" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 11. Preparation of Compound <b>25</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch12" data-index="21" class="article__inlineFigure"><h2 class="fig-label">Scheme 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0021.jpeg" id="gr21" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 12. Preparation of Compound <b>26</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch13" data-index="22" class="article__inlineFigure"><h2 class="fig-label">Scheme 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0022.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0022.jpeg" id="gr22" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 13. Preparation of Compound <b>27</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch14" data-index="23" class="article__inlineFigure"><h2 class="fig-label">Scheme 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0023.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0023.jpeg" id="gr23" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 14. Preparation of Compound <b>28</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch15" data-index="24" class="article__inlineFigure"><h2 class="fig-label">Scheme 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0024.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0024.jpeg" id="gr24" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 15. Preparation of Compound <b>29</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0024.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch15"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70108" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70108" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">This work demonstrates that a variety of unique Asn binders can be hybridized with the THQ<b>-</b>CF<sub>2</sub>H pyrazole LPF fragment to create potent, selective, and structurally diverse CBP probes. Structure-based drug design (SBDD) provided a robust foundation for rapid optimization of novel cores in the hybridization process as evidenced by the development of the benzimidazolone−THQ and indole−THQ series; 43 hybrids were synthesized in the case of the thiazolone/imidazolone series, and optimization of the indole series required the synthesis of 29 analogues. Generally, in SBDD larger structural modifications are associated with greater risk of potency loss. The novelty in this work is that half of the inhibitor was replaced with a completely new scaffold while potency was maintained; furthermore, the fragment necessary for ligand recognition, the Asn binder, underwent significant alteration. Although this was a highly structurally enabled project, in many cases crystal structures were not necessary and hybrids could be effectively designed simply with computationally docked structures. Rarely is SBDD executed in such a predictable manner.</div><div class="NLM_p last">Whereas previously disclosed bromodomain inhibitors were developed by first identifying an Asn-binding starting point and then building into the LPF shelf, our THQ hybridization strategy provides a complementary and unique strategy to efficiently perform hit-to-lead optimization. Although an Asn-binding element was still necessary to maintain activity, these findings illustrate that both CBP bromodomain potency and selectivity over other bromodomains can be controlled via favorable interactions with the LPF shelf and adjacent residues. Optimization of the LPF shelf/ligand interaction provides a robust potency and selectivity handle for the development of CBP inhibitors that can be used to probe the biology of this epigenetic target. Furthermore, we anticipate that this novel hybridization strategy will have a broader application for the discovery of inhibitors of the medicinally relevant bromodomain family beyond CBP and P300.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Experimental Procedures</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37485" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37485" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5-1" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> General Methods</h3><div class="NLM_p last">All solvents and reagents were used as obtained. NMR analysis was performed in a deuterated solvent with a Varian Avance 300 MHz or Bruker Avance 400 or 500 MHz NMR spectrometers referenced to trimethylsilane (TMS). Chemical shifts are expressed as δ units using TMS as the external standard (in NMR description, s = singlet, d = doublet, t = triplet, q = quartet, m = multiplet, and br = broad peak). All coupling constants (<i>J</i>) are reported in hertz. Mass spectra were measured with a Finnigan SSQ710C spectrometer using an ESI source coupled to a Waters 600MS high-performance liquid chromatography (HPLC) system operating in reverse-phase mode with an X-bridge Phenyl column of dimensions 150 mm × 2.6 mm with 5 μm sized particles. Preparatory-scale silica gel chromatography was performed using medium-pressure liquid chromatography (MPLC) on a CombiFlash Companion (Teledyne ISCO) with RediSep normal phase silica gel (35–60 μm) columns and UV detection at 254 nm. Reverse-phase high-performance liquid chromatography (HPLC) was used to purify compounds as needed by elution from a Phenomenex Gemini-NX C18 column (30 × 50 mm, 5 μm) as stationary phase using mobile phase indicated and operated at a 60 mL/min flow rate on an Interchim PuriFlash SFC UV-directed prep instrument. Supercritical fluid (SFC) chromatography was used to purify compounds as needed by elution from a Phenomenex Gemini-NX C18 column (30 × 50 mm, 5 μm) as stationary phase using mobile phase indicated and operated at a 100 mL/min flow rate on a Thar 100 SFC UV-directed prep instrument. Chemical purities were >95% for all final compounds as assessed by LC/MS analysis. The following analytical method was used to determine chemical purity of final compounds: HPLC-Agilent 1200, water with 0.05% TFA, acetonitrile with 0.05% TFA (buffer B), Agilent SB-C18, 1.8 μM, 2.1 × 30 mm, 25 °C, 3–95% buffer B in 8.5 min, 95% in 2.5 min, 400 μL/min, 220 and 254 nm, equipped with Agilent quadrupole 6140, ESI positive, 90–1300 amu.</div></div><div id="sec5-2" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Synthesis and Characterization of (4<i>R</i>)-6-[7-(Difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]-4-methyl-1,3,4,5-tetrahydro-1,5-benzodiazepin-2-one (<b>5</b>)</h3><div class="NLM_p last">(<i>R</i>)-6-Bromo-4-methyl-4,5-dihydro-1<i>H</i>-benzo[<i>b</i>][1,4]diazepin-2(3<i>H</i>)-one (<b>30</b>)<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> and 1-methyl-7-(difluoromethyl)-6-(1-methylpyrazol-4-yl)-1,2,3,4-tetrahydroquinoline (<b>31</b>)<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> were synthesized as previously described. To a solution of (<i>R</i>)-6-bromo-4-methyl-4,5-dihydro-1<i>H</i>-benzo[<i>b</i>][1,4]diazepin-2(3<i>H</i>)-one (<b>30</b>, 100 mg, 0.392 mmol, 1.0 equiv) in 1,4-dioxane (2 mL) was added 7-(difluoromethyl)-6-(1-methylpyrazol-4-yl)-1,2,3,4-tetrahydroquinoline (<b>31,</b> 125 mg, 0.475 mmol, 1.2 equiv), <i>t-</i>BuONa (190 mg, 1.98 mmol, 4.2 equiv), and dichloro[1,3-bis(2,6-di-3-pentylphenyl)imidazol-2-ylidene](3-chloropyridyl)palladium(II) (32 mg, 0.040 mmol, 10 mol %). The mixture was heated to 120 °C for 16 h under a nitrogen atmosphere. After cooling to room temperature, the mixture was filtered, and the filtrate was concentrated in vacuo. The crude residue was diluted with water (10 mL), and the mixture was extracted with EtOAc (10 mL × 3). The combined organic layers were washed with sat. aq NaHCO<sub>3</sub> (10 mL × 3), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuo. The crude residue was purified by reverse-phase chromatography (acetonitrile 10–40%/0.2% formic acid in water) to give (4<i>R</i>)-6-[7-(difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]-4-methyl-1,3,4,5-tetrahydro-1,5-benzodiazepin-2-one (<b>5</b>, 6 mg, 3%) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.71 (s, 1H), 7.53 (s, 1H), 7.40 (s, 1H), 7.07 (s, 1H), 6.98–6.95 (m, 1H), 6.87–6.82 (m, 2H), 6.54 (s, 1H), 6.46 (t, <i>J</i> = 55.6 Hz, 1H), 4.38–4.30 (m, 1H), 4.00–3.94 (m, 4H), 3.59–3.56 (m, 1H), 3.46–3.36 (m, 1H), 3.05–2.97 (m, 1H), 2.95–2.85 (m, 1H), 2.79–2.60 (m, 1H), 2.58–2.44 (m, 1H), 2.15–2.08 (m, 2H), 1.27–1.17 (m, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 438.</div></div><div id="sec5-3" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Synthesis and Characterization of 5-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-<i>N</i>-methyl-1<i>H</i>-indole-3-carboxamide (<b>16</b>)</h3><div id="sec5-3-1" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Step 1, Methyl 5-Bromo-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-indole-3-carboxylate (<b>33</b>)</h4><div class="NLM_p last">To a solution of methyl 5-bromo-1<i>H</i>-indole-3-carboxylate (<b>32</b>, 800 mg, 3.15 mmol, 1.0 equiv) in DMF (3 mL) at 0 °C was added NaH (60% dispersion in mineral oil, 139 mg, 3.48 mmol, 1.1 equiv) portionwise. The mixture was stirred at room temperature for 0.5 h. (2-(Chloromethoxy)ethyl)trimethylsilane (0.61 mL, 3.47 mmol, 1.1 equiv) was added dropwise and the mixture stirred at room temperature for an additional 3 h. The mixture was quenched with water (20 mL) and extracted with EtOAc (20 mL × 3). The combined organic layers were washed with brine (20 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude residue was purified by silica gel chromatography (petroleum ether/EtOAc = 10:1) to give methyl 5-bromo-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-indole-3-carboxylate (<b>33</b>, 800 mg, 66%) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.33 (s, 1H), 7.87 (s, 1H), 7.43–7.38 (m, 2H), 5.48 (s, 2H), 3.94 (s, 3H), 3.47 (t, <i>J</i> = 8.0 Hz, 2H), 0.89 (t, <i>J</i> = 8.0 Hz, 2H), −0.03 (s, 9H).</div></div><div id="sec5-3-2" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Step 2, 5-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-indole-3-carboxylic Acid (<b>34</b>)</h4><div class="NLM_p last">To a solution of methyl 5-bromo-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-indole-3-carboxylate (<b>33</b>, 200 mg, 0.520 mmol, 1.0 equiv) and 7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-1,2,3,4-tetrahydroquinoline (<b>31</b>, 164 mg, 0.623 mmol, 1.2 equiv) in 1,4-dioxane (4 mL) were added chloro(2-dicyclohexylphosphino-2′,6′-di-<i>i</i>-propoxy-1,1′-biphenyl)(2′-amino-1,1′-biphenyl-2-yl)palladium(II) (39 mg, 0.050 mmol, 10 mol %), 2-dicyclohexylphosphino-2′,6′-di-<i>i</i>-propoxy-1,1′-biphenyl (23 mg, 0.050 mmol, 10 mol %), and <i>t</i>-BuONa (150 mg, 1.56 mmol, 3 equiv). The mixture was heated to 120 °C for 12 h under a nitrogen atmosphere. After cooling the reaction to room temperature, DCM (40 mL) was added, and the mixture was washed with water (30 mL × 2). The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude residue was purified by Prep-TLC (DCM/MeOH = 10:1) to give 5-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-indole-3-carboxylic acid (<b>34</b>, 100 mg, 35%) as a white solid. LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 553.</div></div><div id="sec5-3-3" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Step 3, 5-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-<i>N</i>-methyl-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-indole-3-carboxamide (<b>35</b>)</h4><div class="NLM_p last">To a solution of 5-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-indole-3-carboxylic acid (<b>34</b>, 140 mg, 0.247 mmol, 1.0 equiv) and <i>N,N</i>-diisopropylethylamine (0.13 mL, 0.76 mmol, 3.0 equiv) in DMF (3 mL) were added <i>O</i>-(7-azabenzotriazol-1-yl)-<i>N,N,N′,N′</i>-tetramethyluronium hexafluorophosphate (116 mg, 0.305 mmol, 1.2 equiv) and methylamine hydrochloride (21 mg, 0.31 mmol, 1.2 equiv). The reaction was stirred at room temperature for 2 h. EtOAc (50 mL) was added, and the mixture was washed with water (50 mL × 3) and brine (50 mL). The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to give 5-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-<i>N</i>-methyl-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-indole-3-carboxamide (<b>35</b>, 135 mg, 94%) as a white solid that required no further purification. LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 566.</div></div><div id="sec5-3-4" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Step 4, 5-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-<i>N</i>-methyl-1<i>H</i>-indole-3-carboxamide (<b>16</b>)</h4><div class="NLM_p last">To a solution of 5-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-<i>N</i>-methyl-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-indole-3-carboxamide (<b>35</b>, 135 mg, 0.24 mmol) in THF (2 mL) was added TBAF (4.0 mL, 4.0 mmol, 1 M in THF, 17 equiv). The mixture was heated to 50 °C for 40 h under a nitrogen atmosphere. After cooling to room temperature, the mixture was filtered and concentrated in vacuo. EtOAc (40 mL) was added, and the mixture was washed with water (30 mL × 2) and brine (30 mL). The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude residue was purified by reverse-phase chromatography (acetonitrile 28–58%/0.05% NH<sub>4</sub>OH in water) to give 5-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-<i>N</i>-methyl-1<i>H</i>-indole-3-carboxamide (<b>16</b>, 28 mg, 26%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.61 (s, 1H), 8.02 (d, <i>J</i> = 1.2 Hz, 1H), 8.00 (d, <i>J</i> = 2.8 Hz, 1H), 7.87 (d, <i>J</i> = 4.4 Hz, 1H), 7.72 (s, 1H), 7.50 (d, <i>J</i> = 8.4 Hz, 1H), 7.47 (s, 1H), 7.07 (s, 1H), 7.05–7.03 (m, 1H), 6.68 (t, <i>J</i> = 55.2 Hz, 1H), 6.49 (s, 1H), 3.85 (s, 3H), 3.65–3.55 (m, 2H), 2.91–2.86 (m, 2H), 2.75 (d, <i>J</i> = 4.4 Hz, 3H), 2.11–1.98 (m, 2H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 436.</div></div></div><div id="sec5-4" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Synthesis and Characterization of 5-[7-(Difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]-3-methyl-1,3-benzothiazol-2-one (<b>17</b>)</h3><div id="sec5-4-1" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Step 1, 5-Bromo-3<i>H</i>-1,3-benzothiazol-2-one (<b>37</b>)</h4><div class="NLM_p last">To a solution of 2-amino-4-bromobenzenethiol (<b>36</b>, 544 mg, 2.67 mmol, 1.00 equiv) in acetic acid (26.7 mL) was added triphosgene (530 mg, 1.79 mmol, 0.67 equiv). The mixture was heated at reflux for 18 h. After cooling to room temperature, the solution was partially concentrated under reduced pressure; water was added, and the resulting precipitate was removed via filtration and washed with aqueous NaOH (1.0 M). The filtrate was acidified with aq HCl (2 N) to pH 2 and placed in a refrigerator for 12 h. The resulting precipitate was filtered, washed with water, and dried under reduced pressure to give 5-bromo-3<i>H</i>-1,3-benzothiazol-2-one (<b>37</b>, 94 mg, 15%) as a while powder that required no further purification. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.07 (s, 1H), 7.55 (d, <i>J</i> = 8.4 Hz, 1H), 7.30 (d, <i>J</i> = 8.0 Hz, 1H), 7.24 (s, 1H).</div></div><div id="sec5-4-2" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> Step 2, 5-Bromo-3-methyl-1,3-benzothiazol-2-one (<b>38</b>)</h4><div class="NLM_p last">To a solution of 5-bromo-3<i>H</i>-1,3-benzothiazol-2-one (<b>37</b>, 94.4 mg, 0.410 mmol, 1.0 equiv) in water (2.1 mL) was added NaOH (16.4 mg, 0.410 mmol, 1.0 equiv) and then dimethyl sulfate (62.1 mg, 0.492 mmol, 1.2 equiv). After stirring at room temperature for 3 h, the resulting precipitate was filtered, washed with water, and dried under reduced pressure to give 5-bromo-3-methyl-1,3-benzothiazol-2-one (<b>38</b>, 89.3 mg, 89%) as a white powder that required no further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.32–7.29 (m, 2H), 7.20 (dd, <i>J</i> = 1.5, 0.6 Hz, 1H), 3.45 (s, 3H).</div></div><div id="sec5-4-3" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Step 3, 5-[7-(Difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]-3-methyl-1,3-benzothiazol-2-one (<b>17</b>)</h4><div class="NLM_p last">To a vial were added 7-(difluoromethyl)-6-(1-methylpyrazol-4-yl)-1,2,3,4-tetrahydroquinoline (<b>31</b>, 25.0 mg, 0.0950 mmol, 1.0 equiv), 5-bromo-3-methyl-1,3-benzothiazol-2-one (<b>38</b>, 27.8 mg, 0.114 mmol, 1.2 equiv), dichloro[1,3-bis(2,6-di-3-pentylphenyl)imidazol-2-ylidene](3-chloropyridyl)palladium(II) (11.9 mg, 0.0142 mmol, 15 mol %), <i>t-</i>BuONa (18.2 mg, 0.190 mmol, 2.0 equiv), and 1,4-dioxane (0.20 mL). The mixture was sparged with an argon balloon and then heated to 120 °C for 16 h under an argon atmosphere. After cooling the reaction to room temperature, DCM (3 mL) was added, and the reaction mixture was filtered through Celite and concentrated under reduced pressure. The crude residue was purified by reverse-phase preparative HPLC (acetonitrile 30–70%/0.1% ammonium hydroxide in water) to give 5-[7-(difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]-3-methyl-1,3-benzothiazol-2-one (<b>17</b>, 9.6 mg, 24%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.76 (d, <i>J</i> = 0.8 Hz, 1H), 7.66 (d, <i>J</i> = 8.4 Hz, 1H), 7.51 (d, <i>J</i> = 0.8 Hz, 1H), 7.26 (d, <i>J</i> = 2.1 Hz, 1H), 7.18–7.14 (m, 1H), 7.11 (dd, <i>J</i> = 8.4, 2.1 Hz, 1H), 6.87–6.77 (m, 2H), 3.87 (s, 3H), 3.70–3.63 (m, 2H), 3.38 (s, 3H), 2.86 (t, <i>J</i> = 6.4 Hz, 2H), 2.06–1.96 (m, 2H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 427.</div></div></div><div id="sec5-5" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Synthesis and Characterization of 4-[7-(Difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]-<i>N</i>-methyl-quinoline-6-carboxamide (<b>18</b>)</h3><div id="sec5-5-1" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Step 1, Methyl 4-[7-(Difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]quinoline-6-carboxylate (<b>40</b>)</h4><div class="NLM_p last">To a vial were added 7-(difluoromethyl)-6-(1-methylpyrazol-4-yl)-1,2,3,4-tetrahydroquinoline (<b>31</b>, 200 mg, 0.760 mmol, 1.0 equiv), methyl 4-bromoquinoline-6-carboxylate (<b>39</b>, 243 mg, 0.912 mmol, 1.2 equiv), dichloro[1,3-bis(2,6-di-3-pentylphenyl)imidazol-2-ylidene](3-chloropyridyl)palladium(II) (95.2 mg, 0.114 mmol, 15 mol %), <i>t-</i>BuONa (146 mg, 1.52 mmol, 2.0 equiv), and 1,4-dioxane (1.52 mL). The mixture was sparged with an argon balloon and then heated to 120 °C for 16 h under an argon atmosphere. After cooling the reaction to room temperature, DCM (4 mL) was added, and the reaction mixture was filtered through Celite and concentrated under reduced pressure. The residue was purified by silica gel chromatography (1% TEA in heptanes to 100% EtOAc gradient) to afford methyl 4-[7-(difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]quinoline-6-carboxylate (<b>40</b>, 23.6 mg, 7%) as a yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.92 (d, <i>J</i> = 4.9 Hz, 1H), 8.76 (dd, <i>J</i> = 2.0, 0.6 Hz, 1H), 8.31 (dd, <i>J</i> = 8.8, 1.9 Hz, 1H), 8.18 (dd, <i>J</i> = 8.8, 0.6 Hz, 1H), 7.57 (d, <i>J</i> = 0.8 Hz, 1H), 7.45 (d, <i>J</i> = 0.7 Hz, 1H), 7.31–7.27 (m, 1H), 7.18 (t, <i>J</i> = 1.1 Hz, 1H), 6.88 (s, 1H), 6.58–6.30 (m, 1H), 3.96 (s, 3H), 3.96 (s, 3H), 3.80 (dd, <i>J</i> = 6.7, 4.2 Hz, 2H), 3.03 (t, <i>J</i> = 6.6 Hz, 2H), 2.18–2.08 (m, 2H).</div></div><div id="sec5-5-2" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> Step 2, 4-[7-(Difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]-<i>N</i>-methyl-quinoline-6-carboxamide (<b>18</b>)</h4><div class="NLM_p last">To a solution of methyl 4-[7-(difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]quinoline-6-carboxylate (<b>40</b>, 23.4 mg, 0.0522 mmol, 1.0 equiv) in THF (0.5 mL) and water (0.5 mL) was added lithium hydroxide monohydrate (2.6 mg, 0.063 mmol, 1.2 equiv). The mixture was stirred at the room temperature for 1 h. After completion of the reaction, the reaction mixture was concentrated under reduced pressure. The resulting residue was dissolved in DMF (1.0 mL) and TEA (21.1 mg, 0.209 mmol, 4.0 equiv), methylamine hydrochloride (10.6 mg, 0.157 mmol, 3.0 equiv), and TBTU (34.2 mg, 0.104 mmol, 2.0 equiv) were added. The mixture was stirred at room temperature for 14 h and then concentrated under reduced pressure. The crude residue was purified by reverse-phase preparative HPLC (acetonitrile 5–50%/0.1% formic acid in water) to give 4-[7-(difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]-<i>N</i>-methyl-quinoline-6-carboxamide (<b>18</b>, 13.7 mg, 57%, 2 steps) as a yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.96 (d, <i>J</i> = 4.8 Hz, 1H), 8.69 (q, <i>J</i> = 4.5 Hz, 1H), 8.46 (dd, <i>J</i> = 2.0, 0.6 Hz, 1H), 8.19 (dd, <i>J</i> = 8.8, 1.9 Hz, 1H), 8.13 (dd, <i>J</i> = 8.8, 0.6 Hz, 1H), 7.78 (d, <i>J</i> = 0.8 Hz, 1H), 7.53 (d, <i>J</i> = 0.8 Hz, 1H), 7.45 (d, <i>J</i> = 4.8 Hz, 1H), 7.26 (s, 1H), 6.87–6.50 (m, 2H), 3.87 (s, 3H), 3.73 (t, <i>J</i> = 5.4 Hz, 2H), 3.00 (t, <i>J</i> = 6.5 Hz, 2H), 2.80 (d, <i>J</i> = 4.5 Hz, 3H), 2.19–1.90 (m, 2H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 448.</div></div></div><div id="sec5-6" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Synthesis and Characterization of 6-Acetyl-3-[7-(difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]-1-methyl-7,8-dihydro-5<i>H</i>-1,6-naphthyridin-2-one (<b>19</b>)</h3><div id="sec5-6-1" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> Step 1, <i>tert</i>-Butyl 3-Bromo-2-oxo-1,5,7,8-tetrahydro-1,6-naphthyridine-6-carboxylate (<b>42</b>)</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl 2-oxo-1,5,7,8-tetrahydro-1,6-naphthyridine-6-carboxylate (<b>41</b>, 275 mg, 1.10 mmol, 1.0 equiv) in acetic acid (2.2 mL) at 0 °C was added bromine (67.7 μL, 1.32 mmol, 1.2 equiv) portionwise. The mixture was stirred at room temperature for 3 h, then concentrated under reduced pressure. The resulting residue was dissolved in chloroform (4 mL) and water (2.0 mL). To the solution were added Boc<sub>2</sub>O (312 mg, 1.43 mmol, 1.3 equiv) and K<sub>2</sub>CO<sub>3</sub> (304 mg, 2.20 mmol, 2.0 equiv). The mixture was stirred for 14 h at room temperature. The resulting precipitate was filtered, washed with diethyl ether, and dried under reduced pressure to give the first batch of <b>42</b>. The mother liquor was poured into a separatory funnel; the organic layer was separated, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The resulting solid was washed with diethyl ether and then dried under reduced pressure to give the second batch of <i>tert</i>-butyl 3-bromo-2-oxo-1,5,7,8-tetrahydro-1,6-naphthyridine-6-carboxylate (<b>42</b>, 228 mg total, 63%) that required no further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 16/17 H): δ 7.59 (s, 1H), 4.32 (s, 2H), 3.67 (t, <i>J</i> = 5.8 Hz, 2H), 2.74 (t, <i>J</i> = 5.8 Hz, 2H), 1.48 (s, 9H).</div></div><div id="sec5-6-2" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> Step 2, <i>tert</i>-Butyl 3-Bromo-1-methyl-2-oxo-7,8-dihydro-5<i>H</i>-1,6-naphthyridine-6-carboxylate (<b>43</b>)</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl 3-bromo-2-oxo-1,5,7,8-tetrahydro-1,6-naphthyridine-6-carboxylate (<b>42</b>, 124 mg, 0.378 mmol, 1.0 equiv) in DMF (1.5 mL) at 0 °C was added a solution of LiHMDS in hexane (1.0 M, 0.567 mL, 0.567 mmol, 1.5 equiv). After stirring for 15 min, iodomethane (47.1 μL, 0.756 mmol, 2.0 equiv) was added. After stirring for an additional 30 min, the reaction mixture was concentrated under reduced pressure. The residue was purified by silica gel chromatography (100% heptanes to 100% EtOAc gradient) to afford <i>tert</i>-butyl 3-bromo-1-methyl-2-oxo-7,8-dihydro-5<i>H</i>-1,6-naphthyridine-6-carboxylate (<b>43</b>, 87.5 mg, 68%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.50 (s, 1H), 4.31 (t, <i>J</i> = 1.6 Hz, 2H), 3.69 (t, <i>J</i> = 5.8 Hz, 2H), 3.57 (s, 3H), 2.69 (t, <i>J</i> = 1.7 Hz, 2H), 1.49 (s, 9H).</div></div><div id="sec5-6-3" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> Step 3, 6-Acetyl-3-bromo-1-methyl-7,8-dihydro-5<i>H</i>-1,6-naphthyridin-2-one (<b>44</b>)</h4><div class="NLM_p last">A mixture of <i>tert</i>-butyl 3-bromo-1-methyl-2-oxo-7,8-dihydro-5<i>H</i>-1,6-naphthyridine-6-carboxylate (<b>43</b>, 60.0 mg, 0.175 mmol, 1.0 equiv) and trifluoroacetic acid (0.5 mL) in DCM (4 mL) was stirred at room temperature for 2 h. The solvent was concentrated under reduced pressure, and the crude product was redissolved in DCM (4 mL). The solution was cooled to 0 °C before TEA (0.12 mL, 0.87 mmol, 5.0 equiv) and acetic anhydride (33 μL, 0.35 mmol, 2.0 equiv) were added dropwise. The reaction mixture was stirred at room temperature for an additional 3 h before being quenched with water. The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude residue was purified by silica gel chromatography (100% heptanes to 10% MeOH in EtOAc gradient) to afford 6-acetyl-3-bromo-1-methyl-7,8-dihydro-5<i>H</i>-1,6-naphthyridin-2-one (<b>44</b>, 39.8 mg, 80%) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.52 (d, <i>J</i> = 4.6 Hz, 1H), 4.53–4.32 (m, 2H), 3.88–3.74 (m, 2H), 3.57 (d, <i>J</i> = 1.3 Hz, 3H), 2.82–2.64 (m, 2H), 2.17 (d, <i>J</i> = 9.4 Hz, 3H).</div></div><div id="sec5-6-4" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> Step 4, 6-Acetyl-3-[7-(difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]-1-methyl-7,8-dihydro-5<i>H</i>-1,6-naphthyridin-2-one (<b>19</b>)</h4><div class="NLM_p last">To a vial were added 6-acetyl-3-[7-(difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]-1-methyl-7,8-dihydro-5<i>H</i>-1,6-naphthyridin-2-one (<b>44</b>, 12 mg, 0.042 mmol, 1.0 equiv), 7-(difluoromethyl)-6-(1-methylpyrazol-4-yl)-1,2,3,4-tetrahydroquinoline (<b>31</b>, 12 mg, 0.046 mmol, 1.1 equiv), dichloro[1,3-bis(2,6-di-3-pentylphenyl)imidazol-2-ylidene](3-chloropyridyl)palladium(II) (5.3 mg, 0.0063 mmol, 15 mol %), <i>t-</i>BuONa (8.1 mg, 0.084 mmol, 2.0 equiv), and 1,4-dioxane (0.2 mL). The mixture was sparged with an argon balloon and then heated to 120 °C for 16 h under an argon atmosphere. After cooling the reaction to room temperature, DCM (2 mL) was added, and the reaction mixture was filtered through Celite and concentrated under reduced pressure. The crude residue was purified by reverse-phase preparative HPLC (acetonitrile 5–50%/0.1% NH<sub>4</sub>OH in water) to afford 6-acetyl-3-[7-(difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]-1-methyl-7,8-dihydro-5<i>H</i>-1,6-naphthyridin-2-one (<b>19</b>, 4.2 mg, 18%) as a light yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.72 (s, 1H), 7.47 (t, <i>J</i> = 1.1 Hz, 1H), 7.40–7.36 (m, 1H), 7.05 (s, 1H), 6.89–6.57 (m, 1H), 6.42–6.39 (m, 1H), 4.42–4.39 (m, 2H), 3.86 (s, 3H), 3.73 (t, <i>J</i> = 5.9 Hz, 2H), 3.48–3.41 (m, 5H), 2.90–2.79 (m, 4H), 2.10–2.06 (m, 3H), 1.97 (t, <i>J</i> = 6.0 Hz, 2H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 468.</div></div></div><div id="sec5-7" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Synthesis and Characterization of 7-[7-(Difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]-1-methyl-3,4-dihydroquinolin-2-one (<b>20</b>)</h3><div class="NLM_p last">To a vial were added 7-(difluoromethyl)-6-(1-methylpyrazol-4-yl)-1,2,3,4-tetrahydroquinoline (<b>31</b>, 50 mg, 0.19 mmol, 1.0 equiv), 7-bromo-1-methyl-3,4-dihydroquinolin-2(1<i>H</i>)-one (<b>45</b>, 55 mg, 0.23 mmol, 1.2 equiv), dichloro[1,3-bis(2,6-di-3-pentylphenyl)imidazol-2-ylidene](3-chloropyridyl)palladium(II) (24 mg, 0.028 mmol, 15 mol %), <i>t-</i>BuONa (37 mg, 0.38 mmol, 2.0 equiv), and 1,4-dioxane (0.4 mL). The mixture was sparged with an argon balloon and then heated to 120 °C for 16 h under an argon atmosphere. After cooling the reaction to room temperature, DCM (2 mL) was added, and the reaction mixture was filtered through Celite and concentrated under reduced pressure. The crude residue was purified by reverse-phase preparative HPLC (acetonitrile 30–70%/0.1% NH<sub>4</sub>OH in water) to afford 7-[7-(difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]-1-methyl-3,4-dihydroquinolin-2-one (<b>20</b>, 33 mg, 41%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.75 (d, <i>J</i> = 0.8 Hz, 1H), 7.50 (d, <i>J</i> = 0.8 Hz, 1H), 7.24 (d, <i>J</i> = 8.0 Hz, 1H), 7.12 (s, 1H), 6.99 (d, <i>J</i> = 2.0 Hz, 1H), 6.93–6.60 (m, 3H), 3.86 (s, 3H), 3.68–3.59 (m, 2H), 3.21 (s, 3H), 2.91–2.80 (m, 4H), 2.57 (dd, <i>J</i> = 8.6, 6.1 Hz, 2H), 1.99 (p, <i>J</i> = 6.2 Hz, 2H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 423.</div></div><div id="sec5-8" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Synthesis and Characterization of 1-(6-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-4-methoxypyridin-2-yl)-3-methylurea (<b>21</b>)</h3><div id="sec5-8-1" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> Step 1, <i>tert</i>-Butyl (6-Bromo-4-methoxypyridin-2-yl)carbamate (<b>47</b>)</h4><div class="NLM_p last">To a solution of 2,6-dibromo-4-methoxypyridine (<b>46</b>, 2.00 g, 7.49 mmol, 1.0 equiv), <i>tert</i>-butyl carbamate (877 mg, 7.49 mmol, 1.0 equiv), and Cs<sub>2</sub>CO<sub>3</sub> (4.80 g, 15.0 mmol, 2.0 equiv) in 1,4-dioxane (50 mL) were added palladium(II) acetate (168 mg, 0.75 mmol, 10 mol %) and 4,5-bis(diphenylphosphino)-9,9-dimethylxanthene (433 mg, 0.75 mmol, 10 mol %). The reaction mixture was heated to 100 °C for 16 h under a nitrogen atmosphere. After cooling the reaction to room temperature, water (100 mL) was added and extracted with EtOAc (50 mL × 3). The combined organic layers were washed with brine (50 mL × 2), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude residue was purified by silica gel column chromatography (petroleum ether/EtOAc = 9:1) to give <i>tert</i>-butyl (6-bromo-4-methoxypyridin-2-yl)carbamate (<b>47</b>, 900 mg, 39%) as a white solid. LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 303.</div></div><div id="sec5-8-2" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> Step 2, <i>tert</i>-Butyl (6-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-4-methoxypyridin-2-yl)carbamate (<b>48</b>)</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl (6-bromo-4-methoxypyridin-2-yl)carbamate (<b>47</b>, 460 mg, 1.52 mmol, 1.0 equiv), 7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-1,2,3,4-tetrahydroquinoline (<b>31</b>, 400 mg, 1.52 mmol, 1.0 equiv), and <i>t</i>-BuONa (436 mg, 4.54 mmol, 3.0 equiv) in 1,4-dioxane (15 mL) was added dichloro[1,3-bis(2,6-di-3-pentylphenyl)imidazol-2-ylidene](3-chloropyridyl)palladium(II) (120 mg, 0.15 mmol, 10 mol %). The reaction mixture was heated to 120 °C for 16 h under a nitrogen atmosphere. After cooling the reaction to room temperature, water (50 mL) was added and extracted with EtOAc (30 mL × 3). The combined organic layers were washed with brine (30 mL × 2), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude residue was purified by Prep-TLC (petroleum ether/EtOAc = 2:1) to give <i>tert</i>-butyl (6-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-4-methoxypyridin-2-yl)carbamate (<b>48</b>, 100 mg, 14%) as a white solid.</div></div><div id="sec5-8-3" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> Step 3, 6-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-4-methoxypyridin-2-amine (<b>49</b>)</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl (6-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-4-methoxypyridin-2-yl)carbamate (<b>48</b>, 100 mg, 0.21 mmol, 1.0 equiv) in DCM (10 mL) at 0 °C was added trifluoroacetic acid (0.076 mL, 1.03 mmol, 4.9 equiv). The reaction mixture was stirred at room temperature for 5 h and concentrated in vacuo to give 6-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-4-methoxypyridin-2-amine (<b>49</b>) as yellow oil that required no further purification. LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 386.</div></div><div id="sec5-8-4" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> Step 4, 1-(6-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-4-methoxypyridin-2-yl)-3-methylurea (<b>21</b>)</h4><div class="NLM_p last">To a solution of 6-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-4-methoxypyridin-2-amine (<b>49</b>, 40 mg, 0.10 mmol, 1.0 equiv) in anhydrous DMF (2 mL) at 0 °C was added NaH (60% dispersion in mineral oil, 20 mg, 0.50 mmol, 5.0 equiv). After stirring at 0 °C for 10 min, <i>N</i>-methyl-1<i>H</i>-imidazole-1-carboxamide (36 mg, 0.29 mmol, 2.9 equiv) was added. The reaction solution was stirred at room temperature for an additional 2 h. The reaction mixture was poured into water (50 mL) and extracted with EtOAc (30 mL × 3). The combined organic layers were washed with brine (30 mL × 2), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude residue was purified by reverse-phase chromatography (acetonitrile 40–70%/0.2% formic acid in water) to give 1-(6-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-4-methoxypyridin-2-yl)-3-methylurea (<b>21</b>, 8.0 mg, 18% over 2 steps) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.88 (s, 1H), 7.69 (s, 1H), 7.60 (s, 1H), 7.49 (s, 1H), 7.18 (s, 1H),7.06 (s, 1H), 6.58 (t, <i>J</i> = 55.2 Hz, 1H), 6.17 (s, 1H), 5.75 (s, 1H), 3.99 (s, 3H), 3.81–3.76 (m, 2H), 3.79 (s, 3H), 2.82–2.80 (m, 2H), 2.70 (d, <i>J</i> = 4.0 Hz, 3H), 2.06 (t, <i>J</i> = 6.0 Hz, 2H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 443.</div></div></div><div id="sec5-9" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Synthesis and Characterization of 5-[7-(Difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]-<i>N</i>-methyl-pyrazolo[1,5-<i>a</i>]pyrimidine-3-carboxamide (<b>22</b>)</h3><div id="sec5-9-1" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> Step 1, Ethyl 5-[7-(Difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]pyrazolo[1,5-<i>a</i>]pyrimidine-3-carboxylate (<b>51</b>)</h4><div class="NLM_p last">To a solution of ethyl 5-bromopyrazolo[1,5-<i>a</i>]pyrimidine-3-carboxylate (<b>50</b>, 185 mg, 0.684 mmol, 1.2 equiv), 7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-1,2,3,4-tetrahydroquinoline (<b>31</b>, 150 mg, 0.570 mmol, 1.0 equiv), and K<sub>3</sub>PO<sub>4</sub> (249 mg, 1.14 mmol, 2.0 equiv) in <i>t-</i>amyl alcohol (4 mL) was added (2-dicyclohexylphosphino-2′,4′,6′-triisopropyl-1,1′-biphenyl)[2-(2′-amino-1,1′-biphenyl)]palladium(II) methanesulfonate (50.8 mg, 0.0570 mmol, 10 mol %). The reaction mixture was heated to 90 °C for 16 h under a nitrogen atmosphere. After cooling the reaction to room temperature, DCM (2 mL) was added, and the reaction mixture was filtered through Celite and concentrated under reduced pressure. The crude residue was purified by preparative SFC (carbon dioxide 5–50%/0.1% NH<sub>4</sub>OH in methanol) to afford ethyl 5-[7-(difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]pyrazolo[1,5-<i>a</i>]pyrimidine-3-carboxylate (<b>51</b>, 18.5 mg, 7%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.81 (d, <i>J</i> = 7.9 Hz, 1H), 8.32 (s, 1H), 7.91 (d, <i>J</i> = 0.8 Hz, 1H), 7.76 (s, 1H), 7.65 (d, <i>J</i> = 0.8 Hz, 1H), 7.41 (s, 1H), 7.14–6.83 (m, 2H), 4.23 (q, <i>J</i> = 7.1 Hz, 2H), 4.06 (t, <i>J</i> = 6.4 Hz, 2H), 3.91 (s, 3H), 2.83 (t, <i>J</i> = 6.5 Hz, 2H), 2.00 (p, <i>J</i> = 6.5 Hz, 2H), 1.28 (t, <i>J</i> = 7.1 Hz, 3H).</div></div><div id="sec5-9-2" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> Step 2, 5-[7-(Difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]-<i>N</i>-methyl-pyrazolo[1,5-<i>a</i>]pyrimidine-3-carboxamide (<b>22</b>)</h4><div class="NLM_p last">To a solution of ethyl 5-[7-(difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]pyrazolo[1,5-<i>a</i>]pyrimidine-3-carboxylate (<b>51</b>, 18.5 mg, 0.0405 mmol, 1.0 equiv) in THF (0.5 mL) and water (0.5 mL) were added lithium hydroxide monohydrate (17.0 mg, 0.405 mmol, 10 equiv) and Cs<sub>2</sub>CO<sub>3</sub> (20 mg, 0.0614 mmol, 1.5 equiv). The mixture was stirred at 60 °C for 3 d. After completion of the reaction, the reaction mixture was concentrated under reduced pressure. The crude residue was purified by reverse-phase preparative HPLC (acetonitrile 20–60%/0.1% NH<sub>4</sub>OH in water). The resulting carboxylic acid (9.9 mg, 0.0233 mmol, 1.0 equiv) was dissolved in DMF (1.0 mL), and TEA (13 μL, 0.093 mmol, 4.0 equiv), methylamine hydrochloride (4.7 mg, 0.070 mmol, 3.0 equiv), and TBTU (22.5 mg, 0.070 mmol, 3.0 equiv) were added. The mixture was stirred at room temperature for 14 h and then concentrated under reduced pressure. The crude residue was purified by reverse-phase preparative HPLC (acetonitrile 5–50%/0.1% NH<sub>4</sub>OH in water) to give 5-[7-(difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]-<i>N</i>-methyl-pyrazolo[1,5-<i>a</i>]pyrimidine-3-carboxamide (<b>22</b>, 6.5 mg, 37%, 2 steps) as a yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 20/21 H): δ 8.88 (d, <i>J</i> = 7.8 Hz, 1H), 8.26 (s, 1H), 7.93 (d, <i>J</i> = 0.8 Hz, 1H), 7.85 (s, 1H), 7.55 (d, <i>J</i> = 4.8 Hz, 1H), 7.43 (s, 1H), 7.22–6.80 (m, 2H), 4.01 (t, <i>J</i> = 6.4 Hz, 2H), 3.91 (s, 3H), 2.80 (dd, <i>J</i> = 10.5, 5.6 Hz, 5H), 2.01 (t, <i>J</i> = 6.3 Hz, 2H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 438.</div></div></div><div id="sec5-10" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> Synthesis and Characterization of 1-[6-[7-(Difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]-3,4-dihydro-2<i>H</i>-quinolin-1-yl]ethanone (<b>23</b>)</h3><div id="sec5-10-1" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> Step 1, 1-(3,4-Dihydro-2<i>H</i>-quinolin-1-yl)ethanone (<b>53</b>)</h4><div class="NLM_p last">To a solution of 1,2,3,4-tetrahydroquinoline (<b>52</b>, 2.00 g, 15.0 mmol, 1.0 equiv) in DCM (30 mL) at 0 °C was added TEA (3.14 mL, 22.5 mmol, 1.5 equiv) and acetic anhydride (1.70 mL, 18.0 mmol, 1.2 equiv). The mixture was heated to 32 °C for 2 h and then allowed to cool to room temperature. Water (50 mL) was added, and the mixture was extracted with DCM (50 mL × 2). The combined organic layers were washed with brine (30 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and then concentrated under reduced pressure. The residue was purified by silica gel chromatography (100% heptanes to 50% EtOAc in a heptane gradient) to afford 1-(3,4-dihydro-2<i>H</i>-quinolin-1-yl)ethanone (<b>53</b>, 2.32 g, 88%) as a pale yellow oil. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.26–7.05 (m, 4H), 3.81 (t, <i>J</i> = 6.6 Hz, 2H), 2.73 (t, <i>J</i> = 6.7 Hz, 2H), 2.24 (s, 3H), 1.97 (p, <i>J</i> = 6.6 Hz, 2H).</div></div><div id="sec5-10-2" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> Step 2, 1-(6-Bromo-3,4-dihydro-2<i>H</i>-quinolin-1-yl)ethanone (<b>54</b>)</h4><div class="NLM_p last">To a solution of 1-(3,4-dihydro-2<i>H</i>-quinolin-1-yl)ethanone (<b>53</b>, 400 mg, 2.0 mmol, 1.0 equiv) in DCM (10 mL) at 0 °C was added <i>N</i>-bromosuccinimide (400 mg, 2.2 mmol, 1.1 equiv) portionwise. The mixture was stirred at room temperature for 2 h then heated at 45 °C for 14 h. After cooling to room temperature, the reaction was concentrated under reduced pressure. The residue was purified by silica gel chromatography (100% heptanes to 60% EtOAc in a heptane gradient) to afford 1-(6-bromo-3,4-dihydro-2<i>H</i>-quinolin-1-yl)ethanone (<b>54</b>, 0.542 g, 90%) as a yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.40–7.03 (m, 3H), 3.76 (t, <i>J</i> = 6.5 Hz, 2H), 2.71 (t, <i>J</i> = 6.6 Hz, 2H), 2.22 (s, 3H), 2.04–1.86 (m, 2H).</div></div><div id="sec5-10-3" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> Step 3, 1-[6-[7-(Difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]-3,4-dihydro-2<i>H</i>-quinolin-1-yl]ethanone (<b>23</b>)</h4><div class="NLM_p last">To a vial were added 7-(difluoromethyl)-6-(1-methylpyrazol-4-yl)-1,2,3,4-tetrahydroquinoline (<b>31</b>, 70.0 mg, 0.266 mmol, 1.0 equiv), 1-(6-bromo-3,4-dihydro-2<i>H</i>-quinolin-1-yl)ethanone (<b>54</b>, 243 mg, 0.912 mmol, 3.4 equiv), dichloro[1,3-bis(2,6-di-3-pentylphenyl)imidazol-2-ylidene](3-chloropyridyl)palladium(II) (33.3 mg, 0.0399 mmol, 15 mol %), <i>t-</i>BuONa (51.1 mg, 0.532 mmol, 2.0 equiv), and 1,4-dioxane (0.53 mL). The mixture was sparged with an argon balloon and then heated to 120 °C for 16 h under an argon atmosphere. After cooling the reaction to room temperature, DCM (3 mL) was added, and the reaction mixture was filtered through Celite and concentrated under reduced pressure. The crude residue was purified by reverse-phase chromatography (acetonitrile 30–70%/0.1% ammonium hydroxide in water) to give 1-[6-[7-(difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]-3,4-dihydro-2<i>H</i>-quinolin-1-yl]ethanone (<b>23</b>, 40.5 mg, 32%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 330 K): δ 7.72 (s, 1H), 7.53 (s, 1H), 7.48 (d, <i>J</i> = 0.8 Hz, 1H), 7.13–7.02 (m, 3H), 6.87–6.60 (m, 2H), 3.86 (s, 3H), 3.73–3.67 (m, 2H), 3.62–3.55 (m, 2H), 2.83 (t, <i>J</i> = 6.5 Hz, 2H), 2.71 (t, <i>J</i> = 6.6 Hz, 2H), 2.18 (s, 3H), 2.03–1.95 (m, 2H), 1.93–1.84 (m, 2H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 437.</div></div></div><div id="sec5-11" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> Synthesis and Characterization of (<i>R</i>)-5-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-3-methyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>24</b>)</h3><div id="sec5-11-1" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> Step 1, 7-(Difluoromethyl)-1-(2-fluoro-3-nitrophenyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-1,2,3,4-tetrahydroquinoline (<b>56</b>)</h4><div class="NLM_p last">To a solution of 1-bromo-2-fluoro-3-nitrobenzene (<b>55</b>, 300 mg, 1.36 mmol, 1.3 equiv) and 7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-1,2,3,4-tetrahydroquinoline (<b>31</b>, 276 mg, 1.05 mmol, 1.0 equiv) in 1,4-dioxane (2 mL) were added Cs<sub>2</sub>CO<sub>3</sub> (684 mg, 2.1 mmol, 2.0 equiv), palladium(II) acetate (24 mg, 0.10 mmol, 10 mol %), and 4,5-bis(diphenylphosphino)-9,9-dimethylxanthene (61 mg, 0.10 mmol, 10 mol %). The mixture was irradiated in a microwave at 140 °C for 1 h. The mixture was filtered and concentrated in vacuo. The crude residue was purified by Prep-TLC (petroleum ether/EtOAc = 1:1) to give 7-(difluoromethyl)-1-(2-fluoro-3-nitrophenyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-1,2,3,4-tetrahydroquinoline (<b>56</b>, 110 mg, 20%) as a red solid. LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 403.</div></div><div id="sec5-11-2" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> Step 2, (<i>R</i>)-2-((2-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-6-nitrophenyl)amino)propanoic Acid (<b>57</b>)</h4><div class="NLM_p last">To a solution of 7-(difluoromethyl)-1-(2-fluoro-3-nitrophenyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-1,2,3,4-tetrahydroquinoline (<b>56</b>, 300 mg, 0.75 mmol, 1.0 equiv) in DMF (3 mL) were added (<i>R</i>)-2-aminopropanoic acid (100 mg, 1.12 mmol, 1.5 equiv) and Cs<sub>2</sub>CO<sub>3</sub> (486 mg, 1.49 mmol, 2.0 equiv). The reaction mixture was heated to 100 °C for 4 h. After cooling to room temperature, EtOAc (30 mL) was added, and the mixture was washed with water (20 mL × 3). The combined aqueous layers were acidified with HCl (2 N) to pH 4 and then extracted with EtOAc (40 mL × 3). The combined organic layers were washed with brine (20 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuo to give (<i>R</i>)-2-((2-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-6-nitrophenyl)amino)propanoic acid (<b>57</b>, 300 mg, 85%) as a brown solid that required no further purification. LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 472.</div></div><div id="sec5-11-3" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> Step 3, (<i>R</i>)-5-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-3-methyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>24</b>)</h4><div class="NLM_p last">To a solution of (<i>R</i>)-2-((2-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-6-nitrophenyl)amino)propanoic acid (<b>57</b>, 300 mg, 0.64 mmol, 1.0 equiv) in AcOH (4 mL) was added Fe powder (178 mg, 3.18 mmol, 5.0 equiv). The reaction mixture was heated to 100 °C for 3 h under a nitrogen atmosphere. After cooling to room temperature, the mixture was filtered and concentrated in vacuo. The crude residue was purified by reverse-phase chromatography (acetonitrile 36–56%/0.2% formic acid in water) to give (<i>R</i>)-5-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-3-methyl-3,4-dihydroquinoxalin-2(1<i>H</i>)-one (<b>24</b>, 39 mg, 14%) as a yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.36 (s, 1H), 7.73 (s, 1H), 7.48 (s, 1H), 7.22–7.02 (m, 1H), 6.89–6.51 (m, 4H), 6.45–6.24 (m, 1H), 5.41 (d, <i>J</i> = 12.8 Hz, 1H), 3.91–3.66 (m, 1H), 3.86 (s, 3H), 3.50–3.36 (m, 2H), 2.89–2.85 (m, 2H), 2.33–1.94 (m, 2H), 1.28–1.07 (m, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 424.</div></div></div><div id="sec5-12" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> Synthesis and Characterization of 7-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-<i>N</i>-methyl-2,3-dihydrobenzo[<i>f</i>][1,4]oxazepine-4(5<i>H</i>)-carboxamide (<b>25</b>)</h3><div id="sec5-12-1" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> Step 1, <i>tert</i>-Butyl 7-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-2,3-dihydrobenzo[<i>f</i>][1,4]oxazepine-4(5<i>H</i>)-carboxylate (<b>59</b>)</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl 7-bromo-3,5-dihydro-2<i>H</i>-1,4-benzoxazepine-4-carboxylate (<b>58</b>, 800 mg, 2.44 mmol, 1.0 equiv), 7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-1,2,3,4-tetrahydroquinoline (<b>31</b>, 642 mg, 2.44 mmol, 1.0 equiv), and <i>t</i>-BuONa (586 mg, 6.09 mmol, 2.5 equiv) in 1,4-dioxane (6 mL) was added dichloro[1,3-bis(2,6-di-3-pentylphenyl)imidazol-2-ylidene](3-chloropyridyl)palladium(II) (193 mg, 0.24 mmol, 10 mol %). The reaction mixture was heated to 120 °C for 16 h under a nitrogen atmosphere. After cooling to room temperature, the mixture was filtered and concentrated in vacuo. The crude residue was purified by silica gel column chromatography (petroleum ether/EtOAc = 3:2) to give <i>tert</i>-butyl 7-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-2,3-dihydrobenzo[<i>f</i>][1,4]oxazepine-4(5<i>H</i>)-carboxylate (<b>59</b>, 670 mg, 54%) as a yellow solid. LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 511.</div></div><div id="sec5-12-2" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> Step 2, 7-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-2,3,4,5-tetrahydrobenzo[<i>f</i>][1,4]oxazepine (<b>60</b>)</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl 7-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-2,3-dihydrobenzo[<i>f</i>][1,4]oxazepine-4(5<i>H</i>)-carboxylate (<b>59</b>, 870 mg, 1.7 mmol, 1.0 equiv) in DCM (5 mL) at 0 °C was added trifluoroacetic acid (1.26 mL, 17.0 mmol, 10 equiv) dropwise. The reaction mixture was stirred at room temperature for 1 h and concentrated in vacuo to give 7-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-2,3,4,5-tetrahydrobenzo[<i>f</i>][1,4]oxazepine (<b>60</b>, 600 mg, 86%) as a yellow solid that required no further purification. LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 411.</div></div><div id="sec5-12-3" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> Step 3, 7-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-<i>N</i>-methyl-2,3-dihydrobenzo[<i>f</i>][1,4]oxazepine-4(5<i>H</i>)-carboxamide (<b>25</b>)</h4><div class="NLM_p last">To a stirred solution of 7-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-2,3,4,5-tetrahydrobenzo[<i>f</i>][1,4]oxazepine (<b>60</b>, 150 mg, 0.37 mmol, 1.0 equiv) and triethylamine (0.25 mL, 1.83 mmol, 5.0 equiv) in DCM (5 mL) was added <i>N</i>-methyl-1<i>H</i>-imidazole-1-carboxamide (92 mg, 0.74 mmol, 2.0 equiv). The reaction mixture was stirred at room temperature for 16 h and concentrated in vacuo. The crude residue was purified by reverse-phase chromatography (acetonitrile 35–65%/0.2% formic acid in water) to give 7-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-<i>N</i>-methyl-2,3-dihydrobenzo[<i>f</i>][1,4]oxazepine-4(5<i>H</i>)-carboxamide (<b>25</b>, 83 mg, 48%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.74 (s, 1H), 7.49 (s, 1H), 7.31 (d, <i>J</i> = 2.2 Hz, 1H), 7.13–7.04 (m, 2H), 7.03–6.96 (m, 1H), 6.90–6.59 (m, 2H), 6.48–6.43 (m, 1H), 4.42 (s, 2H), 4.01–3.98 (m, 2H), 3.86 (s, 3H), 3.72–3.69 (m, 2H), 3.59–3.51 (m, 2H), 2.88–2.79 (m, 2H), 2.52 (s, 3H), 2.03–1.92 (m, 2H). LCMS (<i>m</i>/<i>z</i>) [M + Na]<sup>+</sup> 490.</div></div></div><div id="sec5-13" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> Synthesis and Characterization of 5-[7-(Difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]-1,3-dimethyl-benzimidazol-2-one (<b>26</b>)</h3><div class="NLM_p last">To a vial was added 7-(difluoromethyl)-6-(1-methylpyrazol-4-yl)-1,2,3,4-tetrahydroquinoline (<b>31</b>, 40.0 mg, 0.152 mmol, 1.0 equiv), 5-bromo-1,3-dimethyl-benzimidazol-2-one (<b>61</b>, 44.0 mg, 0.182 mmol, 1.2 equiv), dichloro[1,3-bis(2,6-di-3-pentylphenyl)imidazol-2-ylidene](3-chloropyridyl)palladium(II) (19.0 mg, 0.0228 mmol, 15 mol %), <i>t-</i>BuONa (29.2 mg, 0.304 mmol, 2.0 equiv), and 1,4-dioxane (0.3 mL). The mixture was sparged with an argon balloon and then heated to 120 °C for 16 h under an argon atmosphere. After cooling the reaction to room temperature, DCM (3 mL) was added, and the reaction mixture was filtered through Celite and concentrated under reduced pressure. The crude residue was purified by reverse-phase chromatography (acetonitrile 20–60%/0.1% ammonium hydroxide in water) to give 5-[7-(difluoromethyl)-6-(1-methylpyrazol-4-yl)-3,4-dihydro-2<i>H</i>-quinolin-1-yl]-1,3-dimethyl-benzimidazol-2-one (<b>26</b>, 40.5 mg, 32%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 20/23 H): δ 7.72 (d, <i>J</i> = 0.8 Hz, 1H), 7.47 (d, <i>J</i> = 0.8 Hz, 1H), 7.20 (d, <i>J</i> = 8.2 Hz, 1H), 7.14 (d, <i>J</i> = 2.0 Hz, 1H), 7.08 (d, <i>J</i> = 1.4 Hz, 1H), 6.98 (dd, <i>J</i> = 8.3, 2.0 Hz, 1H), 6.86–6.55 (m, 2H), 3.86 (s, 3H), 3.64–3.56 (m, 2H), 3.35 (s, 3H), 2.95–2.76 (m, 2H), 2.10–1.93 (m, 2H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 424.</div></div><div id="sec5-14" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> Synthesis and Characterization of 6-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-1,3-dimethyl-4-morpholino-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>27</b>)</h3><div id="sec5-14-1" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> Step 1, 6-Bromo-4-chloro-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>63</b>)</h4><div class="NLM_p last">To a solution of 5-bromo-3-chlorobenzene-1,2-diamine (<b>62</b>, 200 mg, 0.90 mmol, 1.0 equiv) in chloroform (3 mL) was added di(1<i>H</i>-imidazol-1-yl)methanone (220 mg, 1.35 mmol, 1.5 equiv). The mixture was heated to 60 °C for 12 h under a nitrogen atmosphere. After cooling the reaction to room temperature, the white precipitate was collected, washed with chloroform (3 mL × 2), and dried in vacuo to give 6-bromo-4-chloro-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>63</b>, 170 mg, 76%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.35 (s, 1H), 11.05 (s, 1H), 7.21 (d, <i>J</i> = 1.6 Hz, 1H), 7.04 (d, <i>J</i> = 1.6 Hz, 1H).</div></div><div id="sec5-14-2" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> Step 2, 6-Bromo-4-chloro-1,3-dimethyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>64</b>)</h4><div class="NLM_p last">To a stirred solution of 6-bromo-4-chloro-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>63</b>, 170 mg, 0.69 mmol, 1.0 equiv) in DMF (2 mL) at 0 °C was added NaH (60% dispersion in mineral oil, 69 mg, 1.72 mmol, 2.5 equiv), and the mixture was stirred for 15 min. Methyl iodide (0.13 mL, 2.06 mmol, 3.0 equiv) was added dropwise, and the mixture was stirred at room temperature for an additional 12 h. Water (30 mL) was added and extracted with EtOAc (30 mL × 3). The combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude residue was purified by silica gel chromatography (petroleum ether/EtOAc = 5:1) to give title compound (<b>64</b>, 150 mg, 79%) as a white solid. LCMS M/Z (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 275.</div></div><div id="sec5-14-3" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> Step 3, 4-Chloro-6-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-1,3-dimethyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>65</b>)</h4><div class="NLM_p last">To a solution of 7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-1,2,3,4-tetrahydroquinoline (<b>31</b>, 143 mg, 0.54 mmol, 1.0 equiv), 6-bromo-4-chloro-1,3-dimethyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>64</b>, 150 mg, 0.54 mmol, 1.0 equiv), and K<sub>3</sub>PO<sub>4</sub> (347 mg, 1.63 mmol, 3.0 equiv) in 2-methyl-2-butanol (5 mL) was added methanesulfonato(2-dicyclohexylphosphino-2′,4′,6′-tri<i>i</i>-propyl-1,1′-biphenyl)(2′-amino-1,1′-biphenyl-2-yl)palladium(II) (46 mg, 0.054 mmol, 10 mol %). The mixture was heated to 95 °C for 12 h under an argon atmosphere. After cooling the reaction to room temperature, the mixture was filtered and concentrated in vacuo. The crude residue was purified by reverse-phase chromatography (acetonitrile 35–65%/0.2% formic acid in water) to give 4-chloro-6-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-1,3-dimethyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>65</b>, 28 mg, 11%) as a yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.55 (s, 1H), 7.42 (s, 1H), 7.07 (s, 1H), 6.94 (d, <i>J</i> = 1.6 Hz, 1H), 6.88 (s, 1H), 6.80 (d, <i>J</i> = 1.6 Hz, 1H), 6.49 (t, <i>J</i> = 55.6 Hz, 1H), 3.96 (s, 3H), 3.76 (s, 3H), 3.64–3.61 (m, 2H), 3.38 (s, 3H), 2.92–2.89 (m, 2H), 2.11–2.09 (m, 2H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 458.</div></div><div id="sec5-14-4" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> Step 4, 6-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-1,3-dimethyl-4-morpholino-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>27</b>)</h4><div class="NLM_p last">To a solution of 4-chloro-6-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-1,3-dimethyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>65</b>, 200 mg, 0.44 mmol, 1.0 equiv) in 1,4-dioxane (3 mL) were added morpholine (0.038 mL, 0.44 mmol, 1.0 equv), dichloro[1,3-bis(2,6-di-3-pentylphenyl)imidazol-2-ylidene](3-chloropyridyl)palladium(II) (76 mg, 0.10 mmol, 23 mol %), and <i>t</i>-BuONa (277 mg, 2.88 mmol, 6.5 equiv). The mixture was irradiated in a microwave at 140 °C for 0.5 h. EtOAc (30 mL) was added, and the mixture washed with water (30 mL × 2) and brine (30 mL). The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude residue was purified by reverse-phase chromatography (acetonitrile 42–72%/0.2% formic acid in water) to give title compound 6-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-1,3-dimethyl-4-morpholino-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (<b>27</b>, 60 mg, 27%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.73 (d, <i>J</i> = 0.8 Hz, 1H), 7.48 (d, <i>J</i> = 0.8 Hz, 1H), 7.09 (s, 1H), 6.92 (d, <i>J</i> = 1.9 Hz, 1H), 6.90–6.56 (m, 3H), 3.86 (s, 3H), 3.76 (s, 4H), 3.63 (s, 5H), 3.29 (s, 3H), 2.87 (dt, <i>J</i> = 12.3, 5.6 Hz, 6H), 2.10–1.94 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 155.2, 144.5, 141.7, 138.2, 138.1, 132.5, 131.1, 128.8 (t, <i>J</i> = 21 Hz), 126.3, 120.9, 120.6 (t, <i>J</i> = 6.0 Hz), 119.0, 117.6, 114.2, 111.8, 110.3 (t, <i>J</i> = 7.0 Hz), 103.0, 66.6, 53.5, 51.6, 39.0, 29.7, 27.61, 27.56, 22.3. HRMS-ESI (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> calcd for C<sub>27</sub>H<sub>31</sub>O<sub>2</sub>N<sub>6</sub>F<sub>2</sub> = 509.2471; found, 509.2461.</div></div></div><div id="sec5-15" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> Synthesis and Characterization of 5-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxamide (<b>28</b>)</h3><div id="sec5-15-1" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> Step 1, 1-(5-Bromo-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-2,2,2-trichloroethanone (<b>67</b>)</h4><div class="NLM_p last">To a solution of 5-bromo-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine (<b>66</b>, 2.50 g, 12.7 mmol, 1.0 equiv) and aluminum chloride (12.69 g, 95.16 mmol, 7.5 equiv) in DCM (75 mL) at 48 °C was added trichloroacetyl chloride (1.48 mL, 13.3 mmol, 1.05 equiv) dropwise. The reaction mixture was heated to 48 °C for 2 h. After cooling the reaction to room temperature, the mixture was quenched with ice water (100 mL) and extracted with EtOAc (150 mL × 3). The combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude residue was purified by silica gel chromatography (petroleum ether/EtOAc = 1:1) to give 1-(5-bromo-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-2,2,2-trichloroethanone (<b>67</b>, 3.5 g, 81%) as a brown solid. LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 343.</div></div><div id="sec5-15-2" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> Step 2, Methyl 5-Bromo-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylate (<b>68</b>)</h4><div class="NLM_p last">To a solution of 1-(5-bromo-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-2,2,2-trichloroethanone (<b>67</b>, 3.2 g, 9.35 mmol, 1.0 equiv) in MeOH (50 mL) was added potassium hydroxide (577 mg, 10.28 mmol, 1.1 equiv). The reaction was stirred at room temperature for 16 h and concentrated in vacuo. Water (50 mL) was added and extracted with EtOAc (50 mL × 3). The combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to give methyl 5-bromo-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylate (<b>68</b>, 2.0 g, 84%) as a brown solid that required no further purification. LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 255.</div></div><div id="sec5-15-3" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> Step 3, Methyl 5-Bromo-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylate (<b>69</b>)</h4><div class="NLM_p last">To a solution of methyl 5-bromo-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylate (<b>68</b>, 700 mg, 2.74 mmol, 1.0 equiv) in DMF (8 mL) at 0 °C was added NaH (60% dispersion in mineral oil, 154 mg, 3.84 mmol, 1.4 equiv) portionwise. The mixture was stirred at room temperature for 0.5 h. (2-(Chloromethoxy)ethyl)trimethylsilane (0.58 mL, 3.29 mmol, 1.2 equiv) was added dropwise, and the mixture was stirred at room temperature for an additional 1 h. The mixture was quenched with water (20 mL) and extracted with EtOAc (20 mL × 3). The combined organic layers were washed with brine (20 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude residue was purified by silica gel chromatography (petroleum ether/EtOAc = 20:1) to give methyl 5-bromo-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylate (<b>69</b>, 650 mg, 62%) as a white solid.</div></div><div id="sec5-15-4" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> Step 4, Methyl 5-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylate (<b>70</b>)</h4><div class="NLM_p last">To a solution of methyl 5-bromo-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylate (<b>69</b>, 480 mg, 1.25 mmol, 1.0 equiv) in 1,4-dioxane (3 mL) were added 7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-1,2,3,4-tetrahydroquinoline (<b>31</b>, 361 mg, 1.37 mmol, 1.1 equiv), dichloro[1,3-bis(2,6-di-3-pentylphenyl)imidazol-2-ylidene](3-chloropyridyl)palladium(II) (99 mg, 0.12 mmol, 10 mol %) and Cs<sub>2</sub>CO<sub>3</sub> (1.01 g, 3.11 mmol, 2.5 equiv). The mixture was heated to 110 °C for 12 h under a nitrogen atmosphere. EtOAc (30 mL) was added, and the mixture was washed with water (30 mL × 2) and brine (30 mL). The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude residue was purified by silica gel chromatography (petroleum ether/EtOAc = 1:1) to give methyl 5-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylate (<b>70</b>, 300 mg, 42%) as a yellow solid. LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 568.</div></div><div id="sec5-15-5" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> Step 5, 5-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylic Acid (<b>71</b>)</h4><div class="NLM_p last">To a solution of methyl 5-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylate (<b>70</b>, 200 mg, 0.35 mmol, 1.0 equiv) in MeOH (5 mL) and water (1 mL) was added lithium hydroxide monohydrate (45 mg, 1.06 mmol, 3.0 equiv). The reaction was stirred at room temperature for 3 h and concentrated in vacuo. Water (5 mL) was added, and the mixture was acidified with HCl (2 N) to pH 2 and then extracted with EtOAc (20 mL × 3). The combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to give 5-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylic acid (<b>71</b>, 150 mg, 77%) as a brown solid that required no further purification.</div></div><div id="sec5-15-6" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> Step 6, 5-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-<i>N</i>-methyl-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxamide (<b>72</b>)</h4><div class="NLM_p last">To a solution of 5-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylic acid (<b>71</b>, 100 mg, 0.18 mmol, 1.0 equiv) and triethylamine (0.049 mL, 0.35 mmol, 1.9 equiv) in DMF (8 mL) were added <i>O</i>-(7-azabenzotriazol-1-yl)-<i>N,N,N′,N</i>′-tetramethyluronium hexafluorophosphate (80 mg, 0.21 mmol, 1.2 equiv) and methylamine hydrochloride (18 mg, 0.26 mmol, 1.4 equiv). The reaction was stirred at room temperature for 16 h. EtOAc (30 mL) was added, and the mixture was washed with water (30 mL × 3) and brine (30 mL). The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude residue was purified by silica gel chromatography (DCM/MeOH = 9:1) to give 5-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-<i>N</i>-methyl-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxamide (<b>72</b>, 100 mg, 98%) as a brown oil.</div></div><div id="sec5-15-7" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> Step 7, 5-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxamide (<b>28</b>)</h4><div class="NLM_p last">A mixture of 5-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-<i>N</i>-methyl-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxamide (<b>72</b>, 100 mg, 0.18 mmol, 1.0 equiv) and tetrabutylammonium fluoride (1.0 M in THF, 6.0 mL, 6.0 mmol, 3.3 equiv) was heated to 50 °C for 2 d. After cooling the reaction mixture to room temperature, it was concentrated in vacuo. The crude residue was purified by reverse-phase chromatography (acetonitrile 30–60%/0.05% NH<sub>4</sub>OH in water) to give 5-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxamide (<b>28</b>, 16 mg, 20%) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.32 (s, 1H), 8.61 (s, 1H), 7.75 (s, 1H), 7.69 (s, 1H), 7.56 (s, 1H), 7.44 (s, 1H), 7.32 (s, 1H), 7.10 (s, 1H), 6.51 (t, <i>J</i> = 55.6 Hz, 1H), 5.87–5.86 (m, 1H), 3.96 (s, 3H), 3.88–3.85 (m, 2H), 2.99 (d, <i>J</i> = 4.4 Hz, 3H), 2.90–2.88 (m, 2H), 2.08–2.05 (m, 2H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 437.</div></div></div><div id="sec5-16" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> Synthesis and Characterization of 5-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-7-isopropyl-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxamide (<b>29</b>)</h3><div id="sec5-16-1" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> Step 1, 2-Bromo-6-chloro-3-nitropyridine (<b>74</b>)</h4><div class="NLM_p last">To a solution of 6-chloro-3-nitro-2-pyridinylamine (<b>73</b>, 20.0 g, 115 mmol, 1.0 equiv), copper(II) bromide (25.7 g, 115 mmol, 1.0 equiv), and copper(I) bromide (2.48 g, 17.3 mmol, 6.7 equiv) in MeCN (500 mL) at 0 °C was added isopentyl nitrite (20.17 mL, 149.8 mmol, 1.3 equiv). The reaction mixture was heated to 60 °C for 1 h under a nitrogen atmosphere. After cooling the reaction to room temperature, the mixture was concentrated in vacuo. The crude residue was purified by silica gel chromatography (petroleum ether/EtOAc = 5:1) to give 2-bromo-6-chloro-3-nitropyridine (<b>74</b>, 15 g, 55%) as a yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.56 (d, <i>J</i> = 8.4 Hz, 1H), 7.89 (d, <i>J</i> = 8.4 Hz, 1H).</div></div><div id="sec5-16-2" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> Step 2, 7-Bromo-5-chloro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine (<b>75</b>)</h4><div class="NLM_p last">To a solution of 2-bromo-6-chloro-3-nitropyridine (<b>74</b>, 10.0 g, 42.1 mmol, 1.0 equiv) in anhydrous THF (240 mL) at −78 °C was added vinylmagnesium bromide (1.0 M in THF, 329 mL, 329 mmol, 7.8 equiv). The reaction mixture was stirred at −50 °C for 1 h under a nitrogen atmosphere. The mixture was quenched with water (150 mL) and extracted with EtOAc (150 mL × 3). The combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude residue was purified by silica gel chromatography (petroleum ether/EtOAc = 1:1) to give 7-bromo-5-chloro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine (<b>75</b>, 2.6 g, 27%) as a brown solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.13 (s, 1H), 7.76 (s, 1H), 7.69 (s, 1H), 6.64 (s, 1H).</div></div><div id="sec5-16-3" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> Step 3, 1-(7-Bromo-5-chloro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-2,2,2-trichloroethanone (<b>76</b>)</h4><div class="NLM_p last">To a solution of 7-bromo-5-chloro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine (<b>75</b>, 1.9 g, 8.21 mmol, 1.0 equiv) and aluminum chloride (8.21 g, 61.6 mmol, 7.5 equiv) in DCM (60 mL) at 48 °C was added trichloroacetyl chloride (2.74 mL, 24.6 mmol, 3.0 equiv) dropwise. The reaction mixture was heated to 48 °C for 2 h. After cooling the reaction to room temperature, the mixture was quenched with ice water (60 mL) and extracted with EtOAc (70 mL × 3). The combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude residue was purified by silica gel chromatography (petroleum ether/EtOAc = 1:3) to give 7-bromo-5-chloro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine (<b>76</b>, 3.0 g, 97%) as a brown solid. LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 377.</div></div><div id="sec5-16-4" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> Step 4, Methyl 7-Bromo-5-chloro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylate (<b>77</b>)</h4><div class="NLM_p last">To a solution of 1-(7-bromo-5-chloro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-2,2,2-trichloroethanone (<b>76</b>, 3.2 g, 8.49 mmol, 1.0 equiv) in MeOH (50 mL) was added potassium hydroxide (524 mg, 9.34 mmol, 1.1 equiv). The reaction was stirred at room temperature for 16 h and concentrated in vacuo. Water (50 mL) was added and extracted with EtOAc (50 mL × 3). The combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to give methyl 7-bromo-5-chloro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylate (<b>77</b>, 2.0 g, 82%) as a brown solid that required no further purification. LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 291.</div></div><div id="sec5-16-5" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> Step 5, Methyl 5-Chloro-7-(prop-1-en-2-yl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylate (<b>78</b>)</h4><div class="NLM_p last">To a solution of methyl 7-bromo-5-chloro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylate (<b>77</b>, 2.50 g, 8.64 mmol, 1.0 equiv) in 1,4-dioxane (10 mL) and water (1.5 mL) were added [1,1′-bis(diphenylphosphino)ferrocene]dichloropalladium(II) (632 mg, 0.860 mmol, 10 mol %), Na<sub>2</sub>CO<sub>3</sub> (2.75 g, 25.9 mmol, 3.0 equiv), and 4,4,5,5-tetramethyl-2-(prop-1-en-2-yl)-1,3,2-dioxaborolane (1.74 g, 10.4 mmol, 1.2 equiv). The mixture was heated to 95 °C for 16 h under a nitrogen atmosphere. After cooling the reaction to room temperature, the mixture was filtered and concentrated in vacuo. The crude residue was purified by silica gel chromatography (petroleum ether/EtOAc = 5:1) to give methyl 5-chloro-7-(prop-1-en-2-yl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylate (<b>78</b>, 1.8 g, 83%) as a brown solid. LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 251.</div></div><div id="sec5-16-6" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> Step 6, Methyl 5-Chloro-7-(prop-1-en-2-yl)-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylate (<b>79</b>)</h4><div class="NLM_p last">To a solution of methyl 5-chloro-7-(prop-1-en-2-yl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylate (<b>78</b>, 1.50 g, 5.98 mmol, 1.0 equiv) in DMF (15 mL) at 0 °C was added NaH (60% dispersion in mineral oil, 0.50 g, 12 mmol, 2.0 equiv) portionwise. The mixture was stirred at room temperature for 0.5 h. (2-(Chloromethoxy)ethyl)trimethylsilane (1.28 mL, 7.18 mmol, 1.2 equiv) was added dropwise, and the mixture was stirred at room temperature for an additional 4 h. The mixture was quenched with water (15 mL) and extracted with EtOAc (15 mL × 3). The combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude residue was purified by silica gel chromatography (petroleum ether/EtOAc = 7:1) to give methyl 5-chloro-7-(prop-1-en-2-yl)-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylate (<b>79</b>, 310 mg, 14%) as a yellow solid. LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 381.</div></div><div id="sec5-16-7" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> Step 7, Methyl 5-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-7-(prop-1-en-2-yl)-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylate (<b>80</b>)</h4><div class="NLM_p last">To a solution of methyl 5-chloro-7-(prop-1-en-2-yl)-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylate (<b>79</b>, 300 mg, 0.79 mmol, 1.0 equiv) in 1,4-dioxane (8 mL) were added 7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-1,2,3,4-tetrahydroquinoline (<b>31</b>, 207 mg, 0.79 mmol, 1.0 equiv), dichloro[1,3-bis(2,6-di-3-pentylphenyl)imidazol-2-ylidene](3-chloropyridyl)palladium(II) (63 mg, 0.08 mmol, 10 mol %), and Cs<sub>2</sub>CO<sub>3</sub> (770 mg, 2.36 mmol, 3.0 equiv). The mixture was heated to 110 °C for 16 h under a nitrogen atmosphere. EtOAc (30 mL) was added, and the mixture was washed with water (30 mL × 2) and brine (30 mL). The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude residue was purified by silica gel chromatography (petroleum ether/EtOAc = 3:1) to give methyl 5-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-7-(prop-1-en-2-yl)-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylate (<b>80</b>, 310 mg, 65%) as a yellow solid. LCMS LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 608.</div></div><div id="sec5-16-8" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> Step 8, 5-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-7-(prop-1-en-2-yl)-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylic Acid (<b>81</b>)</h4><div class="NLM_p last">To a solution of methyl 5-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-7-(prop-1-en-2-yl)-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylate (<b>80</b>, 310 mg, 0.51 mmol, 1.0 equiv) in MeOH (10 mL), and water (2 mL) was added lithium hydroxide monohydrate (107 mg, 2.55 mmol, 5.0 equiv). The reaction was stirred at room temperature for 12 h under a nitrogen atmosphere and concentrated in vacuo. Water (10 mL) was added, and the mixture was acidified with HCl (2 N) to pH 4 and then extracted with EtOAc (30 mL × 3). The combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to give 5-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-7-(prop-1-en-2-yl)-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylic acid (<b>81</b>, 300 mg, 99%) as a white solid that required no further purification. LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 594.</div></div><div id="sec5-16-9" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> Step 9, 5-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-<i>N</i>-methyl-7-(prop-1-en-2-yl)-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxamide (<b>82</b>)</h4><div class="NLM_p last">To a solution of 5-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-7-(prop-1-en-2-yl)-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxylic acid (<b>81</b>, 300 mg, 0.51 mmol, 1.0 equiv) and <i>N,N</i>-diisopropylethylamine (0.18 mL, 1.04 mmol, 2.0 equiv) in DMF (10 mL) was added <i>O</i>-(7-azabenzotriazol-1-yl)-<i>N,N,N′,N</i>′-tetramethyluronium hexafluorophosphate (298 mg, 0.78 mmol, 1.5 equiv) and methylamine hydrochloride (46 mg, 0.68 mmol, 1.3). The reaction was stirred at room temperature for 16 h. EtOAc (30 mL) was added, and the mixture was washed with water (10 mL × 2) and brine (10 mL). The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude residue was purified by silica gel chromatography (petroleum ether/EtOAc = 6:1) to give 5-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-<i>N</i>-methyl-7-(prop-1-en-2-yl)-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxamide (<b>82</b>, 290 mg, 95%) as a white solid. LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> 607.</div></div><div id="sec5-16-10" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> Step 10, 5-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-<i>N</i>-methyl-7-(prop-1-en-2-yl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxamide (<b>83</b>)</h4><div class="NLM_p last">A mixture of 5-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-<i>N</i>-methyl-7-(prop-1-en-2-yl)-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxamide (<b>82</b>, 290 mg, 0.48 mmol, 1.0 equiv) and tetrabutylammonium fluoride (1.0 M in THF, 10 mL, 10 mmol, 21 equiv) was heated to 55 °C for 2 h under a nitrogen atmosphere. After cooling the reaction to room temperature, the mixture was concentrated in vacuo. The crude residue was dissolved in DCM (50 mL) and washed with brine (300 mL × 4). The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to give 5-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-<i>N</i>-methyl-7-(prop-1-en-2-yl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxamide (<b>83</b>, 210 mg, 92%) as brown oil that required no further purification.</div></div><div id="sec5-16-11" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> Step 11, 5-(7-(Difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-7-isopropyl-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxamide (<b>29</b>)</h4><div class="NLM_p last">To a solution of 5-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-<i>N</i>-methyl-7-(prop-1-en-2-yl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxamide (<b>83</b>, 100 mg, 0.21 mmol, 1.0 equiv) in MeOH (20 mL) was added 10% Pd/C (10 mg, 0.009 mmol). The mixture was stirred at room temperature for 18 h under a hydrogen atmosphere (50 psi). The mixture was filtered, and the filtrate was concentrated in vacuo. The crude residue was purified by reverse-phase chromatography (acetonitrile 20–50%/0.2% formic acid in water) to give 5-(7-(difluoromethyl)-6-(1-methyl-1<i>H</i>-pyrazol-4-yl)-3,4-dihydroquinolin-1(2<i>H</i>)-yl)-7-isopropyl-<i>N</i>-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carboxamide (<b>29</b>, 62 mg, 62%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.94 (s, 1H), 8.11 (s, 1H), 7.90 (q, <i>J</i> = 4.5 Hz, 1H), 7.77 (d, <i>J</i> = 0.9 Hz, 1H), 7.65 (s, 1H), 7.52 (d, <i>J</i> = 0.8 Hz, 1H), 7.34 (s, 1H), 7.12 (d, <i>J</i> = 1.4 Hz, 1H), 6.76 (t, <i>J</i> = 55.2 Hz, 1H), 3.87 (s, 3H), 3.82–3.74 (m, 2H), 3.56–3.45 (m, 1H), 2.86 (t, <i>J</i> = 6.5 Hz, 2H), 2.76 (d, <i>J</i> = 4.5 Hz, 3H), 2.09–1.95 (m, 2H), 1.28 (d, <i>J</i> = 6.8 Hz, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.9, 150.9, 149.8, 142.9, 138.3, 134.1, 131.4, 130.8, 129.8, 128.5, 128.31, 128.28, 128.1, 127.7, 121.5 (t, <i>J</i> = 6.0 Hz), 119.1, 114.3 (t, <i>J</i> = 234 Hz), 113.1, (t, <i>J</i> = 6.0 Hz), 49.3, 39.0, 31.1, 27.6, 25.9, 22.6, 21.9. HRMS-ESI (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> calcd for C<sub>26</sub>H<sub>29</sub>ON<sub>6</sub>F<sub>2</sub> = 479.2365; found, 479.2362.</div></div></div><div id="sec5-17" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> Modeling Methods</h3><div class="NLM_p last">The figures of crystal structures were created using PyMol: All figure were made using PyMOL Molecular Graphics System, version 1.8 Schrödinger, LLC.</div></div><div id="sec5-18" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> Expression and Purification of Bromodomain Proteins</h3><div class="NLM_p last">The expression and purification of bromodomain proteins for binding assays was carried out as previously described.<a onclick="showRef(event, 'ref48 ref62'); return false;" href="javascript:void(0);" class="ref ref48 ref62">(48, 62)</a></div></div><div id="sec5-19" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> Expression and Purification of Bromodomain Proteins for Crystallography</h3><div class="NLM_p last">The CBP bromodomain was expressed in <i>E. coli</i> as an N-terminal His6-FLAG-tagged fusion with a thrombin protease site for removal of the affinity tags. Proteins were purified by sequential Ni affinity and size-exclusion chromatography steps. Affinity tags were removed after the Ni affinity step by dialysis against a thrombin-containing buffer. Additional ion-exchange steps were included as necessary to achieve homogeneous preparations, and the identity of the protein was verified by mass spectrometry.</div></div><div id="sec5-20" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> Time-Resolved Fluorescence Resonance Energy Transfer Assays</h3><div class="NLM_p last">Compound potencies were evaluated in a panel of biochemical-bromodomain-binding assays. Binding of biotinylated small-molecule ligands to recombinant His-tagged bromodomains was assessed by time-resolved fluorescence resonance energy transfer (TR-FRET). Test compounds that compete with the biotinylated ligand for bromodomain binding reduce the TR-FRET signal. All biochemical assay protocols were carried out as previously described.<a onclick="showRef(event, 'ref47 ref55'); return false;" href="javascript:void(0);" class="ref ref47 ref55">(47, 55)</a> Normalized TR-FRET inhibition data were plotted as a function of compound concentration, and the resulting curves were fit to a 4-parameter Hill equation to determine the IC<sub>50</sub> values (constraining upper and/or lower boundaries where necessary). If compound inhibition did not reach 50%, then IC<sub>50</sub> values were reported as greater than the highest concentration tested (50 μM for HTS IC<sub>50</sub> determinations and 10 μM for compound optimization). IC<sub>50</sub> values are reported as the geometric mean of between 2 and 8 replicates. In all cases, replicate values fall within 60% of the geometric mean.</div></div><div id="sec5-21" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> Cellular Assay Protocol</h3><div class="NLM_p last">To determine the inhibition of <i>MYC</i> expression, MV-4-11 cells (ATCC) were plated at 10,000 cells per well in 96-well plates in RPMI1640 media supplemented with 10% fetal bovine serum and 2 mM <span class="smallcaps smallerCapital">l</span>-glutamine. Test compounds diluted in DMSO were transferred to the cell plates, keeping final DMSO concentration consistent at 0.1%, and incubated for 4 h at 37 °C. Lysis and analysis for <i>MYC</i> expression were carried out using QuantiGene 2.0 reagents (Affymetrix/eBioscience, probe set cat # SA-50182) and following the vendor’s instructions. Luminescence was read using an EnVision plate reader (PerkinElmer), and EC<sub>50</sub>s were generated using a 4-parameter nonlinear regression fit.</div></div><div id="sec5-22" class="NLM_sec NLM_sec_level_2"><div id="ac_i92" class="anchor-spacer"></div><h3 class="article-section__title" id="_i92"> In Vitro Metabolic Stability Experiments</h3><div class="NLM_p last">Experiments were carried out as previously described.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a></div></div><div id="sec5-23" class="NLM_sec NLM_sec_level_2"><div id="ac_i93" class="anchor-spacer"></div><h3 class="article-section__title" id="_i93"> In Vivo PK of <b>29</b></h3><div class="NLM_p">Six CD-1 mice were obtained from Charles River Laboratory (Hollister, California, USA). All animals were female, 6–9 weeks old at the time of study, and weighed between 20 and 35 g. Animals (<i>n</i> = 3 per dosing route) were dosed with 1 mg/kg <b>29</b> i.v. (in propyl ethylene glycol 400 (35% v/v), dimethyl sulfoxide (10% v/v), and water (55% v/v)) or 5 mg/kg p.o. (suspended in 0.5% w/v methylcellulose, 0.2% w/v Tween 80). Food and water were available ad libitum to all animals. Serial blood samples (15 μL) were collected by tail nick at 0.033, 0.083, 0.25, 0.5, 1, 3, 8, and 24 h after the intravenous administration and 0.083, 0.25, 0.5, 1, 3, 8, and 24 h after the oral administration. All blood samples were diluted with 60 μL of water containing 1.7 mg/mL of K<sub>2</sub>EDTA and kept at −80 °C until analysis. All animal studies were carried out in accordance with the Guide for the Care and Use of Laboratory Animals as adopted and were approved by the Institution’s Animal Care and Use Committee.</div><div class="NLM_p last">Concentrations of <b>29</b> were determined by a nonvalidated LC–MS/MS assay. The diluted blood samples were prepared for analysis by placing a 15 μL aliquot into a 96-well plate followed by the addition of 150 μL of acetonitrile containing an internal standard mixture (labetalol, indomethacin, and loperamide at ∼1 μg/mL). The samples were vortexed and centrifuged at 4000 rpm for 20 min at 4 °C; 100 μL of the supernatant was diluted with 150 μL of water, and 1 μL of the solution was injected onto an analytical column. A Nexera UPLC System (Shimadzu) coupled with a QTrap 6500 mass spectrometer (AB Sciex, Foster City, CA) was used for sample analysis. The mobile phases were 0.1% of formic acid in water (A) and 0.1% formic acid in acetonitrile (B). The gradient was as follows: starting at 10% B and increased to 90% B until 0.6 min, maintained at 90% for 0.2 min, then from 90% B to 10% B for 0.01 min, and maintained at 10% B for 0.2 min. The total flow rate was 0.6 mL/min, and samples were injected onto a Kinetex F5 (50 × 2.1 mm, 2.6 μm) analytical column with a total run time of 1 min. Data were acquired using multiple reactions monitoring (MRM) in positive ion electrospray mode with an operating source temperature of 550 °C. The MRM transition was <i>m</i>/<i>z</i> 479.4 → 439.1 for <b>29</b> and 477.1 → 266.2 for loperamide. The lower and upper limits of quantitation of the assay for <b>29</b> were 0.001 and 69.6 μM, respectively.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i94"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01372">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87442" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87442" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b01372" class="ext-link">10.1021/acs.jmedchem.7b01372</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Full experimental details and characterization for reported compounds, BROMOscan data for <b>27</b> and <b>29</b>, and crystallography methods for <b>3</b>-CBP, <b>6</b>-CBP, <b>16</b>-CBP, and <b>17</b>-CBP (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01372/suppl_file/jm7b01372_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01372/suppl_file/jm7b01372_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01372/suppl_file/jm7b01372_si_001.pdf">jm7b01372_si_001.pdf (332.5 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01372/suppl_file/jm7b01372_si_002.csv">jm7b01372_si_002.csv (2.54 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB codes for the structures of the CBP bromodomain in complex with <b>3</b>, <b>6</b>, <b>16</b>, and <b>17</b> are <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W0E">5W0E</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6AXQ">6AXQ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6AY3">6AY3</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6AY5">6AY5</a>, respectively.</p><div class="testing" data-doi="10.1021/acs.jmedchem.7b01372" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45686" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45686" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sarah M. Bronner</span> - <span class="hlFld-Affiliation affiliation">Genentech,
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4370-6727" title="Orcid link">http://orcid.org/0000-0003-4370-6727</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f391819c9d9d9681dd809281929bb394969d96dd909c9e"><span class="__cf_email__" data-cfemail="a5c7d7cacbcbc0d78bd6c4d7c4cde5c2c0cbc08bc6cac8">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeremy Murray</span> - <span class="hlFld-Affiliation affiliation">Genentech,
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">F. Anthony Romero</span> - <span class="hlFld-Affiliation affiliation">Genentech,
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7917-1459" title="Orcid link">http://orcid.org/0000-0001-7917-1459</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kwong Wah Lai</span> - <span class="hlFld-Affiliation affiliation">Wuxi
Apptec Co., Ltd., 288 Fute Zhong Road, Waigaoqiao Free Trade Zone, Shanghai 200131, People’s
Republic of China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vickie Tsui</span> - <span class="hlFld-Affiliation affiliation">Genentech,
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Patrick Cyr</span> - <span class="hlFld-Affiliation affiliation">Genentech,
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5320-3483" title="Orcid link">http://orcid.org/0000-0001-5320-3483</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Maureen H. Beresini</span> - <span class="hlFld-Affiliation affiliation">Genentech,
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gladys de leon Boenig</span> - <span class="hlFld-Affiliation affiliation">Genentech,
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhongguo Chen</span> - <span class="hlFld-Affiliation affiliation">Wuxi
Apptec Co., Ltd., 288 Fute Zhong Road, Waigaoqiao Free Trade Zone, Shanghai 200131, People’s
Republic of China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Edna F. Choo</span> - <span class="hlFld-Affiliation affiliation">Genentech,
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4200-8302" title="Orcid link">http://orcid.org/0000-0003-4200-8302</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kevin R. Clark</span> - <span class="hlFld-Affiliation affiliation">Genentech,
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Terry D. Crawford</span> - <span class="hlFld-Affiliation affiliation">Genentech,
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4663-4116" title="Orcid link">http://orcid.org/0000-0003-4663-4116</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hariharan Jayaram</span> - <span class="hlFld-Affiliation affiliation">Editas
Medicine, Inc., 11 Hurley
Street, Cambridge, Massachusetts 02141, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Susan Kaufman</span> - <span class="hlFld-Affiliation affiliation">Genentech,
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ruina Li</span> - <span class="hlFld-Affiliation affiliation">Genentech,
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yingjie Li</span> - <span class="hlFld-Affiliation affiliation">Wuxi
Apptec Co., Ltd., 288 Fute Zhong Road, Waigaoqiao Free Trade Zone, Shanghai 200131, People’s
Republic of China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiangpeng Liao</span> - <span class="hlFld-Affiliation affiliation">Wuxi
Apptec Co., Ltd., 288 Fute Zhong Road, Waigaoqiao Free Trade Zone, Shanghai 200131, People’s
Republic of China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaorong Liang</span> - <span class="hlFld-Affiliation affiliation">Genentech,
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wenfeng Liu</span> - <span class="hlFld-Affiliation affiliation">Wuxi
Apptec Co., Ltd., 288 Fute Zhong Road, Waigaoqiao Free Trade Zone, Shanghai 200131, People’s
Republic of China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Justin Ly</span> - <span class="hlFld-Affiliation affiliation">Genentech,
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jonathan Maher</span> - <span class="hlFld-Affiliation affiliation">Genentech,
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John Wai</span> - <span class="hlFld-Affiliation affiliation">Wuxi
Apptec Co., Ltd., 288 Fute Zhong Road, Waigaoqiao Free Trade Zone, Shanghai 200131, People’s
Republic of China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fei Wang</span> - <span class="hlFld-Affiliation affiliation">Wuxi
Apptec Co., Ltd., 288 Fute Zhong Road, Waigaoqiao Free Trade Zone, Shanghai 200131, People’s
Republic of China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Aijun Zheng</span> - <span class="hlFld-Affiliation affiliation">Wuxi
Apptec Co., Ltd., 288 Fute Zhong Road, Waigaoqiao Free Trade Zone, Shanghai 200131, People’s
Republic of China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaoyu Zhu</span> - <span class="hlFld-Affiliation affiliation">Wuxi
Apptec Co., Ltd., 288 Fute Zhong Road, Waigaoqiao Free Trade Zone, Shanghai 200131, People’s
Republic of China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Steven Magnuson</span> - <span class="hlFld-Affiliation affiliation">Genentech,
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i96">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30763" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30763" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank Mengling Wong, Michael Hayes, and Amber Guillen for compound purification. Baiwei Lin, Deven Wang, and Yutao Jiang are acknowledged for analytical support. Grady Howes, Jan Seerveld, Hao Zheng, and Gigi Yuen are also recognized for help with compound management and logistics. This research used resources of the Advanced Photon Source, a U.S. Department of Energy (DOE) Office of Science User Facility operated for the DOE Office of Science by Argonne National Laboratory under Contract No. DE-AC02-06CH11357. We acknowledge the beamline scientists at APS for assistance in data collection and processing</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i97" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i97"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i98" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i98"> Abbreviations Used</h2><tr><td class="NLM_term">Arg</td><td class="NLM_def"><p class="first last">arginine</p></td></tr><tr><td class="NLM_term">Asn</td><td class="NLM_def"><p class="first last">asparagine</p></td></tr><tr><td class="NLM_term">Asp</td><td class="NLM_def"><p class="first last">aspartic acid</p></td></tr><tr><td class="NLM_term">BC</td><td class="NLM_def"><p class="first last">helices B and C</p></td></tr><tr><td class="NLM_term">BrD</td><td class="NLM_def"><p class="first last">bromodomain</p></td></tr><tr><td class="NLM_term">BRD4(1)</td><td class="NLM_def"><p class="first last">bromodomain 1 of the bromodomain-containing protein 4</p></td></tr><tr><td class="NLM_term">BRD9</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 9</p></td></tr><tr><td class="NLM_term">BRET</td><td class="NLM_def"><p class="first last">bioluminescence resonance energy transfer</p></td></tr><tr><td class="NLM_term">BRPF</td><td class="NLM_def"><p class="first last">bromodomain and plant homeodomain finger containing</p></td></tr><tr><td class="NLM_term">CX (X = 5, 7)</td><td class="NLM_def"><p class="first last">carbon X (X = 5, 7)</p></td></tr><tr><td class="NLM_term">CBP</td><td class="NLM_def"><p class="first last">cyclic adenosine monophosphate response element binding protein binding protein</p></td></tr><tr><td class="NLM_term">CECR2</td><td class="NLM_def"><p class="first last">cat eye syndrome chromosome region, candidate 2</p></td></tr><tr><td class="NLM_term">CF<sub>2</sub>H</td><td class="NLM_def"><p class="first last">difluoromethyl</p></td></tr><tr><td class="NLM_term">Cl<sub>hep</sub></td><td class="NLM_def"><p class="first last">predicted hepatic clearance</p></td></tr><tr><td class="NLM_term">CREBP</td><td class="NLM_def"><p class="first last">cyclic adenosine monophosphate response element binding protein binding protein</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">DNA</td><td class="NLM_def"><p class="first last">DNA</p></td></tr><tr><td class="NLM_term">EC<sub>50</sub></td><td class="NLM_def"><p class="first last">half maximal effective concentration</p></td></tr><tr><td class="NLM_term">g</td><td class="NLM_def"><p class="first last">gram</p></td></tr><tr><td class="NLM_term">GNE</td><td class="NLM_def"><p class="first last">Genentech</p></td></tr><tr><td class="NLM_term">HAT</td><td class="NLM_def"><p class="first last">histone acetyl transferase</p></td></tr><tr><td class="NLM_term">IC<sub>50</sub></td><td class="NLM_def"><p class="first last">half maximal inhibitory concentration</p></td></tr><tr><td class="NLM_term">i.v.</td><td class="NLM_def"><p class="first last">intravenous</p></td></tr><tr><td class="NLM_term">H</td><td class="NLM_def"><p class="first last">human</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high-throughput screen</p></td></tr><tr><td class="NLM_term">K<sub>2</sub>EDTA</td><td class="NLM_def"><p class="first last">dipotassium ethylenediaminetetraacetic acid</p></td></tr><tr><td class="NLM_term">KAc</td><td class="NLM_def"><p class="first last">acetylated lysine</p></td></tr><tr><td class="NLM_term">kg</td><td class="NLM_def"><p class="first last">kilogram</p></td></tr><tr><td class="NLM_term">LC–MS/MS</td><td class="NLM_def"><p class="first last">liquid chromatography mass spectrometry and tandem mass spectrometry</p></td></tr><tr><td class="NLM_term">LE</td><td class="NLM_def"><p class="first last">ligand efficiency</p></td></tr><tr><td class="NLM_term">Leu</td><td class="NLM_def"><p class="first last">leucine</p></td></tr><tr><td class="NLM_term">LM</td><td class="NLM_def"><p class="first last">liver microsomes</p></td></tr><tr><td class="NLM_term">LPF</td><td class="NLM_def"><p class="first last">leucine, proline, and phenylalanine</p></td></tr><tr><td class="NLM_term">M</td><td class="NLM_def"><p class="first last">mouse</p></td></tr><tr><td class="NLM_term">MRM</td><td class="NLM_def"><p class="first last">multiple reactions monitoring</p></td></tr><tr><td class="NLM_term">p.o.</td><td class="NLM_def"><p class="first last">per os</p></td></tr><tr><td class="NLM_term">P300</td><td class="NLM_def"><p class="first last">adenoviral E1A binding protein</p></td></tr><tr><td class="NLM_term">Phe</td><td class="NLM_def"><p class="first last">phenylalanine</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetic</p></td></tr><tr><td class="NLM_term">Pro</td><td class="NLM_def"><p class="first last">proline</p></td></tr><tr><td class="NLM_term">PTM</td><td class="NLM_def"><p class="first last">post-translational modification</p></td></tr><tr><td class="NLM_term">R</td><td class="NLM_def"><p class="first last">rat</p></td></tr><tr><td class="NLM_term">rpm</td><td class="NLM_def"><p class="first last">revolutions per minute</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">SBDD</td><td class="NLM_def"><p class="first last">structure-based drug discovery</p></td></tr><tr><td class="NLM_term">THQ</td><td class="NLM_def"><p class="first last">tetrahydroquinoline</p></td></tr><tr><td class="NLM_term">TR-FRET</td><td class="NLM_def"><p class="first last">time-resolved fluorescence energy transfer</p></td></tr><tr><td class="NLM_term">TRIM24</td><td class="NLM_def"><p class="first last">tripartite motif-containing 24</p></td></tr><tr><td class="NLM_term">v/v</td><td class="NLM_def"><p class="first last">volume per volume</p></td></tr><tr><td class="NLM_term">w/v</td><td class="NLM_def"><p class="first last">weight per volumn</p></td></tr><tr><td class="NLM_term">ZA</td><td class="NLM_def"><p class="first last">helices Z and A.</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i99">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70164" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70164" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 62 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Carey, N.</span> <span class="citation_source-book">The Epigenetics Revolution: How Modern Biology is Rewriting Our Understanding of Genetics, Disease, and Inheritance</span>; <span class="NLM_publisher-name">Columbia University Press</span>: <span class="NLM_publisher-loc">New York, NY</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=N.+Carey&title=The+Epigenetics+Revolution%3A+How+Modern+Biology+is+Rewriting+Our+Understanding+of+Genetics%2C+Disease%2C+and+Inheritance"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DCarey%26aufirst%3DN.%26btitle%3DThe%2520Epigenetics%2520Revolution%253A%2520How%2520Modern%2520Biology%2520is%2520Rewriting%2520Our%2520Understanding%2520of%2520Genetics%252C%2520Disease%252C%2520and%2520Inheritance%26pub%3DColumbia%2520University%2520Press%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Carey, N.</span> <span class="citation_source-book">Epigenetics for Drug Discovery</span>; <span class="NLM_publisher-name">Royal Society of Chemistry</span>: <span class="NLM_publisher-loc">United Kingdom</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1039%2F9781782628484" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=N.+Carey&title=Epigenetics+for+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1039%2F9781782628484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1039%252F9781782628484%26sid%3Dliteratum%253Aachs%26aulast%3DCarey%26aufirst%3DN.%26btitle%3DEpigenetics%2520for%2520Drug%2520Discovery%26pub%3DRoyal%2520Society%2520of%2520Chemistry%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Kanherkar, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhatia-Dey, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Csoka, A. B.</span><span> </span><span class="NLM_article-title">Epigenetics Across the Human Lifespan</span> <span class="citation_source-journal">Front. Cell Dev. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">49</span><span class="refDoi"> DOI: 10.3389/fcell.2014.00067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.3389%2Ffcell.2014.00067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=25364756" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A280%3ADC%252BC2M3ltFajtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2014&pages=49&author=R.+R.+Kanherkarauthor=N.+Bhatia-Deyauthor=A.+B.+Csoka&title=Epigenetics+Across+the+Human+Lifespan&doi=10.3389%2Ffcell.2014.00067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetics across the human lifespan</span></div><div class="casAuthors">Kanherkar Riya R; Bhatia-Dey Naina; Csoka Antonei B</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in cell and developmental biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">49</span>
        ISSN:<span class="NLM_cas:issn">2296-634X</span>.
    </div><div class="casAbstract">Epigenetics has the potential to explain various biological phenomena that have heretofore defied complete explication.  This review describes the various types of endogenous human developmental milestones such as birth, puberty, and menopause, as well as the diverse exogenous environmental factors that influence human health, in a chronological epigenetic context.  We describe the entire course of human life from periconception to death and chronologically note all of the potential internal timepoints and external factors that influence the human epigenome.  Ultimately, the environment presents these various factors to the individual that influence the epigenome, and the unique epigenetic and genetic profile of each individual also modulates the specific response to these factors.  During the course of human life, we are exposed to an environment that abounds with a potent and dynamic milieu capable of triggering chemical changes that activate or silence genes.  There is constant interaction between the external and internal environments that is required for normal development and health maintenance as well as for influencing disease load and resistance.  For example, exposure to pharmaceutical and toxic chemicals, diet, stress, exercise, and other environmental factors are capable of eliciting positive or negative epigenetic modifications with lasting effects on development, metabolism and health.  These can impact the body so profoundly as to permanently alter the epigenetic profile of an individual.  We also present a comprehensive new hypothesis of how these diverse environmental factors cause both direct and indirect epigenetic changes and how this knowledge can ultimately be used to improve personalized medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQwSREcQgl8PDUE-swuRqsHfW6udTcc2eZjNWLTfk1fUbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3ltFajtg%253D%253D&md5=a8f37aee83a71cc6a24b016e631dba0b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.3389%2Ffcell.2014.00067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffcell.2014.00067%26sid%3Dliteratum%253Aachs%26aulast%3DKanherkar%26aufirst%3DR.%2BR.%26aulast%3DBhatia-Dey%26aufirst%3DN.%26aulast%3DCsoka%26aufirst%3DA.%2BB.%26atitle%3DEpigenetics%2520Across%2520the%2520Human%2520Lifespan%26jtitle%3DFront.%2520Cell%2520Dev.%2520Biol.%26date%3D2014%26volume%3D2%26spage%3D49%26doi%3D10.3389%2Ffcell.2014.00067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Dawson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Cancer Epigenetics: From Mechanism to Therapy</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">150</span><span class="NLM_x">, </span> <span class="NLM_fpage">12</span><span class="NLM_x">–</span> <span class="NLM_lpage">27</span><span class="refDoi"> DOI: 10.1016/j.cell.2012.06.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1016%2Fj.cell.2012.06.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=22770212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvFGjsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2012&pages=12-27&author=M.+A.+Dawsonauthor=T.+Kouzarides&title=Cancer+Epigenetics%3A+From+Mechanism+to+Therapy&doi=10.1016%2Fj.cell.2012.06.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer Epigenetics: From Mechanism to Therapy</span></div><div class="casAuthors">Dawson, Mark A.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-27</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The epigenetic regulation of DNA-templated processes has been intensely studied over the last 15 years.  DNA methylation, histone modification, nucleosome remodeling, and RNA-mediated targeting regulate many biol. processes that are fundamental to the genesis of cancer.  Here, we present the basic principles behind these epigenetic pathways and highlight the evidence suggesting that their misregulation can culminate in cancer.  This information, along with the promising clin. and preclin. results seen with epigenetic drugs against chromatin regulators, signifies that it is time to embrace the central role of epigenetics in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuyeeliq_34rVg90H21EOLACvtfcHk0ljkt0Gdy-oxkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvFGjsLg%253D&md5=bc0176364205a2e66e1971fe789d36ae</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.06.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.06.013%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DCancer%2520Epigenetics%253A%2520From%2520Mechanism%2520to%2520Therapy%26jtitle%3DCell%26date%3D2012%26volume%3D150%26spage%3D12%26epage%3D27%26doi%3D10.1016%2Fj.cell.2012.06.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Campbell, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tummino, P. J.</span><span> </span><span class="NLM_article-title">Cancer Epigenetics Drug Discovery and Development: the Challenge of Hitting the Mark</span> <span class="citation_source-journal">J. Clin. Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x">, </span> <span class="NLM_fpage">64</span><span class="NLM_x">–</span> <span class="NLM_lpage">69</span><span class="refDoi"> DOI: 10.1172/JCI71605</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1172%2FJCI71605" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=24382391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntFejug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2014&pages=64-69&author=R.+M.+Campbellauthor=P.+J.+Tummino&title=Cancer+Epigenetics+Drug+Discovery+and+Development%3A+the+Challenge+of+Hitting+the+Mark&doi=10.1172%2FJCI71605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer epigenetics drug discovery and development: the challenge of hitting the mark</span></div><div class="casAuthors">Campbell, Robert M.; Tummino, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">64-69</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    
            (<span class="NLM_cas:orgname">American Society for Clinical Investigation</span>)
        </div><div class="casAbstract">A review.  Over the past several years, there has been rapidly expanding evidence of epigenetic dysregulation in cancer, in which histone and DNA modification play a crit. role in tumor growth and survival.  These findings have gained the attention of the drug discovery and development community, and offer the potential for a second generation of cancer epigenetic agents for patients following the approved "first generation" of DNA methylation (e.g., Dacogen, Vidaza) and broad-spectrum HDAC inhibitors (e.g., Vorinostat, Romidepsin).  This review provides an anal. of prospects for discovery and development of novel cancer agents that target epigenetic proteins.  The authors will examine key examples of epigenetic dysregulation in tumors as well as challenges to epigenetic drug discovery with emerging biol. and novel classes of drug targets.  The authors will also highlight recent successes in cancer epigenetics drug discovery and consider important factors for clin. success in this burgeoning area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozwt2VzcttEbVg90H21EOLACvtfcHk0ljkt0Gdy-oxkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntFejug%253D%253D&md5=69773b6781c3a514621d61956479c6aa</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1172%2FJCI71605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI71605%26sid%3Dliteratum%253Aachs%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26atitle%3DCancer%2520Epigenetics%2520Drug%2520Discovery%2520and%2520Development%253A%2520the%2520Challenge%2520of%2520Hitting%2520the%2520Mark%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2014%26volume%3D124%26spage%3D64%26epage%3D69%26doi%3D10.1172%2FJCI71605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Mau, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yung, R.</span><span> </span><span class="NLM_article-title">Potential of Epigenetic Therapies in Non-Cancerous Conditions</span> <span class="citation_source-journal">Front. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">438</span><span class="refDoi"> DOI: 10.3389/fgene.2014.00438</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.3389%2Ffgene.2014.00438" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=25566322" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A280%3ADC%252BC2MvksFSntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=438&author=T.+Mauauthor=R.+Yung&title=Potential+of+Epigenetic+Therapies+in+Non-Cancerous+Conditions&doi=10.3389%2Ffgene.2014.00438"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Potential of epigenetic therapies in non-cancerous conditions</span></div><div class="casAuthors">Mau Theresa; Yung Raymond</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in genetics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">438</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">There has been an explosion of knowledge in the epigenetics field in the past 20 years.  The first epigenetic therapies have arrived in the clinic for cancer treatments.  In contrast, much of the promise of epigenetic therapies for non-cancerous conditions remains in the laboratories.  The current review will focus on the recent progress that has been made in understanding the pathogenic role of epigenetics in immune and inflammatory conditions, and how the knowledge may provide much needed new therapeutic targets for many autoimmune diseases.  Dietary factors are increasingly recognized as potential modifiers of epigenetic marks that can influence health and diseases across generations.  The current epigenomics revolution will almost certainly complement the explosion of personal genetics medicine to help guide treatment decisions and disease risk stratification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS3SpAGegTY3EQ0lWbKiCflfW6udTcc2eaXOqT8CfwVcLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MvksFSntw%253D%253D&md5=5867589cdc4938ce07745f9db164c6fa</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.3389%2Ffgene.2014.00438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffgene.2014.00438%26sid%3Dliteratum%253Aachs%26aulast%3DMau%26aufirst%3DT.%26aulast%3DYung%26aufirst%3DR.%26atitle%3DPotential%2520of%2520Epigenetic%2520Therapies%2520in%2520Non-Cancerous%2520Conditions%26jtitle%3DFront.%2520Genet.%26date%3D2014%26volume%3D5%26spage%3D438%26doi%3D10.3389%2Ffgene.2014.00438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Kelly, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Carvalho, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, P. A.</span><span> </span><span class="NLM_article-title">Epigenetic Modifications as Therapeutic Targets</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">1069</span><span class="NLM_x">–</span> <span class="NLM_lpage">1078</span><span class="refDoi"> DOI: 10.1038/nbt.1678</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1038%2Fnbt.1678" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2010&pages=1069-1078&author=T.+K.+Kellyauthor=D.+D.+De+Carvalhoauthor=P.+A.+Jones&title=Epigenetic+Modifications+as+Therapeutic+Targets&doi=10.1038%2Fnbt.1678"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1678&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1678%26sid%3Dliteratum%253Aachs%26aulast%3DKelly%26aufirst%3DT.%2BK.%26aulast%3DDe%2BCarvalho%26aufirst%3DD.%2BD.%26aulast%3DJones%26aufirst%3DP.%2BA.%26atitle%3DEpigenetic%2520Modifications%2520as%2520Therapeutic%2520Targets%26jtitle%3DNat.%2520Biotechnol.%26date%3D2010%26volume%3D28%26spage%3D1069%26epage%3D1078%26doi%3D10.1038%2Fnbt.1678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Kwa, F. A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thrimawithana, T. R.</span><span> </span><span class="NLM_article-title">Epigenetic Modifications as Potential Therapeutic Targets in Age-Related Macular Degeneration and Diabetic Retinopathy</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1387</span><span class="NLM_x">–</span> <span class="NLM_lpage">1393</span><span class="refDoi"> DOI: 10.1016/j.drudis.2014.03.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1016%2Fj.drudis.2014.03.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=24717156" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmslakt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=1387-1393&author=F.+A.+A.+Kwaauthor=T.+R.+Thrimawithana&title=Epigenetic+Modifications+as+Potential+Therapeutic+Targets+in+Age-Related+Macular+Degeneration+and+Diabetic+Retinopathy&doi=10.1016%2Fj.drudis.2014.03.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic modifications as potential therapeutic targets in age-related macular degeneration and diabetic retinopathy</span></div><div class="casAuthors">Kwa, Faith A. A.; Thrimawithana, Thilini R.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1387-1393</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Recently, aberrant epigenetic modifications have been identified in the pathogenesis of the posterior eye diseases, age-related macular degeneration (AMD) and diabetic retinopathy (DR).  This has led to the development of alternative therapies that can alter aberrant chromatin-remodelling processes involved in AMD and DR.  These novel therapeutic agents could help to ameliorate the challenges assocd. with current treatments that are limited by variable patient response and disease heterogeneity.  However, research on the use of epigenetic-based therapies in these diseases is relatively young and, therefore, preclin. studies that evaluate their mechanism of action, specificity and adverse effects are warranted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4TpoIQNOL5bVg90H21EOLACvtfcHk0lgwJJO5Wzx5wQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmslakt78%253D&md5=e0bfab23baa28258e85743fbf3b54c7c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.03.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.03.026%26sid%3Dliteratum%253Aachs%26aulast%3DKwa%26aufirst%3DF.%2BA.%2BA.%26aulast%3DThrimawithana%26aufirst%3DT.%2BR.%26atitle%3DEpigenetic%2520Modifications%2520as%2520Potential%2520Therapeutic%2520Targets%2520in%2520Age-Related%2520Macular%2520Degeneration%2520and%2520Diabetic%2520Retinopathy%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2014%26volume%3D19%26spage%3D1387%26epage%3D1393%26doi%3D10.1016%2Fj.drudis.2014.03.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Epigenetic Protein Families: a New Frontier for Drug Discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">384</span><span class="NLM_x">–</span> <span class="NLM_lpage">400</span><span class="refDoi"> DOI: 10.1038/nrd3674</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1038%2Fnrd3674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=22498752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=384-400&author=C.+H.+Arrowsmithauthor=C.+Bountraauthor=P.+V.+Fishauthor=K.+Leeauthor=M.+Schapira&title=Epigenetic+Protein+Families%3A+a+New+Frontier+for+Drug+Discovery&doi=10.1038%2Fnrd3674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic protein families: a new frontier for drug discovery</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Bountra, Chas; Fish, Paul V.; Lee, Kevin; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">384-400</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetic regulation of gene expression is a dynamic and reversible process that establishes normal cellular phenotypes but also contributes to human diseases.  At the mol. level, epigenetic regulation involves hierarchical covalent modification of DNA and the proteins that package DNA, such as histones.  Here, we review the key protein families that mediate epigenetic signalling through the acetylation and methylation of histones, including histone deacetylases, protein methyltransferases, lysine demethylases, bromodomain-contg. proteins and proteins that bind to methylated histones.  These protein families are emerging as druggable classes of enzymes and druggable classes of protein-protein interaction domains.  In this article, we discuss the known links with disease, basic mol. mechanisms of action and recent progress in the pharmacol. modulation of each class of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcD99eCigNtLVg90H21EOLACvtfcHk0lgwJJO5Wzx5wQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D&md5=eb82034466a43107aa74a18ccf6d29f5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrd3674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3674%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DEpigenetic%2520Protein%2520Families%253A%2520a%2520New%2520Frontier%2520for%2520Drug%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D384%26epage%3D400%26doi%3D10.1038%2Fnrd3674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Gnyszka, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jastrzebski, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flis, S.</span><span> </span><span class="NLM_article-title">DNA Methyltransferase Inhibitors and Their Emerging Role in Epigenetic Therapy of Cancer</span> <span class="citation_source-journal">Anticancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">2989</span><span class="NLM_x">–</span> <span class="NLM_lpage">2996</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=23898051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVaqsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2013&pages=2989-2996&author=A.+Gnyszkaauthor=Z.+Jastrzebskiauthor=S.+Flis&title=DNA+Methyltransferase+Inhibitors+and+Their+Emerging+Role+in+Epigenetic+Therapy+of+Cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">DNA methyltransferase inhibitors and their emerging role in epigenetic therapy of cancer</span></div><div class="casAuthors">Gnyszka, Agnieszka; Jastrzebski, Zenon; Flis, Sylwia</div><div class="citationInfo"><span class="NLM_cas:title">Anticancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2989-2996</span>CODEN:
                <span class="NLM_cas:coden">ANTRD4</span>;
        ISSN:<span class="NLM_cas:issn">0250-7005</span>.
    
            (<span class="NLM_cas:orgname">International Institute of Anticancer Research</span>)
        </div><div class="casAbstract">A review.  The DNA methyltransferase (DNMT) inhibitors azacytidine and decitabine are the most successful epigenetic drugs to date and are still the most widely used as epigenetic modulators, even though their application for oncol. diseases is restricted by their relative toxicity and poor chem. stability.  Zebularine (1-(β-D-ribofuranosyl)-1,2-dihydropyrimidin-2-one), a more stable and less toxic cytidine analog, is another inhibitor of DNMT with concomitant inhibitory activity towards cytidine deaminase.  Unfortunately, there is no new information related to the possible clin. applications of zebularine.  Although many new inhibitors of DNMT have been identified, none of them can so far replace azacytidine, decitabine and, to a lesser degree, zebularine.  This review summarizes the current data and knowledge about azacytidine, decitabine and zebularine, and their role in present and possible future epigenetic cancer therapy.  We also discuss the mol. modes of action of these agents with consideration of their different toxicities and demethylation profiles, reflecting their complex and partially overlapping biol. effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPiSuC0_kryrVg90H21EOLACvtfcHk0lgwJJO5Wzx5wQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVaqsLnP&md5=9540e414d3a3291856d8378f3da2292d</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGnyszka%26aufirst%3DA.%26aulast%3DJastrzebski%26aufirst%3DZ.%26aulast%3DFlis%26aufirst%3DS.%26atitle%3DDNA%2520Methyltransferase%2520Inhibitors%2520and%2520Their%2520Emerging%2520Role%2520in%2520Epigenetic%2520Therapy%2520of%2520Cancer%26jtitle%3DAnticancer%2520Res.%26date%3D2013%26volume%3D33%26spage%3D2989%26epage%3D2996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Szyf, M.</span><span> </span><span class="NLM_article-title">The Implications of DNA Methylation for Toxicology: Toward Toxicomethylomics, the Toxicology of DNA Methylation</span> <span class="citation_source-journal">Toxicol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">120</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span><span class="refDoi"> DOI: 10.1093/toxsci/kfr024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1093%2Ftoxsci%2Fkfr024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=21297083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjvFejt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2011&pages=235-255&author=M.+Szyf&title=The+Implications+of+DNA+Methylation+for+Toxicology%3A+Toward+Toxicomethylomics%2C+the+Toxicology+of+DNA+Methylation&doi=10.1093%2Ftoxsci%2Fkfr024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The Implications of DNA Methylation for Toxicology: Toward Toxicomethylomics, the Toxicology of DNA Methylation</span></div><div class="casAuthors">Szyf, Moshe</div><div class="citationInfo"><span class="NLM_cas:title">Toxicological Sciences</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">235-255</span>CODEN:
                <span class="NLM_cas:coden">TOSCF2</span>;
        ISSN:<span class="NLM_cas:issn">1096-6080</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Identifying agents that have a long-term deleterious impact on health but exhibit no immediate toxicity is of prime importance.  It is well established that long-term toxicity of chems. could be caused by their ability to generate changes in the DNA sequence through the process of mutagenesis.  Several assays including the Ames test and its different modifications were developed to assess the mutagenic potential of chems.  These tests have also been employed for assessing the carcinogenic potential of compds.  However, the DNA mol. contains within its chem. structure 2 layers of information: the DNA sequence that bears the ancestral genetic information and the pattern of distribution of covalently bound Me groups on cytosines in DNA.  DNA methylation patterns are generated by an innate program during gestation but are attuned to the environment in utero and throughout life including phys. and social exposures.  DNA function and health could be stably altered by exposure to environmental agents without changing the sequence, just by changing the state of DNA methylation.  The current screening tests do not detect agents that have long-range impact on the phenotype without altering the genotype.  The realization that long-range damage could be caused without changing the DNA sequence has important implications on the way the safety of chems., drugs, and food is assessed and broadens the scope of definition of toxic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVjRfwmFuN6bVg90H21EOLACvtfcHk0ljJj044I9N-Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjvFejt7g%253D&md5=e8d16d77f41868174f4f2abb7eebb8c6</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1093%2Ftoxsci%2Fkfr024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Ftoxsci%252Fkfr024%26sid%3Dliteratum%253Aachs%26aulast%3DSzyf%26aufirst%3DM.%26atitle%3DThe%2520Implications%2520of%2520DNA%2520Methylation%2520for%2520Toxicology%253A%2520Toward%2520Toxicomethylomics%252C%2520the%2520Toxicology%2520of%2520DNA%2520Methylation%26jtitle%3DToxicol.%2520Sci.%26date%3D2011%26volume%3D120%26spage%3D235%26epage%3D255%26doi%3D10.1093%2Ftoxsci%2Fkfr024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Cedar, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergman, Y.</span><span> </span><span class="NLM_article-title">Linking DNA Methylation and Histone Modification: Patterns and Paradigms</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">295</span><span class="NLM_x">–</span> <span class="NLM_lpage">304</span><span class="refDoi"> DOI: 10.1038/nrg2540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1038%2Fnrg2540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=19308066" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksF2rsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=295-304&author=H.+Cedarauthor=Y.+Bergman&title=Linking+DNA+Methylation+and+Histone+Modification%3A+Patterns+and+Paradigms&doi=10.1038%2Fnrg2540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Linking DNA methylation and histone modification: patterns and paradigms</span></div><div class="casAuthors">Cedar, Howard; Bergman, Yehudit</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">295-304</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  DNA methylation and histone modification are important for regulating gene expression and chromatin structure.  New evidence suggests there are multiple levels of crosstalk between these marks.  Understanding this crosstalk will be important for elucidating the role of epigenetics in development and disease.  Both DNA methylation and histone modification are involved in establishing patterns of gene repression during development.  Certain forms of histone methylation cause local formation of heterochromatin, which is readily reversible, whereas DNA methylation leads to stable long-term repression.  It has recently become apparent that DNA methylation and histone modification pathways can be dependent on one another, and that this crosstalk can be mediated by biochem. interactions between SET domain histone methyltransferases and DNA methyltransferases.  Relationships between DNA methylation and histone modification have implications for understanding normal development as well as somatic cell reprogramming and tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbo2-3KvKYDrVg90H21EOLACvtfcHk0ljJj044I9N-Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksF2rsL0%253D&md5=407a040cbccd1afe6cdfbba5dc554a61</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrg2540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg2540%26sid%3Dliteratum%253Aachs%26aulast%3DCedar%26aufirst%3DH.%26aulast%3DBergman%26aufirst%3DY.%26atitle%3DLinking%2520DNA%2520Methylation%2520and%2520Histone%2520Modification%253A%2520Patterns%2520and%2520Paradigms%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2009%26volume%3D10%26spage%3D295%26epage%3D304%26doi%3D10.1038%2Fnrg2540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Verdone, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caserta, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Mauro, E.</span><span> </span><span class="NLM_article-title">Role of Histone Acetylation in the Control of Gene Expression</span> <span class="citation_source-journal">Biochem. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">344</span><span class="NLM_x">–</span> <span class="NLM_lpage">353</span><span class="refDoi"> DOI: 10.1139/o05-041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1139%2Fo05-041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=15959560" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BD2MXntFClsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2005&pages=344-353&author=L.+Verdoneauthor=M.+Casertaauthor=E.+Di+Mauro&title=Role+of+Histone+Acetylation+in+the+Control+of+Gene+Expression&doi=10.1139%2Fo05-041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Role of histone acetylation in the control of gene expression</span></div><div class="casAuthors">Verdone, Loredana; Caserta, Micaela; Di Mauro, Ernesto</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry and Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">344-353</span>CODEN:
                <span class="NLM_cas:coden">BCBIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0829-8211</span>.
    
            (<span class="NLM_cas:orgname">National Research Council of Canada</span>)
        </div><div class="casAbstract">A review.  Histone proteins play structural and functional roles in all nuclear processes.  They undergo different types of covalent modifications, defined in their ensemble as epigenetic because changes in DNA sequences are not involved.  Histone acetylation emerges as a central switch that allows interconversion between permissive and repressive chromatin domains in terms of transcriptional competence.  The mechanisms underlying the histone acetylation-dependent control of gene expression include a direct effect on the stability of nucleosomal arrays and the creation of docking sites for the binding of regulatory proteins.  Histone acetyltransferases and deacetylases are, resp., the enzymes devoted to the addn. and removal of acetyl groups from lysine residues on the histone N-terminal tails.  The enzymes exert fundamental roles in developmental processes and their deregulation has been linked to the progression of diverse human disorders, including cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra07s9zdBpF7Vg90H21EOLACvtfcHk0ljJj044I9N-Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXntFClsbo%253D&md5=4f3d8b79f8cac35e7b0b553028ce96c5</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1139%2Fo05-041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1139%252Fo05-041%26sid%3Dliteratum%253Aachs%26aulast%3DVerdone%26aufirst%3DL.%26aulast%3DCaserta%26aufirst%3DM.%26aulast%3DDi%2BMauro%26aufirst%3DE.%26atitle%3DRole%2520of%2520Histone%2520Acetylation%2520in%2520the%2520Control%2520of%2520Gene%2520Expression%26jtitle%3DBiochem.%2520Cell%2520Biol.%26date%3D2005%26volume%3D83%26spage%3D344%26epage%3D353%26doi%3D10.1139%2Fo05-041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Shahbazian, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grunstein, M.</span><span> </span><span class="NLM_article-title">Functions of Site-Specific Histone Acetylation and Deacetylation</span> <span class="citation_source-journal">Annu. Rev. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">76</span><span class="NLM_x">, </span> <span class="NLM_fpage">75</span><span class="NLM_x">–</span> <span class="NLM_lpage">100</span><span class="refDoi"> DOI: 10.1146/annurev.biochem.76.052705.162114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1146%2Fannurev.biochem.76.052705.162114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=17362198" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVehtb7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2007&pages=75-100&author=M.+D.+Shahbazianauthor=M.+Grunstein&title=Functions+of+Site-Specific+Histone+Acetylation+and+Deacetylation&doi=10.1146%2Fannurev.biochem.76.052705.162114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Functions of site-specific histone acetylation and deacetylation</span></div><div class="casAuthors">Shahbazian, Mona D.; Grunstein, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">75-100</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Histone acetylation regulates many cellular processes, and specific acetylation marks, either singly or in combination, produce distinct outcomes.  For example, the acetylation pattern on newly synthesized histones is important for their assembly into nucleosomes by histone chaperones.  Addnl., the degree of chromatin compaction and folding may be regulated by acetylation of histone H4 at Lys-16.  Histone acetylation also regulates the formation of heterochromatin; deacetylation of H4 Lys-16 is important for the spreading of heterochromatin components, whereas acetylation of this site serves as a barrier to this spreading.  Finally, histone acetylation is crit. for gene transcription, but recent results suggest that deacetylation of certain sites also plays an important role.  There are many histone acetyltransferases (HATs) and deacetylases, with differing preferences for the various histone proteins and for specific sites on individual histones.  Detg. how these enzymes create distinct acetylation patterns and regulate the functional outcome is an important challenge.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2iL3QDzZh-7Vg90H21EOLACvtfcHk0ljJj044I9N-Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVehtb7O&md5=51fb46ba44a7f0c614c812baddb4e65e</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.76.052705.162114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.76.052705.162114%26sid%3Dliteratum%253Aachs%26aulast%3DShahbazian%26aufirst%3DM.%2BD.%26aulast%3DGrunstein%26aufirst%3DM.%26atitle%3DFunctions%2520of%2520Site-Specific%2520Histone%2520Acetylation%2520and%2520Deacetylation%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2007%26volume%3D76%26spage%3D75%26epage%3D100%26doi%3D10.1146%2Fannurev.biochem.76.052705.162114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Grunstein, M.</span><span> </span><span class="NLM_article-title">Histone Acetylation in Chromatin Structure and Transcription</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">389</span><span class="NLM_x">, </span> <span class="NLM_fpage">349</span><span class="NLM_x">–</span> <span class="NLM_lpage">352</span><span class="refDoi"> DOI: 10.1038/38664</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1038%2F38664" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=9311776" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADyaK2sXmsVWntLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=389&publication_year=1997&pages=349-352&author=M.+Grunstein&title=Histone+Acetylation+in+Chromatin+Structure+and+Transcription&doi=10.1038%2F38664"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Histone acetylation in chromatin structure and transcription</span></div><div class="casAuthors">Grunstein, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">389</span>
        (<span class="NLM_cas:issue">6649</span>),
    <span class="NLM_cas:pages">349-352</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">A review with 75 refs.  The amino termini of histones extend from the nucleosomal core and are modified by acetyltransferase and deacetylases during the cell cycle.  These acetylation patterns may direct histone assembly and help regulate the unfolding and activity of genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodv2Ds8rzh-rVg90H21EOLACvtfcHk0lgRO0FmF_Iytw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmsVWntLk%253D&md5=cdd287ff2d87e9c89944366b71aaf6d9</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2F38664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F38664%26sid%3Dliteratum%253Aachs%26aulast%3DGrunstein%26aufirst%3DM.%26atitle%3DHistone%2520Acetylation%2520in%2520Chromatin%2520Structure%2520and%2520Transcription%26jtitle%3DNature%26date%3D1997%26volume%3D389%26spage%3D349%26epage%3D352%26doi%3D10.1038%2F38664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Translating the Histone Code</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">293</span><span class="NLM_x">, </span> <span class="NLM_fpage">1074</span><span class="NLM_x">–</span> <span class="NLM_lpage">1080</span><span class="refDoi"> DOI: 10.1126/science.1063127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1126%2Fscience.1063127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=11498575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2001&pages=1074-1080&author=T.+Jenuweinauthor=C.+D.+Allis&title=Translating+the+Histone+Code&doi=10.1126%2Fscience.1063127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Translating the histone code</span></div><div class="casAuthors">Jenuwein, Thomas; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">5532</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The review.  Chromatin, the physiol. template of all eukaryotic genetic information, is subject to a diverse array of posttranslational modifications that largely impinge on histone amino termini, thereby regulating access to the underlying DNA.  Distinct histone amino-terminal modifications can generate synergistic or antagonistic interaction affinities for chromatin-assocd. proteins, which in turn dictate dynamic transitions between transcriptionally active or transcriptionally silent chromatin states.  The combinatorial nature of histone amino-terminal modifications thus reveals a "histone code" that considerably extends the information potential of the genetic code.  We propose that this epigenetic marking system represents a fundamental regulatory mechanism that has an impact on most, if not all; chromatin-templated processes, with far-reaching consequences for cell fate decisions and both normal and pathol. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUFyHNepKcUbVg90H21EOLACvtfcHk0lgRO0FmF_Iytw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D&md5=021dcd98a5e5bf2ed7eedef1897b2269</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1126%2Fscience.1063127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1063127%26sid%3Dliteratum%253Aachs%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DTranslating%2520the%2520Histone%2520Code%26jtitle%3DScience%26date%3D2001%26volume%3D293%26spage%3D1074%26epage%3D1080%26doi%3D10.1126%2Fscience.1063127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Strahl, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">The Language of Covalent Histone Modifications</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">403</span><span class="NLM_x">, </span> <span class="NLM_fpage">41</span><span class="NLM_x">–</span> <span class="NLM_lpage">45</span><span class="refDoi"> DOI: 10.1038/47412</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1038%2F47412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10638745" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A280%3ADC%252BD3c7gt1arsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=403&publication_year=2000&pages=41-45&author=B.+D.+Strahlauthor=C.+D.+Allis&title=The+Language+of+Covalent+Histone+Modifications&doi=10.1038%2F47412"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The language of covalent histone modifications</span></div><div class="casAuthors">Strahl B D; Allis C D</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">403</span>
        (<span class="NLM_cas:issue">6765</span>),
    <span class="NLM_cas:pages">41-5</span>
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Histone proteins and the nucleosomes they form with DNA are the fundamental building blocks of eukaryotic chromatin.  A diverse array of post-translational modifications that often occur on tail domains of these proteins has been well documented.  Although the function of these highly conserved modifications has remained elusive, converging biochemical and genetic evidence suggests functions in several chromatin-based processes.  We propose that distinct histone modifications, on one or more tails, act sequentially or in combination to form a 'histone code' that is, read by other proteins to bring about distinct downstream events.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSyzhdlYh9sX7t7lRqEFUJVfW6udTcc2eZMMwt9Bc2zzbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3c7gt1arsQ%253D%253D&md5=2eb7d093151c6122ccd70fa2adcfd94b</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2F47412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F47412%26sid%3Dliteratum%253Aachs%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DThe%2520Language%2520of%2520Covalent%2520Histone%2520Modifications%26jtitle%3DNature%26date%3D2000%26volume%3D403%26spage%3D41%26epage%3D45%26doi%3D10.1038%2F47412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Dawson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kouzarides, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huntly, B. J. P.</span><span> </span><span class="NLM_article-title">Targeting Epigenetic Readers in Cancer</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">367</span><span class="NLM_x">, </span> <span class="NLM_fpage">647</span><span class="NLM_x">–</span> <span class="NLM_lpage">657</span><span class="refDoi"> DOI: 10.1056/NEJMra1112635</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1056%2FNEJMra1112635" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=22894577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Chu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=367&publication_year=2012&pages=647-657&author=M.+A.+Dawsonauthor=T.+Kouzaridesauthor=B.+J.+P.+Huntly&title=Targeting+Epigenetic+Readers+in+Cancer&doi=10.1056%2FNEJMra1112635"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting epigenetic readers in cancer</span></div><div class="casAuthors">Dawson, Mark A.; Kouzarides, Tony; Huntly, Brian J. P.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">367</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">647-657</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review on assocn. between epigenetics and chromatin biol., BET bromodomain inhibitors, and role of BET inhibitors in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnz_YQy6Z2T7Vg90H21EOLACvtfcHk0lgRO0FmF_Iytw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Chu77F&md5=f5e1e82c8a1105f798b1f7719138721c</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1056%2FNEJMra1112635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra1112635%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%2BP.%26atitle%3DTargeting%2520Epigenetic%2520Readers%2520in%2520Cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2012%26volume%3D367%26spage%3D647%26epage%3D657%26doi%3D10.1056%2FNEJMra1112635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Targeting Bromodomains: Epigenetic Readers of Lysine Acetylation</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">356</span><span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+Bromodomains%3A+Epigenetic+Readers+of+Lysine+Acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0lje0s6T7JgkPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520Bromodomains%253A%2520Epigenetic%2520Readers%2520of%2520Lysine%2520Acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Romero, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crawford, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsui, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Côté, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magnuson, S.</span><span> </span><span class="NLM_article-title">Disrupting Acetyl-Lysine Recognition: Progress in the Development of Bromodomain Inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">1271</span><span class="NLM_x">–</span> <span class="NLM_lpage">1298</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01514</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01514" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1271-1298&author=F.+A.+Romeroauthor=A.+M.+Taylorauthor=T.+D.+Crawfordauthor=V.+Tsuiauthor=A.+C%C3%B4t%C3%A9author=S.+Magnuson&title=Disrupting+Acetyl-Lysine+Recognition%3A+Progress+in+the+Development+of+Bromodomain+Inhibitors&doi=10.1021%2Facs.jmedchem.5b01514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting Acetyl-Lysine Recognition: Progress in the Development of Bromodomain Inhibitors</span></div><div class="casAuthors">Romero, F. Anthony; Taylor, Alexander M.; Crawford, Terry D.; Tsui, Vickie; Cote, Alexandre; Magnuson, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1271-1298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains, small protein modules that recognize acetylated lysine on histones, play a significant role in the epigenome, where they function as "readers" that ultimately det. the functional outcome of the post-translational modification.  Because the initial discovery of selective BET inhibitors have helped define the role of that protein family in oncol. and inflammation, BET bromodomains have continued to garner the most attention of any other bromodomain.  More recently, non-BET bromodomain inhibitors that are potent and selective have been disclosed for ATAD2, CBP, BRD7/9, BRPF, BRPF/TRIM24, CECR2, SMARCA4, and BAZ2A/B.  Such novel inhibitors can be used to probe the physiol. function of these non-BET bromodomains and further understanding of their role in certain disease states.  Here, we provide an update to the progress in identifying selective bromodomain inhibitors and their use as biol. tools, as well as our perspective on the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgbRrKGlHaVLVg90H21EOLACvtfcHk0lje0s6T7JgkPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO&md5=260352f61301aa88aad48c0b63e6152f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01514%26sid%3Dliteratum%253Aachs%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DDisrupting%2520Acetyl-Lysine%2520Recognition%253A%2520Progress%2520in%2520the%2520Development%2520of%2520Bromodomain%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1271%26epage%3D1298%26doi%3D10.1021%2Facs.jmedchem.5b01514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Sanchez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meslamani, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M.-M.</span><span> </span><span class="NLM_article-title">The Bromodomain: From Epigenome Reader to Druggable Target</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gene Regul. Mech.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">1839</span><span class="NLM_x">, </span> <span class="NLM_fpage">676</span><span class="NLM_x">–</span> <span class="NLM_lpage">685</span><span class="refDoi"> DOI: 10.1016/j.bbagrm.2014.03.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1016%2Fj.bbagrm.2014.03.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=24686119" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtFagtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1839&publication_year=2014&pages=676-685&author=R.+Sanchezauthor=J.+Meslamaniauthor=M.-M.+Zhou&title=The+Bromodomain%3A+From+Epigenome+Reader+to+Druggable+Target&doi=10.1016%2Fj.bbagrm.2014.03.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain: From epigenome reader to druggable target</span></div><div class="casAuthors">Sanchez, Roberto; Meslamani, Jamel; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1839</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">676-685</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a fundamental post-translational modification that plays an important role in the control of gene transcription in chromatin in an ordered fashion.  The bromodomain, the conserved structural module present in transcription-assocd. proteins, functions exclusively to recognize acetyl-lysine on histones and non-histone proteins.  The structural analyses of bromodomains' recognition of lysine-acetylated peptides derived from histones and cellular proteins provide detailed insights into the differences and unifying features of biol. ligand binding selectivity by the bromodomains.  Newly developed small-mol. inhibitors targeting bromodomain proteins further highlight the functional importance of bromodomain/acetyl-lysine binding as a key mechanism in orchestrating mol. interactions and regulation in chromatin biol. and gene transcription.  These new studies argue that modulating bromodomain/acetyl-lysine interactions with small-mol. chems. offer new opportunities to control gene expression in a wide array of human diseases including cancer and inflammation.  This article is part of a Special Issue entitled: Mol. mechanisms of histone modification function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO3Au7jfK0ibVg90H21EOLACvtfcHk0lhq_IbWt7V3yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtFagtrw%253D&md5=95ec21cf13231d9e63bce9b3ccdd6d6f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2014.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2014.03.011%26sid%3Dliteratum%253Aachs%26aulast%3DSanchez%26aufirst%3DR.%26aulast%3DMeslamani%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DThe%2520Bromodomain%253A%2520From%2520Epigenome%2520Reader%2520to%2520Druggable%2520Target%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2014%26volume%3D1839%26spage%3D676%26epage%3D685%26doi%3D10.1016%2Fj.bbagrm.2014.03.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Gallenkamp, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gelato, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haendler, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinmann, H.</span><span> </span><span class="NLM_article-title">Bromodomains and Their Pharmacological Inhibitors</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">438</span><span class="NLM_x">–</span> <span class="NLM_lpage">464</span><span class="refDoi"> DOI: 10.1002/cmdc.201300434</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1002%2Fcmdc.201300434" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=24497428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsl2gtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=438-464&author=D.+Gallenkampauthor=K.+A.+Gelatoauthor=B.+Haendlerauthor=H.+Weinmann&title=Bromodomains+and+Their+Pharmacological+Inhibitors&doi=10.1002%2Fcmdc.201300434"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomains and Their Pharmacological Inhibitors</span></div><div class="casAuthors">Gallenkamp, Daniel; Gelato, Kathy A.; Haendler, Bernard; Weinmann, Hilmar</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">438-464</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Over 60 bromodomains belonging to proteins with very different functions have been identified in humans.  Several of them interact with acetylated lysine residues, leading to the recruitment and stabilization of protein complexes.  The bromodomain and extra-terminal domain (BET) proteins contain tandem bromodomains which bind to acetylated histones and are thereby implicated in a no. of DNA-centered processes, including the regulation of gene expression.  The recent identification of inhibitors of BET and non-BET bromodomains is one of the few examples in which effective blockade of a protein-protein interaction can be achieved with a small mol.  This has led to major strides in the understanding of the function of bromodomain-contg. proteins and their involvement in diseases such as cancer and inflammation.  Indeed, BET bromodomain inhibitors are now being clin. evaluated for the treatment of hematol. tumors and have also been tested in clin. trials for the relatively rare BRD-NUT midline carcinoma.  This review gives an overview of the newest developments in the field, with a focus on the biol. of selected bromodomain proteins on the one hand, and on reported pharmacol. inhibitors on the other, including recent examples from the patent literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon-8_ymytCPbVg90H21EOLACvtfcHk0lhq_IbWt7V3yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsl2gtL4%253D&md5=2ce7c6b5c37001fb8fb901fe04aeb606</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300434%26sid%3Dliteratum%253Aachs%26aulast%3DGallenkamp%26aufirst%3DD.%26aulast%3DGelato%26aufirst%3DK.%2BA.%26aulast%3DHaendler%26aufirst%3DB.%26aulast%3DWeinmann%26aufirst%3DH.%26atitle%3DBromodomains%2520and%2520Their%2520Pharmacological%2520Inhibitors%26jtitle%3DChemMedChem%26date%3D2014%26volume%3D9%26spage%3D438%26epage%3D464%26doi%3D10.1002%2Fcmdc.201300434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Dekker, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haisma, H. J.</span><span> </span><span class="NLM_article-title">Histone Acetyl Transferases as Emerging Drug Targets</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">942</span><span class="NLM_x">–</span> <span class="NLM_lpage">948</span><span class="refDoi"> DOI: 10.1016/j.drudis.2009.06.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1016%2Fj.drudis.2009.06.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=19577000" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFygs7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=942-948&author=F.+J.+Dekkerauthor=H.+J.+Haisma&title=Histone+Acetyl+Transferases+as+Emerging+Drug+Targets&doi=10.1016%2Fj.drudis.2009.06.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Histone acetyl transferases as emerging drug targets</span></div><div class="casAuthors">Dekker, Frank J.; Haisma, Hidde J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">19/20</span>),
    <span class="NLM_cas:pages">942-948</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications, such as acetylation or phosphorylation, play a crucial role in the regulation of gene transcription in eukaryotes.  Different subtypes of histone acetyl transferases (HATs) catalyze the acetylation of histones on specific lysine residues.  A potential role of HATs in the pathol. of cancer, asthma, COPD and viral infection has been described.  This indicates that specific HAT inhibitors are potential tools for pharmacol. research and might find therapeutic applications.  This review focuses on the role of the HATs p300, CBP, PCAF and GCN5 in different diseases and the development of small-mol. inhibitors of these enzymes as potential drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1Ao3F8nEy7bVg90H21EOLACvtfcHk0lhq_IbWt7V3yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFygs7fL&md5=bff2f4e6d437e003e5b17cfb07215676</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2009.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2009.06.008%26sid%3Dliteratum%253Aachs%26aulast%3DDekker%26aufirst%3DF.%2BJ.%26aulast%3DHaisma%26aufirst%3DH.%2BJ.%26atitle%3DHistone%2520Acetyl%2520Transferases%2520as%2520Emerging%2520Drug%2520Targets%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2009%26volume%3D14%26spage%3D942%26epage%3D948%26doi%3D10.1016%2Fj.drudis.2009.06.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Bannister, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">The CBP Co-Activator is a Histone Acetyltransferase</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">384</span><span class="NLM_x">, </span> <span class="NLM_fpage">641</span><span class="NLM_x">–</span> <span class="NLM_lpage">643</span><span class="refDoi"> DOI: 10.1038/384641a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1038%2F384641a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=8967953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADyaK2sXlsVen" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=1996&pages=641-643&author=A.+J.+Bannisterauthor=T.+Kouzarides&title=The+CBP+Co-Activator+is+a+Histone+Acetyltransferase&doi=10.1038%2F384641a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The CBP co-activator is a histone acetyltransferase</span></div><div class="casAuthors">Bannister, Andrew J.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">384</span>
        (<span class="NLM_cas:issue">6610</span>),
    <span class="NLM_cas:pages">641-643</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">The CBP protein acts as a transcriptional adaptor for many different transcription factors by directly contacting DNA-bound activators.  One mechanism by which CBP is thought to stimulate transcription is by recruiting the histone acetyltransferase (HAT) P/CAF to the promoter.  Here we show that CBP has intrinsic HAT activity.  The HAT domain of CBP is adjacent to the binding site for the transcriptional activator E1A.  Although E1A displaces P/CAF from CBP, is does not disrupt the CBP-assocd. HAT activity.  Thus E1A carries HAT activity when complexed with CBP.  Targeting CBP-assocd. HAT activity to specific promoters may therefore be a mechanism by which E1A acts as a transcriptional activator.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiUdYJTsdfILVg90H21EOLACvtfcHk0lgbR2ebVFGNWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlsVen&md5=8c6fd1c0c0679a454a6c10c704200e2f</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2F384641a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F384641a0%26sid%3Dliteratum%253Aachs%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DThe%2520CBP%2520Co-Activator%2520is%2520a%2520Histone%2520Acetyltransferase%26jtitle%3DNature%26date%3D1996%26volume%3D384%26spage%3D641%26epage%3D643%26doi%3D10.1038%2F384641a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Dancy, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span> </span><span class="NLM_article-title">Protein Lysine Acetylation by p300/CBP</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">115</span><span class="NLM_x">, </span> <span class="NLM_fpage">2419</span><span class="NLM_x">–</span> <span class="NLM_lpage">2452</span><span class="refDoi"> DOI: 10.1021/cr500452k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr500452k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVCiu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2015&pages=2419-2452&author=B.+M.+Dancyauthor=P.+A.+Cole&title=Protein+Lysine+Acetylation+by+p300%2FCBP&doi=10.1021%2Fcr500452k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Lysine Acetylation by p300/CBP</span></div><div class="casAuthors">Dancy, Beverley M.; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2419-2452</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The human protein lysine acetyltransferase encoded by the paralogous p300 and CBP genes is involved in regulating the expression of genes.  In this capacity, p300/CBP controls several basic cellular processes such as proliferation and homeostasis, and plays a role in a variety of human diseases, particularly solid tumors.  This review summarizes information related to the current understanding of p300/CBP, including the novel technologies that have co-evolved with p300/CBP research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYgTImMiuo7rVg90H21EOLACvtfcHk0lgbR2ebVFGNWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVCiu7Y%253D&md5=40a46cf68990f4a7d11a338c54081628</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fcr500452k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr500452k%26sid%3Dliteratum%253Aachs%26aulast%3DDancy%26aufirst%3DB.%2BM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DProtein%2520Lysine%2520Acetylation%2520by%2520p300%252FCBP%26jtitle%3DChem.%2520Rev.%26date%3D2015%26volume%3D115%26spage%3D2419%26epage%3D2452%26doi%3D10.1021%2Fcr500452k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Delvecchio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaucher, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aguilar-Gurrieri, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortega, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Panne, D.</span><span> </span><span class="NLM_article-title">Structure of the P300 Catalytic Core and Implications for Chromatin Targeting and HAT Regulation</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1040</span><span class="NLM_x">–</span> <span class="NLM_lpage">1046</span><span class="refDoi"> DOI: 10.1038/nsmb.2642</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1038%2Fnsmb.2642" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=23934153" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Cgtr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=1040-1046&author=M.+Delvecchioauthor=J.+Gaucherauthor=C.+Aguilar-Gurrieriauthor=E.+Ortegaauthor=D.+Panne&title=Structure+of+the+P300+Catalytic+Core+and+Implications+for+Chromatin+Targeting+and+HAT+Regulation&doi=10.1038%2Fnsmb.2642"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the p300 catalytic core and implications for chromatin targeting and HAT regulation</span></div><div class="casAuthors">Delvecchio, Manuela; Gaucher, Jonathan; Aguilar-Gurrieri, Carmen; Ortega, Esther; Panne, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1040-1046</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">CBP and p300 are histone acetyltransferases (HATs) that assoc. with and acetylate transcriptional regulators and chromatin.  Mutations in their catalytic 'cores' are linked to genetic disorders, including cancer.  Here we present the 2.8-Å crystal structure of the catalytic core of human p300 contg. its bromodomain, CH2 region and HAT domain.  The structure reveals that the CH2 region contains a discontinuous PHD domain interrupted by a RING domain.  The bromodomain, PHD, RING and HAT domains adopt an assembled configuration with the RING domain positioned over the HAT substrate-binding pocket.  Disease mutations that disrupt RING attachment led to upregulation of HAT activity, thus revealing an inhibitory role for this domain.  The structure provides a starting point for understanding how chromatin-substrate targeting and HAT regulation are coupled and why mutations in the p300 core lead to dysregulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlYehjRwdhJ7Vg90H21EOLACvtfcHk0lgbR2ebVFGNWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Cgtr3J&md5=e9352198d513eac6fd93fc2896828d09</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2642&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2642%26sid%3Dliteratum%253Aachs%26aulast%3DDelvecchio%26aufirst%3DM.%26aulast%3DGaucher%26aufirst%3DJ.%26aulast%3DAguilar-Gurrieri%26aufirst%3DC.%26aulast%3DOrtega%26aufirst%3DE.%26aulast%3DPanne%26aufirst%3DD.%26atitle%3DStructure%2520of%2520the%2520P300%2520Catalytic%2520Core%2520and%2520Implications%2520for%2520Chromatin%2520Targeting%2520and%2520HAT%2520Regulation%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D1040%26epage%3D1046%26doi%3D10.1038%2Fnsmb.2642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Panagopoulos, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fioretos, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isaksson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samuelsson, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Billstrom, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strombeck, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitelman, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johansson, B.</span><span> </span><span class="NLM_article-title">Fusion of the <i>MORF</i> and <i>CBP</i> Genes in Acute Myeloid Leukemia with the t(10;16)(q22;p13)</span> <span class="citation_source-journal">Hum. Mol. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">395</span><span class="NLM_x">–</span> <span class="NLM_lpage">404</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1093%2Fhmg%2F10.4.395" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=11157802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhsVygtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2001&pages=395-404&author=I.+Panagopoulosauthor=T.+Fioretosauthor=M.+Isakssonauthor=U.+Samuelssonauthor=R.+Billstromauthor=B.+Strombeckauthor=F.+Mitelmanauthor=B.+Johansson&title=Fusion+of+the+MORF+and+CBP+Genes+in+Acute+Myeloid+Leukemia+with+the+t%2810%3B16%29%28q22%3Bp13%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Fusion of the MORF and CBP genes in acute myeloid leukemia with the t(10;16)(q22;p13)</span></div><div class="casAuthors">Panagopoulos, Ioannis; Fioretos, Thoas; Isaksson, Margareth; Samuelsson, Ulf; Billstrom, Rolf; Strombeck, Bodil; Mitelman, Felix; Johansson, Bertil</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">395-404</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The CBP gene at 16p13 fuses to MOZ and MLL as a result of the t(8;16)(p11;p13) in acute (myelo)monocytic leukemias (AML M4/M5) and the t(11;16)(q23;p13) in treatment-related AML, resp.  Here the authors show that a novel t(10;16)(q22;p13) in a childhood AML M5a leads to a MORF-CBP chimera.  RT-PCR using MORF forward and CBP reverse primers amplified a MORF-CBP fusion in which nucleotide 3103 of MORF was fused in-frame with nucleotide 284 of CBP.  Nested RT-PCR with CBP forward and MORF reverse primers generated a CBP-MORF transcript in which nucleotide 283 of CBP was fused in-frame with nucleotide 3104 of MORF.  Genomic analyses revealed that the breaks were close to Alu elements in intron 16 of MORF and intron 2 of CBP and that duplications had occurred near the breakpoints.  A database search using MORF cDNA enabled us to construct an exon-intron map of the MORF gene.  The MORF-CBP protein retains the zinc fingers, two nuclear localization signals, the histone acetyltransferase (HAT) domain, a portion of the acidic domain of MORF and the CBP protein downstream of codon 29.  Thus, the part of CBP encoding the RARA-binding domain, the CREB-binding domain, the three Cys/His-rich regions, the bromodomain, the HAT domain and the Glu-rich domains is present.  In the reciprocal CBP-MORF, part of the acidic domain and the C-terminal Ser- and Met-rich regions of MORF are likely to be driven by the CBP promoter.  Since both fusion transcripts were present, their exact role in the leukemogenic process remains to be elucidated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm-J76O9QaN7Vg90H21EOLACvtfcHk0lgbR2ebVFGNWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhsVygtrg%253D&md5=9f13abf66476fafa1dac70ddfb473e9b</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2F10.4.395&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252F10.4.395%26sid%3Dliteratum%253Aachs%26aulast%3DPanagopoulos%26aufirst%3DI.%26aulast%3DFioretos%26aufirst%3DT.%26aulast%3DIsaksson%26aufirst%3DM.%26aulast%3DSamuelsson%26aufirst%3DU.%26aulast%3DBillstrom%26aufirst%3DR.%26aulast%3DStrombeck%26aufirst%3DB.%26aulast%3DMitelman%26aufirst%3DF.%26aulast%3DJohansson%26aufirst%3DB.%26atitle%3DFusion%2520of%2520the%2520MORF%2520and%2520CBP%2520Genes%2520in%2520Acute%2520Myeloid%2520Leukemia%2520with%2520the%2520t%252810%253B16%2529%2528q22%253Bp13%2529%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2001%26volume%3D10%26spage%3D395%26epage%3D404" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Dutta, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tiu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakamoto, K. M.</span><span> </span><span class="NLM_article-title">CBP/P300 Acetyltransferase Activity in Hematologic Malignancies</span> <span class="citation_source-journal">Mol. Genet. Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">37</span><span class="NLM_x">–</span> <span class="NLM_lpage">43</span><span class="refDoi"> DOI: 10.1016/j.ymgme.2016.06.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1016%2Fj.ymgme.2016.06.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=27380996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFemsb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2016&pages=37-43&author=R.+Duttaauthor=B.+Tiuauthor=K.+M.+Sakamoto&title=CBP%2FP300+Acetyltransferase+Activity+in+Hematologic+Malignancies&doi=10.1016%2Fj.ymgme.2016.06.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">CBP/p300 acetyltransferase activity in hematologic malignancies</span></div><div class="casAuthors">Dutta, Ritika; Tiu, Bruce; Sakamoto, Kathleen M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Genetics and Metabolism</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">37-43</span>CODEN:
                <span class="NLM_cas:coden">MGMEFF</span>;
        ISSN:<span class="NLM_cas:issn">1096-7192</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">CREB binding protein (CBP) and p300 are crit. regulators of hematopoiesis through both their transcriptional coactivator and acetyltransferase activities.  Loss or mutation of CBP/p300 results in hematol. deficiencies in proliferation and differentiation as well as disruption of hematopoietic stem cell renewal and the microenvironment.  Aberrant lysine acetylation mediated by CBP/p300 has recently been implicated in the genesis of multiple hematol. cancers.  Understanding the effects of disrupting the acetyltransferase activity of CBP/p300 could pave the way for new therapeutic approaches to treat patients with these diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMiCksEjVzo7Vg90H21EOLACvtfcHk0ljudpR896m6bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFemsb7J&md5=4c0437f742fb98fbefa871c73dade8f3</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.ymgme.2016.06.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymgme.2016.06.013%26sid%3Dliteratum%253Aachs%26aulast%3DDutta%26aufirst%3DR.%26aulast%3DTiu%26aufirst%3DB.%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26atitle%3DCBP%252FP300%2520Acetyltransferase%2520Activity%2520in%2520Hematologic%2520Malignancies%26jtitle%3DMol.%2520Genet.%2520Metab.%26date%3D2016%26volume%3D119%26spage%3D37%26epage%3D43%26doi%3D10.1016%2Fj.ymgme.2016.06.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Ianculescu, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, D.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siegmund, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stallcup, M. R.</span><span> </span><span class="NLM_article-title">Selective Roles for cAMP Response Element-Binding Protein Binding Protein and P300 Protein as Coregulators for Androgen-Regulated Gene Expression in Advanced Prostate Cancer Cells</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">287</span><span class="NLM_x">, </span> <span class="NLM_fpage">4000</span><span class="NLM_x">–</span> <span class="NLM_lpage">4013</span><span class="refDoi"> DOI: 10.1074/jbc.M111.300194</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1074%2Fjbc.M111.300194" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=22174411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVSjtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=4000-4013&author=I.+Ianculescuauthor=D.-Y.+Wuauthor=K.+D.+Siegmundauthor=M.+R.+Stallcup&title=Selective+Roles+for+cAMP+Response+Element-Binding+Protein+Binding+Protein+and+P300+Protein+as+Coregulators+for+Androgen-Regulated+Gene+Expression+in+Advanced+Prostate+Cancer+Cells&doi=10.1074%2Fjbc.M111.300194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Roles for cAMP Response Element-binding Protein Binding Protein and p300 Protein as Coregulators for Androgen-regulated Gene Expression in Advanced Prostate Cancer Cells</span></div><div class="casAuthors">Ianculescu, Irina; Wu, Dai-Ying; Siegmund, Kimberly D.; Stallcup, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">4000-4013</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The protein acetyltransferases p300 and cAMP response element-binding protein binding protein (CBP) are homologous, ubiquitously expressed proteins that interact with hundreds of proteins involved in transcriptional regulation and are involved globally as transcriptional coregulators.  Although these two proteins acetylate and interact with overlapping sets of proteins, we found that p300 and CBP contribute to androgen-induced regulation of distinct sets of genes in C4-2B prostate cancer cells, a model of advanced prostate cancer.  CBP cannot compensate for the loss of p300 to support androgen-induced expression of many genes, such as TMPRSS2 and PSA.  Global gene expression anal. indicated that 47% of androgen-regulated genes are p300-dependent in these cells, whereas, surprisingly, only 0.3% of them are CBP-dependent.  Chromatin immunopptn. anal. after depletion of cellular p300 indicated that p300 is required for androgen-induced acetylation of histones H3 and H4, methylation of histone H3 at Lys-4, and recruitment of TATA box binding protein (TBP) and RNA polymerase II, but not recruitment of the androgen receptor, on the TMPRSS2 gene in response to androgen.  Thus, p300 is the dominant coregulator of the CBP/p300 pair for androgen-regulated gene expression in C4-2B cells. p300 is required at an early stage of chromatin remodeling and transcription complex assembly after binding of androgen receptor to the gene but before many crit. histone modifications occur.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7uJfPL3B4h7Vg90H21EOLACvtfcHk0ljudpR896m6bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVSjtLg%253D&md5=095ab7582da1b7602e2ab898caa5b094</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.300194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.300194%26sid%3Dliteratum%253Aachs%26aulast%3DIanculescu%26aufirst%3DI.%26aulast%3DWu%26aufirst%3DD.-Y.%26aulast%3DSiegmund%26aufirst%3DK.%2BD.%26aulast%3DStallcup%26aufirst%3DM.%2BR.%26atitle%3DSelective%2520Roles%2520for%2520cAMP%2520Response%2520Element-Binding%2520Protein%2520Binding%2520Protein%2520and%2520P300%2520Protein%2520as%2520Coregulators%2520for%2520Androgen-Regulated%2520Gene%2520Expression%2520in%2520Advanced%2520Prostate%2520Cancer%2520Cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D4000%26epage%3D4013%26doi%3D10.1074%2Fjbc.M111.300194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Iyer, N. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Özdag, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldas, C.</span><span> </span><span class="NLM_article-title">P300/CBP and Cancer</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">4225</span><span class="NLM_x">–</span> <span class="NLM_lpage">4231</span><span class="refDoi"> DOI: 10.1038/sj.onc.1207118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1038%2Fsj.onc.1207118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=15156177" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BD2cXkvVChtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=4225-4231&author=N.+G.+Iyerauthor=H.+%C3%96zdagauthor=C.+Caldas&title=P300%2FCBP+and+Cancer&doi=10.1038%2Fsj.onc.1207118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">p300/CBP and cancer</span></div><div class="casAuthors">Iyer, Narayanan Gopalakrishna; Oezdag, Hilal; Caldas, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">4225-4231</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  P300 and cAMP response element-binding protein (CBP) are adenoviral E1A-binding proteins involved in multiple cellular processes, and function as transcriptional co-factors and histone acetyltransferases.  Germline mutation of CBP results in Rubinstein-Taybi syndrome, which is characterized by an increased predisposition to childhood malignancies.  Furthermore, somatic mutations of p300 and CBP occur in a no. of malignancies.  Chromosome translocations target CBP and, less commonly, p300 in acute myeloid leukemia and treatment-related hematol. disorders.  P300 mutations in solid tumors result in truncated p300 protein products or amino-acid substitutions in crit. protein domains, and these are often assocd. with inactivation of the second allele.  A mouse model confirms that p300 and CBP function as suppressors of hematol. tumor formation.  The involvement of these proteins in crit. tumorigenic pathways (including TGF-β, p53 and Rb) provides a mechanistic route as to how their inactivation could result in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKGBLVLWLqkrVg90H21EOLACvtfcHk0ljudpR896m6bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXkvVChtrc%253D&md5=b68fe6bc258b06515c04a5207410f026</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1207118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1207118%26sid%3Dliteratum%253Aachs%26aulast%3DIyer%26aufirst%3DN.%2BG.%26aulast%3D%25C3%2596zdag%26aufirst%3DH.%26aulast%3DCaldas%26aufirst%3DC.%26atitle%3DP300%252FCBP%2520and%2520Cancer%26jtitle%3DOncogene%26date%3D2004%26volume%3D23%26spage%3D4225%26epage%3D4231%26doi%3D10.1038%2Fsj.onc.1207118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Fujisawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span> </span><span class="NLM_article-title">Functions of Bromodomain-Containing Proteins and Their Roles in Homeostasis and Cancer</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">246</span><span class="NLM_x">–</span> <span class="NLM_lpage">262</span><span class="refDoi"> DOI: 10.1038/nrm.2016.143</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1038%2Fnrm.2016.143" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=28053347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1Kiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=246-262&author=T.+Fujisawaauthor=P.+Filippakopoulos&title=Functions+of+Bromodomain-Containing+Proteins+and+Their+Roles+in+Homeostasis+and+Cancer&doi=10.1038%2Fnrm.2016.143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Functions of bromodomain-containing proteins and their roles in homeostasis and cancer</span></div><div class="casAuthors">Fujisawa, Takao; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">246-262</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Bromodomains (BRDs) are evolutionarily conserved protein-protein interaction modules that are found in a wide range of proteins with diverse catalytic and scaffolding functions and are present in most tissues.  BRDs selectively recognize and bind to acetylated Lys residues - particularly in histones - and thereby have important roles in the regulation of gene expression.  BRD-contg. proteins are frequently dysregulated in cancer, they participate in gene fusions that generate diverse, frequently oncogenic proteins, and many cancer-causing mutations have been mapped to the BRDs themselves.  Importantly, BRDs can be targeted by small-mol. inhibitors, which has stimulated many translational research projects that seek to attenuate the aberrant functions of BRD-contg. proteins in disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-drLvQK1rvLVg90H21EOLACvtfcHk0ljdjMUeH3vHNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1Kiuw%253D%253D&md5=cbccdee8423bad7a426c9cea6566151f</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnrm.2016.143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm.2016.143%26sid%3Dliteratum%253Aachs%26aulast%3DFujisawa%26aufirst%3DT.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DFunctions%2520of%2520Bromodomain-Containing%2520Proteins%2520and%2520Their%2520Roles%2520in%2520Homeostasis%2520and%2520Cancer%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2017%26volume%3D18%26spage%3D246%26epage%3D262%26doi%3D10.1038%2Fnrm.2016.143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de la Cruz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Segal, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merchant, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kharbanda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haverty, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Modrusan, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ly, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choo, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaufman, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beresini, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romero, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magnuson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gascoigne, K. E.</span><span> </span><span class="NLM_article-title">Therapeutic Targeting of the CBP/P300 Bromodomain for the Treatment of Castration Resistant Prostate Cancer</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">77</span><span class="NLM_x">, </span> <span class="NLM_fpage">5564</span><span class="NLM_x">–</span> <span class="NLM_lpage">5575</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-17-0314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1158%2F0008-5472.CAN-17-0314" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=5564-5575&author=L.+Jinauthor=J.+Garciaauthor=E.+Chanauthor=C.+de+la+Cruzauthor=E.+Segalauthor=M.+Merchantauthor=S.+Kharbandaauthor=P.+M.+Havertyauthor=Z.+Modrusanauthor=J.+Lyauthor=E.+F.+Chooauthor=S.+Kaufmanauthor=M.+H.+Beresiniauthor=F.+A.+Romeroauthor=S.+Magnusonauthor=K.+E.+Gascoigne&title=Therapeutic+Targeting+of+the+CBP%2FP300+Bromodomain+for+the+Treatment+of+Castration+Resistant+Prostate+Cancer&doi=10.1158%2F0008-5472.CAN-17-0314"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-17-0314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-17-0314%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DL.%26aulast%3DGarcia%26aufirst%3DJ.%26aulast%3DChan%26aufirst%3DE.%26aulast%3Dde%2Bla%2BCruz%26aufirst%3DC.%26aulast%3DSegal%26aufirst%3DE.%26aulast%3DMerchant%26aufirst%3DM.%26aulast%3DKharbanda%26aufirst%3DS.%26aulast%3DHaverty%26aufirst%3DP.%2BM.%26aulast%3DModrusan%26aufirst%3DZ.%26aulast%3DLy%26aufirst%3DJ.%26aulast%3DChoo%26aufirst%3DE.%2BF.%26aulast%3DKaufman%26aufirst%3DS.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DMagnuson%26aufirst%3DS.%26aulast%3DGascoigne%26aufirst%3DK.%2BE.%26atitle%3DTherapeutic%2520Targeting%2520of%2520the%2520CBP%252FP300%2520Bromodomain%2520for%2520the%2520Treatment%2520of%2520Castration%2520Resistant%2520Prostate%2520Cancer%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D5564%26epage%3D5575%26doi%3D10.1158%2F0008-5472.CAN-17-0314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Faiola, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lymar, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farina, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, E.</span><span> </span><span class="NLM_article-title">Dual Regulation of C-Myc by P300 via Acetylation-Dependent Control of Myc Protein Turnover and Coactivation of Myc-Induced Transcription</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">10220</span><span class="NLM_x">–</span> <span class="NLM_lpage">10234</span><span class="refDoi"> DOI: 10.1128/MCB.25.23.10220-10234.2005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1128%2FMCB.25.23.10220-10234.2005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=16287840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1Ons7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2005&pages=10220-10234&author=F.+Faiolaauthor=X.+Liuauthor=S.+Loauthor=S.+Panauthor=K.+Zhangauthor=E.+Lymarauthor=A.+Farinaauthor=E.+Martinez&title=Dual+Regulation+of+C-Myc+by+P300+via+Acetylation-Dependent+Control+of+Myc+Protein+Turnover+and+Coactivation+of+Myc-Induced+Transcription&doi=10.1128%2FMCB.25.23.10220-10234.2005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Dual regulation of c-Myc by p300 via acetylation-dependent control of Myc protein turnover and coactivation of Myc-induced transcription</span></div><div class="casAuthors">Faiola, Francesco; Liu, Xiaohui; Lo, Szuying; Pan, Songqin; Zhang, Kangling; Lymar, Elena; Farina, Anthony; Martinez, Ernest</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10220-10234</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The c-Myc oncoprotein (Myc) controls cell fate by regulating gene transcription in assocn. with a DNA-binding partner, Max.  While Max lacks a transcription regulatory domain, the N-terminus of Myc contains a transcription activation domain (TAD) that recruits cofactor complexes contg. the histone acetyltransferases (HATs) GCN5 and Tip60.  Here, we report a novel functional interaction between Myc TAD and the p300 coactivator-acetyltransferase.  We show that p300 assocs. with Myc in mammalian cells and in vitro through direct interactions with Myc TAD residues 1 to 110 and acetylates Myc in a TAD-dependent manner in vivo at several lysine residues located between the TAD and DNA-binding domain.  Moreover, the Myc:Max complex is differentially acetylated by p300 and GCN5 and is not acetylated by Tip60 in vitro, suggesting distinct functions for these acetyltransferases.  Whereas p300 and CBP can stabilize Myc independently of acetylation, p300-mediated acetylation results in increased Myc turnover.  In addn., p300 functions as a coactivator that is recruited by Myc to the promoter of the human telomerase reverse transcriptase gene, and p300/CBP stimulates Myc TAD-dependent transcription in a HAT domain-dependent manner.  Our results suggest dual roles for p300/CBP in Myc regulation: as a Myc coactivator that stabilizes Myc and as an inducer of Myc instability via direct Myc acetylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4or2afHXuK7Vg90H21EOLACvtfcHk0ljdjMUeH3vHNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1Ons7fL&md5=bbabfb64492444687f048ee59cdc0210</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1128%2FMCB.25.23.10220-10234.2005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.25.23.10220-10234.2005%26sid%3Dliteratum%253Aachs%26aulast%3DFaiola%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLo%26aufirst%3DS.%26aulast%3DPan%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DLymar%26aufirst%3DE.%26aulast%3DFarina%26aufirst%3DA.%26aulast%3DMartinez%26aufirst%3DE.%26atitle%3DDual%2520Regulation%2520of%2520C-Myc%2520by%2520P300%2520via%2520Acetylation-Dependent%2520Control%2520of%2520Myc%2520Protein%2520Turnover%2520and%2520Coactivation%2520of%2520Myc-Induced%2520Transcription%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2005%26volume%3D25%26spage%3D10220%26epage%3D10234%26doi%3D10.1128%2FMCB.25.23.10220-10234.2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Conery, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Centore, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neiss, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keller, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joshi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spillane, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandy, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatton, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pardo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zawadzke, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bommi-Reddy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gascoigne, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bryant, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mertz, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sims, R. J.</span><span> </span><span class="NLM_article-title">Bromodomain Inhibition of the Transcriptional Coactivators CBP/EP300 as a Therapeutic Strategy to Target the IRF4 Network in Multiple Myeloma</span> <span class="citation_source-journal">eLife</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e10483</span><span class="refDoi"> DOI: 10.7554/eLife.19432</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.7554%2FeLife.19432" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=e10483&author=A.+R.+Coneryauthor=R.+C.+Centoreauthor=A.+Neissauthor=P.+J.+Kellerauthor=S.+Joshiauthor=K.+L.+Spillaneauthor=P.+Sandyauthor=C.+Hattonauthor=E.+Pardoauthor=L.+Zawadzkeauthor=A.+Bommi-Reddyauthor=K.+E.+Gascoigneauthor=B.+M.+Bryantauthor=J.+A.+Mertzauthor=R.+J.+Sims&title=Bromodomain+Inhibition+of+the+Transcriptional+Coactivators+CBP%2FEP300+as+a+Therapeutic+Strategy+to+Target+the+IRF4+Network+in+Multiple+Myeloma&doi=10.7554%2FeLife.19432"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.7554%2FeLife.19432&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.19432%26sid%3Dliteratum%253Aachs%26aulast%3DConery%26aufirst%3DA.%2BR.%26aulast%3DCentore%26aufirst%3DR.%2BC.%26aulast%3DNeiss%26aufirst%3DA.%26aulast%3DKeller%26aufirst%3DP.%2BJ.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DSpillane%26aufirst%3DK.%2BL.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DHatton%26aufirst%3DC.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DZawadzke%26aufirst%3DL.%26aulast%3DBommi-Reddy%26aufirst%3DA.%26aulast%3DGascoigne%26aufirst%3DK.%2BE.%26aulast%3DBryant%26aufirst%3DB.%2BM.%26aulast%3DMertz%26aufirst%3DJ.%2BA.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26atitle%3DBromodomain%2520Inhibition%2520of%2520the%2520Transcriptional%2520Coactivators%2520CBP%252FEP300%2520as%2520a%2520Therapeutic%2520Strategy%2520to%2520Target%2520the%2520IRF4%2520Network%2520in%2520Multiple%2520Myeloma%26jtitle%3DeLife%26date%3D2016%26volume%3D5%26spage%3De10483%26doi%3D10.7554%2FeLife.19432" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Predina, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beier, U. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.-C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapoor, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhatti, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akimova, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singhal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brindle, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albelda, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hancock, W. W.</span><span> </span><span class="NLM_article-title">Inhibition of P300 Impairs Foxp3+ T Regulatory Cell Function and Promotes Antitumor Immunity</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1173</span><span class="NLM_x">–</span> <span class="NLM_lpage">1177</span><span class="refDoi"> DOI: 10.1038/nm.3286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1038%2Fnm.3286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=23955711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1yks77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=1173-1177&author=Y.+Liuauthor=L.+Wangauthor=J.+Predinaauthor=R.+Hanauthor=U.+H.+Beierauthor=L.-C.+S.+Wangauthor=V.+Kapoorauthor=T.+R.+Bhattiauthor=T.+Akimovaauthor=S.+Singhalauthor=P.+K.+Brindleauthor=P.+A.+Coleauthor=S.+M.+Albeldaauthor=W.+W.+Hancock&title=Inhibition+of+P300+Impairs+Foxp3%2B+T+Regulatory+Cell+Function+and+Promotes+Antitumor+Immunity&doi=10.1038%2Fnm.3286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of p300 impairs Foxp3+ T regulatory cell function and promotes antitumor immunity</span></div><div class="casAuthors">Liu, Yujie; Wang, Liqing; Predina, Jarrod; Han, Rongxiang; Beier, Ulf H.; Wang, Liang-Chuan S.; Kapoor, Veena; Bhatti, Tricia R.; Akimova, Tatiana; Singhal, Sunil; Brindle, Paul K.; Cole, Philip A.; Albelda, Steven M.; Hancock, Wayne W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1173-1177</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Forkhead box P3 (Foxp3)+ T regulatory (Treg) cells maintain immune homeostasis and limit autoimmunity but can also curtail host immune responses to various types of tumors.  Foxp3+ Treg cells are therefore considered promising targets to enhance antitumor immunity, and approaches for their therapeutic modulation are being developed.  However, although studies showing that exptl. depleting Foxp3+ Treg cells can enhance antitumor responses provide proof of principle, these studies lack clear translational potential and have various shortcomings.  Histone/protein acetyltransferases (HATs) promote chromatin accessibility, gene transcription and the function of multiple transcription factors and nonhistone proteins.  We now report that conditional deletion or pharmacol. inhibition of one HAT, p300 (also known as Ep300 or KAT3B), in Foxp3+ Treg cells increased T cell receptor-induced apoptosis in Treg cells, impaired Treg cell suppressive function and peripheral Treg cell induction, and limited tumor growth in immunocompetent but not in immunodeficient mice.  Our data thereby demonstrate that p300 is important for Foxp3+ Treg cell function and homeostasis in vivo and in vitro, and identify mechanisms by which appropriate small-mol. inhibitors can diminish Treg cell function without overtly impairing T effector cell responses or inducing autoimmunity.  Collectively, these data suggest a new approach for cancer immunotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoG1uh9kwBI_rVg90H21EOLACvtfcHk0lhalRGevJHqOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1yks77K&md5=b381e7523f686ed8ea8d54aa38327c41</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnm.3286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3286%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DPredina%26aufirst%3DJ.%26aulast%3DHan%26aufirst%3DR.%26aulast%3DBeier%26aufirst%3DU.%2BH.%26aulast%3DWang%26aufirst%3DL.-C.%2BS.%26aulast%3DKapoor%26aufirst%3DV.%26aulast%3DBhatti%26aufirst%3DT.%2BR.%26aulast%3DAkimova%26aufirst%3DT.%26aulast%3DSinghal%26aufirst%3DS.%26aulast%3DBrindle%26aufirst%3DP.%2BK.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DAlbelda%26aufirst%3DS.%2BM.%26aulast%3DHancock%26aufirst%3DW.%2BW.%26atitle%3DInhibition%2520of%2520P300%2520Impairs%2520Foxp3%252B%2520T%2520Regulatory%2520Cell%2520Function%2520and%2520Promotes%2520Antitumor%2520Immunity%26jtitle%3DNat.%2520Med.%26date%3D2013%26volume%3D19%26spage%3D1173%26epage%3D1177%26doi%3D10.1038%2Fnm.3286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Ghosh, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, H.-R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conery, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mertz, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salmeron, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dakle, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mele, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Côté, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayaram, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setser, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poy, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatzivassiliou, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeAlmeida-Nagata, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandy, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatton, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romero, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reimer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crawford, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pardo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, V. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsui, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cochran, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zawadzke, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harmange, J.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bryant, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cummings, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magnuson, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grogan, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellon, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albrecht, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sims, R. J.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lora, J. M.</span><span> </span><span class="NLM_article-title">Regulatory T Cell Modulation by CBP/EP300 Bromodomain Inhibition</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">291</span><span class="NLM_x">, </span> <span class="NLM_fpage">13014</span><span class="NLM_x">–</span> <span class="NLM_lpage">13027</span><span class="refDoi"> DOI: 10.1074/jbc.M115.708560</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1074%2Fjbc.M115.708560" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=27056325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVSksL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=13014-13027&author=S.+Ghoshauthor=A.+Taylorauthor=M.+Chinauthor=H.-R.+Huangauthor=A.+R.+Coneryauthor=J.+A.+Mertzauthor=A.+Salmeronauthor=P.+J.+Dakleauthor=D.+Meleauthor=A.+C%C3%B4t%C3%A9author=H.+Jayaramauthor=J.+W.+Setserauthor=F.+Poyauthor=G.+Hatzivassiliouauthor=D.+DeAlmeida-Nagataauthor=P.+Sandyauthor=C.+Hattonauthor=F.+A.+Romeroauthor=E.+Chiangauthor=T.+Reimerauthor=T.+Crawfordauthor=E.+Pardoauthor=V.+G.+Watsonauthor=V.+Tsuiauthor=A.+G.+Cochranauthor=L.+Zawadzkeauthor=J.-C.+Harmangeauthor=J.+E.+Audiaauthor=B.+M.+Bryantauthor=R.+T.+Cummingsauthor=S.+R.+Magnusonauthor=J.+L.+Groganauthor=S.+F.+Bellonauthor=B.+K.+Albrechtauthor=R.+J.+Simsauthor=J.+M.+Lora&title=Regulatory+T+Cell+Modulation+by+CBP%2FEP300+Bromodomain+Inhibition&doi=10.1074%2Fjbc.M115.708560"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Regulatory T Cell Modulation by CBP/EP300 Bromodomain Inhibition</span></div><div class="casAuthors">Ghosh, Srimoyee; Taylor, Alexander; Chin, Melissa; Huang, Hon-Ren; Conery, Andrew R.; Mertz, Jennifer A.; Salmeron, Andres; Dakle, Pranal J.; Mele, Deanna; Cote, Alexandre; Jayaram, Hari; Setser, Jeremy W.; Poy, Florence; Hatzivassiliou, Georgia; DeAlmeida-Nagata, Denise; Sandy, Peter; Hatton, Charlie; Romero, F. Anthony; Chiang, Eugene; Reimer, Thornik; Crawford, Terry; Pardo, Eneida; Watson, Venita G.; Tsui, Vickie; Cochran, Andrea G.; Zawadzke, Laura; Harmange, Jean-Christophe; Audia, James E.; Bryant, Barbara M.; Cummings, Richard T.; Magnuson, Steven R.; Grogan, Jane L.; Bellon, Steve F.; Albrecht, Brian K.; Sims, Robert J. III; Lora, Jose M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">13014-13027</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Covalent modification of histones is a fundamental mechanism of regulated gene expression in eukaryotes, and interpretation of histone modifications is an essential feature of epigenetic control.  Bromodomains are specialized binding modules that interact with acetylated histones, linking chromatin recognition to gene transcription.  Because of their ability to function in a domain-specific fashion, selective disruption of bromodomain:acetylated histone interactions with chem. probes serves as a powerful means for understanding biol. processes regulated by these chromatin adaptors.  Here the authors describe the discovery and characterization of potent and selective small mol. inhibitors for the bromodomains of CREBBP/EP300 that engage their target in cellular assays.  The authors use these tools to demonstrate a crit. role for CREBBP/EP300 bromodomains in regulatory T cell biol.  Because regulatory T cell recruitment to tumors is a major mechanism of immune evasion by cancer cells, the authors' data highlight the importance of CREBBP/EP300 bromodomain inhibition as a novel, small mol.-based approach for cancer immunotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9maWuQVImyLVg90H21EOLACvtfcHk0lhalRGevJHqOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVSksL3F&md5=bc8b4edba81bdbef30b1d8fcfaa19f1e</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.708560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.708560%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DTaylor%26aufirst%3DA.%26aulast%3DChin%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DH.-R.%26aulast%3DConery%26aufirst%3DA.%2BR.%26aulast%3DMertz%26aufirst%3DJ.%2BA.%26aulast%3DSalmeron%26aufirst%3DA.%26aulast%3DDakle%26aufirst%3DP.%2BJ.%26aulast%3DMele%26aufirst%3DD.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DSetser%26aufirst%3DJ.%2BW.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DHatzivassiliou%26aufirst%3DG.%26aulast%3DDeAlmeida-Nagata%26aufirst%3DD.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DHatton%26aufirst%3DC.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DChiang%26aufirst%3DE.%26aulast%3DReimer%26aufirst%3DT.%26aulast%3DCrawford%26aufirst%3DT.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DWatson%26aufirst%3DV.%2BG.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DZawadzke%26aufirst%3DL.%26aulast%3DHarmange%26aufirst%3DJ.-C.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DBryant%26aufirst%3DB.%2BM.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DMagnuson%26aufirst%3DS.%2BR.%26aulast%3DGrogan%26aufirst%3DJ.%2BL.%26aulast%3DBellon%26aufirst%3DS.%2BF.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26atitle%3DRegulatory%2520T%2520Cell%2520Modulation%2520by%2520CBP%252FEP300%2520Bromodomain%2520Inhibition%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D13014%26epage%3D13027%26doi%3D10.1074%2Fjbc.M115.708560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Sachchidanand</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Resnick-Silverman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mutjaba, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, W.-j.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manfredi, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M.-M.</span><span> </span><span class="NLM_article-title">Target Structure-Based Discovery of Small Molecules that Block Human p53 and CREB Binding Protein Association</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">81</span><span class="NLM_x">–</span> <span class="NLM_lpage">90</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2005.10.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1016%2Fj.chembiol.2005.10.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=16426974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsV2qsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=81-90&author=+Sachchidanandauthor=L.+Resnick-Silvermanauthor=S.+Yanauthor=S.+Mutjabaauthor=W.-j.+Liuauthor=L.+Zengauthor=J.+J.+Manfrediauthor=M.-M.+Zhou&title=Target+Structure-Based+Discovery+of+Small+Molecules+that+Block+Human+p53+and+CREB+Binding+Protein+Association&doi=10.1016%2Fj.chembiol.2005.10.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Target Structure-Based Discovery of Small Molecules that Block Human p53 and CREB Binding Protein Association</span></div><div class="casAuthors">Sachchidanand; Resnick-Silverman, Lois; Yan, Sherry; Mutjaba, Shiraz; Liu, Wen-jun; Zeng, Lei; Manfredi, James J.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">81-90</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Lysine acetylation of human tumor suppressor p53 in response to cellular stress signals is required for its function as a transcription factor that regulates cell cycle arrest, senescence, or apoptosis.  Here, we report small mols. that block lysine 382-acetylated p53 assocn. with the bromodomain of the coactivator CBP, an interaction essential for p53-induced transcription of the cell cycle inhibitor p21 in response to DNA damage.  These chems. were discovered in target structure-guided NMR spectroscopy screening of a focused chem. library constructed based on the structural knowledge of CBP bromodomain/p53-AcK382 binding.  Structural characterization shows that these chems. inhibit CBP/p53 assocn. by binding to the acetyl-lysine binding site of the bromodomain.  Cell-based functional assays demonstrate that the lead chems. can modulate p53 stability and function in response to doxorubicin-induced DNA damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrznkGtHkYXCbVg90H21EOLACvtfcHk0lgNgs07ZaotIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsV2qsQ%253D%253D&md5=a88211df5b27a2d2d37cb09a3bc50821</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2005.10.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2005.10.014%26sid%3Dliteratum%253Aachs%26aulast%3DSachchidanand%26aulast%3DResnick-Silverman%26aufirst%3DL.%26aulast%3DYan%26aufirst%3DS.%26aulast%3DMutjaba%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DW.-j.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DManfredi%26aufirst%3DJ.%2BJ.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DTarget%2520Structure-Based%2520Discovery%2520of%2520Small%2520Molecules%2520that%2520Block%2520Human%2520p53%2520and%2520CREB%2520Binding%2520Protein%2520Association%26jtitle%3DChem.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D81%26epage%3D90%26doi%3D10.1016%2Fj.chembiol.2005.10.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Borah, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mujtaba, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karakikes, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moshkina, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morohashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerona-Navorro, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajjar, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M.-M.</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">531</span><span class="NLM_x">–</span> <span class="NLM_lpage">541</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2010.12.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1016%2Fj.chembiol.2010.12.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=21513889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltFartb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Borah%2C+J.+C.%3B+Mujtaba%2C+S.%3B+Karakikes%2C+I.%3B+Zeng%2C+L.%3B+Muller%2C+M.%3B+Patel%2C+J.%3B+Moshkina%2C+N.%3B+Morohashi%2C+K.%3B+Zhang%2C+W.%3B+Gerona-Navorro%2C+G.%3B+Hajjar%2C+R.+J.%3B+Zhou%2C+M.-M.Chem.+Biol.+2011%2C+18%2C+531%E2%80%93541+10.1016%2Fj.chembiol.2010.12.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">A Small Molecule Binding to the Coactivator CREB-Binding Protein Blocks Apoptosis in Cardiomyocytes</span></div><div class="casAuthors">Borah, Jagat C.; Mujtaba, Shiraz; Karakikes, Ioannis; Zeng, Lei; Muller, Michaela; Patel, Jigneshkumar; Moshkina, Natasha; Morohashi, Keita; Zhang, Weijia; Gerona-Navarro, Guillermo; Hajjar, Roger J.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">531-541</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: As a master transcription factor in cellular responses to external stress, tumor suppressor p53 is tightly regulated.  Excessive p53 activity during myocardial ischemia causes irreversible cellular injury and cardiomyocyte death.  P53 activation is dependent on lysine acetylation by the lysine acetyltransferase and transcriptional coactivator CREB-binding protein (CBP) and on acetylation-directed CBP recruitment for p53 target gene expression.  Here, we report a small mol. ischemin, developed with a structure-guided approach to inhibit the acetyl-lysine binding activity of the bromodomain of CBP.  We show that ischemin alters post-translational modifications on p53 and histones, inhibits p53 interaction with CBP and transcriptional activity in cells, and prevents apoptosis in ischemic cardiomyocytes.  Our study suggests small mol. modulation of acetylation-mediated interactions in gene transcription as a new approach to therapeutic interventions of human disorders such as myocardial ischemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHKOaQAGSYTbVg90H21EOLACvtfcHk0lgNgs07ZaotIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltFartb4%253D&md5=0c1d7500e1fff40bc335d1b10ac3c27a</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.12.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.12.021%26sid%3Dliteratum%253Aachs%26aulast%3DBorah%26aufirst%3DJ.%2BC.%26aulast%3DMujtaba%26aufirst%3DS.%26aulast%3DKarakikes%26aufirst%3DI.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DMuller%26aufirst%3DM.%26aulast%3DPatel%26aufirst%3DJ.%26aulast%3DMoshkina%26aufirst%3DN.%26aulast%3DMorohashi%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DGerona-Navorro%26aufirst%3DG.%26aulast%3DHajjar%26aufirst%3DR.%2BJ.%26aulast%3DZhou%26aufirst%3DM.-M.%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D531%26epage%3D541%26doi%3D10.1016%2Fj.chembiol.2010.12.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Ito, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umehara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishino, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terada, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirouzu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Padmanabhan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoyama, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ito, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, M.</span><span> </span><span class="NLM_article-title">Real-Time Imaging of Histone Deacetylase and Bromodomain Inhibitors</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">495</span><span class="NLM_x">–</span> <span class="NLM_lpage">507</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2011.02.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1016%2Fj.chembiol.2011.02.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=21513886" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltFartL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=495-507&author=T.+Itoauthor=T.+Umeharaauthor=K.+Sasakiauthor=Y.+Nakamuraauthor=N.+Nishinoauthor=T.+Teradaauthor=M.+Shirouzuauthor=B.+Padmanabhanauthor=S.+Yokoyamaauthor=A.+Itoauthor=M.+Yoshida&title=Real-Time+Imaging+of+Histone+Deacetylase+and+Bromodomain+Inhibitors&doi=10.1016%2Fj.chembiol.2011.02.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Real-Time Imaging of Histone H4K12-Specific Acetylation Determines the Modes of Action of Histone Deacetylase and Bromodomain Inhibitors</span></div><div class="casAuthors">Ito, Tamaki; Umehara, Takashi; Sasaki, Kazuki; Nakamura, Yoshihiro; Nishino, Norikazu; Terada, Takaho; Shirouzu, Mikako; Padmanabhan, Balasundaram; Yokoyama, Shigeyuki; Ito, Akihiro; Yoshida, Minoru</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">495-507</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone acetylation constitutes an epigenetic mark for transcriptional regulation.  Here we developed a fluorescent probe to visualize acetylation of histone H4 Lys12 (H4K12) in living cells using fluorescence resonance energy transfer (FRET) and the binding of the BRD2 bromodomain to acetylated H4K12.  Using this probe designated as Histac-K12, we demonstrated that histone H4K12 acetylation is retained in mitosis and that some histone deacetylase (HDAC) inhibitors continue to inhibit cellular HDAC activity even after their removal from the culture.  In addn., a small mol. that interferes with ability of the bromodomain to bind to acetylated H4K12 could be assessed using Histac-K12 in cells.  Thus, Histac-K12 will serve as a powerful tool not only to understand the dynamics of H4K12-specific acetylation but also to characterize small mols. that modulate the acetylation or interaction status of histones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBoA9RjMb5tLVg90H21EOLACvtfcHk0lgNgs07ZaotIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltFartL0%253D&md5=f36a5042c2e9342026884f88f9704954</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.02.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.02.009%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DT.%26aulast%3DUmehara%26aufirst%3DT.%26aulast%3DSasaki%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DNishino%26aufirst%3DN.%26aulast%3DTerada%26aufirst%3DT.%26aulast%3DShirouzu%26aufirst%3DM.%26aulast%3DPadmanabhan%26aufirst%3DB.%26aulast%3DYokoyama%26aufirst%3DS.%26aulast%3DIto%26aufirst%3DA.%26aulast%3DYoshida%26aufirst%3DM.%26atitle%3DReal-Time%2520Imaging%2520of%2520Histone%2520Deacetylase%2520and%2520Bromodomain%2520Inhibitors%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D495%26epage%3D507%26doi%3D10.1016%2Fj.chembiol.2011.02.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Hewings, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uttarkar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vuppusetty, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conway, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span> </span><span class="NLM_article-title">3,5-Dimethylisoxazoles Act as Acetyl-Lysine-Mimetic Bromodomain Ligands</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">6761</span><span class="NLM_x">–</span> <span class="NLM_lpage">6770</span><span class="refDoi"> DOI: 10.1021/jm200640v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200640v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaks7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6761-6770&author=D.+S.+Hewingsauthor=M.+Wangauthor=M.+Philpottauthor=O.+Fedorovauthor=S.+Uttarkarauthor=P.+Filippakopoulosauthor=S.+Picaudauthor=C.+Vuppusettyauthor=B.+Marsdenauthor=S.+Knappauthor=S.+J.+Conwayauthor=T.+D.+Heightman&title=3%2C5-Dimethylisoxazoles+Act+as+Acetyl-Lysine-Mimetic+Bromodomain+Ligands&doi=10.1021%2Fjm200640v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">3,5-Dimethylisoxazoles Act As Acetyl-lysine-mimetic Bromodomain Ligands</span></div><div class="casAuthors">Hewings, David S.; Wang, Minghua; Philpott, Martin; Fedorov, Oleg; Uttarkar, Sagar; Filippakopoulos, Panagis; Picaud, Sarah; Vuppusetty, Chaitanya; Marsden, Brian; Knapp, Stefan; Conway, Stuart J.; Heightman, Tom D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6761-6770</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone-lysine acetylation is a vital chromatin post-translational modification involved in the epigenetic regulation of gene transcription.  Bromodomains bind acetylated lysines, acting as readers of the histone-acetylation code.  Competitive inhibitors of this interaction have antiproliferative and anti-inflammatory properties.  With 57 distinct bromodomains known, the discovery of subtype-selective inhibitors of the histone-bromodomain interaction is of great importance.  We have identified the 3,5-dimethylisoxazole moiety as a novel acetyl-lysine bioisostere, which displaces acetylated histone-mimicking peptides from bromodomains.  Using X-ray crystallog. anal., we have detd. the interactions responsible for the activity and selectivity of 4-substituted 3,5-dimethylisoxazoles against a selection of phylogenetically diverse bromodomains.  By exploiting these interactions, we have developed compd. 4d, which has IC50 values of <5 μM for the bromodomain-contg. proteins BRD2(1) and BRD4(1).  These compds. are promising leads for the further development of selective probes for the bromodomain and extra C-terminal domain (BET) family and CREBBP bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEzK4xMvJhXrVg90H21EOLACvtfcHk0ljf_xtgRc2Hyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaks7rO&md5=7d6f9eff2735c8a7189dec4fb5c5d13d</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm200640v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200640v%26sid%3Dliteratum%253Aachs%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DUttarkar%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DVuppusetty%26aufirst%3DC.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26atitle%3D3%252C5-Dimethylisoxazoles%2520Act%2520as%2520Acetyl-Lysine-Mimetic%2520Bromodomain%2520Ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6761%26epage%3D6770%26doi%3D10.1021%2Fjm200640v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span>I-CBP112: A CREBBP/EP-300-Selective Chemical Probe; SGC, 2015. <a href="http://www.thesgc.org/chemical-probes/I-CBP112" class="extLink">http://www.thesgc.org/chemical-probes/I-CBP112</a> (accessed December 11,<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=I-CBP112%3A+A+CREBBP%2FEP-300-Selective+Chemical+Probe%3B+SGC%2C+2015.+http%3A%2F%2Fwww.thesgc.org%2Fchemical-probes%2FI-CBP112+%28accessed+December+11%2C+2015%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Hewings, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olcina, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freeman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gill, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ritchie, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheppard, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russell, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammond, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conway, S. J.</span><span> </span><span class="NLM_article-title">Optimization of 3,5-Dimethylisoxazole Derivatives as Potent Bromodomain Ligands</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">3217</span><span class="NLM_x">–</span> <span class="NLM_lpage">3227</span><span class="refDoi"> DOI: 10.1021/jm301588r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301588r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktlekuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=3217-3227&author=D.+S.+Hewingsauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=S.+Martinauthor=S.+Picaudauthor=A.+Tumberauthor=C.+Wellsauthor=M.+M.+Olcinaauthor=K.+Freemanauthor=A.+Gillauthor=A.+J.+Ritchieauthor=D.+W.+Sheppardauthor=A.+J.+Russellauthor=E.+M.+Hammondauthor=S.+Knappauthor=P.+E.+Brennanauthor=S.+J.+Conway&title=Optimization+of+3%2C5-Dimethylisoxazole+Derivatives+as+Potent+Bromodomain+Ligands&doi=10.1021%2Fjm301588r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of 3,5-Dimethylisoxazole Derivatives as Potent Bromodomain Ligands</span></div><div class="casAuthors">Hewings, David S.; Fedorov, Oleg; Filippakopoulos, Panagis; Martin, Sarah; Picaud, Sarah; Tumber, Anthony; Wells, Christopher; Olcina, Monica M.; Freeman, Katherine; Gill, Andrew; Ritchie, Alison J.; Sheppard, David W.; Russell, Angela J.; Hammond, Ester M.; Knapp, Stefan; Brennan, Paul E.; Conway, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3217-3227</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">(Isoxazolylphenyl)arylmethanols such as I (R = H; R1 = HO; R2 = H; R3 = Ph), I (R = HO; R1 = H; R2 = H; R3 = Ph), and I (R = H, HO; R1 = HO, H; R2 = Et; R3 = Ph, 3-FC6H4, 4-FC6H4, 3-ClC6H4, 4-ClC6H4) were prepd. as inhibitors of the bromodomain and extra terminal domain (BET) bromodomain family member BRD4(1) for potential use as antitumor agents.  The structures of I (R = H; R1 = HO; R2 = H; R3 = Ph) and I (R = HO; R1 = H; R2 = H; R3 = Ph) bound to human BRD4(1) were detd. by X-ray crystallog. to det. features useful for selective binding to BRD4(1).  I (R = H, HO; R1 = HO, H; R2 = H, MeCO; R3 = Ph) and I (R = H, HO; R1 = HO, H; R2 = Et; R3 = 3-ClC6H4) inhibited a human acute myeloid leukemia cell line in vitro with 3-4 fold reduced inhibition and reduced toxicity in other human cells compared to a known BRD4(1) inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVEQ5OOJGC8bVg90H21EOLACvtfcHk0ljf_xtgRc2Hyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktlekuro%253D&md5=bb4890291db8c3e8d404a75dd9dcbb87</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm301588r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301588r%26sid%3Dliteratum%253Aachs%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DOlcina%26aufirst%3DM.%2BM.%26aulast%3DFreeman%26aufirst%3DK.%26aulast%3DGill%26aufirst%3DA.%26aulast%3DRitchie%26aufirst%3DA.%2BJ.%26aulast%3DSheppard%26aufirst%3DD.%2BW.%26aulast%3DRussell%26aufirst%3DA.%2BJ.%26aulast%3DHammond%26aufirst%3DE.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DOptimization%2520of%25203%252C5-Dimethylisoxazole%2520Derivatives%2520as%2520Potent%2520Bromodomain%2520Ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D3217%26epage%3D3227%26doi%3D10.1021%2Fjm301588r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Hay, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singleton, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tallant, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monteiro, O. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conway, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yapp, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span> </span><span class="NLM_article-title">Discovery and Optimization of Small-Molecule Ligands for the CBP/P300 Bromodomains</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">136</span><span class="NLM_x">, </span> <span class="NLM_fpage">9308</span><span class="NLM_x">–</span> <span class="NLM_lpage">9319</span><span class="refDoi"> DOI: 10.1021/ja412434f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja412434f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVShsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=9308-9319&author=D.+A.+Hayauthor=O.+Fedorovauthor=S.+Martinauthor=D.+C.+Singletonauthor=C.+Tallantauthor=C.+Wellsauthor=S.+Picaudauthor=M.+Philpottauthor=O.+P.+Monteiroauthor=S.+J.+Conwayauthor=A.+Tumberauthor=C.+Yappauthor=P.+Filippakopoulosauthor=M.+E.+Bunnageauthor=S.+Knappauthor=C.+J.+Schofieldauthor=P.+E.+Brennan&title=Discovery+and+Optimization+of+Small-Molecule+Ligands+for+the+CBP%2FP300+Bromodomains&doi=10.1021%2Fja412434f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of Small-Molecule Ligands for the CBP/p300 Bromodomains</span></div><div class="casAuthors">Hay, Duncan A.; Fedorov, Oleg; Martin, Sarah; Singleton, Dean C.; Tallant, Cynthia; Wells, Christopher; Picaud, Sarah; Philpott, Martin; Monteiro, Octovia P.; Rogers, Catherine M.; Conway, Stuart J.; Rooney, Timothy P. C.; Tumber, Anthony; Yapp, Clarence; Filippakopoulos, Panagis; Bunnage, Mark E.; Muller, Susanne; Knapp, Stefan; Schofield, Christopher J.; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">9308-9319</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small-mol. inhibitors that target bromodomains outside of the bromodomain and extra-terminal (BET) sub-family are lacking.  Here, the authors describe highly potent and selective ligands for the bromodomain module of the human lysine acetyl transferase CBP/p300, developed from a series of 5-isoxazolyl-benzimidazoles.  The starting point was a fragment hit, which was optimized into a more potent and selective lead using parallel synthesis employing Suzuki couplings, benzimidazole-forming reactions, and reductive aminations.  The selectivity of the lead compd. against other bromodomain family members was investigated using a thermal stability assay, which revealed some inhibition of the structurally related BET family members.  To address the BET selectivity issue, x-ray crystal structures of the lead compd. bound to the CREB binding protein (CBP) and the first bromodomain of BRD4 (BRD4(1)) were used to guide the design of more selective compds.  The crystal structures obtained revealed two distinct binding modes.  By varying the aryl substitution pattern and developing conformationally constrained analogs, selectivity for CBP over BRD4(1) was increased.  The optimized compd. is highly potent (Kd = 21 nM) and selective, displaying 40-fold selectivity over BRD4(1).  Cellular activity was demonstrated using fluorescence recovery after photo-bleaching (FRAP) and a p53 reporter assay.  The optimized compds. are cell-active and have nanomolar affinity for CBP/p300; therefore, they should be useful in studies investigating the biol. roles of CBP and p300 and to validate the CBP and p300 bromodomains as therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFzgs1DD4QZLVg90H21EOLACvtfcHk0lj4neXnqpKQgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVShsLvP&md5=e5da3e96a4d234453a481c7ee5e788f2</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fja412434f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja412434f%26sid%3Dliteratum%253Aachs%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSingleton%26aufirst%3DD.%2BC.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DDiscovery%2520and%2520Optimization%2520of%2520Small-Molecule%2520Ligands%2520for%2520the%2520CBP%252FP300%2520Bromodomains%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D9308%26epage%3D9319%26doi%3D10.1021%2Fja412434f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Rooney, T. P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cortopassi, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hay, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pryde, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paton, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conway, S. J.</span><span> </span><span class="NLM_article-title">A Series of Potent CREBBP Bromodomain Ligands Reveals an Induced-Fit Pocket Stabilized by a Cation-π Interaction</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">6126</span><span class="NLM_x">–</span> <span class="NLM_lpage">6130</span><span class="refDoi"> DOI: 10.1002/anie.201402750</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1002%2Fanie.201402750" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnvFWgsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=6126-6130&author=T.+P.+C.+Rooneyauthor=P.+Filippakopoulosauthor=O.+Fedorovauthor=S.+Picaudauthor=W.+A.+Cortopassiauthor=D.+A.+Hayauthor=S.+Martinauthor=A.+Tumberauthor=C.+M.+Rogersauthor=M.+Philpottauthor=M.+Wangauthor=A.+L.+Thompsonauthor=T.+D.+Heightmanauthor=D.+C.+Prydeauthor=A.+Cookauthor=R.+S.+Patonauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=P.+E.+Brennanauthor=S.+J.+Conway&title=A+Series+of+Potent+CREBBP+Bromodomain+Ligands+Reveals+an+Induced-Fit+Pocket+Stabilized+by+a+Cation-%CF%80+Interaction&doi=10.1002%2Fanie.201402750"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">A Series of Potent CREBBP Bromodomain Ligands Reveals an Induced-Fit Pocket Stabilized by a Cation-π Interaction</span></div><div class="casAuthors">Rooney, Timothy P. C.; Filippakopoulos, Panagis; Fedorov, Oleg; Picaud, Sarah; Cortopassi, Wilian A.; Hay, Duncan A.; Martin, Sarah; Tumber, Anthony; Rogers, Catherine M.; Philpott, Martin; Wang, Minghua; Thompson, Amber L.; Heightman, Tom D.; Pryde, David C.; Cook, Andrew; Paton, Robert S.; Mueller, Susanne; Knapp, Stefan; Brennan, Paul E.; Conway, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6126-6130</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Benzoxazinone and dihydroquinoxalinone fragments were employed as novel acetyl lysine mimics in the development of CREBBP bromodomain ligands.  While the benzoxazinone series showed low affinity for the CREBBP bromodomain, expansion of the dihydroquinoxalinone series resulted in the first potent inhibitors, e.g., I, of a bromodomain outside the BET family.  Structural and computational studies revealed that an internal hydrogen bond stabilized the protein-bound conformation of the dihydroquinoxalinone series.  The side chain of this series binds in an induced-fit pocket forming a cation-π interaction with R1173 of CREBBP.  Compd. I, the most potent compd., inhibited binding of CREBBP to chromatin in U2OS cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHFgTVUKe7lrVg90H21EOLACvtfcHk0lj4neXnqpKQgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnvFWgsb0%253D&md5=42b12e1fe855d2af02c49f8e1031d39e</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2Fanie.201402750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201402750%26sid%3Dliteratum%253Aachs%26aulast%3DRooney%26aufirst%3DT.%2BP.%2BC.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DCortopassi%26aufirst%3DW.%2BA.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DThompson%26aufirst%3DA.%2BL.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DPryde%26aufirst%3DD.%2BC.%26aulast%3DCook%26aufirst%3DA.%26aulast%3DPaton%26aufirst%3DR.%2BS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DA%2520Series%2520of%2520Potent%2520CREBBP%2520Bromodomain%2520Ligands%2520Reveals%2520an%2520Induced-Fit%2520Pocket%2520Stabilized%2520by%2520a%2520Cation-%25CF%2580%2520Interaction%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D6126%26epage%3D6130%26doi%3D10.1002%2Fanie.201402750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thanasopoulou, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leonards, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meier, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olzscha, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monteiro, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yapp, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Che, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bannister, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parr, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lugo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vecellio, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Mahony, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velichko, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hay, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daniels, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urh, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kouzarides, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwaller, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Generation of a Selective Small Molecule Inhibitor of the CBP/P300 Bromodomain for Leukemia Therapy</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">75</span><span class="NLM_x">, </span> <span class="NLM_fpage">5106</span><span class="NLM_x">–</span> <span class="NLM_lpage">5119</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-15-0236</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1158%2F0008-5472.CAN-15-0236" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=26552700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFensrzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2015&pages=5106-5119&author=S.+Picaudauthor=O.+Fedorovauthor=A.+Thanasopoulouauthor=K.+Leonardsauthor=K.+Jonesauthor=J.+Meierauthor=H.+Olzschaauthor=O.+Monteiroauthor=S.+Martinauthor=M.+Philpottauthor=A.+Tumberauthor=P.+Filippakopoulosauthor=C.+Yappauthor=C.+Wellsauthor=K.+H.+Cheauthor=A.+Bannisterauthor=S.+Robsonauthor=U.+Kumarauthor=N.+Parrauthor=K.+Leeauthor=D.+Lugoauthor=P.+Jeffreyauthor=S.+Taylorauthor=M.+L.+Vecellioauthor=C.+Bountraauthor=P.+E.+Brennanauthor=A.+O%E2%80%99Mahonyauthor=S.+Velichkoauthor=S.+M%C3%BCllerauthor=D.+Hayauthor=D.+L.+Danielsauthor=M.+Urhauthor=N.+B.+La+Thangueauthor=T.+Kouzaridesauthor=R.+Prinjhaauthor=J.+Schwallerauthor=S.+Knapp&title=Generation+of+a+Selective+Small+Molecule+Inhibitor+of+the+CBP%2FP300+Bromodomain+for+Leukemia+Therapy&doi=10.1158%2F0008-5472.CAN-15-0236"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Generation of a Selective Small Molecule Inhibitor of the CBP/p300 Bromodomain for Leukemia Therapy</span></div><div class="casAuthors">Picaud, Sarah; Fedorov, Oleg; Thanasopoulou, Angeliki; Leonards, Katharina; Jones, Katherine; Meier, Julia; Olzscha, Heidi; Monteiro, Octovia; Martin, Sarah; Philpott, Martin; Tumber, Anthony; Filippakopoulos, Panagis; Yapp, Clarence; Wells, Christopher; Che, Ka Hing; Bannister, Andrew; Robson, Samuel; Kumar, Umesh; Parr, Nigel; Lee, Kevin; Lugo, Dave; Jeffrey, Philip; Taylor, Simon; Vecellio, Matteo L.; Bountra, Chas; Brennan, Paul E.; O'Mahony, Alison; Velichko, Sharlene; Mueller, Susanne; Hay, Duncan; Daniels, Danette L.; Urh, Marjeta; La Thangue, Nicholas B.; Kouzarides, Tony; Prinjha, Rab; Schwaller, Juerg; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5106-5119</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The histone acetyltransferases CBP/p300 are involved in recurrent leukemia-assocd. chromosomal translocations and are key regulators of cell growth.  Therefore, efforts to generate inhibitors of CBP/p300 are of clin. value.  We developed a specific and potent acetyl-lysine competitive protein-protein interaction inhibitor, I-CBP112, that targets the CBP/p300 bromodomains.  Exposure of human and mouse leukemic cell lines to I-CBP112 resulted in substantially impaired colony formation and induced cellular differentiation without significant cytotoxicity.  I-CBP112 significantly reduced the leukemia-initiating potential of MLL-AF9+ acute myeloid leukemia cells in a dose-dependent manner in vitro and in vivo.  Interestingly, I-CBP112 increased the cytotoxic activity of BET bromodomain inhibitor JQ1 as well as doxorubicin.  Collectively, we report the development and preclin. evaluation of a novel, potent inhibitor targeting CBP/p300 bromodomains that impairs aberrant self-renewal of leukemic cells.  The synergistic effects of I-CBP112 and current std. therapy (doxorubicin) as well as emerging treatment strategies (BET inhibition) provide new opportunities for combinatorial treatment of leukemia and potentially other cancers.  Cancer Res; 75(23); 5106-19. ©2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWk5dMbmC6K7Vg90H21EOLACvtfcHk0lj4neXnqpKQgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFensrzN&md5=da936fa34139ba27abc748db5f78c95e</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-15-0236&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-15-0236%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DThanasopoulou%26aufirst%3DA.%26aulast%3DLeonards%26aufirst%3DK.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DMeier%26aufirst%3DJ.%26aulast%3DOlzscha%26aufirst%3DH.%26aulast%3DMonteiro%26aufirst%3DO.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DBannister%26aufirst%3DA.%26aulast%3DRobson%26aufirst%3DS.%26aulast%3DKumar%26aufirst%3DU.%26aulast%3DParr%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DVecellio%26aufirst%3DM.%2BL.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DVelichko%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DHay%26aufirst%3DD.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DUrh%26aufirst%3DM.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DPrinjha%26aufirst%3DR.%26aulast%3DSchwaller%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DGeneration%2520of%2520a%2520Selective%2520Small%2520Molecule%2520Inhibitor%2520of%2520the%2520CBP%252FP300%2520Bromodomain%2520for%2520Leukemia%2520Therapy%26jtitle%3DCancer%2520Res.%26date%3D2015%26volume%3D75%26spage%3D5106%26epage%3D5119%26doi%3D10.1158%2F0008-5472.CAN-15-0236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Hammitzsch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tallant, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Mahony, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hay, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, F. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Mossawi, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Wit, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vecellio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wordsworth, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowness, P.</span><span> </span><span class="NLM_article-title">CBP30, a Selective CBP/P300 Bromodomain Inhibitor, Suppresses Human Th17 Responses</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">112</span><span class="NLM_x">, </span> <span class="NLM_fpage">10768</span><span class="NLM_x">–</span> <span class="NLM_lpage">10773</span><span class="refDoi"> DOI: 10.1073/pnas.1501956112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1073%2Fpnas.1501956112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=26261308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlSit7jI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=10768-10773&author=A.+Hammitzschauthor=C.+Tallantauthor=O.+Fedorovauthor=A.+O%E2%80%99Mahonyauthor=P.+E.+Brennanauthor=D.+A.+Hayauthor=F.+O.+Martinezauthor=M.+H.+Al-Mossawiauthor=J.+de+Witauthor=M.+Vecellioauthor=C.+Wellsauthor=P.+Wordsworthauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=P.+Bowness&title=CBP30%2C+a+Selective+CBP%2FP300+Bromodomain+Inhibitor%2C+Suppresses+Human+Th17+Responses&doi=10.1073%2Fpnas.1501956112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">CBP30, a selective CBP/p300 bromodomain inhibitor, suppresses human Th17 responses</span></div><div class="casAuthors">Hammitzsch, Ariane; Tallant, Cynthia; Fedorov, Oleg; O'Mahony, Alison; Brennan, Paul E.; Hay, Duncan A.; Martinez, Fernando O.; Al-Mossawi, M. Hussein; de Wit, Jelle; Vecellio, Matteo; Wells, Christopher; Wordsworth, Paul; Muller, Susanne; Knapp, Stefan; Bowness, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">10768-10773</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Th17 responses are crit. to a variety of human autoimmune diseases, and therapeutic targeting with monoclonal antibodies against IL-17 and IL-23 has shown considerable promise.  Here, the authors report data to support selective bromodomain blockade of the transcriptional coactivators CBP (CREB binding protein) and p300 as an alternative approach to inhibit human Th17 responses.  The authors show that CBP30 has marked mol. specificity for the bromodomains of CBP and p300, compared with 43 other bromodomains.  In unbiased cellular testing on a diverse panel of cultured primary human cells, CBP30 reduced immune cell prodn. of IL-17A and other proinflammatory cytokines.  CBP30 also inhibited IL-17A secretion by Th17 cells from healthy donors and patients with ankylosing spondylitis and psoriatic arthritis.  Transcriptional profiling of human T cells after CBP30 treatment showed a much more restricted effect on gene expression than that obsd. with the pan-BET (bromo and extraterminal domain protein family) bromodomain inhibitor JQ1.  This selective targeting of the CBP/p300 bromodomain by CBP30 will potentially lead to fewer side effects than with the broadly acting epigenetic inhibitors currently in clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4fYjOUV2xW7Vg90H21EOLACvtfcHk0lgEU0jYLiIxCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlSit7jI&md5=fb9de88ecdd6441628ae1455ac11b65c</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1501956112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1501956112%26sid%3Dliteratum%253Aachs%26aulast%3DHammitzsch%26aufirst%3DA.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DMartinez%26aufirst%3DF.%2BO.%26aulast%3DAl-Mossawi%26aufirst%3DM.%2BH.%26aulast%3Dde%2BWit%26aufirst%3DJ.%26aulast%3DVecellio%26aufirst%3DM.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DWordsworth%26aufirst%3DP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBowness%26aufirst%3DP.%26atitle%3DCBP30%252C%2520a%2520Selective%2520CBP%252FP300%2520Bromodomain%2520Inhibitor%252C%2520Suppresses%2520Human%2520Th17%2520Responses%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3D10768%26epage%3D10773%26doi%3D10.1073%2Fpnas.1501956112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Xu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Unzue, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spiliotopoulos, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nevado, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Discovery of CREBB Bromodomain Inhibitors by High-Throughput Docking and Hit Optimization Guided by Molecular Dynamics</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">1340</span><span class="NLM_x">–</span> <span class="NLM_lpage">1349</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00171</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00171" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1340-1349&author=M.+Xuauthor=A.+Unzueauthor=J.+Dongauthor=D.+Spiliotopoulosauthor=C.+Nevadoauthor=A.+Caflisch&title=Discovery+of+CREBB+Bromodomain+Inhibitors+by+High-Throughput+Docking+and+Hit+Optimization+Guided+by+Molecular+Dynamics&doi=10.1021%2Facs.jmedchem.5b00171"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47a&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00171&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00171%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DM.%26aulast%3DUnzue%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DJ.%26aulast%3DSpiliotopoulos%26aufirst%3DD.%26aulast%3DNevado%26aufirst%3DC.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DDiscovery%2520of%2520CREBB%2520Bromodomain%2520Inhibitors%2520by%2520High-Throughput%2520Docking%2520and%2520Hit%2520Optimization%2520Guided%2520by%2520Molecular%2520Dynamics%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1340%26epage%3D1349%26doi%3D10.1021%2Facs.jmedchem.5b00171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit47b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Unzue, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiedmer, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spiliotopoulos, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nevado, C.</span><span> </span><span class="NLM_article-title">Fragment-Based Design of Selective Nano-molar Ligands of the CREBBP Bromodomain</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x"> (</span><span class="NLM_issue">59</span><span class="NLM_x">) </span> <span class="NLM_fpage">1350</span><span class="NLM_x">–</span> <span class="NLM_lpage">1356</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00172</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00172" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1350-1356&issue=59&author=A.+Unzueauthor=M.+Xuauthor=J.+Dongauthor=L.+Wiedmerauthor=D.+Spiliotopoulosauthor=A.+Caflischauthor=C.+Nevado&title=Fragment-Based+Design+of+Selective+Nano-molar+Ligands+of+the+CREBBP+Bromodomain&doi=10.1021%2Facs.jmedchem.5b00172"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47b&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00172&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00172%26sid%3Dliteratum%253Aachs%26aulast%3DUnzue%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DJ.%26aulast%3DWiedmer%26aufirst%3DL.%26aulast%3DSpiliotopoulos%26aufirst%3DD.%26aulast%3DCaflisch%26aufirst%3DA.%26aulast%3DNevado%26aufirst%3DC.%26atitle%3DFragment-Based%2520Design%2520of%2520Selective%2520Nano-molar%2520Ligands%2520of%2520the%2520CREBBP%2520Bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D59%26spage%3D1350%26epage%3D1356%26doi%3D10.1021%2Facs.jmedchem.5b00172" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Taylor, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Côté, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hewitt, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pastor, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leblanc, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nasveschuk, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romero, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crawford, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cantone, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayaram, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setser, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beresini, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Leon Boenig, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conery, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cummings, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dakin, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flynn, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, O. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaufman, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keller, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiefer, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pardo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsui, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zawadzke, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sims, R. J.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cochran, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magnuson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albrecht, B. K.</span><span> </span><span class="NLM_article-title">Fragment-Based Discovery of a Selective and Cell-Active Benzodiazepinone CBP/EP300 Bromodomain Inhibitor (CPI-637)</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">531</span><span class="NLM_x">–</span> <span class="NLM_lpage">536</span><span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00075</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00075" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC28Xkt1Cms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=531-536&author=A.+M.+Taylorauthor=A.+C%C3%B4t%C3%A9author=M.+C.+Hewittauthor=R.+Pastorauthor=Y.+Leblancauthor=C.+G.+Nasveschukauthor=F.+A.+Romeroauthor=T.+D.+Crawfordauthor=N.+Cantoneauthor=H.+Jayaramauthor=J.+Setserauthor=J.+Murrayauthor=M.+H.+Beresiniauthor=G.+de+Leon+Boenigauthor=Z.+Chenauthor=A.+R.+Coneryauthor=R.+T.+Cummingsauthor=L.+A.+Dakinauthor=E.+M.+Flynnauthor=O.+W.+Huangauthor=S.+Kaufmanauthor=P.+J.+Kellerauthor=J.+R.+Kieferauthor=T.+Laiauthor=Y.+Liauthor=J.+Liaoauthor=W.+Liuauthor=H.+Luauthor=E.+Pardoauthor=V.+Tsuiauthor=J.+Wangauthor=Y.+Wangauthor=Z.+Xuauthor=F.+Yanauthor=D.+Yuauthor=L.+Zawadzkeauthor=X.+Zhuauthor=X.+Zhuauthor=R.+J.+Simsauthor=A.+G.+Cochranauthor=S.+Bellonauthor=J.+E.+Audiaauthor=S.+Magnusonauthor=B.+K.+Albrecht&title=Fragment-Based+Discovery+of+a+Selective+and+Cell-Active+Benzodiazepinone+CBP%2FEP300+Bromodomain+Inhibitor+%28CPI-637%29&doi=10.1021%2Facsmedchemlett.6b00075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of a Selective and Cell-Active Benzodiazepinone CBP/EP300 Bromodomain Inhibitor (CPI-637)</span></div><div class="casAuthors">Taylor, Alexander M.; Cote, Alexandre; Hewitt, Michael C.; Pastor, Richard; Leblanc, Yves; Nasveschuk, Christopher G.; Romero, F. Anthony; Crawford, Terry D.; Cantone, Nico; Jayaram, Hariharan; Setser, Jeremy; Murray, Jeremy; Beresini, Maureen H.; de Leon Boenig, Gladys; Chen, Zhongguo; Conery, Andrew R.; Cummings, Richard T.; Dakin, Leslie A.; Flynn, E. Megan; Huang, Oscar W.; Kaufman, Susan; Keller, Patricia J.; Kiefer, James R.; Lai, Tommy; Li, Yingjie; Liao, Jiangpeng; Liu, Wenfeng; Lu, Henry; Pardo, Eneida; Tsui, Vickie; Wang, Jian; Wang, Yongyun; Xu, Zhaowu; Yan, Fen; Yu, Dong; Zawadzke, Laura; Zhu, Xiaoqin; Zhu, Xiaoyu; Sims, Robert J.; Cochran, Andrea G.; Bellon, Steve; Audia, James E.; Magnuson, Steven; Albrecht, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">531-536</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Benzo-fused heterocycles, particularly benzodiazepinones such as I, were prepd. as inhibitors of the bromodomain-contg. proteins CBP (CREB, cAMP response element binding protein) and EP300 (adenoviral E1A-binding protein of 300 kDa) for potential use as antitumor agents; the core benzodiazepinone moiety was identified using fragment-based screening.  The structures of four benzodiazepinones, including I, bound to the bromodomain fragment of human CREB were detd. by X-ray crystallog.  The free energy, enthalpy, and entropy of the binding of I to CREB and its selectivity for CREB and EP300 over other bromodomain-contg. proteins was detd.; I inhibited expression of the transcription factor MYC in human plasmacytoma cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNm7Blycl0-7Vg90H21EOLACvtfcHk0lhtlpujq62rlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xkt1Cms7c%253D&md5=8440a9bb3460ff66d43f15041dbbc19d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00075%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DPastor%26aufirst%3DR.%26aulast%3DLeblanc%26aufirst%3DY.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DCantone%26aufirst%3DN.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DSetser%26aufirst%3DJ.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3Dde%2BLeon%2BBoenig%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DConery%26aufirst%3DA.%2BR.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DHuang%26aufirst%3DO.%2BW.%26aulast%3DKaufman%26aufirst%3DS.%26aulast%3DKeller%26aufirst%3DP.%2BJ.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DLai%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLiao%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DLu%26aufirst%3DH.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DYu%26aufirst%3DD.%26aulast%3DZawadzke%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DMagnuson%26aufirst%3DS.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26atitle%3DFragment-Based%2520Discovery%2520of%2520a%2520Selective%2520and%2520Cell-Active%2520Benzodiazepinone%2520CBP%252FEP300%2520Bromodomain%2520Inhibitor%2520%2528CPI-637%2529%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D531%26epage%3D536%26doi%3D10.1021%2Facsmedchemlett.6b00075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Popp, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tallant, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bracher, F.</span><span> </span><span class="NLM_article-title">Development of Selective CBP/P300 Benzoxazepine Bromodomain Inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">8889</span><span class="NLM_x">–</span> <span class="NLM_lpage">8912</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00774</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00774" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFKis77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8889-8912&author=T.+A.+Poppauthor=C.+Tallantauthor=C.+Rogersauthor=O.+Fedorovauthor=P.+E.+Brennanauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=F.+Bracher&title=Development+of+Selective+CBP%2FP300+Benzoxazepine+Bromodomain+Inhibitors&doi=10.1021%2Facs.jmedchem.6b00774"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Selective CBP/P300 Benzoxazepine Bromodomain Inhibitors</span></div><div class="casAuthors">Popp, Tobias A.; Tallant, Cynthia; Rogers, Catherine; Fedorov, Oleg; Brennan, Paul E.; Mueller, Susanne; Knapp, Stefan; Bracher, Franz</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8889-8912</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">CBP (CREB (cAMP responsive element binding protein) binding protein (CREBBP)) and P300 (adenovirus E1A-assocd. 300 kDa protein) are two closely related histone acetyltransferases (HATs) that play a key role in the regulation of gene transcription.  Both proteins contain a bromodomain flanking the HAT catalytic domain that is important for the targeting of CBP/P300 to chromatin and which offers an opportunity for the development of protein-protein interaction inhibitors.  Here the authors present the development of CBP/P300 bromodomain inhibitors with 2,3,4,5-tetrahydro-1,4-benzoxazepine backbone, an N-acetyl-lysine mimetic scaffold that led to the recent development of the chem. probe I-CBP112.  The authors present comprehensive SAR of this inhibitor class as well as demonstration of cellular on target activity of the most potent and selective inhibitor I, which showed 134 nM affinity for CBP with excellent selectivity over other bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwAZFuNP0NkLVg90H21EOLACvtfcHk0lhtlpujq62rlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFKis77E&md5=6bd914d24abf00a8f074bf1918fb4ee1</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00774&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00774%26sid%3Dliteratum%253Aachs%26aulast%3DPopp%26aufirst%3DT.%2BA.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBracher%26aufirst%3DF.%26atitle%3DDevelopment%2520of%2520Selective%2520CBP%252FP300%2520Benzoxazepine%2520Bromodomain%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8889%26epage%3D8912%26doi%3D10.1021%2Facs.jmedchem.6b00774" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Denny, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flick, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coe, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langille, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basak, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stock, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sahasrabudhe, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonin, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hay, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pletcher, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chekler, E. L. P.</span><span> </span><span class="NLM_article-title">Structure-Based Design of Highly Selective Inhibitors of the CREB Binding Protein Bromodomain</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">5349</span><span class="NLM_x">–</span> <span class="NLM_lpage">5363</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01839</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01839" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1Glu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=5349-5363&author=R.+A.+Dennyauthor=A.+C.+Flickauthor=J.+W.+Coeauthor=J.+Langilleauthor=A.+Basakauthor=S.+Liuauthor=I.+A.+Stockauthor=P.+Sahasrabudheauthor=P.+D.+Boninauthor=D.+A.+Hayauthor=P.+E.+Brennanauthor=M.+T.+Pletcherauthor=L.+H.+Jonesauthor=E.+L.+P.+Chekler&title=Structure-Based+Design+of+Highly+Selective+Inhibitors+of+the+CREB+Binding+Protein+Bromodomain&doi=10.1021%2Facs.jmedchem.6b01839"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Highly Selective Inhibitors of the CREB Binding Protein Bromodomain</span></div><div class="casAuthors">Denny, R. Aldrin; Flick, Andrew C.; Coe, Jotham; Langille, Jonathan; Basak, Arindrajit; Liu, Shenping; Stock, Ingrid; Sahasrabudhe, Parag; Bonin, Paul; Hay, Duncan A.; Brennan, Paul E.; Pletcher, Mathew; Jones, Lyn H.; Chekler, Eugene L. Piatnitski</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5349-5363</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chem. probes are required for preclin. target validation to interrogate novel biol. targets and pathways.  Selective inhibitors of the CREB binding protein (CREBBP)/EP300 bromodomains are required to facilitate the elucidation of biol. assocd. with these important epigenetic targets.  Medicinal chem. optimization that paid particular attention to physiochem. properties delivered chem. probes with desirable potency, selectivity, and permeability attributes.  An important feature of the optimization process was the successful application of rational structure-based drug design to address bromodomain selectivity issues (particularly against the structurally related BRD4 protein).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLWkqF-qAmdbVg90H21EOLACvtfcHk0ljZqw5K8QqfvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1Glu70%253D&md5=23676a973abba59088dedd5502877105</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01839%26sid%3Dliteratum%253Aachs%26aulast%3DDenny%26aufirst%3DR.%2BA.%26aulast%3DFlick%26aufirst%3DA.%2BC.%26aulast%3DCoe%26aufirst%3DJ.%2BW.%26aulast%3DLangille%26aufirst%3DJ.%26aulast%3DBasak%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DStock%26aufirst%3DI.%2BA.%26aulast%3DSahasrabudhe%26aufirst%3DP.%26aulast%3DBonin%26aufirst%3DP.%2BD.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DPletcher%26aufirst%3DM.%2BT.%26aulast%3DJones%26aufirst%3DL.%2BH.%26aulast%3DChekler%26aufirst%3DE.%2BL.%2BP.%26atitle%3DStructure-Based%2520Design%2520of%2520Highly%2520Selective%2520Inhibitors%2520of%2520the%2520CREB%2520Binding%2520Protein%2520Bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D5349%26epage%3D5363%26doi%3D10.1021%2Facs.jmedchem.6b01839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Spiliotopoulos, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wamhoff, E.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deerain, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, J.-R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aretz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rademacher, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Virtual Screen to NMR (VS2NMR): Discovery of Fragment Hits for the CBP Bromodomain</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">2472</span><span class="NLM_x">–</span> <span class="NLM_lpage">2478</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2017.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1016%2Fj.bmcl.2017.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=28410781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtVertrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=2472-2478&author=D.+Spiliotopoulosauthor=J.+Zhuauthor=E.-C.+Wamhoffauthor=N.+Deerainauthor=J.-R.+Marchandauthor=J.+Aretzauthor=C.+Rademacherauthor=A.+Caflisch&title=Virtual+Screen+to+NMR+%28VS2NMR%29%3A+Discovery+of+Fragment+Hits+for+the+CBP+Bromodomain&doi=10.1016%2Fj.bmcl.2017.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screen to NMR (VS2NMR): Discovery of fragment hits for the CBP bromodomain</span></div><div class="casAuthors">Spiliotopoulos, Dimitrios; Zhu, Jian; Wamhoff, Eike-Christian; Deerain, Nicholas; Marchand, Jean-Remy; Aretz, Jonas; Rademacher, Christoph; Caflisch, Amedeo</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2472-2478</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Overexpression of the CREB-binding protein (CBP), a bromodomain-contg. transcription coactivator involved in a variety of cellular processes, has been obsd. in several types of cancer with a correlation to aggressiveness.  The authors have screened a library of nearly 1500 fragments by high-throughput docking into the CBP bromodomain followed by binding energy evaluation using a force field with electrostatic solvation.  Twenty of the 39 fragments selected by virtual screening are pos. in one or more ligand-obsd. NMR expts.  Four crystal structures of the CBP bromodomain in complex with in silico screening hits validate the pose predicted by docking.  Thus, the success ratio of the high-throughput docking procedure is 50% or 10% if one considers the validation by ligand-obsd. NMR spectroscopy or x-ray crystallog., resp.  Compds. 1 and 3 show favorable ligand efficiency in two different in vitro binding assays.  The structure of the CBP bromodomain in the complex with the brominated pyrrole 1 suggests fragment growing by Suzuki coupling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkYM7NMX_QGLVg90H21EOLACvtfcHk0ljZqw5K8QqfvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtVertrg%253D&md5=d71ad8368839c3026fabf75197ba2ac9</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DSpiliotopoulos%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DWamhoff%26aufirst%3DE.-C.%26aulast%3DDeerain%26aufirst%3DN.%26aulast%3DMarchand%26aufirst%3DJ.-R.%26aulast%3DAretz%26aufirst%3DJ.%26aulast%3DRademacher%26aufirst%3DC.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DVirtual%2520Screen%2520to%2520NMR%2520%2528VS2NMR%2529%253A%2520Discovery%2520of%2520Fragment%2520Hits%2520for%2520the%2520CBP%2520Bromodomain%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D2472%26epage%3D2478%26doi%3D10.1016%2Fj.bmcl.2017.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Hügle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lucas, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ostrovskyi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Regenass, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerhardt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Einsle, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hau, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breit, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Günther, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wohlwend, D.</span><span> </span><span class="NLM_article-title">Beyond the BET Family: Targeting CBP/p300 with 4-Acyl Pyrroles</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">12476</span><span class="NLM_x">–</span> <span class="NLM_lpage">12480</span><span class="refDoi"> DOI: 10.1002/anie.201705516</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1002%2Fanie.201705516" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVCrt73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=12476-12480&author=M.+H%C3%BCgleauthor=X.+Lucasauthor=D.+Ostrovskyiauthor=P.+Regenassauthor=S.+Gerhardtauthor=O.+Einsleauthor=M.+Hauauthor=M.+Jungauthor=B.+Breitauthor=S.+G%C3%BCntherauthor=D.+Wohlwend&title=Beyond+the+BET+Family%3A+Targeting+CBP%2Fp300+with+4-Acyl+Pyrroles&doi=10.1002%2Fanie.201705516"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond the BET Family: Targeting CBP/p300 with 4-Acyl Pyrroles</span></div><div class="casAuthors">Huegle, Martin; Lucas, Xavier; Ostrovskyi, Dmytro; Regenass, Pierre; Gerhardt, Stefan; Einsle, Oliver; Hau, Mirjam; Jung, Manfred; Breit, Bernhard; Guenther, Stefan; Wohlwend, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">12476-12480</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Bromodomain and extra-terminal domain (BET) inhibitors are widely used both as chem. tools to study the biol. role of their targets in living organisms and as candidates for drug development against several cancer variants and human disorders.  However, non-BET bromodomains such as those in p300 and CBP are less studied.  XDM-CBP is a highly potent and selective inhibitor for the bromodomains of CBP and p300 derived from a pan-selective BET BRD-binding fragment.  Along with x-ray crystal-structure anal. and thermodn. profiling, XDM-CBP was used in screenings of several cancer cell lines in vitro to study its inhibitory potential on cancer cell proliferation.  XDM-CBP is demonstrated to be a potent and selective CBP/p300 inhibitor that acts on specific cancer cell lines, in particular malignant melanoma, breast cancer, and leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsOlQ2hI7S9bVg90H21EOLACvtfcHk0ljZqw5K8QqfvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVCrt73I&md5=f4e029831a971663cf51ca3ee485f43e</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fanie.201705516&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201705516%26sid%3Dliteratum%253Aachs%26aulast%3DH%25C3%25BCgle%26aufirst%3DM.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DOstrovskyi%26aufirst%3DD.%26aulast%3DRegenass%26aufirst%3DP.%26aulast%3DGerhardt%26aufirst%3DS.%26aulast%3DEinsle%26aufirst%3DO.%26aulast%3DHau%26aufirst%3DM.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DBreit%26aufirst%3DB.%26aulast%3DG%25C3%25BCnther%26aufirst%3DS.%26aulast%3DWohlwend%26aufirst%3DD.%26atitle%3DBeyond%2520the%2520BET%2520Family%253A%2520Targeting%2520CBP%252Fp300%2520with%25204-Acyl%2520Pyrroles%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D12476%26epage%3D12480%26doi%3D10.1002%2Fanie.201705516" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Romero, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsiu, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albrecht, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beresini, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Leon Boenig, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bronner, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, E. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choo, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crawford, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cyr, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Almeida Nagata, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gascoigne, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grogan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatzivassiliou, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunsaker, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaufman, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ly, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maher, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merchant, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ran, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, B.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magnuson, S.</span><span> </span><span class="NLM_article-title">GNE-781, A Highly Advanced Potent and Selective Bromodomain Inhibitor of CBP/P300</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">9162</span><span class="NLM_x">–</span> <span class="NLM_lpage">9183</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00796</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00796" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9162-9183&author=F.+A.+Romeroauthor=J.+Murrayauthor=K.+W.+Laiauthor=V.+Tsiuauthor=B.+K.+Albrechtauthor=M.+H.+Beresiniauthor=G.+de+Leon+Boenigauthor=S.+M.+Bronnerauthor=E.+W.+Chanauthor=K.+Chenauthor=Z.+Chenauthor=E.+F.+Chooauthor=K.+Clarkauthor=T.+D.+Crawfordauthor=P.+Cyrauthor=D.+De+Almeida+Nagataauthor=K.+E.+Gascoigneauthor=J.+Groganauthor=G.+Hatzivassiliouauthor=T.+L.+Hunsakerauthor=S.+Kaufmanauthor=Y.+Liauthor=J.+Liaoauthor=W.+Liuauthor=J.+Lyauthor=J.+Maherauthor=M.+Merchantauthor=Y.+Ranauthor=A.+M.+Taylorauthor=J.+Waiauthor=F.+Wangauthor=X.+Weiauthor=D.+Yuauthor=B.-Y.+Zhuauthor=X.+Zhuauthor=S.+Magnuson&title=GNE-781%2C+A+Highly+Advanced+Potent+and+Selective+Bromodomain+Inhibitor+of+CBP%2FP300&doi=10.1021%2Facs.jmedchem.7b00796"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00796%26sid%3Dliteratum%253Aachs%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DLai%26aufirst%3DK.%2BW.%26aulast%3DTsiu%26aufirst%3DV.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3Dde%2BLeon%2BBoenig%26aufirst%3DG.%26aulast%3DBronner%26aufirst%3DS.%2BM.%26aulast%3DChan%26aufirst%3DE.%2BW.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DChoo%26aufirst%3DE.%2BF.%26aulast%3DClark%26aufirst%3DK.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DCyr%26aufirst%3DP.%26aulast%3DDe%2BAlmeida%2BNagata%26aufirst%3DD.%26aulast%3DGascoigne%26aufirst%3DK.%2BE.%26aulast%3DGrogan%26aufirst%3DJ.%26aulast%3DHatzivassiliou%26aufirst%3DG.%26aulast%3DHunsaker%26aufirst%3DT.%2BL.%26aulast%3DKaufman%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLiao%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DLy%26aufirst%3DJ.%26aulast%3DMaher%26aufirst%3DJ.%26aulast%3DMerchant%26aufirst%3DM.%26aulast%3DRan%26aufirst%3DY.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DWai%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DB.-Y.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DGNE-781%252C%2520A%2520Highly%2520Advanced%2520Potent%2520and%2520Selective%2520Bromodomain%2520Inhibitor%2520of%2520CBP%252FP300%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D9162%26epage%3D9183%26doi%3D10.1021%2Facs.jmedchem.7b00796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Crawford, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romero, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsui, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Leon Boenig, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noland, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ly, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choo, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunsaker, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, E. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merchant, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kharbanda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gascoigne, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaufman, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beresini, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conery, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Côté, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayaram, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiefer, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kleinheinz, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maher, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pardo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poy, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spillane, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yen, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zawadzke, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cummings, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cochran, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albrecht, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magnuson, S.</span><span> </span><span class="NLM_article-title">Discovery of a Potent and Selective in Vivo Probe (GNE-272) for the Bromodomains of CBP/EP300</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">10549</span><span class="NLM_x">–</span> <span class="NLM_lpage">10563</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01022</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01022" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFGhsb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=10549-10563&author=T.+D.+Crawfordauthor=F.+A.+Romeroauthor=K.+W.+Laiauthor=V.+Tsuiauthor=A.+M.+Taylorauthor=G.+de+Leon+Boenigauthor=C.+L.+Nolandauthor=J.+Murrayauthor=J.+Lyauthor=E.+F.+Chooauthor=T.+L.+Hunsakerauthor=E.+W.+Chanauthor=M.+Merchantauthor=S.+Kharbandaauthor=K.+E.+Gascoigneauthor=S.+Kaufmanauthor=M.+H.+Beresiniauthor=J.+Liaoauthor=W.+Liuauthor=K.+X.+Chenauthor=Z.+Chenauthor=A.+R.+Coneryauthor=A.+C%C3%B4t%C3%A9author=H.+Jayaramauthor=Y.+Jiangauthor=J.+R.+Kieferauthor=T.+Kleinheinzauthor=Y.+Liauthor=J.+Maherauthor=E.+Pardoauthor=F.+Poyauthor=K.+L.+Spillaneauthor=F.+Wangauthor=J.+Wangauthor=X.+Weiauthor=Z.+Xuauthor=Z.+Xuauthor=I.+Yenauthor=L.+Zawadzkeauthor=X.+Zhuauthor=S.+Bellonauthor=R.+Cummingsauthor=A.+G.+Cochranauthor=B.+K.+Albrechtauthor=S.+Magnuson&title=Discovery+of+a+Potent+and+Selective+in+Vivo+Probe+%28GNE-272%29+for+the+Bromodomains+of+CBP%2FEP300&doi=10.1021%2Facs.jmedchem.6b01022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent and Selective in Vivo Probe (GNE-272) for the Bromodomains of CBP/EP300</span></div><div class="casAuthors">Crawford, Terry D.; Romero, F. Anthony; Lai, Kwong Wah; Tsui, Vickie; Taylor, Alexander M.; de Leon Boenig, Gladys; Noland, Cameron L.; Murray, Jeremy; Ly, Justin; Choo, Edna F.; Hunsaker, Thomas L.; Chan, Emily W.; Merchant, Mark; Kharbanda, Samir; Gascoigne, Karen E.; Kaufman, Susan; Beresini, Maureen H.; Liao, Jiangpeng; Liu, Wenfeng; Chen, Kevin X.; Chen, Zhongguo; Conery, Andrew R.; Cote, Alexandre; Jayaram, Hariharan; Jiang, Ying; Kiefer, James R.; Kleinheinz, Tracy; Li, Yingjie; Maher, Jonathan; Pardo, Eneida; Poy, Florence; Spillane, Kerry L.; Wang, Fei; Wang, Jian; Wei, Xiaocang; Xu, Zhaowu; Xu, Zhongya; Yen, Ivana; Zawadzke, Laura; Zhu, Xiaoyu; Bellon, Steven; Cummings, Richard; Cochran, Andrea G.; Albrecht, Brian K.; Magnuson, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10549-10563</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The single bromodomain of the closely related transcriptional regulators CBP/EP300 is a target of much recent interest in cancer and immune system regulation.  A co-crystal structure of a ligand-efficient screening hit and the CBP bromodomain guided initial design targeting the LPF shelf, ZA loop, and acetylated lysine binding regions.  Structure-activity relationship studies allowed the authors to identify a more potent analog.  Optimization of permeability and microsomal stability and subsequent improvement of mouse hepatocyte stability afforded 59 (GNE-272, TR-FRET IC50 = 0.02 μM, BRET IC50 = 0.41 μM, BRD4(1) IC50 = 13 μM) that retained the best balance of cell potency, selectivity, and in vivo PK.  GNE-272 showed a marked antiproliferative effect in hematol. cancer cell lines and modulates MYC expression in vivo that corresponds with antitumor activity in an AML tumor model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6T5ODn2Q-2LVg90H21EOLACvtfcHk0lhEaKMB4wA9wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFGhsb%252FO&md5=22ed46cb031022aa6aff23c20d07e7dd</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01022%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DLai%26aufirst%3DK.%2BW.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3Dde%2BLeon%2BBoenig%26aufirst%3DG.%26aulast%3DNoland%26aufirst%3DC.%2BL.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DLy%26aufirst%3DJ.%26aulast%3DChoo%26aufirst%3DE.%2BF.%26aulast%3DHunsaker%26aufirst%3DT.%2BL.%26aulast%3DChan%26aufirst%3DE.%2BW.%26aulast%3DMerchant%26aufirst%3DM.%26aulast%3DKharbanda%26aufirst%3DS.%26aulast%3DGascoigne%26aufirst%3DK.%2BE.%26aulast%3DKaufman%26aufirst%3DS.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3DLiao%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DK.%2BX.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DConery%26aufirst%3DA.%2BR.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DKleinheinz%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DMaher%26aufirst%3DJ.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DSpillane%26aufirst%3DK.%2BL.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DYen%26aufirst%3DI.%26aulast%3DZawadzke%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520a%2520Potent%2520and%2520Selective%2520in%2520Vivo%2520Probe%2520%2528GNE-272%2529%2520for%2520the%2520Bromodomains%2520of%2520CBP%252FEP300%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D10549%26epage%3D10563%26doi%3D10.1021%2Facs.jmedchem.6b01022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Zhou, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shang, Z.</span><span> </span><span class="NLM_article-title">Fluorine Bonding – How Does It Work In Protein–Ligand Interactions?</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">2344</span><span class="NLM_x">–</span> <span class="NLM_lpage">2355</span><span class="refDoi"> DOI: 10.1021/ci9002393</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci9002393" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2017&pages=2344-2355&author=P.+Zhouauthor=J.+Zouauthor=F.+Tianauthor=Z.+Shang&title=Fluorine+Bonding+%E2%80%93+How+Does+It+Work+In+Protein%E2%80%93Ligand+Interactions%3F&doi=10.1021%2Fci9002393"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fci9002393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci9002393%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DP.%26aulast%3DZou%26aufirst%3DJ.%26aulast%3DTian%26aufirst%3DF.%26aulast%3DShang%26aufirst%3DZ.%26atitle%3DFluorine%2520Bonding%2520%25E2%2580%2593%2520How%2520Does%2520It%2520Work%2520In%2520Protein%25E2%2580%2593Ligand%2520Interactions%253F%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D49%26spage%3D2344%26epage%3D2355%26doi%3D10.1021%2Fci9002393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="note"><p class="first last">Details regarding the computational docking protocol are provided in the <a class="ref internalNav" href="#notes1" aria-label="Supporting Information">Supporting Information</a>.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Bennett, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tallant, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monteiro, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meier, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gamble, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitsky, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nunez-Alonso, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haendler, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Discovery of a Chemical Tool Inhibitor Targeting the Bromodomains of TRIM24 and BRPF</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">1642</span><span class="NLM_x">–</span> <span class="NLM_lpage">1647</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00458</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00458" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotlCiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1642-1647&author=J.+Bennettauthor=O.+Fedorovauthor=C.+Tallantauthor=O.+Monteiroauthor=J.+Meierauthor=V.+Gambleauthor=P.+Savitskyauthor=G.+A.+Nunez-Alonsoauthor=B.+Haendlerauthor=C.+Rogersauthor=P.+E.+Brennanauthor=S.+M%C3%BCllerauthor=S.+Knapp&title=Discovery+of+a+Chemical+Tool+Inhibitor+Targeting+the+Bromodomains+of+TRIM24+and+BRPF&doi=10.1021%2Facs.jmedchem.5b00458"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Chemical Tool Inhibitor Targeting the Bromodomains of TRIM24 and BRPF</span></div><div class="casAuthors">Bennett, James; Fedorov, Oleg; Tallant, Cynthia; Monteiro, Octovia; Meier, Julia; Gamble, Vicky; Savitsky, Pavel; Nunez-Alonso, Graciela A.; Haendler, Bernard; Rogers, Catherine; Brennan, Paul E.; Muller, Susanne; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1642-1647</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">TRIM24 is a transcriptional regulator as well as an E3 ubiquitin ligase.  It is overexpressed in diverse tumors, and high expression levels have been linked to poor prognosis in breast cancer patients.  TRIM24 contains a PHD/bromodomain offering the opportunity to develop protein interaction inhibitors that target this protein interaction module.  Here the authors identified potent acetyl-lysine mimetic benzimidazolones TRIM24 bromodomain inhibitors.  The best compd. of this series is a selective BRPF1B/TRIM24 dual inhibitor that bound with a KD of 137 and 222 nM, resp., but exerted good selectivity over other bromodomains.  Cellular activity of the inhibitor was demonstrated using FRAP assays as well as cell viability data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHwwYtxVWed7Vg90H21EOLACvtfcHk0liFxyCNUl28pA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotlCiurg%253D&md5=9464791bb52ab3b14cd20bcdd4f407d4</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00458&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00458%26sid%3Dliteratum%253Aachs%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DMonteiro%26aufirst%3DO.%26aulast%3DMeier%26aufirst%3DJ.%26aulast%3DGamble%26aufirst%3DV.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DNunez-Alonso%26aufirst%3DG.%2BA.%26aulast%3DHaendler%26aufirst%3DB.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520a%2520Chemical%2520Tool%2520Inhibitor%2520Targeting%2520the%2520Bromodomains%2520of%2520TRIM24%2520and%2520BRPF%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1642%26epage%3D1647%26doi%3D10.1021%2Facs.jmedchem.5b00458" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="note"><p class="first last">CBP HTRF IC50 of published CBP inhibitors reported here as measured by Genentech assays.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><div class="note"><p class="first last">For a similar approach for the development of BET inhibitors, see:</p></div><span class="NLM_contrib-group">Sharp, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garnier, J.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, D. C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burns, C. J.</span><span> </span><span class="NLM_article-title">Evaluation of Functional Groups as Acetyl-Lysine Mimetics for BET Bromodomain Inhibition</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1834</span><span class="NLM_x">–</span> <span class="NLM_lpage">1842</span><span class="refDoi"> DOI: 10.1039/C4MD00182F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1039%2FC4MD00182F" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1834-1842&author=P.+P.+Sharpauthor=J.-M.+Garnierauthor=D.+C.+S.+Huangauthor=C.+J.+Burns&title=Evaluation+of+Functional+Groups+as+Acetyl-Lysine+Mimetics+for+BET+Bromodomain+Inhibition&doi=10.1039%2FC4MD00182F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1039%2FC4MD00182F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00182F%26sid%3Dliteratum%253Aachs%26aulast%3DSharp%26aufirst%3DP.%2BP.%26aulast%3DGarnier%26aufirst%3DJ.-M.%26aulast%3DHuang%26aufirst%3DD.%2BC.%2BS.%26aulast%3DBurns%26aufirst%3DC.%2BJ.%26atitle%3DEvaluation%2520of%2520Functional%2520Groups%2520as%2520Acetyl-Lysine%2520Mimetics%2520for%2520BET%2520Bromodomain%2520Inhibition%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D1834%26epage%3D1842%26doi%3D10.1039%2FC4MD00182F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="note"><p class="first last">We found that high potencies in the biochemical assays were required for reasonable potencies to be detected in the MYC assay. Less biochemically potent inhibitors displayed a large cell-shift.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span>BROMOscan<sup>®</sup> recombinant protein binding assay, DiscoveRx Inc. <a href="http://www.discoverx.com" class="extLink">www.discoverx.com</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=BROMOscan%C2%AE+recombinant+protein+binding+assay%2C+DiscoveRx+Inc.+www.discoverx.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Crawford, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsui, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flynn, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Côté, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beresini, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burdick, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cummings, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dakin, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duplessis, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Good, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hewitt, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, H.-R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayaram, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiefer, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nasveschuk, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pardo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poy, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romero, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zawadzke, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albrecht, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magnuson, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellon, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cochran, A. G.</span><span> </span><span class="NLM_article-title">Diving Into the Water: Inducible Binding Conformations for BRD4, TAF1(2), BRD9, and CECR2 Bromodomains</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">5391</span><span class="NLM_x">–</span> <span class="NLM_lpage">5402</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00264</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00264" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC28XosFSnsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5391-5402&author=T.+D.+Crawfordauthor=V.+Tsuiauthor=E.+M.+Flynnauthor=S.+Wangauthor=A.+M.+Taylorauthor=A.+C%C3%B4t%C3%A9author=J.+E.+Audiaauthor=M.+H.+Beresiniauthor=D.+J.+Burdickauthor=R.+T.+Cummingsauthor=L.+A.+Dakinauthor=M.+Duplessisauthor=A.+C.+Goodauthor=M.+C.+Hewittauthor=H.-R.+Huangauthor=H.+Jayaramauthor=J.+R.+Kieferauthor=Y.+Jiangauthor=J.+M.+Murrayauthor=C.+G.+Nasveschukauthor=E.+Pardoauthor=F.+Poyauthor=F.+A.+Romeroauthor=Y.+Tangauthor=J.+Wangauthor=Z.+Xuauthor=L.+E.+Zawadzkeauthor=X.+Zhuauthor=B.+K.+Albrechtauthor=S.+R.+Magnusonauthor=S.+F.+Bellonauthor=A.+G.+Cochran&title=Diving+Into+the+Water%3A+Inducible+Binding+Conformations+for+BRD4%2C+TAF1%282%29%2C+BRD9%2C+and+CECR2+Bromodomains&doi=10.1021%2Facs.jmedchem.6b00264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Diving into the Water: Inducible Binding Conformations for BRD4, TAF1(2), BRD9, and CECR2 Bromodomains</span></div><div class="casAuthors">Crawford, Terry D.; Tsui, Vickie; Flynn, E. Megan; Wang, Shumei; Taylor, Alexander M.; Cote, Alexandre; Audia, James E.; Beresini, Maureen H.; Burdick, Daniel J.; Cummings, Richard; Dakin, Les A.; Duplessis, Martin; Good, Andrew C.; Hewitt, Michael C.; Huang, Hon-Ren; Jayaram, Hariharan; Kiefer, James R.; Jiang, Ying; Murray, Jeremy; Nasveschuk, Christopher G.; Pardo, Eneida; Poy, Florence; Romero, F. Anthony; Tang, Yong; Wang, Jian; Xu, Zhaowu; Zawadzke, Laura E.; Zhu, Xiaoyu; Albrecht, Brian K.; Magnuson, Steven R.; Bellon, Steve; Cochran, Andrea G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5391-5402</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The biol. role played by non-BET bromodomains remains poorly understood, and it is therefore imperative to identify potent and highly selective inhibitors to effectively explore the biol. of individual bromodomain proteins.  A ligand-efficient nonselective bromodomain inhibitor was identified from a 6-Me pyrrolopyridone fragment.  Small hydrophobic substituents replacing the N-Me group were designed directing toward the conserved bromodomain water pocket, and two distinct binding conformations were then obsd.  The substituents either directly displaced and rearranged the conserved solvent network, as in BRD4(1) and TAF1(2), or induced a narrow hydrophobic channel adjacent to the lipophilic shelf, as in BRD9 and CECR2.  The preference of distinct substituents for individual bromodomains provided selectivity handles useful for future lead optimization efforts for selective BRD9, CECR2, and TAF1(2) inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTLnuVCIKdHbVg90H21EOLACvtfcHk0liFxyCNUl28pA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosFSnsbs%253D&md5=bf92e94c5fd3ee36bd8441294ca85185</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00264%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3DBurdick%26aufirst%3DD.%2BJ.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DDuplessis%26aufirst%3DM.%26aulast%3DGood%26aufirst%3DA.%2BC.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DHuang%26aufirst%3DH.-R.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DMurray%26aufirst%3DJ.%2BM.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZawadzke%26aufirst%3DL.%2BE.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DMagnuson%26aufirst%3DS.%2BR.%26aulast%3DBellon%26aufirst%3DS.%2BF.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26atitle%3DDiving%2520Into%2520the%2520Water%253A%2520Inducible%2520Binding%2520Conformations%2520for%2520BRD4%252C%2520TAF1%25282%2529%252C%2520BRD9%252C%2520and%2520CECR2%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5391%26epage%3D5402%26doi%3D10.1021%2Facs.jmedchem.6b00264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47a','cit47b'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':[],'ref57':['cit57'],'ref58':[],'ref59':['cit59'],'ref60':[],'ref61':['cit61'],'ref62':['cit62']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 10 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Michael Brand, James Clayton, Mustafa Moroglu, Matthias Schiedel, Sarah Picaud, Joseph P. Bluck, Anna Skwarska, Hannah Bolland, Anthony K. N. Chan, Corentine M. C. Laurin, Amy R. Scorah, Larissa See, Timothy P. C. Rooney, Katrina H. Andrews, Oleg Fedorov, Gabriella Perell, Prakriti Kalra, Kayla B. Vinh, Wilian A. Cortopassi, Pascal Heitel, Kirsten E. Christensen, Richard I. Cooper, Robert S. Paton, William C. K. Pomerantz, Philip C. Biggin, Ester M. Hammond, Panagis Filippakopoulos, <span class="NLM_string-name hlFld-ContribAuthor">Stuart J. Conway</span>. </span><span class="cited-content_cbyCitation_article-title">Controlling Intramolecular Interactions in the Design of Selective, High-Affinity Ligands for the CREBBP Bromodomain. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (14)
                                     , 10102-10123. <a href="https://doi.org/10.1021/acs.jmedchem.1c00348" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00348</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00348&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00348%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DControlling%252BIntramolecular%252BInteractions%252Bin%252Bthe%252BDesign%252Bof%252BSelective%25252C%252BHigh-Affinity%252BLigands%252Bfor%252Bthe%252BCREBBP%252BBromodomain%26aulast%3DBrand%26aufirst%3DMichael%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D25022021%26date%3D13072021%26volume%3D64%26issue%3D14%26spage%3D10102%26epage%3D10123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Paul N. Mortenson, Daniel A. Erlanson, Iwan J. P. de Esch, Wolfgang Jahnke, <span class="NLM_string-name hlFld-ContribAuthor">Christopher N. Johnson</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-to-Lead Medicinal Chemistry Publications in 2017. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (8)
                                     , 3857-3872. <a href="https://doi.org/10.1021/acs.jmedchem.8b01472" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01472</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01472%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFragment-to-Lead%252BMedicinal%252BChemistry%252BPublications%252Bin%252B2017%26aulast%3DMortenson%26aufirst%3DPaul%2BN.%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D21092018%26date%3D21112018%26volume%3D62%26issue%3D8%26spage%3D3857%26epage%3D3872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhang-Xu  He</span>, <span class="hlFld-ContribAuthor ">Bing-Fei  Wei</span>, <span class="hlFld-ContribAuthor ">Xin  Zhang</span>, <span class="hlFld-ContribAuthor ">Yun-Peng  Gong</span>, <span class="hlFld-ContribAuthor ">Li-Ying  Ma</span>, <span class="hlFld-ContribAuthor ">Wen  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Current development of CBP/p300 inhibitors in the last decade. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>209 </em>, 112861. <a href="https://doi.org/10.1016/j.ejmech.2020.112861" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112861</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112861&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112861%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DCurrent%252Bdevelopment%252Bof%252BCBP%25252Fp300%252Binhibitors%252Bin%252Bthe%252Blast%252Bdecade%26aulast%3DHe%26aufirst%3DZhang-Xu%26date%3D2021%26volume%3D209%26spage%3D112861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ying  Xiong</span>, <span class="hlFld-ContribAuthor ">Mingming  Zhang</span>, <span class="hlFld-ContribAuthor ">Yingxia  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Advances in the Development of CBP/p300 Bromodomain Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Current Medicinal Chemistry</span><span> <strong>2020,</strong> <em>27 </em>
                                    (33)
                                     , 5583-5598. <a href="https://doi.org/10.2174/0929867326666190731141055" title="DOI URL">https://doi.org/10.2174/0929867326666190731141055</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/0929867326666190731141055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F0929867326666190731141055%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Medicinal%2520Chemistry%26atitle%3DRecent%252BAdvances%252Bin%252Bthe%252BDevelopment%252Bof%252BCBP%25252Fp300%252BBromodomain%252BInhibitors%26aulast%3DXiong%26aufirst%3DYing%26date%3D2020%26volume%3D27%26issue%3D33%26spage%3D5583%26epage%3D5598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hong  Lin</span>, <span class="hlFld-ContribAuthor ">Juan I.  Luengo</span>. </span><span class="cited-content_cbyCitation_article-title">Exploiting binding-site arginines in drug design: Recent examples. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (19)
                                     , 127442. <a href="https://doi.org/10.1016/j.bmcl.2020.127442" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.127442</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.127442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.127442%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DExploiting%252Bbinding-site%252Barginines%252Bin%252Bdrug%252Bdesign%25253A%252BRecent%252Bexamples%26aulast%3DLin%26aufirst%3DHong%26date%3D2020%26volume%3D30%26issue%3D19%26spage%3D127442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Feng-cai  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhong-ya  Sun</span>, <span class="hlFld-ContribAuthor ">Li-ping  Liao</span>, <span class="hlFld-ContribAuthor ">Yu  Zuo</span>, <span class="hlFld-ContribAuthor ">Dan  Zhang</span>, <span class="hlFld-ContribAuthor ">Jun  Wang</span>, <span class="hlFld-ContribAuthor ">Yan-tao  Chen</span>, <span class="hlFld-ContribAuthor ">Sen-hao  Xiao</span>, <span class="hlFld-ContribAuthor ">Hao  Jiang</span>, <span class="hlFld-ContribAuthor ">Tian  Lu</span>, <span class="hlFld-ContribAuthor ">Pan  Xu</span>, <span class="hlFld-ContribAuthor ">Li-yan  Yue</span>, <span class="hlFld-ContribAuthor ">Dao-hai  Du</span>, <span class="hlFld-ContribAuthor ">Hao  Zhang</span>, <span class="hlFld-ContribAuthor ">Chuan-peng  Liu</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel CBP bromodomain inhibitors through TR-FRET-based high-throughput screening. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmacologica Sinica</span><span> <strong>2020,</strong> <em>41 </em>
                                    (2)
                                     , 286-292. <a href="https://doi.org/10.1038/s41401-019-0256-2" title="DOI URL">https://doi.org/10.1038/s41401-019-0256-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41401-019-0256-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41401-019-0256-2%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmacologica%2520Sinica%26atitle%3DDiscovery%252Bof%252Bnovel%252BCBP%252Bbromodomain%252Binhibitors%252Bthrough%252BTR-FRET-based%252Bhigh-throughput%252Bscreening%26aulast%3DZhang%26aufirst%3DFeng-cai%26date%3D2020%26date%3D2019%26volume%3D41%26issue%3D2%26spage%3D286%26epage%3D292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jun  Mo</span>, <span class="hlFld-ContribAuthor ">Tingkai  Chen</span>, <span class="hlFld-ContribAuthor ">Hongyu  Yang</span>, <span class="hlFld-ContribAuthor ">Yan  Guo</span>, <span class="hlFld-ContribAuthor ">Qi  Li</span>, <span class="hlFld-ContribAuthor ">Yuting  Qiao</span>, <span class="hlFld-ContribAuthor ">Hongzhi  Lin</span>, <span class="hlFld-ContribAuthor ">Feng  Feng</span>, <span class="hlFld-ContribAuthor ">Wenyuan  Liu</span>, <span class="hlFld-ContribAuthor ">Yao  Chen</span>, <span class="hlFld-ContribAuthor ">Zongliang  Liu</span>, <span class="hlFld-ContribAuthor ">Haopeng  Sun</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis,
              in vitro
              and
              in vivo
              evaluation of benzylpiperidine-linked 1,3-dimethylbenzimidazolinones as cholinesterase inhibitors against Alzheimer’s disease. </span><span class="cited-content_cbyCitation_journal-name">Journal of Enzyme Inhibition and Medicinal Chemistry</span><span> <strong>2020,</strong> <em>35 </em>
                                    (1)
                                     , 330-343. <a href="https://doi.org/10.1080/14756366.2019.1699553" title="DOI URL">https://doi.org/10.1080/14756366.2019.1699553</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14756366.2019.1699553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14756366.2019.1699553%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Enzyme%2520Inhibition%2520and%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%25252C%252Bin%2525C2%2525A0vitro%252Band%252Bin%2525C2%2525A0vivo%252Bevaluation%252Bof%252Bbenzylpiperidine-linked%252B1%25252C3-dimethylbenzimidazolinones%252Bas%252Bcholinesterase%252Binhibitors%252Bagainst%252BAlzheimer%2525E2%252580%252599s%252Bdisease%26aulast%3DMo%26aufirst%3DJun%26date%3D2020%26date%3D2019%26volume%3D35%26issue%3D1%26spage%3D330%26epage%3D343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael A.  Clegg</span>, <span class="hlFld-ContribAuthor ">Nicholas C. O.  Tomkinson</span>, <span class="hlFld-ContribAuthor ">Rab K.  Prinjha</span>, <span class="hlFld-ContribAuthor ">Philip G.  Humphreys</span>. </span><span class="cited-content_cbyCitation_article-title">Advancements in the Development of non‐BET Bromodomain Chemical Probes. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2019,</strong> <em>14 </em>
                                    (4)
                                     , 362-385. <a href="https://doi.org/10.1002/cmdc.201800738" title="DOI URL">https://doi.org/10.1002/cmdc.201800738</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201800738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201800738%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DAdvancements%252Bin%252Bthe%252BDevelopment%252Bof%252Bnon%2525E2%252580%252590BET%252BBromodomain%252BChemical%252BProbes%26aulast%3DClegg%26aufirst%3DMichael%2BA.%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D4%26spage%3D362%26epage%3D385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Osamu  Onomura</span>, <span class="hlFld-ContribAuthor ">Masami  Kuriyama</span>, <span class="hlFld-ContribAuthor ">Satoshi  Kamogawa</span>, <span class="hlFld-ContribAuthor ">Kosuke  Yamamoto</span>. </span><span class="cited-content_cbyCitation_article-title">β-Trichloroacetylation of Cyclic Amines: Application to Synthesis of Chiral Azabicyclo-N-oxyls. </span><span class="cited-content_cbyCitation_journal-name">HETEROCYCLES</span><span> <strong>2019,</strong> <em>99 </em>
                                    (2)
                                     , 1020. <a href="https://doi.org/10.3987/COM-18-S(F)68" title="DOI URL">https://doi.org/10.3987/COM-18-S(F)68</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3987/COM-18-S(F)68&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3987%2FCOM-18-S%28F%2968%26sid%3Dliteratum%253Aachs%26jtitle%3DHETEROCYCLES%26atitle%3D%2525CE%2525B2-Trichloroacetylation%252Bof%252BCyclic%252BAmines%25253A%252BApplication%252Bto%252BSynthesis%252Bof%252BChiral%252BAzabicyclo-N-oxyls%26aulast%3DOnomura%26aufirst%3DOsamu%26date%3D2019%26volume%3D99%26issue%3D2%26spage%3D1020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Meghan E  Breen</span>, <span class="hlFld-ContribAuthor ">Anna K  Mapp</span>. </span><span class="cited-content_cbyCitation_article-title">Modulating the masters: chemical tools to dissect CBP and p300 function. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2018,</strong> <em>45 </em>, 195-203. <a href="https://doi.org/10.1016/j.cbpa.2018.06.005" title="DOI URL">https://doi.org/10.1016/j.cbpa.2018.06.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2018.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2018.06.005%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DModulating%252Bthe%252Bmasters%25253A%252Bchemical%252Btools%252Bto%252Bdissect%252BCBP%252Band%252Bp300%252Bfunction%26aulast%3DBreen%26aufirst%3DMeghan%2BE%26date%3D2018%26volume%3D45%26spage%3D195%26epage%3D203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0025.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0025.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Discovery and optimization of pyrazolopiperidine CBP inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Cocrystal structure of CBP BrD-bound <b>3</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W0E">5W0E</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Effect of hybridization of the benzodiazepinone core of <b>4</b> with THQ-CF<sub>2</sub>H pyrazole on CBP potency and selectivity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (a) Overlay of cocrystal structures of CBP BrD-bound HTS hit <b>6</b> (purple; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6AXQ">6AXQ</a>) and <b>3</b> (cyan; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W0E">5W0E</a>). (b) Docked structures of HTS hit <b>7</b> (orange and pink) overlaid with cocrystal structure of CBP BrD-bound <b>3</b> (cyan; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W0E">5W0E</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Hybridization of HTS hits and known CBP inhibitors with THQ-LPF binder.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Overlay of cocrystal structures of CBP BrD-bound hybrids <b>16</b> (magenta; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6AY3">6AY3</a>), <b>17</b> (green; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6AY5">6AY5</a>), and <b>3</b> (cyan; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W0E">5W0E</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (a) Overlay of cocrystal structures of CBP BrD-bound thiazolone hybrid <b>17</b> (pink; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6AY5">6AY5</a>) and <b>3</b> (cyan; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W0E">5W0E</a>). (b) Overlay of cocrystal structures of CBP BrD-bound indole hybrid <b>16</b> (magenta; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6AY3">6AY3</a>) and <b>3</b> (cyan; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W0E">5W0E</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Optimization of the thiazolone–THQ hybrid.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Optimization of the indole–THQ hybrid.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Preparation of Compound <b>5</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Preparation of Compound <b>16</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Preparation of Compound <b>17</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch4" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Preparation of Compound <b>18</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch5" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0014.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Preparation of Compound <b>19</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch6" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0015.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Preparation of Compound <b>20</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch7" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0016.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Preparation of Compound <b>21</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch8" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0017.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Preparation of Compound <b>22</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch9" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0018.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 9. Preparation of Compound <b>23</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch10" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Scheme 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0019.jpeg" id="rightTab-gr19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 10. Preparation of Compound <b>24</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch11" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Scheme 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0020.jpeg" id="rightTab-gr20" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 11. Preparation of Compound <b>25</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch12" data-index="21" class="article__inlineFigure"><h2 class="fig-label">Scheme 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0021.jpeg" id="rightTab-gr21" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 12. Preparation of Compound <b>26</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch13" data-index="22" class="article__inlineFigure"><h2 class="fig-label">Scheme 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0022.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0022.jpeg" id="rightTab-gr22" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 13. Preparation of Compound <b>27</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch14" data-index="23" class="article__inlineFigure"><h2 class="fig-label">Scheme 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0023.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0023.jpeg" id="rightTab-gr23" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 14. Preparation of Compound <b>28</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch15" data-index="24" class="article__inlineFigure"><h2 class="fig-label">Scheme 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/medium/jm-2017-01372t_0024.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0024.jpeg" id="rightTab-gr24" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 15. Preparation of Compound <b>29</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01372/20171221/images/large/jm-2017-01372t_0024.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01372&amp;id=sch15"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i99">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04530" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04530" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 62 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Carey, N.</span> <span class="citation_source-book">The Epigenetics Revolution: How Modern Biology is Rewriting Our Understanding of Genetics, Disease, and Inheritance</span>; <span class="NLM_publisher-name">Columbia University Press</span>: <span class="NLM_publisher-loc">New York, NY</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=N.+Carey&title=The+Epigenetics+Revolution%3A+How+Modern+Biology+is+Rewriting+Our+Understanding+of+Genetics%2C+Disease%2C+and+Inheritance"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DCarey%26aufirst%3DN.%26btitle%3DThe%2520Epigenetics%2520Revolution%253A%2520How%2520Modern%2520Biology%2520is%2520Rewriting%2520Our%2520Understanding%2520of%2520Genetics%252C%2520Disease%252C%2520and%2520Inheritance%26pub%3DColumbia%2520University%2520Press%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Carey, N.</span> <span class="citation_source-book">Epigenetics for Drug Discovery</span>; <span class="NLM_publisher-name">Royal Society of Chemistry</span>: <span class="NLM_publisher-loc">United Kingdom</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1039%2F9781782628484" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=N.+Carey&title=Epigenetics+for+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1039%2F9781782628484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1039%252F9781782628484%26sid%3Dliteratum%253Aachs%26aulast%3DCarey%26aufirst%3DN.%26btitle%3DEpigenetics%2520for%2520Drug%2520Discovery%26pub%3DRoyal%2520Society%2520of%2520Chemistry%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Kanherkar, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhatia-Dey, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Csoka, A. B.</span><span> </span><span class="NLM_article-title">Epigenetics Across the Human Lifespan</span> <span class="citation_source-journal">Front. Cell Dev. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">49</span><span class="refDoi"> DOI: 10.3389/fcell.2014.00067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.3389%2Ffcell.2014.00067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=25364756" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A280%3ADC%252BC2M3ltFajtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2014&pages=49&author=R.+R.+Kanherkarauthor=N.+Bhatia-Deyauthor=A.+B.+Csoka&title=Epigenetics+Across+the+Human+Lifespan&doi=10.3389%2Ffcell.2014.00067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetics across the human lifespan</span></div><div class="casAuthors">Kanherkar Riya R; Bhatia-Dey Naina; Csoka Antonei B</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in cell and developmental biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">49</span>
        ISSN:<span class="NLM_cas:issn">2296-634X</span>.
    </div><div class="casAbstract">Epigenetics has the potential to explain various biological phenomena that have heretofore defied complete explication.  This review describes the various types of endogenous human developmental milestones such as birth, puberty, and menopause, as well as the diverse exogenous environmental factors that influence human health, in a chronological epigenetic context.  We describe the entire course of human life from periconception to death and chronologically note all of the potential internal timepoints and external factors that influence the human epigenome.  Ultimately, the environment presents these various factors to the individual that influence the epigenome, and the unique epigenetic and genetic profile of each individual also modulates the specific response to these factors.  During the course of human life, we are exposed to an environment that abounds with a potent and dynamic milieu capable of triggering chemical changes that activate or silence genes.  There is constant interaction between the external and internal environments that is required for normal development and health maintenance as well as for influencing disease load and resistance.  For example, exposure to pharmaceutical and toxic chemicals, diet, stress, exercise, and other environmental factors are capable of eliciting positive or negative epigenetic modifications with lasting effects on development, metabolism and health.  These can impact the body so profoundly as to permanently alter the epigenetic profile of an individual.  We also present a comprehensive new hypothesis of how these diverse environmental factors cause both direct and indirect epigenetic changes and how this knowledge can ultimately be used to improve personalized medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQwSREcQgl8PDUE-swuRqsHfW6udTcc2eYa5KRz-CJpv7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3ltFajtg%253D%253D&md5=a8f37aee83a71cc6a24b016e631dba0b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.3389%2Ffcell.2014.00067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffcell.2014.00067%26sid%3Dliteratum%253Aachs%26aulast%3DKanherkar%26aufirst%3DR.%2BR.%26aulast%3DBhatia-Dey%26aufirst%3DN.%26aulast%3DCsoka%26aufirst%3DA.%2BB.%26atitle%3DEpigenetics%2520Across%2520the%2520Human%2520Lifespan%26jtitle%3DFront.%2520Cell%2520Dev.%2520Biol.%26date%3D2014%26volume%3D2%26spage%3D49%26doi%3D10.3389%2Ffcell.2014.00067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Dawson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Cancer Epigenetics: From Mechanism to Therapy</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">150</span><span class="NLM_x">, </span> <span class="NLM_fpage">12</span><span class="NLM_x">–</span> <span class="NLM_lpage">27</span><span class="refDoi"> DOI: 10.1016/j.cell.2012.06.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1016%2Fj.cell.2012.06.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=22770212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvFGjsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2012&pages=12-27&author=M.+A.+Dawsonauthor=T.+Kouzarides&title=Cancer+Epigenetics%3A+From+Mechanism+to+Therapy&doi=10.1016%2Fj.cell.2012.06.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer Epigenetics: From Mechanism to Therapy</span></div><div class="casAuthors">Dawson, Mark A.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-27</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The epigenetic regulation of DNA-templated processes has been intensely studied over the last 15 years.  DNA methylation, histone modification, nucleosome remodeling, and RNA-mediated targeting regulate many biol. processes that are fundamental to the genesis of cancer.  Here, we present the basic principles behind these epigenetic pathways and highlight the evidence suggesting that their misregulation can culminate in cancer.  This information, along with the promising clin. and preclin. results seen with epigenetic drugs against chromatin regulators, signifies that it is time to embrace the central role of epigenetics in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuyeeliq_34rVg90H21EOLACvtfcHk0lh4uweTqV_ChQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvFGjsLg%253D&md5=bc0176364205a2e66e1971fe789d36ae</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.06.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.06.013%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DCancer%2520Epigenetics%253A%2520From%2520Mechanism%2520to%2520Therapy%26jtitle%3DCell%26date%3D2012%26volume%3D150%26spage%3D12%26epage%3D27%26doi%3D10.1016%2Fj.cell.2012.06.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Campbell, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tummino, P. J.</span><span> </span><span class="NLM_article-title">Cancer Epigenetics Drug Discovery and Development: the Challenge of Hitting the Mark</span> <span class="citation_source-journal">J. Clin. Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x">, </span> <span class="NLM_fpage">64</span><span class="NLM_x">–</span> <span class="NLM_lpage">69</span><span class="refDoi"> DOI: 10.1172/JCI71605</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1172%2FJCI71605" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=24382391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntFejug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2014&pages=64-69&author=R.+M.+Campbellauthor=P.+J.+Tummino&title=Cancer+Epigenetics+Drug+Discovery+and+Development%3A+the+Challenge+of+Hitting+the+Mark&doi=10.1172%2FJCI71605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer epigenetics drug discovery and development: the challenge of hitting the mark</span></div><div class="casAuthors">Campbell, Robert M.; Tummino, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">64-69</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    
            (<span class="NLM_cas:orgname">American Society for Clinical Investigation</span>)
        </div><div class="casAbstract">A review.  Over the past several years, there has been rapidly expanding evidence of epigenetic dysregulation in cancer, in which histone and DNA modification play a crit. role in tumor growth and survival.  These findings have gained the attention of the drug discovery and development community, and offer the potential for a second generation of cancer epigenetic agents for patients following the approved "first generation" of DNA methylation (e.g., Dacogen, Vidaza) and broad-spectrum HDAC inhibitors (e.g., Vorinostat, Romidepsin).  This review provides an anal. of prospects for discovery and development of novel cancer agents that target epigenetic proteins.  The authors will examine key examples of epigenetic dysregulation in tumors as well as challenges to epigenetic drug discovery with emerging biol. and novel classes of drug targets.  The authors will also highlight recent successes in cancer epigenetics drug discovery and consider important factors for clin. success in this burgeoning area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozwt2VzcttEbVg90H21EOLACvtfcHk0lh4uweTqV_ChQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntFejug%253D%253D&md5=69773b6781c3a514621d61956479c6aa</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1172%2FJCI71605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI71605%26sid%3Dliteratum%253Aachs%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26atitle%3DCancer%2520Epigenetics%2520Drug%2520Discovery%2520and%2520Development%253A%2520the%2520Challenge%2520of%2520Hitting%2520the%2520Mark%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2014%26volume%3D124%26spage%3D64%26epage%3D69%26doi%3D10.1172%2FJCI71605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Mau, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yung, R.</span><span> </span><span class="NLM_article-title">Potential of Epigenetic Therapies in Non-Cancerous Conditions</span> <span class="citation_source-journal">Front. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">438</span><span class="refDoi"> DOI: 10.3389/fgene.2014.00438</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.3389%2Ffgene.2014.00438" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=25566322" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A280%3ADC%252BC2MvksFSntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=438&author=T.+Mauauthor=R.+Yung&title=Potential+of+Epigenetic+Therapies+in+Non-Cancerous+Conditions&doi=10.3389%2Ffgene.2014.00438"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Potential of epigenetic therapies in non-cancerous conditions</span></div><div class="casAuthors">Mau Theresa; Yung Raymond</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in genetics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">438</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">There has been an explosion of knowledge in the epigenetics field in the past 20 years.  The first epigenetic therapies have arrived in the clinic for cancer treatments.  In contrast, much of the promise of epigenetic therapies for non-cancerous conditions remains in the laboratories.  The current review will focus on the recent progress that has been made in understanding the pathogenic role of epigenetics in immune and inflammatory conditions, and how the knowledge may provide much needed new therapeutic targets for many autoimmune diseases.  Dietary factors are increasingly recognized as potential modifiers of epigenetic marks that can influence health and diseases across generations.  The current epigenomics revolution will almost certainly complement the explosion of personal genetics medicine to help guide treatment decisions and disease risk stratification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS3SpAGegTY3EQ0lWbKiCflfW6udTcc2ebcxgUUn0gSybntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MvksFSntw%253D%253D&md5=5867589cdc4938ce07745f9db164c6fa</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.3389%2Ffgene.2014.00438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffgene.2014.00438%26sid%3Dliteratum%253Aachs%26aulast%3DMau%26aufirst%3DT.%26aulast%3DYung%26aufirst%3DR.%26atitle%3DPotential%2520of%2520Epigenetic%2520Therapies%2520in%2520Non-Cancerous%2520Conditions%26jtitle%3DFront.%2520Genet.%26date%3D2014%26volume%3D5%26spage%3D438%26doi%3D10.3389%2Ffgene.2014.00438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Kelly, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Carvalho, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, P. A.</span><span> </span><span class="NLM_article-title">Epigenetic Modifications as Therapeutic Targets</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">1069</span><span class="NLM_x">–</span> <span class="NLM_lpage">1078</span><span class="refDoi"> DOI: 10.1038/nbt.1678</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1038%2Fnbt.1678" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2010&pages=1069-1078&author=T.+K.+Kellyauthor=D.+D.+De+Carvalhoauthor=P.+A.+Jones&title=Epigenetic+Modifications+as+Therapeutic+Targets&doi=10.1038%2Fnbt.1678"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1678&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1678%26sid%3Dliteratum%253Aachs%26aulast%3DKelly%26aufirst%3DT.%2BK.%26aulast%3DDe%2BCarvalho%26aufirst%3DD.%2BD.%26aulast%3DJones%26aufirst%3DP.%2BA.%26atitle%3DEpigenetic%2520Modifications%2520as%2520Therapeutic%2520Targets%26jtitle%3DNat.%2520Biotechnol.%26date%3D2010%26volume%3D28%26spage%3D1069%26epage%3D1078%26doi%3D10.1038%2Fnbt.1678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Kwa, F. A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thrimawithana, T. R.</span><span> </span><span class="NLM_article-title">Epigenetic Modifications as Potential Therapeutic Targets in Age-Related Macular Degeneration and Diabetic Retinopathy</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1387</span><span class="NLM_x">–</span> <span class="NLM_lpage">1393</span><span class="refDoi"> DOI: 10.1016/j.drudis.2014.03.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1016%2Fj.drudis.2014.03.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=24717156" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmslakt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=1387-1393&author=F.+A.+A.+Kwaauthor=T.+R.+Thrimawithana&title=Epigenetic+Modifications+as+Potential+Therapeutic+Targets+in+Age-Related+Macular+Degeneration+and+Diabetic+Retinopathy&doi=10.1016%2Fj.drudis.2014.03.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic modifications as potential therapeutic targets in age-related macular degeneration and diabetic retinopathy</span></div><div class="casAuthors">Kwa, Faith A. A.; Thrimawithana, Thilini R.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1387-1393</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Recently, aberrant epigenetic modifications have been identified in the pathogenesis of the posterior eye diseases, age-related macular degeneration (AMD) and diabetic retinopathy (DR).  This has led to the development of alternative therapies that can alter aberrant chromatin-remodelling processes involved in AMD and DR.  These novel therapeutic agents could help to ameliorate the challenges assocd. with current treatments that are limited by variable patient response and disease heterogeneity.  However, research on the use of epigenetic-based therapies in these diseases is relatively young and, therefore, preclin. studies that evaluate their mechanism of action, specificity and adverse effects are warranted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4TpoIQNOL5bVg90H21EOLACvtfcHk0lij_VHaILebgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmslakt78%253D&md5=e0bfab23baa28258e85743fbf3b54c7c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.03.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.03.026%26sid%3Dliteratum%253Aachs%26aulast%3DKwa%26aufirst%3DF.%2BA.%2BA.%26aulast%3DThrimawithana%26aufirst%3DT.%2BR.%26atitle%3DEpigenetic%2520Modifications%2520as%2520Potential%2520Therapeutic%2520Targets%2520in%2520Age-Related%2520Macular%2520Degeneration%2520and%2520Diabetic%2520Retinopathy%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2014%26volume%3D19%26spage%3D1387%26epage%3D1393%26doi%3D10.1016%2Fj.drudis.2014.03.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Epigenetic Protein Families: a New Frontier for Drug Discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">384</span><span class="NLM_x">–</span> <span class="NLM_lpage">400</span><span class="refDoi"> DOI: 10.1038/nrd3674</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1038%2Fnrd3674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=22498752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=384-400&author=C.+H.+Arrowsmithauthor=C.+Bountraauthor=P.+V.+Fishauthor=K.+Leeauthor=M.+Schapira&title=Epigenetic+Protein+Families%3A+a+New+Frontier+for+Drug+Discovery&doi=10.1038%2Fnrd3674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic protein families: a new frontier for drug discovery</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Bountra, Chas; Fish, Paul V.; Lee, Kevin; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">384-400</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetic regulation of gene expression is a dynamic and reversible process that establishes normal cellular phenotypes but also contributes to human diseases.  At the mol. level, epigenetic regulation involves hierarchical covalent modification of DNA and the proteins that package DNA, such as histones.  Here, we review the key protein families that mediate epigenetic signalling through the acetylation and methylation of histones, including histone deacetylases, protein methyltransferases, lysine demethylases, bromodomain-contg. proteins and proteins that bind to methylated histones.  These protein families are emerging as druggable classes of enzymes and druggable classes of protein-protein interaction domains.  In this article, we discuss the known links with disease, basic mol. mechanisms of action and recent progress in the pharmacol. modulation of each class of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcD99eCigNtLVg90H21EOLACvtfcHk0lij_VHaILebgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D&md5=eb82034466a43107aa74a18ccf6d29f5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrd3674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3674%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DEpigenetic%2520Protein%2520Families%253A%2520a%2520New%2520Frontier%2520for%2520Drug%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D384%26epage%3D400%26doi%3D10.1038%2Fnrd3674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Gnyszka, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jastrzebski, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flis, S.</span><span> </span><span class="NLM_article-title">DNA Methyltransferase Inhibitors and Their Emerging Role in Epigenetic Therapy of Cancer</span> <span class="citation_source-journal">Anticancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">2989</span><span class="NLM_x">–</span> <span class="NLM_lpage">2996</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=23898051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVaqsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2013&pages=2989-2996&author=A.+Gnyszkaauthor=Z.+Jastrzebskiauthor=S.+Flis&title=DNA+Methyltransferase+Inhibitors+and+Their+Emerging+Role+in+Epigenetic+Therapy+of+Cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">DNA methyltransferase inhibitors and their emerging role in epigenetic therapy of cancer</span></div><div class="casAuthors">Gnyszka, Agnieszka; Jastrzebski, Zenon; Flis, Sylwia</div><div class="citationInfo"><span class="NLM_cas:title">Anticancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2989-2996</span>CODEN:
                <span class="NLM_cas:coden">ANTRD4</span>;
        ISSN:<span class="NLM_cas:issn">0250-7005</span>.
    
            (<span class="NLM_cas:orgname">International Institute of Anticancer Research</span>)
        </div><div class="casAbstract">A review.  The DNA methyltransferase (DNMT) inhibitors azacytidine and decitabine are the most successful epigenetic drugs to date and are still the most widely used as epigenetic modulators, even though their application for oncol. diseases is restricted by their relative toxicity and poor chem. stability.  Zebularine (1-(β-D-ribofuranosyl)-1,2-dihydropyrimidin-2-one), a more stable and less toxic cytidine analog, is another inhibitor of DNMT with concomitant inhibitory activity towards cytidine deaminase.  Unfortunately, there is no new information related to the possible clin. applications of zebularine.  Although many new inhibitors of DNMT have been identified, none of them can so far replace azacytidine, decitabine and, to a lesser degree, zebularine.  This review summarizes the current data and knowledge about azacytidine, decitabine and zebularine, and their role in present and possible future epigenetic cancer therapy.  We also discuss the mol. modes of action of these agents with consideration of their different toxicities and demethylation profiles, reflecting their complex and partially overlapping biol. effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPiSuC0_kryrVg90H21EOLACvtfcHk0lgxr2pMaSBvoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVaqsLnP&md5=9540e414d3a3291856d8378f3da2292d</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGnyszka%26aufirst%3DA.%26aulast%3DJastrzebski%26aufirst%3DZ.%26aulast%3DFlis%26aufirst%3DS.%26atitle%3DDNA%2520Methyltransferase%2520Inhibitors%2520and%2520Their%2520Emerging%2520Role%2520in%2520Epigenetic%2520Therapy%2520of%2520Cancer%26jtitle%3DAnticancer%2520Res.%26date%3D2013%26volume%3D33%26spage%3D2989%26epage%3D2996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Szyf, M.</span><span> </span><span class="NLM_article-title">The Implications of DNA Methylation for Toxicology: Toward Toxicomethylomics, the Toxicology of DNA Methylation</span> <span class="citation_source-journal">Toxicol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">120</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span><span class="refDoi"> DOI: 10.1093/toxsci/kfr024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1093%2Ftoxsci%2Fkfr024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=21297083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjvFejt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2011&pages=235-255&author=M.+Szyf&title=The+Implications+of+DNA+Methylation+for+Toxicology%3A+Toward+Toxicomethylomics%2C+the+Toxicology+of+DNA+Methylation&doi=10.1093%2Ftoxsci%2Fkfr024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The Implications of DNA Methylation for Toxicology: Toward Toxicomethylomics, the Toxicology of DNA Methylation</span></div><div class="casAuthors">Szyf, Moshe</div><div class="citationInfo"><span class="NLM_cas:title">Toxicological Sciences</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">235-255</span>CODEN:
                <span class="NLM_cas:coden">TOSCF2</span>;
        ISSN:<span class="NLM_cas:issn">1096-6080</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Identifying agents that have a long-term deleterious impact on health but exhibit no immediate toxicity is of prime importance.  It is well established that long-term toxicity of chems. could be caused by their ability to generate changes in the DNA sequence through the process of mutagenesis.  Several assays including the Ames test and its different modifications were developed to assess the mutagenic potential of chems.  These tests have also been employed for assessing the carcinogenic potential of compds.  However, the DNA mol. contains within its chem. structure 2 layers of information: the DNA sequence that bears the ancestral genetic information and the pattern of distribution of covalently bound Me groups on cytosines in DNA.  DNA methylation patterns are generated by an innate program during gestation but are attuned to the environment in utero and throughout life including phys. and social exposures.  DNA function and health could be stably altered by exposure to environmental agents without changing the sequence, just by changing the state of DNA methylation.  The current screening tests do not detect agents that have long-range impact on the phenotype without altering the genotype.  The realization that long-range damage could be caused without changing the DNA sequence has important implications on the way the safety of chems., drugs, and food is assessed and broadens the scope of definition of toxic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVjRfwmFuN6bVg90H21EOLACvtfcHk0lgxr2pMaSBvoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjvFejt7g%253D&md5=e8d16d77f41868174f4f2abb7eebb8c6</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1093%2Ftoxsci%2Fkfr024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Ftoxsci%252Fkfr024%26sid%3Dliteratum%253Aachs%26aulast%3DSzyf%26aufirst%3DM.%26atitle%3DThe%2520Implications%2520of%2520DNA%2520Methylation%2520for%2520Toxicology%253A%2520Toward%2520Toxicomethylomics%252C%2520the%2520Toxicology%2520of%2520DNA%2520Methylation%26jtitle%3DToxicol.%2520Sci.%26date%3D2011%26volume%3D120%26spage%3D235%26epage%3D255%26doi%3D10.1093%2Ftoxsci%2Fkfr024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Cedar, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergman, Y.</span><span> </span><span class="NLM_article-title">Linking DNA Methylation and Histone Modification: Patterns and Paradigms</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">295</span><span class="NLM_x">–</span> <span class="NLM_lpage">304</span><span class="refDoi"> DOI: 10.1038/nrg2540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1038%2Fnrg2540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=19308066" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksF2rsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=295-304&author=H.+Cedarauthor=Y.+Bergman&title=Linking+DNA+Methylation+and+Histone+Modification%3A+Patterns+and+Paradigms&doi=10.1038%2Fnrg2540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Linking DNA methylation and histone modification: patterns and paradigms</span></div><div class="casAuthors">Cedar, Howard; Bergman, Yehudit</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">295-304</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  DNA methylation and histone modification are important for regulating gene expression and chromatin structure.  New evidence suggests there are multiple levels of crosstalk between these marks.  Understanding this crosstalk will be important for elucidating the role of epigenetics in development and disease.  Both DNA methylation and histone modification are involved in establishing patterns of gene repression during development.  Certain forms of histone methylation cause local formation of heterochromatin, which is readily reversible, whereas DNA methylation leads to stable long-term repression.  It has recently become apparent that DNA methylation and histone modification pathways can be dependent on one another, and that this crosstalk can be mediated by biochem. interactions between SET domain histone methyltransferases and DNA methyltransferases.  Relationships between DNA methylation and histone modification have implications for understanding normal development as well as somatic cell reprogramming and tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbo2-3KvKYDrVg90H21EOLACvtfcHk0lgxr2pMaSBvoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksF2rsL0%253D&md5=407a040cbccd1afe6cdfbba5dc554a61</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrg2540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg2540%26sid%3Dliteratum%253Aachs%26aulast%3DCedar%26aufirst%3DH.%26aulast%3DBergman%26aufirst%3DY.%26atitle%3DLinking%2520DNA%2520Methylation%2520and%2520Histone%2520Modification%253A%2520Patterns%2520and%2520Paradigms%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2009%26volume%3D10%26spage%3D295%26epage%3D304%26doi%3D10.1038%2Fnrg2540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Verdone, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caserta, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Mauro, E.</span><span> </span><span class="NLM_article-title">Role of Histone Acetylation in the Control of Gene Expression</span> <span class="citation_source-journal">Biochem. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">344</span><span class="NLM_x">–</span> <span class="NLM_lpage">353</span><span class="refDoi"> DOI: 10.1139/o05-041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1139%2Fo05-041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=15959560" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BD2MXntFClsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2005&pages=344-353&author=L.+Verdoneauthor=M.+Casertaauthor=E.+Di+Mauro&title=Role+of+Histone+Acetylation+in+the+Control+of+Gene+Expression&doi=10.1139%2Fo05-041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Role of histone acetylation in the control of gene expression</span></div><div class="casAuthors">Verdone, Loredana; Caserta, Micaela; Di Mauro, Ernesto</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry and Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">344-353</span>CODEN:
                <span class="NLM_cas:coden">BCBIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0829-8211</span>.
    
            (<span class="NLM_cas:orgname">National Research Council of Canada</span>)
        </div><div class="casAbstract">A review.  Histone proteins play structural and functional roles in all nuclear processes.  They undergo different types of covalent modifications, defined in their ensemble as epigenetic because changes in DNA sequences are not involved.  Histone acetylation emerges as a central switch that allows interconversion between permissive and repressive chromatin domains in terms of transcriptional competence.  The mechanisms underlying the histone acetylation-dependent control of gene expression include a direct effect on the stability of nucleosomal arrays and the creation of docking sites for the binding of regulatory proteins.  Histone acetyltransferases and deacetylases are, resp., the enzymes devoted to the addn. and removal of acetyl groups from lysine residues on the histone N-terminal tails.  The enzymes exert fundamental roles in developmental processes and their deregulation has been linked to the progression of diverse human disorders, including cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra07s9zdBpF7Vg90H21EOLACvtfcHk0ljqdX9rQLT72A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXntFClsbo%253D&md5=4f3d8b79f8cac35e7b0b553028ce96c5</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1139%2Fo05-041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1139%252Fo05-041%26sid%3Dliteratum%253Aachs%26aulast%3DVerdone%26aufirst%3DL.%26aulast%3DCaserta%26aufirst%3DM.%26aulast%3DDi%2BMauro%26aufirst%3DE.%26atitle%3DRole%2520of%2520Histone%2520Acetylation%2520in%2520the%2520Control%2520of%2520Gene%2520Expression%26jtitle%3DBiochem.%2520Cell%2520Biol.%26date%3D2005%26volume%3D83%26spage%3D344%26epage%3D353%26doi%3D10.1139%2Fo05-041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Shahbazian, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grunstein, M.</span><span> </span><span class="NLM_article-title">Functions of Site-Specific Histone Acetylation and Deacetylation</span> <span class="citation_source-journal">Annu. Rev. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">76</span><span class="NLM_x">, </span> <span class="NLM_fpage">75</span><span class="NLM_x">–</span> <span class="NLM_lpage">100</span><span class="refDoi"> DOI: 10.1146/annurev.biochem.76.052705.162114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1146%2Fannurev.biochem.76.052705.162114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=17362198" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVehtb7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2007&pages=75-100&author=M.+D.+Shahbazianauthor=M.+Grunstein&title=Functions+of+Site-Specific+Histone+Acetylation+and+Deacetylation&doi=10.1146%2Fannurev.biochem.76.052705.162114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Functions of site-specific histone acetylation and deacetylation</span></div><div class="casAuthors">Shahbazian, Mona D.; Grunstein, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">75-100</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Histone acetylation regulates many cellular processes, and specific acetylation marks, either singly or in combination, produce distinct outcomes.  For example, the acetylation pattern on newly synthesized histones is important for their assembly into nucleosomes by histone chaperones.  Addnl., the degree of chromatin compaction and folding may be regulated by acetylation of histone H4 at Lys-16.  Histone acetylation also regulates the formation of heterochromatin; deacetylation of H4 Lys-16 is important for the spreading of heterochromatin components, whereas acetylation of this site serves as a barrier to this spreading.  Finally, histone acetylation is crit. for gene transcription, but recent results suggest that deacetylation of certain sites also plays an important role.  There are many histone acetyltransferases (HATs) and deacetylases, with differing preferences for the various histone proteins and for specific sites on individual histones.  Detg. how these enzymes create distinct acetylation patterns and regulate the functional outcome is an important challenge.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2iL3QDzZh-7Vg90H21EOLACvtfcHk0ljqdX9rQLT72A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVehtb7O&md5=51fb46ba44a7f0c614c812baddb4e65e</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.76.052705.162114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.76.052705.162114%26sid%3Dliteratum%253Aachs%26aulast%3DShahbazian%26aufirst%3DM.%2BD.%26aulast%3DGrunstein%26aufirst%3DM.%26atitle%3DFunctions%2520of%2520Site-Specific%2520Histone%2520Acetylation%2520and%2520Deacetylation%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2007%26volume%3D76%26spage%3D75%26epage%3D100%26doi%3D10.1146%2Fannurev.biochem.76.052705.162114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Grunstein, M.</span><span> </span><span class="NLM_article-title">Histone Acetylation in Chromatin Structure and Transcription</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">389</span><span class="NLM_x">, </span> <span class="NLM_fpage">349</span><span class="NLM_x">–</span> <span class="NLM_lpage">352</span><span class="refDoi"> DOI: 10.1038/38664</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1038%2F38664" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=9311776" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADyaK2sXmsVWntLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=389&publication_year=1997&pages=349-352&author=M.+Grunstein&title=Histone+Acetylation+in+Chromatin+Structure+and+Transcription&doi=10.1038%2F38664"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Histone acetylation in chromatin structure and transcription</span></div><div class="casAuthors">Grunstein, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">389</span>
        (<span class="NLM_cas:issue">6649</span>),
    <span class="NLM_cas:pages">349-352</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">A review with 75 refs.  The amino termini of histones extend from the nucleosomal core and are modified by acetyltransferase and deacetylases during the cell cycle.  These acetylation patterns may direct histone assembly and help regulate the unfolding and activity of genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodv2Ds8rzh-rVg90H21EOLACvtfcHk0ljqdX9rQLT72A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmsVWntLk%253D&md5=cdd287ff2d87e9c89944366b71aaf6d9</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2F38664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F38664%26sid%3Dliteratum%253Aachs%26aulast%3DGrunstein%26aufirst%3DM.%26atitle%3DHistone%2520Acetylation%2520in%2520Chromatin%2520Structure%2520and%2520Transcription%26jtitle%3DNature%26date%3D1997%26volume%3D389%26spage%3D349%26epage%3D352%26doi%3D10.1038%2F38664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Translating the Histone Code</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">293</span><span class="NLM_x">, </span> <span class="NLM_fpage">1074</span><span class="NLM_x">–</span> <span class="NLM_lpage">1080</span><span class="refDoi"> DOI: 10.1126/science.1063127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1126%2Fscience.1063127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=11498575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2001&pages=1074-1080&author=T.+Jenuweinauthor=C.+D.+Allis&title=Translating+the+Histone+Code&doi=10.1126%2Fscience.1063127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Translating the histone code</span></div><div class="casAuthors">Jenuwein, Thomas; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">5532</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The review.  Chromatin, the physiol. template of all eukaryotic genetic information, is subject to a diverse array of posttranslational modifications that largely impinge on histone amino termini, thereby regulating access to the underlying DNA.  Distinct histone amino-terminal modifications can generate synergistic or antagonistic interaction affinities for chromatin-assocd. proteins, which in turn dictate dynamic transitions between transcriptionally active or transcriptionally silent chromatin states.  The combinatorial nature of histone amino-terminal modifications thus reveals a "histone code" that considerably extends the information potential of the genetic code.  We propose that this epigenetic marking system represents a fundamental regulatory mechanism that has an impact on most, if not all; chromatin-templated processes, with far-reaching consequences for cell fate decisions and both normal and pathol. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUFyHNepKcUbVg90H21EOLACvtfcHk0liUtKK0b-roVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D&md5=021dcd98a5e5bf2ed7eedef1897b2269</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1126%2Fscience.1063127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1063127%26sid%3Dliteratum%253Aachs%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DTranslating%2520the%2520Histone%2520Code%26jtitle%3DScience%26date%3D2001%26volume%3D293%26spage%3D1074%26epage%3D1080%26doi%3D10.1126%2Fscience.1063127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Strahl, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">The Language of Covalent Histone Modifications</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">403</span><span class="NLM_x">, </span> <span class="NLM_fpage">41</span><span class="NLM_x">–</span> <span class="NLM_lpage">45</span><span class="refDoi"> DOI: 10.1038/47412</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1038%2F47412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10638745" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A280%3ADC%252BD3c7gt1arsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=403&publication_year=2000&pages=41-45&author=B.+D.+Strahlauthor=C.+D.+Allis&title=The+Language+of+Covalent+Histone+Modifications&doi=10.1038%2F47412"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The language of covalent histone modifications</span></div><div class="casAuthors">Strahl B D; Allis C D</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">403</span>
        (<span class="NLM_cas:issue">6765</span>),
    <span class="NLM_cas:pages">41-5</span>
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Histone proteins and the nucleosomes they form with DNA are the fundamental building blocks of eukaryotic chromatin.  A diverse array of post-translational modifications that often occur on tail domains of these proteins has been well documented.  Although the function of these highly conserved modifications has remained elusive, converging biochemical and genetic evidence suggests functions in several chromatin-based processes.  We propose that distinct histone modifications, on one or more tails, act sequentially or in combination to form a 'histone code' that is, read by other proteins to bring about distinct downstream events.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSyzhdlYh9sX7t7lRqEFUJVfW6udTcc2eY37dekj3P4q7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3c7gt1arsQ%253D%253D&md5=2eb7d093151c6122ccd70fa2adcfd94b</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2F47412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F47412%26sid%3Dliteratum%253Aachs%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DThe%2520Language%2520of%2520Covalent%2520Histone%2520Modifications%26jtitle%3DNature%26date%3D2000%26volume%3D403%26spage%3D41%26epage%3D45%26doi%3D10.1038%2F47412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Dawson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kouzarides, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huntly, B. J. P.</span><span> </span><span class="NLM_article-title">Targeting Epigenetic Readers in Cancer</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">367</span><span class="NLM_x">, </span> <span class="NLM_fpage">647</span><span class="NLM_x">–</span> <span class="NLM_lpage">657</span><span class="refDoi"> DOI: 10.1056/NEJMra1112635</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1056%2FNEJMra1112635" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=22894577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Chu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=367&publication_year=2012&pages=647-657&author=M.+A.+Dawsonauthor=T.+Kouzaridesauthor=B.+J.+P.+Huntly&title=Targeting+Epigenetic+Readers+in+Cancer&doi=10.1056%2FNEJMra1112635"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting epigenetic readers in cancer</span></div><div class="casAuthors">Dawson, Mark A.; Kouzarides, Tony; Huntly, Brian J. P.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">367</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">647-657</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review on assocn. between epigenetics and chromatin biol., BET bromodomain inhibitors, and role of BET inhibitors in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnz_YQy6Z2T7Vg90H21EOLACvtfcHk0liHeEIdWvFYxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Chu77F&md5=f5e1e82c8a1105f798b1f7719138721c</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1056%2FNEJMra1112635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra1112635%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%2BP.%26atitle%3DTargeting%2520Epigenetic%2520Readers%2520in%2520Cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2012%26volume%3D367%26spage%3D647%26epage%3D657%26doi%3D10.1056%2FNEJMra1112635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Targeting Bromodomains: Epigenetic Readers of Lysine Acetylation</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">356</span><span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+Bromodomains%3A+Epigenetic+Readers+of+Lysine+Acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0liHeEIdWvFYxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520Bromodomains%253A%2520Epigenetic%2520Readers%2520of%2520Lysine%2520Acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Romero, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crawford, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsui, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Côté, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magnuson, S.</span><span> </span><span class="NLM_article-title">Disrupting Acetyl-Lysine Recognition: Progress in the Development of Bromodomain Inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">1271</span><span class="NLM_x">–</span> <span class="NLM_lpage">1298</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01514</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01514" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1271-1298&author=F.+A.+Romeroauthor=A.+M.+Taylorauthor=T.+D.+Crawfordauthor=V.+Tsuiauthor=A.+C%C3%B4t%C3%A9author=S.+Magnuson&title=Disrupting+Acetyl-Lysine+Recognition%3A+Progress+in+the+Development+of+Bromodomain+Inhibitors&doi=10.1021%2Facs.jmedchem.5b01514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting Acetyl-Lysine Recognition: Progress in the Development of Bromodomain Inhibitors</span></div><div class="casAuthors">Romero, F. Anthony; Taylor, Alexander M.; Crawford, Terry D.; Tsui, Vickie; Cote, Alexandre; Magnuson, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1271-1298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains, small protein modules that recognize acetylated lysine on histones, play a significant role in the epigenome, where they function as "readers" that ultimately det. the functional outcome of the post-translational modification.  Because the initial discovery of selective BET inhibitors have helped define the role of that protein family in oncol. and inflammation, BET bromodomains have continued to garner the most attention of any other bromodomain.  More recently, non-BET bromodomain inhibitors that are potent and selective have been disclosed for ATAD2, CBP, BRD7/9, BRPF, BRPF/TRIM24, CECR2, SMARCA4, and BAZ2A/B.  Such novel inhibitors can be used to probe the physiol. function of these non-BET bromodomains and further understanding of their role in certain disease states.  Here, we provide an update to the progress in identifying selective bromodomain inhibitors and their use as biol. tools, as well as our perspective on the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgbRrKGlHaVLVg90H21EOLACvtfcHk0liHeEIdWvFYxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO&md5=260352f61301aa88aad48c0b63e6152f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01514%26sid%3Dliteratum%253Aachs%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DDisrupting%2520Acetyl-Lysine%2520Recognition%253A%2520Progress%2520in%2520the%2520Development%2520of%2520Bromodomain%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1271%26epage%3D1298%26doi%3D10.1021%2Facs.jmedchem.5b01514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Sanchez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meslamani, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M.-M.</span><span> </span><span class="NLM_article-title">The Bromodomain: From Epigenome Reader to Druggable Target</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gene Regul. Mech.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">1839</span><span class="NLM_x">, </span> <span class="NLM_fpage">676</span><span class="NLM_x">–</span> <span class="NLM_lpage">685</span><span class="refDoi"> DOI: 10.1016/j.bbagrm.2014.03.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1016%2Fj.bbagrm.2014.03.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=24686119" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtFagtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1839&publication_year=2014&pages=676-685&author=R.+Sanchezauthor=J.+Meslamaniauthor=M.-M.+Zhou&title=The+Bromodomain%3A+From+Epigenome+Reader+to+Druggable+Target&doi=10.1016%2Fj.bbagrm.2014.03.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain: From epigenome reader to druggable target</span></div><div class="casAuthors">Sanchez, Roberto; Meslamani, Jamel; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1839</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">676-685</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a fundamental post-translational modification that plays an important role in the control of gene transcription in chromatin in an ordered fashion.  The bromodomain, the conserved structural module present in transcription-assocd. proteins, functions exclusively to recognize acetyl-lysine on histones and non-histone proteins.  The structural analyses of bromodomains' recognition of lysine-acetylated peptides derived from histones and cellular proteins provide detailed insights into the differences and unifying features of biol. ligand binding selectivity by the bromodomains.  Newly developed small-mol. inhibitors targeting bromodomain proteins further highlight the functional importance of bromodomain/acetyl-lysine binding as a key mechanism in orchestrating mol. interactions and regulation in chromatin biol. and gene transcription.  These new studies argue that modulating bromodomain/acetyl-lysine interactions with small-mol. chems. offer new opportunities to control gene expression in a wide array of human diseases including cancer and inflammation.  This article is part of a Special Issue entitled: Mol. mechanisms of histone modification function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO3Au7jfK0ibVg90H21EOLACvtfcHk0lgjIRYisSjB6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtFagtrw%253D&md5=95ec21cf13231d9e63bce9b3ccdd6d6f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2014.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2014.03.011%26sid%3Dliteratum%253Aachs%26aulast%3DSanchez%26aufirst%3DR.%26aulast%3DMeslamani%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DThe%2520Bromodomain%253A%2520From%2520Epigenome%2520Reader%2520to%2520Druggable%2520Target%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2014%26volume%3D1839%26spage%3D676%26epage%3D685%26doi%3D10.1016%2Fj.bbagrm.2014.03.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Gallenkamp, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gelato, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haendler, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinmann, H.</span><span> </span><span class="NLM_article-title">Bromodomains and Their Pharmacological Inhibitors</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">438</span><span class="NLM_x">–</span> <span class="NLM_lpage">464</span><span class="refDoi"> DOI: 10.1002/cmdc.201300434</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1002%2Fcmdc.201300434" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=24497428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsl2gtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=438-464&author=D.+Gallenkampauthor=K.+A.+Gelatoauthor=B.+Haendlerauthor=H.+Weinmann&title=Bromodomains+and+Their+Pharmacological+Inhibitors&doi=10.1002%2Fcmdc.201300434"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomains and Their Pharmacological Inhibitors</span></div><div class="casAuthors">Gallenkamp, Daniel; Gelato, Kathy A.; Haendler, Bernard; Weinmann, Hilmar</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">438-464</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Over 60 bromodomains belonging to proteins with very different functions have been identified in humans.  Several of them interact with acetylated lysine residues, leading to the recruitment and stabilization of protein complexes.  The bromodomain and extra-terminal domain (BET) proteins contain tandem bromodomains which bind to acetylated histones and are thereby implicated in a no. of DNA-centered processes, including the regulation of gene expression.  The recent identification of inhibitors of BET and non-BET bromodomains is one of the few examples in which effective blockade of a protein-protein interaction can be achieved with a small mol.  This has led to major strides in the understanding of the function of bromodomain-contg. proteins and their involvement in diseases such as cancer and inflammation.  Indeed, BET bromodomain inhibitors are now being clin. evaluated for the treatment of hematol. tumors and have also been tested in clin. trials for the relatively rare BRD-NUT midline carcinoma.  This review gives an overview of the newest developments in the field, with a focus on the biol. of selected bromodomain proteins on the one hand, and on reported pharmacol. inhibitors on the other, including recent examples from the patent literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon-8_ymytCPbVg90H21EOLACvtfcHk0lgjIRYisSjB6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsl2gtL4%253D&md5=2ce7c6b5c37001fb8fb901fe04aeb606</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300434%26sid%3Dliteratum%253Aachs%26aulast%3DGallenkamp%26aufirst%3DD.%26aulast%3DGelato%26aufirst%3DK.%2BA.%26aulast%3DHaendler%26aufirst%3DB.%26aulast%3DWeinmann%26aufirst%3DH.%26atitle%3DBromodomains%2520and%2520Their%2520Pharmacological%2520Inhibitors%26jtitle%3DChemMedChem%26date%3D2014%26volume%3D9%26spage%3D438%26epage%3D464%26doi%3D10.1002%2Fcmdc.201300434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Dekker, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haisma, H. J.</span><span> </span><span class="NLM_article-title">Histone Acetyl Transferases as Emerging Drug Targets</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">942</span><span class="NLM_x">–</span> <span class="NLM_lpage">948</span><span class="refDoi"> DOI: 10.1016/j.drudis.2009.06.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1016%2Fj.drudis.2009.06.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=19577000" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFygs7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=942-948&author=F.+J.+Dekkerauthor=H.+J.+Haisma&title=Histone+Acetyl+Transferases+as+Emerging+Drug+Targets&doi=10.1016%2Fj.drudis.2009.06.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Histone acetyl transferases as emerging drug targets</span></div><div class="casAuthors">Dekker, Frank J.; Haisma, Hidde J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">19/20</span>),
    <span class="NLM_cas:pages">942-948</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications, such as acetylation or phosphorylation, play a crucial role in the regulation of gene transcription in eukaryotes.  Different subtypes of histone acetyl transferases (HATs) catalyze the acetylation of histones on specific lysine residues.  A potential role of HATs in the pathol. of cancer, asthma, COPD and viral infection has been described.  This indicates that specific HAT inhibitors are potential tools for pharmacol. research and might find therapeutic applications.  This review focuses on the role of the HATs p300, CBP, PCAF and GCN5 in different diseases and the development of small-mol. inhibitors of these enzymes as potential drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1Ao3F8nEy7bVg90H21EOLACvtfcHk0lgjIRYisSjB6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFygs7fL&md5=bff2f4e6d437e003e5b17cfb07215676</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2009.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2009.06.008%26sid%3Dliteratum%253Aachs%26aulast%3DDekker%26aufirst%3DF.%2BJ.%26aulast%3DHaisma%26aufirst%3DH.%2BJ.%26atitle%3DHistone%2520Acetyl%2520Transferases%2520as%2520Emerging%2520Drug%2520Targets%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2009%26volume%3D14%26spage%3D942%26epage%3D948%26doi%3D10.1016%2Fj.drudis.2009.06.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Bannister, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">The CBP Co-Activator is a Histone Acetyltransferase</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">384</span><span class="NLM_x">, </span> <span class="NLM_fpage">641</span><span class="NLM_x">–</span> <span class="NLM_lpage">643</span><span class="refDoi"> DOI: 10.1038/384641a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1038%2F384641a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=8967953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADyaK2sXlsVen" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=1996&pages=641-643&author=A.+J.+Bannisterauthor=T.+Kouzarides&title=The+CBP+Co-Activator+is+a+Histone+Acetyltransferase&doi=10.1038%2F384641a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The CBP co-activator is a histone acetyltransferase</span></div><div class="casAuthors">Bannister, Andrew J.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">384</span>
        (<span class="NLM_cas:issue">6610</span>),
    <span class="NLM_cas:pages">641-643</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">The CBP protein acts as a transcriptional adaptor for many different transcription factors by directly contacting DNA-bound activators.  One mechanism by which CBP is thought to stimulate transcription is by recruiting the histone acetyltransferase (HAT) P/CAF to the promoter.  Here we show that CBP has intrinsic HAT activity.  The HAT domain of CBP is adjacent to the binding site for the transcriptional activator E1A.  Although E1A displaces P/CAF from CBP, is does not disrupt the CBP-assocd. HAT activity.  Thus E1A carries HAT activity when complexed with CBP.  Targeting CBP-assocd. HAT activity to specific promoters may therefore be a mechanism by which E1A acts as a transcriptional activator.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiUdYJTsdfILVg90H21EOLACvtfcHk0lji2GZxTSs2FQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlsVen&md5=8c6fd1c0c0679a454a6c10c704200e2f</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2F384641a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F384641a0%26sid%3Dliteratum%253Aachs%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DThe%2520CBP%2520Co-Activator%2520is%2520a%2520Histone%2520Acetyltransferase%26jtitle%3DNature%26date%3D1996%26volume%3D384%26spage%3D641%26epage%3D643%26doi%3D10.1038%2F384641a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Dancy, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span> </span><span class="NLM_article-title">Protein Lysine Acetylation by p300/CBP</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">115</span><span class="NLM_x">, </span> <span class="NLM_fpage">2419</span><span class="NLM_x">–</span> <span class="NLM_lpage">2452</span><span class="refDoi"> DOI: 10.1021/cr500452k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr500452k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVCiu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2015&pages=2419-2452&author=B.+M.+Dancyauthor=P.+A.+Cole&title=Protein+Lysine+Acetylation+by+p300%2FCBP&doi=10.1021%2Fcr500452k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Lysine Acetylation by p300/CBP</span></div><div class="casAuthors">Dancy, Beverley M.; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2419-2452</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The human protein lysine acetyltransferase encoded by the paralogous p300 and CBP genes is involved in regulating the expression of genes.  In this capacity, p300/CBP controls several basic cellular processes such as proliferation and homeostasis, and plays a role in a variety of human diseases, particularly solid tumors.  This review summarizes information related to the current understanding of p300/CBP, including the novel technologies that have co-evolved with p300/CBP research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYgTImMiuo7rVg90H21EOLACvtfcHk0lji2GZxTSs2FQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVCiu7Y%253D&md5=40a46cf68990f4a7d11a338c54081628</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fcr500452k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr500452k%26sid%3Dliteratum%253Aachs%26aulast%3DDancy%26aufirst%3DB.%2BM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DProtein%2520Lysine%2520Acetylation%2520by%2520p300%252FCBP%26jtitle%3DChem.%2520Rev.%26date%3D2015%26volume%3D115%26spage%3D2419%26epage%3D2452%26doi%3D10.1021%2Fcr500452k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Delvecchio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaucher, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aguilar-Gurrieri, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortega, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Panne, D.</span><span> </span><span class="NLM_article-title">Structure of the P300 Catalytic Core and Implications for Chromatin Targeting and HAT Regulation</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1040</span><span class="NLM_x">–</span> <span class="NLM_lpage">1046</span><span class="refDoi"> DOI: 10.1038/nsmb.2642</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1038%2Fnsmb.2642" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=23934153" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Cgtr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=1040-1046&author=M.+Delvecchioauthor=J.+Gaucherauthor=C.+Aguilar-Gurrieriauthor=E.+Ortegaauthor=D.+Panne&title=Structure+of+the+P300+Catalytic+Core+and+Implications+for+Chromatin+Targeting+and+HAT+Regulation&doi=10.1038%2Fnsmb.2642"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the p300 catalytic core and implications for chromatin targeting and HAT regulation</span></div><div class="casAuthors">Delvecchio, Manuela; Gaucher, Jonathan; Aguilar-Gurrieri, Carmen; Ortega, Esther; Panne, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1040-1046</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">CBP and p300 are histone acetyltransferases (HATs) that assoc. with and acetylate transcriptional regulators and chromatin.  Mutations in their catalytic 'cores' are linked to genetic disorders, including cancer.  Here we present the 2.8-Å crystal structure of the catalytic core of human p300 contg. its bromodomain, CH2 region and HAT domain.  The structure reveals that the CH2 region contains a discontinuous PHD domain interrupted by a RING domain.  The bromodomain, PHD, RING and HAT domains adopt an assembled configuration with the RING domain positioned over the HAT substrate-binding pocket.  Disease mutations that disrupt RING attachment led to upregulation of HAT activity, thus revealing an inhibitory role for this domain.  The structure provides a starting point for understanding how chromatin-substrate targeting and HAT regulation are coupled and why mutations in the p300 core lead to dysregulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlYehjRwdhJ7Vg90H21EOLACvtfcHk0lji2GZxTSs2FQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Cgtr3J&md5=e9352198d513eac6fd93fc2896828d09</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2642&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2642%26sid%3Dliteratum%253Aachs%26aulast%3DDelvecchio%26aufirst%3DM.%26aulast%3DGaucher%26aufirst%3DJ.%26aulast%3DAguilar-Gurrieri%26aufirst%3DC.%26aulast%3DOrtega%26aufirst%3DE.%26aulast%3DPanne%26aufirst%3DD.%26atitle%3DStructure%2520of%2520the%2520P300%2520Catalytic%2520Core%2520and%2520Implications%2520for%2520Chromatin%2520Targeting%2520and%2520HAT%2520Regulation%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D1040%26epage%3D1046%26doi%3D10.1038%2Fnsmb.2642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Panagopoulos, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fioretos, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isaksson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samuelsson, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Billstrom, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strombeck, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitelman, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johansson, B.</span><span> </span><span class="NLM_article-title">Fusion of the <i>MORF</i> and <i>CBP</i> Genes in Acute Myeloid Leukemia with the t(10;16)(q22;p13)</span> <span class="citation_source-journal">Hum. Mol. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">395</span><span class="NLM_x">–</span> <span class="NLM_lpage">404</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1093%2Fhmg%2F10.4.395" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=11157802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhsVygtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2001&pages=395-404&author=I.+Panagopoulosauthor=T.+Fioretosauthor=M.+Isakssonauthor=U.+Samuelssonauthor=R.+Billstromauthor=B.+Strombeckauthor=F.+Mitelmanauthor=B.+Johansson&title=Fusion+of+the+MORF+and+CBP+Genes+in+Acute+Myeloid+Leukemia+with+the+t%2810%3B16%29%28q22%3Bp13%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Fusion of the MORF and CBP genes in acute myeloid leukemia with the t(10;16)(q22;p13)</span></div><div class="casAuthors">Panagopoulos, Ioannis; Fioretos, Thoas; Isaksson, Margareth; Samuelsson, Ulf; Billstrom, Rolf; Strombeck, Bodil; Mitelman, Felix; Johansson, Bertil</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">395-404</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The CBP gene at 16p13 fuses to MOZ and MLL as a result of the t(8;16)(p11;p13) in acute (myelo)monocytic leukemias (AML M4/M5) and the t(11;16)(q23;p13) in treatment-related AML, resp.  Here the authors show that a novel t(10;16)(q22;p13) in a childhood AML M5a leads to a MORF-CBP chimera.  RT-PCR using MORF forward and CBP reverse primers amplified a MORF-CBP fusion in which nucleotide 3103 of MORF was fused in-frame with nucleotide 284 of CBP.  Nested RT-PCR with CBP forward and MORF reverse primers generated a CBP-MORF transcript in which nucleotide 283 of CBP was fused in-frame with nucleotide 3104 of MORF.  Genomic analyses revealed that the breaks were close to Alu elements in intron 16 of MORF and intron 2 of CBP and that duplications had occurred near the breakpoints.  A database search using MORF cDNA enabled us to construct an exon-intron map of the MORF gene.  The MORF-CBP protein retains the zinc fingers, two nuclear localization signals, the histone acetyltransferase (HAT) domain, a portion of the acidic domain of MORF and the CBP protein downstream of codon 29.  Thus, the part of CBP encoding the RARA-binding domain, the CREB-binding domain, the three Cys/His-rich regions, the bromodomain, the HAT domain and the Glu-rich domains is present.  In the reciprocal CBP-MORF, part of the acidic domain and the C-terminal Ser- and Met-rich regions of MORF are likely to be driven by the CBP promoter.  Since both fusion transcripts were present, their exact role in the leukemogenic process remains to be elucidated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm-J76O9QaN7Vg90H21EOLACvtfcHk0liEp-zGU1FT7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhsVygtrg%253D&md5=9f13abf66476fafa1dac70ddfb473e9b</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2F10.4.395&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252F10.4.395%26sid%3Dliteratum%253Aachs%26aulast%3DPanagopoulos%26aufirst%3DI.%26aulast%3DFioretos%26aufirst%3DT.%26aulast%3DIsaksson%26aufirst%3DM.%26aulast%3DSamuelsson%26aufirst%3DU.%26aulast%3DBillstrom%26aufirst%3DR.%26aulast%3DStrombeck%26aufirst%3DB.%26aulast%3DMitelman%26aufirst%3DF.%26aulast%3DJohansson%26aufirst%3DB.%26atitle%3DFusion%2520of%2520the%2520MORF%2520and%2520CBP%2520Genes%2520in%2520Acute%2520Myeloid%2520Leukemia%2520with%2520the%2520t%252810%253B16%2529%2528q22%253Bp13%2529%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2001%26volume%3D10%26spage%3D395%26epage%3D404" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Dutta, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tiu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakamoto, K. M.</span><span> </span><span class="NLM_article-title">CBP/P300 Acetyltransferase Activity in Hematologic Malignancies</span> <span class="citation_source-journal">Mol. Genet. Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">37</span><span class="NLM_x">–</span> <span class="NLM_lpage">43</span><span class="refDoi"> DOI: 10.1016/j.ymgme.2016.06.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1016%2Fj.ymgme.2016.06.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=27380996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFemsb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2016&pages=37-43&author=R.+Duttaauthor=B.+Tiuauthor=K.+M.+Sakamoto&title=CBP%2FP300+Acetyltransferase+Activity+in+Hematologic+Malignancies&doi=10.1016%2Fj.ymgme.2016.06.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">CBP/p300 acetyltransferase activity in hematologic malignancies</span></div><div class="casAuthors">Dutta, Ritika; Tiu, Bruce; Sakamoto, Kathleen M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Genetics and Metabolism</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">37-43</span>CODEN:
                <span class="NLM_cas:coden">MGMEFF</span>;
        ISSN:<span class="NLM_cas:issn">1096-7192</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">CREB binding protein (CBP) and p300 are crit. regulators of hematopoiesis through both their transcriptional coactivator and acetyltransferase activities.  Loss or mutation of CBP/p300 results in hematol. deficiencies in proliferation and differentiation as well as disruption of hematopoietic stem cell renewal and the microenvironment.  Aberrant lysine acetylation mediated by CBP/p300 has recently been implicated in the genesis of multiple hematol. cancers.  Understanding the effects of disrupting the acetyltransferase activity of CBP/p300 could pave the way for new therapeutic approaches to treat patients with these diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMiCksEjVzo7Vg90H21EOLACvtfcHk0liEp-zGU1FT7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFemsb7J&md5=4c0437f742fb98fbefa871c73dade8f3</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.ymgme.2016.06.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymgme.2016.06.013%26sid%3Dliteratum%253Aachs%26aulast%3DDutta%26aufirst%3DR.%26aulast%3DTiu%26aufirst%3DB.%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26atitle%3DCBP%252FP300%2520Acetyltransferase%2520Activity%2520in%2520Hematologic%2520Malignancies%26jtitle%3DMol.%2520Genet.%2520Metab.%26date%3D2016%26volume%3D119%26spage%3D37%26epage%3D43%26doi%3D10.1016%2Fj.ymgme.2016.06.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Ianculescu, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, D.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siegmund, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stallcup, M. R.</span><span> </span><span class="NLM_article-title">Selective Roles for cAMP Response Element-Binding Protein Binding Protein and P300 Protein as Coregulators for Androgen-Regulated Gene Expression in Advanced Prostate Cancer Cells</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">287</span><span class="NLM_x">, </span> <span class="NLM_fpage">4000</span><span class="NLM_x">–</span> <span class="NLM_lpage">4013</span><span class="refDoi"> DOI: 10.1074/jbc.M111.300194</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1074%2Fjbc.M111.300194" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=22174411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVSjtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=4000-4013&author=I.+Ianculescuauthor=D.-Y.+Wuauthor=K.+D.+Siegmundauthor=M.+R.+Stallcup&title=Selective+Roles+for+cAMP+Response+Element-Binding+Protein+Binding+Protein+and+P300+Protein+as+Coregulators+for+Androgen-Regulated+Gene+Expression+in+Advanced+Prostate+Cancer+Cells&doi=10.1074%2Fjbc.M111.300194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Roles for cAMP Response Element-binding Protein Binding Protein and p300 Protein as Coregulators for Androgen-regulated Gene Expression in Advanced Prostate Cancer Cells</span></div><div class="casAuthors">Ianculescu, Irina; Wu, Dai-Ying; Siegmund, Kimberly D.; Stallcup, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">4000-4013</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The protein acetyltransferases p300 and cAMP response element-binding protein binding protein (CBP) are homologous, ubiquitously expressed proteins that interact with hundreds of proteins involved in transcriptional regulation and are involved globally as transcriptional coregulators.  Although these two proteins acetylate and interact with overlapping sets of proteins, we found that p300 and CBP contribute to androgen-induced regulation of distinct sets of genes in C4-2B prostate cancer cells, a model of advanced prostate cancer.  CBP cannot compensate for the loss of p300 to support androgen-induced expression of many genes, such as TMPRSS2 and PSA.  Global gene expression anal. indicated that 47% of androgen-regulated genes are p300-dependent in these cells, whereas, surprisingly, only 0.3% of them are CBP-dependent.  Chromatin immunopptn. anal. after depletion of cellular p300 indicated that p300 is required for androgen-induced acetylation of histones H3 and H4, methylation of histone H3 at Lys-4, and recruitment of TATA box binding protein (TBP) and RNA polymerase II, but not recruitment of the androgen receptor, on the TMPRSS2 gene in response to androgen.  Thus, p300 is the dominant coregulator of the CBP/p300 pair for androgen-regulated gene expression in C4-2B cells. p300 is required at an early stage of chromatin remodeling and transcription complex assembly after binding of androgen receptor to the gene but before many crit. histone modifications occur.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7uJfPL3B4h7Vg90H21EOLACvtfcHk0liEp-zGU1FT7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVSjtLg%253D&md5=095ab7582da1b7602e2ab898caa5b094</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.300194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.300194%26sid%3Dliteratum%253Aachs%26aulast%3DIanculescu%26aufirst%3DI.%26aulast%3DWu%26aufirst%3DD.-Y.%26aulast%3DSiegmund%26aufirst%3DK.%2BD.%26aulast%3DStallcup%26aufirst%3DM.%2BR.%26atitle%3DSelective%2520Roles%2520for%2520cAMP%2520Response%2520Element-Binding%2520Protein%2520Binding%2520Protein%2520and%2520P300%2520Protein%2520as%2520Coregulators%2520for%2520Androgen-Regulated%2520Gene%2520Expression%2520in%2520Advanced%2520Prostate%2520Cancer%2520Cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D4000%26epage%3D4013%26doi%3D10.1074%2Fjbc.M111.300194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Iyer, N. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Özdag, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldas, C.</span><span> </span><span class="NLM_article-title">P300/CBP and Cancer</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">4225</span><span class="NLM_x">–</span> <span class="NLM_lpage">4231</span><span class="refDoi"> DOI: 10.1038/sj.onc.1207118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1038%2Fsj.onc.1207118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=15156177" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BD2cXkvVChtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=4225-4231&author=N.+G.+Iyerauthor=H.+%C3%96zdagauthor=C.+Caldas&title=P300%2FCBP+and+Cancer&doi=10.1038%2Fsj.onc.1207118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">p300/CBP and cancer</span></div><div class="casAuthors">Iyer, Narayanan Gopalakrishna; Oezdag, Hilal; Caldas, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">4225-4231</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  P300 and cAMP response element-binding protein (CBP) are adenoviral E1A-binding proteins involved in multiple cellular processes, and function as transcriptional co-factors and histone acetyltransferases.  Germline mutation of CBP results in Rubinstein-Taybi syndrome, which is characterized by an increased predisposition to childhood malignancies.  Furthermore, somatic mutations of p300 and CBP occur in a no. of malignancies.  Chromosome translocations target CBP and, less commonly, p300 in acute myeloid leukemia and treatment-related hematol. disorders.  P300 mutations in solid tumors result in truncated p300 protein products or amino-acid substitutions in crit. protein domains, and these are often assocd. with inactivation of the second allele.  A mouse model confirms that p300 and CBP function as suppressors of hematol. tumor formation.  The involvement of these proteins in crit. tumorigenic pathways (including TGF-β, p53 and Rb) provides a mechanistic route as to how their inactivation could result in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKGBLVLWLqkrVg90H21EOLACvtfcHk0lh8Cy-JyE99pA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXkvVChtrc%253D&md5=b68fe6bc258b06515c04a5207410f026</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1207118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1207118%26sid%3Dliteratum%253Aachs%26aulast%3DIyer%26aufirst%3DN.%2BG.%26aulast%3D%25C3%2596zdag%26aufirst%3DH.%26aulast%3DCaldas%26aufirst%3DC.%26atitle%3DP300%252FCBP%2520and%2520Cancer%26jtitle%3DOncogene%26date%3D2004%26volume%3D23%26spage%3D4225%26epage%3D4231%26doi%3D10.1038%2Fsj.onc.1207118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Fujisawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span> </span><span class="NLM_article-title">Functions of Bromodomain-Containing Proteins and Their Roles in Homeostasis and Cancer</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">246</span><span class="NLM_x">–</span> <span class="NLM_lpage">262</span><span class="refDoi"> DOI: 10.1038/nrm.2016.143</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1038%2Fnrm.2016.143" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=28053347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1Kiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=246-262&author=T.+Fujisawaauthor=P.+Filippakopoulos&title=Functions+of+Bromodomain-Containing+Proteins+and+Their+Roles+in+Homeostasis+and+Cancer&doi=10.1038%2Fnrm.2016.143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Functions of bromodomain-containing proteins and their roles in homeostasis and cancer</span></div><div class="casAuthors">Fujisawa, Takao; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">246-262</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Bromodomains (BRDs) are evolutionarily conserved protein-protein interaction modules that are found in a wide range of proteins with diverse catalytic and scaffolding functions and are present in most tissues.  BRDs selectively recognize and bind to acetylated Lys residues - particularly in histones - and thereby have important roles in the regulation of gene expression.  BRD-contg. proteins are frequently dysregulated in cancer, they participate in gene fusions that generate diverse, frequently oncogenic proteins, and many cancer-causing mutations have been mapped to the BRDs themselves.  Importantly, BRDs can be targeted by small-mol. inhibitors, which has stimulated many translational research projects that seek to attenuate the aberrant functions of BRD-contg. proteins in disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-drLvQK1rvLVg90H21EOLACvtfcHk0lh8Cy-JyE99pA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1Kiuw%253D%253D&md5=cbccdee8423bad7a426c9cea6566151f</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnrm.2016.143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm.2016.143%26sid%3Dliteratum%253Aachs%26aulast%3DFujisawa%26aufirst%3DT.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DFunctions%2520of%2520Bromodomain-Containing%2520Proteins%2520and%2520Their%2520Roles%2520in%2520Homeostasis%2520and%2520Cancer%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2017%26volume%3D18%26spage%3D246%26epage%3D262%26doi%3D10.1038%2Fnrm.2016.143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de la Cruz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Segal, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merchant, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kharbanda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haverty, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Modrusan, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ly, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choo, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaufman, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beresini, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romero, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magnuson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gascoigne, K. E.</span><span> </span><span class="NLM_article-title">Therapeutic Targeting of the CBP/P300 Bromodomain for the Treatment of Castration Resistant Prostate Cancer</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">77</span><span class="NLM_x">, </span> <span class="NLM_fpage">5564</span><span class="NLM_x">–</span> <span class="NLM_lpage">5575</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-17-0314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1158%2F0008-5472.CAN-17-0314" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=5564-5575&author=L.+Jinauthor=J.+Garciaauthor=E.+Chanauthor=C.+de+la+Cruzauthor=E.+Segalauthor=M.+Merchantauthor=S.+Kharbandaauthor=P.+M.+Havertyauthor=Z.+Modrusanauthor=J.+Lyauthor=E.+F.+Chooauthor=S.+Kaufmanauthor=M.+H.+Beresiniauthor=F.+A.+Romeroauthor=S.+Magnusonauthor=K.+E.+Gascoigne&title=Therapeutic+Targeting+of+the+CBP%2FP300+Bromodomain+for+the+Treatment+of+Castration+Resistant+Prostate+Cancer&doi=10.1158%2F0008-5472.CAN-17-0314"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-17-0314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-17-0314%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DL.%26aulast%3DGarcia%26aufirst%3DJ.%26aulast%3DChan%26aufirst%3DE.%26aulast%3Dde%2Bla%2BCruz%26aufirst%3DC.%26aulast%3DSegal%26aufirst%3DE.%26aulast%3DMerchant%26aufirst%3DM.%26aulast%3DKharbanda%26aufirst%3DS.%26aulast%3DHaverty%26aufirst%3DP.%2BM.%26aulast%3DModrusan%26aufirst%3DZ.%26aulast%3DLy%26aufirst%3DJ.%26aulast%3DChoo%26aufirst%3DE.%2BF.%26aulast%3DKaufman%26aufirst%3DS.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DMagnuson%26aufirst%3DS.%26aulast%3DGascoigne%26aufirst%3DK.%2BE.%26atitle%3DTherapeutic%2520Targeting%2520of%2520the%2520CBP%252FP300%2520Bromodomain%2520for%2520the%2520Treatment%2520of%2520Castration%2520Resistant%2520Prostate%2520Cancer%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D5564%26epage%3D5575%26doi%3D10.1158%2F0008-5472.CAN-17-0314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Faiola, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lymar, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farina, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, E.</span><span> </span><span class="NLM_article-title">Dual Regulation of C-Myc by P300 via Acetylation-Dependent Control of Myc Protein Turnover and Coactivation of Myc-Induced Transcription</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">10220</span><span class="NLM_x">–</span> <span class="NLM_lpage">10234</span><span class="refDoi"> DOI: 10.1128/MCB.25.23.10220-10234.2005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1128%2FMCB.25.23.10220-10234.2005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=16287840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1Ons7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2005&pages=10220-10234&author=F.+Faiolaauthor=X.+Liuauthor=S.+Loauthor=S.+Panauthor=K.+Zhangauthor=E.+Lymarauthor=A.+Farinaauthor=E.+Martinez&title=Dual+Regulation+of+C-Myc+by+P300+via+Acetylation-Dependent+Control+of+Myc+Protein+Turnover+and+Coactivation+of+Myc-Induced+Transcription&doi=10.1128%2FMCB.25.23.10220-10234.2005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Dual regulation of c-Myc by p300 via acetylation-dependent control of Myc protein turnover and coactivation of Myc-induced transcription</span></div><div class="casAuthors">Faiola, Francesco; Liu, Xiaohui; Lo, Szuying; Pan, Songqin; Zhang, Kangling; Lymar, Elena; Farina, Anthony; Martinez, Ernest</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10220-10234</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The c-Myc oncoprotein (Myc) controls cell fate by regulating gene transcription in assocn. with a DNA-binding partner, Max.  While Max lacks a transcription regulatory domain, the N-terminus of Myc contains a transcription activation domain (TAD) that recruits cofactor complexes contg. the histone acetyltransferases (HATs) GCN5 and Tip60.  Here, we report a novel functional interaction between Myc TAD and the p300 coactivator-acetyltransferase.  We show that p300 assocs. with Myc in mammalian cells and in vitro through direct interactions with Myc TAD residues 1 to 110 and acetylates Myc in a TAD-dependent manner in vivo at several lysine residues located between the TAD and DNA-binding domain.  Moreover, the Myc:Max complex is differentially acetylated by p300 and GCN5 and is not acetylated by Tip60 in vitro, suggesting distinct functions for these acetyltransferases.  Whereas p300 and CBP can stabilize Myc independently of acetylation, p300-mediated acetylation results in increased Myc turnover.  In addn., p300 functions as a coactivator that is recruited by Myc to the promoter of the human telomerase reverse transcriptase gene, and p300/CBP stimulates Myc TAD-dependent transcription in a HAT domain-dependent manner.  Our results suggest dual roles for p300/CBP in Myc regulation: as a Myc coactivator that stabilizes Myc and as an inducer of Myc instability via direct Myc acetylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4or2afHXuK7Vg90H21EOLACvtfcHk0lgaVgoY8Yrx9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1Ons7fL&md5=bbabfb64492444687f048ee59cdc0210</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1128%2FMCB.25.23.10220-10234.2005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.25.23.10220-10234.2005%26sid%3Dliteratum%253Aachs%26aulast%3DFaiola%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLo%26aufirst%3DS.%26aulast%3DPan%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DLymar%26aufirst%3DE.%26aulast%3DFarina%26aufirst%3DA.%26aulast%3DMartinez%26aufirst%3DE.%26atitle%3DDual%2520Regulation%2520of%2520C-Myc%2520by%2520P300%2520via%2520Acetylation-Dependent%2520Control%2520of%2520Myc%2520Protein%2520Turnover%2520and%2520Coactivation%2520of%2520Myc-Induced%2520Transcription%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2005%26volume%3D25%26spage%3D10220%26epage%3D10234%26doi%3D10.1128%2FMCB.25.23.10220-10234.2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Conery, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Centore, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neiss, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keller, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joshi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spillane, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandy, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatton, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pardo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zawadzke, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bommi-Reddy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gascoigne, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bryant, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mertz, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sims, R. J.</span><span> </span><span class="NLM_article-title">Bromodomain Inhibition of the Transcriptional Coactivators CBP/EP300 as a Therapeutic Strategy to Target the IRF4 Network in Multiple Myeloma</span> <span class="citation_source-journal">eLife</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e10483</span><span class="refDoi"> DOI: 10.7554/eLife.19432</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.7554%2FeLife.19432" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=e10483&author=A.+R.+Coneryauthor=R.+C.+Centoreauthor=A.+Neissauthor=P.+J.+Kellerauthor=S.+Joshiauthor=K.+L.+Spillaneauthor=P.+Sandyauthor=C.+Hattonauthor=E.+Pardoauthor=L.+Zawadzkeauthor=A.+Bommi-Reddyauthor=K.+E.+Gascoigneauthor=B.+M.+Bryantauthor=J.+A.+Mertzauthor=R.+J.+Sims&title=Bromodomain+Inhibition+of+the+Transcriptional+Coactivators+CBP%2FEP300+as+a+Therapeutic+Strategy+to+Target+the+IRF4+Network+in+Multiple+Myeloma&doi=10.7554%2FeLife.19432"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.7554%2FeLife.19432&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.19432%26sid%3Dliteratum%253Aachs%26aulast%3DConery%26aufirst%3DA.%2BR.%26aulast%3DCentore%26aufirst%3DR.%2BC.%26aulast%3DNeiss%26aufirst%3DA.%26aulast%3DKeller%26aufirst%3DP.%2BJ.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DSpillane%26aufirst%3DK.%2BL.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DHatton%26aufirst%3DC.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DZawadzke%26aufirst%3DL.%26aulast%3DBommi-Reddy%26aufirst%3DA.%26aulast%3DGascoigne%26aufirst%3DK.%2BE.%26aulast%3DBryant%26aufirst%3DB.%2BM.%26aulast%3DMertz%26aufirst%3DJ.%2BA.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26atitle%3DBromodomain%2520Inhibition%2520of%2520the%2520Transcriptional%2520Coactivators%2520CBP%252FEP300%2520as%2520a%2520Therapeutic%2520Strategy%2520to%2520Target%2520the%2520IRF4%2520Network%2520in%2520Multiple%2520Myeloma%26jtitle%3DeLife%26date%3D2016%26volume%3D5%26spage%3De10483%26doi%3D10.7554%2FeLife.19432" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Predina, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beier, U. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.-C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapoor, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhatti, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akimova, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singhal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brindle, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albelda, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hancock, W. W.</span><span> </span><span class="NLM_article-title">Inhibition of P300 Impairs Foxp3+ T Regulatory Cell Function and Promotes Antitumor Immunity</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1173</span><span class="NLM_x">–</span> <span class="NLM_lpage">1177</span><span class="refDoi"> DOI: 10.1038/nm.3286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1038%2Fnm.3286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=23955711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1yks77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=1173-1177&author=Y.+Liuauthor=L.+Wangauthor=J.+Predinaauthor=R.+Hanauthor=U.+H.+Beierauthor=L.-C.+S.+Wangauthor=V.+Kapoorauthor=T.+R.+Bhattiauthor=T.+Akimovaauthor=S.+Singhalauthor=P.+K.+Brindleauthor=P.+A.+Coleauthor=S.+M.+Albeldaauthor=W.+W.+Hancock&title=Inhibition+of+P300+Impairs+Foxp3%2B+T+Regulatory+Cell+Function+and+Promotes+Antitumor+Immunity&doi=10.1038%2Fnm.3286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of p300 impairs Foxp3+ T regulatory cell function and promotes antitumor immunity</span></div><div class="casAuthors">Liu, Yujie; Wang, Liqing; Predina, Jarrod; Han, Rongxiang; Beier, Ulf H.; Wang, Liang-Chuan S.; Kapoor, Veena; Bhatti, Tricia R.; Akimova, Tatiana; Singhal, Sunil; Brindle, Paul K.; Cole, Philip A.; Albelda, Steven M.; Hancock, Wayne W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1173-1177</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Forkhead box P3 (Foxp3)+ T regulatory (Treg) cells maintain immune homeostasis and limit autoimmunity but can also curtail host immune responses to various types of tumors.  Foxp3+ Treg cells are therefore considered promising targets to enhance antitumor immunity, and approaches for their therapeutic modulation are being developed.  However, although studies showing that exptl. depleting Foxp3+ Treg cells can enhance antitumor responses provide proof of principle, these studies lack clear translational potential and have various shortcomings.  Histone/protein acetyltransferases (HATs) promote chromatin accessibility, gene transcription and the function of multiple transcription factors and nonhistone proteins.  We now report that conditional deletion or pharmacol. inhibition of one HAT, p300 (also known as Ep300 or KAT3B), in Foxp3+ Treg cells increased T cell receptor-induced apoptosis in Treg cells, impaired Treg cell suppressive function and peripheral Treg cell induction, and limited tumor growth in immunocompetent but not in immunodeficient mice.  Our data thereby demonstrate that p300 is important for Foxp3+ Treg cell function and homeostasis in vivo and in vitro, and identify mechanisms by which appropriate small-mol. inhibitors can diminish Treg cell function without overtly impairing T effector cell responses or inducing autoimmunity.  Collectively, these data suggest a new approach for cancer immunotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoG1uh9kwBI_rVg90H21EOLACvtfcHk0lgaVgoY8Yrx9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1yks77K&md5=b381e7523f686ed8ea8d54aa38327c41</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnm.3286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3286%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DPredina%26aufirst%3DJ.%26aulast%3DHan%26aufirst%3DR.%26aulast%3DBeier%26aufirst%3DU.%2BH.%26aulast%3DWang%26aufirst%3DL.-C.%2BS.%26aulast%3DKapoor%26aufirst%3DV.%26aulast%3DBhatti%26aufirst%3DT.%2BR.%26aulast%3DAkimova%26aufirst%3DT.%26aulast%3DSinghal%26aufirst%3DS.%26aulast%3DBrindle%26aufirst%3DP.%2BK.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DAlbelda%26aufirst%3DS.%2BM.%26aulast%3DHancock%26aufirst%3DW.%2BW.%26atitle%3DInhibition%2520of%2520P300%2520Impairs%2520Foxp3%252B%2520T%2520Regulatory%2520Cell%2520Function%2520and%2520Promotes%2520Antitumor%2520Immunity%26jtitle%3DNat.%2520Med.%26date%3D2013%26volume%3D19%26spage%3D1173%26epage%3D1177%26doi%3D10.1038%2Fnm.3286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Ghosh, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, H.-R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conery, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mertz, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salmeron, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dakle, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mele, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Côté, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayaram, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setser, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poy, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatzivassiliou, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeAlmeida-Nagata, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandy, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatton, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romero, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reimer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crawford, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pardo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, V. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsui, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cochran, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zawadzke, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harmange, J.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bryant, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cummings, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magnuson, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grogan, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellon, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albrecht, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sims, R. J.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lora, J. M.</span><span> </span><span class="NLM_article-title">Regulatory T Cell Modulation by CBP/EP300 Bromodomain Inhibition</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">291</span><span class="NLM_x">, </span> <span class="NLM_fpage">13014</span><span class="NLM_x">–</span> <span class="NLM_lpage">13027</span><span class="refDoi"> DOI: 10.1074/jbc.M115.708560</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1074%2Fjbc.M115.708560" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=27056325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVSksL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=13014-13027&author=S.+Ghoshauthor=A.+Taylorauthor=M.+Chinauthor=H.-R.+Huangauthor=A.+R.+Coneryauthor=J.+A.+Mertzauthor=A.+Salmeronauthor=P.+J.+Dakleauthor=D.+Meleauthor=A.+C%C3%B4t%C3%A9author=H.+Jayaramauthor=J.+W.+Setserauthor=F.+Poyauthor=G.+Hatzivassiliouauthor=D.+DeAlmeida-Nagataauthor=P.+Sandyauthor=C.+Hattonauthor=F.+A.+Romeroauthor=E.+Chiangauthor=T.+Reimerauthor=T.+Crawfordauthor=E.+Pardoauthor=V.+G.+Watsonauthor=V.+Tsuiauthor=A.+G.+Cochranauthor=L.+Zawadzkeauthor=J.-C.+Harmangeauthor=J.+E.+Audiaauthor=B.+M.+Bryantauthor=R.+T.+Cummingsauthor=S.+R.+Magnusonauthor=J.+L.+Groganauthor=S.+F.+Bellonauthor=B.+K.+Albrechtauthor=R.+J.+Simsauthor=J.+M.+Lora&title=Regulatory+T+Cell+Modulation+by+CBP%2FEP300+Bromodomain+Inhibition&doi=10.1074%2Fjbc.M115.708560"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Regulatory T Cell Modulation by CBP/EP300 Bromodomain Inhibition</span></div><div class="casAuthors">Ghosh, Srimoyee; Taylor, Alexander; Chin, Melissa; Huang, Hon-Ren; Conery, Andrew R.; Mertz, Jennifer A.; Salmeron, Andres; Dakle, Pranal J.; Mele, Deanna; Cote, Alexandre; Jayaram, Hari; Setser, Jeremy W.; Poy, Florence; Hatzivassiliou, Georgia; DeAlmeida-Nagata, Denise; Sandy, Peter; Hatton, Charlie; Romero, F. Anthony; Chiang, Eugene; Reimer, Thornik; Crawford, Terry; Pardo, Eneida; Watson, Venita G.; Tsui, Vickie; Cochran, Andrea G.; Zawadzke, Laura; Harmange, Jean-Christophe; Audia, James E.; Bryant, Barbara M.; Cummings, Richard T.; Magnuson, Steven R.; Grogan, Jane L.; Bellon, Steve F.; Albrecht, Brian K.; Sims, Robert J. III; Lora, Jose M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">13014-13027</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Covalent modification of histones is a fundamental mechanism of regulated gene expression in eukaryotes, and interpretation of histone modifications is an essential feature of epigenetic control.  Bromodomains are specialized binding modules that interact with acetylated histones, linking chromatin recognition to gene transcription.  Because of their ability to function in a domain-specific fashion, selective disruption of bromodomain:acetylated histone interactions with chem. probes serves as a powerful means for understanding biol. processes regulated by these chromatin adaptors.  Here the authors describe the discovery and characterization of potent and selective small mol. inhibitors for the bromodomains of CREBBP/EP300 that engage their target in cellular assays.  The authors use these tools to demonstrate a crit. role for CREBBP/EP300 bromodomains in regulatory T cell biol.  Because regulatory T cell recruitment to tumors is a major mechanism of immune evasion by cancer cells, the authors' data highlight the importance of CREBBP/EP300 bromodomain inhibition as a novel, small mol.-based approach for cancer immunotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9maWuQVImyLVg90H21EOLACvtfcHk0lg1Z2-OL5aErg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVSksL3F&md5=bc8b4edba81bdbef30b1d8fcfaa19f1e</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.708560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.708560%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DTaylor%26aufirst%3DA.%26aulast%3DChin%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DH.-R.%26aulast%3DConery%26aufirst%3DA.%2BR.%26aulast%3DMertz%26aufirst%3DJ.%2BA.%26aulast%3DSalmeron%26aufirst%3DA.%26aulast%3DDakle%26aufirst%3DP.%2BJ.%26aulast%3DMele%26aufirst%3DD.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DSetser%26aufirst%3DJ.%2BW.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DHatzivassiliou%26aufirst%3DG.%26aulast%3DDeAlmeida-Nagata%26aufirst%3DD.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DHatton%26aufirst%3DC.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DChiang%26aufirst%3DE.%26aulast%3DReimer%26aufirst%3DT.%26aulast%3DCrawford%26aufirst%3DT.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DWatson%26aufirst%3DV.%2BG.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DZawadzke%26aufirst%3DL.%26aulast%3DHarmange%26aufirst%3DJ.-C.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DBryant%26aufirst%3DB.%2BM.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DMagnuson%26aufirst%3DS.%2BR.%26aulast%3DGrogan%26aufirst%3DJ.%2BL.%26aulast%3DBellon%26aufirst%3DS.%2BF.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26atitle%3DRegulatory%2520T%2520Cell%2520Modulation%2520by%2520CBP%252FEP300%2520Bromodomain%2520Inhibition%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D13014%26epage%3D13027%26doi%3D10.1074%2Fjbc.M115.708560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Sachchidanand</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Resnick-Silverman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mutjaba, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, W.-j.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manfredi, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M.-M.</span><span> </span><span class="NLM_article-title">Target Structure-Based Discovery of Small Molecules that Block Human p53 and CREB Binding Protein Association</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">81</span><span class="NLM_x">–</span> <span class="NLM_lpage">90</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2005.10.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1016%2Fj.chembiol.2005.10.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=16426974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsV2qsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=81-90&author=+Sachchidanandauthor=L.+Resnick-Silvermanauthor=S.+Yanauthor=S.+Mutjabaauthor=W.-j.+Liuauthor=L.+Zengauthor=J.+J.+Manfrediauthor=M.-M.+Zhou&title=Target+Structure-Based+Discovery+of+Small+Molecules+that+Block+Human+p53+and+CREB+Binding+Protein+Association&doi=10.1016%2Fj.chembiol.2005.10.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Target Structure-Based Discovery of Small Molecules that Block Human p53 and CREB Binding Protein Association</span></div><div class="casAuthors">Sachchidanand; Resnick-Silverman, Lois; Yan, Sherry; Mutjaba, Shiraz; Liu, Wen-jun; Zeng, Lei; Manfredi, James J.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">81-90</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Lysine acetylation of human tumor suppressor p53 in response to cellular stress signals is required for its function as a transcription factor that regulates cell cycle arrest, senescence, or apoptosis.  Here, we report small mols. that block lysine 382-acetylated p53 assocn. with the bromodomain of the coactivator CBP, an interaction essential for p53-induced transcription of the cell cycle inhibitor p21 in response to DNA damage.  These chems. were discovered in target structure-guided NMR spectroscopy screening of a focused chem. library constructed based on the structural knowledge of CBP bromodomain/p53-AcK382 binding.  Structural characterization shows that these chems. inhibit CBP/p53 assocn. by binding to the acetyl-lysine binding site of the bromodomain.  Cell-based functional assays demonstrate that the lead chems. can modulate p53 stability and function in response to doxorubicin-induced DNA damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrznkGtHkYXCbVg90H21EOLACvtfcHk0lg1Z2-OL5aErg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsV2qsQ%253D%253D&md5=a88211df5b27a2d2d37cb09a3bc50821</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2005.10.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2005.10.014%26sid%3Dliteratum%253Aachs%26aulast%3DSachchidanand%26aulast%3DResnick-Silverman%26aufirst%3DL.%26aulast%3DYan%26aufirst%3DS.%26aulast%3DMutjaba%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DW.-j.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DManfredi%26aufirst%3DJ.%2BJ.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DTarget%2520Structure-Based%2520Discovery%2520of%2520Small%2520Molecules%2520that%2520Block%2520Human%2520p53%2520and%2520CREB%2520Binding%2520Protein%2520Association%26jtitle%3DChem.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D81%26epage%3D90%26doi%3D10.1016%2Fj.chembiol.2005.10.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Borah, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mujtaba, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karakikes, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moshkina, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morohashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerona-Navorro, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajjar, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M.-M.</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">531</span><span class="NLM_x">–</span> <span class="NLM_lpage">541</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2010.12.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1016%2Fj.chembiol.2010.12.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=21513889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltFartb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Borah%2C+J.+C.%3B+Mujtaba%2C+S.%3B+Karakikes%2C+I.%3B+Zeng%2C+L.%3B+Muller%2C+M.%3B+Patel%2C+J.%3B+Moshkina%2C+N.%3B+Morohashi%2C+K.%3B+Zhang%2C+W.%3B+Gerona-Navorro%2C+G.%3B+Hajjar%2C+R.+J.%3B+Zhou%2C+M.-M.Chem.+Biol.+2011%2C+18%2C+531%E2%80%93541+10.1016%2Fj.chembiol.2010.12.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">A Small Molecule Binding to the Coactivator CREB-Binding Protein Blocks Apoptosis in Cardiomyocytes</span></div><div class="casAuthors">Borah, Jagat C.; Mujtaba, Shiraz; Karakikes, Ioannis; Zeng, Lei; Muller, Michaela; Patel, Jigneshkumar; Moshkina, Natasha; Morohashi, Keita; Zhang, Weijia; Gerona-Navarro, Guillermo; Hajjar, Roger J.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">531-541</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: As a master transcription factor in cellular responses to external stress, tumor suppressor p53 is tightly regulated.  Excessive p53 activity during myocardial ischemia causes irreversible cellular injury and cardiomyocyte death.  P53 activation is dependent on lysine acetylation by the lysine acetyltransferase and transcriptional coactivator CREB-binding protein (CBP) and on acetylation-directed CBP recruitment for p53 target gene expression.  Here, we report a small mol. ischemin, developed with a structure-guided approach to inhibit the acetyl-lysine binding activity of the bromodomain of CBP.  We show that ischemin alters post-translational modifications on p53 and histones, inhibits p53 interaction with CBP and transcriptional activity in cells, and prevents apoptosis in ischemic cardiomyocytes.  Our study suggests small mol. modulation of acetylation-mediated interactions in gene transcription as a new approach to therapeutic interventions of human disorders such as myocardial ischemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHKOaQAGSYTbVg90H21EOLACvtfcHk0lg1Z2-OL5aErg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltFartb4%253D&md5=0c1d7500e1fff40bc335d1b10ac3c27a</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.12.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.12.021%26sid%3Dliteratum%253Aachs%26aulast%3DBorah%26aufirst%3DJ.%2BC.%26aulast%3DMujtaba%26aufirst%3DS.%26aulast%3DKarakikes%26aufirst%3DI.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DMuller%26aufirst%3DM.%26aulast%3DPatel%26aufirst%3DJ.%26aulast%3DMoshkina%26aufirst%3DN.%26aulast%3DMorohashi%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DGerona-Navorro%26aufirst%3DG.%26aulast%3DHajjar%26aufirst%3DR.%2BJ.%26aulast%3DZhou%26aufirst%3DM.-M.%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D531%26epage%3D541%26doi%3D10.1016%2Fj.chembiol.2010.12.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Ito, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umehara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishino, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terada, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirouzu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Padmanabhan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoyama, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ito, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, M.</span><span> </span><span class="NLM_article-title">Real-Time Imaging of Histone Deacetylase and Bromodomain Inhibitors</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">495</span><span class="NLM_x">–</span> <span class="NLM_lpage">507</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2011.02.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1016%2Fj.chembiol.2011.02.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=21513886" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltFartL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=495-507&author=T.+Itoauthor=T.+Umeharaauthor=K.+Sasakiauthor=Y.+Nakamuraauthor=N.+Nishinoauthor=T.+Teradaauthor=M.+Shirouzuauthor=B.+Padmanabhanauthor=S.+Yokoyamaauthor=A.+Itoauthor=M.+Yoshida&title=Real-Time+Imaging+of+Histone+Deacetylase+and+Bromodomain+Inhibitors&doi=10.1016%2Fj.chembiol.2011.02.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Real-Time Imaging of Histone H4K12-Specific Acetylation Determines the Modes of Action of Histone Deacetylase and Bromodomain Inhibitors</span></div><div class="casAuthors">Ito, Tamaki; Umehara, Takashi; Sasaki, Kazuki; Nakamura, Yoshihiro; Nishino, Norikazu; Terada, Takaho; Shirouzu, Mikako; Padmanabhan, Balasundaram; Yokoyama, Shigeyuki; Ito, Akihiro; Yoshida, Minoru</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">495-507</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone acetylation constitutes an epigenetic mark for transcriptional regulation.  Here we developed a fluorescent probe to visualize acetylation of histone H4 Lys12 (H4K12) in living cells using fluorescence resonance energy transfer (FRET) and the binding of the BRD2 bromodomain to acetylated H4K12.  Using this probe designated as Histac-K12, we demonstrated that histone H4K12 acetylation is retained in mitosis and that some histone deacetylase (HDAC) inhibitors continue to inhibit cellular HDAC activity even after their removal from the culture.  In addn., a small mol. that interferes with ability of the bromodomain to bind to acetylated H4K12 could be assessed using Histac-K12 in cells.  Thus, Histac-K12 will serve as a powerful tool not only to understand the dynamics of H4K12-specific acetylation but also to characterize small mols. that modulate the acetylation or interaction status of histones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBoA9RjMb5tLVg90H21EOLACvtfcHk0lgD77v8NNIWpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltFartL0%253D&md5=f36a5042c2e9342026884f88f9704954</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.02.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.02.009%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DT.%26aulast%3DUmehara%26aufirst%3DT.%26aulast%3DSasaki%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DNishino%26aufirst%3DN.%26aulast%3DTerada%26aufirst%3DT.%26aulast%3DShirouzu%26aufirst%3DM.%26aulast%3DPadmanabhan%26aufirst%3DB.%26aulast%3DYokoyama%26aufirst%3DS.%26aulast%3DIto%26aufirst%3DA.%26aulast%3DYoshida%26aufirst%3DM.%26atitle%3DReal-Time%2520Imaging%2520of%2520Histone%2520Deacetylase%2520and%2520Bromodomain%2520Inhibitors%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D495%26epage%3D507%26doi%3D10.1016%2Fj.chembiol.2011.02.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Hewings, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uttarkar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vuppusetty, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conway, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span> </span><span class="NLM_article-title">3,5-Dimethylisoxazoles Act as Acetyl-Lysine-Mimetic Bromodomain Ligands</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">6761</span><span class="NLM_x">–</span> <span class="NLM_lpage">6770</span><span class="refDoi"> DOI: 10.1021/jm200640v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200640v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaks7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6761-6770&author=D.+S.+Hewingsauthor=M.+Wangauthor=M.+Philpottauthor=O.+Fedorovauthor=S.+Uttarkarauthor=P.+Filippakopoulosauthor=S.+Picaudauthor=C.+Vuppusettyauthor=B.+Marsdenauthor=S.+Knappauthor=S.+J.+Conwayauthor=T.+D.+Heightman&title=3%2C5-Dimethylisoxazoles+Act+as+Acetyl-Lysine-Mimetic+Bromodomain+Ligands&doi=10.1021%2Fjm200640v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">3,5-Dimethylisoxazoles Act As Acetyl-lysine-mimetic Bromodomain Ligands</span></div><div class="casAuthors">Hewings, David S.; Wang, Minghua; Philpott, Martin; Fedorov, Oleg; Uttarkar, Sagar; Filippakopoulos, Panagis; Picaud, Sarah; Vuppusetty, Chaitanya; Marsden, Brian; Knapp, Stefan; Conway, Stuart J.; Heightman, Tom D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6761-6770</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone-lysine acetylation is a vital chromatin post-translational modification involved in the epigenetic regulation of gene transcription.  Bromodomains bind acetylated lysines, acting as readers of the histone-acetylation code.  Competitive inhibitors of this interaction have antiproliferative and anti-inflammatory properties.  With 57 distinct bromodomains known, the discovery of subtype-selective inhibitors of the histone-bromodomain interaction is of great importance.  We have identified the 3,5-dimethylisoxazole moiety as a novel acetyl-lysine bioisostere, which displaces acetylated histone-mimicking peptides from bromodomains.  Using X-ray crystallog. anal., we have detd. the interactions responsible for the activity and selectivity of 4-substituted 3,5-dimethylisoxazoles against a selection of phylogenetically diverse bromodomains.  By exploiting these interactions, we have developed compd. 4d, which has IC50 values of <5 μM for the bromodomain-contg. proteins BRD2(1) and BRD4(1).  These compds. are promising leads for the further development of selective probes for the bromodomain and extra C-terminal domain (BET) family and CREBBP bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEzK4xMvJhXrVg90H21EOLACvtfcHk0lgD77v8NNIWpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaks7rO&md5=7d6f9eff2735c8a7189dec4fb5c5d13d</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm200640v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200640v%26sid%3Dliteratum%253Aachs%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DUttarkar%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DVuppusetty%26aufirst%3DC.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26atitle%3D3%252C5-Dimethylisoxazoles%2520Act%2520as%2520Acetyl-Lysine-Mimetic%2520Bromodomain%2520Ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6761%26epage%3D6770%26doi%3D10.1021%2Fjm200640v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span>I-CBP112: A CREBBP/EP-300-Selective Chemical Probe; SGC, 2015. <a href="http://www.thesgc.org/chemical-probes/I-CBP112" class="extLink">http://www.thesgc.org/chemical-probes/I-CBP112</a> (accessed December 11,<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=I-CBP112%3A+A+CREBBP%2FEP-300-Selective+Chemical+Probe%3B+SGC%2C+2015.+http%3A%2F%2Fwww.thesgc.org%2Fchemical-probes%2FI-CBP112+%28accessed+December+11%2C+2015%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Hewings, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olcina, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freeman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gill, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ritchie, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheppard, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russell, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammond, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conway, S. J.</span><span> </span><span class="NLM_article-title">Optimization of 3,5-Dimethylisoxazole Derivatives as Potent Bromodomain Ligands</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">3217</span><span class="NLM_x">–</span> <span class="NLM_lpage">3227</span><span class="refDoi"> DOI: 10.1021/jm301588r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301588r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktlekuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=3217-3227&author=D.+S.+Hewingsauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=S.+Martinauthor=S.+Picaudauthor=A.+Tumberauthor=C.+Wellsauthor=M.+M.+Olcinaauthor=K.+Freemanauthor=A.+Gillauthor=A.+J.+Ritchieauthor=D.+W.+Sheppardauthor=A.+J.+Russellauthor=E.+M.+Hammondauthor=S.+Knappauthor=P.+E.+Brennanauthor=S.+J.+Conway&title=Optimization+of+3%2C5-Dimethylisoxazole+Derivatives+as+Potent+Bromodomain+Ligands&doi=10.1021%2Fjm301588r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of 3,5-Dimethylisoxazole Derivatives as Potent Bromodomain Ligands</span></div><div class="casAuthors">Hewings, David S.; Fedorov, Oleg; Filippakopoulos, Panagis; Martin, Sarah; Picaud, Sarah; Tumber, Anthony; Wells, Christopher; Olcina, Monica M.; Freeman, Katherine; Gill, Andrew; Ritchie, Alison J.; Sheppard, David W.; Russell, Angela J.; Hammond, Ester M.; Knapp, Stefan; Brennan, Paul E.; Conway, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3217-3227</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">(Isoxazolylphenyl)arylmethanols such as I (R = H; R1 = HO; R2 = H; R3 = Ph), I (R = HO; R1 = H; R2 = H; R3 = Ph), and I (R = H, HO; R1 = HO, H; R2 = Et; R3 = Ph, 3-FC6H4, 4-FC6H4, 3-ClC6H4, 4-ClC6H4) were prepd. as inhibitors of the bromodomain and extra terminal domain (BET) bromodomain family member BRD4(1) for potential use as antitumor agents.  The structures of I (R = H; R1 = HO; R2 = H; R3 = Ph) and I (R = HO; R1 = H; R2 = H; R3 = Ph) bound to human BRD4(1) were detd. by X-ray crystallog. to det. features useful for selective binding to BRD4(1).  I (R = H, HO; R1 = HO, H; R2 = H, MeCO; R3 = Ph) and I (R = H, HO; R1 = HO, H; R2 = Et; R3 = 3-ClC6H4) inhibited a human acute myeloid leukemia cell line in vitro with 3-4 fold reduced inhibition and reduced toxicity in other human cells compared to a known BRD4(1) inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVEQ5OOJGC8bVg90H21EOLACvtfcHk0lip8rqQCHsP-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktlekuro%253D&md5=bb4890291db8c3e8d404a75dd9dcbb87</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm301588r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301588r%26sid%3Dliteratum%253Aachs%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DOlcina%26aufirst%3DM.%2BM.%26aulast%3DFreeman%26aufirst%3DK.%26aulast%3DGill%26aufirst%3DA.%26aulast%3DRitchie%26aufirst%3DA.%2BJ.%26aulast%3DSheppard%26aufirst%3DD.%2BW.%26aulast%3DRussell%26aufirst%3DA.%2BJ.%26aulast%3DHammond%26aufirst%3DE.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DOptimization%2520of%25203%252C5-Dimethylisoxazole%2520Derivatives%2520as%2520Potent%2520Bromodomain%2520Ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D3217%26epage%3D3227%26doi%3D10.1021%2Fjm301588r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Hay, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singleton, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tallant, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monteiro, O. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conway, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yapp, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span> </span><span class="NLM_article-title">Discovery and Optimization of Small-Molecule Ligands for the CBP/P300 Bromodomains</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">136</span><span class="NLM_x">, </span> <span class="NLM_fpage">9308</span><span class="NLM_x">–</span> <span class="NLM_lpage">9319</span><span class="refDoi"> DOI: 10.1021/ja412434f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja412434f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVShsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=9308-9319&author=D.+A.+Hayauthor=O.+Fedorovauthor=S.+Martinauthor=D.+C.+Singletonauthor=C.+Tallantauthor=C.+Wellsauthor=S.+Picaudauthor=M.+Philpottauthor=O.+P.+Monteiroauthor=S.+J.+Conwayauthor=A.+Tumberauthor=C.+Yappauthor=P.+Filippakopoulosauthor=M.+E.+Bunnageauthor=S.+Knappauthor=C.+J.+Schofieldauthor=P.+E.+Brennan&title=Discovery+and+Optimization+of+Small-Molecule+Ligands+for+the+CBP%2FP300+Bromodomains&doi=10.1021%2Fja412434f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of Small-Molecule Ligands for the CBP/p300 Bromodomains</span></div><div class="casAuthors">Hay, Duncan A.; Fedorov, Oleg; Martin, Sarah; Singleton, Dean C.; Tallant, Cynthia; Wells, Christopher; Picaud, Sarah; Philpott, Martin; Monteiro, Octovia P.; Rogers, Catherine M.; Conway, Stuart J.; Rooney, Timothy P. C.; Tumber, Anthony; Yapp, Clarence; Filippakopoulos, Panagis; Bunnage, Mark E.; Muller, Susanne; Knapp, Stefan; Schofield, Christopher J.; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">9308-9319</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small-mol. inhibitors that target bromodomains outside of the bromodomain and extra-terminal (BET) sub-family are lacking.  Here, the authors describe highly potent and selective ligands for the bromodomain module of the human lysine acetyl transferase CBP/p300, developed from a series of 5-isoxazolyl-benzimidazoles.  The starting point was a fragment hit, which was optimized into a more potent and selective lead using parallel synthesis employing Suzuki couplings, benzimidazole-forming reactions, and reductive aminations.  The selectivity of the lead compd. against other bromodomain family members was investigated using a thermal stability assay, which revealed some inhibition of the structurally related BET family members.  To address the BET selectivity issue, x-ray crystal structures of the lead compd. bound to the CREB binding protein (CBP) and the first bromodomain of BRD4 (BRD4(1)) were used to guide the design of more selective compds.  The crystal structures obtained revealed two distinct binding modes.  By varying the aryl substitution pattern and developing conformationally constrained analogs, selectivity for CBP over BRD4(1) was increased.  The optimized compd. is highly potent (Kd = 21 nM) and selective, displaying 40-fold selectivity over BRD4(1).  Cellular activity was demonstrated using fluorescence recovery after photo-bleaching (FRAP) and a p53 reporter assay.  The optimized compds. are cell-active and have nanomolar affinity for CBP/p300; therefore, they should be useful in studies investigating the biol. roles of CBP and p300 and to validate the CBP and p300 bromodomains as therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFzgs1DD4QZLVg90H21EOLACvtfcHk0lip8rqQCHsP-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVShsLvP&md5=e5da3e96a4d234453a481c7ee5e788f2</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fja412434f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja412434f%26sid%3Dliteratum%253Aachs%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSingleton%26aufirst%3DD.%2BC.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DDiscovery%2520and%2520Optimization%2520of%2520Small-Molecule%2520Ligands%2520for%2520the%2520CBP%252FP300%2520Bromodomains%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D9308%26epage%3D9319%26doi%3D10.1021%2Fja412434f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Rooney, T. P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cortopassi, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hay, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pryde, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paton, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conway, S. J.</span><span> </span><span class="NLM_article-title">A Series of Potent CREBBP Bromodomain Ligands Reveals an Induced-Fit Pocket Stabilized by a Cation-π Interaction</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">6126</span><span class="NLM_x">–</span> <span class="NLM_lpage">6130</span><span class="refDoi"> DOI: 10.1002/anie.201402750</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1002%2Fanie.201402750" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnvFWgsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=6126-6130&author=T.+P.+C.+Rooneyauthor=P.+Filippakopoulosauthor=O.+Fedorovauthor=S.+Picaudauthor=W.+A.+Cortopassiauthor=D.+A.+Hayauthor=S.+Martinauthor=A.+Tumberauthor=C.+M.+Rogersauthor=M.+Philpottauthor=M.+Wangauthor=A.+L.+Thompsonauthor=T.+D.+Heightmanauthor=D.+C.+Prydeauthor=A.+Cookauthor=R.+S.+Patonauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=P.+E.+Brennanauthor=S.+J.+Conway&title=A+Series+of+Potent+CREBBP+Bromodomain+Ligands+Reveals+an+Induced-Fit+Pocket+Stabilized+by+a+Cation-%CF%80+Interaction&doi=10.1002%2Fanie.201402750"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">A Series of Potent CREBBP Bromodomain Ligands Reveals an Induced-Fit Pocket Stabilized by a Cation-π Interaction</span></div><div class="casAuthors">Rooney, Timothy P. C.; Filippakopoulos, Panagis; Fedorov, Oleg; Picaud, Sarah; Cortopassi, Wilian A.; Hay, Duncan A.; Martin, Sarah; Tumber, Anthony; Rogers, Catherine M.; Philpott, Martin; Wang, Minghua; Thompson, Amber L.; Heightman, Tom D.; Pryde, David C.; Cook, Andrew; Paton, Robert S.; Mueller, Susanne; Knapp, Stefan; Brennan, Paul E.; Conway, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6126-6130</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Benzoxazinone and dihydroquinoxalinone fragments were employed as novel acetyl lysine mimics in the development of CREBBP bromodomain ligands.  While the benzoxazinone series showed low affinity for the CREBBP bromodomain, expansion of the dihydroquinoxalinone series resulted in the first potent inhibitors, e.g., I, of a bromodomain outside the BET family.  Structural and computational studies revealed that an internal hydrogen bond stabilized the protein-bound conformation of the dihydroquinoxalinone series.  The side chain of this series binds in an induced-fit pocket forming a cation-π interaction with R1173 of CREBBP.  Compd. I, the most potent compd., inhibited binding of CREBBP to chromatin in U2OS cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHFgTVUKe7lrVg90H21EOLACvtfcHk0lip8rqQCHsP-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnvFWgsb0%253D&md5=42b12e1fe855d2af02c49f8e1031d39e</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2Fanie.201402750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201402750%26sid%3Dliteratum%253Aachs%26aulast%3DRooney%26aufirst%3DT.%2BP.%2BC.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DCortopassi%26aufirst%3DW.%2BA.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DThompson%26aufirst%3DA.%2BL.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DPryde%26aufirst%3DD.%2BC.%26aulast%3DCook%26aufirst%3DA.%26aulast%3DPaton%26aufirst%3DR.%2BS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DA%2520Series%2520of%2520Potent%2520CREBBP%2520Bromodomain%2520Ligands%2520Reveals%2520an%2520Induced-Fit%2520Pocket%2520Stabilized%2520by%2520a%2520Cation-%25CF%2580%2520Interaction%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D6126%26epage%3D6130%26doi%3D10.1002%2Fanie.201402750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thanasopoulou, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leonards, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meier, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olzscha, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monteiro, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yapp, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Che, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bannister, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parr, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lugo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vecellio, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Mahony, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velichko, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hay, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daniels, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urh, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kouzarides, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwaller, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Generation of a Selective Small Molecule Inhibitor of the CBP/P300 Bromodomain for Leukemia Therapy</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">75</span><span class="NLM_x">, </span> <span class="NLM_fpage">5106</span><span class="NLM_x">–</span> <span class="NLM_lpage">5119</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-15-0236</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1158%2F0008-5472.CAN-15-0236" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=26552700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFensrzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2015&pages=5106-5119&author=S.+Picaudauthor=O.+Fedorovauthor=A.+Thanasopoulouauthor=K.+Leonardsauthor=K.+Jonesauthor=J.+Meierauthor=H.+Olzschaauthor=O.+Monteiroauthor=S.+Martinauthor=M.+Philpottauthor=A.+Tumberauthor=P.+Filippakopoulosauthor=C.+Yappauthor=C.+Wellsauthor=K.+H.+Cheauthor=A.+Bannisterauthor=S.+Robsonauthor=U.+Kumarauthor=N.+Parrauthor=K.+Leeauthor=D.+Lugoauthor=P.+Jeffreyauthor=S.+Taylorauthor=M.+L.+Vecellioauthor=C.+Bountraauthor=P.+E.+Brennanauthor=A.+O%E2%80%99Mahonyauthor=S.+Velichkoauthor=S.+M%C3%BCllerauthor=D.+Hayauthor=D.+L.+Danielsauthor=M.+Urhauthor=N.+B.+La+Thangueauthor=T.+Kouzaridesauthor=R.+Prinjhaauthor=J.+Schwallerauthor=S.+Knapp&title=Generation+of+a+Selective+Small+Molecule+Inhibitor+of+the+CBP%2FP300+Bromodomain+for+Leukemia+Therapy&doi=10.1158%2F0008-5472.CAN-15-0236"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Generation of a Selective Small Molecule Inhibitor of the CBP/p300 Bromodomain for Leukemia Therapy</span></div><div class="casAuthors">Picaud, Sarah; Fedorov, Oleg; Thanasopoulou, Angeliki; Leonards, Katharina; Jones, Katherine; Meier, Julia; Olzscha, Heidi; Monteiro, Octovia; Martin, Sarah; Philpott, Martin; Tumber, Anthony; Filippakopoulos, Panagis; Yapp, Clarence; Wells, Christopher; Che, Ka Hing; Bannister, Andrew; Robson, Samuel; Kumar, Umesh; Parr, Nigel; Lee, Kevin; Lugo, Dave; Jeffrey, Philip; Taylor, Simon; Vecellio, Matteo L.; Bountra, Chas; Brennan, Paul E.; O'Mahony, Alison; Velichko, Sharlene; Mueller, Susanne; Hay, Duncan; Daniels, Danette L.; Urh, Marjeta; La Thangue, Nicholas B.; Kouzarides, Tony; Prinjha, Rab; Schwaller, Juerg; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5106-5119</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The histone acetyltransferases CBP/p300 are involved in recurrent leukemia-assocd. chromosomal translocations and are key regulators of cell growth.  Therefore, efforts to generate inhibitors of CBP/p300 are of clin. value.  We developed a specific and potent acetyl-lysine competitive protein-protein interaction inhibitor, I-CBP112, that targets the CBP/p300 bromodomains.  Exposure of human and mouse leukemic cell lines to I-CBP112 resulted in substantially impaired colony formation and induced cellular differentiation without significant cytotoxicity.  I-CBP112 significantly reduced the leukemia-initiating potential of MLL-AF9+ acute myeloid leukemia cells in a dose-dependent manner in vitro and in vivo.  Interestingly, I-CBP112 increased the cytotoxic activity of BET bromodomain inhibitor JQ1 as well as doxorubicin.  Collectively, we report the development and preclin. evaluation of a novel, potent inhibitor targeting CBP/p300 bromodomains that impairs aberrant self-renewal of leukemic cells.  The synergistic effects of I-CBP112 and current std. therapy (doxorubicin) as well as emerging treatment strategies (BET inhibition) provide new opportunities for combinatorial treatment of leukemia and potentially other cancers.  Cancer Res; 75(23); 5106-19. ©2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWk5dMbmC6K7Vg90H21EOLACvtfcHk0lgVcw3QCkwJ_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFensrzN&md5=da936fa34139ba27abc748db5f78c95e</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-15-0236&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-15-0236%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DThanasopoulou%26aufirst%3DA.%26aulast%3DLeonards%26aufirst%3DK.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DMeier%26aufirst%3DJ.%26aulast%3DOlzscha%26aufirst%3DH.%26aulast%3DMonteiro%26aufirst%3DO.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DBannister%26aufirst%3DA.%26aulast%3DRobson%26aufirst%3DS.%26aulast%3DKumar%26aufirst%3DU.%26aulast%3DParr%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DVecellio%26aufirst%3DM.%2BL.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DVelichko%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DHay%26aufirst%3DD.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DUrh%26aufirst%3DM.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DPrinjha%26aufirst%3DR.%26aulast%3DSchwaller%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DGeneration%2520of%2520a%2520Selective%2520Small%2520Molecule%2520Inhibitor%2520of%2520the%2520CBP%252FP300%2520Bromodomain%2520for%2520Leukemia%2520Therapy%26jtitle%3DCancer%2520Res.%26date%3D2015%26volume%3D75%26spage%3D5106%26epage%3D5119%26doi%3D10.1158%2F0008-5472.CAN-15-0236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Hammitzsch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tallant, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Mahony, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hay, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, F. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Mossawi, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Wit, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vecellio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wordsworth, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowness, P.</span><span> </span><span class="NLM_article-title">CBP30, a Selective CBP/P300 Bromodomain Inhibitor, Suppresses Human Th17 Responses</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">112</span><span class="NLM_x">, </span> <span class="NLM_fpage">10768</span><span class="NLM_x">–</span> <span class="NLM_lpage">10773</span><span class="refDoi"> DOI: 10.1073/pnas.1501956112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1073%2Fpnas.1501956112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=26261308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlSit7jI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=10768-10773&author=A.+Hammitzschauthor=C.+Tallantauthor=O.+Fedorovauthor=A.+O%E2%80%99Mahonyauthor=P.+E.+Brennanauthor=D.+A.+Hayauthor=F.+O.+Martinezauthor=M.+H.+Al-Mossawiauthor=J.+de+Witauthor=M.+Vecellioauthor=C.+Wellsauthor=P.+Wordsworthauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=P.+Bowness&title=CBP30%2C+a+Selective+CBP%2FP300+Bromodomain+Inhibitor%2C+Suppresses+Human+Th17+Responses&doi=10.1073%2Fpnas.1501956112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">CBP30, a selective CBP/p300 bromodomain inhibitor, suppresses human Th17 responses</span></div><div class="casAuthors">Hammitzsch, Ariane; Tallant, Cynthia; Fedorov, Oleg; O'Mahony, Alison; Brennan, Paul E.; Hay, Duncan A.; Martinez, Fernando O.; Al-Mossawi, M. Hussein; de Wit, Jelle; Vecellio, Matteo; Wells, Christopher; Wordsworth, Paul; Muller, Susanne; Knapp, Stefan; Bowness, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">10768-10773</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Th17 responses are crit. to a variety of human autoimmune diseases, and therapeutic targeting with monoclonal antibodies against IL-17 and IL-23 has shown considerable promise.  Here, the authors report data to support selective bromodomain blockade of the transcriptional coactivators CBP (CREB binding protein) and p300 as an alternative approach to inhibit human Th17 responses.  The authors show that CBP30 has marked mol. specificity for the bromodomains of CBP and p300, compared with 43 other bromodomains.  In unbiased cellular testing on a diverse panel of cultured primary human cells, CBP30 reduced immune cell prodn. of IL-17A and other proinflammatory cytokines.  CBP30 also inhibited IL-17A secretion by Th17 cells from healthy donors and patients with ankylosing spondylitis and psoriatic arthritis.  Transcriptional profiling of human T cells after CBP30 treatment showed a much more restricted effect on gene expression than that obsd. with the pan-BET (bromo and extraterminal domain protein family) bromodomain inhibitor JQ1.  This selective targeting of the CBP/p300 bromodomain by CBP30 will potentially lead to fewer side effects than with the broadly acting epigenetic inhibitors currently in clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4fYjOUV2xW7Vg90H21EOLACvtfcHk0lgVcw3QCkwJ_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlSit7jI&md5=fb9de88ecdd6441628ae1455ac11b65c</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1501956112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1501956112%26sid%3Dliteratum%253Aachs%26aulast%3DHammitzsch%26aufirst%3DA.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DMartinez%26aufirst%3DF.%2BO.%26aulast%3DAl-Mossawi%26aufirst%3DM.%2BH.%26aulast%3Dde%2BWit%26aufirst%3DJ.%26aulast%3DVecellio%26aufirst%3DM.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DWordsworth%26aufirst%3DP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBowness%26aufirst%3DP.%26atitle%3DCBP30%252C%2520a%2520Selective%2520CBP%252FP300%2520Bromodomain%2520Inhibitor%252C%2520Suppresses%2520Human%2520Th17%2520Responses%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3D10768%26epage%3D10773%26doi%3D10.1073%2Fpnas.1501956112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Xu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Unzue, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spiliotopoulos, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nevado, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Discovery of CREBB Bromodomain Inhibitors by High-Throughput Docking and Hit Optimization Guided by Molecular Dynamics</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">1340</span><span class="NLM_x">–</span> <span class="NLM_lpage">1349</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00171</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00171" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1340-1349&author=M.+Xuauthor=A.+Unzueauthor=J.+Dongauthor=D.+Spiliotopoulosauthor=C.+Nevadoauthor=A.+Caflisch&title=Discovery+of+CREBB+Bromodomain+Inhibitors+by+High-Throughput+Docking+and+Hit+Optimization+Guided+by+Molecular+Dynamics&doi=10.1021%2Facs.jmedchem.5b00171"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47a&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00171&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00171%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DM.%26aulast%3DUnzue%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DJ.%26aulast%3DSpiliotopoulos%26aufirst%3DD.%26aulast%3DNevado%26aufirst%3DC.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DDiscovery%2520of%2520CREBB%2520Bromodomain%2520Inhibitors%2520by%2520High-Throughput%2520Docking%2520and%2520Hit%2520Optimization%2520Guided%2520by%2520Molecular%2520Dynamics%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1340%26epage%3D1349%26doi%3D10.1021%2Facs.jmedchem.5b00171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit47b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Unzue, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiedmer, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spiliotopoulos, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nevado, C.</span><span> </span><span class="NLM_article-title">Fragment-Based Design of Selective Nano-molar Ligands of the CREBBP Bromodomain</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x"> (</span><span class="NLM_issue">59</span><span class="NLM_x">) </span> <span class="NLM_fpage">1350</span><span class="NLM_x">–</span> <span class="NLM_lpage">1356</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00172</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00172" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1350-1356&issue=59&author=A.+Unzueauthor=M.+Xuauthor=J.+Dongauthor=L.+Wiedmerauthor=D.+Spiliotopoulosauthor=A.+Caflischauthor=C.+Nevado&title=Fragment-Based+Design+of+Selective+Nano-molar+Ligands+of+the+CREBBP+Bromodomain&doi=10.1021%2Facs.jmedchem.5b00172"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47b&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00172&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00172%26sid%3Dliteratum%253Aachs%26aulast%3DUnzue%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DJ.%26aulast%3DWiedmer%26aufirst%3DL.%26aulast%3DSpiliotopoulos%26aufirst%3DD.%26aulast%3DCaflisch%26aufirst%3DA.%26aulast%3DNevado%26aufirst%3DC.%26atitle%3DFragment-Based%2520Design%2520of%2520Selective%2520Nano-molar%2520Ligands%2520of%2520the%2520CREBBP%2520Bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D59%26spage%3D1350%26epage%3D1356%26doi%3D10.1021%2Facs.jmedchem.5b00172" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Taylor, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Côté, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hewitt, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pastor, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leblanc, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nasveschuk, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romero, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crawford, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cantone, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayaram, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setser, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beresini, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Leon Boenig, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conery, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cummings, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dakin, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flynn, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, O. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaufman, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keller, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiefer, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pardo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsui, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zawadzke, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sims, R. J.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cochran, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magnuson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albrecht, B. K.</span><span> </span><span class="NLM_article-title">Fragment-Based Discovery of a Selective and Cell-Active Benzodiazepinone CBP/EP300 Bromodomain Inhibitor (CPI-637)</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">531</span><span class="NLM_x">–</span> <span class="NLM_lpage">536</span><span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00075</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00075" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC28Xkt1Cms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=531-536&author=A.+M.+Taylorauthor=A.+C%C3%B4t%C3%A9author=M.+C.+Hewittauthor=R.+Pastorauthor=Y.+Leblancauthor=C.+G.+Nasveschukauthor=F.+A.+Romeroauthor=T.+D.+Crawfordauthor=N.+Cantoneauthor=H.+Jayaramauthor=J.+Setserauthor=J.+Murrayauthor=M.+H.+Beresiniauthor=G.+de+Leon+Boenigauthor=Z.+Chenauthor=A.+R.+Coneryauthor=R.+T.+Cummingsauthor=L.+A.+Dakinauthor=E.+M.+Flynnauthor=O.+W.+Huangauthor=S.+Kaufmanauthor=P.+J.+Kellerauthor=J.+R.+Kieferauthor=T.+Laiauthor=Y.+Liauthor=J.+Liaoauthor=W.+Liuauthor=H.+Luauthor=E.+Pardoauthor=V.+Tsuiauthor=J.+Wangauthor=Y.+Wangauthor=Z.+Xuauthor=F.+Yanauthor=D.+Yuauthor=L.+Zawadzkeauthor=X.+Zhuauthor=X.+Zhuauthor=R.+J.+Simsauthor=A.+G.+Cochranauthor=S.+Bellonauthor=J.+E.+Audiaauthor=S.+Magnusonauthor=B.+K.+Albrecht&title=Fragment-Based+Discovery+of+a+Selective+and+Cell-Active+Benzodiazepinone+CBP%2FEP300+Bromodomain+Inhibitor+%28CPI-637%29&doi=10.1021%2Facsmedchemlett.6b00075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of a Selective and Cell-Active Benzodiazepinone CBP/EP300 Bromodomain Inhibitor (CPI-637)</span></div><div class="casAuthors">Taylor, Alexander M.; Cote, Alexandre; Hewitt, Michael C.; Pastor, Richard; Leblanc, Yves; Nasveschuk, Christopher G.; Romero, F. Anthony; Crawford, Terry D.; Cantone, Nico; Jayaram, Hariharan; Setser, Jeremy; Murray, Jeremy; Beresini, Maureen H.; de Leon Boenig, Gladys; Chen, Zhongguo; Conery, Andrew R.; Cummings, Richard T.; Dakin, Leslie A.; Flynn, E. Megan; Huang, Oscar W.; Kaufman, Susan; Keller, Patricia J.; Kiefer, James R.; Lai, Tommy; Li, Yingjie; Liao, Jiangpeng; Liu, Wenfeng; Lu, Henry; Pardo, Eneida; Tsui, Vickie; Wang, Jian; Wang, Yongyun; Xu, Zhaowu; Yan, Fen; Yu, Dong; Zawadzke, Laura; Zhu, Xiaoqin; Zhu, Xiaoyu; Sims, Robert J.; Cochran, Andrea G.; Bellon, Steve; Audia, James E.; Magnuson, Steven; Albrecht, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">531-536</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Benzo-fused heterocycles, particularly benzodiazepinones such as I, were prepd. as inhibitors of the bromodomain-contg. proteins CBP (CREB, cAMP response element binding protein) and EP300 (adenoviral E1A-binding protein of 300 kDa) for potential use as antitumor agents; the core benzodiazepinone moiety was identified using fragment-based screening.  The structures of four benzodiazepinones, including I, bound to the bromodomain fragment of human CREB were detd. by X-ray crystallog.  The free energy, enthalpy, and entropy of the binding of I to CREB and its selectivity for CREB and EP300 over other bromodomain-contg. proteins was detd.; I inhibited expression of the transcription factor MYC in human plasmacytoma cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNm7Blycl0-7Vg90H21EOLACvtfcHk0lhjgnz4-M2ACg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xkt1Cms7c%253D&md5=8440a9bb3460ff66d43f15041dbbc19d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00075%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DPastor%26aufirst%3DR.%26aulast%3DLeblanc%26aufirst%3DY.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DCantone%26aufirst%3DN.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DSetser%26aufirst%3DJ.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3Dde%2BLeon%2BBoenig%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DConery%26aufirst%3DA.%2BR.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DHuang%26aufirst%3DO.%2BW.%26aulast%3DKaufman%26aufirst%3DS.%26aulast%3DKeller%26aufirst%3DP.%2BJ.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DLai%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLiao%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DLu%26aufirst%3DH.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DYu%26aufirst%3DD.%26aulast%3DZawadzke%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DMagnuson%26aufirst%3DS.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26atitle%3DFragment-Based%2520Discovery%2520of%2520a%2520Selective%2520and%2520Cell-Active%2520Benzodiazepinone%2520CBP%252FEP300%2520Bromodomain%2520Inhibitor%2520%2528CPI-637%2529%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D531%26epage%3D536%26doi%3D10.1021%2Facsmedchemlett.6b00075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Popp, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tallant, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bracher, F.</span><span> </span><span class="NLM_article-title">Development of Selective CBP/P300 Benzoxazepine Bromodomain Inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">8889</span><span class="NLM_x">–</span> <span class="NLM_lpage">8912</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00774</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00774" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFKis77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8889-8912&author=T.+A.+Poppauthor=C.+Tallantauthor=C.+Rogersauthor=O.+Fedorovauthor=P.+E.+Brennanauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=F.+Bracher&title=Development+of+Selective+CBP%2FP300+Benzoxazepine+Bromodomain+Inhibitors&doi=10.1021%2Facs.jmedchem.6b00774"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Selective CBP/P300 Benzoxazepine Bromodomain Inhibitors</span></div><div class="casAuthors">Popp, Tobias A.; Tallant, Cynthia; Rogers, Catherine; Fedorov, Oleg; Brennan, Paul E.; Mueller, Susanne; Knapp, Stefan; Bracher, Franz</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8889-8912</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">CBP (CREB (cAMP responsive element binding protein) binding protein (CREBBP)) and P300 (adenovirus E1A-assocd. 300 kDa protein) are two closely related histone acetyltransferases (HATs) that play a key role in the regulation of gene transcription.  Both proteins contain a bromodomain flanking the HAT catalytic domain that is important for the targeting of CBP/P300 to chromatin and which offers an opportunity for the development of protein-protein interaction inhibitors.  Here the authors present the development of CBP/P300 bromodomain inhibitors with 2,3,4,5-tetrahydro-1,4-benzoxazepine backbone, an N-acetyl-lysine mimetic scaffold that led to the recent development of the chem. probe I-CBP112.  The authors present comprehensive SAR of this inhibitor class as well as demonstration of cellular on target activity of the most potent and selective inhibitor I, which showed 134 nM affinity for CBP with excellent selectivity over other bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwAZFuNP0NkLVg90H21EOLACvtfcHk0lgHzkpwPnFAkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFKis77E&md5=6bd914d24abf00a8f074bf1918fb4ee1</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00774&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00774%26sid%3Dliteratum%253Aachs%26aulast%3DPopp%26aufirst%3DT.%2BA.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBracher%26aufirst%3DF.%26atitle%3DDevelopment%2520of%2520Selective%2520CBP%252FP300%2520Benzoxazepine%2520Bromodomain%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8889%26epage%3D8912%26doi%3D10.1021%2Facs.jmedchem.6b00774" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Denny, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flick, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coe, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langille, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basak, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stock, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sahasrabudhe, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonin, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hay, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pletcher, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chekler, E. L. P.</span><span> </span><span class="NLM_article-title">Structure-Based Design of Highly Selective Inhibitors of the CREB Binding Protein Bromodomain</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">5349</span><span class="NLM_x">–</span> <span class="NLM_lpage">5363</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01839</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01839" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1Glu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=5349-5363&author=R.+A.+Dennyauthor=A.+C.+Flickauthor=J.+W.+Coeauthor=J.+Langilleauthor=A.+Basakauthor=S.+Liuauthor=I.+A.+Stockauthor=P.+Sahasrabudheauthor=P.+D.+Boninauthor=D.+A.+Hayauthor=P.+E.+Brennanauthor=M.+T.+Pletcherauthor=L.+H.+Jonesauthor=E.+L.+P.+Chekler&title=Structure-Based+Design+of+Highly+Selective+Inhibitors+of+the+CREB+Binding+Protein+Bromodomain&doi=10.1021%2Facs.jmedchem.6b01839"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Highly Selective Inhibitors of the CREB Binding Protein Bromodomain</span></div><div class="casAuthors">Denny, R. Aldrin; Flick, Andrew C.; Coe, Jotham; Langille, Jonathan; Basak, Arindrajit; Liu, Shenping; Stock, Ingrid; Sahasrabudhe, Parag; Bonin, Paul; Hay, Duncan A.; Brennan, Paul E.; Pletcher, Mathew; Jones, Lyn H.; Chekler, Eugene L. Piatnitski</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5349-5363</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chem. probes are required for preclin. target validation to interrogate novel biol. targets and pathways.  Selective inhibitors of the CREB binding protein (CREBBP)/EP300 bromodomains are required to facilitate the elucidation of biol. assocd. with these important epigenetic targets.  Medicinal chem. optimization that paid particular attention to physiochem. properties delivered chem. probes with desirable potency, selectivity, and permeability attributes.  An important feature of the optimization process was the successful application of rational structure-based drug design to address bromodomain selectivity issues (particularly against the structurally related BRD4 protein).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLWkqF-qAmdbVg90H21EOLACvtfcHk0lgHzkpwPnFAkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1Glu70%253D&md5=23676a973abba59088dedd5502877105</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01839%26sid%3Dliteratum%253Aachs%26aulast%3DDenny%26aufirst%3DR.%2BA.%26aulast%3DFlick%26aufirst%3DA.%2BC.%26aulast%3DCoe%26aufirst%3DJ.%2BW.%26aulast%3DLangille%26aufirst%3DJ.%26aulast%3DBasak%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DStock%26aufirst%3DI.%2BA.%26aulast%3DSahasrabudhe%26aufirst%3DP.%26aulast%3DBonin%26aufirst%3DP.%2BD.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DPletcher%26aufirst%3DM.%2BT.%26aulast%3DJones%26aufirst%3DL.%2BH.%26aulast%3DChekler%26aufirst%3DE.%2BL.%2BP.%26atitle%3DStructure-Based%2520Design%2520of%2520Highly%2520Selective%2520Inhibitors%2520of%2520the%2520CREB%2520Binding%2520Protein%2520Bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D5349%26epage%3D5363%26doi%3D10.1021%2Facs.jmedchem.6b01839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Spiliotopoulos, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wamhoff, E.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deerain, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, J.-R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aretz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rademacher, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Virtual Screen to NMR (VS2NMR): Discovery of Fragment Hits for the CBP Bromodomain</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">2472</span><span class="NLM_x">–</span> <span class="NLM_lpage">2478</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2017.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1016%2Fj.bmcl.2017.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=28410781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtVertrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=2472-2478&author=D.+Spiliotopoulosauthor=J.+Zhuauthor=E.-C.+Wamhoffauthor=N.+Deerainauthor=J.-R.+Marchandauthor=J.+Aretzauthor=C.+Rademacherauthor=A.+Caflisch&title=Virtual+Screen+to+NMR+%28VS2NMR%29%3A+Discovery+of+Fragment+Hits+for+the+CBP+Bromodomain&doi=10.1016%2Fj.bmcl.2017.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screen to NMR (VS2NMR): Discovery of fragment hits for the CBP bromodomain</span></div><div class="casAuthors">Spiliotopoulos, Dimitrios; Zhu, Jian; Wamhoff, Eike-Christian; Deerain, Nicholas; Marchand, Jean-Remy; Aretz, Jonas; Rademacher, Christoph; Caflisch, Amedeo</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2472-2478</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Overexpression of the CREB-binding protein (CBP), a bromodomain-contg. transcription coactivator involved in a variety of cellular processes, has been obsd. in several types of cancer with a correlation to aggressiveness.  The authors have screened a library of nearly 1500 fragments by high-throughput docking into the CBP bromodomain followed by binding energy evaluation using a force field with electrostatic solvation.  Twenty of the 39 fragments selected by virtual screening are pos. in one or more ligand-obsd. NMR expts.  Four crystal structures of the CBP bromodomain in complex with in silico screening hits validate the pose predicted by docking.  Thus, the success ratio of the high-throughput docking procedure is 50% or 10% if one considers the validation by ligand-obsd. NMR spectroscopy or x-ray crystallog., resp.  Compds. 1 and 3 show favorable ligand efficiency in two different in vitro binding assays.  The structure of the CBP bromodomain in the complex with the brominated pyrrole 1 suggests fragment growing by Suzuki coupling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkYM7NMX_QGLVg90H21EOLACvtfcHk0lgHzkpwPnFAkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtVertrg%253D&md5=d71ad8368839c3026fabf75197ba2ac9</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DSpiliotopoulos%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DWamhoff%26aufirst%3DE.-C.%26aulast%3DDeerain%26aufirst%3DN.%26aulast%3DMarchand%26aufirst%3DJ.-R.%26aulast%3DAretz%26aufirst%3DJ.%26aulast%3DRademacher%26aufirst%3DC.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DVirtual%2520Screen%2520to%2520NMR%2520%2528VS2NMR%2529%253A%2520Discovery%2520of%2520Fragment%2520Hits%2520for%2520the%2520CBP%2520Bromodomain%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D2472%26epage%3D2478%26doi%3D10.1016%2Fj.bmcl.2017.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Hügle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lucas, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ostrovskyi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Regenass, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerhardt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Einsle, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hau, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breit, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Günther, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wohlwend, D.</span><span> </span><span class="NLM_article-title">Beyond the BET Family: Targeting CBP/p300 with 4-Acyl Pyrroles</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">12476</span><span class="NLM_x">–</span> <span class="NLM_lpage">12480</span><span class="refDoi"> DOI: 10.1002/anie.201705516</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1002%2Fanie.201705516" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVCrt73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=12476-12480&author=M.+H%C3%BCgleauthor=X.+Lucasauthor=D.+Ostrovskyiauthor=P.+Regenassauthor=S.+Gerhardtauthor=O.+Einsleauthor=M.+Hauauthor=M.+Jungauthor=B.+Breitauthor=S.+G%C3%BCntherauthor=D.+Wohlwend&title=Beyond+the+BET+Family%3A+Targeting+CBP%2Fp300+with+4-Acyl+Pyrroles&doi=10.1002%2Fanie.201705516"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond the BET Family: Targeting CBP/p300 with 4-Acyl Pyrroles</span></div><div class="casAuthors">Huegle, Martin; Lucas, Xavier; Ostrovskyi, Dmytro; Regenass, Pierre; Gerhardt, Stefan; Einsle, Oliver; Hau, Mirjam; Jung, Manfred; Breit, Bernhard; Guenther, Stefan; Wohlwend, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">12476-12480</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Bromodomain and extra-terminal domain (BET) inhibitors are widely used both as chem. tools to study the biol. role of their targets in living organisms and as candidates for drug development against several cancer variants and human disorders.  However, non-BET bromodomains such as those in p300 and CBP are less studied.  XDM-CBP is a highly potent and selective inhibitor for the bromodomains of CBP and p300 derived from a pan-selective BET BRD-binding fragment.  Along with x-ray crystal-structure anal. and thermodn. profiling, XDM-CBP was used in screenings of several cancer cell lines in vitro to study its inhibitory potential on cancer cell proliferation.  XDM-CBP is demonstrated to be a potent and selective CBP/p300 inhibitor that acts on specific cancer cell lines, in particular malignant melanoma, breast cancer, and leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsOlQ2hI7S9bVg90H21EOLACvtfcHk0lhGwQEcW6DXXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVCrt73I&md5=f4e029831a971663cf51ca3ee485f43e</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fanie.201705516&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201705516%26sid%3Dliteratum%253Aachs%26aulast%3DH%25C3%25BCgle%26aufirst%3DM.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DOstrovskyi%26aufirst%3DD.%26aulast%3DRegenass%26aufirst%3DP.%26aulast%3DGerhardt%26aufirst%3DS.%26aulast%3DEinsle%26aufirst%3DO.%26aulast%3DHau%26aufirst%3DM.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DBreit%26aufirst%3DB.%26aulast%3DG%25C3%25BCnther%26aufirst%3DS.%26aulast%3DWohlwend%26aufirst%3DD.%26atitle%3DBeyond%2520the%2520BET%2520Family%253A%2520Targeting%2520CBP%252Fp300%2520with%25204-Acyl%2520Pyrroles%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D12476%26epage%3D12480%26doi%3D10.1002%2Fanie.201705516" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Romero, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsiu, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albrecht, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beresini, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Leon Boenig, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bronner, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, E. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choo, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crawford, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cyr, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Almeida Nagata, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gascoigne, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grogan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatzivassiliou, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunsaker, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaufman, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ly, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maher, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merchant, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ran, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, B.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magnuson, S.</span><span> </span><span class="NLM_article-title">GNE-781, A Highly Advanced Potent and Selective Bromodomain Inhibitor of CBP/P300</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">9162</span><span class="NLM_x">–</span> <span class="NLM_lpage">9183</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00796</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00796" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9162-9183&author=F.+A.+Romeroauthor=J.+Murrayauthor=K.+W.+Laiauthor=V.+Tsiuauthor=B.+K.+Albrechtauthor=M.+H.+Beresiniauthor=G.+de+Leon+Boenigauthor=S.+M.+Bronnerauthor=E.+W.+Chanauthor=K.+Chenauthor=Z.+Chenauthor=E.+F.+Chooauthor=K.+Clarkauthor=T.+D.+Crawfordauthor=P.+Cyrauthor=D.+De+Almeida+Nagataauthor=K.+E.+Gascoigneauthor=J.+Groganauthor=G.+Hatzivassiliouauthor=T.+L.+Hunsakerauthor=S.+Kaufmanauthor=Y.+Liauthor=J.+Liaoauthor=W.+Liuauthor=J.+Lyauthor=J.+Maherauthor=M.+Merchantauthor=Y.+Ranauthor=A.+M.+Taylorauthor=J.+Waiauthor=F.+Wangauthor=X.+Weiauthor=D.+Yuauthor=B.-Y.+Zhuauthor=X.+Zhuauthor=S.+Magnuson&title=GNE-781%2C+A+Highly+Advanced+Potent+and+Selective+Bromodomain+Inhibitor+of+CBP%2FP300&doi=10.1021%2Facs.jmedchem.7b00796"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00796%26sid%3Dliteratum%253Aachs%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DLai%26aufirst%3DK.%2BW.%26aulast%3DTsiu%26aufirst%3DV.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3Dde%2BLeon%2BBoenig%26aufirst%3DG.%26aulast%3DBronner%26aufirst%3DS.%2BM.%26aulast%3DChan%26aufirst%3DE.%2BW.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DChoo%26aufirst%3DE.%2BF.%26aulast%3DClark%26aufirst%3DK.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DCyr%26aufirst%3DP.%26aulast%3DDe%2BAlmeida%2BNagata%26aufirst%3DD.%26aulast%3DGascoigne%26aufirst%3DK.%2BE.%26aulast%3DGrogan%26aufirst%3DJ.%26aulast%3DHatzivassiliou%26aufirst%3DG.%26aulast%3DHunsaker%26aufirst%3DT.%2BL.%26aulast%3DKaufman%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLiao%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DLy%26aufirst%3DJ.%26aulast%3DMaher%26aufirst%3DJ.%26aulast%3DMerchant%26aufirst%3DM.%26aulast%3DRan%26aufirst%3DY.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DWai%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DB.-Y.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DGNE-781%252C%2520A%2520Highly%2520Advanced%2520Potent%2520and%2520Selective%2520Bromodomain%2520Inhibitor%2520of%2520CBP%252FP300%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D9162%26epage%3D9183%26doi%3D10.1021%2Facs.jmedchem.7b00796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Crawford, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romero, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsui, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Leon Boenig, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noland, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ly, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choo, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunsaker, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, E. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merchant, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kharbanda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gascoigne, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaufman, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beresini, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conery, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Côté, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayaram, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiefer, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kleinheinz, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maher, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pardo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poy, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spillane, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yen, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zawadzke, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cummings, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cochran, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albrecht, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magnuson, S.</span><span> </span><span class="NLM_article-title">Discovery of a Potent and Selective in Vivo Probe (GNE-272) for the Bromodomains of CBP/EP300</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">10549</span><span class="NLM_x">–</span> <span class="NLM_lpage">10563</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01022</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01022" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFGhsb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=10549-10563&author=T.+D.+Crawfordauthor=F.+A.+Romeroauthor=K.+W.+Laiauthor=V.+Tsuiauthor=A.+M.+Taylorauthor=G.+de+Leon+Boenigauthor=C.+L.+Nolandauthor=J.+Murrayauthor=J.+Lyauthor=E.+F.+Chooauthor=T.+L.+Hunsakerauthor=E.+W.+Chanauthor=M.+Merchantauthor=S.+Kharbandaauthor=K.+E.+Gascoigneauthor=S.+Kaufmanauthor=M.+H.+Beresiniauthor=J.+Liaoauthor=W.+Liuauthor=K.+X.+Chenauthor=Z.+Chenauthor=A.+R.+Coneryauthor=A.+C%C3%B4t%C3%A9author=H.+Jayaramauthor=Y.+Jiangauthor=J.+R.+Kieferauthor=T.+Kleinheinzauthor=Y.+Liauthor=J.+Maherauthor=E.+Pardoauthor=F.+Poyauthor=K.+L.+Spillaneauthor=F.+Wangauthor=J.+Wangauthor=X.+Weiauthor=Z.+Xuauthor=Z.+Xuauthor=I.+Yenauthor=L.+Zawadzkeauthor=X.+Zhuauthor=S.+Bellonauthor=R.+Cummingsauthor=A.+G.+Cochranauthor=B.+K.+Albrechtauthor=S.+Magnuson&title=Discovery+of+a+Potent+and+Selective+in+Vivo+Probe+%28GNE-272%29+for+the+Bromodomains+of+CBP%2FEP300&doi=10.1021%2Facs.jmedchem.6b01022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent and Selective in Vivo Probe (GNE-272) for the Bromodomains of CBP/EP300</span></div><div class="casAuthors">Crawford, Terry D.; Romero, F. Anthony; Lai, Kwong Wah; Tsui, Vickie; Taylor, Alexander M.; de Leon Boenig, Gladys; Noland, Cameron L.; Murray, Jeremy; Ly, Justin; Choo, Edna F.; Hunsaker, Thomas L.; Chan, Emily W.; Merchant, Mark; Kharbanda, Samir; Gascoigne, Karen E.; Kaufman, Susan; Beresini, Maureen H.; Liao, Jiangpeng; Liu, Wenfeng; Chen, Kevin X.; Chen, Zhongguo; Conery, Andrew R.; Cote, Alexandre; Jayaram, Hariharan; Jiang, Ying; Kiefer, James R.; Kleinheinz, Tracy; Li, Yingjie; Maher, Jonathan; Pardo, Eneida; Poy, Florence; Spillane, Kerry L.; Wang, Fei; Wang, Jian; Wei, Xiaocang; Xu, Zhaowu; Xu, Zhongya; Yen, Ivana; Zawadzke, Laura; Zhu, Xiaoyu; Bellon, Steven; Cummings, Richard; Cochran, Andrea G.; Albrecht, Brian K.; Magnuson, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10549-10563</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The single bromodomain of the closely related transcriptional regulators CBP/EP300 is a target of much recent interest in cancer and immune system regulation.  A co-crystal structure of a ligand-efficient screening hit and the CBP bromodomain guided initial design targeting the LPF shelf, ZA loop, and acetylated lysine binding regions.  Structure-activity relationship studies allowed the authors to identify a more potent analog.  Optimization of permeability and microsomal stability and subsequent improvement of mouse hepatocyte stability afforded 59 (GNE-272, TR-FRET IC50 = 0.02 μM, BRET IC50 = 0.41 μM, BRD4(1) IC50 = 13 μM) that retained the best balance of cell potency, selectivity, and in vivo PK.  GNE-272 showed a marked antiproliferative effect in hematol. cancer cell lines and modulates MYC expression in vivo that corresponds with antitumor activity in an AML tumor model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6T5ODn2Q-2LVg90H21EOLACvtfcHk0lhGwQEcW6DXXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFGhsb%252FO&md5=22ed46cb031022aa6aff23c20d07e7dd</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01022%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DLai%26aufirst%3DK.%2BW.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3Dde%2BLeon%2BBoenig%26aufirst%3DG.%26aulast%3DNoland%26aufirst%3DC.%2BL.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DLy%26aufirst%3DJ.%26aulast%3DChoo%26aufirst%3DE.%2BF.%26aulast%3DHunsaker%26aufirst%3DT.%2BL.%26aulast%3DChan%26aufirst%3DE.%2BW.%26aulast%3DMerchant%26aufirst%3DM.%26aulast%3DKharbanda%26aufirst%3DS.%26aulast%3DGascoigne%26aufirst%3DK.%2BE.%26aulast%3DKaufman%26aufirst%3DS.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3DLiao%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DK.%2BX.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DConery%26aufirst%3DA.%2BR.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DKleinheinz%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DMaher%26aufirst%3DJ.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DSpillane%26aufirst%3DK.%2BL.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DYen%26aufirst%3DI.%26aulast%3DZawadzke%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520a%2520Potent%2520and%2520Selective%2520in%2520Vivo%2520Probe%2520%2528GNE-272%2529%2520for%2520the%2520Bromodomains%2520of%2520CBP%252FEP300%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D10549%26epage%3D10563%26doi%3D10.1021%2Facs.jmedchem.6b01022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Zhou, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shang, Z.</span><span> </span><span class="NLM_article-title">Fluorine Bonding – How Does It Work In Protein–Ligand Interactions?</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">2344</span><span class="NLM_x">–</span> <span class="NLM_lpage">2355</span><span class="refDoi"> DOI: 10.1021/ci9002393</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci9002393" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2017&pages=2344-2355&author=P.+Zhouauthor=J.+Zouauthor=F.+Tianauthor=Z.+Shang&title=Fluorine+Bonding+%E2%80%93+How+Does+It+Work+In+Protein%E2%80%93Ligand+Interactions%3F&doi=10.1021%2Fci9002393"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fci9002393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci9002393%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DP.%26aulast%3DZou%26aufirst%3DJ.%26aulast%3DTian%26aufirst%3DF.%26aulast%3DShang%26aufirst%3DZ.%26atitle%3DFluorine%2520Bonding%2520%25E2%2580%2593%2520How%2520Does%2520It%2520Work%2520In%2520Protein%25E2%2580%2593Ligand%2520Interactions%253F%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D49%26spage%3D2344%26epage%3D2355%26doi%3D10.1021%2Fci9002393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="note"><p class="first last">Details regarding the computational docking protocol are provided in the <a class="ref internalNav" href="#notes1" aria-label="Supporting Information">Supporting Information</a>.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Bennett, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tallant, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monteiro, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meier, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gamble, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitsky, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nunez-Alonso, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haendler, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Discovery of a Chemical Tool Inhibitor Targeting the Bromodomains of TRIM24 and BRPF</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">1642</span><span class="NLM_x">–</span> <span class="NLM_lpage">1647</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00458</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00458" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotlCiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1642-1647&author=J.+Bennettauthor=O.+Fedorovauthor=C.+Tallantauthor=O.+Monteiroauthor=J.+Meierauthor=V.+Gambleauthor=P.+Savitskyauthor=G.+A.+Nunez-Alonsoauthor=B.+Haendlerauthor=C.+Rogersauthor=P.+E.+Brennanauthor=S.+M%C3%BCllerauthor=S.+Knapp&title=Discovery+of+a+Chemical+Tool+Inhibitor+Targeting+the+Bromodomains+of+TRIM24+and+BRPF&doi=10.1021%2Facs.jmedchem.5b00458"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Chemical Tool Inhibitor Targeting the Bromodomains of TRIM24 and BRPF</span></div><div class="casAuthors">Bennett, James; Fedorov, Oleg; Tallant, Cynthia; Monteiro, Octovia; Meier, Julia; Gamble, Vicky; Savitsky, Pavel; Nunez-Alonso, Graciela A.; Haendler, Bernard; Rogers, Catherine; Brennan, Paul E.; Muller, Susanne; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1642-1647</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">TRIM24 is a transcriptional regulator as well as an E3 ubiquitin ligase.  It is overexpressed in diverse tumors, and high expression levels have been linked to poor prognosis in breast cancer patients.  TRIM24 contains a PHD/bromodomain offering the opportunity to develop protein interaction inhibitors that target this protein interaction module.  Here the authors identified potent acetyl-lysine mimetic benzimidazolones TRIM24 bromodomain inhibitors.  The best compd. of this series is a selective BRPF1B/TRIM24 dual inhibitor that bound with a KD of 137 and 222 nM, resp., but exerted good selectivity over other bromodomains.  Cellular activity of the inhibitor was demonstrated using FRAP assays as well as cell viability data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHwwYtxVWed7Vg90H21EOLACvtfcHk0lj9nmsdQNFRFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotlCiurg%253D&md5=9464791bb52ab3b14cd20bcdd4f407d4</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00458&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00458%26sid%3Dliteratum%253Aachs%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DMonteiro%26aufirst%3DO.%26aulast%3DMeier%26aufirst%3DJ.%26aulast%3DGamble%26aufirst%3DV.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DNunez-Alonso%26aufirst%3DG.%2BA.%26aulast%3DHaendler%26aufirst%3DB.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520a%2520Chemical%2520Tool%2520Inhibitor%2520Targeting%2520the%2520Bromodomains%2520of%2520TRIM24%2520and%2520BRPF%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1642%26epage%3D1647%26doi%3D10.1021%2Facs.jmedchem.5b00458" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="note"><p class="first last">CBP HTRF IC50 of published CBP inhibitors reported here as measured by Genentech assays.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><div class="note"><p class="first last">For a similar approach for the development of BET inhibitors, see:</p></div><span class="NLM_contrib-group">Sharp, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garnier, J.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, D. C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burns, C. J.</span><span> </span><span class="NLM_article-title">Evaluation of Functional Groups as Acetyl-Lysine Mimetics for BET Bromodomain Inhibition</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1834</span><span class="NLM_x">–</span> <span class="NLM_lpage">1842</span><span class="refDoi"> DOI: 10.1039/C4MD00182F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=10.1039%2FC4MD00182F" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1834-1842&author=P.+P.+Sharpauthor=J.-M.+Garnierauthor=D.+C.+S.+Huangauthor=C.+J.+Burns&title=Evaluation+of+Functional+Groups+as+Acetyl-Lysine+Mimetics+for+BET+Bromodomain+Inhibition&doi=10.1039%2FC4MD00182F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1039%2FC4MD00182F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00182F%26sid%3Dliteratum%253Aachs%26aulast%3DSharp%26aufirst%3DP.%2BP.%26aulast%3DGarnier%26aufirst%3DJ.-M.%26aulast%3DHuang%26aufirst%3DD.%2BC.%2BS.%26aulast%3DBurns%26aufirst%3DC.%2BJ.%26atitle%3DEvaluation%2520of%2520Functional%2520Groups%2520as%2520Acetyl-Lysine%2520Mimetics%2520for%2520BET%2520Bromodomain%2520Inhibition%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D1834%26epage%3D1842%26doi%3D10.1039%2FC4MD00182F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="note"><p class="first last">We found that high potencies in the biochemical assays were required for reasonable potencies to be detected in the MYC assay. Less biochemically potent inhibitors displayed a large cell-shift.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span>BROMOscan<sup>®</sup> recombinant protein binding assay, DiscoveRx Inc. <a href="http://www.discoverx.com" class="extLink">www.discoverx.com</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=BROMOscan%C2%AE+recombinant+protein+binding+assay%2C+DiscoveRx+Inc.+www.discoverx.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Crawford, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsui, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flynn, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Côté, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beresini, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burdick, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cummings, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dakin, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duplessis, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Good, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hewitt, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, H.-R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayaram, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiefer, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nasveschuk, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pardo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poy, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romero, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zawadzke, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albrecht, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magnuson, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellon, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cochran, A. G.</span><span> </span><span class="NLM_article-title">Diving Into the Water: Inducible Binding Conformations for BRD4, TAF1(2), BRD9, and CECR2 Bromodomains</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">5391</span><span class="NLM_x">–</span> <span class="NLM_lpage">5402</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00264</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00264" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;key=1%3ACAS%3A528%3ADC%252BC28XosFSnsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5391-5402&author=T.+D.+Crawfordauthor=V.+Tsuiauthor=E.+M.+Flynnauthor=S.+Wangauthor=A.+M.+Taylorauthor=A.+C%C3%B4t%C3%A9author=J.+E.+Audiaauthor=M.+H.+Beresiniauthor=D.+J.+Burdickauthor=R.+T.+Cummingsauthor=L.+A.+Dakinauthor=M.+Duplessisauthor=A.+C.+Goodauthor=M.+C.+Hewittauthor=H.-R.+Huangauthor=H.+Jayaramauthor=J.+R.+Kieferauthor=Y.+Jiangauthor=J.+M.+Murrayauthor=C.+G.+Nasveschukauthor=E.+Pardoauthor=F.+Poyauthor=F.+A.+Romeroauthor=Y.+Tangauthor=J.+Wangauthor=Z.+Xuauthor=L.+E.+Zawadzkeauthor=X.+Zhuauthor=B.+K.+Albrechtauthor=S.+R.+Magnusonauthor=S.+F.+Bellonauthor=A.+G.+Cochran&title=Diving+Into+the+Water%3A+Inducible+Binding+Conformations+for+BRD4%2C+TAF1%282%29%2C+BRD9%2C+and+CECR2+Bromodomains&doi=10.1021%2Facs.jmedchem.6b00264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Diving into the Water: Inducible Binding Conformations for BRD4, TAF1(2), BRD9, and CECR2 Bromodomains</span></div><div class="casAuthors">Crawford, Terry D.; Tsui, Vickie; Flynn, E. Megan; Wang, Shumei; Taylor, Alexander M.; Cote, Alexandre; Audia, James E.; Beresini, Maureen H.; Burdick, Daniel J.; Cummings, Richard; Dakin, Les A.; Duplessis, Martin; Good, Andrew C.; Hewitt, Michael C.; Huang, Hon-Ren; Jayaram, Hariharan; Kiefer, James R.; Jiang, Ying; Murray, Jeremy; Nasveschuk, Christopher G.; Pardo, Eneida; Poy, Florence; Romero, F. Anthony; Tang, Yong; Wang, Jian; Xu, Zhaowu; Zawadzke, Laura E.; Zhu, Xiaoyu; Albrecht, Brian K.; Magnuson, Steven R.; Bellon, Steve; Cochran, Andrea G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5391-5402</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The biol. role played by non-BET bromodomains remains poorly understood, and it is therefore imperative to identify potent and highly selective inhibitors to effectively explore the biol. of individual bromodomain proteins.  A ligand-efficient nonselective bromodomain inhibitor was identified from a 6-Me pyrrolopyridone fragment.  Small hydrophobic substituents replacing the N-Me group were designed directing toward the conserved bromodomain water pocket, and two distinct binding conformations were then obsd.  The substituents either directly displaced and rearranged the conserved solvent network, as in BRD4(1) and TAF1(2), or induced a narrow hydrophobic channel adjacent to the lipophilic shelf, as in BRD9 and CECR2.  The preference of distinct substituents for individual bromodomains provided selectivity handles useful for future lead optimization efforts for selective BRD9, CECR2, and TAF1(2) inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTLnuVCIKdHbVg90H21EOLACvtfcHk0liCxG4baqxE2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosFSnsbs%253D&md5=bf92e94c5fd3ee36bd8441294ca85185</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00264%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3DBurdick%26aufirst%3DD.%2BJ.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DDuplessis%26aufirst%3DM.%26aulast%3DGood%26aufirst%3DA.%2BC.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DHuang%26aufirst%3DH.-R.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DMurray%26aufirst%3DJ.%2BM.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZawadzke%26aufirst%3DL.%2BE.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DMagnuson%26aufirst%3DS.%2BR.%26aulast%3DBellon%26aufirst%3DS.%2BF.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26atitle%3DDiving%2520Into%2520the%2520Water%253A%2520Inducible%2520Binding%2520Conformations%2520for%2520BRD4%252C%2520TAF1%25282%2529%252C%2520BRD9%252C%2520and%2520CECR2%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5391%26epage%3D5402%26doi%3D10.1021%2Facs.jmedchem.6b00264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W0E" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W0E','PDB','5W0E'); return false;">PDB: 5W0E</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6AXQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6AXQ','PDB','6AXQ'); return false;">PDB: 6AXQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6AY3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6AY3','PDB','6AY3'); return false;">PDB: 6AY3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6AY5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6AY5','PDB','6AY5'); return false;">PDB: 6AY5</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i94"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01372">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_28846"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b01372">10.1021/acs.jmedchem.7b01372</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Full experimental details and characterization for reported compounds, BROMOscan data for <b>27</b> and <b>29</b>, and crystallography methods for <b>3</b>-CBP, <b>6</b>-CBP, <b>16</b>-CBP, and <b>17</b>-CBP (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01372/suppl_file/jm7b01372_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01372/suppl_file/jm7b01372_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01372/suppl_file/jm7b01372_si_001.pdf">jm7b01372_si_001.pdf (332.5 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01372/suppl_file/jm7b01372_si_002.csv">jm7b01372_si_002.csv (2.54 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB codes for the structures of the CBP bromodomain in complex with <b>3</b>, <b>6</b>, <b>16</b>, and <b>17</b> are <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W0E">5W0E</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6AXQ">6AXQ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6AY3">6AY3</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6AY5">6AY5</a>, respectively.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b01372&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b01372%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-24%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b01372" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                9MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799281579f1247f","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
