
<html lang="en"     class="pb-page"  data-request-id="814b0def-f69b-4f49-afc9-709ad8768d2f"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2020.63.issue-23;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;article:article:10.1021/acs.jmedchem.0c01245;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Potent, Highly Selective, and In Vivo Efficacious, Allosteric MALT1 Inhibitors by Iterative Scaffold Morphing" /></meta><meta name="dc.Creator" content="Carole  Pissot Soldermann" /></meta><meta name="dc.Creator" content="Oliver  Simic" /></meta><meta name="dc.Creator" content="Martin  Renatus" /></meta><meta name="dc.Creator" content="Paulus  Erbel" /></meta><meta name="dc.Creator" content="Samu  Melkko" /></meta><meta name="dc.Creator" content="Markus  Wartmann" /></meta><meta name="dc.Creator" content="Marc  Bigaud" /></meta><meta name="dc.Creator" content="Andreas  Weiss" /></meta><meta name="dc.Creator" content="Paul  McSheehy" /></meta><meta name="dc.Creator" content="Ralf  Endres" /></meta><meta name="dc.Creator" content="Paulo  Santos" /></meta><meta name="dc.Creator" content="Jutta  Blank" /></meta><meta name="dc.Creator" content="Ansgar  Schuffenhauer" /></meta><meta name="dc.Creator" content="Guido  Bold" /></meta><meta name="dc.Creator" content="Nicole  Buschmann" /></meta><meta name="dc.Creator" content="Thomas  Zoller" /></meta><meta name="dc.Creator" content="Eva  Altmann" /></meta><meta name="dc.Creator" content="Paul W.  Manley" /></meta><meta name="dc.Creator" content="Ina  Dix" /></meta><meta name="dc.Creator" content="Elisabeth  Buchdunger" /></meta><meta name="dc.Creator" content="Julien  Scesa" /></meta><meta name="dc.Creator" content="Jean  Quancard" /></meta><meta name="dc.Creator" content="Achim  Schlapbach" /></meta><meta name="dc.Creator" content="Frédéric  Bornancin" /></meta><meta name="dc.Creator" content="Thomas  Radimerski" /></meta><meta name="dc.Creator" content="Catherine H.  Régnier" /></meta><meta name="dc.Description" content="MALT1 plays a central role in immune cell activation by transducing NF-κB signaling, and its proteolytic activity represents a key node for therapeutic intervention. Two cycles of scaffold morphing..." /></meta><meta name="Description" content="MALT1 plays a central role in immune cell activation by transducing NF-κB signaling, and its proteolytic activity represents a key node for therapeutic intervention. Two cycles of scaffold morphing..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 30, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01245" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01245" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01245" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01245" /></link>
        
    
    

<title>Discovery of Potent, Highly Selective, and In Vivo Efficacious, Allosteric MALT1 Inhibitors by Iterative Scaffold Morphing | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01245" /></meta><meta property="og:title" content="Discovery of Potent, Highly Selective, and In Vivo Efficacious, Allosteric MALT1 Inhibitors by Iterative Scaffold Morphing" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0016.jpeg" /></meta><meta property="og:description" content="MALT1 plays a central role in immune cell activation by transducing NF-κB signaling, and its proteolytic activity represents a key node for therapeutic intervention. Two cycles of scaffold morphing of a high-throughput biochemical screening hit resulted in the discovery of MLT-231, which enabled the successful pharmacological validation of MALT1 allosteric inhibition in preclinical models of humoral immune responses and B-cell lymphomas. Herein, we report the structural activity relationships (SARs) and analysis of the physicochemical properties of a pyrazolopyrimidine-derived compound series. In human T-cells and B-cell lymphoma lines, MLT-231 potently and selectively inhibits the proteolytic activity of MALT1 in NF-κB-dependent assays. Both in vitro and in vivo profiling of MLT-231 support further optimization of this in vivo tool compound toward preclinical characterization." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01245"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01245">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01245&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01245&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01245&amp;href=/doi/10.1021/acs.jmedchem.0c01245" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 23</span><span class="cit-fg-pageRange">, 14576-14593</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/23" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01342" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01246" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Potent, Highly Selective, and <i>In Vivo</i> Efficacious, Allosteric MALT1 Inhibitors by Iterative Scaffold Morphing</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Carole Pissot Soldermann</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Carole Pissot Soldermann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#0c6f6d7e636069227c657f7f63784c62637a6d7e78657f226f6361"><span class="__cf_email__" data-cfemail="fe9d9f8c91929bd08e978d8d918abe9091889f8c8a978dd09d9193">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Carole++Pissot+Soldermann">Carole Pissot Soldermann</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7497-3839" title="Orcid link">http://orcid.org/0000-0002-7497-3839</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Oliver Simic</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Oliver Simic</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#2b4447425d4e590558424642486b45445d4a595f425805484446"><span class="__cf_email__" data-cfemail="e28d8e8b948790cc918b8f8b81a28c8d948390968b91cc818d8f">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Oliver++Simic">Oliver Simic</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Martin Renatus</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Martin Renatus</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Martin++Renatus">Martin Renatus</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paulus Erbel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paulus Erbel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paulus++Erbel">Paulus Erbel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Samu Melkko</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Samu Melkko</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Samu++Melkko">Samu Melkko</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Markus Wartmann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Markus Wartmann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Markus++Wartmann">Markus Wartmann</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marc Bigaud</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marc Bigaud</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marc++Bigaud">Marc Bigaud</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andreas Weiss</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andreas Weiss</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andreas++Weiss">Andreas Weiss</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paul McSheehy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paul McSheehy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Basilea Pharmaceutica, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paul++McSheehy">Paul McSheehy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ralf Endres</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ralf Endres</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div><div class="loa-info-affiliations-info">Basilea Pharmaceutica, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ralf++Endres">Ralf Endres</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paulo Santos</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paulo Santos</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div><div class="loa-info-affiliations-info">Basilea Pharmaceutica, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paulo++Santos">Paulo Santos</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jutta Blank</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jutta Blank</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jutta++Blank">Jutta Blank</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ansgar Schuffenhauer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ansgar Schuffenhauer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ansgar++Schuffenhauer">Ansgar Schuffenhauer</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6385-0414" title="Orcid link">http://orcid.org/0000-0001-6385-0414</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Guido Bold</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guido Bold</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guido++Bold">Guido Bold</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nicole Buschmann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicole Buschmann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicole++Buschmann">Nicole Buschmann</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Thomas Zoller</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thomas Zoller</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thomas++Zoller">Thomas Zoller</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Eva Altmann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Eva Altmann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Eva++Altmann">Eva Altmann</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2749-6347" title="Orcid link">http://orcid.org/0000-0002-2749-6347</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paul W. Manley</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paul W. Manley</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paul+W.++Manley">Paul W. Manley</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ina Dix</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ina Dix</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ina++Dix">Ina Dix</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Elisabeth Buchdunger</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elisabeth Buchdunger</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elisabeth++Buchdunger">Elisabeth Buchdunger</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Julien Scesa</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Julien Scesa</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Julien++Scesa">Julien Scesa</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jean Quancard</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jean Quancard</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jean++Quancard">Jean Quancard</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Achim Schlapbach</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Achim Schlapbach</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Achim++Schlapbach">Achim Schlapbach</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Frédéric Bornancin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Frédéric Bornancin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fr%C3%A9d%C3%A9ric++Bornancin">Frédéric Bornancin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Thomas Radimerski</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thomas Radimerski</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Basilea Pharmaceutica, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thomas++Radimerski">Thomas Radimerski</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Catherine H. Régnier</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Catherine H. Régnier</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#046765706c61766d6a612a7661636a6d6176446a6b726576706d772a676b69"><span class="__cf_email__" data-cfemail="f49795809c91869d9a91da8691939a9d9186b49a9b829586809d87da979b99">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Catherine+H.++R%C3%A9gnier">Catherine H. Régnier</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01245&amp;href=/doi/10.1021%2Facs.jmedchem.0c01245" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 23</span><span class="cit-pageRange">, 14576–14593</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 30, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>17 July 2020</li><li><span class="item_label"><b>Published</b> online</span>30 November 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 December 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01245" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01245</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D14576%26pageCount%3D18%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DCarole%2BPissot%2BSoldermann%252C%2BOliver%2BSimic%252C%2BMartin%2BRenatus%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D23%26contentID%3Dacs.jmedchem.0c01245%26title%3DDiscovery%2Bof%2BPotent%252C%2BHighly%2BSelective%252C%2Band%2BIn%2BVivo%2BEfficacious%252C%2BAllosteric%2BMALT1%2BInhibitors%2Bby%2BIterative%2BScaffold%2BMorphing%26numPages%3D18%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D14593%26publicationDate%3DDecember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01245"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2039</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">6</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01245" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Potent, Highly Selective, and In Vivo Efficacious, Allosteric MALT1 Inhibitors by Iterative Scaffold Morphing&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Carole&quot;,&quot;last_name&quot;:&quot;Pissot Soldermann&quot;},{&quot;first_name&quot;:&quot;Oliver&quot;,&quot;last_name&quot;:&quot;Simic&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;Renatus&quot;},{&quot;first_name&quot;:&quot;Paulus&quot;,&quot;last_name&quot;:&quot;Erbel&quot;},{&quot;first_name&quot;:&quot;Samu&quot;,&quot;last_name&quot;:&quot;Melkko&quot;},{&quot;first_name&quot;:&quot;Markus&quot;,&quot;last_name&quot;:&quot;Wartmann&quot;},{&quot;first_name&quot;:&quot;Marc&quot;,&quot;last_name&quot;:&quot;Bigaud&quot;},{&quot;first_name&quot;:&quot;Andreas&quot;,&quot;last_name&quot;:&quot;Weiss&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;McSheehy&quot;},{&quot;first_name&quot;:&quot;Ralf&quot;,&quot;last_name&quot;:&quot;Endres&quot;},{&quot;first_name&quot;:&quot;Paulo&quot;,&quot;last_name&quot;:&quot;Santos&quot;},{&quot;first_name&quot;:&quot;Jutta&quot;,&quot;last_name&quot;:&quot;Blank&quot;},{&quot;first_name&quot;:&quot;Ansgar&quot;,&quot;last_name&quot;:&quot;Schuffenhauer&quot;},{&quot;first_name&quot;:&quot;Guido&quot;,&quot;last_name&quot;:&quot;Bold&quot;},{&quot;first_name&quot;:&quot;Nicole&quot;,&quot;last_name&quot;:&quot;Buschmann&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;Zoller&quot;},{&quot;first_name&quot;:&quot;Eva&quot;,&quot;last_name&quot;:&quot;Altmann&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;W. Manley&quot;},{&quot;first_name&quot;:&quot;Ina&quot;,&quot;last_name&quot;:&quot;Dix&quot;},{&quot;first_name&quot;:&quot;Elisabeth&quot;,&quot;last_name&quot;:&quot;Buchdunger&quot;},{&quot;first_name&quot;:&quot;Julien&quot;,&quot;last_name&quot;:&quot;Scesa&quot;},{&quot;first_name&quot;:&quot;Jean&quot;,&quot;last_name&quot;:&quot;Quancard&quot;},{&quot;first_name&quot;:&quot;Achim&quot;,&quot;last_name&quot;:&quot;Schlapbach&quot;},{&quot;first_name&quot;:&quot;Frédéric&quot;,&quot;last_name&quot;:&quot;Bornancin&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;Radimerski&quot;},{&quot;first_name&quot;:&quot;Catherine&quot;,&quot;last_name&quot;:&quot;H. Régnier&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;30&quot;,&quot;issue&quot;:&quot;23&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;14576-14593&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01245&quot;},&quot;abstract&quot;:&quot;MALT1 plays a central role in immune cell activation by transducing NF-κB signaling, and its proteolytic activity represents a key node for therapeutic intervention. Two cycles of scaffold morphing of a high-throughput biochemical screening hit resulted in the discovery of MLT-231, which enabled the successful pharmacological validation of MALT1 allosteric inhibition in preclinical models of humoral immune responses and B-cell lymphomas. Herein, we report the structural activity relationships (SARs) and analysis of the physicochemical properties of a pyrazolopyrimidine-derived compound series. In human T-cells and B-cell lymphoma lines, MLT-231 potently and selectively inhibits the proteolytic activity of MALT1 in NF-κB-dependent assays. Both in vitro and in vivo profiling of MLT-231 support further optimization of this in vivo tool compound toward preclinical characterization.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01245&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01245" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01245&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01245" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01245&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01245" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01245&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01245&amp;href=/doi/10.1021/acs.jmedchem.0c01245" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01245" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01245" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01245%26sid%3Dliteratum%253Aachs%26pmid%3D33252239%26genre%3Darticle%26aulast%3DPissot%2BSoldermann%26date%3D2020%26atitle%3DDiscovery%2Bof%2BPotent%252C%2BHighly%2BSelective%252C%2Band%2BIn%2BVivo%2BEfficacious%252C%2BAllosteric%2BMALT1%2BInhibitors%2Bby%2BIterative%2BScaffold%2BMorphing%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D23%26spage%3D14576%26epage%3D14593%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292271" title="Urea">Urea</a>,</li><li><a href="/action/doSearch?ConceptID=290795" title="Quinolines">Quinolines</a>,</li><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=291454" title="Solubility">Solubility</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/23" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/jmcmar.2020.63.issue-23/20201210/jmcmar.2020.63.issue-23.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01245&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">MALT1 plays a central role in immune cell activation by transducing NF-κB signaling, and its proteolytic activity represents a key node for therapeutic intervention. Two cycles of scaffold morphing of a high-throughput biochemical screening hit resulted in the discovery of MLT-231, which enabled the successful pharmacological validation of MALT1 allosteric inhibition in preclinical models of humoral immune responses and B-cell lymphomas. Herein, we report the structural activity relationships (SARs) and analysis of the physicochemical properties of a pyrazolopyrimidine-derived compound series. In human T-cells and B-cell lymphoma lines, MLT-231 potently and selectively inhibits the proteolytic activity of MALT1 in NF-κB-dependent assays. Both <i>in vitro</i> and <i>in vivo</i> profiling of MLT-231 support further optimization of this <i>in vivo</i> tool compound toward preclinical characterization.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37665" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37665" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">MALT1, a signaling protein encoded by the mucosa-associated lymphoid tissue lymphoma translocated gene 1, was initially characterized as a key player in nuclear factor-kappa B (NF-κB) activation following T-cell receptor (TCR) or B-cell receptor (BCR) engagement and is involved in regulating a broad range of immune and inflammatory pathways. Prototypically, following receptor activation, MALT1 is engaged together with the caspase recruitment domain family member 11 (CARD11) and the B-cell lymphoma/leukemia 10 (BCL10) proteins to form the CARD11/BCL10/MALT1 (CBM) complex, that triggers IκB kinases (IKK) activation. Structurally related to caspases, MALT1 possesses a unique cysteine protease domain and is classified as a paracaspase.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The role of MALT1 in NF-κB signaling was established well before the discovery in 2008 that the protein possessed bona fide protease activity, first toward BCL10 and A20 (TNFAIP3, a negative regulator of NF-κB) and later toward additional substrates such as CYLD, RelB, Regnase-1, Roquins, HOIL1, and MALT1 itself.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a></div><div class="NLM_p">The development, survival, and proliferation of B-cells essentially depend on canonical and noncanonical NF-κB signaling pathways. Interestingly, MALT1 was first identified in MALT-type B-cell lymphomas, where the t(11;18)(q21;q21) chromosome translocation results in the chimeric cIAP2-MALT1 oncoprotein that drives constitutive NF-κB activation.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Different types of B-cell malignancies are also characterized by constitutive NF-κB signaling, in particular, the activated B-cell-like subtype of diffuse large B-cell lymphoma (ABC-DLBCL), where the sequencing of patient biopsies frequently reveals mutations in NF-κB signaling proteins, such as in the BCR-associated proteins CD79A and CD79B. Downstream of the BCR, oncogenic NF-κB signaling can result from gain-of-function mutations in the coiled-coil domain of the CBM component CARD11, which lead to MALT1 activation independently of upstream phosphorylation events.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><div class="NLM_p last">The transcription factor NF-κB is a master regulator of multiple aspects of innate and adaptive immune functions in normal physiological conditions, and its tight control is mandatory to prevent it from playing a role in pathogenic processes in chronic inflammation and tissue damages of various autoimmune diseases. As a new therapeutic strategy based on NF-κB inhibition, MALT1 represents an attractive protease target for the discovery of small-molecule inhibitors that could dampen NF-κB signaling in immune cells, including B-cell lymphomas. We and others have previously disclosed allosteric MALT1 inhibitors<a onclick="showRef(event, 'ref6 ref7 ref8 ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8 ref9 ref10 ref11">(6−11)</a> and their use as tools to explore the role of MALT1 biology. Here and in the accompanying publication by Quancard <i>et al.</i>, we report the identification of pyrazolo pyrimidines as a new class of allosteric MALT1 inhibitors and their optimization for <i>in vivo</i> activity. We describe herein the lead compound (MLT-231) having nanomolar potency in biochemical and cellular MALT1 protease reporter assays. In addition, the pyrazolopyrimidines are highly selective for MALT1 and demonstrate differential growth inhibition of CARD11 mutant ABC-DLBCL cells compared to Germinal Center B cell-like (GCB) subtype of DLBCL cells. Furthermore, <i>in vivo</i> efficacy was achieved within this series in CD79 mutant ABC-DLBCL xenograft models.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45725" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45725" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Primary SAR of the Chromane Ureas, Pharmacophore Mapping</h3><div class="NLM_p">Our search for novel MALT1 inhibitors started with a biochemical high-throughput screening (HTS) of a library containing approximately one million compounds by monitoring the protease activity using a reporter-labeled peptide substrate.<a onclick="showRef(event, 'ref6 ref10'); return false;" href="javascript:void(0);" class="ref ref6 ref10">(6,10)</a> From this screening, the racemic chromane urea <b>1</b> was identified as a singleton hit. Chiral separation of the two enantiomers showed that the (<i>R</i>)-enantiomer inhibited the protease activity of MALT1, whereas the (<i>S</i>)-enantiomer was inactive (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).</div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Chemical Structure and Biochemical Potency of HTS Hit 1</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01245&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The eutomeric compound <b>2</b> was evaluated for cellular activity and inhibited MALT1 cleavage of the BCL10 substrate with an IC<sub>50</sub> value of 2.2 μM. Given the enantio-differentiation as strong evidence for specific binding, as well as the substantial biochemical and cellular activity, compound <b>2</b> was employed as a starting-point for further optimization.</div><div class="NLM_p">Subsequently, we explored the SAR of the scaffold using racemic chromane urea analogues (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). This campaign showed that substituents at the C6-position afforded potency improvement in the biochemical assay, whereas substitution at C7 and C8 led to a significant drop in activity (not shown). Attaching a halogen atom to C6 gave a fourfold increase in potency, whereas a nitrile resulted in little change (compounds <b>4</b>, <b>6</b>, and <b>5</b><i>vs</i><b>1</b>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR around Chromane (Substitution and Modification)</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0012.gif" alt="" id="GRAPHIC-d7e490-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">R<sub>1</sub></th><th class="colsep0 rowsep0" align="center">A</th><th class="colsep0 rowsep0" align="center">MALT1 biochemical IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">MALT1 cellular BCL10 cleavage IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">0.245</td><td class="colsep0 rowsep0" align="left">2.30<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">0.050</td><td class="colsep0 rowsep0" align="left">n.d.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">CN</td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">0.150</td><td class="colsep0 rowsep0" align="left">1.92</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">0.057</td><td class="colsep0 rowsep0" align="left">1.25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">n.d.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">S</td><td class="colsep0 rowsep0" align="left">0.145</td><td class="colsep0 rowsep0" align="left">4.27</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values are the average of at least two separate determinations.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Cellular activity of eutomeric compound <b>2</b>.</p></div></div></div><div class="NLM_p">Next, we evaluated the role of the chromane-oxygen atom as a potential hydrogen-bond acceptor interacting with MALT1 protein. The corresponding thiochromane, having a weaker hydrogen-bond acceptor character, showed decreased potency (compound <b>8</b><i>vs</i><b>6</b>), whereas the tetrahydronaphthalene analogue (compound <b>1</b><i>vs</i><b>7</b>) was inactive. Based upon these results, we concluded that oxygen was an important pharmacophore element.</div><div class="NLM_p">The role of the central urea motif for binding to MALT1 protein was assessed by <i>N</i>-alkylation or removal of the NH groups (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Replacement of the chromane-linked NH by a methylene in compound <b>9</b>, although tolerated, was fourfold less potent. Replacing the distal urea-NH by a methylene group was not tolerated. <i>N</i>-methylation of either urea nitrogen was detrimental for potency (compounds <b>11</b> and <b>12</b>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SAR around the Urea and Aniline Moieties</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0013.gif" alt="" id="gr11" /></img><div></div></div><div class="NLM_p last">Finally, SAR around the aniline was explored by a high-throughput organic synthesis (HTOS) library approach. Replacing the phenyl ring with substituted heterocycles such as pyridine or adding heteroaryl substituents was tolerated (compound <b>4</b><i>vs</i><b>13 or 14</b>).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> First Iteration of Scaffold Morphing Reveals a New Exit Vector for SAR Exploration</h3><div class="NLM_p">Having developed an understanding of the requirements of the urea and chromane pharmacophore elements and having identified favorable substituents in both C6 as well as in the aniline moiety, we turned our attention toward 4-substituted quinolines. This modification recapitulated the hydrogen-bond acceptor character of the chromane oxygen and offered a readily accessible exit vector to further explore new chemical space at the C4-position. The results are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Expansion to 4-Substituted Quinolines</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0014.gif" alt="" id="gr12" /></img><div></div></div><div class="NLM_p">Consistent with the SAR finding, morphing the chromane into a quinoline was tolerated (compound <b>4</b><i>vs</i><b>15</b>). Addition of a chlorine substituent to C4 led to the first compound with submicromolar activity in the MALT1 cellular BCL10 cleavage assay (compound <b>16</b>). We then explored the SAR surrounding C4 by introducing small <i>N</i>-alkyl substituents. Introduction of an <i>N</i>-methyl tetrahydropyran led to compound <b>17</b>, which showed cellular activity. Further transfer of the SAR from the aniline portion of the chromane urea and introducing an <i>N</i>-methylmorpholine at C4 culminated in compound <b>20</b> (MLT-827), which exhibited a double-digit nanomolar activity in the MALT1 BCL10 cellular assay. This first potent tool compound was used to probe the function of MALT1 protease in lymphocytes,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> myeloid cells,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and keratinocytes.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></div><div class="NLM_p">MLT-827 was selected for further ADME characterization based upon its favorable cellular activity, permeability, and liver microsomal stability. The single-dose pharmacokinetic (PK) profile of MLT-827 was evaluated in Sprague Dawley rats following intravenous (i.v.) and oral (p.o.) administration. The results are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. <i>In Vitro</i> ADME and <i>In Vivo</i> Rat PK Parameters<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a> for MLT-827</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="2" align="center"><i>in vitro</i> ADME</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">log D (7.4)</td><td class="colsep0 rowsep0" align="char" char=".">3.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">thermodynamic solubility pH 6.8 [g/L]</td><td class="colsep0 rowsep0" align="char" char=".">0.0002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">log PAMPA [cm/s]</td><td class="colsep0 rowsep0" align="char" char=".">–4.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RLM CL<sub>int</sub> [μL·min<sup>–1</sup>·mg<sup>–1</sup>]</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">plasma protein binding (rat) (%)</td><td class="colsep0 rowsep0" align="char" char=".">97.5</td></tr></tbody></table><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="4" align="center"><i>in vivo</i> rat PK parameters<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">route (dose in mg·kg<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">i.v. (1.0)</td><td class="colsep0 rowsep0" align="left">p.o. (3.0) suspension</td><td class="colsep0 rowsep0" align="left">p.o. (1.25) solution</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CL [mL·min<sup>–1</sup>·kg<sup>–1</sup>]</td><td class="colsep0 rowsep0" align="left">31 ± 6</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>V</i><sub>SS</sub> [L·kg<sup>–1</sup>]</td><td class="colsep0 rowsep0" align="left">4.8 ± 0.3</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>t</i><sub>1/2</sub> term. [h]</td><td class="colsep0 rowsep0" align="left">2.1 ± 0.4</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC d.n.<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a> [nM h<sup>–1</sup>]</td><td class="colsep0 rowsep0" align="left">1235 ± 205</td><td class="colsep0 rowsep0" align="left">103 ± 37</td><td class="colsep0 rowsep0" align="left">520 ± 54</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">abs. oral BAV [%]</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">8 ± 3</td><td class="colsep0 rowsep0" align="left">42 ± 4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>C</i><sub>max</sub> d.n.<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a> [nM]</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">11 ± 4</td><td class="colsep0 rowsep0" align="left">71 ± 16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>T</i><sub>max</sub> [h]</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">8 ± 0</td><td class="colsep0 rowsep0" align="left">3 ± 0.8</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Male Sprague Dawley rats; each time-point from three single animals; i.v.: bolus administration 1 mg·kg<sup>–1</sup>, dissolved in <i>N</i>-methylpyrrolidone/PEG200 = 30:70, v/v, sampling 0.08, 0.25, 0.5, 1.0, 2.0, 3.0, 4.0, 6.0, 8.0, 24, and 48 h after administration; p.o.: 3 mg·kg<sup>–1</sup>, suspended in 0.5% carboxymethyl cellulose (MC) and Tween 80 in water; or 1.25 mg kg<sup>–1</sup> dissolved in Labrafilm M1944/corn oil/ethanol = 43:39:18, % v/v, sampling 0.25, 0.5, 1.0, 2.0, 3.0, 4.0, 6.0, 8.0, 24 h after administration.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">d.n. = dose normalized.</p></div></div></div><div class="NLM_p">Considering the low <i>in vitro</i> clearance (CL), the observed <i>in vivo</i> CL for MLT-827 was higher than expected, although still in a moderate range. Oral administration of a solution of MLT-827 revealed significantly higher exposure in comparison to the oral administration of a suspension. The limited oral BAV after administration of a suspension was probably because of solubility-limited absorption.</div><div class="NLM_p">A common strategy to increase water solubility at physiological pH without compromising potency is to add a solubilizing moiety containing a basic ionizable group (usually a primary or secondary amine). Such solubilizing side chains were attached either to C4 (compound <b>21</b>) or to the aminopyridine moiety (compound <b>22</b>). While the solubility of the resulting compounds was improved, the ionizable side chains led to measurable human Ether-a-go-go Related Gene (hERG) channel binding<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> with IC<sub>50</sub> values of 9 and 16 μM, respectively.</div><div class="NLM_p">Inhibiting the hERG channel could potentially lead to substantial QT interval prolongation, which is associated with fatal cardiac arrhythmias. A number of marketed drugs, for example, terfenadine or cisapride have been withdrawn because of reports of cardiac sudden death.<a onclick="showRef(event, 'ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17 ref18">(15−18)</a></div><div class="NLM_p">Based upon these findings, instead of continuing this approach to improve solubility, we focused on understanding what drove the limited solubility of the series. Aqueous solubility results from an interplay between three factors: (a) ionization, (b) solute–solvent interactions, and (c) crystal lattice energy. While these three parameters are not completely independent, one may be dominant and thus represent the major solubility barrier. We used our solubility diagnosis matrix on compound <b>17</b> to analyze which of these parameters are driving poor solubility.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> We assessed log 1/<i>S</i><sub>0</sub> – log <i>P</i> (Δ<i>SL</i>), which reflected the difference in log units between the intrinsic solubility and lipophilicity. This can be used as an alternative for <i>T</i><sub>m</sub> in the absence of crystalline material and a value greater than 2 is a sign of high crystal lattice energy. For compound <b>17</b>, the ΔSL value was determined to be 3.4 (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>), suggesting that breaking the crystal lattice should improve solubility.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Solubility Diagnosis on Quinoline Compound <b>17</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">log 1/<i>S</i><sub>6.8</sub> (measured)</td><td class="colsep0 rowsep0" align="center">log 1/<i>S</i><sub>6.8</sub> (extrapolated)</td><td class="colsep0 rowsep0" align="center">log 1/<i>S</i><sub>0</sub></td><td class="colsep0 rowsep0" align="center">FI<sub>6.8</sub></td><td class="colsep0 rowsep0" align="center"><i>e</i> log <i>P</i></td><td class="colsep0 rowsep0" align="center">ΔSL</td><td class="colsep0 rowsep0" align="center"><i>T</i><sub>m</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">compound <b>17</b></td><td class="colsep0 rowsep0" align="center">7.1</td><td class="colsep0 rowsep0" align="center">7.1</td><td class="colsep0 rowsep0" align="center">7.1</td><td class="colsep0 rowsep0" align="center">0</td><td class="colsep0 rowsep0" align="center">3.7</td><td class="colsep0 rowsep0" align="center">3.4</td><td class="colsep0 rowsep0" align="center">N/A</td></tr></tbody></table></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Second Iteration of Scaffold Morphing Leading to Pyrazolopyrimidine Urea</h3><div class="NLM_p">We studied the intermolecular interactions of quinoline compound <b>17</b> in a small-molecule X-ray crystal structure (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Several classical intermolecular water-mediated hydrogen-bonds were identified involving the central urea. Additionally, we observed a nonclassical intermolecular head-to-head interaction between the C6 nitrile and the C5 aromatic hydrogen. The flat conformation of the quinoline urea system led to a tight stacking, which also very likely contributed to the high crystal lattice energy.</div><figure id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Small-molecule X-ray and crystal packing of compound <b>17</b> (mono hydrate). Classical intermolecular hydrogen-bond interactions between the urea nitrogens and water are highlighted in yellow. Additional intermolecular nonclassical CH–N contacts are depicted in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01245&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We concluded that a combination of these three factors was the main driver for the poor solubility of the series, and consequently, we embarked on additional scaffold morphing in an attempt to improve the overall physicochemical properties. Morphing from a 6,6- to 5,6-heterocycle, such as a pyrazolopyrimidine, should maintain an exit vector to the same region as the C4 of the quinoline, which was required for high potency. In addition, morphing from a quinoline to the pyrazolopyrimidine should disrupt the observed nonclassical head-to-head C6 nitrile/C5 aromatic hydrogen interaction.</div><div class="NLM_p">Compound <b>23</b> (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>), where a 2-bromopyrazolo[1,5-<i>a</i>]pyrimidine replaced the 6-cyanoquinoline, exhibited low nanomolar biochemical activity and increased high-throughput equilibrium solubility. Guided by these results, a small selection of substituted morpholines was prepared to investigate their impact on activity and solubility. All compounds showed excellent biochemical inhibition of MALT1, but solubility differed markedly. Introducing the (3<i>S</i>,5<i>S</i>)-3,5-dimethylmorpholine in combination with 2-(trifluoromethyl)pyridin-4-amine resulted in compound <b>27</b> (MLT-231), which showed a biochemical MALT1 IC<sub>50</sub> of 9 nM and high solubility.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. SAR of Selected Pyrazolopyrimidines</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0015.gif" alt="" id="gr13" /></img><div></div></div><div class="NLM_p">To rationalize the observed improvements in solubility, we generated a small-molecule X-ray structure of compound <b>23</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Small-molecule X-ray of compound <b>23</b> overlayed with MLT-827. (A) ORTEP view of compound <b>23</b>. (B) Small-molecule X-ray structures overlays of compound <b>23</b> (cyan) and <b>MLT-827</b> (light yellow) from two orientations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01245&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In contrast to the quinoline ureas, having a completely planar bis-aryl urea core, <b>23</b> revealed some torsion of the morpholine portion, which induced a disruption of planarity of the urea and an increase of the sp3-character. Although some direct interactions of the urea NH-donors and C═O acceptor were observed, overall <b>23</b> showed fewer interactions compared to MLT-827 within the crystal lattice. From this, we concluded that the observed improvement in solubility for <b>23</b> and MLT-231 is likely due to disruption of the molecular planarity.</div><div class="NLM_p">While the potency optimization of the MALT1 inhibitors shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> and <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a> was performed in the absence of any structural information, we obtained at a later stage of the project two tool compounds <b>TC1</b> and <b>TC2</b> from the chromane and pyrazolopyrimidine subseries, which led to a cocrystal structure of dimeric MALT1Casp-Ig3, respectively. Both structures allowed to rationalize most of the observed structure–activity relationship in the corresponding subseries. As described previously,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a><b>TC1</b> and <b>TC2</b> both occupy the “W580 tryptophane pocket” at the interface between the caspase- and the Ig3-domain. This pocket is predominantly surrounded by hydrophobic residues, which explains why compound <b>TC1</b> and <b>TC2</b> are mostly engaged in van-der-Waals interactions with the protein, while only the NH of the central urea moiety is undergoing a polar interaction with the Glu397 COOH. <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A shows that the beneficial C6-chromane halogen interacts with the backbone carbonyl of Arg576. The 4-(aminomethyl)-1<i>H</i>-pyrazol-1-yl) moiety attached to the aniline extends toward the solvent, which explains the tolerance of larger groups in this position. Interestingly, the oxygen-acceptor in the chromane core or the nitrogen-acceptor of the pyrazolopyrimidine core are not engaged in polar interaction with the protein but well tolerated at the bottom of the tryptophane pocket. Overlaying the two structures derived from <b>TC1</b> and <b>TC2</b> illustrates the flexibility of the protein (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). Exploration of the new exit vector in the C4-position of the pyrazolopyrimidine series induces conformational changes in the interdomain helix, allowing opening of a new lipophilic pocket which accommodates the introduced C4-morpholine substituents and likely leads to the increased potency observed for compounds <b>23</b>–<b>27</b>.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cocrystal structures of MALT1 in complex with urea-based inhibitors (A) Cocrystal structure of MALT1 in complex with chromane urea-based inhibitors <b>TC1</b> (orange), 2.3 Å, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AK0">7AK0</a>. (B) Cocrystal structure of MALT1 in complex with pyrazolopyrimidine urea-based inhibitors <b>TC2</b> (green), 2.5 Å, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AK1">7AK1</a>. (C) Overlay of the structures from (A,B) illustrating conformational changes in the inter-domain helix.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01245&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Cellular Profiling of MLT-231</h3><div class="NLM_p">Based upon its overall profile MLT-231 was selected for further characterization and <i>in vivo</i> testing. MLT-231 potently inhibited the enzymatic activity of MALT1 <i>in vitro</i> displaying an IC<sub>50</sub> of 9 nM (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). In an internal assay panel,<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> MLT-231 exhibited high MALT1 selectivity over other proteases (data not shown), which can be attributed to the allosteric mode of inhibition.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Pharmacological Profiling of Compound MLT-231<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">assay</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> [μM] (SD, <i>n</i>)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MALT1 biochemical assay</td><td class="colsep0 rowsep0" align="left">0.009 (±0.005, <i>n</i> = 14)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NF-κB RGA HEK293 cells</td><td class="colsep0 rowsep0" align="left">0.14 (±0.046, <i>n</i> = 4)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IL-2 RGA Jurkat cells</td><td class="colsep0 rowsep0" align="left">0.18 (±0.07, <i>n</i> = 4)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BCL10 cleavage OCI-Ly3 cells</td><td class="colsep0 rowsep0" align="left">0.16 (±0.028, <i>n</i> = 2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">OCI-Ly3/BJAB cell proliferation</td><td class="colsep0 rowsep0" align="left">2 (±0.2)/>10 (<i>n</i> = 2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">huMLR/mMLR/mBM</td><td class="colsep0 rowsep0" align="left">0.19/0.6/>10</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">The pharmacology profile of MLT-231 was obtained using purified recombinant MALT1 protein<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> or cell-based read-outs (see <a class="ref internalNav" href="#sec4" aria-label="Experimental section">Experimental section</a>). Data are presented as mean ± SD of the corresponding <i>n</i> values. RGA: reporter gene assay, MLR: mixed lymphocyte reaction, BM: bone marrow cell proliferation.</p></div></div></div><div class="NLM_p">The biochemical activity translated into the potent cellular activity in two mechanistic models based on oncogenic cIAP2-MALT1-dependent NF-κB reporter gene assay (RGA) in HEK293 cells (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01245/suppl_file/jm0c01245_si_001.pdf" class="ext-link">Figure S1</a>) and on IL-2 RGA in Jurkat T cells stimulated with PMA and an anti-CD28 monoclonal antibody (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). In both murine or human primary immune cells, the mixed lymphocyte reaction (MLR) was inhibited by MLT-231, while no cytotoxic effects were observed on bone marrow proliferation. In the ABC-DLBCL cell line OCI-Ly3 harboring an oncogenic mutation in CARD11 leading to constitutive CBM formation, MALT1 activation, and BCL10 cleavage, MLT-231 specifically prevented endogenous BCL10 cleavage (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>).<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Correspondingly, MLT-231 also inhibited the proliferation of OCI-Ly3 cells in suspension as well as their growth in a soft-agar assay with IC<sub>50</sub> values in the low micromolar range, whereas the compound had no impact on the proliferation of BJAB cells, a GCB-type DLBCL cell line which is independent of NF-κB for growth (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A).</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound MLT-231 inhibits cellular proliferation and cleavage of MALT1 substrates in ABC-DLBCL. (A) Concentration-dependent inhibition of OCI-Ly3 cell growth in soft-agar by MLT-231. The same compound has no impact on BJAB cells from the GC-DLBCL type. (B) Western blots of the indicated proteins in OCI-Ly3 cells treated with increasing concentrations of MLT231 or DMSO for 24 h. The pharmacological inhibition of MALT1 protease activity leads to accumulation of the uncleaved form of its substrates CYLD (upper band), BCL10, and RELB while expression of the NF-κB target gene IRF4 is suppressed.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01245&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">At the molecular level, pharmacological inhibition of MALT1 protease activity by MLT-231 led to the accumulation of the uncleaved form of the MALT1 substrates CYLD, BCL10, and RELB in a concentration-dependent manner. In parallel, the NF-κB target gene IRF4 was repressed upon MALT1 inhibition, confirming the key role of MALT1 in driving NF-κB activity in the ABC-DLBL cell line OCI-Ly3 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B).</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> DMPK Assessment of MLT-231</h3><div class="NLM_p">MLT-231 showed high permeability, moderate thermodynamic solubility, and low-to-moderate clearance in rat and mouse liver microsomes (<a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a>).</div><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. <i>In Vitro</i> ADME for MLT-231</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">assays</th><th class="colsep0 rowsep0" align="center">MLT-231</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">thermodynamic solubility pH 6.8 [g/L]</td><td class="colsep0 rowsep0" align="left">0.004</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Caco-2 Papp A-B [10<sup>–6</sup> cm/s]</td><td class="colsep0 rowsep0" align="left">1.47</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAMPA permeability, log <i>P</i>e [cm/s]</td><td class="colsep0 rowsep0" align="left">–4.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MLM/RLM CLint [μL·min<sup>–1</sup>·mg<sup>–1</sup>]</td><td class="colsep0 rowsep0" align="left">82/42</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mouse/rat PPB (%)</td><td class="colsep0 rowsep0" align="left">98.9/98.7</td></tr></tbody></table></div></div><div class="NLM_p">The single-dose PK profile of MLT-231 was evaluated in female BALB/c mice and male Sprague Dawley rats following either i.v. or p.o. administration. The results are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">Table <a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">9</a></a>. The PK properties of MLT-231 in mouse and rat were characterized by high bioavailability, low-to-moderate clearance, low volume of distribution in mouse, a moderate volume of distribution in rat, and a short terminal half-life.</div><div class="NLM_table-wrap" id="tbl9"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 9. PK Parameters of MLT-231 in BALB/c Mice and Sprague-Dawley Rats from Cassette PK<a class="ref internalNav" href="#t9fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center" char="±">i.v.<a class="ref internalNav" href="#t9fn2" aria-label="b">b</a></th><th class="rowsep1 colsep0" colspan="3" align="center" char="±">p.o.<a class="ref internalNav" href="#t9fn3" aria-label="c">c</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">species</th><th class="colsep0 rowsep0" align="center" char="±">CL [mL min<sup>–1</sup> kg<sup>–1</sup>]</th><th class="colsep0 rowsep0" align="center" char="±"><i>t</i><sub>1/2</sub> [h]</th><th class="colsep0 rowsep0" align="center" char="±"><i>V</i><sub>ss</sub> [L kg<sup>–1</sup>]</th><th class="colsep0 rowsep0" align="center" char="±">AUC<sub>0–24hd.n.</sub> [nM h<sup>–1</sup>]</th><th class="colsep0 rowsep0" align="center" char="±"><i>C</i><sub>maxd.n.</sub> [nM]</th><th class="colsep0 rowsep0" align="center" char="±"><i>F</i> [%]</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mouse</td><td class="colsep0 rowsep0" align="char" char="±">11 ± 1</td><td class="colsep0 rowsep0" align="char" char="±">1.9 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">1.5 ± 0.3</td><td class="colsep0 rowsep0" align="char" char="±">3096 ± 386</td><td class="colsep0 rowsep0" align="char" char="±">549 ± 56</td><td class="colsep0 rowsep0" align="char" char="±">99 ± 12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat</td><td class="colsep0 rowsep0" align="char" char="±">41 ± 9</td><td class="colsep0 rowsep0" align="char" char="±">3.2 ± 0.4</td><td class="colsep0 rowsep0" align="char" char="±">9.4 ± 1.7</td><td class="colsep0 rowsep0" align="char" char="±">547 ± 124</td><td class="colsep0 rowsep0" align="char" char="±">46 ± 10</td><td class="colsep0 rowsep0" align="char" char="±">61 ± 14</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t9fn1"><div class="footnote" id="t9fn1"><sup><sup>a</sup></sup><p class="last">Pharmacokinetic parameters were calculated from blood levels after i.v. or p.o. administration to conscious animals (cassette dosing experiments with two and seven compounds, respectively—serial bleeding of four mice and four rats, respectively).</p></div><div class="footnote" id="t9fn2"><sup><sup>b</sup></sup><p class="last">i.v. administration of 1 mg/kg in NMP/PEG200 (% v 30/70) for rat and in NMP/Plasma (% v 10:90) for mice.</p></div><div class="footnote" id="t9fn3"><sup><sup>c</sup></sup><p class="last">p.o. administration of 3 mg/kg in MC/Tween80/water (% v 0.5:0.5:99) in rat and in mice solution formulation PG/cremophor ELP/8.33% Poloxamer 188 (Lutrol F-68) in H<sub>2</sub>O (% v 20:20:60).</p></div></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> MLT-231 Reduced T Cell-Dependent B Cell Responses in Mice</h3><div class="NLM_p">Mice treated with MLT-231 at 10, 30, or 100 mg/kg showed high (>1 μM) and nearly dose-proportional peak blood levels, that is, 2.6 ± 0.2, 9.3 ± 0.8, and 23.6 ± 1.5 μM, respectively, with 12 h trough levels reduced by 97–98%, that is, 0.04 ± 0.01, 0.12 ± 0.01, and 0.69 ± 0.09 μM, respectively, confirming a fast elimination phase for MLT-231.</div><div class="NLM_p">Under these conditions, MLT-231 dose-dependently reduced the anti-DNP IgM and IgG responses induced by 2,4-dinitrophenol (DNP) coupled to keyhole limpet hemocyanin (KLH) immunization, with about 25% inhibition achieved at 30 mg/kg and nearly complete inhibition attained at 100 mg/kg (equivalent to the effects of Cyclosporine A). Hence, an EC<sub>50</sub> value for MLT-231 in this model could be estimated to be approximately 50 mg/kg (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). No safety issues were identified (bodyweight, behavioral changes, histopathology).</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. MALT1 inhibitor MLT-231 blocks T cell-dependent B cell responses. (A) <i>In vivo</i> proof of concept was evaluated using DNP-KLH-induced antibody response in Balb/c mice treated twice daily (12 h interval) with vehicle or MLT-231 at 10, 30, or 100 mg/kg, as compared to mice treated once daily with CsA at 30 mg/kg. Anti-DNP IgM and IgG titers were assessed 8 days after DNP-KLH immunization. Full efficacy as measured by reduced anti-DNP antibody production was reached at 100 mg/kg BID, comparable to Cyclosporine A (CsA). (B) Dose-dependent efficacy was achieved when comparing the percentage of inhibition of anti-DNP IgM and IgG release in plasma <i>vs</i> the 12 h-trough compound blood exposure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01245&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> MLT-231 Displayed <i>In Vivo</i> Efficacy in the ABC-DLBCL Xenograft Models</h3><div class="NLM_p">To assess the therapeutic potential of MALT1 protease inhibition <i>in vivo</i>, the PK and pharmacodynamic (PD) properties of MLT-231 were tested in the ABC-DLBCL type TMD8 xenograft model (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A).<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> After a single-dose at 100 mg/kg p.o., MLT-231 exhibited good absorption with plasma <i>C</i><sub>max</sub> achieved at 2 h, which was corroborated by a rapid decrease of IL-10 production in the tumor. IL-10 disappearance upon MLT-231 treatment was mirrored by the more proximal readout of increased levels of uncleaved BCL10 in the tumor, both reflecting the blockade of MALT1.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. PK/PD study of the MALT1 inhibitor MLT-231 in ABC-DLBCL xenograft models. (A) PK/PD study of MLT-231 in TMD8 xenograft model (CD79B mutant) after one single oral dose (100 mg/kg). Mean plasma concentration ± SD (blue) and % tumor IL-10 ± SD (upper panel) and % tumor BCL10 ± SD (lower panel) are shown (<i>n</i> = 3 animals per timepoint). (B) Efficacy study in OCI-Ly10 xenograft model (CD79A mutant) after oral administration of either vehicle control or MLT-231 at indicated doses and schedules. Mean tumor volumes ± SEM are shown (<i>n</i> = 8; *<i>p</i> < 0.05 <i>vs</i> vehicle using Kruskal–Wallis, Dunn’s post hoc). (C) Pharmacokinetics of MLT-231 in plasma and tumor post last dose. Mean compound concentrations ± SD are shown in plasma (blue) and tumor (red) from two treated mice per timepoint (<i>n</i> = 2). (D) PD post last dose was monitored by BCL10 levels in tumor lysates (<i>n</i> = 2 mice per timepoint).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01245&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The efficacy of MLT-231 was evaluated in the OCI-Ly10 ABC-DLBCL type xenograft model at doses of 10, 30, and 100 mg/kg in an oral bid schedule for 2 weeks (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). At 100 mg/kg, MLT-231 led to tumor stasis, while being well tolerated (not shown). PK/PD post-therapy was established by determining concentration–time curves of the compound in plasma and tumor and measuring levels of uncleaved BCL10 in tumor lysates (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C). Dose-dependent increase of uncleaved tumor BCL10 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D) confirmed the blockade of MALT1 protease activity correlating with antitumor efficacy in this xenograft model.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Chemistry</h3><div class="NLM_p">The synthetic route for the preparation of the compounds described in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> is illustrated in the following <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Readily available chromanyl-3-amines and analogues thereof were directly reacted with the corresponding isocyanate or activated with p-nitro-phenyl chloroformate prior to reaction with an appropriate aniline to give the desired ureas.</div><figure id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds from <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01245&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) R–NCO, THF, rt, 53–84% yield or (ii) <i>p</i>-nitro-phenyl chloroformate, 1,2-DCE, collidine, 0 °C then aniline addition, 90 °C, 33–41% yield. Reagents and conditions: (iii) 3-chloro-4-methoxyaniline, HATU, DIPEA, DMF, rt, 38% yield; (iv) 2-(3-chloro-4-methoxyphenyl)acetic acid, (COCl)<sub>2</sub>, cat. DMF, TEA, DCM, 0 °C, 43% yield.</p></p></figure><div class="NLM_p">The corresponding acetamides were easily accessible by amide couplings from the commercially available acids and amines to provide <b>9</b> and <b>10</b>.</div><div class="NLM_p">Synthesis of the quinoline ureas shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> started with the corresponding 4-chloro-3-nitro quinoline <b>A</b>, which was treated with various amines to displace the C4-chlorine. Subsequent reduction of the nitro group resulted in the desired 3-amino quinoline <b>C</b> in good to excellent yields over two steps. To the corresponding quinoline, ureas were prepared analogously as described previously, either by activation with a chloro formate or direct treatment with an isocyanate (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>).</div><figure id="sch3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds from <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01245&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) amine, TEA, THF, 0 °C to rt, 30–80% yield; (ii) Fe powder, AcOH, rt, 92–95% yield; (iii) R–NCO, THF, rt, 27–40% yield; (iv) 4-fluoro or 4-nitro-phenyl chloroformate, 1,2-DCE, collidine then aniline, 26–60% yield.</p></p></figure><div class="NLM_p">The synthesis of the pyrazolopyrimidine ureas illustrated in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a> is shown in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>. In order to get access to the pyrazolopyrimidine core methyl 3-methoxy-2-nitroacrylate <b>D</b> was reacted with 5-bromo-1<i>H</i>-pyrazol-3-amine or the corresponding chloro-analogue to give intermediate <b>E</b>, which was cyclized under acidic conditions to the nitropyrazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one <b>F</b>.</div><figure id="sch4" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds from <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01245&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) subst. amino-pyrazole, cat. pTsOH, THF, 53–69% yield; (ii) AcOH, 110 °C, 90–91% yield; (iii) POCl3, DIPEA, 110 °C then TEA, DMAP, substit. morpholine, DCM, −10 °C, 24–56% yield; (iv) SnCl<sub>2</sub>, HCl, EtOH, rt, 97%—quant.; (v) <i>p</i>-F-Ph-COCl, pyridine, THF. rt then aniline, 70 °C, 23–68% yield.</p></p></figure><div class="NLM_p last">Chlorination with POCl<sub>3</sub> and nucleophilic substitution with various morpholines allowed the exploration of the C7 position with different R1 groups, resulting in intermediate <b>G</b>. Reduction of the nitro-group was accomplished by utilizing SnCl<sub>2</sub> to give the corresponding 3-amino-pyrazolopyrimidines <b>H</b> in good to excellent yields. The final formation of the desired ureas <b>23–27</b> was performed by reaction with 4-fluoro-phenylchloroformate and subsequent treatment with the desired anilines.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08430" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08430" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, we have discovered a novel allosteric MALT1 inhibitor derived from a chromane urea series initially identified in a high-throughput screen. Structure activity relationships around the chromane urea scaffold were systematically investigated, leading to a potency plateauing at sub-micromolar level. A first round of scaffold morphing from chromanes to quinolines allowed the exploration of new exit vectors and resulted in a significant improvement in biochemical and cellular activity. Despite its excellent potency, the flat nature of the quinoline urea series resulted in poor physicochemical and pharmacokinetic properties. A second round of morphing toward pyrazolopyrimidine ureas led to conformational changes in the scaffold, which increased the sp3-character and resulted in improved solubility. This series was further advanced and afforded compound <b>27</b> (MLT-231), which displayed good <i>in vitro</i> potency in a biochemical assay measuring MALT1 activity. MLT-231 showed good potency in a wide range of cellular assays measuring the functional impact of MALT1 inhibition on primary immune cells as well as in B cell lymphoma, where blockade of substrate cleavage and NF-κB pathway activation were both achieved. With favorable pharmacokinetic properties, MLT-231 demonstrated <i>in vivo</i> efficacy in a mouse model of antigen-induced antibody response and antitumor activity in an ABC-DLBCL type xenograft model in mouse. This represents the successful pharmacological validation of a novel selective MALT1 allosteric inhibitor in preclinical models of immune diseases and lymphomas.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00686" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00686" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Chemistry</h3><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Compound Synthesis and Characterization</h4><div class="NLM_p last">Unless otherwise stated, all reagents and solvents were purchased from commercial suppliers and used without further purification. <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra were recorded on either a Bruker 400 MHz or a Bruker 600 MHz NMR spectrometer. Chemical shifts are reported in parts per million (ppm) relative to an internal solvent reference in which they were run. Recorded peaks are listed in the order multiplicity (s, singlet; d, doublet; t, triplet; q, quartet; qt, quintet; m, multiplet; br, broad), coupling constants, and number of protons. Final compounds were purified to ≥95% purity as assessed by <sup>1</sup>H NMR spectra and analytical liquid chromatography using the methods below. UPLC and LCMS analyses were conducted using a Waters Acquity UPLC Binary Solvent Manager/Sample Organizer/Sample Manager/Column Manager and a Waters Acquity Photo Diode Array UV–vis detector at 210–450 nm (full scan) with the MS detection performed on a MS Waters Acquity Single Quadrupole Detector (SQD). All flash column chromatography was performed with prepacked Biotage or Teledyne Redisep silica cartridges and performed on either a Biotage or Isco purification system. UPLC/MS analysis was performed using the conditions described in the method below. Preparative reverse-phase HPLC purifications were performed using the method as described below.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Analytical LC–MS Method</h4><div class="NLM_p last">Column Acquity HSS T3 1.8 μm 2.1 × 50 mm at 60 °C; eluent A: water + 0.05% formic acid + 0.05% ammonium acetate; eluent B: acetonitrile + 0.04% formic acid; gradient: from 5 to 98% B in 1.4 min; then 98% B over 0.4 min with solvent flow 1.0 mL/min.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> Preparative Reverse-Phase Gilson HPLC Method</h4><div class="NLM_p">Column Sun Fire prep C18 OBD 5 μm, 30 × 100 mm from WATERS, with water + 0.1% TFA and acetonitrile + 0.1% TFA as the mobile phase. Detection method UV 220–400 nm.</div><div id="sec4_1_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31"> (<i>R</i>)-1-(3-Chloro-4-methoxyphenyl)-3-(chroman-3-yl)urea <b>2</b></h5><div class="NLM_p last">To a stirred solution of (<i>R</i>)-chroman-3-amine (200 mg, 1.3 mmol) in THF (3.0 mL) was added 2-chloro-4-isocyanato-1-methoxybenzene (246 mg, 1.3 mmol) under argon, and the mixture was stirred at rt for 16 h. The mixture was concentrated, and the product was purified by crystallization from hot EtOH to give the title compound (164 mg, 0.49 mmol, 37%). LC–MS: <i>t</i><sub>R</sub> = 1.07 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 333.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.46 (s, 1H), 7.60 (d, <i>J</i> = 2.6 Hz, 1H), 7.17–7.06 (m, 3H), 7.01 (d, <i>J</i> = 8.8 Hz, 1H), 6.86 (t, <i>J</i> = 7.4 Hz, 1H), 6.80 (d, <i>J</i> = 8.1 Hz, 1H), 6.31 (d, <i>J</i> = 7.2 Hz, 1H), 4.24–4.05 (m, 2H), 4.05–3.96 (m, 1H), 3.77 (s, 3H), 3.06 (dd, <i>J</i> = 16.3, 5.2 Hz, 1H), 2.69 (dd, <i>J</i> = 16.6, 4.9 Hz, 1H). UPLC purity: 98%.</div></div><div id="sec4_1_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> 1-(6-Bromochroman-3-yl)-3-(3-chloro-4-methoxyphenyl)urea <b>4</b></h5><div class="NLM_p last">To a stirred solution of 6-bromochroman-3-amine (650 mg, 2.85 mmol) in THF (6.0 mL) was added dropwise a solution of 2-chloro-4-isocyanato-1-methoxybenzene (246 mg, 1.3 mmol) in THF (10 mL) over 5 min, and the mixture was stirred for 90 min at rt. The resulting suspension was cooled in an ice bath, and the solid was filtered off, washed with small amounts of cold THF and Et<sub>2</sub>O, and dried under high vacuum to give the title compound (908 mg, 2.2 mmol, 77%). LC–MS: <i>t</i><sub>R</sub> = 1.25 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 413.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.44 (s, 1H), 7.59 (d, <i>J</i> = 2.6 Hz, 1H), 7.32 (d, <i>J</i> = 2.4 Hz, 1H), 7.26 (dd, <i>J</i> = 8.6, 2.6 Hz, 1H), 7.12 (dd, <i>J</i> = 8.8, 2.5 Hz, 1H), 7.01 (d, <i>J</i> = 9.0 Hz, 1H), 6.78 (d, <i>J</i> = 8.6 Hz, 1H), 6.34 (d, <i>J</i> = 7.2 Hz, 1H), 4.20–4.05 (m, 2H), 4.07–3.98 (m, 1H), 3.77 (s, 3H), 3.06 (dd, <i>J</i> = 16.6, 5.1 Hz, 1H), 2.71 (dd, <i>J</i> = 17.1, 4.9 Hz, 1H). UPLC purity: 100%.</div></div><div id="sec4_1_3_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> 1-(3-Chloro-4-methoxyphenyl)-3-(6-cyanochroman-3-yl)urea <b>5</b></h5><div class="NLM_p last">A mixture of 1-(6-bromochroman-3-yl)-3-(3-chloro-4-methoxyphenyl)urea (206 mg, 0.5 mmol), zinc powder (3.9 mg, 0.06 mmol), Zn(CN)<sub>2</sub> (36 mg, 0.3 mmol), Pd(dba)<sub>2</sub> (9.1 mg, 0.016 mmol), and 1,1′-bis(diphenylphosphino)ferrocene (11.1 mg, 0.02 mmol) in DMA (1.0 mL) was heated under argon at 100 °C for 17 h. The reaction was cooled to rt, and the solvent was removed in vacuo. The resulting residue was triturated with water, the suspension was filtered, and the solid was thoroughly washed with water. The filter cake was dissolved in EtOAc, the organic phase was washed with brine, dried over anhydr. Na<sub>2</sub>SO<sub>4</sub>, and filtered, and the solvent was removed in vacuo. The crude product was purified by flash column chromatography on a silica gel (EtOAc). Residual Pd-traces were removed by dissolution of the product in DCM/MeOH (4:1) and filtration through a StratoSpheresSPE PL-Thiol MP SPE cartridge. The solvent was removed in vacuo, and the solid was recrystallized from EtOH to give the title compound (136 mg, 0.38 mmol, 76%). LC–MS: <i>t</i><sub>R</sub> = 1.00 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 358.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.48 (s, 1H), 7.65 (d, <i>J</i> = 1.9 Hz, 1H), 7.63–7.54 (m, 2H), 7.13 (dt, <i>J</i> = 8.8, 1.9 Hz, 1H), 7.00 (ddd, <i>J</i> = 12.6, 8.8, 1.4 Hz, 2H), 6.51–6.34 (m, 1H), 4.24 (t, <i>J</i> = 6.4 Hz, 1H), 4.14 (t, <i>J</i> = 5.8 Hz, 2H), 3.78 (d, <i>J</i> = 1.5 Hz, 3H), 3.09 (dd, <i>J</i> = 16.3, 4.4 Hz, 1H), 2.77 (d, <i>J</i> = 16.3 Hz, 1H). UPLC purity: 100%.</div></div><div id="sec4_1_3_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> 1-(3-Chloro-4-methoxyphenyl)-3-(6-fluorochroman-3-yl)urea <b>6</b></h5><div class="NLM_p last">To a stirred suspension of 6-fluorochroman-3-amine hydrochloride (30 mg, 0.15 mmol) in THF (1.5 mL) was added TEA (41 μM, 0.30 mmol) at rt. After 5 min, a solution of 2-chloro-4-isocyanato-1-methoxybenzene (27 mg, 0.14 mmol) in THF (1.5 mL) was added, and the mixture was stirred for 16 h at rt. Solids were removed by filtration, washed with THF, and discarded. The filtrate was concentrated, and the solid was triturated with DCM, collected by filtration, washed with DCM, and dried under high vacuum to give the title compound (29 mg, 0.08 mmol, 57%). LC–MS: <i>t</i><sub>R</sub> = 1.03 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 331.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.44 (s, 1H), 7.60 (d, <i>J</i> = 2.6 Hz, 1H), 7.12 (dd, <i>J</i> = 8.8, 2.6 Hz, 1H), 7.01 (d, <i>J</i> = 9.0 Hz, 1H), 6.96 (ddd, <i>J</i> = 17.6, 9.0, 3.1 Hz, 2H), 6.83 (dd, <i>J</i> = 8.8, 4.9 Hz, 1H), 6.33 (d, <i>J</i> = 7.3 Hz, 1H), 4.22–4.06 (m, 2H), 4.05–3.96 (m, 1H), 3.78 (s, 3H), 3.07 (dd, <i>J</i> = 16.8, 5.2 Hz, 1H), 2.71 (dd, <i>J</i> = 16.8, 4.9 Hz, 1H). UPLC purity: 100%.</div></div><div id="sec4_1_3_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> 1-(3-Chloro-4-methoxyphenyl)-3-(1,2,3,4-tetrahydronaphthalen-2-yl)urea <b>7</b></h5><div class="NLM_p last">To a stirred suspension of 1,2,3,4-tetrahydronaphthalen-2-amine hydrochloride (139 mg, 0.76 mmol) in THF (3.0 mL) was added TEA (211 μM, 1.5 mmol) at rt. After 5 min, a solution of 2-chloro-4-isocyanato-1-methoxybenzene (143 mg, 0.76 mmol) in THF (2 mL) was added, and the mixture was stirred for 16 h at rt. The solvent was removed in vacuo, the residue was dissolved in EtOAc, and the organic phase was washed with water and brine. The organic phase was filtered without drying over Na<sub>2</sub>SO<sub>4.</sub> Upon partial removal of the solvent in vacuo, a solid precipitated. Ethanol and water were added, and the precipitate was collected by filtration. The filter cake was dried at 50 °C under high vacuum and purified by recrystallization from hot EtOH to afford the title compound (184 mg, 0.56 mmol, 74%). LC–MS: <i>t</i><sub>R</sub> = 1.10 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 331.3. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.36 (s, 1H), 7.62 (d, <i>J</i> = 2.4 Hz, 1H), 7.14 (dd, <i>J</i> = 9.0, 2.5 Hz, 1H), 7.12–7.05 (m, 4H), 7.02 (d, <i>J</i> = 8.9 Hz, 1H), 6.23 (d, <i>J</i> = 7.5 Hz, 1H), 3.93 (tq, <i>J</i> = 13.8, 7.4 Hz, 1H), 3.78 (s, 3H), 3.01 (dd, <i>J</i> = 16.4, 5.0 Hz, 1H), 2.82 (q, <i>J</i> = 5.6, 5.0 Hz, 2H), 2.63 (dd, <i>J</i> = 16.3, 7.8 Hz, 1H), 1.95 (dq, <i>J</i> = 14.7, 5.4 Hz, 1H), 1.71 (dq, <i>J</i> = 14.8, 7.7 Hz, 1H). UPLC purity: 100%.</div></div><div id="sec4_1_3_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> 1-(3-Chloro-4-methoxyphenyl)-3-(6-fluorothiochroman-3-yl)urea <b>8</b></h5><div class="NLM_p last">To a stirred suspension of 6-fluorothiochroman-3-amine (199 mg, 0.65 mmol) in THF (3.0 mL) was added TEA (182 μM, 1.3 mmol) at rt. After 30 min, a solution of 2-chloro-4-isocyanato-1-methoxy-benzene (120 mg, 0.65 mmol) in THF (2 mL) was added over 10 min, and the resulting clear solution was stirred for 60 min. The solvent was removed in vacuo, and the crude product was purified by flash column chromatography on silica (DCM/MeOH) to give the title compound (204 mg, 0.56 mmol, 85%). LC–MS: <i>t</i><sub>R</sub> = 1.08 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 367.0. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.60 (s, 1H), 7.62 (d, <i>J</i> = 2.5 Hz, 1H), 7.19–7.09 (m, 2H), 7.07–6.92 (m, 3H), 6.29 (d, <i>J</i> = 7.7 Hz, 1H), 4.27 (dh, <i>J</i> = 7.4, 3.6 Hz, 1H), 3.78 (s, 3H), 3.22 (dd, <i>J</i> = 12.5, 3.0 Hz, 1H), 2.99 (ddd, <i>J</i> = 20.1, 14.7, 5.5 Hz, 2H), 2.83 (dd, <i>J</i> = 16.7, 6.5 Hz, 1H). UPLC purity: 100%.</div></div><div id="sec4_1_3_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> <i>N</i>-(3-Chloro-4-methoxyphenyl)-2-(chroman-3-yl)acetamide <b>9</b></h5><div class="NLM_p last">To a solution of 2-(chroman-3-yl)acetic acid (125 mg, 0.65 mmol) in DMF (4 mL) was added DIPEA (340 μL, 1.95 mmol) and HATU (380 mg, 0.99 mmol), and the mixture was stirred for 30 min at rt. 3-Chloro-4-methoxyaniline (165 mg, 1.0 mmol) was added and the reaction was stirred for 72 h at rt. The reaction was quenched by the addition of aq. NaHCO<sub>3</sub>, and the aq. phase was extracted with EtOAc. The combined organic phases were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and filtered, and the solvent was removed in vacuo. Purification by flash chromatography on silica (cyclohexane/EtOAc) gave the title compound (84 mg, 0.25 mmol, 38%), which was recrystallized from hexane/EtOAc. LC–MS: <i>t</i><sub>R</sub> = 1.08 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 332.0. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.99 (s, 1H), 7.97–7.68 (m, 1H), 7.41 (dt, <i>J</i> = 9.0, 1.9 Hz, 1H), 7.22–7.00 (m, 3H), 6.82 (t, <i>J</i> = 7.4 Hz, 1H), 6.78–6.63 (m, 1H), 4.29–4.13 (m, 1H), 3.85 (dd, <i>J</i> = 10.6, 7.6 Hz, 1H), 3.81 (d, <i>J</i> = 1.4 Hz, 3H), 2.88 (dd, <i>J</i> = 16.0, 4.9 Hz, 1H), 2.54–2.53 (m, 1H), 2.47–2.37 (m, 2H), 2.37–2.25 (m, 1H). UPLC purity: 100%.</div></div><div id="sec4_1_3_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> 2-(3-Chloro-4-methoxyphenyl)-<i>N</i>-(chroman-3-yl)acetamide <b>10</b></h5><div class="NLM_p last">To a solution of 2-(3-chloro-4-methoxyphenyl)acetic acid (105 mg, 0.49 mmol) in DCM (3 mL) was added a drop of DMF at rt under argon. The solution was cooled to 0 °C, and oxalyl chloride (84 μL, 0.98 mmol) was added dropwise. After 1 h at 0 °C, the reaction mixture was cannulated into an ice-cold mixture of chroman-3-amine (116 mg, 0.62 mmol) and TEA (0.55 mL, 3.9 mmol) in DCM (3 mL). The mixture was slowly warmed to rt and was stirred for 14 h. The reaction was quenched by addition of water, and the aq. phase was extracted with DCM. The combined organic phases were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub> and filtered, and the solvent was removed in vacuo. Purification by flash chromatography on silica (cyclohexane/EtOAc) gave the title compound (74 mg, 0.22 mmol, 43%). UPLC-MS: <i>t</i><sub>R</sub> = 1.04 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 332.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.29–8.15 (m, 1H), 7.34–7.26 (m, 1H), 7.20–7.14 (m, 1H), 7.12–7.00 (m, 3H), 6.89–6.82 (m, 1H), 6.82–6.74 (m, 1H), 4.13–4.04 (m, 2H), 3.92–3.84 (m, 1H), 3.82 (s, 3H), 3.38 (s, 2H), 3.05–2.94 (m, 1H), 2.75–2.63 (m, 1H). UPLC purity: 100%.</div></div><div id="sec4_1_3_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> 3-(6-Bromochroman-3-yl)-1-(3-chloro-4-methoxyphenyl)-1-methylurea <b>11</b></h5><div class="NLM_p last">To an ice-cold solution of 4-nitrophenyl carbono-chloridate (71 mg, 0.35 mmol) in 1,2-DCE (1 mL) were added collidine (77 μL, 0.58 mmol) and dropwise a solution of 6-bromochroman-3-amine (70 mg, 0.31 mmol) in 1,2-DCE (1.7 mL). The mixture was stirred for 15 min at 0 °C and 45 min at rt. A solution of 3-chloro-4-methoxy-<i>N</i>-methylaniline (50 mg, 0.29 mmol) in 1,2-DCE (0.2 mL) was added, and the mixture as heated for 16 h at 100 °C. The mixture was cooled to rt, washed with 0.1 M aq. NaOH, water, 0.5 M aq. HCl, and brine, dried over anhydr. Na<sub>2</sub>SO<sub>4</sub>, and filtered, and the solvent was removed in vacuo. Purification by flash chromatography on silica (DCM/MeOH) gave the title compound (38 mg, 0.089 mmol, 30%). UPLC-MS: <i>t</i><sub>R</sub> = 1.14 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 427.1. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.33 (d, <i>J</i> = 2.6 Hz, 1H), 7.27 (d, <i>J</i> = 2.4 Hz, 1H), 7.22 (dd, <i>J</i> = 8.6, 2.6 Hz, 1H), 7.17 (dd, <i>J</i> = 8.7, 2.6 Hz, 1H), 7.11 (d, <i>J</i> = 8.8 Hz, 1H), 6.73 (d, <i>J</i> = 8.6 Hz, 1H), 5.92 (d, <i>J</i> = 7.5 Hz, 1H), 4.09 (dd, <i>J</i> = 10.8, 3.1 Hz, 1H), 4.00 (d, <i>J</i> = 8.7 Hz, 1H), 3.85 (s, 3H), 3.74 (t, <i>J</i> = 9.6 Hz, 1H), 2.87 (dd, <i>J</i> = 16.3, 5.6 Hz, 1H), 2.78 (dd, <i>J</i> = 16.4, 9.3 Hz, 1H). UPLC purity: 100%.</div></div><div id="sec4_1_3_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> 1-(6-Bromochroman-3-yl)-3-(3-chloro-4-methoxyphenyl)-1-methylurea <b>12</b></h5><div class="NLM_p last">To a stirred solution of 6-bromo-<i>N</i>-methylchroman-3-amine (12 mg, 0.05 mmol) in THF (0.6 mL) was added 2-chloro-4-isocyanato-1-methoxybenzene (9 mg, 0.05 mmol) at rt, and the mixture was stirred for 1 h. The mixture was diluted with EtOAc and washed with water and brine. The organic phase dried over anhydr. Na<sub>2</sub>SO<sub>4</sub> and filtered, and the solvent was removed in vacuo. The crude product was purified by crystallization from EtOH to give the title compound (6 mg, 0.014 mmol, 28%). UPLC-MS: <i>t</i><sub>R</sub> = 1.23 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 427.1. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.41 (s, 1H), 7.61 (d, <i>J</i> = 2.6 Hz, 1H), 7.38 (dd, <i>J</i> = 8.9, 2.6 Hz, 1H), 7.35 (d, <i>J</i> = 2.5 Hz, 1H), 7.26 (dd, <i>J</i> = 8.8, 2.5 Hz, 1H), 7.05 (d, <i>J</i> = 8.9 Hz, 1H), 6.78 (d, <i>J</i> = 8.6 Hz, 1H), 4.58–4.45 (m, 1H), 4.21–4.13 (m, 1H), 4.08 (t, <i>J</i> = 9.9 Hz, 1H), 3.80 (s, 3H), 3.07 (dd, <i>J</i> = 16.3, 10.3 Hz, 1H), 2.86 (s, 4H). UPLC purity: 100%.</div></div><div id="sec4_1_3_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> (<i>R</i>)-1-(6-Bromochroman-3-yl)-3-(3-chloro-4-(1<i>H</i>-1,2,4-triazol-1-yl)phenyl)urea <b>13</b></h5><div class="NLM_p last">To an ice-cold solution of 4-nitrophenyl carbonochloridate (366 mg, 1.8 mmol) in 1,2-DCE (10 mL) was slowly added a solution of 3-chloro-4-(1<i>H</i>-1,2,4-triazol-1-yl)aniline (294 mg, 1.5 mmol) and collidine (401 μL, 3.0 mmol) in 1,2-DCE (15 mL) <i>via</i> a dropping funnel. After complete addition, the mixture was stirred for 15 min at 0 °C and warmed to rt. After 15 min at rt, a suspension of (<i>R</i>)-6-bromochroman-3-amine hydrochloride (400 mg, 1.5 mmol) and collidine (200 μL, 1.5 mmol) in 1,2-DCE (2 mL) was added, and the mixture was heated for 15 min at 90 °C. After cooling to rt, the mixture was partitioned between DCM and water, and the organic phase was washed with 0.1 M aq. NaOH. The aq. phases were extracted once with DCM, the combined organic phases were washed with 0.5 M aq. HCl, water, and brine, dried over anhydr. Na<sub>2</sub>SO<sub>4</sub>, and filtered, and the solvent was removed in vacuo. The remaining solid was triturated with DCM, collected by filtration, washed with small amounts of DCM, and dried under high vacuum. Recrystallization from EtOH gave the title compound (278 mg, 0.62 mmol, 41%). LC–MS: <i>t</i><sub>R</sub> = 1.01 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 449.9. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.95 (s, 1H), 8.86 (s, 1H), 8.21 (s, 1H), 7.89 (d, <i>J</i> = 2.5 Hz, 1H), 7.48 (d, <i>J</i> = 8.7 Hz, 1H), 7.38–7.24 (m, 3H), 6.80 (d, <i>J</i> = 8.7 Hz, 1H), 6.59 (d, <i>J</i> = 7.0 Hz, 1H), 4.16 (d, <i>J</i> = 9.7 Hz, 2H), 4.08 (dd, <i>J</i> = 11.7, 5.0 Hz, 1H), 3.10 (dd, <i>J</i> = 16.8, 4.9 Hz, 1H), 2.75 (dd, <i>J</i> = 16.9, 4.5 Hz, 1H). UPLC purity: 100%.</div></div><div id="sec4_1_3_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> 1-(6-Bromochroman-3-yl)-3-(5-chloro-6-methoxypyridin-3-yl)urea <b>14</b></h5><div class="NLM_p last">To an ice-cold solution of 5-chloro-6-methoxypyridin-3-amine (63 mg, 0.36 mmol) in THF (2 mL) was added pyridine (43 μL, 0.53 mmol), followed by 4-fluorophenyl carbonochloridate (50 μL, 0.38 mmol). After 5 min, the reaction mixture was slowly warmed to rt and stirred for 1 h. A solution of 6-bromochroman-3-amine (75 mg, 0.33 mmol) in pyridine (2 mL) was added to the mixture, which was heated for 90 h at 70 °C. After cooling to rt, the solvent was removed in vacuo, and the residue was triturated with DCM. The solid was collected by filtration and washed with small amounts of DCM. The final purification by recrystallization from hot EtOH gave the title compound (47 mg, 0.11 mmol, 34%). UPLC-MS: <i>t</i><sub>R</sub> = 1.12 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 414.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.56 (s, 1H), 8.06 (d, <i>J</i> = 2.3 Hz, 1H), 7.99 (d, <i>J</i> = 2.4 Hz, 1H), 7.32 (d, <i>J</i> = 2.4 Hz, 1H), 7.26 (dd, <i>J</i> = 8.9, 2.4 Hz, 1H), 6.79 (d, <i>J</i> = 8.7 Hz, 1H), 6.50 (d, <i>J</i> = 7.1 Hz, 1H), 4.13 (d, <i>J</i> = 10.6 Hz, 2H), 4.08–3.98 (m, 1H), 3.87 (s, 3H), 3.07 (dd, <i>J</i> = 16.7, 5.0 Hz, 1H), 2.83–2.67 (m, 1H). UPLC purity: 100%.</div></div><div id="sec4_1_3_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> 1-(6-Bromoquinolin-3-yl)-3-(3-chloro-4-methoxyphenyl)urea <b>15</b></h5><div class="NLM_p last">To a solution of 6-bromoquinolin-3-amine dihydrochloride (200 mg, 0.68 mmol) in THF (3 mL) was added TEA (0.28 mL, 2.0 mmol), and the mixture was stirred at rt for 20 min. 3-Chloro-4-methoxybenzyl isocyanate (124 mg, 0.68 mmol) was added, and the reaction was stirred at rt for 16 h. The reaction mixture was filtered, and the filter cake was washed with THF. The filtrate was concentrated, and the resulting brown solid was purified by recrystallization from hot EtOH to give the title compound (93 mg, 0.22 mmol, 32%). UPLC-MS: <i>t</i><sub>R</sub> = 1.15 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 407.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.25 (s, 1H), 8.92 (s, 1H), 8.87 (d, <i>J</i> = 2.6 Hz, 1H), 8.48 (d, <i>J</i> = 2.5 Hz, 1H), 8.19 (d, <i>J</i> = 2.2 Hz, 1H), 7.86 (d, <i>J</i> = 8.9 Hz, 1H), 7.70 (d, <i>J</i> = 2.4 Hz, 2H), 7.32 (dd, <i>J</i> = 8.9, 2.6 Hz, 1H), 7.11 (d, <i>J</i> = 9.0 Hz, 1H), 3.82 (s, 3H). UPLC purity: 100%.</div></div><div id="sec4_1_3_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> 1-(6-Bromo-4-chloroquinolin-3-yl)-3-(3-chloro-4-methoxyphenyl)urea <b>16</b></h5><div class="NLM_p last">To an ice-cold solution of 4-nitrophenyl carbonochloridate (92 mg, 0.46 mmol) in 1,2-DCE (10 mL) was added collidine (102 μL, 0.76 mmol), followed by a solution of 5-chloro-6-methoxypyridin-3-amine (60 mg, 0.38 mmol) in 1,2-DCE (2 mL). The mixture was warmed to rt, and the yellow suspension was stirred for 30 min. A solution of 6-bromo-4-chloroquinolin-3-amine (98 mg, 0.38 mmol) in 1,2-DCE (2 mL) was added, and the mixture was heated at 80 °C for 16 h. After cooling to rt, the reaction mixture was diluted with DCM and 0.1 M aq. NaOH. The resulting suspension was filtered, and the filter cake was washed with water and DCM. The filter cake was dried under high vacuum and triturated with MeOH. The solid was collected by filtration, washed with small amounts of MeOH, and dried under high vacuum to give the title compound (101 mg, 60%). UPLC-MS: <i>t</i><sub>R</sub> = 1.27 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 442.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.62 (s, 1H), 9.44 (s, 1H), 8.79 (s, 1H), 8.26 (d, <i>J</i> = 2.2 Hz, 1H), 8.00 (d, <i>J</i> = 8.9 Hz, 1H), 7.87 (dd, <i>J</i> = 8.9, 2.2 Hz, 1H), 7.71 (d, <i>J</i> = 2.6 Hz, 1H), 7.30 (dd, <i>J</i> = 8.9, 2.6 Hz, 1H), 7.13 (d, <i>J</i> = 8.9 Hz, 1H), 3.83 (s, 3H). UPLC purity: 100%.</div></div><div id="sec4_1_3_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> 4-(Methyl(tetrahydro-2<i>H</i>-pyran-4-yl)amino)-3-nitroquinoline-6-carbonitrile <b>B1</b></h5><div class="NLM_p last">To an ice-cold solution of 4-chloro-3-nitroquinoline-6-carbonitrile <b>A</b> (700 mg, 3 mmol) in THF (8 mL) were added TEA (835 μL, 6 mmol) and <i>N</i>-methyltetrahydro-2H-pyran-4-amine (518 mg, 4.5 mmol). The mixture was stirred for 30 min at 0 °C and 2 h at rt. The solvent was removed in vacuo, the residue was dissolved in DCM, washed with water and brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and filtered, and the solvent was removed in vacuo. Purification by flash chromatography on silica (heptane/TBME) gave the title compound (668 mg, 2.1 mmol, 71%) UPLC-MS: <i>t</i><sub>R</sub> = 0.96 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 313.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.15 (d, <i>J</i> = 2.1 Hz, 1H), 8.58 (d, <i>J</i> = 2.0 Hz, 1H), 8.31–7.97 (m, 2H), 4.08–3.74 (m, 3H), 3.35–3.27 (m, 2H), 2.88 (d, <i>J</i> = 2.0 Hz, 3H), 2.07–1.70 (m, 4H). UPLC purity: 100%.</div></div><div id="sec4_1_3_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> 3-Amino-4-(methyl(tetrahydro-2<i>H</i>-pyran-4-yl)amino)quinoline-6-carbonitrile <b>C1</b></h5><div class="NLM_p last">To a mixture of 4-(methyl(tetrahydro-2<i>H</i>-pyran-4-yl)amino)-3-nitroquinoline-6-carbonitrile (665 mg, 2.1 mmol) in AcOH (16.2 mL) and water (1.8 mL) was added Fe powder (1.19 g, 21.3 mmol) under vigorous stirring. After 1 h at rt, the solvent was removed in vacuo, and the residue was taken up in EtOAc and carefully treated under vigorous stirring with sat aq. NaHCO<sub>3</sub> and solid NaHCO<sub>3</sub> (∼3 g). The mixture was stirred for several minutes, and the solids were removed by filtration and thoroughly washed with EtOAc. The filtrate phases were separated, the aq. phase was extracted with EtOAc, the combined organic phases were washed with brine, dried over anhydr. Na<sub>2</sub>SO<sub>4</sub>, and filtered, and the solvent was removed in vacuo to give the title compound (606 mg, 1.99 mmol, 94%), which was used without further purification. UPLC-MS: <i>t</i><sub>R</sub> = 0.80 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 283.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.66 (d, <i>J</i> = 2.0 Hz, 1H), 8.36 (s, 1H), 7.92 (dd, <i>J</i> = 8.5, 1.9 Hz, 1H), 7.60 (d, <i>J</i> = 8.5 Hz, 1H), 5.75 (d, <i>J</i> = 7.9 Hz, 2H), 3.90–3.72 (m, 2H), 3.48 (dq, <i>J</i> = 11.5, 5.3, 4.8 Hz, 1H), 3.35–3.13 (m, 2H), 2.81 (s, 3H), 1.64 (s, 2H), 1.43 (s, 2H). UPLC purity: 93%.</div></div><div id="sec4_1_3_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> 1-(3-Chloro-4-methoxyphenyl)-3-(6-cyano-4-(methyl(tetrahydro-2<i>H</i>-pyran-4-yl)amino)quinolin-3-yl)urea <b>17</b></h5><div class="NLM_p last">3-amino-4-(methyl(tetrahydro-2H-pyran-4-yl)amino)quinoline-6-carbonitrile (120 mg, 0.43 mmol) was dissolved in THF (10 mL) under argon. A solution of 2-chloro-4-isocyanato-1-methoxybenzene (78 mg, 0.43 mmol) in THF (2 mL) was added, and the mixture was stirred for 16 h at rt and 24 h at 35 °C. A solution of 2-chloro-4-isocyanato-1-methoxybenzene (30 mg, 0.16 mmol) in THF (2 mL) was added after additional 24 h and (40 mg, 0.22 mmol) in THF (2 mL) after 48 h. The reaction mixture was cooled to rt and concentrated, DCM was added, and the mixture was sonicated. The undesired solid (1,3-bis(3-chloro-4-methoxyphenyl)urea) was removed by filtration and washed with small amounts of THF. The filtrate was concentrated and purified by flash chromatography on silica (<i>n</i>-hexane/TBME and TBME/DCM/MeOH), followed by crystallization from DCM/iPr<sub>2</sub>O to give the title compound (79 mg, 0.17 mmol, 40%). UPLC-MS: <i>t</i><sub>R</sub> = 1.02 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 466.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.69 (s, 1H), 9.61 (s, 1H), 8.56 (s, 1H), 8.48 (s, 1H), 8.14 (d, <i>J</i> = 8.5 Hz, 1H), 7.90 (d, <i>J</i> = 8.6 Hz, 1H), 7.80–7.60 (m, 1H), 7.41–7.25 (m, 1H), 7.13 (d, <i>J</i> = 8.7 Hz, 1H), 3.83 (s, 5H), 3.70–3.55 (m, 1H), 3.27 (t, <i>J</i> = 11.8 Hz, 3H), 2.96 (s, 3H), 1.67 (s, 2H), 1.59–1.40 (m, 2H). UPLC purity: 100%.</div></div><div id="sec4_1_3_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> 1-(5-Chloro-6-methoxypyridin-3-yl)-3-(6-cyano-4-(methyl(tetrahydro-2<i>H</i>-pyran-4-yl)amino)quinolin-3-yl)urea <b>18</b></h5><div class="NLM_p last">To an ice-cooled solution of 4-nitrophenyl carbonochloridate (61 mg, 0.30 mmol) in 1,2-DCE (2.6 mL) was added collidine (67 μL, 0.5 mmol), followed by 5-chloro-6-methoxypyridin-3-amine (40 mg, 0.25 mmol). The mixture was warmed to rt, and the red solution was stirred for 1 h. A solution of 3-amino-4-(methyl(tetrahydro-2<i>H</i>-pyran-4-yl)amino)quinoline-6-carbonitrile (52 mg, 0.18 mmol) in 1,2-DCE (0.5 mL) was added, and the mixture was heated at 80 °C for 90 min. After cooling to rt, the reaction mixture was dissolved in EtOAc and washed with 0.1 M aq. NaOH, water, and brine, dried over anhydr. Na<sub>2</sub>SO<sub>4</sub>, and filtered, and the solvent was removed in vacuo. The crude product was purified by preparative RP-HPLC, and product-containing fractions were pooled and basified with solid NaHCO<sub>3</sub>. Upon partial removal of the solvent in vacuo, a solid precipitated, which was collected by filtration and dried under high vacuum to give the title compound (32 mg, 0.07 mmol, 26%). UPLC-MS: <i>t</i><sub>R</sub> = 1.06 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 467.3. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.86–9.77 (m, 1H), 9.71–9.63 (m, 1H), 8.68–8.59 (m, 1H), 8.59–8.52 (m, 1H), 8.24–8.16 (m, 1H), 8.16–8.10 (m, 2H), 7.96–7.83 (m, 1H), 4.00–3.87 (m, 3H), 3.87–3.73 (m, 2H), 3.68–3.55 (m, 1H), 3.29–3.20 (m, 2H), 2.95 (s, 3H), 1.79–1.40 (m, 4H). UPLC purity: 94%.</div></div><div id="sec4_1_3_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> (<i>S</i>)-4-(3-Methylmorpholino)-3-nitroquinoline-6-carbonitrile <b>B2</b></h5><div class="NLM_p last">To an ice-cooled solution of 4-chloro-3-nitroquinoline-6-carbonitrile (130 mg, 0.56 mmol) in THF (2 mL) was added TEA (196 μL, 1.39 mmol) and (<i>S</i>)-3-methylmorpholine (96 mg, 1.3 mmol). The mixture was stirred for 60 min at 0 °C and 48 h at rt. The reaction was quenched by the addition of water, and the aq. phase was extracted with EtOAc. The combined organic phases were washed with brine, dried over anhydr. Na<sub>2</sub>SO<sub>4</sub>, and filtered, and the solvent was removed in vacuo to give the title compound (148 mg, 0.45 mmol, 80%), which was used without further purification. UPLC-MS: <i>t</i><sub>R</sub> = 0.96 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 299.2. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.22 (s, 1H), 8.77 (s, 1H), 8.23 (s, 2H), 4.07–3.99 (m, 1H), 3.96 (ddd, <i>J</i> = 10.7, 7.3, 3.0 Hz, 1H), 3.80 (ddd, <i>J</i> = 11.3, 5.7, 3.0 Hz, 1H), 3.66 (s, 1H), 3.55 (dd, <i>J</i> = 11.1, 6.2 Hz, 1H), 3.49 (ddd, <i>J</i> = 12.7, 5.7, 2.9 Hz, 1H), 3.04 (ddd, <i>J</i> = 12.9, 7.4, 2.9 Hz, 1H), 0.91 (d, <i>J</i> = 6.4 Hz, 3H). UPLC purity: 91%.</div></div><div id="sec4_1_3_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> (S)-3-Amino-4-(3-methylmorpholino)quinoline-6-carbonitrile <b>C2</b></h5><div class="NLM_p last">To a mixture of (<i>S</i>)-4-(3-methylmorpholino)-3-nitroquinoline-6-carbonitrile (145 mg, 0.49 mmol) in AcOH (2.5 mL) and water (0.25 mL) was added Fe powder (271 mg, 4.9 mmol), and the reaction was stirred for 90 min at rt. The solvent was removed in vacuo, and the residue was taken up in EtOAc and carefully treated under vigorous stirring with sat aq. NaHCO<sub>3</sub> (∼7 mL) and solid NaHCO<sub>3</sub> (∼1.5 g). Hyflo was added, the mixture was stirred for several minutes, and the solids were removed by filtration and thoroughly washed with EtOAc. The filtrate phases were separated, the aq. phase was extracted with EtOAc, the combined organic phases were washed with sat aq. NaHCO<sub>3</sub> and brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and filtered, and the solvent was removed in vacuo to afford the title compound (123 mg, 0.46 mmol, 94%), which was used without further purification. UPLC-MS: <i>t</i><sub>R</sub> = 0.77 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 269.3. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.78–8.28 (m, 2H), 7.94 (dd, <i>J</i> = 36.1, 8.5 Hz, 1H), 7.61 (t, <i>J</i> = 7.6 Hz, 1H), 5.97 (s, 1H), 5.78 (s, 1H), 3.88–3.66 (m, 4H), 3.52 (d, <i>J</i> = 13.5 Hz, 1H), 3.37 (d, <i>J</i> = 9.9 Hz, 1H), 2.66 (dd, <i>J</i> = 48.0, 12.0 Hz, 1H), 0.59 (dd, <i>J</i> = 10.3, 6.0 Hz, 3H). UPLC purity: 100%.</div></div><div id="sec4_1_3_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> (<i>S</i>)-1-(3-Chloro-4-methoxyphenyl)-3-(6-cyano-4-(3-methylmorpholino)quinolin-3-yl)urea <b>19</b></h5><div class="NLM_p last">To a solution of (<i>S</i>)-3-amino-4-(3-methylmorpholino)quinoline-6-carbonitrile (120 mg, 0.45 mmol) in THF (5 mL) was added 2-chloro-4-isocyanato-1-methoxybenzene (115 mg, 0.63 mmol), and the mixture was heated for 16 h at 40 °C. Additional 2-chloro-4-isocyanato-1-methoxybenzene (58 mg, 0.32 mmol) was added, and stirring was continued for 7 h at 45 °C. The solvent was removed in vacuo, and the crude product was purified by preparative RP-HPLC. The product-containing fractions were pooled and basified with solid NaHCO<sub>3</sub>. Upon partial removal of the solvent in vacuo, a solid precipitated, which was collected by filtration, washed with water, and dried under high vacuum at 60 °C to afford the title compound (54 mg, 0.45 mmol, 27%). UPLC-MS: <i>t</i><sub>R</sub> = 1.01 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 452.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.71 (s, 1H), 8.38 (s, 1H), 8.14 (d, <i>J</i> = 8.6 Hz, 1H), 7.95 (d, <i>J</i> = 8.6 Hz, 1H), 7.73 (d, <i>J</i> = 2.4 Hz, 1H), 7.32 (dd, <i>J</i> = 8.6, 2.6 Hz, 1H), 7.13 (d, <i>J</i> = 8.9 Hz, 1H), 4.05–3.83 (m, 4H), 3.83 (s, 3H), 3.41 (d, <i>J</i> = 37.1 Hz, 2H), 2.93 (d, <i>J</i> = 12.0 Hz, 1H), 0.63 (d, <i>J</i> = 6.2 Hz, 3H) (2 protons not visible in NMR). UPLC purity: 100%.</div></div><div id="sec4_1_3_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> (<i>S</i>)-1-(5-Chloro-6-methoxypyridin-3-yl)-3-(6-cyano-4-(3-methylmorpholino)quinolin-3-yl)urea <b>20</b>, <b>MLT-827</b></h5><div class="NLM_p last">To an ice-cooled solution of 4-nitrophenyl carbonochloridate (83 mg, 0.41 mmol) in 1,2-DCE (2.5 mL) was added a solution of collidine (57 μL, 0.43 mmol) and 5-chloro-6-methoxypyridin-3-amine (59 mg, 0.38 mmol) in 1,2-DCE (1 mL), and the mixture was stirred for 20 min at 0 °C. To this solution was added a solution of (<i>S</i>)-3-amino-4-(3-methylmorpholino)quinoline-6-carbonitrile (71 mg, 0.25 mmol) and collidine (53 μL, 0.4 mmol) in 1,2-DCE (1.2 mL), and the reaction mixture was heated for 6 h at 50 °C. After cooling to rt, the solvent was removed in vacuo, and the crude product was purified by preparative RP-HPLC. Product-containing fractions were pooled and basified with solid NaHCO<sub>3</sub>. Upon partial removal of the solvent in vacuo, a solid precipitated, which was collected by filtration and dried under high vacuum at 60 °C to give the title compound (50 mg, 0.25 mmol, 44%). UPLC-MS: <i>t</i><sub>R</sub> = 1.00 min. MS (ESI): [M<sup>+</sup>] <i>m</i>/<i>z</i>: 453.2. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.81 (br s, 1H), 9.22 (d, <i>J</i> = 137.5 Hz, 1H), 8.73 (s, 1H), 8.53 (s, 1H), 8.27–8.10 (m, 3H), 7.97 (d, <i>J</i> = 8.7 Hz, 1H), 4.00–3.76 (m, 7H), 3.45 (s, 2H), 2.94 (d, <i>J</i> = 12.0 Hz, 1H), 0.64 (d, <i>J</i> = 6.3 Hz, 3H). UPLC purity: 100%.</div></div><div id="sec4_1_3_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> 6-Fluoro-<i>N</i>-methyl-<i>N</i>-(1-methylpiperidin-3-yl)-3-nitroquinolin-4-amine <b>B3</b></h5><div class="NLM_p last">To an ice-cold solution of 4-chloro-6-fluoro-3-nitroquinoline (300 mg, 1.32 mmol) in THF (8 mL) were added TEA (367 μL, 2.65 mmol) and N,1-dimethylpiperidin-3-amine (204 mg, 1.59 mmol). The mixture was stirred for 15 min at 0 °C and 2 h at rt. The reaction was quenched by addition of water, and the aq. phase was extracted with EtOAc. The combined organic phases were washed with brine, dried over anhydr. Na<sub>2</sub>SO<sub>4</sub>, and filtered, and the solvent was removed in vacuo to give the title compound (338 mg, 1.0 mmol, 80%), which was used without further purification. UPLC-MS: <i>t</i><sub>R</sub> = 0.60 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 319.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.02 (s, 1H), 8.16 (dd, <i>J</i> = 9.2, 5.5 Hz, 1H), 7.91–7.61 (m, 2H), 3.63 (tt, <i>J</i> = 10.4, 3.6 Hz, 1H), 3.31 (s, 1H), 2.93 (d, <i>J</i> = 11.2 Hz, 1H), 2.87 (s, 3H), 2.72–2.54 (m, 1H), 2.17 (s, 3H), 1.92 (d, <i>J</i> = 12.2 Hz, 1H), 1.86–1.74 (m, 1H), 1.68 (dt, <i>J</i> = 12.9, 3.3 Hz, 1H), 1.57 (qd, <i>J</i> = 11.9, 3.6 Hz, 1H), 1.45 (tdd, <i>J</i> = 12.3, 8.4, 3.7 Hz, 1H). UPLC purity: 100%.</div></div><div id="sec4_1_3_24" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> 6-Fluoro-N4-methyl-N4-(1-methylpiperidin-3-yl)quinoline-3,4-diamine <b>C3</b></h5><div class="NLM_p last">To a mixture of 6-fluoro-<i>N</i>-methyl-<i>N</i>-(1-methylpiperidin-3-yl)-3-nitroquinolin-4-amine (336 mg, 1.1 mmol) in AcOH (7 mL) and water (0.7 mL) was added Fe powder (1.1 g, 21 mmol), and the reaction was stirred for 60 min at rt. The solvent was removed by coevaporation with toluene in vacuo, and the residue was taken up in EtOAc and carefully treated under vigorous stirring with sat aq. NaHCO<sub>3</sub> (∼25 mL) and solid NaHCO<sub>3.</sub> (∼1.5 g). Solids were removed by filtration and thoroughly washed with EtOAc. The filtrate phases were separated, the aq. phase was extracted with EtOAc, the combined organic phases were washed with sat aq. NaHCO<sub>3</sub> and brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and filtered, and the solvent was removed in vacuo to give the title compound (288 mg, 0.99 mmol, 95%), which was used without further purification. UPLC-MS: <i>t</i><sub>R</sub> = 0.50 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 289.2. UPLC purity: 100%.</div></div><div id="sec4_1_3_25" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> (<i>S</i>)-1-(5-Chloro-6-((1-methylpiperidin-4-yl)oxy)pyridin-3-yl)-3-(6-fluoro-4-(3-methylmorpholino)quinolin-3-yl)urea <b>21</b></h5><div class="NLM_p">To an ice-cold solution of 5-chloro-6-methoxypyridin-3-amine (500 mg, 3.2 mmol) in THF (7 mL) was added pyridine (0.32 mL, 3.94 mmol), followed by 4-fluorophenyl carbonochloridate (443 μL, 3.5 mmol). The reaction mixture was slowly warmed to rt and stirred for 2 h. The reaction was quenched by addition of water, the aq. phase was extracted with EtOAc, the combined organic extracts were dried over anhydr. Na<sub>2</sub>SO<sub>4</sub> and filtered, and the solvent was removed in vacuo to afford 4-fluorophenyl 5-chloro-6-methoxypyridin-3-ylcarbamate (855 mg, 3.15 mmol, 91%), which was used without further purification. UPLC-MS: <i>t</i><sub>R</sub> = 1.06 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 297.1. UPLC purity: 100%.</div><div class="NLM_p last">A mixture of 4-fluorophenyl 5-chloro-6-methoxypyridin-3-ylcarbamate (111 mg, 0.38 mmol) and 6-fluoro-N4-methyl-N4-(1-methylpiperidin-3-yl)quinoline-3,4-diamine (90 mg, 0.31 mmol) in pyridine (1.2 mL) was heated in a microwave for 16 h at 70 °C. Additional 6-fluoro-N4-methyl-N4-(1-methylpiperidin-3-yl)quinoline-3,4-diamine (50 mg, 0.17 mmol) was added, and heating was continued for 1 h to complete conversion. The solvent was removed in vacuo, and the crude product was purified by flash column chromatography on silica (DCM/MeOH) to give the title compound (83 mg, 0.18 mmol, 56%). UPLC-MS: <i>t</i><sub>R</sub> = 0.74 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 473.2. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.73 (s, 1H), 9.49 (s, 1H), 8.57 (s, 1H), 8.19 (d, <i>J</i> = 2.4 Hz, 1H), 8.15 (s, 1H), 8.06 (dd, <i>J</i> = 9.2, 5.6 Hz, 1H), 7.69 (s, 1H), 7.53 (td, <i>J</i> = 8.7, 2.7 Hz, 1H), 3.92 (s, 3H), 3.50–3.46 (m, 1H), 2.93 (s, 3H), 2.68–2.62 (m, 1H), 2.29–2.16 (m, 3H), 1.97–1.86 (m, 2H), 1.61–1.55 (m, 1H), 1.44–1.40 (m, 1H), 1.24 (d, <i>J</i> = 9.6 Hz, 1H) 1 proton hidden. UPLC purity: 100%.</div></div><div id="sec4_1_3_26" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> (<i>S</i>)-4-(6-Fluoro-3-nitroquinolin-4-yl)-3-methylmorpholine <b>B4</b></h5><div class="NLM_p last">To a solution of 4-chloro-6-fluoro-3-nitroquinoline (2.0 g, 8.83 mmol) in THF (50 mL) was added (<i>S</i>)-3-methylmorpholine (1.07 g, 10.6 mmol), followed by TEA (4.9 mL, 35.3 mmol) and the reaction was stirred for 16 h at rt. The reaction mixture was concentrated and partitioned between EtOAc and brine. The aq. phase was extracted with EtOAc, the combined organic extracts were dried over anhydr. Na<sub>2</sub>SO<sub>4</sub> and filtered, and the filtrate was concentrated. The crude product was purified by flash column chromatography on silica (cyclohexane/EtOAc) to give the title compound (0.79 g, 2.60 mmol, 30% yield). UPLC-MS: <i>t</i><sub>R</sub> = 1.02 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 292.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.09 (s, 1H), 8.20 (dd, <i>J</i> = 9.3, 5.4 Hz, 1H), 8.02 (dd, <i>J</i> = 10.0, 2.9 Hz, 1H), 7.87 (td, <i>J</i> = 8.7, 2.8 Hz, 1H), 3.93 (ddd, <i>J</i> = 18.1, 9.8, 2.7 Hz, 2H), 3.81 (dt, <i>J</i> = 11.3, 3.6 Hz, 1H), 3.60–3.33 (m, 3H), 3.11 (ddd, <i>J</i> = 12.4, 8.7, 3.1 Hz, 1H), 0.81 (d, <i>J</i> = 6.1 Hz, 3H). UPLC purity: 96%.</div></div><div id="sec4_1_3_27" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57"> (<i>S</i>)-6-Fluoro-4-(3-methylmorpholino)quinolin-3-amine <b>C4</b></h5><div class="NLM_p last">To a solution of (<i>S</i>)-4-(6-fluoro-3-nitroquinolin-4-yl)-3-methylmorpholine (0.69 g, 2.4 mmol) in EtOAc (150 mL) was added 10% Pd–C (0.252 g, 2.4 mmol) at room temperature. The reaction mixture was degassed, equipped with a hydrogen balloon, and purged with hydrogen under normal pressure. The reaction mixture was stirred for 16 h at ambient temperature and filtered, the solid was washed with EtOAc, and the filtrate was concentrated to give the title compound (0.59 g, 2.168 mmol, 92% yield). UPLC-MS: <i>t</i><sub>R</sub> = 0.80 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 262.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.51 (d, <i>J</i> = 36.2 Hz, 1H), 8.04–7.72 (m, 1H), 7.45 (dd, <i>J</i> = 11.3, 2.7 Hz, 1H), 7.20 (td, <i>J</i> = 8.8, 2.6 Hz, 1H), 5.77 (d, <i>J</i> = 9.0 Hz, 1H), 5.55 (s, 1H), 3.99–3.64 (m, 4H), 3.50 (td, <i>J</i> = 11.1, 10.5, 4.2 Hz, 1H), 3.42–3.16 (m, 1H), 2.65 (dd, <i>J</i> = 29.9, 11.7 Hz, 1H), 0.61 (t, <i>J</i> = 6.4 Hz, 3H). UPLC purity: 96%.</div></div><div id="sec4_1_3_28" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> (<i>S</i>)-1-(5-Chloro-6-((1-methylpiperidin-4-yl)oxy)pyridin-3-yl)-3-(6-fluoro-4-(3-methylmorpholino)quinolin-3-yl)urea <b>22</b></h5><div class="NLM_p">To an ice-cold solution of (<i>S</i>)-6-fluoro-4-(3-methylmorpholino)quinolin-3-amine (204 mg, 0.78 mmol) in THF (10 mL) was added pyridine (95 μL, 1.2 mmol), followed by 4-fluorophenyl carbonochloridate (154 μL, 1.2 mmol). The reaction mixture was slowly warmed to rt and stirred for 1 h. The reaction was quenched by addition of water, the aq. phase was extracted with EtOAc, the combined organic extracts were dried over anhydr. Na<sub>2</sub>SO<sub>4</sub> and filtered, and the solvent was removed in vacuo. The crude product was purified by flash column chromatography on silica (<i>n</i>-heptane/EtOAc) to afford (<i>S</i>)-4-fluorophenyl (6-fluoro-4-(3-methylmorpholino)quinolin-3-yl)carbamate (304 mg, 0.754 mmol, 97% yield). UPLC-MS: <i>t</i><sub>R</sub> = 1.09 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 400.2. UPLC purity: 51%.</div><div class="NLM_p last">A mixture of (<i>S</i>)-4-fluorophenyl (6-fluoro-4-(3-methylmorpholino)quinolin-3-yl)carbamate (100 mg, 0.25 mmol) and 5-chloro-6-((1-methylpiperidin-4-yl)oxy)pyridin-3-amine (61 mg, 0.25 mmol) in acetonitrile (1 mL) was heated in a microwave for 30 min at 90 °C. The solvent was removed in vacuo, and the crude product was purified by the RP-preparative HPLC. Product-containing fractions were lyophilized overnight. The yellow powder was dissolved in a minimal amount of MeOH and passed through a PL-HCO3 MP StratoSphere TM SPE column (500 mg per 6 mL tube), washed thoroughly with MeOH, and the filtrate was concentrated in vacuo. The residue was dissolved in a minimal amount of acetonitrile/water and lyophilized to give the title compound (65 mg, 0.12 mmol, 49%). UPLC-MS: <i>t</i><sub>R</sub> = 0.80 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 529.1. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.29 (s, 2H), 8.24 (s, 1H), 8.18 (d, <i>J</i> = 2.5 Hz, 1H), 8.11 (d, <i>J</i> = 2.5 Hz, 1H), 8.07 (dd, <i>J</i> = 9.2, 5.6 Hz, 1H), 7.88 (d, <i>J</i> = 10.8 Hz, 1H), 7.52 (td, <i>J</i> = 8.7, 2.9 Hz, 1H), 5.14–5.04 (m, 1H), 3.95–3.87 (m, 3H), 3.85–3.78 (m, 1H), 3.57–3.50 (m, 1H), 3.46 (dd, <i>J</i> = 10.9, 9.1 Hz, 1H), 2.92 (d, <i>J</i> = 2.3 Hz, 1H), 2.77–2.67 (m, 2H), 2.43–2.34 (m, 2H), 2.30 (s, 3H), 2.07–1.97 (m, 2H), 1.86–1.71 (m, 2H), 0.70 (d, <i>J</i> = 6.2 Hz, 3H). UPLC purity: 100%.</div></div><div id="sec4_1_3_29" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59"> Methyl 3-Methoxy-2-nitroacrylate <b>D</b></h5><div class="NLM_p last">To a solution of methyl 2-nitroacetate (19.7 mL, 202 mmol) in Ac<sub>2</sub>O (45.7 ml, 483 mmol) under argon was added trimethyl orthoformate (33.2 mL, 303 mmol), and the reaction mixture was refluxed (110 °C) for 3.5 days. The reaction mixture was cooled to rt and concentrated under reduced pressure to give the title compound (38.6 g, 202 mmol, quant.). The material was stored under argon in the freezer and used without further purification in the following step.</div></div><div id="sec4_1_3_30" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60"> Methyl 3-((5-Bromo-1<i>H</i>-pyrazole-3-yl)amino)-2-nitroacrylate <b>E1</b></h5><div class="NLM_p last">To a stirred solution of 3-bromo-1<i>H</i>-pyrazol-5-amine (5.1 g, 31.2 mmol) in dry THF (100 mL) was added pTsOH monohydrate (60 mg, 0.31 mmol) at rt, followed by dropwise addition of methyl 3-methoxy-2-nitroacrylate <b>D</b> (6.04 g, 37.5 mmol) and the reaction mixture was stirred for 4 h at rt. The solvent was removed in vacuo, and the remaining solid was triturated with a <i>n</i>-hexane, sonicated, filtered, washed with <i>n</i>-hexane, and dried to give the title compound (5.2 g, 16.6 mmol, 53%) UPLC-MS: <i>t</i><sub>R</sub> = 0.69 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 291. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.54/13.28 (s, 1H), 11.56/10.87 (m, 1H), 9.07/8.66 (m, 1H), 6.68–6.32 (m, 1H), 3.78 (m, 3H). UPLC purity: 93%.</div></div><div id="sec4_1_3_31" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61"> Methyl 3-((5-Chloro-1<i>H</i>-pyrazole-3-yl)amino)-2-nitroacrylate <b>E2</b></h5><div class="NLM_p last">The title compound was prepared in analogy to the procedure described for the bromo analogue <b>E1</b> from 3-chloro-1<i>H</i>-pyrazol-5-amine (2.96 g, 25 mmol) and methyl 3-methoxy-2-nitroacrylate <b>D</b> (5.51 g, 28.7 mmol) to give the title compound (4.31 g, 17.3 mmol, 69%). UPLC-MS: <i>t</i><sub>R</sub> = 0.69 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 247.1.0. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) mixture of cis/trans isomers: δ 13.59/13.16 (s, 1H), 11.55/10.86 (d, 1H), 9.07/8.66 (d, 1H), 6.47/6.45 (s, 1H), 3.83/3.74 (s, 3H). UPLC purity: 100%.</div></div><div id="sec4_1_3_32" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62"> 2-Bromo-6-nitropyrazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one <b>F1</b></h5><div class="NLM_p last">A solution of methyl 3-(3-bromo-1<i>H</i>-pyrazol-5-ylamino)-2-nitroacrylate <b>E1</b> (5.2 g, 16.6 mmol) in AcOH (40 mL) under argon was heated overnight at 110 °C. The reaction mixture was cooled to rt, and the solid was filtered, washed with a minimal amount of AcOH and <i>n</i>-hexane, and dried to give the title compound (3.91, 15.1 mmol, 91%). UPLC-MS: <i>t</i><sub>R</sub> = 0.47 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 259.0. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.24 (s, 1H), 6.60 (s, 1H). UPLC purity: 99%.</div></div><div id="sec4_1_3_33" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63"> 2-Chloro-6-nitropyrazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one <b>F2</b></h5><div class="NLM_p last">The title compound was prepared in analogy to the procedure for the 2-bromo compound <b>F1</b> from methyl 3-((5-chloro-1<i>H</i>-pyrazole-3-yl)amino)-2-nitroacrylate (4.31 g, 17.3 mmol) to give the title compound (3.38 g, 15.6 mmol, 90%). UPLC-MS: <i>t</i><sub>R</sub> = 0.45 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 215.0. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.20 (s, 1H), 6.53 (s, 1H). UPLC purity: 98%.</div></div><div id="sec4_1_3_34" class="NLM_sec NLM_sec_level_4"><div id="ac_i64" class="anchor-spacer"></div><h5 class="article-section__title" id="_i64"> (<i>S</i>)-4-(2-Bromo-6-nitropyrazolo[1,5-<i>a</i>]pyrimidin-7-yl)-3-methylmorpholine <b>G1</b></h5><div class="NLM_p last">A stirred suspension of 2-bromo-6-nitropyrazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (350 mg, 1.35 mmol) in POCl<sub>3</sub> (8.3 mL, 89 mmol) under argon was cooled to 0 °C and DIPEA (0.52 mL, 3 mmol) was added dropwise under vigorous stirring. The ice-bath was removed after 5 min, and the mixture was heated for 2 h 100 °C. Excess of POCl<sub>3</sub> was removed in vacuo to dryness, and the remaining residue was dissolved in DCM (15 mL) to give <b>solution 1</b>. A separate flask equipped with a dropping funnel, and a thermometer was charged under argon with (<i>S</i>)-3-methylmorpholine hydrochloride (232 mg, 1.7 mmol), TEA (2.4 mL, 17.6 mmol) and DMAP (2 mg) in DCM (15 mL). The mixture was cooled −10 °C, and <b>solution 1</b> was added dropwise <i>via</i> the dropping funnel over 10 min. The reaction mixture was stirred for 1 h at −10 °C and at rt overnight. The solvent was removed in vacuo, and the residue was taken up in EtOAc. The organic phase was washed with sat. aq. NaHCO<sub>3</sub> and brine, dried over anhydr. Na<sub>2</sub>SO<sub>4</sub>, and filtered, and the solvent was removed in vacuo. The crude product was purified by flash column chromatography on silica (cylcohexane/EtOAc) to give the title compound (115 mg, 0.33 mmol, 24%). UPLC-MS: <i>t</i><sub>R</sub> = 1.02 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 343.9. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.91 (s, 1H), 7.00 (s, 1H), 5.00 (d, <i>J</i> = 7.3 Hz, 1H), 4.02 (dd, <i>J</i> = 11.4, 2.9 Hz, 1H), 3.92–3.81 (m, 2H), 3.84–3.73 (m, 1H), 3.61 (dd, <i>J</i> = 11.2, 2.8 Hz, 1H), 2.79 (d, <i>J</i> = 12.6 Hz, 1H), 1.39 (d, <i>J</i> = 6.8 Hz, 3H). UPLC purity: 100%.</div></div><div id="sec4_1_3_35" class="NLM_sec NLM_sec_level_4"><div id="ac_i65" class="anchor-spacer"></div><h5 class="article-section__title" id="_i65"> (<i>S</i>)-2-Bromo-7-(3-methylmorpholino)pyrazolo[1,5-<i>a</i>]pyrimidin-6-amine <b>H1</b></h5><div class="NLM_p last">To a stirred suspension of (<i>S</i>)-4-(2-bromo-6-nitropyrazolo[1,5-<i>a</i>]pyrimidin-7-yl)-3-methylmorpholine (110 mg, 0.32 mmol) in 1.25 M HCl in EtOH (5.1 mL, 6.4 mmol) was added SnCl<sub>2</sub> (305 mg, 1.6 mmol) at rt, and the mixture was stirred for 60 min. The solvent was removed in vacuo, and the residue was taken up in DCM. The mixture was basified with 0.5 M aq. NaOH. Solids were filtered off through a pad of hyflo and washed thoroughly with DCM. The organic phase of the filtrate was separated, and the aqueous phase was extracted with DCM. The combined organic phases were washed with water and brine, dried over anhydr. Na<sub>2</sub>SO<sub>4</sub>, and filtered, and the solvent was removed in vacuo to give the title compound (102 mg, 0.32 mmol, quant.). UPLC-MS: <i>t</i><sub>R</sub> = 0.83 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 314.0. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.33 (s, 1H), 6.72 (s, 1H), 5.14 (s, 2H), 3.95 (tt, <i>J</i> = 6.4, 3.1 Hz, 1H), 3.86–3.70 (m, 3H), 3.59 (td, <i>J</i> = 11.0, 3.7 Hz, 1H), 3.29–3.30 (m, 1H), 2.76 (d, <i>J</i> = 11.6 Hz, 1H), 0.65 (d, <i>J</i> = 6.4 Hz, 3H). UPLC purity: 100%.</div></div><div id="sec4_1_3_36" class="NLM_sec NLM_sec_level_4"><div id="ac_i66" class="anchor-spacer"></div><h5 class="article-section__title" id="_i66"> (<i>S</i>)-1-(2-Bromo-7-(3-methylmorpholino)pyrazolo[1,5-<i>a</i>]pyrimidin-6-yl)-3-(5-chloro-6-methoxypyridin-3-yl)urea <b>23</b></h5><div class="NLM_p last">To an ice-cold solution of 5-chloro-6-methoxypyridin-3-amine (42 mg, 0.26 mmol) in THF (2 mL) was added pyridine (27 μL, 0.33 mmol), followed by 4-fluorophenyl chloroformate (38 μL, 0.29 mmol). After 5 min, the reaction mixture was slowly warmed to rt and stirred for 1 h. A solution of (<i>S</i>)-2-bromo-7-(3-methylmorpholino)pyrazolo[1,5-<i>a</i>]pyrimidin-6-amine (55 mg, 0.18 mmol) in pyridine (2 mL) was added to the mixture, which was heated for 20 h at 70 °C. The solvent was removed in vacuo to give the crude product. The solid was crystallized from a mixture of chloroform, diisopropylether, and heptane to give the title compound (22 mg, 0.04 mmol, 23%). UPLC-MS: <i>t</i><sub>R</sub> = 1.07 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 496.1/498.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.22 (s, 1H), 8.57 (s, 1H), 8.24 (s, 1H), 8.12 (dd, <i>J</i> = 13.3, 2.4 Hz, 2H), 6.86 (s, 1H), 4.18 (s, 1H), 3.90 (s, 3H), 3.87–3.84 (m, 2H), 3.71–3.69 (m, 1H), 3.53–3.49 (dd, <i>J</i> = 11.3, 5.0 Hz, 2H), 3.43–3.39 (m, 1H), 1.04 (d, <i>J</i> = 6.5 Hz, 3H). UPLC purity: 100%.</div></div><div id="sec4_1_3_37" class="NLM_sec NLM_sec_level_4"><div id="ac_i67" class="anchor-spacer"></div><h5 class="article-section__title" id="_i67"> 4-(2-Chloro-6-nitropyrazolo[1,5-<i>a</i>]pyrimidin-7-yl)-3,3-dimethylmorpholine <b>G2</b></h5><div class="NLM_p last">A stirred suspension of 2-chloro-6-nitropyrazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (600 mg, 2.80 mmol) in POCl<sub>3</sub> (10.9 mL, 117 mmol) under argon was cooled to 0 °C, and DIPEA (1.1 mL, 6.2 mmol) was added dropwise under vigorous stirring. The ice-bath was removed after 5 min, and the mixture was heated for 2 h at 100 °C. Excess of POCl<sub>3</sub> was removed in vacuo to dryness, and the remaining residue was dissolved in DCM (30 mL) to give <b>solution 1</b>. A separate flask equipped with a dropping funnel and thermometer was charged under argon with 3,3-dimethylmorpholine (403 mg, 3.5 mmol), TEA (5.0 mL, 36.4 mmol), and DMAP (3 mg) in DCM (30 mL). The mixture was cooled to −10 °C, and <b>solution 1</b> was added dropwise <i>via</i> the dropping funnel over 10 min. The reaction mixture was stirred for 30 min at −10 °C and warmed to rt. The solvent was removed in vacuo, and the residue was taken up in EtOAc. Solids were removed by filtration and washed with EtOAc. The organic phase was washed with sat. aq. NaHCO<sub>3</sub> and brine, dried over anhydr. Na<sub>2</sub>SO<sub>4</sub>, and filtered, and the solvent was removed in vacuo. The crude product was purified by flash column chromatography on silica (<i>n</i>-hexane/EtOAc) to give the title compound (498 mg, 1.60 mmol, 56%). UPLC-MS: <i>t</i><sub>R</sub> = 1.10 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 312.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.96 (s, 1H), 7.05 (s, 1H), 3.98 (s, 1H), 3.83 (t, <i>J</i> = 14.1 Hz, 2H), 3.49 (d, <i>J</i> = 5.2 Hz, 2H), 1.31 (s, 3H), 1.18 (s, 3H). (one proton not visible) UPLC purity: 100%.</div></div><div id="sec4_1_3_38" class="NLM_sec NLM_sec_level_4"><div id="ac_i68" class="anchor-spacer"></div><h5 class="article-section__title" id="_i68"> 2-Chloro-7-(3,3-dimethylmorpholino)pyrazolo[1,5-<i>a</i>]pyrimidin-6-amine <b>H2</b></h5><div class="NLM_p last">To a stirred suspension of 4-(2-chloro-6-nitropyrazolo[1,5-<i>a</i>]pyrimidin-7-yl)-3,3-dimethylmorpholine (498 mg, 1.60 mmol) in 1.25 M HCl in EtOH (25.5 mL, 32 mmol) was added SnCl<sub>2</sub> (1.51 g, 7.98 mmol) in portions at rt, and the mixture was stirred for 90 min. The solvent was removed in vacuo, and the residue was taken up in DCM. The mixture was basified with 0.5 M aq. NaOH. Solids were filtered off through a pad of hyflo and washed thoroughly with DCM. The organic phase of the filtrate was separated, and the aqueous phase was extracted with DCM. The combined organic phases were washed with water and brine, dried over anhydr. Na<sub>2</sub>SO<sub>4</sub>, and filtered, and the solvent was removed in vacuo to give the title compound (441 mg, 1.55 mmol, 97%), which was used without further purification. UPLC-MS: <i>t</i><sub>R</sub> = 0.93 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 282.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.32 (s, 1H), 6.61 (s, 1H), 5.12 (s, 2H), 4.00 (ddd, <i>J</i> = 12.4, 11.0, 3.4 Hz, 1H), 3.88 (dt, <i>J</i> = 10.8, 2.7 Hz, 1H), 3.72 (td, <i>J</i> = 10.9, 3.0 Hz, 1H), 3.62 (d, <i>J</i> = 10.5 Hz, 1H), 3.44 (d, <i>J</i> = 10.5 Hz, 1H), 2.57–2.52 (m, 1H), 1.33 (s, 3H), 1.01 (s, 3H). UPLC purity: 97%.</div></div><div id="sec4_1_3_39" class="NLM_sec NLM_sec_level_4"><div id="ac_i69" class="anchor-spacer"></div><h5 class="article-section__title" id="_i69"> 1-(5-Chloro-6-methoxypyridin-3-yl)-3-(2-chloro-7-(3,3-dimethylmorpholino)pyrazolo[1,5-<i>a</i>]pyrimidin-6-yl)urea <b>24</b></h5><div class="NLM_p last">To an ice-cold solution of 5-chloro-6-methoxypyridin-3-amine (78 mg, 0.45 mmol) in THF (2 mL) was added pyridine (52 μL, 0.65 mmol), followed by 4-fluorophenyl chloroformate (61 μL, 0.47 mmol). After 5 min, the reaction mixture was slowly warmed to rt and stirred for 90 min. A solution of 2-chloro-7-(3,3-dimethylmorpholino)pyrazolo[1,5-<i>a</i>]pyrimidin-6-amine (120 mg, 0.41 mmol) in pyridine (2 mL) was added to the mixture, which was heated for 48 h at 70 °C. The solvent was removed in vacuo, and the crude product was purified twice by flash column chromatography on silica (cylcohexane/EtOAc) to give the title compound (74 mg, 0.16 mmol, 37%). UPLC-MS: <i>t</i><sub>R</sub> = 1.12 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 466.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 9.43–9.35 (m, 1H), 8.81–8.72 (m, 1H), 8.17–8.07 (m, 3H), 6.86–6.78 (m, 1H), 3.93–3.89 (m, 5H), 3.80–3.70 (m, 1H), 3.58–3.41 (m, 2H), 2.96–2.85 (m, 1H), 1.33–1.28 (m, 3H), 1.08–0.98 (m, 3H). UPLC purity: 100%.</div></div><div id="sec4_1_3_40" class="NLM_sec NLM_sec_level_4"><div id="ac_i70" class="anchor-spacer"></div><h5 class="article-section__title" id="_i70"> 1-(2-Chloro-7-(3,3-dimethylmorpholino)pyrazolo[1,5-<i>a</i>]pyrimidin-6-yl)-3-(2-(trifluoromethyl)pyridin-4-yl)urea <b>25</b></h5><div class="NLM_p last">To an ice-cold solution of 2-(trifluoromethyl)pyridin-4-amine (572 mg, 3.53 mmol) in THF (18 mL) was added pyridine (389 μL, 4.8 mmol), followed by 4-fluorophenyl chloroformate (485 μL, 3.7 mmol). After 5 min, the reaction mixture was slowly warmed to rt and stirred for 4.5 h. A solution of 2-chloro-7-(3,3-dimethylmorpholino)pyrazolo[1,5-<i>a</i>]pyrimidin-6-amine (904 mg, 3.2 mmol) in pyridine (18 mL) was added to the mixture, which was heated for 16 h at 70 °C. The solvent was removed in vacuo, the residue was dissolved in DCM, washed with water, sat. aq. NaHCO<sub>3</sub>, and brine, and filtered, and the solvent was removed in vacuo. The crude product was purified by flash column chromatography on silica (DCM/TBME) to give the title compound (1.02 g, 2.2 mmol, 67%). UPLC-MS: <i>t</i><sub>R</sub> = 1.11 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 470.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.08 (s, 1H), 8.74 (s, 1H), 8.56 (d, <i>J</i> = 5.7 Hz, 1H), 8.38 (s, 1H), 8.06 (d, <i>J</i> = 2.1 Hz, 1H), 7.66 (dd, <i>J</i> = 5.7, 2.1 Hz, 1H), 6.86 (s, 1H), 3.99–3.82 (m, 2H), 3.79–3.67 (m, 1H), 3.54–3.41 (m, 2H), 2.93 (dt, <i>J</i> = 13.0, 3.1 Hz, 1H), 1.31 (s, 3H), 1.03 (s, 3H). UPLC purity: 100%.</div></div><div id="sec4_1_3_41" class="NLM_sec NLM_sec_level_4"><div id="ac_i71" class="anchor-spacer"></div><h5 class="article-section__title" id="_i71"> (3<i>S</i>,5<i>S</i>)-4-(2-Chloro-6-nitropyrazolo[1,5-<i>a</i>]pyrimidin-7-yl)-3,5-dimethylmorpholine <b>G3</b></h5><div class="NLM_p last">A stirred suspension of 2-chloro-6-nitropyrazolo[1,5-<i>a</i>]pyrimidin-7(4H)-one (700 mg, 3.26 mmol) in POCl<sub>3</sub> (12.7 mL, 137 mmol) under argon was cooled to 0 °C, and DIPEA (1.27 mL, 7.24 mmol) was added dropwise under vigorous stirring. The ice-bath was removed after 5 min, and the mixture was heated for 90 min at 100 °C. Excess of POCl<sub>3</sub> was removed in vacuo to dryness, and the remaining residue was dissolved in DCM (50 mL) to give <b>solution 1</b>. A separate flask equipped with a dropping funnel and thermometer was charged under argon with (3<i>S</i>,5<i>S</i>)-3,5-dimethylmorpholine (413 mg, 3.6 mmol), TEA (5.9 mL, 42.4 mmol), and DMAP (4 mg) in DCM (50 mL). The mixture was cooled to −10 °C and <b>solution 1</b> was added dropwise <i>via</i> the dropping funnel over 10 min. The reaction mixture was stirred for 30 min at −10 °C and warmed to rt. The solvent was removed in vacuo, and the residue was taken up in EtOAc. Solids were removed by filtration and washed with EtOAc. The organic phase was washed with sat. aq. NaHCO<sub>3</sub> and brine, dried over anhydr. Na<sub>2</sub>SO<sub>4</sub>, and filtered, and the solvent was removed in vacuo. The crude product was purified by flash column chromatography on silica (<i>n</i>-hexane/TBME) to give the title compound (392 mg, 1.625 mmol, 38%). UPLC-MS: <i>t</i><sub>R</sub> = 1.10 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 312.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.09 (s, 1H), 7.06 (s, 1H), 4.60 (dq, <i>J</i> = 7.4, 4.2, 3.7 Hz, 1H), 4.01 (dd, <i>J</i> = 11.2, 3.1 Hz, 1H), 3.95 (dd, <i>J</i> = 11.3, 3.6 Hz, 1H), 3.77 (ddt, <i>J</i> = 8.8, 6.1, 3.0 Hz, 1H), 3.67 (dd, <i>J</i> = 11.2, 3.1 Hz, 1H), 3.29 (s, 1H), 1.22 (d, <i>J</i> = 6.7 Hz, 3H), 0.73 (d, <i>J</i> = 6.1 Hz, 3H). UPLC purity: 100%.</div></div><div id="sec4_1_3_42" class="NLM_sec NLM_sec_level_4"><div id="ac_i72" class="anchor-spacer"></div><h5 class="article-section__title" id="_i72"> 2-Chloro-7-((3<i>S</i>,5<i>S</i>)-3,5-dimethylmorpholino)pyrazolo[1,5-<i>a</i>]pyrimidin-6-amine <b>H3</b></h5><div class="NLM_p last">To a stirred suspension of (3<i>S</i>,5<i>S</i>)-4-(2-chloro-6-nitropyrazolo[1,5-<i>a</i>]pyrimidin-7-yl)-3,5-dimethylmorpholine (392 mg, 1.25 mmol) in 1.25 M HCl in EtOH (20.1 mL, 25.2 mmol) was added SnCl<sub>2</sub> (1.19 g, 6.3 mmol) in portions at rt, and the mixture was stirred for 60 min. The solvent was removed in vacuo, and the residue was taken up in DCM. The mixture was basified with 0.5 M aq. NaOH. Solids were filtered off through a pad of hyflo and washed thoroughly with DCM. The organic phase of the filtrate was separated, and the aqueous phase was extracted with DCM. The combined organic phases were washed with water and brine, dried over anhydr. Na<sub>2</sub>SO<sub>4</sub>, and filtered, and the solvent was removed in vacuo to give the title compound (357 mg, 1.25 mmol, quant.), which was used without further purification. UPLC-MS: <i>t</i><sub>R</sub> = 0.90 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 282.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.31 (s, 1H), 6.60 (s, 1H), 4.97 (s, 2H), 4.26 (ddd, <i>J</i> = 9.8, 6.3, 3.5 Hz, 1H), 3.94 (dd, <i>J</i> = 10.9, 3.1 Hz, 1H), 3.82 (dd, <i>J</i> = 10.8, 3.5 Hz, 1H), 3.52 (dd, <i>J</i> = 10.9, 2.0 Hz, 1H), 3.28–3.23 (m, 2H), 1.06 (d, <i>J</i> = 6.7 Hz, 3H), 0.57 (d, <i>J</i> = 6.2 Hz, 3H). UPLC purity: 99%.</div></div><div id="sec4_1_3_43" class="NLM_sec NLM_sec_level_4"><div id="ac_i73" class="anchor-spacer"></div><h5 class="article-section__title" id="_i73"> 1-(5-Chloro-6-methoxypyridin-3-yl)-3-(2-chloro-7-((3<i>S</i>,5<i>S</i>)-3,5-dimethylmorpholino)pyrazolo[1,5-<i>a</i>]pyrimidin-6-yl)urea <b>26</b></h5><div class="NLM_p last">To an ice-cold solution of 5-chloro-6-methoxypyridin-3-amine (135 mg, 0.85 mmol) in THF (4 mL) was added pyridine (92 μL, 1.1 mmol), followed by 4-fluorophenyl carbonochloridate (131 μL, 0.99 mmol). After 5 min the reaction mixture was slowly warmed to rt and stirred for 1 h. A solution of 2-chloro-7-((3<i>S</i>,5<i>S</i>)-3,5-dimethylmorpholino)pyrazolo[1,5-<i>a</i>]pyrimidin-6-amine (200 mg, 0.71 mmol) in pyridine (4 mL) was added to the mixture, which was heated for 42 h at 70 °C. The solvent was removed in vacuo, and the crude product was purified by flash column chromatography on silica (<i>n</i>-hexane/TBME) to give the title compound (<b>26</b>) (99 mg, 0.21 mmol, 30%). UPLC-MS: <i>t</i><sub>R</sub> = 1.11 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 466.0. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.33 (s, 1H), 8.72 (s, 1H), 8.21–8.05 (m, 2H), 7.96 (s, 1H), 6.83 (s, 1H), 4.26 (td, <i>J</i> = 6.9, 3.2 Hz, 1H), 3.99–3.82 (m, 5H), 3.57 (dq, <i>J</i> = 9.3, 5.7, 4.8 Hz, 1H), 3.48 (dd, <i>J</i> = 11.1, 4.2 Hz, 1H), 3.42–3.34 (m, 1H), 1.01 (d, <i>J</i> = 6.3 Hz, 3H), 0.80 (d, <i>J</i> = 6.3 Hz, 3H). UPLC purity: 100%.</div></div><div id="sec4_1_3_44" class="NLM_sec NLM_sec_level_4"><div id="ac_i74" class="anchor-spacer"></div><h5 class="article-section__title" id="_i74"> 1-(2-Chloro-7-((3<i>S</i>,5<i>S</i>)-3,5-dimethylmorpholino)pyrazolo[1,5-<i>a</i>]pyrimidin-6-yl)-3-(2-(trifluoromethyl)pyridin-4-yl)urea <b>27</b>, <b>MLT-231</b></h5><div class="NLM_p">To an ice-cold solution of 2-(trifluoromethyl)pyridin-4-amine (1.21 g, 7.2 mmol) in THF (40 mL) was added pyridine (797 μL, 9.8 mmol), followed by 4-fluorophenyl chloroformate (992 μL, 7.6 mmol). After 10 min, the reaction mixture was slowly warmed to rt and stirred for 16 h. A solution of 2-chloro-7-((3<i>S</i>,5<i>S</i>)-3,5-dimethylmorpholino)pyrazolo[1,5-<i>a</i>]pyrimidin-6-amine (1.85 g, 6.6 mmol) in pyridine (40 mL) was added to the mixture, which was heated for 4 days at 70 °C. The solvent was removed in vacuo, the residue was dissolved in DCM and washed with brine, dried over anhydr. Na<sub>2</sub>SO<sub>4</sub>, and filtered, and the solvent was removed in vacuo. The crude product was purified by crystallization from MeOH/water to give the title compound (2.3 g, 4.9 mmol, 68%). UPLC-MS: <i>t</i><sub>R</sub> = 1.08 min. MS (ESI) [M<sup>+</sup>] <i>m</i>/<i>z</i>: 470.2. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.03 (s, 1H), 8.70s (s, 1H), 8.56 (d, <i>J</i> = 5.6 Hz, 1H), 8.21 (s, 1H), 8.07 (d, <i>J</i> = 2.1 Hz, 1H), 7.62 (dd, <i>J</i> = 5.7, 2.0 Hz, 1H), 6.86 (s, 1H), 4.25 (pd, <i>J</i> = 6.6, 3.2 Hz, 1H), 3.88 (ddd, <i>J</i> = 11.1, 7.7, 3.3 Hz, 2H), 3.60 (tq, <i>J</i> = 6.7, 3.7, 3.2 Hz, 1H), 3.47 (dd, <i>J</i> = 11.2, 4.3 Hz, 1H), 3.33 (s, 1H), 1.00 (d, <i>J</i> = 6.5 Hz, 3H), 0.80 (d, <i>J</i> = 6.4 Hz, 3H).</div><div class="NLM_p last"><sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 153.12 (1C s), 151.23 (s), 150.88 (s), 148.81 (1C s), 148.06 (1C s), 144.60 (1C s), 142.46 (1C s), 121.57 (1C q(coupling to 19F), 117.33 (1C s),114.86 (1C s), 108.89 (1C s), 95.19 (1C s), 71.83 (1C s), 71.43 (1C s), 51.24 (1C s), 47.25 (1C s), 15.60 (1C s), 14.96 (1C s) (1 signal not visible). UPLC purity: 100%.</div></div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> Biology</h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> Cell Culture</h4><div class="NLM_p last">BJAB and TMD8<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> cells were cultured in RPMI 1640 medium with 10% FCS, OCI-Ly3 in RPMI 1640 GlutaMAX with 20% FCS, and OCI-Ly10 in IMDM with 20% human serum.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> MALT1-Dependent Mechanistic Assays</h4><div class="NLM_p last">The MALT1 biochemical assay, the cIAP2-MALT1-driven NF-κB reporter gene assay (RGA) in HEK293 cells, the human IL2-promoter RGA in Jurkat T cells (clone K22 290_H23), the MALT1-dependent BCL10 cleavage assay in OCI-Ly3 cells, the OCI-Ly3/BJAB cell proliferation assays, the mixed lymphocyte reaction, and the murine bone marrow proliferation assays were performed as described previously.<a onclick="showRef(event, 'ref8 ref9 ref20'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref20">(8,9,20)</a></div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> DLBCL Soft-Agar Assay</h4><div class="NLM_p last">Using BJAB or OCI-Ly3 cell lines, 5000 cells were seeded in 100 mL/well of medium containing 0.4% of 2-hydroxyethylagarose (Sigma-Aldrich) in Ultra-Low Attachment 96-well plate (Costar). Low melting agarose was melted and then cooled to 42 °C before being mixed with the cells. After agarose polymerization, the plate was incubated at 37 °C overnight. 50 mL/well of 3× concentrated compound was added to the cells and incubated at 37 °C in an incubator for 7 days and 14 days for BJAB cells and OCI-Ly3 cells, respectively. Colony formation imaging was performed with a Clone Select Imager (Genetix). To assess cell viability, 15 mL/well of 130 mg/mL resazurin solution (Sigma-Aldrich) was added to the cells and incubated for 1–24 h at 37 °C, depending on cellular metabolic activity. The fluorescent signal was measured at 560/15 nm (exc) and 590/20 nm (em) with a Synergy HT plate reader. Averages of triplicates were calculated for each compound concentration after blank and day 1 value removal. Percentage relative proliferation was determined using the mean value of vehicle-treated cells. The IC<sub>50</sub> values were determined using the Dose response one site 205 model in XLFit software.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> Western Blot Analysis</h4><div class="NLM_p last">Immunoblotting analysis of OCI-Ly3 cells was performed as described previously.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Briefly, 1 × 10<sup>6</sup> OCI-Ly3 cells/mL cells were cultured in RPMI 1640 media with 10% FCS with the desired concentrations of MLT-231 and harvested 24 h post-treatment. Cells were collected, lysed in NP-40 lysis buffer (50 mM Tris-HCl, pH 7.5, 150 mM NaCl, 1.0% NP-40) supplemented with complete protease inhibitor cocktail and phosSTOP phosphatase inhibitor cocktail (Roche Life Science), and clarified by centrifugation. Proteins were separated on 4–12% Bis-Tris SDS–PAGE gradient gels (Life Technologies) and detected by immunoblotting using the following monoclonal and polyclonal antibodies: CYLD, BCL10, RELB, IRF4 (Cell Signaling Technology), and Tubulin (Sigma-Aldrich).</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> Methods for T-Dependent B Cell Responses in Mice</h4><div class="NLM_p">The impacts of MLT-231 on T-dependent (TD) B cell responses in mice were assessed as previously described.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Briefly, keyhole limpet hemocyanin (KLH; Calbiochem) conjugated to DNP (Sigma-Aldrich) was prepared, at a ratio of 1/20, as DNP-KLH stock solution (5 mg/mL) and stored at 20 °C. For induction of TD responses, DNP-KLH was adsorbed onto colloidal aluminum hydroxide (Alu-Gel-S, SERVA: 1 mL DNP-KLH/5 mL Alu-Gel-S/4 mL PBS) and injected i.p. into mice at 100 mg/0.2 mL/mouse.</div><div class="NLM_p">Mice were treated by oral gavage either twice daily (12 h interval) with vehicle or MLT-231 at 10, 30, or 100 mg/kg or once daily with CsA at 30 mg/kg. At 1 h after the first treatment, all mice were injected i.p. with DNP-KLH and monitored until termination after 8 days. Prior to termination, at 12 h after the last treatment, blood samples were collected from the tail vein in order to assess the serum anti-DNP-specific IgG and IgM levels as well as the trough drug blood levels. Antibody titers were measured by ELISA and are expressed as mean ± SEM log 10 dilution values using 50% of the maximal extinction at 405 nm as the endpoint.</div><div class="NLM_p last">An LC–MS/MS method was used to detect MLT-231 in blood. Aliquots (30 μL) of blood samples from mice treated with vehicle or MLT-231 were first spiked with 3 μL of internal standard solution and subsequently deproteinated by adding acetonitrile (150 μL) and centrifuged. A 150 μL aliquot of the resulting supernatant was transferred to a 96-microwell plate, and a 5 μL aliquot of the supernatant was directly injected onto LC–MS/MS.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> Mouse Xenograft Experiments</h4><div class="NLM_p last">Subcutaneous tumors were induced by injecting cells (TMD8 5 × 10<sup>6</sup>) in HBSS containing 50% BD Matrigel or transplantation of tumor fragments from healthy tissue (OCI-Ly10 2 × 2 × 2 mm) in the flank of Scid-beige mice (Taconic). MLT-231 was formulated in 20% propylene-glycol, 20% cremophor ELP, 5% poloxamer-188 (aka Lutrol-68), and 55% water (w/v) and administered orally. For PK/PD studies, treatment was started after randomizing mice into groups of <i>n</i> = 3 for vehicle and dose groups, when the average tumor size had reached approximately 500 mm<sup>3</sup>. Animals from each group were sacrificed at indicated timepoints post treatment. Plasma and tumors were sampled and snap-frozen. For efficacy studies, treatment was initiated when tumors reached a mean tumor volume of around 120–150 mm<sup>3</sup>. At this point, animals were randomized into different treatment groups based on similar tumor size and body weight. The tumor size was measured twice a week with a caliper. The tumor volume was calculated using the formula (length × width) × π/6 and expressed in mm<sup>3</sup>. Data are presented as mean ± SEM. Statistical analysis was performed on ΔTVol by comparing the treatment groups to the vehicle control group at end of study by Kruskal–Wallis, followed by Dunn’s post hoc test. At indicated timepoints post last dose, plasma and tumors were sampled and snap-frozen in liquid nitrogen. Concentrations of MLT-231 in plasma and tumor were determined using UPLC-MS/MS.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> Statement</h3><div class="NLM_p last">Herewith, the authors confirm that all experimental procedures involving animal studies strictly adhered to the Association for Assessment and Accreditation of Laboratory Animal Care International guidelines as published in the Guide for the Care and Use of Laboratory Animals and to Novartis Corporate Animal Welfare policies.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> Crystallization and Structure Determination</h3><div class="NLM_p">Expression and purification of MALT1<sup>Casp1-Ig3</sup> for crystallization and biophysical studies have been described.<a onclick="showRef(event, 'ref20 ref25'); return false;" href="javascript:void(0);" class="ref ref20 ref25">(20,25)</a> MALT1(329–728) at 12 mg/mL, in 25 mM HEPES pH7.5, 50 mM NaCl, and 1 mM TCEP, was mixed with <b>TC1</b> in 100% DMSO at a final concentration of 0.5 mM. TCEP (10 mM) and MgCl<sub>2</sub> (10 mM) were added to the protein before setup. In an Intelli-Plate (Art Robbins Instruments), 300 nL of protein solution was mixed with 200 nL of crystallization solution and 100 nL of seeding solution. Seeds were prepared using crystals of a nonsufficient diffraction quality of the same compound. Best crystals were obtained at 293 K from 0.1 M Bis-Tris pH 5.5, 0.3 M Magnesium Formate.</div><div class="NLM_p">The crystals of the MALT1(329–728)/<b>TC2</b> complex were generated using the same protocol. The best crystals were obtained at 293 K from 0.1 M Tris pH8.5, 25% PEG 3350, 0.2 M MgCl<sub>2</sub>.</div><div class="NLM_p">Crystals were briefly incubated in a cryo buffer (70% crystallization solution/30% glycerol) and flash-cooled in liquid nitrogen. X-ray diffraction experiments were performed at the Swiss Light Source (Villigen, Switzerland) on beamline PXII equipped with a Pilatus Pixel Detector (Dectris, Baden, Switzerland).</div><div class="NLM_p last">Data collection and refinement statistics are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01245/suppl_file/jm0c01245_si_001.pdf" class="ext-link">Table S1</a> found in the Supporting Information.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i84"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01245" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34410" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34410" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01245?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01245</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Data collection and refinement statistics and mechanistic cIAP2-MALT1-dependent NF-κB reporter gene assay (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01245/suppl_file/jm0c01245_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings, biochemical, cellular potency data, and solubility (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01245/suppl_file/jm0c01245_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01245/suppl_file/jm0c01245_si_001.pdf">jm0c01245_si_001.pdf (844.43 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01245/suppl_file/jm0c01245_si_002.csv">jm0c01245_si_002.csv (1.98 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Protein Data Bank codes are the following: <b>TC1</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AK0">7AK0</a>; <b>TC2</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AK1">7AK1</a></p><div class="testing" data-doi="10.1021/acs.jmedchem.0c01245" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58406" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58406" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carole Pissot Soldermann</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7497-3839" title="Orcid link">http://orcid.org/0000-0002-7497-3839</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#305351425f5c551e405943435f44705e5f4651424459431e535f5d"><span class="__cf_email__" data-cfemail="94f7f5e6fbf8f1bae4fde7e7fbe0d4fafbe2f5e6e0fde7baf7fbf9">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Oliver Simic</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#fa9596938c9f88d48993979399ba94958c9b888e9389d4999597"><span class="__cf_email__" data-cfemail="9cf3f0f5eaf9eeb2eff5f1f5ffdcf2f3eafdeee8f5efb2fff3f1">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Catherine H. Régnier</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#2d4e4c5945485f444348035f484a4344485f6d43425b4c5f59445e034e4240"><span class="__cf_email__" data-cfemail="0c6f6d7864697e656269227e696b6265697e4c62637a6d7e78657f226f6361">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin Renatus</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paulus Erbel</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Samu Melkko</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Markus Wartmann</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marc Bigaud</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andreas Weiss</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul McSheehy</span> - <span class="hlFld-Affiliation affiliation">Basilea Pharmaceutica, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ralf Endres</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span>; 
    <span class="hlFld-Affiliation affiliation">Basilea Pharmaceutica, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paulo Santos</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span>; 
    <span class="hlFld-Affiliation affiliation">Basilea Pharmaceutica, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jutta Blank</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ansgar Schuffenhauer</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6385-0414" title="Orcid link">http://orcid.org/0000-0001-6385-0414</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guido Bold</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicole Buschmann</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas Zoller</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Eva Altmann</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2749-6347" title="Orcid link">http://orcid.org/0000-0002-2749-6347</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul W. Manley</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ina Dix</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elisabeth Buchdunger</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Julien Scesa</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jean Quancard</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Achim Schlapbach</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Frédéric Bornancin</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, Novartis Campus, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas Radimerski</span> - <span class="hlFld-Affiliation affiliation">Basilea Pharmaceutica, CH-4002 Basel, Switzerland</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i87">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88080" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88080" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We would like to thank Drs. Marc Lang, René Hersperger, Christian Bruns, Francesco Hofmann, Joerg Eder, and Richard Sedrani for their support and insightful discussions. We also thank Anna Bernardi, Lilian Steiner, Daniela Kaatz, Markus Gaugler, Martin Kessler, Werner Vetterli, Ursula Duerler, Bernard Glasser, Fanny Marque, Anne Cecile D’Alessandro, Joelle Rubert, Rita Andraos-Rey, Danielle Boesch, Frederique Lafossas, Cindy Schmid, Corinne Vedrine, Christine Stefan, Andreas Theuer, Alexandra Buhles, Daniel Weiss, Adeline Unterreiner, and Claire Malinverni for their expert technical support. The TMD8 cell line was kindly provided by Professor Shuji Tohda from the University of Tokyo, Japan.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">CL</td><td class="NLM_def"><p class="first last">clearance</p></td></tr><tr><td class="NLM_term">Clint</td><td class="NLM_def"><p class="first last">intrinsic clearance</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last">diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DLBCL</td><td class="NLM_def"><p class="first last">diffuse large B-cell lymphoma</p></td></tr><tr><td class="NLM_term">Et<sub>3</sub>N</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr><tr><td class="NLM_term">EtOH</td><td class="NLM_def"><p class="first last">ethanol</p></td></tr><tr><td class="NLM_term">HCl</td><td class="NLM_def"><p class="first last">hydrochloric acid</p></td></tr><tr><td class="NLM_term">MC</td><td class="NLM_def"><p class="first last">methylcellulose</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">MLM</td><td class="NLM_def"><p class="first last">mouse liver microsomes</p></td></tr><tr><td class="NLM_term">Papp</td><td class="NLM_def"><p class="first last">apparent permeability</p></td></tr><tr><td class="NLM_term">PMA</td><td class="NLM_def"><p class="first last">phorbol 12-myristate 13-acetate</p></td></tr><tr><td class="NLM_term">RLM</td><td class="NLM_def"><p class="first last">rat liver microsomes</p></td></tr><tr><td class="NLM_term">SD</td><td class="NLM_def"><p class="first last">standard deviation</p></td></tr><tr><td class="NLM_term">TBME</td><td class="NLM_def"><p class="first last"><i>tert</i>-butyl methylether</p></td></tr><tr><td class="NLM_term">TEA</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i89">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31566" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31566" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 25 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Uren, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Rourke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aravind, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pisabarro, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seshagiri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koonin, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixit, V. M.</span></span> <span> </span><span class="NLM_article-title">Identification of Paracaspases and Metacaspases: Two Ancient Families of Caspase-Like Proteins, One of which Plays a Key Role in MALT Lymphoma</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">961</span>– <span class="NLM_lpage">967</span>, <span class="refDoi"> DOI: 10.1016/s1097-2765(00)00094-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1016%2Fs1097-2765%2800%2900094-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=11090634" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnvVentLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2000&pages=961-967&author=A.+G.+Urenauthor=K.+O%E2%80%99Rourkeauthor=L.+Aravindauthor=M.+T.+Pisabarroauthor=S.+Seshagiriauthor=E.+V.+Kooninauthor=V.+M.+Dixit&title=Identification+of+Paracaspases+and+Metacaspases%3A+Two+Ancient+Families+of+Caspase-Like+Proteins%2C+One+of+which+Plays+a+Key+Role+in+MALT+Lymphoma&doi=10.1016%2Fs1097-2765%2800%2900094-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of paracaspases and metacaspases: two ancient families of caspase-like proteins, one of which plays a key role in MALT lymphoma</span></div><div class="casAuthors">Uren, Anthony G.; O'Rourke, Karen; Aravind, L.; Pisabarro, M. Teresa; Seshagiri, Somasekar; Koonin, Eugene V.; Dixi, Vishva M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">961-967</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Caspases are cysteine proteases essential to apoptosis.  The authors have identified two families of caspase-like proteins, Paracaspases (found in metazoans and Dictyostelium) and metacaspases (found in plants, fungi, and protozoa).  Metazoan paracaspase prodomains contain a death domain and Ig domains.  Several plant metacaspase prodomains contain zinc finger motifs resembling those in the plant hypersensitive response/cell death protein Isd-1.  The human paracaspase prodomain binds Bcl10, a protein involved in the t(1;14)(p22;q32) translocation of mucosa-assocd. lymphoid tissue (MALT) lymphoma.  Another MALT lymphoma translocation, t(11;18)(q21;q21), fuses the IAP-2 gene to the MLT1/MALT1 locus, which encodes the human paracaspase.  The authors find that this fusion activates NF-κB and that the caspase domain is required for this function, since mutation of the conserved catalytic cysteine attenuates NF-κB activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqMj67zUtB4LVg90H21EOLACvtfcHk0lhXdk96SKVqcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnvVentLc%253D&md5=c62e4cee69f1574d23182e52369200ca</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fs1097-2765%2800%2900094-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs1097-2765%252800%252900094-0%26sid%3Dliteratum%253Aachs%26aulast%3DUren%26aufirst%3DA.%2BG.%26aulast%3DO%25E2%2580%2599Rourke%26aufirst%3DK.%26aulast%3DAravind%26aufirst%3DL.%26aulast%3DPisabarro%26aufirst%3DM.%2BT.%26aulast%3DSeshagiri%26aufirst%3DS.%26aulast%3DKoonin%26aufirst%3DE.%2BV.%26aulast%3DDixit%26aufirst%3DV.%2BM.%26atitle%3DIdentification%2520of%2520Paracaspases%2520and%2520Metacaspases%253A%2520Two%2520Ancient%2520Families%2520of%2520Caspase-Like%2520Proteins%252C%2520One%2520of%2520which%2520Plays%2520a%2520Key%2520Role%2520in%2520MALT%2520Lymphoma%26jtitle%3DMol.%2520Cell%26date%3D2000%26volume%3D6%26spage%3D961%26epage%3D967%26doi%3D10.1016%2Fs1097-2765%2800%2900094-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Juilland, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thome, M.</span></span> <span> </span><span class="NLM_article-title">Holding All the CARDs: How MALT1 Controls CARMA/CARD-Dependent Signaling</span>. <i>Front. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1927</span>, <span class="refDoi"> DOI: 10.3389/fimmu.2018.01927</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.3389%2Ffimmu.2018.01927" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=30214442" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFSrsrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1927&author=M.+Juillandauthor=M.+Thome&title=Holding+All+the+CARDs%3A+How+MALT1+Controls+CARMA%2FCARD-Dependent+Signaling&doi=10.3389%2Ffimmu.2018.01927"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Holding all the CARDs: how MALT1 controls CARMA/CARD-dependent signaling</span></div><div class="casAuthors">Juilland, Melanie; Thome, Margot</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Immunology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1927/1-1927/15</span>CODEN:
                <span class="NLM_cas:coden">FIRMCW</span>;
        ISSN:<span class="NLM_cas:issn">1664-3224</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">A review.  The scaffold proteins CARMA1-3 (encoded by the genes CARD11, -14 and -10) and CARD9 play major roles in signaling downstream of receptors with immunoreceptor tyrosine activation motifs (ITAMs), G-protein coupled receptors (GPCR) and receptor tyrosine kinases (RTK).  These receptors trigger the formation of oligomeric CARMA/CARD-BCL10-MALT1 (CBM) complexes via kinases of the PKC family.  The CBM in turn regulates gene expression by the activation of NF-κB and AP-1 transcription factors and controls transcript stability.  The paracaspase MALT1 is the only CBM component having an enzymic (proteolytic) activity and has therefore recently gained attention as a potential drug target.  Here we review recent advances in the understanding of the mol. function of the protease MALT1 and summarize how MALT1 scaffold and protease function contribute to the transmission of CBM signals.  Finally, we will highlight how dysregulation of MALT1 function can cause pathologies such as immunodeficiency, autoimmunity, psoriasis, and cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlFMkGmTP2YrVg90H21EOLACvtfcHk0lhXdk96SKVqcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFSrsrrN&md5=55c17de3618a769f61cde83eee85f343</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3389%2Ffimmu.2018.01927&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffimmu.2018.01927%26sid%3Dliteratum%253Aachs%26aulast%3DJuilland%26aufirst%3DM.%26aulast%3DThome%26aufirst%3DM.%26atitle%3DHolding%2520All%2520the%2520CARDs%253A%2520How%2520MALT1%2520Controls%2520CARMA%252FCARD-Dependent%2520Signaling%26jtitle%3DFront.%2520Immunol.%26date%3D2018%26volume%3D9%26spage%3D1927%26doi%3D10.3389%2Ffimmu.2018.01927" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruland, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartjes, L.</span></span> <span> </span><span class="NLM_article-title">CARD-BCL-10-MALT1 Signalling in Protective and Pathological Immunity</span>. <i>Nat. Rev. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">118</span>– <span class="NLM_lpage">134</span>, <span class="refDoi"> DOI: 10.1038/s41577-018-0087-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1038%2Fs41577-018-0087-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=30467369" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnt1Wlu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2019&pages=118-134&author=J.+Rulandauthor=L.+Hartjes&title=CARD-BCL-10-MALT1+Signalling+in+Protective+and+Pathological+Immunity&doi=10.1038%2Fs41577-018-0087-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">CARD-BCL-10-MALT1 signalling in protective and pathological immunity</span></div><div class="casAuthors">Ruland, Juergen; Hartjes, Lara</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Immunology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">118-134</span>CODEN:
                <span class="NLM_cas:coden">NRIABX</span>;
        ISSN:<span class="NLM_cas:issn">1474-1733</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  CARD protein-BCL-10-MALT1 (CBM) signalosomes are multiprotein signalling platforms that control immune and inflammatory pathways in most tissues.  After exposure to distinct immune triggers, these mols. form self-organizing filaments with MALT1 protease activity to regulate canonical nuclear factor-κB (NF-κB) and mitogen-activated protein kinase (MAPK) signalling pathways and the degrdn. of mRNA-binding proteins, which provides two layers of control of inflammatory gene expression.  These CBM-regulated mechanisms are essential for host defense and tissue homeostasis, and numerous genetic alterations in CBM signalling components have been implicated in inherited and acquired immune-mediated diseases.  This Review discusses the regulation and signalling of CBM complexes, their physiol. roles and their pathophysiol. functions in human immunodeficiency diseases, inflammatory disorders and cancers of the immune system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTdKYas0vj17Vg90H21EOLACvtfcHk0lh3VSy1akMicw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnt1Wlu7Y%253D&md5=64a4447502af72534ff4c955813dbe2b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fs41577-018-0087-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41577-018-0087-2%26sid%3Dliteratum%253Aachs%26aulast%3DRuland%26aufirst%3DJ.%26aulast%3DHartjes%26aufirst%3DL.%26atitle%3DCARD-BCL-10-MALT1%2520Signalling%2520in%2520Protective%2520and%2520Pathological%2520Immunity%26jtitle%3DNat.%2520Rev.%2520Immunol.%26date%3D2019%26volume%3D19%26spage%3D118%26epage%3D134%26doi%3D10.1038%2Fs41577-018-0087-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Du, M.-Q.</span></span> <span> </span><span class="NLM_article-title">MALT lymphoma: A paradigm of NF-κB dysregulation</span>. <i>Semin. Canc. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">60</span>, <span class="refDoi"> DOI: 10.1016/j.semcancer.2016.07.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1016%2Fj.semcancer.2016.07.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=27452667" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1ygsr%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2016&pages=49-60&author=M.-Q.+Du&title=MALT+lymphoma%3A+A+paradigm+of+NF-%CE%BAB+dysregulation&doi=10.1016%2Fj.semcancer.2016.07.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">MALT lymphoma: A paradigm of NF-κB dysregulation</span></div><div class="casAuthors">Du, Ming-Qing</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Cancer Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">49-60</span>CODEN:
                <span class="NLM_cas:coden">SECBE7</span>;
        ISSN:<span class="NLM_cas:issn">1044-579X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Extranodal marginal zone lymphoma of mucosa-assocd. lymphoid tissue (MALT lymphoma) invariably arises from a background of chronic microbial infection and/or autoimmune disorder at diverse mucosal sites.  The prolonged chronic infection and/or autoimmunity generate active immune and inflammatory responses that provide a setting for evolution and development of autoreactive B-cells, their expansion and eventual malignant transformation following acquisition of genetic changes.  The immune responses also play a crit. role in sustaining the growth and survival of the transformed cells as shown by complete regression of a high proportion of MALT lymphoma of the stomach, ocular adnexa and skin following anti-microbial treatment.  B-cell receptor engagement by auto-antigen as well as T-cell help including both cognate interaction and bystander help via sol. ligands such as CD40L and BAFF are thought to underpin the immunol. drive in the lymphoma development through activation of the canonical and non-canonical NF-κB pathway resp.  Similarly, the three MALT lymphoma assocd. chromosome translocations, namely t(1;14)(p22;q32)/BCL10-IGH, t(14;18)(q32;q21)/IGH-MALT1,and t(11;18)(q21;q21)/BIRC3 (API2)-MALT1, are also capable of activating both canonical and non-canonical NF-κB pathways.  Furthermore, TNFAIP3 (A20) inactivation by deletion and/or mutation abolishes the auto-neg. feedback to several signalling including BCR and TLR, which connect to the canonical NF-κB activation pathway.  Thus, there is a considerable overlap in the mol. pathways dysregulated by immunol. drive and somatic genetic changes, strongly arguing for their oncogenic cooperation in the development of MALT lymphoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4mYwLSwRG5rVg90H21EOLACvtfcHk0lhLGM07TqNk-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1ygsr%252FM&md5=400f8c53b8fb19e314929c53955a27b6</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.semcancer.2016.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.semcancer.2016.07.003%26sid%3Dliteratum%253Aachs%26aulast%3DDu%26aufirst%3DM.-Q.%26atitle%3DMALT%2520lymphoma%253A%2520A%2520paradigm%2520of%2520NF-%25CE%25BAB%2520dysregulation%26jtitle%3DSemin.%2520Canc.%2520Biol.%26date%3D2016%26volume%3D39%26spage%3D49%26epage%3D60%26doi%3D10.1016%2Fj.semcancer.2016.07.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staudt, L. M.</span></span> <span> </span><span class="NLM_article-title">Aggressive Lymphomas</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>362</i></span>,  <span class="NLM_fpage">1417</span>– <span class="NLM_lpage">1429</span>, <span class="refDoi"> DOI: 10.1056/NEJMra0807082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1056%2FNEJMra0807082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=20393178" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkslajtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=362&publication_year=2010&pages=1417-1429&author=G.+Lenzauthor=L.+M.+Staudt&title=Aggressive+Lymphomas&doi=10.1056%2FNEJMra0807082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Aggressive lymphomas</span></div><div class="casAuthors">Lenz, Georg; Staudt, Louis M.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">362</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1417-1429</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review.  Given the emergence of many new therapeutic agents that affect essential regulatory pathways in lymphomas, the challenge is to identify rational combinations that kill lymphoma cells synergistically.  This effort requires both preclin. evaluation and the development of clin.-trial strategies that include mol. profiling.  Gene expression profiling could be used to det. the lymphoma subtype and to est. the risk assocd. with std. therapy.  Cancer gene resequencing could be used to identify tumors that are likely to be dependent on a particular signaling pathway and to det. which step in the pathway to inhibit.  Finally, the activity of the targeted pathways before and during therapy could be used to assess the efficacy of the treatment.  We expect that this "divide-and-conquer" approach will provide increasingly effective and nontoxic therapies for patients with lymphomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAtbs8gSY6hrVg90H21EOLACvtfcHk0lhLGM07TqNk-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkslajtbY%253D&md5=005333ef8156a770b418a07aae2f19d8</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1056%2FNEJMra0807082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra0807082%26sid%3Dliteratum%253Aachs%26aulast%3DLenz%26aufirst%3DG.%26aulast%3DStaudt%26aufirst%3DL.%2BM.%26atitle%3DAggressive%2520Lymphomas%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2010%26volume%3D362%26spage%3D1417%26epage%3D1429%26doi%3D10.1056%2FNEJMra0807082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pissot Soldermann, C.</span>; <span class="NLM_string-name">Quancard, J.</span>; <span class="NLM_string-name">Schlapbach, A.</span>; <span class="NLM_string-name">Simic, O.</span>; <span class="NLM_string-name">Tintelnot-Blomley, M.</span>; <span class="NLM_string-name">Zoller, T.</span></span> <span> </span><span class="NLM_article-title">Novel Pyrazolo Pyrimidine Derivatives and their use as MALT1 Inhibitors</span>. PCT Int. Appl., <span class="NLM_patent">WO2015181747A1</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=C.+Pissot+Soldermann&author=J.+Quancard&author=A.+Schlapbach&author=O.+Simic&author=M.+Tintelnot-Blomley&author=T.+Zoller&title=Novel+Pyrazolo+Pyrimidine+Derivatives+and+their+use+as+MALT1+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DPissot%2BSoldermann%26aufirst%3DC.%26atitle%3DNovel%2520Pyrazolo%2520Pyrimidine%2520Derivatives%2520and%2520their%2520use%2520as%2520MALT1%2520Inhibitors%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spranger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincendeau, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grau, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raffegerst, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kloo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hlahla, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuenschwander, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peter von Kries, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadian, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dörken, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schendel, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krappmann, D.</span></span> <span> </span><span class="NLM_article-title">Pharmacologic Inhibition of MALT1 Protease by Phenothiazines as a Therapeutic Approach for the Treatment of Aggressive ABC-DLBCL</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">825</span>– <span class="NLM_lpage">837</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2012.11.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1016%2Fj.ccr.2012.11.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=23238017" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVWns77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=825-837&author=D.+Nagelauthor=S.+Sprangerauthor=M.+Vincendeauauthor=M.+Grauauthor=S.+Raffegerstauthor=B.+Klooauthor=D.+Hlahlaauthor=M.+Neuenschwanderauthor=J.+Peter+von+Kriesauthor=K.+Hadianauthor=B.+D%C3%B6rkenauthor=P.+Lenzauthor=G.+Lenzauthor=D.+J.+Schendelauthor=D.+Krappmann&title=Pharmacologic+Inhibition+of+MALT1+Protease+by+Phenothiazines+as+a+Therapeutic+Approach+for+the+Treatment+of+Aggressive+ABC-DLBCL&doi=10.1016%2Fj.ccr.2012.11.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacologic Inhibition of MALT1 Protease by Phenothiazines as a Therapeutic Approach for the Treatment of Aggressive ABC-DLBCL</span></div><div class="casAuthors">Nagel, Daniel; Spranger, Stefani; Vincendeau, Michelle; Grau, Michael; Raffegerst, Silke; Kloo, Bernhard; Hlahla, Daniela; Neuenschwander, Martin; von Kries, Jens Peter; Hadian, Kamyar; Doerken, Bernd; Lenz, Peter; Lenz, Georg; Schendel, Dolores J.; Krappmann, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">825-837</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Proteolytic activity of the mucosa-assocd. lymphoid tissue lymphoma translocation protein-1 (MALT1) paracaspase is required for survival of the activated B cell subtype of diffuse large B cell lymphoma (ABC-DLBCL).  We have identified distinct derivs. of medicinal active phenothiazines, namely mepazine, thioridazine, and promazine, as small mol. inhibitors of the MALT1 protease.  These phenothiazines selectively inhibit cleavage activity of recombinant and cellular MALT1 by a noncompetitive mechanism.  Consequently, the compds. inhibit anti-apoptotic NF-κB signaling and elicit toxic effects selectively on MALT1-dependent ABC-DLBCL cells in vitro and in vivo.  Our data provide a conceptual proof for a clin. application of distinct phenothiazines in the treatment of ABC-DLBCL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-pha_0UjHHLVg90H21EOLACvtfcHk0lhkDFonczGkRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVWns77J&md5=e67e728b09fbaf08defbaf06c1258be9</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2012.11.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2012.11.002%26sid%3Dliteratum%253Aachs%26aulast%3DNagel%26aufirst%3DD.%26aulast%3DSpranger%26aufirst%3DS.%26aulast%3DVincendeau%26aufirst%3DM.%26aulast%3DGrau%26aufirst%3DM.%26aulast%3DRaffegerst%26aufirst%3DS.%26aulast%3DKloo%26aufirst%3DB.%26aulast%3DHlahla%26aufirst%3DD.%26aulast%3DNeuenschwander%26aufirst%3DM.%26aulast%3DPeter%2Bvon%2BKries%26aufirst%3DJ.%26aulast%3DHadian%26aufirst%3DK.%26aulast%3DD%25C3%25B6rken%26aufirst%3DB.%26aulast%3DLenz%26aufirst%3DP.%26aulast%3DLenz%26aufirst%3DG.%26aulast%3DSchendel%26aufirst%3DD.%2BJ.%26aulast%3DKrappmann%26aufirst%3DD.%26atitle%3DPharmacologic%2520Inhibition%2520of%2520MALT1%2520Protease%2520by%2520Phenothiazines%2520as%2520a%2520Therapeutic%2520Approach%2520for%2520the%2520Treatment%2520of%2520Aggressive%2520ABC-DLBCL%26jtitle%3DCancer%2520Cell%26date%3D2012%26volume%3D22%26spage%3D825%26epage%3D837%26doi%3D10.1016%2Fj.ccr.2012.11.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schlapbach, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revesz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pissot Soldermann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoller, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Régnier, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bornancin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radimerski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blank, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renatus, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erbel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melkko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simic, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endres, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wartmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quancard, J.</span></span> <span> </span><span class="NLM_article-title">N-Aryl-Piperidine-4-Carboxamides as a Novel Class of Potent Inhibitors of MALT1 Proteolytic Activity</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">2153</span>– <span class="NLM_lpage">2158</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.05.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1016%2Fj.bmcl.2018.05.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=29759726" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC1cXps1Crsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=2153-2158&author=A.+Schlapbachauthor=L.+Reveszauthor=C.+Pissot+Soldermannauthor=T.+Zollerauthor=C.+H.+R%C3%A9gnierauthor=F.+Bornancinauthor=T.+Radimerskiauthor=J.+Blankauthor=A.+Schuffenhauerauthor=M.+Renatusauthor=P.+Erbelauthor=S.+Melkkoauthor=R.+Hengauthor=O.+Simicauthor=R.+Endresauthor=M.+Wartmannauthor=J.+Quancard&title=N-Aryl-Piperidine-4-Carboxamides+as+a+Novel+Class+of+Potent+Inhibitors+of+MALT1+Proteolytic+Activity&doi=10.1016%2Fj.bmcl.2018.05.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">N-aryl-piperidine-4-carboxamides as a novel class of potent inhibitors of MALT1 proteolytic activity</span></div><div class="casAuthors">Schlapbach, Achim; Revesz, Laszlo; Pissot Soldermann, Carole; Zoller, Thomas; Regnier, Catherine H.; Bornancin, Frederic; Radimerski, Thomas; Blank, Jutta; Schuffenhauer, Ansgar; Renatus, Martin; Erbel, Paulus; Melkko, Samu; Heng, Richard; Simic, Oliver; Endres, Ralf; Wartmann, Markus; Quancard, Jean</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2153-2158</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Starting from a weak screening hit, potent and selective inhibitors of the MALT1 protease function were elaborated.  Advanced compds. displayed high potency in biochem. and cellular assays.  Compds. showed activity in a mechanistic Jurkat T cell activation assay as well as in the B-cell lymphoma line OCI-Ly3, which suggests potential use of MALT1 inhibitors in the treatment of autoimmune diseases as well as B-cell lymphomas with a dysregulated NF-κB pathway.  Initially, rat pharmacokinetic properties of this compd. series were dominated by very high clearance which could be linked to amide cleavage.  Using a rat hepatocyte assay a good in vitro-in vivo correlation could be established which led to the identification of compds. with improved PK properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4Gx09NEhsQ7Vg90H21EOLACvtfcHk0lhkDFonczGkRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXps1Crsbc%253D&md5=f44b5e6bee523fbf3d5d692e0ce9c49c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.05.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.05.017%26sid%3Dliteratum%253Aachs%26aulast%3DSchlapbach%26aufirst%3DA.%26aulast%3DRevesz%26aufirst%3DL.%26aulast%3DPissot%2BSoldermann%26aufirst%3DC.%26aulast%3DZoller%26aufirst%3DT.%26aulast%3DR%25C3%25A9gnier%26aufirst%3DC.%2BH.%26aulast%3DBornancin%26aufirst%3DF.%26aulast%3DRadimerski%26aufirst%3DT.%26aulast%3DBlank%26aufirst%3DJ.%26aulast%3DSchuffenhauer%26aufirst%3DA.%26aulast%3DRenatus%26aufirst%3DM.%26aulast%3DErbel%26aufirst%3DP.%26aulast%3DMelkko%26aufirst%3DS.%26aulast%3DHeng%26aufirst%3DR.%26aulast%3DSimic%26aufirst%3DO.%26aulast%3DEndres%26aufirst%3DR.%26aulast%3DWartmann%26aufirst%3DM.%26aulast%3DQuancard%26aufirst%3DJ.%26atitle%3DN-Aryl-Piperidine-4-Carboxamides%2520as%2520a%2520Novel%2520Class%2520of%2520Potent%2520Inhibitors%2520of%2520MALT1%2520Proteolytic%2520Activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D28%26spage%3D2153%26epage%3D2158%26doi%3D10.1016%2Fj.bmcl.2018.05.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bardet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unterreiner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malinverni, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lafossas, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedrine, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boesch, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glück, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katopodis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renatus, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simic, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlapbach, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quancard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Régnier, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bold, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pissot-Soldermann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carballido, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovarik, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calzascia, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bornancin, F.</span></span> <span> </span><span class="NLM_article-title">The T-Cell Fingerprint of MALT1 Paracaspase Revealed by Selective Inhibition</span>. <i>Immunol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">81</span>– <span class="NLM_lpage">99</span>, <span class="refDoi"> DOI: 10.1111/imcb.1018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1111%2Fimcb.1018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=29359407" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjs1Kitb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=2018&pages=81-99&author=M.+Bardetauthor=A.+Unterreinerauthor=C.+Malinverniauthor=F.+Lafossasauthor=C.+Vedrineauthor=D.+Boeschauthor=Y.+Kolbauthor=D.+Kaiserauthor=A.+Gl%C3%BCckauthor=M.+A.+Schneiderauthor=A.+Katopodisauthor=M.+Renatusauthor=O.+Simicauthor=A.+Schlapbachauthor=J.+Quancardauthor=C.+H.+R%C3%A9gnierauthor=G.+Boldauthor=C.+Pissot-Soldermannauthor=J.+M.+Carballidoauthor=J.+Kovarikauthor=T.+Calzasciaauthor=F.+Bornancin&title=The+T-Cell+Fingerprint+of+MALT1+Paracaspase+Revealed+by+Selective+Inhibition&doi=10.1111%2Fimcb.1018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The T-cell fingerprint of MALT1 paracaspase revealed by selective inhibition</span></div><div class="casAuthors">Bardet, Maureen; Unterreiner, Adeline; Malinverni, Claire; Lafossas, Frederique; Vedrine, Corinne; Boesch, Danielle; Kolb, Yeter; Kaiser, Daniel; Glueck, Anton; Schneider, Martin A.; Katopodis, Andreas; Renatus, Martin; Simic, Oliver; Schlapbach, Achim; Quancard, Jean; Regnier, Catherine H.; Bold, Guido; Pissot-Soldermann, Carole; Carballido, Jose M.; Kovarik, Jiri; Calzascia, Thomas; Bornancin, Frederic</div><div class="citationInfo"><span class="NLM_cas:title">Immunology & Cell Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">81-99</span>CODEN:
                <span class="NLM_cas:coden">ICBIEZ</span>;
        ISSN:<span class="NLM_cas:issn">1440-1711</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Australia, Ltd.</span>)
        </div><div class="casAbstract">Mucosa-assocd. lymphoid tissue lymphoma translocation protein 1 (MALT1) is essential for immune responses triggered by antigen receptors but the contribution of its paracaspase activity is not fully understood.  Here, we studied how MALT1 proteolytic function regulates T-cell activation and fate after engagement of the T-cell receptor pathway.  We show that MLT-827, a potent and selective MALT1 paracaspase inhibitor, does not prevent the initial phase of T-cell activation, in contrast to the pan-protein kinase C inhibitor AEB071.  However, MLT-827 strongly impacted cell expansion after activation.  We demonstrate this is the consequence of profound inhibition of IL-2 prodn. as well as reduced expression of the IL-2 receptor alpha subunit (CD25), resulting from defective canonical NF-κB activation and accelerated mRNA turnover mechanisms.  Accordingly, MLT-827 revealed a unique transcriptional fingerprint of MALT1 protease activity, providing evidence for broad control of T-cell signaling pathways.  Altogether, this first report with a potent and selective inhibitor elucidates how MALT1 paracaspase activity integrates several T-cell activation pathways and indirectly controls gamma-chain receptor dependent survival, to impact on T-cell expansion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcOZ3C5-LsPrVg90H21EOLACvtfcHk0lhkDFonczGkRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjs1Kitb0%253D&md5=0f35cd2ae66e3572e815eb24aacd6445</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1111%2Fimcb.1018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fimcb.1018%26sid%3Dliteratum%253Aachs%26aulast%3DBardet%26aufirst%3DM.%26aulast%3DUnterreiner%26aufirst%3DA.%26aulast%3DMalinverni%26aufirst%3DC.%26aulast%3DLafossas%26aufirst%3DF.%26aulast%3DVedrine%26aufirst%3DC.%26aulast%3DBoesch%26aufirst%3DD.%26aulast%3DKolb%26aufirst%3DY.%26aulast%3DKaiser%26aufirst%3DD.%26aulast%3DGl%25C3%25BCck%26aufirst%3DA.%26aulast%3DSchneider%26aufirst%3DM.%2BA.%26aulast%3DKatopodis%26aufirst%3DA.%26aulast%3DRenatus%26aufirst%3DM.%26aulast%3DSimic%26aufirst%3DO.%26aulast%3DSchlapbach%26aufirst%3DA.%26aulast%3DQuancard%26aufirst%3DJ.%26aulast%3DR%25C3%25A9gnier%26aufirst%3DC.%2BH.%26aulast%3DBold%26aufirst%3DG.%26aulast%3DPissot-Soldermann%26aufirst%3DC.%26aulast%3DCarballido%26aufirst%3DJ.%2BM.%26aulast%3DKovarik%26aufirst%3DJ.%26aulast%3DCalzascia%26aufirst%3DT.%26aulast%3DBornancin%26aufirst%3DF.%26atitle%3DThe%2520T-Cell%2520Fingerprint%2520of%2520MALT1%2520Paracaspase%2520Revealed%2520by%2520Selective%2520Inhibition%26jtitle%3DImmunol.%2520Cell%2520Biol.%26date%3D2018%26volume%3D96%26spage%3D81%26epage%3D99%26doi%3D10.1111%2Fimcb.1018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quancard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fung, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renatus, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israël, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Priatel, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blank, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viner, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blank, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlapbach, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erbel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kizhakkedathu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villard, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersperger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turvey, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bornancin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overall, C. M.</span></span> <span> </span><span class="NLM_article-title">An Allosteric MALT1 Inhibitor is a Molecular Corrector Rescuing Function in an Immunodeficient Patient</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">313</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0222-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1038%2Fs41589-018-0222-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=30692685" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmtF2isbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=304-313&author=J.+Quancardauthor=T.+Kleinauthor=S.-Y.+Fungauthor=M.+Renatusauthor=N.+Hughesauthor=L.+Isra%C3%ABlauthor=J.+J.+Priatelauthor=S.+Kangauthor=M.+A.+Blankauthor=R.+I.+Vinerauthor=J.+Blankauthor=A.+Schlapbachauthor=P.+Erbelauthor=J.+Kizhakkedathuauthor=F.+Villardauthor=R.+Herspergerauthor=S.+E.+Turveyauthor=J.+Ederauthor=F.+Bornancinauthor=C.+M.+Overall&title=An+Allosteric+MALT1+Inhibitor+is+a+Molecular+Corrector+Rescuing+Function+in+an+Immunodeficient+Patient&doi=10.1038%2Fs41589-018-0222-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">An allosteric MALT1 inhibitor is a molecular corrector rescuing function in an immunodeficient patient</span></div><div class="casAuthors">Quancard, Jean; Klein, Theo; Fung, Shan-Yu; Renatus, Martin; Hughes, Nicola; Israel, Laura; Priatel, John J.; Kang, Sohyeong; Blank, Michael A.; Viner, Rosa I.; Blank, Jutta; Schlapbach, Achim; Erbel, Paul; Kizhakkedathu, Jayachandran; Villard, Frederic; Hersperger, Rene; Turvey, Stuart E.; Eder, Joerg; Bornancin, Frederic; Overall, Christopher M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">304-313</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">MALT1 paracaspase is central for lymphocyte antigen-dependent responses including NF-κB activation.  We discovered nanomolar, selective allosteric inhibitors of MALT1 that bind by displacing the side chain of Trp580, locking the protease in an inactive conformation.  Interestingly, we had previously identified a patient homozygous for a MALT1 Trp580-to-serine mutation who suffered from combined immunodeficiency.  We show that the loss of tryptophan weakened interactions between the paracaspase and C-terminal Ig MALT1 domains resulting in protein instability, reduced protein levels and functions.  Upon binding of allosteric inhibitors of increasing potency, we found proportionate increased stabilization of MALT1-W580S to reach that of wild-type MALT1.  With restored levels of stable MALT1 protein, the most potent of the allosteric inhibitors rescued NF-κB and JNK signaling in patient lymphocytes.  Following compd. washout, MALT1 substrate cleavage was partly recovered.  Thus, a mol. corrector rescues an enzyme deficiency by substituting for the mutated residue, inspiring new potential precision therapies to increase mutant enzyme activity in other deficiencies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoowtKlWIeaBbVg90H21EOLACvtfcHk0lj1BGCVYf4D5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmtF2isbo%253D&md5=923fffe83ed681c62ee0164a2d18775e</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0222-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0222-1%26sid%3Dliteratum%253Aachs%26aulast%3DQuancard%26aufirst%3DJ.%26aulast%3DKlein%26aufirst%3DT.%26aulast%3DFung%26aufirst%3DS.-Y.%26aulast%3DRenatus%26aufirst%3DM.%26aulast%3DHughes%26aufirst%3DN.%26aulast%3DIsra%25C3%25ABl%26aufirst%3DL.%26aulast%3DPriatel%26aufirst%3DJ.%2BJ.%26aulast%3DKang%26aufirst%3DS.%26aulast%3DBlank%26aufirst%3DM.%2BA.%26aulast%3DViner%26aufirst%3DR.%2BI.%26aulast%3DBlank%26aufirst%3DJ.%26aulast%3DSchlapbach%26aufirst%3DA.%26aulast%3DErbel%26aufirst%3DP.%26aulast%3DKizhakkedathu%26aufirst%3DJ.%26aulast%3DVillard%26aufirst%3DF.%26aulast%3DHersperger%26aufirst%3DR.%26aulast%3DTurvey%26aufirst%3DS.%2BE.%26aulast%3DEder%26aufirst%3DJ.%26aulast%3DBornancin%26aufirst%3DF.%26aulast%3DOverall%26aufirst%3DC.%2BM.%26atitle%3DAn%2520Allosteric%2520MALT1%2520Inhibitor%2520is%2520a%2520Molecular%2520Corrector%2520Rescuing%2520Function%2520in%2520an%2520Immunodeficient%2520Patient%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D304%26epage%3D313%26doi%3D10.1038%2Fs41589-018-0222-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatcher, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontán, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Us, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casalena, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Quinoline and Thiazolopyridine Allosteric Inhibitors of MALT1</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1694</span>– <span class="NLM_lpage">1698</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.05.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1016%2Fj.bmcl.2019.05.040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=31129051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVKqtb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=1694-1698&author=D.+A.+Scottauthor=J.+M.+Hatcherauthor=H.+Liuauthor=M.+Fuauthor=G.+Duauthor=L.+Font%C3%A1nauthor=I.+Usauthor=G.+Casalenaauthor=Q.+Qiaoauthor=H.+Wuauthor=A.+Melnickauthor=N.+S.+Gray&title=Quinoline+and+Thiazolopyridine+Allosteric+Inhibitors+of+MALT1&doi=10.1016%2Fj.bmcl.2019.05.040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Quinoline and thiazolopyridine allosteric inhibitors of MALT1</span></div><div class="casAuthors">Scott, David A.; Hatcher, John M.; Liu, Hongyan; Fu, Mingpeng; Du, Guangyan; Fontan, Lorena; Us, Ilkay; Casalena, Gabriella; Qiao, Qi; Wu, Hao; Melnick, Ari; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1694-1698</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Quinolines and thiazolopyridines were developed as allosteric inhibitors of MALT1, with good cellular potency and exquisite selectivity.  Mouse pharmacokinetic (PK) profiling showed these to have low in vivo clearance, and moderate oral exposure.  The thiazolopyridines were less lipophilic than the quinolines, and one thiazolopyridine example was active in our hIL10 mouse pharmacodynamic (PD) model upon oral dosing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmNvxQMuqYn7Vg90H21EOLACvtfcHk0lj1BGCVYf4D5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVKqtb%252FN&md5=878f35e729000596b5eb77d979eb34d5</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.05.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.05.040%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BA.%26aulast%3DHatcher%26aufirst%3DJ.%2BM.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DFu%26aufirst%3DM.%26aulast%3DDu%26aufirst%3DG.%26aulast%3DFont%25C3%25A1n%26aufirst%3DL.%26aulast%3DUs%26aufirst%3DI.%26aulast%3DCasalena%26aufirst%3DG.%26aulast%3DQiao%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DQuinoline%2520and%2520Thiazolopyridine%2520Allosteric%2520Inhibitors%2520of%2520MALT1%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D1694%26epage%3D1698%26doi%3D10.1016%2Fj.bmcl.2019.05.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Unterreiner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Touil, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anastasi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubois, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niwa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calzascia, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bornancin, F.</span></span> <span> </span><span class="NLM_article-title">Myeloid Innate Signaling Pathway Regulation by MALT1 Paracaspase Activity</span>. <i>J. Visualized Exp.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>, <span class="NLM_elocation-id">e58439</span> <span class="refDoi"> DOI: 10.3791/58439</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.3791%2F58439" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtF2qt7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=A.+Unterreinerauthor=R.+Touilauthor=D.+Anastasiauthor=N.+Duboisauthor=S.+Niwaauthor=T.+Calzasciaauthor=F.+Bornancin&title=Myeloid+Innate+Signaling+Pathway+Regulation+by+MALT1+Paracaspase+Activity&doi=10.3791%2F58439"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Myeloid innate signaling pathway regulation by MALT1 paracaspase activity</span></div><div class="casAuthors">Unterreiner, Adeline; Touil, Ratiba; Anastasi, Daniele; Dubois, Nicolas; Niwa, Satoru; Calzascia, Thomas; Bornancin, Frederic</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Visualized Experiments</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue">143</span>),
    <span class="NLM_cas:pages">e58439</span>CODEN:
                <span class="NLM_cas:coden">JVEOA4</span>;
        ISSN:<span class="NLM_cas:issn">1940-087X</span>.
    
            (<span class="NLM_cas:orgname">Journal of Visualized Experiments</span>)
        </div><div class="casAbstract">Besides its function in lymphoid cells, which has been addressed by numerous studies, the paracaspase MALT1 also plays an important role in innate cells downstream of pattern recognition receptors.  Best studied are the Dectin-1 and Dectin-2 members of the C-type lectin-like receptor family that induce a SYK- and CARD9-dependent signaling cascade leading to NF-κB activation, in a MALT1-dependent manner.  By contrast, Toll-like receptors (TLR), such as TLR-4, propagate NF-κB activation but signal via an MYD88/IRAK-dependent cascade.  Nonetheless, whether MALT1 might contribute to TLR-4 signaling has remained unclear.  Recent evidence with MLT-827, a potent and selective inhibitor of MALT1 paracaspase activity, indicates that TNF-prodn. downstream of TLR-4 in human myeloid cells is independent of MALT1, as opposed to TNF-prodn. downstream of Dectin-1, which is MALT1 dependent.  Here, we addressed the selective involvement of MALT1 in pattern recognition sensing further, using a variety of human and mouse cellular prepns., and stimulation of Dectin-1, MINCLE or TLR-4 pathways.  We also provided addnl. insights by exploring cytokines beyond TNF-, and by comparing MLT-827 to a SYK inhibitor (Cpd11) and to an IKK inhibitor (AFN700).  Collectively, the data provided further evidence for the MALT1-dependency of C-type lectin-like receptor-signaling by contrast to TLR-signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9hS6I6nQ4_7Vg90H21EOLACvtfcHk0lj1BGCVYf4D5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtF2qt7nM&md5=5c4e736094fcf5dbb38d0e3f69aadb24</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.3791%2F58439&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3791%252F58439%26sid%3Dliteratum%253Aachs%26aulast%3DUnterreiner%26aufirst%3DA.%26aulast%3DTouil%26aufirst%3DR.%26aulast%3DAnastasi%26aufirst%3DD.%26aulast%3DDubois%26aufirst%3DN.%26aulast%3DNiwa%26aufirst%3DS.%26aulast%3DCalzascia%26aufirst%3DT.%26aulast%3DBornancin%26aufirst%3DF.%26atitle%3DMyeloid%2520Innate%2520Signaling%2520Pathway%2520Regulation%2520by%2520MALT1%2520Paracaspase%2520Activity%26jtitle%3DJ.%2520Visualized%2520Exp.%26date%3D2019%26doi%3D10.3791%2F58439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Israël, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huppertz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercado, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ginster, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unterreiner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlierf, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goetschy, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zerwes, H.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolbinger, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bornancin, F.</span></span> <span> </span><span class="NLM_article-title">A CARD10-Dependent Tonic Signalosome Activates MALT1 Paracaspase and Regulates IL-17/TNF-α-Driven Keratinocyte Inflammation</span>. <i>J. Invest. Dermatol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">2075</span>– <span class="NLM_lpage">2079</span>, <span class="refDoi"> DOI: 10.1016/j.jid.2018.03.1503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1016%2Fj.jid.2018.03.1503" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=29571942" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVyjsr3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2018&pages=2075-2079&author=L.+Isra%C3%ABlauthor=M.+Bardetauthor=A.+Huppertzauthor=N.+Mercadoauthor=S.+Ginsterauthor=A.+Unterreinerauthor=A.+Schlierfauthor=J.+F.+Goetschyauthor=H.-G.+Zerwesauthor=L.+Rothauthor=F.+Kolbingerauthor=F.+Bornancin&title=A+CARD10-Dependent+Tonic+Signalosome+Activates+MALT1+Paracaspase+and+Regulates+IL-17%2FTNF-%CE%B1-Driven+Keratinocyte+Inflammation&doi=10.1016%2Fj.jid.2018.03.1503"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A CARD10-Dependent Tonic Signalosome Activates MALT1 Paracaspase and Regulates IL-17/TNF-α-Driven Keratinocyte Inflammation</span></div><div class="casAuthors">Israel, Laura; Bardet, Maureen; Huppertz, Antje; Mercado, Nicolas; Ginster, Stefanie; Unterreiner, Adeline; Schlierf, Anita; Goetschy, Jean Francois; Zerwes, Hans-Gunter; Roth, Lukas; Kolbinger, Frank; Bornancin, Frederic</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Investigative Dermatology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2075-2079</span>CODEN:
                <span class="NLM_cas:coden">JIDEAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-202X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">This study investigated the role of MALT1 in normal human epidermal keratinocytes (NHEKs) stimulated by IL-17/TNF-α, a synergistic proinflammatory combination known to recapitulate key features of psoriasis.  A selective MALT1 inhibitor MLT-827 treated epidermal keratinocytes stimulation with IL-17A/TNF-α showed reduced IL-8 and IL17C induction by MLT-827.  Results show that MALT1 proteolytic function regulates the IL-17/TNF-α pathway in keratinocytes through the mechanism involving a CARD10-dependent pathway, likely through CBM complex formation between CARD10, BCL10, and MALT1A.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf4RyP7yR5arVg90H21EOLACvtfcHk0lhYP5HU-ktZ7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVyjsr3M&md5=ae294c918bdbd030a8015f3149a4ccaa</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.jid.2018.03.1503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jid.2018.03.1503%26sid%3Dliteratum%253Aachs%26aulast%3DIsra%25C3%25ABl%26aufirst%3DL.%26aulast%3DBardet%26aufirst%3DM.%26aulast%3DHuppertz%26aufirst%3DA.%26aulast%3DMercado%26aufirst%3DN.%26aulast%3DGinster%26aufirst%3DS.%26aulast%3DUnterreiner%26aufirst%3DA.%26aulast%3DSchlierf%26aufirst%3DA.%26aulast%3DGoetschy%26aufirst%3DJ.%2BF.%26aulast%3DZerwes%26aufirst%3DH.-G.%26aulast%3DRoth%26aufirst%3DL.%26aulast%3DKolbinger%26aufirst%3DF.%26aulast%3DBornancin%26aufirst%3DF.%26atitle%3DA%2520CARD10-Dependent%2520Tonic%2520Signalosome%2520Activates%2520MALT1%2520Paracaspase%2520and%2520Regulates%2520IL-17%252FTNF-%25CE%25B1-Driven%2520Keratinocyte%2520Inflammation%26jtitle%3DJ.%2520Invest.%2520Dermatol.%26date%3D2018%26volume%3D138%26spage%3D2075%26epage%3D2079%26doi%3D10.1016%2Fj.jid.2018.03.1503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greengrass, P. M.</span>; <span class="NLM_string-name">Stewart, M.</span>; <span class="NLM_string-name">Wood, C. M.</span></span> <span> </span><span class="NLM_article-title">Affinity-Assay for the Human ERG Potassium Channel</span>. PCT Int. Appl. <span class="NLM_patent">WO2003021271A2</span>, <span class="NLM_year">2002</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2002&author=P.+M.+Greengrass&author=M.+Stewart&author=C.+M.+Wood&title=Affinity-Assay+for+the+Human+ERG+Potassium+Channel"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DGreengrass%26aufirst%3DP.%2BM.%26atitle%3DAffinity-Assay%2520for%2520the%2520Human%2520ERG%2520Potassium%2520Channel%26date%3D2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fermini, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fossa, A. A.</span></span> <span> </span><span class="NLM_article-title">The Impact of Drug-Induced QT Interval Prolongation on Drug Discovery and Development</span>. <i>Nat. Rev. Drug Discov.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">439</span>– <span class="NLM_lpage">447</span>, <span class="refDoi"> DOI: 10.1038/nrd1108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1038%2Fnrd1108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=12776219" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BD3sXktFOhu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=439-447&author=B.+Ferminiauthor=A.+A.+Fossa&title=The+Impact+of+Drug-Induced+QT+Interval+Prolongation+on+Drug+Discovery+and+Development&doi=10.1038%2Fnrd1108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The impact of drug-induced QT interval prolongation on drug discovery and development</span></div><div class="casAuthors">Fermini, Bernard; Fossa, Anthony A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">439-447</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  During the past decade, a no. of non-cardiovascular drugs have had their label revised or have been withdrawn from the market because of unexpected post-marketing reports of sudden cardiac death assocd. with a prolongation of the QT interval, and an increased propensity to develop a ventricular tachyarrhythmia called Torsades de Pointes.  Although a direct link between QT interval prolongation and arrhythmogenesis is still unclear, QT prolongation is now the subject of increased regulatory review and is considered a significant risk factor for predicting human safety of New Chem. Entities.  Consequently, pharmaceutical companies are striving to improve the drug discovery and development process to identify, as early as possible, the risk of novel agents, or their metabolites, of causing QT interval prolongation and to make appropriate go/no-go decisions or modify their development program accordingly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZk6HVMpiVQbVg90H21EOLACvtfcHk0lhYP5HU-ktZ7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXktFOhu70%253D&md5=86f7948c80628e39f69a889e76fc0063</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrd1108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1108%26sid%3Dliteratum%253Aachs%26aulast%3DFermini%26aufirst%3DB.%26aulast%3DFossa%26aufirst%3DA.%2BA.%26atitle%3DThe%2520Impact%2520of%2520Drug-Induced%2520QT%2520Interval%2520Prolongation%2520on%2520Drug%2520Discovery%2520and%2520Development%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discov.%26date%3D2003%26volume%3D2%26spage%3D439%26epage%3D447%26doi%3D10.1038%2Fnrd1108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keating, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanguinetti, M. C.</span></span> <span> </span><span class="NLM_article-title">Molecular and Cellular Mechanisms of Cardiac Arrhythmias</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">569</span>– <span class="NLM_lpage">580</span>, <span class="refDoi"> DOI: 10.1016/s0092-8674(01)00243-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1016%2FS0092-8674%2801%2900243-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=11239413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BD3MXis1Kks7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2001&pages=569-580&author=M.+T.+Keatingauthor=M.+C.+Sanguinetti&title=Molecular+and+Cellular+Mechanisms+of+Cardiac+Arrhythmias&doi=10.1016%2Fs0092-8674%2801%2900243-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular and cellular mechanisms of cardiac arrhythmias</span></div><div class="casAuthors">Keating, Mark T.; Sanguinetti, Michael C.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">569-580</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review, with 71 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnj71g2km85bVg90H21EOLACvtfcHk0lgjT6KVgRWQqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXis1Kks7o%253D&md5=b77453d4075b1e560dd5062878ce76db</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2801%2900243-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252801%252900243-4%26sid%3Dliteratum%253Aachs%26aulast%3DKeating%26aufirst%3DM.%2BT.%26aulast%3DSanguinetti%26aufirst%3DM.%2BC.%26atitle%3DMolecular%2520and%2520Cellular%2520Mechanisms%2520of%2520Cardiac%2520Arrhythmias%26jtitle%3DCell%26date%3D2001%26volume%3D104%26spage%3D569%26epage%3D580%26doi%3D10.1016%2Fs0092-8674%2801%2900243-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pearlstein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rampe, D.</span></span> <span> </span><span class="NLM_article-title">Understanding the Structure–Activity Relationship of the Human Ether-a-go-go-Related Gene Cardiac K+Channel. A Model for Bad Behavior</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">2017</span>– <span class="NLM_lpage">2022</span>, <span class="refDoi"> DOI: 10.1021/jm0205651</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0205651" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjtlWhtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=2017-2022&author=R.+Pearlsteinauthor=R.+Vazauthor=D.+Rampe&title=Understanding+the+Structure%E2%80%93Activity+Relationship+of+the+Human+Ether-a-go-go-Related+Gene+Cardiac+K%2BChannel.+A+Model+for+Bad+Behavior&doi=10.1021%2Fjm0205651"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding the structure-activity relationship of the human ether-a-go-go-related gene cardiac K+ channel. A model for bad behavior</span></div><div class="casAuthors">Pearlstein, Robert; Vaz, Roy; Rampe, David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2017-2022</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with refs. on drug/HERG (human ether-a-go-go-related gene) interactions in relation to SAR for antiarrhythmics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIIRvL3sJJG7Vg90H21EOLACvtfcHk0lgjT6KVgRWQqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjtlWhtLw%253D&md5=351cc53cac39afafc4a4a4dbb65362af</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm0205651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0205651%26sid%3Dliteratum%253Aachs%26aulast%3DPearlstein%26aufirst%3DR.%26aulast%3DVaz%26aufirst%3DR.%26aulast%3DRampe%26aufirst%3DD.%26atitle%3DUnderstanding%2520the%2520Structure%25E2%2580%2593Activity%2520Relationship%2520of%2520the%2520Human%2520Ether-a-go-go-Related%2520Gene%2520Cardiac%2520K%252BChannel.%2520A%2520Model%2520for%2520Bad%2520Behavior%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D2017%26epage%3D2022%26doi%3D10.1021%2Fjm0205651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aronov, A.</span></span> <span> </span><span class="NLM_article-title">Predictive in Silico Modeling for hERG Channel Blockers</span>. <i>Drug Discov. Today</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">149</span>– <span class="NLM_lpage">155</span>, <span class="refDoi"> DOI: 10.1016/S1359-6446(04)03278-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1016%2FS1359-6446%2804%2903278-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=15718164" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhsFarsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=149-155&author=A.+Aronov&title=Predictive+in+Silico+Modeling+for+hERG+Channel+Blockers&doi=10.1016%2FS1359-6446%2804%2903278-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Predictive in silico modeling for hERG channel blockers</span></div><div class="casAuthors">Aronov, Alex M.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">149-155</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  HERG-mediated sudden death as a side effect of non-antiarrhythmic drugs has been receiving increased regulatory attention.  Perhaps owing to the unique shape of the ligand-binding site and its hydrophobic character, the hERG channel has been shown to interact with pharmaceuticals of widely varying structure.  Several in silico approaches have attempted to predict hERG channel blockade.  Some of these approaches are aimed primarily at filtering out potential hERG blockers in the context of virtual libraries, others involve understanding structure-activity relationships governing hERG-drug interactions.  This review summarizes the most recent efforts in this emerging field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDgIcgWdJQJrVg90H21EOLACvtfcHk0lh0ruPz8C-juA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhsFarsbY%253D&md5=112b79a48e3d6f5ab3e313f00cf2a4fa</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903278-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903278-7%26sid%3Dliteratum%253Aachs%26aulast%3DAronov%26aufirst%3DA.%26atitle%3DPredictive%2520in%2520Silico%2520Modeling%2520for%2520hERG%2520Channel%2520Blockers%26jtitle%3DDrug%2520Discov.%2520Today%26date%3D2005%26volume%3D10%26spage%3D149%26epage%3D155%26doi%3D10.1016%2FS1359-6446%2804%2903278-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Faller, B.</span>; <span class="NLM_string-name">Skolnik, S.</span>; <span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">ADMET Diagnosis Models</span>. In  <i>Predictive ADMET: Integrative Approaches in Drug Discovery and Development</i>; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>, <span class="NLM_year">2014</span>; pp  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">61</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1002%2F9781118783344.ch3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&pages=47-61&author=B.+Faller&author=S.+Skolnik&author=J.+Wang&title=Predictive+ADMET%3A+Integrative+Approaches+in+Drug+Discovery+and+Development"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2F9781118783344.ch3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118783344.ch3%26sid%3Dliteratum%253Aachs%26aulast%3DFaller%26aufirst%3DB.%26atitle%3DADMET%2520Diagnosis%2520Models%26btitle%3DPredictive%2520ADMET%253A%2520Integrative%2520Approaches%2520in%2520Drug%2520Discovery%2520and%2520Development%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2014%26spage%3D47%26epage%3D61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiesmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leder, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blank, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melkko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Decock, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Arcy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villard, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erbel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freuler, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolay, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alves, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bornancin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renatus, M.</span></span> <span> </span><span class="NLM_article-title">Structural Determinants of MALT1 Protease Activity</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>419</i></span>,  <span class="NLM_fpage">4</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2012.02.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1016%2Fj.jmb.2012.02.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=22366302" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC38Xkt1Ghur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=419&publication_year=2012&pages=4-21&author=C.+Wiesmannauthor=L.+Lederauthor=J.+Blankauthor=A.+Bernardiauthor=S.+Melkkoauthor=A.+Decockauthor=A.+D%E2%80%99Arcyauthor=F.+Villardauthor=P.+Erbelauthor=N.+Hughesauthor=F.+Freulerauthor=R.+Nikolayauthor=J.+Alvesauthor=F.+Bornancinauthor=M.+Renatus&title=Structural+Determinants+of+MALT1+Protease+Activity&doi=10.1016%2Fj.jmb.2012.02.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Determinants of MALT1 Protease Activity</span></div><div class="casAuthors">Wiesmann, Christian; Leder, Lukas; Blank, Jutta; Bernardi, Anna; Melkko, Samu; Decock, Arnaud; D'Arcy, Allan; Villard, Frederic; Erbel, Paulus; Hughes, Nicola; Freuler, Felix; Nikolay, Rainer; Alves, Juliano; Bornancin, Frederic; Renatus, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">419</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">4-21</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The formation of the CBM (CARD11-BCL10-MALT1) complex is pivotal for antigen-receptor-mediated activation of the transcription factor NF-κB.  Signaling is dependent on MALT1 (mucosa-assocd. lymphoid tissue lymphoma translocation protein 1), which not only acts as a scaffolding protein but also possesses proteolytic activity mediated by its caspase-like domain.  It remained unclear how the CBM activates MALT1.  Here, we provide biochem. and structural evidence that MALT1 activation is dependent on its dimerization and show that mutations at the dimer interface abrogate activity in cells.  The unliganded protease presents itself in a dimeric yet inactive state and undergoes substantial conformational changes upon substrate binding.  These structural changes also affect the conformation of the C-terminal Ig-like domain, a domain that is required for MALT1 activity.  Binding to the active site is coupled to a relative movement of caspase and Ig-like domains.  MALT1 binding partners thus may have the potential of tuning MALT1 protease activity without binding directly to the caspase domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDaYh5EEjXC7Vg90H21EOLACvtfcHk0lh0ruPz8C-juA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xkt1Ghur0%253D&md5=a35adbc547fadbe5353ee2d1c2dad767</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2012.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2012.02.018%26sid%3Dliteratum%253Aachs%26aulast%3DWiesmann%26aufirst%3DC.%26aulast%3DLeder%26aufirst%3DL.%26aulast%3DBlank%26aufirst%3DJ.%26aulast%3DBernardi%26aufirst%3DA.%26aulast%3DMelkko%26aufirst%3DS.%26aulast%3DDecock%26aufirst%3DA.%26aulast%3DD%25E2%2580%2599Arcy%26aufirst%3DA.%26aulast%3DVillard%26aufirst%3DF.%26aulast%3DErbel%26aufirst%3DP.%26aulast%3DHughes%26aufirst%3DN.%26aulast%3DFreuler%26aufirst%3DF.%26aulast%3DNikolay%26aufirst%3DR.%26aulast%3DAlves%26aufirst%3DJ.%26aulast%3DBornancin%26aufirst%3DF.%26aulast%3DRenatus%26aufirst%3DM.%26atitle%3DStructural%2520Determinants%2520of%2520MALT1%2520Protease%2520Activity%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D419%26spage%3D4%26epage%3D21%26doi%3D10.1016%2Fj.jmb.2012.02.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tohda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogoshi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, N.</span></span> <span> </span><span class="NLM_article-title">Establishment of a Novel B-Cell Lymphoma Cell Line with Suppressed Growth by Gamma-Secretase Inhibitors</span>. <i>Leuk. Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1385</span>– <span class="NLM_lpage">1390</span>, <span class="refDoi"> DOI: 10.1016/j.leukres.2006.05.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1016%2Fj.leukres.2006.05.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=16780947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BD28Xptlykurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2006&pages=1385-1390&author=S.+Tohdaauthor=T.+Satoauthor=H.+Kogoshiauthor=L.+Fuauthor=S.+Sakanoauthor=N.+Nara&title=Establishment+of+a+Novel+B-Cell+Lymphoma+Cell+Line+with+Suppressed+Growth+by+Gamma-Secretase+Inhibitors&doi=10.1016%2Fj.leukres.2006.05.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Establishment of a novel B-cell lymphoma cell line with suppressed growth by gamma-secretase inhibitors</span></div><div class="casAuthors">Tohda, Shuji; Sato, Taku; Kogoshi, Hanae; Fu, Lu; Sakano, Seiji; Nara, Nobuo</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1385-1390</span>CODEN:
                <span class="NLM_cas:coden">LEREDD</span>;
        ISSN:<span class="NLM_cas:issn">0145-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A novel lymphoma cell line, designated TMD8 was established from cells of a patient with diffuse large B-cell lymphoma.  TMD8 cells expressed HES1 mRNA, suggesting constitutive activation of Notch signaling.  TMD8 cells expressed normal-sized Notch1 protein, and showed no mutations in the NOTCH1 gene.  Cell growth was suppressed by gamma-secretase inhibitors (GSI).  It was reported that GSI suppressed growth of T-cell acute lymphoblastic leukemia (T-ALL) cell lines, which frequently had NOTCH1 mutations.  In addn. to T-ALL, TMD8 is another unique cell line sensitive to GSI, and is useful to study effects of GSI in mol. targeting therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3d8inz_KIn7Vg90H21EOLACvtfcHk0lh0ruPz8C-juA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xptlykurg%253D&md5=9db545b9d7b4f8c40b23a79d771261f5</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.leukres.2006.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.leukres.2006.05.003%26sid%3Dliteratum%253Aachs%26aulast%3DTohda%26aufirst%3DS.%26aulast%3DSato%26aufirst%3DT.%26aulast%3DKogoshi%26aufirst%3DH.%26aulast%3DFu%26aufirst%3DL.%26aulast%3DSakano%26aufirst%3DS.%26aulast%3DNara%26aufirst%3DN.%26atitle%3DEstablishment%2520of%2520a%2520Novel%2520B-Cell%2520Lymphoma%2520Cell%2520Line%2520with%2520Suppressed%2520Growth%2520by%2520Gamma-Secretase%2520Inhibitors%26jtitle%3DLeuk.%2520Res.%26date%3D2006%26volume%3D30%26spage%3D1385%26epage%3D1390%26doi%3D10.1016%2Fj.leukres.2006.05.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Naylor, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratsch, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enns, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuler, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorken, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.-m.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmeier, F.</span></span> <span> </span><span class="NLM_article-title">Protein Kinase C Inhibitor Sotrastaurin Selectively Inhibits the Growth of CD79 Mutant Diffuse Large B-Cell Lymphomas</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">2643</span>– <span class="NLM_lpage">2653</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-10-2525</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1158%2F0008-5472.CAN-10-2525" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=21324920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVOjtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=2643-2653&author=T.+L.+Naylorauthor=H.+Tangauthor=B.+A.+Ratschauthor=A.+Ennsauthor=A.+Looauthor=L.+Chenauthor=P.+Lenzauthor=N.+J.+Watersauthor=W.+Schulerauthor=B.+Dorkenauthor=Y.-m.+Yaoauthor=M.+Warmuthauthor=G.+Lenzauthor=F.+Stegmeier&title=Protein+Kinase+C+Inhibitor+Sotrastaurin+Selectively+Inhibits+the+Growth+of+CD79+Mutant+Diffuse+Large+B-Cell+Lymphomas&doi=10.1158%2F0008-5472.CAN-10-2525"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Kinase C Inhibitor Sotrastaurin Selectively Inhibits the Growth of CD79 Mutant Diffuse Large B-Cell Lymphomas</span></div><div class="casAuthors">Naylor, Tara L.; Tang, Huaping; Ratsch, Boris A.; Enns, Andreas; Loo, Alice; Chen, Liqing; Lenz, Peter; Waters, Nigel J.; Schuler, Walter; Doerken, Bernd; Yao, Yung-mae; Warmuth, Markus; Lenz, Georg; Stegmeier, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2643-2653</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The activated B-cell-like (ABC) subtype of diffuse large B-cell lymphoma (DLBCL) correlates with poor prognosis.  The ABC subtype of DLBCL is assocd. with constitutive activation of the NF-κB pathway, and oncogenic lesions have been identified in its regulators, including CARD11/CARMA1 (caspase recruitment domain-contg. protein 11), A20/TNF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoE3V3gS52btrVg90H21EOLACvtfcHk0lgd8MSFhIzGmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVOjtbs%253D&md5=68f1e5241923456583cc1190da0ed590</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-10-2525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-10-2525%26sid%3Dliteratum%253Aachs%26aulast%3DNaylor%26aufirst%3DT.%2BL.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DRatsch%26aufirst%3DB.%2BA.%26aulast%3DEnns%26aufirst%3DA.%26aulast%3DLoo%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DLenz%26aufirst%3DP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DSchuler%26aufirst%3DW.%26aulast%3DDorken%26aufirst%3DB.%26aulast%3DYao%26aufirst%3DY.-m.%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DLenz%26aufirst%3DG.%26aulast%3DStegmeier%26aufirst%3DF.%26atitle%3DProtein%2520Kinase%2520C%2520Inhibitor%2520Sotrastaurin%2520Selectively%2520Inhibits%2520the%2520Growth%2520of%2520CD79%2520Mutant%2520Diffuse%2520Large%2520B-Cell%2520Lymphomas%26jtitle%3DCancer%2520Res.%26date%3D2011%26volume%3D71%26spage%3D2643%26epage%3D2653%26doi%3D10.1158%2F0008-5472.CAN-10-2525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fung, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blank, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dufour, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolger-Munro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scurll, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Priatel, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweigler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melkko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gold, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viner, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Régnier, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turvey, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overall, C. M.</span></span> <span> </span><span class="NLM_article-title">The paracaspase MALT1 cleaves HOIL1 reducing linear ubiquitination by LUBAC to dampen lymphocyte NF-κB signalling</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">8777</span>, <span class="refDoi"> DOI: 10.1038/ncomms9777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1038%2Fncomms9777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=26525107" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslOns7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=8777&author=T.+Kleinauthor=S.-Y.+Fungauthor=F.+Rennerauthor=M.+A.+Blankauthor=A.+Dufourauthor=S.+Kangauthor=M.+Bolger-Munroauthor=J.+M.+Scurllauthor=J.+J.+Priatelauthor=P.+Schweiglerauthor=S.+Melkkoauthor=M.+R.+Goldauthor=R.+I.+Vinerauthor=C.+H.+R%C3%A9gnierauthor=S.+E.+Turveyauthor=C.+M.+Overall&title=The+paracaspase+MALT1+cleaves+HOIL1+reducing+linear+ubiquitination+by+LUBAC+to+dampen+lymphocyte+NF-%CE%BAB+signalling&doi=10.1038%2Fncomms9777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The paracaspase MALT1 cleaves HOIL1 reducing linear ubiquitination by LUBAC to dampen lymphocyte NF-κB signalling</span></div><div class="casAuthors">Klein, Theo; Fung, Shan-Yu; Renner, Florian; Blank, Michael A.; Dufour, Antoine; Kang, Sohyeong; Bolger-Munro, Madison; Scurll, Joshua M.; Priatel, John J.; Schweigler, Patrick; Melkko, Samu; Gold, Michael R.; Viner, Rosa I.; Regnier, Catherine H.; Turvey, Stuart E.; Overall, Christopher M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8777</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Antigen receptor signalling activates the canonical NF-κB pathway via the CARD11/BCL10/MALT1 (CBM) signalosome involving key, yet ill-defined roles for linear ubiquitination.  The paracaspase MALT1 cleaves and removes neg. checkpoint proteins, amplifying lymphocyte responses in NF-κB activation and in B-cell lymphoma subtypes.  To identify new human MALT1 substrates, we compare B cells from the only known living MALT1mut/mut patient with healthy MALT1+/mut family members using 10-plex Tandem Mass Tag TAILS N-terminal peptide proteomics.  We identify HOIL1 of the linear ubiquitin chain assembly complex as a novel MALT1 substrate.  We show linear ubiquitination at B-cell receptor microclusters and signalosomes.  Late in the NF-κB activation cycle HOIL1 cleavage transiently reduces linear ubiquitination, including of NEMO and RIP1, dampening NF-κB activation and preventing reactivation.  By regulating linear ubiquitination, MALT1 is both a pos. and neg. pleiotropic regulator of the human canonical NF-κB pathway-first promoting activation via the CBM-then triggering HOIL1-dependent neg.-feedback termination, preventing reactivation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3c7mF51tenLVg90H21EOLACvtfcHk0lgd8MSFhIzGmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslOns7fP&md5=dd36a72c870b574f4e8cfc3dff092995</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fncomms9777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms9777%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DT.%26aulast%3DFung%26aufirst%3DS.-Y.%26aulast%3DRenner%26aufirst%3DF.%26aulast%3DBlank%26aufirst%3DM.%2BA.%26aulast%3DDufour%26aufirst%3DA.%26aulast%3DKang%26aufirst%3DS.%26aulast%3DBolger-Munro%26aufirst%3DM.%26aulast%3DScurll%26aufirst%3DJ.%2BM.%26aulast%3DPriatel%26aufirst%3DJ.%2BJ.%26aulast%3DSchweigler%26aufirst%3DP.%26aulast%3DMelkko%26aufirst%3DS.%26aulast%3DGold%26aufirst%3DM.%2BR.%26aulast%3DViner%26aufirst%3DR.%2BI.%26aulast%3DR%25C3%25A9gnier%26aufirst%3DC.%2BH.%26aulast%3DTurvey%26aufirst%3DS.%2BE.%26aulast%3DOverall%26aufirst%3DC.%2BM.%26atitle%3DThe%2520paracaspase%2520MALT1%2520cleaves%2520HOIL1%2520reducing%2520linear%2520ubiquitination%2520by%2520LUBAC%2520to%2520dampen%2520lymphocyte%2520NF-%25CE%25BAB%2520signalling%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D8777%26doi%3D10.1038%2Fncomms9777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bornancin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Touil, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sic, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Touil-Allaoui, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rush, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigaud, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junker-Walker, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkhart, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niwa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katopodis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuesslein-Hildesheim, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weckbecker, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zenke, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinzel, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traggiai, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brüstle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">St. Paul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamurovic, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolink, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Régnier, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mak, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohashi, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calzascia, T.</span></span> <span> </span><span class="NLM_article-title">Deficiency of MALT1 Paracaspase Activity Results in Unbalanced Regulatory and Effector T and B Cell Responses Leading to Multiorgan Inflammation</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>194</i></span>,  <span class="NLM_fpage">3723</span>– <span class="NLM_lpage">3734</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.1402254</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.4049%2Fjimmunol.1402254" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=25762782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVeisrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=194&publication_year=2015&pages=3723-3734&author=F.+Bornancinauthor=F.+Rennerauthor=R.+Touilauthor=H.+Sicauthor=Y.+Kolbauthor=I.+Touil-Allaouiauthor=J.+S.+Rushauthor=P.+A.+Smithauthor=M.+Bigaudauthor=U.+Junker-Walkerauthor=C.+Burkhartauthor=J.+Dawsonauthor=S.+Niwaauthor=A.+Katopodisauthor=B.+Nuesslein-Hildesheimauthor=G.+Weckbeckerauthor=G.+Zenkeauthor=B.+Kinzelauthor=E.+Traggiaiauthor=D.+Brennerauthor=A.+Br%C3%BCstleauthor=M.+St.+Paulauthor=N.+Zamurovicauthor=K.+D.+McCoyauthor=A.+Rolinkauthor=C.+H.+R%C3%A9gnierauthor=T.+W.+Makauthor=P.+S.+Ohashiauthor=D.+D.+Patelauthor=T.+Calzascia&title=Deficiency+of+MALT1+Paracaspase+Activity+Results+in+Unbalanced+Regulatory+and+Effector+T+and+B+Cell+Responses+Leading+to+Multiorgan+Inflammation&doi=10.4049%2Fjimmunol.1402254"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Deficiency of MALT1 Paracaspase Activity Results in Unbalanced Regulatory and Effector T and B Cell Responses Leading to Multiorgan Inflammation</span></div><div class="casAuthors">Bornancin, Frederic; Renner, Florian; Touil, Ratiba; Sic, Heiko; Kolb, Yeter; Touil-Allaoui, Ismahane; Rush, James S.; Smith, Paul A.; Bigaud, Marc; Junker-Walker, Ursula; Burkhart, Christoph; Dawson, Janet; Niwa, Satoru; Katopodis, Andreas; Nuesslein-Hildesheim, Barbara; Weckbecker, Gisbert; Zenke, Gerhard; Kinzel, Bernd; Traggiai, Elisabetta; Brenner, Dirk; Bruestle, Anne; St. Paul, Michael; Zamurovic, Natasa; McCoy, Kathy D.; Rolink, Antonius; Regnier, Catherine H.; Mak, Tak W.; Ohashi, Pamela S.; Patel, Dhavalkumar D.; Calzascia, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">194</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3723-3734</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">The paracaspase MALT1 plays an important role in immune receptor-driven signaling pathways leading to NF-κB activation.  MALT1 promotes signaling by acting as a scaffold, recruiting downstream signaling proteins, as well as by proteolytic cleavage of multiple substrates.  However, the relative contributions of these two different activities to T and B cell function are not well understood.  To investigate how MALT1 proteolytic activity contributes to overall immune cell regulation, we generated MALT1 protease-deficient mice (Malt1PD/PD) and compared their phenotype with that of MALT1 knockout animals (Malt1-/-).  Malt1PD/PD mice displayed defects in multiple cell types including marginal zone B cells, B1 B cells, IL-10-producing B cells, regulatory T cells, and mature T and B cells.  In general, immune defects were more pronounced in Malt1-/- animals.  Both mouse lines showed abrogated B cell responses upon immunization with T-dependent and T-independent Ags.  In vitro, inactivation of MALT1 protease activity caused reduced stimulation-induced T cell proliferation, impaired IL-2 and TNF-α prodn., as well as defective Th17 differentiation.  Consequently, Malt1PD/PD mice were protected in a Th17-dependent exptl. autoimmune encephalomyelitis model.  Surprisingly, Malt1PD/PD animals developed a multiorgan inflammatory pathol., characterized by Th1 and Th2/0 responses and enhanced IgG1 and IgE levels, which was delayed by wild-type regulatory T cell reconstitution.  We therefore propose that the pathol. characterizing Malt1PD/PD animals arises from an immune imbalance featuring pathogenic Th1- and Th2/0-skewed effector responses and reduced immunosuppressive compartments.  These data uncover a previously unappreciated key function of MALT1 protease activity in immune homeostasis and underline its relevance in human health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8H9YU0RgmnbVg90H21EOLACvtfcHk0lgd8MSFhIzGmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVeisrc%253D&md5=8e16455eaab8d70f0fe8cf246aeb1edc</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1402254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1402254%26sid%3Dliteratum%253Aachs%26aulast%3DBornancin%26aufirst%3DF.%26aulast%3DRenner%26aufirst%3DF.%26aulast%3DTouil%26aufirst%3DR.%26aulast%3DSic%26aufirst%3DH.%26aulast%3DKolb%26aufirst%3DY.%26aulast%3DTouil-Allaoui%26aufirst%3DI.%26aulast%3DRush%26aufirst%3DJ.%2BS.%26aulast%3DSmith%26aufirst%3DP.%2BA.%26aulast%3DBigaud%26aufirst%3DM.%26aulast%3DJunker-Walker%26aufirst%3DU.%26aulast%3DBurkhart%26aufirst%3DC.%26aulast%3DDawson%26aufirst%3DJ.%26aulast%3DNiwa%26aufirst%3DS.%26aulast%3DKatopodis%26aufirst%3DA.%26aulast%3DNuesslein-Hildesheim%26aufirst%3DB.%26aulast%3DWeckbecker%26aufirst%3DG.%26aulast%3DZenke%26aufirst%3DG.%26aulast%3DKinzel%26aufirst%3DB.%26aulast%3DTraggiai%26aufirst%3DE.%26aulast%3DBrenner%26aufirst%3DD.%26aulast%3DBr%25C3%25BCstle%26aufirst%3DA.%26aulast%3DSt.%2BPaul%26aufirst%3DM.%26aulast%3DZamurovic%26aufirst%3DN.%26aulast%3DMcCoy%26aufirst%3DK.%2BD.%26aulast%3DRolink%26aufirst%3DA.%26aulast%3DR%25C3%25A9gnier%26aufirst%3DC.%2BH.%26aulast%3DMak%26aufirst%3DT.%2BW.%26aulast%3DOhashi%26aufirst%3DP.%2BS.%26aulast%3DPatel%26aufirst%3DD.%2BD.%26aulast%3DCalzascia%26aufirst%3DT.%26atitle%3DDeficiency%2520of%2520MALT1%2520Paracaspase%2520Activity%2520Results%2520in%2520Unbalanced%2520Regulatory%2520and%2520Effector%2520T%2520and%2520B%2520Cell%2520Responses%2520Leading%2520to%2520Multiorgan%2520Inflammation%26jtitle%3DJ.%2520Immunol.%26date%3D2015%26volume%3D194%26spage%3D3723%26epage%3D3734%26doi%3D10.4049%2Fjimmunol.1402254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erbel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bornancin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiesmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiering, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villard, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Decock, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melkko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spanka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buschmann, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pissot-Soldermann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simic, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beerli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tintelnot-Blomley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beltz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Régnier, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quancard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlapbach, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langlois, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renatus, M.</span></span> <span> </span><span class="NLM_article-title">Stabilizing Inactive Conformations of MALT1 as an Effective Approach to Inhibit Its Protease Activity</span>. <i>Adv. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">2000078</span>, <span class="refDoi"> DOI: 10.1002/adtp.202000078</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1002%2Fadtp.202000078" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslKhs7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2020&pages=2000078&author=N.+Hughesauthor=P.+Erbelauthor=F.+Bornancinauthor=C.+Wiesmannauthor=N.+Schieringauthor=F.+Villardauthor=A.+Decockauthor=B.+Rubiauthor=S.+Melkkoauthor=C.+Spankaauthor=N.+Buschmannauthor=C.+Pissot-Soldermannauthor=O.+Simicauthor=R.+Beerliauthor=M.+Sorgeauthor=M.+Tintelnot-Blomleyauthor=K.+Beltzauthor=C.+H.+R%C3%A9gnierauthor=J.+Quancardauthor=A.+Schlapbachauthor=J.+B.+Langloisauthor=M.+Renatus&title=Stabilizing+Inactive+Conformations+of+MALT1+as+an+Effective+Approach+to+Inhibit+Its+Protease+Activity&doi=10.1002%2Fadtp.202000078"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Stabilizing Inactive Conformations of MALT1 as an Effective Approach to Inhibit Its Protease Activity</span></div><div class="casAuthors">Hughes, Nicola; Erbel, Paul; Bornancin, Frederic; Wiesmann, Christian; Schiering, Nikolaus; Villard, Frederic; Decock, Arnaud; Rubi, Bertran; Melkko, Samu; Spanka, Carsten; Buschmann, Nicole; Pissot-Soldermann, Carole; Simic, Oliver; Beerli, Rene; Sorge, Mickael; Tintelnot-Blomley, Marina; Beltz, Karen; Regnier, Catherine H.; Quancard, Jean; Schlapbach, Achim; Langlois, Jean-Baptiste; Renatus, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Therapeutics (Weinheim, Germany)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2000078</span>CODEN:
                <span class="NLM_cas:coden">ATWGAP</span>;
        ISSN:<span class="NLM_cas:issn">2366-3987</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The paracaspase MALT1 (mucosa assocd. lymphoid tissue lymphoma translocated gene 1) plays an important role in various immune pathways and is proposed as a therapeutic target for autoimmune disorders as well as different types of cancer, such as diffuse large B-cell lymphoma (DLBCL).  Different mechanisms are explored to inhibit the protease activity of MALT1 and two unrelated chem. scaffolds are discovered.  Biophys. and structural studies reveal that both scaffolds stabilize the protease in an inactive conformation.  While one ligand binds to the allosteric site at the interface between the caspase and the Ig3 domain, the other ligand binds to the active site using a so far undescribed mechanism.  Iterative structure-based drug discovery on one scaffold results in the identification of a potent, selective, and orally bioavailable MALT1 inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7qcznIt2xNLVg90H21EOLACvtfcHk0lgTPAnJg3b7Vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslKhs7jO&md5=db679edcbd1dba3d3e991d55e07d5aa8</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fadtp.202000078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fadtp.202000078%26sid%3Dliteratum%253Aachs%26aulast%3DHughes%26aufirst%3DN.%26aulast%3DErbel%26aufirst%3DP.%26aulast%3DBornancin%26aufirst%3DF.%26aulast%3DWiesmann%26aufirst%3DC.%26aulast%3DSchiering%26aufirst%3DN.%26aulast%3DVillard%26aufirst%3DF.%26aulast%3DDecock%26aufirst%3DA.%26aulast%3DRubi%26aufirst%3DB.%26aulast%3DMelkko%26aufirst%3DS.%26aulast%3DSpanka%26aufirst%3DC.%26aulast%3DBuschmann%26aufirst%3DN.%26aulast%3DPissot-Soldermann%26aufirst%3DC.%26aulast%3DSimic%26aufirst%3DO.%26aulast%3DBeerli%26aufirst%3DR.%26aulast%3DSorge%26aufirst%3DM.%26aulast%3DTintelnot-Blomley%26aufirst%3DM.%26aulast%3DBeltz%26aufirst%3DK.%26aulast%3DR%25C3%25A9gnier%26aufirst%3DC.%2BH.%26aulast%3DQuancard%26aufirst%3DJ.%26aulast%3DSchlapbach%26aufirst%3DA.%26aulast%3DLanglois%26aufirst%3DJ.%2BB.%26aulast%3DRenatus%26aufirst%3DM.%26atitle%3DStabilizing%2520Inactive%2520Conformations%2520of%2520MALT1%2520as%2520an%2520Effective%2520Approach%2520to%2520Inhibit%2520Its%2520Protease%2520Activity%26jtitle%3DAdv.%2520Ther.%26date%3D2020%26volume%3D3%26spage%3D2000078%26doi%3D10.1002%2Fadtp.202000078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 6 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Xuewu Liang, Chenxia Sun, Chunpu Li, Haolan Yu, Xiaohui Wei, Xuyi Liu, Wei Bao, Yuqiang Shi, Xiaochen Sun, Mirzadavlat Khamrakulov, Chenghua Yang, <span class="NLM_string-name hlFld-ContribAuthor">Hong Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of Novel Fused Heteroaromatics-Based MALT1 Inhibitors by High-Throughput Screening to Treat B Cell Lymphoma. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (13)
                                     , 9217-9237. <a href="https://doi.org/10.1021/acs.jmedchem.1c00466" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00466</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00466&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00466%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252BNovel%252BFused%252BHeteroaromatics-Based%252BMALT1%252BInhibitors%252Bby%252BHigh-Throughput%252BScreening%252Bto%252BTreat%252BB%252BCell%252BLymphoma%26aulast%3DLiang%26aufirst%3DXuewu%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D15032021%26date%3D28062021%26volume%3D64%26issue%3D13%26spage%3D9217%26epage%3D9237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jean Quancard, Oliver Simic, Carole Pissot Soldermann, Reiner Aichholz, Markus Blatter, Martin Renatus, Paulus Erbel, Samu Melkko, Ralf Endres, Mickael Sorge, Laurence Kieffer, Trixie Wagner, Karen Beltz, Paul Mcsheehy, Markus Wartmann, Catherine H. Régnier, Thomas Calzascia, Thomas Radimerski, Marc Bigaud, Andreas Weiss, Frédéric Bornancin, <span class="NLM_string-name hlFld-ContribAuthor">Achim Schlapbach</span>. </span><span class="cited-content_cbyCitation_article-title">Optimization of the In Vivo Potency of Pyrazolopyrimidine MALT1 Protease Inhibitors by Reducing Metabolism and Increasing Potency in Whole Blood. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (23)
                                     , 14594-14608. <a href="https://doi.org/10.1021/acs.jmedchem.0c01246" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01246</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01246&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01246%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOptimization%252Bof%252Bthe%252BIn%252BVivo%252BPotency%252Bof%252BPyrazolopyrimidine%252BMALT1%252BProtease%252BInhibitors%252Bby%252BReducing%252BMetabolism%252Band%252BIncreasing%252BPotency%252Bin%252BWhole%252BBlood%26aulast%3DQuancard%26aufirst%3DJean%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D17072020%26date%3D20112020%26volume%3D63%26issue%3D23%26spage%3D14594%26epage%3D14608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Isabel  Hamp</span>, <span class="hlFld-ContribAuthor ">Thomas J.  O’Neill</span>, <span class="hlFld-ContribAuthor ">Oliver  Plettenburg</span>, <span class="hlFld-ContribAuthor ">Daniel  Krappmann</span>. </span><span class="cited-content_cbyCitation_article-title">A patent review of MALT1 inhibitors (2013-present). </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Patents</span><span> <strong>2021,</strong> <em>77 </em>, 1-18. <a href="https://doi.org/10.1080/13543776.2021.1951703" title="DOI URL">https://doi.org/10.1080/13543776.2021.1951703</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/13543776.2021.1951703&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F13543776.2021.1951703%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Patents%26atitle%3DA%252Bpatent%252Breview%252Bof%252BMALT1%252Binhibitors%252B%2525282013-present%252529%26aulast%3DHamp%26aufirst%3DIsabel%26date%3D2021%26date%3D2021%26volume%3D77%26spage%3D1%26epage%3D18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kathryn M.  Nelson</span>, <span class="hlFld-ContribAuthor ">Michael A.  Walters</span>. </span><span class="cited-content_cbyCitation_article-title">The Communication of Hit Quality Using Natural History Visualizations (NHVs). </span><span class="cited-content_cbyCitation_journal-name">SLAS DISCOVERY: Advancing the Science of Drug Discovery</span><span> <strong>2021,</strong> <em>36 </em>, 247255522110175. <a href="https://doi.org/10.1177/24725552211017518" title="DOI URL">https://doi.org/10.1177/24725552211017518</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/24725552211017518&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F24725552211017518%26sid%3Dliteratum%253Aachs%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26atitle%3DThe%252BCommunication%252Bof%252BHit%252BQuality%252BUsing%252BNatural%252BHistory%252BVisualizations%252B%252528NHVs%252529%26aulast%3DNelson%26aufirst%3DKathryn%2BM.%26date%3D2021%26date%3D2021%26volume%3D36%26spage%3D247255522110175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Barbara  Kiesewetter</span>, <span class="hlFld-ContribAuthor ">Christiane  Copie-Bergman</span>, <span class="hlFld-ContribAuthor ">Michael  Levy</span>, <span class="hlFld-ContribAuthor ">Fangtian  Wu</span>, <span class="hlFld-ContribAuthor ">Jehan  Dupuis</span>, <span class="hlFld-ContribAuthor ">Caroline  Barau</span>, <span class="hlFld-ContribAuthor ">Luca  Arcaini</span>, <span class="hlFld-ContribAuthor ">Marco  Paulli</span>, <span class="hlFld-ContribAuthor ">Marco  Lucioni</span>, <span class="hlFld-ContribAuthor ">Arturo  Bonometti</span>, <span class="hlFld-ContribAuthor ">Antonio  Salar</span>, <span class="hlFld-ContribAuthor ">Concepción  Fernández-Rodriguez</span>, <span class="hlFld-ContribAuthor ">Miguel A.  Piris</span>, <span class="hlFld-ContribAuthor ">Francesco  Cucco</span>, <span class="hlFld-ContribAuthor ">Rachel  Dobson</span>, <span class="hlFld-ContribAuthor ">Yan  Li</span>, <span class="hlFld-ContribAuthor ">Zi  Chen</span>, <span class="hlFld-ContribAuthor ">Cyrielle  Robe</span>, <span class="hlFld-ContribAuthor ">Ingrid  Simonitsch-Klupp</span>, <span class="hlFld-ContribAuthor ">Andrew  Wotherspoon</span>, <span class="hlFld-ContribAuthor ">Markus  Raderer</span>, <span class="hlFld-ContribAuthor ">Ming Qing  Du</span>. </span><span class="cited-content_cbyCitation_article-title">Genetic Characterization and Clinical Features of Helicobacter pylori Negative Gastric Mucosa-Associated Lymphoid Tissue Lymphoma. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2021,</strong> <em>13 </em>
                                    (12)
                                     , 2993. <a href="https://doi.org/10.3390/cancers13122993" title="DOI URL">https://doi.org/10.3390/cancers13122993</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers13122993&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers13122993%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DGenetic%252BCharacterization%252Band%252BClinical%252BFeatures%252Bof%252BHelicobacter%252Bpylori%252BNegative%252BGastric%252BMucosa-Associated%252BLymphoid%252BTissue%252BLymphoma%26aulast%3DKiesewetter%26aufirst%3DBarbara%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D12%26spage%3D2993" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S. V.  Fedoseev</span>, <span class="hlFld-ContribAuthor ">O. E.  Ershov</span>. </span><span class="cited-content_cbyCitation_article-title">Reaction of 4-Halo-3-hydroxyfuro[3,4-c]pyridin-1(3H)-ones with Morpholine and Thiomorpholine. </span><span class="cited-content_cbyCitation_journal-name">Russian Journal of Organic Chemistry</span><span> <strong>2021,</strong> <em>57 </em>
                                    (3)
                                     , 483-485. <a href="https://doi.org/10.1134/S1070428021030234" title="DOI URL">https://doi.org/10.1134/S1070428021030234</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1134/S1070428021030234&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1134%2FS1070428021030234%26sid%3Dliteratum%253Aachs%26jtitle%3DRussian%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DReaction%252Bof%252B4-Halo-3-hydroxyfuro%25255B3%25252C4-c%25255Dpyridin-1%2525283H%252529-ones%252Bwith%252BMorpholine%252Band%252BThiomorpholine%26aulast%3DFedoseev%26aufirst%3DS.%2BV.%26date%3D2021%26date%3D2021%26volume%3D57%26issue%3D3%26spage%3D483%26epage%3D485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01245&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Chemical Structure and Biochemical Potency of HTS Hit 1</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01245&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Small-molecule X-ray and crystal packing of compound <b>17</b> (mono hydrate). Classical intermolecular hydrogen-bond interactions between the urea nitrogens and water are highlighted in yellow. Additional intermolecular nonclassical CH–N contacts are depicted in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01245&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Small-molecule X-ray of compound <b>23</b> overlayed with MLT-827. (A) ORTEP view of compound <b>23</b>. (B) Small-molecule X-ray structures overlays of compound <b>23</b> (cyan) and <b>MLT-827</b> (light yellow) from two orientations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01245&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cocrystal structures of MALT1 in complex with urea-based inhibitors (A) Cocrystal structure of MALT1 in complex with chromane urea-based inhibitors <b>TC1</b> (orange), 2.3 Å, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AK0">7AK0</a>. (B) Cocrystal structure of MALT1 in complex with pyrazolopyrimidine urea-based inhibitors <b>TC2</b> (green), 2.5 Å, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AK1">7AK1</a>. (C) Overlay of the structures from (A,B) illustrating conformational changes in the inter-domain helix.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01245&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound MLT-231 inhibits cellular proliferation and cleavage of MALT1 substrates in ABC-DLBCL. (A) Concentration-dependent inhibition of OCI-Ly3 cell growth in soft-agar by MLT-231. The same compound has no impact on BJAB cells from the GC-DLBCL type. (B) Western blots of the indicated proteins in OCI-Ly3 cells treated with increasing concentrations of MLT231 or DMSO for 24 h. The pharmacological inhibition of MALT1 protease activity leads to accumulation of the uncleaved form of its substrates CYLD (upper band), BCL10, and RELB while expression of the NF-κB target gene IRF4 is suppressed.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01245&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. MALT1 inhibitor MLT-231 blocks T cell-dependent B cell responses. (A) <i>In vivo</i> proof of concept was evaluated using DNP-KLH-induced antibody response in Balb/c mice treated twice daily (12 h interval) with vehicle or MLT-231 at 10, 30, or 100 mg/kg, as compared to mice treated once daily with CsA at 30 mg/kg. Anti-DNP IgM and IgG titers were assessed 8 days after DNP-KLH immunization. Full efficacy as measured by reduced anti-DNP antibody production was reached at 100 mg/kg BID, comparable to Cyclosporine A (CsA). (B) Dose-dependent efficacy was achieved when comparing the percentage of inhibition of anti-DNP IgM and IgG release in plasma <i>vs</i> the 12 h-trough compound blood exposure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01245&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. PK/PD study of the MALT1 inhibitor MLT-231 in ABC-DLBCL xenograft models. (A) PK/PD study of MLT-231 in TMD8 xenograft model (CD79B mutant) after one single oral dose (100 mg/kg). Mean plasma concentration ± SD (blue) and % tumor IL-10 ± SD (upper panel) and % tumor BCL10 ± SD (lower panel) are shown (<i>n</i> = 3 animals per timepoint). (B) Efficacy study in OCI-Ly10 xenograft model (CD79A mutant) after oral administration of either vehicle control or MLT-231 at indicated doses and schedules. Mean tumor volumes ± SEM are shown (<i>n</i> = 8; *<i>p</i> < 0.05 <i>vs</i> vehicle using Kruskal–Wallis, Dunn’s post hoc). (C) Pharmacokinetics of MLT-231 in plasma and tumor post last dose. Mean compound concentrations ± SD are shown in plasma (blue) and tumor (red) from two treated mice per timepoint (<i>n</i> = 2). (D) PD post last dose was monitored by BCL10 levels in tumor lysates (<i>n</i> = 2 mice per timepoint).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01245&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds from <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01245&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) R–NCO, THF, rt, 53–84% yield or (ii) <i>p</i>-nitro-phenyl chloroformate, 1,2-DCE, collidine, 0 °C then aniline addition, 90 °C, 33–41% yield. Reagents and conditions: (iii) 3-chloro-4-methoxyaniline, HATU, DIPEA, DMF, rt, 38% yield; (iv) 2-(3-chloro-4-methoxyphenyl)acetic acid, (COCl)<sub>2</sub>, cat. DMF, TEA, DCM, 0 °C, 43% yield.</p></p></figure><figure data-id="sch3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds from <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01245&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) amine, TEA, THF, 0 °C to rt, 30–80% yield; (ii) Fe powder, AcOH, rt, 92–95% yield; (iii) R–NCO, THF, rt, 27–40% yield; (iv) 4-fluoro or 4-nitro-phenyl chloroformate, 1,2-DCE, collidine then aniline, 26–60% yield.</p></p></figure><figure data-id="sch4" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/medium/jm0c01245_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds from <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01245/20201203/images/large/jm0c01245_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01245&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) subst. amino-pyrazole, cat. pTsOH, THF, 53–69% yield; (ii) AcOH, 110 °C, 90–91% yield; (iii) POCl3, DIPEA, 110 °C then TEA, DMAP, substit. morpholine, DCM, −10 °C, 24–56% yield; (iv) SnCl<sub>2</sub>, HCl, EtOH, rt, 97%—quant.; (v) <i>p</i>-F-Ph-COCl, pyridine, THF. rt then aniline, 70 °C, 23–68% yield.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i89">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45175" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45175" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 25 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Uren, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Rourke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aravind, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pisabarro, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seshagiri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koonin, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixit, V. M.</span></span> <span> </span><span class="NLM_article-title">Identification of Paracaspases and Metacaspases: Two Ancient Families of Caspase-Like Proteins, One of which Plays a Key Role in MALT Lymphoma</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">961</span>– <span class="NLM_lpage">967</span>, <span class="refDoi"> DOI: 10.1016/s1097-2765(00)00094-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1016%2Fs1097-2765%2800%2900094-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=11090634" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnvVentLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2000&pages=961-967&author=A.+G.+Urenauthor=K.+O%E2%80%99Rourkeauthor=L.+Aravindauthor=M.+T.+Pisabarroauthor=S.+Seshagiriauthor=E.+V.+Kooninauthor=V.+M.+Dixit&title=Identification+of+Paracaspases+and+Metacaspases%3A+Two+Ancient+Families+of+Caspase-Like+Proteins%2C+One+of+which+Plays+a+Key+Role+in+MALT+Lymphoma&doi=10.1016%2Fs1097-2765%2800%2900094-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of paracaspases and metacaspases: two ancient families of caspase-like proteins, one of which plays a key role in MALT lymphoma</span></div><div class="casAuthors">Uren, Anthony G.; O'Rourke, Karen; Aravind, L.; Pisabarro, M. Teresa; Seshagiri, Somasekar; Koonin, Eugene V.; Dixi, Vishva M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">961-967</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Caspases are cysteine proteases essential to apoptosis.  The authors have identified two families of caspase-like proteins, Paracaspases (found in metazoans and Dictyostelium) and metacaspases (found in plants, fungi, and protozoa).  Metazoan paracaspase prodomains contain a death domain and Ig domains.  Several plant metacaspase prodomains contain zinc finger motifs resembling those in the plant hypersensitive response/cell death protein Isd-1.  The human paracaspase prodomain binds Bcl10, a protein involved in the t(1;14)(p22;q32) translocation of mucosa-assocd. lymphoid tissue (MALT) lymphoma.  Another MALT lymphoma translocation, t(11;18)(q21;q21), fuses the IAP-2 gene to the MLT1/MALT1 locus, which encodes the human paracaspase.  The authors find that this fusion activates NF-κB and that the caspase domain is required for this function, since mutation of the conserved catalytic cysteine attenuates NF-κB activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqMj67zUtB4LVg90H21EOLACvtfcHk0lgNeSvF9E_FBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnvVentLc%253D&md5=c62e4cee69f1574d23182e52369200ca</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fs1097-2765%2800%2900094-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs1097-2765%252800%252900094-0%26sid%3Dliteratum%253Aachs%26aulast%3DUren%26aufirst%3DA.%2BG.%26aulast%3DO%25E2%2580%2599Rourke%26aufirst%3DK.%26aulast%3DAravind%26aufirst%3DL.%26aulast%3DPisabarro%26aufirst%3DM.%2BT.%26aulast%3DSeshagiri%26aufirst%3DS.%26aulast%3DKoonin%26aufirst%3DE.%2BV.%26aulast%3DDixit%26aufirst%3DV.%2BM.%26atitle%3DIdentification%2520of%2520Paracaspases%2520and%2520Metacaspases%253A%2520Two%2520Ancient%2520Families%2520of%2520Caspase-Like%2520Proteins%252C%2520One%2520of%2520which%2520Plays%2520a%2520Key%2520Role%2520in%2520MALT%2520Lymphoma%26jtitle%3DMol.%2520Cell%26date%3D2000%26volume%3D6%26spage%3D961%26epage%3D967%26doi%3D10.1016%2Fs1097-2765%2800%2900094-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Juilland, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thome, M.</span></span> <span> </span><span class="NLM_article-title">Holding All the CARDs: How MALT1 Controls CARMA/CARD-Dependent Signaling</span>. <i>Front. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1927</span>, <span class="refDoi"> DOI: 10.3389/fimmu.2018.01927</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.3389%2Ffimmu.2018.01927" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=30214442" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFSrsrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1927&author=M.+Juillandauthor=M.+Thome&title=Holding+All+the+CARDs%3A+How+MALT1+Controls+CARMA%2FCARD-Dependent+Signaling&doi=10.3389%2Ffimmu.2018.01927"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Holding all the CARDs: how MALT1 controls CARMA/CARD-dependent signaling</span></div><div class="casAuthors">Juilland, Melanie; Thome, Margot</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Immunology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1927/1-1927/15</span>CODEN:
                <span class="NLM_cas:coden">FIRMCW</span>;
        ISSN:<span class="NLM_cas:issn">1664-3224</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">A review.  The scaffold proteins CARMA1-3 (encoded by the genes CARD11, -14 and -10) and CARD9 play major roles in signaling downstream of receptors with immunoreceptor tyrosine activation motifs (ITAMs), G-protein coupled receptors (GPCR) and receptor tyrosine kinases (RTK).  These receptors trigger the formation of oligomeric CARMA/CARD-BCL10-MALT1 (CBM) complexes via kinases of the PKC family.  The CBM in turn regulates gene expression by the activation of NF-κB and AP-1 transcription factors and controls transcript stability.  The paracaspase MALT1 is the only CBM component having an enzymic (proteolytic) activity and has therefore recently gained attention as a potential drug target.  Here we review recent advances in the understanding of the mol. function of the protease MALT1 and summarize how MALT1 scaffold and protease function contribute to the transmission of CBM signals.  Finally, we will highlight how dysregulation of MALT1 function can cause pathologies such as immunodeficiency, autoimmunity, psoriasis, and cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlFMkGmTP2YrVg90H21EOLACvtfcHk0lgNeSvF9E_FBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFSrsrrN&md5=55c17de3618a769f61cde83eee85f343</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3389%2Ffimmu.2018.01927&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffimmu.2018.01927%26sid%3Dliteratum%253Aachs%26aulast%3DJuilland%26aufirst%3DM.%26aulast%3DThome%26aufirst%3DM.%26atitle%3DHolding%2520All%2520the%2520CARDs%253A%2520How%2520MALT1%2520Controls%2520CARMA%252FCARD-Dependent%2520Signaling%26jtitle%3DFront.%2520Immunol.%26date%3D2018%26volume%3D9%26spage%3D1927%26doi%3D10.3389%2Ffimmu.2018.01927" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruland, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartjes, L.</span></span> <span> </span><span class="NLM_article-title">CARD-BCL-10-MALT1 Signalling in Protective and Pathological Immunity</span>. <i>Nat. Rev. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">118</span>– <span class="NLM_lpage">134</span>, <span class="refDoi"> DOI: 10.1038/s41577-018-0087-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1038%2Fs41577-018-0087-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=30467369" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnt1Wlu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2019&pages=118-134&author=J.+Rulandauthor=L.+Hartjes&title=CARD-BCL-10-MALT1+Signalling+in+Protective+and+Pathological+Immunity&doi=10.1038%2Fs41577-018-0087-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">CARD-BCL-10-MALT1 signalling in protective and pathological immunity</span></div><div class="casAuthors">Ruland, Juergen; Hartjes, Lara</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Immunology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">118-134</span>CODEN:
                <span class="NLM_cas:coden">NRIABX</span>;
        ISSN:<span class="NLM_cas:issn">1474-1733</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  CARD protein-BCL-10-MALT1 (CBM) signalosomes are multiprotein signalling platforms that control immune and inflammatory pathways in most tissues.  After exposure to distinct immune triggers, these mols. form self-organizing filaments with MALT1 protease activity to regulate canonical nuclear factor-κB (NF-κB) and mitogen-activated protein kinase (MAPK) signalling pathways and the degrdn. of mRNA-binding proteins, which provides two layers of control of inflammatory gene expression.  These CBM-regulated mechanisms are essential for host defense and tissue homeostasis, and numerous genetic alterations in CBM signalling components have been implicated in inherited and acquired immune-mediated diseases.  This Review discusses the regulation and signalling of CBM complexes, their physiol. roles and their pathophysiol. functions in human immunodeficiency diseases, inflammatory disorders and cancers of the immune system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTdKYas0vj17Vg90H21EOLACvtfcHk0lj_NX3B5x3FVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnt1Wlu7Y%253D&md5=64a4447502af72534ff4c955813dbe2b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fs41577-018-0087-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41577-018-0087-2%26sid%3Dliteratum%253Aachs%26aulast%3DRuland%26aufirst%3DJ.%26aulast%3DHartjes%26aufirst%3DL.%26atitle%3DCARD-BCL-10-MALT1%2520Signalling%2520in%2520Protective%2520and%2520Pathological%2520Immunity%26jtitle%3DNat.%2520Rev.%2520Immunol.%26date%3D2019%26volume%3D19%26spage%3D118%26epage%3D134%26doi%3D10.1038%2Fs41577-018-0087-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Du, M.-Q.</span></span> <span> </span><span class="NLM_article-title">MALT lymphoma: A paradigm of NF-κB dysregulation</span>. <i>Semin. Canc. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">60</span>, <span class="refDoi"> DOI: 10.1016/j.semcancer.2016.07.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1016%2Fj.semcancer.2016.07.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=27452667" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1ygsr%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2016&pages=49-60&author=M.-Q.+Du&title=MALT+lymphoma%3A+A+paradigm+of+NF-%CE%BAB+dysregulation&doi=10.1016%2Fj.semcancer.2016.07.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">MALT lymphoma: A paradigm of NF-κB dysregulation</span></div><div class="casAuthors">Du, Ming-Qing</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Cancer Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">49-60</span>CODEN:
                <span class="NLM_cas:coden">SECBE7</span>;
        ISSN:<span class="NLM_cas:issn">1044-579X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Extranodal marginal zone lymphoma of mucosa-assocd. lymphoid tissue (MALT lymphoma) invariably arises from a background of chronic microbial infection and/or autoimmune disorder at diverse mucosal sites.  The prolonged chronic infection and/or autoimmunity generate active immune and inflammatory responses that provide a setting for evolution and development of autoreactive B-cells, their expansion and eventual malignant transformation following acquisition of genetic changes.  The immune responses also play a crit. role in sustaining the growth and survival of the transformed cells as shown by complete regression of a high proportion of MALT lymphoma of the stomach, ocular adnexa and skin following anti-microbial treatment.  B-cell receptor engagement by auto-antigen as well as T-cell help including both cognate interaction and bystander help via sol. ligands such as CD40L and BAFF are thought to underpin the immunol. drive in the lymphoma development through activation of the canonical and non-canonical NF-κB pathway resp.  Similarly, the three MALT lymphoma assocd. chromosome translocations, namely t(1;14)(p22;q32)/BCL10-IGH, t(14;18)(q32;q21)/IGH-MALT1,and t(11;18)(q21;q21)/BIRC3 (API2)-MALT1, are also capable of activating both canonical and non-canonical NF-κB pathways.  Furthermore, TNFAIP3 (A20) inactivation by deletion and/or mutation abolishes the auto-neg. feedback to several signalling including BCR and TLR, which connect to the canonical NF-κB activation pathway.  Thus, there is a considerable overlap in the mol. pathways dysregulated by immunol. drive and somatic genetic changes, strongly arguing for their oncogenic cooperation in the development of MALT lymphoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4mYwLSwRG5rVg90H21EOLACvtfcHk0lj_NX3B5x3FVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1ygsr%252FM&md5=400f8c53b8fb19e314929c53955a27b6</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.semcancer.2016.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.semcancer.2016.07.003%26sid%3Dliteratum%253Aachs%26aulast%3DDu%26aufirst%3DM.-Q.%26atitle%3DMALT%2520lymphoma%253A%2520A%2520paradigm%2520of%2520NF-%25CE%25BAB%2520dysregulation%26jtitle%3DSemin.%2520Canc.%2520Biol.%26date%3D2016%26volume%3D39%26spage%3D49%26epage%3D60%26doi%3D10.1016%2Fj.semcancer.2016.07.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staudt, L. M.</span></span> <span> </span><span class="NLM_article-title">Aggressive Lymphomas</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>362</i></span>,  <span class="NLM_fpage">1417</span>– <span class="NLM_lpage">1429</span>, <span class="refDoi"> DOI: 10.1056/NEJMra0807082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1056%2FNEJMra0807082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=20393178" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkslajtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=362&publication_year=2010&pages=1417-1429&author=G.+Lenzauthor=L.+M.+Staudt&title=Aggressive+Lymphomas&doi=10.1056%2FNEJMra0807082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Aggressive lymphomas</span></div><div class="casAuthors">Lenz, Georg; Staudt, Louis M.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">362</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1417-1429</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review.  Given the emergence of many new therapeutic agents that affect essential regulatory pathways in lymphomas, the challenge is to identify rational combinations that kill lymphoma cells synergistically.  This effort requires both preclin. evaluation and the development of clin.-trial strategies that include mol. profiling.  Gene expression profiling could be used to det. the lymphoma subtype and to est. the risk assocd. with std. therapy.  Cancer gene resequencing could be used to identify tumors that are likely to be dependent on a particular signaling pathway and to det. which step in the pathway to inhibit.  Finally, the activity of the targeted pathways before and during therapy could be used to assess the efficacy of the treatment.  We expect that this "divide-and-conquer" approach will provide increasingly effective and nontoxic therapies for patients with lymphomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAtbs8gSY6hrVg90H21EOLACvtfcHk0lj_NX3B5x3FVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkslajtbY%253D&md5=005333ef8156a770b418a07aae2f19d8</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1056%2FNEJMra0807082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra0807082%26sid%3Dliteratum%253Aachs%26aulast%3DLenz%26aufirst%3DG.%26aulast%3DStaudt%26aufirst%3DL.%2BM.%26atitle%3DAggressive%2520Lymphomas%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2010%26volume%3D362%26spage%3D1417%26epage%3D1429%26doi%3D10.1056%2FNEJMra0807082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pissot Soldermann, C.</span>; <span class="NLM_string-name">Quancard, J.</span>; <span class="NLM_string-name">Schlapbach, A.</span>; <span class="NLM_string-name">Simic, O.</span>; <span class="NLM_string-name">Tintelnot-Blomley, M.</span>; <span class="NLM_string-name">Zoller, T.</span></span> <span> </span><span class="NLM_article-title">Novel Pyrazolo Pyrimidine Derivatives and their use as MALT1 Inhibitors</span>. PCT Int. Appl., <span class="NLM_patent">WO2015181747A1</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=C.+Pissot+Soldermann&author=J.+Quancard&author=A.+Schlapbach&author=O.+Simic&author=M.+Tintelnot-Blomley&author=T.+Zoller&title=Novel+Pyrazolo+Pyrimidine+Derivatives+and+their+use+as+MALT1+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DPissot%2BSoldermann%26aufirst%3DC.%26atitle%3DNovel%2520Pyrazolo%2520Pyrimidine%2520Derivatives%2520and%2520their%2520use%2520as%2520MALT1%2520Inhibitors%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spranger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincendeau, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grau, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raffegerst, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kloo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hlahla, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuenschwander, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peter von Kries, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadian, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dörken, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schendel, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krappmann, D.</span></span> <span> </span><span class="NLM_article-title">Pharmacologic Inhibition of MALT1 Protease by Phenothiazines as a Therapeutic Approach for the Treatment of Aggressive ABC-DLBCL</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">825</span>– <span class="NLM_lpage">837</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2012.11.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1016%2Fj.ccr.2012.11.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=23238017" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVWns77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=825-837&author=D.+Nagelauthor=S.+Sprangerauthor=M.+Vincendeauauthor=M.+Grauauthor=S.+Raffegerstauthor=B.+Klooauthor=D.+Hlahlaauthor=M.+Neuenschwanderauthor=J.+Peter+von+Kriesauthor=K.+Hadianauthor=B.+D%C3%B6rkenauthor=P.+Lenzauthor=G.+Lenzauthor=D.+J.+Schendelauthor=D.+Krappmann&title=Pharmacologic+Inhibition+of+MALT1+Protease+by+Phenothiazines+as+a+Therapeutic+Approach+for+the+Treatment+of+Aggressive+ABC-DLBCL&doi=10.1016%2Fj.ccr.2012.11.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacologic Inhibition of MALT1 Protease by Phenothiazines as a Therapeutic Approach for the Treatment of Aggressive ABC-DLBCL</span></div><div class="casAuthors">Nagel, Daniel; Spranger, Stefani; Vincendeau, Michelle; Grau, Michael; Raffegerst, Silke; Kloo, Bernhard; Hlahla, Daniela; Neuenschwander, Martin; von Kries, Jens Peter; Hadian, Kamyar; Doerken, Bernd; Lenz, Peter; Lenz, Georg; Schendel, Dolores J.; Krappmann, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">825-837</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Proteolytic activity of the mucosa-assocd. lymphoid tissue lymphoma translocation protein-1 (MALT1) paracaspase is required for survival of the activated B cell subtype of diffuse large B cell lymphoma (ABC-DLBCL).  We have identified distinct derivs. of medicinal active phenothiazines, namely mepazine, thioridazine, and promazine, as small mol. inhibitors of the MALT1 protease.  These phenothiazines selectively inhibit cleavage activity of recombinant and cellular MALT1 by a noncompetitive mechanism.  Consequently, the compds. inhibit anti-apoptotic NF-κB signaling and elicit toxic effects selectively on MALT1-dependent ABC-DLBCL cells in vitro and in vivo.  Our data provide a conceptual proof for a clin. application of distinct phenothiazines in the treatment of ABC-DLBCL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-pha_0UjHHLVg90H21EOLACvtfcHk0lj_NX3B5x3FVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVWns77J&md5=e67e728b09fbaf08defbaf06c1258be9</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2012.11.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2012.11.002%26sid%3Dliteratum%253Aachs%26aulast%3DNagel%26aufirst%3DD.%26aulast%3DSpranger%26aufirst%3DS.%26aulast%3DVincendeau%26aufirst%3DM.%26aulast%3DGrau%26aufirst%3DM.%26aulast%3DRaffegerst%26aufirst%3DS.%26aulast%3DKloo%26aufirst%3DB.%26aulast%3DHlahla%26aufirst%3DD.%26aulast%3DNeuenschwander%26aufirst%3DM.%26aulast%3DPeter%2Bvon%2BKries%26aufirst%3DJ.%26aulast%3DHadian%26aufirst%3DK.%26aulast%3DD%25C3%25B6rken%26aufirst%3DB.%26aulast%3DLenz%26aufirst%3DP.%26aulast%3DLenz%26aufirst%3DG.%26aulast%3DSchendel%26aufirst%3DD.%2BJ.%26aulast%3DKrappmann%26aufirst%3DD.%26atitle%3DPharmacologic%2520Inhibition%2520of%2520MALT1%2520Protease%2520by%2520Phenothiazines%2520as%2520a%2520Therapeutic%2520Approach%2520for%2520the%2520Treatment%2520of%2520Aggressive%2520ABC-DLBCL%26jtitle%3DCancer%2520Cell%26date%3D2012%26volume%3D22%26spage%3D825%26epage%3D837%26doi%3D10.1016%2Fj.ccr.2012.11.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schlapbach, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revesz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pissot Soldermann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoller, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Régnier, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bornancin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radimerski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blank, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renatus, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erbel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melkko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simic, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endres, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wartmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quancard, J.</span></span> <span> </span><span class="NLM_article-title">N-Aryl-Piperidine-4-Carboxamides as a Novel Class of Potent Inhibitors of MALT1 Proteolytic Activity</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">2153</span>– <span class="NLM_lpage">2158</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.05.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1016%2Fj.bmcl.2018.05.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=29759726" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC1cXps1Crsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=2153-2158&author=A.+Schlapbachauthor=L.+Reveszauthor=C.+Pissot+Soldermannauthor=T.+Zollerauthor=C.+H.+R%C3%A9gnierauthor=F.+Bornancinauthor=T.+Radimerskiauthor=J.+Blankauthor=A.+Schuffenhauerauthor=M.+Renatusauthor=P.+Erbelauthor=S.+Melkkoauthor=R.+Hengauthor=O.+Simicauthor=R.+Endresauthor=M.+Wartmannauthor=J.+Quancard&title=N-Aryl-Piperidine-4-Carboxamides+as+a+Novel+Class+of+Potent+Inhibitors+of+MALT1+Proteolytic+Activity&doi=10.1016%2Fj.bmcl.2018.05.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">N-aryl-piperidine-4-carboxamides as a novel class of potent inhibitors of MALT1 proteolytic activity</span></div><div class="casAuthors">Schlapbach, Achim; Revesz, Laszlo; Pissot Soldermann, Carole; Zoller, Thomas; Regnier, Catherine H.; Bornancin, Frederic; Radimerski, Thomas; Blank, Jutta; Schuffenhauer, Ansgar; Renatus, Martin; Erbel, Paulus; Melkko, Samu; Heng, Richard; Simic, Oliver; Endres, Ralf; Wartmann, Markus; Quancard, Jean</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2153-2158</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Starting from a weak screening hit, potent and selective inhibitors of the MALT1 protease function were elaborated.  Advanced compds. displayed high potency in biochem. and cellular assays.  Compds. showed activity in a mechanistic Jurkat T cell activation assay as well as in the B-cell lymphoma line OCI-Ly3, which suggests potential use of MALT1 inhibitors in the treatment of autoimmune diseases as well as B-cell lymphomas with a dysregulated NF-κB pathway.  Initially, rat pharmacokinetic properties of this compd. series were dominated by very high clearance which could be linked to amide cleavage.  Using a rat hepatocyte assay a good in vitro-in vivo correlation could be established which led to the identification of compds. with improved PK properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4Gx09NEhsQ7Vg90H21EOLACvtfcHk0lgWgUU3MuxWsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXps1Crsbc%253D&md5=f44b5e6bee523fbf3d5d692e0ce9c49c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.05.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.05.017%26sid%3Dliteratum%253Aachs%26aulast%3DSchlapbach%26aufirst%3DA.%26aulast%3DRevesz%26aufirst%3DL.%26aulast%3DPissot%2BSoldermann%26aufirst%3DC.%26aulast%3DZoller%26aufirst%3DT.%26aulast%3DR%25C3%25A9gnier%26aufirst%3DC.%2BH.%26aulast%3DBornancin%26aufirst%3DF.%26aulast%3DRadimerski%26aufirst%3DT.%26aulast%3DBlank%26aufirst%3DJ.%26aulast%3DSchuffenhauer%26aufirst%3DA.%26aulast%3DRenatus%26aufirst%3DM.%26aulast%3DErbel%26aufirst%3DP.%26aulast%3DMelkko%26aufirst%3DS.%26aulast%3DHeng%26aufirst%3DR.%26aulast%3DSimic%26aufirst%3DO.%26aulast%3DEndres%26aufirst%3DR.%26aulast%3DWartmann%26aufirst%3DM.%26aulast%3DQuancard%26aufirst%3DJ.%26atitle%3DN-Aryl-Piperidine-4-Carboxamides%2520as%2520a%2520Novel%2520Class%2520of%2520Potent%2520Inhibitors%2520of%2520MALT1%2520Proteolytic%2520Activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D28%26spage%3D2153%26epage%3D2158%26doi%3D10.1016%2Fj.bmcl.2018.05.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bardet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unterreiner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malinverni, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lafossas, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedrine, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boesch, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glück, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katopodis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renatus, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simic, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlapbach, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quancard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Régnier, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bold, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pissot-Soldermann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carballido, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovarik, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calzascia, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bornancin, F.</span></span> <span> </span><span class="NLM_article-title">The T-Cell Fingerprint of MALT1 Paracaspase Revealed by Selective Inhibition</span>. <i>Immunol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">81</span>– <span class="NLM_lpage">99</span>, <span class="refDoi"> DOI: 10.1111/imcb.1018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1111%2Fimcb.1018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=29359407" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjs1Kitb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=2018&pages=81-99&author=M.+Bardetauthor=A.+Unterreinerauthor=C.+Malinverniauthor=F.+Lafossasauthor=C.+Vedrineauthor=D.+Boeschauthor=Y.+Kolbauthor=D.+Kaiserauthor=A.+Gl%C3%BCckauthor=M.+A.+Schneiderauthor=A.+Katopodisauthor=M.+Renatusauthor=O.+Simicauthor=A.+Schlapbachauthor=J.+Quancardauthor=C.+H.+R%C3%A9gnierauthor=G.+Boldauthor=C.+Pissot-Soldermannauthor=J.+M.+Carballidoauthor=J.+Kovarikauthor=T.+Calzasciaauthor=F.+Bornancin&title=The+T-Cell+Fingerprint+of+MALT1+Paracaspase+Revealed+by+Selective+Inhibition&doi=10.1111%2Fimcb.1018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The T-cell fingerprint of MALT1 paracaspase revealed by selective inhibition</span></div><div class="casAuthors">Bardet, Maureen; Unterreiner, Adeline; Malinverni, Claire; Lafossas, Frederique; Vedrine, Corinne; Boesch, Danielle; Kolb, Yeter; Kaiser, Daniel; Glueck, Anton; Schneider, Martin A.; Katopodis, Andreas; Renatus, Martin; Simic, Oliver; Schlapbach, Achim; Quancard, Jean; Regnier, Catherine H.; Bold, Guido; Pissot-Soldermann, Carole; Carballido, Jose M.; Kovarik, Jiri; Calzascia, Thomas; Bornancin, Frederic</div><div class="citationInfo"><span class="NLM_cas:title">Immunology & Cell Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">81-99</span>CODEN:
                <span class="NLM_cas:coden">ICBIEZ</span>;
        ISSN:<span class="NLM_cas:issn">1440-1711</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Australia, Ltd.</span>)
        </div><div class="casAbstract">Mucosa-assocd. lymphoid tissue lymphoma translocation protein 1 (MALT1) is essential for immune responses triggered by antigen receptors but the contribution of its paracaspase activity is not fully understood.  Here, we studied how MALT1 proteolytic function regulates T-cell activation and fate after engagement of the T-cell receptor pathway.  We show that MLT-827, a potent and selective MALT1 paracaspase inhibitor, does not prevent the initial phase of T-cell activation, in contrast to the pan-protein kinase C inhibitor AEB071.  However, MLT-827 strongly impacted cell expansion after activation.  We demonstrate this is the consequence of profound inhibition of IL-2 prodn. as well as reduced expression of the IL-2 receptor alpha subunit (CD25), resulting from defective canonical NF-κB activation and accelerated mRNA turnover mechanisms.  Accordingly, MLT-827 revealed a unique transcriptional fingerprint of MALT1 protease activity, providing evidence for broad control of T-cell signaling pathways.  Altogether, this first report with a potent and selective inhibitor elucidates how MALT1 paracaspase activity integrates several T-cell activation pathways and indirectly controls gamma-chain receptor dependent survival, to impact on T-cell expansion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcOZ3C5-LsPrVg90H21EOLACvtfcHk0lgWgUU3MuxWsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjs1Kitb0%253D&md5=0f35cd2ae66e3572e815eb24aacd6445</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1111%2Fimcb.1018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fimcb.1018%26sid%3Dliteratum%253Aachs%26aulast%3DBardet%26aufirst%3DM.%26aulast%3DUnterreiner%26aufirst%3DA.%26aulast%3DMalinverni%26aufirst%3DC.%26aulast%3DLafossas%26aufirst%3DF.%26aulast%3DVedrine%26aufirst%3DC.%26aulast%3DBoesch%26aufirst%3DD.%26aulast%3DKolb%26aufirst%3DY.%26aulast%3DKaiser%26aufirst%3DD.%26aulast%3DGl%25C3%25BCck%26aufirst%3DA.%26aulast%3DSchneider%26aufirst%3DM.%2BA.%26aulast%3DKatopodis%26aufirst%3DA.%26aulast%3DRenatus%26aufirst%3DM.%26aulast%3DSimic%26aufirst%3DO.%26aulast%3DSchlapbach%26aufirst%3DA.%26aulast%3DQuancard%26aufirst%3DJ.%26aulast%3DR%25C3%25A9gnier%26aufirst%3DC.%2BH.%26aulast%3DBold%26aufirst%3DG.%26aulast%3DPissot-Soldermann%26aufirst%3DC.%26aulast%3DCarballido%26aufirst%3DJ.%2BM.%26aulast%3DKovarik%26aufirst%3DJ.%26aulast%3DCalzascia%26aufirst%3DT.%26aulast%3DBornancin%26aufirst%3DF.%26atitle%3DThe%2520T-Cell%2520Fingerprint%2520of%2520MALT1%2520Paracaspase%2520Revealed%2520by%2520Selective%2520Inhibition%26jtitle%3DImmunol.%2520Cell%2520Biol.%26date%3D2018%26volume%3D96%26spage%3D81%26epage%3D99%26doi%3D10.1111%2Fimcb.1018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quancard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fung, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renatus, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israël, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Priatel, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blank, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viner, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blank, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlapbach, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erbel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kizhakkedathu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villard, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersperger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turvey, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bornancin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overall, C. M.</span></span> <span> </span><span class="NLM_article-title">An Allosteric MALT1 Inhibitor is a Molecular Corrector Rescuing Function in an Immunodeficient Patient</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">313</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0222-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1038%2Fs41589-018-0222-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=30692685" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmtF2isbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=304-313&author=J.+Quancardauthor=T.+Kleinauthor=S.-Y.+Fungauthor=M.+Renatusauthor=N.+Hughesauthor=L.+Isra%C3%ABlauthor=J.+J.+Priatelauthor=S.+Kangauthor=M.+A.+Blankauthor=R.+I.+Vinerauthor=J.+Blankauthor=A.+Schlapbachauthor=P.+Erbelauthor=J.+Kizhakkedathuauthor=F.+Villardauthor=R.+Herspergerauthor=S.+E.+Turveyauthor=J.+Ederauthor=F.+Bornancinauthor=C.+M.+Overall&title=An+Allosteric+MALT1+Inhibitor+is+a+Molecular+Corrector+Rescuing+Function+in+an+Immunodeficient+Patient&doi=10.1038%2Fs41589-018-0222-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">An allosteric MALT1 inhibitor is a molecular corrector rescuing function in an immunodeficient patient</span></div><div class="casAuthors">Quancard, Jean; Klein, Theo; Fung, Shan-Yu; Renatus, Martin; Hughes, Nicola; Israel, Laura; Priatel, John J.; Kang, Sohyeong; Blank, Michael A.; Viner, Rosa I.; Blank, Jutta; Schlapbach, Achim; Erbel, Paul; Kizhakkedathu, Jayachandran; Villard, Frederic; Hersperger, Rene; Turvey, Stuart E.; Eder, Joerg; Bornancin, Frederic; Overall, Christopher M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">304-313</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">MALT1 paracaspase is central for lymphocyte antigen-dependent responses including NF-κB activation.  We discovered nanomolar, selective allosteric inhibitors of MALT1 that bind by displacing the side chain of Trp580, locking the protease in an inactive conformation.  Interestingly, we had previously identified a patient homozygous for a MALT1 Trp580-to-serine mutation who suffered from combined immunodeficiency.  We show that the loss of tryptophan weakened interactions between the paracaspase and C-terminal Ig MALT1 domains resulting in protein instability, reduced protein levels and functions.  Upon binding of allosteric inhibitors of increasing potency, we found proportionate increased stabilization of MALT1-W580S to reach that of wild-type MALT1.  With restored levels of stable MALT1 protein, the most potent of the allosteric inhibitors rescued NF-κB and JNK signaling in patient lymphocytes.  Following compd. washout, MALT1 substrate cleavage was partly recovered.  Thus, a mol. corrector rescues an enzyme deficiency by substituting for the mutated residue, inspiring new potential precision therapies to increase mutant enzyme activity in other deficiencies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoowtKlWIeaBbVg90H21EOLACvtfcHk0ljXtZ-OiDThoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmtF2isbo%253D&md5=923fffe83ed681c62ee0164a2d18775e</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0222-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0222-1%26sid%3Dliteratum%253Aachs%26aulast%3DQuancard%26aufirst%3DJ.%26aulast%3DKlein%26aufirst%3DT.%26aulast%3DFung%26aufirst%3DS.-Y.%26aulast%3DRenatus%26aufirst%3DM.%26aulast%3DHughes%26aufirst%3DN.%26aulast%3DIsra%25C3%25ABl%26aufirst%3DL.%26aulast%3DPriatel%26aufirst%3DJ.%2BJ.%26aulast%3DKang%26aufirst%3DS.%26aulast%3DBlank%26aufirst%3DM.%2BA.%26aulast%3DViner%26aufirst%3DR.%2BI.%26aulast%3DBlank%26aufirst%3DJ.%26aulast%3DSchlapbach%26aufirst%3DA.%26aulast%3DErbel%26aufirst%3DP.%26aulast%3DKizhakkedathu%26aufirst%3DJ.%26aulast%3DVillard%26aufirst%3DF.%26aulast%3DHersperger%26aufirst%3DR.%26aulast%3DTurvey%26aufirst%3DS.%2BE.%26aulast%3DEder%26aufirst%3DJ.%26aulast%3DBornancin%26aufirst%3DF.%26aulast%3DOverall%26aufirst%3DC.%2BM.%26atitle%3DAn%2520Allosteric%2520MALT1%2520Inhibitor%2520is%2520a%2520Molecular%2520Corrector%2520Rescuing%2520Function%2520in%2520an%2520Immunodeficient%2520Patient%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D304%26epage%3D313%26doi%3D10.1038%2Fs41589-018-0222-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatcher, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontán, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Us, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casalena, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Quinoline and Thiazolopyridine Allosteric Inhibitors of MALT1</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1694</span>– <span class="NLM_lpage">1698</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.05.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1016%2Fj.bmcl.2019.05.040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=31129051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVKqtb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=1694-1698&author=D.+A.+Scottauthor=J.+M.+Hatcherauthor=H.+Liuauthor=M.+Fuauthor=G.+Duauthor=L.+Font%C3%A1nauthor=I.+Usauthor=G.+Casalenaauthor=Q.+Qiaoauthor=H.+Wuauthor=A.+Melnickauthor=N.+S.+Gray&title=Quinoline+and+Thiazolopyridine+Allosteric+Inhibitors+of+MALT1&doi=10.1016%2Fj.bmcl.2019.05.040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Quinoline and thiazolopyridine allosteric inhibitors of MALT1</span></div><div class="casAuthors">Scott, David A.; Hatcher, John M.; Liu, Hongyan; Fu, Mingpeng; Du, Guangyan; Fontan, Lorena; Us, Ilkay; Casalena, Gabriella; Qiao, Qi; Wu, Hao; Melnick, Ari; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1694-1698</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Quinolines and thiazolopyridines were developed as allosteric inhibitors of MALT1, with good cellular potency and exquisite selectivity.  Mouse pharmacokinetic (PK) profiling showed these to have low in vivo clearance, and moderate oral exposure.  The thiazolopyridines were less lipophilic than the quinolines, and one thiazolopyridine example was active in our hIL10 mouse pharmacodynamic (PD) model upon oral dosing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmNvxQMuqYn7Vg90H21EOLACvtfcHk0ljXtZ-OiDThoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVKqtb%252FN&md5=878f35e729000596b5eb77d979eb34d5</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.05.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.05.040%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BA.%26aulast%3DHatcher%26aufirst%3DJ.%2BM.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DFu%26aufirst%3DM.%26aulast%3DDu%26aufirst%3DG.%26aulast%3DFont%25C3%25A1n%26aufirst%3DL.%26aulast%3DUs%26aufirst%3DI.%26aulast%3DCasalena%26aufirst%3DG.%26aulast%3DQiao%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DQuinoline%2520and%2520Thiazolopyridine%2520Allosteric%2520Inhibitors%2520of%2520MALT1%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D1694%26epage%3D1698%26doi%3D10.1016%2Fj.bmcl.2019.05.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Unterreiner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Touil, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anastasi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubois, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niwa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calzascia, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bornancin, F.</span></span> <span> </span><span class="NLM_article-title">Myeloid Innate Signaling Pathway Regulation by MALT1 Paracaspase Activity</span>. <i>J. Visualized Exp.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>, <span class="NLM_elocation-id">e58439</span> <span class="refDoi"> DOI: 10.3791/58439</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.3791%2F58439" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtF2qt7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=A.+Unterreinerauthor=R.+Touilauthor=D.+Anastasiauthor=N.+Duboisauthor=S.+Niwaauthor=T.+Calzasciaauthor=F.+Bornancin&title=Myeloid+Innate+Signaling+Pathway+Regulation+by+MALT1+Paracaspase+Activity&doi=10.3791%2F58439"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Myeloid innate signaling pathway regulation by MALT1 paracaspase activity</span></div><div class="casAuthors">Unterreiner, Adeline; Touil, Ratiba; Anastasi, Daniele; Dubois, Nicolas; Niwa, Satoru; Calzascia, Thomas; Bornancin, Frederic</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Visualized Experiments</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue">143</span>),
    <span class="NLM_cas:pages">e58439</span>CODEN:
                <span class="NLM_cas:coden">JVEOA4</span>;
        ISSN:<span class="NLM_cas:issn">1940-087X</span>.
    
            (<span class="NLM_cas:orgname">Journal of Visualized Experiments</span>)
        </div><div class="casAbstract">Besides its function in lymphoid cells, which has been addressed by numerous studies, the paracaspase MALT1 also plays an important role in innate cells downstream of pattern recognition receptors.  Best studied are the Dectin-1 and Dectin-2 members of the C-type lectin-like receptor family that induce a SYK- and CARD9-dependent signaling cascade leading to NF-κB activation, in a MALT1-dependent manner.  By contrast, Toll-like receptors (TLR), such as TLR-4, propagate NF-κB activation but signal via an MYD88/IRAK-dependent cascade.  Nonetheless, whether MALT1 might contribute to TLR-4 signaling has remained unclear.  Recent evidence with MLT-827, a potent and selective inhibitor of MALT1 paracaspase activity, indicates that TNF-prodn. downstream of TLR-4 in human myeloid cells is independent of MALT1, as opposed to TNF-prodn. downstream of Dectin-1, which is MALT1 dependent.  Here, we addressed the selective involvement of MALT1 in pattern recognition sensing further, using a variety of human and mouse cellular prepns., and stimulation of Dectin-1, MINCLE or TLR-4 pathways.  We also provided addnl. insights by exploring cytokines beyond TNF-, and by comparing MLT-827 to a SYK inhibitor (Cpd11) and to an IKK inhibitor (AFN700).  Collectively, the data provided further evidence for the MALT1-dependency of C-type lectin-like receptor-signaling by contrast to TLR-signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9hS6I6nQ4_7Vg90H21EOLACvtfcHk0liII0B0HyqNWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtF2qt7nM&md5=5c4e736094fcf5dbb38d0e3f69aadb24</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.3791%2F58439&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3791%252F58439%26sid%3Dliteratum%253Aachs%26aulast%3DUnterreiner%26aufirst%3DA.%26aulast%3DTouil%26aufirst%3DR.%26aulast%3DAnastasi%26aufirst%3DD.%26aulast%3DDubois%26aufirst%3DN.%26aulast%3DNiwa%26aufirst%3DS.%26aulast%3DCalzascia%26aufirst%3DT.%26aulast%3DBornancin%26aufirst%3DF.%26atitle%3DMyeloid%2520Innate%2520Signaling%2520Pathway%2520Regulation%2520by%2520MALT1%2520Paracaspase%2520Activity%26jtitle%3DJ.%2520Visualized%2520Exp.%26date%3D2019%26doi%3D10.3791%2F58439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Israël, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huppertz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercado, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ginster, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unterreiner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlierf, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goetschy, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zerwes, H.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolbinger, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bornancin, F.</span></span> <span> </span><span class="NLM_article-title">A CARD10-Dependent Tonic Signalosome Activates MALT1 Paracaspase and Regulates IL-17/TNF-α-Driven Keratinocyte Inflammation</span>. <i>J. Invest. Dermatol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">2075</span>– <span class="NLM_lpage">2079</span>, <span class="refDoi"> DOI: 10.1016/j.jid.2018.03.1503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1016%2Fj.jid.2018.03.1503" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=29571942" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVyjsr3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2018&pages=2075-2079&author=L.+Isra%C3%ABlauthor=M.+Bardetauthor=A.+Huppertzauthor=N.+Mercadoauthor=S.+Ginsterauthor=A.+Unterreinerauthor=A.+Schlierfauthor=J.+F.+Goetschyauthor=H.-G.+Zerwesauthor=L.+Rothauthor=F.+Kolbingerauthor=F.+Bornancin&title=A+CARD10-Dependent+Tonic+Signalosome+Activates+MALT1+Paracaspase+and+Regulates+IL-17%2FTNF-%CE%B1-Driven+Keratinocyte+Inflammation&doi=10.1016%2Fj.jid.2018.03.1503"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A CARD10-Dependent Tonic Signalosome Activates MALT1 Paracaspase and Regulates IL-17/TNF-α-Driven Keratinocyte Inflammation</span></div><div class="casAuthors">Israel, Laura; Bardet, Maureen; Huppertz, Antje; Mercado, Nicolas; Ginster, Stefanie; Unterreiner, Adeline; Schlierf, Anita; Goetschy, Jean Francois; Zerwes, Hans-Gunter; Roth, Lukas; Kolbinger, Frank; Bornancin, Frederic</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Investigative Dermatology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2075-2079</span>CODEN:
                <span class="NLM_cas:coden">JIDEAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-202X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">This study investigated the role of MALT1 in normal human epidermal keratinocytes (NHEKs) stimulated by IL-17/TNF-α, a synergistic proinflammatory combination known to recapitulate key features of psoriasis.  A selective MALT1 inhibitor MLT-827 treated epidermal keratinocytes stimulation with IL-17A/TNF-α showed reduced IL-8 and IL17C induction by MLT-827.  Results show that MALT1 proteolytic function regulates the IL-17/TNF-α pathway in keratinocytes through the mechanism involving a CARD10-dependent pathway, likely through CBM complex formation between CARD10, BCL10, and MALT1A.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf4RyP7yR5arVg90H21EOLACvtfcHk0liII0B0HyqNWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVyjsr3M&md5=ae294c918bdbd030a8015f3149a4ccaa</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.jid.2018.03.1503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jid.2018.03.1503%26sid%3Dliteratum%253Aachs%26aulast%3DIsra%25C3%25ABl%26aufirst%3DL.%26aulast%3DBardet%26aufirst%3DM.%26aulast%3DHuppertz%26aufirst%3DA.%26aulast%3DMercado%26aufirst%3DN.%26aulast%3DGinster%26aufirst%3DS.%26aulast%3DUnterreiner%26aufirst%3DA.%26aulast%3DSchlierf%26aufirst%3DA.%26aulast%3DGoetschy%26aufirst%3DJ.%2BF.%26aulast%3DZerwes%26aufirst%3DH.-G.%26aulast%3DRoth%26aufirst%3DL.%26aulast%3DKolbinger%26aufirst%3DF.%26aulast%3DBornancin%26aufirst%3DF.%26atitle%3DA%2520CARD10-Dependent%2520Tonic%2520Signalosome%2520Activates%2520MALT1%2520Paracaspase%2520and%2520Regulates%2520IL-17%252FTNF-%25CE%25B1-Driven%2520Keratinocyte%2520Inflammation%26jtitle%3DJ.%2520Invest.%2520Dermatol.%26date%3D2018%26volume%3D138%26spage%3D2075%26epage%3D2079%26doi%3D10.1016%2Fj.jid.2018.03.1503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greengrass, P. M.</span>; <span class="NLM_string-name">Stewart, M.</span>; <span class="NLM_string-name">Wood, C. M.</span></span> <span> </span><span class="NLM_article-title">Affinity-Assay for the Human ERG Potassium Channel</span>. PCT Int. Appl. <span class="NLM_patent">WO2003021271A2</span>, <span class="NLM_year">2002</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2002&author=P.+M.+Greengrass&author=M.+Stewart&author=C.+M.+Wood&title=Affinity-Assay+for+the+Human+ERG+Potassium+Channel"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DGreengrass%26aufirst%3DP.%2BM.%26atitle%3DAffinity-Assay%2520for%2520the%2520Human%2520ERG%2520Potassium%2520Channel%26date%3D2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fermini, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fossa, A. A.</span></span> <span> </span><span class="NLM_article-title">The Impact of Drug-Induced QT Interval Prolongation on Drug Discovery and Development</span>. <i>Nat. Rev. Drug Discov.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">439</span>– <span class="NLM_lpage">447</span>, <span class="refDoi"> DOI: 10.1038/nrd1108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1038%2Fnrd1108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=12776219" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BD3sXktFOhu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=439-447&author=B.+Ferminiauthor=A.+A.+Fossa&title=The+Impact+of+Drug-Induced+QT+Interval+Prolongation+on+Drug+Discovery+and+Development&doi=10.1038%2Fnrd1108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The impact of drug-induced QT interval prolongation on drug discovery and development</span></div><div class="casAuthors">Fermini, Bernard; Fossa, Anthony A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">439-447</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  During the past decade, a no. of non-cardiovascular drugs have had their label revised or have been withdrawn from the market because of unexpected post-marketing reports of sudden cardiac death assocd. with a prolongation of the QT interval, and an increased propensity to develop a ventricular tachyarrhythmia called Torsades de Pointes.  Although a direct link between QT interval prolongation and arrhythmogenesis is still unclear, QT prolongation is now the subject of increased regulatory review and is considered a significant risk factor for predicting human safety of New Chem. Entities.  Consequently, pharmaceutical companies are striving to improve the drug discovery and development process to identify, as early as possible, the risk of novel agents, or their metabolites, of causing QT interval prolongation and to make appropriate go/no-go decisions or modify their development program accordingly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZk6HVMpiVQbVg90H21EOLACvtfcHk0lhcZOVDaZ4Blg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXktFOhu70%253D&md5=86f7948c80628e39f69a889e76fc0063</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrd1108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1108%26sid%3Dliteratum%253Aachs%26aulast%3DFermini%26aufirst%3DB.%26aulast%3DFossa%26aufirst%3DA.%2BA.%26atitle%3DThe%2520Impact%2520of%2520Drug-Induced%2520QT%2520Interval%2520Prolongation%2520on%2520Drug%2520Discovery%2520and%2520Development%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discov.%26date%3D2003%26volume%3D2%26spage%3D439%26epage%3D447%26doi%3D10.1038%2Fnrd1108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keating, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanguinetti, M. C.</span></span> <span> </span><span class="NLM_article-title">Molecular and Cellular Mechanisms of Cardiac Arrhythmias</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">569</span>– <span class="NLM_lpage">580</span>, <span class="refDoi"> DOI: 10.1016/s0092-8674(01)00243-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1016%2FS0092-8674%2801%2900243-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=11239413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BD3MXis1Kks7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2001&pages=569-580&author=M.+T.+Keatingauthor=M.+C.+Sanguinetti&title=Molecular+and+Cellular+Mechanisms+of+Cardiac+Arrhythmias&doi=10.1016%2Fs0092-8674%2801%2900243-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular and cellular mechanisms of cardiac arrhythmias</span></div><div class="casAuthors">Keating, Mark T.; Sanguinetti, Michael C.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">569-580</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review, with 71 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnj71g2km85bVg90H21EOLACvtfcHk0lhcZOVDaZ4Blg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXis1Kks7o%253D&md5=b77453d4075b1e560dd5062878ce76db</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2801%2900243-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252801%252900243-4%26sid%3Dliteratum%253Aachs%26aulast%3DKeating%26aufirst%3DM.%2BT.%26aulast%3DSanguinetti%26aufirst%3DM.%2BC.%26atitle%3DMolecular%2520and%2520Cellular%2520Mechanisms%2520of%2520Cardiac%2520Arrhythmias%26jtitle%3DCell%26date%3D2001%26volume%3D104%26spage%3D569%26epage%3D580%26doi%3D10.1016%2Fs0092-8674%2801%2900243-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pearlstein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rampe, D.</span></span> <span> </span><span class="NLM_article-title">Understanding the Structure–Activity Relationship of the Human Ether-a-go-go-Related Gene Cardiac K+Channel. A Model for Bad Behavior</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">2017</span>– <span class="NLM_lpage">2022</span>, <span class="refDoi"> DOI: 10.1021/jm0205651</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0205651" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjtlWhtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=2017-2022&author=R.+Pearlsteinauthor=R.+Vazauthor=D.+Rampe&title=Understanding+the+Structure%E2%80%93Activity+Relationship+of+the+Human+Ether-a-go-go-Related+Gene+Cardiac+K%2BChannel.+A+Model+for+Bad+Behavior&doi=10.1021%2Fjm0205651"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding the structure-activity relationship of the human ether-a-go-go-related gene cardiac K+ channel. A model for bad behavior</span></div><div class="casAuthors">Pearlstein, Robert; Vaz, Roy; Rampe, David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2017-2022</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with refs. on drug/HERG (human ether-a-go-go-related gene) interactions in relation to SAR for antiarrhythmics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIIRvL3sJJG7Vg90H21EOLACvtfcHk0ljNQI8QFy_S8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjtlWhtLw%253D&md5=351cc53cac39afafc4a4a4dbb65362af</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm0205651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0205651%26sid%3Dliteratum%253Aachs%26aulast%3DPearlstein%26aufirst%3DR.%26aulast%3DVaz%26aufirst%3DR.%26aulast%3DRampe%26aufirst%3DD.%26atitle%3DUnderstanding%2520the%2520Structure%25E2%2580%2593Activity%2520Relationship%2520of%2520the%2520Human%2520Ether-a-go-go-Related%2520Gene%2520Cardiac%2520K%252BChannel.%2520A%2520Model%2520for%2520Bad%2520Behavior%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D2017%26epage%3D2022%26doi%3D10.1021%2Fjm0205651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aronov, A.</span></span> <span> </span><span class="NLM_article-title">Predictive in Silico Modeling for hERG Channel Blockers</span>. <i>Drug Discov. Today</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">149</span>– <span class="NLM_lpage">155</span>, <span class="refDoi"> DOI: 10.1016/S1359-6446(04)03278-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1016%2FS1359-6446%2804%2903278-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=15718164" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhsFarsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=149-155&author=A.+Aronov&title=Predictive+in+Silico+Modeling+for+hERG+Channel+Blockers&doi=10.1016%2FS1359-6446%2804%2903278-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Predictive in silico modeling for hERG channel blockers</span></div><div class="casAuthors">Aronov, Alex M.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">149-155</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  HERG-mediated sudden death as a side effect of non-antiarrhythmic drugs has been receiving increased regulatory attention.  Perhaps owing to the unique shape of the ligand-binding site and its hydrophobic character, the hERG channel has been shown to interact with pharmaceuticals of widely varying structure.  Several in silico approaches have attempted to predict hERG channel blockade.  Some of these approaches are aimed primarily at filtering out potential hERG blockers in the context of virtual libraries, others involve understanding structure-activity relationships governing hERG-drug interactions.  This review summarizes the most recent efforts in this emerging field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDgIcgWdJQJrVg90H21EOLACvtfcHk0ljNQI8QFy_S8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhsFarsbY%253D&md5=112b79a48e3d6f5ab3e313f00cf2a4fa</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903278-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903278-7%26sid%3Dliteratum%253Aachs%26aulast%3DAronov%26aufirst%3DA.%26atitle%3DPredictive%2520in%2520Silico%2520Modeling%2520for%2520hERG%2520Channel%2520Blockers%26jtitle%3DDrug%2520Discov.%2520Today%26date%3D2005%26volume%3D10%26spage%3D149%26epage%3D155%26doi%3D10.1016%2FS1359-6446%2804%2903278-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Faller, B.</span>; <span class="NLM_string-name">Skolnik, S.</span>; <span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">ADMET Diagnosis Models</span>. In  <i>Predictive ADMET: Integrative Approaches in Drug Discovery and Development</i>; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>, <span class="NLM_year">2014</span>; pp  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">61</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1002%2F9781118783344.ch3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&pages=47-61&author=B.+Faller&author=S.+Skolnik&author=J.+Wang&title=Predictive+ADMET%3A+Integrative+Approaches+in+Drug+Discovery+and+Development"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2F9781118783344.ch3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118783344.ch3%26sid%3Dliteratum%253Aachs%26aulast%3DFaller%26aufirst%3DB.%26atitle%3DADMET%2520Diagnosis%2520Models%26btitle%3DPredictive%2520ADMET%253A%2520Integrative%2520Approaches%2520in%2520Drug%2520Discovery%2520and%2520Development%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2014%26spage%3D47%26epage%3D61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiesmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leder, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blank, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melkko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Decock, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Arcy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villard, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erbel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freuler, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolay, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alves, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bornancin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renatus, M.</span></span> <span> </span><span class="NLM_article-title">Structural Determinants of MALT1 Protease Activity</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>419</i></span>,  <span class="NLM_fpage">4</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2012.02.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1016%2Fj.jmb.2012.02.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=22366302" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC38Xkt1Ghur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=419&publication_year=2012&pages=4-21&author=C.+Wiesmannauthor=L.+Lederauthor=J.+Blankauthor=A.+Bernardiauthor=S.+Melkkoauthor=A.+Decockauthor=A.+D%E2%80%99Arcyauthor=F.+Villardauthor=P.+Erbelauthor=N.+Hughesauthor=F.+Freulerauthor=R.+Nikolayauthor=J.+Alvesauthor=F.+Bornancinauthor=M.+Renatus&title=Structural+Determinants+of+MALT1+Protease+Activity&doi=10.1016%2Fj.jmb.2012.02.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Determinants of MALT1 Protease Activity</span></div><div class="casAuthors">Wiesmann, Christian; Leder, Lukas; Blank, Jutta; Bernardi, Anna; Melkko, Samu; Decock, Arnaud; D'Arcy, Allan; Villard, Frederic; Erbel, Paulus; Hughes, Nicola; Freuler, Felix; Nikolay, Rainer; Alves, Juliano; Bornancin, Frederic; Renatus, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">419</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">4-21</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The formation of the CBM (CARD11-BCL10-MALT1) complex is pivotal for antigen-receptor-mediated activation of the transcription factor NF-κB.  Signaling is dependent on MALT1 (mucosa-assocd. lymphoid tissue lymphoma translocation protein 1), which not only acts as a scaffolding protein but also possesses proteolytic activity mediated by its caspase-like domain.  It remained unclear how the CBM activates MALT1.  Here, we provide biochem. and structural evidence that MALT1 activation is dependent on its dimerization and show that mutations at the dimer interface abrogate activity in cells.  The unliganded protease presents itself in a dimeric yet inactive state and undergoes substantial conformational changes upon substrate binding.  These structural changes also affect the conformation of the C-terminal Ig-like domain, a domain that is required for MALT1 activity.  Binding to the active site is coupled to a relative movement of caspase and Ig-like domains.  MALT1 binding partners thus may have the potential of tuning MALT1 protease activity without binding directly to the caspase domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDaYh5EEjXC7Vg90H21EOLACvtfcHk0ljNQI8QFy_S8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xkt1Ghur0%253D&md5=a35adbc547fadbe5353ee2d1c2dad767</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2012.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2012.02.018%26sid%3Dliteratum%253Aachs%26aulast%3DWiesmann%26aufirst%3DC.%26aulast%3DLeder%26aufirst%3DL.%26aulast%3DBlank%26aufirst%3DJ.%26aulast%3DBernardi%26aufirst%3DA.%26aulast%3DMelkko%26aufirst%3DS.%26aulast%3DDecock%26aufirst%3DA.%26aulast%3DD%25E2%2580%2599Arcy%26aufirst%3DA.%26aulast%3DVillard%26aufirst%3DF.%26aulast%3DErbel%26aufirst%3DP.%26aulast%3DHughes%26aufirst%3DN.%26aulast%3DFreuler%26aufirst%3DF.%26aulast%3DNikolay%26aufirst%3DR.%26aulast%3DAlves%26aufirst%3DJ.%26aulast%3DBornancin%26aufirst%3DF.%26aulast%3DRenatus%26aufirst%3DM.%26atitle%3DStructural%2520Determinants%2520of%2520MALT1%2520Protease%2520Activity%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D419%26spage%3D4%26epage%3D21%26doi%3D10.1016%2Fj.jmb.2012.02.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tohda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogoshi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, N.</span></span> <span> </span><span class="NLM_article-title">Establishment of a Novel B-Cell Lymphoma Cell Line with Suppressed Growth by Gamma-Secretase Inhibitors</span>. <i>Leuk. Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1385</span>– <span class="NLM_lpage">1390</span>, <span class="refDoi"> DOI: 10.1016/j.leukres.2006.05.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1016%2Fj.leukres.2006.05.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=16780947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BD28Xptlykurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2006&pages=1385-1390&author=S.+Tohdaauthor=T.+Satoauthor=H.+Kogoshiauthor=L.+Fuauthor=S.+Sakanoauthor=N.+Nara&title=Establishment+of+a+Novel+B-Cell+Lymphoma+Cell+Line+with+Suppressed+Growth+by+Gamma-Secretase+Inhibitors&doi=10.1016%2Fj.leukres.2006.05.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Establishment of a novel B-cell lymphoma cell line with suppressed growth by gamma-secretase inhibitors</span></div><div class="casAuthors">Tohda, Shuji; Sato, Taku; Kogoshi, Hanae; Fu, Lu; Sakano, Seiji; Nara, Nobuo</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1385-1390</span>CODEN:
                <span class="NLM_cas:coden">LEREDD</span>;
        ISSN:<span class="NLM_cas:issn">0145-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A novel lymphoma cell line, designated TMD8 was established from cells of a patient with diffuse large B-cell lymphoma.  TMD8 cells expressed HES1 mRNA, suggesting constitutive activation of Notch signaling.  TMD8 cells expressed normal-sized Notch1 protein, and showed no mutations in the NOTCH1 gene.  Cell growth was suppressed by gamma-secretase inhibitors (GSI).  It was reported that GSI suppressed growth of T-cell acute lymphoblastic leukemia (T-ALL) cell lines, which frequently had NOTCH1 mutations.  In addn. to T-ALL, TMD8 is another unique cell line sensitive to GSI, and is useful to study effects of GSI in mol. targeting therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3d8inz_KIn7Vg90H21EOLACvtfcHk0lhojAe9md3a2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xptlykurg%253D&md5=9db545b9d7b4f8c40b23a79d771261f5</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.leukres.2006.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.leukres.2006.05.003%26sid%3Dliteratum%253Aachs%26aulast%3DTohda%26aufirst%3DS.%26aulast%3DSato%26aufirst%3DT.%26aulast%3DKogoshi%26aufirst%3DH.%26aulast%3DFu%26aufirst%3DL.%26aulast%3DSakano%26aufirst%3DS.%26aulast%3DNara%26aufirst%3DN.%26atitle%3DEstablishment%2520of%2520a%2520Novel%2520B-Cell%2520Lymphoma%2520Cell%2520Line%2520with%2520Suppressed%2520Growth%2520by%2520Gamma-Secretase%2520Inhibitors%26jtitle%3DLeuk.%2520Res.%26date%3D2006%26volume%3D30%26spage%3D1385%26epage%3D1390%26doi%3D10.1016%2Fj.leukres.2006.05.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Naylor, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratsch, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enns, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuler, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorken, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.-m.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmeier, F.</span></span> <span> </span><span class="NLM_article-title">Protein Kinase C Inhibitor Sotrastaurin Selectively Inhibits the Growth of CD79 Mutant Diffuse Large B-Cell Lymphomas</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">2643</span>– <span class="NLM_lpage">2653</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-10-2525</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1158%2F0008-5472.CAN-10-2525" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=21324920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVOjtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=2643-2653&author=T.+L.+Naylorauthor=H.+Tangauthor=B.+A.+Ratschauthor=A.+Ennsauthor=A.+Looauthor=L.+Chenauthor=P.+Lenzauthor=N.+J.+Watersauthor=W.+Schulerauthor=B.+Dorkenauthor=Y.-m.+Yaoauthor=M.+Warmuthauthor=G.+Lenzauthor=F.+Stegmeier&title=Protein+Kinase+C+Inhibitor+Sotrastaurin+Selectively+Inhibits+the+Growth+of+CD79+Mutant+Diffuse+Large+B-Cell+Lymphomas&doi=10.1158%2F0008-5472.CAN-10-2525"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Kinase C Inhibitor Sotrastaurin Selectively Inhibits the Growth of CD79 Mutant Diffuse Large B-Cell Lymphomas</span></div><div class="casAuthors">Naylor, Tara L.; Tang, Huaping; Ratsch, Boris A.; Enns, Andreas; Loo, Alice; Chen, Liqing; Lenz, Peter; Waters, Nigel J.; Schuler, Walter; Doerken, Bernd; Yao, Yung-mae; Warmuth, Markus; Lenz, Georg; Stegmeier, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2643-2653</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The activated B-cell-like (ABC) subtype of diffuse large B-cell lymphoma (DLBCL) correlates with poor prognosis.  The ABC subtype of DLBCL is assocd. with constitutive activation of the NF-κB pathway, and oncogenic lesions have been identified in its regulators, including CARD11/CARMA1 (caspase recruitment domain-contg. protein 11), A20/TNF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoE3V3gS52btrVg90H21EOLACvtfcHk0lhojAe9md3a2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVOjtbs%253D&md5=68f1e5241923456583cc1190da0ed590</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-10-2525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-10-2525%26sid%3Dliteratum%253Aachs%26aulast%3DNaylor%26aufirst%3DT.%2BL.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DRatsch%26aufirst%3DB.%2BA.%26aulast%3DEnns%26aufirst%3DA.%26aulast%3DLoo%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DLenz%26aufirst%3DP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DSchuler%26aufirst%3DW.%26aulast%3DDorken%26aufirst%3DB.%26aulast%3DYao%26aufirst%3DY.-m.%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DLenz%26aufirst%3DG.%26aulast%3DStegmeier%26aufirst%3DF.%26atitle%3DProtein%2520Kinase%2520C%2520Inhibitor%2520Sotrastaurin%2520Selectively%2520Inhibits%2520the%2520Growth%2520of%2520CD79%2520Mutant%2520Diffuse%2520Large%2520B-Cell%2520Lymphomas%26jtitle%3DCancer%2520Res.%26date%3D2011%26volume%3D71%26spage%3D2643%26epage%3D2653%26doi%3D10.1158%2F0008-5472.CAN-10-2525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fung, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blank, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dufour, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolger-Munro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scurll, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Priatel, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweigler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melkko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gold, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viner, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Régnier, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turvey, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overall, C. M.</span></span> <span> </span><span class="NLM_article-title">The paracaspase MALT1 cleaves HOIL1 reducing linear ubiquitination by LUBAC to dampen lymphocyte NF-κB signalling</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">8777</span>, <span class="refDoi"> DOI: 10.1038/ncomms9777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1038%2Fncomms9777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=26525107" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslOns7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=8777&author=T.+Kleinauthor=S.-Y.+Fungauthor=F.+Rennerauthor=M.+A.+Blankauthor=A.+Dufourauthor=S.+Kangauthor=M.+Bolger-Munroauthor=J.+M.+Scurllauthor=J.+J.+Priatelauthor=P.+Schweiglerauthor=S.+Melkkoauthor=M.+R.+Goldauthor=R.+I.+Vinerauthor=C.+H.+R%C3%A9gnierauthor=S.+E.+Turveyauthor=C.+M.+Overall&title=The+paracaspase+MALT1+cleaves+HOIL1+reducing+linear+ubiquitination+by+LUBAC+to+dampen+lymphocyte+NF-%CE%BAB+signalling&doi=10.1038%2Fncomms9777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The paracaspase MALT1 cleaves HOIL1 reducing linear ubiquitination by LUBAC to dampen lymphocyte NF-κB signalling</span></div><div class="casAuthors">Klein, Theo; Fung, Shan-Yu; Renner, Florian; Blank, Michael A.; Dufour, Antoine; Kang, Sohyeong; Bolger-Munro, Madison; Scurll, Joshua M.; Priatel, John J.; Schweigler, Patrick; Melkko, Samu; Gold, Michael R.; Viner, Rosa I.; Regnier, Catherine H.; Turvey, Stuart E.; Overall, Christopher M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8777</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Antigen receptor signalling activates the canonical NF-κB pathway via the CARD11/BCL10/MALT1 (CBM) signalosome involving key, yet ill-defined roles for linear ubiquitination.  The paracaspase MALT1 cleaves and removes neg. checkpoint proteins, amplifying lymphocyte responses in NF-κB activation and in B-cell lymphoma subtypes.  To identify new human MALT1 substrates, we compare B cells from the only known living MALT1mut/mut patient with healthy MALT1+/mut family members using 10-plex Tandem Mass Tag TAILS N-terminal peptide proteomics.  We identify HOIL1 of the linear ubiquitin chain assembly complex as a novel MALT1 substrate.  We show linear ubiquitination at B-cell receptor microclusters and signalosomes.  Late in the NF-κB activation cycle HOIL1 cleavage transiently reduces linear ubiquitination, including of NEMO and RIP1, dampening NF-κB activation and preventing reactivation.  By regulating linear ubiquitination, MALT1 is both a pos. and neg. pleiotropic regulator of the human canonical NF-κB pathway-first promoting activation via the CBM-then triggering HOIL1-dependent neg.-feedback termination, preventing reactivation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3c7mF51tenLVg90H21EOLACvtfcHk0lhojAe9md3a2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslOns7fP&md5=dd36a72c870b574f4e8cfc3dff092995</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fncomms9777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms9777%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DT.%26aulast%3DFung%26aufirst%3DS.-Y.%26aulast%3DRenner%26aufirst%3DF.%26aulast%3DBlank%26aufirst%3DM.%2BA.%26aulast%3DDufour%26aufirst%3DA.%26aulast%3DKang%26aufirst%3DS.%26aulast%3DBolger-Munro%26aufirst%3DM.%26aulast%3DScurll%26aufirst%3DJ.%2BM.%26aulast%3DPriatel%26aufirst%3DJ.%2BJ.%26aulast%3DSchweigler%26aufirst%3DP.%26aulast%3DMelkko%26aufirst%3DS.%26aulast%3DGold%26aufirst%3DM.%2BR.%26aulast%3DViner%26aufirst%3DR.%2BI.%26aulast%3DR%25C3%25A9gnier%26aufirst%3DC.%2BH.%26aulast%3DTurvey%26aufirst%3DS.%2BE.%26aulast%3DOverall%26aufirst%3DC.%2BM.%26atitle%3DThe%2520paracaspase%2520MALT1%2520cleaves%2520HOIL1%2520reducing%2520linear%2520ubiquitination%2520by%2520LUBAC%2520to%2520dampen%2520lymphocyte%2520NF-%25CE%25BAB%2520signalling%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D8777%26doi%3D10.1038%2Fncomms9777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bornancin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Touil, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sic, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Touil-Allaoui, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rush, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigaud, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junker-Walker, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkhart, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niwa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katopodis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuesslein-Hildesheim, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weckbecker, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zenke, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinzel, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traggiai, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brüstle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">St. Paul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamurovic, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolink, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Régnier, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mak, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohashi, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calzascia, T.</span></span> <span> </span><span class="NLM_article-title">Deficiency of MALT1 Paracaspase Activity Results in Unbalanced Regulatory and Effector T and B Cell Responses Leading to Multiorgan Inflammation</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>194</i></span>,  <span class="NLM_fpage">3723</span>– <span class="NLM_lpage">3734</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.1402254</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.4049%2Fjimmunol.1402254" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=25762782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVeisrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=194&publication_year=2015&pages=3723-3734&author=F.+Bornancinauthor=F.+Rennerauthor=R.+Touilauthor=H.+Sicauthor=Y.+Kolbauthor=I.+Touil-Allaouiauthor=J.+S.+Rushauthor=P.+A.+Smithauthor=M.+Bigaudauthor=U.+Junker-Walkerauthor=C.+Burkhartauthor=J.+Dawsonauthor=S.+Niwaauthor=A.+Katopodisauthor=B.+Nuesslein-Hildesheimauthor=G.+Weckbeckerauthor=G.+Zenkeauthor=B.+Kinzelauthor=E.+Traggiaiauthor=D.+Brennerauthor=A.+Br%C3%BCstleauthor=M.+St.+Paulauthor=N.+Zamurovicauthor=K.+D.+McCoyauthor=A.+Rolinkauthor=C.+H.+R%C3%A9gnierauthor=T.+W.+Makauthor=P.+S.+Ohashiauthor=D.+D.+Patelauthor=T.+Calzascia&title=Deficiency+of+MALT1+Paracaspase+Activity+Results+in+Unbalanced+Regulatory+and+Effector+T+and+B+Cell+Responses+Leading+to+Multiorgan+Inflammation&doi=10.4049%2Fjimmunol.1402254"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Deficiency of MALT1 Paracaspase Activity Results in Unbalanced Regulatory and Effector T and B Cell Responses Leading to Multiorgan Inflammation</span></div><div class="casAuthors">Bornancin, Frederic; Renner, Florian; Touil, Ratiba; Sic, Heiko; Kolb, Yeter; Touil-Allaoui, Ismahane; Rush, James S.; Smith, Paul A.; Bigaud, Marc; Junker-Walker, Ursula; Burkhart, Christoph; Dawson, Janet; Niwa, Satoru; Katopodis, Andreas; Nuesslein-Hildesheim, Barbara; Weckbecker, Gisbert; Zenke, Gerhard; Kinzel, Bernd; Traggiai, Elisabetta; Brenner, Dirk; Bruestle, Anne; St. Paul, Michael; Zamurovic, Natasa; McCoy, Kathy D.; Rolink, Antonius; Regnier, Catherine H.; Mak, Tak W.; Ohashi, Pamela S.; Patel, Dhavalkumar D.; Calzascia, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">194</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3723-3734</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">The paracaspase MALT1 plays an important role in immune receptor-driven signaling pathways leading to NF-κB activation.  MALT1 promotes signaling by acting as a scaffold, recruiting downstream signaling proteins, as well as by proteolytic cleavage of multiple substrates.  However, the relative contributions of these two different activities to T and B cell function are not well understood.  To investigate how MALT1 proteolytic activity contributes to overall immune cell regulation, we generated MALT1 protease-deficient mice (Malt1PD/PD) and compared their phenotype with that of MALT1 knockout animals (Malt1-/-).  Malt1PD/PD mice displayed defects in multiple cell types including marginal zone B cells, B1 B cells, IL-10-producing B cells, regulatory T cells, and mature T and B cells.  In general, immune defects were more pronounced in Malt1-/- animals.  Both mouse lines showed abrogated B cell responses upon immunization with T-dependent and T-independent Ags.  In vitro, inactivation of MALT1 protease activity caused reduced stimulation-induced T cell proliferation, impaired IL-2 and TNF-α prodn., as well as defective Th17 differentiation.  Consequently, Malt1PD/PD mice were protected in a Th17-dependent exptl. autoimmune encephalomyelitis model.  Surprisingly, Malt1PD/PD animals developed a multiorgan inflammatory pathol., characterized by Th1 and Th2/0 responses and enhanced IgG1 and IgE levels, which was delayed by wild-type regulatory T cell reconstitution.  We therefore propose that the pathol. characterizing Malt1PD/PD animals arises from an immune imbalance featuring pathogenic Th1- and Th2/0-skewed effector responses and reduced immunosuppressive compartments.  These data uncover a previously unappreciated key function of MALT1 protease activity in immune homeostasis and underline its relevance in human health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8H9YU0RgmnbVg90H21EOLACvtfcHk0liZyjrCoRS4Iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVeisrc%253D&md5=8e16455eaab8d70f0fe8cf246aeb1edc</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1402254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1402254%26sid%3Dliteratum%253Aachs%26aulast%3DBornancin%26aufirst%3DF.%26aulast%3DRenner%26aufirst%3DF.%26aulast%3DTouil%26aufirst%3DR.%26aulast%3DSic%26aufirst%3DH.%26aulast%3DKolb%26aufirst%3DY.%26aulast%3DTouil-Allaoui%26aufirst%3DI.%26aulast%3DRush%26aufirst%3DJ.%2BS.%26aulast%3DSmith%26aufirst%3DP.%2BA.%26aulast%3DBigaud%26aufirst%3DM.%26aulast%3DJunker-Walker%26aufirst%3DU.%26aulast%3DBurkhart%26aufirst%3DC.%26aulast%3DDawson%26aufirst%3DJ.%26aulast%3DNiwa%26aufirst%3DS.%26aulast%3DKatopodis%26aufirst%3DA.%26aulast%3DNuesslein-Hildesheim%26aufirst%3DB.%26aulast%3DWeckbecker%26aufirst%3DG.%26aulast%3DZenke%26aufirst%3DG.%26aulast%3DKinzel%26aufirst%3DB.%26aulast%3DTraggiai%26aufirst%3DE.%26aulast%3DBrenner%26aufirst%3DD.%26aulast%3DBr%25C3%25BCstle%26aufirst%3DA.%26aulast%3DSt.%2BPaul%26aufirst%3DM.%26aulast%3DZamurovic%26aufirst%3DN.%26aulast%3DMcCoy%26aufirst%3DK.%2BD.%26aulast%3DRolink%26aufirst%3DA.%26aulast%3DR%25C3%25A9gnier%26aufirst%3DC.%2BH.%26aulast%3DMak%26aufirst%3DT.%2BW.%26aulast%3DOhashi%26aufirst%3DP.%2BS.%26aulast%3DPatel%26aufirst%3DD.%2BD.%26aulast%3DCalzascia%26aufirst%3DT.%26atitle%3DDeficiency%2520of%2520MALT1%2520Paracaspase%2520Activity%2520Results%2520in%2520Unbalanced%2520Regulatory%2520and%2520Effector%2520T%2520and%2520B%2520Cell%2520Responses%2520Leading%2520to%2520Multiorgan%2520Inflammation%26jtitle%3DJ.%2520Immunol.%26date%3D2015%26volume%3D194%26spage%3D3723%26epage%3D3734%26doi%3D10.4049%2Fjimmunol.1402254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erbel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bornancin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiesmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiering, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villard, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Decock, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melkko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spanka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buschmann, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pissot-Soldermann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simic, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beerli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tintelnot-Blomley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beltz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Régnier, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quancard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlapbach, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langlois, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renatus, M.</span></span> <span> </span><span class="NLM_article-title">Stabilizing Inactive Conformations of MALT1 as an Effective Approach to Inhibit Its Protease Activity</span>. <i>Adv. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">2000078</span>, <span class="refDoi"> DOI: 10.1002/adtp.202000078</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=10.1002%2Fadtp.202000078" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslKhs7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2020&pages=2000078&author=N.+Hughesauthor=P.+Erbelauthor=F.+Bornancinauthor=C.+Wiesmannauthor=N.+Schieringauthor=F.+Villardauthor=A.+Decockauthor=B.+Rubiauthor=S.+Melkkoauthor=C.+Spankaauthor=N.+Buschmannauthor=C.+Pissot-Soldermannauthor=O.+Simicauthor=R.+Beerliauthor=M.+Sorgeauthor=M.+Tintelnot-Blomleyauthor=K.+Beltzauthor=C.+H.+R%C3%A9gnierauthor=J.+Quancardauthor=A.+Schlapbachauthor=J.+B.+Langloisauthor=M.+Renatus&title=Stabilizing+Inactive+Conformations+of+MALT1+as+an+Effective+Approach+to+Inhibit+Its+Protease+Activity&doi=10.1002%2Fadtp.202000078"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Stabilizing Inactive Conformations of MALT1 as an Effective Approach to Inhibit Its Protease Activity</span></div><div class="casAuthors">Hughes, Nicola; Erbel, Paul; Bornancin, Frederic; Wiesmann, Christian; Schiering, Nikolaus; Villard, Frederic; Decock, Arnaud; Rubi, Bertran; Melkko, Samu; Spanka, Carsten; Buschmann, Nicole; Pissot-Soldermann, Carole; Simic, Oliver; Beerli, Rene; Sorge, Mickael; Tintelnot-Blomley, Marina; Beltz, Karen; Regnier, Catherine H.; Quancard, Jean; Schlapbach, Achim; Langlois, Jean-Baptiste; Renatus, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Therapeutics (Weinheim, Germany)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2000078</span>CODEN:
                <span class="NLM_cas:coden">ATWGAP</span>;
        ISSN:<span class="NLM_cas:issn">2366-3987</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The paracaspase MALT1 (mucosa assocd. lymphoid tissue lymphoma translocated gene 1) plays an important role in various immune pathways and is proposed as a therapeutic target for autoimmune disorders as well as different types of cancer, such as diffuse large B-cell lymphoma (DLBCL).  Different mechanisms are explored to inhibit the protease activity of MALT1 and two unrelated chem. scaffolds are discovered.  Biophys. and structural studies reveal that both scaffolds stabilize the protease in an inactive conformation.  While one ligand binds to the allosteric site at the interface between the caspase and the Ig3 domain, the other ligand binds to the active site using a so far undescribed mechanism.  Iterative structure-based drug discovery on one scaffold results in the identification of a potent, selective, and orally bioavailable MALT1 inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7qcznIt2xNLVg90H21EOLACvtfcHk0liZyjrCoRS4Iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslKhs7jO&md5=db679edcbd1dba3d3e991d55e07d5aa8</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fadtp.202000078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fadtp.202000078%26sid%3Dliteratum%253Aachs%26aulast%3DHughes%26aufirst%3DN.%26aulast%3DErbel%26aufirst%3DP.%26aulast%3DBornancin%26aufirst%3DF.%26aulast%3DWiesmann%26aufirst%3DC.%26aulast%3DSchiering%26aufirst%3DN.%26aulast%3DVillard%26aufirst%3DF.%26aulast%3DDecock%26aufirst%3DA.%26aulast%3DRubi%26aufirst%3DB.%26aulast%3DMelkko%26aufirst%3DS.%26aulast%3DSpanka%26aufirst%3DC.%26aulast%3DBuschmann%26aufirst%3DN.%26aulast%3DPissot-Soldermann%26aufirst%3DC.%26aulast%3DSimic%26aufirst%3DO.%26aulast%3DBeerli%26aufirst%3DR.%26aulast%3DSorge%26aufirst%3DM.%26aulast%3DTintelnot-Blomley%26aufirst%3DM.%26aulast%3DBeltz%26aufirst%3DK.%26aulast%3DR%25C3%25A9gnier%26aufirst%3DC.%2BH.%26aulast%3DQuancard%26aufirst%3DJ.%26aulast%3DSchlapbach%26aufirst%3DA.%26aulast%3DLanglois%26aufirst%3DJ.%2BB.%26aulast%3DRenatus%26aufirst%3DM.%26atitle%3DStabilizing%2520Inactive%2520Conformations%2520of%2520MALT1%2520as%2520an%2520Effective%2520Approach%2520to%2520Inhibit%2520Its%2520Protease%2520Activity%26jtitle%3DAdv.%2520Ther.%26date%3D2020%26volume%3D3%26spage%3D2000078%26doi%3D10.1002%2Fadtp.202000078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AK0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AK0','PDB','7AK0'); return false;">PDB: 7AK0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AK1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AK1','PDB','7AK1'); return false;">PDB: 7AK1</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i84"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01245">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_81686"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01245?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01245</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Data collection and refinement statistics and mechanistic cIAP2-MALT1-dependent NF-κB reporter gene assay (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01245/suppl_file/jm0c01245_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings, biochemical, cellular potency data, and solubility (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01245/suppl_file/jm0c01245_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01245/suppl_file/jm0c01245_si_001.pdf">jm0c01245_si_001.pdf (844.43 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01245/suppl_file/jm0c01245_si_002.csv">jm0c01245_si_002.csv (1.98 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Protein Data Bank codes are the following: <b>TC1</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AK0">7AK0</a>; <b>TC2</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AK1">7AK1</a></p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01245&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-23%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01245%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01245" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799b50298c02332","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
