
<html lang="en"     class="pb-page"  data-request-id="aff42302-1de4-4063-95b7-29a6594bd767"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c02000;issue:issue:10.1021/jmcmar.2021.64.issue-5;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Highly Selective Y4 Receptor Antagonist Binds in an Allosteric Binding Pocket" /></meta><meta name="dc.Creator" content="Corinna  Schüß" /></meta><meta name="dc.Creator" content="Oanh  Vu" /></meta><meta name="dc.Creator" content="Mario  Schubert" /></meta><meta name="dc.Creator" content="Yu  Du" /></meta><meta name="dc.Creator" content="Nigam M.  Mishra" /></meta><meta name="dc.Creator" content="Iain R.  Tough" /></meta><meta name="dc.Creator" content="Jan  Stichel" /></meta><meta name="dc.Creator" content="C. David  Weaver" /></meta><meta name="dc.Creator" content="Kyle A.  Emmitte" /></meta><meta name="dc.Creator" content="Helen M.  Cox" /></meta><meta name="dc.Creator" content="Jens  Meiler" /></meta><meta name="dc.Creator" content="Annette G.  Beck-Sickinger" /></meta><meta name="dc.Description" content="Human neuropeptide Y receptors (Y1R, Y2R, Y4R, and Y5R) belong to the superfamily of G protein-coupled receptors and play an important role in the regulation of food intake and energy metabolism. W..." /></meta><meta name="Description" content="Human neuropeptide Y receptors (Y1R, Y2R, Y4R, and Y5R) belong to the superfamily of G protein-coupled receptors and play an important role in the regulation of food intake and energy metabolism. W..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 17, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c02000" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02000" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c02000" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02000" /></link>
        
    
    

<title>Highly Selective Y4 Receptor Antagonist Binds in an Allosteric Binding Pocket | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c02000" /></meta><meta property="og:title" content="Highly Selective Y4 Receptor Antagonist Binds in an Allosteric Binding Pocket" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0009.jpeg" /></meta><meta property="og:description" content="Human neuropeptide Y receptors (Y1R, Y2R, Y4R, and Y5R) belong to the superfamily of G protein-coupled receptors and play an important role in the regulation of food intake and energy metabolism. We identified and characterized the first selective Y4R allosteric antagonist (S)-VU0637120, an important step toward validating Y receptors as therapeutic targets for metabolic diseases. To obtain insight into the antagonistic mechanism of (S)-VU0637120, we conducted a variety of in vitro, ex vivo, and in silico studies. These studies revealed that (S)-VU0637120 selectively inhibits native Y4R function and binds in an allosteric site located below the binding pocket of the endogenous ligand pancreatic polypeptide in the core of the Y4R transmembrane domains. Taken together, our studies provide a first-of-its-kind tool for probing Y4R function and improve the general understanding of allosteric modulation, ultimately contributing to the rational development of allosteric modulators for peptide-activated G protein-coupled receptors (GPCRs)." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c02000"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02000">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c02000&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c02000&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c02000&amp;href=/doi/10.1021/acs.jmedchem.0c02000" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 5</span><span class="cit-fg-pageRange">, 2801-2814</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/5" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c02070" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c02063" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Highly Selective Y<sub>4</sub> Receptor Antagonist Binds in an Allosteric Binding Pocket</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Corinna Schüß</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Corinna Schüß</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Biochemistry, Leipzig University, Leipzig 04103, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Corinna++Sch%C3%BC%C3%9F">Corinna Schüß</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Oanh Vu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Oanh Vu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Vanderbilt University, Nashville, Tennessee 37235, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Oanh++Vu">Oanh Vu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4704-7538" title="Orcid link">http://orcid.org/0000-0003-4704-7538</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mario Schubert</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mario Schubert</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Biochemistry, Leipzig University, Leipzig 04103, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mario++Schubert">Mario Schubert</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yu Du</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yu Du</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology, Vanderbilt University, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yu++Du">Yu Du</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nigam M. Mishra</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nigam M. Mishra</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, UNT System College of Pharmacy, University of North Texas Health Science Center, Fort Worth, Texas 76107, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nigam+M.++Mishra">Nigam M. Mishra</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Iain R. Tough</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Iain R. Tough</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">King’s College London, Wolfson Centre for Age-Related Diseases, Institute of Psychiatry, Psychology & Neuroscience, London SE1 1UL, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Iain+R.++Tough">Iain R. Tough</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jan Stichel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jan Stichel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Biochemistry, Leipzig University, Leipzig 04103, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jan++Stichel">Jan Stichel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">C. David Weaver</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">C. David Weaver</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Vanderbilt University, Nashville, Tennessee 37235, United States</div><div class="loa-info-affiliations-info">Department of Pharmacology, Vanderbilt University, Nashville, Tennessee 37232, United States</div><div class="loa-info-affiliations-info">Institute of Chemical Biology, Vanderbilt University, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=C.+David++Weaver">C. David Weaver</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6886-1195" title="Orcid link">http://orcid.org/0000-0002-6886-1195</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kyle A. Emmitte</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kyle A. Emmitte</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, UNT System College of Pharmacy, University of North Texas Health Science Center, Fort Worth, Texas 76107, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kyle+A.++Emmitte">Kyle A. Emmitte</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6643-3947" title="Orcid link">http://orcid.org/0000-0002-6643-3947</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Helen M. Cox</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Helen M. Cox</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">King’s College London, Wolfson Centre for Age-Related Diseases, Institute of Psychiatry, Psychology & Neuroscience, London SE1 1UL, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Helen+M.++Cox">Helen M. Cox</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jens Meiler</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jens Meiler</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Vanderbilt University, Nashville, Tennessee 37235, United States</div><div class="loa-info-affiliations-info">Department of Pharmacology, Vanderbilt University, Nashville, Tennessee 37232, United States</div><div class="loa-info-affiliations-info">Institute for Drug Discovery, Leipzig University, Leipzig 04103, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#650f000b162508000c0900170904074b0a1702"><span class="__cf_email__" data-cfemail="49232c273a09242c20252c3b25282b67263b2e">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jens++Meiler">Jens Meiler</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Annette G. Beck-Sickinger</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Annette G. Beck-Sickinger</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Biochemistry, Leipzig University, Leipzig 04103, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#aacbc8cfc9c187d9c3c9c1c3c4cdcfd8eadfc4c387c6cfc3dad0c3cd84cecf"><span class="__cf_email__" data-cfemail="4e2f2c2b2d25633d272d252720292b3c0e3b202763222b273e342729602a2b">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Annette+G.++Beck-Sickinger">Annette G. Beck-Sickinger</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4560-8020" title="Orcid link">http://orcid.org/0000-0003-4560-8020</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02000&amp;href=/doi/10.1021%2Facs.jmedchem.0c02000" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 5</span><span class="cit-pageRange">, 2801–2814</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 17, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>30 November 2020</li><li><span class="item_label"><b>Published</b> online</span>17 February 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 March 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c02000" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02000</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2801%26pageCount%3D14%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DCorinna%2BSch%25C3%25BC%25C3%259F%252C%2BOanh%2BVu%252C%2BMario%2BSchubert%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D5%26contentID%3Dacs.jmedchem.0c02000%26title%3DHighly%2BSelective%2BY4%2BReceptor%2BAntagonist%2BBinds%2Bin%2Ban%2BAllosteric%2BBinding%2BPocket%26numPages%3D14%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2814%26publicationDate%3DMarch%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c02000"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">709</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c02000" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Highly Selective Y4 Receptor Antagonist Binds in an Allosteric Binding Pocket&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Corinna&quot;,&quot;last_name&quot;:&quot;Schüß&quot;},{&quot;first_name&quot;:&quot;Oanh&quot;,&quot;last_name&quot;:&quot;Vu&quot;},{&quot;first_name&quot;:&quot;Mario&quot;,&quot;last_name&quot;:&quot;Schubert&quot;},{&quot;first_name&quot;:&quot;Yu&quot;,&quot;last_name&quot;:&quot;Du&quot;},{&quot;first_name&quot;:&quot;Nigam&quot;,&quot;last_name&quot;:&quot;M. Mishra&quot;},{&quot;first_name&quot;:&quot;Iain&quot;,&quot;last_name&quot;:&quot;R. Tough&quot;},{&quot;first_name&quot;:&quot;Jan&quot;,&quot;last_name&quot;:&quot;Stichel&quot;},{&quot;first_name&quot;:&quot;C.&quot;,&quot;last_name&quot;:&quot;David Weaver&quot;},{&quot;first_name&quot;:&quot;Kyle&quot;,&quot;last_name&quot;:&quot;A. Emmitte&quot;},{&quot;first_name&quot;:&quot;Helen&quot;,&quot;last_name&quot;:&quot;M. Cox&quot;},{&quot;first_name&quot;:&quot;Jens&quot;,&quot;last_name&quot;:&quot;Meiler&quot;},{&quot;first_name&quot;:&quot;Annette&quot;,&quot;last_name&quot;:&quot;G. Beck-Sickinger&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;17&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;2801-2814&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c02000&quot;},&quot;abstract&quot;:&quot;Human neuropeptide Y receptors (Y1R, Y2R, Y4R, and Y5R) belong to the superfamily of G protein-coupled receptors and play an important role in the regulation of food intake and energy metabolism. We identified and characterized the first selective Y4R allosteric antagonist (S)-VU0637120, an important step toward validating Y receptors as therapeutic targets for metabolic diseases. To obtain insight into the antagonistic mechanism of (S)-VU0637120, we conducted a variety of in vitro, ex vivo, and in silico studies. These studies revealed that (S)-VU0637120 selectively inhibits native Y4R function and binds in an allosteric site located below the binding pocket of the endogenous ligand pancreatic polypeptide in the core of the Y4R transmembrane domains. Taken together, our studies provide a first-of-its-kind tool for probing Y4R function and improve the general understanding of allosteric modulation, ultimately contributing to the rational development of allosteric modulators for peptide-activated G protein-cou&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02000&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02000" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02000&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02000" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02000&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02000" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02000&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02000&amp;href=/doi/10.1021/acs.jmedchem.0c02000" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c02000" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c02000" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c02000%26sid%3Dliteratum%253Aachs%26pmid%3D33595306%26genre%3Darticle%26aulast%3DSch%25C3%25BC%25C3%259F%26date%3D2021%26atitle%3DHighly%2BSelective%2BY4%2BReceptor%2BAntagonist%2BBinds%2Bin%2Ban%2BAllosteric%2BBinding%2BPocket%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D5%26spage%3D2801%26epage%3D2814%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (10)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291171" title="Antagonists">Antagonists</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/5" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/jmcmar.2021.64.issue-5/20210311/jmcmar.2021.64.issue-5.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/medium/jm0c02000_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02000&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Human neuropeptide Y receptors (Y<sub>1</sub>R, Y<sub>2</sub>R, Y<sub>4</sub>R, and Y<sub>5</sub>R) belong to the superfamily of G protein-coupled receptors and play an important role in the regulation of food intake and energy metabolism. We identified and characterized the first selective Y<sub>4</sub>R allosteric antagonist (<i>S</i>)<i>-</i>VU0637120, an important step toward validating Y receptors as therapeutic targets for metabolic diseases. To obtain insight into the antagonistic mechanism of (<i>S</i>)<i>-</i>VU0637120, we conducted a variety of in vitro, ex vivo, and in silico studies. These studies revealed that (<i>S</i>)<i>-</i>VU0637120 selectively inhibits native Y<sub>4</sub>R function and binds in an allosteric site located below the binding pocket of the endogenous ligand pancreatic polypeptide in the core of the Y<sub>4</sub>R transmembrane domains. Taken together, our studies provide a first-of-its-kind tool for probing Y<sub>4</sub>R function and improve the general understanding of allosteric modulation, ultimately contributing to the rational development of allosteric modulators for peptide-activated G protein-coupled receptors (GPCRs).</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86217" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86217" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The neuropeptide Y system is a multiligand/multireceptor family consisting of four Y receptors (Y<sub>1</sub>R, Y<sub>2</sub>R, Y<sub>4</sub>R, and Y<sub>5</sub>R) and three peptide ligands: neuropeptide Y (NPY), peptide YY (PYY), and pancreatic polypeptide (PP). The NPY family is well known for its regulation of important physiological processes, including energy metabolism and food intake. Therefore, dysregulation of these processes can result in disorders, such as anorexia and obesity. Whereas the Y<sub>1</sub>R and Y<sub>5</sub>R induce orexigenic signals, the Y<sub>2</sub>R and Y<sub>4</sub>R display anorexigenic functions. As Y receptors have overlapping preferences for their peptidic agonists, the development of subtype-selective compounds is essential to enable a better understanding of the function of these receptors and to explore their therapeutic potential.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> The Y<sub>1</sub>R and Y<sub>2</sub>R bind NPY and PYY with high affinity. The Y<sub>4</sub>R has a preference for PP compared to NPY and PYY, and the Y<sub>5</sub>R is activated by all three ligands with comparable potency. Thus, it is very challenging to develop orthosteric ligands with high selectivity for one receptor subtype versus the other Y receptors. To date, a number of agonists and antagonists for Y receptors have been developed.<a onclick="showRef(event, 'ref1 ref2 ref4'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref4">(1,2,4)</a> With respect to Y<sub>4</sub>R, the native ligand PP represents a potent agonist that induces anorexigenic effects in humans.<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a> The major challenge in the development of novel Y<sub>4</sub>R ligands remains obtaining selectivity with respect to the evolutionarily most closely related Y<sub>1</sub>R subtype.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> Selective Y<sub>4</sub>R agonists have been identified by crosslinking two peptide fragments derived from the C-terminal pentapeptide of PP/NPY.<a onclick="showRef(event, 'ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11">(9−11)</a> However, no highly selective small-molecule antagonists for Y<sub>4</sub>R have been described to date.</div><div class="NLM_p">Aiming at the development of small-molecule probes to selectively target the human Y<sub>4</sub>R, approximately 77 000 compounds were tested by high-throughput screening (HTS). We recently reported on the discovery of Y<sub>4</sub>R positive allosteric modulators (PAM) from this experiment.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a> In comparison to PAMs and agonists, the hit rate of negative allosteric modulators (NAM) and antagonists was very low (0.022%, <a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Table S1</a>, Supporting Information), reflecting the challenges in the development of Y<sub>4</sub>R antagonists.<a onclick="showRef(event, 'ref11 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref11 ref14 ref15">(11,14,15)</a></div><div class="NLM_p">In this study, we present the in vitro, in silico, and ex vivo characterization of the first selective, small-molecule Y<sub>4</sub>R allosteric antagonist (<i>S</i>)<i>-</i>VU0637120 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a), including the identification of its binding mode that allows further preclinical studies.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/medium/jm0c02000_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Characterization of VU0637120. (a) Structure of VU0637120, chiral C-atom is labeled by an asterisk. (b) VU0637120 effectively decreases Y<sub>4</sub>R activation by PP in Ca<sup>2+</sup> flux assays using stably transfected COS7_hY<sub>4</sub>R-eYFP_Δ6Gα<sub>qi4myr</sub> cells. Data represent mean ± standard error of the mean (SEM) from <i>N</i> ≥ 2 independent experiments. VU0637120 effect on PP potency (α) and efficacy (β) was quantified using an operational model of allosterism.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> (c) Effect of increasing concentrations of VU0637120 and its enantiopure isomers on Y<sub>4</sub>R activation in response to a submaximal PP (PP EC<sub>80</sub>) concentration in COS7_hY<sub>4</sub>R-eYFP_Δ6Gα<sub>qi4myr</sub> cells. Data are shown as mean ± SEM from at least <i>N</i> ≥ 2 independent experiments. (d) Investigation of arrestin3 (arr3) recruitment to Y<sub>4</sub>R in response to 100 nM PP in the presence of 30 μM of VU0637120. Kinetic bioluminescence resonance energy transfer (BRET) experiments were performed in HEK293 cells transiently expressing Y<sub>4</sub>R-Rluc8 and venus-arr3. Rate constant <i>k</i> of arr3 recruitment is summarized as a bar graph. Data represent the mean ± SEM from <i>N</i> ≥ 4 independent experiments. Statistical analysis was performed using unpaired <i>t</i>-test, ***<i>P</i> <0.001. (e) VU0637120 reduces the specific binding of <sup>125</sup>I-PP to Y<sub>4</sub>R membrane preparations. Binding was investigated in competition binding assays with 60 or 180 pM <sup>125</sup>I-PP. Unspecific binding was determined in the presence of 1 μM PP. Data represent the mean ± SEM from <i>N</i> ≥ 4 independent experiments. (f) PP competition binding in the presence of VU0637120 using <sup>125</sup>I-PP (60 pM) at Y<sub>4</sub>R membrane preparations. Data are shown as mean ± SEM from <i>N</i> ≥ 3 independent experiments. (g) VU0637120 shows high selectivity to the Y<sub>4</sub>R subtype. Activation of Y<sub>1</sub>R, Y<sub>2</sub>R, and Y<sub>5</sub>R by NPY and Y<sub>4</sub>R by PP was investigated in the presence of 30 μM of VU0637120. Receptor signaling was measured in Ca<sup>2+</sup> flux assays using COS7 cells stably expressing one specific Y receptor (Y<sub>1,2,4,5</sub>R-eYFP) and the chimeric G protein Δ6Gα<sub>qi4myr</sub>. Data represent the mean ± SEM from <i>N</i> ≥ 3 independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02000&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41508" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41508" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Chemistry</h3><div class="NLM_p">Synthesis of (<i>S</i>)<i>-</i> and (<i>R</i>)-VU0637120 was carried out according to <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Heteroaryl amine <b>1</b><a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> was coupled to both (<i>S</i>)-1-(<i>tert</i>-butoxycarbonyl)piperidine-2-carboxylic acid (<b>2</b>) and (<i>R</i>)-1-(<i>tert</i>-butoxycarbonyl)piperidine-2-carboxylic acid (<b>3</b>) to afford amides <b>4</b> and <b>5</b>, respectively. Cleavage of the amine-protecting groups under acidic conditions provided the secondary amines <b>6</b> and <b>7</b> in high yield. Finally, the reaction of <b>6</b> and <b>7</b> with methanesulfonyl chloride and trimethylamine gave (<i>S</i>)<i>-</i> and (<i>R</i>)-VU0637120, respectively.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/medium/jm0c02000_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0008.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of (<i>S</i>)<i>-</i> and (<i>R</i>)-VU0637120<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02000&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>N</i>,<i>N</i>-diisopropylethylamine (DIEA), 1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate (HATU), dimethylformamide (DMF), 86% (<b>4</b>), 85% (<b>5</b>); (b) HCl, dioxane, CH<sub>2</sub>Cl<sub>2</sub>, 89% (<b>6</b>), 93% (<b>7</b>); (c) NEt<sub>3</sub>, MeSO<sub>2</sub>Cl, CH<sub>2</sub>Cl<sub>2</sub>, 60% ((<i>S</i>)<i>-</i>VU0637120), 59% ((<i>R</i>)-VU0637120).</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> (<i>S</i>)<i>-</i>VU0637120 Selectively Inhibits Y<sub>4</sub>R Activation by PP through an Allosteric Mechanism</h3><div class="NLM_p">The compound VU0637120 was identified as the most potent Y<sub>4</sub>R small-molecule antagonist in the HTS and is characterized in detail within this study. VU0637120 was first investigated for its effect to modulate the G protein signaling pathway at the Y<sub>4</sub>R by performing Ca<sup>2+</sup> flux assays. These studies confirmed that VU0637120 effectively decreases Y<sub>4</sub>R activation by PP in a concentration-dependent manner. This is indicated by the rightward shift of PP concentration–response curves with increasing VU0637120 concentrations (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b). The activity of VU0637120 at the Y<sub>4</sub>R was quantified using an operational model of allosterism.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> This model allows the calculation of individual parameters like the equilibrium binding constant (<i>K</i><sub>B</sub>) of VU0637120 and the effect of VU0637120 on the modulation of orthosteric agonist potency (α) and efficacy (β) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b). VU0637120 has a negative allosteric effect on the affinity of PP to its Y<sub>4</sub>R orthosteric binding pocket (α = 0.02) and slightly reduces signaling efficacy (β = 0.77). The affinity (<i>K</i><sub>B</sub>) of VU0637120 to its allosteric binding pocket is in the range of 300–400 nM (p<i>K</i><sub>B</sub> 6.4 ± 0.08). VU0637120 has one chiral center (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a). To determine whether both stereoisomers are active, we synthesized enantiopure (<i>S</i>)<i>-</i> and (<i>R</i>)-VU0637120 and tested them for their ability to reduce a submaximal PP response at the Y<sub>4</sub>R (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>c). Here, (<i>S</i>)<i>-</i>VU0637120 was confirmed as the active stereoisomer of the compound, as it is able to efficaciously decrease PP response at the Y<sub>4</sub>R with an IC<sub>50</sub> value of 2.7 μM (pIC<sub>50</sub> 5.6 ± 0.02). In contrast, (<i>R</i>)-VU0637120 is inactive at the Y<sub>4</sub>R up to the top concentration tested. To validate the effect of the active enantiomer (<i>S</i>)<i>-</i>VU0637120, we also tested (<i>S</i>)<i>-</i>VU0637120 on the G protein signaling pathway at the Y<sub>4</sub>R using different functional assays: Ca<sup>2+</sup> flux assay, a hemiequilibrium kinetic assay, and inositol phosphate (IP)-One assay as an equilibrium assay.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Testing of the pure (<i>S</i>)<i>-</i>VU0637120 confirmed the antagonistic activity in Ca<sup>2+</sup> flux and IP-One assays. Schild analysis of the data reveals that the antagonistic effect of (<i>S</i>)<i>-</i>VU0637120 is saturable at concentrations larger than 10 μM, indicating that the compound does not act as a pure orthosteric antagonist and support an allosteric mode of action (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Figure S1</a>, Supporting Information).<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></div><div class="NLM_p">As previous studies have already shown the importance of the arrestin3 (arr3) pathway for Y<sub>4</sub>R signaling,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> we investigated the activity of VU0637120 on arr3 recruitment as an alternative pathway to G protein signaling. PP-induced arr3 recruitment to the Y<sub>4</sub>R was analyzed in kinetic experiments by detecting arr3 recruitment to the Y<sub>4</sub>R constantly over 30 min (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>d). The study demonstrates that VU0637120 slows arr3 recruitment to the Y<sub>4</sub>R in response to PP compared to the dimethyl sulfoxide (DMSO) control, which can be confirmed by the significantly reduced rate constant <i>k</i> of arr3 recruitment to the Y<sub>4</sub>R in the presence of 30 μM of VU0637120. Furthermore, the effect of VU0637120 on the internalization of the Y<sub>4</sub>R was investigated (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Figure S2</a>, Supporting Information). Data show that VU0637120 itself has no effect on the membrane localization of the Y<sub>4</sub>R as the amount of quantified cell surface receptors is comparable to the DMSO control. Stimulation of Y<sub>4</sub>R with either 10 or 100 nM PP in the presence of DMSO induces internalization of the Y<sub>4</sub>R, indicated by a reduced amount of Y<sub>4</sub>R cell surface receptors. In contrast, VU0637120 blocks Y<sub>4</sub>R internalization induced by 10 nM PP and only slightly decreases Y<sub>4</sub>R internalization by 100 nM PP compared to DMSO.</div><div class="NLM_p">To further characterize the effect of VU0637120 on the endogenous ligand PP at the Y<sub>4</sub>R, radioactive ligand-binding studies were performed. Competition binding studies revealed that increasing concentrations of VU0637120 reduce PP binding at the Y<sub>4</sub>R (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>e). Whereas high concentrations of VU0637120 reduce specific binding of 60 pM <sup>125</sup>I-PP (1 × <i>K</i><sub>D</sub>) down to about 15%, the competition curve observed with 180 pM failed to reach a bottom plateau, and even high concentrations of the compound were not able to entirely displace the binding of the orthosteric radioligand. Further binding studies showed that the <i>K</i><sub>i</sub> of PP increases in the presence of VU0637120, confirming that the compound modulates the activity of the endogenous ligand (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>f). These experiments verify an allosteric mechanism of VU0637120, as the binding of a NAM to its receptor often induces a low-affinity state for the binding of the orthosteric ligand, which can result in reduced radioligand binding in competition binding studies.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> This finding is also in agreement with the effect of VU0637120 on the reduction of PP affinity to its Y<sub>4</sub>R orthosteric binding pocket (α = 0.02) in Ca<sup>2+</sup> flux assay. VU0637120 is structurally distinct from all previously described Y receptor antagonists, supporting the proposition of an allosteric mechanism of VU0637120.</div><div class="NLM_p last">With respect to the different functions of Y receptors in the regulation of satiety and energy metabolism, a high selectivity of VU0637120 to the Y<sub>4</sub>R is important.<a onclick="showRef(event, 'ref3 ref24'); return false;" href="javascript:void(0);" class="ref ref3 ref24">(3,24)</a> We first investigated the effect of VU0637120 on the Y<sub>1</sub>R, Y<sub>2</sub>R, and Y<sub>5</sub>R in vitro. A high concentration of 30 μM of VU0637120 has no effect on the efficacy (<i>E</i><sub>Max</sub>) and potency (EC<sub>50</sub>) of NPY-induced response at the Y<sub>1</sub>R, Y<sub>2</sub>R, and Y<sub>5</sub>R, which demonstrates the selectivity of VU0637120 to the Y<sub>4</sub>R subtype (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>g).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Confirmation of (<i>S</i>)<i>-</i>VU0637120 Activity in Ex Vivo Assays on Mouse Tissue</h3><div class="NLM_p">In addition to the in vitro characterization of VU0637120, we investigated the activity of the enantioselective compounds ex vivo in mucosal preparations of mouse descending colon. This tissue was shown previously to express native Y<sub>4</sub>R.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Electrophysiological measurements of vectorial ion transport showed a decrease in the size of rat PP (rPP) responses after exposure of mucosae to the increasing concentrations of (<i>S</i>)<i>-</i>VU0637120, exhibiting an IC<sub>50</sub> value of 3.8 μM (pIC<sub>50</sub> 5.42 ± 0.27). Consistent with results from the Ca<sup>2+</sup> flux assays, the (<i>R</i>)-VU0637120 enantiomer had no effect on rPP signaling (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a). To further study the selectivity of VU0637120 in mouse colon mucosa, PYY-induced activation of Y<sub>1</sub>R and Y<sub>2</sub>R, which are also expressed in these mucosal preparations, was examined. PYY responses were unaffected by either enantiomer (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b), showing selectivity for the Y<sub>4</sub>R, rather than Y<sub>1</sub>R or Y<sub>2</sub>R PYY-induced signaling in these native preparations.</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/medium/jm0c02000_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Investigation of VU0637120 effect ex vivo. (a) (<i>S</i>)<i>-</i>VU0637120 inhibits optimal rPP (30 nM)-mediated inhibition of ion transport in a concentration-dependent manner, while 30 μM of (<i>R</i>)-VU0637120 has no effect on the Y<sub>4</sub>R agonist in the mucosal preparations of mouse descending colon. (b) Subsequent PYY signals (10 nM) are resistant to either enantiomer of VU0637120 at the different concentrations shown, compared to the DMSO control. Ex vivo data represent the mean ± SEM from <i>N</i> = 5 experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02000&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Mutagenesis Studies Identified Y<sub>4</sub>R Key Residues Involved in VU0637120 Activity</h3><div class="NLM_p">To characterize the interaction between VU0637120 and Y<sub>4</sub>R and to gain insight into the mechanism of allosteric compounds, we first tested the activity of VU0637120 at eight Y<sub>4</sub>R/Y<sub>1</sub>R chimeras (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a) to elucidate Y<sub>4</sub>R domains, which are relevant for the interaction with the compound. The activity of VU0637120 at all Y<sub>4</sub>R/Y<sub>1</sub>R chimeras is displayed as the reduction of PP response in the presence of 15 μM antagonist, compared to the DMSO control (bar graphs, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b). The results suggest an important role of transmembrane helix 1 (TM1), TM2, and extracellular loop 2 (ECL2) for the activity of VU0637120, as the exchange of these Y<sub>4</sub>R regions by the corresponding sequences of the Y<sub>1</sub>R drastically reduces the antagonistic activity (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b).</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/medium/jm0c02000_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Investigation of the VU0637120 binding site at the Y<sub>4</sub>R. (a) Schemes of Y<sub>4</sub>R/Y<sub>1</sub>R chimeras, whereby Y<sub>4</sub>R segments are shown in white and Y<sub>1</sub>R segments are colored in black. (b) Importance of TM1, TM2, and ECL2 for VU0637120 activity. Loss of submaximal PP response (PP EC<sub>80</sub>) in the presence of 15 μM VU0637120 at Y<sub>4</sub>R, Y<sub>1</sub>R, and chimeras investigated in Ca<sup>2+</sup> flux assays. Statistical analysis was performed using one-way analysis of variance (ANOVA) and Dunnett’s post-test: *<i>P</i> < 0.05, **<i>P</i> < 0.01, and ***<i>P</i> < 0.001. (c) Effect of increasing concentrations of (<i>S</i>)<i>-</i>VU0637120 at important Y<sub>4</sub>R residues in TM1, TM2, TM3/4, and TM7 in response to a submaximal PP concentration (PP EC<sub>80</sub>) using Ca<sup>2+</sup> flux assay. Receptor activation was investigated in COS7 cells transiently expressing one specific receptor-eYFP construct and the chimeric G protein Δ6Gα<sub>qi4myr</sub>. Data are shown as mean ± SEM from <i>N</i> ≥ 3 independent experiments. Residues are numbered according to the nomenclature of Ballesteros–Weinstein.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> (d) Y<sub>4</sub>R snake plot, residues important for VU0637120 activity are highlighted in green, and residues not important for VU0637120 are marked in black. Adapted from GPCRdb.org.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02000&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Based on these data, we performed an initial screen of 75 variants of Y<sub>4</sub>R with segmental and single-point mutations (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Figure S3</a>, Supporting Information). For the initial mutant screen, the racemic mixture of VU0637120 was used, as it is commercially available and thus less expensive compared to the enantioselective compounds. Based on these results, the systematic investigation of the Y<sub>4</sub>R extracellular domains identified 11 positions, including Y1.39, Q2.58, T2.61, Y2.64, W2.70, S3.28, A3.29, Q3.32, F4.60, F7.35, and L7.36 to be critical for the antagonistic activity of VU0637120 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Figure S3</a>, Supporting Information). To consider the effect of a mutation on the functionality of the receptor, all constructs were characterized for membrane localization (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Figure S4</a>, Supporting Information) and PP activation (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Tables S2 and S3</a>, Supporting Information).</div><div class="NLM_p last">To more thoroughly characterize the binding pocket of the active (<i>S</i>)<i>-</i>VU0637120 enantiomer at the Y<sub>4</sub>R, positions that have been identified in the initial screen to be important for VU0637120 were tested with (<i>S</i>)<i>-</i>VU0637120. The antagonistic activity of the compound was determined by the measurement of submaximal receptor activation (PP EC<sub>80</sub>) in the presence of increasing (<i>S</i>)<i>-</i>VU0637120 concentrations. Here, we confirmed the importance of positions Y1.39, Q2.58, T2.61, Y2.64, W2.70, S3.28, A3.29, Q3.32, F4.60, F7.35, and L7.36 for (<i>S</i>)<i>-</i>VU0637120 as mutations at these positions lead to a dramatic loss of antagonistic activity of (<i>S</i>)<i>-</i>VU0637120 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c). A summary of the results of the mutagenesis data, highlighting important positions for VU0637120, is given in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>d.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Rosetta Docking Identified a Partial Allosteric Binding Pocket of (<i>S</i>)<i>-</i>VU0637120</h3><div class="NLM_p">To further delineate the structural determinants of activity, (<i>S</i>)<i>-</i>VU0637120 was docked using RosettaLigand<a onclick="showRef(event, 'ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28">(26−28)</a> into a Y<sub>4</sub>R homology model (HM) constructed with RosettaCM.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> A similar approach was performed previously to investigate the binding mode of PP (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Figure S5</a>, Supporting Information).<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The resulting docking models were prioritized based on Rosetta predicted binding energy (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Table S4</a>, Supporting Information) and the proximity of (<i>S</i>)<i>-</i>VU0637120 to the residues that were shown to be important in mutagenesis experiments (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">S3</a>). Initial placement of (<i>S</i>)<i>-</i>VU0637120 inside Y<sub>4</sub>R was guided by a significant loss in antagonistic activity of the compound to the variants Y<sub>4</sub>(Y1.39A) and Y<sub>4</sub>(Y1.39F) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">S3</a>). The best-scoring binding pose (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">S6</a>, cluster 1) places the ligand parallel to the membrane plane inside the transmembrane region. This binding mode is supported by the sensitivity of VU0637120 to mutation of F4.60 and A3.29. The loss of VU0637120 function by the introduction of the bigger, hydrophilic asparagine at position A3.29 confirms that the space on top of TM3, below the ECL2, is essential for VU0637120 activity (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">S3</a>). A second round of mutagenesis was conducted to confidently exclude two alternative binding poses (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Figures S6</a>, cluster 2/3 and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">S7</a>). The data reveal that (<i>S</i>)<i>-</i>VU0637120 docking poses of cluster 2 and cluster 3 are unlikely, as (<i>S</i>)<i>-</i>VU0637120 activity is insensitive to mutations at positions predicted to be important for the compound in cluster 2 (D1.27, T2.65, D2.68, N7.32, and M7.43) or cluster 3 (M2.53, S3.35, V3.36, V5.24, W6.48, N7.32, and M7.43). The docking model of cluster 1 is consistent with the mutagenesis data. It shows that (<i>S</i>)<i>-</i>VU0637120 forms polar contacts with Y1.39, Q3.32, and H7.39, CH−π interactions with F4.60 and F7.35, and nonpolar contacts to Y2.64 and L7.36 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Thereby, residues Y2.64, W2.70, F7.35, and L7.36 form the top surface area of the binding site. The distances among the hydroxyl group of Y2.64, the amine group in the indole ring of W2.70, and the oxygen in the amide group of (<i>S</i>)<i>-</i>VU0637120 are close to the range of significant polar contacts (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b). It is possible that those polar interactions exist, changing the orientation of the side chain of W2.70 and Y2.64, and destabilize the PP binding mode.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/medium/jm0c02000_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Predicted binding mode of (<i>S</i>)<i>-</i>VU0637120. (a) Computational docking of (<i>S</i>)<i>-</i>VU0637120 identified an allosteric binding site located below the extracellular interface. (b, c) Contacts of (<i>S</i>)<i>-</i>VU0637120 with most of the important residues (side view and top view). (d) Interaction network among TM1, TM2, TM7, and (<i>S</i>)<i>-</i>VU0637120. Y<sub>4</sub>R homology models were generated using 10 class A G protein-coupled receptor (GPCR) X-ray crystal structures (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ZBQ">5ZBQ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4S0V">4S0V</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZJC">4ZJC</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DHG">5DHG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N6H">4N6H</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DJH">4DJH</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DKL">4DKL</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5GLH">5GLH</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ODU">3ODU</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NDD">5NDD</a>; <a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Table S5</a>, Supporting Information).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02000&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The model further predicts the role of TM1, TM2, and ECL2 for Y<sub>4</sub>R selectivity of VU0637120, which was observed by the investigation of different Y<sub>4</sub>R/Y<sub>1</sub>R chimeras (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a,b). Mutation of specific fragments and positions identified regions 1.40–1.43 and Q2.58 to be highly important (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">S3</a>). In the model, these positions are in close proximity and form polar contacts between TM1 and TM2. In addition, position H7.39 has polar interactions with Y1.39 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>c,d). Thus, these residues might be important for the TM1–TM2 and TM1–TM7 orientation to shape the binding pocket of (<i>S</i>)<i>-</i>VU0637120. The role of the ECL2 was investigated by the mutation of almost the entire loop sequence (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Figure S3</a>, Supporting Information). However, none of these mutations had a strong effect on VU0637120 activity. Thus, the loss of antagonistic activity at the Y<sub>4</sub>R chimera containing the ECL2 of the Y<sub>1</sub>R (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a,b, chimera H) is likely due to conformational effects, which is in agreement with the high sequence variation of the loops between Y<sub>4</sub>R and Y<sub>1</sub>R (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Figure S8</a>, Supporting Information). The docking model locates (<i>S</i>)<i>-</i>VU0637120 directly below the conserved disulfide bridge and suggests a structural role of the ECL2 in the formation of the binding pocket (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a).</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Identification of Y<sub>4</sub>R Gain-of-Function Variants for (<i>S</i>)<i>-</i>VU0637120</h3><div class="NLM_p">Based on the docking of (<i>S</i>)<i>-</i>VU0637120 to the Y<sub>4</sub>R, we used RosettaDesign to predict Y<sub>4</sub>R variants that are favorable for the activity of (<i>S</i>)<i>-</i>VU0637120. The top variants predicted by RosettaDesign were inspected visually. The E6.58R variant was selected because the mutated residue is predicted to form an additional hydrogen bond interaction to the methanesulfonamide group of the compound. Additionally, A3.29V and A3.29P variants were chosen, as both valine and proline could form better hydrophobic interactions to the six-member ring of (<i>S</i>)<i>-</i>VU0637120. Those predicted favorable interactions were confirmed in the docking model of (<i>S</i>)<i>-</i>VU0637120 to single Y<sub>4</sub>R variants, which were reconstructed with RosettaCM<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> and RosettaLigand<a onclick="showRef(event, 'ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28">(26−28)</a> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>a).</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/medium/jm0c02000_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Gain-of-function of (<i>S</i>)<i>-</i>VU0637120 at specific Y<sub>4</sub>R variants. (a) Point mutant models made by Rosetta illustrate the improvement in hydrophobic interactions between (<i>S</i>)<i>-</i>VU0637120 and P3.29 or V3.29, and the additional hydrogen bond between (<i>S</i>)<i>-</i>VU0637120 and R6.58. Ten class A GPCR templates were used for building the Y<sub>4</sub>R homology models (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ZBQ">5ZBQ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4S0V">4S0V</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZJC">4ZJC</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DHG">5DHG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N6H">4N6H</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DJH">4DJH</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DKL">4DKL</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5GLH">5GLH</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ODU">3ODU</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NDD">5NDD</a>; <a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Table S5</a>, Supporting Information). (b) Confirmation of P3.29, V3.29, and R6.58 as Y<sub>4</sub>R gain-of-function variants in vitro. Shown are concentration–response curves of (<i>S</i>)<i>-</i>VU0637120, stimulated with a submaximal PP EC<sub>80</sub> concentration at specifically identified Y<sub>4</sub>R residues. (c) Loss of a submaximal PP concentration in the presence of 15 μM of (<i>S</i>)<i>-</i>VU0637120 at Y<sub>4</sub>R gain-of-function variants, data from <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>(b). Ca<sup>2+</sup> flux assays were performed in COS7 cells transiently expressing one specific Y<sub>4</sub>R-eYFP variant and the chimeric G protein Δ6Gα<sub>qi4myr</sub>. Data are shown as mean ± SEM of <i>N</i> ≥ 3 independent experiments. Comparison of (<i>S</i>)<i>-</i>VU0637120 activity at Y<sub>4</sub>R and variants was performed using one-way ANOVA and Dunnett’s post-test: **<i>P</i> < 0.01; ***<i>P</i> < 0.001. Residues are numbered according to the nomenclature of Ballesteros–Weinstein.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02000&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To check whether we can verify the RosettaDesign prediction in in vitro assays, we created the Y<sub>4</sub>R gain-of-function variants by mutagenesis and tested the effect of (<i>S</i>)<i>-</i>VU0637120 at these constructs. Here, we demonstrate that the antagonistic effect of (<i>S</i>)<i>-</i>VU0637120 on PP response is significantly increased at the variants Y<sub>4</sub>(E6.58R), Y<sub>4</sub>(A3.29P), and Y<sub>4</sub>(A3.29V) compared to the wild-type Y<sub>4</sub>R (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>b,c). These experiments confirm the accuracy of the (<i>S</i>)<i>-</i>VU0637120 binding mode to the Y<sub>4</sub>R as we are able to predict and confirm favorable interactions between distinct Y<sub>4</sub>R residues and functional groups of (<i>S</i>)<i>-</i>VU0637120.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Comparison of the Orthosteric Ligand-Binding Pocket and the Allosteric Binding Site</h3><div class="NLM_p">To map the binding pocket of the allosteric antagonist VU0637120 in comparison to the orthosteric agonist PP, mutagenesis experiments were performed for both ligands (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Tables S2 and S3; Figure S3</a>, Supporting Information). The models of PP docked to Y<sub>4</sub>R were reconstructed as we remade Y<sub>4</sub>R homology models with a more realistic ECL2 loop and an updated list of templates (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Table S5</a>, Supporting Information), which include the Y<sub>1</sub>R crystal structure, discovered in 2018.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Previous studies and our most recent mutagenesis data revealed that positions D2.68, W2.70, N6.55, D6.59, N7.32, F7.35, and M7.43 are important for Y<sub>4</sub>R activation by PP and demonstrated that PP binds at the Y<sub>4</sub>R in a flat binding mode (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">S5</a>).<a onclick="showRef(event, 'ref30 ref32'); return false;" href="javascript:void(0);" class="ref ref30 ref32">(30,32)</a> Additionally, positions T2.61, W2.70, and F7.35 were identified to be critical for both, the activity of the orthosteric peptidic agonist PP and the small-molecule antagonist VU0637120. In contrast, most of the positions that influence VU0637120 activity, including Y1.39, Q2.58, Y2.64, S3.28, A3.29, F4.60, and L7.36, are located in a deeper pocket and fail to affect PP activation (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). These results indicate that VU0637120 binds in a secondary, allosteric binding pocket at the Y<sub>4</sub>R with a slight overlap with the binding site of the endogenous ligand PP. Whereas the Y<sub>4</sub>R-PP docking model indicates a more open conformation of the Y<sub>4</sub>R in complex with PP, the binding of VU0637120 might stabilize a more closed receptor conformation. Thus, it follows that VU0637120 binding might create a low-affinity binding state for the endogeneous ligand PP with reduced agonist binding.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/medium/jm0c02000_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Comparing the binding pocket of PP and (<i>S</i>)<i>-</i>VU0637120. (a, b) Superimposition of docking models of PP and (<i>S</i>)<i>-</i>VU0637120 to Y<sub>4</sub>R. Residues that are important for the activity of PP, (<i>S</i>)<i>-</i>VU0637120, and both are colored in green, red, and yellow, respectively. Group of residues that are represented in the red surface: (*): Y1.39, E1.42, Q2.58, T2.61, S3.28, A3.29, Q3.32, F4.60, L7.36, and H7.39. (c) (<i>S</i>)<i>-</i>VU0637120 binds in a secondary, allosteric binding pocket (red) at the Y<sub>4</sub>R, but this site might slightly overlap with the orthosteric binding site of PP (green). Y<sub>4</sub>R homology models were generated from the X-ray crystal structure of 10 class A GPCRs (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ZBQ">5ZBQ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4S0V">4S0V</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZJC">4ZJC</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DHG">5DHG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N6H">4N6H</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DJH">4DJH</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DKL">4DKL</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5GLH">5GLH</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ODU">3ODU</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NDD">5NDD</a>; <a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Table S5</a>, Supporting Information).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02000&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Interestingly, the binding pocket of (<i>S</i>)<i>-</i>VU0637120 at the Y<sub>4</sub>R overlaps with that of orthosteric antagonists and NAMs of other peptide GPCRs (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Figure S9</a>, Supporting Information). The crystal structure of the related Y<sub>1</sub>R in complex with orthosteric antagonists revealed that the binding pocket of the antagonist BMS-193885 extends from the peptide binding site to the center of the transmembrane domain among TM3, TM4, TM5, and TM6, and partially overlaps with the binding site of VU0637120 at its six-member ring and methanesulfonamide group.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> In contrast, the binding site of (<i>S</i>)<i>-</i>VU0637120 stretches horizontally toward TM1, TM2, and TM7. This region is known as an allosteric pocket in other rhodopsin-like GPCRs such as C–C chemokine receptor type 5 (CCR5)<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and protease-activated receptor 2 (PAR2).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> The interaction with these receptor regions might explain the allosteric character of (<i>S</i>)<i>-</i>VU0637120. Furthermore, positions in TM1 and TM2 were shown to be critical for the Y<sub>4</sub>R selectivity of VU0637120. These findings highlight the potential to develop highly selective Y<sub>4</sub>R ligands by targeting this binding pocket.</div><div class="NLM_p last">The discovery of (<i>S</i>)<i>-</i>VU0637120 defines a novel class of Y<sub>4</sub>R antagonists that are smaller and more selective than the most promising compounds described to date. A small-molecule compound with weak antagonistic activity at the Y<sub>4</sub>R is UR-AK49, but this compound is not Y<sub>4</sub>R selective and can also bind to Y<sub>1</sub>R and Y<sub>5</sub>R.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Previous Y<sub>4</sub>R antagonists were generated by modifying Y<sub>1</sub>R antagonists (BIBP3226,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> BIBO3304<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a>). However, the most active compounds developed by this approach still display substantial activity at Y<sub>1</sub>R (UR-MEK381) or Y<sub>2</sub>R (UR MEK388), respectively.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Recently, a peptide-based Y<sub>4</sub>R antagonist has been identified by modifying the structure of a known Y<sub>4</sub>R hexapeptide agonist.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> However, peptide ligands that activate peptide GPCRs often face challenges with dosing and pharmacokinetic properties such as rapid clearance, poor metabolic stability, and consequently poor bioavailability, as well as weak membrane permeability.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Additionally, it is very challenging to develop selective orthosteric peptide ligands within multiligand/multireceptor systems. Such difficulties have been noted with PP, the native peptide ligand of the Y<sub>4</sub>R, which possesses a half-life in plasma of about 7 min<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> and shows a high affinity for Y<sub>5</sub>R.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Therefore, allosteric modulators, such as (<i>S</i>)<i>-</i>VU0637120, provide a promising new class of ligands to target peptide GPCRs. As these molecules bind to allosteric sites, which are often less conserved compared to orthosteric ligand-binding sites of receptor subtypes, a higher subtype selectivity can be achieved.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Furthermore, allosteric molecules of peptide GPCRs have the advantage to modulate or fine-tune the receptor response by the endogeneous peptide and consequently maintain the normal physiological regulation.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> This study will support the analysis of this relevant class of modulators and facilitate drug development for peptide GPCRs in general.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24682" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24682" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this study, we highlight the characterization of the first-in-class allosteric antagonist (<i>S</i>)<i>-</i>VU0637120 that exhibits clear Y<sub>4</sub>R selectivity in engineered cell lines and in proven native Y<sub>4</sub>R-expressing mouse colon mucosa. We obtained a model for the (<i>S</i>)<i>-</i>VU0637120/Y<sub>4</sub>R complex that not only agrees with known mutagenesis data but also predicted three gain-of-function mutations that increased the activity of (<i>S</i>)<i>-</i>VU0637120. For the enantioselective (<i>S</i>)<i>-</i>VU0637120 compound, we identified an allosteric binding pocket in the core of the Y<sub>4</sub>R transmembrane domains below the endogenous ligand-binding site, which ultimately strengthens our understanding of allosteric modulation of the Y<sub>4</sub>R.</div><div class="NLM_p last">The proposed binding pose of (<i>S</i>)<i>-</i>VU0637120 highlights patterns of locations among allosteric pockets across different GPCR class A subfamilies despite their low sequence identity. The discovery of small, selective Y<sub>4</sub>R allosteric antagonists helps pave the way for validating Y<sub>4</sub>R as a target for treating metabolic diseases such as anorexia and obesity by providing key structural insights that may be extended to other chemotypes. Looking at a broader picture, this study demonstrates that targeting this allosteric binding pocket holds promise for the development of novel, selective compounds as tools for the investigation of peptide GPCRs, which represent about 15% of all GPCRs.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52234" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52234" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Cell Culture</h3><div class="NLM_p last">COS7 cells were obtained from the American Type Culture Collection (ATCC). Wild-type COS7 cells were cultured in Dulbecco’s modified Eagle’s medium (DMEM, Lonza) supplemented with 10% heat-inactivated fetal bovine serum (FBS, Biochrom). The COS7 cell lines stably expressing one specific human Y receptor-eYFP fusion protein (Y<sub>1,2,4,5</sub>R-eYFP) and the chimeric G protein Δ6Gα<sub>qi4myr</sub> were created using pVitro2-hygro-mcs vectors as previously described<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> and cultured in DMEM supplemented with 10% FBS, 133 μg/mL hygromycin (Invivogen), and 1.5 mg/mL G418 sulfate (Invivogen). HEK293 cells were obtained from Deutsche Sammlung von Mikroorganismen und Zellkulturen (DSMZ). Wild-type HEK293 cells were cultured in DMEM/Ham’s F12 (1:1 (v/v), Lonza) supplemented with 15% FBS. Stably transfected HEK293_Y<sub>4</sub>R-eYFP cells were generated using a hY<sub>4</sub>R-eYFP-pVitro2-neo-mcs vector as described<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> and cultured in DMEM/Ham’s F12 (1:1 (v/v), Lonza) with 15% FBS and 100 μg/mL hygromycin. All cell lines were cultured in a humidified atmosphere at 37 °C and 5% CO<sub>2</sub>.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Plasmids</h3><div class="NLM_p last">For the Y<sub>4</sub>R mutant screening, the cDNA of Y<sub>4</sub>R, Y<sub>1</sub>R, Y<sub>4</sub>R/Y<sub>1</sub>R chimeras, and Y<sub>4</sub>R mutants were cloned in a pEYFP_N1 vector (Clontech), C-terminally tagged with an enhanced yellow fluorescent protein (eYFP), as performed in previous studies.<a onclick="showRef(event, 'ref30 ref32'); return false;" href="javascript:void(0);" class="ref ref30 ref32">(30,32)</a> The chimeric G protein Δ6Gα<sub>qi4myr</sub> cDNA was provided by E. Kostenis (Rheinische Friedrich-Wilhelm-Universität, Bonn, Germany)<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> and cloned into a pVitro2-mcs-neo vector. In the arr3 recruitment studies, the Y<sub>4</sub>R was C-terminally tagged to <i>Renilla</i> luciferase 8 (Rluc8) and cloned in a pcDNA3 vector. Arr3 was N-terminally fused to venus fluorescent protein and cloned in a pcDNA3 plasmid.<a onclick="showRef(event, 'ref13 ref20'); return false;" href="javascript:void(0);" class="ref ref13 ref20">(13,20)</a> The sequence of all constructs was confirmed by DNA sequencing.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Generation of Y<sub>4</sub>R/Y<sub>1</sub>R Chimeras and Y<sub>4</sub>R Mutants</h3><div class="NLM_p last">The chimeric Y<sub>1</sub>R/Y<sub>4</sub>R receptors were created using overlap extension polymerase chain reaction (PCR).<a onclick="showRef(event, 'ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref47 ref48">(47,48)</a> In the first PCR, the N-terminal cDNA of the donor receptor subtype was amplified using a sense primer, which annealed before the N-terminal receptor sequence and contained a specific enzyme restriction site. The antisense primer annealed at the end of the desired segment and contained an overhang complementary to the cDNA of the second receptor subtype. The cDNA of the C-terminal fragment was obtained in a second PCR using the cDNA from the second receptor subtype. Here, the sense primer annealed at the beginning of the desired segment and contained a 5′ overhang sequence complementary to the cDNA of the N-terminal segment of the donor receptor, which was created in the first PCR. The antisense primer of the second PCR contained a restriction site sequence and annealed after the receptor sequence. The fragments obtained in both PCRs were purified using agarose gel electrophoresis and Promega Wizard SV Gel and PCR Clean-Up System (Promega). In a third PCR, the receptor fragments were combined using the N-terminal and C-terminal fragments with sense and antisense primers annealing at the beginning or end of the respective fragments to obtain the chimeric cDNA. Afterwards, the PCR product was digested using BgIII and <i>Xba</i>I or <i>Sal</i>I and <i>Not</i>I enzymes (ThermoFischer). The desired products were purified by gel electrophoresis and ligated into a pEYFP_N1 vector using T4 DNA ligase (ThermoFischer). The mutation of single amino acids or 2–4 adjacent residues in the Y<sub>4</sub>R was performed by Quick change site-directed mutagenesis using PCR and appropriate primers. In the PCR, the Phusion (ThermoFischer) or Pfu Turbo DNA polymerase (Agilent) was used with 5× HF (Phusion) or 10× reaction (Pfu) buffer, 100 ng Y<sub>4</sub>R-eYFP_N1 template, 0.25–0.5 μM sense/antisense primers, 200 μM deoxyribonucleotide (dNTP) mix, and 1–10% DMSO. The PCR conditions were adjusted according to the manufacturer’s protocol of the DNA polymerase. Afterwards, the parental DNA was digested using <i>Dpn</i>I (ThermoFischer). The PCR products were transformed into chemically competent <i>Escherichia coli</i><i>DH5α</i> using heat shock (60 s, 42 °C) and single clones were separated on Luria broth (LB) medium-agarose plates containing kanamycin (30 μg/mL, Sigma-Aldrich). The plasmid DNA was prepared using Promega Wizard SV Gel and PCR Clean-Up System or Promega Pureyield Midiprep System (Promega) according to the manufacturer’s protocol. Finally, the sequence of all Y<sub>4</sub>R/Y<sub>1</sub>R chimeras and Y<sub>4</sub>R mutants was confirmed by DNA sequencing.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> High-Throughput Screening Ca<sup>2+</sup> Flux Assay</h3><div class="NLM_p last">Ca<sup>2+</sup> flux screening assays to identify Y<sub>4</sub>R allosteric modulators were performed in a 384-well format using Panoptic (WaveFront Biosciences). The compound screening was performed with stably transfected HEK293 cells coexpressing human Y<sub>4</sub>R and G<sub>qi5</sub> α subunit protein (Addgene plasmid #24501). In a following HTS, 77 021 compounds were tested for Y<sub>4</sub>R modulatory effects and 579 primary hits were detected. Forty-one compounds from the primary hits were reordered as dry compounds and re-evaluated with different concentrations, also for nonspecific effects in wild-type HEK293 cells. Seventeen compounds were confirmed. Seventy-seven thousand compounds were from the Vanderbilt Discovery Collection, a custom library designed in collaboration with Vanderbilt, industry-trained medicinal chemists, Vanderbilt chemoinformaticists, and in collaboration with the vendor, Life Chemicals. The Vanderbilt Discovery Collection is a diverse, druglike library, which had been largely depleted of “problem” compounds (e.g., pan assay interference compounds, PAINS). For the Ca<sup>2+</sup> flux screening, cells were cultured in α minimum essential medium (MEM) (Corning) with 10% FBS (Invitrogen), 1× glutagro (Corning), 800 μg/mL G418 sulfate (Cellgro), and 250 μg/mL zeocin (Life Science) to a confluency of about 90%, then plated in amine-coated 384-well plates (black, clear bottom, Corning) in 20 μL cell culture medium/well without antibiotics using a Multidrop Combi microplate dispenser (ThermoScientific, Thermo Fisher) at a density of 20 000 cells/well. The cells were incubated for 18 h at 37 °C in the presence of 5% CO<sub>2</sub>. Following incubation, the medium was replaced with 20 μL/well fluorescent dye solution, containing 1.0 μM Fluo-8 AM AAT (Bioquest), 0.01% (v/v) pluronic acid F-127 in assay buffer (Hanks’ balanced salt solution (HBSS) buffer, Invitrogen, with 20 mM <i>N</i>-(2-hydroxyethyl)piperazine-<i>N</i>′-ethanesulfonic acid (HEPES), Invitrogen, pH 7.3–7.4) by plate flicking. Following a 60 min incubation at room temperature, fluorescent dye solution was replaced with 20 μL/well assay buffer as described above. The cell plates were loaded into Panoptic (Drug Screening System, WaveFront Biosciences) and imaged at 1 Hz (excitation 480 ± 40 nm, emission 538 ± 40 nm using a three-addition protocol designed to detect agonists, potentiators, and inhibitors). After collecting 10 s of baseline, 20 μL/well of 20 μM test compounds in assay buffer were added to make the final concentration of each compound to be 10 μM. Following a 240 s delay, 10 μL/well of concentration of 5-fold over the PP EC<sub>30</sub> (∼2 nM) in assay buffer containing 0.0125% bovine serum albumin (BSA) (Sigma-Aldrich) was added. After 240 s, a 12 μL/well addition 5-fold over the PP EC<sub>80</sub> (∼20 nM) in assay buffer containing 0.0125% BSA was performed, imaged for another 240 s.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Ca<sup>2+</sup> Flux Assay</h3><div class="NLM_p last">The compound characterization was performed with stably transfected COS7 cells expressing one specific Y receptor subtype (Y<sub>1,2,4,5</sub>R-eYFP) and the chimeric G protein Δ6Gα<sub>qi4myr</sub> as previously described.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> To investigate Y<sub>4</sub>R/Y<sub>1</sub>R chimera and Y<sub>4</sub>R variants, COS7 cells were transiently transfected with one specific receptor construct (C-terminally tagged with eYFP) and the chimeric G protein Δ6Gα<sub>qi4myr</sub> using Metafectene Pro transfection reagent. Therefore, COS7 cells were seeded in 25 cm<sup>2</sup> culture flasks and grown to a confluency of 80%. Each flask was transfected with a total of 4 μg of plasmid DNA (3 μg of receptor-eYFP and 1 μg of Δ6Gα<sub>qi4myr</sub>) using 15 μL of Metafectene Pro according to manufacturer’s protocol over night at 37 °C. On the next day, 20 000 cells per well were reseeded in black, clear bottom 96-well plates (Greiner) and incubated over night at 37 °C. For the assay, the cell culture medium was aspirated, and cells were incubated in 100 μL/well assay buffer (HBSS (Lonza), 20 mM HEPES (Sigma), 2.5 mM probenecid (Sigma), pH 7.4) containing 2.4 μM Fluo2-AM (Abcam) calcium dye. Afterwards, the dye solution was replaced by assay buffer and Ca<sup>2+</sup> signals were detected (excitation 485 nm, emission 525 nm) in a FlexStation III device (Molecular Devices). The compound VU0637120 and agonists (PP/NPY) were added automatically in a two-addition protocol. After 20 s baseline detection, DMSO control or VU0637120 was added to the cells, followed by the addition of PP or NPY after 110 s in a total run time of 140 s. The Ca<sup>2+</sup> signal responses were analyzed as <i>x</i>-fold over basal values and normalized to the maximum activation obtained in the presence of 1 μM PP (Y<sub>4</sub>R, Y<sub>4</sub>R/Y<sub>1</sub>R chimeras and Y<sub>4</sub>R variants) or NPY (Y<sub>1</sub>R, Y<sub>2</sub>R, Y<sub>5</sub>R).</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Y<sub>4</sub>R Membrane Preparations and Radioligand Binding Studies</h3><div class="NLM_p last">Y<sub>4</sub>R membranes were generated from stably transfected HEK293_Y<sub>4</sub>R-eYFP cells, as previously described.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Cells were suspended in Dulbecco’s phosphate-buffered saline (DPBS) and centrifuged at 1800 rpm, 4 °C for 5 min. The cell pellet was resuspended in Tris buffer (50 mM Tris (Sigma), 50 μM Pefabloc (Sigma), pH 7.5) and homogenized in a potter grinder (Potter SB Braun). The cell suspension was centrifuged at 2400 rpm at 4 °C for 20 min; the supernatant was removed and centrifuged for 60 min at 12 000 rpm, 4 °C. The supernatant was discarded and the membrane pellet was resuspended in HEPES buffer (25 mM HEPES, 25 mM CaCl<sub>2</sub> (Fluka), 1 mM MgCl<sub>2</sub> (Fluka), 50 μM Pefabloc (Sigma), pH 7.4). After the homogenization in the potter grinder, the suspension was centrifuged at 12 000 rpm at 4 °C for 60 min. The resulting pellet was resuspended in HEPES buffer and the protein concentration was determined using Bradford protein assay.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> Membrane preparations were frozen in liquid nitrogen and stored at −20 °C. For binding experiments, 0.3 μg of Y<sub>4</sub>R membrane preparation was suspended in HEPES buffer containing either DMSO as control or VU0637120. PP and <sup>125</sup>I-PP (human) solutions were prepared in aqua dest. supplemented with 0.1% BSA (PAA Laboratories). Nonspecific binding was determined in the presence of 1 μM PP. Radioligand solutions (60 or 180 pM <sup>125</sup>I-PP) were added to all samples and incubated for 5 h (at room temperature, 200 rpm). Afterwards, membrane-bound <sup>125</sup>I-PP was separated by filtration using a GFC filter (PerkinElmer), presoaked with 0.1% polyethylenimine (Sigma) in a MicroBeta 96-well filtermate harvester (PerkinElmer). Membranes were washed with cold DPBS, dried for 15 min at 55 °C, and treated with MeltiLex scintillation sheets. For the quantification of radioactivity, membranes were measured in a MicroBeta scintillation counter (PerkinElmer).</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> IP-One Assay</h3><div class="NLM_p last">The IP-One assay was performed in COS7 cells stably expressing the Y<sub>4</sub>R-eYFP and the chimeric G protein Δ6Gα<sub>qi4myr</sub>. Therefore, cells were seeded into white 384-well plates (Greiner) at a density of 8000 cells/well and incubated over night under standard conditions. For the detection of IP production, the IP-One Gq assay kit (Cisbio) was used. On the following day, media was removed, (<i>S</i>)<i>-</i>VU0637120 or DMSO was added followed by the subsequent stimulation with peptide solution in HBSS + 20 mM LiCl for 1 h at 37 °C, 5% CO<sub>2</sub>. Afterwards, cells were lysed with lysis buffer containing antibody 1 and antibody 2 according to the manufacturer’s protocol. Signals were detected using a Tecan Spark microplate reader (Tecan Group AG) measuring emission at 665 and 620 nm.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Arrestin3 Recruitment Assay</h3><div class="NLM_p last">Arrestin3 recruitment to the Y<sub>4</sub>R was measured using BRET assay in HEK293 cells transiently transfected with Y<sub>4</sub>R-Rluc8 and venus-arr3 fusion proteins as previously described.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> For kinetic measurements, signals were measured continuously over 30 min after PP (100 nM) addition. BRET signals were observed as the ratio of fluorescence (venus-arr3) and luminescence (Y<sub>4</sub>R-Rluc8). NetBRET signals were calculated by subtraction of unstimulated controls.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Fluorescence Microscopy</h3><div class="NLM_p last">The internalization of the Y<sub>4</sub>R was investigated using fluorescence microscopy in HEK293 cells stably expressing the Y<sub>4</sub>R-eYFP fusion protein. Cells were seeded in μ-slide eight well (IBIDI) chambers at a density of 180 000 cells/well and incubated over night at 37 °C. For microscopy, the cell culture medium was aspirated and nuclei of the cells were stained with 300 μL/well OptiMEM (serum-reduced media, Gibco) containing Hoechst33342 (Sigma) for 30 min at 37 °C. Afterwards, the solution was aspirated and 150 μL/well OptiMEM containing 30 μM VU0637120 or DMSO control were added. Typically, 50 μL/well PP solution (10 or 100 nM PP final concentration) or OptiMEM (unstimulated samples) was added, and the cells were stimulated for 30 min at 37 °C. The stimulation solution was replaced by 200 μL of OptiMEM. Microscopy was performed using the AxioVert Observer Z1 (ZEISS) fluorescence microscope (YFP: filter set 46, 4′,6-diamidino-2-phenylindole (DAPI): filter set 49, ApoTome, 63×/1.40 oil objective). The images were processed using ZEN2012 (ZEISS) and Axiovision SE64 (ZEISS) software. The detection of the eYFP fluorescence was performed with a constant exposure time (400 ms) to allow the quantification of cell surface receptors as performed previously.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> The cell surface fluorescence was quantified as pixel intensity in black/white pictures using ImageJ software and normalized to the unstimulated DMSO control.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Electrophysiological Measurements</h3><div class="NLM_p last">All of the animal experiments performed in this manuscript were conducted in compliance with the local ethics committee and is covered by the national (U.K.) Home Office Project license (P6EA0199C) held at KCL. Vectorial ion transport was measured across mucosal preparations of mouse descending colon as described previously.<a onclick="showRef(event, 'ref13 ref25'); return false;" href="javascript:void(0);" class="ref ref13 ref25">(13,25)</a> Up to six adjacent preparations were dissected from each colon. Mucosae were placed individually between Ussing chamber halves and voltage-clamped at 0 mV (using a DVC1000; WPI, Sarasota, FL), as described previously. Mucosae with exposed areas of 0.14 cm<sup>2</sup> were bathed in oxygenated Krebs–Henseleit solution on both sides, at 37 °C, and were voltage-clamped at 0 mV. A stable basal <i>I</i><sub>sc</sub> was reached within 20 min after which DMSO (0.3%) or antagonist additions (300 nM to 30 μM of the active enantiomer (<i>S</i>)<i>-</i>VU637120 or 30 μM of inactive (<i>R</i>)-VU637120) were made to the basolateral reservoir. After an intermediary addition of the secretagogue VIP (10 nM, for 10–15 min), an optimal single rPP concentration (30 nM)<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> was added to activate epithelial Y<sub>4</sub>R and reduce <i>I</i><sub>sc</sub> levels, followed 15 min later by PYY (10 nM). Changes in <i>I</i><sub>sc</sub> to each drug addition (all to the basolateral reservoir) were pooled and presented as mean ± SEM. The pIC<sub>50</sub> and statistical analysis were performed using GraphPad Prism software (version 7).</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Homology Modeling of Y<sub>4</sub>R</h3><div class="NLM_p last">Sequences of Y<sub>4</sub>R and the templates were aligned using the Clustal Omega web server.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Multiple sequence alignment was then adjusted to ensure that the helix regions and reserved residues remain aligned, and to remove gaps within transmembrane α-helices. Y<sub>4</sub>R homology models were built using RosettaCM protocol<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> of the Rosetta3.9 software suites.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Addition constraints were set up to account for the disulfide bond between Cys1.25 and Cys7.38, as well as between Cys3.25 and Cys5.25. The output Y<sub>4</sub>R homology models were evaluated by Rosetta total energy score, which includes knowledge-based energy terms such as hydrogen bonds, electrostatic interactions, and van der Waals packing.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Top 10% of output models, which scored most favorably by Rosetta, were clustered based on the RMSD cutoff of 2 Å using BCL::Cluster.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> The clustering procedure was extensively discussed in the precedent protocol paper.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Finally, a Y<sub>4</sub>R model with the best Rosetta total energy score was chosen from each of the 10 largest clusters.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Docking NAM to Y<sub>4</sub>R</h3><div class="NLM_p">As the experimental data suggested that the ligand could interact with residues located on TM1, TM2, and TM7, it is crucial to create enough space among those three transmembrane regions in the homology models before ligand docking. The output Y<sub>4</sub>R homology models from the first round of RosettaCM did not yield enough space for VU0637120 to approach and form hydrogen bonds to the residue Y1.39. Hence, the Y<sub>4</sub>R models were rehybridized with the ligand placed inside the area among important residues. To create a set of templates for the second rounds of RosettaCM, 65 different poses of the ligand were placed inside Y<sub>4</sub>R such that it interacted to TM1, TM2, TM7, and in no more than 4 Å from Y1.39. This way, the interactions between the ligand and the important residues according to the mutagenesis data would be weighed in the placement of helices in MC trajectory and optimization. The resulting models were then filtered for interactions to important residues and hydrogen bond to Y1.39 and clustered. The final 10 cluster representative models were selected to be templates for the docking protocol based on the size of the corresponding clusters as well as the openness of the binding pocket suggested by the mutagenesis data.</div><div class="NLM_p last">A set of 106 conformations of VU0637120 was generated using the ConformerGenerator application of BCL v3.2.2.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> RosettaLigand was used to dock VU0637120 to Y<sub>4</sub>R homology models. The docking protocol is described in a previous publication.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> In addition to Rosetta total energy score, the interface_delta score, which is relative to the predicted binding energy between VU0637120 and Y<sub>4</sub>R, was also extracted for all 20 000 output docking models. We selected models with a total Rosetta score of at least 95% of the best total Rosetta score attained and the top 10% interface delta score. We applied an experimental filter on the selected docking models. The filter makes sure that the selected ligand poses that contact 9 out of the 11 residues were shown to be important to the antagonistic activity by mutagenesis data. A residue is determined to contact the ligand if at least one atom of the residue is at most 5 Å away from any atom of the ligand. Those filtered models were then clustered based on the ligand position with 3 Å RMSD cutoff. For each of the five largest clusters, the interaction contact and strength were computed for each cluster such that the scores are higher when an important residue has high contact frequency to the ligand or favorable binding energy toward the ligand. The interaction contact score is computed as<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/medium/jm0c02000_m001.gif" alt="" id="_i26" /></img></span>where <i>ct</i><sub><i>i</i></sub> and <i>ct</i><sub><i>n</i></sub> are the percents of docking poses that have the ligand contact with residues that have been identified as important and residues that were not confirmed as critical for binding residue, respectively. The weight of 4.7 was chosen to balance the impact of 13 critical with 61 noncritical residues. Similarly, the interaction strength of a cluster <i>t</i> is calculated as<span class="NLM_disp-formula" id="ueq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/medium/jm0c02000_m002.gif" alt="" id="_i27" /></img></span>where <i>ddg</i><sub><i>i</i></sub> and <i>ddg</i><sub><i>n</i></sub> are the predicted binding energies between the ligand and an important residue and noncritical residue, respectively. In the cluster with the most favorable interaction contact and strength scores, a model that best explains the mutagenesis data will be chosen as a putative binding pose of VU0637120 and Y<sub>4</sub>R. Per residue contact and interaction strength are listed in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Tables S4 and S6</a>, Supporting Information.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Predicting Gain-of-Function Y<sub>4</sub>R Variants</h3><div class="NLM_p last">The residues that interact with VU0637120 were redesigned with the Rosetta binding pocket design protocol.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> Predicted variants that are located on TM3, TM4, and TM5 are selected only if they specifically interact with the methylsulfonyl tail and the six-member ring head of the pose of VU0637120 cluster 1. The single-mutant models of the predicted gain-of-function variants are then rebuilt with RosettaCM to verify favorable selective interactions of new residues to VU0637120 cluster 1.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Docking PP to Y<sub>4</sub>R</h3><div class="NLM_p last">PP was docked to Y<sub>4</sub>R homology models (built in step 10) through four rounds of RosettaCM and FlexPepDock with the gradually increasing length of PP (last 5, last 8, and last 23 residues of PP). We removed the ECL2 from the Y<sub>4</sub>R homology models (HM) before docking 5mer PP and added the ECL2 back in after docking 23mer PP. A similar protocol can be found in a previous publication.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> The docking was guided by experimental restraints from mutagenesis studies.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Details of the restraints are described in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Table S7</a>, Supporting Information.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Data Analysis</h3><div class="NLM_p">For data analysis, GraphPad Prism 5.0 (or version 7.0, for electrophysiology) and python matplotlib packages were used. EC<sub>50</sub> and <i>E</i><sub>max</sub> values of concentration–response curves were obtained by nonlinear regression analysis (curve fit). The quantification of the effect of VU0637120 in Ca<sup>2+</sup> flux assays was performed using an operational model of allosterism (<a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>).<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> The parameters of the model describe: <i>E</i><sub>m</sub>—maximum system response; <i>n</i>—related to the slope of response curves; [A] and [B]—concentration of agonist (A) and modulator (B); τ<sub>A</sub> and τ<sub>B</sub>—intrinsic agonism (efficacy) of the agonist (A) and the compound (B); α and β—effect of the modulator on the affinity (α) or the efficacy (β) of the agonist A. In the calculations, the <i>K</i><sub>A</sub> of the agonist PP (log <i>K</i><sub>A</sub> −10.2) was constrained to values determined in previous binding assays.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a><span class="NLM_disp-formula" id="eq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/medium/jm0c02000_m003.gif" alt="" id="_i31" /></img><span class="labelSpan">(1)</span></span></div><div class="NLM_p">The arr3 recruitment rates <i>k</i> were calculated from kinetic traces using the one-phase association equation, as performed previously.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a><span class="NLM_disp-formula" id="eq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/medium/jm0c02000_m004.gif" alt="" id="_i32" /></img><span class="labelSpan">(2)</span></span></div><div class="NLM_p last">Schild analysis for allosteric antagonist was performed using <a class="ref internalNav" href="#eq3" aria-label="eq 3">eq 3</a>, whereas [B] describes the concentration of the allosteric antagonist, EC<sub>50</sub> refers to the EC<sub>50</sub> of the DMSO control, and EC<sub>50</sub>′ to the EC<sub>50</sub> in the presence of (<i>S</i>)<i>-</i>VU0637120.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a><span class="NLM_disp-formula" id="eq3"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/medium/jm0c02000_m005.gif" alt="" id="_i33" /></img><span class="labelSpan">(3)</span></span></div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Compound Synthesis and Characterization</h3><div class="NLM_p last">The compound VU0637120 (product ID: F5141-0188) was purchased from Life Chemicals Inc. (Ukraine). Purity was determined by reversed-phase high-performance liquid chromatography (RP-HPLC) employing two different columns (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Figure S10</a>, Supporting Information). Purity of racemic VU0637120 is >95%. The enantiomers (<i>R</i>)- and (<i>S</i>)<i>-</i>VU0637120 were synthesized and purified individually. <sup>1</sup>H NMR spectra and RP-HPLC were employed to determine purity of the enantioselective compounds. (<i>R</i>)- and (<i>S</i>)<i>-</i>VU0637120 have a purity of >95% (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Figures S10 and S11</a>, Supporting Information). Enantiomeric excess (ee) was determined by chiral supercritical fluid chromatography (SFC) separation performed on a Thar (Waters) Investigator at 220 nm UV (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Figure S12</a>, Supporting Information).</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Synthesis and Purification</h3><div class="NLM_p last">Air sensitive reactions were carried out under a nitrogen atmosphere (Airgas Catalog no. NI UHP300). The following solvents were employed for chemical reactions: dichloromethane (99.9%, extra dry, AcroSeal, Acros Organics Catalog no. 610300010) and <i>N</i>,<i>N</i>-dimethylformamide (anhydrous, 99.8%, packaged under argon in resealable ChemSeal bottles, Alfa Aesar Catalog no. 43997). The following solvents were employed for working up reactions and/or extractions: ethyl acetate (Certified ACS grade, Fisher Chemical Catalog no. E145-20), ethyl ether (anhydrous, BHT stabilized, Certified ACS, Fisher Chemical Catalog no. E138-4), and dichloromethane (not stabilized, HPLC grade, Fisher Chemical Catalog no. D150-4). Brine was prepared from deionized water and sodium chloride (Reagent grade, Fisher Chemical Catalog no. S25541B). Anhydrous sodium sulfate (Lab grade, Fisher Chemical Catalog no. S25568A) was employed for drying organic extracts. Thin-layer chromatography (TLC) was conducted on glass plates coated with silica gel 60 F<sub>254</sub> from Millipore Sigma (Catalog no. 1057150001). Normal phase flash chromatography was carried out on either a CombiFlash EZ Prep or CombiFlash Rf+ automated flash chromatography system, both from Teledyne ISCO. Normal phase flash chromatography was carried out using Redi<i>Sep</i> Rf normal phase disposable flash columns (40–60 μm) from Teledyne ISCO (Catalog nos. 69-2203-304, 69-2203-312, 69-2203-324, 69-2203-340, 69-2203-380, and 69-2203-320). The following solvents were employed for TLC and normal phase chromatography: hexanes (Certified ACS grade, Fisher Chemical Catalog no. H292-20), ethyl acetate (Certified ACS grade, Fisher Chemical Catalog no. E145-20), dichloromethane (not stabilized, HPLC grade, Fisher Chemical Catalog no. D150-4), and methanol (HPLC grade, Fisher Chemical, Catalog no. A452-4). Reverse-phase preparative HPLC was carried out on a CombiFlash EZ Prep automated flash chromatography system equipped with a Redi<i>Sep</i> Prep C18 10 × 250 mm<sup>2</sup>, 100 Å, 5 μm HPLC preparative column from Teledyne ISCO (Catalog no. 692203809). The following solvents were employed for reversed-phase chromatography: acetonitrile (HPLC grade, Fisher Chemical Catalog no. A998SK-4) and water purified using a Milli-Q Advantage A10 Water Purification System from Millipore Sigma.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Compound Characterization</h3><div class="NLM_p last">All NMR spectra were recorded on a 300 MHz Bruker Fourier 300HD NMR spectrometer equipped with a dual <sup>1</sup>H and <sup>13</sup>C probe with <i>Z</i>-Gradient and automatic tuning and matching, full computer control of all shims with TopShim, 24-sample SampleCase automation system, and TopSpin software. All NMR samples were prepared with either methyl sulfoxide-<i>d</i><sub>6</sub> with 0.03% tetramethylsilane (TMS), 99.8 atom % D, Acros Organics Catalog no. 360000100 or chloroform-<i>d</i> with 0.03% TMS, 99.8+ atom % D, Acros Organics Catalog no. 209561000. <sup>1</sup>H and <sup>13</sup>C chemical shifts are reported in δ values in ppm downfield with tetramethylsilane (TMS) as the internal standard. Data are reported as follows: chemical shift, multiplicity (s = singlet, d = doublet, t = triplet, q = quartet, b = broad, m = multiplet), integration, coupling constant (Hz). High-resolution mass spectrometry was conducted on an Agilent 6230 accurate-mass time-of-flight (TOF) LC/MS with the ESI source equipped with MassHunter Walkup software. MS parameters were as follows: fragmentor: 175 V, capillary voltage: 3500 V, nebulizer pressure: 35 psig, drying gas flow: 11 L/min, drying gas temperature: 325 °C. Samples were introduced via an Agilent 1260 Infinity UHPLC comprised of a G4225A HiP Degasser, G1312B binary pump, G1367E ALS, G1316A TCC, and G1315C DAD VL+ with a 5 μL semimicro flow cell with a 6 mm path length. UV absorption was observed at 220 and 254 nm with a 4 nm bandwidth. Column: Agilent Zorbax SB-C18, Rapid Resolution HT, and 1.8 μm, 2.1 × 50 mm<sup>2</sup>. Gradient conditions: hold at 5% CH<sub>3</sub>CN in H<sub>2</sub>O (0.1% formic acid) for 1.0 min, 5–95% CH<sub>3</sub>CN in H<sub>2</sub>O (0.1% formic acid) over 5 min, hold at 95% CH<sub>3</sub>CN in H<sub>2</sub>O (0.1% formic acid) for 1.0 min, and 0.5 mL/min. Chiral SFC separation was performed on a Thar (Waters) Investigator at 220 nm (UV). Column: Chiral Technologies CHIRALPAK IB, 4.6 × 250 mm<sup>2</sup>, and 5 μm. Isocratic conditions: 50% MeOH (0.1% diethanolamine (DEA)) in CO<sub>2</sub> over 5 min. Flow rate: 3.5 mL/min. Column temperature: 40 <b>°</b>C. System backpressure: 100 bar. All final analogs were at least 95% pure according to these analytical methods.</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> (<i>S</i>)<i>-</i>VU0637120 and (<i>R</i>)-VU0637120</h3><div class="NLM_p">Starting material 6,7-dihydro-[1,4]dioxino[2′,3′:4,5]benzo[1,2-<i>d</i>]thiazol-2-amine <b>1</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) is available commercially from several vendors; however, it was prepared according to the literature<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> utilizing the following method. <i>N</i>-Bromosuccinimide (1.30 g, 7.30 mmol) was added in portions to a suspension of NH<sub>4</sub>SCN (1.10 g, 14.5 mmol) in AcOH (32 mL) at 10 °C. The reaction mixture was stirred at the same temperature for 30 min. After filtration to remove the insoluble materials, the filtrate was added to a solution of 3,4-ethylenedioxyaniline (Chem-Impex Catalog no. 27084) (1.00 g, 6.62 mmol) in AcOH (32 mL), and the reaction mixture was stirred at room temperature for 20 h. After removal of the solvent in vacuo, the residue was diluted with ethyl acetate (75 mL), washed with water and brine, and concentrated in vacuo. The crude product obtained was purified by flash chromatography to afford <b>1</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) as a yellow solid (600 mg, 44%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.38 (bs, 2H), 7.41 (s, 1H), 6.96 (s, 1H), and 4.28–4.23 (m, 4H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ 169.79, 146.07, 142.82, 139.86, 122.71, 108.16, 105.57, 64.23, and 64.12. High-resolution mass spectrometry (HRMS), calcd for C<sub>9</sub>H<sub>9</sub>N<sub>2</sub>O<sub>2</sub>S<sup>+</sup> [M + H], 209.0379; found 209.0378.</div><div id="sec4_19_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> <i>tert</i>-Butyl-(<i>S</i>)<i>-</i>2-((6,7-dihydro-[1,4]dioxino[2′,3′:4,5]benzo[1,2-<i>d</i>]thiazol-2-yl)carbamoyl)piperidine-1-carboxylate (<b>4</b>)</h4><div class="NLM_p last"><i>N</i>,<i>N</i>-Diisopropylethylamine (DIEA) (331 μL, 1.92 mmol) was added to a mixture of Boc-<span class="smallcaps smallerCapital">l</span>-homoproline (Chem-Impex Catalog no. 05135) (242 mg, 1.06 mmol), 1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate (HATU) (734 mg, 1.92 mmol) in DMF (4 mL) and stirred for 20 min. A solution of <b>1</b> (200 mg, 960 mmol) in DMF (2 mL) was added dropwise, and the solution was allowed to stir for 12 h. After completion of the reaction as judged by TLC, 15 mL of water was added. The mixture was extracted with ethyl acetate (2×), washed with brine, and dried over anhydrous sodium sulfate. The crude product was purified by flash chromatography using ethyl acetate and hexane as eluent, and the title compound was obtained as an off-white solid (346 mg, 86%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 9.50 (br s, 1H), 7.26 (s, 2H), 5.02 (br s, 1H), 4.30 (s, 4H), 4.06 (m, 1H), 2.91–2.75 (m, 1H), 2.37 (m, 1H), 1.78–1.36 (m, 5H), and 1.51 (s, 9H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 171.18, 169.79, 156.48, 143.42, 143.14, 141.92, 125.10, 108.59, 108.49, 81.52, 64.33, 60.41, 28.36, 24.97, 21.08, 20.56, and 14.22. HRMS, calcd for C<sub>20</sub>H<sub>26</sub>N<sub>3</sub>O<sub>5</sub>S<sup>+</sup> [M + H], 420.1588; found 420.1600.</div></div><div id="sec4_19_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> (<i>S</i>)<i>-N</i>-(6,7-Dihydro-[1,4]dioxino[2′,3′:4,5]benzo[1,2-<i>d</i>]thiazol-2-yl)piperidine-2-carboxamide (<b>6</b>)</h4><div class="NLM_p last">Compound <b>3</b> (340 mg, 0.811 mmol, <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) was dissolved in dichloromethane (6 mL) and cooled to 0 °C in an ice bath. Hydrogen chloride solution (4 mL, 4.0 M in dioxane, 16 mmol) was added dropwise, and the reaction mixture was stirred at room temperature for 2 h. After completion of the reaction as judged by TLC, the majority of the solvent was removed in vacuo, and the product was precipitated by the addition of diethyl ether. The crude product was filtered, washed with diethyl ether (2×), dried, and purified by preparative HPLC using acetonitrile and water as eluent (20% acetonitrile in water). The title compound was obtained as off-white solid (230 mg, 89%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.90 (d, <i>J</i> = 10.0 Hz, 1H), 9.13 (m, 1H), 7.51 (s, 1H), 7.26 (s, 1H), 4.29 (s, 4H), 4.15 (m, 1H), 3.30 (d, <i>J</i> = 11.8 Hz, 1H), 2.96 (m, 1H), 2.29 (d, <i>J</i> = 11.4 Hz, 1H), 1.93–1.41 (m, 5H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.73, 156.70, 143.67, 143.08, 142.04, 124.55, 109.28, 108.37, 64.48, 57.44, 43.80, 27.18, 22.03, and 21.54. HRMS, calcd for C<sub>15</sub>H<sub>18</sub>N<sub>3</sub>O<sub>3</sub>S<sup>+</sup> [M + H], 320.1063; found 320.1067.</div></div><div id="sec4_19_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> (<i>S</i>)<i>-N</i>-(6,7-Dihydro-[1,4]dioxino[2′,3′:4,5]benzo[1,2-<i>d</i>]thiazol-2-yl)-1-(methylsulfonyl)-piperidine-2-carboxamide ((<i>S</i>)<i>-</i>VU0637120)</h4><div class="NLM_p last">Triethylamine (128 μL, 0.936 mmol) was added to a solution of compound <b>4</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) in dichloromethane (2 mL) at 0 °C in an ice bath and stirred for 5 min. Methanesulfonyl chloride (40.0 μL, 0.376 mmol) was added dropwise, and the reaction was warmed to room temperature and stirred for an additional 1 h. After completion of the reaction, water (15 mL) was added to the reaction mixture, and it was extracted with dichloromethane (2×). The crude product was purified by flash chromatography using ethyl acetate and hexane as eluent, and the title compound was obtained as an off-white solid (75 mg, 60%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.39 (br s, 1H), 7.45 (s, 1H), 7.21 (s, 1H), 4.72 (d, <i>J</i> = 4.6 Hz, 1H), 4.27 (s, 4H), 3.67–3.47 (m, 2H), 2.93 (s, 3H), and 1.89–1.34 (m, 6H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 169.13, 156.52, 143.55, 142.80, 142.06, 124.86, 108.62, 108.37, 64.31, 64.26, 55.95, 43.34, 39.66, 26.35, 24.29, and 19.73. HRMS, calcd for C<sub>16</sub>H<sub>20</sub>N<sub>3</sub>O<sub>5</sub>S<sub>2</sub><sup>+</sup> [M + H], 398.0839; found 398.0848. [α]<sub>D</sub><sup>25</sup> −66° (<i>c</i> 0.098, CHCl<sub>3</sub>). 99.6% ee (<i>R</i><sub>T</sub> = 3.74).</div></div><div id="sec4_19_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> <i>tert</i>-Butyl-(<i>S</i>)<i>-</i>2-((6,7-dihydro-[1,4]dioxino[2′,3′:4,5]benzo[1,2-<i>d</i>]thiazol-2-yl)carbamoyl)piperidine-1-carboxylate (<b>5</b>)</h4><div class="NLM_p last">Intermediate <b>6</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) was synthesized analogous to intermediate <b>3</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) using <i>N</i>-Boc-homoproline (Chem-Impex Catalog no. 05499). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 9.51 (br s, 1H), 7.27 (s, 2H), 5.02 (br s, 1H), 4.30 (s, 4H), 4.06 (m, 1H), 2.89–2.75 (m, 1H), 2.39 (m, 1H), 1.77–1.36 (m, 5H), and 1.51 (s, 9H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 169.80, 156.53, 143.43, 143.07, 141.94, 140.90, 125.07, 108.57, 108.50, 81.52, 64.31, 60.42, 28.36, 24.69, 21.08, 20.57, and 14.22. HRMS, C<sub>20</sub>H<sub>26</sub>N<sub>3</sub>O<sub>5</sub>S<sup>+</sup> [M + H], 420.1588; found 420.1607.</div></div><div id="sec4_19_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> (<i>S</i>)<i>-N</i>-(6,7-dihydro-[1,4]dioxino[2′,3′:4,5]benzo[1,2-<i>d</i>]thiazol-2-yl)piperidine-2-carboxamide (<b>7</b>)</h4><div class="NLM_p last">Intermediate <b>7</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) was synthesized analogous to intermediate <b>4</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.83 (d, <i>J</i> = 9.8 Hz, 1H), 9.13 (m, 1H), 7.51 (s, 1H), 7.26 (s, 1H), 5.26 (br s, 1H), 4.29 (s, 4H), 4.13 (m, 1H), 3.30 (d, <i>J</i> = 12.0 Hz, 1H), 3.07 (m, 1H), 2.96 (m, 1H), 2.28 (d, <i>J</i> = 11.4 Hz, 1H), and 1.88–1.47 (m, 5H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.73, 156.71, 143.72, 143.07, 142.05, 124.55, 109.30, 108.38, 64.49, 57.47, 43.81, 27.18, 22.04, and 21.56. HRMS, calcd for C<sub>15</sub>H<sub>18</sub>N<sub>3</sub>O<sub>3</sub>S<sup>+</sup> [M + H], 320.1063; found 320.1075.</div></div><div id="sec4_19_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> (<i>R</i>)-<i>N</i>-(6,7-Dihydro-[1,4]dioxino[2′,3′:4,5]benzo[1,2-<i>d</i>]thiazol-2-yl)-1-(methylsulfonyl)-piperidine-2-carboxamide ((<i>R</i>)-VU0637120)</h4><div class="NLM_p last">(<i>R</i>)-VU0637120 was synthesized analogous to (<i>S</i>)<i>-</i>VU0637120. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.39 (br s, 1H), 7.45 (s, 1H), 7.21 (s, 1H), 4.72 (d, <i>J</i> = 4.2 Hz, 1H), 4.28 (s, 4H), 3.62–3.47 (m, 2H), 2.93 (s, 3H), and 1.89–1.19 (m, 6H). <sup>13</sup>C NMR (75 MHz, CDCl3) δ 168.88, 156.36, 143.57, 142.91, 142.09, 124.95, 108.70, 108.40, 64.34, 64.29, 56.03, 43.43, 39.86, 26.06, 24.26, and 19.82. HRMS, calcd for C<sub>16</sub>H<sub>20</sub>N<sub>3</sub>O<sub>5</sub>S<sub>2</sub><sup>+</sup> [M + H], 398.0839; found 398.0849. [α]<sub>D</sub><sup>25</sup> +73° (<i>c</i> 0.098, CHCl<sub>3</sub>). >99.9% ee (<i>R</i><sub>T</sub> = 2.84).</div></div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> Peptide Synthesis</h3><div class="NLM_p last">The peptides human PP (APLEPVYP GDNATPEQMAQYAADLRRYINMLTRPRY-NH<sub>2</sub>) and porcine NPY (YPSKPDNPGEDAPAEDLARYYSALRHYINLITRQRY-NH2) were synthesized by automated solid-phase peptide synthesis in an automated multiple peptide synthesis robot system (Syro, MultiSynTech) using the 9-fluorenylmethyloxycarbonyl (Fmoc) strategy as described previously.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Peptides were purified by preparative RP-HPLC on a Aeris PEPTIDE XB-C18 column (100 Å, 5 μm, 250 × 2.1 mm<sup>2</sup>, Phenomenex). Purity of the peptides (≥95%, <a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Figures S13 and S14</a>, Supporting Information) was determined by two different analytical RP-HPLC systems using gradients of 20–70% eluent B (0.08% TFA in acetonitrile (ACN)) in eluent A (0.1% trifluoroacetic acid (TFA) in H<sub>2</sub>O) in 40 min. Identity was confirmed by matrix-assisted laser desorption/ionization time-of-flight (MALDI-TOF) mass spectrometry (Ultraflex III MALDI-TOF/TOF, Bruker Daltonics) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Figures S13 and S14</a>, Supporting Information).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i45"><a href="/doi/suppl/10.1021/acs.jmedchem.0c02000" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74778" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74778" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c02000?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02000</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Effect of (<i>S</i>)-VU0637120 on G protein activation, internalization studies, Y<sub>4</sub>R variant screening for the investigation of the VU0637120 binding pocket, RosettaLigand docking clusters, analytics of compounds, and synthetic peptide ligands (Figures S1–S14); small-molecule screening data, PP activation of Y<sub>4</sub>R/Y1R chimeras and Y<sub>4</sub>R variants, templates for Y<sub>4</sub>R homology modeling, overview of docking parameters (Tables S1–S7) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Coordinates of final homology models (A3.29P-Y4-NAM.pdb) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_002.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Coordinates of final homology models (A3.29V-Y4-NAM.pdb) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Coordinates of final homology models (E6.58R-Y4-NAM.pdb) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_004.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Coordinates of final homology models (Y4-NAM-cluster1.pdb) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_005.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Coordinates of final homology models (Y4-NAM-cluster2.pdb) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_006.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Coordinates of final homology models (Y4-NAM-cluster3.pdb) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_007.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Coordinates of final homology models (Y4-PP.pdb) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_008.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_009.csv" class="ext-link">CSV</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_010.tif" class="ext-link">TIF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf">jm0c02000_si_001.pdf (3.11 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_002.pdb">jm0c02000_si_002.pdb (195.89 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_003.pdb">jm0c02000_si_003.pdb (195.89 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_004.pdb">jm0c02000_si_004.pdb (195.89 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_005.pdb">jm0c02000_si_005.pdb (396.11 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_006.pdb">jm0c02000_si_006.pdb (396.11 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_007.pdb">jm0c02000_si_007.pdb (396.11 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_008.pdb">jm0c02000_si_008.pdb (423.43 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_009.csv">jm0c02000_si_009.csv (0.33 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_010.tif">jm0c02000_si_010.tif (295.51 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c02000" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17512" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17512" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jens Meiler</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Vanderbilt University, Nashville, Tennessee 37235, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology, Vanderbilt University, Nashville, Tennessee 37232, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
for Drug Discovery, Leipzig University, Leipzig 04103, Germany</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f69c939885b69b939f9a93849a9794d8998491"><span class="__cf_email__" data-cfemail="b4ded1dac7f4d9d1ddd8d1c6d8d5d69adbc6d3">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Annette G. Beck-Sickinger</span> - <span class="hlFld-Affiliation affiliation">Institute
of Biochemistry, Leipzig University, Leipzig 04103, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4560-8020" title="Orcid link">http://orcid.org/0000-0003-4560-8020</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#0f6e6d6a6c64227c666c646661686a7d4f7a616622636a667f756668216b6a"><span class="__cf_email__" data-cfemail="5332313630387e203a30383a3d34362113263d3a7e3f363a23293a347d3736">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Corinna Schüß</span> - <span class="hlFld-Affiliation affiliation">Institute
of Biochemistry, Leipzig University, Leipzig 04103, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Oanh Vu</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Vanderbilt University, Nashville, Tennessee 37235, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4704-7538" title="Orcid link">http://orcid.org/0000-0003-4704-7538</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mario Schubert</span> - <span class="hlFld-Affiliation affiliation">Institute
of Biochemistry, Leipzig University, Leipzig 04103, Germany</span>; 
    <span>Present Address:
                        Institute of Pharmacology and Toxicology, Technische Universität Dresden, 01307 Dresden, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yu Du</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology, Vanderbilt University, Nashville, Tennessee 37232, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nigam M. Mishra</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Sciences, UNT System College of Pharmacy, University of North Texas Health Science Center, Fort Worth, Texas 76107, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Iain R. Tough</span> - <span class="hlFld-Affiliation affiliation">King’s
College London, Wolfson Centre for Age-Related Diseases, Institute of Psychiatry, Psychology & Neuroscience, London SE1 1UL, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jan Stichel</span> - <span class="hlFld-Affiliation affiliation">Institute
of Biochemistry, Leipzig University, Leipzig 04103, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">C. David Weaver</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Vanderbilt University, Nashville, Tennessee 37235, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology, Vanderbilt University, Nashville, Tennessee 37232, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Chemical Biology, Vanderbilt University, Nashville, Tennessee 37232, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6886-1195" title="Orcid link">http://orcid.org/0000-0002-6886-1195</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kyle A. Emmitte</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Sciences, UNT System College of Pharmacy, University of North Texas Health Science Center, Fort Worth, Texas 76107, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6643-3947" title="Orcid link">http://orcid.org/0000-0002-6643-3947</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Helen M. Cox</span> - <span class="hlFld-Affiliation affiliation">King’s
College London, Wolfson Centre for Age-Related Diseases, Institute of Psychiatry, Psychology & Neuroscience, London SE1 1UL, U.K.</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>C.S., O.V., and M.S. contributed equally to this work. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): C.D.W. is an owner of WaveFront Biosciences, maker of the Panoptic instrument used in high-throughput screening.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e3968-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i48">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13033" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13033" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank K. Löbner, J. Schwesinger, and C. Dammann for technical assistance. We kindly acknowledge the Vanderbilt HTS facility for the help in the identification of the Y<sub>4</sub>R modulators. Dr. A. Rodriguez, B.J. Bender, and N. Kühn are acknowledged for discussions about data analysis and work in docking studies. We are grateful to N. Kett of the Vanderbilt University Warren Center for Neuroscience Drug Discovery for performing the chiral SFC analysis. We acknowledge funding by the National Institutes of Health (Grant R01 DK0973376 to J.M. and C.D.W., grant R01 DA046138 to J.M. and grant 1S10OD021734 for the Panoptic kinetic imaging plate reader), the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) through CRC 1423, project number 421152132, B1 to A.G.B.-S and A7 to J.M., the European Union and the Free State of Saxony (Grants 100148835 and 143213128452 to A.G.B.-S.), and the BBSRC (BB/N006763/1 to H.M.C.).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">ACN</td><td class="NLM_def"><p class="first last">acetonitrile</p></td></tr><tr><td class="NLM_term">arr3</td><td class="NLM_def"><p class="first last">arrestin3</p></td></tr><tr><td class="NLM_term">CCR5</td><td class="NLM_def"><p class="first last">C–C chemokine receptor type 5</p></td></tr><tr><td class="NLM_term">COS7</td><td class="NLM_def"><p class="first last">African green monkey kidney cells</p></td></tr><tr><td class="NLM_term">DMEM</td><td class="NLM_def"><p class="first last">Dulbecco’s modified Eagle’s medium</p></td></tr><tr><td class="NLM_term">dNTP</td><td class="NLM_def"><p class="first last">deoxyribonucleotide</p></td></tr><tr><td class="NLM_term">DPBS</td><td class="NLM_def"><p class="first last">Dulbecco’s phosphate-buffered saline</p></td></tr><tr><td class="NLM_term">ECL</td><td class="NLM_def"><p class="first last">extracellular loop</p></td></tr><tr><td class="NLM_term">eYFP</td><td class="NLM_def"><p class="first last">enhanced yellow fluorescent protein</p></td></tr><tr><td class="NLM_term">FBS</td><td class="NLM_def"><p class="first last">fetal bovine serum</p></td></tr><tr><td class="NLM_term">GPCR</td><td class="NLM_def"><p class="first last">G protein-coupled receptor</p></td></tr><tr><td class="NLM_term">HBSS</td><td class="NLM_def"><p class="first last">Hank’s balanced saline solution</p></td></tr><tr><td class="NLM_term">HEK293</td><td class="NLM_def"><p class="first last">human embryonic kidney cells</p></td></tr><tr><td class="NLM_term">HM</td><td class="NLM_def"><p class="first last">homology model</p></td></tr><tr><td class="NLM_term">IP</td><td class="NLM_def"><p class="first last">inositol phosphate</p></td></tr><tr><td class="NLM_term"><i>K</i><sub>B</sub></td><td class="NLM_def"><p class="first last">equilibrium binding constant</p></td></tr><tr><td class="NLM_term">LB</td><td class="NLM_def"><p class="first last">Luria broth</p></td></tr><tr><td class="NLM_term">MALDI-TOF</td><td class="NLM_def"><p class="first last">matrix-assisted laser desorption/ionization time of flight</p></td></tr><tr><td class="NLM_term">NAM</td><td class="NLM_def"><p class="first last">negative allosteric modulator</p></td></tr><tr><td class="NLM_term">NPY</td><td class="NLM_def"><p class="first last">neuropeptide Y</p></td></tr><tr><td class="NLM_term">PAM</td><td class="NLM_def"><p class="first last">positive allosteric modulator</p></td></tr><tr><td class="NLM_term">PAR2</td><td class="NLM_def"><p class="first last">protease-activated receptor 2</p></td></tr><tr><td class="NLM_term">PP</td><td class="NLM_def"><p class="first last">pancreatic polypeptide</p></td></tr><tr><td class="NLM_term">PYY</td><td class="NLM_def"><p class="first last">peptide YY</p></td></tr><tr><td class="NLM_term">Rluc8</td><td class="NLM_def"><p class="first last"><i>Renilla</i> luciferase 8</p></td></tr><tr><td class="NLM_term">RP-HPLC</td><td class="NLM_def"><p class="first last">reversed-phase high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term">rPP</td><td class="NLM_def"><p class="first last">rat pancreatic polypeptide</p></td></tr><tr><td class="NLM_term">SEM</td><td class="NLM_def"><p class="first last">standard error of the mean</p></td></tr><tr><td class="NLM_term">TM</td><td class="NLM_def"><p class="first last">transmembrane helix</p></td></tr><tr><td class="NLM_term">Y<sub>1/2/4/5</sub>R</td><td class="NLM_def"><p class="first last">neuropeptide Y receptor 1/2/4/5</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i50">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87818" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87818" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 55 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pedragosa-Badia, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stichel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span> <span> </span><span class="NLM_article-title">Neuropeptide Y receptors: how to get subtype selectivity</span>. <i>Front. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">5</span> <span class="refDoi"> DOI: 10.3389/fendo.2013.00005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.3389%2Ffendo.2013.00005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=23382728" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A280%3ADC%252BC3szls1GntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&author=X.+Pedragosa-Badiaauthor=J.+Stichelauthor=A.+G.+Beck-Sickinger&title=Neuropeptide+Y+receptors%3A+how+to+get+subtype+selectivity&doi=10.3389%2Ffendo.2013.00005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Neuropeptide Y receptors: how to get subtype selectivity</span></div><div class="casAuthors">Pedragosa-Badia Xavier; Stichel Jan; Beck-Sickinger Annette G</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in endocrinology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5</span>
        ISSN:<span class="NLM_cas:issn">1664-2392</span>.
    </div><div class="casAbstract">The neuropeptide Y (NPY) system is a multireceptor/multiligand system consisting of four receptors in humans (hY(1), hY(2), hY(4), hY(5)) and three agonists (NPY, PYY, PP) that activate these receptors with different potency.  The relevance of this system in diseases like obesity or cancer, and the different role that each receptor plays influencing different biological processes makes this system suitable for the design of subtype selectivity studies.  In this review we focus on the latest findings within the NPY system, we summarize recent mutagenesis studies, structure activity relationship studies, receptor chimera, and selective ligands focusing also on the binding mode of the native agonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSR-p8hgInRDnl8icjszYl0fW6udTcc2ea_5rWQsy2r57ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3szls1GntA%253D%253D&md5=7f0c792b923299292f75dabb643e4105</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.3389%2Ffendo.2013.00005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffendo.2013.00005%26sid%3Dliteratum%253Aachs%26aulast%3DPedragosa-Badia%26aufirst%3DX.%26aulast%3DStichel%26aufirst%3DJ.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26atitle%3DNeuropeptide%2520Y%2520receptors%253A%2520how%2520to%2520get%2520subtype%2520selectivity%26jtitle%3DFront.%2520Endocrinol.%26date%3D2013%26volume%3D4%26doi%3D10.3389%2Ffendo.2013.00005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yulyaningsih, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herzog, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sainsbury, A.</span></span> <span> </span><span class="NLM_article-title">NPY receptors as potential targets for anti-obesity drug development</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>163</i></span>,  <span class="NLM_fpage">1170</span>– <span class="NLM_lpage">1202</span>, <span class="refDoi"> DOI: 10.1111/j.1476-5381.2011.01363.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1111%2Fj.1476-5381.2011.01363.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=21545413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC3MXps1Sjsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=163&publication_year=2011&pages=1170-1202&author=E.+Yulyaningsihauthor=L.+Zhangauthor=H.+Herzogauthor=A.+Sainsbury&title=NPY+receptors+as+potential+targets+for+anti-obesity+drug+development&doi=10.1111%2Fj.1476-5381.2011.01363.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">NPY receptors as potential targets for anti-obesity drug development</span></div><div class="casAuthors">Yulyaningsih, Ernie; Zhang, Lei; Herzog, Herbert; Sainsbury, Amanda</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">163</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1170-1202</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  The neuropeptide Y system has proven to be one of the most important regulators of feeding behavior and energy homeostasis, thus presenting great potential as a therapeutic target for the treatment of disorders such as obesity and at the other extreme, anorexia.  Due to the initial lack of pharmacol. tools that are active in vivo, functions of the different Y receptors were mainly studied in knockout and transgenic mouse models.  However, over recent years various Y receptor selective peptidic and non-peptidic agonists and antagonists were developed and tested.  Their therapeutic potential in relation to treating obesity and other disorders of energy homeostasis is discussed in this review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoo39ZkydVcgbVg90H21EOLACvtfcHk0lgcsud_LWc1OQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXps1Sjsb0%253D&md5=051cd4220d7aea714e61764917210fcc</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2011.01363.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2011.01363.x%26sid%3Dliteratum%253Aachs%26aulast%3DYulyaningsih%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DHerzog%26aufirst%3DH.%26aulast%3DSainsbury%26aufirst%3DA.%26atitle%3DNPY%2520receptors%2520as%2520potential%2520targets%2520for%2520anti-obesity%2520drug%2520development%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2011%26volume%3D163%26spage%3D1170%26epage%3D1202%26doi%3D10.1111%2Fj.1476-5381.2011.01363.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bijker, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herzog, H.</span></span> <span> </span><span class="NLM_article-title">The neuropeptide Y system: pathophysiological and therapeutic implications in obesity and cancer</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">91</span>– <span class="NLM_lpage">113</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2011.03.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2Fj.pharmthera.2011.03.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=21439311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmsVyltbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2011&pages=91-113&author=L.+Zhangauthor=M.+S.+Bijkerauthor=H.+Herzog&title=The+neuropeptide+Y+system%3A+pathophysiological+and+therapeutic+implications+in+obesity+and+cancer&doi=10.1016%2Fj.pharmthera.2011.03.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The neuropeptide Y system: Pathophysiological and therapeutic implications in obesity and cancer</span></div><div class="casAuthors">Zhang, Lei; Bijker, Martijn S.; Herzog, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">91-113</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  The neuropeptide Y (NPY) system - comprising of neuropeptide Y, peptide YY, pancreatic polypeptide and the corresponding Y receptors through which they act (Y1, Y2, Y4, Y5 and y6) - is well known for its role in the regulation of energy homeostasis and assocd. processes.  Dysfunctions of the system have been implicated in human diseases such as obesity and cancer, raising the possibility that correction of the system may provide therapeutic benefits for these diseases.  In addn. to the regulation of appetite and satiety that has attracted most attention during the past years, insight has also been gained into the crit. role of NPY in the control of energy expenditure, oxidative fuel selection and bone metab.  Studies using conditional knockout models further shed light on the central vs. peripheral, and hypothalamic vs. extra-hypothalamic mechanisms of these regulatory effects of NPY.  Moreover, a role of NPY family peptides and Y receptors in modulating the growth of tumors has emerged.  These findings provide the basis for novel NPY system-targeted strategies to treat obesity as well as cancer.  Such strategies include modifying both sides of the energy balance equation - energy intake vs. energy expenditure - to achieve a greater wt./fat loss by particularly modulating peripheral Y receptor(s) to ameliorate metabolic conditions without interfering with central functions of Y receptors.  In addn., targeting multiple Y receptors and/or multiple systems involved in the regulation of energy balance will have greater beneficial effects.  However, long-term interference with the NPY system to target obesity or cancer related aspects needs to consider potential side effects on bone health.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnPMeJgSGUdLVg90H21EOLACvtfcHk0lgcsud_LWc1OQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmsVyltbY%253D&md5=9817e55c615e9d6084fe17b71ff85d7d</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2011.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2011.03.011%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DBijker%26aufirst%3DM.%2BS.%26aulast%3DHerzog%26aufirst%3DH.%26atitle%3DThe%2520neuropeptide%2520Y%2520system%253A%2520pathophysiological%2520and%2520therapeutic%2520implications%2520in%2520obesity%2520and%2520cancer%26jtitle%3DPharmacol.%2520Ther.%26date%3D2011%26volume%3D131%26spage%3D91%26epage%3D113%26doi%3D10.1016%2Fj.pharmthera.2011.03.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sato, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogino, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mashiko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ando, M.</span></span> <span> </span><span class="NLM_article-title">Modulation of neuropeptide Y receptors for the treatment of obesity</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1401</span>– <span class="NLM_lpage">1415</span>, <span class="refDoi"> DOI: 10.1517/13543770903251722</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1517%2F13543770903251722" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=19743896" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFyhtLfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=1401-1415&author=N.+Satoauthor=Y.+Oginoauthor=S.+Mashikoauthor=M.+Ando&title=Modulation+of+neuropeptide+Y+receptors+for+the+treatment+of+obesity&doi=10.1517%2F13543770903251722"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Modulation of neuropeptide Y receptors for the treatment of obesity</span></div><div class="casAuthors">Sato, Nagaaki; Ogino, Yoshio; Mashiko, Satoshi; Ando, Makoto</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1401-1415</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Background: Neuropeptide Y (NPY) has been demonstrated to have crit. roles in the physiol. control of appetite and energy homeostasis through NPY Y1, Y2, Y4 and Y5 receptors.  A no. of synthetic ligands for NPY receptor subtypes have been developed to date, with Y5 receptor antagonists and Y2 and Y4 receptor agonists advancing into clin. trials.  Methods: A survey of the scientific and patent literature since mid-2006 is presented.  Conclusion: In addn. to the specific modulation of resp. NPY receptor subtypes, recent investigations have revealed that modulation of multiple NPY receptor subtypes produces additive or even synergistic anti-obesity effects.  Development of reliable small mol. Y1, Y2 and Y4 receptor ligands would greatly accelerate investigations and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCyDX0rwHE1LVg90H21EOLACvtfcHk0lhqJW5btwxyJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFyhtLfK&md5=687624982f64ecd0c2b2cf1a70f8f7ba</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1517%2F13543770903251722&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543770903251722%26sid%3Dliteratum%253Aachs%26aulast%3DSato%26aufirst%3DN.%26aulast%3DOgino%26aufirst%3DY.%26aulast%3DMashiko%26aufirst%3DS.%26aulast%3DAndo%26aufirst%3DM.%26atitle%3DModulation%2520of%2520neuropeptide%2520Y%2520receptors%2520for%2520the%2520treatment%2520of%2520obesity%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2009%26volume%3D19%26spage%3D1401%26epage%3D1415%26doi%3D10.1517%2F13543770903251722" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Batterham, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Roux, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frost, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghatei, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bloom, S. R.</span></span> <span> </span><span class="NLM_article-title">Pancreatic polypeptide reduces appetite and food intake in humans</span>. <i>J. Clin. Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">3989</span>– <span class="NLM_lpage">3992</span>, <span class="refDoi"> DOI: 10.1210/jc.2003-030630</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1210%2Fjc.2003-030630" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=12915697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmsFOku7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2003&pages=3989-3992&author=R.+L.+Batterhamauthor=C.+W.+Le+Rouxauthor=M.+A.+Cohenauthor=A.+J.+Parkauthor=S.+M.+Ellisauthor=M.+Pattersonauthor=G.+S.+Frostauthor=M.+A.+Ghateiauthor=S.+R.+Bloom&title=Pancreatic+polypeptide+reduces+appetite+and+food+intake+in+humans&doi=10.1210%2Fjc.2003-030630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Pancreatic polypeptide reduces appetite and food intake in humans</span></div><div class="casAuthors">Batterham, R. L.; Le Roux, C. W.; Cohen, M. A.; Park, A. J.; Ellis, S. M.; Patterson, M.; Frost, G. S.; Ghatei, M. A.; Bloom, S. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3989-3992</span>CODEN:
                <span class="NLM_cas:coden">JCEMAZ</span>;
        ISSN:<span class="NLM_cas:issn">0021-972X</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">Pancreatic polypeptide (PP) is a gut hormone released from the pancreas in response to ingestion of food.  Plasma PP has been shown to be reduced in conditions assocd. with increased food intake and elevated in anorexia nervosa.  In addn. peripheral administration of PP has been shown to decrease food intake in rodents.  These findings suggest that PP may act as a circulating factor that regulates food intake.  Therefore we investigated the effect of i.v. infusion of PP (10 pmol/kg/min) on appetite and food intake in a randomized double-blind placebo-controlled crossover study in ten healthy volunteers.  Infusion of PP reduced appetite and decreased the energy intake at a buffet lunch two hours post-infusion by 21.8±5.7% (P < 0.01).  More importantly the inhibition of food intake was sustained, such that energy intake, as assessed by food diaries, was significantly reduced both the evening of the study and the following morning.  Overall PP infusion reduced cumulative 24-h energy intake by 25.3±5.8%.  In conclusion our data demonstrates that PP causes a sustained decrease in both appetite and food intake.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHQ8iKXhLwcrVg90H21EOLACvtfcHk0ljHpMX-goXETg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmsFOku7o%253D&md5=b28998834ae9a05df6bc678d42a5b9af</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1210%2Fjc.2003-030630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fjc.2003-030630%26sid%3Dliteratum%253Aachs%26aulast%3DBatterham%26aufirst%3DR.%2BL.%26aulast%3DLe%2BRoux%26aufirst%3DC.%2BW.%26aulast%3DCohen%26aufirst%3DM.%2BA.%26aulast%3DPark%26aufirst%3DA.%2BJ.%26aulast%3DEllis%26aufirst%3DS.%2BM.%26aulast%3DPatterson%26aufirst%3DM.%26aulast%3DFrost%26aufirst%3DG.%2BS.%26aulast%3DGhatei%26aufirst%3DM.%2BA.%26aulast%3DBloom%26aufirst%3DS.%2BR.%26atitle%3DPancreatic%2520polypeptide%2520reduces%2520appetite%2520and%2520food%2520intake%2520in%2520humans%26jtitle%3DJ.%2520Clin.%2520Endocrinol.%2520Metab.%26date%3D2003%26volume%3D88%26spage%3D3989%26epage%3D3992%26doi%3D10.1210%2Fjc.2003-030630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jesudason, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGowan, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neary, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philippou, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Small, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frost, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghatei, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bloom, S. R.</span></span> <span> </span><span class="NLM_article-title">Low-dose pancreatic polypeptide inhibits food intake in man</span>. <i>Br. J. Nutr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">426</span>– <span class="NLM_lpage">429</span>, <span class="refDoi"> DOI: 10.1017/S0007114507336799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1017%2FS0007114507336799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=17313701" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjvVSitbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2007&pages=426-429&author=D.+R.+Jesudasonauthor=M.+P.+Monteiroauthor=B.+M.+McGowanauthor=N.+M.+Nearyauthor=A.+J.+Parkauthor=E.+Philippouauthor=C.+J.+Smallauthor=G.+S.+Frostauthor=M.+A.+Ghateiauthor=S.+R.+Bloom&title=Low-dose+pancreatic+polypeptide+inhibits+food+intake+in+man&doi=10.1017%2FS0007114507336799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Low-dose pancreatic polypeptide inhibits food intake in man</span></div><div class="casAuthors">Jesudason, David R.; Monteiro, Mariana P.; McGowan, Barbara M. C.; Neary, Nicola M.; Park, Adrian J.; Philippou, Elena; Small, Caroline J.; Frost, Gary S.; Ghatei, Mohammad A.; Bloom, Stephen R.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Nutrition</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">426-429</span>CODEN:
                <span class="NLM_cas:coden">BJNUAV</span>;
        ISSN:<span class="NLM_cas:issn">0007-1145</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">Pancreatic polypeptide (PP) is a gut hormone released from the pancreas in response to food ingestion and remains elevated for up to 6 h post-prandially.  Plasma levels are elevated in patients with pancreatic tumors.  An i.v. infusion of PP has been reported to reduce food intake in man, suggesting that PP is a satiety hormone.  The authors investigated whether a lower infusion rate of PP would induce significant alterations in energy intake.  The study was randomized and double-blinded.  Fourteen lean fasted volunteers (5 men and 9 women) received 90 min infusions of PP (5 pmol/kg/min) and saline on 2 sep. days.  The dose chosen was half that used in a previous human study which reported a decrease in appetite but at supra-physiol. levels of PP.  One hour after the end of the infusion, a buffet lunch was served and energy intake measured.  PP infusion was assocd. with a significant 11% redn. in energy intake compared with saline.  Preprandial hunger as assessed by a visual analog score was decreased in the PP-treated group compared to saline.  These effects were achieved with plasma levels of PP within the pathophysiol. range of pancreatic tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9qZfa4vo6U7Vg90H21EOLACvtfcHk0lhMsiJXgOVz4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjvVSitbg%253D&md5=755b55137fca7b6029ed92466a79bc05</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1017%2FS0007114507336799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS0007114507336799%26sid%3Dliteratum%253Aachs%26aulast%3DJesudason%26aufirst%3DD.%2BR.%26aulast%3DMonteiro%26aufirst%3DM.%2BP.%26aulast%3DMcGowan%26aufirst%3DB.%2BM.%26aulast%3DNeary%26aufirst%3DN.%2BM.%26aulast%3DPark%26aufirst%3DA.%2BJ.%26aulast%3DPhilippou%26aufirst%3DE.%26aulast%3DSmall%26aufirst%3DC.%2BJ.%26aulast%3DFrost%26aufirst%3DG.%2BS.%26aulast%3DGhatei%26aufirst%3DM.%2BA.%26aulast%3DBloom%26aufirst%3DS.%2BR.%26atitle%3DLow-dose%2520pancreatic%2520polypeptide%2520inhibits%2520food%2520intake%2520in%2520man%26jtitle%3DBr.%2520J.%2520Nutr.%26date%3D2007%26volume%3D97%26spage%3D426%26epage%3D429%26doi%3D10.1017%2FS0007114507336799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frank, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hey-Hawkins, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, P.</span></span> <span> </span><span class="NLM_article-title">Manipulating Y receptor subtype activation of short neuropeptide Y analogs by introducing carbaboranes</span>. <i>Neuropeptides</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">59</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1016/j.npep.2012.12.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2Fj.npep.2012.12.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=23352609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFCqsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2013&pages=59-66&author=S.+Hofmannauthor=R.+Frankauthor=E.+Hey-Hawkinsauthor=A.+G.+Beck-Sickingerauthor=P.+Schmidt&title=Manipulating+Y+receptor+subtype+activation+of+short+neuropeptide+Y+analogs+by+introducing+carbaboranes&doi=10.1016%2Fj.npep.2012.12.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Manipulating Y receptor subtype activation of short neuropeptide Y analogs by introducing carbaboranes</span></div><div class="casAuthors">Hofmann, S.; Frank, R.; Hey-Hawkins, E.; Beck-Sickinger, A. G.; Schmidt, P.</div><div class="citationInfo"><span class="NLM_cas:title">Neuropeptides (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">59-66</span>CODEN:
                <span class="NLM_cas:coden">NRPPDD</span>;
        ISSN:<span class="NLM_cas:issn">0143-4179</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Short selective neuropeptide Y (NPY) analogs are highly attractive because of their facile synthesis.  Based on the reduced-size NPY analog [Pro30, Nle31, Bpa32, Leu34]NPY 28-36 position 32 was identified as a key position to alter the preferential activation pattern of the human neuropeptide Y receptors (hYRs).  By replacing benzoylphenylalanine (Bpa) by a biphenylalanine (Bip) the photostability was first improved while the biol. activity was maintained.  SAR-studies showed that both arom. rings have a high influence on the preferential hYR subtype activation.  Interestingly, replacement of Bpa32 by a strongly hydrophobic moiety changed the hYR subtype preference of the analog.  Whereas the parent compd. is able to activate the human neuropeptide Y1 receptor (hY1R) subtype, the introduction of an Nε-ortho-carbaboranyl propionic acid modified lysine resulted in a loss of activity at the hY1R but in an increased activity at both the hY2R and the hY4R.  However, subsequent receptor internalization studies with this novel analog revealed that receptor internalization can neither be triggered at the hY2R nor at the hY4R suggesting a biased ligand.  Surprisingly, investigations by 1H NMR spectroscopy revealed structural changes in the side chains of residues Pro30 and Leu34 which nicely correlates with the shift from hY1R/hY4R to hY2R/hY4R activation preference.  Thus, position 32 has been identified to switch the bioactive conformation and subsequently influences receptor subtype activation behavior.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvP9VxxSzNyrVg90H21EOLACvtfcHk0lhMsiJXgOVz4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFCqsbw%253D&md5=bc298e83582804b6cad8c2000a99b172</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.npep.2012.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.npep.2012.12.001%26sid%3Dliteratum%253Aachs%26aulast%3DHofmann%26aufirst%3DS.%26aulast%3DFrank%26aufirst%3DR.%26aulast%3DHey-Hawkins%26aufirst%3DE.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26aulast%3DSchmidt%26aufirst%3DP.%26atitle%3DManipulating%2520Y%2520receptor%2520subtype%2520activation%2520of%2520short%2520neuropeptide%2520Y%2520analogs%2520by%2520introducing%2520carbaboranes%26jtitle%3DNeuropeptides%26date%3D2013%26volume%3D47%26spage%3D59%26epage%3D66%26doi%3D10.1016%2Fj.npep.2012.12.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parker, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babij, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balasubramaniam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burrier, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamud, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukhopadhyay, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudinski, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tice, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullins, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salisbury, B. G.</span></span> <span> </span><span class="NLM_article-title">GR231118 (1229U91) and other analogues of the C-terminus of neuropeptide Y are potent neuropeptide Y Y1 receptor antagonists and neuropeptide Y Y4 receptor agonists</span>. <i>Eur. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>349</i></span>,  <span class="NLM_fpage">97</span>– <span class="NLM_lpage">105</span>, <span class="refDoi"> DOI: 10.1016/S0014-2999(98)00171-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2FS0014-2999%2898%2900171-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=9669502" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADyaK1cXjsVehtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=349&publication_year=1998&pages=97-105&author=E.+M.+Parkerauthor=C.+K.+Babijauthor=A.+Balasubramaniamauthor=R.+E.+Burrierauthor=M.+Guzziauthor=F.+Hamudauthor=G.+Mukhopadhyayauthor=M.+S.+Rudinskiauthor=Z.+Taoauthor=M.+Ticeauthor=L.+Xiaauthor=D.+E.+Mullinsauthor=B.+G.+Salisbury&title=GR231118+%281229U91%29+and+other+analogues+of+the+C-terminus+of+neuropeptide+Y+are+potent+neuropeptide+Y+Y1+receptor+antagonists+and+neuropeptide+Y+Y4+receptor+agonists&doi=10.1016%2FS0014-2999%2898%2900171-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">GR231118 (1229U91) and other analogs of the C-terminus of neuropeptide Y are potent neuropeptide Y Y1 receptor antagonists and neuropeptide Y Y4 receptor agonists</span></div><div class="casAuthors">Parker, Eric M.; Babij, Carol K.; Balasubramaniam, Ambikaipakan; Burrier, Robert E.; Guzzi, Mario; Hamud, Fozia; Mukhopadhyay, G.; Rudinski, Mark S.; Tao, Z.; Tice, Melissa; Xia, Ling; Mullins, Deborra E.; Salisbury, Brian G.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmacology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">349</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">97-105</span>CODEN:
                <span class="NLM_cas:coden">EJPHAZ</span>;
        ISSN:<span class="NLM_cas:issn">0014-2999</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">GR231118, BW1911U90, Bis(31/31'){[Cys31, Trp32, Nva34] neuropeptide Y(31-36)} (T-190) and [Trp-Arg-Nva-Arg-Tyr]2-NH2 (T-241) are peptide analogs of the C-terminus of neuropeptide Y that have recently been shown to be antagonists of the neuropeptide Y Y1 receptor.  In this study, the activity of these peptides at each of the cloned neuropeptide Y receptor subtypes is detd. in radioligand binding assays and in functional assays (inhibition of forskolin-stimulated cAMP formation).  GR231118 is a potent antagonist at the human and rat neuropeptide Y Y1 receptors (pA2=10.5 and 10.0, resp.; pKi=10.2 and 10.4, resp.), a potent agonist at the human neuropeptide Y Y4 receptor (pEC50=8.6; pKi=9.6) and a weak agonist at the human and rat neuropeptide Y Y2 and Y5 receptors.  GR231118 also has high affinity for the mouse neuropeptide Y Y6 receptor (pKi=8.8).  Therefore, GR231118 is a relatively selective neuropeptide Y Y1 receptor antagonist, but has appreciable activity at the neuropeptide Y Y4 and Y6 receptors as well.  BW1911U90, T-190 and T-241 are moderately potent neuropeptide Y Y1 receptor antagonists (pA2=7.1, 5.8 and 6.5, resp.; pKi=8.3, 6.5 and 6.8, resp.) and neuropeptide Y Y4 receptor agonists (pEC50=6.8, 6.3 and 6.6, resp.; pKi;8.3, 7.7 and 8.3, resp.).  These data suggest that the C-terminus of neuropeptide Y and related peptides is sufficient for activation of the neuropeptide Y Y4 receptor, but is not sufficient for activation of the neuropeptide Y Y1 receptor.  Because BW1911U90, T-190 and T-241 are significantly less potent at the cloned human neuropeptide Y Y1 receptor than at the neuropeptide Y receptor in human erythroleukemia cells, these cells may express a novel neuropeptide Y receptor with high affinity for these peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnzH4siycw6rVg90H21EOLACvtfcHk0lhIpjyg1rYVBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjsVehtL8%253D&md5=03c10aafff350e9f8862b16b27f27557</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2FS0014-2999%2898%2900171-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-2999%252898%252900171-X%26sid%3Dliteratum%253Aachs%26aulast%3DParker%26aufirst%3DE.%2BM.%26aulast%3DBabij%26aufirst%3DC.%2BK.%26aulast%3DBalasubramaniam%26aufirst%3DA.%26aulast%3DBurrier%26aufirst%3DR.%2BE.%26aulast%3DGuzzi%26aufirst%3DM.%26aulast%3DHamud%26aufirst%3DF.%26aulast%3DMukhopadhyay%26aufirst%3DG.%26aulast%3DRudinski%26aufirst%3DM.%2BS.%26aulast%3DTao%26aufirst%3DZ.%26aulast%3DTice%26aufirst%3DM.%26aulast%3DXia%26aufirst%3DL.%26aulast%3DMullins%26aufirst%3DD.%2BE.%26aulast%3DSalisbury%26aufirst%3DB.%2BG.%26atitle%3DGR231118%2520%25281229U91%2529%2520and%2520other%2520analogues%2520of%2520the%2520C-terminus%2520of%2520neuropeptide%2520Y%2520are%2520potent%2520neuropeptide%2520Y%2520Y1%2520receptor%2520antagonists%2520and%2520neuropeptide%2520Y%2520Y4%2520receptor%2520agonists%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D1998%26volume%3D349%26spage%3D97%26epage%3D105%26doi%3D10.1016%2FS0014-2999%2898%2900171-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Balasubramaniam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullins, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhawan, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knittel, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slack, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herzog, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, E. M.</span></span> <span> </span><span class="NLM_article-title">Neuropeptide Y (NPY) Y4 receptor selective agonists based on NPY(32-36): development of an anorectic Y4 receptor selective agonist with picomolar affinity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">2661</span>– <span class="NLM_lpage">2665</span>, <span class="refDoi"> DOI: 10.1021/jm050907d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050907d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BD28XivFWlsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=2661-2665&author=A.+Balasubramaniamauthor=D.+E.+Mullinsauthor=S.+Linauthor=W.+Zhaiauthor=Z.+Taoauthor=V.+C.+Dhawanauthor=M.+Guzziauthor=J.+J.+Knittelauthor=K.+Slackauthor=H.+Herzogauthor=E.+M.+Parker&title=Neuropeptide+Y+%28NPY%29+Y4+receptor+selective+agonists+based+on+NPY%2832-36%29%3A+development+of+an+anorectic+Y4+receptor+selective+agonist+with+picomolar+affinity&doi=10.1021%2Fjm050907d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Neuropeptide Y (NPY) Y4 Receptor Selective Agonists Based on NPY(32-36): Development of an Anorectic Y4 Receptor Selective Agonist with Picomolar Affinity</span></div><div class="casAuthors">Balasubramaniam, Ambikaipakan; Mullins, Deborra E.; Lin, Shu; Zhai, Wexiu; Tao, Zhiyong; Dhawan, Vikas C.; Guzzi, Mario; Knittel, James J.; Slack, Katy; Herzog, Herbert; Parker, Eric M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2661-2665</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have previously shown [Cys-Trp-Arg-Nva-Arg-Tyr-NH2]2, 1, to be a moderately selective neuropeptide Y (NPY) Y4 receptor agonist.  Toward improving the selectivity and potency for Y4 receptors, we studied the effects of dimerizing H-Trp-Arg-Nva-Arg-Tyr-NH2 using various diamino-dicarboxylic acids contg. either di-, tri-, or tetramethylene spacers.  These parallel dimers, 2A, 2B, 3, 4A, and 4B, and the corresponding linear tandem dimer and trimer analogs, 5 and 6, had enhanced selectivity and affinity for Y4 receptors compared to 1 (Table 1).  Substitution of Trp and Nva with Tyr and Leu, resp., as in 2,7-D/L-diaminosuberic acid derivatized dimer, 7, resulted in a superior Y4 selective agonist with picomolar affinity.  I.P. injection of 7 potently inhibited food intake in fasted mice.  Moreover, 7 (i.p.) inhibited the food intake in wild-type mice and not in Y4-/- knock-out mice, confirming that the actions of 7 on food intake are not due to global effects, but specifically mediated Y4 receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNSefoXjlitLVg90H21EOLACvtfcHk0liiP-G4sgkjeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XivFWlsL0%253D&md5=cfa39f849b60d3e8b56118c2f7b3396a</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm050907d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050907d%26sid%3Dliteratum%253Aachs%26aulast%3DBalasubramaniam%26aufirst%3DA.%26aulast%3DMullins%26aufirst%3DD.%2BE.%26aulast%3DLin%26aufirst%3DS.%26aulast%3DZhai%26aufirst%3DW.%26aulast%3DTao%26aufirst%3DZ.%26aulast%3DDhawan%26aufirst%3DV.%2BC.%26aulast%3DGuzzi%26aufirst%3DM.%26aulast%3DKnittel%26aufirst%3DJ.%2BJ.%26aulast%3DSlack%26aufirst%3DK.%26aulast%3DHerzog%26aufirst%3DH.%26aulast%3DParker%26aufirst%3DE.%2BM.%26atitle%3DNeuropeptide%2520Y%2520%2528NPY%2529%2520Y4%2520receptor%2520selective%2520agonists%2520based%2520on%2520NPY%252832-36%2529%253A%2520development%2520of%2520an%2520anorectic%2520Y4%2520receptor%2520selective%2520agonist%2520with%2520picomolar%2520affinity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D2661%26epage%3D2665%26doi%3D10.1021%2Fjm050907d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ertl, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dukorn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiser, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buschauer, A.</span></span> <span> </span><span class="NLM_article-title">High affinity agonists of the neuropeptide Y (NPY) Y4 receptor derived from the C-terminal pentapeptide of human pancreatic polypeptide (hPP): synthesis, stereochemical discrimination, and radiolabeling</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">6045</span>– <span class="NLM_lpage">6058</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00309</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00309" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC28XosFSmtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=6045-6058&author=K.+K.+Kuhnauthor=T.+Ertlauthor=S.+Dukornauthor=M.+Kellerauthor=G.+Bernhardtauthor=O.+Reiserauthor=A.+Buschauer&title=High+affinity+agonists+of+the+neuropeptide+Y+%28NPY%29+Y4+receptor+derived+from+the+C-terminal+pentapeptide+of+human+pancreatic+polypeptide+%28hPP%29%3A+synthesis%2C+stereochemical+discrimination%2C+and+radiolabeling&doi=10.1021%2Facs.jmedchem.6b00309"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">High Affinity Agonists of the Neuropeptide Y (NPY) Y4 Receptor Derived from the C-Terminal Pentapeptide of Human Pancreatic Polypeptide (hPP): Synthesis, Stereochemical Discrimination, and Radiolabeling</span></div><div class="casAuthors">Kuhn, Kilian K.; Ertl, Thomas; Dukorn, Stefanie; Keller, Max; Bernhardt, Guenther; Reiser, Oliver; Buschauer, Armin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6045-6058</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The diastereomeric mixt. of D/L-2,7-diaminooctanedioyl-bis(YRLRY-NH2) (BVD-74D, 2) was described in the literature as a high affinity Y4 receptor agonist.  Here we report on the synthesis and pharmacol. characterization of the pure diastereomers (2R,7R)- and (2S,7S)-2 and a series of homo- and heterodimeric analogs in which octanedioic acid was used as an achiral linker.  To investigate the role of the Arg residues, one or two arginines were replaced by Ala.  Moreover, Nω-(6-aminohexylaminocarbonyl)Arg was introduced as an arginine replacement (17). (2R,7R)-2 was superior to (2S,7S)-2 in binding and functional cellular assays and equipotent with 17. [3H]Propionylation of one amino group in the linker of (2R,7R)-2 or at the primary amino group in 17 resulted in high affinity Y4R radioligands ([3H]-(2R,7R)-10, [3H]18) with subnanomolar Kd values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa_pLfg6qLerVg90H21EOLACvtfcHk0ljcchf2o1xEeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosFSmtrc%253D&md5=ee7279c046d467c5ec6919187eb0f3aa</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00309%26sid%3Dliteratum%253Aachs%26aulast%3DKuhn%26aufirst%3DK.%2BK.%26aulast%3DErtl%26aufirst%3DT.%26aulast%3DDukorn%26aufirst%3DS.%26aulast%3DKeller%26aufirst%3DM.%26aulast%3DBernhardt%26aufirst%3DG.%26aulast%3DReiser%26aufirst%3DO.%26aulast%3DBuschauer%26aufirst%3DA.%26atitle%3DHigh%2520affinity%2520agonists%2520of%2520the%2520neuropeptide%2520Y%2520%2528NPY%2529%2520Y4%2520receptor%2520derived%2520from%2520the%2520C-terminal%2520pentapeptide%2520of%2520human%2520pancreatic%2520polypeptide%2520%2528hPP%2529%253A%2520synthesis%252C%2520stereochemical%2520discrimination%252C%2520and%2520radiolabeling%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D6045%26epage%3D6058%26doi%3D10.1021%2Facs.jmedchem.6b00309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konieczny, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wifling, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, M.</span></span> <span> </span><span class="NLM_article-title">Oligopeptides as neuropeptide Y Y4 receptor ligands: Identification of a high-affinity tetrapeptide agonist and a hexapeptide antagonist</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">8198</span>– <span class="NLM_lpage">8215</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00426</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00426" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlSju7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=8198-8215&author=A.+Koniecznyauthor=D.+Braunauthor=D.+Wiflingauthor=G.+Bernhardtauthor=M.+Keller&title=Oligopeptides+as+neuropeptide+Y+Y4+receptor+ligands%3A+Identification+of+a+high-affinity+tetrapeptide+agonist+and+a+hexapeptide+antagonist&doi=10.1021%2Facs.jmedchem.0c00426"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Oligopeptides as Neuropeptide Y Y4 Receptor Ligands: Identification of a High-Affinity Tetrapeptide Agonist and a Hexapeptide Antagonist</span></div><div class="casAuthors">Konieczny, Adam; Braun, Diana; Wifling, David; Bernhardt, Guenther; Keller, Max</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8198-8215</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Within the family of neuropeptide Y (NPY) receptors, the Y4 receptor (Y4R) is unique as it prefers pancreatic polypeptide (PP) over NPY and peptide YY (PYY).  Today, low mol. wt. Y4R ligands are lacking, in particular antagonists.  We synthesized a series of peptidic NPY Y4R ligands, derived from the hexapeptide acetyl-Arg-Tyr-Arg-Leu-Arg-Tyr-NH2 (1), reported to be a Y4R partial agonist with high affinity (pKi Y4R: 8.43).  Peptide 1 was N-terminally extended as well as truncated, subjected to a D-amino acid scan, and Leu was replaced by different amino acids.  Compds. were characterized by radioligand competition binding and functional studies (Cai2+ mobilization and β-arrestin 1/2 recruitment).  N-terminal truncation of 1 resulted in a tetrapeptide (Arg-Leu-Arg-Tyr-NH2) being a Y4R partial agonist with retained Y4R affinity (pKi: 8.47).  Remarkably, replacement of Leu in 1 and in derivs. of 1 by Trp turned Y4R agonism to antagonism, giving Y4R antagonists with pKi values ≤ 7.57.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGox8hX-Vehx4rVg90H21EOLACvtfcHk0ljcchf2o1xEeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlSju7jM&md5=326a1d05671d3b43056f14f637fee2c3</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00426&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00426%26sid%3Dliteratum%253Aachs%26aulast%3DKonieczny%26aufirst%3DA.%26aulast%3DBraun%26aufirst%3DD.%26aulast%3DWifling%26aufirst%3DD.%26aulast%3DBernhardt%26aufirst%3DG.%26aulast%3DKeller%26aufirst%3DM.%26atitle%3DOligopeptides%2520as%2520neuropeptide%2520Y%2520Y4%2520receptor%2520ligands%253A%2520Identification%2520of%2520a%2520high-affinity%2520tetrapeptide%2520agonist%2520and%2520a%2520hexapeptide%2520antagonist%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D8198%26epage%3D8215%26doi%3D10.1021%2Facs.jmedchem.0c00426" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sliwoski, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schubert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stichel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weaver, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meiler, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of small-molecule modulators of the human Y4 receptor</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">e0157146</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0157146</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1371%2Fjournal.pone.0157146" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=27294784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFyqtr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&author=G.+Sliwoskiauthor=M.+Schubertauthor=J.+Stichelauthor=D.+Weaverauthor=A.+G.+Beck-Sickingerauthor=J.+Meiler&title=Discovery+of+small-molecule+modulators+of+the+human+Y4+receptor&doi=10.1371%2Fjournal.pone.0157146"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of small-molecule modulators of the human Y4 Receptor</span></div><div class="casAuthors">Sliwoski, Gregory; Schubert, Mario; Stichel, Jan; Weaver, David; Beck-Sickinger, Annette G.; Meiler, Jens</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e0157146/1-e0157146/13</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The human neuropeptide Y4 receptor (Y4R) and its native ligand, pancreatic polypeptide, are critically involved in the regulation of human metab. by signaling satiety and regulating food intake, as well as increasing energy expenditure.  Thus, this receptor represents a putative target for treatment of obesity.  With respect to new approaches to treat complex metabolic disorders, esp. in multi-receptor systems, small mol. allosteric modulators have been in the focus of research in the last years.  However, no pos. allosteric modulators or agonists of the Y4R have been described so far.  In this study, small mol. compds. derived from the Niclosamide scaffold were identified by high-throughput screening to increase Y4R activity.  Compds. were characterized for their potency and their effects at the human Y4R and as well as their selectivity towards Y1R, Y2R and Y5R.  These compds. provide a structure-activity relationship profile around this common scaffold and lay the groundwork for hit-to-lead optimization and characterization of pos. allosteric modulators of the Y4R.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAbv1m3XAlzLVg90H21EOLACvtfcHk0lgnxUjlSc5lCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFyqtr3L&md5=ed0f23077965cf43815202ad89127949</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0157146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0157146%26sid%3Dliteratum%253Aachs%26aulast%3DSliwoski%26aufirst%3DG.%26aulast%3DSchubert%26aufirst%3DM.%26aulast%3DStichel%26aufirst%3DJ.%26aulast%3DWeaver%26aufirst%3DD.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26aulast%3DMeiler%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520small-molecule%2520modulators%2520of%2520the%2520human%2520Y4%2520receptor%26jtitle%3DPLoS%2520One%26date%3D2016%26volume%3D11%26doi%3D10.1371%2Fjournal.pone.0157146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schubert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stichel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sliwoski, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meiler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weaver, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span> <span> </span><span class="NLM_article-title">Identification and characterization of the first selective Y4 receptor positive allosteric modulator</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">7605</span>– <span class="NLM_lpage">7612</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00976</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00976" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlSmtLrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=7605-7612&author=M.+Schubertauthor=J.+Stichelauthor=Y.+Duauthor=I.+R.+Toughauthor=G.+Sliwoskiauthor=J.+Meilerauthor=H.+M.+Coxauthor=C.+D.+Weaverauthor=A.+G.+Beck-Sickinger&title=Identification+and+characterization+of+the+first+selective+Y4+receptor+positive+allosteric+modulator&doi=10.1021%2Facs.jmedchem.7b00976"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Characterization of the First Selective Y4 Receptor Positive Allosteric Modulator</span></div><div class="casAuthors">Schubert, Mario; Stichel, Jan; Du, Yu; Tough, Iain R.; Sliwoski, Gregory; Meiler, Jens; Cox, Helen M.; Weaver, C. David; Beck-Sickinger, Annette G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7605-7612</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The human Y4 receptor (Y4R) and its cognate ligand, pancreatic polypeptide (PP), are involved in the regulation of energy exposure, satiety and food intake.  This system represents a potential target for the treatment of metabolic diseases and has been extensively investigated and validated in vivo.  Here, we present the compd. tBPC (2-(4-tert-Butyl)phenoxycyclohexanol), a novel and selective Y4R pos. allosteric modulator that potentiates Y4R activation in G-protein signaling and arrestin 3 recruitment expts.  The compd. has no effect on the binding of the orthosteric ligands, implying its allosteric mode of action at the Y4R and evidence for a purely efficacy-driven pos. allosteric modulation.  Finally, the ability of tBPC to selectively potentiate Y4R agonism initiated by PP was confirmed in mouse descending colon mucosa prepns. expressing native Y4R, demonstrating Y4R pos. allosteric modulation in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFvYNHioUE8rVg90H21EOLACvtfcHk0ljNE8KvNwynqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlSmtLrL&md5=99b416da5721005fb30d8acff3110f23</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00976&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00976%26sid%3Dliteratum%253Aachs%26aulast%3DSchubert%26aufirst%3DM.%26aulast%3DStichel%26aufirst%3DJ.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DTough%26aufirst%3DI.%2BR.%26aulast%3DSliwoski%26aufirst%3DG.%26aulast%3DMeiler%26aufirst%3DJ.%26aulast%3DCox%26aufirst%3DH.%2BM.%26aulast%3DWeaver%26aufirst%3DC.%2BD.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520the%2520first%2520selective%2520Y4%2520receptor%2520positive%2520allosteric%2520modulator%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D7605%26epage%3D7612%26doi%3D10.1021%2Facs.jmedchem.7b00976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaske, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holzammer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buschauer, A.</span></span> <span> </span><span class="NLM_article-title">Dimeric argininamide-type neuropeptide Y receptor antagonists: chiral discrimination between Y1 and Y4 receptors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">6303</span>– <span class="NLM_lpage">6322</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.08.065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2Fj.bmc.2013.08.065" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=24074877" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFemtLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=6303-6322&author=M.+Kellerauthor=M.+Kaskeauthor=T.+Holzammerauthor=G.+Bernhardtauthor=A.+Buschauer&title=Dimeric+argininamide-type+neuropeptide+Y+receptor+antagonists%3A+chiral+discrimination+between+Y1+and+Y4+receptors&doi=10.1016%2Fj.bmc.2013.08.065"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Dimeric argininamide-type neuropeptide Y receptor antagonists: Chiral discrimination between Y1 and Y4 receptors</span></div><div class="casAuthors">Keller, Max; Kaske, Melanie; Holzammer, Tobias; Bernhardt, Guenther; Buschauer, Armin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6303-6322</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The structurally related peptides neuropeptide Y (NPY), peptide YY (PYY) and pancreatic polypeptide (PP) are endogenous agonists of the NPY receptor (YR) family, which in humans comprises four functionally expressed subtypes, designated Y1R, Y2R, Y4R and Y5R.  Nonpeptide antagonists with high affinity and selectivity have been described for the Y1R, Y2R and Y5R, but such compds. are still lacking for the Y4R.  In this work, the structures of the high affinity selective (R)-argininamide-type Y1R antagonists BIBP3226 and BIBO3304 were linked via the guanidine or urea moieties to give homo-dimeric argininamides with linker lengths ranging from 31 to 41 atoms.  Interestingly, the twin compds. proved to be by far less selective for the Y1R than the R-configured monovalent parent compds.  The decrease in selectivity ratio was most pronounced for Y1R vs. Y4R subtype, resulting in comparable affinities of bivalent ligands for Y1R and Y4R (e.g. UR-MK177 [(R,R)-I (R1 = (CH2O)2CH2CH2X, Y = CH2NHC(O)NH2)]: Ki = 230 nM (Y1R) and 290 nM (Y4R)).  With a Ki value of 130 nM and a Kb value of 20 nM, UR-MK188 [(R,R)-II (R2 = CH2(CH2OCH2)2CH2X, Y = OH)] was superior to all Y4R antagonists known to date.  The S,S-configured optical antipodes of UR-MK177 and UR-MK188 (UR-MEK381 ((S,S)-I) and UR-MEK388 ((S,S)-II)) were synthesized to investigate the stereochem. discrimination by the different receptor subtypes.  Whereas preference for R,R-configured argininamides was characteristic of the Y1R, stereochem. discrimination by the Y4R was not obsd.  This may pave the way to selective Y4R antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRAhCQ0Gt1nLVg90H21EOLACvtfcHk0lg89TKdiXuDyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFemtLzL&md5=5ca4d29cdb12eb2c4ea26396b1c569df</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.08.065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.08.065%26sid%3Dliteratum%253Aachs%26aulast%3DKeller%26aufirst%3DM.%26aulast%3DKaske%26aufirst%3DM.%26aulast%3DHolzammer%26aufirst%3DT.%26aulast%3DBernhardt%26aufirst%3DG.%26aulast%3DBuschauer%26aufirst%3DA.%26atitle%3DDimeric%2520argininamide-type%2520neuropeptide%2520Y%2520receptor%2520antagonists%253A%2520chiral%2520discrimination%2520between%2520Y1%2520and%2520Y4%2520receptors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D6303%26epage%3D6322%26doi%3D10.1016%2Fj.bmc.2013.08.065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Littmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dukorn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buschauer, A.</span></span> <span> </span><span class="NLM_article-title">In Search of NPY Y4R antagonists: Incorporation of carbamoylated arginine, aza-amino acids, or d-amino acids into oligopeptides derived from the C-termini of the endogenous agonists</span>. <i>ACS Omega</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">3616</span>– <span class="NLM_lpage">3631</span>, <span class="refDoi"> DOI: 10.1021/acsomega.7b00451</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.7b00451" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFKhu7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2017&pages=3616-3631&author=K.+K.+Kuhnauthor=T.+Littmannauthor=S.+Dukornauthor=M.+Tanakaauthor=M.+Kellerauthor=T.+Ozawaauthor=G.+Bernhardtauthor=A.+Buschauer&title=In+Search+of+NPY+Y4R+antagonists%3A+Incorporation+of+carbamoylated+arginine%2C+aza-amino+acids%2C+or+d-amino+acids+into+oligopeptides+derived+from+the+C-termini+of+the+endogenous+agonists&doi=10.1021%2Facsomega.7b00451"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">In Search of NPY Y4R Antagonists: Incorporation of Carbamoylated Arginine, Aza-Amino Acids, or D-Amino Acids into Oligopeptides Derived from the C-Termini of the Endogenous Agonists</span></div><div class="casAuthors">Kuhn, Kilian K.; Littmann, Timo; Dukorn, Stefanie; Tanaka, Miho; Keller, Max; Ozawa, Takeaki; Bernhardt, Guenther; Buschauer, Armin</div><div class="citationInfo"><span class="NLM_cas:title">ACS Omega</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3616-3631</span>CODEN:
                <span class="NLM_cas:coden">ACSODF</span>;
        ISSN:<span class="NLM_cas:issn">2470-1343</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The crosslinked pentapeptides (2R,7R)-diaminooctanedioyl-bis(Tyr-Arg-Leu-Arg-Tyr-amide) ((2R,7R)-BVD-74D, (2R,7R)-1) and octanedioyl-bis(Tyr-Arg-Leu-Arg-Tyr-amide) (2) as well as the pentapeptide Ac-Tyr-Arg-Leu-Arg-Tyr-amide (3) were previously described as neuropeptide Y Y4 receptor (Y4R) partial agonists.  Here the authors report on a series of analogs of (2R,7R)-1 and 2 in which Arg2, Leu3 or Arg4 were replaced by the resp. aza-amino acids.  The replacement of Arg2 in 3 with a carbamoylated arginine building block and the extension of the N-terminus by an addnl. arginine led to the high-affinity hexapeptide Ac-Arg-Tyr-Nω-[(4-aminobutyl)aminocarbonyl]Arg-Leu-Arg-Tyr-amide that was used as a precursor for a D-amino acid scan.  The target compds. were investigated for Y4R functional activity in assays with complementary readouts: aequorin Ca2+ and β-arrestin 1 or β-arrestin 2 assays.  In contrast to the parent compds., which are Y4R agonists, several ligands were able to suppress the effect elicited by the endogenous ligand pancreatic polypeptide and therefore represent a novel class of peptide Y4R antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAufjWqDJZnrVg90H21EOLACvtfcHk0lg89TKdiXuDyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFKhu7bE&md5=970422192d233703cda161e6e476b3e5</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facsomega.7b00451&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.7b00451%26sid%3Dliteratum%253Aachs%26aulast%3DKuhn%26aufirst%3DK.%2BK.%26aulast%3DLittmann%26aufirst%3DT.%26aulast%3DDukorn%26aufirst%3DS.%26aulast%3DTanaka%26aufirst%3DM.%26aulast%3DKeller%26aufirst%3DM.%26aulast%3DOzawa%26aufirst%3DT.%26aulast%3DBernhardt%26aufirst%3DG.%26aulast%3DBuschauer%26aufirst%3DA.%26atitle%3DIn%2520Search%2520of%2520NPY%2520Y4R%2520antagonists%253A%2520Incorporation%2520of%2520carbamoylated%2520arginine%252C%2520aza-amino%2520acids%252C%2520or%2520d-amino%2520acids%2520into%2520oligopeptides%2520derived%2520from%2520the%2520C-termini%2520of%2520the%2520endogenous%2520agonists%26jtitle%3DACS%2520Omega%26date%3D2017%26volume%3D2%26spage%3D3616%26epage%3D3631%26doi%3D10.1021%2Facsomega.7b00451" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hirose, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, A.</span></span> <span> </span><span class="NLM_article-title">Fluorescence properties of 5-(5,6-dimethoxybenzothiazol-2-yl)-2-deoxyuridine (dbtU) and oligodeoxyribonucleotides containing dbtU</span>. <i>Eur. J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_fpage">6206</span>– <span class="NLM_lpage">6217</span>, <span class="refDoi"> DOI: 10.1002/ejoc.201100818</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1002%2Fejoc.201100818" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtV2gsbvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&pages=6206-6217&author=W.+Hiroseauthor=K.+Satoauthor=A.+Matsuda&title=Fluorescence+properties+of+5-%285%2C6-dimethoxybenzothiazol-2-yl%29-2-deoxyuridine+%28dbtU%29+and+oligodeoxyribonucleotides+containing+dbtU&doi=10.1002%2Fejoc.201100818"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence Properties of 5-(5,6-Dimethoxybenzothiazol-2-yl)-2'-deoxyuridine (dbtU) and Oligodeoxyribonucleotides Containing dbtU</span></div><div class="casAuthors">Hirose, Wataru; Sato, Kousuke; Matsuda, Akira</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2011</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">6206-6217, S6206/1-S6206/43</span>CODEN:
                <span class="NLM_cas:coden">EJOCFK</span>;
        ISSN:<span class="NLM_cas:issn">1099-0690</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">We describe the synthesis and photo-phys. properties of 11 substituted 5-(benzothiazol-2-yl)-2'-deoxyuridine derivs. and oligodeoxyribonucleotides (ODNs) contg. 5-(5,6-dimethoxybenzothiazol-2-yl)-2'-deoxyuridine (dbtU), which was the strongest fluorescent deriv. among those prepd.  The fluorescence properties of dbtU itself and ODNs contg. dbtU show the same tendency of being weaker in both neutral and acidic soln. and stronger in basic soln.  The ODNs (15mer) contg. 16 combinations of 5'-XbtU-3' and 5'-btUY-3', where X, Y = A, T, G, or C, were synthesized, and their fluorescence intensity and quantum yield in basic soln. were compared.  On av., only the ODN with the 5'-GbtU-3' sequence shows a 7.9-fold lower fluorescence intensity than the other sequences.  Ab initio calcns. of 5'-GbtU-3' and 5'-btUG-3' as models under basic conditions suggest that the lower fluorescence of the ODN contg. the 5'-GbtU-3' sequence is caused by a wider overlap between stacked guanine (Gua) and btUra than that of the 5'-btUG-3' sequence and that the HOMO is delocalized not only on btUra but also on Gua.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6KEFkAz7xELVg90H21EOLACvtfcHk0liNpjnR0O0JGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtV2gsbvE&md5=fb8079bf06d7c4372e19ad6f344b0ea6</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fejoc.201100818&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.201100818%26sid%3Dliteratum%253Aachs%26aulast%3DHirose%26aufirst%3DW.%26aulast%3DSato%26aufirst%3DK.%26aulast%3DMatsuda%26aufirst%3DA.%26atitle%3DFluorescence%2520properties%2520of%25205-%25285%252C6-dimethoxybenzothiazol-2-yl%2529-2-deoxyuridine%2520%2528dbtU%2529%2520and%2520oligodeoxyribonucleotides%2520containing%2520dbtU%26jtitle%3DEur.%2520J.%2520Org.%2520Chem.%26date%3D2011%26spage%3D6206%26epage%3D6217%26doi%3D10.1002%2Fejoc.201100818" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leach, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span> <span> </span><span class="NLM_article-title">Allosteric GPCR modulators: taking advantage of permissive receptor pharmacology</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">382</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2007.06.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2Fj.tips.2007.06.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=17629965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotlWltLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2007&pages=382-389&author=K.+Leachauthor=P.+M.+Sextonauthor=A.+Christopoulos&title=Allosteric+GPCR+modulators%3A+taking+advantage+of+permissive+receptor+pharmacology&doi=10.1016%2Fj.tips.2007.06.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric GPCR modulators: taking advantage of permissive receptor pharmacology</span></div><div class="casAuthors">Leach, Katie; Sexton, Patrick M.; Christopoulos, Arthur</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">382-389</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The past decade has witnessed a dramatic increase in the identification of allosteric modulators of G-protein-coupled receptor (GPCR) activity.  Concomitantly, several new perspectives and hypotheses regarding the way ligands regulate GPCR signaling have also emerged.  Here, we briefly discuss how the concepts of collateral efficacy and permissive agonism-antagonism intersect the field of allosteric GPCR modulation.  Despite the challenges assocd. with detecting and quantifying the myriad of possible allosteric effects on GPCR activity, it is proposed that allosteric ligands offer the exciting prospect of engendering stimulus-bias in orthosteric ligand signaling, thus paving the way for not only receptor-selective but also signaling-pathway-selective therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpHb_QsIW3TbVg90H21EOLACvtfcHk0ljcC21sr3O7bQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotlWltLs%253D&md5=943da2312e7b136cb594ba0346e00301</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2007.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2007.06.004%26sid%3Dliteratum%253Aachs%26aulast%3DLeach%26aufirst%3DK.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DChristopoulos%26aufirst%3DA.%26atitle%3DAllosteric%2520GPCR%2520modulators%253A%2520taking%2520advantage%2520of%2520permissive%2520receptor%2520pharmacology%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2007%26volume%3D28%26spage%3D382%26epage%3D389%26doi%3D10.1016%2Fj.tips.2007.06.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bdioui, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierre, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trinquet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roux, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenakin, T.</span></span> <span> </span><span class="NLM_article-title">Equilibrium assays are required to accurately characterize the activity profiles of drugs modulating Gq-protein-coupled receptors</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">992</span>– <span class="NLM_lpage">1006</span>, <span class="refDoi"> DOI: 10.1124/mol.118.112573</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1124%2Fmol.118.112573" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=29954837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVGgtrvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2018&pages=992-1006&author=S.+Bdiouiauthor=J.+Verdiauthor=N.+Pierreauthor=E.+Trinquetauthor=T.+Rouxauthor=T.+Kenakin&title=Equilibrium+assays+are+required+to+accurately+characterize+the+activity+profiles+of+drugs+modulating+Gq-protein-coupled+receptors&doi=10.1124%2Fmol.118.112573"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Equilibrium assays are required to accurately characterize the activity profiles of drugs modulating Gq-protein-coupled receptors</span></div><div class="casAuthors">Bdioui, Sara; Verdi, Julien; Pierre, Nicolas; Trinquet, Eric; Roux, Thomas; Kenakin, Terry</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">992-1006</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">This paper discusses the process of detg. the activity of candidate mols. targeting Gq-protein activation through G-protein-coupled receptors for possible therapeutic application with two functional assays; calcium release and inositol phosphate metab. [inositol monophosphate (IP1)].  While both are suitable for detecting ligand activity (screening), differences are seen when these assays are used to quant. measure ligand parameters for therapeutic activity.  Specifically, responses for Gq-related pathways present different and dissimulating patterns depending on the functional assay used to assess them.  To investigate the impact of functional assays on the accuracy of compd. pharmacol. profiles, five exemplar mols. [partial agonist, antagonist, inverse agonist, pos. allosteric modulator (PAM) agonist, and pos. beta-PAM] targeting either muscarinic M1 or ghrelin receptors were tested using two functional assays (calcium release and IP1) and the results were compared with theor. pharmacol. models.  The IP1 assay is an equil. assay that is able to det. the correct pharmacol. profiles of all tested compds.  In contrast, the nonequil. nature of calcium assays yields misleading classification of most of the tested compds.  Our study suggests that the use of an equil. assay, such as IP1, is mandatory for the optimal use of pharmacol. models that can both identify mechanisms of action and also convert descriptive-to-predictive data for therapeutic systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1UalfauOGHbVg90H21EOLACvtfcHk0ljcC21sr3O7bQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVGgtrvN&md5=5edfd3e3f251b396820a1cb8f03e2aaa</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1124%2Fmol.118.112573&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.118.112573%26sid%3Dliteratum%253Aachs%26aulast%3DBdioui%26aufirst%3DS.%26aulast%3DVerdi%26aufirst%3DJ.%26aulast%3DPierre%26aufirst%3DN.%26aulast%3DTrinquet%26aufirst%3DE.%26aulast%3DRoux%26aufirst%3DT.%26aulast%3DKenakin%26aufirst%3DT.%26atitle%3DEquilibrium%2520assays%2520are%2520required%2520to%2520accurately%2520characterize%2520the%2520activity%2520profiles%2520of%2520drugs%2520modulating%2520Gq-protein-coupled%2520receptors%26jtitle%3DMol.%2520Pharmacol.%26date%3D2018%26volume%3D94%26spage%3D992%26epage%3D1006%26doi%3D10.1124%2Fmol.118.112573" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenakin, T.</span></span> <span> </span><span class="NLM_article-title">G protein-coupled receptor allosterism and complexing</span>. <i>Pharmacol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">323</span>– <span class="NLM_lpage">374</span>, <span class="refDoi"> DOI: 10.1124/pr.54.2.323</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1124%2Fpr.54.2.323" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=12037145" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BD38XltVOhsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2002&pages=323-374&author=A.+Christopoulosauthor=T.+Kenakin&title=G+protein-coupled+receptor+allosterism+and+complexing&doi=10.1124%2Fpr.54.2.323"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">G protein-coupled receptor allosterism and complexing</span></div><div class="casAuthors">Christopoulos, Arthur; Kenakin, Terry</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">323-374</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">0031-6997</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  G protein-coupled receptors (GPCRs) represent the largest family of cell-surface receptors.  These receptors are natural allosteric proteins because agonist-mediated signaling by GPCRs requires a conformational change in the receptor protein transmitted between two topog. distinct binding sites, one for the agonist and another for the G protein.  It is now becoming increasingly recognized, however, that the agonist-bound GPCR can also form ternary complexes with other ligands or "accessory" proteins and display altered binding and/or signaling properties in relation to the binary agonist-receptor complex.  Allosteric sites on GPCRs represent novel drug targets because allosteric modulators possess a no. of theor. advantages over classic orthosteric ligands, such as a ceiling level to the allosteric effect and a potential for greater GPCR subtype-selectivity.  Because of the noncompetitive nature of allosteric phenomena, the detection and quantification of such effects often relies on a combination of equil. binding, nonequil. kinetic, and functional signaling assays.  This review discusses the development and properties of allosteric receptor models for GPCRs and the detection and quantification of allosteric effects.  Moreover, we provide an overview of the current knowledge regarding,the location of possible allosteric sites on GPCRs and candidate endogenous allosteric modulators.  Finally, we discuss the potential for allosteric effects arising from the formation of GPCR oligomers or GPCRs complexed with accessory cellular proteins.  It is proposed that the study of allosteric phenomena will become of progressively, greater import to the drug discovery process due to the advent of newer and more sensitive GPCR screening-technologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnSA5pRRWTILVg90H21EOLACvtfcHk0lggcPGdFAWTRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltVOhsrs%253D&md5=6c4895bf66778eda4778ac409c7abb70</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1124%2Fpr.54.2.323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.54.2.323%26sid%3Dliteratum%253Aachs%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DKenakin%26aufirst%3DT.%26atitle%3DG%2520protein-coupled%2520receptor%2520allosterism%2520and%2520complexing%26jtitle%3DPharmacol.%2520Rev.%26date%3D2002%26volume%3D54%26spage%3D323%26epage%3D374%26doi%3D10.1124%2Fpr.54.2.323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wanka, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babilon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mörl, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurevich, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span> <span> </span><span class="NLM_article-title">C-terminal motif of human neuropeptide Y4 receptor determines internalization and arrestin recruitment</span>. <i>Cell. Signalling</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">233</span>– <span class="NLM_lpage">239</span>, <span class="refDoi"> DOI: 10.1016/j.cellsig.2016.11.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2Fj.cellsig.2016.11.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=27818291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvV2nsLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2017&pages=233-239&author=L.+Wankaauthor=S.+Babilonauthor=K.+Burkertauthor=K.+M%C3%B6rlauthor=V.+V.+Gurevichauthor=A.+G.+Beck-Sickinger&title=C-terminal+motif+of+human+neuropeptide+Y4+receptor+determines+internalization+and+arrestin+recruitment&doi=10.1016%2Fj.cellsig.2016.11.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">C-terminal motif of human neuropeptide Y4 receptor determines internalization and arrestin recruitment</span></div><div class="casAuthors">Wanka, Lizzy; Babilon, Stefanie; Burkert, Kerstin; Moerl, Karin; Gurevich, Vsevolod V.; Beck-Sickinger, Annette G.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Signalling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">233-239</span>CODEN:
                <span class="NLM_cas:coden">CESIEY</span>;
        ISSN:<span class="NLM_cas:issn">0898-6568</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The human neuropeptide Y4 receptor is a rhodopsin-like G protein-coupled receptor (GPCR), which contributes to anorexigenic signals.  Thus, this receptor is a highly interesting target for metabolic diseases.  As GPCR internalization and trafficking affect receptor signaling and vice versa, we aimed to investigate the mol. mechanism of hY4R desensitization and endocytosis.  The role of distinct segments of the hY4R carboxyl terminus was investigated by fluorescence microscopy, binding assays, inositol turnover expts. and bioluminescence resonance energy transfer assays to examine the internalization behavior of hY4R and its interaction with arrestin-3.  Based on results of C-terminal deletion mutants and substitution of single amino acids, the motif 7.78EESEHLPLSTVHTEVSKGS7.96 was identified, with glutamate, threonine and serine residues playing key roles, based on site-directed mutagenesis.  Thus, we identified the internalization motif for the human neuropeptide Y4 receptor, which regulates arrestin-3 recruitment and receptor endocytosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHsGkMKEJwqLVg90H21EOLACvtfcHk0liYPIPiGnXkWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvV2nsLrP&md5=5ac2a1f796acc477a2e836cd88a5b9c6</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.cellsig.2016.11.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellsig.2016.11.003%26sid%3Dliteratum%253Aachs%26aulast%3DWanka%26aufirst%3DL.%26aulast%3DBabilon%26aufirst%3DS.%26aulast%3DBurkert%26aufirst%3DK.%26aulast%3DM%25C3%25B6rl%26aufirst%3DK.%26aulast%3DGurevich%26aufirst%3DV.%2BV.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26atitle%3DC-terminal%2520motif%2520of%2520human%2520neuropeptide%2520Y4%2520receptor%2520determines%2520internalization%2520and%2520arrestin%2520recruitment%26jtitle%3DCell.%2520Signalling%26date%3D2017%26volume%3D29%26spage%3D233%26epage%3D239%26doi%3D10.1016%2Fj.cellsig.2016.11.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burford, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertekap, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alt, A.</span></span> <span> </span><span class="NLM_article-title">Strategies for the identification of allosteric modulators of G-protein-coupled receptors</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">691</span>– <span class="NLM_lpage">702</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2010.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2Fj.bcp.2010.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=21184747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit12jtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2011&pages=691-702&author=N.+T.+Burfordauthor=J.+Watsonauthor=R.+Bertekapauthor=A.+Alt&title=Strategies+for+the+identification+of+allosteric+modulators+of+G-protein-coupled+receptors&doi=10.1016%2Fj.bcp.2010.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies for the identification of allosteric modulators of G-protein-coupled receptors</span></div><div class="casAuthors">Burford, Neil T.; Watson, John; Bertekap, Robert; Alt, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">691-702</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Once considered a pharmacol. curiosity, allosteric modulation of seven transmembrane domain G-protein-coupled receptors (GPCRs) has emerged as a potentially powerful means to affect receptor function for therapeutic purposes.  Allosteric modulators, which interact with binding sites topol. distinct from the orthosteric ligand binding sites, can potentially provide improved selectivity and safety, along with maintenance of spatial and temporal aspects of GPCR signaling.  Accordingly, drug discovery efforts for GPCRs have increasingly focused on the identification of allosteric modulators.  This review is devoted to an examn. of the strategies, challenges, and opportunities for high-throughput screening for allosteric modulators of GPCRs, with particular focus on the identification of pos. allosteric modulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6GSDSVjkw2bVg90H21EOLACvtfcHk0ljQk53JzCxSNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit12jtLs%253D&md5=d2f2b3304cf9d31d0d37456baf9aec71</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2010.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2010.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DBurford%26aufirst%3DN.%2BT.%26aulast%3DWatson%26aufirst%3DJ.%26aulast%3DBertekap%26aufirst%3DR.%26aulast%3DAlt%26aufirst%3DA.%26atitle%3DStrategies%2520for%2520the%2520identification%2520of%2520allosteric%2520modulators%2520of%2520G-protein-coupled%2520receptors%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2011%26volume%3D81%26spage%3D691%26epage%3D702%26doi%3D10.1016%2Fj.bcp.2010.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ballesteros, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstein, H.</span></span> <span> </span><span class="NLM_article-title">[19] Integrated methods for the construction of three-dimensional models and computational probing of structure-function relations in G protein-coupled receptors</span>. <i>Methods Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">366</span>– <span class="NLM_lpage">428</span>, <span class="refDoi"> DOI: 10.1016/s1043-9471(05)80049-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2FS1043-9471%2805%2980049-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADyaK2MXmt1Kmtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1995&pages=366-428&author=J.+A.+Ballesterosauthor=H.+Weinstein&title=%5B19%5D+Integrated+methods+for+the+construction+of+three-dimensional+models+and+computational+probing+of+structure-function+relations+in+G+protein-coupled+receptors&doi=10.1016%2Fs1043-9471%2805%2980049-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Integrated methods for the construction of three-dimensional models and computational probing of structure-function relations in G protein-coupled receptors</span></div><div class="casAuthors">Ballesteros, Juan A.; Weinstein, Harel</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Neurosciences</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">366-428</span>CODEN:
                <span class="NLM_cas:coden">MENEE5</span>;
        ISSN:<span class="NLM_cas:issn">1043-9471</span>.
    </div><div class="casAbstract">A review, with 135 refs., on approaches that can be used to resolve the apparent ambiguities that burden the pharmacol. testing of G protein-coupled receptor (GPCR) models, based on the integration of structural information about the receptor, about mutants, and about the changes induced by ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfENoRspqa7LVg90H21EOLACvtfcHk0ljQk53JzCxSNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmt1Kmtrk%253D&md5=d3f99e9d07cea245a770f3f099bc8cc7</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2FS1043-9471%2805%2980049-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1043-9471%252805%252980049-7%26sid%3Dliteratum%253Aachs%26aulast%3DBallesteros%26aufirst%3DJ.%2BA.%26aulast%3DWeinstein%26aufirst%3DH.%26atitle%3D%255B19%255D%2520Integrated%2520methods%2520for%2520the%2520construction%2520of%2520three-dimensional%2520models%2520and%2520computational%2520probing%2520of%2520structure-function%2520relations%2520in%2520G%2520protein-coupled%2520receptors%26jtitle%3DMethods%2520Neurosci.%26date%3D1995%26volume%3D25%26spage%3D366%26epage%3D428%26doi%3D10.1016%2Fs1043-9471%2805%2980049-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Isberg, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mordalski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munk, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rataj, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harpsoe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vroling, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bojarski, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vriend, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gloriam, D. E.</span></span> <span> </span><span class="NLM_article-title">GPCRdb: an information system for G protein-coupled receptors</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">D356</span>– <span class="NLM_lpage">D364</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv1178</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1093%2Fnar%2Fgkv1178" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=26582914" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2gu7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D356-D364&author=V.+Isbergauthor=S.+Mordalskiauthor=C.+Munkauthor=K.+Ratajauthor=K.+Harpsoeauthor=A.+S.+Hauserauthor=B.+Vrolingauthor=A.+J.+Bojarskiauthor=G.+Vriendauthor=D.+E.+Gloriam&title=GPCRdb%3A+an+information+system+for+G+protein-coupled+receptors&doi=10.1093%2Fnar%2Fgkv1178"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">GPCRdb: an information system for G protein-coupled receptors</span></div><div class="casAuthors">Isberg, Vignir; Mordalski, Stefan; Munk, Christian; Rataj, Krzysztof; Harpsoee, Kasper; Hauser, Alexander S.; Vroling, Bas; Bojarski, Andrzej J.; Vriend, Gert; Gloriam, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D356-D364</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Recent developments in G protein-coupled receptor (GPCR) structural biol. and pharmacol. have greatly enhanced our knowledge of receptor structure-function relations, and have helped improve the scientific foundation for drug design studies.  The GPCR database, GPCRdb, serves a dual role in disseminating and enabling new scientific developments by providing ref. data, anal. tools and interactive diagrams.  This paper highlights new features in the fifth major GPCRdb release: (i) GPCR crystal structure browsing, superposition and display of ligand interactions; (ii) direct deposition by users of point mutations and their effects on ligand binding; (iii) refined snake and helix box residue diagram looks; and (iv) phylogenetic trees with receptor classification color schemes.  Under the hood, the entire GPCRdb front- and back-ends have been recoded within one infrastructure, ensuring a smooth browsing experience and development.  GPCRdb is available at http://www.gpcrdb.org/ and it's open source code at https://bitbucket.org/gpcr/protwis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPnn-bXMy54bVg90H21EOLACvtfcHk0lgM9MyJu3kW6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2gu7bJ&md5=ee7db0ef81d65cf6e5ab8750b990bb78</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv1178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv1178%26sid%3Dliteratum%253Aachs%26aulast%3DIsberg%26aufirst%3DV.%26aulast%3DMordalski%26aufirst%3DS.%26aulast%3DMunk%26aufirst%3DC.%26aulast%3DRataj%26aufirst%3DK.%26aulast%3DHarpsoe%26aufirst%3DK.%26aulast%3DHauser%26aufirst%3DA.%2BS.%26aulast%3DVroling%26aufirst%3DB.%26aulast%3DBojarski%26aufirst%3DA.%2BJ.%26aulast%3DVriend%26aufirst%3DG.%26aulast%3DGloriam%26aufirst%3DD.%2BE.%26atitle%3DGPCRdb%253A%2520an%2520information%2520system%2520for%2520G%2520protein-coupled%2520receptors%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3DD356%26epage%3DD364%26doi%3D10.1093%2Fnar%2Fgkv1178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riepler, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enriquez, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldock, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herzog, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sainsbury, A.</span></span> <span> </span><span class="NLM_article-title">Y2 and Y4 receptor signaling synergistically act on energy expenditure and physical activity</span>. <i>Am. J. Physiol.: Regul., Integr. Comp. Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>299</i></span>,  <span class="NLM_fpage">R1618</span>– <span class="NLM_lpage">R1628</span>, <span class="refDoi"> DOI: 10.1152/ajpregu.00345.2010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1152%2Fajpregu.00345.2010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=20881101" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=299&publication_year=2010&pages=R1618-R1628&author=L.+Zhangauthor=S.+J.+Rieplerauthor=N.+Turnerauthor=R.+F.+Enriquezauthor=I.+C.+Leeauthor=P.+A.+Baldockauthor=H.+Herzogauthor=A.+Sainsbury&title=Y2+and+Y4+receptor+signaling+synergistically+act+on+energy+expenditure+and+physical+activity&doi=10.1152%2Fajpregu.00345.2010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1152%2Fajpregu.00345.2010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajpregu.00345.2010%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DRiepler%26aufirst%3DS.%2BJ.%26aulast%3DTurner%26aufirst%3DN.%26aulast%3DEnriquez%26aufirst%3DR.%2BF.%26aulast%3DLee%26aufirst%3DI.%2BC.%26aulast%3DBaldock%26aufirst%3DP.%2BA.%26aulast%3DHerzog%26aufirst%3DH.%26aulast%3DSainsbury%26aufirst%3DA.%26atitle%3DY2%2520and%2520Y4%2520receptor%2520signaling%2520synergistically%2520act%2520on%2520energy%2520expenditure%2520and%2520physical%2520activity%26jtitle%3DAm.%2520J.%2520Physiol.%253A%2520Regul.%252C%2520Integr.%2520Comp.%2520Physiol.%26date%3D2010%26volume%3D299%26spage%3DR1618%26epage%3DR1628%26doi%3D10.1152%2Fajpregu.00345.2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tough, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holliday, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, H. M.</span></span> <span> </span><span class="NLM_article-title">Y(4) receptors mediate the inhibitory responses of pancreatic polypeptide in human and mouse colon mucosa</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>319</i></span>,  <span class="NLM_fpage">20</span>– <span class="NLM_lpage">30</span>, <span class="refDoi"> DOI: 10.1124/jpet.106.106500</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1124%2Fjpet.106.106500" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=16807358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVCksL%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=319&publication_year=2006&pages=20-30&author=I.+R.+Toughauthor=N.+D.+Hollidayauthor=H.+M.+Cox&title=Y%284%29+receptors+mediate+the+inhibitory+responses+of+pancreatic+polypeptide+in+human+and+mouse+colon+mucosa&doi=10.1124%2Fjpet.106.106500"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Y4 receptors mediate the inhibitory responses of pancreatic polypeptide in human and mouse colon mucosa</span></div><div class="casAuthors">Tough, Iain R.; Holliday, Nicholas D.; Cox, Helen M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">319</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">20-30</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The antisecretory effects of several Y agonists, including pancreatic polypeptide (PP), indicate the presence of Y1, Y2, and Y4 receptors in mouse and human (h) colon mucosa.  Here, we used prepns. from human and from wild-type (WT), Y4, and Y1 receptor knockout (-/-) mice, alongside Y4 receptor-transfected cells to define the relative functional contribution of the Y4 receptor.  First, rat (r) PP antisecretory responses were lost in murine Y4-/- prepns., but hPP and Pro34 peptide YY (PYY) costimulated Y4 and Y1 receptors in WT mucosa.  The Y1 antagonist/Y4 agonist GR231118 [(IIe,Glu,Pro,Dpr,Tyr,Arg,Leu,Arg,Try-NH2)-2-cyclic(2,4'),(2',4)-diamide] elicited small Y4-mediated antisecretory responses in human tissues pretreated with the Y1 antagonist, BIBO3304 (R)-N-[(4-(aminocarbonylaminomethyl)-phenylmethyl)]-N2-[(diphenylacetyl)-argininamiden trifluoroacetate], and attenuated Y4-mediated hPP responses in mouse and human mucosa.  GR231118 and rPP were also antisecretory in hY4-transfected epithelial monolayers but were partial agonists compared with hPP at this receptor.  In Y4-transfected human embryonic kidney (HEK) 293 cells, Y4 ligands displaced [125I]hPP binding with orders of affinity (pKi) at human (hPP = rPP > GR231118 > Pro34PYY = PYY) and mouse (rPP = hPP > GR231118 > Pro34PYY > PYY) Y4 receptors.  GR23118- and rPP-stimulated guanosine 5'-3-O-(thio)triphosphate binding through hY4 receptors with significantly lower efficacy than hPP.  GR231118 marginally increased basal but abolished further PP-induced hY4 internalization to recycling (transferrin-labeled) pathways in HEK293 cells.  Taken together, these findings show that Y4 receptors play a definitive role in attenuating colonic anion transport and may be useful targets for novel antidiarrheal agents due to their limited peripheral expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHuAe6MTFvirVg90H21EOLACvtfcHk0lhEwE1ruk0vMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVCksL%252FI&md5=cf51b740f94ff70872f85de3d9bf815a</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1124%2Fjpet.106.106500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.106.106500%26sid%3Dliteratum%253Aachs%26aulast%3DTough%26aufirst%3DI.%2BR.%26aulast%3DHolliday%26aufirst%3DN.%2BD.%26aulast%3DCox%26aufirst%3DH.%2BM.%26atitle%3DY%25284%2529%2520receptors%2520mediate%2520the%2520inhibitory%2520responses%2520of%2520pancreatic%2520polypeptide%2520in%2520human%2520and%2520mouse%2520colon%2520mucosa%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2006%26volume%3D319%26spage%3D20%26epage%3D30%26doi%3D10.1124%2Fjpet.106.106500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meiler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span> <span> </span><span class="NLM_article-title">ROSETTALIGAND: protein-small molecule docking with full side-chain flexibility</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">538</span>– <span class="NLM_lpage">548</span>, <span class="refDoi"> DOI: 10.1002/prot.21086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1002%2Fprot.21086" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=16972285" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFWqt7nJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2006&pages=538-548&author=J.+Meilerauthor=D.+Baker&title=ROSETTALIGAND%3A+protein-small+molecule+docking+with+full+side-chain+flexibility&doi=10.1002%2Fprot.21086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">ROSETTALIGAND: protein-small molecule docking with full side-chain flexibility</span></div><div class="casAuthors">Meiler, Jens; Baker, David</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">538-548</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Protein-small mol. docking algorithms provide a means to model the structure of protein-small mol. complexes in structural detail and play an important role in drug development.  In recent years the necessity of simulating protein side-chain flexibility for an accurate prediction of the protein-small mol. interfaces has become apparent, and an increasing no. of docking algorithms probe different approaches to include protein flexibility.  Here the authors describe a new method for docking small mols. into protein binding sites employing a Monte Carlo minimization procedure in which the rigid body position and orientation of the small mol. and the protein side-chain conformations are optimized simultaneously.  The energy function comprises van der Waals (VDW) interactions, an implicit solvation model, an explicit orientation hydrogen bonding potential, and an electrostatics model.  In an evaluation of the scoring function the computed energy correlated with exptl. small mol. binding energy with a correlation coeff. of 0.63 across a diverse set of 229 protein-small mol. complexes.  The docking method produced lowest energy models with a root mean square deviation (RMSD) smaller than 2 Å in 71 out of 100 protein-small mol. crystal structure complexes (self-docking).  In cross-docking calcns. in which both protein side-chain and small mol. internal degrees of freedom were varied the lowest energy predictions had RMSDs less than 2 Å in 14 of 20 test cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb1x4K4yjDk7Vg90H21EOLACvtfcHk0lhR-busUlBaww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFWqt7nJ&md5=0e5580cc490c11a7e9c6265c27a14408</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fprot.21086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21086%26sid%3Dliteratum%253Aachs%26aulast%3DMeiler%26aufirst%3DJ.%26aulast%3DBaker%26aufirst%3DD.%26atitle%3DROSETTALIGAND%253A%2520protein-small%2520molecule%2520docking%2520with%2520full%2520side-chain%2520flexibility%26jtitle%3DProteins%26date%3D2006%26volume%3D65%26spage%3D538%26epage%3D548%26doi%3D10.1002%2Fprot.21086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Combs, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deluca, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deluca, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemmon, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nannemann, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willis, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheehan, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meiler, J.</span></span> <span> </span><span class="NLM_article-title">Small-molecule ligand docking into comparative models with Rosetta</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1277</span>– <span class="NLM_lpage">1298</span>, <span class="refDoi"> DOI: 10.1038/nprot.2013.074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1038%2Fnprot.2013.074" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=23744289" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC3sXovFartLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1277-1298&author=S.+A.+Combsauthor=S.+L.+Delucaauthor=S.+H.+Delucaauthor=G.+H.+Lemmonauthor=D.+P.+Nannemannauthor=E.+D.+Nguyenauthor=J.+R.+Willisauthor=J.+H.+Sheehanauthor=J.+Meiler&title=Small-molecule+ligand+docking+into+comparative+models+with+Rosetta&doi=10.1038%2Fnprot.2013.074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule ligand docking into comparative models with Rosetta</span></div><div class="casAuthors">Combs, Steven A.; DeLuca, Samuel L.; DeLuca, Stephanie H.; Lemmon, Gordon H.; Nannemann, David P.; Nguyen, Elizabeth D.; Willis, Jordan R.; Sheehan, Jonathan H.; Meiler, Jens</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1277-1298</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Structure-based drug design is frequently used to accelerate the development of small-mol. therapeutics.  Although substantial progress has been made in X-ray crystallog. and NMR spectroscopy, the availability of high-resoln. structures is limited owing to the frequent inability to crystallize or obtain sufficient NMR restraints for large or flexible proteins.  Computational methods can be used to both predict unknown protein structures and model ligand interactions when exptl. data are unavailable.  This paper describes a comprehensive and detailed protocol using the Rosetta modeling suite to dock small-mol. ligands into comparative models.  In the protocol presented here, we review the comparative modeling process, including sequence alignment, threading and loop building.  Next, we cover docking a small-mol. ligand into the protein comparative model.  In addn., we discuss criteria that can improve ligand docking into comparative models.  Finally, and importantly, we present a strategy for assessing model quality.  The entire protocol is presented on a single example selected solely for didactic purposes.  The results are therefore not representative and do not replace benchmarks published elsewhere.  We also provide an addnl. tutorial so that the user can gain hands-on experience in using Rosetta.  The protocol should take 5-7 h, with addnl. time allocated for computer generation of models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwkuPWDwMFFbVg90H21EOLACvtfcHk0libPkweANdQ-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXovFartLs%253D&md5=17577fd22e6ca3bd13a2c3aa66ba9b25</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2013.074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2013.074%26sid%3Dliteratum%253Aachs%26aulast%3DCombs%26aufirst%3DS.%2BA.%26aulast%3DDeluca%26aufirst%3DS.%2BL.%26aulast%3DDeluca%26aufirst%3DS.%2BH.%26aulast%3DLemmon%26aufirst%3DG.%2BH.%26aulast%3DNannemann%26aufirst%3DD.%2BP.%26aulast%3DNguyen%26aufirst%3DE.%2BD.%26aulast%3DWillis%26aufirst%3DJ.%2BR.%26aulast%3DSheehan%26aufirst%3DJ.%2BH.%26aulast%3DMeiler%26aufirst%3DJ.%26atitle%3DSmall-molecule%2520ligand%2520docking%2520into%2520comparative%2520models%2520with%2520Rosetta%26jtitle%3DNat.%2520Protoc.%26date%3D2013%26volume%3D8%26spage%3D1277%26epage%3D1298%26doi%3D10.1038%2Fnprot.2013.074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lemmon, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meiler, J.</span></span> <span> </span><span class="NLM_article-title">Rosetta ligand docking with flexible XML protocols</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>819</i></span>,  <span class="NLM_fpage">143</span>– <span class="NLM_lpage">155</span>, <span class="refDoi"> DOI: 10.1007/978-1-61779-465-0_10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1007%2F978-1-61779-465-0_10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=22183535" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFSjur7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=819&publication_year=2012&pages=143-155&author=G.+Lemmonauthor=J.+Meiler&title=Rosetta+ligand+docking+with+flexible+XML+protocols&doi=10.1007%2F978-1-61779-465-0_10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Rosetta ligand docking with flexible XML protocols</span></div><div class="casAuthors">Lemmon, Gordon; Meiler, Jens</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">819</span>
        (<span class="NLM_cas:issue">Computational Drug Discovery and Design</span>),
    <span class="NLM_cas:pages">143-155</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Rosetta Ligand is premiere software for predicting how a protein and a small mol. interact.  Benchmark studies demonstrate that 70% of the top scoring Rosetta Ligand predicted interfaces are within 2 Å RMSD from the crystal structure.  The latest release of Rosetta ligand software includes many new features, such as (1) docking of multiple ligands simultaneously, (2) representing ligands as fragments for greater flexibility, (3) redesign of the interface during docking, and (4) an XML script based interface that gives the user full control of the ligand docking protocol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5flEQBFyUqbVg90H21EOLACvtfcHk0lgGSBEfoOIEmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFSjur7P&md5=d815380e73b10f59adc5a8f27f7d2653</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-465-0_10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-465-0_10%26sid%3Dliteratum%253Aachs%26aulast%3DLemmon%26aufirst%3DG.%26aulast%3DMeiler%26aufirst%3DJ.%26atitle%3DRosetta%2520ligand%2520docking%2520with%2520flexible%2520XML%2520protocols%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2012%26volume%3D819%26spage%3D143%26epage%3D155%26doi%3D10.1007%2F978-1-61779-465-0_10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiMaio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miles, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunette, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span> <span> </span><span class="NLM_article-title">High-resolution comparative modeling with RosettaCM</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1735</span>– <span class="NLM_lpage">1742</span>, <span class="refDoi"> DOI: 10.1016/j.str.2013.08.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2Fj.str.2013.08.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=24035711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVCltrrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=1735-1742&author=Y.+Songauthor=F.+DiMaioauthor=R.+Y.+Wangauthor=D.+Kimauthor=C.+Milesauthor=T.+Brunetteauthor=J.+Thompsonauthor=D.+Baker&title=High-resolution+comparative+modeling+with+RosettaCM&doi=10.1016%2Fj.str.2013.08.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">High-Resolution Comparative Modeling with RosettaCM</span></div><div class="casAuthors">Song, Yifan; DiMaio, Frank; Wang, Ray Yu-Ruei; Kim, David; Miles, Chris; Brunette, T. J.; Thompson, James; Baker, David</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1735-1742</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">We describe an improved method for comparative modeling, RosettaCM, which optimizes a phys. realistic all-atom energy function over the conformational space defined by homologous structures.  Given a set of sequence alignments, RosettaCM assembles topologies by recombining aligned segments in Cartesian space and building unaligned regions de novo in torsion space.  The junctions between segments are regularized using a loop closure method combining fragment superposition with gradient-based minimization.  The energies of the resulting models are optimized by all-atom refinement, and the most representative low-energy model is selected.  The CASP10 expt. suggests that RosettaCM yields models with more accurate side-chain and backbone conformations than other methods when the sequence identity to the templates is greater than ∼15%.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorujEuJW1hmbVg90H21EOLACvtfcHk0lgg9PamdHfkpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVCltrrP&md5=299d5b1f30a1fdd0d311804158cc5053</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2013.08.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2013.08.005%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DY.%26aulast%3DDiMaio%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DR.%2BY.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DMiles%26aufirst%3DC.%26aulast%3DBrunette%26aufirst%3DT.%26aulast%3DThompson%26aufirst%3DJ.%26aulast%3DBaker%26aufirst%3DD.%26atitle%3DHigh-resolution%2520comparative%2520modeling%2520with%2520RosettaCM%26jtitle%3DStructure%26date%3D2013%26volume%3D21%26spage%3D1735%26epage%3D1742%26doi%3D10.1016%2Fj.str.2013.08.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pedragosa-Badia, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sliwoski, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong
Nguyen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stichel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufmann, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meiler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span> <span> </span><span class="NLM_article-title">Pancreatic polypeptide is recognized by two hydrophobic domains of the human Y4 receptor binding pocket</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">5846</span>– <span class="NLM_lpage">5859</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.502021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1074%2Fjbc.M113.502021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=24375409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjsVKktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=5846-5859&author=X.+Pedragosa-Badiaauthor=G.+R.+Sliwoskiauthor=E.+Dong%0ANguyenauthor=D.+Lindnerauthor=J.+Stichelauthor=K.+W.+Kaufmannauthor=J.+Meilerauthor=A.+G.+Beck-Sickinger&title=Pancreatic+polypeptide+is+recognized+by+two+hydrophobic+domains+of+the+human+Y4+receptor+binding+pocket&doi=10.1074%2Fjbc.M113.502021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Pancreatic Polypeptide Is Recognized by Two Hydrophobic Domains of the Human Y4 Receptor Binding Pocket</span></div><div class="casAuthors">Pedragosa-Badia, Xavier; Sliwoski, Gregory R.; Nguyen, Elizabeth Dong; Lindner, Diana; Stichel, Jan; Kaufmann, Kristian W.; Meiler, Jens; Beck-Sickinger, Annette G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">5846-5859</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Structural characterization of the human Y4 receptor (hY4R) interaction with human pancreatic polypeptide (hPP) is crucial, not only for understanding its biol. function but also for testing treatment strategies for obesity that target this interaction.  Here, the interaction of receptor mutants with pancreatic polypeptide analogs was studied through double-cycle mutagenesis.  To guide mutagenesis and interpret results, a three-dimensional comparative model of the hY4R-hPP complex was constructed based on all available class A G protein-coupled receptor crystal structures and refined using exptl. data.  Our study reveals that residues of the hPP and the hY4R form a complex network consisting of ionic interactions, hydrophobic interactions, and hydrogen binding.  Residues Tyr2.64, Asp2.68, Asn6.55, Asn7.32, and Phe7.35 of Y4R are found to be important in receptor activation by hPP.  Specifically, Tyr2.64 interacts with Tyr27 of hPP through hydrophobic contacts.  Asn7.32 is affected by modifications on position Arg33 of hPP, suggesting a hydrogen bond between these two residues.  Likewise, we find that Phe7.35 is affected by modifications of hPP at positions 33 and 36, indicating interactions between these three amino acids.  Taken together, we demonstrate that the top of transmembrane helix 2 (TM2) and the top of transmembrane helixes 6 and 7 (TM6-TM7) form the core of the peptide binding pocket.  These findings will contribute to the rational design of ligands that bind the receptor more effectively to produce an enhanced agonistic or antagonistic effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkLgd7H3oyYrVg90H21EOLACvtfcHk0liDQLpel9hznQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjsVKktL4%253D&md5=cf4bf542381418e47815b8b139c8f99b</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.502021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.502021%26sid%3Dliteratum%253Aachs%26aulast%3DPedragosa-Badia%26aufirst%3DX.%26aulast%3DSliwoski%26aufirst%3DG.%2BR.%26aulast%3DDong%2BNguyen%26aufirst%3DE.%26aulast%3DLindner%26aufirst%3DD.%26aulast%3DStichel%26aufirst%3DJ.%26aulast%3DKaufmann%26aufirst%3DK.%2BW.%26aulast%3DMeiler%26aufirst%3DJ.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26atitle%3DPancreatic%2520polypeptide%2520is%2520recognized%2520by%2520two%2520hydrophobic%2520domains%2520of%2520the%2520human%2520Y4%2520receptor%2520binding%2520pocket%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D5846%26epage%3D5859%26doi%3D10.1074%2Fjbc.M113.502021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kögler, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wifling, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plank, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Littmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larhammar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huster, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meiler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buschauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, B.</span></span> <span> </span><span class="NLM_article-title">Structural basis of ligand binding modes at the neuropeptide Y Y1 receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>556</i></span>,  <span class="NLM_fpage">520</span>– <span class="NLM_lpage">524</span>, <span class="refDoi"> DOI: 10.1038/s41586-018-0046-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1038%2Fs41586-018-0046-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=29670288" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC1cXosVCgur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=556&publication_year=2018&pages=520-524&author=Z.+Yangauthor=S.+Hanauthor=M.+Kellerauthor=A.+Kaiserauthor=B.+J.+Benderauthor=M.+Bosseauthor=K.+Burkertauthor=L.+M.+K%C3%B6glerauthor=D.+Wiflingauthor=G.+Bernhardtauthor=N.+Plankauthor=T.+Littmannauthor=P.+Schmidtauthor=C.+Yiauthor=B.+Liauthor=S.+Yeauthor=R.+Zhangauthor=B.+Xuauthor=D.+Larhammarauthor=R.+C.+Stevensauthor=D.+Husterauthor=J.+Meilerauthor=Q.+Zhaoauthor=A.+G.+Beck-Sickingerauthor=A.+Buschauerauthor=B.+Wu&title=Structural+basis+of+ligand+binding+modes+at+the+neuropeptide+Y+Y1+receptor&doi=10.1038%2Fs41586-018-0046-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of ligand binding modes at the neuropeptide Y Y1 receptor</span></div><div class="casAuthors">Yang, Zhenlin; Han, Shuo; Keller, Max; Kaiser, Anette; Bender, Brian J.; Bosse, Mathias; Burkert, Kerstin; Kogler, Lisa M.; Wifling, David; Bernhardt, Guenther; Plank, Nicole; Littmann, Timo; Schmidt, Peter; Yi, Cuiying; Li, Beibei; Ye, Sheng; Zhang, Rongguang; Xu, Bo; Larhammar, Dan; Stevens, Raymond C.; Huster, Daniel; Meiler, Jens; Zhao, Qiang; Beck-Sickinger, Annette G.; Buschauer, Armin; Wu, Beili</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">556</span>
        (<span class="NLM_cas:issue">7702</span>),
    <span class="NLM_cas:pages">520-524</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Neuropeptide Y (NPY) receptors belong to the G-protein-coupled receptor superfamily and have important roles in food intake, anxiety and cancer biol.1,2.  The NPY-Y receptor system has emerged as one of the most complex networks with three peptide ligands (NPY, peptide YY and pancreatic polypeptide) binding to four receptors in most mammals, namely the Y1, Y2, Y4 and Y5 receptors, with different affinity and selectivity3.  NPY is the most powerful stimulant of food intake and this effect is primarily mediated by the Y1 receptor (Y1R)4.  A no. of peptides and small-mol. compds. have been characterized as Y1R antagonists and have shown clin. potential in the treatment of obesity4, tumor1 and bone loss5.  However, their clin. usage has been hampered by low potency and selectivity, poor brain penetration ability or lack of oral bioavailability6.  Here we report crystal structures of the human Y1R bound to the two selective antagonists UR-MK299 and BMS-193885 at 2.7 and 3.0 Å resoln., resp.  The structures combined with mutagenesis studies reveal the binding modes of Y1R to several structurally diverse antagonists and the determinants of ligand selectivity.  The Y1R structure and mol. docking of the endogenous agonist NPY, together with NMR, photo-crosslinking and functional studies, provide insights into the binding behavior of the agonist and for the first time, to our knowledge, det. the interaction of its N terminus with the receptor.  These insights into Y1R can enable structure-based drug discovery that targets NPY receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr584bBD3wdnbVg90H21EOLACvtfcHk0lh8rKJE5Gaugw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXosVCgur8%253D&md5=79289fb85d5f65b94fbf873ec4ea31f3</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fs41586-018-0046-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-018-0046-x%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DHan%26aufirst%3DS.%26aulast%3DKeller%26aufirst%3DM.%26aulast%3DKaiser%26aufirst%3DA.%26aulast%3DBender%26aufirst%3DB.%2BJ.%26aulast%3DBosse%26aufirst%3DM.%26aulast%3DBurkert%26aufirst%3DK.%26aulast%3DK%25C3%25B6gler%26aufirst%3DL.%2BM.%26aulast%3DWifling%26aufirst%3DD.%26aulast%3DBernhardt%26aufirst%3DG.%26aulast%3DPlank%26aufirst%3DN.%26aulast%3DLittmann%26aufirst%3DT.%26aulast%3DSchmidt%26aufirst%3DP.%26aulast%3DYi%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DYe%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DXu%26aufirst%3DB.%26aulast%3DLarhammar%26aufirst%3DD.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DHuster%26aufirst%3DD.%26aulast%3DMeiler%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26aulast%3DBuschauer%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DB.%26atitle%3DStructural%2520basis%2520of%2520ligand%2520binding%2520modes%2520at%2520the%2520neuropeptide%2520Y%2520Y1%2520receptor%26jtitle%3DNature%26date%3D2018%26volume%3D556%26spage%3D520%26epage%3D524%26doi%3D10.1038%2Fs41586-018-0046-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Merten, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Römpler, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mörl, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schöneberg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span> <span> </span><span class="NLM_article-title">Receptor subtype-specific docking of Asp6.59 with C-terminal arginine residues in Y receptor ligands</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">7543</span>– <span class="NLM_lpage">7551</span>, <span class="refDoi"> DOI: 10.1074/jbc.M608902200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1074%2Fjbc.M608902200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=17204471" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitlSrtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=7543-7551&author=N.+Mertenauthor=D.+Lindnerauthor=N.+Rabeauthor=H.+R%C3%B6mplerauthor=K.+M%C3%B6rlauthor=T.+Sch%C3%B6nebergauthor=A.+G.+Beck-Sickinger&title=Receptor+subtype-specific+docking+of+Asp6.59+with+C-terminal+arginine+residues+in+Y+receptor+ligands&doi=10.1074%2Fjbc.M608902200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Receptor Subtype-specific Docking of Asp6.59 with C-terminal Arginine Residues in Y Receptor Ligands</span></div><div class="casAuthors">Merten, Nicole; Lindner, Diana; Rabe, Nadine; Roempler, Holger; Moerl, Karin; Schoeneberg, Torsten; Beck-Sickinger, Annette G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">7543-7551</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Y receptors (YRs) are G protein-coupled receptors whose Y1R, Y2R, and Y5R subtypes preferentially bind neuropeptide Y (NPY) and peptide YY, whereas mammalian Y4Rs show a higher affinity for pancreatic polypeptide (PP).  Comparison of YR orthologs and paralogs revealed Asp6.59 to be fully conserved throughout all of the YRs reported so far.  By replacing this conserved aspartic acid residue with alanine, asparagine, glutamate, and arginine, we now show that this residue plays a crucial role in binding and signal transduction of NPY/PP at all YRs.  Sensitivity to distinct replacements is, however, receptor subtype-specific.  Next, we performed a complementary mutagenesis approach to identify the contact site of the ligand.  Surprisingly, this conserved residue interacts with two different ligand arginine residues by ionic interactions; although in Y2R and Y5R, Arg33 is the binding partner of Asp6.59, in Y1R and Y4R, Arg35 of human PP and NPY interacts with Asp6.59.  Furthermore, Arg25 of PP and NPY is involved in ligand binding only at Y2R and Y5R.  This suggests significant differences in the docking of YR ligands between Y1/4R and Y2/5R and provides new insights into the mol. binding mode of peptide agonists at GPCRs.  Furthermore, the proposed model of a subtype-specific binding mode is in agreement with the evolution of YRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMfokRVrtUB7Vg90H21EOLACvtfcHk0lh8rKJE5Gaugw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitlSrtLs%253D&md5=78e41ddb73bcfd8d23163759159d1bb2</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M608902200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M608902200%26sid%3Dliteratum%253Aachs%26aulast%3DMerten%26aufirst%3DN.%26aulast%3DLindner%26aufirst%3DD.%26aulast%3DRabe%26aufirst%3DN.%26aulast%3DR%25C3%25B6mpler%26aufirst%3DH.%26aulast%3DM%25C3%25B6rl%26aufirst%3DK.%26aulast%3DSch%25C3%25B6neberg%26aufirst%3DT.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26atitle%3DReceptor%2520subtype-specific%2520docking%2520of%2520Asp6.59%2520with%2520C-terminal%2520arginine%2520residues%2520in%2520Y%2520receptor%2520ligands%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D7543%26epage%3D7551%26doi%3D10.1074%2Fjbc.M608902200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kufareva, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenalti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, B.</span></span> <span> </span><span class="NLM_article-title">Structure of the CCR5 chemokine receptor-HIV entry inhibitor maraviroc complex</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>341</i></span>,  <span class="NLM_fpage">1387</span>– <span class="NLM_lpage">1390</span>, <span class="refDoi"> DOI: 10.1126/science.1241475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1126%2Fscience.1241475" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=24030490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVCmu7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=1387-1390&author=Q.+Tanauthor=Y.+Zhuauthor=J.+Liauthor=Z.+Chenauthor=G.+W.+Hanauthor=I.+Kufarevaauthor=T.+Liauthor=L.+Maauthor=G.+Fenaltiauthor=J.+Liauthor=W.+Zhangauthor=X.+Xieauthor=H.+Yangauthor=H.+Jiangauthor=V.+Cherezovauthor=H.+Liuauthor=R.+C.+Stevensauthor=Q.+Zhaoauthor=B.+Wu&title=Structure+of+the+CCR5+chemokine+receptor-HIV+entry+inhibitor+maraviroc+complex&doi=10.1126%2Fscience.1241475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the CCR5 Chemokine Receptor-HIV Entry Inhibitor Maraviroc Complex</span></div><div class="casAuthors">Tan, Qiuxiang; Zhu, Ya; Li, Jian; Chen, Zhuxi; Han, Gye Won; Kufareva, Irina; Li, Tingting; Ma, Limin; Fenalti, Gustavo; Li, Jing; Zhang, Wenru; Xie, Xin; Yang, Huaiyu; Jiang, Hualiang; Cherezov, Vadim; Liu, Hong; Stevens, Raymond C.; Zhao, Qiang; Wu, Beili</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6152</span>),
    <span class="NLM_cas:pages">1387-1390</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The CCR5 chemokine receptor acts as a co-receptor for HIV-1 viral entry.  Here we report the 2.7 angstrom-resoln. crystal structure of human CCR5 bound to the marketed HIV drug maraviroc.  The structure reveals a ligand-binding site that is distinct from the proposed major recognition sites for chemokines and the viral glycoprotein gp120, providing insights into the mechanism of allosteric inhibition of chemokine signaling and viral entry.  A comparison between CCR5 and CXCR4 crystal structures, along with models of co-receptor-gp120-V3 complexes, suggests that different charge distributions and steric hindrances caused by residue substitutions may be major determinants of HIV-1 co-receptor selectivity.  These high-resoln. insights into CCR5 can enable structure-based drug discovery for the treatment of HIV-1 infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt5VKeO5Hmb7Vg90H21EOLACvtfcHk0ljK2YrBU_tNKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVCmu7vJ&md5=730ee5644a2a144cd49cd0e9a15eba2e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1126%2Fscience.1241475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1241475%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DQ.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DKufareva%26aufirst%3DI.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DFenalti%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DB.%26atitle%3DStructure%2520of%2520the%2520CCR5%2520chemokine%2520receptor-HIV%2520entry%2520inhibitor%2520maraviroc%2520complex%26jtitle%3DScience%26date%3D2013%26volume%3D341%26spage%3D1387%26epage%3D1390%26doi%3D10.1126%2Fscience.1241475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, R. K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiez-Vandal, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlenker, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aggeler, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dore, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geschwindner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grebner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermansson, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jazayeri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prihandoko, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rappas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soutter, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snijder, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundstrom, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tehan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troast, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiggin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhukov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dekker, N.</span></span> <span> </span><span class="NLM_article-title">Structural insight into allosteric modulation of protease-activated receptor 2</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>545</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">115</span>, <span class="refDoi"> DOI: 10.1038/nature22309</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1038%2Fnature22309" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=28445455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmslehsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=545&publication_year=2017&pages=112-115&author=R.+K.+Y.+Chengauthor=C.+Fiez-Vandalauthor=O.+Schlenkerauthor=K.+Edmanauthor=B.+Aggelerauthor=D.+G.+Brownauthor=G.+A.+Brownauthor=R.+M.+Cookeauthor=C.+E.+Dumelinauthor=A.+S.+Doreauthor=S.+Geschwindnerauthor=C.+Grebnerauthor=N.+O.+Hermanssonauthor=A.+Jazayeriauthor=P.+Johanssonauthor=L.+Leongauthor=R.+Prihandokoauthor=M.+Rappasauthor=H.+Soutterauthor=A.+Snijderauthor=L.+Sundstromauthor=B.+Tehanauthor=P.+Thorntonauthor=D.+Troastauthor=G.+Wigginauthor=A.+Zhukovauthor=F.+H.+Marshallauthor=N.+Dekker&title=Structural+insight+into+allosteric+modulation+of+protease-activated+receptor+2&doi=10.1038%2Fnature22309"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insight into allosteric modulation of protease-activated receptor 2</span></div><div class="casAuthors">Cheng, Robert K. Y.; Fiez-Vandal, Cedric; Schlenker, Oliver; Edman, Karl; Aggeler, Birte; Brown, Dean G.; Brown, Giles A.; Cooke, Robert M.; Dumelin, Christoph E.; Dore, Andrew S.; Geschwindner, Stefan; Grebner, Christoph; Hermansson, Nils-Olov; Jazayeri, Ali; Johansson, Patrik; Leong, Louis; Prihandoko, Rudi; Rappas, Mathieu; Soutter, Holly; Snijder, Arjan; Sundstrom, Linda; Tehan, Benjamin; Thornton, Peter; Troast, Dawn; Wiggin, Giselle; Zhukov, Andrei; Marshall, Fiona H.; Dekker, Niek</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">545</span>
        (<span class="NLM_cas:issue">7652</span>),
    <span class="NLM_cas:pages">112-115</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protease-activated receptors (PARs) are a family of G-protein-coupled receptors (GPCRs) that are irreversibly activated by proteolytic cleavage of the N-terminus, which unmasks a tethered peptide ligand that binds and activates the transmembrane receptor domain, eliciting a cellular cascade in response to inflammatory signals and other stimuli.  PARs are implicated in a wide range of diseases, such as cancer and inflammation.  PARs have been the subject of major pharmaceutical research efforts, but the discovery of small-mol. antagonists that effectively bind them has proved challenging.  The only marketed drug targeting a PAR is vorapaxar, a selective antagonist of PAR1 used to prevent thrombosis.  The structure of PAR1 in complex with vorapaxar has been reported previously.  Despite sequence homol. across the PAR isoforms, discovery of PAR2 antagonists has been less successful, although GB88 has been described as a weak antagonist.  Here, we report crystal structures of PAR2 in complex with two distinct antagonists and a blocking antibody.  The antagonist AZ8838 binds in a fully occluded pocket near the extracellular surface.  Functional and binding studies reveal that AZ8838 exhibits slow binding kinetics, which is an attractive feature for a PAR2 antagonist competing against a tethered ligand.  Antagonist AZ3451 binds to a remote allosteric site outside the helical bundle.  We propose that antagonist binding prevents structural rearrangements required for receptor activation and signalling.  We also show that a blocking antibody antigen-binding fragment binds to the extracellular surface of PAR2, preventing access of the tethered ligand to the peptide-binding site.  These structures provide a basis for the development of selective PAR2 antagonists for a range of therapeutic uses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSz2vmZJ2f6LVg90H21EOLACvtfcHk0lgSscYg7AmJPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmslehsrs%253D&md5=6a0737757c891336437d4419c3fd6c45</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnature22309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22309%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DR.%2BK.%2BY.%26aulast%3DFiez-Vandal%26aufirst%3DC.%26aulast%3DSchlenker%26aufirst%3DO.%26aulast%3DEdman%26aufirst%3DK.%26aulast%3DAggeler%26aufirst%3DB.%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DBrown%26aufirst%3DG.%2BA.%26aulast%3DCooke%26aufirst%3DR.%2BM.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DDore%26aufirst%3DA.%2BS.%26aulast%3DGeschwindner%26aufirst%3DS.%26aulast%3DGrebner%26aufirst%3DC.%26aulast%3DHermansson%26aufirst%3DN.%2BO.%26aulast%3DJazayeri%26aufirst%3DA.%26aulast%3DJohansson%26aufirst%3DP.%26aulast%3DLeong%26aufirst%3DL.%26aulast%3DPrihandoko%26aufirst%3DR.%26aulast%3DRappas%26aufirst%3DM.%26aulast%3DSoutter%26aufirst%3DH.%26aulast%3DSnijder%26aufirst%3DA.%26aulast%3DSundstrom%26aufirst%3DL.%26aulast%3DTehan%26aufirst%3DB.%26aulast%3DThornton%26aufirst%3DP.%26aulast%3DTroast%26aufirst%3DD.%26aulast%3DWiggin%26aufirst%3DG.%26aulast%3DZhukov%26aufirst%3DA.%26aulast%3DMarshall%26aufirst%3DF.%2BH.%26aulast%3DDekker%26aufirst%3DN.%26atitle%3DStructural%2520insight%2520into%2520allosteric%2520modulation%2520of%2520protease-activated%2520receptor%25202%26jtitle%3DNature%26date%3D2017%26volume%3D545%26spage%3D112%26epage%3D115%26doi%3D10.1038%2Fnature22309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ziemek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabrele, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buschauer, A.</span></span> <span> </span><span class="NLM_article-title">Determination of affinity and activity of ligands at the human neuropeptide Y Y4 receptor by flow cytometry and aequorin luminescence</span>. <i>J. Recept. Signal Transduction</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">217</span>– <span class="NLM_lpage">233</span>, <span class="refDoi"> DOI: 10.1080/10799890701505206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1080%2F10799890701505206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=17885919" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVGrs7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=217-233&author=R.+Ziemekauthor=E.+Schneiderauthor=A.+Krausauthor=C.+Cabreleauthor=A.+G.+Beck-Sickingerauthor=G.+Bernhardtauthor=A.+Buschauer&title=Determination+of+affinity+and+activity+of+ligands+at+the+human+neuropeptide+Y+Y4+receptor+by+flow+cytometry+and+aequorin+luminescence&doi=10.1080%2F10799890701505206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Determination of Affinity and Activity of Ligands at the Human Neuropeptide Y Y4 Receptor by Flow Cytometry and Aequorin Luminescence</span></div><div class="casAuthors">Ziemek, Ralf; Schneider, Erich; Kraus, Anja; Cabrele, Chiara; Beck-Sickinger, Annette G.; Bernhardt, Guenther; Buschauer, Armin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Receptors and Signal Transduction</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">217-233</span>CODEN:
                <span class="NLM_cas:coden">JRSTCT</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">Fluorescence-labeled neuropeptide Y (NPY) has been used in flow cytometric binding assays for the detn. of affinity consts. of NPY Y1, Y2, and Y5 receptor ligands.  Because the binding of fluorescent NPY is insufficient for competition studies at the human Y4 receptor (hY4R), the authors replaced Glu 4 in hPP with Lys for the derivatization with cyanine-5.  Because cy5-[K4]hPP has high affinity (Kd 5.6 nM) to the hY4R, it was used as a probe in a flow cytometric binding assay.  Specific binding of cy5-[K4]hPP to hY4R was visualized by confocal microscopy.  The hY4R, the chimeric G protein Gqi5 and mitochondrially targeted apoaequorin were stably coexpressed in CHO cells.  Aequorin luminescence was quantified in a microplate reader and by a CCD camera.  By application of these methods 3-cyclohexyl-N-[(3-1H-imidazol-4-ylpropylamino)(imino)methyl]propanamide (UR-AK49)was discovered as the first nonpeptidic Y4R antagonist (pKi 4.17), a lead to be optimized in terms of potency and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8DMLWtdk7rrVg90H21EOLACvtfcHk0lgSscYg7AmJPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVGrs7fI&md5=3c280bdb617c42f53c0745a20bac73a9</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1080%2F10799890701505206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10799890701505206%26sid%3Dliteratum%253Aachs%26aulast%3DZiemek%26aufirst%3DR.%26aulast%3DSchneider%26aufirst%3DE.%26aulast%3DKraus%26aufirst%3DA.%26aulast%3DCabrele%26aufirst%3DC.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26aulast%3DBernhardt%26aufirst%3DG.%26aulast%3DBuschauer%26aufirst%3DA.%26atitle%3DDetermination%2520of%2520affinity%2520and%2520activity%2520of%2520ligands%2520at%2520the%2520human%2520neuropeptide%2520Y%2520Y4%2520receptor%2520by%2520flow%2520cytometry%2520and%2520aequorin%2520luminescence%26jtitle%3DJ.%2520Recept.%2520Signal%2520Transduction%26date%3D2007%26volume%3D27%26spage%3D217%26epage%3D233%26doi%3D10.1080%2F10799890701505206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rudolf, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberlein, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieland, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willim, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Entzeroth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wienen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doods, H. N.</span></span> <span> </span><span class="NLM_article-title">The first highly potent and selective non-peptide neuropeptide Y Y1 receptor antagonist: BIBP3226</span>. <i>Eur. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>271</i></span>,  <span class="NLM_fpage">R11</span>– <span class="NLM_lpage">R13</span>, <span class="refDoi"> DOI: 10.1016/0014-2999(94)90822-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2F0014-2999%2894%2990822-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=7705422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADyaK2MXjt1Sntrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=271&publication_year=1994&pages=R11-R13&author=K.+Rudolfauthor=W.+Eberleinauthor=W.+Engelauthor=H.+A.+Wielandauthor=K.+D.+Willimauthor=M.+Entzerothauthor=W.+Wienenauthor=A.+G.+Beck-Sickingerauthor=H.+N.+Doods&title=The+first+highly+potent+and+selective+non-peptide+neuropeptide+Y+Y1+receptor+antagonist%3A+BIBP3226&doi=10.1016%2F0014-2999%2894%2990822-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The first highly potent and selective non-peptide neuropeptide Y Y1 receptor antagonist: BIBP3226</span></div><div class="casAuthors">Rudolf, Klaus; Eberlein, Wolfgang; Engel, Wolfhard; Wieland, Heike A.; Willim, Klaus D.; Entzeroth, Michael; Wienen, Wolfgang; Beck-Sickinger, Annette G.; Doods, Henri N.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmacology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">271</span>
        (<span class="NLM_cas:issue">2/3</span>),
    <span class="NLM_cas:pages">R11-R13</span>CODEN:
                <span class="NLM_cas:coden">EJPHAZ</span>;
        ISSN:<span class="NLM_cas:issn">0014-2999</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The design and subsequent in vitro and in vivo biol. characterization of the first potent and selective non-peptide neuropeptide Y Y1 receptor antagonist, BIBP3226 (I; (R)-N2-(diphenylacetyl)-N-[(4- hydroxyphenyl)methyl]-argininamide) is reported.  BIBP3226 displaced 125I-labeled neuropeptide Y with high affinity (Ki = 7 nM) from the human neuropeptide Y Y1 receptor and proved to be highly selective.  BIBP3226 displayed potent antagonistic properties both in in vitro and in vivo models and thus represents the first selective non-peptide neuropeptide Y Y1 receptor antagonist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoy6B_rW0NxBLVg90H21EOLACvtfcHk0liYhUiX3U5WIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXjt1Sntrw%253D&md5=d0c87027fe726aacc2146433884d163a</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2F0014-2999%2894%2990822-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0014-2999%252894%252990822-2%26sid%3Dliteratum%253Aachs%26aulast%3DRudolf%26aufirst%3DK.%26aulast%3DEberlein%26aufirst%3DW.%26aulast%3DEngel%26aufirst%3DW.%26aulast%3DWieland%26aufirst%3DH.%2BA.%26aulast%3DWillim%26aufirst%3DK.%2BD.%26aulast%3DEntzeroth%26aufirst%3DM.%26aulast%3DWienen%26aufirst%3DW.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26aulast%3DDoods%26aufirst%3DH.%2BN.%26atitle%3DThe%2520first%2520highly%2520potent%2520and%2520selective%2520non-peptide%2520neuropeptide%2520Y%2520Y1%2520receptor%2520antagonist%253A%2520BIBP3226%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D1994%26volume%3D271%26spage%3DR11%26epage%3DR13%26doi%3D10.1016%2F0014-2999%2894%2990822-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wieland, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberlein, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolf, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doods, H. N.</span></span> <span> </span><span class="NLM_article-title">Subtype selectivity of the novel nonpeptide neuropeptide Y Y1 receptor antagonist BIBO 3304 and its effect on feeding in rodents</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">549</span>– <span class="NLM_lpage">555</span>, <span class="refDoi"> DOI: 10.1038/sj.bjp.0702084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1038%2Fsj.bjp.0702084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=9806339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADyaK1cXmvVSqt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=1998&pages=549-555&author=H.+A.+Wielandauthor=W.+Engelauthor=W.+Eberleinauthor=K.+Rudolfauthor=H.+N.+Doods&title=Subtype+selectivity+of+the+novel+nonpeptide+neuropeptide+Y+Y1+receptor+antagonist+BIBO+3304+and+its+effect+on+feeding+in+rodents&doi=10.1038%2Fsj.bjp.0702084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Subtype selectivity of the novel nonpeptide neuropeptide Y Y1 receptor antagonist BIBO 3304 and its effect on feeding in rodents</span></div><div class="casAuthors">Wieland, H. A.; Engel, W.; Eberlein, W.; Rudolf, K.; Doods, H. N.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">549-555</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Stockton Press</span>)
        </div><div class="casAbstract">The novel Y1-selective argininamide deriv. BIBO 3304 ((R)-N-[[4-(aminocarbonylaminomethyl)-phenyl]methyl]-N2-(diphenylacetyl)-argininamide trifluoroacetate) has been synthesized and was examd. for its subtype selectivity, its in vitro antagonistic properties and its food intake inhibitory properties.  BIBO 3304 displayed subnanomolar affinity for both the human and the rat Y1 receptor (IC50 values 0.38±0.06 nM and 0.72±0.42 nM, resp.).  The inactive enantiomer of BIBO 3304 (BIBO 3457) had low affinity for both the human and rat Y1 receptor subtype (IC50>1000 nM).  BIBO 3304 showed low affinity for the human Y2 receptor, human and rat Y4 receptor as well as for the human and rat Y5 receptor (IC50 values > 1000 nM).  30 μG BIBO 3304 administered into the paraventricular nucleus inhibited the feeding response induced by 1 μg NPY as well as the hyperphagia induced by a 24 h fast implying a role for Y1 receptors in NPY mediated feeding.  The inactive enantiomer had no effect.  BIBO 3304 inhibits neither the galanin nor the noradrenaline induced orexigenic response, but it blocked feeding behavior elicited by both [Leu31, Pro34]NPY and NPY (3-36) suggesting an interplay between different NPY receptor subtypes in feeding behavior.  The present study reveals that BIBO 3304 is a subtype selective nonpeptide antagonist with subnanomolar affinity for the Y1 receptor subtype that significantly inhibits food intake induced by application of NPY or by fasting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUdOSOrjTrcbVg90H21EOLACvtfcHk0lj-XhfEMSRFaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmvVSqt78%253D&md5=b7f588c39cb25b1adcd9dc0ae8f3f5a8</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0702084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0702084%26sid%3Dliteratum%253Aachs%26aulast%3DWieland%26aufirst%3DH.%2BA.%26aulast%3DEngel%26aufirst%3DW.%26aulast%3DEberlein%26aufirst%3DW.%26aulast%3DRudolf%26aufirst%3DK.%26aulast%3DDoods%26aufirst%3DH.%2BN.%26atitle%3DSubtype%2520selectivity%2520of%2520the%2520novel%2520nonpeptide%2520neuropeptide%2520Y%2520Y1%2520receptor%2520antagonist%2520BIBO%25203304%2520and%2520its%2520effect%2520on%2520feeding%2520in%2520rodents%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D1998%26volume%3D125%26spage%3D549%26epage%3D555%26doi%3D10.1038%2Fsj.bjp.0702084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D.</span></span> <span> </span><span class="NLM_article-title">The future of peptide-based drugs</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">136</span>– <span class="NLM_lpage">147</span>, <span class="refDoi"> DOI: 10.1111/cbdd.12055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1111%2Fcbdd.12055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=23253135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlt1yktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2013&pages=136-147&author=D.+J.+Craikauthor=D.+P.+Fairlieauthor=S.+Lirasauthor=D.+Price&title=The+future+of+peptide-based+drugs&doi=10.1111%2Fcbdd.12055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The future of peptide-based drugs</span></div><div class="casAuthors">Craik, David J.; Fairlie, David P.; Liras, Spiros; Price, David</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">136-147</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  The suite of currently used drugs can be divided into two categories - traditional "small mol." drugs with typical mol. wts. of <500 Da but with oral bioavailability, and much larger "biologics" typically >5000 Da that are not orally bioavailable and need to be delivered via injection.  Due to their small size, conventional small mol. drugs may suffer from reduced target selectivity that often ultimately manifests in human side-effects, whereas protein therapeutics tend to be exquisitely specific for their targets due to many more interactions with them, but this comes at a cost of low bioavailability, poor membrane permeability, and metabolic instability.  The time has now come to reinvestigate new drug leads that fit between these two mol. wt. extremes, with the goal of combining advantages of small mols. (cost, conformational restriction, membrane permeability, metabolic stability, oral bioavailability) with those of proteins (natural components, target specificity, high potency).  This article uses selected examples of peptides to highlight the importance of peptide drugs, some potential new opportunities for their exploitation, and some difficult challenges ahead in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlaEZbrPXLPbVg90H21EOLACvtfcHk0lhxC-IApiYsew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlt1yktA%253D%253D&md5=1e7e4b7e725541587fad855e6a28cadf</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12055%26sid%3Dliteratum%253Aachs%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DPrice%26aufirst%3DD.%26atitle%3DThe%2520future%2520of%2520peptide-based%2520drugs%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2013%26volume%3D81%26spage%3D136%26epage%3D147%26doi%3D10.1111%2Fcbdd.12055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adrian, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenberg, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besterman, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bloom, S. R.</span></span> <span> </span><span class="NLM_article-title">Pharmacokinetics of pancreatic polypeptide in man</span>. <i>Gut</i> <span class="NLM_year" style="font-weight: bold;">1978</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">907</span>– <span class="NLM_lpage">909</span>, <span class="refDoi"> DOI: 10.1136/gut.19.10.907</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1136%2Fgut.19.10.907" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=568585" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADyaE1MXkvV2ksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1978&pages=907-909&author=T.+E.+Adrianauthor=G.+R.+Greenbergauthor=H.+S.+Bestermanauthor=S.+R.+Bloom&title=Pharmacokinetics+of+pancreatic+polypeptide+in+man&doi=10.1136%2Fgut.19.10.907"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacokinetics of pancreatic polypeptide in man</span></div><div class="casAuthors">Adrian, T. E.; Greenberg, G. R.; Besterman, H. S.; Bloom, S. R.</div><div class="citationInfo"><span class="NLM_cas:title">Gut</span>
        (<span class="NLM_cas:date">1978</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">907-9</span>CODEN:
                <span class="NLM_cas:coden">GUTTAK</span>;
        ISSN:<span class="NLM_cas:issn">0017-5749</span>.
    </div><div class="casAbstract">Pure bovine pancreatic polypeptide  [59763-91-6] (1, 3, or 5 pmol/kg/min) infused i.v. for 60 min into healthy subjects produced mean plasma levels of 203, 575, and 930 pmol/L, resp.  The mean disappearance half-time on stopping the infusion was 6.9 min.  The metabolic clearance rate was 5.1 mL/kg/min and the apparent vol. of distribution was 51 nL/kg.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1BerHjBAmt7Vg90H21EOLACvtfcHk0lhxC-IApiYsew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1MXkvV2ksQ%253D%253D&md5=285d86d43cf7f83e4190f8e53daf0a9e</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1136%2Fgut.19.10.907&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fgut.19.10.907%26sid%3Dliteratum%253Aachs%26aulast%3DAdrian%26aufirst%3DT.%2BE.%26aulast%3DGreenberg%26aufirst%3DG.%2BR.%26aulast%3DBesterman%26aufirst%3DH.%2BS.%26aulast%3DBloom%26aufirst%3DS.%2BR.%26atitle%3DPharmacokinetics%2520of%2520pancreatic%2520polypeptide%2520in%2520man%26jtitle%3DGut%26date%3D1978%26volume%3D19%26spage%3D907%26epage%3D909%26doi%3D10.1136%2Fgut.19.10.907" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerald, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Criscione, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batzl-Hartmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaysse, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durkin, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laz, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linemeyer, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaffhauser, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitebread, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofbauer, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taber, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branchek, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinshank, R. L.</span></span> <span> </span><span class="NLM_article-title">A receptor subtype involved in neuropeptide-Y-induced food intake</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>382</i></span>,  <span class="NLM_fpage">168</span>– <span class="NLM_lpage">171</span>, <span class="refDoi"> DOI: 10.1038/382168a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1038%2F382168a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=8700207" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADyaK28XktlSkur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=382&publication_year=1996&pages=168-171&author=C.+Geraldauthor=M.+W.+Walkerauthor=L.+Criscioneauthor=E.+L.+Gustafsonauthor=C.+Batzl-Hartmannauthor=K.+E.+Smithauthor=P.+Vaysseauthor=M.+M.+Durkinauthor=T.+M.+Lazauthor=D.+L.+Linemeyerauthor=A.+O.+Schaffhauserauthor=S.+Whitebreadauthor=K.+G.+Hofbauerauthor=R.+I.+Taberauthor=T.+A.+Branchekauthor=R.+L.+Weinshank&title=A+receptor+subtype+involved+in+neuropeptide-Y-induced+food+intake&doi=10.1038%2F382168a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A receptor subtype involved in neuropeptide-Y-induced food intake</span></div><div class="casAuthors">Gerald, Christophe; Walker, Mary W.; Criscione, Leoluca; Gustafson, Eric L.; Batzl-Hartmann, Christine; Smith, Kelli E.; Vaysse, Pierre; Durkin, Margaret M.; Laz, Thomas M.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">382</span>
        (<span class="NLM_cas:issue">6587</span>),
    <span class="NLM_cas:pages">168-171</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Neuropeptide Y (NPY) is a powerful stimulant of food intake and is proposed to activate a hypothalamic 'feeding' receptor distinct from previously cloned Y-type receptors.  This receptor was first suggested to explain a feeding response to NPY and related peptides, including NPY2-36, that differed from their activities at the Y1 receptor.  Here the authors report the expression cloning of a novel Y-type receptor from rat hypothalamus, which the authors name Y5.  The complementary DNA encodes a 456-amino-acid protein with less than 35% overall identity to known Y-type receptors.  The mRNA is found primarily in the central nervous system, including the paraventricular nucleus of the hypothalamus.  The extent to which selected peptides can inhibit adenylate cyclase through the Y5 receptor and stimulate food intake in rats correspond well.  The authors' data support the idea that the Y5 receptor is the postulated 'feeding' receptor, and may provide a new method for the study and treatment of obesity and eating disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPe_-2lRIPvrVg90H21EOLACvtfcHk0liHPfHTLY6Q2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XktlSkur0%253D&md5=50601131172ab87888112111c412901b</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2F382168a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F382168a0%26sid%3Dliteratum%253Aachs%26aulast%3DGerald%26aufirst%3DC.%26aulast%3DWalker%26aufirst%3DM.%2BW.%26aulast%3DCriscione%26aufirst%3DL.%26aulast%3DGustafson%26aufirst%3DE.%2BL.%26aulast%3DBatzl-Hartmann%26aufirst%3DC.%26aulast%3DSmith%26aufirst%3DK.%2BE.%26aulast%3DVaysse%26aufirst%3DP.%26aulast%3DDurkin%26aufirst%3DM.%2BM.%26aulast%3DLaz%26aufirst%3DT.%2BM.%26aulast%3DLinemeyer%26aufirst%3DD.%2BL.%26aulast%3DSchaffhauser%26aufirst%3DA.%2BO.%26aulast%3DWhitebread%26aufirst%3DS.%26aulast%3DHofbauer%26aufirst%3DK.%2BG.%26aulast%3DTaber%26aufirst%3DR.%2BI.%26aulast%3DBranchek%26aufirst%3DT.%2BA.%26aulast%3DWeinshank%26aufirst%3DR.%2BL.%26atitle%3DA%2520receptor%2520subtype%2520involved%2520in%2520neuropeptide-Y-induced%2520food%2520intake%26jtitle%3DNature%26date%3D1996%26volume%3D382%26spage%3D168%26epage%3D171%26doi%3D10.1038%2F382168a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span> <span> </span><span class="NLM_article-title">Allosteric modulation and functional selectivity of G protein-coupled receptors</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">e237</span>– <span class="NLM_lpage">e243</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2012.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2Fj.ddtec.2012.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=24050274" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=e237-e243&author=Z.+G.+Gaoauthor=K.+A.+Jacobson&title=Allosteric+modulation+and+functional+selectivity+of+G+protein-coupled+receptors&doi=10.1016%2Fj.ddtec.2012.08.004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2012.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2012.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DZ.%2BG.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DAllosteric%2520modulation%2520and%2520functional%2520selectivity%2520of%2520G%2520protein-coupled%2520receptors%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2013%26volume%3D10%26spage%3De237%26epage%3De243%26doi%3D10.1016%2Fj.ddtec.2012.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wootten, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span> <span> </span><span class="NLM_article-title">Emerging paradigms in GPCR allostery: implications for drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">630</span>– <span class="NLM_lpage">644</span>, <span class="refDoi"> DOI: 10.1038/nrd4052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1038%2Fnrd4052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=23903222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1SjtbvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=630-644&author=D.+Woottenauthor=A.+Christopoulosauthor=P.+M.+Sexton&title=Emerging+paradigms+in+GPCR+allostery%3A+implications+for+drug+discovery&doi=10.1038%2Fnrd4052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging paradigms in GPCR allostery: implications for drug discovery</span></div><div class="casAuthors">Wootten, Denise; Christopoulos, Arthur; Sexton, Patrick M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">630-644</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Allosteric ligands bind to G protein-coupled receptors (GPCRs; also known as seven-transmembrane receptors) at sites that are distinct from the sites to which endogenous ligands bind.  The existence of allosteric ligands has enriched the ways in which the functions of GPCRs can be manipulated for potential therapeutic benefit, yet the complexity of their actions provides both challenges and opportunities for drug screening and development.  Converging avenues of research in areas such as biased signalling by allosteric ligands and the mechanisms by which allosteric ligands modulate the effects of diverse endogenous ligands have provided new insights into how interactions between allosteric ligands and GPCRs could be exploited for drug discovery.  These new findings have the potential to alter how screening for allosteric drugs is performed and may increase the chances of success in the development of allosteric modulators as clin. lead compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxRlmQUCUlsLVg90H21EOLACvtfcHk0lj-ftEKZAN4aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1SjtbvO&md5=59b74e5fbd5eeb832785dd879daa7c77</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnrd4052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4052%26sid%3Dliteratum%253Aachs%26aulast%3DWootten%26aufirst%3DD.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26atitle%3DEmerging%2520paradigms%2520in%2520GPCR%2520allostery%253A%2520implications%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D630%26epage%3D644%26doi%3D10.1038%2Fnrd4052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Graaf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span> <span> </span><span class="NLM_article-title">Structure and function of peptide-binding G protein-coupled receptors</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>429</i></span>,  <span class="NLM_fpage">2726</span>– <span class="NLM_lpage">2745</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2017.06.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2Fj.jmb.2017.06.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=28705763" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFygtrbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=429&publication_year=2017&pages=2726-2745&author=F.+Wuauthor=G.+Songauthor=C.+de+Graafauthor=R.+C.+Stevens&title=Structure+and+function+of+peptide-binding+G+protein-coupled+receptors&doi=10.1016%2Fj.jmb.2017.06.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and function of peptide-binding G protein-coupled receptors</span></div><div class="casAuthors">Wu, Fan; Song, Gaojie; de Graaf, Chris; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">429</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2726-2745</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  G protein-coupled receptors (GPCRs) are the largest family of cell surface receptors and are important human drug targets.  Of the 826 human GPCRs, 118 of them recognize endogenous peptide or protein ligands, and 30 of the 118 are targeted by approved drug mols., including the very high-profile class B glucagon-like peptide 1 receptor.  Here, we analyze the 21 exptl. detd. 3-dimensional structures of the known peptide-binding GPCRs in relation to the endogenous peptides and drug mols. that modulate their cell signaling processes.  The integrated analyses reveal that half of the marketed drugs and most of the drugs in clin. trials that interact with peptide GPCRs are small mols. with a wide range of binding modes distinct from those of large peptide ligands.  As we continue to collect addnl. data on these receptors from orthogonal approaches, including NMR and electron microscopy, we are beginning to understand how these receptors interact with their ligands at the mol. level and how improving the pharmacol. of GPCR signal transduction requires us to study these receptors using multiple biophys. techniques.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrn7cZECdBHg7Vg90H21EOLACvtfcHk0lhVYslFM6Tk8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFygtrbE&md5=0b1f5ad22516e40f7552218862023e13</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2017.06.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2017.06.022%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DF.%26aulast%3DSong%26aufirst%3DG.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DStructure%2520and%2520function%2520of%2520peptide-binding%2520G%2520protein-coupled%2520receptors%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2017%26volume%3D429%26spage%3D2726%26epage%3D2745%26doi%3D10.1016%2Fj.jmb.2017.06.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mäde, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babilon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jolly, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wanka, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellmann-Sickert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz Gimenez, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mörl, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurevich, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span> <span> </span><span class="NLM_article-title">Peptide modifications differentially alter G protein-coupled receptor internalization and signaling bias</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">10067</span>– <span class="NLM_lpage">10071</span>, <span class="refDoi"> DOI: 10.1002/anie.201403750</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1002%2Fanie.201403750" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A280%3ADC%252BC2cbmsVOrsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=10067-10071&author=V.+M%C3%A4deauthor=S.+Babilonauthor=N.+Jollyauthor=L.+Wankaauthor=K.+Bellmann-Sickertauthor=L.+E.+Diaz+Gimenezauthor=K.+M%C3%B6rlauthor=H.+M.+Coxauthor=V.+V.+Gurevichauthor=A.+G.+Beck-Sickinger&title=Peptide+modifications+differentially+alter+G+protein-coupled+receptor+internalization+and+signaling+bias&doi=10.1002%2Fanie.201403750"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide modifications differentially alter G protein-coupled receptor internalization and signaling bias</span></div><div class="casAuthors">Made Veronika; Babilon Stefanie; Jolly Navjeet; Wanka Lizzy; Bellmann-Sickert Kathrin; Diaz Gimenez Luis E; Morl Karin; Cox Helen M; Gurevich Vsevolod V; Beck-Sickinger Annette G</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">10067-71</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Although G protein-coupled receptors (GPCRs) are targeted by more clinically used drugs than any other type of protein, their ligand development is particularly challenging.  Humans have four neuropeptide Y receptors: hY1R and hY5R are orexigenic, while hY2R and hY4R are anorexigenic, and represent important anti-obesity drug targets.  We show for the first time that PEGylation and lipidation, chemical modifications that prolong the plasma half-lives of peptides, confer additional benefits.  Both modifications enhance pancreatic polypeptide preference for hY2R/hY4R over hY1R/hY5R.  Lipidation biases the ligand towards arrestin recruitment and internalization, whereas PEGylation confers the opposite bias.  These effects were independent of the cell system and modified residue.  We thus provide novel insights into the mode of action of peptide modifications and open innovative venues for generating peptide agonists with extended therapeutic potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ9ClV0SIs_P2BAiYWy6n3SfW6udTcc2earF7eu7MyiJbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cbmsVOrsg%253D%253D&md5=24666ef122d28ca19ca72fada334feaf</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2Fanie.201403750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201403750%26sid%3Dliteratum%253Aachs%26aulast%3DM%25C3%25A4de%26aufirst%3DV.%26aulast%3DBabilon%26aufirst%3DS.%26aulast%3DJolly%26aufirst%3DN.%26aulast%3DWanka%26aufirst%3DL.%26aulast%3DBellmann-Sickert%26aufirst%3DK.%26aulast%3DDiaz%2BGimenez%26aufirst%3DL.%2BE.%26aulast%3DM%25C3%25B6rl%26aufirst%3DK.%26aulast%3DCox%26aufirst%3DH.%2BM.%26aulast%3DGurevich%26aufirst%3DV.%2BV.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26atitle%3DPeptide%2520modifications%2520differentially%2520alter%2520G%2520protein-coupled%2520receptor%2520internalization%2520and%2520signaling%2520bias%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D10067%26epage%3D10071%26doi%3D10.1002%2Fanie.201403750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Böhme, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stichel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walther, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mörl, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span> <span> </span><span class="NLM_article-title">Agonist induced receptor internalization of neuropeptide Y receptor subtypes depends on third intracellular loop and C-terminus</span>. <i>Cell. Signalling</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1740</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1016/j.cellsig.2008.05.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2Fj.cellsig.2008.05.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=18598760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A280%3ADC%252BD1crjtFSltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2008&pages=1740-1749&author=I.+B%C3%B6hmeauthor=J.+Stichelauthor=C.+Waltherauthor=K.+M%C3%B6rlauthor=A.+G.+Beck-Sickinger&title=Agonist+induced+receptor+internalization+of+neuropeptide+Y+receptor+subtypes+depends+on+third+intracellular+loop+and+C-terminus&doi=10.1016%2Fj.cellsig.2008.05.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Agonist induced receptor internalization of neuropeptide Y receptor subtypes depends on third intracellular loop and C-terminus</span></div><div class="casAuthors">Bohme Ilka; Stichel Jan; Walther Cornelia; Morl Karin; Beck-Sickinger Annette G</div><div class="citationInfo"><span class="NLM_cas:title">Cellular signalling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1740-9</span>
        ISSN:<span class="NLM_cas:issn">0898-6568</span>.
    </div><div class="casAbstract">Agonist stimulation of G-protein coupled receptors (GPCRs) results in the redistribution of the receptor from the cell surface into intracellular compartments through the process of endocytosis.  Monitoring ligand-mediated internalization of GPCRs in living cells has become experimentally accessible by applying fluorescent reagents and fluorescence microscopy.  By using cell lines that transiently, stably or endogenously express the human Y receptor (hYR) subtypes hY(1)R, hY(2)R, hY(4)R and hY(5)R and differently fluorescently tagged receptor proteins we were able to unravel further details concerning the internalization behavior of this multi-receptor/multi-ligand system.  For the first time we could show that also the hY(2)R is internalized with a rate which is comparable to the hY(1)R and the hY(4)R.  In contrast, the hY(5)R was internalized much slower and the rate remained unaffected by co-expression with other hYR subtypes.  Furthermore receptor subtype co-expressing cells and selectively binding peptides revealed a receptor subtype selective internalization.  By using novel hY(5)/hY(2) receptor chimera the receptor subtype dependent differences in hY receptor internalization could be identified on a molecular level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS1_oavs96kyXO4gDb7JggCfW6udTcc2earF7eu7MyiJbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1crjtFSltQ%253D%253D&md5=5eb7dfb86e75c35b92b95b5ee3775f39</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.cellsig.2008.05.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellsig.2008.05.017%26sid%3Dliteratum%253Aachs%26aulast%3DB%25C3%25B6hme%26aufirst%3DI.%26aulast%3DStichel%26aufirst%3DJ.%26aulast%3DWalther%26aufirst%3DC.%26aulast%3DM%25C3%25B6rl%26aufirst%3DK.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26atitle%3DAgonist%2520induced%2520receptor%2520internalization%2520of%2520neuropeptide%2520Y%2520receptor%2520subtypes%2520depends%2520on%2520third%2520intracellular%2520loop%2520and%2520C-terminus%26jtitle%3DCell.%2520Signalling%26date%3D2008%26volume%3D20%26spage%3D1740%26epage%3D1749%26doi%3D10.1016%2Fj.cellsig.2008.05.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kostenis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degtyarev, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conklin, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wess, J.</span></span> <span> </span><span class="NLM_article-title">The N-terminal extension of Gαq is critical for constraining the selectivity of receptor coupling</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>272</i></span>,  <span class="NLM_fpage">19107</span>– <span class="NLM_lpage">19110</span>, <span class="refDoi"> DOI: 10.1074/jbc.272.31.19107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1074%2Fjbc.272.31.19107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=9235898" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADyaK2sXltFSnuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=1997&pages=19107-19110&author=E.+Kostenisauthor=M.+Y.+Degtyarevauthor=B.+R.+Conklinauthor=J.+Wess&title=The+N-terminal+extension+of+G%CE%B1q+is+critical+for+constraining+the+selectivity+of+receptor+coupling&doi=10.1074%2Fjbc.272.31.19107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">The N-terminal extension of Gαq is critical for constraining the selectivity of receptor coupling</span></div><div class="casAuthors">Kostenis, Evi; Degtyarev, Michael Y.; Conklin, Bruce R.; Wess, Jurgen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">272</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">19107-19110</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Characteristically, an individual member of the superfamily of G protein-coupled receptors can interact only with a limited no. of the many structurally closely related G protein heterotrimers that are expressed within a cell.  Interestingly, the N termini of two G protein α subunits, Gαq and Gα11, differ from those of other α subunits in that they display a unique, highly conserved six-amino extension.  To test the hypothesis that this sequence element is crit. for proper receptor recognition, we prepd. a Gαq deletion mutant (-6q) lacking these first six amino acids.  The -6q construct (or wild type Gαq as a control) was coexpressed (in COS-7 cells) with several different Gi/o- or Gs-coupled receptors, and ligand-induced increases in inositol phosphate prodn. were detd. as a measure of G protein activation.  Whereas these receptors did not efficiently interact with wild type Gαq, most of them gained the ability to productively coupled to -6q.  Addnl. expts. indicated that the obsd. functional promiscuity of -6q is not due to overexpression (as compared with wild type Gαq) or a lack of palmitoylation.  We conclude that the N-terminal extension characteristic for Gαq/11 proteins is crit. for constraining the receptor coupling selectivity of these subunits, indicative of a novel mechanism by which the fidelity of receptor-G protein interactions can be regulated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzkDUJrMf9qbVg90H21EOLACvtfcHk0lhV5_VfngyAIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXltFSnuro%253D&md5=77e697d194898507c9cc27b508225279</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1074%2Fjbc.272.31.19107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.272.31.19107%26sid%3Dliteratum%253Aachs%26aulast%3DKostenis%26aufirst%3DE.%26aulast%3DDegtyarev%26aufirst%3DM.%2BY.%26aulast%3DConklin%26aufirst%3DB.%2BR.%26aulast%3DWess%26aufirst%3DJ.%26atitle%3DThe%2520N-terminal%2520extension%2520of%2520G%25CE%25B1q%2520is%2520critical%2520for%2520constraining%2520the%2520selectivity%2520of%2520receptor%2520coupling%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1997%26volume%3D272%26spage%3D19107%26epage%3D19110%26doi%3D10.1074%2Fjbc.272.31.19107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bryksin, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumura, I.</span></span> <span> </span><span class="NLM_article-title">Overlap extension PCR cloning: a simple and reliable way to create recombinant plasmids</span>. <i>BioTechniques</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">463</span>– <span class="NLM_lpage">465</span>, <span class="refDoi"> DOI: 10.2144/000113418</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.2144%2F000113418" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=20569222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjtVeisr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2010&pages=463-465&author=A.+V.+Bryksinauthor=I.+Matsumura&title=Overlap+extension+PCR+cloning%3A+a+simple+and+reliable+way+to+create+recombinant+plasmids&doi=10.2144%2F000113418"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Overlap extension PCR cloning: a simple and reliable way to create recombinant plasmids</span></div><div class="casAuthors">Bryksin, Anton V.; Matsumura, Ichiro</div><div class="citationInfo"><span class="NLM_cas:title">BioTechniques</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">463-465</span>CODEN:
                <span class="NLM_cas:coden">BTNQDO</span>;
        ISSN:<span class="NLM_cas:issn">0736-6205</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">Here we describe a straightforward, efficient, and reliable way to clone an insert of choice into a plasmid of choice without restriction endonucleases or T4 DNA ligase.  Chimeric primers contg. plasmid sequence at the 5' ends and insert sequence at the 3' ends were used to PCR-amplify insertion sequences of various sizes, namely the genes for GFP (g/f), β-D-glucuronidase (gusA), and β-galactosidase (lacZ), as well as the entire luxABCDE operon.  These inserts were employed as mega-primers in a second PCR with a circular plasmid template.  The original plasmid templates were then destroyed in restriction digests with DpnI, and the overlap extension PCR products were used to transform competent Escherichia coli cells.  Phusion DNA polymerase was used for the amplification and fusion reactions, so both reactions were easy to monitor and optimize.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQWvKNOXGKP7Vg90H21EOLACvtfcHk0lghJbzsS51ljg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjtVeisr8%253D&md5=22658b78ee0555be56a20beff6b4225b</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.2144%2F000113418&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2144%252F000113418%26sid%3Dliteratum%253Aachs%26aulast%3DBryksin%26aufirst%3DA.%2BV.%26aulast%3DMatsumura%26aufirst%3DI.%26atitle%3DOverlap%2520extension%2520PCR%2520cloning%253A%2520a%2520simple%2520and%2520reliable%2520way%2520to%2520create%2520recombinant%2520plasmids%26jtitle%3DBioTechniques%26date%3D2010%26volume%3D48%26spage%3D463%26epage%3D465%26doi%3D10.2144%2F000113418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ho, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horton, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pullen, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pease, L. R.</span></span> <span> </span><span class="NLM_article-title">Site-directed mutagenesis by overlap extension using the polymerase chain reaction</span>. <i>Gene</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">59</span>, <span class="refDoi"> DOI: 10.1016/0378-1119(89)90358-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2F0378-1119%2889%2990358-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=2744487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADyaL1MXktVaitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=1989&pages=51-59&author=S.+N.+Hoauthor=H.+D.+Huntauthor=R.+M.+Hortonauthor=J.+K.+Pullenauthor=L.+R.+Pease&title=Site-directed+mutagenesis+by+overlap+extension+using+the+polymerase+chain+reaction&doi=10.1016%2F0378-1119%2889%2990358-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Site-directed mutagenesis by overlap extension using the polymerase chain reaction</span></div><div class="casAuthors">Ho, Steffan N.; Hunt, Henry D.; Horton, Robert M.; Pullen, Jeffrey K.; Pease, Larry R.</div><div class="citationInfo"><span class="NLM_cas:title">Gene</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">51-9</span>CODEN:
                <span class="NLM_cas:coden">GENED6</span>;
        ISSN:<span class="NLM_cas:issn">0378-1119</span>.
    </div><div class="casAbstract">Overlap extension represents a new approach to genetic engineering.  Complementary oligodeoxyribonucleotide (oligo) primers and the polymerase chain reaction are used to generate two DNA fragments having overlapping ends.  These fragments are combined in a subsequent fusion reaction in which the overlapping ends anneal, allowing the 3' overlap of each strand to serve as a primer for the 3' extension of the complementary strand.  The resulting fusion product is amplified further by PCR.  Specific alterations in the nucleotide (nt) sequence can be introduced by incorporating nucleotide changes into the overlapping oligo primers.  Using this technique of site-directed mutagenesis, 3 variants of a mouse major histocompatibility complex class-I gene have been generated, cloned and analyzed.  Screening of mutant clones revealed at least a 98% efficiency of mutagenesis.  All clones sequenced contained the desired mutations, and a low frequency of random substitution estd. to occur at approx. 1 in 400 nt was detected.  This method represents a significant improvement over std. methods of site-directed mutagenesis because it is much faster, simpler and approaches 100% efficiency in the generation of mutant product.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqebn9qfaVP9rVg90H21EOLACvtfcHk0lghJbzsS51ljg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXktVaitrk%253D&md5=a2587cbdaec79a12306c022825570e19</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2F0378-1119%2889%2990358-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0378-1119%252889%252990358-2%26sid%3Dliteratum%253Aachs%26aulast%3DHo%26aufirst%3DS.%2BN.%26aulast%3DHunt%26aufirst%3DH.%2BD.%26aulast%3DHorton%26aufirst%3DR.%2BM.%26aulast%3DPullen%26aufirst%3DJ.%2BK.%26aulast%3DPease%26aufirst%3DL.%2BR.%26atitle%3DSite-directed%2520mutagenesis%2520by%2520overlap%2520extension%2520using%2520the%2520polymerase%2520chain%2520reaction%26jtitle%3DGene%26date%3D1989%26volume%3D77%26spage%3D51%26epage%3D59%26doi%3D10.1016%2F0378-1119%2889%2990358-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bradford, M. M.</span></span> <span> </span><span class="NLM_article-title">A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1976</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">248</span>– <span class="NLM_lpage">254</span>, <span class="refDoi"> DOI: 10.1016/0003-2697(76)90527-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2F0003-2697%2876%2990527-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=942051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADyaE28XksVehtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=1976&pages=248-254&author=M.+M.+Bradford&title=A+rapid+and+sensitive+method+for+the+quantitation+of+microgram+quantities+of+protein+utilizing+the+principle+of+protein-dye+binding&doi=10.1016%2F0003-2697%2876%2990527-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding</span></div><div class="casAuthors">Bradford, Marion M.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">248-54</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">A protein detn. method that involves the binding of coomassie Brilliant Blue G 250 to protein is described.  The binding of the dye to protein causes a shift in the absorption max. of the dye from 465 to 595 nm, and it is the increase in absorption at 595 nm that is monitored.  This assay is very reproducible and rapid with the dye binding process virtually complete in ∼ 2 min with good color stability for 1 hr.  There is little or no interference from cations such as Na+ or K+ nor from carbohydrates such as sucrose.  A small amt. of color is developed in the presence of strongly alk. buffering agents, but the assay may be run accurately by the use of proper buffer controls.  The only components found to give excessive interfering color in the assay are relatively large amts. of detergents such as Na dodecyl sulfate, Triton X 100, and commercial glassware detergents.  Interference by small amts. of detergent may be eliminated by the use of proper control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN_T_d0trcNrVg90H21EOLACvtfcHk0lghJbzsS51ljg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28XksVehtrY%253D&md5=43f388a493becc4193c7a49d9de0e1db</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2F0003-2697%2876%2990527-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-2697%252876%252990527-3%26sid%3Dliteratum%253Aachs%26aulast%3DBradford%26aufirst%3DM.%2BM.%26atitle%3DA%2520rapid%2520and%2520sensitive%2520method%2520for%2520the%2520quantitation%2520of%2520microgram%2520quantities%2520of%2520protein%2520utilizing%2520the%2520principle%2520of%2520protein-dye%2520binding%26jtitle%3DAnal.%2520Biochem.%26date%3D1976%26volume%3D72%26spage%3D248%26epage%3D254%26doi%3D10.1016%2F0003-2697%2876%2990527-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sievers, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilm, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dineen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karplus, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McWilliam, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remmert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Söding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higgins, D. G.</span></span> <span> </span><span class="NLM_article-title">Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega</span>. <i>Mol. Syst. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">539</span> <span class="refDoi"> DOI: 10.1038/msb.2011.75</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1038%2Fmsb.2011.75" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=21988835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A280%3ADC%252BC3MbgtlantA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&author=F.+Sieversauthor=A.+Wilmauthor=D.+Dineenauthor=T.+J.+Gibsonauthor=K.+Karplusauthor=W.+Liauthor=R.+Lopezauthor=H.+McWilliamauthor=M.+Remmertauthor=J.+S%C3%B6dingauthor=J.+D.+Thompsonauthor=D.+G.+Higgins&title=Fast%2C+scalable+generation+of+high-quality+protein+multiple+sequence+alignments+using+Clustal+Omega&doi=10.1038%2Fmsb.2011.75"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega</span></div><div class="casAuthors">Sievers Fabian; Wilm Andreas; Dineen David; Gibson Toby J; Karplus Kevin; Li Weizhong; Lopez Rodrigo; McWilliam Hamish; Remmert Michael; Soding Johannes; Thompson Julie D; Higgins Desmond G</div><div class="citationInfo"><span class="NLM_cas:title">Molecular systems biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">539</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Multiple sequence alignments are fundamental to many sequence analysis methods.  Most alignments are computed using the progressive alignment heuristic.  These methods are starting to become a bottleneck in some analysis pipelines when faced with data sets of the size of many thousands of sequences.  Some methods allow computation of larger data sets while sacrificing quality, and others produce high-quality alignments, but scale badly with the number of sequences.  In this paper, we describe a new program called Clustal Omega, which can align virtually any number of protein sequences quickly and that delivers accurate alignments.  The accuracy of the package on smaller test cases is similar to that of the high-quality aligners.  On larger data sets, Clustal Omega outperforms other packages in terms of execution time and quality.  Clustal Omega also has powerful features for adding sequences to and exploiting information in existing alignments, making use of the vast amount of precomputed information in public databases like Pfam.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQD21C9zX5warjnyZkbE8yXfW6udTcc2eaBk0Y961mwE7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MbgtlantA%253D%253D&md5=27dd8fac22447528e6d09e5fd1066133</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fmsb.2011.75&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmsb.2011.75%26sid%3Dliteratum%253Aachs%26aulast%3DSievers%26aufirst%3DF.%26aulast%3DWilm%26aufirst%3DA.%26aulast%3DDineen%26aufirst%3DD.%26aulast%3DGibson%26aufirst%3DT.%2BJ.%26aulast%3DKarplus%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLopez%26aufirst%3DR.%26aulast%3DMcWilliam%26aufirst%3DH.%26aulast%3DRemmert%26aufirst%3DM.%26aulast%3DS%25C3%25B6ding%26aufirst%3DJ.%26aulast%3DThompson%26aufirst%3DJ.%2BD.%26aulast%3DHiggins%26aufirst%3DD.%2BG.%26atitle%3DFast%252C%2520scalable%2520generation%2520of%2520high-quality%2520protein%2520multiple%2520sequence%2520alignments%2520using%2520Clustal%2520Omega%26jtitle%3DMol.%2520Syst.%2520Biol.%26date%3D2011%26volume%3D7%26doi%3D10.1038%2Fmsb.2011.75" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bender, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cisneros, A.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duran, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokits, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sangha, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauer, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sevy, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sliwoski, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheehan, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiMaio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meiler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moretti, R.</span></span> <span> </span><span class="NLM_article-title">Protocols for molecular modeling with Rosetta3 and RosettaScripts</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">4748</span>– <span class="NLM_lpage">4763</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.6b00444</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.6b00444" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1ynt73M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=4748-4763&author=B.+J.+Benderauthor=A.+Cisnerosauthor=A.+M.+Duranauthor=J.+A.+Finnauthor=D.+Fuauthor=A.+D.+Lokitsauthor=B.+K.+Muellerauthor=A.+K.+Sanghaauthor=M.+F.+Sauerauthor=A.+M.+Sevyauthor=G.+Sliwoskiauthor=J.+H.+Sheehanauthor=F.+DiMaioauthor=J.+Meilerauthor=R.+Moretti&title=Protocols+for+molecular+modeling+with+Rosetta3+and+RosettaScripts&doi=10.1021%2Facs.biochem.6b00444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Protocols for Molecular Modeling with Rosetta3 and RosettaScripts</span></div><div class="casAuthors">Bender, Brian J.; Cisneros, Alberto; Duran, Amanda M.; Finn, Jessica A.; Fu, Darwin; Lokits, Alyssa D.; Mueller, Benjamin K.; Sangha, Amandeep K.; Sauer, Marion F.; Sevy, Alexander M.; Sliwoski, Gregory; Sheehan, Jonathan H.; DiMaio, Frank; Meiler, Jens; Moretti, Rocco</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">4748-4763</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Previously, we published an article providing an overview of the Rosetta suite of biomacromol. modeling software and a series of step-by-step tutorials [Kaufmann, K. W. et al. (2010) Biochem. 49, 2987-2998].  The overwhelming pos. response we received to this publication motivates us to here share the next iteration of these tutorials which feature de novo folding, comparative modeling, loop construction, protein docking, small mol. docking, and protein design.  This updated and expanded set of tutorials is needed, as since 2010 Rosetta has been fully re into an object-oriented protein modeling program Rosetta3.  Notable improvements include a substantially improved energy function, an XML-like language termed "RosettaScripts" for flexibly specifying modeling task, new anal. tools, the addn. of the TopologyBroker to control conformational sampling, and support for multiple templates in comparative modeling.  Rosetta's ability to model systems with sym. proteins, membrane proteins, non-canonical amino acids and RNA has also been greatly expanded and improved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCdqi-g3EfybVg90H21EOLACvtfcHk0livY-Ev95NVBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1ynt73M&md5=c540bd135fdf2753665249347e1779a8</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.6b00444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.6b00444%26sid%3Dliteratum%253Aachs%26aulast%3DBender%26aufirst%3DB.%2BJ.%26aulast%3DCisneros%26aufirst%3DA.%26aulast%3DDuran%26aufirst%3DA.%2BM.%26aulast%3DFinn%26aufirst%3DJ.%2BA.%26aulast%3DFu%26aufirst%3DD.%26aulast%3DLokits%26aufirst%3DA.%2BD.%26aulast%3DMueller%26aufirst%3DB.%2BK.%26aulast%3DSangha%26aufirst%3DA.%2BK.%26aulast%3DSauer%26aufirst%3DM.%2BF.%26aulast%3DSevy%26aufirst%3DA.%2BM.%26aulast%3DSliwoski%26aufirst%3DG.%26aulast%3DSheehan%26aufirst%3DJ.%2BH.%26aulast%3DDiMaio%26aufirst%3DF.%26aulast%3DMeiler%26aufirst%3DJ.%26aulast%3DMoretti%26aufirst%3DR.%26atitle%3DProtocols%2520for%2520molecular%2520modeling%2520with%2520Rosetta3%2520and%2520RosettaScripts%26jtitle%3DBiochemistry%26date%3D2016%26volume%3D55%26spage%3D4748%26epage%3D4763%26doi%3D10.1021%2Facs.biochem.6b00444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, N.</span>; <span class="NLM_string-name">Woetzel, N.</span>; <span class="NLM_string-name">Meiler, J.</span></span> In  <i>Bcl::Cluster: A Method for Clustering Biological Molecules Coupled with Visualization in the Pymol Molecular Graphics System</i>, 2011 IEEE 1st International Conference on Computational Advances in Bio and Medical Sciences (ICCABS), <span class="NLM_year">2011</span>; pp  <span class="NLM_fpage">13</span>– <span class="NLM_lpage">18</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&pages=13-18&author=N.+Alexander&author=N.+Woetzel&author=J.+Meiler&title=Bcl%3A%3ACluster%3A+A+Method+for+Clustering+Biological+Molecules+Coupled+with+Visualization+in+the+Pymol+Molecular+Graphics+System"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DAlexander%26aufirst%3DN.%26jtitle%3DBcl%253A%253ACluster%253A%2520A%2520Method%2520for%2520Clustering%2520Biological%2520Molecules%2520Coupled%2520with%2520Visualization%2520in%2520the%2520Pymol%2520Molecular%2520Graphics%2520System%26date%3D2011%26spage%3D13%26epage%3D18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kothiwale, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendenhall, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meiler, J.</span></span> <span> </span><span class="NLM_article-title">BCL::Conf: small molecule conformational sampling using a knowledge based rotamer library</span>. <i>J. Cheminf.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">47</span> <span class="refDoi"> DOI: 10.1186/s13321-015-0095-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1186%2Fs13321-015-0095-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC28Xos1ejtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&author=S.+Kothiwaleauthor=J.+L.+Mendenhallauthor=J.+Meiler&title=BCL%3A%3AConf%3A+small+molecule+conformational+sampling+using+a+knowledge+based+rotamer+library&doi=10.1186%2Fs13321-015-0095-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">BCL::CONF: small molecule conformational sampling using a knowledge based rotamer library</span></div><div class="casAuthors">Kothiwale, Sandeepkumar; Mendenhall, Jeffrey L.; Meiler, Jens</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">47/1-47/15</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">The interaction of a small mol. with a protein target depends on its ability to adopt a three-dimensional structure that is complementary.  Therefore, complete and rapid prediction of the conformational space a small mol. can sample is crit. for both structure- and ligand-based drug discovery algorithms such as small mol. docking or three-dimensional quant. structure-activity relationships.  Here we have derived a database of small mol. fragments frequently sampled in exptl. structures within the Cambridge Structure Database and the Protein Data Bank.  Likely conformations of these fragments are stored as 'rotamers' in analogy to amino acid side chain rotamer libraries used for rapid sampling of protein conformational space.  Explicit fragments take into account correlations between multiple torsion bonds and effect of substituents on torsional profiles.  A conformational ensemble for small mols. can then be generated by recombining fragment rotamers with a Monte Carlo search strategy.  BCL::CONF was benchmarked against other conformer generator methods including CONFGEN, MOE, OMEGA and RDKIT in its ability to recover exptl. detd. protein bound conformations of small mols., diversity of conformational ensembles, and sampling rate.  BCL::CONF recovers at least one conformation with a root mean square deviation of 2 Å or better to the exptl. structure for 99 % of the small mols. in the VERNALIS benchmark dataset.  The 'rotamer' approach will allow integration of BCL::CONF into resp. computational biol. programs such as ROSETTA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroCP70my1ks7Vg90H21EOLACvtfcHk0livY-Ev95NVBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xos1ejtLY%253D&md5=a08c0fd16b2b4a526a41f9ac6f6559d0</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1186%2Fs13321-015-0095-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-015-0095-1%26sid%3Dliteratum%253Aachs%26aulast%3DKothiwale%26aufirst%3DS.%26aulast%3DMendenhall%26aufirst%3DJ.%2BL.%26aulast%3DMeiler%26aufirst%3DJ.%26atitle%3DBCL%253A%253AConf%253A%2520small%2520molecule%2520conformational%2520sampling%2520using%2520a%2520knowledge%2520based%2520rotamer%2520library%26jtitle%3DJ.%2520Cheminf.%26date%3D2015%26volume%3D7%26doi%3D10.1186%2Fs13321-015-0095-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moretti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allison, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meiler, J.</span></span> <span> </span><span class="NLM_article-title">Rosetta and the design of ligand binding sites</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>1414</i></span>,  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">62</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-3569-7_4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1007%2F978-1-4939-3569-7_4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=27094285" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpsFWrsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1414&publication_year=2016&pages=47-62&author=R.+Morettiauthor=B.+J.+Benderauthor=B.+Allisonauthor=J.+Meiler&title=Rosetta+and+the+design+of+ligand+binding+sites&doi=10.1007%2F978-1-4939-3569-7_4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Rosetta and the Design of Ligand Binding Sites</span></div><div class="casAuthors">Moretti, Rocco; Bender, Brian J.; Allison, Brittany; Meiler, Jens</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">1414</span>
        (<span class="NLM_cas:issue">Computational Design of Ligand Binding Proteins</span>),
    <span class="NLM_cas:pages">47-62</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Proteins that bind small mols. (ligands) can be used as biosensors, signal modulators, and sequestering agents.  When naturally occurring proteins for a particular target ligand are not available, artificial proteins can be computationally designed.  We present a protocol based on RosettaLigand to redesign an existing protein pocket to bind a target ligand.  Starting with a protein structure and the structure of the ligand, Rosetta can optimize both the placement of the ligand in the pocket and the identity and conformation of the surrounding sidechains, yielding proteins that bind the target compd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpGdSJp5nS2bVg90H21EOLACvtfcHk0livY-Ev95NVBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpsFWrsrk%253D&md5=5bd1f176ef8793b183ea5a926307204c</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-3569-7_4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-3569-7_4%26sid%3Dliteratum%253Aachs%26aulast%3DMoretti%26aufirst%3DR.%26aulast%3DBender%26aufirst%3DB.%2BJ.%26aulast%3DAllison%26aufirst%3DB.%26aulast%3DMeiler%26aufirst%3DJ.%26atitle%3DRosetta%2520and%2520the%2520design%2520of%2520ligand%2520binding%2520sites%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2016%26volume%3D1414%26spage%3D47%26epage%3D62%26doi%3D10.1007%2F978-1-4939-3569-7_4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kenakin, T. P.</span></span> <span> </span><span class="NLM_article-title">Allosteric Drug Antagonism. A Pharmacology Primer—Theory</span>.  <i>Applications, and Methods</i>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_publisher-name">Elsevier Academic Press</span>: <span class="NLM_publisher-loc">Amsterdam, Boston</span>, <span class="NLM_year">2006</span>; pp  <span class="NLM_fpage">143</span>– <span class="NLM_lpage">145</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&pages=143-145&author=T.+P.+Kenakin&title=Applications%2C+and+Methods"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DKenakin%26aufirst%3DT.%2BP.%26atitle%3DAllosteric%2520Drug%2520Antagonism.%2520A%2520Pharmacology%2520Primer%25E2%2580%2594Theory%26btitle%3DApplications%252C%2520and%2520Methods%26pub%3DElsevier%2520Academic%2520Press%26date%3D2006%26spage%3D143%26epage%3D145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/medium/jm0c02000_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02000&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/medium/jm0c02000_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Characterization of VU0637120. (a) Structure of VU0637120, chiral C-atom is labeled by an asterisk. (b) VU0637120 effectively decreases Y<sub>4</sub>R activation by PP in Ca<sup>2+</sup> flux assays using stably transfected COS7_hY<sub>4</sub>R-eYFP_Δ6Gα<sub>qi4myr</sub> cells. Data represent mean ± standard error of the mean (SEM) from <i>N</i> ≥ 2 independent experiments. VU0637120 effect on PP potency (α) and efficacy (β) was quantified using an operational model of allosterism.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> (c) Effect of increasing concentrations of VU0637120 and its enantiopure isomers on Y<sub>4</sub>R activation in response to a submaximal PP (PP EC<sub>80</sub>) concentration in COS7_hY<sub>4</sub>R-eYFP_Δ6Gα<sub>qi4myr</sub> cells. Data are shown as mean ± SEM from at least <i>N</i> ≥ 2 independent experiments. (d) Investigation of arrestin3 (arr3) recruitment to Y<sub>4</sub>R in response to 100 nM PP in the presence of 30 μM of VU0637120. Kinetic bioluminescence resonance energy transfer (BRET) experiments were performed in HEK293 cells transiently expressing Y<sub>4</sub>R-Rluc8 and venus-arr3. Rate constant <i>k</i> of arr3 recruitment is summarized as a bar graph. Data represent the mean ± SEM from <i>N</i> ≥ 4 independent experiments. Statistical analysis was performed using unpaired <i>t</i>-test, ***<i>P</i> <0.001. (e) VU0637120 reduces the specific binding of <sup>125</sup>I-PP to Y<sub>4</sub>R membrane preparations. Binding was investigated in competition binding assays with 60 or 180 pM <sup>125</sup>I-PP. Unspecific binding was determined in the presence of 1 μM PP. Data represent the mean ± SEM from <i>N</i> ≥ 4 independent experiments. (f) PP competition binding in the presence of VU0637120 using <sup>125</sup>I-PP (60 pM) at Y<sub>4</sub>R membrane preparations. Data are shown as mean ± SEM from <i>N</i> ≥ 3 independent experiments. (g) VU0637120 shows high selectivity to the Y<sub>4</sub>R subtype. Activation of Y<sub>1</sub>R, Y<sub>2</sub>R, and Y<sub>5</sub>R by NPY and Y<sub>4</sub>R by PP was investigated in the presence of 30 μM of VU0637120. Receptor signaling was measured in Ca<sup>2+</sup> flux assays using COS7 cells stably expressing one specific Y receptor (Y<sub>1,2,4,5</sub>R-eYFP) and the chimeric G protein Δ6Gα<sub>qi4myr</sub>. Data represent the mean ± SEM from <i>N</i> ≥ 3 independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02000&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/medium/jm0c02000_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0008.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of (<i>S</i>)<i>-</i> and (<i>R</i>)-VU0637120<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02000&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>N</i>,<i>N</i>-diisopropylethylamine (DIEA), 1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate (HATU), dimethylformamide (DMF), 86% (<b>4</b>), 85% (<b>5</b>); (b) HCl, dioxane, CH<sub>2</sub>Cl<sub>2</sub>, 89% (<b>6</b>), 93% (<b>7</b>); (c) NEt<sub>3</sub>, MeSO<sub>2</sub>Cl, CH<sub>2</sub>Cl<sub>2</sub>, 60% ((<i>S</i>)<i>-</i>VU0637120), 59% ((<i>R</i>)-VU0637120).</p></p></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/medium/jm0c02000_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Investigation of VU0637120 effect ex vivo. (a) (<i>S</i>)<i>-</i>VU0637120 inhibits optimal rPP (30 nM)-mediated inhibition of ion transport in a concentration-dependent manner, while 30 μM of (<i>R</i>)-VU0637120 has no effect on the Y<sub>4</sub>R agonist in the mucosal preparations of mouse descending colon. (b) Subsequent PYY signals (10 nM) are resistant to either enantiomer of VU0637120 at the different concentrations shown, compared to the DMSO control. Ex vivo data represent the mean ± SEM from <i>N</i> = 5 experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02000&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/medium/jm0c02000_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Investigation of the VU0637120 binding site at the Y<sub>4</sub>R. (a) Schemes of Y<sub>4</sub>R/Y<sub>1</sub>R chimeras, whereby Y<sub>4</sub>R segments are shown in white and Y<sub>1</sub>R segments are colored in black. (b) Importance of TM1, TM2, and ECL2 for VU0637120 activity. Loss of submaximal PP response (PP EC<sub>80</sub>) in the presence of 15 μM VU0637120 at Y<sub>4</sub>R, Y<sub>1</sub>R, and chimeras investigated in Ca<sup>2+</sup> flux assays. Statistical analysis was performed using one-way analysis of variance (ANOVA) and Dunnett’s post-test: *<i>P</i> < 0.05, **<i>P</i> < 0.01, and ***<i>P</i> < 0.001. (c) Effect of increasing concentrations of (<i>S</i>)<i>-</i>VU0637120 at important Y<sub>4</sub>R residues in TM1, TM2, TM3/4, and TM7 in response to a submaximal PP concentration (PP EC<sub>80</sub>) using Ca<sup>2+</sup> flux assay. Receptor activation was investigated in COS7 cells transiently expressing one specific receptor-eYFP construct and the chimeric G protein Δ6Gα<sub>qi4myr</sub>. Data are shown as mean ± SEM from <i>N</i> ≥ 3 independent experiments. Residues are numbered according to the nomenclature of Ballesteros–Weinstein.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> (d) Y<sub>4</sub>R snake plot, residues important for VU0637120 activity are highlighted in green, and residues not important for VU0637120 are marked in black. Adapted from GPCRdb.org.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02000&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/medium/jm0c02000_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Predicted binding mode of (<i>S</i>)<i>-</i>VU0637120. (a) Computational docking of (<i>S</i>)<i>-</i>VU0637120 identified an allosteric binding site located below the extracellular interface. (b, c) Contacts of (<i>S</i>)<i>-</i>VU0637120 with most of the important residues (side view and top view). (d) Interaction network among TM1, TM2, TM7, and (<i>S</i>)<i>-</i>VU0637120. Y<sub>4</sub>R homology models were generated using 10 class A G protein-coupled receptor (GPCR) X-ray crystal structures (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ZBQ">5ZBQ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4S0V">4S0V</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZJC">4ZJC</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DHG">5DHG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N6H">4N6H</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DJH">4DJH</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DKL">4DKL</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5GLH">5GLH</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ODU">3ODU</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NDD">5NDD</a>; <a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Table S5</a>, Supporting Information).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02000&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/medium/jm0c02000_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Gain-of-function of (<i>S</i>)<i>-</i>VU0637120 at specific Y<sub>4</sub>R variants. (a) Point mutant models made by Rosetta illustrate the improvement in hydrophobic interactions between (<i>S</i>)<i>-</i>VU0637120 and P3.29 or V3.29, and the additional hydrogen bond between (<i>S</i>)<i>-</i>VU0637120 and R6.58. Ten class A GPCR templates were used for building the Y<sub>4</sub>R homology models (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ZBQ">5ZBQ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4S0V">4S0V</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZJC">4ZJC</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DHG">5DHG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N6H">4N6H</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DJH">4DJH</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DKL">4DKL</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5GLH">5GLH</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ODU">3ODU</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NDD">5NDD</a>; <a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Table S5</a>, Supporting Information). (b) Confirmation of P3.29, V3.29, and R6.58 as Y<sub>4</sub>R gain-of-function variants in vitro. Shown are concentration–response curves of (<i>S</i>)<i>-</i>VU0637120, stimulated with a submaximal PP EC<sub>80</sub> concentration at specifically identified Y<sub>4</sub>R residues. (c) Loss of a submaximal PP concentration in the presence of 15 μM of (<i>S</i>)<i>-</i>VU0637120 at Y<sub>4</sub>R gain-of-function variants, data from <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>(b). Ca<sup>2+</sup> flux assays were performed in COS7 cells transiently expressing one specific Y<sub>4</sub>R-eYFP variant and the chimeric G protein Δ6Gα<sub>qi4myr</sub>. Data are shown as mean ± SEM of <i>N</i> ≥ 3 independent experiments. Comparison of (<i>S</i>)<i>-</i>VU0637120 activity at Y<sub>4</sub>R and variants was performed using one-way ANOVA and Dunnett’s post-test: **<i>P</i> < 0.01; ***<i>P</i> < 0.001. Residues are numbered according to the nomenclature of Ballesteros–Weinstein.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02000&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/medium/jm0c02000_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Comparing the binding pocket of PP and (<i>S</i>)<i>-</i>VU0637120. (a, b) Superimposition of docking models of PP and (<i>S</i>)<i>-</i>VU0637120 to Y<sub>4</sub>R. Residues that are important for the activity of PP, (<i>S</i>)<i>-</i>VU0637120, and both are colored in green, red, and yellow, respectively. Group of residues that are represented in the red surface: (*): Y1.39, E1.42, Q2.58, T2.61, S3.28, A3.29, Q3.32, F4.60, L7.36, and H7.39. (c) (<i>S</i>)<i>-</i>VU0637120 binds in a secondary, allosteric binding pocket (red) at the Y<sub>4</sub>R, but this site might slightly overlap with the orthosteric binding site of PP (green). Y<sub>4</sub>R homology models were generated from the X-ray crystal structure of 10 class A GPCRs (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ZBQ">5ZBQ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4S0V">4S0V</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZJC">4ZJC</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DHG">5DHG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N6H">4N6H</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DJH">4DJH</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DKL">4DKL</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5GLH">5GLH</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ODU">3ODU</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NDD">5NDD</a>; <a href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf" class="ext-link">Table S5</a>, Supporting Information).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02000/20210305/images/large/jm0c02000_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02000&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i50">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23266" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23266" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 55 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pedragosa-Badia, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stichel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span> <span> </span><span class="NLM_article-title">Neuropeptide Y receptors: how to get subtype selectivity</span>. <i>Front. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">5</span> <span class="refDoi"> DOI: 10.3389/fendo.2013.00005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.3389%2Ffendo.2013.00005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=23382728" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A280%3ADC%252BC3szls1GntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&author=X.+Pedragosa-Badiaauthor=J.+Stichelauthor=A.+G.+Beck-Sickinger&title=Neuropeptide+Y+receptors%3A+how+to+get+subtype+selectivity&doi=10.3389%2Ffendo.2013.00005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Neuropeptide Y receptors: how to get subtype selectivity</span></div><div class="casAuthors">Pedragosa-Badia Xavier; Stichel Jan; Beck-Sickinger Annette G</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in endocrinology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5</span>
        ISSN:<span class="NLM_cas:issn">1664-2392</span>.
    </div><div class="casAbstract">The neuropeptide Y (NPY) system is a multireceptor/multiligand system consisting of four receptors in humans (hY(1), hY(2), hY(4), hY(5)) and three agonists (NPY, PYY, PP) that activate these receptors with different potency.  The relevance of this system in diseases like obesity or cancer, and the different role that each receptor plays influencing different biological processes makes this system suitable for the design of subtype selectivity studies.  In this review we focus on the latest findings within the NPY system, we summarize recent mutagenesis studies, structure activity relationship studies, receptor chimera, and selective ligands focusing also on the binding mode of the native agonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSR-p8hgInRDnl8icjszYl0fW6udTcc2ebMNLTcKwruZ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3szls1GntA%253D%253D&md5=7f0c792b923299292f75dabb643e4105</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.3389%2Ffendo.2013.00005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffendo.2013.00005%26sid%3Dliteratum%253Aachs%26aulast%3DPedragosa-Badia%26aufirst%3DX.%26aulast%3DStichel%26aufirst%3DJ.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26atitle%3DNeuropeptide%2520Y%2520receptors%253A%2520how%2520to%2520get%2520subtype%2520selectivity%26jtitle%3DFront.%2520Endocrinol.%26date%3D2013%26volume%3D4%26doi%3D10.3389%2Ffendo.2013.00005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yulyaningsih, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herzog, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sainsbury, A.</span></span> <span> </span><span class="NLM_article-title">NPY receptors as potential targets for anti-obesity drug development</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>163</i></span>,  <span class="NLM_fpage">1170</span>– <span class="NLM_lpage">1202</span>, <span class="refDoi"> DOI: 10.1111/j.1476-5381.2011.01363.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1111%2Fj.1476-5381.2011.01363.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=21545413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC3MXps1Sjsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=163&publication_year=2011&pages=1170-1202&author=E.+Yulyaningsihauthor=L.+Zhangauthor=H.+Herzogauthor=A.+Sainsbury&title=NPY+receptors+as+potential+targets+for+anti-obesity+drug+development&doi=10.1111%2Fj.1476-5381.2011.01363.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">NPY receptors as potential targets for anti-obesity drug development</span></div><div class="casAuthors">Yulyaningsih, Ernie; Zhang, Lei; Herzog, Herbert; Sainsbury, Amanda</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">163</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1170-1202</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  The neuropeptide Y system has proven to be one of the most important regulators of feeding behavior and energy homeostasis, thus presenting great potential as a therapeutic target for the treatment of disorders such as obesity and at the other extreme, anorexia.  Due to the initial lack of pharmacol. tools that are active in vivo, functions of the different Y receptors were mainly studied in knockout and transgenic mouse models.  However, over recent years various Y receptor selective peptidic and non-peptidic agonists and antagonists were developed and tested.  Their therapeutic potential in relation to treating obesity and other disorders of energy homeostasis is discussed in this review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoo39ZkydVcgbVg90H21EOLACvtfcHk0liCRIS13MU5cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXps1Sjsb0%253D&md5=051cd4220d7aea714e61764917210fcc</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2011.01363.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2011.01363.x%26sid%3Dliteratum%253Aachs%26aulast%3DYulyaningsih%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DHerzog%26aufirst%3DH.%26aulast%3DSainsbury%26aufirst%3DA.%26atitle%3DNPY%2520receptors%2520as%2520potential%2520targets%2520for%2520anti-obesity%2520drug%2520development%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2011%26volume%3D163%26spage%3D1170%26epage%3D1202%26doi%3D10.1111%2Fj.1476-5381.2011.01363.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bijker, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herzog, H.</span></span> <span> </span><span class="NLM_article-title">The neuropeptide Y system: pathophysiological and therapeutic implications in obesity and cancer</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">91</span>– <span class="NLM_lpage">113</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2011.03.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2Fj.pharmthera.2011.03.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=21439311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmsVyltbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2011&pages=91-113&author=L.+Zhangauthor=M.+S.+Bijkerauthor=H.+Herzog&title=The+neuropeptide+Y+system%3A+pathophysiological+and+therapeutic+implications+in+obesity+and+cancer&doi=10.1016%2Fj.pharmthera.2011.03.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The neuropeptide Y system: Pathophysiological and therapeutic implications in obesity and cancer</span></div><div class="casAuthors">Zhang, Lei; Bijker, Martijn S.; Herzog, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">91-113</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  The neuropeptide Y (NPY) system - comprising of neuropeptide Y, peptide YY, pancreatic polypeptide and the corresponding Y receptors through which they act (Y1, Y2, Y4, Y5 and y6) - is well known for its role in the regulation of energy homeostasis and assocd. processes.  Dysfunctions of the system have been implicated in human diseases such as obesity and cancer, raising the possibility that correction of the system may provide therapeutic benefits for these diseases.  In addn. to the regulation of appetite and satiety that has attracted most attention during the past years, insight has also been gained into the crit. role of NPY in the control of energy expenditure, oxidative fuel selection and bone metab.  Studies using conditional knockout models further shed light on the central vs. peripheral, and hypothalamic vs. extra-hypothalamic mechanisms of these regulatory effects of NPY.  Moreover, a role of NPY family peptides and Y receptors in modulating the growth of tumors has emerged.  These findings provide the basis for novel NPY system-targeted strategies to treat obesity as well as cancer.  Such strategies include modifying both sides of the energy balance equation - energy intake vs. energy expenditure - to achieve a greater wt./fat loss by particularly modulating peripheral Y receptor(s) to ameliorate metabolic conditions without interfering with central functions of Y receptors.  In addn., targeting multiple Y receptors and/or multiple systems involved in the regulation of energy balance will have greater beneficial effects.  However, long-term interference with the NPY system to target obesity or cancer related aspects needs to consider potential side effects on bone health.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnPMeJgSGUdLVg90H21EOLACvtfcHk0liGmefX3bxRQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmsVyltbY%253D&md5=9817e55c615e9d6084fe17b71ff85d7d</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2011.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2011.03.011%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DBijker%26aufirst%3DM.%2BS.%26aulast%3DHerzog%26aufirst%3DH.%26atitle%3DThe%2520neuropeptide%2520Y%2520system%253A%2520pathophysiological%2520and%2520therapeutic%2520implications%2520in%2520obesity%2520and%2520cancer%26jtitle%3DPharmacol.%2520Ther.%26date%3D2011%26volume%3D131%26spage%3D91%26epage%3D113%26doi%3D10.1016%2Fj.pharmthera.2011.03.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sato, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogino, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mashiko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ando, M.</span></span> <span> </span><span class="NLM_article-title">Modulation of neuropeptide Y receptors for the treatment of obesity</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1401</span>– <span class="NLM_lpage">1415</span>, <span class="refDoi"> DOI: 10.1517/13543770903251722</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1517%2F13543770903251722" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=19743896" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFyhtLfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=1401-1415&author=N.+Satoauthor=Y.+Oginoauthor=S.+Mashikoauthor=M.+Ando&title=Modulation+of+neuropeptide+Y+receptors+for+the+treatment+of+obesity&doi=10.1517%2F13543770903251722"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Modulation of neuropeptide Y receptors for the treatment of obesity</span></div><div class="casAuthors">Sato, Nagaaki; Ogino, Yoshio; Mashiko, Satoshi; Ando, Makoto</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1401-1415</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Background: Neuropeptide Y (NPY) has been demonstrated to have crit. roles in the physiol. control of appetite and energy homeostasis through NPY Y1, Y2, Y4 and Y5 receptors.  A no. of synthetic ligands for NPY receptor subtypes have been developed to date, with Y5 receptor antagonists and Y2 and Y4 receptor agonists advancing into clin. trials.  Methods: A survey of the scientific and patent literature since mid-2006 is presented.  Conclusion: In addn. to the specific modulation of resp. NPY receptor subtypes, recent investigations have revealed that modulation of multiple NPY receptor subtypes produces additive or even synergistic anti-obesity effects.  Development of reliable small mol. Y1, Y2 and Y4 receptor ligands would greatly accelerate investigations and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCyDX0rwHE1LVg90H21EOLACvtfcHk0liGmefX3bxRQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFyhtLfK&md5=687624982f64ecd0c2b2cf1a70f8f7ba</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1517%2F13543770903251722&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543770903251722%26sid%3Dliteratum%253Aachs%26aulast%3DSato%26aufirst%3DN.%26aulast%3DOgino%26aufirst%3DY.%26aulast%3DMashiko%26aufirst%3DS.%26aulast%3DAndo%26aufirst%3DM.%26atitle%3DModulation%2520of%2520neuropeptide%2520Y%2520receptors%2520for%2520the%2520treatment%2520of%2520obesity%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2009%26volume%3D19%26spage%3D1401%26epage%3D1415%26doi%3D10.1517%2F13543770903251722" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Batterham, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Roux, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frost, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghatei, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bloom, S. R.</span></span> <span> </span><span class="NLM_article-title">Pancreatic polypeptide reduces appetite and food intake in humans</span>. <i>J. Clin. Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">3989</span>– <span class="NLM_lpage">3992</span>, <span class="refDoi"> DOI: 10.1210/jc.2003-030630</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1210%2Fjc.2003-030630" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=12915697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmsFOku7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2003&pages=3989-3992&author=R.+L.+Batterhamauthor=C.+W.+Le+Rouxauthor=M.+A.+Cohenauthor=A.+J.+Parkauthor=S.+M.+Ellisauthor=M.+Pattersonauthor=G.+S.+Frostauthor=M.+A.+Ghateiauthor=S.+R.+Bloom&title=Pancreatic+polypeptide+reduces+appetite+and+food+intake+in+humans&doi=10.1210%2Fjc.2003-030630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Pancreatic polypeptide reduces appetite and food intake in humans</span></div><div class="casAuthors">Batterham, R. L.; Le Roux, C. W.; Cohen, M. A.; Park, A. J.; Ellis, S. M.; Patterson, M.; Frost, G. S.; Ghatei, M. A.; Bloom, S. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3989-3992</span>CODEN:
                <span class="NLM_cas:coden">JCEMAZ</span>;
        ISSN:<span class="NLM_cas:issn">0021-972X</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">Pancreatic polypeptide (PP) is a gut hormone released from the pancreas in response to ingestion of food.  Plasma PP has been shown to be reduced in conditions assocd. with increased food intake and elevated in anorexia nervosa.  In addn. peripheral administration of PP has been shown to decrease food intake in rodents.  These findings suggest that PP may act as a circulating factor that regulates food intake.  Therefore we investigated the effect of i.v. infusion of PP (10 pmol/kg/min) on appetite and food intake in a randomized double-blind placebo-controlled crossover study in ten healthy volunteers.  Infusion of PP reduced appetite and decreased the energy intake at a buffet lunch two hours post-infusion by 21.8±5.7% (P < 0.01).  More importantly the inhibition of food intake was sustained, such that energy intake, as assessed by food diaries, was significantly reduced both the evening of the study and the following morning.  Overall PP infusion reduced cumulative 24-h energy intake by 25.3±5.8%.  In conclusion our data demonstrates that PP causes a sustained decrease in both appetite and food intake.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHQ8iKXhLwcrVg90H21EOLACvtfcHk0liGmefX3bxRQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmsFOku7o%253D&md5=b28998834ae9a05df6bc678d42a5b9af</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1210%2Fjc.2003-030630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fjc.2003-030630%26sid%3Dliteratum%253Aachs%26aulast%3DBatterham%26aufirst%3DR.%2BL.%26aulast%3DLe%2BRoux%26aufirst%3DC.%2BW.%26aulast%3DCohen%26aufirst%3DM.%2BA.%26aulast%3DPark%26aufirst%3DA.%2BJ.%26aulast%3DEllis%26aufirst%3DS.%2BM.%26aulast%3DPatterson%26aufirst%3DM.%26aulast%3DFrost%26aufirst%3DG.%2BS.%26aulast%3DGhatei%26aufirst%3DM.%2BA.%26aulast%3DBloom%26aufirst%3DS.%2BR.%26atitle%3DPancreatic%2520polypeptide%2520reduces%2520appetite%2520and%2520food%2520intake%2520in%2520humans%26jtitle%3DJ.%2520Clin.%2520Endocrinol.%2520Metab.%26date%3D2003%26volume%3D88%26spage%3D3989%26epage%3D3992%26doi%3D10.1210%2Fjc.2003-030630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jesudason, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGowan, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neary, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philippou, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Small, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frost, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghatei, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bloom, S. R.</span></span> <span> </span><span class="NLM_article-title">Low-dose pancreatic polypeptide inhibits food intake in man</span>. <i>Br. J. Nutr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">426</span>– <span class="NLM_lpage">429</span>, <span class="refDoi"> DOI: 10.1017/S0007114507336799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1017%2FS0007114507336799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=17313701" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjvVSitbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2007&pages=426-429&author=D.+R.+Jesudasonauthor=M.+P.+Monteiroauthor=B.+M.+McGowanauthor=N.+M.+Nearyauthor=A.+J.+Parkauthor=E.+Philippouauthor=C.+J.+Smallauthor=G.+S.+Frostauthor=M.+A.+Ghateiauthor=S.+R.+Bloom&title=Low-dose+pancreatic+polypeptide+inhibits+food+intake+in+man&doi=10.1017%2FS0007114507336799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Low-dose pancreatic polypeptide inhibits food intake in man</span></div><div class="casAuthors">Jesudason, David R.; Monteiro, Mariana P.; McGowan, Barbara M. C.; Neary, Nicola M.; Park, Adrian J.; Philippou, Elena; Small, Caroline J.; Frost, Gary S.; Ghatei, Mohammad A.; Bloom, Stephen R.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Nutrition</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">426-429</span>CODEN:
                <span class="NLM_cas:coden">BJNUAV</span>;
        ISSN:<span class="NLM_cas:issn">0007-1145</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">Pancreatic polypeptide (PP) is a gut hormone released from the pancreas in response to food ingestion and remains elevated for up to 6 h post-prandially.  Plasma levels are elevated in patients with pancreatic tumors.  An i.v. infusion of PP has been reported to reduce food intake in man, suggesting that PP is a satiety hormone.  The authors investigated whether a lower infusion rate of PP would induce significant alterations in energy intake.  The study was randomized and double-blinded.  Fourteen lean fasted volunteers (5 men and 9 women) received 90 min infusions of PP (5 pmol/kg/min) and saline on 2 sep. days.  The dose chosen was half that used in a previous human study which reported a decrease in appetite but at supra-physiol. levels of PP.  One hour after the end of the infusion, a buffet lunch was served and energy intake measured.  PP infusion was assocd. with a significant 11% redn. in energy intake compared with saline.  Preprandial hunger as assessed by a visual analog score was decreased in the PP-treated group compared to saline.  These effects were achieved with plasma levels of PP within the pathophysiol. range of pancreatic tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9qZfa4vo6U7Vg90H21EOLACvtfcHk0liGmefX3bxRQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjvVSitbg%253D&md5=755b55137fca7b6029ed92466a79bc05</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1017%2FS0007114507336799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS0007114507336799%26sid%3Dliteratum%253Aachs%26aulast%3DJesudason%26aufirst%3DD.%2BR.%26aulast%3DMonteiro%26aufirst%3DM.%2BP.%26aulast%3DMcGowan%26aufirst%3DB.%2BM.%26aulast%3DNeary%26aufirst%3DN.%2BM.%26aulast%3DPark%26aufirst%3DA.%2BJ.%26aulast%3DPhilippou%26aufirst%3DE.%26aulast%3DSmall%26aufirst%3DC.%2BJ.%26aulast%3DFrost%26aufirst%3DG.%2BS.%26aulast%3DGhatei%26aufirst%3DM.%2BA.%26aulast%3DBloom%26aufirst%3DS.%2BR.%26atitle%3DLow-dose%2520pancreatic%2520polypeptide%2520inhibits%2520food%2520intake%2520in%2520man%26jtitle%3DBr.%2520J.%2520Nutr.%26date%3D2007%26volume%3D97%26spage%3D426%26epage%3D429%26doi%3D10.1017%2FS0007114507336799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frank, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hey-Hawkins, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, P.</span></span> <span> </span><span class="NLM_article-title">Manipulating Y receptor subtype activation of short neuropeptide Y analogs by introducing carbaboranes</span>. <i>Neuropeptides</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">59</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1016/j.npep.2012.12.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2Fj.npep.2012.12.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=23352609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFCqsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2013&pages=59-66&author=S.+Hofmannauthor=R.+Frankauthor=E.+Hey-Hawkinsauthor=A.+G.+Beck-Sickingerauthor=P.+Schmidt&title=Manipulating+Y+receptor+subtype+activation+of+short+neuropeptide+Y+analogs+by+introducing+carbaboranes&doi=10.1016%2Fj.npep.2012.12.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Manipulating Y receptor subtype activation of short neuropeptide Y analogs by introducing carbaboranes</span></div><div class="casAuthors">Hofmann, S.; Frank, R.; Hey-Hawkins, E.; Beck-Sickinger, A. G.; Schmidt, P.</div><div class="citationInfo"><span class="NLM_cas:title">Neuropeptides (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">59-66</span>CODEN:
                <span class="NLM_cas:coden">NRPPDD</span>;
        ISSN:<span class="NLM_cas:issn">0143-4179</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Short selective neuropeptide Y (NPY) analogs are highly attractive because of their facile synthesis.  Based on the reduced-size NPY analog [Pro30, Nle31, Bpa32, Leu34]NPY 28-36 position 32 was identified as a key position to alter the preferential activation pattern of the human neuropeptide Y receptors (hYRs).  By replacing benzoylphenylalanine (Bpa) by a biphenylalanine (Bip) the photostability was first improved while the biol. activity was maintained.  SAR-studies showed that both arom. rings have a high influence on the preferential hYR subtype activation.  Interestingly, replacement of Bpa32 by a strongly hydrophobic moiety changed the hYR subtype preference of the analog.  Whereas the parent compd. is able to activate the human neuropeptide Y1 receptor (hY1R) subtype, the introduction of an Nε-ortho-carbaboranyl propionic acid modified lysine resulted in a loss of activity at the hY1R but in an increased activity at both the hY2R and the hY4R.  However, subsequent receptor internalization studies with this novel analog revealed that receptor internalization can neither be triggered at the hY2R nor at the hY4R suggesting a biased ligand.  Surprisingly, investigations by 1H NMR spectroscopy revealed structural changes in the side chains of residues Pro30 and Leu34 which nicely correlates with the shift from hY1R/hY4R to hY2R/hY4R activation preference.  Thus, position 32 has been identified to switch the bioactive conformation and subsequently influences receptor subtype activation behavior.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvP9VxxSzNyrVg90H21EOLACvtfcHk0lilLLv_c7r-wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFCqsbw%253D&md5=bc298e83582804b6cad8c2000a99b172</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.npep.2012.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.npep.2012.12.001%26sid%3Dliteratum%253Aachs%26aulast%3DHofmann%26aufirst%3DS.%26aulast%3DFrank%26aufirst%3DR.%26aulast%3DHey-Hawkins%26aufirst%3DE.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26aulast%3DSchmidt%26aufirst%3DP.%26atitle%3DManipulating%2520Y%2520receptor%2520subtype%2520activation%2520of%2520short%2520neuropeptide%2520Y%2520analogs%2520by%2520introducing%2520carbaboranes%26jtitle%3DNeuropeptides%26date%3D2013%26volume%3D47%26spage%3D59%26epage%3D66%26doi%3D10.1016%2Fj.npep.2012.12.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parker, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babij, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balasubramaniam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burrier, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamud, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukhopadhyay, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudinski, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tice, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullins, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salisbury, B. G.</span></span> <span> </span><span class="NLM_article-title">GR231118 (1229U91) and other analogues of the C-terminus of neuropeptide Y are potent neuropeptide Y Y1 receptor antagonists and neuropeptide Y Y4 receptor agonists</span>. <i>Eur. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>349</i></span>,  <span class="NLM_fpage">97</span>– <span class="NLM_lpage">105</span>, <span class="refDoi"> DOI: 10.1016/S0014-2999(98)00171-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2FS0014-2999%2898%2900171-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=9669502" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADyaK1cXjsVehtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=349&publication_year=1998&pages=97-105&author=E.+M.+Parkerauthor=C.+K.+Babijauthor=A.+Balasubramaniamauthor=R.+E.+Burrierauthor=M.+Guzziauthor=F.+Hamudauthor=G.+Mukhopadhyayauthor=M.+S.+Rudinskiauthor=Z.+Taoauthor=M.+Ticeauthor=L.+Xiaauthor=D.+E.+Mullinsauthor=B.+G.+Salisbury&title=GR231118+%281229U91%29+and+other+analogues+of+the+C-terminus+of+neuropeptide+Y+are+potent+neuropeptide+Y+Y1+receptor+antagonists+and+neuropeptide+Y+Y4+receptor+agonists&doi=10.1016%2FS0014-2999%2898%2900171-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">GR231118 (1229U91) and other analogs of the C-terminus of neuropeptide Y are potent neuropeptide Y Y1 receptor antagonists and neuropeptide Y Y4 receptor agonists</span></div><div class="casAuthors">Parker, Eric M.; Babij, Carol K.; Balasubramaniam, Ambikaipakan; Burrier, Robert E.; Guzzi, Mario; Hamud, Fozia; Mukhopadhyay, G.; Rudinski, Mark S.; Tao, Z.; Tice, Melissa; Xia, Ling; Mullins, Deborra E.; Salisbury, Brian G.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmacology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">349</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">97-105</span>CODEN:
                <span class="NLM_cas:coden">EJPHAZ</span>;
        ISSN:<span class="NLM_cas:issn">0014-2999</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">GR231118, BW1911U90, Bis(31/31'){[Cys31, Trp32, Nva34] neuropeptide Y(31-36)} (T-190) and [Trp-Arg-Nva-Arg-Tyr]2-NH2 (T-241) are peptide analogs of the C-terminus of neuropeptide Y that have recently been shown to be antagonists of the neuropeptide Y Y1 receptor.  In this study, the activity of these peptides at each of the cloned neuropeptide Y receptor subtypes is detd. in radioligand binding assays and in functional assays (inhibition of forskolin-stimulated cAMP formation).  GR231118 is a potent antagonist at the human and rat neuropeptide Y Y1 receptors (pA2=10.5 and 10.0, resp.; pKi=10.2 and 10.4, resp.), a potent agonist at the human neuropeptide Y Y4 receptor (pEC50=8.6; pKi=9.6) and a weak agonist at the human and rat neuropeptide Y Y2 and Y5 receptors.  GR231118 also has high affinity for the mouse neuropeptide Y Y6 receptor (pKi=8.8).  Therefore, GR231118 is a relatively selective neuropeptide Y Y1 receptor antagonist, but has appreciable activity at the neuropeptide Y Y4 and Y6 receptors as well.  BW1911U90, T-190 and T-241 are moderately potent neuropeptide Y Y1 receptor antagonists (pA2=7.1, 5.8 and 6.5, resp.; pKi=8.3, 6.5 and 6.8, resp.) and neuropeptide Y Y4 receptor agonists (pEC50=6.8, 6.3 and 6.6, resp.; pKi;8.3, 7.7 and 8.3, resp.).  These data suggest that the C-terminus of neuropeptide Y and related peptides is sufficient for activation of the neuropeptide Y Y4 receptor, but is not sufficient for activation of the neuropeptide Y Y1 receptor.  Because BW1911U90, T-190 and T-241 are significantly less potent at the cloned human neuropeptide Y Y1 receptor than at the neuropeptide Y receptor in human erythroleukemia cells, these cells may express a novel neuropeptide Y receptor with high affinity for these peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnzH4siycw6rVg90H21EOLACvtfcHk0lilLLv_c7r-wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjsVehtL8%253D&md5=03c10aafff350e9f8862b16b27f27557</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2FS0014-2999%2898%2900171-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-2999%252898%252900171-X%26sid%3Dliteratum%253Aachs%26aulast%3DParker%26aufirst%3DE.%2BM.%26aulast%3DBabij%26aufirst%3DC.%2BK.%26aulast%3DBalasubramaniam%26aufirst%3DA.%26aulast%3DBurrier%26aufirst%3DR.%2BE.%26aulast%3DGuzzi%26aufirst%3DM.%26aulast%3DHamud%26aufirst%3DF.%26aulast%3DMukhopadhyay%26aufirst%3DG.%26aulast%3DRudinski%26aufirst%3DM.%2BS.%26aulast%3DTao%26aufirst%3DZ.%26aulast%3DTice%26aufirst%3DM.%26aulast%3DXia%26aufirst%3DL.%26aulast%3DMullins%26aufirst%3DD.%2BE.%26aulast%3DSalisbury%26aufirst%3DB.%2BG.%26atitle%3DGR231118%2520%25281229U91%2529%2520and%2520other%2520analogues%2520of%2520the%2520C-terminus%2520of%2520neuropeptide%2520Y%2520are%2520potent%2520neuropeptide%2520Y%2520Y1%2520receptor%2520antagonists%2520and%2520neuropeptide%2520Y%2520Y4%2520receptor%2520agonists%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D1998%26volume%3D349%26spage%3D97%26epage%3D105%26doi%3D10.1016%2FS0014-2999%2898%2900171-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Balasubramaniam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullins, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhawan, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knittel, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slack, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herzog, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, E. M.</span></span> <span> </span><span class="NLM_article-title">Neuropeptide Y (NPY) Y4 receptor selective agonists based on NPY(32-36): development of an anorectic Y4 receptor selective agonist with picomolar affinity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">2661</span>– <span class="NLM_lpage">2665</span>, <span class="refDoi"> DOI: 10.1021/jm050907d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050907d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BD28XivFWlsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=2661-2665&author=A.+Balasubramaniamauthor=D.+E.+Mullinsauthor=S.+Linauthor=W.+Zhaiauthor=Z.+Taoauthor=V.+C.+Dhawanauthor=M.+Guzziauthor=J.+J.+Knittelauthor=K.+Slackauthor=H.+Herzogauthor=E.+M.+Parker&title=Neuropeptide+Y+%28NPY%29+Y4+receptor+selective+agonists+based+on+NPY%2832-36%29%3A+development+of+an+anorectic+Y4+receptor+selective+agonist+with+picomolar+affinity&doi=10.1021%2Fjm050907d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Neuropeptide Y (NPY) Y4 Receptor Selective Agonists Based on NPY(32-36): Development of an Anorectic Y4 Receptor Selective Agonist with Picomolar Affinity</span></div><div class="casAuthors">Balasubramaniam, Ambikaipakan; Mullins, Deborra E.; Lin, Shu; Zhai, Wexiu; Tao, Zhiyong; Dhawan, Vikas C.; Guzzi, Mario; Knittel, James J.; Slack, Katy; Herzog, Herbert; Parker, Eric M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2661-2665</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have previously shown [Cys-Trp-Arg-Nva-Arg-Tyr-NH2]2, 1, to be a moderately selective neuropeptide Y (NPY) Y4 receptor agonist.  Toward improving the selectivity and potency for Y4 receptors, we studied the effects of dimerizing H-Trp-Arg-Nva-Arg-Tyr-NH2 using various diamino-dicarboxylic acids contg. either di-, tri-, or tetramethylene spacers.  These parallel dimers, 2A, 2B, 3, 4A, and 4B, and the corresponding linear tandem dimer and trimer analogs, 5 and 6, had enhanced selectivity and affinity for Y4 receptors compared to 1 (Table 1).  Substitution of Trp and Nva with Tyr and Leu, resp., as in 2,7-D/L-diaminosuberic acid derivatized dimer, 7, resulted in a superior Y4 selective agonist with picomolar affinity.  I.P. injection of 7 potently inhibited food intake in fasted mice.  Moreover, 7 (i.p.) inhibited the food intake in wild-type mice and not in Y4-/- knock-out mice, confirming that the actions of 7 on food intake are not due to global effects, but specifically mediated Y4 receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNSefoXjlitLVg90H21EOLACvtfcHk0lgDwte18WKB_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XivFWlsL0%253D&md5=cfa39f849b60d3e8b56118c2f7b3396a</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm050907d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050907d%26sid%3Dliteratum%253Aachs%26aulast%3DBalasubramaniam%26aufirst%3DA.%26aulast%3DMullins%26aufirst%3DD.%2BE.%26aulast%3DLin%26aufirst%3DS.%26aulast%3DZhai%26aufirst%3DW.%26aulast%3DTao%26aufirst%3DZ.%26aulast%3DDhawan%26aufirst%3DV.%2BC.%26aulast%3DGuzzi%26aufirst%3DM.%26aulast%3DKnittel%26aufirst%3DJ.%2BJ.%26aulast%3DSlack%26aufirst%3DK.%26aulast%3DHerzog%26aufirst%3DH.%26aulast%3DParker%26aufirst%3DE.%2BM.%26atitle%3DNeuropeptide%2520Y%2520%2528NPY%2529%2520Y4%2520receptor%2520selective%2520agonists%2520based%2520on%2520NPY%252832-36%2529%253A%2520development%2520of%2520an%2520anorectic%2520Y4%2520receptor%2520selective%2520agonist%2520with%2520picomolar%2520affinity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D2661%26epage%3D2665%26doi%3D10.1021%2Fjm050907d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ertl, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dukorn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiser, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buschauer, A.</span></span> <span> </span><span class="NLM_article-title">High affinity agonists of the neuropeptide Y (NPY) Y4 receptor derived from the C-terminal pentapeptide of human pancreatic polypeptide (hPP): synthesis, stereochemical discrimination, and radiolabeling</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">6045</span>– <span class="NLM_lpage">6058</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00309</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00309" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC28XosFSmtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=6045-6058&author=K.+K.+Kuhnauthor=T.+Ertlauthor=S.+Dukornauthor=M.+Kellerauthor=G.+Bernhardtauthor=O.+Reiserauthor=A.+Buschauer&title=High+affinity+agonists+of+the+neuropeptide+Y+%28NPY%29+Y4+receptor+derived+from+the+C-terminal+pentapeptide+of+human+pancreatic+polypeptide+%28hPP%29%3A+synthesis%2C+stereochemical+discrimination%2C+and+radiolabeling&doi=10.1021%2Facs.jmedchem.6b00309"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">High Affinity Agonists of the Neuropeptide Y (NPY) Y4 Receptor Derived from the C-Terminal Pentapeptide of Human Pancreatic Polypeptide (hPP): Synthesis, Stereochemical Discrimination, and Radiolabeling</span></div><div class="casAuthors">Kuhn, Kilian K.; Ertl, Thomas; Dukorn, Stefanie; Keller, Max; Bernhardt, Guenther; Reiser, Oliver; Buschauer, Armin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6045-6058</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The diastereomeric mixt. of D/L-2,7-diaminooctanedioyl-bis(YRLRY-NH2) (BVD-74D, 2) was described in the literature as a high affinity Y4 receptor agonist.  Here we report on the synthesis and pharmacol. characterization of the pure diastereomers (2R,7R)- and (2S,7S)-2 and a series of homo- and heterodimeric analogs in which octanedioic acid was used as an achiral linker.  To investigate the role of the Arg residues, one or two arginines were replaced by Ala.  Moreover, Nω-(6-aminohexylaminocarbonyl)Arg was introduced as an arginine replacement (17). (2R,7R)-2 was superior to (2S,7S)-2 in binding and functional cellular assays and equipotent with 17. [3H]Propionylation of one amino group in the linker of (2R,7R)-2 or at the primary amino group in 17 resulted in high affinity Y4R radioligands ([3H]-(2R,7R)-10, [3H]18) with subnanomolar Kd values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa_pLfg6qLerVg90H21EOLACvtfcHk0lgDwte18WKB_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosFSmtrc%253D&md5=ee7279c046d467c5ec6919187eb0f3aa</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00309%26sid%3Dliteratum%253Aachs%26aulast%3DKuhn%26aufirst%3DK.%2BK.%26aulast%3DErtl%26aufirst%3DT.%26aulast%3DDukorn%26aufirst%3DS.%26aulast%3DKeller%26aufirst%3DM.%26aulast%3DBernhardt%26aufirst%3DG.%26aulast%3DReiser%26aufirst%3DO.%26aulast%3DBuschauer%26aufirst%3DA.%26atitle%3DHigh%2520affinity%2520agonists%2520of%2520the%2520neuropeptide%2520Y%2520%2528NPY%2529%2520Y4%2520receptor%2520derived%2520from%2520the%2520C-terminal%2520pentapeptide%2520of%2520human%2520pancreatic%2520polypeptide%2520%2528hPP%2529%253A%2520synthesis%252C%2520stereochemical%2520discrimination%252C%2520and%2520radiolabeling%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D6045%26epage%3D6058%26doi%3D10.1021%2Facs.jmedchem.6b00309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konieczny, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wifling, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, M.</span></span> <span> </span><span class="NLM_article-title">Oligopeptides as neuropeptide Y Y4 receptor ligands: Identification of a high-affinity tetrapeptide agonist and a hexapeptide antagonist</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">8198</span>– <span class="NLM_lpage">8215</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00426</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00426" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlSju7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=8198-8215&author=A.+Koniecznyauthor=D.+Braunauthor=D.+Wiflingauthor=G.+Bernhardtauthor=M.+Keller&title=Oligopeptides+as+neuropeptide+Y+Y4+receptor+ligands%3A+Identification+of+a+high-affinity+tetrapeptide+agonist+and+a+hexapeptide+antagonist&doi=10.1021%2Facs.jmedchem.0c00426"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Oligopeptides as Neuropeptide Y Y4 Receptor Ligands: Identification of a High-Affinity Tetrapeptide Agonist and a Hexapeptide Antagonist</span></div><div class="casAuthors">Konieczny, Adam; Braun, Diana; Wifling, David; Bernhardt, Guenther; Keller, Max</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8198-8215</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Within the family of neuropeptide Y (NPY) receptors, the Y4 receptor (Y4R) is unique as it prefers pancreatic polypeptide (PP) over NPY and peptide YY (PYY).  Today, low mol. wt. Y4R ligands are lacking, in particular antagonists.  We synthesized a series of peptidic NPY Y4R ligands, derived from the hexapeptide acetyl-Arg-Tyr-Arg-Leu-Arg-Tyr-NH2 (1), reported to be a Y4R partial agonist with high affinity (pKi Y4R: 8.43).  Peptide 1 was N-terminally extended as well as truncated, subjected to a D-amino acid scan, and Leu was replaced by different amino acids.  Compds. were characterized by radioligand competition binding and functional studies (Cai2+ mobilization and β-arrestin 1/2 recruitment).  N-terminal truncation of 1 resulted in a tetrapeptide (Arg-Leu-Arg-Tyr-NH2) being a Y4R partial agonist with retained Y4R affinity (pKi: 8.47).  Remarkably, replacement of Leu in 1 and in derivs. of 1 by Trp turned Y4R agonism to antagonism, giving Y4R antagonists with pKi values ≤ 7.57.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGox8hX-Vehx4rVg90H21EOLACvtfcHk0lgDwte18WKB_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlSju7jM&md5=326a1d05671d3b43056f14f637fee2c3</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00426&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00426%26sid%3Dliteratum%253Aachs%26aulast%3DKonieczny%26aufirst%3DA.%26aulast%3DBraun%26aufirst%3DD.%26aulast%3DWifling%26aufirst%3DD.%26aulast%3DBernhardt%26aufirst%3DG.%26aulast%3DKeller%26aufirst%3DM.%26atitle%3DOligopeptides%2520as%2520neuropeptide%2520Y%2520Y4%2520receptor%2520ligands%253A%2520Identification%2520of%2520a%2520high-affinity%2520tetrapeptide%2520agonist%2520and%2520a%2520hexapeptide%2520antagonist%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D8198%26epage%3D8215%26doi%3D10.1021%2Facs.jmedchem.0c00426" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sliwoski, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schubert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stichel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weaver, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meiler, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of small-molecule modulators of the human Y4 receptor</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">e0157146</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0157146</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1371%2Fjournal.pone.0157146" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=27294784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFyqtr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&author=G.+Sliwoskiauthor=M.+Schubertauthor=J.+Stichelauthor=D.+Weaverauthor=A.+G.+Beck-Sickingerauthor=J.+Meiler&title=Discovery+of+small-molecule+modulators+of+the+human+Y4+receptor&doi=10.1371%2Fjournal.pone.0157146"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of small-molecule modulators of the human Y4 Receptor</span></div><div class="casAuthors">Sliwoski, Gregory; Schubert, Mario; Stichel, Jan; Weaver, David; Beck-Sickinger, Annette G.; Meiler, Jens</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e0157146/1-e0157146/13</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The human neuropeptide Y4 receptor (Y4R) and its native ligand, pancreatic polypeptide, are critically involved in the regulation of human metab. by signaling satiety and regulating food intake, as well as increasing energy expenditure.  Thus, this receptor represents a putative target for treatment of obesity.  With respect to new approaches to treat complex metabolic disorders, esp. in multi-receptor systems, small mol. allosteric modulators have been in the focus of research in the last years.  However, no pos. allosteric modulators or agonists of the Y4R have been described so far.  In this study, small mol. compds. derived from the Niclosamide scaffold were identified by high-throughput screening to increase Y4R activity.  Compds. were characterized for their potency and their effects at the human Y4R and as well as their selectivity towards Y1R, Y2R and Y5R.  These compds. provide a structure-activity relationship profile around this common scaffold and lay the groundwork for hit-to-lead optimization and characterization of pos. allosteric modulators of the Y4R.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAbv1m3XAlzLVg90H21EOLACvtfcHk0li852e4DSy2Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFyqtr3L&md5=ed0f23077965cf43815202ad89127949</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0157146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0157146%26sid%3Dliteratum%253Aachs%26aulast%3DSliwoski%26aufirst%3DG.%26aulast%3DSchubert%26aufirst%3DM.%26aulast%3DStichel%26aufirst%3DJ.%26aulast%3DWeaver%26aufirst%3DD.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26aulast%3DMeiler%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520small-molecule%2520modulators%2520of%2520the%2520human%2520Y4%2520receptor%26jtitle%3DPLoS%2520One%26date%3D2016%26volume%3D11%26doi%3D10.1371%2Fjournal.pone.0157146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schubert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stichel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sliwoski, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meiler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weaver, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span> <span> </span><span class="NLM_article-title">Identification and characterization of the first selective Y4 receptor positive allosteric modulator</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">7605</span>– <span class="NLM_lpage">7612</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00976</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00976" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlSmtLrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=7605-7612&author=M.+Schubertauthor=J.+Stichelauthor=Y.+Duauthor=I.+R.+Toughauthor=G.+Sliwoskiauthor=J.+Meilerauthor=H.+M.+Coxauthor=C.+D.+Weaverauthor=A.+G.+Beck-Sickinger&title=Identification+and+characterization+of+the+first+selective+Y4+receptor+positive+allosteric+modulator&doi=10.1021%2Facs.jmedchem.7b00976"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Characterization of the First Selective Y4 Receptor Positive Allosteric Modulator</span></div><div class="casAuthors">Schubert, Mario; Stichel, Jan; Du, Yu; Tough, Iain R.; Sliwoski, Gregory; Meiler, Jens; Cox, Helen M.; Weaver, C. David; Beck-Sickinger, Annette G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7605-7612</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The human Y4 receptor (Y4R) and its cognate ligand, pancreatic polypeptide (PP), are involved in the regulation of energy exposure, satiety and food intake.  This system represents a potential target for the treatment of metabolic diseases and has been extensively investigated and validated in vivo.  Here, we present the compd. tBPC (2-(4-tert-Butyl)phenoxycyclohexanol), a novel and selective Y4R pos. allosteric modulator that potentiates Y4R activation in G-protein signaling and arrestin 3 recruitment expts.  The compd. has no effect on the binding of the orthosteric ligands, implying its allosteric mode of action at the Y4R and evidence for a purely efficacy-driven pos. allosteric modulation.  Finally, the ability of tBPC to selectively potentiate Y4R agonism initiated by PP was confirmed in mouse descending colon mucosa prepns. expressing native Y4R, demonstrating Y4R pos. allosteric modulation in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFvYNHioUE8rVg90H21EOLACvtfcHk0li852e4DSy2Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlSmtLrL&md5=99b416da5721005fb30d8acff3110f23</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00976&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00976%26sid%3Dliteratum%253Aachs%26aulast%3DSchubert%26aufirst%3DM.%26aulast%3DStichel%26aufirst%3DJ.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DTough%26aufirst%3DI.%2BR.%26aulast%3DSliwoski%26aufirst%3DG.%26aulast%3DMeiler%26aufirst%3DJ.%26aulast%3DCox%26aufirst%3DH.%2BM.%26aulast%3DWeaver%26aufirst%3DC.%2BD.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520the%2520first%2520selective%2520Y4%2520receptor%2520positive%2520allosteric%2520modulator%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D7605%26epage%3D7612%26doi%3D10.1021%2Facs.jmedchem.7b00976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaske, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holzammer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buschauer, A.</span></span> <span> </span><span class="NLM_article-title">Dimeric argininamide-type neuropeptide Y receptor antagonists: chiral discrimination between Y1 and Y4 receptors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">6303</span>– <span class="NLM_lpage">6322</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.08.065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2Fj.bmc.2013.08.065" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=24074877" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFemtLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=6303-6322&author=M.+Kellerauthor=M.+Kaskeauthor=T.+Holzammerauthor=G.+Bernhardtauthor=A.+Buschauer&title=Dimeric+argininamide-type+neuropeptide+Y+receptor+antagonists%3A+chiral+discrimination+between+Y1+and+Y4+receptors&doi=10.1016%2Fj.bmc.2013.08.065"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Dimeric argininamide-type neuropeptide Y receptor antagonists: Chiral discrimination between Y1 and Y4 receptors</span></div><div class="casAuthors">Keller, Max; Kaske, Melanie; Holzammer, Tobias; Bernhardt, Guenther; Buschauer, Armin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6303-6322</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The structurally related peptides neuropeptide Y (NPY), peptide YY (PYY) and pancreatic polypeptide (PP) are endogenous agonists of the NPY receptor (YR) family, which in humans comprises four functionally expressed subtypes, designated Y1R, Y2R, Y4R and Y5R.  Nonpeptide antagonists with high affinity and selectivity have been described for the Y1R, Y2R and Y5R, but such compds. are still lacking for the Y4R.  In this work, the structures of the high affinity selective (R)-argininamide-type Y1R antagonists BIBP3226 and BIBO3304 were linked via the guanidine or urea moieties to give homo-dimeric argininamides with linker lengths ranging from 31 to 41 atoms.  Interestingly, the twin compds. proved to be by far less selective for the Y1R than the R-configured monovalent parent compds.  The decrease in selectivity ratio was most pronounced for Y1R vs. Y4R subtype, resulting in comparable affinities of bivalent ligands for Y1R and Y4R (e.g. UR-MK177 [(R,R)-I (R1 = (CH2O)2CH2CH2X, Y = CH2NHC(O)NH2)]: Ki = 230 nM (Y1R) and 290 nM (Y4R)).  With a Ki value of 130 nM and a Kb value of 20 nM, UR-MK188 [(R,R)-II (R2 = CH2(CH2OCH2)2CH2X, Y = OH)] was superior to all Y4R antagonists known to date.  The S,S-configured optical antipodes of UR-MK177 and UR-MK188 (UR-MEK381 ((S,S)-I) and UR-MEK388 ((S,S)-II)) were synthesized to investigate the stereochem. discrimination by the different receptor subtypes.  Whereas preference for R,R-configured argininamides was characteristic of the Y1R, stereochem. discrimination by the Y4R was not obsd.  This may pave the way to selective Y4R antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRAhCQ0Gt1nLVg90H21EOLACvtfcHk0lgCKYxcUfeSvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFemtLzL&md5=5ca4d29cdb12eb2c4ea26396b1c569df</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.08.065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.08.065%26sid%3Dliteratum%253Aachs%26aulast%3DKeller%26aufirst%3DM.%26aulast%3DKaske%26aufirst%3DM.%26aulast%3DHolzammer%26aufirst%3DT.%26aulast%3DBernhardt%26aufirst%3DG.%26aulast%3DBuschauer%26aufirst%3DA.%26atitle%3DDimeric%2520argininamide-type%2520neuropeptide%2520Y%2520receptor%2520antagonists%253A%2520chiral%2520discrimination%2520between%2520Y1%2520and%2520Y4%2520receptors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D6303%26epage%3D6322%26doi%3D10.1016%2Fj.bmc.2013.08.065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Littmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dukorn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buschauer, A.</span></span> <span> </span><span class="NLM_article-title">In Search of NPY Y4R antagonists: Incorporation of carbamoylated arginine, aza-amino acids, or d-amino acids into oligopeptides derived from the C-termini of the endogenous agonists</span>. <i>ACS Omega</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">3616</span>– <span class="NLM_lpage">3631</span>, <span class="refDoi"> DOI: 10.1021/acsomega.7b00451</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.7b00451" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFKhu7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2017&pages=3616-3631&author=K.+K.+Kuhnauthor=T.+Littmannauthor=S.+Dukornauthor=M.+Tanakaauthor=M.+Kellerauthor=T.+Ozawaauthor=G.+Bernhardtauthor=A.+Buschauer&title=In+Search+of+NPY+Y4R+antagonists%3A+Incorporation+of+carbamoylated+arginine%2C+aza-amino+acids%2C+or+d-amino+acids+into+oligopeptides+derived+from+the+C-termini+of+the+endogenous+agonists&doi=10.1021%2Facsomega.7b00451"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">In Search of NPY Y4R Antagonists: Incorporation of Carbamoylated Arginine, Aza-Amino Acids, or D-Amino Acids into Oligopeptides Derived from the C-Termini of the Endogenous Agonists</span></div><div class="casAuthors">Kuhn, Kilian K.; Littmann, Timo; Dukorn, Stefanie; Tanaka, Miho; Keller, Max; Ozawa, Takeaki; Bernhardt, Guenther; Buschauer, Armin</div><div class="citationInfo"><span class="NLM_cas:title">ACS Omega</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3616-3631</span>CODEN:
                <span class="NLM_cas:coden">ACSODF</span>;
        ISSN:<span class="NLM_cas:issn">2470-1343</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The crosslinked pentapeptides (2R,7R)-diaminooctanedioyl-bis(Tyr-Arg-Leu-Arg-Tyr-amide) ((2R,7R)-BVD-74D, (2R,7R)-1) and octanedioyl-bis(Tyr-Arg-Leu-Arg-Tyr-amide) (2) as well as the pentapeptide Ac-Tyr-Arg-Leu-Arg-Tyr-amide (3) were previously described as neuropeptide Y Y4 receptor (Y4R) partial agonists.  Here the authors report on a series of analogs of (2R,7R)-1 and 2 in which Arg2, Leu3 or Arg4 were replaced by the resp. aza-amino acids.  The replacement of Arg2 in 3 with a carbamoylated arginine building block and the extension of the N-terminus by an addnl. arginine led to the high-affinity hexapeptide Ac-Arg-Tyr-Nω-[(4-aminobutyl)aminocarbonyl]Arg-Leu-Arg-Tyr-amide that was used as a precursor for a D-amino acid scan.  The target compds. were investigated for Y4R functional activity in assays with complementary readouts: aequorin Ca2+ and β-arrestin 1 or β-arrestin 2 assays.  In contrast to the parent compds., which are Y4R agonists, several ligands were able to suppress the effect elicited by the endogenous ligand pancreatic polypeptide and therefore represent a novel class of peptide Y4R antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAufjWqDJZnrVg90H21EOLACvtfcHk0lgCKYxcUfeSvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFKhu7bE&md5=970422192d233703cda161e6e476b3e5</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facsomega.7b00451&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.7b00451%26sid%3Dliteratum%253Aachs%26aulast%3DKuhn%26aufirst%3DK.%2BK.%26aulast%3DLittmann%26aufirst%3DT.%26aulast%3DDukorn%26aufirst%3DS.%26aulast%3DTanaka%26aufirst%3DM.%26aulast%3DKeller%26aufirst%3DM.%26aulast%3DOzawa%26aufirst%3DT.%26aulast%3DBernhardt%26aufirst%3DG.%26aulast%3DBuschauer%26aufirst%3DA.%26atitle%3DIn%2520Search%2520of%2520NPY%2520Y4R%2520antagonists%253A%2520Incorporation%2520of%2520carbamoylated%2520arginine%252C%2520aza-amino%2520acids%252C%2520or%2520d-amino%2520acids%2520into%2520oligopeptides%2520derived%2520from%2520the%2520C-termini%2520of%2520the%2520endogenous%2520agonists%26jtitle%3DACS%2520Omega%26date%3D2017%26volume%3D2%26spage%3D3616%26epage%3D3631%26doi%3D10.1021%2Facsomega.7b00451" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hirose, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, A.</span></span> <span> </span><span class="NLM_article-title">Fluorescence properties of 5-(5,6-dimethoxybenzothiazol-2-yl)-2-deoxyuridine (dbtU) and oligodeoxyribonucleotides containing dbtU</span>. <i>Eur. J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_fpage">6206</span>– <span class="NLM_lpage">6217</span>, <span class="refDoi"> DOI: 10.1002/ejoc.201100818</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1002%2Fejoc.201100818" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtV2gsbvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&pages=6206-6217&author=W.+Hiroseauthor=K.+Satoauthor=A.+Matsuda&title=Fluorescence+properties+of+5-%285%2C6-dimethoxybenzothiazol-2-yl%29-2-deoxyuridine+%28dbtU%29+and+oligodeoxyribonucleotides+containing+dbtU&doi=10.1002%2Fejoc.201100818"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence Properties of 5-(5,6-Dimethoxybenzothiazol-2-yl)-2'-deoxyuridine (dbtU) and Oligodeoxyribonucleotides Containing dbtU</span></div><div class="casAuthors">Hirose, Wataru; Sato, Kousuke; Matsuda, Akira</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2011</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">6206-6217, S6206/1-S6206/43</span>CODEN:
                <span class="NLM_cas:coden">EJOCFK</span>;
        ISSN:<span class="NLM_cas:issn">1099-0690</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">We describe the synthesis and photo-phys. properties of 11 substituted 5-(benzothiazol-2-yl)-2'-deoxyuridine derivs. and oligodeoxyribonucleotides (ODNs) contg. 5-(5,6-dimethoxybenzothiazol-2-yl)-2'-deoxyuridine (dbtU), which was the strongest fluorescent deriv. among those prepd.  The fluorescence properties of dbtU itself and ODNs contg. dbtU show the same tendency of being weaker in both neutral and acidic soln. and stronger in basic soln.  The ODNs (15mer) contg. 16 combinations of 5'-XbtU-3' and 5'-btUY-3', where X, Y = A, T, G, or C, were synthesized, and their fluorescence intensity and quantum yield in basic soln. were compared.  On av., only the ODN with the 5'-GbtU-3' sequence shows a 7.9-fold lower fluorescence intensity than the other sequences.  Ab initio calcns. of 5'-GbtU-3' and 5'-btUG-3' as models under basic conditions suggest that the lower fluorescence of the ODN contg. the 5'-GbtU-3' sequence is caused by a wider overlap between stacked guanine (Gua) and btUra than that of the 5'-btUG-3' sequence and that the HOMO is delocalized not only on btUra but also on Gua.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6KEFkAz7xELVg90H21EOLACvtfcHk0lhdRWF6Ai0KkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtV2gsbvE&md5=fb8079bf06d7c4372e19ad6f344b0ea6</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fejoc.201100818&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.201100818%26sid%3Dliteratum%253Aachs%26aulast%3DHirose%26aufirst%3DW.%26aulast%3DSato%26aufirst%3DK.%26aulast%3DMatsuda%26aufirst%3DA.%26atitle%3DFluorescence%2520properties%2520of%25205-%25285%252C6-dimethoxybenzothiazol-2-yl%2529-2-deoxyuridine%2520%2528dbtU%2529%2520and%2520oligodeoxyribonucleotides%2520containing%2520dbtU%26jtitle%3DEur.%2520J.%2520Org.%2520Chem.%26date%3D2011%26spage%3D6206%26epage%3D6217%26doi%3D10.1002%2Fejoc.201100818" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leach, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span> <span> </span><span class="NLM_article-title">Allosteric GPCR modulators: taking advantage of permissive receptor pharmacology</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">382</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2007.06.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2Fj.tips.2007.06.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=17629965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotlWltLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2007&pages=382-389&author=K.+Leachauthor=P.+M.+Sextonauthor=A.+Christopoulos&title=Allosteric+GPCR+modulators%3A+taking+advantage+of+permissive+receptor+pharmacology&doi=10.1016%2Fj.tips.2007.06.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric GPCR modulators: taking advantage of permissive receptor pharmacology</span></div><div class="casAuthors">Leach, Katie; Sexton, Patrick M.; Christopoulos, Arthur</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">382-389</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The past decade has witnessed a dramatic increase in the identification of allosteric modulators of G-protein-coupled receptor (GPCR) activity.  Concomitantly, several new perspectives and hypotheses regarding the way ligands regulate GPCR signaling have also emerged.  Here, we briefly discuss how the concepts of collateral efficacy and permissive agonism-antagonism intersect the field of allosteric GPCR modulation.  Despite the challenges assocd. with detecting and quantifying the myriad of possible allosteric effects on GPCR activity, it is proposed that allosteric ligands offer the exciting prospect of engendering stimulus-bias in orthosteric ligand signaling, thus paving the way for not only receptor-selective but also signaling-pathway-selective therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpHb_QsIW3TbVg90H21EOLACvtfcHk0lhdRWF6Ai0KkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotlWltLs%253D&md5=943da2312e7b136cb594ba0346e00301</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2007.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2007.06.004%26sid%3Dliteratum%253Aachs%26aulast%3DLeach%26aufirst%3DK.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DChristopoulos%26aufirst%3DA.%26atitle%3DAllosteric%2520GPCR%2520modulators%253A%2520taking%2520advantage%2520of%2520permissive%2520receptor%2520pharmacology%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2007%26volume%3D28%26spage%3D382%26epage%3D389%26doi%3D10.1016%2Fj.tips.2007.06.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bdioui, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierre, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trinquet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roux, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenakin, T.</span></span> <span> </span><span class="NLM_article-title">Equilibrium assays are required to accurately characterize the activity profiles of drugs modulating Gq-protein-coupled receptors</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">992</span>– <span class="NLM_lpage">1006</span>, <span class="refDoi"> DOI: 10.1124/mol.118.112573</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1124%2Fmol.118.112573" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=29954837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVGgtrvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2018&pages=992-1006&author=S.+Bdiouiauthor=J.+Verdiauthor=N.+Pierreauthor=E.+Trinquetauthor=T.+Rouxauthor=T.+Kenakin&title=Equilibrium+assays+are+required+to+accurately+characterize+the+activity+profiles+of+drugs+modulating+Gq-protein-coupled+receptors&doi=10.1124%2Fmol.118.112573"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Equilibrium assays are required to accurately characterize the activity profiles of drugs modulating Gq-protein-coupled receptors</span></div><div class="casAuthors">Bdioui, Sara; Verdi, Julien; Pierre, Nicolas; Trinquet, Eric; Roux, Thomas; Kenakin, Terry</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">992-1006</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">This paper discusses the process of detg. the activity of candidate mols. targeting Gq-protein activation through G-protein-coupled receptors for possible therapeutic application with two functional assays; calcium release and inositol phosphate metab. [inositol monophosphate (IP1)].  While both are suitable for detecting ligand activity (screening), differences are seen when these assays are used to quant. measure ligand parameters for therapeutic activity.  Specifically, responses for Gq-related pathways present different and dissimulating patterns depending on the functional assay used to assess them.  To investigate the impact of functional assays on the accuracy of compd. pharmacol. profiles, five exemplar mols. [partial agonist, antagonist, inverse agonist, pos. allosteric modulator (PAM) agonist, and pos. beta-PAM] targeting either muscarinic M1 or ghrelin receptors were tested using two functional assays (calcium release and IP1) and the results were compared with theor. pharmacol. models.  The IP1 assay is an equil. assay that is able to det. the correct pharmacol. profiles of all tested compds.  In contrast, the nonequil. nature of calcium assays yields misleading classification of most of the tested compds.  Our study suggests that the use of an equil. assay, such as IP1, is mandatory for the optimal use of pharmacol. models that can both identify mechanisms of action and also convert descriptive-to-predictive data for therapeutic systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1UalfauOGHbVg90H21EOLACvtfcHk0lhdRWF6Ai0KkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVGgtrvN&md5=5edfd3e3f251b396820a1cb8f03e2aaa</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1124%2Fmol.118.112573&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.118.112573%26sid%3Dliteratum%253Aachs%26aulast%3DBdioui%26aufirst%3DS.%26aulast%3DVerdi%26aufirst%3DJ.%26aulast%3DPierre%26aufirst%3DN.%26aulast%3DTrinquet%26aufirst%3DE.%26aulast%3DRoux%26aufirst%3DT.%26aulast%3DKenakin%26aufirst%3DT.%26atitle%3DEquilibrium%2520assays%2520are%2520required%2520to%2520accurately%2520characterize%2520the%2520activity%2520profiles%2520of%2520drugs%2520modulating%2520Gq-protein-coupled%2520receptors%26jtitle%3DMol.%2520Pharmacol.%26date%3D2018%26volume%3D94%26spage%3D992%26epage%3D1006%26doi%3D10.1124%2Fmol.118.112573" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenakin, T.</span></span> <span> </span><span class="NLM_article-title">G protein-coupled receptor allosterism and complexing</span>. <i>Pharmacol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">323</span>– <span class="NLM_lpage">374</span>, <span class="refDoi"> DOI: 10.1124/pr.54.2.323</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1124%2Fpr.54.2.323" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=12037145" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BD38XltVOhsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2002&pages=323-374&author=A.+Christopoulosauthor=T.+Kenakin&title=G+protein-coupled+receptor+allosterism+and+complexing&doi=10.1124%2Fpr.54.2.323"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">G protein-coupled receptor allosterism and complexing</span></div><div class="casAuthors">Christopoulos, Arthur; Kenakin, Terry</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">323-374</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">0031-6997</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  G protein-coupled receptors (GPCRs) represent the largest family of cell-surface receptors.  These receptors are natural allosteric proteins because agonist-mediated signaling by GPCRs requires a conformational change in the receptor protein transmitted between two topog. distinct binding sites, one for the agonist and another for the G protein.  It is now becoming increasingly recognized, however, that the agonist-bound GPCR can also form ternary complexes with other ligands or "accessory" proteins and display altered binding and/or signaling properties in relation to the binary agonist-receptor complex.  Allosteric sites on GPCRs represent novel drug targets because allosteric modulators possess a no. of theor. advantages over classic orthosteric ligands, such as a ceiling level to the allosteric effect and a potential for greater GPCR subtype-selectivity.  Because of the noncompetitive nature of allosteric phenomena, the detection and quantification of such effects often relies on a combination of equil. binding, nonequil. kinetic, and functional signaling assays.  This review discusses the development and properties of allosteric receptor models for GPCRs and the detection and quantification of allosteric effects.  Moreover, we provide an overview of the current knowledge regarding,the location of possible allosteric sites on GPCRs and candidate endogenous allosteric modulators.  Finally, we discuss the potential for allosteric effects arising from the formation of GPCR oligomers or GPCRs complexed with accessory cellular proteins.  It is proposed that the study of allosteric phenomena will become of progressively, greater import to the drug discovery process due to the advent of newer and more sensitive GPCR screening-technologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnSA5pRRWTILVg90H21EOLACvtfcHk0lhdRWF6Ai0KkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltVOhsrs%253D&md5=6c4895bf66778eda4778ac409c7abb70</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1124%2Fpr.54.2.323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.54.2.323%26sid%3Dliteratum%253Aachs%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DKenakin%26aufirst%3DT.%26atitle%3DG%2520protein-coupled%2520receptor%2520allosterism%2520and%2520complexing%26jtitle%3DPharmacol.%2520Rev.%26date%3D2002%26volume%3D54%26spage%3D323%26epage%3D374%26doi%3D10.1124%2Fpr.54.2.323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wanka, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babilon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mörl, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurevich, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span> <span> </span><span class="NLM_article-title">C-terminal motif of human neuropeptide Y4 receptor determines internalization and arrestin recruitment</span>. <i>Cell. Signalling</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">233</span>– <span class="NLM_lpage">239</span>, <span class="refDoi"> DOI: 10.1016/j.cellsig.2016.11.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2Fj.cellsig.2016.11.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=27818291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvV2nsLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2017&pages=233-239&author=L.+Wankaauthor=S.+Babilonauthor=K.+Burkertauthor=K.+M%C3%B6rlauthor=V.+V.+Gurevichauthor=A.+G.+Beck-Sickinger&title=C-terminal+motif+of+human+neuropeptide+Y4+receptor+determines+internalization+and+arrestin+recruitment&doi=10.1016%2Fj.cellsig.2016.11.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">C-terminal motif of human neuropeptide Y4 receptor determines internalization and arrestin recruitment</span></div><div class="casAuthors">Wanka, Lizzy; Babilon, Stefanie; Burkert, Kerstin; Moerl, Karin; Gurevich, Vsevolod V.; Beck-Sickinger, Annette G.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Signalling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">233-239</span>CODEN:
                <span class="NLM_cas:coden">CESIEY</span>;
        ISSN:<span class="NLM_cas:issn">0898-6568</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The human neuropeptide Y4 receptor is a rhodopsin-like G protein-coupled receptor (GPCR), which contributes to anorexigenic signals.  Thus, this receptor is a highly interesting target for metabolic diseases.  As GPCR internalization and trafficking affect receptor signaling and vice versa, we aimed to investigate the mol. mechanism of hY4R desensitization and endocytosis.  The role of distinct segments of the hY4R carboxyl terminus was investigated by fluorescence microscopy, binding assays, inositol turnover expts. and bioluminescence resonance energy transfer assays to examine the internalization behavior of hY4R and its interaction with arrestin-3.  Based on results of C-terminal deletion mutants and substitution of single amino acids, the motif 7.78EESEHLPLSTVHTEVSKGS7.96 was identified, with glutamate, threonine and serine residues playing key roles, based on site-directed mutagenesis.  Thus, we identified the internalization motif for the human neuropeptide Y4 receptor, which regulates arrestin-3 recruitment and receptor endocytosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHsGkMKEJwqLVg90H21EOLACvtfcHk0lhvhAbiGIPGBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvV2nsLrP&md5=5ac2a1f796acc477a2e836cd88a5b9c6</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.cellsig.2016.11.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellsig.2016.11.003%26sid%3Dliteratum%253Aachs%26aulast%3DWanka%26aufirst%3DL.%26aulast%3DBabilon%26aufirst%3DS.%26aulast%3DBurkert%26aufirst%3DK.%26aulast%3DM%25C3%25B6rl%26aufirst%3DK.%26aulast%3DGurevich%26aufirst%3DV.%2BV.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26atitle%3DC-terminal%2520motif%2520of%2520human%2520neuropeptide%2520Y4%2520receptor%2520determines%2520internalization%2520and%2520arrestin%2520recruitment%26jtitle%3DCell.%2520Signalling%26date%3D2017%26volume%3D29%26spage%3D233%26epage%3D239%26doi%3D10.1016%2Fj.cellsig.2016.11.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burford, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertekap, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alt, A.</span></span> <span> </span><span class="NLM_article-title">Strategies for the identification of allosteric modulators of G-protein-coupled receptors</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">691</span>– <span class="NLM_lpage">702</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2010.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2Fj.bcp.2010.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=21184747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit12jtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2011&pages=691-702&author=N.+T.+Burfordauthor=J.+Watsonauthor=R.+Bertekapauthor=A.+Alt&title=Strategies+for+the+identification+of+allosteric+modulators+of+G-protein-coupled+receptors&doi=10.1016%2Fj.bcp.2010.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies for the identification of allosteric modulators of G-protein-coupled receptors</span></div><div class="casAuthors">Burford, Neil T.; Watson, John; Bertekap, Robert; Alt, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">691-702</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Once considered a pharmacol. curiosity, allosteric modulation of seven transmembrane domain G-protein-coupled receptors (GPCRs) has emerged as a potentially powerful means to affect receptor function for therapeutic purposes.  Allosteric modulators, which interact with binding sites topol. distinct from the orthosteric ligand binding sites, can potentially provide improved selectivity and safety, along with maintenance of spatial and temporal aspects of GPCR signaling.  Accordingly, drug discovery efforts for GPCRs have increasingly focused on the identification of allosteric modulators.  This review is devoted to an examn. of the strategies, challenges, and opportunities for high-throughput screening for allosteric modulators of GPCRs, with particular focus on the identification of pos. allosteric modulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6GSDSVjkw2bVg90H21EOLACvtfcHk0lhvhAbiGIPGBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit12jtLs%253D&md5=d2f2b3304cf9d31d0d37456baf9aec71</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2010.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2010.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DBurford%26aufirst%3DN.%2BT.%26aulast%3DWatson%26aufirst%3DJ.%26aulast%3DBertekap%26aufirst%3DR.%26aulast%3DAlt%26aufirst%3DA.%26atitle%3DStrategies%2520for%2520the%2520identification%2520of%2520allosteric%2520modulators%2520of%2520G-protein-coupled%2520receptors%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2011%26volume%3D81%26spage%3D691%26epage%3D702%26doi%3D10.1016%2Fj.bcp.2010.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ballesteros, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstein, H.</span></span> <span> </span><span class="NLM_article-title">[19] Integrated methods for the construction of three-dimensional models and computational probing of structure-function relations in G protein-coupled receptors</span>. <i>Methods Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">366</span>– <span class="NLM_lpage">428</span>, <span class="refDoi"> DOI: 10.1016/s1043-9471(05)80049-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2FS1043-9471%2805%2980049-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADyaK2MXmt1Kmtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1995&pages=366-428&author=J.+A.+Ballesterosauthor=H.+Weinstein&title=%5B19%5D+Integrated+methods+for+the+construction+of+three-dimensional+models+and+computational+probing+of+structure-function+relations+in+G+protein-coupled+receptors&doi=10.1016%2Fs1043-9471%2805%2980049-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Integrated methods for the construction of three-dimensional models and computational probing of structure-function relations in G protein-coupled receptors</span></div><div class="casAuthors">Ballesteros, Juan A.; Weinstein, Harel</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Neurosciences</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">366-428</span>CODEN:
                <span class="NLM_cas:coden">MENEE5</span>;
        ISSN:<span class="NLM_cas:issn">1043-9471</span>.
    </div><div class="casAbstract">A review, with 135 refs., on approaches that can be used to resolve the apparent ambiguities that burden the pharmacol. testing of G protein-coupled receptor (GPCR) models, based on the integration of structural information about the receptor, about mutants, and about the changes induced by ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfENoRspqa7LVg90H21EOLACvtfcHk0lhvhAbiGIPGBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmt1Kmtrk%253D&md5=d3f99e9d07cea245a770f3f099bc8cc7</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2FS1043-9471%2805%2980049-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1043-9471%252805%252980049-7%26sid%3Dliteratum%253Aachs%26aulast%3DBallesteros%26aufirst%3DJ.%2BA.%26aulast%3DWeinstein%26aufirst%3DH.%26atitle%3D%255B19%255D%2520Integrated%2520methods%2520for%2520the%2520construction%2520of%2520three-dimensional%2520models%2520and%2520computational%2520probing%2520of%2520structure-function%2520relations%2520in%2520G%2520protein-coupled%2520receptors%26jtitle%3DMethods%2520Neurosci.%26date%3D1995%26volume%3D25%26spage%3D366%26epage%3D428%26doi%3D10.1016%2Fs1043-9471%2805%2980049-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Isberg, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mordalski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munk, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rataj, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harpsoe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vroling, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bojarski, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vriend, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gloriam, D. E.</span></span> <span> </span><span class="NLM_article-title">GPCRdb: an information system for G protein-coupled receptors</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">D356</span>– <span class="NLM_lpage">D364</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv1178</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1093%2Fnar%2Fgkv1178" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=26582914" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2gu7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D356-D364&author=V.+Isbergauthor=S.+Mordalskiauthor=C.+Munkauthor=K.+Ratajauthor=K.+Harpsoeauthor=A.+S.+Hauserauthor=B.+Vrolingauthor=A.+J.+Bojarskiauthor=G.+Vriendauthor=D.+E.+Gloriam&title=GPCRdb%3A+an+information+system+for+G+protein-coupled+receptors&doi=10.1093%2Fnar%2Fgkv1178"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">GPCRdb: an information system for G protein-coupled receptors</span></div><div class="casAuthors">Isberg, Vignir; Mordalski, Stefan; Munk, Christian; Rataj, Krzysztof; Harpsoee, Kasper; Hauser, Alexander S.; Vroling, Bas; Bojarski, Andrzej J.; Vriend, Gert; Gloriam, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D356-D364</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Recent developments in G protein-coupled receptor (GPCR) structural biol. and pharmacol. have greatly enhanced our knowledge of receptor structure-function relations, and have helped improve the scientific foundation for drug design studies.  The GPCR database, GPCRdb, serves a dual role in disseminating and enabling new scientific developments by providing ref. data, anal. tools and interactive diagrams.  This paper highlights new features in the fifth major GPCRdb release: (i) GPCR crystal structure browsing, superposition and display of ligand interactions; (ii) direct deposition by users of point mutations and their effects on ligand binding; (iii) refined snake and helix box residue diagram looks; and (iv) phylogenetic trees with receptor classification color schemes.  Under the hood, the entire GPCRdb front- and back-ends have been recoded within one infrastructure, ensuring a smooth browsing experience and development.  GPCRdb is available at http://www.gpcrdb.org/ and it's open source code at https://bitbucket.org/gpcr/protwis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPnn-bXMy54bVg90H21EOLACvtfcHk0lhBFrE6GG4LLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2gu7bJ&md5=ee7db0ef81d65cf6e5ab8750b990bb78</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv1178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv1178%26sid%3Dliteratum%253Aachs%26aulast%3DIsberg%26aufirst%3DV.%26aulast%3DMordalski%26aufirst%3DS.%26aulast%3DMunk%26aufirst%3DC.%26aulast%3DRataj%26aufirst%3DK.%26aulast%3DHarpsoe%26aufirst%3DK.%26aulast%3DHauser%26aufirst%3DA.%2BS.%26aulast%3DVroling%26aufirst%3DB.%26aulast%3DBojarski%26aufirst%3DA.%2BJ.%26aulast%3DVriend%26aufirst%3DG.%26aulast%3DGloriam%26aufirst%3DD.%2BE.%26atitle%3DGPCRdb%253A%2520an%2520information%2520system%2520for%2520G%2520protein-coupled%2520receptors%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3DD356%26epage%3DD364%26doi%3D10.1093%2Fnar%2Fgkv1178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riepler, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enriquez, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldock, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herzog, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sainsbury, A.</span></span> <span> </span><span class="NLM_article-title">Y2 and Y4 receptor signaling synergistically act on energy expenditure and physical activity</span>. <i>Am. J. Physiol.: Regul., Integr. Comp. Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>299</i></span>,  <span class="NLM_fpage">R1618</span>– <span class="NLM_lpage">R1628</span>, <span class="refDoi"> DOI: 10.1152/ajpregu.00345.2010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1152%2Fajpregu.00345.2010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=20881101" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=299&publication_year=2010&pages=R1618-R1628&author=L.+Zhangauthor=S.+J.+Rieplerauthor=N.+Turnerauthor=R.+F.+Enriquezauthor=I.+C.+Leeauthor=P.+A.+Baldockauthor=H.+Herzogauthor=A.+Sainsbury&title=Y2+and+Y4+receptor+signaling+synergistically+act+on+energy+expenditure+and+physical+activity&doi=10.1152%2Fajpregu.00345.2010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1152%2Fajpregu.00345.2010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajpregu.00345.2010%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DRiepler%26aufirst%3DS.%2BJ.%26aulast%3DTurner%26aufirst%3DN.%26aulast%3DEnriquez%26aufirst%3DR.%2BF.%26aulast%3DLee%26aufirst%3DI.%2BC.%26aulast%3DBaldock%26aufirst%3DP.%2BA.%26aulast%3DHerzog%26aufirst%3DH.%26aulast%3DSainsbury%26aufirst%3DA.%26atitle%3DY2%2520and%2520Y4%2520receptor%2520signaling%2520synergistically%2520act%2520on%2520energy%2520expenditure%2520and%2520physical%2520activity%26jtitle%3DAm.%2520J.%2520Physiol.%253A%2520Regul.%252C%2520Integr.%2520Comp.%2520Physiol.%26date%3D2010%26volume%3D299%26spage%3DR1618%26epage%3DR1628%26doi%3D10.1152%2Fajpregu.00345.2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tough, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holliday, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, H. M.</span></span> <span> </span><span class="NLM_article-title">Y(4) receptors mediate the inhibitory responses of pancreatic polypeptide in human and mouse colon mucosa</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>319</i></span>,  <span class="NLM_fpage">20</span>– <span class="NLM_lpage">30</span>, <span class="refDoi"> DOI: 10.1124/jpet.106.106500</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1124%2Fjpet.106.106500" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=16807358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVCksL%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=319&publication_year=2006&pages=20-30&author=I.+R.+Toughauthor=N.+D.+Hollidayauthor=H.+M.+Cox&title=Y%284%29+receptors+mediate+the+inhibitory+responses+of+pancreatic+polypeptide+in+human+and+mouse+colon+mucosa&doi=10.1124%2Fjpet.106.106500"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Y4 receptors mediate the inhibitory responses of pancreatic polypeptide in human and mouse colon mucosa</span></div><div class="casAuthors">Tough, Iain R.; Holliday, Nicholas D.; Cox, Helen M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">319</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">20-30</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The antisecretory effects of several Y agonists, including pancreatic polypeptide (PP), indicate the presence of Y1, Y2, and Y4 receptors in mouse and human (h) colon mucosa.  Here, we used prepns. from human and from wild-type (WT), Y4, and Y1 receptor knockout (-/-) mice, alongside Y4 receptor-transfected cells to define the relative functional contribution of the Y4 receptor.  First, rat (r) PP antisecretory responses were lost in murine Y4-/- prepns., but hPP and Pro34 peptide YY (PYY) costimulated Y4 and Y1 receptors in WT mucosa.  The Y1 antagonist/Y4 agonist GR231118 [(IIe,Glu,Pro,Dpr,Tyr,Arg,Leu,Arg,Try-NH2)-2-cyclic(2,4'),(2',4)-diamide] elicited small Y4-mediated antisecretory responses in human tissues pretreated with the Y1 antagonist, BIBO3304 (R)-N-[(4-(aminocarbonylaminomethyl)-phenylmethyl)]-N2-[(diphenylacetyl)-argininamiden trifluoroacetate], and attenuated Y4-mediated hPP responses in mouse and human mucosa.  GR231118 and rPP were also antisecretory in hY4-transfected epithelial monolayers but were partial agonists compared with hPP at this receptor.  In Y4-transfected human embryonic kidney (HEK) 293 cells, Y4 ligands displaced [125I]hPP binding with orders of affinity (pKi) at human (hPP = rPP > GR231118 > Pro34PYY = PYY) and mouse (rPP = hPP > GR231118 > Pro34PYY > PYY) Y4 receptors.  GR23118- and rPP-stimulated guanosine 5'-3-O-(thio)triphosphate binding through hY4 receptors with significantly lower efficacy than hPP.  GR231118 marginally increased basal but abolished further PP-induced hY4 internalization to recycling (transferrin-labeled) pathways in HEK293 cells.  Taken together, these findings show that Y4 receptors play a definitive role in attenuating colonic anion transport and may be useful targets for novel antidiarrheal agents due to their limited peripheral expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHuAe6MTFvirVg90H21EOLACvtfcHk0lhBFrE6GG4LLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVCksL%252FI&md5=cf51b740f94ff70872f85de3d9bf815a</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1124%2Fjpet.106.106500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.106.106500%26sid%3Dliteratum%253Aachs%26aulast%3DTough%26aufirst%3DI.%2BR.%26aulast%3DHolliday%26aufirst%3DN.%2BD.%26aulast%3DCox%26aufirst%3DH.%2BM.%26atitle%3DY%25284%2529%2520receptors%2520mediate%2520the%2520inhibitory%2520responses%2520of%2520pancreatic%2520polypeptide%2520in%2520human%2520and%2520mouse%2520colon%2520mucosa%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2006%26volume%3D319%26spage%3D20%26epage%3D30%26doi%3D10.1124%2Fjpet.106.106500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meiler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span> <span> </span><span class="NLM_article-title">ROSETTALIGAND: protein-small molecule docking with full side-chain flexibility</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">538</span>– <span class="NLM_lpage">548</span>, <span class="refDoi"> DOI: 10.1002/prot.21086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1002%2Fprot.21086" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=16972285" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFWqt7nJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2006&pages=538-548&author=J.+Meilerauthor=D.+Baker&title=ROSETTALIGAND%3A+protein-small+molecule+docking+with+full+side-chain+flexibility&doi=10.1002%2Fprot.21086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">ROSETTALIGAND: protein-small molecule docking with full side-chain flexibility</span></div><div class="casAuthors">Meiler, Jens; Baker, David</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">538-548</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Protein-small mol. docking algorithms provide a means to model the structure of protein-small mol. complexes in structural detail and play an important role in drug development.  In recent years the necessity of simulating protein side-chain flexibility for an accurate prediction of the protein-small mol. interfaces has become apparent, and an increasing no. of docking algorithms probe different approaches to include protein flexibility.  Here the authors describe a new method for docking small mols. into protein binding sites employing a Monte Carlo minimization procedure in which the rigid body position and orientation of the small mol. and the protein side-chain conformations are optimized simultaneously.  The energy function comprises van der Waals (VDW) interactions, an implicit solvation model, an explicit orientation hydrogen bonding potential, and an electrostatics model.  In an evaluation of the scoring function the computed energy correlated with exptl. small mol. binding energy with a correlation coeff. of 0.63 across a diverse set of 229 protein-small mol. complexes.  The docking method produced lowest energy models with a root mean square deviation (RMSD) smaller than 2 Å in 71 out of 100 protein-small mol. crystal structure complexes (self-docking).  In cross-docking calcns. in which both protein side-chain and small mol. internal degrees of freedom were varied the lowest energy predictions had RMSDs less than 2 Å in 14 of 20 test cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb1x4K4yjDk7Vg90H21EOLACvtfcHk0lhBFrE6GG4LLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFWqt7nJ&md5=0e5580cc490c11a7e9c6265c27a14408</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fprot.21086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21086%26sid%3Dliteratum%253Aachs%26aulast%3DMeiler%26aufirst%3DJ.%26aulast%3DBaker%26aufirst%3DD.%26atitle%3DROSETTALIGAND%253A%2520protein-small%2520molecule%2520docking%2520with%2520full%2520side-chain%2520flexibility%26jtitle%3DProteins%26date%3D2006%26volume%3D65%26spage%3D538%26epage%3D548%26doi%3D10.1002%2Fprot.21086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Combs, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deluca, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deluca, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemmon, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nannemann, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willis, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheehan, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meiler, J.</span></span> <span> </span><span class="NLM_article-title">Small-molecule ligand docking into comparative models with Rosetta</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1277</span>– <span class="NLM_lpage">1298</span>, <span class="refDoi"> DOI: 10.1038/nprot.2013.074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1038%2Fnprot.2013.074" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=23744289" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC3sXovFartLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1277-1298&author=S.+A.+Combsauthor=S.+L.+Delucaauthor=S.+H.+Delucaauthor=G.+H.+Lemmonauthor=D.+P.+Nannemannauthor=E.+D.+Nguyenauthor=J.+R.+Willisauthor=J.+H.+Sheehanauthor=J.+Meiler&title=Small-molecule+ligand+docking+into+comparative+models+with+Rosetta&doi=10.1038%2Fnprot.2013.074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule ligand docking into comparative models with Rosetta</span></div><div class="casAuthors">Combs, Steven A.; DeLuca, Samuel L.; DeLuca, Stephanie H.; Lemmon, Gordon H.; Nannemann, David P.; Nguyen, Elizabeth D.; Willis, Jordan R.; Sheehan, Jonathan H.; Meiler, Jens</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1277-1298</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Structure-based drug design is frequently used to accelerate the development of small-mol. therapeutics.  Although substantial progress has been made in X-ray crystallog. and NMR spectroscopy, the availability of high-resoln. structures is limited owing to the frequent inability to crystallize or obtain sufficient NMR restraints for large or flexible proteins.  Computational methods can be used to both predict unknown protein structures and model ligand interactions when exptl. data are unavailable.  This paper describes a comprehensive and detailed protocol using the Rosetta modeling suite to dock small-mol. ligands into comparative models.  In the protocol presented here, we review the comparative modeling process, including sequence alignment, threading and loop building.  Next, we cover docking a small-mol. ligand into the protein comparative model.  In addn., we discuss criteria that can improve ligand docking into comparative models.  Finally, and importantly, we present a strategy for assessing model quality.  The entire protocol is presented on a single example selected solely for didactic purposes.  The results are therefore not representative and do not replace benchmarks published elsewhere.  We also provide an addnl. tutorial so that the user can gain hands-on experience in using Rosetta.  The protocol should take 5-7 h, with addnl. time allocated for computer generation of models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwkuPWDwMFFbVg90H21EOLACvtfcHk0lgBPhjAR_12Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXovFartLs%253D&md5=17577fd22e6ca3bd13a2c3aa66ba9b25</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2013.074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2013.074%26sid%3Dliteratum%253Aachs%26aulast%3DCombs%26aufirst%3DS.%2BA.%26aulast%3DDeluca%26aufirst%3DS.%2BL.%26aulast%3DDeluca%26aufirst%3DS.%2BH.%26aulast%3DLemmon%26aufirst%3DG.%2BH.%26aulast%3DNannemann%26aufirst%3DD.%2BP.%26aulast%3DNguyen%26aufirst%3DE.%2BD.%26aulast%3DWillis%26aufirst%3DJ.%2BR.%26aulast%3DSheehan%26aufirst%3DJ.%2BH.%26aulast%3DMeiler%26aufirst%3DJ.%26atitle%3DSmall-molecule%2520ligand%2520docking%2520into%2520comparative%2520models%2520with%2520Rosetta%26jtitle%3DNat.%2520Protoc.%26date%3D2013%26volume%3D8%26spage%3D1277%26epage%3D1298%26doi%3D10.1038%2Fnprot.2013.074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lemmon, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meiler, J.</span></span> <span> </span><span class="NLM_article-title">Rosetta ligand docking with flexible XML protocols</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>819</i></span>,  <span class="NLM_fpage">143</span>– <span class="NLM_lpage">155</span>, <span class="refDoi"> DOI: 10.1007/978-1-61779-465-0_10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1007%2F978-1-61779-465-0_10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=22183535" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFSjur7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=819&publication_year=2012&pages=143-155&author=G.+Lemmonauthor=J.+Meiler&title=Rosetta+ligand+docking+with+flexible+XML+protocols&doi=10.1007%2F978-1-61779-465-0_10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Rosetta ligand docking with flexible XML protocols</span></div><div class="casAuthors">Lemmon, Gordon; Meiler, Jens</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">819</span>
        (<span class="NLM_cas:issue">Computational Drug Discovery and Design</span>),
    <span class="NLM_cas:pages">143-155</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Rosetta Ligand is premiere software for predicting how a protein and a small mol. interact.  Benchmark studies demonstrate that 70% of the top scoring Rosetta Ligand predicted interfaces are within 2 Å RMSD from the crystal structure.  The latest release of Rosetta ligand software includes many new features, such as (1) docking of multiple ligands simultaneously, (2) representing ligands as fragments for greater flexibility, (3) redesign of the interface during docking, and (4) an XML script based interface that gives the user full control of the ligand docking protocol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5flEQBFyUqbVg90H21EOLACvtfcHk0lgBPhjAR_12Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFSjur7P&md5=d815380e73b10f59adc5a8f27f7d2653</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-465-0_10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-465-0_10%26sid%3Dliteratum%253Aachs%26aulast%3DLemmon%26aufirst%3DG.%26aulast%3DMeiler%26aufirst%3DJ.%26atitle%3DRosetta%2520ligand%2520docking%2520with%2520flexible%2520XML%2520protocols%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2012%26volume%3D819%26spage%3D143%26epage%3D155%26doi%3D10.1007%2F978-1-61779-465-0_10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiMaio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miles, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunette, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span> <span> </span><span class="NLM_article-title">High-resolution comparative modeling with RosettaCM</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1735</span>– <span class="NLM_lpage">1742</span>, <span class="refDoi"> DOI: 10.1016/j.str.2013.08.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2Fj.str.2013.08.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=24035711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVCltrrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=1735-1742&author=Y.+Songauthor=F.+DiMaioauthor=R.+Y.+Wangauthor=D.+Kimauthor=C.+Milesauthor=T.+Brunetteauthor=J.+Thompsonauthor=D.+Baker&title=High-resolution+comparative+modeling+with+RosettaCM&doi=10.1016%2Fj.str.2013.08.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">High-Resolution Comparative Modeling with RosettaCM</span></div><div class="casAuthors">Song, Yifan; DiMaio, Frank; Wang, Ray Yu-Ruei; Kim, David; Miles, Chris; Brunette, T. J.; Thompson, James; Baker, David</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1735-1742</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">We describe an improved method for comparative modeling, RosettaCM, which optimizes a phys. realistic all-atom energy function over the conformational space defined by homologous structures.  Given a set of sequence alignments, RosettaCM assembles topologies by recombining aligned segments in Cartesian space and building unaligned regions de novo in torsion space.  The junctions between segments are regularized using a loop closure method combining fragment superposition with gradient-based minimization.  The energies of the resulting models are optimized by all-atom refinement, and the most representative low-energy model is selected.  The CASP10 expt. suggests that RosettaCM yields models with more accurate side-chain and backbone conformations than other methods when the sequence identity to the templates is greater than ∼15%.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorujEuJW1hmbVg90H21EOLACvtfcHk0lgBPhjAR_12Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVCltrrP&md5=299d5b1f30a1fdd0d311804158cc5053</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2013.08.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2013.08.005%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DY.%26aulast%3DDiMaio%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DR.%2BY.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DMiles%26aufirst%3DC.%26aulast%3DBrunette%26aufirst%3DT.%26aulast%3DThompson%26aufirst%3DJ.%26aulast%3DBaker%26aufirst%3DD.%26atitle%3DHigh-resolution%2520comparative%2520modeling%2520with%2520RosettaCM%26jtitle%3DStructure%26date%3D2013%26volume%3D21%26spage%3D1735%26epage%3D1742%26doi%3D10.1016%2Fj.str.2013.08.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pedragosa-Badia, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sliwoski, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong
Nguyen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stichel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufmann, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meiler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span> <span> </span><span class="NLM_article-title">Pancreatic polypeptide is recognized by two hydrophobic domains of the human Y4 receptor binding pocket</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">5846</span>– <span class="NLM_lpage">5859</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.502021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1074%2Fjbc.M113.502021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=24375409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjsVKktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=5846-5859&author=X.+Pedragosa-Badiaauthor=G.+R.+Sliwoskiauthor=E.+Dong%0ANguyenauthor=D.+Lindnerauthor=J.+Stichelauthor=K.+W.+Kaufmannauthor=J.+Meilerauthor=A.+G.+Beck-Sickinger&title=Pancreatic+polypeptide+is+recognized+by+two+hydrophobic+domains+of+the+human+Y4+receptor+binding+pocket&doi=10.1074%2Fjbc.M113.502021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Pancreatic Polypeptide Is Recognized by Two Hydrophobic Domains of the Human Y4 Receptor Binding Pocket</span></div><div class="casAuthors">Pedragosa-Badia, Xavier; Sliwoski, Gregory R.; Nguyen, Elizabeth Dong; Lindner, Diana; Stichel, Jan; Kaufmann, Kristian W.; Meiler, Jens; Beck-Sickinger, Annette G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">5846-5859</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Structural characterization of the human Y4 receptor (hY4R) interaction with human pancreatic polypeptide (hPP) is crucial, not only for understanding its biol. function but also for testing treatment strategies for obesity that target this interaction.  Here, the interaction of receptor mutants with pancreatic polypeptide analogs was studied through double-cycle mutagenesis.  To guide mutagenesis and interpret results, a three-dimensional comparative model of the hY4R-hPP complex was constructed based on all available class A G protein-coupled receptor crystal structures and refined using exptl. data.  Our study reveals that residues of the hPP and the hY4R form a complex network consisting of ionic interactions, hydrophobic interactions, and hydrogen binding.  Residues Tyr2.64, Asp2.68, Asn6.55, Asn7.32, and Phe7.35 of Y4R are found to be important in receptor activation by hPP.  Specifically, Tyr2.64 interacts with Tyr27 of hPP through hydrophobic contacts.  Asn7.32 is affected by modifications on position Arg33 of hPP, suggesting a hydrogen bond between these two residues.  Likewise, we find that Phe7.35 is affected by modifications of hPP at positions 33 and 36, indicating interactions between these three amino acids.  Taken together, we demonstrate that the top of transmembrane helix 2 (TM2) and the top of transmembrane helixes 6 and 7 (TM6-TM7) form the core of the peptide binding pocket.  These findings will contribute to the rational design of ligands that bind the receptor more effectively to produce an enhanced agonistic or antagonistic effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkLgd7H3oyYrVg90H21EOLACvtfcHk0lj6yAdI7Hu9Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjsVKktL4%253D&md5=cf4bf542381418e47815b8b139c8f99b</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.502021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.502021%26sid%3Dliteratum%253Aachs%26aulast%3DPedragosa-Badia%26aufirst%3DX.%26aulast%3DSliwoski%26aufirst%3DG.%2BR.%26aulast%3DDong%2BNguyen%26aufirst%3DE.%26aulast%3DLindner%26aufirst%3DD.%26aulast%3DStichel%26aufirst%3DJ.%26aulast%3DKaufmann%26aufirst%3DK.%2BW.%26aulast%3DMeiler%26aufirst%3DJ.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26atitle%3DPancreatic%2520polypeptide%2520is%2520recognized%2520by%2520two%2520hydrophobic%2520domains%2520of%2520the%2520human%2520Y4%2520receptor%2520binding%2520pocket%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D5846%26epage%3D5859%26doi%3D10.1074%2Fjbc.M113.502021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kögler, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wifling, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plank, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Littmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larhammar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huster, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meiler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buschauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, B.</span></span> <span> </span><span class="NLM_article-title">Structural basis of ligand binding modes at the neuropeptide Y Y1 receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>556</i></span>,  <span class="NLM_fpage">520</span>– <span class="NLM_lpage">524</span>, <span class="refDoi"> DOI: 10.1038/s41586-018-0046-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1038%2Fs41586-018-0046-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=29670288" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC1cXosVCgur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=556&publication_year=2018&pages=520-524&author=Z.+Yangauthor=S.+Hanauthor=M.+Kellerauthor=A.+Kaiserauthor=B.+J.+Benderauthor=M.+Bosseauthor=K.+Burkertauthor=L.+M.+K%C3%B6glerauthor=D.+Wiflingauthor=G.+Bernhardtauthor=N.+Plankauthor=T.+Littmannauthor=P.+Schmidtauthor=C.+Yiauthor=B.+Liauthor=S.+Yeauthor=R.+Zhangauthor=B.+Xuauthor=D.+Larhammarauthor=R.+C.+Stevensauthor=D.+Husterauthor=J.+Meilerauthor=Q.+Zhaoauthor=A.+G.+Beck-Sickingerauthor=A.+Buschauerauthor=B.+Wu&title=Structural+basis+of+ligand+binding+modes+at+the+neuropeptide+Y+Y1+receptor&doi=10.1038%2Fs41586-018-0046-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of ligand binding modes at the neuropeptide Y Y1 receptor</span></div><div class="casAuthors">Yang, Zhenlin; Han, Shuo; Keller, Max; Kaiser, Anette; Bender, Brian J.; Bosse, Mathias; Burkert, Kerstin; Kogler, Lisa M.; Wifling, David; Bernhardt, Guenther; Plank, Nicole; Littmann, Timo; Schmidt, Peter; Yi, Cuiying; Li, Beibei; Ye, Sheng; Zhang, Rongguang; Xu, Bo; Larhammar, Dan; Stevens, Raymond C.; Huster, Daniel; Meiler, Jens; Zhao, Qiang; Beck-Sickinger, Annette G.; Buschauer, Armin; Wu, Beili</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">556</span>
        (<span class="NLM_cas:issue">7702</span>),
    <span class="NLM_cas:pages">520-524</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Neuropeptide Y (NPY) receptors belong to the G-protein-coupled receptor superfamily and have important roles in food intake, anxiety and cancer biol.1,2.  The NPY-Y receptor system has emerged as one of the most complex networks with three peptide ligands (NPY, peptide YY and pancreatic polypeptide) binding to four receptors in most mammals, namely the Y1, Y2, Y4 and Y5 receptors, with different affinity and selectivity3.  NPY is the most powerful stimulant of food intake and this effect is primarily mediated by the Y1 receptor (Y1R)4.  A no. of peptides and small-mol. compds. have been characterized as Y1R antagonists and have shown clin. potential in the treatment of obesity4, tumor1 and bone loss5.  However, their clin. usage has been hampered by low potency and selectivity, poor brain penetration ability or lack of oral bioavailability6.  Here we report crystal structures of the human Y1R bound to the two selective antagonists UR-MK299 and BMS-193885 at 2.7 and 3.0 Å resoln., resp.  The structures combined with mutagenesis studies reveal the binding modes of Y1R to several structurally diverse antagonists and the determinants of ligand selectivity.  The Y1R structure and mol. docking of the endogenous agonist NPY, together with NMR, photo-crosslinking and functional studies, provide insights into the binding behavior of the agonist and for the first time, to our knowledge, det. the interaction of its N terminus with the receptor.  These insights into Y1R can enable structure-based drug discovery that targets NPY receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr584bBD3wdnbVg90H21EOLACvtfcHk0lj6yAdI7Hu9Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXosVCgur8%253D&md5=79289fb85d5f65b94fbf873ec4ea31f3</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fs41586-018-0046-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-018-0046-x%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DHan%26aufirst%3DS.%26aulast%3DKeller%26aufirst%3DM.%26aulast%3DKaiser%26aufirst%3DA.%26aulast%3DBender%26aufirst%3DB.%2BJ.%26aulast%3DBosse%26aufirst%3DM.%26aulast%3DBurkert%26aufirst%3DK.%26aulast%3DK%25C3%25B6gler%26aufirst%3DL.%2BM.%26aulast%3DWifling%26aufirst%3DD.%26aulast%3DBernhardt%26aufirst%3DG.%26aulast%3DPlank%26aufirst%3DN.%26aulast%3DLittmann%26aufirst%3DT.%26aulast%3DSchmidt%26aufirst%3DP.%26aulast%3DYi%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DYe%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DXu%26aufirst%3DB.%26aulast%3DLarhammar%26aufirst%3DD.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DHuster%26aufirst%3DD.%26aulast%3DMeiler%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26aulast%3DBuschauer%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DB.%26atitle%3DStructural%2520basis%2520of%2520ligand%2520binding%2520modes%2520at%2520the%2520neuropeptide%2520Y%2520Y1%2520receptor%26jtitle%3DNature%26date%3D2018%26volume%3D556%26spage%3D520%26epage%3D524%26doi%3D10.1038%2Fs41586-018-0046-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Merten, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Römpler, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mörl, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schöneberg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span> <span> </span><span class="NLM_article-title">Receptor subtype-specific docking of Asp6.59 with C-terminal arginine residues in Y receptor ligands</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">7543</span>– <span class="NLM_lpage">7551</span>, <span class="refDoi"> DOI: 10.1074/jbc.M608902200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1074%2Fjbc.M608902200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=17204471" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitlSrtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=7543-7551&author=N.+Mertenauthor=D.+Lindnerauthor=N.+Rabeauthor=H.+R%C3%B6mplerauthor=K.+M%C3%B6rlauthor=T.+Sch%C3%B6nebergauthor=A.+G.+Beck-Sickinger&title=Receptor+subtype-specific+docking+of+Asp6.59+with+C-terminal+arginine+residues+in+Y+receptor+ligands&doi=10.1074%2Fjbc.M608902200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Receptor Subtype-specific Docking of Asp6.59 with C-terminal Arginine Residues in Y Receptor Ligands</span></div><div class="casAuthors">Merten, Nicole; Lindner, Diana; Rabe, Nadine; Roempler, Holger; Moerl, Karin; Schoeneberg, Torsten; Beck-Sickinger, Annette G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">7543-7551</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Y receptors (YRs) are G protein-coupled receptors whose Y1R, Y2R, and Y5R subtypes preferentially bind neuropeptide Y (NPY) and peptide YY, whereas mammalian Y4Rs show a higher affinity for pancreatic polypeptide (PP).  Comparison of YR orthologs and paralogs revealed Asp6.59 to be fully conserved throughout all of the YRs reported so far.  By replacing this conserved aspartic acid residue with alanine, asparagine, glutamate, and arginine, we now show that this residue plays a crucial role in binding and signal transduction of NPY/PP at all YRs.  Sensitivity to distinct replacements is, however, receptor subtype-specific.  Next, we performed a complementary mutagenesis approach to identify the contact site of the ligand.  Surprisingly, this conserved residue interacts with two different ligand arginine residues by ionic interactions; although in Y2R and Y5R, Arg33 is the binding partner of Asp6.59, in Y1R and Y4R, Arg35 of human PP and NPY interacts with Asp6.59.  Furthermore, Arg25 of PP and NPY is involved in ligand binding only at Y2R and Y5R.  This suggests significant differences in the docking of YR ligands between Y1/4R and Y2/5R and provides new insights into the mol. binding mode of peptide agonists at GPCRs.  Furthermore, the proposed model of a subtype-specific binding mode is in agreement with the evolution of YRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMfokRVrtUB7Vg90H21EOLACvtfcHk0lj6yAdI7Hu9Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitlSrtLs%253D&md5=78e41ddb73bcfd8d23163759159d1bb2</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M608902200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M608902200%26sid%3Dliteratum%253Aachs%26aulast%3DMerten%26aufirst%3DN.%26aulast%3DLindner%26aufirst%3DD.%26aulast%3DRabe%26aufirst%3DN.%26aulast%3DR%25C3%25B6mpler%26aufirst%3DH.%26aulast%3DM%25C3%25B6rl%26aufirst%3DK.%26aulast%3DSch%25C3%25B6neberg%26aufirst%3DT.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26atitle%3DReceptor%2520subtype-specific%2520docking%2520of%2520Asp6.59%2520with%2520C-terminal%2520arginine%2520residues%2520in%2520Y%2520receptor%2520ligands%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D7543%26epage%3D7551%26doi%3D10.1074%2Fjbc.M608902200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kufareva, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenalti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, B.</span></span> <span> </span><span class="NLM_article-title">Structure of the CCR5 chemokine receptor-HIV entry inhibitor maraviroc complex</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>341</i></span>,  <span class="NLM_fpage">1387</span>– <span class="NLM_lpage">1390</span>, <span class="refDoi"> DOI: 10.1126/science.1241475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1126%2Fscience.1241475" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=24030490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVCmu7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=1387-1390&author=Q.+Tanauthor=Y.+Zhuauthor=J.+Liauthor=Z.+Chenauthor=G.+W.+Hanauthor=I.+Kufarevaauthor=T.+Liauthor=L.+Maauthor=G.+Fenaltiauthor=J.+Liauthor=W.+Zhangauthor=X.+Xieauthor=H.+Yangauthor=H.+Jiangauthor=V.+Cherezovauthor=H.+Liuauthor=R.+C.+Stevensauthor=Q.+Zhaoauthor=B.+Wu&title=Structure+of+the+CCR5+chemokine+receptor-HIV+entry+inhibitor+maraviroc+complex&doi=10.1126%2Fscience.1241475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the CCR5 Chemokine Receptor-HIV Entry Inhibitor Maraviroc Complex</span></div><div class="casAuthors">Tan, Qiuxiang; Zhu, Ya; Li, Jian; Chen, Zhuxi; Han, Gye Won; Kufareva, Irina; Li, Tingting; Ma, Limin; Fenalti, Gustavo; Li, Jing; Zhang, Wenru; Xie, Xin; Yang, Huaiyu; Jiang, Hualiang; Cherezov, Vadim; Liu, Hong; Stevens, Raymond C.; Zhao, Qiang; Wu, Beili</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6152</span>),
    <span class="NLM_cas:pages">1387-1390</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The CCR5 chemokine receptor acts as a co-receptor for HIV-1 viral entry.  Here we report the 2.7 angstrom-resoln. crystal structure of human CCR5 bound to the marketed HIV drug maraviroc.  The structure reveals a ligand-binding site that is distinct from the proposed major recognition sites for chemokines and the viral glycoprotein gp120, providing insights into the mechanism of allosteric inhibition of chemokine signaling and viral entry.  A comparison between CCR5 and CXCR4 crystal structures, along with models of co-receptor-gp120-V3 complexes, suggests that different charge distributions and steric hindrances caused by residue substitutions may be major determinants of HIV-1 co-receptor selectivity.  These high-resoln. insights into CCR5 can enable structure-based drug discovery for the treatment of HIV-1 infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt5VKeO5Hmb7Vg90H21EOLACvtfcHk0lgvEdl0edXmJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVCmu7vJ&md5=730ee5644a2a144cd49cd0e9a15eba2e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1126%2Fscience.1241475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1241475%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DQ.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DKufareva%26aufirst%3DI.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DFenalti%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DB.%26atitle%3DStructure%2520of%2520the%2520CCR5%2520chemokine%2520receptor-HIV%2520entry%2520inhibitor%2520maraviroc%2520complex%26jtitle%3DScience%26date%3D2013%26volume%3D341%26spage%3D1387%26epage%3D1390%26doi%3D10.1126%2Fscience.1241475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, R. K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiez-Vandal, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlenker, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aggeler, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dore, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geschwindner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grebner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermansson, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jazayeri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prihandoko, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rappas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soutter, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snijder, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundstrom, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tehan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troast, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiggin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhukov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dekker, N.</span></span> <span> </span><span class="NLM_article-title">Structural insight into allosteric modulation of protease-activated receptor 2</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>545</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">115</span>, <span class="refDoi"> DOI: 10.1038/nature22309</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1038%2Fnature22309" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=28445455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmslehsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=545&publication_year=2017&pages=112-115&author=R.+K.+Y.+Chengauthor=C.+Fiez-Vandalauthor=O.+Schlenkerauthor=K.+Edmanauthor=B.+Aggelerauthor=D.+G.+Brownauthor=G.+A.+Brownauthor=R.+M.+Cookeauthor=C.+E.+Dumelinauthor=A.+S.+Doreauthor=S.+Geschwindnerauthor=C.+Grebnerauthor=N.+O.+Hermanssonauthor=A.+Jazayeriauthor=P.+Johanssonauthor=L.+Leongauthor=R.+Prihandokoauthor=M.+Rappasauthor=H.+Soutterauthor=A.+Snijderauthor=L.+Sundstromauthor=B.+Tehanauthor=P.+Thorntonauthor=D.+Troastauthor=G.+Wigginauthor=A.+Zhukovauthor=F.+H.+Marshallauthor=N.+Dekker&title=Structural+insight+into+allosteric+modulation+of+protease-activated+receptor+2&doi=10.1038%2Fnature22309"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insight into allosteric modulation of protease-activated receptor 2</span></div><div class="casAuthors">Cheng, Robert K. Y.; Fiez-Vandal, Cedric; Schlenker, Oliver; Edman, Karl; Aggeler, Birte; Brown, Dean G.; Brown, Giles A.; Cooke, Robert M.; Dumelin, Christoph E.; Dore, Andrew S.; Geschwindner, Stefan; Grebner, Christoph; Hermansson, Nils-Olov; Jazayeri, Ali; Johansson, Patrik; Leong, Louis; Prihandoko, Rudi; Rappas, Mathieu; Soutter, Holly; Snijder, Arjan; Sundstrom, Linda; Tehan, Benjamin; Thornton, Peter; Troast, Dawn; Wiggin, Giselle; Zhukov, Andrei; Marshall, Fiona H.; Dekker, Niek</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">545</span>
        (<span class="NLM_cas:issue">7652</span>),
    <span class="NLM_cas:pages">112-115</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protease-activated receptors (PARs) are a family of G-protein-coupled receptors (GPCRs) that are irreversibly activated by proteolytic cleavage of the N-terminus, which unmasks a tethered peptide ligand that binds and activates the transmembrane receptor domain, eliciting a cellular cascade in response to inflammatory signals and other stimuli.  PARs are implicated in a wide range of diseases, such as cancer and inflammation.  PARs have been the subject of major pharmaceutical research efforts, but the discovery of small-mol. antagonists that effectively bind them has proved challenging.  The only marketed drug targeting a PAR is vorapaxar, a selective antagonist of PAR1 used to prevent thrombosis.  The structure of PAR1 in complex with vorapaxar has been reported previously.  Despite sequence homol. across the PAR isoforms, discovery of PAR2 antagonists has been less successful, although GB88 has been described as a weak antagonist.  Here, we report crystal structures of PAR2 in complex with two distinct antagonists and a blocking antibody.  The antagonist AZ8838 binds in a fully occluded pocket near the extracellular surface.  Functional and binding studies reveal that AZ8838 exhibits slow binding kinetics, which is an attractive feature for a PAR2 antagonist competing against a tethered ligand.  Antagonist AZ3451 binds to a remote allosteric site outside the helical bundle.  We propose that antagonist binding prevents structural rearrangements required for receptor activation and signalling.  We also show that a blocking antibody antigen-binding fragment binds to the extracellular surface of PAR2, preventing access of the tethered ligand to the peptide-binding site.  These structures provide a basis for the development of selective PAR2 antagonists for a range of therapeutic uses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSz2vmZJ2f6LVg90H21EOLACvtfcHk0lgvEdl0edXmJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmslehsrs%253D&md5=6a0737757c891336437d4419c3fd6c45</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnature22309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22309%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DR.%2BK.%2BY.%26aulast%3DFiez-Vandal%26aufirst%3DC.%26aulast%3DSchlenker%26aufirst%3DO.%26aulast%3DEdman%26aufirst%3DK.%26aulast%3DAggeler%26aufirst%3DB.%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DBrown%26aufirst%3DG.%2BA.%26aulast%3DCooke%26aufirst%3DR.%2BM.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DDore%26aufirst%3DA.%2BS.%26aulast%3DGeschwindner%26aufirst%3DS.%26aulast%3DGrebner%26aufirst%3DC.%26aulast%3DHermansson%26aufirst%3DN.%2BO.%26aulast%3DJazayeri%26aufirst%3DA.%26aulast%3DJohansson%26aufirst%3DP.%26aulast%3DLeong%26aufirst%3DL.%26aulast%3DPrihandoko%26aufirst%3DR.%26aulast%3DRappas%26aufirst%3DM.%26aulast%3DSoutter%26aufirst%3DH.%26aulast%3DSnijder%26aufirst%3DA.%26aulast%3DSundstrom%26aufirst%3DL.%26aulast%3DTehan%26aufirst%3DB.%26aulast%3DThornton%26aufirst%3DP.%26aulast%3DTroast%26aufirst%3DD.%26aulast%3DWiggin%26aufirst%3DG.%26aulast%3DZhukov%26aufirst%3DA.%26aulast%3DMarshall%26aufirst%3DF.%2BH.%26aulast%3DDekker%26aufirst%3DN.%26atitle%3DStructural%2520insight%2520into%2520allosteric%2520modulation%2520of%2520protease-activated%2520receptor%25202%26jtitle%3DNature%26date%3D2017%26volume%3D545%26spage%3D112%26epage%3D115%26doi%3D10.1038%2Fnature22309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ziemek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabrele, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buschauer, A.</span></span> <span> </span><span class="NLM_article-title">Determination of affinity and activity of ligands at the human neuropeptide Y Y4 receptor by flow cytometry and aequorin luminescence</span>. <i>J. Recept. Signal Transduction</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">217</span>– <span class="NLM_lpage">233</span>, <span class="refDoi"> DOI: 10.1080/10799890701505206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1080%2F10799890701505206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=17885919" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVGrs7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=217-233&author=R.+Ziemekauthor=E.+Schneiderauthor=A.+Krausauthor=C.+Cabreleauthor=A.+G.+Beck-Sickingerauthor=G.+Bernhardtauthor=A.+Buschauer&title=Determination+of+affinity+and+activity+of+ligands+at+the+human+neuropeptide+Y+Y4+receptor+by+flow+cytometry+and+aequorin+luminescence&doi=10.1080%2F10799890701505206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Determination of Affinity and Activity of Ligands at the Human Neuropeptide Y Y4 Receptor by Flow Cytometry and Aequorin Luminescence</span></div><div class="casAuthors">Ziemek, Ralf; Schneider, Erich; Kraus, Anja; Cabrele, Chiara; Beck-Sickinger, Annette G.; Bernhardt, Guenther; Buschauer, Armin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Receptors and Signal Transduction</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">217-233</span>CODEN:
                <span class="NLM_cas:coden">JRSTCT</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">Fluorescence-labeled neuropeptide Y (NPY) has been used in flow cytometric binding assays for the detn. of affinity consts. of NPY Y1, Y2, and Y5 receptor ligands.  Because the binding of fluorescent NPY is insufficient for competition studies at the human Y4 receptor (hY4R), the authors replaced Glu 4 in hPP with Lys for the derivatization with cyanine-5.  Because cy5-[K4]hPP has high affinity (Kd 5.6 nM) to the hY4R, it was used as a probe in a flow cytometric binding assay.  Specific binding of cy5-[K4]hPP to hY4R was visualized by confocal microscopy.  The hY4R, the chimeric G protein Gqi5 and mitochondrially targeted apoaequorin were stably coexpressed in CHO cells.  Aequorin luminescence was quantified in a microplate reader and by a CCD camera.  By application of these methods 3-cyclohexyl-N-[(3-1H-imidazol-4-ylpropylamino)(imino)methyl]propanamide (UR-AK49)was discovered as the first nonpeptidic Y4R antagonist (pKi 4.17), a lead to be optimized in terms of potency and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8DMLWtdk7rrVg90H21EOLACvtfcHk0lgvEdl0edXmJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVGrs7fI&md5=3c280bdb617c42f53c0745a20bac73a9</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1080%2F10799890701505206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10799890701505206%26sid%3Dliteratum%253Aachs%26aulast%3DZiemek%26aufirst%3DR.%26aulast%3DSchneider%26aufirst%3DE.%26aulast%3DKraus%26aufirst%3DA.%26aulast%3DCabrele%26aufirst%3DC.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26aulast%3DBernhardt%26aufirst%3DG.%26aulast%3DBuschauer%26aufirst%3DA.%26atitle%3DDetermination%2520of%2520affinity%2520and%2520activity%2520of%2520ligands%2520at%2520the%2520human%2520neuropeptide%2520Y%2520Y4%2520receptor%2520by%2520flow%2520cytometry%2520and%2520aequorin%2520luminescence%26jtitle%3DJ.%2520Recept.%2520Signal%2520Transduction%26date%3D2007%26volume%3D27%26spage%3D217%26epage%3D233%26doi%3D10.1080%2F10799890701505206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rudolf, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberlein, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieland, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willim, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Entzeroth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wienen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doods, H. N.</span></span> <span> </span><span class="NLM_article-title">The first highly potent and selective non-peptide neuropeptide Y Y1 receptor antagonist: BIBP3226</span>. <i>Eur. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>271</i></span>,  <span class="NLM_fpage">R11</span>– <span class="NLM_lpage">R13</span>, <span class="refDoi"> DOI: 10.1016/0014-2999(94)90822-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2F0014-2999%2894%2990822-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=7705422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADyaK2MXjt1Sntrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=271&publication_year=1994&pages=R11-R13&author=K.+Rudolfauthor=W.+Eberleinauthor=W.+Engelauthor=H.+A.+Wielandauthor=K.+D.+Willimauthor=M.+Entzerothauthor=W.+Wienenauthor=A.+G.+Beck-Sickingerauthor=H.+N.+Doods&title=The+first+highly+potent+and+selective+non-peptide+neuropeptide+Y+Y1+receptor+antagonist%3A+BIBP3226&doi=10.1016%2F0014-2999%2894%2990822-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The first highly potent and selective non-peptide neuropeptide Y Y1 receptor antagonist: BIBP3226</span></div><div class="casAuthors">Rudolf, Klaus; Eberlein, Wolfgang; Engel, Wolfhard; Wieland, Heike A.; Willim, Klaus D.; Entzeroth, Michael; Wienen, Wolfgang; Beck-Sickinger, Annette G.; Doods, Henri N.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmacology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">271</span>
        (<span class="NLM_cas:issue">2/3</span>),
    <span class="NLM_cas:pages">R11-R13</span>CODEN:
                <span class="NLM_cas:coden">EJPHAZ</span>;
        ISSN:<span class="NLM_cas:issn">0014-2999</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The design and subsequent in vitro and in vivo biol. characterization of the first potent and selective non-peptide neuropeptide Y Y1 receptor antagonist, BIBP3226 (I; (R)-N2-(diphenylacetyl)-N-[(4- hydroxyphenyl)methyl]-argininamide) is reported.  BIBP3226 displaced 125I-labeled neuropeptide Y with high affinity (Ki = 7 nM) from the human neuropeptide Y Y1 receptor and proved to be highly selective.  BIBP3226 displayed potent antagonistic properties both in in vitro and in vivo models and thus represents the first selective non-peptide neuropeptide Y Y1 receptor antagonist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoy6B_rW0NxBLVg90H21EOLACvtfcHk0lh1PzU9SUsaQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXjt1Sntrw%253D&md5=d0c87027fe726aacc2146433884d163a</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2F0014-2999%2894%2990822-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0014-2999%252894%252990822-2%26sid%3Dliteratum%253Aachs%26aulast%3DRudolf%26aufirst%3DK.%26aulast%3DEberlein%26aufirst%3DW.%26aulast%3DEngel%26aufirst%3DW.%26aulast%3DWieland%26aufirst%3DH.%2BA.%26aulast%3DWillim%26aufirst%3DK.%2BD.%26aulast%3DEntzeroth%26aufirst%3DM.%26aulast%3DWienen%26aufirst%3DW.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26aulast%3DDoods%26aufirst%3DH.%2BN.%26atitle%3DThe%2520first%2520highly%2520potent%2520and%2520selective%2520non-peptide%2520neuropeptide%2520Y%2520Y1%2520receptor%2520antagonist%253A%2520BIBP3226%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D1994%26volume%3D271%26spage%3DR11%26epage%3DR13%26doi%3D10.1016%2F0014-2999%2894%2990822-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wieland, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberlein, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolf, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doods, H. N.</span></span> <span> </span><span class="NLM_article-title">Subtype selectivity of the novel nonpeptide neuropeptide Y Y1 receptor antagonist BIBO 3304 and its effect on feeding in rodents</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">549</span>– <span class="NLM_lpage">555</span>, <span class="refDoi"> DOI: 10.1038/sj.bjp.0702084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1038%2Fsj.bjp.0702084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=9806339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADyaK1cXmvVSqt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=1998&pages=549-555&author=H.+A.+Wielandauthor=W.+Engelauthor=W.+Eberleinauthor=K.+Rudolfauthor=H.+N.+Doods&title=Subtype+selectivity+of+the+novel+nonpeptide+neuropeptide+Y+Y1+receptor+antagonist+BIBO+3304+and+its+effect+on+feeding+in+rodents&doi=10.1038%2Fsj.bjp.0702084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Subtype selectivity of the novel nonpeptide neuropeptide Y Y1 receptor antagonist BIBO 3304 and its effect on feeding in rodents</span></div><div class="casAuthors">Wieland, H. A.; Engel, W.; Eberlein, W.; Rudolf, K.; Doods, H. N.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">549-555</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Stockton Press</span>)
        </div><div class="casAbstract">The novel Y1-selective argininamide deriv. BIBO 3304 ((R)-N-[[4-(aminocarbonylaminomethyl)-phenyl]methyl]-N2-(diphenylacetyl)-argininamide trifluoroacetate) has been synthesized and was examd. for its subtype selectivity, its in vitro antagonistic properties and its food intake inhibitory properties.  BIBO 3304 displayed subnanomolar affinity for both the human and the rat Y1 receptor (IC50 values 0.38±0.06 nM and 0.72±0.42 nM, resp.).  The inactive enantiomer of BIBO 3304 (BIBO 3457) had low affinity for both the human and rat Y1 receptor subtype (IC50>1000 nM).  BIBO 3304 showed low affinity for the human Y2 receptor, human and rat Y4 receptor as well as for the human and rat Y5 receptor (IC50 values > 1000 nM).  30 μG BIBO 3304 administered into the paraventricular nucleus inhibited the feeding response induced by 1 μg NPY as well as the hyperphagia induced by a 24 h fast implying a role for Y1 receptors in NPY mediated feeding.  The inactive enantiomer had no effect.  BIBO 3304 inhibits neither the galanin nor the noradrenaline induced orexigenic response, but it blocked feeding behavior elicited by both [Leu31, Pro34]NPY and NPY (3-36) suggesting an interplay between different NPY receptor subtypes in feeding behavior.  The present study reveals that BIBO 3304 is a subtype selective nonpeptide antagonist with subnanomolar affinity for the Y1 receptor subtype that significantly inhibits food intake induced by application of NPY or by fasting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUdOSOrjTrcbVg90H21EOLACvtfcHk0lh1PzU9SUsaQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmvVSqt78%253D&md5=b7f588c39cb25b1adcd9dc0ae8f3f5a8</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0702084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0702084%26sid%3Dliteratum%253Aachs%26aulast%3DWieland%26aufirst%3DH.%2BA.%26aulast%3DEngel%26aufirst%3DW.%26aulast%3DEberlein%26aufirst%3DW.%26aulast%3DRudolf%26aufirst%3DK.%26aulast%3DDoods%26aufirst%3DH.%2BN.%26atitle%3DSubtype%2520selectivity%2520of%2520the%2520novel%2520nonpeptide%2520neuropeptide%2520Y%2520Y1%2520receptor%2520antagonist%2520BIBO%25203304%2520and%2520its%2520effect%2520on%2520feeding%2520in%2520rodents%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D1998%26volume%3D125%26spage%3D549%26epage%3D555%26doi%3D10.1038%2Fsj.bjp.0702084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D.</span></span> <span> </span><span class="NLM_article-title">The future of peptide-based drugs</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">136</span>– <span class="NLM_lpage">147</span>, <span class="refDoi"> DOI: 10.1111/cbdd.12055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1111%2Fcbdd.12055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=23253135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlt1yktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2013&pages=136-147&author=D.+J.+Craikauthor=D.+P.+Fairlieauthor=S.+Lirasauthor=D.+Price&title=The+future+of+peptide-based+drugs&doi=10.1111%2Fcbdd.12055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The future of peptide-based drugs</span></div><div class="casAuthors">Craik, David J.; Fairlie, David P.; Liras, Spiros; Price, David</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">136-147</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  The suite of currently used drugs can be divided into two categories - traditional "small mol." drugs with typical mol. wts. of <500 Da but with oral bioavailability, and much larger "biologics" typically >5000 Da that are not orally bioavailable and need to be delivered via injection.  Due to their small size, conventional small mol. drugs may suffer from reduced target selectivity that often ultimately manifests in human side-effects, whereas protein therapeutics tend to be exquisitely specific for their targets due to many more interactions with them, but this comes at a cost of low bioavailability, poor membrane permeability, and metabolic instability.  The time has now come to reinvestigate new drug leads that fit between these two mol. wt. extremes, with the goal of combining advantages of small mols. (cost, conformational restriction, membrane permeability, metabolic stability, oral bioavailability) with those of proteins (natural components, target specificity, high potency).  This article uses selected examples of peptides to highlight the importance of peptide drugs, some potential new opportunities for their exploitation, and some difficult challenges ahead in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlaEZbrPXLPbVg90H21EOLACvtfcHk0lh1PzU9SUsaQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlt1yktA%253D%253D&md5=1e7e4b7e725541587fad855e6a28cadf</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12055%26sid%3Dliteratum%253Aachs%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DPrice%26aufirst%3DD.%26atitle%3DThe%2520future%2520of%2520peptide-based%2520drugs%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2013%26volume%3D81%26spage%3D136%26epage%3D147%26doi%3D10.1111%2Fcbdd.12055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adrian, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenberg, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besterman, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bloom, S. R.</span></span> <span> </span><span class="NLM_article-title">Pharmacokinetics of pancreatic polypeptide in man</span>. <i>Gut</i> <span class="NLM_year" style="font-weight: bold;">1978</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">907</span>– <span class="NLM_lpage">909</span>, <span class="refDoi"> DOI: 10.1136/gut.19.10.907</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1136%2Fgut.19.10.907" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=568585" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADyaE1MXkvV2ksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1978&pages=907-909&author=T.+E.+Adrianauthor=G.+R.+Greenbergauthor=H.+S.+Bestermanauthor=S.+R.+Bloom&title=Pharmacokinetics+of+pancreatic+polypeptide+in+man&doi=10.1136%2Fgut.19.10.907"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacokinetics of pancreatic polypeptide in man</span></div><div class="casAuthors">Adrian, T. E.; Greenberg, G. R.; Besterman, H. S.; Bloom, S. R.</div><div class="citationInfo"><span class="NLM_cas:title">Gut</span>
        (<span class="NLM_cas:date">1978</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">907-9</span>CODEN:
                <span class="NLM_cas:coden">GUTTAK</span>;
        ISSN:<span class="NLM_cas:issn">0017-5749</span>.
    </div><div class="casAbstract">Pure bovine pancreatic polypeptide  [59763-91-6] (1, 3, or 5 pmol/kg/min) infused i.v. for 60 min into healthy subjects produced mean plasma levels of 203, 575, and 930 pmol/L, resp.  The mean disappearance half-time on stopping the infusion was 6.9 min.  The metabolic clearance rate was 5.1 mL/kg/min and the apparent vol. of distribution was 51 nL/kg.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1BerHjBAmt7Vg90H21EOLACvtfcHk0lgfBvpx586y4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1MXkvV2ksQ%253D%253D&md5=285d86d43cf7f83e4190f8e53daf0a9e</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1136%2Fgut.19.10.907&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fgut.19.10.907%26sid%3Dliteratum%253Aachs%26aulast%3DAdrian%26aufirst%3DT.%2BE.%26aulast%3DGreenberg%26aufirst%3DG.%2BR.%26aulast%3DBesterman%26aufirst%3DH.%2BS.%26aulast%3DBloom%26aufirst%3DS.%2BR.%26atitle%3DPharmacokinetics%2520of%2520pancreatic%2520polypeptide%2520in%2520man%26jtitle%3DGut%26date%3D1978%26volume%3D19%26spage%3D907%26epage%3D909%26doi%3D10.1136%2Fgut.19.10.907" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerald, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Criscione, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batzl-Hartmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaysse, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durkin, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laz, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linemeyer, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaffhauser, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitebread, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofbauer, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taber, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branchek, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinshank, R. L.</span></span> <span> </span><span class="NLM_article-title">A receptor subtype involved in neuropeptide-Y-induced food intake</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>382</i></span>,  <span class="NLM_fpage">168</span>– <span class="NLM_lpage">171</span>, <span class="refDoi"> DOI: 10.1038/382168a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1038%2F382168a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=8700207" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADyaK28XktlSkur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=382&publication_year=1996&pages=168-171&author=C.+Geraldauthor=M.+W.+Walkerauthor=L.+Criscioneauthor=E.+L.+Gustafsonauthor=C.+Batzl-Hartmannauthor=K.+E.+Smithauthor=P.+Vaysseauthor=M.+M.+Durkinauthor=T.+M.+Lazauthor=D.+L.+Linemeyerauthor=A.+O.+Schaffhauserauthor=S.+Whitebreadauthor=K.+G.+Hofbauerauthor=R.+I.+Taberauthor=T.+A.+Branchekauthor=R.+L.+Weinshank&title=A+receptor+subtype+involved+in+neuropeptide-Y-induced+food+intake&doi=10.1038%2F382168a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A receptor subtype involved in neuropeptide-Y-induced food intake</span></div><div class="casAuthors">Gerald, Christophe; Walker, Mary W.; Criscione, Leoluca; Gustafson, Eric L.; Batzl-Hartmann, Christine; Smith, Kelli E.; Vaysse, Pierre; Durkin, Margaret M.; Laz, Thomas M.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">382</span>
        (<span class="NLM_cas:issue">6587</span>),
    <span class="NLM_cas:pages">168-171</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Neuropeptide Y (NPY) is a powerful stimulant of food intake and is proposed to activate a hypothalamic 'feeding' receptor distinct from previously cloned Y-type receptors.  This receptor was first suggested to explain a feeding response to NPY and related peptides, including NPY2-36, that differed from their activities at the Y1 receptor.  Here the authors report the expression cloning of a novel Y-type receptor from rat hypothalamus, which the authors name Y5.  The complementary DNA encodes a 456-amino-acid protein with less than 35% overall identity to known Y-type receptors.  The mRNA is found primarily in the central nervous system, including the paraventricular nucleus of the hypothalamus.  The extent to which selected peptides can inhibit adenylate cyclase through the Y5 receptor and stimulate food intake in rats correspond well.  The authors' data support the idea that the Y5 receptor is the postulated 'feeding' receptor, and may provide a new method for the study and treatment of obesity and eating disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPe_-2lRIPvrVg90H21EOLACvtfcHk0lgfBvpx586y4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XktlSkur0%253D&md5=50601131172ab87888112111c412901b</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2F382168a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F382168a0%26sid%3Dliteratum%253Aachs%26aulast%3DGerald%26aufirst%3DC.%26aulast%3DWalker%26aufirst%3DM.%2BW.%26aulast%3DCriscione%26aufirst%3DL.%26aulast%3DGustafson%26aufirst%3DE.%2BL.%26aulast%3DBatzl-Hartmann%26aufirst%3DC.%26aulast%3DSmith%26aufirst%3DK.%2BE.%26aulast%3DVaysse%26aufirst%3DP.%26aulast%3DDurkin%26aufirst%3DM.%2BM.%26aulast%3DLaz%26aufirst%3DT.%2BM.%26aulast%3DLinemeyer%26aufirst%3DD.%2BL.%26aulast%3DSchaffhauser%26aufirst%3DA.%2BO.%26aulast%3DWhitebread%26aufirst%3DS.%26aulast%3DHofbauer%26aufirst%3DK.%2BG.%26aulast%3DTaber%26aufirst%3DR.%2BI.%26aulast%3DBranchek%26aufirst%3DT.%2BA.%26aulast%3DWeinshank%26aufirst%3DR.%2BL.%26atitle%3DA%2520receptor%2520subtype%2520involved%2520in%2520neuropeptide-Y-induced%2520food%2520intake%26jtitle%3DNature%26date%3D1996%26volume%3D382%26spage%3D168%26epage%3D171%26doi%3D10.1038%2F382168a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span> <span> </span><span class="NLM_article-title">Allosteric modulation and functional selectivity of G protein-coupled receptors</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">e237</span>– <span class="NLM_lpage">e243</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2012.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2Fj.ddtec.2012.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=24050274" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=e237-e243&author=Z.+G.+Gaoauthor=K.+A.+Jacobson&title=Allosteric+modulation+and+functional+selectivity+of+G+protein-coupled+receptors&doi=10.1016%2Fj.ddtec.2012.08.004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2012.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2012.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DZ.%2BG.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DAllosteric%2520modulation%2520and%2520functional%2520selectivity%2520of%2520G%2520protein-coupled%2520receptors%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2013%26volume%3D10%26spage%3De237%26epage%3De243%26doi%3D10.1016%2Fj.ddtec.2012.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wootten, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span> <span> </span><span class="NLM_article-title">Emerging paradigms in GPCR allostery: implications for drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">630</span>– <span class="NLM_lpage">644</span>, <span class="refDoi"> DOI: 10.1038/nrd4052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1038%2Fnrd4052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=23903222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1SjtbvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=630-644&author=D.+Woottenauthor=A.+Christopoulosauthor=P.+M.+Sexton&title=Emerging+paradigms+in+GPCR+allostery%3A+implications+for+drug+discovery&doi=10.1038%2Fnrd4052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging paradigms in GPCR allostery: implications for drug discovery</span></div><div class="casAuthors">Wootten, Denise; Christopoulos, Arthur; Sexton, Patrick M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">630-644</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Allosteric ligands bind to G protein-coupled receptors (GPCRs; also known as seven-transmembrane receptors) at sites that are distinct from the sites to which endogenous ligands bind.  The existence of allosteric ligands has enriched the ways in which the functions of GPCRs can be manipulated for potential therapeutic benefit, yet the complexity of their actions provides both challenges and opportunities for drug screening and development.  Converging avenues of research in areas such as biased signalling by allosteric ligands and the mechanisms by which allosteric ligands modulate the effects of diverse endogenous ligands have provided new insights into how interactions between allosteric ligands and GPCRs could be exploited for drug discovery.  These new findings have the potential to alter how screening for allosteric drugs is performed and may increase the chances of success in the development of allosteric modulators as clin. lead compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxRlmQUCUlsLVg90H21EOLACvtfcHk0lgfBvpx586y4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1SjtbvO&md5=59b74e5fbd5eeb832785dd879daa7c77</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnrd4052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4052%26sid%3Dliteratum%253Aachs%26aulast%3DWootten%26aufirst%3DD.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26atitle%3DEmerging%2520paradigms%2520in%2520GPCR%2520allostery%253A%2520implications%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D630%26epage%3D644%26doi%3D10.1038%2Fnrd4052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Graaf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span> <span> </span><span class="NLM_article-title">Structure and function of peptide-binding G protein-coupled receptors</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>429</i></span>,  <span class="NLM_fpage">2726</span>– <span class="NLM_lpage">2745</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2017.06.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2Fj.jmb.2017.06.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=28705763" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFygtrbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=429&publication_year=2017&pages=2726-2745&author=F.+Wuauthor=G.+Songauthor=C.+de+Graafauthor=R.+C.+Stevens&title=Structure+and+function+of+peptide-binding+G+protein-coupled+receptors&doi=10.1016%2Fj.jmb.2017.06.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and function of peptide-binding G protein-coupled receptors</span></div><div class="casAuthors">Wu, Fan; Song, Gaojie; de Graaf, Chris; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">429</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2726-2745</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  G protein-coupled receptors (GPCRs) are the largest family of cell surface receptors and are important human drug targets.  Of the 826 human GPCRs, 118 of them recognize endogenous peptide or protein ligands, and 30 of the 118 are targeted by approved drug mols., including the very high-profile class B glucagon-like peptide 1 receptor.  Here, we analyze the 21 exptl. detd. 3-dimensional structures of the known peptide-binding GPCRs in relation to the endogenous peptides and drug mols. that modulate their cell signaling processes.  The integrated analyses reveal that half of the marketed drugs and most of the drugs in clin. trials that interact with peptide GPCRs are small mols. with a wide range of binding modes distinct from those of large peptide ligands.  As we continue to collect addnl. data on these receptors from orthogonal approaches, including NMR and electron microscopy, we are beginning to understand how these receptors interact with their ligands at the mol. level and how improving the pharmacol. of GPCR signal transduction requires us to study these receptors using multiple biophys. techniques.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrn7cZECdBHg7Vg90H21EOLACvtfcHk0lggvQr6k2YvpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFygtrbE&md5=0b1f5ad22516e40f7552218862023e13</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2017.06.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2017.06.022%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DF.%26aulast%3DSong%26aufirst%3DG.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DStructure%2520and%2520function%2520of%2520peptide-binding%2520G%2520protein-coupled%2520receptors%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2017%26volume%3D429%26spage%3D2726%26epage%3D2745%26doi%3D10.1016%2Fj.jmb.2017.06.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mäde, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babilon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jolly, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wanka, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellmann-Sickert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz Gimenez, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mörl, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurevich, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span> <span> </span><span class="NLM_article-title">Peptide modifications differentially alter G protein-coupled receptor internalization and signaling bias</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">10067</span>– <span class="NLM_lpage">10071</span>, <span class="refDoi"> DOI: 10.1002/anie.201403750</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1002%2Fanie.201403750" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A280%3ADC%252BC2cbmsVOrsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=10067-10071&author=V.+M%C3%A4deauthor=S.+Babilonauthor=N.+Jollyauthor=L.+Wankaauthor=K.+Bellmann-Sickertauthor=L.+E.+Diaz+Gimenezauthor=K.+M%C3%B6rlauthor=H.+M.+Coxauthor=V.+V.+Gurevichauthor=A.+G.+Beck-Sickinger&title=Peptide+modifications+differentially+alter+G+protein-coupled+receptor+internalization+and+signaling+bias&doi=10.1002%2Fanie.201403750"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide modifications differentially alter G protein-coupled receptor internalization and signaling bias</span></div><div class="casAuthors">Made Veronika; Babilon Stefanie; Jolly Navjeet; Wanka Lizzy; Bellmann-Sickert Kathrin; Diaz Gimenez Luis E; Morl Karin; Cox Helen M; Gurevich Vsevolod V; Beck-Sickinger Annette G</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">10067-71</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Although G protein-coupled receptors (GPCRs) are targeted by more clinically used drugs than any other type of protein, their ligand development is particularly challenging.  Humans have four neuropeptide Y receptors: hY1R and hY5R are orexigenic, while hY2R and hY4R are anorexigenic, and represent important anti-obesity drug targets.  We show for the first time that PEGylation and lipidation, chemical modifications that prolong the plasma half-lives of peptides, confer additional benefits.  Both modifications enhance pancreatic polypeptide preference for hY2R/hY4R over hY1R/hY5R.  Lipidation biases the ligand towards arrestin recruitment and internalization, whereas PEGylation confers the opposite bias.  These effects were independent of the cell system and modified residue.  We thus provide novel insights into the mode of action of peptide modifications and open innovative venues for generating peptide agonists with extended therapeutic potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ9ClV0SIs_P2BAiYWy6n3SfW6udTcc2eYGyaeXJJPbx7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cbmsVOrsg%253D%253D&md5=24666ef122d28ca19ca72fada334feaf</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2Fanie.201403750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201403750%26sid%3Dliteratum%253Aachs%26aulast%3DM%25C3%25A4de%26aufirst%3DV.%26aulast%3DBabilon%26aufirst%3DS.%26aulast%3DJolly%26aufirst%3DN.%26aulast%3DWanka%26aufirst%3DL.%26aulast%3DBellmann-Sickert%26aufirst%3DK.%26aulast%3DDiaz%2BGimenez%26aufirst%3DL.%2BE.%26aulast%3DM%25C3%25B6rl%26aufirst%3DK.%26aulast%3DCox%26aufirst%3DH.%2BM.%26aulast%3DGurevich%26aufirst%3DV.%2BV.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26atitle%3DPeptide%2520modifications%2520differentially%2520alter%2520G%2520protein-coupled%2520receptor%2520internalization%2520and%2520signaling%2520bias%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D10067%26epage%3D10071%26doi%3D10.1002%2Fanie.201403750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Böhme, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stichel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walther, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mörl, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span> <span> </span><span class="NLM_article-title">Agonist induced receptor internalization of neuropeptide Y receptor subtypes depends on third intracellular loop and C-terminus</span>. <i>Cell. Signalling</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1740</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1016/j.cellsig.2008.05.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2Fj.cellsig.2008.05.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=18598760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A280%3ADC%252BD1crjtFSltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2008&pages=1740-1749&author=I.+B%C3%B6hmeauthor=J.+Stichelauthor=C.+Waltherauthor=K.+M%C3%B6rlauthor=A.+G.+Beck-Sickinger&title=Agonist+induced+receptor+internalization+of+neuropeptide+Y+receptor+subtypes+depends+on+third+intracellular+loop+and+C-terminus&doi=10.1016%2Fj.cellsig.2008.05.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Agonist induced receptor internalization of neuropeptide Y receptor subtypes depends on third intracellular loop and C-terminus</span></div><div class="casAuthors">Bohme Ilka; Stichel Jan; Walther Cornelia; Morl Karin; Beck-Sickinger Annette G</div><div class="citationInfo"><span class="NLM_cas:title">Cellular signalling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1740-9</span>
        ISSN:<span class="NLM_cas:issn">0898-6568</span>.
    </div><div class="casAbstract">Agonist stimulation of G-protein coupled receptors (GPCRs) results in the redistribution of the receptor from the cell surface into intracellular compartments through the process of endocytosis.  Monitoring ligand-mediated internalization of GPCRs in living cells has become experimentally accessible by applying fluorescent reagents and fluorescence microscopy.  By using cell lines that transiently, stably or endogenously express the human Y receptor (hYR) subtypes hY(1)R, hY(2)R, hY(4)R and hY(5)R and differently fluorescently tagged receptor proteins we were able to unravel further details concerning the internalization behavior of this multi-receptor/multi-ligand system.  For the first time we could show that also the hY(2)R is internalized with a rate which is comparable to the hY(1)R and the hY(4)R.  In contrast, the hY(5)R was internalized much slower and the rate remained unaffected by co-expression with other hYR subtypes.  Furthermore receptor subtype co-expressing cells and selectively binding peptides revealed a receptor subtype selective internalization.  By using novel hY(5)/hY(2) receptor chimera the receptor subtype dependent differences in hY receptor internalization could be identified on a molecular level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS1_oavs96kyXO4gDb7JggCfW6udTcc2eYGyaeXJJPbx7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1crjtFSltQ%253D%253D&md5=5eb7dfb86e75c35b92b95b5ee3775f39</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.cellsig.2008.05.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellsig.2008.05.017%26sid%3Dliteratum%253Aachs%26aulast%3DB%25C3%25B6hme%26aufirst%3DI.%26aulast%3DStichel%26aufirst%3DJ.%26aulast%3DWalther%26aufirst%3DC.%26aulast%3DM%25C3%25B6rl%26aufirst%3DK.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26atitle%3DAgonist%2520induced%2520receptor%2520internalization%2520of%2520neuropeptide%2520Y%2520receptor%2520subtypes%2520depends%2520on%2520third%2520intracellular%2520loop%2520and%2520C-terminus%26jtitle%3DCell.%2520Signalling%26date%3D2008%26volume%3D20%26spage%3D1740%26epage%3D1749%26doi%3D10.1016%2Fj.cellsig.2008.05.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kostenis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degtyarev, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conklin, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wess, J.</span></span> <span> </span><span class="NLM_article-title">The N-terminal extension of Gαq is critical for constraining the selectivity of receptor coupling</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>272</i></span>,  <span class="NLM_fpage">19107</span>– <span class="NLM_lpage">19110</span>, <span class="refDoi"> DOI: 10.1074/jbc.272.31.19107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1074%2Fjbc.272.31.19107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=9235898" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADyaK2sXltFSnuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=1997&pages=19107-19110&author=E.+Kostenisauthor=M.+Y.+Degtyarevauthor=B.+R.+Conklinauthor=J.+Wess&title=The+N-terminal+extension+of+G%CE%B1q+is+critical+for+constraining+the+selectivity+of+receptor+coupling&doi=10.1074%2Fjbc.272.31.19107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">The N-terminal extension of Gαq is critical for constraining the selectivity of receptor coupling</span></div><div class="casAuthors">Kostenis, Evi; Degtyarev, Michael Y.; Conklin, Bruce R.; Wess, Jurgen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">272</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">19107-19110</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Characteristically, an individual member of the superfamily of G protein-coupled receptors can interact only with a limited no. of the many structurally closely related G protein heterotrimers that are expressed within a cell.  Interestingly, the N termini of two G protein α subunits, Gαq and Gα11, differ from those of other α subunits in that they display a unique, highly conserved six-amino extension.  To test the hypothesis that this sequence element is crit. for proper receptor recognition, we prepd. a Gαq deletion mutant (-6q) lacking these first six amino acids.  The -6q construct (or wild type Gαq as a control) was coexpressed (in COS-7 cells) with several different Gi/o- or Gs-coupled receptors, and ligand-induced increases in inositol phosphate prodn. were detd. as a measure of G protein activation.  Whereas these receptors did not efficiently interact with wild type Gαq, most of them gained the ability to productively coupled to -6q.  Addnl. expts. indicated that the obsd. functional promiscuity of -6q is not due to overexpression (as compared with wild type Gαq) or a lack of palmitoylation.  We conclude that the N-terminal extension characteristic for Gαq/11 proteins is crit. for constraining the receptor coupling selectivity of these subunits, indicative of a novel mechanism by which the fidelity of receptor-G protein interactions can be regulated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzkDUJrMf9qbVg90H21EOLACvtfcHk0liEcnOHLKrZaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXltFSnuro%253D&md5=77e697d194898507c9cc27b508225279</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1074%2Fjbc.272.31.19107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.272.31.19107%26sid%3Dliteratum%253Aachs%26aulast%3DKostenis%26aufirst%3DE.%26aulast%3DDegtyarev%26aufirst%3DM.%2BY.%26aulast%3DConklin%26aufirst%3DB.%2BR.%26aulast%3DWess%26aufirst%3DJ.%26atitle%3DThe%2520N-terminal%2520extension%2520of%2520G%25CE%25B1q%2520is%2520critical%2520for%2520constraining%2520the%2520selectivity%2520of%2520receptor%2520coupling%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1997%26volume%3D272%26spage%3D19107%26epage%3D19110%26doi%3D10.1074%2Fjbc.272.31.19107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bryksin, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumura, I.</span></span> <span> </span><span class="NLM_article-title">Overlap extension PCR cloning: a simple and reliable way to create recombinant plasmids</span>. <i>BioTechniques</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">463</span>– <span class="NLM_lpage">465</span>, <span class="refDoi"> DOI: 10.2144/000113418</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.2144%2F000113418" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=20569222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjtVeisr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2010&pages=463-465&author=A.+V.+Bryksinauthor=I.+Matsumura&title=Overlap+extension+PCR+cloning%3A+a+simple+and+reliable+way+to+create+recombinant+plasmids&doi=10.2144%2F000113418"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Overlap extension PCR cloning: a simple and reliable way to create recombinant plasmids</span></div><div class="casAuthors">Bryksin, Anton V.; Matsumura, Ichiro</div><div class="citationInfo"><span class="NLM_cas:title">BioTechniques</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">463-465</span>CODEN:
                <span class="NLM_cas:coden">BTNQDO</span>;
        ISSN:<span class="NLM_cas:issn">0736-6205</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">Here we describe a straightforward, efficient, and reliable way to clone an insert of choice into a plasmid of choice without restriction endonucleases or T4 DNA ligase.  Chimeric primers contg. plasmid sequence at the 5' ends and insert sequence at the 3' ends were used to PCR-amplify insertion sequences of various sizes, namely the genes for GFP (g/f), β-D-glucuronidase (gusA), and β-galactosidase (lacZ), as well as the entire luxABCDE operon.  These inserts were employed as mega-primers in a second PCR with a circular plasmid template.  The original plasmid templates were then destroyed in restriction digests with DpnI, and the overlap extension PCR products were used to transform competent Escherichia coli cells.  Phusion DNA polymerase was used for the amplification and fusion reactions, so both reactions were easy to monitor and optimize.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQWvKNOXGKP7Vg90H21EOLACvtfcHk0liEcnOHLKrZaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjtVeisr8%253D&md5=22658b78ee0555be56a20beff6b4225b</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.2144%2F000113418&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2144%252F000113418%26sid%3Dliteratum%253Aachs%26aulast%3DBryksin%26aufirst%3DA.%2BV.%26aulast%3DMatsumura%26aufirst%3DI.%26atitle%3DOverlap%2520extension%2520PCR%2520cloning%253A%2520a%2520simple%2520and%2520reliable%2520way%2520to%2520create%2520recombinant%2520plasmids%26jtitle%3DBioTechniques%26date%3D2010%26volume%3D48%26spage%3D463%26epage%3D465%26doi%3D10.2144%2F000113418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ho, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horton, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pullen, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pease, L. R.</span></span> <span> </span><span class="NLM_article-title">Site-directed mutagenesis by overlap extension using the polymerase chain reaction</span>. <i>Gene</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">59</span>, <span class="refDoi"> DOI: 10.1016/0378-1119(89)90358-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2F0378-1119%2889%2990358-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=2744487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADyaL1MXktVaitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=1989&pages=51-59&author=S.+N.+Hoauthor=H.+D.+Huntauthor=R.+M.+Hortonauthor=J.+K.+Pullenauthor=L.+R.+Pease&title=Site-directed+mutagenesis+by+overlap+extension+using+the+polymerase+chain+reaction&doi=10.1016%2F0378-1119%2889%2990358-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Site-directed mutagenesis by overlap extension using the polymerase chain reaction</span></div><div class="casAuthors">Ho, Steffan N.; Hunt, Henry D.; Horton, Robert M.; Pullen, Jeffrey K.; Pease, Larry R.</div><div class="citationInfo"><span class="NLM_cas:title">Gene</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">51-9</span>CODEN:
                <span class="NLM_cas:coden">GENED6</span>;
        ISSN:<span class="NLM_cas:issn">0378-1119</span>.
    </div><div class="casAbstract">Overlap extension represents a new approach to genetic engineering.  Complementary oligodeoxyribonucleotide (oligo) primers and the polymerase chain reaction are used to generate two DNA fragments having overlapping ends.  These fragments are combined in a subsequent fusion reaction in which the overlapping ends anneal, allowing the 3' overlap of each strand to serve as a primer for the 3' extension of the complementary strand.  The resulting fusion product is amplified further by PCR.  Specific alterations in the nucleotide (nt) sequence can be introduced by incorporating nucleotide changes into the overlapping oligo primers.  Using this technique of site-directed mutagenesis, 3 variants of a mouse major histocompatibility complex class-I gene have been generated, cloned and analyzed.  Screening of mutant clones revealed at least a 98% efficiency of mutagenesis.  All clones sequenced contained the desired mutations, and a low frequency of random substitution estd. to occur at approx. 1 in 400 nt was detected.  This method represents a significant improvement over std. methods of site-directed mutagenesis because it is much faster, simpler and approaches 100% efficiency in the generation of mutant product.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqebn9qfaVP9rVg90H21EOLACvtfcHk0liEcnOHLKrZaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXktVaitrk%253D&md5=a2587cbdaec79a12306c022825570e19</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2F0378-1119%2889%2990358-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0378-1119%252889%252990358-2%26sid%3Dliteratum%253Aachs%26aulast%3DHo%26aufirst%3DS.%2BN.%26aulast%3DHunt%26aufirst%3DH.%2BD.%26aulast%3DHorton%26aufirst%3DR.%2BM.%26aulast%3DPullen%26aufirst%3DJ.%2BK.%26aulast%3DPease%26aufirst%3DL.%2BR.%26atitle%3DSite-directed%2520mutagenesis%2520by%2520overlap%2520extension%2520using%2520the%2520polymerase%2520chain%2520reaction%26jtitle%3DGene%26date%3D1989%26volume%3D77%26spage%3D51%26epage%3D59%26doi%3D10.1016%2F0378-1119%2889%2990358-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bradford, M. M.</span></span> <span> </span><span class="NLM_article-title">A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1976</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">248</span>– <span class="NLM_lpage">254</span>, <span class="refDoi"> DOI: 10.1016/0003-2697(76)90527-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1016%2F0003-2697%2876%2990527-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=942051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADyaE28XksVehtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=1976&pages=248-254&author=M.+M.+Bradford&title=A+rapid+and+sensitive+method+for+the+quantitation+of+microgram+quantities+of+protein+utilizing+the+principle+of+protein-dye+binding&doi=10.1016%2F0003-2697%2876%2990527-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding</span></div><div class="casAuthors">Bradford, Marion M.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">248-54</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">A protein detn. method that involves the binding of coomassie Brilliant Blue G 250 to protein is described.  The binding of the dye to protein causes a shift in the absorption max. of the dye from 465 to 595 nm, and it is the increase in absorption at 595 nm that is monitored.  This assay is very reproducible and rapid with the dye binding process virtually complete in ∼ 2 min with good color stability for 1 hr.  There is little or no interference from cations such as Na+ or K+ nor from carbohydrates such as sucrose.  A small amt. of color is developed in the presence of strongly alk. buffering agents, but the assay may be run accurately by the use of proper buffer controls.  The only components found to give excessive interfering color in the assay are relatively large amts. of detergents such as Na dodecyl sulfate, Triton X 100, and commercial glassware detergents.  Interference by small amts. of detergent may be eliminated by the use of proper control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN_T_d0trcNrVg90H21EOLACvtfcHk0liEcnOHLKrZaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28XksVehtrY%253D&md5=43f388a493becc4193c7a49d9de0e1db</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2F0003-2697%2876%2990527-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-2697%252876%252990527-3%26sid%3Dliteratum%253Aachs%26aulast%3DBradford%26aufirst%3DM.%2BM.%26atitle%3DA%2520rapid%2520and%2520sensitive%2520method%2520for%2520the%2520quantitation%2520of%2520microgram%2520quantities%2520of%2520protein%2520utilizing%2520the%2520principle%2520of%2520protein-dye%2520binding%26jtitle%3DAnal.%2520Biochem.%26date%3D1976%26volume%3D72%26spage%3D248%26epage%3D254%26doi%3D10.1016%2F0003-2697%2876%2990527-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sievers, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilm, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dineen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karplus, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McWilliam, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remmert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Söding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higgins, D. G.</span></span> <span> </span><span class="NLM_article-title">Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega</span>. <i>Mol. Syst. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">539</span> <span class="refDoi"> DOI: 10.1038/msb.2011.75</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1038%2Fmsb.2011.75" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=21988835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A280%3ADC%252BC3MbgtlantA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&author=F.+Sieversauthor=A.+Wilmauthor=D.+Dineenauthor=T.+J.+Gibsonauthor=K.+Karplusauthor=W.+Liauthor=R.+Lopezauthor=H.+McWilliamauthor=M.+Remmertauthor=J.+S%C3%B6dingauthor=J.+D.+Thompsonauthor=D.+G.+Higgins&title=Fast%2C+scalable+generation+of+high-quality+protein+multiple+sequence+alignments+using+Clustal+Omega&doi=10.1038%2Fmsb.2011.75"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega</span></div><div class="casAuthors">Sievers Fabian; Wilm Andreas; Dineen David; Gibson Toby J; Karplus Kevin; Li Weizhong; Lopez Rodrigo; McWilliam Hamish; Remmert Michael; Soding Johannes; Thompson Julie D; Higgins Desmond G</div><div class="citationInfo"><span class="NLM_cas:title">Molecular systems biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">539</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Multiple sequence alignments are fundamental to many sequence analysis methods.  Most alignments are computed using the progressive alignment heuristic.  These methods are starting to become a bottleneck in some analysis pipelines when faced with data sets of the size of many thousands of sequences.  Some methods allow computation of larger data sets while sacrificing quality, and others produce high-quality alignments, but scale badly with the number of sequences.  In this paper, we describe a new program called Clustal Omega, which can align virtually any number of protein sequences quickly and that delivers accurate alignments.  The accuracy of the package on smaller test cases is similar to that of the high-quality aligners.  On larger data sets, Clustal Omega outperforms other packages in terms of execution time and quality.  Clustal Omega also has powerful features for adding sequences to and exploiting information in existing alignments, making use of the vast amount of precomputed information in public databases like Pfam.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQD21C9zX5warjnyZkbE8yXfW6udTcc2ebYyZUUCb-R4bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MbgtlantA%253D%253D&md5=27dd8fac22447528e6d09e5fd1066133</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fmsb.2011.75&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmsb.2011.75%26sid%3Dliteratum%253Aachs%26aulast%3DSievers%26aufirst%3DF.%26aulast%3DWilm%26aufirst%3DA.%26aulast%3DDineen%26aufirst%3DD.%26aulast%3DGibson%26aufirst%3DT.%2BJ.%26aulast%3DKarplus%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLopez%26aufirst%3DR.%26aulast%3DMcWilliam%26aufirst%3DH.%26aulast%3DRemmert%26aufirst%3DM.%26aulast%3DS%25C3%25B6ding%26aufirst%3DJ.%26aulast%3DThompson%26aufirst%3DJ.%2BD.%26aulast%3DHiggins%26aufirst%3DD.%2BG.%26atitle%3DFast%252C%2520scalable%2520generation%2520of%2520high-quality%2520protein%2520multiple%2520sequence%2520alignments%2520using%2520Clustal%2520Omega%26jtitle%3DMol.%2520Syst.%2520Biol.%26date%3D2011%26volume%3D7%26doi%3D10.1038%2Fmsb.2011.75" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bender, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cisneros, A.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duran, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokits, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sangha, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauer, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sevy, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sliwoski, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheehan, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiMaio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meiler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moretti, R.</span></span> <span> </span><span class="NLM_article-title">Protocols for molecular modeling with Rosetta3 and RosettaScripts</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">4748</span>– <span class="NLM_lpage">4763</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.6b00444</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.6b00444" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1ynt73M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=4748-4763&author=B.+J.+Benderauthor=A.+Cisnerosauthor=A.+M.+Duranauthor=J.+A.+Finnauthor=D.+Fuauthor=A.+D.+Lokitsauthor=B.+K.+Muellerauthor=A.+K.+Sanghaauthor=M.+F.+Sauerauthor=A.+M.+Sevyauthor=G.+Sliwoskiauthor=J.+H.+Sheehanauthor=F.+DiMaioauthor=J.+Meilerauthor=R.+Moretti&title=Protocols+for+molecular+modeling+with+Rosetta3+and+RosettaScripts&doi=10.1021%2Facs.biochem.6b00444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Protocols for Molecular Modeling with Rosetta3 and RosettaScripts</span></div><div class="casAuthors">Bender, Brian J.; Cisneros, Alberto; Duran, Amanda M.; Finn, Jessica A.; Fu, Darwin; Lokits, Alyssa D.; Mueller, Benjamin K.; Sangha, Amandeep K.; Sauer, Marion F.; Sevy, Alexander M.; Sliwoski, Gregory; Sheehan, Jonathan H.; DiMaio, Frank; Meiler, Jens; Moretti, Rocco</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">4748-4763</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Previously, we published an article providing an overview of the Rosetta suite of biomacromol. modeling software and a series of step-by-step tutorials [Kaufmann, K. W. et al. (2010) Biochem. 49, 2987-2998].  The overwhelming pos. response we received to this publication motivates us to here share the next iteration of these tutorials which feature de novo folding, comparative modeling, loop construction, protein docking, small mol. docking, and protein design.  This updated and expanded set of tutorials is needed, as since 2010 Rosetta has been fully re into an object-oriented protein modeling program Rosetta3.  Notable improvements include a substantially improved energy function, an XML-like language termed "RosettaScripts" for flexibly specifying modeling task, new anal. tools, the addn. of the TopologyBroker to control conformational sampling, and support for multiple templates in comparative modeling.  Rosetta's ability to model systems with sym. proteins, membrane proteins, non-canonical amino acids and RNA has also been greatly expanded and improved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCdqi-g3EfybVg90H21EOLACvtfcHk0ljS89tmfhjavw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1ynt73M&md5=c540bd135fdf2753665249347e1779a8</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.6b00444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.6b00444%26sid%3Dliteratum%253Aachs%26aulast%3DBender%26aufirst%3DB.%2BJ.%26aulast%3DCisneros%26aufirst%3DA.%26aulast%3DDuran%26aufirst%3DA.%2BM.%26aulast%3DFinn%26aufirst%3DJ.%2BA.%26aulast%3DFu%26aufirst%3DD.%26aulast%3DLokits%26aufirst%3DA.%2BD.%26aulast%3DMueller%26aufirst%3DB.%2BK.%26aulast%3DSangha%26aufirst%3DA.%2BK.%26aulast%3DSauer%26aufirst%3DM.%2BF.%26aulast%3DSevy%26aufirst%3DA.%2BM.%26aulast%3DSliwoski%26aufirst%3DG.%26aulast%3DSheehan%26aufirst%3DJ.%2BH.%26aulast%3DDiMaio%26aufirst%3DF.%26aulast%3DMeiler%26aufirst%3DJ.%26aulast%3DMoretti%26aufirst%3DR.%26atitle%3DProtocols%2520for%2520molecular%2520modeling%2520with%2520Rosetta3%2520and%2520RosettaScripts%26jtitle%3DBiochemistry%26date%3D2016%26volume%3D55%26spage%3D4748%26epage%3D4763%26doi%3D10.1021%2Facs.biochem.6b00444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, N.</span>; <span class="NLM_string-name">Woetzel, N.</span>; <span class="NLM_string-name">Meiler, J.</span></span> In  <i>Bcl::Cluster: A Method for Clustering Biological Molecules Coupled with Visualization in the Pymol Molecular Graphics System</i>, 2011 IEEE 1st International Conference on Computational Advances in Bio and Medical Sciences (ICCABS), <span class="NLM_year">2011</span>; pp  <span class="NLM_fpage">13</span>– <span class="NLM_lpage">18</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&pages=13-18&author=N.+Alexander&author=N.+Woetzel&author=J.+Meiler&title=Bcl%3A%3ACluster%3A+A+Method+for+Clustering+Biological+Molecules+Coupled+with+Visualization+in+the+Pymol+Molecular+Graphics+System"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DAlexander%26aufirst%3DN.%26jtitle%3DBcl%253A%253ACluster%253A%2520A%2520Method%2520for%2520Clustering%2520Biological%2520Molecules%2520Coupled%2520with%2520Visualization%2520in%2520the%2520Pymol%2520Molecular%2520Graphics%2520System%26date%3D2011%26spage%3D13%26epage%3D18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kothiwale, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendenhall, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meiler, J.</span></span> <span> </span><span class="NLM_article-title">BCL::Conf: small molecule conformational sampling using a knowledge based rotamer library</span>. <i>J. Cheminf.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">47</span> <span class="refDoi"> DOI: 10.1186/s13321-015-0095-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1186%2Fs13321-015-0095-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC28Xos1ejtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&author=S.+Kothiwaleauthor=J.+L.+Mendenhallauthor=J.+Meiler&title=BCL%3A%3AConf%3A+small+molecule+conformational+sampling+using+a+knowledge+based+rotamer+library&doi=10.1186%2Fs13321-015-0095-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">BCL::CONF: small molecule conformational sampling using a knowledge based rotamer library</span></div><div class="casAuthors">Kothiwale, Sandeepkumar; Mendenhall, Jeffrey L.; Meiler, Jens</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">47/1-47/15</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">The interaction of a small mol. with a protein target depends on its ability to adopt a three-dimensional structure that is complementary.  Therefore, complete and rapid prediction of the conformational space a small mol. can sample is crit. for both structure- and ligand-based drug discovery algorithms such as small mol. docking or three-dimensional quant. structure-activity relationships.  Here we have derived a database of small mol. fragments frequently sampled in exptl. structures within the Cambridge Structure Database and the Protein Data Bank.  Likely conformations of these fragments are stored as 'rotamers' in analogy to amino acid side chain rotamer libraries used for rapid sampling of protein conformational space.  Explicit fragments take into account correlations between multiple torsion bonds and effect of substituents on torsional profiles.  A conformational ensemble for small mols. can then be generated by recombining fragment rotamers with a Monte Carlo search strategy.  BCL::CONF was benchmarked against other conformer generator methods including CONFGEN, MOE, OMEGA and RDKIT in its ability to recover exptl. detd. protein bound conformations of small mols., diversity of conformational ensembles, and sampling rate.  BCL::CONF recovers at least one conformation with a root mean square deviation of 2 Å or better to the exptl. structure for 99 % of the small mols. in the VERNALIS benchmark dataset.  The 'rotamer' approach will allow integration of BCL::CONF into resp. computational biol. programs such as ROSETTA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroCP70my1ks7Vg90H21EOLACvtfcHk0lho3rbg1KGwfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xos1ejtLY%253D&md5=a08c0fd16b2b4a526a41f9ac6f6559d0</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1186%2Fs13321-015-0095-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-015-0095-1%26sid%3Dliteratum%253Aachs%26aulast%3DKothiwale%26aufirst%3DS.%26aulast%3DMendenhall%26aufirst%3DJ.%2BL.%26aulast%3DMeiler%26aufirst%3DJ.%26atitle%3DBCL%253A%253AConf%253A%2520small%2520molecule%2520conformational%2520sampling%2520using%2520a%2520knowledge%2520based%2520rotamer%2520library%26jtitle%3DJ.%2520Cheminf.%26date%3D2015%26volume%3D7%26doi%3D10.1186%2Fs13321-015-0095-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moretti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allison, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meiler, J.</span></span> <span> </span><span class="NLM_article-title">Rosetta and the design of ligand binding sites</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>1414</i></span>,  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">62</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-3569-7_4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=10.1007%2F978-1-4939-3569-7_4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=27094285" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpsFWrsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1414&publication_year=2016&pages=47-62&author=R.+Morettiauthor=B.+J.+Benderauthor=B.+Allisonauthor=J.+Meiler&title=Rosetta+and+the+design+of+ligand+binding+sites&doi=10.1007%2F978-1-4939-3569-7_4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Rosetta and the Design of Ligand Binding Sites</span></div><div class="casAuthors">Moretti, Rocco; Bender, Brian J.; Allison, Brittany; Meiler, Jens</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">1414</span>
        (<span class="NLM_cas:issue">Computational Design of Ligand Binding Proteins</span>),
    <span class="NLM_cas:pages">47-62</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Proteins that bind small mols. (ligands) can be used as biosensors, signal modulators, and sequestering agents.  When naturally occurring proteins for a particular target ligand are not available, artificial proteins can be computationally designed.  We present a protocol based on RosettaLigand to redesign an existing protein pocket to bind a target ligand.  Starting with a protein structure and the structure of the ligand, Rosetta can optimize both the placement of the ligand in the pocket and the identity and conformation of the surrounding sidechains, yielding proteins that bind the target compd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpGdSJp5nS2bVg90H21EOLACvtfcHk0lho3rbg1KGwfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpsFWrsrk%253D&md5=5bd1f176ef8793b183ea5a926307204c</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-3569-7_4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-3569-7_4%26sid%3Dliteratum%253Aachs%26aulast%3DMoretti%26aufirst%3DR.%26aulast%3DBender%26aufirst%3DB.%2BJ.%26aulast%3DAllison%26aufirst%3DB.%26aulast%3DMeiler%26aufirst%3DJ.%26atitle%3DRosetta%2520and%2520the%2520design%2520of%2520ligand%2520binding%2520sites%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2016%26volume%3D1414%26spage%3D47%26epage%3D62%26doi%3D10.1007%2F978-1-4939-3569-7_4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kenakin, T. P.</span></span> <span> </span><span class="NLM_article-title">Allosteric Drug Antagonism. A Pharmacology Primer—Theory</span>.  <i>Applications, and Methods</i>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_publisher-name">Elsevier Academic Press</span>: <span class="NLM_publisher-loc">Amsterdam, Boston</span>, <span class="NLM_year">2006</span>; pp  <span class="NLM_fpage">143</span>– <span class="NLM_lpage">145</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&pages=143-145&author=T.+P.+Kenakin&title=Applications%2C+and+Methods"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DKenakin%26aufirst%3DT.%2BP.%26atitle%3DAllosteric%2520Drug%2520Antagonism.%2520A%2520Pharmacology%2520Primer%25E2%2580%2594Theory%26btitle%3DApplications%252C%2520and%2520Methods%26pub%3DElsevier%2520Academic%2520Press%26date%3D2006%26spage%3D143%26epage%3D145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ZBQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ZBQ','PDB','5ZBQ'); return false;">PDB: 5ZBQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4S0V" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4S0V','PDB','4S0V'); return false;">PDB: 4S0V</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZJC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZJC','PDB','4ZJC'); return false;">PDB: 4ZJC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DHG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DHG','PDB','5DHG'); return false;">PDB: 5DHG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N6H" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N6H','PDB','4N6H'); return false;">PDB: 4N6H</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DJH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DJH','PDB','4DJH'); return false;">PDB: 4DJH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DKL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DKL','PDB','4DKL'); return false;">PDB: 4DKL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5GLH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5GLH','PDB','5GLH'); return false;">PDB: 5GLH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ODU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ODU','PDB','3ODU'); return false;">PDB: 3ODU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NDD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NDD','PDB','5NDD'); return false;">PDB: 5NDD</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i45"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02000">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_22182"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c02000?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02000</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Effect of (<i>S</i>)-VU0637120 on G protein activation, internalization studies, Y<sub>4</sub>R variant screening for the investigation of the VU0637120 binding pocket, RosettaLigand docking clusters, analytics of compounds, and synthetic peptide ligands (Figures S1–S14); small-molecule screening data, PP activation of Y<sub>4</sub>R/Y1R chimeras and Y<sub>4</sub>R variants, templates for Y<sub>4</sub>R homology modeling, overview of docking parameters (Tables S1–S7) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Coordinates of final homology models (A3.29P-Y4-NAM.pdb) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_002.pdb">PDB</a>)</p></li><li><p class="inline">Coordinates of final homology models (A3.29V-Y4-NAM.pdb) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_003.pdb">PDB</a>)</p></li><li><p class="inline">Coordinates of final homology models (E6.58R-Y4-NAM.pdb) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_004.pdb">PDB</a>)</p></li><li><p class="inline">Coordinates of final homology models (Y4-NAM-cluster1.pdb) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_005.pdb">PDB</a>)</p></li><li><p class="inline">Coordinates of final homology models (Y4-NAM-cluster2.pdb) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_006.pdb">PDB</a>)</p></li><li><p class="inline">Coordinates of final homology models (Y4-NAM-cluster3.pdb) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_007.pdb">PDB</a>)</p></li><li><p class="inline">Coordinates of final homology models (Y4-PP.pdb) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_008.pdb">PDB</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_009.csv">CSV</a>, <a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_010.tif">TIF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_001.pdf">jm0c02000_si_001.pdf (3.11 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_002.pdb">jm0c02000_si_002.pdb (195.89 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_003.pdb">jm0c02000_si_003.pdb (195.89 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_004.pdb">jm0c02000_si_004.pdb (195.89 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_005.pdb">jm0c02000_si_005.pdb (396.11 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_006.pdb">jm0c02000_si_006.pdb (396.11 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_007.pdb">jm0c02000_si_007.pdb (396.11 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_008.pdb">jm0c02000_si_008.pdb (423.43 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_009.csv">jm0c02000_si_009.csv (0.33 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02000/suppl_file/jm0c02000_si_010.tif">jm0c02000_si_010.tif (295.51 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c02000&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c02000%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-5%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c02000" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799316ad9f13c46","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
