
<html lang="en"     class="pb-page"  data-request-id="d2ae4948-7e35-453d-880f-a32cf9b35921"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2020.63.issue-15;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.jmedchem.0c00630;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery and Characterization of Pure RhlR Antagonists against Pseudomonas aeruginosa Infections" /></meta><meta name="dc.Creator" content="SangJin  Nam" /></meta><meta name="dc.Creator" content="So-Young  Ham" /></meta><meta name="dc.Creator" content="Hongmok  Kwon" /></meta><meta name="dc.Creator" content="Han-Shin  Kim" /></meta><meta name="dc.Creator" content="Suhyun  Moon" /></meta><meta name="dc.Creator" content="Jeong-Hoon  Lee" /></meta><meta name="dc.Creator" content="Taehyeong  Lim" /></meta><meta name="dc.Creator" content="Sang-Hyun  Son" /></meta><meta name="dc.Creator" content="Hee-Deung  Park" /></meta><meta name="dc.Creator" content="Youngjoo  Byun" /></meta><meta name="dc.Description" content="Pseudomonas aeruginosa (P. aeruginosa) is an opportunistic human pathogen that forms biofilms and produces virulence factors via quorum sensing (QS). Blocking the QS system in P. aeruginosa is an e..." /></meta><meta name="Description" content="Pseudomonas aeruginosa (P. aeruginosa) is an opportunistic human pathogen that forms biofilms and produces virulence factors via quorum sensing (QS). Blocking the QS system in P. aeruginosa is an e..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 22, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00630" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00630" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00630" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00630" /></link>
        
    
    

<title>Discovery and Characterization of Pure RhlR Antagonists against Pseudomonas aeruginosa Infections | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00630" /></meta><meta property="og:title" content="Discovery and Characterization of Pure RhlR Antagonists against Pseudomonas aeruginosa Infections" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0013.jpeg" /></meta><meta property="og:description" content="Pseudomonas aeruginosa (P. aeruginosa) is an opportunistic human pathogen that forms biofilms and produces virulence factors via quorum sensing (QS). Blocking the QS system in P. aeruginosa is an excellent strategy to reduce biofilm formation and the production of virulence factors. RhlR plays an essential role in the QS system of P. aeruginosa. We synthesized 55 analogues based on the chemical structure of 4-gingerol and evaluated their RhlR inhibitory activities using the cell-based reporter strain assay. Comprehensive structure–activity relationship studies identified the alkynyl ketone 30 as the most potent RhlR antagonist. This compound displayed selective RhlR antagonism over LasR and PqsR, strong inhibition of biofilm formation, and reduced production of virulence factors in P. aeruginosa. Furthermore, the survival rate of Tenebrio molitor larvae treated with 30 in vivo greatly improved. Therefore, compound 30, a pure RhlR antagonist, can be utilized for developing QS-modulating molecules in the control of P. aeruginosa infections." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00630"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00630">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00630&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00630&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00630&amp;href=/doi/10.1021/acs.jmedchem.0c00630" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 15</span><span class="cit-fg-pageRange">, 8388-8407</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/15" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00613" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00754" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery and Characterization of Pure RhlR Antagonists against <i>Pseudomonas aeruginosa</i> Infections</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">SangJin Nam</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">SangJin Nam</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy, Korea University, 2511 Sejong-ro, Jochiwon-eup, Sejong 30019, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=SangJin++Nam">SangJin Nam</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">So-Young Ham</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">So-Young Ham</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Civil, Environmental and Architectural Engineering, Korea University, 145 Anam-ro, Seongbuk-Gu, Seoul 02841, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=So-Young++Ham">So-Young Ham</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hongmok Kwon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hongmok Kwon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy, Korea University, 2511 Sejong-ro, Jochiwon-eup, Sejong 30019, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hongmok++Kwon">Hongmok Kwon</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Han-Shin Kim</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Han-Shin Kim</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Korean Peninsula Infrastructure Cooperation Team, Korea Institute of Civil Engineering and Building Technology (KICT), Goyang-si, Gyeonggi-do 10223, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Han-Shin++Kim">Han-Shin Kim</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Suhyun Moon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Suhyun Moon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy, Korea University, 2511 Sejong-ro, Jochiwon-eup, Sejong 30019, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Suhyun++Moon">Suhyun Moon</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jeong-Hoon Lee</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jeong-Hoon Lee</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Civil, Environmental and Architectural Engineering, Korea University, 145 Anam-ro, Seongbuk-Gu, Seoul 02841, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jeong-Hoon++Lee">Jeong-Hoon Lee</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Taehyeong Lim</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Taehyeong Lim</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy, Korea University, 2511 Sejong-ro, Jochiwon-eup, Sejong 30019, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Taehyeong++Lim">Taehyeong Lim</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sang-Hyun Son</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sang-Hyun Son</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy, Korea University, 2511 Sejong-ro, Jochiwon-eup, Sejong 30019, Republic of Korea</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sang-Hyun++Son">Sang-Hyun Son</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8624-078X" title="Orcid link">http://orcid.org/0000-0002-8624-078X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hee-Deung Park</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hee-Deung Park</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Civil, Environmental and Architectural Engineering, Korea University, 145 Anam-ro, Seongbuk-Gu, Seoul 02841, Republic of Korea</div><div class="loa-info-affiliations-info">KU-KIST Graduate School of Converging Science and Technology, Korea University, 145 Anam-ro, Seongbuk-Gu, Seoul 02841, Republic of Korea</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#2a424f4f4e4f5f444d6a4145584f4b044b49044158"><span class="__cf_email__" data-cfemail="9df5f8f8f9f8e8f3faddf6f2eff8fcb3fcfeb3f6ef">[email protected]</span></a>. Phone: +82-2-3290-4861. Fax: +82-2-928-7656.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hee-Deung++Park">Hee-Deung Park</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Youngjoo Byun</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Youngjoo Byun</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy, Korea University, 2511 Sejong-ro, Jochiwon-eup, Sejong 30019, Republic of Korea</div><div class="loa-info-affiliations-info">Biomedical Research Center, Korea University Guro Hospital, 148 Gurodong-ro, Guro-gu, Seoul 08308, Republic of Korea</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#ff86959d868a91cebf94908d9a9ed19e9cd1948d"><span class="__cf_email__" data-cfemail="e891828a919d86d9a883879a8d89c6898bc6839a">[email protected]</span></a>. Phone: +82-44-860-1619. Fax: +82-44-860-1607.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Youngjoo++Byun">Youngjoo Byun</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0297-7734" title="Orcid link">http://orcid.org/0000-0002-0297-7734</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00630&amp;href=/doi/10.1021%2Facs.jmedchem.0c00630" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 15</span><span class="cit-pageRange">, 8388–8407</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 22, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>23 April 2020</li><li><span class="item_label"><b>Published</b> online</span>22 July 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 August 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00630" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00630</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8388%26pageCount%3D20%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DSangJin%2BNam%252C%2BSo-Young%2BHam%252C%2BHongmok%2BKwon%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D15%26contentID%3Dacs.jmedchem.0c00630%26title%3DDiscovery%2Band%2BCharacterization%2Bof%2BPure%2BRhlR%2BAntagonists%2Bagainst%2BPseudomonas%2Baeruginosa%2BInfections%26numPages%3D20%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8407%26publicationDate%3DAugust%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00630"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">764</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00630" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery and Characterization of Pure RhlR Antagonists against Pseudomonas aeruginosa Infections&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;SangJin&quot;,&quot;last_name&quot;:&quot;Nam&quot;},{&quot;first_name&quot;:&quot;So-Young&quot;,&quot;last_name&quot;:&quot;Ham&quot;},{&quot;first_name&quot;:&quot;Hongmok&quot;,&quot;last_name&quot;:&quot;Kwon&quot;},{&quot;first_name&quot;:&quot;Han-Shin&quot;,&quot;last_name&quot;:&quot;Kim&quot;},{&quot;first_name&quot;:&quot;Suhyun&quot;,&quot;last_name&quot;:&quot;Moon&quot;},{&quot;first_name&quot;:&quot;Jeong-Hoon&quot;,&quot;last_name&quot;:&quot;Lee&quot;},{&quot;first_name&quot;:&quot;Taehyeong&quot;,&quot;last_name&quot;:&quot;Lim&quot;},{&quot;first_name&quot;:&quot;Sang-Hyun&quot;,&quot;last_name&quot;:&quot;Son&quot;},{&quot;first_name&quot;:&quot;Hee-Deung&quot;,&quot;last_name&quot;:&quot;Park&quot;},{&quot;first_name&quot;:&quot;Youngjoo&quot;,&quot;last_name&quot;:&quot;Byun&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;22&quot;,&quot;issue&quot;:&quot;15&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;8388-8407&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00630&quot;},&quot;abstract&quot;:&quot;Pseudomonas aeruginosa (P. aeruginosa) is an opportunistic human pathogen that forms biofilms and produces virulence factors via quorum sensing (QS). Blocking the QS system in P. aeruginosa is an excellent strategy to reduce biofilm formation and the production of virulence factors. RhlR plays an essential role in the QS system of P. aeruginosa. We synthesized 55 analogues based on the chemical structure of 4-gingerol and evaluated their RhlR inhibitory activities using the cell-based reporter strain assay. Comprehensive structure–activity relationship studies identified the alkynyl ketone 30 as the most potent RhlR antagonist. This compound displayed selective RhlR antagonism over LasR and PqsR, strong inhibition of biofilm formation, and reduced production of virulence factors in P. aeruginosa. Furthermore, the survival rate of Tenebrio molitor larvae treated with 30 in vivo greatly improved. Therefore, compound 30, a pure RhlR antagonist, can be utilized for developing QS-modulating molecules in the contro&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00630&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00630" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00630&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00630" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00630&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00630" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00630&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00630&amp;href=/doi/10.1021/acs.jmedchem.0c00630" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00630" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00630" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00630%26sid%3Dliteratum%253Aachs%26pmid%3D32696644%26genre%3Darticle%26aulast%3DNam%26date%3D2020%26atitle%3DDiscovery%2Band%2BCharacterization%2Bof%2BPure%2BRhlR%2BAntagonists%2Bagainst%2BPseudomonas%2Baeruginosa%2BInfections%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D15%26spage%3D8388%26epage%3D8407%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (7)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292518" title="Biofilms">Biofilms</a>,</li><li><a href="/action/doSearch?ConceptID=291456" title="Hydrocarbons">Hydrocarbons</a>,</li><li><a href="/action/doSearch?ConceptID=291171" title="Antagonists">Antagonists</a>,</li><li><a href="/action/doSearch?ConceptID=291241" title="Phenyls">Phenyls</a>,</li><li><a href="/action/doSearch?ConceptID=291945" title="Ketones">Ketones</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/15" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/jmcmar.2020.63.issue-15/20200813/jmcmar.2020.63.issue-15.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/medium/jm0c00630_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00630&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText"><i>Pseudomonas aeruginosa</i> (<i>P. aeruginosa</i>) is an opportunistic human pathogen that forms biofilms and produces virulence factors <i>via</i> quorum sensing (QS). Blocking the QS system in <i>P. aeruginosa</i> is an excellent strategy to reduce biofilm formation and the production of virulence factors. RhlR plays an essential role in the QS system of <i>P. aeruginosa</i>. We synthesized 55 analogues based on the chemical structure of 4-gingerol and evaluated their RhlR inhibitory activities using the cell-based reporter strain assay. Comprehensive structure–activity relationship studies identified the alkynyl ketone <b>30</b> as the most potent RhlR antagonist. This compound displayed selective RhlR antagonism over LasR and PqsR, strong inhibition of biofilm formation, and reduced production of virulence factors in <i>P. aeruginosa</i>. Furthermore, the survival rate of <i>Tenebrio molitor</i> larvae treated with <b>30</b><i>in vivo</i> greatly improved. Therefore, compound <b>30</b>, a pure RhlR antagonist, can be utilized for developing QS-modulating molecules in the control of <i>P. aeruginosa</i> infections.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03858" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03858" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p"><i>Pseudomonas aeruginosa</i> (<i>P. aeruginosa</i>) is a causative human pathogen that induces chronic diseases in immune-compromised patients.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> It induces infections in burn wounds, cystic fibrosis, acute leukemia, organ transplantation, and intravenous drug addiction patients.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> The World Health Organization (WHO) categorizes it as a critical priority pathogen that requires considerable attention.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a><i>P. aeruginosa</i> is resistant to conventional antibiotic treatments mostly because of the formation of thick biofilms.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> It can colonize on various surfaces by forming biofilms in which bacterial cells embedded within self-produced exopolysaccharides.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Biofilms are widely found in medical, dental, agricultural, industrial, and environmental settings.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> In particular, biofilms in medical settings are associated with about 80% of bacterial infections in humans and increased antibiotic resistance.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></div><div class="NLM_p">Quorum sensing (QS) is a bacterial cell-to-cell communication process that occurs <i>via</i> chemical signal molecules and allows bacteria to share information in response to environmental changes.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Once the bacterial density reaches a certain threshold, signaling molecules (autoinducers) bind to their cognate receptor proteins and alter gene expression to regulate collective behaviors.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a><i>P. aeruginosa</i> possesses three major QS systems (<i>las</i>, <i>rhl</i>, and <i>pqs</i>) that are tightly interconnected with each other. Gram-negative bacteria including <i>P. aeruginosa</i> use <i>N</i>-acyl-<span class="smallcaps smallerCapital">l</span>-homoserine lactones (AHLs) as autoinducers of QS.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Typically, AHLs are produced by LuxI-type synthases (<i>e.g.</i>, LasI and RhlI) and recognized by cytoplasmic LuxR-type receptors (<i>e.g.</i>, LasR and RhlR). LasI and RhlI produce <i>N</i>-(3-oxo-dodecanoyl)-<span class="smallcaps smallerCapital">l</span>-homoserine lactone (OdDHL, <b>1a</b>) and <i>N</i>-butyryl-<span class="smallcaps smallerCapital">l</span>-homoserine lactone (BHL, <b>1b</b>) as autoinducers, respectively (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> In addition to these systems, the <i>pqs</i> circuit is the third system regulated via <i>Pseudomonas</i> quinolone signal (PQS) recognized by PqsR. These three systems are hierarchically controlled. The <i>las</i> system activates <i>rhl</i> and <i>pqs</i> systems, and the <i>rhl</i> system represses the <i>pqs</i> system. This interactive signal-network regulation of <i>P. aeruginosa</i> leads to biofilm formation, the production of virulence factors, and the modulation of host immune response.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Therefore, small molecules that modulate the recognition of autoinducers toward their cognate receptors have the potential to control virulence factors and biofilm formation of <i>P. aeruginosa</i>.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/medium/jm0c00630_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of endogenous and synthetic molecules interacting with the QS receptor of <i>P. aeruginosa</i>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00630&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Most QS inhibitors of <i>P. aeruginosa</i> focus on targeting LasR because it is located at the top of the <i>P. aeruginosa</i> QS network hierarchy.<a onclick="showRef(event, 'ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16">(14−16)</a> However, even though RhlR also plays an important role in the QS process of <i>P. aeruginosa</i>, RhlR modulators have rarely been reported.<a onclick="showRef(event, 'ref17 ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19 ref20">(17−20)</a> Up to date, few modulators for the interaction between BHL and RhlR as agonists or antagonists have been reported.<a onclick="showRef(event, 'ref21 ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23 ref24">(21−24)</a> Blackwell and co-workers reported that BHL analogues with RhlR agonism activity reduced the production of <i>P. aeruginosa</i> virulence factors such as pyocyanin.<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a> They also reported mixed LasR/RhlR antagonists which increased pyocyanin production.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Bassler and co-workers recently conducted an experiment with BHL-independent and active <i>rhlR</i> mutants, which showed that RhlR drives biofilm formation and production of virulence factors in <i>P. aeruginosa</i>.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></div><div class="NLM_p last">Previously, we reported that (<i>S</i>)-6-gingerol (<b>1c</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) from ginger extracts is a moderate LasR antagonist of <i>P. aeruginosa</i>.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> It reduced the biofilm formation, production of virulence factors, and expression of QS-related genes in <i>P. aeruginosa</i>. We performed comprehensive structure–activity relationship (SAR) studies of gingerol analogues and identified the LasR antagonist that was more potent than (<i>S</i>)-6-gingerol.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Based on our previous studies, we hypothesized that gingerols with alkyl chains shorter than <b>1c</b> might bind to RhlR because of their structural similarity to BHL. We describe the design, synthesis, and biochemical characterization of gingerol analogues as pure RhlR antagonists. We have established the SAR of gingerol analogues by extensively modifying their chemical structures. The pure and potent RhlR antagonist which we discovered in this study can be utilized in the discovery of novel agents that inhibit the biofilm formation and production of virulence factors and in the elucidation of <i>rhl</i>-RhlR mechanism in QS network of <i>P. aeruginosa</i>.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78254" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78254" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As an attempt to discover pure RhlR antagonists, we first screened the relative RhlR activities of in-house gingerol analogues with various alkyl chain lengths, from 4-gingerol to 10-gingerol (see Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_001.pdf" class="ext-link">Scheme S1</a>), which were previously reported.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> We used dimethyl sulfoxide (DMSO) as a negative control and (<i>S</i>)-6-gingerol (<b>1c</b>) and <i>N</i>-cyclopentylbutyramide (<b>1d</b>) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) as positive controls in <i>in vitro</i> biological evaluation studies. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, the more potent RhlR antagonism activities increased with the shorter alkyl chain lengths of gingerols, suggesting that gingerols with the shorter alkyl chain have a higher affinity for RhlR. Interestingly, all tested gingerols showed very low RhlR agonistic activity (<4%). Compound <b>2</b> (4-gingerol) with the shortest alkyl chain in the series was the most potent with a relative RhlR activity of 54%. As the chemical structure of <b>2</b> is more similar to that of BHL than the others, it was assumed that it competed against BHL for binding to RhlR. Based on preliminary results, we used 4-gingerol as an initial hit compound for further structural modification.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Relative RhlR Activity (%) of the Synthesized Compounds</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">relative RhlR activity</th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">relative RhlR activity</th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">relative RhlR activity</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">antagonsim<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">agonism<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">antagonsim</th><th class="colsep0 rowsep0" align="center" char=".">agonism</th><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">antagonsim</th><th class="colsep0 rowsep0" align="center" char=".">agonism</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1c</b></td><td class="colsep0 rowsep0" align="char" char=".">75**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td><td class="colsep0 rowsep0" align="left"><b>11p</b></td><td class="colsep0 rowsep0" align="char" char=".">72**</td><td class="colsep0 rowsep0" align="char" char=".">2**</td><td class="colsep0 rowsep0" align="left"><b>14b</b></td><td class="colsep0 rowsep0" align="char" char=".">49**</td><td class="colsep0 rowsep0" align="char" char=".">2**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1d</b></td><td class="colsep0 rowsep0" align="char" char=".">54**</td><td class="colsep0 rowsep0" align="char" char=".">4**</td><td class="colsep0 rowsep0" align="left"><b>11q</b></td><td class="colsep0 rowsep0" align="char" char=".">43**</td><td class="colsep0 rowsep0" align="char" char=".">1**</td><td class="colsep0 rowsep0" align="left"><b>15a</b></td><td class="colsep0 rowsep0" align="char" char=".">55**</td><td class="colsep0 rowsep0" align="char" char=".">5**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="char" char=".">54**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td><td class="colsep0 rowsep0" align="left"><b>11r</b></td><td class="colsep0 rowsep0" align="char" char=".">74**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td><td class="colsep0 rowsep0" align="left"><b>15b</b></td><td class="colsep0 rowsep0" align="char" char=".">41**</td><td class="colsep0 rowsep0" align="char" char=".">2**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="char" char=".">62**</td><td class="colsep0 rowsep0" align="char" char=".">2**</td><td class="colsep0 rowsep0" align="left"><b>12a</b></td><td class="colsep0 rowsep0" align="char" char=".">69**</td><td class="colsep0 rowsep0" align="char" char=".">4**</td><td class="colsep0 rowsep0" align="left"><b>16a</b></td><td class="colsep0 rowsep0" align="char" char=".">61**</td><td class="colsep0 rowsep0" align="char" char=".">5**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="char" char=".">76**</td><td class="colsep0 rowsep0" align="char" char=".">4**</td><td class="colsep0 rowsep0" align="left"><b>12b</b></td><td class="colsep0 rowsep0" align="char" char=".">88*</td><td class="colsep0 rowsep0" align="char" char=".">4**</td><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="char" char=".">29**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="char" char=".">83**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td><td class="colsep0 rowsep0" align="left"><b>12c</b></td><td class="colsep0 rowsep0" align="char" char=".">90*</td><td class="colsep0 rowsep0" align="char" char=".">2**</td><td class="colsep0 rowsep0" align="left"><b>23a</b></td><td class="colsep0 rowsep0" align="char" char=".">32**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="char" char=".">90*</td><td class="colsep0 rowsep0" align="char" char=".">2**</td><td class="colsep0 rowsep0" align="left"><b>12d</b></td><td class="colsep0 rowsep0" align="char" char=".">64**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td><td class="colsep0 rowsep0" align="left"><b>23b</b></td><td class="colsep0 rowsep0" align="char" char=".">27**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="char" char=".">41**</td><td class="colsep0 rowsep0" align="char" char=".">2**</td><td class="colsep0 rowsep0" align="left"><b>12e</b></td><td class="colsep0 rowsep0" align="char" char=".">81**</td><td class="colsep0 rowsep0" align="char" char=".">2**</td><td class="colsep0 rowsep0" align="left"><b>23c</b></td><td class="colsep0 rowsep0" align="char" char=".">34**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="char" char=".">36**</td><td class="colsep0 rowsep0" align="char" char=".">2**</td><td class="colsep0 rowsep0" align="left"><b>12f</b></td><td class="colsep0 rowsep0" align="char" char=".">71**</td><td class="colsep0 rowsep0" align="char" char=".">2**</td><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="char" char=".">79**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11a</b></td><td class="colsep0 rowsep0" align="char" char=".">65**</td><td class="colsep0 rowsep0" align="char" char=".">4**</td><td class="colsep0 rowsep0" align="left"><b>12g</b></td><td class="colsep0 rowsep0" align="char" char=".">51**</td><td class="colsep0 rowsep0" align="char" char=".">2**</td><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="char" char=".">45**</td><td class="colsep0 rowsep0" align="char" char=".">2**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11b</b></td><td class="colsep0 rowsep0" align="char" char=".">93*</td><td class="colsep0 rowsep0" align="char" char=".">4**</td><td class="colsep0 rowsep0" align="left"><b>12h</b></td><td class="colsep0 rowsep0" align="char" char=".">69**</td><td class="colsep0 rowsep0" align="char" char=".">2**</td><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="char" char=".">12**</td><td class="colsep0 rowsep0" align="char" char=".">4**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11c</b></td><td class="colsep0 rowsep0" align="char" char=".">94</td><td class="colsep0 rowsep0" align="char" char=".">2**</td><td class="colsep0 rowsep0" align="left"><b>12i</b></td><td class="colsep0 rowsep0" align="char" char=".">76**</td><td class="colsep0 rowsep0" align="char" char=".">1**</td><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="char" char=".">16**</td><td class="colsep0 rowsep0" align="char" char=".">4**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11d</b></td><td class="colsep0 rowsep0" align="char" char=".">51**</td><td class="colsep0 rowsep0" align="char" char=".">1**</td><td class="colsep0 rowsep0" align="left"><b>12j</b></td><td class="colsep0 rowsep0" align="char" char=".">74**</td><td class="colsep0 rowsep0" align="char" char=".">1**</td><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="char" char=".">17**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11e</b></td><td class="colsep0 rowsep0" align="char" char=".">78**</td><td class="colsep0 rowsep0" align="char" char=".">2**</td><td class="colsep0 rowsep0" align="left"><b>12k</b></td><td class="colsep0 rowsep0" align="char" char=".">47**</td><td class="colsep0 rowsep0" align="char" char=".">2**</td><td class="colsep0 rowsep0" align="left"><b>33a</b></td><td class="colsep0 rowsep0" align="char" char=".">32**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11f</b></td><td class="colsep0 rowsep0" align="char" char=".">55**</td><td class="colsep0 rowsep0" align="char" char=".">1**</td><td class="colsep0 rowsep0" align="left"><b>12l</b></td><td class="colsep0 rowsep0" align="char" char=".">94</td><td class="colsep0 rowsep0" align="char" char=".">1**</td><td class="colsep0 rowsep0" align="left"><b>33b</b></td><td class="colsep0 rowsep0" align="char" char=".">25**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11g</b></td><td class="colsep0 rowsep0" align="char" char=".">33**</td><td class="colsep0 rowsep0" align="char" char=".">1**</td><td class="colsep0 rowsep0" align="left"><b>12m</b></td><td class="colsep0 rowsep0" align="char" char=".">73**</td><td class="colsep0 rowsep0" align="char" char=".">1**</td><td class="colsep0 rowsep0" align="left"><b>33c</b></td><td class="colsep0 rowsep0" align="char" char=".">38**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11h</b></td><td class="colsep0 rowsep0" align="char" char=".">64**</td><td class="colsep0 rowsep0" align="char" char=".">1**</td><td class="colsep0 rowsep0" align="left"><b>12n</b></td><td class="colsep0 rowsep0" align="char" char=".">98</td><td class="colsep0 rowsep0" align="char" char=".">1**</td><td class="colsep0 rowsep0" align="left"><b>34a</b></td><td class="colsep0 rowsep0" align="char" char=".">71**</td><td class="colsep0 rowsep0" align="char" char=".">4**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11i</b></td><td class="colsep0 rowsep0" align="char" char=".">65**</td><td class="colsep0 rowsep0" align="char" char=".">2**</td><td class="colsep0 rowsep0" align="left"><b>12o</b></td><td class="colsep0 rowsep0" align="char" char=".">49**</td><td class="colsep0 rowsep0" align="char" char=".">1**</td><td class="colsep0 rowsep0" align="left"><b>34b</b></td><td class="colsep0 rowsep0" align="char" char=".">67**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11j</b></td><td class="colsep0 rowsep0" align="char" char=".">53**</td><td class="colsep0 rowsep0" align="char" char=".">2**</td><td class="colsep0 rowsep0" align="left"><b>12p</b></td><td class="colsep0 rowsep0" align="char" char=".">82**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td><td class="colsep0 rowsep0" align="left"><b>34c</b></td><td class="colsep0 rowsep0" align="char" char=".">64**</td><td class="colsep0 rowsep0" align="char" char=".">2**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11k</b></td><td class="colsep0 rowsep0" align="char" char=".">43**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td><td class="colsep0 rowsep0" align="left"><b>12q</b></td><td class="colsep0 rowsep0" align="char" char=".">79**</td><td class="colsep0 rowsep0" align="char" char=".">1**</td><td class="colsep0 rowsep0" align="left"><b>35a</b></td><td class="colsep0 rowsep0" align="char" char=".">74**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11l</b></td><td class="colsep0 rowsep0" align="char" char=".">69**</td><td class="colsep0 rowsep0" align="char" char=".">1**</td><td class="colsep0 rowsep0" align="left"><b>12r</b></td><td class="colsep0 rowsep0" align="char" char=".">85**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td><td class="colsep0 rowsep0" align="left"><b>35b</b></td><td class="colsep0 rowsep0" align="char" char=".">72**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11m</b></td><td class="colsep0 rowsep0" align="char" char=".">60**</td><td class="colsep0 rowsep0" align="char" char=".">2**</td><td class="colsep0 rowsep0" align="left"><b>13a</b></td><td class="colsep0 rowsep0" align="char" char=".">51**</td><td class="colsep0 rowsep0" align="char" char=".">4**</td><td class="colsep0 rowsep0" align="left"><b>35c</b></td><td class="colsep0 rowsep0" align="char" char=".">67**</td><td class="colsep0 rowsep0" align="char" char=".">4**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11n</b></td><td class="colsep0 rowsep0" align="char" char=".">91</td><td class="colsep0 rowsep0" align="char" char=".">1**</td><td class="colsep0 rowsep0" align="left"><b>13b</b></td><td class="colsep0 rowsep0" align="char" char=".">41**</td><td class="colsep0 rowsep0" align="char" char=".">2**</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11o</b></td><td class="colsep0 rowsep0" align="char" char=".">20**</td><td class="colsep0 rowsep0" align="char" char=".">2**</td><td class="colsep0 rowsep0" align="left"><b>14a</b></td><td class="colsep0 rowsep0" align="char" char=".">55**</td><td class="colsep0 rowsep0" align="char" char=".">4**</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">DMSO (negative control) and <b>1c</b> and <b>1d</b> (positive controls) were used.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">RhlR antagonism activity of the compound (100 μM) in the presence of <b>1b</b> (10 μM).</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">RhlR agonism activity of the compound (100 μM). (**) <i>P</i> < 0.005 and (*) <i>P</i> < 0.05 as compared with the control.</p></div></div></div><div class="NLM_p">To establish the SAR of 4-gingerol for RhlR antagonism, we performed structural modification as follows: (1) variation of the substituents in the phenyl ring, (2) introduction of a double or triple bond between the phenyl ring and the carbonyl group to increase rotational rigidity, (3) removal of β-hydroxyl group, and (4) change of absolute configuration. First, we synthesized 34 derivatives to determine the effect of the substituents in the phenyl ring on RhlR antagonism. 3′-OMe and 4′-OH groups of the phenyl ring in 4-gingerol were replaced with diverse functional groups (−F, −Cl, −OH, −OMe, −OEt, −Me, and −N(CH<sub>3</sub>)<sub>2</sub>). <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> describes the synthesis of 4-gingerol derivatives with various substituents of the phenyl ring. Compounds <b>10a–10r</b> were synthesized from commercial benzaldehydes by treating 10% NaOH in acetone and ethanol (or water) at 25 °C. The reaction of <b>10a–10r</b> with lithium diisopropylamide (LDA) at −78 °C, followed by the addition of <i>n</i>-butanal afforded compounds <b>11a–11r</b> in 24–60% yield. Low-to-moderate yield in this step might be because of side reactions such as elimination or dimer formation. Compounds <b>12a–12r</b> were obtained in 40–93% yield by reducing <b>11a–11r</b> with hydrogen gas in the presence of 10% Pd/C.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/medium/jm0c00630_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 4-Gingerol Derivatives with the Variation of Phenyl Rings<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00630&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) acetone, EtOH (or water), 10% NaOH, rt, 2 to 24 h, 32%−quantitative yield; (ii) LDA 1.0 M in THF/hexanes, <i>n</i>-butanal, THF, −78 °C, 2–12 h, 24–60% yield; (iii) H<sub>2</sub> gas, 10% Pd/C, MeOH, rt, 2 h, 40–93% yield.</p></p></figure><div class="NLM_p">As summarized in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, compound <b>11b</b> without any substituent in the phenyl region completely lost its RhlR antagonistic property with a relative RhlR activity of 93%, indicating that a polar functional group in the phenyl ring is required for binding to RhlR. To assess the necessity of 4′-OH and 3′-OMe groups in the phenyl ring for RhlR antagonism, we removed the −OH group at 4′-position (<b>11c</b>) and the −OMe group at 3′-position (<b>11d</b>). As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, compound <b>11c</b> displayed a reduced ability to inhibit RhlR (relative 94% RhlR activity). However, compound <b>11d</b> showed slightly stronger RhlR antagonism than <b>11a</b> (51 vs 65%). However, compound <b>11f</b> with an −OMe group at 4′-position was less potent than <b>11d</b>, implying that the presence of the polar group at 4′-position is favorable for RhlR-binding affinity. In the case of the 3′-position, compound <b>11e</b> with an −OH group only showed stronger RhlR antagonism than compound <b>11c</b> with an −OMe group only (78 vs 94%).</div><div class="NLM_p">To expand the initial SAR of 4-gingerol, we synthesized a variety of monosubstituted (<b>11g–11j</b>) and disubstituted (<b>11k–11r</b>) analogues. Among the monosubstituted compounds, <b>11g</b> with 4′-F was the most potent with a relative RhlR activity of 33%. Interestingly, compound <b>11j</b> (53%) with a bulky 4′-N(CH<sub>3</sub>)<sub>2</sub> was more potent than <b>11a</b>, whereas <b>11h</b> (64%) with 4′-Cl and <b>11i</b> (65%) with 4′-Me were comparable to <b>11a</b>. A similar trend was observed with the di-substituted compounds. Compound <b>11o</b> with 3′,4′-di-F was the most potent in this series (relative RhlR activity, 20%). Notably, these trends indicated that the F-substitution in the phenyl ring is preferred for the structural modification of the phenyl ring. Compound <b>11q</b> (43%) with 3′-OMe and 4′-F was more active than <b>11a</b> with 3′-OMe and 4′-OH, suggesting that the F group at 4′-position can replace the OH group of 4-gingerol. However, compound <b>11r</b> (74%) with 3′-F and 5′-OMe was less potent than <b>11a</b>, confirming the necessity of the small and polar groups (<i>i.e.</i>, −F and −OH) at 4′-position. As expected, the hydrophobic −Me group (<b>11p</b>, 72%) at both 3′- and 4′-position showed decreased RhlR inhibitory activity. Compound <b>11n</b> with 3′-OEt and 4′-OH dramatically decreased the antagonistic activity of RhlR (91%), implying that the region of RhlR which interacts with the 3′-position of 4-gingerol was highly sensitive to the substituent size. The SAR results indicated that the 4′-position is related to the hydrogen-bonding interaction with RhlR, whereas the 3′-position is sensitive to the bulkiness of the substituent.</div><div class="NLM_p">The effect of rotational rigidity on RhlR antagonism was evaluated by comparing the compounds (<b>11a–11r</b>) with a double bond between the phenyl ring and the carbonyl group with the corresponding compounds (<b>12a–12r</b>) with a single bond. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, the compounds with a double bond (<b>11a–11r</b>) displayed stronger RhlR antagonism than those with a single bond (<b>12a–12r</b>), irrespective of the substituents in the phenyl ring. This result implied that the restricted rotational flexibility between the phenyl ring and the carbonyl group significantly increased the binding affinity for RhlR. Overall, the double-bond compound <b>11o</b>, substituted with the F group at both 3′- and 4′-positions, showed the most potent RhlR antagonism with low RhlR agonist activity (2%) among the compounds with variations in the phenyl ring.</div><div class="NLM_p">The synthesis of 4-gingerol derivatives with variations in the middle section is outlined in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_001.pdf" class="ext-link">Scheme S2</a> (see the Supporting Information). Compounds (<b>13a–13b</b>) without the β-hydroxyl group were synthesized by reacting 2-heptanone with vanillin or 3,4-difluorobenzaldehyde under basic conditions at 25 °C for 72 h in 30–50% yield. The α,β-unsaturated carbonyl group of <b>13a–13b</b> was reduced by two different reagents: (1) H<sub>2</sub> and Pd/C and (2) NaBH<sub>4</sub>. The double bond was reduced to the single bond via catalytic hydrogenation, providing compounds <b>14a–14b</b> in ∼76% yield. The carbonyl group was reduced to the secondary alcohol by treatment with NaBH<sub>4</sub> to afford compounds <b>15a</b> and <b>15b</b> in 94 and 90% yield, respectively. Compound <b>15a</b> was reduced to the single-bond compound <b>16a</b> in 60% yield. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, compound <b>13b</b> (41%) without the β-hydroxyl group was more potent than <b>11a</b> (65%), indicating that the β-hydroxyl group was not essential for binding to RhlR. The α,β-unsaturated carbonyl analogue (<b>13b</b>) was more potent than the corresponding α,β-saturated one (<b>14b</b>). The reduction of the α,β-unsaturated carbonyl group to alcohol (<b>15b</b>, 41%) maintained the RhlR-binding affinity. However, the RhlR antagonistic activities of 3,4-difluorophenyl ring analogues (<b>13a–15a</b>) were almost similar.</div><div class="NLM_p">Gingerol derivatives with a triple bond in the middle section were prepared by applying the synthetic route described in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. The phenolic OH of vanillin was protected with the <i>tert</i>-butyldimethylsilyl (TBDMS) group, affording compound <b>17</b>. The reaction of <b>17</b> with CBr<sub>4</sub> and PPh<sub>3</sub> in dichloromethane provided the dibromo alkene <b>18</b> in 98% yield. The elimination and lithiation of <b>18</b> in the presence of 3 equiv of <i>n</i>-BuLi afforded the terminal alkyne <b>19</b> in 96% yield.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Compound <b>20</b> was obtained by reacting <b>19</b> with Weinreb amide. The alkynyl ketone <b>21</b> was prepared from <b>20</b> by using tetrabutylammoniumfluoride (TBAF) in 94% yield. Enantiomerically enriched alkynyl alcohol analogues (<b>23b–23c</b>) were prepared from <b>20</b> by using chiral catalysts, such as RuCl[(<i>R</i>,<i>R</i>)-TsDPEN(mesitylene)] or RuCl[(<i>S</i>,<i>S</i>)-TsDPEN(mesitylene)],<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and desilylation with TBAF. Compound <b>23a</b>, a racemic mixture, was obtained from <b>20</b> by treatment of NaBH<sub>4</sub> and subsequent desilylation. The alkynyl alcohols (<b>23a–23c</b>) were analyzed by chiral reversed-phase (RP) high-performance liquid chromatography (HPLC). As compound <b>23a</b> was a racemic mixture, two distinct peaks were observed at 9.91 and 12.58 min in chiral HPLC with a 1:1 ratio (see the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_001.pdf" class="ext-link">Supporting Information</a>). Compound <b>23b</b> with an (<i>R</i>)-configuration was eluted at 12.54 min, whereas compound <b>23c</b> with (<i>S</i>)-configuration was eluted at 9.88 min with ee values >99% (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_001.pdf" class="ext-link">Figure S1</a>). (<i>Z</i>)-selective reduction of the alkyne (<b>22a</b>) to cis-isomer (<b>25</b>) was accomplished by the treatment of the Lindlar catalyst and 1,4-benzoquinone under an atmosphere of hydrogen, followed by desilylation with TBAF.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The alkynyl alcohol <b>27</b> was obtained from <b>22a</b> in 94% yield via catalytic hydrogenation and subsequent desilylation.</div><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/medium/jm0c00630_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of 4-Gingerol Derivatives with the Variation of Rotational Rigidity and Absolute Configuration<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00630&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) TBDMS chloride, imidazole, CH<sub>2</sub>Cl<sub>2</sub>, rt, 18 h, 94% yield; (ii) PPh<sub>3</sub>, CBr<sub>4</sub>, CH<sub>2</sub>Cl<sub>2</sub>, rt, 3 h, 98% yield; (iii) <i>n</i>-BuLi 1.6 M in hexanes (3.0 equiv), THF, rt, 2 h, 96% yield; (iv) <i>n</i>-BuLi 1.6 M in hexanes (1.1 equiv), <i>N</i>-methoxy-<i>N</i>-methylhexanamide, THF, −78 °C to rt, 16 h, 75% yield; (v) <i>n</i>-Bu<sub>4</sub>NF (0.1 M in THF) rt, 1 h, 94% yield (for <b>21</b>), 94% yield (for <b>23a</b>), 94% yield (for <b>23b</b>), 93% yield (for <b>23c</b>), 70% yield (for <b>25</b>), and 94% yield (for <b>27</b>); (vi) NaBH<sub>4</sub>, MeOH, rt, 1 h, 77% yield (for <b>22a</b>), RuCl[(<i>R</i>,<i>R</i>)-TsDPEN(mesitylene)], KOH, 2-propanol, 4 h, 88% yield (for <b>22b</b>), and RuCl[(<i>S</i>,<i>S</i>)-TsDPEN(mesitylene)], KOH, 2-propanol, 4 h, 68% yield (for <b>22c</b>); (vii) Lindlar cat., H<sub>2</sub> gas, 1,4-benzoquinone, MeOH, 0 °C, 1 h (from <b>22a</b>); (viii) H<sub>2</sub> gas, 10% Pd/C, MeOH, rt, 2 h, 94% yield (from <b>22a</b>).</p></p></figure><div class="NLM_p">The alkynyl ketone <b>21</b> displayed strong relative RhlR activity (29%) than the corresponding alkenyl ketone <b>13b</b> (41%) and alkanyl ketone <b>14b</b> (49%) (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Similarly, the alkynyl alcohol <b>23a</b> (32%) was also more potent than the alkene <b>15b</b> (41%) and the alkane <b>27</b> (45%). The relative RhlR activities of the alkynyl ketone (<b>21</b>) or the alkynyl alcohol compounds (<b>23a–23c</b>) were <34%. They were more potent than the reported RhlR antagonist (<b>1d</b>, 54%). Regarding the effect of the absolute configuration on RhlR affinity, the (<i>R</i>)-isomer <b>23b</b> (27%) was stronger than the corresponding (<i>S</i>)-isomer <b>23c</b> (34%) and the racemate <b>23a</b> (32%). The (<i>Z</i>)-alkenyl alcohol <b>25</b> with a relative RhlR activity of 79% was much weaker than the (<i>E</i>)-alkenyl alcohol <b>15b</b> (41%), implying that the cis-isomer may not be properly located in the RhlR active site. Overall, SAR data indicated that the carbonyl group or the hydroxyl group at the γ-position from the phenyl group is important for binding to RhlR.</div><div class="NLM_p">Based on the SAR studies of 4-gingerol analogues, we synthesized 3,4-difluorophenyl derivatives which showed the most potent RhlR-binding activity in the variation of the phenyl ring. As described in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>, the alkynyl ketones (<b>30–32</b>) and the alkynyl alcohols (<b>33a–33c</b>, <b>34a–34c</b>, and <b>35a–35c</b>) were prepared from commercial 3,4-difluorobenzaldehyde by applying the same synthetic strategy used for 4-gingerol derivatives. The purity and ee ratio of the final compounds were analyzed using chiral HPLC (see Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_001.pdf" class="ext-link">Figure S2</a>). The alkynyl ketones (<b>30–32</b>) were more potent than the corresponding alkene and alkane compounds, with relative RhlR activities <17% at 100 μM. In addition, the (<i>R</i>)-stereoisomers (<b>33b–35b</b>) were also more active than the corresponding (<i>S</i>)-stereoisomers (<b>33c–35c</b>) or the racemates (<b>33a–35a</b>). Furthermore, the compounds with the shorter alkyl chain length (<b>30</b> and <b>33a–33c</b>) were stronger than the corresponding ones with the longer alkyl chain lengths (<b>31</b>, <b>32</b>, <b>34a–34c</b>, and <b>35a–35c</b>). The alkynyl ketone with the <i>n</i>-propyl group (<b>30</b>) displayed the most potent activity in this series, with a relative RhlR activity of 12%, which is a much stronger activity profile than the reported antagonist <b>1d</b> (54%).</div><figure id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/medium/jm0c00630_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of 3,4-Difluorophenyl Derivatives<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00630&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) PPh<sub>3</sub>, CBr<sub>4</sub>, CH<sub>2</sub>Cl<sub>2</sub>, rt, 3 h, 97% yield; (ii) <i>n</i>-BuLi 1.6 M in hexanes (3.0 equiv), THF, rt, 2 h; (iii) <i>n</i>-BuLi 1.6 M in hexanes (1.1 equiv), appropriate Weinreb amides, THF, −78 °C to rt, 16 h, 72% yield (for <b>30</b>), 88% yield (for <b>31</b>), and 72% yield (for <b>32</b>); (iv) NaBH<sub>4</sub>, MeOH, rt, 1 h (for <b>33a–35a</b>), 88% yield (for <b>33a</b>), 80% yield (for <b>34a</b>), and 86% yield (for <b>35a</b>), RuCl[(<i>R</i>,<i>R</i>)-TsDPEN(mesitylene)], KOH, 2-propanol, 4 h (for <b>33b–35b</b>), 86% yield (for <b>33b</b>), 82% yield (for <b>34b</b>), and 80% yield (for <b>35b</b>), and RuCl[(<i>S</i>,<i>S</i>)-TsDPEN(mesitylene)], KOH, 2-propanol, 4 h (for <b>33c–35c</b>), 86% yield (for <b>33c</b>), 82% yield (for <b>34c</b>), and 82% yield (for <b>35c</b>).</p></p></figure><div class="NLM_p">To elucidate the binding mode of compound <b>30</b> in RhlR, we performed <i>in silico</i> molecular docking studies of <b>30</b> and <b>1d</b> with a RhlR homology model. Compound <b>30</b> interacted with RhlR via a π–π stacking interaction with Tyr72 and a hydrogen bond with Trp68, whereas <b>1d</b> made hydrogen-bonding interactions with Asp81, Thr121, and Ser135 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Trp68 was reported as one of the key amino acids in a BHL-independent and active <i>rhlR</i> mutants.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> As the alkyl chain length gets longer (<b>30</b> vs <b>31</b> and <b>32</b>), the phenyl moiety made weaker π–π stacking interactions with Tyr72 (see Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_001.pdf" class="ext-link">Figure S3</a>), elucidating the importance of the π–π stacking interaction in 3,4-difluorophenyl analogues.</div><figure id="fig2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/medium/jm0c00630_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Docked poses of <b>30</b> (yellow) and <b>1d</b> (pink) with the RhlR homology model. Red and black dotted lines represent π–π stacking and hydrogen-bonding interactions, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00630&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To validate the SAR results and compare the activities under the same experimental conditions, we selected 11 representative compounds (<b>11a</b>, <b>11c</b>, <b>11k</b>, <b>11o</b>, <b>13a</b>, <b>30</b>, <b>31</b>, <b>32</b>, <b>33a</b>, <b>33b</b>, and <b>33c</b>) and determined RhlR antagonism, RhlR agonism, static biofilm formation, bacterial growth, and rhamnolipid production (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The removal of the OH group at the 4′-position of the phenyl ring (<b>11c</b>) significantly reduced the RhlR antagonism activity profile. In contrast, the replacement of −OMe at the 3′-position of the phenyl ring with −OH (<b>11k</b>) increased RhlR antagonism. A comparison of <b>11o</b> and <b>13a</b> confirmed that the β-hydroxyl group in the middle section is not essential for RhlR activity. The alkynyl ketone compounds (<b>30–32</b>) with restricted rotational flexibility exhibited increased RhlR activity. We also confirmed that the (<i>R</i>)-alkynol (<b>33b</b>) is a better RhlR antagonist than the racemate (<b>33a</b>) and the (<i>S</i>)-alkynol (<b>33c</b>). Next, we determined the RhlR agonism activity of the selected compounds. As summarized in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, none of the synthesized compounds showed RhlR agonism activity even at 1000 μM. However, the reported RhlR antagonist <b>1d</b> exhibited RhlR agonism with a relative RhlR agonistic activity of 37% at 1000 μM. Compared with the control, compounds <b>11o</b>, <b>13a</b>, <b>30</b>, <b>31</b>, and <b>32</b> significantly inhibited biofilm formation at 100 μM. In particular, compound <b>30</b> showed approximately 47% relative biofilm formation of <i>P. aeruginosa</i>, which is much more potent than (<i>S</i>)-6-gingerol (<b>1c</b>) and the known RhlR antagonist (<b>1d</b>). Next, we measured the effect of bacterial growth inhibition by the compounds at 10 and 100 μM concentrations. Compounds <b>31</b> and <b>32</b> slightly inhibited bacterial growth at 100 μM. However, compound <b>30</b> had no effect on bacterial growth at the same concentration. Furthermore, we examined the production of rhamnolipid, a representative virulence factor in <i>P. aeruginosa</i>.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> RhlR is known to directly regulate the expression of rhamnolipid production enzymes. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, the most potent RhlR antagonist <b>30</b> (12% of relative RhlR activity) displayed the lowest rhamnolipid production (42%) among the selected compounds. Moreover, compound <b>30</b> showed stronger inhibition activities on biofilm formation and rhamnolipid production without inhibiting bacterial growth than the other compounds (<b>7</b>, <b>8</b>, <b>11a</b>, <b>11c</b>, <b>11d</b>, <b>11f</b>, <b>21</b>, <b>23a</b>, <b>23b</b>, and <b>23c</b>) with high RhlR activity but not included in the representative set (see Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_001.pdf" class="ext-link">Table S1</a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. <i>In Vitro</i> Biological Data of the Representative Compounds</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/medium/jm0c00630_0012.gif" alt="" id="GRAPHIC-d7e2373-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center" char=".">relative RhlR activity (%)</th><th class="rowsep1 colsep0" colspan="6" align="center" char=".">relative biological activity (%)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">antagonism<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">agonism<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">biofilm formation<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">bacterial growth<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">rhamnolipid production<a class="ref internalNav" href="#t2fn6" aria-label="f">f</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">10</th><th class="colsep0 rowsep0" align="center" char=".">100</th><th class="colsep0 rowsep0" align="center" char=".">100</th><th class="colsep0 rowsep0" align="center" char=".">1000</th><th class="colsep0 rowsep0" align="center" char=".">10</th><th class="colsep0 rowsep0" align="center" char=".">100</th><th class="colsep0 rowsep0" align="center" char=".">10</th><th class="colsep0 rowsep0" align="center" char=".">100</th><th class="colsep0 rowsep0" align="center" char=".">10</th><th class="colsep0 rowsep0" align="center" char=".">100</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1c</b></td><td class="colsep0 rowsep0" align="char" char=".">86*</td><td class="colsep0 rowsep0" align="char" char=".">75**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td><td class="colsep0 rowsep0" align="char" char=".">5**</td><td class="colsep0 rowsep0" align="char" char=".">86</td><td class="colsep0 rowsep0" align="char" char=".">71**</td><td class="colsep0 rowsep0" align="char" char=".">96*</td><td class="colsep0 rowsep0" align="char" char=".">101</td><td class="colsep0 rowsep0" align="char" char=".">76*</td><td class="colsep0 rowsep0" align="char" char=".">68*</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1d</b></td><td class="colsep0 rowsep0" align="char" char=".">74*</td><td class="colsep0 rowsep0" align="char" char=".">54**</td><td class="colsep0 rowsep0" align="char" char=".">4**</td><td class="colsep0 rowsep0" align="char" char=".">37**</td><td class="colsep0 rowsep0" align="char" char=".">84</td><td class="colsep0 rowsep0" align="char" char=".">69**</td><td class="colsep0 rowsep0" align="char" char=".">95*</td><td class="colsep0 rowsep0" align="char" char=".">98</td><td class="colsep0 rowsep0" align="char" char=".">58**</td><td class="colsep0 rowsep0" align="char" char=".">45**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11a</b></td><td class="colsep0 rowsep0" align="char" char=".">73**</td><td class="colsep0 rowsep0" align="char" char=".">65**</td><td class="colsep0 rowsep0" align="char" char=".">4**</td><td class="colsep0 rowsep0" align="char" char=".">5**</td><td class="colsep0 rowsep0" align="char" char=".">82</td><td class="colsep0 rowsep0" align="char" char=".">65**</td><td class="colsep0 rowsep0" align="char" char=".">102</td><td class="colsep0 rowsep0" align="char" char=".">95</td><td class="colsep0 rowsep0" align="char" char=".">73*</td><td class="colsep0 rowsep0" align="char" char=".">57*</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11c</b></td><td class="colsep0 rowsep0" align="char" char=".">89</td><td class="colsep0 rowsep0" align="char" char=".">94</td><td class="colsep0 rowsep0" align="char" char=".">2**</td><td class="colsep0 rowsep0" align="char" char=".">4**</td><td class="colsep0 rowsep0" align="char" char=".">88</td><td class="colsep0 rowsep0" align="char" char=".">73**</td><td class="colsep0 rowsep0" align="char" char=".">96</td><td class="colsep0 rowsep0" align="char" char=".">95*</td><td class="colsep0 rowsep0" align="char" char=".">91</td><td class="colsep0 rowsep0" align="char" char=".">83</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11k</b></td><td class="colsep0 rowsep0" align="char" char=".">60**</td><td class="colsep0 rowsep0" align="char" char=".">43**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td><td class="colsep0 rowsep0" align="char" char=".">5**</td><td class="colsep0 rowsep0" align="char" char=".">84*</td><td class="colsep0 rowsep0" align="char" char=".">72**</td><td class="colsep0 rowsep0" align="char" char=".">99</td><td class="colsep0 rowsep0" align="char" char=".">104*</td><td class="colsep0 rowsep0" align="char" char=".">56*</td><td class="colsep0 rowsep0" align="char" char=".">50*</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11o</b></td><td class="colsep0 rowsep0" align="char" char=".">60**</td><td class="colsep0 rowsep0" align="char" char=".">20**</td><td class="colsep0 rowsep0" align="char" char=".">2**</td><td class="colsep0 rowsep0" align="char" char=".">5**</td><td class="colsep0 rowsep0" align="char" char=".">72*</td><td class="colsep0 rowsep0" align="char" char=".">53**</td><td class="colsep0 rowsep0" align="char" char=".">101</td><td class="colsep0 rowsep0" align="char" char=".">105*</td><td class="colsep0 rowsep0" align="char" char=".">63*</td><td class="colsep0 rowsep0" align="char" char=".">50**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13a</b></td><td class="colsep0 rowsep0" align="char" char=".">53**</td><td class="colsep0 rowsep0" align="char" char=".">51**</td><td class="colsep0 rowsep0" align="char" char=".">4**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td><td class="colsep0 rowsep0" align="char" char=".">80*</td><td class="colsep0 rowsep0" align="char" char=".">61**</td><td class="colsep0 rowsep0" align="char" char=".">103</td><td class="colsep0 rowsep0" align="char" char=".">103</td><td class="colsep0 rowsep0" align="char" char=".">64**</td><td class="colsep0 rowsep0" align="char" char=".">62*</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="char" char=".">31**</td><td class="colsep0 rowsep0" align="char" char=".">12**</td><td class="colsep0 rowsep0" align="char" char=".">4**</td><td class="colsep0 rowsep0" align="char" char=".">4**</td><td class="colsep0 rowsep0" align="char" char=".">68*</td><td class="colsep0 rowsep0" align="char" char=".">47**</td><td class="colsep0 rowsep0" align="char" char=".">103</td><td class="colsep0 rowsep0" align="char" char=".">99</td><td class="colsep0 rowsep0" align="char" char=".">57**</td><td class="colsep0 rowsep0" align="char" char=".">42*</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="char" char=".">29**</td><td class="colsep0 rowsep0" align="char" char=".">16**</td><td class="colsep0 rowsep0" align="char" char=".">4**</td><td class="colsep0 rowsep0" align="char" char=".">6**</td><td class="colsep0 rowsep0" align="char" char=".">75*</td><td class="colsep0 rowsep0" align="char" char=".">55**</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">89**</td><td class="colsep0 rowsep0" align="char" char=".">59**</td><td class="colsep0 rowsep0" align="char" char=".">49**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="char" char=".">39**</td><td class="colsep0 rowsep0" align="char" char=".">17**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td><td class="colsep0 rowsep0" align="char" char=".">5**</td><td class="colsep0 rowsep0" align="char" char=".">71*</td><td class="colsep0 rowsep0" align="char" char=".">56**</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">88**</td><td class="colsep0 rowsep0" align="char" char=".">60**</td><td class="colsep0 rowsep0" align="char" char=".">58**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33a</b></td><td class="colsep0 rowsep0" align="char" char=".">45**</td><td class="colsep0 rowsep0" align="char" char=".">32**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td><td class="colsep0 rowsep0" align="char" char=".">5**</td><td class="colsep0 rowsep0" align="char" char=".">73**</td><td class="colsep0 rowsep0" align="char" char=".">60**</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">104*</td><td class="colsep0 rowsep0" align="char" char=".">73*</td><td class="colsep0 rowsep0" align="char" char=".">49**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33b</b></td><td class="colsep0 rowsep0" align="char" char=".">45**</td><td class="colsep0 rowsep0" align="char" char=".">25**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td><td class="colsep0 rowsep0" align="char" char=".">5**</td><td class="colsep0 rowsep0" align="char" char=".">74*</td><td class="colsep0 rowsep0" align="char" char=".">60**</td><td class="colsep0 rowsep0" align="char" char=".">98</td><td class="colsep0 rowsep0" align="char" char=".">102</td><td class="colsep0 rowsep0" align="char" char=".">60*</td><td class="colsep0 rowsep0" align="char" char=".">48**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33c</b></td><td class="colsep0 rowsep0" align="char" char=".">59**</td><td class="colsep0 rowsep0" align="char" char=".">38**</td><td class="colsep0 rowsep0" align="char" char=".">3**</td><td class="colsep0 rowsep0" align="char" char=".">5**</td><td class="colsep0 rowsep0" align="char" char=".">79*</td><td class="colsep0 rowsep0" align="char" char=".">64**</td><td class="colsep0 rowsep0" align="char" char=".">98</td><td class="colsep0 rowsep0" align="char" char=".">99</td><td class="colsep0 rowsep0" align="char" char=".">59*</td><td class="colsep0 rowsep0" align="char" char=".">52**</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">DMSO (negative control) and <b>1c</b> and <b>1d</b> (positive controls) were used.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">RhlR antagonism activity of the compound (10 or 100 μM) in the presence of <b>1b</b> (10 μM).</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">RhlR agonism activity of the compound (100 or 1000 μM).</p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last">Biofilm formed by <i>P. aeruginosa</i> at 10 or 100 μM compound in static conditions.</p></div><div class="footnote" id="t2fn5"><sup><sup>e</sup></sup><p class="last">Growth of <i>P. aeruginosa</i> at 10 or 100 μM compound for 24 h.</p></div><div class="footnote" id="t2fn6"><sup><sup>f</sup></sup><p class="last">Rhamnolipid produced by <i>P. aeruginosa</i> at 10 or 100 μM compound. (**) <i>P</i> < 0.005 and (*) <i>P</i> < 0.05 as compared with the control.</p></div></div></div><div class="NLM_p">Next, we performed the comprehensive biological assays of the most potent RhlR antagonist (<b>30</b>) against <i>P. aeruginosa</i>, using <b>1c</b>, <b>1d</b>, and 4-gingerol (<b>12a</b>) as positive controls. First, the RhlR IC<sub>50</sub> values of all four compounds were measured at different concentrations (0–100 μM). The IC<sub>50</sub> of <b>1c</b>, <b>1d</b>, <b>12a</b>, and <b>30</b> were 945, 86, 218, and 26 μM, respectively (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A), confirming that compound <b>30</b> was the most potent RhlR antagonist. We also determined the agonist and antagonist activities of the compounds against LasR and PqsR, as <i>las</i> and <i>pqs</i> systems are also involved in the QS processes of <i>P. aeruginosa</i>.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> All four compounds showed neither LasR- and PqsR-agonistic activities nor PqsR-antagonism activity at 0.1 and 10 μM, respectively (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B,C). However, (<i>S</i>)-6-gingerol (<b>1c</b>) displayed moderate LasR antagonism, as previously reported.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> 4-Gingerol (<b>12a</b>) exhibited no LasR antagonism, confirming the result of our previous study.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The most potent RhlR antagonist <b>30</b> showed high selectivity for RhlR over LasR and PqsR, indicating that it is a pure RhlR antagonist.</div><figure id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/medium/jm0c00630_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. <i>In vitro</i> biological activities of compounds <b>1c</b>, <b>1d</b>, <b>12a</b>, and <b>30</b>. DMSO (negative control) and <b>1c</b>, <b>1d</b>, and <b>12a</b> (positive controls) were used. (A) Relative RhlR activity dose–response curves. (B) Relative QS receptor (LasR and PqsR) agonism activity at 0.1 and 10 μM of compounds, respectively. (C) Relative QS receptor (LasR and PqsR) antagonism activity at 0.1 and 10 μM of compounds, respectively. (D) Static biofilm formation of <i>P. aeruginosa</i> at 10 or 100 μM of compounds. (**) <i>P</i> < 0.005 and (*) <i>P</i> < 0.05 as compared with the control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00630&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D, all four compounds strongly inhibited <i>P. aeruginosa</i> biofilm formation by 14–32% (10 μM) and 29–53% (100 μM), without affecting bacterial growth (see Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_001.pdf" class="ext-link">Figure S4</a>). Dynamic biofilm inhibition by using a drip-flow reactor was analyzed at 48 h of the reactor operation. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A, the control condition displayed a typical mushroom morphology with 44.15 μm<sup>3</sup>/μm<sup>2</sup> volume and 53.90 μm thickness. However, treatment with the other compounds (<b>1c</b>, <b>1d</b>, and <b>30</b>) showed thinner and sparser formation of biofilm, as well as smaller volume and thickness, than that with the control. In particular, compound <b>30</b> inhibited biofilm formation by 74% and decreased the amount of carbohydrate and protein by 39 and 72%, respectively, as compared with the control (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B,C).</div><figure id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/medium/jm0c00630_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Biofilm formation by <i>P. aeruginosa</i> with compounds <b>1c</b>, <b>1d</b>, and <b>30</b> at 10 μM in flow conditions. DMSO (negative control) and <b>1c</b>, <b>1d</b>, and <b>12a</b> (positive controls) were used. (A) CLSM images with biofilm volume and thickness. (B) Amount of carbohydrate in extracellular polymeric substance (EPS). (C) Amount of protein in EPS. (**) <i>P</i> < 0.005 and (*) <i>P</i> < 0.05 as compared with the control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00630&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In the next virulence production experiment, compound <b>30</b> reduced significantly rhamnolipid production by <i>P. aeruginosa</i> at 10 and 100 μM concentrations (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A), indicating that RhlR antagonists can regulate rhamnolipid production and play an important role in biofilm development stages.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> In general, rhamnolipid increases the hydrophobicity of cells and bacterial twitching motility, affecting attachment and maturation in biofilm formation stages.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a><i>P. aeruginosa</i><i>rhlA</i> mutants which are downregulated for the synthesis of rhamnolipid did not form mushroom-like biofilms,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> suggesting that the RhlR antagonist-induced reduction of rhamnolipid production may control biofilm formation in <i>P. aeruginosa</i>.</div><figure id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/medium/jm0c00630_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Production of virulence factors and expression of QS-induced genes by <i>P. aeruginosa</i> with compounds <b>1c</b>, <b>1d</b>, <b>12a</b>, and <b>30</b>. DMSO (C, negative control) and <b>1c</b>, <b>1d</b>, and <b>12a</b> (positive controls) were used. (A) Rhamnolipid production by wild-type <i>P. aeruginosa</i> at 10 or 100 μM compound. (B) Relative <i>rhlA</i> expression of biofilm cells at 10 μM compound. (C) Pyocyanin production by wild-type <i>P. aeruginosa</i> and <i>rhlR</i> mutants at 10 μM compound. (D) Relative QS-related gene expression of wild-type <i>P. aeruginosa</i> and <i>rhlR</i> mutants. (**) <i>P</i> < 0.005 and (*) <i>P</i> < 0.05 as compared with the control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00630&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We hypothesized that compound <b>30</b>-induced inhibition of biofilm formation and virulence factor production are closely associated with the <i>rhl</i> system. To verify this hypothesis, we measured rhamnolipid synthesis gene (<i>rhlA</i>) expression in compound <b>30</b>-treated biofilm cells using the reverse transcription polymerase chain reaction (RT-qPCR) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). The RhlR antagonists (<b>1d</b>, <b>12a</b>, and <b>30</b>) downregulated <i>rhlA</i> expression of biofilm cells more significantly (65–79%) than compound <b>1c</b> (38%), indicating that <i>rhlA</i> expression can be controlled by RhlR antagonists. These RT-qPCR results demonstrated that compound <b>30</b> could inhibit <i>P. aeruginosa</i> biofilm formation and virulence factor production by downregulating the <i>rhlA</i> expression of <i>P. aeruginosa</i>. However, the <i>rhl</i> system was reported to negatively modulate the <i>pqs</i> system, leading to increased pyocyanin production.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Considering the relationship between the <i>rhl</i> and <i>pqs</i> systems, Blackwell and co-workers developed an RhlR agonist and used it as an anti-virulence strategy against pathogens.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The RhlR agonist strongly inhibited pyocyanin production by suppressing <i>pqs</i> signaling. Based on their results, we expected that RhlR antagonists would increase pyocyanin production, thus negatively affecting biofilm formation and virulence production. Surprisingly, <i>rhlR</i> mutants produced less pyocyanin than wild-type <i>P. aeruginosa,</i> as shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C. In addition, compound <b>30</b> reduced pyocyanin production by 44% as compared with the control. However, there was no difference of virulence factor production between compound <b>30</b> and the control in <i>rhlR</i> mutants (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C), indicating that compound <b>30</b> reduced pyocyanin production by blocking RhlR. This difference might be related to the stage when pyocyanin is produced in <i>P. aeruginosa</i>. RhlR agonists are suggested to suppress the <i>pqs</i> system only before basal pyocyanin is produced.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> According to the microarray experiment by Givskov and co-workers, <i>rhlR</i> mutants repressed <i>rhl</i> system-related genes, especially rhamnolipid production-related genes (<i>rhlA</i> and <i>rhlB</i>), but insignificantly affect <i>las</i>- or <i>pqs</i>-related system genes.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Pyocyanin production-related genes (<i>phzA</i>-<i>D</i>) were also downregulated in <i>rhlR</i> mutants. Bassler and co-workers reported the downregulation of pyocyanin biosynthetic genes (<i>phzA1</i>-<i>G1</i>) and decreased pyocyanin production in <i>rhlR</i> mutants.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Similarly, according to our RT-qPCR experiment results, <i>rhlR</i> mutants decreased <i>rhlA</i> and <i>phzC1</i> expression (<i>P</i> < 0.005) without affecting the expression of other QS-related genes (<i>lasR</i>, <i>lasB</i>, and <i>pqsR</i>) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D).</div><div class="NLM_p">Encouraged by <i>in vitro</i> assay results, we determined the effect of compound <b>30</b> on <i>Tenebrio molitor</i> (<i>T. molitor</i>) larvae mortality. Larvae injected with <i>P. aeruginosa</i> started to die in the initial incubation time and 70% of them died at the end of the 20-day incubation period (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). The survival rate of compound <b>30</b>-treated larvae was greatly improved, with approximately 80% larvae surviving at the end of the incubation period. In addition, larvae injected with <i>rhlR</i> mutants of <i>P. aeruginosa</i> showed a 90% survival rate after 20 days. Moreover, when <i>rhlR</i> mutants were treated with different concentrations of rhamnolipid (0.01–10 μM) in larvae, their mortality was increased depending on the rhamnolipid concentration, emphasizing the key role of RhlR in the production of the virulence factor (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_001.pdf" class="ext-link">Figure S5</a>). Based on <i>in vitro</i> and <i>in vivo</i> results, we propose that the pure RhlR antagonist <b>30</b> inhibits rhamnolipid and pyocyanin production by inactivating virulence factor production gene expression (<i>e.g.</i>, <i>rhl</i> and <i>phz</i> genes), as described in a working model (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>).</div><figure id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/medium/jm0c00630_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Mortality of <i>T. molitor</i> larvae injected <i>P. aeruginosa</i> with compounds <b>1c</b>, <b>1d</b>, <b>12a</b>, and <b>30</b> at 10 μM. DMSO (negative control) and <b>1c</b>, <b>1d</b>, and <b>12a</b> (positive controls) were used.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00630&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/medium/jm0c00630_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Working model to control the QS system of <i>P. aeruginosa</i> by compound <b>30</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00630&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81381" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81381" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">RhlR is one of the key LuxR-type receptors in the QS network of <i>P. aeruginosa</i> and considered as an attractive target protein for the discovery of biofilm inhibitors. We systemically synthesized 55 gingerol analogues and evaluated their relative RhlR activities using QS reporter strain assay. Comprehensive SAR studies identified the alkynyl ketone <b>30</b> as the most potent RhlR antagonist. Compound <b>30</b> displayed high RhlR antagonism (no RhlR agonism at 1000 μM), strong inhibition of biofilm formation in static and dynamic settings, and reduced production of virulence factors such as rhamnolipid and pyocyanin in <i>P. aeruginosa</i>. It also displayed selective affinity for RhlR over LasR and PqsR. Furthermore, compound <b>30</b> significantly increased the <i>in vivo</i> survival rate of <i>T. molitor</i> larvae as compared with the control. In conclusion, the pure and potent RhlR antagonist <b>30</b> which we discovered for the first time can be utilized for investigating <i>rhl</i>-related QS mechanism and developing a novel antivirulence strategy to control <i>P. aeruginosa</i> infections.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72633" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72633" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> General</h3><div class="NLM_p last">All chemicals and solvents used in the reaction were purchased from Sigma-Aldrich, TCI, and Acros and were used without further purification. The reaction progress was monitored by thin-layer chromatography (TLC) on precoated silica gel plates with silica gel 60F<sub>254</sub> (Merck; Darmstadt, Germany) and visualized by UV254 light and/or KMnO<sub>4</sub> staining for detection purposes. Column chromatography was performed on a silica gel (silica gel 60; 230–400 mesh ASTM, Merck, Darmstadt, Germany). Nuclear magnetic resonance (NMR) spectra were recorded at room temperature on either a Bruker BioSpin AVANCE 300 MHz NMR (<sup>1</sup>H, 300 MHz; <sup>13</sup>C, 75 MHz) or a Bruker UltraShield 600 MHz Plus (<sup>1</sup>H, 600 MHz; <sup>13</sup>C, 150 MHz) spectrometer. All chemical shifts are reported in parts per million (ppm) from tetramethylsilane (δ = 0) and were measured relative to the solvent in which the sample was analyzed (CDCl<sub>3</sub>: δ 7.26 for <sup>1</sup>H NMR, δ 77.0 for <sup>13</sup>C NMR; MeOH-<i>d</i><sub>4</sub>: δ 3.31 for <sup>1</sup>H NMR, δ 49.0 for <sup>13</sup>C NMR). The <sup>1</sup>H NMR shift values are reported as chemical shift (δ), the corresponding integral, multiplicity (s = singlet, br = broad, d = doublet, t = triplet, q = quartet, m = multiplet, dd = doublet of doublets, td = triplet of doublets, qd = quartet of doublets), coupling constant (<i>J</i> in Hz), and assignments. High-resolution mass spectra (HRMS) were recorded on an Agilent 6530 Accurate Mass Q-TOF LC/MS spectrometer. The purity of all final compounds was measured by analytical reverse-phase HPLC on an Agilent 1260 Infinity (Agilent) with a C18 column (Phenomenex, 150 mm × 4.6 mm, 3 μm, 110 Å). RP-HPLC was performed using the following isocratic conditions: for method A, the mobile phase was acetonitrile and water (30:70, v/v); for method B, the mobile phase was acetonitrile and water with 0.1% trifluoroacetic acid (TFA) (30:70, v/v); for method C, the mobile phase was acetonitrile and water (40:60, v/v); for method D, the mobile phase was acetonitrile and water (50:50, v/v); for method E, the mobile phase was acetonitrile and water (55:45, v/v); for method F, the mobile phase was acetonitrile and water (60:40, v/v); for method G, the mobile phase was methanol and water (50:50, v/v); for method H, the mobile phase was methanol and water (60:40, v/v); and for method I, the mobile phase was methanol and water (70:30, v/v). All compounds were eluted with a flow rate of 1 mL/min and monitored at UV detector (220 nm or 254 nm). The purity of the tested compounds was >95%.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Chemical Synthesis</h3><div class="NLM_p">Compounds <b>2–8</b> were reported by our previous study.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The chemical structure and name for compounds <b>2–8</b> can be found in the Supporting Information (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_001.pdf" class="ext-link">Scheme S1</a>).</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i13" class="anchor-spacer"></div><h4 class="article-section__title" id="_i13"> General Procedure A for Compounds <b>11a–11r</b></h4><div class="NLM_p">To a stirred solution of benzylideneacetone compound in tetrahydrofuran (THF) (20 mL) was added LDA (1.2–3.6 equiv) at −78 °C. The solution was stirred under argon for 1 h at the same temperature, and then, <i>n</i>-butanal (10.0 equiv) was added dropwise. The reaction mixture was vigorously stirred at the same temperature until TLC analysis indicated complete conversion (typically 3–24 h), quenched with aqueous NH<sub>4</sub>Cl (10 mL), and extracted with EtOAc (3 × 25 mL). The organic layer was washed with brine, dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude residue was purified by column chromatography on a silica gel to furnish compounds.</div><div id="sec4_2_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i14" class="anchor-spacer"></div><h5 class="article-section__title" id="_i14"> (<i>E</i>)-5-Hydroxy-1-(4-hydroxy-3-methoxyphenyl)oct-1-en-3-one (<b>11a</b>)</h5><div class="NLM_p last">Compound <b>11a</b> was prepared in 60% yield as a yellow oil following the same procedure as described in the general procedure A with (<i>E</i>)-4-(4-hydroxy-3-methoxyphenyl)but-3-en-2-one (300 mg, 1.56 mmol) and LDA (4.10 mL, 2.4 equiv) under stirring for 3 h. The crude residue was purified by column chromatography on a silica gel (toluene/EtOAc = 6:1 to 3:1, v/v). <i>R</i><sub><i>f</i></sub> 0.40 (toluene/EtOAc = 2:1, v/v). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.53 (d, <i>J</i> = 16.1 Hz, 1H), 7.13 (dd, <i>J</i> = 1.8 and 8.2 Hz, 1H), 7.08 (d, <i>J</i> = 1.7 Hz, 1H), 6.96 (d, <i>J</i> = 8.2 Hz, 1H), 6.61 (d, <i>J</i> = 16.1 Hz, 1H), 5.94 (s, 1H), 4.23–4.09 (m, 1H), 3.96 (s, 3H), 3.30 (d, <i>J</i> = 2.9 Hz, 1H), 2.90 (dd, <i>J</i> = 2.7 and 17.1 Hz, 1H), 2.75 (q, <i>J</i> = 9.0 Hz, 1H), 1.59–1.36 (m, 4H), 0.97 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 201.03, 148.50, 146.89, 143.87, 126.72, 124.15, 123.76, 114.87, 109.51, 67.65, 55.98, 46.52, 38.68, 18.75, 14.05, 13.97. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>20</sub>O<sub>4</sub> [M – H]<sup>−</sup>, 263.1289; found, 263.1304. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 6.34 min, method G, <i>t</i><sub>R</sub> = 7.33 min).</div></div><div id="sec4_2_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i15" class="anchor-spacer"></div><h5 class="article-section__title" id="_i15"> (<i>E</i>)-5-Hydroxy-1-phenyloct-1-en-3-one (<b>11b</b>)</h5><div class="NLM_p last">Compound <b>11b</b> was prepared in 36% yield as a yellow oil following the same procedure as described in the general procedure A with (<i>E</i>)-4-phenylbut-3-en-2-one (300 mg, 2.05 mmol) and LDA (2.70 mL, 1.2 equiv) under stirring for 2 h. The crude residue was purified by column chromatography on a silica gel (toluene/EtOAc = 6:1 to 3:1, v/v). <i>R</i><sub><i>f</i></sub> 0.40 (toluene/EtOAc = 2:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.58 (d, <i>J</i> = 16.2 Hz, 1H), 7.56–7.54 (m, 2H), 7.43–7.39 (m, 3H), 6.74 (d, <i>J</i> = 16.2 Hz, 1H), 4.19–4.13 (m, 1H), 3.17 (d, <i>J</i> = 3.30 Hz, 1H), 2.88 (dd, <i>J</i> = 2.6 and 17.3 Hz, 1H), 2.77 (q, <i>J</i> = 9.1 Hz, 1H), 1.62–1.36 (m, 4H), 0.96 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 211.24, 140.75, 128.56, 128.29, 126.22, 67.36, 49.32, 45.06, 38.59, 29.53, 18.65, 13.98. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>18</sub>O<sub>2</sub> [M – H]<sup>−</sup>, 217.1234; found, 217.1252. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 12.91 min, method G, <i>t</i><sub>R</sub> = 15.79 min).</div></div><div id="sec4_2_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i16" class="anchor-spacer"></div><h5 class="article-section__title" id="_i16"> (<i>E</i>)-5-Hydroxy-1-(3-methoxyphenyl)oct-1-en-3-one (<b>11c</b>)</h5><div class="NLM_p last">Compound <b>11c</b> was prepared in 41% yield as a yellow oil following the same procedure as described in the general procedure A with (<i>E</i>)-4-(3-methoxyphenyl)but-3-en-2-one (300 mg, 1.70 mmol) and LDA (2.23 mL, 1.2 equiv) under stirring for 4 h. The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 10:1 to 6:1, v/v). <i>R</i><sub><i>f</i></sub> 0.48 (hexane/EtOAc = 2:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.55 (d, <i>J</i> = 16.1 Hz, 1H), 7.33 (t, <i>J</i> = 7.9 Hz, 1H), 7.16 (d, <i>J</i> = 7.6 Hz, 1H), 7.08 (s, 1H), 6.98 (d, <i>J</i> = 8.2 Hz, 1H), 6.73 (d, <i>J</i> = 16.2 Hz, 1H), 4.17 (brs, 1H), 3.86 (s, 3H), 3.22 (brs, 1H), 2.91–2.88 (m, 1H), 2.80–2.76 (m, 1H), 1.61–1.40 (m, 4H), 0.97 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 201.04, 159.98, 143.48, 135.60, 130.02, 126.65, 121.15, 116.66, 113.20, 67.61, 55.35, 46.86, 38.69, 18.75, 14.04. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>20</sub>O<sub>3</sub> [M – H]<sup>−</sup>, 247.1339; found, 247.1344. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 14.25 min, method G, <i>t</i><sub>R</sub> = 19.69 min).</div></div><div id="sec4_2_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i17" class="anchor-spacer"></div><h5 class="article-section__title" id="_i17"> (<i>E</i>)-5-Hydroxy-1-(4-hydroxyphenyl)oct-1-en-3-one (<b>11d</b>)</h5><div class="NLM_p last">Compound <b>11d</b> was prepared in 43% yield as a yellow oil following the same procedure as described in the general procedure A with (<i>E</i>)-4-(4-hydroxyphenyl)but-3-en-2-one (250 mg, 1.54 mmol) and LDA (4.10 mL, 2.4 equiv) under stirring for 6 h. The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 10:1 to 3:1, v/v). <i>R</i><sub><i>f</i></sub> 0.42 (hexane/EtOAc = 2:1, v/v). <sup>1</sup>H NMR (600 MHz, MeOD): δ 7.61 (d, <i>J</i> = 16.1 Hz, 1H), 7.53 (d, <i>J</i> = 8.6 Hz, 1H), 6.84 (d, <i>J</i> = 8.6 Hz, 1H), 6.72 (d, <i>J</i> = 16.1 Hz, 1H), 4.14–4.12 (m, 1H), 2.88–2.84 (m, 1H), 2.78–2.75 (m, 1H), 1.55–1.41 (m, 4H), 0.97 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, MeOD): δ 200.62, 160.22, 144.09, 130.21, 125.86, 123.16, 115.54, 67.81, 39.25, 18.45, 12.98. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>18</sub>O<sub>3</sub> [M – H]<sup>−</sup>, 233.1183; found, 233.1213. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 6.18 min, method G, <i>t</i><sub>R</sub> = 8.08 min).</div></div><div id="sec4_2_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i18" class="anchor-spacer"></div><h5 class="article-section__title" id="_i18"> (<i>E</i>)-5-Hydroxy-1-(3-hydroxyphenyl)oct-1-en-3-one (<b>11e</b>)</h5><div class="NLM_p last">Compound <b>11e</b> was prepared in 34% yield as a yellow oil following the same procedure as described in the general procedure A with (<i>E</i>)-4-(3-hydroxyphenyl)but-3-en-2-one (250 mg, 1.54 mmol) and LDA (4.10 mL, 2.4 equiv) under stirring for 6 h. The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 10:1 to 3:1, v/v). <i>R</i><sub><i>f</i></sub> 0.30 (hexane/EtOAc = 2:1, v/v). <sup>1</sup>H NMR (600 MHz, MeOD): δ 7.58 (d, <i>J</i> = 16.1 Hz, 1H), 7.24 (t, <i>J</i> = 7.9 Hz, 1H), 7.11 (d, <i>J</i> = 7.6 Hz, 1H), 7.06 (s, 1H), 6.87 (dd, <i>J</i> = 8.1 and 1.8 Hz, 1H), 6.80 (d, <i>J</i> = 16.2 Hz, 1H), 4.16–4.11 (m, 1H), 2.90–2.86 (m, 1H), 2.79–2.77 (m, 1H), 1.54–1.40 (m, 4H), 0.97 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, MeOD): δ 200.49, 157.73, 143.65, 135.85, 129.65, 126.18, 119.78, 117.53, 114.14, 67.66, 39.25, 18.45, 12.96. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>18</sub>O<sub>3</sub> [M – H]<sup>−</sup>, 233.1183; found, 233.1200. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 5.68 min, method G, <i>t</i><sub>R</sub> = 8.17 min).</div></div><div id="sec4_2_1_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i19" class="anchor-spacer"></div><h5 class="article-section__title" id="_i19"> (<i>E</i>)-5-Hydroxy-1-(4-methoxyphenyl)oct-1-en-3-one (<b>11f</b>)</h5><div class="NLM_p last">Compound <b>11f</b> was prepared in 48% yield as a white oil following the same procedure as described in the general procedure A with (<i>E</i>)-4-(4-methoxyphenyl)but-3-en-2-one (300 mg, 1.70 mmol) and LDA (2.30 mL, 1.2 equiv) under stirring for 2 h. The crude residue was purified by column chromatography on a silica gel (toluene/EtOAc = 10:1 to 3:1, v/v). <i>R</i><sub><i>f</i></sub> 0.55 (toluene/EtOAc = 2:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.55–7.50 (m, 3H), 6.93 (d, <i>J</i> = 8.7 Hz, 2H), 6.62 (d, <i>J</i> = 16.2 Hz, 1H), 4.14 (brs, 1H), 3.85 (s, 3H), 3.27 (brs, 1H), 2.88–2.85 (m, 1H), 2.75–2.71 (m, 1H), 1.62–1.42 (m, 4H), 0.95 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 201.01, 161.83, 143.39, 130.19, 126.87, 124.18, 114.49, 67.73, 55.41, 46.71, 38.75, 18.75, 14.04. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>20</sub>O<sub>3</sub> [M – H]<sup>−</sup>, 247.1339; found, 247.1357. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 12.88 min, method G, <i>t</i><sub>R</sub> = 18.64 min).</div></div><div id="sec4_2_1_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i20" class="anchor-spacer"></div><h5 class="article-section__title" id="_i20"> (<i>E</i>)-1-(4-Fluorophenyl)-5-hydroxyoct-1-en-3-one (<b>11g</b>)</h5><div class="NLM_p last">Compound <b>11g</b> was prepared in 38% yield as a yellow oil following the same procedure as described in the general procedure A with (<i>E</i>)-4-(4-fluorophenyl)but-3-en-2-one (200 mg, 1.22 mmol) and LDA (1.60 mL, 1.2 equiv) under stirring for 7 h. The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 10:1 to 6:1, v/v). <i>R</i><sub><i>f</i></sub> 0.40 (hexane/EtOAc = 2:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.56–7.53 (m, 3H), 7.10 (t, <i>J</i> = 8.6 Hz, 2H), 6.66 (d, <i>J</i> = 16.2 Hz, 1H), 4.17 (brs, 1H), 3.13 (s, 1H), 2.88–2.85 (m, 1H), 2.77–2.73 (m, 1H), 1.59–1.40 (m, 4H), 0.96 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 200.74, 165.03, 163.36, 142.15, 130.50, 130.48, 130.35, 130.30, 126.08, 126.07, 116.30, 116.16, 67.61, 47.04, 38.70, 18.74, 14.02. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>17</sub>FO<sub>2</sub> [M – H]<sup>−</sup>, 235.1140; found, 235.1155. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 14.12 min, method G, <i>t</i><sub>R</sub> = 17.64 min).</div></div><div id="sec4_2_1_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i21" class="anchor-spacer"></div><h5 class="article-section__title" id="_i21"> (<i>E</i>)-1-(4-Chlorophenyl)-5-hydroxyoct-1-en-3-one (<b>11h</b>)</h5><div class="NLM_p last">Compound <b>11h</b> was prepared in 41% yield as a yellow oil following the same procedure as described in the general procedure A with (<i>E</i>)-4-(4-(dimethylamino)phenyl)but-3-en-2-one (200 mg, 1.11 mmol) and LDA (1.40 mL, 1.2 equiv) under stirring for 6 h. The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 12:1 to 6:1, v/v). <i>R</i><sub><i>f</i></sub> 0.35 (hexane/EtOAc = 2:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.52 (d, <i>J</i> = 16.2 Hz, 1H), 7.48 (d, <i>J</i> = 8.8 Hz, 2H), 7.37 (d, <i>J</i> = 8.5 Hz, 2H), 6.70 (d, <i>J</i> = 16.2 Hz, 1H), 4.15 (brs, 1H), 3.13 (brs, 1H), 2.87–2.84 (m, 1H), 2.78–2.74 (m, 1H), 1.59–1.39 (m, 4H), 0.95 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 200.64, 141.91, 136.66, 132.75, 129.54, 129.28, 126.72, 67.58, 47.21, 38.75, 18.75, 14.02. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>17</sub>ClO<sub>2</sub> [M – H]<sup>−</sup>, 251.0844; found, 251.0862. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 22.36 min, method G, <i>t</i><sub>R</sub> = 34.88 min).</div></div><div id="sec4_2_1_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i22" class="anchor-spacer"></div><h5 class="article-section__title" id="_i22"> (<i>E</i>)-5-Hydroxy-1-(<i>p</i>-tolyl)oct-1-en-3-one (<b>11i</b>)</h5><div class="NLM_p last">Compound <b>11i</b> was prepared in 29% yield as a yellow oil following the same procedure as described in the general procedure A with (<i>E</i>)-4-(p-tolyl)but-3-en-2-one (300 mg, 1.87 mmol) and LDA (2.48 mL, 1.2 equiv) under stirring for 4 h. The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 8:1 to 4:1, v/v). <i>R</i><sub><i>f</i></sub> 0.45 (hexane/EtOAc = 2:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.55 (d, <i>J</i> = 16.2 Hz, 1H), 7.45 (d, <i>J</i> = 8.2 Hz, 2H), 7.21 (d, <i>J</i> = 8.0 Hz, 2H), 6.69 (d, <i>J</i> = 16.2 Hz, 1H), 4.16–4.14 (m, 1H), 3.25 (d, <i>J</i> = 3.2 Hz, 1H), 2.88–2.85 (m, 1H), 2.77–2.72 (m, 1H), 2.38 (s, 1H), 1.59–1.39 (m, 4H), 0.95 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 201.10, 143.63, 141.37, 131.48, 129.77, 128.44, 125.46, 46.77, 38.72, 21.53, 18.75, 14.04. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>20</sub>O<sub>2</sub> [M – H]<sup>−</sup>, 231.1390; found, 231.1404. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 19.34 min, method G, <i>t</i><sub>R</sub> = 30.62 min).</div></div><div id="sec4_2_1_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i23" class="anchor-spacer"></div><h5 class="article-section__title" id="_i23"> (<i>E</i>)-1-(4-(Dimethylamino)phenyl)-5-hydroxyoct-1-en-3-one (<b>11j</b>)</h5><div class="NLM_p last">Compound <b>11j</b> was prepared in 60% yield as a yellow oil following the same procedure as described in the general procedure A with (<i>E</i>)-4-(4-(dimethylamino)phenyl)but-3-en-2-one (200 mg, 1.06 mmol) and LDA (1.40 mL, 1.2 equiv) under stirring for 8 h. The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 10:1 to 3:1, v/v). <i>R</i><sub><i>f</i></sub> 0.31 (hexane/EtOAc = 2:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.52 (d, <i>J</i> = 16.0 Hz, 1H), 7.45 (d, <i>J</i> = 8.9 Hz, 2H), 6.67 (d, <i>J</i> = 8.9 Hz, 2H), 6.54 (d, <i>J</i> = 16.0 Hz, 1H), 4.13–4.11 (m, 1H), 3.50–3.49 (m, 1H), 3.04 (s, 6H), 2.88–2.84 (m, 1H), 2.71–2.67 (m, 1H), 1.60–1.39 (m, 4H), 0.95 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 201.06, 152.15, 144.52, 130.34, 121.75, 121.40, 111.84, 67.90, 46.22, 40.10, 38.78, 18.77, 14.08. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>16</sub>H<sub>23</sub>NO<sub>2</sub> [M – H]<sup>−</sup>, 260.1656; found, 260.1674. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 17.13 min, method G, <i>t</i><sub>R</sub> = 30.68 min).</div></div><div id="sec4_2_1_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24"> (<i>E</i>)-1-(3,4-Dihydroxyphenyl)-5-hydroxyoct-1-en-3-one (<b>11k</b>)</h5><div class="NLM_p last">Compound <b>11k</b> was prepared in 30% yield as a yellow oil following the same procedure as described in the general procedure A with (<i>E</i>)-4-(3,4-dihydroxyphenyl)but-3-en-2-one (150 mg, 0.84 mmol) and LDA (3.30 mL, 3.6 equiv) under stirring for 6 h. The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 3:1 to 1:1, v/v). <i>R</i><sub><i>f</i></sub> 0.50 (hexane/EtOAc = 1:3, v/v). <sup>1</sup>H NMR (600 MHz, MeOD): δ 7.53 (d, <i>J</i> = 16.1 Hz, 1H), 7.10 (d, <i>J</i> = 1.8 Hz, 1H), 7.01 (dd, <i>J</i> = 1.8 and 8.2 Hz, 1H), 6.81 (d, <i>J</i> = 8.2 Hz, 1H), 6.64 (d, <i>J</i> = 16.1 Hz, 1H), 4.18–4.05 (m, 1H), 2.84 (q, <i>J</i> = 8.2 Hz, 1H), 2.74 (dd, <i>J</i> = 4.4 and 15.4 Hz, 1H), 1.57–1.35 (m, 4H), 0.96 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, MeOD): δ 200.69, 148.58, 145.48, 144.61, 126.45, 123.18, 122.21, 115.20, 114.03, 67.86, 39.23, 18.47, 13.00. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>18</sub>O<sub>4</sub> [M – H]<sup>−</sup>, 249.1132; found, 249.1232. >95% purity (as determined by RP-HPLC, method B, <i>t</i><sub>R</sub> = 5.21 min, method G, <i>t</i><sub>R</sub> = 5.60 min).</div></div><div id="sec4_2_1_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i25" class="anchor-spacer"></div><h5 class="article-section__title" id="_i25"> (<i>E</i>)-1-(3,4-Dimethoxyphenyl)-5-hydroxyoct-1-en-3-one (<b>11l</b>)</h5><div class="NLM_p last">Compound <b>11l</b> was prepared in 54% yield as a yellow oil following the same procedure as described in the general procedure A with (<i>E</i>)-4-(3,4-dimethoxyphenyl)but-3-en-2-one (200 mg, 0.97 mmol) and LDA (1.30 mL, 1.2 equiv) under stirring for 1 h. The crude residue was purified by column chromatography on a silica gel (toluene/EtOAc = 10:1 to 3:1, v/v). <i>R</i><sub><i>f</i></sub> 0.50 (toluene/EtOAc = 2:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.53 (d, <i>J</i> = 16.1 Hz, 1H), 7.14 (d, <i>J</i> = 8.3 Hz, 1H), 7.07 (s, 1H), 6.89 (d, <i>J</i> = 8.3 Hz, 1H), 6.61 (d, <i>J</i> = 16.1 Hz, 1H), 4.15 (brs, 1H), 3.93 (s, 6H), 3.25 (s, 1H), 2.89–2.86 (m, 1H), 2.77–2.72 (m, 1H), 1.61–1.40 (m, 4H), 0.96 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 200.95, 151.60, 149.30, 143.66, 127.13, 124.41, 123.33, 111.10, 109.74, 67.72, 56.00, 55.91, 46.63, 38.73, 18.74, 14.04. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>16</sub>H<sub>22</sub>O<sub>4</sub> [M – H]<sup>−</sup>, 277.1445; found, 277.1478. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 8.92 min, method G, <i>t</i><sub>R</sub> = 11.13 min).</div></div><div id="sec4_2_1_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i26" class="anchor-spacer"></div><h5 class="article-section__title" id="_i26"> (<i>E</i>)-1-(3-Fluoro-4-hydroxyphenyl)-5-hydroxyoct-1-en-3-one (<b>11m</b>)</h5><div class="NLM_p last">Compound <b>11m</b> was prepared in 45% yield as a yellow oil following the same procedure as described in the general procedure A with (<i>E</i>)-4-(3-fluoro-4-hydroxyphenyl)but-3-en-2-one (100 mg, 0.55 mmol) and LDA (1.50 mL, 2.4 equiv) under stirring for 4 h. The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 10:1 to 3:1, v/v). <i>R</i><sub><i>f</i></sub> 0.25 (hexane/EtOAc = 2:1, v/v). <sup>1</sup>H NMR (600 MHz, MeOD): δ 7.56 (d, <i>J</i> = 16.1 Hz, 1H), 7.42 (d, <i>J</i> = 12.1 Hz, 1H), 7.31 (d, <i>J</i> = 8.4 Hz, 1H), 6.95 (t, <i>J</i> = 8.6 Hz, 3H), 6.73 (d, <i>J</i> = 16.1 Hz, 1H), 4.14 (brs, 1H), 3.26 (s, 1H), 2.88–2.83 (m, 1H), 2.78–2.75 (m, 1H), 1.53–1.40 (m, 4H), 0.96 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, MeOD): δ 200.36, 152.43, 150.82, 147.75, 147.66, 142.75, 142.74, 126.70, 126.66, 125.76, 125.74, 124.49, 117.67, 117.65, 115.12, 114.99, 67.73, 39.25, 18.46, 12.99. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>17</sub>FO<sub>3</sub> [M – H]<sup>−</sup>, 251.1089; found, 251.1117. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 6.49 min, method G, <i>t</i><sub>R</sub> = 8.91 min).</div></div><div id="sec4_2_1_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i27" class="anchor-spacer"></div><h5 class="article-section__title" id="_i27"> (<i>E</i>)-1-(3-Ethoxy-4-hydroxyphenyl)-5-hydroxyoct-1-en-3-one (<b>11n</b>)</h5><div class="NLM_p last">Compound <b>11n</b> was prepared in 39% yield as a yellow oil following the same procedure as described in the general procedure A with (<i>E</i>)-4-(3-ethoxy-4-hydroxyphenyl)but-3-en-2-one (200 mg, 0.97 mmol) and LDA (2.60 mL, 2.4 equiv) under stirring for 3 h. The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 10:1 to 3:1, v/v). <i>R</i><sub><i>f</i></sub> 0.24 (hexane/EtOAc = 2:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.50 (d, <i>J</i> = 16.1 Hz, 1H), 7.10 (d, <i>J</i> = 8.1 Hz, 1H), 7.04 (s, 1H), 6.94 (d, <i>J</i> = 8.2 Hz, 1H), 6.58 (d, <i>J</i> = 16.1 Hz, 1H), 5.94 (s, 1H), 4.18–4.15 (q, <i>J</i> = 7.0 Hz, 2H), 3.26 (s, 1H), 2.88–2.85 (m, 1H), 2.75–2.71 (m, 1H), 1.50–1.47 (m, 4H), 0.95 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 201.02, 148.69, 146.22, 143.98, 126.64, 124.05, 123.59, 114.89, 110.44, 67.76, 64.64, 46.52, 38.70, 18.75, 14.79, 14.05. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>16</sub>H<sub>22</sub>O<sub>4</sub> [M – H]<sup>−</sup>, 277.1445; found, 277.1472. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 8.42 min, method G, <i>t</i><sub>R</sub> = 10.65 min).</div></div><div id="sec4_2_1_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> (<i>E</i>)-1-(3,4-Difluorophenyl)-5-hydroxyoct-1-en-3-one (<b>11o</b>)</h5><div class="NLM_p last">Compound <b>11o</b> was prepared in 38% yield as a yellow oil following the same procedure as described in the general procedure A with (<i>E</i>)-4-(3,4-difluorophenyl)but-3-en-2-one (200 mg, 1.10 mmol) and LDA (1.60 mL, 1.2 equiv) under stirring for 18 h. The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 10:1 to 2:1, v/v). <i>R</i><sub><i>f</i></sub> 0.38 (hexane/EtOAc = 2:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.47 (d, <i>J</i> = 16.1 Hz, 1H), 7.38 (t, <i>J</i> = 9.2 Hz, 1H), 7.30–7.28 (m, 1H), 7.20 (q, <i>J</i> = 8.7 Hz, 1H), 6.64 (d, <i>J</i> = 16.1 Hz, 1H), 4.16 (brs, 1H), 3.09 (s, 1H), 2.86–2.83 (m, 1H), 2.78–2.74 (m, 1H), 1.60–1.36 (m, 4H), 0.96 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 200.32, 152.61, 152.52, 151.52, 151.43, 150.92, 150.84, 149.87, 149.78, 140.88, 131.56, 131.52, 131.49, 127.09, 127.08, 125.26, 125.23, 125.21, 125.19, 118.01, 117.89, 116.58, 116.47. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>16</sub>F<sub>2</sub>O<sub>2</sub> [M – H]<sup>−</sup>, 253.1045; found, 253.1062. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 17.12 min, method G, <i>t</i><sub>R</sub> = 22.40 min).</div></div><div id="sec4_2_1_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29"> (<i>E</i>)-1-(3,4-Dimethylphenyl)-5-hydroxyoct-1-en-3-one (<b>11p</b>)</h5><div class="NLM_p last">Compound <b>11p</b> was prepared in 37% yield as a yellow oil following the same procedure as described in the general procedure A with (<i>E</i>)-4-(3,4-dimethylphenyl)but-3-en-2-one (200 mg, 1.15 mmol) and LDA (1.50 mL, 1.2 equiv) under stirring for 8 h. The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 8:1 to 4:1, v/v). <i>R</i><sub><i>f</i></sub> 0.55 (hexane/EtOAc = 2:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.53 (d, <i>J</i> = 16.1 Hz, 1H), 7.32 (s, 1H), 7.29 (d, <i>J</i> = 7.8 Hz, 1H), 7.16 (d, <i>J</i> = 7.7 Hz, 1H), 6.68 (d, <i>J</i> = 16.2 Hz, 1H), 4.16–4.12 (m, 1H), 3.29 (brs, 1H), 2.88–2.85 (m, 1H), 2.76–2.72 (m, 1H), 2.28 (s, 6H), 1.60–1.39 (m, 4H), 0.95 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 201.12, 143.86, 140.14, 137.30, 131.87, 130.30, 129.63, 126.10, 125.31, 67.67, 46.73, 38.73, 19.88, 19.75, 18.76, 14.05. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>16</sub>H<sub>22</sub>O<sub>2</sub> [M – H]<sup>−</sup>, 245.1547; found, 245.1558. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 28.39 min, method H, <i>t</i><sub>R</sub> = 19.67 min).</div></div><div id="sec4_2_1_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> (<i>E</i>)-1-(4-Fluoro-3-methoxyphenyl)-5-hydroxyoct-1-en-3-one (<b>11q</b>)</h5><div class="NLM_p last">Compound <b>11q</b> was prepared in 46% yield as a white oil following the same procedure as described in the general procedure A with (<i>E</i>)-4-(4-fluoro-3-methoxyphenyl)but-3-en-2-one (120 mg, 0.62 mmol) and LDA (0.70 mL, 1.2 equiv) under stirring for 2 h. The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 10:1 to 3:1, v/v). <i>R</i><sub><i>f</i></sub> 0.48 (hexane/EtOAc = 2:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.51 (d, <i>J</i> = 16.1 Hz, 1H), 7.14 (d, <i>J</i> = 8.0 Hz, 1H), 7.12–7.08 (m, 2H), 6.64 (d, <i>J</i> = 16.1 Hz, 1H), 4.17–4.14 (m, 1H), 3.93 (s, 3H), 3.15 (brs, 1H), 2.88–2.85 (m, 1H), 2.78–2.74 (m, 1H), 1.59–1.40 (m, 4H), 0.96 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 200.74, 154.87, 153.19, 148.18, 148.10, 142.57, 130.89, 130.87, 126.18, 126.16, 122.10, 122.06, 116.67, 116.55, 112.46, 67.62, 56.27, 46.95, 38.71, 18.75, 14.03. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>19</sub>FO<sub>3</sub> [M – H]<sup>−</sup>, 265.1245; found, 265.1496. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 15.14 min, method G, <i>t</i><sub>R</sub> = 19.00 min).</div></div><div id="sec4_2_1_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31"> (<i>E</i>)-1-(3-Fluoro-5-methoxyphenyl)-5-hydroxyoct-1-en-3-one (<b>11r</b>)</h5><div class="NLM_p last">Compound <b>11r</b> was prepared in 24% yield as a yellow oil following the same procedure as described in the general procedure A with (<i>E</i>)-4-(3-fluoro-5-methoxyphenyl)but-3-en-2-one (130 mg, 0.67 mmol) and LDA (0.88 mL, 1.2 equiv) under stirring for 6 h. The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 10:1 to 6:1, v/v). <i>R</i><sub><i>f</i></sub> 0.40 (hexane/EtOAc = 2:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.46 (d, <i>J</i> = 16.2 Hz, 1H), 6.86–6.85 (m, 2H), 6.70–6.65 (m, 2H), 4.17–4.14 (m, 1H), 3.83 (s, 3H), 3.13 (brs, 1H), 2.87–2.84 (m, 1H), 2.79–2.74 (m, 1H), 1.59–1.40 (m, 4H), 0.95 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 200.68, 164.57, 162.93, 161.26, 161.18, 142.14, 142.12, 136.79, 136.72, 127.55, 110.12, 110.11, 107.09, 106.94, 103.86, 103.70, 67.56, 55.69, 47.21, 38.70, 18.74, 14.01. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>19</sub>FO<sub>3</sub> [M – H]<sup>−</sup>, 265.1245; found, 265.1262. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 19.07 min, method H, <i>t</i><sub>R</sub> = 11.43 min).</div></div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> General Procedure B for Compounds <b>12a–12r</b></h4><div class="NLM_p">To a stirred solution of <b>11a–11r</b> in MeOH (8 mL) was added 10% Pd/C (0.03 equiv). The solution was then stirred in an atmosphere of H<sub>2</sub> gas for 2 h. The reaction mixture was filtered through a celite pad and concentrated under reduced pressure. The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 4:1, v/v) to furnish compounds <b>12a–12r</b>.</div><div id="sec4_2_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> 5-Hydroxy-1-(4-hydroxy-3-methoxyphenyl)octan-3-one (<b>12a</b>)</h5><div class="NLM_p last">Compound <b>12a</b> was prepared in 45% yield as a colorless oil by following the same procedure as described in the general procedure B with <b>11a</b> (109 mg, 0.412 mmol). <i>R</i><sub><i>f</i></sub> 0.38 (hexane/EtOAc = 1:1, v/v). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 6.82 (d, <i>J</i> = 8.2 Hz, 1H), 6.73–6.60 (m, 2H), 5.71 (s, 1H), 4.04 (brs, 1H), 3.86 (s, 3H), 3.06 (brs, 1H), 2.90–2.65 (m, 4H), 2.62–2.41 (m, 2H), 1.56–1.21 (m, 4H), 0.91 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 211.53, 146.45, 143.93, 132.63, 120.74, 114.37, 111.02, 67.38, 67.27, 55.88, 49.34, 45.44, 38.54, 29.28, 18.65, 13.98. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>22</sub>O<sub>3</sub> [M – H]<sup>−</sup>, 265.1445; found, 265.1455. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 6.09 min, method G, <i>t</i><sub>R</sub> = 5.49 min).</div></div><div id="sec4_2_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> 5-Hydroxy-1-phenyloctan-3-one (<b>12b</b>)</h5><div class="NLM_p last">Compound <b>12b</b> was prepared in 52% yield as a colorless oil by following the same procedure as described in the general procedure B with <b>11b</b> (50 mg, 0.229 mmol). <i>R</i><sub><i>f</i></sub> 0.65 (hexane/EtOAc = 1:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.29–7.26 (m, 2H), 7.20–7.17 (m, 3H), 4.05–4.03 (m, 1H), 2.94 (s, 1H), 2.90 (t, <i>J</i> = 7.6 Hz, 2H), 2.76 (t, <i>J</i> = 7.6 Hz, 2H), 2.57–2.47 (m, 2H), 1.49–1.25 (m, 4H), 0.91 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 211.24, 140.75, 128.56, 128.29, 126.22, 67.36, 49.32, 45.06, 38.59, 29.53, 18.65, 13.98. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>20</sub>O<sub>2</sub> [M – H]<sup>−</sup>, 219.1390; found, 219.1414. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 14.38 min, method G, <i>t</i><sub>R</sub> = 16.36 min).</div></div><div id="sec4_2_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> 5-Hydroxy-1-(3-methoxyphenyl)octan-3-one (<b>12c</b>)</h5><div class="NLM_p last">Compound <b>12c</b> was prepared in 72% yield as a colorless oil by following the same procedure as described in the general procedure B with <b>11c</b> (50 mg, 0.201 mmol). <i>R</i><sub><i>f</i></sub> 0.51 (hexane/EtOAc = 1:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.20 (t, <i>J</i> = 7.9 Hz, 1H), 6.77–6.72 (m, 3H), 4.05–4.03 (m, 1H), 3.79 (s, 3H), 2.90 (d, <i>J</i> = 3.4 Hz, 1H), 2.88 (t, <i>J</i> = 7.6 Hz, 2H), 2.76 (t, <i>J</i> = 7.6 Hz, 2H), 2.58–2.55 (m, 1H), 2.52–2.47 (m, 1H), 1.50–1.31 (m, 4H), 0.92 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 211.15, 159.75, 142.38, 129.54, 120.61, 114.13, 111.46, 67.36, 55.16, 49.33, 44.96, 38.61, 29.55, 18.65, 13.98. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>22</sub>O<sub>3</sub> [M – H]<sup>−</sup>, 250.1496; found, 249.1518. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 13.80 min, method G, <i>t</i><sub>R</sub> = 15.38 min).</div></div><div id="sec4_2_2_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> 5-Hydroxy-1-(4-hydroxyphenyl)octan-3-one (<b>12d</b>)</h5><div class="NLM_p last">Compound <b>12d</b> was prepared in 72% yield as a colorless oil by following the same procedure as described in the general procedure B with <b>11d</b> (88 mg, 0.376 mmol). <i>R</i><sub><i>f</i></sub> 0.45 (hexane/EtOAc = 1:1, v/v). <sup>1</sup>H NMR (600 MHz, MeOD): δ 7.02 (d, <i>J</i> = 8.5 Hz, 1H), 6.70 (d, <i>J</i> = 8.5 Hz, 1H), 4.04–4.01 (m, 1H), 2.77 (s, 4H), 2.58–2.54 (m, 1H), 2.52–2.48 (m, 1H), 1.46–1.33 (m, 4H), 0.93 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, MeOD): δ 210.62, 155.21, 131.87, 128.88, 114.78, 67.24, 49.89, 45.03, 39.19, 28.39, 18.37, 12.94. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>20</sub>O<sub>3</sub> [M – H]<sup>−</sup>, 235.1339; found, 235.1360. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 6.02 min, method G, <i>t</i><sub>R</sub> = 6.14 min).</div></div><div id="sec4_2_2_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> 5-Hydroxy-1-(3-hydroxyphenyl)octan-3-one (<b>12e</b>)</h5><div class="NLM_p last">Compound <b>12e</b> was prepared in 68% yield as a colorless oil by following the same procedure as described in the general procedure B with <b>11e</b> (50 mg, 0.213 mmol). <i>R</i><sub><i>f</i></sub> 0.35 (hexane/EtOAc = 1:1, v/v). <sup>1</sup>H NMR (600 MHz, MeOD): δ 7.07 (t, <i>J</i> = 7.8 Hz, 1H), 6.67 (d, <i>J</i> = 7.6 Hz, 1H), 6.64 (s, 1H), 6.61–6.60 (m, 1H), 4.05–4.02 (m, 1H), 2.80 (s, 4H), 2.59–2.55 (m, 1H), 2.53–2.50 (m, 1H), 1.48–1.31 (m, 4H), 0.94 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, MeOD): δ 210.35, 157.09, 142.67, 129.03, 119.16, 114.81, 112.55, 67.26, 49.87, 44.58, 39.20, 29.14, 18.37, 12.93. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>20</sub>O<sub>3</sub> [M – H]<sup>−</sup>, 235.1339; found, 235.1360. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 5.57 min, method G, <i>t</i><sub>R</sub> = 5.23 min).</div></div><div id="sec4_2_2_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> 5-Hydroxy-1-(4-methoxyphenyl)octan-3-one (<b>12f</b>)</h5><div class="NLM_p last">Compound <b>12f</b> was prepared in 80% yield as a colorless oil by following the same procedure as described in the general procedure B with <b>11f</b> (40 mg, 0.161 mmol). <i>R</i><sub><i>f</i></sub> 0.40 (hexane/EtOAc = 1:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.09 (d, <i>J</i> = 8.5 Hz, 2H), 6.82 (d, <i>J</i> = 8.4 Hz, 2H), 4.03 (brs, 1H), 3.78 (s, 3H), 2.93 (s, 1H), 2.84 (t, <i>J</i> = 7.5 Hz, 2H), 2.73 (t, <i>J</i> = 7.6 Hz, 2H), 2.57–2.54 (m, 1H), 2.50–2.46 (m, 1H), 1.49–1.29 (m, 4H), 0.91 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 211.46, 158.03, 132.75, 129.40, 129.22, 113.96, 67.35, 55.27, 49.32, 45.34, 38.57, 28.70, 18.65, 13.98. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>22</sub>O<sub>3</sub> [M – H]<sup>−</sup>, 249.1496; found, 249.1527. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 13.20 min, method G, <i>t</i><sub>R</sub> = 14.78 min).</div></div><div id="sec4_2_2_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> 1-(4-Fluorophenyl)-5-hydroxyoctan-3-one (<b>12g</b>)</h5><div class="NLM_p last">Compound <b>12g</b> was prepared in 65% yield as a colorless oil by following the same procedure as described in the general procedure B with <b>11g</b> (84 mg, 0.356 mmol). <i>R</i><sub><i>f</i></sub> 0.41 (hexane/EtOAc = 1:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.14–7.12 (m, 3H), 6.96 (t, <i>J</i> = 8.7 Hz, 2H), 5.56 (s, 1H), 4.04–4.03 (brs, 1H), 4.05 (brs, 1H), 2.90 (s, 1H), 2.87 (t, <i>J</i> = 7.6 Hz, 2H), 2.74 (t, <i>J</i> = 7.5 Hz, 2H), 2.57–2.54 (m, 1H), 2.51–2.47 (m, 1H), 1.49–1.32 (m, 4H), 0.92 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 210.91, 162.22, 160.60, 136.41, 136.40, 129.73, 129.68, 115.35, 115.20, 67.36, 49.38, 45.09, 38.63, 28.64, 18.64, 13.96. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>19</sub>FO<sub>2</sub> [M – H]<sup>−</sup>, 237.1296; found, 237.1309. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 12.21 min, method G, <i>t</i><sub>R</sub> = 18.56 min).</div></div><div id="sec4_2_2_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> 1-(4-Chlorophenyl)-5-hydroxyoctan-3-one (<b>12h</b>)</h5><div class="NLM_p last">Compound <b>12h</b> was prepared in 40% yield as a colorless oil by following the same procedure as described in the general procedure B with <b>11h</b> (48 mg, 0.190 mmol). <i>R</i><sub><i>f</i></sub> 0.45 (hexane/EtOAc = 1:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.30–7.26 (m, 2H), 7.21–7.17 (m, 2H), 4.05–4.03 (m, 1H), 2.91 (t, <i>J</i> = 7.6 Hz, 2H), 2.76 (t, <i>J</i> = 7.6 Hz, 2H), 2.58–2.55 (m, 1H), 2.51–2.47 (m, 1H), 1.51–1.31 (m, 4H), 0.92 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 211.24, 140.75, 128.56, 128.29, 126.23, 67.36, 49.31, 45.07, 38.59, 29.53, 18.65, 13.98. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>19</sub>ClO<sub>2</sub> [M – H]<sup>−</sup>, 253.1001; found, 253.1430. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 11.95 min, method G, <i>t</i><sub>R</sub> = 16.89 min).</div></div><div id="sec4_2_2_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> 5-Hydroxy-1-(<i>p</i>-tolyl)octan-3-one (<b>12i</b>)</h5><div class="NLM_p last">Compound <b>12i</b> was prepared in 70% yield as a yellow oil by following the same procedure as described in the general procedure B with <b>11i</b> (107 mg, 0.461 mmol). <i>R</i><sub><i>f</i></sub> 0.48 (hexane/EtOAc = 1:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.07 (dd, <i>J</i> = 8.0 and 18.2 Hz, 4H), 4.05–4.02 (m, 1H), 2.92 (d, <i>J</i> = 3.5 Hz, 1H), 2.86 (t, <i>J</i> = 7.6 Hz, 2H), 2.74 (t, <i>J</i> = 7.6 Hz, 2H), 2.57–2.54 (m, 1H), 2.50–2.46 (m, 1H), 2.31 (s, 3H), 1.49–1.31 (m, 4H), 0.91 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 211.32, 137.66, 135.69, 129.23, 128.17, 67.36, 49.36, 45.21, 38.65, 29.13, 21.00, 18.67, 14.00. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>22</sub>O<sub>2</sub> [M – H]<sup>−</sup>, 233.1547; found, 233.1575. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 21.76 min, method G, <i>t</i><sub>R</sub> = 32.04 min).</div></div><div id="sec4_2_2_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> 1-(4-(Dimethylamino)phenyl)-5-hydroxyoctan-3-one (<b>12j</b>)</h5><div class="NLM_p last">Compound <b>12j</b> was prepared in 63% yield as a colorless oil by following the same procedure as described in the general procedure B with <b>11j</b> (80 mg, 0.306 mmol). <i>R</i><sub><i>f</i></sub> 0.60 (hexane/EtOAc = 1:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.04 (d, <i>J</i> = 8.1 Hz, 2H), 6.68 (d, <i>J</i> = 8.1 Hz, 2H), 4.02 (brs, 1H), 2.98 (s, 1H), 2.90 (s, 6H), 2.80 (t, <i>J</i> = 7.3 Hz, 2H), 2.70 (t, <i>J</i> = 7.5 Hz, 2H), 2.57–2.54 (m, 1H), 2.50–2.45 (m, 1H), 1.47–1.33 (m, 4H), 0.91 (t, <i>J</i> = 6.8 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 211.83, 149.28, 128.88, 128.66, 113.04, 67.35, 49.30, 45.53, 40.83, 38.59, 28.68, 18.67, 14.00. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>16</sub>H<sub>25</sub>NO<sub>2</sub> [M – H]<sup>−</sup>, 262.1812; found, 262.1828. >95% purity (as determined by RP-HPLC, method D, <i>t</i><sub>R</sub> = 6.20 min, method G, <i>t</i><sub>R</sub> = 23.83 min).</div></div><div id="sec4_2_2_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> 1-(3,4-Dihydroxyphenyl)-5-hydroxyoctan-3-one (<b>12k</b>)</h5><div class="NLM_p last">Compound <b>12k</b> was prepared in 93% yield as a colorless oil by following the same procedure as described in the general procedure B with <b>11k</b> (36 mg, 0.144 mmol). <i>R</i><sub><i>f</i></sub> 0.55 (hexane/EtOAc = 1:3, v/v). <sup>1</sup>H NMR (600 MHz, MeOD): δ 6.67 (d, <i>J</i> = 8.2 Hz, 1H), 6.64 (d, <i>J</i> = 1.8 Hz, 1H), 6.52 (dd, <i>J</i> = 1.8 and 8.2 Hz, 1H), 4.07–3.98 (m, 1H), 2.80–2.69 (m, 4H), 2.57 (q, <i>J</i> = 8.2 Hz, 1H), 2.50 (dd, <i>J</i> = 4.4 and 15.4 Hz, 1H), 1.53–1.28 (m, 4H), 0.93 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 210.85, 144.78, 143.07, 132.68, 119.13, 115.09, 114.95, 67.25, 49.93, 45.01, 39.18, 28.65, 18.36, 12.91. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>16</sub>H<sub>25</sub>NO<sub>2</sub> [M – H]<sup>−</sup>, 251.1289; found, 251.1387. >95% purity (as determined by RP-HPLC, method A, <i>t</i><sub>R</sub> = 4.86 min, method G, <i>t</i><sub>R</sub> = 3.93 min).</div></div><div id="sec4_2_2_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> 1-(3,4-Dimethoxyphenyl)-5-hydroxyoctan-3-one (<b>12l</b>)</h5><div class="NLM_p last">Compound <b>12l</b> was prepared in 63% yield as a colorless oil by following the same procedure as described in the general procedure B with <b>11l</b> (59 mg, 0.212 mmol). <i>R</i><sub><i>f</i></sub> 0.33 (hexane/EtOAc = 1:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 6.78 (d, <i>J</i> = 8.4 Hz, 1H), 6.71–6.70 (m, 2H), 4.04 (brs, 1H), 3.87 (s, 3H), 3.85 (s, 3H), 2.92 (s, 1H), 2.85 (t, <i>J</i> = 7.5 Hz, 2H), 2.74 (t, <i>J</i> = 7.5 Hz, 2H), 2.58–2.55 (m, 1H), 2.52–2.47 (m, 1H), 1.48–1.32 (m, 4H), 0.92 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 211.40, 148.93, 147.45, 133.35, 120.07, 111.67, 111.34, 67.38, 55.93, 55.85, 49.37, 45.34, 38.60, 29.20, 18.65, 13.98. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>16</sub>H<sub>24</sub>O<sub>4</sub> [M – H]<sup>−</sup>, 279.1602; found, 279.1620. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 8.52 min, method G, <i>t</i><sub>R</sub> = 8.61 min).</div></div><div id="sec4_2_2_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> 1-(3-Fluoro-4-hydroxyphenyl)-5-hydroxyoctan-3-one (<b>12m</b>)</h5><div class="NLM_p last">Compound <b>12m</b> was prepared in 63% yield as a colorless oil by following the same procedure as described in the general procedure B with <b>11m</b> (55 mg, 0.218 mmol). <i>R</i><sub><i>f</i></sub> 0.66 (hexane/EtOAc = 1:1, v/v). <sup>1</sup>H NMR (600 MHz, MeOD): δ 6.91 (d, <i>J</i> = 12.2 Hz, 1H), 6.82–6.81 (m, 2H), 4.04 (brs, 1H), 2.79 (s, 4H), 2.59–2.55 (m, 1H), 2.53–2.49 (m, 1H), 1.46–1.31 (m, 4H), 0.94 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, MeOD): δ 210.17, 152.10, 150.51, 142.74, 142.66, 133.11, 133.07, 123.85, 123.83, 117.20, 117.19, 115.41, 115.29, 67.24, 49.85, 44.62, 39.21, 28.11, 18.36, 12.91. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>19</sub>FO<sub>3</sub> [M – H]<sup>−</sup>, 253.1245; found, 253.1275. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 6.38 min, method G, <i>t</i><sub>R</sub> = 6.04 min).</div></div><div id="sec4_2_2_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> 1-(3-Ethoxy-4-hydroxyphenyl)-5-hydroxyoctan-3-one (<b>12n</b>)</h5><div class="NLM_p last">Compound <b>12n</b> was prepared in 45% yield as a colorless oil by following the same procedure as described in the general procedure B with <b>11n</b> (60 mg, 0.216 mmol). <i>R</i><sub><i>f</i></sub> 0.35 (hexane/EtOAc = 1:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 6.82 (d, <i>J</i> = 8.0 Hz, 1H), 6.67 (s, 1H), 6.65 (d, <i>J</i> = 8.0 Hz, 1H), 5.56 (s, 1H), 4.10–4.07 (q, <i>J</i> = 7.0 Hz, 1H), 4.05 (brs, 1H), 2.92 (s, 1H), 2.82 (t, <i>J</i> = 7.5 Hz, 2H), 2.72 (t, <i>J</i> = 7.5 Hz, 2H), 2.57–2.54 (m, 1H), 2.50–2.46 (m, 1H), 1.48–1.31 (m, 7H), 1.25 (s, 1H), 0.91 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 211.49, 145.73, 144.12, 132.56, 120.65, 114.34, 111.93, 67.38, 64.44, 49.37, 45.46, 38.59, 29.29, 18.65, 14.91, 13.97. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>16</sub>H<sub>24</sub>O<sub>4</sub> [M – H]<sup>−</sup>, 279.1602; found, 279.1646. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 8.06 min, method G, <i>t</i><sub>R</sub> = 7.73 min).</div></div><div id="sec4_2_2_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> 1-(3,4-Difluorophenyl)-5-hydroxyoctan-3-one (<b>12o</b>)</h5><div class="NLM_p last">Compound <b>12o</b> was prepared in 81% yield as a colorless oil by following the same procedure as described in the general procedure B with <b>11o</b> (49 mg, 0.193 mmol). <i>R</i><sub><i>f</i></sub> 0.40 (hexane/EtOAc = 1:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.05 (q, <i>J</i> = 9.0 Hz, 1H), 6.98 (t, <i>J</i> = 9.5 Hz, 1H), 6.89–6.87 (m, 1H), 4.07–4.04 (m, 1H), 2.86 (t, <i>J</i> = 7.4 Hz, 2H), 2.80 (s, 1H), 2.74 (t, <i>J</i> = 7.4 Hz, 2H), 2.58–2.54 (m, 1H), 2.52–2.48 (m, 1H), 1.49–1.33 (m, 4H), 0.92 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 210.35, 151.03, 150.95, 149.77, 149.69, 149.39, 149.31, 148.14, 148.05, 137.78, 137.75, 137.74, 137.72, 124.21, 124.18, 124.16, 124.14, 117.21, 117.19, 117.10, 117.08, 67.38, 49.38, 44.66, 38.66, 28.51, 18.64, 13.94. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>18</sub>F<sub>2</sub>O<sub>2</sub> [M – H]<sup>−</sup>, 255.1202; found, 255.1235. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 18.49 min, method G, <i>t</i><sub>R</sub> = 21.95 min).</div></div><div id="sec4_2_2_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> 1-(3,4-Dimethylphenyl)-5-hydroxyoctan-3-one (<b>12p</b>)</h5><div class="NLM_p last">Compound <b>12p</b> was prepared in 72% yield as a colorless oil by following the same procedure as described in the general procedure B with <b>11p</b> (83 mg, 0.337 mmol). <i>R</i><sub><i>f</i></sub> 0.60 (hexane/EtOAc = 1:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.04 (d, <i>J</i> = 7.6 Hz, 1H), 6.94 (s, 1H), 6.90 (d, <i>J</i> = 7.6 Hz, 1H), 4.05–4.01 (m, 1H), 2.94 (d, <i>J</i> = 3.5 Hz, 1H), 2.83 (t, <i>J</i> = 7.6 Hz, 2H), 2.73 (t, <i>J</i> = 7.6 Hz, 2H), 2.58–2.55 (m, 1H), 2.51–2.46 (m, 1H), 2.23 (s, 3H), 2.22 (s, 3H), 1.49–1.31 (m, 4H), 0.91 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 211.43, 138.14, 136.67, 134.35, 129.80, 129.64, 125.57, 67.37, 49.32, 45.29, 38.63, 29.12, 19.75, 19.31, 18.67, 14.00. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>16</sub>H<sub>24</sub>O<sub>2</sub> [M – H]<sup>−</sup>, 247.1703; found, 247.1725. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 31.01 min, method G, <i>t</i><sub>R</sub> = 21.80 min).</div></div><div id="sec4_2_2_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> 1-(4-Fluoro-3-methoxyphenyl)-5-hydroxyoctan-3-one (<b>12q</b>)</h5><div class="NLM_p last">Compound <b>12q</b> was prepared in 62% yield as a colorless oil by following the same procedure as described in the general procedure B with <b>11q</b> (48 mg, 0.180 mmol). <i>R</i><sub><i>f</i></sub> 0.50 (hexane/EtOAc = 1:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 6.97 (dd, <i>J</i> = 8.2 Hz, and 11.3 Hz, 1H), 6.78 (dd, <i>J</i> = 8.1 and 1.9 Hz, 1H), 6.69–6.67 (m, 1H), 4.06–4.03 (m, 1H), 3.87 (s, 3H), 2.88–2.84 (m, 3H), 2.75 (t, <i>J</i> = 7.4 Hz, 2H), 2.58–2.55 (m, 1H), 2.52–2.48 (m, 1H), 1.50–1.31 (m, 4H), 0.92 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 210.91, 151.86, 150.24, 147.46, 147.39, 137.10, 137.08, 120.30, 120.25, 115.95, 115.83, 113.67, 67.38, 56.23, 49.40, 45.10, 38.63, 29.15, 18.64, 13.96. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>21</sub>FO<sub>3</sub> [M – H]<sup>−</sup>, 267.1402; found, 267.1426. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 14.58 min, method G, <i>t</i><sub>R</sub> = 14.25 min).</div></div><div id="sec4_2_2_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> 1-(3-Fluoro-5-methoxyphenyl)-5-hydroxyoctan-3-one (<b>12r</b>)</h5><div class="NLM_p last">Compound <b>12r</b> was prepared in 40% yield as a colorless oil by following the same procedure as described in the general procedure B with <b>11r</b> (43 mg, 0.162 mmol). <i>R</i><sub><i>f</i></sub> 0.45 (hexane/EtOAc = 1:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 6.51 (s, 1H), 6.49–6.44 (m, 2H), 4.06–4.04 (m, 1H), 3.77 (s, 3H), 2.86–2.84 (m, 3H), 2.75 (t, <i>J</i> = 7.6 Hz, 2H), 2.59–2.55 (m, 1H), 2.52–2.48 (m, 1H), 1.50–1.32 (m, 4H), 0.92 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 210.64, 164.48, 162.85, 160.97, 160.90, 143.85, 143.79, 110.03, 110.01, 107.45, 107.31, 99.37, 99.20, 67.37, 55.47, 49.34, 44.53, 38.62, 29.38, 29.36, 18.65, 13.96. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>21</sub>FO<sub>3</sub> [M – H]<sup>−</sup>, 267.1402; found, 267.1419. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 17.95 min, method G, <i>t</i><sub>R</sub> = 22.68 min).</div></div><div id="sec4_2_2_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> (<i>E</i>)-1-(3,4-Difluorophenyl)oct-1-en-3-one (<b>13a</b>)</h5><div class="NLM_p last">To a solution of 3,4-difluorobenzaldehyde (500 mg, 3.52 mmol) in MeOH (8 mL) were added <span class="smallcaps smallerCapital">l</span>-proline (61.0 mg, 0.53 mmol) and heptan-2-one (0.49 mL, 3.52 mmol) at 25 °C under argon. After 30 min, trimethylamine (0.12 mL, 0.88 mmol) was introduced. The reaction mixture was stirred at 25 °C for 72 h, quenched with water, and then extracted with EtOAc (3 × 25 mL). The combined organic layer was dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 12:1 to 6:1, v/v) to furnish compound <b>13a</b> in 50% yield as a colorless oil. <i>R</i><sub><i>f</i></sub> 0.75 (hexane/EtOAc = 4:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.45 (d, <i>J</i> = 16.1 Hz, 1H), 7.40–7.35 (m, 1H), 7.30–7.26 (m, 1H), 7.21–7.15 (m, 1H), 6.65 (d, <i>J</i> = 16.1 Hz, 1H), 2.64 (t, <i>J</i> = 7.4 Hz, 2H), 1.72–1.63 (m, 2H), 1.40–1.29 (m, 4H), 0.91 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 200.05, 152.38, 152.29, 151.48, 151.40, 150.69, 150.61, 149.83, 149.74, 139.76, 131.93, 131.90, 131.86, 126.95, 126.93, 125.05, 125.02, 125.00, 124.98, 117.90, 117.78, 116.41, 116.30, 41.27, 31.46, 23.91, 22.48, 13.92. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>16</sub>F<sub>2</sub>O [M – H]<sup>−</sup>, 237.1096; found, 237.1126. >95% purity (as determined by RP-HPLC, method E, <i>t</i><sub>R</sub> = 16.72 min, method I, <i>t</i><sub>R</sub> = 11.93 min).</div></div><div id="sec4_2_2_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> (<i>E</i>)-1-(4-Hydroxy-3-methoxyphenyl)oct-1-en-3-one (<b>13b</b>)</h5><div class="NLM_p last">Compound <b>13b</b> was prepared in 30% yield as a yellow oil by following the same procedure as described for the synthesis of <b>13a</b> but with vanillin (500 mg, 3.29 mmol) instead of 3,4-difluorobenzaldehyde. <i>R</i><sub><i>f</i></sub> 0.44 (hexane/EtOAc = 5:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.48 (d, <i>J</i> = 16.0 Hz, 1H), 7.10 (dd, <i>J</i> = 1.6 and 8.2 Hz, 1H), 7.05 (d, <i>J</i> = 1.7 Hz, 1H), 6.93 (d, <i>J</i> = 8.2 Hz, 1H), 6.60 (d, <i>J</i> = 15.6 Hz, 1H), 6.00 (brs, 1H), 3.93 (s, 3H), 2.64 (t, <i>J</i> = 7.2 Hz, 2H), 1.81–1.69 (m, 2H), 1.49–1.21 (m, 4H), 0.87 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 200.93, 148.23, 146.94, 142.74, 127.05, 124.06, 123.38, 114.88, 109.52, 55.95, 40.64, 31.56, 24.26, 22.52, 13.97. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>20</sub>O<sub>3</sub> [M – H]<sup>−</sup>, 247.1339; found, 247.1406. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 18.10 min, method H, <i>t</i><sub>R</sub> = 13.32 min).</div></div><div id="sec4_2_2_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> 1-(3,4-Difluorophenyl)octan-3-one (<b>14a</b>)</h5><div class="NLM_p last">Compound <b>14a</b> was prepared in 76% yield as a colorless oil by following the same procedure as described in the general procedure B with <b>13a</b> (30 mg, 0.126 mmol). <i>R</i><sub><i>f</i></sub> 0.77 (hexane/EtOAc = 4:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.10–7.01 (m, 1H), 7.01–6.94 (m, 1H), 6.92–6.82 (m, 1H), 2.85 (t, <i>J</i> = 7.4 Hz, 2H), 2.70 (t, <i>J</i> = 7.4 Hz, 2H), 2.37 (t, <i>J</i> = 7.4 Hz, 2H), 1.59–1.51 (m, 2H), 1.34–1.19 (m, 4H), 0.88 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 209.73, 151.00, 150.92, 149.70, 149.62, 149.36, 149.28, 148.07, 147.99, 138.20, 138.17, 138.14, 124.24, 124.22, 124.21, 124.19, 117.21, 117.12, 117.10, 117.01, 43.80, 43.05, 31.36, 28.80, 23.47, 22.43, 13.89. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>18</sub>F<sub>2</sub>O [M – H]<sup>−</sup>, 239.1253; found, 239.1288. >95% purity (as determined by RP-HPLC, method H, <i>t</i><sub>R</sub> = 10.08 min, method I, <i>t</i><sub>R</sub> = 11.68 min).</div></div><div id="sec4_2_2_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> 1-(4-Hydroxy-3-methoxyphenyl)octan-3-one (<b>14b</b>)</h5><div class="NLM_p last">Compound <b>14b</b> was prepared in 75% yield as a colorless oil by following the same procedure as described in the general procedure B with <b>13b</b> (21 mg, 0.085 mmol). <i>R</i><sub><i>f</i></sub> 0.51 (hexane/EtOAc = 5:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 6.81 (d, <i>J</i> = 7.8 Hz, 1H), 6.69 (s, 1H), 6.67 (d, <i>J</i> = 7.8 Hz, 1H), 5.47 (brs, 1H), 3.87 (s, 3H), 2.82 (t, <i>J</i> = 7.8 Hz, 2H), 2.69 (t, <i>J</i> = 7.8 Hz, 2H), 2.37 (t, <i>J</i> = 7.8 Hz, 2H), 1.68–1.51 (m, 2H), 1.38–1.21 (m, 6H), 0.87 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 210.65, 146.38, 143.87, 133.15, 120.78, 114.31, 111.06, 55.88, 44.63, 43.12, 31.40, 30.33, 29.71, 29.56, 23.50, 22.46, 13.92. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>22</sub>O<sub>3</sub> [M – H]<sup>−</sup>, 249.1496; found, 249.1531. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 17.50 min, method H, <i>t</i><sub>R</sub> = 9.64 min).</div></div><div id="sec4_2_2_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> (<i>E</i>)-1-(3,4-Difluorophenyl)oct-1-en-3-ol (<b>15a</b>)</h5><div class="NLM_p last">To a stirred solution of <b>13a</b> (55.0 mg, 0.231 mmol) in MeOH (8 mL) was added NaBH<sub>4</sub> (12.2 mg, 0.315 mmol) at 0 °C. The reaction mixture was stirred for 1 h at the room temperature. The reaction mixture was concentrated and then extracted with EtOAc (3 × 25 mL). The organic layer was washed with water, dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 8:1, v/v) to furnish compound <b>15a</b> in 94% yield as a colorless oil. <i>R</i><sub><i>f</i></sub> 0.51 (hexane/EtOAc = 4:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.23–7.16 (m, 1H), 7.14–7.06 (m, 2H), 6.50 (d, <i>J</i> = 15.9 Hz, 1H), 6.16 (dd, <i>J</i> = 6.5 and 13.1 Hz, 1H), 4.29 (q, <i>J</i> = 6.5 Hz, 1H), 1.78 (brs, 1H), 1.71–1.55 (m, 2H), 1.51–1.25 (m, 6H), 0.92 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 151.25, 150.65, 150.57, 149.70, 149.61, 149.00, 148.92, 134.12, 134.08, 134.05, 133.76, 133.75, 128.02, 122.66, 122.64, 122.62, 122.60, 117.34, 117.23, 114.79, 114.68, 72.73, 37.35, 31.76, 25.10, 22.60, 14.03. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>18</sub>F<sub>2</sub>O [M – H]<sup>−</sup>, 239.1253; found, 239.1290. >95% purity (as determined by RP-HPLC, method E, <i>t</i><sub>R</sub> = 10.05 min, method I, <i>t</i><sub>R</sub> = 9.67 min).</div></div><div id="sec4_2_2_24" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> (<i>E</i>)-4-(3-Hydroxyoct-1-en-1-yl)-2-methoxyphenol (<b>15b</b>)</h5><div class="NLM_p last">Compound <b>15b</b> was prepared in 90% yield as a brown oil by following the same procedure as described for the synthesis of <b>15a</b> but with <b>13b</b> (31 mg, 0.125 mmol) instead of <b>13a</b>. <i>R</i><sub><i>f</i></sub> 0.36 (hexane/EtOAc = 2:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 6.94 (d, <i>J</i> = 1.6 Hz, 1H), 6.92–6.88 (m, 2H), 6.50 (d, <i>J</i> = 15.8 Hz, 1H), 6.08 (q, <i>J</i> = 7.1 Hz, 1H), 5.67 (s, 1H), 4.27 (dd, <i>J</i> = 6.5 and 13.3 Hz, 1H), 3.93 (s, 3H), 1.72–1.58 (m, 4H), 1.37–1.30 (m, 4H), 0.92 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 146.63, 145.52, 130.33, 130.31, 129.33, 120.29, 114.4, 108.27, 73.33, 55.90, 37.42, 31.81, 25.21, 22.63, 14.07. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>22</sub>O<sub>3</sub> [M – H]<sup>−</sup>, 249.1496; found, 249.1525. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 11.86 min, method H, <i>t</i><sub>R</sub> = 9.02 min).</div></div><div id="sec4_2_2_25" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57"> 1-(3,4-Difluorophenyl)octan-3-ol (<b>16a</b>)</h5><div class="NLM_p last">Compound <b>16a</b> was prepared in 60% yield as a colorless oil by following the same procedure as described in the general procedure B with <b>15a</b> (30 mg, 0.125 mmol). <i>R</i><sub><i>f</i></sub> 0.58 (hexane/EtOAc = 4:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.09–7.02 (m, 1H), 7.02–6.96 (m, 1H), 6.93–6.86 (m, 1H), 3.64–3.55 (m, 1H), 2.81–2.72 (m, 1H), 2.67–2.59 (m, 1H), 1.80–1.64 (m, 2H), 1.53–1.23 (m, 9H), 0.89 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 151.02, 150.94, 149.54, 149.45, 149.38, 149.30, 147.91, 147.83, 139.23, 139.20, 139.17, 124.18, 124.16, 124.14, 124.12, 117.14, 117.03, 117.01, 116.90, 71.09, 38.80, 37.68, 31.84, 31.24, 25.28, 22.62, 14.02. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>20</sub>F<sub>2</sub>O [M – H]<sup>−</sup>, 241.1409; found, 241.1452. >95% purity (as determined by RP-HPLC, method E, <i>t</i><sub>R</sub> = 12.84 min, method I, <i>t</i><sub>R</sub> = 14.86 min).</div></div><div id="sec4_2_2_26" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> 4-((<i>tert</i>-Butyldimethylsilyl)oxy)-3-methoxybenzaldehyde (<b>17</b>)</h5><div class="NLM_p last">To a stirred solution of vanillin (1.00 g, 6.57 mmol) dissolved in dry CH<sub>2</sub>Cl<sub>2</sub> (50 mL) were added imidazole (1.29 g, 18.96 mmol) and TBDMS chloride (1.42 g, 9.39 mmol) at 0 °C. The reaction mixture was stirred under argon for 16 h at room temperature, quenched with water, and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 25 mL). The organic layer was washed with water, dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 4:1, v/v) to furnish compound <b>17</b> in 94% yield as a colorless oil. <i>R</i><sub><i>f</i></sub> 0.89 (hexane/EtOAc = 1:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.85 (s, 1H), 7.40 (d, <i>J</i> = 1.9 Hz, 1H), 7.37 (dd, <i>J</i> = 4.0 and 8.0 Hz, 1H), 6.97 (d, <i>J</i> = 8.0 Hz, 1H), 3.87 (s, 3H), 1.00 (s, 9H), 0.20 (s, 6H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 191.19, 151.80, 151.51, 131.09, 126.40, 120.87, 110.25, 55.61, 25.74, 18.66, −4.41.</div></div><div id="sec4_2_2_27" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59"> <i>tert</i>-Butyl(4-(2,2-dibromovinyl)-2-methoxyphenoxy)dimethylsilane (<b>18</b>)</h5><div class="NLM_p last">To a stirred solution of CBr<sub>4</sub> (3.37 g, 10.14 mmol) dissolved in dry CH<sub>2</sub>Cl<sub>2</sub> (30 mL) was slowly added PPh<sub>3</sub> (5.32 g, 20.28 mmol) at 0 °C. After stirring under argon for 1 h at the same temperature, a solution of compound <b>17</b> (1.35 g, 5.07 mmol) in dry CH<sub>2</sub>Cl<sub>2</sub> (10 mL) was slowly added over 10 min. The reaction mixture was stirred under argon for 2 h and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 25 mL). The organic layer was washed with water, dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 15:1, v/v) to furnish compound <b>18</b> in 98% yield as a colorless oil. <i>R</i><sub><i>f</i></sub> 0.87 (hexane/EtOAc = 8:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.42 (s, 1H), 7.19 (d, <i>J</i> = 2.0 Hz, 1H), 7.03 (dd, <i>J</i> = 4.1 and 8.3 Hz, 1H), 6.84 (d, <i>J</i> = 8.2 Hz, 1H), 3.83 (s, 3H), 1.02 (s, 9H), 0.19 (s, 6H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 150.62, 145.64, 136.59, 128.77, 121.96, 120.68, 112.03, 87.10, 55.51, 25.69, 25.66, 18.47, −4.58.</div></div><div id="sec4_2_2_28" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60"> 4-(2,2-Dibromovinyl)-1,2-difluorobenzene (<b>28</b>)</h5><div class="NLM_p last">Compound <b>28</b> was prepared in 97% yield as a colorless oil by following the same procedure as described for the synthesis of <b>18</b> but with 3,4-difluorobenzaldehyde (1.35 g, 9.50 mmol) instead of <b>17</b>. The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 10:1, v/v). <i>R</i><sub><i>f</i></sub> 0.85 (hexane/EtOAc = 4:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.46 (q, <i>J</i> = 9.6 Hz, 1H), 7.39 (s, 1H), 7.23–7.20 (m, 1H), 7.15 (q, <i>J</i> = 9.1 Hz, 1H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 151.01, 150.94, 150.86, 149.35, 149.29, 149.27, 149.21, 134.73, 132.19, 132.15, 125.18, 125.15, 125.13, 125.11, 117.42, 117.30, 117.26, 117.14, 116.60, 91.07, 91.06.</div></div><div id="sec4_2_2_29" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61"> <i>tert</i>-Butyl(4-ethynyl-2-methoxyphenoxy)dimethylsilane (<b>19</b>)</h5><div class="NLM_p last">To a stirred solution of compound <b>18</b> (1.41 g, 3.34 mmol) in dry THF (20 mL) was added <i>n</i>-BuLi (1.6 M in hexanes, 5.30 mL, 8.35 mmol) at −78 °C. The reaction mixture was stirred under argon for 2 h at the same temperature, quenched with aqueous NH<sub>4</sub>Cl (10 mL), and extracted with EtOAc (3 × 25 mL). The organic layer was washed with water, dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 16:1 to 10:1, v/v) to furnish compound <b>19</b> in 96% yield as a colorless oil. <i>R</i><sub><i>f</i></sub> 0.80 (hexane/EtOAc = 8:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.00 (dd, <i>J</i> = 4.0 and 8.1 Hz, 1H), 6.97 (d, <i>J</i> = 1.9 Hz, 1H), 3.80 (s, 3H), 2.99 (s, 1H), 0.99 (s, 9H), 0.15 (s, 6H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 150.66, 146.23, 125.46, 120.90, 115.66, 115.04, 83.96, 75.57, 55.45, 25.67, 18.47, −4.64.</div></div><div id="sec4_2_2_30" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62"> 4-Ethynyl-1,2-difluorobenzene (<b>29</b>)</h5><div class="NLM_p last">To a stirred solution of compound <b>28</b> (800 mg, 2.69 mmol) in dry THF (40 mL) was added <i>n</i>-BuLi (1.6 M in hexanes, 4.30 mL, 6.71 mmol) at −78 °C. The reaction mixture was stirred under argon for 2 h at the same temperature, quenched with aqueous NH<sub>4</sub>Cl (10 mL), and extracted with hexane (3 × 25 mL). The organic layer was washed with water, dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure at low temperature (20 °C) to provide compound <b>29</b> which was used in the next step without further purification.</div></div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> General Procedure C for Compounds <b>20</b> and <b>30–32</b></h4><div class="NLM_p">To a stirred solution of appropriate ethynylbenzene (400 mg) in THF (40 mL) was added <i>n</i>-BuLi (1.6 M in hexanes, 1.1 equiv) at −78 °C. The solution was stirred under argon for 1 h at the same temperature, and then appropriate Weinreb amides (1.5 equiv) were added dropwise. The reaction mixture was stirred under argon at the same temperature until TLC analysis indicated complete conversion (typically 10–12 h), quenched with aqueous NH<sub>4</sub>Cl (10 mL), and extracted with EtOAc (3 × 25 mL). The organic layer was washed with brine, dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude residue was purified by column chromatography on a silica gel (hexane/ether = 120:1 to 80:1, v/v) to furnish compounds <b>20</b> and <b>30–32</b>.</div><div id="sec4_2_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i64" class="anchor-spacer"></div><h5 class="article-section__title" id="_i64"> 1-(4-((<i>tert</i>-Butyldimethylsilyl)oxy)-3-methoxyphenyl)oct-1-yn-3-one (<b>20</b>)</h5><div class="NLM_p last">Compound <b>20</b> was prepared in 75% yield as a colorless oil by following the same procedure as described in the general procedure C with <b>19</b> (400 mg, 1.52 mmol) and <i>N</i>-methoxy-<i>N</i>-methylhexanamide (364 mg, 2.28 mmol). <i>R</i><sub><i>f</i></sub> 0.75 (hexane/EtOAc = 8:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.12 (d, <i>J</i> = 8.1 Hz, 1H), 7.07 (s, 1H), 6.84 (d, <i>J</i> = 8.2 Hz, 1H), 3.82 (s, 3H), 2.65 (t, <i>J</i> = 7.4 Hz, 2H), 1.77–1.75 (m, 2H), 1.38–1.37 (m, 4H), 1.00 (s, 9H), 0.93 (t, <i>J</i> = 6.5 Hz, 3H), 0.18 (s, 6H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 188.36, 150.90, 148.19, 127.18, 121.14, 116.48, 112.74, 91.88, 87.48, 55.51, 45.41, 31.21, 25.62, 23.99, 22.43, 18.48, 13.92, −4.59.</div></div><div id="sec4_2_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i65" class="anchor-spacer"></div><h5 class="article-section__title" id="_i65"> 1-(3,4-Difluorophenyl)hex-1-yn-3-one (<b>30</b>)</h5><div class="NLM_p last">Compound <b>30</b> was prepared in 72% yield as a colorless oil by following the same procedure as described in the general procedure C with <b>29</b> (400 mg, 2.90 mmol) and <i>N</i>-methoxy-<i>N</i>-methylbutyramide (577 mg, 4.40 mmol). <i>R</i><sub><i>f</i></sub> 0.69 (hexane/ether = 6:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.39 (t, <i>J</i> = 7.9 Hz, 1H), 7.36–7.32 (m, 1H), 7.19 (q, <i>J</i> = 8.4 Hz, 1H), 2.64 (t, <i>J</i> = 7.4 Hz, 2H), 1.82–1.72 (m, 2H), 1.00 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 187.63, 152.92, 152.84, 151.23, 151.14, 150.93, 150.84, 149.26, 149.18, 130.06, 130.04, 130.02, 130.00, 121.97, 121.85, 118.06, 117.94, 116.94, 116.91, 116.89, 116.86, 87.83, 87.53, 47.29, 17.55, 13.48. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>12</sub>H<sub>10</sub>F<sub>2</sub>O [M – H]<sup>−</sup>, 207.0627; found, 207.0630. >95% purity (as determined by RP-HPLC, method E, <i>t</i><sub>R</sub> = 8.91 min, method I, <i>t</i><sub>R</sub> = 5.56 min).</div></div><div id="sec4_2_3_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i66" class="anchor-spacer"></div><h5 class="article-section__title" id="_i66"> 1-(3,4-Difluorophenyl)hept-1-yn-3-one (<b>31</b>)</h5><div class="NLM_p last">Compound <b>31</b> was prepared in 88% yield as a colorless oil by following the same procedure as described in the general procedure C with <b>29</b> (400 mg, 2.90 mmol) and <i>N</i>-methoxy-<i>N</i>-methylpentanamide (638 mg, 4.40 mmol). <i>R</i><sub><i>f</i></sub> 0.72 (hexane/ether = 6:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.39 (t, <i>J</i> = 7.9 Hz, 1H), 7.36–7.29 (m, 1H), 7.19 (q, <i>J</i> = 8.4 Hz, 1H), 2.66 (t, <i>J</i> = 7.4 Hz, 2H), 1.76–1.68 (m, 2H), 1.44–1.35 (m, 2H), 0.95 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 187.82, 152.96, 152.88, 151.27, 151.19, 150.97, 150.88, 149.31, 149.22, 130.07, 130.05, 130.03, 130.00, 122.03, 121.90, 118.09, 117.97, 116.88, 87.89, 87.60, 45.21, 26.12, 22.14, 13.87. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>13</sub>H<sub>12</sub>F<sub>2</sub>O [M – H]<sup>−</sup>, 221.0783; found, 221.0813. >95% purity (as determined by RP-HPLC, method E, <i>t</i><sub>R</sub> = 12.70 min, method I, <i>t</i><sub>R</sub> = 7.93 min).</div></div><div id="sec4_2_3_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i67" class="anchor-spacer"></div><h5 class="article-section__title" id="_i67"> 1-(3,4-Difluorophenyl)oct-1-yn-3-one (<b>32</b>)</h5><div class="NLM_p last">Compound <b>32</b> was prepared in 72% yield as a colorless oil by following the same procedure as described in the general procedure C with <b>29</b> (400 mg, 2.90 mmol) and <i>N</i>-methoxy-<i>N</i>-methylhexanamide (694 mg, 4.40 mmol). <i>R</i><sub><i>f</i></sub> 0.79 (hexane/EtOAc = 6:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.39 (t, <i>J</i> = 7.9 Hz, 1H), 7.36–7.29 (m, 1H), 7.19 (q, <i>J</i> = 8.4 Hz, 1H), 2.65 (t, <i>J</i> = 7.4 Hz, 2H), 1.77–1.69 (m, 2H), 1.41–1.30 (m, 4H), 0.92 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 187.87, 152.89, 151.28, 150.89, 149.23, 130.07, 130.04, 130.02, 130.00, 122.03, 121.91, 118.10, 117.97, 116.96, 87.89, 87.62, 45.46, 31.13, 23.74, 22.39, 13.89. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>14</sub>F<sub>2</sub>O [M – H]<sup>−</sup>, 235.0940; found, 235.0971. >95% purity (as determined by RP-HPLC, method E, <i>t</i><sub>R</sub> = 18.50 min, method I, <i>t</i><sub>R</sub> = 12.86 min).</div></div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> General Procedure D for Compounds <b>21</b>, <b>23a</b>, <b>23b</b>, <b>23c</b>, <b>25</b>, and <b>27</b></h4><div class="NLM_p">To a stirred solution of the silyl protected compound in THF (5 mL) was added tetrabutylammonium fluoride solution (1 M in THF, 2.0 equiv) at 0 °C. The reaction mixture was stirred under argon at the same temperature until TLC analysis indicated complete conversion (typically 1 h), quenched with aqueous NH<sub>4</sub>Cl (10 mL), and extracted with EtOAc (3 × 25 mL). The organic layer was washed with brine, dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc) to furnish compounds.</div><div id="sec4_2_4_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i69" class="anchor-spacer"></div><h5 class="article-section__title" id="_i69"> 1-(4-Hydroxy-3-methoxyphenyl)oct-1-yn-3-one (<b>21</b>)</h5><div class="NLM_p last">Compound <b>21</b> (27.0 mg, 0.075 mmol) was prepared in 94% yield as a white oil by following the same procedure as described in the general procedure D with <b>20</b>. <i>R</i><sub><i>f</i></sub> 0.21 (hexane/EtOAc = 5:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.17 (dd, <i>J</i> = 1.8 and 7.8 Hz, 1H), 7.06 (d, <i>J</i> = 1.8 Hz, 1H), 6.91 (d, <i>J</i> = 7.8 Hz, 1H), 5.99 (brs, 1H), 3.91 (s, 3H), 2.64 (t, <i>J</i> = 7.8 Hz, 2H), 1.77–1.71 (m, 2H), 1.42–1.33 (m, 4H), 0.92 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 188.40, 148.53, 146.35, 128.02, 115.18, 114.87, 111.33, 91.93, 87.29, 56.10, 45.37, 31.20, 23.98, 22.43, 13.92. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>18</sub>O<sub>3</sub> [M – H]<sup>−</sup>, 245.1183; found, 245.1261. >95% purity (as determined by RP-HPLC, method F, <i>t</i><sub>R</sub> = 5.67 min, method H, <i>t</i><sub>R</sub> = 19.33 min).</div></div><div id="sec4_2_4_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i70" class="anchor-spacer"></div><h5 class="article-section__title" id="_i70"> 4-(3-Hydroxyoct-1-yn-1-yl)-2-methoxyphenol (<b>23a</b>)</h5><div class="NLM_p last">Compound <b>23a</b> (28.0 mg, 0.077 mmol) was prepared in 94% yield as a white oil by following the same procedure as described in the general procedure D with <b>22a</b>. <i>R</i><sub><i>f</i></sub> 0.21 (hexane/EtOAc = 5:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 6.98 (dd, <i>J</i> = 1.8 and 15.6 Hz, 1H), 6.92 (d, <i>J</i> = 1.8 Hz, 1H), 6.84 (d, <i>J</i> = 8.4 Hz, 1H), 5.77 (brs, 1H), 4.58 (t, <i>J</i> = 6.6 Hz, 1H), 3.88 (s, 3H), 1.92 (brs, 1H), 1.89–1.79 (m, 2H), 1.45–1.32 (m, 4H), 0.91 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 146.28, 146.13, 125.66, 114.49, 114.23, 113.95, 88.33, 84.97, 63.09, 55.98, 37.97, 31.50, 24.95, 22.59, 14.03. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>20</sub>O<sub>3</sub> [M – H]<sup>−</sup>, 247.1339; found, 247.1397. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 12.46 min, method H, <i>t</i><sub>R</sub> = 10.92 min).</div></div><div id="sec4_2_4_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i71" class="anchor-spacer"></div><h5 class="article-section__title" id="_i71"> (<i>R</i>)-4-(3-Hydroxyoct-1-yn-1-yl)-2-methoxyphenol (<b>23b</b>)</h5><div class="NLM_p last">Compound <b>23b</b> (28.0 mg, 0.077 mmol) was prepared in 94% yield as a white oil by following the same procedure as described in the general procedure D with <b>22b</b>. <i>R</i><sub><i>f</i></sub> 0.21 (hexane/EtOAc = 5:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 6.98 (dd, <i>J</i> = 1.7 and 8.2 Hz, 1H), 6.92 (d, <i>J</i> = 1.7 Hz, 1H), 6.84 (d, <i>J</i> = 8.2 Hz, 1H), 5.72 (brs, 1H), 4.58 (q, <i>J</i> = 6.4 Hz, 1H), 3.89 (s, 3H), 1.87–1.73 (m, 3H), 1.58–1.47 (m, 2H), 1.39–1.30 (m, 4H), 0.91 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 146.29, 146.12, 125.67, 114.46, 114.23, 113.93, 88.32, 84.97, 63.10, 55.98, 37.98, 31.50, 24.94, 22.59, 14.03. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>20</sub>O<sub>3</sub> [M – H]<sup>−</sup>, 247.1339; found, 247.1397. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 7.43 min, method H, <i>t</i><sub>R</sub> = 10.95 min).</div></div><div id="sec4_2_4_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i72" class="anchor-spacer"></div><h5 class="article-section__title" id="_i72"> (<i>S</i>)-4-(3-Hydroxyoct-1-yn-1-yl)-2-methoxyphenol (<b>23c</b>)</h5><div class="NLM_p last">Compound <b>23c</b> (20.0 mg, 0.055 mmol) was prepared in 93% yield as a yellow oil by following the same procedure as described in the general procedure D with <b>22c</b>. <i>R</i><sub><i>f</i></sub> 0.21 (hexane/EtOAc = 5:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 6.97 (t, <i>J</i> = 6.6 Hz, 1H), 6.92 (d, <i>J</i> = 1.8 Hz, 1H), 6.84 (d, <i>J</i> = 7.8 Hz, 1H), 4.58 (t, <i>J</i> = 6.6 Hz, 1H), 3.88 (s, 3H), 1.89–1.76 (m, 2H), 1.63–1.55 (m, 2H), 1.43–1.31 (m, 4H), 0.90 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 146.30, 146.15, 125.65, 114.50, 114.21, 113.96, 88.34, 84.97, 63.09, 55.97, 37.97, 31.50, 24.95, 22.59, 14.03. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>20</sub>O<sub>3</sub> [M – H]<sup>−</sup>, 247.1339; found, 247.1415. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 7.70 min, method H, <i>t</i><sub>R</sub> = 10.95 min).</div></div><div id="sec4_2_4_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i73" class="anchor-spacer"></div><h5 class="article-section__title" id="_i73"> 1-(4-((<i>tert</i>-Butyldimethylsilyl)oxy)-3-methoxyphenyl)oct-1-yn-3-ol (<b>22a</b>)</h5><div class="NLM_p last">To a stirred solution of <b>20</b> (210 mg, 0.582 mmol) in MeOH 15 mL was added NaBH<sub>4</sub> (33.0 mg, 0.874 mmol) at 0 °C. The reaction mixture was stirred under argon for 1 h at the room temperature. The reaction mixture was concentrated and then extracted with EtOAc (3 × 25 mL). The organic layer was washed with water, dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 8:1, v/v) to furnish compound <b>22a</b> in 77% yield as a colorless oil. <i>R</i><sub><i>f</i></sub> 0.59 (hexane/EtOAc = 4:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 6.94–6.92 (m, 2H), 6.77 (d, <i>J</i> = 7.9 Hz, 1H), 4.58 (q, <i>J</i> = 6.4 Hz, 1H), 3.79 (s, 3H), 1.92 (d, <i>J</i> = 5.2 Hz, 1H), 1.81–1.76 (m, 2H), 1.53–1.49 (m, 2H), 1.35–1.34 (m, 4H), 0.98 (s, 9H), 0.91 (t, <i>J</i> = 7.0 Hz, 3H), 0.15 (s, 6H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 150.66, 145.84, 124.96, 120.90, 115.69, 115.31, 88.72, 85.01, 63.08, 55.44, 37.97, 31.51, 25.68, 24.95, 22.60, 18.47, 14.04, −4.65.</div></div><div id="sec4_2_4_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i74" class="anchor-spacer"></div><h5 class="article-section__title" id="_i74"> (<i>Z</i>)-4-(3-Hydroxyoct-1-en-1-yl)-2-methoxyphenol (<b>25</b>)</h5><div class="NLM_p">To a solution of <b>22a</b> (70.0 mg, 0.19 mmol) in MeOH (3 mL) were added benzoquinone (7 mg, 0.06 mmol) and Lindlar catalyst (7 mg, 0.02 mmol), and the mixture was stirred at 0 °C for 1 h under an atmosphere of hydrogen (balloon). After the complete conversion of compound <b>22a</b> (TLC, toluene/EtOAc = 10:1, v/v), the reaction mixture was filtered through a pad of celite and washed with MeOH (3 × 5 mL). The combined filtrates were concentrated under reduced pressure to give the crude product, which was purified by silica gel column chromatography (toluene/EtOAc = 10:1, v/v) to furnish compound <b>24</b> as a colorless oil. <i>R</i><sub><i>f</i></sub> 0.36 (toluene/EtOAc = 10:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 6.83 (d, <i>J</i> = 1.8 Hz, 1H), 6.81 (d, <i>J</i> = 8.4 Hz, 1H), 6.77 (dd, <i>J</i> = 1.8 and 8.1 Hz, 1H), 6.46 (d, <i>J</i> = 11.4 Hz, 1H), 5.57 (dd, <i>J</i> = 9.0 and 12.0 Hz, 1H), 4.60 (dd, <i>J</i> = 7.2 and 15.3 Hz, 1H), 3.80 (s, 3H), 1.69–1.62 (m, 1H), 1.61 (brs, 1H), 1.59–1.52 (m, 1H), 1.46–1.21 (m, 6H), 1.00 (s, 9H), 0.88 (t, <i>J</i> = 6.6 Hz, 3H), 0.16 (s, 6H).</div><div class="NLM_p last">Compound <b>25</b> (34 mg, 0.14 mmol) was prepared in 70% yield (for two steps) as a colorless oil by following the same procedure as described in the general procedure D with <b>24</b>. <i>R</i><sub><i>f</i></sub> 0.21 (toluene/EtOAc = 10:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 6.89 (d, <i>J</i> = 8.4 Hz, 1H), 6.88–6.86 (m, 1H), 6.85–6.81 (m, 1H), 6.48 (d, <i>J</i> = 11.4 Hz, 1H), 5.64 (s, 1H), 5.57 (dd, <i>J</i> = 9.0 and 11.7 Hz, 1H), 4.56–4.51 (m, 1H), 3.90 (s, 3H), 1.71–1.63 (m, 1H), 1.61–1.54 (m, 1H), 1.52 (brs, 1H), 1.46–1.23 (m, 6H), 0.88 (t, <i>J</i> = 6.6 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 146.36, 145.14, 133.27, 131.25, 129.15, 122.29, 114.34, 111.50, 68.20, 56.01, 37.94, 31.94, 25.31, 22.74, 14.13. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>22</sub>O<sub>3</sub> [M – H]<sup>−</sup>, 249.1496; found, 249.1523. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 11.09 min, method H, <i>t</i><sub>R</sub> = 8.78 min).</div></div><div id="sec4_2_4_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i75" class="anchor-spacer"></div><h5 class="article-section__title" id="_i75"> 1-(4-((<i>tert</i>-Butyldimethylsilyl)oxy)-3-methoxyphenyl)octan-3-ol (<b>26</b>)</h5><div class="NLM_p last">Compound <b>26</b> was prepared in 94% yield as a colorless oil by following the same procedure as described in the general procedure B with <b>22a</b> (33.0 mg, 0.091 mmol). <i>R</i><sub><i>f</i></sub> 0.60 (hexane/EtOAc = 5:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 6.75 (d, <i>J</i> = 8.0 Hz, 1H), 6.69 (d, <i>J</i> = 2.0 Hz, 1H), 6.64 (dd, <i>J</i> = 2.0 and 8.0 Hz, 1H), 3.79 (s, 3H), 3.65–3.57 (m, 1H), 2.75–2.66 (m, 1H), 2.64–2.55 (m, 1H), 1.81–1.66 (m, 2H), 1.54–1.38 (m, 3H), 1.38–1.22 (m, 6H), 0.99 (s, 9H), 0.87 (t, <i>J</i> = 7.2 Hz, 3H), 0.14 (s, 6H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 150.75, 143.04, 135.66, 120.72, 120.41, 112.50, 71.50, 55.50, 39.22, 37.59, 31.89, 31.81, 25.76, 25.32, 22.65, 18.45, 14.06, −4.63.</div></div><div id="sec4_2_4_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i76" class="anchor-spacer"></div><h5 class="article-section__title" id="_i76"> 1-(4-Hydroxy-3-methoxyphenyl)octan-3-one (<b>27</b>)</h5><div class="NLM_p last">Compound <b>27</b> was prepared in 94% yield as a white oil by following the same procedure as described in the general procedure D with <b>26</b> (32.0 mg, 0.087 mmol). <i>R</i><sub><i>f</i></sub> 0.21 (hexane/EtOAc = 5:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 6.83 (d, <i>J</i> = 7.9 Hz, 1H), 6.72–6.67 (m, 2H), 5.50 (brs, 1H), 3.87 (s, 3H), 3.65–3.59 (m, 1H), 2.77–2.69 (m, 1H), 2.65–2.55 (m, 1H), 1.82–1.66 (m, 2H), 1.53–1.37 (m, 4H), 1.37–1.22 (m, 6H), 0.89 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 146.41, 143.68, 134.13, 120.90, 114.26, 110.99, 71.52, 55.88, 39.37, 37.59, 31.89, 31.80, 25.30, 22.64, 14.05. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>24</sub>O<sub>3</sub> [M – H]<sup>−</sup>, 251.1652; found, 251.1683. >95% purity (as determined by RP-HPLC, method C, <i>t</i><sub>R</sub> = 5.37 min, method G, <i>t</i><sub>R</sub> = 29.83 min).</div></div><div id="sec4_2_4_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i77" class="anchor-spacer"></div><h5 class="article-section__title" id="_i77"> 1-(3,4-Difluorophenyl)hex-1-yn-3-ol (<b>33a</b>)</h5><div class="NLM_p last">Compound <b>33a</b> was prepared in 88% yield as a colorless oil by following the same procedure as described for the synthesis of <b>22a</b> but with <b>30</b> (50.0 mg, 0.240 mmol) instead of <b>20</b>. <i>R</i><sub><i>f</i></sub> 0.25 (hexane/EtOAc = 10:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.26–7.20 (m, 1H), 7.18–7.13 (m, 1H), 7.12–7.05 (m, 1H), 4.59 (t, <i>J</i> = 6.7 Hz, 1H), 2.11 (brs, 1H), 1.83–1.72 (m, 2H), 1.59–1.49 (m, 2H), 0.98 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 151.46, 151.37, 150.78, 150.69, 149.79, 149.71, 149.13, 149.04, 135.49, 128.34, 128.31, 128.30, 128.27, 120.69, 120.56, 119.56, 119.53, 119.51, 119.48, 117.48, 117.36, 90.77, 90.75, 82.73, 62.63, 39.83, 18.48, 13.74. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>12</sub>H<sub>12</sub>F<sub>2</sub>O [M – H]<sup>−</sup>, 209.0783; found, 209.0820. >95% purity (as determined by RP-HPLC, method E, <i>t</i><sub>R</sub> = 5.43 min, method I, <i>t</i><sub>R</sub> = 4.90 min).</div></div><div id="sec4_2_4_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i78" class="anchor-spacer"></div><h5 class="article-section__title" id="_i78"> 1-(3,4-Difluorophenyl)hept-1-yn-3-ol (<b>34a</b>)</h5><div class="NLM_p last">Compound <b>34a</b> was prepared in 80% yield as a colorless oil by following the same procedure as described for the synthesis of <b>22a</b> but with <b>31</b> (17.0 mg, 0.076 mmol) instead of <b>20</b>. <i>R</i><sub><i>f</i></sub> 0.29 (hexane/EtOAc = 10:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.26–7.21 (m, 1H), 7.18–7.14 (m, 1H), 7.10 (q, <i>J</i> = 8.4 Hz, 1H), 4.57 (dd, <i>J</i> = 6.4 and 12.3 Hz, 1H), 1.88 (d, <i>J</i> = 5.3 Hz, 1H), 1.85–1.73 (m, 2H), 1.55–1.43 (m, 2H), 1.43–1.34 (m, 2H), 0.94 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 151.39, 150.79, 150.71, 149.81, 149.72, 149.15, 128.36, 128.33, 128.31, 128.29, 120.71, 120.59, 119.56, 119.50, 117.50, 117.38, 90.76, 82.75, 62.89, 37.50, 27.34, 22.38, 14.01. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>13</sub>H<sub>14</sub>F<sub>2</sub>O [M – H]<sup>−</sup>, 223.0940; found, 223.0966. >95% purity (as determined by RP-HPLC, method E, <i>t</i><sub>R</sub> = 7.35 min, method I, <i>t</i><sub>R</sub> = 6.77 min).</div></div><div id="sec4_2_4_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i79" class="anchor-spacer"></div><h5 class="article-section__title" id="_i79"> 1-(3,4-Difluorophenyl)oct-1-yn-3-ol (<b>35a</b>)</h5><div class="NLM_p last">Compound <b>35a</b> was prepared in 86% yield as a colorless oil by following the same procedure as described for the synthesis of <b>22a</b> but with <b>32</b> (58.0 mg, 0.245 mmol) instead of <b>20</b>. <i>R</i><sub><i>f</i></sub> 0.35 (hexane/EtOAc = 10:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.26–7.20 (m, 1H), 7.18–7.13 (m, 1H), 7.09 (q, <i>J</i> = 8.4 Hz, 1H), 4.57 (dd, <i>J</i> = 6.4 and 12.3 Hz, 1H), 2.00 (d, <i>J</i> = 5.6 Hz, 1H), 1.86–1.71 (m, 2H), 1.57–1.43 (m, 2H), 1.40–1.21 (m, 4H), 0.91 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 151.38, 150.79, 150.70, 149.80, 149.71, 149.14, 149.05, 128.35, 128.32, 128.31, 128.28, 120.70, 120.57, 119.56, 119.54, 119.51, 119.49, 117.49, 117.37, 90.78, 82.74, 62.89, 37.74, 31.45, 24.87, 22.56, 14.00. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>16</sub>F<sub>2</sub>O [M – H]<sup>−</sup>, 237.1096; found, 237.1138. >95% purity (as determined by RP-HPLC, method E, <i>t</i><sub>R</sub> = 10.34 min, method I, <i>t</i><sub>R</sub> = 9.96 min).</div></div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> General Procedure E for Compounds <b>22b</b>, <b>22c</b>, <b>33b</b>, <b>34b</b>, <b>35b</b>, <b>33c</b>, <b>34c</b>, and <b>35c</b></h4><div class="NLM_p">To a 0.1 M solution of an ynone compound in 2-propanol were added KOH and catalyst with the ratio of ynone/catalyst/KOH = 200:1:1.2. The reaction mixture was stirred under argon at room temperature until TLC analysis indicated complete conversion (typically 4 h) and concentrated under reduced pressure. The crude residue was purified by column chromatography on a silica gel to furnish stereoselective compounds. The ee values were determined by the chiral HPLC analyses on a chiral column (CHIRALPAK IG, 10% ethanol in hexane).</div><div id="sec4_2_5_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i81" class="anchor-spacer"></div><h5 class="article-section__title" id="_i81"> (<i>R</i>)-1-(4-((<i>tert</i>-Butyldimethylsilyl)oxy)-3-methoxyphenyl)oct-1-yn-3-ol (<b>22b</b>)</h5><div class="NLM_p last">Compound <b>22b</b> was prepared in 88% yield as a colorless oil following the same procedure as described in the general procedure E with <b>20</b> (150 mg, 0.416 mmol) in 2-propanol (4.16 mL), RuCl[(<i>R</i>,<i>R</i>)-TsDPEN(mesitylene)] (1.29 mg, 2.08 μmol) and KOH (0.138 mg, 2.50 μmol). The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 12:1, v/v) to furnish compound <b>22b</b>. <i>R</i><sub><i>f</i></sub> 0.59 (hexane/EtOAc = 4:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 6.94–6.92 (m, 2H), 6.77 (d, <i>J</i> = 7.9 Hz, 1H), 4.58 (q, <i>J</i> = 6.4 Hz, 1H), 3.79 (s, 3H), 1.90 (d, <i>J</i> = 5.5 Hz, 1H), 1.81–1.76 (m, 2H), 1.53–1.50 (m, 2H), 1.36–1.33 (m, 4H), 0.98 (s, 9H), 0.91 (t, <i>J</i> = 7.0 Hz, 3H), 0.15 (s, 6H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 150.66, 145.84, 124.95, 120.90, 115.68, 115.30, 88.71, 85.02, 63.09, 55.44, 37.97, 31.51, 25.68, 24.95, 22.60, 18.47, 14.04, −4.65.</div></div><div id="sec4_2_5_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82"> (<i>S</i>)-1-(4-((<i>tert</i>-Butyldimethylsilyl)oxy)-3-methoxyphenyl)oct-1-yn-3-ol (<b>22c</b>)</h5><div class="NLM_p last">Compound <b>22c</b> was prepared in 68% yield as a colorless oil following the same procedure as described in the general procedure E with <b>20</b> (150 mg, 0.416 mmol) in 2-propanol (4.16 mL), RuCl[(<i>S</i>,<i>S</i>)-TsDPEN(mesitylene)] (1.29 mg, 2.08 μmol) and KOH (0.138 mg, 2.50 μmol). The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 12:1, v/v) to furnish compound <b>22c</b>. <i>R</i><sub><i>f</i></sub> 0.59 (hexane/EtOAc = 4:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 6.94–6.92 (m, 2H), 6.77 (d, <i>J</i> = 7.9 Hz, 1H), 4.58 (t, <i>J</i> = 6.5 Hz, 1H), 3.79 (s, 3H), 1.91 (s, 1H), 1.81–1.76 (m, 2H), 1.53–1.49 (m, 2H), 1.35–1.34 (m, 4H), 0.98 (s, 9H), 0.91 (t, <i>J</i> = 7.0 Hz, 3H), 0.15 (s, 6H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 150.66, 145.84, 124.95, 120.90, 115.68, 115.31, 88.71, 85.01, 63.08, 55.44, 37.97, 31.51, 25.68, 24.95, 22.60, 18.47, 14.04, −4.65.</div></div><div id="sec4_2_5_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i83" class="anchor-spacer"></div><h5 class="article-section__title" id="_i83"> (<i>R</i>)-1-(3,4-Difluorophenyl)hex-1-yn-3-ol (<b>33b</b>)</h5><div class="NLM_p last">Compound <b>33b</b> was prepared in 86% yield as a colorless oil following the same procedure as described in the general procedure E with <b>30</b> (100 mg, 0.48 mmol) in 2-propanol (4.8 mL), RuCl[(<i>R</i>,<i>R</i>)-TsDPEN(mesitylene)] (1.49 mg, 2.40 μmol) and KOH (0.16 mg, 2.90 μmol). The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 20:1, v/v) to furnish compound <b>33b</b>. <i>R</i><sub><i>f</i></sub> 0.25 (hexane/EtOAc = 10:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.25–7.19 (m, 1H), 7.18–7.13 (m, 1H), 7.12–7.04 (m, 1H), 4.59 (t, <i>J</i> = 2.9 Hz, 1H), 2.18 (brs, 1H), 1.83–1.72 (m, 2H), 1.58–1.49 (m, 2H), 0.98 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 151.45, 151.37, 150.78, 150.69, 149.79, 149.70, 149.12, 149.04, 128.34, 128.31, 128.29, 128.27, 120.68, 120.56, 119.56, 119.53, 119.51, 119.48, 117.48, 117.36, 90.77, 90.76, 82.72, 62.62, 39.83, 18.49, 13.74. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>12</sub>H<sub>12</sub>F<sub>2</sub>O [M – H]<sup>−</sup>, 209.0783; found, 209.0811. >95% purity (as determined by RP-HPLC, method E, <i>t</i><sub>R</sub> = 5.42 min, method I, <i>t</i><sub>R</sub> = 4.86 min).</div></div><div id="sec4_2_5_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i84" class="anchor-spacer"></div><h5 class="article-section__title" id="_i84"> (<i>R</i>)-1-(3,4-Difluorophenyl)hept-1-yn-3-ol (<b>34b</b>)</h5><div class="NLM_p last">Compound <b>34b</b> was prepared in 82% yield as a colorless oil following the same procedure as described in the general procedure E with <b>31</b> (100 mg, 0.45 mmol) in 2-propanol (4.5 mL), RuCl[(<i>R</i>,<i>R</i>)-TsDPEN(mesitylene)] (1.40 mg, 2.30 μmol) and KOH (0.15 mg, 2.70 μmol). The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 20:1, v/v) to furnish compound <b>34b</b>. <i>R</i><sub><i>f</i></sub> 0.29 (hexane/EtOAc = 10:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.26–7.19 (m, 1H), 7.19–7.13 (m, 1H), 7.12–7.05 (m, 1H), 4.57 (t, <i>J</i> = 2.9 Hz, 1H), 2.18 (brs, 1H), 1.85–1.74 (m, 2H), 1.53–1.45 (m, 2H), 1.42–1.34 (m, 2H), 0.94 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 150.78, 150.69, 149.79, 149.70, 149.12, 149.04, 128.34, 128.32, 128.30, 128.28, 120.69, 120.56, 119.54, 119.49, 117.48, 117.36, 90.80, 82.71, 62.85, 37.48, 27.35, 22.37, 13.99. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>13</sub>H<sub>14</sub>F<sub>2</sub>O [M – H]<sup>−</sup>, 223.0940; found, 223.0973. >95% purity (as determined by RP-HPLC, method E, <i>t</i><sub>R</sub> = 7.32 min, method I, <i>t</i><sub>R</sub> = 6.64 min).</div></div><div id="sec4_2_5_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i85" class="anchor-spacer"></div><h5 class="article-section__title" id="_i85"> (<i>R</i>)-1-(3,4-Difluorophenyl)oct-1-yn-3-ol (<b>35b</b>)</h5><div class="NLM_p last">Compound <b>35b</b> was prepared in 80% yield as a colorless oil following the same procedure as described in the general procedure E with <b>32</b> (100 mg, 0.42 mmol) in 2-propanol (4.2 mL), RuCl[(<i>R</i>,<i>R</i>)-TsDPEN(mesitylene)] (1.31 mg, 2.10 μmol) and KOH (0.14 mg, 2.50 μmol). The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 20:1, v/v) to furnish compound <b>35b</b>. <i>R</i><sub><i>f</i></sub> 0.35 (hexane/EtOAc = 10:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.25–7.21 (m, 1H), 7.20–7.15 (m, 1H), 7.14–7.03 (m, 1H), 4.57 (dd, <i>J</i> = 6.0 and 10.8 Hz, 1H), 2.28 (d, <i>J</i> = 4.2 Hz, 1H), 1.89–1.72 (m, 2H), 1.58–1.42 (m, 2H), 1.38–1.33 (m, 4H), 0.90 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 151.45, 151.36, 150.77, 150.69, 149.78, 149.70, 149.12, 149.04, 128.33, 128.31, 128.29, 128.27, 120.68, 120.55, 119.58, 119.55, 119.53, 119.50, 117.47, 117.35, 90.82, 82.70, 62.85, 37.72, 31.44, 24.88, 22.56, 13.98. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>16</sub>F<sub>2</sub>O [M – H]<sup>−</sup>, 237.1096; found, 237.1139. >95% purity (as determined by RP-HPLC, method E, <i>t</i><sub>R</sub> = 10.26 min, method I, <i>t</i><sub>R</sub> = 11.03 min).</div></div><div id="sec4_2_5_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i86" class="anchor-spacer"></div><h5 class="article-section__title" id="_i86"> (<i>S</i>)-1-(3,4-Difluorophenyl)hex-1-yn-3-ol (<b>33c</b>)</h5><div class="NLM_p last">Compound <b>33c</b> was prepared in 86% yield as a colorless oil following the same procedure as described in the general procedure E with <b>30</b> (100 mg, 0.48 mmol) in 2-propanol (4.8 mL), RuCl[(<i>S</i>,<i>S</i>)-TsDPEN(mesitylene)] (1.49 mg, 2.40 μmol) and KOH (0.16 mg, 2.90 μmol). The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 20:1, v/v) to furnish compound <b>33c</b>. <i>R</i><sub><i>f</i></sub> 0.25 (hexane/EtOAc = 10:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.26–7.18 (m, 1H), 7.18–7.12 (m, 1H), 7.12–7.03 (m, 1H), 4.59 (t, <i>J</i> = 2.9 Hz, 1H), 2.39 (brs, 1H), 1.84–1.72 (m, 2H), 1.59–1.47 (m, 2H), 0.98 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 151.44, 151.36, 150.76, 150.68, 149.77, 149.69, 149.12, 149.03, 128.33, 128.30, 128.28, 128.26, 120.67, 120.54, 119.57, 119.54, 119.52, 119.49, 117.46, 117.35, 90.79, 82.71, 82.69, 62.59, 39.81, 18.49, 13.72. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>12</sub>H<sub>12</sub>F<sub>2</sub>O [M – H]<sup>−</sup>, 209.0783; found, 209.0816. >95% purity (as determined by RP-HPLC, method E, <i>t</i><sub>R</sub> = 5.42 min, method I, <i>t</i><sub>R</sub> = 4.88 min).</div></div><div id="sec4_2_5_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i87" class="anchor-spacer"></div><h5 class="article-section__title" id="_i87"> (<i>S</i>)-1-(3,4-Difluorophenyl)hept-1-yn-3-ol (<b>34c</b>)</h5><div class="NLM_p last">Compound <b>29c</b> was prepared in 82% yield as a colorless oil following the same procedure as described in the general procedure E with <b>31</b> (100 mg, 0.45 mmol) in 2-propanol (4.5 mL), RuCl[(<i>S</i>,<i>S</i>)-TsDPEN(mesitylene)] (1.4 mg, 2.30 μmol) and KOH (0.15 mg, 2.70 μmol). The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 20:1, v/v) to furnish compound <b>34c</b>. <i>R</i><sub><i>f</i></sub> 0.29 (hexane/EtOAc = 10:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.26–7.19 (m, 1H), 7.19–7.12 (m, 1H), 7.12–7.04 (m, 1H), 4.57 (t, <i>J</i> = 2.9 Hz, 1H), 2.37 (brs, 1H), 1.86–1.72 (m, 2H), 1.54–1.43 (m, 2H), 1.43–1.34 (m, 2H), 0.93 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 151.44, 151.36, 150.77, 150.68, 149.77, 149.69, 149.12, 149.03, 128.33, 128.31, 128.29, 128.27, 120.67, 120.55, 119.58, 119.55, 119.53, 119.50, 117.46, 117.34, 90.83, 90.82, 82.69, 62.82, 37.47, 27.35, 22.37, 13.97. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>13</sub>H<sub>14</sub>F<sub>2</sub>O [M – H]<sup>−</sup>, 223.0940; found, 223.0971. >95% purity (as determined by RP-HPLC, method E, <i>t</i><sub>R</sub> = 7.32 min, method I, <i>t</i><sub>R</sub> = 6.72 min).</div></div><div id="sec4_2_5_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i88" class="anchor-spacer"></div><h5 class="article-section__title" id="_i88"> (<i>S</i>)-1-(3,4-Difluorophenyl)oct-1-yn-3-ol (<b>35c</b>)</h5><div class="NLM_p last">Compound <b>35c</b> was prepared in 82% yield as a colorless oil following the same procedure as described in the general procedure E with <b>32</b> (100 mg, 0.42 mmol) in 2-propanol (4.2 mL), RuCl[(<i>S</i>,<i>S</i>)-TsDPEN(mesitylene)] (1.31 mg, 2.10 μmol) and KOH (0.14 mg, 2.50 μmol). The crude residue was purified by column chromatography on a silica gel (hexane/EtOAc = 20:1, v/v) to furnish compound <b>35c</b>. <i>R</i><sub><i>f</i></sub> 0.35 (hexane/EtOAc = 10:1, v/v). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.26–7.19 (m, 1H), 7.19–7.12 (m, 1H), 7.12–7.04 (m, 1H), 4.57 (t, <i>J</i> = 2.9 Hz, 1H), 2.31 (brs, 1H), 1.85–1.72 (m, 2H), 1.56–1.44 (m, 2H), 1.39–1.28 (m, 4H), 0.90 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 151.44, 151.36, 150.77, 150.68, 149.78, 149.69, 149.12, 149.04, 128.33, 128.31, 128.29, 128.26, 120.67, 120.55, 119.58, 119.55, 119.53, 119.50, 117.47, 117.35, 90.82, 82.70, 62.85, 37.72, 31.44, 24.88, 22.56, 13.97. HRMS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>16</sub>F<sub>2</sub>O [M – H]<sup>−</sup>, 237.1096; found, 237.1137. >95% purity (as determined by RP-HPLC, method E, <i>t</i><sub>R</sub> = 10.31 min, method I, <i>t</i><sub>R</sub> = 9.79 min).</div></div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> Chiral HPLC Analysis</h3><div class="NLM_p last">Enantiomeric excess (ee) of the compounds was determined by the chiral HPLC analyses using a chiral column [CHIRALPAK IG (4.6 i.d × 250 mm)]. Chromatographic analysis was performed on high-performance liquid chromatography (Agilent 1260 series) by two methods (A and B). Method A was applied over 30 min at a flow rate of 1 mL/min with an isocratic of 10% ethanol in hexane (v/v). The autosampler and the column compartment temperatures were set at 25 °C. UV detection was carried out at a wavelength of 254 nm. The sample (5 μL) was injected. Method B was applied over 30 min at a flow rate of 0.8 mL/min with an isocratic of 1% ethanol in hexane (v/v). The autosampler and the column compartment temperatures were set at 25 °C. UV detection was carried out at a wavelength of 254 nm. The sample (3 μL) was injected.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> QS Reporter Strain Assay</h3><div class="NLM_p last"><i>Escherichia coli</i> (<i>E. coli</i>) DH5α contains a LuxR homologue SdiA that can detect exogenous AHL synthesized by other microbial species, especially BHL.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> To avoid this possible interference, we used <i>sdiA</i> mutants cotransformed with two plasmids, pJN105R (RhlR expression plasmid) and pSC11 (RhlA::<i>lacZ</i> fusion plasmid), for RhlR reporter strain assay. An overnight culture of reporter strain (1%, OD 595 nm = 1.0) with 10 μg/mL gentamicin and 50 μg/mL ampicillin was incubated in a Luria–Bertani medium (Difco, Detroit, MI, USA) at 37 °C for 2 h. Incubated reporter strain (OD 595 nm = 0.3) with positive controls or the compounds (0–1000 μM), BHL (0–10 μM) (Sigma-Aldrich), and 0.4% arabinose (Sigma-Aldrich) was reacted at 37 °C for 1.5 h. β-Galactosidase activity was estimated using a Tropix plus kit (Applied Biosystems, CA, USA), and OD 595 nm and luminescence were measured on a VICTOR ×5 multimode plate reader (PerkinElmer, Waltham, MA, USA). Relative RhlR activity was quantified by dividing luminescence with OD 595 nm. For LasR reporter strain assay, <i>E. coli</i> DH5α cotransformed with two plasmids, pJN105L (LasR expression plasmid) and pSC11 (lasI::<i>lacZ</i> fusion plasmid), and OdDHL (Sigma-Aldrich) were used. In addition, for the PqsR reporter strain assay, <i>E. coli</i> DH5α cotransformed with two plasmids, pJN105P (PqsR expression plasmid) and pSC11 (PqsA::<i>lacZ</i> fusion plasmid), and PQS (Sigma-Aldrich) were used.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> Static Biofilm Formation Assay</h3><div class="NLM_p last">An overnight culture of <i>P. aeruginosa</i> (OD 595 nm = 1.0) was diluted with an AB medium (1:20) (300 mM NaCl, 50 mM MgSO<sub>4</sub>, 0.2% vitamin-free casamino acids, 10 mM potassium phosphate, 1 mM <span class="smallcaps smallerCapital">l</span>-arginine, and 1% glucose, pH 7.5) with positive controls or the compounds (0–100 μM) in borosilicate bottles. After incubation at 37 °C for 24 h without shaking, OD 595 nm of the suspended culture was measured on the multimode plate reader to measure the growth inhibition activity. The biofilm cells attached to the bottles were washed two times with phosphate-buffered saline (PBS; 137 mM NaCl, 2.7 mM KCl, 10 mM Na<sub>2</sub>HPO<sub>4</sub>, and 2 mM KH<sub>2</sub>PO<sub>4</sub>, pH 7.2). Then, the remained biofilm cells were stained using 0.1% crystal violet for 10 min and washed with deionized water to remove unbound crystal violet. The bounded crystal violet to biofilm cells was eluted in 100% ethanol, and the OD of the eluted ethanol samples was measured at 545 nm using the multimode plate reader.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i92" class="anchor-spacer"></div><h3 class="article-section__title" id="_i92"> Dynamic Biofilm Formation Assay</h3><div class="NLM_p last">Glass slides were inserted into a drip-flow reactor, and an AB medium containing 5% of <i>P. aeruginosa</i> (OD 595 nm = 1.0) with positive controls or the compounds (0–10 μM) was continuously fed into the reactor using a peristaltic pump (Masterflex C/L tubing pumps, Cole-Parmer, IL, USA) at 0.3 mL/min. After the operation at 37 °C for 48 h, the unattached biofilm cells were removed with PBS. The remained biofilm cells were stained with 4′,6-diamidino-2-phenylindole (Sigma-Aldrich) for 10 min and observed using confocal laser scanning microscopy (CLSM; Carl Zeiss LSM700, Jena, Germany). CLSM images of biofilm cells were obtained under blue fluorescence light (excitation wavelength: 350 nm, emission wavelength: 470 nm) and a 20× objective lens [W NAchroplan ×20/0.5W (DIC) M27] using Z-stack mode in Zen 2011 program (Carl Zeiss). Biofilm volume and thickness were measured by Comstat2 in the ImageJ program based on the CLSM images.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i93" class="anchor-spacer"></div><h3 class="article-section__title" id="_i93"> Extracellular Polymeric Substance Analysis</h3><div class="NLM_p last">Biofilm samples were prepared after drip-flow reactor operation by scrapping cells attached to the slides. After resuspension of centrifuged biofilm cells with 0.01 M KCl, the suspension was disrupted with a sonicator (VCX 750, SONICS, Newtown, CT, USA) for 4 cycles of 5 s of operation and 5 s of pause at a power level of 3.5 Hz for carbohydrate and protein analysis. The supernatant of the sonicated suspension was filtered through a 0.22 μm membrane filter (Millex filter, Carl Roth, Karlsruhe, Germany). For protein analysis, mixtures containing 40 μL of the filtrate and 200 μL of the Lowry reagent (L3540, Sigma-Aldrich) were incubated for 10 min at room temperature. After adding 20 μL of the Folin-Ciocalteu reagent (Sigma-Aldrich) and incubating at room temperature for 30 min, OD 750 nm was measured using the multimode plate reader. The amount of protein was quantified by dividing OD 750 nm by OD 595 nm. For carbohydrate analysis, mixtures containing 50 μL of the filtrate and 150 μL of 99.9% sulfuric acid were incubated for 30 min at room temperature. After adding 5% phenol and incubating at 90 °C for 5 min, OD 490 nm was measured using the multimode plate reader. The amount of protein was quantified by dividing OD 490 nm by OD 595 nm.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i94" class="anchor-spacer"></div><h3 class="article-section__title" id="_i94"> Rhamnolipid Production Assay</h3><div class="NLM_p last">An overnight culture of <i>P. aeruginosa</i> (OD 595 nm = 1.0) was inoculated in an AB medium (1:100) with positive controls or the compounds (0–100 μM) and then incubated using a shaking incubator at 37 °C for 24 h. The culture was centrifuged at 12,000<i>g</i> at 4 °C for 5 min. Crude rhamnolipid was initially extracted twice by mixing 200 μL of the supernatant and 400 μL of 100% diethyl ether (Junsei, Tokyo, Japan). The ether fraction was transferred and evaporated into a new tube. The dry sample was eluted in 20 μL of deionized water and then reacted with 180 μL of Orcinol solution [0.19% Orcinol (Sigma-Aldrich) in 53% H<sub>2</sub>SO<sub>4</sub>]. The reacted sample was boiled at 80 °C for 30 min and cooled at room temperature for 15 min. The amounts of rhamnolipid were measured at OD 421 nm and normalized with OD 595 nm in bacterial culture using the multimode plate reader.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i95" class="anchor-spacer"></div><h3 class="article-section__title" id="_i95"> Pyocyanin Production Assay</h3><div class="NLM_p last">Diluted overnight culture of <i>P. aeruginosa</i> (OD 595 nm = 1.0) (1:100) using an AB medium with positive controls or the compounds (0–10 μM) was incubated at 37 °C for 24 h and centrifuged at 10,000 rpm for 10 min. To extract pyocyanin, the supernatant was reacted with 50% TFA (Sigma-Aldrich) at 25 °C for 1 h. The reacted supernatant was centrifuged at 10,000 rpm for 10 min and then passed through a 0.22 μm filter. The filtered supernatants were analyzed by a 1260 InfinityII Prep-HPLC System (Agilent Technologies, Santa Clara, CA, USA) and EC-C18 column (4.6 × 150 mm, Agilent Technologies). The detailed HPLC conditions were as follows: 99:1 water/TFA (v/v) mobile phase; 10 μL injection volume; 25 °C temperature; and 0.5 mL/min flow rate. The retention time of pyocyanin (Sigma-Aldrich) was 20 min. The height of analyzed peaks at 20 min of retention time was detected for estimating the amounts of pyocyanin.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i96" class="anchor-spacer"></div><h3 class="article-section__title" id="_i96"> Mortality Experiment of <i>T. molitor</i> Larvae</h3><div class="NLM_p last">The mortality of <i>T. molitor</i> larvae with positive controls or the compounds (0–10 μM) was measured according to the previous study.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> An overnight culture of <i>P. aeruginosa</i> (OD 595 nm = 1.0) was centrifuged at 10,000 rpm for 10 min. The filtered supernatant (10 μL) through a 0.22 μm filter was injected into larvae using syringe needles. The larvae were maintained in Petri dishes at 25 °C for 20 days. The mortality of larvae was observed every 5 days.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i97" class="anchor-spacer"></div><h3 class="article-section__title" id="_i97"> RT-qPCR Analysis</h3><div class="NLM_p last">An overnight culture of <i>P. aeruginosa</i> (OD 595 nm = 1.0) was diluted (1:20) with an AB medium with positive controls or the compounds (0–10 μM) in borosilicate bottles. Following incubation at 37 °C for 24 h, biofilm cells were collected for RNA extraction. Total RNA was extracted using the TRI Reagent (Molecular Research Center, OH, USA) following the manufacturer’s instruction. RT-qPCR was performed to quantify and compare the levels of QS-related gene expression. SYBR Premix Ex TaqTM (Takara, Shiga, Japan), CFX-96 real-time system (Bio-Rad, Hercules, CA, USA), and QS-related gene primer sets were used for RT-qPCR.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Thermal profiles of the RT-qPCR were as follows: initial denaturation at 95 °C for 10 s, followed by 40 cycles of denaturation at 95 °C for 10 s, annealing at 60 °C for 10 s, and extension at 63 °C for 34 s. The fluorescent signal intensity was collected at the end of the extension step.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i98" class="anchor-spacer"></div><h3 class="article-section__title" id="_i98"> <i>In Silico</i> Docking Studies</h3><div class="NLM_p last">A homology model of RhlR based on the crystal structure of SdiA in complex with 3-oxo-C6-homoserine lactone (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y15">4Y15</a>)<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> was generated. The protein sequence of RhlR was obtained from NCBI protein database (<a href="http://ncbi.nlm.nih.gov/protein" class="extLink">http://ncbi.nlm.nih.gov/protein</a>) as the FASTA format. The homology model of RhlR was generated using the SWISS-MODEL (<a href="https://swissmodel.expasy.org" class="extLink">https://swissmodel.expasy.org</a>). All final compounds were generated as a 3D structure by <i>Chem3D Pro</i> (ver 12.0) and the group of compounds was saved as .<i>sdf</i> file. “<i>Sanitize</i>” preparation protocol in <i>SYBYL-X 2.1.1</i> (Tripos Inc., St Louis) was applied to ligand preparation and optimization process. The prepared final compounds were docked into the RhlR homology model using the <i>Surflex-Dock GeomX</i> module of <i>SYBYL-X 2.1.1</i>. The protein minimization for molecular docking was performed by the “<i>POWELL</i>” method with “<i>Tripos</i>” Force Field setting. The initial optimization option was set to <i>None</i>. “<i>Surflex-Dock protomol</i>” was used to guide the docking site as defined by the “<i>Residues</i>” method with the selected amino acids (Tyr43, Val60, Tyr64, Trp68, Asp81, Ile84, Ser135; radius setting: 4.0; those amino acids were selected based on the active site of SdiA, the template protein of RhlR homology model). Other docking parameters were kept to the default values.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i99" class="anchor-spacer"></div><h3 class="article-section__title" id="_i99"> Statistical Analysis</h3><div class="NLM_p last"><i>P</i> values were estimated by the Student’s <i>t</i>-test (SigmaPlot version 10, Systat Software Inc., San Jose, CA, USA).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i100"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00630" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01507" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01507" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00630?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00630</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Analytical data (<sup>1</sup>H NMR, <sup>13</sup>C NMR and HPLC spectra) of the key intermediates and final compounds and biological data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">PDB coordinates of homology RhlR model in complex with compound <b>30</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_002.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">PDB coordinates of homology RhlR model in complex with compound <b>31</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">PDB coordinates of homology RhlR model in complex with compound <b>32</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_004.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">PDB coordinates of homology RhlR model in complex with compound <b>1d</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_005.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Molecular formula strings of <b>11a–11r</b>, <b>12a–12r</b>, <b>13a</b> and <b>13b</b>, <b>14a</b> and <b>14b</b>, <b>15a</b> and <b>15b</b>, <b>16a</b>, <b>21</b>, <b>23a–23c</b>, <b>25</b>, <b>27</b>, <b>30–32</b>, <b>33a–33c</b>, <b>34a–34c</b>, and <b>35a–35c</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_006.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">Molecular formula strings and biological data for final compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_007.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_001.pdf">jm0c00630_si_001.pdf (5.72 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_002.pdb">jm0c00630_si_002.pdb (302.13 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_003.pdb">jm0c00630_si_003.pdb (302.37 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_004.pdb">jm0c00630_si_004.pdb (302.61 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_005.pdb">jm0c00630_si_005.pdb (302.37 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_006.xlsx">jm0c00630_si_006.xlsx (16.86 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_007.csv">jm0c00630_si_007.csv (3.72 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c00630" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46866" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46866" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hee-Deung Park</span> - <span class="hlFld-Affiliation affiliation">School
of Civil, Environmental and Architectural Engineering, Korea University, 145 Anam-ro, Seongbuk-Gu, Seoul 02841, Republic of Korea</span>; 
    <span class="hlFld-Affiliation affiliation">KU-KIST
Graduate School of Converging Science and Technology, Korea University, 145
Anam-ro, Seongbuk-Gu, Seoul 02841, Republic of Korea</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#08606d6d6c6d7d666f4863677a6d6926696b26637a"><span class="__cf_email__" data-cfemail="bed6dbdbdadbcbd0d9fed5d1ccdbdf90dfdd90d5cc">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Youngjoo Byun</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy, Korea University, 2511 Sejong-ro,
Jochiwon-eup, Sejong 30019, Republic of Korea</span>; 
    <span class="hlFld-Affiliation affiliation">Biomedical
Research Center, Korea University Guro Hospital, 148 Gurodong-ro, Guro-gu, Seoul 08308, Republic of Korea</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0297-7734" title="Orcid link">http://orcid.org/0000-0002-0297-7734</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#7a031018030f144b3a1115081f1b541b19541108"><span class="__cf_email__" data-cfemail="6b120109121e055a2b0004190e0a450a08450019">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">SangJin Nam</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy, Korea University, 2511 Sejong-ro,
Jochiwon-eup, Sejong 30019, Republic of Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">So-Young Ham</span> - <span class="hlFld-Affiliation affiliation">School
of Civil, Environmental and Architectural Engineering, Korea University, 145 Anam-ro, Seongbuk-Gu, Seoul 02841, Republic of Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hongmok Kwon</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy, Korea University, 2511 Sejong-ro,
Jochiwon-eup, Sejong 30019, Republic of Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Han-Shin Kim</span> - <span class="hlFld-Affiliation affiliation">Korean
Peninsula Infrastructure Cooperation Team, Korea Institute of Civil Engineering and Building Technology (KICT), Goyang-si, Gyeonggi-do 10223, Republic of Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Suhyun Moon</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy, Korea University, 2511 Sejong-ro,
Jochiwon-eup, Sejong 30019, Republic of Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeong-Hoon Lee</span> - <span class="hlFld-Affiliation affiliation">School
of Civil, Environmental and Architectural Engineering, Korea University, 145 Anam-ro, Seongbuk-Gu, Seoul 02841, Republic of Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Taehyeong Lim</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy, Korea University, 2511 Sejong-ro,
Jochiwon-eup, Sejong 30019, Republic of Korea</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sang-Hyun Son</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy, Korea University, 2511 Sejong-ro,
Jochiwon-eup, Sejong 30019, Republic of Korea</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8624-078X" title="Orcid link">http://orcid.org/0000-0002-8624-078X</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>S.N. and S.Y.H. equal contribution. S.N., S.-Y.H., H.-S.K., S.-H.S., H.-D.P., and Y.B. were involved in the initial stages of the project, including the design of gingerol analogues and the development of biological experiments. S.N., H.K., S.M., T.L., and S.-H.S. synthesized and analyzed gingerol analogues. S.-Y.H., H.-S.K., and J.-H.L. performed biological experiments. S.N., S.-Y.H., H.-S.K., S.-H.S., H.-D.P., and Y.B. analyzed the data and wrote the paper. All authors contributed to editing the final manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i102">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73158" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73158" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the National Research Foundation of Korea (2019R1A6A1A03031807 and 2020R1A2C2005919 to Y.B.) and by Korea Ministry of Environment (MOE) (1485016734 to H.D.P.). The authors thank professor Joon-Hee Lee at Pusan National University for kindly providing QS reporter strains and professor You-Hee Cho at CHA University for kindly providing <i>rhlR</i> mutants.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">ACN</td><td class="NLM_def"><p class="first last">acetonitrile</p></td></tr><tr><td class="NLM_term">AHL</td><td class="NLM_def"><p class="first last"><i>N</i>-acyl-<span class="smallcaps smallerCapital">l</span>-homoserine lactones</p></td></tr><tr><td class="NLM_term">BHL</td><td class="NLM_def"><p class="first last"><i>N</i>-butyryl-<span class="smallcaps smallerCapital">l</span>-homoserine lactone</p></td></tr><tr><td class="NLM_term">CLSM</td><td class="NLM_def"><p class="first last">confocal laser scanning microscopy</p></td></tr><tr><td class="NLM_term">ESI</td><td class="NLM_def"><p class="first last">electrospray ionization</p></td></tr><tr><td class="NLM_term">HRMS</td><td class="NLM_def"><p class="first last">high-resolution mass spectra</p></td></tr><tr><td class="NLM_term">IC<sub>50</sub></td><td class="NLM_def"><p class="first last">half maximal inhibitory concentration</p></td></tr><tr><td class="NLM_term">LDA</td><td class="NLM_def"><p class="first last">diisopropylamide</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">nuclear magnetic resonance</p></td></tr><tr><td class="NLM_term">OdDHL</td><td class="NLM_def"><p class="first last"><i>N</i>-(3-oxo-dodecanoyl)-<span class="smallcaps smallerCapital">l</span>-homoserine lactone</p></td></tr><tr><td class="NLM_term">PBS</td><td class="NLM_def"><p class="first last">phosphate-buffered saline</p></td></tr><tr><td class="NLM_term">PQS</td><td class="NLM_def"><p class="first last"><i>Pseudomonas</i> quinolone signal</p></td></tr><tr><td class="NLM_term">QS</td><td class="NLM_def"><p class="first last">quorum sensing</p></td></tr><tr><td class="NLM_term">RP-HPLC</td><td class="NLM_def"><p class="first last">reverse-phase high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">TBAF</td><td class="NLM_def"><p class="first last">tetrabutylammoniumfluoride</p></td></tr><tr><td class="NLM_term">TBDMS</td><td class="NLM_def"><p class="first last"><i>tert</i>-butyldimethylsilyl</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">TLC</td><td class="NLM_def"><p class="first last">thin-layer chromatography</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i104">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84870" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84870" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 42 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Delden, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iglewski, B. H.</span></span> <span> </span><span class="NLM_article-title">Cell-to-cell signaling and Pseudomonas aeruginosa infections</span>. <i>Emerging Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">551</span>– <span class="NLM_lpage">560</span>, <span class="refDoi"> DOI: 10.3201/eid0404.980405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.3201%2Feid0404.980405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=9866731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADyaK1MXptlam" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1998&pages=551-560&author=C.+Van%0ADeldenauthor=B.+H.+Iglewski&title=Cell-to-cell+signaling+and+Pseudomonas+aeruginosa+infections&doi=10.3201%2Feid0404.980405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Cell-to-cell signaling and Pseudomonas aeruginosa infections</span></div><div class="casAuthors">Van Delden, Christian; Iglewski, Barbara H.</div><div class="citationInfo"><span class="NLM_cas:title">Emerging Infectious Diseases</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">551-560</span>CODEN:
                <span class="NLM_cas:coden">EIDIFA</span>;
        ISSN:<span class="NLM_cas:issn">1080-6040</span>.
    
            (<span class="NLM_cas:orgname">National Center for Infectious Diseases, Centers for Disease Control and Prevention</span>)
        </div><div class="casAbstract">A review with 82 refs.  Pseudomonas aeruginosa is a bacterium responsible for severe nosocomial infections, life-threatening infections in immunocompromised persons, and chronic infections in cystic fibrosis patients.  The bacterium's virulence depends on a large no. of cell-assocd. and extracellular factors.  Cell-to-cell signaling systems control the expression and allow a coordinated, cell-d.-dependent prodn. of many extracellular virulence factors.  The authors discuss the possible role of cell-to-cell signaling in the pathogenesis of P. aeruginosa infections and present a rationale for targeting cell-to-cell signaling systems in the development of new therapeutic approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-y7jqiLUG4rVg90H21EOLACvtfcHk0lhkPjO-6hEcAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXptlam&md5=ddb679006ff63cad39e2ffd22297935c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.3201%2Feid0404.980405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3201%252Feid0404.980405%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDelden%26aufirst%3DC.%26aulast%3DIglewski%26aufirst%3DB.%2BH.%26atitle%3DCell-to-cell%2520signaling%2520and%2520Pseudomonas%2520aeruginosa%2520infections%26jtitle%3DEmerging%2520Infect.%2520Dis.%26date%3D1998%26volume%3D4%26spage%3D551%26epage%3D560%26doi%3D10.3201%2Feid0404.980405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bodey, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolivar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fainstein, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadeja, L.</span></span> <span> </span><span class="NLM_article-title">Infections caused by Pseudomonas aeruginosa</span>. <i>Rev. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">279</span>– <span class="NLM_lpage">313</span>, <span class="refDoi"> DOI: 10.1093/clinids/5.2.279</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1093%2Fclinids%2F5.2.279" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=6405475" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A280%3ADyaL3s7ps1Ojtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1983&pages=279-313&author=G.+P.+Bodeyauthor=R.+Bolivarauthor=V.+Fainsteinauthor=L.+Jadeja&title=Infections+caused+by+Pseudomonas+aeruginosa&doi=10.1093%2Fclinids%2F5.2.279"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Infections caused by Pseudomonas aeruginosa</span></div><div class="casAuthors">Bodey G P; Bolivar R; Fainstein V; Jadeja L</div><div class="citationInfo"><span class="NLM_cas:title">Reviews of infectious diseases</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">279-313</span>
        ISSN:<span class="NLM_cas:issn">0162-0886</span>.
    </div><div class="casAbstract">Pseudomonas aeruginosa has emerged as an important pathogen during the past two decades.  It causes between 10% and 20% of infections in most hospitals.  Pseudomonas infection is especially prevalent among patients with burn wounds, cystic fibrosis, acute leukemia, organ transplants, and intravenous-drug addiction.  P. aeruginosa is a common nosocomial contaminant, and epidemics have been traced to many items in the hospital environment.  Patients who are hospitalized for extended periods are frequently colonized by this organism and are at increased risk of developing infection.  The most serious infections include malignant external otitis, endophthalmitis, endocarditis, meningitis, pneumonia, and septicemia.  The likelihood of recovery from pseudomonas infection is related to the severity of the patient's underlying disease process.  The introduction of the antipseudomonal aminoglycosides and penicillins has improved substantially the prognosis of these infections.  Ticarcillin and carbenicillin have been especially beneficial in neutropenic patients; however, prompt institution of therapy is mandatory for optimal benefit.  Many new drugs with antipseudomonal activity, including penicillins, cephalosporins, and other beta-lactams, have been introduced in recent years and offer the potential for new approaches to therapy for these infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRWY-abeIF33Fm8bhe6roZYfW6udTcc2eZGnAnmgyu-H7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL3s7ps1Ojtg%253D%253D&md5=05e486264586ee11af383e23c8bdb83a</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1093%2Fclinids%2F5.2.279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fclinids%252F5.2.279%26sid%3Dliteratum%253Aachs%26aulast%3DBodey%26aufirst%3DG.%2BP.%26aulast%3DBolivar%26aufirst%3DR.%26aulast%3DFainstein%26aufirst%3DV.%26aulast%3DJadeja%26aufirst%3DL.%26atitle%3DInfections%2520caused%2520by%2520Pseudomonas%2520aeruginosa%26jtitle%3DRev.%2520Infect.%2520Dis.%26date%3D1983%26volume%3D5%26spage%3D279%26epage%3D313%26doi%3D10.1093%2Fclinids%2F5.2.279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Potron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poirel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordmann, P.</span></span> <span> </span><span class="NLM_article-title">Emerging broad-spectrum resistance in Pseudomonas aeruginosa and Acinetobacter baumannii: Mechanisms and epidemiology</span>. <i>Int. J. Antimicrob. Agents</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">568</span>– <span class="NLM_lpage">585</span>, <span class="refDoi"> DOI: 10.1016/j.ijantimicag.2015.03.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1016%2Fj.ijantimicag.2015.03.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=25857949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsFGqsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2015&pages=568-585&author=A.+Potronauthor=L.+Poirelauthor=P.+Nordmann&title=Emerging+broad-spectrum+resistance+in+Pseudomonas+aeruginosa+and+Acinetobacter+baumannii%3A+Mechanisms+and+epidemiology&doi=10.1016%2Fj.ijantimicag.2015.03.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging broad-spectrum resistance in Pseudomonas aeruginosa and Acinetobacter baumannii: Mechanisms and epidemiology</span></div><div class="casAuthors">Potron, Anais; Poirel, Laurent; Nordmann, Patrice</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Antimicrobial Agents</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">568-585</span>CODEN:
                <span class="NLM_cas:coden">IAAGEA</span>;
        ISSN:<span class="NLM_cas:issn">0924-8579</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Multidrug resistance is quite common among non-fermenting Gram-neg. rods, in particular among clin. relevant species including Pseudomonas aeruginosa and Acinetobacter baumannii.  These bacterial species, which are mainly nosocomial pathogens, possess a diversity of resistance mechanisms that may lead to multidrug or even pandrug resistance.  Extended-spectrum β-lactamases (ESBLs) conferring resistance to broad-spectrum cephalosporins, carbapenemases conferring resistance to carbapenems, and 16S rRNA methylases conferring resistance to all clin. relevant aminoglycosides are the most important causes of concern.  Concomitant resistance to fluoroquinolones, polymyxins (colistin) and tigecycline may lead to pandrug resistance.  The most important mechanisms of resistance in P. aeruginosa and A. baumannii and their most recent dissemination worldwide are detailed here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofJ_AESIF1RLVg90H21EOLACvtfcHk0lj4NP1zILQKvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsFGqsLc%253D&md5=112b4b11ae62345c2f0493d66540b4f2</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.ijantimicag.2015.03.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijantimicag.2015.03.001%26sid%3Dliteratum%253Aachs%26aulast%3DPotron%26aufirst%3DA.%26aulast%3DPoirel%26aufirst%3DL.%26aulast%3DNordmann%26aufirst%3DP.%26atitle%3DEmerging%2520broad-spectrum%2520resistance%2520in%2520Pseudomonas%2520aeruginosa%2520and%2520Acinetobacter%2520baumannii%253A%2520Mechanisms%2520and%2520epidemiology%26jtitle%3DInt.%2520J.%2520Antimicrob.%2520Agents%26date%3D2015%26volume%3D45%26spage%3D568%26epage%3D585%26doi%3D10.1016%2Fj.ijantimicag.2015.03.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pendleton, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorman, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilmore, B. F.</span></span> <span> </span><span class="NLM_article-title">Clinical relevance of the ESKAPE pathogens</span>. <i>Expert Rev. Anti-Infect. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">297</span>– <span class="NLM_lpage">308</span>, <span class="refDoi"> DOI: 10.1586/eri.13.12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1586%2Feri.13.12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=23458769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlKisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=297-308&author=J.+N.+Pendletonauthor=S.+P.+Gormanauthor=B.+F.+Gilmore&title=Clinical+relevance+of+the+ESKAPE+pathogens&doi=10.1586%2Feri.13.12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical relevance of the ESKAPE pathogens</span></div><div class="casAuthors">Pendleton, Jack N.; Gorman, Sean P.; Gilmore, Brendan F.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Review of Anti-Infective Therapy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">297-308</span>CODEN:
                <span class="NLM_cas:coden">ERATCK</span>;
        ISSN:<span class="NLM_cas:issn">1478-7210</span>.
    
            (<span class="NLM_cas:orgname">Expert Reviews Ltd.</span>)
        </div><div class="casAbstract">A review.  In recent years, the Infectious Diseases Society of America has highlighted a faction of antibiotic-resistant bacteria (Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa and Enterobacter spp.) - acronymically dubbed the ESKAPE pathogens' - capable of escaping' the biocidal action of antibiotics and mutually representing new paradigms in pathogenesis, transmission and resistance.  This review aims to consolidate clin. relevant background information on the ESKAPE pathogens and provide a contemporary summary of bacterial resistance, alongside pertinent microbiol. considerations necessary to face the mounting threat of antimicrobial resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql_dEmz1UF37Vg90H21EOLACvtfcHk0lhA60xR04WzIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlKisbc%253D&md5=7168500f85d84236a255e20dbf18930a</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1586%2Feri.13.12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1586%252Feri.13.12%26sid%3Dliteratum%253Aachs%26aulast%3DPendleton%26aufirst%3DJ.%2BN.%26aulast%3DGorman%26aufirst%3DS.%2BP.%26aulast%3DGilmore%26aufirst%3DB.%2BF.%26atitle%3DClinical%2520relevance%2520of%2520the%2520ESKAPE%2520pathogens%26jtitle%3DExpert%2520Rev.%2520Anti-Infect.%2520Ther.%26date%3D2013%26volume%3D11%26spage%3D297%26epage%3D308%26doi%3D10.1586%2Feri.13.12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Flemming, H.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neu, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wozniak, D. J.</span></span> <span> </span><span class="NLM_article-title">The EPS matrix: The “house of biofilm cells”</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>189</i></span>,  <span class="NLM_fpage">7945</span>– <span class="NLM_lpage">7947</span>, <span class="refDoi"> DOI: 10.1128/jb.00858-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1128%2Fjb.00858-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=17675377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlyht7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=189&publication_year=2007&pages=7945-7947&author=H.-C.+Flemmingauthor=T.+R.+Neuauthor=D.+J.+Wozniak&title=The+EPS+matrix%3A+The+%E2%80%9Chouse+of+biofilm+cells%E2%80%9D&doi=10.1128%2Fjb.00858-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The EPS matrix: the "house of biofilm cells</span></div><div class="casAuthors">Flemming, Hans-Curt; Neu, Thomas R.; Wozniak, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">189</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7945-7947</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5pGo1ZUBw7Vg90H21EOLACvtfcHk0lhA60xR04WzIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlyht7zN&md5=b2865c83312667d3fff6accd52d19a3d</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1128%2Fjb.00858-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fjb.00858-07%26sid%3Dliteratum%253Aachs%26aulast%3DFlemming%26aufirst%3DH.-C.%26aulast%3DNeu%26aufirst%3DT.%2BR.%26aulast%3DWozniak%26aufirst%3DD.%2BJ.%26atitle%3DThe%2520EPS%2520matrix%253A%2520The%2520%25E2%2580%259Chouse%2520of%2520biofilm%2520cells%25E2%2580%259D%26jtitle%3DJ.%2520Bacteriol.%26date%3D2007%26volume%3D189%26spage%3D7945%26epage%3D7947%26doi%3D10.1128%2Fjb.00858-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costerton, J. W.</span></span> <span> </span><span class="NLM_article-title">Overview of microbial biofilms</span>. <i>J. Ind. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">137</span>– <span class="NLM_lpage">140</span>, <span class="refDoi"> DOI: 10.1007/bf01569816</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1007%2FBF01569816" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=8519468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADyaK2MXovVKmt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1995&pages=137-140&author=J.+W.+Costerton&title=Overview+of+microbial+biofilms&doi=10.1007%2Fbf01569816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of microbial biofilms</span></div><div class="casAuthors">Costerton, J. W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Industrial Microbiology</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">137-40</span>CODEN:
                <span class="NLM_cas:coden">JIMIE7</span>;
        ISSN:<span class="NLM_cas:issn">0169-4146</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Scientific & Medical Division</span>)
        </div><div class="casAbstract">A review with 26 refs., stressing the common features of microbial biofilms that should be remembered as this concept is used to seek to understand bacterial behavior in hundreds of aquatic ecosystems studied by dozens of subspecies of microbiologists reared in sharply different scientific and academic conventions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVZo-nAqeZA7Vg90H21EOLACvtfcHk0lgVmEKeOtjPhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXovVKmt70%253D&md5=c51547fec50900c404293da856b009b0</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1007%2FBF01569816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF01569816%26sid%3Dliteratum%253Aachs%26aulast%3DCosterton%26aufirst%3DJ.%2BW.%26atitle%3DOverview%2520of%2520microbial%2520biofilms%26jtitle%3DJ.%2520Ind.%2520Microbiol.%26date%3D1995%26volume%3D15%26spage%3D137%26epage%3D140%26doi%3D10.1007%2Fbf01569816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">William Costerton, J.</span></span> <span> </span><span class="NLM_article-title">Antibiotic resistance of bacteria in biofilms</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>358</i></span>,  <span class="NLM_fpage">135</span>– <span class="NLM_lpage">138</span>, <span class="refDoi"> DOI: 10.1016/s0140-6736(01)05321-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1016%2FS0140-6736%2801%2905321-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=11463434" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlt1Shsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=358&publication_year=2001&pages=135-138&author=P.+S.+Stewartauthor=J.+William+Costerton&title=Antibiotic+resistance+of+bacteria+in+biofilms&doi=10.1016%2Fs0140-6736%2801%2905321-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotic resistance of bacteria in biofilms</span></div><div class="casAuthors">Stewart, P. S.; William Costerton, J.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">358</span>
        (<span class="NLM_cas:issue">9276</span>),
    <span class="NLM_cas:pages">135-138</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Lancet Ltd.</span>)
        </div><div class="casAbstract">A brief review with 40 refs.  Bacteria that adhere to implanted medical devices or damaged tissue can encase themselves in a hydrated matrix of polysaccharide and protein, and form a slimy layer known as a biofilm.  Antibiotic resistance of bacteria in the biofilm mode of growth contributes to the chronicity of infections such as those assocd. with implanted medical devices.  The mechanisms of resistance in biofilms are different from the now familiar plasmids, transposons, and mutations that confer innate resistance to individual bacterial cells.  In biofilms, resistance seems to depend on multicellular strategies.  We summarize the features of biofilm infections, review emerging mechanisms of resistance, and discuss potential therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonpYgJ4rKbWrVg90H21EOLACvtfcHk0lgVmEKeOtjPhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlt1Shsbs%253D&md5=c97535521d93dc62ae2e2a69556ff0a8</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2801%2905321-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252801%252905321-1%26sid%3Dliteratum%253Aachs%26aulast%3DStewart%26aufirst%3DP.%2BS.%26aulast%3DWilliam%2BCosterton%26aufirst%3DJ.%26atitle%3DAntibiotic%2520resistance%2520of%2520bacteria%2520in%2520biofilms%26jtitle%3DLancet%26date%3D2001%26volume%3D358%26spage%3D135%26epage%3D138%26doi%3D10.1016%2Fs0140-6736%2801%2905321-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bassler, B. L.</span></span> <span> </span><span class="NLM_article-title">How bacteria talk to each other: Regulation of gene expression by quorum sensing</span>. <i>Curr. Opin. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">582</span>– <span class="NLM_lpage">587</span>, <span class="refDoi"> DOI: 10.1016/s1369-5274(99)00025-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1016%2FS1369-5274%2899%2900025-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10607620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BD3cXivF2jsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=1999&pages=582-587&author=B.+L.+Bassler&title=How+bacteria+talk+to+each+other%3A+Regulation+of+gene+expression+by+quorum+sensing&doi=10.1016%2Fs1369-5274%2899%2900025-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">How bacteria talk to each other: regulation of gene expression by quorum sensing</span></div><div class="casAuthors">Bassler, Bonnie L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Microbiology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">582-587</span>CODEN:
                <span class="NLM_cas:coden">COMIF7</span>;
        ISSN:<span class="NLM_cas:issn">1369-5274</span>.
    
            (<span class="NLM_cas:orgname">Current Biology Publications</span>)
        </div><div class="casAbstract">A review with 58 refs.  Quorum sensing, or the control of gene expression in response to cell d., is used by both Gram-neg. and Gram-pos. bacteria to regulate a variety of physiol. functions.  In all cases, quorum sensing involves the prodn. and detection of extracellular signalling mols. called autoinducers.  While universal signalling themes exist, variations in the design of the extracellular signals, the signal detection apparatuses, and the biochem. mechanisms of signal relay have allowed quorum sensing systems to be exquisitely adapted for their varied uses.  Recent studies show that quorum sensing modulates both intra- and inter-species cell-cell communication, and it plays a major role in enabling bacteria to architect complex community structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbtw2G8PkKx7Vg90H21EOLACvtfcHk0lgVmEKeOtjPhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXivF2jsA%253D%253D&md5=e3409a7308c52997a4240d2459bc0e13</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2FS1369-5274%2899%2900025-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1369-5274%252899%252900025-9%26sid%3Dliteratum%253Aachs%26aulast%3DBassler%26aufirst%3DB.%2BL.%26atitle%3DHow%2520bacteria%2520talk%2520to%2520each%2520other%253A%2520Regulation%2520of%2520gene%2520expression%2520by%2520quorum%2520sensing%26jtitle%3DCurr.%2520Opin.%2520Microbiol.%26date%3D1999%26volume%3D2%26spage%3D582%26epage%3D587%26doi%3D10.1016%2Fs1369-5274%2899%2900025-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassler, B. L.</span></span> <span> </span><span class="NLM_article-title">Quorum sensing in bacteria</span>. <i>Annu. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">165</span>– <span class="NLM_lpage">199</span>, <span class="refDoi"> DOI: 10.1146/annurev.micro.55.1.165</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1146%2Fannurev.micro.55.1.165" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=11544353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnslKjt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2001&pages=165-199&author=M.+B.+Millerauthor=B.+L.+Bassler&title=Quorum+sensing+in+bacteria&doi=10.1146%2Fannurev.micro.55.1.165"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Quorum sensing in bacteria</span></div><div class="casAuthors">Miller, Melissa B.; Bassler, Bonnie L.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Microbiology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">165-199</span>CODEN:
                <span class="NLM_cas:coden">ARMIAZ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4227</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Quorum sensing is the regulation of gene expression in response to fluctuations in cell-population d.  Quorum sensing bacteria produce and release chem. signal mols. called autoinducers that increase in concn. as a function of cell d.  The detection of a minimal threshold stimulatory concn. of an autoinducer leads to an alteration in gene expression.  Gram-pos. and Gram-neg. bacteria use quorum sensing communication circuits to regulate a diverse array of physiol. activities.  These processes include symbiosis, virulence, competence, conjugation, antibiotic prodn., motility, sporulation, and biofilm formation.  In general, Gram-neg. bacteria use acylated homoserine lactones as autoinducers, and Gram-pos. bacteria use processed oligo-peptides to communicate.  Recent advances in the field indicate that cell-cell communication via autoinducers occurs both within and between bacterial species.  Furthermore, there is mounting data suggesting that bacterial autoinducers elicit specific responses from host organisms.  Although the nature of the chem. signals, the signal relay mechanisms, and the target genes controlled by bacterial quorum sensing systems differ, in every case the ability to communicate with one another allows bacteria to coordinate the gene expression, and therefore the behavior, of the entire community.  Presumably, this process bestows upon bacteria some of the qualities of higher organisms.  The evolution of quorum sensing systems in bacteria could, therefore, have been one of the early steps in the development of multicellularity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxPVpte_A5T7Vg90H21EOLACvtfcHk0lj6lm1oMtkxgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnslKjt70%253D&md5=55a377efef69610fd36f46ebfa3cb536</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1146%2Fannurev.micro.55.1.165&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.micro.55.1.165%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DM.%2BB.%26aulast%3DBassler%26aufirst%3DB.%2BL.%26atitle%3DQuorum%2520sensing%2520in%2520bacteria%26jtitle%3DAnnu.%2520Rev.%2520Microbiol.%26date%3D2001%26volume%3D55%26spage%3D165%26epage%3D199%26doi%3D10.1146%2Fannurev.micro.55.1.165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iglewski, B. H.</span></span> <span> </span><span class="NLM_article-title">P. aeruginosa quorum-sensing systems and virulence</span>. <i>Curr. Opin. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">56</span>– <span class="NLM_lpage">60</span>, <span class="refDoi"> DOI: 10.1016/s1369-5274(03)00008-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1016%2FS1369-5274%2803%2900008-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=12615220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhs1Kktb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2003&pages=56-60&author=R.+Smithauthor=B.+H.+Iglewski&title=P.+aeruginosa+quorum-sensing+systems+and+virulence&doi=10.1016%2Fs1369-5274%2803%2900008-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">P. aeruginosa quorum sensing systems and virulence</span></div><div class="casAuthors">Smith, Roger S.; Iglewski, Barbara H.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Microbiology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-60</span>CODEN:
                <span class="NLM_cas:coden">COMIF7</span>;
        ISSN:<span class="NLM_cas:issn">1369-5274</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Quorum sensing is an important mechanism for the regulation of genes in many Gram-neg. and Gram-pos. bacteria.  In the opportunistic pathogen Pseudomonas aeruginosa, the absence of one or more components of the quorum-sensing system results in a significant redn. in virulence.  Recent advances in the past year have demonstrated that the quorum-sensing signal mol. 3O-C12-HSL is also a potent stimulator of multiple eukaryotic cells and thus may alter the host response during P. aeruginosa infections.  Therefore, via the regulation of multiple factors and the prodn. of 3O-C12-HSL, quorum-sensing systems have a significant effect on the virulence of the bacteria and also on how the host responds to P. aeruginosa infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYjqAmBbtql7Vg90H21EOLACvtfcHk0lj6lm1oMtkxgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhs1Kktb8%253D&md5=0c8ce2ae45b8d39f6472ccf413d458ea</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2FS1369-5274%2803%2900008-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1369-5274%252803%252900008-0%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DR.%26aulast%3DIglewski%26aufirst%3DB.%2BH.%26atitle%3DP.%2520aeruginosa%2520quorum-sensing%2520systems%2520and%2520virulence%26jtitle%3DCurr.%2520Opin.%2520Microbiol.%26date%3D2003%26volume%3D6%26spage%3D56%26epage%3D60%26doi%3D10.1016%2Fs1369-5274%2803%2900008-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pesci, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seed, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iglewski, B. H.</span></span> <span> </span><span class="NLM_article-title">Regulation of las and rhl quorum sensing in Pseudomonas aeruginosa</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>179</i></span>,  <span class="NLM_fpage">3127</span>– <span class="NLM_lpage">3132</span>, <span class="refDoi"> DOI: 10.1128/jb.179.10.3127-3132.1997</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1128%2Fjb.179.10.3127-3132.1997" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=9150205" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADyaK2sXjtFGlsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=1997&pages=3127-3132&author=E.+C.+Pesciauthor=J.+P.+Pearsonauthor=P.+C.+Seedauthor=B.+H.+Iglewski&title=Regulation+of+las+and+rhl+quorum+sensing+in+Pseudomonas+aeruginosa&doi=10.1128%2Fjb.179.10.3127-3132.1997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of las and rhl quorum sensing in Pseudomonas aeruginosa</span></div><div class="casAuthors">Pesci, Everett; Pearson, James P.; Seed, Patrick C.; Iglewski, Barbara H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">179</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3127-3132</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The prodn. of several virulence factors by Pseudomonas aeruginosa is controlled according to cell d. through two quorum-sensing systems, las and rhl.  The las system is comprised of the transcriptional activator protein LasR and of LasI, which directs the synthesis of the autoinducer PAI-1.  Similarly, the rhl system consists of the transcriptional activator protein RhIR and of RhlI, which directs synthesis of the autoinducer PAI-2 (formerly referred to as factor 2).  To study the interrelation between the two P. aeruginosa quorum-sensing systems, we fused a lacZ reporter gene to lasR, rhlR, and rhlA and monitored expression of these three genes under various conditions.  Our data indicate that lasR and rhlR are expressed in a growth-dependent manner, with activation of each gene occurring during the last half of log-phase growth.  We also show that the las quorum-sensing system controls the rhl quorum-sensing system in two ways.  First, we found that LasR and PAI-1 activated rhlR transcription.  Second, we showed that PAI-1 blocked PAI-2 from binding to RhlR, thereby inhibiting the expression of rhlA.  Our data thus indicate that the las system exerts two levels of control on RhlR, transcriptional and posttranslational.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_cg-ekDpYwbVg90H21EOLACvtfcHk0liy8ff0293WNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjtFGlsLk%253D&md5=fd8dcf9774aec865a6c36bd3b3a0a536</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1128%2Fjb.179.10.3127-3132.1997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fjb.179.10.3127-3132.1997%26sid%3Dliteratum%253Aachs%26aulast%3DPesci%26aufirst%3DE.%2BC.%26aulast%3DPearson%26aufirst%3DJ.%2BP.%26aulast%3DSeed%26aufirst%3DP.%2BC.%26aulast%3DIglewski%26aufirst%3DB.%2BH.%26atitle%3DRegulation%2520of%2520las%2520and%2520rhl%2520quorum%2520sensing%2520in%2520Pseudomonas%2520aeruginosa%26jtitle%3DJ.%2520Bacteriol.%26date%3D1997%26volume%3D179%26spage%3D3127%26epage%3D3132%26doi%3D10.1128%2Fjb.179.10.3127-3132.1997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilder, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diggle, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuster, M.</span></span> <span> </span><span class="NLM_article-title">Cooperation and cheating in Pseudomonas aeruginosa: The roles of the las, rhl and pqs quorum-sensing systems</span>. <i>ISME J.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1332</span>– <span class="NLM_lpage">1343</span>, <span class="refDoi"> DOI: 10.1038/ismej.2011.13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1038%2Fismej.2011.13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=21368905" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptVynurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2011&pages=1332-1343&author=C.+N.+Wilderauthor=S.+P.+Diggleauthor=M.+Schuster&title=Cooperation+and+cheating+in+Pseudomonas+aeruginosa%3A+The+roles+of+the+las%2C+rhl+and+pqs+quorum-sensing+systems&doi=10.1038%2Fismej.2011.13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Cooperation and cheating in Pseudomonas aeruginosa: the roles of the las, rhl and pqs quorum-sensing systems</span></div><div class="casAuthors">Wilder, Cara N.; Diggle, Stephen P.; Schuster, Martin</div><div class="citationInfo"><span class="NLM_cas:title">ISME Journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1332-1343, S1332/1-S1332/9</span>CODEN:
                <span class="NLM_cas:coden">IJSOCF</span>;
        ISSN:<span class="NLM_cas:issn">1751-7362</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Pseudomonas aeruginosa coordinates the transcription of hundreds of genes, including many virulence genes, through three hierarchically arranged quorum-sensing (QS) systems, namely las, rhl and pqs.  Each system consists of genes involved in autoinducer synthesis, lasI, rhlI and pqsABCDH, as well as cognate-regulatory genes, lasR, rhlR and pqsR.  In this study, we analyzed the social behavior of signal-blind (ΔlasR, ΔrhlR, ΔpqsR) and signal-neg. (ΔlasI, ΔrhlI, ΔpqsA) mutants from each QS system.  As each system controls extracellular common goods but differs in the extent of regulatory control, we hypothesized that all signal-blind mutants can behave as cheaters that vary in their ability to invade a QS-proficient population.  We found that lasR and pqsR, but not rhlR, mutants evolve from a wild-type ancestor in vitro under conditions that favor QS.  Accordingly, defined lasR and pqsR mutants enriched in wild-type co-culture, whereas rhlR and all signal-neg. mutants did not.  Both lasR and pqsR mutants enriched with neg. frequency dependence, suggesting social interactions with the wild type, although the pqsR mutant also grew well on its own.  Taken together, the lasR mutant behaved as a typical cheater, as reported previously.  However, the pqsR and rhlR mutants exhibited more complex behaviors, which can be sufficiently explained by pos. and neg. pleiotropic effects through differential regulation of pqs gene expression in the interconnected QS network.  The evolutionary approach adopted here may account for the prevalence of naturally occurring QS mutants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7eGlUKXXMZbVg90H21EOLACvtfcHk0liy8ff0293WNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptVynurw%253D&md5=9a8b62bd7b9825a92a88ed696280fc4f</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fismej.2011.13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fismej.2011.13%26sid%3Dliteratum%253Aachs%26aulast%3DWilder%26aufirst%3DC.%2BN.%26aulast%3DDiggle%26aufirst%3DS.%2BP.%26aulast%3DSchuster%26aufirst%3DM.%26atitle%3DCooperation%2520and%2520cheating%2520in%2520Pseudomonas%2520aeruginosa%253A%2520The%2520roles%2520of%2520the%2520las%252C%2520rhl%2520and%2520pqs%2520quorum-sensing%2520systems%26jtitle%3DISME%2520J.%26date%3D2011%26volume%3D5%26spage%3D1332%26epage%3D1343%26doi%3D10.1038%2Fismej.2011.13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Givskov, M.</span></span> <span> </span><span class="NLM_article-title">Quorum-sensing inhibitors as anti-pathogenic drugs</span>. <i>Int. J. Med. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>296</i></span>,  <span class="NLM_fpage">149</span>– <span class="NLM_lpage">161</span>, <span class="refDoi"> DOI: 10.1016/j.ijmm.2006.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1016%2Fj.ijmm.2006.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=16503194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BD28XksFSru70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=296&publication_year=2006&pages=149-161&author=T.+B.+Rasmussenauthor=M.+Givskov&title=Quorum-sensing+inhibitors+as+anti-pathogenic+drugs&doi=10.1016%2Fj.ijmm.2006.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Quorum-sensing inhibitors as anti-pathogenic drugs</span></div><div class="casAuthors">Rasmussen, Thomas B.; Givskov, Michael</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Medical Microbiology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">296</span>
        (<span class="NLM_cas:issue">2-3</span>),
    <span class="NLM_cas:pages">149-161</span>CODEN:
                <span class="NLM_cas:coden">IMEMFV</span>;
        ISSN:<span class="NLM_cas:issn">1438-4221</span>.
    
            (<span class="NLM_cas:orgname">Elsevier GmbH</span>)
        </div><div class="casAbstract">A review.  Quorum-sensing (QS) signalling systems of pathogens are central regulators for the expression of virulence factors and represent highly attractive targets for the development of novel therapeutics.  In Pseudomonas aeruginosa, QS systems are also involved in elevated antibiotic tolerance of biofilms as well as elevated tolerance to the activity of the innate immune system.  Gram-neg. bacteria commonly use N-acyl homoserine lactones (AHL) as QS signal mols.  The use of signal mol. based drugs to attenuate bacterial pathogenicity rather than bacterial growth is attractive for several reasons, particularly considering the emergence of increasingly antibiotic-resistant bacteria.  Compds. capable of this type of interference have been termed anti-pathogenic drugs.  A large variety of synthetic AHL analogs and natural products libraries have been screened and a no. of QS inhibitors (QSI) have been identified.  Promising QSI compds. have been shown to make biofilms more susceptible to antimicrobial treatments, and are capable of reducing mortality and virulence as well as promoting clearance of bacteria in exptl. animal models of infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqi6hHo5ELXLVg90H21EOLACvtfcHk0lg34TaT7h9BAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XksFSru70%253D&md5=dc7ab0360ef9fe9b8b7bda6c31e2e53d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.ijmm.2006.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijmm.2006.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DRasmussen%26aufirst%3DT.%2BB.%26aulast%3DGivskov%26aufirst%3DM.%26atitle%3DQuorum-sensing%2520inhibitors%2520as%2520anti-pathogenic%2520drugs%26jtitle%3DInt.%2520J.%2520Med.%2520Microbiol.%26date%3D2006%26volume%3D296%26spage%3D149%26epage%3D161%26doi%3D10.1016%2Fj.ijmm.2006.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noto, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nichols-O’Neill, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, L. J.</span></span> <span> </span><span class="NLM_article-title">Potent irreversible inhibitors of LasR quorum sensing in Pseudomonas aeruginosa</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">162</span>– <span class="NLM_lpage">167</span>, <span class="refDoi"> DOI: 10.1021/ml500459f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500459f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFyhtL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=162-167&author=K.+T.+O%E2%80%99Brienauthor=J.+G.+Notoauthor=L.+Nichols-O%E2%80%99Neillauthor=L.+J.+Perez&title=Potent+irreversible+inhibitors+of+LasR+quorum+sensing+in+Pseudomonas+aeruginosa&doi=10.1021%2Fml500459f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Potent Irreversible Inhibitors of LasR Quorum Sensing in Pseudomonas aeruginosa</span></div><div class="casAuthors">O'Brien, Kevin T.; Noto, Joseph G.; Nichols-O'Neill, Luke; Perez, Lark J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">162-167</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Antagonism of quorum sensing represents a promising new antivirulence approach for the treatment of bacterial infection.  The development of a novel series of non-natural irreversible antagonists of P. aeruginosa LasR is described.  The lead compds. identified (25 and 28) display potent LasR antagonist activity and inhibit expression of the P. aeruginosa virulence factors pyocyanin and biofilm formation in PAO1 and PA14.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs_0FnrDKcmbVg90H21EOLACvtfcHk0lg34TaT7h9BAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFyhtL%252FN&md5=affb26c8bcb4e9db5158491fb8c83c2a</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fml500459f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500459f%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DK.%2BT.%26aulast%3DNoto%26aufirst%3DJ.%2BG.%26aulast%3DNichols-O%25E2%2580%2599Neill%26aufirst%3DL.%26aulast%3DPerez%26aufirst%3DL.%2BJ.%26atitle%3DPotent%2520irreversible%2520inhibitors%2520of%2520LasR%2520quorum%2520sensing%2520in%2520Pseudomonas%2520aeruginosa%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D162%26epage%3D167%26doi%3D10.1021%2Fml500459f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Geske, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattmann, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackwell, H. E.</span></span> <span> </span><span class="NLM_article-title">Evaluation of a focused library of N-aryl L-homoserine lactones reveals a new set of potent quorum sensing modulators</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">5978</span>– <span class="NLM_lpage">5981</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2008.07.089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1016%2Fj.bmcl.2008.07.089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=18760602" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOht7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=5978-5981&author=G.+D.+Geskeauthor=M.+E.+Mattmannauthor=H.+E.+Blackwell&title=Evaluation+of+a+focused+library+of+N-aryl+L-homoserine+lactones+reveals+a+new+set+of+potent+quorum+sensing+modulators&doi=10.1016%2Fj.bmcl.2008.07.089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of a focused library of N-aryl -homoserine lactones reveals a new set of potent quorum sensing modulators</span></div><div class="casAuthors">Geske, Grant D.; Mattmann, Margrith E.; Blackwell, Helen E.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5978-5981</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A focused library of N-aryl -homoserine lactones was designed around known lactone leads and evaluated for antagonistic and agonistic activity against quorum-sensing receptors in Agrobacterium tumefaciens, Pseudomonas aeruginosa, and Vibrio fischeri.  Several compds. were identified with significantly heightened activities relative to the lead compds., and new structure-activity relationships (SARs) were delineated.  Notably, 4-substituted N-phenoxyacetyl and 3-substituted N-phenylpropionyl -homoserine lactones were identified as potent antagonists of TraR and LuxR, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZBUYV5E_TQbVg90H21EOLACvtfcHk0lgyswLd0AQ1EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOht7jK&md5=b099166a3f71a7e8504ed440d85253d3</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.07.089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.07.089%26sid%3Dliteratum%253Aachs%26aulast%3DGeske%26aufirst%3DG.%2BD.%26aulast%3DMattmann%26aufirst%3DM.%2BE.%26aulast%3DBlackwell%26aufirst%3DH.%2BE.%26atitle%3DEvaluation%2520of%2520a%2520focused%2520library%2520of%2520N-aryl%2520L-homoserine%2520lactones%2520reveals%2520a%2520new%2520set%2520of%2520potent%2520quorum%2520sensing%2520modulators%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D5978%26epage%3D5981%26doi%3D10.1016%2Fj.bmcl.2008.07.089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McInnis, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackwell, H. E.</span></span> <span> </span><span class="NLM_article-title">Thiolactone modulators of quorum sensing revealed through library design and screening</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">4820</span>– <span class="NLM_lpage">4828</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2011.06.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1016%2Fj.bmc.2011.06.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=21798746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvFWiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=4820-4828&author=C.+E.+McInnisauthor=H.+E.+Blackwell&title=Thiolactone+modulators+of+quorum+sensing+revealed+through+library+design+and+screening&doi=10.1016%2Fj.bmc.2011.06.071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Thiolactone modulators of quorum sensing revealed through library design and screening</span></div><div class="casAuthors">McInnis, Christine E.; Blackwell, Helen E.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4820-4828</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Quorum sensing (QS) is a process by which bacteria use small mols. or peptidic signals to assess their local population densities.  At sufficiently high d., bacteria can alter gene expression levels to regulate group behaviors involved in a range of important and diverse phenotypes, including virulence factor prodn., biofilm formation, root nodulation, and bioluminescence.  Gram-neg. bacteria most commonly use N-acylated L-homoserine lactones (AHLs) as their QS signals.  The AHL lactone ring is hydrolyzed relatively rapidly at biol. pH, and the ring-opened product is QS inactive.  The authors sought to identify AHL analogs with heightened hydrolytic stability, and thereby potentially heightened activity, for use as non-native modulators of bacterial QS.  As part of this effort, they probed the utility of thiolactone analogs in the current study as QS agonists and antagonists in Gram-neg. bacteria.  A focused library of thiolactone analogs was designed and rapidly synthesized in soln.  The authors examd. the activity of the library as agonists and antagonists of LuxR-type QS receptors in Pseudomonas aeruginosa (LasR), Vibrio fischeri (LuxR), and Agrobacterium tumefaciens (TraR) using bacterial reporter strains.  The thiolactone library contained several highly active compds., including some of the most active LuxR inhibitors and the most active synthetic TraR agonist reported to date.  Anal. of a representative thiolactone analog revealed that its hydrolysis half-life was almost double that of its parent AHL in bacterial growth medium.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMrPNBC5qdAbVg90H21EOLACvtfcHk0lgyswLd0AQ1EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvFWiurg%253D&md5=8c51d9121e29019218e48b6403d48684</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.06.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.06.071%26sid%3Dliteratum%253Aachs%26aulast%3DMcInnis%26aufirst%3DC.%2BE.%26aulast%3DBlackwell%26aufirst%3DH.%2BE.%26atitle%3DThiolactone%2520modulators%2520of%2520quorum%2520sensing%2520revealed%2520through%2520library%2520design%2520and%2520screening%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D4820%26epage%3D4828%26doi%3D10.1016%2Fj.bmc.2011.06.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oshri, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zrihen, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shner, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omer Bendori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eldar, A.</span></span> <span> </span><span class="NLM_article-title">Selection for increased quorum-sensing cooperation in Pseudomonas aeruginosa through the shut-down of a drug resistance pump</span>. <i>ISME J.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2458</span>– <span class="NLM_lpage">2469</span>, <span class="refDoi"> DOI: 10.1038/s41396-018-0205-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1038%2Fs41396-018-0205-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=29925881" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFyjtL%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2018&pages=2458-2469&author=R.+D.+Oshriauthor=K.+S.+Zrihenauthor=I.+Shnerauthor=S.+Omer+Bendoriauthor=A.+Eldar&title=Selection+for+increased+quorum-sensing+cooperation+in+Pseudomonas+aeruginosa+through+the+shut-down+of+a+drug+resistance+pump&doi=10.1038%2Fs41396-018-0205-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Selection for increased quorum-sensing cooperation in Pseudomonas aeruginosa through the shut-down of a drug resistance pump</span></div><div class="casAuthors">Oshri, Ron D.; Zrihen, Keren S.; Shner, Itzhak; Omer Bendori, Shira; Eldar, Avigdor</div><div class="citationInfo"><span class="NLM_cas:title">ISME Journal</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2458-2469</span>CODEN:
                <span class="NLM_cas:coden">IJSOCF</span>;
        ISSN:<span class="NLM_cas:issn">1751-7362</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The opportunistic pathogen Pseudomonas aeruginosa employs a hierarchical quorum-sensing network to regulate virulence factor prodn. that cooperatively benefit the population at a cost to the individual.  It has been argued that the evolution of a cooperative mutant in a quorum sensing-suppressed population would be hampered through its exploitation by neighboring non-mutant cells.  It remains unclear whether mechanisms which overcome this exploitation exist.  Here we investigate the regain of quorum-sensing cooperation by evolving a mutant of the lasR master quorum-sensing regulator.  The mutant regained partial cooperative growth through null mutations in mexT, which codes for an activator of the MexEF-OprN multidrug-resistant pump.  We find that these mutations enhance cooperative growth in both the lasR mutant and wild-type backgrounds through the activation of the RhlIR system.  We show that the regain of cooperation in mexT mutants is mediated by the redn. in MexEF-OprN activity, whereas an addnl. source of private benefit is mostly mexEF-oprN-independent.  Finally, we show that addn. of antibiotics for which resistance is mediated by MexEF-OprN prevents the selection of increased cooperation at sub-MIC concns.  MexT, therefore, not only links private and public goods, but also exposes conflicts between selection for antibiotic resistance and enhanced cooperation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoG8GawxJ1EzrVg90H21EOLACvtfcHk0lhEo_rAQRcbfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFyjtL%252FF&md5=fbf6aefdfb543702ec8349ba652637ba</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fs41396-018-0205-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41396-018-0205-y%26sid%3Dliteratum%253Aachs%26aulast%3DOshri%26aufirst%3DR.%2BD.%26aulast%3DZrihen%26aufirst%3DK.%2BS.%26aulast%3DShner%26aufirst%3DI.%26aulast%3DOmer%2BBendori%26aufirst%3DS.%26aulast%3DEldar%26aufirst%3DA.%26atitle%3DSelection%2520for%2520increased%2520quorum-sensing%2520cooperation%2520in%2520Pseudomonas%2520aeruginosa%2520through%2520the%2520shut-down%2520of%2520a%2520drug%2520resistance%2520pump%26jtitle%3DISME%2520J.%26date%3D2018%26volume%3D12%26spage%3D2458%26epage%3D2469%26doi%3D10.1038%2Fs41396-018-0205-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Induction and inhibition of Pseudomonas aeruginosa quorum sensing by synthetic autoinducer analogs</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">81</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1016/s1074-5521(03)00002-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1016%2FS1074-5521%2803%2900002-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=12573701" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BD3sXps1Srtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2003&pages=81-89&author=K.+M.+Smithauthor=Y.+Buauthor=H.+Suga&title=Induction+and+inhibition+of+Pseudomonas+aeruginosa+quorum+sensing+by+synthetic+autoinducer+analogs&doi=10.1016%2Fs1074-5521%2803%2900002-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Induction and Inhibition of Pseudomonas aeruginosa Quorum Sensing by Synthetic Autoinducer Analogs</span></div><div class="casAuthors">Smith, Kristina M.; Bu, Yigong; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">81-89</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We synthesized a library of Pseudomonas aeruginosa autoinducer analogs with variation targeted to the homoserine lactone (HSL) moiety and discovered a new agonist, 3-oxo-C12-(2-aminocyclohexanol), capable of activating LasR as a transcription factor.  We reconstructed two sets of focused libraries against the quorum-sensing transcription factors LasR and RhlR, resp.  Opposing the prediction that both proteins should have the same binding site for HSL, it was surprising to find that these two related proteins respond to different structural motifs.  This suggests that the HSL binding site differs in these proteins.  We also found that subtle structural modifications to the agonists yielded compds. with antagonist activity.  We performed a series of assays to show that inhibition of quorum sensing by these antagonists significantly reduced the prodn. of virulence factors and biofilm formation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKQ50GqXLterVg90H21EOLACvtfcHk0lhEo_rAQRcbfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXps1Srtw%253D%253D&md5=322f1933c805207386bf14169398b93d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2FS1074-5521%2803%2900002-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1074-5521%252803%252900002-4%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DK.%2BM.%26aulast%3DBu%26aufirst%3DY.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DInduction%2520and%2520inhibition%2520of%2520Pseudomonas%2520aeruginosa%2520quorum%2520sensing%2520by%2520synthetic%2520autoinducer%2520analogs%26jtitle%3DChem.%2520Biol.%26date%3D2003%26volume%3D10%26spage%3D81%26epage%3D89%26doi%3D10.1016%2Fs1074-5521%2803%2900002-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Loughlin, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siryaporn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drescher, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semmelhack, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassler, B. L.</span></span> <span> </span><span class="NLM_article-title">A quorum-sensing inhibitor blocks Pseudomonas aeruginosa virulence and biofilm formation</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">17981</span>– <span class="NLM_lpage">17986</span>, <span class="refDoi"> DOI: 10.1073/pnas.1316981110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1073%2Fpnas.1316981110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=24143808" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVWmur3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=17981-17986&author=C.+T.+O%E2%80%99Loughlinauthor=L.+C.+Millerauthor=A.+Siryapornauthor=K.+Drescherauthor=M.+F.+Semmelhackauthor=B.+L.+Bassler&title=A+quorum-sensing+inhibitor+blocks+Pseudomonas+aeruginosa+virulence+and+biofilm+formation&doi=10.1073%2Fpnas.1316981110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">A quorum sensing inhibitor blocks Pseudomonas aeruginosa virulence and biofilm formation</span></div><div class="casAuthors">O'Loughlin, Colleen T.; Miller, Laura C.; Siryaporn, Albert; Drescher, Knut; Semmelhack, Martin F.; Bassler, Bonnie L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">17981-17986,S17981/1-S17981/56</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Quorum sensing is a chem. communication process that bacteria use to regulate collective behaviors.  Disabling quorum-sensing circuits with small mols. has been proposed as a potential strategy to prevent bacterial pathogenicity.  The human pathogen Pseudomonas aeruginosa uses quorum sensing to control virulence and biofilm formation.  Here, the authors analyze synthetic mols. for inhibition of the two P. aeruginosa quorum sensing receptors, LasR and RhlR.  The most effective compd., meta-bromo-thiolactone (mBTL), inhibits both the prodn. of the virulence factor pyocyanin and biofilm formation.  The mBTL also protects Caenorhabditis elegans and human lung epithelial cells from killing by P. aeruginosa.  Both LasR and RhlR are partially inhibited by mBTL in vivo and in vitro; however, RhlR, not LasR, is the relevant in vivo target.  More potent antagonists do not exhibit superior function in impeding virulence.  Because LasR and RhlR reciprocally control crucial virulence factors, appropriately tuning rather than completely inhibiting their activities appears to hold the key to blocking pathogenesis in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq34j__e2GFKrVg90H21EOLACvtfcHk0lhEo_rAQRcbfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVWmur3F&md5=efa2b960af182ca3945d81bf70d2a37b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1316981110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1316981110%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Loughlin%26aufirst%3DC.%2BT.%26aulast%3DMiller%26aufirst%3DL.%2BC.%26aulast%3DSiryaporn%26aufirst%3DA.%26aulast%3DDrescher%26aufirst%3DK.%26aulast%3DSemmelhack%26aufirst%3DM.%2BF.%26aulast%3DBassler%26aufirst%3DB.%2BL.%26atitle%3DA%2520quorum-sensing%2520inhibitor%2520blocks%2520Pseudomonas%2520aeruginosa%2520virulence%2520and%2520biofilm%2520formation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D17981%26epage%3D17986%26doi%3D10.1073%2Fpnas.1316981110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Furiga, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lajoie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Hage, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baziard, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roques, C.</span></span> <span> </span><span class="NLM_article-title">Impairment of Pseudomonas aeruginosa biofilm resistance to antibiotics by combining the drugs with a new quorum-sensing inhibitor</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1676</span>– <span class="NLM_lpage">1686</span>, <span class="refDoi"> DOI: 10.1128/aac.02533-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1128%2FAAC.02533-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsF2ntbjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2016&pages=1676-1686&author=A.+Furigaauthor=B.+Lajoieauthor=S.+El+Hageauthor=G.+Baziardauthor=C.+Roques&title=Impairment+of+Pseudomonas+aeruginosa+biofilm+resistance+to+antibiotics+by+combining+the+drugs+with+a+new+quorum-sensing+inhibitor&doi=10.1128%2Faac.02533-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Impairment of Pseudomonas aeruginosa biofilm resistance to antibiotics by combining the drugs with a new quorum-sensing inhibitor</span></div><div class="casAuthors">Furiga, Aurelie; Lajoie, Barbora; El Hage, Salome; Baziard, Genevieve; Roques, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1676-1686</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Pseudomonas aeruginosa plays an important role in chronic lung infections among patients with cystic fibrosis (CF) through its ability to form antibiotic-resistant biofilms.  In P. aeruginosa, biofilm development and the prodn. of several virulence factors are mainly regulated by the rhl and las quorum-sensing (QS) systems, which are controlled by two N-acyl-homoserine lactone signal mols.  In a previous study, we discovered an original QS inhibitor, N-(2-pyrimidyl)butanamide, called C11, based on the structure of C4-homoserine lactone, and found that it is able to significantly inhibit P. aeruginosa biofilm formation.  However, recent data indicate that P. aeruginosa grows under anaerobic conditions and forms biofilms in the lungs of CF patients that are denser and more robust than those formed under aerobic conditions.  Our confocal microscopy observations of P. aeruginosa biofilms developed under aerobic and anaerobic conditions confirmed that the biofilms formed under these 2 conditions have radically different architectures.  C11 showed significant dose-dependent antibiofilm activity on biofilms grown under both aerobic and anaerobic conditions, with a greater inhibitory effect being seen under conditions of anaerobiosis.  Gene expression analyses performed by quant. reverse transcriptase PCR showed that C11 led to the significant downregulation of rhl QS regulatory genes but also to the downregulation of both las QS regulatory genes and QS system-regulated virulence genes, rhlA and lasB.  Furthermore, the activity of C11 in combination with antibiotics against P. aeruginosa biofilms was tested, and synergistic antibiofilm activity between C11 and ciprofloxacin, tobramycin, and colistin was obtained under both aerobic and anaerobic conditions.  This study demonstrates that C11 may increase the efficacy of treatments for P. aeruginosa infections by increasing the susceptibility of biofilms to antibiotics and by attenuating the pathogenicity of the bacterium.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-yOX6KkutobVg90H21EOLACvtfcHk0ljTJZj_tib6Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsF2ntbjI&md5=367c1cf09e9fe5d032071f44bc3cf915</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1128%2FAAC.02533-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.02533-15%26sid%3Dliteratum%253Aachs%26aulast%3DFuriga%26aufirst%3DA.%26aulast%3DLajoie%26aufirst%3DB.%26aulast%3DEl%2BHage%26aufirst%3DS.%26aulast%3DBaziard%26aufirst%3DG.%26aulast%3DRoques%26aufirst%3DC.%26atitle%3DImpairment%2520of%2520Pseudomonas%2520aeruginosa%2520biofilm%2520resistance%2520to%2520antibiotics%2520by%2520combining%2520the%2520drugs%2520with%2520a%2520new%2520quorum-sensing%2520inhibitor%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2016%26volume%3D60%26spage%3D1676%26epage%3D1686%26doi%3D10.1128%2Faac.02533-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soukarieh, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stocks, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cámara, M.</span></span> <span> </span><span class="NLM_article-title">Pseudomonas aeruginosa quorum sensing systems as drug discovery targets: Current position and future perspectives</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">10385</span>– <span class="NLM_lpage">10402</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00540</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00540" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlSmurvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=10385-10402&author=F.+Soukariehauthor=P.+Williamsauthor=M.+J.+Stocksauthor=M.+C%C3%A1mara&title=Pseudomonas+aeruginosa+quorum+sensing+systems+as+drug+discovery+targets%3A+Current+position+and+future+perspectives&doi=10.1021%2Facs.jmedchem.8b00540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Pseudomonas aeruginosa Quorum Sensing Systems as Drug Discovery Targets: Current Position and Future Perspectives</span></div><div class="casAuthors">Soukarieh, Fadi; Williams, Paul; Stocks, Michael J.; Camara, Miguel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10385-10402</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Antimicrobial resistance (AMR) is a serious threat to public health globally, manifested by the frequent emergence of multi-drug resistant pathogens that render current chemotherapy inadequate.  Health organizations worldwide have recognized the severity of this crisis and implemented action plans to contain its adverse consequences and prolong the utility of conventional antibiotics.  Hence, there is a pressing need for new classes of antibacterial agents with novel modes of action.  Quorum sensing (QS), a communication system employed by bacterial populations to coordinate virulence gene expression, is a potential target that has been intensively investigated over the last decade.  This Perspective will focus on recent advances in targeting the three main quorum sensing systems (las, rhl and pqs) of a major opportunistic human pathogen, Pseudomonas aeruginosa, and will specifically evaluate the medicinal chem. strategies devised to develop QS inhibitors from a drug discovery perspective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKqjOmBMF_u7Vg90H21EOLACvtfcHk0ljTJZj_tib6Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlSmurvE&md5=ce4ff141208ec755b91345d7a1fae049</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00540%26sid%3Dliteratum%253Aachs%26aulast%3DSoukarieh%26aufirst%3DF.%26aulast%3DWilliams%26aufirst%3DP.%26aulast%3DStocks%26aufirst%3DM.%2BJ.%26aulast%3DC%25C3%25A1mara%26aufirst%3DM.%26atitle%3DPseudomonas%2520aeruginosa%2520quorum%2520sensing%2520systems%2520as%2520drug%2520discovery%2520targets%253A%2520Current%2520position%2520and%2520future%2520perspectives%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D10385%26epage%3D10402%26doi%3D10.1021%2Facs.jmedchem.8b00540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eibergen, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattmann, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackwell, H. E.</span></span> <span> </span><span class="NLM_article-title">Potent and selective modulation of the RhlR quorum sensing receptor by using non-native ligands: An emerging target for virulence control in Pseudomonas aeruginosa</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">2348</span>– <span class="NLM_lpage">2356</span>, <span class="refDoi"> DOI: 10.1002/cbic.201500357</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1002%2Fcbic.201500357" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=26460240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1egs7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=2348-2356&author=N.+R.+Eibergenauthor=J.+D.+Mooreauthor=M.+E.+Mattmannauthor=H.+E.+Blackwell&title=Potent+and+selective+modulation+of+the+RhlR+quorum+sensing+receptor+by+using+non-native+ligands%3A+An+emerging+target+for+virulence+control+in+Pseudomonas+aeruginosa&doi=10.1002%2Fcbic.201500357"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and Selective Modulation of the RhlR Quorum Sensing Receptor by Using Non-native Ligands: An Emerging Target for Virulence Control in Pseudomonas aeruginosa</span></div><div class="casAuthors">Eibergen, Nora R.; Moore, Joseph D.; Mattmann, Margrith E.; Blackwell, Helen E.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2348-2356</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Pseudomonas aeruginosa uses N-acylated L-homoserine lactone signals and a triumvirate of LuxR-type receptor proteins (LasR, RhlR, and QscR) for quorum sensing (QS).  Each of these receptors can contribute to QS activation or repression and, thereby, the control of myriad virulence phenotypes in this pathogen.  LasR has traditionally been considered to be at the top of the QS receptor hierarchy in P. aeruginosa; however, recent reports suggest that RhlR plays a more prominent role in infection than originally predicted, in some circumstances superseding that of LasR.  Herein, the authors report the characterization of a set of synthetic, small-mol. agonists and antagonists of RhlR.  Using E. coli reporter strains, it was shown that many of these compds. can selectively activate or inhibit RhlR instead of LasR and QscR.  Moreover, several mols. maintain their activities in P. aeruginosa at concns. analogous to native RhlR signal levels.  These compds. represent useful chem. probes to study the role of RhlR in the complex QS circuitry of P. aeruginosa, its direct (and indirect) effects on virulence, and its overall merit as a target for anti-infective therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaLc7CY8sMxLVg90H21EOLACvtfcHk0lhLiQZXCcbHVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1egs7%252FE&md5=5299d6c73bd982144096752d1b08dc5a</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201500357&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201500357%26sid%3Dliteratum%253Aachs%26aulast%3DEibergen%26aufirst%3DN.%2BR.%26aulast%3DMoore%26aufirst%3DJ.%2BD.%26aulast%3DMattmann%26aufirst%3DM.%2BE.%26aulast%3DBlackwell%26aufirst%3DH.%2BE.%26atitle%3DPotent%2520and%2520selective%2520modulation%2520of%2520the%2520RhlR%2520quorum%2520sensing%2520receptor%2520by%2520using%2520non-native%2520ligands%253A%2520An%2520emerging%2520target%2520for%2520virulence%2520control%2520in%2520Pseudomonas%2520aeruginosa%26jtitle%3DChemBioChem%26date%3D2015%26volume%3D16%26spage%3D2348%26epage%3D2356%26doi%3D10.1002%2Fcbic.201500357" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Library screening for synthetic agonists and antagonists of a Pseudomonas aeruginosa autoinducer</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">563</span>– <span class="NLM_lpage">571</span>, <span class="refDoi"> DOI: 10.1016/s1074-5521(03)00107-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1016%2Fs1074-5521%2803%2900107-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=12837389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BD3sXltVSqsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2003&pages=563-571&author=K.+M.+Smithauthor=Y.+Buauthor=H.+Suga&title=Library+screening+for+synthetic+agonists+and+antagonists+of+a+Pseudomonas+aeruginosa+autoinducer&doi=10.1016%2Fs1074-5521%2803%2900107-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Library Screening for Synthetic Agonists and Antagonists of a Pseudomonas aeruginosa Autoinducer</span></div><div class="casAuthors">Smith, Kristina M.; Bu, Yigong; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">563-571</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The autoinducer (AI) that initiates the quorum sensing (QS) signaling cascade in Pseudomonas aeruginosa is an acyl-homoserine lactone (acyl-HSL).  We initiated a study of the requirements for binding of the AI to its protein effector LasR by synthesizing a library of analogs with the HSL moiety replaced with different amines and alcs.  We tested each compd. for both agonist and antagonist activity using a QS-controlled reporter gene assay and found several new agonists and antagonists.  A representative antagonist was further tested for its ability to inhibit virulence factors.  This data progresses our understanding of the LasR-AI interaction toward the rational design of therapeutic inhibitors of QS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMFP7aVBkHYbVg90H21EOLACvtfcHk0lhLiQZXCcbHVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXltVSqsr4%253D&md5=f3c37fa88ce2b3de8d75603cded4a8e4</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fs1074-5521%2803%2900107-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs1074-5521%252803%252900107-8%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DK.%2BM.%26aulast%3DBu%26aufirst%3DY.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DLibrary%2520screening%2520for%2520synthetic%2520agonists%2520and%2520antagonists%2520of%2520a%2520Pseudomonas%2520aeruginosa%2520autoinducer%26jtitle%3DChem.%2520Biol.%26date%3D2003%26volume%3D10%26spage%3D563%26epage%3D571%26doi%3D10.1016%2Fs1074-5521%2803%2900107-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Annapoorani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umamageswaran, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parameswari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandian, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravi, A. V.</span></span> <span> </span><span class="NLM_article-title">Computational discovery of putative quorum sensing inhibitors against LasR and RhlR receptor proteins of Pseudomonas aeruginosa</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1077</span>, <span class="refDoi"> DOI: 10.1007/s10822-012-9599-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1007%2Fs10822-012-9599-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=22986632" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFWkt7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=1067-1077&author=A.+Annapooraniauthor=V.+Umamageswaranauthor=R.+Parameswariauthor=S.+K.+Pandianauthor=A.+V.+Ravi&title=Computational+discovery+of+putative+quorum+sensing+inhibitors+against+LasR+and+RhlR+receptor+proteins+of+Pseudomonas+aeruginosa&doi=10.1007%2Fs10822-012-9599-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Computational discovery of putative quorum sensing inhibitors against LasR and RhlR receptor proteins of Pseudomonas aeruginosa</span></div><div class="casAuthors">Annapoorani, Angusamy; Umamageswaran, Venugopal; Parameswari, Radhakrishnan; Pandian, Shunmugiah Karutha; Ravi, Arumugam Veera</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1067-1077</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Drugs have been discovered in the past mainly either by identification of active components from traditional remedies or by unpredicted discovery.  A key motivation for the study of structure based virtual screening is the exploitation of such information to design targeted drugs.  In this study, structure-based virtual screening was used in the search for putative quorum sensing inhibitors (QSI) of Pseudomonas aeruginosa.  The virtual screening program Glide version 5.5 was applied to screen 1,920 natural compds./drugs against LasR and RhlR receptor proteins of P. aeruginosa.  Based on the results of in silico docking anal., five top ranking compds. namely rosmarinic acid, naringin, chlorogenic acid, morin and mangiferin were subjected to in vitro bioassays against lab. strain PAO1 and two more antibiotic-resistant clin. isolates, P. aeruginosa AS1 (GU447237) and P. aeruginosa AS2 (GU447238).  Among 5 compds. studied, all but mangiferin showed significant inhibition in the prodn. of protease, elastase and hemolysin.  Further, all five compds. potentially inhibited biofilm-related behaviors.  This interaction study provided promising ligands to inhibit the quorum sensing- mediated virulence factor prodn. in P. aeruginosa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJ4R1ilIt18rVg90H21EOLACvtfcHk0liszXXiH29YkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFWkt7%252FI&md5=d230c270761f761d6e1a3f8768b46ad5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9599-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9599-1%26sid%3Dliteratum%253Aachs%26aulast%3DAnnapoorani%26aufirst%3DA.%26aulast%3DUmamageswaran%26aufirst%3DV.%26aulast%3DParameswari%26aufirst%3DR.%26aulast%3DPandian%26aufirst%3DS.%2BK.%26aulast%3DRavi%26aufirst%3DA.%2BV.%26atitle%3DComputational%2520discovery%2520of%2520putative%2520quorum%2520sensing%2520inhibitors%2520against%2520LasR%2520and%2520RhlR%2520receptor%2520proteins%2520of%2520Pseudomonas%2520aeruginosa%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26spage%3D1067%26epage%3D1077%26doi%3D10.1007%2Fs10822-012-9599-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shepardson-Fungairino, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Combs, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagarajan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackwell, H. E.</span></span> <span> </span><span class="NLM_article-title">Structure-function analyses of the N-butanoyl L-homoserine lactone quorum-sensing signal define features critical to activity in RhlR</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2655</span>– <span class="NLM_lpage">2662</span>, <span class="refDoi"> DOI: 10.1021/acschembio.8b00577</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b00577" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFait7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=2655-2662&author=M.+E.+Boursierauthor=J.+D.+Mooreauthor=K.+M.+Heitmanauthor=S.+P.+Shepardson-Fungairinoauthor=J.+B.+Combsauthor=L.+C.+Koenigauthor=D.+Shinauthor=E.+C.+Brownauthor=R.+Nagarajanauthor=H.+E.+Blackwell&title=Structure-function+analyses+of+the+N-butanoyl+L-homoserine+lactone+quorum-sensing+signal+define+features+critical+to+activity+in+RhlR&doi=10.1021%2Facschembio.8b00577"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Function Analyses of the N-Butanoyl L-Homoserine Lactone Quorum-Sensing Signal Define Features Critical to Activity in RhlR</span></div><div class="casAuthors">Boursier, Michelle E.; Moore, Joseph D.; Heitman, Katherine M.; Shepardson-Fungairino, Sally P.; Combs, Joshua B.; Koenig, Lea C.; Shin, Daniel; Brown, Eric C.; Nagarajan, Rajesh; Blackwell, Helen E.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2655-2662</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pseudomonas aeruginosa is an opportunistic pathogen that coordinates the prodn. of many virulence phenotypes at high population d. via quorum sensing (QS).  The LuxR-type receptor RhlR plays an important role in the P. aeruginosa QS process, and there is considerable interest in the development of chem. approaches to modulate the activity of this protein.  RhlR is activated by a simple, low mol. wt. N-acyl L-homoserine lactone signal, N-butanoyl-L-homoserine lactone (BHL).  Despite the emerging prominence of RhlR in QS pathways, there has been limited exploration of the chem. features of the BHL scaffold that are crit. to its function.  In the current study, we sought to systematically delineate the structure-activity relationships (SARs) driving BHL activity for the first time.  A focused library of BHL analogs was designed, synthesized, and evaluated in cell-based reporter gene assays for RhlR agonism and antagonism.  These investigations allowed us to define a series of SARs for BHL-type ligands and identify structural motifs crit. for both activation and inhibition of the RhlR receptor.  Notably, we identified agonists that have ∼10-fold higher potencies in RhlR relative to BHL, are highly selective for RhlR agonism over LasR, and are active in the P. aeruginosa background.  These compds. and the SARs reported herein should pave a route toward new chem. strategies to study RhlR in P. aeruginosa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2CSUdtm4ISbVg90H21EOLACvtfcHk0liszXXiH29YkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFait7bF&md5=bb38808c632048f6967c4e5f8f695842</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b00577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b00577%26sid%3Dliteratum%253Aachs%26aulast%3DBoursier%26aufirst%3DM.%2BE.%26aulast%3DMoore%26aufirst%3DJ.%2BD.%26aulast%3DHeitman%26aufirst%3DK.%2BM.%26aulast%3DShepardson-Fungairino%26aufirst%3DS.%2BP.%26aulast%3DCombs%26aufirst%3DJ.%2BB.%26aulast%3DKoenig%26aufirst%3DL.%2BC.%26aulast%3DShin%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DE.%2BC.%26aulast%3DNagarajan%26aufirst%3DR.%26aulast%3DBlackwell%26aufirst%3DH.%2BE.%26atitle%3DStructure-function%2520analyses%2520of%2520the%2520N-butanoyl%2520L-homoserine%2520lactone%2520quorum-sensing%2520signal%2520define%2520features%2520critical%2520to%2520activity%2520in%2520RhlR%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D2655%26epage%3D2662%26doi%3D10.1021%2Facschembio.8b00577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Welsh, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eibergen, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackwell, H. E.</span></span> <span> </span><span class="NLM_article-title">Small molecule disruption of quorum sensing cross-regulation in Pseudomonas aeruginosa causes major and unexpected alterations to virulence phenotypes</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">1510</span>– <span class="NLM_lpage">1519</span>, <span class="refDoi"> DOI: 10.1021/ja5110798</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja5110798" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC2MXns12qtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=1510-1519&author=M.+A.+Welshauthor=N.+R.+Eibergenauthor=J.+D.+Mooreauthor=H.+E.+Blackwell&title=Small+molecule+disruption+of+quorum+sensing+cross-regulation+in+Pseudomonas+aeruginosa+causes+major+and+unexpected+alterations+to+virulence+phenotypes&doi=10.1021%2Fja5110798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Disruption of Quorum Sensing Cross-Regulation in Pseudomonas aeruginosa Causes Major and Unexpected Alterations to Virulence Phenotypes</span></div><div class="casAuthors">Welsh, Michael A.; Eibergen, Nora R.; Moore, Joseph D.; Blackwell, Helen E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1510-1519</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The opportunistic pathogen Pseudomonas aeruginosa uses three interwoven quorum-sensing (QS) circuits-Las, Rhl, and Pqs-to regulate the global expression of myriad virulence-assocd. genes.  Interception of these signaling networks with small mols. represents an emerging strategy for the development of anti-infective agents against this bacterium.  In the current study, we applied a chem. approach to investigate how the Las-Rhl-Pqs QS hierarchy coordinates key virulence phenotypes in wild-type P. aeruginosa.  We screened a focused library of synthetic, non-native N-acyl L-homoserine lactones and identified compds. that can drastically alter prodn. of two important virulence factors: pyocyanin and rhamnolipid.  We demonstrate that these mols. act by targeting RhlR in P. aeruginosa, a QS receptor that has seen far less scrutiny to date relative to other circuitry.  Unexpectedly, modulation of RhlR activity by a single compd. induces inverse regulation of pyocyanin and rhamnolipid, a result that was not predicted using genetic approaches to interrogate QS in P. aeruginosa.  Further, we show that certain RhlR agonists strongly repress Pqs signaling, revealing disruption of Rhl-Pqs cross-regulation as a novel mechanism for QS inhibition.  These compds. significantly expand the known repertoire of chem. probes available to study RhlR in P. aeruginosa.  Moreover, our results suggest that designing chem. agents to disrupt Rhl-Pqs crosstalk could be an effective antivirulence strategy to fight this common pathogen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmeYI_3kIpQbVg90H21EOLACvtfcHk0li7oMAOJnI-gQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXns12qtw%253D%253D&md5=8cf2dfd6d0c33bf6aa1142d00683a86a</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fja5110798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja5110798%26sid%3Dliteratum%253Aachs%26aulast%3DWelsh%26aufirst%3DM.%2BA.%26aulast%3DEibergen%26aufirst%3DN.%2BR.%26aulast%3DMoore%26aufirst%3DJ.%2BD.%26aulast%3DBlackwell%26aufirst%3DH.%2BE.%26atitle%3DSmall%2520molecule%2520disruption%2520of%2520quorum%2520sensing%2520cross-regulation%2520in%2520Pseudomonas%2520aeruginosa%2520causes%2520major%2520and%2520unexpected%2520alterations%2520to%2520virulence%2520phenotypes%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D1510%26epage%3D1519%26doi%3D10.1021%2Fja5110798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCready, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paczkowski, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassler, B. L.</span></span> <span> </span><span class="NLM_article-title">An autoinducer-independent RhlR quorum-sensing receptor enables analysis of RhlR regulation</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>, <span class="NLM_elocation-id">e1007820</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1007820</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1371%2Fjournal.ppat.1007820" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=31194839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFGks7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&author=A.+R.+McCreadyauthor=J.+E.+Paczkowskiauthor=J.-P.+Congauthor=B.+L.+Bassler&title=An+autoinducer-independent+RhlR+quorum-sensing+receptor+enables+analysis+of+RhlR+regulation&doi=10.1371%2Fjournal.ppat.1007820"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">An autoinducer-independent RhlR quorum-sensing receptor enables analysis of RhlR regulation</span></div><div class="casAuthors">McCready, Amelia R.; Paczkowski, Jon E.; Cong, Jian-Ping; Bassler, Bonnie L.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e1007820</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Quorum sensing is a chem. communication process that bacteria use to coordinate group behaviors.  Pseudomonas aeruginosa, an opportunistic pathogen, employs multiple quorum-sensing systems to control behaviors including virulence factor prodn. and biofilm formation.  One P. aeruginosa quorum-sensing receptor, called RhlR, binds the cognate autoinducer N-butryl-homoserine lactone (C4HSL), and the RhlR:C4HSL complex activates transcription of target quorum-sensing genes.  Here, we use a genetic screen to identify RhlR mutants that function independently of the autoinducer.  The RhlR Y64F W68F V133F triple mutant, which we call RhlR*, exhibits ligand-independent activity in vitro and in vivo.  RhlR* can drive wildtype biofilm formation and infection in a nematode animal model.  The ability of RhlR* to properly regulate quorum-sensing-controlled genes in vivo depends on the quorum-sensing regulator RsaL keeping RhlR* activity in check.  RhlR is known to function together with PqsE to control prodn. of the virulence factor called pyocyanin.  Likewise, RhlR* requires PqsE for pyocyanin prodn. in planktonic cultures, however, PqsE is dispensable for RhlR*-driven pyocyanin prodn. on surfaces.  Finally, wildtype RhlR protein is not sufficiently stabilized by C4HSL to allow purifn.  However, wildtype RhlR can be stabilized by the synthetic ligand mBTL (meta-bromo-thiolactone) and RhlR* is stable without a ligand.  These features enabled purifn. of the RhlR:mBTL complex and of RhlR* for in vitro examn. of their biochem. activities.  To our knowledge, this work reports the first RhlR protein purifn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpy48nibUhtx7Vg90H21EOLACvtfcHk0li7oMAOJnI-gQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFGks7vO&md5=92bf15dba7796bec8c95a05d6b60ef25</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1007820&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1007820%26sid%3Dliteratum%253Aachs%26aulast%3DMcCready%26aufirst%3DA.%2BR.%26aulast%3DPaczkowski%26aufirst%3DJ.%2BE.%26aulast%3DCong%26aufirst%3DJ.-P.%26aulast%3DBassler%26aufirst%3DB.%2BL.%26atitle%3DAn%2520autoinducer-independent%2520RhlR%2520quorum-sensing%2520receptor%2520enables%2520analysis%2520of%2520RhlR%2520regulation%26jtitle%3DPLoS%2520Pathog.%26date%3D2019%26volume%3D15%26doi%3D10.1371%2Fjournal.ppat.1007820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.-D.</span></span> <span> </span><span class="NLM_article-title">6-Gingerol reduces Pseudomonas aeruginosa biofilm formation and virulence via quorum sensing inhibition</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">8656</span>, <span class="refDoi"> DOI: 10.1038/srep08656</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1038%2Fsrep08656" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=25728862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVCrtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=8656&author=H.-S.+Kimauthor=S.-H.+Leeauthor=Y.+Byunauthor=H.-D.+Park&title=6-Gingerol+reduces+Pseudomonas+aeruginosa+biofilm+formation+and+virulence+via+quorum+sensing+inhibition&doi=10.1038%2Fsrep08656"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">6-Gingerol reduces Pseudomonas aeruginosa biofilm formation and virulence via quorum sensing inhibition</span></div><div class="casAuthors">Kim, Han-Shin; Lee, Sang-Hoon; Byun, Youngjoo; Park, Hee-Deung</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8656</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Pseudomonas aeruginosa is a well-known pathogenic bacterium that forms biofilms and produces virulence factors via quorum sensing (QS).  Interfering with normal QS interactions between signal mols. and their cognate receptors is a developing strategy for attenuating its virulence.  Here we tested the hypothesis that 6-gingerol, a pungent oil of fresh ginger, reduces biofilm formation and virulence by antagonistically binding to P. aeruginosa QS receptors.  In silico studies demonstrated mol. binding occurs between 6-gingerol and the QS receptor LasR through hydrogen bonding and hydrophobic interactions.  Exptl. 6-gingerol reduced biofilm formation, several virulence factors (e.g., exoprotease, rhamnolipid, and pyocyanin), and mice mortality.  Further transcriptome analyses demonstrated that 6-gingerol successfully repressed QS-induced genes, specifically those related to the prodn. of virulence factors.  These results strongly support our hypothesis and offer insight into the mol. mechanism that caused QS gene repression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSqLFnQR49jLVg90H21EOLACvtfcHk0linnMXU-cVKdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVCrtbY%253D&md5=35b0f3cac58fefde5cf68e03f5397efa</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fsrep08656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep08656%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DH.-S.%26aulast%3DLee%26aufirst%3DS.-H.%26aulast%3DByun%26aufirst%3DY.%26aulast%3DPark%26aufirst%3DH.-D.%26atitle%3D6-Gingerol%2520reduces%2520Pseudomonas%2520aeruginosa%2520biofilm%2520formation%2520and%2520virulence%2520via%2520quorum%2520sensing%2520inhibition%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D8656%26doi%3D10.1038%2Fsrep08656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ham, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cha, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bang, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Son, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byun, Y.</span></span> <span> </span><span class="NLM_article-title">Structure-activity relationships of 6-and 8-gingerol analogs as anti-biofilm agents</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">9821</span>– <span class="NLM_lpage">9837</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01426</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01426" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVSnur%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9821-9837&author=H.+Choiauthor=S.-Y.+Hamauthor=E.+Chaauthor=Y.+Shinauthor=H.-S.+Kimauthor=J.+K.+Bangauthor=S.-H.+Sonauthor=H.-D.+Parkauthor=Y.+Byun&title=Structure-activity+relationships+of+6-and+8-gingerol+analogs+as+anti-biofilm+agents&doi=10.1021%2Facs.jmedchem.7b01426"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationships of 6- and 8-Gingerol Analogs as Anti-Biofilm Agents</span></div><div class="casAuthors">Choi, Hyunsuk; Ham, So-Young; Cha, Eunji; Shin, Yujin; Kim, Han-Shin; Bang, Jeong Kyu; Son, Sang-Hyun; Park, Hee-Deung; Byun, Youngjoo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9821-9837</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pseudomonas aeruginosa is a causative agent of chronic infections in immunocompromised patients.  Disruption of quorum sensing circuits is an attractive strategy for treating diseases assocd. with P. aeruginosa infection.  In this study, the authors designed and synthesized a series of gingerol analogs targeting LasR, a master regulator of quorum sensing networks in P. aeruginosa.  Structure-activity relationship studies showed that a hydrogen-bonding interaction in the head section, stereochem. and rotational rigidity in the middle section, and optimal alkyl chain length in the tail section are important factors for the enhancement of LasR-binding affinity and for the inhibition of biofilm formation.  The most potent compd. I, an analog of (R)-8-gingerol with restricted rotation, showed stronger LasR-binding affinity and inhibition of biofilm formation than the known LasR antagonist (S)-6-gingerol.  This new LasR antagonist can be used as an early lead compd. for the development of antibiofilm agents to treat P. aeruginosa infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFyS9hVAiLXLVg90H21EOLACvtfcHk0linnMXU-cVKdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVSnur%252FJ&md5=73007847e98d059046ae29327c27d817</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01426&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01426%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DH.%26aulast%3DHam%26aufirst%3DS.-Y.%26aulast%3DCha%26aufirst%3DE.%26aulast%3DShin%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DH.-S.%26aulast%3DBang%26aufirst%3DJ.%2BK.%26aulast%3DSon%26aufirst%3DS.-H.%26aulast%3DPark%26aufirst%3DH.-D.%26aulast%3DByun%26aufirst%3DY.%26atitle%3DStructure-activity%2520relationships%2520of%25206-and%25208-gingerol%2520analogs%2520as%2520anti-biofilm%2520agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D9821%26epage%3D9837%26doi%3D10.1021%2Facs.jmedchem.7b01426" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tassano, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dondo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galatini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riva, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banfi, L.</span></span> <span> </span><span class="NLM_article-title">Conjugation of hydroxytyrosol with other natural phenolic fragments: From waste to antioxidants and antitumour compounds</span>. <i>Eur. J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>2015</i></span>,  <span class="NLM_fpage">6710</span>– <span class="NLM_lpage">6726</span>, <span class="refDoi"> DOI: 10.1002/ejoc.201500931</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1002%2Fejoc.201500931" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFWrsrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2015&publication_year=2015&pages=6710-6726&author=E.+Tassanoauthor=A.+Alamaauthor=A.+Bassoauthor=G.+Dondoauthor=A.+Galatiniauthor=R.+Rivaauthor=L.+Banfi&title=Conjugation+of+hydroxytyrosol+with+other+natural+phenolic+fragments%3A+From+waste+to+antioxidants+and+antitumour+compounds&doi=10.1002%2Fejoc.201500931"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Conjugation of Hydroxytyrosol with Other Natural Phenolic Fragments: From Waste to Antioxidants and Antitumour Compounds</span></div><div class="casAuthors">Tassano, Erika; Alama, Angela; Basso, Andrea; Dondo, Giancarlo; Galatini, Andrea; Riva, Renata; Banfi, Luca</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">2015</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">6710-6726</span>CODEN:
                <span class="NLM_cas:coden">EJOCFK</span>;
        ISSN:<span class="NLM_cas:issn">1099-0690</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Hydroxytyrosol (I), a natural polyphenol that can be extd. from olive mill waste waters, was converted into a series of more complex and lipophilic analogs by conjugation with other naturally derived (poly)phenolic fragments.  Ether and triazole linkers were employed.  A small library of compds. was prepd., stressing step economy and operational simplicity.  Some of these substances were proven to have activity equal or superior to that of the parent hydroxytyrosol in radical scavenging assays as well as in cytotoxicity tests against tumor cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoG5GoGXvsi5LVg90H21EOLACvtfcHk0lismrEHccqVIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFWrsrrN&md5=bc83086e9f313191bb4e5650b242c3ea</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fejoc.201500931&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.201500931%26sid%3Dliteratum%253Aachs%26aulast%3DTassano%26aufirst%3DE.%26aulast%3DAlama%26aufirst%3DA.%26aulast%3DBasso%26aufirst%3DA.%26aulast%3DDondo%26aufirst%3DG.%26aulast%3DGalatini%26aufirst%3DA.%26aulast%3DRiva%26aufirst%3DR.%26aulast%3DBanfi%26aufirst%3DL.%26atitle%3DConjugation%2520of%2520hydroxytyrosol%2520with%2520other%2520natural%2520phenolic%2520fragments%253A%2520From%2520waste%2520to%2520antioxidants%2520and%2520antitumour%2520compounds%26jtitle%3DEur.%2520J.%2520Org.%2520Chem.%26date%3D2015%26volume%3D2015%26spage%3D6710%26epage%3D6726%26doi%3D10.1002%2Fejoc.201500931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsumura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashiguchi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikariya, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noyori, R.</span></span> <span> </span><span class="NLM_article-title">Asymmetric transfer hydrogenation of α, β-acetylenic ketones</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">8738</span>– <span class="NLM_lpage">8739</span>, <span class="refDoi"> DOI: 10.1021/ja971570a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja971570a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADyaK2sXmtVens7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=1997&pages=8738-8739&author=K.+Matsumuraauthor=S.+Hashiguchiauthor=T.+Ikariyaauthor=R.+Noyori&title=Asymmetric+transfer+hydrogenation+of+%CE%B1%2C+%CE%B2-acetylenic+ketones&doi=10.1021%2Fja971570a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Asymmetric Transfer Hydrogenation of α,β-Acetylenic Ketones</span></div><div class="casAuthors">Matsumura, Kazuhiko; Hashiguchi, Shohei; Ikariya, Takao; Noyori, Ryoji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">8738-8739</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A range of achiral α,β-acetylenic ketones is converted to propargylic alcs. with high enantiomeric excess and in high yield by transfer hydrogenation using 2-propanol as the hydrogen donor.  The asym. reaction using arene-Ru(II) catalysts modified with (R,R)- or (S,S)-N-p-toluenesulfonyl-1,2-diphenylethylenediamine proceeds with a substrate/catalyst molar ratio of 100-1000 under neutral conditions and at room temp., allowing chemo- and enantioselective satn. of the carbonyl function while leaving the acetylenic linkage intact.  β-Amino alcs. such as (1S,2S)-2-(methylamino)-1,2-diphenylethanol are also usable as chiral auxiliaries.  The enantiomeric purity of the propargylic alcs. is consistently high regardless of the bulkiness of substituents.  Redn. of chiral alkynyl Cbz-aminoalkyl ketones gives the enantiomerically pure alcs. with contiguous stereogenic carbons, where the carbonyl faces are differentiated by the chirality of the Ru catalyst.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8efeo1MIaKLVg90H21EOLACvtfcHk0lismrEHccqVIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmtVens7w%253D&md5=8b9b4a49190d9681c8950826f24a2d55</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fja971570a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja971570a%26sid%3Dliteratum%253Aachs%26aulast%3DMatsumura%26aufirst%3DK.%26aulast%3DHashiguchi%26aufirst%3DS.%26aulast%3DIkariya%26aufirst%3DT.%26aulast%3DNoyori%26aufirst%3DR.%26atitle%3DAsymmetric%2520transfer%2520hydrogenation%2520of%2520%25CE%25B1%252C%2520%25CE%25B2-acetylenic%2520ketones%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1997%26volume%3D119%26spage%3D8738%26epage%3D8739%26doi%3D10.1021%2Fja971570a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazmaier, U.</span></span> <span> </span><span class="NLM_article-title">Stereoselective modification of N-(alpha-hydroxyacyl)-glycinesters via palladium-catalyzed allylic alkylation</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">4595</span>– <span class="NLM_lpage">4599</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.9b01497</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.9b01497" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVOntrrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2019&pages=4595-4599&author=A.+Hornauthor=U.+Kazmaier&title=Stereoselective+modification+of+N-%28alpha-hydroxyacyl%29-glycinesters+via+palladium-catalyzed+allylic+alkylation&doi=10.1021%2Facs.orglett.9b01497"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Stereoselective Modification of N-(α-Hydroxyacyl)-glycinesters via Palladium-Catalyzed Allylic Alkylation</span></div><div class="casAuthors">Horn, Alexander; Kazmaier, Uli</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4595-4599</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">N-(α-Hydroxyacyl)-glycinesters can be used as excellent nucleophiles in Pd-catalyzed allylic alkylation.  The method allows for the stereoselective introduction of a wide range of side chains, including highly functionalized ones.  Both diastereomers can be accessed through variation of the reaction conditions.  Furthermore, the use of stannylated carbonates introduces vinylstannane motifs, which are eligible for subsequent C-C coupling reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVhlVAbob4bLVg90H21EOLACvtfcHk0lismrEHccqVIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVOntrrM&md5=17c3a77a32c603a6925bfd4a74382a7b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.9b01497&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.9b01497%26sid%3Dliteratum%253Aachs%26aulast%3DHorn%26aufirst%3DA.%26aulast%3DKazmaier%26aufirst%3DU.%26atitle%3DStereoselective%2520modification%2520of%2520N-%2528alpha-hydroxyacyl%2529-glycinesters%2520via%2520palladium-catalyzed%2520allylic%2520alkylation%26jtitle%3DOrg.%2520Lett.%26date%3D2019%26volume%3D21%26spage%3D4595%26epage%3D4599%26doi%3D10.1021%2Facs.orglett.9b01497" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ochsner, U. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiser, J.</span></span> <span> </span><span class="NLM_article-title">Autoinducer-mediated regulation of rhamnolipid biosurfactant synthesis in Pseudomonas aeruginosa</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">6424</span>– <span class="NLM_lpage">6428</span>, <span class="refDoi"> DOI: 10.1073/pnas.92.14.6424</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1073%2Fpnas.92.14.6424" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=7604006" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADyaK2MXmslKqur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=1995&pages=6424-6428&author=U.+A.+Ochsnerauthor=J.+Reiser&title=Autoinducer-mediated+regulation+of+rhamnolipid+biosurfactant+synthesis+in+Pseudomonas+aeruginosa&doi=10.1073%2Fpnas.92.14.6424"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Autoinducer-mediated regulation of rhamnolipid biosurfactant synthesis in Pseudomonas aeruginosa</span></div><div class="casAuthors">Ochsner, Urs A.; Reiser, Jakob</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6424-8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The opportunistic human pathogen Pseudomonas aeruginosa produces a variety of virulence factors, including exotoxin A, elastase, alk. protease, alginate, phospholipases, and extracellular rhamnolipids.  The previously characterized rhlABR gene cluster encodes a regulatory protein (RhlR) and a rhamnosyltransferase (RhlAB), both of which are required for rhamnolipid synthesis.  Another gene, rhlI, was now identified downstream of the rhlABR gene cluster.  The putative RhlI protein shares significant sequence similarity with bacterial autoinducer synthetases of the LuxI type.  A P. aeruginosa rhlI mutant strain carrying a disrupted rhlI gene was unable to produce rhamnolipids and lacked rhamnosyltransferase activity.  Rhamnolipid synthesis was restored by introducing a wild-type rhlI gene into such strains or, alternatively, by adding either the cell-free spent supernatant from a P. aeruginosa wild-type strain or synthetic N-acylhomoserine lactones.  Half-maximal induction of rhamnolipid synthesis in the rhlI mutant strain required 0.5 μM N-butyrylhomoserine lactone or 10 μM N-(3-oxohexanoyl)homoserine lactone.  The P. aeruginosa rhlA promoter was active in the heterologous host Pseudomonas putida when both the rhlR and rhlI genes were present or when the rhlR gene alone was supplied together with synthetic N-acylhomoserine lactones.  The RhlR-RhlI regulatory system was essential for the prodn. of elastase as well, and cross-communication between the RhlR-RhlI rhamnolipid regulatory system and the LasR-LasI elastase regulatory system was demonstrated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFoNIMwsQQE7Vg90H21EOLACvtfcHk0lhHWPoaGTVujA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmslKqur0%253D&md5=45fcdfa709509c6c3ce07f7e0b76ab57</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1073%2Fpnas.92.14.6424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.92.14.6424%26sid%3Dliteratum%253Aachs%26aulast%3DOchsner%26aufirst%3DU.%2BA.%26aulast%3DReiser%26aufirst%3DJ.%26atitle%3DAutoinducer-mediated%2520regulation%2520of%2520rhamnolipid%2520biosurfactant%2520synthesis%2520in%2520Pseudomonas%2520aeruginosa%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1995%26volume%3D92%26spage%3D6424%26epage%3D6428%26doi%3D10.1073%2Fpnas.92.14.6424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nickzad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Déziel, E.</span></span> <span> </span><span class="NLM_article-title">The involvement of rhamnolipids in microbial cell adhesion and biofilm development-an approach for control?</span>. <i>Lett. Appl. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">453</span>, <span class="refDoi"> DOI: 10.1111/lam.12211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1111%2Flam.12211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=24372465" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtl2itLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2014&pages=447-453&author=A.+Nickzadauthor=E.+D%C3%A9ziel&title=The+involvement+of+rhamnolipids+in+microbial+cell+adhesion+and+biofilm+development-an+approach+for+control%3F&doi=10.1111%2Flam.12211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">The involvement of rhamnolipids in microbial cell adhesion and biofilm development - an approach for control?</span></div><div class="casAuthors">Nickzad, A.; Deziel, E.</div><div class="citationInfo"><span class="NLM_cas:title">Letters in Applied Microbiology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">447-453</span>CODEN:
                <span class="NLM_cas:coden">LAMIE7</span>;
        ISSN:<span class="NLM_cas:issn">0266-8254</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Biofilms are omnipresent in clin. and industrial settings and most of the times cause detrimental side effects.  Finding efficient strategies to control surface-growing communities of micro-organisms remains a significant challenge.  Rhamnolipids are extracellular secondary metabolites with surface-active properties mainly produced by Pseudomonas aeruginosa.  There is growing evidence for the implication of this biosurfactant in different stages of biofilm development of this bacterium.  Furthermore, rhamnolipids display a significant potential as anti-adhesive and disrupting agents against established biofilms formed by several bacterial and fungal species.  Their low toxicity, biodegradability, efficiency and specificity, compared to synthetic surfactants typically used in biofilm control, might compensate for the economic hurdle still linked to their superior prodn. costs and make them promising antifouling agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpt6j2_3bBSbVg90H21EOLACvtfcHk0lhHWPoaGTVujA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtl2itLs%253D&md5=7575bc61e392564b72503aa1d9e5e6c3</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1111%2Flam.12211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Flam.12211%26sid%3Dliteratum%253Aachs%26aulast%3DNickzad%26aufirst%3DA.%26aulast%3DD%25C3%25A9ziel%26aufirst%3DE.%26atitle%3DThe%2520involvement%2520of%2520rhamnolipids%2520in%2520microbial%2520cell%2520adhesion%2520and%2520biofilm%2520development-an%2520approach%2520for%2520control%253F%26jtitle%3DLett.%2520Appl.%2520Microbiol.%26date%3D2014%26volume%3D58%26spage%3D447%26epage%3D453%26doi%3D10.1111%2Flam.12211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Solano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echeverz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasa, I.</span></span> <span> </span><span class="NLM_article-title">Biofilm dispersion and quorum sensing</span>. <i>Curr. Opin. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">96</span>– <span class="NLM_lpage">104</span>, <span class="refDoi"> DOI: 10.1016/j.mib.2014.02.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1016%2Fj.mib.2014.02.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=24657330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmslais7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2014&pages=96-104&author=C.+Solanoauthor=M.+Echeverzauthor=I.+Lasa&title=Biofilm+dispersion+and+quorum+sensing&doi=10.1016%2Fj.mib.2014.02.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Biofilm dispersion and quorum sensing</span></div><div class="casAuthors">Solano, Cristina; Echeverz, Maite; Lasa, Inigo</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Microbiology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">96-104</span>CODEN:
                <span class="NLM_cas:coden">COMIF7</span>;
        ISSN:<span class="NLM_cas:issn">1369-5274</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Biofilm development and quorum sensing (QS) are closely interconnected processes.  Biofilm formation is a cooperative group behavior that involves bacterial populations living embedded in a self-produced extracellular matrix.  QS is a cell-cell communication mechanism that synchronizes gene expression in response to population cell d.  Intuitively, it would appear that QS might coordinate the switch to a biofilm lifestyle when the population d. reaches a threshold level.  However, compelling evidence obtained in different bacterial species coincides in that activation of QS occurs in the formed biofilm and activates the maturation and disassembly of the biofilm in a coordinate manner.  The aim of this review is to illustrate, using four bacterial pathogens as examples, the emergent concept that QS activates the biofilm dispersion process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqt_zGsdhS-bVg90H21EOLACvtfcHk0li-eLZ9DBR4qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmslais7Y%253D&md5=46db3d97efd74b26c75679a57fdfd1b4</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.mib.2014.02.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mib.2014.02.008%26sid%3Dliteratum%253Aachs%26aulast%3DSolano%26aufirst%3DC.%26aulast%3DEcheverz%26aufirst%3DM.%26aulast%3DLasa%26aufirst%3DI.%26atitle%3DBiofilm%2520dispersion%2520and%2520quorum%2520sensing%26jtitle%3DCurr.%2520Opin.%2520Microbiol.%26date%3D2014%26volume%3D18%26spage%3D96%26epage%3D104%26doi%3D10.1016%2Fj.mib.2014.02.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pamp, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolker-Nielsen, T.</span></span> <span> </span><span class="NLM_article-title">Multiple roles of biosurfactants in structural biofilm development by Pseudomonas aeruginosa</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>189</i></span>,  <span class="NLM_fpage">2531</span>– <span class="NLM_lpage">2539</span>, <span class="refDoi"> DOI: 10.1128/jb.01515-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1128%2FJB.01515-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=17220224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjt1GksLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=189&publication_year=2007&pages=2531-2539&author=S.+J.+Pampauthor=T.+Tolker-Nielsen&title=Multiple+roles+of+biosurfactants+in+structural+biofilm+development+by+Pseudomonas+aeruginosa&doi=10.1128%2Fjb.01515-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple roles of biosurfactants in structural biofilm development by Pseudomonas aeruginosa</span></div><div class="casAuthors">Pamp, Sunje Johanna; Tolker-Nielsen, Tim</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">189</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2531-2539</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Recent studies have indicated that biosurfactants produced by Pseudomonas aeruginosa play a role both in maintaining channels between multi-cellular structures in biofilms and in dispersal of cells from biofilms.  Through the use of flow cell technol. and enhanced confocal laser scanning microscopy, we have obtained results which suggest that the biosurfactants produced by P. aeruginosa play addnl. roles in structural biofilm development.  We present genetic evidence that during biofilm development by P. aeruginosa, biosurfactants promote microcolony formation in the initial phase and facilitate migration-dependent structural development in the later phase.  P. aeruginosa rhlA mutants, deficient in synthesis of biosurfactants, were not capable of forming microcolonies in the initial phase of biofilm formation.  Expts. involving two-color-coded mixed-strain biofilms showed that P. aeruginosa rhlA mutants were defective in migration-dependent development of mushroom-shaped multi-cellular structures in the later phase of biofilm formation.  Expts. involving three-color-coded mixed-strain P. aeruginosa biofilms demonstrated that the wild-type and rhlA and pilA mutant strains formed distinct subpopulations on top of each other dependent on their ability to migrate and produce biosurfactants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojNxJWxWbAQLVg90H21EOLACvtfcHk0li-eLZ9DBR4qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjt1GksLo%253D&md5=49ff7bcfa3ad2dc7b8604b1223b55f8c</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1128%2FJB.01515-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.01515-06%26sid%3Dliteratum%253Aachs%26aulast%3DPamp%26aufirst%3DS.%2BJ.%26aulast%3DTolker-Nielsen%26aufirst%3DT.%26atitle%3DMultiple%2520roles%2520of%2520biosurfactants%2520in%2520structural%2520biofilm%2520development%2520by%2520Pseudomonas%2520aeruginosa%26jtitle%3DJ.%2520Bacteriol.%26date%3D2007%26volume%3D189%26spage%3D2531%26epage%3D2539%26doi%3D10.1128%2Fjb.01515-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skindersoe, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alhede, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phipps, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, P. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bjarnsholt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolker-Nielsen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Høiby, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Givskov, M.</span></span> <span> </span><span class="NLM_article-title">Effects of antibiotics on quorum sensing in Pseudomonas aeruginosa</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">3648</span>– <span class="NLM_lpage">3663</span>, <span class="refDoi"> DOI: 10.1128/aac.01230-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1128%2FAAC.01230-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=18644954" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1SrtbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2008&pages=3648-3663&author=M.+E.+Skindersoeauthor=M.+Alhedeauthor=R.+Phippsauthor=L.+Yangauthor=P.+O.+Jensenauthor=T.+B.+Rasmussenauthor=T.+Bjarnsholtauthor=T.+Tolker-Nielsenauthor=N.+H%C3%B8ibyauthor=M.+Givskov&title=Effects+of+antibiotics+on+quorum+sensing+in+Pseudomonas+aeruginosa&doi=10.1128%2Faac.01230-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of antibiotics on quorum sensing in Pseudomonas aeruginosa</span></div><div class="casAuthors">Skindersoe, Mette E.; Alhede, Morten; Phipps, Richard; Yang, Liang; Jensen, Peter O.; Rasmussen, Thomas B.; Bjarnsholt, Thomas; Tolker-Nielsen, Tim; Hoeiby, Niels; Givskov, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3648-3663</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">During infection, Pseudomonas aeruginosa employs bacterial communication (quorum sensing [QS]) to coordinate the expression of tissue-damaging factors.  QS-controlled gene expression plays a pivotal role in the virulence of P. aeruginosa, and QS-deficient mutants cause less severe infections in animal infection models.  Treatment of cystic fibrosis (CF) patients chronically infected with P. aeruginosa with the macrolide antibiotic azithromycin (AZM) has been demonstrated to improve the clin. outcome.  Several studies indicate that AZM may accomplish its beneficial action in CF patients by impeding QS, thereby reducing the pathogenicity of P. aeruginosa.  This led us to investigate whether QS inhibition is a common feature of antibiotics.  We present the results of a screening of 12 antibiotics for their QS-inhibitory activities using a previously described QS inhibitor selector 1 strain.  Three of the antibiotics tested, AZM, ceftazidime (CFT), and ciprofloxacin (CPR), were very active in the assay and were further examd. for their effects on QS-regulated virulence factor prodn. in P. aeruginosa.  The effects of the three antibiotics administered at subinhibitory concns. were investigated by use of DNA microarrays.  Consistent results from the virulence factor assays, reverse transcription-PCR, and the DNA microarrays support the finding that AZM, CFT, and CPR decrease the expression of a range of QS-regulated virulence factors.  The data suggest that the underlying mechanism may be mediated by changes in membrane permeability, thereby influencing the flux of N-3-oxo-dodecanoyl-L-homoserine lactone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpj0zKNJdCINrVg90H21EOLACvtfcHk0li-eLZ9DBR4qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1SrtbbJ&md5=059375c0aba1089b5150fd5bdecf4ae4</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1128%2FAAC.01230-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01230-07%26sid%3Dliteratum%253Aachs%26aulast%3DSkindersoe%26aufirst%3DM.%2BE.%26aulast%3DAlhede%26aufirst%3DM.%26aulast%3DPhipps%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DJensen%26aufirst%3DP.%2BO.%26aulast%3DRasmussen%26aufirst%3DT.%2BB.%26aulast%3DBjarnsholt%26aufirst%3DT.%26aulast%3DTolker-Nielsen%26aufirst%3DT.%26aulast%3DH%25C3%25B8iby%26aufirst%3DN.%26aulast%3DGivskov%26aufirst%3DM.%26atitle%3DEffects%2520of%2520antibiotics%2520on%2520quorum%2520sensing%2520in%2520Pseudomonas%2520aeruginosa%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2008%26volume%3D52%26spage%3D3648%26epage%3D3663%26doi%3D10.1128%2Faac.01230-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moustafa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassler, B. L.</span></span> <span> </span><span class="NLM_article-title">The RhlR quorum-sensing receptor controls Pseudomonas aeruginosa pathogenesis and biofilm development independently of its canonical homoserine lactone autoinducer</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>, <span class="NLM_elocation-id">e1006504</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1006504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1371%2Fjournal.ppat.1006504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=28715477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC1cXivVWgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&author=S.+Mukherjeeauthor=D.+Moustafaauthor=C.+D.+Smithauthor=J.+B.+Goldbergauthor=B.+L.+Bassler&title=The+RhlR+quorum-sensing+receptor+controls+Pseudomonas+aeruginosa+pathogenesis+and+biofilm+development+independently+of+its+canonical+homoserine+lactone+autoinducer&doi=10.1371%2Fjournal.ppat.1006504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The RhlR quorum-sensing receptor controls Pseudomonas aeruginosa pathogenesis and biofilm development independently of its canonical homoserine lactone autoinducer</span></div><div class="casAuthors">Mukherjee, Sampriti; Moustafa, Dina; Smith, Chari D.; Goldberg, Joanna B.; Bassler, Bonnie L.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e1006504/1-e1006504/25</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Quorum sensing (QS) is a bacterial cell-to-cell communication process that relies on the prodn., release, and response to extracellular signaling mols. called autoinducers.  QS controls virulence and biofilm formation in the human pathogen Pseudomonas aeruginosa.  P. aeruginosa possesses two canonical LuxI/R-type QS systems, LasI/R and RhlI/R, which produce and detect 3OC12-homoserine lactone and C4-homoserine lactone, resp.  Here, we use biofilm analyses, reporter assays, RNA-seq studies, and animal infection assays to show that RhlR directs both RhlI-dependent and RhlI-independent regulons.  In the absence of RhlI, RhlR controls the expression of genes required for biofilm formation as well as genes encoding virulence factors.  Consistent with these findings, ΔrhlR and ΔrhlI mutants have radically different biofilm phenotypes and the ΔrhlI mutant displays full virulence in animals whereas the ΔrhlR mutant is attenuated.  The ΔrhlI mutant cell-free culture fluids contain an activity that stimulates RhlR-dependent gene expression.  We propose a model in which RhlR responds to an alternative ligand, in addn. to its canonical C4-homoserine lactone autoinducer.  This alternate ligand promotes a RhlR-dependent transcriptional program in the absence of RhlI.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfizuX4Th-oLVg90H21EOLACvtfcHk0li-eLZ9DBR4qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXivVWgs7c%253D&md5=843c61c61fe3c63ab5d1bca3c3dcb790</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1006504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1006504%26sid%3Dliteratum%253Aachs%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DMoustafa%26aufirst%3DD.%26aulast%3DSmith%26aufirst%3DC.%2BD.%26aulast%3DGoldberg%26aufirst%3DJ.%2BB.%26aulast%3DBassler%26aufirst%3DB.%2BL.%26atitle%3DThe%2520RhlR%2520quorum-sensing%2520receptor%2520controls%2520Pseudomonas%2520aeruginosa%2520pathogenesis%2520and%2520biofilm%2520development%2520independently%2520of%2520its%2520canonical%2520homoserine%2520lactone%2520autoinducer%26jtitle%3DPLoS%2520Pathog.%26date%3D2017%26volume%3D13%26doi%3D10.1371%2Fjournal.ppat.1006504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lindsay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmer, B. M. M.</span></span> <span> </span><span class="NLM_article-title">Effect of sdiA on biosensors of N-acylhomoserine lactones</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>187</i></span>,  <span class="NLM_fpage">5054</span>– <span class="NLM_lpage">5058</span>, <span class="refDoi"> DOI: 10.1128/jb.187.14.5054-5058.2005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1128%2FJB.187.14.5054-5058.2005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=15995228" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmtVyiu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=187&publication_year=2005&pages=5054-5058&author=A.+Lindsayauthor=B.+M.+M.+Ahmer&title=Effect+of+sdiA+on+biosensors+of+N-acylhomoserine+lactones&doi=10.1128%2Fjb.187.14.5054-5058.2005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of sdiA on biosensors of N-acylhomoserine lactones</span></div><div class="casAuthors">Lindsay, Amber; Ahmer, Brian M. M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">187</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5054-5058</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Many gram-neg. bacteria synthesize N-acylhomoserine lactones (AHLs) and then use transcription factors of the LuxR family to sense and respond to AHL accumulation in the environment; this phenomenon is termed quorum sensing.  Bacteria produce a variety of AHLs, and numerous bacterial reporter strains, or biosensors, that can detect subsets of these mols. have been constructed.  Many of these are based on Escherichia coli because this species does not produce AHLs.  However, both Escherichia and Salmonella spp. contain a LuxR homolog named SdiA that can detect exogenous AHL synthesized by other microbial species.  In this study we have detd. that sdiA of E. coli and Salmonella spp. can activate an RhlR-based biosensor plasmid in response to AHLs other than what the biosensor was designed to detect.  SdiA does not activate LuxR-, LasR-, or AhyR-based biosensor plasmids, although the presence of sdiA in E. coli does interfere with the function of the AhyR-based biosensor.  Because sdiA interferes with the function of two of the four reporters, we have constructed a set of E. coli biosensor strains that lack sdiA.  The set includes control reporters that allow the luxR dependence of responses to be detd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpI99L1ykDmebVg90H21EOLACvtfcHk0li2Ei7glzCGtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmtVyiu78%253D&md5=84f4b4e2159e1cb32f46b47b11bd9c80</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1128%2FJB.187.14.5054-5058.2005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.187.14.5054-5058.2005%26sid%3Dliteratum%253Aachs%26aulast%3DLindsay%26aufirst%3DA.%26aulast%3DAhmer%26aufirst%3DB.%2BM.%2BM.%26atitle%3DEffect%2520of%2520sdiA%2520on%2520biosensors%2520of%2520N-acylhomoserine%2520lactones%26jtitle%3DJ.%2520Bacteriol.%26date%3D2005%26volume%3D187%26spage%3D5054%26epage%3D5058%26doi%3D10.1128%2Fjb.187.14.5054-5058.2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collins, T. J.</span></span> <span> </span><span class="NLM_article-title">ImageJ for microscopy</span>. <i>Biotechniques</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">S25</span>– <span class="NLM_lpage">S30</span>, <span class="refDoi"> DOI: 10.2144/000112517</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.2144%2F000112517" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=17936939" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A280%3ADC%252BD2snht1agtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2007&pages=S25-S30&author=T.+J.+Collins&title=ImageJ+for+microscopy&doi=10.2144%2F000112517"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">ImageJ for microscopy</span></div><div class="casAuthors">Collins Tony J</div><div class="citationInfo"><span class="NLM_cas:title">BioTechniques</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">1 Suppl</span>),
    <span class="NLM_cas:pages">25-30</span>
        ISSN:<span class="NLM_cas:issn">0736-6205</span>.
    </div><div class="casAbstract">ImageJ is an essential tool for us that fulfills most of our routine image processing and analysis requirements.  The near-comprehensive range of import filters that allow easy access to image and meta-data, a broad suite processing and analysis routine, and enthusiastic support from a friendly mailing list are invaluable for all microscopy labs and facilities-not just those on a budget.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTICP2q04csYUmhnSqMnvzDfW6udTcc2eb1YK2e04sRv7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2snht1agtg%253D%253D&md5=452a9e7d1b2118ded24c1ea3ffc13232</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.2144%2F000112517&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2144%252F000112517%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DT.%2BJ.%26atitle%3DImageJ%2520for%2520microscopy%26jtitle%3DBiotechniques%26date%3D2007%26volume%3D43%26spage%3DS25%26epage%3DS30%26doi%3D10.2144%2F000112517" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">So, Y.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeom, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, B.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-H.</span></span> <span> </span><span class="NLM_article-title">Protease IV, a quorum sensing-dependent protease of Pseudomonas aeruginosa modulates insect innate immunity</span>. <i>Mol. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">1298</span>– <span class="NLM_lpage">1314</span>, <span class="refDoi"> DOI: 10.1111/mmi.12830</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1111%2Fmmi.12830" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=25315216" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVClsbfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2014&pages=1298-1314&author=S.-J.+Parkauthor=S.-K.+Kimauthor=Y.-I.+Soauthor=H.-Y.+Parkauthor=X.-H.+Liauthor=D.+H.+Yeomauthor=M.-N.+Leeauthor=B.-L.+Leeauthor=J.-H.+Lee&title=Protease+IV%2C+a+quorum+sensing-dependent+protease+of+Pseudomonas+aeruginosa+modulates+insect+innate+immunity&doi=10.1111%2Fmmi.12830"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Protease IV, a quorum sensing-dependent protease of Pseudomonas aeruginosa modulates insect innate immunity</span></div><div class="casAuthors">Park, Su-Jin; Kim, Soo-Kyoung; So, Yong-In; Park, Ha-Young; Li, Xi-Hui; Yeom, Doo Hwan; Lee, Mi-Nan; Lee, Bok-Luel; Lee, Joon-Hee</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Microbiology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1298-1314</span>CODEN:
                <span class="NLM_cas:coden">MOMIEE</span>;
        ISSN:<span class="NLM_cas:issn">0950-382X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Summary : In Pseudomonas aeruginosa, quorum sensing (QS) plays an essential role in pathogenesis and the QS response controls many virulence factors.  Using a mealworm, Tenebrio molitor as a host model, we found that Protease IV, a QS-regulated exoprotease of P. aeruginosa functions as a key virulence effector causing the melanization and death of T. molitor larvae.  Protease IV was able to degrade zymogens of spaetzle processing enzyme (SPE) and SPE-activating enzyme (SAE) without the activation of the antimicrobial peptide (AMP) prodn.  Since SPE and SAE function to activate spaetzle, a ligand of Toll receptor in the innate immune system of T. molitor, we suggest that Protease IV may interfere with the activation of the Toll signaling.  Independently of the Toll pathway, the melanization response, another innate immunity was still generated, since Protease IV directly converted Tenebrio prophenoloxidase into active phenoloxidase.  Protease IV also worked as an important factor in the virulence to brine shrimp and nematode.  These results suggest that Protease IV provides P. aeruginosa with a sophisticated way to escape the immune attack of host by interfering with the prodn. of AMPs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsxhx_6u3AfbVg90H21EOLACvtfcHk0li2Ei7glzCGtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVClsbfJ&md5=9b07fc021e13f59071aa711eecd4a27a</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1111%2Fmmi.12830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fmmi.12830%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DS.-J.%26aulast%3DKim%26aufirst%3DS.-K.%26aulast%3DSo%26aufirst%3DY.-I.%26aulast%3DPark%26aufirst%3DH.-Y.%26aulast%3DLi%26aufirst%3DX.-H.%26aulast%3DYeom%26aufirst%3DD.%2BH.%26aulast%3DLee%26aufirst%3DM.-N.%26aulast%3DLee%26aufirst%3DB.-L.%26aulast%3DLee%26aufirst%3DJ.-H.%26atitle%3DProtease%2520IV%252C%2520a%2520quorum%2520sensing-dependent%2520protease%2520of%2520Pseudomonas%2520aeruginosa%2520modulates%2520insect%2520innate%2520immunity%26jtitle%3DMol.%2520Microbiol.%26date%3D2014%26volume%3D94%26spage%3D1298%26epage%3D1314%26doi%3D10.1111%2Fmmi.12830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, N. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacMillan, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sperandio, V.</span></span> <span> </span><span class="NLM_article-title">Structural and mechanistic roles of novel chemical ligands on the SdiA quorum-sensing transcription regulator</span>. <i>mBio</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e02429-14</span> <span class="refDoi"> DOI: 10.1128/mbio.02429-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1128%2FmBio.02429-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=25827420" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&author=Y.+Nguyenauthor=N.+X.+Nguyenauthor=J.+L.+Rogersauthor=J.+Liaoauthor=J.+B.+MacMillanauthor=Y.+Jiangauthor=V.+Sperandio&title=Structural+and+mechanistic+roles+of+novel+chemical+ligands+on+the+SdiA+quorum-sensing+transcription+regulator&doi=10.1128%2Fmbio.02429-14"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1128%2FmBio.02429-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FmBio.02429-14%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DY.%26aulast%3DNguyen%26aufirst%3DN.%2BX.%26aulast%3DRogers%26aufirst%3DJ.%2BL.%26aulast%3DLiao%26aufirst%3DJ.%26aulast%3DMacMillan%26aufirst%3DJ.%2BB.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DSperandio%26aufirst%3DV.%26atitle%3DStructural%2520and%2520mechanistic%2520roles%2520of%2520novel%2520chemical%2520ligands%2520on%2520the%2520SdiA%2520quorum-sensing%2520transcription%2520regulator%26jtitle%3DmBio%26date%3D2015%26volume%3D6%26doi%3D10.1128%2Fmbio.02429-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/medium/jm0c00630_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00630&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/medium/jm0c00630_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of endogenous and synthetic molecules interacting with the QS receptor of <i>P. aeruginosa</i>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00630&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/medium/jm0c00630_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 4-Gingerol Derivatives with the Variation of Phenyl Rings<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00630&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) acetone, EtOH (or water), 10% NaOH, rt, 2 to 24 h, 32%−quantitative yield; (ii) LDA 1.0 M in THF/hexanes, <i>n</i>-butanal, THF, −78 °C, 2–12 h, 24–60% yield; (iii) H<sub>2</sub> gas, 10% Pd/C, MeOH, rt, 2 h, 40–93% yield.</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/medium/jm0c00630_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of 4-Gingerol Derivatives with the Variation of Rotational Rigidity and Absolute Configuration<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00630&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) TBDMS chloride, imidazole, CH<sub>2</sub>Cl<sub>2</sub>, rt, 18 h, 94% yield; (ii) PPh<sub>3</sub>, CBr<sub>4</sub>, CH<sub>2</sub>Cl<sub>2</sub>, rt, 3 h, 98% yield; (iii) <i>n</i>-BuLi 1.6 M in hexanes (3.0 equiv), THF, rt, 2 h, 96% yield; (iv) <i>n</i>-BuLi 1.6 M in hexanes (1.1 equiv), <i>N</i>-methoxy-<i>N</i>-methylhexanamide, THF, −78 °C to rt, 16 h, 75% yield; (v) <i>n</i>-Bu<sub>4</sub>NF (0.1 M in THF) rt, 1 h, 94% yield (for <b>21</b>), 94% yield (for <b>23a</b>), 94% yield (for <b>23b</b>), 93% yield (for <b>23c</b>), 70% yield (for <b>25</b>), and 94% yield (for <b>27</b>); (vi) NaBH<sub>4</sub>, MeOH, rt, 1 h, 77% yield (for <b>22a</b>), RuCl[(<i>R</i>,<i>R</i>)-TsDPEN(mesitylene)], KOH, 2-propanol, 4 h, 88% yield (for <b>22b</b>), and RuCl[(<i>S</i>,<i>S</i>)-TsDPEN(mesitylene)], KOH, 2-propanol, 4 h, 68% yield (for <b>22c</b>); (vii) Lindlar cat., H<sub>2</sub> gas, 1,4-benzoquinone, MeOH, 0 °C, 1 h (from <b>22a</b>); (viii) H<sub>2</sub> gas, 10% Pd/C, MeOH, rt, 2 h, 94% yield (from <b>22a</b>).</p></p></figure><figure data-id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/medium/jm0c00630_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of 3,4-Difluorophenyl Derivatives<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00630&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) PPh<sub>3</sub>, CBr<sub>4</sub>, CH<sub>2</sub>Cl<sub>2</sub>, rt, 3 h, 97% yield; (ii) <i>n</i>-BuLi 1.6 M in hexanes (3.0 equiv), THF, rt, 2 h; (iii) <i>n</i>-BuLi 1.6 M in hexanes (1.1 equiv), appropriate Weinreb amides, THF, −78 °C to rt, 16 h, 72% yield (for <b>30</b>), 88% yield (for <b>31</b>), and 72% yield (for <b>32</b>); (iv) NaBH<sub>4</sub>, MeOH, rt, 1 h (for <b>33a–35a</b>), 88% yield (for <b>33a</b>), 80% yield (for <b>34a</b>), and 86% yield (for <b>35a</b>), RuCl[(<i>R</i>,<i>R</i>)-TsDPEN(mesitylene)], KOH, 2-propanol, 4 h (for <b>33b–35b</b>), 86% yield (for <b>33b</b>), 82% yield (for <b>34b</b>), and 80% yield (for <b>35b</b>), and RuCl[(<i>S</i>,<i>S</i>)-TsDPEN(mesitylene)], KOH, 2-propanol, 4 h (for <b>33c–35c</b>), 86% yield (for <b>33c</b>), 82% yield (for <b>34c</b>), and 82% yield (for <b>35c</b>).</p></p></figure><figure data-id="fig2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/medium/jm0c00630_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Docked poses of <b>30</b> (yellow) and <b>1d</b> (pink) with the RhlR homology model. Red and black dotted lines represent π–π stacking and hydrogen-bonding interactions, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00630&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/medium/jm0c00630_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. <i>In vitro</i> biological activities of compounds <b>1c</b>, <b>1d</b>, <b>12a</b>, and <b>30</b>. DMSO (negative control) and <b>1c</b>, <b>1d</b>, and <b>12a</b> (positive controls) were used. (A) Relative RhlR activity dose–response curves. (B) Relative QS receptor (LasR and PqsR) agonism activity at 0.1 and 10 μM of compounds, respectively. (C) Relative QS receptor (LasR and PqsR) antagonism activity at 0.1 and 10 μM of compounds, respectively. (D) Static biofilm formation of <i>P. aeruginosa</i> at 10 or 100 μM of compounds. (**) <i>P</i> < 0.005 and (*) <i>P</i> < 0.05 as compared with the control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00630&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/medium/jm0c00630_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Biofilm formation by <i>P. aeruginosa</i> with compounds <b>1c</b>, <b>1d</b>, and <b>30</b> at 10 μM in flow conditions. DMSO (negative control) and <b>1c</b>, <b>1d</b>, and <b>12a</b> (positive controls) were used. (A) CLSM images with biofilm volume and thickness. (B) Amount of carbohydrate in extracellular polymeric substance (EPS). (C) Amount of protein in EPS. (**) <i>P</i> < 0.005 and (*) <i>P</i> < 0.05 as compared with the control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00630&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/medium/jm0c00630_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Production of virulence factors and expression of QS-induced genes by <i>P. aeruginosa</i> with compounds <b>1c</b>, <b>1d</b>, <b>12a</b>, and <b>30</b>. DMSO (C, negative control) and <b>1c</b>, <b>1d</b>, and <b>12a</b> (positive controls) were used. (A) Rhamnolipid production by wild-type <i>P. aeruginosa</i> at 10 or 100 μM compound. (B) Relative <i>rhlA</i> expression of biofilm cells at 10 μM compound. (C) Pyocyanin production by wild-type <i>P. aeruginosa</i> and <i>rhlR</i> mutants at 10 μM compound. (D) Relative QS-related gene expression of wild-type <i>P. aeruginosa</i> and <i>rhlR</i> mutants. (**) <i>P</i> < 0.005 and (*) <i>P</i> < 0.05 as compared with the control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00630&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/medium/jm0c00630_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Mortality of <i>T. molitor</i> larvae injected <i>P. aeruginosa</i> with compounds <b>1c</b>, <b>1d</b>, <b>12a</b>, and <b>30</b> at 10 μM. DMSO (negative control) and <b>1c</b>, <b>1d</b>, and <b>12a</b> (positive controls) were used.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00630&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/medium/jm0c00630_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Working model to control the QS system of <i>P. aeruginosa</i> by compound <b>30</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00630/20200806/images/large/jm0c00630_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00630&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i104">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58776" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58776" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 42 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Delden, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iglewski, B. H.</span></span> <span> </span><span class="NLM_article-title">Cell-to-cell signaling and Pseudomonas aeruginosa infections</span>. <i>Emerging Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">551</span>– <span class="NLM_lpage">560</span>, <span class="refDoi"> DOI: 10.3201/eid0404.980405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.3201%2Feid0404.980405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=9866731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADyaK1MXptlam" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1998&pages=551-560&author=C.+Van%0ADeldenauthor=B.+H.+Iglewski&title=Cell-to-cell+signaling+and+Pseudomonas+aeruginosa+infections&doi=10.3201%2Feid0404.980405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Cell-to-cell signaling and Pseudomonas aeruginosa infections</span></div><div class="casAuthors">Van Delden, Christian; Iglewski, Barbara H.</div><div class="citationInfo"><span class="NLM_cas:title">Emerging Infectious Diseases</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">551-560</span>CODEN:
                <span class="NLM_cas:coden">EIDIFA</span>;
        ISSN:<span class="NLM_cas:issn">1080-6040</span>.
    
            (<span class="NLM_cas:orgname">National Center for Infectious Diseases, Centers for Disease Control and Prevention</span>)
        </div><div class="casAbstract">A review with 82 refs.  Pseudomonas aeruginosa is a bacterium responsible for severe nosocomial infections, life-threatening infections in immunocompromised persons, and chronic infections in cystic fibrosis patients.  The bacterium's virulence depends on a large no. of cell-assocd. and extracellular factors.  Cell-to-cell signaling systems control the expression and allow a coordinated, cell-d.-dependent prodn. of many extracellular virulence factors.  The authors discuss the possible role of cell-to-cell signaling in the pathogenesis of P. aeruginosa infections and present a rationale for targeting cell-to-cell signaling systems in the development of new therapeutic approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-y7jqiLUG4rVg90H21EOLACvtfcHk0lih4EVPQN44WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXptlam&md5=ddb679006ff63cad39e2ffd22297935c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.3201%2Feid0404.980405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3201%252Feid0404.980405%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDelden%26aufirst%3DC.%26aulast%3DIglewski%26aufirst%3DB.%2BH.%26atitle%3DCell-to-cell%2520signaling%2520and%2520Pseudomonas%2520aeruginosa%2520infections%26jtitle%3DEmerging%2520Infect.%2520Dis.%26date%3D1998%26volume%3D4%26spage%3D551%26epage%3D560%26doi%3D10.3201%2Feid0404.980405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bodey, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolivar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fainstein, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadeja, L.</span></span> <span> </span><span class="NLM_article-title">Infections caused by Pseudomonas aeruginosa</span>. <i>Rev. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">279</span>– <span class="NLM_lpage">313</span>, <span class="refDoi"> DOI: 10.1093/clinids/5.2.279</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1093%2Fclinids%2F5.2.279" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=6405475" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A280%3ADyaL3s7ps1Ojtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1983&pages=279-313&author=G.+P.+Bodeyauthor=R.+Bolivarauthor=V.+Fainsteinauthor=L.+Jadeja&title=Infections+caused+by+Pseudomonas+aeruginosa&doi=10.1093%2Fclinids%2F5.2.279"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Infections caused by Pseudomonas aeruginosa</span></div><div class="casAuthors">Bodey G P; Bolivar R; Fainstein V; Jadeja L</div><div class="citationInfo"><span class="NLM_cas:title">Reviews of infectious diseases</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">279-313</span>
        ISSN:<span class="NLM_cas:issn">0162-0886</span>.
    </div><div class="casAbstract">Pseudomonas aeruginosa has emerged as an important pathogen during the past two decades.  It causes between 10% and 20% of infections in most hospitals.  Pseudomonas infection is especially prevalent among patients with burn wounds, cystic fibrosis, acute leukemia, organ transplants, and intravenous-drug addiction.  P. aeruginosa is a common nosocomial contaminant, and epidemics have been traced to many items in the hospital environment.  Patients who are hospitalized for extended periods are frequently colonized by this organism and are at increased risk of developing infection.  The most serious infections include malignant external otitis, endophthalmitis, endocarditis, meningitis, pneumonia, and septicemia.  The likelihood of recovery from pseudomonas infection is related to the severity of the patient's underlying disease process.  The introduction of the antipseudomonal aminoglycosides and penicillins has improved substantially the prognosis of these infections.  Ticarcillin and carbenicillin have been especially beneficial in neutropenic patients; however, prompt institution of therapy is mandatory for optimal benefit.  Many new drugs with antipseudomonal activity, including penicillins, cephalosporins, and other beta-lactams, have been introduced in recent years and offer the potential for new approaches to therapy for these infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRWY-abeIF33Fm8bhe6roZYfW6udTcc2eZf2YPeb1Ir5rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL3s7ps1Ojtg%253D%253D&md5=05e486264586ee11af383e23c8bdb83a</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1093%2Fclinids%2F5.2.279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fclinids%252F5.2.279%26sid%3Dliteratum%253Aachs%26aulast%3DBodey%26aufirst%3DG.%2BP.%26aulast%3DBolivar%26aufirst%3DR.%26aulast%3DFainstein%26aufirst%3DV.%26aulast%3DJadeja%26aufirst%3DL.%26atitle%3DInfections%2520caused%2520by%2520Pseudomonas%2520aeruginosa%26jtitle%3DRev.%2520Infect.%2520Dis.%26date%3D1983%26volume%3D5%26spage%3D279%26epage%3D313%26doi%3D10.1093%2Fclinids%2F5.2.279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Potron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poirel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordmann, P.</span></span> <span> </span><span class="NLM_article-title">Emerging broad-spectrum resistance in Pseudomonas aeruginosa and Acinetobacter baumannii: Mechanisms and epidemiology</span>. <i>Int. J. Antimicrob. Agents</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">568</span>– <span class="NLM_lpage">585</span>, <span class="refDoi"> DOI: 10.1016/j.ijantimicag.2015.03.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1016%2Fj.ijantimicag.2015.03.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=25857949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsFGqsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2015&pages=568-585&author=A.+Potronauthor=L.+Poirelauthor=P.+Nordmann&title=Emerging+broad-spectrum+resistance+in+Pseudomonas+aeruginosa+and+Acinetobacter+baumannii%3A+Mechanisms+and+epidemiology&doi=10.1016%2Fj.ijantimicag.2015.03.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging broad-spectrum resistance in Pseudomonas aeruginosa and Acinetobacter baumannii: Mechanisms and epidemiology</span></div><div class="casAuthors">Potron, Anais; Poirel, Laurent; Nordmann, Patrice</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Antimicrobial Agents</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">568-585</span>CODEN:
                <span class="NLM_cas:coden">IAAGEA</span>;
        ISSN:<span class="NLM_cas:issn">0924-8579</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Multidrug resistance is quite common among non-fermenting Gram-neg. rods, in particular among clin. relevant species including Pseudomonas aeruginosa and Acinetobacter baumannii.  These bacterial species, which are mainly nosocomial pathogens, possess a diversity of resistance mechanisms that may lead to multidrug or even pandrug resistance.  Extended-spectrum β-lactamases (ESBLs) conferring resistance to broad-spectrum cephalosporins, carbapenemases conferring resistance to carbapenems, and 16S rRNA methylases conferring resistance to all clin. relevant aminoglycosides are the most important causes of concern.  Concomitant resistance to fluoroquinolones, polymyxins (colistin) and tigecycline may lead to pandrug resistance.  The most important mechanisms of resistance in P. aeruginosa and A. baumannii and their most recent dissemination worldwide are detailed here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofJ_AESIF1RLVg90H21EOLACvtfcHk0lgQR_NmWaTl6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsFGqsLc%253D&md5=112b4b11ae62345c2f0493d66540b4f2</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.ijantimicag.2015.03.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijantimicag.2015.03.001%26sid%3Dliteratum%253Aachs%26aulast%3DPotron%26aufirst%3DA.%26aulast%3DPoirel%26aufirst%3DL.%26aulast%3DNordmann%26aufirst%3DP.%26atitle%3DEmerging%2520broad-spectrum%2520resistance%2520in%2520Pseudomonas%2520aeruginosa%2520and%2520Acinetobacter%2520baumannii%253A%2520Mechanisms%2520and%2520epidemiology%26jtitle%3DInt.%2520J.%2520Antimicrob.%2520Agents%26date%3D2015%26volume%3D45%26spage%3D568%26epage%3D585%26doi%3D10.1016%2Fj.ijantimicag.2015.03.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pendleton, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorman, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilmore, B. F.</span></span> <span> </span><span class="NLM_article-title">Clinical relevance of the ESKAPE pathogens</span>. <i>Expert Rev. Anti-Infect. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">297</span>– <span class="NLM_lpage">308</span>, <span class="refDoi"> DOI: 10.1586/eri.13.12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1586%2Feri.13.12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=23458769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlKisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=297-308&author=J.+N.+Pendletonauthor=S.+P.+Gormanauthor=B.+F.+Gilmore&title=Clinical+relevance+of+the+ESKAPE+pathogens&doi=10.1586%2Feri.13.12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical relevance of the ESKAPE pathogens</span></div><div class="casAuthors">Pendleton, Jack N.; Gorman, Sean P.; Gilmore, Brendan F.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Review of Anti-Infective Therapy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">297-308</span>CODEN:
                <span class="NLM_cas:coden">ERATCK</span>;
        ISSN:<span class="NLM_cas:issn">1478-7210</span>.
    
            (<span class="NLM_cas:orgname">Expert Reviews Ltd.</span>)
        </div><div class="casAbstract">A review.  In recent years, the Infectious Diseases Society of America has highlighted a faction of antibiotic-resistant bacteria (Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa and Enterobacter spp.) - acronymically dubbed the ESKAPE pathogens' - capable of escaping' the biocidal action of antibiotics and mutually representing new paradigms in pathogenesis, transmission and resistance.  This review aims to consolidate clin. relevant background information on the ESKAPE pathogens and provide a contemporary summary of bacterial resistance, alongside pertinent microbiol. considerations necessary to face the mounting threat of antimicrobial resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql_dEmz1UF37Vg90H21EOLACvtfcHk0lgQR_NmWaTl6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlKisbc%253D&md5=7168500f85d84236a255e20dbf18930a</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1586%2Feri.13.12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1586%252Feri.13.12%26sid%3Dliteratum%253Aachs%26aulast%3DPendleton%26aufirst%3DJ.%2BN.%26aulast%3DGorman%26aufirst%3DS.%2BP.%26aulast%3DGilmore%26aufirst%3DB.%2BF.%26atitle%3DClinical%2520relevance%2520of%2520the%2520ESKAPE%2520pathogens%26jtitle%3DExpert%2520Rev.%2520Anti-Infect.%2520Ther.%26date%3D2013%26volume%3D11%26spage%3D297%26epage%3D308%26doi%3D10.1586%2Feri.13.12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Flemming, H.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neu, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wozniak, D. J.</span></span> <span> </span><span class="NLM_article-title">The EPS matrix: The “house of biofilm cells”</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>189</i></span>,  <span class="NLM_fpage">7945</span>– <span class="NLM_lpage">7947</span>, <span class="refDoi"> DOI: 10.1128/jb.00858-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1128%2Fjb.00858-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=17675377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlyht7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=189&publication_year=2007&pages=7945-7947&author=H.-C.+Flemmingauthor=T.+R.+Neuauthor=D.+J.+Wozniak&title=The+EPS+matrix%3A+The+%E2%80%9Chouse+of+biofilm+cells%E2%80%9D&doi=10.1128%2Fjb.00858-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The EPS matrix: the "house of biofilm cells</span></div><div class="casAuthors">Flemming, Hans-Curt; Neu, Thomas R.; Wozniak, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">189</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7945-7947</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5pGo1ZUBw7Vg90H21EOLACvtfcHk0lgQR_NmWaTl6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlyht7zN&md5=b2865c83312667d3fff6accd52d19a3d</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1128%2Fjb.00858-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fjb.00858-07%26sid%3Dliteratum%253Aachs%26aulast%3DFlemming%26aufirst%3DH.-C.%26aulast%3DNeu%26aufirst%3DT.%2BR.%26aulast%3DWozniak%26aufirst%3DD.%2BJ.%26atitle%3DThe%2520EPS%2520matrix%253A%2520The%2520%25E2%2580%259Chouse%2520of%2520biofilm%2520cells%25E2%2580%259D%26jtitle%3DJ.%2520Bacteriol.%26date%3D2007%26volume%3D189%26spage%3D7945%26epage%3D7947%26doi%3D10.1128%2Fjb.00858-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costerton, J. W.</span></span> <span> </span><span class="NLM_article-title">Overview of microbial biofilms</span>. <i>J. Ind. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">137</span>– <span class="NLM_lpage">140</span>, <span class="refDoi"> DOI: 10.1007/bf01569816</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1007%2FBF01569816" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=8519468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADyaK2MXovVKmt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1995&pages=137-140&author=J.+W.+Costerton&title=Overview+of+microbial+biofilms&doi=10.1007%2Fbf01569816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of microbial biofilms</span></div><div class="casAuthors">Costerton, J. W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Industrial Microbiology</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">137-40</span>CODEN:
                <span class="NLM_cas:coden">JIMIE7</span>;
        ISSN:<span class="NLM_cas:issn">0169-4146</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Scientific & Medical Division</span>)
        </div><div class="casAbstract">A review with 26 refs., stressing the common features of microbial biofilms that should be remembered as this concept is used to seek to understand bacterial behavior in hundreds of aquatic ecosystems studied by dozens of subspecies of microbiologists reared in sharply different scientific and academic conventions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVZo-nAqeZA7Vg90H21EOLACvtfcHk0ljKLzYSiWZNXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXovVKmt70%253D&md5=c51547fec50900c404293da856b009b0</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1007%2FBF01569816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF01569816%26sid%3Dliteratum%253Aachs%26aulast%3DCosterton%26aufirst%3DJ.%2BW.%26atitle%3DOverview%2520of%2520microbial%2520biofilms%26jtitle%3DJ.%2520Ind.%2520Microbiol.%26date%3D1995%26volume%3D15%26spage%3D137%26epage%3D140%26doi%3D10.1007%2Fbf01569816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">William Costerton, J.</span></span> <span> </span><span class="NLM_article-title">Antibiotic resistance of bacteria in biofilms</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>358</i></span>,  <span class="NLM_fpage">135</span>– <span class="NLM_lpage">138</span>, <span class="refDoi"> DOI: 10.1016/s0140-6736(01)05321-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1016%2FS0140-6736%2801%2905321-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=11463434" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlt1Shsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=358&publication_year=2001&pages=135-138&author=P.+S.+Stewartauthor=J.+William+Costerton&title=Antibiotic+resistance+of+bacteria+in+biofilms&doi=10.1016%2Fs0140-6736%2801%2905321-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotic resistance of bacteria in biofilms</span></div><div class="casAuthors">Stewart, P. S.; William Costerton, J.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">358</span>
        (<span class="NLM_cas:issue">9276</span>),
    <span class="NLM_cas:pages">135-138</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Lancet Ltd.</span>)
        </div><div class="casAbstract">A brief review with 40 refs.  Bacteria that adhere to implanted medical devices or damaged tissue can encase themselves in a hydrated matrix of polysaccharide and protein, and form a slimy layer known as a biofilm.  Antibiotic resistance of bacteria in the biofilm mode of growth contributes to the chronicity of infections such as those assocd. with implanted medical devices.  The mechanisms of resistance in biofilms are different from the now familiar plasmids, transposons, and mutations that confer innate resistance to individual bacterial cells.  In biofilms, resistance seems to depend on multicellular strategies.  We summarize the features of biofilm infections, review emerging mechanisms of resistance, and discuss potential therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonpYgJ4rKbWrVg90H21EOLACvtfcHk0ljKLzYSiWZNXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlt1Shsbs%253D&md5=c97535521d93dc62ae2e2a69556ff0a8</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2801%2905321-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252801%252905321-1%26sid%3Dliteratum%253Aachs%26aulast%3DStewart%26aufirst%3DP.%2BS.%26aulast%3DWilliam%2BCosterton%26aufirst%3DJ.%26atitle%3DAntibiotic%2520resistance%2520of%2520bacteria%2520in%2520biofilms%26jtitle%3DLancet%26date%3D2001%26volume%3D358%26spage%3D135%26epage%3D138%26doi%3D10.1016%2Fs0140-6736%2801%2905321-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bassler, B. L.</span></span> <span> </span><span class="NLM_article-title">How bacteria talk to each other: Regulation of gene expression by quorum sensing</span>. <i>Curr. Opin. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">582</span>– <span class="NLM_lpage">587</span>, <span class="refDoi"> DOI: 10.1016/s1369-5274(99)00025-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1016%2FS1369-5274%2899%2900025-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10607620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BD3cXivF2jsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=1999&pages=582-587&author=B.+L.+Bassler&title=How+bacteria+talk+to+each+other%3A+Regulation+of+gene+expression+by+quorum+sensing&doi=10.1016%2Fs1369-5274%2899%2900025-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">How bacteria talk to each other: regulation of gene expression by quorum sensing</span></div><div class="casAuthors">Bassler, Bonnie L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Microbiology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">582-587</span>CODEN:
                <span class="NLM_cas:coden">COMIF7</span>;
        ISSN:<span class="NLM_cas:issn">1369-5274</span>.
    
            (<span class="NLM_cas:orgname">Current Biology Publications</span>)
        </div><div class="casAbstract">A review with 58 refs.  Quorum sensing, or the control of gene expression in response to cell d., is used by both Gram-neg. and Gram-pos. bacteria to regulate a variety of physiol. functions.  In all cases, quorum sensing involves the prodn. and detection of extracellular signalling mols. called autoinducers.  While universal signalling themes exist, variations in the design of the extracellular signals, the signal detection apparatuses, and the biochem. mechanisms of signal relay have allowed quorum sensing systems to be exquisitely adapted for their varied uses.  Recent studies show that quorum sensing modulates both intra- and inter-species cell-cell communication, and it plays a major role in enabling bacteria to architect complex community structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbtw2G8PkKx7Vg90H21EOLACvtfcHk0ljKLzYSiWZNXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXivF2jsA%253D%253D&md5=e3409a7308c52997a4240d2459bc0e13</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2FS1369-5274%2899%2900025-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1369-5274%252899%252900025-9%26sid%3Dliteratum%253Aachs%26aulast%3DBassler%26aufirst%3DB.%2BL.%26atitle%3DHow%2520bacteria%2520talk%2520to%2520each%2520other%253A%2520Regulation%2520of%2520gene%2520expression%2520by%2520quorum%2520sensing%26jtitle%3DCurr.%2520Opin.%2520Microbiol.%26date%3D1999%26volume%3D2%26spage%3D582%26epage%3D587%26doi%3D10.1016%2Fs1369-5274%2899%2900025-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassler, B. L.</span></span> <span> </span><span class="NLM_article-title">Quorum sensing in bacteria</span>. <i>Annu. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">165</span>– <span class="NLM_lpage">199</span>, <span class="refDoi"> DOI: 10.1146/annurev.micro.55.1.165</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1146%2Fannurev.micro.55.1.165" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=11544353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnslKjt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2001&pages=165-199&author=M.+B.+Millerauthor=B.+L.+Bassler&title=Quorum+sensing+in+bacteria&doi=10.1146%2Fannurev.micro.55.1.165"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Quorum sensing in bacteria</span></div><div class="casAuthors">Miller, Melissa B.; Bassler, Bonnie L.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Microbiology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">165-199</span>CODEN:
                <span class="NLM_cas:coden">ARMIAZ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4227</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Quorum sensing is the regulation of gene expression in response to fluctuations in cell-population d.  Quorum sensing bacteria produce and release chem. signal mols. called autoinducers that increase in concn. as a function of cell d.  The detection of a minimal threshold stimulatory concn. of an autoinducer leads to an alteration in gene expression.  Gram-pos. and Gram-neg. bacteria use quorum sensing communication circuits to regulate a diverse array of physiol. activities.  These processes include symbiosis, virulence, competence, conjugation, antibiotic prodn., motility, sporulation, and biofilm formation.  In general, Gram-neg. bacteria use acylated homoserine lactones as autoinducers, and Gram-pos. bacteria use processed oligo-peptides to communicate.  Recent advances in the field indicate that cell-cell communication via autoinducers occurs both within and between bacterial species.  Furthermore, there is mounting data suggesting that bacterial autoinducers elicit specific responses from host organisms.  Although the nature of the chem. signals, the signal relay mechanisms, and the target genes controlled by bacterial quorum sensing systems differ, in every case the ability to communicate with one another allows bacteria to coordinate the gene expression, and therefore the behavior, of the entire community.  Presumably, this process bestows upon bacteria some of the qualities of higher organisms.  The evolution of quorum sensing systems in bacteria could, therefore, have been one of the early steps in the development of multicellularity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxPVpte_A5T7Vg90H21EOLACvtfcHk0ljKLzYSiWZNXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnslKjt70%253D&md5=55a377efef69610fd36f46ebfa3cb536</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1146%2Fannurev.micro.55.1.165&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.micro.55.1.165%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DM.%2BB.%26aulast%3DBassler%26aufirst%3DB.%2BL.%26atitle%3DQuorum%2520sensing%2520in%2520bacteria%26jtitle%3DAnnu.%2520Rev.%2520Microbiol.%26date%3D2001%26volume%3D55%26spage%3D165%26epage%3D199%26doi%3D10.1146%2Fannurev.micro.55.1.165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iglewski, B. H.</span></span> <span> </span><span class="NLM_article-title">P. aeruginosa quorum-sensing systems and virulence</span>. <i>Curr. Opin. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">56</span>– <span class="NLM_lpage">60</span>, <span class="refDoi"> DOI: 10.1016/s1369-5274(03)00008-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1016%2FS1369-5274%2803%2900008-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=12615220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhs1Kktb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2003&pages=56-60&author=R.+Smithauthor=B.+H.+Iglewski&title=P.+aeruginosa+quorum-sensing+systems+and+virulence&doi=10.1016%2Fs1369-5274%2803%2900008-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">P. aeruginosa quorum sensing systems and virulence</span></div><div class="casAuthors">Smith, Roger S.; Iglewski, Barbara H.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Microbiology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-60</span>CODEN:
                <span class="NLM_cas:coden">COMIF7</span>;
        ISSN:<span class="NLM_cas:issn">1369-5274</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Quorum sensing is an important mechanism for the regulation of genes in many Gram-neg. and Gram-pos. bacteria.  In the opportunistic pathogen Pseudomonas aeruginosa, the absence of one or more components of the quorum-sensing system results in a significant redn. in virulence.  Recent advances in the past year have demonstrated that the quorum-sensing signal mol. 3O-C12-HSL is also a potent stimulator of multiple eukaryotic cells and thus may alter the host response during P. aeruginosa infections.  Therefore, via the regulation of multiple factors and the prodn. of 3O-C12-HSL, quorum-sensing systems have a significant effect on the virulence of the bacteria and also on how the host responds to P. aeruginosa infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYjqAmBbtql7Vg90H21EOLACvtfcHk0lge4w5LWAT6iA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhs1Kktb8%253D&md5=0c8ce2ae45b8d39f6472ccf413d458ea</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2FS1369-5274%2803%2900008-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1369-5274%252803%252900008-0%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DR.%26aulast%3DIglewski%26aufirst%3DB.%2BH.%26atitle%3DP.%2520aeruginosa%2520quorum-sensing%2520systems%2520and%2520virulence%26jtitle%3DCurr.%2520Opin.%2520Microbiol.%26date%3D2003%26volume%3D6%26spage%3D56%26epage%3D60%26doi%3D10.1016%2Fs1369-5274%2803%2900008-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pesci, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seed, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iglewski, B. H.</span></span> <span> </span><span class="NLM_article-title">Regulation of las and rhl quorum sensing in Pseudomonas aeruginosa</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>179</i></span>,  <span class="NLM_fpage">3127</span>– <span class="NLM_lpage">3132</span>, <span class="refDoi"> DOI: 10.1128/jb.179.10.3127-3132.1997</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1128%2Fjb.179.10.3127-3132.1997" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=9150205" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADyaK2sXjtFGlsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=1997&pages=3127-3132&author=E.+C.+Pesciauthor=J.+P.+Pearsonauthor=P.+C.+Seedauthor=B.+H.+Iglewski&title=Regulation+of+las+and+rhl+quorum+sensing+in+Pseudomonas+aeruginosa&doi=10.1128%2Fjb.179.10.3127-3132.1997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of las and rhl quorum sensing in Pseudomonas aeruginosa</span></div><div class="casAuthors">Pesci, Everett; Pearson, James P.; Seed, Patrick C.; Iglewski, Barbara H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">179</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3127-3132</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The prodn. of several virulence factors by Pseudomonas aeruginosa is controlled according to cell d. through two quorum-sensing systems, las and rhl.  The las system is comprised of the transcriptional activator protein LasR and of LasI, which directs the synthesis of the autoinducer PAI-1.  Similarly, the rhl system consists of the transcriptional activator protein RhIR and of RhlI, which directs synthesis of the autoinducer PAI-2 (formerly referred to as factor 2).  To study the interrelation between the two P. aeruginosa quorum-sensing systems, we fused a lacZ reporter gene to lasR, rhlR, and rhlA and monitored expression of these three genes under various conditions.  Our data indicate that lasR and rhlR are expressed in a growth-dependent manner, with activation of each gene occurring during the last half of log-phase growth.  We also show that the las quorum-sensing system controls the rhl quorum-sensing system in two ways.  First, we found that LasR and PAI-1 activated rhlR transcription.  Second, we showed that PAI-1 blocked PAI-2 from binding to RhlR, thereby inhibiting the expression of rhlA.  Our data thus indicate that the las system exerts two levels of control on RhlR, transcriptional and posttranslational.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_cg-ekDpYwbVg90H21EOLACvtfcHk0lge4w5LWAT6iA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjtFGlsLk%253D&md5=fd8dcf9774aec865a6c36bd3b3a0a536</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1128%2Fjb.179.10.3127-3132.1997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fjb.179.10.3127-3132.1997%26sid%3Dliteratum%253Aachs%26aulast%3DPesci%26aufirst%3DE.%2BC.%26aulast%3DPearson%26aufirst%3DJ.%2BP.%26aulast%3DSeed%26aufirst%3DP.%2BC.%26aulast%3DIglewski%26aufirst%3DB.%2BH.%26atitle%3DRegulation%2520of%2520las%2520and%2520rhl%2520quorum%2520sensing%2520in%2520Pseudomonas%2520aeruginosa%26jtitle%3DJ.%2520Bacteriol.%26date%3D1997%26volume%3D179%26spage%3D3127%26epage%3D3132%26doi%3D10.1128%2Fjb.179.10.3127-3132.1997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilder, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diggle, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuster, M.</span></span> <span> </span><span class="NLM_article-title">Cooperation and cheating in Pseudomonas aeruginosa: The roles of the las, rhl and pqs quorum-sensing systems</span>. <i>ISME J.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1332</span>– <span class="NLM_lpage">1343</span>, <span class="refDoi"> DOI: 10.1038/ismej.2011.13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1038%2Fismej.2011.13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=21368905" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptVynurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2011&pages=1332-1343&author=C.+N.+Wilderauthor=S.+P.+Diggleauthor=M.+Schuster&title=Cooperation+and+cheating+in+Pseudomonas+aeruginosa%3A+The+roles+of+the+las%2C+rhl+and+pqs+quorum-sensing+systems&doi=10.1038%2Fismej.2011.13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Cooperation and cheating in Pseudomonas aeruginosa: the roles of the las, rhl and pqs quorum-sensing systems</span></div><div class="casAuthors">Wilder, Cara N.; Diggle, Stephen P.; Schuster, Martin</div><div class="citationInfo"><span class="NLM_cas:title">ISME Journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1332-1343, S1332/1-S1332/9</span>CODEN:
                <span class="NLM_cas:coden">IJSOCF</span>;
        ISSN:<span class="NLM_cas:issn">1751-7362</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Pseudomonas aeruginosa coordinates the transcription of hundreds of genes, including many virulence genes, through three hierarchically arranged quorum-sensing (QS) systems, namely las, rhl and pqs.  Each system consists of genes involved in autoinducer synthesis, lasI, rhlI and pqsABCDH, as well as cognate-regulatory genes, lasR, rhlR and pqsR.  In this study, we analyzed the social behavior of signal-blind (ΔlasR, ΔrhlR, ΔpqsR) and signal-neg. (ΔlasI, ΔrhlI, ΔpqsA) mutants from each QS system.  As each system controls extracellular common goods but differs in the extent of regulatory control, we hypothesized that all signal-blind mutants can behave as cheaters that vary in their ability to invade a QS-proficient population.  We found that lasR and pqsR, but not rhlR, mutants evolve from a wild-type ancestor in vitro under conditions that favor QS.  Accordingly, defined lasR and pqsR mutants enriched in wild-type co-culture, whereas rhlR and all signal-neg. mutants did not.  Both lasR and pqsR mutants enriched with neg. frequency dependence, suggesting social interactions with the wild type, although the pqsR mutant also grew well on its own.  Taken together, the lasR mutant behaved as a typical cheater, as reported previously.  However, the pqsR and rhlR mutants exhibited more complex behaviors, which can be sufficiently explained by pos. and neg. pleiotropic effects through differential regulation of pqs gene expression in the interconnected QS network.  The evolutionary approach adopted here may account for the prevalence of naturally occurring QS mutants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7eGlUKXXMZbVg90H21EOLACvtfcHk0lge4w5LWAT6iA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptVynurw%253D&md5=9a8b62bd7b9825a92a88ed696280fc4f</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fismej.2011.13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fismej.2011.13%26sid%3Dliteratum%253Aachs%26aulast%3DWilder%26aufirst%3DC.%2BN.%26aulast%3DDiggle%26aufirst%3DS.%2BP.%26aulast%3DSchuster%26aufirst%3DM.%26atitle%3DCooperation%2520and%2520cheating%2520in%2520Pseudomonas%2520aeruginosa%253A%2520The%2520roles%2520of%2520the%2520las%252C%2520rhl%2520and%2520pqs%2520quorum-sensing%2520systems%26jtitle%3DISME%2520J.%26date%3D2011%26volume%3D5%26spage%3D1332%26epage%3D1343%26doi%3D10.1038%2Fismej.2011.13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Givskov, M.</span></span> <span> </span><span class="NLM_article-title">Quorum-sensing inhibitors as anti-pathogenic drugs</span>. <i>Int. J. Med. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>296</i></span>,  <span class="NLM_fpage">149</span>– <span class="NLM_lpage">161</span>, <span class="refDoi"> DOI: 10.1016/j.ijmm.2006.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1016%2Fj.ijmm.2006.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=16503194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BD28XksFSru70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=296&publication_year=2006&pages=149-161&author=T.+B.+Rasmussenauthor=M.+Givskov&title=Quorum-sensing+inhibitors+as+anti-pathogenic+drugs&doi=10.1016%2Fj.ijmm.2006.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Quorum-sensing inhibitors as anti-pathogenic drugs</span></div><div class="casAuthors">Rasmussen, Thomas B.; Givskov, Michael</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Medical Microbiology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">296</span>
        (<span class="NLM_cas:issue">2-3</span>),
    <span class="NLM_cas:pages">149-161</span>CODEN:
                <span class="NLM_cas:coden">IMEMFV</span>;
        ISSN:<span class="NLM_cas:issn">1438-4221</span>.
    
            (<span class="NLM_cas:orgname">Elsevier GmbH</span>)
        </div><div class="casAbstract">A review.  Quorum-sensing (QS) signalling systems of pathogens are central regulators for the expression of virulence factors and represent highly attractive targets for the development of novel therapeutics.  In Pseudomonas aeruginosa, QS systems are also involved in elevated antibiotic tolerance of biofilms as well as elevated tolerance to the activity of the innate immune system.  Gram-neg. bacteria commonly use N-acyl homoserine lactones (AHL) as QS signal mols.  The use of signal mol. based drugs to attenuate bacterial pathogenicity rather than bacterial growth is attractive for several reasons, particularly considering the emergence of increasingly antibiotic-resistant bacteria.  Compds. capable of this type of interference have been termed anti-pathogenic drugs.  A large variety of synthetic AHL analogs and natural products libraries have been screened and a no. of QS inhibitors (QSI) have been identified.  Promising QSI compds. have been shown to make biofilms more susceptible to antimicrobial treatments, and are capable of reducing mortality and virulence as well as promoting clearance of bacteria in exptl. animal models of infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqi6hHo5ELXLVg90H21EOLACvtfcHk0lhQESc9PzP5yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XksFSru70%253D&md5=dc7ab0360ef9fe9b8b7bda6c31e2e53d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.ijmm.2006.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijmm.2006.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DRasmussen%26aufirst%3DT.%2BB.%26aulast%3DGivskov%26aufirst%3DM.%26atitle%3DQuorum-sensing%2520inhibitors%2520as%2520anti-pathogenic%2520drugs%26jtitle%3DInt.%2520J.%2520Med.%2520Microbiol.%26date%3D2006%26volume%3D296%26spage%3D149%26epage%3D161%26doi%3D10.1016%2Fj.ijmm.2006.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noto, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nichols-O’Neill, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, L. J.</span></span> <span> </span><span class="NLM_article-title">Potent irreversible inhibitors of LasR quorum sensing in Pseudomonas aeruginosa</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">162</span>– <span class="NLM_lpage">167</span>, <span class="refDoi"> DOI: 10.1021/ml500459f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500459f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFyhtL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=162-167&author=K.+T.+O%E2%80%99Brienauthor=J.+G.+Notoauthor=L.+Nichols-O%E2%80%99Neillauthor=L.+J.+Perez&title=Potent+irreversible+inhibitors+of+LasR+quorum+sensing+in+Pseudomonas+aeruginosa&doi=10.1021%2Fml500459f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Potent Irreversible Inhibitors of LasR Quorum Sensing in Pseudomonas aeruginosa</span></div><div class="casAuthors">O'Brien, Kevin T.; Noto, Joseph G.; Nichols-O'Neill, Luke; Perez, Lark J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">162-167</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Antagonism of quorum sensing represents a promising new antivirulence approach for the treatment of bacterial infection.  The development of a novel series of non-natural irreversible antagonists of P. aeruginosa LasR is described.  The lead compds. identified (25 and 28) display potent LasR antagonist activity and inhibit expression of the P. aeruginosa virulence factors pyocyanin and biofilm formation in PAO1 and PA14.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs_0FnrDKcmbVg90H21EOLACvtfcHk0lhQESc9PzP5yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFyhtL%252FN&md5=affb26c8bcb4e9db5158491fb8c83c2a</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fml500459f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500459f%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DK.%2BT.%26aulast%3DNoto%26aufirst%3DJ.%2BG.%26aulast%3DNichols-O%25E2%2580%2599Neill%26aufirst%3DL.%26aulast%3DPerez%26aufirst%3DL.%2BJ.%26atitle%3DPotent%2520irreversible%2520inhibitors%2520of%2520LasR%2520quorum%2520sensing%2520in%2520Pseudomonas%2520aeruginosa%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D162%26epage%3D167%26doi%3D10.1021%2Fml500459f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Geske, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattmann, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackwell, H. E.</span></span> <span> </span><span class="NLM_article-title">Evaluation of a focused library of N-aryl L-homoserine lactones reveals a new set of potent quorum sensing modulators</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">5978</span>– <span class="NLM_lpage">5981</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2008.07.089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1016%2Fj.bmcl.2008.07.089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=18760602" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOht7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=5978-5981&author=G.+D.+Geskeauthor=M.+E.+Mattmannauthor=H.+E.+Blackwell&title=Evaluation+of+a+focused+library+of+N-aryl+L-homoserine+lactones+reveals+a+new+set+of+potent+quorum+sensing+modulators&doi=10.1016%2Fj.bmcl.2008.07.089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of a focused library of N-aryl -homoserine lactones reveals a new set of potent quorum sensing modulators</span></div><div class="casAuthors">Geske, Grant D.; Mattmann, Margrith E.; Blackwell, Helen E.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5978-5981</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A focused library of N-aryl -homoserine lactones was designed around known lactone leads and evaluated for antagonistic and agonistic activity against quorum-sensing receptors in Agrobacterium tumefaciens, Pseudomonas aeruginosa, and Vibrio fischeri.  Several compds. were identified with significantly heightened activities relative to the lead compds., and new structure-activity relationships (SARs) were delineated.  Notably, 4-substituted N-phenoxyacetyl and 3-substituted N-phenylpropionyl -homoserine lactones were identified as potent antagonists of TraR and LuxR, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZBUYV5E_TQbVg90H21EOLACvtfcHk0lhQESc9PzP5yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOht7jK&md5=b099166a3f71a7e8504ed440d85253d3</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.07.089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.07.089%26sid%3Dliteratum%253Aachs%26aulast%3DGeske%26aufirst%3DG.%2BD.%26aulast%3DMattmann%26aufirst%3DM.%2BE.%26aulast%3DBlackwell%26aufirst%3DH.%2BE.%26atitle%3DEvaluation%2520of%2520a%2520focused%2520library%2520of%2520N-aryl%2520L-homoserine%2520lactones%2520reveals%2520a%2520new%2520set%2520of%2520potent%2520quorum%2520sensing%2520modulators%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D5978%26epage%3D5981%26doi%3D10.1016%2Fj.bmcl.2008.07.089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McInnis, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackwell, H. E.</span></span> <span> </span><span class="NLM_article-title">Thiolactone modulators of quorum sensing revealed through library design and screening</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">4820</span>– <span class="NLM_lpage">4828</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2011.06.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1016%2Fj.bmc.2011.06.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=21798746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvFWiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=4820-4828&author=C.+E.+McInnisauthor=H.+E.+Blackwell&title=Thiolactone+modulators+of+quorum+sensing+revealed+through+library+design+and+screening&doi=10.1016%2Fj.bmc.2011.06.071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Thiolactone modulators of quorum sensing revealed through library design and screening</span></div><div class="casAuthors">McInnis, Christine E.; Blackwell, Helen E.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4820-4828</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Quorum sensing (QS) is a process by which bacteria use small mols. or peptidic signals to assess their local population densities.  At sufficiently high d., bacteria can alter gene expression levels to regulate group behaviors involved in a range of important and diverse phenotypes, including virulence factor prodn., biofilm formation, root nodulation, and bioluminescence.  Gram-neg. bacteria most commonly use N-acylated L-homoserine lactones (AHLs) as their QS signals.  The AHL lactone ring is hydrolyzed relatively rapidly at biol. pH, and the ring-opened product is QS inactive.  The authors sought to identify AHL analogs with heightened hydrolytic stability, and thereby potentially heightened activity, for use as non-native modulators of bacterial QS.  As part of this effort, they probed the utility of thiolactone analogs in the current study as QS agonists and antagonists in Gram-neg. bacteria.  A focused library of thiolactone analogs was designed and rapidly synthesized in soln.  The authors examd. the activity of the library as agonists and antagonists of LuxR-type QS receptors in Pseudomonas aeruginosa (LasR), Vibrio fischeri (LuxR), and Agrobacterium tumefaciens (TraR) using bacterial reporter strains.  The thiolactone library contained several highly active compds., including some of the most active LuxR inhibitors and the most active synthetic TraR agonist reported to date.  Anal. of a representative thiolactone analog revealed that its hydrolysis half-life was almost double that of its parent AHL in bacterial growth medium.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMrPNBC5qdAbVg90H21EOLACvtfcHk0liPyKYJNLkrag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvFWiurg%253D&md5=8c51d9121e29019218e48b6403d48684</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.06.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.06.071%26sid%3Dliteratum%253Aachs%26aulast%3DMcInnis%26aufirst%3DC.%2BE.%26aulast%3DBlackwell%26aufirst%3DH.%2BE.%26atitle%3DThiolactone%2520modulators%2520of%2520quorum%2520sensing%2520revealed%2520through%2520library%2520design%2520and%2520screening%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D4820%26epage%3D4828%26doi%3D10.1016%2Fj.bmc.2011.06.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oshri, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zrihen, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shner, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omer Bendori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eldar, A.</span></span> <span> </span><span class="NLM_article-title">Selection for increased quorum-sensing cooperation in Pseudomonas aeruginosa through the shut-down of a drug resistance pump</span>. <i>ISME J.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2458</span>– <span class="NLM_lpage">2469</span>, <span class="refDoi"> DOI: 10.1038/s41396-018-0205-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1038%2Fs41396-018-0205-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=29925881" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFyjtL%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2018&pages=2458-2469&author=R.+D.+Oshriauthor=K.+S.+Zrihenauthor=I.+Shnerauthor=S.+Omer+Bendoriauthor=A.+Eldar&title=Selection+for+increased+quorum-sensing+cooperation+in+Pseudomonas+aeruginosa+through+the+shut-down+of+a+drug+resistance+pump&doi=10.1038%2Fs41396-018-0205-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Selection for increased quorum-sensing cooperation in Pseudomonas aeruginosa through the shut-down of a drug resistance pump</span></div><div class="casAuthors">Oshri, Ron D.; Zrihen, Keren S.; Shner, Itzhak; Omer Bendori, Shira; Eldar, Avigdor</div><div class="citationInfo"><span class="NLM_cas:title">ISME Journal</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2458-2469</span>CODEN:
                <span class="NLM_cas:coden">IJSOCF</span>;
        ISSN:<span class="NLM_cas:issn">1751-7362</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The opportunistic pathogen Pseudomonas aeruginosa employs a hierarchical quorum-sensing network to regulate virulence factor prodn. that cooperatively benefit the population at a cost to the individual.  It has been argued that the evolution of a cooperative mutant in a quorum sensing-suppressed population would be hampered through its exploitation by neighboring non-mutant cells.  It remains unclear whether mechanisms which overcome this exploitation exist.  Here we investigate the regain of quorum-sensing cooperation by evolving a mutant of the lasR master quorum-sensing regulator.  The mutant regained partial cooperative growth through null mutations in mexT, which codes for an activator of the MexEF-OprN multidrug-resistant pump.  We find that these mutations enhance cooperative growth in both the lasR mutant and wild-type backgrounds through the activation of the RhlIR system.  We show that the regain of cooperation in mexT mutants is mediated by the redn. in MexEF-OprN activity, whereas an addnl. source of private benefit is mostly mexEF-oprN-independent.  Finally, we show that addn. of antibiotics for which resistance is mediated by MexEF-OprN prevents the selection of increased cooperation at sub-MIC concns.  MexT, therefore, not only links private and public goods, but also exposes conflicts between selection for antibiotic resistance and enhanced cooperation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoG8GawxJ1EzrVg90H21EOLACvtfcHk0liPyKYJNLkrag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFyjtL%252FF&md5=fbf6aefdfb543702ec8349ba652637ba</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fs41396-018-0205-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41396-018-0205-y%26sid%3Dliteratum%253Aachs%26aulast%3DOshri%26aufirst%3DR.%2BD.%26aulast%3DZrihen%26aufirst%3DK.%2BS.%26aulast%3DShner%26aufirst%3DI.%26aulast%3DOmer%2BBendori%26aufirst%3DS.%26aulast%3DEldar%26aufirst%3DA.%26atitle%3DSelection%2520for%2520increased%2520quorum-sensing%2520cooperation%2520in%2520Pseudomonas%2520aeruginosa%2520through%2520the%2520shut-down%2520of%2520a%2520drug%2520resistance%2520pump%26jtitle%3DISME%2520J.%26date%3D2018%26volume%3D12%26spage%3D2458%26epage%3D2469%26doi%3D10.1038%2Fs41396-018-0205-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Induction and inhibition of Pseudomonas aeruginosa quorum sensing by synthetic autoinducer analogs</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">81</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1016/s1074-5521(03)00002-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1016%2FS1074-5521%2803%2900002-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=12573701" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BD3sXps1Srtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2003&pages=81-89&author=K.+M.+Smithauthor=Y.+Buauthor=H.+Suga&title=Induction+and+inhibition+of+Pseudomonas+aeruginosa+quorum+sensing+by+synthetic+autoinducer+analogs&doi=10.1016%2Fs1074-5521%2803%2900002-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Induction and Inhibition of Pseudomonas aeruginosa Quorum Sensing by Synthetic Autoinducer Analogs</span></div><div class="casAuthors">Smith, Kristina M.; Bu, Yigong; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">81-89</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We synthesized a library of Pseudomonas aeruginosa autoinducer analogs with variation targeted to the homoserine lactone (HSL) moiety and discovered a new agonist, 3-oxo-C12-(2-aminocyclohexanol), capable of activating LasR as a transcription factor.  We reconstructed two sets of focused libraries against the quorum-sensing transcription factors LasR and RhlR, resp.  Opposing the prediction that both proteins should have the same binding site for HSL, it was surprising to find that these two related proteins respond to different structural motifs.  This suggests that the HSL binding site differs in these proteins.  We also found that subtle structural modifications to the agonists yielded compds. with antagonist activity.  We performed a series of assays to show that inhibition of quorum sensing by these antagonists significantly reduced the prodn. of virulence factors and biofilm formation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKQ50GqXLterVg90H21EOLACvtfcHk0lgRbn_jwDilUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXps1Srtw%253D%253D&md5=322f1933c805207386bf14169398b93d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2FS1074-5521%2803%2900002-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1074-5521%252803%252900002-4%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DK.%2BM.%26aulast%3DBu%26aufirst%3DY.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DInduction%2520and%2520inhibition%2520of%2520Pseudomonas%2520aeruginosa%2520quorum%2520sensing%2520by%2520synthetic%2520autoinducer%2520analogs%26jtitle%3DChem.%2520Biol.%26date%3D2003%26volume%3D10%26spage%3D81%26epage%3D89%26doi%3D10.1016%2Fs1074-5521%2803%2900002-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Loughlin, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siryaporn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drescher, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semmelhack, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassler, B. L.</span></span> <span> </span><span class="NLM_article-title">A quorum-sensing inhibitor blocks Pseudomonas aeruginosa virulence and biofilm formation</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">17981</span>– <span class="NLM_lpage">17986</span>, <span class="refDoi"> DOI: 10.1073/pnas.1316981110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1073%2Fpnas.1316981110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=24143808" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVWmur3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=17981-17986&author=C.+T.+O%E2%80%99Loughlinauthor=L.+C.+Millerauthor=A.+Siryapornauthor=K.+Drescherauthor=M.+F.+Semmelhackauthor=B.+L.+Bassler&title=A+quorum-sensing+inhibitor+blocks+Pseudomonas+aeruginosa+virulence+and+biofilm+formation&doi=10.1073%2Fpnas.1316981110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">A quorum sensing inhibitor blocks Pseudomonas aeruginosa virulence and biofilm formation</span></div><div class="casAuthors">O'Loughlin, Colleen T.; Miller, Laura C.; Siryaporn, Albert; Drescher, Knut; Semmelhack, Martin F.; Bassler, Bonnie L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">17981-17986,S17981/1-S17981/56</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Quorum sensing is a chem. communication process that bacteria use to regulate collective behaviors.  Disabling quorum-sensing circuits with small mols. has been proposed as a potential strategy to prevent bacterial pathogenicity.  The human pathogen Pseudomonas aeruginosa uses quorum sensing to control virulence and biofilm formation.  Here, the authors analyze synthetic mols. for inhibition of the two P. aeruginosa quorum sensing receptors, LasR and RhlR.  The most effective compd., meta-bromo-thiolactone (mBTL), inhibits both the prodn. of the virulence factor pyocyanin and biofilm formation.  The mBTL also protects Caenorhabditis elegans and human lung epithelial cells from killing by P. aeruginosa.  Both LasR and RhlR are partially inhibited by mBTL in vivo and in vitro; however, RhlR, not LasR, is the relevant in vivo target.  More potent antagonists do not exhibit superior function in impeding virulence.  Because LasR and RhlR reciprocally control crucial virulence factors, appropriately tuning rather than completely inhibiting their activities appears to hold the key to blocking pathogenesis in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq34j__e2GFKrVg90H21EOLACvtfcHk0lgRbn_jwDilUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVWmur3F&md5=efa2b960af182ca3945d81bf70d2a37b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1316981110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1316981110%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Loughlin%26aufirst%3DC.%2BT.%26aulast%3DMiller%26aufirst%3DL.%2BC.%26aulast%3DSiryaporn%26aufirst%3DA.%26aulast%3DDrescher%26aufirst%3DK.%26aulast%3DSemmelhack%26aufirst%3DM.%2BF.%26aulast%3DBassler%26aufirst%3DB.%2BL.%26atitle%3DA%2520quorum-sensing%2520inhibitor%2520blocks%2520Pseudomonas%2520aeruginosa%2520virulence%2520and%2520biofilm%2520formation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D17981%26epage%3D17986%26doi%3D10.1073%2Fpnas.1316981110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Furiga, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lajoie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Hage, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baziard, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roques, C.</span></span> <span> </span><span class="NLM_article-title">Impairment of Pseudomonas aeruginosa biofilm resistance to antibiotics by combining the drugs with a new quorum-sensing inhibitor</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1676</span>– <span class="NLM_lpage">1686</span>, <span class="refDoi"> DOI: 10.1128/aac.02533-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1128%2FAAC.02533-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsF2ntbjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2016&pages=1676-1686&author=A.+Furigaauthor=B.+Lajoieauthor=S.+El+Hageauthor=G.+Baziardauthor=C.+Roques&title=Impairment+of+Pseudomonas+aeruginosa+biofilm+resistance+to+antibiotics+by+combining+the+drugs+with+a+new+quorum-sensing+inhibitor&doi=10.1128%2Faac.02533-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Impairment of Pseudomonas aeruginosa biofilm resistance to antibiotics by combining the drugs with a new quorum-sensing inhibitor</span></div><div class="casAuthors">Furiga, Aurelie; Lajoie, Barbora; El Hage, Salome; Baziard, Genevieve; Roques, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1676-1686</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Pseudomonas aeruginosa plays an important role in chronic lung infections among patients with cystic fibrosis (CF) through its ability to form antibiotic-resistant biofilms.  In P. aeruginosa, biofilm development and the prodn. of several virulence factors are mainly regulated by the rhl and las quorum-sensing (QS) systems, which are controlled by two N-acyl-homoserine lactone signal mols.  In a previous study, we discovered an original QS inhibitor, N-(2-pyrimidyl)butanamide, called C11, based on the structure of C4-homoserine lactone, and found that it is able to significantly inhibit P. aeruginosa biofilm formation.  However, recent data indicate that P. aeruginosa grows under anaerobic conditions and forms biofilms in the lungs of CF patients that are denser and more robust than those formed under aerobic conditions.  Our confocal microscopy observations of P. aeruginosa biofilms developed under aerobic and anaerobic conditions confirmed that the biofilms formed under these 2 conditions have radically different architectures.  C11 showed significant dose-dependent antibiofilm activity on biofilms grown under both aerobic and anaerobic conditions, with a greater inhibitory effect being seen under conditions of anaerobiosis.  Gene expression analyses performed by quant. reverse transcriptase PCR showed that C11 led to the significant downregulation of rhl QS regulatory genes but also to the downregulation of both las QS regulatory genes and QS system-regulated virulence genes, rhlA and lasB.  Furthermore, the activity of C11 in combination with antibiotics against P. aeruginosa biofilms was tested, and synergistic antibiofilm activity between C11 and ciprofloxacin, tobramycin, and colistin was obtained under both aerobic and anaerobic conditions.  This study demonstrates that C11 may increase the efficacy of treatments for P. aeruginosa infections by increasing the susceptibility of biofilms to antibiotics and by attenuating the pathogenicity of the bacterium.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-yOX6KkutobVg90H21EOLACvtfcHk0lgRbn_jwDilUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsF2ntbjI&md5=367c1cf09e9fe5d032071f44bc3cf915</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1128%2FAAC.02533-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.02533-15%26sid%3Dliteratum%253Aachs%26aulast%3DFuriga%26aufirst%3DA.%26aulast%3DLajoie%26aufirst%3DB.%26aulast%3DEl%2BHage%26aufirst%3DS.%26aulast%3DBaziard%26aufirst%3DG.%26aulast%3DRoques%26aufirst%3DC.%26atitle%3DImpairment%2520of%2520Pseudomonas%2520aeruginosa%2520biofilm%2520resistance%2520to%2520antibiotics%2520by%2520combining%2520the%2520drugs%2520with%2520a%2520new%2520quorum-sensing%2520inhibitor%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2016%26volume%3D60%26spage%3D1676%26epage%3D1686%26doi%3D10.1128%2Faac.02533-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soukarieh, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stocks, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cámara, M.</span></span> <span> </span><span class="NLM_article-title">Pseudomonas aeruginosa quorum sensing systems as drug discovery targets: Current position and future perspectives</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">10385</span>– <span class="NLM_lpage">10402</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00540</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00540" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlSmurvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=10385-10402&author=F.+Soukariehauthor=P.+Williamsauthor=M.+J.+Stocksauthor=M.+C%C3%A1mara&title=Pseudomonas+aeruginosa+quorum+sensing+systems+as+drug+discovery+targets%3A+Current+position+and+future+perspectives&doi=10.1021%2Facs.jmedchem.8b00540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Pseudomonas aeruginosa Quorum Sensing Systems as Drug Discovery Targets: Current Position and Future Perspectives</span></div><div class="casAuthors">Soukarieh, Fadi; Williams, Paul; Stocks, Michael J.; Camara, Miguel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10385-10402</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Antimicrobial resistance (AMR) is a serious threat to public health globally, manifested by the frequent emergence of multi-drug resistant pathogens that render current chemotherapy inadequate.  Health organizations worldwide have recognized the severity of this crisis and implemented action plans to contain its adverse consequences and prolong the utility of conventional antibiotics.  Hence, there is a pressing need for new classes of antibacterial agents with novel modes of action.  Quorum sensing (QS), a communication system employed by bacterial populations to coordinate virulence gene expression, is a potential target that has been intensively investigated over the last decade.  This Perspective will focus on recent advances in targeting the three main quorum sensing systems (las, rhl and pqs) of a major opportunistic human pathogen, Pseudomonas aeruginosa, and will specifically evaluate the medicinal chem. strategies devised to develop QS inhibitors from a drug discovery perspective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKqjOmBMF_u7Vg90H21EOLACvtfcHk0lhqNpodIC_w-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlSmurvE&md5=ce4ff141208ec755b91345d7a1fae049</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00540%26sid%3Dliteratum%253Aachs%26aulast%3DSoukarieh%26aufirst%3DF.%26aulast%3DWilliams%26aufirst%3DP.%26aulast%3DStocks%26aufirst%3DM.%2BJ.%26aulast%3DC%25C3%25A1mara%26aufirst%3DM.%26atitle%3DPseudomonas%2520aeruginosa%2520quorum%2520sensing%2520systems%2520as%2520drug%2520discovery%2520targets%253A%2520Current%2520position%2520and%2520future%2520perspectives%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D10385%26epage%3D10402%26doi%3D10.1021%2Facs.jmedchem.8b00540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eibergen, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattmann, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackwell, H. E.</span></span> <span> </span><span class="NLM_article-title">Potent and selective modulation of the RhlR quorum sensing receptor by using non-native ligands: An emerging target for virulence control in Pseudomonas aeruginosa</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">2348</span>– <span class="NLM_lpage">2356</span>, <span class="refDoi"> DOI: 10.1002/cbic.201500357</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1002%2Fcbic.201500357" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=26460240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1egs7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=2348-2356&author=N.+R.+Eibergenauthor=J.+D.+Mooreauthor=M.+E.+Mattmannauthor=H.+E.+Blackwell&title=Potent+and+selective+modulation+of+the+RhlR+quorum+sensing+receptor+by+using+non-native+ligands%3A+An+emerging+target+for+virulence+control+in+Pseudomonas+aeruginosa&doi=10.1002%2Fcbic.201500357"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and Selective Modulation of the RhlR Quorum Sensing Receptor by Using Non-native Ligands: An Emerging Target for Virulence Control in Pseudomonas aeruginosa</span></div><div class="casAuthors">Eibergen, Nora R.; Moore, Joseph D.; Mattmann, Margrith E.; Blackwell, Helen E.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2348-2356</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Pseudomonas aeruginosa uses N-acylated L-homoserine lactone signals and a triumvirate of LuxR-type receptor proteins (LasR, RhlR, and QscR) for quorum sensing (QS).  Each of these receptors can contribute to QS activation or repression and, thereby, the control of myriad virulence phenotypes in this pathogen.  LasR has traditionally been considered to be at the top of the QS receptor hierarchy in P. aeruginosa; however, recent reports suggest that RhlR plays a more prominent role in infection than originally predicted, in some circumstances superseding that of LasR.  Herein, the authors report the characterization of a set of synthetic, small-mol. agonists and antagonists of RhlR.  Using E. coli reporter strains, it was shown that many of these compds. can selectively activate or inhibit RhlR instead of LasR and QscR.  Moreover, several mols. maintain their activities in P. aeruginosa at concns. analogous to native RhlR signal levels.  These compds. represent useful chem. probes to study the role of RhlR in the complex QS circuitry of P. aeruginosa, its direct (and indirect) effects on virulence, and its overall merit as a target for anti-infective therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaLc7CY8sMxLVg90H21EOLACvtfcHk0lhqNpodIC_w-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1egs7%252FE&md5=5299d6c73bd982144096752d1b08dc5a</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201500357&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201500357%26sid%3Dliteratum%253Aachs%26aulast%3DEibergen%26aufirst%3DN.%2BR.%26aulast%3DMoore%26aufirst%3DJ.%2BD.%26aulast%3DMattmann%26aufirst%3DM.%2BE.%26aulast%3DBlackwell%26aufirst%3DH.%2BE.%26atitle%3DPotent%2520and%2520selective%2520modulation%2520of%2520the%2520RhlR%2520quorum%2520sensing%2520receptor%2520by%2520using%2520non-native%2520ligands%253A%2520An%2520emerging%2520target%2520for%2520virulence%2520control%2520in%2520Pseudomonas%2520aeruginosa%26jtitle%3DChemBioChem%26date%3D2015%26volume%3D16%26spage%3D2348%26epage%3D2356%26doi%3D10.1002%2Fcbic.201500357" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Library screening for synthetic agonists and antagonists of a Pseudomonas aeruginosa autoinducer</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">563</span>– <span class="NLM_lpage">571</span>, <span class="refDoi"> DOI: 10.1016/s1074-5521(03)00107-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1016%2Fs1074-5521%2803%2900107-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=12837389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BD3sXltVSqsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2003&pages=563-571&author=K.+M.+Smithauthor=Y.+Buauthor=H.+Suga&title=Library+screening+for+synthetic+agonists+and+antagonists+of+a+Pseudomonas+aeruginosa+autoinducer&doi=10.1016%2Fs1074-5521%2803%2900107-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Library Screening for Synthetic Agonists and Antagonists of a Pseudomonas aeruginosa Autoinducer</span></div><div class="casAuthors">Smith, Kristina M.; Bu, Yigong; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">563-571</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The autoinducer (AI) that initiates the quorum sensing (QS) signaling cascade in Pseudomonas aeruginosa is an acyl-homoserine lactone (acyl-HSL).  We initiated a study of the requirements for binding of the AI to its protein effector LasR by synthesizing a library of analogs with the HSL moiety replaced with different amines and alcs.  We tested each compd. for both agonist and antagonist activity using a QS-controlled reporter gene assay and found several new agonists and antagonists.  A representative antagonist was further tested for its ability to inhibit virulence factors.  This data progresses our understanding of the LasR-AI interaction toward the rational design of therapeutic inhibitors of QS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMFP7aVBkHYbVg90H21EOLACvtfcHk0lhqNpodIC_w-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXltVSqsr4%253D&md5=f3c37fa88ce2b3de8d75603cded4a8e4</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fs1074-5521%2803%2900107-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs1074-5521%252803%252900107-8%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DK.%2BM.%26aulast%3DBu%26aufirst%3DY.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DLibrary%2520screening%2520for%2520synthetic%2520agonists%2520and%2520antagonists%2520of%2520a%2520Pseudomonas%2520aeruginosa%2520autoinducer%26jtitle%3DChem.%2520Biol.%26date%3D2003%26volume%3D10%26spage%3D563%26epage%3D571%26doi%3D10.1016%2Fs1074-5521%2803%2900107-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Annapoorani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umamageswaran, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parameswari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandian, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravi, A. V.</span></span> <span> </span><span class="NLM_article-title">Computational discovery of putative quorum sensing inhibitors against LasR and RhlR receptor proteins of Pseudomonas aeruginosa</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1077</span>, <span class="refDoi"> DOI: 10.1007/s10822-012-9599-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1007%2Fs10822-012-9599-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=22986632" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFWkt7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=1067-1077&author=A.+Annapooraniauthor=V.+Umamageswaranauthor=R.+Parameswariauthor=S.+K.+Pandianauthor=A.+V.+Ravi&title=Computational+discovery+of+putative+quorum+sensing+inhibitors+against+LasR+and+RhlR+receptor+proteins+of+Pseudomonas+aeruginosa&doi=10.1007%2Fs10822-012-9599-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Computational discovery of putative quorum sensing inhibitors against LasR and RhlR receptor proteins of Pseudomonas aeruginosa</span></div><div class="casAuthors">Annapoorani, Angusamy; Umamageswaran, Venugopal; Parameswari, Radhakrishnan; Pandian, Shunmugiah Karutha; Ravi, Arumugam Veera</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1067-1077</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Drugs have been discovered in the past mainly either by identification of active components from traditional remedies or by unpredicted discovery.  A key motivation for the study of structure based virtual screening is the exploitation of such information to design targeted drugs.  In this study, structure-based virtual screening was used in the search for putative quorum sensing inhibitors (QSI) of Pseudomonas aeruginosa.  The virtual screening program Glide version 5.5 was applied to screen 1,920 natural compds./drugs against LasR and RhlR receptor proteins of P. aeruginosa.  Based on the results of in silico docking anal., five top ranking compds. namely rosmarinic acid, naringin, chlorogenic acid, morin and mangiferin were subjected to in vitro bioassays against lab. strain PAO1 and two more antibiotic-resistant clin. isolates, P. aeruginosa AS1 (GU447237) and P. aeruginosa AS2 (GU447238).  Among 5 compds. studied, all but mangiferin showed significant inhibition in the prodn. of protease, elastase and hemolysin.  Further, all five compds. potentially inhibited biofilm-related behaviors.  This interaction study provided promising ligands to inhibit the quorum sensing- mediated virulence factor prodn. in P. aeruginosa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJ4R1ilIt18rVg90H21EOLACvtfcHk0lj73y6lcXdGiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFWkt7%252FI&md5=d230c270761f761d6e1a3f8768b46ad5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9599-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9599-1%26sid%3Dliteratum%253Aachs%26aulast%3DAnnapoorani%26aufirst%3DA.%26aulast%3DUmamageswaran%26aufirst%3DV.%26aulast%3DParameswari%26aufirst%3DR.%26aulast%3DPandian%26aufirst%3DS.%2BK.%26aulast%3DRavi%26aufirst%3DA.%2BV.%26atitle%3DComputational%2520discovery%2520of%2520putative%2520quorum%2520sensing%2520inhibitors%2520against%2520LasR%2520and%2520RhlR%2520receptor%2520proteins%2520of%2520Pseudomonas%2520aeruginosa%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26spage%3D1067%26epage%3D1077%26doi%3D10.1007%2Fs10822-012-9599-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shepardson-Fungairino, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Combs, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagarajan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackwell, H. E.</span></span> <span> </span><span class="NLM_article-title">Structure-function analyses of the N-butanoyl L-homoserine lactone quorum-sensing signal define features critical to activity in RhlR</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2655</span>– <span class="NLM_lpage">2662</span>, <span class="refDoi"> DOI: 10.1021/acschembio.8b00577</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b00577" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFait7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=2655-2662&author=M.+E.+Boursierauthor=J.+D.+Mooreauthor=K.+M.+Heitmanauthor=S.+P.+Shepardson-Fungairinoauthor=J.+B.+Combsauthor=L.+C.+Koenigauthor=D.+Shinauthor=E.+C.+Brownauthor=R.+Nagarajanauthor=H.+E.+Blackwell&title=Structure-function+analyses+of+the+N-butanoyl+L-homoserine+lactone+quorum-sensing+signal+define+features+critical+to+activity+in+RhlR&doi=10.1021%2Facschembio.8b00577"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Function Analyses of the N-Butanoyl L-Homoserine Lactone Quorum-Sensing Signal Define Features Critical to Activity in RhlR</span></div><div class="casAuthors">Boursier, Michelle E.; Moore, Joseph D.; Heitman, Katherine M.; Shepardson-Fungairino, Sally P.; Combs, Joshua B.; Koenig, Lea C.; Shin, Daniel; Brown, Eric C.; Nagarajan, Rajesh; Blackwell, Helen E.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2655-2662</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pseudomonas aeruginosa is an opportunistic pathogen that coordinates the prodn. of many virulence phenotypes at high population d. via quorum sensing (QS).  The LuxR-type receptor RhlR plays an important role in the P. aeruginosa QS process, and there is considerable interest in the development of chem. approaches to modulate the activity of this protein.  RhlR is activated by a simple, low mol. wt. N-acyl L-homoserine lactone signal, N-butanoyl-L-homoserine lactone (BHL).  Despite the emerging prominence of RhlR in QS pathways, there has been limited exploration of the chem. features of the BHL scaffold that are crit. to its function.  In the current study, we sought to systematically delineate the structure-activity relationships (SARs) driving BHL activity for the first time.  A focused library of BHL analogs was designed, synthesized, and evaluated in cell-based reporter gene assays for RhlR agonism and antagonism.  These investigations allowed us to define a series of SARs for BHL-type ligands and identify structural motifs crit. for both activation and inhibition of the RhlR receptor.  Notably, we identified agonists that have ∼10-fold higher potencies in RhlR relative to BHL, are highly selective for RhlR agonism over LasR, and are active in the P. aeruginosa background.  These compds. and the SARs reported herein should pave a route toward new chem. strategies to study RhlR in P. aeruginosa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2CSUdtm4ISbVg90H21EOLACvtfcHk0lj73y6lcXdGiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFait7bF&md5=bb38808c632048f6967c4e5f8f695842</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b00577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b00577%26sid%3Dliteratum%253Aachs%26aulast%3DBoursier%26aufirst%3DM.%2BE.%26aulast%3DMoore%26aufirst%3DJ.%2BD.%26aulast%3DHeitman%26aufirst%3DK.%2BM.%26aulast%3DShepardson-Fungairino%26aufirst%3DS.%2BP.%26aulast%3DCombs%26aufirst%3DJ.%2BB.%26aulast%3DKoenig%26aufirst%3DL.%2BC.%26aulast%3DShin%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DE.%2BC.%26aulast%3DNagarajan%26aufirst%3DR.%26aulast%3DBlackwell%26aufirst%3DH.%2BE.%26atitle%3DStructure-function%2520analyses%2520of%2520the%2520N-butanoyl%2520L-homoserine%2520lactone%2520quorum-sensing%2520signal%2520define%2520features%2520critical%2520to%2520activity%2520in%2520RhlR%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D2655%26epage%3D2662%26doi%3D10.1021%2Facschembio.8b00577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Welsh, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eibergen, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackwell, H. E.</span></span> <span> </span><span class="NLM_article-title">Small molecule disruption of quorum sensing cross-regulation in Pseudomonas aeruginosa causes major and unexpected alterations to virulence phenotypes</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">1510</span>– <span class="NLM_lpage">1519</span>, <span class="refDoi"> DOI: 10.1021/ja5110798</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja5110798" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC2MXns12qtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=1510-1519&author=M.+A.+Welshauthor=N.+R.+Eibergenauthor=J.+D.+Mooreauthor=H.+E.+Blackwell&title=Small+molecule+disruption+of+quorum+sensing+cross-regulation+in+Pseudomonas+aeruginosa+causes+major+and+unexpected+alterations+to+virulence+phenotypes&doi=10.1021%2Fja5110798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Disruption of Quorum Sensing Cross-Regulation in Pseudomonas aeruginosa Causes Major and Unexpected Alterations to Virulence Phenotypes</span></div><div class="casAuthors">Welsh, Michael A.; Eibergen, Nora R.; Moore, Joseph D.; Blackwell, Helen E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1510-1519</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The opportunistic pathogen Pseudomonas aeruginosa uses three interwoven quorum-sensing (QS) circuits-Las, Rhl, and Pqs-to regulate the global expression of myriad virulence-assocd. genes.  Interception of these signaling networks with small mols. represents an emerging strategy for the development of anti-infective agents against this bacterium.  In the current study, we applied a chem. approach to investigate how the Las-Rhl-Pqs QS hierarchy coordinates key virulence phenotypes in wild-type P. aeruginosa.  We screened a focused library of synthetic, non-native N-acyl L-homoserine lactones and identified compds. that can drastically alter prodn. of two important virulence factors: pyocyanin and rhamnolipid.  We demonstrate that these mols. act by targeting RhlR in P. aeruginosa, a QS receptor that has seen far less scrutiny to date relative to other circuitry.  Unexpectedly, modulation of RhlR activity by a single compd. induces inverse regulation of pyocyanin and rhamnolipid, a result that was not predicted using genetic approaches to interrogate QS in P. aeruginosa.  Further, we show that certain RhlR agonists strongly repress Pqs signaling, revealing disruption of Rhl-Pqs cross-regulation as a novel mechanism for QS inhibition.  These compds. significantly expand the known repertoire of chem. probes available to study RhlR in P. aeruginosa.  Moreover, our results suggest that designing chem. agents to disrupt Rhl-Pqs crosstalk could be an effective antivirulence strategy to fight this common pathogen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmeYI_3kIpQbVg90H21EOLACvtfcHk0lj73y6lcXdGiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXns12qtw%253D%253D&md5=8cf2dfd6d0c33bf6aa1142d00683a86a</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fja5110798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja5110798%26sid%3Dliteratum%253Aachs%26aulast%3DWelsh%26aufirst%3DM.%2BA.%26aulast%3DEibergen%26aufirst%3DN.%2BR.%26aulast%3DMoore%26aufirst%3DJ.%2BD.%26aulast%3DBlackwell%26aufirst%3DH.%2BE.%26atitle%3DSmall%2520molecule%2520disruption%2520of%2520quorum%2520sensing%2520cross-regulation%2520in%2520Pseudomonas%2520aeruginosa%2520causes%2520major%2520and%2520unexpected%2520alterations%2520to%2520virulence%2520phenotypes%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D1510%26epage%3D1519%26doi%3D10.1021%2Fja5110798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCready, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paczkowski, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassler, B. L.</span></span> <span> </span><span class="NLM_article-title">An autoinducer-independent RhlR quorum-sensing receptor enables analysis of RhlR regulation</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>, <span class="NLM_elocation-id">e1007820</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1007820</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1371%2Fjournal.ppat.1007820" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=31194839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFGks7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&author=A.+R.+McCreadyauthor=J.+E.+Paczkowskiauthor=J.-P.+Congauthor=B.+L.+Bassler&title=An+autoinducer-independent+RhlR+quorum-sensing+receptor+enables+analysis+of+RhlR+regulation&doi=10.1371%2Fjournal.ppat.1007820"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">An autoinducer-independent RhlR quorum-sensing receptor enables analysis of RhlR regulation</span></div><div class="casAuthors">McCready, Amelia R.; Paczkowski, Jon E.; Cong, Jian-Ping; Bassler, Bonnie L.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e1007820</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Quorum sensing is a chem. communication process that bacteria use to coordinate group behaviors.  Pseudomonas aeruginosa, an opportunistic pathogen, employs multiple quorum-sensing systems to control behaviors including virulence factor prodn. and biofilm formation.  One P. aeruginosa quorum-sensing receptor, called RhlR, binds the cognate autoinducer N-butryl-homoserine lactone (C4HSL), and the RhlR:C4HSL complex activates transcription of target quorum-sensing genes.  Here, we use a genetic screen to identify RhlR mutants that function independently of the autoinducer.  The RhlR Y64F W68F V133F triple mutant, which we call RhlR*, exhibits ligand-independent activity in vitro and in vivo.  RhlR* can drive wildtype biofilm formation and infection in a nematode animal model.  The ability of RhlR* to properly regulate quorum-sensing-controlled genes in vivo depends on the quorum-sensing regulator RsaL keeping RhlR* activity in check.  RhlR is known to function together with PqsE to control prodn. of the virulence factor called pyocyanin.  Likewise, RhlR* requires PqsE for pyocyanin prodn. in planktonic cultures, however, PqsE is dispensable for RhlR*-driven pyocyanin prodn. on surfaces.  Finally, wildtype RhlR protein is not sufficiently stabilized by C4HSL to allow purifn.  However, wildtype RhlR can be stabilized by the synthetic ligand mBTL (meta-bromo-thiolactone) and RhlR* is stable without a ligand.  These features enabled purifn. of the RhlR:mBTL complex and of RhlR* for in vitro examn. of their biochem. activities.  To our knowledge, this work reports the first RhlR protein purifn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpy48nibUhtx7Vg90H21EOLACvtfcHk0lhyUI3qtxRMgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFGks7vO&md5=92bf15dba7796bec8c95a05d6b60ef25</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1007820&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1007820%26sid%3Dliteratum%253Aachs%26aulast%3DMcCready%26aufirst%3DA.%2BR.%26aulast%3DPaczkowski%26aufirst%3DJ.%2BE.%26aulast%3DCong%26aufirst%3DJ.-P.%26aulast%3DBassler%26aufirst%3DB.%2BL.%26atitle%3DAn%2520autoinducer-independent%2520RhlR%2520quorum-sensing%2520receptor%2520enables%2520analysis%2520of%2520RhlR%2520regulation%26jtitle%3DPLoS%2520Pathog.%26date%3D2019%26volume%3D15%26doi%3D10.1371%2Fjournal.ppat.1007820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.-D.</span></span> <span> </span><span class="NLM_article-title">6-Gingerol reduces Pseudomonas aeruginosa biofilm formation and virulence via quorum sensing inhibition</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">8656</span>, <span class="refDoi"> DOI: 10.1038/srep08656</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1038%2Fsrep08656" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=25728862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVCrtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=8656&author=H.-S.+Kimauthor=S.-H.+Leeauthor=Y.+Byunauthor=H.-D.+Park&title=6-Gingerol+reduces+Pseudomonas+aeruginosa+biofilm+formation+and+virulence+via+quorum+sensing+inhibition&doi=10.1038%2Fsrep08656"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">6-Gingerol reduces Pseudomonas aeruginosa biofilm formation and virulence via quorum sensing inhibition</span></div><div class="casAuthors">Kim, Han-Shin; Lee, Sang-Hoon; Byun, Youngjoo; Park, Hee-Deung</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8656</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Pseudomonas aeruginosa is a well-known pathogenic bacterium that forms biofilms and produces virulence factors via quorum sensing (QS).  Interfering with normal QS interactions between signal mols. and their cognate receptors is a developing strategy for attenuating its virulence.  Here we tested the hypothesis that 6-gingerol, a pungent oil of fresh ginger, reduces biofilm formation and virulence by antagonistically binding to P. aeruginosa QS receptors.  In silico studies demonstrated mol. binding occurs between 6-gingerol and the QS receptor LasR through hydrogen bonding and hydrophobic interactions.  Exptl. 6-gingerol reduced biofilm formation, several virulence factors (e.g., exoprotease, rhamnolipid, and pyocyanin), and mice mortality.  Further transcriptome analyses demonstrated that 6-gingerol successfully repressed QS-induced genes, specifically those related to the prodn. of virulence factors.  These results strongly support our hypothesis and offer insight into the mol. mechanism that caused QS gene repression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSqLFnQR49jLVg90H21EOLACvtfcHk0lhyUI3qtxRMgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVCrtbY%253D&md5=35b0f3cac58fefde5cf68e03f5397efa</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fsrep08656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep08656%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DH.-S.%26aulast%3DLee%26aufirst%3DS.-H.%26aulast%3DByun%26aufirst%3DY.%26aulast%3DPark%26aufirst%3DH.-D.%26atitle%3D6-Gingerol%2520reduces%2520Pseudomonas%2520aeruginosa%2520biofilm%2520formation%2520and%2520virulence%2520via%2520quorum%2520sensing%2520inhibition%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D8656%26doi%3D10.1038%2Fsrep08656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ham, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cha, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bang, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Son, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byun, Y.</span></span> <span> </span><span class="NLM_article-title">Structure-activity relationships of 6-and 8-gingerol analogs as anti-biofilm agents</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">9821</span>– <span class="NLM_lpage">9837</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01426</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01426" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVSnur%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9821-9837&author=H.+Choiauthor=S.-Y.+Hamauthor=E.+Chaauthor=Y.+Shinauthor=H.-S.+Kimauthor=J.+K.+Bangauthor=S.-H.+Sonauthor=H.-D.+Parkauthor=Y.+Byun&title=Structure-activity+relationships+of+6-and+8-gingerol+analogs+as+anti-biofilm+agents&doi=10.1021%2Facs.jmedchem.7b01426"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationships of 6- and 8-Gingerol Analogs as Anti-Biofilm Agents</span></div><div class="casAuthors">Choi, Hyunsuk; Ham, So-Young; Cha, Eunji; Shin, Yujin; Kim, Han-Shin; Bang, Jeong Kyu; Son, Sang-Hyun; Park, Hee-Deung; Byun, Youngjoo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9821-9837</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pseudomonas aeruginosa is a causative agent of chronic infections in immunocompromised patients.  Disruption of quorum sensing circuits is an attractive strategy for treating diseases assocd. with P. aeruginosa infection.  In this study, the authors designed and synthesized a series of gingerol analogs targeting LasR, a master regulator of quorum sensing networks in P. aeruginosa.  Structure-activity relationship studies showed that a hydrogen-bonding interaction in the head section, stereochem. and rotational rigidity in the middle section, and optimal alkyl chain length in the tail section are important factors for the enhancement of LasR-binding affinity and for the inhibition of biofilm formation.  The most potent compd. I, an analog of (R)-8-gingerol with restricted rotation, showed stronger LasR-binding affinity and inhibition of biofilm formation than the known LasR antagonist (S)-6-gingerol.  This new LasR antagonist can be used as an early lead compd. for the development of antibiofilm agents to treat P. aeruginosa infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFyS9hVAiLXLVg90H21EOLACvtfcHk0lhyUI3qtxRMgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVSnur%252FJ&md5=73007847e98d059046ae29327c27d817</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01426&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01426%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DH.%26aulast%3DHam%26aufirst%3DS.-Y.%26aulast%3DCha%26aufirst%3DE.%26aulast%3DShin%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DH.-S.%26aulast%3DBang%26aufirst%3DJ.%2BK.%26aulast%3DSon%26aufirst%3DS.-H.%26aulast%3DPark%26aufirst%3DH.-D.%26aulast%3DByun%26aufirst%3DY.%26atitle%3DStructure-activity%2520relationships%2520of%25206-and%25208-gingerol%2520analogs%2520as%2520anti-biofilm%2520agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D9821%26epage%3D9837%26doi%3D10.1021%2Facs.jmedchem.7b01426" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tassano, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dondo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galatini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riva, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banfi, L.</span></span> <span> </span><span class="NLM_article-title">Conjugation of hydroxytyrosol with other natural phenolic fragments: From waste to antioxidants and antitumour compounds</span>. <i>Eur. J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>2015</i></span>,  <span class="NLM_fpage">6710</span>– <span class="NLM_lpage">6726</span>, <span class="refDoi"> DOI: 10.1002/ejoc.201500931</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1002%2Fejoc.201500931" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFWrsrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2015&publication_year=2015&pages=6710-6726&author=E.+Tassanoauthor=A.+Alamaauthor=A.+Bassoauthor=G.+Dondoauthor=A.+Galatiniauthor=R.+Rivaauthor=L.+Banfi&title=Conjugation+of+hydroxytyrosol+with+other+natural+phenolic+fragments%3A+From+waste+to+antioxidants+and+antitumour+compounds&doi=10.1002%2Fejoc.201500931"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Conjugation of Hydroxytyrosol with Other Natural Phenolic Fragments: From Waste to Antioxidants and Antitumour Compounds</span></div><div class="casAuthors">Tassano, Erika; Alama, Angela; Basso, Andrea; Dondo, Giancarlo; Galatini, Andrea; Riva, Renata; Banfi, Luca</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">2015</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">6710-6726</span>CODEN:
                <span class="NLM_cas:coden">EJOCFK</span>;
        ISSN:<span class="NLM_cas:issn">1099-0690</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Hydroxytyrosol (I), a natural polyphenol that can be extd. from olive mill waste waters, was converted into a series of more complex and lipophilic analogs by conjugation with other naturally derived (poly)phenolic fragments.  Ether and triazole linkers were employed.  A small library of compds. was prepd., stressing step economy and operational simplicity.  Some of these substances were proven to have activity equal or superior to that of the parent hydroxytyrosol in radical scavenging assays as well as in cytotoxicity tests against tumor cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoG5GoGXvsi5LVg90H21EOLACvtfcHk0lh9W9IBZ8BhVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFWrsrrN&md5=bc83086e9f313191bb4e5650b242c3ea</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fejoc.201500931&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.201500931%26sid%3Dliteratum%253Aachs%26aulast%3DTassano%26aufirst%3DE.%26aulast%3DAlama%26aufirst%3DA.%26aulast%3DBasso%26aufirst%3DA.%26aulast%3DDondo%26aufirst%3DG.%26aulast%3DGalatini%26aufirst%3DA.%26aulast%3DRiva%26aufirst%3DR.%26aulast%3DBanfi%26aufirst%3DL.%26atitle%3DConjugation%2520of%2520hydroxytyrosol%2520with%2520other%2520natural%2520phenolic%2520fragments%253A%2520From%2520waste%2520to%2520antioxidants%2520and%2520antitumour%2520compounds%26jtitle%3DEur.%2520J.%2520Org.%2520Chem.%26date%3D2015%26volume%3D2015%26spage%3D6710%26epage%3D6726%26doi%3D10.1002%2Fejoc.201500931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsumura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashiguchi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikariya, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noyori, R.</span></span> <span> </span><span class="NLM_article-title">Asymmetric transfer hydrogenation of α, β-acetylenic ketones</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">8738</span>– <span class="NLM_lpage">8739</span>, <span class="refDoi"> DOI: 10.1021/ja971570a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja971570a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADyaK2sXmtVens7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=1997&pages=8738-8739&author=K.+Matsumuraauthor=S.+Hashiguchiauthor=T.+Ikariyaauthor=R.+Noyori&title=Asymmetric+transfer+hydrogenation+of+%CE%B1%2C+%CE%B2-acetylenic+ketones&doi=10.1021%2Fja971570a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Asymmetric Transfer Hydrogenation of α,β-Acetylenic Ketones</span></div><div class="casAuthors">Matsumura, Kazuhiko; Hashiguchi, Shohei; Ikariya, Takao; Noyori, Ryoji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">8738-8739</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A range of achiral α,β-acetylenic ketones is converted to propargylic alcs. with high enantiomeric excess and in high yield by transfer hydrogenation using 2-propanol as the hydrogen donor.  The asym. reaction using arene-Ru(II) catalysts modified with (R,R)- or (S,S)-N-p-toluenesulfonyl-1,2-diphenylethylenediamine proceeds with a substrate/catalyst molar ratio of 100-1000 under neutral conditions and at room temp., allowing chemo- and enantioselective satn. of the carbonyl function while leaving the acetylenic linkage intact.  β-Amino alcs. such as (1S,2S)-2-(methylamino)-1,2-diphenylethanol are also usable as chiral auxiliaries.  The enantiomeric purity of the propargylic alcs. is consistently high regardless of the bulkiness of substituents.  Redn. of chiral alkynyl Cbz-aminoalkyl ketones gives the enantiomerically pure alcs. with contiguous stereogenic carbons, where the carbonyl faces are differentiated by the chirality of the Ru catalyst.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8efeo1MIaKLVg90H21EOLACvtfcHk0lh9W9IBZ8BhVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmtVens7w%253D&md5=8b9b4a49190d9681c8950826f24a2d55</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fja971570a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja971570a%26sid%3Dliteratum%253Aachs%26aulast%3DMatsumura%26aufirst%3DK.%26aulast%3DHashiguchi%26aufirst%3DS.%26aulast%3DIkariya%26aufirst%3DT.%26aulast%3DNoyori%26aufirst%3DR.%26atitle%3DAsymmetric%2520transfer%2520hydrogenation%2520of%2520%25CE%25B1%252C%2520%25CE%25B2-acetylenic%2520ketones%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1997%26volume%3D119%26spage%3D8738%26epage%3D8739%26doi%3D10.1021%2Fja971570a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazmaier, U.</span></span> <span> </span><span class="NLM_article-title">Stereoselective modification of N-(alpha-hydroxyacyl)-glycinesters via palladium-catalyzed allylic alkylation</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">4595</span>– <span class="NLM_lpage">4599</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.9b01497</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.9b01497" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVOntrrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2019&pages=4595-4599&author=A.+Hornauthor=U.+Kazmaier&title=Stereoselective+modification+of+N-%28alpha-hydroxyacyl%29-glycinesters+via+palladium-catalyzed+allylic+alkylation&doi=10.1021%2Facs.orglett.9b01497"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Stereoselective Modification of N-(α-Hydroxyacyl)-glycinesters via Palladium-Catalyzed Allylic Alkylation</span></div><div class="casAuthors">Horn, Alexander; Kazmaier, Uli</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4595-4599</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">N-(α-Hydroxyacyl)-glycinesters can be used as excellent nucleophiles in Pd-catalyzed allylic alkylation.  The method allows for the stereoselective introduction of a wide range of side chains, including highly functionalized ones.  Both diastereomers can be accessed through variation of the reaction conditions.  Furthermore, the use of stannylated carbonates introduces vinylstannane motifs, which are eligible for subsequent C-C coupling reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVhlVAbob4bLVg90H21EOLACvtfcHk0lh9W9IBZ8BhVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVOntrrM&md5=17c3a77a32c603a6925bfd4a74382a7b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.9b01497&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.9b01497%26sid%3Dliteratum%253Aachs%26aulast%3DHorn%26aufirst%3DA.%26aulast%3DKazmaier%26aufirst%3DU.%26atitle%3DStereoselective%2520modification%2520of%2520N-%2528alpha-hydroxyacyl%2529-glycinesters%2520via%2520palladium-catalyzed%2520allylic%2520alkylation%26jtitle%3DOrg.%2520Lett.%26date%3D2019%26volume%3D21%26spage%3D4595%26epage%3D4599%26doi%3D10.1021%2Facs.orglett.9b01497" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ochsner, U. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiser, J.</span></span> <span> </span><span class="NLM_article-title">Autoinducer-mediated regulation of rhamnolipid biosurfactant synthesis in Pseudomonas aeruginosa</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">6424</span>– <span class="NLM_lpage">6428</span>, <span class="refDoi"> DOI: 10.1073/pnas.92.14.6424</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1073%2Fpnas.92.14.6424" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=7604006" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADyaK2MXmslKqur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=1995&pages=6424-6428&author=U.+A.+Ochsnerauthor=J.+Reiser&title=Autoinducer-mediated+regulation+of+rhamnolipid+biosurfactant+synthesis+in+Pseudomonas+aeruginosa&doi=10.1073%2Fpnas.92.14.6424"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Autoinducer-mediated regulation of rhamnolipid biosurfactant synthesis in Pseudomonas aeruginosa</span></div><div class="casAuthors">Ochsner, Urs A.; Reiser, Jakob</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6424-8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The opportunistic human pathogen Pseudomonas aeruginosa produces a variety of virulence factors, including exotoxin A, elastase, alk. protease, alginate, phospholipases, and extracellular rhamnolipids.  The previously characterized rhlABR gene cluster encodes a regulatory protein (RhlR) and a rhamnosyltransferase (RhlAB), both of which are required for rhamnolipid synthesis.  Another gene, rhlI, was now identified downstream of the rhlABR gene cluster.  The putative RhlI protein shares significant sequence similarity with bacterial autoinducer synthetases of the LuxI type.  A P. aeruginosa rhlI mutant strain carrying a disrupted rhlI gene was unable to produce rhamnolipids and lacked rhamnosyltransferase activity.  Rhamnolipid synthesis was restored by introducing a wild-type rhlI gene into such strains or, alternatively, by adding either the cell-free spent supernatant from a P. aeruginosa wild-type strain or synthetic N-acylhomoserine lactones.  Half-maximal induction of rhamnolipid synthesis in the rhlI mutant strain required 0.5 μM N-butyrylhomoserine lactone or 10 μM N-(3-oxohexanoyl)homoserine lactone.  The P. aeruginosa rhlA promoter was active in the heterologous host Pseudomonas putida when both the rhlR and rhlI genes were present or when the rhlR gene alone was supplied together with synthetic N-acylhomoserine lactones.  The RhlR-RhlI regulatory system was essential for the prodn. of elastase as well, and cross-communication between the RhlR-RhlI rhamnolipid regulatory system and the LasR-LasI elastase regulatory system was demonstrated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFoNIMwsQQE7Vg90H21EOLACvtfcHk0ljO53S0FIIpbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmslKqur0%253D&md5=45fcdfa709509c6c3ce07f7e0b76ab57</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1073%2Fpnas.92.14.6424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.92.14.6424%26sid%3Dliteratum%253Aachs%26aulast%3DOchsner%26aufirst%3DU.%2BA.%26aulast%3DReiser%26aufirst%3DJ.%26atitle%3DAutoinducer-mediated%2520regulation%2520of%2520rhamnolipid%2520biosurfactant%2520synthesis%2520in%2520Pseudomonas%2520aeruginosa%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1995%26volume%3D92%26spage%3D6424%26epage%3D6428%26doi%3D10.1073%2Fpnas.92.14.6424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nickzad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Déziel, E.</span></span> <span> </span><span class="NLM_article-title">The involvement of rhamnolipids in microbial cell adhesion and biofilm development-an approach for control?</span>. <i>Lett. Appl. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">453</span>, <span class="refDoi"> DOI: 10.1111/lam.12211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1111%2Flam.12211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=24372465" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtl2itLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2014&pages=447-453&author=A.+Nickzadauthor=E.+D%C3%A9ziel&title=The+involvement+of+rhamnolipids+in+microbial+cell+adhesion+and+biofilm+development-an+approach+for+control%3F&doi=10.1111%2Flam.12211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">The involvement of rhamnolipids in microbial cell adhesion and biofilm development - an approach for control?</span></div><div class="casAuthors">Nickzad, A.; Deziel, E.</div><div class="citationInfo"><span class="NLM_cas:title">Letters in Applied Microbiology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">447-453</span>CODEN:
                <span class="NLM_cas:coden">LAMIE7</span>;
        ISSN:<span class="NLM_cas:issn">0266-8254</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Biofilms are omnipresent in clin. and industrial settings and most of the times cause detrimental side effects.  Finding efficient strategies to control surface-growing communities of micro-organisms remains a significant challenge.  Rhamnolipids are extracellular secondary metabolites with surface-active properties mainly produced by Pseudomonas aeruginosa.  There is growing evidence for the implication of this biosurfactant in different stages of biofilm development of this bacterium.  Furthermore, rhamnolipids display a significant potential as anti-adhesive and disrupting agents against established biofilms formed by several bacterial and fungal species.  Their low toxicity, biodegradability, efficiency and specificity, compared to synthetic surfactants typically used in biofilm control, might compensate for the economic hurdle still linked to their superior prodn. costs and make them promising antifouling agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpt6j2_3bBSbVg90H21EOLACvtfcHk0ljO53S0FIIpbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtl2itLs%253D&md5=7575bc61e392564b72503aa1d9e5e6c3</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1111%2Flam.12211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Flam.12211%26sid%3Dliteratum%253Aachs%26aulast%3DNickzad%26aufirst%3DA.%26aulast%3DD%25C3%25A9ziel%26aufirst%3DE.%26atitle%3DThe%2520involvement%2520of%2520rhamnolipids%2520in%2520microbial%2520cell%2520adhesion%2520and%2520biofilm%2520development-an%2520approach%2520for%2520control%253F%26jtitle%3DLett.%2520Appl.%2520Microbiol.%26date%3D2014%26volume%3D58%26spage%3D447%26epage%3D453%26doi%3D10.1111%2Flam.12211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Solano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echeverz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasa, I.</span></span> <span> </span><span class="NLM_article-title">Biofilm dispersion and quorum sensing</span>. <i>Curr. Opin. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">96</span>– <span class="NLM_lpage">104</span>, <span class="refDoi"> DOI: 10.1016/j.mib.2014.02.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1016%2Fj.mib.2014.02.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=24657330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmslais7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2014&pages=96-104&author=C.+Solanoauthor=M.+Echeverzauthor=I.+Lasa&title=Biofilm+dispersion+and+quorum+sensing&doi=10.1016%2Fj.mib.2014.02.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Biofilm dispersion and quorum sensing</span></div><div class="casAuthors">Solano, Cristina; Echeverz, Maite; Lasa, Inigo</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Microbiology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">96-104</span>CODEN:
                <span class="NLM_cas:coden">COMIF7</span>;
        ISSN:<span class="NLM_cas:issn">1369-5274</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Biofilm development and quorum sensing (QS) are closely interconnected processes.  Biofilm formation is a cooperative group behavior that involves bacterial populations living embedded in a self-produced extracellular matrix.  QS is a cell-cell communication mechanism that synchronizes gene expression in response to population cell d.  Intuitively, it would appear that QS might coordinate the switch to a biofilm lifestyle when the population d. reaches a threshold level.  However, compelling evidence obtained in different bacterial species coincides in that activation of QS occurs in the formed biofilm and activates the maturation and disassembly of the biofilm in a coordinate manner.  The aim of this review is to illustrate, using four bacterial pathogens as examples, the emergent concept that QS activates the biofilm dispersion process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqt_zGsdhS-bVg90H21EOLACvtfcHk0ljO53S0FIIpbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmslais7Y%253D&md5=46db3d97efd74b26c75679a57fdfd1b4</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.mib.2014.02.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mib.2014.02.008%26sid%3Dliteratum%253Aachs%26aulast%3DSolano%26aufirst%3DC.%26aulast%3DEcheverz%26aufirst%3DM.%26aulast%3DLasa%26aufirst%3DI.%26atitle%3DBiofilm%2520dispersion%2520and%2520quorum%2520sensing%26jtitle%3DCurr.%2520Opin.%2520Microbiol.%26date%3D2014%26volume%3D18%26spage%3D96%26epage%3D104%26doi%3D10.1016%2Fj.mib.2014.02.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pamp, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolker-Nielsen, T.</span></span> <span> </span><span class="NLM_article-title">Multiple roles of biosurfactants in structural biofilm development by Pseudomonas aeruginosa</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>189</i></span>,  <span class="NLM_fpage">2531</span>– <span class="NLM_lpage">2539</span>, <span class="refDoi"> DOI: 10.1128/jb.01515-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1128%2FJB.01515-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=17220224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjt1GksLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=189&publication_year=2007&pages=2531-2539&author=S.+J.+Pampauthor=T.+Tolker-Nielsen&title=Multiple+roles+of+biosurfactants+in+structural+biofilm+development+by+Pseudomonas+aeruginosa&doi=10.1128%2Fjb.01515-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple roles of biosurfactants in structural biofilm development by Pseudomonas aeruginosa</span></div><div class="casAuthors">Pamp, Sunje Johanna; Tolker-Nielsen, Tim</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">189</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2531-2539</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Recent studies have indicated that biosurfactants produced by Pseudomonas aeruginosa play a role both in maintaining channels between multi-cellular structures in biofilms and in dispersal of cells from biofilms.  Through the use of flow cell technol. and enhanced confocal laser scanning microscopy, we have obtained results which suggest that the biosurfactants produced by P. aeruginosa play addnl. roles in structural biofilm development.  We present genetic evidence that during biofilm development by P. aeruginosa, biosurfactants promote microcolony formation in the initial phase and facilitate migration-dependent structural development in the later phase.  P. aeruginosa rhlA mutants, deficient in synthesis of biosurfactants, were not capable of forming microcolonies in the initial phase of biofilm formation.  Expts. involving two-color-coded mixed-strain biofilms showed that P. aeruginosa rhlA mutants were defective in migration-dependent development of mushroom-shaped multi-cellular structures in the later phase of biofilm formation.  Expts. involving three-color-coded mixed-strain P. aeruginosa biofilms demonstrated that the wild-type and rhlA and pilA mutant strains formed distinct subpopulations on top of each other dependent on their ability to migrate and produce biosurfactants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojNxJWxWbAQLVg90H21EOLACvtfcHk0ljO53S0FIIpbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjt1GksLo%253D&md5=49ff7bcfa3ad2dc7b8604b1223b55f8c</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1128%2FJB.01515-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.01515-06%26sid%3Dliteratum%253Aachs%26aulast%3DPamp%26aufirst%3DS.%2BJ.%26aulast%3DTolker-Nielsen%26aufirst%3DT.%26atitle%3DMultiple%2520roles%2520of%2520biosurfactants%2520in%2520structural%2520biofilm%2520development%2520by%2520Pseudomonas%2520aeruginosa%26jtitle%3DJ.%2520Bacteriol.%26date%3D2007%26volume%3D189%26spage%3D2531%26epage%3D2539%26doi%3D10.1128%2Fjb.01515-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skindersoe, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alhede, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phipps, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, P. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bjarnsholt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolker-Nielsen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Høiby, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Givskov, M.</span></span> <span> </span><span class="NLM_article-title">Effects of antibiotics on quorum sensing in Pseudomonas aeruginosa</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">3648</span>– <span class="NLM_lpage">3663</span>, <span class="refDoi"> DOI: 10.1128/aac.01230-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1128%2FAAC.01230-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=18644954" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1SrtbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2008&pages=3648-3663&author=M.+E.+Skindersoeauthor=M.+Alhedeauthor=R.+Phippsauthor=L.+Yangauthor=P.+O.+Jensenauthor=T.+B.+Rasmussenauthor=T.+Bjarnsholtauthor=T.+Tolker-Nielsenauthor=N.+H%C3%B8ibyauthor=M.+Givskov&title=Effects+of+antibiotics+on+quorum+sensing+in+Pseudomonas+aeruginosa&doi=10.1128%2Faac.01230-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of antibiotics on quorum sensing in Pseudomonas aeruginosa</span></div><div class="casAuthors">Skindersoe, Mette E.; Alhede, Morten; Phipps, Richard; Yang, Liang; Jensen, Peter O.; Rasmussen, Thomas B.; Bjarnsholt, Thomas; Tolker-Nielsen, Tim; Hoeiby, Niels; Givskov, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3648-3663</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">During infection, Pseudomonas aeruginosa employs bacterial communication (quorum sensing [QS]) to coordinate the expression of tissue-damaging factors.  QS-controlled gene expression plays a pivotal role in the virulence of P. aeruginosa, and QS-deficient mutants cause less severe infections in animal infection models.  Treatment of cystic fibrosis (CF) patients chronically infected with P. aeruginosa with the macrolide antibiotic azithromycin (AZM) has been demonstrated to improve the clin. outcome.  Several studies indicate that AZM may accomplish its beneficial action in CF patients by impeding QS, thereby reducing the pathogenicity of P. aeruginosa.  This led us to investigate whether QS inhibition is a common feature of antibiotics.  We present the results of a screening of 12 antibiotics for their QS-inhibitory activities using a previously described QS inhibitor selector 1 strain.  Three of the antibiotics tested, AZM, ceftazidime (CFT), and ciprofloxacin (CPR), were very active in the assay and were further examd. for their effects on QS-regulated virulence factor prodn. in P. aeruginosa.  The effects of the three antibiotics administered at subinhibitory concns. were investigated by use of DNA microarrays.  Consistent results from the virulence factor assays, reverse transcription-PCR, and the DNA microarrays support the finding that AZM, CFT, and CPR decrease the expression of a range of QS-regulated virulence factors.  The data suggest that the underlying mechanism may be mediated by changes in membrane permeability, thereby influencing the flux of N-3-oxo-dodecanoyl-L-homoserine lactone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpj0zKNJdCINrVg90H21EOLACvtfcHk0ljFsEr2J2RV9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1SrtbbJ&md5=059375c0aba1089b5150fd5bdecf4ae4</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1128%2FAAC.01230-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01230-07%26sid%3Dliteratum%253Aachs%26aulast%3DSkindersoe%26aufirst%3DM.%2BE.%26aulast%3DAlhede%26aufirst%3DM.%26aulast%3DPhipps%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DJensen%26aufirst%3DP.%2BO.%26aulast%3DRasmussen%26aufirst%3DT.%2BB.%26aulast%3DBjarnsholt%26aufirst%3DT.%26aulast%3DTolker-Nielsen%26aufirst%3DT.%26aulast%3DH%25C3%25B8iby%26aufirst%3DN.%26aulast%3DGivskov%26aufirst%3DM.%26atitle%3DEffects%2520of%2520antibiotics%2520on%2520quorum%2520sensing%2520in%2520Pseudomonas%2520aeruginosa%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2008%26volume%3D52%26spage%3D3648%26epage%3D3663%26doi%3D10.1128%2Faac.01230-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moustafa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassler, B. L.</span></span> <span> </span><span class="NLM_article-title">The RhlR quorum-sensing receptor controls Pseudomonas aeruginosa pathogenesis and biofilm development independently of its canonical homoserine lactone autoinducer</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>, <span class="NLM_elocation-id">e1006504</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1006504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1371%2Fjournal.ppat.1006504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=28715477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC1cXivVWgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&author=S.+Mukherjeeauthor=D.+Moustafaauthor=C.+D.+Smithauthor=J.+B.+Goldbergauthor=B.+L.+Bassler&title=The+RhlR+quorum-sensing+receptor+controls+Pseudomonas+aeruginosa+pathogenesis+and+biofilm+development+independently+of+its+canonical+homoserine+lactone+autoinducer&doi=10.1371%2Fjournal.ppat.1006504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The RhlR quorum-sensing receptor controls Pseudomonas aeruginosa pathogenesis and biofilm development independently of its canonical homoserine lactone autoinducer</span></div><div class="casAuthors">Mukherjee, Sampriti; Moustafa, Dina; Smith, Chari D.; Goldberg, Joanna B.; Bassler, Bonnie L.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e1006504/1-e1006504/25</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Quorum sensing (QS) is a bacterial cell-to-cell communication process that relies on the prodn., release, and response to extracellular signaling mols. called autoinducers.  QS controls virulence and biofilm formation in the human pathogen Pseudomonas aeruginosa.  P. aeruginosa possesses two canonical LuxI/R-type QS systems, LasI/R and RhlI/R, which produce and detect 3OC12-homoserine lactone and C4-homoserine lactone, resp.  Here, we use biofilm analyses, reporter assays, RNA-seq studies, and animal infection assays to show that RhlR directs both RhlI-dependent and RhlI-independent regulons.  In the absence of RhlI, RhlR controls the expression of genes required for biofilm formation as well as genes encoding virulence factors.  Consistent with these findings, ΔrhlR and ΔrhlI mutants have radically different biofilm phenotypes and the ΔrhlI mutant displays full virulence in animals whereas the ΔrhlR mutant is attenuated.  The ΔrhlI mutant cell-free culture fluids contain an activity that stimulates RhlR-dependent gene expression.  We propose a model in which RhlR responds to an alternative ligand, in addn. to its canonical C4-homoserine lactone autoinducer.  This alternate ligand promotes a RhlR-dependent transcriptional program in the absence of RhlI.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfizuX4Th-oLVg90H21EOLACvtfcHk0ljFsEr2J2RV9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXivVWgs7c%253D&md5=843c61c61fe3c63ab5d1bca3c3dcb790</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1006504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1006504%26sid%3Dliteratum%253Aachs%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DMoustafa%26aufirst%3DD.%26aulast%3DSmith%26aufirst%3DC.%2BD.%26aulast%3DGoldberg%26aufirst%3DJ.%2BB.%26aulast%3DBassler%26aufirst%3DB.%2BL.%26atitle%3DThe%2520RhlR%2520quorum-sensing%2520receptor%2520controls%2520Pseudomonas%2520aeruginosa%2520pathogenesis%2520and%2520biofilm%2520development%2520independently%2520of%2520its%2520canonical%2520homoserine%2520lactone%2520autoinducer%26jtitle%3DPLoS%2520Pathog.%26date%3D2017%26volume%3D13%26doi%3D10.1371%2Fjournal.ppat.1006504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lindsay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmer, B. M. M.</span></span> <span> </span><span class="NLM_article-title">Effect of sdiA on biosensors of N-acylhomoserine lactones</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>187</i></span>,  <span class="NLM_fpage">5054</span>– <span class="NLM_lpage">5058</span>, <span class="refDoi"> DOI: 10.1128/jb.187.14.5054-5058.2005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1128%2FJB.187.14.5054-5058.2005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=15995228" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmtVyiu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=187&publication_year=2005&pages=5054-5058&author=A.+Lindsayauthor=B.+M.+M.+Ahmer&title=Effect+of+sdiA+on+biosensors+of+N-acylhomoserine+lactones&doi=10.1128%2Fjb.187.14.5054-5058.2005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of sdiA on biosensors of N-acylhomoserine lactones</span></div><div class="casAuthors">Lindsay, Amber; Ahmer, Brian M. M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">187</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5054-5058</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Many gram-neg. bacteria synthesize N-acylhomoserine lactones (AHLs) and then use transcription factors of the LuxR family to sense and respond to AHL accumulation in the environment; this phenomenon is termed quorum sensing.  Bacteria produce a variety of AHLs, and numerous bacterial reporter strains, or biosensors, that can detect subsets of these mols. have been constructed.  Many of these are based on Escherichia coli because this species does not produce AHLs.  However, both Escherichia and Salmonella spp. contain a LuxR homolog named SdiA that can detect exogenous AHL synthesized by other microbial species.  In this study we have detd. that sdiA of E. coli and Salmonella spp. can activate an RhlR-based biosensor plasmid in response to AHLs other than what the biosensor was designed to detect.  SdiA does not activate LuxR-, LasR-, or AhyR-based biosensor plasmids, although the presence of sdiA in E. coli does interfere with the function of the AhyR-based biosensor.  Because sdiA interferes with the function of two of the four reporters, we have constructed a set of E. coli biosensor strains that lack sdiA.  The set includes control reporters that allow the luxR dependence of responses to be detd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpI99L1ykDmebVg90H21EOLACvtfcHk0ljFsEr2J2RV9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmtVyiu78%253D&md5=84f4b4e2159e1cb32f46b47b11bd9c80</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1128%2FJB.187.14.5054-5058.2005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.187.14.5054-5058.2005%26sid%3Dliteratum%253Aachs%26aulast%3DLindsay%26aufirst%3DA.%26aulast%3DAhmer%26aufirst%3DB.%2BM.%2BM.%26atitle%3DEffect%2520of%2520sdiA%2520on%2520biosensors%2520of%2520N-acylhomoserine%2520lactones%26jtitle%3DJ.%2520Bacteriol.%26date%3D2005%26volume%3D187%26spage%3D5054%26epage%3D5058%26doi%3D10.1128%2Fjb.187.14.5054-5058.2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collins, T. J.</span></span> <span> </span><span class="NLM_article-title">ImageJ for microscopy</span>. <i>Biotechniques</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">S25</span>– <span class="NLM_lpage">S30</span>, <span class="refDoi"> DOI: 10.2144/000112517</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.2144%2F000112517" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=17936939" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A280%3ADC%252BD2snht1agtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2007&pages=S25-S30&author=T.+J.+Collins&title=ImageJ+for+microscopy&doi=10.2144%2F000112517"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">ImageJ for microscopy</span></div><div class="casAuthors">Collins Tony J</div><div class="citationInfo"><span class="NLM_cas:title">BioTechniques</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">1 Suppl</span>),
    <span class="NLM_cas:pages">25-30</span>
        ISSN:<span class="NLM_cas:issn">0736-6205</span>.
    </div><div class="casAbstract">ImageJ is an essential tool for us that fulfills most of our routine image processing and analysis requirements.  The near-comprehensive range of import filters that allow easy access to image and meta-data, a broad suite processing and analysis routine, and enthusiastic support from a friendly mailing list are invaluable for all microscopy labs and facilities-not just those on a budget.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTICP2q04csYUmhnSqMnvzDfW6udTcc2eaZkVdZr9RaGbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2snht1agtg%253D%253D&md5=452a9e7d1b2118ded24c1ea3ffc13232</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.2144%2F000112517&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2144%252F000112517%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DT.%2BJ.%26atitle%3DImageJ%2520for%2520microscopy%26jtitle%3DBiotechniques%26date%3D2007%26volume%3D43%26spage%3DS25%26epage%3DS30%26doi%3D10.2144%2F000112517" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">So, Y.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeom, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, B.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-H.</span></span> <span> </span><span class="NLM_article-title">Protease IV, a quorum sensing-dependent protease of Pseudomonas aeruginosa modulates insect innate immunity</span>. <i>Mol. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">1298</span>– <span class="NLM_lpage">1314</span>, <span class="refDoi"> DOI: 10.1111/mmi.12830</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1111%2Fmmi.12830" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=25315216" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVClsbfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2014&pages=1298-1314&author=S.-J.+Parkauthor=S.-K.+Kimauthor=Y.-I.+Soauthor=H.-Y.+Parkauthor=X.-H.+Liauthor=D.+H.+Yeomauthor=M.-N.+Leeauthor=B.-L.+Leeauthor=J.-H.+Lee&title=Protease+IV%2C+a+quorum+sensing-dependent+protease+of+Pseudomonas+aeruginosa+modulates+insect+innate+immunity&doi=10.1111%2Fmmi.12830"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Protease IV, a quorum sensing-dependent protease of Pseudomonas aeruginosa modulates insect innate immunity</span></div><div class="casAuthors">Park, Su-Jin; Kim, Soo-Kyoung; So, Yong-In; Park, Ha-Young; Li, Xi-Hui; Yeom, Doo Hwan; Lee, Mi-Nan; Lee, Bok-Luel; Lee, Joon-Hee</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Microbiology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1298-1314</span>CODEN:
                <span class="NLM_cas:coden">MOMIEE</span>;
        ISSN:<span class="NLM_cas:issn">0950-382X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Summary : In Pseudomonas aeruginosa, quorum sensing (QS) plays an essential role in pathogenesis and the QS response controls many virulence factors.  Using a mealworm, Tenebrio molitor as a host model, we found that Protease IV, a QS-regulated exoprotease of P. aeruginosa functions as a key virulence effector causing the melanization and death of T. molitor larvae.  Protease IV was able to degrade zymogens of spaetzle processing enzyme (SPE) and SPE-activating enzyme (SAE) without the activation of the antimicrobial peptide (AMP) prodn.  Since SPE and SAE function to activate spaetzle, a ligand of Toll receptor in the innate immune system of T. molitor, we suggest that Protease IV may interfere with the activation of the Toll signaling.  Independently of the Toll pathway, the melanization response, another innate immunity was still generated, since Protease IV directly converted Tenebrio prophenoloxidase into active phenoloxidase.  Protease IV also worked as an important factor in the virulence to brine shrimp and nematode.  These results suggest that Protease IV provides P. aeruginosa with a sophisticated way to escape the immune attack of host by interfering with the prodn. of AMPs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsxhx_6u3AfbVg90H21EOLACvtfcHk0ligtejS4xSgaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVClsbfJ&md5=9b07fc021e13f59071aa711eecd4a27a</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1111%2Fmmi.12830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fmmi.12830%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DS.-J.%26aulast%3DKim%26aufirst%3DS.-K.%26aulast%3DSo%26aufirst%3DY.-I.%26aulast%3DPark%26aufirst%3DH.-Y.%26aulast%3DLi%26aufirst%3DX.-H.%26aulast%3DYeom%26aufirst%3DD.%2BH.%26aulast%3DLee%26aufirst%3DM.-N.%26aulast%3DLee%26aufirst%3DB.-L.%26aulast%3DLee%26aufirst%3DJ.-H.%26atitle%3DProtease%2520IV%252C%2520a%2520quorum%2520sensing-dependent%2520protease%2520of%2520Pseudomonas%2520aeruginosa%2520modulates%2520insect%2520innate%2520immunity%26jtitle%3DMol.%2520Microbiol.%26date%3D2014%26volume%3D94%26spage%3D1298%26epage%3D1314%26doi%3D10.1111%2Fmmi.12830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, N. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacMillan, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sperandio, V.</span></span> <span> </span><span class="NLM_article-title">Structural and mechanistic roles of novel chemical ligands on the SdiA quorum-sensing transcription regulator</span>. <i>mBio</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e02429-14</span> <span class="refDoi"> DOI: 10.1128/mbio.02429-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=10.1128%2FmBio.02429-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;key=25827420" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&author=Y.+Nguyenauthor=N.+X.+Nguyenauthor=J.+L.+Rogersauthor=J.+Liaoauthor=J.+B.+MacMillanauthor=Y.+Jiangauthor=V.+Sperandio&title=Structural+and+mechanistic+roles+of+novel+chemical+ligands+on+the+SdiA+quorum-sensing+transcription+regulator&doi=10.1128%2Fmbio.02429-14"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1128%2FmBio.02429-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FmBio.02429-14%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DY.%26aulast%3DNguyen%26aufirst%3DN.%2BX.%26aulast%3DRogers%26aufirst%3DJ.%2BL.%26aulast%3DLiao%26aufirst%3DJ.%26aulast%3DMacMillan%26aufirst%3DJ.%2BB.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DSperandio%26aufirst%3DV.%26atitle%3DStructural%2520and%2520mechanistic%2520roles%2520of%2520novel%2520chemical%2520ligands%2520on%2520the%2520SdiA%2520quorum-sensing%2520transcription%2520regulator%26jtitle%3DmBio%26date%3D2015%26volume%3D6%26doi%3D10.1128%2Fmbio.02429-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y15" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y15','PDB','4Y15'); return false;">PDB: 4Y15</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i100"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00630">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_65078"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00630?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00630</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Analytical data (<sup>1</sup>H NMR, <sup>13</sup>C NMR and HPLC spectra) of the key intermediates and final compounds and biological data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_001.pdf">PDF</a>)</p></li><li><p class="inline">PDB coordinates of homology RhlR model in complex with compound <b>30</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_002.pdb">PDB</a>)</p></li><li><p class="inline">PDB coordinates of homology RhlR model in complex with compound <b>31</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_003.pdb">PDB</a>)</p></li><li><p class="inline">PDB coordinates of homology RhlR model in complex with compound <b>32</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_004.pdb">PDB</a>)</p></li><li><p class="inline">PDB coordinates of homology RhlR model in complex with compound <b>1d</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_005.pdb">PDB</a>)</p></li><li><p class="inline">Molecular formula strings of <b>11a–11r</b>, <b>12a–12r</b>, <b>13a</b> and <b>13b</b>, <b>14a</b> and <b>14b</b>, <b>15a</b> and <b>15b</b>, <b>16a</b>, <b>21</b>, <b>23a–23c</b>, <b>25</b>, <b>27</b>, <b>30–32</b>, <b>33a–33c</b>, <b>34a–34c</b>, and <b>35a–35c</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_006.xlsx">XLSX</a>)</p></li><li><p class="inline">Molecular formula strings and biological data for final compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_007.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_001.pdf">jm0c00630_si_001.pdf (5.72 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_002.pdb">jm0c00630_si_002.pdb (302.13 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_003.pdb">jm0c00630_si_003.pdb (302.37 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_004.pdb">jm0c00630_si_004.pdb (302.61 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_005.pdb">jm0c00630_si_005.pdb (302.37 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_006.xlsx">jm0c00630_si_006.xlsx (16.86 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00630/suppl_file/jm0c00630_si_007.csv">jm0c00630_si_007.csv (3.72 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00630&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-15%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00630%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00630" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799c04d7c073da0","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
