
<html lang="en"     class="pb-page"  data-request-id="513fbaf4-6ef3-4607-b66a-a34aac81f6df"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.jmedchem.9b00096;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2019.62.issue-9"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Pyrrolo[3,2-d]pyrimidin-4-one Derivatives as a New Class of Potent and Cell-Active Inhibitors of P300/CBP-Associated Factor Bromodomain" /></meta><meta name="dc.Creator" content="Luyi  Huang" /></meta><meta name="dc.Creator" content="Hui  Li" /></meta><meta name="dc.Creator" content="Linli  Li" /></meta><meta name="dc.Creator" content="Lu  Niu" /></meta><meta name="dc.Creator" content="Raina  Seupel" /></meta><meta name="dc.Creator" content="Chengyong  Wu" /></meta><meta name="dc.Creator" content="Wei  Cheng" /></meta><meta name="dc.Creator" content="Chong  Chen" /></meta><meta name="dc.Creator" content="Bisen  Ding" /></meta><meta name="dc.Creator" content="Paul E.  Brennan" /></meta><meta name="dc.Creator" content="Shengyong  Yang" /></meta><meta name="dc.Description" content="Herein, we report the discovery of a series of new P300/CBP-associated factor (PCAF) bromodomain (BRD) inhibitors, which were obtained through a hit discovery process and subsequent structure-based..." /></meta><meta name="Description" content="Herein, we report the discovery of a series of new P300/CBP-associated factor (PCAF) bromodomain (BRD) inhibitors, which were obtained through a hit discovery process and subsequent structure-based..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 18, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00096" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00096" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00096" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00096" /></link>
        
    
    

<title>Discovery of Pyrrolo[3,2-d]pyrimidin-4-one Derivatives as a New Class of Potent and Cell-Active Inhibitors of P300/CBP-Associated Factor Bromodomain | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00096" /></meta><meta property="og:title" content="Discovery of Pyrrolo[3,2-d]pyrimidin-4-one Derivatives as a New Class of Potent and Cell-Active Inhibitors of P300/CBP-Associated Factor Bromodomain" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0018.jpeg" /></meta><meta property="og:description" content="Herein, we report the discovery of a series of new P300/CBP-associated factor (PCAF) bromodomain (BRD) inhibitors, which were obtained through a hit discovery process and subsequent structure-based optimization and structure–activity relationship analyses toward a retrieved hit compound (12). Among these inhibitors, (R,R)-36n is the most potent one with an IC50 of 7 nM in homogeneous time-resolved fluorescence assay and a KD of 78 nM in isothermal titration calorimetry assay. This compound also exhibited activity against GCN5 and FALZ, but weak or no activity against other 29 BRD proteins and 422 kinases, indicating considerable selectivity. X-ray cocrystal structure analysis revealed the molecular interaction mode and the precise stereochemistry required for bioactivity. Cellular activity, preliminary RNA-seq analysis, and pharmacokinetic properties were also examined for this compound. Collectively, this study provides a versatile tool molecule to explore molecular mechanisms of PCAF BRD regulation and also offers a new lead compound for drug discovery targeting PCAF." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00096"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00096">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00096&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00096&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00096&amp;href=/doi/10.1021/acs.jmedchem.9b00096" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 4526-4542</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00056" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00128" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Pyrrolo[3,2-<i>d</i>]pyrimidin-4-one Derivatives as a New Class of Potent and Cell-Active Inhibitors of P300/CBP-Associated Factor Bromodomain</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Luyi Huang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Luyi Huang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu 610041, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Luyi++Huang">Luyi Huang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hui Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hui Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu 610041, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hui++Li">Hui Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Linli Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Linli Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Drug Targeting and Drug Delivery System of Ministry of Education, West China School of Pharmacy, Sichuan University, Sichuan 610041, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Linli++Li">Linli Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lu Niu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lu Niu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu 610041, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lu++Niu">Lu Niu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Raina Seupel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Raina Seupel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Oxford, Old Road Campus Research Building, Roosevelt Drive, Oxford OX3 7DQ, U.K.</div><div class="loa-info-affiliations-info">Target Discovery Institute, University of Oxford, NDM Research Building, Roosevelt Drive, Oxford OX3 7FZ, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Raina++Seupel">Raina Seupel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chengyong Wu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chengyong Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu 610041, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chengyong++Wu">Chengyong Wu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wei Cheng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wei Cheng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu 610041, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wei++Cheng">Wei Cheng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chong Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chong Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu 610041, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chong++Chen">Chong Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bisen Ding</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bisen Ding</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu 610041, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bisen++Ding">Bisen Ding</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paul E. Brennan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paul E. Brennan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Oxford, Old Road Campus Research Building, Roosevelt Drive, Oxford OX3 7DQ, U.K.</div><div class="loa-info-affiliations-info">Target Discovery Institute, University of Oxford, NDM Research Building, Roosevelt Drive, Oxford OX3 7FZ, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paul+E.++Brennan">Paul E. Brennan</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8950-7646" title="Orcid link">http://orcid.org/0000-0002-8950-7646</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Shengyong Yang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shengyong Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu 610041, P. R. China</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#423b232c25313b023121376c2726376c212c"><span class="__cf_email__" data-cfemail="e69f878881959fa6958593c8838293c88588">[email protected]</span></a>. Phone: +86-28-85164063. Fax: +86-28-85164060.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shengyong++Yang">Shengyong Yang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5147-3746" title="Orcid link">http://orcid.org/0000-0001-5147-3746</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00096&amp;href=/doi/10.1021%2Facs.jmedchem.9b00096" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 4526–4542</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 18, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>15 January 2019</li><li><span class="item_label"><b>Published</b> online</span>18 April 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 9 May 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00096" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00096</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4526%26pageCount%3D17%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DLuyi%2BHuang%252C%2BHui%2BLi%252C%2BLinli%2BLi%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D9%26contentID%3Dacs.jmedchem.9b00096%26title%3DDiscovery%2Bof%2BPyrrolo%255B3%252C2-d%255Dpyrimidin-4-one%2BDerivatives%2Bas%2Ba%2BNew%2BClass%2Bof%2BPotent%2Band%2BCell-Active%2BInhibitors%2Bof%2BP300%252FCBP-Associated%2BFactor%2BBromodomain%26numPages%3D17%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4542%26publicationDate%3DMay%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00096"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2124</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">2</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00096" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Pyrrolo[3,2-d]pyrimidin-4-one Derivatives as a New Class of Potent and Cell-Active Inhibitors of P300/CBP-Associated Factor Bromodomain&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Luyi&quot;,&quot;last_name&quot;:&quot;Huang&quot;},{&quot;first_name&quot;:&quot;Hui&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Linli&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Lu&quot;,&quot;last_name&quot;:&quot;Niu&quot;},{&quot;first_name&quot;:&quot;Raina&quot;,&quot;last_name&quot;:&quot;Seupel&quot;},{&quot;first_name&quot;:&quot;Chengyong&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Wei&quot;,&quot;last_name&quot;:&quot;Cheng&quot;},{&quot;first_name&quot;:&quot;Chong&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Bisen&quot;,&quot;last_name&quot;:&quot;Ding&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;E. Brennan&quot;},{&quot;first_name&quot;:&quot;Shengyong&quot;,&quot;last_name&quot;:&quot;Yang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;18&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;4526-4542&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00096&quot;},&quot;abstract&quot;:&quot;Herein, we report the discovery of a series of new P300/CBP-associated factor (PCAF) bromodomain (BRD) inhibitors, which were obtained through a hit discovery process and subsequent structure-based optimization and structure–activity relationship analyses toward a retrieved hit compound (12). Among these inhibitors, (R,R)-36n is the most potent one with an IC50 of 7 nM in homogeneous time-resolved fluorescence assay and a KD of 78 nM in isothermal titration calorimetry assay. This compound also exhibited activity against GCN5 and FALZ, but weak or no activity against other 29 BRD proteins and 422 kinases, indicating considerable selectivity. X-ray cocrystal structure analysis revealed the molecular interaction mode and the precise stereochemistry required for bioactivity. Cellular activity, preliminary RNA-seq analysis, and pharmacokinetic properties were also examined for this compound. Collectively, this study provides a versatile tool molecule to explore molecular mechanisms of PCAF BRD regulation and also&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00096&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00096" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00096&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00096" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00096&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00096" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00096&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00096&amp;href=/doi/10.1021/acs.jmedchem.9b00096" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00096" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00096" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00096%26sid%3Dliteratum%253Aachs%26pmid%3D30998845%26genre%3Darticle%26aulast%3DHuang%26date%3D2019%26atitle%3DDiscovery%2Bof%2BPyrrolo%255B3%252C2-d%255Dpyrimidin-4-one%2BDerivatives%2Bas%2Ba%2BNew%2BClass%2Bof%2BPotent%2Band%2BCell-Active%2BInhibitors%2Bof%2BP300%252FCBP-Associated%2BFactor%2BBromodomain%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D9%26spage%3D4526%26epage%3D4542%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292304" title="Optimization">Optimization</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291321" title="Bioactivity">Bioactivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/jmcmar.2019.62.issue-9/20190509/jmcmar.2019.62.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00096&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Herein, we report the discovery of a series of new P300/CBP-associated factor (PCAF) bromodomain (BRD) inhibitors, which were obtained through a hit discovery process and subsequent structure-based optimization and structure–activity relationship analyses toward a retrieved hit compound (<b>12</b>). Among these inhibitors, (<i>R</i>,<i>R</i>)-<b>36n</b> is the most potent one with an IC<sub>50</sub> of 7 nM in homogeneous time-resolved fluorescence assay and a <i>K</i><sub>D</sub> of 78 nM in isothermal titration calorimetry assay. This compound also exhibited activity against GCN5 and FALZ, but weak or no activity against other 29 BRD proteins and 422 kinases, indicating considerable selectivity. X-ray cocrystal structure analysis revealed the molecular interaction mode and the precise stereochemistry required for bioactivity. Cellular activity, preliminary RNA-seq analysis, and pharmacokinetic properties were also examined for this compound. Collectively, this study provides a versatile tool molecule to explore molecular mechanisms of PCAF BRD regulation and also offers a new lead compound for drug discovery targeting PCAF.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">1.  Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35307" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35307" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Bromodomains (BRDs) are highly conserved epigenetic “reader” protein modules that recognize ε-<i>N</i>-acetylated lysine marks on proteins<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> and play a key role in the epigenetic regulation of gene transcription. A total of 61 BRDs are identified in the human proteome, which are present in 46 diverse proteins.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Among these BRDs, BET (bromo and extra-terminal) BRDs are the most extensively studied subfamily (subfamily II of BRD phylogenetic tree). A large number of selective and potent BET BRD inhibitors have been discovered,<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> which have provided versatile tools for biofunctional studies and further led to numerous translational studies on this subfamily of BRD proteins.<a onclick="showRef(event, 'ref5 ref6 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref8 ref9">(5−9)</a> In contrast, BRDs in non-BET subfamilies have received less attention. Biological functions of many non-BET BRDs in physiological and pathological conditions are still not clear, and inhibitors of these BRDs are also much less.<a onclick="showRef(event, 'ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12">(10−12)</a></div><div class="NLM_p">The P300/CBP-associated factor (PCAF), also known as lysine acetyltransferase 2B (KAT2B), is a BRD-containing protein, which belongs to subfamily I of the BRD phylogenetic tree.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> PCAF is a multidomain protein including an acetyltransferase (HAT) domain,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> a N-terminal E3 ubiquitin ligase domain,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> and a C-terminal BRD,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> which has been implicated in a number of disparate disease pathologies, and small-molecule modulators have great potential as therapeutics.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Regulation mechanisms and roles in diseases of these domains, particularly the C-terminal BRD, are far from understood. Small-molecule inhibitors might provide tool molecules to unravel the functions of the PCAF BRD on the one hand, and on the other hand, offer potential lead compounds for drug development targeting PCAF BRD.</div><div class="NLM_p">To date, a number of small-molecule inhibitors of PCAF BRD have been reported, which are summarized in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. Zeng et al. reported the first PCAF inhibitor <b>1</b>, which could disrupt the PCAF-BRD/Tat-AcK50 interaction in vitro.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Subsequent structural optimization of <b>1</b> generated a more potent compound <b>2</b>.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Navratilova et al.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> and Chaikuad et al.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> separately used fragment-based screening approaches to identify PCAF BRD inhibitors, and a number of inhibitors (fragments) were retrieved, for example, compound <b>3</b> and <b>4</b>. In 2016, Genentech and Constellation disclosed potent dual PCAF/GCN5 BRDs inhibitors in three patents, and compounds <b>5–7</b> shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> are representative ones, although no selectivity data were reported.<a onclick="showRef(event, 'ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23">(21−23)</a> Later, Brennan and co-workers reported a [1,2,4]triazolo[3,4-<i>a</i>]phthalazine derivative, <b>8</b> (<b><i>L</i>-Moses</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), as a potent, selective, and cell-active PCAF probe.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Researchers from GlaxoSmithKline published a potent PCAF/GCN5 BRDs inhibitor <b>9</b>,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and very recently, they further derived a PROTAC compound (GSK699) from <b>9</b>.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative PCAF BRD inhibitors publicly reported.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00096&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Despite these recent advances, there are still very few reported potent and selective PCAF BRD inhibitors. The aim of this investigation was to identify more novel potent and selective PCAF BRD inhibitors. To achieve this goal, we first performed virtual screening (VS) against an in-house chemical database to retrieve hit compounds, which led to the discovery of three hits. The most potent hit compound, which contains a new pyrrolo[3,2-<i>d</i>]pyrimidin-4-one scaffold, was then selected for subsequent structural optimization. Structure–activity relationship (SAR) analyses resulted in the discovery of a series of new PCAF BRD inhibitors containing the scaffold pyrrolo[3,2-<i>d</i>]pyrimidin-4-one. For the most active compound, further investigations including selectivity profiling, ligand–receptor interaction analysis, cellular activity, preliminary RNA-seq (RNA-sequencing) analysis, and pharmacokinetic studies were carried out.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">2.  Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26751" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26751" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4">2.1.  Retrieving of Hit Compounds</h3><div class="NLM_p">To discover more potent PCAF BRD inhibitors with new scaffolds, we first conducted a VS study against an in-house chemical database (for details of the VS, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00096/suppl_file/jm9b00096_si_001.pdf" class="ext-link">Supporting Information</a> Figure S1). Selected hit compounds in VS were then subjected to a differential scanning fluorimetry (DSF) assay at a compound concentration of 20 μM. Compounds with a thermal shift (Δ<i>T</i><sub>m</sub>) of ≥1 °C in the DSF assay were then validated by isothermal titration calorimetry (ITC) assay. We finally obtained three weakly active compounds, <b>10</b>, <b>11</b>, and <b>12</b>, which showed <i>K</i><sub>D</sub> values of 45, 7.8, and 2.4 μM in the ITC assay, respectively (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Compound <b>10</b> shows some similarity to previously reported BRD7/9,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> BAZ2A/B,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and CBP/EP300<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29,30)</a> BRD inhibitors, and compound <b>11</b> contains the previously reported phthalazinone scaffold,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> but compound <b>12</b> harbors a new scaffold, pyrrolo[3,2-<i>d</i>]pyrimidin-4-one, which has not been reported previously in BRD inhibitors. We chose compound <b>12</b> to conduct further structural optimization because it is the most potent compound and has a novel scaffold.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Three active compounds found by VS and subsequent bioactivity evaluation. (A) Chemical structures of <b>10</b>, <b>11</b>, and <b>12</b>. (B) Isothermal titration data of the interaction of hit compounds (for <b>10</b> and <b>11</b>, 50 μM ligand and 500 μM PCAF BRD were used; for <b>12</b>, 20 μM ligand and 200 μM PCAF BRD were used) with PCAF BRD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00096&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To facilitate the following structural optimization of <b>12</b>, an X-ray cocrystal structure of PCAF in complex with <b>12</b> was solved with a resolution of 2.1 Å resolution (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6J3O">6J3O</a>). As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, compound <b>12</b> bound to the acetyllysine-binding site and had been well defined by electron density. This compound forms hydrogen bonds with four amino acids, Pro747, Asn803, Tyr760, and Tyr809, either directly or via water-mediated interaction. The methyl substituent at the 3-position was in the water channel occupied by the acetyl lysine of the histone ligand. We also observed a π–π stacking interaction between the pyrrolo[3,2-<i>d</i>]pyrimidin-4-one core and the benzene ring of Tyr809. The basic piperidine group forms a salt bridge with the acidic side chain of Glu756, which is further stabilized by the backbone amide nitrogen of Lys753 through a hydrogen bond. Here, it is necessary to mention that compound <b>12</b> seems to make the same interactions with PCAF BRD as compound (<i>R</i>)-<b>23</b> in ref <a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a>, and the only difference is that compound <b>12</b> forms an additional hydrogen bond with Asn803.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cocrystal structure of PCAF/<b>12</b> guided the structural optimization. (A) Cocrystal structure of compound <b>12</b> (yellow sticks) in complex with PCAF BRD (pale cyan cartoon) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6J3O">6J3O</a>). Related residues are shown as green sticks. (B) Experimental electron density map (2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub>) contoured at 1σ around the hit <b>12</b>. (C) Focused regions in the structural optimization and SAR analyses.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00096&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5">2.2.  Structural Optimization and SAR Analyses of Pyrrolo[3,2-<i>d</i>]pyrimidin-4-one Derivatives</h3><div class="NLM_p">Structural optimization of hit compound <b>12</b> was focused on three regions, namely, the 2-amino moiety (region I), the 3-position (region II), and the 5-position substituent (region III) of pyrrolo[3,2-<i>d</i>]pyrimidin-4-one (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). A number of new pyrrolo[3,2-<i>d</i>]pyrimidin-4-one derivatives were designed and synthesized. Bioactivities of these compounds against PCAF BRD were measured by DSF and ITC assays.</div><div id="sec2_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i6" class="anchor-spacer"></div><h4 class="article-section__title" id="_i6">2.2.1.  Impact of 2-Amino Moiety (Region I)</h4><div class="NLM_p">To explore the impact of the 2-amino moiety of compound <b>12</b> on the bioactivity, we varied region I and fixed regions II and III with their original substituents. According to the cocrystal structure of the <b>12</b>-PCAF complex (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A), the basic <i>N</i>-methyl piperidine in region I, which forms a salt bridge with acidic Glu756, plays an important role in the ligand–receptor interaction. However, the <i>N</i>-methyl piperidine group is not large enough to occupy the ZA channel. Accordingly, all substituents used in region I were required to contain a secondary or tertiary amine and be larger than <i>N</i>-methyl piperidine to make potential interactions with Trp746 and Lys753. Based on this observation and inspiration from the SAR study of compound <b>9</b>,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> we designed and synthesized a total of 10 compounds [(<i>R</i>,<i>R</i>)-<b>24a</b>, (<i>S</i>,<i>S</i>)-<b>24a</b>, <i>cis</i>-<b>24b</b>, <i>endo</i>-<b>24c–e</b>, <i>endo</i>-<b>25a</b>, <i>exo</i>-<b>25a</b>, and <i>endo</i>-<b>26a–b</b>].</div><div class="NLM_p"><a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> shows synthetic routes for these compounds. Commercially available 2,4-dichloro-5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidine (<b>13</b>) underwent alkaline hydrolysis to produce intermediate <b>14</b>. Boc-protection of <b>14</b> gave <b>15</b>. Subsequent methylation and Boc-deprotection of <b>15</b> generated the key intermediate <b>16</b>. (<i>R</i>)/(<i>S</i>)-<b>17a</b> or <b>17b</b> (racemate) reacted with picolinic acid to produce (<i>R</i>)/(<i>S</i>)-<b>18a</b> or <b>18b</b>, which underwent C(sp<sup>3</sup>)–H arylation with iodobenzene to afford enantiomerically pure (<i>R</i>,<i>R</i>)/(<i>S</i>,<i>S</i>)-<b>19a</b> or <i>cis</i>-<b>19b</b>, respectively.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Compounds (<i>R</i>,<i>R</i>)/(<i>S</i>,<i>S</i>)-<b>19a</b> and <i>cis</i>-<b>19b</b> were further converted to (<i>R</i>,<i>R</i>)/(<i>S</i>,<i>S</i>)-<b>20a</b> and <i>cis</i>-<b>20b</b> by Boc-deprotection, reductive amination, and subsequent alkaline hydrolysis. (<i>R</i>,<i>R</i>)/(<i>S</i>,<i>S</i>)-<b>20a</b> and <i>cis</i>-<b>20b</b> together with two commercially available amines <b>21–22</b> were subjected to a tractable S<sub>N</sub>Ar reaction with <b>16</b> to afford target products (<i>R</i>,<i>R</i>)-<b>24a</b>, (<i>S</i>,<i>S</i>)-<b>24a</b>, <i>cis</i>-<b>24b</b>, and <i>endo</i>-<b>24c–d</b>. <b>23</b> (<i>endo</i>/<i>exo</i> mixture) reacted with <b>16</b> to afford <i>endo</i>- and <i>exo</i>-<b>24</b>e. Acidic N-deprotection of <i>endo</i>- and <i>exo</i>-<b>24e</b> afforded <i>endo</i>- and <i>exo</i>-<b>25a</b>, respectively. <i>N</i>-alkylation of <i>endo</i>-<b>25a</b> with (<i>R</i>)-2-phenyloxirane or (<i>R</i>)-2-bromo-1-(pyridin-2-yl)ethan-1-ol produced <i>endo</i>-<b>26a–b</b>.</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of (<i>R</i>,<i>R</i>)-<b>24a</b>, (<i>S</i>,<i>S</i>)-<b>24a</b>, <i>cis</i>-<b>24b</b>, <i>endo</i>-<b>24c–e</b>, <i>endo</i>-<b>25a</b>, <i>exo</i>-<b>25a</b>, and <i>endo</i>-<b>26a–b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00096&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 1 M NaOH (aq), 100 °C, 16 h, 90%; (b) Boc<sub>2</sub>O, TEA, DMAP, DMF, rt, 82%; (c) iodomethane, NaH, anhydrous DMF, 40 °C, overnight; (d) TFA, DCM, rt, 46% over two steps; (e) picolinic acid, HATU, DIPEA, DCM, rt, 83–88%; (f) iodobenzene, Pd(OAc)<sub>2</sub>, Ag<sub>2</sub>CO<sub>3</sub>, 2,6-dimethylbenzoic acid, <i>t</i>-BuOH, 120 °C, 24 h, 62–77%, >99% ee; (g) TFA, DCM, rt; (h) 37% formaldehyde, NaBH(OAc)<sub>3</sub>, AcOH, DCM, rt, 8 h, 84%; (i) NaOH, <i>i</i>-PrOH, 85 °C, 18 h, 78–85%; (j) <b>16</b>, primary amines, DIPEA, NMP, 150 °C, 2–5 h, 12–37%; (k) (<i>R</i>)-2-phenyloxirane or (<i>R</i>)-2-bromo-1-(pyridin-2-yl)ethan-1-ol, TEA, MeCN, 70 °C, overnight, 47–58%.</p></p></figure><div class="NLM_p">The synthesized compounds were assayed against the PCAF BRD, and bioactivities of these compounds are presented in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. (<i>R</i>,<i>R</i>)-<b>24a</b>, containing a (<i>R</i>,<i>R</i>)-1-methyl-5-phenylpiperidin-3-amino group, showed a thermal shift (Δ<i>T</i><sub>m</sub>) of 6.3 °C in DSF assay, and a <i>K</i><sub>D</sub> of 152 nM in ITC assay, whereas its enantiomer (<i>S</i>,<i>S</i>)-<b>24a</b> lost activity. <i>Cis</i>-<b>24b</b> with piperidine in (<i>R</i>,<i>R</i>)-<b>24a</b> replaced by azacycloheptane also displayed considerable activity in both assays (Δ<i>T</i><sub>m</sub> = 3.2 °C and <i>K</i><sub>D</sub> = 347 nM), but relatively weaker potency compared with those of (<i>R</i>,<i>R</i>)-<b>24a</b>. Among the remaining seven compounds containing a bridged piperidine, <i>endo</i>-<b>24c</b> was the most potent one (Δ<i>T</i><sub>m</sub> = 7.3 °C and <i>K</i><sub>D</sub> = 0.215 μM). The <i>N</i>-methyl removal product <i>endo</i>-<b>25a</b> slightly decreased in bioactivity (Δ<i>T</i><sub>m</sub> = 6.6 °C and <i>K</i><sub>D</sub> = 0.242 μM), whereas the diastereomer <i>exo</i>-<b>25a</b> did not exhibit activity. <i>Endo</i>-<b>24e</b>, <i>endo</i>-<b>26a</b>, and <i>endo</i>-<b>26b</b> with a bulky substituent in the bridged piperidine displayed a significant reduced activity in both assays. <i>Endo</i>-<b>24d</b>, which contains a 2-carbon-bridged piperidine (tropane), showed no activity.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Bioactivities of Region I-Substituted Compounds against PCAF BRD</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0012.gif" alt="" id="GRAPHIC-d7e1197-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0013.gif" alt="" id="gr12" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Values are averages of triplicates ± SEM (2 μM PCAF, 10 μM compound).</p></div></div><div></div></div></div><div id="sec2_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9">2.2.2.  Influence of 3- and 5-Position Substitutions (Region II and III)</h4><div class="NLM_p">To examine the influence of 3- and 5-position substitutions of pyrrolo[3,2-<i>d</i>]pyrimidin-4-one core, we fixed 2-position (region I) with its optimal group, (<i>R</i>,<i>R</i>)-1-methyl-5-phenylpiperidin-3-amino, and varied the 3- or 5-position substituents. Six new compounds ((<i>R</i>,<i>R</i>)-<b>32a–f</b>) were designed and synthesized (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>); for the 3-position substituents, we introduced unsaturated alkyl groups because it has been reported that unsaturated alkyl groups at this position might influence the bioactivity or selectivity across BRDs.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Intermediate <b>15</b> underwent an alkylation reaction with various haloalkanes followed by Boc-deprotection which gave <b>27a–d</b>. Methylation of <b>14</b> with methyl iodide afforded <b>28</b>. Selective hydrolysis of <b>29</b> with 1 M NaOH generated <b>30</b>, and methylation of <b>30</b> resulted in the formation of <b>31</b> in an excellent yield. Finally, <b>27a–d</b>, <b>28</b>, and <b>31</b> reacted with (<i>R</i>,<i>R</i>)-<b>20a</b> to produce (<i>R</i>,<i>R</i>)-<b>32a–f</b>, respectively.</div><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of (<i>R</i>,<i>R</i>)-<b>32a–f</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00096&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) haloalkanes, NaH, anhydrous DMF, 50 °C, overnight, 34–78%; (b) TFA, DCM, rt; (c) 1 M NaOH (aq), 100 °C, 95%; (d) (<i>R</i>,<i>R</i>)-<b>20a</b>, DIPEA, NMP, 150 °C, 2 h, 13–47%.</p></p></figure><div class="NLM_p"><a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> shows the bioactivities of these compounds. From <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, we can see that the longer the (unsaturated) alkyl chain of 3-position, the worse the bioactivity ((<i>R</i>,<i>R</i>)-<b>32a–d</b>), indicating that a bulky group at 3-position is unfavorable for bioactivity. To explore the impact of 5-position substitution of pyrrolo[3,2-<i>d</i>]pyrimidin-4-one, we first added a methyl group at 5-position. Bioactivity of the resulting compound (<i>R</i>,<i>R</i>)-<b>32e</b> decreased significantly, implying that the hydrogen bond (NH···O) is important and substitution at this position is not favored (see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). Displacing the nitrogen atom on 5-position with a sulfur atom ((<i>R</i>,<i>R</i>)-<b>32f</b>) also decreased the binding affinity, demonstrating again the importance of the hydrogen bond (NH···O).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Bioactivities of Compounds (<i>R</i>,<i>R</i>)-<b>32a–f</b> against PCAF BRD</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0014.gif" alt="" id="GRAPHIC-d7e1382-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0015.gif" alt="" id="gr13" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Values are averages of triplicates ± SEM (2 μM PCAF, 10 μM compound).</p></div></div><div></div></div></div><div id="sec2_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12">2.2.3.  Effect of Various Substituted 3-Amino-5-phenylpiperidine (Region I)</h4><div class="NLM_p">In preceding structural optimization, we obtained a potent compound, (<i>R</i>,<i>R</i>)-<b>24a</b>. SAR analyses indicated that any changes in region II and III were not favored for bioactivity improvement. We thus turned back to recheck region I and examined the possible effect of various substituents on the piperidine ring (R<sup>3</sup> and R<sup>4</sup>). Fifteen compounds [(<i>R</i>,<i>R</i>)-<b>36a–n</b> and (<i>S</i>,<i>S</i>)-<b>36n</b>] were synthesized.</div><div class="NLM_p"><a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a> outlines the reaction routes for these compounds. Compound (<i>R</i>,<i>R</i>)- or (<i>S</i>,<i>S</i>)-<b>18a</b> reacted with various aryl iodides via C(sp<sup>3</sup>)–H arylation to afford enantiomerically pure (<i>R</i>,<i>R</i>)-<b>19a</b>, (<i>R</i>,<i>R</i>)-<b>33a–j</b> and (<i>S</i>,<i>S</i>)-<b>19a</b>.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Compounds (<i>R</i>,<i>R</i>)-<b>19a</b>, (<i>R</i>,<i>R</i>)-<b>33a–j</b>, and (<i>S</i>,<i>S</i>)-<b>19a</b> were further converted to (<i>R</i>,<i>R</i>)-<b>34a–m</b> and (<i>S</i>,<i>S</i>)-<b>34m</b> by Boc-deprotection and subsequent reductive amination or nucleophilic substitution. Cleavage of the bidentate directing groups of (<i>R</i>,<i>R</i>)-<b>34a–m</b> and (<i>S</i>,<i>S</i>)-<b>34m</b> with the use of NaOH/<i>i</i>-PrOH resulted in amines (<i>R</i>,<i>R</i>)-<b>35a–m</b> and (<i>S</i>,<i>S</i>)-<b>35m</b>, which were subjected to S<sub>N</sub>Ar reaction with <b>16</b> to afford target products (<i>R</i>,<i>R</i>)-<b>36a–i</b>, (<i>R</i>,<i>R</i>)-<b>36k–n</b>, and (<i>S</i>,<i>S</i>)-<b>36n</b>. Pd/C catalyzed debenzylation of (<i>R</i>,<i>R</i>)-<b>36i</b> produced (<i>R</i>,<i>R</i>)-<b>36j</b>.</div><figure id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds (<i>R</i>,<i>R</i>)-<b>36a–n</b> and (<i>S</i>,<i>S</i>)-<b>36n</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00096&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) aryl iodides, Pd(OAc)<sub>2</sub>, Ag<sub>2</sub>CO<sub>3</sub>, 2,6-dimethylbenzoic acid, <i>t</i>-BuOH, 120 °C, 24 h, 52–76%, >99% ee; (b) TFA, DCM, rt; (c) haloalkanes, TEA, MeCN, 80 °C, overnight, 76%; (d) 37% formaldehyde (aq), NaBH(OAc)<sub>3</sub>, AcOH, DCM, rt, 73%; (e) NaOH, <i>i</i>-PrOH, 85 °C, 18 h; (f) <b>16</b>, DIPEA, NMP, 150 °C, 2 h, 11–26%; (g) (<i>R</i>,<i>R</i>)-<b>36i</b>, 10% Pd/C, H<sub>2</sub>, MeOH, rt, 65%.</p></p></figure><div class="NLM_p">Bioactivities of these compounds are shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. Increasing the size of the <i>N</i>-alkyl substituent (R<sup>4</sup>) from methyl to allyl ((<i>R</i>,<i>R</i>)-<b>36a</b>), hydroxyethyl ((<i>R</i>,<i>R</i>)-<b>36b</b>), and phenethyl ((<i>R</i>,<i>R</i>)-<b>36c</b>) decreased the bioactivity. Meta- and/or para-substituted phenyl groups at R<sup>3</sup> ((<i>R</i>,<i>R</i>)-<b>36d–l</b>) delivered potent activities, which are comparable to those of (<i>R</i>,<i>R</i>)-<b>24a</b>. (<i>R</i>,<i>R</i>)-<b>36m</b>, bearing a naphthalene-2-yl, enhanced the activity (<i>K</i><sub>D</sub>: 0.103 μM). (<i>R</i>,<i>R</i>)-<b>36n</b> with R<sup>3</sup> substituted by 2,3-dihydro-1,4-benzodioxin-6-yl showed the most potent activity among all the synthesized compounds (<i>K</i><sub>D</sub>: 0.078 μM, Δ<i>T</i><sub>m</sub>: 9.5 °C) (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A,C). Of special note is the fact that the enantiomer (<i>S</i>,<i>S</i>)-<b>36n</b> showed no obvious activity.</div><figure id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Identification of (<i>R</i>,<i>R</i>)-<b>36n</b> as a potent inhibitor of PCAF BRD. (A) Chemical structure of (<i>R</i>,<i>R</i>)-<b>36n</b>. (B) HTRF assays of (<i>R</i>,<i>R</i>)-<b>36n</b> and <b><i>L</i>-Moses</b> on GST-PCAF. (C–E) ITC titration curves for the binding of (<i>R</i>,<i>R</i>)-<b>36n</b> (10 μM, 200 μL in cell) to PCAF (C, 100 μM, 40 μL in syringe), GCN5 (D, 160 μM, 40 μL in syringe), and FALZ (E, 125 μM, 40 μL in syringe).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00096&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Bioactivities of Compounds (<i>R</i>,<i>R</i>)-<b>36a–n</b> and (<i>S</i>,<i>S</i>)-<b>36n</b> against PCAF BRD</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0016.gif" alt="" id="GRAPHIC-d7e1884-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0017.gif" alt="" id="gr14" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><p class="first last"><sup>a</sup>Values are averages of triplicates ± SEM (2 μM PCAF, 10 μM compound).</p></div></div><div></div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15">2.3.  Validation of Top Active Compounds by Biochemical Assay</h3><div class="NLM_p">In the abovementioned SAR studies, DSF and ITC assays were used to measure the bioactivity of the synthesized compounds, and a number of active compounds were identified. Because both DSF and ITC belong to biophysical assay, we then used homogeneous time-resolved fluorescence (HTRF) assay, which is a biochemical assay, to further validate the bioactivity. Here, just the top active compounds were tested. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, all of the tested compounds showed high potency in the HTRF assay. (<i>R</i>,<i>R</i>)-<b>36n</b> is again the most active one with an IC<sub>50</sub> of 7 nM. For comparison, we also tested the bioactivity of <b><i>L</i>-Moses</b>, which showed an IC<sub>50</sub> of 36 nM in the HTRF assay (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Biochemical Activities (HTRF, IC<sub>50</sub>) of Compounds That Showed Potent Activity in ITC and DSF Assays</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">(<i>R</i>,<i>R</i>)-<b>24a</b></td><td class="colsep0 rowsep0" align="char" char=".">0.041</td><td class="colsep0 rowsep0" align="left">(<i>R</i>,<i>R</i>)-<b>36i</b></td><td class="colsep0 rowsep0" align="char" char=".">0.014</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>endo</i>-<b>24c</b></td><td class="colsep0 rowsep0" align="char" char=".">0.073</td><td class="colsep0 rowsep0" align="left">(<i>R</i>,<i>R</i>)-<b>36j</b></td><td class="colsep0 rowsep0" align="char" char=".">0.030</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>endo</i>-<b>25a</b></td><td class="colsep0 rowsep0" align="char" char=".">0.125</td><td class="colsep0 rowsep0" align="left">(<i>R</i>,<i>R</i>)-<b>36k</b></td><td class="colsep0 rowsep0" align="char" char=".">0.025</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">(<i>R</i>,<i>R</i>)-<b>36d</b></td><td class="colsep0 rowsep0" align="char" char=".">0.022</td><td class="colsep0 rowsep0" align="left">(<i>R</i>,<i>R</i>)-<b>36l</b></td><td class="colsep0 rowsep0" align="char" char=".">0.010</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">(<i>R</i>,<i>R</i>)-<b>36e</b></td><td class="colsep0 rowsep0" align="char" char=".">0.032</td><td class="colsep0 rowsep0" align="left">(<i>R</i>,<i>R</i>)-<b>36m</b></td><td class="colsep0 rowsep0" align="char" char=".">0.047</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">(<i>R</i>,<i>R</i>)-<b>36f</b></td><td class="colsep0 rowsep0" align="char" char=".">0.035</td><td class="colsep0 rowsep0" align="left">(<i>R</i>,<i>R</i>)-<b>36n</b></td><td class="colsep0 rowsep0" align="char" char=".">0.007</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">(<i>R</i>,<i>R</i>)-<b>36g</b></td><td class="colsep0 rowsep0" align="char" char=".">0.017</td><td class="colsep0 rowsep0" align="left"><b><i>L</i>-Moses</b></td><td class="colsep0 rowsep0" align="char" char=".">0.036</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">(<i>R</i>,<i>R</i>)-<b>36h</b></td><td class="colsep0 rowsep0" align="char" char=".">0.030</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr></tbody></table></div></div><div class="NLM_p last">Overall, the abovementioned structural optimization and SAR studies finally led to the discovery of compound (<i>R</i>,<i>R</i>)-<b>36n</b>, which is among the most potent PCAF BRD inhibitors currently reported. Subsequently, further studies including selectivity profiling, interaction mode analyses, cellular activity evaluation, preliminary RNA-seq analysis, and pharmacokinetic studies were conducted on (<i>R</i>,<i>R</i>)-<b>36n</b>.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17">2.4.  Selectivity of (<i>R</i>,<i>R</i>)-<b>36n</b></h3><div class="NLM_p">To examine the selectivity of (<i>R</i>,<i>R</i>)-<b>36n</b>, we first used the DSF assay to measure activities of this compound against a panel of 12 BRDs available in our laboratory. (<i>R</i>,<i>R</i>)-<b>36n</b> showed the highest Δ<i>T</i><sub>m</sub> (9.5 °C) against PCAF. Besides PCAF, it also exhibited potent activity against GCN5 (Δ<i>T</i><sub>m</sub> = 7.3 °C) and FALZ (Δ<i>T</i><sub>m</sub> = 5.3 °C). To verify these activities, ITC was then used to determine the binding affinities of (<i>R</i>,<i>R</i>)-<b>36n</b> against GCN5 and FALZ, which gave <i>K</i><sub>D</sub> values of 105 and 917 nM, respectively (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D,E). This is not strange because PCAF BRD, GCN5 BRD, and FALZ BRD are highly homologous and share 69% sequence similarity. For the other 9 BRDs tested, (<i>R</i>,<i>R</i>)-<b>36n</b> displayed obviously weak or no activity (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00096/suppl_file/jm9b00096_si_001.pdf" class="ext-link">Supporting Information</a> Table S1).</div><figure id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Selectivity of (<i>R</i>,<i>R</i>)-<b>36n</b>. (A) Thermal shift analysis of (<i>R</i>,<i>R</i>)-<b>36n</b> (10 μM) against 12 BRDs, which belong to six distinct BRD subfamilies. The concentration of BRDs used was 2 μM. The heat map shows the relative Δ<i>T</i><sub>m</sub>, where the red color indicates a positive Δ<i>T</i><sub>m</sub> value and the green color indicates a negative Δ<i>T</i><sub>m</sub> value. (B) DiscoverX BROMOscan BRD cross-screen of (<i>R</i>,<i>R</i>)-<b>36n</b> at 1 μM. Binding activity is expressed as a percentage of the control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00096&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Further, the commercial BROMOscan assay was used to test the activity of this compound against a panel of 32 BRDs. Consistent with the results from previous DSF assays, (<i>R</i>,<i>R</i>)-<b>36n</b> (1 μM) showed potent activity against PCAF, GCN5, and FALZ. It also showed activity but much weaker against CECR2, BRPF, and BAZ2 families. For the remaining 23 BRDs, (<i>R</i>,<i>R</i>)-<b>36n</b> did not show activity (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00096/suppl_file/jm9b00096_si_001.pdf" class="ext-link">Supporting Information</a> Table S2).</div><div class="NLM_p last">In addition, we also tested the activity of (<i>R</i>,<i>R</i>)-<b>36n</b> against a panel of 422 kinases by Eurofins KinaseProfiler. Here, a single concentration (10 μM) of (<i>R</i>,<i>R</i>)-<b>36n</b> was used. The results showed that (<i>R</i>,<i>R</i>)-<b>36n</b> had no activity against all these kinases (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00096/suppl_file/jm9b00096_si_001.pdf" class="ext-link">Supporting Information</a> Table S3).</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18">2.5.  Interaction Mode of (<i>R</i>,<i>R</i>)-<b>36n</b> with Its Receptor</h3><div class="NLM_p">To understand the interaction mode of (<i>R</i>,<i>R</i>)-<b>36n</b> with PCAF BRD, we tried to culture the cocrystal of (<i>R</i>,<i>R</i>)-<b>36n</b> with PCAF BRD. Unfortunately, it was unsuccessful for some unknown reasons. We thus transferred to culture the cocrystal of (<i>R</i>,<i>R</i>)-<b>36n</b> bound to GCN5 BRD. As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>, (<i>R</i>,<i>R</i>)-<b>36n</b> occupies the KAc (acetylated lysine) binding site. Interactions found in the <b>12</b>-BRD complex (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A), including hydrogen bonding, π-stacking interaction, and salt bridge, all exist in the (<i>R</i>,<i>R</i>)-<b>36n</b>/BRD complex. Besides, we observed an additional edge-to-face aromatic interaction between the 2,3-dihydro-1,4-benzodioxine group and Trp751 in the (<i>R</i>,<i>R</i>)-<b>36n</b>/BRD complex, which may explain the higher activity of (<i>R</i>,<i>R</i>)-<b>36n</b> compared with that of <b>12</b>.</div><figure id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Structure and binding mode of (<i>R</i>,<i>R</i>)-<b>36n</b>. (A) X-ray cocrystal structure of (<i>R</i>,<i>R</i>)-<b>36n</b> (green sticks) bound to GCN5 BRD (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6J3P">6J3P</a>), with a conserved water network depicted (red spheres); yellow dashed lines indicate hydrogen bonds or salt bridges. (B) Surface view of the complex of GCN5 and (<i>R</i>,<i>R</i>)-<b>36n</b>. Mesh: 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> omitted electron density map contoured at 1.0σ.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00096&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19">2.6.  Bioactivity of (<i>R</i>,<i>R</i>)-<b>36n</b> in Intact Cells</h3><div class="NLM_p">The NanoBRET assay was adopted to assess the cell permeability and target engagement potential of (<i>R</i>,<i>R</i>)-<b>36n</b>.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> HEK293T cells were co-transfected with NanoLuc-tagged full-length PCAF and Halo-tagged histone H3.3 (Promega). The tagged PCAF-histone interaction in intact cells could be disrupted by competition with the test compound. In this assay, (<i>R</i>,<i>R</i>)-<b>36n</b> showed clear dose-dependent displacement of full length PCAF-NanoLuc from histone H3.3-HaloTag in HEK293T cells with an IC<sub>50</sub> of 118 nM, indicating that (<i>R</i>,<i>R</i>)-<b>36n</b> could pass through the cell membrane and target PCAF, whereas the enantiomer (<i>S</i>,<i>S</i>)-<b>36n</b> displayed no effect (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>).</div><figure id="fig7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. NanoBRET dose–response curves of (<i>R</i>,<i>R</i>)-<b>36n</b> and (<i>S</i>,<i>S</i>)-<b>36n</b> after 24 h treatment using N-terminal-nanoLuc-PCAF and C-terminal HaloTag-H3.3 in HEK293T cells (graph represents <i>n</i> = 2 biological replicates).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00096&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20">2.7.  Effect of (<i>R</i>,<i>R</i>)-<b>36n</b> on Global Gene Expression</h3><div class="NLM_p">RNA-seq (RNA-sequencing) analysis was adopted to explore the effect of (<i>R</i>,<i>R</i>)-<b>36n</b> (2.5 μM) on global gene expression. In this assay, (<i>S</i>,<i>S</i>)-<b>36n</b> (2.5 μM) and dimethyl sulfoxide (DMSO) were taken as negative and blank controls, respectively. Mouse embryonic fibroblast (MEF) cells were used. MEF cells were treated for 48 h in advance and then collected for RNA-seq analysis. RNA from each set (<i>n</i> = 3) was used to analyze the global gene expression. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>, treatment of (<i>R</i>,<i>R</i>)-<b>36n</b> significantly influenced the expression of 58 genes (>1.5-fold), among which 49 genes (39 upregulated and 10 downregulated) were uniquely impacted by (<i>R</i>,<i>R</i>)-<b>36n</b>, which might be attributed to the role of PCAF/GCN5 BRDs. Even so, the biology significance of the gene up- or downregulation needs further in-depth studies.</div><figure id="fig8" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Effect of (<i>R</i>,<i>R</i>)-<b>36n</b> on global gene expression. (A) Venn diagram showing a number of genes significantly up- or downregulated greater than 1.5 fold (<i>p</i> < 0.05) following (<i>R</i>,<i>R</i>)-<b>36n</b>, (<i>S</i>,<i>S</i>)-<b>36n</b> or DMSO treatment for 48 h in MEF cells. The green circle represents the number of influenced genes by (<i>R</i>,<i>R</i>)-<b>36n</b> relative to blank control, the red circle means the differential gene of (<i>S</i>,<i>S</i>)-<b>36n</b> relative to the blank control. (B) Hierarchical clustering heat map of selected genes showing differential regulation by (<i>R</i>,<i>R</i>)-<b>36n</b> and (<i>S</i>,<i>S</i>)-<b>36n</b> [genes up- or downregulated by >1.5-fold and <i>p</i> < 0.05 of (<i>R</i>,<i>R</i>)-<b>36n</b>]. Data from three donors are shown for each experiment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00096&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21">2.8.  Preliminary Assessment for the Pharmacokinetic Properties of (<i>R</i>,<i>R</i>)-<b>36n</b></h3><div class="NLM_p">The pharmacokinetic characteristics of (<i>R</i>,<i>R</i>)-<b>36n</b> were assessed on rats. A dose of 10 mg/kg was administrated through intravenous infusion or oral administration (for the blood concentration–time profiles, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00096/suppl_file/jm9b00096_si_001.pdf" class="ext-link">Supporting Information</a> Figure S3). The key oral administration pharmacokinetic parameters calculated are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>. The area under the concentration–time curve (AUC<sub>0-∞</sub>) is 2486 h·ng/mL. The half-life (<i>T</i><sub>1/2</sub>) and the maximum plasma concentration (<i>C</i><sub>max</sub>) are 1.51 h and 1418 ng/mL, respectively. Importantly, (<i>R</i>,<i>R</i>)-<b>36n</b> has an excellent bioavailability of 52%. In addition, the measured in vitro clearance of (<i>R</i>,<i>R</i>)-<b>36n</b> is 45 mL/min/kg in human liver microsomes (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00096/suppl_file/jm9b00096_si_001.pdf" class="ext-link">Supporting Information</a> Table S5). The aqueous solubility and log <i>D</i><sub>pH7.4</sub> measured by HPLC are 480 μM and 1.9, respectively. Because it is the unbound drug that drives efficacy according to the free drug hypothesis, we thus determined the fraction unbound (fu<sub>b</sub>) in the Sprague-Dawley rat plasma, which gave an average value of 5.8%. All of these results indicate that (<i>R</i>,<i>R</i>)-<b>36n</b> possesses favorable pharmacokinetic properties.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Key Pharmacokinetic Parameters of (<i>R</i>,<i>R</i>)-<b>36n</b> (p.o.)</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">parameter</th><th class="colsep0 rowsep0" align="center" char="±">value<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>T</i><sub>1/2</sub> (h)</td><td class="colsep0 rowsep0" align="char" char="±">1.51 ± 0.23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>T</i><sub>max</sub> (h)</td><td class="colsep0 rowsep0" align="char" char="±">0.67 ± 0.29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>C</i><sub>max</sub> (ng/mL)</td><td class="colsep0 rowsep0" align="char" char="±">1418 ± 252</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC<sub>(0-<i>t</i>)</sub> (h·ng/mL)</td><td class="colsep0 rowsep0" align="char" char="±">2479 ± 399</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC<sub>(0–∞)</sub> (h·ng/mL)</td><td class="colsep0 rowsep0" align="char" char="±">2486 ± 403</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>F</i> (%)</td><td class="colsep0 rowsep0" align="char" char="±">52 ± 8.3</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">Expressed as mean ± SD, <i>n</i> = 3.</p></div></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">3.  Concluding Remarks</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63460" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63460" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In this investigation, we discovered a potent PCAR BRD inhibitor, (<i>R</i>,<i>R</i>)-<b>36n</b>. This compound contains a new scaffold, pyrrolo[3,2-<i>d</i>]pyrimidin-4-one. Besides, it also contains a substituted piperidine, which is a common feature and presents in two other reported series of active compounds (<b>7</b> and <b>9</b> in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref23 ref25'); return false;" href="javascript:void(0);" class="ref ref23 ref25">(23,25)</a> In addition to the PCAF inhibition, (<i>R</i>,<i>R</i>)-<b>36n</b> also showed activity against GCN5 and FALZ, but displayed very weak or no activity against other 29 BRDs and 422 kinases, indicating considerable selectivity. Interestingly, the enantiomer (<i>S</i>,<i>S</i>)-<b>36n</b> did not show activity against PCAF, implying that (<i>R</i>,<i>R</i>)-<b>36n</b> could be a very suitable probe in investigating the biologic functions of PCAF. RNA-seq analysis demonstrated that (<i>R</i>,<i>R</i>)-<b>36n</b> treatment affected the expression of 49 genes on MEF cells. However, the effects of the gene expression changes on biological functions are unknown, which needs further in-depth studies. Overall, we have obtained a potent and selective PCAF inhibitor. Nevertheless, the exploring of biological functions and medicinal applications of this compound still needs intensive studies.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">4.  Experimental Procedures</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27756" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27756" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25">4.1.  Chemistry</h3><div class="NLM_p">All reagents and solvents were purchased from commercial suppliers without further purification unless otherwise indicated. All reactions were monitored by thin-layer chromatography and visualized with UV light, ninhydrin stain, or iodine stain. Column chromatography was performed on pre-packed silica gel columns using a Biotage Isolera One flash purification system (LPLC). HPLC was performed on a Waters 2695 HPLC system. Gemini C18 reversed-phase column (4.6 mm Φ × 150 mm, 5 μm) was used for purity analysis, solubility and log <i>D</i> determinations, and Daicel Chiralpak IE chiral column (part no. 85325; 4.6 mm Φ × 150 mm; 5 μm) for enantiomeric excess (ee) determination. <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra were recorded on a Bruker AV-400 spectrometer. Coupling constants (<i>J</i>) are expressed in hertz (Hz). Chemical shifts are reported as parts per million (ppm) relative to an internal solvent reference. The following abbreviations were used in the NMR descriptions: s, singlet; d, doublet; t, triplet; q, quartet; m, multiplet; dd, double doublet; and br, broad peak. High-resolution ESI-MS data were recorded on an Agilent 1200-G6410A mass spectrometer. Purity of screening compounds was evaluated by NMR spectroscopy and HPLC analysis. All compounds had purity ≥95% by HPLC.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26">4.1.1.  2-Chloro-3-methyl-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one (<b>16</b>)</h4><div class="NLM_p">Step 1: 2,4-dichloro-5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidine (<b>13</b>, 25 g, 133 mmol) was added to 200 mL 1 M NaOH solution and stirred at 100 °C for 16 h. Upon cooling to rt, the dark brown solution was acidified to pH 5 with 3 N HCl. The precipitate was collected by filtration onto a sintered-glass funnel, washed with water (2 × 50 mL), and dried in vacuo to give 2-chloro-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one (<b>14</b>, 20.3 g, 120 mmol, 90%) as a brown solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.81 (s, 1H), 12.25 (s, 1H), 7.41 (t, <i>J</i> = 3.0 Hz, 1H), 6.35 (dd, <i>J</i> = 2.8, 2.0 Hz, 1H).</div><div class="NLM_p">Step 2: Compound <b>14</b> (20.3 g, 120 mmol), Boc<sub>2</sub>O (28.7 g, 132 mmol), triethylamine (13.3 g, 132 mmol), and DMAP (500 mg, 2.25 mmol) were suspended in 150 mL DMF. The mixture was stirred overnight at ambient temperature. Then, 450 mL water was added and the resulting solution was acidified to pH 6 with citric acid. The precipitate was collected by filtration onto a sintered-glass funnel, washed with water (2 × 50 mL), and dried in vacuo to give <i>tert</i>-butyl 2-chloro-4-oxo-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidine-5-carboxylate (<b>15</b>, 26.6 g, 98.6 mmol, 82%) as a brown solid. The product was used without further purification. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.07 (s, 1H), 7.85 (d, <i>J</i> = 3.5 Hz, 1H), 6.55 (d, <i>J</i> = 3.5 Hz, 1H), 1.58 (s, 9H).</div><div class="NLM_p last">Step 3: NaH (60% in mineral oil, 1.9 g, 47.5 mmol) was added portionwise to a stirred solution of <b>15</b> (8.7 g, 32.3 mmol) in anhydrous DMF (100 mL) at 0 °C. Twenty minutes later, iodomethane (6.67 g, 47 mmol) was added and the mixture was reacted at 40 °C overnight. After cooling to rt, the mixture was carefully diluted with water and extracted with ethyl acetate (EtOAc). The combined organic phase was added trifluoroacetic acid (TFA) and stirred for another 5 h. Then, the solvent was removed under reduced pressure. The residue was purified by Biotage Isolera LPLC to give <b>16</b> (2.7 g, 14.7 mmol, 46%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.28 (s, 1H), 7.43 (t, <i>J</i> = 2.9 Hz, 1H), 6.35 (t, <i>J</i> = 2.2 Hz, 1H), 3.62 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.30, 142.50, 141.19, 129.18, 116.20, 102.98, 32.96.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27">4.1.2.  <i>tert</i>-Butyl (3<i>R</i>)-3-(pyridine-2-amido)piperidine-1-carboxylate ((<i>R</i>)-<b>18a</b>)</h4><div class="NLM_p last">To a stirred solution of picolinic acid (11.1 g, 130 mmol), <i>tert</i>-butyl (3<i>R</i>)-3-[(pyridin-2-yl)amido]piperidine-1-carboxylate ((<i>R</i>)-<b>17a</b>, 25.0 g, 125 mmol), and DIPEA (15.0 g, 155 mmol) in dichloromethane (DCM) (250 mL) was added HATU (50.0 g, 131.5 mmol). The mixture was stirred at ambient temperature overnight and concentrated under reduced pressure. The residue was dissolved in EtOAc (200 mL) and washed with 0.05 M HCl (2 × 100 mL) and sat. aq NaHCO<sub>3</sub> (2 × 100 mL). The organic layer was concentrated in vacuo and purified by Biotage Isolera LPLC to give the title compound (<i>R</i>)-<b>18a</b> (32.1 g, 107 mmol, 84%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.64 (dt, <i>J</i> = 4.7, 1.3 Hz, 1H), 8.55 (d, <i>J</i> = 8.2 Hz, 1H), 8.10–7.97 (m, 2H), 7.61 (ddd, <i>J</i> = 7.4, 4.8, 1.5 Hz, 1H), 3.96–3.43 (m, 3H), 3.26–2.77 (m, 2H), 1.93–1.79 (m, 1H), 1.79–1.58 (m, 2H), 1.57–1.20 (m, 10H). Chiral HPLC (Chiralpak IE-H column, 4.6 mm × 25 cm, 20% <i>i</i>-PrOH/hexane, 1 mL/min): <i>R</i><sub>t</sub> = 25.86 min, >99% ee.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28">4.1.3.  <i>tert</i>-Butyl (3<i>R</i>,5<i>R</i>)-3-phenyl-5-(pyridine-2-amido)piperidine-1-carboxylate ((<i>R</i>,<i>R</i>)-<b>19a</b>)</h4><div class="NLM_p last">A pressure vial (250 mL) equipped with a magnetic stirring bar was charged with compound (<i>R</i>)-<b>18a</b> (7.2 g, 23.6 mmol), iodobenzene (24.1 g, 117.89 mmol), silver carbonate (6.5 g, 23.6 mmol), palladium acetate (529 mg, 2.36 mmol), 2,6-dimethylbenzoic acid (885 mg, 5.9 mmol), and 60 mL <i>t</i>-BuOH. The vessel was flushed with argon, sealed with a crimp cap, and heated to 120 °C. After 24 h, the reaction vessel was removed from the oil bath, cooled to room temperature, and added DCM (60 mL) to the reaction mixture. The mixture was thoroughly stirred for 10 min, and the solids were removed by filtration. The filtrate was concentrated under reduced pressure and purified by Biotage Isolera LPLC to give the title compound (<i>R</i>,<i>R</i>)-<b>19a</b> (6.91 g, 18.1 mmol, 77%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.74–8.62 (m, 2H), 8.10–7.96 (m, 2H), 7.61 (ddd, <i>J</i> = 7.4, 4.7, 1.4 Hz, 1H), 7.38–7.20 (m, 5H), 4.17–3.91 (m, 3H), 2.91–2.63 (m, 3H), 2.08–1.92 (m, 2H), 1.43 (s, 9H). Chiral HPLC (Chiralpak IE-H column, 4.6 mm × 25 cm, 20% EtOH/hexane, 1 mL/min): <i>R</i><sub>t</sub> = 31.52 min, >99% ee.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29">4.1.4.  (3<i>R</i>,5<i>R</i>)-1-methyl-5-phenylpiperidin-3-amine Dihydrochloride ((<i>R</i>,<i>R</i>)-<b>20a</b>)</h4><div class="NLM_p">Step 1: To a stirred solution of (<i>R</i>,<i>R</i>)-<b>19a</b> (6.8 g, 17.8 mmol) in DCM (80 mL) was added TFA (6 mL). The resulting mixture was stirred at room temperature for 4 h. The solvent was removed under reduced pressure, and the crude product was redissolved in water (60 mL). The solution was basified to pH 10 with 15% aq NaOH and extracted twice with EtOAc. The combined organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to give <i>N</i>-[(3<i>R</i>,5<i>R</i>)-5-phenylpiperidin-3-yl]pyridine-2-carboxamide (4.72 g, 16.8 mmol, 94%) as a pale-yellow solid that required no further purification.</div><div class="NLM_p">Step 2: A stirred solution of <i>N</i>-[(3<i>R</i>,5<i>R</i>)-5-phenylpiperidin-3-yl]pyridine-2-carboxamide (3.2 g, 9.03 mmol) in methanol (80 mL) was treated with glacial acetic acid (0.1 mL) and 37% w/v formaldehyde in water (1.1 mL), then with sodium triacetoxyborohydride (4.1 g, 19.33 mmol) portionwise (1.25 g every 20 min). Two hours after the final addition, the solvent was removed in vacuo, and the residue was partitioned between EtOAc and water. The aqueous phase was extracted twice with EtOAc, and the combined organics were washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. Purification of the residue by Biotage Isolera LPLC gave <i>N</i>-[(3<i>R</i>,5<i>R</i>)-1-methyl-5-phenylpiperidin-3-yl]pyridine-2-carboxamide (2.82 g, 9.55 mmol, 84%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.64 (dt, <i>J</i> = 4.8, 1.3 Hz, 1H), 8.56 (d, <i>J</i> = 8.7 Hz, 1H), 8.08–7.95 (m, 2H), 7.60 (ddd, <i>J</i> = 7.3, 4.7, 1.5 Hz, 1H), 7.36–7.25 (m, 4H), 7.25–7.16 (m, 1H), 4.20–4.06 (m, 1H), 2.98–2.80 (m, 3H), 2.24 (s, 3H), 2.01–1.84 (m, 3H), 1.75 (q, <i>J</i> = 12.2 Hz, 1H).</div><div class="NLM_p last">Step 3: To a suspension of <i>N</i>-[(3<i>R</i>,5<i>R</i>)-1-methyl-5-phenylpiperidin-3-yl]pyridine-2-carboxamide (2.8 g, 9.48 mmol) in <i>i</i>-PrOH (80 mL) was added NaOH (3.8 g, 95 mmol). The mixture was stirred at 85 °C for 18 h. Then, the solvent was removed under reduced pressure and water (50 mL) was added. The solution was extracted with EtOAc (50 mL × 2). The combined organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and added 15% HCl in ethanol (1 mL). Finally, the solvent was removed in vacuo to give (<i>R</i>,<i>R</i>)-<b>20a</b> (2.11 g, 85%, 8.0 mmol) as a pale-white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.78 (s, 1H), 8.80 (s, 3H), 7.44–7.21 (m, 5H), 3.81–3.70 (m, 1H), 3.64–3.56 (m, 1H), 3.54–3.46 (m, 1H), 3.37–3.26 (m, 1H), 3.25–3.05 (m, 2H), 2.84 (s, 3H), 2.28 (d, <i>J</i> = 12.3 Hz, 1H), 1.82 (q, <i>J</i> = 12.3 Hz, 1H).</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30">4.1.5.  3-Methyl-2-{[(3<i>R</i>,5<i>R</i>)-1-methyl-5-phenylpiperidin-3-yl]amino}-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one ((<i>R</i>,<i>R</i>)-<b>24a</b>)</h4><div class="NLM_p last">A solution of <b>16</b> (200 mg, 1.09 mmol) in NMP was treated with (<i>R</i>,<i>R</i>)-<b>20a</b> (344 mg, 1.31 mmol) and DIPEA (183 mg, 1.42 mmol). Then, the reaction was heated to 150 °C and stirred at this temperature for 2 h. After cooling to ambient temperature, the mixture was diluted with EtOAc and water. The biphasic solution was extracted twice with EtOAc. The combined organics were washed with brine, dried over anhydrous MgSO<sub>4</sub>, and concentrated in vacuo. Purification of the residue by Biotage Isolera LPLC gave the title compound (<i>R</i>,<i>R</i>)-<b>24a</b> (53 mg, 0.157 mmol, 14%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.37 (s, 1H), 7.36–7.13 (m, 5H), 7.11 (t, <i>J</i> = 2.9 Hz, 1H), 6.01 (t, <i>J</i> = 2.5 Hz, 1H), 5.97 (d, <i>J</i> = 7.7 Hz, 1H), 4.22–4.10 (m, 1H), 3.31 (s, 3H), 3.07 (dd, <i>J</i> = 10.4, 4.1 Hz, 1H), 2.92–2.78 (m, 2H), 2.20 (s, 3H), 2.04 (dd, <i>J</i> = 12.0, 4.2 Hz, 1H), 1.91–1.75 (m, 2H), 1.57 (q, <i>J</i> = 12.1 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.48, 149.57, 144.80, 143.97, 128.88, 127.80, 127.47, 126.86, 112.40, 101.18, 62.49, 60.33, 48.62, 46.14, 41.62, 37.11, 27.72. HRMS <i>m</i>/<i>z</i>: 338.1985 (M + H<sup>+</sup>, C<sub>21</sub>H<sub>25</sub>N<sub>5</sub>O<sub>3</sub>, requires 337.1903). Chiral HPLC (Chiralpak IE-H column, 4.6 mm × 25 cm, 25% <i>i</i>-PrOH/hexane, 1 mL/min): <i>R</i><sub>t</sub> = 16.53 min, >99% ee.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31">4.1.6.  3-Methyl-2-{[(3<i>S</i>,5<i>S</i>)-1-methyl-5-phenylpiperidin-3-yl]amino}-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one ((<i>S</i>,<i>S</i>)-<b>24a</b>)</h4><div class="NLM_p last">The title compound (<i>S</i>,<i>S</i>)-<b>24a</b> was prepared in the same manner as shown for (<i>R</i>,<i>R</i>)-<b>24a</b> except that <i>tert</i>-butyl (3<i>S</i>)-3-aminopiperidine-1-carboxylate (<i>S</i>)-<b>17a</b> was used instead. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.42 (s, 1H), 7.36–7.19 (m, 5H), 7.15 (t, <i>J</i> = 2.9 Hz, 1H), 6.05 (t, <i>J</i> = 2.5 Hz, 1H), 6.00 (d, <i>J</i> = 7.7 Hz, 1H), 4.26–4.13 (m, 1H), 3.36 (s, 3H), 3.10 (dd, <i>J</i> = 10.4, 4.2 Hz, 1H), 2.95–2.82 (m, 2H), 2.23 (s, 3H), 2.11–2.04 (m, 1H), 1.93–1.76 (m, 2H), 1.62 (q, <i>J</i> = 12.1 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.47, 149.57, 144.80, 144.06, 128.88, 127.77, 127.47, 126.84, 112.40, 101.18, 62.63, 60.46, 48.68, 46.23, 41.70, 37.16, 27.71. HRMS <i>m</i>/<i>z</i>: 338.1987 (M + H<sup>+</sup>, C<sub>19</sub>H<sub>23</sub>N<sub>5</sub>O, requires 337.1903). Chiral HPLC (Chiralpak IE-H column, 4.6 mm × 25 cm, 25% <i>i</i>-PrOH/hexane, 1 mL/min): <i>R</i><sub>t</sub> = 14.23 min, >99% ee.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32">4.1.7.  3-Methyl-2-[(<i>cis</i>-1-methyl-6-phenylazepan-4-yl)amino]-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one (<i>cis</i>-<b>24b</b>)</h4><div class="NLM_p last">The title compound <i>cis</i>-<b>24b</b> was prepared in the same manner as shown for (<i>R</i>,<i>R</i>)-<b>24a</b> except that <i>tert</i>-butyl 4-aminoazepane-1-carboxylate <b>17b</b> was used instead. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.39 (s, 1H), 7.31–7.22 (m, 4H), 7.19–7.10 (m, 2H), 6.06 (d, <i>J</i> = 7.6 Hz, 1H), 6.02 (t, <i>J</i> = 2.4 Hz, 1H), 4.33–4.20 (m, 1H), 3.35 (s, 3H), 3.06–2.95 (m, 1H), 2.73–2.62 (m, 4H), 2.32 (s, 3H), 2.17–1.97 (m, 3H), 1.90–1.77 (m, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.52, 149.35, 147.02, 144.93, 128.83, 127.65, 127.29, 126.40, 112.31, 101.21, 65.54, 53.85, 51.10, 47.36, 43.07, 42.07, 34.76, 27.62. HRMS <i>m</i>/<i>z</i>: 352.2137 (M + H<sup>+</sup>, C<sub>20</sub>H<sub>25</sub>N<sub>5</sub>O, requires 351.2059).</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33">4.1.8.  3-Methyl-2-{[(1<i>R</i>,3<i>r</i>,5<i>S</i>)-9-methyl-9-azabicyclo[3.3.1]nonan-3-yl]amino}-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one (<i>endo</i>-<b>24c</b>)</h4><div class="NLM_p last">Compound <b>16</b> (100 mg, 0.54 mmol), (1<i>R</i>,3<i>r</i>,5<i>S</i>)-9-methyl-9-azabicyclo[3.3.1]nonan-3-amine (100 mg, 0.65 mmol), and DIPEA (100 mg) were added to 4 mL NMP. The suspension was reacted at 150 °C for 2 h. After cooling to ambient temperature, the mixture was diluted with EtOAc and water. The resulting solution was extracted twice with EtOAc. The combined organic layer was concentrated and purified by Biotage Isolera LPLC to give <i>endo</i>-<b>24c</b> (37 mg, 12.3 mmol, 22.5%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.36 (s, 1H), 7.13 (t, <i>J</i> = 2.9 Hz, 1H), 6.03 (t, <i>J</i> = 2.4 Hz, 1H), 5.84 (d, <i>J</i> = 8.5 Hz, 1H), 4.39 (dtd, <i>J</i> = 14.4, 12.0, 6.5 Hz, 1H), 3.36 (s, 3H), 2.97 (d, <i>J</i> = 11.1 Hz, 2H), 2.41 (s, 3H), 2.29 (td, <i>J</i> = 12.1, 6.2 Hz, 2H), 2.18–2.03 (m, 1H), 1.92 (tt, <i>J</i> = 13.4, 4.3 Hz, 2H), 1.49–1.37 (m, 3H), 0.90 (d, <i>J</i> = 12.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.51, 149.78, 145.10, 127.68, 112.22, 101.06, 51.22, 43.31, 32.88, 27.67, 23.60, 14.54. HRMS <i>m</i>/<i>z</i>: 302.1981 (M + H<sup>+</sup>, C<sub>16</sub>H<sub>23</sub>N<sub>5</sub>O, requires 301.1903).</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34">4.1.9.  2-{[(1<i>R</i>,3<i>R</i>,5<i>S</i>)-8-Benzyl-8-azabicyclo[3.2.1]octan-3-yl]amino}-3-methyl-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one (<i>endo</i>-<b>24d</b>)</h4><div class="NLM_p last">The title compound <i>endo</i>-<b>24d</b> was prepared in the same manner as shown for <i>endo</i>-<b>24c</b> except that commercial (1<i>R</i>,3<i>R</i>,5<i>S</i>)-8-benzyl-8-azabicyclo[3.2.1]octan-3-amine was used instead. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.46 (s, 1H), 7.42–7.20 (m, 5H), 7.16 (t, <i>J</i> = 2.9 Hz, 1H), 6.06 (t, <i>J</i> = 2.4 Hz, 1H), 5.56 (d, <i>J</i> = 2.9 Hz, 1H), 4.03 (t, <i>J</i> = 6.6 Hz, 1H), 3.53 (s, 2H), 3.42 (s, 3H), 3.12 (s, 2H), 2.09–1.94 (m, 6H), 1.89–1.80 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.66, 149.61, 144.53, 128.81, 128.57, 127.63, 127.09, 112.84, 101.47, 58.15, 56.06, 43.69, 35.83, 27.76, 26.40. HRMS <i>m</i>/<i>z</i>: 364.2139 (M + H<sup>+</sup>, C<sub>21</sub>H<sub>25</sub>N<sub>5</sub>O, requires 363.2059).</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35">4.1.10.  3-Methyl-2-{[(1<i>R</i>,3<i>r</i>,5<i>S</i>)-9-azabicyclo[3.3.1]nonan-3-yl]amino}-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one (<i>endo</i>-<b>25a</b>) and 3-Methyl-2-{[(1<i>R</i>,3<i>s</i>,5<i>S</i>)-9-azabicyclo[3.3.1]nonan-3-yl]amino}-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one (<i>exo</i>-<b>25a</b>)</h4><div class="NLM_p">Step 1: <i>Endo</i>-<b>24e</b> and <i>exo</i>-<b>24e</b> were prepared in the same manner as shown for <i>endo</i>-<b>24c</b> except that commercial <i>tert</i>-butyl 3-amino-9-azabicyclo[3.3.1]nonane-9-carboxylate (endo/exo = 2:3; 3.14 g, 131 mmol) was used instead. The crude products were purified by Biotage Isolera LPLC to give <i>endo</i>-<b>24e</b> (830 mg, 2.14 mmol) and <i>exo</i>-<b>24e</b> (1.2 g, 3.1 mmol) as white solids. <i>endo</i>-<b>24e</b>: <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.39 (s, 1H), 7.14 (t, <i>J</i> = 2.9 Hz, 1H), 6.03–5.91 (m, 2H), 4.34 (dd, <i>J</i> = 20.2, 11.5 Hz, 2H), 3.90–3.79 (m, 1H), 3.35 (s, 3H), 2.37–2.00 (m, 4H), 1.57–1.33 (m, 15H). HRMS <i>m</i>/<i>z</i>: 388.2281 (M + H<sup>+</sup>, C<sub>21</sub>H<sub>25</sub>N<sub>5</sub>O<sub>3</sub>, requires 387.2270). Chiral HPLC (Chiralpak IE-H column, 4.6 mm × 25 cm, 20% <i>i</i>-PrOH/hexane, 1 mL/min): <i>R</i><sub>t</sub> = 17.14 min, >99% ee. <i>exo</i>-<b>24e</b>: <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.40 (s, 1H), 7.14 (t, <i>J</i> = 2.9 Hz, 1H), 6.01 (t, <i>J</i> = 2.4 Hz, 1H), 5.86 (d, <i>J</i> = 7.9 Hz, 1H), 4.97–4.84 (m, 1H), 4.23 (d, <i>J</i> = 16.3 Hz, 2H), 3.33 (s, 3H), 2.06–1.86 (m, 3H), 1.75–1.62 (m, 6H), 1.42 (s, 9H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.47, 153.38, 149.62, 144.85, 127.70, 112.34, 101.27, 78.78, 47.68, 46.08, 44.96, 36.78, 36.36, 29.44, 28.69, 27.68. HRMS <i>m</i>/<i>z</i>: 388.2237 (M + H<sup>+</sup>, C<sub>21</sub>H<sub>25</sub>N<sub>5</sub>O<sub>3</sub>, requires 387.2270). Chiral HPLC (Chiralpak IE-H column, 4.6 mm × 25 cm, 20% <i>i</i>-PrOH/hexane, 1 mL/min): <i>R</i><sub>t</sub> = 14.37 min, >99% ee.</div><div class="NLM_p last">Step 2: To a stirred solution of <i>endo</i>-<b>24e</b> in DCM at room temperature was added TFA. After reaction completion, the solvent was removed under reduced pressure, and the residue was basified and purified to give <i>endo</i>-<b>25a</b> as a white solid. <i>endo</i>-<b>25a</b>: <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.41 (s, 1H), 7.15 (d, <i>J</i> = 2.8 Hz, 1H), 6.02 (d, <i>J</i> = 2.8 Hz, 1H), 5.84 (d, <i>J</i> = 8.2 Hz, 1H), 4.28–4.12 (m, 1H), 3.37 (s, 3H), 3.29–3.23 (m, 2H), 2.19 (td, <i>J</i> = 11.7, 5.9 Hz, 2H), 2.10–1.97 (m, 1H), 1.56 (tt, <i>J</i> = 12.8, 4.2 Hz, 2H), 1.50–1.41 (m, 1H), 1.39–1.25 (m, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.51, 149.66, 145.00, 127.69, 112.30, 101.02, 45.23, 43.68, 32.69, 32.23, 27.70, 14.54. HRMS <i>m</i>/<i>z</i>: 288.1827 (M + H<sup>+</sup>, C<sub>21</sub>H<sub>25</sub>N<sub>5</sub>O<sub>3</sub>, requires 287.1746). <i>exo</i>-<b>25a</b> was prepared from <i>exo</i>-<b>24e</b> using the procedure described for compound <i>endo</i>-<b>25a</b> as a white solid. <i>exo</i>-<b>25a</b>: <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.40 (s, 1H), 7.14 (d, <i>J</i> = 2.8 Hz, 1H), 6.00 (d, <i>J</i> = 2.8 Hz, 1H), 5.81 (d, <i>J</i> = 7.7 Hz, 1H), 4.82 (tq, <i>J</i> = 12.2, 6.4 Hz, 1H), 3.35 (s, 3H), 3.18–3.13 (m, 2H), 2.05–1.86 (m, 3H), 1.85–1.60 (m, 7H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.52, 149.66, 144.96, 127.67, 112.31, 101.24, 47.31, 45.40, 38.01, 30.43, 27.67, 21.03. HRMS <i>m</i>/<i>z</i>: 288.1819 (M + H<sup>+</sup>, C<sub>21</sub>H<sub>25</sub>N<sub>5</sub>O<sub>3</sub>, requires 287.1746).</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36">4.1.11.  3-Methyl-2-{[(1<i>R</i>,3<i>r</i>,5<i>S</i>)-9-[(2<i>R</i>)-2-hydroxy-2-phenylethyl]-9-azabicyclo[3.3.1]nonan-3-yl]amino}-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one (<i>endo</i>-<b>26a</b>)</h4><div class="NLM_p last">To a stirred solution of <i>endo</i>-<b>25a</b> (100 mg, 0.35 mmol) and triethylamine (61 mg, 0.6 mmol) in MeCN was added (<i>R</i>)-2-phenyloxirane (60 mg, 0.5 mmol). The mixture was reacted at 70 °C overnight. After cooling to room temperature, the mixture was concentrated under reduced pressure, and the residue was purified by Biotage Isolera LPLC to give <i>endo</i>-<b>26a</b> in 58% yield as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.38 (s, 1H), 7.44–7.29 (m, 4H), 7.27–7.20 (m, 1H), 7.15 (t, <i>J</i> = 2.9 Hz, 1H), 6.05 (t, <i>J</i> = 2.4 Hz, 1H), 5.81 (d, <i>J</i> = 8.3 Hz, 1H), 4.89 (s, 1H), 4.57 (dd, <i>J</i> = 7.8, 5.1 Hz, 1H), 4.42–4.27 (m, 1H), 3.37 (s, 3H), 3.14–2.96 (m, 2H), 2.79 (dd, <i>J</i> = 13.1, 5.1 Hz, 1H), 2.68 (dd, <i>J</i> = 13.0, 7.8 Hz, 1H), 2.40–2.19 (m, 2H), 2.16–2.00 (m, 1H), 1.89–1.63 (m, 2H), 1.49–1.31 (m, 3H), 1.08–0.94 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.53, 149.73, 145.05, 128.24, 127.73, 127.14, 126.57, 112.28, 101.11, 71.39, 61.31, 51.48, 49.95, 43.34, 32.34, 27.71, 26.78, 26.33, 14.22. HRMS <i>m</i>/<i>z</i>: 408.2405 (M + H<sup>+</sup>, C<sub>23</sub>H<sub>29</sub>N<sub>5</sub>O<sub>2</sub>, requires 407.2321).</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37">4.1.12.  3-Methyl-2-{[(1<i>R</i>,3<i>r</i>,5<i>S</i>)-9-[(2<i>S</i>)-2-hydroxy-2-(pyridin-2-yl)ethyl]-9-azabicyclo[3.3.1]nonan-3-yl]amino}-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one (<i>endo</i>-<b>26b</b>)</h4><div class="NLM_p last">The title compound was prepared from <i>endo</i>-<b>25a</b> (100 mg, 0.75 mmol) and (<i>R</i>)-2-bromo-1-(pyridin-2-yl)ethan-1-ol (162 mg, 0.8 mmol) using the procedure described for compound <i>endo</i>-<b>26a</b> in 47% yield as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.38 (s, 1H), 8.57–8.43 (m, 1H), 7.79 (td, <i>J</i> = 7.7, 1.9 Hz, 1H), 7.53 (d, <i>J</i> = 7.8 Hz, 1H), 7.25 (dd, <i>J</i> = 7.4, 4.9 Hz, 1H), 7.15 (t, <i>J</i> = 2.9 Hz, 1H), 6.07 (t, <i>J</i> = 2.4 Hz, 1H), 5.80 (d, <i>J</i> = 8.3 Hz, 1H), 5.10 (s, 1H), 4.62 (dd, <i>J</i> = 7.9, 4.1 Hz, 1H), 4.39–4.26 (m, 1H), 3.37 (s, 3H), 3.13–2.96 (m, 3H), 2.70 (dd, <i>J</i> = 13.1, 7.9 Hz, 1H), 2.29 (dp, <i>J</i> = 16.2, 5.1 Hz, 2H), 2.15–2.01 (m, 1H), 1.86–1.67 (m, 2H), 1.47–1.33 (m, 3H), 1.00 (d, <i>J</i> = 12.8 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.80, 154.53, 149.72, 148.66, 145.03, 136.80, 127.73, 122.45, 121.03, 112.26, 101.12, 72.86, 59.56, 51.48, 49.98, 43.31, 32.38, 27.71, 26.71, 26.28, 14.22. HRMS <i>m</i>/<i>z</i>: 409.2359 (M + H<sup>+</sup>, C<sub>22</sub>H<sub>28</sub>N<sub>6</sub>O<sub>2</sub>, requires 408.2274).</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38">4.1.13.  2-Chloro-3-ethyl-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one (<b>27a</b>)</h4><div class="NLM_p last"><b>27a</b> was prepared from <b>15</b> (2.0 g, 7.42 mmol) and iodoethane (1.5 g, 9.65 mmol) using the procedure described for <b>16</b> in 51% yield as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.29 (s, 1H), 7.43 (t, <i>J</i> = 2.9 Hz, 1H), 6.35 (dd, <i>J</i> = 2.9, 2.0 Hz, 1H), 4.24 (q, <i>J</i> = 7.1 Hz, 2H), 1.26 (t, <i>J</i> = 7.1 Hz, 3H).</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39">4.1.14.  2-Chloro-3-(prop-2-en-1-yl)-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one (<b>27b</b>)</h4><div class="NLM_p last"><b>27b</b> was prepared from <b>15</b> (2.0 g, 7.42 mmol) and 3-bromoprop-1-ene (1.17 g, 9.65 mmol) using the procedure described for <b>16</b> in 49% yield as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.33 (s, 1H), 7.46 (t, <i>J</i> = 2.9 Hz, 1H), 6.38 (t, <i>J</i> = 2.4 Hz, 1H), 6.02–5.90 (m, 1H), 5.20 (dq, <i>J</i> = 10.5, 1.6 Hz, 1H), 5.02 (dq, <i>J</i> = 17.1, 1.7 Hz, 1H), 4.83 (dt, <i>J</i> = 4.9, 1.8 Hz, 2H).</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40">4.1.15.  3-(But-3-en-1-yl)-2-chloro-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one (<b>27c</b>)</h4><div class="NLM_p last"><b>27c</b> was prepared from <b>15</b> (2.0 g, 7.42 mmol) and 4-bromobut-1-ene (1.30 g, 9.65 mmol) using the procedure described for <b>16</b> in 34% yield as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.29 (s, 1H), 7.44 (t, <i>J</i> = 2.9 Hz, 1H), 6.35 (dd, <i>J</i> = 2.9, 2.0 Hz, 1H), 5.90–5.80 (m, 1H), 5.09–5.00 (m, 2H), 4.32–4.22 (m, 2H), 2.45 (q, <i>J</i> = 7.1 Hz, 2H).</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41">4.1.16.  3-(But-2-en-1-yl)-2-chloro-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one (<b>27d</b>)</h4><div class="NLM_p last"><b>27d</b> was prepared from <b>15</b> (2.0 g, 7.42 mmol) and 1-bromobut-2-ene (1.30 g, 9.65 mmol) using the procedure described for <b>16</b> in 39% yield as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.31 (s, 1H), 7.45 (t, <i>J</i> = 3.0 Hz, 1H), 6.36 (dd, <i>J</i> = 2.8, 2.0 Hz, 1H), 5.64–5.52 (m, 2H), 4.81–4.70 (m, 2H), 1.80–1.72 (m, 1H), 1.68–1.62 (m, 2H).</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42">4.1.17.  2-Chloro-3,5-dimethyl-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one (<b>28</b>)</h4><div class="NLM_p last">NaH (60% in mineral oil, 400 mg, 10.0 mmol) was added portionwise to a stirred solution of <b>14</b> (1.2 g, 7.08 mmol) in anhydrous DMF (30 mL) at 0 °C. Twenty minutes later, iodomethane (2.27 g, 16.0 mmol) was added. The mixture was reacted at 60 °C for 5 h. After cooling to rt, the reaction mixture was diluted with water and extracted with EtOAc. The combined organic phase was concentrated under reduced pressure, and the residue was purified by Biotage Isolera LPLC (PE/EA 10:1 to 2:1) to give <b>28</b> (1.09 g, 78%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.43 (d, <i>J</i> = 2.9 Hz, 1H), 6.30 (d, <i>J</i> = 2.9 Hz, 1H), 3.98 (s, 3H), 3.58 (s, 3H).</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43">4.1.18.  2-Chloro-3-methyl-3<i>H</i>,4<i>H</i>-thieno[3,2-<i>d</i>]pyrimidin-4-one (<b>31</b>)</h4><div class="NLM_p">Step 1<i>:</i> 2-chloro-3<i>H</i>,4<i>H</i>-thieno[3,2-<i>d</i>]pyrimidin-4-one <b>30</b> (1.73 g, 9.27 mmol, 95%) was prepared from <b>29</b> (2.0 g, 9.75 mmol) using the procedure described for <b>14</b> as a yellow solid.</div><div class="NLM_p last">Step 2: <b>31</b> (1.43 g, 77%) was prepared from <b>30</b> (1.73 g, 9.27 mmol) and iodomethane (1.8 g, 12.7 mmol) using the procedure described for <b>28</b> as a yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.24 (d, <i>J</i> = 5.2 Hz, 1H), 7.37 (d, <i>J</i> = 5.2 Hz, 1H), 3.64 (s, 3H).</div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44">4.1.19.  3-Ethyl-2-{[(3<i>R</i>,5<i>R</i>)-1-methyl-5-phenylpiperidin-3-yl]amino}-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one ((<i>R</i>,<i>R</i>)-<b>32a</b>)</h4><div class="NLM_p last">The title compound was prepared from <b>27a</b> (180 mg, 0.91 mmol) and (<i>R</i>,<i>R</i>)-<b>20a</b> (264 mg, 1.0 mmol) using the procedure described for compound (<i>R</i>,<i>R</i>)-<b>24a</b> in 13% yield as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.40 (s, 1H), 7.36–7.19 (m, 5H), 7.15 (t, <i>J</i> = 2.9 Hz, 1H), 6.09–6.01 (m, 2H), 4.25 (dtt, <i>J</i> = 11.5, 7.5, 4.0 Hz, 1H), 4.07 (q, <i>J</i> = 7.0 Hz, 2H), 3.09 (dd, <i>J</i> = 10.5, 4.1 Hz, 1H), 2.96–2.83 (m, 2H), 2.24 (s, 3H), 2.03 (dd, <i>J</i> = 10.0, 6.1 Hz, 1H), 1.94–1.77 (m, 2H), 1.65 (q, <i>J</i> = 12.1 Hz, 1H), 1.12 (t, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.30, 148.65, 144.82, 144.09, 128.87, 127.77, 127.48, 126.84, 112.57, 101.13, 62.53, 60.43, 48.56, 46.22, 41.75, 37.19, 34.90, 13.73. HRMS <i>m</i>/<i>z</i>: 352.2136 (M + H<sup>+</sup>, C<sub>20</sub>H<sub>25</sub>N<sub>5</sub>O, requires 351.2059).</div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45">4.1.20.  2-{[(3<i>R</i>,5<i>R</i>)-1-Methyl-5-phenylpiperidin-3-yl]amino}-3-(prop-2-en-1-yl)-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one ((<i>R</i>,<i>R</i>)-<b>32b</b>)</h4><div class="NLM_p last">The title compound was prepared from <b>27b</b> (190 mg, 0.91 mmol) and (<i>R</i>,<i>R</i>)-<b>20a</b> (264 mg, 1.0 mmol) using the procedure described for compound (<i>R</i>,<i>R</i>)-<b>24a</b> in 17% yield as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.46 (s, 1H), 7.34–7.16 (m, 6H), 6.06 (t, <i>J</i> = 2.4 Hz, 1H), 5.91–5.80 (m, 1H), 5.76 (d, <i>J</i> = 7.8 Hz, 1H), 5.12–5.05 (m, 1H), 5.01–4.93 (m, 1H), 4.71 (d, <i>J</i> = 4.8 Hz, 2H), 4.21 (dd, <i>J</i> = 7.2, 3.7 Hz, 1H), 3.06 (dd, <i>J</i> = 10.6, 4.3 Hz, 1H), 2.95–2.81 (m, 2H), 2.23 (s, 3H), 2.01 (d, <i>J</i> = 12.3 Hz, 1H), 1.89 (t, <i>J</i> = 10.8 Hz, 1H), 1.76 (t, <i>J</i> = 10.5 Hz, 1H), 1.56 (q, <i>J</i> = 12.1 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.19, 148.76, 144.99, 144.04, 133.40, 128.87, 128.01, 127.47, 126.84, 115.94, 112.31, 101.25, 62.45, 60.39, 48.62, 46.20, 46.17, 41.68, 41.52, 37.29. HRMS <i>m</i>/<i>z</i>: 364.2133 (M + H<sup>+</sup>, C<sub>21</sub>H<sub>25</sub>N<sub>5</sub>O, requires 363.2059).</div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46">4.1.21.  3-(But-3-en-1-yl)-2-{[(3<i>R</i>,5<i>R</i>)-1-methyl-5-phenylpiperidin-3-yl]amino}-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one ((<i>R</i>,<i>R</i>)-<b>32c</b>)</h4><div class="NLM_p last">The title compound was prepared from <b>27c</b> (180 mg, 0.8 mmol) and (<i>R</i>,<i>R</i>)-<b>20a</b> (233 mg, 0.88 mmol) using the procedure described for compound (<i>R</i>,<i>R</i>)-<b>24a</b> in 19% yield as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.38 (s, 1H), 7.36–7.26 (m, 4H), 7.26–7.18 (m, 1H), 7.15 (t, <i>J</i> = 2.9 Hz, 1H), 6.08–5.99 (m, 2H), 5.82 (ddt, <i>J</i> = 17.0, 10.2, 6.7 Hz, 1H), 5.05–4.93 (m, 2H), 4.24 (td, <i>J</i> = 11.4, 9.4, 5.2 Hz, 1H), 4.13 (dd, <i>J</i> = 8.4, 6.3 Hz, 2H), 3.12–3.04 (m, 1H), 2.97–2.79 (m, 2H), 2.32 (q, <i>J</i> = 7.1 Hz, 2H), 2.24 (s, 3H), 2.04 (dd, <i>J</i> = 10.0, 6.1 Hz, 1H), 1.93–1.78 (m, 2H), 1.63 (q, <i>J</i> = 12.1 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.49, 148.74, 144.76, 144.09, 135.68, 128.88, 127.81, 127.48, 126.84, 117.24, 112.52, 101.17, 62.50, 60.39, 48.63, 46.22, 41.75, 38.76, 37.23, 32.27. HRMS <i>m</i>/<i>z</i>: 378.2293 (M + H<sup>+</sup>, C<sub>22</sub>H<sub>27</sub>N<sub>5</sub>O, requires 377.2216).</div></div><div id="sec4_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47">4.1.22.  3-[(2<i>E</i>/<i>Z</i>)-But-2-en-1-yl]-2-{[(3<i>R</i>,5<i>R</i>)-1-methyl-5-phenylpiperidin-3-yl]amino}-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one ((<i>R</i>,<i>R</i>)-<b>32d</b>)</h4><div class="NLM_p last">The title compound was prepared from <b>27d</b> (180 mg, 0.8 mmol) and (<i>R</i>,<i>R</i>)-<b>20a</b> (233 mg, 0.88 mmol) using the procedure described for compound (<i>R</i>,<i>R</i>)-<b>24a</b> in 14% yield as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.42 (s, 1H), 7.36–7.19 (m, 5H), 7.16 (t, <i>J</i> = 2.9 Hz, 1H), 6.05 (t, <i>J</i> = 2.4 Hz, 1H), 5.76 (d, <i>J</i> = 7.8 Hz, 1H), 5.59–5.40 (m, 2H), 4.62 (d, <i>J</i> = 4.9 Hz, 2H), 4.24–4.12 (m, 1H), 3.06 (dd, 1H), 2.93–2.82 (m, 2H), 2.24 (s, 3H), 2.02 (dt, <i>J</i> = 12.4, 3.9 Hz, 1H), 1.95–1.87 (m, 1H), 1.76 (t, 1H), 1.65–1.50 (m, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.20, 148.77, 144.87, 144.02, 128.89, 127.94, 127.69, 127.48, 126.86, 125.86, 112.44, 101.20, 62.43, 60.40, 48.61, 46.18, 41.65, 40.82, 37.25, 17.86. HRMS <i>m</i>/<i>z</i>: 378.2295 (M + H<sup>+</sup>, C<sub>22</sub>H<sub>27</sub>N<sub>5</sub>O, requires 377.2216).</div></div><div id="sec4_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48">4.1.23.  3,5-Dimethyl-2-{[(3<i>R</i>,5<i>R</i>)-1-methyl-5-phenylpiperidin-3-yl]amino}-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one ((<i>R</i>,<i>R</i>)-<b>32e</b>)</h4><div class="NLM_p last">The title compound was prepared from <b>28</b> (150 mg, 0.76 mmol) and (<i>R</i>,<i>R</i>)-<b>20a</b> (224 mg, 0.85 mmol) using the procedure described for compound (<i>R</i>,<i>R</i>)-<b>24a</b> in 23% yield as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.36–7.19 (m, 5H), 7.15 (d, <i>J</i> = 2.8 Hz, 1H), 6.03 (d, <i>J</i> = 7.8 Hz, 1H), 5.99 (d, <i>J</i> = 2.8 Hz, 1H), 4.24–4.13 (m, 1H), 3.88 (s, 3H), 3.33 (s, 3H), 3.09 (dd, <i>J</i> = 10.4, 4.1 Hz, 1H), 2.97–2.80 (m, 2H), 2.23 (s, 3H), 2.07 (dd, <i>J</i> = 11.5, 4.2 Hz, 1H), 1.85 (dt, <i>J</i> = 27.9, 10.6 Hz, 2H), 1.62 (q, <i>J</i> = 12.1 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.90, 149.66, 145.21, 144.04, 132.37, 128.87, 127.47, 126.84, 112.17, 100.02, 62.60, 60.42, 48.62, 46.21, 41.68, 37.13, 35.56, 27.45. HRMS <i>m</i>/<i>z</i>: 352.2136 (M + H<sup>+</sup>, C<sub>20</sub>H<sub>25</sub>N<sub>5</sub>O, requires 351.2059).</div></div><div id="sec4_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49">4.1.24.  3-Methyl-2-{[(3<i>R</i>,5<i>R</i>)-1-methyl-5-phenylpiperidin-3-yl]amino}-3<i>H</i>,4<i>H</i>-thieno[3,2-<i>d</i>]pyrimidin-4-one ((<i>R</i>,<i>R</i>)-<b>32f</b>)</h4><div class="NLM_p last">The title compound was prepared from <b>31</b> (150 mg, 0.75 mmol) and (<i>R</i>,<i>R</i>)-<b>20a</b> (211 mg, 0.8 mmol) using the procedure described for compound (<i>R</i>,<i>R</i>)-<b>24a</b> in 47% yield as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.96 (d, <i>J</i> = 5.3 Hz, 1H), 7.38–7.26 (m, 4H), 7.25–7.20 (m, 1H), 7.10 (d, <i>J</i> = 5.3 Hz, 1H), 6.60 (d, <i>J</i> = 7.9 Hz, 1H), 4.30 (ddd, <i>J</i> = 18.5, 9.7, 6.3 Hz, 1H), 3.40 (s, 3H), 3.08 (dd, <i>J</i> = 10.5, 4.2 Hz, 1H), 2.98–2.81 (m, 2H), 2.24 (s, 3H), 2.07 (dt, <i>J</i> = 12.1, 3.8 Hz, 1H), 1.88 (q, <i>J</i> = 10.6 Hz, 2H), 1.69 (q, <i>J</i> = 12.1 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 158.20, 157.96, 152.92, 143.91, 134.95, 128.88, 127.46, 126.87, 124.49, 112.10, 62.48, 60.12, 48.90, 46.18, 41.63, 36.89, 28.16. HRMS <i>m</i>/<i>z</i>: 355.1560 (M + H<sup>+</sup>, C<sub>19</sub>H<sub>22</sub>N<sub>4</sub>OS, requires 354.1514).</div></div><div id="sec4_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50">4.1.25.  <i>tert</i>-Butyl (3<i>R</i>,5<i>R</i>)-3-(2,3-dihydro-1,4-benzodioxin-6-yl)-5-(pyridine-2-amido)piperidine-1-carboxylate ((<i>R</i>,<i>R</i>)-<b>33j</b>)</h4><div class="NLM_p last">A pressure vial (100 mL) equipped with a magnetic stirring bar was charged with compound (<i>R</i>)-<b>18a</b> (2.5 g, 8.19 mmol), silver carbonate (2.26 g, 8.19 mmol), Pd(OAc)<sub>2</sub> (184 mg, 0.82 mmol), 6-iodo-2,3-dihydro-1,4-benzodioxine (5.0 g, 19.1 mmol), 2,6-dimethylbenzoic acid (307 mg, 2.05 mmol), and 30 mL <i>t</i>-BuOH. The vessel was flushed with argon, sealed with a crimp cap, and heated to 120 °C. After 24 h, the reaction vessel was removed from the oil bath and cooled to room temperature, and DCM (50 mL) was added to the reaction mixture. The mixture was thoroughly stirred for 10 min, and the solids were removed by filtration, which was additionally rinsed with DCM (50 mL). The combined filtrates were concentrated under reduced pressure, and the residue was purified by Biotage Isolera LPLC to give the title compound (<i>R</i>,<i>R</i>)-<b>33j</b> (2.36 g, 5.37 mmol, 66%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.73–8.60 (m, 2H), 8.08–7.97 (m, 2H), 7.61 (ddd, <i>J</i> = 7.4, 4.7, 1.5 Hz, 1H), 6.83–6.68 (m, 3H), 4.32–4.16 (m, 4H), 4.09 (d, <i>J</i> = 12.2 Hz, 1H), 3.99–3.87 (m, 2H), 2.01–1.86 (m, 2H), 1.42 (s, 9H). Chiral HPLC (Chiralpak IE-H column, 4.6 mm × 25 cm, 30% EtOH/hexane, 1 mL/min): <i>R</i><sub>t</sub> = 28.43 min, >99% ee.</div></div><div id="sec4_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51">4.1.26.  <i>N</i>-[(3<i>R</i>,5<i>R</i>)-5-(2,3-Dihydro-1,4-benzodioxin-6-yl)-1-methylpiperidin-3-yl]pyridine-2-carboxamide ((<i>R</i>,<i>R</i>)-<b>34m</b>)</h4><div class="NLM_p">Step 1: To a stirred solution of (<i>R</i>,<i>R</i>)-<b>33j</b> (2.3 g, 5.23 mmol) in DCM was added TFA (4 mL). The resulting mixture was stirred at ambient temperature for 4 h. Then, the solvent was removed under reduced pressure, and the crude product was redissolved in water (50 mL). The solution was basified to pH 10 with 15% aq NaOH and extracted with EtOAc (80 mL × 3). The combined organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to give <i>N</i>-[(3<i>R</i>,5<i>R</i>)-5-(2,3-dihydro-1,4-benzodioxin-6-yl)piperidin-3-yl]pyridine-2-carboxamide (1.45 g, 4.27 mmol, 82%) as a pale-yellow solid, which was used without further purification. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.68–8.59 (m, 1H), 8.45 (d, <i>J</i> = 8.7 Hz, 1H), 8.08–7.92 (m, 2H), 7.60 (ddd, <i>J</i> = 7.4, 4.8, 1.5 Hz, 1H), 6.76 (d, <i>J</i> = 8.1 Hz, 1H), 6.72–6.62 (m, 2H), 4.31–4.10 (m, 4H), 4.01–3.88 (m, 1H), 3.17 (d, <i>J</i> = 3.6 Hz, 1H), 3.07–2.95 (m, 1H), 2.94–2.84 (m, 1H), 2.69–2.58 (m, 1H), 2.48–2.38 (m, 2H), 2.33 (t, <i>J</i> = 11.7 Hz, 1H), 2.01–1.91 (m, 1H), 1.77 (q, <i>J</i> = 12.1 Hz, 1H).</div><div class="NLM_p last">Step 2: To a stirred solution of <i>N</i>-[(3<i>R</i>,5<i>R</i>)-5-(2,3-dihydro-1,4-benzodioxin-6-yl)piperidin-3-yl]pyridine-2-carboxamide (1.45 g, 4.27 mmol) and glacial acetic acid (0.1 mL) in methanol (50 mL) was added 37% w/v formaldehyde (0.42 mL). Then, sodium triacetoxyborohydride (2.0 g, 9.44 mmol) was added portionwise (0.5 g every 20 min). Two hours after the final addition, the solvent was removed in vacuo, and the residue was partitioned between EtOAc and water. The biphasic solution was extracted twice with EtOAc. Then, the combined organics were washed twice with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. Purification of the residue by Biotage Isolera LPLC gave (<i>R</i>,<i>R</i>)-<b>34m</b> (1.1 g, 3.11 mmol, 73%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.64 (dt, <i>J</i> = 4.9, 1.3 Hz, 1H), 8.53 (d, <i>J</i> = 8.7 Hz, 1H), 8.07–7.93 (m, 2H), 7.60 (ddd, <i>J</i> = 7.6, 4.7, 1.5 Hz, 1H), 6.81–6.66 (m, 3H), 4.28–4.17 (m, 4H), 4.15–4.02 (m, 1H), 2.91 (dd, <i>J</i> = 10.1, 4.1 Hz, 1H), 2.83–2.72 (m, 2H), 2.21 (s, 3H), 2.01–1.76 (m, 3H), 1.67 (q, <i>J</i> = 12.0 Hz, 1H).</div></div><div id="sec4_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52">4.1.27.  (3<i>R</i>,5<i>R</i>)-5-(2,3-Dihydro-1,4-benzodioxin-6-yl)-1-methylpiperidin-3-amine ((<i>R</i>,<i>R</i>)-<b>35m</b>)</h4><div class="NLM_p last">Compound (<i>R</i>,<i>R</i>)-<b>34m</b> (1.1 g, 3.11 mmol) was suspended in <i>i</i>-PrOH (40 mL) in a 100 mL round-bottom flask. NaOH (1.24 g, 31.1 mmol) was added. The reaction mixture was stirred at 85 °C for 18 h. Then, the solvent was removed under reduced pressure and water (50 mL) was added. The solution was extracted with EtOAc (50 mL × 2), dried over Na<sub>2</sub>SO<sub>4</sub>, and filtered, and the solvent was removed to give the title compound (<i>R</i>,<i>R</i>)-<b>35m</b> (593 mg, 2.39 mmol, 77%) as a pale-yellow solid without further purification. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 6.75 (d, <i>J</i> = 8.1 Hz, 1H), 6.71–6.60 (m, 2H), 4.28–4.13 (m, 4H), 2.84–2.59 (m, 4H), 2.15 (s, 3H), 1.89–1.78 (m, 1H), 1.67 (t, <i>J</i> = 10.7 Hz, 1H), 1.59–1.28 (m, 3H), 1.04 (q, <i>J</i> = 12.0 Hz, 1H).</div></div><div id="sec4_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53">4.1.28.  3-Methyl-2-{[(3<i>R</i>,5<i>R</i>)-5-phenyl-1-(prop-2-en-1-yl)piperidin-3-yl]amino}-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one ((<i>R</i>,<i>R</i>)-<b>36a</b>)</h4><div class="NLM_p">Step 1: To a solution of <i>N</i>-[(3<i>R</i>,5<i>R</i>)-5-phenylpiperidin-3-yl]pyridine-2-carboxamide (800 mg, 2.84 mmol) and Et<sub>3</sub>N (324 mg, 3.2 mmol) in MeCN was added 3-bromoprop-1-ene (387 mg, 3.2 mmol). The mixture was reacted at 70 °C for 8 h. After cooling to ambient temperature, the mixture was concentrated and the residue was purified by LPLC to give (<i>R</i>,<i>R</i>)-<b>34a</b> (690 mg, 2.15 mmol, 76%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.67–8.61 (m, 1H), 8.57 (d, <i>J</i> = 8.7 Hz, 1H), 8.08–7.95 (m, 2H), 7.60 (ddd, <i>J</i> = 7.3, 4.7, 1.5 Hz, 1H), 7.37–7.24 (m, 4H), 7.24–7.16 (m, 1H), 5.93–5.78 (m, 1H), 5.22–5.08 (m, 2H), 4.17–4.06 (m, 1H), 3.12–2.96 (m, 3H), 2.96–2.83 (m, 2H), 2.07–1.89 (m, 3H), 1.80 (q, <i>J</i> = 12.0 Hz, 1H).</div><div class="NLM_p">Step 2: (3<i>R</i>,5<i>R</i>)-1-methyl-5-phenylpiperidin-3-amine ((<i>R</i>,<i>R</i>)-<b>35a</b>) was prepared from (<i>R</i>,<i>R</i>)-<b>34a</b> using the procedure described for (<i>R</i>,<i>R</i>)-<b>20a</b> as colorless oil. The product was used immediately without further purification. LC-MS: <i>m</i>/<i>z</i> 191.1 (M + H)<sup>+</sup>.</div><div class="NLM_p last">Step 3: The title compound (<i>R</i>,<i>R</i>)-<b>36a</b> was prepared from <b>16</b> and (<i>R</i>,<i>R</i>)-<b>35a</b> using the procedure described for (<i>R</i>,<i>R</i>)-<b>24a</b> as a white solid. This product was further purified by reverse-phase HPLC. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.44 (s, 1H), 7.35–7.17 (m, 6H), 6.18–5.97 (m, 2H), 5.89–5.78 (m, 1H), 5.28–5.04 (m, 2H), 4.32–4.14 (m, 1H), 3.37 (s, 3H), 3.18 (dd, <i>J</i> = 10.5, 4.3 Hz, 1H), 3.05–2.84 (m, 4H), 2.11 (dd, <i>J</i> = 12.0, 4.1 Hz, 1H), 1.97–1.87 (m, 2H), 1.68 (q, <i>J</i> = 12.7 Hz, 1H).</div></div><div id="sec4_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54">4.1.29.  2-{[(3<i>R</i>,5<i>R</i>)-1-(2-Hydroxyethyl)-5-phenylpiperidin-3-yl]amino}-3-methyl-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one ((<i>R</i>,<i>R</i>)-<b>36b</b>)</h4><div class="NLM_p last">The title compound (<i>R</i>,<i>R</i>)-<b>36b</b> was prepared in the same manner as shown for (<i>R</i>,<i>R</i>)-<b>36a</b> except that 2-bromoethan-1-ol was used instead. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.41 (s, 1H), 7.35–7.19 (m, 5H), 7.16 (t, <i>J</i> = 2.9 Hz, 1H), 6.06 (t, <i>J</i> = 2.4 Hz, 1H), 6.00 (d, <i>J</i> = 7.8 Hz, 1H), 4.42 (t, <i>J</i> = 5.3 Hz, 1H), 4.25–4.14 (m, 1H), 3.57–3.48 (m, 2H), 3.36 (s, 3H), 3.18 (dd, <i>J</i> = 10.7, 4.1 Hz, 1H), 2.97 (dd, <i>J</i> = 10.9, 3.7 Hz, 1H), 2.94–2.83 (m, 1H), 2.50–2.41 (m, 2H), 2.12–1.93 (m, 3H), 1.65 (q, <i>J</i> = 12.2 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.48, 149.58, 144.82, 144.19, 128.87, 127.78, 127.51, 126.82, 112.38, 101.18, 60.82, 60.61, 59.12, 58.96, 48.67, 41.74, 37.74, 27.70. HRMS <i>m</i>/<i>z</i>: 368.2083 (M + H<sup>+</sup>, C<sub>20</sub>H<sub>25</sub>N<sub>5</sub>O<sub>2</sub>, requires 367.2008).</div></div><div id="sec4_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55">4.1.30.  3-Methyl-2-{[(3<i>R</i>,5<i>R</i>)-5-phenyl-1-(2-phenylethyl)piperidin-3-yl]amino}-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one ((<i>R</i>,<i>R</i>)-<b>36c</b>)</h4><div class="NLM_p last">The title compound (<i>R</i>,<i>R</i>)-<b>36b</b> was prepared in the same manner as shown for (<i>R</i>,<i>R</i>)-<b>36a</b> except that (2-bromoethyl)benzene was used instead. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.43 (s, 1H), 7.36–7.14 (m, 11H), 6.07 (t, <i>J</i> = 2.4 Hz, 1H), 6.03 (d, <i>J</i> = 7.7 Hz, 1H), 4.30–4.12 (m, 1H), 3.37 (s, 3H), 3.32–3.26 (m, 1H), 3.01 (dd, <i>J</i> = 10.8, 3.6 Hz, 1H), 2.91 (td, <i>J</i> = 12.1, 11.5, 3.7 Hz, 1H), 2.82–2.69 (m, 2H), 2.62 (t, <i>J</i> = 7.8 Hz, 2H), 2.22–1.88 (m, 3H), 1.69 (q, <i>J</i> = 12.1 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.49, 149.59, 144.85, 144.15, 140.91, 129.20, 128.87, 128.66, 127.81, 127.54, 126.83, 126.25, 112.40, 101.20, 60.42, 60.11, 58.11, 48.75, 41.73, 37.73, 33.10, 27.70. HRMS <i>m</i>/<i>z</i>: 428.2452 (M + H<sup>+</sup>, C<sub>26</sub>H<sub>29</sub>N<sub>5</sub>O, requires 427.2372).</div></div><div id="sec4_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56">4.1.31.  2-{[(3<i>R</i>,5<i>R</i>)-5-(3-bromophenyl)-1-methylpiperidin-3-yl]amino}-3-methyl-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one ((<i>R</i>,<i>R</i>)-<b>36d</b>)</h4><div class="NLM_p last">The title compound (<i>R</i>,<i>R</i>)-<b>36d</b> was prepared in the same manner as shown for (<i>R</i>,<i>R</i>)-<b>24a</b> except that 1-bromo-3-iodobenzene was used instead. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.43 (s, 1H), 7.47 (t, <i>J</i> = 1.4 Hz, 1H), 7.45–7.40 (m, 1H), 7.32–7.26 (m, 2H), 7.16 (t, <i>J</i> = 2.9 Hz, 1H), 6.05 (t, <i>J</i> = 2.4 Hz, 1H), 5.99 (d, <i>J</i> = 7.7 Hz, 1H), 4.25–4.12 (m, 1H), 3.36 (s, 3H), 3.10 (dd, <i>J</i> = 10.5, 4.2 Hz, 1H), 3.01–2.83 (m, 2H), 2.23 (s, 3H), 2.13–2.03 (m, 1H), 1.91–1.76 (m, 2H), 1.61 (q, <i>J</i> = 12.2 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.46, 149.53, 146.90, 144.79, 131.07, 130.28, 129.75, 127.78, 126.72, 122.22, 112.41, 101.18, 62.22, 60.35, 48.51, 46.15, 41.22, 36.85, 27.69. HRMS <i>m</i>/<i>z</i>: 416.1082 (M + H<sup>+</sup>, C<sub>19</sub>H<sub>22</sub>BrN<sub>5</sub>O, requires 415.1008).</div></div><div id="sec4_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57">4.1.32.  2-{[(3<i>R</i>,5<i>R</i>)-5-(4-methoxyphenyl)-1-methylpiperidin-3-yl]amino}-3-methyl-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one ((<i>R</i>,<i>R</i>)-<b>36e</b>)</h4><div class="NLM_p last">The title compound (<i>R</i>,<i>R</i>)-<b>36e</b> was prepared in the same manner as shown for (<i>R</i>,<i>R</i>)-<b>24a</b> except that 1-iodo-4-methoxybenzene was used instead. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.43 (s, 1H), 7.23–7.12 (m, 3H), 6.92–6.83 (m, 2H), 6.06 (t, <i>J</i> = 2.4 Hz, 1H), 6.00 (d, <i>J</i> = 7.7 Hz, 1H), 4.26–4.12 (m, 1H), 3.73 (s, 3H), 3.10 (dd, <i>J</i> = 10.5, 4.1 Hz, 1H), 2.89–2.77 (m, 2H), 2.23 (s, 3H), 2.05 (d, <i>J</i> = 11.9 Hz, 1H), 1.82 (dt, <i>J</i> = 12.6, 10.6 Hz, 2H), 1.58 (q, <i>J</i> = 12.2 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 158.23, 154.47, 149.58, 144.81, 136.02, 128.36, 127.78, 114.25, 112.40, 101.18, 62.93, 60.42, 55.46, 48.69, 46.20, 40.81, 37.35, 27.71. HRMS <i>m</i>/<i>z</i>: 368.2089 (M + H<sup>+</sup>, C<sub>20</sub>H<sub>25</sub>N<sub>5</sub>O<sub>2</sub>, requires 367.2008).</div></div><div id="sec4_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58">4.1.33.  2-{[(3<i>R</i>,5<i>R</i>)-5-(4-Butylphenyl)-1-methylpiperidin-3-yl]amino}-3-methyl-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one ((<i>R</i>,<i>R</i>)-<b>36f</b>)</h4><div class="NLM_p last">The title compound (<i>R</i>,<i>R</i>)-<b>36f</b> was prepared in the same manner as shown for (<i>R</i>,<i>R</i>)-<b>24a</b> except that 1-butyl-4-iodobenzene was used instead. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.44 (s, 1H), 7.28–7.05 (m, 5H), 6.07 (t, <i>J</i> = 2.5 Hz, 1H), 6.00 (d, <i>J</i> = 7.8 Hz, 1H), 4.29–4.11 (m, 1H), 3.37 (s, 3H), 3.11 (dd, <i>J</i> = 10.5, 4.2 Hz, 1H), 2.93–2.77 (m, 2H), 2.57–2.50 (m, 2H), 2.23 (s, 3H), 2.07 (d, <i>J</i> = 12.1 Hz, 1H), 1.91–1.75 (m, 2H), 1.67–1.48 (m, 3H), 1.38–1.20 (m, 2H), 0.88 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.49, 149.57, 144.83, 141.23, 140.71, 128.72, 127.76, 127.28, 112.42, 101.18, 62.77, 60.47, 48.70, 46.22, 41.30, 37.19, 34.88, 33.63, 27.70, 22.22, 14.23. HRMS <i>m</i>/<i>z</i>: 394.2610 (M + H<sup>+</sup>, C<sub>23</sub>H<sub>31</sub>N<sub>5</sub>O, requires 393.2529).</div></div><div id="sec4_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59">4.1.34.  3-Methyl-2-{[(3<i>R</i>,5<i>R</i>)-1-methyl-5-[4-(trifluoromethoxy)phenyl]piperidin-3-yl]amino}-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one ((<i>R</i>,<i>R</i>)-<b>36g</b>)</h4><div class="NLM_p last">The title compound (<i>R</i>,<i>R</i>)-<b>36g</b> was prepared in the same manner as shown for (<i>R</i>,<i>R</i>)-<b>24a</b> except that 1-iodo-4-(trifluoromethoxy)benzene was used instead. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.44 (s, 1H), 7.46–7.37 (m, 2H), 7.31 (d, <i>J</i> = 8.2 Hz, 2H), 7.16 (t, <i>J</i> = 2.9 Hz, 1H), 6.06 (t, <i>J</i> = 2.4 Hz, 1H), 6.01 (d, <i>J</i> = 7.7 Hz, 1H), 4.29–4.13 (m, 1H), 3.36 (s, 3H), 3.11 (dd, <i>J</i> = 10.5, 4.1 Hz, 1H), 3.04–2.94 (m, 1H), 2.87 (dd, <i>J</i> = 11.0, 3.6 Hz, 1H), 2.24 (s, 3H), 2.10 (dt, <i>J</i> = 12.9, 3.8 Hz, 1H), 1.92–1.77 (m, 2H), 1.60 (q, <i>J</i> = 12.2 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.46, 149.55, 147.28, 144.79, 143.50, 129.28, 127.77, 121.85, 121.44, 119.30, 112.42, 101.18, 62.32, 60.36, 48.58, 46.17, 40.99, 37.10, 27.70. HRMS <i>m</i>/<i>z</i>: 422.1832 (M + H<sup>+</sup>, C<sub>20</sub>H<sub>22</sub>F<sub>3</sub>N<sub>5</sub>O<sub>2</sub>, requires 421.1726).</div></div><div id="sec4_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60">4.1.35.  2-{[(3<i>R</i>,5<i>R</i>)-5-[4-chloro-3-(trifluoromethyl)phenyl]-1-methylpiperidin-3-yl]amino}-3-methyl-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one ((<i>R</i>,<i>R</i>)-<b>36h</b>)</h4><div class="NLM_p last">The title compound (<i>R</i>,<i>R</i>)-<b>36h</b> was prepared in the same manner as shown for (<i>R</i>,<i>R</i>)-<b>24a</b> except that 1-chloro-4-iodo-2-(trifluoromethyl)benzene was used instead. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.45 (s, 1H), 7.76–7.60 (m, 3H), 7.17 (t, <i>J</i> = 2.9 Hz, 1H), 6.06 (t, <i>J</i> = 2.4 Hz, 1H), 6.02 (d, <i>J</i> = 7.7 Hz, 1H), 4.29–4.17 (m, 1H), 3.38 (s, 3H), 3.16–3.04 (m, 2H), 2.90 (dd, <i>J</i> = 10.9, 3.9 Hz, 1H), 2.25 (s, 3H), 2.12 (dt, <i>J</i> = 12.4, 4.0 Hz, 1H), 1.93–1.82 (m, 2H), 1.63 (q, <i>J</i> = 12.2 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.46, 149.52, 144.77, 144.08, 133.38, 132.14, 129.01, 127.78, 126.87, 126.76, 126.71, 112.43, 101.18, 61.88, 60.24, 48.42, 46.08, 40.74, 36.66, 27.69. HRMS <i>m</i>/<i>z</i>: 440.1476 (M + H<sup>+</sup>, C<sub>20</sub>H<sub>21</sub>ClF<sub>3</sub>N<sub>5</sub>O, requires 439.1387).</div></div><div id="sec4_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61">4.1.36.  2-{[(3<i>R</i>,5<i>R</i>)-5-[4-(Benzyloxy)phenyl]-1-methylpiperidin-3-yl]amino}-3-methyl-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one ((<i>R</i>,<i>R</i>)-<b>36i</b>)</h4><div class="NLM_p last">The title compound (<i>R</i>,<i>R</i>)-<b>36i</b> was prepared in the same manner as shown for (<i>R</i>,<i>R</i>)-<b>24a</b> except that 1-(benzyloxy)-4-iodobenzene was used instead. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.43 (s, 1H), 7.45–7.37 (m, 4H), 7.35–7.30 (m, 1H), 7.21–7.14 (m, 3H), 7.00–6.94 (m, 2H), 6.06 (t, <i>J</i> = 2.4 Hz, 1H), 5.98 (d, <i>J</i> = 7.7 Hz, 1H), 5.07 (s, 2H), 4.25–4.12 (m, 1H), 3.36 (s, 3H), 3.13–3.05 (m, 1H), 2.87–2.76 (m, 2H), 2.22 (s, 3H), 2.09–2.01 (m, 1H), 1.86–1.74 (m, 2H), 1.57 (q, <i>J</i> = 12.3 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 157.33, 154.48, 149.58, 144.82, 137.70, 136.32, 128.86, 128.39, 128.20, 128.05, 127.76, 115.17, 112.41, 101.19, 69.62, 62.92, 60.46, 48.71, 46.22, 40.84, 37.37, 27.71. HRMS <i>m</i>/<i>z</i>: 444.2394 (M + H<sup>+</sup>, C<sub>26</sub>H<sub>29</sub>N<sub>5</sub>O<sub>2</sub>, requires 443.2321).</div></div><div id="sec4_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62">4.1.37.  2-{[(3<i>R</i>,5<i>R</i>)-5-(4-hydroxyphenyl)-1-methylpiperidin-3-yl]amino}-3-methyl-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one ((<i>R</i>,<i>R</i>)-<b>36j</b>)</h4><div class="NLM_p last">A stirred solution of (<i>R</i>,<i>R</i>)-<b>36i</b> (660 mg, 1.49 mmol) and 10% Pd/C in MeOH was carefully evacuated and backfilled with a H<sub>2</sub> atmosphere and finally allowed to stir at ambient temperature overnight with a H<sub>2</sub> balloon attached. Upon reaction completion, the suspension was filtered through Celite with MeOH washings. The filtrate was then concentrated, and the residue was purified by Biotage Isolera LPLC (DCM/MeOH/Et<sub>3</sub>N 60:1:0.2 to 15:1:0.05) to give (<i>R</i>,<i>R</i>)-<b>36j</b> (340 mg, 65%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.42 (s, 1H), 9.21 (s, 1H), 7.15 (t, <i>J</i> = 2.9 Hz, 1H), 7.10–7.03 (m, 2H), 6.74–6.69 (m, 2H), 6.05 (t, <i>J</i> = 2.4 Hz, 1H), 5.99 (d, <i>J</i> = 7.7 Hz, 1H), 4.24–4.11 (m, 1H), 3.36 (s, 3H), 3.14–3.06 (m, 1H), 2.85–2.77 (m, 2H), 2.23 (s, 3H), 2.03 (d, <i>J</i> = 12.4 Hz, 1H), 1.87–1.77 (m, 2H), 1.55 (q, <i>J</i> = 12.3 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 156.24, 154.48, 149.58, 144.81, 134.13, 128.25, 127.78, 115.60, 112.41, 101.18, 62.87, 60.25, 48.64, 46.09, 40.72, 37.38, 27.73. HRMS <i>m</i>/<i>z</i>: 354.1927 (M + H<sup>+</sup>, C<sub>19</sub>H<sub>23</sub>N<sub>5</sub>O<sub>2</sub>, requires 353.1852).</div></div><div id="sec4_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63">4.1.38.  3-Methyl-2-{[(3<i>R</i>,5<i>R</i>)-1-methyl-5-(3-phenylphenyl)piperidin-3-yl]amino}-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one ((<i>R</i>,<i>R</i>)-<b>36k</b>)</h4><div class="NLM_p last">The title compound (<i>R</i>,<i>R</i>)-<b>36k</b> was prepared in the same manner as shown for (<i>R</i>,<i>R</i>)-<b>24a</b> except that 1-iodo-3-phenylbenzene was used instead. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.45 (s, 1H), 7.70–7.62 (m, 2H), 7.57–7.34 (m, 6H), 7.31–7.25 (m, 1H), 7.17 (t, <i>J</i> = 2.9 Hz, 1H), 6.08 (t, <i>J</i> = 2.4 Hz, 1H), 6.01 (d, <i>J</i> = 7.8 Hz, 1H), 4.33–4.18 (m, 1H), 3.38 (s, 3H), 3.14 (dd, <i>J</i> = 10.4, 4.1 Hz, 1H), 3.07–2.87 (m, 2H), 2.25 (s, 3H), 2.18–2.11 (m, 1H), 1.96 (t, <i>J</i> = 10.5 Hz, 1H), 1.85 (t, <i>J</i> = 10.5 Hz, 1H), 1.71 (q, <i>J</i> = 12.1 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.50, 149.59, 144.85, 144.73, 140.82, 140.80, 129.48, 129.34, 127.85, 127.80, 127.20, 126.62, 126.03, 125.29, 112.43, 101.21, 62.66, 60.47, 48.68, 46.20, 41.72, 37.02, 27.71. HRMS <i>m</i>/<i>z</i>: 414.2290 (M + H<sup>+</sup>, C<sub>25</sub>H<sub>27</sub>N<sub>5</sub>O, requires 413.2216).</div></div><div id="sec4_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64">4.1.39.  3-Methyl-2-{[(3<i>R</i>,5<i>R</i>)-1-methyl-5-(4-phenoxyphenyl)piperidin-3-yl]amino}-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one ((<i>R</i>,<i>R</i>)-<b>36l</b>)</h4><div class="NLM_p last">The title compound (<i>R</i>,<i>R</i>)-<b>36l</b> was prepared in the same manner as shown for (<i>R</i>,<i>R</i>)-<b>24a</b> except that 1-iodo-4-phenoxybenzene was used instead. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.44 (s, 1H), 7.45–7.32 (m, 2H), 7.34–7.24 (m, 2H), 7.20–7.08 (m, 2H), 7.03–6.89 (m, 4H), 6.06 (t, <i>J</i> = 2.4 Hz, 1H), 6.00 (d, <i>J</i> = 8.5 Hz, 1H), 4.28–4.13 (m, 1H), 3.15–3.07 (m, 1H), 2.98–2.81 (m, 2H), 2.24 (s, 3H), 2.14–2.05 (m, 1H), 1.93–1.72 (m, 3H), 1.60 (q, <i>J</i> = 12.1 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 157.35, 155.48, 154.48, 149.57, 144.81, 139.19, 130.46, 128.96, 127.77, 123.72, 119.15, 118.89, 112.42, 101.19, 62.68, 60.38, 48.66, 46.19, 40.94, 37.27, 27.71. HRMS <i>m</i>/<i>z</i>: 430.2237 (M + H<sup>+</sup>, C<sub>25</sub>H<sub>27</sub>N<sub>5</sub>O<sub>2</sub>, requires 429.2165).</div></div><div id="sec4_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65">4.1.40.  3-Methyl-2-{[(3<i>R</i>,5<i>R</i>)-1-methyl-5-(naphthalen-2-yl)piperidin-3-yl]amino}-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one ((<i>R</i>,<i>R</i>)-<b>36m</b>)</h4><div class="NLM_p last">The title compound (<i>R</i>,<i>R</i>)-<b>36m</b> was prepared in the same manner as shown for (<i>R</i>,<i>R</i>)-<b>24a</b> except that 2-iodonaphthalene was used instead. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.41 (s, 1H), 7.87 (d, <i>J</i> = 7.8 Hz, 3H), 7.77 (s, 1H), 7.57–7.40 (m, 3H), 7.17 (t, <i>J</i> = 2.9 Hz, 1H), 6.12–6.00 (m, 2H), 4.31–4.19 (m, 1H), 3.37 (s, 3H), 3.19–3.03 (m, 2H), 3.00–2.92 (m, 1H), 2.27 (s, 3H), 2.22–2.12 (m, 1H), 2.00 (t, <i>J</i> = 11.1 Hz, 1H), 1.88 (t, <i>J</i> = 10.5 Hz, 1H), 1.73 (q, <i>J</i> = 12.1 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.50, 149.61, 144.83, 141.63, 133.61, 132.36, 128.27, 127.98, 127.88, 126.61, 126.51, 125.92, 125.30, 112.38, 101.20, 62.44, 60.43, 48.72, 46.23, 41.66, 37.07, 27.73. HRMS <i>m</i>/<i>z</i>: 388.2136 (M + H<sup>+</sup>, C<sub>23</sub>H<sub>25</sub>N<sub>5</sub>O, requires 387.2059).</div></div><div id="sec4_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66">4.1.41.  2-{[(3<i>R</i>,5<i>R</i>)-5-(2,3-dihydro-1,4-benzodioxin-6-yl)-1-methylpiperidin-3-yl]amino}-3-methyl-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one ((<i>R</i>,<i>R</i>)-<b>36n</b>)</h4><div class="NLM_p last">A solution of DIPEA (183 mg, 1.42 mmol) in NMP (5 mL) was treated with <b>16</b> (200 mg, 1.09 mmol) and (<i>R</i>,<i>R</i>)-<b>35m</b> (325 mg, 1.31 mmol). The resulting mixture was heated to 150 °C and stirred at this temperature for 2 h. After cooling to ambient temperature, the mixture was diluted with water and extracted with EtOAc. The combined organic phase was washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuo. Purification of the residue by Biotage Isolera LPLC gave the title compound (<i>R</i>,<i>R</i>)-<b>36n</b> (53 mg, 134 mmol, 12%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.42 (s, 1H), 7.15 (t, <i>J</i> = 2.9 Hz, 1H), 6.82–6.69 (m, 3H), 6.05 (t, <i>J</i> = 2.4 Hz, 1H), 5.98 (d, <i>J</i> = 7.8 Hz, 1H), 4.27–4.13 (m, 5H), 3.35 (s, 3H), 3.09 (dd, <i>J</i> = 10.7, 4.3 Hz, 1H), 2.90–2.76 (m, 2H), 2.21 (s, 3H), 2.02 (d, <i>J</i> = 10.2, 6.4 Hz, 1H), 1.89–1.77 (m, 2H), 1.55 (q, <i>J</i> = 12.2 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.46, 149.55, 144.79, 143.63, 142.31, 137.10, 127.77, 120.08, 117.31, 115.95, 112.40, 101.17, 64.54, 64.44, 62.73, 60.33, 48.55, 46.12, 40.82, 37.17, 27.70. HRMS <i>m</i>/<i>z</i>: 396.2035 (M + H<sup>+</sup>, C<sub>21</sub>H<sub>25</sub>N<sub>5</sub>O<sub>3</sub>, requires 395.1957). Chiral HPLC (Chiralpak IE-H column, 4.6 mm × 25 cm, <i>i</i>-PrOH/Hexane/EtN<sub>3</sub> = 30:70:0.14, 1 mL/min): <i>R</i><sub>t</sub> = 31.54 min, >99% ee.</div></div><div id="sec4_1_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67">4.1.42.  3-Methyl-2-{[(3<i>S</i>,5<i>S</i>)-1-methyl-5-phenylpiperidin-3-yl]amino}-3<i>H</i>,4<i>H</i>,5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-4-one ((<i>S</i>,<i>S</i>)-<b>36n</b>)</h4><div class="NLM_p last">The title compound (<i>S</i>,<i>S</i>)-<b>36n</b> was prepared as a white solid by the same procedures as (<i>R</i>,<i>R</i>)-<b>36n</b> except the starting material <i>tert</i>-butyl (3<i>S</i>)-3-(pyridine-2-amido)piperidine-1-carboxylate (<i>S</i>)-<b>18a</b> was used instead. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.43 (d, <i>J</i> = 3.0 Hz, 1H), 6.84–6.66 (m, 3H), 6.41 (d, <i>J</i> = 8.1 Hz, 1H), 6.26 (d, <i>J</i> = 3.0 Hz, 1H), 4.46–4.30 (m, 1H), 4.21 (s, 4H), 4.01 (s, 3H), 3.31 (s, 3H), 3.11–2.96 (m, 1H), 2.91–2.74 (m, 2H), 2.24 (s, 3H), 2.04–1.79 (m, 3H), 1.69 (q, <i>J</i> = 12.0 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 151.12, 150.92, 150.33, 143.64, 142.34, 137.01, 135.21, 120.12, 117.30, 115.98, 113.83, 100.49, 64.55, 64.45, 62.48, 59.70, 47.58, 46.14, 40.69, 36.85, 36.73. HRMS <i>m</i>/<i>z</i>: 396.2040 (M + H<sup>+</sup>, C<sub>21</sub>H<sub>25</sub>N<sub>5</sub>O<sub>3</sub>, requires 395.1957). Chiral HPLC (Chiralpak IE-H column, 4.6 mm × 25 cm, <i>i</i>-PrOH/hexane/EtN<sub>3</sub> = 30:70:0.14, 1 mL/min): <i>R</i><sub>t</sub> = 21.65 min, >99% ee.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68">4.2.  Protein Expression and Purification</h3><div class="NLM_p">For the ITC, thermal-shift assay (TSA), and crystallogenesis, human PCAF, GCN5, BRD4(1), ATAD2, BAZ2B, BRD9, BRPF1, BRPF3, EP300, FALZ, TAF1(1), and TRIM24 BRDs were expressed and purified according to the protocols previously described.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> All proteins were expressed as N-terminal His-tagged fusions and purified using Ni-chelating affinity chromatography. Then, these proteins were purified by gel filtration using Superdex200 column, eluted with buffer containing 10 mM HEPES pH 7.5 and 500 mM NaCl, concentrated to 3 mg/mL, and frozen at −80 °C for thermal shift assays. PCAF and GCN5 BRDs were further treated with TEV protease at 4 °C overnight to cleave N-terminal His-tag. The two proteins were then purified by gel filtration chromatography (Superdex 200; GE-Healthcare), concentrated to 12 mg/mL, flash-frozen in liquid nitrogen, and stored at −80 °C for ITC assays and crystallogenesis.</div><div class="NLM_p last">For the HTRF experiments, GST-tagged PCAF was cloned into pET28a vector and overexpressed in <i>Escherichia coli</i> Rosetta (DE3). Purification was carried on GST-affinity resin (Thermo Scientific), and reduced glutathione was used for protein release. GST-PCAF was further purified by gel filtration chromatography on a Superdex 200 column (GE Healthcare) using a buffer of 25 mM HEPES (pH 7.5) and 300 mM NaCl.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69">4.3.  Thermal-Shift Assay</h3><div class="NLM_p last">The TSA experiments were performed at least in duplicate. Proteins were prepared in 10 mM HEPES pH 7.5, 500 mM NaCl, and assayed in low-profile PCR tubes (Bio-Rad, TLS0851) at a final concentration of 2 μM in 20 μL volume. Compounds were added at a final concentration of 10 μM (final concentration 0.1% DMSO). SYPRO Orange dye (Thermo Fisher Scientific) was added at a dilution of 1:1000. The thermal melting experiments were carried using a Bio-Rad CFX96 RT-PCR system. The tubes were first equilibrated at 25 °C for 3 min, and then the tubes were heated from 25 to 95 °C with a step of 1 °C/min. Raw fluorescence data were recorded using CFX Maestro, and melting temperature shifts were calculated as previously described.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70">4.4.  Isothermal Titration Calorimetry</h3><div class="NLM_p last">ITC was used to evaluate the thermodynamics parameters of the binding between PCAF BRD and all the final compounds. Titrations were carried out on a MicroCal ITC200 microcalorimeter (Malvern Instruments). All experiments were carried out at 25 °C in 25 mM HEPES (pH 7.5), 300 mM NaCl, 0.5 mM TCEP. Compounds were diluted directly in the same batch buffer prior to experiments. Each experiment was designed as reverse titrations experiments (protein in the syringe and ligand in the cell) using an initial injection of 0.4 μL followed by 19 injections of 2 μL. The first injection (generally 0.4 μL) was included in the titration protocol in order to remove air bubbles trapped in the syringe prior to the titration. Background dilution heat was subtracted from each experiment. Thermodynamic parameters were calculated using Δ<i>G</i> = Δ<i>H</i> – <i>T</i>Δ<i>S</i> = −<i>RT</i> ln <i>K</i><sub>A</sub>, where <i>K</i><sub>D</sub> = 1/<i>K</i><sub>A</sub>. Δ<i>G</i>, ΔH, and Δ<i>S</i> are changes in free energy, enthalpy and entropy, respectively. Independent single site binding models were employed in data analysis.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71">4.5.  Homogeneous Time-Resolved Fluorescence</h3><div class="NLM_p last">HTRF assay was carried out using a Cisbio EPIgeneous Binding Domain Kit B (62BDBPEG) using the standard assay protocol with GST-PCAF BRD and biotinylated substrate <b>46</b>. The protein GST-PCAF BRD was produced and purified in house. The biotinylated substrate <b>46</b> was a kind gift from Prof. Dr. P. E. Brennan (Oxford University). The IC<sub>50</sub> measurements were performed in duplicate. 4 μL GST-PCAF BRD (100 nM), 2 μL tested compounds, 4 μL compound <b>46</b> (500 nM), 5 μL SA-XL665 (250 nM), and 5 μL 1× GST Ab-Eu<sup>3+</sup> were added into a 96-well low-volume plate (Cisbio 66PL96025), and then the plate was sealed and incubated at room temperature for 15 h. Readings were recorded using a Cytation Hybrid Multi-Mode Reader with an excitation filter at 337 nm and fluorescence measurement at 620 and 665 nm (60 μs integration delay and 400 μs integration time). The IC<sub>50</sub> values were normalized and fitted with Prism 7.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72">4.6.  BROMOscan</h3><div class="NLM_p last">BRD profiling was provided by DiscoveRx Corp. using their BROMOscan platform (<a href="http://www.discoverx.com/" class="extLink">http://www.discoverx.com/</a>).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> (<i>R</i>,<i>R</i>)-<b>36n</b> was profiled at 1 μM against 32 recombinant BRDs.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73">4.7.  Eurofins KinaseProfiler</h3><div class="NLM_p last">Kinase profiling was performed by Eurofins (Dundee, U. K., <a href="https://www.eurofins.com/" class="extLink">https://www.eurofins.com/</a>). (<i>R</i>,<i>R</i>)-<b>24a</b>, <i>endo</i>-<b>24c</b>, and (<i>R</i>,<i>R</i>)-<b>36n</b> were profiled at a single concentration of 10 μM against a panel of 422 kinases.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74">4.8.  Crystallization and Structure Determination</h3><div class="NLM_p">PCAF and GCN5 BRDs were crystallized by vapor diffusion in sitting drops. For PCAF, crystals grew in a mixture of 1 μL protein solution at 16 mg/ml (25 mM HEPES, pH 7.5, 300 mM NaCl, 0.5 mM TCEP) and 1 μL reservoir solution (100 mM HEPES pH 8.2, 26% PEG10000 (v/v), 4% glycol) at 4 °C. Then, overnight soaking of compound <b>12</b> was performed. For GCN5, crystals with compound (<i>R</i>,<i>R</i>)-<b>36n</b> were grown by mixing 1 μL of the protein (12 mg/mL and 4 mM final ligand concentration) with an equal volume of reservoir solution containing 0.2 M ammonium acetate, 0.1 M Tris pH 8.5, 25% w/v polyethylene glycol 3350 at 20 °C. Diffraction quality crystals grew within 5–7 days. All crystals were cryoprotected with reservoir solution supplemented with 20% glycerol (v/v) before being plunge-frozen in liquid nitrogen.</div><div class="NLM_p last">Diffraction data were collected at the Shanghai Synchrotron Light Source (Shanghai, China). Data were indexed, integrated, and scaled using HKL2000<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> or XDS.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Structures were solved by molecular replacement with Phaser<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> using 5mkx.pdb as the template for PCAF BRD and 5mlj.pdb as the template for GCN5 BRD. Initial models were refined alternating cycles of automatic refinement with Phenix.refine<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39,40)</a> and manual model building with WinCOOT 0.8.9.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> The structure figures were prepared using Pymol. For the data collection and refinement statistics, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00096/suppl_file/jm9b00096_si_001.pdf" class="ext-link">Supporting Information</a> Table S4.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75">4.9.  NanoBRET</h3><div class="NLM_p last">HEK293T cells (4 × 10<sup>5</sup>/mL) were plated in 6-well plate and co-transfected with Histone H3.3-Halotag and PCAF-Nanoluciferase (Promega). Twenty hours after transfection, cells were digested, collected, and exchanged into Phenol red-free DMEM containing 4% FBS. Cell density was adjusted to 2 × 10<sup>5</sup>/mL and then re-plated into a 96-well assay white plate (Corning Costar #3917) in the absence (blank control) or the presence of 100 nM NanoBRET 618 fluorescent ligand (Promega N1661). Compounds or DMSO (vehicle control) were diluted using media and then added into a 96-well plate at indicated concentrations. After that, the 96-well plate was incubated for 18 h at 37 °C in the presence of 5% CO<sub>2</sub>. NanoBRET Nano-Glo substrate (Promega N1661) was diluted 100 times using media and then added 25 μL to each sample well. Readings were recorded within 10 min using Thermo Scientific Varioskan LUX equipped with 460/80 nm bandpass and 610 nm longpass filter module. The corrected BRET ratio was calculated as the following formula: NanoBRET corrected ratio = ligand (610 nm/460 nm) – blank control (610 nm/460 nm). BRET ratios are expressed as milliBRET units (mBU), where 1 mBU corresponds to the corrected BRET ratio multiplied by 1000.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76">4.10.  RNA-seq</h3><div class="NLM_p">Total RNA was extracted using the TRIzol reagent according to the manufacturer’s protocol. RNA purity and quantification were evaluated using a NanoDrop 2000 spectrophotometer (Thermo Scientific, USA). RNA integrity was assessed using the Agilent 2100 Bioanalyzer (Agilent Technologies, Santa Clara, CA, USA). Then, the libraries were constructed using TruSeq Stranded mRNA LT Sample Prep Kit (Illumina, San Diego, CA, USA) according to the manufacturer’s instructions. The transcriptome sequencing and analysis were conducted by OE Biotech Co., Ltd. (Shanghai, China).</div><div class="NLM_p">The libraries were sequenced on an Illumina HiSeq X Ten platform and 150 bp paired-end reads were generated. Raw data (raw reads) of fastq format were first processed using trimmomatic. After removing low quality reads and reads containing adapter or ploy-N from raw data, the clean reads were mapped to the human genome using HISAT2.</div><div class="NLM_p last">The FPKM value of each gene was calculated using Cufflinks, and the read counts of each gene were obtained by HTSeq-count. Differential expression analysis was performed using the DESeq (2012) R package. The <i>P</i> value <0.05 and fold change >1.5 or <1/1.5 was set as the threshold for significantly differential expression.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77">4.11.  Microsomal Stability Assay</h3><div class="NLM_p last">(<i>R</i>,<i>R</i>)-<b>36n</b> (1 μM) was incubated with 0.5 mg/mL human liver microsomes. NADPH was maintained at 1 mM in 1000 μL of reaction volume. The reaction was then evaluated at 0, 5, 15, 30, and 45 min and was terminated by the addition of acetonitrile. Samples were centrifuged for 15 min at 6000 rpm, and the supernatant was analyzed using LC-MS. Percentage parent remaining was calculated considering percent parent area at 0 min as 100%.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78">4.12.  Plasma Protein Binding</h3><div class="NLM_p last">Plasma protein binding was measured using a previously published protocol.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a></div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79">4.13.  Rats Pharmacokinetic Studies</h3><div class="NLM_p">Animal studies were conducted under the approval of the Experimental Animal Management Committee of Sichuan University. The pharmacokinetics analysis of (<i>R</i>,<i>R</i>)-<b>36n</b> was conducted in male Sprague-Dawley rats. The 6 to 8 week old Sprague-Dawley rats were treated with a single dose of (<i>R,R</i>)-<b>36n</b> at 10 mg/kg by intravenous tail vein injection and oral gavage administration (2.5% (v/v) ethanol and 2.5% tween-80 in saline, pH 7.0). Serial blood samples (200 μL) were collected from the jugular vein at designated times. Blood samples were put on ice and centrifuged to obtain plasma samples (6800<i>g</i>, 6 min under 4 °C) within 2 h. All blood samples were stored at approximately at −80 °C until analysis.</div><div class="NLM_p last">The blood samples were prepared for analysis by placing a 20 μL aliquot into a 96-well plate followed by the addition of 400 μL MeOH of acetonitrile containing 100 ng/mL IS. The mixture was vortexed for 1 min and centrifuged at 18 000<i>g</i> for 7 min. An aliquot of 1 μL supernatant was injected for LC-MS/MS analysis. Noncompartmental pharmacokinetic parameters were fitted using DAS software (Enterprise, version 2.0, Mathematical Pharmacology Professional Committee of China).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i80"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00096" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18314" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18314" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00096" class="ext-link">10.1021/acs.jmedchem.9b00096</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Flowchart of hits identification against PCAF BRD; DSF results of test compounds against a panel of 12 BRDs; BROMOscan assay of (<i>R</i>,<i>R</i>)-<b>36n</b> against 32 BRDs; kinase inhibition profiles of (<i>R</i>,<i>R</i>)-<b>24a</b>, <i>endo</i>-<b>24c</b> and (<i>R</i>,<i>R</i>)-<b>36n</b>; ITC curves of selected compounds against PCAF; X-ray data collection and refinement statistics; <sup>1</sup>H spectra, <sup>13</sup>C NMR spectra; HRMS spectra; and chiral HPLC traces (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00096/suppl_file/jm9b00096_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00096/suppl_file/jm9b00096_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00096/suppl_file/jm9b00096_si_001.pdf">jm9b00096_si_001.pdf (5.02 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00096/suppl_file/jm9b00096_si_002.csv">jm9b00096_si_002.csv (2.47 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates and structure factors have been deposited in the Protein Data Bank under the following accession codes: 6J3O (PCAF/compound <b>12</b>), 6J3P (GCN5/compound (<i>R</i>,<i>R</i>)-<b>36n</b>). Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b00096" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58263" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58263" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shengyong Yang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu 610041, P. R. China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5147-3746" title="Orcid link">http://orcid.org/0000-0001-5147-3746</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#afd6cec1c8dcd6efdcccda81cacbda81ccc1"><span class="__cf_email__" data-cfemail="ccb5ada2abbfb58cbfafb9e2a9a8b9e2afa2">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Luyi Huang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu 610041, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hui Li</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu 610041, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Linli Li</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Drug Targeting and Drug Delivery System of Ministry
of Education, West China School of Pharmacy, Sichuan University, Sichuan 610041, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lu Niu</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu 610041, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Raina Seupel</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Oxford, Old Road Campus Research Building,
Roosevelt Drive, Oxford OX3 7DQ, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Target
Discovery Institute, University of Oxford, NDM Research Building, Roosevelt
Drive, Oxford OX3 7FZ, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chengyong Wu</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu 610041, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wei Cheng</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu 610041, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chong Chen</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu 610041, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bisen Ding</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Cancer Center, West China Hospital, Sichuan University, Chengdu 610041, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul E. Brennan</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Oxford, Old Road Campus Research Building,
Roosevelt Drive, Oxford OX3 7DQ, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Target
Discovery Institute, University of Oxford, NDM Research Building, Roosevelt
Drive, Oxford OX3 7FZ, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8950-7646" title="Orcid link">http://orcid.org/0000-0002-8950-7646</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>L.H., H.L., and L.L. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i83">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37760" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37760" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the National Natural Science Foundation of China (81573349, 81773633, and 21772130), National Science and Technology Major Project (2018ZX09711002-014-002, 2018ZX09711002-011-019, 2018ZX09201018, and 2018ZX09711003-003-006), and 1.3.5 project for disciplines of excellence, West China Hospital, Sichuan University. The authors thank the staff from BL17U1, BL18U, BL19U1 beamlines of National Facility for Protein Science Shanghai (NFPS) at Shanghai Synchrotron Radiation Facility for assistance during data collection.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extra terminal domain</p></td></tr><tr><td class="NLM_term">HAT</td><td class="NLM_def"><p class="first last">histone acetyltransferase</p></td></tr><tr><td class="NLM_term">BRD4(1)</td><td class="NLM_def"><p class="first last">bromodomain containing protein 4, first bromodomain</p></td></tr><tr><td class="NLM_term">S<sub>N</sub>Ar</td><td class="NLM_def"><p class="first last">nucleophilic aromatic substitution</p></td></tr><tr><td class="NLM_term">BRD9</td><td class="NLM_def"><p class="first last">bromodomain containing protein 9</p></td></tr><tr><td class="NLM_term">BAZ2</td><td class="NLM_def"><p class="first last">bromodomain adjacent to zinc finger domain 2</p></td></tr><tr><td class="NLM_term">BRPF</td><td class="NLM_def"><p class="first last">bromodomain and PHD finger</p></td></tr><tr><td class="NLM_term">CECR2</td><td class="NLM_def"><p class="first last">cat eye syndrome chromosome region, candidate 2</p></td></tr><tr><td class="NLM_term">FALZ</td><td class="NLM_def"><p class="first last">fetal Alzheimer-50 clone 1 protein</p></td></tr><tr><td class="NLM_term">TAF1</td><td class="NLM_def"><p class="first last">TBP-associated factor RNA polymerase 1</p></td></tr><tr><td class="NLM_term">EP300</td><td class="NLM_def"><p class="first last">E1A-binding protein, 300 kDa</p></td></tr><tr><td class="NLM_term">DSF</td><td class="NLM_def"><p class="first last">differential scanning fluorimetry</p></td></tr><tr><td class="NLM_term">GCN5</td><td class="NLM_def"><p class="first last">general control nonderepressible 5</p></td></tr><tr><td class="NLM_term">KAT2</td><td class="NLM_def"><p class="first last">lysine acetyl-transferase 2</p></td></tr><tr><td class="NLM_term">PCAF</td><td class="NLM_def"><p class="first last">p300/CBPassociated factor</p></td></tr><tr><td class="NLM_term">HTRF</td><td class="NLM_def"><p class="first last">homogeneous time-resolved fluorescence</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">DMAP</td><td class="NLM_def"><p class="first last">4-dimethylaminepyridine</p></td></tr><tr><td class="NLM_term">RNA-seq</td><td class="NLM_def"><p class="first last">RNA-sequencing</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i85">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54462" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54462" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 42 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.-P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+M%C3%BCllerauthor=T.+Pawsonauthor=A.-C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0lizKhnPEVxgTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">356</span>, <span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0lgiTr75EPJJXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S.</span></span> <span> </span><span class="NLM_article-title">Disrupting acetyl-lysine recognition: progress in the development of bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1271</span>– <span class="NLM_lpage">1298</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01514</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01514" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1271-1298&author=F.+A.+Romeroauthor=A.+M.+Taylorauthor=T.+D.+Crawfordauthor=V.+Tsuiauthor=A.+C%C3%B4t%C3%A9author=S.+Magnuson&title=Disrupting+acetyl-lysine+recognition%3A+progress+in+the+development+of+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b01514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting Acetyl-Lysine Recognition: Progress in the Development of Bromodomain Inhibitors</span></div><div class="casAuthors">Romero, F. Anthony; Taylor, Alexander M.; Crawford, Terry D.; Tsui, Vickie; Cote, Alexandre; Magnuson, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1271-1298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains, small protein modules that recognize acetylated lysine on histones, play a significant role in the epigenome, where they function as "readers" that ultimately det. the functional outcome of the post-translational modification.  Because the initial discovery of selective BET inhibitors have helped define the role of that protein family in oncol. and inflammation, BET bromodomains have continued to garner the most attention of any other bromodomain.  More recently, non-BET bromodomain inhibitors that are potent and selective have been disclosed for ATAD2, CBP, BRD7/9, BRPF, BRPF/TRIM24, CECR2, SMARCA4, and BAZ2A/B.  Such novel inhibitors can be used to probe the physiol. function of these non-BET bromodomains and further understanding of their role in certain disease states.  Here, we provide an update to the progress in identifying selective bromodomain inhibitors and their use as biol. tools, as well as our perspective on the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgbRrKGlHaVLVg90H21EOLACvtfcHk0lgiTr75EPJJXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO&md5=260352f61301aa88aad48c0b63e6152f</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01514%26sid%3Dliteratum%253Aachs%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DDisrupting%2520acetyl-lysine%2520recognition%253A%2520progress%2520in%2520the%2520development%2520of%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1271%26epage%3D1298%26doi%3D10.1021%2Facs.jmedchem.5b01514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Drug discovery targeting bromodomain-containing protein 4</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4533</span>– <span class="NLM_lpage">4558</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01761</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01761" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4533-4558&author=Z.+Liuauthor=P.+Wangauthor=H.+Chenauthor=E.+A.+Woldauthor=B.+Tianauthor=A.+R.+Brasierauthor=J.+Zhou&title=Drug+discovery+targeting+bromodomain-containing+protein+4&doi=10.1021%2Facs.jmedchem.6b01761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Discovery Targeting Bromodomain-Containing Protein 4</span></div><div class="casAuthors">Liu, Zhiqing; Wang, Pingyuan; Chen, Haiying; Wold, Eric A.; Tian, Bing; Brasier, Allan R.; Zhou, Jia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4533-4558</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  BRD4, the most extensively studied member of BET family, is an epigenetic regulator that localizes to DNA via binding acetylated histones and controls the expression of therapeutically important gene regulatory networks through recruiting transcription factors to form mediator complexes, phosphorylating RNA polymerase II and by its intrinsic histone acetyltransferase activity.  Disrupting the protein-protein interactions between BRD4 and acetyl-lysine has been shown to effectively block cell proliferation in cancer, cytokine prodn. in acute inflammation, etc.  To date, significant efforts have been devoted to the development of BRD4 inhibitors, and consequently, a dozen have progressed into human clin. trials.  Herein, the authors summarize the advances in drug discovery and development of BRD4 inhibitors by focusing on their chemotypes, in vitro and in vivo activity, selectivity, relevant mechanisms of action and therapeutic potential.  Opportunities and challenges to achieve selective and efficacious BRD4 inhibitors as a viable therapeutic strategy for human diseases are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6IAERH4iggrVg90H21EOLACvtfcHk0li-F2EUspbHJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D&md5=6d7ba5844073fac8c916b1249af402e5</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01761%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWold%26aufirst%3DE.%2BA.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDrug%2520discovery%2520targeting%2520bromodomain-containing%2520protein%25204%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4533%26epage%3D4558%26doi%3D10.1021%2Facs.jmedchem.6b01761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giotopoulos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, W.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitski, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huthmacher, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gudgin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auger, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doehner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delwel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burnett, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntly, B. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span> <span> </span><span class="NLM_article-title">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>478</i></span>,  <span class="NLM_fpage">529</span>– <span class="NLM_lpage">533</span>, <span class="refDoi"> DOI: 10.1038/nature10509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1038%2Fnature10509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=21964340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=529-533&author=M.+A.+Dawsonauthor=R.+K.+Prinjhaauthor=A.+Dittmannauthor=G.+Giotopoulosauthor=M.+Bantscheffauthor=W.-I.+Chanauthor=S.+C.+Robsonauthor=C.-w.+Chungauthor=C.+Hopfauthor=M.+M.+Savitskiauthor=C.+Huthmacherauthor=E.+Gudginauthor=D.+Lugoauthor=S.+Beinkeauthor=T.+D.+Chapmanauthor=E.+J.+Robertsauthor=P.+E.+Sodenauthor=K.+R.+Augerauthor=O.+Mirguetauthor=K.+Doehnerauthor=R.+Delwelauthor=A.+K.+Burnettauthor=P.+Jeffreyauthor=G.+Drewesauthor=K.+Leeauthor=B.+J.+P.+Huntlyauthor=T.+Kouzarides&title=Inhibition+of+BET+recruitment+to+chromatin+as+an+effective+treatment+for+MLL-fusion+leukaemia&doi=10.1038%2Fnature10509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span></div><div class="casAuthors">Dawson, Mark A.; Prinjha, Rab K.; Dittman, Antje; Giotopoulos, George; Bantscheff, Marcus; Chan, Wai-In; Robson, Samuel C.; Chung, Chun-wa; Hopf, Carsten; Savitski, Mikhail M.; Huthmacher, Carola; Gudgin, Emma; Lugo, Dave; Beinke, Soren; Chapman, Trevor D.; Roberts, Emma J.; Soden, Peter E.; Auger, Kurt R.; Mirguet, Olivier; Doehner, Konstanze; Delwel, Ruud; Burnett, Alan K.; Jeffrey, Phillip; Drewes, Gerard; Lee, Kevin; Huntly, Brian J. P.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">529-533</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recurrent chromosomal translocations involving the mixed lineage leukemia (MLL) gene initiate aggressive forms of leukemia, which are often refractory to conventional therapies.  Many MLL-fusion partners are members of the super elongation complex (SEC), a crit. regulator of transcriptional elongation, suggesting that aberrant control of this process has an important role in leukemia induction.  Here we use a global proteomic strategy to demonstrate that MLL fusions, as part of SEC and the polymerase-assocd. factor complex (PAFc), are assocd. with the BET family of acetyl-lysine recognizing, chromatin adaptor' proteins.  These data provided the basis for therapeutic intervention in MLL-fusion leukemia, via the displacement of the BET family of proteins from chromatin.  We show that a novel small mol. inhibitor of the BET family, GSK1210151A (I-BET151), has profound efficacy against human and murine MLL-fusion leukemic cell lines, through the induction of early cell cycle arrest and apoptosis.  I-BET151 treatment in two human leukemia cell lines with different MLL fusions alters the expression of a common set of genes whose function may account for these phenotypic changes.  The mode of action of I-BET151 is, at least in part, due to the inhibition of transcription at key genes (BCL2, C-MYC and CDK6) through the displacement of BRD3/4, PAFc and SEC components from chromatin.  In vivo studies indicate that I-BET151 has significant therapeutic value, providing survival benefit in two distinct mouse models of murine MLL-AF9 and human MLL-AF4 leukemia.  Finally, the efficacy of I-BET151 against human leukemia stem cells is demonstrated, providing further evidence of its potent therapeutic potential.  These findings establish the displacement of BET proteins from chromatin as a promising epigenetic therapy for these aggressive leukemias.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4FbgJpjftv7Vg90H21EOLACvtfcHk0li-F2EUspbHJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO&md5=dffb4b81da4ca8291c27139d0df86324</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature10509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10509%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DDittmann%26aufirst%3DA.%26aulast%3DGiotopoulos%26aufirst%3DG.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DW.-I.%26aulast%3DRobson%26aufirst%3DS.%2BC.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DHopf%26aufirst%3DC.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DHuthmacher%26aufirst%3DC.%26aulast%3DGudgin%26aufirst%3DE.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DAuger%26aufirst%3DK.%2BR.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DDoehner%26aufirst%3DK.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DBurnett%26aufirst%3DA.%2BK.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%2BP.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DInhibition%2520of%2520BET%2520recruitment%2520to%2520chromatin%2520as%2520an%2520effective%2520treatment%2520for%2520MLL-fusion%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D529%26epage%3D533%26doi%3D10.1038%2Fnature10509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delmore, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Issa, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahl, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobs, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kastritis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilpatrick, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paranal, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schinzel, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heffernan, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergsagel, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghobrial, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsiades, C. S.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition as a therapeutic strategy to target c-Myc</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>146</i></span>,  <span class="NLM_fpage">904</span>– <span class="NLM_lpage">917</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.08.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1016%2Fj.cell.2011.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=21889194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2011&pages=904-917&author=J.+E.+Delmoreauthor=G.+C.+Issaauthor=M.+E.+Lemieuxauthor=P.+B.+Rahlauthor=J.+Shiauthor=H.+M.+Jacobsauthor=E.+Kastritisauthor=T.+Gilpatrickauthor=R.+M.+Paranalauthor=J.+Qiauthor=M.+Chesiauthor=A.+C.+Schinzelauthor=M.+R.+McKeownauthor=T.+P.+Heffernanauthor=C.+R.+Vakocauthor=P.+L.+Bergsagelauthor=I.+M.+Ghobrialauthor=P.+G.+Richardsonauthor=R.+A.+Youngauthor=W.+C.+Hahnauthor=K.+C.+Andersonauthor=A.+L.+Kungauthor=J.+E.+Bradnerauthor=C.+S.+Mitsiades&title=BET+bromodomain+inhibition+as+a+therapeutic+strategy+to+target+c-Myc&doi=10.1016%2Fj.cell.2011.08.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">BET Bromodomain Inhibition as a Therapeutic Strategy to Target c-Myc</span></div><div class="casAuthors">Delmore, Jake E.; Issa, Ghayas C.; Lemieux, Madeleine E.; Rahl, Peter B.; Shi, Jun-Wei; Jacobs, Hannah M.; Kastritis, Efstathios; Gilpatrick, Timothy; Paranal, Ronald M.; Qi, Jun; Chesi, Marta; Schinzel, Anna C.; McKeown, Michael R.; Heffernan, Timothy P.; Vakoc, Christopher R.; Bergsagel, P. Leif; Ghobrial, Irene M.; Richardson, Paul G.; Young, Richard A.; Hahn, William C.; Anderson, Kenneth C.; Kung, Andrew L.; Bradner, James E.; Mitsiades, Constantine S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">904-917</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: MYC contributes to the pathogenesis of a majority of human cancers, yet strategies to modulate the function of the c-Myc oncoprotein do not exist.  Toward this objective, we have targeted MYC transcription by interfering with chromatin-dependent signal transduction to RNA polymerase, specifically by inhibiting the acetyl-lysine recognition domains (bromodomains) of putative coactivator proteins implicated in transcriptional initiation and elongation.  Using a selective small-mol. bromodomain inhibitor, JQ1, we identify BET bromodomain proteins as regulatory factors for c-Myc.  BET inhibition by JQ1 downregulates MYC transcription, followed by genome-wide downregulation of Myc-dependent target genes.  In exptl. models of multiple myeloma, a Myc-dependent hematol. malignancy, JQ1 produces a potent antiproliferative effect assocd. with cell-cycle arrest and cellular senescence.  Efficacy of JQ1 in three murine models of multiple myeloma establishes the therapeutic rationale for BET bromodomain inhibition in this disease and other malignancies characterized by pathol. activation of c-Myc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpevD1QHKmOhLVg90H21EOLACvtfcHk0li-F2EUspbHJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF&md5=013674b8224214a9a1c8f0b75583a5a6</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DDelmore%26aufirst%3DJ.%2BE.%26aulast%3DIssa%26aufirst%3DG.%2BC.%26aulast%3DLemieux%26aufirst%3DM.%2BE.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DJacobs%26aufirst%3DH.%2BM.%26aulast%3DKastritis%26aufirst%3DE.%26aulast%3DGilpatrick%26aufirst%3DT.%26aulast%3DParanal%26aufirst%3DR.%2BM.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DChesi%26aufirst%3DM.%26aulast%3DSchinzel%26aufirst%3DA.%2BC.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DHeffernan%26aufirst%3DT.%2BP.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DBergsagel%26aufirst%3DP.%2BL.%26aulast%3DGhobrial%26aufirst%3DI.%2BM.%26aulast%3DRichardson%26aufirst%3DP.%2BG.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DMitsiades%26aufirst%3DC.%2BS.%26atitle%3DBET%2520bromodomain%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520to%2520target%2520c-Myc%26jtitle%3DCell%26date%3D2011%26volume%3D146%26spage%3D904%26epage%3D917%26doi%3D10.1016%2Fj.cell.2011.08.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandwani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marazzi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span> <span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1119</span>– <span class="NLM_lpage">1123</span>, <span class="refDoi"> DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.-w.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0lhhxa9aGdHV5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brotherton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diez-Dacal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">19754</span>– <span class="NLM_lpage">19759</span>, <span class="refDoi"> DOI: 10.1073/pnas.1310658110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1073%2Fpnas.1310658110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=24248379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=19754-19759&author=S.+Picaudauthor=C.+Wellsauthor=I.+Felletarauthor=D.+Brothertonauthor=S.+Martinauthor=P.+Savitskyauthor=B.+Diez-Dacalauthor=M.+Philpottauthor=C.+Bountraauthor=H.+Lingardauthor=O.+Fedorovauthor=S.+Mullerauthor=P.+E.+Brennanauthor=S.+Knappauthor=P.+Filippakopoulos&title=RVX-208%2C+an+inhibitor+of+BET+transcriptional+regulators+with+selectivity+for+the+second+bromodomain&doi=10.1073%2Fpnas.1310658110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span></div><div class="casAuthors">Picaud, Sarah; Wells, Christopher; Felletar, Ildiko; Brotherton, Deborah; Martin, Sarah; Savitsky, Pavel; Diez-Dacal, Beatriz; Philpott, Martin; Bountra, Chas; Lingard, Hannah; Fedorov, Oleg; Muller, Susanne; Brennan, Paul E.; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">19754-19759,S19754/1-S19754/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomains have emerged as attractive candidates for the development of inhibitors targeting gene transcription.  Inhibitors of the bromo and extraterminal (BET) family recently showed promising activity in diverse disease models.  However, the pleiotropic nature of BET proteins regulating tissue-specific transcription has raised safety concerns and suggested that attempts should be made for domain-specific targeting.  Here, we report that RVX-208, a compd. currently in phase II clin. trials, is a BET bromodomain inhibitor specific for second bromodomains (BD2s).  Cocrystal structures revealed binding modes of RVX-208 and its synthetic precursor, and fluorescent recovery after photobleaching demonstrated that RVX-208 displaces BET proteins from chromatin.  However, gene-expression data showed that BD2 inhibition only modestly affects BET-dependent gene transcription.  Our data demonstrate the feasibility of specific targeting within the BET family resulting in different transcriptional outcomes and highlight the importance of BD1 in transcriptional regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqamW8vnIopKrVg90H21EOLACvtfcHk0lhhxa9aGdHV5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN&md5=c918702c9fdb91241bc837732730f7f6</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1310658110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1310658110%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DBrotherton%26aufirst%3DD.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DDiez-Dacal%26aufirst%3DB.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DRVX-208%252C%2520an%2520inhibitor%2520of%2520BET%2520transcriptional%2520regulators%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D19754%26epage%3D19759%26doi%3D10.1073%2Fpnas.1310658110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witwicki, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabassum, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janiszewska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin Huh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doherty, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammed, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stover, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ekram, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peluffo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Santos, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krop, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duarte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anders, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winer, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Letai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barry, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirley Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Response and resistance to BET bromodomain inhibitors in triple-negative breast cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>529</i></span>,  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">417</span>, <span class="refDoi"> DOI: 10.1038/nature16508</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1038%2Fnature16508" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=26735014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC28Xns1Gmtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=529&publication_year=2016&pages=413-417&author=S.+Shuauthor=C.+Y.+Linauthor=H.+H.+Heauthor=R.+M.+Witwickiauthor=D.+P.+Tabassumauthor=J.+M.+Robertsauthor=M.+Janiszewskaauthor=S.+Jin+Huhauthor=Y.+Liangauthor=J.+Ryanauthor=E.+Dohertyauthor=H.+Mohammedauthor=H.+Guoauthor=D.+G.+Stoverauthor=M.+B.+Ekramauthor=G.+Peluffoauthor=J.+Brownauthor=C.+D%E2%80%99Santosauthor=I.+E.+Kropauthor=D.+Dillonauthor=M.+McKeownauthor=C.+Ottauthor=J.+Qiauthor=M.+Niauthor=P.+K.+Raoauthor=M.+Duarteauthor=S.-Y.+Wuauthor=C.-M.+Chiangauthor=L.+Andersauthor=R.+A.+Youngauthor=E.+P.+Winerauthor=A.+Letaiauthor=W.+T.+Barryauthor=J.+S.+Carrollauthor=H.+W.+Longauthor=M.+Brownauthor=X.+Shirley+Liuauthor=C.+A.+Meyerauthor=J.+E.+Bradner&title=Response+and+resistance+to+BET+bromodomain+inhibitors+in+triple-negative+breast+cancer&doi=10.1038%2Fnature16508"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Response and resistance to BET bromodomain inhibitors in triple-negative breast cancer</span></div><div class="casAuthors">Shu, Shaokun; Lin, Charles Y.; He, Housheng Hansen; Witwicki, Robert M.; Tabassum, Doris P.; Roberts, Justin M.; Janiszewska, Michalina; Huh, Sung Jin; Liang, Yi; Ryan, Jeremy; Doherty, Ernest; Mohammed, Hisham; Guo, Hao; Stover, Daniel G.; Ekram, Muhammad B.; Peluffo, Guillermo; Brown, Jonathan; D'Santos, Clive; Krop, Ian E.; Dillon, Deborah; McKeown, Michael; Ott, Christopher; Qi, Jun; Ni, Min; Rao, Prakash K.; Duarte, Melissa; Wu, Shwu-Yuan; Chiang, Cheng-Ming; Anders, Lars; Young, Richard A.; Winer, Eric P.; Letai, Antony; Barry, William T.; Carroll, Jason S.; Long, Henry W.; Brown, Myles; Shirley Liu, X.; Meyer, Clifford A.; Bradner, James E.; Polyak, Kornelia</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">529</span>
        (<span class="NLM_cas:issue">7586</span>),
    <span class="NLM_cas:pages">413-417</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Triple-neg. breast cancer (TNBC) is a heterogeneous and clin. aggressive disease for which there is no targeted therapy.  BET bromodomain inhibitors, which have shown efficacy in several models of cancer, have not been evaluated in TNBC.  These inhibitors displace BET bromodomain proteins such as BRD4 from chromatin by competing with their acetyl-lysine recognition modules, leading to inhibition of oncogenic transcriptional programs.  Here we report the preferential sensitivity of TNBCs to BET bromodomain inhibition in vitro and in vivo, establishing a rationale for clin. investigation and further motivation to understand mechanisms of resistance.  In paired cell lines selected for acquired resistance to BET inhibition from previously sensitive TNBCs, we failed to identify gatekeeper mutations, new driver events or drug pump activation.  BET-resistant TNBC cells remain dependent on wild-type BRD4, which supports transcription and cell proliferation in a bromodomain-independent manner.  Proteomic studies of resistant TNBC identify strong assocn. with MED1 and hyper-phosphorylation of BRD4 attributable to decreased activity of PP2A, identified here as a principal BRD4 serine phosphatase.  Together, these studies provide a rationale for BET inhibition in TNBC and present mechanism-based combination strategies to anticipate clin. drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZu5A85pdQDbVg90H21EOLACvtfcHk0ljEmPD_J9kjnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xns1Gmtw%253D%253D&md5=50a03861d9156ee6e344e95c5f5361b9</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnature16508&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature16508%26sid%3Dliteratum%253Aachs%26aulast%3DShu%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DHe%26aufirst%3DH.%2BH.%26aulast%3DWitwicki%26aufirst%3DR.%2BM.%26aulast%3DTabassum%26aufirst%3DD.%2BP.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DJaniszewska%26aufirst%3DM.%26aulast%3DJin%2BHuh%26aufirst%3DS.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DRyan%26aufirst%3DJ.%26aulast%3DDoherty%26aufirst%3DE.%26aulast%3DMohammed%26aufirst%3DH.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DStover%26aufirst%3DD.%2BG.%26aulast%3DEkram%26aufirst%3DM.%2BB.%26aulast%3DPeluffo%26aufirst%3DG.%26aulast%3DBrown%26aufirst%3DJ.%26aulast%3DD%25E2%2580%2599Santos%26aufirst%3DC.%26aulast%3DKrop%26aufirst%3DI.%2BE.%26aulast%3DDillon%26aufirst%3DD.%26aulast%3DMcKeown%26aufirst%3DM.%26aulast%3DOtt%26aufirst%3DC.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DNi%26aufirst%3DM.%26aulast%3DRao%26aufirst%3DP.%2BK.%26aulast%3DDuarte%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DS.-Y.%26aulast%3DChiang%26aufirst%3DC.-M.%26aulast%3DAnders%26aufirst%3DL.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DWiner%26aufirst%3DE.%2BP.%26aulast%3DLetai%26aufirst%3DA.%26aulast%3DBarry%26aufirst%3DW.%2BT.%26aulast%3DCarroll%26aufirst%3DJ.%2BS.%26aulast%3DLong%26aufirst%3DH.%2BW.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DShirley%2BLiu%26aufirst%3DX.%26aulast%3DMeyer%26aufirst%3DC.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DResponse%2520and%2520resistance%2520to%2520BET%2520bromodomain%2520inhibitors%2520in%2520triple-negative%2520breast%2520cancer%26jtitle%3DNature%26date%3D2016%26volume%3D529%26spage%3D413%26epage%3D417%26doi%3D10.1038%2Fnature16508" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Theodoulou, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Progress in the development of non-BET bromodomain chemical probes</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">477</span>– <span class="NLM_lpage">487</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1002%2Fcmdc.201500540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=26749027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC28Xks1ehtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=477-487&author=N.+H.+Theodoulouauthor=N.+C.+O.+Tomkinsonauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Progress+in+the+development+of+non-BET+bromodomain+chemical+probes&doi=10.1002%2Fcmdc.201500540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Progress in the Development of non-BET Bromodomain Chemical Probes</span></div><div class="casAuthors">Theodoulou, Natalie H.; Tomkinson, Nicholas C. O.; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">477-487</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The bromodomain and extra terminal (BET) family of bromodomains have been the focus of extensive research, leading to the development of many potent, selective chem. probes and recent clin. assets.  The profound biol. assocd. with BET bromodomain inhibition has provided a convincing rationale for targeting bromodomains for the treatment of disease.  However, the BET family represents just eight of the at least 56 human bromodomains identified to date.  Until recently, there has been significantly less interest in non-BET bromodomains, leaving a vast area of research and the majority of this new target class yet to be thoroughly investigated.  It has been widely reported that several non-BET bromodomain contg. proteins are assocd. with various diseases including cancer and HIV.  Therefore, the development of chem. probes for non-BET bromodomains will facilitate elucidation of their precise biol. roles and potentially lead to the development of new medicines.  This review summarises the progress made towards the development of non-BET bromodomain chem. probes to date.  In addn., we highlight the potential for future work in this new and exciting area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYLj_VyHOS5bVg90H21EOLACvtfcHk0ljEmPD_J9kjnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xks1ehtA%253D%253D&md5=1e1339bf09a10bde8621c18344172fa6</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500540%26sid%3Dliteratum%253Aachs%26aulast%3DTheodoulou%26aufirst%3DN.%2BH.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DProgress%2520in%2520the%2520development%2520of%2520non-BET%2520bromodomain%2520chemical%2520probes%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D477%26epage%3D487%26doi%3D10.1002%2Fcmdc.201500540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moustakim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, P. G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Chemical probes and inhibitors of bromodomains outside the BET family</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">2246</span>– <span class="NLM_lpage">2264</span>, <span class="refDoi"> DOI: 10.1039/c6md00373g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1039%2Fc6md00373g" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=29170712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVOhsbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=2246-2264&author=M.+Moustakimauthor=P.+G.+K.+Clarkauthor=D.+A.+Hayauthor=D.+J.+Dixonauthor=P.+E.+Brennan&title=Chemical+probes+and+inhibitors+of+bromodomains+outside+the+BET+family&doi=10.1039%2Fc6md00373g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical probes and inhibitors of bromodomains outside the BET family</span></div><div class="casAuthors">Moustakim, Moses; Clark, Peter G. K.; Hay, Duncan A.; Dixon, Darren J.; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2246-2264</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">In the last five years, the development of inhibitors of bromodomains has emerged as an area of intensive worldwide research.  Emerging evidence has implicated a no. of non-BET bromodomains in the onset and progression of diseases such as cancer, HIV infection and inflammation.  The development and use of small mol. chem. probes has been fundamental to pre-clin. evaluation of bromodomains as targets.  Recent efforts are described highlighting the development of potent, selective and cell active non-BET bromodomain inhibitors and their therapeutic potential.  Over half of typical bromodomains now have reported ligands, but those with atypical binding site residues remain resistant to chem. probe discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYJvhj3Z0Ry7Vg90H21EOLACvtfcHk0ljEmPD_J9kjnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVOhsbbF&md5=df4c5718b15ff7d401a0cd9974abc236</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1039%2Fc6md00373g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc6md00373g%26sid%3Dliteratum%253Aachs%26aulast%3DMoustakim%26aufirst%3DM.%26aulast%3DClark%26aufirst%3DP.%2BG.%2BK.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DDixon%26aufirst%3DD.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DChemical%2520probes%2520and%2520inhibitors%2520of%2520bromodomains%2520outside%2520the%2520BET%2520family%26jtitle%3DMedChemComm%26date%3D2016%26volume%3D7%26spage%3D2246%26epage%3D2264%26doi%3D10.1039%2Fc6md00373g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clegg, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Advancements in the development of non-BET bromodomain chemical probes</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">362</span>– <span class="NLM_lpage">385</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201800738</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1002%2Fcmdc.201800738" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=30624862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1Ohs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=362-385&author=M.+A.+Cleggauthor=N.+C.+O.+Tomkinsonauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Advancements+in+the+development+of+non-BET+bromodomain+chemical+probes&doi=10.1002%2Fcmdc.201800738"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Advancements in the Development of non-BET Bromodomain Chemical Probes</span></div><div class="casAuthors">Clegg, Michael A.; Tomkinson, Nicholas C. O.; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">362-385</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The bromodomain and extra terminal (BET) family of bromodomain-contg. proteins (BCPs) have been the subject of extensive research over the past decade, resulting in a plethora of high-quality chem. probes for their tandem bromodomains.  In turn, these chem. probes have helped reveal the profound biol. role of the BET bromodomains and their role in disease, ultimately leading to a no. of mols. in active clin. development.  However, the BET subfamily represents just 8/61 of the known human bromodomains, and attention has now expanded to the biol. role of the remaining 53 non-BET bromodomains.  Rapid growth of this research area has been accompanied by a greater understanding of the requirements for an effective bromodomain chem. probe and has led to a no. of new non-BET bromodomain chem. probes being developed.  Advances since Dec. 2015 are discussed, highlighting the strengths/caveats of each mol., and the value they add toward validating the non-BET bromodomains as tractable therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_U9uvpaCEd7Vg90H21EOLACvtfcHk0lhskiLEPykikg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1Ohs70%253D&md5=a93505759fe45ffd0352de2f0e17885a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201800738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201800738%26sid%3Dliteratum%253Aachs%26aulast%3DClegg%26aufirst%3DM.%2BA.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DAdvancements%2520in%2520the%2520development%2520of%2520non-BET%2520bromodomain%2520chemical%2520probes%26jtitle%3DChemMedChem%26date%3D2019%26volume%3D14%26spage%3D362%26epage%3D385%26doi%3D10.1002%2Fcmdc.201800738" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lau, O. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Courtney, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassilev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marzilli, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cotter, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakatani, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">P300/CBP-associated factor histone acetyltransferase processing of a peptide substrate. Kinetic analysis of the catalytic mechanism</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>275</i></span>,  <span class="NLM_fpage">21953</span>– <span class="NLM_lpage">21959</span>, <span class="refDoi"> DOI: 10.1074/jbc.m003219200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1074%2Fjbc.m003219200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=275&publication_year=2000&pages=21953-21959&author=O.+D.+Lauauthor=A.+D.+Courtneyauthor=A.+Vassilevauthor=L.+A.+Marzilliauthor=R.+J.+Cotterauthor=Y.+Nakataniauthor=P.+A.+Cole&title=P300%2FCBP-associated+factor+histone+acetyltransferase+processing+of+a+peptide+substrate.+Kinetic+analysis+of+the+catalytic+mechanism&doi=10.1074%2Fjbc.m003219200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m003219200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m003219200%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DO.%2BD.%26aulast%3DCourtney%26aufirst%3DA.%2BD.%26aulast%3DVassilev%26aufirst%3DA.%26aulast%3DMarzilli%26aufirst%3DL.%2BA.%26aulast%3DCotter%26aufirst%3DR.%2BJ.%26aulast%3DNakatani%26aufirst%3DY.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DP300%252FCBP-associated%2520factor%2520histone%2520acetyltransferase%2520processing%2520of%2520a%2520peptide%2520substrate.%2520Kinetic%2520analysis%2520of%2520the%2520catalytic%2520mechanism%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2000%26volume%3D275%26spage%3D21953%26epage%3D21959%26doi%3D10.1074%2Fjbc.m003219200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Linares, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiernan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Triboulet, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chable-Bessia, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latreille, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuvier, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacroix, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Cam, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coux, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benkirane, M.</span></span> <span> </span><span class="NLM_article-title">Intrinsic ubiquitination activity of PCAF controls the stability of the oncoprotein Hdm2</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">331</span>– <span class="NLM_lpage">338</span>, <span class="refDoi"> DOI: 10.1038/ncb1545</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1038%2Fncb1545" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=17293853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BD2sXit12rsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2007&pages=331-338&author=L.+K.+Linaresauthor=R.+Kiernanauthor=R.+Tribouletauthor=C.+Chable-Bessiaauthor=D.+Latreilleauthor=O.+Cuvierauthor=M.+Lacroixauthor=L.+Le+Camauthor=O.+Couxauthor=M.+Benkirane&title=Intrinsic+ubiquitination+activity+of+PCAF+controls+the+stability+of+the+oncoprotein+Hdm2&doi=10.1038%2Fncb1545"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Intrinsic ubiquitination activity of PCAF controls the stability of the oncoprotein Hdm2</span></div><div class="casAuthors">Linares, Laetitia K.; Kiernan, Rosemary; Triboulet, Robinson; Chable-Bessia, Christine; Latreille, Daniel; Cuvier, Olivier; Lacroix, Matthieu; Le Cam, Laurent; Coux, Olivier; Benkirane, Monsef</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">331-338</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The p300-CBP-assocd. factor (PCAF) is a histone acetyltransferase (HAT) involved in the reversible acetylation of various transcriptional regulators, including the tumor suppressor p53.  It is implicated in many cellular processes, such as transcription, differentiation, proliferation and apoptosis.  We obsd. that knockdown of PCAF expression in HeLa or U2OS cell lines induces stabilization of the oncoprotein Hdm2, a RING finger E3 ligase primarily known for its role in controlling p53 stability.  To investigate the mol. basis of this effect, we examd. whether PCAF is involved in Hdm2 ubiquitination.  Here, we show that PCAF, in addn. to its acetyltransferase activity, possesses an intrinsic ubiquitination activity that is crit. for controlling Hdm2 expression levels, and thus p53 functions.  Our data highlight a regulatory crosstalk between PCAF and Hdm2 activities, which is likely to have a central role in the subtle control of p53 activity after DNA damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIuN86NzWK97Vg90H21EOLACvtfcHk0lhskiLEPykikg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXit12rsr8%253D&md5=93136ff023459d9d4d037a7c97bf12a2</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fncb1545&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb1545%26sid%3Dliteratum%253Aachs%26aulast%3DLinares%26aufirst%3DL.%2BK.%26aulast%3DKiernan%26aufirst%3DR.%26aulast%3DTriboulet%26aufirst%3DR.%26aulast%3DChable-Bessia%26aufirst%3DC.%26aulast%3DLatreille%26aufirst%3DD.%26aulast%3DCuvier%26aufirst%3DO.%26aulast%3DLacroix%26aufirst%3DM.%26aulast%3DLe%2BCam%26aufirst%3DL.%26aulast%3DCoux%26aufirst%3DO.%26aulast%3DBenkirane%26aufirst%3DM.%26atitle%3DIntrinsic%2520ubiquitination%2520activity%2520of%2520PCAF%2520controls%2520the%2520stability%2520of%2520the%2520oncoprotein%2520Hdm2%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2007%26volume%3D9%26spage%3D331%26epage%3D338%26doi%3D10.1038%2Fncb1545" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mujtaba, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farooq, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">Structural basis of lysine-acetylated HIV-1 Tat recognition by PCAF bromodomain</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">575</span>– <span class="NLM_lpage">586</span>, <span class="refDoi"> DOI: 10.1016/s1097-2765(02)00483-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1016%2Fs1097-2765%2802%2900483-5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=575-586&author=S.+Mujtabaauthor=Y.+Heauthor=L.+Zengauthor=A.+Farooqauthor=J.+E.+Carlsonauthor=M.+Ottauthor=E.+Verdinauthor=M.-M.+Zhou&title=Structural+basis+of+lysine-acetylated+HIV-1+Tat+recognition+by+PCAF+bromodomain&doi=10.1016%2Fs1097-2765%2802%2900483-5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fs1097-2765%2802%2900483-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs1097-2765%252802%252900483-5%26sid%3Dliteratum%253Aachs%26aulast%3DMujtaba%26aufirst%3DS.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DFarooq%26aufirst%3DA.%26aulast%3DCarlson%26aufirst%3DJ.%2BE.%26aulast%3DOtt%26aufirst%3DM.%26aulast%3DVerdin%26aufirst%3DE.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DStructural%2520basis%2520of%2520lysine-acetylated%2520HIV-1%2520Tat%2520recognition%2520by%2520PCAF%2520bromodomain%26jtitle%3DMol.%2520Cell%26date%3D2002%26volume%3D9%26spage%3D575%26epage%3D586%26doi%3D10.1016%2Fs1097-2765%2802%2900483-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, F.</span></span> <span> </span><span class="NLM_article-title">PCAF fine-tunes hepatic metabolic syndrome, inflammatory disease, and cancer</span>. <i>J. Cell. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">5787</span>– <span class="NLM_lpage">5800</span>, <span class="refDoi"> DOI: 10.1111/jcmm.13877</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1111%2Fjcmm.13877" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=30216660" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A280%3ADC%252BB3c3ovFCqsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2018&pages=5787-5800&author=T.+Wangauthor=W.+Yaoauthor=Y.+Shaoauthor=R.+Zhengauthor=F.+Huang&title=PCAF+fine-tunes+hepatic+metabolic+syndrome%2C+inflammatory+disease%2C+and+cancer&doi=10.1111%2Fjcmm.13877"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">PCAF fine-tunes hepatic metabolic syndrome, inflammatory disease, and cancer</span></div><div class="casAuthors">Wang Tongxin; Yao Weilei; Shao Yafei; Zheng Ruilong; Huang Feiruo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of cellular and molecular medicine</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5787-5800</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The P300/CBP-associating factor (PCAF), a histone acetyltransferase, is involved in metabolic and pathogenic diseases, particularly of the liver.  The effects of PCAF on fine-tuning liver diseases are extremely complex and vary according to different pathological conditions.  This enzyme has dichotomous functions, depending on differently modified sites, which regulate the activities of various enzymes, metabolic functions, and gene expression.  Here, we summarize the most recent findings on the functions and targets of PCAF in various metabolic and immunological processes in the liver and review these new discoveries and models of PCAF biology in three areas: hepatic metabolic syndrome, inflammatory disease, and cancer.  Finally, we discuss the potential implications of these findings for therapeutic interventions in liver diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS9R6QYjknpJ0NHe_gFldTbfW6udTcc2eYbXQaNFwfqI7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3ovFCqsQ%253D%253D&md5=ca754ef104057fc72eb7bb3ca5f59a1d</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1111%2Fjcmm.13877&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjcmm.13877%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DT.%26aulast%3DYao%26aufirst%3DW.%26aulast%3DShao%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DR.%26aulast%3DHuang%26aufirst%3DF.%26atitle%3DPCAF%2520fine-tunes%2520hepatic%2520metabolic%2520syndrome%252C%2520inflammatory%2520disease%252C%2520and%2520cancer%26jtitle%3DJ.%2520Cell.%2520Mol.%2520Med.%26date%3D2018%26volume%3D22%26spage%3D5787%26epage%3D5800%26doi%3D10.1111%2Fjcmm.13877" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mujtaba, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">Selective small molecules blocking HIV-1 Tat and coactivator PCAF association</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">2376</span>– <span class="NLM_lpage">2377</span>, <span class="refDoi"> DOI: 10.1021/ja044885g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja044885g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2005&pages=2376-2377&author=L.+Zengauthor=J.+Liauthor=M.+Mullerauthor=S.+Yanauthor=S.+Mujtabaauthor=C.+Panauthor=Z.+Wangauthor=M.-M.+Zhou&title=Selective+small+molecules+blocking+HIV-1+Tat+and+coactivator+PCAF+association&doi=10.1021%2Fja044885g"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fja044885g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja044885g%26sid%3Dliteratum%253Aachs%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DMuller%26aufirst%3DM.%26aulast%3DYan%26aufirst%3DS.%26aulast%3DMujtaba%26aufirst%3DS.%26aulast%3DPan%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DSelective%2520small%2520molecules%2520blocking%2520HIV-1%2520Tat%2520and%2520coactivator%2520PCAF%2520association%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2005%26volume%3D127%26spage%3D2376%26epage%3D2377%26doi%3D10.1021%2Fja044885g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span> <span> </span><span class="NLM_article-title">Fluorescence polarization for the evaluation of small-molecule inhibitors of PCAF BRD/Tat-AcK50 association</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">928</span>– <span class="NLM_lpage">931</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201300499</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1002%2Fcmdc.201300499" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=24474698" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtleisb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=928-931&author=P.+Huauthor=X.+Wangauthor=B.+Zhangauthor=S.+Zhangauthor=Q.+Wangauthor=Z.+Wang&title=Fluorescence+polarization+for+the+evaluation+of+small-molecule+inhibitors+of+PCAF+BRD%2FTat-AcK50+association&doi=10.1002%2Fcmdc.201300499"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence Polarization for the Evaluation of Small-Molecule Inhibitors of PCAF BRD/Tat-AcK50 Association</span></div><div class="casAuthors">Hu, Ping; Wang, Xinghui; Zhang, Baiqun; Zhang, Shuai; Wang, Qiang; Wang, Zhiyong</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">928-931</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A fluorescence polarization competitive assay was developed to efficiently screen and evaluate inhibitors of PCAF bromodomain/Tat-AcK50 protein-peptide interaction.  A series of pyridine 1-oxide derivs. were synthesized and evaluated.  Some of the novel compds., 2-(3-aminopropylamino) pyridine 1-oxide derivs., could be effective inhibitors of PCAF bromodomain/Tat-AcK50 assocn.  Specifically, 2-(3-aminopropylamino)-5-(hydroxymethyl)pyridine 1-oxide hydrochloride (15) and the 5-((3-aminopropylamino)methyl) deriv. (20) were found to be effective ligands for the PCAF BRD pocket.  First preliminary cellular studies indicate that these small-mol. inhibitors have lower cytotoxicities and are potential leads for the anti-HIV/AIDS therapeutic strategy by targeting host-cell protein PCAF BRD to block HIV replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2Jg2NNDsHHrVg90H21EOLACvtfcHk0lgvqtryJ2TEKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtleisb0%253D&md5=97dc23c85843ad1611f5927fd2ee540c</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300499%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DZ.%26atitle%3DFluorescence%2520polarization%2520for%2520the%2520evaluation%2520of%2520small-molecule%2520inhibitors%2520of%2520PCAF%2520BRD%252FTat-AcK50%2520association%26jtitle%3DChemMedChem%26date%3D2014%26volume%3D9%26spage%3D928%26epage%3D931%26doi%3D10.1002%2Fcmdc.201300499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Navratilova, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aristotelous, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filappakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span> <span> </span><span class="NLM_article-title">Discovery of new bromodomain scaffolds by biosensor fragment screening</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1213</span>– <span class="NLM_lpage">1218</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00154</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00154" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=1213-1218&author=I.+Navratilovaauthor=T.+Aristotelousauthor=S.+Picaudauthor=A.+Chaikuadauthor=S.+Knappauthor=P.+Filappakopoulosauthor=A.+L.+Hopkins&title=Discovery+of+new+bromodomain+scaffolds+by+biosensor+fragment+screening&doi=10.1021%2Facsmedchemlett.6b00154"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00154%26sid%3Dliteratum%253Aachs%26aulast%3DNavratilova%26aufirst%3DI.%26aulast%3DAristotelous%26aufirst%3DT.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilappakopoulos%26aufirst%3DP.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DDiscovery%2520of%2520new%2520bromodomain%2520scaffolds%2520by%2520biosensor%2520fragment%2520screening%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D1213%26epage%3D1218%26doi%3D10.1021%2Facsmedchemlett.6b00154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Temperini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollander, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nachane, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegal, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based identification of inhibitory fragments targeting the p300/CBP-associated factor bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1648</span>– <span class="NLM_lpage">1653</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01719</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01719" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1648-1653&author=A.+Chaikuadauthor=S.+Langauthor=P.+E.+Brennanauthor=C.+Temperiniauthor=O.+Fedorovauthor=J.+Hollanderauthor=R.+Nachaneauthor=C.+Abellauthor=S.+M%C3%BCllerauthor=G.+Siegalauthor=S.+Knapp&title=Structure-based+identification+of+inhibitory+fragments+targeting+the+p300%2FCBP-associated+factor+bromodomain&doi=10.1021%2Facs.jmedchem.5b01719"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01719&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01719%26sid%3Dliteratum%253Aachs%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DLang%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DTemperini%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DHollander%26aufirst%3DJ.%26aulast%3DNachane%26aufirst%3DR.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DSiegal%26aufirst%3DG.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DStructure-based%2520identification%2520of%2520inhibitory%2520fragments%2520targeting%2520the%2520p300%252FCBP-associated%2520factor%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1648%26epage%3D1653%26doi%3D10.1021%2Facs.jmedchem.5b01719" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span>; <span class="NLM_string-name">Cote, A.</span>; <span class="NLM_string-name">Crawford, T.</span>; <span class="NLM_string-name">Duplessis, M.</span>; <span class="NLM_string-name">Good, A.
C.</span>; <span class="NLM_string-name">Leblanc, Y.</span>; <span class="NLM_string-name">Magnuson, S.</span>; <span class="NLM_string-name">Nasveschuk, C. G.</span>; <span class="NLM_string-name">Pastor, R.</span>; <span class="NLM_string-name">Romero, F. A.</span>; <span class="NLM_string-name">Taylor, A. M.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Compounds and Uses Thereof</span>. WO <span class="NLM_patent">2016036873 A1</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=B.+K.+Albrecht&author=A.+Cote&author=T.+Crawford&author=M.+Duplessis&author=A.%0AC.+Good&author=Y.+Leblanc&author=S.+Magnuson&author=C.+G.+Nasveschuk&author=R.+Pastor&author=F.+A.+Romero&author=A.+M.+Taylor&title=Therapeutic+Compounds+and+Uses+Thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26atitle%3DTherapeutic%2520Compounds%2520and%2520Uses%2520Thereof%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span>; <span class="NLM_string-name">Cote, A.</span>; <span class="NLM_string-name">Crawford, T.</span>; <span class="NLM_string-name">Duplessis, M.</span>; <span class="NLM_string-name">Good, A.
C.</span>; <span class="NLM_string-name">Leblanc, Y.</span>; <span class="NLM_string-name">Magnuson, S.</span>; <span class="NLM_string-name">Nasveschuk, C. G.</span>; <span class="NLM_string-name">Pastor, R.</span>; <span class="NLM_string-name">Romero, F. A.</span>; <span class="NLM_string-name">Taylor, A. M.</span></span> <span> </span><span class="NLM_article-title">Phthalazine Derivatives of Forumula (I) as PCAF and GCN5 Inhibitors for Use in the Treatment of Cancer</span>. WO <span class="NLM_patent">2016036954 A1</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=B.+K.+Albrecht&author=A.+Cote&author=T.+Crawford&author=M.+Duplessis&author=A.%0AC.+Good&author=Y.+Leblanc&author=S.+Magnuson&author=C.+G.+Nasveschuk&author=R.+Pastor&author=F.+A.+Romero&author=A.+M.+Taylor&title=Phthalazine+Derivatives+of+Forumula+%28I%29+as+PCAF+and+GCN5+Inhibitors+for+Use+in+the+Treatment+of+Cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26atitle%3DPhthalazine%2520Derivatives%2520of%2520Forumula%2520%2528I%2529%2520as%2520PCAF%2520and%2520GCN5%2520Inhibitors%2520for%2520Use%2520in%2520the%2520Treatment%2520of%2520Cancer%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span>; <span class="NLM_string-name">Burdick, D. J.</span>; <span class="NLM_string-name">Cote, A.</span>; <span class="NLM_string-name">Duplessis, M.</span>; <span class="NLM_string-name">Nasveschuk, C. G.</span>; <span class="NLM_string-name">Taylor, A. M.</span></span> <span> </span><span class="NLM_article-title">Pyridazinone Derivatives and Their Use in the Treatment of Cancer</span>. WO <span class="NLM_patent">2016112298 A1</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=B.+K.+Albrecht&author=D.+J.+Burdick&author=A.+Cote&author=M.+Duplessis&author=C.+G.+Nasveschuk&author=A.+M.+Taylor&title=Pyridazinone+Derivatives+and+Their+Use+in+the+Treatment+of+Cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26atitle%3DPyridazinone%2520Derivatives%2520and%2520Their%2520Use%2520in%2520the%2520Treatment%2520of%2520Cancer%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moustakim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, P. G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trulli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes de Arriba, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehebauer, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendez-Johnson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, C.-F. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorrell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Discovery of a PCAF bromodomain chemical probe</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">827</span>– <span class="NLM_lpage">831</span>, <span class="refDoi"> DOI: 10.1002/anie.201610816</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1002%2Fanie.201610816" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC28XitV2mtL3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=827-831&author=M.+Moustakimauthor=P.+G.+K.+Clarkauthor=L.+Trulliauthor=A.+L.+Fuentes+de+Arribaauthor=M.+T.+Ehebauerauthor=A.+Chaikuadauthor=E.+J.+Murphyauthor=J.+Mendez-Johnsonauthor=D.+Danielsauthor=C.-F.+D.+Houauthor=Y.-H.+Linauthor=J.+R.+Walkerauthor=R.+Huiauthor=H.+Yangauthor=L.+Dorrellauthor=C.+M.+Rogersauthor=O.+P.+Monteiroauthor=O.+Fedorovauthor=K.+V.+M.+Huberauthor=S.+Knappauthor=J.+Heerauthor=D.+J.+Dixonauthor=P.+E.+Brennan&title=Discovery+of+a+PCAF+bromodomain+chemical+probe&doi=10.1002%2Fanie.201610816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a PCAF Bromodomain Chemical Probe</span></div><div class="casAuthors">Moustakim, Moses; Clark, Peter G. K.; Trulli, Laura; Fuentes de Arriba, Angel L.; Ehebauer, Matthias T.; Chaikuad, Apirat; Murphy, Emma J.; Mendez-Johnson, Jacqui; Daniels, Danette; Hou, Chun-Feng D.; Lin, Yu-Hui; Walker, John R.; Hui, Raymond; Yang, Hongbing; Dorrell, Lucy; Rogers, Catherine M.; Monteiro, Octovia P.; Fedorov, Oleg; Huber, Kilian V. M.; Knapp, Stefan; Heer, Jag; Dixon, Darren J.; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">827-831</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The p300/CBP-assocd. factor (PCAF) and related GCN5 bromodomain-contg. lysine acetyl transferases are members of subfamily I of the bromodomain phylogenetic tree.  Iterative cycles of rational inhibitor design and biophys. characterization led to the discovery of the triazolopthalazine-based L-45 (dubbed L-Moses) as the first potent, selective, and cell-active PCAF bromodomain (Brd) inhibitor.  Synthesis from readily available (1R,2S)-(-)-norephedrine furnished L-45 in enantiopure form.  L-45 was shown to disrupt PCAF-Brd histone H3.3 interaction in cells using a nanoBRET assay, and a co-crystal structure of L-45 with the homologous Brd PfGCN5 from Plasmodium falciparum rationalizes the high selectivity for PCAF and GCN5 bromodomains.  Compd. L-45 shows no observable cytotoxicity in peripheral blood mononuclear cells (PBMC), good cell-permeability, and metabolic stability in human and mouse liver microsomes, supporting its potential for in vivo use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3qpUenJBD_7Vg90H21EOLACvtfcHk0lhfT3IL5JfTLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitV2mtL3M&md5=26c4dedfb4c441d47cf2bbd44885c09a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fanie.201610816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201610816%26sid%3Dliteratum%253Aachs%26aulast%3DMoustakim%26aufirst%3DM.%26aulast%3DClark%26aufirst%3DP.%2BG.%2BK.%26aulast%3DTrulli%26aufirst%3DL.%26aulast%3DFuentes%2Bde%2BArriba%26aufirst%3DA.%2BL.%26aulast%3DEhebauer%26aufirst%3DM.%2BT.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DMurphy%26aufirst%3DE.%2BJ.%26aulast%3DMendez-Johnson%26aufirst%3DJ.%26aulast%3DDaniels%26aufirst%3DD.%26aulast%3DHou%26aufirst%3DC.-F.%2BD.%26aulast%3DLin%26aufirst%3DY.-H.%26aulast%3DWalker%26aufirst%3DJ.%2BR.%26aulast%3DHui%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DDorrell%26aufirst%3DL.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHeer%26aufirst%3DJ.%26aulast%3DDixon%26aufirst%3DD.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DDiscovery%2520of%2520a%2520PCAF%2520bromodomain%2520chemical%2520probe%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D827%26epage%3D831%26doi%3D10.1002%2Fanie.201610816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayhow, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renaux, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suckling, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent, cell penetrant, and selective P300/CBP-associated factor (PCAF)/general control nonderepressible 5 (GCN5) bromodomain chemical probe</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">695</span>– <span class="NLM_lpage">709</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01566</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01566" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFCgs77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=695-709&author=P.+G.+Humphreysauthor=P.+Bamboroughauthor=C.-w.+Chungauthor=P.+D.+Craggsauthor=L.+Gordonauthor=P.+Grandiauthor=T.+G.+Hayhowauthor=J.+Hussainauthor=K.+L.+Jonesauthor=M.+Lindonauthor=A.-M.+Michonauthor=J.+F.+Renauxauthor=C.+J.+Sucklingauthor=D.+F.+Toughauthor=R.+K.+Prinjha&title=Discovery+of+a+potent%2C+cell+penetrant%2C+and+selective+P300%2FCBP-associated+factor+%28PCAF%29%2Fgeneral+control+nonderepressible+5+%28GCN5%29+bromodomain+chemical+probe&doi=10.1021%2Facs.jmedchem.6b01566"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent, Cell Penetrant, and Selective p300/CBP-Associated Factor (PCAF)/General Control Nonderepressible 5 (GCN5) Bromodomain Chemical Probe</span></div><div class="casAuthors">Humphreys, Philip G.; Bamborough, Paul; Chung, Chun-wa; Craggs, Peter D.; Gordon, Laurie; Grandi, Paola; Hayhow, Thomas G.; Hussain, Jameed; Jones, Katherine L.; Lindon, Matthew; Michon, Anne-Marie; Renaux, Jessica F.; Suckling, Colin J.; Tough, David F.; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">695-709</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P300/CREB binding protein assocd. factor (PCAF/KAT2B) and general control non-derepressible 5 (GCN5/KAT2A) are multidomain proteins that have been implicated in retroviral infection, inflammation pathways and cancer development.  However, outside of viral replication, little is known about the dependence of these effects on the C-terminal bromodomain.  Herein, the authors report GSK4027 as a chem. probe for the PCAF/GCN5 bromodomain, together with GSK4028 as an enantiomeric neg. control.  The probe was optimized from a weakly potent, non-selective pyridazinone hit to deliver high potency for the PCAF/GCN5 bromodomain, high soly., cellular target engagement and ≥18,000-fold selectivity over the BET family, together with ≥70-fold selectivity over the wider bromodomain families.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBkLwjcv1aZLVg90H21EOLACvtfcHk0lhNd82FnagXUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFCgs77N&md5=fd4fdbe8827e26769fc17b09120c7826</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01566%26sid%3Dliteratum%253Aachs%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DHayhow%26aufirst%3DT.%2BG.%26aulast%3DHussain%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DK.%2BL.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DRenaux%26aufirst%3DJ.%2BF.%26aulast%3DSuckling%26aufirst%3DC.%2BJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DDiscovery%2520of%2520a%2520potent%252C%2520cell%2520penetrant%252C%2520and%2520selective%2520P300%252FCBP-associated%2520factor%2520%2528PCAF%2529%252Fgeneral%2520control%2520nonderepressible%25205%2520%2528GCN5%2529%2520bromodomain%2520chemical%2520probe%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D695%26epage%3D709%26doi%3D10.1021%2Facs.jmedchem.6b01566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bassi, Z. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fillmore, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maller, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galwey, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waddington, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parravicini, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macmillan-Jones, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gongora, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span> <span> </span><span class="NLM_article-title">Modulating PCAF/GCN5 immune cell function through a PROTAC approach</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2862</span>– <span class="NLM_lpage">2867</span>, <span class="refDoi"> DOI: 10.1021/acschembio.8b00705</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b00705" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1OktL7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=2862-2867&author=Z.+I.+Bassiauthor=M.+C.+Fillmoreauthor=A.+H.+Miahauthor=T.+D.+Chapmanauthor=C.+Mallerauthor=E.+J.+Robertsauthor=L.+C.+Davisauthor=D.+E.+Lewisauthor=N.+W.+Galweyauthor=K.+E.+Waddingtonauthor=V.+Parraviciniauthor=A.+L.+Macmillan-Jonesauthor=C.+Gongoraauthor=P.+G.+Humphreysauthor=I.+Churcherauthor=R.+K.+Prinjhaauthor=D.+F.+Tough&title=Modulating+PCAF%2FGCN5+immune+cell+function+through+a+PROTAC+approach&doi=10.1021%2Facschembio.8b00705"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Modulating PCAF/GCN5 Immune Cell Function through a PROTAC Approach</span></div><div class="casAuthors">Bassi, Zuni I.; Fillmore, Martin C.; Miah, Afjal H.; Chapman, Trevor D.; Maller, Claire; Roberts, Emma J.; Davis, Lauren C.; Lewis, Darcy E.; Galwey, Nicholas W.; Waddington, Kirsty E.; Parravicini, Valentino; Macmillan-Jones, Abigail L.; Gongora, Celine; Humphreys, Philip G.; Churcher, Ian; Prinjha, Rab K.; Tough, David F.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2862-2867</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P300/CBP-assocd. factor (PCAF) and general control nonderepressible 5 (GCN5) are closely related epigenetic proteins, each contg. an acetyltransferase domain and a bromodomain.  Consistent with reported roles for these proteins in immune function, we find that PCAF-deficient macrophages exhibit a markedly reduced ability to produce cytokines upon stimulation with lipopolysaccharide (LPS).  Investigating the potential to target this pathway pharmacol., we show that chem. inhibition of the PCAF/GCN5 bromodomains is insufficient to recapitulate the diminished inflammatory response of PCAF-deficient immune cells.  However, by generating the first PCAF/GCN5 proteolysis targeting chimera (PROTAC), we identify small mols. able to degrade PCAF/GCN5 and to potently modulate the expression of multiple inflammatory mediators in LPS-stimulated macrophages and dendritic cells.  Our data illustrate the power of the PROTAC approach in the context of multidomain proteins, revealing a novel anti-inflammatory therapeutic opportunity for targeting PCAF/GCN5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUuwZ6JZtCirVg90H21EOLACvtfcHk0lhNd82FnagXUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1OktL7I&md5=6783f39397381536ed40ac9e1b20fa5b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b00705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b00705%26sid%3Dliteratum%253Aachs%26aulast%3DBassi%26aufirst%3DZ.%2BI.%26aulast%3DFillmore%26aufirst%3DM.%2BC.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DMaller%26aufirst%3DC.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DDavis%26aufirst%3DL.%2BC.%26aulast%3DLewis%26aufirst%3DD.%2BE.%26aulast%3DGalwey%26aufirst%3DN.%2BW.%26aulast%3DWaddington%26aufirst%3DK.%2BE.%26aulast%3DParravicini%26aufirst%3DV.%26aulast%3DMacmillan-Jones%26aufirst%3DA.%2BL.%26aulast%3DGongora%26aufirst%3DC.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DTough%26aufirst%3DD.%2BF.%26atitle%3DModulating%2520PCAF%252FGCN5%2520immune%2520cell%2520function%2520through%2520a%2520PROTAC%2520approach%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D2862%26epage%3D2867%26doi%3D10.1021%2Facschembio.8b00705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of potent and selective inhibitors of BRD7 and BRD9 bromodomains</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1381</span>– <span class="NLM_lpage">1386</span>, <span class="refDoi"> DOI: 10.1039/c5md00152h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1039%2Fc5md00152h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVSgtrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=1381-1386&author=D.+A.+Hayauthor=C.+M.+Rogersauthor=O.+Fedorovauthor=C.+Tallantauthor=S.+Martinauthor=O.+P.+Monteiroauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=C.+J.+Schofieldauthor=P.+E.+Brennan&title=Design+and+synthesis+of+potent+and+selective+inhibitors+of+BRD7+and+BRD9+bromodomains&doi=10.1039%2Fc5md00152h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Design and synthesis of potent and selective inhibitors of BRD7 and BRD9 bromodomains</span></div><div class="casAuthors">Hay, Duncan A.; Rogers, Catherine M.; Fedorov, Oleg; Tallant, Cynthia; Martin, Sarah; Monteiro, Octovia P.; Muller, Susanne; Knapp, Stefan; Schofield, Christopher J.; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1381-1386</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Emerging evidence suggests bromodomain-contg. proteins 7 and 9 (BRD7 and BRD9) have roles in the regulation of human transcription and disease including cancer.  We describe potent and selective inhibitors of the BRD7 and BRD9 bromodomains intended for use as tools to elucidate the biol. roles of BRD7 and BRD9 in healthy and diseased cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrynQ2AJnreDLVg90H21EOLACvtfcHk0lhzczQchS5V0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVSgtrvJ&md5=6a93d7300f7815eaf5ca654f873078d3</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1039%2Fc5md00152h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc5md00152h%26sid%3Dliteratum%253Aachs%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520potent%2520and%2520selective%2520inhibitors%2520of%2520BRD7%2520and%2520BRD9%2520bromodomains%26jtitle%3DMedChemComm%26date%3D2015%26volume%3D6%26spage%3D1381%26epage%3D1386%26doi%3D10.1039%2Fc5md00152h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lesniak, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selenski, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span> <span> </span><span class="NLM_article-title">Discovery and characterization of GSK2801, a selective chemical probe for the bromodomains BAZ2A and BAZ2B</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1410</span>– <span class="NLM_lpage">1424</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00209</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00209" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC2MXltFSlsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1410-1424&author=P.+Chenauthor=A.+Chaikuadauthor=P.+Bamboroughauthor=M.+Bantscheffauthor=C.+Bountraauthor=C.-w.+Chungauthor=O.+Fedorovauthor=P.+Grandiauthor=D.+Jungauthor=R.+Lesniakauthor=M.+Lindonauthor=S.+M%C3%BCllerauthor=M.+Philpottauthor=R.+Prinjhaauthor=C.+Rogersauthor=C.+Selenskiauthor=C.+Tallantauthor=T.+Wernerauthor=T.+M.+Willsonauthor=S.+Knappauthor=D.+H.+Drewry&title=Discovery+and+characterization+of+GSK2801%2C+a+selective+chemical+probe+for+the+bromodomains+BAZ2A+and+BAZ2B&doi=10.1021%2Facs.jmedchem.5b00209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Characterization of GSK2801, a Selective Chemical Probe for the Bromodomains BAZ2A and BAZ2B</span></div><div class="casAuthors">Chen, Peiling; Chaikuad, Apirat; Bamborough, Paul; Bantscheff, Marcus; Bountra, Chas; Chung, Chun-wa; Fedorov, Oleg; Grandi, Paola; Jung, David; Lesniak, Robert; Lindon, Matthew; Muller, Susanne; Philpott, Martin; Prinjha, Rab; Rogers, Catherine; Selenski, Carolyn; Tallant, Cynthia; Werner, Thilo; Willson, Timothy M.; Knapp, Stefan; Drewry, David H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1410-1424</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains are acetyl-lysine specific protein interaction domains that have recently emerged as a new target class for the development of inhibitors that modulate gene transcription.  The two closely related bromodomain contg. proteins BAZ2A and BAZ2B constitute the central scaffolding protein of the nucleolar remodeling complex (NoRC) that regulates the expression of noncoding RNAs.  However, BAZ2 bromodomains have low predicted druggability and so far no selective inhibitors have been published.  Here we report the development of GSK2801, a potent, selective and cell active acetyl-lysine competitive inhibitor of BAZ2A and BAZ2B bromodomains as well as the inactive control compd. GSK8573.  GSK2801 binds to BAZ2 bromodomains with dissocn. consts. (KD) of 136 and 257 nM for BAZ2B and BAZ2A, resp.  Crystal structures demonstrated a canonical acetyl-lysine competitive binding mode.  Cellular activity was demonstrated using fluorescent recovery after photobleaching (FRAP) monitoring displacement of GFP-BAZ2A from acetylated chromatin.  A pharmacokinetic study in mice showed that GSK2801 had reasonable in vivo exposure after oral dosing, with modest clearance and reasonable plasma stability.  Thus, GSK2801 represents a versatile tool compd. for cellular and in vivo studies to understand the role of BAZ2 bromodomains in chromatin biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprwxgNLFBYCrVg90H21EOLACvtfcHk0lhzczQchS5V0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXltFSlsbY%253D&md5=700e8966cb5932a1558a7d8cc2976b43</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00209%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DP.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DJung%26aufirst%3DD.%26aulast%3DLesniak%26aufirst%3DR.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DPrinjha%26aufirst%3DR.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DSelenski%26aufirst%3DC.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26atitle%3DDiscovery%2520and%2520characterization%2520of%2520GSK2801%252C%2520a%2520selective%2520chemical%2520probe%2520for%2520the%2520bromodomains%2520BAZ2A%2520and%2520BAZ2B%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1410%26epage%3D1424%26doi%3D10.1021%2Facs.jmedchem.5b00209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smaill, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery and optimization of 1-(1H-indol-1-yl)ethanone derivatives as CBP/EP300 bromodomain inhibitors for the treatment of castration-resistant prostate cancer</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>147</i></span>,  <span class="NLM_fpage">238</span>– <span class="NLM_lpage">252</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.01.</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1016%2Fj.ejmech.2018.01." title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=2018&pages=238-252&author=Q.+Xiangauthor=C.+Wangauthor=Y.+Zhangauthor=X.+Xueauthor=M.+Songauthor=C.+Zhangauthor=C.+Liauthor=C.+Wuauthor=K.+Liauthor=X.+Huiauthor=Y.+Zhouauthor=J.+B.+Smaillauthor=A.+V.+Pattersonauthor=D.+Wuauthor=K.+Dingauthor=Y.+Xu&title=Discovery+and+optimization+of+1-%281H-indol-1-yl%29ethanone+derivatives+as+CBP%2FEP300+bromodomain+inhibitors+for+the+treatment+of+castration-resistant+prostate+cancer&doi=10.1016%2Fj.ejmech.2018.01."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.01.&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.01.%26sid%3Dliteratum%253Aachs%26aulast%3DXiang%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DXue%26aufirst%3DX.%26aulast%3DSong%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DK.%26aulast%3DHui%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DSmaill%26aufirst%3DJ.%2BB.%26aulast%3DPatterson%26aufirst%3DA.%2BV.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DDing%26aufirst%3DK.%26aulast%3DXu%26aufirst%3DY.%26atitle%3DDiscovery%2520and%2520optimization%2520of%25201-%25281H-indol-1-yl%2529ethanone%2520derivatives%2520as%2520CBP%252FEP300%2520bromodomain%2520inhibitors%2520for%2520the%2520treatment%2520of%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D147%26spage%3D238%26epage%3D252%26doi%3D10.1016%2Fj.ejmech.2018.01." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Unzue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lolli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zechner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolbois, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caflisch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nevado, C.</span></span> <span> </span><span class="NLM_article-title">The ″gatekeeper″ residue influences the mode of binding of acetyl indoles to bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3087</span>– <span class="NLM_lpage">3097</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01757</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01757" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC28Xkt1CntL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=3087-3097&author=A.+Unzueauthor=H.+Zhaoauthor=G.+Lolliauthor=J.+Dongauthor=J.+Zhuauthor=M.+Zechnerauthor=A.+Dolboisauthor=A.+Caflischauthor=C.+Nevado&title=The+%E2%80%B3gatekeeper%E2%80%B3+residue+influences+the+mode+of+binding+of+acetyl+indoles+to+bromodomains&doi=10.1021%2Facs.jmedchem.5b01757"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">The "Gatekeeper" Residue Influences the Mode of Binding of Acetyl Indoles to Bromodomains</span></div><div class="casAuthors">Unzue, Andrea; Zhao, Hongtao; Lolli, Graziano; Dong, Jing; Zhu, Jian; Zechner, Melanie; Dolbois, Aymeric; Caflisch, Amedeo; Nevado, Cristina</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3087-3097</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small-mol. hits for the bromodomains of CREBBP and BAZ2B have been identified by scaffold hopping followed by docking of a set of ∼200 compds. contg. the acetyl indole scaffold.  Chem. synthesis of nearly 30 derivs. has resulted in ligands of representatives of three subfamilies of human bromodomains with favorable ligand efficiency.  The X-ray crystal structures of three different bromodomains (CREBBP, BAZ2B, and BRPF1b) in complex with acetyl indole derivs. reveal the influence of the gatekeeper residue on the orientation of small-mol. ligands in the acetyl lysine binding site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLvogt-C6f_LVg90H21EOLACvtfcHk0lhzczQchS5V0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xkt1CntL0%253D&md5=35b2490b53eb162d0009d68b8f11262e</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01757&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01757%26sid%3Dliteratum%253Aachs%26aulast%3DUnzue%26aufirst%3DA.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DLolli%26aufirst%3DG.%26aulast%3DDong%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DZechner%26aufirst%3DM.%26aulast%3DDolbois%26aufirst%3DA.%26aulast%3DCaflisch%26aufirst%3DA.%26aulast%3DNevado%26aufirst%3DC.%26atitle%3DThe%2520%25E2%2580%25B3gatekeeper%25E2%2580%25B3%2520residue%2520influences%2520the%2520mode%2520of%2520binding%2520of%2520acetyl%2520indoles%2520to%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D3087%26epage%3D3097%26doi%3D10.1021%2Facs.jmedchem.5b01757" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Steijvoort, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaval, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulago, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maes, B. U. W.</span></span> <span> </span><span class="NLM_article-title">Remote functionalization: palladium-catalyzed C5(sp3)-H arylation of 1-boc-3-aminopiperidine through the use of a bidentate directing group</span>. <i>ACS Catal.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">4486</span>– <span class="NLM_lpage">4490</span>, <span class="refDoi"> DOI: 10.1021/acscatal.6b00841</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscatal.6b00841" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVKmsL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=4486-4490&author=B.+F.+Van%0ASteijvoortauthor=N.+Kavalauthor=A.+A.+Kulagoauthor=B.+U.+W.+Maes&title=Remote+functionalization%3A+palladium-catalyzed+C5%28sp3%29-H+arylation+of+1-boc-3-aminopiperidine+through+the+use+of+a+bidentate+directing+group&doi=10.1021%2Facscatal.6b00841"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Remote Functionalization: Palladium-Catalyzed C5(sp3)-H Arylation of 1-Boc-3-aminopiperidine through the Use of a Bidentate Directing Group</span></div><div class="casAuthors">Van Steijvoort, Ben F.; Kaval, Nadya; Kulago, Artem A.; Maes, Bert U. W.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Catalysis</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4486-4490</span>CODEN:
                <span class="NLM_cas:coden">ACCACS</span>;
        ISSN:<span class="NLM_cas:issn">2155-5435</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A protocol for the Pd-catalyzed C5(sp3)-H arylation of readily available 1-Boc-3-(picolinoylamino)piperidine with iodo(hetero)arenes is reported.  The substrate can be obtained from a biorenewable feedstock, namely L-arginine.  The use of the correct N1 protective group is decisive to obtain arylation.  The addn. of a catalytic amt. of 2,6-dimethylbenzoic acid and performing the reaction at high concn. are important to achieve a high conversion and yield.  The procedure gives arylated 1-Boc-3-(picolinoylamino)piperidines in a regiospecific (C5) and stereospecific (cis) manner.  Orthogonal cleavage of the amide over the carbamate group allows one to further selectively derivatize the amino moieties of the piperidine scaffold.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8Ay5gpO-0I7Vg90H21EOLACvtfcHk0lgqFBIFB65r_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVKmsL7E&md5=cc0186e8b700e9fca735b6d077d28c79</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facscatal.6b00841&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscatal.6b00841%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BSteijvoort%26aufirst%3DB.%2BF.%26aulast%3DKaval%26aufirst%3DN.%26aulast%3DKulago%26aufirst%3DA.%2BA.%26aulast%3DMaes%26aufirst%3DB.%2BU.%2BW.%26atitle%3DRemote%2520functionalization%253A%2520palladium-catalyzed%2520C5%2528sp3%2529-H%2520arylation%2520of%25201-boc-3-aminopiperidine%2520through%2520the%2520use%2520of%2520a%2520bidentate%2520directing%2520group%26jtitle%3DACS%2520Catal.%26date%3D2016%26volume%3D6%26spage%3D4486%26epage%3D4490%26doi%3D10.1021%2Facscatal.6b00841" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresini, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burdick, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakin, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duplessis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Good, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span> <span> </span><span class="NLM_article-title">Diving into the water: inducible binding conformations for BRD4, TAF1(2), BRD9, and CECR2 bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">5391</span>– <span class="NLM_lpage">5402</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00264</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00264" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC28XosFSnsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5391-5402&author=T.+D.+Crawfordauthor=V.+Tsuiauthor=E.+M.+Flynnauthor=S.+Wangauthor=A.+M.+Taylorauthor=A.+C%C3%B4t%C3%A9author=J.+E.+Audiaauthor=M.+H.+Beresiniauthor=D.+J.+Burdickauthor=R.+Cummingsauthor=L.+A.+Dakinauthor=M.+Duplessisauthor=A.+C.+Goodauthor=M.+C.+Hewittauthor=H.-R.+Huangauthor=H.+Jayaramauthor=J.+R.+Kieferauthor=Y.+Jiangauthor=J.+Murrayauthor=C.+G.+Nasveschukauthor=E.+Pardoauthor=F.+Poyauthor=F.+A.+Romeroauthor=Y.+Tangauthor=J.+Wangauthor=Z.+Xuauthor=L.+E.+Zawadzkeauthor=X.+Zhuauthor=B.+K.+Albrechtauthor=S.+R.+Magnusonauthor=S.+Bellonauthor=A.+G.+Cochran&title=Diving+into+the+water%3A+inducible+binding+conformations+for+BRD4%2C+TAF1%282%29%2C+BRD9%2C+and+CECR2+bromodomains&doi=10.1021%2Facs.jmedchem.6b00264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Diving into the Water: Inducible Binding Conformations for BRD4, TAF1(2), BRD9, and CECR2 Bromodomains</span></div><div class="casAuthors">Crawford, Terry D.; Tsui, Vickie; Flynn, E. Megan; Wang, Shumei; Taylor, Alexander M.; Cote, Alexandre; Audia, James E.; Beresini, Maureen H.; Burdick, Daniel J.; Cummings, Richard; Dakin, Les A.; Duplessis, Martin; Good, Andrew C.; Hewitt, Michael C.; Huang, Hon-Ren; Jayaram, Hariharan; Kiefer, James R.; Jiang, Ying; Murray, Jeremy; Nasveschuk, Christopher G.; Pardo, Eneida; Poy, Florence; Romero, F. Anthony; Tang, Yong; Wang, Jian; Xu, Zhaowu; Zawadzke, Laura E.; Zhu, Xiaoyu; Albrecht, Brian K.; Magnuson, Steven R.; Bellon, Steve; Cochran, Andrea G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5391-5402</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The biol. role played by non-BET bromodomains remains poorly understood, and it is therefore imperative to identify potent and highly selective inhibitors to effectively explore the biol. of individual bromodomain proteins.  A ligand-efficient nonselective bromodomain inhibitor was identified from a 6-Me pyrrolopyridone fragment.  Small hydrophobic substituents replacing the N-Me group were designed directing toward the conserved bromodomain water pocket, and two distinct binding conformations were then obsd.  The substituents either directly displaced and rearranged the conserved solvent network, as in BRD4(1) and TAF1(2), or induced a narrow hydrophobic channel adjacent to the lipophilic shelf, as in BRD9 and CECR2.  The preference of distinct substituents for individual bromodomains provided selectivity handles useful for future lead optimization efforts for selective BRD9, CECR2, and TAF1(2) inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTLnuVCIKdHbVg90H21EOLACvtfcHk0lgqFBIFB65r_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosFSnsbs%253D&md5=bf92e94c5fd3ee36bd8441294ca85185</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00264%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3DBurdick%26aufirst%3DD.%2BJ.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DDuplessis%26aufirst%3DM.%26aulast%3DGood%26aufirst%3DA.%2BC.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DHuang%26aufirst%3DH.-R.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZawadzke%26aufirst%3DL.%2BE.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DMagnuson%26aufirst%3DS.%2BR.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26atitle%3DDiving%2520into%2520the%2520water%253A%2520inducible%2520binding%2520conformations%2520for%2520BRD4%252C%2520TAF1%25282%2529%252C%2520BRD9%252C%2520and%2520CECR2%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5391%26epage%3D5402%26doi%3D10.1021%2Facs.jmedchem.6b00264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodroofe, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwinn, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Méndez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">NanoBRET--a novel BRET platform for the analysis of protein-protein interactions</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1797</span>– <span class="NLM_lpage">1804</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00143</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00143" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovFKkt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1797-1804&author=T.+Machleidtauthor=C.+C.+Woodroofeauthor=M.+K.+Schwinnauthor=J.+M%C3%A9ndezauthor=M.+B.+Robersauthor=K.+Zimmermanauthor=P.+Ottoauthor=D.+L.+Danielsauthor=T.+A.+Kirklandauthor=K.+V.+Wood&title=NanoBRET%2D%2Da+novel+BRET+platform+for+the+analysis+of+protein-protein+interactions&doi=10.1021%2Facschembio.5b00143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">NanoBRET-A Novel BRET Platform for the Analysis of Protein-Protein Interactions</span></div><div class="casAuthors">Machleidt, Thomas; Woodroofe, Carolyn C.; Schwinn, Marie K.; Mendez, Jacqui; Robers, Matthew B.; Zimmerman, Kris; Otto, Paul; Daniels, Danette L.; Kirkland, Thomas A.; Wood, Keith V.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1797-1804</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dynamic interactions between proteins comprise a key mechanism for temporal control of cellular function and thus hold promise for development of novel drug therapies.  It remains tech. challenging, however, to quant. characterize these interactions within the biol. relevant context of living cells.  Although, bioluminescence resonance energy transfer (BRET) has often been used for this purpose, its general applicability has been hindered by limited sensitivity and dynamic range.  We have addressed this by combining an extremely bright luciferase (Nanoluc) with a means for tagging intracellular proteins with a long-wavelength fluorophore (HaloTag).  The small size (19 kDa), high emission intensity, and relatively narrow spectrum (460 nm peak intensity) make Nanoluc luciferase well suited as an energy donor.  By selecting an efficient red-emitting fluorophore (635 nm peak intensity) for attachment onto the HaloTag, an overall spectral sepn. exceeding 175 nm was achieved.  This combination of greater light intensity with improved spectral resoln. results in substantially increased detection sensitivity and dynamic range over current BRET technologies.  Enhanced performance is demonstrated using several established model systems, as well as the ability to image BRET in individual cells.  The capabilities are further exhibited in a novel assay developed for analyzing the interactions of bromodomain proteins with chromatin in living cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6-h9HlReNabVg90H21EOLACvtfcHk0ljp-S2TS0IMng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovFKkt7c%253D&md5=9f5f773b6cd14c40074a2ffbd6472008</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00143%26sid%3Dliteratum%253Aachs%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DWoodroofe%26aufirst%3DC.%2BC.%26aulast%3DSchwinn%26aufirst%3DM.%2BK.%26aulast%3DM%25C3%25A9ndez%26aufirst%3DJ.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DNanoBRET--a%2520novel%2520BRET%2520platform%2520for%2520the%2520analysis%2520of%2520protein-protein%2520interactions%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1797%26epage%3D1804%26doi%3D10.1021%2Facschembio.5b00143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niesen, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span> <span> </span><span class="NLM_article-title">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">2212</span>– <span class="NLM_lpage">2221</span>, <span class="refDoi"> DOI: 10.1038/nprot.2007.321</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1038%2Fnprot.2007.321" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=17853878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2212-2221&author=F.+H.+Niesenauthor=H.+Berglundauthor=M.+Vedadi&title=The+use+of+differential+scanning+fluorimetry+to+detect+ligand+interactions+that+promote+protein+stability&doi=10.1038%2Fnprot.2007.321"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span></div><div class="casAuthors">Niesen, Frank H.; Berglund, Helena; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2212-2221</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Differential scanning fluorometry (DSF) is a rapid and inexpensive screening method to identify low-mol.-wt. ligands that bind and stabilize purified proteins.  The temp. at which a protein unfolds is measured by an increase in the fluorescence of a dye with affinity for hydrophobic parts of the protein, which are exposed as the protein unfolds.  A simple fitting procedure allows quick calcn. of the transition midpoint; the difference in the temp. of this midpoint in the presence and absence of ligand is related to the binding affinity of the small mol., which can be a low-mol.-wt. compd., a peptide or a nucleic acid.  DSF is best performed using a conventional real-time PCR instrument.  Ligand solns. from a storage plate are added to a soln. of protein and dye, distributed into the wells of the PCR plate and fluorescence intensity measured as the temp. is raised gradually.  Results can be obtained in a single day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomViqp6Tq7krVg90H21EOLACvtfcHk0ljp-S2TS0IMng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N&md5=8a74dbb348ab81179614ab4ae5797d94</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.321%26sid%3Dliteratum%253Aachs%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DBerglund%26aufirst%3DH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DThe%2520use%2520of%2520differential%2520scanning%2520fluorimetry%2520to%2520detect%2520ligand%2520interactions%2520that%2520promote%2520protein%2520stability%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D2%26spage%3D2212%26epage%3D2221%26doi%3D10.1038%2Fnprot.2007.321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickle, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D.</span></span> <span> </span><span class="NLM_article-title">BROMOscan a high throughput, quantitative ligand binding platform identifies best-in-class bromodomain inhibitors from a screen of mature compounds targeting other protein classes</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">4238</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.am2013-4238</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1158%2F1538-7445.am2013-4238" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=4238&author=E.+Quinnauthor=L.+Wodickaauthor=P.+Ciceriauthor=G.+Pallaresauthor=E.+Pickleauthor=A.+Torreyauthor=M.+Floydauthor=J.+Huntauthor=D.+Treiber&title=BROMOscan+a+high+throughput%2C+quantitative+ligand+binding+platform+identifies+best-in-class+bromodomain+inhibitors+from+a+screen+of+mature+compounds+targeting+other+protein+classes&doi=10.1158%2F1538-7445.am2013-4238"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.am2013-4238&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.am2013-4238%26sid%3Dliteratum%253Aachs%26aulast%3DQuinn%26aufirst%3DE.%26aulast%3DWodicka%26aufirst%3DL.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPickle%26aufirst%3DE.%26aulast%3DTorrey%26aufirst%3DA.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%26aulast%3DTreiber%26aufirst%3DD.%26atitle%3DBROMOscan%2520a%2520high%2520throughput%252C%2520quantitative%2520ligand%2520binding%2520platform%2520identifies%2520best-in-class%2520bromodomain%2520inhibitors%2520from%2520a%2520screen%2520of%2520mature%2520compounds%2520targeting%2520other%2520protein%2520classes%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D4238%26doi%3D10.1158%2F1538-7445.am2013-4238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1016/s0076-6879(97)76066-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode&doi=10.1016%2Fs0076-6879%2897%2976066-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0ljp-S2TS0IMng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2Fs0076-6879%2897%2976066-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Xds</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2Fs0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lhClLCwMgsIQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2Fs0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/s0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2Fs0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lhClLCwMgsIQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2Fs0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/s0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2Fs0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lhClLCwMgsIQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2Fs0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustyakimov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urzhumtsev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">Towards automated crystallographic structure refinement with phenix.refine</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">352</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1107/s0907444912001308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1107%2Fs0907444912001308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=22505256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=352-367&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=N.+Echolsauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=M.+Mustyakimovauthor=T.+C.+Terwilligerauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=Towards+automated+crystallographic+structure+refinement+with+phenix.refine&doi=10.1107%2Fs0907444912001308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Towards automated crystallographic structure refinement with phenix.refine</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Echols, Nathaniel; Headd, Jeffrey J.; Moriarty, Nigel W.; Mustyakimov, Marat; Terwilliger, Thomas C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">352-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.refine is a program within the PHENIX package that supports crystallog. structure refinement against exptl. data with a wide range of upper resoln. limits using a large repertoire of model parameterizations.  It has several automation features and is also highly flexible.  Several hundred parameters enable extensive customizations for complex use cases.  Multiple user-defined refinement strategies can be applied to specific parts of the model in a single refinement run.  An intuitive graphical user interface is available to guide novice users and to assist advanced users in managing refinement projects.  X-ray or neutron diffraction data can be used sep. or jointly in refinement. phenix.refine is tightly integrated into the PHENIX suite, where it serves as a crit. component in automated model building, final structure refinement, structure validation and deposition to the wwPDB.  This paper presents an overview of the major phenix.refine features, with extensive literature refs. for readers interested in more detailed discussions of the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnhn9QwIEkTbVg90H21EOLACvtfcHk0licoKKeHF93_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D&md5=1cc7da2cea396f1a75646ea709140cf1</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2Fs0907444912001308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444912001308%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DMustyakimov%26aufirst%3DM.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DTowards%2520automated%2520crystallographic%2520structure%2520refinement%2520with%2520phenix.refine%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D352%26epage%3D367%26doi%3D10.1107%2Fs0907444912001308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0licoKKeHF93_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomenius, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munchhof, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klaus, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riera, T. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span> <span> </span><span class="NLM_article-title">Novel oxindole sulfonamides and sulfamides: EPZ031686, the first orally bioavailable small molecule SMYD3 inhibitor</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">134</span>– <span class="NLM_lpage">138</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00272</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00272" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVahsrnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=134-138&author=L.+H.+Mitchellauthor=P.+A.+Boriack-Sjodinauthor=S.+Smithauthor=M.+Thomeniusauthor=N.+Riouxauthor=M.+Munchhofauthor=J.+E.+Millsauthor=C.+Klausauthor=J.+Totmanauthor=T.+V.+Rieraauthor=A.+Raimondiauthor=S.+L.+Jacquesauthor=K.+Westauthor=M.+Foleyauthor=N.+J.+Watersauthor=K.+W.+Kuntzauthor=T.+J.+Wigleauthor=M.+P.+Scottauthor=R.+A.+Copelandauthor=J.+J.+Smithauthor=R.+Chesworth&title=Novel+oxindole+sulfonamides+and+sulfamides%3A+EPZ031686%2C+the+first+orally+bioavailable+small+molecule+SMYD3+inhibitor&doi=10.1021%2Facsmedchemlett.5b00272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Oxindole Sulfonamides and Sulfamides: EPZ031686, the First Orally Bioavailable Small Molecule SMYD3 Inhibitor</span></div><div class="casAuthors">Mitchell, Lorna H.; Boriack-Sjodin, P. Ann; Smith, Sherri; Thomenius, Michael; Rioux, Nathalie; Munchhof, Michael; Mills, James E.; Klaus, Christine; Totman, Jennifer; Riera, Thomas V.; Raimondi, Alejandra; Jacques, Suzanne L.; West, Kip; Foley, Megan; Waters, Nigel J.; Kuntz, Kevin W.; Wigle, Tim J.; Scott, Margaret Porter; Copeland, Robert A.; Smith, Jesse J.; Chesworth, Richard</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">134-138</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SMYD3 has been implicated in a range of cancers; however, until now no potent selective small mol. inhibitors have been available for target validation studies.  A novel oxindole series of SMYD3 inhibitors was identified through screening of the Epizyme proprietary histone methyltransferase-biased library.  Potency optimization afforded two tool compds., sulfonamide EPZ031686 and sulfamide EPZ030456, with cellular potency at a level sufficient to probe the in vitro biol. of SMYD3 inhibition.  EPZ031686 shows good bioavailability following oral dosing in mice making it a suitable tool for potential in vivo target validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQtO_uOhxeYrVg90H21EOLACvtfcHk0licoKKeHF93_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVahsrnJ&md5=3fd7612cb086796a0bcc5c6e0c1f1352</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00272%26sid%3Dliteratum%253Aachs%26aulast%3DMitchell%26aufirst%3DL.%2BH.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DSmith%26aufirst%3DS.%26aulast%3DThomenius%26aufirst%3DM.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DMunchhof%26aufirst%3DM.%26aulast%3DMills%26aufirst%3DJ.%2BE.%26aulast%3DKlaus%26aufirst%3DC.%26aulast%3DTotman%26aufirst%3DJ.%26aulast%3DRiera%26aufirst%3DT.%2BV.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DWest%26aufirst%3DK.%26aulast%3DFoley%26aufirst%3DM.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26atitle%3DNovel%2520oxindole%2520sulfonamides%2520and%2520sulfamides%253A%2520EPZ031686%252C%2520the%2520first%2520orally%2520bioavailable%2520small%2520molecule%2520SMYD3%2520inhibitor%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D134%26epage%3D138%26doi%3D10.1021%2Facsmedchemlett.5b00272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 2 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alessandra  Cipriano</span>, <span class="hlFld-ContribAuthor ">Gianluca  Sbardella</span>, <span class="hlFld-ContribAuthor ">Alessio  Ciulli</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting epigenetic reader domains by chemical biology. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2020,</strong> <em>57 </em>, 82-94. <a href="https://doi.org/10.1016/j.cbpa.2020.05.006" title="DOI URL">https://doi.org/10.1016/j.cbpa.2020.05.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2020.05.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2020.05.006%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DTargeting%252Bepigenetic%252Breader%252Bdomains%252Bby%252Bchemical%252Bbiology%26aulast%3DCipriano%26aufirst%3DAlessandra%26date%3D2020%26volume%3D57%26spage%3D82%26epage%3D94" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cristiano  Bolchi</span>, <span class="hlFld-ContribAuthor ">Francesco  Bavo</span>, <span class="hlFld-ContribAuthor ">Rebecca  Appiani</span>, <span class="hlFld-ContribAuthor ">Gabriella  Roda</span>, <span class="hlFld-ContribAuthor ">Marco  Pallavicini</span>. </span><span class="cited-content_cbyCitation_article-title">1,4-Benzodioxane, an evergreen, versatile scaffold in medicinal chemistry: A review of its recent applications in drug design. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>200 </em>, 112419. <a href="https://doi.org/10.1016/j.ejmech.2020.112419" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112419</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112419%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3D1%25252C4-Benzodioxane%25252C%252Ban%252Bevergreen%25252C%252Bversatile%252Bscaffold%252Bin%252Bmedicinal%252Bchemistry%25253A%252BA%252Breview%252Bof%252Bits%252Brecent%252Bapplications%252Bin%252Bdrug%252Bdesign%26aulast%3DBolchi%26aufirst%3DCristiano%26date%3D2020%26volume%3D200%26spage%3D112419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00096&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative PCAF BRD inhibitors publicly reported.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00096&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Three active compounds found by VS and subsequent bioactivity evaluation. (A) Chemical structures of <b>10</b>, <b>11</b>, and <b>12</b>. (B) Isothermal titration data of the interaction of hit compounds (for <b>10</b> and <b>11</b>, 50 μM ligand and 500 μM PCAF BRD were used; for <b>12</b>, 20 μM ligand and 200 μM PCAF BRD were used) with PCAF BRD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00096&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cocrystal structure of PCAF/<b>12</b> guided the structural optimization. (A) Cocrystal structure of compound <b>12</b> (yellow sticks) in complex with PCAF BRD (pale cyan cartoon) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6J3O">6J3O</a>). Related residues are shown as green sticks. (B) Experimental electron density map (2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub>) contoured at 1σ around the hit <b>12</b>. (C) Focused regions in the structural optimization and SAR analyses.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00096&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of (<i>R</i>,<i>R</i>)-<b>24a</b>, (<i>S</i>,<i>S</i>)-<b>24a</b>, <i>cis</i>-<b>24b</b>, <i>endo</i>-<b>24c–e</b>, <i>endo</i>-<b>25a</b>, <i>exo</i>-<b>25a</b>, and <i>endo</i>-<b>26a–b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00096&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 1 M NaOH (aq), 100 °C, 16 h, 90%; (b) Boc<sub>2</sub>O, TEA, DMAP, DMF, rt, 82%; (c) iodomethane, NaH, anhydrous DMF, 40 °C, overnight; (d) TFA, DCM, rt, 46% over two steps; (e) picolinic acid, HATU, DIPEA, DCM, rt, 83–88%; (f) iodobenzene, Pd(OAc)<sub>2</sub>, Ag<sub>2</sub>CO<sub>3</sub>, 2,6-dimethylbenzoic acid, <i>t</i>-BuOH, 120 °C, 24 h, 62–77%, >99% ee; (g) TFA, DCM, rt; (h) 37% formaldehyde, NaBH(OAc)<sub>3</sub>, AcOH, DCM, rt, 8 h, 84%; (i) NaOH, <i>i</i>-PrOH, 85 °C, 18 h, 78–85%; (j) <b>16</b>, primary amines, DIPEA, NMP, 150 °C, 2–5 h, 12–37%; (k) (<i>R</i>)-2-phenyloxirane or (<i>R</i>)-2-bromo-1-(pyridin-2-yl)ethan-1-ol, TEA, MeCN, 70 °C, overnight, 47–58%.</p></p></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of (<i>R</i>,<i>R</i>)-<b>32a–f</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00096&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) haloalkanes, NaH, anhydrous DMF, 50 °C, overnight, 34–78%; (b) TFA, DCM, rt; (c) 1 M NaOH (aq), 100 °C, 95%; (d) (<i>R</i>,<i>R</i>)-<b>20a</b>, DIPEA, NMP, 150 °C, 2 h, 13–47%.</p></p></figure><figure data-id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds (<i>R</i>,<i>R</i>)-<b>36a–n</b> and (<i>S</i>,<i>S</i>)-<b>36n</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00096&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) aryl iodides, Pd(OAc)<sub>2</sub>, Ag<sub>2</sub>CO<sub>3</sub>, 2,6-dimethylbenzoic acid, <i>t</i>-BuOH, 120 °C, 24 h, 52–76%, >99% ee; (b) TFA, DCM, rt; (c) haloalkanes, TEA, MeCN, 80 °C, overnight, 76%; (d) 37% formaldehyde (aq), NaBH(OAc)<sub>3</sub>, AcOH, DCM, rt, 73%; (e) NaOH, <i>i</i>-PrOH, 85 °C, 18 h; (f) <b>16</b>, DIPEA, NMP, 150 °C, 2 h, 11–26%; (g) (<i>R</i>,<i>R</i>)-<b>36i</b>, 10% Pd/C, H<sub>2</sub>, MeOH, rt, 65%.</p></p></figure><figure data-id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Identification of (<i>R</i>,<i>R</i>)-<b>36n</b> as a potent inhibitor of PCAF BRD. (A) Chemical structure of (<i>R</i>,<i>R</i>)-<b>36n</b>. (B) HTRF assays of (<i>R</i>,<i>R</i>)-<b>36n</b> and <b><i>L</i>-Moses</b> on GST-PCAF. (C–E) ITC titration curves for the binding of (<i>R</i>,<i>R</i>)-<b>36n</b> (10 μM, 200 μL in cell) to PCAF (C, 100 μM, 40 μL in syringe), GCN5 (D, 160 μM, 40 μL in syringe), and FALZ (E, 125 μM, 40 μL in syringe).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00096&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Selectivity of (<i>R</i>,<i>R</i>)-<b>36n</b>. (A) Thermal shift analysis of (<i>R</i>,<i>R</i>)-<b>36n</b> (10 μM) against 12 BRDs, which belong to six distinct BRD subfamilies. The concentration of BRDs used was 2 μM. The heat map shows the relative Δ<i>T</i><sub>m</sub>, where the red color indicates a positive Δ<i>T</i><sub>m</sub> value and the green color indicates a negative Δ<i>T</i><sub>m</sub> value. (B) DiscoverX BROMOscan BRD cross-screen of (<i>R</i>,<i>R</i>)-<b>36n</b> at 1 μM. Binding activity is expressed as a percentage of the control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00096&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Structure and binding mode of (<i>R</i>,<i>R</i>)-<b>36n</b>. (A) X-ray cocrystal structure of (<i>R</i>,<i>R</i>)-<b>36n</b> (green sticks) bound to GCN5 BRD (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6J3P">6J3P</a>), with a conserved water network depicted (red spheres); yellow dashed lines indicate hydrogen bonds or salt bridges. (B) Surface view of the complex of GCN5 and (<i>R</i>,<i>R</i>)-<b>36n</b>. Mesh: 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> omitted electron density map contoured at 1.0σ.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00096&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. NanoBRET dose–response curves of (<i>R</i>,<i>R</i>)-<b>36n</b> and (<i>S</i>,<i>S</i>)-<b>36n</b> after 24 h treatment using N-terminal-nanoLuc-PCAF and C-terminal HaloTag-H3.3 in HEK293T cells (graph represents <i>n</i> = 2 biological replicates).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00096&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/medium/jm-2019-000965_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Effect of (<i>R</i>,<i>R</i>)-<b>36n</b> on global gene expression. (A) Venn diagram showing a number of genes significantly up- or downregulated greater than 1.5 fold (<i>p</i> < 0.05) following (<i>R</i>,<i>R</i>)-<b>36n</b>, (<i>S</i>,<i>S</i>)-<b>36n</b> or DMSO treatment for 48 h in MEF cells. The green circle represents the number of influenced genes by (<i>R</i>,<i>R</i>)-<b>36n</b> relative to blank control, the red circle means the differential gene of (<i>S</i>,<i>S</i>)-<b>36n</b> relative to the blank control. (B) Hierarchical clustering heat map of selected genes showing differential regulation by (<i>R</i>,<i>R</i>)-<b>36n</b> and (<i>S</i>,<i>S</i>)-<b>36n</b> [genes up- or downregulated by >1.5-fold and <i>p</i> < 0.05 of (<i>R</i>,<i>R</i>)-<b>36n</b>]. Data from three donors are shown for each experiment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00096/20190502/images/large/jm-2019-000965_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00096&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i85">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46758" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46758" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 42 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.-P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+M%C3%BCllerauthor=T.+Pawsonauthor=A.-C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0lh9qpSr5EeDJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">356</span>, <span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0lig1tquz5wzBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S.</span></span> <span> </span><span class="NLM_article-title">Disrupting acetyl-lysine recognition: progress in the development of bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1271</span>– <span class="NLM_lpage">1298</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01514</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01514" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1271-1298&author=F.+A.+Romeroauthor=A.+M.+Taylorauthor=T.+D.+Crawfordauthor=V.+Tsuiauthor=A.+C%C3%B4t%C3%A9author=S.+Magnuson&title=Disrupting+acetyl-lysine+recognition%3A+progress+in+the+development+of+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b01514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting Acetyl-Lysine Recognition: Progress in the Development of Bromodomain Inhibitors</span></div><div class="casAuthors">Romero, F. Anthony; Taylor, Alexander M.; Crawford, Terry D.; Tsui, Vickie; Cote, Alexandre; Magnuson, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1271-1298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains, small protein modules that recognize acetylated lysine on histones, play a significant role in the epigenome, where they function as "readers" that ultimately det. the functional outcome of the post-translational modification.  Because the initial discovery of selective BET inhibitors have helped define the role of that protein family in oncol. and inflammation, BET bromodomains have continued to garner the most attention of any other bromodomain.  More recently, non-BET bromodomain inhibitors that are potent and selective have been disclosed for ATAD2, CBP, BRD7/9, BRPF, BRPF/TRIM24, CECR2, SMARCA4, and BAZ2A/B.  Such novel inhibitors can be used to probe the physiol. function of these non-BET bromodomains and further understanding of their role in certain disease states.  Here, we provide an update to the progress in identifying selective bromodomain inhibitors and their use as biol. tools, as well as our perspective on the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgbRrKGlHaVLVg90H21EOLACvtfcHk0lgCvthHr3_8Mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO&md5=260352f61301aa88aad48c0b63e6152f</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01514%26sid%3Dliteratum%253Aachs%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DDisrupting%2520acetyl-lysine%2520recognition%253A%2520progress%2520in%2520the%2520development%2520of%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1271%26epage%3D1298%26doi%3D10.1021%2Facs.jmedchem.5b01514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Drug discovery targeting bromodomain-containing protein 4</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4533</span>– <span class="NLM_lpage">4558</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01761</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01761" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4533-4558&author=Z.+Liuauthor=P.+Wangauthor=H.+Chenauthor=E.+A.+Woldauthor=B.+Tianauthor=A.+R.+Brasierauthor=J.+Zhou&title=Drug+discovery+targeting+bromodomain-containing+protein+4&doi=10.1021%2Facs.jmedchem.6b01761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Discovery Targeting Bromodomain-Containing Protein 4</span></div><div class="casAuthors">Liu, Zhiqing; Wang, Pingyuan; Chen, Haiying; Wold, Eric A.; Tian, Bing; Brasier, Allan R.; Zhou, Jia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4533-4558</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  BRD4, the most extensively studied member of BET family, is an epigenetic regulator that localizes to DNA via binding acetylated histones and controls the expression of therapeutically important gene regulatory networks through recruiting transcription factors to form mediator complexes, phosphorylating RNA polymerase II and by its intrinsic histone acetyltransferase activity.  Disrupting the protein-protein interactions between BRD4 and acetyl-lysine has been shown to effectively block cell proliferation in cancer, cytokine prodn. in acute inflammation, etc.  To date, significant efforts have been devoted to the development of BRD4 inhibitors, and consequently, a dozen have progressed into human clin. trials.  Herein, the authors summarize the advances in drug discovery and development of BRD4 inhibitors by focusing on their chemotypes, in vitro and in vivo activity, selectivity, relevant mechanisms of action and therapeutic potential.  Opportunities and challenges to achieve selective and efficacious BRD4 inhibitors as a viable therapeutic strategy for human diseases are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6IAERH4iggrVg90H21EOLACvtfcHk0lgCvthHr3_8Mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D&md5=6d7ba5844073fac8c916b1249af402e5</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01761%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWold%26aufirst%3DE.%2BA.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDrug%2520discovery%2520targeting%2520bromodomain-containing%2520protein%25204%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4533%26epage%3D4558%26doi%3D10.1021%2Facs.jmedchem.6b01761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giotopoulos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, W.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitski, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huthmacher, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gudgin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auger, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doehner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delwel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burnett, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntly, B. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span> <span> </span><span class="NLM_article-title">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>478</i></span>,  <span class="NLM_fpage">529</span>– <span class="NLM_lpage">533</span>, <span class="refDoi"> DOI: 10.1038/nature10509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1038%2Fnature10509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=21964340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=529-533&author=M.+A.+Dawsonauthor=R.+K.+Prinjhaauthor=A.+Dittmannauthor=G.+Giotopoulosauthor=M.+Bantscheffauthor=W.-I.+Chanauthor=S.+C.+Robsonauthor=C.-w.+Chungauthor=C.+Hopfauthor=M.+M.+Savitskiauthor=C.+Huthmacherauthor=E.+Gudginauthor=D.+Lugoauthor=S.+Beinkeauthor=T.+D.+Chapmanauthor=E.+J.+Robertsauthor=P.+E.+Sodenauthor=K.+R.+Augerauthor=O.+Mirguetauthor=K.+Doehnerauthor=R.+Delwelauthor=A.+K.+Burnettauthor=P.+Jeffreyauthor=G.+Drewesauthor=K.+Leeauthor=B.+J.+P.+Huntlyauthor=T.+Kouzarides&title=Inhibition+of+BET+recruitment+to+chromatin+as+an+effective+treatment+for+MLL-fusion+leukaemia&doi=10.1038%2Fnature10509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span></div><div class="casAuthors">Dawson, Mark A.; Prinjha, Rab K.; Dittman, Antje; Giotopoulos, George; Bantscheff, Marcus; Chan, Wai-In; Robson, Samuel C.; Chung, Chun-wa; Hopf, Carsten; Savitski, Mikhail M.; Huthmacher, Carola; Gudgin, Emma; Lugo, Dave; Beinke, Soren; Chapman, Trevor D.; Roberts, Emma J.; Soden, Peter E.; Auger, Kurt R.; Mirguet, Olivier; Doehner, Konstanze; Delwel, Ruud; Burnett, Alan K.; Jeffrey, Phillip; Drewes, Gerard; Lee, Kevin; Huntly, Brian J. P.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">529-533</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recurrent chromosomal translocations involving the mixed lineage leukemia (MLL) gene initiate aggressive forms of leukemia, which are often refractory to conventional therapies.  Many MLL-fusion partners are members of the super elongation complex (SEC), a crit. regulator of transcriptional elongation, suggesting that aberrant control of this process has an important role in leukemia induction.  Here we use a global proteomic strategy to demonstrate that MLL fusions, as part of SEC and the polymerase-assocd. factor complex (PAFc), are assocd. with the BET family of acetyl-lysine recognizing, chromatin adaptor' proteins.  These data provided the basis for therapeutic intervention in MLL-fusion leukemia, via the displacement of the BET family of proteins from chromatin.  We show that a novel small mol. inhibitor of the BET family, GSK1210151A (I-BET151), has profound efficacy against human and murine MLL-fusion leukemic cell lines, through the induction of early cell cycle arrest and apoptosis.  I-BET151 treatment in two human leukemia cell lines with different MLL fusions alters the expression of a common set of genes whose function may account for these phenotypic changes.  The mode of action of I-BET151 is, at least in part, due to the inhibition of transcription at key genes (BCL2, C-MYC and CDK6) through the displacement of BRD3/4, PAFc and SEC components from chromatin.  In vivo studies indicate that I-BET151 has significant therapeutic value, providing survival benefit in two distinct mouse models of murine MLL-AF9 and human MLL-AF4 leukemia.  Finally, the efficacy of I-BET151 against human leukemia stem cells is demonstrated, providing further evidence of its potent therapeutic potential.  These findings establish the displacement of BET proteins from chromatin as a promising epigenetic therapy for these aggressive leukemias.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4FbgJpjftv7Vg90H21EOLACvtfcHk0li-7MBxvAjduA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO&md5=dffb4b81da4ca8291c27139d0df86324</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature10509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10509%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DDittmann%26aufirst%3DA.%26aulast%3DGiotopoulos%26aufirst%3DG.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DW.-I.%26aulast%3DRobson%26aufirst%3DS.%2BC.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DHopf%26aufirst%3DC.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DHuthmacher%26aufirst%3DC.%26aulast%3DGudgin%26aufirst%3DE.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DAuger%26aufirst%3DK.%2BR.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DDoehner%26aufirst%3DK.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DBurnett%26aufirst%3DA.%2BK.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%2BP.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DInhibition%2520of%2520BET%2520recruitment%2520to%2520chromatin%2520as%2520an%2520effective%2520treatment%2520for%2520MLL-fusion%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D529%26epage%3D533%26doi%3D10.1038%2Fnature10509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delmore, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Issa, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahl, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobs, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kastritis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilpatrick, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paranal, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schinzel, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heffernan, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergsagel, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghobrial, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsiades, C. S.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition as a therapeutic strategy to target c-Myc</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>146</i></span>,  <span class="NLM_fpage">904</span>– <span class="NLM_lpage">917</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.08.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1016%2Fj.cell.2011.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=21889194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2011&pages=904-917&author=J.+E.+Delmoreauthor=G.+C.+Issaauthor=M.+E.+Lemieuxauthor=P.+B.+Rahlauthor=J.+Shiauthor=H.+M.+Jacobsauthor=E.+Kastritisauthor=T.+Gilpatrickauthor=R.+M.+Paranalauthor=J.+Qiauthor=M.+Chesiauthor=A.+C.+Schinzelauthor=M.+R.+McKeownauthor=T.+P.+Heffernanauthor=C.+R.+Vakocauthor=P.+L.+Bergsagelauthor=I.+M.+Ghobrialauthor=P.+G.+Richardsonauthor=R.+A.+Youngauthor=W.+C.+Hahnauthor=K.+C.+Andersonauthor=A.+L.+Kungauthor=J.+E.+Bradnerauthor=C.+S.+Mitsiades&title=BET+bromodomain+inhibition+as+a+therapeutic+strategy+to+target+c-Myc&doi=10.1016%2Fj.cell.2011.08.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">BET Bromodomain Inhibition as a Therapeutic Strategy to Target c-Myc</span></div><div class="casAuthors">Delmore, Jake E.; Issa, Ghayas C.; Lemieux, Madeleine E.; Rahl, Peter B.; Shi, Jun-Wei; Jacobs, Hannah M.; Kastritis, Efstathios; Gilpatrick, Timothy; Paranal, Ronald M.; Qi, Jun; Chesi, Marta; Schinzel, Anna C.; McKeown, Michael R.; Heffernan, Timothy P.; Vakoc, Christopher R.; Bergsagel, P. Leif; Ghobrial, Irene M.; Richardson, Paul G.; Young, Richard A.; Hahn, William C.; Anderson, Kenneth C.; Kung, Andrew L.; Bradner, James E.; Mitsiades, Constantine S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">904-917</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: MYC contributes to the pathogenesis of a majority of human cancers, yet strategies to modulate the function of the c-Myc oncoprotein do not exist.  Toward this objective, we have targeted MYC transcription by interfering with chromatin-dependent signal transduction to RNA polymerase, specifically by inhibiting the acetyl-lysine recognition domains (bromodomains) of putative coactivator proteins implicated in transcriptional initiation and elongation.  Using a selective small-mol. bromodomain inhibitor, JQ1, we identify BET bromodomain proteins as regulatory factors for c-Myc.  BET inhibition by JQ1 downregulates MYC transcription, followed by genome-wide downregulation of Myc-dependent target genes.  In exptl. models of multiple myeloma, a Myc-dependent hematol. malignancy, JQ1 produces a potent antiproliferative effect assocd. with cell-cycle arrest and cellular senescence.  Efficacy of JQ1 in three murine models of multiple myeloma establishes the therapeutic rationale for BET bromodomain inhibition in this disease and other malignancies characterized by pathol. activation of c-Myc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpevD1QHKmOhLVg90H21EOLACvtfcHk0lgcPfuILiJJ_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF&md5=013674b8224214a9a1c8f0b75583a5a6</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DDelmore%26aufirst%3DJ.%2BE.%26aulast%3DIssa%26aufirst%3DG.%2BC.%26aulast%3DLemieux%26aufirst%3DM.%2BE.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DJacobs%26aufirst%3DH.%2BM.%26aulast%3DKastritis%26aufirst%3DE.%26aulast%3DGilpatrick%26aufirst%3DT.%26aulast%3DParanal%26aufirst%3DR.%2BM.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DChesi%26aufirst%3DM.%26aulast%3DSchinzel%26aufirst%3DA.%2BC.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DHeffernan%26aufirst%3DT.%2BP.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DBergsagel%26aufirst%3DP.%2BL.%26aulast%3DGhobrial%26aufirst%3DI.%2BM.%26aulast%3DRichardson%26aufirst%3DP.%2BG.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DMitsiades%26aufirst%3DC.%2BS.%26atitle%3DBET%2520bromodomain%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520to%2520target%2520c-Myc%26jtitle%3DCell%26date%3D2011%26volume%3D146%26spage%3D904%26epage%3D917%26doi%3D10.1016%2Fj.cell.2011.08.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandwani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marazzi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span> <span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1119</span>– <span class="NLM_lpage">1123</span>, <span class="refDoi"> DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.-w.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0lgcPfuILiJJ_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brotherton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diez-Dacal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">19754</span>– <span class="NLM_lpage">19759</span>, <span class="refDoi"> DOI: 10.1073/pnas.1310658110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1073%2Fpnas.1310658110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=24248379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=19754-19759&author=S.+Picaudauthor=C.+Wellsauthor=I.+Felletarauthor=D.+Brothertonauthor=S.+Martinauthor=P.+Savitskyauthor=B.+Diez-Dacalauthor=M.+Philpottauthor=C.+Bountraauthor=H.+Lingardauthor=O.+Fedorovauthor=S.+Mullerauthor=P.+E.+Brennanauthor=S.+Knappauthor=P.+Filippakopoulos&title=RVX-208%2C+an+inhibitor+of+BET+transcriptional+regulators+with+selectivity+for+the+second+bromodomain&doi=10.1073%2Fpnas.1310658110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span></div><div class="casAuthors">Picaud, Sarah; Wells, Christopher; Felletar, Ildiko; Brotherton, Deborah; Martin, Sarah; Savitsky, Pavel; Diez-Dacal, Beatriz; Philpott, Martin; Bountra, Chas; Lingard, Hannah; Fedorov, Oleg; Muller, Susanne; Brennan, Paul E.; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">19754-19759,S19754/1-S19754/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomains have emerged as attractive candidates for the development of inhibitors targeting gene transcription.  Inhibitors of the bromo and extraterminal (BET) family recently showed promising activity in diverse disease models.  However, the pleiotropic nature of BET proteins regulating tissue-specific transcription has raised safety concerns and suggested that attempts should be made for domain-specific targeting.  Here, we report that RVX-208, a compd. currently in phase II clin. trials, is a BET bromodomain inhibitor specific for second bromodomains (BD2s).  Cocrystal structures revealed binding modes of RVX-208 and its synthetic precursor, and fluorescent recovery after photobleaching demonstrated that RVX-208 displaces BET proteins from chromatin.  However, gene-expression data showed that BD2 inhibition only modestly affects BET-dependent gene transcription.  Our data demonstrate the feasibility of specific targeting within the BET family resulting in different transcriptional outcomes and highlight the importance of BD1 in transcriptional regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqamW8vnIopKrVg90H21EOLACvtfcHk0lhWWj4rrEZETw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN&md5=c918702c9fdb91241bc837732730f7f6</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1310658110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1310658110%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DBrotherton%26aufirst%3DD.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DDiez-Dacal%26aufirst%3DB.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DRVX-208%252C%2520an%2520inhibitor%2520of%2520BET%2520transcriptional%2520regulators%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D19754%26epage%3D19759%26doi%3D10.1073%2Fpnas.1310658110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witwicki, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabassum, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janiszewska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin Huh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doherty, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammed, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stover, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ekram, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peluffo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Santos, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krop, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duarte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anders, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winer, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Letai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barry, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirley Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Response and resistance to BET bromodomain inhibitors in triple-negative breast cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>529</i></span>,  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">417</span>, <span class="refDoi"> DOI: 10.1038/nature16508</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1038%2Fnature16508" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=26735014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC28Xns1Gmtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=529&publication_year=2016&pages=413-417&author=S.+Shuauthor=C.+Y.+Linauthor=H.+H.+Heauthor=R.+M.+Witwickiauthor=D.+P.+Tabassumauthor=J.+M.+Robertsauthor=M.+Janiszewskaauthor=S.+Jin+Huhauthor=Y.+Liangauthor=J.+Ryanauthor=E.+Dohertyauthor=H.+Mohammedauthor=H.+Guoauthor=D.+G.+Stoverauthor=M.+B.+Ekramauthor=G.+Peluffoauthor=J.+Brownauthor=C.+D%E2%80%99Santosauthor=I.+E.+Kropauthor=D.+Dillonauthor=M.+McKeownauthor=C.+Ottauthor=J.+Qiauthor=M.+Niauthor=P.+K.+Raoauthor=M.+Duarteauthor=S.-Y.+Wuauthor=C.-M.+Chiangauthor=L.+Andersauthor=R.+A.+Youngauthor=E.+P.+Winerauthor=A.+Letaiauthor=W.+T.+Barryauthor=J.+S.+Carrollauthor=H.+W.+Longauthor=M.+Brownauthor=X.+Shirley+Liuauthor=C.+A.+Meyerauthor=J.+E.+Bradner&title=Response+and+resistance+to+BET+bromodomain+inhibitors+in+triple-negative+breast+cancer&doi=10.1038%2Fnature16508"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Response and resistance to BET bromodomain inhibitors in triple-negative breast cancer</span></div><div class="casAuthors">Shu, Shaokun; Lin, Charles Y.; He, Housheng Hansen; Witwicki, Robert M.; Tabassum, Doris P.; Roberts, Justin M.; Janiszewska, Michalina; Huh, Sung Jin; Liang, Yi; Ryan, Jeremy; Doherty, Ernest; Mohammed, Hisham; Guo, Hao; Stover, Daniel G.; Ekram, Muhammad B.; Peluffo, Guillermo; Brown, Jonathan; D'Santos, Clive; Krop, Ian E.; Dillon, Deborah; McKeown, Michael; Ott, Christopher; Qi, Jun; Ni, Min; Rao, Prakash K.; Duarte, Melissa; Wu, Shwu-Yuan; Chiang, Cheng-Ming; Anders, Lars; Young, Richard A.; Winer, Eric P.; Letai, Antony; Barry, William T.; Carroll, Jason S.; Long, Henry W.; Brown, Myles; Shirley Liu, X.; Meyer, Clifford A.; Bradner, James E.; Polyak, Kornelia</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">529</span>
        (<span class="NLM_cas:issue">7586</span>),
    <span class="NLM_cas:pages">413-417</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Triple-neg. breast cancer (TNBC) is a heterogeneous and clin. aggressive disease for which there is no targeted therapy.  BET bromodomain inhibitors, which have shown efficacy in several models of cancer, have not been evaluated in TNBC.  These inhibitors displace BET bromodomain proteins such as BRD4 from chromatin by competing with their acetyl-lysine recognition modules, leading to inhibition of oncogenic transcriptional programs.  Here we report the preferential sensitivity of TNBCs to BET bromodomain inhibition in vitro and in vivo, establishing a rationale for clin. investigation and further motivation to understand mechanisms of resistance.  In paired cell lines selected for acquired resistance to BET inhibition from previously sensitive TNBCs, we failed to identify gatekeeper mutations, new driver events or drug pump activation.  BET-resistant TNBC cells remain dependent on wild-type BRD4, which supports transcription and cell proliferation in a bromodomain-independent manner.  Proteomic studies of resistant TNBC identify strong assocn. with MED1 and hyper-phosphorylation of BRD4 attributable to decreased activity of PP2A, identified here as a principal BRD4 serine phosphatase.  Together, these studies provide a rationale for BET inhibition in TNBC and present mechanism-based combination strategies to anticipate clin. drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZu5A85pdQDbVg90H21EOLACvtfcHk0liLXu9CJZ8aDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xns1Gmtw%253D%253D&md5=50a03861d9156ee6e344e95c5f5361b9</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnature16508&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature16508%26sid%3Dliteratum%253Aachs%26aulast%3DShu%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DHe%26aufirst%3DH.%2BH.%26aulast%3DWitwicki%26aufirst%3DR.%2BM.%26aulast%3DTabassum%26aufirst%3DD.%2BP.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DJaniszewska%26aufirst%3DM.%26aulast%3DJin%2BHuh%26aufirst%3DS.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DRyan%26aufirst%3DJ.%26aulast%3DDoherty%26aufirst%3DE.%26aulast%3DMohammed%26aufirst%3DH.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DStover%26aufirst%3DD.%2BG.%26aulast%3DEkram%26aufirst%3DM.%2BB.%26aulast%3DPeluffo%26aufirst%3DG.%26aulast%3DBrown%26aufirst%3DJ.%26aulast%3DD%25E2%2580%2599Santos%26aufirst%3DC.%26aulast%3DKrop%26aufirst%3DI.%2BE.%26aulast%3DDillon%26aufirst%3DD.%26aulast%3DMcKeown%26aufirst%3DM.%26aulast%3DOtt%26aufirst%3DC.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DNi%26aufirst%3DM.%26aulast%3DRao%26aufirst%3DP.%2BK.%26aulast%3DDuarte%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DS.-Y.%26aulast%3DChiang%26aufirst%3DC.-M.%26aulast%3DAnders%26aufirst%3DL.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DWiner%26aufirst%3DE.%2BP.%26aulast%3DLetai%26aufirst%3DA.%26aulast%3DBarry%26aufirst%3DW.%2BT.%26aulast%3DCarroll%26aufirst%3DJ.%2BS.%26aulast%3DLong%26aufirst%3DH.%2BW.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DShirley%2BLiu%26aufirst%3DX.%26aulast%3DMeyer%26aufirst%3DC.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DResponse%2520and%2520resistance%2520to%2520BET%2520bromodomain%2520inhibitors%2520in%2520triple-negative%2520breast%2520cancer%26jtitle%3DNature%26date%3D2016%26volume%3D529%26spage%3D413%26epage%3D417%26doi%3D10.1038%2Fnature16508" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Theodoulou, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Progress in the development of non-BET bromodomain chemical probes</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">477</span>– <span class="NLM_lpage">487</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1002%2Fcmdc.201500540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=26749027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC28Xks1ehtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=477-487&author=N.+H.+Theodoulouauthor=N.+C.+O.+Tomkinsonauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Progress+in+the+development+of+non-BET+bromodomain+chemical+probes&doi=10.1002%2Fcmdc.201500540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Progress in the Development of non-BET Bromodomain Chemical Probes</span></div><div class="casAuthors">Theodoulou, Natalie H.; Tomkinson, Nicholas C. O.; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">477-487</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The bromodomain and extra terminal (BET) family of bromodomains have been the focus of extensive research, leading to the development of many potent, selective chem. probes and recent clin. assets.  The profound biol. assocd. with BET bromodomain inhibition has provided a convincing rationale for targeting bromodomains for the treatment of disease.  However, the BET family represents just eight of the at least 56 human bromodomains identified to date.  Until recently, there has been significantly less interest in non-BET bromodomains, leaving a vast area of research and the majority of this new target class yet to be thoroughly investigated.  It has been widely reported that several non-BET bromodomain contg. proteins are assocd. with various diseases including cancer and HIV.  Therefore, the development of chem. probes for non-BET bromodomains will facilitate elucidation of their precise biol. roles and potentially lead to the development of new medicines.  This review summarises the progress made towards the development of non-BET bromodomain chem. probes to date.  In addn., we highlight the potential for future work in this new and exciting area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYLj_VyHOS5bVg90H21EOLACvtfcHk0liQ362Sr8vi3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xks1ehtA%253D%253D&md5=1e1339bf09a10bde8621c18344172fa6</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500540%26sid%3Dliteratum%253Aachs%26aulast%3DTheodoulou%26aufirst%3DN.%2BH.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DProgress%2520in%2520the%2520development%2520of%2520non-BET%2520bromodomain%2520chemical%2520probes%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D477%26epage%3D487%26doi%3D10.1002%2Fcmdc.201500540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moustakim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, P. G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Chemical probes and inhibitors of bromodomains outside the BET family</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">2246</span>– <span class="NLM_lpage">2264</span>, <span class="refDoi"> DOI: 10.1039/c6md00373g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1039%2Fc6md00373g" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=29170712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVOhsbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=2246-2264&author=M.+Moustakimauthor=P.+G.+K.+Clarkauthor=D.+A.+Hayauthor=D.+J.+Dixonauthor=P.+E.+Brennan&title=Chemical+probes+and+inhibitors+of+bromodomains+outside+the+BET+family&doi=10.1039%2Fc6md00373g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical probes and inhibitors of bromodomains outside the BET family</span></div><div class="casAuthors">Moustakim, Moses; Clark, Peter G. K.; Hay, Duncan A.; Dixon, Darren J.; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2246-2264</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">In the last five years, the development of inhibitors of bromodomains has emerged as an area of intensive worldwide research.  Emerging evidence has implicated a no. of non-BET bromodomains in the onset and progression of diseases such as cancer, HIV infection and inflammation.  The development and use of small mol. chem. probes has been fundamental to pre-clin. evaluation of bromodomains as targets.  Recent efforts are described highlighting the development of potent, selective and cell active non-BET bromodomain inhibitors and their therapeutic potential.  Over half of typical bromodomains now have reported ligands, but those with atypical binding site residues remain resistant to chem. probe discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYJvhj3Z0Ry7Vg90H21EOLACvtfcHk0lhtCl7xxM5fCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVOhsbbF&md5=df4c5718b15ff7d401a0cd9974abc236</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1039%2Fc6md00373g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc6md00373g%26sid%3Dliteratum%253Aachs%26aulast%3DMoustakim%26aufirst%3DM.%26aulast%3DClark%26aufirst%3DP.%2BG.%2BK.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DDixon%26aufirst%3DD.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DChemical%2520probes%2520and%2520inhibitors%2520of%2520bromodomains%2520outside%2520the%2520BET%2520family%26jtitle%3DMedChemComm%26date%3D2016%26volume%3D7%26spage%3D2246%26epage%3D2264%26doi%3D10.1039%2Fc6md00373g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clegg, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Advancements in the development of non-BET bromodomain chemical probes</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">362</span>– <span class="NLM_lpage">385</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201800738</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1002%2Fcmdc.201800738" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=30624862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1Ohs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=362-385&author=M.+A.+Cleggauthor=N.+C.+O.+Tomkinsonauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Advancements+in+the+development+of+non-BET+bromodomain+chemical+probes&doi=10.1002%2Fcmdc.201800738"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Advancements in the Development of non-BET Bromodomain Chemical Probes</span></div><div class="casAuthors">Clegg, Michael A.; Tomkinson, Nicholas C. O.; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">362-385</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The bromodomain and extra terminal (BET) family of bromodomain-contg. proteins (BCPs) have been the subject of extensive research over the past decade, resulting in a plethora of high-quality chem. probes for their tandem bromodomains.  In turn, these chem. probes have helped reveal the profound biol. role of the BET bromodomains and their role in disease, ultimately leading to a no. of mols. in active clin. development.  However, the BET subfamily represents just 8/61 of the known human bromodomains, and attention has now expanded to the biol. role of the remaining 53 non-BET bromodomains.  Rapid growth of this research area has been accompanied by a greater understanding of the requirements for an effective bromodomain chem. probe and has led to a no. of new non-BET bromodomain chem. probes being developed.  Advances since Dec. 2015 are discussed, highlighting the strengths/caveats of each mol., and the value they add toward validating the non-BET bromodomains as tractable therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_U9uvpaCEd7Vg90H21EOLACvtfcHk0ljNnxL__ARS3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1Ohs70%253D&md5=a93505759fe45ffd0352de2f0e17885a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201800738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201800738%26sid%3Dliteratum%253Aachs%26aulast%3DClegg%26aufirst%3DM.%2BA.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DAdvancements%2520in%2520the%2520development%2520of%2520non-BET%2520bromodomain%2520chemical%2520probes%26jtitle%3DChemMedChem%26date%3D2019%26volume%3D14%26spage%3D362%26epage%3D385%26doi%3D10.1002%2Fcmdc.201800738" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lau, O. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Courtney, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassilev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marzilli, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cotter, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakatani, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">P300/CBP-associated factor histone acetyltransferase processing of a peptide substrate. Kinetic analysis of the catalytic mechanism</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>275</i></span>,  <span class="NLM_fpage">21953</span>– <span class="NLM_lpage">21959</span>, <span class="refDoi"> DOI: 10.1074/jbc.m003219200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1074%2Fjbc.m003219200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=275&publication_year=2000&pages=21953-21959&author=O.+D.+Lauauthor=A.+D.+Courtneyauthor=A.+Vassilevauthor=L.+A.+Marzilliauthor=R.+J.+Cotterauthor=Y.+Nakataniauthor=P.+A.+Cole&title=P300%2FCBP-associated+factor+histone+acetyltransferase+processing+of+a+peptide+substrate.+Kinetic+analysis+of+the+catalytic+mechanism&doi=10.1074%2Fjbc.m003219200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m003219200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m003219200%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DO.%2BD.%26aulast%3DCourtney%26aufirst%3DA.%2BD.%26aulast%3DVassilev%26aufirst%3DA.%26aulast%3DMarzilli%26aufirst%3DL.%2BA.%26aulast%3DCotter%26aufirst%3DR.%2BJ.%26aulast%3DNakatani%26aufirst%3DY.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DP300%252FCBP-associated%2520factor%2520histone%2520acetyltransferase%2520processing%2520of%2520a%2520peptide%2520substrate.%2520Kinetic%2520analysis%2520of%2520the%2520catalytic%2520mechanism%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2000%26volume%3D275%26spage%3D21953%26epage%3D21959%26doi%3D10.1074%2Fjbc.m003219200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Linares, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiernan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Triboulet, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chable-Bessia, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latreille, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuvier, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacroix, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Cam, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coux, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benkirane, M.</span></span> <span> </span><span class="NLM_article-title">Intrinsic ubiquitination activity of PCAF controls the stability of the oncoprotein Hdm2</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">331</span>– <span class="NLM_lpage">338</span>, <span class="refDoi"> DOI: 10.1038/ncb1545</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1038%2Fncb1545" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=17293853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BD2sXit12rsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2007&pages=331-338&author=L.+K.+Linaresauthor=R.+Kiernanauthor=R.+Tribouletauthor=C.+Chable-Bessiaauthor=D.+Latreilleauthor=O.+Cuvierauthor=M.+Lacroixauthor=L.+Le+Camauthor=O.+Couxauthor=M.+Benkirane&title=Intrinsic+ubiquitination+activity+of+PCAF+controls+the+stability+of+the+oncoprotein+Hdm2&doi=10.1038%2Fncb1545"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Intrinsic ubiquitination activity of PCAF controls the stability of the oncoprotein Hdm2</span></div><div class="casAuthors">Linares, Laetitia K.; Kiernan, Rosemary; Triboulet, Robinson; Chable-Bessia, Christine; Latreille, Daniel; Cuvier, Olivier; Lacroix, Matthieu; Le Cam, Laurent; Coux, Olivier; Benkirane, Monsef</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">331-338</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The p300-CBP-assocd. factor (PCAF) is a histone acetyltransferase (HAT) involved in the reversible acetylation of various transcriptional regulators, including the tumor suppressor p53.  It is implicated in many cellular processes, such as transcription, differentiation, proliferation and apoptosis.  We obsd. that knockdown of PCAF expression in HeLa or U2OS cell lines induces stabilization of the oncoprotein Hdm2, a RING finger E3 ligase primarily known for its role in controlling p53 stability.  To investigate the mol. basis of this effect, we examd. whether PCAF is involved in Hdm2 ubiquitination.  Here, we show that PCAF, in addn. to its acetyltransferase activity, possesses an intrinsic ubiquitination activity that is crit. for controlling Hdm2 expression levels, and thus p53 functions.  Our data highlight a regulatory crosstalk between PCAF and Hdm2 activities, which is likely to have a central role in the subtle control of p53 activity after DNA damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIuN86NzWK97Vg90H21EOLACvtfcHk0ljNnxL__ARS3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXit12rsr8%253D&md5=93136ff023459d9d4d037a7c97bf12a2</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fncb1545&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb1545%26sid%3Dliteratum%253Aachs%26aulast%3DLinares%26aufirst%3DL.%2BK.%26aulast%3DKiernan%26aufirst%3DR.%26aulast%3DTriboulet%26aufirst%3DR.%26aulast%3DChable-Bessia%26aufirst%3DC.%26aulast%3DLatreille%26aufirst%3DD.%26aulast%3DCuvier%26aufirst%3DO.%26aulast%3DLacroix%26aufirst%3DM.%26aulast%3DLe%2BCam%26aufirst%3DL.%26aulast%3DCoux%26aufirst%3DO.%26aulast%3DBenkirane%26aufirst%3DM.%26atitle%3DIntrinsic%2520ubiquitination%2520activity%2520of%2520PCAF%2520controls%2520the%2520stability%2520of%2520the%2520oncoprotein%2520Hdm2%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2007%26volume%3D9%26spage%3D331%26epage%3D338%26doi%3D10.1038%2Fncb1545" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mujtaba, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farooq, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">Structural basis of lysine-acetylated HIV-1 Tat recognition by PCAF bromodomain</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">575</span>– <span class="NLM_lpage">586</span>, <span class="refDoi"> DOI: 10.1016/s1097-2765(02)00483-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1016%2Fs1097-2765%2802%2900483-5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=575-586&author=S.+Mujtabaauthor=Y.+Heauthor=L.+Zengauthor=A.+Farooqauthor=J.+E.+Carlsonauthor=M.+Ottauthor=E.+Verdinauthor=M.-M.+Zhou&title=Structural+basis+of+lysine-acetylated+HIV-1+Tat+recognition+by+PCAF+bromodomain&doi=10.1016%2Fs1097-2765%2802%2900483-5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fs1097-2765%2802%2900483-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs1097-2765%252802%252900483-5%26sid%3Dliteratum%253Aachs%26aulast%3DMujtaba%26aufirst%3DS.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DFarooq%26aufirst%3DA.%26aulast%3DCarlson%26aufirst%3DJ.%2BE.%26aulast%3DOtt%26aufirst%3DM.%26aulast%3DVerdin%26aufirst%3DE.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DStructural%2520basis%2520of%2520lysine-acetylated%2520HIV-1%2520Tat%2520recognition%2520by%2520PCAF%2520bromodomain%26jtitle%3DMol.%2520Cell%26date%3D2002%26volume%3D9%26spage%3D575%26epage%3D586%26doi%3D10.1016%2Fs1097-2765%2802%2900483-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, F.</span></span> <span> </span><span class="NLM_article-title">PCAF fine-tunes hepatic metabolic syndrome, inflammatory disease, and cancer</span>. <i>J. Cell. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">5787</span>– <span class="NLM_lpage">5800</span>, <span class="refDoi"> DOI: 10.1111/jcmm.13877</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1111%2Fjcmm.13877" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=30216660" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A280%3ADC%252BB3c3ovFCqsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2018&pages=5787-5800&author=T.+Wangauthor=W.+Yaoauthor=Y.+Shaoauthor=R.+Zhengauthor=F.+Huang&title=PCAF+fine-tunes+hepatic+metabolic+syndrome%2C+inflammatory+disease%2C+and+cancer&doi=10.1111%2Fjcmm.13877"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">PCAF fine-tunes hepatic metabolic syndrome, inflammatory disease, and cancer</span></div><div class="casAuthors">Wang Tongxin; Yao Weilei; Shao Yafei; Zheng Ruilong; Huang Feiruo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of cellular and molecular medicine</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5787-5800</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The P300/CBP-associating factor (PCAF), a histone acetyltransferase, is involved in metabolic and pathogenic diseases, particularly of the liver.  The effects of PCAF on fine-tuning liver diseases are extremely complex and vary according to different pathological conditions.  This enzyme has dichotomous functions, depending on differently modified sites, which regulate the activities of various enzymes, metabolic functions, and gene expression.  Here, we summarize the most recent findings on the functions and targets of PCAF in various metabolic and immunological processes in the liver and review these new discoveries and models of PCAF biology in three areas: hepatic metabolic syndrome, inflammatory disease, and cancer.  Finally, we discuss the potential implications of these findings for therapeutic interventions in liver diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS9R6QYjknpJ0NHe_gFldTbfW6udTcc2ebtUqoSLqxE8rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3ovFCqsQ%253D%253D&md5=ca754ef104057fc72eb7bb3ca5f59a1d</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1111%2Fjcmm.13877&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjcmm.13877%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DT.%26aulast%3DYao%26aufirst%3DW.%26aulast%3DShao%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DR.%26aulast%3DHuang%26aufirst%3DF.%26atitle%3DPCAF%2520fine-tunes%2520hepatic%2520metabolic%2520syndrome%252C%2520inflammatory%2520disease%252C%2520and%2520cancer%26jtitle%3DJ.%2520Cell.%2520Mol.%2520Med.%26date%3D2018%26volume%3D22%26spage%3D5787%26epage%3D5800%26doi%3D10.1111%2Fjcmm.13877" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mujtaba, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">Selective small molecules blocking HIV-1 Tat and coactivator PCAF association</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">2376</span>– <span class="NLM_lpage">2377</span>, <span class="refDoi"> DOI: 10.1021/ja044885g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja044885g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2005&pages=2376-2377&author=L.+Zengauthor=J.+Liauthor=M.+Mullerauthor=S.+Yanauthor=S.+Mujtabaauthor=C.+Panauthor=Z.+Wangauthor=M.-M.+Zhou&title=Selective+small+molecules+blocking+HIV-1+Tat+and+coactivator+PCAF+association&doi=10.1021%2Fja044885g"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fja044885g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja044885g%26sid%3Dliteratum%253Aachs%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DMuller%26aufirst%3DM.%26aulast%3DYan%26aufirst%3DS.%26aulast%3DMujtaba%26aufirst%3DS.%26aulast%3DPan%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DSelective%2520small%2520molecules%2520blocking%2520HIV-1%2520Tat%2520and%2520coactivator%2520PCAF%2520association%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2005%26volume%3D127%26spage%3D2376%26epage%3D2377%26doi%3D10.1021%2Fja044885g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span> <span> </span><span class="NLM_article-title">Fluorescence polarization for the evaluation of small-molecule inhibitors of PCAF BRD/Tat-AcK50 association</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">928</span>– <span class="NLM_lpage">931</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201300499</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1002%2Fcmdc.201300499" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=24474698" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtleisb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=928-931&author=P.+Huauthor=X.+Wangauthor=B.+Zhangauthor=S.+Zhangauthor=Q.+Wangauthor=Z.+Wang&title=Fluorescence+polarization+for+the+evaluation+of+small-molecule+inhibitors+of+PCAF+BRD%2FTat-AcK50+association&doi=10.1002%2Fcmdc.201300499"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence Polarization for the Evaluation of Small-Molecule Inhibitors of PCAF BRD/Tat-AcK50 Association</span></div><div class="casAuthors">Hu, Ping; Wang, Xinghui; Zhang, Baiqun; Zhang, Shuai; Wang, Qiang; Wang, Zhiyong</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">928-931</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A fluorescence polarization competitive assay was developed to efficiently screen and evaluate inhibitors of PCAF bromodomain/Tat-AcK50 protein-peptide interaction.  A series of pyridine 1-oxide derivs. were synthesized and evaluated.  Some of the novel compds., 2-(3-aminopropylamino) pyridine 1-oxide derivs., could be effective inhibitors of PCAF bromodomain/Tat-AcK50 assocn.  Specifically, 2-(3-aminopropylamino)-5-(hydroxymethyl)pyridine 1-oxide hydrochloride (15) and the 5-((3-aminopropylamino)methyl) deriv. (20) were found to be effective ligands for the PCAF BRD pocket.  First preliminary cellular studies indicate that these small-mol. inhibitors have lower cytotoxicities and are potential leads for the anti-HIV/AIDS therapeutic strategy by targeting host-cell protein PCAF BRD to block HIV replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2Jg2NNDsHHrVg90H21EOLACvtfcHk0ljrLO1Knqu6bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtleisb0%253D&md5=97dc23c85843ad1611f5927fd2ee540c</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300499%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DZ.%26atitle%3DFluorescence%2520polarization%2520for%2520the%2520evaluation%2520of%2520small-molecule%2520inhibitors%2520of%2520PCAF%2520BRD%252FTat-AcK50%2520association%26jtitle%3DChemMedChem%26date%3D2014%26volume%3D9%26spage%3D928%26epage%3D931%26doi%3D10.1002%2Fcmdc.201300499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Navratilova, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aristotelous, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filappakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span> <span> </span><span class="NLM_article-title">Discovery of new bromodomain scaffolds by biosensor fragment screening</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1213</span>– <span class="NLM_lpage">1218</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00154</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00154" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=1213-1218&author=I.+Navratilovaauthor=T.+Aristotelousauthor=S.+Picaudauthor=A.+Chaikuadauthor=S.+Knappauthor=P.+Filappakopoulosauthor=A.+L.+Hopkins&title=Discovery+of+new+bromodomain+scaffolds+by+biosensor+fragment+screening&doi=10.1021%2Facsmedchemlett.6b00154"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00154%26sid%3Dliteratum%253Aachs%26aulast%3DNavratilova%26aufirst%3DI.%26aulast%3DAristotelous%26aufirst%3DT.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilappakopoulos%26aufirst%3DP.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DDiscovery%2520of%2520new%2520bromodomain%2520scaffolds%2520by%2520biosensor%2520fragment%2520screening%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D1213%26epage%3D1218%26doi%3D10.1021%2Facsmedchemlett.6b00154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Temperini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollander, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nachane, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegal, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based identification of inhibitory fragments targeting the p300/CBP-associated factor bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1648</span>– <span class="NLM_lpage">1653</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01719</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01719" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1648-1653&author=A.+Chaikuadauthor=S.+Langauthor=P.+E.+Brennanauthor=C.+Temperiniauthor=O.+Fedorovauthor=J.+Hollanderauthor=R.+Nachaneauthor=C.+Abellauthor=S.+M%C3%BCllerauthor=G.+Siegalauthor=S.+Knapp&title=Structure-based+identification+of+inhibitory+fragments+targeting+the+p300%2FCBP-associated+factor+bromodomain&doi=10.1021%2Facs.jmedchem.5b01719"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01719&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01719%26sid%3Dliteratum%253Aachs%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DLang%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DTemperini%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DHollander%26aufirst%3DJ.%26aulast%3DNachane%26aufirst%3DR.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DSiegal%26aufirst%3DG.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DStructure-based%2520identification%2520of%2520inhibitory%2520fragments%2520targeting%2520the%2520p300%252FCBP-associated%2520factor%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1648%26epage%3D1653%26doi%3D10.1021%2Facs.jmedchem.5b01719" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span>; <span class="NLM_string-name">Cote, A.</span>; <span class="NLM_string-name">Crawford, T.</span>; <span class="NLM_string-name">Duplessis, M.</span>; <span class="NLM_string-name">Good, A.
C.</span>; <span class="NLM_string-name">Leblanc, Y.</span>; <span class="NLM_string-name">Magnuson, S.</span>; <span class="NLM_string-name">Nasveschuk, C. G.</span>; <span class="NLM_string-name">Pastor, R.</span>; <span class="NLM_string-name">Romero, F. A.</span>; <span class="NLM_string-name">Taylor, A. M.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Compounds and Uses Thereof</span>. WO <span class="NLM_patent">2016036873 A1</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=B.+K.+Albrecht&author=A.+Cote&author=T.+Crawford&author=M.+Duplessis&author=A.%0AC.+Good&author=Y.+Leblanc&author=S.+Magnuson&author=C.+G.+Nasveschuk&author=R.+Pastor&author=F.+A.+Romero&author=A.+M.+Taylor&title=Therapeutic+Compounds+and+Uses+Thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26atitle%3DTherapeutic%2520Compounds%2520and%2520Uses%2520Thereof%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span>; <span class="NLM_string-name">Cote, A.</span>; <span class="NLM_string-name">Crawford, T.</span>; <span class="NLM_string-name">Duplessis, M.</span>; <span class="NLM_string-name">Good, A.
C.</span>; <span class="NLM_string-name">Leblanc, Y.</span>; <span class="NLM_string-name">Magnuson, S.</span>; <span class="NLM_string-name">Nasveschuk, C. G.</span>; <span class="NLM_string-name">Pastor, R.</span>; <span class="NLM_string-name">Romero, F. A.</span>; <span class="NLM_string-name">Taylor, A. M.</span></span> <span> </span><span class="NLM_article-title">Phthalazine Derivatives of Forumula (I) as PCAF and GCN5 Inhibitors for Use in the Treatment of Cancer</span>. WO <span class="NLM_patent">2016036954 A1</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=B.+K.+Albrecht&author=A.+Cote&author=T.+Crawford&author=M.+Duplessis&author=A.%0AC.+Good&author=Y.+Leblanc&author=S.+Magnuson&author=C.+G.+Nasveschuk&author=R.+Pastor&author=F.+A.+Romero&author=A.+M.+Taylor&title=Phthalazine+Derivatives+of+Forumula+%28I%29+as+PCAF+and+GCN5+Inhibitors+for+Use+in+the+Treatment+of+Cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26atitle%3DPhthalazine%2520Derivatives%2520of%2520Forumula%2520%2528I%2529%2520as%2520PCAF%2520and%2520GCN5%2520Inhibitors%2520for%2520Use%2520in%2520the%2520Treatment%2520of%2520Cancer%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span>; <span class="NLM_string-name">Burdick, D. J.</span>; <span class="NLM_string-name">Cote, A.</span>; <span class="NLM_string-name">Duplessis, M.</span>; <span class="NLM_string-name">Nasveschuk, C. G.</span>; <span class="NLM_string-name">Taylor, A. M.</span></span> <span> </span><span class="NLM_article-title">Pyridazinone Derivatives and Their Use in the Treatment of Cancer</span>. WO <span class="NLM_patent">2016112298 A1</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=B.+K.+Albrecht&author=D.+J.+Burdick&author=A.+Cote&author=M.+Duplessis&author=C.+G.+Nasveschuk&author=A.+M.+Taylor&title=Pyridazinone+Derivatives+and+Their+Use+in+the+Treatment+of+Cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26atitle%3DPyridazinone%2520Derivatives%2520and%2520Their%2520Use%2520in%2520the%2520Treatment%2520of%2520Cancer%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moustakim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, P. G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trulli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes de Arriba, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehebauer, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendez-Johnson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, C.-F. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorrell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Discovery of a PCAF bromodomain chemical probe</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">827</span>– <span class="NLM_lpage">831</span>, <span class="refDoi"> DOI: 10.1002/anie.201610816</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1002%2Fanie.201610816" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC28XitV2mtL3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=827-831&author=M.+Moustakimauthor=P.+G.+K.+Clarkauthor=L.+Trulliauthor=A.+L.+Fuentes+de+Arribaauthor=M.+T.+Ehebauerauthor=A.+Chaikuadauthor=E.+J.+Murphyauthor=J.+Mendez-Johnsonauthor=D.+Danielsauthor=C.-F.+D.+Houauthor=Y.-H.+Linauthor=J.+R.+Walkerauthor=R.+Huiauthor=H.+Yangauthor=L.+Dorrellauthor=C.+M.+Rogersauthor=O.+P.+Monteiroauthor=O.+Fedorovauthor=K.+V.+M.+Huberauthor=S.+Knappauthor=J.+Heerauthor=D.+J.+Dixonauthor=P.+E.+Brennan&title=Discovery+of+a+PCAF+bromodomain+chemical+probe&doi=10.1002%2Fanie.201610816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a PCAF Bromodomain Chemical Probe</span></div><div class="casAuthors">Moustakim, Moses; Clark, Peter G. K.; Trulli, Laura; Fuentes de Arriba, Angel L.; Ehebauer, Matthias T.; Chaikuad, Apirat; Murphy, Emma J.; Mendez-Johnson, Jacqui; Daniels, Danette; Hou, Chun-Feng D.; Lin, Yu-Hui; Walker, John R.; Hui, Raymond; Yang, Hongbing; Dorrell, Lucy; Rogers, Catherine M.; Monteiro, Octovia P.; Fedorov, Oleg; Huber, Kilian V. M.; Knapp, Stefan; Heer, Jag; Dixon, Darren J.; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">827-831</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The p300/CBP-assocd. factor (PCAF) and related GCN5 bromodomain-contg. lysine acetyl transferases are members of subfamily I of the bromodomain phylogenetic tree.  Iterative cycles of rational inhibitor design and biophys. characterization led to the discovery of the triazolopthalazine-based L-45 (dubbed L-Moses) as the first potent, selective, and cell-active PCAF bromodomain (Brd) inhibitor.  Synthesis from readily available (1R,2S)-(-)-norephedrine furnished L-45 in enantiopure form.  L-45 was shown to disrupt PCAF-Brd histone H3.3 interaction in cells using a nanoBRET assay, and a co-crystal structure of L-45 with the homologous Brd PfGCN5 from Plasmodium falciparum rationalizes the high selectivity for PCAF and GCN5 bromodomains.  Compd. L-45 shows no observable cytotoxicity in peripheral blood mononuclear cells (PBMC), good cell-permeability, and metabolic stability in human and mouse liver microsomes, supporting its potential for in vivo use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3qpUenJBD_7Vg90H21EOLACvtfcHk0lhjZBfOzwTSEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitV2mtL3M&md5=26c4dedfb4c441d47cf2bbd44885c09a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fanie.201610816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201610816%26sid%3Dliteratum%253Aachs%26aulast%3DMoustakim%26aufirst%3DM.%26aulast%3DClark%26aufirst%3DP.%2BG.%2BK.%26aulast%3DTrulli%26aufirst%3DL.%26aulast%3DFuentes%2Bde%2BArriba%26aufirst%3DA.%2BL.%26aulast%3DEhebauer%26aufirst%3DM.%2BT.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DMurphy%26aufirst%3DE.%2BJ.%26aulast%3DMendez-Johnson%26aufirst%3DJ.%26aulast%3DDaniels%26aufirst%3DD.%26aulast%3DHou%26aufirst%3DC.-F.%2BD.%26aulast%3DLin%26aufirst%3DY.-H.%26aulast%3DWalker%26aufirst%3DJ.%2BR.%26aulast%3DHui%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DDorrell%26aufirst%3DL.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHeer%26aufirst%3DJ.%26aulast%3DDixon%26aufirst%3DD.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DDiscovery%2520of%2520a%2520PCAF%2520bromodomain%2520chemical%2520probe%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D827%26epage%3D831%26doi%3D10.1002%2Fanie.201610816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayhow, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renaux, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suckling, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent, cell penetrant, and selective P300/CBP-associated factor (PCAF)/general control nonderepressible 5 (GCN5) bromodomain chemical probe</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">695</span>– <span class="NLM_lpage">709</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01566</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01566" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFCgs77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=695-709&author=P.+G.+Humphreysauthor=P.+Bamboroughauthor=C.-w.+Chungauthor=P.+D.+Craggsauthor=L.+Gordonauthor=P.+Grandiauthor=T.+G.+Hayhowauthor=J.+Hussainauthor=K.+L.+Jonesauthor=M.+Lindonauthor=A.-M.+Michonauthor=J.+F.+Renauxauthor=C.+J.+Sucklingauthor=D.+F.+Toughauthor=R.+K.+Prinjha&title=Discovery+of+a+potent%2C+cell+penetrant%2C+and+selective+P300%2FCBP-associated+factor+%28PCAF%29%2Fgeneral+control+nonderepressible+5+%28GCN5%29+bromodomain+chemical+probe&doi=10.1021%2Facs.jmedchem.6b01566"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent, Cell Penetrant, and Selective p300/CBP-Associated Factor (PCAF)/General Control Nonderepressible 5 (GCN5) Bromodomain Chemical Probe</span></div><div class="casAuthors">Humphreys, Philip G.; Bamborough, Paul; Chung, Chun-wa; Craggs, Peter D.; Gordon, Laurie; Grandi, Paola; Hayhow, Thomas G.; Hussain, Jameed; Jones, Katherine L.; Lindon, Matthew; Michon, Anne-Marie; Renaux, Jessica F.; Suckling, Colin J.; Tough, David F.; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">695-709</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P300/CREB binding protein assocd. factor (PCAF/KAT2B) and general control non-derepressible 5 (GCN5/KAT2A) are multidomain proteins that have been implicated in retroviral infection, inflammation pathways and cancer development.  However, outside of viral replication, little is known about the dependence of these effects on the C-terminal bromodomain.  Herein, the authors report GSK4027 as a chem. probe for the PCAF/GCN5 bromodomain, together with GSK4028 as an enantiomeric neg. control.  The probe was optimized from a weakly potent, non-selective pyridazinone hit to deliver high potency for the PCAF/GCN5 bromodomain, high soly., cellular target engagement and ≥18,000-fold selectivity over the BET family, together with ≥70-fold selectivity over the wider bromodomain families.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBkLwjcv1aZLVg90H21EOLACvtfcHk0lhjZBfOzwTSEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFCgs77N&md5=fd4fdbe8827e26769fc17b09120c7826</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01566%26sid%3Dliteratum%253Aachs%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DHayhow%26aufirst%3DT.%2BG.%26aulast%3DHussain%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DK.%2BL.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DRenaux%26aufirst%3DJ.%2BF.%26aulast%3DSuckling%26aufirst%3DC.%2BJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DDiscovery%2520of%2520a%2520potent%252C%2520cell%2520penetrant%252C%2520and%2520selective%2520P300%252FCBP-associated%2520factor%2520%2528PCAF%2529%252Fgeneral%2520control%2520nonderepressible%25205%2520%2528GCN5%2529%2520bromodomain%2520chemical%2520probe%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D695%26epage%3D709%26doi%3D10.1021%2Facs.jmedchem.6b01566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bassi, Z. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fillmore, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maller, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galwey, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waddington, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parravicini, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macmillan-Jones, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gongora, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span> <span> </span><span class="NLM_article-title">Modulating PCAF/GCN5 immune cell function through a PROTAC approach</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2862</span>– <span class="NLM_lpage">2867</span>, <span class="refDoi"> DOI: 10.1021/acschembio.8b00705</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b00705" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1OktL7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=2862-2867&author=Z.+I.+Bassiauthor=M.+C.+Fillmoreauthor=A.+H.+Miahauthor=T.+D.+Chapmanauthor=C.+Mallerauthor=E.+J.+Robertsauthor=L.+C.+Davisauthor=D.+E.+Lewisauthor=N.+W.+Galweyauthor=K.+E.+Waddingtonauthor=V.+Parraviciniauthor=A.+L.+Macmillan-Jonesauthor=C.+Gongoraauthor=P.+G.+Humphreysauthor=I.+Churcherauthor=R.+K.+Prinjhaauthor=D.+F.+Tough&title=Modulating+PCAF%2FGCN5+immune+cell+function+through+a+PROTAC+approach&doi=10.1021%2Facschembio.8b00705"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Modulating PCAF/GCN5 Immune Cell Function through a PROTAC Approach</span></div><div class="casAuthors">Bassi, Zuni I.; Fillmore, Martin C.; Miah, Afjal H.; Chapman, Trevor D.; Maller, Claire; Roberts, Emma J.; Davis, Lauren C.; Lewis, Darcy E.; Galwey, Nicholas W.; Waddington, Kirsty E.; Parravicini, Valentino; Macmillan-Jones, Abigail L.; Gongora, Celine; Humphreys, Philip G.; Churcher, Ian; Prinjha, Rab K.; Tough, David F.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2862-2867</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P300/CBP-assocd. factor (PCAF) and general control nonderepressible 5 (GCN5) are closely related epigenetic proteins, each contg. an acetyltransferase domain and a bromodomain.  Consistent with reported roles for these proteins in immune function, we find that PCAF-deficient macrophages exhibit a markedly reduced ability to produce cytokines upon stimulation with lipopolysaccharide (LPS).  Investigating the potential to target this pathway pharmacol., we show that chem. inhibition of the PCAF/GCN5 bromodomains is insufficient to recapitulate the diminished inflammatory response of PCAF-deficient immune cells.  However, by generating the first PCAF/GCN5 proteolysis targeting chimera (PROTAC), we identify small mols. able to degrade PCAF/GCN5 and to potently modulate the expression of multiple inflammatory mediators in LPS-stimulated macrophages and dendritic cells.  Our data illustrate the power of the PROTAC approach in the context of multidomain proteins, revealing a novel anti-inflammatory therapeutic opportunity for targeting PCAF/GCN5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUuwZ6JZtCirVg90H21EOLACvtfcHk0ljjR8SR51SXEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1OktL7I&md5=6783f39397381536ed40ac9e1b20fa5b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b00705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b00705%26sid%3Dliteratum%253Aachs%26aulast%3DBassi%26aufirst%3DZ.%2BI.%26aulast%3DFillmore%26aufirst%3DM.%2BC.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DMaller%26aufirst%3DC.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DDavis%26aufirst%3DL.%2BC.%26aulast%3DLewis%26aufirst%3DD.%2BE.%26aulast%3DGalwey%26aufirst%3DN.%2BW.%26aulast%3DWaddington%26aufirst%3DK.%2BE.%26aulast%3DParravicini%26aufirst%3DV.%26aulast%3DMacmillan-Jones%26aufirst%3DA.%2BL.%26aulast%3DGongora%26aufirst%3DC.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DTough%26aufirst%3DD.%2BF.%26atitle%3DModulating%2520PCAF%252FGCN5%2520immune%2520cell%2520function%2520through%2520a%2520PROTAC%2520approach%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D2862%26epage%3D2867%26doi%3D10.1021%2Facschembio.8b00705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of potent and selective inhibitors of BRD7 and BRD9 bromodomains</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1381</span>– <span class="NLM_lpage">1386</span>, <span class="refDoi"> DOI: 10.1039/c5md00152h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1039%2Fc5md00152h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVSgtrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=1381-1386&author=D.+A.+Hayauthor=C.+M.+Rogersauthor=O.+Fedorovauthor=C.+Tallantauthor=S.+Martinauthor=O.+P.+Monteiroauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=C.+J.+Schofieldauthor=P.+E.+Brennan&title=Design+and+synthesis+of+potent+and+selective+inhibitors+of+BRD7+and+BRD9+bromodomains&doi=10.1039%2Fc5md00152h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Design and synthesis of potent and selective inhibitors of BRD7 and BRD9 bromodomains</span></div><div class="casAuthors">Hay, Duncan A.; Rogers, Catherine M.; Fedorov, Oleg; Tallant, Cynthia; Martin, Sarah; Monteiro, Octovia P.; Muller, Susanne; Knapp, Stefan; Schofield, Christopher J.; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1381-1386</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Emerging evidence suggests bromodomain-contg. proteins 7 and 9 (BRD7 and BRD9) have roles in the regulation of human transcription and disease including cancer.  We describe potent and selective inhibitors of the BRD7 and BRD9 bromodomains intended for use as tools to elucidate the biol. roles of BRD7 and BRD9 in healthy and diseased cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrynQ2AJnreDLVg90H21EOLACvtfcHk0ljjR8SR51SXEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVSgtrvJ&md5=6a93d7300f7815eaf5ca654f873078d3</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1039%2Fc5md00152h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc5md00152h%26sid%3Dliteratum%253Aachs%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520potent%2520and%2520selective%2520inhibitors%2520of%2520BRD7%2520and%2520BRD9%2520bromodomains%26jtitle%3DMedChemComm%26date%3D2015%26volume%3D6%26spage%3D1381%26epage%3D1386%26doi%3D10.1039%2Fc5md00152h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lesniak, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selenski, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span> <span> </span><span class="NLM_article-title">Discovery and characterization of GSK2801, a selective chemical probe for the bromodomains BAZ2A and BAZ2B</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1410</span>– <span class="NLM_lpage">1424</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00209</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00209" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC2MXltFSlsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1410-1424&author=P.+Chenauthor=A.+Chaikuadauthor=P.+Bamboroughauthor=M.+Bantscheffauthor=C.+Bountraauthor=C.-w.+Chungauthor=O.+Fedorovauthor=P.+Grandiauthor=D.+Jungauthor=R.+Lesniakauthor=M.+Lindonauthor=S.+M%C3%BCllerauthor=M.+Philpottauthor=R.+Prinjhaauthor=C.+Rogersauthor=C.+Selenskiauthor=C.+Tallantauthor=T.+Wernerauthor=T.+M.+Willsonauthor=S.+Knappauthor=D.+H.+Drewry&title=Discovery+and+characterization+of+GSK2801%2C+a+selective+chemical+probe+for+the+bromodomains+BAZ2A+and+BAZ2B&doi=10.1021%2Facs.jmedchem.5b00209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Characterization of GSK2801, a Selective Chemical Probe for the Bromodomains BAZ2A and BAZ2B</span></div><div class="casAuthors">Chen, Peiling; Chaikuad, Apirat; Bamborough, Paul; Bantscheff, Marcus; Bountra, Chas; Chung, Chun-wa; Fedorov, Oleg; Grandi, Paola; Jung, David; Lesniak, Robert; Lindon, Matthew; Muller, Susanne; Philpott, Martin; Prinjha, Rab; Rogers, Catherine; Selenski, Carolyn; Tallant, Cynthia; Werner, Thilo; Willson, Timothy M.; Knapp, Stefan; Drewry, David H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1410-1424</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains are acetyl-lysine specific protein interaction domains that have recently emerged as a new target class for the development of inhibitors that modulate gene transcription.  The two closely related bromodomain contg. proteins BAZ2A and BAZ2B constitute the central scaffolding protein of the nucleolar remodeling complex (NoRC) that regulates the expression of noncoding RNAs.  However, BAZ2 bromodomains have low predicted druggability and so far no selective inhibitors have been published.  Here we report the development of GSK2801, a potent, selective and cell active acetyl-lysine competitive inhibitor of BAZ2A and BAZ2B bromodomains as well as the inactive control compd. GSK8573.  GSK2801 binds to BAZ2 bromodomains with dissocn. consts. (KD) of 136 and 257 nM for BAZ2B and BAZ2A, resp.  Crystal structures demonstrated a canonical acetyl-lysine competitive binding mode.  Cellular activity was demonstrated using fluorescent recovery after photobleaching (FRAP) monitoring displacement of GFP-BAZ2A from acetylated chromatin.  A pharmacokinetic study in mice showed that GSK2801 had reasonable in vivo exposure after oral dosing, with modest clearance and reasonable plasma stability.  Thus, GSK2801 represents a versatile tool compd. for cellular and in vivo studies to understand the role of BAZ2 bromodomains in chromatin biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprwxgNLFBYCrVg90H21EOLACvtfcHk0ljjR8SR51SXEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXltFSlsbY%253D&md5=700e8966cb5932a1558a7d8cc2976b43</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00209%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DP.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DJung%26aufirst%3DD.%26aulast%3DLesniak%26aufirst%3DR.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DPrinjha%26aufirst%3DR.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DSelenski%26aufirst%3DC.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26atitle%3DDiscovery%2520and%2520characterization%2520of%2520GSK2801%252C%2520a%2520selective%2520chemical%2520probe%2520for%2520the%2520bromodomains%2520BAZ2A%2520and%2520BAZ2B%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1410%26epage%3D1424%26doi%3D10.1021%2Facs.jmedchem.5b00209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smaill, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery and optimization of 1-(1H-indol-1-yl)ethanone derivatives as CBP/EP300 bromodomain inhibitors for the treatment of castration-resistant prostate cancer</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>147</i></span>,  <span class="NLM_fpage">238</span>– <span class="NLM_lpage">252</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.01.</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1016%2Fj.ejmech.2018.01." title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=2018&pages=238-252&author=Q.+Xiangauthor=C.+Wangauthor=Y.+Zhangauthor=X.+Xueauthor=M.+Songauthor=C.+Zhangauthor=C.+Liauthor=C.+Wuauthor=K.+Liauthor=X.+Huiauthor=Y.+Zhouauthor=J.+B.+Smaillauthor=A.+V.+Pattersonauthor=D.+Wuauthor=K.+Dingauthor=Y.+Xu&title=Discovery+and+optimization+of+1-%281H-indol-1-yl%29ethanone+derivatives+as+CBP%2FEP300+bromodomain+inhibitors+for+the+treatment+of+castration-resistant+prostate+cancer&doi=10.1016%2Fj.ejmech.2018.01."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.01.&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.01.%26sid%3Dliteratum%253Aachs%26aulast%3DXiang%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DXue%26aufirst%3DX.%26aulast%3DSong%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DK.%26aulast%3DHui%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DSmaill%26aufirst%3DJ.%2BB.%26aulast%3DPatterson%26aufirst%3DA.%2BV.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DDing%26aufirst%3DK.%26aulast%3DXu%26aufirst%3DY.%26atitle%3DDiscovery%2520and%2520optimization%2520of%25201-%25281H-indol-1-yl%2529ethanone%2520derivatives%2520as%2520CBP%252FEP300%2520bromodomain%2520inhibitors%2520for%2520the%2520treatment%2520of%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D147%26spage%3D238%26epage%3D252%26doi%3D10.1016%2Fj.ejmech.2018.01." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Unzue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lolli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zechner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolbois, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caflisch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nevado, C.</span></span> <span> </span><span class="NLM_article-title">The ″gatekeeper″ residue influences the mode of binding of acetyl indoles to bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3087</span>– <span class="NLM_lpage">3097</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01757</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01757" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC28Xkt1CntL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=3087-3097&author=A.+Unzueauthor=H.+Zhaoauthor=G.+Lolliauthor=J.+Dongauthor=J.+Zhuauthor=M.+Zechnerauthor=A.+Dolboisauthor=A.+Caflischauthor=C.+Nevado&title=The+%E2%80%B3gatekeeper%E2%80%B3+residue+influences+the+mode+of+binding+of+acetyl+indoles+to+bromodomains&doi=10.1021%2Facs.jmedchem.5b01757"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">The "Gatekeeper" Residue Influences the Mode of Binding of Acetyl Indoles to Bromodomains</span></div><div class="casAuthors">Unzue, Andrea; Zhao, Hongtao; Lolli, Graziano; Dong, Jing; Zhu, Jian; Zechner, Melanie; Dolbois, Aymeric; Caflisch, Amedeo; Nevado, Cristina</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3087-3097</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small-mol. hits for the bromodomains of CREBBP and BAZ2B have been identified by scaffold hopping followed by docking of a set of ∼200 compds. contg. the acetyl indole scaffold.  Chem. synthesis of nearly 30 derivs. has resulted in ligands of representatives of three subfamilies of human bromodomains with favorable ligand efficiency.  The X-ray crystal structures of three different bromodomains (CREBBP, BAZ2B, and BRPF1b) in complex with acetyl indole derivs. reveal the influence of the gatekeeper residue on the orientation of small-mol. ligands in the acetyl lysine binding site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLvogt-C6f_LVg90H21EOLACvtfcHk0li_ZObgrlWUSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xkt1CntL0%253D&md5=35b2490b53eb162d0009d68b8f11262e</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01757&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01757%26sid%3Dliteratum%253Aachs%26aulast%3DUnzue%26aufirst%3DA.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DLolli%26aufirst%3DG.%26aulast%3DDong%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DZechner%26aufirst%3DM.%26aulast%3DDolbois%26aufirst%3DA.%26aulast%3DCaflisch%26aufirst%3DA.%26aulast%3DNevado%26aufirst%3DC.%26atitle%3DThe%2520%25E2%2580%25B3gatekeeper%25E2%2580%25B3%2520residue%2520influences%2520the%2520mode%2520of%2520binding%2520of%2520acetyl%2520indoles%2520to%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D3087%26epage%3D3097%26doi%3D10.1021%2Facs.jmedchem.5b01757" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Steijvoort, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaval, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulago, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maes, B. U. W.</span></span> <span> </span><span class="NLM_article-title">Remote functionalization: palladium-catalyzed C5(sp3)-H arylation of 1-boc-3-aminopiperidine through the use of a bidentate directing group</span>. <i>ACS Catal.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">4486</span>– <span class="NLM_lpage">4490</span>, <span class="refDoi"> DOI: 10.1021/acscatal.6b00841</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscatal.6b00841" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVKmsL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=4486-4490&author=B.+F.+Van%0ASteijvoortauthor=N.+Kavalauthor=A.+A.+Kulagoauthor=B.+U.+W.+Maes&title=Remote+functionalization%3A+palladium-catalyzed+C5%28sp3%29-H+arylation+of+1-boc-3-aminopiperidine+through+the+use+of+a+bidentate+directing+group&doi=10.1021%2Facscatal.6b00841"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Remote Functionalization: Palladium-Catalyzed C5(sp3)-H Arylation of 1-Boc-3-aminopiperidine through the Use of a Bidentate Directing Group</span></div><div class="casAuthors">Van Steijvoort, Ben F.; Kaval, Nadya; Kulago, Artem A.; Maes, Bert U. W.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Catalysis</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4486-4490</span>CODEN:
                <span class="NLM_cas:coden">ACCACS</span>;
        ISSN:<span class="NLM_cas:issn">2155-5435</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A protocol for the Pd-catalyzed C5(sp3)-H arylation of readily available 1-Boc-3-(picolinoylamino)piperidine with iodo(hetero)arenes is reported.  The substrate can be obtained from a biorenewable feedstock, namely L-arginine.  The use of the correct N1 protective group is decisive to obtain arylation.  The addn. of a catalytic amt. of 2,6-dimethylbenzoic acid and performing the reaction at high concn. are important to achieve a high conversion and yield.  The procedure gives arylated 1-Boc-3-(picolinoylamino)piperidines in a regiospecific (C5) and stereospecific (cis) manner.  Orthogonal cleavage of the amide over the carbamate group allows one to further selectively derivatize the amino moieties of the piperidine scaffold.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8Ay5gpO-0I7Vg90H21EOLACvtfcHk0li_ZObgrlWUSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVKmsL7E&md5=cc0186e8b700e9fca735b6d077d28c79</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facscatal.6b00841&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscatal.6b00841%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BSteijvoort%26aufirst%3DB.%2BF.%26aulast%3DKaval%26aufirst%3DN.%26aulast%3DKulago%26aufirst%3DA.%2BA.%26aulast%3DMaes%26aufirst%3DB.%2BU.%2BW.%26atitle%3DRemote%2520functionalization%253A%2520palladium-catalyzed%2520C5%2528sp3%2529-H%2520arylation%2520of%25201-boc-3-aminopiperidine%2520through%2520the%2520use%2520of%2520a%2520bidentate%2520directing%2520group%26jtitle%3DACS%2520Catal.%26date%3D2016%26volume%3D6%26spage%3D4486%26epage%3D4490%26doi%3D10.1021%2Facscatal.6b00841" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresini, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burdick, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakin, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duplessis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Good, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span> <span> </span><span class="NLM_article-title">Diving into the water: inducible binding conformations for BRD4, TAF1(2), BRD9, and CECR2 bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">5391</span>– <span class="NLM_lpage">5402</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00264</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00264" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC28XosFSnsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5391-5402&author=T.+D.+Crawfordauthor=V.+Tsuiauthor=E.+M.+Flynnauthor=S.+Wangauthor=A.+M.+Taylorauthor=A.+C%C3%B4t%C3%A9author=J.+E.+Audiaauthor=M.+H.+Beresiniauthor=D.+J.+Burdickauthor=R.+Cummingsauthor=L.+A.+Dakinauthor=M.+Duplessisauthor=A.+C.+Goodauthor=M.+C.+Hewittauthor=H.-R.+Huangauthor=H.+Jayaramauthor=J.+R.+Kieferauthor=Y.+Jiangauthor=J.+Murrayauthor=C.+G.+Nasveschukauthor=E.+Pardoauthor=F.+Poyauthor=F.+A.+Romeroauthor=Y.+Tangauthor=J.+Wangauthor=Z.+Xuauthor=L.+E.+Zawadzkeauthor=X.+Zhuauthor=B.+K.+Albrechtauthor=S.+R.+Magnusonauthor=S.+Bellonauthor=A.+G.+Cochran&title=Diving+into+the+water%3A+inducible+binding+conformations+for+BRD4%2C+TAF1%282%29%2C+BRD9%2C+and+CECR2+bromodomains&doi=10.1021%2Facs.jmedchem.6b00264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Diving into the Water: Inducible Binding Conformations for BRD4, TAF1(2), BRD9, and CECR2 Bromodomains</span></div><div class="casAuthors">Crawford, Terry D.; Tsui, Vickie; Flynn, E. Megan; Wang, Shumei; Taylor, Alexander M.; Cote, Alexandre; Audia, James E.; Beresini, Maureen H.; Burdick, Daniel J.; Cummings, Richard; Dakin, Les A.; Duplessis, Martin; Good, Andrew C.; Hewitt, Michael C.; Huang, Hon-Ren; Jayaram, Hariharan; Kiefer, James R.; Jiang, Ying; Murray, Jeremy; Nasveschuk, Christopher G.; Pardo, Eneida; Poy, Florence; Romero, F. Anthony; Tang, Yong; Wang, Jian; Xu, Zhaowu; Zawadzke, Laura E.; Zhu, Xiaoyu; Albrecht, Brian K.; Magnuson, Steven R.; Bellon, Steve; Cochran, Andrea G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5391-5402</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The biol. role played by non-BET bromodomains remains poorly understood, and it is therefore imperative to identify potent and highly selective inhibitors to effectively explore the biol. of individual bromodomain proteins.  A ligand-efficient nonselective bromodomain inhibitor was identified from a 6-Me pyrrolopyridone fragment.  Small hydrophobic substituents replacing the N-Me group were designed directing toward the conserved bromodomain water pocket, and two distinct binding conformations were then obsd.  The substituents either directly displaced and rearranged the conserved solvent network, as in BRD4(1) and TAF1(2), or induced a narrow hydrophobic channel adjacent to the lipophilic shelf, as in BRD9 and CECR2.  The preference of distinct substituents for individual bromodomains provided selectivity handles useful for future lead optimization efforts for selective BRD9, CECR2, and TAF1(2) inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTLnuVCIKdHbVg90H21EOLACvtfcHk0lhXnaq4sOrNpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosFSnsbs%253D&md5=bf92e94c5fd3ee36bd8441294ca85185</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00264%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3DBurdick%26aufirst%3DD.%2BJ.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DDuplessis%26aufirst%3DM.%26aulast%3DGood%26aufirst%3DA.%2BC.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DHuang%26aufirst%3DH.-R.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZawadzke%26aufirst%3DL.%2BE.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DMagnuson%26aufirst%3DS.%2BR.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26atitle%3DDiving%2520into%2520the%2520water%253A%2520inducible%2520binding%2520conformations%2520for%2520BRD4%252C%2520TAF1%25282%2529%252C%2520BRD9%252C%2520and%2520CECR2%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5391%26epage%3D5402%26doi%3D10.1021%2Facs.jmedchem.6b00264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodroofe, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwinn, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Méndez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">NanoBRET--a novel BRET platform for the analysis of protein-protein interactions</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1797</span>– <span class="NLM_lpage">1804</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00143</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00143" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovFKkt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1797-1804&author=T.+Machleidtauthor=C.+C.+Woodroofeauthor=M.+K.+Schwinnauthor=J.+M%C3%A9ndezauthor=M.+B.+Robersauthor=K.+Zimmermanauthor=P.+Ottoauthor=D.+L.+Danielsauthor=T.+A.+Kirklandauthor=K.+V.+Wood&title=NanoBRET%2D%2Da+novel+BRET+platform+for+the+analysis+of+protein-protein+interactions&doi=10.1021%2Facschembio.5b00143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">NanoBRET-A Novel BRET Platform for the Analysis of Protein-Protein Interactions</span></div><div class="casAuthors">Machleidt, Thomas; Woodroofe, Carolyn C.; Schwinn, Marie K.; Mendez, Jacqui; Robers, Matthew B.; Zimmerman, Kris; Otto, Paul; Daniels, Danette L.; Kirkland, Thomas A.; Wood, Keith V.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1797-1804</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dynamic interactions between proteins comprise a key mechanism for temporal control of cellular function and thus hold promise for development of novel drug therapies.  It remains tech. challenging, however, to quant. characterize these interactions within the biol. relevant context of living cells.  Although, bioluminescence resonance energy transfer (BRET) has often been used for this purpose, its general applicability has been hindered by limited sensitivity and dynamic range.  We have addressed this by combining an extremely bright luciferase (Nanoluc) with a means for tagging intracellular proteins with a long-wavelength fluorophore (HaloTag).  The small size (19 kDa), high emission intensity, and relatively narrow spectrum (460 nm peak intensity) make Nanoluc luciferase well suited as an energy donor.  By selecting an efficient red-emitting fluorophore (635 nm peak intensity) for attachment onto the HaloTag, an overall spectral sepn. exceeding 175 nm was achieved.  This combination of greater light intensity with improved spectral resoln. results in substantially increased detection sensitivity and dynamic range over current BRET technologies.  Enhanced performance is demonstrated using several established model systems, as well as the ability to image BRET in individual cells.  The capabilities are further exhibited in a novel assay developed for analyzing the interactions of bromodomain proteins with chromatin in living cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6-h9HlReNabVg90H21EOLACvtfcHk0lhXnaq4sOrNpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovFKkt7c%253D&md5=9f5f773b6cd14c40074a2ffbd6472008</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00143%26sid%3Dliteratum%253Aachs%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DWoodroofe%26aufirst%3DC.%2BC.%26aulast%3DSchwinn%26aufirst%3DM.%2BK.%26aulast%3DM%25C3%25A9ndez%26aufirst%3DJ.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DNanoBRET--a%2520novel%2520BRET%2520platform%2520for%2520the%2520analysis%2520of%2520protein-protein%2520interactions%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1797%26epage%3D1804%26doi%3D10.1021%2Facschembio.5b00143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niesen, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span> <span> </span><span class="NLM_article-title">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">2212</span>– <span class="NLM_lpage">2221</span>, <span class="refDoi"> DOI: 10.1038/nprot.2007.321</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1038%2Fnprot.2007.321" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=17853878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2212-2221&author=F.+H.+Niesenauthor=H.+Berglundauthor=M.+Vedadi&title=The+use+of+differential+scanning+fluorimetry+to+detect+ligand+interactions+that+promote+protein+stability&doi=10.1038%2Fnprot.2007.321"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span></div><div class="casAuthors">Niesen, Frank H.; Berglund, Helena; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2212-2221</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Differential scanning fluorometry (DSF) is a rapid and inexpensive screening method to identify low-mol.-wt. ligands that bind and stabilize purified proteins.  The temp. at which a protein unfolds is measured by an increase in the fluorescence of a dye with affinity for hydrophobic parts of the protein, which are exposed as the protein unfolds.  A simple fitting procedure allows quick calcn. of the transition midpoint; the difference in the temp. of this midpoint in the presence and absence of ligand is related to the binding affinity of the small mol., which can be a low-mol.-wt. compd., a peptide or a nucleic acid.  DSF is best performed using a conventional real-time PCR instrument.  Ligand solns. from a storage plate are added to a soln. of protein and dye, distributed into the wells of the PCR plate and fluorescence intensity measured as the temp. is raised gradually.  Results can be obtained in a single day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomViqp6Tq7krVg90H21EOLACvtfcHk0lhXnaq4sOrNpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N&md5=8a74dbb348ab81179614ab4ae5797d94</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.321%26sid%3Dliteratum%253Aachs%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DBerglund%26aufirst%3DH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DThe%2520use%2520of%2520differential%2520scanning%2520fluorimetry%2520to%2520detect%2520ligand%2520interactions%2520that%2520promote%2520protein%2520stability%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D2%26spage%3D2212%26epage%3D2221%26doi%3D10.1038%2Fnprot.2007.321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickle, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D.</span></span> <span> </span><span class="NLM_article-title">BROMOscan a high throughput, quantitative ligand binding platform identifies best-in-class bromodomain inhibitors from a screen of mature compounds targeting other protein classes</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">4238</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.am2013-4238</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1158%2F1538-7445.am2013-4238" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=4238&author=E.+Quinnauthor=L.+Wodickaauthor=P.+Ciceriauthor=G.+Pallaresauthor=E.+Pickleauthor=A.+Torreyauthor=M.+Floydauthor=J.+Huntauthor=D.+Treiber&title=BROMOscan+a+high+throughput%2C+quantitative+ligand+binding+platform+identifies+best-in-class+bromodomain+inhibitors+from+a+screen+of+mature+compounds+targeting+other+protein+classes&doi=10.1158%2F1538-7445.am2013-4238"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.am2013-4238&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.am2013-4238%26sid%3Dliteratum%253Aachs%26aulast%3DQuinn%26aufirst%3DE.%26aulast%3DWodicka%26aufirst%3DL.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPickle%26aufirst%3DE.%26aulast%3DTorrey%26aufirst%3DA.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%26aulast%3DTreiber%26aufirst%3DD.%26atitle%3DBROMOscan%2520a%2520high%2520throughput%252C%2520quantitative%2520ligand%2520binding%2520platform%2520identifies%2520best-in-class%2520bromodomain%2520inhibitors%2520from%2520a%2520screen%2520of%2520mature%2520compounds%2520targeting%2520other%2520protein%2520classes%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D4238%26doi%3D10.1158%2F1538-7445.am2013-4238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1016/s0076-6879(97)76066-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode&doi=10.1016%2Fs0076-6879%2897%2976066-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lgGKtsnIO5F8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2Fs0076-6879%2897%2976066-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Xds</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2Fs0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lgGKtsnIO5F8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2Fs0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/s0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2Fs0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0li2HVD5MgW7IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2Fs0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/s0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2Fs0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0li2HVD5MgW7IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2Fs0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustyakimov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urzhumtsev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">Towards automated crystallographic structure refinement with phenix.refine</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">352</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1107/s0907444912001308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1107%2Fs0907444912001308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=22505256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=352-367&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=N.+Echolsauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=M.+Mustyakimovauthor=T.+C.+Terwilligerauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=Towards+automated+crystallographic+structure+refinement+with+phenix.refine&doi=10.1107%2Fs0907444912001308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Towards automated crystallographic structure refinement with phenix.refine</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Echols, Nathaniel; Headd, Jeffrey J.; Moriarty, Nigel W.; Mustyakimov, Marat; Terwilliger, Thomas C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">352-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.refine is a program within the PHENIX package that supports crystallog. structure refinement against exptl. data with a wide range of upper resoln. limits using a large repertoire of model parameterizations.  It has several automation features and is also highly flexible.  Several hundred parameters enable extensive customizations for complex use cases.  Multiple user-defined refinement strategies can be applied to specific parts of the model in a single refinement run.  An intuitive graphical user interface is available to guide novice users and to assist advanced users in managing refinement projects.  X-ray or neutron diffraction data can be used sep. or jointly in refinement. phenix.refine is tightly integrated into the PHENIX suite, where it serves as a crit. component in automated model building, final structure refinement, structure validation and deposition to the wwPDB.  This paper presents an overview of the major phenix.refine features, with extensive literature refs. for readers interested in more detailed discussions of the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnhn9QwIEkTbVg90H21EOLACvtfcHk0li2HVD5MgW7IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D&md5=1cc7da2cea396f1a75646ea709140cf1</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2Fs0907444912001308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444912001308%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DMustyakimov%26aufirst%3DM.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DTowards%2520automated%2520crystallographic%2520structure%2520refinement%2520with%2520phenix.refine%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D352%26epage%3D367%26doi%3D10.1107%2Fs0907444912001308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lj6H5kcKZZIhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomenius, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munchhof, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klaus, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riera, T. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span> <span> </span><span class="NLM_article-title">Novel oxindole sulfonamides and sulfamides: EPZ031686, the first orally bioavailable small molecule SMYD3 inhibitor</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">134</span>– <span class="NLM_lpage">138</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00272</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00272" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVahsrnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=134-138&author=L.+H.+Mitchellauthor=P.+A.+Boriack-Sjodinauthor=S.+Smithauthor=M.+Thomeniusauthor=N.+Riouxauthor=M.+Munchhofauthor=J.+E.+Millsauthor=C.+Klausauthor=J.+Totmanauthor=T.+V.+Rieraauthor=A.+Raimondiauthor=S.+L.+Jacquesauthor=K.+Westauthor=M.+Foleyauthor=N.+J.+Watersauthor=K.+W.+Kuntzauthor=T.+J.+Wigleauthor=M.+P.+Scottauthor=R.+A.+Copelandauthor=J.+J.+Smithauthor=R.+Chesworth&title=Novel+oxindole+sulfonamides+and+sulfamides%3A+EPZ031686%2C+the+first+orally+bioavailable+small+molecule+SMYD3+inhibitor&doi=10.1021%2Facsmedchemlett.5b00272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Oxindole Sulfonamides and Sulfamides: EPZ031686, the First Orally Bioavailable Small Molecule SMYD3 Inhibitor</span></div><div class="casAuthors">Mitchell, Lorna H.; Boriack-Sjodin, P. Ann; Smith, Sherri; Thomenius, Michael; Rioux, Nathalie; Munchhof, Michael; Mills, James E.; Klaus, Christine; Totman, Jennifer; Riera, Thomas V.; Raimondi, Alejandra; Jacques, Suzanne L.; West, Kip; Foley, Megan; Waters, Nigel J.; Kuntz, Kevin W.; Wigle, Tim J.; Scott, Margaret Porter; Copeland, Robert A.; Smith, Jesse J.; Chesworth, Richard</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">134-138</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SMYD3 has been implicated in a range of cancers; however, until now no potent selective small mol. inhibitors have been available for target validation studies.  A novel oxindole series of SMYD3 inhibitors was identified through screening of the Epizyme proprietary histone methyltransferase-biased library.  Potency optimization afforded two tool compds., sulfonamide EPZ031686 and sulfamide EPZ030456, with cellular potency at a level sufficient to probe the in vitro biol. of SMYD3 inhibition.  EPZ031686 shows good bioavailability following oral dosing in mice making it a suitable tool for potential in vivo target validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQtO_uOhxeYrVg90H21EOLACvtfcHk0lj6H5kcKZZIhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVahsrnJ&md5=3fd7612cb086796a0bcc5c6e0c1f1352</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00272%26sid%3Dliteratum%253Aachs%26aulast%3DMitchell%26aufirst%3DL.%2BH.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DSmith%26aufirst%3DS.%26aulast%3DThomenius%26aufirst%3DM.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DMunchhof%26aufirst%3DM.%26aulast%3DMills%26aufirst%3DJ.%2BE.%26aulast%3DKlaus%26aufirst%3DC.%26aulast%3DTotman%26aufirst%3DJ.%26aulast%3DRiera%26aufirst%3DT.%2BV.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DWest%26aufirst%3DK.%26aulast%3DFoley%26aufirst%3DM.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26atitle%3DNovel%2520oxindole%2520sulfonamides%2520and%2520sulfamides%253A%2520EPZ031686%252C%2520the%2520first%2520orally%2520bioavailable%2520small%2520molecule%2520SMYD3%2520inhibitor%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D134%26epage%3D138%26doi%3D10.1021%2Facsmedchemlett.5b00272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6J3O" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6J3O','PDB','6J3O'); return false;">PDB: 6J3O</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6J3P" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6J3P','PDB','6J3P'); return false;">PDB: 6J3P</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i80"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00096">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_01315"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00096">10.1021/acs.jmedchem.9b00096</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Flowchart of hits identification against PCAF BRD; DSF results of test compounds against a panel of 12 BRDs; BROMOscan assay of (<i>R</i>,<i>R</i>)-<b>36n</b> against 32 BRDs; kinase inhibition profiles of (<i>R</i>,<i>R</i>)-<b>24a</b>, <i>endo</i>-<b>24c</b> and (<i>R</i>,<i>R</i>)-<b>36n</b>; ITC curves of selected compounds against PCAF; X-ray data collection and refinement statistics; <sup>1</sup>H spectra, <sup>13</sup>C NMR spectra; HRMS spectra; and chiral HPLC traces (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00096/suppl_file/jm9b00096_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00096/suppl_file/jm9b00096_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00096/suppl_file/jm9b00096_si_001.pdf">jm9b00096_si_001.pdf (5.02 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00096/suppl_file/jm9b00096_si_002.csv">jm9b00096_si_002.csv (2.47 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates and structure factors have been deposited in the Protein Data Bank under the following accession codes: 6J3O (PCAF/compound <b>12</b>), 6J3P (GCN5/compound (<i>R</i>,<i>R</i>)-<b>36n</b>). Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00096&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00096%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-9" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00096" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799b13319963cb2","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
